
<html lang="en"     class="pb-page"  data-request-id="036ce484-b5f5-43b8-8cfd-88b6ab432c1f"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;article:article:10.1021/acs.jmedchem.5b01630;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;issue:issue:10.1021/jmcmar.2016.59.issue-6;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="dc.Title" content="Structure-Based Optimization of a Small Molecule Antagonist of the Interaction Between WD Repeat-Containing Protein 5 (WDR5) and Mixed-Lineage Leukemia 1 (MLL1)" /></meta><meta name="dc.Creator" content="Matthäus  Getlik" /></meta><meta name="dc.Creator" content="David  Smil" /></meta><meta name="dc.Creator" content="Carlos  Zepeda-Velázquez" /></meta><meta name="dc.Creator" content="Yuri  Bolshan" /></meta><meta name="dc.Creator" content="Gennady  Poda" /></meta><meta name="dc.Creator" content="Hong  Wu" /></meta><meta name="dc.Creator" content="Aiping  Dong" /></meta><meta name="dc.Creator" content="Ekaterina  Kuznetsova" /></meta><meta name="dc.Creator" content="Richard  Marcellus" /></meta><meta name="dc.Creator" content="Guillermo  Senisterra" /></meta><meta name="dc.Creator" content="Ludmila  Dombrovski" /></meta><meta name="dc.Creator" content="Taraneh  Hajian" /></meta><meta name="dc.Creator" content="Taira  Kiyota" /></meta><meta name="dc.Creator" content="Matthieu  Schapira" /></meta><meta name="dc.Creator" content="Cheryl H.  Arrowsmith" /></meta><meta name="dc.Creator" content="Peter J.  Brown" /></meta><meta name="dc.Creator" content="Masoud  Vedadi" /></meta><meta name="dc.Creator" content="Rima  Al-awar" /></meta><meta name="dc.Description" content="WD repeat-containing protein 5 (WDR5) is an important component of the multiprotein complex essential for activating mixed-lineage leukemia 1 (MLL1). Rearrangement of the MLL1 gene is associated wi..." /></meta><meta name="Description" content="WD repeat-containing protein 5 (WDR5) is an important component of the multiprotein complex essential for activating mixed-lineage leukemia 1 (MLL1). Rearrangement of the MLL1 gene is associated wi..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="March 9, 2016" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.5b01630" /></meta><meta name="dc.Language" content="EN" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2016 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.5b01630" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.5b01630" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.5b01630" /></link>
        
    
    

<title>Structure-Based Optimization of a Small Molecule Antagonist of the Interaction Between WD Repeat-Containing Protein 5 (WDR5) and Mixed-Lineage Leukemia 1 (MLL1) | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.5b01630" /></meta><meta property="og:title" content="Structure-Based Optimization of a Small Molecule Antagonist of the Interaction Between WD Repeat-Containing Protein 5 (WDR5) and Mixed-Lineage Leukemia 1 (MLL1)" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01630/20160318/images/large/jm-2015-01630k_0016.jpeg" /></meta><meta property="og:description" content="WD repeat-containing protein 5 (WDR5) is an important component of the multiprotein complex essential for activating mixed-lineage leukemia 1 (MLL1). Rearrangement of the MLL1 gene is associated with onset and progression of acute myeloid and lymphoblastic leukemias, and targeting the WDR5-MLL1 interaction may result in new cancer therapeutics. Our previous work showed that binding of small molecule ligands to WDR5 can modulate its interaction with MLL1, suppressing MLL1 methyltransferase activity. Initial structure–activity relationship studies identified N-(2-(4-methylpiperazin-1-yl)-5-substituted-phenyl) benzamides as potent and selective antagonists of this protein–protein interaction. Guided by crystal structure data and supported by in silico library design, we optimized the scaffold by varying the C-1 benzamide and C-5 substituents. This allowed us to develop the first highly potent (Kdisp &lt; 100 nM) small molecule antagonists of the WDR5-MLL1 interaction and demonstrate that N-(4-(4-methylpiperazin-1-yl)-3′-(morpholinomethyl)-[1,1′-biphenyl]-3-yl)-6-oxo-4-(trifluoromethyl)-1,6-dihydropyridine-3-carboxamide 16d (OICR-9429) is a potent and selective chemical probe suitable to help dissect the biological role of WDR5." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.5b01630"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.5b01630">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.5b01630&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.5b01630&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.5b01630&amp;href=/doi/10.1021/acs.jmedchem.5b01630" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2016</span><span class="cit-fg-volume">, 59</span><span class="cit-fg-issue">, 6</span><span class="cit-fg-pageRange">, 2478-2496</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/59/6" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.5b01609" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.5b01654" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Structure-Based Optimization of a Small Molecule Antagonist of the Interaction Between WD Repeat-Containing Protein 5 (WDR5) and Mixed-Lineage Leukemia 1 (MLL1)</span></h1><ul class="loa non-jats-loa"><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Matth%C3%A4us++Getlik">Matthäus Getlik</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=David++Smil">David Smil</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Carlos++Zepeda-Vel%C3%A1zquez">Carlos Zepeda-Velázquez</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Yuri++Bolshan">Yuri Bolshan</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Gennady++Poda">Gennady Poda</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>∥</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Hong++Wu">Hong Wu</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Aiping++Dong">Aiping Dong</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Ekaterina++Kuznetsova">Ekaterina Kuznetsova</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Richard++Marcellus">Richard Marcellus</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Guillermo++Senisterra">Guillermo Senisterra</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Ludmila++Dombrovski">Ludmila Dombrovski</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Taraneh++Hajian">Taraneh Hajian</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Taira++Kiyota">Taira Kiyota</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Matthieu++Schapira">Matthieu Schapira</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Cheryl+H.++Arrowsmith">Cheryl H. Arrowsmith</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Peter+J.++Brown">Peter J. Brown</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Masoud++Vedadi">Masoud Vedadi</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Rima++Al-awar">Rima Al-awar</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span></span></li><span class="comma-separator"></span></ul><div class="affiliations"><div class="view-affs"><span class="title">View Author Information</span><i class="caret-icon icon-angle-down"></i></div><div class="all-aff-infos hlFld-Affiliation"><div class="aff-info" id="aff1"><span class="aff-symbol">†</span> <span class="aff-text">Drug Discovery Program, Ontario Institute for Cancer Research, 661 University Avenue, MaRS Centre, West Tower, Toronto, Ontario M5G 0A3, Canada</span></div><div class="aff-info" id="aff2"><span class="aff-symbol">‡</span> <span class="aff-text">Structural Genomics Consortium, University of Toronto, 101 College Street, MaRS Centre, South Tower, Toronto, Ontario M5G 1L7, Canada</span></div><div class="aff-info" id="aff3"><span class="aff-symbol">∥</span> <span class="aff-text">Leslie Dan Faculty of Pharmacy, University of Toronto, 144 College Street, Toronto, Ontario M5S 3M2, Canada</span></div><div class="aff-info" id="aff4"><span class="aff-symbol">§</span> <span class="aff-text">Department of Pharmacology and Toxicology, University of Toronto, Medical Sciences Building, 1 King’s College Circle, Toronto, Ontario M5S 1A8, Canada</span></div><div class="corresp-info"><strong>*</strong>Phone: +1 (416) 673-8577. E-mail: <a href="/cdn-cgi/l/email-protection#99cbf0f4f8b7d8f5f8eef8ebd9f6f0faebb7f6f7b7faf8"><span class="__cf_email__" data-cfemail="b0e2d9ddd19ef1dcd1c7d1c2f0dfd9d3c29edfde9ed3d1">[email protected]</span></a></div></div></div><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.5b01630&amp;href=/doi/10.1021%2Facs.jmedchem.5b01630" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2016</span></span><span class="cit-volume">, 59</span><span class="cit-issue">, 6</span><span class="cit-pageRange">, 2478–2496</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">March 9, 2016</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>16 October 2015</li><li><span class="item_label"><b>Published</b> online</span>9 March 2016</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 24 March 2016</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.5b01630" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.5b01630</a></div><div class="article_header-article-copyright"><strong>Copyright © 2016 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D2478%26pageCount%3D19%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DMatth%25C3%25A4us%2BGetlik%252C%2BDavid%2BSmil%252C%2BCarlos%2BZepeda-Vel%25C3%25A1zquez%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D59%26issueNum%3D6%26contentID%3Dacs.jmedchem.5b01630%26title%3DStructure-Based%2BOptimization%2Bof%2Ba%2BSmall%2BMolecule%2BAntagonist%2Bof%2Bthe%2BInteraction%2BBetween%2BWD%2BRepeat-Containing%2BProtein%2B5%2B%2528WDR5%2529%2Band%2BMixed-Lineage%2BLeukemia%2B1%2B%2528MLL1%2529%26numPages%3D19%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D2496%26publicationDate%3DMarch%2B2016">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.5b01630"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">2978</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">38</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.5b01630" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Structure-Based Optimization of a Small Molecule Antagonist of the Interaction Between WD Repeat-Containing Protein 5 (WDR5) and Mixed-Lineage Leukemia 1 (MLL1)&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Matthäus&quot;,&quot;last_name&quot;:&quot;Getlik&quot;},{&quot;first_name&quot;:&quot;David&quot;,&quot;last_name&quot;:&quot;Smil&quot;},{&quot;first_name&quot;:&quot;Carlos&quot;,&quot;last_name&quot;:&quot;Zepeda-Velázquez&quot;},{&quot;first_name&quot;:&quot;Yuri&quot;,&quot;last_name&quot;:&quot;Bolshan&quot;},{&quot;first_name&quot;:&quot;Gennady&quot;,&quot;last_name&quot;:&quot;Poda&quot;},{&quot;first_name&quot;:&quot;Hong&quot;,&quot;last_name&quot;:&quot;Wu&quot;},{&quot;first_name&quot;:&quot;Aiping&quot;,&quot;last_name&quot;:&quot;Dong&quot;},{&quot;first_name&quot;:&quot;Ekaterina&quot;,&quot;last_name&quot;:&quot;Kuznetsova&quot;},{&quot;first_name&quot;:&quot;Richard&quot;,&quot;last_name&quot;:&quot;Marcellus&quot;},{&quot;first_name&quot;:&quot;Guillermo&quot;,&quot;last_name&quot;:&quot;Senisterra&quot;},{&quot;first_name&quot;:&quot;Ludmila&quot;,&quot;last_name&quot;:&quot;Dombrovski&quot;},{&quot;first_name&quot;:&quot;Taraneh&quot;,&quot;last_name&quot;:&quot;Hajian&quot;},{&quot;first_name&quot;:&quot;Taira&quot;,&quot;last_name&quot;:&quot;Kiyota&quot;},{&quot;first_name&quot;:&quot;Matthieu&quot;,&quot;last_name&quot;:&quot;Schapira&quot;},{&quot;first_name&quot;:&quot;Cheryl&quot;,&quot;last_name&quot;:&quot;H. Arrowsmith&quot;},{&quot;first_name&quot;:&quot;Peter&quot;,&quot;last_name&quot;:&quot;J. Brown&quot;},{&quot;first_name&quot;:&quot;Masoud&quot;,&quot;last_name&quot;:&quot;Vedadi&quot;},{&quot;first_name&quot;:&quot;Rima&quot;,&quot;last_name&quot;:&quot;Al-awar&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2016&quot;,&quot;month&quot;:&quot;03&quot;,&quot;day&quot;:&quot;09&quot;,&quot;issue&quot;:&quot;6&quot;,&quot;volume&quot;:&quot;59&quot;,&quot;pages&quot;:&quot;2478-2496&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.5b01630&quot;},&quot;abstract&quot;:&quot;WD repeat-containing protein 5 (WDR5) is an important component of the multiprotein complex essential for activating mixed-lineage leukemia 1 (MLL1). Rearrangement of the MLL1 gene is associated with onset and progression of acute myeloid and lymphoblastic leukemias, and targeting the WDR5-MLL1 interaction may result in new cancer therapeutics. Our previous work showed that binding of small molecule ligands to WDR5 can modulate its interaction with MLL1, suppressing MLL1 methyltransferase activity. Initial structure–activity relationship studies identified N-(2-(4-methylpiperazin-1-yl)-5-substituted-phenyl) benzamides as potent and selective antagonists of this protein–protein interaction. Guided by crystal structure data and supported by in silico library design, we optimized the scaffold by varying the C-1 benzamide and C-5 substituents. This allowed us to develop the first highly potent (Kdisp &lt; 100 nM) small molecule antagonists of the WDR5-MLL1 interaction and demonstrate that N-(4-(4-methylpiperazin-1-y&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.5b01630&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.5b01630" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.5b01630&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.5b01630" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.5b01630&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.5b01630" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.5b01630&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.5b01630&amp;href=/doi/10.1021/acs.jmedchem.5b01630" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.5b01630" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.5b01630" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (3 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.5b01630&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.5b01630%26sid%3Dliteratum%253Aachs%26pmid%3D26958703%26genre%3Darticle%26aulast%3DGetlik%26date%3D2016%26atitle%3DStructure-Based%2BOptimization%2Bof%2Ba%2BSmall%2BMolecule%2BAntagonist%2Bof%2Bthe%2BInteraction%2BBetween%2BWD%2BRepeat-Containing%2BProtein%2B5%2B%2528WDR5%2529%2Band%2BMixed-Lineage%2BLeukemia%2B1%2B%2528MLL1%2529%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D59%26issue%3D6%26spage%3D2478%26epage%3D2496%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (2)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=291171" title="Antagonists">Antagonists</a>,</li><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=290676" title="Substituents">Substituents</a>,</li><li><a href="/action/doSearch?ConceptID=292529" title="Aromatic compounds">Aromatic compounds</a>,</li><li><a href="/action/doSearch?ConceptID=291352" title="Screening assays">Screening assays</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/59/6" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/jmcmar.2016.59.issue-6/20160324/jmcmar.2016.59.issue-6.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="f_tgr1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01630/20160318/images/medium/jm-2015-01630k_0016.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01630/20160318/images/large/jm-2015-01630k_0016.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01630&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">WD repeat-containing protein 5 (WDR5) is an important component of the multiprotein complex essential for activating mixed-lineage leukemia 1 (MLL1). Rearrangement of the MLL1 gene is associated with onset and progression of acute myeloid and lymphoblastic leukemias, and targeting the WDR5-MLL1 interaction may result in new cancer therapeutics. Our previous work showed that binding of small molecule ligands to WDR5 can modulate its interaction with MLL1, suppressing MLL1 methyltransferase activity. Initial structure–activity relationship studies identified <i>N</i>-(2-(4-methylpiperazin-1-yl)-5-substituted-phenyl) benzamides as potent and selective antagonists of this protein–protein interaction. Guided by crystal structure data and supported by in silico library design, we optimized the scaffold by varying the C-1 benzamide and C-5 substituents. This allowed us to develop the first highly potent (<i>K</i><sub>disp</sub> < 100 nM) small molecule antagonists of the WDR5-MLL1 interaction and demonstrate that <i>N</i>-(4-(4-methylpiperazin-1-yl)-3′-(morpholinomethyl)-[1,1′-biphenyl]-3-yl)-6-oxo-4-(trifluoromethyl)-1,6-dihydropyridine-3-carboxamide <b>16d</b> (OICR-9429) is a potent and selective chemical probe suitable to help dissect the biological role of WDR5.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i1">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_15284" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_15284" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Mixed-lineage leukemia 1 (MLL1) is a member of the Set1 family of methyl transferases which catalyze the mono-, di-, and trimethylation of histone 3 at lysine 4 (H3K4) and facilitate transcriptional initiation. MLL1 is often deregulated in acute myeloid and lymphoblastic leukemias with a common defect being a gene translocation involving chromosome 11, leading to fusion of the MLL1 gene with other genes. The resulting MLL1 fusion proteins induce leukemia by constitutively activating the expression of HOXA9 and MEIS1, genes that are essential for self-renewal of hematopoietic stem cells and whose expression is usually down-regulated after differentiation.<a onclick="showRef(event, 'ref1 ref2'); return false;" href="javascript:void(0);" class="ref ref1 ref2">(1, 2)</a></div><div class="NLM_p">The WD repeat-containing protein 5 (WDR5) binds to MLL1 and has been shown to be crucial for the stability and methyltransferase activity of the MLL1 complex (MLL1, RbBP5, Ash2L, WDR5, and DPY-30). In the absence of any of these proteins, the catalytic activity of the core complex is significantly diminished.<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> WDR5 belongs to a family of proteins that consist of four or more 40 amino acid repeats forming a circular β-propeller structure, usually ending with a tryptophan-aspartic acid peptide (WD).<a onclick="showRef(event, 'ref4 ref5'); return false;" href="javascript:void(0);" class="ref ref4 ref5">(4, 5)</a> WDR5 was reported to facilitate H3K4 methylation by MLL1 through binding to the N-terminal region of histone H3 via key interactions with arginine 2. Interestingly, the conserved WDR5-interacting (WIN) motif of MLL1 binds to the same binding site in WDR5, with R3765 forming a dense hydrogen-bond network to the surrounding amino acid residues.<a onclick="showRef(event, 'ref6 ref7'); return false;" href="javascript:void(0);" class="ref ref6 ref7">(6, 7)</a></div><div class="NLM_p">Given the critical role of WDR5 within the MLL1 multiprotein complex, antagonists of the WDR5-MLL1 interaction could serve to dissociate the complex and inhibit MLL1 activity. Well-characterized antagonists of such interaction would be valuable tools for advancing our understanding of the roles of this complex and the implications of MLL1 dysregulation in cancers. To that end, the small molecule inhibitor <b>1</b> (SCF-I2, <a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>)<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> along with the linear peptide <b>2</b> (MM-101, <a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>)<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> and cyclic peptide <b>3</b> (MM-401, <a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>)<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> have been reported, with the peptides shown to directly bind the arginine binding site of WDR5 in order to disrupt its interaction with MLL1.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01630/20160318/images/medium/jm-2015-01630k_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01630/20160318/images/large/jm-2015-01630k_0001.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Chemical structures of reported WD repeat domain antagonists. (A) Racemic 1,1′-binaphthyl-2,2′-dicarboxylic acid, an inhibitor of yeast Cdc4. (B) Acyclic and cyclic peptide inhibitors that occupy the arginine binding site of WDR5.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01630/20160318/images/large/jm-2015-01630k_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01630&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Compound <b>3</b> was reported as having a 0.9 nM binding affinity to WDR5 as determined using a fluorescence polarization (FP) assay and to inhibit the MLL1 methyltransferase activity (IC<sub>50</sub> = 0.32 μM).<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> When tested on human leukemia cell lines, the compound was found to arrest cell growth of MV4:11, Molm13, and KOPN8 lines with a GI<sub>50</sub> of 12.42, 23.97, and 29.73 μM respectively.</div><div class="NLM_p">We had previously identified the first nonpeptide small molecule antagonists of the WDR5-MLL1 interaction<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> and have reported initial structure-activity relationship (SAR) findings for the <i>N</i>-(2-(4-methylpiperazin-1-yl)-5-substituted-phenyl) benzamide scaffold.<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> Optimization of this scaffold for potency, solubility, and selectivity resulted in the discovery of <i>N</i>-(4-(4-methylpiperazin-1-yl)-3′-(morpholinomethyl)-[1,1′-biphenyl]-3-yl)-6-oxo-4-(trifluoromethyl)-1,6-dihydropyridine-3-carboxamide <b>16d</b> (OICR-9429), the most potent small molecule antagonist of the WDR5-MLL1 interaction reported to-date, and one built on a chemotype which can potentially address the liabilities commonly associated with peptide-based compounds.<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a></div><div class="NLM_p last">Compound <b>16d</b> has significant biological activity in two separate cancer models that are dependent on WDR5/MLL1 activity. The mutated form (p30) of C/EBPα, a transcription factor that is a master regulator of myeloid gene expression programs in the hematopoietic system, interacts preferentially with WDR5, and cells with the C/EBPαp30 mutation are sensitive to <b>16d</b> as well as to WDR5 knock down.<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> Significantly, C/EBPα mutations are present in 9% of patients with acute myeloid leukemia (AML). In a separate study, gain-of-function (GOF) p53 mutants were found to upregulate chromatin regulatory genes including MLL1, and breast cancer cells with a GOF p53 mutation were sensitive to MLL1 knockdown as well as <b>16d</b>.<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> More recently, <b>16d</b> was shown to be effective at reducing N-Myc/WDR5 complex formation, N-Myc target gene expression, and neuroblastomal cell growth, thereby identifying WDR5 as a key cofactor for N-Myc-regulated transcriptional activation and tumorigenesis and as a novel therapeutic target for MYCN-amplified neuroblastomas.<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> Here, we present the structure-based design, synthesis, and biochemical evaluation of <b>16d</b> and the intermediate compounds leading to its development.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_16501" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_16501" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i3" class="anchor-spacer"></div><h3 class="article-section__title" id="_i3"> Compound Synthesis and Structure–Activity Relationship Studies</h3><div class="NLM_p">Having previously leveraged the WDR5-<b>4</b> co-crystal structure (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3SMR">3SMR</a>)<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> to guide the optimization of our initial screening hit <b>4</b>, an antagonist of the WDR5-MLL1 interaction with a <i>K</i><sub>disp</sub> = 7 ± 1 μM, we reported the development of advanced compound <b>5</b> which exhibited a 25-fold improvement in potency (<a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>).<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> While key structural features such as the central phenyl ring, C-1 position amide, C-2 position 4-methylpiperazine, and C-5 position nitro group were maintained following the initial SAR study, introduction of a 3-methyl and 4-fluoro substituent into the benzamide portion of the scaffold was critical for improving potency to <i>K</i><sub>disp</sub> = 0.27 ± 0.1 μM. A co-crystal structure of WDR5-<b>5</b> (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4IA9">4IA9</a>)<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> revealed that crucial interactions of the <i>N</i>-(2-(4-methylpiperazin-1-yl)-5-nitrophenyl) benzamide scaffold with WDR5 were maintained, and the newly introduced methyl and fluoro substituents added to the potency by interacting with the surrounding hydrophobic side chains of A47, A65, Y260, I305, and L321.</div><div class="NLM_p">From this initial work, as well as additional experiments,<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> it was apparent that even minor modifications of the 4-methylpiperazine at C-2 were not tolerated, consistent with it being a key anchor within the arginine-binding cleft of WDR5. Similarly, given the critical hydrogen bond (H-bond) formed between the C-1 amide N–H and the S91 side chain as evident in the available WDR5-<b>5</b> co-crystal structure, modifications such as reversal of the amide, methylation of the amide N–H, and replacement of the amide with a urea or sulfonamide all resulted in a complete loss of activity.<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> We therefore decided to preserve the parent 4-methylpiperazine substituent and amide linkage during all of the SAR work reported in this publication. However, we aimed to fully explore a range of substituents at C-5 and see if additional changes to the benzamide moiety could further improve the potency.</div><div class="NLM_p">The initial published SAR efforts at the C-5 position indicated that introduction of aromatic heterocycles such as 2-furanyl or 4-pyridyl led to modest improvements in potency.<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> The co-crystal structure of WDR5 in complex with <b>5</b> revealed that this position is partially solvent exposed and that the C-5 nitro substituent forms favorable van der Waals interactions with nearby residues F133, Y191 and P173 (<a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>C).</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01630/20160318/images/medium/jm-2015-01630k_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01630/20160318/images/large/jm-2015-01630k_0002.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Chemical structures and binding mode of WDR5-MLL1 interaction antagonists. (A) Initial medium-throughput screening hit compound <b>4</b>. (B) Advanced antagonist <b>5</b> obtained following an initial round of SAR studies. (C) Crystal structure of WDR5 (gray) in complex with <b>5</b> (green; PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4IA9">4IA9</a>). The antagonist resides in the MLL1 arginine-binding pocket and forms a direct H-bond to the side chain of S91 as well as water mediated H-bonds to the backbone of C261. Substituents on the benzamide add hydrophobic interactions with aliphatic side chains surrounding a hydrophobic subpocket.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01630/20160318/images/large/jm-2015-01630k_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01630&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Given that the nitro group potentially reduces cellular permeability by contributing significantly to the compound’s polar surface area (PSA), and because of its potential in vivo liability, we felt that this part of the molecule should be more broadly explored through generation of a focused compound collection designed to not only replace the nitro group but to also capitalize on the potential van der Waals interactions with the aforementioned aromatic residues F133 and Y191 as well as F149. With this as the goal, we introduced bulkier aromatic residues at C-5 with the intention of capturing some interactions with these aromatic side chains. In addition, the incorporation of substituents that would serve to increase overall compound solubility was considered. In order to generate the compound set, an efficient synthetic route was established which allowed for the direct functionalization of the C-5 position via Suzuki coupling as the final step (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>).</div><figure id="sch1" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01630/20160318/images/medium/jm-2015-01630k_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01630/20160318/images/large/jm-2015-01630k_0005.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of 5-Substituted 2-(4-Methylpiperazin-1-yl)phenyl)-3-methylbenzamides<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01630/20160318/images/large/jm-2015-01630k_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01630&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) 1-methylpiperazine, DIPEA, DMSO, 80 °C; (b) Fe<sub>(s)</sub>, CaCl<sub>2</sub>, water/MeOH (1:1), 100 °C; (c) 3-methylbenzoyl chloride, pyridine, CH<sub>2</sub>Cl<sub>2</sub>, rt; (d) boronic acid/ester, Pd(PPh<sub>3</sub>)<sub>4</sub>, Na<sub>2</sub>CO<sub>3</sub>, toluene/water (1:1), 115 °C.</p></p></figure><div class="NLM_p">The route involved initial preparation of 5-bromo-2-(4-methylpiperazin-1-yl)aniline (<b>7</b>) by substitution of 4-bromo-1-fluoro-2-nitrobenzene (<b>6</b>) with 1-methylpiperazine, followed by iron-mediated reduction of the intermediate 1-(4-bromo-2-nitrophenyl)-4-methylpiperazine. Coupling of <b>7</b> with 3-methylbenzoyl chloride readily afforded the precursor <i>N</i>-(5-bromo-2-(4-methylpiperazin-1-yl)phenyl)-3-methylbenzamide (<b>8</b>). The 3-methyl group on the benzamide had been selected as the fixed parent substituent because it was not expected to interfere with the Suzuki reaction and was among the more potent benzamide substituents found during the initial SAR work.<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> The Suzuki couplings were then carried out on <b>8</b> using standard coupling conditions, allowing for access to a collection of derivatives (<b>9a</b>–<b>w</b>) (<a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>). Subsequently, the affinity of the compounds was evaluated using a pair of fluorescence polarization (FP) assays detecting the displacement of either fluorescein-labeled H3 (1–15) peptide (<i>K</i><sub>D</sub> = 167 ± 22 nM) or 9-Ala-FAM ((Ac)-ARAEVHLRK-(Ahx-Ahx)-K(5,6-FAM)) peptide (<i>K</i><sub>D</sub> = 19 ± 3 nM)<a onclick="showRef(event, 'ref12 ref16'); return false;" href="javascript:void(0);" class="ref ref12 ref16">(12, 16)</a> from the WDR5 binding site.<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> For each experiment, we chose a suitable binding peptide based on our estimation of compound affinity. The H3 (1–15) peptide was used when compound activity was anticipated to be above 200 nM, while the 9-Ala-FAM peptide was employed with compounds expected to have potencies below 200 nM. Binding of these peptides to WDR5 increased the FP signal which was then decreased upon displacement of these peptides by the small molecule antagonists. Reduction of signal was monitored at various compound concentrations, and <i>K</i><sub>displacement</sub> (<i>K</i><sub>disp</sub>) values were calculated by fitting the data to a hyperbolic function as described in the <a href="/doi/suppl/10.1021/acs.jmedchem.5b01630/suppl_file/jm5b01630_si_001.pdf" class="ext-link">Supporting Information</a>.</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Binding Affinities of Various 5-Substituted 2-(4-Methylpiperazin-1-yl)phenyl)-3-methylbenzamides</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01630/20160318/images/medium/jm-2015-01630k_0009.gif" alt="" id="fx1" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01630/20160318/images/medium/jm-2015-01630k_0010.gif" alt="" id="fx2" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup>Table a</sup><p class="last">Average of triplicate measurements.</p></div><div class="footnote" id="t1fn2"><sup>Table b</sup><p class="last">FP measurement made using fluorescein-labeled H3 (1–15) peptide displacement.</p></div></div><div></div></div><div class="NLM_p">Represented within the newly generated set were compounds containing larger heterocyclic substituents at C-5 such as the naphthyl (<b>9b</b>–<b>c</b>), dihydrobenzofuranyl (<b>9d</b>–<b>e</b>), quinolinyl (<b>9g</b>–<b>h</b>), benzothiophenyl (<b>9i</b>–<b>l</b>), and indolyl (<b>9m</b>–<b>o</b>) derivatives. Introduction of these groups generally results in potencies greater than that of the previously reported C-5 nitro compound (<i>K</i><sub>disp</sub> = 1.1 μM),<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> with <i>K</i><sub>disp</sub> values typically under 1 μM. Several exceptions, compounds <b>9b</b>, <b>9g</b>, <b>9i</b>, <b>9m</b>, and <b>9r</b>, all appear to have C-5 substituents featuring an L-shaped orientation with respect to the central phenyl core, resulting in an intramolecular hydrogen clash which likely diminishes the ability of these compounds to adopt the very planar binding mode required for greater potency. Compounds <b>8</b> and <b>9a</b>, functionalized at the C-5 position with relatively small, nonaromatic substituents, also show diminished potency (<i>K</i><sub>disp</sub> = 9.70 and 19.85 μM, respectively), pointing to the benefit of appropriate, larger aromatic groups at this position.</div><div class="NLM_p">To understand the binding contribution and orientation of the viable aromatic substituents, the crystal structures of WDR5 in complex with compounds <b>9d</b>, <b>9e</b>, <b>9h</b>, and <b>9o</b> were solved (PDB codes <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5EAR">5EAR</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5EAP">5EAP</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5EAL">5EAL</a>, and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5EAM">5EAM</a> respectively). Superposition of the resultant structures reveals a high degree of alignment among the various C-5 substituents, particularly with respect to the aromatic ring portion of each moiety which is directly attached to the C-5 position and which interacts in a parallel π-stacking orientation with the side chain of F133 (<a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>).</div><figure id="fig3" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01630/20160318/images/medium/jm-2015-01630k_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01630/20160318/images/large/jm-2015-01630k_0003.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Structural alignment of WDR5 crystal structures in complex with <b>4</b> (gray), <b>9d</b> (green), <b>9e</b> (cyan), <b>9h</b> (pink), and <b>9o</b> (yellow). The inhibitors occupy the MLL1 binding site and form water-mediated H-bonds to residue C261 as well as a direct H-bond to S91. The phenyl rings of the heterocyclic substituents attached at the C-5 position show a high degree of alignment.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01630/20160318/images/large/jm-2015-01630k_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01630&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">This residue also π-stacks in a similar manner with the compound’s central phenyl ring, and taken together, these interactions serve to stabilize the almost planar geometry observed between the two aromatic rings. Furthermore, both the C-5 substituents of <b>9h</b> and <b>9o</b> interact in a T-shape and parallel arrangement with the side chains of F149 and Y191 respectively, presumably accounting for the relatively high potency of these compounds (<i>K</i><sub>disp</sub> = 0.30 and 0.39 μM, respectively). Interestingly, when comparing the potencies of the quinoline-substituted compound <b>9h</b> (<i>K</i><sub>disp</sub> = 0.30 μM) and naphthalene-substituted compound <b>9c</b> (<i>K</i><sub>disp</sub> = 0.88 μM), it is apparent that the electron-deficient quinoline substituent serves to further stabilize this interaction, consistent with previous observations that stacking between electron-poor and electron-rich aromatic rings is beneficial.<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> Introduction of a dihydrobenzofuran substituent at C-5 (compounds <b>9d</b>–<b>e</b>) decreased potency (<i>K</i><sub>disp</sub> = 0.70 and 0.48 μM, respectively), but did confer greater solubility<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> to the compounds when compared to derivatives such as <b>9c</b> and <b>9h</b>. In the case of <b>9d</b>–<b>e</b>, a parallel π-stacking interaction with the side chain of F133 is observed, although the furanyl portions of each substituent point toward the solvent and do not further interact with the protein.</div><div class="NLM_p">Overall, it is apparent that the introduction of a bicyclic heterocycle at the C-5 position of the antagonist core adds favorable aromatic interactions with the nearby side chains of residues Y191, F149, and F133, ultimately resulting in greater binding affinity. Unfortunately, incorporation of these heterocycles often yields poorly soluble compounds, so the introduction of pyrrolidine and morpholine-substituted phenyl rings at C-5 was explored in order to address this issue. The resulting compounds <b>9p</b>–<b>t</b> displayed potencies ranging from <i>K</i><sub>disp</sub> = 0.57 to 15 μM and did show improved solubility<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> relative to other compounds in the series.</div><div class="NLM_p">While attempting to identify C-5 substituents capable of conferring potency and/or solubility to the scaffold, a parallel effort was made to optimize the benzamide moiety. Co-crystal structures of WDR5 in complex with antagonists <b>4</b> and <b>5</b> revealed no direct H-bonding interaction of the benzamide substituents with the protein, and in particular, failure to engage residue D107. In order to identify suitable candidates for replacement of the phenyl derivatives that had been predominantly used at the benzamide position,<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> a chemoinformatics approach was employed. Using Accelrys Pipeline Pilot,<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> we performed an exhaustive in silico enumeration of all (benz)amides accessible using commercially available benzoyl and acid halides, benzoic acids/esters, and heterocyclic carboxylic acids (based on the eMolecules Plus database),<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> with specific consideration given to moieties that could potentially interact with the side chain of D107. The virtual focused library was reduced by applying OICR HTS filters that include calculated physical properties filters and filters to eliminate reactive moieties, entities with more than one carboxyl or acid chloride groups, and reagents from suppliers with a long lead time. About 1200 acyl halides and 9000 acids/esters were first enumerated and then docked into the WDR5-<b>4</b> co-crystal structure (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3SMR">3SMR</a>) as their <i>N</i>-(2-(4-methylpiperazin-1-yl)-5-nitrophenyl) amides using the Glide docking program.<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> The obtained docking scores were then used to rank the top 2000 compounds, of which 50 broadly representative compounds were selected for synthesis. A mix of factors such as the compound’s docking score, chemotype, and anticipated synthetic difficulty were all considered during this selection, with preference given to high-scoring and more readily accessible compounds featuring relatively simple substituents (i.e., primarily those containing a limited number of heteroatoms).</div><div class="NLM_p">In order to access the various benzamides in the target set, synthesis began by reaction of commercially available 2-fluoro-5-nitroaniline (<b>10</b>) with selected acid chlorides and benzoic acids to provide intermediate benzamides <b>11a</b>–<b>t</b> (<a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>).</div><figure id="sch2" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01630/20160318/images/medium/jm-2015-01630k_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01630/20160318/images/large/jm-2015-01630k_0006.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of <i>N</i>-(2-(4-Methylpiperazin-1-yl)-5-nitrophenyl) Benzamides<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01630/20160318/images/large/jm-2015-01630k_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01630&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) (i) acid chloride, pyridine, DCM, rt; or (ii) benzoic acid, SOCl<sub>2</sub>, toluene, 70 °C, then amine <b>10</b>, pyridine, DCM, rt; or (iii) benzoic acid, POCl<sub>3</sub>, toluene, 70 °C, then amine <b>10</b>, pyridine, DCM, rt; (b) 1-methylpiperazine, DIPEA, DMF, 80 °C.</p></p></figure><div class="NLM_p">Acid chlorides were successfully reacted directly with <b>10</b> under basic conditions, whereas the reaction of benzoic acids using standard coupling agents such as HATU or HOBt failed to yield any product, as did in situ generation of the reactive anhydride using ethyl chloroformate. We overcame the recalcitrant reaction through use of the corresponding acid chlorides which were first prepared by refluxing the benzoic acids with either SOCl<sub>2</sub> or POCl<sub>3</sub> and then reacted with <b>10</b> under basic conditions. Final elaboration of these intermediates <b>11a</b>–<b>t</b> was then accomplished through the addition of 1-methylpiperazine to afford benzamides <b>12a</b>–<b>t</b> (<a class="ref internalNav" href="#tbl2" aria-label="Table 2">Table 2</a>).</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Binding Affinities of Various <i>N</i>-(2-(4-Methylpiperazin-1-yl)-5-nitrophenyl) Benzamides</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01630/20160318/images/medium/jm-2015-01630k_0011.gif" alt="" id="fx3" /></img><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="left" /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compound</th><th class="colsep0 rowsep0" align="center">R</th><th class="colsep0 rowsep0" align="center" char="."><i>K</i><sub>disp</sub> [μM]<a class="ref internalNav" href="#t2fn1" aria-label="a">a</a><sup>,</sup><a class="ref internalNav" href="#t2fn2" aria-label="b">b</a></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>12a</b></td><td class="colsep0 rowsep0" align="left">2-CF<sub>3</sub>, 5-F</td><td class="colsep0 rowsep0" align="char" char=".">>50</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>12b</b></td><td class="colsep0 rowsep0" align="left">2-CF<sub>3</sub>, 4-OH</td><td class="colsep0 rowsep0" align="char" char=".">1.26</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>12c</b></td><td class="colsep0 rowsep0" align="left">2-Cl, 4-CF<sub>3</sub></td><td class="colsep0 rowsep0" align="char" char=".">13.67</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>12d</b></td><td class="colsep0 rowsep0" align="left">2-Cl, 5-CF<sub>3</sub></td><td class="colsep0 rowsep0" align="char" char="."> >50</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>12e</b></td><td class="colsep0 rowsep0" align="left">2-Cl, 5-Me</td><td class="colsep0 rowsep0" align="char" char=".">2.27</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>12f</b></td><td class="colsep0 rowsep0" align="left">2-Cl, 6-F</td><td class="colsep0 rowsep0" align="char" char=".">>50</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>12g</b></td><td class="colsep0 rowsep0" align="left">3-CF<sub>3</sub>, 4-OMe</td><td class="colsep0 rowsep0" align="char" char=".">>50</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>12h</b></td><td class="colsep0 rowsep0" align="left">3-Me, 5-Me</td><td class="colsep0 rowsep0" align="char" char=".">0.29</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>12i</b></td><td class="colsep0 rowsep0" align="left">3-Me, 5-CF<sub>3</sub></td><td class="colsep0 rowsep0" align="char" char=".">0.85</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>12j</b></td><td class="colsep0 rowsep0" align="left">3-F, 5-CF<sub>3</sub></td><td class="colsep0 rowsep0" align="char" char=".">>50</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>12k</b></td><td class="colsep0 rowsep0" align="left">3-Cl, 5-Cl</td><td class="colsep0 rowsep0" align="char" char=".">0.58</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>12l</b></td><td class="colsep0 rowsep0" align="left">3-OH, 5-CF<sub>3</sub></td><td class="colsep0 rowsep0" align="char" char=".">0.54</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>12m</b></td><td class="colsep0 rowsep0" align="left">2-F, 5-SO<sub>2</sub>NH<sub>2</sub></td><td class="colsep0 rowsep0" align="char" char=".">>50</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>12n</b></td><td class="colsep0 rowsep0" align="left">2-F, 3-F, 5-OH</td><td class="colsep0 rowsep0" align="char" char=".">0.40</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>12o</b></td><td class="colsep0 rowsep0" align="left">2-F, 3-Cl, 5-CF<sub>3</sub></td><td class="colsep0 rowsep0" align="char" char=".">>50</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>12p</b></td><td class="colsep0 rowsep0" align="left">2-Cl, 3-Me, 6-F</td><td class="colsep0 rowsep0" align="char" char=".">>50</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>12q</b></td><td class="colsep0 rowsep0" align="left">2-F, 3-Me, 4-F</td><td class="colsep0 rowsep0" align="char" char=".">1.36</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>12r</b></td><td class="colsep0 rowsep0" align="left">2-Me, 3-F, 5-F</td><td class="colsep0 rowsep0" align="char" char=".">2.34</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>12s</b></td><td class="colsep0 rowsep0" align="left">3-Me, 4-F, 5-Me</td><td class="colsep0 rowsep0" align="char" char=".">0.24</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>12t</b></td><td class="colsep0 rowsep0" align="left">2-F, 3-Me, 4-F, 5-Me, 6-F</td><td class="colsep0 rowsep0" align="char" char=".">0.29</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup>a</sup><p class="last">Average of triplicate measurements.</p></div><div class="footnote" id="t2fn2"><sup>b</sup><p class="last">FP measurement made using fluorescein-labeled H3 (1–15) peptide displacement.</p></div></div></div><div class="NLM_p">Heterocyclic carboxylic acids were coupled using a modified approach that relied on first reacting 1-methylpiperazine with commercially available <b>10</b> to afford 2-(4-methylpiperazin-1-yl)-5-nitroaniline (<b>13</b>) (<a class="ref internalNav" href="#sch3" aria-label="Scheme 3">Scheme 3</a>). This intermediate was subsequently reacted with the acid chlorides generated from the various heterocyclic carboxylic acids using POCl<sub>3</sub>, affording the final amides <b>14a</b>–<b>j</b> (<a class="ref internalNav" href="#tbl3" aria-label="Table 3">Table 3</a>). All generated compounds were then evaluated in the biochemical fluorescence polarization assay as described earlier.</div><figure id="sch3" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01630/20160318/images/medium/jm-2015-01630k_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01630/20160318/images/large/jm-2015-01630k_0007.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Synthesis of <i>N</i>-(2-(4-Methylpiperazin-1-yl)-5-nitrophenyl) Amides<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01630/20160318/images/large/jm-2015-01630k_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01630&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) 1-methylpiperazine, DIPEA, 80 °C; (b) RCO<sub>2</sub>H, POCl<sub>3</sub>, cat. DMAP, xylene, 120 °C.</p></p></figure><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. Binding Affinities of Various <i>N</i>-(2-(4-Methylpiperazin-1-yl)-5-nitrophenyl) Amides</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01630/20160318/images/medium/jm-2015-01630k_0012.gif" alt="" id="fx4" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01630/20160318/images/medium/jm-2015-01630k_0013.gif" alt="" id="fx5" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t3fn1"><div class="footnote" id="t3fn1"><sup>Table a</sup><p class="last">Average of triplicate measurements.</p></div><div class="footnote" id="t3fn2"><sup>Table b</sup><p class="last">FP measurement made using fluorescein-labeled H3 (1–15) peptide displacement.</p></div></div><div></div></div><div class="NLM_p">Compounds <b>12a</b>–<b>t</b> (<a class="ref internalNav" href="#tbl2" aria-label="Table 2">Table 2</a>) were intended to serve as a direct extension of compound <b>5</b>, bearing new substitution patterns which we hoped would better fill the hydrophobic subpocket hitherto occupied by the 2-Cl-3-Me-4-F-phenyl moiety. Significantly, of the disubstituted compounds in the series (<b>12a</b>–<b>m</b>), compound <b>12h</b> is the most potent (<i>K</i><sub>disp</sub> = 0.29 μM) and hints at the greater capacity of the 3,5-dimethylated benzamide to occupy the hydrophobic shelf. While tri- or penta-substitution of the benzamide (compounds <b>12n</b>–<b>s</b> and <b>12t</b>, respectively) did not significantly improve potency, addition of a fluorine to the 4-position of <b>12h</b> was well tolerated, giving rise to <b>12s</b> with a comparable <i>K</i><sub>disp</sub> = 0.24 μM. The only compound bearing a substituent positioned to potentially form an H-bond with the side-chain of D107, 5-hydroxylated compound <b>12n</b>, was reasonably potent (<i>K</i><sub>disp</sub> = 0.40 μM) but does not likely engage in the anticipated interaction. An orthogonal analysis of the most potent compounds (<i>K</i><sub>disp</sub> < 0.3 μM) using surface plasmon resonance (SPR) correlated well with the observed FP values and underscored that <b>12h</b> (<i>K</i><sub>D</sub> = 0.39 μM), <b>12s</b> (<i>K</i><sub>D</sub> = 0.33 μM), and <b>12t</b> (<i>K</i><sub>D</sub> = 0.34 μM) showed no marked improvement in binding over previously reported compound <b>5</b> (<i>K</i><sub>D</sub> = 0.38 μM). Surprisingly, all appeared to bind more strongly than <b>9h</b> (<i>K</i><sub>D</sub> = 0.72 μM), the most potent compound from the initial series with a comparable FP value (<i>K</i><sub>disp</sub> = 0.30 μM).</div><div class="NLM_p">The size of the heterocyclic moieties in analogs <b>14a</b>–<b>j</b> (<a class="ref internalNav" href="#tbl3" aria-label="Table 3">Table 3</a>), which were identified using the in silico approach, is consistent with our previous observation that larger quinolinyl or naphthyl substituents are well tolerated in the hydrophobic pocket.<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> Docking studies also suggest that the 4-hydroxyl substituents on antagonists such as <b>14b</b>–<b>c</b> may potentially form H-bonds to the side chain of D107 and that the nonhydroxylated aryl rings may occupy the hydrophobic pocket lined by residues A65, A47, Y260, and L321. Unfortunately, the 2-position methyl substituent on <b>14b</b> renders it inactive (presumably due to steric effects), while the activity of <b>14c</b> is little different from that of the previously reported 1-naphthyl analog,<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> suggesting that no favorable interactions are gained from the additional 4-position hydroxyl group. Moving the hydroxyl group to the 6-position only resulted in weak inhibitor <b>14d</b> despite the fact that docking predicts a flip of the entire quinolinyl moiety in order for an H-bond with D107 to be established. Introduction of pyrazolo pyridine (<b>14e</b>), benzimidazole (<b>14f</b>–<b>g</b>), and triazolo pyridine (<b>14h</b>) groups resulted in inactive compounds, while the benzo pyrazoles <b>14i</b>–<b>j</b> gave weaker antagonists. Within this heterocyclic series, only the 4-(trifluoromethyl)pyridinone compound (<b>14a</b>) showed exceptionally strong binding affinity with <i>K</i><sub>disp</sub> = 0.20 μM. As this was close to the detection limit of the FP assay using the H3 (1–15) peptide, a more sensitive version employing fluorescein-labeled 9-Ala-FAM peptide (<i>K</i><sub>D</sub> = 19 ± 3 nM) was developed in order to more accurately assess the potency of this compound. Upon retesting with the new assay format, <b>14a</b> was shown to have a <i>K</i><sub>disp</sub> = 0.079 μM, making it a very potent and suitable chemotype for further development.</div><div class="NLM_p">Following the discovery of the 4-(trifluoromethyl)pyridinone moiety, we turned our attention toward the synthesis of antagonists combining this feature with some of the more active or solubilizing C-5 substituents reported elsewhere in this study. To that end, we prepared compounds <b>16a</b>–<b>h</b> according to the route outlined in <a class="ref internalNav" href="#sch4" aria-label="Scheme 4">Scheme 4</a>.</div><figure id="sch4" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01630/20160318/images/medium/jm-2015-01630k_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01630/20160318/images/large/jm-2015-01630k_0008.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 4. Synthesis of 5-Substituted <i>N</i>-(2-(4-Methylpiperazin-1-yl)phenyl)-6-oxo-4-(trifluoromethyl)-1,6-dihydropyridine-3-carboxamides<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01630/20160318/images/large/jm-2015-01630k_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01630&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) 1-methylpiperazine, DIPEA, DMSO, 80 °C; (b) Fe<sub>(s)</sub>, CaCl<sub>2</sub>, water/MeOH (1:1), 100 °C; (c) 6-oxo-4-(trifluoromethyl)-1,6-dihydropyridine-3-carboxylic acid, SOCl<sub>2</sub>, 80 °C, then aniline <b>7</b>, DCM, pyridine, rt; (d) boronic acid/ester, Xphos Pd G2, Na<sub>2</sub>CO<sub>3</sub>, 1,4-dioxane/water (5:3), 120 °C.</p></p></figure><div class="NLM_p">To start, the coupling of 5-bromo-2-(4-methylpiperazin-1-yl)aniline (<b>7</b>) to 6-oxo-4-(trifluoromethyl)-1,6-dihydropyridine-3-carbonyl chloride (generated from 6-oxo-4-(trifluoromethyl)-1,6-dihydropyridine-3-carboxylic acid with SOCl<sub>2</sub>) produced bromide <b>15</b>, the intermediate through which the C-5 position aryl moiety was to be introduced. This was accomplished using the requisite boronic acid/ester and standard Suzuki coupling conditions, giving rise to antagonists <b>16a</b>–<b>h</b> (<a class="ref internalNav" href="#tbl4" aria-label="Table 4">Table 4</a>).</div><div class="NLM_table-wrap" id="tbl4"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 4. Binding Affinities of Various <i>N</i>-(2-(4-Methylpiperazin-1-yl)phenyl)-6-oxo-4-(trifluoromethyl)-1,6-dihydropyridine-3-carboxamides</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01630/20160318/images/medium/jm-2015-01630k_0014.gif" alt="" id="fx6" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01630/20160318/images/medium/jm-2015-01630k_0015.gif" alt="" id="fx7" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t4fn1"><div class="footnote" id="t4fn1"><sup>Table a</sup><p class="last">Average of triplicate measurements.</p></div><div class="footnote" id="t4fn2"><sup>Table b</sup><p class="last">FP measurement made using fluorescein-labeled 9-Ala-FAM peptide displacement.</p></div><div class="footnote" id="t4fn3"><sup>Table c</sup><p class="last">SPR measurement.</p></div></div><div></div></div><div class="NLM_p">Because the <b>16a</b>–<b>h</b> antagonist set yielded many compounds with <i>K</i><sub>disp</sub> values <100 nM using the FP assay, we also evaluated the entire series using SPR as an orthogonal method to confirm binding affinity. The measured <i>K</i><sub>D</sub> values obtained are also presented in <a class="ref internalNav" href="#tbl4" aria-label="Table 4">Table 4</a> and show fairly strong correlation with the recorded FP assay values. One of the principal compounds of interest is <b>16b</b>, intended to bring together the potent C-5 3-quinoline substituent described earlier with the 4-(trifluoromethyl)pyridinone moiety. By eliminating the nitro group and improving both the observed <i>K</i><sub>disp</sub> and <i>K</i><sub>D</sub> values, we were provided with what we considered to be a new lead compound in the overall program. As such, we first chose to screen <b>16b</b> for selectivity against a broad panel of epigenetic and nonepigenetic targets including G-protein coupled receptors (GPCRs), ion channels, transporters, and kinases to determine whether the compound was selective enough to warrant additional investigation.<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> Although <b>16b</b> was inactive against all other targets, it did inhibit receptor tyrosine kinases FLT3 and FLT4 with IC<sub>50</sub> values of 0.5 and 0.6 μM, respectively, and a set of other protein kinases at a low micromolar level. We employed a structure-based approach to predict the binding mode of <b>16b</b> to a selected subset of protein kinases in DFG-in and DFG-out (DFG being the Asp-Phe-Gly motif) conformations to dial out this undesirable set of activities. Thus, docking of <b>16b</b> into the X-ray crystal structures of Abl (PDB codes <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4FOC">4FOC</a>, DFG-in, and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3CS9">3CS9</a>, DFG-out), ALK (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3GXL">3GXL</a>, DFG-in), CHK2 (PDB codes <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2W7X">2W7X</a> and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4A9U">4A9U</a>, both DFG-in), and KDR (PDB codes <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3VO3">3VO3</a> and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4AGD">4AGD</a>, both DFG-out) identified the 3-quinoline substituent at C-5 to be the likely hinge-binding moiety. It is known that, with exception of the PIM family kinases, eliminating the key hinge-binding moiety in the ligand results in a significant reduction (or complete loss) of kinase inhibitory activity. To address the kinase inhibitory liability as well as the rather poor aqueous solubility of this compound, alternate C-5 substituents which showed acceptable activity in the initial screen (<a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>) and which could potentially prevent kinase hinge-binding while also enhancing solubility were docked into the above-mentioned kinase crystal structures. In the docking experiments with Glide, compounds <b>16a</b>, <b>16c</b>, and <b>16d</b> did not appear to interact meaningfully with the kinases, whereas compounds <b>16b</b> and <b>16e</b> were suggested to be hinge binders.</div><div class="NLM_p">Based on these in silico results, we decided to focus our attention on the morpholine- containing compound <b>16d</b> as a probe candidate. Running the compound through the same selectivity panel as described above showed that this compound was inactive against all other targets. Compound <b>16d</b> was co-crystallized with WDR5 to yield a high-resolution structure of the WDR5-<b>16d</b> complex (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4QL1">4QL1</a>) (<a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>).</div><figure id="fig4" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01630/20160318/images/medium/jm-2015-01630k_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01630/20160318/images/large/jm-2015-01630k_0004.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. X-ray crystal structure of the WDR5-<b>16d</b> complex (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4QL1">4QL1</a>). Compound <b>16d</b> (green) binds in the arginine binding site of WDR5 (gray). The experimental electron density (purple mesh) of the ligand is shown at a 1.5 Å resolution (2<i>F</i><sub>o</sub> – <i>F</i><sub>c</sub> map contoured at 1.5 σ). In addition to the binding interactions observed in previous derivatives, the nitrogen of the newly introduced pyridinone substituent forms a direct H-bond to the D107 side chain. A water-mediated H-bond between the pyridinone oxygen and the A65 carbonyl further stabilizes binding of the ligand. The pendant morpholino moiety points into the bulk solvent and does not interact with the protein. As a result of this solvent exposure, the electron density of the morpholine ring is not well-defined.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01630/20160318/images/large/jm-2015-01630k_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01630&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Recapitulated in this co-crystal structure are the direct and water-mediated H-bonds with S91 and C261, the importance of the latter clearly demonstrated through the synthesis of a <b>16d</b> analog morpholine at the C-2 position instead of methylpiperazine. Based on the fact that the WDR5 pocket recognizes highly basic arginine-containing side chains and also based on the analysis of the electrostatic potential on the Connolly surface of the pocket, we hypothesized that the positively charged nitrogen of the methylpiperazine is of paramount importance for binding to WDR5. Indeed, elimination of the protonated N1 atom of the methylpiperazine through replacement of the C-2 position substituent with morpholine leads to complete loss of antagonism. Because such a small structural change resulted in almost complete inactivation, this compound, <i>N</i>-(4-morpholino-3′-(morpholinomethyl)-[1,1′-biphenyl]-3-yl)-6-oxo-4-(trifluoromethyl)-1,6-dihydropyridine-3-carboxamide (OICR-0547), serves as a suitable negative control for biological experiments.<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a></div><div class="NLM_p last">Seen for the first time in the WDR5-<b>16d</b> structure is a new, direct H-bond to D107, with the trifluoromethyl group filling out the adjacent hydrophobic pocket by forming van der Waals contacts with residues A47, L321, and I305. This group also serves to push the entire pyridinone moiety toward the D107 face of the pocket, decreasing the bonding distance while also serving to facilitate the H-bond through an inductive electron-withdrawing effect that lengthens the pyridinone N–H bond. Finally, a water-mediated H-bond between the pyridinone oxygen and A65 confers additional stabilization to this binding pose. Elsewhere, the phenyl ring of the C-5 substituent interacts in a parallel π-stacking manner with the side chains of F133 and Y191. Compared to the C-5 quinoline of <b>16b</b>, which interacts in a T-shape arrangement with the side chain of F149, a slight rotation of the phenyl ring also enables parallel π-stacking with Y191.</div></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i12" class="anchor-spacer"></div><h3 class="article-section__title" id="_i12"> Cell-Based and Pharmacokinetic Assessment</h3><div class="NLM_p">Having selected <b>16d</b> as a chemical probe candidate, the compound was evaluated for its ability to inhibit proliferation in N-terminal C/EBPα p30-expressing (<i>Cebpa</i><sup><i>p30/p30</i></sup>) human AML cells.<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> The p30-dependent increase in self-renewal was shown to require WDR5 binding. We recently reported that direct antagonism of the WDR5-MLL1 interaction with <b>16d</b> reduced the viability of primary human AML cells with N-terminal C/EBPα mutations by about 50% (mean value, <i>n</i> = 8) at 5 μM. This result prompted not only the adoption of <b>16d</b> as a viable in vitro chemical probe of WDR5-MLL1 interaction but also spurred us to assess the utility of this compound for in vivo applications.</div><div class="NLM_p last">Although <b>16d</b> was measured as having suboptimal Caco-2 permeability and a high efflux ratio (apparent permeability rates were <i>P</i><sub>app</sub> A → B = 0.1 × 10<sup>–6</sup> cm/sec and <i>P</i><sub>app</sub> B → A = 16.8 × 10<sup>–6</sup> cm/s), it was potent enough to show significant activity in cells.<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> In order to potentially extend the use of <b>16d</b> into murine in vivo models of AML,<a onclick="showRef(event, 'ref23 ref24'); return false;" href="javascript:void(0);" class="ref ref23 ref24">(23, 24)</a> we also undertook an assessment of the compound’s stability and pharmacokinetic (PK) parameters in mice. Initial mouse liver microsomal and plasma stability assessments showed the compound to be stable, returning 85% and 100% of the parent respectively after 0.5 h. For PK studies, <b>16d</b> was administered to mice (female NOD-SCID) at 3 mg/kg IV (<i>n</i> = 3) and 30 mg/kg IP (<i>n</i> = 3), with blood collection conducted 6 times up to 24 h postinjection. The intravenous (IV) dosing protocol yielding a <i>C</i><sub>max</sub> of 697 ng/mL (1.26 μM) at <i>T</i><sub>max</sub> = 5 min (the first blood sample was drawn 5 min after the tail vein injection) and with a significant <i>T</i><sub>1/2</sub> = 287 min (4.78 h). The compound exposure was also reasonable with an AUC = 1012 h·ng/mL, while clearance was moderate at Cl = 3233 mL/h/kg, and the <i>V</i><sub>D</sub> = 15.6 L/kg (plasma confined). The IP dosing produced a <i>C</i><sub>max</sub> of 3790 ng/mL (6.85 μM) at <i>T</i><sub>max</sub> = 15 min, with an AUC = 6813 h·ng/mL and bioavailability <i>F</i> = 67%. Taken together, these parameters suggest that <b>16d</b> constitutes a reasonable starting point for further optimization leading to an in vivo probe of the WDR5-MLL1 interaction and possibly an AML therapeutic.</div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i13">Conclusion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_73224" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_73224" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">Medium-throughput screening of the OICR 16,000 diversity set initially led to the identification of a low-micromolar WDR5 antagonist, compound <b>4</b> (<i>K</i><sub>disp</sub> = 7 μM).<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> Its co-crystal structure in complex with WDR5 then facilitated early SAR exploration via structure-based analog design that ultimately led to the development of <b>5</b>, a reasonably potent antagonist of the WDR5-MLL1 interaction (<i>K</i><sub>disp</sub> = 0.27 μM, <i>K</i><sub>D</sub> = 0.38 μM).<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> Additional structural insights into the nature of the antagonist binding site and binding mode were then obtained through the WDR5-<b>5</b> complex crystal structure, allowing for further optimization of this scaffold. The SAR observed through variation of the C-5 substituent revealed several suitable moieties that improved the overall properties of the compound, although significant improvements in potency remained elusive. Design and enumeration of a focused virtual compound library of about 10,200 analogs, and subsequent docking experiments with Glide enabled the discovery of the 4-(trifluoromethyl)pyridin-2(1<i>H</i>)-one moiety to replace the C-1 benzamide. This substituent was critical in realizing the antagonist potencies of <i>K</i><sub>disp</sub> < 100 nM. A chemoinformatics-supported approach was then used to select a suitable C-5 substituent from the earlier development set to pair with the 4-(trifluoromethyl)pyridin-2(1<i>H</i>)-one group, resulting in antagonist <b>16d.</b> This compound showed significant in vitro activity (<i>K</i><sub>disp</sub> = 64 ± 4 nM) as well as the ability to reduce the viability of primary human AML cells with N-terminal C/EBPα mutations (IC<sub>50</sub> ≈ 5 μM). A reasonable pharmacokinetic profile (via IP administration) suggests that <b>16d</b> may not only be a viable chemical probe for exploring the WDR5-MLL1 interaction but also may serve as a suitable starting point for development of a therapeutic for MLL1-driven acute myeloid leukemias.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i14">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_60354" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_60354" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i15" class="anchor-spacer"></div><h3 class="article-section__title" id="_i15"> Focused Library Design and Molecular Docking</h3><div class="NLM_p last">Focused library design and reaction-based virtual enumeration was performed with Accelrys Pipeline Pilot (currently, BIOVIA, <a href="http://www.3ds.com/products-services/biovia/" class="extLink">http://www.3ds.com/products-services/biovia/</a>). Structures, vendor information, and prices of all commercially available building blocks were assembled in a flat SD file based on the database table dumps from eMolecules Plus (<a href="https://www.emolecules.com/info/building-blocks" class="extLink">https://www.emolecules.com/info/building-blocks</a>). These available building blocks were fed into Pipeline Pilot Markush substructure searches to yield all commercially available benzoyl and acid halides, benzoic acids and esters, and heterocyclic carboxylic acids. The lists were then used to virtually enumerate all (benz)amides accessible via commercially available reagents. OICR HTS filters were used to reduce the number of enumerated products by eliminating compounds with undesirable calculated physical properties (MW, PSA, calculated octanol–water partition coefficient, log <i>P</i>, number of aromatic rings, number of H-bond acceptors and donors, number of rotatable bonds, individual halogens counts, positively and negatively charged groups counts). As a result, about 10,200 (benz)amides were enumerated: 1200 based on available acid halides and 9000 based on available acids/esters. The obtained library was prepared with the ligprep procedure from Schrödinger, Inc. Accordingly, the molecules were ionized to pH 7 with Epik, specific chiralities were retained while all other chiral centers were enumerated, and no tautomers were generated. The resulting structures were energy minimized with OPLS_2005 force field. To avoid docking instability and to obtain the best possible results during the docking stage with Glide, 10 different conformers were generated for each molecule with the ConfGen Mixed torsional/Low-mode sampling procedure using 1 × 10<sup>6</sup> steps and a 21 kJ/mol energy window for saving structures. The protein grid calculation was done based on the WDR5-<b>4</b> co-crystal structure (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3SMR">3SMR</a>, resolved to 1.8 Å) with one H-bond constraint defined with the side chain of S91. Docking was performed with the Glide docking program using the SP scoring function (<a href="http://www.schrodinger.com" class="extLink">http://www.schrodinger.com</a>). Each compound was subject to postdocking minimization using a set of 100 ligand poses. Out of all the poses generated for multiple conformers, one best docking pose was retained for each compound. The obtained SP Glide docking scores were used to rank the top 2000 virtual compounds that were visually inspected. The 50 best looking analogs were selected and prioritized for synthesis.</div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i16" class="anchor-spacer"></div><h3 class="article-section__title" id="_i16"> Chemistry</h3><div class="NLM_p">All oxygen- and/or moisture-sensitive reactions were carried out under a nitrogen atmosphere. Solvent removal from reaction mixtures was performed by rotary evaporation under reduced pressure at 40 °C unless otherwise noted. All reagents and laboratory grade solvents were purchased from commercial vendors and used as received, without further purification. The yields given refer to chromatographically purified and spectroscopically pure compounds, unless stated otherwise. Nuclear magnetic resonance (NMR) spectra for <sup>1</sup>H, <sup>13</sup>C, DEPTq, COSY, HSQC, and HMBC were recorded on a Bruker Avance-III 500 MHz spectrometer (500 MHz for <sup>1</sup>H, 125 MHz for <sup>13</sup>C). All <sup>1</sup>H NMR spectra were referenced relative to SiMe<sub>4</sub> through a resonance of the employed deuterated solvent or proteo impurity of the solvent; chloroform (7.26 ppm), acetone (2.05 ppm), DMSO (3.33 ppm), and methanol (3.31 ppm) for <sup>1</sup>H NMR; chloroform (77.00 ppm), acetone (29.84 ppm), DMSO (39.52 ppm), and methanol (49.00 ppm) for <sup>13</sup>C NMR. NMR data are reported as follows: chemical shifts (δ), multiplicity (br = broad, s = singlet, d = doublet, t = triplet, q = quartet, m = multiplet); coupling constant(s) (<i>J</i>) in Hz; integration. Unless otherwise noted, NMR data were collected at 25 °C. Flash column chromatography was performed using a Biotage SP1 system fitted with a KP-SIL SNAP Silica Gel (60 Å mesh) Flash Cartridge (FSKO-1107). Compound purity determination was conducted by UV absorbance at 254 nm during tandem liquid chromatography/mass spectrometry (LCMS) using a Waters Acquity separations module. All compounds reported in this publication were assessed as having ≥95% purity using this method. Compound identity was determined via low-resolution mass spectrometry (LRMS) conducted in positive ion mode using a Waters Acquity SQD mass spectrometer (electrospray ionization source) fitted with a PDA detector. Mobile phase A consisted of 0.1% formic acid in water, while mobile phase B consisted of 0.1% formic acid in acetonitrile. Two types of columns were used: Column 1, Acquity UPLC CSH C18 (2.1 × 50 mm, 130 Å, 1.7 μm; part no. 186005296); and Column 2, Acquity UPLC BEH C8 (2.1 × 50 mm, 130 Å, 1.7 μm; part no. 186002877). Column temperature was maintained at 25 °C, and the sample solution injection volume was 1 μL. Leucine Enkephalin was used as lock mass, and MassLynx 4.1 was used for data analysis. Analytical thin-layer chromatography (TLC) was performed on aluminum sheets coated with silica gel 60 F254 (0.2 mm, VWR International, Darmstadt, Germany). Visualization was accomplished with UV light and aqueous potassium permanganate (KMnO<sub>4</sub>) stain followed by heating. High-resolution mass spectrometry (HRMS) was conducted using a Waters Xevo quadrupole-time-of-flight (QTOF) hybrid mass spectrometer system coupled with an Acquity ultraperformance liquid chromatography (UPLC) system.</div><div id="sec4_2_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i17" class="anchor-spacer"></div><h4 class="article-section__title" id="_i17"> 5-Bromo-2-(4-methylpiperazin-1-yl)aniline (<b>7</b>)</h4><div class="NLM_p last">A 100 mL round-bottom flask was charged with 4-bromo-1-fluoro-2-nitrobenzene (<b>6</b>) (3.33 mL, 26.5 mmol), 1-methylpiperazine (3.28 mL, 29.2 mmol), and <i>N</i>,<i>N</i>-diisopropylethylamine (9.12 mL, 53.1 mmol) in DMSO (20 mL). The reaction turned red and was stirred for 3 h at 80 °C during which time it gradually turned brown. After cooling to 23 °C and dilution with water (80 mL), the aqueous layer was extracted with EtOAc (4 × 50 mL). The combined organic extracts were dried (Na<sub>2</sub>SO<sub>4</sub>), filtered, and concentrated under reduced pressure. The residue was purified by flash column chromatography on silica gel (0–30% MeOH/EtOAc) to afford the intermediate 1-(4-bromo-2-nitrophenyl)-4-methylpiperazine (8.22 g, 100% yield) as a red oil. Subsequently, the 1-(4-bromo-2-nitrophenyl)-4-methylpiperazine (7.94 g, 27 mmol), iron powder (7.39 g, 132 mmol), and calcium chloride (3.52 g, 32 mmol) were dissolved in a mixture of water and MeOH (1:1, 100 mL) and heated at reflux for 3 h. The solution was then basified with 1 M NaOH, diluted with brine, and extracted with EtOAc (5 × 50 mL). The combined organic extracts were dried (MgSO<sub>4</sub>), filtered, and concentrated under reduced pressure. The residue was purified by flash column chromatography on silica gel (5–30% MeOH/EtOAc) to afford the title compound (2.75 g, 39% yield). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 6.83 (d, <i>J</i> = 2.4 Hz, 1H), 6.80 (d, <i>J</i> = 8.3 Hz, 1H), 6.65 (dd, <i>J</i> = 2.4, 8.3 Hz, 1H), 4.97 (s, 2H), 2.76 (br s, 4H), 2.47 (br s, 4H), 2.22 (s, 3H); <sup>13</sup>C NMR (125 MHz, DMSO-<i>d</i><sub>6</sub>) δ 144.3, 137.4, 120.9, 118.5, 116.2, 116.1, 55.0 (2), 50.1 (2), 45.8. HRMS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>11</sub>H<sub>17</sub>BrN<sub>3</sub> [M + H]<sup>+</sup>: 270.0606, found: 270.0612.</div></div><div id="sec4_2_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i18" class="anchor-spacer"></div><h4 class="article-section__title" id="_i18"> <i>N</i>-(5-Bromo-2-(4-methylpiperazin-1-yl)phenyl)-3-methylbenzamide (<b>8</b>)</h4><div class="NLM_p last">5-Bromo-2-(4-methylpiperazin-1-yl)aniline (<b>7</b>) (1.5 g, 5.55 mmol) was dissolved in CH<sub>2</sub>Cl<sub>2</sub> (25 mL) prior to the addition of pyridine (584 μL, 7 mmol) and 3-methylbenzoyl chloride (807 μL, 7 mmol). After 1 h at 23 °C the reaction mixture was diluted with EtOAc and 1 M NaOH, and the aqueous layer extracted with EtOAc (3 × 15 mL). The organic layers were combined, dried (MgSO<sub>4</sub>), filtered, and concentrated under reduced pressure to give a brown solid. This crude product was purified by flash column chromatography on silica gel (5–30% MeOH/EtOAc) to afford the title compound (2.12 g, 93% yield) as an off-white solid. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.58 (s, 1H), 8.39 (d, <i>J</i> = 2.4 Hz, 1H), 7.74–7.71 (m, 2H), 7.49–7.45 (m, 2H), 7.30 (dd, <i>J</i> = 2.4, 8.6 Hz, 1H), 7.25 (d, <i>J</i> = 8.6 Hz, 1H), 2.86 (t, <i>J</i> = 4.8 Hz, 4H), 2.51 (br s, 4H), 2.43 (s, 3H), 2.24 (s, 3H); <sup>13</sup>C NMR (125 MHz, DMSO-<i>d</i><sub>6</sub>) δ 164.4, 142.1, 138.2, 134.5, 134.1, 132.7, 128.8, 127.5, 126.8, 124.1, 123.0, 122.7, 116.6, 55.4 (2), 51.3 (2), 45.8, 20.9. HRMS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>19</sub>H<sub>23</sub>BrN<sub>3</sub>O [M + H]<sup>+</sup>: 388.1025, found: 388.1036.</div></div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i19" class="anchor-spacer"></div><h3 class="article-section__title" id="_i19"> General Procedure A for the Synthesis of Compounds <b>9a</b>–<b>w</b></h3><div class="NLM_p">A 5 mL reaction vial was charged with <i>N</i>-(5-bromo-2-(4-methylpiperazin-1-yl)phenyl)-3-methylbenzamide (<b>8</b>) (50 mg, 0.13 mmol), the boronic acid/ester (0.17 mmol), sodium carbonate (68 mg, 0.64 mmol), Pd(Ph<sub>3</sub>P)<sub>4</sub> (15 mg, 0.01 mmol), and toluene/water (1:1, 2 mL). The reaction mixture was then heated at 115 °C overnight. After cooling to 23 °C, the solution was diluted with EtOAc (1 mL) and water (1 mL), and the aqueous layer extracted with EtOAc (3 × 2 mL). The combined organic layers were then dried (MgSO<sub>4</sub>), filtered, and concentrated under reduced pressure. The crude product was purified by flash column chromatography on silica gel (5–50% MeOH/EtOAc) to afford the desired compound.</div><div id="sec4_3_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i20" class="anchor-spacer"></div><h4 class="article-section__title" id="_i20"> <i>N</i>-(5-Cyclopropyl-2-(4-methylpiperazin-1-yl)phenyl)-3-methylbenzamide (<b>9a</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_3" aria-label="general procedure A">general procedure A</a> (17 mg, 35% yield). <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ 9.25 (br s, 1H), 8.30 (s, 1H), 7.76 (s, 1H), 7.65 (d, <i>J</i> = 6.6 Hz, 1H), 7.43–7.38 (m, 2H), 7.18 (d, <i>J</i> = 8.2 Hz, 1H), 6.84 (dd, <i>J</i> = 2.0, 4.3 Hz, 1H), 3.35–2.75 (m, 8H), 2.66 (br s, 3H), 2.47 (s, 3H), 1.95–1.89 (m, 1H), 0.99–0.95 (m, 2H), 0.76–0.72 (m, 2H); <sup>13</sup>C NMR (125 MHz, CDCl<sub>3</sub>) δ 165.1, 142.7, 139.0, 138.1, 135.4, 133.9, 132.7, 129.0, 128.1, 123.9, 121.4, 121.1, 116.9, 56.0 (2), 51.5, 45.5, 21.7 (2), 15.7, 9.5. HRMS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>22</sub>H<sub>28</sub>N<sub>3</sub>O [M + H]<sup>+</sup>: 350.2232, found: 350.2230.</div></div><div id="sec4_3_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i21" class="anchor-spacer"></div><h4 class="article-section__title" id="_i21"> 3-Methyl-<i>N</i>-(2-(4-methylpiperazin-1-yl)-5-(naphthalen-1-yl)phenyl)benzamide (<b>9b</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_3" aria-label="general procedure A">general procedure A</a> (26 mg, 44% yield). <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ 9.32 (br s, 1H), 8.71 (d, <i>J</i> = 1.5 Hz, 1H), 7.99 (d, <i>J</i> = 8.3 Hz, 1H), 7.90 (d, <i>J</i> = 8.2 Hz, 1H), 7.86 (d, <i>J</i> = 8.2 Hz, 1H), 7.77 (s, 1H), 7.68 (d, <i>J</i> = 7.3 Hz, 1H), 7.53–7.38 (m, 7H), 7.27 (dd, <i>J</i> = 1.9, 8.1 Hz, 1H), 3.27 (br s, 4H), 2.96 (br s, 4H), 2.64 (br s, 3H), 2.46 (s, 3H); <sup>13</sup>C NMR (125 MHz, CDCl<sub>3</sub>) δ 165.2, 139.7, 139.5, 139.2, 139.1, 135.2, 134.0, 133.6, 132.9, 131.7, 129.0, 128.5, 128.1, 128.0, 127.3, 126.3, 126.1, 126.1, 125.9, 125.5, 123.8, 121.8, 121.0, 55.5 (2), 50.8, 45.0, 21.7 (2). HRMS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>29</sub>H<sub>30</sub>N<sub>3</sub>O [M + H]<sup>+</sup>: 436.2389, found: 436.2397.</div></div><div id="sec4_3_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i22" class="anchor-spacer"></div><h4 class="article-section__title" id="_i22"> 3-Methyl-<i>N</i>-(2-(4-methylpiperazin-1-yl)-5-(naphthalen-2-yl)phenyl)benzamide (<b>9c</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_3" aria-label="general procedure A">general procedure A</a> (9 mg, 15% yield). <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ 9.24 (br s, 1H), 8.98 (s, 1H), 8.11 (d, <i>J</i> = 1.1 Hz, 1H), 7.91 (d, <i>J</i> = 8.1 Hz, 1H), 7.86 (d, <i>J</i> = 8.1 Hz, 1H), 7.81–7.79 (m, 2H), 7.69 (d, <i>J</i> = 7.1 Hz, 1H), 7.53–7.40 (m, 6H), 3.27 (br s, 4H), 2.97 (br s, 4H), 2.70 (br s, 3H), 2.49 (s, 3H); <sup>13</sup>C NMR (125 MHz, CDCl<sub>3</sub>) δ 165.7, 140.8, 139.3, 137.7, 137.4, 135.0, 133.8, 133.7, 133.2, 133.1, 129.1, 128.8, 128.5, 128.5, 127.9, 126.6, 126.4, 126.2, 125.6, 124.0, 123.8, 121.8, 54.3, 49.5 (2), 43.9, 29.9, 21.8 (2). HRMS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>29</sub>H<sub>30</sub>N<sub>3</sub>O [M + H]<sup>+</sup>: 436.2389, found: 436.2389.</div></div><div id="sec4_3_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i23" class="anchor-spacer"></div><h4 class="article-section__title" id="_i23"> <i>N</i>-(5-(2,3-Dihydrobenzofuran-7-yl)-2-(4-methylpiperazin-1-yl)phenyl)-3-methylbenzamide (<b>9d</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_3" aria-label="general procedure A">general procedure A</a> (36 mg, 62% yield). <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ 9.23 (br s, 1H), 8.87 (d, <i>J</i> = 2.0 Hz, 1H), 7.78 (s, 1H), 7.68 (d, <i>J</i> = 7.0 Hz, 1H), 7.50 (dd, <i>J</i> = 2.0, 8.3 Hz, 1H), 7.44–7.38 (m, 2H), 7.34–7.32 (m, 2H), 7.18 (dd, <i>J</i> = 1.1, 7.3 Hz, 1H), 6.93 (t, <i>J</i> = 7.6 Hz, 1H), 4.64 (t, <i>J</i> = 8.8 Hz, 2H), 3.35–2.80 (m, 10H), 2.65 (s, 3H), 2.47 (s, 3H); <sup>13</sup>C NMR (125 MHz, CDCl<sub>3</sub>) δ 165.3, 157.4, 139.3, 138.1, 136.7, 135.4, 133.4, 133.0, 129.0, 128.4, 128.1, 128.0, 124.9, 124.5, 123.6, 122.8, 121.4, 121.1, 120.7, 71.4, 54.7 (2), 49.3 (2), 43.8, 30.0, 21.7. HRMS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>27</sub>H<sub>30</sub>N<sub>3</sub>O<sub>2</sub> [M + H]<sup>+</sup>: 428.2338, found: 428.2342.</div></div><div id="sec4_3_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i24" class="anchor-spacer"></div><h4 class="article-section__title" id="_i24"> <i>N</i>-(5-(2,3-Dihydrobenzofuran-5-yl)-2-(4-methylpiperazin-1-yl)phenyl)-3-methylbenzamide (<b>9e</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_3" aria-label="general procedure A">general procedure A</a> (35 mg, 60% yield). <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ 9.27 (br s, 1H), 8.79 (s, 1H), 7.79 (s, 1H), 7.71–7.67 (m, 1H), 7.51 (d, <i>J</i> = 1.2 Hz, 1H), 7.44–7.38 (m, 3H), 7.33–7.29 (m, 2H), 6.84 (d, <i>J</i> = 8.3 Hz, 1H), 4.62 (t, <i>J</i> = 8.5 Hz, 2H), 3.32–2.82 (m, 10H), 2.66 (s, 3H), 2.48 (s, 3H); <sup>13</sup>C NMR (125 MHz, CDCl<sub>3</sub>) δ 165.3, 160.2, 140.3, 139.3, 138.0, 135.3, 133.8, 133.1, 133.0, 129.0, 128.3, 128.0, 127.3, 124.0, 123.6, 122.7, 121.7, 118.6, 109.6, 71.7, 55.0 (2), 49.7 (2), 44.1, 29.9, 21.8. HRMS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>27</sub>H<sub>30</sub>N<sub>3</sub>O<sub>2</sub> [M + H]<sup>+</sup>: 428.2338, found: 428.2353.</div></div><div id="sec4_3_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i25" class="anchor-spacer"></div><h4 class="article-section__title" id="_i25"> <i>N</i>-(5-(2,3-Dihydrobenzo[<i>b</i>][1,4]dioxin-6-yl)-2-(4-methylpiperazin-1-yl)phenyl)-3-methylbenzamide (<b>9f</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_3" aria-label="general procedure A">general procedure A</a> (8 mg, 13% yield). <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ 9.24 (br s, 1H), 8.77 (d, <i>J</i> = 1.5 Hz, 1H), 7.77 (s, 1H), 7.67 (d, <i>J</i> = 7.1 Hz, 1H), 7.44–7.39 (m, 3H), 7.32–7.27 (m, 2H), 7.17 (d, <i>J</i> = 2.2 Hz, 1H), 7.14 (dd, <i>J</i> = 2.3, 8.2 Hz, 1H), 6.92 (d, <i>J</i> = 8.4 Hz, 1H), 4.30 (s, 4H), 3.21 (br s, 4H), 2.96 (br s, 4H), 2.66 (br s, 3H), 2.47 (s, 3H); <sup>13</sup>C NMR (125 MHz, CDCl<sub>3</sub>) δ 165.8, 143.9, 143.9, 139.2, 136.7, 136.5, 134.8, 133.4, 133.4, 133.2, 129.0, 129.6, 123.9, 123.6. 121.5, 121.1, 120.6, 117.8, 116.2, 64.7, 64.6, 53.8 (2), 49.6, 43.8, 21.7 (2). HRMS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>27</sub>H<sub>30</sub>N<sub>3</sub>O<sub>3</sub> [M + H]<sup>+</sup>: 444.2287, found: 444.2294.</div></div><div id="sec4_3_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i26" class="anchor-spacer"></div><h4 class="article-section__title" id="_i26"> <i>N</i>-(5-(Isoquinolin-4-yl)-2-(4-methylpiperazin-1-yl)phenyl)-3-methylbenzamide (<b>9g</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_3" aria-label="general procedure A">general procedure A</a> (31 mg, 52% yield). <sup>1</sup>H NMR (500 MHz, CD<sub>3</sub>OD) δ 9.21 (br s, 1H), 8.61 (d, <i>J</i> = 1.9 Hz, 1H), 8.36 (s, 1H), 8.14 (d, <i>J</i> = 8.0 Hz, 1H), 8.01 (d, <i>J</i> = 8.5 Hz, 1H), 7.77–7.67 (m, 4H), 7.44–7.41 (m, 3H), 7.29 (dd, <i>J</i> = 2.0, 8.1 Hz, 1H), 3.03 (t, <i>J</i> = 4.9 Hz, 4H), 2.68 (br s, 4H), 2.43 (s, 3H), 2.38 (s, 3H); <sup>13</sup>C NMR (125 MHz, CD<sub>3</sub>OD) δ 167.7, 152.9, 144.4, 142.7, 140.3, 136.0, 135.7, 135.1, 134.8, 134.6, 134.1, 132.8, 130.2, 130.1, 129.6, 129.1, 128.9, 127.7, 125.9, 125.4, 124.0, 122.5, 57.0 (2), 52.9, 46.3, 21.6 (2). HRMS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>28</sub>H<sub>29</sub>N<sub>4</sub>O [M + H]<sup>+</sup>: 437.2341, found: 437.2339.</div></div><div id="sec4_3_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i27" class="anchor-spacer"></div><h4 class="article-section__title" id="_i27"> 3-Methyl-<i>N</i>-(2-(4-methylpiperazin-1-yl)-5-(quinolin-3-yl)phenyl)benzamide (<b>9h</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_3" aria-label="general procedure A">general procedure A</a> (26 mg, 44% yield). <sup>1</sup>H NMR (500 MHz, CD<sub>3</sub>OD) δ 9.01 (d, <i>J</i> = 2.3 Hz, 1H), 8.61 (d, <i>J</i> = 2.2 Hz, 1H), 8.36 (d, <i>J</i> = 2.2 Hz, 1H), 7.99 (d, <i>J</i> = 8.5 Hz, 1H), 7.89 (d, <i>J</i> = 7.9 Hz, 1H), 7.72–7.67 (m, 3H), 7.57–7.47 (m, 1H), 7.43–7.37 (m, 3H), 7.29 (d, <i>J</i> = 8.2 Hz, 1H), 2.90 (t, <i>J</i> = 4.8 Hz, 4H), 2.60 (br s, 4H), 2.40 (s, 3H), 2.33 (s, 3H); <sup>13</sup>C NMR (125 MHz, CD<sub>3</sub>OD) δ 167.5, 150.4, 147.9, 144.4, 140.3, 135.9, 135.4, 135.1, 135.0, 134.9, 134.2, 131.2, 130.1, 129.8, 129.7, 129.1, 128.9, 128.7, 125.4, 124.8, 122.7, 120.8, 57.0 (2), 52.8, 46.4, 21.6 (2). HRMS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>28</sub>H<sub>29</sub>N<sub>4</sub>O [M + H]<sup>+</sup>: 437.2341, found: 437.2336.</div></div><div id="sec4_3_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i28" class="anchor-spacer"></div><h4 class="article-section__title" id="_i28"> <i>N</i>-(5-(Benzo[b]thiophen-3-yl)-2-(4-methylpiperazin-1-yl)phenyl)-3-methylbenzamide (<b>9i</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_3" aria-label="general procedure A">general procedure A</a> (11 mg, 15% yield). <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ 9.30 (br s, 1H), 8.82 (s, 1H), 8.03 (d, <i>J</i> = 8.1 Hz, 1H), 7.92 (d, <i>J</i> = 7.8 Hz, 1H), 7.79 (s, 1H), 7.69 (d, <i>J</i> = 7.7 Hz, 1H), 7.47 (s, 1H), 7.44–7.36 (m, 6H), 3.25 (br s, 4H), 2.98 (br s, 4H), 2.64 (s, 3H), 2.48 (s, 3H); <sup>13</sup>C NMR (125 MHz, CDCl3) δ 165.3, 140.9, 139.2, 139.2, 137.8, 137.4, 135.2, 134.7, 133.9, 133.0, 129.0, 128.3, 124.7, 124.6 (2C), 124.1, 123.7, 123.1 (2C), 121.7, 120.6, 55.1 (2), 50.0, 44.4, 21.7 (2). HRMS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>27</sub>H<sub>28</sub>N<sub>3</sub>OS [M + H]<sup>+</sup>: 442.1953, found: 442.1962.</div></div><div id="sec4_3_10" class="NLM_sec NLM_sec_level_3"><div id="ac_i29" class="anchor-spacer"></div><h4 class="article-section__title" id="_i29"> <i>N</i>-(5-(Benzo[b]thiophen-2-yl)-2-(4-methylpiperazin-1-yl)phenyl)-3-methylbenzamide (<b>9j</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_3" aria-label="general procedure A">general procedure A</a> (9 mg, 13% yield). <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ 9.18 (br s, 1H), 9.01 (s, 1H), 7.84–7.77 (m, 3H), 7.68 (d, <i>J</i> = 5.6 Hz, 1H), 7.63 (s, 1H), 7.48–7.42 (m, 3H), 7.38–7.30 (m, 3H), 3.59–2.86 (m, 8H), 2.70 (br s, 3H), 2.49 (s, 3H); <sup>13</sup>C NMR (125 MHz, CDCl<sub>3</sub>) δ 165.6, 143.1, 140.8, 139.9, 139.4, 138.0, 134.9, 133.9, 133.6, 133.2, 129.1, 128.5, 125.4, 124.8, 123.7, 123.0, 122.6, 122.5, 122.0, 121.7, 120.7, 54.4 (2), 49.4, 44.0 (2), 21.8. HRMS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>27</sub>H<sub>28</sub>N<sub>3</sub>OS [M + H]<sup>+</sup>: 442.1953, found: 442.1961.</div></div><div id="sec4_3_11" class="NLM_sec NLM_sec_level_3"><div id="ac_i30" class="anchor-spacer"></div><h4 class="article-section__title" id="_i30"> 3-Methyl-<i>N</i>-(5-(5-methylbenzo[b]thiophen-2-yl)-2-(4-methylpiperazin-1-yl)phenyl)benzamide (<b>9k</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_3" aria-label="general procedure A">general procedure A</a> (14 mg, 23% yield). <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ 9.06 (s, 1H), 8.96 (s, 1H), 7.78 (s, 1H), 7.70–7.64 (m, 2H), 7.59–7.53 (m, 2H), 7.47–7.41 (m, 3H), 7.34 (d, <i>J</i> = 8.2 Hz, 1H), 7.15 (dd, <i>J</i> = 1.3, 8.2 Hz, 1H), 3.95–2.64 (m, 11H), 2.48 (s, 3H), 2.47 (s, 3H); <sup>13</sup>C NMR (125 MHz, CDCl<sub>3</sub>) δ 165.3, 143.8, 141.2, 139.2, 137.0, 135.1, 134.5, 134.1, 133.0, 133.0, 129.0, 129.0, 128.2, 126.4, 123.8, 123.8, 122.3, 122.1, 121.7, 120.0, 117.8, 55.3 (2), 50.5, 44.5, 21.7 (2), 21.6. HRMS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>28</sub>H<sub>30</sub>N<sub>3</sub>OS [M + H]<sup>+</sup>: 456.2110, found: 456.2115.</div></div><div id="sec4_3_12" class="NLM_sec NLM_sec_level_3"><div id="ac_i31" class="anchor-spacer"></div><h4 class="article-section__title" id="_i31"> <i>N</i>-(5-(Dibenzo[b,d]thiophen-2-yl)-2-(4-methylpiperazin-1-yl)phenyl)-3-methylbenzamide (<b>9l</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_3" aria-label="general procedure A">general procedure A</a> (26 mg, 39% yield). <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ 9.27 (s, 1H), 8.97 (d, <i>J</i> = 1.4 Hz, 1H), 8.40 (d, <i>J</i> = 1.5 Hz, 1H), 8.28–8.24 (m, 1H), 7.91 (d, <i>J</i> = 8.6 Hz, 1H), 7.88–7.85 (m, 1H), 7.81 (br s, 1H), 7.75 (dd, <i>J</i> = 1.9, 8.4 Hz, 1H), 7.70 (d, <i>J</i> = 7.1 Hz, 1H), 7.50–7.46 (m, 3H), 7.45–7.40 (m, 3H), 3.26 (br s, 4H), 3.00 (br s, 4H), 2.70 (s, 3H), 2.49 (s, 3H); <sup>13</sup>C NMR (125 MHz, CDCl<sub>3</sub>) δ 165.3, 140.1, 140.1, 139.3, 139.1, 138.8, 137.4, 136.3, 135.8, 135.3, 134.3, 132.9, 129.0, 128.1, 127.0, 126.4, 124.6, 124.0, 123.2, 123.1, 123.0, 122.0, 121.6, 120.2, 118.6, 55.9 (2), 51.5, 45.7, 21.7 (2). HRMS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>31</sub>H<sub>30</sub>N<sub>3</sub>OS [M + H]<sup>+</sup>: 492.2110, found: 492.2115.</div></div><div id="sec4_3_13" class="NLM_sec NLM_sec_level_3"><div id="ac_i32" class="anchor-spacer"></div><h4 class="article-section__title" id="_i32"> 3-Methyl-<i>N</i>-(5-(1-methyl-1<i>H</i>-indol-4-yl)-2-(4-methylpiperazin-1-yl)phenyl)benzamide (<b>9m</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_3" aria-label="general procedure A">general procedure A</a> (30 mg, 51% yield). <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ 9.17 (br s, 1H), 8.86 (s, 1H), 7.73 (s, 1H), 7.61 (d, <i>J</i> = 7.0 Hz, 1H), 7.42 (dd, <i>J</i> = 1.9, 8.3 Hz, 1H), 7.36–7.30 (m, 3H), 7.27–7.22 (m, 2H), 7.19 (dd, <i>J</i> = 1.5, 7.0 Hz, 1H),7.06 (d, <i>J</i> = 3.2 Hz, 1H), 6.74 (d, <i>J</i> = 3.0 Hz, 1H), 3.77 (s, 3H), 3.18 (br s, 4H), 2.91 (br s, 4H), 2.58 (br s, 3H), 2.40 (br s, 3H); <sup>13</sup>C NMR (125 MHz, CDCl<sub>3</sub>) δ 165.2, 139.9, 139.9, 138.9, 137.5, 135.4, 133.7, 133.6, 132.8, 129.6, 129.0, 128.3, 126.8, 124.8, 123.7, 122.0, 121.4, 120.8, 119.6, 108.8, 100.8, 55.3, 50.4, 44.7, 33.2, 21.7. HRMS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>28</sub>H<sub>31</sub>N<sub>4</sub>O [M + H]<sup>+</sup>: 439.2498, found: 439.2496.</div></div><div id="sec4_3_14" class="NLM_sec NLM_sec_level_3"><div id="ac_i33" class="anchor-spacer"></div><h4 class="article-section__title" id="_i33"> 3-Methyl-<i>N</i>-(5-(1-methyl-1<i>H</i>-indol-6-yl)-2-(4-methylpiperazin-1-yl)phenyl)benzamide (<b>9n</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_3" aria-label="general procedure A">general procedure A</a> (39 mg, 66% yield). <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ 9.26 (br s, 1H), 8.86 (s, 1H), 7.74 (s, 1H), 7.64 (d, <i>J</i> = 6.5 Hz, 1H), 7.60 (d, <i>J</i> = 8.5 Hz, 1H), 7.52 (s, 1H), 7.39–7.33 (m, 4H), 7.29 (d, <i>J</i> = 8.2 Hz, 1H),7.02 (d, <i>J</i> = 3.0 Hz, 1H), 6.43 (d, <i>J</i> = 3.0 Hz, 1H), 3.79 (s, 3H), 3.14 (br s, 4H), 2.85 (br s, 4H), 2.57 (br s, 3H), 2.41 (br s, 3H); <sup>13</sup>C NMR (125 MHz, CDCl<sub>3</sub>) δ 165.3, 41.0, 139.1, 139.0, 137.4, 135.4, 134.5, 134.0, 132.9, 129.8, 129.0, 128.1, 128.2, 123.8, 123.2, 121.5, 121.2, 119.4, 118.9, 108.0, 101.0, 56.5 (2), 50.8, 45.0, 33.1 (2), 21.7. HRMS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>28</sub>H<sub>31</sub>N<sub>4</sub>O [M + H]<sup>+</sup>: 439.2498, found: 439.2496.</div></div><div id="sec4_3_15" class="NLM_sec NLM_sec_level_3"><div id="ac_i34" class="anchor-spacer"></div><h4 class="article-section__title" id="_i34"> <i>N</i>-(5-(1<i>H</i>-Indol-5-yl)-2-(4-methylpiperazin-1-yl)phenyl)-3-methylbenzamide (<b>9o</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_3" aria-label="general procedure A">general procedure A</a> (27 mg, 47% yield). <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ 9.54 (s, 1H), 8.91 (d, <i>J</i> = 1.9 Hz, 1H), 8.35 (s, 1H), 7.87 (s, 1H), 7.78 (s, 1H), 7.72 (d, <i>J</i> = 7.4 Hz, 1H), 7.46 (dd, <i>J</i> = 1.6, 8.5 Hz, 1H), 7.39–7.32 (m, 4H), 7.24 (d, <i>J</i> = 8.2 Hz, 1H), 7.16 (d, <i>J</i> = 2.6 Hz, 1H), 6.54–6.53 (m, 1H), 2.97 (s, 4H), 2.61 (br s, 4H), 2.43 (s, 3H), 2.35 (s, 3H); <sup>13</sup>C NMR (125 MHz, CDCl<sub>3</sub>) δ 165.9, 141.0, 139.0, 138.9, 135.7, 134.8, 133.2, 132.8, 131.8, 128.8, 128.4, 128.3, 127.9, 125.2, 125.1, 123.6, 123.3, 121.2, 118.9, 111.4, 102.1, 55.0 (2), 50.2, 29.6, 21.2 (2). HRMS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>27</sub>H<sub>28</sub>N<sub>4</sub>O [M + H]<sup>+</sup>: 425.2341, found: 425.2328.</div></div><div id="sec4_3_16" class="NLM_sec NLM_sec_level_3"><div id="ac_i35" class="anchor-spacer"></div><h4 class="article-section__title" id="_i35"> 3-Methyl-<i>N</i>-(4-(4-methylpiperazin-1-yl)-3′-(pyrrolidin-1-yl)-[1,1′-biphenyl]-3-yl)benzamide (<b>9p</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_3" aria-label="general procedure A">general procedure A</a> (9 mg, 15% yield). <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ 9.45 (br s, 1H), 8.85 (d, <i>J</i> = 1.6 Hz, 1H), 7.77 (s, 1H), 7.70 (d, <i>J</i> = 7.1 Hz, 1H), 7.40–7.32 (m, 3H), 7.27–7.23 (m, 2H), 6.91 (d, <i>J</i> = 7.6 Hz, 1H), 6.79 (s, 1H), 6.53 (dd, <i>J</i> = 2.0, 8.2 Hz, 1H), 3.34–3.32 (m, 4H), 3.03 (br s, 4H), 2.71 (br s, 3H), 2.44–2.42 (m, 7H), 2.02–1.97 (m, 4H); <sup>13</sup>C NMR (125 MHz, CDCl<sub>3</sub>) δ 165.2, 148.5, 141.7, 140.6, 139.6, 139.0, 135.3, 133.9, 132.8, 129.6, 129.0, 129.0, 128.1, 128.1, 123.8, 123.1, 121.3, 118.8, 114.8, 111.1, 110.6, 55.6, 51.0 (2), 47.9, 45.2, 25.6 (2), 21.7. HRMS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>29</sub>H<sub>35</sub>N<sub>4</sub>O [M + H]<sup>+</sup>: 455.2811, found: 455.2811.</div></div><div id="sec4_3_17" class="NLM_sec NLM_sec_level_3"><div id="ac_i36" class="anchor-spacer"></div><h4 class="article-section__title" id="_i36"> 3-Methyl-<i>N</i>-(4-(4-methylpiperazin-1-yl)-4′-(pyrrolidin-1-yl)-[1,1′-biphenyl]-3-yl)benzamide (<b>9q</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_3" aria-label="general procedure A">general procedure A</a> (18 mg, 29% yield). <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ 9.45 (br s, 1H), 8.82 (d, <i>J</i> = 1.4 Hz, 1H), 7.80 (s, 1H), 7.72 (d, <i>J</i> = 7.7 Hz, 1H), 7.57–7.54 (m, 2H), 7.44–7.37 (m, 2H), 7.30 (dd, <i>J</i> = 2.0, 8.4 Hz, 1H), 7.27–7.25 (m, 1H), 6.63–6.60 (m, 2H), 3.33–3.31 (m, 4H), 3.08 (br s, 4H), 2.78 (br s, 3H), 2.49 (br s, 3H), 2.47 (s, 4H), 2.03–2.00 (m, 4H); <sup>13</sup>C NMR (125 MHz, CDCl<sub>3</sub>) δ 165.0, 147.6, 139.7, 138.9, 138.6, 135.4, 134.1, 132.7, 128.9, 128.1, 128.0, 127.5, 123.9, 121.6, 121.3, 117.3, 112.1, 55.9, 51.5, 47.8 (2), 45.6, 25.7 (2), 21.7. HRMS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>29</sub>H<sub>35</sub>N<sub>4</sub>O [M + H]<sup>+</sup>: 455.2811, found: 455.2815.</div></div><div id="sec4_3_18" class="NLM_sec NLM_sec_level_3"><div id="ac_i37" class="anchor-spacer"></div><h4 class="article-section__title" id="_i37"> 3-Methyl-<i>N</i>-(4-(4-methylpiperazin-1-yl)-2′-(morpholinomethyl)-[1,1′-biphenyl]-3-yl)benzamide (<b>9r</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_3" aria-label="general procedure A">general procedure A</a> (27 mg, 42% yield). <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ 9.37 (br s, 1H), 8.55 (s, 1H), 7.77 (s, 1H), 7.70 (s, 1H), 7.44–7.28 (m, 7H), 7.13 (d, <i>J</i> = 6.7 Hz, 1H), 3.73 (br s, 4H), 3.36–2.97 (m, 20H); <sup>13</sup>C NMR (125 MHz, CDCl<sub>3</sub>) δ 165.0, 142.6, 140.1, 139.2, 138.9, 135.3, 133.5, 132.8, 130.5, 130.3, 129.0, 128.0, 127.5, 127.3, 125.2, 124.0, 120.8, 120.5, 67.0, 60.1, 56.2 (2), 53.4 (2), 52.1 (2), 46.1, 21.7 (2). HRMS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>30</sub>H<sub>37</sub>N<sub>4</sub>O<sub>2</sub> [M + H]<sup>+</sup>: 485.2917, found: 485.2924.</div></div><div id="sec4_3_19" class="NLM_sec NLM_sec_level_3"><div id="ac_i38" class="anchor-spacer"></div><h4 class="article-section__title" id="_i38"> 3-Methyl-<i>N</i>-(4-(4-methylpiperazin-1-yl)-3′-(morpholinomethyl)-[1,1′-biphenyl]-3-yl)benzamide (<b>9s</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_3" aria-label="general procedure A">general procedure A</a> (15 mg, 23% yield). <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ 9.44 (br s, 1H), 8.87 (d, <i>J</i> = 1.9 Hz, 1H), 7.80 (s, 1H), 7.73 (d, <i>J</i> = 7.3 Hz, 1H), 7.63 (s, 1H), 7.57 (d, <i>J</i> = 7.3 Hz, 1H), 7.45–7.31 (m, 6H), 3.78 (s, 4H), 3.65 (s, 3H), 3.11 (s, 4H), 2.82 (br s, 3H), 2.56 (br s, 4H), 2.52 (br s, 3H), 2.48 (s, 3H); <sup>13</sup>C NMR (125 MHz, DMSO-<i>d</i><sub>6</sub>) δ 165.1, 140.9, 139.6, 139.0, 138.8, 135.0, 133.9, 132.8, 129.2, 128.9, 128.0, 127.4, 123.6, 123.0, 121.5, 118.7, 77.3, 77.0, 76.8, 65.6, 62.6, 55.2 (2), 52.8 (2), 50.4 (2), 44.7, 21.5 (2). HRMS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>30</sub>H<sub>37</sub>N<sub>4</sub>O<sub>2</sub> [M + H]<sup>+</sup>: 485.2917, found: 485.2924.</div></div><div id="sec4_3_20" class="NLM_sec NLM_sec_level_3"><div id="ac_i39" class="anchor-spacer"></div><h4 class="article-section__title" id="_i39"> 3-Methyl-<i>N</i>-(4-(4-methylpiperazin-1-yl)-4′-(morpholinomethyl)-[1,1′-biphenyl]-3-yl)benzamide (<b>9t</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_3" aria-label="general procedure A">general procedure A</a> (17 mg, 27% yield). <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ 9.30 (br s, 1H), 8.85 (s, 1H), 7.79 (s, 1H), 7.69 (d, <i>J</i> = 7.2 Hz, 1H), 7.64 (d, <i>J</i> = 7.9 Hz, 2H), 7.49 (br s, 2H), 7.45–7.40 (m, 2H), 7.35 (s, 2H), 3.94–3.64 (m, 7H), 3.36–2.97 (m, 8H), 2.62 (br s, 6H), 2.48 (s, 3H); <sup>13</sup>C NMR (125 MHz, CDCl<sub>3</sub>) δ 165.2, 139.9, 139.1, 138.9, 135.2, 134.1, 132.9, 130.6, 129.2, 129.0, 128.1, 127.7, 127.6, 123.9, 122.8, 121.6, 118.6, 66.1, 62.7, 55.7 (2), 53.1 (2), 51.0 (2), 45.3, 21.8 (2). HRMS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>30</sub>H<sub>37</sub>N<sub>4</sub>O<sub>2</sub> [M + H]<sup>+</sup>: 485.2917, found: 485.2920.</div></div><div id="sec4_3_21" class="NLM_sec NLM_sec_level_3"><div id="ac_i40" class="anchor-spacer"></div><h4 class="article-section__title" id="_i40"> 3-Methyl-<i>N</i>-(5-(5-methyl-1-phenyl-1<i>H</i>-pyrazol-4-yl)-2-(4-methylpiperazin-1-yl)phenyl)benzamide (<b>9u</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_3" aria-label="general procedure A">general procedure A</a> (13 mg, 21% yield). <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ 9.29 (s, 1H), 8.70 (d, <i>J</i> = 1.9 Hz, 1H), 7.83 (s, 1H), 7.79 (s, 1H), 7.68 (d, <i>J</i> = 6.8 Hz, 1H), 7.51 (s, 2H), 7.50 (s, 2H), 7.44–7.39 (m, 3H), 7.34 (d, <i>J</i> = 8.1 Hz, 1H), 7.22 (dd, <i>J</i> = 2.0, 8.2 Hz, 1H), 3.23 (br s, 4H), 2.98 (br s, 4H), 2.65 (s, 3H), 2.51 (s, 3H), 2.47 (s, 3H); <sup>13</sup>C NMR (125 MHz, CDCl<sub>3</sub>) δ 165.3, 139.6, 139.2, 139.6, 137.1, 135.8, 135.0, 133.5, 132.9, 132.6, 129.3, 129.2, 128.8, 128.2, 128.0, 125.2, 123.7, 123.3, 121.6, 121.2, 54.5, 49.7 (2), 43.7, 21.5 (2), 12.1. HRMS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>29</sub>H<sub>32</sub>N<sub>5</sub>O [M + H]<sup>+</sup>: 466.2607, found: 466.2608.</div></div><div id="sec4_3_22" class="NLM_sec NLM_sec_level_3"><div id="ac_i41" class="anchor-spacer"></div><h4 class="article-section__title" id="_i41"> <i>N</i>-(5-(1-Isopentyl-1<i>H</i>-pyrazol-4-yl)-2-(4-methylpiperazin-1-yl)phenyl)-3-methylbenzamide (<b>9v</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_3" aria-label="general procedure A">general procedure A</a> (11 mg, 18% yield). <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ 9.28 (br s, 1H), 8.72 (d, <i>J</i> = 1.8 Hz, 1H), 7.79 (d, <i>J</i> = 0.6 Hz, 1H), 7.78 (br s, 1H), 7.71 (s, 1H), 7.67 (d, <i>J</i> = 7.2 Hz, 1H), 7.45–7.40 (m, 2H), 7.28 (d, <i>J</i> = 8.3 Hz, 1H), 7.25 (dd, <i>J</i> = 2.0, 8.4 Hz, 1H), 4.16 (t, <i>J</i> = 7.6 Hz, 2H), 3.26 (br s, 4H), 2.94 (br s, 4H), 2.67 (br s, 3H), 2.48 (s, 3H), 1.81 (q, <i>J</i> = 7.1 Hz, 2H), 1.65–1.58 (m, 1H), 0.98 (s, 3H), 0.96 (s, 3H); <sup>13</sup>C NMR (125 MHz, CDCl<sub>3</sub>) δ 165.3, 139.3, 137.3, 136.6, 135.3, 134.0, 133.1, 132.1, 129.1, 128.3, 126.5, 123.5, 123.1, 122.3, 122.0, 121.5, 117.3, 54.8, 51.0, 49.4, 43.9, 39.4, 25.8, 22.5, 21.8. HRMS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>27</sub>H<sub>35</sub>N<sub>5</sub>O [M + H]<sup>+</sup>: 446.2920, found: 446.2926.</div></div><div id="sec4_3_23" class="NLM_sec NLM_sec_level_3"><div id="ac_i42" class="anchor-spacer"></div><h4 class="article-section__title" id="_i42"> <i>N</i>-(5-(6-Acetamidopyridin-3-yl)-2-(4-methylpiperazin-1-yl)phenyl)-3-methylbenzamide (<b>9w</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_3" aria-label="general procedure A">general procedure A</a> (14 mg, 23% yield). <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ 9.21 (br s, 1H), 9.01 (s, 1H), 8.74 (s, 1H), 8.47 (s, 1H), 8.34 (d, <i>J</i> = 8.8 Hz, 1H), 8.04 (dd, <i>J</i> = 2.0, 8.9 Hz, 1H), 7.76 (s, 1H), 7.63 (q, <i>J</i> = 6.2 Hz, 1H), 7.46–7.40 (m, 3H), 7.31 (dd, <i>J</i> = 1.8, 8.5 Hz, 1H), 3.78 (t, <i>J</i> = 11.5 Hz, 2H), 3.67 (d, <i>J</i> = 11.5 Hz, 2H), 3.15 (d, <i>J</i> = 13.0 Hz, 2H), 3.08 (t, <i>J</i> = 11.5 Hz, 2H), 2.91 (s, 3H), 2.47 (s, 3H), 2.29 (s, 3H); <sup>13</sup>C NMR (125 MHz, CDCl<sub>3</sub>) δ 165.6, 148.1, 144.9, 140.8, 139.6, 134.8, 134.6, 133.8, 133.5, 131.9, 129.2, 128.4, 123.6, 123.0, 122.7, 118.8, 117.6, 54.6 (2), 53.6, 49.2 (2), 43.9, 29.9, 25.1, 21.8. HRMS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>26</sub>H<sub>30</sub>N<sub>5</sub>O<sub>2</sub> [M + H]<sup>+</sup>: 444.2400, found: 444.2394.</div></div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i43" class="anchor-spacer"></div><h3 class="article-section__title" id="_i43"> General Procedure B for the Synthesis of Compounds <b>12a</b>,<b>c</b>,<b>f</b>,<b>h</b>,<b>k</b>,<b>q</b>,<b>s</b></h3><div class="NLM_p last">To a solution of commercially available 2-fluoro-5-nitroaniline (<b>10</b>) (0.100 g, 0.64 mmol) in CH<sub>2</sub>Cl<sub>2</sub> (3 mL) was added the commercial acid chloride (0.83 mmol) at 23 °C. This was followed by the addition of pyridine (0.07 mL, 0.83 mmol), and the resulting solution was stirred at 23 °C for 4 h. The reaction was subsequently diluted with CH<sub>2</sub>Cl<sub>2</sub> (5 mL) and washed with water (10 mL). The organic phase was then separated, dried (Na<sub>2</sub>SO<sub>4</sub>), filtered, and concentrated. The residue was purified by flash column chromatography on silica gel using an appropriate EtOAc/hexanes gradient as the eluent, providing the intermediate fluoro-benzamide <b>11</b>. To a solution of the fluoro-benzamide <b>11</b> (0.50 mmol) in DMF (4 mL) at 23 °C was added 1-methylpiperazine (0.072 mL, 0.65 mmol), followed by <i>N</i>,<i>N</i>-diisopropylethylamine (0.113 mL, 0.881 mmol). The resulting solution was then heated at 80 °C for 1 h and subsequently cooled to 23 °C. Following dilution with ethyl acetate (40 mL), the organic phase was washed with water (2 × 20 mL), separated, dried (Na<sub>2</sub>SO<sub>4</sub>), filtered, and concentrated. The residue was purified by trituration from EtOAc with hexanes or by flash column chromatography on silica gel using a common 5–20% MeOH/EtOAc gradient to afford the desired product.</div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i44" class="anchor-spacer"></div><h3 class="article-section__title" id="_i44"> General Procedure C for the Synthesis of Compounds <b>12d</b>,<b>e</b>,<b>g</b>,<b>j</b>,<b>o</b>,<b>p</b>,<b>t</b></h3><div class="NLM_p last">To a solution of the benzoic acid (0.530 mmol) in toluene (2 mL) was added SOCl<sub>2</sub> (0.20 mL, 2.65 mmol) at 23 °C. The resulting solution was stirred vigorously and heated at 70 °C for 2 h and subsequently cooled back down to 23 °C. All of the solvent was then removed under reduced pressure and the crude acid chloride used in the subsequent reaction with no further purification. To a solution of commercially available 2-fluoro-5-nitroaniline (<b>10</b>) (0.100 g, 0.64 mmol) in CH<sub>2</sub>Cl<sub>2</sub> (3 mL) was added the freshly prepared acid chloride (0.83 mmol) at 23 °C. This was followed by the addition of pyridine (0.07 mL, 0.83 mmol) and stirring of the resulting solution at 23 °C for 4 h. The reaction mixture was subsequently diluted with CH<sub>2</sub>Cl<sub>2</sub> (5 mL) and washed with water (10 mL). The organic phase was then separated, dried (Na<sub>2</sub>SO<sub>4</sub>), filtered, and concentrated. The residue was purified by flash column chromatography on silica gel using an appropriate EtOAc/hexanes gradient as the eluent, providing the intermediate fluoro-benzamide <b>11</b>. To a solution of the fluoro-benzamide <b>11</b> (0.500 mmol) in DMF (4 mL) at 23 °C was added 1-methylpiperazine (0.072 mL, 0.650 mmol) followed by <i>N</i>,<i>N</i>-diisopropylethylamine (0.113 mL, 0.881 mmol). The resulting solution was heated at 80 °C for 1 h and subsequently cooled to 23 °C. Following dilution with ethyl acetate (40 mL), the organic phase was washed with water (2 × 20 mL), separated, dried (Na<sub>2</sub>SO<sub>4</sub>), filtered, and concentrated. The residue was purified by trituration from EtOAc with hexanes or by flash column chromatography on silica gel using a common 5–20% MeOH/EtOAc gradient to afford the desired product.</div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i45" class="anchor-spacer"></div><h3 class="article-section__title" id="_i45"> General Procedure D for the Synthesis of Compounds <b>12b</b>,<b>i</b>,<b>l</b>–<b>n</b>,<b>r</b></h3><div class="NLM_p">Identical to <a class="ref internalNav" href="#sec4_5" aria-label="general procedure C">general procedure C</a>, but using POCl<sub>3</sub> in place of SOCl<sub>2</sub>.</div><div id="sec4_6_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i46" class="anchor-spacer"></div><h4 class="article-section__title" id="_i46"> 5-Fluoro-<i>N</i>-(2-(4-methylpiperazin-1-yl)-5-nitrophenyl)-2-(trifluoromethyl)benzamide (<b>12a</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_4" aria-label="general procedure B">general procedure B</a> (74 mg, 90% yield). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.06 (s, 1H), 8.60 (d, <i>J</i> = 3.0 Hz, 1H), 8.06 (dd, <i>J</i> = 9.0, 2.5 Hz, 1H), 7.96 (dd, <i>J</i> = 8.5, 5.0 Hz, 1H), 7.69 (d, <i>J</i> = 7.0 Hz, 1H), 7.59 (t, <i>J</i> = 8.5 Hz, 1H), 7.27 (d, <i>J</i> = 9.0 Hz, 1H), 3.10 (br t, 4H), 2.48 (br s, 4H), 2.22 (s, 3H). <sup>13</sup>C NMR (125 MHz, DMSO-<i>d</i><sub>6</sub>) δ 165.2, 164.0 (d, <sup>a1</sup><i>J</i><sub>C–F</sub> = 250.0 Hz), 152.0, 141.7, 138.7 (d, <sup>a4</sup><i>J</i><sub>C–F</sub> = 6.3 Hz), 130.3, 130.1 (q, <sup>b3</sup><i>J</i><sub>C–F</sub> = 5.0 Hz), 124.0 (q, <sup>b1</sup><i>J</i><sub>C–F</sub> = 271.3 Hz), 122.9 (q, <sup>b2</sup><i>J</i><sub>C–F</sub> = 28.8 Hz), 122.2, 120.7, 120.0, 117.6 (d, <sup>a3</sup><i>J</i><sub>C–F</sub> = 21.3 Hz), 116.6 (d, <sup>a2</sup><i>J</i><sub>C–F</sub> = 25.0 Hz), 54.5 (2), 50.5 (2), 46.2. HRMS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>19</sub>H<sub>18</sub>F<sub>4</sub>N<sub>4</sub>O<sub>3</sub> [M + H]<sup>+</sup>: 427.1393, found: 427.1389.</div></div><div id="sec4_6_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i47" class="anchor-spacer"></div><h4 class="article-section__title" id="_i47"> 4-Hydroxy-<i>N</i>-(2-(4-methylpiperazin-1-yl)-5-nitrophenyl)-2-(trifluoromethyl)benzamide (<b>12b</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_6" aria-label="general procedure D">general procedure D</a> (14 mg, 15% yield). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.59 (br s, 1H), 9.68 (s, 1H), 8.58 (s, 1H), 8.04 (dd, <i>J</i> = 9.0, 2.5 Hz, 1H), 7.58 (d, <i>J</i> = 8.5 Hz, 1H), 7.28 (d, <i>J</i> = 9.0 Hz, 1H), 7.18 (s, 1H), 7.13 (d, <i>J</i> = 8.0 Hz, 1H), 3.10 (br s, 4H), 2.51 (br s, 4H), 2.25 (s, 3H). <sup>13</sup>C NMR (125 MHz, DMSO-<i>d</i><sub>6</sub>) δ 166.6, 159.4, 151.6, 141.9, 131.1, 130.9, 127.4 (q, <sup>1</sup><i>J</i><sub>C–F</sub> = 193.8 Hz), 121.8 (2), 120.2, 120.1, 119.0 (2), 113.7, 54.6 (2), 50.4 (2), 46.0. HRMS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>19</sub>H<sub>19</sub>F<sub>3</sub>N<sub>4</sub>O<sub>4</sub> [M + H]<sup>+</sup>: 425.1437, found: 425.1438.</div></div><div id="sec4_6_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i48" class="anchor-spacer"></div><h4 class="article-section__title" id="_i48"> 2-Chloro-<i>N</i>-(2-(4-methylpiperazin-1-yl)-5-nitrophenyl)-4-(trifluoromethyl)benzamide (<b>12c</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_4" aria-label="general procedure B">general procedure B</a> (72 mg, 85% yield). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.10 (br s, 1H), 8.71 (d, <i>J</i> = 2.5 Hz, 1H), 8.12 (s, 1H), 8.07 (dd, <i>J</i> = 9.0, 2.5 Hz, 1H), 7.89 (d, <i>J</i> = 8.0 Hz, 1H), 7.83 (d, <i>J</i> = 8.5 Hz, 1H), 7.27 (d, <i>J</i> = 9.0 Hz, 1H), 3.11 (br t, 4H), 2.49 (br s, 4H), 2.21 (s, 3H). <sup>13</sup>C NMR (125 MHz, DMSO-<i>d</i><sub>6</sub>) δ 164.7, 152.2, 141.8, 137.7, 134.8, 131.3, 130.3, 128.3 (q, <sup>2</sup><i>J</i><sub>C–F</sub> = 32.5 Hz), 128.3 (d, <sup>4</sup><i>J</i><sub>C–F</sub> = 2.5 Hz), 126.7 (d, <sup>3</sup><i>J</i><sub>C–F</sub> = 3.8 Hz), 124.1 (q, <sup>1</sup><i>J</i><sub>C–F</sub> = 271.3 Hz), 122.2, 121.2, 120.0, 54.6 (2), 50.6 (2), 46.2. HRMS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>19</sub>H<sub>18</sub>ClF<sub>3</sub>N<sub>4</sub>O<sub>3</sub> [M + H]<sup>+</sup>: 443.1098, found: 443.1105.</div></div><div id="sec4_6_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i49" class="anchor-spacer"></div><h4 class="article-section__title" id="_i49"> 2-Chloro-<i>N</i>-(2-(4-methylpiperazin-1-yl)-5-nitrophenyl)-5-(trifluoromethyl)benzamide (<b>12d</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_5" aria-label="general procedure C">general procedure C</a> (65 mg, 71% yield). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.11 (s, 1H), 8.71 (d, <i>J</i> = 2.5 Hz, 1H), 8.12 (s, 1H), 8.07 (dd, <i>J</i> = 9.0, 2.5 Hz, 1H), 7.90 (d, <i>J</i> = 8.5 Hz, 1H), 7.83 (d, <i>J</i> = 8.5 Hz, 1H), 7.28 (d, <i>J</i> = 8.5 Hz, 1H), 3.12 (br t, 4H), 2.50 (br t, 4H), 2.22 (s, 3H). <sup>13</sup>C NMR (125 MHz, DMSO-<i>d</i><sub>6</sub>) δ 164.7, 152.2, 141.7, 137.7, 134.8, 131.3, 130.2, 128.3 (q, <sup>2</sup><i>J</i><sub>C–F</sub> = 32.5 Hz), 128.3 (q, <sup>3</sup><i>J</i><sub>C–F</sub> = 3.8 Hz), 126.7 (q, <sup>3</sup><i>J</i><sub>C–F</sub> = 3.8 Hz), 124.1 (q, <sup>1</sup><i>J</i><sub>C–F</sub> = 395.0 Hz), 122.3, 121.3, 120.0, 54.6 (2), 50.6 (2), 46.2. HRMS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>19</sub>H<sub>18</sub>ClF<sub>3</sub>N<sub>4</sub>O<sub>3</sub> [M + H]<sup>+</sup>: 443.1098, found: 443.1107.</div></div><div id="sec4_6_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i50" class="anchor-spacer"></div><h4 class="article-section__title" id="_i50"> 2-Chloro-5-methyl-<i>N</i>-(2-(4-methylpiperazin-1-yl)-5-nitrophenyl)benzamide (<b>12e</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_5" aria-label="general procedure C">general procedure C</a> (66 mg, 88% yield). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.83 (s, 1H), 8.67 (br s, 1H), 8.05 (dd, <i>J</i> = 8.5, 3.0 Hz, 1H), 7.48 (s, 1H), 7.46 (d, <i>J</i> = 8.5 Hz, 1H), 7.34 (d, <i>J</i> = 8.0 Hz, 1H), 7.29 (d, <i>J</i> = 9.0 Hz, 1H), 3.10 (br t, 4H), 2.49 (br t, 4H), 2.36 (s, 3H), 2.22 (s, 3H). <sup>13</sup>C NMR (125 MHz, DMSO-<i>d</i><sub>6</sub>) δ 165.8, 151.8, 141.9, 137.4, 136.2, 132.4, 130.8, 130.6, 130.0 (2) 127.2, 121.9, 120.2, 54.76 (2), 50.6 (2), 46.2, 20.7. HRMS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>19</sub>H<sub>21</sub>ClN<sub>4</sub>O<sub>3</sub> [M + H]<sup>+</sup>: 389.1392, found: 389.1390.</div></div><div id="sec4_6_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i51" class="anchor-spacer"></div><h4 class="article-section__title" id="_i51"> 2-Chloro-6-fluoro-<i>N</i>-(2-(4-methylpiperazin-1-yl)-5-nitrophenyl)benzamide (<b>12f</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_4" aria-label="general procedure B">general procedure B</a> (51 mg, 68% yield). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.33 (s, 1H), 8.54 (d, <i>J</i> = 2.5 Hz, 1H), 8.07 (dd, <i>J</i> = 9.0, 2.5 Hz, 1H), 7.59–7.53 (m, 1H), 7.46 (d, <i>J</i> = 8.0 Hz, 1H), 7.40 (t, <i>J</i> = 9.0 Hz, 1H), 7.26 (d, <i>J</i> = 9.0 Hz, 1H), 3.11 (br t, 4H), 2.49 (br s, 4H), 2.22 (s, 3H). <sup>13</sup>C NMR (125 MHz, DMSO-<i>d</i><sub>6</sub>) δ 161.7, 159.3 (d, <sup>1</sup><i>J</i><sub>C–F</sub> = 246.3 Hz), 152.2, 141.5, 132.3 (d, <sup>3</sup><i>J</i><sub>C–F</sub> = 10.0 Hz), 131.4 (d, <sup>4</sup><i>J</i><sub>C–F</sub> = 5.0 Hz), 129.8, 126.2, 126.0 (d, <sup>5</sup><i>J</i><sub>C–F</sub> = 3.8 Hz), 122.4, 120.9, 119.9, 115.2 (d, <sup>2</sup><i>J</i><sub>C–F</sub> = 21.3 Hz), 54.5 (2), 50.5 (2), 46.2. HRMS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>18</sub>H<sub>18</sub>ClFN<sub>4</sub>O<sub>3</sub> [M + H]<sup>+</sup>: 393.1130, found: 393.1129.</div></div><div id="sec4_6_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i52" class="anchor-spacer"></div><h4 class="article-section__title" id="_i52"> 4-Methoxy-<i>N</i>-(2-(4-methylpiperazin-1-yl)-5-nitrophenyl)-3-(trifluoromethyl)benzamide (<b>12g</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_5" aria-label="general procedure C">general procedure C</a> (71 mg, 82% yield). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.86 (s, 1H), 8.64 (d, <i>J</i> = 2.5 Hz, 1H), 8.28 (d, <i>J</i> = 9.0 Hz, 1H), 8.24 (s, 1H), 8.05 (dd, <i>J</i> = 9.0, 2.5 Hz, 1H), 7.48 (d, <i>J</i> = 8.5 Hz, 1H), 7.31 (d, <i>J</i> = 9.0 Hz, 1H), 4.01 (s, 3H), 3.08 (br t, 4H), 2.49 (br s, 4H), 2.22 (s, 3H). <sup>13</sup>C NMR (125 MHz, DMSO-<i>d</i><sub>6</sub>) δ 163.9, 160.2, 151.9, 142.0, 134.4, 131.1, 126.9 (q, <sup>3</sup><i>J</i><sub>C–F</sub> = 5.0 Hz), 123.9 (q, <sup>1</sup><i>J</i><sub>C–F</sub> = 270.0 Hz), 121.7, 120.5, 120.2, 117.2 (q, <sup>2</sup><i>J</i><sub>C–F</sub> = 30.0 Hz), 113.6 (2), 57.2 (2), 55.0 (2), 50.5, 46.2. HRMS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>20</sub>H<sub>21</sub>F<sub>3</sub>N<sub>4</sub>O<sub>4</sub> [M + H]<sup>+</sup>: 439.1593, found: 439.1595.</div></div><div id="sec4_6_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i53" class="anchor-spacer"></div><h4 class="article-section__title" id="_i53"> 3,5-Dimethyl-<i>N</i>-(2-(4-methylpiperazin-1-yl)-5-nitrophenyl)benzamide (<b>12h</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_4" aria-label="general procedure B">general procedure B</a> (70 mg, 90% yield). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.60 (s, 1H), 8.76 (d, <i>J</i> = 2.5 Hz, 1H), 8.03 (dd, <i>J</i> = 9.0, 2.5 Hz, 1H), 7.58 (s, 2H), 7.36 (d, <i>J</i> = 9.0 Hz, 1H), 3.07 (br t, 4H), 2.51 (br s, 4H), 2.38 (s, 6H), 2.24 (s, 3H). <sup>13</sup>C NMR (125 MHz, DMSO-<i>d</i><sub>6</sub>) δ 165.5, 151.0, 142.5, 138.5, 134.4, 133.9, 131.9, 125.5 (2), 121.2, 120.6, 118.7, 55.3 (2), 50.7 (2), 46.3, 21.3 (2). HRMS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>20</sub>H<sub>24</sub>N<sub>4</sub>O<sub>3</sub> [M + H]<sup>+</sup>: 369.1927, found: 369.1927.</div></div><div id="sec4_6_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i54" class="anchor-spacer"></div><h4 class="article-section__title" id="_i54"> 3-Methyl-<i>N</i>-(2-(4-methylpiperazin-1-yl)-5-nitrophenyl)-5-(trifluoromethyl)benzamide (<b>12i</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_6" aria-label="general procedure D">general procedure D</a> (4 mg, 4% yield). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.98 (br s, 1H), 8.63 (d, <i>J</i> = 2.5 Hz, 1H), 8.09 (s, 2H), 8.06 (dd, <i>J</i> = 9.5, 3.0 Hz, 1H), 7.84 (s, 1H), 7.32 (d, <i>J</i> = 9.0 Hz, 1H), 4.01 (s, 3H), 3.09 (br t, 4H), 2.48 (br s, 4H), 2.21 (s, 3H). <sup>13</sup>C NMR (125 MHz, DMSO-<i>d</i><sub>6</sub>) δ 164.4, 152.0, 142.0, 140.5, 135.4, 132.7, 130.9, 129.7 (q, <sup>2</sup><i>J</i><sub>C–F</sub> = 32.5 Hz), 129.3, 124.4 (q, <sup>1</sup><i>J</i><sub>C–F</sub> = 271.3 Hz), 122.0, 121.7, 120.6, 120.3, 55.0 (2), 50.5 (2), 46.2, 21.2. HRMS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>20</sub>H<sub>21</sub>F<sub>3</sub>N<sub>4</sub>O<sub>3</sub> [M + H]<sup>+</sup>: 423.1644, found: 423.1640.</div></div><div id="sec4_6_10" class="NLM_sec NLM_sec_level_3"><div id="ac_i55" class="anchor-spacer"></div><h4 class="article-section__title" id="_i55"> 3-Fluoro-<i>N</i>-(2-(4-methylpiperazin-1-yl)-5-nitrophenyl)-5-(trifluoromethyl)benzamide (<b>12j</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_5" aria-label="general procedure C">general procedure C</a> (78 mg, 80% yield). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.15 (s, 1H), 8.56 (d, <i>J</i> = 2.0 Hz, 1H), 8.18 (s, 1H), 8.11 (d, <i>J</i> = 8.5 Hz, 1H), 8.07 (dd, <i>J</i> = 9.0, 3.0 Hz, 1H), 8.02 (d, <i>J</i> = 8.5 Hz, 1H), 7.30 (d, <i>J</i> = 9.0 Hz, 1H), 3.12 (br t, 4H), 2.48 (br t, 4H), 2.21 (s, 3H). <sup>13</sup>C NMR (125 MHz, DMSO-<i>d</i><sub>6</sub>) δ 163.2, 162.4 (d, <sup>a1</sup><i>J</i><sub>C–F</sub> = 246.3 Hz), 152.5, 141.6, 118.1 (d, <sup>a3</sup><i>J</i><sub>C–F</sub> = 11.3 Hz), 131.7 (q, <sup>b2</sup><i>J</i><sub>C–F</sub> = 33.8 Hz), 130.16, 124.6, 122.5, 121.7, 121.0 (m), 120.1, 119.5 (d, <sup>a2</sup><i>J</i><sub>C–F</sub> = 22.5 Hz), 116.7 (q, <sup>b3</sup><i>J</i><sub>C–F</sub> = 21.3 Hz), 54.9 (2), 50.4 (2), 46.1. HRMS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>19</sub>H<sub>18</sub>F<sub>4</sub>N<sub>4</sub>O<sub>3</sub> [M + H]<sup>+</sup>: 427.1393, found: 427.1400.</div></div><div id="sec4_6_11" class="NLM_sec NLM_sec_level_3"><div id="ac_i56" class="anchor-spacer"></div><h4 class="article-section__title" id="_i56"> 3,5-Dichloro-<i>N</i>-(2-(4-methylpiperazin-1-yl)-5-nitrophenyl)benzamide (<b>12k</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_4" aria-label="general procedure B">general procedure B</a> (61 mg, 77% yield). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.01 (s, 1H), 8.51 (d, <i>J</i> = 2.5 Hz, 1H), 8.05 (dd, <i>J</i> = 9.0, 2.5 Hz, 1H), 7.97 (s, 1H), 7.90 (s, 1H), 7.28 (d, <i>J</i> = 9.0 Hz, 1H), 3.10 (br t, 4H), 2.47 (br t, 4H), 2.21 (s, 3H). <sup>13</sup>C NMR (125 MHz, DMSO-<i>d</i><sub>6</sub>) δ 163.1, 152.4, 141.6, 137.8, 134.9 (2), 131.7, 130.2, 126.9 (2), 122.4, 121.5, 120.1, 54.9 (2), 50.4 (2), 46.2. HRMS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>18</sub>H<sub>18</sub>Cl<sub>2</sub>N<sub>4</sub>O<sub>3</sub> [M + H]<sup>+</sup>: 409.0834, found: 409.0844.</div></div><div id="sec4_6_12" class="NLM_sec NLM_sec_level_3"><div id="ac_i57" class="anchor-spacer"></div><h4 class="article-section__title" id="_i57"> 3-Hydroxy-<i>N</i>-(2-(4-methylpiperazin-1-yl)-5-nitrophenyl)-5-(trifluoromethyl)benzamide (<b>12l</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_6" aria-label="general procedure D">general procedure D</a> (8 mg, 14% yield). <sup>1</sup>H NMR (500 MHz, CD<sub>3</sub>OD) δ 10.96 (br s, 1H), 9.91 (s, 1H), 8.64 (d, <i>J</i> = 2.9 Hz, 1H), 8.07 (dd, <i>J</i> = 2.7, 8.9 Hz, 1H), 7.72 (s, 1H), 7.64 (s, 1H), 7.33 (d, <i>J</i> = 9.1 Hz, 2H), 3.10 (t, <i>J</i> = 4.7 Hz, 4H), 2.50 (s, 4H), 2.23 (s, 3H); <sup>13</sup>C NMR (125 MHz, CD<sub>3</sub>OD) δ 163.6, 158.4, 151.4, 141.5, 136.5, 130.6, 130.4, 130.3, 123.7 (d, <sup>1</sup><i>J</i><sub>C–F</sub> = 272.7 Hz, 1C), 121.4, 119.9 (d, <sup>3</sup><i>J</i><sub>C–F</sub> = 10.0 Hz, 1C), 118.5, 115.0, 114.2 (d, <sup>4</sup><i>J</i><sub>C–F</sub> = 3.7 Hz, 1C), 54.5 (2), 50.0 (2), 45.6. HRMS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>19</sub>H<sub>20</sub>F<sub>3</sub>N<sub>4</sub>O<sub>4</sub> [M + H]<sup>+</sup>: 425.1437, found: 425.1438.</div></div><div id="sec4_6_13" class="NLM_sec NLM_sec_level_3"><div id="ac_i58" class="anchor-spacer"></div><h4 class="article-section__title" id="_i58"> 2-Fluoro-<i>N</i>-(2-(4-methylpiperazin-1-yl)-5-nitrophenyl)-5-sulfamoylbenzamide (<b>12m</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_6" aria-label="general procedure D">general procedure D</a> (26 mg, 23% yield). <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ 9.96 (d, <i>J</i> = 6.9 Hz, 1H), 8.96 (d, <i>J</i> = 1.7 Hz, 1H), 8.37 (dd, <i>J</i> = 2.0, 6.7 Hz, 1H), 8.07–8.02 (m, 2H), 7.68–7.64 (m, 1H), 7.57 (s, 2H), 7.41 (d, <i>J</i> = 9.0 Hz, 1H), 3.10–3.07 (m, 4H), 2.60 (br s, 4H), 2.30 (s, 3H); <sup>13</sup>C NMR (125 MHz, CDCl<sub>3</sub>) δ 160.9, 160.7 (d, <sup>1</sup><i>J</i><sub>C–F</sub> = 255.8 Hz, 1C), 150.0, 142.3, 141.0, 131.3, 131.1 (d, <sup>3</sup><i>J</i><sub>C–F</sub> = 10.4 Hz, 1C), 128.9 (d, <sup>4</sup><i>J</i><sub>C–F</sub> = 3.6 Hz, 1C), 122.9 (d, <sup>3</sup><i>J</i><sub>C–F</sub> = 14.8 Hz, 1C), 120.9, 120.5, 117.7, 117.5 (d, <sup>2</sup><i>J</i><sub>C–F</sub> = 25.0 Hz, 1C), 54.2 (2), 50.3 (2), 45.4. HRMS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>18</sub>H<sub>21</sub>FN<sub>5</sub>O<sub>5</sub>S [M + H]<sup>+</sup>: 438.1247, found: 438.1240.</div></div><div id="sec4_6_14" class="NLM_sec NLM_sec_level_3"><div id="ac_i59" class="anchor-spacer"></div><h4 class="article-section__title" id="_i59"> 2,3-Difluoro-5-hydroxy-<i>N</i>-(2-(4-methylpiperazin-1-yl)-5-nitrophenyl)benzamide (<b>12n</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_6" aria-label="general procedure D">general procedure D</a> (11 mg, 13% yield). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.60 (br s, 1H), 8.88 (d, <i>J</i> = 2.5 Hz, 1H), 8.01 (dd, <i>J</i> = 9.0, 2.5 Hz, 1H), 7.35 (q, <i>J</i> = 10.0 Hz, 1H), 7.31 (d, <i>J</i> = 9.0 Hz, 1H), 6.73 (m, 1H), 3.08 (br t, 4H), 2.60 (br s, 4H), 2.29 (s, 3H). <sup>13</sup>C NMR (125 MHz, DMSO-<i>d</i><sub>6</sub>) δ 161.3, 155.6, 150.0, 141.8 (2), 131.1, 120.5, 119.8 (2), 118.8 (d, <sup>2</sup><i>J</i><sub>C–F</sub> = 18.9 Hz), 117.9, 113.2 (d, <sup>3</sup><i>J</i><sub>C–F</sub> = 12.3 Hz), 111.9, 54.1 (2), 50.0 (2), 45.3. HRMS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>18</sub>H<sub>18</sub>F<sub>2</sub>N<sub>4</sub>O<sub>4</sub> [M + H]<sup>+</sup>: 393.1374, found: 393.1387.</div></div><div id="sec4_6_15" class="NLM_sec NLM_sec_level_3"><div id="ac_i60" class="anchor-spacer"></div><h4 class="article-section__title" id="_i60"> 3-Chloro-2-fluoro-<i>N</i>-(2-(4-methylpiperazin-1-yl)-5-nitrophenyl)-5-(trifluoromethyl) benzamide (<b>12o</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_5" aria-label="general procedure C">general procedure C</a> (31 mg, 51% yield). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.12 (br s, 1H), 8.83 (br s, 1H), 8.35 (d, <i>J</i> = 9.0 Hz, 1H), 8.19 (d, <i>J</i> = 2.8 Hz, 1H), 8.07 (dd, <i>J</i> = 9.0, 2.8 Hz, 1H), 7.36 (d, <i>J</i> = 9.0 Hz, 1H), 3.10 (br s, 4H), 2.56 (br s, 4H), 2.27 (s, 3H). <sup>13</sup>C NMR (125 MHz, DMSO-<i>d</i><sub>6</sub>) δ 161.0, 157.2 (d, <sup>a1</sup><i>J</i><sub>C–F</sub> = 250.0 Hz), 151.5, 142.2, 130.8, 127.1, 126.8 (q, <sup>b2</sup><i>J</i><sub>C–F</sub> = 50.0 Hz), 126.6, 126.5, 123.2 (q, <sup>b1</sup><i>J</i><sub>C–F</sub> = 295.1 Hz), 122.5, 122.1, 120.5, 119.8, 54.7 (2), 50.7 (2), 46.2. HRMS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>19</sub>H<sub>17</sub>ClF<sub>4</sub>N<sub>4</sub>O<sub>3</sub> [M + H]<sup>+</sup>: 461.1004, found: 461.1003.</div></div><div id="sec4_6_16" class="NLM_sec NLM_sec_level_3"><div id="ac_i61" class="anchor-spacer"></div><h4 class="article-section__title" id="_i61"> 2-Chloro-6-fluoro-3-methyl-<i>N</i>-(2-(4-methylpiperazin-1-yl)-5-nitrophenyl)benzamide (<b>12p</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_5" aria-label="general procedure C">general procedure C</a> (55 mg, 59% yield). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.29 (s, 1H), 8.54 (d, <i>J</i> = 2.5 Hz, 1H), 8.07 (dd, <i>J</i> = 9.0, 2.5 Hz, 1H), 7.52 (m, 1H), 7.31 (t, <i>J</i> = 9.0 Hz, 1H), 7.26 (d, <i>J</i> = 9.0 Hz, 1H), 3.11 (br t, 4H), 2.49 (br s, 4H), 2.37 (s, 3H), 2.22 (s, 3H). <sup>13</sup>C NMR (125 MHz, DMSO-<i>d</i><sub>6</sub>) δ 162.0, 157.4 (d, <sup>1</sup><i>J</i><sub>C–F</sub> = 243.8 Hz), 152.2, 141.5, 132.8, 132.7 (d, <sup>4</sup><i>J</i><sub>C–F</sub> = 13.8 Hz), 131.1 (d, <sup>5</sup><i>J</i><sub>C–F</sub> = 6.3 Hz), 129.8, 126.1 (d, <sup>2</sup><i>J</i><sub>C–F</sub> = 22.5 Hz), 122.1, 120.9, 119.9, 114.8 (d, <sup>3</sup><i>J</i><sub>C–F</sub> = 21.3 Hz), 54.6 (2), 50.4 (2), 46.2, 19.5. HRMS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>19</sub>H<sub>20</sub>ClFN<sub>4</sub>O<sub>3</sub> [M + H]<sup>+</sup>: 407.1286, found: 407.1293.</div></div><div id="sec4_6_17" class="NLM_sec NLM_sec_level_3"><div id="ac_i62" class="anchor-spacer"></div><h4 class="article-section__title" id="_i62"> 2,4-Difluoro-3-methyl-<i>N</i>-(2-(4-methylpiperazin-1-yl)-5-nitrophenyl)benzamide (<b>12q</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_4" aria-label="general procedure B">general procedure B</a> (34 mg, 86% yield). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.78 (s, 1H), 9.00 (s, 1H), 8.02 (dd, <i>J</i> = 9.0, 2.5 Hz, 1H), 7.85 (q, <i>J</i> = 7.0 Hz, 1H), 7.41 (d, <i>J</i> = 9.0 Hz, 1H), 7.27 (t, <i>J</i> = 8.5 Hz, 1H), 3.02 (br t, 4H), 2.51 (br s, 4H), 2.25 (s, 3H). <sup>13</sup>C NMR (125 MHz, DMSO-<i>d</i><sub>6</sub>) δ 163.1 (dd, <sup>1</sup><i>J</i><sub>C–F</sub> = 248.8, 8.8 Hz), 161.7, 158.9 (dd, <sup>1</sup><i>J</i><sub>C–F</sub> = 247.3, 8.8 Hz), 150.1, 143.0, 132.5, 129.8 (dd, <sup>3</sup><i>J</i><sub>C–F</sub> = 11.3, 3.8 Hz), 121.2, 120.9, 118.9 (dd, <sup>3</sup><i>J</i><sub>C–F</sub> = 11.3, 3.8 Hz), 117.2, 114.0 (t, <sup>2</sup><i>J</i><sub>C–F</sub> = 21.3 Hz), 112.3 (dd, <sup>2</sup><i>J</i><sub>C–F</sub> = 22.5, 2.5 Hz), 55.0 (2), 51.2 (2), 46.3, 7.4. HRMS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>19</sub>H<sub>20</sub>F<sub>2</sub>N<sub>4</sub>O<sub>3</sub> [M + H]<sup>+</sup>: 391.1582, found: 391.1585.</div></div><div id="sec4_6_18" class="NLM_sec NLM_sec_level_3"><div id="ac_i63" class="anchor-spacer"></div><h4 class="article-section__title" id="_i63"> 3,5-Difluoro-2-methyl-<i>N</i>-(2-(4-methylpiperazin-1-yl)-5-nitrophenyl)benzamide (<b>12r</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_6" aria-label="general procedure D">general procedure D</a> (71 mg, 68% yield). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.39 (s, 1H), 8.62 (s, 1H), 8.05 (dd, <i>J</i> = 2.7, 8.8 Hz, 1H), 7.33 (d, <i>J</i> = 8.8 Hz, 1H), 7.06–7.04 (m, 1H), 7.01–6.97 (m, 1H), 3.06 (s, 4H), 2.67 (s, 4H), 2.43 (s, 6H); <sup>13</sup>C NMR (125 MHz, DMSO-<i>d</i><sub>6</sub>) δ 165.4 (t, <sup>4</sup><i>J</i><sub>C–F</sub> = 2.9, 3.5 Hz, 1C), 162.0 (dd, <sup>1,3</sup><i>J</i><sub>C–F</sub> = 11.4, 249.3 Hz, 1C), 160.9 (dd, <sup>1,3</sup><i>J</i><sub>C–F</sub> = 12.7, 249.3 Hz, 1C), 147.1, 145.2, 138.5 (m), 133.6, 121.0, 120.7 (dd, <sup>2,4</sup><i>J</i><sub>C–F</sub> = 4.1, 18.2 Hz, 1C), 120.1, 115.3, 109.5 (dd, <sup>2,4</sup><i>J</i><sub>C–F</sub> = 3.7, 22.7 Hz, 1C), 106.2 (m), 55.4 (2), 51.8 (2), 46.0, 11.4. HRMS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>19</sub>H<sub>21</sub>F<sub>2</sub>N<sub>4</sub>O<sub>3</sub> [M + H]<sup>+</sup>: 391.1582, found: 391.1589.</div></div><div id="sec4_6_19" class="NLM_sec NLM_sec_level_3"><div id="ac_i64" class="anchor-spacer"></div><h4 class="article-section__title" id="_i64"> 4-Fluoro-3,5-dimethyl-<i>N</i>-(2-(4-methylpiperazin-1-yl)-5-nitrophenyl)benzamide (<b>12s</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_5" aria-label="general procedure C">general procedure C</a> (49 mg, 52% yield). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.63 (s, 1H), 8.66 (d, <i>J</i> = 2.5 Hz, 1H), 8.04 (dd, <i>J</i> = 8.5, 2.5 Hz, 1H), 7.74 (d, <i>J</i> = 6.5 Hz, 1H), 7.33 (d, <i>J</i> = 9.0 Hz, 1H), 3.08 (br t, 4H), 2.50 (br s, 4H), 2.32 (s, 3H), 2.19 (s, 6H). <sup>13</sup>C NMR (125 MHz, DMSO-<i>d</i><sub>6</sub>) δ 164.7, 161.9 (d, <sup>1</sup><i>J</i><sub>C–F</sub> = 247.5 Hz), 151.4, 142.2, 131.4, 129.9 (d, <sup>4</sup><i>J</i><sub>C–F</sub> = 2.5 Hz), 129.1 (2), 129.0, 124.8 (d, <sup>2</sup><i>J</i><sub>C–F</sub> = 18.9 Hz), 121.5, 120.4, 119.6, 55.2 (2), 50.6 (2), 46.3, 14.8 (2) (d, <sup>3</sup><i>J</i><sub>C–F</sub> = 3.8 Hz). HRMS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>20</sub>H<sub>23</sub>FN<sub>4</sub>O<sub>3</sub> [M + H]<sup>+</sup>: 387.1832, found: 387.1834.</div></div><div id="sec4_6_20" class="NLM_sec NLM_sec_level_3"><div id="ac_i65" class="anchor-spacer"></div><h4 class="article-section__title" id="_i65"> 2,4,6-Trifluoro-3,5-dimethyl-<i>N</i>-(2-(4-methylpiperazin-1-yl)-5-nitrophenyl)benzamide (<b>12t</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_5" aria-label="general procedure C">general procedure C</a> (17 mg, 22% yield). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.19 (s, 1H), 8.55 (d, <i>J</i> = 2.5 Hz, 1H), 8.06 (dd, <i>J</i> = 8.5, 3.0 Hz, 1H), 7.29 (d, <i>J</i> = 8.5 Hz, 1H), 3.09 (br t, 4H), 2.51 (br s, 4H), 2.23 (s, 3H), 2.18 (s, 6H). <sup>13</sup>C NMR (125 MHz, DMSO-<i>d</i><sub>6</sub>) δ 160.7, 159.3, 158.8, 156.5 (d, <sup>3</sup><i>J</i><sub>C–F</sub> = 12.5 Hz), 154.5 (d, <sup>3</sup><i>J</i><sub>C–F</sub> = 11.3 Hz), 151.9, 141.6, 129.9, 122.3, 120.5, 120.1, 111.5 (t, <sup>2</sup><i>J</i><sub>C–F</sub> = 21.1 Hz), 109.4 (t, <sup>2</sup><i>J</i><sub>C–F</sub> = 21.3 Hz), 54.4 (2), 50.5 (2), 46.2, 7.2 (2). HRMS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>20</sub>H<sub>21</sub>F<sub>3</sub>N<sub>4</sub>O<sub>3</sub> [M + H]<sup>+</sup>: 423.1644, found: 423.1653.</div></div><div id="sec4_6_21" class="NLM_sec NLM_sec_level_3"><div id="ac_i66" class="anchor-spacer"></div><h4 class="article-section__title" id="_i66"> 2-(4-Methylpiperazin-1-yl)-5-nitroaniline (<b>13</b>)</h4><div class="NLM_p last">A mixture of 2-fluoro-5-nitroaniline (<b>10</b>) (2.13 g, 13.7 mmol), 1-methylpiperazine (5 mL, 45 mmol) and <i>N</i>,<i>N</i>-diisopropylethylamine (3.6 mL, 20 mmol) was heated at 80 °C for 3 h. After cooling to 23 °C, the yellow precipitate was filtered off, washed with CH<sub>2</sub>Cl<sub>2</sub>, and dried under reduced pressure to afford the title compound (2.42 g, 75% yield). <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ 7.62 (dd, <i>J</i> = 2.6,8.7 Hz, 1H), 7.55 (d, <i>J</i> = 2.6 Hz, 1H), 6.99 (d, <i>J</i> = 8.7 Hz, 1H), 4.11 (br s, 2H), 3.03 (br s, 4H), 2.61 (br s, 4H), 2.38 (s, 3H); <sup>13</sup>C NMR (125 MHz, CDCl<sub>3</sub>) δ 145.2, 144.4, 14.6, 119.2, 114.6, 109.7, 55.6 (2), 50.3 (2), 46.3. HRMS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>11</sub>H<sub>17</sub>N<sub>4</sub>O<sub>2</sub> [M + H]<sup>+</sup>: 237.1352, found: 237.1357.</div></div></div><div id="sec4_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i67" class="anchor-spacer"></div><h3 class="article-section__title" id="_i67"> General Procedure E for the Synthesis of Compounds <b>14c</b>–<b>g</b>,<b>i</b>,<b>j</b></h3><div class="NLM_p">A solution of 2-(4-methylpiperazin-1-yl)-5-nitroaniline (<b>13</b>) (50 mg, 0.21 mmol) and the desired carboxylic acid (0.24 mmol) in xylene (2 mL) were heated at 120 °C for 5 min before POCl<sub>3</sub> (14 μL, 0.16 mmol) and DMAP (2.9 mg, 0.02 mmol) were added. The reaction mixture was heated at 120 °C for a further 2 h before cooling to 23 °C. The solvent was evaporated, and saturated aqueous sodium bicarbonate solution (2 mL) along with EtOAc (2 mL) were added to the residue. The aqueous layer was extracted with EtOAc (3 × 15 mL), and the combined organic extracts dried (MgSO<sub>4</sub>), filtered, and concentrated. The residue was purified by flash column chromatography on silica gel (5–30% MeOH/EtOAc) or precipitation from EtOAc with hexanes to afford the desired product.</div><div id="sec4_7_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i68" class="anchor-spacer"></div><h4 class="article-section__title" id="_i68"> <i>N</i>-(2-(4-Methylpiperazin-1-yl)-5-nitrophenyl)-6-oxo-4-(trifluoromethyl)-1,6-dihydropyridine-3-carboxamide (<b>14a</b>)</h4><div class="NLM_p last">The 6-hydroxy-4-(trifluoromethyl)nicotinic acid (117 mg, 0.554 mmol) was suspended in SOCl<sub>2</sub> (1.506 mL, 20.76 mmol) to form a white slurry. The suspension was stirred at 70 °C for 2 h until the solution went clear. The remaining SOCl<sub>2</sub> was removed under reduced pressure, and the residue dried under high vacuum for 1 h. The dried residue was subsequently dissolved in CH<sub>2</sub>Cl<sub>2</sub> (10 mL) and added dropwise over 20–30 min to a solution of 2-(4-methylpiperazin-1-yl)-5-nitroaniline (<b>13</b>) (109 mg, 0.461 mmol) and pyridine (0.074 mL, 0.923 mmol) in CH<sub>2</sub>Cl<sub>2</sub> (5.00 mL). After 2 h the reaction mixture was diluted with saturated aqueous sodium bicarbonate solution (10 mL) and extracted with CH<sub>2</sub>Cl<sub>2</sub> (5 × 10 mL) (sonication was necessary to dissolve the black solid formed). The combined organic extracts were dried (Na<sub>2</sub>SO<sub>4</sub>), filtered, and concentrated under reduced pressure. The residue was purified by flash column chromatography on silica gel (0–90% MeOH/CH<sub>2</sub>Cl<sub>2</sub>, 1% NH<sub>4</sub>Ac) to afford the title compound (59 mg, 30% yield) as a white solid. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.77 (s, 1H), 8.53 (d, <i>J</i> = 2.7 Hz, 1H), 8.04 (dd, <i>J</i> = 9.0, 2.8 Hz, 1H), 7.98 (s, 1H), 7.26 (d, <i>J</i> = 9.0 Hz, 1H), 6.82 (s, 1H), 3.34 (s, 4H), 3.10–3.04 (m, 4H), 2.24 (s, 3H); <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>) δ 163.06, 161.06, 151.62, 141.28, 139.55, 138.64 (q, <sup>2</sup><i>J</i><sub>C–F</sub> = 32.4 Hz), 130.24, 121.97 (q, <sup>1</sup><i>J</i><sub>C–F</sub> = 275.2 Hz), 121.48, 120.64, 119.49, 118.95 (q, <sup>3</sup><i>J</i><sub>C–F</sub> = 5.6 Hz), 110.99, 54.03, 49.89, 45.57; <sup>19</sup>F NMR (471 MHz, DMSO-<i>d</i><sub>6</sub>) δ −61.51. HRMS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>18</sub>H<sub>19</sub>F<sub>3</sub>N<sub>5</sub>O<sub>4</sub> [M + H]<sup>+</sup>: 426.1389, found: 426.1380.</div></div><div id="sec4_7_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i69" class="anchor-spacer"></div><h4 class="article-section__title" id="_i69"> 3-Hydroxy-2-methyl-<i>N</i>-(2-(4-methylpiperazin-1-yl)-5-nitrophenyl)quinoline-4-carboxamide (<b>14b</b>)</h4><div class="NLM_p last">To a solution of 3-hydroxy-2-methylquinoline-4-carboxylic acid (100 mg, 0.48 mmol) and triethlyamine (233 μL, 1.67 mmol) in EtOAc (5 mL) was added SOCl<sub>2</sub> (35 μL, 0.48 mmol) at 0 °C. The ice bath was then removed, and the solution allowed to stir at 23 °C for 1.5 h. Subsequently, 2-(4-methylpiperazin-1-yl)-5-nitroaniline (<b>13</b>) (90 mg, 0.382 mmol) was added, and the temperature was raised to 80 °C. After stirring for 1.5 h, the reaction mixture was cooled to 23 °C, and diluted with water (5 mL). The aqueous phase was extracted with EtOAc (3 × 10 mL), and the combined organic extracts were dried (MgSO<sub>4</sub>), filtered, and concentrated under reduced pressure. The residue was purified by flash column chromatography on silica gel (5–30% MeOH/EtOAc) to afford the title compound (25 mg, 11% yield). <sup>1</sup>H NMR (500 MHz, CD<sub>3</sub>OD) δ 9.32 (d, <i>J</i> = 2.7 Hz, 1H), 9.26 (d, <i>J</i> = 8.6 Hz, 1H), 7.96 (dd, <i>J</i> = 2.7, 9.0 Hz, 1H), 7.74 (dd, <i>J</i> = 1.1, 8.2 Hz, 1H), 7.56–7.48 (m, 1H), 7.39–7.35 (m, 1H), 7.27–7.22 (m, 2H), 3.18 (br s, 4H), 2.98 (t, <i>J</i> = 4.6 Hz, 4H), 2.65 (s, 3H), 2.50 (s, 3H); <sup>13</sup>C NMR (125 MHz, CD<sub>3</sub>OD) δ 170.8, 165.2, 161.5, 150.7, 144.8, 139.4, 135.3, 130.6, 128.1, 127.5, 127.3, 125.7, 124.3, 123.5, 120.2, 120.2, 118.9, 111.6, 55.5 (2), 51.3 (2), 46.1, 45.8. HRMS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>22</sub>H<sub>23</sub>N<sub>5</sub>O<sub>4</sub> [M + H]<sup>+</sup>: 422.1828, found: 422.1830.</div></div><div id="sec4_7_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i70" class="anchor-spacer"></div><h4 class="article-section__title" id="_i70"> 3-Hydroxy-<i>N</i>-(2-(4-methylpiperazin-1-yl)-5-nitrophenyl)quinoline-4-carboxamide (<b>14c</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_7" aria-label="general procedure E">general procedure E</a> (4 mg, 6% yield). <sup>1</sup>H NMR (500 MHz, CD<sub>3</sub>OD) δ 9.41 (d, <i>J</i> = 2.7 Hz, 1H), 9.19 (d, <i>J</i> = 8.8 Hz, 1H), 8.52 (s, 1H), 7.97 (dd, <i>J</i> = 2.7 Hz, 1H), 7.80 (dd, <i>J</i> = 1.3, 8.2 Hz, 1H), 7.48–7.45 (m, 1H), 7.35–7.30 (m, 2H), 3.35 (s, 1H), 3.20 (br s, 4H), 3.19 (br s, 4H), 2.70 (s, 3H); <sup>13</sup>C NMR (125 MHz, CD<sub>3</sub>OD) δ 169.6, 163.0, 152.1, 149.6, 145.4, 141.1, 135.5, 130.4, 129.3, 128.7, 126.1, 124.6, 120.9, 120.2, 118.4, 114.8, 55.4 (2), 51.0 (2), 45.0. HRMS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>21</sub>H<sub>22</sub>N<sub>5</sub>O<sub>4</sub> [M + H]<sup>+</sup>: 408.1672, found: 408.1666.</div></div><div id="sec4_7_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i71" class="anchor-spacer"></div><h4 class="article-section__title" id="_i71"> 6-Hydroxy-2-methyl-<i>N</i>-(2-(4-methylpiperazin-1-yl)-5-nitrophenyl)quinoline-4-carboxamide (<b>14d</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_7" aria-label="general procedure E">general procedure E</a> (25 mg, 23% yield). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.31 (br s, 1H), 10.13 (s, 1H), 10.11 (s, 1H), 8.84 (s, 1H), 8.11 (dd, <i>J</i> = 2.7, 9.0 Hz, 1H), 7.87 (d, <i>J</i> = 9.0 Hz, 1H), 7.57 (s, 1H), 7.43 (d, <i>J</i> = 2.6 Hz, 1H), 7.34 (dd, <i>J</i> = 2.7, 9.0 Hz, 1H), 3.35 (br s, 4H), 3.07 (br s, 4H), 2.66 (s, 3H), 2.63 (br s, 3H); <sup>13</sup>C NMR (125 MHz, DMSO-<i>d</i><sub>6</sub>) δ 166.3, 155.7, 154.5, 150.2, 143.0, 141.9, 139.6, 130.6, 130.2, 123.6, 122.2, 121.6, 120.2, 120.0, 119.9, 105.6, 52.4 (2), 47.7 (2), 24.4 (2). HRMS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>22</sub>H<sub>24</sub>N<sub>5</sub>O<sub>4</sub> [M + H]<sup>+</sup>: 422.1828, found: 422.1829.</div></div><div id="sec4_7_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i72" class="anchor-spacer"></div><h4 class="article-section__title" id="_i72"> <i>N</i>-(2-(4-Methylpiperazin-1-yl)-5-nitrophenyl)pyrazolo[1,5-<i>a</i>]pyridine-7-carboxamide (<b>14e</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_7" aria-label="general procedure E">general procedure E</a> (12 mg, 24% yield). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 13.17 (s, 1H), 9.39 (d, <i>J</i> = 2.7 Hz, 1H), 8.35 (d, <i>J</i> = 2.4 Hz, 1H), 8.10 (dd, <i>J</i> = 1.3, 8.8 Hz, 1H), 8.05 (dd, <i>J</i> = 2.9, 8.9 Hz, 1H), 8.03 (dd, <i>J</i> = 1.3, 7.1 Hz, 1H), 7.53–7.50 (m, 1H), 7.47 (d, <i>J</i> = 8.9 Hz, 1H), 7.01 (d, <i>J</i> = 2.5 Hz, 1H), 3.03 (br s, 4H), 2.61 (br s, 4H), 2.30 (s, 3H); <sup>13</sup>C NMR (125 MHz, DMSO-<i>d</i><sub>6</sub>) δ 158.2, 149.3, 142.9, 141.3, 140.9, 132.9, 130.7, 123.8, 122.6, 120.7, 120.2, 118.2, 115.6, 98.9, 54.5 (2), 50.8 (2), 45.7. HRMS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>19</sub>H<sub>21</sub>N<sub>6</sub>O<sub>3</sub> [M + H]<sup>+</sup>: 381.1675, found: 381.1688.</div></div><div id="sec4_7_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i73" class="anchor-spacer"></div><h4 class="article-section__title" id="_i73"> 5-Fluoro-<i>N</i>-(2-(4-methylpiperazin-1-yl)-5-nitrophenyl)-1<i>H</i>-benzo[<i>d</i>]imidazole-7-carboxamide (<b>14f</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_7" aria-label="general procedure E">general procedure E</a> (7 mg, 11% yield). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 13.24 (br s, 1H), 12.26 (s, 1H), 9.37(s, 1H), 8.64 (s, 1H), 8.01 (dd, <i>J</i> = 2.6, 8.8 Hz, 1H), 7.77 (dd, <i>J</i> = 2.3, 10.6 Hz, 1H), 7.71 (dd, <i>J</i> = 2.4, 8.5 Hz, 1H), 7.43 (d, <i>J</i> = 9.0 Hz, 1H), 3.02 (br s, 4H), 2.61 (br s, 4H), 2.28 (s, 3H); <sup>13</sup>C NMR (125 MHz, DMSO-<i>d</i><sub>6</sub>) δ 162.4, 158.5 (d, <sup>1</sup><i>J</i><sub>C–F</sub> = 237.3 Hz, 1C), 149.3, 144.4, 142.8, 136.8, 134.2 (d, <sup>3</sup><i>J</i><sub>C–F</sub> = 11.5 Hz, 1C), 133.2, 122.6 (d, <sup>3</sup><i>J</i><sub>C–F</sub> = 9.2 Hz, 1C), 120.5, 119.7, 115.6, 110.6 (d, <sup>2</sup><i>J</i><sub>C–F</sub> = 27.1 Hz, 1C), 103.1 (d, <sup>2</sup><i>J</i><sub>C–F</sub> = 27.1 Hz, 1C), 54.3 (2), 50.9 (2), 45.8. HRMS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>19</sub>H<sub>20</sub>FN<sub>6</sub>O<sub>3</sub> [M + H]<sup>+</sup>: 399.1581, found: 399.1582.</div></div><div id="sec4_7_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i74" class="anchor-spacer"></div><h4 class="article-section__title" id="_i74"> <i>N</i>-(2-(4-Methylpiperazin-1-yl)-5-nitrophenyl)-1<i>H</i>-benzo[<i>d</i>]imidazole-5-carboxamide (<b>14g</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_7" aria-label="general procedure E">general procedure E</a> (15 mg, 19% yield). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.85 (br s, 1H), 9.67 (br s, 1H), 8.75 (d, <i>J</i> = 2.5 Hz, 1H), 8.41 (s, 1H), 8.30 (br s, 1H), 8.04 (dd, <i>J</i> = 9.0, 2.5 Hz, 1H), 7.87 (d, <i>J</i> = 8.5 Hz, 1H), 7.74 (d, <i>J</i> = 8.5 Hz, 1H), 7.35 (d, <i>J</i> = 9.0 Hz, 1H), 3.09 (br t, 4H), 2.51 (br s, 4H), 2.24 (s, 3H). <sup>13</sup>C NMR (125 MHz, DMSO-<i>d</i><sub>6</sub>) δ 165.9, 151.4, 144.9, 143.9, 142.3, 140.0, 131.9, 128.0, 121.2, 120.4 (2), 119.5, 113.6, 110.1, 55.1 (2), 50.7 (2), 46.2. HRMS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>19</sub>H<sub>20</sub>N<sub>6</sub>O<sub>3</sub> [M + H]<sup>+</sup>: 381.1675, found: 381.1671.</div></div><div id="sec4_7_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i75" class="anchor-spacer"></div><h4 class="article-section__title" id="_i75"> 3-Methyl-<i>N</i>-(2-(4-methylpiperazin-1-yl)-5-nitrophenyl)-[1,2,4]triazolo[4,3-<i>a</i>]pyridine-6-carboxamide (<b>14h</b>)</h4><div class="NLM_p last">3-Methyl-[1,2,4]triazolo[4,3-<i>a</i>]pyridine-6-carboxylic acid (48.7 mg, 0.275 mmol) was suspended in SOCl<sub>2</sub> (1 mL) and stirred at 60 °C for 1 h. Excess SOCl<sub>2</sub> was removed under reduced pressure, and the residue was dried under high vacuum for 1 h. The dried residue was then dissolved in CH<sub>2</sub>Cl<sub>2</sub> (3 mL) and added dropwise to a solution of 2-(4-methylpiperazin-1-yl)-5-nitroaniline (<b>13</b>) (50 mg, 0.21 mmol) and pyridine (22 μL, 0.28 mmol) in CH<sub>2</sub>Cl<sub>2</sub> (2 mL). After 1.5 h, the reaction mixture was diluted with saturated aqueous sodium bicarbonate solution (5 mL) and extracted with CH<sub>2</sub>Cl<sub>2</sub> (3 × 5 mL). The combined organic extracts were dried (MgSO<sub>4</sub>), filtered, and concentrated under reduced pressure. The residue was purified by flash column chromatography on silica gel (5–30% MeOH/EtOAc) to afford the title compound (6 mg, 6% yield). <sup>1</sup>H NMR (500 MHz, CD<sub>3</sub>OD) δ 9.14 (t, <i>J</i> = 1.3 Hz, 1H), 8.82 (d, <i>J</i> = 2.8 Hz, 1H), 8.19 (dd, <i>J</i> = 2.7, 8.9 Hz, 1H), 8.01 (dd, <i>J</i> = 1.7, 9.7 Hz, 1H), 7.88 (dd, <i>J</i> = 1.0, 9.5 Hz, 1H), 7.49 (d, <i>J</i> = 8.9 Hz, 1H), 3.62 (br s, 4H), 3.43 (br s, 2H), 3.30 (br s, 2H), 3.01 (s, 3H), 2.93 (s, 3H); <sup>13</sup>C NMR (125 MHz, CD<sub>3</sub>OD) δ 165.2, 150.8, 150.6, 147.5, 145.5, 133.1, 129.0, 127.7, 123.5, 123.0, 122.1, 121.9, 115.8, 115.5, 54.9 (2), 43.8 (2), 10.2. HRMS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>19</sub>H<sub>21</sub>N<sub>7</sub>O<sub>3</sub> [M + H]<sup>+</sup>: 396.1784, found: 396.1779.</div></div><div id="sec4_7_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i76" class="anchor-spacer"></div><h4 class="article-section__title" id="_i76"> <i>N</i>-(2-(4-Methylpiperazin-1-yl)-5-nitrophenyl)-1<i>H</i>-indazole-6-carboxamide (<b>14i</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_7" aria-label="general procedure E">general procedure E</a> (8 mg, 8% yield). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 13.51 (br s, 1H), 9.80 (s, 1H), 8.70 (d, <i>J</i> = 2.8 Hz, 1H), 8.20 (d, <i>J</i> = 7.5 Hz, 2H), 8.06 (dd, <i>J</i> = 2.7, 8.9 Hz, 1H), 7.94 (d, <i>J</i> = 8.4 Hz, 1H), 7.70 (dd, <i>J</i> = 1.3, 8.4 Hz, 1H), 7.34 (d, <i>J</i> = 9.0 Hz, 1H), 3.10 (t, <i>J</i> = 5.0 Hz, 4H), 2.51 (br s, 4H), 2.23 (s, 3H); <sup>13</sup>C NMR (125 MHz, DMSO-<i>d</i><sub>6</sub>) δ 165.3, 151.2, 141.6, 139.3, 133.7, 131.6, 130.9, 124.7, 121.1, 120.9, 119.9, 119.5, 119.0, 111.0, 54.6 (2), 50.1 (2), 45.7. HRMS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>19</sub>H<sub>22</sub>N<sub>6</sub>O<sub>3</sub> [M + H]<sup>+</sup>: 381.1675, found: 381.1670.</div></div><div id="sec4_7_10" class="NLM_sec NLM_sec_level_3"><div id="ac_i77" class="anchor-spacer"></div><h4 class="article-section__title" id="_i77"> 1-Methyl-<i>N</i>-(2-(4-methylpiperazin-1-yl)-5-nitrophenyl)-1<i>H</i>-indazole-3-carboxamide (<b>14j</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_7" aria-label="general procedure E">general procedure E</a> (31 mg, 41% yield). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.87 (br s, 1H), 9.26 (d, <i>J</i> = 2.5 Hz, 1H), 8.26 (d, <i>J</i> = 8.0 Hz, 1H), 8.01 (dd, <i>J</i> = 9.0, 3.0 Hz, 1H), 7.83 (d, <i>J</i> = 8.5 Hz, 1H), 7.56 (t, <i>J</i> = 7.5 Hz, 1H), 7.45 (d, <i>J</i> = 8.5 Hz, 1H), 7.40 (t, <i>J</i> = 7.5 Hz, 1H), 4.23 (s, 3H), 3.04 (br t, 4H), 2.51 (br s, 4H), 2.33 (s, 3H). <sup>13</sup>C NMR (125 MHz, DMSO-<i>d</i><sub>6</sub>) δ 160.2, 148.3, 143.7, 141.9, 136.3, 133.0, 127.6, 123.8, 122.5, 121.8, 121.2, 119.6, 123.8, 111.4, 55.4 (2), 51.4 (2), 46.4, 36.9. HRMS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>20</sub>H<sub>22</sub>N<sub>6</sub>O<sub>3</sub> [M + H]<sup>+</sup>: 395.1832, found: 395.1828.</div></div><div id="sec4_7_11" class="NLM_sec NLM_sec_level_3"><div id="ac_i78" class="anchor-spacer"></div><h4 class="article-section__title" id="_i78"> <i>N</i>-(5-Bromo-2-(4-methylpiperazin-1-yl)phenyl)-6-oxo-4-(trifluoromethyl)-1,6-dihydropyridine-3-carboxamide (<b>15</b>)</h4><div class="NLM_p last">The 6-hydroxy-4-(trifluoromethyl)nicotinic acid (1.890 g, 8.94 mmol) was suspended in SOCl<sub>2</sub> (24.33 mL, 335 mmol) and stirred at 80 °C for 2 h. The solution became clear and was then cooled to 23 °C. The excess SOCl<sub>2</sub> was removed under reduced pressure, and the resulting solid was dried under vacuum for 2 h. The dry residue was dissolved in CH<sub>2</sub>Cl<sub>2</sub> (10 mL) and added over a 30 min period to a solution of 5-bromo-2-(4-methylpiperazin-1-yl)aniline (<b>7</b>) (2.013 g, 7.45 mmol) and pyridine (1.801 mL, 22.36 mmol) in CH<sub>2</sub>Cl<sub>2</sub> (20 mL). The resulting mixture was then stirred for 12 h. The reaction was diluted with saturated aqueous sodium bicarbonate solution (100 mL), sonicated to dissolve any solid particles, and extracted with CH<sub>2</sub>Cl<sub>2</sub> (4 × 50 mL). The combined organic extracts were dried (Na<sub>2</sub>SO<sub>4</sub>), filtered, and concentrated under reduced pressure. The residue was purified by flash column chromatography on silica gel (0–100%, 89% CH<sub>2</sub>Cl<sub>2</sub>, 10% MeOH, 1% NH<sub>4</sub>Ac/CH<sub>2</sub>Cl<sub>2</sub>) to afford the title compound (366 mg, 10% yield) as a yellow solid. <sup>1</sup>H NMR (500 MHz, MeOD) δ 8.20 (d, <i>J</i> = 2.3 Hz, 1H), 7.95 (s, 1H), 7.32 (dd, <i>J</i> = 8.6, 2.3 Hz, 1H), 7.18 (d, <i>J</i> = 8.6 Hz, 1H), 6.91 (s, 1H), 2.95 (t, <i>J</i> = 4.8 Hz, 4H), 2.69 (s, 4H), 2.40 (s, 3H); <sup>13</sup>C NMR (126 MHz, MeOD) δ 164.7, 164.1, 144.2, 141.4 (q, <sup>2</sup><i>J</i><sub>C–F</sub> = 33.2 Hz, 1C), 139.7, 135.1, 129.6, 126.4, 123.5, 123.3 (q, <sup>1</sup><i>J</i><sub>C–F</sub> = 274.7 Hz, 1C), 120.48 (q, <sup>3</sup><i>J</i><sub>C–F</sub> = 5.4 Hz, 1C), 118.5, 114.5, 56.1, 52.3, 45.9; <sup>19</sup>F NMR (471 MHz, MeOD) δ −63.94. HRMS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>18</sub>H<sub>19</sub>BrF<sub>3</sub>N<sub>4</sub>O<sub>2</sub> [M + H]<sup>+</sup>: 459.0643, found: 459.0647.</div></div></div><div id="sec4_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i79" class="anchor-spacer"></div><h3 class="article-section__title" id="_i79"> General Procedure F for Synthesis of Compounds <b>16a</b>–<b>h</b></h3><div class="NLM_p">The <i>N</i>-(5-bromo-2-(4-methylpiperazin-1-yl)phenyl)-6-oxo-4-(trifluoromethyl)-1,6-dihydropyridine-3-carboxamide (<b>15</b>) (1 equiv), boronic acid/ester (3 equiv), sodium carbonate (10 equiv), XPhos (0.2 equiv), and XPhos Pd G2 (0.2 equiv) in 1,4-dioxane and water (5:3, 0.02 M) were mixed in a 5 mL microwave vial. The mixture was stirred for 5 min, degassed, purged with N<sub>2</sub>, and irradiated for 60 min at 120 °C. After cooling to 23 °C, all solvents were removed under reduced pressure, and the crude material purified using flash column chromatography on silica gel (0–100%, 89% CH<sub>2</sub>Cl<sub>2</sub>, 10% MeOH, 1% NH<sub>4</sub>Ac/CH<sub>2</sub>Cl<sub>2</sub>) to afford the desired compound.</div><div id="sec4_8_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i80" class="anchor-spacer"></div><h4 class="article-section__title" id="_i80"> <i>N</i>-(5-(2,3-Dihydrobenzofuran-5-yl)-2-(4-methylpiperazin-1-yl)phenyl)-6-oxo-4-(trifluoromethyl)-1,6-dihydropyridine-3-carboxamide (<b>16a</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_8" aria-label="general procedure F">general procedure F</a> (50 mg, 44% yield) as a white solid. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.39 (s, 1H), 8.05 (s, 1H), 7.97 (s, 1H), 7.45 (s, 1H), 7.36 (dd, <i>J</i> = 8.3, 1.8 Hz, 1H), 7.31 (d, <i>J</i> = 8.2 Hz, 1H), 7.21 (d, <i>J</i> = 8.3 Hz, 1H), 6.83 (d, <i>J</i> = 8.4 Hz, 1H), 6.82 (s, 1H), 4.56 (t, <i>J</i> = 8.7 Hz, 2H), 3.24 (t, <i>J</i> = 8.6 Hz, 2H), 2.87 (s, 4H), 2.48 (s, 4H), 2.23 (s, 3H); <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>) δ 161.0, 159.6, 157.6, 141.5, 137.5, 137.0 (q, <sup>2</sup><i>J</i><sub>C–F</sub> = 32.5 Hz, 1C), 134.3, 130.7, 126.6, 124.5, 121.5, 121.4, 120.4 (q, <sup>1</sup><i>J</i><sub>C–F</sub> = 275.1 Hz, 1C), 119.5, 118.8, 117.3 (q, <sup>3</sup><i>J</i><sub>C–F</sub> = 5.5 Hz, 1C), 110.26, 107.54, 69.51, 53.20, 49.61, 44.11, 27.45; <sup>19</sup>F NMR (471 MHz, DMSO-<i>d</i><sub>6</sub>) δ −61.43. HRMS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>26</sub>H<sub>26</sub>F<sub>3</sub>N<sub>4</sub>O<sub>3</sub> [M + H]<sup>+</sup>: 499.1957, found: 499.1955.</div></div><div id="sec4_8_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i81" class="anchor-spacer"></div><h4 class="article-section__title" id="_i81"> <i>N</i>-(2-(4-Methylpiperazin-1-yl)-5-(quinolin-3-yl)phenyl)-6-oxo-4-(trifluoromethyl)-1,6-dihydropyridine-3-carboxamide (<b>16b</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_8" aria-label="general procedure F">general procedure F</a> (31 mg, 97% yield). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.55 (s, 1H), 9.19 (d, <i>J</i> = 2.0 Hz, 1H), 8.54 (d, <i>J</i> = 1.7 Hz, 1H), 8.24 (d, <i>J</i> = 1.5 Hz, 1H), 8.08 (d, <i>J</i> = 8.1 Hz, 1H), 8.05 (d, <i>J</i> = 8.4 Hz, 1H), 8.01 (s, 1H), 7.76 (t, <i>J</i> = 7.6 Hz, 1H), 7.69 (dd, <i>J</i> = 8.3, 1.9 Hz, 1H), 7.64 (t, <i>J</i> = 7.5 Hz, 1H), 7.33 (d, <i>J</i> = 8.3 Hz, 1H), 6.82 (s, 1H), 3.35 (s, 4H), 2.94 (s, 4H), 2.24 (s, 3H); <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>) δ 163.4, 161.8, 149.7, 147.2, 145.7, 140.0, 139.1 (q, <sup>2</sup><i>J</i><sub>C–F</sub> = 32.7 Hz, 1C), 132.9, 132.6, 132.5, 129.9, 129.1, 128.8, 128.2, 127.5, 124.9, 123.3, 122.5 (q, <sup>1</sup><i>J</i><sub>C–F</sub> = 275.2 Hz, 1C), 121.2, 119.3 (q, <sup>3</sup><i>J</i><sub>C–F</sub> = 5.5 Hz, 1C), 112.1, 55.2, 51.5, 46.2; <sup>19</sup>F NMR (471 MHz, DMSO-<i>d</i><sub>6</sub>) δ −61.39. HRMS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>27</sub>H<sub>25</sub>F<sub>3</sub>N<sub>5</sub>O<sub>2</sub> [M + H]<sup>+</sup>: 508.1960, found: 508.1969.</div></div><div id="sec4_8_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i82" class="anchor-spacer"></div><h4 class="article-section__title" id="_i82"> <i>N</i>-(5-(1<i>H</i>-Indol-5-yl)-2-(4-methylpiperazin-1-yl)phenyl)-6-oxo-4-(trifluoromethyl)-1,6-dihydropyridine-3-carboxamide (<b>16c</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_8" aria-label="general procedure F">general procedure F</a> (47 mg, 26% yield). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.50 (br s, 1H), 11.15 (br s, 1H), 9.42 (s, 1H), 8.16 (s, 1H), 7.99 (s, 1H), 7.75 (s, 1H), 7.45 (d, <i>J</i> = 8.0 Hz, 1H), 7.38 (s, 1H), 7.34 (d, <i>J</i> = 8.3 Hz, 1H), 7.25 (d, <i>J</i> = 8.3 Hz, 1H), 6.84 (s, 1H), 6.50 (br s, 1H), 2.94–2.85 (m, 4H), 2.51 (br s, 4H), 2.25 (s, 3H); <sup>13</sup>C NMR (125 MHz, DMSO-<i>d</i><sub>6</sub>) δ 163.1, 161.6, 143.3, 139.5, 139.1 (q, <sup>2</sup><i>J</i><sub>C–F</sub> = 32.7 Hz), 137.9, 135.9, 132.8, 131.4, 128.7, 126.6, 123.9, 122.0, 120.9, 120.6, 119.5, 122.4 (q, <sup>1</sup><i>J</i><sub>C–F</sub> = 275.2 Hz), 118.2, 112.46, 112.3, 102.0, 55.4 (2), 51.8 (2), 46.23. HRMS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>26</sub>H<sub>25</sub>F<sub>3</sub>N<sub>5</sub>O<sub>2</sub> [M + H]<sup>+</sup>: 496.1960, found: 496.1963.</div></div><div id="sec4_8_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i83" class="anchor-spacer"></div><h4 class="article-section__title" id="_i83"> <i>N</i>-(4-(4-Methylpiperazin-1-yl)-3′-(morpholinomethyl)-[1,1′-biphenyl]-3-yl)-6-oxo-4-(trifluoromethyl)-1,6-dihydropyridine-3-carboxamide (<b>16d</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_8" aria-label="general procedure F">general procedure F</a> (47 mg, 26% yield). <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>) δ 8.95 (s, 1H), 8.64 (d, <i>J</i> = 0.9 Hz, 1H), 7.89 (s, 1H), 7.55 (s, 1H), 7.50 (d, <i>J</i> = 7.7 Hz, 1H), 7.38–7.33 (m, 2H), 7.32–7.28 (m, 2H), 6.97 (s, 1H), 3.72–3.68 (m, 4H), 3.55 (s, 2H), 2.97–2.90 (m, 4H), 2.63 (s, 4H), 2.49–2.44 (m, 4H), 2.36 (s, 3H); <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>) δ 163.6, 161.9, 140.8, 140.6, 140.0 (q, <sup>2</sup><i>J</i><sub>C–F</sub> = 33.3 Hz, 1C), 139.0, 138.4, 138.3, 133.7, 128.8, 128.4, 128.1, 126.2, 123.5, 121.8 (q, <sup>1</sup><i>J</i><sub>C–F</sub> = 275.4 Hz, 1C), 121.6, 119.6 (q, <sup>3</sup><i>J</i><sub>C–F</sub> = 5.3 Hz, 1C), 118.6, 116.4, 114.8, 67.1, 63.5, 53.7, 52.4, 46.0, 40.9; <sup>19</sup>F NMR (471 MHz, CDCl<sub>3</sub>) δ −62.33. HRMS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>29</sub>H<sub>33</sub>F<sub>3</sub>N<sub>5</sub>O<sub>3</sub> [M + H]<sup>+</sup>: 556.2535, found: 556.2526.</div></div><div id="sec4_8_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i84" class="anchor-spacer"></div><h4 class="article-section__title" id="_i84"> <i>N</i>-(5-(6-Acetamidopyridin-3-yl)-2-(4-methylpiperazin-1-yl)phenyl)-6-oxo-4-(trifluoromethyl)-1,6-dihydropyridine-3-carboxamide (<b>16e</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_8" aria-label="general procedure F">general procedure F</a> (47 mg, 26% yield). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.55 (br s, 1H), 10.60 (s, 1H), 9.48 (s, 1H), 8.55 (d, <i>J</i> = 2.2 Hz, 1H), 8.16 (d, <i>J</i> = 8.7 Hz, 1H), 8.10–8.06 (m, 1H), 8.02–7.97 (m, 2H), 7.50 (dd, <i>J</i> = 1.8, 8.3 Hz, 1H), 7.27 (d, <i>J</i> = 8.3 Hz, 1H), 6.82 (s, 1H), 2.93–2.86 (m, 4H), 2.24 (s, 3H), 2.15–2.09 (m, 3H); <sup>13</sup>C NMR (125 MHz, DMSO-<i>d</i><sub>6</sub>) δ 170.6, 163.6, 162.8, 150.9, 145.5, 143.6, 140.2, 138.4, 136.0, 133.8, 132.6, 132.0, 124.0, 123.1, 120.9, 120.9, 119.1, 113.9, 113.2, 54.7 (2), 51.0 (2), 44.4, 22.6; <sup>19</sup>F NMR (471 MHz, DMSO-<i>d</i><sub>6</sub>) δ −61.41. HRMS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>25</sub>H<sub>26</sub>F<sub>3</sub>N<sub>6</sub>O<sub>3</sub> [M + H]<sup>+</sup>: 515.2018, found: 515.2012.</div></div><div id="sec4_8_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i85" class="anchor-spacer"></div><h4 class="article-section__title" id="_i85"> <i>N</i>-(5-(6-Aminopyridin-3-yl)-2-(4-methylpiperazin-1-yl)phenyl)-6-oxo-4-(trifluoromethyl)-1,6-dihydropyridine-3-carboxamide (<b>16f</b>)</h4><div class="NLM_p last">The title compound was prepared by using <a class="ref internalNav" href="#sec4_8" aria-label="general procedure F">general procedure F</a> (154 mg, 58% yield). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.30 (br s, 1H), 8.45 (s, 1H), 8.16 (d, <i>J</i> = 2.5 Hz, 1H), 8.08 (s, 1H), 8.04 (s, 1H), 7.61 (dd, <i>J</i> = 8.5, 2.5 Hz, 1H), 7.32 (dd, <i>J</i> = 8.0, 2.0 Hz, 1H), 7.22 (d, <i>J</i> = 8.0 Hz, 1H), 6.67 (br s, 1H), 6.53 (d, <i>J</i> = 8.5 Hz, 1H), 6.04 (s, 2H), 2.86 (br t, 4H), 2.48 (br s, 4H), 2.23 (s, 3H). <sup>13</sup>C NMR (125 MHz, DMSO-<i>d</i><sub>6</sub>) δ 165.7, 163.6, 159.5, 145.9, 142.9 (2), 135.6, 134.4, 133.4, 124.2, 122.3, 121.1, 119.9, 117.5, 111.6, 108.5 (2), 55.4 (2), 51.8 (2), 46.2. HRMS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>23</sub>H<sub>24</sub>F<sub>3</sub>N<sub>6</sub>O<sub>2</sub> [M + H]<sup>+</sup>: 473.1913, found: 473.1909.</div></div><div id="sec4_8_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i86" class="anchor-spacer"></div><h4 class="article-section__title" id="_i86"> <i>N</i>-(4′-Amino-4-(4-methylpiperazin-1-yl)-[1,1′-biphenyl]-3-yl)-6-oxo-4-(trifluoromethyl)-1,6-dihydropyridine-3-carboxamide (<b>16g</b>)</h4><div class="NLM_p last">The title compound was prepared by using <a class="ref internalNav" href="#sec4_8" aria-label="general procedure F">general procedure F</a> (59 mg, 64% yield). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.30 (br s, 1H), 8.42 (s, 1H), 8.06 (d, <i>J</i> = 2.0 Hz, 1H), 8.00 (s, 1H), 7.31–7.26 (m, 3H), 7.19 (d, <i>J</i> = 10.0 Hz, 1H), 6.72 (br s, 1H), 6.66–6.62 (m, 2H), 5.20 (br s, 2H), 2.86 (br t, 4H), 2.48 (br s, 4H), 2.23 (s, 3H). <sup>13</sup>C NMR (125 MHz, DMSO-<i>d</i><sub>6</sub>) δ 165.7, 163.4, 148.7, 142.5, 141.6, 137.0, 133.l, 128.1, 127.6, 127.4 (2), 122.3, 120.9, 120.0, 118.2, 114.7 (2), 112.0, 55.5 (2), 51.9 (2), 46.2; <sup>19</sup>F NMR (471 MHz, DMSO-<i>d</i><sub>6</sub>) δ −61.15. HRMS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>24</sub>H<sub>25</sub>F<sub>3</sub>N<sub>5</sub>O<sub>2</sub> [M + H]<sup>+</sup>: 472.1960, found: 472.1952.</div></div><div id="sec4_8_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i87" class="anchor-spacer"></div><h4 class="article-section__title" id="_i87"> <i>N</i>-(2-(4-Methylpiperazin-1-yl)-5-(1-(2-morpholinoethyl)-1<i>H</i>-pyrazol-4-yl)phenyl)-6-oxo-4-(trifluoromethyl)-1,6-dihydropyridine-3-carboxamide (<b>16h</b>)</h4><div class="NLM_p last">The title compound was prepared according to <a class="ref internalNav" href="#sec4_8" aria-label="general procedure F">general procedure F</a> (42 mg, 68% yield). <sup>1</sup>H NMR (500 MHz, MeOD) δ 8.20 (d, <i>J</i> = 1.6 Hz, 1H), 8.01 (s, 1H), 7.98 (s, 1H), 7.80 (s, 1H), 7.38 (dd, <i>J</i> = 8.3, 2.0 Hz, 1H), 7.25 (d, <i>J</i> = 8.3 Hz, 1H), 6.92 (s, 1H), 4.29 (t, <i>J</i> = 6.5 Hz, 2H), 3.68–3.64 (m, 4H), 2.96 (t, <i>J</i> = 4.7 Hz, 4H), 2.82 (t, <i>J</i> = 6.5 Hz, 2H), 2.68 (s, 4H), 2.52–2.47 (m, 4H), 2.38 (s, 3H); <sup>13</sup>C NMR (126 MHz, MeOD) δ 164.8, 164.4, 143.3, 141.4 (q, <sup>2</sup><i>J</i><sub>C–F</sub> = 33.2 Hz, 1C), 139.8, 137.5, 134.1, 130.8, 128.9, 123.8, 123.7, 123.4 (q, <sup>1</sup><i>J</i><sub>C–F</sub> = 274.7 Hz, 1C), 122.2, 120.3 (q, <sup>3</sup><i>J</i><sub>C–F</sub> = 4.5 Hz, 1C), 114.8, 67.9, 59.2, 56.3, 54.7, 52.6, 50.3, 46.0; <sup>19</sup>F NMR (471 MHz, MeOD) δ −63.80. HRMS (ESI) <i>m</i>/<i>z</i> calcd for C<sub>27</sub>H<sub>33</sub>F<sub>3</sub>N<sub>7</sub>O<sub>3</sub> [M + H]<sup>+</sup>: 560.2597, found: 560.2595.</div></div></div><div id="sec4_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i88" class="anchor-spacer"></div><h3 class="article-section__title" id="_i88"> Cloning, Expression, and Purification of Human WDR5</h3><div class="NLM_p last">The DNA fragment encoding human WDR5 (residues 1–334) was amplified by PCR and subcloned into pET28-LIC vector, downstream of the poly histidine coding region. The protein was expressed in <i>E. coli</i> BL21 (DE3)-V2R-pRARE2 strain by addition of 1 mM isopropyl-1-thio-<span class="smallcaps smallerCapital">d</span>-galactopyranoside (IPTG) and incubated overnight at 15 °C. Harvested cells were resuspended in 20 mM Tris buffer (pH 7.5) supplemented with 500 mM NaCl, 5 mM imidazole, and 5% glycerol. The cells were lysed chemically, followed by sonication at a frequency of 8.5 (10 s on/10 s off) for 4 min. After clarification of the crude extract by high-speed centrifugation, the lysate was loaded onto DE52 and passed onto a Ni-NTA column. The column was washed and eluted sequentially with 20 mM Tris (pH 7.5), 500 mM NaCl, 5% glycerol, 30 mM and 250 mM imidazole. Thrombin was added while the protein was being dialyzed against 20 mM Tris buffer (pH 7.5), 2.5 mM CaCl<sub>2</sub>, and 500 mM NaCl. To remove the cut his-tag and his-tagged protein, dialyzed protein solution was passed through a Ni-NTA column. Flow through was dialyzed again against 20 mM Tris buffer (pH 7.5), 2.5 mM CaCl<sub>2</sub>, and 500 mM NaCl. Pure protein (>95%) was further concentrated to higher than 20 mg/mL and stored at −80 °C after flash freezing.</div></div><div id="sec4_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i89" class="anchor-spacer"></div><h3 class="article-section__title" id="_i89"> Fluorescence Polarization (FP) Binding Assays</h3><div class="NLM_p last">H3(1–15) (ARTKQTARKSTGGKA) and 9-Ala-FAM ((Ac)-ARAEVHLRK-(Ahx-Ahx)-K(5,6-FAM)) (where Ahx represents 6 amino hexanoic acid linkers)<a onclick="showRef(event, 'ref12 ref16'); return false;" href="javascript:void(0);" class="ref ref12 ref16">(12, 16)</a> peptides for WDR5 were synthesized, N-terminal-labeled with isothiocyanate-fluorescein, and purified by Tufts University Core Services (Boston, MA). Compound binding assays were performed at a constant labeled peptide concentration of 30 or 20 nM for H3 (1–15) and 9-Ala-FAM respectively. WDR5 concentrations of 0.3 μM for H3 (1–15) and 0.05 μM for 9-Ala-FAM were used. For both the H3 (1–15) and 9-Ala-FAM peptides, 80 mM sodium phosphate buffer (pH 6.5), 20 mM KCl, and 0.008% Triton X-100 were used. For the H3 (1–15) peptide, FP assays were measured in 10 μL aliquots in 384-well Axygen plates using a Synergy 4 microplate reader (BioTek) with an excitation wavelength of 485 nm and emission wavelength of 528 nm. For the 9-Ala-FAM peptide, FP assays were measured in 125 μL aliquots in 96-well Microfluor plates using a ViewLux instrument (PerkinElmer) with an excitation wavelength of 480 nm and emission wavelength of 540 nm. To determine <i>K</i><sub>disp</sub> values, the data were fit to a hyperbolic function using Sigma Plot software (Systat Software). The <i>K</i><sub>disp</sub> values represent the average of quadruplicate measurements.</div></div><div id="sec4_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i90" class="anchor-spacer"></div><h3 class="article-section__title" id="_i90"> Crystallization and Structure Determination</h3><div class="NLM_p last">Purified WDR5 protein (10 mg/mL) was mixed with each compound at a 1:5 molar ratio of protein to compound and crystallized using the sitting-drop vapor-diffusion method at 20 °C by adding 1 μL of protein solution to 1 μL of the reservoir solution containing 25% PEG [poly(ethylene glycol)] 3350, 0.1 M ammonium sulfate 0.1 M BisTris, pH 6.5 (compound <b>9d</b>), 28% PEG 2000 monomethyl ether, 0.1 M BisTris, pH 6.5 (compound <b>9e</b>), 20% PEG 5000 monomethyl ether, 0.1 M BisTris, pH 6.5 (compound <b>9h</b>), and 25% PEG3350, 0.1 M ammonium sulfate, 0.1 M BisTris, pH 6.5 (compound <b>9o</b>), respectively. Crystals were soaked in the corresponding mother liquor supplemented with 20% ethylene glycol as cryoprotectant before freezing in liquid nitrogen. Diffraction data for crystals of the WDR5-compound complexes were collected on a Rigaku FR-E SuperBright instrument or at beamline of the Canadian Macromolecular Crystallography Facility. The program suite HKL3000 was used to integrate and scale the data sets. All WDR5 co-crystal structures were determined by molecular replacement using PHASER, with coordinates from PDB entry <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4IA9">4IA9</a> as the search template. The graphic program Coot was used for manual model refinement and visualization. Refmac was used to refine the model. MolProbity was used to validate the refined structure. The structures have been deposited into the PDB under codes <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3SMR">3SMR</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4IA9">4IA9</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5EAR">5EAR</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5EAP">5EAP</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5EAL">5EAL</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5EAM">5EAM</a>,and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4QL1">4QL1</a>. Crystal diffraction data and refinement statistics for the structures of WDR5-<b>9d</b>, <b>9e</b>, <b>9h,</b> and <b>9o</b> complexes are shown in <a href="/doi/suppl/10.1021/acs.jmedchem.5b01630/suppl_file/jm5b01630_si_001.pdf" class="ext-link">Supporting Information</a>, Table 1.</div></div><div id="sec4_12" class="NLM_sec NLM_sec_level_2"><div id="ac_i91" class="anchor-spacer"></div><h3 class="article-section__title" id="_i91"> Pharmacokinetics</h3><div class="NLM_p last">All PK assessments were performed by InterVivo Solutions (Mississauga, Ontario, Canada). Compound <b>16d</b> was administered to mice (female NOD-SCID) at 3 mg/kg IV (<i>n</i> = 3) and 30 mg/kg IP (<i>n</i> = 3), along with an undosed control (<i>n</i> = 3). Blood samples were collected from all mice at 5, 15, 30 min and 1, 2, 4, 6, and 24 h. Plasma was separated from blood by centrifugation and stored at −80 °C until analysis. The pharmacokinetic parameters <i>C</i><sub>max</sub>, <i>T</i><sub>max</sub>, <i>t</i><sub>1/2</sub>, AUC, Cl, and <i>V</i><sub>D</sub> were evaluated.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i92"><a href="/doi/suppl/10.1021/acs.jmedchem.5b01630">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_35483" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_35483" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a href="/" class="ext-link">ACS Publications website</a> at DOI: <a href="/doi/abs/10.1021/acs.jmedchem.5b01630" class="ext-link">10.1021/acs.jmedchem.5b01630</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Synthetic procedures and compound characterization data, NMR spectra of select compounds, details of the focused library design and molecular docking, methods for cloning, expression, and purification of human WDR5, protocols for the fluorescence polarization (FP) binding assays, crystallization and structure determination methods, and how pharmacokinetic parameters were obtained (<a href="/doi/suppl/10.1021/acs.jmedchem.5b01630/suppl_file/jm5b01630_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a href="/doi/suppl/10.1021/acs.jmedchem.5b01630/suppl_file/jm5b01630_si_002.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.5b01630/suppl_file/jm5b01630_si_001.pdf">jm5b01630_si_001.pdf (1.29 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.5b01630/suppl_file/jm5b01630_si_002.csv">jm5b01630_si_002.csv (4.68 kb)</a></li></ul></div></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">PDB codes for the structures of WDR5 bound to various antagonists are as follows: WDR5-<b>4</b>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3SMR">3SMR</a>; WDR5-<b>5</b>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4IA9">4IA9</a>; WDR5-<b>9d</b>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5EAR">5EAR</a>; WDR5-<b>9e</b>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5EAP">5EAP</a>; WDR5-<b>9h</b>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5EAL">5EAL</a>; WDR5-<b>9o</b>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5EAM">5EAM</a>; WDR5-<b>16d</b>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4QL1">4QL1</a>.</p><div class="testing" data-doi="10.1021/acs.jmedchem.5b01630" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_60022" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_60022" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Rima Al-awar</span> - <span class="hlFld-Affiliation affiliation">Drug
Discovery Program, Ontario Institute for
Cancer Research, 661
University Avenue, MaRS Centre, West Tower, Toronto, Ontario M5G 0A3, Canada</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Pharmacology and Toxicology, University
of Toronto, Medical Sciences Building, 1 King’s College Circle, Toronto, Ontario M5S 1A8, Canada</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#287a414549066944495f495a6847414b5a064746064b49"><span class="__cf_email__" data-cfemail="d486bdb9b5fa95b8b5a3b5a694bbbdb7a6fabbbafab7b5">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Matthäus Getlik</span> - <span class="hlFld-Affiliation affiliation">Drug
Discovery Program, Ontario Institute for
Cancer Research, 661
University Avenue, MaRS Centre, West Tower, Toronto, Ontario M5G 0A3, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">David Smil</span> - <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium, University of Toronto, 101 College Street, MaRS Centre, South Tower, Toronto, Ontario M5G 1L7, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Carlos Zepeda-Velázquez</span> - <span class="hlFld-Affiliation affiliation">Drug
Discovery Program, Ontario Institute for
Cancer Research, 661
University Avenue, MaRS Centre, West Tower, Toronto, Ontario M5G 0A3, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yuri Bolshan</span> - <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium, University of Toronto, 101 College Street, MaRS Centre, South Tower, Toronto, Ontario M5G 1L7, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Gennady Poda</span> - <span class="hlFld-Affiliation affiliation">Drug
Discovery Program, Ontario Institute for
Cancer Research, 661
University Avenue, MaRS Centre, West Tower, Toronto, Ontario M5G 0A3, Canada</span>; 
    <span class="hlFld-Affiliation affiliation">Leslie
Dan Faculty of Pharmacy, University of Toronto, 144 College Street, Toronto, Ontario M5S 3M2, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Hong Wu</span> - <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium, University of Toronto, 101 College Street, MaRS Centre, South Tower, Toronto, Ontario M5G 1L7, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Aiping Dong</span> - <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium, University of Toronto, 101 College Street, MaRS Centre, South Tower, Toronto, Ontario M5G 1L7, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ekaterina Kuznetsova</span> - <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium, University of Toronto, 101 College Street, MaRS Centre, South Tower, Toronto, Ontario M5G 1L7, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Richard Marcellus</span> - <span class="hlFld-Affiliation affiliation">Drug
Discovery Program, Ontario Institute for
Cancer Research, 661
University Avenue, MaRS Centre, West Tower, Toronto, Ontario M5G 0A3, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Guillermo Senisterra</span> - <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium, University of Toronto, 101 College Street, MaRS Centre, South Tower, Toronto, Ontario M5G 1L7, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ludmila Dombrovski</span> - <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium, University of Toronto, 101 College Street, MaRS Centre, South Tower, Toronto, Ontario M5G 1L7, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Taraneh Hajian</span> - <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium, University of Toronto, 101 College Street, MaRS Centre, South Tower, Toronto, Ontario M5G 1L7, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Taira Kiyota</span> - <span class="hlFld-Affiliation affiliation">Drug
Discovery Program, Ontario Institute for
Cancer Research, 661
University Avenue, MaRS Centre, West Tower, Toronto, Ontario M5G 0A3, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Matthieu Schapira</span> - <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium, University of Toronto, 101 College Street, MaRS Centre, South Tower, Toronto, Ontario M5G 1L7, Canada</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Pharmacology and Toxicology, University
of Toronto, Medical Sciences Building, 1 King’s College Circle, Toronto, Ontario M5S 1A8, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Cheryl H. Arrowsmith</span> - <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium, University of Toronto, 101 College Street, MaRS Centre, South Tower, Toronto, Ontario M5G 1L7, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Peter J. Brown</span> - <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium, University of Toronto, 101 College Street, MaRS Centre, South Tower, Toronto, Ontario M5G 1L7, Canada</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Masoud Vedadi</span> - <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium, University of Toronto, 101 College Street, MaRS Centre, South Tower, Toronto, Ontario M5G 1L7, Canada</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Pharmacology and Toxicology, University
of Toronto, Medical Sciences Building, 1 King’s College Circle, Toronto, Ontario M5S 1A8, Canada</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>M.G. and D.S. contributed equally to this work.  M.G., D.S., and C. Z.-V. designed and synthesized compounds. Y.B. synthesized compounds. G.P. designed compounds, enumerated and evaluated focused virtual libraries and performed docking experiments against WDR5 and protein kinases. E.K. and G.S. performed all FP assays. R.M. performed all SPR experiments. H.W. and L.D. purified and co-crystallized WDR5. A.D. solved the crystal structures. T.H. purified WDR5 for in vitro assays. T.K. performed Caco-2 studies. M.S. contributed to the design of compounds and analyzed data. C.H.A. and P.J.B. contributed to experimental design, data analysis, and editing of the paper. M.G., D.S., C. Z.-V., G.P., M.V., and R.A. reviewed data, contributed to experimental design, and wrote the paper. R.A. and M.V. were the principal investigators, designed experiments, analyzed data, and supervised the experiments.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ACK-d7e7660-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i98">Acknowledgment</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_22867" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_22867" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">This work was supported by the Ontario Institute for Cancer Research whose funding is provided by the Government of Ontario through the Ontario Ministry of Economic Development and Innovation. Additional support came from the Structural Genomics Consortium, a registered charity (no. 1097737) that receives funds from AbbVie, Bayer Pharma AG, Boehringer Ingelheim, Canada Foundation for Innovation, Eshelman Institute for Innovation, Genome Canada, Innovative Medicines Initiative (EU/EFPIA), Janssen, Merck & Co., Novartis Pharma AG, Pfizer, São Paulo Research Foundation-FAPESP, Takeda, and Wellcome Trust. Research described in this paper was performed using beamline 08ID-1 at the Canadian Light Source, which is supported by the Natural Sciences and Engineering Research Council of Canada, the National Research Council Canada, the Canadian Institutes of Health Research, the Province of Saskatchewan, Western Economic Diversification Canada, and the University of Saskatchewan. C.Z.-V. thanks CONACyT for a postdoctoral fellowship. We would also like to thank Dr. Ahmed Aman and Dr. Taira Kiyota for assistance in obtaining the HRMS of all final synthesized compounds and Dr. John R. Walker for review of the crystallography model.</p></div><div id="" class="NLM_sec NLM_sec_level_1"><div id="ac_i99" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i99"> </h2><div class="NLM_p last"><table summary="" class="NLM_def-list" id="dl1"><div id="ac_i100" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i100"> Abbreviations Used</h2><tr><td class="NLM_term">AML</td><td class="NLM_def"><p class="first last">acute myeloid leukemia</p></td></tr><tr><td class="NLM_term">Ash2L</td><td class="NLM_def"><p class="first last">absent, small, or homeotic 2 like protein</p></td></tr><tr><td class="NLM_term">C/EBPα</td><td class="NLM_def"><p class="first last">CCAAT/enhancer binding protein alpha</p></td></tr><tr><td class="NLM_term">DFG</td><td class="NLM_def"><p class="first last">Asp-Phe-Gly</p></td></tr><tr><td class="NLM_term">DIPEA</td><td class="NLM_def"><p class="first last"><i>N</i>,<i>N</i>-diisopropylethylamine</p></td></tr><tr><td class="NLM_term">DMAP</td><td class="NLM_def"><p class="first last">4-(<i>N</i>,<i>N</i>-dimethylamino)pyridine</p></td></tr><tr><td class="NLM_term">DMSO</td><td class="NLM_def"><p class="first last">dimethyl sulfoxide</p></td></tr><tr><td class="NLM_term">EtOAc</td><td class="NLM_def"><p class="first last">ethyl acetate</p></td></tr><tr><td class="NLM_term">FP</td><td class="NLM_def"><p class="first last">fluorescence polarization</p></td></tr><tr><td class="NLM_term">GOF</td><td class="NLM_def"><p class="first last">gain-of-function</p></td></tr><tr><td class="NLM_term">GPCR</td><td class="NLM_def"><p class="first last">G-protein coupled receptor</p></td></tr><tr><td class="NLM_term">HOXA9</td><td class="NLM_def"><p class="first last">homeobox A9</p></td></tr><tr><td class="NLM_term">HRMS</td><td class="NLM_def"><p class="first last">high-resolution mass spectrometry</p></td></tr><tr><td class="NLM_term">HTS</td><td class="NLM_def"><p class="first last">high-throughput screening</p></td></tr><tr><td class="NLM_term">IV</td><td class="NLM_def"><p class="first last">intravenous</p></td></tr><tr><td class="NLM_term">MEIS1</td><td class="NLM_def"><p class="first last">Meis homeobox 1</p></td></tr><tr><td class="NLM_term">MeOH</td><td class="NLM_def"><p class="first last">methanol</p></td></tr><tr><td class="NLM_term">MLL1</td><td class="NLM_def"><p class="first last">mixed-lineage leukemia</p></td></tr><tr><td class="NLM_term">N-Myc</td><td class="NLM_def"><p class="first last">neuroblastoma-derived myelocytomatosis</p></td></tr><tr><td class="NLM_term">PK</td><td class="NLM_def"><p class="first last">pharmacokinetic</p></td></tr><tr><td class="NLM_term">PSA</td><td class="NLM_def"><p class="first last">polar surface area</p></td></tr><tr><td class="NLM_term">RbBP5</td><td class="NLM_def"><p class="first last">retinoblastoma-binding protein 5</p></td></tr><tr><td class="NLM_term">SAR</td><td class="NLM_def"><p class="first last">structure–activity relationship</p></td></tr><tr><td class="NLM_term">SPR</td><td class="NLM_def"><p class="first last">surface plasmon resonance</p></td></tr><tr><td class="NLM_term">WDR5</td><td class="NLM_def"><p class="first last">WD repeat-containing protein 5</p></td></tr><tr><td class="NLM_term">WIN</td><td class="NLM_def"><p class="first last">WDR5-interacting</p></td></tr></table></div></div><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i101">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_64257" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_64257" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 24 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group">Wu, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shu, H.-B.</span><span> </span><span class="NLM_article-title">MLL1/WDR5 complex in leukemogenesis and epigenetic regulation</span> <span class="citation_source-journal">Aizheng</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">30</span><span class="NLM_x">, </span> <span class="NLM_fpage">240</span><span class="NLM_x">–</span> <span class="NLM_lpage">246</span><span class="refDoi"> DOI: 10.5732/cjc.011.10055</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01630&amp;key=10.5732%2Fcjc.011.10055" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01630&amp;key=21439245" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2011&pages=240-246&author=M.+Wuauthor=H.-B.+Shu&title=MLL1%2FWDR5+complex+in+leukemogenesis+and+epigenetic+regulation&doi=10.5732%2Fcjc.011.10055"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.5732%2Fcjc.011.10055&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.5732%252Fcjc.011.10055%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DM.%26aulast%3DShu%26aufirst%3DH.-B.%26atitle%3DMLL1%252FWDR5%2520complex%2520in%2520leukemogenesis%2520and%2520epigenetic%2520regulation%26jtitle%3DAizheng%26date%3D2011%26volume%3D30%26spage%3D240%26epage%3D246%26doi%3D10.5732%2Fcjc.011.10055" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group">Slany, R. K.</span><span> </span><span class="NLM_article-title">The molecular biology of mixed-lineage leukemia</span> <span class="citation_source-journal">Haematologica</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">94</span><span class="NLM_x">, </span> <span class="NLM_fpage">984</span><span class="NLM_x">–</span> <span class="NLM_lpage">993</span><span class="refDoi"> DOI: 10.3324/haematol.2008.002436</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01630&amp;key=10.3324%2Fhaematol.2008.002436" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=94&publication_year=2009&pages=984-993&author=R.+K.+Slany&title=The+molecular+biology+of+mixed-lineage+leukemia&doi=10.3324%2Fhaematol.2008.002436"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.3324%2Fhaematol.2008.002436&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3324%252Fhaematol.2008.002436%26sid%3Dliteratum%253Aachs%26aulast%3DSlany%26aufirst%3DR.%2BK.%26atitle%3DThe%2520molecular%2520biology%2520of%2520mixed-lineage%2520leukemia%26jtitle%3DHaematologica%26date%3D2009%26volume%3D94%26spage%3D984%26epage%3D993%26doi%3D10.3324%2Fhaematol.2008.002436" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group">Karatas, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Townsend, E. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cao, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bernard, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lei, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dou, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, S.</span><span> </span><span class="NLM_article-title">High-affinity, small-molecule peptidomimetic inhibitors of MLL1/WDR5 protein-protein interaction</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">135</span><span class="NLM_x">, </span> <span class="NLM_fpage">669</span><span class="NLM_x">–</span> <span class="NLM_lpage">682</span><span class="refDoi"> DOI: 10.1021/ja306028q</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja306028q" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=135&publication_year=2013&pages=669-682&author=H.+Karatasauthor=E.+C.+Townsendauthor=F.+Caoauthor=Y.+Chenauthor=D.+Bernardauthor=L.+Liuauthor=M.+Leiauthor=Y.+Douauthor=S.+Wang&title=High-affinity%2C+small-molecule+peptidomimetic+inhibitors+of+MLL1%2FWDR5+protein-protein+interaction&doi=10.1021%2Fja306028q"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1021%2Fja306028q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja306028q%26sid%3Dliteratum%253Aachs%26aulast%3DKaratas%26aufirst%3DH.%26aulast%3DTownsend%26aufirst%3DE.%2BC.%26aulast%3DCao%26aufirst%3DF.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DBernard%26aufirst%3DD.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DLei%26aufirst%3DM.%26aulast%3DDou%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DHigh-affinity%252C%2520small-molecule%2520peptidomimetic%2520inhibitors%2520of%2520MLL1%252FWDR5%2520protein-protein%2520interaction%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2013%26volume%3D135%26spage%3D669%26epage%3D682%26doi%3D10.1021%2Fja306028q" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group">Bissinger, E.-M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heinke, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sippl, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jung, M.</span><span> </span><span class="NLM_article-title">Targeting epigenetic modifiers: inhibitors of histone methyltransferases</span> <span class="citation_source-journal">MedChemComm</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">1</span><span class="NLM_x">, </span> <span class="NLM_fpage">114</span><span class="NLM_x">–</span> <span class="NLM_lpage">124</span><span class="refDoi"> DOI: 10.1039/c0md00062k</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01630&amp;key=10.1039%2Fc0md00062k" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2010&pages=114-124&author=E.-M.+Bissingerauthor=R.+Heinkeauthor=W.+Sipplauthor=M.+Jung&title=Targeting+epigenetic+modifiers%3A+inhibitors+of+histone+methyltransferases&doi=10.1039%2Fc0md00062k"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1039%2Fc0md00062k&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc0md00062k%26sid%3Dliteratum%253Aachs%26aulast%3DBissinger%26aufirst%3DE.-M.%26aulast%3DHeinke%26aufirst%3DR.%26aulast%3DSippl%26aufirst%3DW.%26aulast%3DJung%26aufirst%3DM.%26atitle%3DTargeting%2520epigenetic%2520modifiers%253A%2520inhibitors%2520of%2520histone%2520methyltransferases%26jtitle%3DMedChemComm%26date%3D2010%26volume%3D1%26spage%3D114%26epage%3D124%26doi%3D10.1039%2Fc0md00062k" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group">Li, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roberts, R.</span><span> </span><span class="NLM_article-title">Human Genome and Diseases. WD-repeat proteins: structure characteristics, biological function, and their involvement in human diseases</span> <span class="citation_source-journal">Cell. Mol. Life Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">58</span><span class="NLM_x">, </span> <span class="NLM_fpage">2085</span><span class="NLM_x">–</span> <span class="NLM_lpage">2097</span><span class="refDoi"> DOI: 10.1007/PL00000838</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01630&amp;key=10.1007%2FPL00000838" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01630&amp;key=11814058" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01630&amp;key=1%3ACAS%3A528%3ADC%252BD38XhtFakug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2001&pages=2085-2097&author=D.+Liauthor=R.+Roberts&title=Human+Genome+and+Diseases.+WD-repeat+proteins%3A+structure+characteristics%2C+biological+function%2C+and+their+involvement+in+human+diseases&doi=10.1007%2FPL00000838"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">WD-repeat proteins: structure characteristics, biological function, and their involvement in human diseases</span></div><div class="casAuthors">Li, D.; Roberts, R.</div><div class="citationInfo"><span class="NLM_cas:title">Cellular and Molecular Life Sciences</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">2085-2097</span>CODEN:
                <span class="NLM_cas:coden">CMLSFI</span>;
        ISSN:<span class="NLM_cas:issn">1420-682X</span>.
    
            (<span class="NLM_cas:orgname">Birkhaeuser Verlag</span>)
        </div><div class="casAbstract">A review.  Defined by the presence of four or more repeating units contg. a conserved core of approx. 40 amino acids that usually ending with tryptophan-aspartic acid (WD), WD-repeat proteins belong to a large and fast-expanding conservative protein family.  As demonstrated by the crystal structure of the G protein β subunit, all WD-repeat proteins are speculated to form a circularized β propeller structure.  The importance of these proteins is not only demonstrated by their crit. roles in many essential biol. functions ranging from signal transduction, transcription regulation, to apoptosis, but is also recognized by their assocn. with several human diseases.  Defining the function of a WD-repeat protein is the current challenge.  It is however, paramount to uncover the function of individual WD-repeat proteins, explore the protein interaction mechanism through WD-repeat domains and, ultimately, understand the complex biol. processes and organisms themselves.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGphQmbI-2-bprVg90H21EOLACvtfcHk0lhmkQwh17Wj9g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XhtFakug%253D%253D&md5=651e8fb04b017537c5f26f9b19a0b45c</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1007%2FPL00000838&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252FPL00000838%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DD.%26aulast%3DRoberts%26aufirst%3DR.%26atitle%3DHuman%2520Genome%2520and%2520Diseases.%2520WD-repeat%2520proteins%253A%2520structure%2520characteristics%252C%2520biological%2520function%252C%2520and%2520their%2520involvement%2520in%2520human%2520diseases%26jtitle%3DCell.%2520Mol.%2520Life%2520Sci.%26date%3D2001%26volume%3D58%26spage%3D2085%26epage%3D2097%26doi%3D10.1007%2FPL00000838" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group">Patel, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dharmarajan, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cosgrove, M. S.</span><span> </span><span class="NLM_article-title">Structure of WDR5 bound to mixed lineage leukemia protein-1 peptide</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">283</span><span class="NLM_x">, </span> <span class="NLM_fpage">32158</span><span class="NLM_x">–</span> <span class="NLM_lpage">32161</span><span class="refDoi"> DOI: 10.1074/jbc.C800164200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01630&amp;key=10.1074%2Fjbc.C800164200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01630&amp;key=18829459" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01630&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtlOjtL7M" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=283&publication_year=2008&pages=32158-32161&author=A.+Patelauthor=V.+Dharmarajanauthor=M.+S.+Cosgrove&title=Structure+of+WDR5+bound+to+mixed+lineage+leukemia+protein-1+peptide&doi=10.1074%2Fjbc.C800164200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Structure of WDR5 Bound to Mixed Lineage Leukemia Protein-1 Peptide</span></div><div class="casAuthors">Patel, Anamika; Dharmarajan, Venkatasubramanian; Cosgrove, Michael S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">283</span>
        (<span class="NLM_cas:issue">47</span>),
    <span class="NLM_cas:pages">32158-32161</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">The mixed lineage leukemia protein-1 (MLL1) catalyzes histone H3 lysine 4 methylation and is regulated by interaction with WDR5 (WD-repeat protein-5), RbBP5 (retinoblastoma-binding protein-5), and the Ash2L (absent, small, homeotic disks-2-like) oncoprotein.  In the accompanying investigation, we describe the identification of a conserved arginine contg. motif, called the "Win" or WDR5 interaction motif, that is essential for the assembly and H3K4 dimethylation activity of the MLL1 core complex.  Here we present a 1.7-Å crystal structure of WDR5 bound to a peptide derived from the MLL1 Win motif.  Our results show that Arg-3765 of MLL1 is bound in the same arginine binding pocket on WDR5 that was previously suggested to bind histone H3.  Thermodn. binding expts. show that the MLL1 Win peptide is preferentially recognized by WDR5.  These results are consistent with a model in which WDR5 recognizes Arg-3765 of MLL1, which is essential for the assembly and enzymic activity of the MLL1 core complex.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq2hQy6dZIN-LVg90H21EOLACvtfcHk0lhmkQwh17Wj9g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtlOjtL7M&md5=770e511a13cf8ada19b37e0cb7115eaf</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1074%2Fjbc.C800164200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.C800164200%26sid%3Dliteratum%253Aachs%26aulast%3DPatel%26aufirst%3DA.%26aulast%3DDharmarajan%26aufirst%3DV.%26aulast%3DCosgrove%26aufirst%3DM.%2BS.%26atitle%3DStructure%2520of%2520WDR5%2520bound%2520to%2520mixed%2520lineage%2520leukemia%2520protein-1%2520peptide%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2008%26volume%3D283%26spage%3D32158%26epage%3D32161%26doi%3D10.1074%2Fjbc.C800164200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group">Patel, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vought, V. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dharmarajan, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cosgrove, M. S.</span><span> </span><span class="NLM_article-title">A conserved arginine-containing motif crucial for the assembly and enzymatic activity of the mixed lineage leukemia protein-1 core complex</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">283</span><span class="NLM_x">, </span> <span class="NLM_fpage">32162</span><span class="NLM_x">–</span> <span class="NLM_lpage">32175</span><span class="refDoi"> DOI: 10.1074/jbc.M806317200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01630&amp;key=10.1074%2Fjbc.M806317200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01630&amp;key=18829457" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01630&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtlOjtL7N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=283&publication_year=2008&pages=32162-32175&author=A.+Patelauthor=V.+E.+Voughtauthor=V.+Dharmarajanauthor=M.+S.+Cosgrove&title=A+conserved+arginine-containing+motif+crucial+for+the+assembly+and+enzymatic+activity+of+the+mixed+lineage+leukemia+protein-1+core+complex&doi=10.1074%2Fjbc.M806317200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">A Conserved Arginine-containing Motif Crucial for the Assembly and Enzymatic Activity of the Mixed Lineage Leukemia Protein-1 Core Complex</span></div><div class="casAuthors">Patel, Anamika; Vought, Valarie E.; Dharmarajan, Venkatasubramanian; Cosgrove, Michael S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">283</span>
        (<span class="NLM_cas:issue">47</span>),
    <span class="NLM_cas:pages">32162-32175</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">The mixed lineage leukemia protein-1 (MLL1) belongs to the SET1 family of histone H3 lysine 4 methyltransferases.  Recent studies indicate that the catalytic subunits of SET1 family members are regulated by interaction with a conserved core group of proteins that include the WD repeat protein-5 (WDR5), retinoblastoma-binding protein-5 (RbBP5), and the absent small homeotic-2-like protein (Ash2L).  It has been suggested that WDR5 functions to bridge the interactions between the catalytic and regulatory subunits of SET1 family complexes.  However, the mol. details of these interactions are unknown.  To gain insight into the interactions among these proteins, we have detd. the biophys. basis for the interaction between the human WDR5 and MLL1.  Our studies reveal that WDR5 preferentially recognizes a previously unidentified and conserved arginine-contg. motif, called the "Win" or WDR5 interaction motif, which is located in the N-SET region of MLL1 and other SET1 family members.  Surprisingly, our structural and functional studies show that WDR5 recognizes arginine 3765 of the MLL1 Win motif using the same arginine-binding pocket on WDR5 that was previously shown to bind histone H3.  We demonstrate that WDR5's recognition of arginine 3765 of MLL1 is essential for the assembly and enzymic activity of the MLL1 core complex in vitro.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqSRgLh_lnhMbVg90H21EOLACvtfcHk0lgHUoDndQb4aA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtlOjtL7N&md5=0d2e92e9ab6d549b4d7e175aa7957335</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M806317200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M806317200%26sid%3Dliteratum%253Aachs%26aulast%3DPatel%26aufirst%3DA.%26aulast%3DVought%26aufirst%3DV.%2BE.%26aulast%3DDharmarajan%26aufirst%3DV.%26aulast%3DCosgrove%26aufirst%3DM.%2BS.%26atitle%3DA%2520conserved%2520arginine-containing%2520motif%2520crucial%2520for%2520the%2520assembly%2520and%2520enzymatic%2520activity%2520of%2520the%2520mixed%2520lineage%2520leukemia%2520protein-1%2520core%2520complex%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2008%26volume%3D283%26spage%3D32162%26epage%3D32175%26doi%3D10.1074%2Fjbc.M806317200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group">Orlicky, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Neduva, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Elowe, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, E. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sicheri, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tyers, M.</span><span> </span><span class="NLM_article-title">An allosteric inhibitor of substrate recognition by the SCF<sup>Cdc4</sup> ubiquitin ligase</span> <span class="citation_source-journal">Nat. Biotechnol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">28</span><span class="NLM_x">, </span> <span class="NLM_fpage">733</span><span class="NLM_x">–</span> <span class="NLM_lpage">737</span><span class="refDoi"> DOI: 10.1038/nbt.1646</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01630&amp;key=10.1038%2Fnbt.1646" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01630&amp;key=20581844" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01630&amp;key=1%3ACAS%3A528%3ADC%252BC3cXnvFyksbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2010&pages=733-737&author=S.+Orlickyauthor=X.+Tangauthor=V.+Neduvaauthor=N.+Eloweauthor=E.+D.+Brownauthor=F.+Sicheriauthor=M.+Tyers&title=An+allosteric+inhibitor+of+substrate+recognition+by+the+SCFCdc4+ubiquitin+ligase&doi=10.1038%2Fnbt.1646"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">An allosteric inhibitor of substrate recognition by the SCFCdc4 ubiquitin ligase</span></div><div class="casAuthors">Orlicky, Stephen; Tang, Xiaojing; Neduva, Victor; Elowe, Nadine; Brown, Eric D.; Sicheri, Frank; Tyers, Mike</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">733-737</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The specificity of SCF ubiquitin ligase-mediated protein degrdn. is detd. by F-box proteins.  We identified a biplanar dicarboxylic acid compd., called SCF-I2, as an inhibitor of substrate recognition by the yeast F-box protein Cdc4 using a fluorescence polarization screen to monitor the displacement of a fluorescein-labeled phosphodegron peptide.  SCF-I2 inhibits the binding and ubiquitination of full-length phosphorylated substrates by SCFCdc4.  A co-crystal structure reveals that SCF-I2 inserts itself between the β-strands of blades 5 and 6 of the WD40 propeller domain of Cdc4 at a site that is 25 Å away from the substrate binding site.  Long-range transmission of SCF-I2 interactions distorts the substrate binding pocket and impedes recognition of key determinants in the Cdc4 phosphodegron.  Mutation of the SCF-I2 binding site abrogates its inhibitory effect and explains specificity in the allosteric inhibition mechanism.  Mammalian WD40 domain proteins may exhibit similar allosteric responsiveness and hence represent an extensive class of druggable target.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGptnWKaMi1kdLVg90H21EOLACvtfcHk0lgHUoDndQb4aA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXnvFyksbw%253D&md5=23197bd195811ad8503efb5d43457ccf</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1038%2Fnbt.1646&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt.1646%26sid%3Dliteratum%253Aachs%26aulast%3DOrlicky%26aufirst%3DS.%26aulast%3DTang%26aufirst%3DX.%26aulast%3DNeduva%26aufirst%3DV.%26aulast%3DElowe%26aufirst%3DN.%26aulast%3DBrown%26aufirst%3DE.%2BD.%26aulast%3DSicheri%26aufirst%3DF.%26aulast%3DTyers%26aufirst%3DM.%26atitle%3DAn%2520allosteric%2520inhibitor%2520of%2520substrate%2520recognition%2520by%2520the%2520SCFCdc4%2520ubiquitin%2520ligase%26jtitle%3DNat.%2520Biotechnol.%26date%3D2010%26volume%3D28%26spage%3D733%26epage%3D737%26doi%3D10.1038%2Fnbt.1646" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group">Cao, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Townsend, E. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Karatas, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ouillette, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hess, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Atadja, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lei, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Qin, Z. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Malek, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dou, Y.</span><span> </span><span class="NLM_article-title">Targeting MLL1 H3K4 methyltransferase activity in mixed-lineage leukemia</span> <span class="citation_source-journal">Mol. Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">247</span><span class="NLM_x">–</span> <span class="NLM_lpage">261</span><span class="refDoi"> DOI: 10.1016/j.molcel.2013.12.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01630&amp;key=10.1016%2Fj.molcel.2013.12.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01630&amp;key=24389101" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01630&amp;key=1%3ACAS%3A528%3ADC%252BC2cXisVKruw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2014&pages=247-261&author=F.+Caoauthor=E.+C.+Townsendauthor=H.+Karatasauthor=J.+Xuauthor=L.+Liauthor=S.+Leeauthor=L.+Liuauthor=Y.+Chenauthor=P.+Ouilletteauthor=J.+Zhuauthor=J.+L.+Hessauthor=P.+Atadjaauthor=M.+Leiauthor=Z.+S.+Qinauthor=S.+Malekauthor=S.+Wangauthor=Y.+Dou&title=Targeting+MLL1+H3K4+methyltransferase+activity+in+mixed-lineage+leukemia&doi=10.1016%2Fj.molcel.2013.12.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting MLL1 H3K4 Methyltransferase Activity in Mixed-Lineage Leukemia</span></div><div class="casAuthors">Cao, Fang; Townsend, Elizabeth C.; Karatas, Hacer; Xu, Jing; Li, Li; Lee, Shirley; Liu, Liu; Chen, Yong; Ouillette, Peter; Zhu, Jidong; Hess, Jay L.; Atadja, Peter; Lei, Ming; Qin, Zhaohui S.; Malek, Sami; Wang, Shaomeng; Dou, Yali</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">247-261</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Here we report a comprehensive characterization of our recently developed inhibitor MM-401 that targets the MLL1 H3K4 methyltransferase activity.  MM-401 is able to specifically inhibit MLL1 activity by blocking MLL1-WDR5 interaction and thus the complex assembly.  This targeting strategy does not affect other mixed-lineage leukemia (MLL) family histone methyltransferases (HMTs), revealing a unique regulatory feature for the MLL1 complex.  Using MM-401 and its enantiomer control MM-NC-401, we show that inhibiting MLL1 methyltransferase activity specifically blocks proliferation of MLL cells by inducing cell-cycle arrest, apoptosis, and myeloid differentiation without general toxicity to normal bone marrow cells or non-MLL cells.  More importantly, transcriptome analyses show that MM-401 induces changes in gene expression similar to those of MLL1 deletion, supporting a predominant role of MLL1 activity in regulating MLL1-dependent leukemia transcription program.  We envision broad applications for MM-401 in basic and translational research.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGov9sdJIz15pbVg90H21EOLACvtfcHk0lgHUoDndQb4aA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXisVKruw%253D%253D&md5=f3e482ba3474b39e1bbdde0d10407d1f</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2013.12.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2013.12.001%26sid%3Dliteratum%253Aachs%26aulast%3DCao%26aufirst%3DF.%26aulast%3DTownsend%26aufirst%3DE.%2BC.%26aulast%3DKaratas%26aufirst%3DH.%26aulast%3DXu%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DLee%26aufirst%3DS.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DOuillette%26aufirst%3DP.%26aulast%3DZhu%26aufirst%3DJ.%26aulast%3DHess%26aufirst%3DJ.%2BL.%26aulast%3DAtadja%26aufirst%3DP.%26aulast%3DLei%26aufirst%3DM.%26aulast%3DQin%26aufirst%3DZ.%2BS.%26aulast%3DMalek%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DS.%26aulast%3DDou%26aufirst%3DY.%26atitle%3DTargeting%2520MLL1%2520H3K4%2520methyltransferase%2520activity%2520in%2520mixed-lineage%2520leukemia%26jtitle%3DMol.%2520Cell%26date%3D2014%26volume%3D53%26spage%3D247%26epage%3D261%26doi%3D10.1016%2Fj.molcel.2013.12.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group">Senisterra, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allali-Hassani, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wasney, G. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barsyte-Lovejoy, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dombrovski, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dong, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nguyen, K. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smil, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bolshan, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hajian, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">He, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Seitova, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chau, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Poda, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Couture, J. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Al-Awar, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schapira, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vedadi, M.</span><span> </span><span class="NLM_article-title">Small-molecule inhibition of MLL1 activity by disruption of its interaction with WDR5</span> <span class="citation_source-journal">Biochem. J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">449</span><span class="NLM_x">, </span> <span class="NLM_fpage">151</span><span class="NLM_x">–</span> <span class="NLM_lpage">159</span><span class="refDoi"> DOI: 10.1042/BJ20121280</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01630&amp;key=10.1042%2FBJ20121280" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01630&amp;key=22989411" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01630&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvVSmtrzL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=449&publication_year=2013&pages=151-159&author=G.+Senisterraauthor=H.+Wuauthor=A.+Allali-Hassaniauthor=G.+A.+Wasneyauthor=D.+Barsyte-Lovejoyauthor=L.+Dombrovskiauthor=A.+Dongauthor=K.+T.+Nguyenauthor=D.+Smilauthor=Y.+Bolshanauthor=T.+Hajianauthor=H.+Heauthor=A.+Seitovaauthor=I.+Chauauthor=F.+Liauthor=G.+Podaauthor=J.+F.+Coutureauthor=P.+J.+Brownauthor=R.+Al-Awarauthor=M.+Schapiraauthor=C.+H.+Arrowsmithauthor=M.+Vedadi&title=Small-molecule+inhibition+of+MLL1+activity+by+disruption+of+its+interaction+with+WDR5&doi=10.1042%2FBJ20121280"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Small-molecule inhibition of MLL activity by disruption of its interaction with WDR5</span></div><div class="casAuthors">Senisterra, Guillermo; Wu, Hong; Allali-Hassani, Abdellah; Wasney, Gregory A.; Barsyte-Lovejoy, Dalia; Dombrovski, Ludmila; Dong, Aiping; Nguyen, Kong T.; Smil, David; Bolshan, Yuri; Hajian, Taraneh; He, Hao; Seitova, Alma; Chau, Irene; Li, Fengling; Poda, Gennadiy; Couture, Jean-Francois; Brown, Peter J.; Al-Awar, Rima; Schapira, Matthieu; Arrowsmith, Cheryl H.; Vedadi, Masoud</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical Journal</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">449</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">151-159</span>CODEN:
                <span class="NLM_cas:coden">BIJOAK</span>;
        ISSN:<span class="NLM_cas:issn">0264-6021</span>.
    
            (<span class="NLM_cas:orgname">Portland Press Ltd.</span>)
        </div><div class="casAbstract">WDR5 (WD40 repeat protein 5) is an essential component of the human trithorax-like family of SET1 [Su(var)3-9 enhancer-of-zeste trithorax 1] methyltransferase complexes that carry out trimethylation of histone 3 Lys4 (H3K4me3), play key roles in development and are abnormally expressed in many cancers.  In the present study, we show that the interaction between WDR5 and peptides from the catalytic domain of MLL (mixed-lineage leukemia protein) (KMT2) can be antagonized with a small mol.  Structural and biophys. anal. show that this antagonist binds in the WDR5 peptide-binding pocket with a Kd of 450 nM and inhibits the catalytic activity of the MLL core complex in vitro.  The degree of inhibition was enhanced at lower protein concns. consistent with a role for WDR5 in directly stabilizing the MLL multiprotein complex.  Our data demonstrate inhibition of an important protein-protein interaction and form the basis for further development of inhibitors of WDR5-dependent enzymes implicated in MLL-rearranged leukemias or other cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrAilf3m7N6c7Vg90H21EOLACvtfcHk0lh58b71rtSvww"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvVSmtrzL&md5=ad00c4e1bfd48099aa8040199f9311be</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1042%2FBJ20121280&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252FBJ20121280%26sid%3Dliteratum%253Aachs%26aulast%3DSenisterra%26aufirst%3DG.%26aulast%3DWu%26aufirst%3DH.%26aulast%3DAllali-Hassani%26aufirst%3DA.%26aulast%3DWasney%26aufirst%3DG.%2BA.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DDombrovski%26aufirst%3DL.%26aulast%3DDong%26aufirst%3DA.%26aulast%3DNguyen%26aufirst%3DK.%2BT.%26aulast%3DSmil%26aufirst%3DD.%26aulast%3DBolshan%26aufirst%3DY.%26aulast%3DHajian%26aufirst%3DT.%26aulast%3DHe%26aufirst%3DH.%26aulast%3DSeitova%26aufirst%3DA.%26aulast%3DChau%26aufirst%3DI.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DPoda%26aufirst%3DG.%26aulast%3DCouture%26aufirst%3DJ.%2BF.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DAl-Awar%26aufirst%3DR.%26aulast%3DSchapira%26aufirst%3DM.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DVedadi%26aufirst%3DM.%26atitle%3DSmall-molecule%2520inhibition%2520of%2520MLL1%2520activity%2520by%2520disruption%2520of%2520its%2520interaction%2520with%2520WDR5%26jtitle%3DBiochem.%2520J.%26date%3D2013%26volume%3D449%26spage%3D151%26epage%3D159%26doi%3D10.1042%2FBJ20121280" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group">Bolshan, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Getlik, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuznetsova, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wasney, G. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hajian, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Poda, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nguyen, K. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dombrovski, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dong, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Senisterra, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schapira, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Al-awar, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vedadi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smil, D.</span><span> </span><span class="NLM_article-title">Synthesis, optimization, and evaluation of novel small molecules as antagonists of WDR5-MLL1 interaction</span> <span class="citation_source-journal">ACS Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x">, </span> <span class="NLM_fpage">353</span><span class="NLM_x">–</span> <span class="NLM_lpage">357</span><span class="refDoi"> DOI: 10.1021/ml300467n</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ml300467n" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2013&pages=353-357&author=Y.+Bolshanauthor=M.+Getlikauthor=E.+Kuznetsovaauthor=G.+A.+Wasneyauthor=T.+Hajianauthor=G.+Podaauthor=K.+T.+Nguyenauthor=H.+Wuauthor=L.+Dombrovskiauthor=A.+Dongauthor=G.+Senisterraauthor=M.+Schapiraauthor=C.+H.+Arrowsmithauthor=P.+J.+Brownauthor=R.+Al-awarauthor=M.+Vedadiauthor=D.+Smil&title=Synthesis%2C+optimization%2C+and+evaluation+of+novel+small+molecules+as+antagonists+of+WDR5-MLL1+interaction&doi=10.1021%2Fml300467n"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1021%2Fml300467n&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fml300467n%26sid%3Dliteratum%253Aachs%26aulast%3DBolshan%26aufirst%3DY.%26aulast%3DGetlik%26aufirst%3DM.%26aulast%3DKuznetsova%26aufirst%3DE.%26aulast%3DWasney%26aufirst%3DG.%2BA.%26aulast%3DHajian%26aufirst%3DT.%26aulast%3DPoda%26aufirst%3DG.%26aulast%3DNguyen%26aufirst%3DK.%2BT.%26aulast%3DWu%26aufirst%3DH.%26aulast%3DDombrovski%26aufirst%3DL.%26aulast%3DDong%26aufirst%3DA.%26aulast%3DSenisterra%26aufirst%3DG.%26aulast%3DSchapira%26aufirst%3DM.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DAl-awar%26aufirst%3DR.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DSmil%26aufirst%3DD.%26atitle%3DSynthesis%252C%2520optimization%252C%2520and%2520evaluation%2520of%2520novel%2520small%2520molecules%2520as%2520antagonists%2520of%2520WDR5-MLL1%2520interaction%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2013%26volume%3D4%26spage%3D353%26epage%3D357%26doi%3D10.1021%2Fml300467n" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group">Grebien, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vedadi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Getlik, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Giambruno, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grover, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Avellino, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Skucha, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vittori, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuznetsova, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smil, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barsyte-Lovejoy, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Poda, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schapira, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dong, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Senisterra, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stukalov, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huber, K. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schönegger, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marcellus, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bilban, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bock, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zuber, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bennett, K. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Al-awar, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Delwel, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nerlov, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Superti-Furga, G.</span><span> </span><span class="NLM_article-title">Pharmacological targeting of the WDR5-MLL1 interaction in C/EBPα <i>N</i>-terminal leukemia</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">571</span><span class="NLM_x">–</span> <span class="NLM_lpage">578</span><span class="refDoi"> DOI: 10.1038/nchembio.1859</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01630&amp;key=10.1038%2Fnchembio.1859" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01630&amp;key=26167872" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01630&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht1WltLfM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2015&pages=571-578&author=F.+Grebienauthor=M.+Vedadiauthor=M.+Getlikauthor=R.+Giambrunoauthor=A.+Groverauthor=R.+Avellinoauthor=A.+Skuchaauthor=S.+Vittoriauthor=E.+Kuznetsovaauthor=D.+Smilauthor=D.+Barsyte-Lovejoyauthor=F.+Liauthor=G.+Podaauthor=M.+Schapiraauthor=H.+Wuauthor=A.+Dongauthor=G.+Senisterraauthor=A.+Stukalovauthor=K.+V.+Huberauthor=A.+Sch%C3%B6neggerauthor=R.+Marcellusauthor=M.+Bilbanauthor=C.+Bockauthor=P.+J.+Brownauthor=J.+Zuberauthor=K.+L.+Bennettauthor=R.+Al-awarauthor=R.+Delwelauthor=C.+Nerlovauthor=C.+H.+Arrowsmithauthor=G.+Superti-Furga&title=Pharmacological+targeting+of+the+WDR5-MLL1+interaction+in+C%2FEBP%CE%B1+N-terminal+leukemia&doi=10.1038%2Fnchembio.1859"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Pharmacological targeting of the Wdr5-MLL interaction in C/EBPα N-terminal leukemia</span></div><div class="casAuthors">Grebien, Florian; Vedadi, Masoud; Getlik, Matthaeus; Giambruno, Roberto; Grover, Amit; Avellino, Roberto; Skucha, Anna; Vittori, Sarah; Kuznetsova, Ekaterina; Smil, David; Barsyte-Lovejoy, Dalia; Li, Fengling; Poda, Gennadiy; Schapira, Matthieu; Wu, Hong; Dong, Aiping; Senisterra, Guillermo; Stukalov, Alexey; Huber, Kilian V. M.; Schoenegger, Andreas; Marcellus, Richard; Bilban, Martin; Bock, Christoph; Brown, Peter J.; Zuber, Johannes; Bennett, Keiryn L.; Al-awar, Rima; Delwel, Ruud; Nerlov, Claus; Arrowsmith, Cheryl H.; Superti-Furga, Giulio</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">571-578</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The CEBPA gene is mutated in 9% of patients with acute myeloid leukemia (AML).  Selective expression of a short (30-kDa) CCAAT-enhancer binding protein-α (C/EBPα) translational isoform, termed p30, represents the most common type of CEBPA mutation in AML.  The mol. mechanisms underlying p30-mediated transformation remain incompletely understood.  We show that C/EBPα p30, but not the normal p42 isoform, preferentially interacts with Wdr5, a key component of SET/MLL (SET-domain/mixed-lineage leukemia) histone-methyltransferase complexes.  Accordingly, p30-bound genomic regions were enriched for MLL-dependent H3K4me3 marks.  The p30-dependent increase in self-renewal and inhibition of myeloid differentiation required Wdr5, as downregulation of the latter inhibited proliferation and restored differentiation in p30-dependent AML models.  OICR-9429 is a new small-mol. antagonist of the Wdr5-MLL interaction.  This compd. selectively inhibited proliferation and induced differentiation in p30-expressing human AML cells.  Our data reveal the mechanism of p30-dependent transformation and establish the essential p30 cofactor Wdr5 as a therapeutic target in CEBPA-mutant AML.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqLFPcxLpW4sbVg90H21EOLACvtfcHk0lh58b71rtSvww"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXht1WltLfM&md5=91c1937ebcfd0552a8cf02e354f8c663</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1859&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1859%26sid%3Dliteratum%253Aachs%26aulast%3DGrebien%26aufirst%3DF.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DGetlik%26aufirst%3DM.%26aulast%3DGiambruno%26aufirst%3DR.%26aulast%3DGrover%26aufirst%3DA.%26aulast%3DAvellino%26aufirst%3DR.%26aulast%3DSkucha%26aufirst%3DA.%26aulast%3DVittori%26aufirst%3DS.%26aulast%3DKuznetsova%26aufirst%3DE.%26aulast%3DSmil%26aufirst%3DD.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DPoda%26aufirst%3DG.%26aulast%3DSchapira%26aufirst%3DM.%26aulast%3DWu%26aufirst%3DH.%26aulast%3DDong%26aufirst%3DA.%26aulast%3DSenisterra%26aufirst%3DG.%26aulast%3DStukalov%26aufirst%3DA.%26aulast%3DHuber%26aufirst%3DK.%2BV.%26aulast%3DSch%25C3%25B6negger%26aufirst%3DA.%26aulast%3DMarcellus%26aufirst%3DR.%26aulast%3DBilban%26aufirst%3DM.%26aulast%3DBock%26aufirst%3DC.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DZuber%26aufirst%3DJ.%26aulast%3DBennett%26aufirst%3DK.%2BL.%26aulast%3DAl-awar%26aufirst%3DR.%26aulast%3DDelwel%26aufirst%3DR.%26aulast%3DNerlov%26aufirst%3DC.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DSuperti-Furga%26aufirst%3DG.%26atitle%3DPharmacological%2520targeting%2520of%2520the%2520WDR5-MLL1%2520interaction%2520in%2520C%252FEBP%25CE%25B1%2520N-terminal%2520leukemia%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2015%26volume%3D11%26spage%3D571%26epage%3D578%26doi%3D10.1038%2Fnchembio.1859" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group">Zhu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sammons, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Donahue, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dou, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vedadi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Getlik, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barsyte-Lovejoy, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Al-awar, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Katona, B. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shilatifard, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hua, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Berger, S. L.</span><span> </span><span class="NLM_article-title">Gain-of-function p53 mutants co-opt chromatin pathways to drive cancer growth</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">525</span><span class="NLM_x">, </span> <span class="NLM_fpage">206</span><span class="NLM_x">–</span> <span class="NLM_lpage">211</span><span class="refDoi"> DOI: 10.1038/nature15251</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01630&amp;key=10.1038%2Fnature15251" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01630&amp;key=26331536" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01630&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsVCis7%252FI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=525&publication_year=2015&pages=206-211&author=J.+Zhuauthor=M.+A.+Sammonsauthor=G.+Donahueauthor=Z.+Douauthor=M.+Vedadiauthor=M.+Getlikauthor=D.+Barsyte-Lovejoyauthor=R.+Al-awarauthor=B.+W.+Katonaauthor=A.+Shilatifardauthor=J.+Huangauthor=X.+Huaauthor=C.+H.+Arrowsmithauthor=S.+L.+Berger&title=Gain-of-function+p53+mutants+co-opt+chromatin+pathways+to+drive+cancer+growth&doi=10.1038%2Fnature15251"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Gain-of-function p53 mutants co-opt chromatin pathways to drive cancer growth</span></div><div class="casAuthors">Zhu, Jiajun; Sammons, Morgan A.; Donahue, Greg; Dou, Zhixun; Vedadi, Masoud; Getlik, Matthaus; Barsyte-Lovejoy, Dalia; Al-awar, Rima; Katona, Bryson W.; Shilatifard, Ali; Huang, Jing; Hua, Xianxin; Arrowsmith, Cheryl H.; Berger, Shelley L.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">525</span>
        (<span class="NLM_cas:issue">7568</span>),
    <span class="NLM_cas:pages">206-211</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">TP53 (which encodes p53 protein) is the most frequently mutated gene among all human cancers.  Prevalent p53 missense mutations abrogate its tumor suppressive function and lead to a 'gain-of-function' (GOF) that promotes cancer.  Here we show that p53 GOF mutants bind to and upregulate chromatin regulatory genes, including the methyltransferases MLL1 (also known as KMT2A), MLL2 (also known as KMT2D), and acetyltransferase MOZ (also known as KAT6A or MYST3), resulting in genome-wide increases of histone methylation and acetylation.  Anal. of The Cancer Genome Atlas shows specific upregulation of MLL1, MLL2, and MOZ in p53 GOF patient-derived tumors, but not in wild-type p53 or p53 null tumors.  Cancer cell proliferation is markedly lowered by genetic knockdown of MLL1 or by pharmacol. inhibition of the MLL1 methyltransferase complex.  Our study reveals a novel chromatin mechanism underlying the progression of tumors with GOF p53, and suggests new possibilities for designing combinatorial chromatin-based therapies for treating individual cancers driven by prevalent GOF p53 mutations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrG4nSmyzIWUbVg90H21EOLACvtfcHk0ljdzE1qOZ1nQQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsVCis7%252FI&md5=e37e60db0774b56a8df447d46acaad2a</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1038%2Fnature15251&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature15251%26sid%3Dliteratum%253Aachs%26aulast%3DZhu%26aufirst%3DJ.%26aulast%3DSammons%26aufirst%3DM.%2BA.%26aulast%3DDonahue%26aufirst%3DG.%26aulast%3DDou%26aufirst%3DZ.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DGetlik%26aufirst%3DM.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DAl-awar%26aufirst%3DR.%26aulast%3DKatona%26aufirst%3DB.%2BW.%26aulast%3DShilatifard%26aufirst%3DA.%26aulast%3DHuang%26aufirst%3DJ.%26aulast%3DHua%26aufirst%3DX.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DBerger%26aufirst%3DS.%2BL.%26atitle%3DGain-of-function%2520p53%2520mutants%2520co-opt%2520chromatin%2520pathways%2520to%2520drive%2520cancer%2520growth%26jtitle%3DNature%26date%3D2015%26volume%3D525%26spage%3D206%26epage%3D211%26doi%3D10.1038%2Fnature15251" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group">Sun, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bell, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carter, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gherardi, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Poulos, R. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Milazzo, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wong, J. W. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Al-Awar, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tee, A. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, P. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Atmadibrata, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wong, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Trahair, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhao, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shohet, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Haupt, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schulte, J. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vedadi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">MacKenzie, K. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hüttelmaier, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perini, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marshall, G. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Braithwaite, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, T.</span><span> </span><span class="NLM_article-title">WDR5 supports an N-Myc transcriptional complex that drives a protumorigenic gene expression signature in neuroblastoma</span> <span class="citation_source-journal">Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">75</span><span class="NLM_x">, </span> <span class="NLM_fpage">5143</span><span class="NLM_x">–</span> <span class="NLM_lpage">5154</span><span class="refDoi"> DOI: 10.1158/0008-5472.CAN-15-0423</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01630&amp;key=10.1158%2F0008-5472.CAN-15-0423" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01630&amp;key=26471359" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01630&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvFensrzI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=75&publication_year=2015&pages=5143-5154&author=Y.+Sunauthor=J.+L.+Bellauthor=D.+Carterauthor=S.+Gherardiauthor=R.+C.+Poulosauthor=G.+Milazzoauthor=J.+W.+H.+Wongauthor=R.+Al-Awarauthor=A.+E.+Teeauthor=P.+Y.+Liuauthor=B.+Liuauthor=B.+Atmadibrataauthor=M.+Wongauthor=T.+Trahairauthor=Q.+Zhaoauthor=J.+M.+Shohetauthor=Y.+Hauptauthor=J.+H.+Schulteauthor=P.+J.+Brownauthor=C.+H.+Arrowsmithauthor=M.+Vedadiauthor=K.+L.+MacKenzieauthor=S.+H%C3%BCttelmaierauthor=G.+Periniauthor=G.+M.+Marshallauthor=A.+Braithwaiteauthor=T.+Liu&title=WDR5+supports+an+N-Myc+transcriptional+complex+that+drives+a+protumorigenic+gene+expression+signature+in+neuroblastoma&doi=10.1158%2F0008-5472.CAN-15-0423"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">WDR5 Supports an N-Myc Transcriptional Complex That Drives a Protumorigenic Gene Expression Signature in Neuroblastoma</span></div><div class="casAuthors">Sun, Yuting; Bell, Jessica L.; Carter, Daniel; Gherardi, Samuele; Poulos, Rebecca C.; Milazzo, Giorgio; Wong, Jason W. H.; Al-Awar, Rima; Tee, Andrew E.; Liu, Pei Y.; Liu, Bing; Atmadibrata, Bernard; Wong, Matthew; Trahair, Toby; Zhao, Quan; Shohet, Jason M.; Haupt, Ygal; Schulte, Johannes H.; Brown, Peter J.; Arrowsmith, Cheryl H.; Vedadi, Masoud; MacKenzie, Karen L.; Huettelmaier, Stefan; Perini, Giovanni; Marshall, Glenn M.; Braithwaite, Antony; Liu, Tao</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">75</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">5143-5154</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">MYCN gene amplification in neuroblastoma drives a gene expression program that correlates strongly with aggressive disease.  Mechanistically, trimethylation of histone H3 lysine 4 (H3K4) at target gene promoters is a strict prerequisite for this transcriptional program to be enacted.  WDR5 is a histone H3K4 presenter that has been found to have an essential role in H3K4 trimethylation.  For this reason, in this study, we investigated the relationship between WDR5-mediated H3K4 trimethylation and N-Myc transcriptional programs in neuroblastoma cells.  N-Myc upregulated WDR5 expression in neuroblastoma cells.  Gene expression anal. revealed that WDR5 target genes included those with MYC-binding elements at promoters such as MDM2.  We showed that WDR5 could form a protein complex at the MDM2 promoter with N-Myc, but not p53, leading to histone H3K4 trimethylation and activation of MDM2 transcription.  RNAi-mediated attenuation of WDR5 upregulated expression of wild-type but not mutant p53, an effect assocd. with growth inhibition and apoptosis.  Similarly, a small-mol. antagonist of WDR5 reduced N-Myc/WDR5 complex formation, N-Myc target gene expression, and cell growth in neuroblastoma cells.  In MYCN-transgenic mice, WDR5 was overexpressed in precancerous ganglion and neuroblastoma cells compared with normal ganglion cells.  Clin., elevated levels of WDR5 in neuroblastoma specimens were an independent predictor of poor overall survival.  Overall, our results identify WDR5 as a key cofactor for N-Myc-regulated transcriptional activation and tumorigenesis and as a novel therapeutic target for MYCN-amplified neuroblastomas.  Cancer Res; 75(23); 5143-54. ©2015 AACR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpdt2xhQOC6ibVg90H21EOLACvtfcHk0lg5RP0uXOshSA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvFensrzI&md5=26513b24ba2ba6b82cc691534f5939cd</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-15-0423&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-15-0423%26sid%3Dliteratum%253Aachs%26aulast%3DSun%26aufirst%3DY.%26aulast%3DBell%26aufirst%3DJ.%2BL.%26aulast%3DCarter%26aufirst%3DD.%26aulast%3DGherardi%26aufirst%3DS.%26aulast%3DPoulos%26aufirst%3DR.%2BC.%26aulast%3DMilazzo%26aufirst%3DG.%26aulast%3DWong%26aufirst%3DJ.%2BW.%2BH.%26aulast%3DAl-Awar%26aufirst%3DR.%26aulast%3DTee%26aufirst%3DA.%2BE.%26aulast%3DLiu%26aufirst%3DP.%2BY.%26aulast%3DLiu%26aufirst%3DB.%26aulast%3DAtmadibrata%26aufirst%3DB.%26aulast%3DWong%26aufirst%3DM.%26aulast%3DTrahair%26aufirst%3DT.%26aulast%3DZhao%26aufirst%3DQ.%26aulast%3DShohet%26aufirst%3DJ.%2BM.%26aulast%3DHaupt%26aufirst%3DY.%26aulast%3DSchulte%26aufirst%3DJ.%2BH.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DMacKenzie%26aufirst%3DK.%2BL.%26aulast%3DH%25C3%25BCttelmaier%26aufirst%3DS.%26aulast%3DPerini%26aufirst%3DG.%26aulast%3DMarshall%26aufirst%3DG.%2BM.%26aulast%3DBraithwaite%26aufirst%3DA.%26aulast%3DLiu%26aufirst%3DT.%26atitle%3DWDR5%2520supports%2520an%2520N-Myc%2520transcriptional%2520complex%2520that%2520drives%2520a%2520protumorigenic%2520gene%2520expression%2520signature%2520in%2520neuroblastoma%26jtitle%3DCancer%2520Res.%26date%3D2015%26volume%3D75%26spage%3D5143%26epage%3D5154%26doi%3D10.1158%2F0008-5472.CAN-15-0423" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="note"><p class="first last">Data not shown. Unpublished results.</p></div><div class="citationLinks"><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><div class="note"><p class="first last">In order to obtain a better signal in the FP assay used for this publication, we synthesized the described 9-Ala-FAM peptide based on the 10-Ala-FAM peptide reported in</p></div><span class="NLM_contrib-group">Karatas, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Townsend, E. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bernard, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dou, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, S.</span><span> </span><span class="NLM_article-title">Analysis of the binding of mixed-lineage leukemia 1 (MLL1) and histone 3 peptides to WD repeat domain 5 (WDR5) for the design of inhibitors of the MLL1-WDR5 interaction</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">5179</span><span class="NLM_x">–</span> <span class="NLM_lpage">5185</span><span class="refDoi"> DOI: 10.1021/jm100139b</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm100139b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=5179-5185&author=H.+Karatasauthor=E.+C.+Townsendauthor=D.+Bernardauthor=Y.+Douauthor=S.+Wang&title=Analysis+of+the+binding+of+mixed-lineage+leukemia+1+%28MLL1%29+and+histone+3+peptides+to+WD+repeat+domain+5+%28WDR5%29+for+the+design+of+inhibitors+of+the+MLL1-WDR5+interaction&doi=10.1021%2Fjm100139b"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1021%2Fjm100139b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm100139b%26sid%3Dliteratum%253Aachs%26aulast%3DKaratas%26aufirst%3DH.%26aulast%3DTownsend%26aufirst%3DE.%2BC.%26aulast%3DBernard%26aufirst%3DD.%26aulast%3DDou%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DAnalysis%2520of%2520the%2520binding%2520of%2520mixed-lineage%2520leukemia%25201%2520%2528MLL1%2529%2520and%2520histone%25203%2520peptides%2520to%2520WD%2520repeat%2520domain%25205%2520%2528WDR5%2529%2520for%2520the%2520design%2520of%2520inhibitors%2520of%2520the%2520MLL1-WDR5%2520interaction%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D5179%26epage%3D5185%26doi%3D10.1021%2Fjm100139b" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group">Bissantz, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuhn, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stahl, M.</span><span> </span><span class="NLM_article-title">A medicinal chemist’s guide to molecular interactions</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">5061</span><span class="NLM_x">–</span> <span class="NLM_lpage">5084</span><span class="refDoi"> DOI: 10.1021/jm100112j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm100112j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01630&amp;key=1%3ACAS%3A528%3ADC%252BC3cXjvFKjtbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=5061-5084&author=C.+Bissantzauthor=B.+Kuhnauthor=M.+Stahl&title=A+medicinal+chemist%E2%80%99s+guide+to+molecular+interactions&doi=10.1021%2Fjm100112j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">A Medicinal Chemist's Guide to Molecular Interactions</span></div><div class="casAuthors">Bissantz, Caterina; Kuhn, Bernd; Stahl, Martin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">5061-5084</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGraV8HPbwsihbVg90H21EOLACvtfcHk0lg5RP0uXOshSA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXjvFKjtbs%253D&md5=f39cf8700267cdc311c73de4d433bab0</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1021%2Fjm100112j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm100112j%26sid%3Dliteratum%253Aachs%26aulast%3DBissantz%26aufirst%3DC.%26aulast%3DKuhn%26aufirst%3DB.%26aulast%3DStahl%26aufirst%3DM.%26atitle%3DA%2520medicinal%2520chemist%25E2%2580%2599s%2520guide%2520to%2520molecular%2520interactions%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D5061%26epage%3D5084%26doi%3D10.1021%2Fjm100112j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="note"><p class="first last">Qualitative finding based on the apparent solubility of the compounds in common organic solvents such as acetonitrile, methanol, and DMSO observed during the course of experimental work, measurements using SPR, and compound characterization by NMR.</p></div><div class="citationLinks"><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span>Currently BIOVIA; Virtual Biosphere and Materials. <a href="http://www.3ds.com/products-services/biovia/" class="extLink">http://www.3ds.com/products-services/biovia/</a> (accessed February 17,<span class="NLM_x"> </span><span class="NLM_year">2016</span><span class="NLM_x">) </span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Currently+BIOVIA%3B+Virtual+Biosphere+and+Materials.+http%3A%2F%2Fwww.3ds.com%2Fproducts-services%2Fbiovia%2F+%28accessed+February+17%2C+2016%29+."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweb%26sid%3Dliteratum%253Aachs%26jtitle%3DBIOVIA%253B%2520Virtual%2520Biosphere%2520and%2520Materials%26date%3D2016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span>eMolecules. <a href="http://www.emolecules.com" class="extLink">http://www.emolecules.com</a> (accessed February 17,<span class="NLM_x"> </span><span class="NLM_year">2016</span><span class="NLM_x">) </span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=eMolecules.+http%3A%2F%2Fwww.emolecules.com+%28accessed+February+17%2C+2016%29+."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweb%26sid%3Dliteratum%253Aachs%26jtitle%3DeMolecules%26date%3D2016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span>Glide docking program. <a href="http://www.schrodinger.com" class="extLink">http://www.schrodinger.com</a> (accessed February 17,<span class="NLM_x"> </span><span class="NLM_year">2016</span><span class="NLM_x">) </span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Glide+docking+program.+http%3A%2F%2Fwww.schrodinger.com+%28accessed+February+17%2C+2016%29+."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweb%26sid%3Dliteratum%253Aachs%26jtitle%3DGlide%2520docking%2520program%26date%3D2016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span>Cerep. <a href="http://www.cerep.fr/cerep/users/pages/catalog/profiles/catalog.asp" class="extLink">http://www.cerep.fr/cerep/users/pages/catalog/profiles/catalog.asp</a> (accessed February 17,<span class="NLM_x"> </span><span class="NLM_year">2016</span><span class="NLM_x">) </span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Cerep.+http%3A%2F%2Fwww.cerep.fr%2Fcerep%2Fusers%2Fpages%2Fcatalog%2Fprofiles%2Fcatalog.asp+%28accessed+February+17%2C+2016%29+."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweb%26sid%3Dliteratum%253Aachs%26jtitle%3DCerep%26date%3D2016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group">Zuber, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Radtke, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pardee, T. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhao, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rappaport, A. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luo, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCurrach, M. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, M.-M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dolan, M. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kogan, S. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Downing, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lowe, S. W.</span><span> </span><span class="NLM_article-title">Mouse models of human AML accurately predict chemotherapy response</span> <span class="citation_source-journal">Genes Dev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">23</span><span class="NLM_x">, </span> <span class="NLM_fpage">877</span><span class="NLM_x">–</span> <span class="NLM_lpage">889</span><span class="refDoi"> DOI: 10.1101/gad.1771409</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01630&amp;key=10.1101%2Fgad.1771409" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01630&amp;key=19339691" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01630&amp;key=1%3ACAS%3A528%3ADC%252BD1MXks1Srurs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2009&pages=877-889&author=J.+Zuberauthor=I.+Radtkeauthor=T.+S.+Pardeeauthor=Z.+Zhaoauthor=A.+R.+Rappaportauthor=W.+Luoauthor=M.+E.+McCurrachauthor=M.-M.+Yangauthor=M.+E.+Dolanauthor=S.+C.+Koganauthor=J.+R.+Downingauthor=S.+W.+Lowe&title=Mouse+models+of+human+AML+accurately+predict+chemotherapy+response&doi=10.1101%2Fgad.1771409"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Mouse models of human AML accurately predict chemotherapy response</span></div><div class="casAuthors">Zuber, Johannes; Radtke, Ina; Pardee, Timothy S.; Zhao, Zhen; Rappaport, Amy R.; Luo, Weijun; McCurrach, Mila E.; Yang, Miao-Miao; Dolan, M. Eileen; Kogan, Scott C.; Downing, James R.; Lowe, Scott W.</div><div class="citationInfo"><span class="NLM_cas:title">Genes & Development</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">877-889</span>CODEN:
                <span class="NLM_cas:coden">GEDEEP</span>;
        ISSN:<span class="NLM_cas:issn">0890-9369</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">The genetic heterogeneity of cancer influences the trajectory of tumor progression and may underlie clin. variation in therapy response.  To model such heterogeneity, we produced genetically and pathol. accurate mouse models of common forms of human acute myeloid leukemia (AML) and developed methods to mimic std. induction chemotherapy and efficiently monitor therapy response.  We see that murine AMLs harboring two common human AML genotypes show remarkably diverse responses to conventional therapy that mirror clin. experience.  Specifically, murine leukemias expressing the AML1/ETO fusion oncoprotein, assocd. with a favorable prognosis in patients, show a dramatic response to induction chemotherapy owing to robust activation of the p53 tumor suppressor network.  Conversely, murine leukemias expressing MLL fusion proteins, assocd. with a dismal prognosis in patients, are drug-resistant due to an attenuated p53 response.  These studies highlight the importance of genetic information in guiding the treatment of human AML, functionally establish the p53 network as a central determinant of chemotherapy response in AML, and demonstrate that genetically engineered mouse models of human cancer can accurately predict therapy response in patients.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpjBGFSrry7j7Vg90H21EOLACvtfcHk0lhsO-CA07nkkQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXks1Srurs%253D&md5=2307241c6a572fe14a6ce00af7a93f1c</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1101%2Fgad.1771409&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fgad.1771409%26sid%3Dliteratum%253Aachs%26aulast%3DZuber%26aufirst%3DJ.%26aulast%3DRadtke%26aufirst%3DI.%26aulast%3DPardee%26aufirst%3DT.%2BS.%26aulast%3DZhao%26aufirst%3DZ.%26aulast%3DRappaport%26aufirst%3DA.%2BR.%26aulast%3DLuo%26aufirst%3DW.%26aulast%3DMcCurrach%26aufirst%3DM.%2BE.%26aulast%3DYang%26aufirst%3DM.-M.%26aulast%3DDolan%26aufirst%3DM.%2BE.%26aulast%3DKogan%26aufirst%3DS.%2BC.%26aulast%3DDowning%26aufirst%3DJ.%2BR.%26aulast%3DLowe%26aufirst%3DS.%2BW.%26atitle%3DMouse%2520models%2520of%2520human%2520AML%2520accurately%2520predict%2520chemotherapy%2520response%26jtitle%3DGenes%2520Dev.%26date%3D2009%26volume%3D23%26spage%3D877%26epage%3D889%26doi%3D10.1101%2Fgad.1771409" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group">McCormack, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bruserud, Ø</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gjertsen, B. T.</span><span> </span><span class="NLM_article-title">Animal models of acute myelogenous leukaemia - development, application and future perspectives</span> <span class="citation_source-journal">Leukemia</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">687</span><span class="NLM_x">–</span> <span class="NLM_lpage">706</span><span class="refDoi"> DOI: 10.1038/sj.leu.2403670</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01630&amp;key=10.1038%2Fsj.leu.2403670" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2005&pages=687-706&author=E.+McCormackauthor=%C3%98+Bruserudauthor=B.+T.+Gjertsen&title=Animal+models+of+acute+myelogenous+leukaemia+-+development%2C+application+and+future+perspectives&doi=10.1038%2Fsj.leu.2403670"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1038%2Fsj.leu.2403670&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.leu.2403670%26sid%3Dliteratum%253Aachs%26aulast%3DMcCormack%26aufirst%3DE.%26aulast%3DBruserud%26aufirst%3D%25C3%2598%26aulast%3DGjertsen%26aufirst%3DB.%2BT.%26atitle%3DAnimal%2520models%2520of%2520acute%2520myelogenous%2520leukaemia%2520-%2520development%252C%2520application%2520and%2520future%2520perspectives%26jtitle%3DLeukemia%26date%3D2005%26volume%3D19%26spage%3D687%26epage%3D706%26doi%3D10.1038%2Fsj.leu.2403670" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':[],'ref16':['cit16'],'ref17':['cit17'],'ref18':[],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 38 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors"><span class="NLM_string-name hlFld-ContribAuthor">Antti Poso</span>. </span><span class="cited-content_cbyCitation_article-title">The Future of Medicinal Chemistry, PROTAC, and Undruggable Drug Targets. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021</strong>, Article ASAP.</span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.1c01126&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.1c01126%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DThe%252BFuture%252Bof%252BMedicinal%252BChemistry%25252C%252BPROTAC%25252C%252Band%252BUndruggable%252BDrug%252BTargets%26aulast%3DPoso%26aufirst%3DAntti%26date%3D2021%26date%3D2021%26date%3D05072021%26volume%3D0%26issue%3D0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Weilin Chen, Xin Chen, Dongdong Li, Jianrui Zhou, Zhengyu Jiang, Qidong You, <span class="NLM_string-name hlFld-ContribAuthor">Xiaoke Guo</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of DDO-2213 as a Potent and Orally Bioavailable Inhibitor of the WDR5–Mixed Lineage Leukemia 1 Protein–Protein Interaction for the Treatment of MLL Fusion Leukemia. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>64 </em>
                                    (12)
                                     , 8221-8245. <a href="https://doi.org/10.1021/acs.jmedchem.1c00091" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.1c00091</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.1c00091&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.1c00091%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252BDDO-2213%252Bas%252Ba%252BPotent%252Band%252BOrally%252BBioavailable%252BInhibitor%252Bof%252Bthe%252BWDR5%2525E2%252580%252593Mixed%252BLineage%252BLeukemia%252B1%252BProtein%2525E2%252580%252593Protein%252BInteraction%252Bfor%252Bthe%252BTreatment%252Bof%252BMLL%252BFusion%252BLeukemia%26aulast%3DChen%26aufirst%3DWeilin%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D19012021%26date%3D09062021%26volume%3D64%26issue%3D12%26spage%3D8221%26epage%3D8245" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Anja Dölle, Bikash Adhikari, Andreas Krämer, Janik Weckesser, Nicola Berner, Lena-Marie Berger, Mathias Diebold, Magdalena M. Szewczyk, Dalia Barsyte-Lovejoy, Cheryl H. Arrowsmith, Jakob Gebel, Frank Löhr, Volker Dötsch, Martin Eilers, Stephanie Heinzlmeir, Bernhard Kuster, Christoph Sotriffer, Elmar Wolf, <span class="NLM_string-name hlFld-ContribAuthor">Stefan Knapp</span>. </span><span class="cited-content_cbyCitation_article-title">Design, Synthesis, and Evaluation of WD-Repeat-Containing Protein 5 (WDR5) Degraders. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021</strong>, Article ASAP.</span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.1c00146&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.1c00146%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDesign%25252C%252BSynthesis%25252C%252Band%252BEvaluation%252Bof%252BWD-Repeat-Containing%252BProtein%252B5%252B%252528WDR5%252529%252BDegraders%26aulast%3DD%25C3%25B6lle%26aufirst%3DAnja%26date%3D2021%26date%3D2021%26date%3D25012021%26volume%3D0%26issue%3D0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Jianhua Tian, Kevin B. Teuscher, Erin R. Aho, Joseph R. Alvarado, Jonathan J. Mills, Kenneth M. Meyers, Rocco D. Gogliotti, Changho Han, Jonathan D. Macdonald, Jiqing Sai, J. Grace Shaw, John L. Sensintaffar, Bin Zhao, Tyson A. Rietz, Lance R. Thomas, William G. Payne, William J. Moore, Gordon M. Stott, Jumpei Kondo, Masahiro Inoue, Robert J. Coffey, William P. Tansey, Shaun R. Stauffer, Taekyu Lee, <span class="NLM_string-name hlFld-ContribAuthor">Stephen W. Fesik</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery and Structure-Based Optimization of Potent and Selective WD Repeat Domain 5 (WDR5) Inhibitors Containing a Dihydroisoquinolinone Bicyclic Core. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (2)
                                     , 656-675. <a href="https://doi.org/10.1021/acs.jmedchem.9b01608" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b01608</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b01608&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.9b01608%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Band%252BStructure-Based%252BOptimization%252Bof%252BPotent%252Band%252BSelective%252BWD%252BRepeat%252BDomain%252B5%252B%252528WDR5%252529%252BInhibitors%252BContaining%252Ba%252BDihydroisoquinolinone%252BBicyclic%252BCore%26aulast%3DTian%26aufirst%3DJianhua%26date%3D2020%26date%3D2019%26date%3D2019%26date%3D27092019%26date%3D07012020%26date%3D20122019%26volume%3D63%26issue%3D2%26spage%3D656%26epage%3D675" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Jonathan D. Macdonald, Selena Chacón Simon, Changho Han, Feng Wang, J. Grace Shaw, Jennifer E. Howes, Jiqing Sai, Joannes P. Yuh, Demarco Camper, Bethany M. Alicie, Joseph Alvarado, Sameer Nikhar, William Payne, Erin R. Aho, Joshua A. Bauer, Bin Zhao, Jason Phan, Lance R. Thomas, Olivia W. Rossanese, William P. Tansey, Alex G. Waterson, Shaun R. Stauffer, <span class="NLM_string-name hlFld-ContribAuthor">Stephen W. Fesik</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery and Optimization of Salicylic Acid-Derived Sulfonamide Inhibitors of the WD Repeat-Containing Protein 5–MYC Protein–Protein Interaction. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2019,</strong> <em>62 </em>
                                    (24)
                                     , 11232-11259. <a href="https://doi.org/10.1021/acs.jmedchem.9b01411" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b01411</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b01411&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.9b01411%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Band%252BOptimization%252Bof%252BSalicylic%252BAcid-Derived%252BSulfonamide%252BInhibitors%252Bof%252Bthe%252BWD%252BRepeat-Containing%252BProtein%252B5%2525E2%252580%252593MYC%252BProtein%2525E2%252580%252593Protein%252BInteraction%26aulast%3DMacdonald%26aufirst%3DJonathan%2BD.%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D26082019%26date%3D05122019%26date%3D14112019%26volume%3D62%26issue%3D24%26spage%3D11232%26epage%3D11259" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Feng Wang, Kyu Ok Jeon, James M. Salovich, Jonathan D. Macdonald, Joseph Alvarado, Rocco D. Gogliotti, Jason Phan, Edward T. Olejniczak, Qi Sun, Shidong Wang, DeMarco Camper, Joannes P. Yuh, J. Grace Shaw, Jiqing Sai, Olivia W. Rossanese, William P. Tansey, Shaun R. Stauffer, <span class="NLM_string-name hlFld-ContribAuthor">Stephen W. Fesik</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of Potent 2-Aryl-6,7-dihydro-5H-pyrrolo[1,2-a]imidazoles as WDR5-WIN-Site Inhibitors Using Fragment-Based Methods and Structure-Based Design. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2018,</strong> <em>61 </em>
                                    (13)
                                     , 5623-5642. <a href="https://doi.org/10.1021/acs.jmedchem.8b00375" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.8b00375</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.8b00375&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.8b00375%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252BPotent%252B2-Aryl-6%25252C7-dihydro-5H-pyrrolo%25255B1%25252C2-a%25255Dimidazoles%252Bas%252BWDR5-WIN-Site%252BInhibitors%252BUsing%252BFragment-Based%252BMethods%252Band%252BStructure-Based%252BDesign%26aulast%3DWang%26aufirst%3DFeng%26date%3D2018%26date%3D2018%26date%3D2018%26date%3D08032018%26date%3D29062018%26date%3D11062018%26volume%3D61%26issue%3D13%26spage%3D5623%26epage%3D5642" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">H. Ümit  Kaniskan</span>, <span class="hlFld-ContribAuthor ">Michael L.  Martini</span>, and <span class="hlFld-ContribAuthor ">Jian  Jin</span>  . </span><span class="cited-content_cbyCitation_article-title">Inhibitors of Protein Methyltransferases and Demethylases. </span><span class="cited-content_cbyCitation_journal-name">Chemical Reviews</span><span> <strong>2018,</strong> <em>118 </em>
                                    (3)
                                     , 989-1068. <a href="https://doi.org/10.1021/acs.chemrev.6b00801" title="DOI URL">https://doi.org/10.1021/acs.chemrev.6b00801</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.chemrev.6b00801&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.chemrev.6b00801%26sid%3Dliteratum%253Aachs%26jtitle%3DChemical%2520Reviews%26atitle%3DInhibitors%252Bof%252BProtein%252BMethyltransferases%252Band%252BDemethylases%26aulast%3DKaniskan%26aufirst%3DH.%2B%25C3%259Cmit%26date%3D2018%26date%3D2017%26date%3D2016%26date%3D30112016%26date%3D24032017%26date%3D14022018%26volume%3D118%26issue%3D3%26spage%3D989%26epage%3D1068" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Hacer  Karatas</span>, <span class="hlFld-ContribAuthor ">Yangbing  Li</span>, <span class="hlFld-ContribAuthor ">Liu  Liu</span>, <span class="hlFld-ContribAuthor ">Jiao  Ji</span>, <span class="hlFld-ContribAuthor ">Shirley  Lee</span>, <span class="hlFld-ContribAuthor ">Yong  Chen</span>, <span class="hlFld-ContribAuthor ">Jiuling  Yang</span>, <span class="hlFld-ContribAuthor ">Liyue  Huang</span>, <span class="hlFld-ContribAuthor ">Denzil  Bernard</span>, <span class="hlFld-ContribAuthor ">Jing  Xu</span>, <span class="hlFld-ContribAuthor ">Elizabeth C.  Townsend</span>, <span class="hlFld-ContribAuthor ">Fang  Cao</span>, <span class="hlFld-ContribAuthor ">Xu  Ran</span>, <span class="hlFld-ContribAuthor ">Xiaoqin  Li</span>, <span class="hlFld-ContribAuthor ">Bo  Wen</span>, <span class="hlFld-ContribAuthor ">Duxin  Sun</span>, <span class="hlFld-ContribAuthor ">Jeanne A  Stuckey</span>, <span class="hlFld-ContribAuthor ">Ming  Lei</span>, <span class="hlFld-ContribAuthor ">Yali  Dou</span>, and <span class="hlFld-ContribAuthor ">Shaomeng  Wang</span>  . </span><span class="cited-content_cbyCitation_article-title">Discovery of a Highly Potent, Cell-Permeable Macrocyclic Peptidomimetic (MM-589) Targeting the WD Repeat Domain 5 Protein (WDR5)–Mixed Lineage Leukemia (MLL) Protein–Protein Interaction. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2017,</strong> <em>60 </em>
                                    (12)
                                     , 4818-4839. <a href="https://doi.org/10.1021/acs.jmedchem.6b01796" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.6b01796</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.6b01796&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.6b01796%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252Ba%252BHighly%252BPotent%25252C%252BCell-Permeable%252BMacrocyclic%252BPeptidomimetic%252B%252528MM-589%252529%252BTargeting%252Bthe%252BWD%252BRepeat%252BDomain%252B5%252BProtein%252B%252528WDR5%252529%2525E2%252580%252593Mixed%252BLineage%252BLeukemia%252B%252528MLL%252529%252BProtein%2525E2%252580%252593Protein%252BInteraction%26aulast%3DKaratas%26aufirst%3DHacer%26date%3D2017%26date%3D2017%26date%3D2016%26date%3D09122016%26date%3D12062017%26date%3D22062017%26volume%3D60%26issue%3D12%26spage%3D4818%26epage%3D4839" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ting-Ting  Yao</span>, <span class="hlFld-ContribAuthor ">Shao-Wei  Fang</span>, <span class="hlFld-ContribAuthor ">Zhong-Shan  Li</span>, <span class="hlFld-ContribAuthor ">Dou-Xin  Xiao</span>, <span class="hlFld-ContribAuthor ">Jing-Li  Cheng</span>, <span class="hlFld-ContribAuthor ">Hua-Zhou  Ying</span>, <span class="hlFld-ContribAuthor ">Yong-Jun  Du</span>, <span class="hlFld-ContribAuthor ">Jin-Hao  Zhao</span>, and <span class="hlFld-ContribAuthor ">Xiao-Wu  Dong</span>  . </span><span class="cited-content_cbyCitation_article-title">Discovery of Novel Succinate Dehydrogenase Inhibitors by the Integration of in Silico Library Design and Pharmacophore Mapping. </span><span class="cited-content_cbyCitation_journal-name">Journal of Agricultural and Food Chemistry</span><span> <strong>2017,</strong> <em>65 </em>
                                    (15)
                                     , 3204-3211. <a href="https://doi.org/10.1021/acs.jafc.7b00249" title="DOI URL">https://doi.org/10.1021/acs.jafc.7b00249</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jafc.7b00249&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jafc.7b00249%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Agricultural%2520and%2520Food%2520Chemistry%26atitle%3DDiscovery%252Bof%252BNovel%252BSuccinate%252BDehydrogenase%252BInhibitors%252Bby%252Bthe%252BIntegration%252Bof%252Bin%252BSilico%252BLibrary%252BDesign%252Band%252BPharmacophore%252BMapping%26aulast%3DYao%26aufirst%3DTing-Ting%26date%3D2017%26date%3D2017%26date%3D2017%26date%3D2017%26date%3D2017%26date%3D17012017%26date%3D29032017%26date%3D30032017%26date%3D05042017%26date%3D19042017%26date%3D30032017%26volume%3D65%26issue%3D15%26spage%3D3204%26epage%3D3211" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xin  Li</span>, <span class="hlFld-ContribAuthor ">Yongcheng  Song</span>. </span><span class="cited-content_cbyCitation_article-title">Structure, function and inhibition of critical protein–protein interactions involving mixed lineage leukemia 1 and its fusion oncoproteins. </span><span class="cited-content_cbyCitation_journal-name">Journal of Hematology & Oncology</span><span> <strong>2021,</strong> <em>14 </em>
                                    (1)
                                     <a href="https://doi.org/10.1186/s13045-021-01057-7" title="DOI URL">https://doi.org/10.1186/s13045-021-01057-7</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1186/s13045-021-01057-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1186%2Fs13045-021-01057-7%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Hematology%2520%2526%2520Oncology%26atitle%3DStructure%25252C%252Bfunction%252Band%252Binhibition%252Bof%252Bcritical%252Bprotein%2525E2%252580%252593protein%252Binteractions%252Binvolving%252Bmixed%252Blineage%252Bleukemia%252B1%252Band%252Bits%252Bfusion%252Boncoproteins%26aulast%3DLi%26aufirst%3DXin%26date%3D2021%26date%3D2021%26volume%3D14%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Weilin  Chen</span>, <span class="hlFld-ContribAuthor ">Xin  Chen</span>, <span class="hlFld-ContribAuthor ">Dongdong  Li</span>, <span class="hlFld-ContribAuthor ">Xianghan  Wang</span>, <span class="hlFld-ContribAuthor ">Guanlu  Long</span>, <span class="hlFld-ContribAuthor ">Zhengyu  Jiang</span>, <span class="hlFld-ContribAuthor ">Qidong  You</span>, <span class="hlFld-ContribAuthor ">Xiaoke  Guo</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of a potent MLL1 and WDR5 protein-protein interaction inhibitor with in vivo antitumor activity. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>223 </em>, 113677. <a href="https://doi.org/10.1016/j.ejmech.2021.113677" title="DOI URL">https://doi.org/10.1016/j.ejmech.2021.113677</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2021.113677&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2021.113677%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252Ba%252Bpotent%252BMLL1%252Band%252BWDR5%252Bprotein-protein%252Binteraction%252Binhibitor%252Bwith%252Bin%2525C2%2525A0vivo%252Bantitumor%252Bactivity%26aulast%3DChen%26aufirst%3DWeilin%26date%3D2021%26volume%3D223%26spage%3D113677" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ali  Imran</span>, <span class="hlFld-ContribAuthor ">Brandon S.  Moyer</span>, <span class="hlFld-ContribAuthor ">Ashley J.  Canning</span>, <span class="hlFld-ContribAuthor ">Dan  Kalina</span>, <span class="hlFld-ContribAuthor ">Thomas M.  Duncan</span>, <span class="hlFld-ContribAuthor ">Kelsey J.  Moody</span>, <span class="hlFld-ContribAuthor ">Aaron J.  Wolfe</span>, <span class="hlFld-ContribAuthor ">Michael S.  Cosgrove</span>, <span class="hlFld-ContribAuthor ">Liviu  Movileanu</span>. </span><span class="cited-content_cbyCitation_article-title">Kinetics of the multitasking high-affinity Win binding site of WDR5 in restricted and unrestricted conditions. </span><span class="cited-content_cbyCitation_journal-name">Biochemical Journal</span><span> <strong>2021,</strong> <em>478 </em>
                                    (11)
                                     , 2145-2161. <a href="https://doi.org/10.1042/BCJ20210253" title="DOI URL">https://doi.org/10.1042/BCJ20210253</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1042/BCJ20210253&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1042%2FBCJ20210253%26sid%3Dliteratum%253Aachs%26jtitle%3DBiochemical%2520Journal%26atitle%3DKinetics%252Bof%252Bthe%252Bmultitasking%252Bhigh-affinity%252BWin%252Bbinding%252Bsite%252Bof%252BWDR5%252Bin%252Brestricted%252Band%252Bunrestricted%252Bconditions%26aulast%3DImran%26aufirst%3DAli%26date%3D2021%26date%3D2021%26volume%3D478%26issue%3D11%26spage%3D2145%26epage%3D2161" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jiawen  Cao</span>, <span class="hlFld-ContribAuthor ">Tiantian  Fan</span>, <span class="hlFld-ContribAuthor ">Yanlian  Li</span>, <span class="hlFld-ContribAuthor ">Zhiyan  Du</span>, <span class="hlFld-ContribAuthor ">Lin  Chen</span>, <span class="hlFld-ContribAuthor ">Ying  Wang</span>, <span class="hlFld-ContribAuthor ">Xin  Wang</span>, <span class="hlFld-ContribAuthor ">Jingkang  Shen</span>, <span class="hlFld-ContribAuthor ">Xun  Huang</span>, <span class="hlFld-ContribAuthor ">Bing  Xiong</span>, <span class="hlFld-ContribAuthor ">Danyan  Cao</span>. </span><span class="cited-content_cbyCitation_article-title">Phage-Display Based Discovery and Characterization of Peptide Ligands against WDR5. </span><span class="cited-content_cbyCitation_journal-name">Molecules</span><span> <strong>2021,</strong> <em>26 </em>
                                    (5)
                                     , 1225. <a href="https://doi.org/10.3390/molecules26051225" title="DOI URL">https://doi.org/10.3390/molecules26051225</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/molecules26051225&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fmolecules26051225%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecules%26atitle%3DPhage-Display%252BBased%252BDiscovery%252Band%252BCharacterization%252Bof%252BPeptide%252BLigands%252Bagainst%252BWDR5%26aulast%3DCao%26aufirst%3DJiawen%26date%3D2021%26date%3D2021%26volume%3D26%26issue%3D5%26spage%3D1225" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xiaoqing  Ye</span>, <span class="hlFld-ContribAuthor ">Gang  Chen</span>, <span class="hlFld-ContribAuthor ">Jia  Jin</span>, <span class="hlFld-ContribAuthor ">Binzhong  Zhang</span>, <span class="hlFld-ContribAuthor ">Yinda  Wang</span>, <span class="hlFld-ContribAuthor ">Zhenhai  Cai</span>, <span class="hlFld-ContribAuthor ">Fei  Ye</span>. </span><span class="cited-content_cbyCitation_article-title">The Development of Inhibitors Targeting the Mixed Lineage Leukemia 1 (MLL1)-WD Repeat Domain 5 Protein (WDR5) Protein- Protein Interaction. </span><span class="cited-content_cbyCitation_journal-name">Current Medicinal Chemistry</span><span> <strong>2020,</strong> <em>27 </em>
                                    (33)
                                     , 5530-5542. <a href="https://doi.org/10.2174/0929867326666190528080514" title="DOI URL">https://doi.org/10.2174/0929867326666190528080514</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.2174/0929867326666190528080514&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.2174%2F0929867326666190528080514%26sid%3Dliteratum%253Aachs%26jtitle%3DCurrent%2520Medicinal%2520Chemistry%26atitle%3DThe%252BDevelopment%252Bof%252BInhibitors%252BTargeting%252Bthe%252BMixed%252BLineage%252BLeukemia%252B1%252B%252528MLL1%252529-WD%252BRepeat%252BDomain%252B5%252BProtein%252B%252528WDR5%252529%252BProtein-%252BProtein%252BInteraction%26aulast%3DYe%26aufirst%3DXiaoqing%26date%3D2020%26volume%3D27%26issue%3D33%26spage%3D5530%26epage%3D5542" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Brian M  Linhares</span>, <span class="hlFld-ContribAuthor ">Jolanta  Grembecka</span>, <span class="hlFld-ContribAuthor ">Tomasz  Cierpicki</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting epigenetic protein–protein interactions with small-molecule inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Future Medicinal Chemistry</span><span> <strong>2020,</strong> <em>12 </em>
                                    (14)
                                     , 1305-1326. <a href="https://doi.org/10.4155/fmc-2020-0082" title="DOI URL">https://doi.org/10.4155/fmc-2020-0082</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.4155/fmc-2020-0082&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.4155%2Ffmc-2020-0082%26sid%3Dliteratum%253Aachs%26jtitle%3DFuture%2520Medicinal%2520Chemistry%26atitle%3DTargeting%252Bepigenetic%252Bprotein%2525E2%252580%252593protein%252Binteractions%252Bwith%252Bsmall-molecule%252Binhibitors%26aulast%3DLinhares%26aufirst%3DBrian%2BM%26date%3D2020%26volume%3D12%26issue%3D14%26spage%3D1305%26epage%3D1326" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Kangjie  Lin</span>, <span class="hlFld-ContribAuthor ">Youjun  Xu</span>, <span class="hlFld-ContribAuthor ">Jianfeng  Pei</span>, <span class="hlFld-ContribAuthor ">Luhua  Lai</span>. </span><span class="cited-content_cbyCitation_article-title">Automatic retrosynthetic route planning using template-free models. </span><span class="cited-content_cbyCitation_journal-name">Chemical Science</span><span> <strong>2020,</strong> <em>11 </em>
                                    (12)
                                     , 3355-3364. <a href="https://doi.org/10.1039/C9SC03666K" title="DOI URL">https://doi.org/10.1039/C9SC03666K</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/C9SC03666K&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FC9SC03666K%26sid%3Dliteratum%253Aachs%26jtitle%3DChemical%2520Science%26atitle%3DAutomatic%252Bretrosynthetic%252Broute%252Bplanning%252Busing%252Btemplate-free%252Bmodels%26aulast%3DLin%26aufirst%3DKangjie%26date%3D2020%26date%3D2020%26volume%3D11%26issue%3D12%26spage%3D3355%26epage%3D3364" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Wei-Lin  Chen</span>, <span class="hlFld-ContribAuthor ">Dong-Dong  Li</span>, <span class="hlFld-ContribAuthor ">Xin  Chen</span>, <span class="hlFld-ContribAuthor ">Ying-Zhe  Wang</span>, <span class="hlFld-ContribAuthor ">Jun-Jie  Xu</span>, <span class="hlFld-ContribAuthor ">Zheng-Yu  Jiang</span>, <span class="hlFld-ContribAuthor ">Qi-Dong  You</span>, <span class="hlFld-ContribAuthor ">Xiao-Ke  Guo</span>. </span><span class="cited-content_cbyCitation_article-title">Proton pump inhibitors selectively suppress MLL rearranged leukemia cells via disrupting MLL1-WDR5 protein-protein interaction. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>188 </em>, 112027. <a href="https://doi.org/10.1016/j.ejmech.2019.112027" title="DOI URL">https://doi.org/10.1016/j.ejmech.2019.112027</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2019.112027&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2019.112027%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DProton%252Bpump%252Binhibitors%252Bselectively%252Bsuppress%252BMLL%252Brearranged%252Bleukemia%252Bcells%252Bvia%252Bdisrupting%252BMLL1-WDR5%252Bprotein-protein%252Binteraction%26aulast%3DChen%26aufirst%3DWei-Lin%26date%3D2020%26volume%3D188%26spage%3D112027" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Fei  Ye</span>, <span class="hlFld-ContribAuthor ">Jing  Huang</span>, <span class="hlFld-ContribAuthor ">Hongbo  Wang</span>, <span class="hlFld-ContribAuthor ">Cheng  Luo</span>, <span class="hlFld-ContribAuthor ">Kehao  Zhao</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting epigenetic machinery: Emerging novel allosteric inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Pharmacology & Therapeutics</span><span> <strong>2019,</strong> <em>204 </em>, 107406. <a href="https://doi.org/10.1016/j.pharmthera.2019.107406" title="DOI URL">https://doi.org/10.1016/j.pharmthera.2019.107406</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.pharmthera.2019.107406&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.pharmthera.2019.107406%26sid%3Dliteratum%253Aachs%26jtitle%3DPharmacology%2520%2526%2520Therapeutics%26atitle%3DTargeting%252Bepigenetic%252Bmachinery%25253A%252BEmerging%252Bnovel%252Ballosteric%252Binhibitors%26aulast%3DYe%26aufirst%3DFei%26date%3D2019%26volume%3D204%26spage%3D107406" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Matthew L.  Dennis</span>, <span class="hlFld-ContribAuthor ">Benjamin J.  Morrow</span>, <span class="hlFld-ContribAuthor ">Olan  Dolezal</span>, <span class="hlFld-ContribAuthor ">Anthony N.  Cuzzupe</span>, <span class="hlFld-ContribAuthor ">Alexandra E.  Stupple</span>, <span class="hlFld-ContribAuthor ">Janet  Newman</span>, <span class="hlFld-ContribAuthor ">John  Bentley</span>, <span class="hlFld-ContribAuthor ">Meghan  Hattarki</span>, <span class="hlFld-ContribAuthor ">Stewart D.  Nuttall</span>, <span class="hlFld-ContribAuthor ">Richard C.  Foitzik</span>, <span class="hlFld-ContribAuthor ">Ian P.  Street</span>, <span class="hlFld-ContribAuthor ">Paul A.  Stupple</span>, <span class="hlFld-ContribAuthor ">Brendon J.  Monahan</span>, <span class="hlFld-ContribAuthor ">Thomas. S.  Peat</span>. </span><span class="cited-content_cbyCitation_article-title">Fragment screening for a protein-protein interaction inhibitor to WDR5. </span><span class="cited-content_cbyCitation_journal-name">Structural Dynamics</span><span> <strong>2019,</strong> <em>6 </em>
                                    (6)
                                     , 064701. <a href="https://doi.org/10.1063/1.5122849" title="DOI URL">https://doi.org/10.1063/1.5122849</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1063/1.5122849&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1063%2F1.5122849%26sid%3Dliteratum%253Aachs%26jtitle%3DStructural%2520Dynamics%26atitle%3DFragment%252Bscreening%252Bfor%252Ba%252Bprotein-protein%252Binteraction%252Binhibitor%252Bto%252BWDR5%26aulast%3DDennis%26aufirst%3DMatthew%2BL.%26date%3D2019%26volume%3D6%26issue%3D6%26spage%3D064701" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xiaoqing  Ye</span>, <span class="hlFld-ContribAuthor ">Rukang  Zhang</span>, <span class="hlFld-ContribAuthor ">Fulin  Lian</span>, <span class="hlFld-ContribAuthor ">Weiyao  Zhang</span>, <span class="hlFld-ContribAuthor ">Wenchao  Lu</span>, <span class="hlFld-ContribAuthor ">Jie  Han</span>, <span class="hlFld-ContribAuthor ">Naixia  Zhang</span>, <span class="hlFld-ContribAuthor ">Jia  Jin</span>, <span class="hlFld-ContribAuthor ">Cheng  Luo</span>, <span class="hlFld-ContribAuthor ">Kaixian  Chen</span>, <span class="hlFld-ContribAuthor ">Fei  Ye</span>, <span class="hlFld-ContribAuthor ">Hong  Ding</span>. </span><span class="cited-content_cbyCitation_article-title">The identification of novel small-molecule inhibitors targeting WDR5-MLL1 interaction through fluorescence polarization based high-throughput screening. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry Letters</span><span> <strong>2019,</strong> <em>29 </em>
                                    (4)
                                     , 638-645. <a href="https://doi.org/10.1016/j.bmcl.2018.12.035" title="DOI URL">https://doi.org/10.1016/j.bmcl.2018.12.035</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmcl.2018.12.035&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmcl.2018.12.035%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DThe%252Bidentification%252Bof%252Bnovel%252Bsmall-molecule%252Binhibitors%252Btargeting%252BWDR5-MLL1%252Binteraction%252Bthrough%252Bfluorescence%252Bpolarization%252Bbased%252Bhigh-throughput%252Bscreening%26aulast%3DYe%26aufirst%3DXiaoqing%26date%3D2019%26volume%3D29%26issue%3D4%26spage%3D638%26epage%3D645" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Gianluca  Sbardella</span>. </span><span class="cited-content_cbyCitation_article-title">Methyl-Readers and Inhibitors. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2019,</strong>,, 339-399. <a href="https://doi.org/10.1007/7355_2019_78" title="DOI URL">https://doi.org/10.1007/7355_2019_78</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/7355_2019_78&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2F7355_2019_78%26sid%3Dliteratum%253Aachs%26atitle%3DMethyl-Readers%252Band%252BInhibitors%26aulast%3DSbardella%26aufirst%3DGianluca%26date%3D2019%26date%3D2019%26spage%3D339%26epage%3D399%26pub%3DSpringer%2520International%2520Publishing%26atitle%3DChemical%252BEpigenetics%26aulast%3DMai%26aufirst%3DAntonello%26date%3D2020%26volume%3D33" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Tomoya  Hirano</span>, <span class="hlFld-ContribAuthor ">Shuichi  Mori</span>, <span class="hlFld-ContribAuthor ">Hiroyuki  Kagechika</span>. </span><span class="cited-content_cbyCitation_article-title">Recent Advances in Chemical Tools for the Regulation and Study of Protein Lysine Methyltransferases. </span><span class="cited-content_cbyCitation_journal-name">The Chemical Record</span><span> <strong>2018,</strong> <em>18 </em>
                                    (12)
                                     , 1745-1759. <a href="https://doi.org/10.1002/tcr.201800034" title="DOI URL">https://doi.org/10.1002/tcr.201800034</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/tcr.201800034&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Ftcr.201800034%26sid%3Dliteratum%253Aachs%26jtitle%3DThe%2520Chemical%2520Record%26atitle%3DRecent%252BAdvances%252Bin%252BChemical%252BTools%252Bfor%252Bthe%252BRegulation%252Band%252BStudy%252Bof%252BProtein%252BLysine%252BMethyltransferases%26aulast%3DHirano%26aufirst%3DTomoya%26date%3D2018%26date%3D2018%26volume%3D18%26issue%3D12%26spage%3D1745%26epage%3D1759" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Beth K.  Neilsen</span>, <span class="hlFld-ContribAuthor ">Binita  Chakraborty</span>, <span class="hlFld-ContribAuthor ">Jamie L.  McCall</span>, <span class="hlFld-ContribAuthor ">Danielle E.  Frodyma</span>, <span class="hlFld-ContribAuthor ">Richard L.  Sleightholm</span>, <span class="hlFld-ContribAuthor ">Kurt W.  Fisher</span>, <span class="hlFld-ContribAuthor ">Robert E.  Lewis</span>. </span><span class="cited-content_cbyCitation_article-title">WDR5 supports colon cancer cells by promoting methylation of H3K4 and suppressing DNA damage. </span><span class="cited-content_cbyCitation_journal-name">BMC Cancer</span><span> <strong>2018,</strong> <em>18 </em>
                                    (1)
                                     <a href="https://doi.org/10.1186/s12885-018-4580-6" title="DOI URL">https://doi.org/10.1186/s12885-018-4580-6</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1186/s12885-018-4580-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1186%2Fs12885-018-4580-6%26sid%3Dliteratum%253Aachs%26jtitle%3DBMC%2520Cancer%26atitle%3DWDR5%252Bsupports%252Bcolon%252Bcancer%252Bcells%252Bby%252Bpromoting%252Bmethylation%252Bof%252BH3K4%252Band%252Bsuppressing%252BDNA%252Bdamage%26aulast%3DNeilsen%26aufirst%3DBeth%2BK.%26date%3D2018%26date%3D2018%26volume%3D18%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Kebin  Lu</span>, <span class="hlFld-ContribAuthor ">He  Tao</span>, <span class="hlFld-ContribAuthor ">Xiaomin  Si</span>, <span class="hlFld-ContribAuthor ">Qingjuan  Chen</span>. </span><span class="cited-content_cbyCitation_article-title">The Histone H3 Lysine 4 Presenter WDR5 as an Oncogenic Protein and Novel Epigenetic Target in Cancer. </span><span class="cited-content_cbyCitation_journal-name">Frontiers in Oncology</span><span> <strong>2018,</strong> <em>8 </em><a href="https://doi.org/10.3389/fonc.2018.00502" title="DOI URL">https://doi.org/10.3389/fonc.2018.00502</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3389/fonc.2018.00502&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3389%2Ffonc.2018.00502%26sid%3Dliteratum%253Aachs%26jtitle%3DFrontiers%2520in%2520Oncology%26atitle%3DThe%252BHistone%252BH3%252BLysine%252B4%252BPresenter%252BWDR5%252Bas%252Ban%252BOncogenic%252BProtein%252Band%252BNovel%252BEpigenetic%252BTarget%252Bin%252BCancer%26aulast%3DLu%26aufirst%3DKebin%26date%3D2018%26date%3D2018%26volume%3D8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Guillermo  Senisterra</span>, <span class="hlFld-ContribAuthor ">Hugh Y.  Zhu</span>, <span class="hlFld-ContribAuthor ">Xiao  Luo</span>, <span class="hlFld-ContribAuthor ">Hailong  Zhang</span>, <span class="hlFld-ContribAuthor ">Guoliang  Xun</span>, <span class="hlFld-ContribAuthor ">Chunliang  Lu</span>, <span class="hlFld-ContribAuthor ">Wen  Xiao</span>, <span class="hlFld-ContribAuthor ">Taraneh  Hajian</span>, <span class="hlFld-ContribAuthor ">Peter  Loppnau</span>, <span class="hlFld-ContribAuthor ">Irene  Chau</span>, <span class="hlFld-ContribAuthor ">Fengling  Li</span>, <span class="hlFld-ContribAuthor ">Abdellah  Allali-Hassani</span>, <span class="hlFld-ContribAuthor ">Peter  Atadja</span>, <span class="hlFld-ContribAuthor ">Counde  Oyang</span>, <span class="hlFld-ContribAuthor ">En  Li</span>, <span class="hlFld-ContribAuthor ">Peter J.  Brown</span>, <span class="hlFld-ContribAuthor ">Cheryl H.  Arrowsmith</span>, <span class="hlFld-ContribAuthor ">Kehao  Zhao</span>, <span class="hlFld-ContribAuthor ">Zhengtian  Yu</span>, <span class="hlFld-ContribAuthor ">Masoud  Vedadi</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of Small-Molecule Antagonists of the H3K9me3 Binding to UHRF1 Tandem Tudor Domain. </span><span class="cited-content_cbyCitation_journal-name">SLAS DISCOVERY: Advancing the Science of Drug Discovery</span><span> <strong>2018,</strong> <em>23 </em>
                                    (9)
                                     , 930-940. <a href="https://doi.org/10.1177/2472555218766278" title="DOI URL">https://doi.org/10.1177/2472555218766278</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1177/2472555218766278&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1177%2F2472555218766278%26sid%3Dliteratum%253Aachs%26jtitle%3DSLAS%2520DISCOVERY%253A%2520Advancing%2520the%2520Science%2520of%2520Drug%2520Discovery%26atitle%3DDiscovery%252Bof%252BSmall-Molecule%252BAntagonists%252Bof%252Bthe%252BH3K9me3%252BBinding%252Bto%252BUHRF1%252BTandem%252BTudor%252BDomain%26aulast%3DSenisterra%26aufirst%3DGuillermo%26date%3D2018%26date%3D2018%26volume%3D23%26issue%3D9%26spage%3D930%26epage%3D940" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xiong  Zhang</span>, <span class="hlFld-ContribAuthor ">Xingling  Zheng</span>, <span class="hlFld-ContribAuthor ">Hong  Yang</span>, <span class="hlFld-ContribAuthor ">Juan  Yan</span>, <span class="hlFld-ContribAuthor ">Xuhong  Fu</span>, <span class="hlFld-ContribAuthor ">Rongrui  Wei</span>, <span class="hlFld-ContribAuthor ">Xiaowei  Xu</span>, <span class="hlFld-ContribAuthor ">Zhuqing  Zhang</span>, <span class="hlFld-ContribAuthor ">Aisong  Yu</span>, <span class="hlFld-ContribAuthor ">Kaixin  Zhou</span>, <span class="hlFld-ContribAuthor ">Jian  Ding</span>, <span class="hlFld-ContribAuthor ">Meiyu  Geng</span>, <span class="hlFld-ContribAuthor ">Xun  Huang</span>. </span><span class="cited-content_cbyCitation_article-title">Piribedil disrupts the MLL1-WDR5 interaction and sensitizes MLL-rearranged acute myeloid leukemia (AML) to doxorubicin-induced apoptosis. </span><span class="cited-content_cbyCitation_journal-name">Cancer Letters</span><span> <strong>2018,</strong> <em>431 </em>, 150-160. <a href="https://doi.org/10.1016/j.canlet.2018.05.034" title="DOI URL">https://doi.org/10.1016/j.canlet.2018.05.034</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.canlet.2018.05.034&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.canlet.2018.05.034%26sid%3Dliteratum%253Aachs%26jtitle%3DCancer%2520Letters%26atitle%3DPiribedil%252Bdisrupts%252Bthe%252BMLL1-WDR5%252Binteraction%252Band%252Bsensitizes%252BMLL-rearranged%252Bacute%252Bmyeloid%252Bleukemia%252B%252528AML%252529%252Bto%252Bdoxorubicin-induced%252Bapoptosis%26aulast%3DZhang%26aufirst%3DXiong%26date%3D2018%26volume%3D431%26spage%3D150%26epage%3D160" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ankit  Gupta</span>, <span class="hlFld-ContribAuthor ">Jing  Xu</span>, <span class="hlFld-ContribAuthor ">Shirley  Lee</span>, <span class="hlFld-ContribAuthor ">Steven T.  Tsai</span>, <span class="hlFld-ContribAuthor ">Bo  Zhou</span>, <span class="hlFld-ContribAuthor ">Kohei  Kurosawa</span>, <span class="hlFld-ContribAuthor ">Michael S.  Werner</span>, <span class="hlFld-ContribAuthor ">Akiko  Koide</span>, <span class="hlFld-ContribAuthor ">Alexander J.  Ruthenburg</span>, <span class="hlFld-ContribAuthor ">Yali  Dou</span>, <span class="hlFld-ContribAuthor ">Shohei  Koide</span>. </span><span class="cited-content_cbyCitation_article-title">Facile target validation in an animal model with intracellularly expressed monobodies. </span><span class="cited-content_cbyCitation_journal-name">Nature Chemical Biology</span><span> <strong>2018,</strong> <em>14 </em>
                                    (9)
                                     , 895-900. <a href="https://doi.org/10.1038/s41589-018-0099-z" title="DOI URL">https://doi.org/10.1038/s41589-018-0099-z</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41589-018-0099-z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41589-018-0099-z%26sid%3Dliteratum%253Aachs%26jtitle%3DNature%2520Chemical%2520Biology%26atitle%3DFacile%252Btarget%252Bvalidation%252Bin%252Ban%252Banimal%252Bmodel%252Bwith%252Bintracellularly%252Bexpressed%252Bmonobodies%26aulast%3DGupta%26aufirst%3DAnkit%26date%3D2018%26date%3D2018%26volume%3D14%26issue%3D9%26spage%3D895%26epage%3D900" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Wenchao  Lu</span>, <span class="hlFld-ContribAuthor ">Rukang  Zhang</span>, <span class="hlFld-ContribAuthor ">Hao  Jiang</span>, <span class="hlFld-ContribAuthor ">Huimin  Zhang</span>, <span class="hlFld-ContribAuthor ">Cheng  Luo</span>. </span><span class="cited-content_cbyCitation_article-title">Computer-Aided Drug Design in Epigenetics. </span><span class="cited-content_cbyCitation_journal-name">Frontiers in Chemistry</span><span> <strong>2018,</strong> <em>6 </em><a href="https://doi.org/10.3389/fchem.2018.00057" title="DOI URL">https://doi.org/10.3389/fchem.2018.00057</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3389/fchem.2018.00057&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3389%2Ffchem.2018.00057%26sid%3Dliteratum%253Aachs%26jtitle%3DFrontiers%2520in%2520Chemistry%26atitle%3DComputer-Aided%252BDrug%252BDesign%252Bin%252BEpigenetics%26aulast%3DLu%26aufirst%3DWenchao%26date%3D2018%26date%3D2018%26volume%3D6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Tomasz  Klucznik</span>, <span class="hlFld-ContribAuthor ">Barbara  Mikulak-Klucznik</span>, <span class="hlFld-ContribAuthor ">Michael P.  McCormack</span>, <span class="hlFld-ContribAuthor ">Heather  Lima</span>, <span class="hlFld-ContribAuthor ">Sara  Szymkuć</span>, <span class="hlFld-ContribAuthor ">Manishabrata  Bhowmick</span>, <span class="hlFld-ContribAuthor ">Karol  Molga</span>, <span class="hlFld-ContribAuthor ">Yubai  Zhou</span>, <span class="hlFld-ContribAuthor ">Lindsey  Rickershauser</span>, <span class="hlFld-ContribAuthor ">Ewa P.  Gajewska</span>, <span class="hlFld-ContribAuthor ">Alexei  Toutchkine</span>, <span class="hlFld-ContribAuthor ">Piotr  Dittwald</span>, <span class="hlFld-ContribAuthor ">Michał P.  Startek</span>, <span class="hlFld-ContribAuthor ">Gregory J.  Kirkovits</span>, <span class="hlFld-ContribAuthor ">Rafał  Roszak</span>, <span class="hlFld-ContribAuthor ">Ariel  Adamski</span>, <span class="hlFld-ContribAuthor ">Bianka  Sieredzińska</span>, <span class="hlFld-ContribAuthor ">Milan  Mrksich</span>, <span class="hlFld-ContribAuthor ">Sarah L.J.  Trice</span>, <span class="hlFld-ContribAuthor ">Bartosz A.  Grzybowski</span>. </span><span class="cited-content_cbyCitation_article-title">Efficient Syntheses of Diverse, Medicinally Relevant Targets Planned by Computer and Executed in the Laboratory. </span><span class="cited-content_cbyCitation_journal-name">Chem</span><span> <strong>2018,</strong> <em>4 </em>
                                    (3)
                                     , 522-532. <a href="https://doi.org/10.1016/j.chempr.2018.02.002" title="DOI URL">https://doi.org/10.1016/j.chempr.2018.02.002</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.chempr.2018.02.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.chempr.2018.02.002%26sid%3Dliteratum%253Aachs%26jtitle%3DChem%26atitle%3DEfficient%252BSyntheses%252Bof%252BDiverse%25252C%252BMedicinally%252BRelevant%252BTargets%252BPlanned%252Bby%252BComputer%252Band%252BExecuted%252Bin%252Bthe%252BLaboratory%26aulast%3DKlucznik%26aufirst%3DTomasz%26date%3D2018%26volume%3D4%26issue%3D3%26spage%3D522%26epage%3D532" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Wei-Lin  Chen</span>, <span class="hlFld-ContribAuthor ">Dong-Dong  Li</span>, <span class="hlFld-ContribAuthor ">Zhi-Hui  Wang</span>, <span class="hlFld-ContribAuthor ">Xiao-Li  Xu</span>, <span class="hlFld-ContribAuthor ">Xiao-Jin  Zhang</span>, <span class="hlFld-ContribAuthor ">Zheng-Yu  Jiang</span>, <span class="hlFld-ContribAuthor ">Xiao-Ke  Guo</span>, <span class="hlFld-ContribAuthor ">Qi-Dong  You</span>. </span><span class="cited-content_cbyCitation_article-title">Design, synthesis, and initial evaluation of affinity-based small molecular probe for detection of WDR5. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic Chemistry</span><span> <strong>2018,</strong> <em>76 </em>, 380-385. <a href="https://doi.org/10.1016/j.bioorg.2017.11.018" title="DOI URL">https://doi.org/10.1016/j.bioorg.2017.11.018</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bioorg.2017.11.018&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bioorg.2017.11.018%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520Chemistry%26atitle%3DDesign%25252C%252Bsynthesis%25252C%252Band%252Binitial%252Bevaluation%252Bof%252Baffinity-based%252Bsmall%252Bmolecular%252Bprobe%252Bfor%252Bdetection%252Bof%252BWDR5%26aulast%3DChen%26aufirst%3DWei-Lin%26date%3D2018%26volume%3D76%26spage%3D380%26epage%3D385" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Liyan  Yue</span>, <span class="hlFld-ContribAuthor ">Wei  Wan</span>, <span class="hlFld-ContribAuthor ">Pan  Xu</span>, <span class="hlFld-ContribAuthor ">Linjuan  Li</span>, <span class="hlFld-ContribAuthor ">Chen  Wang</span>, <span class="hlFld-ContribAuthor ">Yuanyuan  Zhang</span>, <span class="hlFld-ContribAuthor ">Heng  Xu</span>, <span class="hlFld-ContribAuthor ">Rukang  Zhang</span>, <span class="hlFld-ContribAuthor ">Junchi  Hu</span>, <span class="hlFld-ContribAuthor ">Wenchao  Lu</span>, <span class="hlFld-ContribAuthor ">Hao  Jiang</span>, <span class="hlFld-ContribAuthor ">Cheng  Luo</span>. </span><span class="cited-content_cbyCitation_article-title">Small Molecule Inhibitors Targeting New Targets of Protein-Protein Interactions. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2018,</strong>,, 179-211. <a href="https://doi.org/10.1007/978-981-13-0773-7_7" title="DOI URL">https://doi.org/10.1007/978-981-13-0773-7_7</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/978-981-13-0773-7_7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2F978-981-13-0773-7_7%26sid%3Dliteratum%253Aachs%26atitle%3DSmall%252BMolecule%252BInhibitors%252BTargeting%252BNew%252BTargets%252Bof%252BProtein-Protein%252BInteractions%26aulast%3DYue%26aufirst%3DLiyan%26date%3D2018%26date%3D2018%26spage%3D179%26epage%3D211%26pub%3DSpringer%2520Singapore%26atitle%3DTargeting%252BProtein-Protein%252BInteractions%252Bby%252BSmall%252BMolecules%26aulast%3DSheng%26aufirst%3DChunquan%26date%3D2018%26volume%3D15" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Zhi-Hui  Wang</span>, <span class="hlFld-ContribAuthor ">Dong-Dong  Li</span>, <span class="hlFld-ContribAuthor ">Wei-Lin  Chen</span>, <span class="hlFld-ContribAuthor ">Qi-Dong  You</span>, <span class="hlFld-ContribAuthor ">Xiao-Ke  Guo</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting protein-protein interaction between MLL1 and reciprocal proteins for leukemia therapy. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry</span><span> <strong>2018,</strong> <em>26 </em>
                                    (2)
                                     , 356-365. <a href="https://doi.org/10.1016/j.bmc.2017.11.045" title="DOI URL">https://doi.org/10.1016/j.bmc.2017.11.045</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmc.2017.11.045&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmc.2017.11.045%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%26atitle%3DTargeting%252Bprotein-protein%252Binteraction%252Bbetween%252BMLL1%252Band%252Breciprocal%252Bproteins%252Bfor%252Bleukemia%252Btherapy%26aulast%3DWang%26aufirst%3DZhi-Hui%26date%3D2018%26volume%3D26%26issue%3D2%26spage%3D356%26epage%3D365" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Matthieu  Schapira</span>, <span class="hlFld-ContribAuthor ">Mike  Tyers</span>, <span class="hlFld-ContribAuthor ">Maricel  Torrent</span>, <span class="hlFld-ContribAuthor ">Cheryl H.  Arrowsmith</span>. </span><span class="cited-content_cbyCitation_article-title">WD40 repeat domain proteins: a novel target class?. </span><span class="cited-content_cbyCitation_journal-name">Nature Reviews Drug Discovery</span><span> <strong>2017,</strong> <em>16 </em>
                                    (11)
                                     , 773-786. <a href="https://doi.org/10.1038/nrd.2017.179" title="DOI URL">https://doi.org/10.1038/nrd.2017.179</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/nrd.2017.179&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fnrd.2017.179%26sid%3Dliteratum%253Aachs%26jtitle%3DNature%2520Reviews%2520Drug%2520Discovery%26atitle%3DWD40%252Brepeat%252Bdomain%252Bproteins%25253A%252Ba%252Bnovel%252Btarget%252Bclass%25253F%26aulast%3DSchapira%26aufirst%3DMatthieu%26date%3D2017%26date%3D2017%26volume%3D16%26issue%3D11%26spage%3D773%26epage%3D786" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jong-Hee  Lee</span>, <span class="hlFld-ContribAuthor ">Kyle R.  Salci</span>, <span class="hlFld-ContribAuthor ">Jennifer C.  Reid</span>, <span class="hlFld-ContribAuthor ">Luca  Orlando</span>, <span class="hlFld-ContribAuthor ">Borko  Tanasijevic</span>, <span class="hlFld-ContribAuthor ">Zoya  Shapovalova</span>, <span class="hlFld-ContribAuthor ">Mickie  Bhatia</span>. </span><span class="cited-content_cbyCitation_article-title">Brief Report: Human Acute Myeloid Leukemia Reprogramming to Pluripotency Is a Rare Event and Selects for Patient Hematopoietic Cells Devoid of Leukemic Mutations. </span><span class="cited-content_cbyCitation_journal-name">STEM CELLS</span><span> <strong>2017,</strong> <em>35 </em>
                                    (9)
                                     , 2095-2102. <a href="https://doi.org/10.1002/stem.2655" title="DOI URL">https://doi.org/10.1002/stem.2655</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/stem.2655&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fstem.2655%26sid%3Dliteratum%253Aachs%26jtitle%3DSTEM%2520CELLS%26atitle%3DBrief%252BReport%25253A%252BHuman%252BAcute%252BMyeloid%252BLeukemia%252BReprogramming%252Bto%252BPluripotency%252BIs%252Ba%252BRare%252BEvent%252Band%252BSelects%252Bfor%252BPatient%252BHematopoietic%252BCells%252BDevoid%252Bof%252BLeukemic%252BMutations%26aulast%3DLee%26aufirst%3DJong-Hee%26date%3D2017%26date%3D2017%26volume%3D35%26issue%3D9%26spage%3D2095%26epage%3D2102" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">H. Ümit  Kaniskan</span>, <span class="hlFld-ContribAuthor ">Jian  Jin</span>. </span><span class="cited-content_cbyCitation_article-title">Recent progress in developing selective inhibitors of protein methyltransferases. </span><span class="cited-content_cbyCitation_journal-name">Current Opinion in Chemical Biology</span><span> <strong>2017,</strong> <em>39 </em>, 100-108. <a href="https://doi.org/10.1016/j.cbpa.2017.06.013" title="DOI URL">https://doi.org/10.1016/j.cbpa.2017.06.013</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.cbpa.2017.06.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.cbpa.2017.06.013%26sid%3Dliteratum%253Aachs%26jtitle%3DCurrent%2520Opinion%2520in%2520Chemical%2520Biology%26atitle%3DRecent%252Bprogress%252Bin%252Bdeveloping%252Bselective%252Binhibitors%252Bof%252Bprotein%252Bmethyltransferases%26aulast%3DKaniskan%26aufirst%3DH.%2B%25C3%259Cmit%26date%3D2017%26volume%3D39%26spage%3D100%26epage%3D108" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Dong-Dong  Li</span>, <span class="hlFld-ContribAuthor ">Zhi-Hui  Wang</span>, <span class="hlFld-ContribAuthor ">Wei-Lin  Chen</span>, <span class="hlFld-ContribAuthor ">Yi-Yue  Xie</span>, <span class="hlFld-ContribAuthor ">Qi-Dong  You</span>, <span class="hlFld-ContribAuthor ">Xiao-Ke  Guo</span>. </span><span class="cited-content_cbyCitation_article-title">Structure-based design of ester compounds to inhibit MLL complex catalytic activity by targeting mixed lineage leukemia 1 (MLL1)–WDR5 interaction. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry</span><span> <strong>2016,</strong> <em>24 </em>
                                    (22)
                                     , 6109-6118. <a href="https://doi.org/10.1016/j.bmc.2016.09.073" title="DOI URL">https://doi.org/10.1016/j.bmc.2016.09.073</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmc.2016.09.073&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmc.2016.09.073%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%26atitle%3DStructure-based%252Bdesign%252Bof%252Bester%252Bcompounds%252Bto%252Binhibit%252BMLL%252Bcomplex%252Bcatalytic%252Bactivity%252Bby%252Btargeting%252Bmixed%252Blineage%252Bleukemia%252B1%252B%252528MLL1%252529%2525E2%252580%252593WDR5%252Binteraction%26aulast%3DLi%26aufirst%3DDong-Dong%26date%3D2016%26volume%3D24%26issue%3D22%26spage%3D6109%26epage%3D6118" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Dong-Dong  Li</span>, <span class="hlFld-ContribAuthor ">Wei-Lin  Chen</span>, <span class="hlFld-ContribAuthor ">Zhi-Hui  Wang</span>, <span class="hlFld-ContribAuthor ">Yi-Yue  Xie</span>, <span class="hlFld-ContribAuthor ">Xiao-Li  Xu</span>, <span class="hlFld-ContribAuthor ">Zheng-Yu  Jiang</span>, <span class="hlFld-ContribAuthor ">Xiao-Jin  Zhang</span>, <span class="hlFld-ContribAuthor ">Qi-Dong  You</span>, <span class="hlFld-ContribAuthor ">Xiao-Ke  Guo</span>. </span><span class="cited-content_cbyCitation_article-title">High-affinity small molecular blockers of mixed lineage leukemia 1 (MLL1)-WDR5 interaction inhibit MLL1 complex H3K4 methyltransferase activity. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2016,</strong> <em>124 </em>, 480-489. <a href="https://doi.org/10.1016/j.ejmech.2016.08.036" title="DOI URL">https://doi.org/10.1016/j.ejmech.2016.08.036</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2016.08.036&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2016.08.036%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DHigh-affinity%252Bsmall%252Bmolecular%252Bblockers%252Bof%252Bmixed%252Blineage%252Bleukemia%252B1%252B%252528MLL1%252529-WDR5%252Binteraction%252Binhibit%252BMLL1%252Bcomplex%252BH3K4%252Bmethyltransferase%252Bactivity%26aulast%3DLi%26aufirst%3DDong-Dong%26date%3D2016%26volume%3D124%26spage%3D480%26epage%3D489" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Nilda L.  Alicea-Velázquez</span>, <span class="hlFld-ContribAuthor ">Stephen A.  Shinsky</span>, <span class="hlFld-ContribAuthor ">Daniel M.  Loh</span>, <span class="hlFld-ContribAuthor ">Jeong-Heon  Lee</span>, <span class="hlFld-ContribAuthor ">David G.  Skalnik</span>, <span class="hlFld-ContribAuthor ">Michael S.  Cosgrove</span>. </span><span class="cited-content_cbyCitation_article-title">Targeted Disruption of the Interaction between WD-40 Repeat Protein 5 (WDR5) and Mixed Lineage Leukemia (MLL)/SET1 Family Proteins Specifically Inhibits MLL1 and SETd1A Methyltransferase Complexes. </span><span class="cited-content_cbyCitation_journal-name">Journal of Biological Chemistry</span><span> <strong>2016,</strong> <em>291 </em>
                                    (43)
                                     , 22357-22372. <a href="https://doi.org/10.1074/jbc.M116.752626" title="DOI URL">https://doi.org/10.1074/jbc.M116.752626</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1074/jbc.M116.752626&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1074%2Fjbc.M116.752626%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Biological%2520Chemistry%26atitle%3DTargeted%252BDisruption%252Bof%252Bthe%252BInteraction%252Bbetween%252BWD-40%252BRepeat%252BProtein%252B5%252B%252528WDR5%252529%252Band%252BMixed%252BLineage%252BLeukemia%252B%252528MLL%252529%25252FSET1%252BFamily%252BProteins%252BSpecifically%252BInhibits%252BMLL1%252Band%252BSETd1A%252BMethyltransferase%252BComplexes%26aulast%3DAlicea-Vel%25C3%25A1zquez%26aufirst%3DNilda%2BL.%26date%3D2016%26volume%3D291%26issue%3D43%26spage%3D22357%26epage%3D22372" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="f_tgr1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01630/20160318/images/medium/jm-2015-01630k_0016.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01630/20160318/images/large/jm-2015-01630k_0016.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01630&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01630/20160318/images/medium/jm-2015-01630k_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01630/20160318/images/large/jm-2015-01630k_0001.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Chemical structures of reported WD repeat domain antagonists. (A) Racemic 1,1′-binaphthyl-2,2′-dicarboxylic acid, an inhibitor of yeast Cdc4. (B) Acyclic and cyclic peptide inhibitors that occupy the arginine binding site of WDR5.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01630/20160318/images/large/jm-2015-01630k_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01630&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01630/20160318/images/medium/jm-2015-01630k_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01630/20160318/images/large/jm-2015-01630k_0002.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Chemical structures and binding mode of WDR5-MLL1 interaction antagonists. (A) Initial medium-throughput screening hit compound <b>4</b>. (B) Advanced antagonist <b>5</b> obtained following an initial round of SAR studies. (C) Crystal structure of WDR5 (gray) in complex with <b>5</b> (green; PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4IA9">4IA9</a>). The antagonist resides in the MLL1 arginine-binding pocket and forms a direct H-bond to the side chain of S91 as well as water mediated H-bonds to the backbone of C261. Substituents on the benzamide add hydrophobic interactions with aliphatic side chains surrounding a hydrophobic subpocket.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01630/20160318/images/large/jm-2015-01630k_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01630&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01630/20160318/images/medium/jm-2015-01630k_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01630/20160318/images/large/jm-2015-01630k_0005.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of 5-Substituted 2-(4-Methylpiperazin-1-yl)phenyl)-3-methylbenzamides<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01630/20160318/images/large/jm-2015-01630k_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01630&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) 1-methylpiperazine, DIPEA, DMSO, 80 °C; (b) Fe<sub>(s)</sub>, CaCl<sub>2</sub>, water/MeOH (1:1), 100 °C; (c) 3-methylbenzoyl chloride, pyridine, CH<sub>2</sub>Cl<sub>2</sub>, rt; (d) boronic acid/ester, Pd(PPh<sub>3</sub>)<sub>4</sub>, Na<sub>2</sub>CO<sub>3</sub>, toluene/water (1:1), 115 °C.</p></p></figure><figure data-id="fig3" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01630/20160318/images/medium/jm-2015-01630k_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01630/20160318/images/large/jm-2015-01630k_0003.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Structural alignment of WDR5 crystal structures in complex with <b>4</b> (gray), <b>9d</b> (green), <b>9e</b> (cyan), <b>9h</b> (pink), and <b>9o</b> (yellow). The inhibitors occupy the MLL1 binding site and form water-mediated H-bonds to residue C261 as well as a direct H-bond to S91. The phenyl rings of the heterocyclic substituents attached at the C-5 position show a high degree of alignment.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01630/20160318/images/large/jm-2015-01630k_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01630&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch2" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01630/20160318/images/medium/jm-2015-01630k_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01630/20160318/images/large/jm-2015-01630k_0006.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of <i>N</i>-(2-(4-Methylpiperazin-1-yl)-5-nitrophenyl) Benzamides<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01630/20160318/images/large/jm-2015-01630k_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01630&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) (i) acid chloride, pyridine, DCM, rt; or (ii) benzoic acid, SOCl<sub>2</sub>, toluene, 70 °C, then amine <b>10</b>, pyridine, DCM, rt; or (iii) benzoic acid, POCl<sub>3</sub>, toluene, 70 °C, then amine <b>10</b>, pyridine, DCM, rt; (b) 1-methylpiperazine, DIPEA, DMF, 80 °C.</p></p></figure><figure data-id="sch3" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01630/20160318/images/medium/jm-2015-01630k_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01630/20160318/images/large/jm-2015-01630k_0007.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Synthesis of <i>N</i>-(2-(4-Methylpiperazin-1-yl)-5-nitrophenyl) Amides<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01630/20160318/images/large/jm-2015-01630k_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01630&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) 1-methylpiperazine, DIPEA, 80 °C; (b) RCO<sub>2</sub>H, POCl<sub>3</sub>, cat. DMAP, xylene, 120 °C.</p></p></figure><figure data-id="sch4" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01630/20160318/images/medium/jm-2015-01630k_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01630/20160318/images/large/jm-2015-01630k_0008.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 4. Synthesis of 5-Substituted <i>N</i>-(2-(4-Methylpiperazin-1-yl)phenyl)-6-oxo-4-(trifluoromethyl)-1,6-dihydropyridine-3-carboxamides<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01630/20160318/images/large/jm-2015-01630k_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01630&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) 1-methylpiperazine, DIPEA, DMSO, 80 °C; (b) Fe<sub>(s)</sub>, CaCl<sub>2</sub>, water/MeOH (1:1), 100 °C; (c) 6-oxo-4-(trifluoromethyl)-1,6-dihydropyridine-3-carboxylic acid, SOCl<sub>2</sub>, 80 °C, then aniline <b>7</b>, DCM, pyridine, rt; (d) boronic acid/ester, Xphos Pd G2, Na<sub>2</sub>CO<sub>3</sub>, 1,4-dioxane/water (5:3), 120 °C.</p></p></figure><figure data-id="fig4" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01630/20160318/images/medium/jm-2015-01630k_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01630/20160318/images/large/jm-2015-01630k_0004.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. X-ray crystal structure of the WDR5-<b>16d</b> complex (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4QL1">4QL1</a>). Compound <b>16d</b> (green) binds in the arginine binding site of WDR5 (gray). The experimental electron density (purple mesh) of the ligand is shown at a 1.5 Å resolution (2<i>F</i><sub>o</sub> – <i>F</i><sub>c</sub> map contoured at 1.5 σ). In addition to the binding interactions observed in previous derivatives, the nitrogen of the newly introduced pyridinone substituent forms a direct H-bond to the D107 side chain. A water-mediated H-bond between the pyridinone oxygen and the A65 carbonyl further stabilizes binding of the ligand. The pendant morpholino moiety points into the bulk solvent and does not interact with the protein. As a result of this solvent exposure, the electron density of the morpholine ring is not well-defined.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-6/acs.jmedchem.5b01630/20160318/images/large/jm-2015-01630k_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01630&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i101">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_57620" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_57620" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 24 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group">Wu, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shu, H.-B.</span><span> </span><span class="NLM_article-title">MLL1/WDR5 complex in leukemogenesis and epigenetic regulation</span> <span class="citation_source-journal">Aizheng</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">30</span><span class="NLM_x">, </span> <span class="NLM_fpage">240</span><span class="NLM_x">–</span> <span class="NLM_lpage">246</span><span class="refDoi"> DOI: 10.5732/cjc.011.10055</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01630&amp;key=10.5732%2Fcjc.011.10055" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01630&amp;key=21439245" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2011&pages=240-246&author=M.+Wuauthor=H.-B.+Shu&title=MLL1%2FWDR5+complex+in+leukemogenesis+and+epigenetic+regulation&doi=10.5732%2Fcjc.011.10055"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.5732%2Fcjc.011.10055&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.5732%252Fcjc.011.10055%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DM.%26aulast%3DShu%26aufirst%3DH.-B.%26atitle%3DMLL1%252FWDR5%2520complex%2520in%2520leukemogenesis%2520and%2520epigenetic%2520regulation%26jtitle%3DAizheng%26date%3D2011%26volume%3D30%26spage%3D240%26epage%3D246%26doi%3D10.5732%2Fcjc.011.10055" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group">Slany, R. K.</span><span> </span><span class="NLM_article-title">The molecular biology of mixed-lineage leukemia</span> <span class="citation_source-journal">Haematologica</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">94</span><span class="NLM_x">, </span> <span class="NLM_fpage">984</span><span class="NLM_x">–</span> <span class="NLM_lpage">993</span><span class="refDoi"> DOI: 10.3324/haematol.2008.002436</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01630&amp;key=10.3324%2Fhaematol.2008.002436" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=94&publication_year=2009&pages=984-993&author=R.+K.+Slany&title=The+molecular+biology+of+mixed-lineage+leukemia&doi=10.3324%2Fhaematol.2008.002436"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.3324%2Fhaematol.2008.002436&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3324%252Fhaematol.2008.002436%26sid%3Dliteratum%253Aachs%26aulast%3DSlany%26aufirst%3DR.%2BK.%26atitle%3DThe%2520molecular%2520biology%2520of%2520mixed-lineage%2520leukemia%26jtitle%3DHaematologica%26date%3D2009%26volume%3D94%26spage%3D984%26epage%3D993%26doi%3D10.3324%2Fhaematol.2008.002436" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group">Karatas, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Townsend, E. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cao, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bernard, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lei, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dou, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, S.</span><span> </span><span class="NLM_article-title">High-affinity, small-molecule peptidomimetic inhibitors of MLL1/WDR5 protein-protein interaction</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">135</span><span class="NLM_x">, </span> <span class="NLM_fpage">669</span><span class="NLM_x">–</span> <span class="NLM_lpage">682</span><span class="refDoi"> DOI: 10.1021/ja306028q</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja306028q" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=135&publication_year=2013&pages=669-682&author=H.+Karatasauthor=E.+C.+Townsendauthor=F.+Caoauthor=Y.+Chenauthor=D.+Bernardauthor=L.+Liuauthor=M.+Leiauthor=Y.+Douauthor=S.+Wang&title=High-affinity%2C+small-molecule+peptidomimetic+inhibitors+of+MLL1%2FWDR5+protein-protein+interaction&doi=10.1021%2Fja306028q"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1021%2Fja306028q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja306028q%26sid%3Dliteratum%253Aachs%26aulast%3DKaratas%26aufirst%3DH.%26aulast%3DTownsend%26aufirst%3DE.%2BC.%26aulast%3DCao%26aufirst%3DF.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DBernard%26aufirst%3DD.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DLei%26aufirst%3DM.%26aulast%3DDou%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DHigh-affinity%252C%2520small-molecule%2520peptidomimetic%2520inhibitors%2520of%2520MLL1%252FWDR5%2520protein-protein%2520interaction%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2013%26volume%3D135%26spage%3D669%26epage%3D682%26doi%3D10.1021%2Fja306028q" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group">Bissinger, E.-M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heinke, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sippl, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jung, M.</span><span> </span><span class="NLM_article-title">Targeting epigenetic modifiers: inhibitors of histone methyltransferases</span> <span class="citation_source-journal">MedChemComm</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">1</span><span class="NLM_x">, </span> <span class="NLM_fpage">114</span><span class="NLM_x">–</span> <span class="NLM_lpage">124</span><span class="refDoi"> DOI: 10.1039/c0md00062k</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01630&amp;key=10.1039%2Fc0md00062k" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2010&pages=114-124&author=E.-M.+Bissingerauthor=R.+Heinkeauthor=W.+Sipplauthor=M.+Jung&title=Targeting+epigenetic+modifiers%3A+inhibitors+of+histone+methyltransferases&doi=10.1039%2Fc0md00062k"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1039%2Fc0md00062k&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc0md00062k%26sid%3Dliteratum%253Aachs%26aulast%3DBissinger%26aufirst%3DE.-M.%26aulast%3DHeinke%26aufirst%3DR.%26aulast%3DSippl%26aufirst%3DW.%26aulast%3DJung%26aufirst%3DM.%26atitle%3DTargeting%2520epigenetic%2520modifiers%253A%2520inhibitors%2520of%2520histone%2520methyltransferases%26jtitle%3DMedChemComm%26date%3D2010%26volume%3D1%26spage%3D114%26epage%3D124%26doi%3D10.1039%2Fc0md00062k" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group">Li, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roberts, R.</span><span> </span><span class="NLM_article-title">Human Genome and Diseases. WD-repeat proteins: structure characteristics, biological function, and their involvement in human diseases</span> <span class="citation_source-journal">Cell. Mol. Life Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">58</span><span class="NLM_x">, </span> <span class="NLM_fpage">2085</span><span class="NLM_x">–</span> <span class="NLM_lpage">2097</span><span class="refDoi"> DOI: 10.1007/PL00000838</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01630&amp;key=10.1007%2FPL00000838" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01630&amp;key=11814058" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01630&amp;key=1%3ACAS%3A528%3ADC%252BD38XhtFakug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2001&pages=2085-2097&author=D.+Liauthor=R.+Roberts&title=Human+Genome+and+Diseases.+WD-repeat+proteins%3A+structure+characteristics%2C+biological+function%2C+and+their+involvement+in+human+diseases&doi=10.1007%2FPL00000838"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">WD-repeat proteins: structure characteristics, biological function, and their involvement in human diseases</span></div><div class="casAuthors">Li, D.; Roberts, R.</div><div class="citationInfo"><span class="NLM_cas:title">Cellular and Molecular Life Sciences</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">2085-2097</span>CODEN:
                <span class="NLM_cas:coden">CMLSFI</span>;
        ISSN:<span class="NLM_cas:issn">1420-682X</span>.
    
            (<span class="NLM_cas:orgname">Birkhaeuser Verlag</span>)
        </div><div class="casAbstract">A review.  Defined by the presence of four or more repeating units contg. a conserved core of approx. 40 amino acids that usually ending with tryptophan-aspartic acid (WD), WD-repeat proteins belong to a large and fast-expanding conservative protein family.  As demonstrated by the crystal structure of the G protein β subunit, all WD-repeat proteins are speculated to form a circularized β propeller structure.  The importance of these proteins is not only demonstrated by their crit. roles in many essential biol. functions ranging from signal transduction, transcription regulation, to apoptosis, but is also recognized by their assocn. with several human diseases.  Defining the function of a WD-repeat protein is the current challenge.  It is however, paramount to uncover the function of individual WD-repeat proteins, explore the protein interaction mechanism through WD-repeat domains and, ultimately, understand the complex biol. processes and organisms themselves.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGphQmbI-2-bprVg90H21EOLACvtfcHk0lgkHI7pwhYU_w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XhtFakug%253D%253D&md5=651e8fb04b017537c5f26f9b19a0b45c</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1007%2FPL00000838&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252FPL00000838%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DD.%26aulast%3DRoberts%26aufirst%3DR.%26atitle%3DHuman%2520Genome%2520and%2520Diseases.%2520WD-repeat%2520proteins%253A%2520structure%2520characteristics%252C%2520biological%2520function%252C%2520and%2520their%2520involvement%2520in%2520human%2520diseases%26jtitle%3DCell.%2520Mol.%2520Life%2520Sci.%26date%3D2001%26volume%3D58%26spage%3D2085%26epage%3D2097%26doi%3D10.1007%2FPL00000838" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group">Patel, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dharmarajan, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cosgrove, M. S.</span><span> </span><span class="NLM_article-title">Structure of WDR5 bound to mixed lineage leukemia protein-1 peptide</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">283</span><span class="NLM_x">, </span> <span class="NLM_fpage">32158</span><span class="NLM_x">–</span> <span class="NLM_lpage">32161</span><span class="refDoi"> DOI: 10.1074/jbc.C800164200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01630&amp;key=10.1074%2Fjbc.C800164200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01630&amp;key=18829459" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01630&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtlOjtL7M" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=283&publication_year=2008&pages=32158-32161&author=A.+Patelauthor=V.+Dharmarajanauthor=M.+S.+Cosgrove&title=Structure+of+WDR5+bound+to+mixed+lineage+leukemia+protein-1+peptide&doi=10.1074%2Fjbc.C800164200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Structure of WDR5 Bound to Mixed Lineage Leukemia Protein-1 Peptide</span></div><div class="casAuthors">Patel, Anamika; Dharmarajan, Venkatasubramanian; Cosgrove, Michael S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">283</span>
        (<span class="NLM_cas:issue">47</span>),
    <span class="NLM_cas:pages">32158-32161</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">The mixed lineage leukemia protein-1 (MLL1) catalyzes histone H3 lysine 4 methylation and is regulated by interaction with WDR5 (WD-repeat protein-5), RbBP5 (retinoblastoma-binding protein-5), and the Ash2L (absent, small, homeotic disks-2-like) oncoprotein.  In the accompanying investigation, we describe the identification of a conserved arginine contg. motif, called the "Win" or WDR5 interaction motif, that is essential for the assembly and H3K4 dimethylation activity of the MLL1 core complex.  Here we present a 1.7-Å crystal structure of WDR5 bound to a peptide derived from the MLL1 Win motif.  Our results show that Arg-3765 of MLL1 is bound in the same arginine binding pocket on WDR5 that was previously suggested to bind histone H3.  Thermodn. binding expts. show that the MLL1 Win peptide is preferentially recognized by WDR5.  These results are consistent with a model in which WDR5 recognizes Arg-3765 of MLL1, which is essential for the assembly and enzymic activity of the MLL1 core complex.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq2hQy6dZIN-LVg90H21EOLACvtfcHk0lgkHI7pwhYU_w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtlOjtL7M&md5=770e511a13cf8ada19b37e0cb7115eaf</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1074%2Fjbc.C800164200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.C800164200%26sid%3Dliteratum%253Aachs%26aulast%3DPatel%26aufirst%3DA.%26aulast%3DDharmarajan%26aufirst%3DV.%26aulast%3DCosgrove%26aufirst%3DM.%2BS.%26atitle%3DStructure%2520of%2520WDR5%2520bound%2520to%2520mixed%2520lineage%2520leukemia%2520protein-1%2520peptide%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2008%26volume%3D283%26spage%3D32158%26epage%3D32161%26doi%3D10.1074%2Fjbc.C800164200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group">Patel, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vought, V. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dharmarajan, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cosgrove, M. S.</span><span> </span><span class="NLM_article-title">A conserved arginine-containing motif crucial for the assembly and enzymatic activity of the mixed lineage leukemia protein-1 core complex</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">283</span><span class="NLM_x">, </span> <span class="NLM_fpage">32162</span><span class="NLM_x">–</span> <span class="NLM_lpage">32175</span><span class="refDoi"> DOI: 10.1074/jbc.M806317200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01630&amp;key=10.1074%2Fjbc.M806317200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01630&amp;key=18829457" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01630&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtlOjtL7N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=283&publication_year=2008&pages=32162-32175&author=A.+Patelauthor=V.+E.+Voughtauthor=V.+Dharmarajanauthor=M.+S.+Cosgrove&title=A+conserved+arginine-containing+motif+crucial+for+the+assembly+and+enzymatic+activity+of+the+mixed+lineage+leukemia+protein-1+core+complex&doi=10.1074%2Fjbc.M806317200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">A Conserved Arginine-containing Motif Crucial for the Assembly and Enzymatic Activity of the Mixed Lineage Leukemia Protein-1 Core Complex</span></div><div class="casAuthors">Patel, Anamika; Vought, Valarie E.; Dharmarajan, Venkatasubramanian; Cosgrove, Michael S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">283</span>
        (<span class="NLM_cas:issue">47</span>),
    <span class="NLM_cas:pages">32162-32175</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">The mixed lineage leukemia protein-1 (MLL1) belongs to the SET1 family of histone H3 lysine 4 methyltransferases.  Recent studies indicate that the catalytic subunits of SET1 family members are regulated by interaction with a conserved core group of proteins that include the WD repeat protein-5 (WDR5), retinoblastoma-binding protein-5 (RbBP5), and the absent small homeotic-2-like protein (Ash2L).  It has been suggested that WDR5 functions to bridge the interactions between the catalytic and regulatory subunits of SET1 family complexes.  However, the mol. details of these interactions are unknown.  To gain insight into the interactions among these proteins, we have detd. the biophys. basis for the interaction between the human WDR5 and MLL1.  Our studies reveal that WDR5 preferentially recognizes a previously unidentified and conserved arginine-contg. motif, called the "Win" or WDR5 interaction motif, which is located in the N-SET region of MLL1 and other SET1 family members.  Surprisingly, our structural and functional studies show that WDR5 recognizes arginine 3765 of the MLL1 Win motif using the same arginine-binding pocket on WDR5 that was previously shown to bind histone H3.  We demonstrate that WDR5's recognition of arginine 3765 of MLL1 is essential for the assembly and enzymic activity of the MLL1 core complex in vitro.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqSRgLh_lnhMbVg90H21EOLACvtfcHk0ljojHJnc1oopw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtlOjtL7N&md5=0d2e92e9ab6d549b4d7e175aa7957335</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M806317200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M806317200%26sid%3Dliteratum%253Aachs%26aulast%3DPatel%26aufirst%3DA.%26aulast%3DVought%26aufirst%3DV.%2BE.%26aulast%3DDharmarajan%26aufirst%3DV.%26aulast%3DCosgrove%26aufirst%3DM.%2BS.%26atitle%3DA%2520conserved%2520arginine-containing%2520motif%2520crucial%2520for%2520the%2520assembly%2520and%2520enzymatic%2520activity%2520of%2520the%2520mixed%2520lineage%2520leukemia%2520protein-1%2520core%2520complex%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2008%26volume%3D283%26spage%3D32162%26epage%3D32175%26doi%3D10.1074%2Fjbc.M806317200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group">Orlicky, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Neduva, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Elowe, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, E. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sicheri, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tyers, M.</span><span> </span><span class="NLM_article-title">An allosteric inhibitor of substrate recognition by the SCF<sup>Cdc4</sup> ubiquitin ligase</span> <span class="citation_source-journal">Nat. Biotechnol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">28</span><span class="NLM_x">, </span> <span class="NLM_fpage">733</span><span class="NLM_x">–</span> <span class="NLM_lpage">737</span><span class="refDoi"> DOI: 10.1038/nbt.1646</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01630&amp;key=10.1038%2Fnbt.1646" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01630&amp;key=20581844" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01630&amp;key=1%3ACAS%3A528%3ADC%252BC3cXnvFyksbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2010&pages=733-737&author=S.+Orlickyauthor=X.+Tangauthor=V.+Neduvaauthor=N.+Eloweauthor=E.+D.+Brownauthor=F.+Sicheriauthor=M.+Tyers&title=An+allosteric+inhibitor+of+substrate+recognition+by+the+SCFCdc4+ubiquitin+ligase&doi=10.1038%2Fnbt.1646"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">An allosteric inhibitor of substrate recognition by the SCFCdc4 ubiquitin ligase</span></div><div class="casAuthors">Orlicky, Stephen; Tang, Xiaojing; Neduva, Victor; Elowe, Nadine; Brown, Eric D.; Sicheri, Frank; Tyers, Mike</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">733-737</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The specificity of SCF ubiquitin ligase-mediated protein degrdn. is detd. by F-box proteins.  We identified a biplanar dicarboxylic acid compd., called SCF-I2, as an inhibitor of substrate recognition by the yeast F-box protein Cdc4 using a fluorescence polarization screen to monitor the displacement of a fluorescein-labeled phosphodegron peptide.  SCF-I2 inhibits the binding and ubiquitination of full-length phosphorylated substrates by SCFCdc4.  A co-crystal structure reveals that SCF-I2 inserts itself between the β-strands of blades 5 and 6 of the WD40 propeller domain of Cdc4 at a site that is 25 Å away from the substrate binding site.  Long-range transmission of SCF-I2 interactions distorts the substrate binding pocket and impedes recognition of key determinants in the Cdc4 phosphodegron.  Mutation of the SCF-I2 binding site abrogates its inhibitory effect and explains specificity in the allosteric inhibition mechanism.  Mammalian WD40 domain proteins may exhibit similar allosteric responsiveness and hence represent an extensive class of druggable target.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGptnWKaMi1kdLVg90H21EOLACvtfcHk0ljojHJnc1oopw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXnvFyksbw%253D&md5=23197bd195811ad8503efb5d43457ccf</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1038%2Fnbt.1646&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt.1646%26sid%3Dliteratum%253Aachs%26aulast%3DOrlicky%26aufirst%3DS.%26aulast%3DTang%26aufirst%3DX.%26aulast%3DNeduva%26aufirst%3DV.%26aulast%3DElowe%26aufirst%3DN.%26aulast%3DBrown%26aufirst%3DE.%2BD.%26aulast%3DSicheri%26aufirst%3DF.%26aulast%3DTyers%26aufirst%3DM.%26atitle%3DAn%2520allosteric%2520inhibitor%2520of%2520substrate%2520recognition%2520by%2520the%2520SCFCdc4%2520ubiquitin%2520ligase%26jtitle%3DNat.%2520Biotechnol.%26date%3D2010%26volume%3D28%26spage%3D733%26epage%3D737%26doi%3D10.1038%2Fnbt.1646" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group">Cao, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Townsend, E. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Karatas, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ouillette, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hess, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Atadja, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lei, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Qin, Z. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Malek, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dou, Y.</span><span> </span><span class="NLM_article-title">Targeting MLL1 H3K4 methyltransferase activity in mixed-lineage leukemia</span> <span class="citation_source-journal">Mol. Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">247</span><span class="NLM_x">–</span> <span class="NLM_lpage">261</span><span class="refDoi"> DOI: 10.1016/j.molcel.2013.12.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01630&amp;key=10.1016%2Fj.molcel.2013.12.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01630&amp;key=24389101" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01630&amp;key=1%3ACAS%3A528%3ADC%252BC2cXisVKruw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2014&pages=247-261&author=F.+Caoauthor=E.+C.+Townsendauthor=H.+Karatasauthor=J.+Xuauthor=L.+Liauthor=S.+Leeauthor=L.+Liuauthor=Y.+Chenauthor=P.+Ouilletteauthor=J.+Zhuauthor=J.+L.+Hessauthor=P.+Atadjaauthor=M.+Leiauthor=Z.+S.+Qinauthor=S.+Malekauthor=S.+Wangauthor=Y.+Dou&title=Targeting+MLL1+H3K4+methyltransferase+activity+in+mixed-lineage+leukemia&doi=10.1016%2Fj.molcel.2013.12.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting MLL1 H3K4 Methyltransferase Activity in Mixed-Lineage Leukemia</span></div><div class="casAuthors">Cao, Fang; Townsend, Elizabeth C.; Karatas, Hacer; Xu, Jing; Li, Li; Lee, Shirley; Liu, Liu; Chen, Yong; Ouillette, Peter; Zhu, Jidong; Hess, Jay L.; Atadja, Peter; Lei, Ming; Qin, Zhaohui S.; Malek, Sami; Wang, Shaomeng; Dou, Yali</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">247-261</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Here we report a comprehensive characterization of our recently developed inhibitor MM-401 that targets the MLL1 H3K4 methyltransferase activity.  MM-401 is able to specifically inhibit MLL1 activity by blocking MLL1-WDR5 interaction and thus the complex assembly.  This targeting strategy does not affect other mixed-lineage leukemia (MLL) family histone methyltransferases (HMTs), revealing a unique regulatory feature for the MLL1 complex.  Using MM-401 and its enantiomer control MM-NC-401, we show that inhibiting MLL1 methyltransferase activity specifically blocks proliferation of MLL cells by inducing cell-cycle arrest, apoptosis, and myeloid differentiation without general toxicity to normal bone marrow cells or non-MLL cells.  More importantly, transcriptome analyses show that MM-401 induces changes in gene expression similar to those of MLL1 deletion, supporting a predominant role of MLL1 activity in regulating MLL1-dependent leukemia transcription program.  We envision broad applications for MM-401 in basic and translational research.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGov9sdJIz15pbVg90H21EOLACvtfcHk0ljojHJnc1oopw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXisVKruw%253D%253D&md5=f3e482ba3474b39e1bbdde0d10407d1f</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2013.12.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2013.12.001%26sid%3Dliteratum%253Aachs%26aulast%3DCao%26aufirst%3DF.%26aulast%3DTownsend%26aufirst%3DE.%2BC.%26aulast%3DKaratas%26aufirst%3DH.%26aulast%3DXu%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DLee%26aufirst%3DS.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DOuillette%26aufirst%3DP.%26aulast%3DZhu%26aufirst%3DJ.%26aulast%3DHess%26aufirst%3DJ.%2BL.%26aulast%3DAtadja%26aufirst%3DP.%26aulast%3DLei%26aufirst%3DM.%26aulast%3DQin%26aufirst%3DZ.%2BS.%26aulast%3DMalek%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DS.%26aulast%3DDou%26aufirst%3DY.%26atitle%3DTargeting%2520MLL1%2520H3K4%2520methyltransferase%2520activity%2520in%2520mixed-lineage%2520leukemia%26jtitle%3DMol.%2520Cell%26date%3D2014%26volume%3D53%26spage%3D247%26epage%3D261%26doi%3D10.1016%2Fj.molcel.2013.12.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group">Senisterra, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allali-Hassani, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wasney, G. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barsyte-Lovejoy, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dombrovski, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dong, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nguyen, K. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smil, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bolshan, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hajian, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">He, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Seitova, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chau, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Poda, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Couture, J. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Al-Awar, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schapira, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vedadi, M.</span><span> </span><span class="NLM_article-title">Small-molecule inhibition of MLL1 activity by disruption of its interaction with WDR5</span> <span class="citation_source-journal">Biochem. J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">449</span><span class="NLM_x">, </span> <span class="NLM_fpage">151</span><span class="NLM_x">–</span> <span class="NLM_lpage">159</span><span class="refDoi"> DOI: 10.1042/BJ20121280</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01630&amp;key=10.1042%2FBJ20121280" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01630&amp;key=22989411" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01630&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvVSmtrzL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=449&publication_year=2013&pages=151-159&author=G.+Senisterraauthor=H.+Wuauthor=A.+Allali-Hassaniauthor=G.+A.+Wasneyauthor=D.+Barsyte-Lovejoyauthor=L.+Dombrovskiauthor=A.+Dongauthor=K.+T.+Nguyenauthor=D.+Smilauthor=Y.+Bolshanauthor=T.+Hajianauthor=H.+Heauthor=A.+Seitovaauthor=I.+Chauauthor=F.+Liauthor=G.+Podaauthor=J.+F.+Coutureauthor=P.+J.+Brownauthor=R.+Al-Awarauthor=M.+Schapiraauthor=C.+H.+Arrowsmithauthor=M.+Vedadi&title=Small-molecule+inhibition+of+MLL1+activity+by+disruption+of+its+interaction+with+WDR5&doi=10.1042%2FBJ20121280"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Small-molecule inhibition of MLL activity by disruption of its interaction with WDR5</span></div><div class="casAuthors">Senisterra, Guillermo; Wu, Hong; Allali-Hassani, Abdellah; Wasney, Gregory A.; Barsyte-Lovejoy, Dalia; Dombrovski, Ludmila; Dong, Aiping; Nguyen, Kong T.; Smil, David; Bolshan, Yuri; Hajian, Taraneh; He, Hao; Seitova, Alma; Chau, Irene; Li, Fengling; Poda, Gennadiy; Couture, Jean-Francois; Brown, Peter J.; Al-Awar, Rima; Schapira, Matthieu; Arrowsmith, Cheryl H.; Vedadi, Masoud</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical Journal</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">449</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">151-159</span>CODEN:
                <span class="NLM_cas:coden">BIJOAK</span>;
        ISSN:<span class="NLM_cas:issn">0264-6021</span>.
    
            (<span class="NLM_cas:orgname">Portland Press Ltd.</span>)
        </div><div class="casAbstract">WDR5 (WD40 repeat protein 5) is an essential component of the human trithorax-like family of SET1 [Su(var)3-9 enhancer-of-zeste trithorax 1] methyltransferase complexes that carry out trimethylation of histone 3 Lys4 (H3K4me3), play key roles in development and are abnormally expressed in many cancers.  In the present study, we show that the interaction between WDR5 and peptides from the catalytic domain of MLL (mixed-lineage leukemia protein) (KMT2) can be antagonized with a small mol.  Structural and biophys. anal. show that this antagonist binds in the WDR5 peptide-binding pocket with a Kd of 450 nM and inhibits the catalytic activity of the MLL core complex in vitro.  The degree of inhibition was enhanced at lower protein concns. consistent with a role for WDR5 in directly stabilizing the MLL multiprotein complex.  Our data demonstrate inhibition of an important protein-protein interaction and form the basis for further development of inhibitors of WDR5-dependent enzymes implicated in MLL-rearranged leukemias or other cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrAilf3m7N6c7Vg90H21EOLACvtfcHk0lifXuRQRzPyKw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvVSmtrzL&md5=ad00c4e1bfd48099aa8040199f9311be</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1042%2FBJ20121280&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252FBJ20121280%26sid%3Dliteratum%253Aachs%26aulast%3DSenisterra%26aufirst%3DG.%26aulast%3DWu%26aufirst%3DH.%26aulast%3DAllali-Hassani%26aufirst%3DA.%26aulast%3DWasney%26aufirst%3DG.%2BA.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DDombrovski%26aufirst%3DL.%26aulast%3DDong%26aufirst%3DA.%26aulast%3DNguyen%26aufirst%3DK.%2BT.%26aulast%3DSmil%26aufirst%3DD.%26aulast%3DBolshan%26aufirst%3DY.%26aulast%3DHajian%26aufirst%3DT.%26aulast%3DHe%26aufirst%3DH.%26aulast%3DSeitova%26aufirst%3DA.%26aulast%3DChau%26aufirst%3DI.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DPoda%26aufirst%3DG.%26aulast%3DCouture%26aufirst%3DJ.%2BF.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DAl-Awar%26aufirst%3DR.%26aulast%3DSchapira%26aufirst%3DM.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DVedadi%26aufirst%3DM.%26atitle%3DSmall-molecule%2520inhibition%2520of%2520MLL1%2520activity%2520by%2520disruption%2520of%2520its%2520interaction%2520with%2520WDR5%26jtitle%3DBiochem.%2520J.%26date%3D2013%26volume%3D449%26spage%3D151%26epage%3D159%26doi%3D10.1042%2FBJ20121280" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group">Bolshan, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Getlik, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuznetsova, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wasney, G. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hajian, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Poda, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nguyen, K. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dombrovski, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dong, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Senisterra, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schapira, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Al-awar, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vedadi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smil, D.</span><span> </span><span class="NLM_article-title">Synthesis, optimization, and evaluation of novel small molecules as antagonists of WDR5-MLL1 interaction</span> <span class="citation_source-journal">ACS Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x">, </span> <span class="NLM_fpage">353</span><span class="NLM_x">–</span> <span class="NLM_lpage">357</span><span class="refDoi"> DOI: 10.1021/ml300467n</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ml300467n" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2013&pages=353-357&author=Y.+Bolshanauthor=M.+Getlikauthor=E.+Kuznetsovaauthor=G.+A.+Wasneyauthor=T.+Hajianauthor=G.+Podaauthor=K.+T.+Nguyenauthor=H.+Wuauthor=L.+Dombrovskiauthor=A.+Dongauthor=G.+Senisterraauthor=M.+Schapiraauthor=C.+H.+Arrowsmithauthor=P.+J.+Brownauthor=R.+Al-awarauthor=M.+Vedadiauthor=D.+Smil&title=Synthesis%2C+optimization%2C+and+evaluation+of+novel+small+molecules+as+antagonists+of+WDR5-MLL1+interaction&doi=10.1021%2Fml300467n"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1021%2Fml300467n&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fml300467n%26sid%3Dliteratum%253Aachs%26aulast%3DBolshan%26aufirst%3DY.%26aulast%3DGetlik%26aufirst%3DM.%26aulast%3DKuznetsova%26aufirst%3DE.%26aulast%3DWasney%26aufirst%3DG.%2BA.%26aulast%3DHajian%26aufirst%3DT.%26aulast%3DPoda%26aufirst%3DG.%26aulast%3DNguyen%26aufirst%3DK.%2BT.%26aulast%3DWu%26aufirst%3DH.%26aulast%3DDombrovski%26aufirst%3DL.%26aulast%3DDong%26aufirst%3DA.%26aulast%3DSenisterra%26aufirst%3DG.%26aulast%3DSchapira%26aufirst%3DM.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DAl-awar%26aufirst%3DR.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DSmil%26aufirst%3DD.%26atitle%3DSynthesis%252C%2520optimization%252C%2520and%2520evaluation%2520of%2520novel%2520small%2520molecules%2520as%2520antagonists%2520of%2520WDR5-MLL1%2520interaction%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2013%26volume%3D4%26spage%3D353%26epage%3D357%26doi%3D10.1021%2Fml300467n" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group">Grebien, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vedadi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Getlik, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Giambruno, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grover, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Avellino, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Skucha, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vittori, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuznetsova, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smil, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barsyte-Lovejoy, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Poda, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schapira, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dong, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Senisterra, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stukalov, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huber, K. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schönegger, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marcellus, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bilban, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bock, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zuber, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bennett, K. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Al-awar, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Delwel, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nerlov, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Superti-Furga, G.</span><span> </span><span class="NLM_article-title">Pharmacological targeting of the WDR5-MLL1 interaction in C/EBPα <i>N</i>-terminal leukemia</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">571</span><span class="NLM_x">–</span> <span class="NLM_lpage">578</span><span class="refDoi"> DOI: 10.1038/nchembio.1859</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01630&amp;key=10.1038%2Fnchembio.1859" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01630&amp;key=26167872" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01630&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht1WltLfM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2015&pages=571-578&author=F.+Grebienauthor=M.+Vedadiauthor=M.+Getlikauthor=R.+Giambrunoauthor=A.+Groverauthor=R.+Avellinoauthor=A.+Skuchaauthor=S.+Vittoriauthor=E.+Kuznetsovaauthor=D.+Smilauthor=D.+Barsyte-Lovejoyauthor=F.+Liauthor=G.+Podaauthor=M.+Schapiraauthor=H.+Wuauthor=A.+Dongauthor=G.+Senisterraauthor=A.+Stukalovauthor=K.+V.+Huberauthor=A.+Sch%C3%B6neggerauthor=R.+Marcellusauthor=M.+Bilbanauthor=C.+Bockauthor=P.+J.+Brownauthor=J.+Zuberauthor=K.+L.+Bennettauthor=R.+Al-awarauthor=R.+Delwelauthor=C.+Nerlovauthor=C.+H.+Arrowsmithauthor=G.+Superti-Furga&title=Pharmacological+targeting+of+the+WDR5-MLL1+interaction+in+C%2FEBP%CE%B1+N-terminal+leukemia&doi=10.1038%2Fnchembio.1859"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Pharmacological targeting of the Wdr5-MLL interaction in C/EBPα N-terminal leukemia</span></div><div class="casAuthors">Grebien, Florian; Vedadi, Masoud; Getlik, Matthaeus; Giambruno, Roberto; Grover, Amit; Avellino, Roberto; Skucha, Anna; Vittori, Sarah; Kuznetsova, Ekaterina; Smil, David; Barsyte-Lovejoy, Dalia; Li, Fengling; Poda, Gennadiy; Schapira, Matthieu; Wu, Hong; Dong, Aiping; Senisterra, Guillermo; Stukalov, Alexey; Huber, Kilian V. M.; Schoenegger, Andreas; Marcellus, Richard; Bilban, Martin; Bock, Christoph; Brown, Peter J.; Zuber, Johannes; Bennett, Keiryn L.; Al-awar, Rima; Delwel, Ruud; Nerlov, Claus; Arrowsmith, Cheryl H.; Superti-Furga, Giulio</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">571-578</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The CEBPA gene is mutated in 9% of patients with acute myeloid leukemia (AML).  Selective expression of a short (30-kDa) CCAAT-enhancer binding protein-α (C/EBPα) translational isoform, termed p30, represents the most common type of CEBPA mutation in AML.  The mol. mechanisms underlying p30-mediated transformation remain incompletely understood.  We show that C/EBPα p30, but not the normal p42 isoform, preferentially interacts with Wdr5, a key component of SET/MLL (SET-domain/mixed-lineage leukemia) histone-methyltransferase complexes.  Accordingly, p30-bound genomic regions were enriched for MLL-dependent H3K4me3 marks.  The p30-dependent increase in self-renewal and inhibition of myeloid differentiation required Wdr5, as downregulation of the latter inhibited proliferation and restored differentiation in p30-dependent AML models.  OICR-9429 is a new small-mol. antagonist of the Wdr5-MLL interaction.  This compd. selectively inhibited proliferation and induced differentiation in p30-expressing human AML cells.  Our data reveal the mechanism of p30-dependent transformation and establish the essential p30 cofactor Wdr5 as a therapeutic target in CEBPA-mutant AML.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqLFPcxLpW4sbVg90H21EOLACvtfcHk0lh7sCElsgJnqw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXht1WltLfM&md5=91c1937ebcfd0552a8cf02e354f8c663</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1859&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1859%26sid%3Dliteratum%253Aachs%26aulast%3DGrebien%26aufirst%3DF.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DGetlik%26aufirst%3DM.%26aulast%3DGiambruno%26aufirst%3DR.%26aulast%3DGrover%26aufirst%3DA.%26aulast%3DAvellino%26aufirst%3DR.%26aulast%3DSkucha%26aufirst%3DA.%26aulast%3DVittori%26aufirst%3DS.%26aulast%3DKuznetsova%26aufirst%3DE.%26aulast%3DSmil%26aufirst%3DD.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DPoda%26aufirst%3DG.%26aulast%3DSchapira%26aufirst%3DM.%26aulast%3DWu%26aufirst%3DH.%26aulast%3DDong%26aufirst%3DA.%26aulast%3DSenisterra%26aufirst%3DG.%26aulast%3DStukalov%26aufirst%3DA.%26aulast%3DHuber%26aufirst%3DK.%2BV.%26aulast%3DSch%25C3%25B6negger%26aufirst%3DA.%26aulast%3DMarcellus%26aufirst%3DR.%26aulast%3DBilban%26aufirst%3DM.%26aulast%3DBock%26aufirst%3DC.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DZuber%26aufirst%3DJ.%26aulast%3DBennett%26aufirst%3DK.%2BL.%26aulast%3DAl-awar%26aufirst%3DR.%26aulast%3DDelwel%26aufirst%3DR.%26aulast%3DNerlov%26aufirst%3DC.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DSuperti-Furga%26aufirst%3DG.%26atitle%3DPharmacological%2520targeting%2520of%2520the%2520WDR5-MLL1%2520interaction%2520in%2520C%252FEBP%25CE%25B1%2520N-terminal%2520leukemia%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2015%26volume%3D11%26spage%3D571%26epage%3D578%26doi%3D10.1038%2Fnchembio.1859" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group">Zhu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sammons, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Donahue, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dou, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vedadi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Getlik, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barsyte-Lovejoy, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Al-awar, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Katona, B. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shilatifard, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hua, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Berger, S. L.</span><span> </span><span class="NLM_article-title">Gain-of-function p53 mutants co-opt chromatin pathways to drive cancer growth</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">525</span><span class="NLM_x">, </span> <span class="NLM_fpage">206</span><span class="NLM_x">–</span> <span class="NLM_lpage">211</span><span class="refDoi"> DOI: 10.1038/nature15251</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01630&amp;key=10.1038%2Fnature15251" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01630&amp;key=26331536" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01630&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsVCis7%252FI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=525&publication_year=2015&pages=206-211&author=J.+Zhuauthor=M.+A.+Sammonsauthor=G.+Donahueauthor=Z.+Douauthor=M.+Vedadiauthor=M.+Getlikauthor=D.+Barsyte-Lovejoyauthor=R.+Al-awarauthor=B.+W.+Katonaauthor=A.+Shilatifardauthor=J.+Huangauthor=X.+Huaauthor=C.+H.+Arrowsmithauthor=S.+L.+Berger&title=Gain-of-function+p53+mutants+co-opt+chromatin+pathways+to+drive+cancer+growth&doi=10.1038%2Fnature15251"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Gain-of-function p53 mutants co-opt chromatin pathways to drive cancer growth</span></div><div class="casAuthors">Zhu, Jiajun; Sammons, Morgan A.; Donahue, Greg; Dou, Zhixun; Vedadi, Masoud; Getlik, Matthaus; Barsyte-Lovejoy, Dalia; Al-awar, Rima; Katona, Bryson W.; Shilatifard, Ali; Huang, Jing; Hua, Xianxin; Arrowsmith, Cheryl H.; Berger, Shelley L.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">525</span>
        (<span class="NLM_cas:issue">7568</span>),
    <span class="NLM_cas:pages">206-211</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">TP53 (which encodes p53 protein) is the most frequently mutated gene among all human cancers.  Prevalent p53 missense mutations abrogate its tumor suppressive function and lead to a 'gain-of-function' (GOF) that promotes cancer.  Here we show that p53 GOF mutants bind to and upregulate chromatin regulatory genes, including the methyltransferases MLL1 (also known as KMT2A), MLL2 (also known as KMT2D), and acetyltransferase MOZ (also known as KAT6A or MYST3), resulting in genome-wide increases of histone methylation and acetylation.  Anal. of The Cancer Genome Atlas shows specific upregulation of MLL1, MLL2, and MOZ in p53 GOF patient-derived tumors, but not in wild-type p53 or p53 null tumors.  Cancer cell proliferation is markedly lowered by genetic knockdown of MLL1 or by pharmacol. inhibition of the MLL1 methyltransferase complex.  Our study reveals a novel chromatin mechanism underlying the progression of tumors with GOF p53, and suggests new possibilities for designing combinatorial chromatin-based therapies for treating individual cancers driven by prevalent GOF p53 mutations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrG4nSmyzIWUbVg90H21EOLACvtfcHk0lh7sCElsgJnqw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsVCis7%252FI&md5=e37e60db0774b56a8df447d46acaad2a</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1038%2Fnature15251&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature15251%26sid%3Dliteratum%253Aachs%26aulast%3DZhu%26aufirst%3DJ.%26aulast%3DSammons%26aufirst%3DM.%2BA.%26aulast%3DDonahue%26aufirst%3DG.%26aulast%3DDou%26aufirst%3DZ.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DGetlik%26aufirst%3DM.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DAl-awar%26aufirst%3DR.%26aulast%3DKatona%26aufirst%3DB.%2BW.%26aulast%3DShilatifard%26aufirst%3DA.%26aulast%3DHuang%26aufirst%3DJ.%26aulast%3DHua%26aufirst%3DX.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DBerger%26aufirst%3DS.%2BL.%26atitle%3DGain-of-function%2520p53%2520mutants%2520co-opt%2520chromatin%2520pathways%2520to%2520drive%2520cancer%2520growth%26jtitle%3DNature%26date%3D2015%26volume%3D525%26spage%3D206%26epage%3D211%26doi%3D10.1038%2Fnature15251" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group">Sun, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bell, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carter, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gherardi, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Poulos, R. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Milazzo, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wong, J. W. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Al-Awar, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tee, A. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, P. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Atmadibrata, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wong, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Trahair, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhao, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shohet, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Haupt, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schulte, J. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vedadi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">MacKenzie, K. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hüttelmaier, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perini, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marshall, G. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Braithwaite, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, T.</span><span> </span><span class="NLM_article-title">WDR5 supports an N-Myc transcriptional complex that drives a protumorigenic gene expression signature in neuroblastoma</span> <span class="citation_source-journal">Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">75</span><span class="NLM_x">, </span> <span class="NLM_fpage">5143</span><span class="NLM_x">–</span> <span class="NLM_lpage">5154</span><span class="refDoi"> DOI: 10.1158/0008-5472.CAN-15-0423</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01630&amp;key=10.1158%2F0008-5472.CAN-15-0423" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01630&amp;key=26471359" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01630&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvFensrzI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=75&publication_year=2015&pages=5143-5154&author=Y.+Sunauthor=J.+L.+Bellauthor=D.+Carterauthor=S.+Gherardiauthor=R.+C.+Poulosauthor=G.+Milazzoauthor=J.+W.+H.+Wongauthor=R.+Al-Awarauthor=A.+E.+Teeauthor=P.+Y.+Liuauthor=B.+Liuauthor=B.+Atmadibrataauthor=M.+Wongauthor=T.+Trahairauthor=Q.+Zhaoauthor=J.+M.+Shohetauthor=Y.+Hauptauthor=J.+H.+Schulteauthor=P.+J.+Brownauthor=C.+H.+Arrowsmithauthor=M.+Vedadiauthor=K.+L.+MacKenzieauthor=S.+H%C3%BCttelmaierauthor=G.+Periniauthor=G.+M.+Marshallauthor=A.+Braithwaiteauthor=T.+Liu&title=WDR5+supports+an+N-Myc+transcriptional+complex+that+drives+a+protumorigenic+gene+expression+signature+in+neuroblastoma&doi=10.1158%2F0008-5472.CAN-15-0423"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">WDR5 Supports an N-Myc Transcriptional Complex That Drives a Protumorigenic Gene Expression Signature in Neuroblastoma</span></div><div class="casAuthors">Sun, Yuting; Bell, Jessica L.; Carter, Daniel; Gherardi, Samuele; Poulos, Rebecca C.; Milazzo, Giorgio; Wong, Jason W. H.; Al-Awar, Rima; Tee, Andrew E.; Liu, Pei Y.; Liu, Bing; Atmadibrata, Bernard; Wong, Matthew; Trahair, Toby; Zhao, Quan; Shohet, Jason M.; Haupt, Ygal; Schulte, Johannes H.; Brown, Peter J.; Arrowsmith, Cheryl H.; Vedadi, Masoud; MacKenzie, Karen L.; Huettelmaier, Stefan; Perini, Giovanni; Marshall, Glenn M.; Braithwaite, Antony; Liu, Tao</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">75</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">5143-5154</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">MYCN gene amplification in neuroblastoma drives a gene expression program that correlates strongly with aggressive disease.  Mechanistically, trimethylation of histone H3 lysine 4 (H3K4) at target gene promoters is a strict prerequisite for this transcriptional program to be enacted.  WDR5 is a histone H3K4 presenter that has been found to have an essential role in H3K4 trimethylation.  For this reason, in this study, we investigated the relationship between WDR5-mediated H3K4 trimethylation and N-Myc transcriptional programs in neuroblastoma cells.  N-Myc upregulated WDR5 expression in neuroblastoma cells.  Gene expression anal. revealed that WDR5 target genes included those with MYC-binding elements at promoters such as MDM2.  We showed that WDR5 could form a protein complex at the MDM2 promoter with N-Myc, but not p53, leading to histone H3K4 trimethylation and activation of MDM2 transcription.  RNAi-mediated attenuation of WDR5 upregulated expression of wild-type but not mutant p53, an effect assocd. with growth inhibition and apoptosis.  Similarly, a small-mol. antagonist of WDR5 reduced N-Myc/WDR5 complex formation, N-Myc target gene expression, and cell growth in neuroblastoma cells.  In MYCN-transgenic mice, WDR5 was overexpressed in precancerous ganglion and neuroblastoma cells compared with normal ganglion cells.  Clin., elevated levels of WDR5 in neuroblastoma specimens were an independent predictor of poor overall survival.  Overall, our results identify WDR5 as a key cofactor for N-Myc-regulated transcriptional activation and tumorigenesis and as a novel therapeutic target for MYCN-amplified neuroblastomas.  Cancer Res; 75(23); 5143-54. ©2015 AACR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpdt2xhQOC6ibVg90H21EOLACvtfcHk0lhXP7epnybYgw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvFensrzI&md5=26513b24ba2ba6b82cc691534f5939cd</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-15-0423&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-15-0423%26sid%3Dliteratum%253Aachs%26aulast%3DSun%26aufirst%3DY.%26aulast%3DBell%26aufirst%3DJ.%2BL.%26aulast%3DCarter%26aufirst%3DD.%26aulast%3DGherardi%26aufirst%3DS.%26aulast%3DPoulos%26aufirst%3DR.%2BC.%26aulast%3DMilazzo%26aufirst%3DG.%26aulast%3DWong%26aufirst%3DJ.%2BW.%2BH.%26aulast%3DAl-Awar%26aufirst%3DR.%26aulast%3DTee%26aufirst%3DA.%2BE.%26aulast%3DLiu%26aufirst%3DP.%2BY.%26aulast%3DLiu%26aufirst%3DB.%26aulast%3DAtmadibrata%26aufirst%3DB.%26aulast%3DWong%26aufirst%3DM.%26aulast%3DTrahair%26aufirst%3DT.%26aulast%3DZhao%26aufirst%3DQ.%26aulast%3DShohet%26aufirst%3DJ.%2BM.%26aulast%3DHaupt%26aufirst%3DY.%26aulast%3DSchulte%26aufirst%3DJ.%2BH.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DMacKenzie%26aufirst%3DK.%2BL.%26aulast%3DH%25C3%25BCttelmaier%26aufirst%3DS.%26aulast%3DPerini%26aufirst%3DG.%26aulast%3DMarshall%26aufirst%3DG.%2BM.%26aulast%3DBraithwaite%26aufirst%3DA.%26aulast%3DLiu%26aufirst%3DT.%26atitle%3DWDR5%2520supports%2520an%2520N-Myc%2520transcriptional%2520complex%2520that%2520drives%2520a%2520protumorigenic%2520gene%2520expression%2520signature%2520in%2520neuroblastoma%26jtitle%3DCancer%2520Res.%26date%3D2015%26volume%3D75%26spage%3D5143%26epage%3D5154%26doi%3D10.1158%2F0008-5472.CAN-15-0423" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="note"><p class="first last">Data not shown. Unpublished results.</p></div><div class="citationLinks"><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><div class="note"><p class="first last">In order to obtain a better signal in the FP assay used for this publication, we synthesized the described 9-Ala-FAM peptide based on the 10-Ala-FAM peptide reported in</p></div><span class="NLM_contrib-group">Karatas, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Townsend, E. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bernard, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dou, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, S.</span><span> </span><span class="NLM_article-title">Analysis of the binding of mixed-lineage leukemia 1 (MLL1) and histone 3 peptides to WD repeat domain 5 (WDR5) for the design of inhibitors of the MLL1-WDR5 interaction</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">5179</span><span class="NLM_x">–</span> <span class="NLM_lpage">5185</span><span class="refDoi"> DOI: 10.1021/jm100139b</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm100139b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=5179-5185&author=H.+Karatasauthor=E.+C.+Townsendauthor=D.+Bernardauthor=Y.+Douauthor=S.+Wang&title=Analysis+of+the+binding+of+mixed-lineage+leukemia+1+%28MLL1%29+and+histone+3+peptides+to+WD+repeat+domain+5+%28WDR5%29+for+the+design+of+inhibitors+of+the+MLL1-WDR5+interaction&doi=10.1021%2Fjm100139b"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1021%2Fjm100139b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm100139b%26sid%3Dliteratum%253Aachs%26aulast%3DKaratas%26aufirst%3DH.%26aulast%3DTownsend%26aufirst%3DE.%2BC.%26aulast%3DBernard%26aufirst%3DD.%26aulast%3DDou%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DAnalysis%2520of%2520the%2520binding%2520of%2520mixed-lineage%2520leukemia%25201%2520%2528MLL1%2529%2520and%2520histone%25203%2520peptides%2520to%2520WD%2520repeat%2520domain%25205%2520%2528WDR5%2529%2520for%2520the%2520design%2520of%2520inhibitors%2520of%2520the%2520MLL1-WDR5%2520interaction%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D5179%26epage%3D5185%26doi%3D10.1021%2Fjm100139b" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group">Bissantz, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuhn, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stahl, M.</span><span> </span><span class="NLM_article-title">A medicinal chemist’s guide to molecular interactions</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">5061</span><span class="NLM_x">–</span> <span class="NLM_lpage">5084</span><span class="refDoi"> DOI: 10.1021/jm100112j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm100112j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01630&amp;key=1%3ACAS%3A528%3ADC%252BC3cXjvFKjtbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=5061-5084&author=C.+Bissantzauthor=B.+Kuhnauthor=M.+Stahl&title=A+medicinal+chemist%E2%80%99s+guide+to+molecular+interactions&doi=10.1021%2Fjm100112j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">A Medicinal Chemist's Guide to Molecular Interactions</span></div><div class="casAuthors">Bissantz, Caterina; Kuhn, Bernd; Stahl, Martin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">5061-5084</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGraV8HPbwsihbVg90H21EOLACvtfcHk0lhXP7epnybYgw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXjvFKjtbs%253D&md5=f39cf8700267cdc311c73de4d433bab0</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1021%2Fjm100112j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm100112j%26sid%3Dliteratum%253Aachs%26aulast%3DBissantz%26aufirst%3DC.%26aulast%3DKuhn%26aufirst%3DB.%26aulast%3DStahl%26aufirst%3DM.%26atitle%3DA%2520medicinal%2520chemist%25E2%2580%2599s%2520guide%2520to%2520molecular%2520interactions%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D5061%26epage%3D5084%26doi%3D10.1021%2Fjm100112j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="note"><p class="first last">Qualitative finding based on the apparent solubility of the compounds in common organic solvents such as acetonitrile, methanol, and DMSO observed during the course of experimental work, measurements using SPR, and compound characterization by NMR.</p></div><div class="citationLinks"><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span>Currently BIOVIA; Virtual Biosphere and Materials. <a href="http://www.3ds.com/products-services/biovia/" class="extLink">http://www.3ds.com/products-services/biovia/</a> (accessed February 17,<span class="NLM_x"> </span><span class="NLM_year">2016</span><span class="NLM_x">) </span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Currently+BIOVIA%3B+Virtual+Biosphere+and+Materials.+http%3A%2F%2Fwww.3ds.com%2Fproducts-services%2Fbiovia%2F+%28accessed+February+17%2C+2016%29+."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweb%26sid%3Dliteratum%253Aachs%26jtitle%3DBIOVIA%253B%2520Virtual%2520Biosphere%2520and%2520Materials%26date%3D2016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span>eMolecules. <a href="http://www.emolecules.com" class="extLink">http://www.emolecules.com</a> (accessed February 17,<span class="NLM_x"> </span><span class="NLM_year">2016</span><span class="NLM_x">) </span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=eMolecules.+http%3A%2F%2Fwww.emolecules.com+%28accessed+February+17%2C+2016%29+."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweb%26sid%3Dliteratum%253Aachs%26jtitle%3DeMolecules%26date%3D2016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span>Glide docking program. <a href="http://www.schrodinger.com" class="extLink">http://www.schrodinger.com</a> (accessed February 17,<span class="NLM_x"> </span><span class="NLM_year">2016</span><span class="NLM_x">) </span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Glide+docking+program.+http%3A%2F%2Fwww.schrodinger.com+%28accessed+February+17%2C+2016%29+."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweb%26sid%3Dliteratum%253Aachs%26jtitle%3DGlide%2520docking%2520program%26date%3D2016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span>Cerep. <a href="http://www.cerep.fr/cerep/users/pages/catalog/profiles/catalog.asp" class="extLink">http://www.cerep.fr/cerep/users/pages/catalog/profiles/catalog.asp</a> (accessed February 17,<span class="NLM_x"> </span><span class="NLM_year">2016</span><span class="NLM_x">) </span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Cerep.+http%3A%2F%2Fwww.cerep.fr%2Fcerep%2Fusers%2Fpages%2Fcatalog%2Fprofiles%2Fcatalog.asp+%28accessed+February+17%2C+2016%29+."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweb%26sid%3Dliteratum%253Aachs%26jtitle%3DCerep%26date%3D2016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group">Zuber, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Radtke, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pardee, T. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhao, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rappaport, A. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luo, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCurrach, M. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, M.-M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dolan, M. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kogan, S. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Downing, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lowe, S. W.</span><span> </span><span class="NLM_article-title">Mouse models of human AML accurately predict chemotherapy response</span> <span class="citation_source-journal">Genes Dev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">23</span><span class="NLM_x">, </span> <span class="NLM_fpage">877</span><span class="NLM_x">–</span> <span class="NLM_lpage">889</span><span class="refDoi"> DOI: 10.1101/gad.1771409</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01630&amp;key=10.1101%2Fgad.1771409" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01630&amp;key=19339691" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01630&amp;key=1%3ACAS%3A528%3ADC%252BD1MXks1Srurs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2009&pages=877-889&author=J.+Zuberauthor=I.+Radtkeauthor=T.+S.+Pardeeauthor=Z.+Zhaoauthor=A.+R.+Rappaportauthor=W.+Luoauthor=M.+E.+McCurrachauthor=M.-M.+Yangauthor=M.+E.+Dolanauthor=S.+C.+Koganauthor=J.+R.+Downingauthor=S.+W.+Lowe&title=Mouse+models+of+human+AML+accurately+predict+chemotherapy+response&doi=10.1101%2Fgad.1771409"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Mouse models of human AML accurately predict chemotherapy response</span></div><div class="casAuthors">Zuber, Johannes; Radtke, Ina; Pardee, Timothy S.; Zhao, Zhen; Rappaport, Amy R.; Luo, Weijun; McCurrach, Mila E.; Yang, Miao-Miao; Dolan, M. Eileen; Kogan, Scott C.; Downing, James R.; Lowe, Scott W.</div><div class="citationInfo"><span class="NLM_cas:title">Genes & Development</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">877-889</span>CODEN:
                <span class="NLM_cas:coden">GEDEEP</span>;
        ISSN:<span class="NLM_cas:issn">0890-9369</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">The genetic heterogeneity of cancer influences the trajectory of tumor progression and may underlie clin. variation in therapy response.  To model such heterogeneity, we produced genetically and pathol. accurate mouse models of common forms of human acute myeloid leukemia (AML) and developed methods to mimic std. induction chemotherapy and efficiently monitor therapy response.  We see that murine AMLs harboring two common human AML genotypes show remarkably diverse responses to conventional therapy that mirror clin. experience.  Specifically, murine leukemias expressing the AML1/ETO fusion oncoprotein, assocd. with a favorable prognosis in patients, show a dramatic response to induction chemotherapy owing to robust activation of the p53 tumor suppressor network.  Conversely, murine leukemias expressing MLL fusion proteins, assocd. with a dismal prognosis in patients, are drug-resistant due to an attenuated p53 response.  These studies highlight the importance of genetic information in guiding the treatment of human AML, functionally establish the p53 network as a central determinant of chemotherapy response in AML, and demonstrate that genetically engineered mouse models of human cancer can accurately predict therapy response in patients.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpjBGFSrry7j7Vg90H21EOLACvtfcHk0liL5HUjQFu1dA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXks1Srurs%253D&md5=2307241c6a572fe14a6ce00af7a93f1c</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1101%2Fgad.1771409&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fgad.1771409%26sid%3Dliteratum%253Aachs%26aulast%3DZuber%26aufirst%3DJ.%26aulast%3DRadtke%26aufirst%3DI.%26aulast%3DPardee%26aufirst%3DT.%2BS.%26aulast%3DZhao%26aufirst%3DZ.%26aulast%3DRappaport%26aufirst%3DA.%2BR.%26aulast%3DLuo%26aufirst%3DW.%26aulast%3DMcCurrach%26aufirst%3DM.%2BE.%26aulast%3DYang%26aufirst%3DM.-M.%26aulast%3DDolan%26aufirst%3DM.%2BE.%26aulast%3DKogan%26aufirst%3DS.%2BC.%26aulast%3DDowning%26aufirst%3DJ.%2BR.%26aulast%3DLowe%26aufirst%3DS.%2BW.%26atitle%3DMouse%2520models%2520of%2520human%2520AML%2520accurately%2520predict%2520chemotherapy%2520response%26jtitle%3DGenes%2520Dev.%26date%3D2009%26volume%3D23%26spage%3D877%26epage%3D889%26doi%3D10.1101%2Fgad.1771409" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group">McCormack, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bruserud, Ø</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gjertsen, B. T.</span><span> </span><span class="NLM_article-title">Animal models of acute myelogenous leukaemia - development, application and future perspectives</span> <span class="citation_source-journal">Leukemia</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">687</span><span class="NLM_x">–</span> <span class="NLM_lpage">706</span><span class="refDoi"> DOI: 10.1038/sj.leu.2403670</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01630&amp;key=10.1038%2Fsj.leu.2403670" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2005&pages=687-706&author=E.+McCormackauthor=%C3%98+Bruserudauthor=B.+T.+Gjertsen&title=Animal+models+of+acute+myelogenous+leukaemia+-+development%2C+application+and+future+perspectives&doi=10.1038%2Fsj.leu.2403670"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1038%2Fsj.leu.2403670&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.leu.2403670%26sid%3Dliteratum%253Aachs%26aulast%3DMcCormack%26aufirst%3DE.%26aulast%3DBruserud%26aufirst%3D%25C3%2598%26aulast%3DGjertsen%26aufirst%3DB.%2BT.%26atitle%3DAnimal%2520models%2520of%2520acute%2520myelogenous%2520leukaemia%2520-%2520development%252C%2520application%2520and%2520future%2520perspectives%26jtitle%3DLeukemia%26date%3D2005%26volume%3D19%26spage%3D687%26epage%3D706%26doi%3D10.1038%2Fsj.leu.2403670" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3SMR" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3SMR','PDB','3SMR'); return false;">PDB: 3SMR</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4IA9" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4IA9','PDB','4IA9'); return false;">PDB: 4IA9</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5EAR" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5EAR','PDB','5EAR'); return false;">PDB: 5EAR</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5EAP" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5EAP','PDB','5EAP'); return false;">PDB: 5EAP</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5EAL" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5EAL','PDB','5EAL'); return false;">PDB: 5EAL</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5EAM" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5EAM','PDB','5EAM'); return false;">PDB: 5EAM</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4FOC" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4FOC','PDB','4FOC'); return false;">PDB: 4FOC</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3CS9" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3CS9','PDB','3CS9'); return false;">PDB: 3CS9</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3GXL" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3GXL','PDB','3GXL'); return false;">PDB: 3GXL</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2W7X" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2W7X','PDB','2W7X'); return false;">PDB: 2W7X</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4A9U" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4A9U','PDB','4A9U'); return false;">PDB: 4A9U</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3VO3" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3VO3','PDB','3VO3'); return false;">PDB: 3VO3</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4AGD" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4AGD','PDB','4AGD'); return false;">PDB: 4AGD</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4QL1" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4QL1','PDB','4QL1'); return false;">PDB: 4QL1</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i92"><a href="/doi/suppl/10.1021/acs.jmedchem.5b01630">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_70377"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a class="ext-link" href="/">ACS Publications website</a> at DOI: <a class="ext-link" href="/doi/abs/10.1021/acs.jmedchem.5b01630">10.1021/acs.jmedchem.5b01630</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Synthetic procedures and compound characterization data, NMR spectra of select compounds, details of the focused library design and molecular docking, methods for cloning, expression, and purification of human WDR5, protocols for the fluorescence polarization (FP) binding assays, crystallization and structure determination methods, and how pharmacokinetic parameters were obtained (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.5b01630/suppl_file/jm5b01630_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.5b01630/suppl_file/jm5b01630_si_002.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.5b01630/suppl_file/jm5b01630_si_001.pdf">jm5b01630_si_001.pdf (1.29 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.5b01630/suppl_file/jm5b01630_si_002.csv">jm5b01630_si_002.csv (4.68 kb)</a></li></ul></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">PDB codes for the structures of WDR5 bound to various antagonists are as follows: WDR5-<b>4</b>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3SMR">3SMR</a>; WDR5-<b>5</b>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4IA9">4IA9</a>; WDR5-<b>9d</b>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5EAR">5EAR</a>; WDR5-<b>9e</b>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5EAP">5EAP</a>; WDR5-<b>9h</b>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5EAL">5EAL</a>; WDR5-<b>9o</b>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5EAM">5EAM</a>; WDR5-<b>16d</b>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4QL1">4QL1</a>.</p><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.5b01630&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.5b01630%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2016.59.issue-6%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.5b01630" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                3MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"679955390c3b3d42","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
