
<html lang="en"     class="pb-page"  data-request-id="055d43da-9bb5-4f30-bf0b-d07ea68eb6c5"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;article:article:10.1021/acs.jmedchem.0c01487;ctype:string:Journal Content;issue:issue:10.1021/jmcmar.2021.64.issue-5;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Targeting Bromodomain and Extraterminal Proteins for Drug Discovery: From Current Progress to Technological Development" /></meta><meta name="dc.Creator" content="Pan  Tang" /></meta><meta name="dc.Creator" content="Jifa  Zhang" /></meta><meta name="dc.Creator" content="Jie  Liu" /></meta><meta name="dc.Creator" content="Cheng-Ming  Chiang" /></meta><meta name="dc.Creator" content="Liang  Ouyang" /></meta><meta name="dc.Description" content="Bromodomain and extraterminal (BET) proteins bind acetylated lysine residues in histones and nonhistone proteins via tandem bromodomains and regulate chromatin dynamics, cellular processes, and dis..." /></meta><meta name="Description" content="Bromodomain and extraterminal (BET) proteins bind acetylated lysine residues in histones and nonhistone proteins via tandem bromodomains and regulate chromatin dynamics, cellular processes, and dis..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="February 22, 2021" /></meta><meta name="dc.Type" content="review-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.0c01487" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2021 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c01487" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.0c01487" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c01487" /></link>
        
    
    

<title>Targeting Bromodomain and Extraterminal Proteins for Drug Discovery: From Current Progress to Technological Development | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.0c01487" /></meta><meta property="og:title" content="Targeting Bromodomain and Extraterminal Proteins for Drug Discovery: From Current Progress to Technological Development" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01487/20210305/images/large/jm0c01487_0007.jpeg" /></meta><meta property="og:description" content="Bromodomain and extraterminal (BET) proteins bind acetylated lysine residues in histones and nonhistone proteins via tandem bromodomains and regulate chromatin dynamics, cellular processes, and disease procession. Thus targeting BET proteins is a promising strategy for treating various diseases, especially malignant tumors and chronic inflammation. Many pan-BET small-molecule inhibitors have been described, and some of them are in clinical evaluation. Nevertheless, the limited clinical efficacy of the current BET inhibitors is also evident and has inspired the development of new technologies to improve their clinical outcomes and minimize unwanted side effects. In this Review, we summarize the latest protein characteristics and biological functions of BRD4 as an example of BET proteins, analyze the clinical development status and preclinical resistance mechanisms, and discuss recent advances in BRD4-selective inhibitors, dual-target BET inhibitors, proteolysis targeting chimera degraders, and protein–protein interaction inhibitors." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.0c01487"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c01487">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.0c01487&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.0c01487&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.0c01487&amp;href=/doi/10.1021/acs.jmedchem.0c01487" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2021</span><span class="cit-fg-volume">, 64</span><span class="cit-fg-issue">, 5</span><span class="cit-fg-pageRange">, 2419-2435</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/64/5" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.0c01180" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Perspective</span><a href="/doi/10.1021/acs.jmedchem.0c01746" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Targeting Bromodomain and Extraterminal Proteins for Drug Discovery: From Current Progress to Technological Development</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Pan Tang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Pan Tang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">State Key Laboratory of Biotherapy and Cancer Center, National Clinical Research Center for Geriatrics, West China Hospital of Sichuan University, Chengdu 610041, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Pan++Tang">Pan Tang</a></span></div><div class="loa-info-bio"><span><a class="internalNav" href="#ath1">View Biography</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Jifa Zhang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jifa Zhang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">State Key Laboratory of Biotherapy and Cancer Center, National Clinical Research Center for Geriatrics, West China Hospital of Sichuan University, Chengdu 610041, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jifa++Zhang">Jifa Zhang</a></span></div><div class="loa-info-bio"><span><a class="internalNav" href="#ath2">View Biography</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Jie Liu</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jie Liu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">State Key Laboratory of Biotherapy and Cancer Center, National Clinical Research Center for Geriatrics, West China Hospital of Sichuan University, Chengdu 610041, China</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#472b2e322d2e22757776760734243269222332692429"><span class="__cf_email__" data-cfemail="600c09150a090552505151201303154e0504154e030e">[email protected]</span></a> (J.L.).</span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jie++Liu">Jie Liu</a></span></div><div class="loa-info-bio"><span><a class="internalNav" href="#ath3">View Biography</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Cheng-Ming Chiang</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Cheng-Ming Chiang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Simmons Comprehensive Cancer Center, Department of Pharmacology, and Department of Biochemistry, University of Texas Southwestern Medical Center, Dallas, Texas 75390, United States</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#bbd8d3ded5dc96d6d2d5dc95d8d3d2dad5dcfbcecfc8d4cecfd3ccdec8cfdec9d595dedfce"><span class="__cf_email__" data-cfemail="53303b363d347e3e3a3d347d303b3a323d34132627203c26273b2436202736213d7d363726">[email protected]</span></a> (C.-M.C.).</span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Cheng-Ming++Chiang">Cheng-Ming Chiang</a></span></div><div class="loa-info-bio"><span><a class="internalNav" href="#ath4">View Biography</a></span></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Liang Ouyang</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Liang Ouyang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">State Key Laboratory of Biotherapy and Cancer Center, National Clinical Research Center for Geriatrics, West China Hospital of Sichuan University, Chengdu 610041, China</div></div><span class="conrtib-corresp"><strong>*</strong>E-mail: <a href="/cdn-cgi/l/email-protection#137c666a727d747f7a727d74536070663d7677663d707d">ouy<span class="__cf_email__" data-cfemail="f5949b92999c949b92b5869680db909180db969b">[email protected]</span></a> (L.O.).</span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Liang++Ouyang">Liang Ouyang</a></span></div><div class="loa-info-bio"><span><a class="internalNav" href="#ath5">View Biography</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-5537-8834" title="Orcid link">http://orcid.org/0000-0001-5537-8834</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c01487&amp;href=/doi/10.1021%2Facs.jmedchem.0c01487" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2021</span></span><span class="cit-volume">, 64</span><span class="cit-issue">, 5</span><span class="cit-pageRange">, 2419–2435</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">February 22, 2021</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>26 August 2020</li><li><span class="item_label"><b>Published</b> online</span>22 February 2021</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 11 March 2021</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.0c01487" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c01487</a></div><div class="article_header-article-copyright"><strong>Copyright © 2021 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D2419%26pageCount%3D17%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DPan%2BTang%252C%2BJifa%2BZhang%252C%2BJie%2BLiu%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D64%26issueNum%3D5%26contentID%3Dacs.jmedchem.0c01487%26title%3DTargeting%2BBromodomain%2Band%2BExtraterminal%2BProteins%2Bfor%2BDrug%2BDiscovery%253A%2BFrom%2BCurrent%2BProgress%2Bto%2BTechnological%2BDevelopment%26numPages%3D17%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D2435%26publicationDate%3DMarch%2B2021">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.0c01487"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">2177</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">5</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.0c01487" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Targeting Bromodomain and Extraterminal Proteins for Drug Discovery: From Current Progress to Technological Development&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Pan&quot;,&quot;last_name&quot;:&quot;Tang&quot;},{&quot;first_name&quot;:&quot;Jifa&quot;,&quot;last_name&quot;:&quot;Zhang&quot;},{&quot;first_name&quot;:&quot;Jie&quot;,&quot;last_name&quot;:&quot;Liu&quot;},{&quot;first_name&quot;:&quot;Cheng-Ming&quot;,&quot;last_name&quot;:&quot;Chiang&quot;},{&quot;first_name&quot;:&quot;Liang&quot;,&quot;last_name&quot;:&quot;Ouyang&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2021&quot;,&quot;month&quot;:&quot;02&quot;,&quot;day&quot;:&quot;22&quot;,&quot;issue&quot;:&quot;5&quot;,&quot;volume&quot;:&quot;64&quot;,&quot;pages&quot;:&quot;2419-2435&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.0c01487&quot;},&quot;abstract&quot;:&quot;Bromodomain and extraterminal (BET) proteins bind acetylated lysine residues in histones and nonhistone proteins via tandem bromodomains and regulate chromatin dynamics, cellular processes, and disease procession. Thus targeting BET proteins is a promising strategy for treating various diseases, especially malignant tumors and chronic inflammation. Many pan-BET small-molecule inhibitors have been described, and some of them are in clinical evaluation. Nevertheless, the limited clinical efficacy of the current BET inhibitors is also evident and has inspired the development of new technologies to improve their clinical outcomes and minimize unwanted side effects. In this Review, we summarize the latest protein characteristics and biological functions of BRD4 as an example of BET proteins, analyze the clinical development status and preclinical resistance mechanisms, and discuss recent advances in BRD4-selective inhibitors, dual-target BET inhibitors, proteolysis targeting chimera degraders, and protein–protein &quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c01487&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c01487" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c01487&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c01487" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c01487&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c01487" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c01487&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c01487&amp;href=/doi/10.1021/acs.jmedchem.0c01487" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.0c01487" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.0c01487" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (5 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c01487&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.0c01487%26sid%3Dliteratum%253Aachs%26pmid%3D33616410%26genre%3Darticle%26aulast%3DTang%26date%3D2021%26atitle%3DTargeting%2BBromodomain%2Band%2BExtraterminal%2BProteins%2Bfor%2BDrug%2BDiscovery%253A%2BFrom%2BCurrent%2BProgress%2Bto%2BTechnological%2BDevelopment%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D64%26issue%3D5%26spage%3D2419%26epage%3D2435%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=292544" title="Targeting">Targeting</a>,</li><li><a href="/action/doSearch?ConceptID=292258" title="Cancer">Cancer</a>,</li><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=291793" title="Inhibition">Inhibition</a>,</li><li><a href="/action/doSearch?ConceptID=291711" title="Selectivity">Selectivity</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/64/5" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/jmcmar.2021.64.issue-5/20210311/jmcmar.2021.64.issue-5.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01487/20210305/images/medium/jm0c01487_0007.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01487/20210305/images/large/jm0c01487_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01487&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Bromodomain and extraterminal (BET) proteins bind acetylated lysine residues in histones and nonhistone proteins via tandem bromodomains and regulate chromatin dynamics, cellular processes, and disease procession. Thus targeting BET proteins is a promising strategy for treating various diseases, especially malignant tumors and chronic inflammation. Many pan-BET small-molecule inhibitors have been described, and some of them are in clinical evaluation. Nevertheless, the limited clinical efficacy of the current BET inhibitors is also evident and has inspired the development of new technologies to improve their clinical outcomes and minimize unwanted side effects. In this Review, we summarize the latest protein characteristics and biological functions of BRD4 as an example of BET proteins, analyze the clinical development status and preclinical resistance mechanisms, and discuss recent advances in BRD4-selective inhibitors, dual-target BET inhibitors, proteolysis targeting chimera degraders, and protein–protein interaction inhibitors.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">1.  Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_50338" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_50338" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">In humans, 61 bromodomains, each composed of ∼110 amino acids forming four antiparallel α helices (αZ, αA, αB, and αC) and two hydrophobic (ZA and BC) loops, in 46 different proteins have been described.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> Among these, bromodomain and extraterminal (BET) proteins constitute a unique group with four family members, bromodomain-containing protein 4 (BRD4), BRD3, BRD2, and testis-specific BRDT, featuring the presence of two N-terminal tandem bromodomains (BD1 and BD2) and an ET (extraterminal) domain (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>A).<a onclick="showRef(event, 'ref2 ref3'); return false;" href="javascript:void(0);" class="ref ref2 ref3">(2,3)</a> Notably, BRD4 and BRDT contain an extended C-terminal motif (CTM) not found in BRD2 and BRD3. BET proteins are epigenetic “readers” binding specifically to <i>N</i>-ε-acetyllysine (Kac) residues in histones and nonhistone proteins via their BD1 and BD2 bromodomains to engage in chromatin dynamics and transcriptional regulation.<a onclick="showRef(event, 'ref4 ref5'); return false;" href="javascript:void(0);" class="ref ref4 ref5">(4,5)</a> However, the molecular distinction underlying the functional redundancy and unique aspects of each BET protein, with respect to their acetylated histone and nonhistone protein recognition, remains elusive.<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> Here we summarize the structural and functional characteristics of BRD4, which is the most understood member of the BET family, as an example of BET proteins.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01487/20210305/images/medium/jm0c01487_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01487/20210305/images/large/jm0c01487_0001.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. (A) Domain organization of human BET proteins. (B) Conserved sequence of BRD4 BD1 and BRD4 BD2. Key varying residues are marked in different colors. (C) Recruitment of P-TEFb facilitates BRD4-enhanced Pol II elongation. BRD4 binds to Kac sites on histones and recruits BRD4 interactors and the P-TEFb complex, leading to enhanced transcriptional elongation by Ser2-phosphorylated Pol II. (D) BRD4 function in DNA damage response in both NHEJ and HR pathways.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01487/20210305/images/large/jm0c01487_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01487&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">BRD4 has three characterized protein isoforms with different C-terminal amino acid residues—one long (L) and two short isoforms, S(a) and S(b)—resulting from alternative mRNA splicing.<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> Amino acids 1–719 are common among the three isoforms, including two highly conserved N-terminal tandem bromodomains (BD1 and BD2), the N-terminal cluster of phosphorylation sites (NPS), the basic residue-enriched interaction domain (BID), the extraterminal (ET) domain, and the C-terminal cluster of phosphorylation sites (CPS; see <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>A). Interestingly, the conformation of BD2 in a closed or open configuration is controlled by the downstream NPS, whose extent of phosphorylation modulated by casein kinase II (CK2)<a onclick="showRef(event, 'ref8 ref9'); return false;" href="javascript:void(0);" class="ref ref8 ref9">(8,9)</a> and protein phosphatase 2A (PP2A)<a onclick="showRef(event, 'ref10 ref11'); return false;" href="javascript:void(0);" class="ref ref10 ref11">(10,11)</a> regulates BRD4 association with different transcriptional regulators such as p53, AP-1, NF-κB, and EN1.<a onclick="showRef(event, 'ref7 ref10 ref12'); return false;" href="javascript:void(0);" class="ref ref7 ref10 ref12">(7,10,12)</a> Although BET bromodomains are structurally conserved in general, the ZA and BC loops between BD1 and BD2 are slightly different in sequence and length, thus contributing to the context-specific binding of acetylated lysine residues (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>B). Sequence differences between BD1 and BD2 provide a structural basis for designing selective inhibitors of BD1 or BD2. Besides sequence variation, BD1 and BD2 also exhibit functional differences in which BD1 primarily serves as a chromatin-binding module and BD2 is typically used for factor recruitment,<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> hence partly accounting for BD1 being indispensable for steady-state gene expression and both BD1 and BD2 being seemingly required for the acute gene response induced by inflammatory stimulation.<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> In addition, the conserved BID and phospho-NPS each interact directly with p53 but with different functional outcomes in regulating p53 binding to its target sequences.<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> The ET domain of BRD4 interacts with select chromatin regulators via specific structural contacts.<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> A unique CTM present in BRD4-L but absent in BRD4-S(a) and BRD4-S(b) functions to recruit transcriptional regulators, such as positive transcription elongation factor b (P-TEFb)<a onclick="showRef(event, 'ref16 ref17'); return false;" href="javascript:void(0);" class="ref ref16 ref17">(16,17)</a> and human papillomavirus (HPV)-encoded E2 protein that is capable of inhibiting AP-1-dependent HPV transcription via specific E2-binding sites.<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> Both BRD4-L and BRD4-S(a) can induce liquid–liquid phase separation (LLPS) <i>in vitro</i> and in the cell to mediate transcription complex assembly and regulate gene transcription.<a onclick="showRef(event, 'ref18 ref19 ref20'); return false;" href="javascript:void(0);" class="ref ref18 ref19 ref20">(18−20)</a> Recent research reveals that the expression level of BRD4-S(a), rather than BRD4-L, is closely related to the number and size of BRD4 puncta formed in the cell.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> An oncogenic function of BRD4-S(a) and a tumor-suppressive activity of BRD4-L recently revealed in breast cancer development further indicate the importance of target selectivity in anti-BRD4 drug development.<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a></div><div class="NLM_p">BRD4 interacts with both cyclin T1 (cycT1) and cyclin-dependent kinase 9 (CDK9) subunits of the dimeric P-TEFb complex<a onclick="showRef(event, 'ref8 ref16 ref21'); return false;" href="javascript:void(0);" class="ref ref8 ref16 ref21">(8,16,21)</a> and recruits transcription factors such as MYC,<a onclick="showRef(event, 'ref22 ref23'); return false;" href="javascript:void(0);" class="ref ref22 ref23">(22,23)</a> p53,<a onclick="showRef(event, 'ref8 ref24'); return false;" href="javascript:void(0);" class="ref ref8 ref24">(8,24)</a> AP-1,<a onclick="showRef(event, 'ref7 ref8 ref10'); return false;" href="javascript:void(0);" class="ref ref7 ref8 ref10">(7,8,10)</a> NF-κB,<a onclick="showRef(event, 'ref10 ref25'); return false;" href="javascript:void(0);" class="ref ref10 ref25">(10,25)</a> TWIST,<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> SNAIL,<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> EN1,<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> YAP1,<a onclick="showRef(event, 'ref28 ref29'); return false;" href="javascript:void(0);" class="ref ref28 ref29">(28,29)</a> TEAD4,<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> and NF-YA<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> to facilitate RNA polymerase II (Pol II)-dependent transcription (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>C).<a onclick="showRef(event, 'ref13 ref30 ref31'); return false;" href="javascript:void(0);" class="ref ref13 ref30 ref31">(13,30,31)</a> BRD4 is also implicated in DNA repair to control genome stability and cancer development. The role of BRD4 in the DNA damage response is not limited to nonhomologous end joining (NHEJ)<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> and also involves homologous recombination (HR)<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>D). BRD4 is associated with R-loop formation, which enhances double-strand break (DSB) formation and engages in the activation of the NHEJ pathway implicated in immunoglobulin class switch recombination<a onclick="showRef(event, 'ref32 ref33'); return false;" href="javascript:void(0);" class="ref ref32 ref33">(32,33)</a> and oncogenic gene rearrangement.<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> In contrast, the inhibition of BRD4 induces the loss of C-terminal binding protein-interacting protein (CtIP) involved in DNA end resection and single-strand DNA (ssDNA) generation, resulting in a decrease in the HR competency.<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> The involvement of BRD4 in cancer initiation and progression can be attributed to chromosomal translocation leading to the formation of oncogenic fusion drivers (e.g., BRD4-NUT)<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> through epigenetic control via the recruitment of cell-type-specific transcriptional regulators<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> or by modulating the activity<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> or recruitment of cell-cycle-promoting factors such as CDK4/6.<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> In NUT midline carcinoma (NMC), BRD4-NUT fusion oncoprotein induces the hyperacetylation of chromatin and the formation of a unique nuclear subcompartment that squelches key transcription components or chromatin regulators, leading to the inhibition of transcription initiation by RNA Pol II.<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> Another study identified CDK9 as a kinase implicated in BRD4 hyperphosphorylation that contributes to its oncogenic activity,<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> consistent with the finding of enhanced BRD4 phosphorylation due to the reduced PP2A activity during triple-negative breast cancer progression.<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> In addition, BRD4 is also involved in blastic plasmacytoid dendritic cell neoplasm (BPDCN),<a onclick="showRef(event, 'ref41 ref42'); return false;" href="javascript:void(0);" class="ref ref41 ref42">(41,42)</a> Cornelia-de-Lange-like syndrome,<a onclick="showRef(event, 'ref43'); return false;" href="javascript:void(0);" class="ref ref43">(43)</a> and fragile X syndrome.<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a> Expanded research on BRD4 protein isoforms with gain- and loss-of-function approaches conducted in various breast cancer cells and mouse models, in conjunction with genome-wide transcriptome and binding profiling, unequivocally establishes the opposing functions of BRD4 isoforms in breast cancer and significantly enhances our understanding of gene targets and pathways associated with cancer progression.<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a></div><div class="NLM_p last">Over the past decade, numerous pan-BET inhibitors have been generated, and some of them are currently in clinical trials, such as OTX-015, CPI-0610, and I-BET762.<a onclick="showRef(event, 'ref45'); return false;" href="javascript:void(0);" class="ref ref45">(45)</a> Although small-molecule pan-BET inhibitors show promising effects in clinical evaluation, there are still problems and challenges to overcome, such as the moderate clinical efficacy and the occurrence of preclinical resistance.<a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a> Encouragingly, with recent advances in pharmaceutical chemistry and chemical biology, new technologies that facilitate the development of more selective inhibitors or other new-generation inhibitors targeting specific functional aspects of BET proteins have been used to assuage some clinical concerns.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">2.  Current Clinical Development Status and Observations</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_17366" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_17366" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">In 2017, pan-BET inhibitors available at that time were reviewed in detail regarding their chemical types, activities, and therapeutic potentials.<a onclick="showRef(event, 'ref45'); return false;" href="javascript:void(0);" class="ref ref45">(45)</a> At present, some pan-BET inhibitors are in clinical trials for treating different cancers (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). However, the clinical advancement of most pan-BET inhibitors is still in the early stages and mainly focuses on cancer treatment. The results of clinical trials published to date raise several key issues to be considered. First, some inhibitors display toxicity during clinical trials, including dose-limiting toxicity (DLT). The most common adverse effects are thrombocytopenia, diarrhea, fatigue, vomiting, anemia, and hyperbilirubinemia. A commonly observed DLT is thrombocytopenia with undamaged platelet function. Reduced platelet counts were reversible and could be restored within 1 week after drug withdrawal.<a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a> In addition, complications and immunodeficiency are potential undesirable reactions to the use of pan-BET inhibitors.<a onclick="showRef(event, 'ref47'); return false;" href="javascript:void(0);" class="ref ref47">(47)</a> The phase I trial of BAY1238097 (NCT02369029) was terminated early due to severe toxic effects, and INCB057643 (NCT02711137) was also discontinued because of safety concerns. Second, several inhibitors exhibited much lower antitumor activity than those observed in preclinical trials. In a dose study of CPI-0610 given to 44 patients with relapsed or refractory lymphoma, only 2 showed a complete response, and 1 had a partial response. In addition, studies with OTX-015 (NCT02698176 and NCT02296476), as reported for I-BET151 (NCT02630251), were forced to terminate because of its limited efficacy in solid tumors. Pan-BET inhibitors show a significant survival advantage in tumor models, which are often abrogated in transplants of drug-resistant cells.<a onclick="showRef(event, 'ref48'); return false;" href="javascript:void(0);" class="ref ref48">(48)</a> The main reason for the lack of clinical efficacy is likely due to a failure to identify the right group of patients most likely to benefit from pan-BET inhibitor treatment, resulting in ineffectual dosing before DLT occurs. Moreover, modest outcomes may be partly attributed to the development of resistance, diminishing the clinical response to pan-BET inhibitors.<a onclick="showRef(event, 'ref49'); return false;" href="javascript:void(0);" class="ref ref49">(49)</a></div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Pan-BET Inhibitors in Cancer-Related Clinical Trials<a class="ref internalNav" href="#tbl1-fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">drug</th><th class="colsep0 rowsep0" align="center">sponsor</th><th class="colsep0 rowsep0" align="center">phase</th><th class="colsep0 rowsep0" align="center">indication</th><th class="colsep0 rowsep0" align="center">NUT identifier</th><th class="colsep0 rowsep0" align="center">status</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" rowspan="9" align="left">I-BET762</td><td class="colsep0 rowsep0" rowspan="8" align="left">GlaxoSmithKline</td><td class="colsep0 rowsep0" align="left">I</td><td class="colsep0 rowsep0" align="left">NMC, other solid tumors</td><td class="colsep0 rowsep0" align="left">NCT01587703</td><td class="colsep0 rowsep0" align="left">completed</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">I</td><td class="colsep0 rowsep0" align="left">CRPC</td><td class="colsep0 rowsep0" align="left">NCT03150056</td><td class="colsep0 rowsep0" align="left">active, not recruiting</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">I</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">NCT02706535</td><td class="colsep0 rowsep0" align="left">completed</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">I</td><td class="colsep0 rowsep0" align="left">solid tumors, lymphomas</td><td class="colsep0 rowsep0" align="left">NCT03925428</td><td class="colsep0 rowsep0" align="left">withdrawn</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">NMC</td><td class="colsep0 rowsep0" align="left">NCT03702036</td><td class="colsep0 rowsep0" align="left">no longer available</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">II</td><td class="colsep0 rowsep0" align="left">hematological malignancies</td><td class="colsep0 rowsep0" align="left">NCT01943851</td><td class="colsep0 rowsep0" align="left">completed</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">II</td><td class="colsep0 rowsep0" align="left">HR+/HER2– advanced or metastatic breast cancer</td><td class="colsep0 rowsep0" align="left">NCT02964507</td><td class="colsep0 rowsep0" align="left">active, not recruiting</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">II</td><td class="colsep0 rowsep0" align="left">solid tumor</td><td class="colsep0 rowsep0" align="left">NCT03266159</td><td class="colsep0 rowsep0" align="left">withdrawn</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">National Cancer Institute</td><td class="colsep0 rowsep0" align="left">I/II</td><td class="colsep0 rowsep0" align="left">NUT carcinoma</td><td class="colsep0 rowsep0" align="left">NCT04116359</td><td class="colsep0 rowsep0" align="left">withdrawn</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" rowspan="6" align="left">OTX015</td><td class="colsep0 rowsep0" rowspan="2" align="left">Merck Sharp & Dohme Corp</td><td class="colsep0 rowsep0" align="left">I</td><td class="colsep0 rowsep0" align="left">NMC, TNBC, NSCLC, CRPC</td><td class="colsep0 rowsep0" align="left">NCT02698176</td><td class="colsep0 rowsep0" align="left">terminated</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">I</td><td class="colsep0 rowsep0" align="left">AML, DLBCL</td><td class="colsep0 rowsep0" align="left">NCT02698189</td><td class="colsep0 rowsep0" align="left">active, not recruiting</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" rowspan="4" align="left">Oncoethix GmbH</td><td class="colsep0 rowsep0" align="left">I</td><td class="colsep0 rowsep0" align="left">NMC, TNBC, NSCLC, CRPC</td><td class="colsep0 rowsep0" align="left">NCT02259114</td><td class="colsep0 rowsep0" align="left">completed</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">I/II</td><td class="colsep0 rowsep0" align="left">AML</td><td class="colsep0 rowsep0" align="left">NCT02303782</td><td class="colsep0 rowsep0" align="left">withdrawn</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">I</td><td class="colsep0 rowsep0" align="left">AML, DLBCL, ALL, MM</td><td class="colsep0 rowsep0" align="left">NCT01713582</td><td class="colsep0 rowsep0" align="left">completed</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">II</td><td class="colsep0 rowsep0" align="left">GBM</td><td class="colsep0 rowsep0" align="left">NCT02296476</td><td class="colsep0 rowsep0" align="left">terminated</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" rowspan="2" align="left">TEN-010</td><td class="colsep0 rowsep0" rowspan="2" align="left">Hoffmann-La Roche</td><td class="colsep0 rowsep0" align="left">I</td><td class="colsep0 rowsep0" align="left">AML, myelodysplastic syndromes</td><td class="colsep0 rowsep0" align="left">NCT02308761</td><td class="colsep0 rowsep0" align="left">completed</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">I</td><td class="colsep0 rowsep0" align="left">solid tumors, AST</td><td class="colsep0 rowsep0" align="left">NCT01987362</td><td class="colsep0 rowsep0" align="left">completed</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Bay1238097</td><td class="colsep0 rowsep0" align="left">Bayer</td><td class="colsep0 rowsep0" align="left">I</td><td class="colsep0 rowsep0" align="left">solid tumors, lymphoma</td><td class="colsep0 rowsep0" align="left">NCT02369029</td><td class="colsep0 rowsep0" align="left">terminated</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" rowspan="4" align="left">CPI-0610</td><td class="colsep0 rowsep0" align="left">University of Texas Southwestern Medical Center</td><td class="colsep0 rowsep0" align="left">II</td><td class="colsep0 rowsep0" align="left">malignant peripheral nerve sheath tumors</td><td class="colsep0 rowsep0" align="left">NCT02986919</td><td class="colsep0 rowsep0" align="left">withdrawn</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" rowspan="3" align="left">Constellation Pharmaceuticals</td><td class="colsep0 rowsep0" align="left">I</td><td class="colsep0 rowsep0" align="left">previously treated MM</td><td class="colsep0 rowsep0" align="left">NCT02157636</td><td class="colsep0 rowsep0" align="left">completed</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">I</td><td class="colsep0 rowsep0" align="left">progressive lymphoma</td><td class="colsep0 rowsep0" align="left">NCT01949883</td><td class="colsep0 rowsep0" align="left">completed</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">II</td><td class="colsep0 rowsep0" align="left">myelofibrosis, MDS/MPN, U</td><td class="colsep0 rowsep0" align="left">NCT02158858</td><td class="colsep0 rowsep0" align="left">recruiting</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">I-BET151</td><td class="colsep0 rowsep0" align="left">GlaxoSmithKline</td><td class="colsep0 rowsep0" align="left">I</td><td class="colsep0 rowsep0" align="left">metastatic, unresectable solid tumors</td><td class="colsep0 rowsep0" align="left">NCT02630251</td><td class="colsep0 rowsep0" align="left">terminated</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" rowspan="2" align="left">INCB054329</td><td class="colsep0 rowsep0" rowspan="2" align="left">Incyte Corporation</td><td class="colsep0 rowsep0" align="left">I/II</td><td class="colsep0 rowsep0" align="left">advanced-stage cancer</td><td class="colsep0 rowsep0" align="left">NCT02431260</td><td class="colsep0 rowsep0" align="left">terminated</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">metastatic renal cell carcinoma</td><td class="colsep0 rowsep0" align="left">NCT03896815</td><td class="colsep0 rowsep0" align="left">no longer available</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" rowspan="3" align="left">INCB057643</td><td class="colsep0 rowsep0" rowspan="3" align="left">Incyte Corporation</td><td class="colsep0 rowsep0" align="left">I/II</td><td class="colsep0 rowsep0" align="left">advanced-stage cancer</td><td class="colsep0 rowsep0" align="left">NCT02711137</td><td class="colsep0 rowsep0" align="left">terminated</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">I</td><td class="colsep0 rowsep0" align="left">primary myelofibrosis, secondary myelofibrosis</td><td class="colsep0 rowsep0" align="left">NCT04279847</td><td class="colsep0 rowsep0" align="left">not yet recruiting</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">I/II</td><td class="colsep0 rowsep0" align="left">solid tumors, advanced malignancies, metastatic cancer</td><td class="colsep0 rowsep0" align="left">NCT02959437</td><td class="colsep0 rowsep0" align="left">active, not recruiting</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" rowspan="2" align="left">ADZ5153</td><td class="colsep0 rowsep0" align="left">AstraZeneca</td><td class="colsep0 rowsep0" align="left">I</td><td class="colsep0 rowsep0" align="left">malignant solid tumors, lymphoma, ovarian cancer</td><td class="colsep0 rowsep0" align="left">NCT03205176</td><td class="colsep0 rowsep0" align="left">recruiting</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Acerta Pharma BV|AstraZeneca</td><td class="colsep0 rowsep0" align="left">I</td><td class="colsep0 rowsep0" align="left">NHL, DLBCL, non-Hodgkin’s lymphoma</td><td class="colsep0 rowsep0" align="left">NCT03527147</td><td class="colsep0 rowsep0" align="left">recruiting</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">ABBV-075</td><td class="colsep0 rowsep0" rowspan="2" align="left">AbbVie</td><td class="colsep0 rowsep0" align="left">I</td><td class="colsep0 rowsep0" align="left">AST</td><td class="colsep0 rowsep0" align="left">NCT02391480</td><td class="colsep0 rowsep0" align="left">completed</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">ABBV-744</td><td class="colsep0 rowsep0" align="left">I</td><td class="colsep0 rowsep0" align="left">CRPC, AML</td><td class="colsep0 rowsep0" align="left">NCT03360006</td><td class="colsep0 rowsep0" align="left">recruiting</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" rowspan="2" align="left">BMS-986158</td><td class="colsep0 rowsep0" align="left">Dana-Farber Cancer Institute</td><td class="colsep0 rowsep0" align="left">I</td><td class="colsep0 rowsep0" align="left">solid tumor, lymphoma, brain tumor (pediatric)</td><td class="colsep0 rowsep0" align="left">NCT03936465</td><td class="colsep0 rowsep0" align="left">recruiting</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Bristol-Myers Squibb</td><td class="colsep0 rowsep0" align="left">I/II</td><td class="colsep0 rowsep0" align="left">AST</td><td class="colsep0 rowsep0" align="left">NCT02419417</td><td class="colsep0 rowsep0" align="left">recruiting</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" rowspan="2" align="left">PLX51107</td><td class="colsep0 rowsep0" align="left">M.D. Anderson Cancer Center</td><td class="colsep0 rowsep0" align="left">I</td><td class="colsep0 rowsep0" align="left">AML, myelodysplastic syndrome, MDS/MPN, U</td><td class="colsep0 rowsep0" align="left">NCT04022785</td><td class="colsep0 rowsep0" align="left">recruiting</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Plexxikon</td><td class="colsep0 rowsep0" align="left">I</td><td class="colsep0 rowsep0" align="left">solid tumors, AML, myelodysplastic syndrome, NHL</td><td class="colsep0 rowsep0" align="left">NCT02683395</td><td class="colsep0 rowsep0" align="left">terminated</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">BI 894999</td><td class="colsep0 rowsep0" align="left">Boehringer Ingelheim</td><td class="colsep0 rowsep0" align="left">I</td><td class="colsep0 rowsep0" align="left">neoplasms</td><td class="colsep0 rowsep0" align="left">NCT02516553</td><td class="colsep0 rowsep0" align="left">recruiting</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" rowspan="4" align="left">ZEN003694</td><td class="colsep0 rowsep0" rowspan="4" align="left">Zenith Epigenetics</td><td class="colsep0 rowsep0" align="left">I/II</td><td class="colsep0 rowsep0" align="left">MCRPC</td><td class="colsep0 rowsep0" align="left">NCT04145375</td><td class="colsep0 rowsep0" align="left">enrolling by invitation</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">I</td><td class="colsep0 rowsep0" align="left">TNBC</td><td class="colsep0 rowsep0" align="left">NCT03901469</td><td class="colsep0 rowsep0" align="left">recruiting</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">II</td><td class="colsep0 rowsep0" align="left">MCRPC</td><td class="colsep0 rowsep0" align="left">NCT02705469</td><td class="colsep0 rowsep0" align="left">completed</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">I/II</td><td class="colsep0 rowsep0" align="left">MCRPC</td><td class="colsep0 rowsep0" align="left">NCT02711956</td><td class="colsep0 rowsep0" align="left">active, not recruiting</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" rowspan="2" align="left">GS-5829</td><td class="colsep0 rowsep0" rowspan="2" align="left">Gilead Sciences</td><td class="colsep0 rowsep0" align="left">I/II</td><td class="colsep0 rowsep0" align="left">metastatic CRPC</td><td class="colsep0 rowsep0" align="left">NCT02607228</td><td class="colsep0 rowsep0" align="left">completed</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">I</td><td class="colsep0 rowsep0" align="left">solid tumors, lymphomas</td><td class="colsep0 rowsep0" align="left">NCT02392611</td><td class="colsep0 rowsep0" align="left">completed</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">FT-1101</td><td class="colsep0 rowsep0" align="left">Forma Therapeutics</td><td class="colsep0 rowsep0" align="left">I</td><td class="colsep0 rowsep0" align="left">AML, MDS</td><td class="colsep0 rowsep0" align="left">NCT02543879</td><td class="colsep0 rowsep0" align="left">completed</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-tbl1-fn1"><div class="footnote" id="tbl1-fn1"><sup><sup>a</sup></sup><p class="last">Sourced from <a href="https://clinicaltrials.gov/" class="extLink">https://clinicaltrials.gov/</a> (January 2021).</p></div></div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i5">3.  Resistance Mechanisms</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_66373" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_66373" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">In related research of BRD4, problems with preclinical drug resistance have attracted significant attention. In drug resistance, the activation of alternative or compensatory pathways such as WNT/β-catenin and hedgehog signaling can maintain MYC expression independently of BRD4 (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>).<a onclick="showRef(event, 'ref48 ref49 ref50'); return false;" href="javascript:void(0);" class="ref ref48 ref49 ref50">(48−50)</a> In pan-BET inhibitor-resistant leukemia cells, genome-wide BRD4 binding is decreased upon BET inhibitor treatment, but alternative binding by β-catenin to key oncogenic sites or nearby WNT-responsive enhancers maintains MYC expression.<a onclick="showRef(event, 'ref48'); return false;" href="javascript:void(0);" class="ref ref48">(48)</a> In addition, the activation of the hedgehog signaling pathway is another mechanism to restore MYC expression through altered transcription programming.<a onclick="showRef(event, 'ref49'); return false;" href="javascript:void(0);" class="ref ref49">(49)</a></div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01487/20210305/images/medium/jm0c01487_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01487/20210305/images/large/jm0c01487_0002.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Resistance mechanisms.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01487/20210305/images/large/jm0c01487_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01487&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The phosphorylation of BRD4, which is crucial for chromatin binding and factor recruitment,<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> also has a great effect on the drug resistance, as shown in triple-negative breast cancer (TNBC) cells.<a onclick="showRef(event, 'ref11 ref12'); return false;" href="javascript:void(0);" class="ref ref11 ref12">(11,12)</a> Significantly increased BRD4 phosphorylation (or hyperphosphorylation) due to reduced PP2A activity results in a stronger association of BRD4 with the MED1 component of the mediator complex that recruits RNA Pol II to the promoter region, thereby promoting bromodomain-independent gene activation via an alternative protein–protein (i.e., BRD4–mediator) recruitment mechanism.<a onclick="showRef(event, 'ref11 ref51'); return false;" href="javascript:void(0);" class="ref ref11 ref51">(11,51)</a> BRD4 hyperphosphorylation has also been observed in NMC to support its oncogenic activity.<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a></div><div class="NLM_p">Furthermore, adapted apoptotic signaling through AMPK/ULK1-mediated autophagy represents another resistance mechanism to BRD4 inhibitors in certain JQ1-resistant acute myelogenous leukemia (AML) cells, where reduced apoptosis is achieved by JQ1-induced prosurvival autophagy, as the pharmacological inhibition of AMPK leads to diminished ULK1-mediated autophagy and increased apoptosis.<a onclick="showRef(event, 'ref52'); return false;" href="javascript:void(0);" class="ref ref52">(52)</a></div><div class="NLM_p">Conceivably, the activity of the E3 ubiquitin ligase adaptor SPOP (speckle-type POZ protein) and deubiquitinase DUB3 that directly modulate the BRD4 protein stability may potentially account for the drug resistance, as a deficiency in SPOP binding to BRD4 results in the enhanced BRD4-dependent expression of RAC1 GTPase and the activation of cholesterol biosynthetic enzymes and AKT-mTORC1 to mediate intrinsic resistance.<a onclick="showRef(event, 'ref53 ref54 ref55'); return false;" href="javascript:void(0);" class="ref ref53 ref54 ref55">(53−55)</a> Likewise, a mutation in NCOR2 that negatively regulates DUB3 gene transcription frequently observed in castration-resistant prostate cancer (CRPC) leads to enhanced DUB3 and BRD4 protein levels,<a onclick="showRef(event, 'ref56'); return false;" href="javascript:void(0);" class="ref ref56">(56)</a> thus contributing to resistance to BRD4 inhibitor treatment. Moreover, a mutation in PI3KCA has also been identified as a resistance determinant to BRD4 inhibitors based on whole-genome shRNA dropout screens performed in 77 breast cancer cell lines.<a onclick="showRef(event, 'ref57'); return false;" href="javascript:void(0);" class="ref ref57">(57)</a></div><div class="NLM_p last">Notably, in preclinical models, the complexity of drug resistance with diverse factors and pathways involved greatly diminishes the clinical responses. It is indeed a significant challenge to overcome resistance to BET inhibitors, and no compounds targeting specific gene mutation have thus far been reported.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i6">4.  Development of Next-Generation Inhibitors</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_05364" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_05364" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Poor and short-term clinical responses have become a significant issue in the use of pan-BET small-molecule inhibitors for the clinical treatment of cancer. Thus higher and stricter requirements for the development and rational use of inhibitors are essential. New technologies have been developed to facilitate the discovery of novel inhibitors, such as BD1-, BD2-, or BRD4-selective inhibitors, BET proteolysis targeting chimeras (PROTACs), dual-target inhibitors, and protein–protein interaction (PPI) inhibitors. Among these inhibitors, pan-BET inhibitors show no selectivity against specific BET protein isoforms or bromodomains, BD1- and BD2-selective inhibitors imply pan-BET activity toward BRD2/3/4/T but with selectivity for specific bromodomains, and BRD4-selective inhibitors mean target preference for BRD4 rather than BRD2/3/T but without bromodomain selectivity. A recent report depicting functional differences between BD1 and BD2 highlights the potential for developing BD1-specific inhibitors in cancer therapy and BD2-specific inhibitors in immunoinflammation that may be more favorable compared with nonselective BET bromodomain targeting.<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> Dual-target inhibitors with bimolecular binding to two different proteins, such as dual BET/kinase inhibitors, display either synergistic effects or synthetic lethality that may reduce the drug resistance frequently seen with monotherapy during cancer treatment. PROTACs targeting BET proteins that allow the prompt removal of a large amount of BET proteins in the cell show strong antitumor activity in preclinical studies. Undoubtedly, PPI inhibitors blocking BRD4 interaction with other proteins offer another line of new approaches to drug discovery and cancer treatment. The development of these more selective BET inhibitors with alternative mechanisms of action is of vital importance to overcome the limited clinical efficacy and off-target toxicities often associated with the use of pan-BET inhibitors.</div><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i7" class="anchor-spacer"></div><h3 class="article-section__title" id="_i7">4.1.  BD1-Selective Inhibitors</h3><div class="NLM_p">BD1 and BD2 each consist of four antiparallel α-helixes (αZ, αA, αB, and αC) with different lengths linked by two hydrophobic loops (ZA and BC) that together form a hydrophobic Kac recognition pocket. Amino acid differences in the binding pockets of BD1 and BD2 lead to narrower binding regions and differences in the polarity and hydrophobicity of BD2. Poor selectivity among BET proteins and between BD1 and BD2 has incurred unwanted side effects and reduced the efficacy of BET bromodomain inhibitors in clinical trials.<a onclick="showRef(event, 'ref58'); return false;" href="javascript:void(0);" class="ref ref58">(58)</a> Thus selectively targeting only BD1 or BD2 in BRD4 can provide better specificity and avoid unnecessary bioreactions triggered by the other domain inhibition as well as poor clinical evaluation due to the concurrent targeting of both BD1 and BD2 with differences in functions and structures. Gilan et al. have developed respective BD1- and BD2-specific inhibitors for better disease treatment and efficacy based on the functional and structural differences between BD1 and BD2.<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> Because BD1 and BD2 within the same protein are structurally and functionally more diverse compared with the analogous BD1 or BD2 found in different BET proteins,<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a> it is in fact easier to develop inhibitors selective for BD1 or BD2 than for individual BET family members.</div><div class="NLM_p">Many BD1-selective inhibitors have been discovered, including Olinone (<b>1</b>)<a onclick="showRef(event, 'ref59 ref60'); return false;" href="javascript:void(0);" class="ref ref59 ref60">(59,60)</a> and MS436 (<b>2</b>),<a onclick="showRef(event, 'ref60'); return false;" href="javascript:void(0);" class="ref ref60">(60)</a> with the 3D crystal structure of the latter with BRD4-BD1 also shown on the right (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>A). ZL0580 (<b>3</b>) (IC<sub>50</sub> = 163 nM), a BRD4 BD1-selective inhibitor, was discovered by structure-guided drug design.<a onclick="showRef(event, 'ref61'); return false;" href="javascript:void(0);" class="ref ref61">(61)</a> Overlay analysis reveals that <b>3</b> can access the Kac-binding pocket in BD1 with binding notably different from that of JQ1. Compound <b>3</b> induces the epigenetic suppression of HIV in multiple <i>in vitro</i> and <i>ex vivo</i> models. Another BD1-selective inhibitor, LT052 (<b>4</b>),<a onclick="showRef(event, 'ref62'); return false;" href="javascript:void(0);" class="ref ref62">(62)</a> with more than 100-fold selectivity for BRD4-BD1 over BRD4-BD2 and exhibiting a stable level in liver microsomal metabolic analysis inhibits monosodium urate (MSU)-induced apoptosis in human THP-1 monocytic cells and improves symptoms of acute gouty arthritis via BRD4/NF-κB/NLRP3 inflammasome signaling pathways. Chromone derivatives ZL0513 (<b>5</b>) and ZL0516 (<b>6</b>) both show high binding affinity for BD1 with a value of 67 and 84 nM, respectively. <i>In vivo</i> pharmacokinetics (PK) profiles between compounds <b>5</b> and <b>6</b> are similar (half life 3 to 4 h, maximum concentration ∼600 ng/mL, AUC ∼5000 ng·h/mL), and the oral bioavailability of compounds <b>5</b> and <b>6</b> of around 35–38% (20 mg/kg) is generally acceptable in rats.<a onclick="showRef(event, 'ref63'); return false;" href="javascript:void(0);" class="ref ref63">(63)</a> GSK778 (<b>7</b>), identified by the functional and structural comparison between BD1 and BD2 showing 130 times higher affinity for BD1 than BD2, inhibits not only the growth and vitality of human cancer cell lines but also the clone-forming ability of human primary AML cells.<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> Further chemical modification of compound <b>7</b> leads to another potent, selective, and cell-permeable inhibitor of BD1 (GSK789 (<b>8</b>)), displaying more than 1000-fold selectivity for BD1 over BD2 and achieving a submicromolar inhibition of monocyte chemoattractant protein-1 (MCP-1) release in human whole blood (pIC<sub>50</sub> = 6.5).<a onclick="showRef(event, 'ref64'); return false;" href="javascript:void(0);" class="ref ref64">(64)</a> Starting with I-BET151 guided by structural information leads to imidazoquinoline compound <b>9</b> with 150-fold BD1 selectivity over BD2.<a onclick="showRef(event, 'ref65'); return false;" href="javascript:void(0);" class="ref ref65">(65)</a> Compound <b>9</b> reduces the production of IL-6 and MCP-1 in LPS-stimulated human peripheral blood mononuclear cells (PBMCs) and whole blood and suppresses MV4-11 lymphoblast leukemia cell growth (pIC<sub>50</sub> = 7.0), indicating that BD1-selective inhibitor compound <b>9</b> is sufficient to maintain the anti-inflammatory and antiproliferative activity of pan-BET inhibition. Compound <b>10</b>, another BD1-selective inhibitor (28 times higher affinity for BRD4-BD1 than BRD4-BD2), displays similar selectivity against the BD2 of BRD2 and BRD3 (25 and 33 times, respectively).<a onclick="showRef(event, 'ref66'); return false;" href="javascript:void(0);" class="ref ref66">(66)</a> This study also reveals that the underlying BD1 selectivity lies in the flexibility of the BD1-conserved YNKP motif and the substitution of structured waters in the Kac-binding site, which, unlike typical pan-BET inhibitor-suppressed MYC expression, leads to enhanced MYC expression in compound <b>10</b>-treated MM.1S multiple myeloid cells. This unexpected result needs to be further investigated.</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01487/20210305/images/medium/jm0c01487_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01487/20210305/images/large/jm0c01487_0003.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Reported novel selective inhibitors <b>1</b>–<b>20</b>. (A) Chemical structures of BRD4 BD1-selective inhibitors. The docking pose of compounds <b>2</b> and <b>6</b> with BRD4–BD1 in ribbon representation is shown on the right. Key residues, including Pro82, Gln85, Lys91, Tyr97, and Asn140, are shown in green sticks, and the amide ketone of compound <b>2</b> interacts with the critical conserved residue Asn140 to form a hydrogen bond. Similarly, in the docking pose of compound <b>6</b>, key residues Asn140, Tyr97, Gln85, and Pro82 are green. The carbonyl O atom of compound <b>6</b> interacts with the conserved residue Asn140, forming a direct H bond, and with Tyr97, forming an indirect hydrogen bond mediated by a water molecule. (B) Reported BRD4 BD2-selective inhibitors. (C) Chemical structures of BRD4-selective inhibitors.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01487/20210305/images/large/jm0c01487_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01487&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i8" class="anchor-spacer"></div><h3 class="article-section__title" id="_i8">4.2.  BD2-Selective Inhibitors</h3><div class="NLM_p last">Representative BD2-selective inhibitors identified to date are shown in <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>B. The tetrahydroquinoxaline derivative GSK340 (<b>11</b>) with BD2 selectivity displays good solubility in both thermodynamic and kinetic solubility assays.<a onclick="showRef(event, 'ref67'); return false;" href="javascript:void(0);" class="ref ref67">(67)</a> In addition, compound <b>11</b> is cell-permeable and reduces the release of the MCP-1 cytokine in human PBMCs and whole blood. BY27 (<b>12</b>) is a potent BD2-selective inhibitor with 7 times higher selectivity for BD2 over BD1.<a onclick="showRef(event, 'ref68'); return false;" href="javascript:void(0);" class="ref ref68">(68)</a> In MV4-11 cells, compound <b>12</b> reduced the c-MYC protein level and increased the p21 expression in a dose-dependent manner. Compared with I-BET762, which caused 11% weight loss at 60 mg/kg, compound <b>12</b> is less toxic (exhibiting 7% weight loss at 150 mg/kg) at high doses and has 67% inhibition of tumor growth in a MV4-11 mouse xenograft model. Currently, ABBV-744 (<b>13</b>),<a onclick="showRef(event, 'ref69'); return false;" href="javascript:void(0);" class="ref ref69">(69)</a> a highly effective inhibitor of the BD2 domain with more than 300 times higher selectivity for BD2 over BD1, is synthesized and shows moderate preclinical PK profiles, including a moderate to high volume of distribution, a 2.0–4.4 h half life, and moderate clearance.<a onclick="showRef(event, 'ref70'); return false;" href="javascript:void(0);" class="ref ref70">(70)</a> The structural modification of an ABBV-075 analog leads to compound <b>13</b>. 2,6-Dimethylphenyl ether in compound <b>13</b> contributes to BD2 selectivity, with a tertiary alcohol on the central phenyl ring suppressing the activity against BRD4-BD1 and a fluorine atom on the phenyl ether improving the pharmacokinetic properties of compound <b>13</b>. In a myelodysplastic SKM-1 cell-derived xenograft mouse model, compound <b>13</b> displays 83% tumor growth inhibition with minimal (2%) weight loss over 21 days at a dose of 18.8 mg/kg, whereas ABBV-075 has similar tumor growth inhibition with 7% weight loss at the maximally tolerated dose of 1 mg/kg, suggesting that more selective BET inhibitors may improve the tolerability, albeit modestly. Compound <b>13</b> is currently in phase I clinical evaluation. GSK046 (iBET-BD2, <b>14</b>), although less effective than <b>8</b> in the human cancer cell lines examined, exhibits more than 300 times higher selectivity for BD2 over BD1 and appears to ameliorate inflammatory disease in preclinical models.<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> A recent study shows that compound <b>14</b> has physicochemical and pharmacokinetic properties suitable for use <i>in vivo</i>, and it retains the ability to effectively inhibit the release of MCP-1 cytokines (pIC<sub>50</sub> = 7.5) in the cellular and whole blood environment.<a onclick="showRef(event, 'ref71'); return false;" href="javascript:void(0);" class="ref ref71">(71)</a> Interestingly, the inhibition of BD1 or BD2 leads to the suppression of MCP-1 despite the fact that these two types of inhibitors have distinct selectivity profiles. Another study applying a template hopping approach based on the orthogonal template (compound <b>14</b>) leads to the synthesis of compounds GSK620 (<b>15</b>) and GSK549 (<b>16</b>) with high selectivity and low levels of inhibition against indicators of hepatoxicity.<a onclick="showRef(event, 'ref72'); return false;" href="javascript:void(0);" class="ref ref72">(72)</a> SJ018 (<b>17</b>) and SJ432 (<b>18</b>), two tetrahydroquinoline analogs, inhibit the expression of MYC and downstream targets but display no rebound effects, as compared with JQ1 in pediatric tumor cell lines, suggesting that BD2-selective inhibitors are effective candidates for the design of antitumor drugs for MYC-driven pediatric malignancies.<a onclick="showRef(event, 'ref73'); return false;" href="javascript:void(0);" class="ref ref73">(73)</a></div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i9" class="anchor-spacer"></div><h3 class="article-section__title" id="_i9">4.3.  BRD4-Selective Inhibitors</h3><div class="NLM_p last">Besides BD1- and BD2-selective inhibitors, the specific targeting of BRD4 rather than other BET proteins is also emerging. The ubiquitously expressed BRD2, BRD3, and BRD4 have a broad range of functions and participate in housekeeping and lineage-specific gene regulation in somatic and progenitor/stem cells by interacting with distinct partner proteins. BRD2 associates with E2F transcription factors to regulate E2F-dependent cell differentiation and proliferation genes and interacts with RNA Pol II to engage in transcription elongation in embryonic kidney cells through hyperacetylated nucleosomes due to the lack of a CTM.<a onclick="showRef(event, 'ref74 ref75 ref76'); return false;" href="javascript:void(0);" class="ref ref74 ref75 ref76">(74−76)</a> Similar to BRD2, BRD3 is mainly involved in the regulation of gene transcription related to the development of embryonic stem cells, but unlike BRD4, it interacts with RNA Pol II in a P-TEFb-independent manner.<a onclick="showRef(event, 'ref77'); return false;" href="javascript:void(0);" class="ref ref77">(77)</a> The germ-cell-specific BRDT associates with multiple interactors during spermatogenesis and oogenesis.<a onclick="showRef(event, 'ref78 ref79'); return false;" href="javascript:void(0);" class="ref ref78 ref79">(78,79)</a> Mice treated with the pan-BET inhibitor JQ1 had a reduced spermatozoa number and reduced motility, indicating that BRDT could be an ideal target for contraceptive drug development.<a onclick="showRef(event, 'ref80 ref81'); return false;" href="javascript:void(0);" class="ref ref80 ref81">(80,81)</a> Considering the intricate physiological functions of BET family proteins, it is important to improve the target selectivity of BRD4 inhibitors in specific biological processes and disease progression without incurring unintended side effects, especially in the treatment of cancer and inflammation.<a onclick="showRef(event, 'ref58'); return false;" href="javascript:void(0);" class="ref ref58">(58)</a> Two BRD4-selective inhibitors reported in recent years are shown in <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>C. ZL0454 (<b>19</b>) and ZL0420 (<b>20</b>) are two BRD4-selective inhibitors displaying 30–60-fold selectivity over BRD2, 50–90-fold selectivity over BRD3, and 70–120-fold selectivity over BRDT.<a onclick="showRef(event, 'ref82'); return false;" href="javascript:void(0);" class="ref ref82">(82)</a> Molecular docking shows that compounds <b>19</b> and <b>20</b> occupy the Kac-binding pocket of BRD4 in a similar way. <i>In vivo</i> experiments further found that <b>19</b> and <b>20</b> almost completely blocked the TLR3 (toll-like receptor 3)-dependent expression of innate immune genes, including ISG54, ISG56, IL-8, and Groβ in human primary small airway epithelial cells (hSAECs). Given the shared functions between BRD4 and the other BET family proteins yet to be completely defined, it is difficult to predict what types of diseases are most suited for BRD4-selective targeting. Nevertheless, therapy with fewer unintended targets has the potential to improve the clinical outcomes.</div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i10" class="anchor-spacer"></div><h3 class="article-section__title" id="_i10">4.4.  Dual Pharmacophores</h3><div class="NLM_p">In recent years, multitarget approaches targeting different proteins and inhibiting kinases have received increasing attention in anticancer drug development.<a onclick="showRef(event, 'ref83'); return false;" href="javascript:void(0);" class="ref ref83">(83)</a> A previous study that reported that the CDK inhibitor Dinaciclib could interact with the Kac-binding pocket of BRDT provided the first indication that kinase inhibitors might be used to rationally design a new generation of BRD4 inhibitors.<a onclick="showRef(event, 'ref84'); return false;" href="javascript:void(0);" class="ref ref84">(84)</a> Since then, on the basis of the idea that polypharmacological inhibitors with multitargets would have a more robust action, many dual BET/kinase inhibitors have been described.<a onclick="showRef(event, 'ref85 ref86'); return false;" href="javascript:void(0);" class="ref ref85 ref86">(85,86)</a> Some of the recently discovered dual-target inhibitors are shown in <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>.</div><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01487/20210305/images/medium/jm0c01487_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01487/20210305/images/large/jm0c01487_0004.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Chemical structures of dual-target inhibitors <b>21</b>–<b>31</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01487/20210305/images/large/jm0c01487_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01487&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Histone deacetylases (HDACs) reduce the acetylation level of lysine residues where BET proteins recognize and promote factor recruitment to targeted gene loci, thereby regulating gene transcription.<a onclick="showRef(event, 'ref2 ref87 ref88 ref89'); return false;" href="javascript:void(0);" class="ref ref2 ref87 ref88 ref89">(2,87−89)</a> It is logical to cotarget BET proteins and HDACs for the more efficient repression of oncogenic gene transcription and tumor cell proliferation.<a onclick="showRef(event, 'ref90 ref91'); return false;" href="javascript:void(0);" class="ref ref90 ref91">(90,91)</a> Compound <b>21</b>, a highly potent dual BET/HDAC inhibitor, was reported with balanced activity against BRD4 BD1 and HDAC1.<a onclick="showRef(event, 'ref92'); return false;" href="javascript:void(0);" class="ref ref92">(92)</a> Compound <b>22</b> shows the inhibition of BRD4 at a rate of 88% at 10 μM and strong HDAC3 inhibition with an IC<sub>50</sub> value of 5 nM.<a onclick="showRef(event, 'ref93'); return false;" href="javascript:void(0);" class="ref ref93">(93)</a> Compound <b>23</b>, another dual BRD4/HDAC inhibitor, induces autophagic cell death in colorectal cancer (CRC) cells at nanomolar levels.<a onclick="showRef(event, 'ref90'); return false;" href="javascript:void(0);" class="ref ref90">(90)</a> Compound <b>24</b> induces HL-60 AML cell death with c-MYC-inhibitory activity and suppresses cell growth in BET inhibitor-resistant cells.<a onclick="showRef(event, 'ref94'); return false;" href="javascript:void(0);" class="ref ref94">(94)</a> Compounds <b>25</b> and <b>26</b> also exhibit potent antiproliferative activity in human leukemia cell lines.<a onclick="showRef(event, 'ref95 ref96'); return false;" href="javascript:void(0);" class="ref ref95 ref96">(95,96)</a></div><div class="NLM_p">Because BRD4 plays a critical role in DNA repair (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>D), the inhibition of BRD4 shows synthetic lethality with poly(ADP-ribose) polymerase-1 (PARP1) inhibitors, resulting in HR deficiency and providing a strategy for cancer treatment.<a onclick="showRef(event, 'ref35 ref97'); return false;" href="javascript:void(0);" class="ref ref35 ref97">(35,97)</a> Compound <b>27</b>, designed with the conception of synthetic lethality, is the first dual BRD4/PARP1 inhibitor reported that triggers breast cancer cell apoptosis and induces cell-cycle arrest in the G1 phase.<a onclick="showRef(event, 'ref98'); return false;" href="javascript:void(0);" class="ref ref98">(98)</a> A recent study reports that polo-like kinase 1 (PLK1) upregulates BRD4-controlled MYC transcription and protein stability, with MYC, in turn, enhancing the PLK1 gene transcription, thus forming a PLK1-MYC feed-forward circuit in double-hit lymphoma (DHL) cells.<a onclick="showRef(event, 'ref99'); return false;" href="javascript:void(0);" class="ref ref99">(99)</a> Via the chemical modification of the pre-existing dual BRD4/PLK1 inhibitor BI-2536 (<b>28</b>), the dual BET/PLK1 inhibitor WNY0824 (<b>29</b>) was shown to suppress CRPC cell proliferation as well as the tumor growth in an enzalutamide-resistant CRPC xenograft mouse model by blocking BRD4-dependent androgen-receptor- and MYC-driven transcription programs.<a onclick="showRef(event, 'ref100'); return false;" href="javascript:void(0);" class="ref ref100">(100)</a> Fine-tuning the selectivity of inhibitors can also be achieved through structural guidance. Compound <b>30</b> is identified as a well-balanced dual BRD4/PLK1 inhibitor, illustrating that the methylated amide and the cyclopentyl group of the compound <b>28</b> scaffold are important for maintaining bromodomain-binding activity but are less critical for PLK1-inhibitory activity.<a onclick="showRef(event, 'ref101'); return false;" href="javascript:void(0);" class="ref ref101">(101)</a> Another study also chose compound <b>28</b> as the starting compound but changed PLK1 inhibition to anaplastic lymphoma kinase (ALK) targeting with the derived inhibitor compound <b>31</b>, maintaining the BRD4-inhibitory activity but shifting the kinase selectivity from diminished PLK1 inhibition to enhanced ALK inhibition, which shows the remarkable suppression of ALK<sup>F1174L</sup>- and BRD4-dependent MYCN-driven gene transcription in pediatric neuroblastoma cells.<a onclick="showRef(event, 'ref102'); return false;" href="javascript:void(0);" class="ref ref102">(102)</a> These strategies provide new directions for the development of novel dual-target inhibitors via the chemical modification of existing pharmacophores targeting, respectively, BET proteins and kinases. However, this new line of research also poses significant challenges in how to properly balance the selectivity between BET proteins and specific kinases in designing dual-target inhibitors with enhanced antitumor activity. A report documenting that PLK inhibition, rather than BRD4 inhibition, is the main action for the anticancer activity of compound <b>28</b> in MV4-11 cells may provide some mechanistic hints about this specific compound.<a onclick="showRef(event, 'ref103'); return false;" href="javascript:void(0);" class="ref ref103">(103)</a></div><div class="NLM_p">It should be mentioned that many dual-target inhibitors were discovered through their unexpected BET-targeting activity. For instance, TG101209 and BI-2536, originally reported as JAK2 inhibitor and PLK1 inhibitor, respectively, were found to have strong inhibitory potential against BRD4 through binding to the Kac pocket, providing a rationale for the development of dual-target inhibitors.<a onclick="showRef(event, 'ref104'); return false;" href="javascript:void(0);" class="ref ref104">(104)</a> As the products of a new technology, dual-target inhibitors display great potential in cancer therapy, even though they are still in the preclinical stage. Combining two pharmacophores cotargeting BET proteins and HDACs based on their pharmacological activity and safety evaluation, for example, <b>21</b> synthesized by linking (+)-JQ1 to HDAC inhibitors SAHA and CI-944,<a onclick="showRef(event, 'ref92'); return false;" href="javascript:void(0);" class="ref ref92">(92)</a> has become a popular trend in the discovery of dual-target inhibitors.<a onclick="showRef(event, 'ref105'); return false;" href="javascript:void(0);" class="ref ref105">(105)</a> For dual-target inhibitors that have already been developed, structural modification may result in new inhibitors with different biological activities, such as compounds <b>29</b>, <b>30</b>, and <b>31</b>, optimized from compound <b>28</b>.<a onclick="showRef(event, 'ref96 ref102'); return false;" href="javascript:void(0);" class="ref ref96 ref102">(96,102)</a></div><div class="NLM_p last">Additional kinase inhibitors to be considered include those involved in the PI3K-AKT-mTOR pathway that regulates MYC expression, stability, post-translational modification, and MYC protein synthesis<a onclick="showRef(event, 'ref106'); return false;" href="javascript:void(0);" class="ref ref106">(106)</a> as well as CDK9, which phosphorylates the Pol II CTM at Ser2, resulting in the stimulation of transcription elongation, and is associated with BRD4 hyperphosphorylation in NMC.<a onclick="showRef(event, 'ref40 ref83 ref107'); return false;" href="javascript:void(0);" class="ref ref40 ref83 ref107">(40,83,107)</a> CK2, which phosphorylates BRD4, implicated in chromatin targeting and factor recruitment,<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> is another promising candidate for the future development of potent dual-target inhibitors for cancer therapy.</div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i11" class="anchor-spacer"></div><h3 class="article-section__title" id="_i11">4.5.  PROTACs</h3><div class="NLM_p">PROTACs, initially proposed by Deshaies and colleagues,<a onclick="showRef(event, 'ref108'); return false;" href="javascript:void(0);" class="ref ref108">(108)</a> are an effective way to inactivate protein function via compound-mediated proteolysis. PROTACs targeting BET proteins have distinct pharmacological effects compared with the corresponding small-molecule inhibitors because they can disrupt the function of the entire protein rather than just the bromodomain activity.<a onclick="showRef(event, 'ref46 ref109'); return false;" href="javascript:void(0);" class="ref ref46 ref109">(46,109)</a> A sustained loss of BET proteins has superior antiproliferative effects <i>in vitro</i> while showing better anticancer activity <i>in vivo</i> with no obvious toxicity in a mouse tumor regression study.<a onclick="showRef(event, 'ref110'); return false;" href="javascript:void(0);" class="ref ref110">(110)</a></div><div class="NLM_p">The currently developed BET degraders use ligand binding to the cereblon (CRBN) or von Hipple-Lindau (VHL) component of an E3 ubiquitin ligase. Compounds <b>32</b>–<b>38</b> are CRBN-based degraders, whereas compounds <b>39</b>–<b>45</b> are based on the VHL component (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>A). Among them, dBET1 (<b>32</b>) containing an eight-atom linker is the first degrader displaying remarkable selectivity in depleting BET proteins with more potent inhibition compared with JQ1 against MV4-11 cells, DHL-4 lymphoma cells, and primary blasts from leukemia patients.<a onclick="showRef(event, 'ref111'); return false;" href="javascript:void(0);" class="ref ref111">(111)</a> ARV-825 (<b>33</b>) with a different linker type also achieves an effective degradation of BRD4 (DC<sub>50</sub> < 1 nM), resulting in strong and lasting c-MYC inhibition in comparison with pan-BET inhibitors JQ1 and OTX015.<a onclick="showRef(event, 'ref112'); return false;" href="javascript:void(0);" class="ref ref112">(112)</a> BETd-246 (<b>34</b>) induces BET protein degradation at low nanomolar concentrations within an hour in TNBC cells by downregulating many proliferation/apoptosis-related genes, leading to growth inhibition and apoptosis.<a onclick="showRef(event, 'ref113'); return false;" href="javascript:void(0);" class="ref ref113">(113)</a> Another degrader, BETd-260 (<b>35</b>), shows picomolar potency (IC<sub>50</sub> = 51 pM) in inhibiting RS4-11 lymphoblastic leukemia cell growth.<a onclick="showRef(event, 'ref110'); return false;" href="javascript:void(0);" class="ref ref110">(110)</a></div><figure id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01487/20210305/images/medium/jm0c01487_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01487/20210305/images/large/jm0c01487_0005.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Reported PROTAC degraders and PPI inhibitors. (A) PROTAC degraders based on CRL4<sup>CRBN</sup> (<b>32</b>–<b>38</b>) and CRL2<sup>VHL</sup> E3 ligase (<b>39</b>–<b>45</b>). (B) Chemical structures of <b>46</b>–<b>48</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01487/20210305/images/large/jm0c01487_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01487&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">QCA570 (<b>36</b>) was designed and synthesized based on the 1,4-oxazepine structure with more than 1000 times higher potency compared with <b>33</b>. Significantly, in both the MV4-11 and RS4-11 acute leukemia xenograft models, compound <b>36</b> shows complete and long-lasting tumor regression at 5 mg/kg without apparent toxicity.<a onclick="showRef(event, 'ref114'); return false;" href="javascript:void(0);" class="ref ref114">(114)</a> A recent study highlights that DP1 (<b>37</b>) not only achieves rapid, stable, and persistent BRD4 degradation in SU-DHL-4 lymphoma cells but also has significant inhibitory effects on immunodeficient mice with xenotransplanted lymphoma cells.<a onclick="showRef(event, 'ref115'); return false;" href="javascript:void(0);" class="ref ref115">(115)</a> Cellular degradation assays showed that ZXH-3-26 (<b>38</b>) exhibits a DC<sub>50/5h</sub> of ∼5 nM and BRD4 selectivity without inducing significant inhibition or degradation of BRD2 and BRD3,<a onclick="showRef(event, 'ref116'); return false;" href="javascript:void(0);" class="ref ref116">(116)</a> a result similar to that observed with AT1 (<b>39</b>) displaying BRD4-selective depletion but negligible activity against BRD2 and BRD3 at 1 to 3 μM of <b>39</b> treatment.<a onclick="showRef(event, 'ref117'); return false;" href="javascript:void(0);" class="ref ref117">(117)</a> ARV-771 (<b>40</b>), a small-molecule pan-BET degrader, reduces the protein levels of androgen receptor (AR) signaling in a CRPC mouse xenotransplantation model, providing the first reported BET degrader activity against a solid tumor.<a onclick="showRef(event, 'ref118'); return false;" href="javascript:void(0);" class="ref ref118">(118)</a> Compound <b>40</b> also presents superior pharmacological properties compared with those of <b>33</b> in mantle cell lymphoma (MCL) Mino cells.<a onclick="showRef(event, 'ref119'); return false;" href="javascript:void(0);" class="ref ref119">(119)</a> MZ1 (<b>41</b>), a JQ1-based BRD4 degrader, exhibits BRD4-targeting preference in cell-based assays but comparable <i>in vitro</i> binding affinities to individually purified BRD2, BRD3, and BRD4 bromodomains measured by isothermal calorimetry (ITC), suggesting that MZ1-induced proximity and accessible surface lysine residues on BRD4, versus BRD2 and BRD3, might be differentially contacted by the VHL E3 ligase module <i>in vivo</i> and are presumably regulated by BET interactors that are unique to each family member.<a onclick="showRef(event, 'ref120'); return false;" href="javascript:void(0);" class="ref ref120">(120)</a> Adding a cyclizing linker to compound <b>40</b> leads to a macrocyclic PROTAC, macroPROTAC-1 (<b>42</b>), with cellular activity similar to that of compound <b>41</b>, thus providing a new strategy for drug design.<a onclick="showRef(event, 'ref121'); return false;" href="javascript:void(0);" class="ref ref121">(121)</a></div><div class="NLM_p">The targeting effects between triazolodiazepine JQ1 and tetrahydroquinoline I-BET762 and the optimal length of linkers connecting the BET-binding ligand JQ1 or I-BET762 to the VHL ligand VH032 for degrader design were examined by synthesizing a series of I-BET762-derived degraders, including MZP-54 (<b>43</b>), MZP-55 (<b>44</b>), and MZP-61 (<b>45</b>),<a onclick="showRef(event, 'ref122'); return false;" href="javascript:void(0);" class="ref ref122">(122)</a> for comparison with MZ1-related compounds, with the results showing that JQ1-based PROTACs are more effective at inducing BET protein degradation, even though JQ1 alone is not as potent as I-BET762 in BET protein binding. This parallel comparison indicates that stronger inhibitors may not necessarily produce more effective PROTAC degraders, and the linker length is critical in PROTAC design. It is important to note that PROTAC binding to incidental or unintended targets through substrate or E3 ligase association does not guarantee substrate degradation.<a onclick="showRef(event, 'ref123'); return false;" href="javascript:void(0);" class="ref ref123">(123)</a> These BET degraders are effective in inhibiting cancer cell proliferation <i>in vivo</i> and <i>in vitro</i>. In mice, compounds <b>34</b>, <b>35</b>, and <b>36</b> show good tolerance without obvious toxicity.<a onclick="showRef(event, 'ref124'); return false;" href="javascript:void(0);" class="ref ref124">(124)</a></div><div class="NLM_p last">BET-targeting PROTACs are still in the preclinical stage, and thus their therapeutic potential is presently unknown. For the current degraders, the main challenge is to improve the bioavailability index through precise modification without significantly increasing the molecular size. More selective and tissue-specific degraders are also needed for clinical trials. A strategy involving antibody–PROTAC conjugation provides a proof of concept for tissue-specific BRD4 degradation.<a onclick="showRef(event, 'ref125'); return false;" href="javascript:void(0);" class="ref ref125">(125)</a> A trastuzumab–PROTAC conjugate (Ab-PROTAC) with blocked degradation activity by an antibody linker that can be selectively released to produce an active PROTAC demonstrates the significant potential for tissue-specific target degradation. It should be mentioned that targeting BET proteins for degradation is not limited to ubiquitination. The lysosomal/autophagy pathway provides cargo-specific degradation through a different molecular mechanism.<a onclick="showRef(event, 'ref126'); return false;" href="javascript:void(0);" class="ref ref126">(126)</a></div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i12" class="anchor-spacer"></div><h3 class="article-section__title" id="_i12">4.6.  Targeting the Protein–Protein Interaction Network</h3><div class="NLM_p">The PPI forms signal nodes and hubs in normal cells and cancer cells and transmits pathophysiological cues along molecular networks to promote tumorigenesis, migration, invasion, and metastasis.<a onclick="showRef(event, 'ref127'); return false;" href="javascript:void(0);" class="ref ref127">(127)</a> Therefore, blocking the PPI is an effective way to inactivate network function and signal transmission. DC-1 (<b>46</b>) and DC-2 (<b>47</b>), two peptoids (i.e., N-substituted oligoglycines; <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>B) isolated from a combinatorial compound library of ∼38 500 peptoids,<a onclick="showRef(event, 'ref128'); return false;" href="javascript:void(0);" class="ref ref128">(128)</a> bind specifically to phospho-NPS in human BRD4, which provides an effective strategy to inhibit the phosphorylation-dependent gene-specific function of BRD4 and factor-regulated transcription programs.<a onclick="showRef(event, 'ref10 ref12'); return false;" href="javascript:void(0);" class="ref ref10 ref12">(10,12)</a> The iridium(III) transition-metal-based irreversible PPI inhibitor <b>48</b> blocks BRD4-BD1 but not BRD4-BD2, binding to a tetraacetylated histone H4 peptide with IC<sub>50</sub> = 70 nM, and suppresses melanoma cell growth <i>in vitro</i> and mouse xenograft tumor growth <i>in vivo</i> by inhibiting BRD4-driven MYC and BCL-2 gene transcription.<a onclick="showRef(event, 'ref129'); return false;" href="javascript:void(0);" class="ref ref129">(129)</a> Clearly, PPI inhibitors provide another promising strategy for BRD4-targeted disease treatment.</div><div class="NLM_p last">Compared with other technologies, targeting the PPI associated with BRD4 is less reported, in part due to the challenge in identifying the <i>in vivo</i> targets and the mechanisms of action as well as the difficulty in establishing effective screening systems.<a onclick="showRef(event, 'ref130'); return false;" href="javascript:void(0);" class="ref ref130">(130)</a> Encouragingly, targeting phospho-NPS in BRD4, as illustrated with compounds <b>46</b> and <b>47</b>, has been shown to be a promising and effective way to inhibit BRD4 association with specific transcription factors without blocking the chromatin-binding activity of BRD4, providing a rationale for targeting nonbromodomain surfaces of BRD4 to block specific disease-associated pathways with minimal perturbation of normal BRD4 function.<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> A unique interface formed between BRD4 and GATA4 interaction in cardiomyocytes that regulates the bioenergy homeostasis of the adult heart would be an interesting region for the future development of BRD4-targeting PPI inhibitors.<a onclick="showRef(event, 'ref131'); return false;" href="javascript:void(0);" class="ref ref131">(131)</a></div></div></div><div id="sec5" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i13">5.  Conclusions and Future Perspectives</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_14802" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_14802" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">In mouse tumor models, BD1-selective inhibition shows reduced proliferation, cell-cycle arrest, and apoptosis, as seen with pan-BET inhibitors, whereas BD2-selective inhibition is more effective in suppressing inflammation, metabolic diseases, and fibrosis by reducing the negative effects of enhancer reprogramming and minimizing the maladaptive transcription of disease-causing genes on potential enhancers.<a onclick="showRef(event, 'ref132'); return false;" href="javascript:void(0);" class="ref ref132">(132)</a> The development of BRD4-selective inhibitors is in an early stage. Considering the oncogenic role the BRD4 short isoform plays in the extracellular matrix (ECM) network during breast cancer progression<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> and the potential active gene transcription through LLPS,<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> selective targeting of the BRD4 short isoform without cross-inhibiting the tumor-suppressive function of the BRD4 long isoform is a critical strategy for future drug development. The advancement in BRD4 protein characterization also sheds new light on the design of selective inhibitors targeting a specific region of BRD4, such as phospho-NPS, which provides more selective interaction with cancer-associated transcription factors. Further development of inhibitors targeting other BET proteins could also be beneficial to disease treatment or birth control, such as BRDT-specific targeting for contraceptives. Conceivably, research on the structural understanding of BET proteins will contribute to the development of more selective inhibitors. A computational protein structure prediction is helpful to simulate the protein structure, with refinement making the model closer to the natural structure,<a onclick="showRef(event, 'ref133'); return false;" href="javascript:void(0);" class="ref ref133">(133)</a> thus allowing the exploration of whether or how small molecules bind to the protein through docking.<a onclick="showRef(event, 'ref134'); return false;" href="javascript:void(0);" class="ref ref134">(134)</a> All of these provide new methodologies for the further development of selective inhibitors.</div><div class="NLM_p">At present, dual kinase/BRD4 inhibitors are still in the early stages. A rational design of dual kinase/BET inhibitors may be conceptually easier than designing bifunctional ligands targeting two different kinases due to the similarity of their catalytic pockets. It is also challenging to design dual-target inhibitors with balanced selectivity. Here computational methods are useful for predicting and identifying new ligands with dual activity by simulating the binding modes of ligands and using ligand-based and structure-based approaches. Moreover, the physical and chemical properties of dual-target inhibitors may be poor due to an increase in the relative molecular weight and structural complexity. How to balance the target selectivity of dual-target inhibitors to maximize the clinical efficacy is another challenge. It is likely that partial coverage of the kinase and full coverage of BRD4 may achieve the maximum outcome.</div><div class="NLM_p">The pharmacological effects of PROTACs are not limited to only protein loss but also affect the related subcellular localization and transcriptional regulation, thus improving the pharmacological efficacy through this additional function. Another feature of PROTACs is that they require a relatively low dose to achieve pharmacological effects. This also reduces the possibility of off targets and thus improves the drug tolerance. In addition, PROTACs targeting BET proteins, in a way similar to gene knockout, can cause a more lasting loss of protein expression than other targeted inhibitors, namely, a more prolonged inhibitory effect.</div><div class="NLM_p">The PPI is an important part of the functional exploration of BRD4. Besides the well-known bromodomain, the nonbromodomain PPI is also of great research value, especially the phospho region of BRD4 targeted by the existing peptoid inhibitors. Accordingly, PPI inhibitors, with in-depth mechanistic studies and chemical probing, have great therapeutic potential warranting further development.</div><div class="NLM_p last">All of these new technologies have shown great potential based on <i>in vitro</i> and <i>in vivo</i> bench studies. The primary challenge is to transfer the preclinical antitumor activity of compounds to clinical efficacy. Altogether, the use of small-molecule inhibitors targeting BRD4 is expected to be effective for the treatment of a variety of cancers. Interdisciplinary cooperation among investigators is of great need to make groundbreaking drug discoveries in this exciting area of research.</div></div><div class="NLM_back"><div class="article_supporting-info" id=""></div><div class="testing" data-doi="10.1021/acs.jmedchem.0c01487" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_70768" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_70768" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jie Liu</span> - <span class="hlFld-Affiliation affiliation">State
Key Laboratory of Biotherapy and Cancer Center, National Clinical
Research Center for Geriatrics, West China
Hospital of Sichuan University, Chengdu 610041, China</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#1975706c73707c2b292828596a7a6c377c7d6c377a77"><span class="__cf_email__" data-cfemail="15797c607f7c7027252424556676603b7071603b767b">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Cheng-Ming Chiang</span> - <span class="hlFld-Affiliation affiliation">Simmons
Comprehensive Cancer Center, Department of Pharmacology, and Department
of Biochemistry, University of Texas Southwestern
Medical Center, Dallas, Texas 75390, United States</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#d7b4bfb2b9b0fababeb9b0f9b4bfbeb6b9b097a2a3a4b8a2a3bfa0b2a4a3b2a5b9f9b2b3a2"><span class="__cf_email__" data-cfemail="b0d3d8d5ded79dddd9ded79ed3d8d9d1ded7f0c5c4c3dfc5c4d8c7d5c3c4d5c2de9ed5d4c5">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Liang Ouyang</span> - <span class="hlFld-Affiliation affiliation">State
Key Laboratory of Biotherapy and Cancer Center, National Clinical
Research Center for Geriatrics, West China
Hospital of Sichuan University, Chengdu 610041, China</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-5537-8834" title="Orcid link">http://orcid.org/0000-0001-5537-8834</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#157a606c747b72797c747b72556676603b7071603b767b"><span class="__cf_email__" data-cfemail="640b111d050a03080d050a03241707114a0100114a070a">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Pan Tang</span> - <span class="hlFld-Affiliation affiliation">State
Key Laboratory of Biotherapy and Cancer Center, National Clinical
Research Center for Geriatrics, West China
Hospital of Sichuan University, Chengdu 610041, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jifa Zhang</span> - <span class="hlFld-Affiliation affiliation">State
Key Laboratory of Biotherapy and Cancer Center, National Clinical
Research Center for Geriatrics, West China
Hospital of Sichuan University, Chengdu 610041, China</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>P.T. and J.Z. contributed equally to this work.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="bio" rid="ath1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="biographies">Biographies</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_23117" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_23117" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><figure id="ath1" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Pan Tang</h2><div class="figure-bottom-links"><div class="download-hi-res-img"></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01487&amp;id=BIO-d7e1835-autogenerated"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><b>Pan Tang</b> received her B.S. degree in Pharmacy English from Shenyang Pharmaceutical University in 2018. She is currently a Master’s student at the State Key Laboratory of Biotherapy and Cancer Center, West China Hospital and Collaborative Innovation Center of Biotherapy, Sichuan University. Her research interest focuses on the anticancer mechanism of small-molecule compounds.</p></figure></div><div class="bio" rid="ath2"><figure id="ath2" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Jifa Zhang</h2><div class="figure-bottom-links"><div class="download-hi-res-img"></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01487&amp;id=BIO-d7e1840-autogenerated"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><b>Jifa Zhang</b> received his Ph.D. degree from Southwest Jiaotong University in 2018 and is currently a postdoctoral student at the State Key Laboratory of Biotherapy and Cancer Center, West China Hospital and Collaborative Innovation Center of Biotherapy, Sichuan University. His research interests focus on the discovery, optimization, and molecular mechanism of candidate small-molecule compounds with antitumor activity.</p></figure></div><div class="bio" rid="ath3"><figure id="ath3" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Jie Liu</h2><div class="figure-bottom-links"><div class="download-hi-res-img"></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01487&amp;id=BIO-d7e1845-autogenerated"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><b>Jie Liu</b> received his Ph.D. degree in Medicinal Chemistry from West China School of Pharmacy, Sichuan University in 2010. He is now a Professor at State Key Laboratory of Biotherapy, West China Hospital, Sichuan University. He has long been engaged in the application of transition-metal-mediated organic synthesis in small-molecule-targeted drugs and pharmacochemical biology.</p></figure></div><div class="bio" rid="ath4"><figure id="ath4" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Cheng-Ming Chiang</h2><div class="figure-bottom-links"><div class="download-hi-res-img"></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01487&amp;id=BIO-d7e1850-autogenerated"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><b>Cheng-Ming Chiang</b> received his Ph.D. from the Department of Biochemistry, University of Rochester in 1991. He has been an Assistant Professor in the Department of Biochemistry at the University of Illinois Urbana–Champaign (1995–2000), an Associate Professor in the Department of Biochemistry at Case Western Reserve University in Cleveland, Ohio (2000–2007), and a Professor at Simmons Comprehensive Cancer Center, Department of Biochemistry, and Department of Pharmacology at the University of Texas Southwestern Medical Center (2007–present). His research interests primarily focus on transcription mechanisms, gene regulation, epigenetic control, chromatin dynamics, posttranslational modification, human papillomavirus, cancer, and therapeutics targeting phospho-BRD4 and BET proteins.</p></figure></div><div class="bio" rid="ath5"><figure id="ath5" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Liang Ouyang</h2><div class="figure-bottom-links"><div class="download-hi-res-img"></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01487&amp;id=BIO-d7e1855-autogenerated"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><b>Liang Ouyang</b> received his Ph.D. degree in Medicinal Chemistry from West China School of Pharmacy, Sichuan University, in 2010. He is now Professor at State Key Laboratory of Biotherapy, West China Hospital, Sichuan University. He has long-time interests in the design and synthesis of drugs based on original targeting of autophagy, basic research on the application of lead compound screening, the development of new drug target recognition methods and drug discovery, and taking advantage of interdisciplinary approaches including systems biology, structural biology, computer-aided drug design, medicinal chemistry, chemical biology, and molecular biology.</p></figure></div><div class="ack" id="ack1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i20">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_68753" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_68753" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">Funding for the authors’ research was sponsored in part by the National Natural Science Foundation of China, grant numbers 81922064 (to L.O.), 81874290 (to L.O.) and 81903502 (to J.Z.); the National Science and Technology Major Project of the Ministry of Science and Technology of China, grant number 2018ZX09735005 (to L.O.); the National Major Scientific and Technological Special Project for ‘Signficant New Drugs Development’, grant number 2018ZX09201018-021 (to L.O.); the Sichuan Science and Technology Program, grant number 2019YFS0003 (to L.O.); the Sichuan Science and Technology Program, grant number 2020YJ0091 (to J.L.); the China Postdoctoral Science Foundation, grant number 2020M673268 (to J.Z.); the U.S. National Institutes of Health (NIH), grant number 1RO1CA251698-01 (to C.-M.C.); and the Cancer Prevention and Research Institute of Texas (CPRIT), grant numbers RP180349 and RP190077 (to C.-M.C.).</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations Used</td></tr><tr><td class="NLM_term">ALK</td><td class="NLM_def"><p class="first last">anaplastic lymphoma kinase</p></td></tr><tr><td class="NLM_term">ALL</td><td class="NLM_def"><p class="first last">acute lymphoblastic leukemia</p></td></tr><tr><td class="NLM_term">AML</td><td class="NLM_def"><p class="first last">acute myelogenous leukemia</p></td></tr><tr><td class="NLM_term">AR</td><td class="NLM_def"><p class="first last">androgen receptor</p></td></tr><tr><td class="NLM_term">AST</td><td class="NLM_def"><p class="first last">advanced solid tumor</p></td></tr><tr><td class="NLM_term">BET</td><td class="NLM_def"><p class="first last">bromodomain and extraterminal</p></td></tr><tr><td class="NLM_term">BID</td><td class="NLM_def"><p class="first last">basic residue-enriched interaction domain</p></td></tr><tr><td class="NLM_term">BPDCN</td><td class="NLM_def"><p class="first last">blastic plasmacytoid dendritic cell neoplasm</p></td></tr><tr><td class="NLM_term">BRD4</td><td class="NLM_def"><p class="first last">bromodomain-containing protein 4</p></td></tr><tr><td class="NLM_term">CDK9</td><td class="NLM_def"><p class="first last">cyclin-dependent kinase 9</p></td></tr><tr><td class="NLM_term">CK2</td><td class="NLM_def"><p class="first last">casein kinase II</p></td></tr><tr><td class="NLM_term">CPS</td><td class="NLM_def"><p class="first last">C-terminal cluster of phosphorylation sites</p></td></tr><tr><td class="NLM_term">CRBN</td><td class="NLM_def"><p class="first last">cereblon</p></td></tr><tr><td class="NLM_term">CRC</td><td class="NLM_def"><p class="first last">colorectal cancer</p></td></tr><tr><td class="NLM_term">CRPC</td><td class="NLM_def"><p class="first last">castration-resistant prostate cancer</p></td></tr><tr><td class="NLM_term">CtIP</td><td class="NLM_def"><p class="first last">C-terminal binding protein-interacting protein</p></td></tr><tr><td class="NLM_term">CTM</td><td class="NLM_def"><p class="first last">C-terminal motif</p></td></tr><tr><td class="NLM_term">DHL</td><td class="NLM_def"><p class="first last">double-hit lymphoma</p></td></tr><tr><td class="NLM_term">DLBCL</td><td class="NLM_def"><p class="first last">diffuse large B-cell lymphoma</p></td></tr><tr><td class="NLM_term">DLT</td><td class="NLM_def"><p class="first last">dose-limiting toxicity</p></td></tr><tr><td class="NLM_term">DSB</td><td class="NLM_def"><p class="first last">DNA double-strand break</p></td></tr><tr><td class="NLM_term">ECM</td><td class="NLM_def"><p class="first last">extracellular matrix</p></td></tr><tr><td class="NLM_term">ET</td><td class="NLM_def"><p class="first last">extraterminal</p></td></tr><tr><td class="NLM_term">GBM</td><td class="NLM_def"><p class="first last">glioblastoma multiform</p></td></tr><tr><td class="NLM_term">HDAC</td><td class="NLM_def"><p class="first last">histone deacetylase</p></td></tr><tr><td class="NLM_term">HPV</td><td class="NLM_def"><p class="first last">human papillomavirus</p></td></tr><tr><td class="NLM_term">HR</td><td class="NLM_def"><p class="first last">homologous recombination</p></td></tr><tr><td class="NLM_term">hSAEC</td><td class="NLM_def"><p class="first last">human primary small airway epithelial cell</p></td></tr><tr><td class="NLM_term">ITC</td><td class="NLM_def"><p class="first last">isothermal calorimetry</p></td></tr><tr><td class="NLM_term">Kac</td><td class="NLM_def"><p class="first last"><i>N</i>-ε-acetyllysine</p></td></tr><tr><td class="NLM_term">LLPS</td><td class="NLM_def"><p class="first last">liquid–liquid phase separation</p></td></tr><tr><td class="NLM_term">MCL</td><td class="NLM_def"><p class="first last">mantle cell lymphoma</p></td></tr><tr><td class="NLM_term">MCP-1</td><td class="NLM_def"><p class="first last">monocyte chemoattractant protein-1</p></td></tr><tr><td class="NLM_term">MCRPC</td><td class="NLM_def"><p class="first last">metastatic castration-resistant prostate cancer</p></td></tr><tr><td class="NLM_term">MDS</td><td class="NLM_def"><p class="first last">myelodysplastic syndrome</p></td></tr><tr><td class="NLM_term">MM</td><td class="NLM_def"><p class="first last">multiple myeloma</p></td></tr><tr><td class="NLM_term">MPN, U</td><td class="NLM_def"><p class="first last">myeloproliferative neoplasm-unclassifiable</p></td></tr><tr><td class="NLM_term">NHEJ</td><td class="NLM_def"><p class="first last">nonhomologous end joining</p></td></tr><tr><td class="NLM_term">NHL</td><td class="NLM_def"><p class="first last">non-Hodgkin’s lymphoma</p></td></tr><tr><td class="NLM_term">NMC</td><td class="NLM_def"><p class="first last">NUT midline carcinoma</p></td></tr><tr><td class="NLM_term">NPS</td><td class="NLM_def"><p class="first last">N-terminal cluster of phosphorylation sites</p></td></tr><tr><td class="NLM_term">NSCLC</td><td class="NLM_def"><p class="first last">nonsmall cell lung cancer</p></td></tr><tr><td class="NLM_term">PARP1</td><td class="NLM_def"><p class="first last">poly(ADP-ribose) polymerase-1</p></td></tr><tr><td class="NLM_term">PBMC</td><td class="NLM_def"><p class="first last">peripheral blood mononuclear cell</p></td></tr><tr><td class="NLM_term">PK</td><td class="NLM_def"><p class="first last">pharmacokinetics</p></td></tr><tr><td class="NLM_term">PLK1</td><td class="NLM_def"><p class="first last">polo-like kinase 1</p></td></tr><tr><td class="NLM_term">PP2A</td><td class="NLM_def"><p class="first last">protein phosphatase 2A</p></td></tr><tr><td class="NLM_term">PPI</td><td class="NLM_def"><p class="first last">protein–protein interaction</p></td></tr><tr><td class="NLM_term">PROTAC</td><td class="NLM_def"><p class="first last">proteolysis targeting chimera</p></td></tr><tr><td class="NLM_term">P-TEFb</td><td class="NLM_def"><p class="first last">positive transcription elongation factor b</p></td></tr><tr><td class="NLM_term">SPOP</td><td class="NLM_def"><p class="first last">speckle-type POZ protein</p></td></tr><tr><td class="NLM_term">TLP3</td><td class="NLM_def"><p class="first last">toll-like receptor 3</p></td></tr><tr><td class="NLM_term">TNBC</td><td class="NLM_def"><p class="first last">triple-negative breast cancer</p></td></tr><tr><td class="NLM_term">VHL</td><td class="NLM_def"><p class="first last">von Hipple-Lindau.</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i22">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_62043" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_62043" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 134 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Picaud, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mangos, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keates, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lambert, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barsyte-Lovejoy, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Felletar, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Volkmer, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pawson, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gingras, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span> <span> </span><span class="NLM_article-title">Histone recognition and large-scale structural analysis of the human bromodomain family</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>149</i></span>,  <span class="NLM_fpage">214</span>– <span class="NLM_lpage">231</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2012.02.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1016%2Fj.cell.2012.02.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=22464331" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BC38XltVamurs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=149&publication_year=2012&pages=214-231&author=P.+Filippakopoulosauthor=S.+Picaudauthor=M.+Mangosauthor=T.+Keatesauthor=J.+P.+Lambertauthor=D.+Barsyte-Lovejoyauthor=I.+Felletarauthor=R.+Volkmerauthor=S.+Mullerauthor=T.+Pawsonauthor=A.+C.+Gingrasauthor=C.+H.+Arrowsmithauthor=S.+Knapp&title=Histone+recognition+and+large-scale+structural+analysis+of+the+human+bromodomain+family&doi=10.1016%2Fj.cell.2012.02.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Histone Recognition and Large-Scale Structural Analysis of the Human Bromodomain Family</span></div><div class="casAuthors">Filippakopoulos, Panagis; Picaud, Sarah; Mangos, Maria; Keates, Tracy; Lambert, Jean-Philippe; Barsyte-Lovejoy, Dalia; Felletar, Ildiko; Volkmer, Rudolf; Muller, Susanne; Pawson, Tony; Gingras, Anne-Claude; Arrowsmith, Cheryl H.; Knapp, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">149</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">214-231</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Bromodomains (BRDs) are protein interaction modules that specifically recognize ε-N-lysine acetylation motifs, a key event in the reading process of epigenetic marks.  The 61 BRDs in the human genome cluster into eight families based on structure/sequence similarity.  Here, we present 29 high-resoln. crystal structures, covering all BRD families.  These proteins are: ASH1L, ATAD2, BAZ2B, BPTF, BRD1, BRD3(1), BRD3(2), BRD4(1), BRD4(2), BRD9, BRDT(1), CECR2, EP300, CREBBP, GCN5L2, KIAA1240, PB1(1), PB1(2), PB1(3), PB1(4), PB1(5), PB1(6), PCAF, PHIP(2), TAF1(2), WDR9(2), BRD4(1).  Comprehensive crossfamily structural anal. identifies conserved and family-specific structural features that are necessary for specific acetylation-dependent substrate recognition.  Screening of more than 30 representative BRDs against systematic histone-peptide arrays identifies new BRD substrates and reveals a strong influence of flanking posttranslational modifications, such as acetylation and phosphorylation, suggesting that BRDs recognize combinations of marks rather than singly acetylated sequences.  We further uncovered a structural mechanism for the simultaneous binding and recognition of diverse diacetyl-contg. peptides by BRD4.  These data provide a foundation for structure-based drug design of specific inhibitors for this emerging target family.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoK2FSIcPoXZLVg90H21EOLACvtfcHk0ljHccTPlZIoYA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XltVamurs%253D&md5=05583a01a1cd4c3b59fc59c00e4c4777</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2012.02.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2012.02.013%26sid%3Dliteratum%253Aachs%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DMangos%26aufirst%3DM.%26aulast%3DKeates%26aufirst%3DT.%26aulast%3DLambert%26aufirst%3DJ.%2BP.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DFelletar%26aufirst%3DI.%26aulast%3DVolkmer%26aufirst%3DR.%26aulast%3DMuller%26aufirst%3DS.%26aulast%3DPawson%26aufirst%3DT.%26aulast%3DGingras%26aufirst%3DA.%2BC.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DKnapp%26aufirst%3DS.%26atitle%3DHistone%2520recognition%2520and%2520large-scale%2520structural%2520analysis%2520of%2520the%2520human%2520bromodomain%2520family%26jtitle%3DCell%26date%3D2012%26volume%3D149%26spage%3D214%26epage%3D231%26doi%3D10.1016%2Fj.cell.2012.02.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wu, S. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiang, C. M.</span></span> <span> </span><span class="NLM_article-title">The double bromodomain-containing chromatin adaptor BRD4 and transcriptional regulation</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>282</i></span>,  <span class="NLM_fpage">13141</span>– <span class="NLM_lpage">13145</span>, <span class="refDoi"> DOI: 10.1074/jbc.R700001200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1074%2Fjbc.R700001200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=17329240" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BD2sXks1Ghtbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=282&publication_year=2007&pages=13141-13145&author=S.+Y.+Wuauthor=C.+M.+Chiang&title=The+double+bromodomain-containing+chromatin+adaptor+BRD4+and+transcriptional+regulation&doi=10.1074%2Fjbc.R700001200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">The Double Bromodomain-containing Chromatin Adaptor Brd4 and Transcriptional Regulation</span></div><div class="casAuthors">Wu, Shwu-Yuan; Chiang, Cheng-Ming</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">282</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">13141-13145</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">A review.  Brd4 is a double bromodomain-contg. protein that binds preferentially to acetylated chromatin.  It belongs to the BET (bromodomains and extraterminal) family that includes mammalian Brd2, Brd3, Brd4, Brdt, Drosophila Fsh, yeast Bdf1, Bdf2, and corresponding homologues in other species.  Brd4 is essential for cellular growth and has been implicated in cell cycle control, DNA replication, and gene rearrangement found in t(15;19)-assocd. carcinomas.  Recently, Brd4 has been found in several transcription complexes, including the general cofactor Mediator and the P-TEFb elongation factor, and is capable of stimulating HIV-1 transcription in a Tat-independent manner.  In addn., Brd4 is used as a cellular adaptor by some animal and human papillomaviruses (HPV) for anchoring viral genomes to mitotic chromosomes.  This tethering, mediated by Brd4 interaction with virus-encoded E2 protein, facilitates viral genome segregation during mitosis.  Interestingly, Brd4 is also identified in a transcriptional silencing complex assembled by HPV E2 and turns out to be the long sought cellular corepressor that inhibits the expression of HPV-encoded E6 and E7 oncoproteins that antagonize p53 and pRB tumor suppressor activity, resp.  The dual role of Brd4 in gene activation and repression illustrates how a dynamic chromatin-binding adaptor is able to recruit distinct transcriptional regulators to modulate promoter activity through cell cycle progression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGohccDdZC9iKLVg90H21EOLACvtfcHk0ljHccTPlZIoYA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXks1Ghtbo%253D&md5=b025a5222b510a7009eca4289370b1d4</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1074%2Fjbc.R700001200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.R700001200%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DS.%2BY.%26aulast%3DChiang%26aufirst%3DC.%2BM.%26atitle%3DThe%2520double%2520bromodomain-containing%2520chromatin%2520adaptor%2520BRD4%2520and%2520transcriptional%2520regulation%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2007%26volume%3D282%26spage%3D13141%26epage%3D13145%26doi%3D10.1074%2Fjbc.R700001200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dhalluin, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carlson, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aggarwal, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, M.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, M.-M.</span></span> <span> </span><span class="NLM_article-title">Structure and ligand of a histone acetyltransferase bromodomain</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>399</i></span>,  <span class="NLM_fpage">491</span>– <span class="NLM_lpage">496</span>, <span class="refDoi"> DOI: 10.1038/20974</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1038%2F20974" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10365964" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADyaK1MXjvV2isLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=399&publication_year=1999&pages=491-496&author=C.+Dhalluinauthor=J.+E.+Carlsonauthor=L.+Zengauthor=C.+Heauthor=A.+K.+Aggarwalauthor=M.-M.+Zhouauthor=M.-M.+Zhou&title=Structure+and+ligand+of+a+histone+acetyltransferase+bromodomain&doi=10.1038%2F20974"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Structure and ligand of a histone acetyltransferase bromodomain</span></div><div class="casAuthors">Dhalluin, Christophe; Carlson, Justin E.; Zeng, Lei; He, Cheng; Aggarwal, Aneel K.; Zhou, Ming-Ming</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London)</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">399</span>
        (<span class="NLM_cas:issue">6735</span>),
    <span class="NLM_cas:pages">491-496</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Macmillan Magazines</span>)
        </div><div class="casAbstract">Histone acetylation is important in chromatin remodelling and gene activation.  Nearly all known histone-acetyltransferase (HAT)-assocd. transcriptional co-activators contain bromodomains, which are ∼110-amino-acid modules found in many chromatin-assocd. proteins.  Despite the wide occurrence of these bromodomains, their three-dimensional structure and binding partners remain unknown.  Here we report the soln. structure of the bromodomain of the HAT co-activator P/CAF (p300/CBP-assocd. factor).  The structure reveals an unusual left-handed up-and-down four-helix bundle.  In addn., we show by a combination of structural and site-directed mutagenesis studies that bromodomains can interact specifically with acetylated lysine, making them the first known protein modules to do so.  The nature of the recognition of acetyl-lysine by the P/CAF bromodomain is similar to that of acetyl-CoA by histone acetyltransferase.  Thus, the bromodomain is functionally linked to the HAT activity of co-activators in the regulation of gene transcription.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrsTbwcjeBG_rVg90H21EOLACvtfcHk0lj3aWwd0SGFOA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXjvV2isLo%253D&md5=ff1063ba81d81b431065dd6e3751ba79</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1038%2F20974&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F20974%26sid%3Dliteratum%253Aachs%26aulast%3DDhalluin%26aufirst%3DC.%26aulast%3DCarlson%26aufirst%3DJ.%2BE.%26aulast%3DZeng%26aufirst%3DL.%26aulast%3DHe%26aufirst%3DC.%26aulast%3DAggarwal%26aufirst%3DA.%2BK.%26aulast%3DZhou%26aufirst%3DM.-M.%26aulast%3DZhou%26aufirst%3DM.-M.%26atitle%3DStructure%2520and%2520ligand%2520of%2520a%2520histone%2520acetyltransferase%2520bromodomain%26jtitle%3DNature%26date%3D1999%26volume%3D399%26spage%3D491%26epage%3D496%26doi%3D10.1038%2F20974" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Devaiah, B. N</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Case-Borden, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gegonne, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hsu, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meerzaman, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dey, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ozato, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singer, D. S</span></span> <span> </span><span class="NLM_article-title">Singer D. S. BRD4 is a histone acetyltransferase that evicts nucleosomes from chromatin</span>. <i>Nat. Struct. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">540</span>– <span class="NLM_lpage">548</span>, <span class="refDoi"> DOI: 10.1038/nsmb.3228</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1038%2Fnsmb.3228" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=27159561" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BC28XnsFaitbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2016&pages=540-548&author=B.+N+Devaiahauthor=C.+Case-Bordenauthor=A.+Gegonneauthor=C.+H.+Hsuauthor=Q.+Chenauthor=D.+Meerzamanauthor=A.+Deyauthor=K.+Ozatoauthor=D.+S+Singer&title=Singer+D.+S.+BRD4+is+a+histone+acetyltransferase+that+evicts+nucleosomes+from+chromatin&doi=10.1038%2Fnsmb.3228"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">BRD4 is a histone acetyltransferase that evicts nucleosomes from chromatin</span></div><div class="casAuthors">Devaiah, Ballachanda N.; Case-Borden, Chanelle; Gegonne, Anne; Hsu, Chih Hao; Chen, Qingrong; Meerzaman, Daoud; Dey, Anup; Ozato, Keiko; Singer, Dinah S.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Structural & Molecular Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">540-548</span>CODEN:
                <span class="NLM_cas:coden">NSMBCU</span>;
        ISSN:<span class="NLM_cas:issn">1545-9993</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Bromodomain protein 4 (BRD4) is a chromatin-binding protein implicated in cancer and autoimmune diseases that functions as a scaffold for transcription factors at promoters and super-enhancers.  Although chromatin decompaction and transcriptional activation of target genes are assocd. with BRD4 binding, the mechanisms involved are unknown.  We report that BRD4 is a histone acetyltransferase (HAT) that acetylates histones H3 and H4 with a pattern distinct from those of other HATs.  Both mouse and human BRD4 have intrinsic HAT activity.  Importantly, BRD4 acetylates H3 K122, a residue crit. for nucleosome stability, thus resulting in nucleosome eviction and chromatin decompaction.  Nucleosome clearance by BRD4 occurs genome wide, including at its targets MYC, FOS and AURKB (Aurora B kinase), resulting in increased transcription.  These findings suggest a model wherein BRD4 actively links chromatin structure and transcription: it mediates chromatin decompaction by acetylating and evicting nucleosomes at target genes, thereby activating transcription.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpiIuN9x3auYLVg90H21EOLACvtfcHk0lj3aWwd0SGFOA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XnsFaitbw%253D&md5=ce482733951b61a9cf0d024b8cd60d30</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1038%2Fnsmb.3228&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnsmb.3228%26sid%3Dliteratum%253Aachs%26aulast%3DDevaiah%26aufirst%3DB.%2BN%26aulast%3DCase-Borden%26aufirst%3DC.%26aulast%3DGegonne%26aufirst%3DA.%26aulast%3DHsu%26aufirst%3DC.%2BH.%26aulast%3DChen%26aufirst%3DQ.%26aulast%3DMeerzaman%26aufirst%3DD.%26aulast%3DDey%26aufirst%3DA.%26aulast%3DOzato%26aufirst%3DK.%26aulast%3DSinger%26aufirst%3DD.%2BS%26atitle%3DSinger%2520D.%2520S.%2520BRD4%2520is%2520a%2520histone%2520acetyltransferase%2520that%2520evicts%2520nucleosomes%2520from%2520chromatin%26jtitle%3DNat.%2520Struct.%2520Mol.%2520Biol.%26date%3D2016%26volume%3D23%26spage%3D540%26epage%3D548%26doi%3D10.1038%2Fnsmb.3228" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span> <span> </span><span class="NLM_article-title">Targeting bromodomains: epigenetic readers of lysine acetylation</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">337</span>– <span class="NLM_lpage">356</span>, <span class="refDoi"> DOI: 10.1038/nrd4286</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1038%2Fnrd4286" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=24751816" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BC2cXmsFalsr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2014&pages=337-356&author=P.+Filippakopoulosauthor=S.+Knapp&title=Targeting+bromodomains%3A+epigenetic+readers+of+lysine+acetylation&doi=10.1038%2Fnrd4286"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting bromodomains: epigenetic readers of lysine acetylation</span></div><div class="casAuthors">Filippakopoulos, Panagis; Knapp, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">337-356</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Lysine acetylation is a key mechanism that regulates chromatin structure; aberrant acetylation levels have been linked to the development of several diseases.  Acetyl-lysine modifications create docking sites for bromodomains, which are small interaction modules found on diverse proteins, some of which have a key role in the acetylation-dependent assembly of transcriptional regulator complexes.  These complexes can then initiate transcriptional programs that result in phenotypic changes.  The recent discovery of potent and highly specific inhibitors for the BET (bromodomain and extra-terminal) family of bromodomains has stimulated intensive research activity in diverse therapeutic areas, particularly in oncol., where BET proteins regulate the expression of key oncogenes and anti-apoptotic proteins.  In addn., targeting BET bromodomains could hold potential for the treatment of inflammation and viral infection.  Here, we highlight recent progress in the development of bromodomain inhibitors, and their potential applications in drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrOSIcwBJKj4LVg90H21EOLACvtfcHk0lj3aWwd0SGFOA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXmsFalsr4%253D&md5=77049c82ce6a19b4685e3b7950399a07</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1038%2Fnrd4286&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd4286%26sid%3Dliteratum%253Aachs%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DKnapp%26aufirst%3DS.%26atitle%3DTargeting%2520bromodomains%253A%2520epigenetic%2520readers%2520of%2520lysine%2520acetylation%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2014%26volume%3D13%26spage%3D337%26epage%3D356%26doi%3D10.1038%2Fnrd4286" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Spriano, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stathis, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bertoni, F.</span></span> <span> </span><span class="NLM_article-title">Targeting BET bromodomain proteins in cancer: The example of lymphomas</span>. <i>Pharmacol. Ther.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>215</i></span>,  <span class="NLM_fpage">107631</span>, <span class="refDoi"> DOI: 10.1016/j.pharmthera.2020.107631</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1016%2Fj.pharmthera.2020.107631" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=32693114" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhsVyisrbI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=215&publication_year=2020&pages=107631&author=F.+Sprianoauthor=A.+Stathisauthor=F.+Bertoni&title=Targeting+BET+bromodomain+proteins+in+cancer%3A+The+example+of+lymphomas&doi=10.1016%2Fj.pharmthera.2020.107631"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting BET bromodomain proteins in cancer: The example of lymphomas</span></div><div class="casAuthors">Spriano, Filippo; Stathis, Anastasios; Bertoni, Francesco</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacology & Therapeutics</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">215</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">107631</span>CODEN:
                <span class="NLM_cas:coden">PHTHDT</span>;
        ISSN:<span class="NLM_cas:issn">0163-7258</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">A review.  The Bromo- and Extra-Terminal domain (BET) family proteins act as ''readers'' of acetylated histones and they are important transcription regulators.  BRD2, BRD3, BRD4 and BRDT, part of the BET family, are important in different tumors, where upregulation or translocation often occurs.  The potential of targeting BET proteins as anti-cancer treatment originated with data obtained with a first series of compds., and there are now several data supporting BET inhibition in both solid tumors and hematol. malignancies.  Despite very pos. preclin. data in different tumor types, the clin. results have been so far moderate.  Using lymphoma as an example to review the data produced in the lab. and in the context of the early clin. trials, we discuss the modalities to make BET targeting more efficient both generating novel generation of compds. and by exploring the combination with small mols. affecting various signaling pathways, BCL2, or DNA damage response signaling, but also with addnl. epigenetic agents and with immunotherapy.  We also discuss the mechanisms of resistance and the toxicity profiles so far reported.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp3Jn61Q24_wrVg90H21EOLACvtfcHk0lg6BnwOA2fNGQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhsVyisrbI&md5=cdb034e619849cb681f48c5b03f7cde8</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1016%2Fj.pharmthera.2020.107631&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.pharmthera.2020.107631%26sid%3Dliteratum%253Aachs%26aulast%3DSpriano%26aufirst%3DF.%26aulast%3DStathis%26aufirst%3DA.%26aulast%3DBertoni%26aufirst%3DF.%26atitle%3DTargeting%2520BET%2520bromodomain%2520proteins%2520in%2520cancer%253A%2520The%2520example%2520of%2520lymphomas%26jtitle%3DPharmacol.%2520Ther.%26date%3D2020%26volume%3D215%26spage%3D107631%26doi%3D10.1016%2Fj.pharmthera.2020.107631" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wu, S. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, C. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, H. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, C. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zuo, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsai, S. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsai, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ge, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wan, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiang, C. M.</span></span> <span> </span><span class="NLM_article-title">Opposing functions of BRD4 isoforms in breast cancer</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>78</i></span>,  <span class="NLM_fpage">1114</span>– <span class="NLM_lpage">1132</span>, <span class="refDoi"> DOI: 10.1016/j.molcel.2020.04.034</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1016%2Fj.molcel.2020.04.034" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=32446320" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhtVaqt7jP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=78&publication_year=2020&pages=1114-1132&author=S.+Y.+Wuauthor=C.+F.+Leeauthor=H.+T.+Laiauthor=C.+T.+Yuauthor=J.+E.+Leeauthor=H.+Zuoauthor=S.+Y.+Tsaiauthor=M.+J.+Tsaiauthor=K.+Geauthor=Y.+Wanauthor=C.+M.+Chiang&title=Opposing+functions+of+BRD4+isoforms+in+breast+cancer&doi=10.1016%2Fj.molcel.2020.04.034"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Opposing Functions of BRD4 Isoforms in Breast Cancer</span></div><div class="casAuthors">Wu, Shwu-Yuan; Lee, Chien-Fei; Lai, Hsien-Tsung; Yu, Cheng-Tai; Lee, Ji-Eun; Zuo, Hao; Tsai, Sophia Y.; Tsai, Ming-Jer; Ge, Kai; Wan, Yihong; Chiang, Cheng-Ming</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">78</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1114-1132.e10</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Bromodomain-contg. protein 4 (BRD4) is a cancer therapeutic target in ongoing clin. trials disrupting primarily BRD4-regulated transcription programs.  The role of BRD4 in cancer has been attributed mainly to the abundant long isoform (BRD4-L).  Here we show, by isoform-specific knockdown and endogenous protein detection, along with transgene expression, the less abundant BRD4 short isoform (BRD4-S) is oncogenic while BRD4-L is tumor-suppressive in breast cancer cell proliferation and migration, as well as mammary tumor formation and metastasis.  Through integrated RNA-seq, genome-wide ChIP-seq, and CUT&RUN assocn. profiling, we identify the Engrailed-1 (EN1) homeobox transcription factor as a key BRD4-S coregulator, particularly in triple-neg. breast cancer.  BRD4-S and EN1 comodulate the extracellular matrix (ECM)-assocd. matrisome network, including type II cystatin gene cluster, mucin 5, and cathepsin loci, via enhancer regulation of cancer-assocd. genes and pathways.  Our work highlights the importance of targeted therapies for the oncogenic, but not tumor-suppressive, activity of BRD4.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo9HICVoywZRbVg90H21EOLACvtfcHk0lg6BnwOA2fNGQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhtVaqt7jP&md5=f330aa48d11b0dd2b7fb8193b4b05dbd</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2020.04.034&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2020.04.034%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DS.%2BY.%26aulast%3DLee%26aufirst%3DC.%2BF.%26aulast%3DLai%26aufirst%3DH.%2BT.%26aulast%3DYu%26aufirst%3DC.%2BT.%26aulast%3DLee%26aufirst%3DJ.%2BE.%26aulast%3DZuo%26aufirst%3DH.%26aulast%3DTsai%26aufirst%3DS.%2BY.%26aulast%3DTsai%26aufirst%3DM.%2BJ.%26aulast%3DGe%26aufirst%3DK.%26aulast%3DWan%26aufirst%3DY.%26aulast%3DChiang%26aufirst%3DC.%2BM.%26atitle%3DOpposing%2520functions%2520of%2520BRD4%2520isoforms%2520in%2520breast%2520cancer%26jtitle%3DMol.%2520Cell%26date%3D2020%26volume%3D78%26spage%3D1114%26epage%3D1132%26doi%3D10.1016%2Fj.molcel.2020.04.034" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wu, S. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, A. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, H. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiang, C. M.</span></span> <span> </span><span class="NLM_article-title">Phospho switch triggers BRD4 chromatin binding and activator recruitment for gene-specific targeting</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>49</i></span>,  <span class="NLM_fpage">843</span>– <span class="NLM_lpage">857</span>, <span class="refDoi"> DOI: 10.1016/j.molcel.2012.12.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1016%2Fj.molcel.2012.12.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=23317504" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BC3sXnvFersQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2013&pages=843-857&author=S.+Y.+Wuauthor=A.+Y.+Leeauthor=H.+T.+Laiauthor=H.+Zhangauthor=C.+M.+Chiang&title=Phospho+switch+triggers+BRD4+chromatin+binding+and+activator+recruitment+for+gene-specific+targeting&doi=10.1016%2Fj.molcel.2012.12.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Phospho switch triggers Brd4 chromatin binding and activator recruitment for gene-specific targeting</span></div><div class="casAuthors">Wu, Shwu-Yuan; Lee, A-Young; Lai, Hsien-Tsung; Zhang, Hong; Chiang, Cheng-Ming</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">843-857</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Bromodomain-contg. protein 4 (Brd4) is an epigenetic reader and transcriptional regulator recently identified as a cancer therapeutic target for acute myeloid leukemia, multiple myeloma, and Burkitt's lymphoma.  Although chromatin targeting is a crucial function of Brd4, there is little understanding of how bromodomains that bind acetylated histones are regulated, nor how the gene-specific activity of Brd4 is detd.  Via interaction screen and domain mapping, we identified p53 as a functional partner of Brd4.  Interestingly, Brd4 assocn. with p53 is modulated by casein kinase II (CK2)-mediated phosphorylation of a conserved acidic region in Brd4 that selectively contacts either a juxtaposed bromodomain or an adjacent basic region to dictate the ability of Brd4 binding to chromatin and also the recruitment of p53 to regulated promoters.  The unmasking of bromodomains and activator recruitment, concurrently triggered by the CK2 phospho switch, provide an intriguing mechanism for gene-specific targeting by a universal epigenetic reader.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoAKJqWUMTaTbVg90H21EOLACvtfcHk0lg6BnwOA2fNGQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXnvFersQ%253D%253D&md5=39eeb31c048a047419a169e0991c8dd8</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2012.12.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2012.12.006%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DS.%2BY.%26aulast%3DLee%26aufirst%3DA.%2BY.%26aulast%3DLai%26aufirst%3DH.%2BT.%26aulast%3DZhang%26aufirst%3DH.%26aulast%3DChiang%26aufirst%3DC.%2BM.%26atitle%3DPhospho%2520switch%2520triggers%2520BRD4%2520chromatin%2520binding%2520and%2520activator%2520recruitment%2520for%2520gene-specific%2520targeting%26jtitle%3DMol.%2520Cell%26date%3D2013%26volume%3D49%26spage%3D843%26epage%3D857%26doi%3D10.1016%2Fj.molcel.2012.12.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, H. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ge, J. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeid, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harris, I. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jovanović, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murphy, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qiu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Endress, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reyes, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lim, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Font-Tello, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Syamala, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiao, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reddy Chilamakuri, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Papachristou, E. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">D’Santos, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anand, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hinohara, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McDonald, T. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luoma, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Modiste, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen, Q. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michel, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cejas, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kadoch, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jaffe, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wucherpfennig, K. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Long, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carroll, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brugge, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michor, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Polyak, K.</span></span> <span> </span><span class="NLM_article-title">Synthetic lethal and resistance interactions with BET bromodomain inhibitors in triple-negative breast cancer</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>78</i></span>,  <span class="NLM_fpage">1096</span>– <span class="NLM_lpage">1113</span>, <span class="refDoi"> DOI: 10.1016/j.molcel.2020.04.027</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1016%2Fj.molcel.2020.04.027" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=32416067" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BB3cXpsl2jt7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=78&publication_year=2020&pages=1096-1113&author=S.+Shuauthor=H.+J.+Wuauthor=J.+Y.+Geauthor=R.+Zeidauthor=I.+S.+Harrisauthor=B.+Jovanovi%C4%87author=K.+Murphyauthor=B.+Wangauthor=X.+Qiuauthor=J.+E.+Endressauthor=J.+Reyesauthor=K.+Limauthor=A.+Font-Telloauthor=S.+Syamalaauthor=T.+Xiaoauthor=C.+S.+Reddy+Chilamakuriauthor=E.+K.+Papachristouauthor=C.+D%E2%80%99Santosauthor=J.+Anandauthor=K.+Hinoharaauthor=W.+Liauthor=T.+O.+McDonaldauthor=A.+Luomaauthor=R.+J.+Modisteauthor=Q.+D.+Nguyenauthor=B.+Michelauthor=P.+Cejasauthor=C.+Kadochauthor=J.+D.+Jaffeauthor=K.+W.+Wucherpfennigauthor=J.+Qiauthor=X.+S.+Liuauthor=H.+Longauthor=M.+Brownauthor=J.+S.+Carrollauthor=J.+S.+Bruggeauthor=J.+Bradnerauthor=F.+Michorauthor=K.+Polyak&title=Synthetic+lethal+and+resistance+interactions+with+BET+bromodomain+inhibitors+in+triple-negative+breast+cancer&doi=10.1016%2Fj.molcel.2020.04.027"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Synthetic Lethal and Resistance Interactions with BET Bromodomain Inhibitors in Triple-Negative Breast Cancer</span></div><div class="casAuthors">Shu, Shaokun; Wu, Hua-Jun; Ge, Jennifer Y.; Zeid, Rhamy; Harris, Isaac S.; Jovanovic, Bojana; Murphy, Katherine; Wang, Binbin; Qiu, Xintao; Endress, Jennifer E.; Reyes, Jaime; Lim, Klothilda; Font-Tello, Alba; Syamala, Sudeepa; Xiao, Tengfei; Reddy Chilamakuri, Chandra Sekhar; Papachristou, Evangelia K.; D'Santos, Clive; Anand, Jayati; Hinohara, Kunihiko; Li, Wei; McDonald, Thomas O.; Luoma, Adrienne; Modiste, Rebecca J.; Nguyen, Quang-De; Michel, Brittany; Cejas, Paloma; Kadoch, Cigall; Jaffe, Jacob D.; Wucherpfennig, Kai W.; Qi, Jun; Liu, X. Shirley; Long, Henry; Brown, Myles; Carroll, Jason S.; Brugge, Joan S.; Bradner, James; Michor, Franziska; Polyak, Kornelia</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">78</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1096-1113.e8</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">BET bromodomain inhibitors (BBDIs) are candidate therapeutic agents for triple-neg. breast cancer (TNBC) and other cancer types, but inherent and acquired resistance to BBDIs limits their potential clin. use.  Using CRISPR and small-mol. inhibitor screens combined with comprehensive mol. profiling of BBDI response and resistance, we identified synthetic lethal interactions with BBDIs and genes that, when deleted, confer resistance.  We obsd. synergy with regulators of cell cycle progression, YAP, AXL, and SRC signaling, and chemotherapeutic agents.  We also uncovered functional similarities and differences among BRD2, BRD4, and BRD7.  Although deletion of BRD2 enhances sensitivity to BBDIs, BRD7 loss leads to gain of TEAD-YAP chromatin binding and luminal features assocd. with BBDI resistance.  Single-cell RNA-seq, ATAC-seq, and cellular barcoding anal. of BBDI responses in sensitive and resistant cell lines highlight significant heterogeneity among samples and demonstrate that BBDI resistance can be pre-existing or acquired.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo4Sz5TPbxoKLVg90H21EOLACvtfcHk0lhAcGzE6vzk0Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXpsl2jt7g%253D&md5=8aabf40e0d9ca1c33c7876a0d68c1d65</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2020.04.027&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2020.04.027%26sid%3Dliteratum%253Aachs%26aulast%3DShu%26aufirst%3DS.%26aulast%3DWu%26aufirst%3DH.%2BJ.%26aulast%3DGe%26aufirst%3DJ.%2BY.%26aulast%3DZeid%26aufirst%3DR.%26aulast%3DHarris%26aufirst%3DI.%2BS.%26aulast%3DJovanovi%25C4%2587%26aufirst%3DB.%26aulast%3DMurphy%26aufirst%3DK.%26aulast%3DWang%26aufirst%3DB.%26aulast%3DQiu%26aufirst%3DX.%26aulast%3DEndress%26aufirst%3DJ.%2BE.%26aulast%3DReyes%26aufirst%3DJ.%26aulast%3DLim%26aufirst%3DK.%26aulast%3DFont-Tello%26aufirst%3DA.%26aulast%3DSyamala%26aufirst%3DS.%26aulast%3DXiao%26aufirst%3DT.%26aulast%3DReddy%2BChilamakuri%26aufirst%3DC.%2BS.%26aulast%3DPapachristou%26aufirst%3DE.%2BK.%26aulast%3DD%25E2%2580%2599Santos%26aufirst%3DC.%26aulast%3DAnand%26aufirst%3DJ.%26aulast%3DHinohara%26aufirst%3DK.%26aulast%3DLi%26aufirst%3DW.%26aulast%3DMcDonald%26aufirst%3DT.%2BO.%26aulast%3DLuoma%26aufirst%3DA.%26aulast%3DModiste%26aufirst%3DR.%2BJ.%26aulast%3DNguyen%26aufirst%3DQ.%2BD.%26aulast%3DMichel%26aufirst%3DB.%26aulast%3DCejas%26aufirst%3DP.%26aulast%3DKadoch%26aufirst%3DC.%26aulast%3DJaffe%26aufirst%3DJ.%2BD.%26aulast%3DWucherpfennig%26aufirst%3DK.%2BW.%26aulast%3DQi%26aufirst%3DJ.%26aulast%3DLiu%26aufirst%3DX.%2BS.%26aulast%3DLong%26aufirst%3DH.%26aulast%3DBrown%26aufirst%3DM.%26aulast%3DCarroll%26aufirst%3DJ.%2BS.%26aulast%3DBrugge%26aufirst%3DJ.%2BS.%26aulast%3DBradner%26aufirst%3DJ.%26aulast%3DMichor%26aufirst%3DF.%26aulast%3DPolyak%26aufirst%3DK.%26atitle%3DSynthetic%2520lethal%2520and%2520resistance%2520interactions%2520with%2520BET%2520bromodomain%2520inhibitors%2520in%2520triple-negative%2520breast%2520cancer%26jtitle%3DMol.%2520Cell%26date%3D2020%26volume%3D78%26spage%3D1096%26epage%3D1113%26doi%3D10.1016%2Fj.molcel.2020.04.027" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wu, S. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nin, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, A. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simanski, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kodadek, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiang, C. M.</span></span> <span> </span><span class="NLM_article-title">BRD4 phosphorylation regulates HPV E2-mediated viral transcription, origin replication, and cellular MMP-9 expression</span>. <i>Cell Rep.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">1733</span>– <span class="NLM_lpage">1748</span>, <span class="refDoi"> DOI: 10.1016/j.celrep.2016.07.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1016%2Fj.celrep.2016.07.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=27477287" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BC28Xht1KlsbnO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2016&pages=1733-1748&author=S.+Y.+Wuauthor=D.+S.+Ninauthor=A.+Y.+Leeauthor=S.+Simanskiauthor=T.+Kodadekauthor=C.+M.+Chiang&title=BRD4+phosphorylation+regulates+HPV+E2-mediated+viral+transcription%2C+origin+replication%2C+and+cellular+MMP-9+expression&doi=10.1016%2Fj.celrep.2016.07.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">BRD4 Phosphorylation Regulates HPV E2-Mediated Viral Transcription, Origin Replication, and Cellular MMP-9 Expression</span></div><div class="casAuthors">Wu, Shwu-Yuan; Nin, Dawn Sijin; Lee, A-Young; Simanski, Scott; Kodadek, Thomas; Chiang, Cheng-Ming</div><div class="citationInfo"><span class="NLM_cas:title">Cell Reports</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1733-1748</span>CODEN:
                <span class="NLM_cas:coden">CREED8</span>;
        ISSN:<span class="NLM_cas:issn">2211-1247</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Post-translational modification can modulate protein conformation and alter binding partner recruitment within gene regulatory regions.  Here, we report that bromodomain-contg. protein 4 (BRD4), a transcription co-factor and chromatin regulator, uses a phosphorylation-induced switch mechanism to recruit E2 protein encoded by cancer-assocd. human papillomavirus (HPV) to viral early gene and cellular matrix metalloproteinase-9 (MMP-9) promoters.  Enhanced MMP-9 expression, induced upon keratinocyte differentiation, occurs via BRD4-dependent recruitment of active AP-1 and NF-κB to their target sequences.  This is triggered by replacement of AP-1 family members JunB and JunD by c-Jun and by re-localization of NF-κB from the cytoplasm to the nucleus.  In addn., BRD4 phosphorylation is crit. for E2- and origin-dependent HPV DNA replication.  A class of phospho-BRD4-targeting compds., distinct from the BET bromodomain inhibitors, effectively blocks BRD4 phosphorylation-specific functions in transcription and factor recruitment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGomvZx5RmFIRLVg90H21EOLACvtfcHk0lhAcGzE6vzk0Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xht1KlsbnO&md5=10bde9118396d8a5a1f4614fb64a566c</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1016%2Fj.celrep.2016.07.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.celrep.2016.07.001%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DS.%2BY.%26aulast%3DNin%26aufirst%3DD.%2BS.%26aulast%3DLee%26aufirst%3DA.%2BY.%26aulast%3DSimanski%26aufirst%3DS.%26aulast%3DKodadek%26aufirst%3DT.%26aulast%3DChiang%26aufirst%3DC.%2BM.%26atitle%3DBRD4%2520phosphorylation%2520regulates%2520HPV%2520E2-mediated%2520viral%2520transcription%252C%2520origin%2520replication%252C%2520and%2520cellular%2520MMP-9%2520expression%26jtitle%3DCell%2520Rep.%26date%3D2016%26volume%3D16%26spage%3D1733%26epage%3D1748%26doi%3D10.1016%2Fj.celrep.2016.07.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, C. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, H. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Witwicki, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tabassum, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Janiszewska, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin Huh, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ryan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doherty, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mohammed, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stover, D. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ekram, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peluffo, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">D’Santos, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krop, I. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dillon, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McKeown, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ott, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ni, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rao, P. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duarte, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, S.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiang, C.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anders, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winer, E. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Letai, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barry, W. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carroll, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Long, H. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shirley Liu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meyer, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Polyak, K.</span></span> <span> </span><span class="NLM_article-title">Response and resistance to BET bromodomain inhibitors in triple-negative breast cancer</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>529</i></span>,  <span class="NLM_fpage">413</span>– <span class="NLM_lpage">417</span>, <span class="refDoi"> DOI: 10.1038/nature16508</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1038%2Fnature16508" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=26735014" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BC28Xns1Gmtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=529&publication_year=2016&pages=413-417&author=S.+Shuauthor=C.+Y.+Linauthor=H.+H.+Heauthor=R.+M.+Witwickiauthor=D.+P.+Tabassumauthor=J.+M.+Robertsauthor=M.+Janiszewskaauthor=S.+Jin+Huhauthor=Y.+Liangauthor=J.+Ryanauthor=E.+Dohertyauthor=H.+Mohammedauthor=H.+Guoauthor=D.+G.+Stoverauthor=M.+B.+Ekramauthor=G.+Peluffoauthor=J.+Brownauthor=C.+D%E2%80%99Santosauthor=I.+E.+Kropauthor=D.+Dillonauthor=M.+McKeownauthor=C.+Ottauthor=J.+Qiauthor=M.+Niauthor=P.+K.+Raoauthor=M.+Duarteauthor=S.-Y.+Wuauthor=C.-M.+Chiangauthor=L.+Andersauthor=R.+A.+Youngauthor=E.+P.+Winerauthor=A.+Letaiauthor=W.+T.+Barryauthor=J.+S.+Carrollauthor=H.+W.+Longauthor=M.+Brownauthor=X.+Shirley+Liuauthor=C.+A.+Meyerauthor=J.+E.+Bradnerauthor=K.+Polyak&title=Response+and+resistance+to+BET+bromodomain+inhibitors+in+triple-negative+breast+cancer&doi=10.1038%2Fnature16508"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Response and resistance to BET bromodomain inhibitors in triple-negative breast cancer</span></div><div class="casAuthors">Shu, Shaokun; Lin, Charles Y.; He, Housheng Hansen; Witwicki, Robert M.; Tabassum, Doris P.; Roberts, Justin M.; Janiszewska, Michalina; Huh, Sung Jin; Liang, Yi; Ryan, Jeremy; Doherty, Ernest; Mohammed, Hisham; Guo, Hao; Stover, Daniel G.; Ekram, Muhammad B.; Peluffo, Guillermo; Brown, Jonathan; D'Santos, Clive; Krop, Ian E.; Dillon, Deborah; McKeown, Michael; Ott, Christopher; Qi, Jun; Ni, Min; Rao, Prakash K.; Duarte, Melissa; Wu, Shwu-Yuan; Chiang, Cheng-Ming; Anders, Lars; Young, Richard A.; Winer, Eric P.; Letai, Antony; Barry, William T.; Carroll, Jason S.; Long, Henry W.; Brown, Myles; Shirley Liu, X.; Meyer, Clifford A.; Bradner, James E.; Polyak, Kornelia</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">529</span>
        (<span class="NLM_cas:issue">7586</span>),
    <span class="NLM_cas:pages">413-417</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Triple-neg. breast cancer (TNBC) is a heterogeneous and clin. aggressive disease for which there is no targeted therapy.  BET bromodomain inhibitors, which have shown efficacy in several models of cancer, have not been evaluated in TNBC.  These inhibitors displace BET bromodomain proteins such as BRD4 from chromatin by competing with their acetyl-lysine recognition modules, leading to inhibition of oncogenic transcriptional programs.  Here we report the preferential sensitivity of TNBCs to BET bromodomain inhibition in vitro and in vivo, establishing a rationale for clin. investigation and further motivation to understand mechanisms of resistance.  In paired cell lines selected for acquired resistance to BET inhibition from previously sensitive TNBCs, we failed to identify gatekeeper mutations, new driver events or drug pump activation.  BET-resistant TNBC cells remain dependent on wild-type BRD4, which supports transcription and cell proliferation in a bromodomain-independent manner.  Proteomic studies of resistant TNBC identify strong assocn. with MED1 and hyper-phosphorylation of BRD4 attributable to decreased activity of PP2A, identified here as a principal BRD4 serine phosphatase.  Together, these studies provide a rationale for BET inhibition in TNBC and present mechanism-based combination strategies to anticipate clin. drug resistance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqZu5A85pdQDbVg90H21EOLACvtfcHk0lhWDXuvj3ERQA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xns1Gmtw%253D%253D&md5=50a03861d9156ee6e344e95c5f5361b9</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1038%2Fnature16508&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature16508%26sid%3Dliteratum%253Aachs%26aulast%3DShu%26aufirst%3DS.%26aulast%3DLin%26aufirst%3DC.%2BY.%26aulast%3DHe%26aufirst%3DH.%2BH.%26aulast%3DWitwicki%26aufirst%3DR.%2BM.%26aulast%3DTabassum%26aufirst%3DD.%2BP.%26aulast%3DRoberts%26aufirst%3DJ.%2BM.%26aulast%3DJaniszewska%26aufirst%3DM.%26aulast%3DJin%2BHuh%26aufirst%3DS.%26aulast%3DLiang%26aufirst%3DY.%26aulast%3DRyan%26aufirst%3DJ.%26aulast%3DDoherty%26aufirst%3DE.%26aulast%3DMohammed%26aufirst%3DH.%26aulast%3DGuo%26aufirst%3DH.%26aulast%3DStover%26aufirst%3DD.%2BG.%26aulast%3DEkram%26aufirst%3DM.%2BB.%26aulast%3DPeluffo%26aufirst%3DG.%26aulast%3DBrown%26aufirst%3DJ.%26aulast%3DD%25E2%2580%2599Santos%26aufirst%3DC.%26aulast%3DKrop%26aufirst%3DI.%2BE.%26aulast%3DDillon%26aufirst%3DD.%26aulast%3DMcKeown%26aufirst%3DM.%26aulast%3DOtt%26aufirst%3DC.%26aulast%3DQi%26aufirst%3DJ.%26aulast%3DNi%26aufirst%3DM.%26aulast%3DRao%26aufirst%3DP.%2BK.%26aulast%3DDuarte%26aufirst%3DM.%26aulast%3DWu%26aufirst%3DS.-Y.%26aulast%3DChiang%26aufirst%3DC.-M.%26aulast%3DAnders%26aufirst%3DL.%26aulast%3DYoung%26aufirst%3DR.%2BA.%26aulast%3DWiner%26aufirst%3DE.%2BP.%26aulast%3DLetai%26aufirst%3DA.%26aulast%3DBarry%26aufirst%3DW.%2BT.%26aulast%3DCarroll%26aufirst%3DJ.%2BS.%26aulast%3DLong%26aufirst%3DH.%2BW.%26aulast%3DBrown%26aufirst%3DM.%26aulast%3DShirley%2BLiu%26aufirst%3DX.%26aulast%3DMeyer%26aufirst%3DC.%2BA.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26aulast%3DPolyak%26aufirst%3DK.%26atitle%3DResponse%2520and%2520resistance%2520to%2520BET%2520bromodomain%2520inhibitors%2520in%2520triple-negative%2520breast%2520cancer%26jtitle%3DNature%26date%3D2016%26volume%3D529%26spage%3D413%26epage%3D417%26doi%3D10.1038%2Fnature16508" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chiang, C. M.</span></span> <span> </span><span class="NLM_article-title">Phospho-BRD4: transcription plasticity and drug targeting</span>. <i>Drug Discovery Today: Technol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">17</span>– <span class="NLM_lpage">22</span>, <span class="refDoi"> DOI: 10.1016/j.ddtec.2016.05.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1016%2Fj.ddtec.2016.05.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=27769352" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A280%3ADC%252BC2srkvFOjsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2016&pages=17-22&author=C.+M.+Chiang&title=Phospho-BRD4%3A+transcription+plasticity+and+drug+targeting&doi=10.1016%2Fj.ddtec.2016.05.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Phospho-BRD4: transcription plasticity and drug targeting</span></div><div class="casAuthors">Chiang Cheng-Ming</div><div class="citationInfo"><span class="NLM_cas:title">Drug discovery today. Technologies</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">17-22</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BRD4 is an epigenetic regulator and transcription cofactor whose phosphorylation by CK2 and dephosphorylation by PP2A modulates its function in chromatin targeting, factor recruitment, and cancer progression.  While the bromodomains of BET family proteins, including BRD4, BRD2, BRD3 and BRDT, have been the primary targets of small compounds such as JQ1, I-BET and MS417 that show promising anticancer effects against some hematopoietic cancer and solid tumors, drug resistance upon prolonged treatment necessitates a better understanding of alternative pathways underlying not only the resistance but also persistent BET protein dependence for identifying new targets and effective combination therapy strategies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSEothR-V9Y-GhHGZei4OW9fW6udTcc2eYPPT_7lLu5Kbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2srkvFOjsA%253D%253D&md5=3337d696a5a0fdba97c265d2c15fd8df</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1016%2Fj.ddtec.2016.05.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ddtec.2016.05.003%26sid%3Dliteratum%253Aachs%26aulast%3DChiang%26aufirst%3DC.%2BM.%26atitle%3DPhospho-BRD4%253A%2520transcription%2520plasticity%2520and%2520drug%2520targeting%26jtitle%3DDrug%2520Discovery%2520Today%253A%2520Technol.%26date%3D2016%26volume%3D19%26spage%3D17%26epage%3D22%26doi%3D10.1016%2Fj.ddtec.2016.05.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cheung, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jaganathan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharma, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Konuma, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ren, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olson, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaplan, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Littman, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walsh, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiong, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, M. M.</span></span> <span> </span><span class="NLM_article-title">Distinct roles of BRD2 and BRD4 in potentiating the transcriptional program for Th17 cell differentiation</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>65</i></span>,  <span class="NLM_fpage">1068</span>– <span class="NLM_lpage">1080</span>, <span class="refDoi"> DOI: 10.1016/j.molcel.2016.12.022</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1016%2Fj.molcel.2016.12.022" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=28262505" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BC2sXjvVWqu78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=65&publication_year=2017&pages=1068-1080&author=K.+L.+Cheungauthor=F.+Zhangauthor=A.+Jaganathanauthor=R.+Sharmaauthor=Q.+Zhangauthor=T.+Konumaauthor=T.+Shenauthor=J.+Y.+Leeauthor=C.+Renauthor=C.+H.+Chenauthor=G.+Luauthor=M.+R.+Olsonauthor=W.+Zhangauthor=M.+H.+Kaplanauthor=D.+R.+Littmanauthor=M.+J.+Walshauthor=H.+Xiongauthor=L.+Zengauthor=M.+M.+Zhou&title=Distinct+roles+of+BRD2+and+BRD4+in+potentiating+the+transcriptional+program+for+Th17+cell+differentiation&doi=10.1016%2Fj.molcel.2016.12.022"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Distinct Roles of Brd2 and Brd4 in Potentiating the Transcriptional Program for Th17 Cell Differentiation</span></div><div class="casAuthors">Cheung, Ka Lung; Zhang, Fan; Jaganathan, Anbalagan; Sharma, Rajal; Zhang, Qiang; Konuma, Tsuyoshi; Shen, Tong; Lee, June-Yong; Ren, Chunyan; Chen, Chih-Hung; Lu, Geming; Olson, Matthew R.; Zhang, Weijia; Kaplan, Mark H.; Littman, Dan R.; Walsh, Martin J.; Xiong, Huabao; Zeng, Lei; Zhou, Ming-Ming</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">65</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1068-1080.e5</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">The BET proteins are major transcriptional regulators and have emerged as new drug targets, but their functional distinction has remained elusive.  In this study, we report that the BET family members Brd2 and Brd4 exert distinct genomic functions at genes whose transcription they co-regulate during mouse T helper 17 (Th17) cell differentiation.  Brd2 is assocd. with the chromatin insulator CTCF and the cohesin complex to support cis-regulatory enhancer assembly for gene transcriptional activation.  In this context, Brd2 binds the transcription factor Stat3 in an acetylation-sensitive manner and facilitates Stat3 recruitment to active enhancers occupied with transcription factors Irf4 and Batf.  In parallel, Brd4 temporally controls RNA polymerase II (Pol II) processivity during transcription elongation through cyclin T1 and Cdk9 recruitment and Pol II Ser2 phosphorylation.  Collectively, our study uncovers both sep. and interdependent Brd2 and Brd4 functions in potentiating the genetic program required for Th17 cell development and adaptive immunity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp06k1TG5JMHrVg90H21EOLACvtfcHk0lhWDXuvj3ERQA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXjvVWqu78%253D&md5=2f2cd782230c84d9e128b04a2d8bba27</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2016.12.022&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2016.12.022%26sid%3Dliteratum%253Aachs%26aulast%3DCheung%26aufirst%3DK.%2BL.%26aulast%3DZhang%26aufirst%3DF.%26aulast%3DJaganathan%26aufirst%3DA.%26aulast%3DSharma%26aufirst%3DR.%26aulast%3DZhang%26aufirst%3DQ.%26aulast%3DKonuma%26aufirst%3DT.%26aulast%3DShen%26aufirst%3DT.%26aulast%3DLee%26aufirst%3DJ.%2BY.%26aulast%3DRen%26aufirst%3DC.%26aulast%3DChen%26aufirst%3DC.%2BH.%26aulast%3DLu%26aufirst%3DG.%26aulast%3DOlson%26aufirst%3DM.%2BR.%26aulast%3DZhang%26aufirst%3DW.%26aulast%3DKaplan%26aufirst%3DM.%2BH.%26aulast%3DLittman%26aufirst%3DD.%2BR.%26aulast%3DWalsh%26aufirst%3DM.%2BJ.%26aulast%3DXiong%26aufirst%3DH.%26aulast%3DZeng%26aufirst%3DL.%26aulast%3DZhou%26aufirst%3DM.%2BM.%26atitle%3DDistinct%2520roles%2520of%2520BRD2%2520and%2520BRD4%2520in%2520potentiating%2520the%2520transcriptional%2520program%2520for%2520Th17%2520cell%2520differentiation%26jtitle%3DMol.%2520Cell%26date%3D2017%26volume%3D65%26spage%3D1068%26epage%3D1080%26doi%3D10.1016%2Fj.molcel.2016.12.022" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gilan, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rioja, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knezevic, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bell, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yeung, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harker, N. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lam, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bamborough, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petretich, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Urh, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Atkinson, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bassil, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vassiliadis, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burr, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Preston, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wellaway, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Werner, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michon, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gobbetti, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumar, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soden, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haynes, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vappiani, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tough, D. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dawson, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bantscheff, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindon, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drewes, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Demont, E. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daniels, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grandi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dawson, M. A.</span></span> <span> </span><span class="NLM_article-title">Selective targeting of BD1 and BD2 of the BET proteins in cancer and immunoinflammation</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>368</i></span>,  <span class="NLM_fpage">387</span>– <span class="NLM_lpage">394</span>, <span class="refDoi"> DOI: 10.1126/science.aaz8455</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1126%2Fscience.aaz8455" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=32193360" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BB3cXnslKrtrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=368&publication_year=2020&pages=387-394&author=O.+Gilanauthor=I.+Riojaauthor=K.+Knezevicauthor=M.+J.+Bellauthor=M.+M.+Yeungauthor=N.+R.+Harkerauthor=E.+Lamauthor=C.+W.+Chungauthor=P.+Bamboroughauthor=M.+Petretichauthor=M.+Urhauthor=S.+J.+Atkinsonauthor=A.+K.+Bassilauthor=E.+J.+Robertsauthor=D.+Vassiliadisauthor=M.+L.+Burrauthor=A.+Prestonauthor=C.+Wellawayauthor=T.+Wernerauthor=J.+R.+Grayauthor=A.+M.+Michonauthor=T.+Gobbettiauthor=V.+Kumarauthor=P.+E.+Sodenauthor=A.+Haynesauthor=J.+Vappianiauthor=D.+F.+Toughauthor=S.+Taylorauthor=S.+J.+Dawsonauthor=M.+Bantscheffauthor=M.+Lindonauthor=G.+Drewesauthor=E.+H.+Demontauthor=D.+L.+Danielsauthor=P.+Grandiauthor=R.+K.+Prinjhaauthor=M.+A.+Dawson&title=Selective+targeting+of+BD1+and+BD2+of+the+BET+proteins+in+cancer+and+immunoinflammation&doi=10.1126%2Fscience.aaz8455"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Selective targeting of BD1 and BD2 of the BET proteins in cancer and immunoinflammation</span></div><div class="casAuthors">Gilan, Omer; Rioja, Inmaculada; Knezevic, Kathy; Bell, Matthew J.; Yeung, Miriam M.; Harker, Nicola R.; Lam, Enid Y. N.; Chung, Chun-wa; Bamborough, Paul; Petretich, Massimo; Urh, Marjeta; Atkinson, Stephen J.; Bassil, Anna K.; Roberts, Emma J.; Vassiliadis, Dane; Burr, Marian L.; Preston, Alex G. S.; Wellaway, Christopher; Werner, Thilo; Gray, James R.; Michon, Anne-Marie; Gobbetti, Thomas; Kumar, Vinod; Soden, Peter E.; Haynes, Andrea; Vappiani, Johanna; Tough, David F.; Taylor, Simon; Dawson, Sarah-Jane; Bantscheff, Marcus; Lindon, Matthew; Drewes, Gerard; Demont, Emmanuel H.; Daniels, Danette L.; Grandi, Paola; Prinjha, Rab K.; Dawson, Mark A.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">368</span>
        (<span class="NLM_cas:issue">6489</span>),
    <span class="NLM_cas:pages">387-394</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">1095-9203</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">The two tandem bromodomains of the BET (bromodomain and extraterminal domain) proteins enable chromatin binding to facilitate transcription.  Drugs that inhibit both bromodomains equally have shown efficacy in certain malignant and inflammatory conditions.  To explore the individual functional contributions of the first (BD1) and second (BD2) bromodomains in biol. and therapy, we developed selective BD1 and BD2 inhibitors.  We found that steady-state gene expression primarily requires BD1, whereas the rapid increase of gene expression induced by inflammatory stimuli requires both BD1 and BD2 of all BET proteins.  BD1 inhibitors phenocopied the effects of pan-BET inhibitors in cancer models, whereas BD2 inhibitors were predominantly effective in models of inflammatory and autoimmune disease.  These insights into the differential requirement of BD1 and BD2 for the maintenance and induction of gene expression may guide future BET-targeted therapies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr078AuJubsqbVg90H21EOLACvtfcHk0lh8PdOo1pzxIQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXnslKrtrg%253D&md5=d78cfea091fcf5ee18735b02b2120dd7</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1126%2Fscience.aaz8455&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.aaz8455%26sid%3Dliteratum%253Aachs%26aulast%3DGilan%26aufirst%3DO.%26aulast%3DRioja%26aufirst%3DI.%26aulast%3DKnezevic%26aufirst%3DK.%26aulast%3DBell%26aufirst%3DM.%2BJ.%26aulast%3DYeung%26aufirst%3DM.%2BM.%26aulast%3DHarker%26aufirst%3DN.%2BR.%26aulast%3DLam%26aufirst%3DE.%26aulast%3DChung%26aufirst%3DC.%2BW.%26aulast%3DBamborough%26aufirst%3DP.%26aulast%3DPetretich%26aufirst%3DM.%26aulast%3DUrh%26aufirst%3DM.%26aulast%3DAtkinson%26aufirst%3DS.%2BJ.%26aulast%3DBassil%26aufirst%3DA.%2BK.%26aulast%3DRoberts%26aufirst%3DE.%2BJ.%26aulast%3DVassiliadis%26aufirst%3DD.%26aulast%3DBurr%26aufirst%3DM.%2BL.%26aulast%3DPreston%26aufirst%3DA.%26aulast%3DWellaway%26aufirst%3DC.%26aulast%3DWerner%26aufirst%3DT.%26aulast%3DGray%26aufirst%3DJ.%2BR.%26aulast%3DMichon%26aufirst%3DA.%2BM.%26aulast%3DGobbetti%26aufirst%3DT.%26aulast%3DKumar%26aufirst%3DV.%26aulast%3DSoden%26aufirst%3DP.%2BE.%26aulast%3DHaynes%26aufirst%3DA.%26aulast%3DVappiani%26aufirst%3DJ.%26aulast%3DTough%26aufirst%3DD.%2BF.%26aulast%3DTaylor%26aufirst%3DS.%26aulast%3DDawson%26aufirst%3DS.%2BJ.%26aulast%3DBantscheff%26aufirst%3DM.%26aulast%3DLindon%26aufirst%3DM.%26aulast%3DDrewes%26aufirst%3DG.%26aulast%3DDemont%26aufirst%3DE.%2BH.%26aulast%3DDaniels%26aufirst%3DD.%2BL.%26aulast%3DGrandi%26aufirst%3DP.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DDawson%26aufirst%3DM.%2BA.%26atitle%3DSelective%2520targeting%2520of%2520BD1%2520and%2520BD2%2520of%2520the%2520BET%2520proteins%2520in%2520cancer%2520and%2520immunoinflammation%26jtitle%3DScience%26date%3D2020%26volume%3D368%26spage%3D387%26epage%3D394%26doi%3D10.1126%2Fscience.aaz8455" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Crowe, B. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Larue, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hess, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kvaratskhelia, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Foster, M. P.</span></span> <span> </span><span class="NLM_article-title">Structure of the BRD4 ET domain bound to a C-terminal motif from gamma-retroviral integrases reveals a conserved mechanism of interaction</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>113</i></span>,  <span class="NLM_fpage">2086</span>– <span class="NLM_lpage">2091</span>, <span class="refDoi"> DOI: 10.1073/pnas.1516813113</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1073%2Fpnas.1516813113" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=26858406" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BC28Xit12ku7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=113&publication_year=2016&pages=2086-2091&author=B.+L.+Croweauthor=R.+C.+Larueauthor=C.+Yuanauthor=S.+Hessauthor=M.+Kvaratskheliaauthor=M.+P.+Foster&title=Structure+of+the+BRD4+ET+domain+bound+to+a+C-terminal+motif+from+gamma-retroviral+integrases+reveals+a+conserved+mechanism+of+interaction&doi=10.1073%2Fpnas.1516813113"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Structure of the Brd4 ET domain bound to a C-terminal motif from γ-retroviral integrases reveals a conserved mechanism of interaction</span></div><div class="casAuthors">Crowe, Brandon L.; Larue, Ross C.; Yuan, Chunhua; Hess, Sonja; Kvaratskhelia, Mamuka; Foster, Mark P.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">113</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2086-2091</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The bromodomain and extraterminal domain (BET) protein family (Brd2, Brd3, Brd4, and Brdt) are promising therapeutic targets for a range of diseases linked to transcriptional activation, cancer, viral latency, and viral integration.  Tandem bromodomains selectively tether BET proteins to chromatin by engaging cognate acetylated histone marks, and the extraterminal (ET) domain is the focal point for recruiting a range of cellular and viral proteins.  BET proteins guide γ-retroviral integration to transcription start sites and enhancers through bimodal interaction with chromatin and the γ-retroviral integrase (IN).  We report the NMR-derived soln. structure of the Brd4 ET domain bound to a conserved peptide sequence from the C terminus of murine leukemia virus (MLV) IN.  The complex reveals a protein-protein interaction governed by the binding-coupled folding of disordered regions in both interacting partners to form a well-structured intermol. three-stranded β sheet.  In addn., we show that a peptide comprising the ET binding motif (EBM) of MLV IN can disrupt the cognate interaction of Brd4 with histone-lysine N-methyltransferase NSD3, and that substitutions of Brd4 ET residues essential for binding MLV IN also impair interaction of Brd4 with a no. of cellular partners involved in transcriptional regulation and chromatin remodeling.  This suggests that γ-retroviruses have evolved the EBM to mimic a cognate interaction motif to achieve effective integration in host chromatin.  Collectively, our findings identify key structural features of the ET domain of Brd4 that allow for interactions with both cellular and viral proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoTRmI3Hgkr8LVg90H21EOLACvtfcHk0lh8PdOo1pzxIQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xit12ku7Y%253D&md5=9a03dcb089e5d800b39c8c42374d8661</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1516813113&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1516813113%26sid%3Dliteratum%253Aachs%26aulast%3DCrowe%26aufirst%3DB.%2BL.%26aulast%3DLarue%26aufirst%3DR.%2BC.%26aulast%3DYuan%26aufirst%3DC.%26aulast%3DHess%26aufirst%3DS.%26aulast%3DKvaratskhelia%26aufirst%3DM.%26aulast%3DFoster%26aufirst%3DM.%2BP.%26atitle%3DStructure%2520of%2520the%2520BRD4%2520ET%2520domain%2520bound%2520to%2520a%2520C-terminal%2520motif%2520from%2520gamma-retroviral%2520integrases%2520reveals%2520a%2520conserved%2520mechanism%2520of%2520interaction%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2016%26volume%3D113%26spage%3D2086%26epage%3D2091%26doi%3D10.1073%2Fpnas.1516813113" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bisgrove, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mahmoudi, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Henklein, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verdin, E.</span></span> <span> </span><span class="NLM_article-title">Conserved P-TEFb-interacting domain of BRD4 inhibits HIV transcription</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>104</i></span>,  <span class="NLM_fpage">13690</span>– <span class="NLM_lpage">13695</span>, <span class="refDoi"> DOI: 10.1073/pnas.0705053104</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1073%2Fpnas.0705053104" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=17690245" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BD2sXpvVOjsLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=104&publication_year=2007&pages=13690-13695&author=D.+A.+Bisgroveauthor=T.+Mahmoudiauthor=P.+Henkleinauthor=E.+Verdin&title=Conserved+P-TEFb-interacting+domain+of+BRD4+inhibits+HIV+transcription&doi=10.1073%2Fpnas.0705053104"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Conserved P-TEFb-interacting domain of BRD4 inhibits HIV transcription</span></div><div class="casAuthors">Bisgrove, Dwayne A.; Mahmoudi, Tokameh; Henklein, Peter; Verdin, Eric</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">104</span>
        (<span class="NLM_cas:issue">34</span>),
    <span class="NLM_cas:pages">13690-13695</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">We have identified a conserved region in the C-terminal domain of bromodomain-contg. protein 4 (BRD4) that mediates its specific interaction with pos. transcription elongation factor b (P-TEFb).  This domain is highly conserved in testis-specific bromodomain protein (BRDT) and Drosophila fs(1)h.  Both BRDT and fs(1)h specifically interact with P-TEFb in mammalian cells, and this interaction depends on their C-terminal domains.  Overexpression of the BRD4 P-TEFb-interacting domain disrupts the interaction between the HIV transactivator Tat and P-TEFb and suppresses the ability of Tat to transactivate the HIV promoter.  Incubation of cells with a synthetic peptide contg. the C-terminal domain of BRD4 interferes with transactivation of the HIV promoter by the Tat protein.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq7C_2ZREaOELVg90H21EOLACvtfcHk0liwg659WcGAvw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXpvVOjsLc%253D&md5=6091d31044bc49d054c0c4b89f690384</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0705053104&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0705053104%26sid%3Dliteratum%253Aachs%26aulast%3DBisgrove%26aufirst%3DD.%2BA.%26aulast%3DMahmoudi%26aufirst%3DT.%26aulast%3DHenklein%26aufirst%3DP.%26aulast%3DVerdin%26aufirst%3DE.%26atitle%3DConserved%2520P-TEFb-interacting%2520domain%2520of%2520BRD4%2520inhibits%2520HIV%2520transcription%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2007%26volume%3D104%26spage%3D13690%26epage%3D13695%26doi%3D10.1073%2Fpnas.0705053104" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Itzen, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greifenberg, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bosken, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geyer, M.</span></span> <span> </span><span class="NLM_article-title">BRD4 activates P-TEFb for RNA polymerase II CTD phosphorylation</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>42</i></span>,  <span class="NLM_fpage">7577</span>– <span class="NLM_lpage">7590</span>, <span class="refDoi"> DOI: 10.1093/nar/gku449</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1093%2Fnar%2Fgku449" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=24860166" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtFCqsrnE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2014&pages=7577-7590&author=F.+Itzenauthor=A.+K.+Greifenbergauthor=C.+A.+Boskenauthor=M.+Geyer&title=BRD4+activates+P-TEFb+for+RNA+polymerase+II+CTD+phosphorylation&doi=10.1093%2Fnar%2Fgku449"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Brd4 activates P-TEFb for RNA polymerase II CTD phosphorylation</span></div><div class="casAuthors">Itzen, Friederike; Greifenberg, Ann Katrin; Boesken, Christian A.; Geyer, Matthias</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">7577-7590</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Bromodomain protein Brd4 regulates the transcription of signal-inducible genes.  This is achieved by recruiting pos. transcription elongation factor P-TEFb to promoters by its P-TEFb interaction domain (PID).  Here, the authors show that Brd4 stimulates the kinase activity of P-TEFb for phosphorylation of the C-terminal domain (CTD) of RNA polymerase II over basal levels.  The CTD phosphorylation satn. levels, the preferences for pre-phosphorylated substrates, and the phosphorylation specificity for Ser-5 of the CTD however remained unchanged.  Inhibition of P-TEFb by coupling factor Hexim1 was relieved by Brd4, although no mutual displacement with the cyclin T-binding domain of Hexim1 was obsd.  The Brd4 PID showed a surprising sequence motif similarity to the trans-activating Tat protein from HIV-1, which included a core RxL motif, a polybasic cluster known as an arginine-rich motif, and a C-terminal leucine motif.  The mutation of these motifs to alanine significantly diminished the stimulatory effect of Brd4 and fully abrogated its activation potential in the presence of Hexim1.  Yet, the protein was not found to bind cyclin T1 as Tat, but only P-TEFb with a Kd of 0.5 μM.  The data suggested a model where Brd4 acts on the kinase subunit of P-TEFb to relieve inhibition and stimulate substrate recognition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo3Ihalme7shrVg90H21EOLACvtfcHk0liwg659WcGAvw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtFCqsrnE&md5=2d2d10fee6b55d4ef43950195344ca79</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgku449&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgku449%26sid%3Dliteratum%253Aachs%26aulast%3DItzen%26aufirst%3DF.%26aulast%3DGreifenberg%26aufirst%3DA.%2BK.%26aulast%3DBosken%26aufirst%3DC.%2BA.%26aulast%3DGeyer%26aufirst%3DM.%26atitle%3DBRD4%2520activates%2520P-TEFb%2520for%2520RNA%2520polymerase%2520II%2520CTD%2520phosphorylation%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2014%26volume%3D42%26spage%3D7577%26epage%3D7590%26doi%3D10.1093%2Fnar%2Fgku449" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sabari, B. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dall’Agnese, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boija, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klein, I. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coffey, E. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shrinivas, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abraham, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hannett, N. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zamudio, A. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manteiga, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, Y. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Day, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schuijers, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vasile, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Malik, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hnisz, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, T. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cisse, I. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roeder, R. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharp, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chakraborty, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, R. A.</span></span> <span> </span><span class="NLM_article-title">Coactivator condensation at super-enhancers links phase separation and gene control</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>361</i></span>,  <span class="NLM_fpage">eaar3958</span>, <span class="refDoi"> DOI: 10.1126/science.aar3958</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1126%2Fscience.aar3958" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=29930091" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=361&publication_year=2018&pages=eaar3958&author=B.+R.+Sabariauthor=A.+Dall%E2%80%99Agneseauthor=A.+Boijaauthor=I.+A.+Kleinauthor=E.+L.+Coffeyauthor=K.+Shrinivasauthor=B.+J.+Abrahamauthor=N.+M.+Hannettauthor=A.+V.+Zamudioauthor=J.+C.+Manteigaauthor=C.+H.+Liauthor=Y.+E.+Guoauthor=D.+S.+Dayauthor=J.+Schuijersauthor=E.+Vasileauthor=S.+Malikauthor=D.+Hniszauthor=T.+I.+Leeauthor=I.+I.+Cisseauthor=R.+G.+Roederauthor=P.+A.+Sharpauthor=A.+K.+Chakrabortyauthor=R.+A.+Young&title=Coactivator+condensation+at+super-enhancers+links+phase+separation+and+gene+control&doi=10.1126%2Fscience.aar3958"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1126%2Fscience.aar3958&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.aar3958%26sid%3Dliteratum%253Aachs%26aulast%3DSabari%26aufirst%3DB.%2BR.%26aulast%3DDall%25E2%2580%2599Agnese%26aufirst%3DA.%26aulast%3DBoija%26aufirst%3DA.%26aulast%3DKlein%26aufirst%3DI.%2BA.%26aulast%3DCoffey%26aufirst%3DE.%2BL.%26aulast%3DShrinivas%26aufirst%3DK.%26aulast%3DAbraham%26aufirst%3DB.%2BJ.%26aulast%3DHannett%26aufirst%3DN.%2BM.%26aulast%3DZamudio%26aufirst%3DA.%2BV.%26aulast%3DManteiga%26aufirst%3DJ.%2BC.%26aulast%3DLi%26aufirst%3DC.%2BH.%26aulast%3DGuo%26aufirst%3DY.%2BE.%26aulast%3DDay%26aufirst%3DD.%2BS.%26aulast%3DSchuijers%26aufirst%3DJ.%26aulast%3DVasile%26aufirst%3DE.%26aulast%3DMalik%26aufirst%3DS.%26aulast%3DHnisz%26aufirst%3DD.%26aulast%3DLee%26aufirst%3DT.%2BI.%26aulast%3DCisse%26aufirst%3DI.%2BI.%26aulast%3DRoeder%26aufirst%3DR.%2BG.%26aulast%3DSharp%26aufirst%3DP.%2BA.%26aulast%3DChakraborty%26aufirst%3DA.%2BK.%26aulast%3DYoung%26aufirst%3DR.%2BA.%26atitle%3DCoactivator%2520condensation%2520at%2520super-enhancers%2520links%2520phase%2520separation%2520and%2520gene%2520control%26jtitle%3DScience%26date%3D2018%26volume%3D361%26spage%3Deaar3958%26doi%3D10.1126%2Fscience.aar3958" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gibson, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doolittle, L. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schneider, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jensen, L. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gamarra, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Henry, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerlich, D. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Redding, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosen, M. K.</span></span> <span> </span><span class="NLM_article-title">Organization of chromatin by intrinsic and regulated phase separation</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>179</i></span>,  <span class="NLM_fpage">470</span>– <span class="NLM_lpage">484</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2019.08.037</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1016%2Fj.cell.2019.08.037" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=31543265" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhvVentbzE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=179&publication_year=2019&pages=470-484&author=B.+A.+Gibsonauthor=L.+K.+Doolittleauthor=M.+Schneiderauthor=L.+E.+Jensenauthor=N.+Gamarraauthor=L.+Henryauthor=D.+W.+Gerlichauthor=S.+Reddingauthor=M.+K.+Rosen&title=Organization+of+chromatin+by+intrinsic+and+regulated+phase+separation&doi=10.1016%2Fj.cell.2019.08.037"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Organization of Chromatin by Intrinsic and Regulated Phase Separation</span></div><div class="casAuthors">Gibson, Bryan A.; Doolittle, Lynda K.; Schneider, Maximillian W. G.; Jensen, Liv E.; Gamarra, Nathan; Henry, Lisa; Gerlich, Daniel W.; Redding, Sy; Rosen, Michael K.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">179</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">470-484.e21</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Eukaryotic chromatin is highly condensed but dynamically accessible to regulation and organized into subdomains.  We demonstrate that reconstituted chromatin undergoes histone tail-driven liq.-liq. phase sepn. (LLPS) in physiol. salt and when microinjected into cell nuclei, producing dense and dynamic droplets.  Linker histone H1 and internucleosome linker lengths shared across eukaryotes promote phase sepn. of chromatin, tune droplet properties, and coordinate to form condensates of consistent d. in manners that parallel chromatin behavior in cells.  Histone acetylation by p300 antagonizes chromatin phase sepn., dissolving droplets in vitro and decreasing droplet formation in nuclei.  In the presence of multi-bromodomain proteins, such as BRD4, highly acetylated chromatin forms a new phase-sepd. state with droplets of distinct phys. properties, which can be immiscible with unmodified chromatin droplets, mimicking nuclear chromatin subdomains.  Our data suggest a framework, based on intrinsic phase sepn. of the chromatin polymer, for understanding the organization and regulation of eukaryotic genomes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGosqITN5BU2ZrVg90H21EOLACvtfcHk0liKEuMaM9vEhg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhvVentbzE&md5=27efce722f7a1f0be19097a034e0d200</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2019.08.037&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2019.08.037%26sid%3Dliteratum%253Aachs%26aulast%3DGibson%26aufirst%3DB.%2BA.%26aulast%3DDoolittle%26aufirst%3DL.%2BK.%26aulast%3DSchneider%26aufirst%3DM.%26aulast%3DJensen%26aufirst%3DL.%2BE.%26aulast%3DGamarra%26aufirst%3DN.%26aulast%3DHenry%26aufirst%3DL.%26aulast%3DGerlich%26aufirst%3DD.%2BW.%26aulast%3DRedding%26aufirst%3DS.%26aulast%3DRosen%26aufirst%3DM.%2BK.%26atitle%3DOrganization%2520of%2520chromatin%2520by%2520intrinsic%2520and%2520regulated%2520phase%2520separation%26jtitle%3DCell%26date%3D2019%26volume%3D179%26spage%3D470%26epage%3D484%26doi%3D10.1016%2Fj.cell.2019.08.037" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Han, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gu, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jia, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jaganathan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Babault, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, L.</span></span> <span> </span><span class="NLM_article-title">Roles of the BRD4 short isoform in phase separation and active gene transcription</span>. <i>Nat. Struct. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">333</span>– <span class="NLM_lpage">341</span>, <span class="refDoi"> DOI: 10.1038/s41594-020-0394-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1038%2Fs41594-020-0394-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=32203489" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BB3cXltVSisLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2020&pages=333-341&author=X.+Hanauthor=D.+Yuauthor=R.+Guauthor=Y.+Jiaauthor=Q.+Wangauthor=A.+Jaganathanauthor=X.+Yangauthor=M.+Yuauthor=N.+Babaultauthor=C.+Zhaoauthor=H.+Yiauthor=Q.+Zhangauthor=M.+M.+Zhouauthor=L.+Zeng&title=Roles+of+the+BRD4+short+isoform+in+phase+separation+and+active+gene+transcription&doi=10.1038%2Fs41594-020-0394-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Roles of the BRD4 short isoform in phase separation and active gene transcription</span></div><div class="casAuthors">Han, Xinye; Yu, Di; Gu, Ruirui; Jia, Yanjie; Wang, Qi; Jaganathan, Anbalagan; Yang, Xuelan; Yu, Miaomiao; Babault, Nicolas; Zhao, Chengcheng; Yi, Huanfa; Zhang, Qiang; Zhou, Ming-Ming; Zeng, Lei</div><div class="citationInfo"><span class="NLM_cas:title">Nature Structural & Molecular Biology</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">333-341</span>CODEN:
                <span class="NLM_cas:coden">NSMBCU</span>;
        ISSN:<span class="NLM_cas:issn">1545-9993</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">BRD4, a major tandem-bromodomain-contg. transcription regulator, has two isoforms.  The long isoform (BRD4L) has an extended C terminus that binds transcription cofactors, while the short isoform (BRD4S) lacks this C-terminal extension.  Unlike BRD4L, the role of BRD4S in gene transcription remains unclear.  Here, we report that, in human cancer cells, BRD4S forms nuclear puncta that possess liq.-like properties and that colocalize with BRD4L, MED1 and sites of histone H3 lysine 27 acetylation.  BRD4 puncta are correlated with BRD4S but not BRD4L expression levels.  BRD4S knockdown reduces BRD4S condensation, and ectopic expression promotes puncta formation and target gene transcription.  BRD4S nuclear condensation is mediated by its intrinsically disordered regions and binding of its bromodomains to DNA and acetylated chromatin, resp., and BRD4S phosphorylation diminishes BRD4 condensation.  Our study illuminates a previously unappreciated role of BRD4S in organizing chromatin and transcription factors through phase sepn. to sustain gene transcription in chromatin for cancer cell proliferation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoePsB81fBjY7Vg90H21EOLACvtfcHk0liKEuMaM9vEhg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXltVSisLw%253D&md5=fcd3cd7541a8c2a60ee84a57a19ab62d</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1038%2Fs41594-020-0394-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41594-020-0394-8%26sid%3Dliteratum%253Aachs%26aulast%3DHan%26aufirst%3DX.%26aulast%3DYu%26aufirst%3DD.%26aulast%3DGu%26aufirst%3DR.%26aulast%3DJia%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DQ.%26aulast%3DJaganathan%26aufirst%3DA.%26aulast%3DYang%26aufirst%3DX.%26aulast%3DYu%26aufirst%3DM.%26aulast%3DBabault%26aufirst%3DN.%26aulast%3DZhao%26aufirst%3DC.%26aulast%3DYi%26aufirst%3DH.%26aulast%3DZhang%26aufirst%3DQ.%26aulast%3DZhou%26aufirst%3DM.%2BM.%26aulast%3DZeng%26aufirst%3DL.%26atitle%3DRoles%2520of%2520the%2520BRD4%2520short%2520isoform%2520in%2520phase%2520separation%2520and%2520active%2520gene%2520transcription%26jtitle%3DNat.%2520Struct.%2520Mol.%2520Biol.%26date%3D2020%26volume%3D27%26spage%3D333%26epage%3D341%26doi%3D10.1038%2Fs41594-020-0394-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sakurai, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Inamochi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Inoue, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hariya, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kawamura, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamada, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dey, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nishiyama, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kubota, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ozato, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goda, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mochizuki, K.</span></span> <span> </span><span class="NLM_article-title">BRD4 regulates adiponectin gene induction by recruiting the P-TEFb complex to the transcribed region of the gene</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">11962</span>, <span class="refDoi"> DOI: 10.1038/s41598-017-12342-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1038%2Fs41598-017-12342-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=28931940" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A280%3ADC%252BC1M%252FgslSqtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2017&pages=11962&author=N.+Sakuraiauthor=Y.+Inamochiauthor=T.+Inoueauthor=N.+Hariyaauthor=M.+Kawamuraauthor=M.+Yamadaauthor=A.+Deyauthor=A.+Nishiyamaauthor=T.+Kubotaauthor=K.+Ozatoauthor=T.+Godaauthor=K.+Mochizuki&title=BRD4+regulates+adiponectin+gene+induction+by+recruiting+the+P-TEFb+complex+to+the+transcribed+region+of+the+gene&doi=10.1038%2Fs41598-017-12342-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">BRD4 regulates adiponectin gene induction by recruiting the P-TEFb complex to the transcribed region of the gene</span></div><div class="casAuthors">Sakurai Naoko; Inamochi Yuko; Yamada Masami; Goda Toshinao; Mochizuki Kazuki; Inoue Takuya; Hariya Natsuyo; Kawamura Musashi; Mochizuki Kazuki; Dey Anup; Nishiyama Akira; Ozato Keiko; Kubota Takeo</div><div class="citationInfo"><span class="NLM_cas:title">Scientific reports</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">11962</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">We previously reported that induction of the adipocyte-specific gene adiponectin (Adipoq) during 3T3-L1 adipocyte differentiation is closely associated with epigenetic memory histone H3 acetylation on the transcribed region of the gene.  We used 3T3-L1 adipocytes and Brd4 heterozygous mice to investigate whether the induction of Adipoq during adipocyte differentiation is regulated by histone acetylation and the binding protein bromodomain containing 4 (BRD4) on the transcribed region.  Depletion of BRD4 by shRNA and inhibition by (+)-JQ1, an inhibitor of BET family proteins including BRD4, reduced Adipoq expression and lipid droplet accumulation in 3T3-L1 adipocytes.  Additionally, the depletion and inhibition of BRD4 reduced the expression of many insulin sensitivity-related genes, including genes related to lipid droplet accumulation in adipocytes.  BRD4 depletion reduced P-TEFb recruitment and histone acetylation on the transcribed region of the Adipoq gene.  The expression levels of Adipoq and fatty acid synthesis-related genes and the circulating ADIPOQ protein level were lower in Brd4 heterozygous mice than in wild-type mice at 21 days after birth.  These findings indicate that BRD4 regulates the Adipoq gene by recruiting P-TEFb onto acetylated histones in the transcribed region of the gene and regulates adipocyte differentiation by regulating the expression of genes related to insulin sensitivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcR6_YYS8G5ZW0cLvDNbOD4DfW6udTcc2eb7Qms93z1i77ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1M%252FgslSqtA%253D%253D&md5=b5db7bd76c71c96e00dc12496816b3f6</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1038%2Fs41598-017-12342-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41598-017-12342-2%26sid%3Dliteratum%253Aachs%26aulast%3DSakurai%26aufirst%3DN.%26aulast%3DInamochi%26aufirst%3DY.%26aulast%3DInoue%26aufirst%3DT.%26aulast%3DHariya%26aufirst%3DN.%26aulast%3DKawamura%26aufirst%3DM.%26aulast%3DYamada%26aufirst%3DM.%26aulast%3DDey%26aufirst%3DA.%26aulast%3DNishiyama%26aufirst%3DA.%26aulast%3DKubota%26aufirst%3DT.%26aulast%3DOzato%26aufirst%3DK.%26aulast%3DGoda%26aufirst%3DT.%26aulast%3DMochizuki%26aufirst%3DK.%26atitle%3DBRD4%2520regulates%2520adiponectin%2520gene%2520induction%2520by%2520recruiting%2520the%2520P-TEFb%2520complex%2520to%2520the%2520transcribed%2520region%2520of%2520the%2520gene%26jtitle%3DSci.%2520Rep.%26date%3D2017%26volume%3D7%26spage%3D11962%26doi%3D10.1038%2Fs41598-017-12342-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sabo, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amati, B.</span></span> <span> </span><span class="NLM_article-title">BRD4 and MYC-clarifying regulatory specificity</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>360</i></span>,  <span class="NLM_fpage">713</span>– <span class="NLM_lpage">714</span>, <span class="refDoi"> DOI: 10.1126/science.aat6664</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1126%2Fscience.aat6664" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=29773735" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtVCmur7K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=360&publication_year=2018&pages=713-714&author=A.+Saboauthor=B.+Amati&title=BRD4+and+MYC-clarifying+regulatory+specificity&doi=10.1126%2Fscience.aat6664"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">BRD4 and MYC-clarifying regulatory specificity</span></div><div class="casAuthors">Sabo, Arianna; Amati, Bruno</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">360</span>
        (<span class="NLM_cas:issue">6390</span>),
    <span class="NLM_cas:pages">713-714</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr7ZGDU1lOAM7Vg90H21EOLACvtfcHk0li0duteB9l5Vw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtVCmur7K&md5=fe1fc2f0d68a8231dd9ff02224071671</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1126%2Fscience.aat6664&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.aat6664%26sid%3Dliteratum%253Aachs%26aulast%3DSabo%26aufirst%3DA.%26aulast%3DAmati%26aufirst%3DB.%26atitle%3DBRD4%2520and%2520MYC-clarifying%2520regulatory%2520specificity%26jtitle%3DScience%26date%3D2018%26volume%3D360%26spage%3D713%26epage%3D714%26doi%3D10.1126%2Fscience.aat6664" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Devaiah, B. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Akman, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uppal, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weissman, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baranello, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nie, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Levens, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singer, D. S.</span></span> <span> </span><span class="NLM_article-title">MYC protein stability is negatively regulated by BRD4</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>117</i></span>,  <span class="NLM_fpage">13457</span>– <span class="NLM_lpage">13467</span>, <span class="refDoi"> DOI: 10.1073/pnas.1919507117</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1073%2Fpnas.1919507117" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=32482868" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhtF2jtLvF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=117&publication_year=2020&pages=13457-13467&author=B.+N.+Devaiahauthor=J.+Muauthor=B.+Akmanauthor=S.+Uppalauthor=J.+D.+Weissmanauthor=D.+Chengauthor=L.+Baranelloauthor=Z.+Nieauthor=D.+Levensauthor=D.+S.+Singer&title=MYC+protein+stability+is+negatively+regulated+by+BRD4&doi=10.1073%2Fpnas.1919507117"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">MYC protein stability is negatively regulated by BRD4</span></div><div class="casAuthors">Devaiah, Ballachanda N.; Mu, Jie; Akman, Ben; Uppal, Sheetal; Weissman, Jocelyn D.; Cheng, Dan; Baranello, Laura; Nie, Zuqin; Levens, David; Singer, Dinah S.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">117</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">13457-13467</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The protooncogene MYC regulates a variety of cellular processes, including proliferation and metab.  Maintaining MYC at homeostatic levels is crit. to normal cell function; overexpression drives many cancers.  MYC stability is regulated through phosphorylation: phosphorylation at Thr58 signals degrdn. while Ser62 phosphorylation leads to its stabilization and functional activation.  The bromodomain protein 4 (BRD4) is a transcriptional and epigenetic regulator with intrinsic kinase and histone acetyltransferase (HAT) activities that activates transcription of key protooncogenes, including MYC.  We report that BRD4 phosphorylates MYC at Thr58, leading to MYC ubiquitination and degrdn., thereby regulating MYC target genes.  Importantly, BRD4 degrdn., but not inhibition, results in increased levels of MYC protein.  Conversely, MYC inhibits BRD4's HAT activity, suggesting that MYC regulates its own transcription by limiting BRD4-mediated chromatin remodeling of its locus.  The MYC stabilizing kinase, ERK1, regulates MYC levels directly and indirectly by inhibiting BRD4 kinase activity.  These findings demonstrate that BRD4 neg. regulates MYC levels, which is counteracted by ERK1 activation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqtwATxk8kt87Vg90H21EOLACvtfcHk0li0duteB9l5Vw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhtF2jtLvF&md5=77dec456bd0a0507e76d054f547c0430</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1919507117&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1919507117%26sid%3Dliteratum%253Aachs%26aulast%3DDevaiah%26aufirst%3DB.%2BN.%26aulast%3DMu%26aufirst%3DJ.%26aulast%3DAkman%26aufirst%3DB.%26aulast%3DUppal%26aufirst%3DS.%26aulast%3DWeissman%26aufirst%3DJ.%2BD.%26aulast%3DCheng%26aufirst%3DD.%26aulast%3DBaranello%26aufirst%3DL.%26aulast%3DNie%26aufirst%3DZ.%26aulast%3DLevens%26aufirst%3DD.%26aulast%3DSinger%26aufirst%3DD.%2BS.%26atitle%3DMYC%2520protein%2520stability%2520is%2520negatively%2520regulated%2520by%2520BRD4%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2020%26volume%3D117%26spage%3D13457%26epage%3D13467%26doi%3D10.1073%2Fpnas.1919507117" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rahnamoun, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramsey, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Komives, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lauberth, S. M.</span></span> <span> </span><span class="NLM_article-title">RNAs interact with BRD4 to promote enhanced chromatin engagement and transcription activation</span>. <i>Nat. Struct. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">687</span>– <span class="NLM_lpage">697</span>, <span class="refDoi"> DOI: 10.1038/s41594-018-0102-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1038%2Fs41594-018-0102-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=30076409" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhsVGjtr7O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2018&pages=687-697&author=H.+Rahnamounauthor=J.+Leeauthor=Z.+Sunauthor=H.+Luauthor=K.+M.+Ramseyauthor=E.+A.+Komivesauthor=S.+M.+Lauberth&title=RNAs+interact+with+BRD4+to+promote+enhanced+chromatin+engagement+and+transcription+activation&doi=10.1038%2Fs41594-018-0102-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">RNAs interact with BRD4 to promote enhanced chromatin engagement and transcription activation</span></div><div class="casAuthors">Rahnamoun, Homa; Lee, Jihoon; Sun, Zhengxi; Lu, Hanbin; Ramsey, Kristen M.; Komives, Elizabeth A.; Lauberth, Shannon M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Structural & Molecular Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">687-697</span>CODEN:
                <span class="NLM_cas:coden">NSMBCU</span>;
        ISSN:<span class="NLM_cas:issn">1545-9993</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">The bromodomain and extra-terminal motif (BET) protein BRD4 binds to acetylated histones at enhancers and promoters via its bromodomains (BDs) to regulate transcriptional elongation.  In human colorectal cancer cells, we found that BRD4 was recruited to enhancers that were co-occupied by mutant p53 and supported the synthesis of enhancer-directed transcripts (eRNAs) in response to chronic immune signaling.  BRD4 selectively assocd. with eRNAs that were produced from BRD4-bound enhancers.  Using biochem. and biophys. methods, we found that BRD4 BDs function cooperatively as docking sites for eRNAs and that the BDs of BRD2, BRD3, BRDT, BRG1, and BRD7 directly interact with eRNAs.  BRD4-eRNA interactions increased BRD4 binding to acetylated histones in vitro and augmented BRD4 enhancer recruitment and transcriptional cofactor activities.  Our results suggest a mechanism by which eRNAs are directly involved in gene regulation by modulating enhancer interactions and transcriptional functions of BRD4.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGriyuXqZrKRy7Vg90H21EOLACvtfcHk0li0duteB9l5Vw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhsVGjtr7O&md5=c3470d4854f5c90e7e770d3dbe7cd5d8</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1038%2Fs41594-018-0102-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41594-018-0102-0%26sid%3Dliteratum%253Aachs%26aulast%3DRahnamoun%26aufirst%3DH.%26aulast%3DLee%26aufirst%3DJ.%26aulast%3DSun%26aufirst%3DZ.%26aulast%3DLu%26aufirst%3DH.%26aulast%3DRamsey%26aufirst%3DK.%2BM.%26aulast%3DKomives%26aufirst%3DE.%2BA.%26aulast%3DLauberth%26aufirst%3DS.%2BM.%26atitle%3DRNAs%2520interact%2520with%2520BRD4%2520to%2520promote%2520enhanced%2520chromatin%2520engagement%2520and%2520transcription%2520activation%26jtitle%3DNat.%2520Struct.%2520Mol.%2520Biol.%26date%3D2018%26volume%3D25%26spage%3D687%26epage%3D697%26doi%3D10.1038%2Fs41594-018-0102-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ozato, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L. F.</span></span> <span> </span><span class="NLM_article-title">BRD4 coactivates transcriptional activation of NF-kappaB via specific binding to acetylated RelA</span>. <i>Mol. Cell. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">1375</span>– <span class="NLM_lpage">1387</span>, <span class="refDoi"> DOI: 10.1128/MCB.01365-08</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1128%2FMCB.01365-08" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=19103749" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BD1MXitlylsrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2009&pages=1375-1387&author=B.+Huangauthor=X.+D.+Yangauthor=M.+M.+Zhouauthor=K.+Ozatoauthor=L.+F.+Chen&title=BRD4+coactivates+transcriptional+activation+of+NF-kappaB+via+specific+binding+to+acetylated+RelA&doi=10.1128%2FMCB.01365-08"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Brd4 coactivates transcriptional activation of NF-κB via specific binding to acetylated RelA</span></div><div class="casAuthors">Huang, Bo; Yang, Xiao-Dong; Zhou, Ming-Ming; Ozato, Keiko; Chen, Lin-Feng</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Biology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1375-1387</span>CODEN:
                <span class="NLM_cas:coden">MCEBD4</span>;
        ISSN:<span class="NLM_cas:issn">0270-7306</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Acetylation of the RelA subunit of NF-κB, esp. at lysine-310, is crit. for the transcriptional activation of NF-κB and the expression of inflammatory genes.  In this study, we demonstrate that bromodomains of Brd4 bind to acetylated lysine-310.  Brd4 enhances transcriptional activation of NF-κB and the expression of a subset of NF-κB-responsive inflammatory genes in an acetylated lysine-310-dependent manner.  Bromodomains of Brd4 and acetylated lysine-310 of RelA are both required for the mutual interaction and coactivation function of Brd4.  Finally, we demonstrate that Brd4 further recruits CDK9 to phosphorylate C-terminal domain of RNA polymerase II and facilitate the transcription of NF-κB-dependent inflammatory genes.  Our results identify Brd4 as a novel coactivator of NF-κB through specifically binding to acetylated lysine-310 of RelA.  In addn., these studies reveal a mechanism by which acetylated RelA stimulates the transcriptional activity of NF-κB and the NF-κB-dependent inflammatory response.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoBnS4p5wzT2LVg90H21EOLACvtfcHk0lgEWm7iYVw2VQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXitlylsrc%253D&md5=ae4e16c84d17c5e0e268ff80703a4608</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1128%2FMCB.01365-08&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMCB.01365-08%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DB.%26aulast%3DYang%26aufirst%3DX.%2BD.%26aulast%3DZhou%26aufirst%3DM.%2BM.%26aulast%3DOzato%26aufirst%3DK.%26aulast%3DChen%26aufirst%3DL.%2BF.%26atitle%3DBRD4%2520coactivates%2520transcriptional%2520activation%2520of%2520NF-kappaB%2520via%2520specific%2520binding%2520to%2520acetylated%2520RelA%26jtitle%3DMol.%2520Cell.%2520Biol.%26date%3D2009%26volume%3D29%26spage%3D1375%26epage%3D1387%26doi%3D10.1128%2FMCB.01365-08" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tao, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rusinova, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, Y. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evers, B. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, B. P.</span></span> <span> </span><span class="NLM_article-title">Disrupting the interaction of BRD4 with diacetylated Twist suppresses tumorigenesis in basal-like breast cancer</span>. <i>Cancer Cell</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">210</span>– <span class="NLM_lpage">225</span>, <span class="refDoi"> DOI: 10.1016/j.ccr.2014.01.028</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1016%2Fj.ccr.2014.01.028" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=24525235" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BC2cXitlCisrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2014&pages=210-225&author=J.+Shiauthor=Y.+Wangauthor=L.+Zengauthor=Y.+Wuauthor=J.+Dengauthor=Q.+Zhangauthor=Y.+Linauthor=J.+Liauthor=T.+Kangauthor=M.+Taoauthor=E.+Rusinovaauthor=G.+Zhangauthor=C.+Wangauthor=H.+Zhuauthor=J.+Yaoauthor=Y.+X.+Zengauthor=B.+M.+Eversauthor=M.+M.+Zhouauthor=B.+P.+Zhou&title=Disrupting+the+interaction+of+BRD4+with+diacetylated+Twist+suppresses+tumorigenesis+in+basal-like+breast+cancer&doi=10.1016%2Fj.ccr.2014.01.028"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Disrupting the Interaction of BRD4 with Diacetylated Twist Suppresses Tumorigenesis in Basal-like Breast Cancer</span></div><div class="casAuthors">Shi, Jian; Wang, Yifan; Zeng, Lei; Wu, Yadi; Deng, Jiong; Zhang, Qiang; Lin, Yiwei; Li, Junlin; Kang, Tiebang; Tao, Min; Rusinova, Elena; Zhang, Guangtao; Wang, Chi; Zhu, Haining; Yao, Jun; Zeng, Yi-Xin; Evers, B. Mark; Zhou, Ming-Ming; Zhou, Binhua P.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">210-225</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Twist is a key transcription activator of epithelial-mesenchymal transition (EMT).  It remains unclear how Twist induces gene expression.  Here we report a mechanism by which Twist recruits BRD4 to direct WNT5A expression in basal-like breast cancer (BLBC).  Twist contains a "histone H4-mimic" GK-X-GK motif that is diacetylated by Tip60.  The diacetylated Twist binds the second bromodomain of BRD4, whose first bromodomain interacts with acetylated H4, thereby constructing an activated Twist/BRD4/P-TEFb/RNA-Pol II complex at the WNT5A promoter and enhancer.  Pharmacol. inhibition of the Twist-BRD4 assocn. reduced WNT5A expression and suppressed invasion, cancer stem cell (CSC)-like properties, and tumorigenicity of BLBC cells.  Our study indicates that the interaction with BRD4 is crit. for the oncogenic function of Twist in BLBC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGovIzriI8A6ZrVg90H21EOLACvtfcHk0lgEWm7iYVw2VQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXitlCisrs%253D&md5=d207cbc1abb187a53d807452b4f10d8f</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1016%2Fj.ccr.2014.01.028&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccr.2014.01.028%26sid%3Dliteratum%253Aachs%26aulast%3DShi%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DZeng%26aufirst%3DL.%26aulast%3DWu%26aufirst%3DY.%26aulast%3DDeng%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DQ.%26aulast%3DLin%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DKang%26aufirst%3DT.%26aulast%3DTao%26aufirst%3DM.%26aulast%3DRusinova%26aufirst%3DE.%26aulast%3DZhang%26aufirst%3DG.%26aulast%3DWang%26aufirst%3DC.%26aulast%3DZhu%26aufirst%3DH.%26aulast%3DYao%26aufirst%3DJ.%26aulast%3DZeng%26aufirst%3DY.%2BX.%26aulast%3DEvers%26aufirst%3DB.%2BM.%26aulast%3DZhou%26aufirst%3DM.%2BM.%26aulast%3DZhou%26aufirst%3DB.%2BP.%26atitle%3DDisrupting%2520the%2520interaction%2520of%2520BRD4%2520with%2520diacetylated%2520Twist%2520suppresses%2520tumorigenesis%2520in%2520basal-like%2520breast%2520cancer%26jtitle%3DCancer%2520Cell%26date%3D2014%26volume%3D25%26spage%3D210%26epage%3D225%26doi%3D10.1016%2Fj.ccr.2014.01.028" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tian, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Su, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">An, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Du, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shan, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiang, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H.</span></span> <span> </span><span class="NLM_article-title">Inhibition of BRD4 suppresses the malignancy of breast cancer cells via regulation of Snail</span>. <i>Cell Death Differ.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">255</span>– <span class="NLM_lpage">268</span>, <span class="refDoi"> DOI: 10.1038/s41418-019-0353-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1038%2Fs41418-019-0353-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=31114028" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhtFSmsbnF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2020&pages=255-268&author=L.+Luauthor=Z.+Chenauthor=X.+Linauthor=L.+Tianauthor=Q.+Suauthor=P.+Anauthor=W.+Liauthor=Y.+Wuauthor=J.+Duauthor=H.+Shanauthor=C.+M.+Chiangauthor=H.+Wang&title=Inhibition+of+BRD4+suppresses+the+malignancy+of+breast+cancer+cells+via+regulation+of+Snail&doi=10.1038%2Fs41418-019-0353-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of BRD4 suppresses the malignancy of breast cancer cells via regulation of Snail</span></div><div class="casAuthors">Lu, Linlin; Chen, Zhuojia; Lin, Xinyao; Tian, Lin; Su, Qiao; An, Panpan; Li, Wuguo; Wu, Yingmin; Du, Jun; Shan, Hong; Chiang, Cheng-Ming; Wang, Hongsheng</div><div class="citationInfo"><span class="NLM_cas:title">Cell Death & Differentiation</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">255-268</span>CODEN:
                <span class="NLM_cas:coden">CDDIEK</span>;
        ISSN:<span class="NLM_cas:issn">1350-9047</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">The mechanistic action of bromodomain-contg. protein 4 (BRD4) in cancer motility, including epithelial-mesenchymal transition (EMT), remains largely undefined.  We found that targeted inhibition of BRD4 reduces migration, invasion, in vivo growth of patient-derived xenograft (PDX), and lung colonization of breast cancer (BC) cells.  Inhibition of BRD4 rapidly decreases the expression of Snail, a powerful EMT transcription factor (EMT-TF), via diminishing its protein stability and transcription.  Protein kinase D1 (PRKD1) is responsible for BRD4-regulated Snail protein stability by triggering phosphorylation at Ser11 of Snail and then inducing proteasome-mediated degrdn.  BRD4 inhibition also suppresses the expression of Gli1, a key transductor of Hedgehog (Hh) required to activate the transcription of SNAI1, in BC cells.  The GACCACC sequence (-341 to -333) in the SNAI1 promoter is responsible for Gli1-induced transcription of SNAI1.  Clin., BRD4 and Snail levels are increased in lung-metastasized, estrogen receptor-neg. (ER-), and progesterone receptor-neg. (PR-) breast cancers and correlate with the expression of mesenchymal markers.  Collectively, BRD4 can regulate malignancy of breast cancer cells via both transcriptional and post-translational regulation of Snail.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrhdNJ6b7jcFbVg90H21EOLACvtfcHk0lgEWm7iYVw2VQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhtFSmsbnF&md5=9bee526409f844f81cfaaadce9779da2</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1038%2Fs41418-019-0353-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41418-019-0353-2%26sid%3Dliteratum%253Aachs%26aulast%3DLu%26aufirst%3DL.%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DLin%26aufirst%3DX.%26aulast%3DTian%26aufirst%3DL.%26aulast%3DSu%26aufirst%3DQ.%26aulast%3DAn%26aufirst%3DP.%26aulast%3DLi%26aufirst%3DW.%26aulast%3DWu%26aufirst%3DY.%26aulast%3DDu%26aufirst%3DJ.%26aulast%3DShan%26aufirst%3DH.%26aulast%3DChiang%26aufirst%3DC.%2BM.%26aulast%3DWang%26aufirst%3DH.%26atitle%3DInhibition%2520of%2520BRD4%2520suppresses%2520the%2520malignancy%2520of%2520breast%2520cancer%2520cells%2520via%2520regulation%2520of%2520Snail%26jtitle%3DCell%2520Death%2520Differ.%26date%3D2020%26volume%3D27%26spage%3D255%26epage%3D268%26doi%3D10.1038%2Fs41418-019-0353-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zanconato, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Battilana, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Forcato, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Filippi, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Azzolin, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manfrin, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quaranta, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Di Biagio, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sigismondo, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guzzardo, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lejeune, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haendler, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krijgsveld, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fassan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bicciato, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cordenonsi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Piccolo, S.</span></span> <span> </span><span class="NLM_article-title">Transcriptional addiction in cancer cells is mediated by YAP/TAZ through BRD4</span>. <i>Nat. Med.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">1599</span>– <span class="NLM_lpage">1610</span>, <span class="refDoi"> DOI: 10.1038/s41591-018-0158-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1038%2Fs41591-018-0158-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=30224758" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhslCjtrbN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2018&pages=1599-1610&author=F.+Zanconatoauthor=G.+Battilanaauthor=M.+Forcatoauthor=L.+Filippiauthor=L.+Azzolinauthor=A.+Manfrinauthor=E.+Quarantaauthor=D.+Di+Biagioauthor=G.+Sigismondoauthor=V.+Guzzardoauthor=P.+Lejeuneauthor=B.+Haendlerauthor=J.+Krijgsveldauthor=M.+Fassanauthor=S.+Bicciatoauthor=M.+Cordenonsiauthor=S.+Piccolo&title=Transcriptional+addiction+in+cancer+cells+is+mediated+by+YAP%2FTAZ+through+BRD4&doi=10.1038%2Fs41591-018-0158-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Transcriptional addiction in cancer cells is mediated by YAP/TAZ through BRD4</span></div><div class="casAuthors">Zanconato, Francesca; Battilana, Giusy; Forcato, Mattia; Filippi, Letizia; Azzolin, Luca; Manfrin, Andrea; Quaranta, Erika; Di Biagio, Daniele; Sigismondo, Gianluca; Guzzardo, Vincenza; Lejeune, Pascale; Haendler, Bernard; Krijgsveld, Jeroen; Fassan, Matteo; Bicciato, Silvio; Cordenonsi, Michelangelo; Piccolo, Stefano</div><div class="citationInfo"><span class="NLM_cas:title">Nature Medicine (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1599-1610</span>CODEN:
                <span class="NLM_cas:coden">NAMEFI</span>;
        ISSN:<span class="NLM_cas:issn">1078-8956</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Cancer cells rely on dysregulated gene expression.  This establishes specific transcriptional addictions that may be therapeutically exploited.  Yet, the mechanisms that are ultimately responsible for these addictions are poorly understood.  Here, we investigated the transcriptional dependencies of transformed cells to the transcription factors YAP and TAZ.  YAP/TAZ phys. engage the general coactivator bromodomain-contg. protein 4 (BRD4), dictating the genome-wide assocn. of BRD4 to chromatin.  YAP/TAZ flag a large set of enhancers with super-enhancer-like functional properties.  YAP/TAZ-bound enhancers mediate the recruitment of BRD4 and RNA polymerase II at YAP/TAZ-regulated promoters, boosting the expression of a host of growth-regulating genes.  Treatment with small-mol. inhibitors of BRD4 blunts YAP/TAZ pro-tumorigenic activity in several cell or tissue contexts, causes the regression of pre-established, YAP/TAZ-addicted neoplastic lesions and reverts drug resistance.  This work sheds light on essential mediators, mechanisms and genome-wide regulatory elements that are responsible for transcriptional addiction in cancer and lays the groundwork for a rational use of BET inhibitors according to YAP/TAZ biol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqqz-uXLracArVg90H21EOLACvtfcHk0ljQx8X7_5-xpQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhslCjtrbN&md5=131d262f43c61023f6b4f6a6d99a94ea</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1038%2Fs41591-018-0158-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41591-018-0158-8%26sid%3Dliteratum%253Aachs%26aulast%3DZanconato%26aufirst%3DF.%26aulast%3DBattilana%26aufirst%3DG.%26aulast%3DForcato%26aufirst%3DM.%26aulast%3DFilippi%26aufirst%3DL.%26aulast%3DAzzolin%26aufirst%3DL.%26aulast%3DManfrin%26aufirst%3DA.%26aulast%3DQuaranta%26aufirst%3DE.%26aulast%3DDi%2BBiagio%26aufirst%3DD.%26aulast%3DSigismondo%26aufirst%3DG.%26aulast%3DGuzzardo%26aufirst%3DV.%26aulast%3DLejeune%26aufirst%3DP.%26aulast%3DHaendler%26aufirst%3DB.%26aulast%3DKrijgsveld%26aufirst%3DJ.%26aulast%3DFassan%26aufirst%3DM.%26aulast%3DBicciato%26aufirst%3DS.%26aulast%3DCordenonsi%26aufirst%3DM.%26aulast%3DPiccolo%26aufirst%3DS.%26atitle%3DTranscriptional%2520addiction%2520in%2520cancer%2520cells%2520is%2520mediated%2520by%2520YAP%252FTAZ%2520through%2520BRD4%26jtitle%3DNat.%2520Med.%26date%3D2018%26volume%3D24%26spage%3D1599%26epage%3D1610%26doi%3D10.1038%2Fs41591-018-0158-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Song, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pool Pizzi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, A. W</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huo, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhutani, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weston, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shanbhag, N. D</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ajani, J. A.</span></span> <span> </span><span class="NLM_article-title">Targeting Hippo coactivator YAP1 through BET bromodomain inhibition in esophageal adenocarcinoma</span>. <i>Mol. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">1410</span>– <span class="NLM_lpage">1426</span>, <span class="refDoi"> DOI: 10.1002/1878-0261.12667</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1002%2F1878-0261.12667" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=32175692" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhsleltbnJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2020&pages=1410-1426&author=S.+Songauthor=Y.+Liauthor=Y.+Xuauthor=L.+Maauthor=M.+Pool+Pizziauthor=J.+Jinauthor=A.+W+Scottauthor=L.+Huoauthor=Y.+Wangauthor=J.+H.+Leeauthor=M.+S.+Bhutaniauthor=B.+Westonauthor=N.+D+Shanbhagauthor=R.+L.+Johnsonauthor=J.+A.+Ajani&title=Targeting+Hippo+coactivator+YAP1+through+BET+bromodomain+inhibition+in+esophageal+adenocarcinoma&doi=10.1002%2F1878-0261.12667"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting Hippo coactivator YAP1 through BET bromodomain inhibition in esophageal adenocarcinoma</span></div><div class="casAuthors">Song, Shumei; Li, Yuan; Xu, Yan; Ma, Lang; Pool Pizzi, Melissa; Jin, Jiankang; Scott, Ailing W.; Huo, Longfei; Wang, Ying; Lee, Jeffrey H.; Bhutani, Manoop S.; Weston, Brian; Shanbhag, Namita D.; Johnson, Randy L.; Ajani, Jaffer A.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Oncology</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1410-1426</span>CODEN:
                <span class="NLM_cas:coden">MOONC3</span>;
        ISSN:<span class="NLM_cas:issn">1878-0261</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons Ltd.</span>)
        </div><div class="casAbstract">Hippo/YAP1 signaling is a major regulator of organ size, cancer stemness, and aggressive phenotype.  Thus, targeting YAP1 may provide a novel therapeutic strategy for tumors with high YAP1 expression in esophageal cancer (EC).  Chromatin immunopptn. (ChiP) and quant. ChiP-PCR were used to det. the regulation of the chromatin remodeling protein bromodomain-contg. protein 4 (BRD4) on YAP1.  The role of the bromodomain and extraterminal motif (BET) inhibitor JQ1, an established BRD4 inhibitor, on inhibition of YAP1 in EC cells was dissected using western blot, immunofluorescence, qPCR, and transient transfection.  The antitumor activities of BET inhibitor were further examd. by variety of functional assays, cell proliferation (MTS), tumorsphere, and ALDH1+ labeling in vitro and in vivo.  Here, we show that BRD4 regulates YAP1 expression and transcription.  ChiP assays revealed that BRD4 directly occupies YAP1 promoter and that JQ1 robustly blocks BRD4 binding to the YAP1 promoter.  Consequently, JQ1 strongly suppresses constitutive or induced YAP1 expression and transcription in EC cells and YAP1/Tead downstream transcriptional activity.  Intriguingly, radiation-resistant cells that acquire strong cancer stem cell traits and an aggressive phenotype can be effectively suppressed by JQ1 as assessed by cell proliferation, tumorsphere formation, and redn. in the ALDH1+ cells.  Moreover, effects of JQ1 are synergistically amplified by the addn. of docetaxel in vitro and in vivo.  Our results demonstrate that BRD4 is a crit. regulator of Hippo/YAP1 signaling and that BRD4 inhibitor JQ1 represents a new class of inhibitor of Hippo/YAP1 signaling, primarily targeting YAP1 high and therapy-resistant cancer cells enriched with cancer stem cell properties.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpOA8KqDxe2ALVg90H21EOLACvtfcHk0ljQx8X7_5-xpQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhsleltbnJ&md5=04d25857ce6910c64c99092a3e8b4c5f</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1002%2F1878-0261.12667&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F1878-0261.12667%26sid%3Dliteratum%253Aachs%26aulast%3DSong%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DXu%26aufirst%3DY.%26aulast%3DMa%26aufirst%3DL.%26aulast%3DPool%2BPizzi%26aufirst%3DM.%26aulast%3DJin%26aufirst%3DJ.%26aulast%3DScott%26aufirst%3DA.%2BW%26aulast%3DHuo%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DLee%26aufirst%3DJ.%2BH.%26aulast%3DBhutani%26aufirst%3DM.%2BS.%26aulast%3DWeston%26aufirst%3DB.%26aulast%3DShanbhag%26aufirst%3DN.%2BD%26aulast%3DJohnson%26aufirst%3DR.%2BL.%26aulast%3DAjani%26aufirst%3DJ.%2BA.%26atitle%3DTargeting%2520Hippo%2520coactivator%2520YAP1%2520through%2520BET%2520bromodomain%2520inhibition%2520in%2520esophageal%2520adenocarcinoma%26jtitle%3DMol.%2520Oncol.%26date%3D2020%26volume%3D14%26spage%3D1410%26epage%3D1426%26doi%3D10.1002%2F1878-0261.12667" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chiang, C. M.</span></span> <span> </span><span class="NLM_article-title">BRD4 engagement from chromatin targeting to transcriptional regulation: selective contact with acetylated histone H3 and H4</span>. <i>F1000 Biol. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">98</span>, <span class="refDoi"> DOI: 10.3410/B1-98</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.3410%2FB1-98" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=20495683" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A280%3ADC%252BC3MrlvVygtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2009&pages=98&author=C.+M.+Chiang&title=BRD4+engagement+from+chromatin+targeting+to+transcriptional+regulation%3A+selective+contact+with+acetylated+histone+H3+and+H4&doi=10.3410%2FB1-98"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Brd4 engagement from chromatin targeting to transcriptional regulation: selective contact with acetylated histone H3 and H4</span></div><div class="casAuthors">Chiang Cheng-Ming</div><div class="citationInfo"><span class="NLM_cas:title">F1000 biology reports</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">98</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Bromodomain-containing protein 4 (Brd4) contains two tandem bromodomains (BD1 and BD2) that bind preferentially to acetylated lysine residues found in histones and nonhistone proteins.  This molecular recognition allows Brd4 to associate with acetylated chromatin throughout the cell cycle and regulates transcription at targeted loci.  Recruitment of positive transcription elongation factor b, and possibly the general initiation cofactor Mediator as well, plays an important role in Brd4-regulated transcription.  Selective contacts with acetyl-lysines in nucleosomal histones and chromatin-binding factors likely provide a molecular switch modulating the steps from chromatin targeting to transcriptional regulation, thus further expanding the 'acetylation code' for combinatorial regulation in eukaryotes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcR81nP81pAX6sKL0FlyGHDofW6udTcc2eYRbpClr1jfAbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3MrlvVygtQ%253D%253D&md5=d575a30cf8b3d918367d6580c2c0d64a</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.3410%2FB1-98&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3410%252FB1-98%26sid%3Dliteratum%253Aachs%26aulast%3DChiang%26aufirst%3DC.%2BM.%26atitle%3DBRD4%2520engagement%2520from%2520chromatin%2520targeting%2520to%2520transcriptional%2520regulation%253A%2520selective%2520contact%2520with%2520acetylated%2520histone%2520H3%2520and%2520H4%26jtitle%3DF1000%2520Biol.%2520Rep.%26date%3D2009%26volume%3D1%26spage%3D98%26doi%3D10.3410%2FB1-98" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jonkers, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lis, J. T.</span></span> <span> </span><span class="NLM_article-title">Getting up to speed with transcription elongation by RNA polymerase II</span>. <i>Nat. Rev. Mol. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">167</span>– <span class="NLM_lpage">177</span>, <span class="refDoi"> DOI: 10.1038/nrm3953</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1038%2Fnrm3953" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=25693130" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BC2MXjtFemsrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2015&pages=167-177&author=I.+Jonkersauthor=J.+T.+Lis&title=Getting+up+to+speed+with+transcription+elongation+by+RNA+polymerase+II&doi=10.1038%2Fnrm3953"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Getting up to speed with transcription elongation by RNA polymerase II</span></div><div class="casAuthors">Jonkers, Iris; Lis, John T.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Molecular Cell Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">167-177</span>CODEN:
                <span class="NLM_cas:coden">NRMCBP</span>;
        ISSN:<span class="NLM_cas:issn">1471-0072</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Recent advances in sequencing techniques that measure nascent transcripts and that reveal the positioning of RNA polymerase II (Pol II) have shown that the pausing of Pol II in promoter-proximal regions and its release to initiate a phase of productive elongation are key steps in transcription regulation.  Moreover, after the release of Pol II from the promoter-proximal region, elongation rates are highly dynamic throughout the transcription of a gene, and vary on a gene-by-gene basis.  Interestingly, Pol II elongation rates affect co-transcriptional processes such as splicing, termination and genome stability.  Increasing nos. of factors and regulatory mechanisms have been assocd. with the steps of transcription elongation by Pol II, revealing that elongation is a highly complex process.  Elongation is thus now recognized as a key phase in the regulation of transcription by Pol II.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqmoGYHNxHU1bVg90H21EOLACvtfcHk0lj1qmvA1ko_cw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXjtFemsrw%253D&md5=6b8aa1cef47c37a0fd26a83f60cbab49</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1038%2Fnrm3953&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrm3953%26sid%3Dliteratum%253Aachs%26aulast%3DJonkers%26aufirst%3DI.%26aulast%3DLis%26aufirst%3DJ.%2BT.%26atitle%3DGetting%2520up%2520to%2520speed%2520with%2520transcription%2520elongation%2520by%2520RNA%2520polymerase%2520II%26jtitle%3DNat.%2520Rev.%2520Mol.%2520Cell%2520Biol.%26date%3D2015%26volume%3D16%26spage%3D167%26epage%3D177%26doi%3D10.1038%2Fnrm3953" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Stanlie, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yousif, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Akiyama, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Honjo, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Begum, N. A.</span></span> <span> </span><span class="NLM_article-title">Chromatin reader BRD4 functions in Ig class switching as a repair complex adaptor of nonhomologous end-joining</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">97</span>– <span class="NLM_lpage">110</span>, <span class="refDoi"> DOI: 10.1016/j.molcel.2014.05.018</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1016%2Fj.molcel.2014.05.018" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=24954901" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtVams73E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2014&pages=97-110&author=A.+Stanlieauthor=A.+S.+Yousifauthor=H.+Akiyamaauthor=T.+Honjoauthor=N.+A.+Begum&title=Chromatin+reader+BRD4+functions+in+Ig+class+switching+as+a+repair+complex+adaptor+of+nonhomologous+end-joining&doi=10.1016%2Fj.molcel.2014.05.018"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Chromatin reader Brd4 functions in Ig class switching as a repair complex adaptor of nonhomologous end-joining</span></div><div class="casAuthors">Stanlie, Andre; Yousif, Ashraf S.; Akiyama, Hideo; Honjo, Tasuku; Begum, Nasim A.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">97-110</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Class switch recombination (CSR) is a B cell-specific genomic alteration induced by activation-induced cytidine deaminase (AID)-dependent DNA break at the Ig heavy-chain locus, followed by repair.  Although chromatin-assocd. factors in promoting AID-induced DNA break have been widely reported, the involvement of chromatin adaptors at the repair phase of CSR remains unknown.  Here, we show that the acetylated histone reader Brd4 is crit. for nonhomologous end-joining (NHEJ) repair of AID- and I-SceI-induced DNA breaks.  Brd4 was recruited to the DNA break regions, and its depletion from the chromatin caused CSR impairment without affecting the DNA break generation.  Inhibition of Brd4 suppressed the accumulation of 53BP1 and uracil DNA glycosylase at the switch regions, perturbed the switch junctional microhomol., and reduced Igh/c-myc translocation.  We conclude that Brd4 serves as a chromatin platform required for the recruitment of repair components during CSR and general DNA damage.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGocjLJOukIdgbVg90H21EOLACvtfcHk0lj1qmvA1ko_cw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtVams73E&md5=af90286ee2aa03ffd10c9b6b6087c742</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2014.05.018&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2014.05.018%26sid%3Dliteratum%253Aachs%26aulast%3DStanlie%26aufirst%3DA.%26aulast%3DYousif%26aufirst%3DA.%2BS.%26aulast%3DAkiyama%26aufirst%3DH.%26aulast%3DHonjo%26aufirst%3DT.%26aulast%3DBegum%26aufirst%3DN.%2BA.%26atitle%3DChromatin%2520reader%2520BRD4%2520functions%2520in%2520Ig%2520class%2520switching%2520as%2520a%2520repair%2520complex%2520adaptor%2520of%2520nonhomologous%2520end-joining%26jtitle%3DMol.%2520Cell%26date%3D2014%26volume%3D55%26spage%3D97%26epage%3D110%26doi%3D10.1016%2Fj.molcel.2014.05.018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kim, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, S. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gong, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Battenhouse, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boutz, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bashyal, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Refvik, S. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiang, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xhemalce, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paull, T. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brodbelt, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marcotte, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, K. M.</span></span> <span> </span><span class="NLM_article-title">Systematic bromodomain protein screens identify homologous recombination and R-loop suppression pathways involved in genome integrity</span>. <i>Genes Dev.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>33</i></span>,  <span class="NLM_fpage">1751</span>– <span class="NLM_lpage">1774</span>, <span class="refDoi"> DOI: 10.1101/gad.331231.119</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1101%2Fgad.331231.119" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=31753913" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhtVSlt77J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2019&pages=1751-1774&author=J.+J.+Kimauthor=S.+Y.+Leeauthor=F.+Gongauthor=A.+M.+Battenhouseauthor=D.+R.+Boutzauthor=A.+Bashyalauthor=S.+T.+Refvikauthor=C.+M.+Chiangauthor=B.+Xhemalceauthor=T.+T.+Paullauthor=J.+S.+Brodbeltauthor=E.+M.+Marcotteauthor=K.+M.+Miller&title=Systematic+bromodomain+protein+screens+identify+homologous+recombination+and+R-loop+suppression+pathways+involved+in+genome+integrity&doi=10.1101%2Fgad.331231.119"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Systematic bromodomain protein screens identify homologous recombination and R-loop suppression pathways involved in genome integrity</span></div><div class="casAuthors">Kim, Jae Jin; Lee, Seo Yun; Gong, Fade; Battenhouse, Anna M.; Boutz, Daniel R.; Bashyal, Aarti; Refvik, Samantha T.; Chiang, Cheng-Ming; Xhemalce, Blerta; Paull, Tanya T.; Brodbelt, Jennifer S.; Marcotte, Edward M.; Miller, Kyle M.</div><div class="citationInfo"><span class="NLM_cas:title">Genes & Development</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">23-24</span>),
    <span class="NLM_cas:pages">1751-1774</span>CODEN:
                <span class="NLM_cas:coden">GEDEEP</span>;
        ISSN:<span class="NLM_cas:issn">0890-9369</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">Bromodomain proteins (BRD) are key chromatin regulators of genome function and stability as well as therapeutic targets in cancer.  Here, we systematically delineate the contribution of human BRD proteins for genome stability and DNA double-strand break (DSB) repair using several cell-based assays and proteomic interaction network anal.  Applying these approaches, we identify 24 of the 42 BRD proteins as promoters of DNA repair and/or genome integrity.  We identified a BRD-reader function of PCAF that bound TIP60-mediated histone acetylations at DSBs to recruit a DUB complex to deubiquitylate histone H2BK120, to allowing direct acetylation by PCAF, and repair of DSBs by homologous recombination.  We also discovered the bromo-and-extra-terminal (BET) BRD proteins, BRD2 and BRD4, as neg. regulators of transcription-assocd. RNA-DNA hybrids (R-loops) as inhibition of BRD2 or BRD4 increased R-loop formation, which generated DSBs.  These breaks were reliant on topoisomerase II, and BRD2 directly bound and activated topoisomerase I, a known restrainer of R-loops.  Thus, comprehensive interactome and functional profiling of BRD proteins revealed new homologous recombination and genome stability pathways, providing a framework to understand genome maintenance by BRD proteins and the effects of their pharmacol. inhibition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrS0sg29K3XdbVg90H21EOLACvtfcHk0lj1qmvA1ko_cw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhtVSlt77J&md5=31149336b6a185fb24186b32c6019886</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1101%2Fgad.331231.119&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fgad.331231.119%26sid%3Dliteratum%253Aachs%26aulast%3DKim%26aufirst%3DJ.%2BJ.%26aulast%3DLee%26aufirst%3DS.%2BY.%26aulast%3DGong%26aufirst%3DF.%26aulast%3DBattenhouse%26aufirst%3DA.%2BM.%26aulast%3DBoutz%26aufirst%3DD.%2BR.%26aulast%3DBashyal%26aufirst%3DA.%26aulast%3DRefvik%26aufirst%3DS.%2BT.%26aulast%3DChiang%26aufirst%3DC.%2BM.%26aulast%3DXhemalce%26aufirst%3DB.%26aulast%3DPaull%26aufirst%3DT.%2BT.%26aulast%3DBrodbelt%26aufirst%3DJ.%2BS.%26aulast%3DMarcotte%26aufirst%3DE.%2BM.%26aulast%3DMiller%26aufirst%3DK.%2BM.%26atitle%3DSystematic%2520bromodomain%2520protein%2520screens%2520identify%2520homologous%2520recombination%2520and%2520R-loop%2520suppression%2520pathways%2520involved%2520in%2520genome%2520integrity%26jtitle%3DGenes%2520Dev.%26date%3D2019%26volume%3D33%26spage%3D1751%26epage%3D1774%26doi%3D10.1101%2Fgad.331231.119" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baek, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramanand, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharp, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Welti, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rodrigues, D. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dolling, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Figueiredo, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sumanasuriya, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crespo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aslam, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yin, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mukherjee, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kanchwala, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hughes, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Halsey, W. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiang, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xing, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raj, G. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burma, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Bono, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mani, R. S.</span></span> <span> </span><span class="NLM_article-title">BRD4 promotes DNA repair and mediates the formation of TMPRSS2-ERG gene rearrangements in prostate cancer</span>. <i>Cell Rep.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">796</span>– <span class="NLM_lpage">808</span>, <span class="refDoi"> DOI: 10.1016/j.celrep.2017.12.078</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1016%2Fj.celrep.2017.12.078" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=29346775" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhvFahurY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2018&pages=796-808&author=X.+Liauthor=G.+Baekauthor=S.+G.+Ramanandauthor=A.+Sharpauthor=Y.+Gaoauthor=W.+Yuanauthor=J.+Weltiauthor=D.+N.+Rodriguesauthor=D.+Dollingauthor=I.+Figueiredoauthor=S.+Sumanasuriyaauthor=M.+Crespoauthor=A.+Aslamauthor=R.+Liauthor=Y.+Yinauthor=B.+Mukherjeeauthor=M.+Kanchwalaauthor=A.+M.+Hughesauthor=W.+S.+Halseyauthor=C.+M.+Chiangauthor=C.+Xingauthor=G.+V.+Rajauthor=S.+Burmaauthor=J.+de+Bonoauthor=R.+S.+Mani&title=BRD4+promotes+DNA+repair+and+mediates+the+formation+of+TMPRSS2-ERG+gene+rearrangements+in+prostate+cancer&doi=10.1016%2Fj.celrep.2017.12.078"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">BRD4 Promotes DNA Repair and Mediates the Formation of TMPRSS2-ERG Gene Rearrangements in Prostate Cancer</span></div><div class="casAuthors">Li, Xiangyi; Baek, Guem Hee; Ramanand, Susmita G.; Sharp, Adam; Gao, Yunpeng; Yuan, Wei; Welti, Jon; Rodrigues, Daniel N.; Dolling, David; Figueiredo, Ines; Sumanasuriya, Semini; Crespo, Mateus; Aslam, Adam; Li, Rui; Yin, Yi; Mukherjee, Bipasha; Kanchwala, Mohammed; Hughes, Ashley M.; Halsey, Wendy S.; Chiang, Cheng-Ming; Xing, Chao; Raj, Ganesh V.; Burma, Sandeep; de Bono, Johann; Mani, Ram S.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Reports</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">796-808</span>CODEN:
                <span class="NLM_cas:coden">CREED8</span>;
        ISSN:<span class="NLM_cas:issn">2211-1247</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">BRD4 belongs to the bromodomain and extraterminal (BET) family of chromatin reader proteins that bind acetylated histones and regulate gene expression.  Pharmacol. inhibition of BRD4 by BET inhibitors (BETi) has indicated antitumor activity against multiple cancer types.  We show that BRD4 is essential for the repair of DNA double-strand breaks (DSBs) and mediates the formation of oncogenic gene rearrangements by engaging the non-homologous end joining (NHEJ) pathway.  Mechanistically, genome-wide DNA breaks are assocd. with enhanced acetylation of histone H4, leading to BRD4 recruitment, and stable establishment of the DNA repair complex.  In support of this, we also show that, in clin. tumor samples, BRD4 protein levels are neg. assocd. with outcome after prostate cancer (PCa) radiation therapy.  Thus, in addn. to regulating gene expression, BRD4 is also a central player in the repair of DNA DSBs, with significant implications for cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr1nW-LR5WHMbVg90H21EOLACvtfcHk0lgMXLtHAIIyrg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhvFahurY%253D&md5=877fe254c14f13633c5528481c39ff92</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1016%2Fj.celrep.2017.12.078&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.celrep.2017.12.078%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DX.%26aulast%3DBaek%26aufirst%3DG.%26aulast%3DRamanand%26aufirst%3DS.%2BG.%26aulast%3DSharp%26aufirst%3DA.%26aulast%3DGao%26aufirst%3DY.%26aulast%3DYuan%26aufirst%3DW.%26aulast%3DWelti%26aufirst%3DJ.%26aulast%3DRodrigues%26aufirst%3DD.%2BN.%26aulast%3DDolling%26aufirst%3DD.%26aulast%3DFigueiredo%26aufirst%3DI.%26aulast%3DSumanasuriya%26aufirst%3DS.%26aulast%3DCrespo%26aufirst%3DM.%26aulast%3DAslam%26aufirst%3DA.%26aulast%3DLi%26aufirst%3DR.%26aulast%3DYin%26aufirst%3DY.%26aulast%3DMukherjee%26aufirst%3DB.%26aulast%3DKanchwala%26aufirst%3DM.%26aulast%3DHughes%26aufirst%3DA.%2BM.%26aulast%3DHalsey%26aufirst%3DW.%2BS.%26aulast%3DChiang%26aufirst%3DC.%2BM.%26aulast%3DXing%26aufirst%3DC.%26aulast%3DRaj%26aufirst%3DG.%2BV.%26aulast%3DBurma%26aufirst%3DS.%26aulast%3Dde%2BBono%26aufirst%3DJ.%26aulast%3DMani%26aufirst%3DR.%2BS.%26atitle%3DBRD4%2520promotes%2520DNA%2520repair%2520and%2520mediates%2520the%2520formation%2520of%2520TMPRSS2-ERG%2520gene%2520rearrangements%2520in%2520prostate%2520cancer%26jtitle%3DCell%2520Rep.%26date%3D2018%26volume%3D22%26spage%3D796%26epage%3D808%26doi%3D10.1016%2Fj.celrep.2017.12.078" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sun, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeong, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vellano, C. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ju, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meric-Bernstam, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yap, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hattersley, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Connor, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fawell, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, S. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peng, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mills, G. B.</span></span> <span> </span><span class="NLM_article-title">BRD4 inhibition is synthetic lethal with PARP inhibitors through the induction of homologous recombination deficiency</span>. <i>Cancer Cell</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>33</i></span>,  <span class="NLM_fpage">401</span>– <span class="NLM_lpage">416</span>, <span class="refDoi"> DOI: 10.1016/j.ccell.2018.01.019</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1016%2Fj.ccell.2018.01.019" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=29533782" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BC1cXksVent7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2018&pages=401-416&author=C.+Sunauthor=J.+Yinauthor=Y.+Fangauthor=J.+Chenauthor=K.+J.+Jeongauthor=X.+Chenauthor=C.+P.+Vellanoauthor=Z.+Juauthor=W.+Zhaoauthor=D.+Zhangauthor=Y.+Luauthor=F.+Meric-Bernstamauthor=T.+A.+Yapauthor=M.+Hattersleyauthor=M.+J.+O%E2%80%99Connorauthor=H.+Chenauthor=S.+Fawellauthor=S.+Y.+Linauthor=G.+Pengauthor=G.+B.+Mills&title=BRD4+inhibition+is+synthetic+lethal+with+PARP+inhibitors+through+the+induction+of+homologous+recombination+deficiency&doi=10.1016%2Fj.ccell.2018.01.019"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">BRD4 Inhibition Is Synthetic Lethal with PARP Inhibitors through the Induction of Homologous Recombination Deficiency</span></div><div class="casAuthors">Sun, Chaoyang; Yin, Jun; Fang, Yong; Chen, Jian; Jeong, Kang Jin; Chen, Xiaohua; Vellano, Christopher P.; Ju, Zhenlin; Zhao, Wei; Zhang, Dong; Lu, Yiling; Meric-Bernstam, Funda; Yap, Timothy A.; Hattersley, Maureen; O'Connor, Mark J.; Chen, Huawei; Fawell, Stephen; Lin, Shiaw-Yih; Peng, Guang; Mills, Gordon B.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">401-416.e8</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Poly(ADP-ribose) polymerase inhibitors (PARPi) are selectively active in cells with homologous recombination (HR) deficiency (HRD) caused by mutations in BRCA1, BRCA2, and other pathway members.  We sought small mols. that induce HRD in HR-competent cells to induce synthetic lethality with PARPi and extend the utility of PARPi.  We demonstrated that inhibition of bromodomain contg. 4 (BRD4) induced HRD and sensitized cells across multiple tumor lineages to PARPi regardless of BRCA1/2, TP53, RAS, or BRAF mutation status through depletion of the DNA double-stand break resection protein CtIP (C-terminal binding protein interacting protein).  Importantly, BRD4 inhibitor (BRD4i) treatment reversed multiple mechanisms of resistance to PARPi.  Furthermore, PARPi and BRD4i are synergistic in multiple in vivo models.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGooOJrqnT3mVLVg90H21EOLACvtfcHk0lgMXLtHAIIyrg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXksVent7c%253D&md5=460bd6deafbbfdaf581a8711c67be116</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1016%2Fj.ccell.2018.01.019&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccell.2018.01.019%26sid%3Dliteratum%253Aachs%26aulast%3DSun%26aufirst%3DC.%26aulast%3DYin%26aufirst%3DJ.%26aulast%3DFang%26aufirst%3DY.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DJeong%26aufirst%3DK.%2BJ.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DVellano%26aufirst%3DC.%2BP.%26aulast%3DJu%26aufirst%3DZ.%26aulast%3DZhao%26aufirst%3DW.%26aulast%3DZhang%26aufirst%3DD.%26aulast%3DLu%26aufirst%3DY.%26aulast%3DMeric-Bernstam%26aufirst%3DF.%26aulast%3DYap%26aufirst%3DT.%2BA.%26aulast%3DHattersley%26aufirst%3DM.%26aulast%3DO%25E2%2580%2599Connor%26aufirst%3DM.%2BJ.%26aulast%3DChen%26aufirst%3DH.%26aulast%3DFawell%26aufirst%3DS.%26aulast%3DLin%26aufirst%3DS.%2BY.%26aulast%3DPeng%26aufirst%3DG.%26aulast%3DMills%26aufirst%3DG.%2BB.%26atitle%3DBRD4%2520inhibition%2520is%2520synthetic%2520lethal%2520with%2520PARP%2520inhibitors%2520through%2520the%2520induction%2520of%2520homologous%2520recombination%2520deficiency%26jtitle%3DCancer%2520Cell%26date%3D2018%26volume%3D33%26spage%3D401%26epage%3D416%26doi%3D10.1016%2Fj.ccell.2018.01.019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">French, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miyoshi, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kubonishi, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grier, H. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perez-Atayde, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fletcher, J. A.</span></span> <span> </span><span class="NLM_article-title">BRD4-NUT fusion oncogene: a novel mechanism in aggressive carcinoma</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">304</span>– <span class="NLM_lpage">307</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=12543779" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BD3sXnt1Klug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2003&pages=304-307&author=C.+A.+Frenchauthor=I.+Miyoshiauthor=I.+Kubonishiauthor=H.+E.+Grierauthor=A.+R.+Perez-Ataydeauthor=J.+A.+Fletcher&title=BRD4-NUT+fusion+oncogene%3A+a+novel+mechanism+in+aggressive+carcinoma"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">BRD4-NUT Fusion Oncogene: A Novel Mechanism in Aggressive Carcinoma</span></div><div class="casAuthors">French, Christopher A.; Miyoshi, Isao; Kubonishi, Ichiro; Grier, Holcombe E.; Perez-Atayde, Antonio R.; Fletcher, Jonathan A.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">304-307</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">The poorly differentiated carcinoma with t(15;19)(q13,p13.1) is characterized by its highly aggressive, invariably lethal clin. course.  The chromosome 19 translocation breakpoint targets the BRD4 double bromodomain-contg. gene, which functions in regulation of cell cycle progression.  Herein the authors demonstrate that BRD4 is fused with nearly the entire transcript of the novel 15q13 gene, NUT (nuclear protein in testis), forming a 6.4-kb fusion oncogene, BRD4-NUT.  NUT, like BRD4, is predicted to encode a nuclear protein but, unlike the ubiquitous BRD4 transcript, is expressed only in testis.  These findings establish a model to elucidate the oncogenic consequences of unscheduled NUT expression and altered BRD4 function.  Very few fusion oncogenes have been identified in epithelial tumors, and BRD4-NUT is the first fusion oncogene mechanism identified in a highly lethal form of carcinoma.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrTa0K_kBu2pLVg90H21EOLACvtfcHk0lh7Ta_xRZLmKg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXnt1Klug%253D%253D&md5=e927ea5874b2013c7467f981e10d7efd</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DFrench%26aufirst%3DC.%2BA.%26aulast%3DMiyoshi%26aufirst%3DI.%26aulast%3DKubonishi%26aufirst%3DI.%26aulast%3DGrier%26aufirst%3DH.%2BE.%26aulast%3DPerez-Atayde%26aufirst%3DA.%2BR.%26aulast%3DFletcher%26aufirst%3DJ.%2BA.%26atitle%3DBRD4-NUT%2520fusion%2520oncogene%253A%2520a%2520novel%2520mechanism%2520in%2520aggressive%2520carcinoma%26jtitle%3DCancer%2520Res.%26date%3D2003%26volume%3D63%26spage%3D304%26epage%3D307" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, Y. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waring, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dey, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ozato, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peng, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ge, K.</span></span> <span> </span><span class="NLM_article-title">BRD4 binds to active enhancers to control cell identity gene induction in adipogenesis and myogenesis</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">2217</span>, <span class="refDoi"> DOI: 10.1038/s41467-017-02403-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1038%2Fs41467-017-02403-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=29263365" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A280%3ADC%252BC1MzjslSlsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=2217&author=J.+E.+Leeauthor=Y.+K.+Parkauthor=S.+Parkauthor=Y.+Jangauthor=N.+Waringauthor=A.+Deyauthor=K.+Ozatoauthor=B.+Laiauthor=W.+Pengauthor=K.+Ge&title=BRD4+binds+to+active+enhancers+to+control+cell+identity+gene+induction+in+adipogenesis+and+myogenesis&doi=10.1038%2Fs41467-017-02403-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Brd4 binds to active enhancers to control cell identity gene induction in adipogenesis and myogenesis</span></div><div class="casAuthors">Lee Ji-Eun; Park Young-Kwon; Park Sarah; Jang Younghoon; Waring Nicholas; Lai Binbin; Ge Kai; Waring Nicholas; Lai Binbin; Peng Weiqun; Dey Anup; Ozato Keiko</div><div class="citationInfo"><span class="NLM_cas:title">Nature communications</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">2217</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The epigenomic reader Brd4 is an important drug target for cancers.  However, its role in cell differentiation and animal development remains largely unclear.  Using two conditional knockout mouse strains and derived cells, we demonstrate that Brd4 controls cell identity gene induction and is essential for adipogenesis and myogenesis.  Brd4 co-localizes with lineage-determining transcription factors (LDTFs) on active enhancers during differentiation.  LDTFs coordinate with H3K4 mono-methyltransferases MLL3/MLL4 (KMT2C/KMT2D) and H3K27 acetyltransferases CBP/p300 to recruit Brd4 to enhancers activated during differentiation.  Brd4 deletion prevents the enrichment of Mediator and RNA polymerase II transcription machinery, but not that of LDTFs, MLL3/MLL4-mediated H3K4me1, and CBP/p300-mediated H3K27ac, on enhancers.  Consequently, Brd4 deletion prevents enhancer RNA production, cell identity gene induction and cell differentiation.  Interestingly, Brd4 is dispensable for maintaining cell identity genes in differentiated cells.  These findings identify Brd4 as an enhancer epigenomic reader that links active enhancers with cell identity gene induction in differentiation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcREMjSSm_naSGZyxfu64m4ffW6udTcc2eYdHD30pQivt7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1MzjslSlsQ%253D%253D&md5=35aa3eb2ed0f9c1f38a6a2d09b438957</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1038%2Fs41467-017-02403-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41467-017-02403-5%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DJ.%2BE.%26aulast%3DPark%26aufirst%3DY.%2BK.%26aulast%3DPark%26aufirst%3DS.%26aulast%3DJang%26aufirst%3DY.%26aulast%3DWaring%26aufirst%3DN.%26aulast%3DDey%26aufirst%3DA.%26aulast%3DOzato%26aufirst%3DK.%26aulast%3DLai%26aufirst%3DB.%26aulast%3DPeng%26aufirst%3DW.%26aulast%3DGe%26aufirst%3DK.%26atitle%3DBRD4%2520binds%2520to%2520active%2520enhancers%2520to%2520control%2520cell%2520identity%2520gene%2520induction%2520in%2520adipogenesis%2520and%2520myogenesis%26jtitle%3DNat.%2520Commun.%26date%3D2017%26volume%3D8%26spage%3D2217%26doi%3D10.1038%2Fs41467-017-02403-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ge, J. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kwon, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jovanovic, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murphy, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gulvady, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fassl, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trinh, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heavey, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luoma, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paweletz, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thorner, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wucherpfennig, K. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sicinski, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mcdonald, T. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pellman, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michor, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Polyak, K.</span></span> <span> </span><span class="NLM_article-title">Acquired resistance to combined BET and CDK4/6 inhibition in triple-negative breast cancer</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">2350</span>, <span class="refDoi"> DOI: 10.1038/s41467-020-16170-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1038%2Fs41467-020-16170-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=32393766" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BB3cXpt1Khsr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2020&pages=2350&author=J.+Y.+Geauthor=S.+Shuauthor=M.+Kwonauthor=B.+Jovanovicauthor=K.+Murphyauthor=A.+Gulvadyauthor=A.+Fasslauthor=A.+Trinhauthor=Y.+Kuangauthor=G.+A.+Heaveyauthor=A.+Luomaauthor=C.+Paweletzauthor=A.+R.+Thornerauthor=K.+W.+Wucherpfennigauthor=J.+Qiauthor=M.+Brownauthor=P.+Sicinskiauthor=T.+O.+Mcdonaldauthor=D.+Pellmanauthor=F.+Michorauthor=K.+Polyak&title=Acquired+resistance+to+combined+BET+and+CDK4%2F6+inhibition+in+triple-negative+breast+cancer&doi=10.1038%2Fs41467-020-16170-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Acquired resistance to combined BET and CDK4/6 inhibition in triple-negative breast cancer</span></div><div class="casAuthors">Ge, Jennifer Y.; Shu, Shaokun; Kwon, Mijung; Jovanovic, Bojana; Murphy, Katherine; Gulvady, Anushree; Fassl, Anne; Trinh, Anne; Kuang, Yanan; Heavey, Grace A.; Luoma, Adrienne; Paweletz, Cloud; Thorner, Aaron R.; Wucherpfennig, Kai W.; Qi, Jun; Brown, Myles; Sicinski, Piotr; McDonald, Thomas O.; Pellman, David; Michor, Franziska; Polyak, Kornelia</div><div class="citationInfo"><span class="NLM_cas:title">Nature Communications</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">2350</span>CODEN:
                <span class="NLM_cas:coden">NCAOBW</span>;
        ISSN:<span class="NLM_cas:issn">2041-1723</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Abstr.: BET inhibitors are promising therapeutic agents for the treatment of triple-neg. breast cancer (TNBC), but the rapid emergence of resistance necessitates investigation of combination therapies and their effects on tumor evolution.  Here, we show that palbociclib, a CDK4/6 inhibitor, and paclitaxel, a microtubule inhibitor, synergize with the BET inhibitor JQ1 in TNBC lines.  High-complexity DNA barcoding and math. modeling indicate a high rate of de novo acquired resistance to these drugs relative to pre-existing resistance.  We demonstrate that the combination of JQ1 and palbociclib induces cell division errors, which can increase the chance of developing aneuploidy.  Characterizing acquired resistance to combination treatment at a single cell level shows heterogeneous mechanisms including activation of G1-S and senescence pathways.  Our results establish a rationale for further investigation of combined BET and CDK4/6 inhibition in TNBC and suggest novel mechanisms of action for these drugs and new vulnerabilities in cells after emergence of resistance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq8HPBD7AnPVLVg90H21EOLACvtfcHk0lh7Ta_xRZLmKg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXpt1Khsr4%253D&md5=ff7d4b06ea3b3a7f6eec1e97764e169b</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1038%2Fs41467-020-16170-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41467-020-16170-3%26sid%3Dliteratum%253Aachs%26aulast%3DGe%26aufirst%3DJ.%2BY.%26aulast%3DShu%26aufirst%3DS.%26aulast%3DKwon%26aufirst%3DM.%26aulast%3DJovanovic%26aufirst%3DB.%26aulast%3DMurphy%26aufirst%3DK.%26aulast%3DGulvady%26aufirst%3DA.%26aulast%3DFassl%26aufirst%3DA.%26aulast%3DTrinh%26aufirst%3DA.%26aulast%3DKuang%26aufirst%3DY.%26aulast%3DHeavey%26aufirst%3DG.%2BA.%26aulast%3DLuoma%26aufirst%3DA.%26aulast%3DPaweletz%26aufirst%3DC.%26aulast%3DThorner%26aufirst%3DA.%2BR.%26aulast%3DWucherpfennig%26aufirst%3DK.%2BW.%26aulast%3DQi%26aufirst%3DJ.%26aulast%3DBrown%26aufirst%3DM.%26aulast%3DSicinski%26aufirst%3DP.%26aulast%3DMcdonald%26aufirst%3DT.%2BO.%26aulast%3DPellman%26aufirst%3DD.%26aulast%3DMichor%26aufirst%3DF.%26aulast%3DPolyak%26aufirst%3DK.%26atitle%3DAcquired%2520resistance%2520to%2520combined%2520BET%2520and%2520CDK4%252F6%2520inhibition%2520in%2520triple-negative%2520breast%2520cancer%26jtitle%3DNat.%2520Commun.%26date%3D2020%26volume%3D11%26spage%3D2350%26doi%3D10.1038%2Fs41467-020-16170-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rosencrance, C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ammouri, H. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ge, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rendleman, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marshall, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eagen, K. P.</span></span> <span> </span><span class="NLM_article-title">Chromatin hyperacetylation impacts chromosome folding by forming a nuclear subcompartment</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>78</i></span>,  <span class="NLM_fpage">112</span>– <span class="NLM_lpage">126</span>, <span class="refDoi"> DOI: 10.1016/j.molcel.2020.03.018</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1016%2Fj.molcel.2020.03.018" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=32243828" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BB3cXmsFOjsbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=78&publication_year=2020&pages=112-126&author=C.+D.+Rosencranceauthor=H.+N.+Ammouriauthor=Q.+Yuauthor=T.+Geauthor=E.+J.+Rendlemanauthor=S.+A.+Marshallauthor=K.+P.+Eagen&title=Chromatin+hyperacetylation+impacts+chromosome+folding+by+forming+a+nuclear+subcompartment&doi=10.1016%2Fj.molcel.2020.03.018"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Chromatin Hyperacetylation Impacts Chromosome Folding by Forming a Nuclear Subcompartment</span></div><div class="casAuthors">Rosencrance, Celeste D.; Ammouri, Haneen N.; Yu, Qi; Ge, Tiffany; Rendleman, Emily J.; Marshall, Stacy A.; Eagen, Kyle P.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">78</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">112-126.e12</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Delineating how chromosomes fold at length scales beyond one megabase remains obscure relative to smaller-scale folding into TADs, loops, and nucleosomes.  We find that rather than simply unfolding chromatin, histone hyperacetylation results in interactions between distant genomic loci sepd. by tens to hundreds of megabases, even in the absence of transcription.  These hyperacetylated "megadomains" are formed by the BRD4-NUT fusion oncoprotein, interact both within and between chromosomes, and form a specific nuclear subcompartment that has elevated gene activity with respect to other subcompartments.  Pharmacol. degrdn. of BRD4-NUT results in collapse of megadomains and attenuation of the interactions between them.  In contrast, these interactions persist and contacts between newly acetylated regions are formed after inhibiting RNA polymerase II initiation.  Our structure-function approach thus reveals that broad chromatin domains of identical biochem. compn., independent of transcription, form nuclear subcompartments, and also indicates the potential of altering chromosome structure for treating human disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp7CF3cuc8CRrVg90H21EOLACvtfcHk0lg46KQkSQGXog"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXmsFOjsbk%253D&md5=2772ff70e2e2f1b07b4d7a77185d4b76</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2020.03.018&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2020.03.018%26sid%3Dliteratum%253Aachs%26aulast%3DRosencrance%26aufirst%3DC.%2BD.%26aulast%3DAmmouri%26aufirst%3DH.%2BN.%26aulast%3DYu%26aufirst%3DQ.%26aulast%3DGe%26aufirst%3DT.%26aulast%3DRendleman%26aufirst%3DE.%2BJ.%26aulast%3DMarshall%26aufirst%3DS.%2BA.%26aulast%3DEagen%26aufirst%3DK.%2BP.%26atitle%3DChromatin%2520hyperacetylation%2520impacts%2520chromosome%2520folding%2520by%2520forming%2520a%2520nuclear%2520subcompartment%26jtitle%3DMol.%2520Cell%26date%3D2020%26volume%3D78%26spage%3D112%26epage%3D126%26doi%3D10.1016%2Fj.molcel.2020.03.018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, X. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kulej, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Macdonald, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiang, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garcia, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">You, J.</span></span> <span> </span><span class="NLM_article-title">Uncovering BRD4 hyperphosphorylation associated with cellular transformation in NUT midline carcinoma</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>114</i></span>,  <span class="NLM_fpage">E5352</span>– <span class="NLM_lpage">E5361</span>, <span class="refDoi"> DOI: 10.1073/pnas.1703071114</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1073%2Fpnas.1703071114" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=28630312" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtVWmtLjM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=114&publication_year=2017&pages=E5352-E5361&author=R.+Wangauthor=X.+J.+Caoauthor=K.+Kulejauthor=W.+Liuauthor=T.+Maauthor=M.+Macdonaldauthor=C.+M.+Chiangauthor=B.+A.+Garciaauthor=J.+You&title=Uncovering+BRD4+hyperphosphorylation+associated+with+cellular+transformation+in+NUT+midline+carcinoma&doi=10.1073%2Fpnas.1703071114"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Uncovering BRD4 hyperphosphorylation associated with cellular transformation in NUT midline carcinoma</span></div><div class="casAuthors">Wang, Ranran; Cao, Xing-Jun; Kulej, Katarzyna; Liu, Wei; Ma, Tongcui; MacDonald, Margo; Chiang, Cheng-Ming; Garcia, Benjamin A.; You, Jianxin</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">114</span>
        (<span class="NLM_cas:issue">27</span>),
    <span class="NLM_cas:pages">E5352-E5361</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The epigenetic reader BRD4 plays a vital role in transcriptional regulation, cellular growth control, and cell-cycle progression.  Dysregulation of BRD4 function has been implicated in the pathogenesis of a wide range of cancers.  However, how BRD4 is regulated to maintain its normal function in healthy cells and how alteration of this process leads to cancer remain poorly understood.  In this study, we discovered that BRD4 is hyperphosphorylated in NUT midline carcinoma and identified CDK9 as a potential kinase mediating BRD4 hyperphosphorylation.  Disruption of BRD4 hyperphosphorylation using both chem. and mol. inhibitors led to the repression of BRD4 downstream oncogenes and abrogation of cellular transformation.  BRD4 hyperphosphorylation is also obsd. in other cancers displaying enhanced BRD4 oncogenic activity.  Our study revealed a mechanism that may regulate BRD4 biol. function through phosphorylation, which, when dysregulated, could lead to oncogenesis.  Our finding points to strategies to target the aberrant BRD4 signaling specifically for cancer intervention.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo3oN-XMSFJurVg90H21EOLACvtfcHk0lg46KQkSQGXog"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtVWmtLjM&md5=51fc4e5afb1d86b8a4e30b0c1cbc066e</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1703071114&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1703071114%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DR.%26aulast%3DCao%26aufirst%3DX.%2BJ.%26aulast%3DKulej%26aufirst%3DK.%26aulast%3DLiu%26aufirst%3DW.%26aulast%3DMa%26aufirst%3DT.%26aulast%3DMacdonald%26aufirst%3DM.%26aulast%3DChiang%26aufirst%3DC.%2BM.%26aulast%3DGarcia%26aufirst%3DB.%2BA.%26aulast%3DYou%26aufirst%3DJ.%26atitle%3DUncovering%2520BRD4%2520hyperphosphorylation%2520associated%2520with%2520cellular%2520transformation%2520in%2520NUT%2520midline%2520carcinoma%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2017%26volume%3D114%26spage%3DE5352%26epage%3DE5361%26doi%3D10.1073%2Fpnas.1703071114" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ceribelli, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hou, Z. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kelly, P. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, D. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wright, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ganapathi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evbuomwan, M. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pittaluga, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaffer, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marcucci, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Forman, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiao, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guha, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferrer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chaperot, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plumas, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jaffe, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reizis, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Staudt, L. M.</span></span> <span> </span><span class="NLM_article-title">A druggable TCF4- and BRD4-dependent transcriptional network sustains malignancy in blastic plasmacytoid dendritic cell neoplasm</span>. <i>Cancer Cell</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">764</span>– <span class="NLM_lpage">778</span>, <span class="refDoi"> DOI: 10.1016/j.ccell.2016.10.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1016%2Fj.ccell.2016.10.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=27846392" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvVOgsLfN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2016&pages=764-778&author=M.+Ceribelliauthor=Z.+E.+Houauthor=P.+N.+Kellyauthor=D.+W.+Huangauthor=G.+Wrightauthor=K.+Ganapathiauthor=M.+O.+Evbuomwanauthor=S.+Pittalugaauthor=A.+L.+Shafferauthor=G.+Marcucciauthor=S.+J.+Formanauthor=W.+Xiaoauthor=R.+Guhaauthor=X.+Zhangauthor=M.+Ferrerauthor=L.+Chaperotauthor=J.+Plumasauthor=E.+S.+Jaffeauthor=C.+J.+Thomasauthor=B.+Reizisauthor=L.+M.+Staudt&title=A+druggable+TCF4-+and+BRD4-dependent+transcriptional+network+sustains+malignancy+in+blastic+plasmacytoid+dendritic+cell+neoplasm&doi=10.1016%2Fj.ccell.2016.10.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">A Druggable TCF4- and BRD4-Dependent Transcriptional Network Sustains Malignancy in Blastic Plasmacytoid Dendritic Cell Neoplasm</span></div><div class="casAuthors">Ceribelli, Michele; Hou, Zhiying Esther; Kelly, Priscilla N.; Huang, Da Wei; Wright, George; Ganapathi, Karthik; Evbuomwan, Moses O.; Pittaluga, Stefania; Shaffer, Arthur L.; Marcucci, Guido; Forman, Stephen J.; Xiao, Wenming; Guha, Rajarshi; Zhang, Xiaohu; Ferrer, Marc; Chaperot, Laurence; Plumas, Joel; Jaffe, Elaine S.; Thomas, Craig J.; Reizis, Boris; Staudt, Louis M.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">764-778</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Blastic plasmacytoid dendritic cell neoplasm (BPDCN) is an aggressive and largely incurable hematol. malignancy originating from plasmacytoid dendritic cells (pDCs).  Using RNAi screening, we identified the E-box transcription factor TCF4 as a master regulator of the BPDCN oncogenic program.  TCF4 served as a faithful diagnostic marker of BPDCN, and its downregulation caused the loss of the BPDCN-specific gene expression program and apoptosis.  High-throughput drug screening revealed that bromodomain and extra-terminal domain inhibitors (BETis) induced BPDCN apoptosis, which was attributable to disruption of a BPDCN-specific transcriptional network controlled by TCF4-dependent super-enhancers.  BETis retarded the growth of BPDCN xenografts, supporting their clin. evaluation in this recalcitrant malignancy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqIY_1k7hZSwrVg90H21EOLACvtfcHk0lg46KQkSQGXog"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvVOgsLfN&md5=8b21e09a917549c508f1df0916eb7412</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1016%2Fj.ccell.2016.10.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccell.2016.10.002%26sid%3Dliteratum%253Aachs%26aulast%3DCeribelli%26aufirst%3DM.%26aulast%3DHou%26aufirst%3DZ.%2BE.%26aulast%3DKelly%26aufirst%3DP.%2BN.%26aulast%3DHuang%26aufirst%3DD.%2BW.%26aulast%3DWright%26aufirst%3DG.%26aulast%3DGanapathi%26aufirst%3DK.%26aulast%3DEvbuomwan%26aufirst%3DM.%2BO.%26aulast%3DPittaluga%26aufirst%3DS.%26aulast%3DShaffer%26aufirst%3DA.%2BL.%26aulast%3DMarcucci%26aufirst%3DG.%26aulast%3DForman%26aufirst%3DS.%2BJ.%26aulast%3DXiao%26aufirst%3DW.%26aulast%3DGuha%26aufirst%3DR.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DFerrer%26aufirst%3DM.%26aulast%3DChaperot%26aufirst%3DL.%26aulast%3DPlumas%26aufirst%3DJ.%26aulast%3DJaffe%26aufirst%3DE.%2BS.%26aulast%3DThomas%26aufirst%3DC.%2BJ.%26aulast%3DReizis%26aufirst%3DB.%26aulast%3DStaudt%26aufirst%3DL.%2BM.%26atitle%3DA%2520druggable%2520TCF4-%2520and%2520BRD4-dependent%2520transcriptional%2520network%2520sustains%2520malignancy%2520in%2520blastic%2520plasmacytoid%2520dendritic%2520cell%2520neoplasm%26jtitle%3DCancer%2520Cell%26date%3D2016%26volume%3D30%26spage%3D764%26epage%3D778%26doi%3D10.1016%2Fj.ccell.2016.10.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kleppe, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Levine, R. L.</span></span> <span> </span><span class="NLM_article-title">An unexpected chink in the transcriptional armor of plasmacytoid dendritic neoplasms</span>. <i>Cancer Cell</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">659</span>– <span class="NLM_lpage">660</span>, <span class="refDoi"> DOI: 10.1016/j.ccell.2016.10.016</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1016%2Fj.ccell.2016.10.016" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=27846385" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvVOgsb7M" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2016&pages=659-660&author=M.+Kleppeauthor=R.+L.+Levine&title=An+unexpected+chink+in+the+transcriptional+armor+of+plasmacytoid+dendritic+neoplasms&doi=10.1016%2Fj.ccell.2016.10.016"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">An Unexpected Chink in the Transcriptional Armor of Plasmacytoid Dendritic Neoplasms</span></div><div class="casAuthors">Kleppe, Maria; Levine, Ross L.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">659-660</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">In this issue of Cancer Cell, Ceribelli et al. use functional genomic and chem. screening to reveal the existence of a TCF4/BRD4-dependent oncogenic regulatory network in blastic plasmacytoid dendritic cell neoplasm (BPDCN) and demonstrate that BPDCN cells are highly sensitive to therapeutic targeting of this novel dependency.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqiVskGYzNKq7Vg90H21EOLACvtfcHk0liZTj_yaA1PFQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvVOgsb7M&md5=0cc7bda6c1edd829af3a3ae19dea5a6c</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1016%2Fj.ccell.2016.10.016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccell.2016.10.016%26sid%3Dliteratum%253Aachs%26aulast%3DKleppe%26aufirst%3DM.%26aulast%3DLevine%26aufirst%3DR.%2BL.%26atitle%3DAn%2520unexpected%2520chink%2520in%2520the%2520transcriptional%2520armor%2520of%2520plasmacytoid%2520dendritic%2520neoplasms%26jtitle%3DCancer%2520Cell%26date%3D2016%26volume%3D30%26spage%3D659%26epage%3D660%26doi%3D10.1016%2Fj.ccell.2016.10.016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Olley, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ansari, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bengani, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grimes, G. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rhodes, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">von Kriegsheim, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blatnik, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stewart, F. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wakeling, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carroll, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ross, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, S.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bickmore, W. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pradeepa, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">FitzPatrick, D. R.</span></span> <span> </span><span class="NLM_article-title">Fitzpatrick D. R. BRD4 interacts with NIPBL and BRD4 is mutated in a Cornelia de Lange-like syndrome</span>. <i>Nat. Genet.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>50</i></span>,  <span class="NLM_fpage">329</span>– <span class="NLM_lpage">332</span>, <span class="refDoi"> DOI: 10.1038/s41588-018-0042-y</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1038%2Fs41588-018-0042-y" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=29379197" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BC1cXkslGiu78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2018&pages=329-332&author=G.+Olleyauthor=M.+Ansariauthor=H.+Benganiauthor=G.+R.+Grimesauthor=J.+Rhodesauthor=A.+von+Kriegsheimauthor=A.+Blatnikauthor=F.+J.+Stewartauthor=E.+Wakelingauthor=N.+Carrollauthor=A.+Rossauthor=S.-M.+Parkauthor=W.+A.+Bickmoreauthor=M.+M.+Pradeepaauthor=D.+R.+FitzPatrick&title=Fitzpatrick+D.+R.+BRD4+interacts+with+NIPBL+and+BRD4+is+mutated+in+a+Cornelia+de+Lange-like+syndrome&doi=10.1038%2Fs41588-018-0042-y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">BRD4 interacts with NIPBL and BRD4 is mutated in a Cornelia de Lange-like syndrome</span></div><div class="casAuthors">Olley, Gabrielle; Ansari, Morad; Bengani, Hemant; Grimes, Graeme R.; Rhodes, James; von Kriegsheim, Alex; Blatnik, Ana; Stewart, Fiona J.; Wakeling, Emma; Carroll, Nicola; Ross, Alison; Park, Soo-Mi; Bickmore, Wendy A.; Pradeepa, Madapura M.; FitzPatrick, David R.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Genetics</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">329-332</span>CODEN:
                <span class="NLM_cas:coden">NGENEC</span>;
        ISSN:<span class="NLM_cas:issn">1061-4036</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">We found that the clin. phenotype assocd. with BRD4 haploinsufficiency overlapped with that of Cornelia de Lange syndrome (CdLS), which is most often caused by mutation of NIPBL.  More typical CdLS was obsd. with a de novo BRD4 missense variant, which retained the ability to coimmunoppt. with NIPBL, but bound poorly to acetylated histones.  BRD4 and NIPBL displayed correlated binding at super-enhancers and appeared to co-regulate developmental gene expression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGodrM8Io0YGb7Vg90H21EOLACvtfcHk0liZTj_yaA1PFQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXkslGiu78%253D&md5=87ef00b8de0b134ea874bc9092bdd632</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1038%2Fs41588-018-0042-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41588-018-0042-y%26sid%3Dliteratum%253Aachs%26aulast%3DOlley%26aufirst%3DG.%26aulast%3DAnsari%26aufirst%3DM.%26aulast%3DBengani%26aufirst%3DH.%26aulast%3DGrimes%26aufirst%3DG.%2BR.%26aulast%3DRhodes%26aufirst%3DJ.%26aulast%3Dvon%2BKriegsheim%26aufirst%3DA.%26aulast%3DBlatnik%26aufirst%3DA.%26aulast%3DStewart%26aufirst%3DF.%2BJ.%26aulast%3DWakeling%26aufirst%3DE.%26aulast%3DCarroll%26aufirst%3DN.%26aulast%3DRoss%26aufirst%3DA.%26aulast%3DPark%26aufirst%3DS.-M.%26aulast%3DBickmore%26aufirst%3DW.%2BA.%26aulast%3DPradeepa%26aufirst%3DM.%2BM.%26aulast%3DFitzPatrick%26aufirst%3DD.%2BR.%26atitle%3DFitzpatrick%2520D.%2520R.%2520BRD4%2520interacts%2520with%2520NIPBL%2520and%2520BRD4%2520is%2520mutated%2520in%2520a%2520Cornelia%2520de%2520Lange-like%2520syndrome%26jtitle%3DNat.%2520Genet.%26date%3D2018%26volume%3D50%26spage%3D329%26epage%3D332%26doi%3D10.1038%2Fs41588-018-0042-y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Korb, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herre, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zucker-Scharff, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gresack, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allis, C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Darnell, R. B.</span></span> <span> </span><span class="NLM_article-title">Excess translation of epigenetic regulators contributes to Fragile X Syndrome and is alleviated by BRD4 inhibition</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>170</i></span>,  <span class="NLM_fpage">1209</span>– <span class="NLM_lpage">1223</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2017.07.033</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1016%2Fj.cell.2017.07.033" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=28823556" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtlKmsr3O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=170&publication_year=2017&pages=1209-1223&author=E.+Korbauthor=M.+Herreauthor=I.+Zucker-Scharffauthor=J.+Gresackauthor=C.+D.+Allisauthor=R.+B.+Darnell&title=Excess+translation+of+epigenetic+regulators+contributes+to+Fragile+X+Syndrome+and+is+alleviated+by+BRD4+inhibition&doi=10.1016%2Fj.cell.2017.07.033"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Excess Translation of Epigenetic Regulators Contributes to Fragile X Syndrome and Is Alleviated by Brd4 Inhibition</span></div><div class="casAuthors">Korb, Erica; Herre, Margaret; Zucker-Scharff, Ilana; Gresack, Jodi; Allis, C. David; Darnell, Robert B.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">170</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1209-1223.e20</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Fragile X syndrome (FXS) is a leading genetic cause of intellectual disability and autism.  FXS results from the loss of function of fragile X mental retardation protein (FMRP), which represses translation of target transcripts.  Most of the well-characterized target transcripts of FMRP are synaptic proteins, yet targeting these proteins has not provided effective treatments.  We examd. a group of FMRP targets that encode transcriptional regulators, particularly chromatin-assocd. proteins.  Loss of FMRP in mice results in widespread changes in chromatin regulation and aberrant gene expression.  To det. if targeting epigenetic factors could reverse phenotypes assocd. with the disorder, we focused on Brd4, a BET protein and chromatin reader targeted by FMRP.  Inhibition of Brd4 function alleviated many of the phenotypes assocd. with FXS.  We conclude that loss of FMRP results in significant epigenetic misregulation and that targeting transcription via epigenetic regulators like Brd4 may provide new treatments for FXS.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrJePhuggwmkLVg90H21EOLACvtfcHk0liZTj_yaA1PFQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtlKmsr3O&md5=0a374238040e3f466323ddf17bd4b372</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2017.07.033&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2017.07.033%26sid%3Dliteratum%253Aachs%26aulast%3DKorb%26aufirst%3DE.%26aulast%3DHerre%26aufirst%3DM.%26aulast%3DZucker-Scharff%26aufirst%3DI.%26aulast%3DGresack%26aufirst%3DJ.%26aulast%3DAllis%26aufirst%3DC.%2BD.%26aulast%3DDarnell%26aufirst%3DR.%2BB.%26atitle%3DExcess%2520translation%2520of%2520epigenetic%2520regulators%2520contributes%2520to%2520Fragile%2520X%2520Syndrome%2520and%2520is%2520alleviated%2520by%2520BRD4%2520inhibition%26jtitle%3DCell%26date%3D2017%26volume%3D170%26spage%3D1209%26epage%3D1223%26doi%3D10.1016%2Fj.cell.2017.07.033" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wold, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tian, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brasier, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, J.</span></span> <span> </span><span class="NLM_article-title">Drug discovery targeting bromodomain-containing protein 4</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">4533</span>– <span class="NLM_lpage">4558</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b01761</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01761" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BC2sXis1yiurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=4533-4558&author=Z.+Liuauthor=P.+Wangauthor=H.+Chenauthor=E.+A.+Woldauthor=B.+Tianauthor=A.+R.+Brasierauthor=J.+Zhou&title=Drug+discovery+targeting+bromodomain-containing+protein+4&doi=10.1021%2Facs.jmedchem.6b01761"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Drug Discovery Targeting Bromodomain-Containing Protein 4</span></div><div class="casAuthors">Liu, Zhiqing; Wang, Pingyuan; Chen, Haiying; Wold, Eric A.; Tian, Bing; Brasier, Allan R.; Zhou, Jia</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">4533-4558</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  BRD4, the most extensively studied member of BET family, is an epigenetic regulator that localizes to DNA via binding acetylated histones and controls the expression of therapeutically important gene regulatory networks through recruiting transcription factors to form mediator complexes, phosphorylating RNA polymerase II and by its intrinsic histone acetyltransferase activity.  Disrupting the protein-protein interactions between BRD4 and acetyl-lysine has been shown to effectively block cell proliferation in cancer, cytokine prodn. in acute inflammation, etc.  To date, significant efforts have been devoted to the development of BRD4 inhibitors, and consequently, a dozen have progressed into human clin. trials.  Herein, the authors summarize the advances in drug discovery and development of BRD4 inhibitors by focusing on their chemotypes, in vitro and in vivo activity, selectivity, relevant mechanisms of action and therapeutic potential.  Opportunities and challenges to achieve selective and efficacious BRD4 inhibitors as a viable therapeutic strategy for human diseases are also highlighted.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq6IAERH4iggrVg90H21EOLACvtfcHk0ljcAsLhndSRKw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXis1yiurg%253D&md5=6d7ba5844073fac8c916b1249af402e5</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01761&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01761%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DZ.%26aulast%3DWang%26aufirst%3DP.%26aulast%3DChen%26aufirst%3DH.%26aulast%3DWold%26aufirst%3DE.%2BA.%26aulast%3DTian%26aufirst%3DB.%26aulast%3DBrasier%26aufirst%3DA.%2BR.%26aulast%3DZhou%26aufirst%3DJ.%26atitle%3DDrug%2520discovery%2520targeting%2520bromodomain-containing%2520protein%25204%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D4533%26epage%3D4558%26doi%3D10.1021%2Facs.jmedchem.6b01761" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bechter, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schoffski, P.</span></span> <span> </span><span class="NLM_article-title">Make your best BET: The emerging role of BET inhibitor treatment in malignant tumors</span>. <i>Pharmacol. Ther.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>208</i></span>,  <span class="NLM_fpage">107479</span>, <span class="refDoi"> DOI: 10.1016/j.pharmthera.2020.107479</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1016%2Fj.pharmthera.2020.107479" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=31931101" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BB3cXotFyksQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=208&publication_year=2020&pages=107479&author=O.+Bechterauthor=P.+Schoffski&title=Make+your+best+BET%3A+The+emerging+role+of+BET+inhibitor+treatment+in+malignant+tumors&doi=10.1016%2Fj.pharmthera.2020.107479"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Make your best BET the emerging role of BET inhibitor treatment in malignant tumors</span></div><div class="casAuthors">Bechter, Oliver; Schoffski, Patrick</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacology & Therapeutics</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">208</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">107479</span>CODEN:
                <span class="NLM_cas:coden">PHTHDT</span>;
        ISSN:<span class="NLM_cas:issn">0163-7258</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">Bromodomains are protein-protein interaction modules with a great diversity in terms of no. of proteins and their function.  The bromodomain and extraterminal protein (BET) represents a distinct subclass of bromodomain proteins mainly involved in transcriptional regulation via their interaction with acetylated chromatin.  In cancer cells BET proteins are found to be altered in many ways such as overexpression, mutations and fusions of BET proteins or their interference with cancer relevant signaling pathways and transcriptional programs in order to sustain cancer growth and viability.  Blocking BET protein function with small mols. is assocd. with therapeutic activity.  Consequently, a variety of small mols. have been developed and a no. of phase I clin. trials have explored their tolerability and efficacy in patients with solid tumors and hematol. malignancies.  We will review the rational for applying BET inhibitors in the clinic and we will discuss the toxicity profile as well as efficacy of this new class of protein inhibitors.  We will also highlight the emerging problem of treatment resistance and the potential these drugs might have when combined with other anti-cancer therapies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqiNKo5rStKX7Vg90H21EOLACvtfcHk0ljcAsLhndSRKw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXotFyksQ%253D%253D&md5=0f19577c9c572057afcb09414e02886e</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1016%2Fj.pharmthera.2020.107479&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.pharmthera.2020.107479%26sid%3Dliteratum%253Aachs%26aulast%3DBechter%26aufirst%3DO.%26aulast%3DSchoffski%26aufirst%3DP.%26atitle%3DMake%2520your%2520best%2520BET%253A%2520The%2520emerging%2520role%2520of%2520BET%2520inhibitor%2520treatment%2520in%2520malignant%2520tumors%26jtitle%3DPharmacol.%2520Ther.%26date%3D2020%26volume%3D208%26spage%3D107479%26doi%3D10.1016%2Fj.pharmthera.2020.107479" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bolden, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tasdemir, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dow, L. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Es, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilkinson, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clevers, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lowe, S. W.</span></span> <span> </span><span class="NLM_article-title">Inducible in vivo silencing of BRD4 identifies potential toxicities of sustained BET protein inhibition</span>. <i>Cell Rep.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">1919</span>– <span class="NLM_lpage">1929</span>, <span class="refDoi"> DOI: 10.1016/j.celrep.2014.08.025</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1016%2Fj.celrep.2014.08.025" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=25242322" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsFylur7E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2014&pages=1919-1929&author=J.+E.+Boldenauthor=N.+Tasdemirauthor=L.+E.+Dowauthor=J.+H.+van+Esauthor=J.+E.+Wilkinsonauthor=Z.+Zhaoauthor=H.+Cleversauthor=S.+W.+Lowe&title=Inducible+in+vivo+silencing+of+BRD4+identifies+potential+toxicities+of+sustained+BET+protein+inhibition&doi=10.1016%2Fj.celrep.2014.08.025"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Inducible In Vivo Silencing of Brd4 Identifies Potential Toxicities of Sustained BET Protein Inhibition</span></div><div class="casAuthors">Bolden, Jessica E.; Tasdemir, Nilgun; Dow, Lukas E.; van Es, Johan H.; Wilkinson, John E.; Zhao, Zhen; Clevers, Hans; Lowe, Scott W.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Reports</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1919-1929</span>CODEN:
                <span class="NLM_cas:coden">CREED8</span>;
        ISSN:<span class="NLM_cas:issn">2211-1247</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">BET family proteins are novel therapeutic targets for cancer and inflammation and represent the first chromatin readers against which small-mol. inhibitors have been developed.  First-generation BET inhibitors have shown therapeutic efficacy in preclin. models, but the consequences of sustained BET protein inhibition in normal tissues remain poorly characterized.  Using an inducible and reversible transgenic RNAi mouse model, we show that strong suppression of the BET protein Brd4 in adult animals has dramatic effects in multiple tissues.  Brd4-depleted mice display reversible epidermal hyperplasia, alopecia, and decreased cellular diversity and stem cell depletion in the small intestine.  Furthermore, Brd4-suppressed intestines are sensitive to organ stress and show impaired regeneration following irradn., suggesting that concurrent Brd4 suppression and certain cytotoxic therapies may induce undesirable synergistic effects.  These findings provide important insight into Brd4 function in normal tissues and, importantly, predict several potential outcomes assocd. with potent and sustained BET protein inhibition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqGFqmSiDt98LVg90H21EOLACvtfcHk0ljcAsLhndSRKw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsFylur7E&md5=9de00d3d76679ca7997cb245300a19af</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1016%2Fj.celrep.2014.08.025&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.celrep.2014.08.025%26sid%3Dliteratum%253Aachs%26aulast%3DBolden%26aufirst%3DJ.%2BE.%26aulast%3DTasdemir%26aufirst%3DN.%26aulast%3DDow%26aufirst%3DL.%2BE.%26aulast%3Dvan%2BEs%26aufirst%3DJ.%2BH.%26aulast%3DWilkinson%26aufirst%3DJ.%2BE.%26aulast%3DZhao%26aufirst%3DZ.%26aulast%3DClevers%26aufirst%3DH.%26aulast%3DLowe%26aufirst%3DS.%2BW.%26atitle%3DInducible%2520in%2520vivo%2520silencing%2520of%2520BRD4%2520identifies%2520potential%2520toxicities%2520of%2520sustained%2520BET%2520protein%2520inhibition%26jtitle%3DCell%2520Rep.%26date%3D2014%26volume%3D8%26spage%3D1919%26epage%3D1929%26doi%3D10.1016%2Fj.celrep.2014.08.025" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fong, C. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilan, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lam, E. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rubin, A. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ftouni, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tyler, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stanley, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sinha, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yeh, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morison, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giotopoulos, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lugo, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeffrey, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carpenter, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gregory, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramsay, R. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lane, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abdel-Wahab, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kouzarides, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnstone, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dawson, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huntly, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Papenfuss, A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dawson, M. A.</span></span> <span> </span><span class="NLM_article-title">BET inhibitor resistance emerges from leukaemia stem cells</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>525</i></span>,  <span class="NLM_fpage">538</span>– <span class="NLM_lpage">542</span>, <span class="refDoi"> DOI: 10.1038/nature14888</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1038%2Fnature14888" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=26367796" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsFeiu7fL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=525&publication_year=2015&pages=538-542&author=C.+Y.+Fongauthor=O.+Gilanauthor=E.+Y.+Lamauthor=A.+F.+Rubinauthor=S.+Ftouniauthor=D.+Tylerauthor=K.+Stanleyauthor=D.+Sinhaauthor=P.+Yehauthor=J.+Morisonauthor=G.+Giotopoulosauthor=D.+Lugoauthor=P.+Jeffreyauthor=S.+C.+Leeauthor=C.+Carpenterauthor=R.+Gregoryauthor=R.+G.+Ramsayauthor=S.+W.+Laneauthor=O.+Abdel-Wahabauthor=T.+Kouzaridesauthor=R.+W.+Johnstoneauthor=S.+J.+Dawsonauthor=B.+J.+Huntlyauthor=R.+K.+Prinjhaauthor=A.+T.+Papenfussauthor=M.+A.+Dawson&title=BET+inhibitor+resistance+emerges+from+leukaemia+stem+cells&doi=10.1038%2Fnature14888"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">BET inhibitor resistance emerges from leukaemia stem cells</span></div><div class="casAuthors">Fong, Chun Yew; Gilan, Omer; Lam, Enid Y. N.; Rubin, Alan F.; Ftouni, Sarah; Tyler, Dean; Stanley, Kym; Sinha, Devbarna; Yeh, Paul; Morison, Jessica; Giotopoulos, George; Lugo, Dave; Jeffrey, Philip; Lee, Stanley Chun-Wei; Carpenter, Christopher; Gregory, Richard; Ramsay, Robert G.; Lane, Steven W.; Abdel-Wahab, Omar; Kouzarides, Tony; Johnstone, Ricky W.; Dawson, Sarah-Jane; Huntly, Brian J. P.; Prinjha, Rab K.; Papenfuss, Anthony T.; Dawson, Mark A.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">525</span>
        (<span class="NLM_cas:issue">7570</span>),
    <span class="NLM_cas:pages">538-542</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Bromodomain and extra terminal protein (BET) inhibitors are first-in-class targeted therapies that deliver a new therapeutic opportunity by directly targeting bromodomain proteins that bind acetylated chromatin marks.  Early clin. trials have shown promise, esp. in acute myeloid leukemia, and therefore the evaluation of resistance mechanisms is crucial to optimize the clin. efficacy of these drugs.  Here we use primary mouse hematopoietic stem and progenitor cells immortalized with the fusion protein MLL-AF9 to generate several single-cell clones that demonstrate resistance, in vitro and in vivo, to the prototypical BET inhibitor, I-BET.  Resistance to I-BET confers cross-resistance to chem. distinct BET inhibitors such as JQ1, as well as resistance to genetic knockdown of BET proteins.  Resistance is not mediated through increased drug efflux or metab., but is shown to emerge from leukemia stem cells both ex vivo and in vivo.  Chromatin-bound BRD4 is globally reduced in resistant cells, whereas the expression of key target genes such as Myc remains unaltered, highlighting the existence of alternative mechanisms to regulate transcription.  We demonstrate that resistance to BET inhibitors, in human and mouse leukemia cells, is in part a consequence of increased Wnt/β-catenin signaling, and neg. regulation of this pathway results in restoration of sensitivity to I-BET in vitro and in vivo.  Together, these findings provide new insights into the biol. of acute myeloid leukemia, highlight potential therapeutic limitations of BET inhibitors, and identify strategies that may enhance the clin. utility of these unique targeted therapies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq7Lg1n64SCWrVg90H21EOLACvtfcHk0lgUQ7jlG_5jFw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsFeiu7fL&md5=47d1769e077b1c6c92ff52a051368404</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1038%2Fnature14888&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature14888%26sid%3Dliteratum%253Aachs%26aulast%3DFong%26aufirst%3DC.%2BY.%26aulast%3DGilan%26aufirst%3DO.%26aulast%3DLam%26aufirst%3DE.%2BY.%26aulast%3DRubin%26aufirst%3DA.%2BF.%26aulast%3DFtouni%26aufirst%3DS.%26aulast%3DTyler%26aufirst%3DD.%26aulast%3DStanley%26aufirst%3DK.%26aulast%3DSinha%26aufirst%3DD.%26aulast%3DYeh%26aufirst%3DP.%26aulast%3DMorison%26aufirst%3DJ.%26aulast%3DGiotopoulos%26aufirst%3DG.%26aulast%3DLugo%26aufirst%3DD.%26aulast%3DJeffrey%26aufirst%3DP.%26aulast%3DLee%26aufirst%3DS.%2BC.%26aulast%3DCarpenter%26aufirst%3DC.%26aulast%3DGregory%26aufirst%3DR.%26aulast%3DRamsay%26aufirst%3DR.%2BG.%26aulast%3DLane%26aufirst%3DS.%2BW.%26aulast%3DAbdel-Wahab%26aufirst%3DO.%26aulast%3DKouzarides%26aufirst%3DT.%26aulast%3DJohnstone%26aufirst%3DR.%2BW.%26aulast%3DDawson%26aufirst%3DS.%2BJ.%26aulast%3DHuntly%26aufirst%3DB.%2BJ.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DPapenfuss%26aufirst%3DA.%2BT.%26aulast%3DDawson%26aufirst%3DM.%2BA.%26atitle%3DBET%2520inhibitor%2520resistance%2520emerges%2520from%2520leukaemia%2520stem%2520cells%26jtitle%3DNature%26date%3D2015%26volume%3D525%26spage%3D538%26epage%3D542%26doi%3D10.1038%2Fnature14888" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rathert, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roth, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neumann, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muerdter, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roe, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muhar, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deswal, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cerny-Reiterer, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peter, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jude, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoffmann, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boryn, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Axelsson, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schweifer, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tontsch-Grunt, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dow, L. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gianni, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pearson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valent, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stark, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kraut, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vakoc, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zuber, J.</span></span> <span> </span><span class="NLM_article-title">Transcriptional plasticity promotes primary and acquired resistance to BET inhibition</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>525</i></span>,  <span class="NLM_fpage">543</span>– <span class="NLM_lpage">547</span>, <span class="refDoi"> DOI: 10.1038/nature14898</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1038%2Fnature14898" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=26367798" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsFeiu7fO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=525&publication_year=2015&pages=543-547&author=P.+Rathertauthor=M.+Rothauthor=T.+Neumannauthor=F.+Muerdterauthor=J.+S.+Roeauthor=M.+Muharauthor=S.+Deswalauthor=S.+Cerny-Reitererauthor=B.+Peterauthor=J.+Judeauthor=T.+Hoffmannauthor=L.+M.+Borynauthor=E.+Axelssonauthor=N.+Schweiferauthor=U.+Tontsch-Gruntauthor=L.+E.+Dowauthor=D.+Gianniauthor=M.+Pearsonauthor=P.+Valentauthor=A.+Starkauthor=N.+Krautauthor=C.+R.+Vakocauthor=J.+Zuber&title=Transcriptional+plasticity+promotes+primary+and+acquired+resistance+to+BET+inhibition&doi=10.1038%2Fnature14898"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Transcriptional plasticity promotes primary and acquired resistance to BET inhibition</span></div><div class="casAuthors">Rathert, Philipp; Roth, Mareike; Neumann, Tobias; Muerdter, Felix; Roe, Jae-Seok; Muhar, Matthias; Deswal, Sumit; Cerny-Reiterer, Sabine; Peter, Barbara; Jude, Julian; Hoffmann, Thomas; Boryn, Lukasz M.; Axelsson, Elin; Schweifer, Norbert; Tontsch-Grunt, Ulrike; Dow, Lukas E.; Gianni, Davide; Pearson, Mark; Valent, Peter; Stark, Alexander; Kraut, Norbert; Vakoc, Christopher R.; Zuber, Johannes</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">525</span>
        (<span class="NLM_cas:issue">7570</span>),
    <span class="NLM_cas:pages">543-547</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Following the discovery of BRD4 as a non-oncogene addiction target in acute myeloid leukemia (AML), bromodomain and extra terminal protein (BET) inhibitors are being explored as a promising therapeutic avenue in numerous cancers.  While clin. trials have reported single-agent activity in advanced hematol. malignancies, mechanisms detg. the response to BET inhibition remain poorly understood.  To identify factors involved in primary and acquired BET resistance in leukemia, here we perform a chromatin-focused RNAi screen in a sensitive MLL-AF9;NrasG12D-driven AML mouse model, and investigate dynamic transcriptional profiles in sensitive and resistant mouse and human leukemias.  Our screen shows that suppression of the PRC2 complex, contrary to effects in other contexts, promotes BET inhibitor resistance in AML.  PRC2 suppression does not directly affect the regulation of Brd4-dependent transcripts, but facilitates the remodelling of regulatory pathways that restore the transcription of key targets such as Myc.  Similarly, while BET inhibition triggers acute MYC repression in human leukemias regardless of their sensitivity, resistant leukemias are uniformly characterized by their ability to rapidly restore MYC transcription.  This process involves the activation and recruitment of WNT signaling components, which compensate for the loss of BRD4 and drive resistance in various cancer models.  Dynamic chromatin immunopptn. sequencing and self-transcribing active regulatory region sequencing of enhancer profiles reveal that BET-resistant states are characterized by remodelled regulatory landscapes, involving the activation of a focal MYC enhancer that recruits WNT machinery in response to BET inhibition.  Together, our results identify and validate WNT signaling as a driver and candidate biomarker of primary and acquired BET resistance in leukemia, and implicate the rewiring of transcriptional programs as an important mechanism promoting resistance to BET inhibitors and, potentially, other chromatin-targeted therapies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrDgeY3nFr4GrVg90H21EOLACvtfcHk0lgUQ7jlG_5jFw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsFeiu7fO&md5=6d6cf4c4123ad6a698b88f02f95a2ebe</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1038%2Fnature14898&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature14898%26sid%3Dliteratum%253Aachs%26aulast%3DRathert%26aufirst%3DP.%26aulast%3DRoth%26aufirst%3DM.%26aulast%3DNeumann%26aufirst%3DT.%26aulast%3DMuerdter%26aufirst%3DF.%26aulast%3DRoe%26aufirst%3DJ.%2BS.%26aulast%3DMuhar%26aufirst%3DM.%26aulast%3DDeswal%26aufirst%3DS.%26aulast%3DCerny-Reiterer%26aufirst%3DS.%26aulast%3DPeter%26aufirst%3DB.%26aulast%3DJude%26aufirst%3DJ.%26aulast%3DHoffmann%26aufirst%3DT.%26aulast%3DBoryn%26aufirst%3DL.%2BM.%26aulast%3DAxelsson%26aufirst%3DE.%26aulast%3DSchweifer%26aufirst%3DN.%26aulast%3DTontsch-Grunt%26aufirst%3DU.%26aulast%3DDow%26aufirst%3DL.%2BE.%26aulast%3DGianni%26aufirst%3DD.%26aulast%3DPearson%26aufirst%3DM.%26aulast%3DValent%26aufirst%3DP.%26aulast%3DStark%26aufirst%3DA.%26aulast%3DKraut%26aufirst%3DN.%26aulast%3DVakoc%26aufirst%3DC.%2BR.%26aulast%3DZuber%26aufirst%3DJ.%26atitle%3DTranscriptional%2520plasticity%2520promotes%2520primary%2520and%2520acquired%2520resistance%2520to%2520BET%2520inhibition%26jtitle%3DNature%26date%3D2015%26volume%3D525%26spage%3D543%26epage%3D547%26doi%3D10.1038%2Fnature14898" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kumar, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raza, S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knab, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chow, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kwok, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bentrem, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Popovic, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ebine, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Licht, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Munshi, H. G.</span></span> <span> </span><span class="NLM_article-title">GLI2-dependent c-MYC upregulation mediates resistance of pancreatic cancer cells to the BET bromodomain inhibitor JQ1</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">9489</span>, <span class="refDoi"> DOI: 10.1038/srep09489</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1038%2Fsrep09489" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=25807524" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtFOjurjO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2015&pages=9489&author=K.+Kumarauthor=S.+S.+Razaauthor=L.+M.+Knabauthor=C.+R.+Chowauthor=B.+Kwokauthor=D.+J.+Bentremauthor=R.+Popovicauthor=K.+Ebineauthor=J.+D.+Lichtauthor=H.+G.+Munshi&title=GLI2-dependent+c-MYC+upregulation+mediates+resistance+of+pancreatic+cancer+cells+to+the+BET+bromodomain+inhibitor+JQ1&doi=10.1038%2Fsrep09489"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">GLI2-dependent c-MYC upregulation mediates resistance of pancreatic cancer cells to the BET bromodomain inhibitor JQ1</span></div><div class="casAuthors">Kumar, Krishan; Raza, Sania S.; Knab, Lawrence M.; Chow, Christina R.; Kwok, Benjamin; Bentrem, David J.; Popovic, Relja; Ebine, Kazumi; Licht, Jonathan D.; Munshi, Hidayatullah G.</div><div class="citationInfo"><span class="NLM_cas:title">Scientific Reports</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">9489</span>CODEN:
                <span class="NLM_cas:coden">SRCEC3</span>;
        ISSN:<span class="NLM_cas:issn">2045-2322</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">JQ1 and I-BET151 are selective inhibitors of BET bromodomain proteins that have efficacy against a no. of different cancers.  Since the effectiveness of targeted therapies is often limited by development of resistance, we examd. whether it was possible for cancer cells to develop resistance to the BET inhibitor JQ1.  Here we show that pancreatic cancer cells developing resistance to JQ1 demonstrate cross-resistance to I-BET151 and insensitivity to BRD4 downregulation.  The resistant cells maintain expression of c-MYC, increase expression of JQ1-target genes FOSL1 and HMGA2, and demonstrate evidence of epithelial-mesenchymal transition (EMT).  However, reverting EMT fails to sensitize the resistant cells to JQ1 treatment.  Importantly, the JQ1-resistant cells remain dependent on c-MYC that now becomes co-regulated by high levels of GLI2.  Furthermore, downregulating GLI2 re-sensitizes the resistant cells to JQ1.  Overall, these results identify a mechanism by which cancer cells develop resistance to BET inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp6mUgUS37czbVg90H21EOLACvtfcHk0lgUQ7jlG_5jFw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtFOjurjO&md5=c5f9dad7134441f6c7edbd6dcc7b40e8</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1038%2Fsrep09489&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsrep09489%26sid%3Dliteratum%253Aachs%26aulast%3DKumar%26aufirst%3DK.%26aulast%3DRaza%26aufirst%3DS.%2BS.%26aulast%3DKnab%26aufirst%3DL.%2BM.%26aulast%3DChow%26aufirst%3DC.%2BR.%26aulast%3DKwok%26aufirst%3DB.%26aulast%3DBentrem%26aufirst%3DD.%2BJ.%26aulast%3DPopovic%26aufirst%3DR.%26aulast%3DEbine%26aufirst%3DK.%26aulast%3DLicht%26aufirst%3DJ.%2BD.%26aulast%3DMunshi%26aufirst%3DH.%2BG.%26atitle%3DGLI2-dependent%2520c-MYC%2520upregulation%2520mediates%2520resistance%2520of%2520pancreatic%2520cancer%2520cells%2520to%2520the%2520BET%2520bromodomain%2520inhibitor%2520JQ1%26jtitle%3DSci.%2520Rep.%26date%3D2015%26volume%3D5%26spage%3D9489%26doi%3D10.1038%2Fsrep09489" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Settleman, J.</span></span> <span> </span><span class="NLM_article-title">Cancer: BET on drug resistance</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>529</i></span>,  <span class="NLM_fpage">289</span>– <span class="NLM_lpage">290</span>, <span class="refDoi"> DOI: 10.1038/nature16863</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1038%2Fnature16863" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=26735017" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BC28Xns1GlsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=529&publication_year=2016&pages=289-290&author=J.+Settleman&title=Cancer%3A+BET+on+drug+resistance&doi=10.1038%2Fnature16863"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Cancer Bet on drug resistance</span></div><div class="casAuthors">Settleman, Jeff</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">529</span>
        (<span class="NLM_cas:issue">7586</span>),
    <span class="NLM_cas:pages">289-290</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Inhibitors of the BET bromodomain proteins are promising cancer therapeutics, but tumor cells are likely to become resistant to these drugs.  Anticipated mechanisms of resistance have now been described.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq3IC8VaowJQLVg90H21EOLACvtfcHk0lglwY2T3p5BMg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xns1GlsA%253D%253D&md5=9ce8b053ecf769ab9117983eb80b749f</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1038%2Fnature16863&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature16863%26sid%3Dliteratum%253Aachs%26aulast%3DSettleman%26aufirst%3DJ.%26atitle%3DCancer%253A%2520BET%2520on%2520drug%2520resistance%26jtitle%3DNature%26date%3D2016%26volume%3D529%26spage%3D289%26epage%3D290%26doi%3D10.1038%2Fnature16863" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jang, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eom, J. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeung, H. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheong, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Min, Y. H.</span></span> <span> </span><span class="NLM_article-title">AMPK-ULK1-mediated autophagy confers resistance to BET Inhibitor JQ1 in acute myeloid leukemia stem cells</span>. <i>Clin. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">2781</span>– <span class="NLM_lpage">2794</span>, <span class="refDoi"> DOI: 10.1158/1078-0432.CCR-16-1903</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1158%2F1078-0432.CCR-16-1903" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=27864418" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BC2sXpt1Sms7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2017&pages=2781-2794&author=J.+E.+Jangauthor=J.+I.+Eomauthor=H.+K.+Jeungauthor=J.+W.+Cheongauthor=J.+Y.+Leeauthor=J.+S.+Kimauthor=Y.+H.+Min&title=AMPK-ULK1-mediated+autophagy+confers+resistance+to+BET+Inhibitor+JQ1+in+acute+myeloid+leukemia+stem+cells&doi=10.1158%2F1078-0432.CCR-16-1903"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">AMPK-ULK1-Mediated Autophagy Confers Resistance to BET Inhibitor JQ1 in Acute Myeloid Leukemia Stem Cells</span></div><div class="casAuthors">Jang, Ji Eun; Eom, Ju-In; Jeung, Hoi-Kyung; Cheong, June-Won; Lee, Jung Yeon; Kim, Jin Seok; Min, Yoo Hong</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2781-2794</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Purpose: Bromodomain and extraterminal domain (BET) inhibitors are promising epigenetic agents for the treatment of various subsets of acute myeloid leukemia (AML).  However, the resistance of leukemia stem cells (LSC) to BET inhibitors remains a major challenge.  In this study, we evaluated the mechanisms underlying LSC resistance to the BET inhibitor JQ1.  Exptl. Design: We evaluated the levels of apoptosis and autophagy induced by JQ1 in LSC-like leukemia cell lines and primary CD34+CD38- leukemic blasts obtained from AML cases with normal karyotype without recurrent mutations.  Results: JQ1 effectively induced apoptosis in a concn.-dependent manner in JQ1-sensitive AML cells.  However, in JQ1-resistant AML LSCs, JQ1 induced little apoptosis and led to upregulation of beclin-1, increased LC3-II lipidation, formation of autophagosomes, and downregulation of p62/SQSTM1.  Inhibition of autophagy by pharmacol. inhibitors or knockdown of beclin-1 using specific siRNA enhanced JQ1-induced apoptosis in resistant cells, indicating that prosurvival autophagy occurred in these cells.  Independent of mTOR signaling, activation of the AMPK (pThr172)/ULK1 (pSer555) pathway was found to be assocd. with JQ1-induced autophagy in resistant cells.  AMPK inhibition using the pharmacol. inhibitor compd. C or by knockdown of AMPKα suppressed autophagy and promoted JQ1-induced apoptosis in AML LSCs.  Conclusions: These findings revealed that prosurvival autophagy was one of the mechanisms involved in the resistance AML LSCs to JQ1.  Targeting the AMPK/ULK1 pathway or inhibition of autophagy could be an effective therapeutic strategy for combating resistance to BET inhibitors in AML and other types of cancer.  Clin Cancer Res; 23(11); 2781-94. 2016 AACR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqHOzsCo6fOvLVg90H21EOLACvtfcHk0lglwY2T3p5BMg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXpt1Sms7o%253D&md5=f15df13e0addda969161b31973857176</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1158%2F1078-0432.CCR-16-1903&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1078-0432.CCR-16-1903%26sid%3Dliteratum%253Aachs%26aulast%3DJang%26aufirst%3DJ.%2BE.%26aulast%3DEom%26aufirst%3DJ.%2BI.%26aulast%3DJeung%26aufirst%3DH.%2BK.%26aulast%3DCheong%26aufirst%3DJ.%2BW.%26aulast%3DLee%26aufirst%3DJ.%2BY.%26aulast%3DKim%26aufirst%3DJ.%2BS.%26aulast%3DMin%26aufirst%3DY.%2BH.%26atitle%3DAMPK-ULK1-mediated%2520autophagy%2520confers%2520resistance%2520to%2520BET%2520Inhibitor%2520JQ1%2520in%2520acute%2520myeloid%2520leukemia%2520stem%2520cells%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D2017%26volume%3D23%26spage%3D2781%26epage%3D2794%26doi%3D10.1158%2F1078-0432.CCR-16-1903" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ren, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ye, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pan, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, H.</span></span> <span> </span><span class="NLM_article-title">Intrinsic BET inhibitor resistance in SPOP-mutated prostate cancer is mediated by BET protein stabilization and AKT-mTORC1 activation</span>. <i>Nat. Med.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">1055</span>– <span class="NLM_lpage">1062</span>, <span class="refDoi"> DOI: 10.1038/nm.4379</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1038%2Fnm.4379" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=28805822" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtlKjtbbO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2017&pages=1055-1062&author=P.+Zhangauthor=D.+Wangauthor=Y.+Zhaoauthor=S.+Renauthor=K.+Gaoauthor=Z.+Yeauthor=S.+Wangauthor=C.+W.+Panauthor=Y.+Zhuauthor=Y.+Yanauthor=Y.+Yangauthor=D.+Wuauthor=Y.+Heauthor=J.+Zhangauthor=D.+Luauthor=X.+Liuauthor=L.+Yuauthor=S.+Zhaoauthor=Y.+Liauthor=D.+Linauthor=Y.+Wangauthor=L.+Wangauthor=Y.+Chenauthor=Y.+Sunauthor=C.+Wangauthor=H.+Huang&title=Intrinsic+BET+inhibitor+resistance+in+SPOP-mutated+prostate+cancer+is+mediated+by+BET+protein+stabilization+and+AKT-mTORC1+activation&doi=10.1038%2Fnm.4379"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Intrinsic BET inhibitor resistance in SPOP-mutated prostate cancer is mediated by BET protein stabilization and AKT-mTORC1 activation</span></div><div class="casAuthors">Zhang, Pingzhao; Wang, Dejie; Zhao, Yu; Ren, Shancheng; Gao, Kun; Ye, Zhenqing; Wang, Shangqian; Pan, Chun-Wu; Zhu, Yasheng; Yan, Yuqian; Yang, Yinhui; Wu, Di; He, Yundong; Zhang, Jun; Lu, Daru; Liu, Xiuping; Yu, Long; Zhao, Shimin; Li, Yao; Lin, Dong; Wang, Yuzhuo; Wang, Liguo; Chen, Yu; Sun, Yinghao; Wang, Chenji; Huang, Haojie</div><div class="citationInfo"><span class="NLM_cas:title">Nature Medicine (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1055-1062</span>CODEN:
                <span class="NLM_cas:coden">NAMEFI</span>;
        ISSN:<span class="NLM_cas:issn">1078-8956</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Bromodomain and extraterminal domain (BET) protein inhibitors are emerging as promising anticancer therapies.  The gene encoding the E3 ubiquitin ligase substrate-binding adaptor speckle-type POZ protein (SPOP) is the most frequently mutated in primary prostate cancer.  Here we demonstrate that wild-type SPOP binds to and induces ubiquitination and proteasomal degrdn. of BET proteins (BRD2, BRD3 and BRD4) by recognizing a degron motif common among them.  In contrast, prostate cancer-assocd. SPOP mutants show impaired binding to BET proteins, resulting in decreased proteasomal degrdn. and accumulation of these proteins in prostate cancer cell lines and patient specimens and causing resistance to BET inhibitors.  Transcriptome and BRD4 cistrome analyses reveal enhanced expression of the GTPase RAC1 and cholesterol-biosynthesis-assocd. genes together with activation of AKT-mTORC1 signaling as a consequence of BRD4 stabilization.  Our data show that resistance to BET inhibitors in SPOP-mutant prostate cancer can be overcome by combination with AKT inhibitors and further support the evaluation of SPOP mutations as biomarkers to guide BET-inhibitor-oriented therapy in patients with prostate cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoJ-PBm8f3OarVg90H21EOLACvtfcHk0lglwY2T3p5BMg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtlKjtbbO&md5=01e4c46e7e5334055a81205c5dac9057</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1038%2Fnm.4379&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnm.4379%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DP.%26aulast%3DWang%26aufirst%3DD.%26aulast%3DZhao%26aufirst%3DY.%26aulast%3DRen%26aufirst%3DS.%26aulast%3DGao%26aufirst%3DK.%26aulast%3DYe%26aufirst%3DZ.%26aulast%3DWang%26aufirst%3DS.%26aulast%3DPan%26aufirst%3DC.%2BW.%26aulast%3DZhu%26aufirst%3DY.%26aulast%3DYan%26aufirst%3DY.%26aulast%3DYang%26aufirst%3DY.%26aulast%3DWu%26aufirst%3DD.%26aulast%3DHe%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DLu%26aufirst%3DD.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DYu%26aufirst%3DL.%26aulast%3DZhao%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DLin%26aufirst%3DD.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DSun%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DC.%26aulast%3DHuang%26aufirst%3DH.%26atitle%3DIntrinsic%2520BET%2520inhibitor%2520resistance%2520in%2520SPOP-mutated%2520prostate%2520cancer%2520is%2520mediated%2520by%2520BET%2520protein%2520stabilization%2520and%2520AKT-mTORC1%2520activation%26jtitle%3DNat.%2520Med.%26date%3D2017%26volume%3D23%26spage%3D1055%26epage%3D1062%26doi%3D10.1038%2Fnm.4379" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref54"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref54'); return false;" data-citation="" class="refNumLink">54</a></strong><div class="NLM_citation" id="cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dai, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gan, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhong, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shimizu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beca, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blattner, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vasudevan, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buckley, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buser, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Inuzuka, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beck, A. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wild, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garraway, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rubin, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barbieri, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, K. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muthuswamy, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, W.</span></span> <span> </span><span class="NLM_article-title">Prostate cancer-associated SPOP mutations confer resistance to BET inhibitors through stabilization of BRD4</span>. <i>Nat. Med.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">1063</span>– <span class="NLM_lpage">1071</span>, <span class="refDoi"> DOI: 10.1038/nm.4378</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1038%2Fnm.4378" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=28805820" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtlKjtbnE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2017&pages=1063-1071&author=X.+Daiauthor=W.+Ganauthor=X.+Liauthor=S.+Wangauthor=W.+Zhangauthor=L.+Huangauthor=S.+Liuauthor=Q.+Zhongauthor=J.+Guoauthor=J.+Zhangauthor=T.+Chenauthor=K.+Shimizuauthor=F.+Becaauthor=M.+Blattnerauthor=D.+Vasudevanauthor=D.+L.+Buckleyauthor=J.+Qiauthor=L.+Buserauthor=P.+Liuauthor=H.+Inuzukaauthor=A.+H.+Beckauthor=L.+Wangauthor=P.+J.+Wildauthor=L.+A.+Garrawayauthor=M.+A.+Rubinauthor=C.+E.+Barbieriauthor=K.+K.+Wongauthor=S.+K.+Muthuswamyauthor=J.+Huangauthor=Y.+Chenauthor=J.+E.+Bradnerauthor=W.+Wei&title=Prostate+cancer-associated+SPOP+mutations+confer+resistance+to+BET+inhibitors+through+stabilization+of+BRD4&doi=10.1038%2Fnm.4378"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Prostate cancer-associated SPOP mutations confer resistance to BET inhibitors through stabilization of BRD4</span></div><div class="casAuthors">Dai, Xiangpeng; Gan, Wenjian; Li, Xiaoning; Wang, Shangqian; Zhang, Wei; Huang, Ling; Liu, Shengwu; Zhong, Qing; Guo, Jianping; Zhang, Jinfang; Chen, Ting; Shimizu, Kouhei; Beca, Francisco; Blattner, Mirjam; Vasudevan, Divya; Buckley, Dennis L.; Qi, Jun; Buser, Lorenz; Liu, Pengda; Inuzuka, Hiroyuki; Beck, Andrew H.; Wang, Liewei; Wild, Peter J.; Garraway, Levi A.; Rubin, Mark A.; Barbieri, Christopher E.; Wong, Kwok-Kin; Muthuswamy, Senthil K.; Huang, Jiaoti; Chen, Yu; Bradner, James E.; Wei, Wenyi</div><div class="citationInfo"><span class="NLM_cas:title">Nature Medicine (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1063-1071</span>CODEN:
                <span class="NLM_cas:coden">NAMEFI</span>;
        ISSN:<span class="NLM_cas:issn">1078-8956</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">The bromodomain and extraterminal (BET) family of proteins comprises four members-BRD2, BRD3, BRD4 and the testis-specific isoform BRDT-that largely function as transcriptional coactivators and play crit. roles in various cellular processes, including the cell cycle, apoptosis, migration and invasion.  BET proteins enhance the oncogenic functions of major cancer drivers by elevating the expression of these drivers, such as c-Myc in leukemia, or by promoting the transcriptional activities of oncogenic factors, such as AR and ERG in prostate cancer.  Pathol., BET proteins are frequently overexpressed and are clin. linked to various types of human cancer; they are therefore being pursued as attractive therapeutic targets for selective inhibition in patients with cancer.  To this end, a no. of bromodomain inhibitors, including JQ1 and I-BET, have been developed and have shown promising outcomes in early clin. trials.  Although resistance to BET inhibitors has been documented in preclin. models, the mol. mechanisms underlying acquired resistance are largely unknown.  Here we report that cullin-3SPOP earmarks BET proteins, including BRD2, BRD3 and BRD4, for ubiquitination-mediated degrdn.  Pathol., prostate cancer-assocd. SPOP mutants fail to interact with and promote the degrdn. of BET proteins, leading to their elevated abundance in SPOP-mutant prostate cancer.  As a result, prostate cancer cell lines and organoids derived from individuals harboring SPOP mutations are more resistant to BET-inhibitor-induced cell growth arrest and apoptosis.  Therefore, our results elucidate the tumor-suppressor role of SPOP in prostate cancer in which it acts as a neg. regulator of BET protein stability and also provide a mol. mechanism for resistance to BET inhibitors in individuals with prostate cancer bearing SPOP mutations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqDt_5oFy4JlLVg90H21EOLACvtfcHk0lhYpn3ZFEpQWg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtlKjtbnE&md5=dbbb73a496d18b8310d675cd2503853f</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1038%2Fnm.4378&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnm.4378%26sid%3Dliteratum%253Aachs%26aulast%3DDai%26aufirst%3DX.%26aulast%3DGan%26aufirst%3DW.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DS.%26aulast%3DZhang%26aufirst%3DW.%26aulast%3DHuang%26aufirst%3DL.%26aulast%3DLiu%26aufirst%3DS.%26aulast%3DZhong%26aufirst%3DQ.%26aulast%3DGuo%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DT.%26aulast%3DShimizu%26aufirst%3DK.%26aulast%3DBeca%26aufirst%3DF.%26aulast%3DBlattner%26aufirst%3DM.%26aulast%3DVasudevan%26aufirst%3DD.%26aulast%3DBuckley%26aufirst%3DD.%2BL.%26aulast%3DQi%26aufirst%3DJ.%26aulast%3DBuser%26aufirst%3DL.%26aulast%3DLiu%26aufirst%3DP.%26aulast%3DInuzuka%26aufirst%3DH.%26aulast%3DBeck%26aufirst%3DA.%2BH.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DWild%26aufirst%3DP.%2BJ.%26aulast%3DGarraway%26aufirst%3DL.%2BA.%26aulast%3DRubin%26aufirst%3DM.%2BA.%26aulast%3DBarbieri%26aufirst%3DC.%2BE.%26aulast%3DWong%26aufirst%3DK.%2BK.%26aulast%3DMuthuswamy%26aufirst%3DS.%2BK.%26aulast%3DHuang%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26aulast%3DWei%26aufirst%3DW.%26atitle%3DProstate%2520cancer-associated%2520SPOP%2520mutations%2520confer%2520resistance%2520to%2520BET%2520inhibitors%2520through%2520stabilization%2520of%2520BRD4%26jtitle%3DNat.%2520Med.%26date%3D2017%26volume%3D23%26spage%3D1063%26epage%3D1071%26doi%3D10.1038%2Fnm.4378" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref55"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref55'); return false;" data-citation="" class="refNumLink">55</a></strong><div class="NLM_citation" id="cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Janouskova, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">El Tekle, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bellini, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Udeshi, N. D</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rinaldi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ulbricht, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernasocchi, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Civenni, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Losa, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Svinkina, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bielski, C. M</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kryukov, G. V</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cascione, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Napoli, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Enchev, R. I</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mutch, D. G</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carney, M. E</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berchuck, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winterhoff, B. J N</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Broaddus, R. R</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schraml, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moch, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bertoni, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Catapano, C. V</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peter, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carr, S. A</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garraway, L. A</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wild, P. J</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Theurillat, J.-P. P</span></span> <span> </span><span class="NLM_article-title">Opposing effects of cancer-type-specific SPOP mutants on BET protein degradation and sensitivity to BET inhibitors</span>. <i>Nat. Med.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">1046</span>– <span class="NLM_lpage">1054</span>, <span class="refDoi"> DOI: 10.1038/nm.4372</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1038%2Fnm.4372" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=28805821" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtlKjtbbN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2017&pages=1046-1054&author=H.+Janouskovaauthor=G.+El+Tekleauthor=E.+Belliniauthor=N.+D+Udeshiauthor=A.+Rinaldiauthor=A.+Ulbrichtauthor=T.+Bernasocchiauthor=G.+Civenniauthor=M.+Losaauthor=T.+Svinkinaauthor=C.+M+Bielskiauthor=G.+V+Kryukovauthor=L.+Cascioneauthor=S.+Napoliauthor=R.+I+Enchevauthor=D.+G+Mutchauthor=M.+E+Carneyauthor=A.+Berchuckauthor=B.+J+N+Winterhoffauthor=R.+R+Broaddusauthor=P.+Schramlauthor=H.+Mochauthor=F.+Bertoniauthor=C.+V+Catapanoauthor=M.+Peterauthor=S.+A+Carrauthor=L.+A+Garrawayauthor=P.+J+Wildauthor=J.-P.+P+Theurillat&title=Opposing+effects+of+cancer-type-specific+SPOP+mutants+on+BET+protein+degradation+and+sensitivity+to+BET+inhibitors&doi=10.1038%2Fnm.4372"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Opposing effects of cancer-type-specific SPOP mutants on BET protein degradation and sensitivity to BET inhibitors</span></div><div class="casAuthors">Janouskova, Hana; El Tekle, Geniver; Bellini, Elisa; Udeshi, Namrata D.; Rinaldi, Anna; Ulbricht, Anna; Bernasocchi, Tiziano; Civenni, Gianluca; Losa, Marco; Svinkina, Tanya; Bielski, Craig M.; Kryukov, Gregory V.; Cascione, Luciano; Napoli, Sara; Enchev, Radoslav I.; Mutch, David G.; Carney, Michael E.; Berchuck, Andrew; Winterhoff, Boris J. N.; Broaddus, Russell R.; Schraml, Peter; Moch, Holger; Bertoni, Francesco; Catapano, Carlo V.; Peter, Matthias; Carr, Steven A.; Garraway, Levi A.; Wild, Peter J.; Theurillat, Jean-Philippe P.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Medicine (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1046-1054</span>CODEN:
                <span class="NLM_cas:coden">NAMEFI</span>;
        ISSN:<span class="NLM_cas:issn">1078-8956</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">It is generally assumed that recurrent mutations within a given cancer driver gene elicit similar drug responses.  Cancer genome studies have identified recurrent but divergent missense mutations affecting the substrate-recognition domain of the ubiquitin ligase adaptor SPOP in endometrial and prostate cancers.  The therapeutic implications of these mutations remain incompletely understood.  Here we analyzed changes in the ubiquitin landscape induced by endometrial cancer-assocd. SPOP mutations and identified BRD2, BRD3 and BRD4 proteins (BETs) as SPOP-CUL3 substrates that are preferentially degraded by endometrial cancer-assocd. SPOP mutants.  The resulting redn. of BET protein levels sensitized cancer cells to BET inhibitors.  Conversely, prostate cancer-specific SPOP mutations resulted in impaired degrdn. of BETs, promoting their resistance to pharmacol. inhibition.  These results uncover an oncogenomics paradox, whereby mutations mapping to the same domain evoke opposing drug susceptibilities.  Specifically, we provide a mol. rationale for the use of BET inhibitors to treat patients with endometrial but not prostate cancer who harbor SPOP mutations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq_yEfPkA6U9rVg90H21EOLACvtfcHk0lhYpn3ZFEpQWg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtlKjtbbN&md5=cf43e77dd3d6ed7c3a2fb4efca9167cb</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1038%2Fnm.4372&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnm.4372%26sid%3Dliteratum%253Aachs%26aulast%3DJanouskova%26aufirst%3DH.%26aulast%3DEl%2BTekle%26aufirst%3DG.%26aulast%3DBellini%26aufirst%3DE.%26aulast%3DUdeshi%26aufirst%3DN.%2BD%26aulast%3DRinaldi%26aufirst%3DA.%26aulast%3DUlbricht%26aufirst%3DA.%26aulast%3DBernasocchi%26aufirst%3DT.%26aulast%3DCivenni%26aufirst%3DG.%26aulast%3DLosa%26aufirst%3DM.%26aulast%3DSvinkina%26aufirst%3DT.%26aulast%3DBielski%26aufirst%3DC.%2BM%26aulast%3DKryukov%26aufirst%3DG.%2BV%26aulast%3DCascione%26aufirst%3DL.%26aulast%3DNapoli%26aufirst%3DS.%26aulast%3DEnchev%26aufirst%3DR.%2BI%26aulast%3DMutch%26aufirst%3DD.%2BG%26aulast%3DCarney%26aufirst%3DM.%2BE%26aulast%3DBerchuck%26aufirst%3DA.%26aulast%3DWinterhoff%26aufirst%3DB.%2BJ%2BN%26aulast%3DBroaddus%26aufirst%3DR.%2BR%26aulast%3DSchraml%26aufirst%3DP.%26aulast%3DMoch%26aufirst%3DH.%26aulast%3DBertoni%26aufirst%3DF.%26aulast%3DCatapano%26aufirst%3DC.%2BV%26aulast%3DPeter%26aufirst%3DM.%26aulast%3DCarr%26aufirst%3DS.%2BA%26aulast%3DGarraway%26aufirst%3DL.%2BA%26aulast%3DWild%26aufirst%3DP.%2BJ%26aulast%3DTheurillat%26aufirst%3DJ.-P.%2BP%26atitle%3DOpposing%2520effects%2520of%2520cancer-type-specific%2520SPOP%2520mutants%2520on%2520BET%2520protein%2520degradation%2520and%2520sensitivity%2520to%2520BET%2520inhibitors%26jtitle%3DNat.%2520Med.%26date%3D2017%26volume%3D23%26spage%3D1046%26epage%3D1054%26doi%3D10.1038%2Fnm.4372" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref56"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref56'); return false;" data-citation="" class="refNumLink">56</a></strong><div class="NLM_citation" id="cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jin, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ye, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jimenez, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, H.</span></span> <span> </span><span class="NLM_article-title">DUB3 promotes BET inhibitor resistance and cancer progression by deubiquitinating BRD4</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>71</i></span>,  <span class="NLM_fpage">592</span>– <span class="NLM_lpage">605</span>, <span class="refDoi"> DOI: 10.1016/j.molcel.2018.06.036</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1016%2Fj.molcel.2018.06.036" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=30057199" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhsVSmsb%252FO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=71&publication_year=2018&pages=592-605&author=X.+Jinauthor=Y.+Yanauthor=D.+Wangauthor=D.+Dingauthor=T.+Maauthor=Z.+Yeauthor=R.+Jimenezauthor=L.+Wangauthor=H.+Wuauthor=H.+Huang&title=DUB3+promotes+BET+inhibitor+resistance+and+cancer+progression+by+deubiquitinating+BRD4&doi=10.1016%2Fj.molcel.2018.06.036"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">DUB3 Promotes BET Inhibitor Resistance and Cancer Progression by Deubiquitinating BRD4</span></div><div class="casAuthors">Jin, Xin; Yan, Yuqian; Wang, Dejie; Ding, Donglin; Ma, Tao; Ye, Zhenqing; Jimenez, Rafael; Wang, Liguo; Wu, Heshui; Huang, Haojie</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">71</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">592-605.e4</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">The bromodomain and extra-terminal domain (BET) protein BRD4 is emerging as a promising anticancer therapeutic target.  However, resistance to BET inhibitors often occurs, and it has been linked to aberrant degrdn. of BRD4 protein in cancer.  Here, we demonstrate that the deubiquitinase DUB3 binds to BRD4 and promotes its deubiquitination and stabilization.  Expression of DUB3 is transcriptionally repressed by the NCOR2-HDAC10 complex.  The NCOR2 gene is frequently deleted in castration-resistant prostate cancer patient specimens, and loss of NCOR2 induces elevation of DUB3 and BRD4 proteins in cancer cells.  DUB3-proficient prostate cancer cells are resistant to the BET inhibitor JQ1 in vitro and in mice, but this effect is diminished by DUB3 inhibitory agents such as CDK4/6 inhibitor in a RB-independent manner.  Our findings identify a previously unrecognized mechanism causing BRD4 upregulation and drug resistance, suggesting that DUB3 is a viable therapeutic target to overcome BET inhibitor resistance in cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGozU4LZMIj7-LVg90H21EOLACvtfcHk0lhi_x5rg0y1CQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhsVSmsb%252FO&md5=45ae817ec1b9ac71dc2402b4618a1794</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2018.06.036&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2018.06.036%26sid%3Dliteratum%253Aachs%26aulast%3DJin%26aufirst%3DX.%26aulast%3DYan%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DD.%26aulast%3DDing%26aufirst%3DD.%26aulast%3DMa%26aufirst%3DT.%26aulast%3DYe%26aufirst%3DZ.%26aulast%3DJimenez%26aufirst%3DR.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DWu%26aufirst%3DH.%26aulast%3DHuang%26aufirst%3DH.%26atitle%3DDUB3%2520promotes%2520BET%2520inhibitor%2520resistance%2520and%2520cancer%2520progression%2520by%2520deubiquitinating%2520BRD4%26jtitle%3DMol.%2520Cell%26date%3D2018%26volume%3D71%26spage%3D592%26epage%3D605%26doi%3D10.1016%2Fj.molcel.2018.06.036" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref57"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref57'); return false;" data-citation="" class="refNumLink">57</a></strong><div class="NLM_citation" id="cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Marcotte, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sayad, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanchez-Garcia, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reimand, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haider, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Virtanen, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bader, G. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mills, G. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pe’Er, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moffat, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neel, B. G.</span></span> <span> </span><span class="NLM_article-title">Functional genomic landscape of human breast cancer drivers, vulnerabilities, and resistance</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>164</i></span>,  <span class="NLM_fpage">293</span>– <span class="NLM_lpage">309</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2015.11.062</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1016%2Fj.cell.2015.11.062" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=26771497" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtFShtr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=164&publication_year=2016&pages=293-309&author=R.+Marcotteauthor=A.+Sayadauthor=K.+R.+Brownauthor=F.+Sanchez-Garciaauthor=J.+Reimandauthor=M.+Haiderauthor=C.+Virtanenauthor=J.+E.+Bradnerauthor=G.+D.+Baderauthor=G.+B.+Millsauthor=D.+Pe%E2%80%99Erauthor=J.+Moffatauthor=B.+G.+Neel&title=Functional+genomic+landscape+of+human+breast+cancer+drivers%2C+vulnerabilities%2C+and+resistance&doi=10.1016%2Fj.cell.2015.11.062"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">Functional Genomic Landscape of Human Breast Cancer Drivers, Vulnerabilities, and Resistance</span></div><div class="casAuthors">Marcotte, Richard; Sayad, Azin; Brown, Kevin R.; Sanchez-Garcia, Felix; Reimand, Juri; Haider, Maliha; Virtanen, Carl; Bradner, James E.; Bader, Gary D.; Mills, Gordon B.; Pe'er, Dana; Moffat, Jason; Neel, Benjamin G.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">164</span>
        (<span class="NLM_cas:issue">1-2</span>),
    <span class="NLM_cas:pages">293-309</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Large-scale genomic studies have identified multiple somatic aberrations in breast cancer, including copy no. alterations and point mutations.  Still, identifying causal variants and emergent vulnerabilities that arise as a consequence of genetic alterations remain major challenges.  We performed whole-genome small hairpin RNA (shRNA) "dropout screens" on 77 breast cancer cell lines.  Using a hierarchical linear regression algorithm to score our screen results and integrate them with accompanying detailed genetic and proteomic information, we identify vulnerabilities in breast cancer, including candidate "drivers," and reveal general functional genomic properties of cancer cells.  Comparisons of gene essentiality with drug sensitivity data suggest potential resistance mechanisms, effects of existing anti-cancer drugs, and opportunities for combination therapy.  Finally, we demonstrate the utility of this large dataset by identifying BRD4 as a potential target in luminal breast cancer and PIK3CA mutations as a resistance determinant for BET-inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrc0920t-4Go7Vg90H21EOLACvtfcHk0lhi_x5rg0y1CQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtFShtr0%253D&md5=ebb0a807e744c8fefea76bb4f100fbca</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2015.11.062&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2015.11.062%26sid%3Dliteratum%253Aachs%26aulast%3DMarcotte%26aufirst%3DR.%26aulast%3DSayad%26aufirst%3DA.%26aulast%3DBrown%26aufirst%3DK.%2BR.%26aulast%3DSanchez-Garcia%26aufirst%3DF.%26aulast%3DReimand%26aufirst%3DJ.%26aulast%3DHaider%26aufirst%3DM.%26aulast%3DVirtanen%26aufirst%3DC.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26aulast%3DBader%26aufirst%3DG.%2BD.%26aulast%3DMills%26aufirst%3DG.%2BB.%26aulast%3DPe%25E2%2580%2599Er%26aufirst%3DD.%26aulast%3DMoffat%26aufirst%3DJ.%26aulast%3DNeel%26aufirst%3DB.%2BG.%26atitle%3DFunctional%2520genomic%2520landscape%2520of%2520human%2520breast%2520cancer%2520drivers%252C%2520vulnerabilities%252C%2520and%2520resistance%26jtitle%3DCell%26date%3D2016%26volume%3D164%26spage%3D293%26epage%3D309%26doi%3D10.1016%2Fj.cell.2015.11.062" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref58"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref58'); return false;" data-citation="" class="refNumLink">58</a></strong><div class="NLM_citation" id="cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cochran, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Conery, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sims, R. R.</span></span> <span> </span><span class="NLM_article-title">Bromodomains: a new target class for drug development</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">609</span>– <span class="NLM_lpage">628</span>, <span class="refDoi"> DOI: 10.1038/s41573-019-0030-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1038%2Fs41573-019-0030-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=31273347" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhtlWitbnL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2019&pages=609-628&author=A.+G.+Cochranauthor=A.+R.+Coneryauthor=R.+R.+Sims&title=Bromodomains%3A+a+new+target+class+for+drug+development&doi=10.1038%2Fs41573-019-0030-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">Bromodomains: a new target class for drug development</span></div><div class="casAuthors">Cochran, Andrea G.; Conery, Andrew R.; Sims, III, Robert J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">609-628</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Less than a decade ago, it was shown that bromodomains, acetyl lysine 'reader' modules found in proteins with varied functions, were highly tractable small-mol. targets.  This is an unusual property for protein-protein or protein-peptide interaction domains, and it prompted a wave of chem. probe discovery to understand the biol. potential of new agents that targeted bromodomains.  The original examples, inhibitors of the bromodomain and extra-terminal (BET) class of bromodomains, showed enticing anti-inflammatory and anticancer activities, and several compds. have since advanced to human clin. trials.  Here, we review the current state of BET inhibitor biol. in relation to clin. development, and we discuss the next wave of bromodomain inhibitors with clin. potential in oncol. and non-oncol. indications.  The lessons learned from BET inhibitor programs should affect efforts to develop drugs that target non-BET bromodomains and other epigenetic readers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr6QwxqowW9NbVg90H21EOLACvtfcHk0lhi_x5rg0y1CQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhtlWitbnL&md5=88e56eec8edeb7c325d28156ec131ed8</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1038%2Fs41573-019-0030-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41573-019-0030-7%26sid%3Dliteratum%253Aachs%26aulast%3DCochran%26aufirst%3DA.%2BG.%26aulast%3DConery%26aufirst%3DA.%2BR.%26aulast%3DSims%26aufirst%3DR.%2BR.%26atitle%3DBromodomains%253A%2520a%2520new%2520target%2520class%2520for%2520drug%2520development%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2019%26volume%3D18%26spage%3D609%26epage%3D628%26doi%3D10.1038%2Fs41573-019-0030-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref59"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref59'); return false;" data-citation="" class="refNumLink">59</a></strong><div class="NLM_citation" id="cit59"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rodriguez, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerona-Navarro, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Osman, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, M. M.</span></span> <span> </span><span class="NLM_article-title">In silico design and molecular basis for the selectivity of Olinone toward the first over the second bromodomain of BRD4</span>. <i>Proteins: Struct., Funct., Genet.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>88</i></span>,  <span class="NLM_fpage">414</span>– <span class="NLM_lpage">430</span>, <span class="refDoi"> DOI: 10.1002/prot.25818</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1002%2Fprot.25818" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=31587361" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=88&publication_year=2020&pages=414-430&author=Y.+Rodriguezauthor=G.+Gerona-Navarroauthor=R.+Osmanauthor=M.+M.+Zhou&title=In+silico+design+and+molecular+basis+for+the+selectivity+of+Olinone+toward+the+first+over+the+second+bromodomain+of+BRD4&doi=10.1002%2Fprot.25818"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1002%2Fprot.25818&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fprot.25818%26sid%3Dliteratum%253Aachs%26aulast%3DRodriguez%26aufirst%3DY.%26aulast%3DGerona-Navarro%26aufirst%3DG.%26aulast%3DOsman%26aufirst%3DR.%26aulast%3DZhou%26aufirst%3DM.%2BM.%26atitle%3DIn%2520silico%2520design%2520and%2520molecular%2520basis%2520for%2520the%2520selectivity%2520of%2520Olinone%2520toward%2520the%2520first%2520over%2520the%2520second%2520bromodomain%2520of%2520BRD4%26jtitle%3DProteins%253A%2520Struct.%252C%2520Funct.%252C%2520Genet.%26date%3D2020%26volume%3D88%26spage%3D414%26epage%3D430%26doi%3D10.1002%2Fprot.25818" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref60"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref60'); return false;" data-citation="" class="refNumLink">60</a></strong><div class="NLM_citation" id="cit60"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Divakaran, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Talluri, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ayoub, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mishra, N. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cui, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Widen, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berndt, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, J. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carlson, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Topczewski, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schonbrunn, E. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harki, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pomerantz, W.</span></span> <span> </span><span class="NLM_article-title">Molecular basis for the N-terminal bromodomain-and-extra-terminal-family selectivity of a dual kinase-bromodomain inhibitor</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">9316</span>– <span class="NLM_lpage">9334</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b01248</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b01248" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhvVSis7fM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=9316-9334&author=A.+Divakaranauthor=S.+K.+Talluriauthor=A.+M.+Ayoubauthor=N.+K.+Mishraauthor=H.+Cuiauthor=J.+C.+Widenauthor=N.+Berndtauthor=J.+Y.+Zhuauthor=A.+S.+Carlsonauthor=J.+J.+Topczewskiauthor=E.+K.+Schonbrunnauthor=D.+A.+Harkiauthor=W.+Pomerantz&title=Molecular+basis+for+the+N-terminal+bromodomain-and-extra-terminal-family+selectivity+of+a+dual+kinase-bromodomain+inhibitor&doi=10.1021%2Facs.jmedchem.8b01248"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular Basis for the N-Terminal Bromodomain-and-Extra-Terminal-Family Selectivity of a Dual Kinase-Bromodomain Inhibitor</span></div><div class="casAuthors">Divakaran, Anand; Talluri, Siva K.; Ayoub, Alex M.; Mishra, Neeraj K.; Cui, Huarui; Widen, John C.; Berndt, Norbert; Zhu, Jin-Yi; Carlson, Angela S.; Topczewski, Joseph J.; Schonbrunn, Ernst K.; Harki, Daniel A.; Pomerantz, William C. K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">9316-9334</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">As regulators of transcription, epigenetic proteins that interpret post-translational modifications to N-terminal histone tails are essential for maintaining cellular homeostasis.  When dysregulated, "reader" proteins become drivers of disease.  In the case of bromodomains, which recognize N-ε-acetylated lysine, selective inhibition of individual bromodomain-and-extra-terminal (BET)-family bromodomains has proven challenging.  We describe the >55-fold N-terminal-BET bromodomain selectivity of 1,4,5-trisubstituted-imidazole dual kinase-bromodomain inhibitors.  Selectivity for the BRD4 N-terminal bromodomain (BRD4(1)) over its second bromodomain (BRD4(2)) arises from the displacement of ordered waters and the conformational flexibility of lysine-141 in BRD4(1).  Cellular efficacy was demonstrated via redn. of c-Myc expression, inhibition of NF-κB signaling, and suppression of IL-8 prodn. through potential synergistic inhibition of BRD4(1) and p38α.  These dual inhibitors provide a new scaffold for domain-selective inhibition of BRD4, the aberrant function of which plays a key role in cancer and inflammatory signaling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpZhnL9cPdDi7Vg90H21EOLACvtfcHk0ljEw9pqdTI2nA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhvVSis7fM&md5=5176ef754a898b3e03fc56e2662ab399</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b01248%26sid%3Dliteratum%253Aachs%26aulast%3DDivakaran%26aufirst%3DA.%26aulast%3DTalluri%26aufirst%3DS.%2BK.%26aulast%3DAyoub%26aufirst%3DA.%2BM.%26aulast%3DMishra%26aufirst%3DN.%2BK.%26aulast%3DCui%26aufirst%3DH.%26aulast%3DWiden%26aufirst%3DJ.%2BC.%26aulast%3DBerndt%26aufirst%3DN.%26aulast%3DZhu%26aufirst%3DJ.%2BY.%26aulast%3DCarlson%26aufirst%3DA.%2BS.%26aulast%3DTopczewski%26aufirst%3DJ.%2BJ.%26aulast%3DSchonbrunn%26aufirst%3DE.%2BK.%26aulast%3DHarki%26aufirst%3DD.%2BA.%26aulast%3DPomerantz%26aufirst%3DW.%26atitle%3DMolecular%2520basis%2520for%2520the%2520N-terminal%2520bromodomain-and-extra-terminal-family%2520selectivity%2520of%2520a%2520dual%2520kinase-bromodomain%2520inhibitor%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D9316%26epage%3D9334%26doi%3D10.1021%2Facs.jmedchem.8b01248" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref61"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref61'); return false;" data-citation="" class="refNumLink">61</a></strong><div class="NLM_citation" id="cit61"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Niu, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alamer, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fan, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Endsley, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gelman, B. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tian, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michael, N. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robb, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ananworanich, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, H.</span></span> <span> </span><span class="NLM_article-title">Structure-guided drug design identifies a BRD4-selective small molecule that suppresses HIV.</span>. <i>J. Clin. Invest.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>129</i></span>,  <span class="NLM_fpage">3361</span>– <span class="NLM_lpage">3373</span>, <span class="refDoi"> DOI: 10.1172/JCI120633</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1172%2FJCI120633" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=31329163" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A280%3ADC%252BB3Mzps1ynsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=129&publication_year=2019&pages=3361-3373&author=Q.+Niuauthor=Z.+Liuauthor=E.+Alamerauthor=X.+Fanauthor=H.+Chenauthor=J.+Endsleyauthor=B.+B.+Gelmanauthor=B.+Tianauthor=J.+H.+Kimauthor=N.+L.+Michaelauthor=M.+L.+Robbauthor=J.+Ananworanichauthor=J.+Zhouauthor=H.+Hu&title=Structure-guided+drug+design+identifies+a+BRD4-selective+small+molecule+that+suppresses+HIV.&doi=10.1172%2FJCI120633"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-guided drug design identifies a BRD4-selective small molecule that suppresses HIV</span></div><div class="casAuthors">Niu Qingli; Alamer Edrous; Fan Xiuzhen; Endsley Janice; Hu Haitao; Niu Qingli; Alamer Edrous; Fan Xiuzhen; Endsley Janice; Hu Haitao; Liu Zhiqing; Chen Haiying; Zhou Jia; Gelman Benjamin B; Tian Bing; Kim Jerome H; Michael Nelson L; Robb Merlin L; Ananworanich Jintanat; Michael Nelson L; Robb Merlin L; Ananworanich Jintanat; Ananworanich Jintanat</div><div class="citationInfo"><span class="NLM_cas:title">The Journal of clinical investigation</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">129</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">3361-3373</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">HIV integrates its provirus into the host genome and establishes latent infection.  Antiretroviral therapy (ART) can control HIV viremia, but cannot eradicate or cure the virus.  Approaches targeting host epigenetic machinery to repress HIV, leading to an aviremic state free of ART, are needed.  Bromodomain and extraterminal (BET) family protein BRD4 is an epigenetic reader involved in HIV transcriptional regulation.  Using structure-guided drug design, we identified a small molecule (ZL0580) that induced epigenetic suppression of HIV via BRD4.  We showed that ZL0580 induced HIV suppression in multiple in vitro and ex vivo cell models.  Combination treatment of cells of aviremic HIV-infected individuals with ART and ZL0580 revealed that ZL0580 accelerated HIV suppression during ART and delayed viral rebound after ART cessation.  Mechanistically different from the BET/BRD4 pan-inhibitor JQ1, which nonselectively binds to BD1 and BD2 domains of all BET proteins, ZL0580 selectively bound to BD1 domain of BRD4.  We further demonstrate that ZL0580 induced HIV suppression by inhibiting Tat transactivation and transcription elongation as well as by inducing repressive chromatin structure at the HIV promoter.  Our findings establish a proof of concept for modulation of BRD4 to epigenetically suppress HIV and provide a promising chemical scaffold for the development of probes and/or therapeutic agents for HIV epigenetic silencing.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcS-v5rtAsoTqZ0d-O_J5C0AfW6udTcc2eakTyDNsVcHNLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3Mzps1ynsg%253D%253D&md5=b6b2d6b47419a88ecdf646cf5d3f859b</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1172%2FJCI120633&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1172%252FJCI120633%26sid%3Dliteratum%253Aachs%26aulast%3DNiu%26aufirst%3DQ.%26aulast%3DLiu%26aufirst%3DZ.%26aulast%3DAlamer%26aufirst%3DE.%26aulast%3DFan%26aufirst%3DX.%26aulast%3DChen%26aufirst%3DH.%26aulast%3DEndsley%26aufirst%3DJ.%26aulast%3DGelman%26aufirst%3DB.%2BB.%26aulast%3DTian%26aufirst%3DB.%26aulast%3DKim%26aufirst%3DJ.%2BH.%26aulast%3DMichael%26aufirst%3DN.%2BL.%26aulast%3DRobb%26aufirst%3DM.%2BL.%26aulast%3DAnanworanich%26aufirst%3DJ.%26aulast%3DZhou%26aufirst%3DJ.%26aulast%3DHu%26aufirst%3DH.%26atitle%3DStructure-guided%2520drug%2520design%2520identifies%2520a%2520BRD4-selective%2520small%2520molecule%2520that%2520suppresses%2520HIV.%26jtitle%3DJ.%2520Clin.%2520Invest.%26date%3D2019%26volume%3D129%26spage%3D3361%26epage%3D3373%26doi%3D10.1172%2FJCI120633" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref62"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref62'); return false;" data-citation="" class="refNumLink">62</a></strong><div class="NLM_citation" id="cit62"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cui, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span> <span> </span><span class="NLM_article-title">Discovery of benzo[cd]indol-2(1H)-ones and pyrrolo[4,3,2-de] quinolin-2(1H)-ones as bromodomain and extra-terminal domain (BET) inhibitors with selectivity for the first bromodomain with potential high efficiency against acute gouty arthritis</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">11080</span>– <span class="NLM_lpage">11107</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b01010</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b01010" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BC1MXit1yqur7E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=11080-11107&author=F.+Jiangauthor=Q.+Huauthor=Z.+Zhangauthor=H.+Liauthor=H.+Liauthor=D.+Zhangauthor=H.+Liauthor=Y.+Maauthor=J.+Xuauthor=H.+Chenauthor=Y.+Cuiauthor=Y.+Zhiauthor=Y.+Zhangauthor=J.+Xuauthor=J.+Zhuauthor=T.+Luauthor=Y.+Chen&title=Discovery+of+benzo%5Bcd%5Dindol-2%281H%29-ones+and+pyrrolo%5B4%2C3%2C2-de%5D+quinolin-2%281H%29-ones+as+bromodomain+and+extra-terminal+domain+%28BET%29+inhibitors+with+selectivity+for+the+first+bromodomain+with+potential+high+efficiency+against+acute+gouty+arthritis&doi=10.1021%2Facs.jmedchem.9b01010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit62R"><div class="casContent"><span class="casTitleNuber">62</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Benzo[cd]indol-2(1H)-ones and Pyrrolo[4,3,2-de]quinolin-2(1H)-ones as Bromodomain and Extra-Terminal Domain (BET) Inhibitors with Selectivity for the First Bromodomain with Potential High Efficiency against Acute Gouty Arthritis</span></div><div class="casAuthors">Jiang, Fei; Hu, Qinghua; Zhang, Zhimin; Li, Hongmei; Li, Huili; Zhang, Dewei; Li, Hanwen; Ma, Yu; Xu, Jingjing; Chen, Haifang; Cui, Yong; Zhi, Yanle; Zhang, Yanmin; Xu, Junyu; Zhu, Jiapeng; Lu, Tao; Chen, Yadong</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">11080-11107</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The bromodomain and extra-terminal domain (BET) family of proteins are readers which specifically recognize histone-acetylated lysine residues.  Each BET bromodomain protein contains two highly homologous domains: the first bromodomain (BD1) and the second bromodomain (BD2).  Pan-BET bromodomain inhibition is a potential therapy for various cancers and immune-inflammatory diseases, but only few reported inhibitors show selectivity within the BET family.  Herein, we identified a series of benzo[cd]indol-2(1H)-ones and pyrrolo[4,3,2-de]quinolin-2(1H)-ones with good selectivity for BET BD1.  Through structure-based optimization, highly active and selective compds. are ultimately obtained.  The representative compds. are the first reported inhibitors with selectivity more than 100-fold for BRD4(1) over BRD4(2).  Among them, we further show that 68 (LT052) mediates BRD4/NF-κB/NLRP3 signaling inflammatory pathways with comparable protein expression and significantly improves symptoms of gout arthritis in a rat model.  Therefore, selective pharmacol. modulation of individual bromodomains could represent a strategy for the treatment of acute gouty arthritis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrav2e3P7XQ1LVg90H21EOLACvtfcHk0lgkIuadeFc9wg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXit1yqur7E&md5=5cce669e137812d20f8ba2a9785f4666</span></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b01010%26sid%3Dliteratum%253Aachs%26aulast%3DJiang%26aufirst%3DF.%26aulast%3DHu%26aufirst%3DQ.%26aulast%3DZhang%26aufirst%3DZ.%26aulast%3DLi%26aufirst%3DH.%26aulast%3DLi%26aufirst%3DH.%26aulast%3DZhang%26aufirst%3DD.%26aulast%3DLi%26aufirst%3DH.%26aulast%3DMa%26aufirst%3DY.%26aulast%3DXu%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DH.%26aulast%3DCui%26aufirst%3DY.%26aulast%3DZhi%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DXu%26aufirst%3DJ.%26aulast%3DZhu%26aufirst%3DJ.%26aulast%3DLu%26aufirst%3DT.%26aulast%3DChen%26aufirst%3DY.%26atitle%3DDiscovery%2520of%2520benzo%255Bcd%255Dindol-2%25281H%2529-ones%2520and%2520pyrrolo%255B4%252C3%252C2-de%255D%2520quinolin-2%25281H%2529-ones%2520as%2520bromodomain%2520and%2520extra-terminal%2520domain%2520%2528BET%2529%2520inhibitors%2520with%2520selectivity%2520for%2520the%2520first%2520bromodomain%2520with%2520potential%2520high%2520efficiency%2520against%2520acute%2520gouty%2520arthritis%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D11080%26epage%3D11107%26doi%3D10.1021%2Facs.jmedchem.9b01010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref63"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref63'); return false;" data-citation="" class="refNumLink">63</a></strong><div class="NLM_citation" id="cit63"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wold, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leonard, P. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Joseph, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brasier, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tian, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, J.</span></span> <span> </span><span class="NLM_article-title">Discovery of orally bioavailable chromone derivatives as potent and selective BRD4 inhibitors: scaffold hopping, optimization, and pharmacological evaluation</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">5242</span>– <span class="NLM_lpage">5256</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.0c00035</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.0c00035" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A280%3ADC%252BB38zhslKksA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=5242-5256&author=Z.+Liuauthor=H.+Chenauthor=P.+Wangauthor=Y.+Liauthor=E.+A.+Woldauthor=P.+G.+Leonardauthor=S.+Josephauthor=A.+R.+Brasierauthor=B.+Tianauthor=J.+Zhou&title=Discovery+of+orally+bioavailable+chromone+derivatives+as+potent+and+selective+BRD4+inhibitors%3A+scaffold+hopping%2C+optimization%2C+and+pharmacological+evaluation&doi=10.1021%2Facs.jmedchem.0c00035"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit63R"><div class="casContent"><span class="casTitleNuber">63</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Orally Bioavailable Chromone Derivatives as Potent and Selective BRD4 Inhibitors: Scaffold Hopping, Optimization, and Pharmacological Evaluation</span></div><div class="casAuthors">Liu Zhiqing; Chen Haiying; Wang Pingyuan; Li Yi; Wold Eric A; Zhou Jia; Leonard Paul G; Joseph Sarah; Brasier Allan R; Tian Bing; Tian Bing; Zhou Jia; Zhou Jia</div><div class="citationInfo"><span class="NLM_cas:title">Journal of medicinal chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">5242-5256</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Bromodomain-containing protein 4 (BRD4) represents a promising drug target for anti-inflammatory therapeutics.  Herein, we report the design, synthesis, and pharmacological evaluation of novel chromone derivatives via scaffold hopping to discover a new class of orally bioavailable BRD4-selective inhibitors.  Two potent BRD4 bromodomain 1 (BD1)-selective inhibitors 44 (ZL0513) and 45 (ZL0516) have been discovered with high binding affinity (IC50 values of 67-84 nM) and good selectivity over other BRD family proteins and distant BD-containing proteins.  Both compounds significantly inhibited the expression of Toll-like receptor-induced inflammatory genes in vitro and airway inflammation in murine models.  The cocrystal structure of 45 in complex with human BRD4 BD1 at a high resolution of 2.0 ÅA has been solved, offering a solid structural basis for its binding validation and further structure-based optimization.  These BRD4 BD1 inhibitors demonstrated impressive in vivo efficacy and overall promising pharmacokinetic properties, indicating their therapeutic potential for the treatment of inflammatory diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcR1QLaeKXMGxJ0d-O_J5C0AfW6udTcc2eYNQJea5AlvqLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB38zhslKksA%253D%253D&md5=d65e4298cbfff545c6c333713e4db210</span></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.0c00035&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.0c00035%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DZ.%26aulast%3DChen%26aufirst%3DH.%26aulast%3DWang%26aufirst%3DP.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DWold%26aufirst%3DE.%2BA.%26aulast%3DLeonard%26aufirst%3DP.%2BG.%26aulast%3DJoseph%26aufirst%3DS.%26aulast%3DBrasier%26aufirst%3DA.%2BR.%26aulast%3DTian%26aufirst%3DB.%26aulast%3DZhou%26aufirst%3DJ.%26atitle%3DDiscovery%2520of%2520orally%2520bioavailable%2520chromone%2520derivatives%2520as%2520potent%2520and%2520selective%2520BRD4%2520inhibitors%253A%2520scaffold%2520hopping%252C%2520optimization%252C%2520and%2520pharmacological%2520evaluation%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D5242%26epage%3D5256%26doi%3D10.1021%2Facs.jmedchem.0c00035" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref64"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref64'); return false;" data-citation="" class="refNumLink">64</a></strong><div class="NLM_citation" id="cit64"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Watson, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bamborough, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barnett, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C.-w.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grandi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petretich, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phillipou, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rioja, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soden, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Werner, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Demont, E. H.</span></span> <span> </span><span class="NLM_article-title">Demont E. H. GSK789: a selective inhibitor of the first bromodomains (BD1) of the bromo and extra terminal domain (BET) proteins</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">9045</span>– <span class="NLM_lpage">9069</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.0c00614</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.0c00614" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhsVWrsLzJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=9045-9069&author=R.+J.+Watsonauthor=P.+Bamboroughauthor=H.+Barnettauthor=C.-w.+Chungauthor=R.+Davisauthor=L.+Gordonauthor=P.+Grandiauthor=M.+Petretichauthor=A.+Phillipouauthor=R.+K.+Prinjhaauthor=I.+Riojaauthor=P.+Sodenauthor=T.+Wernerauthor=E.+H.+Demont&title=Demont+E.+H.+GSK789%3A+a+selective+inhibitor+of+the+first+bromodomains+%28BD1%29+of+the+bromo+and+extra+terminal+domain+%28BET%29+proteins&doi=10.1021%2Facs.jmedchem.0c00614"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit64R"><div class="casContent"><span class="casTitleNuber">64</span><div class="casTitle"><span class="NLM_cas:atitle">GSK789: A Selective Inhibitor of the First Bromodomains (BD1) of the Bromo and Extra Terminal Domain (BET) Proteins</span></div><div class="casAuthors">Watson, Robert J.; Bamborough, Paul; Barnett, Heather; Chung, Chun-wa; Davis, Rob; Gordon, Laurie; Grandi, Paola; Petretich, Massimo; Phillipou, Alex; Prinjha, Rab K.; Rioja, Inmaculada; Soden, Peter; Werner, Thilo; Demont, Emmanuel H.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">9045-9069</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Pan-bromodomain and extra terminal (BET) inhibitors interact equipotently with all eight bromodomains of the BET family of proteins.  They have shown profound efficacy in vitro and in vivo in oncol. and immunomodulatory models, and a no. of them are currently in clin. trials where significant safety signals have been reported.  It is therefore important to understand the functional contribution of each bromodomain to assess the opportunity to tease apart efficacy and toxicity.  This article discloses the in vitro and cellular activity profiles of GSK789(I), a potent, cell-permeable, and highly selective inhibitor of the first bromodomains of the BET family.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGozid6PMjhvvbVg90H21EOLACvtfcHk0lgkIuadeFc9wg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhsVWrsLzJ&md5=c86b4d2c85dcfa62854debf225c8076c</span></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.0c00614&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.0c00614%26sid%3Dliteratum%253Aachs%26aulast%3DWatson%26aufirst%3DR.%2BJ.%26aulast%3DBamborough%26aufirst%3DP.%26aulast%3DBarnett%26aufirst%3DH.%26aulast%3DChung%26aufirst%3DC.-w.%26aulast%3DDavis%26aufirst%3DR.%26aulast%3DGordon%26aufirst%3DL.%26aulast%3DGrandi%26aufirst%3DP.%26aulast%3DPetretich%26aufirst%3DM.%26aulast%3DPhillipou%26aufirst%3DA.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DRioja%26aufirst%3DI.%26aulast%3DSoden%26aufirst%3DP.%26aulast%3DWerner%26aufirst%3DT.%26aulast%3DDemont%26aufirst%3DE.%2BH.%26atitle%3DDemont%2520E.%2520H.%2520GSK789%253A%2520a%2520selective%2520inhibitor%2520of%2520the%2520first%2520bromodomains%2520%2528BD1%2529%2520of%2520the%2520bromo%2520and%2520extra%2520terminal%2520domain%2520%2528BET%2529%2520proteins%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D9045%26epage%3D9069%26doi%3D10.1021%2Facs.jmedchem.0c00614" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref65"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref65'); return false;" data-citation="" class="refNumLink">65</a></strong><div class="NLM_citation" id="cit65"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wellaway, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bamborough, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernard, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Craggs, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cutler, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Demont, E. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evans, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karamshi, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lewis, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindon, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitchell, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rioja, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soden, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watson, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Willis, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woolven, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wyspianska, B. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kerr, W. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span> <span> </span><span class="NLM_article-title">Structure-based design of a bromodomain and extraterminal domain (BET) inhibitor selective for the N-terminal bromodomains that retains an anti-inflammatory and antiproliferative phenotype</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">9020</span>– <span class="NLM_lpage">9044</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.0c00566</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.0c00566" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhsV2lt77M" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=9020-9044&author=C.+R.+Wellawayauthor=P.+Bamboroughauthor=S.+G.+Bernardauthor=C.+W.+Chungauthor=P.+D.+Craggsauthor=L.+Cutlerauthor=E.+H.+Demontauthor=J.+P.+Evansauthor=L.+Gordonauthor=B.+Karamshiauthor=A.+J.+Lewisauthor=M.+J.+Lindonauthor=D.+J.+Mitchellauthor=I.+Riojaauthor=P.+E.+Sodenauthor=S.+Taylorauthor=R.+J.+Watsonauthor=R.+Willisauthor=J.+M.+Woolvenauthor=B.+S.+Wyspianskaauthor=W.+J.+Kerrauthor=R.+K.+Prinjha&title=Structure-based+design+of+a+bromodomain+and+extraterminal+domain+%28BET%29+inhibitor+selective+for+the+N-terminal+bromodomains+that+retains+an+anti-inflammatory+and+antiproliferative+phenotype&doi=10.1021%2Facs.jmedchem.0c00566"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit65R"><div class="casContent"><span class="casTitleNuber">65</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Based Design of a Bromodomain and Extraterminal Domain (BET) Inhibitor Selective for the N-Terminal Bromodomains That Retains an Anti-inflammatory and Antiproliferative Phenotype</span></div><div class="casAuthors">Wellaway, Christopher R.; Bamborough, Paul; Bernard, Sharon G.; Chung, Chun-wa; Craggs, Peter D.; Cutler, Leanne; Demont, Emmanuel H.; Evans, John P.; Gordon, Laurie; Karamshi, Bhumika; Lewis, Antonia J.; Lindon, Matthew J.; Mitchell, Darren J.; Rioja, Inmaculada; Soden, Peter E.; Taylor, Simon; Watson, Robert J.; Willis, Rob; Woolven, James M.; Wyspianska, Beata S.; Kerr, William J.; Prinjha, Rab K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">9020-9044</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The bromodomain and extraterminal domain (BET) family of epigenetic regulators comprises four proteins (BRD2, BRD3, BRD4, BRDT), each contg. tandem bromodomains.  To date, small mol. inhibitors of these proteins typically bind all eight bromodomains of the family with similar affinity, resulting in a diverse range of biol. effects.  To enable further understanding of the broad phenotype characteristic of pan-BET inhibition, the development of inhibitors selective for individual, or sets of, bromodomains within the family is required.  In this regard, we report the discovery of a potent probe mol. possessing up to 150-fold selectivity for the N-terminal bromodomains (BD1s) over the C-terminal bromodomains (BD2s) of the BETs.  Guided by structural information, a specific amino acid difference between BD1 and BD2 domains was targeted for selective interaction with chem. functionality appended to the previously developed I-BET151 scaffold.  Data presented herein demonstrate that selective inhibition of BD1 domains is sufficient to drive anti-inflammatory and antiproliferative effects.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpkUREcHUhFfbVg90H21EOLACvtfcHk0lhzFw9sm__KqA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhsV2lt77M&md5=52b2505bb0a36f526a08bb9bad1f2c26</span></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.0c00566&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.0c00566%26sid%3Dliteratum%253Aachs%26aulast%3DWellaway%26aufirst%3DC.%2BR.%26aulast%3DBamborough%26aufirst%3DP.%26aulast%3DBernard%26aufirst%3DS.%2BG.%26aulast%3DChung%26aufirst%3DC.%2BW.%26aulast%3DCraggs%26aufirst%3DP.%2BD.%26aulast%3DCutler%26aufirst%3DL.%26aulast%3DDemont%26aufirst%3DE.%2BH.%26aulast%3DEvans%26aufirst%3DJ.%2BP.%26aulast%3DGordon%26aufirst%3DL.%26aulast%3DKaramshi%26aufirst%3DB.%26aulast%3DLewis%26aufirst%3DA.%2BJ.%26aulast%3DLindon%26aufirst%3DM.%2BJ.%26aulast%3DMitchell%26aufirst%3DD.%2BJ.%26aulast%3DRioja%26aufirst%3DI.%26aulast%3DSoden%26aufirst%3DP.%2BE.%26aulast%3DTaylor%26aufirst%3DS.%26aulast%3DWatson%26aufirst%3DR.%2BJ.%26aulast%3DWillis%26aufirst%3DR.%26aulast%3DWoolven%26aufirst%3DJ.%2BM.%26aulast%3DWyspianska%26aufirst%3DB.%2BS.%26aulast%3DKerr%26aufirst%3DW.%2BJ.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26atitle%3DStructure-based%2520design%2520of%2520a%2520bromodomain%2520and%2520extraterminal%2520domain%2520%2528BET%2529%2520inhibitor%2520selective%2520for%2520the%2520N-terminal%2520bromodomains%2520that%2520retains%2520an%2520anti-inflammatory%2520and%2520antiproliferative%2520phenotype%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D9020%26epage%3D9044%26doi%3D10.1021%2Facs.jmedchem.0c00566" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref66"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref66'); return false;" data-citation="" class="refNumLink">66</a></strong><div class="NLM_citation" id="cit66"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cui, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Divakaran, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pandey, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zahid, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoell, Z. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ellingson, M. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aihara, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harki, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pomerantz, W. C. K.</span></span> <span> </span><span class="NLM_article-title">Selective N-terminal BRD4 bromodomain inhibitors by targeting non-conserved residues and structured water displacement</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2021</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">1220</span>– <span class="NLM_lpage">1226</span>, <span class="refDoi"> DOI: 10.1002/anie.202008625</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1002%2Fanie.202008625" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BB3cXitlOqtLjI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2021&pages=1220-1226&author=H.+Cuiauthor=A.+Divakaranauthor=A.+K.+Pandeyauthor=J.+A.+Johnsonauthor=H.+Zahidauthor=Z.+J.+Hoellauthor=M.+O.+Ellingsonauthor=K.+Shiauthor=H.+Aiharaauthor=D.+A.+Harkiauthor=W.+C.+K.+Pomerantz&title=Selective+N-terminal+BRD4+bromodomain+inhibitors+by+targeting+non-conserved+residues+and+structured+water+displacement&doi=10.1002%2Fanie.202008625"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit66R"><div class="casContent"><span class="casTitleNuber">66</span><div class="casTitle"><span class="NLM_cas:atitle">Selective N-Terminal BET Bromodomain Inhibitors by Targeting Non-Conserved Residues and Structured Water Displacement**</span></div><div class="casAuthors">Cui, Huarui; Divakaran, Anand; Pandey, Anil K.; Johnson, Jorden A.; Zahid, Huda; Hoell, Zachariah J.; Ellingson, Mikael O.; Shi, Ke; Aihara, Hideki; Harki, Daniel A.; Pomerantz, William C. K.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2021</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1220-1226</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Bromodomain and extra-terminal (BET) family proteins, BRD2-4 and T, are important drug targets; however, the biol. functions of each bromodomain remain ill-defined.  Chem. probes that selectively inhibit a single BET bromodomain are lacking, although pan inhibitors of the first (D1), and second (D2), bromodomain are known.  Here, we develop selective BET D1 inhibitors with preferred binding to BRD4 D1.  In competitive inhibition assays, we show that our lead compd. is 9-33 fold selective for BRD4 D1 over the other BET bromodomains.  X-ray crystallog. supports a role for the selectivity based on reorganization of a non-conserved lysine and displacement of an addnl. structured water in the BRD4 D1 binding site relative to our prior lead.  Whereas pan-D1 inhibitors displace BRD4 from MYC enhancers, BRD4 D1 inhibition in MM.1S cells is insufficient for stopping Myc expression and may lead to its upregulation.  Future anal. of BRD4 D1 gene regulation may shed light on differential BET bromodomain functions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr19lACXxcpUrVg90H21EOLACvtfcHk0lhzFw9sm__KqA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXitlOqtLjI&md5=0f0379a17c98e5c1b9c436dc0952e9a7</span></div><a href="/servlet/linkout?suffix=cit66&amp;dbid=16384&amp;doi=10.1002%2Fanie.202008625&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.202008625%26sid%3Dliteratum%253Aachs%26aulast%3DCui%26aufirst%3DH.%26aulast%3DDivakaran%26aufirst%3DA.%26aulast%3DPandey%26aufirst%3DA.%2BK.%26aulast%3DJohnson%26aufirst%3DJ.%2BA.%26aulast%3DZahid%26aufirst%3DH.%26aulast%3DHoell%26aufirst%3DZ.%2BJ.%26aulast%3DEllingson%26aufirst%3DM.%2BO.%26aulast%3DShi%26aufirst%3DK.%26aulast%3DAihara%26aufirst%3DH.%26aulast%3DHarki%26aufirst%3DD.%2BA.%26aulast%3DPomerantz%26aufirst%3DW.%2BC.%2BK.%26atitle%3DSelective%2520N-terminal%2520BRD4%2520bromodomain%2520inhibitors%2520by%2520targeting%2520non-conserved%2520residues%2520and%2520structured%2520water%2520displacement%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2021%26volume%3D60%26spage%3D1220%26epage%3D1226%26doi%3D10.1002%2Fanie.202008625" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref67"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref67'); return false;" data-citation="" class="refNumLink">67</a></strong><div class="NLM_citation" id="cit67"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Law, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Atkinson, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bamborough, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Demont, E. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindon, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watson, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hirst, D. J.</span></span> <span> </span><span class="NLM_article-title">Discovery of tetrahydroquinoxalines as bromodomain and extra-terminal domain (BET) inhibitors with selectivity for the second bromodomain</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">4317</span>– <span class="NLM_lpage">4334</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b01666</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b01666" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BC1cXns1Sjtbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=4317-4334&author=R.+P.+Lawauthor=S.+J.+Atkinsonauthor=P.+Bamboroughauthor=C.+W.+Chungauthor=E.+H.+Demontauthor=L.+J.+Gordonauthor=M.+Lindonauthor=R.+K.+Prinjhaauthor=A.+Watsonauthor=D.+J.+Hirst&title=Discovery+of+tetrahydroquinoxalines+as+bromodomain+and+extra-terminal+domain+%28BET%29+inhibitors+with+selectivity+for+the+second+bromodomain&doi=10.1021%2Facs.jmedchem.7b01666"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit67R"><div class="casContent"><span class="casTitleNuber">67</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Tetrahydroquinoxalines as Bromodomain and Extra-Terminal Domain (BET) Inhibitors with Selectivity for the Second Bromodomain</span></div><div class="casAuthors">Law, Robert P.; Atkinson, Stephen J.; Bamborough, Paul; Chung, Chun-wa; Demont, Emmanuel H.; Gordon, Laurie J.; Lindon, Matthew; Prinjha, Rab K.; Watson, Allan J. B.; Hirst, David J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">4317-4334</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The bromodomain and extra-terminal domain (BET) family of proteins bind acetylated lysine residues on histone proteins.  The four BET bromodomains-BRD2, BRD3, BRD4, and BRDT-each contain two bromodomain modules.  BET bromodomain inhibition is a potential therapy for various cancers and immunoinflammatory diseases, but few reported inhibitors show selectivity within the BET family.  Inhibitors with selectivity for the first or second bromodomain are desired to aid investigation of the biol. function of these domains.  Focused library screening identified a series of tetrahydroquinoxalines with selectivity for the second bromodomains of the BET family (BD2).  Structure-guided optimization of the template improved potency, selectivity, and physicochem. properties, culminating in potent BET inhibitors with BD2 selectivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGobBKYwu7NyTbVg90H21EOLACvtfcHk0li6W54vhCBVYg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXns1Sjtbc%253D&md5=078db91b35feaf1958fa8c33d0544387</span></div><a href="/servlet/linkout?suffix=cit67&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b01666&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b01666%26sid%3Dliteratum%253Aachs%26aulast%3DLaw%26aufirst%3DR.%2BP.%26aulast%3DAtkinson%26aufirst%3DS.%2BJ.%26aulast%3DBamborough%26aufirst%3DP.%26aulast%3DChung%26aufirst%3DC.%2BW.%26aulast%3DDemont%26aufirst%3DE.%2BH.%26aulast%3DGordon%26aufirst%3DL.%2BJ.%26aulast%3DLindon%26aufirst%3DM.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DWatson%26aufirst%3DA.%26aulast%3DHirst%26aufirst%3DD.%2BJ.%26atitle%3DDiscovery%2520of%2520tetrahydroquinoxalines%2520as%2520bromodomain%2520and%2520extra-terminal%2520domain%2520%2528BET%2529%2520inhibitors%2520with%2520selectivity%2520for%2520the%2520second%2520bromodomain%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D4317%26epage%3D4334%26doi%3D10.1021%2Facs.jmedchem.7b01666" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref68"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref68'); return false;" data-citation="" class="refNumLink">68</a></strong><div class="NLM_citation" id="cit68"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lyu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y.</span></span> <span> </span><span class="NLM_article-title">Discovery, structural insight, and bioactivities of BY27 as a selective inhibitor of the second bromodomains of BET proteins</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>182</i></span>,  <span class="NLM_fpage">111633</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2019.111633</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1016%2Fj.ejmech.2019.111633" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=31461688" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhs1Kktb%252FJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=182&publication_year=2019&pages=111633&author=D.+Chenauthor=T.+Luauthor=Z.+Yanauthor=W.+Luauthor=F.+Zhouauthor=X.+Lyuauthor=B.+Xuauthor=H.+Jiangauthor=K.+Chenauthor=C.+Luoauthor=Y.+Zhao&title=Discovery%2C+structural+insight%2C+and+bioactivities+of+BY27+as+a+selective+inhibitor+of+the+second+bromodomains+of+BET+proteins&doi=10.1016%2Fj.ejmech.2019.111633"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit68R"><div class="casContent"><span class="casTitleNuber">68</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery, structural insight, and bioactivities of BY27 as a selective inhibitor of the second bromodomains of BET proteins</span></div><div class="casAuthors">Chen, Deheng; Lu, Tian; Yan, Ziqin; Lu, Wenchao; Zhou, Feilong; Lyu, Xilin; Xu, Biling; Jiang, Hualiang; Chen, Kaixian; Luo, Cheng; Zhao, Yujun</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">182</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">111633</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">Recently, selective inhibition of BET BD2 is emerging as a promising strategy for drug discovery.  Despite significant progress in this area, systematic studies of selective BET BD2 inhibitors are still few.  In this study, we report the discovery of a potent and selective BET BD2 inhibitor BY27 (47).  Our high resoln. co-crystal structures of 47/BRD2 BD1 and BD2 showed that the triazole group of 47, water mols., H433 and N429 in BRD2 BD2 established a water-bridged H-bonding network, which is responsible for the obsd. selectivities.  DNA microarray anal. of HepG2 cells treated with 47 or OTX015 demonstrated the transcriptome impact differences between a BET BD2 selective inhibitor and a pan BET inhibitor.  In a MV4-11 mouse xenograft model, 47 caused 67% of tumor growth inhibition and was less toxic than a pan BET inhibitor 1 at high doses.  We conclude that the improved safety profile of selective BET BD2 inhibitors warrant future studies in BET assocd. diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGre_6szzmO9ZbVg90H21EOLACvtfcHk0ljhQqVIp92FPQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhs1Kktb%252FJ&md5=e3a21e4e692b25a94e0035a2e75d33f7</span></div><a href="/servlet/linkout?suffix=cit68&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2019.111633&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2019.111633%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DD.%26aulast%3DLu%26aufirst%3DT.%26aulast%3DYan%26aufirst%3DZ.%26aulast%3DLu%26aufirst%3DW.%26aulast%3DZhou%26aufirst%3DF.%26aulast%3DLyu%26aufirst%3DX.%26aulast%3DXu%26aufirst%3DB.%26aulast%3DJiang%26aufirst%3DH.%26aulast%3DChen%26aufirst%3DK.%26aulast%3DLuo%26aufirst%3DC.%26aulast%3DZhao%26aufirst%3DY.%26atitle%3DDiscovery%252C%2520structural%2520insight%252C%2520and%2520bioactivities%2520of%2520BY27%2520as%2520a%2520selective%2520inhibitor%2520of%2520the%2520second%2520bromodomains%2520of%2520BET%2520proteins%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2019%26volume%3D182%26spage%3D111633%26doi%3D10.1016%2Fj.ejmech.2019.111633" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref69"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref69'); return false;" data-citation="" class="refNumLink">69</a></strong><div class="NLM_citation" id="cit69"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Faivre, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mcdaniel, K. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Albert, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mantena, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plotnik, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilcox, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bui, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheppard, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sehgal, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uziel, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hessler, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lam, L. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bellin, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mehta, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fidanze, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pratt, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hasvold, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Panchal, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nicolette, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fossey, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Longenecker, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bigelow, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Torrent, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosenberg, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kati, W. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, Y.</span></span> <span> </span><span class="NLM_article-title">Selective inhibition of the BD2 bromodomain of BET proteins in prostate cancer</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>578</i></span>,  <span class="NLM_fpage">306</span>– <span class="NLM_lpage">310</span>, <span class="refDoi"> DOI: 10.1038/s41586-020-1930-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1038%2Fs41586-020-1930-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=31969702" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BB3cXis1Smsr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=578&publication_year=2020&pages=306-310&author=E.+J.+Faivreauthor=K.+F.+Mcdanielauthor=D.+H.+Albertauthor=S.+R.+Mantenaauthor=J.+P.+Plotnikauthor=D.+Wilcoxauthor=L.+Zhangauthor=M.+H.+Buiauthor=G.+S.+Sheppardauthor=L.+Wangauthor=V.+Sehgalauthor=X.+Linauthor=X.+Huangauthor=X.+Luauthor=T.+Uzielauthor=P.+Hesslerauthor=L.+T.+Lamauthor=R.+J.+Bellinauthor=G.+Mehtaauthor=S.+Fidanzeauthor=J.+K.+Prattauthor=D.+Liuauthor=L.+A.+Hasvoldauthor=C.+Sunauthor=S.+C.+Panchalauthor=J.+J.+Nicoletteauthor=S.+L.+Fosseyauthor=C.+H.+Parkauthor=K.+Longeneckerauthor=L.+Bigelowauthor=M.+Torrentauthor=S.+H.+Rosenbergauthor=W.+M.+Katiauthor=Y.+Shen&title=Selective+inhibition+of+the+BD2+bromodomain+of+BET+proteins+in+prostate+cancer&doi=10.1038%2Fs41586-020-1930-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit69R"><div class="casContent"><span class="casTitleNuber">69</span><div class="casTitle"><span class="NLM_cas:atitle">Selective inhibition of the BD2 bromodomain of BET proteins in prostate cancer</span></div><div class="casAuthors">Faivre, Emily J.; McDaniel, Keith F.; Albert, Daniel H.; Mantena, Srinivasa R.; Plotnik, Joshua P.; Wilcox, Denise; Zhang, Lu; Bui, Mai H.; Sheppard, George S.; Wang, Le; Sehgal, Vasudha; Lin, Xiaoyu; Huang, Xiaoli; Lu, Xin; Uziel, Tamar; Hessler, Paul; Lam, Lloyd T.; Bellin, Richard J.; Mehta, Gaurav; Fidanze, Steve; Pratt, John K.; Liu, Dachun; Hasvold, Lisa A.; Sun, Chaohong; Panchal, Sanjay C.; Nicolette, John J.; Fossey, Stacey L.; Park, Chang H.; Longenecker, Kenton; Bigelow, Lance; Torrent, Maricel; Rosenberg, Saul H.; Kati, Warren M.; Shen, Yu</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">578</span>
        (<span class="NLM_cas:issue">7794</span>),
    <span class="NLM_cas:pages">306-310</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Abstr.: Proteins of the bromodomain and extra-terminal (BET) domain family are epigenetic readers that bind acetylated histones through their bromodomains to regulate gene transcription.  Dual-bromodomain BET inhibitors (DbBi) that bind with similar affinities to the first (BD1) and second (BD2) bromodomains of BRD2, BRD3, BRD4 and BRDt have displayed modest clin. activity in monotherapy cancer trials.  A reduced no. of thrombocytes in the blood (thrombocytopenia) as well as symptoms of gastrointestinal toxicity are dose-limiting adverse events for some types of DbBi1-5.  Given that similar haematol. and gastrointestinal defects were obsd. after genetic silencing of Brd4 in mice6, the platelet and gastrointestinal toxicities may represent on-target activities assocd. with BET inhibition.  The two individual bromodomains in BET family proteins may have distinct functions7-9 and different cellular phenotypes after pharmacol. inhibition of one or both bromodomains have been reported10,11, suggesting that selectively targeting one of the bromodomains may result in a different efficacy and tolerability profile compared with DbBi.  Available compds. that are selective to individual domains lack sufficient potency and the pharmacokinetics properties that are required for in vivo efficacy and tolerability assessment10-13.  Here we carried out a medicinal chem. campaign that led to the discovery of ABBV-744, a highly potent and selective inhibitor of the BD2 domain of BET family proteins with drug-like properties.  In contrast to the broad range of cell growth inhibition induced by DbBi, the antiproliferative activity of ABBV-744 was largely, but not exclusively, restricted to cell lines of acute myeloid leukemia and prostate cancer that expressed the full-length androgen receptor (AR).  ABBV-744 retained robust activity in prostate cancer xenografts, and showed fewer platelet and gastrointestinal toxicities than the DbBi ABBV-07514.  Analyses of RNA expression and chromatin immunopptn. followed by sequencing revealed that ABBV-744 displaced BRD4 from AR-contg. super-enhancers and inhibited AR-dependent transcription, with less impact on global transcription compared with ABBV-075.  These results underscore the potential value of selectively targeting the BD2 domain of BET family proteins for cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrF98kER1y9pLVg90H21EOLACvtfcHk0ljhQqVIp92FPQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXis1Smsr0%253D&md5=800d9106c33084390fd8cbf1ac7f1766</span></div><a href="/servlet/linkout?suffix=cit69&amp;dbid=16384&amp;doi=10.1038%2Fs41586-020-1930-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41586-020-1930-8%26sid%3Dliteratum%253Aachs%26aulast%3DFaivre%26aufirst%3DE.%2BJ.%26aulast%3DMcdaniel%26aufirst%3DK.%2BF.%26aulast%3DAlbert%26aufirst%3DD.%2BH.%26aulast%3DMantena%26aufirst%3DS.%2BR.%26aulast%3DPlotnik%26aufirst%3DJ.%2BP.%26aulast%3DWilcox%26aufirst%3DD.%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DBui%26aufirst%3DM.%2BH.%26aulast%3DSheppard%26aufirst%3DG.%2BS.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DSehgal%26aufirst%3DV.%26aulast%3DLin%26aufirst%3DX.%26aulast%3DHuang%26aufirst%3DX.%26aulast%3DLu%26aufirst%3DX.%26aulast%3DUziel%26aufirst%3DT.%26aulast%3DHessler%26aufirst%3DP.%26aulast%3DLam%26aufirst%3DL.%2BT.%26aulast%3DBellin%26aufirst%3DR.%2BJ.%26aulast%3DMehta%26aufirst%3DG.%26aulast%3DFidanze%26aufirst%3DS.%26aulast%3DPratt%26aufirst%3DJ.%2BK.%26aulast%3DLiu%26aufirst%3DD.%26aulast%3DHasvold%26aufirst%3DL.%2BA.%26aulast%3DSun%26aufirst%3DC.%26aulast%3DPanchal%26aufirst%3DS.%2BC.%26aulast%3DNicolette%26aufirst%3DJ.%2BJ.%26aulast%3DFossey%26aufirst%3DS.%2BL.%26aulast%3DPark%26aufirst%3DC.%2BH.%26aulast%3DLongenecker%26aufirst%3DK.%26aulast%3DBigelow%26aufirst%3DL.%26aulast%3DTorrent%26aufirst%3DM.%26aulast%3DRosenberg%26aufirst%3DS.%2BH.%26aulast%3DKati%26aufirst%3DW.%2BM.%26aulast%3DShen%26aufirst%3DY.%26atitle%3DSelective%2520inhibition%2520of%2520the%2520BD2%2520bromodomain%2520of%2520BET%2520proteins%2520in%2520prostate%2520cancer%26jtitle%3DNature%26date%3D2020%26volume%3D578%26spage%3D306%26epage%3D310%26doi%3D10.1038%2Fs41586-020-1930-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref70"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref70'); return false;" data-citation="" class="refNumLink">70</a></strong><div class="NLM_citation" id="cit70"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sheppard, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fidanze, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hasvold, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pratt, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Longenecker, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qiu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Torrent, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kovar, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bui, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Faivre, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilcox, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Albert, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Magoc, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rajaraman, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kati, W. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mcdaniel, K. F.</span></span> <span> </span><span class="NLM_article-title">Discovery of N-Ethyl-4-[2-(4-fluoro-2,6-dimethyl-phenoxy)-5-(1-hydroxy-1-methyl-ethyl)phenyl]- 6-methyl-7-oxo-1H-pyrrolo[2,3-c]pyridine-2-carboxamide (ABBV-744), a BET bromodomain inhibitor with selectivity for the second bromodomain</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">5585</span>– <span class="NLM_lpage">5623</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.0c00628</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.0c00628" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BB3cXnvVeisrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=5585-5623&author=G.+S.+Sheppardauthor=L.+Wangauthor=S.+D.+Fidanzeauthor=L.+A.+Hasvoldauthor=D.+Liuauthor=J.+K.+Prattauthor=C.+H.+Parkauthor=K.+Longeneckerauthor=W.+Qiuauthor=M.+Torrentauthor=P.+J.+Kovarauthor=M.+Buiauthor=E.+Faivreauthor=X.+Huangauthor=X.+Linauthor=D.+Wilcoxauthor=L.+Zhangauthor=Y.+Shenauthor=D.+H.+Albertauthor=T.+J.+Magocauthor=G.+Rajaramanauthor=W.+M.+Katiauthor=K.+F.+Mcdaniel&title=Discovery+of+N-Ethyl-4-%5B2-%284-fluoro-2%2C6-dimethyl-phenoxy%29-5-%281-hydroxy-1-methyl-ethyl%29phenyl%5D-+6-methyl-7-oxo-1H-pyrrolo%5B2%2C3-c%5Dpyridine-2-carboxamide+%28ABBV-744%29%2C+a+BET+bromodomain+inhibitor+with+selectivity+for+the+second+bromodomain&doi=10.1021%2Facs.jmedchem.0c00628"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit70R"><div class="casContent"><span class="casTitleNuber">70</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of N-Ethyl-4-[2-(4-fluoro-2,6-dimethyl-phenoxy)-5-(1-hydroxy-1-methyl-ethyl)phenyl]-6-methyl-7-oxo-1H-pyrrolo[2,3-c]pyridine-2-carboxamide (ABBV-744), a BET Bromodomain Inhibitor with Selectivity for the Second Bromodomain</span></div><div class="casAuthors">Sheppard, George S.; Wang, Le; Fidanze, Steven D.; Hasvold, Lisa A.; Liu, Dachun; Pratt, John K.; Park, Chang H.; Longenecker, Kenton; Qiu, Wei; Torrent, Maricel; Kovar, Peter J.; Bui, Mai; Faivre, Emily; Huang, Xiaoli; Lin, Xiaoyu; Wilcox, Denise; Zhang, Lu; Shen, Yu; Albert, Daniel H.; Magoc, Terrance J.; Rajaraman, Ganesh; Kati, Warren M.; McDaniel, Keith F.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">5585-5623</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The BET family of proteins consists of BRD2, BRD3, BRD4, and BRDt.  Each protein contains two distinct bromodomains (BD1 and BD2).  BET family bromodomain inhibitors under clin. development for oncol. bind to each of the eight bromodomains with similar affinities.  We hypothesized that it may be possible to achieve an improved therapeutic index by selectively targeting subsets of the BET bromodomains.  Both BD1 and BD2 are highly conserved across family members (>70% identity), whereas BD1 and BD2 from the same protein exhibit a larger degree of divergence (~ 40% identity), suggesting selectivity between BD1 and BD2 of all family members would be more straightforward to achieve.  Exploiting the Asp144/His437 and Ile146/Val439 sequence differences (BRD4 BD1/BD2 numbering) allowed the identification of compd. 27 demonstrating greater than 100-fold selectivity for BRD4 BD2 over BRD4 BD1.  Further optimization to improve BD2 selectivity and oral bioavailability resulted in the clin. development compd. 46 (ABBV-744).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoJtZIcWro0xLVg90H21EOLACvtfcHk0lhuKgyydNmgnQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXnvVeisrk%253D&md5=675827659c3c7b6099ccd667f4c60f95</span></div><a href="/servlet/linkout?suffix=cit70&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.0c00628&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.0c00628%26sid%3Dliteratum%253Aachs%26aulast%3DSheppard%26aufirst%3DG.%2BS.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DFidanze%26aufirst%3DS.%2BD.%26aulast%3DHasvold%26aufirst%3DL.%2BA.%26aulast%3DLiu%26aufirst%3DD.%26aulast%3DPratt%26aufirst%3DJ.%2BK.%26aulast%3DPark%26aufirst%3DC.%2BH.%26aulast%3DLongenecker%26aufirst%3DK.%26aulast%3DQiu%26aufirst%3DW.%26aulast%3DTorrent%26aufirst%3DM.%26aulast%3DKovar%26aufirst%3DP.%2BJ.%26aulast%3DBui%26aufirst%3DM.%26aulast%3DFaivre%26aufirst%3DE.%26aulast%3DHuang%26aufirst%3DX.%26aulast%3DLin%26aufirst%3DX.%26aulast%3DWilcox%26aufirst%3DD.%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DShen%26aufirst%3DY.%26aulast%3DAlbert%26aufirst%3DD.%2BH.%26aulast%3DMagoc%26aufirst%3DT.%2BJ.%26aulast%3DRajaraman%26aufirst%3DG.%26aulast%3DKati%26aufirst%3DW.%2BM.%26aulast%3DMcdaniel%26aufirst%3DK.%2BF.%26atitle%3DDiscovery%2520of%2520N-Ethyl-4-%255B2-%25284-fluoro-2%252C6-dimethyl-phenoxy%2529-5-%25281-hydroxy-1-methyl-ethyl%2529phenyl%255D-%25206-methyl-7-oxo-1H-pyrrolo%255B2%252C3-c%255Dpyridine-2-carboxamide%2520%2528ABBV-744%2529%252C%2520a%2520BET%2520bromodomain%2520inhibitor%2520with%2520selectivity%2520for%2520the%2520second%2520bromodomain%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D5585%26epage%3D5623%26doi%3D10.1021%2Facs.jmedchem.0c00628" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref71"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref71'); return false;" data-citation="" class="refNumLink">71</a></strong><div class="NLM_citation" id="cit71"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Preston, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Atkinson, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bamborough, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Craggs, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grandi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindon, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michon, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitchell, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rianjongdee, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rioja, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seal, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wall, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watson, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woolven, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Demont, E. H.</span></span> <span> </span><span class="NLM_article-title">Design and synthesis of a highly selective and in vivo-capable inhibitor of the second bromodomain of the bromodomain and extra terminal domain family of proteins</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">9070</span>– <span class="NLM_lpage">9092</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.0c00605</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.0c00605" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhsVWrsLrK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=9070-9092&author=A.+Prestonauthor=S.+Atkinsonauthor=P.+Bamboroughauthor=C.+W.+Chungauthor=P.+D.+Craggsauthor=L.+Gordonauthor=P.+Grandiauthor=J.+Grayauthor=E.+J.+Jonesauthor=M.+Lindonauthor=A.+M.+Michonauthor=D.+J.+Mitchellauthor=R.+K.+Prinjhaauthor=F.+Rianjongdeeauthor=I.+Riojaauthor=J.+Sealauthor=S.+Taylorauthor=I.+Wallauthor=R.+J.+Watsonauthor=J.+Woolvenauthor=E.+H.+Demont&title=Design+and+synthesis+of+a+highly+selective+and+in+vivo-capable+inhibitor+of+the+second+bromodomain+of+the+bromodomain+and+extra+terminal+domain+family+of+proteins&doi=10.1021%2Facs.jmedchem.0c00605"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit71R"><div class="casContent"><span class="casTitleNuber">71</span><div class="casTitle"><span class="NLM_cas:atitle">Design and Synthesis of a Highly Selective and In Vivo-Capable Inhibitor of the Second Bromodomain of the Bromodomain and Extra Terminal Domain Family of Proteins</span></div><div class="casAuthors">Preston, Alex; Atkinson, Stephen; Bamborough, Paul; Chung, Chun-wa; Craggs, Peter D.; Gordon, Laurie; Grandi, Paola; Gray, James R. J.; Jones, Emma J.; Lindon, Matthew; Michon, Anne-Marie; Mitchell, Darren J.; Prinjha, Rab K.; Rianjongdee, Francesco; Rioja, Inmaculada; Seal, Jonathan; Taylor, Simon; Wall, Ian; Watson, Robert J.; Woolven, James; Demont, Emmanuel H.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">9070-9092</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Pan-bromodomain and extra terminal domain (BET) inhibitors interact equipotently with the eight bromodomains of the BET family of proteins and have shown profound efficacy in a no. of in vitro phenotypic assays and in vivo pre-clin. models in inflammation or oncol.  A no. of these inhibitors have progressed to the clinic where pharmacol.-driven adverse events have been reported.  To better understand the contribution of each domain to their efficacy and improve their safety profile, selective inhibitors are required.  This article discloses the profile of GSK046, also known as iBET-BD2(I), a highly selective inhibitor of the second bromodomains of the BET proteins that has undergone extensive pre-clin. in vitro and in vivo characterization.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpXn2qUNkpo27Vg90H21EOLACvtfcHk0lhuKgyydNmgnQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhsVWrsLrK&md5=08f4c47868e0a5acd5d443adba06d016</span></div><a href="/servlet/linkout?suffix=cit71&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.0c00605%26sid%3Dliteratum%253Aachs%26aulast%3DPreston%26aufirst%3DA.%26aulast%3DAtkinson%26aufirst%3DS.%26aulast%3DBamborough%26aufirst%3DP.%26aulast%3DChung%26aufirst%3DC.%2BW.%26aulast%3DCraggs%26aufirst%3DP.%2BD.%26aulast%3DGordon%26aufirst%3DL.%26aulast%3DGrandi%26aufirst%3DP.%26aulast%3DGray%26aufirst%3DJ.%26aulast%3DJones%26aufirst%3DE.%2BJ.%26aulast%3DLindon%26aufirst%3DM.%26aulast%3DMichon%26aufirst%3DA.%2BM.%26aulast%3DMitchell%26aufirst%3DD.%2BJ.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DRianjongdee%26aufirst%3DF.%26aulast%3DRioja%26aufirst%3DI.%26aulast%3DSeal%26aufirst%3DJ.%26aulast%3DTaylor%26aufirst%3DS.%26aulast%3DWall%26aufirst%3DI.%26aulast%3DWatson%26aufirst%3DR.%2BJ.%26aulast%3DWoolven%26aufirst%3DJ.%26aulast%3DDemont%26aufirst%3DE.%2BH.%26atitle%3DDesign%2520and%2520synthesis%2520of%2520a%2520highly%2520selective%2520and%2520in%2520vivo-capable%2520inhibitor%2520of%2520the%2520second%2520bromodomain%2520of%2520the%2520bromodomain%2520and%2520extra%2520terminal%2520domain%2520family%2520of%2520proteins%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D9070%26epage%3D9092%26doi%3D10.1021%2Facs.jmedchem.0c00605" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref72"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref72'); return false;" data-citation="" class="refNumLink">72</a></strong><div class="NLM_citation" id="cit72"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Seal, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Atkinson, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aylott, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bamborough, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Copley, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grandi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harrison, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayhow, T. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindon, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Messenger, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michon, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitchell, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Preston, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rioja, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wall, I. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watson, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woolven, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Demont, E. H.</span></span> <span> </span><span class="NLM_article-title">The optimization of a novel, weak bromo and extra terminal domain (BET) bromodomain fragment ligand to a potent and selective second bromodomain (BD2) inhibitor</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">9093</span>– <span class="NLM_lpage">9126</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.0c00796</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.0c00796" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhsVCmurzI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=9093-9126&author=J.+T.+Sealauthor=S.+J.+Atkinsonauthor=H.+Aylottauthor=P.+Bamboroughauthor=C.+W.+Chungauthor=R.+Copleyauthor=L.+Gordonauthor=P.+Grandiauthor=J.+Grayauthor=L.+A.+Harrisonauthor=T.+G.+Hayhowauthor=M.+Lindonauthor=C.+Messengerauthor=A.+M.+Michonauthor=D.+Mitchellauthor=A.+Prestonauthor=R.+K.+Prinjhaauthor=I.+Riojaauthor=S.+Taylorauthor=I.+D.+Wallauthor=R.+J.+Watsonauthor=J.+M.+Woolvenauthor=E.+H.+Demont&title=The+optimization+of+a+novel%2C+weak+bromo+and+extra+terminal+domain+%28BET%29+bromodomain+fragment+ligand+to+a+potent+and+selective+second+bromodomain+%28BD2%29+inhibitor&doi=10.1021%2Facs.jmedchem.0c00796"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit72R"><div class="casContent"><span class="casTitleNuber">72</span><div class="casTitle"><span class="NLM_cas:atitle">The Optimization of a Novel, Weak Bromo and Extra Terminal Domain (BET) Bromodomain Fragment Ligand to a Potent and Selective Second Bromodomain (BD2) Inhibitor</span></div><div class="casAuthors">Seal, Jonathan T.; Atkinson, Stephen J.; Aylott, Helen; Bamborough, Paul; Chung, Chun-wa; Copley, Royston C. B.; Gordon, Laurie; Grandi, Paola; Gray, James R. J.; Harrison, Lee A.; Hayhow, Thomas G.; Lindon, Matthew; Messenger, Cassie; Michon, Anne-Marie; Mitchell, Darren; Preston, Alex; Prinjha, Rab K.; Rioja, Inmaculada; Taylor, Simon; Wall, Ian D.; Watson, Robert J.; Woolven, James M.; Demont, Emmanuel H.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">9093-9126</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The profound efficacy, yet assocd. toxicity of pan-BET inhibitors is well documented.  The possibility of an ameliorated safety profile driven by significantly selective (>100-fold) inhibition of a subset of the eight bromodomains is enticing, but challenging given the close homol.  Herein, we describe the X-ray crystal structure-directed optimization of a novel weak fragment ligand with a pan-second bromodomain (BD2) bias, to potent and highly BD2 selective inhibitors.  A template hopping approach, enabled by our parallel research into an orthogonal template (15, GSK046, I), was the basis for the high selectivity obsd.  This culminated in two tool mols., 20 (GSK620) and 56 (GSK549), which showed an anti-inflammatory phenotype in human whole blood, confirming their cellular target engagement.  Excellent broad selectivity, developability, and in vivo oral pharmacokinetics characterize these tools, which we hope will be of broad utility to the field of epigenetics research.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoS04XaSgoDyLVg90H21EOLACvtfcHk0lgRr6qAirSoXg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhsVCmurzI&md5=c562a8c9e4c38abac04149ee96e2f594</span></div><a href="/servlet/linkout?suffix=cit72&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.0c00796%26sid%3Dliteratum%253Aachs%26aulast%3DSeal%26aufirst%3DJ.%2BT.%26aulast%3DAtkinson%26aufirst%3DS.%2BJ.%26aulast%3DAylott%26aufirst%3DH.%26aulast%3DBamborough%26aufirst%3DP.%26aulast%3DChung%26aufirst%3DC.%2BW.%26aulast%3DCopley%26aufirst%3DR.%26aulast%3DGordon%26aufirst%3DL.%26aulast%3DGrandi%26aufirst%3DP.%26aulast%3DGray%26aufirst%3DJ.%26aulast%3DHarrison%26aufirst%3DL.%2BA.%26aulast%3DHayhow%26aufirst%3DT.%2BG.%26aulast%3DLindon%26aufirst%3DM.%26aulast%3DMessenger%26aufirst%3DC.%26aulast%3DMichon%26aufirst%3DA.%2BM.%26aulast%3DMitchell%26aufirst%3DD.%26aulast%3DPreston%26aufirst%3DA.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DRioja%26aufirst%3DI.%26aulast%3DTaylor%26aufirst%3DS.%26aulast%3DWall%26aufirst%3DI.%2BD.%26aulast%3DWatson%26aufirst%3DR.%2BJ.%26aulast%3DWoolven%26aufirst%3DJ.%2BM.%26aulast%3DDemont%26aufirst%3DE.%2BH.%26atitle%3DThe%2520optimization%2520of%2520a%2520novel%252C%2520weak%2520bromo%2520and%2520extra%2520terminal%2520domain%2520%2528BET%2529%2520bromodomain%2520fragment%2520ligand%2520to%2520a%2520potent%2520and%2520selective%2520second%2520bromodomain%2520%2528BD2%2529%2520inhibitor%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D9093%26epage%3D9126%26doi%3D10.1021%2Facs.jmedchem.0c00796" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref73"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref73'); return false;" data-citation="" class="refNumLink">73</a></strong><div class="NLM_citation" id="cit73"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Slavish, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chi, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yun, M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsurkan, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martinez, N. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jonchere, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chai, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Connelly, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waddell, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Das, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neale, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shadrick, W. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olsen, R. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Freeman, K. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Low, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Price, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, B. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bharatham, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boyd, V. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morfouace, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roussel, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Savic, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guy, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">White, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shelat, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Potter, P. M.</span></span> <span> </span><span class="NLM_article-title">Bromodomain-selective BET inhibitors are potent antitumor agents against MYC-driven pediatric cancer</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>80</i></span>,  <span class="NLM_fpage">3507</span>– <span class="NLM_lpage">3518</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.CAN-19-3934</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1158%2F0008-5472.CAN-19-3934" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=32651255" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BB3cXitF2nu7vP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=80&publication_year=2020&pages=3507-3518&author=P.+J.+Slavishauthor=L.+Chiauthor=M.+K.+Yunauthor=L.+Tsurkanauthor=N.+E.+Martinezauthor=B.+Jonchereauthor=S.+C.+Chaiauthor=M.+Connellyauthor=M.+B.+Waddellauthor=S.+Dasauthor=G.+Nealeauthor=Z.+Liauthor=W.+R.+Shadrickauthor=R.+R.+Olsenauthor=K.+W.+Freemanauthor=J.+A.+Lowauthor=J.+E.+Priceauthor=B.+M.+Youngauthor=N.+Bharathamauthor=V.+A.+Boydauthor=J.+Yangauthor=R.+E.+Leeauthor=M.+Morfouaceauthor=M.+F.+Rousselauthor=T.+Chenauthor=D.+Savicauthor=R.+K.+Guyauthor=S.+W.+Whiteauthor=A.+A.+Shelatauthor=P.+M.+Potter&title=Bromodomain-selective+BET+inhibitors+are+potent+antitumor+agents+against+MYC-driven+pediatric+cancer&doi=10.1158%2F0008-5472.CAN-19-3934"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit73R"><div class="casContent"><span class="casTitleNuber">73</span><div class="casTitle"><span class="NLM_cas:atitle">Bromodomain-selective BET inhibitors are potent antitumor agents against MYC-driven pediatric cancer</span></div><div class="casAuthors">Slavish, P. Jake; Chi, Liying; Yun, Mi-Kyung; Tsurkan, Lyudmila; Martinez, Nancy E.; Jonchere, Barbara; Chai, Sergio C.; Connelly, Michele; Waddell, M. Brett; Das, Sourav; Neale, Geoffrey; Li, Zhenmei; Shadrick, William R.; Olsen, Rachelle R.; Freeman, Kevin W.; Low, Jonathan A.; Price, Jeanine E.; Young, Brandon M.; Bharatham, Nagakumar; Boyd, Vincent A.; Yang, Jun; Lee, Richard E.; Morfouace, Marie; Roussel, Martine F.; Chen, Taosheng; Savic, Daniel; Guy, R. Kiplin; White, Stephen W.; Shelat, Anang A.; Potter, Philip M.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">80</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">3507-3518</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Inhibition of members of the bromodomain and extraterminal (BET) family of proteins has proven a valid strategy for cancer chemotherapy.  All BET identified to date contain two bromodomains (BD; BD1 and BD2) that are necessary for recognition of acetylated lysine residues in the N-terminal regions of histones.  Chem. matter that targets BET (BETi) also interact via these domains.  Mol. and cellular data indicate that BD1 and BD2 have different biol. roles depending upon their cellular context, with BD2 particularly assocd. with cancer.  We have therefore pursued the development of BD2-selective mols. both as chem. probes and as potential leads for drug development.  Here we report the structure-based generation of a novel series of tetrahydroquinoline analogs that exhibit >50-fold selectivity for BD2 vs. BD1.  This selective targeting resulted in engagement with BD-contg. proteins in cells, resulting in modulation of MYC proteins and downstream targets.  These compds. were potent cytotoxins toward numerous pediatric cancer cell lines and were minimally toxic to nontumorigenic cells.  In addn., unlike the pan BETi (+)-JQ1, these BD2-selective inhibitors demonstrated no rebound expression effects.  Finally, we report a pharmacokinetic-optimized, metabolically stable deriv. that induced growth delay in a neuroblastoma xenograft model with minimal toxicity.  We conclude that BD2-selective agents are valid candidates for antitumor drug design for pediatric malignancies driven by the MYC oncogene.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGphIgrDNGwlb7Vg90H21EOLACvtfcHk0lgRr6qAirSoXg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXitF2nu7vP&md5=297abd22381a331adb3d68cd0b2aedea</span></div><a href="/servlet/linkout?suffix=cit73&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-19-3934&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-19-3934%26sid%3Dliteratum%253Aachs%26aulast%3DSlavish%26aufirst%3DP.%2BJ.%26aulast%3DChi%26aufirst%3DL.%26aulast%3DYun%26aufirst%3DM.%2BK.%26aulast%3DTsurkan%26aufirst%3DL.%26aulast%3DMartinez%26aufirst%3DN.%2BE.%26aulast%3DJonchere%26aufirst%3DB.%26aulast%3DChai%26aufirst%3DS.%2BC.%26aulast%3DConnelly%26aufirst%3DM.%26aulast%3DWaddell%26aufirst%3DM.%2BB.%26aulast%3DDas%26aufirst%3DS.%26aulast%3DNeale%26aufirst%3DG.%26aulast%3DLi%26aufirst%3DZ.%26aulast%3DShadrick%26aufirst%3DW.%2BR.%26aulast%3DOlsen%26aufirst%3DR.%2BR.%26aulast%3DFreeman%26aufirst%3DK.%2BW.%26aulast%3DLow%26aufirst%3DJ.%2BA.%26aulast%3DPrice%26aufirst%3DJ.%2BE.%26aulast%3DYoung%26aufirst%3DB.%2BM.%26aulast%3DBharatham%26aufirst%3DN.%26aulast%3DBoyd%26aufirst%3DV.%2BA.%26aulast%3DYang%26aufirst%3DJ.%26aulast%3DLee%26aufirst%3DR.%2BE.%26aulast%3DMorfouace%26aufirst%3DM.%26aulast%3DRoussel%26aufirst%3DM.%2BF.%26aulast%3DChen%26aufirst%3DT.%26aulast%3DSavic%26aufirst%3DD.%26aulast%3DGuy%26aufirst%3DR.%2BK.%26aulast%3DWhite%26aufirst%3DS.%2BW.%26aulast%3DShelat%26aufirst%3DA.%2BA.%26aulast%3DPotter%26aufirst%3DP.%2BM.%26atitle%3DBromodomain-selective%2520BET%2520inhibitors%2520are%2520potent%2520antitumor%2520agents%2520against%2520MYC-driven%2520pediatric%2520cancer%26jtitle%3DCancer%2520Res.%26date%3D2020%26volume%3D80%26spage%3D3507%26epage%3D3518%26doi%3D10.1158%2F0008-5472.CAN-19-3934" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref74"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref74'); return false;" data-citation="" class="refNumLink">74</a></strong><div class="NLM_citation" id="cit74"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Leroy, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rickards, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flint, S. J.</span></span> <span> </span><span class="NLM_article-title">The double bromodomain proteins BRD2 and BRD3 couple histone acetylation to transcription</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">51</span>– <span class="NLM_lpage">60</span>, <span class="refDoi"> DOI: 10.1016/j.molcel.2008.01.018</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1016%2Fj.molcel.2008.01.018" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=18406326" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BD1cXkvFegurk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2008&pages=51-60&author=G.+Leroyauthor=B.+Rickardsauthor=S.+J.+Flint&title=The+double+bromodomain+proteins+BRD2+and+BRD3+couple+histone+acetylation+to+transcription&doi=10.1016%2Fj.molcel.2008.01.018"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit74R"><div class="casContent"><span class="casTitleNuber">74</span><div class="casTitle"><span class="NLM_cas:atitle">The double bromodomain proteins Brd2 and Brd3 couple histone acetylation to transcription</span></div><div class="casAuthors">LeRoy, Gary; Rickards, Brenden; Flint, S. J.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">51-60</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Posttranslational histone modifications are crucial for the modulation of chromatin structure and regulation of transcription.  Bromodomains present in many chromatin-assocd. proteins recognize acetylated lysines in the unstructured N-terminal regions of histones.  Here, we report that the double bromodomain proteins Brd2 and Brd3 assoc. preferentially in vivo with hyperacetylated chromatin along the entire lengths of transcribed genes.  Brd2- and Brd3-assocd. chromatin is significantly enriched in H4K5, H4K12, and H3K14 acetylation and contains relatively little dimethylated H3K9.  Both Brd2 and Brd3 allowed RNA polymerase II to transcribe through nucleosomes in a defined transcription system.  Such activity depended on specific histone H4 modifications known to be recognized by the Brd proteins.  We also demonstrate that Brd2 has intrinsic histone chaperone activity and is required for transcription of the cyclin D1 gene in vivo.  These data identify proteins that render nucleosomes marked by acetylation permissive to the passage of elongating RNA polymerase II.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoNyXwkpnVoo7Vg90H21EOLACvtfcHk0ljfTeluyJ_0rQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXkvFegurk%253D&md5=63c9910684a1e43fe4ab1f6841443fc7</span></div><a href="/servlet/linkout?suffix=cit74&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2008.01.018&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2008.01.018%26sid%3Dliteratum%253Aachs%26aulast%3DLeroy%26aufirst%3DG.%26aulast%3DRickards%26aufirst%3DB.%26aulast%3DFlint%26aufirst%3DS.%2BJ.%26atitle%3DThe%2520double%2520bromodomain%2520proteins%2520BRD2%2520and%2520BRD3%2520couple%2520histone%2520acetylation%2520to%2520transcription%26jtitle%3DMol.%2520Cell%26date%3D2008%26volume%3D30%26spage%3D51%26epage%3D60%26doi%3D10.1016%2Fj.molcel.2008.01.018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref75"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref75'); return false;" data-citation="" class="refNumLink">75</a></strong><div class="NLM_citation" id="cit75"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Loven, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoke, H. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, C. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lau, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Orlando, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vakoc, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, T. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, R. A.</span></span> <span> </span><span class="NLM_article-title">Selective inhibition of tumor oncogenes by disruption of super-enhancers</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>153</i></span>,  <span class="NLM_fpage">320</span>– <span class="NLM_lpage">334</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2013.03.036</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1016%2Fj.cell.2013.03.036" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=23582323" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BC3sXlvVCgsrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=153&publication_year=2013&pages=320-334&author=J.+Lovenauthor=H.+A.+Hokeauthor=C.+Y.+Linauthor=A.+Lauauthor=D.+A.+Orlandoauthor=C.+R.+Vakocauthor=J.+E.+Bradnerauthor=T.+I.+Leeauthor=R.+A.+Young&title=Selective+inhibition+of+tumor+oncogenes+by+disruption+of+super-enhancers&doi=10.1016%2Fj.cell.2013.03.036"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit75R"><div class="casContent"><span class="casTitleNuber">75</span><div class="casTitle"><span class="NLM_cas:atitle">Selective Inhibition of Tumor Oncogenes by Disruption of Super-Enhancers</span></div><div class="casAuthors">Loven, Jakob; Hoke, Heather A.; Lin, Charles Y.; Lau, Ashley; Orlando, David A.; Vakoc, Christopher R.; Bradner, James E.; Lee, Tong Ihn; Young, Richard A.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">153</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">320-334</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Chromatin regulators have become attractive targets for cancer therapy, but it is unclear why inhibition of these ubiquitous regulators should have gene-specific effects in tumor cells.  Here, we investigate how inhibition of the widely expressed transcriptional coactivator BRD4 leads to selective inhibition of the MYC oncogene in multiple myeloma (MM).  BRD4 and Mediator were found to co-occupy thousands of enhancers assocd. with active genes.  They also co-occupied a small set of exceptionally large super-enhancers assocd. with genes that feature prominently in MM biol., including the MYC oncogene.  Treatment of MM tumor cells with the BET-bromodomain inhibitor JQ1 led to preferential loss of BRD4 at super-enhancers and consequent transcription elongation defects that preferentially impacted genes with super-enhancers, including MYC.  Super-enhancers were found at key oncogenic drivers in many other tumor cells.  These observations have implications for the discovery of cancer therapeutics directed at components of super-enhancers in diverse tumor types.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrM-NqFYuGMqrVg90H21EOLACvtfcHk0ljfTeluyJ_0rQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXlvVCgsrY%253D&md5=66b6ccb5b8a6039832910443fd8b4664</span></div><a href="/servlet/linkout?suffix=cit75&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2013.03.036&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2013.03.036%26sid%3Dliteratum%253Aachs%26aulast%3DLoven%26aufirst%3DJ.%26aulast%3DHoke%26aufirst%3DH.%2BA.%26aulast%3DLin%26aufirst%3DC.%2BY.%26aulast%3DLau%26aufirst%3DA.%26aulast%3DOrlando%26aufirst%3DD.%2BA.%26aulast%3DVakoc%26aufirst%3DC.%2BR.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26aulast%3DLee%26aufirst%3DT.%2BI.%26aulast%3DYoung%26aufirst%3DR.%2BA.%26atitle%3DSelective%2520inhibition%2520of%2520tumor%2520oncogenes%2520by%2520disruption%2520of%2520super-enhancers%26jtitle%3DCell%26date%3D2013%26volume%3D153%26spage%3D320%26epage%3D334%26doi%3D10.1016%2Fj.cell.2013.03.036" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref76"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref76'); return false;" data-citation="" class="refNumLink">76</a></strong><div class="NLM_citation" id="cit76"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Denis, G. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mccomb, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Faller, D. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sinha, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Romesser, P. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Costello, C. E.</span></span> <span> </span><span class="NLM_article-title">Identification of transcription complexes that contain the double bromodomain protein BRD2 and chromatin remodeling machines</span>. <i>J. Proteome Res.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">502</span>– <span class="NLM_lpage">511</span>, <span class="refDoi"> DOI: 10.1021/pr050430u</span> </span><div class="citationLinks">[<a href="/doi/10.1021/pr050430u" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtlSru7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2006&pages=502-511&author=G.+V.+Denisauthor=M.+E.+Mccombauthor=D.+V.+Fallerauthor=A.+Sinhaauthor=P.+B.+Romesserauthor=C.+E.+Costello&title=Identification+of+transcription+complexes+that+contain+the+double+bromodomain+protein+BRD2+and+chromatin+remodeling+machines&doi=10.1021%2Fpr050430u"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit76R"><div class="casContent"><span class="casTitleNuber">76</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of Transcription Complexes that Contain the Double Bromodomain Protein Brd2 and Chromatin Remodeling Machines</span></div><div class="casAuthors">Denis, Gerald V.; McComb, Mark E.; Faller, Douglas V.; Sinha, Anupama; Romesser, Paul B.; Costello, Catherine E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Proteome Research</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">502-511</span>CODEN:
                <span class="NLM_cas:coden">JPROBS</span>;
        ISSN:<span class="NLM_cas:issn">1535-3893</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We use affinity purifn. of the double bromodomain protein Brd2 to isolate a multicomponent nuclear complex from cultured cells, and apply mass spectrometry/proteomics methods to identify the participants.  We then confirm by immunoblot several transcription co-activators and co-repressors, proteins of the Swi/Snf chromatin remodeling complex, which regulate transcription control of cyclin A.  This multiprotein complex is likely to contribute to cell cycle control and play a role in proliferation and cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGooVmPLcNO2ILVg90H21EOLACvtfcHk0ljfTeluyJ_0rQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtlSru7Y%253D&md5=42184d6213946e9110843b36709a3c81</span></div><a href="/servlet/linkout?suffix=cit76&amp;dbid=16384&amp;doi=10.1021%2Fpr050430u&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fpr050430u%26sid%3Dliteratum%253Aachs%26aulast%3DDenis%26aufirst%3DG.%2BV.%26aulast%3DMccomb%26aufirst%3DM.%2BE.%26aulast%3DFaller%26aufirst%3DD.%2BV.%26aulast%3DSinha%26aufirst%3DA.%26aulast%3DRomesser%26aufirst%3DP.%2BB.%26aulast%3DCostello%26aufirst%3DC.%2BE.%26atitle%3DIdentification%2520of%2520transcription%2520complexes%2520that%2520contain%2520the%2520double%2520bromodomain%2520protein%2520BRD2%2520and%2520chromatin%2520remodeling%2520machines%26jtitle%3DJ.%2520Proteome%2520Res.%26date%3D2006%26volume%3D5%26spage%3D502%26epage%3D511%26doi%3D10.1021%2Fpr050430u" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref77"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref77'); return false;" data-citation="" class="refNumLink">77</a></strong><div class="NLM_citation" id="cit77"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lambert, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Picaud, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fujisawa, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hou, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Savitsky, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uuskula-Reimand, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gupta, G. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abdouni, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, Z. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tucholska, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knight, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gonzalez-Badillo, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">St-Denis, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Newman, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stucki, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pelletier, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bandeira, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gingras, A. C.</span></span> <span> </span><span class="NLM_article-title">Interactome rewiring following pharmacological targeting of BET bromodomains</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>73</i></span>,  <span class="NLM_fpage">621</span>– <span class="NLM_lpage">638</span>, <span class="refDoi"> DOI: 10.1016/j.molcel.2018.11.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1016%2Fj.molcel.2018.11.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=30554943" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BC1cXisFWqsbjO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=73&publication_year=2019&pages=621-638&author=J.+P.+Lambertauthor=S.+Picaudauthor=T.+Fujisawaauthor=H.+Houauthor=P.+Savitskyauthor=L.+Uuskula-Reimandauthor=G.+D.+Guptaauthor=H.+Abdouniauthor=Z.+Y.+Linauthor=M.+Tucholskaauthor=J.+Knightauthor=B.+Gonzalez-Badilloauthor=N.+St-Denisauthor=J.+A.+Newmanauthor=M.+Stuckiauthor=L.+Pelletierauthor=N.+Bandeiraauthor=M.+D.+Wilsonauthor=P.+Filippakopoulosauthor=A.+C.+Gingras&title=Interactome+rewiring+following+pharmacological+targeting+of+BET+bromodomains&doi=10.1016%2Fj.molcel.2018.11.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit77R"><div class="casContent"><span class="casTitleNuber">77</span><div class="casTitle"><span class="NLM_cas:atitle">Interactome Rewiring Following Pharmacological Targeting of BET Bromodomains</span></div><div class="casAuthors">Lambert, Jean-Philippe; Picaud, Sarah; Fujisawa, Takao; Hou, Huayun; Savitsky, Pavel; Uuskula-Reimand, Liis; Gupta, Gagan D.; Abdouni, Hala; Lin, Zhen-Yuan; Tucholska, Monika; Knight, James D. R.; Gonzalez-Badillo, Beatriz; St-Denis, Nicole; Newman, Joseph A.; Stucki, Manuel; Pelletier, Laurence; Bandeira, Nuno; Wilson, Michael D.; Filippakopoulos, Panagis; Gingras, Anne-Claude</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">73</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">621-638.e17</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Targeting bromodomains (BRDs) of the bromo-and-extra-terminal (BET) family offers opportunities for therapeutic intervention in cancer and other diseases.  Here, we profile the interactomes of BRD2, BRD3, BRD4, and BRDT following treatment with the pan-BET BRD inhibitor JQ1, revealing broad rewiring of the interaction landscape, with three distinct classes of behavior for the 603 unique interactors identified.  A group of proteins assoc. in a JQ1-sensitive manner with BET BRDs through canonical and new binding modes, while two classes of extra-terminal (ET)-domain binding motifs mediate acetylation-independent interactions.  Last, we identify an unexpected increase in several interactions following JQ1 treatment that define neg. functions for BRD3 in the regulation of rRNA synthesis and potentially RNAPII-dependent gene expression that result in decreased cell proliferation.  Together, our data highlight the contributions of BET protein modules to their interactomes allowing for a better understanding of pharmacol. rewiring in response to JQ1.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGojeAO2NIyKGLVg90H21EOLACvtfcHk0ljfTeluyJ_0rQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXisFWqsbjO&md5=70895d9ff04263449c67fbea254140b5</span></div><a href="/servlet/linkout?suffix=cit77&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2018.11.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2018.11.006%26sid%3Dliteratum%253Aachs%26aulast%3DLambert%26aufirst%3DJ.%2BP.%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DFujisawa%26aufirst%3DT.%26aulast%3DHou%26aufirst%3DH.%26aulast%3DSavitsky%26aufirst%3DP.%26aulast%3DUuskula-Reimand%26aufirst%3DL.%26aulast%3DGupta%26aufirst%3DG.%2BD.%26aulast%3DAbdouni%26aufirst%3DH.%26aulast%3DLin%26aufirst%3DZ.%2BY.%26aulast%3DTucholska%26aufirst%3DM.%26aulast%3DKnight%26aufirst%3DJ.%26aulast%3DGonzalez-Badillo%26aufirst%3DB.%26aulast%3DSt-Denis%26aufirst%3DN.%26aulast%3DNewman%26aufirst%3DJ.%2BA.%26aulast%3DStucki%26aufirst%3DM.%26aulast%3DPelletier%26aufirst%3DL.%26aulast%3DBandeira%26aufirst%3DN.%26aulast%3DWilson%26aufirst%3DM.%2BD.%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DGingras%26aufirst%3DA.%2BC.%26atitle%3DInteractome%2520rewiring%2520following%2520pharmacological%2520targeting%2520of%2520BET%2520bromodomains%26jtitle%3DMol.%2520Cell%26date%3D2019%26volume%3D73%26spage%3D621%26epage%3D638%26doi%3D10.1016%2Fj.molcel.2018.11.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref78"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref78'); return false;" data-citation="" class="refNumLink">78</a></strong><div class="NLM_citation" id="cit78"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Leroy, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rickards, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flint, S. J.</span></span> <span> </span><span class="NLM_article-title">The double bromodomain proteins BRD2 and BRD3 couple histone acetylation to transcription</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">51</span>– <span class="NLM_lpage">60</span>, <span class="refDoi"> DOI: 10.1016/j.molcel.2008.01.018</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1016%2Fj.molcel.2008.01.018" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=18406326" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BD1cXkvFegurk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2008&pages=51-60&author=G.+Leroyauthor=B.+Rickardsauthor=S.+J.+Flint&title=The+double+bromodomain+proteins+BRD2+and+BRD3+couple+histone+acetylation+to+transcription&doi=10.1016%2Fj.molcel.2008.01.018"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit78R"><div class="casContent"><span class="casTitleNuber">78</span><div class="casTitle"><span class="NLM_cas:atitle">The double bromodomain proteins Brd2 and Brd3 couple histone acetylation to transcription</span></div><div class="casAuthors">LeRoy, Gary; Rickards, Brenden; Flint, S. J.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">51-60</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Posttranslational histone modifications are crucial for the modulation of chromatin structure and regulation of transcription.  Bromodomains present in many chromatin-assocd. proteins recognize acetylated lysines in the unstructured N-terminal regions of histones.  Here, we report that the double bromodomain proteins Brd2 and Brd3 assoc. preferentially in vivo with hyperacetylated chromatin along the entire lengths of transcribed genes.  Brd2- and Brd3-assocd. chromatin is significantly enriched in H4K5, H4K12, and H3K14 acetylation and contains relatively little dimethylated H3K9.  Both Brd2 and Brd3 allowed RNA polymerase II to transcribe through nucleosomes in a defined transcription system.  Such activity depended on specific histone H4 modifications known to be recognized by the Brd proteins.  We also demonstrate that Brd2 has intrinsic histone chaperone activity and is required for transcription of the cyclin D1 gene in vivo.  These data identify proteins that render nucleosomes marked by acetylation permissive to the passage of elongating RNA polymerase II.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoNyXwkpnVoo7Vg90H21EOLACvtfcHk0ljRoERC1ckVHg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXkvFegurk%253D&md5=63c9910684a1e43fe4ab1f6841443fc7</span></div><a href="/servlet/linkout?suffix=cit78&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2008.01.018&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2008.01.018%26sid%3Dliteratum%253Aachs%26aulast%3DLeroy%26aufirst%3DG.%26aulast%3DRickards%26aufirst%3DB.%26aulast%3DFlint%26aufirst%3DS.%2BJ.%26atitle%3DThe%2520double%2520bromodomain%2520proteins%2520BRD2%2520and%2520BRD3%2520couple%2520histone%2520acetylation%2520to%2520transcription%26jtitle%3DMol.%2520Cell%26date%3D2008%26volume%3D30%26spage%3D51%26epage%3D60%26doi%3D10.1016%2Fj.molcel.2008.01.018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref79"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref79'); return false;" data-citation="" class="refNumLink">79</a></strong><div class="NLM_citation" id="cit79"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lamonica, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kadauke, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campbell, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gamsjaeger, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Billin, A. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hardison, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mackay, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blobel, G. A.</span></span> <span> </span><span class="NLM_article-title">Bromodomain protein BRD3 associates with acetylated GATA1 to promote its chromatin occupancy at erythroid target genes</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>108</i></span>,  <span class="NLM_fpage">E159</span>– <span class="NLM_lpage">E168</span>, <span class="refDoi"> DOI: 10.1073/pnas.1102140108</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1073%2Fpnas.1102140108" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=21536911" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A280%3ADC%252BC3MrmtlGgtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=108&publication_year=2011&pages=E159-E168&author=J.+M.+Lamonicaauthor=W.+Dengauthor=S.+Kadaukeauthor=A.+E.+Campbellauthor=R.+Gamsjaegerauthor=H.+Wangauthor=Y.+Chengauthor=A.+N.+Billinauthor=R.+C.+Hardisonauthor=J.+P.+Mackayauthor=G.+A.+Blobel&title=Bromodomain+protein+BRD3+associates+with+acetylated+GATA1+to+promote+its+chromatin+occupancy+at+erythroid+target+genes&doi=10.1073%2Fpnas.1102140108"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit79R"><div class="casContent"><span class="casTitleNuber">79</span><div class="casTitle"><span class="NLM_cas:atitle">Bromodomain protein Brd3 associates with acetylated GATA1 to promote its chromatin occupancy at erythroid target genes</span></div><div class="casAuthors">Lamonica Janine M; Deng Wulan; Kadauke Stephan; Campbell Amy E; Gamsjaeger Roland; Wang Hongxin; Cheng Yong; Billin Andrew N; Hardison Ross C; Mackay Joel P; Blobel Gerd A</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">108</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">E159-68</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Acetylation of histones triggers association with bromodomain-containing proteins that regulate diverse chromatin-related processes.  Although acetylation of transcription factors has been appreciated for some time, the mechanistic consequences are less well understood.  The hematopoietic transcription factor GATA1 is acetylated at conserved lysines that are required for its stable association with chromatin.  We show that the BET family protein Brd3 binds via its first bromodomain (BD1) to GATA1 in an acetylation-dependent manner in vitro and in vivo.  Mutation of a single residue in BD1 that is involved in acetyl-lysine binding abrogated recruitment of Brd3 by GATA1, demonstrating that acetylation of GATA1 is essential for Brd3 association with chromatin.  Notably, Brd3 is recruited by GATA1 to both active and repressed target genes in a fashion seemingly independent of histone acetylation.  Anti-Brd3 ChIP followed by massively parallel sequencing in GATA1-deficient erythroid precursor cells and those that are GATA1 replete revealed that GATA1 is a major determinant of Brd3 recruitment to genomic targets within chromatin.  A pharmacologic compound that occupies the acetyl-lysine binding pockets of Brd3 bromodomains disrupts the Brd3-GATA1 interaction, diminishes the chromatin occupancy of both proteins, and inhibits erythroid maturation.  Together these findings provide a mechanism for GATA1 acetylation and suggest that Brd3 "reads" acetyl marks on nuclear factors to promote their stable association with chromatin.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQeZ8VSdvvG7p6KlWarZTAzfW6udTcc2eZhc2BB4D1uILntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3MrmtlGgtQ%253D%253D&md5=b81056e4aee1351dce3899fb8d4e86e8</span></div><a href="/servlet/linkout?suffix=cit79&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1102140108&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1102140108%26sid%3Dliteratum%253Aachs%26aulast%3DLamonica%26aufirst%3DJ.%2BM.%26aulast%3DDeng%26aufirst%3DW.%26aulast%3DKadauke%26aufirst%3DS.%26aulast%3DCampbell%26aufirst%3DA.%2BE.%26aulast%3DGamsjaeger%26aufirst%3DR.%26aulast%3DWang%26aufirst%3DH.%26aulast%3DCheng%26aufirst%3DY.%26aulast%3DBillin%26aufirst%3DA.%2BN.%26aulast%3DHardison%26aufirst%3DR.%2BC.%26aulast%3DMackay%26aufirst%3DJ.%2BP.%26aulast%3DBlobel%26aufirst%3DG.%2BA.%26atitle%3DBromodomain%2520protein%2520BRD3%2520associates%2520with%2520acetylated%2520GATA1%2520to%2520promote%2520its%2520chromatin%2520occupancy%2520at%2520erythroid%2520target%2520genes%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2011%26volume%3D108%26spage%3DE159%26epage%3DE168%26doi%3D10.1073%2Fpnas.1102140108" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref80"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref80'); return false;" data-citation="" class="refNumLink">80</a></strong><div class="NLM_citation" id="cit80"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Matzuk, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mckeown, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Agno, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lemieux, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Picaud, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, R. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span> <span> </span><span class="NLM_article-title">Small-molecule inhibition of BRDT for male contraception</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>150</i></span>,  <span class="NLM_fpage">673</span>– <span class="NLM_lpage">684</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2012.06.045</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1016%2Fj.cell.2012.06.045" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=22901802" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BC38Xht1aktrbO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=150&publication_year=2012&pages=673-684&author=M.+M.+Matzukauthor=M.+R.+Mckeownauthor=P.+Filippakopoulosauthor=Q.+Liauthor=L.+Maauthor=J.+E.+Agnoauthor=M.+E.+Lemieuxauthor=S.+Picaudauthor=R.+N.+Yuauthor=J.+Qiauthor=S.+Knappauthor=J.+E.+Bradner&title=Small-molecule+inhibition+of+BRDT+for+male+contraception&doi=10.1016%2Fj.cell.2012.06.045"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit80R"><div class="casContent"><span class="casTitleNuber">80</span><div class="casTitle"><span class="NLM_cas:atitle">Small-Molecule Inhibition of BRDT for Male Contraception</span></div><div class="casAuthors">Matzuk, Martin M.; McKeown, Michael R.; Filippakopoulos, Panagis; Li, Qinglei; Ma, Lang; Agno, Julio E.; Lemieux, Madeleine E.; Picaud, Sarah; Yu, Richard N.; Qi, Jun; Knapp, Stefan; Bradner, James E.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">150</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">673-684</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A pharmacol. approach to male contraception remains a longstanding challenge in medicine.  Toward this objective, the authors explored the spermatogenic effects of a selective small-mol. inhibitor (JQ1) of the bromodomain and extraterminal (BET) subfamily of epigenetic reader proteins.  Here, the authors report potent inhibition of the testis-specific member BRDT, which is essential for chromatin remodeling during spermatogenesis.  Biochem. and crystallog. studies confirm that occupancy of the BRDT acetyl-lysine binding pocket by JQ1 prevents recognition of acetylated histone H4.  Treatment of mice with JQ1 reduced seminiferous tubule area, testis size, and spermatozoa no. and motility without affecting hormone levels.  Although JQ1-treated males mate normally, inhibitory effects of JQ1 evident at the spermatocyte and round spermatid stages cause a complete and reversible contraceptive effect.  These data establish a new contraceptive that can cross the blood:testis boundary and inhibit bromodomain activity during spermatogenesis, providing a lead compd. targeting the male germ cell for contraception.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqlxVWUgXTC77Vg90H21EOLACvtfcHk0lgxFq2N7m4WMw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xht1aktrbO&md5=ccfa1b443c156612714e4b7f88590041</span></div><a href="/servlet/linkout?suffix=cit80&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2012.06.045&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2012.06.045%26sid%3Dliteratum%253Aachs%26aulast%3DMatzuk%26aufirst%3DM.%2BM.%26aulast%3DMckeown%26aufirst%3DM.%2BR.%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DLi%26aufirst%3DQ.%26aulast%3DMa%26aufirst%3DL.%26aulast%3DAgno%26aufirst%3DJ.%2BE.%26aulast%3DLemieux%26aufirst%3DM.%2BE.%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DYu%26aufirst%3DR.%2BN.%26aulast%3DQi%26aufirst%3DJ.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26atitle%3DSmall-molecule%2520inhibition%2520of%2520BRDT%2520for%2520male%2520contraception%26jtitle%3DCell%26date%3D2012%26volume%3D150%26spage%3D673%26epage%3D684%26doi%3D10.1016%2Fj.cell.2012.06.045" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref81"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref81'); return false;" data-citation="" class="refNumLink">81</a></strong><div class="NLM_citation" id="cit81"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bryant, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berger, S. L.</span></span> <span> </span><span class="NLM_article-title">Low-hanging fruit: targeting BRDT in the testes</span>. <i>EMBO J.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>31</i></span>,  <span class="NLM_fpage">3788</span>– <span class="NLM_lpage">3789</span>, <span class="refDoi"> DOI: 10.1038/emboj.2012.259</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1038%2Femboj.2012.259" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=22960636" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtlWgu7rP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2012&pages=3788-3789&author=J.+M.+Bryantauthor=S.+L.+Berger&title=Low-hanging+fruit%3A+targeting+BRDT+in+the+testes&doi=10.1038%2Femboj.2012.259"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit81R"><div class="casContent"><span class="casTitleNuber">81</span><div class="casTitle"><span class="NLM_cas:atitle">Low-hanging fruit: targeting Brdt in the testes</span></div><div class="casAuthors">Bryant, Jessica M.; Berger, Shelley L.</div><div class="citationInfo"><span class="NLM_cas:title">EMBO Journal</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">3788-3789</span>CODEN:
                <span class="NLM_cas:coden">EMJODG</span>;
        ISSN:<span class="NLM_cas:issn">0261-4189</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The title research of Gaucher, J. et al. (ibid, advance online publication, 24 August 2012; doi:10.1038/emboj.2012.233) and Matzuk, M. M. et al. (Cell, 2012, 150:pg. 673-684) is reviewed with commentary.  In this issue of The EMBO Journal, Gaucher et al. (2012) demonstrate that Brdt, a testes-specific member of the BET family of double bromodomain proteins, is intricately involved in the transcriptional regulation and genome reorganization required for the prodn. of viable, fertile sperm.  These findings are complimented by a recent publication in Cell (Matzuk et al, 2012) where JQ1, a small mol. BET inhibitor, is effective as a male contraceptive in mice.  Both studies establish Brdt as an integral factor in sperm development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGouzkYXqg4y87Vg90H21EOLACvtfcHk0lgxFq2N7m4WMw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtlWgu7rP&md5=0d779b87988844846f2549e375a16230</span></div><a href="/servlet/linkout?suffix=cit81&amp;dbid=16384&amp;doi=10.1038%2Femboj.2012.259&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Femboj.2012.259%26sid%3Dliteratum%253Aachs%26aulast%3DBryant%26aufirst%3DJ.%2BM.%26aulast%3DBerger%26aufirst%3DS.%2BL.%26atitle%3DLow-hanging%2520fruit%253A%2520targeting%2520BRDT%2520in%2520the%2520testes%26jtitle%3DEMBO%2520J.%26date%3D2012%26volume%3D31%26spage%3D3788%26epage%3D3789%26doi%3D10.1038%2Femboj.2012.259" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref82"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref82'); return false;" data-citation="" class="refNumLink">82</a></strong><div class="NLM_citation" id="cit82"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tian, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brasier, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, J.</span></span> <span> </span><span class="NLM_article-title">Discovery of potent and selective BRD4 inhibitors capable of blocking TLR3-induced acute airway inflammation</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>151</i></span>,  <span class="NLM_fpage">450</span>– <span class="NLM_lpage">461</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2018.04.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1016%2Fj.ejmech.2018.04.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=29649741" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BC1cXnt1KgsLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=151&publication_year=2018&pages=450-461&author=Z.+Liuauthor=B.+Tianauthor=H.+Chenauthor=P.+Wangauthor=A.+R.+Brasierauthor=J.+Zhou&title=Discovery+of+potent+and+selective+BRD4+inhibitors+capable+of+blocking+TLR3-induced+acute+airway+inflammation&doi=10.1016%2Fj.ejmech.2018.04.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit82R"><div class="casContent"><span class="casTitleNuber">82</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of potent and selective BRD4 inhibitors capable of blocking TLR3-induced acute airway inflammation</span></div><div class="casAuthors">Liu, Zhiqing; Tian, Bing; Chen, Haiying; Wang, Pingyuan; Brasier, Allan R.; Zhou, Jia</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">151</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">450-461</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">A series of diverse small mols. have been designed and synthesized through structure-based drug design by taking advantage of fragment merging and elaboration approaches.  Compds. ZL0420 (28) and ZL0454 (35) were identified as potent and selective BRD4 inhibitors with nanomolar binding affinities to bromodomains (BDs) of BRD4.  Both of them can be well docked into the acetyl-lysine (KAc) binding pocket of BRD4, forming key interactions including the crit. hydrogen bonds with Asn140 directly and Tyr97 indirectly via a H2O mol.  Both compds. 28 and 35 exhibited submicromolar potency of inhibiting the TLR3-dependent innate immune gene program, including ISG54, ISG56, IL-8, and Groβ genes in cultured human small airway epithelial cells (hSAECs).  More importantly, they also demonstrated potent efficacy reducing airway inflammation in a mouse model with low toxicity, indicating a proof of concept that BRD4 inhibitors may offer the therapeutic potential to block the viral-induced airway inflammation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqLyejdKuiGi7Vg90H21EOLACvtfcHk0lgxFq2N7m4WMw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXnt1KgsLc%253D&md5=564b088a350abef9964d97bbecbb52b1</span></div><a href="/servlet/linkout?suffix=cit82&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2018.04.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2018.04.006%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DZ.%26aulast%3DTian%26aufirst%3DB.%26aulast%3DChen%26aufirst%3DH.%26aulast%3DWang%26aufirst%3DP.%26aulast%3DBrasier%26aufirst%3DA.%2BR.%26aulast%3DZhou%26aufirst%3DJ.%26atitle%3DDiscovery%2520of%2520potent%2520and%2520selective%2520BRD4%2520inhibitors%2520capable%2520of%2520blocking%2520TLR3-induced%2520acute%2520airway%2520inflammation%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2018%26volume%3D151%26spage%3D450%26epage%3D461%26doi%3D10.1016%2Fj.ejmech.2018.04.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref83"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref83'); return false;" data-citation="" class="refNumLink">83</a></strong><div class="NLM_citation" id="cit83"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lv, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mou, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fan, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pei, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiong, B.</span></span> <span> </span><span class="NLM_article-title">Rational design and evaluation of 6-(Pyrimidin-2-ylamino)-3,4-dihydroquinoxalin-2(1H)-ones as polypharmacological inhibitors of BET and kinases</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">9787</span>– <span class="NLM_lpage">9802</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.0c00962</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.0c00962" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref83/cit83&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhsFOisLvN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=9787-9802&author=K.+Lvauthor=W.+Chenauthor=D.+Chenauthor=J.+Mouauthor=H.+Zhangauthor=T.+Fanauthor=Y.+Liauthor=D.+Caoauthor=X.+Wangauthor=L.+Chenauthor=J.+Shenauthor=D.+Peiauthor=B.+Xiong&title=Rational+design+and+evaluation+of+6-%28Pyrimidin-2-ylamino%29-3%2C4-dihydroquinoxalin-2%281H%29-ones+as+polypharmacological+inhibitors+of+BET+and+kinases&doi=10.1021%2Facs.jmedchem.0c00962"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit83R"><div class="casContent"><span class="casTitleNuber">83</span><div class="casTitle"><span class="NLM_cas:atitle">Rational Design and Evaluation of 6-(Pyrimidin-2-ylamino)-3,4-dihydroquinoxalin-2(1H)-ones as Polypharmacological Inhibitors of BET and Kinases</span></div><div class="casAuthors">Lv, Kaikai; Chen, Weicong; Chen, Danqi; Mou, Jie; Zhang, Huijie; Fan, Tiantian; Li, Yanlian; Cao, Danyan; Wang, Xin; Chen, Lin; Shen, Jingkang; Pei, Dongsheng; Xiong, Bing</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">9787-9802</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Cancer exhibits diverse heterogeneity with a complicated mol. basis that usually harbors genetic and epigenetic abnormality, which poses a big challenge for single-target agents.  In the current work, we proposed a hybrid strategy by incorporating pharmacophores that bind to the acetylated lysine binding pocket of BET proteins with a typical kinase hinge binder to generate novel polypharmacol. inhibitors of BET and kinases.  Through elaborating the core structure of 6-(pyrimidin-2-ylamino)-3,4-dihydroquinoxalin-2(1H)-one, we demonstrated that this rational design can produce high potent inhibitors of CDK9 and BET proteins.  In this series, compd. 40(I) was identified as the potential lead compd. with balanced activities of BRD4 (IC50 = 12.7 nM) and CDK9 (IC50 = 22.4 nM), as well as good antiproliferative activities on a small cancer cell panel.  Together, the current study provided a new method for the discovery of bromodomain and kinase dual inhibitors rather than only being discovered by serendipity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGquGRr7R1VS9LVg90H21EOLACvtfcHk0lgZhj3mK-TQbg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhsFOisLvN&md5=4305f1e5c45c8e0dd994b042a4ee3bb7</span></div><a href="/servlet/linkout?suffix=cit83&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.0c00962&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.0c00962%26sid%3Dliteratum%253Aachs%26aulast%3DLv%26aufirst%3DK.%26aulast%3DChen%26aufirst%3DW.%26aulast%3DChen%26aufirst%3DD.%26aulast%3DMou%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DH.%26aulast%3DFan%26aufirst%3DT.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DCao%26aufirst%3DD.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DChen%26aufirst%3DL.%26aulast%3DShen%26aufirst%3DJ.%26aulast%3DPei%26aufirst%3DD.%26aulast%3DXiong%26aufirst%3DB.%26atitle%3DRational%2520design%2520and%2520evaluation%2520of%25206-%2528Pyrimidin-2-ylamino%2529-3%252C4-dihydroquinoxalin-2%25281H%2529-ones%2520as%2520polypharmacological%2520inhibitors%2520of%2520BET%2520and%2520kinases%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D9787%26epage%3D9802%26doi%3D10.1021%2Facs.jmedchem.0c00962" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref84"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref84'); return false;" data-citation="" class="refNumLink">84</a></strong><div class="NLM_citation" id="cit84"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Martin, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olesen, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Georg, G. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schonbrunn, E.</span></span> <span> </span><span class="NLM_article-title">Cyclin-dependent kinase inhibitor dinaciclib interacts with the acetyl-lysine recognition site of bromodomains</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">2360</span>– <span class="NLM_lpage">2365</span>, <span class="refDoi"> DOI: 10.1021/cb4003283</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb4003283" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref84/cit84&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtl2rtbvN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&pages=2360-2365&author=M.+P.+Martinauthor=S.+H.+Olesenauthor=G.+I.+Georgauthor=E.+Schonbrunn&title=Cyclin-dependent+kinase+inhibitor+dinaciclib+interacts+with+the+acetyl-lysine+recognition+site+of+bromodomains&doi=10.1021%2Fcb4003283"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit84R"><div class="casContent"><span class="casTitleNuber">84</span><div class="casTitle"><span class="NLM_cas:atitle">Cyclin-Dependent Kinase Inhibitor Dinaciclib Interacts with the Acetyl-Lysine Recognition Site of Bromodomains</span></div><div class="casAuthors">Martin, Mathew P.; Olesen, Sanne H.; Georg, Gunda I.; Schonbrunn, Ernst</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2360-2365</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Bromodomain-contg. proteins are considered atypical kinases, but their potential to interact with kinase inhibitors is unknown.  Dinaciclib is a potent inhibitor of cyclin-dependent kinases (CDKs), which recently advanced to Phase III clin. trials for the treatment of leukemia.  The authors detd. the crystal structure of dinaciclib in complex with CDK2 at 1.7 Å resoln., revealing an elaborate network of binding interactions in the ATP site, which explains the extraordinary potency and selectivity of this inhibitor.  Remarkably, dinaciclib also interacted with the acetyl-lysine recognition site of the bromodomain testis-specific protein BRDT, a member of the BET family of bromodomains.  The binding mode of dinaciclib to BRDT at 2.0 Å resoln. suggests that general kinase inhibitors (hinge binders) possess a previously unrecognized potential to act as protein-protein inhibitors of bromodomains.  The findings may provide a new structural framework for the design of next-generation bromodomain inhibitors using the vast chem. space of kinase inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGonrxl9QUxkE7Vg90H21EOLACvtfcHk0lgZhj3mK-TQbg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtl2rtbvN&md5=1ae089e8e571ad11aadcdc4a1a5769b5</span></div><a href="/servlet/linkout?suffix=cit84&amp;dbid=16384&amp;doi=10.1021%2Fcb4003283&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb4003283%26sid%3Dliteratum%253Aachs%26aulast%3DMartin%26aufirst%3DM.%2BP.%26aulast%3DOlesen%26aufirst%3DS.%2BH.%26aulast%3DGeorg%26aufirst%3DG.%2BI.%26aulast%3DSchonbrunn%26aufirst%3DE.%26atitle%3DCyclin-dependent%2520kinase%2520inhibitor%2520dinaciclib%2520interacts%2520with%2520the%2520acetyl-lysine%2520recognition%2520site%2520of%2520bromodomains%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2013%26volume%3D8%26spage%3D2360%26epage%3D2365%26doi%3D10.1021%2Fcb4003283" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref85"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref85'); return false;" data-citation="" class="refNumLink">85</a></strong><div class="NLM_citation" id="cit85"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Carlino, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rastelli, G.</span></span> <span> </span><span class="NLM_article-title">Dual kinase-bromodomain inhibitors in anticancer drug discovery: a structural and pharmacological perspective</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">9305</span>– <span class="NLM_lpage">9320</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b00438</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b00438" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref85/cit85&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsVWnt7rP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=9305-9320&author=L.+Carlinoauthor=G.+Rastelli&title=Dual+kinase-bromodomain+inhibitors+in+anticancer+drug+discovery%3A+a+structural+and+pharmacological+perspective&doi=10.1021%2Facs.jmedchem.6b00438"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit85R"><div class="casContent"><span class="casTitleNuber">85</span><div class="casTitle"><span class="NLM_cas:atitle">Dual Kinase-Bromodomain Inhibitors in Anticancer Drug Discovery: A Structural and Pharmacological Perspective</span></div><div class="casAuthors">Carlino, Luca; Rastelli, Giulio</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">9305-9320</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Protein kinases play crucial roles in several cell transformation processes and are validated drug targets for many human diseases, including cancer.  Nevertheless, most tumors have eluded the effects of inhibition of a single kinase by activating resistance mechanisms and/or alternative pathways and escape mechanisms.  In recent years, multitarget approaches directed toward inhibition of kinases and targets of different families have received increasing attention.  In particular, co-targeting kinases and bromodomain epigenetic reader proteins has rapidly emerged as a promising approach to cancer drug development.  In this manuscript, we will review the recent discoveries that led to the identification and optimization of dual kinase/bromodomain inhibitors.  We will analyze and compare the structural features required for dual inhibition and comment on the potential of this approach in anticancer drug discovery.  Moreover, we will introduce computational approaches useful for the identification of dual kinase/bromodomain inhibitors and generate ad hoc pharmacophore and docking models.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqTkHDSBWCGdbVg90H21EOLACvtfcHk0lgZhj3mK-TQbg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsVWnt7rP&md5=1351a235bba09782160a86ae3f904833</span></div><a href="/servlet/linkout?suffix=cit85&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b00438&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b00438%26sid%3Dliteratum%253Aachs%26aulast%3DCarlino%26aufirst%3DL.%26aulast%3DRastelli%26aufirst%3DG.%26atitle%3DDual%2520kinase-bromodomain%2520inhibitors%2520in%2520anticancer%2520drug%2520discovery%253A%2520a%2520structural%2520and%2520pharmacological%2520perspective%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D9305%26epage%3D9320%26doi%3D10.1021%2Facs.jmedchem.6b00438" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref86"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref86'); return false;" data-citation="" class="refNumLink">86</a></strong><div class="NLM_citation" id="cit86"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sun, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ouyang, L.</span></span> <span> </span><span class="NLM_article-title">Dual-target kinase drug design: current strategies and future directions in cancer therapy</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>188</i></span>,  <span class="NLM_fpage">112025</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2019.112025</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref86/cit86&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1016%2Fj.ejmech.2019.112025" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref86/cit86&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=31931340" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref86/cit86&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhtFCjtLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=188&publication_year=2020&pages=112025&author=D.+Sunauthor=Y.+Zhaoauthor=S.+Zhangauthor=L.+Zhangauthor=B.+Liuauthor=L.+Ouyang&title=Dual-target+kinase+drug+design%3A+current+strategies+and+future+directions+in+cancer+therapy&doi=10.1016%2Fj.ejmech.2019.112025"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit86R"><div class="casContent"><span class="casTitleNuber">86</span><div class="casTitle"><span class="NLM_cas:atitle">Dual-target kinase drug design: Current strategies and future directions in cancer therapy</span></div><div class="casAuthors">Sun, Dejuan; Zhao, Yuqian; Zhang, Shouyue; Zhang, Lan; Liu, Bo; Ouyang, Liang</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">188</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">112025</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">A review.  Protein kinases are well-known to orchestrate the activation of signaling cascades in response to extracellular and intracellular stimuli to control cell proliferation and survival.  The perturbation of such kinases by some mutation or abnormal protein expressions has been closely linked to cancer.  Drug development aimed at several targetable kinases may alter their participated pathways that are able to trigger carcinogenesis.  A series of small-mol. drugs have been approved for the current cancer therapy.  However, their complicated inherent mechanisms may lead to the resistance to such small mols.  Consequently, development of new dual-target kinase drugs simultaneously aimed at two targetable kinases may improve their anti-tumor efficiency and solve resistant mechanism problems.  In this review, we focus on summarizing an overview of the current strategies of dual-target kinase drug design, including medicinal chem. strategies and computational approaches.  Taken together, we believe the above-mentioned strategies will provide a new insight into future directions of dual-target kinase drug design to improve potential cancer therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoL4dbUqruISrVg90H21EOLACvtfcHk0lgZhj3mK-TQbg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhtFCjtLg%253D&md5=7b4830183b0b7045331f6d62ded69b1d</span></div><a href="/servlet/linkout?suffix=cit86&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2019.112025&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2019.112025%26sid%3Dliteratum%253Aachs%26aulast%3DSun%26aufirst%3DD.%26aulast%3DZhao%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DS.%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DLiu%26aufirst%3DB.%26aulast%3DOuyang%26aufirst%3DL.%26atitle%3DDual-target%2520kinase%2520drug%2520design%253A%2520current%2520strategies%2520and%2520future%2520directions%2520in%2520cancer%2520therapy%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2020%26volume%3D188%26spage%3D112025%26doi%3D10.1016%2Fj.ejmech.2019.112025" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref87"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref87'); return false;" data-citation="" class="refNumLink">87</a></strong><div class="NLM_citation" id="cit87"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mazur, P. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herner, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mello, S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wirth, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hausmann, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanchez-Rivera, F. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lofgren, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuschma, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hahn, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vangala, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trajkovic-Arsic, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gupta, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heid, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Noel, P. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Braren, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Erkan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kleeff, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sipos, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sayles, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heikenwalder, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hessmann, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ellenrieder, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Esposito, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacks, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khatri, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sweet-Cordero, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Attardi, L. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmid, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schneider, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sage, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siveke, J. T.</span></span> <span> </span><span class="NLM_article-title">Combined inhibition of BET family proteins and histone deacetylases as a potential epigenetics-based therapy for pancreatic ductal adenocarcinoma</span>. <i>Nat. Med.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">1163</span>– <span class="NLM_lpage">1171</span>, <span class="refDoi"> DOI: 10.1038/nm.3952</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref87/cit87&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1038%2Fnm.3952" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref87/cit87&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=26390243" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref87/cit87&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsFersL%252FE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2015&pages=1163-1171&author=P.+K.+Mazurauthor=A.+Hernerauthor=S.+S.+Melloauthor=M.+Wirthauthor=S.+Hausmannauthor=F.+J.+Sanchez-Riveraauthor=S.+M.+Lofgrenauthor=T.+Kuschmaauthor=S.+A.+Hahnauthor=D.+Vangalaauthor=M.+Trajkovic-Arsicauthor=A.+Guptaauthor=I.+Heidauthor=P.+B.+Noelauthor=R.+Brarenauthor=M.+Erkanauthor=J.+Kleeffauthor=B.+Siposauthor=L.+C.+Saylesauthor=M.+Heikenwalderauthor=E.+Hessmannauthor=V.+Ellenriederauthor=I.+Espositoauthor=T.+Jacksauthor=J.+E.+Bradnerauthor=P.+Khatriauthor=E.+A.+Sweet-Corderoauthor=L.+D.+Attardiauthor=R.+M.+Schmidauthor=G.+Schneiderauthor=J.+Sageauthor=J.+T.+Siveke&title=Combined+inhibition+of+BET+family+proteins+and+histone+deacetylases+as+a+potential+epigenetics-based+therapy+for+pancreatic+ductal+adenocarcinoma&doi=10.1038%2Fnm.3952"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit87R"><div class="casContent"><span class="casTitleNuber">87</span><div class="casTitle"><span class="NLM_cas:atitle">Combined inhibition of BET family proteins and histone deacetylases as a potential epigenetics-based therapy for pancreatic ductal adenocarcinoma</span></div><div class="casAuthors">Mazur, Pawel K.; Herner, Alexander; Mello, Stephano S.; Wirth, Matthias; Hausmann, Simone; Sanchez-Rivera, Francisco J.; Lofgren, Shane M.; Kuschma, Timo; Hahn, Stephan A.; Vangala, Deepak; Trajkovic-Arsic, Marija; Gupta, Aayush; Heid, Irina; Noel, Peter B.; Braren, Rickmer; Erkan, Mert; Kleeff, Jorg; Sipos, Bence; Sayles, Leanne C.; Heikenwalder, Mathias; Hessmann, Elisabeth; Ellenrieder, Volker; Esposito, Irene; Jacks, Tyler; Bradner, James E.; Khatri, Purvesh; Sweet-Cordero, E. Alejandro; Attardi, Laura D.; Schmid, Roland M.; Schneider, Guenter; Sage, Julien; Siveke, Jens T.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Medicine (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1163-1171</span>CODEN:
                <span class="NLM_cas:coden">NAMEFI</span>;
        ISSN:<span class="NLM_cas:issn">1078-8956</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Pancreatic ductal adenocarcinoma (PDAC) is one of the most lethal human cancers and shows resistance to any therapeutic strategy used.  Here the authors tested small-mol. inhibitors targeting chromatin regulators as possible therapeutic agents in PDAC.  The authors show that JQ1, an inhibitor of the bromodomain and extraterminal (BET) family of proteins, suppresses PDAC development in mice by inhibiting both MYC activity and inflammatory signals.  The histone deacetylase (HDAC) inhibitor SAHA synergizes with JQ1 to augment cell death and more potently suppress advanced PDAC.  Finally, using a CRISPR-Cas9-based method for gene editing directly in the mouse adult pancreas, the authors show that de-repression of p57 (also known as KIP2 or CDKN1C) upon combined BET and HDAC inhibition is required for the induction of combination therapy-induced cell death in PDAC.  SAHA is approved for human use, and mols. similar to JQ1 are being tested in clin. trials.  Thus, these studies identify a promising epigenetic-based therapeutic strategy that may be rapidly implemented in fatal human tumors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqmoDwfBKhuTrVg90H21EOLACvtfcHk0liVSwQhDvyWCw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsFersL%252FE&md5=d9acebe4751ac1fd33f0297e5134ee40</span></div><a href="/servlet/linkout?suffix=cit87&amp;dbid=16384&amp;doi=10.1038%2Fnm.3952&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnm.3952%26sid%3Dliteratum%253Aachs%26aulast%3DMazur%26aufirst%3DP.%2BK.%26aulast%3DHerner%26aufirst%3DA.%26aulast%3DMello%26aufirst%3DS.%2BS.%26aulast%3DWirth%26aufirst%3DM.%26aulast%3DHausmann%26aufirst%3DS.%26aulast%3DSanchez-Rivera%26aufirst%3DF.%2BJ.%26aulast%3DLofgren%26aufirst%3DS.%2BM.%26aulast%3DKuschma%26aufirst%3DT.%26aulast%3DHahn%26aufirst%3DS.%2BA.%26aulast%3DVangala%26aufirst%3DD.%26aulast%3DTrajkovic-Arsic%26aufirst%3DM.%26aulast%3DGupta%26aufirst%3DA.%26aulast%3DHeid%26aufirst%3DI.%26aulast%3DNoel%26aufirst%3DP.%2BB.%26aulast%3DBraren%26aufirst%3DR.%26aulast%3DErkan%26aufirst%3DM.%26aulast%3DKleeff%26aufirst%3DJ.%26aulast%3DSipos%26aufirst%3DB.%26aulast%3DSayles%26aufirst%3DL.%2BC.%26aulast%3DHeikenwalder%26aufirst%3DM.%26aulast%3DHessmann%26aufirst%3DE.%26aulast%3DEllenrieder%26aufirst%3DV.%26aulast%3DEsposito%26aufirst%3DI.%26aulast%3DJacks%26aufirst%3DT.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26aulast%3DKhatri%26aufirst%3DP.%26aulast%3DSweet-Cordero%26aufirst%3DE.%2BA.%26aulast%3DAttardi%26aufirst%3DL.%2BD.%26aulast%3DSchmid%26aufirst%3DR.%2BM.%26aulast%3DSchneider%26aufirst%3DG.%26aulast%3DSage%26aufirst%3DJ.%26aulast%3DSiveke%26aufirst%3DJ.%2BT.%26atitle%3DCombined%2520inhibition%2520of%2520BET%2520family%2520proteins%2520and%2520histone%2520deacetylases%2520as%2520a%2520potential%2520epigenetics-based%2520therapy%2520for%2520pancreatic%2520ductal%2520adenocarcinoma%26jtitle%3DNat.%2520Med.%26date%3D2015%26volume%3D21%26spage%3D1163%26epage%3D1171%26doi%3D10.1038%2Fnm.3952" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref88"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref88'); return false;" data-citation="" class="refNumLink">88</a></strong><div class="NLM_citation" id="cit88"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vakoc, C. R.</span></span> <span> </span><span class="NLM_article-title">The mechanisms behind the therapeutic activity of BET bromodomain inhibition</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">728</span>– <span class="NLM_lpage">736</span>, <span class="refDoi"> DOI: 10.1016/j.molcel.2014.05.016</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref88/cit88&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1016%2Fj.molcel.2014.05.016" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref88/cit88&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=24905006" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref88/cit88&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BC2cXpslCrtrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2014&pages=728-736&author=J.+Shiauthor=C.+R.+Vakoc&title=The+mechanisms+behind+the+therapeutic+activity+of+BET+bromodomain+inhibition&doi=10.1016%2Fj.molcel.2014.05.016"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit88R"><div class="casContent"><span class="casTitleNuber">88</span><div class="casTitle"><span class="NLM_cas:atitle">The Mechanisms behind the Therapeutic Activity of BET Bromodomain Inhibition</span></div><div class="casAuthors">Shi, Junwei; Vakoc, Christopher R.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">728-736</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">A review.  The bromodomain and extraterminal (BET) protein Brd4 recruits transcriptional regulatory complexes to acetylated chromatin.  While Brd4 is considered to be a general transcriptional regulator, pharmacol. inhibition of BET proteins shows therapeutic activity in a variety of different pathologies, particularly in models of cancer and inflammation.  Such effects have been attributed to a specific set of downstream target genes whose expression is disproportionately sensitive to pharmacol. targeting of BET proteins.  Emerging evidence links the transcriptional consequences of BET inhibition to the assocn. of Brd4 with enhancer elements, which tend to be involved in lineage-specific gene regulation.  Furthermore, Brd4 engages in direct regulatory interactions with several DNA-binding transcription factors to influence their disease-relevant functions.  Here we review the current understanding of mol. mechanisms that underlie the promising therapeutic effects of BET bromodomain inhibition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpJpSFRzwcR1rVg90H21EOLACvtfcHk0liVSwQhDvyWCw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXpslCrtrk%253D&md5=f49bf3524ec378576fb1a25c64a464ae</span></div><a href="/servlet/linkout?suffix=cit88&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2014.05.016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2014.05.016%26sid%3Dliteratum%253Aachs%26aulast%3DShi%26aufirst%3DJ.%26aulast%3DVakoc%26aufirst%3DC.%2BR.%26atitle%3DThe%2520mechanisms%2520behind%2520the%2520therapeutic%2520activity%2520of%2520BET%2520bromodomain%2520inhibition%26jtitle%3DMol.%2520Cell%26date%3D2014%26volume%3D54%26spage%3D728%26epage%3D736%26doi%3D10.1016%2Fj.molcel.2014.05.016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref89"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref89'); return false;" data-citation="" class="refNumLink">89</a></strong><div class="NLM_citation" id="cit89"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Donati, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lorenzini, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciarrocchi, A.</span></span> <span> </span><span class="NLM_article-title">Ciarrocchi A. BRD4 and cancer: going beyond transcriptional regulation</span>. <i>Mol. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">164</span>, <span class="refDoi"> DOI: 10.1186/s12943-018-0915-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref89/cit89&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1186%2Fs12943-018-0915-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref89/cit89&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=30466442" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref89/cit89&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BC1MXptlWgu78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2018&pages=164&author=B.+Donatiauthor=E.+Lorenziniauthor=A.+Ciarrocchi&title=Ciarrocchi+A.+BRD4+and+cancer%3A+going+beyond+transcriptional+regulation&doi=10.1186%2Fs12943-018-0915-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit89R"><div class="casContent"><span class="casTitleNuber">89</span><div class="casTitle"><span class="NLM_cas:atitle">BRD4 and Cancer: going beyond transcriptional regulation</span></div><div class="casAuthors">Donati, Benedetta; Lorenzini, Eugenia; Ciarrocchi, Alessia</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cancer</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">164</span>CODEN:
                <span class="NLM_cas:coden">MCOACG</span>;
        ISSN:<span class="NLM_cas:issn">1476-4598</span>.
    
            (<span class="NLM_cas:orgname">BioMed Central Ltd.</span>)
        </div><div class="casAbstract">A review.  BRD4, member of the Bromodomain and Extraterminal (BET) protein family, is largely acknowledged in cancer for its role in super-enhancers (SEs) organization and oncogenes expression regulation.  Inhibition of BRD4 shortcuts the communication between SEs and target promoters with a subsequent cell-specific repression of oncogenes to which cancer cells are addicted and cell death.  To date, this is the most credited mechanism of action of BET inhibitors, a class of small mols. targeting BET proteins which are currently in clin. trials in several cancer settings.  However, recent evidence indicates that BRD4 relevance in cancer goes beyond its role in transcription regulation and identifies this protein as a keeper of genome stability.  Indeed, a non-transcriptional role of BRD4 in controlling DNA damage checkpoint activation and repair as well as telomere maintenance has been proposed, throwing new lights into the multiple functions of this protein and opening new perspectives on the use of BETi in cancer.  Here we discuss the current available information on non-canonical, non-transcriptional functions of BRD4 and on their implications in cancer biol.  Integrating this information with the already known BRD4 role in gene expression regulation, we propose a "common" model to explain BRD4 genomic function.  Furthermore, in light of the transversal function of BRD4, we provide new interpretation for the cytotoxic activity of BETi and we discuss new possibilities for a wide and focused employment of these drugs in clin. settings.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpbYn0mnQW7ELVg90H21EOLACvtfcHk0liVSwQhDvyWCw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXptlWgu78%253D&md5=2ddfde1363d430aa67e15a52fdc2335e</span></div><a href="/servlet/linkout?suffix=cit89&amp;dbid=16384&amp;doi=10.1186%2Fs12943-018-0915-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs12943-018-0915-9%26sid%3Dliteratum%253Aachs%26aulast%3DDonati%26aufirst%3DB.%26aulast%3DLorenzini%26aufirst%3DE.%26aulast%3DCiarrocchi%26aufirst%3DA.%26atitle%3DCiarrocchi%2520A.%2520BRD4%2520and%2520cancer%253A%2520going%2520beyond%2520transcriptional%2520regulation%26jtitle%3DMol.%2520Cancer%26date%3D2018%26volume%3D17%26spage%3D164%26doi%3D10.1186%2Fs12943-018-0915-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref90"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref90'); return false;" data-citation="" class="refNumLink">90</a></strong><div class="NLM_citation" id="cit90"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pan, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, G.</span></span> <span> </span><span class="NLM_article-title">Discovery of thieno[2,3-d] pyrimidine-based hydroxamic acid derivatives as bromodomain-containing protein 4/histone deacetylase dual inhibitors induce autophagic cell death in colorectal carcinoma cells</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">3678</span>– <span class="NLM_lpage">3700</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b02178</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b02178" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref90/cit90&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BB3cXksFajtLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=3678-3700&author=Z.+Panauthor=X.+Liauthor=Y.+Wangauthor=Q.+Jiangauthor=L.+Jiangauthor=M.+Zhangauthor=N.+Zhangauthor=F.+Wuauthor=B.+Liuauthor=G.+He&title=Discovery+of+thieno%5B2%2C3-d%5D+pyrimidine-based+hydroxamic+acid+derivatives+as+bromodomain-containing+protein+4%2Fhistone+deacetylase+dual+inhibitors+induce+autophagic+cell+death+in+colorectal+carcinoma+cells&doi=10.1021%2Facs.jmedchem.9b02178"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit90R"><div class="casContent"><span class="casTitleNuber">90</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Thieno[2,3-d]pyrimidine-Based Hydroxamic Acid Derivatives as Bromodomain-Containing Protein 4/Histone Deacetylase Dual Inhibitors Induce Autophagic Cell Death in Colorectal Carcinoma Cells</span></div><div class="casAuthors">Pan, Zhaoping; Li, Xiang; Wang, Yujia; Jiang, Qinglin; Jiang, Li; Zhang, Min; Zhang, Nan; Wu, Fengbo; Liu, Bo; He, Gu</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">3678-3700</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Bromodomain-contg. protein 4 (BRD4) and histone deacetylases (HDAC) are both attractive epigenetic targets in cancer and other chronic diseases.  Based on the integrated fragment-based drug design, synthesis, and in vitro and in vivo evaluations, a series of novel thieno[2,3-d]pyrimidine-based hydroxamic acid derivs. are discovered as selective BRD4-HDAC dual inhibitors.  Compd. 17c is the most potent inhibitor for BRD4 and HDAC with IC50 values at nanomolar levels, as well as the expression level of c-Myc, and increases the acetylation of histone H3.  Moreover, 17c presents inhibitory effects on the proliferation of colorectal carcinoma (CRC) cells via inducing autophagic cell death.  It also has a good pharmacokinetic profile in rats and oral bioavailability of 40.5%.  In the HCT-116 xenograft in vivo models, 17c displays potent inhibitory efficiency on tumor growth by inducing autophagic cell death and suppressing IL6-JAK-STAT signaling pathways.  Our results suggest that the BRD4-HDAC dual inhibition might be an attractive therapeutic strategy for CRC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr2FHKdnuD3QLVg90H21EOLACvtfcHk0lhsjM-aWY9d8Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXksFajtLs%253D&md5=ce6446afe48f80165d2b21aee04571f6</span></div><a href="/servlet/linkout?suffix=cit90&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b02178%26sid%3Dliteratum%253Aachs%26aulast%3DPan%26aufirst%3DZ.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DJiang%26aufirst%3DQ.%26aulast%3DJiang%26aufirst%3DL.%26aulast%3DZhang%26aufirst%3DM.%26aulast%3DZhang%26aufirst%3DN.%26aulast%3DWu%26aufirst%3DF.%26aulast%3DLiu%26aufirst%3DB.%26aulast%3DHe%26aufirst%3DG.%26atitle%3DDiscovery%2520of%2520thieno%255B2%252C3-d%255D%2520pyrimidine-based%2520hydroxamic%2520acid%2520derivatives%2520as%2520bromodomain-containing%2520protein%25204%252Fhistone%2520deacetylase%2520dual%2520inhibitors%2520induce%2520autophagic%2520cell%2520death%2520in%2520colorectal%2520carcinoma%2520cells%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D3678%26epage%3D3700%26doi%3D10.1021%2Facs.jmedchem.9b02178" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref91"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref91'); return false;" data-citation="" class="refNumLink">91</a></strong><div class="NLM_citation" id="cit91"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lan, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geng, M.</span></span> <span> </span><span class="NLM_article-title">Feedback activation of leukemia inhibitory factor receptor limits response to histone deacetylase inhibitors in breast cancer</span>. <i>Cancer Cell</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">459</span>– <span class="NLM_lpage">473</span>, <span class="refDoi"> DOI: 10.1016/j.ccell.2016.08.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref91/cit91&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1016%2Fj.ccell.2016.08.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref91/cit91&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=27622335" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref91/cit91&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsFWitrnI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2016&pages=459-473&author=H.+Zengauthor=J.+Quauthor=N.+Jinauthor=J.+Xuauthor=C.+Linauthor=Y.+Chenauthor=X.+Yangauthor=X.+Heauthor=S.+Tangauthor=X.+Lanauthor=X.+Yangauthor=Z.+Chenauthor=M.+Huangauthor=J.+Dingauthor=M.+Geng&title=Feedback+activation+of+leukemia+inhibitory+factor+receptor+limits+response+to+histone+deacetylase+inhibitors+in+breast+cancer&doi=10.1016%2Fj.ccell.2016.08.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit91R"><div class="casContent"><span class="casTitleNuber">91</span><div class="casTitle"><span class="NLM_cas:atitle">Feedback Activation of Leukemia Inhibitory Factor Receptor Limits Response to Histone Deacetylase Inhibitors in Breast Cancer</span></div><div class="casAuthors">Zeng, Hanlin; Qu, Jia; Jin, Nan; Xu, Jun; Lin, Chenchu; Chen, Yi; Yang, Xinying; He, Xiang; Tang, Shuai; Lan, Xiaojing; Yang, Xiaotong; Chen, Ziqi; Huang, Min; Ding, Jian; Geng, Meiyu</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">459-473</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Histone deacetylase (HDAC) inhibitors have demonstrated clin. benefits in subtypes of hematol. malignancies.  However, the efficacy of HDAC inhibitors in solid tumors remains uncertain.  This study takes breast cancer as a model to understand mechanisms accounting for limited response of HDAC inhibitors in solid tumors and to seek combination solns.  We discover that feedback activation of leukemia inhibitory factor receptor (LIFR) signaling in breast cancer limits the response to HDAC inhibition.  Mechanistically, HDAC inhibition increases histone acetylation at the LIFR gene promoter, which recruits bromodomain protein BRD4, upregulates LIFR expression, and activates JAK1-STAT3 signaling.  Importantly, JAK1 or BRD4 inhibition sensitizes breast cancer to HDAC inhibitors, implicating combination inhibition of HDAC with JAK1 or BRD4 as potential therapies for breast cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrL24sYbhFYeLVg90H21EOLACvtfcHk0lhsjM-aWY9d8Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsFWitrnI&md5=9ce620c7a19519a95edd714f8012518c</span></div><a href="/servlet/linkout?suffix=cit91&amp;dbid=16384&amp;doi=10.1016%2Fj.ccell.2016.08.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccell.2016.08.001%26sid%3Dliteratum%253Aachs%26aulast%3DZeng%26aufirst%3DH.%26aulast%3DQu%26aufirst%3DJ.%26aulast%3DJin%26aufirst%3DN.%26aulast%3DXu%26aufirst%3DJ.%26aulast%3DLin%26aufirst%3DC.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DYang%26aufirst%3DX.%26aulast%3DHe%26aufirst%3DX.%26aulast%3DTang%26aufirst%3DS.%26aulast%3DLan%26aufirst%3DX.%26aulast%3DYang%26aufirst%3DX.%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DHuang%26aufirst%3DM.%26aulast%3DDing%26aufirst%3DJ.%26aulast%3DGeng%26aufirst%3DM.%26atitle%3DFeedback%2520activation%2520of%2520leukemia%2520inhibitory%2520factor%2520receptor%2520limits%2520response%2520to%2520histone%2520deacetylase%2520inhibitors%2520in%2520breast%2520cancer%26jtitle%3DCancer%2520Cell%26date%3D2016%26volume%3D30%26spage%3D459%26epage%3D473%26doi%3D10.1016%2Fj.ccell.2016.08.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref92"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref92'); return false;" data-citation="" class="refNumLink">92</a></strong><div class="NLM_citation" id="cit92"><span><span class="NLM_contrib-group"><span class="NLM_string-name">He, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheng, C.</span></span> <span> </span><span class="NLM_article-title">Potent Dual BET/HDAC Inhibitors for Efficient Treatment of Pancreatic Cancer</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">3028</span>– <span class="NLM_lpage">3032</span>, <span class="refDoi"> DOI: 10.1002/anie.201915896</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref92/cit92&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1002%2Fanie.201915896" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref92/cit92&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BB3cXitFegtLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2020&pages=3028-3032&author=S.+Heauthor=G.+Dongauthor=Y.+Liauthor=S.+Wuauthor=W.+Wangauthor=C.+Sheng&title=Potent+Dual+BET%2FHDAC+Inhibitors+for+Efficient+Treatment+of+Pancreatic+Cancer&doi=10.1002%2Fanie.201915896"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit92R"><div class="casContent"><span class="casTitleNuber">92</span><div class="casTitle"><span class="NLM_cas:atitle">Potent Dual BET/HDAC Inhibitors for Efficient Treatment of Pancreatic Cancer</span></div><div class="casAuthors">He, Shipeng; Dong, Guoqiang; Li, Yu; Wu, Shanchao; Wang, Wei; Sheng, Chunquan</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">3028-3032</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">As one of the most aggressive and lethal human malignancies with extremely poor prognosis, there is an urgent demand of more effective therapy for the treatment of pancreatic cancer.  Reported here is a new, effective therapeutic strategy and the design of small-mol. inhibitors that simultaneously target bromodomain and extra-terminal (BET) and histone deacetylase (HDAC), potentially serving as promising therapeutic agents for pancreatic cancer.  A highly potent dual inhibitor (13 a) is identified to possess excellent and balanced activities against BRD4 BD1 (IC50=11 nM) and HDAC1 (IC50=21 nM).  Notably, this compd. shows higher in vitro and in vivo antitumor potency than the BET inhibitor (+)-JQ1 and the HDAC inhibitor vorinostat, either alone or and in combination, highlighting the advantages of BET/HDAC dual inhibitors for more effective treatment of pancreatic cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp9BFNcLfkA4bVg90H21EOLACvtfcHk0lhsjM-aWY9d8Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXitFegtLw%253D&md5=78db62cb533611939d9abd0e654ff5f3</span></div><a href="/servlet/linkout?suffix=cit92&amp;dbid=16384&amp;doi=10.1002%2Fanie.201915896&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201915896%26sid%3Dliteratum%253Aachs%26aulast%3DHe%26aufirst%3DS.%26aulast%3DDong%26aufirst%3DG.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DWu%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DW.%26aulast%3DSheng%26aufirst%3DC.%26atitle%3DPotent%2520Dual%2520BET%252FHDAC%2520Inhibitors%2520for%2520Efficient%2520Treatment%2520of%2520Pancreatic%2520Cancer%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2020%26volume%3D59%26spage%3D3028%26epage%3D3032%26doi%3D10.1002%2Fanie.201915896" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref93"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref93'); return false;" data-citation="" class="refNumLink">93</a></strong><div class="NLM_citation" id="cit93"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, D.</span></span> <span> </span><span class="NLM_article-title">Design, synthesis and biological evaluation of novel indole derivatives as potential HDAC/BRD4 dual inhibitors and anti-leukemia agents</span>. <i>Bioorg. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>84</i></span>,  <span class="NLM_fpage">410</span>– <span class="NLM_lpage">417</span>, <span class="refDoi"> DOI: 10.1016/j.bioorg.2018.12.011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref93/cit93&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1016%2Fj.bioorg.2018.12.011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref93/cit93&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=30554080" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref93/cit93&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BC1cXisFaqtb%252FI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=84&publication_year=2019&pages=410-417&author=G.+Chengauthor=Z.+Wangauthor=J.+Yangauthor=Y.+Baoauthor=Q.+Xuauthor=L.+Zhaoauthor=D.+Liu&title=Design%2C+synthesis+and+biological+evaluation+of+novel+indole+derivatives+as+potential+HDAC%2FBRD4+dual+inhibitors+and+anti-leukemia+agents&doi=10.1016%2Fj.bioorg.2018.12.011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit93R"><div class="casContent"><span class="casTitleNuber">93</span><div class="casTitle"><span class="NLM_cas:atitle">Design, synthesis and biological evaluation of novel indole derivatives as potential HDAC/BRD4 dual inhibitors and anti-leukemia agents</span></div><div class="casAuthors">Cheng, Gaoliang; Wang, Zhi; Yang, Jinyu; Bao, Yu; Xu, Qihao; Zhao, Linxiang; Liu, Dan</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">84</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">410-417</span>CODEN:
                <span class="NLM_cas:coden">BOCMBM</span>;
        ISSN:<span class="NLM_cas:issn">0045-2068</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">HDAC inhibitors and BRD4 inhibitors were considered to be potent anti-cancer agents.  Recent studies have demonstrated that HDAC and BRD4 participate in the regulation of some signal paths like PI3K-AKT.  In this work, a series of indole derivs. that combine the inhibitory activities of BRD4 and HDAC into one mol. were designed and synthesized through the structure-based design method.  Most compds. showed potent HDAC inhibitory activity and moderate BRD4 inhibitory activity.  In vitro anti-proliferation activities of the synthesized compds. were also evaluated.  Among them, 19f was the most potent inhibitor against HDAC3 with IC50 value of 5 nM and BRD4 inhibition rate of 88% at 10 μM.  It was confirmed that 19f could up-regulate the expression of Ac-H3 and reduce the expression of c-Myc by western blot anal.  These results indicated that 19f was a potent dual HDAC/BRD4 inhibitor and deserved further investigation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqkPM3yaQ5gE7Vg90H21EOLACvtfcHk0lgZVyF9RbcRzQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXisFaqtb%252FI&md5=e2c81b5d67fd9f49e6805c6b544438b4</span></div><a href="/servlet/linkout?suffix=cit93&amp;dbid=16384&amp;doi=10.1016%2Fj.bioorg.2018.12.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bioorg.2018.12.011%26sid%3Dliteratum%253Aachs%26aulast%3DCheng%26aufirst%3DG.%26aulast%3DWang%26aufirst%3DZ.%26aulast%3DYang%26aufirst%3DJ.%26aulast%3DBao%26aufirst%3DY.%26aulast%3DXu%26aufirst%3DQ.%26aulast%3DZhao%26aufirst%3DL.%26aulast%3DLiu%26aufirst%3DD.%26atitle%3DDesign%252C%2520synthesis%2520and%2520biological%2520evaluation%2520of%2520novel%2520indole%2520derivatives%2520as%2520potential%2520HDAC%252FBRD4%2520dual%2520inhibitors%2520and%2520anti-leukemia%2520agents%26jtitle%3DBioorg.%2520Chem.%26date%3D2019%26volume%3D84%26spage%3D410%26epage%3D417%26doi%3D10.1016%2Fj.bioorg.2018.12.011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref94"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref94'); return false;" data-citation="" class="refNumLink">94</a></strong><div class="NLM_citation" id="cit94"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Amemiya, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamaguchi, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hashimoto, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Noguchi-Yachide, T.</span></span> <span> </span><span class="NLM_article-title">Synthesis and evaluation of novel dual BRD4/HDAC inhibitors</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">3677</span>– <span class="NLM_lpage">3684</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2017.04.043</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref94/cit94&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1016%2Fj.bmc.2017.04.043" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref94/cit94&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=28549889" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref94/cit94&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BC2sXotlyitL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2017&pages=3677-3684&author=S.+Amemiyaauthor=T.+Yamaguchiauthor=Y.+Hashimotoauthor=T.+Noguchi-Yachide&title=Synthesis+and+evaluation+of+novel+dual+BRD4%2FHDAC+inhibitors&doi=10.1016%2Fj.bmc.2017.04.043"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit94R"><div class="casContent"><span class="casTitleNuber">94</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis and evaluation of novel dual BRD4/HDAC inhibitors</span></div><div class="casAuthors">Amemiya, Seika; Yamaguchi, Takao; Hashimoto, Yuichi; Noguchi-Yachide, Tomomi</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">3677-3684</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Epigenetic regulation of gene expression via histone acetylation modulates many cellular processes, including apoptosis, the cell cycle, cell growth and differentiation, and inhibitors are promising drug candidates.  We have previously developed inhibitors of BRD4, which recognizes acetylated lysine residue on histones and recruits transcription elongation factor to the transcription start site, while inhibitors of histone deacetylase (HDAC), which catalyzes the removal of acetyl groups on histones, are already in clin. use for cancer treatment.  Based on the idea that polypharmacol. agents with multiple targets would have a more robust action, we set out to develop dual BRD4/HDAC inhibitors.  Here, we describe the design and synthesis of N6-[2-(7-hydroxyamino-7-oxoheptyloxy)benzoyl]adenine (I) as a BRD4/HDAC dual inhibitor.  This compd. showed HL-60 cell growth-inhibitory and apoptosis-inducing activity, as well as all-trans retinoic acid (ATRA)-induced HL-60 cell differentiation-enhancing activity, and c-MYC prodn.-inhibitory activity.  Interestingly, it also showed growth-inhibitory activity towards BRD4 inhibitor-resistant cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqw5w5uPYmY37Vg90H21EOLACvtfcHk0lgZVyF9RbcRzQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXotlyitL4%253D&md5=c9099dc3d40c966d150a13fab28c2b9f</span></div><a href="/servlet/linkout?suffix=cit94&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2017.04.043&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2017.04.043%26sid%3Dliteratum%253Aachs%26aulast%3DAmemiya%26aufirst%3DS.%26aulast%3DYamaguchi%26aufirst%3DT.%26aulast%3DHashimoto%26aufirst%3DY.%26aulast%3DNoguchi-Yachide%26aufirst%3DT.%26atitle%3DSynthesis%2520and%2520evaluation%2520of%2520novel%2520dual%2520BRD4%252FHDAC%2520inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2017%26volume%3D25%26spage%3D3677%26epage%3D3684%26doi%3D10.1016%2Fj.bmc.2017.04.043" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref95"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref95'); return false;" data-citation="" class="refNumLink">95</a></strong><div class="NLM_citation" id="cit95"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hou, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ran, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bian, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span> <span> </span><span class="NLM_article-title">Targeting epigenetic reader and eraser: Rational design, synthesis and in vitro evaluation of dimethylisoxazoles derivatives as BRD4/HDAC dual inhibitors</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">2931</span>– <span class="NLM_lpage">2935</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2016.04.034</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref95/cit95&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1016%2Fj.bmcl.2016.04.034" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref95/cit95&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=27142751" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref95/cit95&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BC28XntFCmt7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2016&pages=2931-2935&author=Z.+Zhangauthor=S.+Houauthor=H.+Chenauthor=T.+Ranauthor=F.+Jiangauthor=Y.+Bianauthor=D.+Zhangauthor=Y.+Zhiauthor=L.+Wangauthor=L.+Zhangauthor=H.+Liauthor=Y.+Zhangauthor=W.+Tangauthor=T.+Luauthor=Y.+Chen&title=Targeting+epigenetic+reader+and+eraser%3A+Rational+design%2C+synthesis+and+in+vitro+evaluation+of+dimethylisoxazoles+derivatives+as+BRD4%2FHDAC+dual+inhibitors&doi=10.1016%2Fj.bmcl.2016.04.034"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit95R"><div class="casContent"><span class="casTitleNuber">95</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting epigenetic reader and eraser: Rational design, synthesis and in vitro evaluation of dimethylisoxazoles derivatives as BRD4/HDAC dual inhibitors</span></div><div class="casAuthors">Zhang, Zhimin; Hou, Shaohua; Chen, Hongli; Ran, Ting; Jiang, Fei; Bian, Yuanyuan; Zhang, Dewei; Zhi, Yanle; Wang, Lu; Zhang, Li; Li, Hongmei; Zhang, Yanmin; Tang, Weifang; Lu, Tao; Chen, Yadong</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2931-2935</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The bromodomain protein module and histone deacetylase (HDAC), which recognize and remove acetylated lysine, resp., have emerged as important epigenetic therapeutic targets in cancer treatments.  Herein the authors presented a novel design approach for cancer drug development by combination of bromodomain and HDAC inhibitory activity in one mol.  The designed compds. were synthesized which showed inhibitory activity against bromodomain 4 and HDAC1.  The representative dual bromodomain/HDAC inhibitors, compds. N1-(3-benzoyl-5-(3,5-dimethylisoxazol-4-yl)phenyl)-N7-hydroxyheptanediamide and N1-(3-benzoyl-5-(3,5-dimethylisoxazol-4-yl)phenyl)-N8-hydroxyoctanediamide showed potent antiproliferative activities against human leukemia cell line K562 and MV4-11 in cellular assays.  This work may lay the foundation for developing dual bromodomain/HDAC inhibitors as potential anticancer therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGriS0VB1G-QoLVg90H21EOLACvtfcHk0lgZVyF9RbcRzQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XntFCmt7o%253D&md5=a6ebdbd6402e3117a2f9d96ed071790c</span></div><a href="/servlet/linkout?suffix=cit95&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2016.04.034&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2016.04.034%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DZ.%26aulast%3DHou%26aufirst%3DS.%26aulast%3DChen%26aufirst%3DH.%26aulast%3DRan%26aufirst%3DT.%26aulast%3DJiang%26aufirst%3DF.%26aulast%3DBian%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DD.%26aulast%3DZhi%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DLi%26aufirst%3DH.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DTang%26aufirst%3DW.%26aulast%3DLu%26aufirst%3DT.%26aulast%3DChen%26aufirst%3DY.%26atitle%3DTargeting%2520epigenetic%2520reader%2520and%2520eraser%253A%2520Rational%2520design%252C%2520synthesis%2520and%2520in%2520vitro%2520evaluation%2520of%2520dimethylisoxazoles%2520derivatives%2520as%2520BRD4%252FHDAC%2520dual%2520inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2016%26volume%3D26%26spage%3D2931%26epage%3D2935%26doi%3D10.1016%2Fj.bmcl.2016.04.034" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref96"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref96'); return false;" data-citation="" class="refNumLink">96</a></strong><div class="NLM_citation" id="cit96"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shao, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L.</span></span> <span> </span><span class="NLM_article-title">Structure-based design, synthesis and in vitro antiproliferative effects studies of novel dual BRD4/HDAC inhibitors</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">4051</span>– <span class="NLM_lpage">4055</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2017.07.054</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref96/cit96&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1016%2Fj.bmcl.2017.07.054" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref96/cit96&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=28765013" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref96/cit96&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BC2sXht1KqsLrP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2017&pages=4051-4055&author=M.+Shaoauthor=L.+Heauthor=L.+Zhengauthor=L.+Huangauthor=Y.+Zhouauthor=T.+Wangauthor=Y.+Chenauthor=M.+Shenauthor=F.+Wangauthor=Z.+Yangauthor=L.+Chen&title=Structure-based+design%2C+synthesis+and+in+vitro+antiproliferative+effects+studies+of+novel+dual+BRD4%2FHDAC+inhibitors&doi=10.1016%2Fj.bmcl.2017.07.054"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit96R"><div class="casContent"><span class="casTitleNuber">96</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-based design, synthesis and in vitro antiproliferative effects studies of novel dual BRD4/HDAC inhibitors</span></div><div class="casAuthors">Shao, Mingfeng; He, Linhong; Zheng, Li; Huang, Lingxiao; Zhou, Yuanyuan; Wang, Taijing; Chen, Yong; Shen, Mingsheng; Wang, Fang; Yang, Zhuang; Chen, Lijuan</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">4051-4055</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Histone acetylation marks play important roles in controlling gene expressions and are removed by histone deacetylases (HDACs).  These marks are read by bromodomain and extra-terminal (BET) proteins, whose targeted inhibitors are under clin. investigation.  BET and HDAC inhibitors have been demonstrated to be synergistically killing in Myc-induced murine lymphoma.  Herein, the authors combine the inhibitory activities of BET and HDAC into one mol. through structure-based design method and evaluate its function.  The majority of these synthesized compds. showed inhibitory activity against second bromodomains(BRD) of BRD4 and HDAC1.  Among them, I presented antiproliferative effects against human acute myelogenous leukemia (AML) cell lines in vitro, and I is confirmed to reduce the expression of Myc by Western blot anal.  Those results indicated that I is a potent dual BRD4/HDAC inhibitor and deserves further investigation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpkMaHqVu_LM7Vg90H21EOLACvtfcHk0ljc2g4_RRJd2w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXht1KqsLrP&md5=c525f1a2083699dd5f6f6a262f477f24</span></div><a href="/servlet/linkout?suffix=cit96&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2017.07.054&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2017.07.054%26sid%3Dliteratum%253Aachs%26aulast%3DShao%26aufirst%3DM.%26aulast%3DHe%26aufirst%3DL.%26aulast%3DZheng%26aufirst%3DL.%26aulast%3DHuang%26aufirst%3DL.%26aulast%3DZhou%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DT.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DShen%26aufirst%3DM.%26aulast%3DWang%26aufirst%3DF.%26aulast%3DYang%26aufirst%3DZ.%26aulast%3DChen%26aufirst%3DL.%26atitle%3DStructure-based%2520design%252C%2520synthesis%2520and%2520in%2520vitro%2520antiproliferative%2520effects%2520studies%2520of%2520novel%2520dual%2520BRD4%252FHDAC%2520inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2017%26volume%3D27%26spage%3D4051%26epage%3D4055%26doi%3D10.1016%2Fj.bmcl.2017.07.054" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref97"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref97'); return false;" data-citation="" class="refNumLink">97</a></strong><div class="NLM_citation" id="cit97"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shan, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fan, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanyi, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fan, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Montone, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dang, C. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span> <span> </span><span class="NLM_article-title">Repression of BET activity sensitizes homologous recombination-proficient cancers to PARP inhibition</span>. <i>Sci. Transl. Med.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">eaal1645</span>, <span class="refDoi"> DOI: 10.1126/scitranslmed.aal1645</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref97/cit97&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1126%2Fscitranslmed.aal1645" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref97/cit97&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=28747513" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2017&pages=eaal1645&author=L.+Yangauthor=Y.+Zhangauthor=W.+Shanauthor=Z.+Huauthor=J.+Yuanauthor=J.+Piauthor=Y.+Wangauthor=L.+Fanauthor=Z.+Tangauthor=C.+Liauthor=X.+Huauthor=J.+L.+Tanyiauthor=Y.+Fanauthor=Q.+Huangauthor=K.+Montoneauthor=C.+V.+Dangauthor=L.+Zhang&title=Repression+of+BET+activity+sensitizes+homologous+recombination-proficient+cancers+to+PARP+inhibition&doi=10.1126%2Fscitranslmed.aal1645"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit97&amp;dbid=16384&amp;doi=10.1126%2Fscitranslmed.aal1645&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscitranslmed.aal1645%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DL.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DShan%26aufirst%3DW.%26aulast%3DHu%26aufirst%3DZ.%26aulast%3DYuan%26aufirst%3DJ.%26aulast%3DPi%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DFan%26aufirst%3DL.%26aulast%3DTang%26aufirst%3DZ.%26aulast%3DLi%26aufirst%3DC.%26aulast%3DHu%26aufirst%3DX.%26aulast%3DTanyi%26aufirst%3DJ.%2BL.%26aulast%3DFan%26aufirst%3DY.%26aulast%3DHuang%26aufirst%3DQ.%26aulast%3DMontone%26aufirst%3DK.%26aulast%3DDang%26aufirst%3DC.%2BV.%26aulast%3DZhang%26aufirst%3DL.%26atitle%3DRepression%2520of%2520BET%2520activity%2520sensitizes%2520homologous%2520recombination-proficient%2520cancers%2520to%2520PARP%2520inhibition%26jtitle%3DSci.%2520Transl.%2520Med.%26date%3D2017%26volume%3D9%26spage%3Deaal1645%26doi%3D10.1126%2Fscitranslmed.aal1645" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref98"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref98'); return false;" data-citation="" class="refNumLink">98</a></strong><div class="NLM_citation" id="cit98"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ouyang, L.</span></span> <span> </span><span class="NLM_article-title">Design, synthesis, and biological evaluation of quinazolin-4(3H)-one derivatives co-targeting poly(ADP-ribose) polymerase-1 and bromodomain containing protein 4 for breast cancer therapy</span>. <i>Acta Pharm. Sin. B</i> <span class="NLM_year" style="font-weight: bold;">2021</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">156</span>– <span class="NLM_lpage">180</span>, <span class="refDoi"> DOI: 10.1016/j.apsb.2020.06.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref98/cit98&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1016%2Fj.apsb.2020.06.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref98/cit98&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=33532187" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref98/cit98&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A280%3ADC%252BB3srovVGlsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2021&pages=156-180&author=X.+Changauthor=D.+Sunauthor=F.+Shiauthor=G.+Wangauthor=Y.+Chenauthor=K.+Zhangauthor=H.+Tanauthor=J.+Liuauthor=B.+Liuauthor=L.+Ouyang&title=Design%2C+synthesis%2C+and+biological+evaluation+of+quinazolin-4%283H%29-one+derivatives+co-targeting+poly%28ADP-ribose%29+polymerase-1+and+bromodomain+containing+protein+4+for+breast+cancer+therapy&doi=10.1016%2Fj.apsb.2020.06.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit98R"><div class="casContent"><span class="casTitleNuber">98</span><div class="casTitle"><span class="NLM_cas:atitle">Design, synthesis, and biological evaluation of quinazolin-4(3H)-one derivatives co-targeting poly(ADP-ribose) polymerase-1 and bromodomain containing protein 4 for breast cancer therapy</span></div><div class="casAuthors">Chang Xiaosa; Sun Dejuan; Shi Danfeng; Wang Guan; Chen Yanmei; Zhang Kai; Tan Huidan; Liu Jie; Liu Bo; Ouyang Liang</div><div class="citationInfo"><span class="NLM_cas:title">Acta pharmaceutica Sinica. B</span>
        (<span class="NLM_cas:date">2021</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">156-180</span>
        ISSN:<span class="NLM_cas:issn">2211-3835</span>.
    </div><div class="casAbstract">This study was aimed to design the first dual-target small-molecule inhibitor co-targeting poly (ADP-ribose) polymerase-1 (PARP1) and bromodomain containing protein 4 (BRD4), which had important cross relation in the global network of breast cancer, reflecting the synthetic lethal effect.  A series of new BRD4 and PARP1 dual-target inhibitors were discovered and synthesized by fragment-based combinatorial screening and activity assays that together led to the chemical optimization.  Among these compounds, 19d was selected and exhibited micromole enzymatic potencies against BRD4 and PARP1, respectively.  Compound 19d was further shown to efficiently modulate the expression of BRD4 and PARP1.  Subsequently, compound 19d was found to induce breast cancer cell apoptosis and stimulate cell cycle arrest at G1 phase.  Following pharmacokinetic studies, compound 19d showed its antitumor activity in breast cancer susceptibility gene 1/2 (BRCA1/2) wild-type MDA-MB-468 and MCF-7 xenograft models without apparent toxicity and loss of body weight.  These results together demonstrated that a highly potent dual-targeted inhibitor was successfully synthesized and indicated that co-targeting of BRD4 and PARP1 based on the concept of synthetic lethality would be a promising therapeutic strategy for breast cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTHS4WGcbBgy2HOrQ6qeX-mfW6udTcc2eZ2H6rKUCwI7Lntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3srovVGlsQ%253D%253D&md5=7108868f2fa08b8bcd5131ec8dd1d281</span></div><a href="/servlet/linkout?suffix=cit98&amp;dbid=16384&amp;doi=10.1016%2Fj.apsb.2020.06.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.apsb.2020.06.003%26sid%3Dliteratum%253Aachs%26aulast%3DChang%26aufirst%3DX.%26aulast%3DSun%26aufirst%3DD.%26aulast%3DShi%26aufirst%3DF.%26aulast%3DWang%26aufirst%3DG.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DK.%26aulast%3DTan%26aufirst%3DH.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DLiu%26aufirst%3DB.%26aulast%3DOuyang%26aufirst%3DL.%26atitle%3DDesign%252C%2520synthesis%252C%2520and%2520biological%2520evaluation%2520of%2520quinazolin-4%25283H%2529-one%2520derivatives%2520co-targeting%2520poly%2528ADP-ribose%2529%2520polymerase-1%2520and%2520bromodomain%2520containing%2520protein%25204%2520for%2520breast%2520cancer%2520therapy%26jtitle%3DActa%2520Pharm.%2520Sin.%2520B%26date%3D2021%26volume%3D11%26spage%3D156%26epage%3D180%26doi%3D10.1016%2Fj.apsb.2020.06.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref99"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref99'); return false;" data-citation="" class="refNumLink">99</a></strong><div class="NLM_citation" id="cit99"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ren, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bi, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lwin, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Silva, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shah, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koomen, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chavez, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pham, L. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sudalagunta, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wan, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dalton, W. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moscinski, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shain, K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vose, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cleveland, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sotomayor, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tao, J.</span></span> <span> </span><span class="NLM_article-title">PLK1 stabilizes a MYC-dependent kinase network in aggressive B cell lymphomas</span>. <i>J. Clin. Invest.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>128</i></span>,  <span class="NLM_fpage">5517</span>– <span class="NLM_lpage">5530</span>, <span class="refDoi"> DOI: 10.1172/JCI122533</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref99/cit99&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1172%2FJCI122533" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref99/cit99&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=30260324" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref99/cit99&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A280%3ADC%252BB3czjt1yntw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=128&publication_year=2018&pages=5517-5530&author=Y.+Renauthor=C.+Biauthor=X.+Zhaoauthor=T.+Lwinauthor=C.+Wangauthor=J.+Yuanauthor=A.+S.+Silvaauthor=B.+D.+Shahauthor=B.+Fangauthor=T.+Liauthor=J.+M.+Koomenauthor=H.+Jiangauthor=J.+C.+Chavezauthor=L.+V.+Phamauthor=P.+R.+Sudalaguntaauthor=L.+Wanauthor=X.+Wangauthor=W.+S.+Daltonauthor=L.+C.+Moscinskiauthor=K.+H.+Shainauthor=J.+Voseauthor=J.+L.+Clevelandauthor=E.+M.+Sotomayorauthor=K.+Fuauthor=J.+Tao&title=PLK1+stabilizes+a+MYC-dependent+kinase+network+in+aggressive+B+cell+lymphomas&doi=10.1172%2FJCI122533"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit99R"><div class="casContent"><span class="casTitleNuber">99</span><div class="casTitle"><span class="NLM_cas:atitle">PLK1 stabilizes a MYC-dependent kinase network in aggressive B cell lymphomas</span></div><div class="casAuthors">Ren Yuan; Zhao Xiaohong; Lwin Tint; Li Tao; Jiang Huijuan; Moscinski Lynn C; Tao Jianguo; Bi Chengfeng; Wang Cheng; Yuan Ji; Fu Kai; Silva Ariosto S; Sudalagunta Praneeth R; Shah Bijal D; Chavez Julio C; Dalton William S; Shain Kenneth H; Tao Jianguo; Fang Bin; Koomen John M; Jiang Huijuan; Pham Lan V; Wan Lixin; Wang Xuefeng; Vose Julie; Cleveland John L; Sotomayor Eduardo M</div><div class="citationInfo"><span class="NLM_cas:title">The Journal of clinical investigation</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">128</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">5517-5530</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Concordant activation of MYC and BCL-2 oncoproteins in double-hit lymphoma (DHL) results in aggressive disease that is refractory to treatment.  By integrating activity-based proteomic profiling and drug screens, polo-like kinase-1 (PLK1) was identified as an essential regulator of the MYC-dependent kinome in DHL.  Notably, PLK1 was expressed at high levels in DHL, correlated with MYC expression, and connoted poor outcome.  Further, PLK1 signaling augmented MYC protein stability, and in turn, MYC directly induced PLK1 transcription, establishing a feed-forward MYC-PLK1 circuit in DHL.  Finally, inhibition of PLK1 triggered degradation of MYC and of the antiapoptotic protein MCL-1, and PLK1 inhibitors showed synergy with BCL-2 antagonists in blocking DHL cell growth, survival, and tumorigenicity, supporting clinical targeting of PLK1 in DHL.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRyRm5q5xX4CfxQvRxYU81tfW6udTcc2ebfusADzxtKIbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3czjt1yntw%253D%253D&md5=fec6af4dad461a48831839f52913b704</span></div><a href="/servlet/linkout?suffix=cit99&amp;dbid=16384&amp;doi=10.1172%2FJCI122533&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1172%252FJCI122533%26sid%3Dliteratum%253Aachs%26aulast%3DRen%26aufirst%3DY.%26aulast%3DBi%26aufirst%3DC.%26aulast%3DZhao%26aufirst%3DX.%26aulast%3DLwin%26aufirst%3DT.%26aulast%3DWang%26aufirst%3DC.%26aulast%3DYuan%26aufirst%3DJ.%26aulast%3DSilva%26aufirst%3DA.%2BS.%26aulast%3DShah%26aufirst%3DB.%2BD.%26aulast%3DFang%26aufirst%3DB.%26aulast%3DLi%26aufirst%3DT.%26aulast%3DKoomen%26aufirst%3DJ.%2BM.%26aulast%3DJiang%26aufirst%3DH.%26aulast%3DChavez%26aufirst%3DJ.%2BC.%26aulast%3DPham%26aufirst%3DL.%2BV.%26aulast%3DSudalagunta%26aufirst%3DP.%2BR.%26aulast%3DWan%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DDalton%26aufirst%3DW.%2BS.%26aulast%3DMoscinski%26aufirst%3DL.%2BC.%26aulast%3DShain%26aufirst%3DK.%2BH.%26aulast%3DVose%26aufirst%3DJ.%26aulast%3DCleveland%26aufirst%3DJ.%2BL.%26aulast%3DSotomayor%26aufirst%3DE.%2BM.%26aulast%3DFu%26aufirst%3DK.%26aulast%3DTao%26aufirst%3DJ.%26atitle%3DPLK1%2520stabilizes%2520a%2520MYC-dependent%2520kinase%2520network%2520in%2520aggressive%2520B%2520cell%2520lymphomas%26jtitle%3DJ.%2520Clin.%2520Invest.%26date%3D2018%26volume%3D128%26spage%3D5517%26epage%3D5530%26doi%3D10.1172%2FJCI122533" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref100"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref100'); return false;" data-citation="" class="refNumLink">100</a></strong><div class="NLM_citation" id="cit100"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiao, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zuo, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, L.</span></span> <span> </span><span class="NLM_article-title">Novel dual BET and PLK1 inhibitor WNY0824 exerts potent antitumor effects in CRPC by inhibiting transcription factor function and inducing mitotic abnormality</span>. <i>Mol. Cancer Ther.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">1221</span>– <span class="NLM_lpage">1231</span>, <span class="refDoi"> DOI: 10.1158/1535-7163.MCT-19-0578</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref100/cit100&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1158%2F1535-7163.MCT-19-0578" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref100/cit100&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=32220972" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref100/cit100&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhvVems7%252FO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2020&pages=1221-1231&author=Y.+Xuauthor=Q.+Wangauthor=K.+Xiaoauthor=Z.+Liuauthor=L.+Zhaoauthor=X.+Songauthor=X.+Huauthor=Z.+Fengauthor=T.+Gaoauthor=W.+Zuoauthor=J.+Zengauthor=N.+Wangauthor=L.+Yu&title=Novel+dual+BET+and+PLK1+inhibitor+WNY0824+exerts+potent+antitumor+effects+in+CRPC+by+inhibiting+transcription+factor+function+and+inducing+mitotic+abnormality&doi=10.1158%2F1535-7163.MCT-19-0578"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit100R"><div class="casContent"><span class="casTitleNuber">100</span><div class="casTitle"><span class="NLM_cas:atitle">Novel dual BET and PLK1 inhibitor WNY0824 exerts potent antitumor effects in crpc by inhibiting transcription factor function and inducing mitotic abnormality</span></div><div class="casAuthors">Xu, Ying; Wang, Qianqian; Xiao, Kunjie; Liu, Zhihao; Zhao, Lifeng; Song, Xuejiao; Hu, Xi; Feng, Zhanzhan; Gao, Tiantao; Zuo, Weiqiong; Zeng, Jun; Wang, Ningyu; Yu, Luoting</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cancer Therapeutics</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1221-1231</span>CODEN:
                <span class="NLM_cas:coden">MCTOCF</span>;
        ISSN:<span class="NLM_cas:issn">1535-7163</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Castration-resistant prostate cancer (CRPC) is a lethal disease with few treatment alternatives once patients become resistant to second-generation antiandrogens.  In CRPC, BET proteins are key regulators of AR- and MYC-mediated transcription, while the PLK1 inhibitor potentially downregulates AR and MYC besides influencing the cell cycle.  Therefore, synchronous inhibition of BET and PLK1 would be a promising approach for CRPC therapy.  This study developed a dual BET and PLK1 inhibitor WNY0824 with nanomolar and equipotent inhibition of BRD4 and PLK1.  In vitro, WNY0824 exhibited excellent antiproliferation activity on AR-pos. CRPC cells and induced apoptosis.  These activities are attributable to its disruption of the AR-transcriptional program and the inhibition of the ETS pathway.  Furthermore, WNY0824 downregulated MYC and induced mitotic abnormality.  In vivo, oral WNY0824 administration suppressed tumor growth in the CRPC xenograft model of enzalutamide resistance.  These findings suggest that WNY0824 is a selective dual BET and PLK1 inhibitor with potent anti-CRPC oncogenic activity and provides insights into the development of other novel dual BET- and PLK1-inhibiting drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpNhuOBmxkBB7Vg90H21EOLACvtfcHk0li7O_f3eFCshQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhvVems7%252FO&md5=74c4a259cded1d4cb89a2d2cbcb36442</span></div><a href="/servlet/linkout?suffix=cit100&amp;dbid=16384&amp;doi=10.1158%2F1535-7163.MCT-19-0578&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1535-7163.MCT-19-0578%26sid%3Dliteratum%253Aachs%26aulast%3DXu%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DQ.%26aulast%3DXiao%26aufirst%3DK.%26aulast%3DLiu%26aufirst%3DZ.%26aulast%3DZhao%26aufirst%3DL.%26aulast%3DSong%26aufirst%3DX.%26aulast%3DHu%26aufirst%3DX.%26aulast%3DFeng%26aufirst%3DZ.%26aulast%3DGao%26aufirst%3DT.%26aulast%3DZuo%26aufirst%3DW.%26aulast%3DZeng%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DN.%26aulast%3DYu%26aufirst%3DL.%26atitle%3DNovel%2520dual%2520BET%2520and%2520PLK1%2520inhibitor%2520WNY0824%2520exerts%2520potent%2520antitumor%2520effects%2520in%2520CRPC%2520by%2520inhibiting%2520transcription%2520factor%2520function%2520and%2520inducing%2520mitotic%2520abnormality%26jtitle%3DMol.%2520Cancer%2520Ther.%26date%3D2020%26volume%3D19%26spage%3D1221%26epage%3D1231%26doi%3D10.1158%2F1535-7163.MCT-19-0578" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref101"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref101'); return false;" data-citation="" class="refNumLink">101</a></strong><div class="NLM_citation" id="cit101"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yosief, H. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dai, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dhawan, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muthengi, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buckley, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perry, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, W.</span></span> <span> </span><span class="NLM_article-title">Structure-guided design and development of potent and selective dual bromodomain 4 (BRD4)/polo-like kinase 1 (PLK1) inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">7785</span>– <span class="NLM_lpage">7795</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b00765</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b00765" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref101/cit101&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhsFCksr3L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=7785-7795&author=S.+Liuauthor=H.+O.+Yosiefauthor=L.+Daiauthor=H.+Huangauthor=G.+Dhawanauthor=X.+Zhangauthor=A.+M.+Muthengiauthor=J.+Robertsauthor=D.+L.+Buckleyauthor=J.+A.+Perryauthor=L.+Wuauthor=J.+E.+Bradnerauthor=J.+Qiauthor=W.+Zhang&title=Structure-guided+design+and+development+of+potent+and+selective+dual+bromodomain+4+%28BRD4%29%2Fpolo-like+kinase+1+%28PLK1%29+inhibitors&doi=10.1021%2Facs.jmedchem.8b00765"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit101R"><div class="casContent"><span class="casTitleNuber">101</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Guided Design and Development of Potent and Selective Dual Bromodomain 4 (BRD4)/Polo-like Kinase 1 (PLK1) Inhibitors</span></div><div class="casAuthors">Liu, Shuai; Yosief, Hailemichael O.; Dai, Lingling; Huang, He; Dhawan, Gagan; Zhang, Xiaofeng; Muthengi, Alex M.; Roberts, Justin; Buckley, Dennis L.; Perry, Jennifer A.; Wu, Lei; Bradner, James E.; Qi, Jun; Zhang, Wei</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">7785-7795</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The simultaneous inhibition of polo-like kinase 1 (PLK1) and BRD4 bromodomain by a single mol. could lead to the development of an effective therapeutic strategy for a variety of diseases in which PLK1 and BRD4 are implicated.  Compd. 23 has been found to be a potent dual kinase-bromodomain inhibitor (BRD4-BD1 IC50 = 28 nM, PLK1 IC50 = 40 nM).  Compd. 6 was found to be the most selective PLK1 inhibitor over BRD4 in our series (BRD4-BD1 IC50 = 2579 nM, PLK1 IC50 = 9.9 nM).  Mol. docking studies with 23 and BRD4-BD1/PLK1 as well as with 6 corroborate the biochem. assay results.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoUL2G-DnFzU7Vg90H21EOLACvtfcHk0liFRb0J6SCFqg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhsFCksr3L&md5=e34a58694f0827d33d31516200e2c0d8</span></div><a href="/servlet/linkout?suffix=cit101&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b00765%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DS.%26aulast%3DYosief%26aufirst%3DH.%2BO.%26aulast%3DDai%26aufirst%3DL.%26aulast%3DHuang%26aufirst%3DH.%26aulast%3DDhawan%26aufirst%3DG.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DMuthengi%26aufirst%3DA.%2BM.%26aulast%3DRoberts%26aufirst%3DJ.%26aulast%3DBuckley%26aufirst%3DD.%2BL.%26aulast%3DPerry%26aufirst%3DJ.%2BA.%26aulast%3DWu%26aufirst%3DL.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26aulast%3DQi%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DW.%26atitle%3DStructure-guided%2520design%2520and%2520development%2520of%2520potent%2520and%2520selective%2520dual%2520bromodomain%25204%2520%2528BRD4%2529%252Fpolo-like%2520kinase%25201%2520%2528PLK1%2529%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D7785%26epage%3D7795%26doi%3D10.1021%2Facs.jmedchem.8b00765" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref102"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref102'); return false;" data-citation="" class="refNumLink">102</a></strong><div class="NLM_citation" id="cit102"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Watts, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heidenreich, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tucker, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raab, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Strebhardt, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chesler, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bellenie, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoelder, S.</span></span> <span> </span><span class="NLM_article-title">Designing dual inhibitors of anaplastic lymphoma kinase (ALK) and bromodomain-4 (BRD4) by tuning kinase selectivity</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">2618</span>– <span class="NLM_lpage">2637</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b01947</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b01947" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref102/cit102&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BC1MXjtlGmu74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=2618-2637&author=E.+Wattsauthor=D.+Heidenreichauthor=E.+Tuckerauthor=M.+Raabauthor=K.+Strebhardtauthor=L.+Cheslerauthor=S.+Knappauthor=B.+Bellenieauthor=S.+Hoelder&title=Designing+dual+inhibitors+of+anaplastic+lymphoma+kinase+%28ALK%29+and+bromodomain-4+%28BRD4%29+by+tuning+kinase+selectivity&doi=10.1021%2Facs.jmedchem.8b01947"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit102R"><div class="casContent"><span class="casTitleNuber">102</span><div class="casTitle"><span class="NLM_cas:atitle">Designing Dual Inhibitors of Anaplastic Lymphoma Kinase (ALK) and Bromodomain-4 (BRD4) by Tuning Kinase Selectivity</span></div><div class="casAuthors">Watts, Ellen; Heidenreich, David; Tucker, Elizabeth; Raab, Monika; Strebhardt, Klaus; Chesler, Louis; Knapp, Stefan; Bellenie, Benjamin; Hoelder, Swen</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">2618-2637</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Concomitant inhibition of anaplastic lymphoma kinase (ALK) and bromodomain-4 (BRD4) is a potential therapeutic strategy for targeting two key oncogenic drivers that co-segregate in a significant fraction of high-risk neuroblastoma patients, mutation of ALK and amplification of MYCN.  Starting from known dual polo-like kinase (PLK)-1-BRD4 inhibitor BI-2536, we employed structure-based design to redesign this series toward compds. with a dual ALK-BRD4 profile.  These efforts led to compd. (R)-2-((2-ethoxy-4-(1-methylpiperidin-4-yl)phenyl)amino)-7-ethyl-5-methyl-8-((4-methylthiophen-2-yl)methyl)-7,8-dihydropteridin-6(5H)-one (16k) demonstrating improved ALK activity and significantly reduced PLK-1 activity, while maintaining BRD4 activity and overall kinome selectivity.  We demonstrate the compds.' on-target engagement with ALK and BRD4 in cells as well as favorable broad kinase and bromodomain selectivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGop6qx6PSj2KLVg90H21EOLACvtfcHk0liFRb0J6SCFqg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXjtlGmu74%253D&md5=e00fdf8eb20acc2ef1c8777b2afb750d</span></div><a href="/servlet/linkout?suffix=cit102&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b01947&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b01947%26sid%3Dliteratum%253Aachs%26aulast%3DWatts%26aufirst%3DE.%26aulast%3DHeidenreich%26aufirst%3DD.%26aulast%3DTucker%26aufirst%3DE.%26aulast%3DRaab%26aufirst%3DM.%26aulast%3DStrebhardt%26aufirst%3DK.%26aulast%3DChesler%26aufirst%3DL.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DBellenie%26aufirst%3DB.%26aulast%3DHoelder%26aufirst%3DS.%26atitle%3DDesigning%2520dual%2520inhibitors%2520of%2520anaplastic%2520lymphoma%2520kinase%2520%2528ALK%2529%2520and%2520bromodomain-4%2520%2528BRD4%2529%2520by%2520tuning%2520kinase%2520selectivity%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D2618%26epage%3D2637%26doi%3D10.1021%2Facs.jmedchem.8b01947" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref103"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref103'); return false;" data-citation="" class="refNumLink">103</a></strong><div class="NLM_citation" id="cit103"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Koblan, L. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buckley, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ott, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fitzgerald, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ember, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, J. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Remillard, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vittori, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schonbrunn, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span> <span> </span><span class="NLM_article-title">Assessment of bromodomain target engagement by a series of BI2536 analogues with miniaturized BET-BRET</span>. <i>ChemMedChem</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">2575</span>– <span class="NLM_lpage">2581</span>, <span class="refDoi"> DOI: 10.1002/cmdc.201600502</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref103/cit103&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1002%2Fcmdc.201600502" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref103/cit103&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=27862999" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref103/cit103&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvVChsbfK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2016&pages=2575-2581&author=L.+W.+Koblanauthor=D.+L.+Buckleyauthor=C.+J.+Ottauthor=M.+E.+Fitzgeraldauthor=S.+W.+Emberauthor=J.+Y.+Zhuauthor=S.+Liuauthor=J.+M.+Robertsauthor=D.+Remillardauthor=S.+Vittoriauthor=W.+Zhangauthor=E.+Schonbrunnauthor=J.+E.+Bradner&title=Assessment+of+bromodomain+target+engagement+by+a+series+of+BI2536+analogues+with+miniaturized+BET-BRET&doi=10.1002%2Fcmdc.201600502"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit103R"><div class="casContent"><span class="casTitleNuber">103</span><div class="casTitle"><span class="NLM_cas:atitle">Assessment of Bromodomain Target Engagement by a Series of BI2536 Analogues with Miniaturized BET-BRET</span></div><div class="casAuthors">Koblan, Luke W.; Buckley, Dennis L.; Ott, Christopher J.; Fitzgerald, Mark E.; Ember, Stuart W. J.; Zhu, Jin-Yi; Liu, Shuai; Roberts, Justin M.; Remillard, David; Vittori, Sarah; Zhang, Wei; Schonbrunn, Ernst; Bradner, James E.</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">2575-2581</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Evaluating the engagement of a small mol. ligand with a protein target in cells provides useful information for chem. probe optimization and pharmaceutical development.  While several techniques exist that can be performed in a low-throughput manner, systematic evaluation of large compd. libraries remains a challenge.  In-cell engagement measurements are esp. useful when evaluating compd. classes suspected to target multiple cellular factors.  In this study we used a bioluminescent resonant energy transfer assay to assess bromodomain engagement by a compd. series contg. bromodomain- and kinase-biasing polypharmacophores based on the known dual BRD4 bromodomain/PLK1 kinase inhibitor BI2536.  With this assay, we discovered several novel agents with bromodomain-selective specificity profiles and cellular activity.  Thus, this platform aids in distinguishing mols. whose cellular activity is difficult to assess due to polypharmacol. effects.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpiVVtnPQvEQrVg90H21EOLACvtfcHk0liFRb0J6SCFqg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvVChsbfK&md5=a6d7c6a58870eb6208e18eb32068d821</span></div><a href="/servlet/linkout?suffix=cit103&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.201600502&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.201600502%26sid%3Dliteratum%253Aachs%26aulast%3DKoblan%26aufirst%3DL.%2BW.%26aulast%3DBuckley%26aufirst%3DD.%2BL.%26aulast%3DOtt%26aufirst%3DC.%2BJ.%26aulast%3DFitzgerald%26aufirst%3DM.%2BE.%26aulast%3DEmber%26aufirst%3DS.%2BW.%26aulast%3DZhu%26aufirst%3DJ.%2BY.%26aulast%3DLiu%26aufirst%3DS.%26aulast%3DRoberts%26aufirst%3DJ.%2BM.%26aulast%3DRemillard%26aufirst%3DD.%26aulast%3DVittori%26aufirst%3DS.%26aulast%3DZhang%26aufirst%3DW.%26aulast%3DSchonbrunn%26aufirst%3DE.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26atitle%3DAssessment%2520of%2520bromodomain%2520target%2520engagement%2520by%2520a%2520series%2520of%2520BI2536%2520analogues%2520with%2520miniaturized%2520BET-BRET%26jtitle%3DChemMedChem%26date%3D2016%26volume%3D11%26spage%3D2575%26epage%3D2581%26doi%3D10.1002%2Fcmdc.201600502" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref104"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref104'); return false;" data-citation="" class="refNumLink">104</a></strong><div class="NLM_citation" id="cit104"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ciceri, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Mahony, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hunt, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lasater, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pallares, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Picaud, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wells, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wodicka, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shah, N. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Treiber, D. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span> <span> </span><span class="NLM_article-title">Dual kinase-bromodomain inhibitors for rationally designed polypharmacology</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">305</span>– <span class="NLM_lpage">312</span>, <span class="refDoi"> DOI: 10.1038/nchembio.1471</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref104/cit104&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1038%2Fnchembio.1471" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref104/cit104&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=24584101" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref104/cit104&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BC2cXjtl2itro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2014&pages=305-312&author=P.+Ciceriauthor=S.+Mullerauthor=A.+O%E2%80%99Mahonyauthor=O.+Fedorovauthor=P.+Filippakopoulosauthor=J.+P.+Huntauthor=E.+A.+Lasaterauthor=G.+Pallaresauthor=S.+Picaudauthor=C.+Wellsauthor=S.+Martinauthor=L.+M.+Wodickaauthor=N.+P.+Shahauthor=D.+K.+Treiberauthor=S.+Knapp&title=Dual+kinase-bromodomain+inhibitors+for+rationally+designed+polypharmacology&doi=10.1038%2Fnchembio.1471"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit104R"><div class="casContent"><span class="casTitleNuber">104</span><div class="casTitle"><span class="NLM_cas:atitle">Dual kinase-bromodomain inhibitors for rationally designed polypharmacology</span></div><div class="casAuthors">Ciceri, Pietro; Mueller, Susanne; O'Mahony, Alison; Fedorov, Oleg; Filippakopoulos, Panagis; Hunt, Jeremy P.; Lasater, Elisabeth A.; Pallares, Gabriel; Picaud, Sarah; Wells, Christopher; Martin, Sarah; Wodicka, Lisa M.; Shah, Neil P.; Treiber, Daniel K.; Knapp, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">305-312</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Concomitant inhibition of multiple cancer-driving kinases is an established strategy to improve the durability of clin. responses to targeted therapies.  The difficulty of discovering kinase inhibitors with an appropriate multitarget profile has, however, necessitated the application of combination therapies, which can pose major clin. development challenges.  Epigenetic reader domains of the bromodomain family have recently emerged as new targets for cancer therapy.  Here the authors report that several clin. kinase inhibitors also inhibit bromodomains with therapeutically relevant potencies and are best classified as dual kinase-bromodomain inhibitors.  Nanomolar activity on BRD4 by BI-2536 and TG-101348, which are clin. PLK1 and JAK2-FLT3 kinase inhibitors, resp., is particularly noteworthy as these combinations of activities on independent oncogenic pathways exemplify a new strategy for rational single-agent polypharmacol. targeting.  Furthermore, structure-activity relationships and co-crystal structures identify design features that enable a general platform for the rational design of dual kinase-bromodomain inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGotPedDED4FJLVg90H21EOLACvtfcHk0lgKpW4lni_LLA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXjtl2itro%253D&md5=a00117db2648ea38dd03750faf3b3cb9</span></div><a href="/servlet/linkout?suffix=cit104&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1471&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1471%26sid%3Dliteratum%253Aachs%26aulast%3DCiceri%26aufirst%3DP.%26aulast%3DMuller%26aufirst%3DS.%26aulast%3DO%25E2%2580%2599Mahony%26aufirst%3DA.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DHunt%26aufirst%3DJ.%2BP.%26aulast%3DLasater%26aufirst%3DE.%2BA.%26aulast%3DPallares%26aufirst%3DG.%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DWells%26aufirst%3DC.%26aulast%3DMartin%26aufirst%3DS.%26aulast%3DWodicka%26aufirst%3DL.%2BM.%26aulast%3DShah%26aufirst%3DN.%2BP.%26aulast%3DTreiber%26aufirst%3DD.%2BK.%26aulast%3DKnapp%26aufirst%3DS.%26atitle%3DDual%2520kinase-bromodomain%2520inhibitors%2520for%2520rationally%2520designed%2520polypharmacology%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2014%26volume%3D10%26spage%3D305%26epage%3D312%26doi%3D10.1038%2Fnchembio.1471" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref105"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref105'); return false;" data-citation="" class="refNumLink">105</a></strong><div class="NLM_citation" id="cit105"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wan, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xia, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duan, G.</span></span> <span> </span><span class="NLM_article-title">Dual-target inhibitors based on HDACs: novel antitumor agents for cancer therapy</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">8977</span>– <span class="NLM_lpage">9002</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.0c00491</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.0c00491" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref105/cit105&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BB3cXnslShtL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=8977-9002&author=T.+Liuauthor=Y.+Wanauthor=Y.+Xiaoauthor=C.+Xiaauthor=G.+Duan&title=Dual-target+inhibitors+based+on+HDACs%3A+novel+antitumor+agents+for+cancer+therapy&doi=10.1021%2Facs.jmedchem.0c00491"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit105R"><div class="casContent"><span class="casTitleNuber">105</span><div class="casTitle"><span class="NLM_cas:atitle">Dual-Target Inhibitors Based on HDACs: Novel Antitumor Agents for Cancer Therapy</span></div><div class="casAuthors">Liu, Tingting; Wan, Yichao; Xiao, Yuliang; Xia, Chengcai; Duan, Guiyun</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">8977-9002</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Histone deacetylases (HDACs) play an important role in regulating target gene expression.  They have been highlighted as a novel category of anticancer targets, and their inhibition can induce apoptosis, differentiation, and growth arrest in cancer cells.  In view of the fact that HDAC inhibitors and other antitumor agents, such as BET inhibitors, topoisomerase inhibitors, and RTK pathway inhibitors, exert a synergistic effect on cellular processes in cancer cells, the combined inhibition of two targets is regarded as a rational strategy to improve the effectiveness of these single-target drugs for cancer treatment.  In this review, we discuss the theor. basis for designing HDAC-involved dual-target drugs and provide insight into the structure-activity relationships of these dual-target agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGohbD4ktOU4jbVg90H21EOLACvtfcHk0lgKpW4lni_LLA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXnslShtL8%253D&md5=fa6764d35879c6a8973d57231854f821</span></div><a href="/servlet/linkout?suffix=cit105&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.0c00491&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.0c00491%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DT.%26aulast%3DWan%26aufirst%3DY.%26aulast%3DXiao%26aufirst%3DY.%26aulast%3DXia%26aufirst%3DC.%26aulast%3DDuan%26aufirst%3DG.%26atitle%3DDual-target%2520inhibitors%2520based%2520on%2520HDACs%253A%2520novel%2520antitumor%2520agents%2520for%2520cancer%2520therapy%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D8977%26epage%3D9002%26doi%3D10.1021%2Facs.jmedchem.0c00491" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref106"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref106'); return false;" data-citation="" class="refNumLink">106</a></strong><div class="NLM_citation" id="cit106"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Carlson, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cui, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Divakaran, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brunner, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pomerantz, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Topczewski, J. J.</span></span> <span> </span><span class="NLM_article-title">Systematically mitigating the p38alpha activity of triazole-based BET inhibitors</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">1296</span>– <span class="NLM_lpage">1301</span>, <span class="refDoi"> DOI: 10.1021/acsmedchemlett.9b00227</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.9b00227" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref106/cit106&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhsFSku7jE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2019&pages=1296-1301&author=A.+S.+Carlsonauthor=H.+Cuiauthor=A.+Divakaranauthor=J.+A.+Johnsonauthor=R.+M.+Brunnerauthor=W.+Pomerantzauthor=J.+J.+Topczewski&title=Systematically+mitigating+the+p38alpha+activity+of+triazole-based+BET+inhibitors&doi=10.1021%2Facsmedchemlett.9b00227"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit106R"><div class="casContent"><span class="casTitleNuber">106</span><div class="casTitle"><span class="NLM_cas:atitle">Systematically Mitigating the p38α Activity of Triazole-based BET Inhibitors</span></div><div class="casAuthors">Carlson, Angela S.; Cui, Huarui; Divakaran, Anand; Johnson, Jorden A.; Brunner, Ryan M.; Pomerantz, William C. K.; Topczewski, Joseph J.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1296-1301</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The Bromodomain and Extra Terminal (BET) family of proteins recognize post-translational N-ε-acetylated lysine modifications and act as "reader" proteins in regulating transcription.  This core function makes bromodomain inhibitors interesting targets for the development of potential cancer, inflammation, and heart disease treatments.  Several dual kinase-bromodomain inhibitors have been identified by screening kinase inhibitor libraries against BET proteins.  Although potentially useful from a polypharmacol. standpoint, multi-target binding complicates deciphering mol. mechanisms.  This report describes a systematic approach to mitigating kinase activity in a dual kinase-bromodomain inhibitor based on a 1,2,3-triazole core.  This SAR study enhanced BET activity while reducing the p38α kinase activity >200,000 fold.  A BRD4-D1 co-crystal structure indicates that the 1,2,3-triazole is acting as a N-ε-acetylated lysine mimic.  A BRD4 sensitive cell line, MM.1S, was used to demonstrate activity in cells, which is further sup-ported by reduced c-Myc expression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo2d-QBOuUZHLVg90H21EOLACvtfcHk0lgKpW4lni_LLA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhsFSku7jE&md5=c4f776357db94fca3bdcd4bdc8f377d8</span></div><a href="/servlet/linkout?suffix=cit106&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.9b00227&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.9b00227%26sid%3Dliteratum%253Aachs%26aulast%3DCarlson%26aufirst%3DA.%2BS.%26aulast%3DCui%26aufirst%3DH.%26aulast%3DDivakaran%26aufirst%3DA.%26aulast%3DJohnson%26aufirst%3DJ.%2BA.%26aulast%3DBrunner%26aufirst%3DR.%2BM.%26aulast%3DPomerantz%26aufirst%3DW.%26aulast%3DTopczewski%26aufirst%3DJ.%2BJ.%26atitle%3DSystematically%2520mitigating%2520the%2520p38alpha%2520activity%2520of%2520triazole-based%2520BET%2520inhibitors%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2019%26volume%3D10%26spage%3D1296%26epage%3D1301%26doi%3D10.1021%2Facsmedchemlett.9b00227" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref107"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref107'); return false;" data-citation="" class="refNumLink">107</a></strong><div class="NLM_citation" id="cit107"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yoon, H. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, J.</span></span> <span> </span><span class="NLM_article-title">Regulation of P-TEFb elongation complex activity by CDK9 acetylation</span>. <i>Mol. Cell. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">4641</span>– <span class="NLM_lpage">4651</span>, <span class="refDoi"> DOI: 10.1128/MCB.00857-06</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref107/cit107&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1128%2FMCB.00857-06" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref107/cit107&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=17452463" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref107/cit107&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BD2sXnsVWrs7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2007&pages=4641-4651&author=J.+Fuauthor=H.+G.+Yoonauthor=J.+Qinauthor=J.+Wong&title=Regulation+of+P-TEFb+elongation+complex+activity+by+CDK9+acetylation&doi=10.1128%2FMCB.00857-06"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit107R"><div class="casContent"><span class="casTitleNuber">107</span><div class="casTitle"><span class="NLM_cas:atitle">Regulation of P-TEFb elongation complex activity by CDK9 acetylation</span></div><div class="casAuthors">Fu, Junjiang; Yoon, Ho-Geun; Qin, Jun; Wong, Jiemin</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Biology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">4641-4651</span>CODEN:
                <span class="NLM_cas:coden">MCEBD4</span>;
        ISSN:<span class="NLM_cas:issn">0270-7306</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">P-TEFb, comprised of CDK9 and a cyclin T subunit, is a global transcriptional elongation factor important for most RNA polymerase II (pol II) transcription.  P-TEFb facilitates transcription elongation in part by phosphorylating Ser2 of the heptapeptide repeat of the carboxy-terminal domain (CTD) of the largest subunit of pol II.  Previous studies have shown that P-TEFb is subjected to neg. regulation by forming an inactive complex with 7SK small RNA and HEXIM1.  In an effort to investigate the mol. mechanism by which corepressor N-CoR mediates transcription repression, we identified HEXIM1 as an N-CoR-interacting protein.  This finding led us to test whether the P-TEFb complex is regulated by acetylation.  We demonstrate that CDK9 is an acetylated protein in cells and can be acetylated by p300 in vitro.  Through both in vitro and in vivo assays, we identified lysine 44 of CDK9 as a major acetylation site.  We present evidence that CDK9 is regulated by N-CoR and its assocd. HDAC3 and that acetylation of CDK9 affects its ability to phosphorylate the CTD of pol II.  These results suggest that acetylation of CDK9 is an important posttranslational modification that is involved in regulating P-TEFb transcriptional elongation function.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo1MWK4pAUTKrVg90H21EOLACvtfcHk0lgxsvoi4p5v0Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXnsVWrs7c%253D&md5=24a3b35574a6f126baa8bcd4a1fd62a8</span></div><a href="/servlet/linkout?suffix=cit107&amp;dbid=16384&amp;doi=10.1128%2FMCB.00857-06&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMCB.00857-06%26sid%3Dliteratum%253Aachs%26aulast%3DFu%26aufirst%3DJ.%26aulast%3DYoon%26aufirst%3DH.%2BG.%26aulast%3DQin%26aufirst%3DJ.%26aulast%3DWong%26aufirst%3DJ.%26atitle%3DRegulation%2520of%2520P-TEFb%2520elongation%2520complex%2520activity%2520by%2520CDK9%2520acetylation%26jtitle%3DMol.%2520Cell.%2520Biol.%26date%3D2007%26volume%3D27%26spage%3D4641%26epage%3D4651%26doi%3D10.1128%2FMCB.00857-06" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref108"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref108'); return false;" data-citation="" class="refNumLink">108</a></strong><div class="NLM_citation" id="cit108"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sakamoto, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, K. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumagai, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mercurio, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deshaies, R. J.</span></span> <span> </span><span class="NLM_article-title">Protacs: chimeric molecules that target proteins to the Skp1-Cullin-F box complex for ubiquitination and degradation</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>98</i></span>,  <span class="NLM_fpage">8554</span>– <span class="NLM_lpage">8559</span>, <span class="refDoi"> DOI: 10.1073/pnas.141230798</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref108/cit108&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1073%2Fpnas.141230798" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref108/cit108&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=11438690" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref108/cit108&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BD3MXls1Wisbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=98&publication_year=2001&pages=8554-8559&author=K.+M.+Sakamotoauthor=K.+B.+Kimauthor=A.+Kumagaiauthor=F.+Mercurioauthor=C.+M.+Crewsauthor=R.+J.+Deshaies&title=Protacs%3A+chimeric+molecules+that+target+proteins+to+the+Skp1-Cullin-F+box+complex+for+ubiquitination+and+degradation&doi=10.1073%2Fpnas.141230798"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit108R"><div class="casContent"><span class="casTitleNuber">108</span><div class="casTitle"><span class="NLM_cas:atitle">Protacs: chimeric molecules that target proteins to the Skp1-Cullin-F box complex for ubiquitination and degradation</span></div><div class="casAuthors">Sakamoto, Kathleen M.; Kim, Kyung B.; Kumagai, Akiko; Mercurio, Frank; Crews, Craig M.; Deshaies, Raymond J.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">98</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">8554-8559</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The intracellular levels of many proteins are regulated by ubiquitin-dependent proteolysis.  One of the best-characterized enzymes that catalyzes the attachment of ubiquitin to proteins is a ubiquitin ligase complex, Skp1-Cullin-F box complex contg. Hrt1 (SCF).  We sought to artificially target a protein to the SCF complex for ubiquitination and degrdn.  To this end, we tested methionine aminopeptidase-2 (MetAP-2), which covalently binds the angiogenesis inhibitor ovalicin.  A chimeric compd., protein-targeting chimeric mol. 1 (Protac-1), was synthesized to recruit MetAP-2 to SCF.  One domain of Protac-1 contains the IκBα phosphopeptide that is recognized by the F-box protein β-TRCP, whereas the other domain is composed of ovalicin.  We show that MetAP-2 can be tethered to SCFβ-TRCP, ubiquitinated, and degraded in a Protac-1-dependent manner.  In the future, this approach may be useful for conditional inactivation of proteins, and for targeting disease-causing proteins for destruction.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp49fYtuEVlJrVg90H21EOLACvtfcHk0lgxsvoi4p5v0Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXls1Wisbk%253D&md5=540358035222f745f7b6367b38781a21</span></div><a href="/servlet/linkout?suffix=cit108&amp;dbid=16384&amp;doi=10.1073%2Fpnas.141230798&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.141230798%26sid%3Dliteratum%253Aachs%26aulast%3DSakamoto%26aufirst%3DK.%2BM.%26aulast%3DKim%26aufirst%3DK.%2BB.%26aulast%3DKumagai%26aufirst%3DA.%26aulast%3DMercurio%26aufirst%3DF.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26aulast%3DDeshaies%26aufirst%3DR.%2BJ.%26atitle%3DProtacs%253A%2520chimeric%2520molecules%2520that%2520target%2520proteins%2520to%2520the%2520Skp1-Cullin-F%2520box%2520complex%2520for%2520ubiquitination%2520and%2520degradation%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2001%26volume%3D98%26spage%3D8554%26epage%3D8559%26doi%3D10.1073%2Fpnas.141230798" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref109"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref109'); return false;" data-citation="" class="refNumLink">109</a></strong><div class="NLM_citation" id="cit109"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lai, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Induced protein degradation: an emerging drug discovery paradigm</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">101</span>– <span class="NLM_lpage">114</span>, <span class="refDoi"> DOI: 10.1038/nrd.2016.211</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref109/cit109&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1038%2Fnrd.2016.211" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref109/cit109&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=27885283" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref109/cit109&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvFCit7rP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2017&pages=101-114&author=A.+C.+Laiauthor=C.+M.+Crews&title=Induced+protein+degradation%3A+an+emerging+drug+discovery+paradigm&doi=10.1038%2Fnrd.2016.211"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit109R"><div class="casContent"><span class="casTitleNuber">109</span><div class="casTitle"><span class="NLM_cas:atitle">Induced protein degradation: an emerging drug discovery paradigm</span></div><div class="casAuthors">Lai, Ashton C.; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">101-114</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Small-mol. drug discovery has traditionally focused on occupancy of a binding site that directly affects protein function, and this approach typically precludes targeting proteins that lack such amenable sites.  Furthermore, high systemic drug exposures may be needed to maintain sufficient target inhibition in vivo, increasing the risk of undesirable off-target effects.  Induced protein degrdn. is an alternative approach that is event-driven: upon drug binding, the target protein is tagged for elimination.  Emerging technologies based on proteolysis-targeting chimaeras (PROTACs) that exploit cellular quality control machinery to selectively degrade target proteins are attracting considerable attention in the pharmaceutical industry owing to the advantages they could offer over traditional small-mol. strategies.  These advantages include the potential to reduce systemic drug exposure, the ability to counteract increased target protein expression that often accompanies inhibition of protein function and the potential ability to target proteins that are not currently therapeutically tractable, such as transcription factors, scaffolding and regulatory proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoj-O7zXGOpXLVg90H21EOLACvtfcHk0lgxsvoi4p5v0Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvFCit7rP&md5=a01ff43e9b696fe4dc03f0815158a98f</span></div><a href="/servlet/linkout?suffix=cit109&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2016.211&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2016.211%26sid%3Dliteratum%253Aachs%26aulast%3DLai%26aufirst%3DA.%2BC.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DInduced%2520protein%2520degradation%253A%2520an%2520emerging%2520drug%2520discovery%2520paradigm%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2017%26volume%3D16%26spage%3D101%26epage%3D114%26doi%3D10.1038%2Fnrd.2016.211" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref110"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref110'); return false;" data-citation="" class="refNumLink">110</a></strong><div class="NLM_citation" id="cit110"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bai, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fernandez-Salas, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mceachern, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Przybranowski, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wen, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span> <span> </span><span class="NLM_article-title">Discovery of a small-molecule degrader of bromodomain and extra-terminal (BET) proteins with picomolar cellular potencies and capable of achieving tumor regression</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">462</span>– <span class="NLM_lpage">481</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b01816</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01816" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref110/cit110&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BC2sXkvVajsb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=462-481&author=B.+Zhouauthor=J.+Huauthor=F.+Xuauthor=Z.+Chenauthor=L.+Baiauthor=E.+Fernandez-Salasauthor=M.+Linauthor=L.+Liuauthor=C.+Y.+Yangauthor=Y.+Zhaoauthor=D.+Mceachernauthor=S.+Przybranowskiauthor=B.+Wenauthor=D.+Sunauthor=S.+Wang&title=Discovery+of+a+small-molecule+degrader+of+bromodomain+and+extra-terminal+%28BET%29+proteins+with+picomolar+cellular+potencies+and+capable+of+achieving+tumor+regression&doi=10.1021%2Facs.jmedchem.6b01816"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit110R"><div class="casContent"><span class="casTitleNuber">110</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a Small-Molecule Degrader of Bromodomain and Extra-Terminal (BET) Proteins with Picomolar Cellular Potencies and Capable of Achieving Tumor Regression</span></div><div class="casAuthors">Zhou, Bing; Hu, Jiantao; Xu, Fuming; Chen, Zhuo; Bai, Longchuan; Fernandez-Salas, Ester; Lin, Mei; Liu, Liu; Yang, Chao-Yie; Zhao, Yujun; McEachern, Donna; Przybranowski, Sally; Wen, Bo; Sun, Duxin; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">462-481</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The bromodomain and extra-terminal (BET) family proteins, consisting of BRD2, BRD3, BRD4, and testis-specific BRDT members, are epigenetic "readers" and play a key role in the regulation of gene transcription.  BET proteins are considered to be attractive therapeutic targets for cancer and other human diseases.  Recently, heterobifunctional small-mol. BET degraders have been designed based upon the proteolysis targeting chimera (PROTAC) concept to induce BET protein degrdn.  Herein, we present our design, synthesis, and evaluation of a new class of PROTAC BET degraders.  One of the most promising compds., 23, effectively degrades BRD4 protein at concns. as low as 30 pM in the RS4;11 leukemia cell line, achieves an IC50 value of 51 pM in inhibition of RS4;11 cell growth and induces rapid tumor regression in vivo against RS4;11 xenograft tumors.  These data establish that compd. 23 (BETd-260/ZBC260) is a highly potent and efficacious BET degrader.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqqz2Lwp52x1rVg90H21EOLACvtfcHk0lhnNmRQKhXvlA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXkvVajsb8%253D&md5=6b3a10ecd192a5812ef581dca1fdc82a</span></div><a href="/servlet/linkout?suffix=cit110&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01816&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01816%26sid%3Dliteratum%253Aachs%26aulast%3DZhou%26aufirst%3DB.%26aulast%3DHu%26aufirst%3DJ.%26aulast%3DXu%26aufirst%3DF.%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DBai%26aufirst%3DL.%26aulast%3DFernandez-Salas%26aufirst%3DE.%26aulast%3DLin%26aufirst%3DM.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DYang%26aufirst%3DC.%2BY.%26aulast%3DZhao%26aufirst%3DY.%26aulast%3DMceachern%26aufirst%3DD.%26aulast%3DPrzybranowski%26aufirst%3DS.%26aulast%3DWen%26aufirst%3DB.%26aulast%3DSun%26aufirst%3DD.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DDiscovery%2520of%2520a%2520small-molecule%2520degrader%2520of%2520bromodomain%2520and%2520extra-terminal%2520%2528BET%2529%2520proteins%2520with%2520picomolar%2520cellular%2520potencies%2520and%2520capable%2520of%2520achieving%2520tumor%2520regression%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D462%26epage%3D481%26doi%3D10.1021%2Facs.jmedchem.6b01816" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref111"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref111'); return false;" data-citation="" class="refNumLink">111</a></strong><div class="NLM_citation" id="cit111"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Winter, G. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buckley, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paulk, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Souza, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dhe-Paganon, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span> <span> </span><span class="NLM_article-title">DRUG DEVELOPMENT. Phthalimide conjugation as a strategy for in vivo target protein degradation</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>348</i></span>,  <span class="NLM_fpage">1376</span>– <span class="NLM_lpage">1381</span>, <span class="refDoi"> DOI: 10.1126/science.aab1433</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref111/cit111&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1126%2Fscience.aab1433" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref111/cit111&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=25999370" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref111/cit111&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BC2MXpvFansLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=348&publication_year=2015&pages=1376-1381&author=G.+E.+Winterauthor=D.+L.+Buckleyauthor=J.+Paulkauthor=J.+M.+Robertsauthor=A.+Souzaauthor=S.+Dhe-Paganonauthor=J.+E.+Bradner&title=DRUG+DEVELOPMENT.+Phthalimide+conjugation+as+a+strategy+for+in+vivo+target+protein+degradation&doi=10.1126%2Fscience.aab1433"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit111R"><div class="casContent"><span class="casTitleNuber">111</span><div class="casTitle"><span class="NLM_cas:atitle">Phthalimide conjugation as a strategy for in vivo target protein degradation</span></div><div class="casAuthors">Winter, Georg E.; Buckley, Dennis L.; Paulk, Joshiawa; Roberts, Justin M.; Souza, Amanda; Dhe-Paganon, Sirano; Bradner, James E.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">348</span>
        (<span class="NLM_cas:issue">6241</span>),
    <span class="NLM_cas:pages">1376-1381</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">The development of effective pharmacol. inhibitors of multidomain scaffold proteins, notably transcription factors, is a particularly challenging problem.  In part, this is because many small-mol. antagonists disrupt the activity of only one domain in the target protein.  The authors devised a chem. strategy that promotes ligand-dependent target protein degrdn. using as an example the transcriptional coactivator BRD4, a protein crit. for cancer cell growth and survival.  The authors appended a competitive antagonist of BET bromodomains to a phthalimide moiety to hijack the cereblon E3 ubiquitin ligase complex.  The resultant compd., dBET1, induced highly selective cereblon-dependent BET protein degrdn. in vitro and in vivo and delayed leukemia progression in mice.  A second series of probes resulted in selective degrdn. of the cytosolic protein FKBP12.  This chem. strategy for controlling target protein stability may have implications for therapeutically targeting previously intractable proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp1ZW4zjRjXi7Vg90H21EOLACvtfcHk0lhnNmRQKhXvlA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXpvFansLk%253D&md5=b0277cc95d318d926b96e7a119066e3b</span></div><a href="/servlet/linkout?suffix=cit111&amp;dbid=16384&amp;doi=10.1126%2Fscience.aab1433&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.aab1433%26sid%3Dliteratum%253Aachs%26aulast%3DWinter%26aufirst%3DG.%2BE.%26aulast%3DBuckley%26aufirst%3DD.%2BL.%26aulast%3DPaulk%26aufirst%3DJ.%26aulast%3DRoberts%26aufirst%3DJ.%2BM.%26aulast%3DSouza%26aufirst%3DA.%26aulast%3DDhe-Paganon%26aufirst%3DS.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26atitle%3DDRUG%2520DEVELOPMENT.%2520Phthalimide%2520conjugation%2520as%2520a%2520strategy%2520for%2520in%2520vivo%2520target%2520protein%2520degradation%26jtitle%3DScience%26date%3D2015%26volume%3D348%26spage%3D1376%26epage%3D1381%26doi%3D10.1126%2Fscience.aab1433" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref112"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref112'); return false;" data-citation="" class="refNumLink">112</a></strong><div class="NLM_citation" id="cit112"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qian, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Altieri, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raina, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hines, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winkler, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crew, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coleman, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Hijacking the E3 ubiquitin ligase cereblon to efficiently target BRD4</span>. <i>Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">755</span>– <span class="NLM_lpage">763</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2015.05.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref112/cit112&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1016%2Fj.chembiol.2015.05.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref112/cit112&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=26051217" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref112/cit112&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtVWrtbrM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2015&pages=755-763&author=J.+Luauthor=Y.+Qianauthor=M.+Altieriauthor=H.+Dongauthor=J.+Wangauthor=K.+Rainaauthor=J.+Hinesauthor=J.+D.+Winklerauthor=A.+P.+Crewauthor=K.+Colemanauthor=C.+M.+Crews&title=Hijacking+the+E3+ubiquitin+ligase+cereblon+to+efficiently+target+BRD4&doi=10.1016%2Fj.chembiol.2015.05.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit112R"><div class="casContent"><span class="casTitleNuber">112</span><div class="casTitle"><span class="NLM_cas:atitle">Hijacking the E3 Ubiquitin Ligase Cereblon to Efficiently Target BRD4</span></div><div class="casAuthors">Lu, Jing; Qian, Yimin; Altieri, Martha; Dong, Hanqing; Wang, Jing; Raina, Kanak; Hines, John; Winkler, James D.; Crew, Andrew P.; Coleman, Kevin; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">755-763</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">BRD4, a bromodomain and extraterminal domain (BET) family member, is an attractive target in multiple pathol. settings, particularly cancer.  While BRD4 inhibitors have shown some promise in MYC-driven malignancies such as Burkitt's lymphoma (BL), we show that BRD4 inhibitors lead to robust BRD4 protein accumulation, which may account for their limited suppression of MYC expression, modest antiproliferative activity, and lack of apoptotic induction.  To address these limitations we designed ARV-825, a hetero-bifunctional PROTAC (Proteolysis Targeting Chimera) that recruits BRD4 to the E3 ubiquitin ligase cereblon, leading to fast, efficient, and prolonged degrdn. of BRD4 in all BL cell lines tested.  Consequently, ARV-825 more effectively suppresses c-MYC levels and downstream signaling than small-mol. BRD4 inhibitors, resulting in more effective cell proliferation inhibition and apoptosis induction in BL.  Our findings provide strong evidence that cereblon-based PROTACs provide a better and more efficient strategy in targeting BRD4 than traditional small-mol. inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq6Ol4grCG2RbVg90H21EOLACvtfcHk0lhnNmRQKhXvlA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtVWrtbrM&md5=a64675e7a0cd006a8737be9730f447e3</span></div><a href="/servlet/linkout?suffix=cit112&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2015.05.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2015.05.009%26sid%3Dliteratum%253Aachs%26aulast%3DLu%26aufirst%3DJ.%26aulast%3DQian%26aufirst%3DY.%26aulast%3DAltieri%26aufirst%3DM.%26aulast%3DDong%26aufirst%3DH.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DRaina%26aufirst%3DK.%26aulast%3DHines%26aufirst%3DJ.%26aulast%3DWinkler%26aufirst%3DJ.%2BD.%26aulast%3DCrew%26aufirst%3DA.%2BP.%26aulast%3DColeman%26aufirst%3DK.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DHijacking%2520the%2520E3%2520ubiquitin%2520ligase%2520cereblon%2520to%2520efficiently%2520target%2520BRD4%26jtitle%3DChem.%2520Biol.%26date%3D2015%26volume%3D22%26spage%3D755%26epage%3D763%26doi%3D10.1016%2Fj.chembiol.2015.05.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref113"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref113'); return false;" data-citation="" class="refNumLink">113</a></strong><div class="NLM_citation" id="cit113"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bai, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ji, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mceachern, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Przybranowski, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fernandez-Salas, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dou, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wen, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meagher, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stuckey, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayes, D. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ellis, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span> <span> </span><span class="NLM_article-title">Targeted degradation of BET proteins in triple-negative breast cancer</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>77</i></span>,  <span class="NLM_fpage">2476</span>– <span class="NLM_lpage">2487</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.CAN-16-2622</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref113/cit113&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1158%2F0008-5472.CAN-16-2622" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref113/cit113&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=28209615" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref113/cit113&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BC2sXmvFakurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=77&publication_year=2017&pages=2476-2487&author=L.+Baiauthor=B.+Zhouauthor=C.+Y.+Yangauthor=J.+Jiauthor=D.+Mceachernauthor=S.+Przybranowskiauthor=H.+Jiangauthor=J.+Huauthor=F.+Xuauthor=Y.+Zhaoauthor=L.+Liuauthor=E.+Fernandez-Salasauthor=J.+Xuauthor=Y.+Douauthor=B.+Wenauthor=D.+Sunauthor=J.+Meagherauthor=J.+Stuckeyauthor=D.+F.+Hayesauthor=S.+Liauthor=M.+J.+Ellisauthor=S.+Wang&title=Targeted+degradation+of+BET+proteins+in+triple-negative+breast+cancer&doi=10.1158%2F0008-5472.CAN-16-2622"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit113R"><div class="casContent"><span class="casTitleNuber">113</span><div class="casTitle"><span class="NLM_cas:atitle">Targeted Degradation of BET Proteins in Triple-Negative Breast Cancer</span></div><div class="casAuthors">Bai, Longchuan; Zhou, Bing; Yang, Chao-Yie; Ji, Jiao; McEachern, Donna; Przybranowski, Sally; Jiang, Hui; Hu, Jiantao; Xu, Fuming; Zhao, Yujun; Liu, Liu; Fernandez-Salas, Ester; Xu, Jing; Dou, Yali; Wen, Bo; Sun, Duxin; Meagher, Jennifer; Stuckey, Jeanne; Hayes, Daniel F.; Li, Shunqiang; Ellis, Matthew J.; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">77</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2476-2487</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Triple-neg. breast cancers (TNBC) remain clin. challenging with a lack of options for targeted therapy.  In this study, we report the development of a second-generation BET protein degrader, BETd-246, which exhibits superior selectivity, potency, and antitumor activity.  In human TNBC cells, BETd-246 induced degrdn. of BET proteins at low nanomolar concns. within 1 h of exposure, resulting in robust growth inhibition and apoptosis.  BETd-246 was more potent and effective in TNBC cells than its parental BET inhibitor compd. BETi-211.  RNA-seq anal. revealed predominant downregulation of a large no. of genes involved in proliferation and apoptosis in cells treated with BETd-246, as compared with BETi-211 treatment that upregulated and downregulated a similar no. of genes.  Functional investigations identified the MCL1 gene as a crit. downstream effector for BET degraders, which synergized with small-mol. inhibitors of BCL-xL in triggering apoptosis.  In multiple murine xenograft models of human breast cancer, BETd-246 and a further optimized analog BETd-260 effectively depleted BET proteins in tumors and exhibited strong antitumor activities at well-tolerated dosing schedules.  Overall, our findings show that targeting BET proteins for degrdn. represents an effective therapeutic strategy for TNBC treatment.  Cancer Res; 77(9); 2476-87. ©2017 AACR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq8M1VaSNIBCLVg90H21EOLACvtfcHk0lipI-ojDZr7dg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXmvFakurg%253D&md5=84c0e0b23e61a5503f8d88385cc88ae5</span></div><a href="/servlet/linkout?suffix=cit113&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-16-2622&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-16-2622%26sid%3Dliteratum%253Aachs%26aulast%3DBai%26aufirst%3DL.%26aulast%3DZhou%26aufirst%3DB.%26aulast%3DYang%26aufirst%3DC.%2BY.%26aulast%3DJi%26aufirst%3DJ.%26aulast%3DMceachern%26aufirst%3DD.%26aulast%3DPrzybranowski%26aufirst%3DS.%26aulast%3DJiang%26aufirst%3DH.%26aulast%3DHu%26aufirst%3DJ.%26aulast%3DXu%26aufirst%3DF.%26aulast%3DZhao%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DFernandez-Salas%26aufirst%3DE.%26aulast%3DXu%26aufirst%3DJ.%26aulast%3DDou%26aufirst%3DY.%26aulast%3DWen%26aufirst%3DB.%26aulast%3DSun%26aufirst%3DD.%26aulast%3DMeagher%26aufirst%3DJ.%26aulast%3DStuckey%26aufirst%3DJ.%26aulast%3DHayes%26aufirst%3DD.%2BF.%26aulast%3DLi%26aufirst%3DS.%26aulast%3DEllis%26aufirst%3DM.%2BJ.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DTargeted%2520degradation%2520of%2520BET%2520proteins%2520in%2520triple-negative%2520breast%2520cancer%26jtitle%3DCancer%2520Res.%26date%3D2017%26volume%3D77%26spage%3D2476%26epage%3D2487%26doi%3D10.1158%2F0008-5472.CAN-16-2622" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref114"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref114'); return false;" data-citation="" class="refNumLink">114</a></strong><div class="NLM_citation" id="cit114"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Qin, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fernandez-Salas, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mceachern, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Przybranowski, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stuckey, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meagher, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bai, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ziazadeh, D. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wen, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span> <span> </span><span class="NLM_article-title">Discovery of QCA570 as an exceptionally potent and efficacious proteolysis targeting chimera (PROTAC) degrader of the bromodomain and extra-terminal (BET)proteins capable of inducing complete and durable tumor regression</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">6685</span>– <span class="NLM_lpage">6704</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b00506</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b00506" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref114/cit114&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtlaqsLrE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=6685-6704&author=C.+Qinauthor=Y.+Huauthor=B.+Zhouauthor=E.+Fernandez-Salasauthor=C.+Y.+Yangauthor=L.+Liuauthor=D.+Mceachernauthor=S.+Przybranowskiauthor=M.+Wangauthor=J.+Stuckeyauthor=J.+Meagherauthor=L.+Baiauthor=Z.+Chenauthor=M.+Linauthor=J.+Yangauthor=D.+N.+Ziazadehauthor=F.+Xuauthor=J.+Huauthor=W.+Xiangauthor=L.+Huangauthor=S.+Liauthor=B.+Wenauthor=D.+Sunauthor=S.+Wang&title=Discovery+of+QCA570+as+an+exceptionally+potent+and+efficacious+proteolysis+targeting+chimera+%28PROTAC%29+degrader+of+the+bromodomain+and+extra-terminal+%28BET%29proteins+capable+of+inducing+complete+and+durable+tumor+regression&doi=10.1021%2Facs.jmedchem.8b00506"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit114R"><div class="casContent"><span class="casTitleNuber">114</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of QCA570 as an Exceptionally Potent and Efficacious Proteolysis Targeting Chimera (PROTAC) Degrader of the Bromodomain and Extra-Terminal (BET) Proteins Capable of Inducing Complete and Durable Tumor Regression</span></div><div class="casAuthors">Qin, Chong; Hu, Yang; Zhou, Bing; Fernandez-Salas, Ester; Yang, Chao-Yie; Liu, Liu; McEachern, Donna; Przybranowski, Sally; Wang, Mi; Stuckey, Jeanne; Meagher, Jennifer; Bai, Longchuan; Chen, Zhuo; Lin, Mei; Yang, Jiuling; Ziazadeh, Danya N.; Xu, Fuming; Hu, Jiantao; Xing, Weiguo; Huang, Liyue; Li, Siwei; Wen, Bo; Sun, Duxin; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">6685-6704</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Proteins of the bromodomain and extra-terminal (BET) family are epigenetics "readers" and promising therapeutic targets for cancer and other human diseases.  We describe herein a structure-guided design of [1,4]oxazepines as a new class of BET inhibitors and our subsequent design, synthesis, and evaluation of proteolysis-targeting chimeric (PROTAC) small-mol. BET degraders.  Our efforts have led to the discovery of extremely potent BET degraders, exemplified by QCA570, which effectively induces degrdn. of BET proteins and inhibits cell growth in human acute leukemia cell lines even at low picomolar concns.  QCA570 achieves complete and durable tumor regression in leukemia xenograft models in mice at well-tolerated dose-schedules.  QCA570 is the most potent and efficacious BET degrader reported to date.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo6ZUIsfX3p6LVg90H21EOLACvtfcHk0lipI-ojDZr7dg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtlaqsLrE&md5=2a731730641372f4526543e03d9ac76f</span></div><a href="/servlet/linkout?suffix=cit114&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b00506&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b00506%26sid%3Dliteratum%253Aachs%26aulast%3DQin%26aufirst%3DC.%26aulast%3DHu%26aufirst%3DY.%26aulast%3DZhou%26aufirst%3DB.%26aulast%3DFernandez-Salas%26aufirst%3DE.%26aulast%3DYang%26aufirst%3DC.%2BY.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DMceachern%26aufirst%3DD.%26aulast%3DPrzybranowski%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DStuckey%26aufirst%3DJ.%26aulast%3DMeagher%26aufirst%3DJ.%26aulast%3DBai%26aufirst%3DL.%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DLin%26aufirst%3DM.%26aulast%3DYang%26aufirst%3DJ.%26aulast%3DZiazadeh%26aufirst%3DD.%2BN.%26aulast%3DXu%26aufirst%3DF.%26aulast%3DHu%26aufirst%3DJ.%26aulast%3DXiang%26aufirst%3DW.%26aulast%3DHuang%26aufirst%3DL.%26aulast%3DLi%26aufirst%3DS.%26aulast%3DWen%26aufirst%3DB.%26aulast%3DSun%26aufirst%3DD.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DDiscovery%2520of%2520QCA570%2520as%2520an%2520exceptionally%2520potent%2520and%2520efficacious%2520proteolysis%2520targeting%2520chimera%2520%2528PROTAC%2529%2520degrader%2520of%2520the%2520bromodomain%2520and%2520extra-terminal%2520%2528BET%2529proteins%2520capable%2520of%2520inducing%2520complete%2520and%2520durable%2520tumor%2520regression%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D6685%26epage%3D6704%26doi%3D10.1021%2Facs.jmedchem.8b00506" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref115"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref115'); return false;" data-citation="" class="refNumLink">115</a></strong><div class="NLM_citation" id="cit115"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mi, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pei, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duan, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span> <span> </span><span class="NLM_article-title">In vivo target protein degradation induced by PROTACs based on E3 ligase DCAF15</span>. <i>Signal Transduction Targeted Ther.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">129</span>, <span class="refDoi"> DOI: 10.1038/s41392-020-00245-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref115/cit115&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1038%2Fs41392-020-00245-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref115/cit115&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=32713946" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref115/cit115&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhsVGlu77N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2020&pages=129&author=L.+Liauthor=D.+Miauthor=H.+Peiauthor=Q.+Duanauthor=X.+Wangauthor=W.+Zhouauthor=J.+Jinauthor=D.+Liauthor=M.+Liuauthor=Y.+Chen&title=In+vivo+target+protein+degradation+induced+by+PROTACs+based+on+E3+ligase+DCAF15&doi=10.1038%2Fs41392-020-00245-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit115R"><div class="casContent"><span class="casTitleNuber">115</span><div class="casTitle"><span class="NLM_cas:atitle">In vivo target protein degradation induced by PROTACs based on E3 ligase DCAF15</span></div><div class="casAuthors">Li, Liang; Mi, Dazhao; Pei, Haixiang; Duan, Qiuhui; Wang, Xinyue; Zhou, Wenbo; Jin, Jianping; Li, Dali; Liu, Mingyao; Chen, Yihua</div><div class="citationInfo"><span class="NLM_cas:title">Signal Transduction and Targeted Therapy</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">129</span>CODEN:
                <span class="NLM_cas:coden">STTTCB</span>;
        ISSN:<span class="NLM_cas:issn">2059-3635</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Proteolysis targeting chimeras (PROTACs) facilitate ubiquitination and degrdn. of the targets by cellular endogenous proteasome system.  To design PROTACs hijacking the E3 ligase complex CRL4-DCAF15, BRD4 was selected as the target protein since which is well-studied member of BET.  To investigate the mechanism of DP1-induced BRD4 degrdn., we explored whether DP1 induced degrdn. of BRD4 isdependent on binding to both BRD4 and DCAF15 and proteasome functions.  Collectively, our study provides a series of evidence that the DCAF15 can serve as a valuable addn. to the limited E3 ligases used in PROTACs design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqZi2tCZj8aWLVg90H21EOLACvtfcHk0lipI-ojDZr7dg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhsVGlu77N&md5=a9b7410e4316998f21009da41db6f8c0</span></div><a href="/servlet/linkout?suffix=cit115&amp;dbid=16384&amp;doi=10.1038%2Fs41392-020-00245-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41392-020-00245-0%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DL.%26aulast%3DMi%26aufirst%3DD.%26aulast%3DPei%26aufirst%3DH.%26aulast%3DDuan%26aufirst%3DQ.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DZhou%26aufirst%3DW.%26aulast%3DJin%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DD.%26aulast%3DLiu%26aufirst%3DM.%26aulast%3DChen%26aufirst%3DY.%26atitle%3DIn%2520vivo%2520target%2520protein%2520degradation%2520induced%2520by%2520PROTACs%2520based%2520on%2520E3%2520ligase%2520DCAF15%26jtitle%3DSignal%2520Transduction%2520Targeted%2520Ther.%26date%3D2020%26volume%3D5%26spage%3D129%26doi%3D10.1038%2Fs41392-020-00245-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref116"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref116'); return false;" data-citation="" class="refNumLink">116</a></strong><div class="NLM_citation" id="cit116"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nowak, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deangelo, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buckley, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Donovan, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">An, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Safaee, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jedrychowski, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ponthier, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ishoey, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mancias, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fischer, E. S.</span></span> <span> </span><span class="NLM_article-title">Plasticity in binding confers selectivity in ligand-induced protein degradation</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">706</span>– <span class="NLM_lpage">714</span>, <span class="refDoi"> DOI: 10.1038/s41589-018-0055-y</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref116/cit116&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1038%2Fs41589-018-0055-y" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref116/cit116&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=29892083" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref116/cit116&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtFWlurbK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2018&pages=706-714&author=R.+P.+Nowakauthor=S.+L.+Deangeloauthor=D.+Buckleyauthor=Z.+Heauthor=K.+A.+Donovanauthor=J.+Anauthor=N.+Safaeeauthor=M.+P.+Jedrychowskiauthor=C.+M.+Ponthierauthor=M.+Ishoeyauthor=T.+Zhangauthor=J.+D.+Manciasauthor=N.+S.+Grayauthor=J.+E.+Bradnerauthor=E.+S.+Fischer&title=Plasticity+in+binding+confers+selectivity+in+ligand-induced+protein+degradation&doi=10.1038%2Fs41589-018-0055-y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit116R"><div class="casContent"><span class="casTitleNuber">116</span><div class="casTitle"><span class="NLM_cas:atitle">Plasticity in binding confers selectivity in ligand-induced protein degradation</span></div><div class="casAuthors">Nowak, Radoslaw P.; De Angelo, Stephen L.; Buckley, Dennis; He, Zhixiang; Donovan, Katherine A.; An, Jian; Safaee, Nozhat; Jedrychowski, Mark P.; Ponthier, Charles M.; Ishoey, Mette; Zhang, Tinghu; Mancias, Joseph D.; Gray, Nathanael S.; Bradner, James E.; Fischer, Eric S.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">706-714</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Heterobifunctional small-mol. degraders that induce protein degrdn. through ligase-mediated ubiquitination have shown considerable promise as a new pharmacol. modality.  However, we currently lack a detailed understanding of the mol. basis for target recruitment and selectivity, which is critically required to enable rational design of degraders.  Here we utilized a comprehensive characterization of the ligand-dependent CRBN-BRD4 interaction to demonstrate that binding between proteins that have not evolved to interact is plastic.  Multiple x-ray crystal structures showed that plasticity resulted in several distinct low-energy binding conformations that were selectively bound by ligands.  We demonstrated that computational protein-protein docking can reveal the underlying interprotein contacts and inform the design of a BRD4 selective degrader that can discriminate between highly homologous BET bromodomains.  These findings that plastic interprotein contacts confer selectivity for ligand-induced protein dimerization provide a conceptual framework for the development of heterobifunctional ligands.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGreQasCW8bz2LVg90H21EOLACvtfcHk0lgOMafTVWIGbA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtFWlurbK&md5=e4876c43b1eeb4558c23121b7a9fac27</span></div><a href="/servlet/linkout?suffix=cit116&amp;dbid=16384&amp;doi=10.1038%2Fs41589-018-0055-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41589-018-0055-y%26sid%3Dliteratum%253Aachs%26aulast%3DNowak%26aufirst%3DR.%2BP.%26aulast%3DDeangelo%26aufirst%3DS.%2BL.%26aulast%3DBuckley%26aufirst%3DD.%26aulast%3DHe%26aufirst%3DZ.%26aulast%3DDonovan%26aufirst%3DK.%2BA.%26aulast%3DAn%26aufirst%3DJ.%26aulast%3DSafaee%26aufirst%3DN.%26aulast%3DJedrychowski%26aufirst%3DM.%2BP.%26aulast%3DPonthier%26aufirst%3DC.%2BM.%26aulast%3DIshoey%26aufirst%3DM.%26aulast%3DZhang%26aufirst%3DT.%26aulast%3DMancias%26aufirst%3DJ.%2BD.%26aulast%3DGray%26aufirst%3DN.%2BS.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26aulast%3DFischer%26aufirst%3DE.%2BS.%26atitle%3DPlasticity%2520in%2520binding%2520confers%2520selectivity%2520in%2520ligand-induced%2520protein%2520degradation%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2018%26volume%3D14%26spage%3D706%26epage%3D714%26doi%3D10.1038%2Fs41589-018-0055-y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref117"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref117'); return false;" data-citation="" class="refNumLink">117</a></strong><div class="NLM_citation" id="cit117"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gadd, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Testa, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lucas, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lamont, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zengerle, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span> <span> </span><span class="NLM_article-title">Structural basis of PROTAC cooperative recognition for selective protein degradation</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">514</span>– <span class="NLM_lpage">521</span>, <span class="refDoi"> DOI: 10.1038/nchembio.2329</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref117/cit117&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1038%2Fnchembio.2329" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref117/cit117&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=28288108" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref117/cit117&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BC2sXkvFahtbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2017&pages=514-521&author=M.+S.+Gaddauthor=A.+Testaauthor=X.+Lucasauthor=K.+H.+Chanauthor=W.+Chenauthor=D.+J.+Lamontauthor=M.+Zengerleauthor=A.+Ciulli&title=Structural+basis+of+PROTAC+cooperative+recognition+for+selective+protein+degradation&doi=10.1038%2Fnchembio.2329"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit117R"><div class="casContent"><span class="casTitleNuber">117</span><div class="casTitle"><span class="NLM_cas:atitle">Structural basis of PROTAC cooperative recognition for selective protein degradation</span></div><div class="casAuthors">Gadd, Morgan S.; Testa, Andrea; Lucas, Xavier; Chan, Kwok-Ho; Chen, Wenzhang; Lamont, Douglas J.; Zengerle, Michael; Ciulli, Alessio</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">514-521</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Inducing macromol. interactions with small mols. to activate cellular signaling is a challenging goal.  PROTACs (proteolysis-targeting chimeras) are bifunctional mols. that recruit a target protein in proximity to an E3 ubiquitin ligase to trigger protein degrdn.  Structural elucidation of the key ternary ligase-PROTAC-target species and its impact on target degrdn. selectivity remain elusive.  Here, we solved the crystal structure of Brd4 degrader MZ1 in complex with human VHL and the Brd4 bromodomain (Brd4BD2).  The ligand folds into itself to allow formation of specific intermol. interactions in the ternary complex.  Isothermal titrn. calorimetry studies, supported by surface mutagenesis and proximity assays, were consistent with pronounced cooperative formation of ternary complexes with Brd4BD2.  Structure-based-designed compd. AT1 exhibited highly selective depletion of Brd4 in cells.  These results elucidate how PROTAC-induced de novo contacts dictate preferential recruitment of a target protein into a stable and cooperative complex with an E3 ligase for selective degrdn.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoJri00jB5YJLVg90H21EOLACvtfcHk0lgOMafTVWIGbA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXkvFahtbg%253D&md5=dfee13913106d362d0972d7b1a62d700</span></div><a href="/servlet/linkout?suffix=cit117&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.2329&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.2329%26sid%3Dliteratum%253Aachs%26aulast%3DGadd%26aufirst%3DM.%2BS.%26aulast%3DTesta%26aufirst%3DA.%26aulast%3DLucas%26aufirst%3DX.%26aulast%3DChan%26aufirst%3DK.%2BH.%26aulast%3DChen%26aufirst%3DW.%26aulast%3DLamont%26aufirst%3DD.%2BJ.%26aulast%3DZengerle%26aufirst%3DM.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DStructural%2520basis%2520of%2520PROTAC%2520cooperative%2520recognition%2520for%2520selective%2520protein%2520degradation%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2017%26volume%3D13%26spage%3D514%26epage%3D521%26doi%3D10.1038%2Fnchembio.2329" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref118"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref118'); return false;" data-citation="" class="refNumLink">118</a></strong><div class="NLM_citation" id="cit118"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Raina, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qian, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Altieri, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rossi, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winkler, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crew, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coleman, K. G.</span></span> <span> </span><span class="NLM_article-title">PROTAC-induced BET protein degradation as a therapy for castration-resistant prostate cancer</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>113</i></span>,  <span class="NLM_fpage">7124</span>– <span class="NLM_lpage">7129</span>, <span class="refDoi"> DOI: 10.1073/pnas.1521738113</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref118/cit118&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1073%2Fpnas.1521738113" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref118/cit118&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=27274052" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref118/cit118&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BC28Xptlyqs7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=113&publication_year=2016&pages=7124-7129&author=K.+Rainaauthor=J.+Luauthor=Y.+Qianauthor=M.+Altieriauthor=D.+Gordonauthor=A.+M.+Rossiauthor=J.+Wangauthor=X.+Chenauthor=H.+Dongauthor=K.+Siuauthor=J.+D.+Winklerauthor=A.+P.+Crewauthor=C.+M.+Crewsauthor=K.+G.+Coleman&title=PROTAC-induced+BET+protein+degradation+as+a+therapy+for+castration-resistant+prostate+cancer&doi=10.1073%2Fpnas.1521738113"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit118R"><div class="casContent"><span class="casTitleNuber">118</span><div class="casTitle"><span class="NLM_cas:atitle">PROTAC-induced BET protein degradation as a therapy for castration-resistant prostate cancer</span></div><div class="casAuthors">Raina, Kanak; Lu, Jing; Qian, Yimin; Altieri, Martha; Gordon, Deborah; Rossi, Ann Marie K.; Wang, Jing; Chen, Xin; Dong, Hanqing; Siu, Kam; Winkler, James D.; Crew, Andrew P.; Crews, Craig M.; Coleman, Kevin G.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">113</span>
        (<span class="NLM_cas:issue">26</span>),
    <span class="NLM_cas:pages">7124-7129</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Prostate cancer has the second highest incidence among cancers in men worldwide and is the second leading cause of cancer deaths of men in the United States.  Although androgen deprivation can initially lead to remission, the disease often progresses to castration-resistant prostate cancer (CRPC), which is still reliant on androgen receptor (AR) signaling and is assocd. with a poor prognosis.  Some success against CRPC has been achieved by drugs that target AR signaling, but secondary resistance invariably emerges, and new therapies are urgently needed.  Recently, inhibitors of bromodomain and extra-terminal (BET) family proteins have shown growth-inhibitory activity in preclin. models of CRPC.  Here, we demonstrate that ARV-771, a small-mol. pan-BET degrader based on proteolysis-targeting chimera (PROTAC) technol., demonstrates dramatically improved efficacy in cellular models of CRPC as compared with BET inhibition.  Unlike BET inhibitors, ARV-771 results in suppression of both AR signaling and AR levels and leads to tumor regression in a CRPC mouse xenograft model.  This study is, to our knowledge, the first to demonstrate efficacy with a small-mol. BET degrader in a solid-tumor malignancy and potentially represents an important therapeutic advance in the treatment of CRPC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrD7QtYgZVig7Vg90H21EOLACvtfcHk0lgOMafTVWIGbA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xptlyqs7s%253D&md5=34218e685ad72950f3eb85586110c43d</span></div><a href="/servlet/linkout?suffix=cit118&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1521738113&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1521738113%26sid%3Dliteratum%253Aachs%26aulast%3DRaina%26aufirst%3DK.%26aulast%3DLu%26aufirst%3DJ.%26aulast%3DQian%26aufirst%3DY.%26aulast%3DAltieri%26aufirst%3DM.%26aulast%3DGordon%26aufirst%3DD.%26aulast%3DRossi%26aufirst%3DA.%2BM.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DDong%26aufirst%3DH.%26aulast%3DSiu%26aufirst%3DK.%26aulast%3DWinkler%26aufirst%3DJ.%2BD.%26aulast%3DCrew%26aufirst%3DA.%2BP.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26aulast%3DColeman%26aufirst%3DK.%2BG.%26atitle%3DPROTAC-induced%2520BET%2520protein%2520degradation%2520as%2520a%2520therapy%2520for%2520castration-resistant%2520prostate%2520cancer%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2016%26volume%3D113%26spage%3D7124%26epage%3D7129%26doi%3D10.1073%2Fpnas.1521738113" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref119"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref119'); return false;" data-citation="" class="refNumLink">119</a></strong><div class="NLM_citation" id="cit119"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sun, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fiskus, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qian, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rajapakshe, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raina, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coleman, K. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crew, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saenz, D. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mill, C. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nowak, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jain, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khoury, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coarfa, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhalla, K. N.</span></span> <span> </span><span class="NLM_article-title">BET protein proteolysis targeting chimera (PROTAC) exerts potent lethal activity against mantle cell lymphoma cells</span>. <i>Leukemia</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>32</i></span>,  <span class="NLM_fpage">343</span>– <span class="NLM_lpage">352</span>, <span class="refDoi"> DOI: 10.1038/leu.2017.207</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref119/cit119&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1038%2Fleu.2017.207" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref119/cit119&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=28663582" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref119/cit119&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BC2sXht1eqt77J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2018&pages=343-352&author=B.+Sunauthor=W.+Fiskusauthor=Y.+Qianauthor=K.+Rajapaksheauthor=K.+Rainaauthor=K.+G.+Colemanauthor=A.+P.+Crewauthor=A.+Shenauthor=D.+T.+Saenzauthor=C.+P.+Millauthor=A.+J.+Nowakauthor=N.+Jainauthor=L.+Zhangauthor=M.+Wangauthor=J.+D.+Khouryauthor=C.+Coarfaauthor=C.+M.+Crewsauthor=K.+N.+Bhalla&title=BET+protein+proteolysis+targeting+chimera+%28PROTAC%29+exerts+potent+lethal+activity+against+mantle+cell+lymphoma+cells&doi=10.1038%2Fleu.2017.207"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit119R"><div class="casContent"><span class="casTitleNuber">119</span><div class="casTitle"><span class="NLM_cas:atitle">Bet protein proteolysis targeting chimera (protac) exerts potent lethal activity against mantle cell lymphoma cells</span></div><div class="casAuthors">Sun, B.; Fiskus, W.; Qian, Y.; Rajapakshe, K.; Raina, K.; Coleman, K. G.; Crew, A. P.; Shen, A.; Saenz, D. T.; Mill, C. P.; Nowak, A. J.; Jain, N.; Zhang, L.; Wang, M.; Khoury, J. D.; Coarfa, C.; Crews, C. M.; Bhalla, K. N.</div><div class="citationInfo"><span class="NLM_cas:title">Leukemia</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">32</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">343-352</span>CODEN:
                <span class="NLM_cas:coden">LEUKED</span>;
        ISSN:<span class="NLM_cas:issn">0887-6924</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Bromodomain extraterminal protein (BETP) inhibitors transcriptionally repress oncoproteins and nuclear factor-κB (NF-κB) target genes that undermines the growth and survival of mantle cell lymphoma (MCL) cells.  However, BET bromodomain inhibitor (BETi) treatment causes accumulation of BETPs, assocd. with reversible binding and incomplete inhibition of BRD4 that potentially compromises the activity of BETi in MCL cells.  Unlike BETi, BET-PROTACs (proteolysis-targeting chimera) ARV-825 and ARV-771 (Arvinas, Inc.) recruit and utilize an E3-ubiquitin ligase to effectively degrade BETPs in MCL cells.  BET-PROTACs induce more apoptosis than BETi of MCL cells, including those resistant to ibrutinib.  BET-PROTAC treatment induced more perturbations in the mRNA and protein expressions than BETi, with depletion of c-Myc, CDK4, cyclin D1 and the NF-κB transcriptional targets Bcl-xL, XIAP and BTK, while inducing the levels of HEXIM1, NOXA and CDKN1A/p21.  Treatment with ARV-771, which possesses superior pharmacol. properties compared with ARV-825, inhibited the in vivo growth and induced greater survival improvement than the BETi OTX015 of immune-depleted mice engrafted with MCL cells.  Cotreatment of ARV-771 with ibrutinib or the BCL2 antagonist venetoclax or CDK4/6 inhibitor palbociclib synergistically induced apoptosis of MCL cells.  These studies highlight promising and superior preclin. activity of BET-PROTAC than BETi, requiring further in vivo evaluation of BET-PROTAC as a therapy for ibrutinib-sensitive or -resistant MCL.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAZxihoDZILrVg90H21EOLACvtfcHk0li4RAd6DFtLVQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXht1eqt77J&md5=f771eee574daf11546b93866ea314381</span></div><a href="/servlet/linkout?suffix=cit119&amp;dbid=16384&amp;doi=10.1038%2Fleu.2017.207&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fleu.2017.207%26sid%3Dliteratum%253Aachs%26aulast%3DSun%26aufirst%3DB.%26aulast%3DFiskus%26aufirst%3DW.%26aulast%3DQian%26aufirst%3DY.%26aulast%3DRajapakshe%26aufirst%3DK.%26aulast%3DRaina%26aufirst%3DK.%26aulast%3DColeman%26aufirst%3DK.%2BG.%26aulast%3DCrew%26aufirst%3DA.%2BP.%26aulast%3DShen%26aufirst%3DA.%26aulast%3DSaenz%26aufirst%3DD.%2BT.%26aulast%3DMill%26aufirst%3DC.%2BP.%26aulast%3DNowak%26aufirst%3DA.%2BJ.%26aulast%3DJain%26aufirst%3DN.%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DKhoury%26aufirst%3DJ.%2BD.%26aulast%3DCoarfa%26aufirst%3DC.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26aulast%3DBhalla%26aufirst%3DK.%2BN.%26atitle%3DBET%2520protein%2520proteolysis%2520targeting%2520chimera%2520%2528PROTAC%2529%2520exerts%2520potent%2520lethal%2520activity%2520against%2520mantle%2520cell%2520lymphoma%2520cells%26jtitle%3DLeukemia%26date%3D2018%26volume%3D32%26spage%3D343%26epage%3D352%26doi%3D10.1038%2Fleu.2017.207" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref120"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref120'); return false;" data-citation="" class="refNumLink">120</a></strong><div class="NLM_citation" id="cit120"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zengerle, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span> <span> </span><span class="NLM_article-title">Selective small molecule induced degradation of the BET bromodomain protein BRD4</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">1770</span>– <span class="NLM_lpage">1777</span>, <span class="refDoi"> DOI: 10.1021/acschembio.5b00216</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.5b00216" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref120/cit120&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BC2MXptlOgu70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=1770-1777&author=M.+Zengerleauthor=K.+H.+Chanauthor=A.+Ciulli&title=Selective+small+molecule+induced+degradation+of+the+BET+bromodomain+protein+BRD4&doi=10.1021%2Facschembio.5b00216"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit120R"><div class="casContent"><span class="casTitleNuber">120</span><div class="casTitle"><span class="NLM_cas:atitle">Selective Small Molecule Induced Degradation of the BET Bromodomain Protein BRD4</span></div><div class="casAuthors">Zengerle, Michael; Chan, Kwok-Ho; Ciulli, Alessio</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1770-1777</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The Bromo- and Extra-Terminal (BET) proteins BRD2, BRD3, and BRD4 play important roles in transcriptional regulation, epigenetics, and cancer and are the targets of pan-BET selective bromodomain inhibitor JQ1.  However, the lack of intra-BET selectivity limits the scope of current inhibitors as probes for target validation and could lead to unwanted side effects or toxicity in a therapeutic setting.  We designed Proteolysis Targeted Chimeras (PROTACs) that tether JQ1 to a ligand for the E3 ubiquitin ligase VHL, aimed at triggering the intracellular destruction of BET proteins.  Compd. MZ1 potently and rapidly induces reversible, long-lasting, and unexpectedly selective removal of BRD4 over BRD2 and BRD3.  The activity of MZ1 is dependent on binding to VHL but is achieved at a sufficiently low concn. not to induce stabilization of HIF-1α.  Gene expression profiles of selected cancer-related genes responsive to JQ1 reveal distinct and more limited transcriptional responses induced by MZ1, consistent with selective suppression of BRD4.  Our discovery opens up new opportunities to elucidate the cellular phenotypes and therapeutic implications assocd. with selective targeting of BRD4.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrEmSv7p9MZOrVg90H21EOLACvtfcHk0li4RAd6DFtLVQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXptlOgu70%253D&md5=b7514fd274eff8e4bb6a771f5aaf9ab5</span></div><a href="/servlet/linkout?suffix=cit120&amp;dbid=16384&amp;doi=10.1021%2Facschembio.5b00216&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.5b00216%26sid%3Dliteratum%253Aachs%26aulast%3DZengerle%26aufirst%3DM.%26aulast%3DChan%26aufirst%3DK.%2BH.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DSelective%2520small%2520molecule%2520induced%2520degradation%2520of%2520the%2520BET%2520bromodomain%2520protein%2520BRD4%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2015%26volume%3D10%26spage%3D1770%26epage%3D1777%26doi%3D10.1021%2Facschembio.5b00216" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref121"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref121'); return false;" data-citation="" class="refNumLink">121</a></strong><div class="NLM_citation" id="cit121"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Testa, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hughes, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lucas, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wright, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span> <span> </span><span class="NLM_article-title">Structure-based design of a macrocyclic PROTAC</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">1727</span>– <span class="NLM_lpage">1734</span>, <span class="refDoi"> DOI: 10.1002/anie.201914396</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref121/cit121&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1002%2Fanie.201914396" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref121/cit121&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitlyitbjK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2020&pages=1727-1734&author=A.+Testaauthor=S.+J.+Hughesauthor=X.+Lucasauthor=J.+E.+Wrightauthor=A.+Ciulli&title=Structure-based+design+of+a+macrocyclic+PROTAC&doi=10.1002%2Fanie.201914396"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit121R"><div class="casContent"><span class="casTitleNuber">121</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Based Design of a Macrocyclic PROTAC</span></div><div class="casAuthors">Testa, Andrea; Hughes, Scott J.; Lucas, Xavier; Wright, Jane E.; Ciulli, Alessio</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1727-1734</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Constraining a mol. in its bioactive conformation via macrocyclization represents an attractive strategy to rationally design functional chem. probes.  While this approach has been applied to enzyme inhibitors or receptor antagonists, to date it remains unprecedented for bifunctional mols. that bring proteins together, such as PROTAC degraders.  Herein, the authors report the design and synthesis of a macrocyclic PROTAC by adding a cyclizing linker to the BET degrader MZ1.  A cocrystal structure of macroPROTAC-1 bound in a ternary complex with VHL and the second bromodomain of Brd4 validated the rational design.  Biophys. studies revealed enhanced discrimination between the second and the first bromodomains of BET proteins.  Despite a 12-fold loss of binary binding affinity for Brd4, macroPROTAC-1 exhibited cellular activity comparable to MZ1.  The findings support macrocyclization as an advantageous strategy to enhance PROTAC degrdn. potency and selectivity between homologous targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqZjLgy_V3_abVg90H21EOLACvtfcHk0li4RAd6DFtLVQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitlyitbjK&md5=6f3f279f6220ec30b30ca3d2c931da95</span></div><a href="/servlet/linkout?suffix=cit121&amp;dbid=16384&amp;doi=10.1002%2Fanie.201914396&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201914396%26sid%3Dliteratum%253Aachs%26aulast%3DTesta%26aufirst%3DA.%26aulast%3DHughes%26aufirst%3DS.%2BJ.%26aulast%3DLucas%26aufirst%3DX.%26aulast%3DWright%26aufirst%3DJ.%2BE.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DStructure-based%2520design%2520of%2520a%2520macrocyclic%2520PROTAC%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2020%26volume%3D59%26spage%3D1727%26epage%3D1734%26doi%3D10.1002%2Fanie.201914396" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref122"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref122'); return false;" data-citation="" class="refNumLink">122</a></strong><div class="NLM_citation" id="cit122"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chan, K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zengerle, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Testa, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span> <span> </span><span class="NLM_article-title">Impact of target warhead and linkage vector on inducing protein degradation: comparison of bromodomain and extra-terminal (BET) degraders derived from triazolodiazepine (JQ1) and tetrahydroquinoline (I-BET726) BET inhibitor scaffolds</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">504</span>– <span class="NLM_lpage">513</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b01912</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01912" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref122/cit122&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BC2sXpslSktL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=504-513&author=K.+H.+Chanauthor=M.+Zengerleauthor=A.+Testaauthor=A.+Ciulli&title=Impact+of+target+warhead+and+linkage+vector+on+inducing+protein+degradation%3A+comparison+of+bromodomain+and+extra-terminal+%28BET%29+degraders+derived+from+triazolodiazepine+%28JQ1%29+and+tetrahydroquinoline+%28I-BET726%29+BET+inhibitor+scaffolds&doi=10.1021%2Facs.jmedchem.6b01912"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit122R"><div class="casContent"><span class="casTitleNuber">122</span><div class="casTitle"><span class="NLM_cas:atitle">Impact of Target Warhead and Linkage Vector on Inducing Protein Degradation: Comparison of Bromodomain and Extra-Terminal (BET) Degraders Derived from Triazolodiazepine (JQ1) and Tetrahydroquinoline (I-BET726) BET Inhibitor Scaffolds</span></div><div class="casAuthors">Chan, Kwok-Ho; Zengerle, Michael; Testa, Andrea; Ciulli, Alessio</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">504-513</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The design of proteolysis-targeting chimeras (PROTACs) is a powerful small-mol. approach for inducing protein degrdn.  PROTACs conjugate a target warhead to an E3 ubiquitin ligase ligand via a linker.  Here the authors examd. the impact of derivatizing two different BET bromodomain inhibitors, triazolodiazepine JQ1 and the more potent tetrahydroquinoline I-BET726, via distinct exit vectors, using different polyethylene glycol linkers to VHL ligand VH032.  Triazolodiazepine PROTACs exhibited pos. cooperativities of ternary complex formation and were more potent degraders than tetrahydroquinoline compds., which showed neg. cooperativities instead.  Marked dependency on linker length was obsd. for BET-degrading and cMyc-driven antiproliferative activities in acute myeloid leukemia cell lines.  This work exemplifies as a cautionary tale how a more potent inhibitor does not necessarily generate more potent PROTACs and underscores the key roles played by the conjugation.  The provided insights and framework for structure-activity relationships of bivalent degraders are anticipated to have wide future applicability.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqvEL41TefwTbVg90H21EOLACvtfcHk0lhdz4vM_47mRw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXpslSktL4%253D&md5=c3bfd9377d8e7805ba962cd0180431af</span></div><a href="/servlet/linkout?suffix=cit122&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01912&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01912%26sid%3Dliteratum%253Aachs%26aulast%3DChan%26aufirst%3DK.%2BH.%26aulast%3DZengerle%26aufirst%3DM.%26aulast%3DTesta%26aufirst%3DA.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DImpact%2520of%2520target%2520warhead%2520and%2520linkage%2520vector%2520on%2520inducing%2520protein%2520degradation%253A%2520comparison%2520of%2520bromodomain%2520and%2520extra-terminal%2520%2528BET%2529%2520degraders%2520derived%2520from%2520triazolodiazepine%2520%2528JQ1%2529%2520and%2520tetrahydroquinoline%2520%2528I-BET726%2529%2520BET%2520inhibitor%2520scaffolds%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D504%26epage%3D513%26doi%3D10.1021%2Facs.jmedchem.6b01912" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref123"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref123'); return false;" data-citation="" class="refNumLink">123</a></strong><div class="NLM_citation" id="cit123"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bondeson, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, B. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burslem, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buhimschi, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hines, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jaime-Figueroa, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamman, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ishchenko, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Lessons in PROTAC design from selective degradation with a promiscuous warhead</span>. <i>Cell Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">78</span>– <span class="NLM_lpage">87</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2017.09.010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref123/cit123&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1016%2Fj.chembiol.2017.09.010" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref123/cit123&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=29129718" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref123/cit123&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsl2lsrjN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2018&pages=78-87&author=D.+P.+Bondesonauthor=B.+E.+Smithauthor=G.+M.+Burslemauthor=A.+D.+Buhimschiauthor=J.+Hinesauthor=S.+Jaime-Figueroaauthor=J.+Wangauthor=B.+D.+Hammanauthor=A.+Ishchenkoauthor=C.+M.+Crews&title=Lessons+in+PROTAC+design+from+selective+degradation+with+a+promiscuous+warhead&doi=10.1016%2Fj.chembiol.2017.09.010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit123R"><div class="casContent"><span class="casTitleNuber">123</span><div class="casTitle"><span class="NLM_cas:atitle">Lessons in PROTAC Design from Selective Degradation with a Promiscuous Warhead</span></div><div class="casAuthors">Bondeson, Daniel P.; Smith, Blake E.; Burslem, George M.; Buhimschi, Alexandru D.; Hines, John; Jaime-Figueroa, Saul; Wang, Jing; Hamman, Brian D.; Ishchenko, Alexey; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">78-87.e5</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Inhibiting protein function selectively is a major goal of modern drug discovery.  Here, we report a previously understudied benefit of small mol. proteolysis-targeting chimeras (PROTACs) that recruit E3 ubiquitin ligases to target proteins for their ubiquitination and subsequent proteasome-mediated degrdn.  Using promiscuous CRBN- and VHL-recruiting PROTACs that bind >50 kinases, we show that only a subset of bound targets is degraded.  The basis of this selectivity relies on protein-protein interactions between the E3 ubiquitin ligase and the target protein, as illustrated by engaged proteins that are not degraded as a result of unstable ternary complexes with PROTAC-recruited E3 ligases.  In contrast, weak PROTAC:target protein affinity can be stabilized by high-affinity target:PROTAC:ligase trimer interactions, leading to efficient degrdn.  This study highlights design guidelines for generating potent PROTACs as well as possibilities for degrading undruggable proteins immune to traditional small-mol. inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoISOBudOLyFrVg90H21EOLACvtfcHk0lhdz4vM_47mRw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsl2lsrjN&md5=30fac502705a2344c4261cf531e6bf9d</span></div><a href="/servlet/linkout?suffix=cit123&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2017.09.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2017.09.010%26sid%3Dliteratum%253Aachs%26aulast%3DBondeson%26aufirst%3DD.%2BP.%26aulast%3DSmith%26aufirst%3DB.%2BE.%26aulast%3DBurslem%26aufirst%3DG.%2BM.%26aulast%3DBuhimschi%26aufirst%3DA.%2BD.%26aulast%3DHines%26aufirst%3DJ.%26aulast%3DJaime-Figueroa%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DHamman%26aufirst%3DB.%2BD.%26aulast%3DIshchenko%26aufirst%3DA.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DLessons%2520in%2520PROTAC%2520design%2520from%2520selective%2520degradation%2520with%2520a%2520promiscuous%2520warhead%26jtitle%3DCell%2520Chem.%2520Biol.%26date%3D2018%26volume%3D25%26spage%3D78%26epage%3D87%26doi%3D10.1016%2Fj.chembiol.2017.09.010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref124"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref124'); return false;" data-citation="" class="refNumLink">124</a></strong><div class="NLM_citation" id="cit124"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qin, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bai, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span> <span> </span><span class="NLM_article-title">Small-molecule PROTAC degraders of the Bromodomain and Extra Terminal (BET) proteins - A review</span>. <i>Drug Discovery Today: Technol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>31</i></span>,  <span class="NLM_fpage">43</span>– <span class="NLM_lpage">51</span>, <span class="refDoi"> DOI: 10.1016/j.ddtec.2019.04.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref124/cit124&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1016%2Fj.ddtec.2019.04.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref124/cit124&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=31200858" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref124/cit124&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A280%3ADC%252BB3M3mvFWrsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2019&pages=43-51&author=C.+Y.+Yangauthor=C.+Qinauthor=L.+Baiauthor=S.+Wang&title=Small-molecule+PROTAC+degraders+of+the+Bromodomain+and+Extra+Terminal+%28BET%29+proteins+-+A+review&doi=10.1016%2Fj.ddtec.2019.04.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit124R"><div class="casContent"><span class="casTitleNuber">124</span><div class="casTitle"><span class="NLM_cas:atitle">Small-molecule PROTAC degraders of the Bromodomain and Extra Terminal (BET) proteins - A review</span></div><div class="casAuthors">Yang Chao-Yie; Qin Chong; Bai Longchuan; Wang Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Drug discovery today. Technologies</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">43-51</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The PROteolysis TArgeting Chimeric (PROTAC) concept has provided an opportunity for the discovery and development of a completely new type of therapy involving induction of protein degradation.  The BET proteins, comprised of BRD2, BRD3, BRD4 and the testis-specific BRDT protein, are epigenetic readers and master transcription coactivators.  Extremely potent and efficacious small-molecule PROTAC degraders of the BET proteins, based on available, potent and selective BET inhibitors, have been reported.  BET degraders differ from BET inhibitors in their cellular potency, phenotypic effects, pharmacokinetic properties and toxicity profiles.  Herein, we provide a review of BET degraders and the differential outcome observed in the cellular and animal models for BET degraders in comparison to BET inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQCRHissLLXoQvupmecNw_lfW6udTcc2eaEDMmUFt8A87ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3M3mvFWrsA%253D%253D&md5=de5790c1db28d56717ff76374072ad81</span></div><a href="/servlet/linkout?suffix=cit124&amp;dbid=16384&amp;doi=10.1016%2Fj.ddtec.2019.04.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ddtec.2019.04.001%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DC.%2BY.%26aulast%3DQin%26aufirst%3DC.%26aulast%3DBai%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DSmall-molecule%2520PROTAC%2520degraders%2520of%2520the%2520Bromodomain%2520and%2520Extra%2520Terminal%2520%2528BET%2529%2520proteins%2520-%2520A%2520review%26jtitle%3DDrug%2520Discovery%2520Today%253A%2520Technol.%26date%3D2019%26volume%3D31%26spage%3D43%26epage%3D51%26doi%3D10.1016%2Fj.ddtec.2019.04.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref125"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref125'); return false;" data-citation="" class="refNumLink">125</a></strong><div class="NLM_citation" id="cit125"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Maneiro, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Forte, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shchepinova, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kounde, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chudasama, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baker, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tate, E. W.</span></span> <span> </span><span class="NLM_article-title">Antibody-PROTAC conjugates enable HER2-dependent targeted protein degradation of BRD4</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">1306</span>– <span class="NLM_lpage">1312</span>, <span class="refDoi"> DOI: 10.1021/acschembio.0c00285</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.0c00285" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref125/cit125&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BB3cXotFehsLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2020&pages=1306-1312&author=M.+A.+Maneiroauthor=N.+Forteauthor=M.+M.+Shchepinovaauthor=C.+S.+Koundeauthor=V.+Chudasamaauthor=J.+R.+Bakerauthor=E.+W.+Tate&title=Antibody-PROTAC+conjugates+enable+HER2-dependent+targeted+protein+degradation+of+BRD4&doi=10.1021%2Facschembio.0c00285"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit125R"><div class="casContent"><span class="casTitleNuber">125</span><div class="casTitle"><span class="NLM_cas:atitle">Antibody-PROTAC Conjugates Enable HER2-Dependent Targeted Protein Degradation of BRD4</span></div><div class="casAuthors">Maneiro, Maria; Forte, Nafsika; Shchepinova, Maria M.; Kounde, Cyrille S.; Chudasama, Vijay; Baker, James Richard; Tate, Edward W.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1306-1312</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Targeting protein degrdn. with Proteolysis-Targeting Chimeras (PROTACs) is an area of great current interest in drug discovery.  Nevertheless, although the high effectiveness of PROTACs against a wide variety of targets has been established, most degraders reported to date display limited intrinsic tissue selectivity and do not discriminate between cells of different types.  Here, we describe a strategy for selective protein degrdn. in a specific cell type.  We report the design and synthesis of a trastuzumab-PROTAC conjugate (Ab-PROTAC 3) in which E3 ligase-directed degrader activity is caged with an antibody linker which can be hydrolyzed following antibody-PROTAC internalization, releasing the active PROTAC and inducing catalytic protein degrdn.  We show that 3 selectively targets bromodomain-contg. protein 4 (BRD4) for degrdn. only in HER2 pos. breast cancer cell lines, while sparing HER2 neg. cells.  Using live cell confocal microscopy, we show internalization and lysosomal trafficking of the conjugate specifically in HER2 pos. cells, leading to the release of active PROTAC in quantities sufficient to induce potent BRD4 degrdn.  These studies demonstrate proof-of-concept for tissue-specific BRD4 degrdn., overcoming limitations of PROTAC selectivity, with significant potential for application to novel targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPwGNLaGmRbbVg90H21EOLACvtfcHk0lifoBPlybASCg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXotFehsLo%253D&md5=544b53e54d3f0696f0b425c3f6e48e46</span></div><a href="/servlet/linkout?suffix=cit125&amp;dbid=16384&amp;doi=10.1021%2Facschembio.0c00285&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.0c00285%26sid%3Dliteratum%253Aachs%26aulast%3DManeiro%26aufirst%3DM.%2BA.%26aulast%3DForte%26aufirst%3DN.%26aulast%3DShchepinova%26aufirst%3DM.%2BM.%26aulast%3DKounde%26aufirst%3DC.%2BS.%26aulast%3DChudasama%26aufirst%3DV.%26aulast%3DBaker%26aufirst%3DJ.%2BR.%26aulast%3DTate%26aufirst%3DE.%2BW.%26atitle%3DAntibody-PROTAC%2520conjugates%2520enable%2520HER2-dependent%2520targeted%2520protein%2520degradation%2520of%2520BRD4%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2020%26volume%3D15%26spage%3D1306%26epage%3D1312%26doi%3D10.1021%2Facschembio.0c00285" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref126"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref126'); return false;" data-citation="" class="refNumLink">126</a></strong><div class="NLM_citation" id="cit126"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Takahashi, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moriyama, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakamura, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miki, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takahashi, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sato, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Akaike, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Itto-Nakama, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arimoto, H.</span></span> <span> </span><span class="NLM_article-title">AUTACs: cargo-specific degraders using selective autophagy</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>76</i></span>,  <span class="NLM_fpage">797</span>– <span class="NLM_lpage">810</span>, <span class="refDoi"> DOI: 10.1016/j.molcel.2019.09.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref126/cit126&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1016%2Fj.molcel.2019.09.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref126/cit126&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=31606272" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref126/cit126&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhvFKjsr%252FE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=76&publication_year=2019&pages=797-810&author=D.+Takahashiauthor=J.+Moriyamaauthor=T.+Nakamuraauthor=E.+Mikiauthor=E.+Takahashiauthor=A.+Satoauthor=T.+Akaikeauthor=K.+Itto-Nakamaauthor=H.+Arimoto&title=AUTACs%3A+cargo-specific+degraders+using+selective+autophagy&doi=10.1016%2Fj.molcel.2019.09.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit126R"><div class="casContent"><span class="casTitleNuber">126</span><div class="casTitle"><span class="NLM_cas:atitle">AUTACs: Cargo-Specific Degraders Using Selective Autophagy</span></div><div class="casAuthors">Takahashi, Daiki; Moriyama, Jun; Nakamura, Tomoe; Miki, Erika; Takahashi, Eriko; Sato, Ayami; Akaike, Takaaki; Itto-Nakama, Kaori; Arimoto, Hirokazu</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">76</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">797-810.e10</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Protein silencing represents an essential tool in biomedical research.  Targeted protein degrdn. (TPD) strategies exemplified by PROTACs are rapidly emerging as modalities in drug discovery.  However, the scope of current TPD techniques is limited because many intracellular materials are not substrates of proteasomal clearance.  Here, we described a novel targeted-clearance strategy (autophagy-targeting chimera [AUTAC]) that contains a degrdn. tag (guanine derivs.) and a warhead to provide target specificity.  As expected from the substrate scope of autophagy, AUTAC degraded fragmented mitochondria as well as proteins.  Mitochondria-targeted AUTAC accelerated both the removal of dysfunctional fragmented mitochondria and the biogenesis of functionally normal mitochondria in patient-derived fibroblast cells.  Cytoprotective effects against acute mitochondrial injuries were also seen.  Canonical autophagy is viewed as a nonselective bulk decompn. system, and none of the available autophagy-inducing agents exhibit useful cargo selectivity.  With its target specificity, AUTAC provides a new modality for research on autophagy-based drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqEB3iYVVOYJbVg90H21EOLACvtfcHk0lifoBPlybASCg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhvFKjsr%252FE&md5=2eb738f4a51d83e7aee6d69e47bf2e00</span></div><a href="/servlet/linkout?suffix=cit126&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2019.09.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2019.09.009%26sid%3Dliteratum%253Aachs%26aulast%3DTakahashi%26aufirst%3DD.%26aulast%3DMoriyama%26aufirst%3DJ.%26aulast%3DNakamura%26aufirst%3DT.%26aulast%3DMiki%26aufirst%3DE.%26aulast%3DTakahashi%26aufirst%3DE.%26aulast%3DSato%26aufirst%3DA.%26aulast%3DAkaike%26aufirst%3DT.%26aulast%3DItto-Nakama%26aufirst%3DK.%26aulast%3DArimoto%26aufirst%3DH.%26atitle%3DAUTACs%253A%2520cargo-specific%2520degraders%2520using%2520selective%2520autophagy%26jtitle%3DMol.%2520Cell%26date%3D2019%26volume%3D76%26spage%3D797%26epage%3D810%26doi%3D10.1016%2Fj.molcel.2019.09.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref127"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref127'); return false;" data-citation="" class="refNumLink">127</a></strong><div class="NLM_citation" id="cit127"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ivanov, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khuri, F. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fu, H.</span></span> <span> </span><span class="NLM_article-title">Targeting protein-protein interactions as an anticancer strategy</span>. <i>Trends Pharmacol. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>34</i></span>,  <span class="NLM_fpage">393</span>– <span class="NLM_lpage">400</span>, <span class="refDoi"> DOI: 10.1016/j.tips.2013.04.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref127/cit127&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1016%2Fj.tips.2013.04.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref127/cit127&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=23725674" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref127/cit127&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BC3sXosVGjt7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=2013&pages=393-400&author=A.+A.+Ivanovauthor=F.+R.+Khuriauthor=H.+Fu&title=Targeting+protein-protein+interactions+as+an+anticancer+strategy&doi=10.1016%2Fj.tips.2013.04.007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit127R"><div class="casContent"><span class="casTitleNuber">127</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting protein-protein interactions as an anticancer strategy</span></div><div class="casAuthors">Ivanov, Andrei A.; Khuri, Fadlo R.; Fu, Haian</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Pharmacological Sciences</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">34</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">393-400</span>CODEN:
                <span class="NLM_cas:coden">TPHSDY</span>;
        ISSN:<span class="NLM_cas:issn">0165-6147</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  The emergence and convergence of cancer genomics, targeted therapies, and network oncol. have significantly expanded the landscape of protein-protein interaction (PPI) networks in cancer for therapeutic discovery.  Extensive biol. and clin. investigations have led to the identification of protein interaction hubs and nodes that are crit. for the acquisition and maintenance of characteristics of cancer essential for cell transformation.  Such cancer-enabling PPIs have become promising therapeutic targets.  With technol. advances in PPI modulator discovery and validation of PPI-targeting agents in clin. settings, targeting of PPI interfaces as an anticancer strategy has become a reality.  Future research directed at genomics-based PPI target discovery, PPI interface characterization, PPI-focused chem. library design, and patient-genomic subpopulation-driven clin. studies is expected to accelerate the development of the next generation of PPI-based anticancer agents for personalized precision medicine.  Here we briefly review prominent PPIs that mediate cancer-acquired properties, highlight recognized challenges and promising clin. results in targeting PPIs, and outline emerging opportunities.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqKeFT8Id1vhbVg90H21EOLACvtfcHk0lifoBPlybASCg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXosVGjt7g%253D&md5=2f4772a4abfa6640b6af922671834fa3</span></div><a href="/servlet/linkout?suffix=cit127&amp;dbid=16384&amp;doi=10.1016%2Fj.tips.2013.04.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tips.2013.04.007%26sid%3Dliteratum%253Aachs%26aulast%3DIvanov%26aufirst%3DA.%2BA.%26aulast%3DKhuri%26aufirst%3DF.%2BR.%26aulast%3DFu%26aufirst%3DH.%26atitle%3DTargeting%2520protein-protein%2520interactions%2520as%2520an%2520anticancer%2520strategy%26jtitle%3DTrends%2520Pharmacol.%2520Sci.%26date%3D2013%26volume%3D34%26spage%3D393%26epage%3D400%26doi%3D10.1016%2Fj.tips.2013.04.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref128"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref128'); return false;" data-citation="" class="refNumLink">128</a></strong><div class="NLM_citation" id="cit128"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cai, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, A. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiang, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kodadek, T.</span></span> <span> </span><span class="NLM_article-title">Peptoid ligands that bind selectively to phosphoproteins</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">4960</span>– <span class="NLM_lpage">4964</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2011.06.011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref128/cit128&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1016%2Fj.bmcl.2011.06.011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref128/cit128&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=21742492" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref128/cit128&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtVaisL%252FJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2011&pages=4960-4964&author=D.+Caiauthor=A.+Y.+Leeauthor=C.+M.+Chiangauthor=T.+Kodadek&title=Peptoid+ligands+that+bind+selectively+to+phosphoproteins&doi=10.1016%2Fj.bmcl.2011.06.011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit128R"><div class="casContent"><span class="casTitleNuber">128</span><div class="casTitle"><span class="NLM_cas:atitle">Peptoid ligands that bind selectively to phosphoproteins</span></div><div class="casAuthors">Cai, Di; Lee, A.-Young; Chiang, Cheng-Ming; Kodadek, Thomas</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">4960-4964</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Synthetic equiv. of phosphoprotein-specific antibodies would be valuable reagents for biol. research, since these antibodies can often be difficult to produce.  Protein phosphorylation is thought to result in significant conformational changes in most substrate proteins.  Therefore, one approach might be to simply screen combinatorial libraries for ligands to the phosphorylated state in the hope of isolating a ligand that binds to a pocket created by the conformational shift.  In this study, the authors probe this strategy by screening a peptoid library for ligands to the phosphorylated form of the Brd4 chromatin adaptor and transcriptional coactivator protein.  The authors find that peptoids with high selectivity for binding to the phosphorylation form of Brd4 can indeed be isolated in this screen.  Moreover, these ligands do not bind promiscuously to other phospho-proteins.  However, attempts to employ these reagents as antibody substitutes in an immunoaffinity purifn.-like application showed that they do not perform as well as bona fide antibodies and that significant optimization will be required.  This study highlights the potential and current limitations of a naive library screening strategy for phosphoprotein-specific antibody surrogates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqf0autNVBtLLVg90H21EOLACvtfcHk0lhL6gOTDwQeag"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtVaisL%252FJ&md5=d91d1c1daa186fd809289d528760ce79</span></div><a href="/servlet/linkout?suffix=cit128&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2011.06.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2011.06.011%26sid%3Dliteratum%253Aachs%26aulast%3DCai%26aufirst%3DD.%26aulast%3DLee%26aufirst%3DA.%2BY.%26aulast%3DChiang%26aufirst%3DC.%2BM.%26aulast%3DKodadek%26aufirst%3DT.%26atitle%3DPeptoid%2520ligands%2520that%2520bind%2520selectively%2520to%2520phosphoproteins%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2011%26volume%3D21%26spage%3D4960%26epage%3D4964%26doi%3D10.1016%2Fj.bmcl.2011.06.011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref129"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref129'); return false;" data-citation="" class="refNumLink">129</a></strong><div class="NLM_citation" id="cit129"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhong, H.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leung, K.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, C. C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peng, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, S.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, D.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">David Wang, H.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leung, C.-H.</span></span> <span> </span><span class="NLM_article-title">An iridium(iii)-based irreversible protein-protein interaction inhibitor of BRD4 as a potent anticancer agent</span>. <i>Chem. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">5400</span>– <span class="NLM_lpage">5408</span>, <span class="refDoi"> DOI: 10.1039/C5SC02321A</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref129/cit129&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1039%2FC5SC02321A" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref129/cit129&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=28757943" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref129/cit129&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht1Grtr7E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=5400-5408&author=H.-J.+Zhongauthor=L.+Luauthor=K.-H.+Leungauthor=C.+C.+L.+Wongauthor=C.+Pengauthor=S.-C.+Yanauthor=D.-L.+Maauthor=Z.+Caiauthor=H.-M.+David+Wangauthor=C.-H.+Leung&title=An+iridium%28iii%29-based+irreversible+protein-protein+interaction+inhibitor+of+BRD4+as+a+potent+anticancer+agent&doi=10.1039%2FC5SC02321A"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit129R"><div class="casContent"><span class="casTitleNuber">129</span><div class="casTitle"><span class="NLM_cas:atitle">An iridium(III)-based irreversible protein-protein interaction inhibitor of BRD4 as a potent anticancer agent</span></div><div class="casAuthors">Zhong, Hai-Jing; Lu, Lihua; Leung, Ka-Ho; Wong, Catherine C. L.; Peng, Chao; Yan, Siu-Cheong; Ma, Dik-Lung; Cai, Zongwei; David Wang, Hui-Min; Leung, Chung-Hang</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Science</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">5400-5408</span>CODEN:
                <span class="NLM_cas:coden">CSHCCN</span>;
        ISSN:<span class="NLM_cas:issn">2041-6520</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Bromodomain-contg. protein 4 (BRD4) has recently emerged as an attractive epigenetic target for anticancer therapy.  In this study, an iridium(III) complex is reported as the first metal-based, irreversible inhibitor of BRD4.  Complex 1a is able to antagonize the BRD4-acetylated histone protein-protein interaction (PPI) in vitro, and to bind BRD4 and down-regulate c-myc oncogenic expression in cellulo.  Chromatin immunopptn. (ChIP) anal. revealed that 1a could modulate the interaction between BRD4 and chromatin in melanoma cells, particular at the MYC promoter.  Finally, the complex showed potent activity against melanoma xenografts in an in vivo mouse model.  To our knowledge, this is the first report of a Group 9 metal complex inhibiting the PPI of a member of the bromodomain and extraterminal domain (BET) family.  We envision that complex 1a may serve as a useful scaffold for the development of more potent epigenetic agents against cancers such as melanoma.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpqMXspNfXFlbVg90H21EOLACvtfcHk0lhL6gOTDwQeag"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXht1Grtr7E&md5=ae176cbfd3621dd7939c234d643db84e</span></div><a href="/servlet/linkout?suffix=cit129&amp;dbid=16384&amp;doi=10.1039%2FC5SC02321A&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC5SC02321A%26sid%3Dliteratum%253Aachs%26aulast%3DZhong%26aufirst%3DH.-J.%26aulast%3DLu%26aufirst%3DL.%26aulast%3DLeung%26aufirst%3DK.-H.%26aulast%3DWong%26aufirst%3DC.%2BC.%2BL.%26aulast%3DPeng%26aufirst%3DC.%26aulast%3DYan%26aufirst%3DS.-C.%26aulast%3DMa%26aufirst%3DD.-L.%26aulast%3DCai%26aufirst%3DZ.%26aulast%3DDavid%2BWang%26aufirst%3DH.-M.%26aulast%3DLeung%26aufirst%3DC.-H.%26atitle%3DAn%2520iridium%2528iii%2529-based%2520irreversible%2520protein-protein%2520interaction%2520inhibitor%2520of%2520BRD4%2520as%2520a%2520potent%2520anticancer%2520agent%26jtitle%3DChem.%2520Sci.%26date%3D2015%26volume%3D6%26spage%3D5400%26epage%3D5408%26doi%3D10.1039%2FC5SC02321A" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref130"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref130'); return false;" data-citation="" class="refNumLink">130</a></strong><div class="NLM_citation" id="cit130"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Leung, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, Q. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, D. L.</span></span> <span> </span><span class="NLM_article-title">Emerging screening approaches in the development of Nrf2-Keap1 protein-protein interaction inhibitors</span>. <i>Int. J. Mol. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">4445</span>, <span class="refDoi"> DOI: 10.3390/ijms20184445</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref130/cit130&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.3390%2Fijms20184445" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref130/cit130&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BB3cXovFSgsbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2019&pages=4445&author=C.+H.+Leungauthor=J.+T.+Zhangauthor=G.+J.+Yangauthor=H.+Liuauthor=Q.+B.+Hanauthor=D.+L.+Ma&title=Emerging+screening+approaches+in+the+development+of+Nrf2-Keap1+protein-protein+interaction+inhibitors&doi=10.3390%2Fijms20184445"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit130R"><div class="casContent"><span class="casTitleNuber">130</span><div class="casTitle"><span class="NLM_cas:atitle">Emerging screening approaches in the development of Nrf2-Keap1 protein-protein interaction inhibitors</span></div><div class="casAuthors">Leung, Chung-Hang; Zhang, Jia-Tong; Yang, Guan-Jun; Liu, Hao; Han, Quan-Bin; Ma, Dik-Lung</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Molecular Sciences</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">4445</span>CODEN:
                <span class="NLM_cas:coden">IJMCFK</span>;
        ISSN:<span class="NLM_cas:issn">1422-0067</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">A review.  Due to role of the Keap1-Nrf2 protein-protein interaction (PPI) in protecting cells from oxidative stress, the development of small mol. inhibitors that inhibit this interaction has arisen as a viable approach to combat maladies caused by oxidative stress, such as cancers, neurodegenerative disease and diabetes.  To obtain specific and genuine Keap1-Nrf2 inhibitors, many efforts have been made towards developing new screening approaches.  However, there is no inhibitor for this target entering the clinic for the treatment of human diseases.  New strategies to identify novel bioactive compds. from large mol. databases and accelerate the developmental process of the clin. application of Keap1-Nrf2 protein-protein interaction inhibitors are greatly needed.  In this review, we have summarized virtual screening and other methods for discovering new lead compds. against the Keap1-Nrf2 protein-protein interaction.  We also discuss the advantages and limitations of different strategies, and the potential of this PPI as a drug target in disease therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqjIj82HYmo1bVg90H21EOLACvtfcHk0lhL6gOTDwQeag"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXovFSgsbo%253D&md5=bf4681884449ccb168a226d399b21747</span></div><a href="/servlet/linkout?suffix=cit130&amp;dbid=16384&amp;doi=10.3390%2Fijms20184445&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fijms20184445%26sid%3Dliteratum%253Aachs%26aulast%3DLeung%26aufirst%3DC.%2BH.%26aulast%3DZhang%26aufirst%3DJ.%2BT.%26aulast%3DYang%26aufirst%3DG.%2BJ.%26aulast%3DLiu%26aufirst%3DH.%26aulast%3DHan%26aufirst%3DQ.%2BB.%26aulast%3DMa%26aufirst%3DD.%2BL.%26atitle%3DEmerging%2520screening%2520approaches%2520in%2520the%2520development%2520of%2520Nrf2-Keap1%2520protein-protein%2520interaction%2520inhibitors%26jtitle%3DInt.%2520J.%2520Mol.%2520Sci.%26date%3D2019%26volume%3D20%26spage%3D4445%26doi%3D10.3390%2Fijms20184445" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref131"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref131'); return false;" data-citation="" class="refNumLink">131</a></strong><div class="NLM_citation" id="cit131"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Padmanabhan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alexanian, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Linares-Saldana, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gonzalez-Teran, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andreoletti, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Connolly, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hsu, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duan, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winchester, S. A.B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Felix, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perez-Bermejo, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shah, P. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haldar, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jain, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Srivastava, D.</span></span> <span> </span><span class="NLM_article-title">Srivastava D. BRD4 interacts with GATA4 to govern mitochondrial homeostasis in adult cardiomyocytes</span>. <i>Circulation</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>142</i></span>,  <span class="NLM_fpage">2338</span>– <span class="NLM_lpage">2355</span>, <span class="refDoi"> DOI: 10.1161/CIRCULATIONAHA.120.047753</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref131/cit131&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1161%2FCIRCULATIONAHA.120.047753" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref131/cit131&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=33094644" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref131/cit131&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BB3MXms1Snuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=142&publication_year=2020&pages=2338-2355&author=A.+Padmanabhanauthor=M.+Alexanianauthor=R.+Linares-Saldanaauthor=B.+Gonzalez-Teranauthor=G.+Andreolettiauthor=Y.+Huangauthor=A.+J.+Connollyauthor=W.+Kimauthor=A.+Hsuauthor=Q.+Duanauthor=S.+A.B.+Winchesterauthor=F.+Felixauthor=J.+A.+Perez-Bermejoauthor=Q.+Wangauthor=L.+Liauthor=P.+P.+Shahauthor=S.+M.+Haldarauthor=R.+Jainauthor=D.+Srivastava&title=Srivastava+D.+BRD4+interacts+with+GATA4+to+govern+mitochondrial+homeostasis+in+adult+cardiomyocytes&doi=10.1161%2FCIRCULATIONAHA.120.047753"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit131R"><div class="casContent"><span class="casTitleNuber">131</span><div class="casTitle"><span class="NLM_cas:atitle">BRD4 (bromodomain-containing protein 4) interacts with GATA4 (GATA binding protein 4) to govern mitochondrial homeostasis in adult cardiomyocytes</span></div><div class="casAuthors">Padmanabhan, Arun; Alexanian, Michael; Linares-Saldana, Ricardo; Gonzalez-Teran, Barbara; Andreoletti, Gaia; Huang, Yu; Connolly, Andrew J.; Kim, Wonho; Hsu, Austin; Duan, Qiming; Winchester, Sarah A. B.; Felix, Franco; Perez-Bermejo, Juan A.; Wang, Qiaohong; Li, Li; Shah, Parisha P.; Haldar, Saptarsi M.; Jain, Rajan; Srivastava, Deepak</div><div class="citationInfo"><span class="NLM_cas:title">Circulation</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">142</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">2338-2355</span>CODEN:
                <span class="NLM_cas:coden">CIRCAZ</span>;
        ISSN:<span class="NLM_cas:issn">0009-7322</span>.
    
            (<span class="NLM_cas:orgname">Lippincott Williams & Wilkins</span>)
        </div><div class="casAbstract">Gene regulatory networks control tissue homeostasis and disease progression in a cell type-specific manner.  Ubiquitously expressed chromatin regulators modulate these networks, yet the mechanisms governing how tissue specificity of their function is achieved are poorly understood.  BRD4 (bromodomain-contg. protein 4), a member of the BET (bromo- and extraterminal domain) family of ubiquitously expressed acetyl-lysine reader proteins, plays a pivotal role as a coactivator of enhancer signaling across diverse tissue types in both health and disease and has been implicated as a pharmacol. target in heart failure.  However, the cell-specific role of BRD4 in adult cardiomyocytes remains unknown.  We combined conditional mouse genetics, unbiased transcriptomic and epigenomic analyses, and classic mol. biol. and biochem. approaches to understand the mechanism by which BRD4 in adult cardiomyocyte homeostasis.  Here, we show that cardiomyocyte-specific deletion of Brd4 in adult mice leads to acute deterioration of cardiac contractile function with mutant animals demonstrating a transcriptomic signature characterized by decreased expression of genes crit. for mitochondrial energy prodn.  Genome-wide occupancy data show that BRD4 enriches at many downregulated genes (including the master coactivators Ppargc1a, Ppargc1b, and their downstream targets) and preferentially colocalizes with GATA4 (GATA binding protein 4), a lineage-detg. cardiac transcription factor not previously implicated in regulation of adult cardiac metab.  BRD4 and GATA4 form an endogenous complex in cardiomyocytes and interact in a bromodomain-independent manner, revealing a new functional interaction partner for BRD4 that can direct its locus and tissue specificity.  These results highlight a novel role for a BRD4-GATA4 module in cooperative regulation of a cardiomyocyte-specific gene program governing bioenergetic homeostasis in the adult heart.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqdl4WwdZyHlbVg90H21EOLACvtfcHk0lijvaTFwEhHiw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3MXms1Snuw%253D%253D&md5=f3e0c0725b70e82f10a9fbfd78cf03b1</span></div><a href="/servlet/linkout?suffix=cit131&amp;dbid=16384&amp;doi=10.1161%2FCIRCULATIONAHA.120.047753&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1161%252FCIRCULATIONAHA.120.047753%26sid%3Dliteratum%253Aachs%26aulast%3DPadmanabhan%26aufirst%3DA.%26aulast%3DAlexanian%26aufirst%3DM.%26aulast%3DLinares-Saldana%26aufirst%3DR.%26aulast%3DGonzalez-Teran%26aufirst%3DB.%26aulast%3DAndreoletti%26aufirst%3DG.%26aulast%3DHuang%26aufirst%3DY.%26aulast%3DConnolly%26aufirst%3DA.%2BJ.%26aulast%3DKim%26aufirst%3DW.%26aulast%3DHsu%26aufirst%3DA.%26aulast%3DDuan%26aufirst%3DQ.%26aulast%3DWinchester%26aufirst%3DS.%2BA.B.%26aulast%3DFelix%26aufirst%3DF.%26aulast%3DPerez-Bermejo%26aufirst%3DJ.%2BA.%26aulast%3DWang%26aufirst%3DQ.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DShah%26aufirst%3DP.%2BP.%26aulast%3DHaldar%26aufirst%3DS.%2BM.%26aulast%3DJain%26aufirst%3DR.%26aulast%3DSrivastava%26aufirst%3DD.%26atitle%3DSrivastava%2520D.%2520BRD4%2520interacts%2520with%2520GATA4%2520to%2520govern%2520mitochondrial%2520homeostasis%2520in%2520adult%2520cardiomyocytes%26jtitle%3DCirculation%26date%3D2020%26volume%3D142%26spage%3D2338%26epage%3D2355%26doi%3D10.1161%2FCIRCULATIONAHA.120.047753" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref132"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref132'); return false;" data-citation="" class="refNumLink">132</a></strong><div class="NLM_citation" id="cit132"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kulikowski, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rakai, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, N.</span></span> <span> </span><span class="NLM_article-title">Inhibitors of bromodomain and extra-terminal proteins for treating multiple human diseases</span>. <i>Med. Res. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2021</span>,  <span class="NLM_volume"><i>41</i></span>,  <span class="NLM_fpage">223</span>– <span class="NLM_lpage">245</span>, <span class="refDoi"> DOI: 10.1002/med.21730</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref132/cit132&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1002%2Fmed.21730" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref132/cit132&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=32926459" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref132/cit132&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BB3MXhsVagtro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2021&pages=223-245&author=E.+Kulikowskiauthor=B.+D.+Rakaiauthor=N.+Wong&title=Inhibitors+of+bromodomain+and+extra-terminal+proteins+for+treating+multiple+human+diseases&doi=10.1002%2Fmed.21730"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit132R"><div class="casContent"><span class="casTitleNuber">132</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibitors of bromodomain and extra-terminal proteins for treating multiple human diseases</span></div><div class="casAuthors">Kulikowski, Ewelina; Rakai, Brooke D.; Wong, Norman C. W.</div><div class="citationInfo"><span class="NLM_cas:title">Medicinal Research Reviews</span>
        (<span class="NLM_cas:date">2021</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">223-245</span>CODEN:
                <span class="NLM_cas:coden">MRREDD</span>;
        ISSN:<span class="NLM_cas:issn">0198-6325</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">Clin. development of bromodomain and extra-terminal (BET) protein inhibitors differs from the traditional course of drug development.  These drugs are simultaneously being evaluated for treating a wide spectrum of human diseases due to their novel mechanism of action.  BET proteins are epigenetic "readers," which play a primary role in transcription.  Here, we briefly describe the BET family of proteins, of which BRD4 has been studied most extensively.  We discuss BRD4 activity at latent enhancers as an example of BET protein function.  We examine BRD4 redistribution and enhancer reprogramming in embryonic development, cancer, cardiovascular, autoimmune, and metabolic diseases, presenting hallmark studies that highlight BET proteins as attractive targets for therapeutic intervention.  We review the currently available approaches to targeting BET proteins, methods of selectively targeting individual bromodomains, and review studies that compare the effects of selective BET inhibition to those of pan-BET inhibition.  Lastly, we examine the current clin. landscape of BET inhibitor development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqiAZK3p5-347Vg90H21EOLACvtfcHk0lijvaTFwEhHiw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3MXhsVagtro%253D&md5=a63df8a1142c18a4c3e02d4567ded365</span></div><a href="/servlet/linkout?suffix=cit132&amp;dbid=16384&amp;doi=10.1002%2Fmed.21730&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fmed.21730%26sid%3Dliteratum%253Aachs%26aulast%3DKulikowski%26aufirst%3DE.%26aulast%3DRakai%26aufirst%3DB.%2BD.%26aulast%3DWong%26aufirst%3DN.%26atitle%3DInhibitors%2520of%2520bromodomain%2520and%2520extra-terminal%2520proteins%2520for%2520treating%2520multiple%2520human%2520diseases%26jtitle%3DMed.%2520Res.%2520Rev.%26date%3D2021%26volume%3D41%26spage%3D223%26epage%3D245%26doi%3D10.1002%2Fmed.21730" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref133"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref133'); return false;" data-citation="" class="refNumLink">133</a></strong><div class="NLM_citation" id="cit133"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Feig, M.</span></span> <span> </span><span class="NLM_article-title">Computational protein structure refinement: almost there, yet still so far to go</span>. <i>Wiley Interdiscip Rev. Comput. Mol. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">e1307</span>, <span class="refDoi"> DOI: 10.1002/wcms.1307</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref133/cit133&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1002%2Fwcms.1307" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref133/cit133&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=30613211" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2017&pages=e1307&author=M.+Feig&title=Computational+protein+structure+refinement%3A+almost+there%2C+yet+still+so+far+to+go&doi=10.1002%2Fwcms.1307"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit133&amp;dbid=16384&amp;doi=10.1002%2Fwcms.1307&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fwcms.1307%26sid%3Dliteratum%253Aachs%26aulast%3DFeig%26aufirst%3DM.%26atitle%3DComputational%2520protein%2520structure%2520refinement%253A%2520almost%2520there%252C%2520yet%2520still%2520so%2520far%2520to%2520go%26jtitle%3DWiley%2520Interdiscip%2520Rev.%2520Comput.%2520Mol.%2520Sci.%26date%3D2017%26volume%3D7%26spage%3De1307%26doi%3D10.1002%2Fwcms.1307" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref134"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref134'); return false;" data-citation="" class="refNumLink">134</a></strong><div class="NLM_citation" id="cit134"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cerutti, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Case, D. A.</span></span> <span> </span><span class="NLM_article-title">Molecular dynamics simulations of macromolecular crystals</span>. <i>Wiley Interdiscip. Rev.: Comput. Mol. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">e1402</span>, <span class="refDoi"> DOI: 10.1002/wcms.1402</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref134/cit134&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1002%2Fwcms.1402" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref134/cit134&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=31662799" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2019&pages=e1402&author=D.+S.+Ceruttiauthor=D.+A.+Case&title=Molecular+dynamics+simulations+of+macromolecular+crystals&doi=10.1002%2Fwcms.1402"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit134&amp;dbid=16384&amp;doi=10.1002%2Fwcms.1402&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fwcms.1402%26sid%3Dliteratum%253Aachs%26aulast%3DCerutti%26aufirst%3DD.%2BS.%26aulast%3DCase%26aufirst%3DD.%2BA.%26atitle%3DMolecular%2520dynamics%2520simulations%2520of%2520macromolecular%2520crystals%26jtitle%3DWiley%2520Interdiscip.%2520Rev.%253A%2520Comput.%2520Mol.%2520Sci.%26date%3D2019%26volume%3D9%26spage%3De1402%26doi%3D10.1002%2Fwcms.1402" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52'],'ref53':['cit53'],'ref54':['cit54'],'ref55':['cit55'],'ref56':['cit56'],'ref57':['cit57'],'ref58':['cit58'],'ref59':['cit59'],'ref60':['cit60'],'ref61':['cit61'],'ref62':['cit62'],'ref63':['cit63'],'ref64':['cit64'],'ref65':['cit65'],'ref66':['cit66'],'ref67':['cit67'],'ref68':['cit68'],'ref69':['cit69'],'ref70':['cit70'],'ref71':['cit71'],'ref72':['cit72'],'ref73':['cit73'],'ref74':['cit74'],'ref75':['cit75'],'ref76':['cit76'],'ref77':['cit77'],'ref78':['cit78'],'ref79':['cit79'],'ref80':['cit80'],'ref81':['cit81'],'ref82':['cit82'],'ref83':['cit83'],'ref84':['cit84'],'ref85':['cit85'],'ref86':['cit86'],'ref87':['cit87'],'ref88':['cit88'],'ref89':['cit89'],'ref90':['cit90'],'ref91':['cit91'],'ref92':['cit92'],'ref93':['cit93'],'ref94':['cit94'],'ref95':['cit95'],'ref96':['cit96'],'ref97':['cit97'],'ref98':['cit98'],'ref99':['cit99'],'ref100':['cit100'],'ref101':['cit101'],'ref102':['cit102'],'ref103':['cit103'],'ref104':['cit104'],'ref105':['cit105'],'ref106':['cit106'],'ref107':['cit107'],'ref108':['cit108'],'ref109':['cit109'],'ref110':['cit110'],'ref111':['cit111'],'ref112':['cit112'],'ref113':['cit113'],'ref114':['cit114'],'ref115':['cit115'],'ref116':['cit116'],'ref117':['cit117'],'ref118':['cit118'],'ref119':['cit119'],'ref120':['cit120'],'ref121':['cit121'],'ref122':['cit122'],'ref123':['cit123'],'ref124':['cit124'],'ref125':['cit125'],'ref126':['cit126'],'ref127':['cit127'],'ref128':['cit128'],'ref129':['cit129'],'ref130':['cit130'],'ref131':['cit131'],'ref132':['cit132'],'ref133':['cit133'],'ref134':['cit134']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 5 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Adam I. Green, <span class="NLM_string-name hlFld-ContribAuthor">George M. Burslem</span>. </span><span class="cited-content_cbyCitation_article-title">Focused Libraries for Epigenetic Drug Discovery: The Importance of Isosteres. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>64 </em>
                                    (11)
                                     , 7231-7240. <a href="https://doi.org/10.1021/acs.jmedchem.1c00592" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.1c00592</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.1c00592&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.1c00592%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DFocused%252BLibraries%252Bfor%252BEpigenetic%252BDrug%252BDiscovery%25253A%252BThe%252BImportance%252Bof%252BIsosteres%26aulast%3DGreen%26aufirst%3DAdam%2BI.%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D31032021%26date%3D27052021%26volume%3D64%26issue%3D11%26spage%3D7231%26epage%3D7240" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ying-Chao  Duan</span>, <span class="hlFld-ContribAuthor ">Shao-Jie  Zhang</span>, <span class="hlFld-ContribAuthor ">Xiao-Jing  Shi</span>, <span class="hlFld-ContribAuthor ">Lin-Feng  Jin</span>, <span class="hlFld-ContribAuthor ">Tong  Yu</span>, <span class="hlFld-ContribAuthor ">Yu  Song</span>, <span class="hlFld-ContribAuthor ">Yuan-Yuan  Guan</span>. </span><span class="cited-content_cbyCitation_article-title">Research progress of dual inhibitors targeting crosstalk between histone epigenetic modulators for cancer therapy. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>222 </em>, 113588. <a href="https://doi.org/10.1016/j.ejmech.2021.113588" title="DOI URL">https://doi.org/10.1016/j.ejmech.2021.113588</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2021.113588&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2021.113588%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DResearch%252Bprogress%252Bof%252Bdual%252Binhibitors%252Btargeting%252Bcrosstalk%252Bbetween%252Bhistone%252Bepigenetic%252Bmodulators%252Bfor%252Bcancer%252Btherapy%26aulast%3DDuan%26aufirst%3DYing-Chao%26date%3D2021%26volume%3D222%26spage%3D113588" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Tian  Xu</span>, <span class="hlFld-ContribAuthor ">Jifa  Zhang</span>, <span class="hlFld-ContribAuthor ">Chengcan  Yang</span>, <span class="hlFld-ContribAuthor ">Ryszard  Pluta</span>, <span class="hlFld-ContribAuthor ">Guan  Wang</span>, <span class="hlFld-ContribAuthor ">Tinghong  Ye</span>, <span class="hlFld-ContribAuthor ">Liang  Ouyang</span>. </span><span class="cited-content_cbyCitation_article-title">Identification and optimization of 3-bromo-N’-(4-hydroxybenzylidene)-4-methylbenzohydrazide derivatives as mTOR inhibitors that induce autophagic cell death and apoptosis in triple-negative breast cancer. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>219 </em>, 113424. <a href="https://doi.org/10.1016/j.ejmech.2021.113424" title="DOI URL">https://doi.org/10.1016/j.ejmech.2021.113424</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2021.113424&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2021.113424%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DIdentification%252Band%252Boptimization%252Bof%252B3-bromo-N%2525E2%252580%252599-%2525284-hydroxybenzylidene%252529-4-methylbenzohydrazide%252Bderivatives%252Bas%252BmTOR%252Binhibitors%252Bthat%252Binduce%252Bautophagic%252Bcell%252Bdeath%252Band%252Bapoptosis%252Bin%252Btriple-negative%252Bbreast%252Bcancer%26aulast%3DXu%26aufirst%3DTian%26date%3D2021%26volume%3D219%26spage%3D113424" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Zhangxu  He</span>, <span class="hlFld-ContribAuthor ">Haomiao  Jiao</span>, <span class="hlFld-ContribAuthor ">Qi  An</span>, <span class="hlFld-ContribAuthor ">Xin  Zhang</span>, <span class="hlFld-ContribAuthor ">Dan  Zengyangzong</span>, <span class="hlFld-ContribAuthor ">Jiale  Xu</span>, <span class="hlFld-ContribAuthor ">Hongmin  Liu</span>, <span class="hlFld-ContribAuthor ">Liying  Ma</span>, <span class="hlFld-ContribAuthor ">Wen  Zhao</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of novel 4-phenylquinazoline-based BRD4 inhibitors for cardiac fibrosis. </span><span class="cited-content_cbyCitation_journal-name">Acta Pharmaceutica Sinica B</span><span> <strong>2021,</strong> <em>399 </em><a href="https://doi.org/10.1016/j.apsb.2021.07.018" title="DOI URL">https://doi.org/10.1016/j.apsb.2021.07.018</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.apsb.2021.07.018&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.apsb.2021.07.018%26sid%3Dliteratum%253Aachs%26jtitle%3DActa%2520Pharmaceutica%2520Sinica%2520B%26atitle%3DDiscovery%252Bof%252Bnovel%252B4-phenylquinazoline-based%252BBRD4%252Binhibitors%252Bfor%252Bcardiac%252Bfibrosis%26aulast%3DHe%26aufirst%3DZhangxu%26date%3D2021%26volume%3D399" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Laura  Belver</span>, <span class="hlFld-ContribAuthor ">Robert  Albero</span>, <span class="hlFld-ContribAuthor ">Adolfo A.  Ferrando</span>. </span><span class="cited-content_cbyCitation_article-title">Deregulation of enhancer structure, function, and dynamics in acute lymphoblastic leukemia. </span><span class="cited-content_cbyCitation_journal-name">Trends in Immunology</span><span> <strong>2021,</strong> <em>42 </em>
                                    (5)
                                     , 418-431. <a href="https://doi.org/10.1016/j.it.2021.03.005" title="DOI URL">https://doi.org/10.1016/j.it.2021.03.005</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.it.2021.03.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.it.2021.03.005%26sid%3Dliteratum%253Aachs%26jtitle%3DTrends%2520in%2520Immunology%26atitle%3DDeregulation%252Bof%252Benhancer%252Bstructure%25252C%252Bfunction%25252C%252Band%252Bdynamics%252Bin%252Bacute%252Blymphoblastic%252Bleukemia%26aulast%3DBelver%26aufirst%3DLaura%26date%3D2021%26volume%3D42%26issue%3D5%26spage%3D418%26epage%3D431" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01487/20210305/images/medium/jm0c01487_0007.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01487/20210305/images/large/jm0c01487_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01487&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01487/20210305/images/medium/jm0c01487_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01487/20210305/images/large/jm0c01487_0001.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. (A) Domain organization of human BET proteins. (B) Conserved sequence of BRD4 BD1 and BRD4 BD2. Key varying residues are marked in different colors. (C) Recruitment of P-TEFb facilitates BRD4-enhanced Pol II elongation. BRD4 binds to Kac sites on histones and recruits BRD4 interactors and the P-TEFb complex, leading to enhanced transcriptional elongation by Ser2-phosphorylated Pol II. (D) BRD4 function in DNA damage response in both NHEJ and HR pathways.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01487/20210305/images/large/jm0c01487_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01487&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01487/20210305/images/medium/jm0c01487_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01487/20210305/images/large/jm0c01487_0002.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Resistance mechanisms.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01487/20210305/images/large/jm0c01487_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01487&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01487/20210305/images/medium/jm0c01487_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01487/20210305/images/large/jm0c01487_0003.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Reported novel selective inhibitors <b>1</b>–<b>20</b>. (A) Chemical structures of BRD4 BD1-selective inhibitors. The docking pose of compounds <b>2</b> and <b>6</b> with BRD4–BD1 in ribbon representation is shown on the right. Key residues, including Pro82, Gln85, Lys91, Tyr97, and Asn140, are shown in green sticks, and the amide ketone of compound <b>2</b> interacts with the critical conserved residue Asn140 to form a hydrogen bond. Similarly, in the docking pose of compound <b>6</b>, key residues Asn140, Tyr97, Gln85, and Pro82 are green. The carbonyl O atom of compound <b>6</b> interacts with the conserved residue Asn140, forming a direct H bond, and with Tyr97, forming an indirect hydrogen bond mediated by a water molecule. (B) Reported BRD4 BD2-selective inhibitors. (C) Chemical structures of BRD4-selective inhibitors.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01487/20210305/images/large/jm0c01487_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01487&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01487/20210305/images/medium/jm0c01487_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01487/20210305/images/large/jm0c01487_0004.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Chemical structures of dual-target inhibitors <b>21</b>–<b>31</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01487/20210305/images/large/jm0c01487_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01487&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01487/20210305/images/medium/jm0c01487_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01487/20210305/images/large/jm0c01487_0005.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Reported PROTAC degraders and PPI inhibitors. (A) PROTAC degraders based on CRL4<sup>CRBN</sup> (<b>32</b>–<b>38</b>) and CRL2<sup>VHL</sup> E3 ligase (<b>39</b>–<b>45</b>). (B) Chemical structures of <b>46</b>–<b>48</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-5/acs.jmedchem.0c01487/20210305/images/large/jm0c01487_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01487&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i22">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_23168" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_23168" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 134 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Picaud, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mangos, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keates, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lambert, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barsyte-Lovejoy, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Felletar, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Volkmer, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pawson, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gingras, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span> <span> </span><span class="NLM_article-title">Histone recognition and large-scale structural analysis of the human bromodomain family</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>149</i></span>,  <span class="NLM_fpage">214</span>– <span class="NLM_lpage">231</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2012.02.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1016%2Fj.cell.2012.02.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=22464331" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BC38XltVamurs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=149&publication_year=2012&pages=214-231&author=P.+Filippakopoulosauthor=S.+Picaudauthor=M.+Mangosauthor=T.+Keatesauthor=J.+P.+Lambertauthor=D.+Barsyte-Lovejoyauthor=I.+Felletarauthor=R.+Volkmerauthor=S.+Mullerauthor=T.+Pawsonauthor=A.+C.+Gingrasauthor=C.+H.+Arrowsmithauthor=S.+Knapp&title=Histone+recognition+and+large-scale+structural+analysis+of+the+human+bromodomain+family&doi=10.1016%2Fj.cell.2012.02.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Histone Recognition and Large-Scale Structural Analysis of the Human Bromodomain Family</span></div><div class="casAuthors">Filippakopoulos, Panagis; Picaud, Sarah; Mangos, Maria; Keates, Tracy; Lambert, Jean-Philippe; Barsyte-Lovejoy, Dalia; Felletar, Ildiko; Volkmer, Rudolf; Muller, Susanne; Pawson, Tony; Gingras, Anne-Claude; Arrowsmith, Cheryl H.; Knapp, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">149</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">214-231</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Bromodomains (BRDs) are protein interaction modules that specifically recognize ε-N-lysine acetylation motifs, a key event in the reading process of epigenetic marks.  The 61 BRDs in the human genome cluster into eight families based on structure/sequence similarity.  Here, we present 29 high-resoln. crystal structures, covering all BRD families.  These proteins are: ASH1L, ATAD2, BAZ2B, BPTF, BRD1, BRD3(1), BRD3(2), BRD4(1), BRD4(2), BRD9, BRDT(1), CECR2, EP300, CREBBP, GCN5L2, KIAA1240, PB1(1), PB1(2), PB1(3), PB1(4), PB1(5), PB1(6), PCAF, PHIP(2), TAF1(2), WDR9(2), BRD4(1).  Comprehensive crossfamily structural anal. identifies conserved and family-specific structural features that are necessary for specific acetylation-dependent substrate recognition.  Screening of more than 30 representative BRDs against systematic histone-peptide arrays identifies new BRD substrates and reveals a strong influence of flanking posttranslational modifications, such as acetylation and phosphorylation, suggesting that BRDs recognize combinations of marks rather than singly acetylated sequences.  We further uncovered a structural mechanism for the simultaneous binding and recognition of diverse diacetyl-contg. peptides by BRD4.  These data provide a foundation for structure-based drug design of specific inhibitors for this emerging target family.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoK2FSIcPoXZLVg90H21EOLACvtfcHk0li02afwzriamw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XltVamurs%253D&md5=05583a01a1cd4c3b59fc59c00e4c4777</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2012.02.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2012.02.013%26sid%3Dliteratum%253Aachs%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DMangos%26aufirst%3DM.%26aulast%3DKeates%26aufirst%3DT.%26aulast%3DLambert%26aufirst%3DJ.%2BP.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DFelletar%26aufirst%3DI.%26aulast%3DVolkmer%26aufirst%3DR.%26aulast%3DMuller%26aufirst%3DS.%26aulast%3DPawson%26aufirst%3DT.%26aulast%3DGingras%26aufirst%3DA.%2BC.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DKnapp%26aufirst%3DS.%26atitle%3DHistone%2520recognition%2520and%2520large-scale%2520structural%2520analysis%2520of%2520the%2520human%2520bromodomain%2520family%26jtitle%3DCell%26date%3D2012%26volume%3D149%26spage%3D214%26epage%3D231%26doi%3D10.1016%2Fj.cell.2012.02.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wu, S. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiang, C. M.</span></span> <span> </span><span class="NLM_article-title">The double bromodomain-containing chromatin adaptor BRD4 and transcriptional regulation</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>282</i></span>,  <span class="NLM_fpage">13141</span>– <span class="NLM_lpage">13145</span>, <span class="refDoi"> DOI: 10.1074/jbc.R700001200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1074%2Fjbc.R700001200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=17329240" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BD2sXks1Ghtbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=282&publication_year=2007&pages=13141-13145&author=S.+Y.+Wuauthor=C.+M.+Chiang&title=The+double+bromodomain-containing+chromatin+adaptor+BRD4+and+transcriptional+regulation&doi=10.1074%2Fjbc.R700001200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">The Double Bromodomain-containing Chromatin Adaptor Brd4 and Transcriptional Regulation</span></div><div class="casAuthors">Wu, Shwu-Yuan; Chiang, Cheng-Ming</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">282</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">13141-13145</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">A review.  Brd4 is a double bromodomain-contg. protein that binds preferentially to acetylated chromatin.  It belongs to the BET (bromodomains and extraterminal) family that includes mammalian Brd2, Brd3, Brd4, Brdt, Drosophila Fsh, yeast Bdf1, Bdf2, and corresponding homologues in other species.  Brd4 is essential for cellular growth and has been implicated in cell cycle control, DNA replication, and gene rearrangement found in t(15;19)-assocd. carcinomas.  Recently, Brd4 has been found in several transcription complexes, including the general cofactor Mediator and the P-TEFb elongation factor, and is capable of stimulating HIV-1 transcription in a Tat-independent manner.  In addn., Brd4 is used as a cellular adaptor by some animal and human papillomaviruses (HPV) for anchoring viral genomes to mitotic chromosomes.  This tethering, mediated by Brd4 interaction with virus-encoded E2 protein, facilitates viral genome segregation during mitosis.  Interestingly, Brd4 is also identified in a transcriptional silencing complex assembled by HPV E2 and turns out to be the long sought cellular corepressor that inhibits the expression of HPV-encoded E6 and E7 oncoproteins that antagonize p53 and pRB tumor suppressor activity, resp.  The dual role of Brd4 in gene activation and repression illustrates how a dynamic chromatin-binding adaptor is able to recruit distinct transcriptional regulators to modulate promoter activity through cell cycle progression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGohccDdZC9iKLVg90H21EOLACvtfcHk0li02afwzriamw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXks1Ghtbo%253D&md5=b025a5222b510a7009eca4289370b1d4</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1074%2Fjbc.R700001200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.R700001200%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DS.%2BY.%26aulast%3DChiang%26aufirst%3DC.%2BM.%26atitle%3DThe%2520double%2520bromodomain-containing%2520chromatin%2520adaptor%2520BRD4%2520and%2520transcriptional%2520regulation%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2007%26volume%3D282%26spage%3D13141%26epage%3D13145%26doi%3D10.1074%2Fjbc.R700001200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dhalluin, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carlson, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aggarwal, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, M.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, M.-M.</span></span> <span> </span><span class="NLM_article-title">Structure and ligand of a histone acetyltransferase bromodomain</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>399</i></span>,  <span class="NLM_fpage">491</span>– <span class="NLM_lpage">496</span>, <span class="refDoi"> DOI: 10.1038/20974</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1038%2F20974" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10365964" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADyaK1MXjvV2isLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=399&publication_year=1999&pages=491-496&author=C.+Dhalluinauthor=J.+E.+Carlsonauthor=L.+Zengauthor=C.+Heauthor=A.+K.+Aggarwalauthor=M.-M.+Zhouauthor=M.-M.+Zhou&title=Structure+and+ligand+of+a+histone+acetyltransferase+bromodomain&doi=10.1038%2F20974"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Structure and ligand of a histone acetyltransferase bromodomain</span></div><div class="casAuthors">Dhalluin, Christophe; Carlson, Justin E.; Zeng, Lei; He, Cheng; Aggarwal, Aneel K.; Zhou, Ming-Ming</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London)</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">399</span>
        (<span class="NLM_cas:issue">6735</span>),
    <span class="NLM_cas:pages">491-496</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Macmillan Magazines</span>)
        </div><div class="casAbstract">Histone acetylation is important in chromatin remodelling and gene activation.  Nearly all known histone-acetyltransferase (HAT)-assocd. transcriptional co-activators contain bromodomains, which are ∼110-amino-acid modules found in many chromatin-assocd. proteins.  Despite the wide occurrence of these bromodomains, their three-dimensional structure and binding partners remain unknown.  Here we report the soln. structure of the bromodomain of the HAT co-activator P/CAF (p300/CBP-assocd. factor).  The structure reveals an unusual left-handed up-and-down four-helix bundle.  In addn., we show by a combination of structural and site-directed mutagenesis studies that bromodomains can interact specifically with acetylated lysine, making them the first known protein modules to do so.  The nature of the recognition of acetyl-lysine by the P/CAF bromodomain is similar to that of acetyl-CoA by histone acetyltransferase.  Thus, the bromodomain is functionally linked to the HAT activity of co-activators in the regulation of gene transcription.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrsTbwcjeBG_rVg90H21EOLACvtfcHk0lh0rhk1jxV9uA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXjvV2isLo%253D&md5=ff1063ba81d81b431065dd6e3751ba79</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1038%2F20974&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F20974%26sid%3Dliteratum%253Aachs%26aulast%3DDhalluin%26aufirst%3DC.%26aulast%3DCarlson%26aufirst%3DJ.%2BE.%26aulast%3DZeng%26aufirst%3DL.%26aulast%3DHe%26aufirst%3DC.%26aulast%3DAggarwal%26aufirst%3DA.%2BK.%26aulast%3DZhou%26aufirst%3DM.-M.%26aulast%3DZhou%26aufirst%3DM.-M.%26atitle%3DStructure%2520and%2520ligand%2520of%2520a%2520histone%2520acetyltransferase%2520bromodomain%26jtitle%3DNature%26date%3D1999%26volume%3D399%26spage%3D491%26epage%3D496%26doi%3D10.1038%2F20974" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Devaiah, B. N</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Case-Borden, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gegonne, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hsu, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meerzaman, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dey, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ozato, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singer, D. S</span></span> <span> </span><span class="NLM_article-title">Singer D. S. BRD4 is a histone acetyltransferase that evicts nucleosomes from chromatin</span>. <i>Nat. Struct. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">540</span>– <span class="NLM_lpage">548</span>, <span class="refDoi"> DOI: 10.1038/nsmb.3228</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1038%2Fnsmb.3228" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=27159561" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BC28XnsFaitbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2016&pages=540-548&author=B.+N+Devaiahauthor=C.+Case-Bordenauthor=A.+Gegonneauthor=C.+H.+Hsuauthor=Q.+Chenauthor=D.+Meerzamanauthor=A.+Deyauthor=K.+Ozatoauthor=D.+S+Singer&title=Singer+D.+S.+BRD4+is+a+histone+acetyltransferase+that+evicts+nucleosomes+from+chromatin&doi=10.1038%2Fnsmb.3228"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">BRD4 is a histone acetyltransferase that evicts nucleosomes from chromatin</span></div><div class="casAuthors">Devaiah, Ballachanda N.; Case-Borden, Chanelle; Gegonne, Anne; Hsu, Chih Hao; Chen, Qingrong; Meerzaman, Daoud; Dey, Anup; Ozato, Keiko; Singer, Dinah S.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Structural & Molecular Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">540-548</span>CODEN:
                <span class="NLM_cas:coden">NSMBCU</span>;
        ISSN:<span class="NLM_cas:issn">1545-9993</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Bromodomain protein 4 (BRD4) is a chromatin-binding protein implicated in cancer and autoimmune diseases that functions as a scaffold for transcription factors at promoters and super-enhancers.  Although chromatin decompaction and transcriptional activation of target genes are assocd. with BRD4 binding, the mechanisms involved are unknown.  We report that BRD4 is a histone acetyltransferase (HAT) that acetylates histones H3 and H4 with a pattern distinct from those of other HATs.  Both mouse and human BRD4 have intrinsic HAT activity.  Importantly, BRD4 acetylates H3 K122, a residue crit. for nucleosome stability, thus resulting in nucleosome eviction and chromatin decompaction.  Nucleosome clearance by BRD4 occurs genome wide, including at its targets MYC, FOS and AURKB (Aurora B kinase), resulting in increased transcription.  These findings suggest a model wherein BRD4 actively links chromatin structure and transcription: it mediates chromatin decompaction by acetylating and evicting nucleosomes at target genes, thereby activating transcription.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpiIuN9x3auYLVg90H21EOLACvtfcHk0lh0rhk1jxV9uA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XnsFaitbw%253D&md5=ce482733951b61a9cf0d024b8cd60d30</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1038%2Fnsmb.3228&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnsmb.3228%26sid%3Dliteratum%253Aachs%26aulast%3DDevaiah%26aufirst%3DB.%2BN%26aulast%3DCase-Borden%26aufirst%3DC.%26aulast%3DGegonne%26aufirst%3DA.%26aulast%3DHsu%26aufirst%3DC.%2BH.%26aulast%3DChen%26aufirst%3DQ.%26aulast%3DMeerzaman%26aufirst%3DD.%26aulast%3DDey%26aufirst%3DA.%26aulast%3DOzato%26aufirst%3DK.%26aulast%3DSinger%26aufirst%3DD.%2BS%26atitle%3DSinger%2520D.%2520S.%2520BRD4%2520is%2520a%2520histone%2520acetyltransferase%2520that%2520evicts%2520nucleosomes%2520from%2520chromatin%26jtitle%3DNat.%2520Struct.%2520Mol.%2520Biol.%26date%3D2016%26volume%3D23%26spage%3D540%26epage%3D548%26doi%3D10.1038%2Fnsmb.3228" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span> <span> </span><span class="NLM_article-title">Targeting bromodomains: epigenetic readers of lysine acetylation</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">337</span>– <span class="NLM_lpage">356</span>, <span class="refDoi"> DOI: 10.1038/nrd4286</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1038%2Fnrd4286" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=24751816" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BC2cXmsFalsr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2014&pages=337-356&author=P.+Filippakopoulosauthor=S.+Knapp&title=Targeting+bromodomains%3A+epigenetic+readers+of+lysine+acetylation&doi=10.1038%2Fnrd4286"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting bromodomains: epigenetic readers of lysine acetylation</span></div><div class="casAuthors">Filippakopoulos, Panagis; Knapp, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">337-356</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Lysine acetylation is a key mechanism that regulates chromatin structure; aberrant acetylation levels have been linked to the development of several diseases.  Acetyl-lysine modifications create docking sites for bromodomains, which are small interaction modules found on diverse proteins, some of which have a key role in the acetylation-dependent assembly of transcriptional regulator complexes.  These complexes can then initiate transcriptional programs that result in phenotypic changes.  The recent discovery of potent and highly specific inhibitors for the BET (bromodomain and extra-terminal) family of bromodomains has stimulated intensive research activity in diverse therapeutic areas, particularly in oncol., where BET proteins regulate the expression of key oncogenes and anti-apoptotic proteins.  In addn., targeting BET bromodomains could hold potential for the treatment of inflammation and viral infection.  Here, we highlight recent progress in the development of bromodomain inhibitors, and their potential applications in drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrOSIcwBJKj4LVg90H21EOLACvtfcHk0lh0rhk1jxV9uA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXmsFalsr4%253D&md5=77049c82ce6a19b4685e3b7950399a07</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1038%2Fnrd4286&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd4286%26sid%3Dliteratum%253Aachs%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DKnapp%26aufirst%3DS.%26atitle%3DTargeting%2520bromodomains%253A%2520epigenetic%2520readers%2520of%2520lysine%2520acetylation%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2014%26volume%3D13%26spage%3D337%26epage%3D356%26doi%3D10.1038%2Fnrd4286" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Spriano, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stathis, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bertoni, F.</span></span> <span> </span><span class="NLM_article-title">Targeting BET bromodomain proteins in cancer: The example of lymphomas</span>. <i>Pharmacol. Ther.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>215</i></span>,  <span class="NLM_fpage">107631</span>, <span class="refDoi"> DOI: 10.1016/j.pharmthera.2020.107631</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1016%2Fj.pharmthera.2020.107631" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=32693114" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhsVyisrbI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=215&publication_year=2020&pages=107631&author=F.+Sprianoauthor=A.+Stathisauthor=F.+Bertoni&title=Targeting+BET+bromodomain+proteins+in+cancer%3A+The+example+of+lymphomas&doi=10.1016%2Fj.pharmthera.2020.107631"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting BET bromodomain proteins in cancer: The example of lymphomas</span></div><div class="casAuthors">Spriano, Filippo; Stathis, Anastasios; Bertoni, Francesco</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacology & Therapeutics</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">215</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">107631</span>CODEN:
                <span class="NLM_cas:coden">PHTHDT</span>;
        ISSN:<span class="NLM_cas:issn">0163-7258</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">A review.  The Bromo- and Extra-Terminal domain (BET) family proteins act as ''readers'' of acetylated histones and they are important transcription regulators.  BRD2, BRD3, BRD4 and BRDT, part of the BET family, are important in different tumors, where upregulation or translocation often occurs.  The potential of targeting BET proteins as anti-cancer treatment originated with data obtained with a first series of compds., and there are now several data supporting BET inhibition in both solid tumors and hematol. malignancies.  Despite very pos. preclin. data in different tumor types, the clin. results have been so far moderate.  Using lymphoma as an example to review the data produced in the lab. and in the context of the early clin. trials, we discuss the modalities to make BET targeting more efficient both generating novel generation of compds. and by exploring the combination with small mols. affecting various signaling pathways, BCL2, or DNA damage response signaling, but also with addnl. epigenetic agents and with immunotherapy.  We also discuss the mechanisms of resistance and the toxicity profiles so far reported.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp3Jn61Q24_wrVg90H21EOLACvtfcHk0lh0rhk1jxV9uA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhsVyisrbI&md5=cdb034e619849cb681f48c5b03f7cde8</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1016%2Fj.pharmthera.2020.107631&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.pharmthera.2020.107631%26sid%3Dliteratum%253Aachs%26aulast%3DSpriano%26aufirst%3DF.%26aulast%3DStathis%26aufirst%3DA.%26aulast%3DBertoni%26aufirst%3DF.%26atitle%3DTargeting%2520BET%2520bromodomain%2520proteins%2520in%2520cancer%253A%2520The%2520example%2520of%2520lymphomas%26jtitle%3DPharmacol.%2520Ther.%26date%3D2020%26volume%3D215%26spage%3D107631%26doi%3D10.1016%2Fj.pharmthera.2020.107631" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wu, S. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, C. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, H. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, C. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zuo, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsai, S. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsai, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ge, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wan, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiang, C. M.</span></span> <span> </span><span class="NLM_article-title">Opposing functions of BRD4 isoforms in breast cancer</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>78</i></span>,  <span class="NLM_fpage">1114</span>– <span class="NLM_lpage">1132</span>, <span class="refDoi"> DOI: 10.1016/j.molcel.2020.04.034</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1016%2Fj.molcel.2020.04.034" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=32446320" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhtVaqt7jP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=78&publication_year=2020&pages=1114-1132&author=S.+Y.+Wuauthor=C.+F.+Leeauthor=H.+T.+Laiauthor=C.+T.+Yuauthor=J.+E.+Leeauthor=H.+Zuoauthor=S.+Y.+Tsaiauthor=M.+J.+Tsaiauthor=K.+Geauthor=Y.+Wanauthor=C.+M.+Chiang&title=Opposing+functions+of+BRD4+isoforms+in+breast+cancer&doi=10.1016%2Fj.molcel.2020.04.034"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Opposing Functions of BRD4 Isoforms in Breast Cancer</span></div><div class="casAuthors">Wu, Shwu-Yuan; Lee, Chien-Fei; Lai, Hsien-Tsung; Yu, Cheng-Tai; Lee, Ji-Eun; Zuo, Hao; Tsai, Sophia Y.; Tsai, Ming-Jer; Ge, Kai; Wan, Yihong; Chiang, Cheng-Ming</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">78</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1114-1132.e10</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Bromodomain-contg. protein 4 (BRD4) is a cancer therapeutic target in ongoing clin. trials disrupting primarily BRD4-regulated transcription programs.  The role of BRD4 in cancer has been attributed mainly to the abundant long isoform (BRD4-L).  Here we show, by isoform-specific knockdown and endogenous protein detection, along with transgene expression, the less abundant BRD4 short isoform (BRD4-S) is oncogenic while BRD4-L is tumor-suppressive in breast cancer cell proliferation and migration, as well as mammary tumor formation and metastasis.  Through integrated RNA-seq, genome-wide ChIP-seq, and CUT&RUN assocn. profiling, we identify the Engrailed-1 (EN1) homeobox transcription factor as a key BRD4-S coregulator, particularly in triple-neg. breast cancer.  BRD4-S and EN1 comodulate the extracellular matrix (ECM)-assocd. matrisome network, including type II cystatin gene cluster, mucin 5, and cathepsin loci, via enhancer regulation of cancer-assocd. genes and pathways.  Our work highlights the importance of targeted therapies for the oncogenic, but not tumor-suppressive, activity of BRD4.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo9HICVoywZRbVg90H21EOLACvtfcHk0li-8kAMIe8t9Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhtVaqt7jP&md5=f330aa48d11b0dd2b7fb8193b4b05dbd</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2020.04.034&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2020.04.034%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DS.%2BY.%26aulast%3DLee%26aufirst%3DC.%2BF.%26aulast%3DLai%26aufirst%3DH.%2BT.%26aulast%3DYu%26aufirst%3DC.%2BT.%26aulast%3DLee%26aufirst%3DJ.%2BE.%26aulast%3DZuo%26aufirst%3DH.%26aulast%3DTsai%26aufirst%3DS.%2BY.%26aulast%3DTsai%26aufirst%3DM.%2BJ.%26aulast%3DGe%26aufirst%3DK.%26aulast%3DWan%26aufirst%3DY.%26aulast%3DChiang%26aufirst%3DC.%2BM.%26atitle%3DOpposing%2520functions%2520of%2520BRD4%2520isoforms%2520in%2520breast%2520cancer%26jtitle%3DMol.%2520Cell%26date%3D2020%26volume%3D78%26spage%3D1114%26epage%3D1132%26doi%3D10.1016%2Fj.molcel.2020.04.034" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wu, S. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, A. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, H. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiang, C. M.</span></span> <span> </span><span class="NLM_article-title">Phospho switch triggers BRD4 chromatin binding and activator recruitment for gene-specific targeting</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>49</i></span>,  <span class="NLM_fpage">843</span>– <span class="NLM_lpage">857</span>, <span class="refDoi"> DOI: 10.1016/j.molcel.2012.12.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1016%2Fj.molcel.2012.12.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=23317504" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BC3sXnvFersQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2013&pages=843-857&author=S.+Y.+Wuauthor=A.+Y.+Leeauthor=H.+T.+Laiauthor=H.+Zhangauthor=C.+M.+Chiang&title=Phospho+switch+triggers+BRD4+chromatin+binding+and+activator+recruitment+for+gene-specific+targeting&doi=10.1016%2Fj.molcel.2012.12.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Phospho switch triggers Brd4 chromatin binding and activator recruitment for gene-specific targeting</span></div><div class="casAuthors">Wu, Shwu-Yuan; Lee, A-Young; Lai, Hsien-Tsung; Zhang, Hong; Chiang, Cheng-Ming</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">843-857</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Bromodomain-contg. protein 4 (Brd4) is an epigenetic reader and transcriptional regulator recently identified as a cancer therapeutic target for acute myeloid leukemia, multiple myeloma, and Burkitt's lymphoma.  Although chromatin targeting is a crucial function of Brd4, there is little understanding of how bromodomains that bind acetylated histones are regulated, nor how the gene-specific activity of Brd4 is detd.  Via interaction screen and domain mapping, we identified p53 as a functional partner of Brd4.  Interestingly, Brd4 assocn. with p53 is modulated by casein kinase II (CK2)-mediated phosphorylation of a conserved acidic region in Brd4 that selectively contacts either a juxtaposed bromodomain or an adjacent basic region to dictate the ability of Brd4 binding to chromatin and also the recruitment of p53 to regulated promoters.  The unmasking of bromodomains and activator recruitment, concurrently triggered by the CK2 phospho switch, provide an intriguing mechanism for gene-specific targeting by a universal epigenetic reader.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoAKJqWUMTaTbVg90H21EOLACvtfcHk0li-8kAMIe8t9Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXnvFersQ%253D%253D&md5=39eeb31c048a047419a169e0991c8dd8</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2012.12.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2012.12.006%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DS.%2BY.%26aulast%3DLee%26aufirst%3DA.%2BY.%26aulast%3DLai%26aufirst%3DH.%2BT.%26aulast%3DZhang%26aufirst%3DH.%26aulast%3DChiang%26aufirst%3DC.%2BM.%26atitle%3DPhospho%2520switch%2520triggers%2520BRD4%2520chromatin%2520binding%2520and%2520activator%2520recruitment%2520for%2520gene-specific%2520targeting%26jtitle%3DMol.%2520Cell%26date%3D2013%26volume%3D49%26spage%3D843%26epage%3D857%26doi%3D10.1016%2Fj.molcel.2012.12.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, H. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ge, J. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeid, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harris, I. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jovanović, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murphy, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qiu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Endress, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reyes, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lim, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Font-Tello, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Syamala, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiao, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reddy Chilamakuri, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Papachristou, E. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">D’Santos, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anand, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hinohara, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McDonald, T. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luoma, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Modiste, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen, Q. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michel, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cejas, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kadoch, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jaffe, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wucherpfennig, K. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Long, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carroll, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brugge, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michor, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Polyak, K.</span></span> <span> </span><span class="NLM_article-title">Synthetic lethal and resistance interactions with BET bromodomain inhibitors in triple-negative breast cancer</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>78</i></span>,  <span class="NLM_fpage">1096</span>– <span class="NLM_lpage">1113</span>, <span class="refDoi"> DOI: 10.1016/j.molcel.2020.04.027</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1016%2Fj.molcel.2020.04.027" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=32416067" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BB3cXpsl2jt7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=78&publication_year=2020&pages=1096-1113&author=S.+Shuauthor=H.+J.+Wuauthor=J.+Y.+Geauthor=R.+Zeidauthor=I.+S.+Harrisauthor=B.+Jovanovi%C4%87author=K.+Murphyauthor=B.+Wangauthor=X.+Qiuauthor=J.+E.+Endressauthor=J.+Reyesauthor=K.+Limauthor=A.+Font-Telloauthor=S.+Syamalaauthor=T.+Xiaoauthor=C.+S.+Reddy+Chilamakuriauthor=E.+K.+Papachristouauthor=C.+D%E2%80%99Santosauthor=J.+Anandauthor=K.+Hinoharaauthor=W.+Liauthor=T.+O.+McDonaldauthor=A.+Luomaauthor=R.+J.+Modisteauthor=Q.+D.+Nguyenauthor=B.+Michelauthor=P.+Cejasauthor=C.+Kadochauthor=J.+D.+Jaffeauthor=K.+W.+Wucherpfennigauthor=J.+Qiauthor=X.+S.+Liuauthor=H.+Longauthor=M.+Brownauthor=J.+S.+Carrollauthor=J.+S.+Bruggeauthor=J.+Bradnerauthor=F.+Michorauthor=K.+Polyak&title=Synthetic+lethal+and+resistance+interactions+with+BET+bromodomain+inhibitors+in+triple-negative+breast+cancer&doi=10.1016%2Fj.molcel.2020.04.027"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Synthetic Lethal and Resistance Interactions with BET Bromodomain Inhibitors in Triple-Negative Breast Cancer</span></div><div class="casAuthors">Shu, Shaokun; Wu, Hua-Jun; Ge, Jennifer Y.; Zeid, Rhamy; Harris, Isaac S.; Jovanovic, Bojana; Murphy, Katherine; Wang, Binbin; Qiu, Xintao; Endress, Jennifer E.; Reyes, Jaime; Lim, Klothilda; Font-Tello, Alba; Syamala, Sudeepa; Xiao, Tengfei; Reddy Chilamakuri, Chandra Sekhar; Papachristou, Evangelia K.; D'Santos, Clive; Anand, Jayati; Hinohara, Kunihiko; Li, Wei; McDonald, Thomas O.; Luoma, Adrienne; Modiste, Rebecca J.; Nguyen, Quang-De; Michel, Brittany; Cejas, Paloma; Kadoch, Cigall; Jaffe, Jacob D.; Wucherpfennig, Kai W.; Qi, Jun; Liu, X. Shirley; Long, Henry; Brown, Myles; Carroll, Jason S.; Brugge, Joan S.; Bradner, James; Michor, Franziska; Polyak, Kornelia</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">78</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1096-1113.e8</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">BET bromodomain inhibitors (BBDIs) are candidate therapeutic agents for triple-neg. breast cancer (TNBC) and other cancer types, but inherent and acquired resistance to BBDIs limits their potential clin. use.  Using CRISPR and small-mol. inhibitor screens combined with comprehensive mol. profiling of BBDI response and resistance, we identified synthetic lethal interactions with BBDIs and genes that, when deleted, confer resistance.  We obsd. synergy with regulators of cell cycle progression, YAP, AXL, and SRC signaling, and chemotherapeutic agents.  We also uncovered functional similarities and differences among BRD2, BRD4, and BRD7.  Although deletion of BRD2 enhances sensitivity to BBDIs, BRD7 loss leads to gain of TEAD-YAP chromatin binding and luminal features assocd. with BBDI resistance.  Single-cell RNA-seq, ATAC-seq, and cellular barcoding anal. of BBDI responses in sensitive and resistant cell lines highlight significant heterogeneity among samples and demonstrate that BBDI resistance can be pre-existing or acquired.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo4Sz5TPbxoKLVg90H21EOLACvtfcHk0liKl-AbYn-yWw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXpsl2jt7g%253D&md5=8aabf40e0d9ca1c33c7876a0d68c1d65</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2020.04.027&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2020.04.027%26sid%3Dliteratum%253Aachs%26aulast%3DShu%26aufirst%3DS.%26aulast%3DWu%26aufirst%3DH.%2BJ.%26aulast%3DGe%26aufirst%3DJ.%2BY.%26aulast%3DZeid%26aufirst%3DR.%26aulast%3DHarris%26aufirst%3DI.%2BS.%26aulast%3DJovanovi%25C4%2587%26aufirst%3DB.%26aulast%3DMurphy%26aufirst%3DK.%26aulast%3DWang%26aufirst%3DB.%26aulast%3DQiu%26aufirst%3DX.%26aulast%3DEndress%26aufirst%3DJ.%2BE.%26aulast%3DReyes%26aufirst%3DJ.%26aulast%3DLim%26aufirst%3DK.%26aulast%3DFont-Tello%26aufirst%3DA.%26aulast%3DSyamala%26aufirst%3DS.%26aulast%3DXiao%26aufirst%3DT.%26aulast%3DReddy%2BChilamakuri%26aufirst%3DC.%2BS.%26aulast%3DPapachristou%26aufirst%3DE.%2BK.%26aulast%3DD%25E2%2580%2599Santos%26aufirst%3DC.%26aulast%3DAnand%26aufirst%3DJ.%26aulast%3DHinohara%26aufirst%3DK.%26aulast%3DLi%26aufirst%3DW.%26aulast%3DMcDonald%26aufirst%3DT.%2BO.%26aulast%3DLuoma%26aufirst%3DA.%26aulast%3DModiste%26aufirst%3DR.%2BJ.%26aulast%3DNguyen%26aufirst%3DQ.%2BD.%26aulast%3DMichel%26aufirst%3DB.%26aulast%3DCejas%26aufirst%3DP.%26aulast%3DKadoch%26aufirst%3DC.%26aulast%3DJaffe%26aufirst%3DJ.%2BD.%26aulast%3DWucherpfennig%26aufirst%3DK.%2BW.%26aulast%3DQi%26aufirst%3DJ.%26aulast%3DLiu%26aufirst%3DX.%2BS.%26aulast%3DLong%26aufirst%3DH.%26aulast%3DBrown%26aufirst%3DM.%26aulast%3DCarroll%26aufirst%3DJ.%2BS.%26aulast%3DBrugge%26aufirst%3DJ.%2BS.%26aulast%3DBradner%26aufirst%3DJ.%26aulast%3DMichor%26aufirst%3DF.%26aulast%3DPolyak%26aufirst%3DK.%26atitle%3DSynthetic%2520lethal%2520and%2520resistance%2520interactions%2520with%2520BET%2520bromodomain%2520inhibitors%2520in%2520triple-negative%2520breast%2520cancer%26jtitle%3DMol.%2520Cell%26date%3D2020%26volume%3D78%26spage%3D1096%26epage%3D1113%26doi%3D10.1016%2Fj.molcel.2020.04.027" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wu, S. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nin, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, A. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simanski, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kodadek, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiang, C. M.</span></span> <span> </span><span class="NLM_article-title">BRD4 phosphorylation regulates HPV E2-mediated viral transcription, origin replication, and cellular MMP-9 expression</span>. <i>Cell Rep.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">1733</span>– <span class="NLM_lpage">1748</span>, <span class="refDoi"> DOI: 10.1016/j.celrep.2016.07.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1016%2Fj.celrep.2016.07.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=27477287" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BC28Xht1KlsbnO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2016&pages=1733-1748&author=S.+Y.+Wuauthor=D.+S.+Ninauthor=A.+Y.+Leeauthor=S.+Simanskiauthor=T.+Kodadekauthor=C.+M.+Chiang&title=BRD4+phosphorylation+regulates+HPV+E2-mediated+viral+transcription%2C+origin+replication%2C+and+cellular+MMP-9+expression&doi=10.1016%2Fj.celrep.2016.07.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">BRD4 Phosphorylation Regulates HPV E2-Mediated Viral Transcription, Origin Replication, and Cellular MMP-9 Expression</span></div><div class="casAuthors">Wu, Shwu-Yuan; Nin, Dawn Sijin; Lee, A-Young; Simanski, Scott; Kodadek, Thomas; Chiang, Cheng-Ming</div><div class="citationInfo"><span class="NLM_cas:title">Cell Reports</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1733-1748</span>CODEN:
                <span class="NLM_cas:coden">CREED8</span>;
        ISSN:<span class="NLM_cas:issn">2211-1247</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Post-translational modification can modulate protein conformation and alter binding partner recruitment within gene regulatory regions.  Here, we report that bromodomain-contg. protein 4 (BRD4), a transcription co-factor and chromatin regulator, uses a phosphorylation-induced switch mechanism to recruit E2 protein encoded by cancer-assocd. human papillomavirus (HPV) to viral early gene and cellular matrix metalloproteinase-9 (MMP-9) promoters.  Enhanced MMP-9 expression, induced upon keratinocyte differentiation, occurs via BRD4-dependent recruitment of active AP-1 and NF-κB to their target sequences.  This is triggered by replacement of AP-1 family members JunB and JunD by c-Jun and by re-localization of NF-κB from the cytoplasm to the nucleus.  In addn., BRD4 phosphorylation is crit. for E2- and origin-dependent HPV DNA replication.  A class of phospho-BRD4-targeting compds., distinct from the BET bromodomain inhibitors, effectively blocks BRD4 phosphorylation-specific functions in transcription and factor recruitment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGomvZx5RmFIRLVg90H21EOLACvtfcHk0liKl-AbYn-yWw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xht1KlsbnO&md5=10bde9118396d8a5a1f4614fb64a566c</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1016%2Fj.celrep.2016.07.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.celrep.2016.07.001%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DS.%2BY.%26aulast%3DNin%26aufirst%3DD.%2BS.%26aulast%3DLee%26aufirst%3DA.%2BY.%26aulast%3DSimanski%26aufirst%3DS.%26aulast%3DKodadek%26aufirst%3DT.%26aulast%3DChiang%26aufirst%3DC.%2BM.%26atitle%3DBRD4%2520phosphorylation%2520regulates%2520HPV%2520E2-mediated%2520viral%2520transcription%252C%2520origin%2520replication%252C%2520and%2520cellular%2520MMP-9%2520expression%26jtitle%3DCell%2520Rep.%26date%3D2016%26volume%3D16%26spage%3D1733%26epage%3D1748%26doi%3D10.1016%2Fj.celrep.2016.07.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, C. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, H. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Witwicki, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tabassum, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Janiszewska, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin Huh, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ryan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doherty, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mohammed, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stover, D. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ekram, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peluffo, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">D’Santos, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krop, I. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dillon, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McKeown, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ott, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ni, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rao, P. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duarte, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, S.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiang, C.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anders, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winer, E. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Letai, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barry, W. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carroll, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Long, H. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shirley Liu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meyer, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Polyak, K.</span></span> <span> </span><span class="NLM_article-title">Response and resistance to BET bromodomain inhibitors in triple-negative breast cancer</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>529</i></span>,  <span class="NLM_fpage">413</span>– <span class="NLM_lpage">417</span>, <span class="refDoi"> DOI: 10.1038/nature16508</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1038%2Fnature16508" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=26735014" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BC28Xns1Gmtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=529&publication_year=2016&pages=413-417&author=S.+Shuauthor=C.+Y.+Linauthor=H.+H.+Heauthor=R.+M.+Witwickiauthor=D.+P.+Tabassumauthor=J.+M.+Robertsauthor=M.+Janiszewskaauthor=S.+Jin+Huhauthor=Y.+Liangauthor=J.+Ryanauthor=E.+Dohertyauthor=H.+Mohammedauthor=H.+Guoauthor=D.+G.+Stoverauthor=M.+B.+Ekramauthor=G.+Peluffoauthor=J.+Brownauthor=C.+D%E2%80%99Santosauthor=I.+E.+Kropauthor=D.+Dillonauthor=M.+McKeownauthor=C.+Ottauthor=J.+Qiauthor=M.+Niauthor=P.+K.+Raoauthor=M.+Duarteauthor=S.-Y.+Wuauthor=C.-M.+Chiangauthor=L.+Andersauthor=R.+A.+Youngauthor=E.+P.+Winerauthor=A.+Letaiauthor=W.+T.+Barryauthor=J.+S.+Carrollauthor=H.+W.+Longauthor=M.+Brownauthor=X.+Shirley+Liuauthor=C.+A.+Meyerauthor=J.+E.+Bradnerauthor=K.+Polyak&title=Response+and+resistance+to+BET+bromodomain+inhibitors+in+triple-negative+breast+cancer&doi=10.1038%2Fnature16508"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Response and resistance to BET bromodomain inhibitors in triple-negative breast cancer</span></div><div class="casAuthors">Shu, Shaokun; Lin, Charles Y.; He, Housheng Hansen; Witwicki, Robert M.; Tabassum, Doris P.; Roberts, Justin M.; Janiszewska, Michalina; Huh, Sung Jin; Liang, Yi; Ryan, Jeremy; Doherty, Ernest; Mohammed, Hisham; Guo, Hao; Stover, Daniel G.; Ekram, Muhammad B.; Peluffo, Guillermo; Brown, Jonathan; D'Santos, Clive; Krop, Ian E.; Dillon, Deborah; McKeown, Michael; Ott, Christopher; Qi, Jun; Ni, Min; Rao, Prakash K.; Duarte, Melissa; Wu, Shwu-Yuan; Chiang, Cheng-Ming; Anders, Lars; Young, Richard A.; Winer, Eric P.; Letai, Antony; Barry, William T.; Carroll, Jason S.; Long, Henry W.; Brown, Myles; Shirley Liu, X.; Meyer, Clifford A.; Bradner, James E.; Polyak, Kornelia</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">529</span>
        (<span class="NLM_cas:issue">7586</span>),
    <span class="NLM_cas:pages">413-417</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Triple-neg. breast cancer (TNBC) is a heterogeneous and clin. aggressive disease for which there is no targeted therapy.  BET bromodomain inhibitors, which have shown efficacy in several models of cancer, have not been evaluated in TNBC.  These inhibitors displace BET bromodomain proteins such as BRD4 from chromatin by competing with their acetyl-lysine recognition modules, leading to inhibition of oncogenic transcriptional programs.  Here we report the preferential sensitivity of TNBCs to BET bromodomain inhibition in vitro and in vivo, establishing a rationale for clin. investigation and further motivation to understand mechanisms of resistance.  In paired cell lines selected for acquired resistance to BET inhibition from previously sensitive TNBCs, we failed to identify gatekeeper mutations, new driver events or drug pump activation.  BET-resistant TNBC cells remain dependent on wild-type BRD4, which supports transcription and cell proliferation in a bromodomain-independent manner.  Proteomic studies of resistant TNBC identify strong assocn. with MED1 and hyper-phosphorylation of BRD4 attributable to decreased activity of PP2A, identified here as a principal BRD4 serine phosphatase.  Together, these studies provide a rationale for BET inhibition in TNBC and present mechanism-based combination strategies to anticipate clin. drug resistance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqZu5A85pdQDbVg90H21EOLACvtfcHk0lgEGyyNKjsjjw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xns1Gmtw%253D%253D&md5=50a03861d9156ee6e344e95c5f5361b9</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1038%2Fnature16508&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature16508%26sid%3Dliteratum%253Aachs%26aulast%3DShu%26aufirst%3DS.%26aulast%3DLin%26aufirst%3DC.%2BY.%26aulast%3DHe%26aufirst%3DH.%2BH.%26aulast%3DWitwicki%26aufirst%3DR.%2BM.%26aulast%3DTabassum%26aufirst%3DD.%2BP.%26aulast%3DRoberts%26aufirst%3DJ.%2BM.%26aulast%3DJaniszewska%26aufirst%3DM.%26aulast%3DJin%2BHuh%26aufirst%3DS.%26aulast%3DLiang%26aufirst%3DY.%26aulast%3DRyan%26aufirst%3DJ.%26aulast%3DDoherty%26aufirst%3DE.%26aulast%3DMohammed%26aufirst%3DH.%26aulast%3DGuo%26aufirst%3DH.%26aulast%3DStover%26aufirst%3DD.%2BG.%26aulast%3DEkram%26aufirst%3DM.%2BB.%26aulast%3DPeluffo%26aufirst%3DG.%26aulast%3DBrown%26aufirst%3DJ.%26aulast%3DD%25E2%2580%2599Santos%26aufirst%3DC.%26aulast%3DKrop%26aufirst%3DI.%2BE.%26aulast%3DDillon%26aufirst%3DD.%26aulast%3DMcKeown%26aufirst%3DM.%26aulast%3DOtt%26aufirst%3DC.%26aulast%3DQi%26aufirst%3DJ.%26aulast%3DNi%26aufirst%3DM.%26aulast%3DRao%26aufirst%3DP.%2BK.%26aulast%3DDuarte%26aufirst%3DM.%26aulast%3DWu%26aufirst%3DS.-Y.%26aulast%3DChiang%26aufirst%3DC.-M.%26aulast%3DAnders%26aufirst%3DL.%26aulast%3DYoung%26aufirst%3DR.%2BA.%26aulast%3DWiner%26aufirst%3DE.%2BP.%26aulast%3DLetai%26aufirst%3DA.%26aulast%3DBarry%26aufirst%3DW.%2BT.%26aulast%3DCarroll%26aufirst%3DJ.%2BS.%26aulast%3DLong%26aufirst%3DH.%2BW.%26aulast%3DBrown%26aufirst%3DM.%26aulast%3DShirley%2BLiu%26aufirst%3DX.%26aulast%3DMeyer%26aufirst%3DC.%2BA.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26aulast%3DPolyak%26aufirst%3DK.%26atitle%3DResponse%2520and%2520resistance%2520to%2520BET%2520bromodomain%2520inhibitors%2520in%2520triple-negative%2520breast%2520cancer%26jtitle%3DNature%26date%3D2016%26volume%3D529%26spage%3D413%26epage%3D417%26doi%3D10.1038%2Fnature16508" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chiang, C. M.</span></span> <span> </span><span class="NLM_article-title">Phospho-BRD4: transcription plasticity and drug targeting</span>. <i>Drug Discovery Today: Technol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">17</span>– <span class="NLM_lpage">22</span>, <span class="refDoi"> DOI: 10.1016/j.ddtec.2016.05.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1016%2Fj.ddtec.2016.05.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=27769352" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A280%3ADC%252BC2srkvFOjsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2016&pages=17-22&author=C.+M.+Chiang&title=Phospho-BRD4%3A+transcription+plasticity+and+drug+targeting&doi=10.1016%2Fj.ddtec.2016.05.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Phospho-BRD4: transcription plasticity and drug targeting</span></div><div class="casAuthors">Chiang Cheng-Ming</div><div class="citationInfo"><span class="NLM_cas:title">Drug discovery today. Technologies</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">17-22</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BRD4 is an epigenetic regulator and transcription cofactor whose phosphorylation by CK2 and dephosphorylation by PP2A modulates its function in chromatin targeting, factor recruitment, and cancer progression.  While the bromodomains of BET family proteins, including BRD4, BRD2, BRD3 and BRDT, have been the primary targets of small compounds such as JQ1, I-BET and MS417 that show promising anticancer effects against some hematopoietic cancer and solid tumors, drug resistance upon prolonged treatment necessitates a better understanding of alternative pathways underlying not only the resistance but also persistent BET protein dependence for identifying new targets and effective combination therapy strategies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSEothR-V9Y-GhHGZei4OW9fW6udTcc2eaZiqk7uKOaGrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2srkvFOjsA%253D%253D&md5=3337d696a5a0fdba97c265d2c15fd8df</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1016%2Fj.ddtec.2016.05.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ddtec.2016.05.003%26sid%3Dliteratum%253Aachs%26aulast%3DChiang%26aufirst%3DC.%2BM.%26atitle%3DPhospho-BRD4%253A%2520transcription%2520plasticity%2520and%2520drug%2520targeting%26jtitle%3DDrug%2520Discovery%2520Today%253A%2520Technol.%26date%3D2016%26volume%3D19%26spage%3D17%26epage%3D22%26doi%3D10.1016%2Fj.ddtec.2016.05.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cheung, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jaganathan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharma, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Konuma, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ren, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olson, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaplan, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Littman, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walsh, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiong, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, M. M.</span></span> <span> </span><span class="NLM_article-title">Distinct roles of BRD2 and BRD4 in potentiating the transcriptional program for Th17 cell differentiation</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>65</i></span>,  <span class="NLM_fpage">1068</span>– <span class="NLM_lpage">1080</span>, <span class="refDoi"> DOI: 10.1016/j.molcel.2016.12.022</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1016%2Fj.molcel.2016.12.022" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=28262505" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BC2sXjvVWqu78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=65&publication_year=2017&pages=1068-1080&author=K.+L.+Cheungauthor=F.+Zhangauthor=A.+Jaganathanauthor=R.+Sharmaauthor=Q.+Zhangauthor=T.+Konumaauthor=T.+Shenauthor=J.+Y.+Leeauthor=C.+Renauthor=C.+H.+Chenauthor=G.+Luauthor=M.+R.+Olsonauthor=W.+Zhangauthor=M.+H.+Kaplanauthor=D.+R.+Littmanauthor=M.+J.+Walshauthor=H.+Xiongauthor=L.+Zengauthor=M.+M.+Zhou&title=Distinct+roles+of+BRD2+and+BRD4+in+potentiating+the+transcriptional+program+for+Th17+cell+differentiation&doi=10.1016%2Fj.molcel.2016.12.022"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Distinct Roles of Brd2 and Brd4 in Potentiating the Transcriptional Program for Th17 Cell Differentiation</span></div><div class="casAuthors">Cheung, Ka Lung; Zhang, Fan; Jaganathan, Anbalagan; Sharma, Rajal; Zhang, Qiang; Konuma, Tsuyoshi; Shen, Tong; Lee, June-Yong; Ren, Chunyan; Chen, Chih-Hung; Lu, Geming; Olson, Matthew R.; Zhang, Weijia; Kaplan, Mark H.; Littman, Dan R.; Walsh, Martin J.; Xiong, Huabao; Zeng, Lei; Zhou, Ming-Ming</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">65</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1068-1080.e5</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">The BET proteins are major transcriptional regulators and have emerged as new drug targets, but their functional distinction has remained elusive.  In this study, we report that the BET family members Brd2 and Brd4 exert distinct genomic functions at genes whose transcription they co-regulate during mouse T helper 17 (Th17) cell differentiation.  Brd2 is assocd. with the chromatin insulator CTCF and the cohesin complex to support cis-regulatory enhancer assembly for gene transcriptional activation.  In this context, Brd2 binds the transcription factor Stat3 in an acetylation-sensitive manner and facilitates Stat3 recruitment to active enhancers occupied with transcription factors Irf4 and Batf.  In parallel, Brd4 temporally controls RNA polymerase II (Pol II) processivity during transcription elongation through cyclin T1 and Cdk9 recruitment and Pol II Ser2 phosphorylation.  Collectively, our study uncovers both sep. and interdependent Brd2 and Brd4 functions in potentiating the genetic program required for Th17 cell development and adaptive immunity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp06k1TG5JMHrVg90H21EOLACvtfcHk0lj35-4s6s8Gcg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXjvVWqu78%253D&md5=2f2cd782230c84d9e128b04a2d8bba27</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2016.12.022&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2016.12.022%26sid%3Dliteratum%253Aachs%26aulast%3DCheung%26aufirst%3DK.%2BL.%26aulast%3DZhang%26aufirst%3DF.%26aulast%3DJaganathan%26aufirst%3DA.%26aulast%3DSharma%26aufirst%3DR.%26aulast%3DZhang%26aufirst%3DQ.%26aulast%3DKonuma%26aufirst%3DT.%26aulast%3DShen%26aufirst%3DT.%26aulast%3DLee%26aufirst%3DJ.%2BY.%26aulast%3DRen%26aufirst%3DC.%26aulast%3DChen%26aufirst%3DC.%2BH.%26aulast%3DLu%26aufirst%3DG.%26aulast%3DOlson%26aufirst%3DM.%2BR.%26aulast%3DZhang%26aufirst%3DW.%26aulast%3DKaplan%26aufirst%3DM.%2BH.%26aulast%3DLittman%26aufirst%3DD.%2BR.%26aulast%3DWalsh%26aufirst%3DM.%2BJ.%26aulast%3DXiong%26aufirst%3DH.%26aulast%3DZeng%26aufirst%3DL.%26aulast%3DZhou%26aufirst%3DM.%2BM.%26atitle%3DDistinct%2520roles%2520of%2520BRD2%2520and%2520BRD4%2520in%2520potentiating%2520the%2520transcriptional%2520program%2520for%2520Th17%2520cell%2520differentiation%26jtitle%3DMol.%2520Cell%26date%3D2017%26volume%3D65%26spage%3D1068%26epage%3D1080%26doi%3D10.1016%2Fj.molcel.2016.12.022" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gilan, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rioja, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knezevic, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bell, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yeung, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harker, N. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lam, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bamborough, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petretich, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Urh, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Atkinson, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bassil, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vassiliadis, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burr, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Preston, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wellaway, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Werner, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michon, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gobbetti, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumar, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soden, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haynes, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vappiani, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tough, D. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dawson, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bantscheff, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindon, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drewes, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Demont, E. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daniels, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grandi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dawson, M. A.</span></span> <span> </span><span class="NLM_article-title">Selective targeting of BD1 and BD2 of the BET proteins in cancer and immunoinflammation</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>368</i></span>,  <span class="NLM_fpage">387</span>– <span class="NLM_lpage">394</span>, <span class="refDoi"> DOI: 10.1126/science.aaz8455</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1126%2Fscience.aaz8455" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=32193360" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BB3cXnslKrtrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=368&publication_year=2020&pages=387-394&author=O.+Gilanauthor=I.+Riojaauthor=K.+Knezevicauthor=M.+J.+Bellauthor=M.+M.+Yeungauthor=N.+R.+Harkerauthor=E.+Lamauthor=C.+W.+Chungauthor=P.+Bamboroughauthor=M.+Petretichauthor=M.+Urhauthor=S.+J.+Atkinsonauthor=A.+K.+Bassilauthor=E.+J.+Robertsauthor=D.+Vassiliadisauthor=M.+L.+Burrauthor=A.+Prestonauthor=C.+Wellawayauthor=T.+Wernerauthor=J.+R.+Grayauthor=A.+M.+Michonauthor=T.+Gobbettiauthor=V.+Kumarauthor=P.+E.+Sodenauthor=A.+Haynesauthor=J.+Vappianiauthor=D.+F.+Toughauthor=S.+Taylorauthor=S.+J.+Dawsonauthor=M.+Bantscheffauthor=M.+Lindonauthor=G.+Drewesauthor=E.+H.+Demontauthor=D.+L.+Danielsauthor=P.+Grandiauthor=R.+K.+Prinjhaauthor=M.+A.+Dawson&title=Selective+targeting+of+BD1+and+BD2+of+the+BET+proteins+in+cancer+and+immunoinflammation&doi=10.1126%2Fscience.aaz8455"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Selective targeting of BD1 and BD2 of the BET proteins in cancer and immunoinflammation</span></div><div class="casAuthors">Gilan, Omer; Rioja, Inmaculada; Knezevic, Kathy; Bell, Matthew J.; Yeung, Miriam M.; Harker, Nicola R.; Lam, Enid Y. N.; Chung, Chun-wa; Bamborough, Paul; Petretich, Massimo; Urh, Marjeta; Atkinson, Stephen J.; Bassil, Anna K.; Roberts, Emma J.; Vassiliadis, Dane; Burr, Marian L.; Preston, Alex G. S.; Wellaway, Christopher; Werner, Thilo; Gray, James R.; Michon, Anne-Marie; Gobbetti, Thomas; Kumar, Vinod; Soden, Peter E.; Haynes, Andrea; Vappiani, Johanna; Tough, David F.; Taylor, Simon; Dawson, Sarah-Jane; Bantscheff, Marcus; Lindon, Matthew; Drewes, Gerard; Demont, Emmanuel H.; Daniels, Danette L.; Grandi, Paola; Prinjha, Rab K.; Dawson, Mark A.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">368</span>
        (<span class="NLM_cas:issue">6489</span>),
    <span class="NLM_cas:pages">387-394</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">1095-9203</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">The two tandem bromodomains of the BET (bromodomain and extraterminal domain) proteins enable chromatin binding to facilitate transcription.  Drugs that inhibit both bromodomains equally have shown efficacy in certain malignant and inflammatory conditions.  To explore the individual functional contributions of the first (BD1) and second (BD2) bromodomains in biol. and therapy, we developed selective BD1 and BD2 inhibitors.  We found that steady-state gene expression primarily requires BD1, whereas the rapid increase of gene expression induced by inflammatory stimuli requires both BD1 and BD2 of all BET proteins.  BD1 inhibitors phenocopied the effects of pan-BET inhibitors in cancer models, whereas BD2 inhibitors were predominantly effective in models of inflammatory and autoimmune disease.  These insights into the differential requirement of BD1 and BD2 for the maintenance and induction of gene expression may guide future BET-targeted therapies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr078AuJubsqbVg90H21EOLACvtfcHk0lj35-4s6s8Gcg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXnslKrtrg%253D&md5=d78cfea091fcf5ee18735b02b2120dd7</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1126%2Fscience.aaz8455&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.aaz8455%26sid%3Dliteratum%253Aachs%26aulast%3DGilan%26aufirst%3DO.%26aulast%3DRioja%26aufirst%3DI.%26aulast%3DKnezevic%26aufirst%3DK.%26aulast%3DBell%26aufirst%3DM.%2BJ.%26aulast%3DYeung%26aufirst%3DM.%2BM.%26aulast%3DHarker%26aufirst%3DN.%2BR.%26aulast%3DLam%26aufirst%3DE.%26aulast%3DChung%26aufirst%3DC.%2BW.%26aulast%3DBamborough%26aufirst%3DP.%26aulast%3DPetretich%26aufirst%3DM.%26aulast%3DUrh%26aufirst%3DM.%26aulast%3DAtkinson%26aufirst%3DS.%2BJ.%26aulast%3DBassil%26aufirst%3DA.%2BK.%26aulast%3DRoberts%26aufirst%3DE.%2BJ.%26aulast%3DVassiliadis%26aufirst%3DD.%26aulast%3DBurr%26aufirst%3DM.%2BL.%26aulast%3DPreston%26aufirst%3DA.%26aulast%3DWellaway%26aufirst%3DC.%26aulast%3DWerner%26aufirst%3DT.%26aulast%3DGray%26aufirst%3DJ.%2BR.%26aulast%3DMichon%26aufirst%3DA.%2BM.%26aulast%3DGobbetti%26aufirst%3DT.%26aulast%3DKumar%26aufirst%3DV.%26aulast%3DSoden%26aufirst%3DP.%2BE.%26aulast%3DHaynes%26aufirst%3DA.%26aulast%3DVappiani%26aufirst%3DJ.%26aulast%3DTough%26aufirst%3DD.%2BF.%26aulast%3DTaylor%26aufirst%3DS.%26aulast%3DDawson%26aufirst%3DS.%2BJ.%26aulast%3DBantscheff%26aufirst%3DM.%26aulast%3DLindon%26aufirst%3DM.%26aulast%3DDrewes%26aufirst%3DG.%26aulast%3DDemont%26aufirst%3DE.%2BH.%26aulast%3DDaniels%26aufirst%3DD.%2BL.%26aulast%3DGrandi%26aufirst%3DP.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DDawson%26aufirst%3DM.%2BA.%26atitle%3DSelective%2520targeting%2520of%2520BD1%2520and%2520BD2%2520of%2520the%2520BET%2520proteins%2520in%2520cancer%2520and%2520immunoinflammation%26jtitle%3DScience%26date%3D2020%26volume%3D368%26spage%3D387%26epage%3D394%26doi%3D10.1126%2Fscience.aaz8455" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Crowe, B. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Larue, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hess, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kvaratskhelia, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Foster, M. P.</span></span> <span> </span><span class="NLM_article-title">Structure of the BRD4 ET domain bound to a C-terminal motif from gamma-retroviral integrases reveals a conserved mechanism of interaction</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>113</i></span>,  <span class="NLM_fpage">2086</span>– <span class="NLM_lpage">2091</span>, <span class="refDoi"> DOI: 10.1073/pnas.1516813113</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1073%2Fpnas.1516813113" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=26858406" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BC28Xit12ku7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=113&publication_year=2016&pages=2086-2091&author=B.+L.+Croweauthor=R.+C.+Larueauthor=C.+Yuanauthor=S.+Hessauthor=M.+Kvaratskheliaauthor=M.+P.+Foster&title=Structure+of+the+BRD4+ET+domain+bound+to+a+C-terminal+motif+from+gamma-retroviral+integrases+reveals+a+conserved+mechanism+of+interaction&doi=10.1073%2Fpnas.1516813113"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Structure of the Brd4 ET domain bound to a C-terminal motif from γ-retroviral integrases reveals a conserved mechanism of interaction</span></div><div class="casAuthors">Crowe, Brandon L.; Larue, Ross C.; Yuan, Chunhua; Hess, Sonja; Kvaratskhelia, Mamuka; Foster, Mark P.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">113</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2086-2091</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The bromodomain and extraterminal domain (BET) protein family (Brd2, Brd3, Brd4, and Brdt) are promising therapeutic targets for a range of diseases linked to transcriptional activation, cancer, viral latency, and viral integration.  Tandem bromodomains selectively tether BET proteins to chromatin by engaging cognate acetylated histone marks, and the extraterminal (ET) domain is the focal point for recruiting a range of cellular and viral proteins.  BET proteins guide γ-retroviral integration to transcription start sites and enhancers through bimodal interaction with chromatin and the γ-retroviral integrase (IN).  We report the NMR-derived soln. structure of the Brd4 ET domain bound to a conserved peptide sequence from the C terminus of murine leukemia virus (MLV) IN.  The complex reveals a protein-protein interaction governed by the binding-coupled folding of disordered regions in both interacting partners to form a well-structured intermol. three-stranded β sheet.  In addn., we show that a peptide comprising the ET binding motif (EBM) of MLV IN can disrupt the cognate interaction of Brd4 with histone-lysine N-methyltransferase NSD3, and that substitutions of Brd4 ET residues essential for binding MLV IN also impair interaction of Brd4 with a no. of cellular partners involved in transcriptional regulation and chromatin remodeling.  This suggests that γ-retroviruses have evolved the EBM to mimic a cognate interaction motif to achieve effective integration in host chromatin.  Collectively, our findings identify key structural features of the ET domain of Brd4 that allow for interactions with both cellular and viral proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoTRmI3Hgkr8LVg90H21EOLACvtfcHk0lj35-4s6s8Gcg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xit12ku7Y%253D&md5=9a03dcb089e5d800b39c8c42374d8661</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1516813113&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1516813113%26sid%3Dliteratum%253Aachs%26aulast%3DCrowe%26aufirst%3DB.%2BL.%26aulast%3DLarue%26aufirst%3DR.%2BC.%26aulast%3DYuan%26aufirst%3DC.%26aulast%3DHess%26aufirst%3DS.%26aulast%3DKvaratskhelia%26aufirst%3DM.%26aulast%3DFoster%26aufirst%3DM.%2BP.%26atitle%3DStructure%2520of%2520the%2520BRD4%2520ET%2520domain%2520bound%2520to%2520a%2520C-terminal%2520motif%2520from%2520gamma-retroviral%2520integrases%2520reveals%2520a%2520conserved%2520mechanism%2520of%2520interaction%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2016%26volume%3D113%26spage%3D2086%26epage%3D2091%26doi%3D10.1073%2Fpnas.1516813113" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bisgrove, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mahmoudi, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Henklein, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verdin, E.</span></span> <span> </span><span class="NLM_article-title">Conserved P-TEFb-interacting domain of BRD4 inhibits HIV transcription</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>104</i></span>,  <span class="NLM_fpage">13690</span>– <span class="NLM_lpage">13695</span>, <span class="refDoi"> DOI: 10.1073/pnas.0705053104</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1073%2Fpnas.0705053104" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=17690245" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BD2sXpvVOjsLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=104&publication_year=2007&pages=13690-13695&author=D.+A.+Bisgroveauthor=T.+Mahmoudiauthor=P.+Henkleinauthor=E.+Verdin&title=Conserved+P-TEFb-interacting+domain+of+BRD4+inhibits+HIV+transcription&doi=10.1073%2Fpnas.0705053104"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Conserved P-TEFb-interacting domain of BRD4 inhibits HIV transcription</span></div><div class="casAuthors">Bisgrove, Dwayne A.; Mahmoudi, Tokameh; Henklein, Peter; Verdin, Eric</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">104</span>
        (<span class="NLM_cas:issue">34</span>),
    <span class="NLM_cas:pages">13690-13695</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">We have identified a conserved region in the C-terminal domain of bromodomain-contg. protein 4 (BRD4) that mediates its specific interaction with pos. transcription elongation factor b (P-TEFb).  This domain is highly conserved in testis-specific bromodomain protein (BRDT) and Drosophila fs(1)h.  Both BRDT and fs(1)h specifically interact with P-TEFb in mammalian cells, and this interaction depends on their C-terminal domains.  Overexpression of the BRD4 P-TEFb-interacting domain disrupts the interaction between the HIV transactivator Tat and P-TEFb and suppresses the ability of Tat to transactivate the HIV promoter.  Incubation of cells with a synthetic peptide contg. the C-terminal domain of BRD4 interferes with transactivation of the HIV promoter by the Tat protein.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq7C_2ZREaOELVg90H21EOLACvtfcHk0liF4cMfGTGlpA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXpvVOjsLc%253D&md5=6091d31044bc49d054c0c4b89f690384</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0705053104&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0705053104%26sid%3Dliteratum%253Aachs%26aulast%3DBisgrove%26aufirst%3DD.%2BA.%26aulast%3DMahmoudi%26aufirst%3DT.%26aulast%3DHenklein%26aufirst%3DP.%26aulast%3DVerdin%26aufirst%3DE.%26atitle%3DConserved%2520P-TEFb-interacting%2520domain%2520of%2520BRD4%2520inhibits%2520HIV%2520transcription%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2007%26volume%3D104%26spage%3D13690%26epage%3D13695%26doi%3D10.1073%2Fpnas.0705053104" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Itzen, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greifenberg, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bosken, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geyer, M.</span></span> <span> </span><span class="NLM_article-title">BRD4 activates P-TEFb for RNA polymerase II CTD phosphorylation</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>42</i></span>,  <span class="NLM_fpage">7577</span>– <span class="NLM_lpage">7590</span>, <span class="refDoi"> DOI: 10.1093/nar/gku449</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1093%2Fnar%2Fgku449" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=24860166" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtFCqsrnE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2014&pages=7577-7590&author=F.+Itzenauthor=A.+K.+Greifenbergauthor=C.+A.+Boskenauthor=M.+Geyer&title=BRD4+activates+P-TEFb+for+RNA+polymerase+II+CTD+phosphorylation&doi=10.1093%2Fnar%2Fgku449"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Brd4 activates P-TEFb for RNA polymerase II CTD phosphorylation</span></div><div class="casAuthors">Itzen, Friederike; Greifenberg, Ann Katrin; Boesken, Christian A.; Geyer, Matthias</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">7577-7590</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Bromodomain protein Brd4 regulates the transcription of signal-inducible genes.  This is achieved by recruiting pos. transcription elongation factor P-TEFb to promoters by its P-TEFb interaction domain (PID).  Here, the authors show that Brd4 stimulates the kinase activity of P-TEFb for phosphorylation of the C-terminal domain (CTD) of RNA polymerase II over basal levels.  The CTD phosphorylation satn. levels, the preferences for pre-phosphorylated substrates, and the phosphorylation specificity for Ser-5 of the CTD however remained unchanged.  Inhibition of P-TEFb by coupling factor Hexim1 was relieved by Brd4, although no mutual displacement with the cyclin T-binding domain of Hexim1 was obsd.  The Brd4 PID showed a surprising sequence motif similarity to the trans-activating Tat protein from HIV-1, which included a core RxL motif, a polybasic cluster known as an arginine-rich motif, and a C-terminal leucine motif.  The mutation of these motifs to alanine significantly diminished the stimulatory effect of Brd4 and fully abrogated its activation potential in the presence of Hexim1.  Yet, the protein was not found to bind cyclin T1 as Tat, but only P-TEFb with a Kd of 0.5 μM.  The data suggested a model where Brd4 acts on the kinase subunit of P-TEFb to relieve inhibition and stimulate substrate recognition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo3Ihalme7shrVg90H21EOLACvtfcHk0liF4cMfGTGlpA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtFCqsrnE&md5=2d2d10fee6b55d4ef43950195344ca79</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgku449&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgku449%26sid%3Dliteratum%253Aachs%26aulast%3DItzen%26aufirst%3DF.%26aulast%3DGreifenberg%26aufirst%3DA.%2BK.%26aulast%3DBosken%26aufirst%3DC.%2BA.%26aulast%3DGeyer%26aufirst%3DM.%26atitle%3DBRD4%2520activates%2520P-TEFb%2520for%2520RNA%2520polymerase%2520II%2520CTD%2520phosphorylation%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2014%26volume%3D42%26spage%3D7577%26epage%3D7590%26doi%3D10.1093%2Fnar%2Fgku449" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sabari, B. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dall’Agnese, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boija, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klein, I. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coffey, E. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shrinivas, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abraham, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hannett, N. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zamudio, A. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manteiga, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, Y. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Day, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schuijers, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vasile, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Malik, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hnisz, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, T. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cisse, I. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roeder, R. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharp, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chakraborty, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, R. A.</span></span> <span> </span><span class="NLM_article-title">Coactivator condensation at super-enhancers links phase separation and gene control</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>361</i></span>,  <span class="NLM_fpage">eaar3958</span>, <span class="refDoi"> DOI: 10.1126/science.aar3958</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1126%2Fscience.aar3958" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=29930091" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=361&publication_year=2018&pages=eaar3958&author=B.+R.+Sabariauthor=A.+Dall%E2%80%99Agneseauthor=A.+Boijaauthor=I.+A.+Kleinauthor=E.+L.+Coffeyauthor=K.+Shrinivasauthor=B.+J.+Abrahamauthor=N.+M.+Hannettauthor=A.+V.+Zamudioauthor=J.+C.+Manteigaauthor=C.+H.+Liauthor=Y.+E.+Guoauthor=D.+S.+Dayauthor=J.+Schuijersauthor=E.+Vasileauthor=S.+Malikauthor=D.+Hniszauthor=T.+I.+Leeauthor=I.+I.+Cisseauthor=R.+G.+Roederauthor=P.+A.+Sharpauthor=A.+K.+Chakrabortyauthor=R.+A.+Young&title=Coactivator+condensation+at+super-enhancers+links+phase+separation+and+gene+control&doi=10.1126%2Fscience.aar3958"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1126%2Fscience.aar3958&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.aar3958%26sid%3Dliteratum%253Aachs%26aulast%3DSabari%26aufirst%3DB.%2BR.%26aulast%3DDall%25E2%2580%2599Agnese%26aufirst%3DA.%26aulast%3DBoija%26aufirst%3DA.%26aulast%3DKlein%26aufirst%3DI.%2BA.%26aulast%3DCoffey%26aufirst%3DE.%2BL.%26aulast%3DShrinivas%26aufirst%3DK.%26aulast%3DAbraham%26aufirst%3DB.%2BJ.%26aulast%3DHannett%26aufirst%3DN.%2BM.%26aulast%3DZamudio%26aufirst%3DA.%2BV.%26aulast%3DManteiga%26aufirst%3DJ.%2BC.%26aulast%3DLi%26aufirst%3DC.%2BH.%26aulast%3DGuo%26aufirst%3DY.%2BE.%26aulast%3DDay%26aufirst%3DD.%2BS.%26aulast%3DSchuijers%26aufirst%3DJ.%26aulast%3DVasile%26aufirst%3DE.%26aulast%3DMalik%26aufirst%3DS.%26aulast%3DHnisz%26aufirst%3DD.%26aulast%3DLee%26aufirst%3DT.%2BI.%26aulast%3DCisse%26aufirst%3DI.%2BI.%26aulast%3DRoeder%26aufirst%3DR.%2BG.%26aulast%3DSharp%26aufirst%3DP.%2BA.%26aulast%3DChakraborty%26aufirst%3DA.%2BK.%26aulast%3DYoung%26aufirst%3DR.%2BA.%26atitle%3DCoactivator%2520condensation%2520at%2520super-enhancers%2520links%2520phase%2520separation%2520and%2520gene%2520control%26jtitle%3DScience%26date%3D2018%26volume%3D361%26spage%3Deaar3958%26doi%3D10.1126%2Fscience.aar3958" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gibson, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doolittle, L. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schneider, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jensen, L. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gamarra, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Henry, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerlich, D. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Redding, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosen, M. K.</span></span> <span> </span><span class="NLM_article-title">Organization of chromatin by intrinsic and regulated phase separation</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>179</i></span>,  <span class="NLM_fpage">470</span>– <span class="NLM_lpage">484</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2019.08.037</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1016%2Fj.cell.2019.08.037" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=31543265" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhvVentbzE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=179&publication_year=2019&pages=470-484&author=B.+A.+Gibsonauthor=L.+K.+Doolittleauthor=M.+Schneiderauthor=L.+E.+Jensenauthor=N.+Gamarraauthor=L.+Henryauthor=D.+W.+Gerlichauthor=S.+Reddingauthor=M.+K.+Rosen&title=Organization+of+chromatin+by+intrinsic+and+regulated+phase+separation&doi=10.1016%2Fj.cell.2019.08.037"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Organization of Chromatin by Intrinsic and Regulated Phase Separation</span></div><div class="casAuthors">Gibson, Bryan A.; Doolittle, Lynda K.; Schneider, Maximillian W. G.; Jensen, Liv E.; Gamarra, Nathan; Henry, Lisa; Gerlich, Daniel W.; Redding, Sy; Rosen, Michael K.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">179</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">470-484.e21</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Eukaryotic chromatin is highly condensed but dynamically accessible to regulation and organized into subdomains.  We demonstrate that reconstituted chromatin undergoes histone tail-driven liq.-liq. phase sepn. (LLPS) in physiol. salt and when microinjected into cell nuclei, producing dense and dynamic droplets.  Linker histone H1 and internucleosome linker lengths shared across eukaryotes promote phase sepn. of chromatin, tune droplet properties, and coordinate to form condensates of consistent d. in manners that parallel chromatin behavior in cells.  Histone acetylation by p300 antagonizes chromatin phase sepn., dissolving droplets in vitro and decreasing droplet formation in nuclei.  In the presence of multi-bromodomain proteins, such as BRD4, highly acetylated chromatin forms a new phase-sepd. state with droplets of distinct phys. properties, which can be immiscible with unmodified chromatin droplets, mimicking nuclear chromatin subdomains.  Our data suggest a framework, based on intrinsic phase sepn. of the chromatin polymer, for understanding the organization and regulation of eukaryotic genomes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGosqITN5BU2ZrVg90H21EOLACvtfcHk0lgMub8HktZhlA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhvVentbzE&md5=27efce722f7a1f0be19097a034e0d200</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2019.08.037&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2019.08.037%26sid%3Dliteratum%253Aachs%26aulast%3DGibson%26aufirst%3DB.%2BA.%26aulast%3DDoolittle%26aufirst%3DL.%2BK.%26aulast%3DSchneider%26aufirst%3DM.%26aulast%3DJensen%26aufirst%3DL.%2BE.%26aulast%3DGamarra%26aufirst%3DN.%26aulast%3DHenry%26aufirst%3DL.%26aulast%3DGerlich%26aufirst%3DD.%2BW.%26aulast%3DRedding%26aufirst%3DS.%26aulast%3DRosen%26aufirst%3DM.%2BK.%26atitle%3DOrganization%2520of%2520chromatin%2520by%2520intrinsic%2520and%2520regulated%2520phase%2520separation%26jtitle%3DCell%26date%3D2019%26volume%3D179%26spage%3D470%26epage%3D484%26doi%3D10.1016%2Fj.cell.2019.08.037" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Han, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gu, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jia, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jaganathan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Babault, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, L.</span></span> <span> </span><span class="NLM_article-title">Roles of the BRD4 short isoform in phase separation and active gene transcription</span>. <i>Nat. Struct. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">333</span>– <span class="NLM_lpage">341</span>, <span class="refDoi"> DOI: 10.1038/s41594-020-0394-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1038%2Fs41594-020-0394-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=32203489" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BB3cXltVSisLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2020&pages=333-341&author=X.+Hanauthor=D.+Yuauthor=R.+Guauthor=Y.+Jiaauthor=Q.+Wangauthor=A.+Jaganathanauthor=X.+Yangauthor=M.+Yuauthor=N.+Babaultauthor=C.+Zhaoauthor=H.+Yiauthor=Q.+Zhangauthor=M.+M.+Zhouauthor=L.+Zeng&title=Roles+of+the+BRD4+short+isoform+in+phase+separation+and+active+gene+transcription&doi=10.1038%2Fs41594-020-0394-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Roles of the BRD4 short isoform in phase separation and active gene transcription</span></div><div class="casAuthors">Han, Xinye; Yu, Di; Gu, Ruirui; Jia, Yanjie; Wang, Qi; Jaganathan, Anbalagan; Yang, Xuelan; Yu, Miaomiao; Babault, Nicolas; Zhao, Chengcheng; Yi, Huanfa; Zhang, Qiang; Zhou, Ming-Ming; Zeng, Lei</div><div class="citationInfo"><span class="NLM_cas:title">Nature Structural & Molecular Biology</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">333-341</span>CODEN:
                <span class="NLM_cas:coden">NSMBCU</span>;
        ISSN:<span class="NLM_cas:issn">1545-9993</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">BRD4, a major tandem-bromodomain-contg. transcription regulator, has two isoforms.  The long isoform (BRD4L) has an extended C terminus that binds transcription cofactors, while the short isoform (BRD4S) lacks this C-terminal extension.  Unlike BRD4L, the role of BRD4S in gene transcription remains unclear.  Here, we report that, in human cancer cells, BRD4S forms nuclear puncta that possess liq.-like properties and that colocalize with BRD4L, MED1 and sites of histone H3 lysine 27 acetylation.  BRD4 puncta are correlated with BRD4S but not BRD4L expression levels.  BRD4S knockdown reduces BRD4S condensation, and ectopic expression promotes puncta formation and target gene transcription.  BRD4S nuclear condensation is mediated by its intrinsically disordered regions and binding of its bromodomains to DNA and acetylated chromatin, resp., and BRD4S phosphorylation diminishes BRD4 condensation.  Our study illuminates a previously unappreciated role of BRD4S in organizing chromatin and transcription factors through phase sepn. to sustain gene transcription in chromatin for cancer cell proliferation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoePsB81fBjY7Vg90H21EOLACvtfcHk0lgMub8HktZhlA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXltVSisLw%253D&md5=fcd3cd7541a8c2a60ee84a57a19ab62d</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1038%2Fs41594-020-0394-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41594-020-0394-8%26sid%3Dliteratum%253Aachs%26aulast%3DHan%26aufirst%3DX.%26aulast%3DYu%26aufirst%3DD.%26aulast%3DGu%26aufirst%3DR.%26aulast%3DJia%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DQ.%26aulast%3DJaganathan%26aufirst%3DA.%26aulast%3DYang%26aufirst%3DX.%26aulast%3DYu%26aufirst%3DM.%26aulast%3DBabault%26aufirst%3DN.%26aulast%3DZhao%26aufirst%3DC.%26aulast%3DYi%26aufirst%3DH.%26aulast%3DZhang%26aufirst%3DQ.%26aulast%3DZhou%26aufirst%3DM.%2BM.%26aulast%3DZeng%26aufirst%3DL.%26atitle%3DRoles%2520of%2520the%2520BRD4%2520short%2520isoform%2520in%2520phase%2520separation%2520and%2520active%2520gene%2520transcription%26jtitle%3DNat.%2520Struct.%2520Mol.%2520Biol.%26date%3D2020%26volume%3D27%26spage%3D333%26epage%3D341%26doi%3D10.1038%2Fs41594-020-0394-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sakurai, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Inamochi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Inoue, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hariya, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kawamura, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamada, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dey, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nishiyama, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kubota, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ozato, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goda, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mochizuki, K.</span></span> <span> </span><span class="NLM_article-title">BRD4 regulates adiponectin gene induction by recruiting the P-TEFb complex to the transcribed region of the gene</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">11962</span>, <span class="refDoi"> DOI: 10.1038/s41598-017-12342-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1038%2Fs41598-017-12342-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=28931940" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A280%3ADC%252BC1M%252FgslSqtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2017&pages=11962&author=N.+Sakuraiauthor=Y.+Inamochiauthor=T.+Inoueauthor=N.+Hariyaauthor=M.+Kawamuraauthor=M.+Yamadaauthor=A.+Deyauthor=A.+Nishiyamaauthor=T.+Kubotaauthor=K.+Ozatoauthor=T.+Godaauthor=K.+Mochizuki&title=BRD4+regulates+adiponectin+gene+induction+by+recruiting+the+P-TEFb+complex+to+the+transcribed+region+of+the+gene&doi=10.1038%2Fs41598-017-12342-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">BRD4 regulates adiponectin gene induction by recruiting the P-TEFb complex to the transcribed region of the gene</span></div><div class="casAuthors">Sakurai Naoko; Inamochi Yuko; Yamada Masami; Goda Toshinao; Mochizuki Kazuki; Inoue Takuya; Hariya Natsuyo; Kawamura Musashi; Mochizuki Kazuki; Dey Anup; Nishiyama Akira; Ozato Keiko; Kubota Takeo</div><div class="citationInfo"><span class="NLM_cas:title">Scientific reports</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">11962</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">We previously reported that induction of the adipocyte-specific gene adiponectin (Adipoq) during 3T3-L1 adipocyte differentiation is closely associated with epigenetic memory histone H3 acetylation on the transcribed region of the gene.  We used 3T3-L1 adipocytes and Brd4 heterozygous mice to investigate whether the induction of Adipoq during adipocyte differentiation is regulated by histone acetylation and the binding protein bromodomain containing 4 (BRD4) on the transcribed region.  Depletion of BRD4 by shRNA and inhibition by (+)-JQ1, an inhibitor of BET family proteins including BRD4, reduced Adipoq expression and lipid droplet accumulation in 3T3-L1 adipocytes.  Additionally, the depletion and inhibition of BRD4 reduced the expression of many insulin sensitivity-related genes, including genes related to lipid droplet accumulation in adipocytes.  BRD4 depletion reduced P-TEFb recruitment and histone acetylation on the transcribed region of the Adipoq gene.  The expression levels of Adipoq and fatty acid synthesis-related genes and the circulating ADIPOQ protein level were lower in Brd4 heterozygous mice than in wild-type mice at 21 days after birth.  These findings indicate that BRD4 regulates the Adipoq gene by recruiting P-TEFb onto acetylated histones in the transcribed region of the gene and regulates adipocyte differentiation by regulating the expression of genes related to insulin sensitivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcR6_YYS8G5ZW0cLvDNbOD4DfW6udTcc2eYhmK_gjd4pTLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1M%252FgslSqtA%253D%253D&md5=b5db7bd76c71c96e00dc12496816b3f6</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1038%2Fs41598-017-12342-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41598-017-12342-2%26sid%3Dliteratum%253Aachs%26aulast%3DSakurai%26aufirst%3DN.%26aulast%3DInamochi%26aufirst%3DY.%26aulast%3DInoue%26aufirst%3DT.%26aulast%3DHariya%26aufirst%3DN.%26aulast%3DKawamura%26aufirst%3DM.%26aulast%3DYamada%26aufirst%3DM.%26aulast%3DDey%26aufirst%3DA.%26aulast%3DNishiyama%26aufirst%3DA.%26aulast%3DKubota%26aufirst%3DT.%26aulast%3DOzato%26aufirst%3DK.%26aulast%3DGoda%26aufirst%3DT.%26aulast%3DMochizuki%26aufirst%3DK.%26atitle%3DBRD4%2520regulates%2520adiponectin%2520gene%2520induction%2520by%2520recruiting%2520the%2520P-TEFb%2520complex%2520to%2520the%2520transcribed%2520region%2520of%2520the%2520gene%26jtitle%3DSci.%2520Rep.%26date%3D2017%26volume%3D7%26spage%3D11962%26doi%3D10.1038%2Fs41598-017-12342-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sabo, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amati, B.</span></span> <span> </span><span class="NLM_article-title">BRD4 and MYC-clarifying regulatory specificity</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>360</i></span>,  <span class="NLM_fpage">713</span>– <span class="NLM_lpage">714</span>, <span class="refDoi"> DOI: 10.1126/science.aat6664</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1126%2Fscience.aat6664" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=29773735" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtVCmur7K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=360&publication_year=2018&pages=713-714&author=A.+Saboauthor=B.+Amati&title=BRD4+and+MYC-clarifying+regulatory+specificity&doi=10.1126%2Fscience.aat6664"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">BRD4 and MYC-clarifying regulatory specificity</span></div><div class="casAuthors">Sabo, Arianna; Amati, Bruno</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">360</span>
        (<span class="NLM_cas:issue">6390</span>),
    <span class="NLM_cas:pages">713-714</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr7ZGDU1lOAM7Vg90H21EOLACvtfcHk0ljUJfaZ6zx2WA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtVCmur7K&md5=fe1fc2f0d68a8231dd9ff02224071671</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1126%2Fscience.aat6664&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.aat6664%26sid%3Dliteratum%253Aachs%26aulast%3DSabo%26aufirst%3DA.%26aulast%3DAmati%26aufirst%3DB.%26atitle%3DBRD4%2520and%2520MYC-clarifying%2520regulatory%2520specificity%26jtitle%3DScience%26date%3D2018%26volume%3D360%26spage%3D713%26epage%3D714%26doi%3D10.1126%2Fscience.aat6664" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Devaiah, B. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Akman, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uppal, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weissman, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baranello, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nie, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Levens, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singer, D. S.</span></span> <span> </span><span class="NLM_article-title">MYC protein stability is negatively regulated by BRD4</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>117</i></span>,  <span class="NLM_fpage">13457</span>– <span class="NLM_lpage">13467</span>, <span class="refDoi"> DOI: 10.1073/pnas.1919507117</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1073%2Fpnas.1919507117" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=32482868" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhtF2jtLvF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=117&publication_year=2020&pages=13457-13467&author=B.+N.+Devaiahauthor=J.+Muauthor=B.+Akmanauthor=S.+Uppalauthor=J.+D.+Weissmanauthor=D.+Chengauthor=L.+Baranelloauthor=Z.+Nieauthor=D.+Levensauthor=D.+S.+Singer&title=MYC+protein+stability+is+negatively+regulated+by+BRD4&doi=10.1073%2Fpnas.1919507117"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">MYC protein stability is negatively regulated by BRD4</span></div><div class="casAuthors">Devaiah, Ballachanda N.; Mu, Jie; Akman, Ben; Uppal, Sheetal; Weissman, Jocelyn D.; Cheng, Dan; Baranello, Laura; Nie, Zuqin; Levens, David; Singer, Dinah S.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">117</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">13457-13467</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The protooncogene MYC regulates a variety of cellular processes, including proliferation and metab.  Maintaining MYC at homeostatic levels is crit. to normal cell function; overexpression drives many cancers.  MYC stability is regulated through phosphorylation: phosphorylation at Thr58 signals degrdn. while Ser62 phosphorylation leads to its stabilization and functional activation.  The bromodomain protein 4 (BRD4) is a transcriptional and epigenetic regulator with intrinsic kinase and histone acetyltransferase (HAT) activities that activates transcription of key protooncogenes, including MYC.  We report that BRD4 phosphorylates MYC at Thr58, leading to MYC ubiquitination and degrdn., thereby regulating MYC target genes.  Importantly, BRD4 degrdn., but not inhibition, results in increased levels of MYC protein.  Conversely, MYC inhibits BRD4's HAT activity, suggesting that MYC regulates its own transcription by limiting BRD4-mediated chromatin remodeling of its locus.  The MYC stabilizing kinase, ERK1, regulates MYC levels directly and indirectly by inhibiting BRD4 kinase activity.  These findings demonstrate that BRD4 neg. regulates MYC levels, which is counteracted by ERK1 activation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqtwATxk8kt87Vg90H21EOLACvtfcHk0ljUJfaZ6zx2WA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhtF2jtLvF&md5=77dec456bd0a0507e76d054f547c0430</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1919507117&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1919507117%26sid%3Dliteratum%253Aachs%26aulast%3DDevaiah%26aufirst%3DB.%2BN.%26aulast%3DMu%26aufirst%3DJ.%26aulast%3DAkman%26aufirst%3DB.%26aulast%3DUppal%26aufirst%3DS.%26aulast%3DWeissman%26aufirst%3DJ.%2BD.%26aulast%3DCheng%26aufirst%3DD.%26aulast%3DBaranello%26aufirst%3DL.%26aulast%3DNie%26aufirst%3DZ.%26aulast%3DLevens%26aufirst%3DD.%26aulast%3DSinger%26aufirst%3DD.%2BS.%26atitle%3DMYC%2520protein%2520stability%2520is%2520negatively%2520regulated%2520by%2520BRD4%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2020%26volume%3D117%26spage%3D13457%26epage%3D13467%26doi%3D10.1073%2Fpnas.1919507117" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rahnamoun, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramsey, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Komives, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lauberth, S. M.</span></span> <span> </span><span class="NLM_article-title">RNAs interact with BRD4 to promote enhanced chromatin engagement and transcription activation</span>. <i>Nat. Struct. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">687</span>– <span class="NLM_lpage">697</span>, <span class="refDoi"> DOI: 10.1038/s41594-018-0102-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1038%2Fs41594-018-0102-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=30076409" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhsVGjtr7O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2018&pages=687-697&author=H.+Rahnamounauthor=J.+Leeauthor=Z.+Sunauthor=H.+Luauthor=K.+M.+Ramseyauthor=E.+A.+Komivesauthor=S.+M.+Lauberth&title=RNAs+interact+with+BRD4+to+promote+enhanced+chromatin+engagement+and+transcription+activation&doi=10.1038%2Fs41594-018-0102-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">RNAs interact with BRD4 to promote enhanced chromatin engagement and transcription activation</span></div><div class="casAuthors">Rahnamoun, Homa; Lee, Jihoon; Sun, Zhengxi; Lu, Hanbin; Ramsey, Kristen M.; Komives, Elizabeth A.; Lauberth, Shannon M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Structural & Molecular Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">687-697</span>CODEN:
                <span class="NLM_cas:coden">NSMBCU</span>;
        ISSN:<span class="NLM_cas:issn">1545-9993</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">The bromodomain and extra-terminal motif (BET) protein BRD4 binds to acetylated histones at enhancers and promoters via its bromodomains (BDs) to regulate transcriptional elongation.  In human colorectal cancer cells, we found that BRD4 was recruited to enhancers that were co-occupied by mutant p53 and supported the synthesis of enhancer-directed transcripts (eRNAs) in response to chronic immune signaling.  BRD4 selectively assocd. with eRNAs that were produced from BRD4-bound enhancers.  Using biochem. and biophys. methods, we found that BRD4 BDs function cooperatively as docking sites for eRNAs and that the BDs of BRD2, BRD3, BRDT, BRG1, and BRD7 directly interact with eRNAs.  BRD4-eRNA interactions increased BRD4 binding to acetylated histones in vitro and augmented BRD4 enhancer recruitment and transcriptional cofactor activities.  Our results suggest a mechanism by which eRNAs are directly involved in gene regulation by modulating enhancer interactions and transcriptional functions of BRD4.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGriyuXqZrKRy7Vg90H21EOLACvtfcHk0lgYcMhSLulcrg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhsVGjtr7O&md5=c3470d4854f5c90e7e770d3dbe7cd5d8</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1038%2Fs41594-018-0102-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41594-018-0102-0%26sid%3Dliteratum%253Aachs%26aulast%3DRahnamoun%26aufirst%3DH.%26aulast%3DLee%26aufirst%3DJ.%26aulast%3DSun%26aufirst%3DZ.%26aulast%3DLu%26aufirst%3DH.%26aulast%3DRamsey%26aufirst%3DK.%2BM.%26aulast%3DKomives%26aufirst%3DE.%2BA.%26aulast%3DLauberth%26aufirst%3DS.%2BM.%26atitle%3DRNAs%2520interact%2520with%2520BRD4%2520to%2520promote%2520enhanced%2520chromatin%2520engagement%2520and%2520transcription%2520activation%26jtitle%3DNat.%2520Struct.%2520Mol.%2520Biol.%26date%3D2018%26volume%3D25%26spage%3D687%26epage%3D697%26doi%3D10.1038%2Fs41594-018-0102-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ozato, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L. F.</span></span> <span> </span><span class="NLM_article-title">BRD4 coactivates transcriptional activation of NF-kappaB via specific binding to acetylated RelA</span>. <i>Mol. Cell. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">1375</span>– <span class="NLM_lpage">1387</span>, <span class="refDoi"> DOI: 10.1128/MCB.01365-08</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1128%2FMCB.01365-08" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=19103749" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BD1MXitlylsrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2009&pages=1375-1387&author=B.+Huangauthor=X.+D.+Yangauthor=M.+M.+Zhouauthor=K.+Ozatoauthor=L.+F.+Chen&title=BRD4+coactivates+transcriptional+activation+of+NF-kappaB+via+specific+binding+to+acetylated+RelA&doi=10.1128%2FMCB.01365-08"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Brd4 coactivates transcriptional activation of NF-κB via specific binding to acetylated RelA</span></div><div class="casAuthors">Huang, Bo; Yang, Xiao-Dong; Zhou, Ming-Ming; Ozato, Keiko; Chen, Lin-Feng</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Biology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1375-1387</span>CODEN:
                <span class="NLM_cas:coden">MCEBD4</span>;
        ISSN:<span class="NLM_cas:issn">0270-7306</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Acetylation of the RelA subunit of NF-κB, esp. at lysine-310, is crit. for the transcriptional activation of NF-κB and the expression of inflammatory genes.  In this study, we demonstrate that bromodomains of Brd4 bind to acetylated lysine-310.  Brd4 enhances transcriptional activation of NF-κB and the expression of a subset of NF-κB-responsive inflammatory genes in an acetylated lysine-310-dependent manner.  Bromodomains of Brd4 and acetylated lysine-310 of RelA are both required for the mutual interaction and coactivation function of Brd4.  Finally, we demonstrate that Brd4 further recruits CDK9 to phosphorylate C-terminal domain of RNA polymerase II and facilitate the transcription of NF-κB-dependent inflammatory genes.  Our results identify Brd4 as a novel coactivator of NF-κB through specifically binding to acetylated lysine-310 of RelA.  In addn., these studies reveal a mechanism by which acetylated RelA stimulates the transcriptional activity of NF-κB and the NF-κB-dependent inflammatory response.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoBnS4p5wzT2LVg90H21EOLACvtfcHk0lgYcMhSLulcrg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXitlylsrc%253D&md5=ae4e16c84d17c5e0e268ff80703a4608</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1128%2FMCB.01365-08&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMCB.01365-08%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DB.%26aulast%3DYang%26aufirst%3DX.%2BD.%26aulast%3DZhou%26aufirst%3DM.%2BM.%26aulast%3DOzato%26aufirst%3DK.%26aulast%3DChen%26aufirst%3DL.%2BF.%26atitle%3DBRD4%2520coactivates%2520transcriptional%2520activation%2520of%2520NF-kappaB%2520via%2520specific%2520binding%2520to%2520acetylated%2520RelA%26jtitle%3DMol.%2520Cell.%2520Biol.%26date%3D2009%26volume%3D29%26spage%3D1375%26epage%3D1387%26doi%3D10.1128%2FMCB.01365-08" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tao, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rusinova, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, Y. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evers, B. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, B. P.</span></span> <span> </span><span class="NLM_article-title">Disrupting the interaction of BRD4 with diacetylated Twist suppresses tumorigenesis in basal-like breast cancer</span>. <i>Cancer Cell</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">210</span>– <span class="NLM_lpage">225</span>, <span class="refDoi"> DOI: 10.1016/j.ccr.2014.01.028</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1016%2Fj.ccr.2014.01.028" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=24525235" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BC2cXitlCisrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2014&pages=210-225&author=J.+Shiauthor=Y.+Wangauthor=L.+Zengauthor=Y.+Wuauthor=J.+Dengauthor=Q.+Zhangauthor=Y.+Linauthor=J.+Liauthor=T.+Kangauthor=M.+Taoauthor=E.+Rusinovaauthor=G.+Zhangauthor=C.+Wangauthor=H.+Zhuauthor=J.+Yaoauthor=Y.+X.+Zengauthor=B.+M.+Eversauthor=M.+M.+Zhouauthor=B.+P.+Zhou&title=Disrupting+the+interaction+of+BRD4+with+diacetylated+Twist+suppresses+tumorigenesis+in+basal-like+breast+cancer&doi=10.1016%2Fj.ccr.2014.01.028"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Disrupting the Interaction of BRD4 with Diacetylated Twist Suppresses Tumorigenesis in Basal-like Breast Cancer</span></div><div class="casAuthors">Shi, Jian; Wang, Yifan; Zeng, Lei; Wu, Yadi; Deng, Jiong; Zhang, Qiang; Lin, Yiwei; Li, Junlin; Kang, Tiebang; Tao, Min; Rusinova, Elena; Zhang, Guangtao; Wang, Chi; Zhu, Haining; Yao, Jun; Zeng, Yi-Xin; Evers, B. Mark; Zhou, Ming-Ming; Zhou, Binhua P.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">210-225</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Twist is a key transcription activator of epithelial-mesenchymal transition (EMT).  It remains unclear how Twist induces gene expression.  Here we report a mechanism by which Twist recruits BRD4 to direct WNT5A expression in basal-like breast cancer (BLBC).  Twist contains a "histone H4-mimic" GK-X-GK motif that is diacetylated by Tip60.  The diacetylated Twist binds the second bromodomain of BRD4, whose first bromodomain interacts with acetylated H4, thereby constructing an activated Twist/BRD4/P-TEFb/RNA-Pol II complex at the WNT5A promoter and enhancer.  Pharmacol. inhibition of the Twist-BRD4 assocn. reduced WNT5A expression and suppressed invasion, cancer stem cell (CSC)-like properties, and tumorigenicity of BLBC cells.  Our study indicates that the interaction with BRD4 is crit. for the oncogenic function of Twist in BLBC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGovIzriI8A6ZrVg90H21EOLACvtfcHk0lgYcMhSLulcrg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXitlCisrs%253D&md5=d207cbc1abb187a53d807452b4f10d8f</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1016%2Fj.ccr.2014.01.028&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccr.2014.01.028%26sid%3Dliteratum%253Aachs%26aulast%3DShi%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DZeng%26aufirst%3DL.%26aulast%3DWu%26aufirst%3DY.%26aulast%3DDeng%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DQ.%26aulast%3DLin%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DKang%26aufirst%3DT.%26aulast%3DTao%26aufirst%3DM.%26aulast%3DRusinova%26aufirst%3DE.%26aulast%3DZhang%26aufirst%3DG.%26aulast%3DWang%26aufirst%3DC.%26aulast%3DZhu%26aufirst%3DH.%26aulast%3DYao%26aufirst%3DJ.%26aulast%3DZeng%26aufirst%3DY.%2BX.%26aulast%3DEvers%26aufirst%3DB.%2BM.%26aulast%3DZhou%26aufirst%3DM.%2BM.%26aulast%3DZhou%26aufirst%3DB.%2BP.%26atitle%3DDisrupting%2520the%2520interaction%2520of%2520BRD4%2520with%2520diacetylated%2520Twist%2520suppresses%2520tumorigenesis%2520in%2520basal-like%2520breast%2520cancer%26jtitle%3DCancer%2520Cell%26date%3D2014%26volume%3D25%26spage%3D210%26epage%3D225%26doi%3D10.1016%2Fj.ccr.2014.01.028" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tian, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Su, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">An, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Du, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shan, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiang, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H.</span></span> <span> </span><span class="NLM_article-title">Inhibition of BRD4 suppresses the malignancy of breast cancer cells via regulation of Snail</span>. <i>Cell Death Differ.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">255</span>– <span class="NLM_lpage">268</span>, <span class="refDoi"> DOI: 10.1038/s41418-019-0353-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1038%2Fs41418-019-0353-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=31114028" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhtFSmsbnF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2020&pages=255-268&author=L.+Luauthor=Z.+Chenauthor=X.+Linauthor=L.+Tianauthor=Q.+Suauthor=P.+Anauthor=W.+Liauthor=Y.+Wuauthor=J.+Duauthor=H.+Shanauthor=C.+M.+Chiangauthor=H.+Wang&title=Inhibition+of+BRD4+suppresses+the+malignancy+of+breast+cancer+cells+via+regulation+of+Snail&doi=10.1038%2Fs41418-019-0353-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of BRD4 suppresses the malignancy of breast cancer cells via regulation of Snail</span></div><div class="casAuthors">Lu, Linlin; Chen, Zhuojia; Lin, Xinyao; Tian, Lin; Su, Qiao; An, Panpan; Li, Wuguo; Wu, Yingmin; Du, Jun; Shan, Hong; Chiang, Cheng-Ming; Wang, Hongsheng</div><div class="citationInfo"><span class="NLM_cas:title">Cell Death & Differentiation</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">255-268</span>CODEN:
                <span class="NLM_cas:coden">CDDIEK</span>;
        ISSN:<span class="NLM_cas:issn">1350-9047</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">The mechanistic action of bromodomain-contg. protein 4 (BRD4) in cancer motility, including epithelial-mesenchymal transition (EMT), remains largely undefined.  We found that targeted inhibition of BRD4 reduces migration, invasion, in vivo growth of patient-derived xenograft (PDX), and lung colonization of breast cancer (BC) cells.  Inhibition of BRD4 rapidly decreases the expression of Snail, a powerful EMT transcription factor (EMT-TF), via diminishing its protein stability and transcription.  Protein kinase D1 (PRKD1) is responsible for BRD4-regulated Snail protein stability by triggering phosphorylation at Ser11 of Snail and then inducing proteasome-mediated degrdn.  BRD4 inhibition also suppresses the expression of Gli1, a key transductor of Hedgehog (Hh) required to activate the transcription of SNAI1, in BC cells.  The GACCACC sequence (-341 to -333) in the SNAI1 promoter is responsible for Gli1-induced transcription of SNAI1.  Clin., BRD4 and Snail levels are increased in lung-metastasized, estrogen receptor-neg. (ER-), and progesterone receptor-neg. (PR-) breast cancers and correlate with the expression of mesenchymal markers.  Collectively, BRD4 can regulate malignancy of breast cancer cells via both transcriptional and post-translational regulation of Snail.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrhdNJ6b7jcFbVg90H21EOLACvtfcHk0livZoBZcSF9Zg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhtFSmsbnF&md5=9bee526409f844f81cfaaadce9779da2</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1038%2Fs41418-019-0353-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41418-019-0353-2%26sid%3Dliteratum%253Aachs%26aulast%3DLu%26aufirst%3DL.%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DLin%26aufirst%3DX.%26aulast%3DTian%26aufirst%3DL.%26aulast%3DSu%26aufirst%3DQ.%26aulast%3DAn%26aufirst%3DP.%26aulast%3DLi%26aufirst%3DW.%26aulast%3DWu%26aufirst%3DY.%26aulast%3DDu%26aufirst%3DJ.%26aulast%3DShan%26aufirst%3DH.%26aulast%3DChiang%26aufirst%3DC.%2BM.%26aulast%3DWang%26aufirst%3DH.%26atitle%3DInhibition%2520of%2520BRD4%2520suppresses%2520the%2520malignancy%2520of%2520breast%2520cancer%2520cells%2520via%2520regulation%2520of%2520Snail%26jtitle%3DCell%2520Death%2520Differ.%26date%3D2020%26volume%3D27%26spage%3D255%26epage%3D268%26doi%3D10.1038%2Fs41418-019-0353-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zanconato, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Battilana, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Forcato, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Filippi, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Azzolin, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manfrin, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quaranta, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Di Biagio, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sigismondo, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guzzardo, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lejeune, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haendler, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krijgsveld, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fassan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bicciato, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cordenonsi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Piccolo, S.</span></span> <span> </span><span class="NLM_article-title">Transcriptional addiction in cancer cells is mediated by YAP/TAZ through BRD4</span>. <i>Nat. Med.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">1599</span>– <span class="NLM_lpage">1610</span>, <span class="refDoi"> DOI: 10.1038/s41591-018-0158-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1038%2Fs41591-018-0158-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=30224758" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhslCjtrbN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2018&pages=1599-1610&author=F.+Zanconatoauthor=G.+Battilanaauthor=M.+Forcatoauthor=L.+Filippiauthor=L.+Azzolinauthor=A.+Manfrinauthor=E.+Quarantaauthor=D.+Di+Biagioauthor=G.+Sigismondoauthor=V.+Guzzardoauthor=P.+Lejeuneauthor=B.+Haendlerauthor=J.+Krijgsveldauthor=M.+Fassanauthor=S.+Bicciatoauthor=M.+Cordenonsiauthor=S.+Piccolo&title=Transcriptional+addiction+in+cancer+cells+is+mediated+by+YAP%2FTAZ+through+BRD4&doi=10.1038%2Fs41591-018-0158-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Transcriptional addiction in cancer cells is mediated by YAP/TAZ through BRD4</span></div><div class="casAuthors">Zanconato, Francesca; Battilana, Giusy; Forcato, Mattia; Filippi, Letizia; Azzolin, Luca; Manfrin, Andrea; Quaranta, Erika; Di Biagio, Daniele; Sigismondo, Gianluca; Guzzardo, Vincenza; Lejeune, Pascale; Haendler, Bernard; Krijgsveld, Jeroen; Fassan, Matteo; Bicciato, Silvio; Cordenonsi, Michelangelo; Piccolo, Stefano</div><div class="citationInfo"><span class="NLM_cas:title">Nature Medicine (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1599-1610</span>CODEN:
                <span class="NLM_cas:coden">NAMEFI</span>;
        ISSN:<span class="NLM_cas:issn">1078-8956</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Cancer cells rely on dysregulated gene expression.  This establishes specific transcriptional addictions that may be therapeutically exploited.  Yet, the mechanisms that are ultimately responsible for these addictions are poorly understood.  Here, we investigated the transcriptional dependencies of transformed cells to the transcription factors YAP and TAZ.  YAP/TAZ phys. engage the general coactivator bromodomain-contg. protein 4 (BRD4), dictating the genome-wide assocn. of BRD4 to chromatin.  YAP/TAZ flag a large set of enhancers with super-enhancer-like functional properties.  YAP/TAZ-bound enhancers mediate the recruitment of BRD4 and RNA polymerase II at YAP/TAZ-regulated promoters, boosting the expression of a host of growth-regulating genes.  Treatment with small-mol. inhibitors of BRD4 blunts YAP/TAZ pro-tumorigenic activity in several cell or tissue contexts, causes the regression of pre-established, YAP/TAZ-addicted neoplastic lesions and reverts drug resistance.  This work sheds light on essential mediators, mechanisms and genome-wide regulatory elements that are responsible for transcriptional addiction in cancer and lays the groundwork for a rational use of BET inhibitors according to YAP/TAZ biol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqqz-uXLracArVg90H21EOLACvtfcHk0livZoBZcSF9Zg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhslCjtrbN&md5=131d262f43c61023f6b4f6a6d99a94ea</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1038%2Fs41591-018-0158-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41591-018-0158-8%26sid%3Dliteratum%253Aachs%26aulast%3DZanconato%26aufirst%3DF.%26aulast%3DBattilana%26aufirst%3DG.%26aulast%3DForcato%26aufirst%3DM.%26aulast%3DFilippi%26aufirst%3DL.%26aulast%3DAzzolin%26aufirst%3DL.%26aulast%3DManfrin%26aufirst%3DA.%26aulast%3DQuaranta%26aufirst%3DE.%26aulast%3DDi%2BBiagio%26aufirst%3DD.%26aulast%3DSigismondo%26aufirst%3DG.%26aulast%3DGuzzardo%26aufirst%3DV.%26aulast%3DLejeune%26aufirst%3DP.%26aulast%3DHaendler%26aufirst%3DB.%26aulast%3DKrijgsveld%26aufirst%3DJ.%26aulast%3DFassan%26aufirst%3DM.%26aulast%3DBicciato%26aufirst%3DS.%26aulast%3DCordenonsi%26aufirst%3DM.%26aulast%3DPiccolo%26aufirst%3DS.%26atitle%3DTranscriptional%2520addiction%2520in%2520cancer%2520cells%2520is%2520mediated%2520by%2520YAP%252FTAZ%2520through%2520BRD4%26jtitle%3DNat.%2520Med.%26date%3D2018%26volume%3D24%26spage%3D1599%26epage%3D1610%26doi%3D10.1038%2Fs41591-018-0158-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Song, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pool Pizzi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, A. W</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huo, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhutani, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weston, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shanbhag, N. D</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ajani, J. A.</span></span> <span> </span><span class="NLM_article-title">Targeting Hippo coactivator YAP1 through BET bromodomain inhibition in esophageal adenocarcinoma</span>. <i>Mol. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">1410</span>– <span class="NLM_lpage">1426</span>, <span class="refDoi"> DOI: 10.1002/1878-0261.12667</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1002%2F1878-0261.12667" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=32175692" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhsleltbnJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2020&pages=1410-1426&author=S.+Songauthor=Y.+Liauthor=Y.+Xuauthor=L.+Maauthor=M.+Pool+Pizziauthor=J.+Jinauthor=A.+W+Scottauthor=L.+Huoauthor=Y.+Wangauthor=J.+H.+Leeauthor=M.+S.+Bhutaniauthor=B.+Westonauthor=N.+D+Shanbhagauthor=R.+L.+Johnsonauthor=J.+A.+Ajani&title=Targeting+Hippo+coactivator+YAP1+through+BET+bromodomain+inhibition+in+esophageal+adenocarcinoma&doi=10.1002%2F1878-0261.12667"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting Hippo coactivator YAP1 through BET bromodomain inhibition in esophageal adenocarcinoma</span></div><div class="casAuthors">Song, Shumei; Li, Yuan; Xu, Yan; Ma, Lang; Pool Pizzi, Melissa; Jin, Jiankang; Scott, Ailing W.; Huo, Longfei; Wang, Ying; Lee, Jeffrey H.; Bhutani, Manoop S.; Weston, Brian; Shanbhag, Namita D.; Johnson, Randy L.; Ajani, Jaffer A.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Oncology</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1410-1426</span>CODEN:
                <span class="NLM_cas:coden">MOONC3</span>;
        ISSN:<span class="NLM_cas:issn">1878-0261</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons Ltd.</span>)
        </div><div class="casAbstract">Hippo/YAP1 signaling is a major regulator of organ size, cancer stemness, and aggressive phenotype.  Thus, targeting YAP1 may provide a novel therapeutic strategy for tumors with high YAP1 expression in esophageal cancer (EC).  Chromatin immunopptn. (ChiP) and quant. ChiP-PCR were used to det. the regulation of the chromatin remodeling protein bromodomain-contg. protein 4 (BRD4) on YAP1.  The role of the bromodomain and extraterminal motif (BET) inhibitor JQ1, an established BRD4 inhibitor, on inhibition of YAP1 in EC cells was dissected using western blot, immunofluorescence, qPCR, and transient transfection.  The antitumor activities of BET inhibitor were further examd. by variety of functional assays, cell proliferation (MTS), tumorsphere, and ALDH1+ labeling in vitro and in vivo.  Here, we show that BRD4 regulates YAP1 expression and transcription.  ChiP assays revealed that BRD4 directly occupies YAP1 promoter and that JQ1 robustly blocks BRD4 binding to the YAP1 promoter.  Consequently, JQ1 strongly suppresses constitutive or induced YAP1 expression and transcription in EC cells and YAP1/Tead downstream transcriptional activity.  Intriguingly, radiation-resistant cells that acquire strong cancer stem cell traits and an aggressive phenotype can be effectively suppressed by JQ1 as assessed by cell proliferation, tumorsphere formation, and redn. in the ALDH1+ cells.  Moreover, effects of JQ1 are synergistically amplified by the addn. of docetaxel in vitro and in vivo.  Our results demonstrate that BRD4 is a crit. regulator of Hippo/YAP1 signaling and that BRD4 inhibitor JQ1 represents a new class of inhibitor of Hippo/YAP1 signaling, primarily targeting YAP1 high and therapy-resistant cancer cells enriched with cancer stem cell properties.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpOA8KqDxe2ALVg90H21EOLACvtfcHk0livZoBZcSF9Zg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhsleltbnJ&md5=04d25857ce6910c64c99092a3e8b4c5f</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1002%2F1878-0261.12667&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F1878-0261.12667%26sid%3Dliteratum%253Aachs%26aulast%3DSong%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DXu%26aufirst%3DY.%26aulast%3DMa%26aufirst%3DL.%26aulast%3DPool%2BPizzi%26aufirst%3DM.%26aulast%3DJin%26aufirst%3DJ.%26aulast%3DScott%26aufirst%3DA.%2BW%26aulast%3DHuo%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DLee%26aufirst%3DJ.%2BH.%26aulast%3DBhutani%26aufirst%3DM.%2BS.%26aulast%3DWeston%26aufirst%3DB.%26aulast%3DShanbhag%26aufirst%3DN.%2BD%26aulast%3DJohnson%26aufirst%3DR.%2BL.%26aulast%3DAjani%26aufirst%3DJ.%2BA.%26atitle%3DTargeting%2520Hippo%2520coactivator%2520YAP1%2520through%2520BET%2520bromodomain%2520inhibition%2520in%2520esophageal%2520adenocarcinoma%26jtitle%3DMol.%2520Oncol.%26date%3D2020%26volume%3D14%26spage%3D1410%26epage%3D1426%26doi%3D10.1002%2F1878-0261.12667" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chiang, C. M.</span></span> <span> </span><span class="NLM_article-title">BRD4 engagement from chromatin targeting to transcriptional regulation: selective contact with acetylated histone H3 and H4</span>. <i>F1000 Biol. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">98</span>, <span class="refDoi"> DOI: 10.3410/B1-98</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.3410%2FB1-98" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=20495683" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A280%3ADC%252BC3MrlvVygtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2009&pages=98&author=C.+M.+Chiang&title=BRD4+engagement+from+chromatin+targeting+to+transcriptional+regulation%3A+selective+contact+with+acetylated+histone+H3+and+H4&doi=10.3410%2FB1-98"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Brd4 engagement from chromatin targeting to transcriptional regulation: selective contact with acetylated histone H3 and H4</span></div><div class="casAuthors">Chiang Cheng-Ming</div><div class="citationInfo"><span class="NLM_cas:title">F1000 biology reports</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">98</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Bromodomain-containing protein 4 (Brd4) contains two tandem bromodomains (BD1 and BD2) that bind preferentially to acetylated lysine residues found in histones and nonhistone proteins.  This molecular recognition allows Brd4 to associate with acetylated chromatin throughout the cell cycle and regulates transcription at targeted loci.  Recruitment of positive transcription elongation factor b, and possibly the general initiation cofactor Mediator as well, plays an important role in Brd4-regulated transcription.  Selective contacts with acetyl-lysines in nucleosomal histones and chromatin-binding factors likely provide a molecular switch modulating the steps from chromatin targeting to transcriptional regulation, thus further expanding the 'acetylation code' for combinatorial regulation in eukaryotes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcR81nP81pAX6sKL0FlyGHDofW6udTcc2eaevPybUKAx7Lntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3MrlvVygtQ%253D%253D&md5=d575a30cf8b3d918367d6580c2c0d64a</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.3410%2FB1-98&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3410%252FB1-98%26sid%3Dliteratum%253Aachs%26aulast%3DChiang%26aufirst%3DC.%2BM.%26atitle%3DBRD4%2520engagement%2520from%2520chromatin%2520targeting%2520to%2520transcriptional%2520regulation%253A%2520selective%2520contact%2520with%2520acetylated%2520histone%2520H3%2520and%2520H4%26jtitle%3DF1000%2520Biol.%2520Rep.%26date%3D2009%26volume%3D1%26spage%3D98%26doi%3D10.3410%2FB1-98" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jonkers, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lis, J. T.</span></span> <span> </span><span class="NLM_article-title">Getting up to speed with transcription elongation by RNA polymerase II</span>. <i>Nat. Rev. Mol. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">167</span>– <span class="NLM_lpage">177</span>, <span class="refDoi"> DOI: 10.1038/nrm3953</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1038%2Fnrm3953" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=25693130" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BC2MXjtFemsrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2015&pages=167-177&author=I.+Jonkersauthor=J.+T.+Lis&title=Getting+up+to+speed+with+transcription+elongation+by+RNA+polymerase+II&doi=10.1038%2Fnrm3953"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Getting up to speed with transcription elongation by RNA polymerase II</span></div><div class="casAuthors">Jonkers, Iris; Lis, John T.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Molecular Cell Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">167-177</span>CODEN:
                <span class="NLM_cas:coden">NRMCBP</span>;
        ISSN:<span class="NLM_cas:issn">1471-0072</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Recent advances in sequencing techniques that measure nascent transcripts and that reveal the positioning of RNA polymerase II (Pol II) have shown that the pausing of Pol II in promoter-proximal regions and its release to initiate a phase of productive elongation are key steps in transcription regulation.  Moreover, after the release of Pol II from the promoter-proximal region, elongation rates are highly dynamic throughout the transcription of a gene, and vary on a gene-by-gene basis.  Interestingly, Pol II elongation rates affect co-transcriptional processes such as splicing, termination and genome stability.  Increasing nos. of factors and regulatory mechanisms have been assocd. with the steps of transcription elongation by Pol II, revealing that elongation is a highly complex process.  Elongation is thus now recognized as a key phase in the regulation of transcription by Pol II.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqmoGYHNxHU1bVg90H21EOLACvtfcHk0lhrKuxxjx-RwA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXjtFemsrw%253D&md5=6b8aa1cef47c37a0fd26a83f60cbab49</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1038%2Fnrm3953&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrm3953%26sid%3Dliteratum%253Aachs%26aulast%3DJonkers%26aufirst%3DI.%26aulast%3DLis%26aufirst%3DJ.%2BT.%26atitle%3DGetting%2520up%2520to%2520speed%2520with%2520transcription%2520elongation%2520by%2520RNA%2520polymerase%2520II%26jtitle%3DNat.%2520Rev.%2520Mol.%2520Cell%2520Biol.%26date%3D2015%26volume%3D16%26spage%3D167%26epage%3D177%26doi%3D10.1038%2Fnrm3953" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Stanlie, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yousif, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Akiyama, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Honjo, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Begum, N. A.</span></span> <span> </span><span class="NLM_article-title">Chromatin reader BRD4 functions in Ig class switching as a repair complex adaptor of nonhomologous end-joining</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">97</span>– <span class="NLM_lpage">110</span>, <span class="refDoi"> DOI: 10.1016/j.molcel.2014.05.018</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1016%2Fj.molcel.2014.05.018" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=24954901" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtVams73E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2014&pages=97-110&author=A.+Stanlieauthor=A.+S.+Yousifauthor=H.+Akiyamaauthor=T.+Honjoauthor=N.+A.+Begum&title=Chromatin+reader+BRD4+functions+in+Ig+class+switching+as+a+repair+complex+adaptor+of+nonhomologous+end-joining&doi=10.1016%2Fj.molcel.2014.05.018"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Chromatin reader Brd4 functions in Ig class switching as a repair complex adaptor of nonhomologous end-joining</span></div><div class="casAuthors">Stanlie, Andre; Yousif, Ashraf S.; Akiyama, Hideo; Honjo, Tasuku; Begum, Nasim A.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">97-110</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Class switch recombination (CSR) is a B cell-specific genomic alteration induced by activation-induced cytidine deaminase (AID)-dependent DNA break at the Ig heavy-chain locus, followed by repair.  Although chromatin-assocd. factors in promoting AID-induced DNA break have been widely reported, the involvement of chromatin adaptors at the repair phase of CSR remains unknown.  Here, we show that the acetylated histone reader Brd4 is crit. for nonhomologous end-joining (NHEJ) repair of AID- and I-SceI-induced DNA breaks.  Brd4 was recruited to the DNA break regions, and its depletion from the chromatin caused CSR impairment without affecting the DNA break generation.  Inhibition of Brd4 suppressed the accumulation of 53BP1 and uracil DNA glycosylase at the switch regions, perturbed the switch junctional microhomol., and reduced Igh/c-myc translocation.  We conclude that Brd4 serves as a chromatin platform required for the recruitment of repair components during CSR and general DNA damage.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGocjLJOukIdgbVg90H21EOLACvtfcHk0lhrKuxxjx-RwA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtVams73E&md5=af90286ee2aa03ffd10c9b6b6087c742</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2014.05.018&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2014.05.018%26sid%3Dliteratum%253Aachs%26aulast%3DStanlie%26aufirst%3DA.%26aulast%3DYousif%26aufirst%3DA.%2BS.%26aulast%3DAkiyama%26aufirst%3DH.%26aulast%3DHonjo%26aufirst%3DT.%26aulast%3DBegum%26aufirst%3DN.%2BA.%26atitle%3DChromatin%2520reader%2520BRD4%2520functions%2520in%2520Ig%2520class%2520switching%2520as%2520a%2520repair%2520complex%2520adaptor%2520of%2520nonhomologous%2520end-joining%26jtitle%3DMol.%2520Cell%26date%3D2014%26volume%3D55%26spage%3D97%26epage%3D110%26doi%3D10.1016%2Fj.molcel.2014.05.018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kim, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, S. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gong, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Battenhouse, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boutz, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bashyal, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Refvik, S. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiang, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xhemalce, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paull, T. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brodbelt, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marcotte, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, K. M.</span></span> <span> </span><span class="NLM_article-title">Systematic bromodomain protein screens identify homologous recombination and R-loop suppression pathways involved in genome integrity</span>. <i>Genes Dev.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>33</i></span>,  <span class="NLM_fpage">1751</span>– <span class="NLM_lpage">1774</span>, <span class="refDoi"> DOI: 10.1101/gad.331231.119</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1101%2Fgad.331231.119" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=31753913" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhtVSlt77J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2019&pages=1751-1774&author=J.+J.+Kimauthor=S.+Y.+Leeauthor=F.+Gongauthor=A.+M.+Battenhouseauthor=D.+R.+Boutzauthor=A.+Bashyalauthor=S.+T.+Refvikauthor=C.+M.+Chiangauthor=B.+Xhemalceauthor=T.+T.+Paullauthor=J.+S.+Brodbeltauthor=E.+M.+Marcotteauthor=K.+M.+Miller&title=Systematic+bromodomain+protein+screens+identify+homologous+recombination+and+R-loop+suppression+pathways+involved+in+genome+integrity&doi=10.1101%2Fgad.331231.119"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Systematic bromodomain protein screens identify homologous recombination and R-loop suppression pathways involved in genome integrity</span></div><div class="casAuthors">Kim, Jae Jin; Lee, Seo Yun; Gong, Fade; Battenhouse, Anna M.; Boutz, Daniel R.; Bashyal, Aarti; Refvik, Samantha T.; Chiang, Cheng-Ming; Xhemalce, Blerta; Paull, Tanya T.; Brodbelt, Jennifer S.; Marcotte, Edward M.; Miller, Kyle M.</div><div class="citationInfo"><span class="NLM_cas:title">Genes & Development</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">23-24</span>),
    <span class="NLM_cas:pages">1751-1774</span>CODEN:
                <span class="NLM_cas:coden">GEDEEP</span>;
        ISSN:<span class="NLM_cas:issn">0890-9369</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">Bromodomain proteins (BRD) are key chromatin regulators of genome function and stability as well as therapeutic targets in cancer.  Here, we systematically delineate the contribution of human BRD proteins for genome stability and DNA double-strand break (DSB) repair using several cell-based assays and proteomic interaction network anal.  Applying these approaches, we identify 24 of the 42 BRD proteins as promoters of DNA repair and/or genome integrity.  We identified a BRD-reader function of PCAF that bound TIP60-mediated histone acetylations at DSBs to recruit a DUB complex to deubiquitylate histone H2BK120, to allowing direct acetylation by PCAF, and repair of DSBs by homologous recombination.  We also discovered the bromo-and-extra-terminal (BET) BRD proteins, BRD2 and BRD4, as neg. regulators of transcription-assocd. RNA-DNA hybrids (R-loops) as inhibition of BRD2 or BRD4 increased R-loop formation, which generated DSBs.  These breaks were reliant on topoisomerase II, and BRD2 directly bound and activated topoisomerase I, a known restrainer of R-loops.  Thus, comprehensive interactome and functional profiling of BRD proteins revealed new homologous recombination and genome stability pathways, providing a framework to understand genome maintenance by BRD proteins and the effects of their pharmacol. inhibition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrS0sg29K3XdbVg90H21EOLACvtfcHk0lhn2GCr-vPeyA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhtVSlt77J&md5=31149336b6a185fb24186b32c6019886</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1101%2Fgad.331231.119&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fgad.331231.119%26sid%3Dliteratum%253Aachs%26aulast%3DKim%26aufirst%3DJ.%2BJ.%26aulast%3DLee%26aufirst%3DS.%2BY.%26aulast%3DGong%26aufirst%3DF.%26aulast%3DBattenhouse%26aufirst%3DA.%2BM.%26aulast%3DBoutz%26aufirst%3DD.%2BR.%26aulast%3DBashyal%26aufirst%3DA.%26aulast%3DRefvik%26aufirst%3DS.%2BT.%26aulast%3DChiang%26aufirst%3DC.%2BM.%26aulast%3DXhemalce%26aufirst%3DB.%26aulast%3DPaull%26aufirst%3DT.%2BT.%26aulast%3DBrodbelt%26aufirst%3DJ.%2BS.%26aulast%3DMarcotte%26aufirst%3DE.%2BM.%26aulast%3DMiller%26aufirst%3DK.%2BM.%26atitle%3DSystematic%2520bromodomain%2520protein%2520screens%2520identify%2520homologous%2520recombination%2520and%2520R-loop%2520suppression%2520pathways%2520involved%2520in%2520genome%2520integrity%26jtitle%3DGenes%2520Dev.%26date%3D2019%26volume%3D33%26spage%3D1751%26epage%3D1774%26doi%3D10.1101%2Fgad.331231.119" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baek, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramanand, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharp, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Welti, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rodrigues, D. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dolling, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Figueiredo, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sumanasuriya, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crespo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aslam, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yin, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mukherjee, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kanchwala, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hughes, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Halsey, W. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiang, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xing, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raj, G. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burma, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Bono, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mani, R. S.</span></span> <span> </span><span class="NLM_article-title">BRD4 promotes DNA repair and mediates the formation of TMPRSS2-ERG gene rearrangements in prostate cancer</span>. <i>Cell Rep.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">796</span>– <span class="NLM_lpage">808</span>, <span class="refDoi"> DOI: 10.1016/j.celrep.2017.12.078</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1016%2Fj.celrep.2017.12.078" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=29346775" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhvFahurY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2018&pages=796-808&author=X.+Liauthor=G.+Baekauthor=S.+G.+Ramanandauthor=A.+Sharpauthor=Y.+Gaoauthor=W.+Yuanauthor=J.+Weltiauthor=D.+N.+Rodriguesauthor=D.+Dollingauthor=I.+Figueiredoauthor=S.+Sumanasuriyaauthor=M.+Crespoauthor=A.+Aslamauthor=R.+Liauthor=Y.+Yinauthor=B.+Mukherjeeauthor=M.+Kanchwalaauthor=A.+M.+Hughesauthor=W.+S.+Halseyauthor=C.+M.+Chiangauthor=C.+Xingauthor=G.+V.+Rajauthor=S.+Burmaauthor=J.+de+Bonoauthor=R.+S.+Mani&title=BRD4+promotes+DNA+repair+and+mediates+the+formation+of+TMPRSS2-ERG+gene+rearrangements+in+prostate+cancer&doi=10.1016%2Fj.celrep.2017.12.078"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">BRD4 Promotes DNA Repair and Mediates the Formation of TMPRSS2-ERG Gene Rearrangements in Prostate Cancer</span></div><div class="casAuthors">Li, Xiangyi; Baek, Guem Hee; Ramanand, Susmita G.; Sharp, Adam; Gao, Yunpeng; Yuan, Wei; Welti, Jon; Rodrigues, Daniel N.; Dolling, David; Figueiredo, Ines; Sumanasuriya, Semini; Crespo, Mateus; Aslam, Adam; Li, Rui; Yin, Yi; Mukherjee, Bipasha; Kanchwala, Mohammed; Hughes, Ashley M.; Halsey, Wendy S.; Chiang, Cheng-Ming; Xing, Chao; Raj, Ganesh V.; Burma, Sandeep; de Bono, Johann; Mani, Ram S.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Reports</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">796-808</span>CODEN:
                <span class="NLM_cas:coden">CREED8</span>;
        ISSN:<span class="NLM_cas:issn">2211-1247</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">BRD4 belongs to the bromodomain and extraterminal (BET) family of chromatin reader proteins that bind acetylated histones and regulate gene expression.  Pharmacol. inhibition of BRD4 by BET inhibitors (BETi) has indicated antitumor activity against multiple cancer types.  We show that BRD4 is essential for the repair of DNA double-strand breaks (DSBs) and mediates the formation of oncogenic gene rearrangements by engaging the non-homologous end joining (NHEJ) pathway.  Mechanistically, genome-wide DNA breaks are assocd. with enhanced acetylation of histone H4, leading to BRD4 recruitment, and stable establishment of the DNA repair complex.  In support of this, we also show that, in clin. tumor samples, BRD4 protein levels are neg. assocd. with outcome after prostate cancer (PCa) radiation therapy.  Thus, in addn. to regulating gene expression, BRD4 is also a central player in the repair of DNA DSBs, with significant implications for cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr1nW-LR5WHMbVg90H21EOLACvtfcHk0lhn2GCr-vPeyA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhvFahurY%253D&md5=877fe254c14f13633c5528481c39ff92</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1016%2Fj.celrep.2017.12.078&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.celrep.2017.12.078%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DX.%26aulast%3DBaek%26aufirst%3DG.%26aulast%3DRamanand%26aufirst%3DS.%2BG.%26aulast%3DSharp%26aufirst%3DA.%26aulast%3DGao%26aufirst%3DY.%26aulast%3DYuan%26aufirst%3DW.%26aulast%3DWelti%26aufirst%3DJ.%26aulast%3DRodrigues%26aufirst%3DD.%2BN.%26aulast%3DDolling%26aufirst%3DD.%26aulast%3DFigueiredo%26aufirst%3DI.%26aulast%3DSumanasuriya%26aufirst%3DS.%26aulast%3DCrespo%26aufirst%3DM.%26aulast%3DAslam%26aufirst%3DA.%26aulast%3DLi%26aufirst%3DR.%26aulast%3DYin%26aufirst%3DY.%26aulast%3DMukherjee%26aufirst%3DB.%26aulast%3DKanchwala%26aufirst%3DM.%26aulast%3DHughes%26aufirst%3DA.%2BM.%26aulast%3DHalsey%26aufirst%3DW.%2BS.%26aulast%3DChiang%26aufirst%3DC.%2BM.%26aulast%3DXing%26aufirst%3DC.%26aulast%3DRaj%26aufirst%3DG.%2BV.%26aulast%3DBurma%26aufirst%3DS.%26aulast%3Dde%2BBono%26aufirst%3DJ.%26aulast%3DMani%26aufirst%3DR.%2BS.%26atitle%3DBRD4%2520promotes%2520DNA%2520repair%2520and%2520mediates%2520the%2520formation%2520of%2520TMPRSS2-ERG%2520gene%2520rearrangements%2520in%2520prostate%2520cancer%26jtitle%3DCell%2520Rep.%26date%3D2018%26volume%3D22%26spage%3D796%26epage%3D808%26doi%3D10.1016%2Fj.celrep.2017.12.078" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sun, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeong, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vellano, C. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ju, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meric-Bernstam, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yap, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hattersley, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Connor, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fawell, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, S. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peng, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mills, G. B.</span></span> <span> </span><span class="NLM_article-title">BRD4 inhibition is synthetic lethal with PARP inhibitors through the induction of homologous recombination deficiency</span>. <i>Cancer Cell</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>33</i></span>,  <span class="NLM_fpage">401</span>– <span class="NLM_lpage">416</span>, <span class="refDoi"> DOI: 10.1016/j.ccell.2018.01.019</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1016%2Fj.ccell.2018.01.019" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=29533782" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BC1cXksVent7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2018&pages=401-416&author=C.+Sunauthor=J.+Yinauthor=Y.+Fangauthor=J.+Chenauthor=K.+J.+Jeongauthor=X.+Chenauthor=C.+P.+Vellanoauthor=Z.+Juauthor=W.+Zhaoauthor=D.+Zhangauthor=Y.+Luauthor=F.+Meric-Bernstamauthor=T.+A.+Yapauthor=M.+Hattersleyauthor=M.+J.+O%E2%80%99Connorauthor=H.+Chenauthor=S.+Fawellauthor=S.+Y.+Linauthor=G.+Pengauthor=G.+B.+Mills&title=BRD4+inhibition+is+synthetic+lethal+with+PARP+inhibitors+through+the+induction+of+homologous+recombination+deficiency&doi=10.1016%2Fj.ccell.2018.01.019"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">BRD4 Inhibition Is Synthetic Lethal with PARP Inhibitors through the Induction of Homologous Recombination Deficiency</span></div><div class="casAuthors">Sun, Chaoyang; Yin, Jun; Fang, Yong; Chen, Jian; Jeong, Kang Jin; Chen, Xiaohua; Vellano, Christopher P.; Ju, Zhenlin; Zhao, Wei; Zhang, Dong; Lu, Yiling; Meric-Bernstam, Funda; Yap, Timothy A.; Hattersley, Maureen; O'Connor, Mark J.; Chen, Huawei; Fawell, Stephen; Lin, Shiaw-Yih; Peng, Guang; Mills, Gordon B.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">401-416.e8</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Poly(ADP-ribose) polymerase inhibitors (PARPi) are selectively active in cells with homologous recombination (HR) deficiency (HRD) caused by mutations in BRCA1, BRCA2, and other pathway members.  We sought small mols. that induce HRD in HR-competent cells to induce synthetic lethality with PARPi and extend the utility of PARPi.  We demonstrated that inhibition of bromodomain contg. 4 (BRD4) induced HRD and sensitized cells across multiple tumor lineages to PARPi regardless of BRCA1/2, TP53, RAS, or BRAF mutation status through depletion of the DNA double-stand break resection protein CtIP (C-terminal binding protein interacting protein).  Importantly, BRD4 inhibitor (BRD4i) treatment reversed multiple mechanisms of resistance to PARPi.  Furthermore, PARPi and BRD4i are synergistic in multiple in vivo models.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGooOJrqnT3mVLVg90H21EOLACvtfcHk0lhn2GCr-vPeyA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXksVent7c%253D&md5=460bd6deafbbfdaf581a8711c67be116</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1016%2Fj.ccell.2018.01.019&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccell.2018.01.019%26sid%3Dliteratum%253Aachs%26aulast%3DSun%26aufirst%3DC.%26aulast%3DYin%26aufirst%3DJ.%26aulast%3DFang%26aufirst%3DY.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DJeong%26aufirst%3DK.%2BJ.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DVellano%26aufirst%3DC.%2BP.%26aulast%3DJu%26aufirst%3DZ.%26aulast%3DZhao%26aufirst%3DW.%26aulast%3DZhang%26aufirst%3DD.%26aulast%3DLu%26aufirst%3DY.%26aulast%3DMeric-Bernstam%26aufirst%3DF.%26aulast%3DYap%26aufirst%3DT.%2BA.%26aulast%3DHattersley%26aufirst%3DM.%26aulast%3DO%25E2%2580%2599Connor%26aufirst%3DM.%2BJ.%26aulast%3DChen%26aufirst%3DH.%26aulast%3DFawell%26aufirst%3DS.%26aulast%3DLin%26aufirst%3DS.%2BY.%26aulast%3DPeng%26aufirst%3DG.%26aulast%3DMills%26aufirst%3DG.%2BB.%26atitle%3DBRD4%2520inhibition%2520is%2520synthetic%2520lethal%2520with%2520PARP%2520inhibitors%2520through%2520the%2520induction%2520of%2520homologous%2520recombination%2520deficiency%26jtitle%3DCancer%2520Cell%26date%3D2018%26volume%3D33%26spage%3D401%26epage%3D416%26doi%3D10.1016%2Fj.ccell.2018.01.019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">French, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miyoshi, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kubonishi, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grier, H. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perez-Atayde, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fletcher, J. A.</span></span> <span> </span><span class="NLM_article-title">BRD4-NUT fusion oncogene: a novel mechanism in aggressive carcinoma</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">304</span>– <span class="NLM_lpage">307</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=12543779" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BD3sXnt1Klug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2003&pages=304-307&author=C.+A.+Frenchauthor=I.+Miyoshiauthor=I.+Kubonishiauthor=H.+E.+Grierauthor=A.+R.+Perez-Ataydeauthor=J.+A.+Fletcher&title=BRD4-NUT+fusion+oncogene%3A+a+novel+mechanism+in+aggressive+carcinoma"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">BRD4-NUT Fusion Oncogene: A Novel Mechanism in Aggressive Carcinoma</span></div><div class="casAuthors">French, Christopher A.; Miyoshi, Isao; Kubonishi, Ichiro; Grier, Holcombe E.; Perez-Atayde, Antonio R.; Fletcher, Jonathan A.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">304-307</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">The poorly differentiated carcinoma with t(15;19)(q13,p13.1) is characterized by its highly aggressive, invariably lethal clin. course.  The chromosome 19 translocation breakpoint targets the BRD4 double bromodomain-contg. gene, which functions in regulation of cell cycle progression.  Herein the authors demonstrate that BRD4 is fused with nearly the entire transcript of the novel 15q13 gene, NUT (nuclear protein in testis), forming a 6.4-kb fusion oncogene, BRD4-NUT.  NUT, like BRD4, is predicted to encode a nuclear protein but, unlike the ubiquitous BRD4 transcript, is expressed only in testis.  These findings establish a model to elucidate the oncogenic consequences of unscheduled NUT expression and altered BRD4 function.  Very few fusion oncogenes have been identified in epithelial tumors, and BRD4-NUT is the first fusion oncogene mechanism identified in a highly lethal form of carcinoma.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrTa0K_kBu2pLVg90H21EOLACvtfcHk0li2_rItnG-7NA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXnt1Klug%253D%253D&md5=e927ea5874b2013c7467f981e10d7efd</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DFrench%26aufirst%3DC.%2BA.%26aulast%3DMiyoshi%26aufirst%3DI.%26aulast%3DKubonishi%26aufirst%3DI.%26aulast%3DGrier%26aufirst%3DH.%2BE.%26aulast%3DPerez-Atayde%26aufirst%3DA.%2BR.%26aulast%3DFletcher%26aufirst%3DJ.%2BA.%26atitle%3DBRD4-NUT%2520fusion%2520oncogene%253A%2520a%2520novel%2520mechanism%2520in%2520aggressive%2520carcinoma%26jtitle%3DCancer%2520Res.%26date%3D2003%26volume%3D63%26spage%3D304%26epage%3D307" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, Y. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waring, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dey, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ozato, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peng, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ge, K.</span></span> <span> </span><span class="NLM_article-title">BRD4 binds to active enhancers to control cell identity gene induction in adipogenesis and myogenesis</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">2217</span>, <span class="refDoi"> DOI: 10.1038/s41467-017-02403-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1038%2Fs41467-017-02403-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=29263365" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A280%3ADC%252BC1MzjslSlsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=2217&author=J.+E.+Leeauthor=Y.+K.+Parkauthor=S.+Parkauthor=Y.+Jangauthor=N.+Waringauthor=A.+Deyauthor=K.+Ozatoauthor=B.+Laiauthor=W.+Pengauthor=K.+Ge&title=BRD4+binds+to+active+enhancers+to+control+cell+identity+gene+induction+in+adipogenesis+and+myogenesis&doi=10.1038%2Fs41467-017-02403-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Brd4 binds to active enhancers to control cell identity gene induction in adipogenesis and myogenesis</span></div><div class="casAuthors">Lee Ji-Eun; Park Young-Kwon; Park Sarah; Jang Younghoon; Waring Nicholas; Lai Binbin; Ge Kai; Waring Nicholas; Lai Binbin; Peng Weiqun; Dey Anup; Ozato Keiko</div><div class="citationInfo"><span class="NLM_cas:title">Nature communications</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">2217</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The epigenomic reader Brd4 is an important drug target for cancers.  However, its role in cell differentiation and animal development remains largely unclear.  Using two conditional knockout mouse strains and derived cells, we demonstrate that Brd4 controls cell identity gene induction and is essential for adipogenesis and myogenesis.  Brd4 co-localizes with lineage-determining transcription factors (LDTFs) on active enhancers during differentiation.  LDTFs coordinate with H3K4 mono-methyltransferases MLL3/MLL4 (KMT2C/KMT2D) and H3K27 acetyltransferases CBP/p300 to recruit Brd4 to enhancers activated during differentiation.  Brd4 deletion prevents the enrichment of Mediator and RNA polymerase II transcription machinery, but not that of LDTFs, MLL3/MLL4-mediated H3K4me1, and CBP/p300-mediated H3K27ac, on enhancers.  Consequently, Brd4 deletion prevents enhancer RNA production, cell identity gene induction and cell differentiation.  Interestingly, Brd4 is dispensable for maintaining cell identity genes in differentiated cells.  These findings identify Brd4 as an enhancer epigenomic reader that links active enhancers with cell identity gene induction in differentiation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcREMjSSm_naSGZyxfu64m4ffW6udTcc2eaMEebw0WM6w7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1MzjslSlsQ%253D%253D&md5=35aa3eb2ed0f9c1f38a6a2d09b438957</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1038%2Fs41467-017-02403-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41467-017-02403-5%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DJ.%2BE.%26aulast%3DPark%26aufirst%3DY.%2BK.%26aulast%3DPark%26aufirst%3DS.%26aulast%3DJang%26aufirst%3DY.%26aulast%3DWaring%26aufirst%3DN.%26aulast%3DDey%26aufirst%3DA.%26aulast%3DOzato%26aufirst%3DK.%26aulast%3DLai%26aufirst%3DB.%26aulast%3DPeng%26aufirst%3DW.%26aulast%3DGe%26aufirst%3DK.%26atitle%3DBRD4%2520binds%2520to%2520active%2520enhancers%2520to%2520control%2520cell%2520identity%2520gene%2520induction%2520in%2520adipogenesis%2520and%2520myogenesis%26jtitle%3DNat.%2520Commun.%26date%3D2017%26volume%3D8%26spage%3D2217%26doi%3D10.1038%2Fs41467-017-02403-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ge, J. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kwon, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jovanovic, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murphy, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gulvady, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fassl, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trinh, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heavey, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luoma, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paweletz, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thorner, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wucherpfennig, K. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sicinski, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mcdonald, T. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pellman, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michor, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Polyak, K.</span></span> <span> </span><span class="NLM_article-title">Acquired resistance to combined BET and CDK4/6 inhibition in triple-negative breast cancer</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">2350</span>, <span class="refDoi"> DOI: 10.1038/s41467-020-16170-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1038%2Fs41467-020-16170-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=32393766" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BB3cXpt1Khsr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2020&pages=2350&author=J.+Y.+Geauthor=S.+Shuauthor=M.+Kwonauthor=B.+Jovanovicauthor=K.+Murphyauthor=A.+Gulvadyauthor=A.+Fasslauthor=A.+Trinhauthor=Y.+Kuangauthor=G.+A.+Heaveyauthor=A.+Luomaauthor=C.+Paweletzauthor=A.+R.+Thornerauthor=K.+W.+Wucherpfennigauthor=J.+Qiauthor=M.+Brownauthor=P.+Sicinskiauthor=T.+O.+Mcdonaldauthor=D.+Pellmanauthor=F.+Michorauthor=K.+Polyak&title=Acquired+resistance+to+combined+BET+and+CDK4%2F6+inhibition+in+triple-negative+breast+cancer&doi=10.1038%2Fs41467-020-16170-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Acquired resistance to combined BET and CDK4/6 inhibition in triple-negative breast cancer</span></div><div class="casAuthors">Ge, Jennifer Y.; Shu, Shaokun; Kwon, Mijung; Jovanovic, Bojana; Murphy, Katherine; Gulvady, Anushree; Fassl, Anne; Trinh, Anne; Kuang, Yanan; Heavey, Grace A.; Luoma, Adrienne; Paweletz, Cloud; Thorner, Aaron R.; Wucherpfennig, Kai W.; Qi, Jun; Brown, Myles; Sicinski, Piotr; McDonald, Thomas O.; Pellman, David; Michor, Franziska; Polyak, Kornelia</div><div class="citationInfo"><span class="NLM_cas:title">Nature Communications</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">2350</span>CODEN:
                <span class="NLM_cas:coden">NCAOBW</span>;
        ISSN:<span class="NLM_cas:issn">2041-1723</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Abstr.: BET inhibitors are promising therapeutic agents for the treatment of triple-neg. breast cancer (TNBC), but the rapid emergence of resistance necessitates investigation of combination therapies and their effects on tumor evolution.  Here, we show that palbociclib, a CDK4/6 inhibitor, and paclitaxel, a microtubule inhibitor, synergize with the BET inhibitor JQ1 in TNBC lines.  High-complexity DNA barcoding and math. modeling indicate a high rate of de novo acquired resistance to these drugs relative to pre-existing resistance.  We demonstrate that the combination of JQ1 and palbociclib induces cell division errors, which can increase the chance of developing aneuploidy.  Characterizing acquired resistance to combination treatment at a single cell level shows heterogeneous mechanisms including activation of G1-S and senescence pathways.  Our results establish a rationale for further investigation of combined BET and CDK4/6 inhibition in TNBC and suggest novel mechanisms of action for these drugs and new vulnerabilities in cells after emergence of resistance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq8HPBD7AnPVLVg90H21EOLACvtfcHk0li2_rItnG-7NA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXpt1Khsr4%253D&md5=ff7d4b06ea3b3a7f6eec1e97764e169b</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1038%2Fs41467-020-16170-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41467-020-16170-3%26sid%3Dliteratum%253Aachs%26aulast%3DGe%26aufirst%3DJ.%2BY.%26aulast%3DShu%26aufirst%3DS.%26aulast%3DKwon%26aufirst%3DM.%26aulast%3DJovanovic%26aufirst%3DB.%26aulast%3DMurphy%26aufirst%3DK.%26aulast%3DGulvady%26aufirst%3DA.%26aulast%3DFassl%26aufirst%3DA.%26aulast%3DTrinh%26aufirst%3DA.%26aulast%3DKuang%26aufirst%3DY.%26aulast%3DHeavey%26aufirst%3DG.%2BA.%26aulast%3DLuoma%26aufirst%3DA.%26aulast%3DPaweletz%26aufirst%3DC.%26aulast%3DThorner%26aufirst%3DA.%2BR.%26aulast%3DWucherpfennig%26aufirst%3DK.%2BW.%26aulast%3DQi%26aufirst%3DJ.%26aulast%3DBrown%26aufirst%3DM.%26aulast%3DSicinski%26aufirst%3DP.%26aulast%3DMcdonald%26aufirst%3DT.%2BO.%26aulast%3DPellman%26aufirst%3DD.%26aulast%3DMichor%26aufirst%3DF.%26aulast%3DPolyak%26aufirst%3DK.%26atitle%3DAcquired%2520resistance%2520to%2520combined%2520BET%2520and%2520CDK4%252F6%2520inhibition%2520in%2520triple-negative%2520breast%2520cancer%26jtitle%3DNat.%2520Commun.%26date%3D2020%26volume%3D11%26spage%3D2350%26doi%3D10.1038%2Fs41467-020-16170-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rosencrance, C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ammouri, H. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ge, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rendleman, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marshall, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eagen, K. P.</span></span> <span> </span><span class="NLM_article-title">Chromatin hyperacetylation impacts chromosome folding by forming a nuclear subcompartment</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>78</i></span>,  <span class="NLM_fpage">112</span>– <span class="NLM_lpage">126</span>, <span class="refDoi"> DOI: 10.1016/j.molcel.2020.03.018</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1016%2Fj.molcel.2020.03.018" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=32243828" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BB3cXmsFOjsbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=78&publication_year=2020&pages=112-126&author=C.+D.+Rosencranceauthor=H.+N.+Ammouriauthor=Q.+Yuauthor=T.+Geauthor=E.+J.+Rendlemanauthor=S.+A.+Marshallauthor=K.+P.+Eagen&title=Chromatin+hyperacetylation+impacts+chromosome+folding+by+forming+a+nuclear+subcompartment&doi=10.1016%2Fj.molcel.2020.03.018"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Chromatin Hyperacetylation Impacts Chromosome Folding by Forming a Nuclear Subcompartment</span></div><div class="casAuthors">Rosencrance, Celeste D.; Ammouri, Haneen N.; Yu, Qi; Ge, Tiffany; Rendleman, Emily J.; Marshall, Stacy A.; Eagen, Kyle P.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">78</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">112-126.e12</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Delineating how chromosomes fold at length scales beyond one megabase remains obscure relative to smaller-scale folding into TADs, loops, and nucleosomes.  We find that rather than simply unfolding chromatin, histone hyperacetylation results in interactions between distant genomic loci sepd. by tens to hundreds of megabases, even in the absence of transcription.  These hyperacetylated "megadomains" are formed by the BRD4-NUT fusion oncoprotein, interact both within and between chromosomes, and form a specific nuclear subcompartment that has elevated gene activity with respect to other subcompartments.  Pharmacol. degrdn. of BRD4-NUT results in collapse of megadomains and attenuation of the interactions between them.  In contrast, these interactions persist and contacts between newly acetylated regions are formed after inhibiting RNA polymerase II initiation.  Our structure-function approach thus reveals that broad chromatin domains of identical biochem. compn., independent of transcription, form nuclear subcompartments, and also indicates the potential of altering chromosome structure for treating human disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp7CF3cuc8CRrVg90H21EOLACvtfcHk0ljXfDpfAp0Kbw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXmsFOjsbk%253D&md5=2772ff70e2e2f1b07b4d7a77185d4b76</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2020.03.018&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2020.03.018%26sid%3Dliteratum%253Aachs%26aulast%3DRosencrance%26aufirst%3DC.%2BD.%26aulast%3DAmmouri%26aufirst%3DH.%2BN.%26aulast%3DYu%26aufirst%3DQ.%26aulast%3DGe%26aufirst%3DT.%26aulast%3DRendleman%26aufirst%3DE.%2BJ.%26aulast%3DMarshall%26aufirst%3DS.%2BA.%26aulast%3DEagen%26aufirst%3DK.%2BP.%26atitle%3DChromatin%2520hyperacetylation%2520impacts%2520chromosome%2520folding%2520by%2520forming%2520a%2520nuclear%2520subcompartment%26jtitle%3DMol.%2520Cell%26date%3D2020%26volume%3D78%26spage%3D112%26epage%3D126%26doi%3D10.1016%2Fj.molcel.2020.03.018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, X. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kulej, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Macdonald, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiang, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garcia, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">You, J.</span></span> <span> </span><span class="NLM_article-title">Uncovering BRD4 hyperphosphorylation associated with cellular transformation in NUT midline carcinoma</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>114</i></span>,  <span class="NLM_fpage">E5352</span>– <span class="NLM_lpage">E5361</span>, <span class="refDoi"> DOI: 10.1073/pnas.1703071114</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1073%2Fpnas.1703071114" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=28630312" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtVWmtLjM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=114&publication_year=2017&pages=E5352-E5361&author=R.+Wangauthor=X.+J.+Caoauthor=K.+Kulejauthor=W.+Liuauthor=T.+Maauthor=M.+Macdonaldauthor=C.+M.+Chiangauthor=B.+A.+Garciaauthor=J.+You&title=Uncovering+BRD4+hyperphosphorylation+associated+with+cellular+transformation+in+NUT+midline+carcinoma&doi=10.1073%2Fpnas.1703071114"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Uncovering BRD4 hyperphosphorylation associated with cellular transformation in NUT midline carcinoma</span></div><div class="casAuthors">Wang, Ranran; Cao, Xing-Jun; Kulej, Katarzyna; Liu, Wei; Ma, Tongcui; MacDonald, Margo; Chiang, Cheng-Ming; Garcia, Benjamin A.; You, Jianxin</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">114</span>
        (<span class="NLM_cas:issue">27</span>),
    <span class="NLM_cas:pages">E5352-E5361</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The epigenetic reader BRD4 plays a vital role in transcriptional regulation, cellular growth control, and cell-cycle progression.  Dysregulation of BRD4 function has been implicated in the pathogenesis of a wide range of cancers.  However, how BRD4 is regulated to maintain its normal function in healthy cells and how alteration of this process leads to cancer remain poorly understood.  In this study, we discovered that BRD4 is hyperphosphorylated in NUT midline carcinoma and identified CDK9 as a potential kinase mediating BRD4 hyperphosphorylation.  Disruption of BRD4 hyperphosphorylation using both chem. and mol. inhibitors led to the repression of BRD4 downstream oncogenes and abrogation of cellular transformation.  BRD4 hyperphosphorylation is also obsd. in other cancers displaying enhanced BRD4 oncogenic activity.  Our study revealed a mechanism that may regulate BRD4 biol. function through phosphorylation, which, when dysregulated, could lead to oncogenesis.  Our finding points to strategies to target the aberrant BRD4 signaling specifically for cancer intervention.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo3oN-XMSFJurVg90H21EOLACvtfcHk0ljXfDpfAp0Kbw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtVWmtLjM&md5=51fc4e5afb1d86b8a4e30b0c1cbc066e</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1703071114&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1703071114%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DR.%26aulast%3DCao%26aufirst%3DX.%2BJ.%26aulast%3DKulej%26aufirst%3DK.%26aulast%3DLiu%26aufirst%3DW.%26aulast%3DMa%26aufirst%3DT.%26aulast%3DMacdonald%26aufirst%3DM.%26aulast%3DChiang%26aufirst%3DC.%2BM.%26aulast%3DGarcia%26aufirst%3DB.%2BA.%26aulast%3DYou%26aufirst%3DJ.%26atitle%3DUncovering%2520BRD4%2520hyperphosphorylation%2520associated%2520with%2520cellular%2520transformation%2520in%2520NUT%2520midline%2520carcinoma%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2017%26volume%3D114%26spage%3DE5352%26epage%3DE5361%26doi%3D10.1073%2Fpnas.1703071114" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ceribelli, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hou, Z. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kelly, P. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, D. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wright, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ganapathi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evbuomwan, M. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pittaluga, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaffer, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marcucci, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Forman, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiao, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guha, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferrer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chaperot, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plumas, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jaffe, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reizis, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Staudt, L. M.</span></span> <span> </span><span class="NLM_article-title">A druggable TCF4- and BRD4-dependent transcriptional network sustains malignancy in blastic plasmacytoid dendritic cell neoplasm</span>. <i>Cancer Cell</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">764</span>– <span class="NLM_lpage">778</span>, <span class="refDoi"> DOI: 10.1016/j.ccell.2016.10.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1016%2Fj.ccell.2016.10.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=27846392" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvVOgsLfN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2016&pages=764-778&author=M.+Ceribelliauthor=Z.+E.+Houauthor=P.+N.+Kellyauthor=D.+W.+Huangauthor=G.+Wrightauthor=K.+Ganapathiauthor=M.+O.+Evbuomwanauthor=S.+Pittalugaauthor=A.+L.+Shafferauthor=G.+Marcucciauthor=S.+J.+Formanauthor=W.+Xiaoauthor=R.+Guhaauthor=X.+Zhangauthor=M.+Ferrerauthor=L.+Chaperotauthor=J.+Plumasauthor=E.+S.+Jaffeauthor=C.+J.+Thomasauthor=B.+Reizisauthor=L.+M.+Staudt&title=A+druggable+TCF4-+and+BRD4-dependent+transcriptional+network+sustains+malignancy+in+blastic+plasmacytoid+dendritic+cell+neoplasm&doi=10.1016%2Fj.ccell.2016.10.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">A Druggable TCF4- and BRD4-Dependent Transcriptional Network Sustains Malignancy in Blastic Plasmacytoid Dendritic Cell Neoplasm</span></div><div class="casAuthors">Ceribelli, Michele; Hou, Zhiying Esther; Kelly, Priscilla N.; Huang, Da Wei; Wright, George; Ganapathi, Karthik; Evbuomwan, Moses O.; Pittaluga, Stefania; Shaffer, Arthur L.; Marcucci, Guido; Forman, Stephen J.; Xiao, Wenming; Guha, Rajarshi; Zhang, Xiaohu; Ferrer, Marc; Chaperot, Laurence; Plumas, Joel; Jaffe, Elaine S.; Thomas, Craig J.; Reizis, Boris; Staudt, Louis M.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">764-778</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Blastic plasmacytoid dendritic cell neoplasm (BPDCN) is an aggressive and largely incurable hematol. malignancy originating from plasmacytoid dendritic cells (pDCs).  Using RNAi screening, we identified the E-box transcription factor TCF4 as a master regulator of the BPDCN oncogenic program.  TCF4 served as a faithful diagnostic marker of BPDCN, and its downregulation caused the loss of the BPDCN-specific gene expression program and apoptosis.  High-throughput drug screening revealed that bromodomain and extra-terminal domain inhibitors (BETis) induced BPDCN apoptosis, which was attributable to disruption of a BPDCN-specific transcriptional network controlled by TCF4-dependent super-enhancers.  BETis retarded the growth of BPDCN xenografts, supporting their clin. evaluation in this recalcitrant malignancy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqIY_1k7hZSwrVg90H21EOLACvtfcHk0ljSThCWfhD9yA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvVOgsLfN&md5=8b21e09a917549c508f1df0916eb7412</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1016%2Fj.ccell.2016.10.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccell.2016.10.002%26sid%3Dliteratum%253Aachs%26aulast%3DCeribelli%26aufirst%3DM.%26aulast%3DHou%26aufirst%3DZ.%2BE.%26aulast%3DKelly%26aufirst%3DP.%2BN.%26aulast%3DHuang%26aufirst%3DD.%2BW.%26aulast%3DWright%26aufirst%3DG.%26aulast%3DGanapathi%26aufirst%3DK.%26aulast%3DEvbuomwan%26aufirst%3DM.%2BO.%26aulast%3DPittaluga%26aufirst%3DS.%26aulast%3DShaffer%26aufirst%3DA.%2BL.%26aulast%3DMarcucci%26aufirst%3DG.%26aulast%3DForman%26aufirst%3DS.%2BJ.%26aulast%3DXiao%26aufirst%3DW.%26aulast%3DGuha%26aufirst%3DR.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DFerrer%26aufirst%3DM.%26aulast%3DChaperot%26aufirst%3DL.%26aulast%3DPlumas%26aufirst%3DJ.%26aulast%3DJaffe%26aufirst%3DE.%2BS.%26aulast%3DThomas%26aufirst%3DC.%2BJ.%26aulast%3DReizis%26aufirst%3DB.%26aulast%3DStaudt%26aufirst%3DL.%2BM.%26atitle%3DA%2520druggable%2520TCF4-%2520and%2520BRD4-dependent%2520transcriptional%2520network%2520sustains%2520malignancy%2520in%2520blastic%2520plasmacytoid%2520dendritic%2520cell%2520neoplasm%26jtitle%3DCancer%2520Cell%26date%3D2016%26volume%3D30%26spage%3D764%26epage%3D778%26doi%3D10.1016%2Fj.ccell.2016.10.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kleppe, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Levine, R. L.</span></span> <span> </span><span class="NLM_article-title">An unexpected chink in the transcriptional armor of plasmacytoid dendritic neoplasms</span>. <i>Cancer Cell</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">659</span>– <span class="NLM_lpage">660</span>, <span class="refDoi"> DOI: 10.1016/j.ccell.2016.10.016</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1016%2Fj.ccell.2016.10.016" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=27846385" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvVOgsb7M" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2016&pages=659-660&author=M.+Kleppeauthor=R.+L.+Levine&title=An+unexpected+chink+in+the+transcriptional+armor+of+plasmacytoid+dendritic+neoplasms&doi=10.1016%2Fj.ccell.2016.10.016"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">An Unexpected Chink in the Transcriptional Armor of Plasmacytoid Dendritic Neoplasms</span></div><div class="casAuthors">Kleppe, Maria; Levine, Ross L.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">659-660</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">In this issue of Cancer Cell, Ceribelli et al. use functional genomic and chem. screening to reveal the existence of a TCF4/BRD4-dependent oncogenic regulatory network in blastic plasmacytoid dendritic cell neoplasm (BPDCN) and demonstrate that BPDCN cells are highly sensitive to therapeutic targeting of this novel dependency.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqiVskGYzNKq7Vg90H21EOLACvtfcHk0ljSThCWfhD9yA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvVOgsb7M&md5=0cc7bda6c1edd829af3a3ae19dea5a6c</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1016%2Fj.ccell.2016.10.016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccell.2016.10.016%26sid%3Dliteratum%253Aachs%26aulast%3DKleppe%26aufirst%3DM.%26aulast%3DLevine%26aufirst%3DR.%2BL.%26atitle%3DAn%2520unexpected%2520chink%2520in%2520the%2520transcriptional%2520armor%2520of%2520plasmacytoid%2520dendritic%2520neoplasms%26jtitle%3DCancer%2520Cell%26date%3D2016%26volume%3D30%26spage%3D659%26epage%3D660%26doi%3D10.1016%2Fj.ccell.2016.10.016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Olley, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ansari, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bengani, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grimes, G. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rhodes, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">von Kriegsheim, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blatnik, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stewart, F. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wakeling, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carroll, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ross, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, S.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bickmore, W. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pradeepa, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">FitzPatrick, D. R.</span></span> <span> </span><span class="NLM_article-title">Fitzpatrick D. R. BRD4 interacts with NIPBL and BRD4 is mutated in a Cornelia de Lange-like syndrome</span>. <i>Nat. Genet.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>50</i></span>,  <span class="NLM_fpage">329</span>– <span class="NLM_lpage">332</span>, <span class="refDoi"> DOI: 10.1038/s41588-018-0042-y</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1038%2Fs41588-018-0042-y" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=29379197" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BC1cXkslGiu78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2018&pages=329-332&author=G.+Olleyauthor=M.+Ansariauthor=H.+Benganiauthor=G.+R.+Grimesauthor=J.+Rhodesauthor=A.+von+Kriegsheimauthor=A.+Blatnikauthor=F.+J.+Stewartauthor=E.+Wakelingauthor=N.+Carrollauthor=A.+Rossauthor=S.-M.+Parkauthor=W.+A.+Bickmoreauthor=M.+M.+Pradeepaauthor=D.+R.+FitzPatrick&title=Fitzpatrick+D.+R.+BRD4+interacts+with+NIPBL+and+BRD4+is+mutated+in+a+Cornelia+de+Lange-like+syndrome&doi=10.1038%2Fs41588-018-0042-y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">BRD4 interacts with NIPBL and BRD4 is mutated in a Cornelia de Lange-like syndrome</span></div><div class="casAuthors">Olley, Gabrielle; Ansari, Morad; Bengani, Hemant; Grimes, Graeme R.; Rhodes, James; von Kriegsheim, Alex; Blatnik, Ana; Stewart, Fiona J.; Wakeling, Emma; Carroll, Nicola; Ross, Alison; Park, Soo-Mi; Bickmore, Wendy A.; Pradeepa, Madapura M.; FitzPatrick, David R.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Genetics</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">329-332</span>CODEN:
                <span class="NLM_cas:coden">NGENEC</span>;
        ISSN:<span class="NLM_cas:issn">1061-4036</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">We found that the clin. phenotype assocd. with BRD4 haploinsufficiency overlapped with that of Cornelia de Lange syndrome (CdLS), which is most often caused by mutation of NIPBL.  More typical CdLS was obsd. with a de novo BRD4 missense variant, which retained the ability to coimmunoppt. with NIPBL, but bound poorly to acetylated histones.  BRD4 and NIPBL displayed correlated binding at super-enhancers and appeared to co-regulate developmental gene expression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGodrM8Io0YGb7Vg90H21EOLACvtfcHk0ljSThCWfhD9yA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXkslGiu78%253D&md5=87ef00b8de0b134ea874bc9092bdd632</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1038%2Fs41588-018-0042-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41588-018-0042-y%26sid%3Dliteratum%253Aachs%26aulast%3DOlley%26aufirst%3DG.%26aulast%3DAnsari%26aufirst%3DM.%26aulast%3DBengani%26aufirst%3DH.%26aulast%3DGrimes%26aufirst%3DG.%2BR.%26aulast%3DRhodes%26aufirst%3DJ.%26aulast%3Dvon%2BKriegsheim%26aufirst%3DA.%26aulast%3DBlatnik%26aufirst%3DA.%26aulast%3DStewart%26aufirst%3DF.%2BJ.%26aulast%3DWakeling%26aufirst%3DE.%26aulast%3DCarroll%26aufirst%3DN.%26aulast%3DRoss%26aufirst%3DA.%26aulast%3DPark%26aufirst%3DS.-M.%26aulast%3DBickmore%26aufirst%3DW.%2BA.%26aulast%3DPradeepa%26aufirst%3DM.%2BM.%26aulast%3DFitzPatrick%26aufirst%3DD.%2BR.%26atitle%3DFitzpatrick%2520D.%2520R.%2520BRD4%2520interacts%2520with%2520NIPBL%2520and%2520BRD4%2520is%2520mutated%2520in%2520a%2520Cornelia%2520de%2520Lange-like%2520syndrome%26jtitle%3DNat.%2520Genet.%26date%3D2018%26volume%3D50%26spage%3D329%26epage%3D332%26doi%3D10.1038%2Fs41588-018-0042-y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Korb, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herre, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zucker-Scharff, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gresack, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allis, C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Darnell, R. B.</span></span> <span> </span><span class="NLM_article-title">Excess translation of epigenetic regulators contributes to Fragile X Syndrome and is alleviated by BRD4 inhibition</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>170</i></span>,  <span class="NLM_fpage">1209</span>– <span class="NLM_lpage">1223</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2017.07.033</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1016%2Fj.cell.2017.07.033" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=28823556" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtlKmsr3O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=170&publication_year=2017&pages=1209-1223&author=E.+Korbauthor=M.+Herreauthor=I.+Zucker-Scharffauthor=J.+Gresackauthor=C.+D.+Allisauthor=R.+B.+Darnell&title=Excess+translation+of+epigenetic+regulators+contributes+to+Fragile+X+Syndrome+and+is+alleviated+by+BRD4+inhibition&doi=10.1016%2Fj.cell.2017.07.033"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Excess Translation of Epigenetic Regulators Contributes to Fragile X Syndrome and Is Alleviated by Brd4 Inhibition</span></div><div class="casAuthors">Korb, Erica; Herre, Margaret; Zucker-Scharff, Ilana; Gresack, Jodi; Allis, C. David; Darnell, Robert B.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">170</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1209-1223.e20</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Fragile X syndrome (FXS) is a leading genetic cause of intellectual disability and autism.  FXS results from the loss of function of fragile X mental retardation protein (FMRP), which represses translation of target transcripts.  Most of the well-characterized target transcripts of FMRP are synaptic proteins, yet targeting these proteins has not provided effective treatments.  We examd. a group of FMRP targets that encode transcriptional regulators, particularly chromatin-assocd. proteins.  Loss of FMRP in mice results in widespread changes in chromatin regulation and aberrant gene expression.  To det. if targeting epigenetic factors could reverse phenotypes assocd. with the disorder, we focused on Brd4, a BET protein and chromatin reader targeted by FMRP.  Inhibition of Brd4 function alleviated many of the phenotypes assocd. with FXS.  We conclude that loss of FMRP results in significant epigenetic misregulation and that targeting transcription via epigenetic regulators like Brd4 may provide new treatments for FXS.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrJePhuggwmkLVg90H21EOLACvtfcHk0ljDkThbOSXebA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtlKmsr3O&md5=0a374238040e3f466323ddf17bd4b372</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2017.07.033&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2017.07.033%26sid%3Dliteratum%253Aachs%26aulast%3DKorb%26aufirst%3DE.%26aulast%3DHerre%26aufirst%3DM.%26aulast%3DZucker-Scharff%26aufirst%3DI.%26aulast%3DGresack%26aufirst%3DJ.%26aulast%3DAllis%26aufirst%3DC.%2BD.%26aulast%3DDarnell%26aufirst%3DR.%2BB.%26atitle%3DExcess%2520translation%2520of%2520epigenetic%2520regulators%2520contributes%2520to%2520Fragile%2520X%2520Syndrome%2520and%2520is%2520alleviated%2520by%2520BRD4%2520inhibition%26jtitle%3DCell%26date%3D2017%26volume%3D170%26spage%3D1209%26epage%3D1223%26doi%3D10.1016%2Fj.cell.2017.07.033" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wold, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tian, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brasier, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, J.</span></span> <span> </span><span class="NLM_article-title">Drug discovery targeting bromodomain-containing protein 4</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">4533</span>– <span class="NLM_lpage">4558</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b01761</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01761" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BC2sXis1yiurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=4533-4558&author=Z.+Liuauthor=P.+Wangauthor=H.+Chenauthor=E.+A.+Woldauthor=B.+Tianauthor=A.+R.+Brasierauthor=J.+Zhou&title=Drug+discovery+targeting+bromodomain-containing+protein+4&doi=10.1021%2Facs.jmedchem.6b01761"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Drug Discovery Targeting Bromodomain-Containing Protein 4</span></div><div class="casAuthors">Liu, Zhiqing; Wang, Pingyuan; Chen, Haiying; Wold, Eric A.; Tian, Bing; Brasier, Allan R.; Zhou, Jia</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">4533-4558</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  BRD4, the most extensively studied member of BET family, is an epigenetic regulator that localizes to DNA via binding acetylated histones and controls the expression of therapeutically important gene regulatory networks through recruiting transcription factors to form mediator complexes, phosphorylating RNA polymerase II and by its intrinsic histone acetyltransferase activity.  Disrupting the protein-protein interactions between BRD4 and acetyl-lysine has been shown to effectively block cell proliferation in cancer, cytokine prodn. in acute inflammation, etc.  To date, significant efforts have been devoted to the development of BRD4 inhibitors, and consequently, a dozen have progressed into human clin. trials.  Herein, the authors summarize the advances in drug discovery and development of BRD4 inhibitors by focusing on their chemotypes, in vitro and in vivo activity, selectivity, relevant mechanisms of action and therapeutic potential.  Opportunities and challenges to achieve selective and efficacious BRD4 inhibitors as a viable therapeutic strategy for human diseases are also highlighted.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq6IAERH4iggrVg90H21EOLACvtfcHk0ljDkThbOSXebA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXis1yiurg%253D&md5=6d7ba5844073fac8c916b1249af402e5</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01761&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01761%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DZ.%26aulast%3DWang%26aufirst%3DP.%26aulast%3DChen%26aufirst%3DH.%26aulast%3DWold%26aufirst%3DE.%2BA.%26aulast%3DTian%26aufirst%3DB.%26aulast%3DBrasier%26aufirst%3DA.%2BR.%26aulast%3DZhou%26aufirst%3DJ.%26atitle%3DDrug%2520discovery%2520targeting%2520bromodomain-containing%2520protein%25204%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D4533%26epage%3D4558%26doi%3D10.1021%2Facs.jmedchem.6b01761" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bechter, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schoffski, P.</span></span> <span> </span><span class="NLM_article-title">Make your best BET: The emerging role of BET inhibitor treatment in malignant tumors</span>. <i>Pharmacol. Ther.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>208</i></span>,  <span class="NLM_fpage">107479</span>, <span class="refDoi"> DOI: 10.1016/j.pharmthera.2020.107479</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1016%2Fj.pharmthera.2020.107479" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=31931101" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BB3cXotFyksQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=208&publication_year=2020&pages=107479&author=O.+Bechterauthor=P.+Schoffski&title=Make+your+best+BET%3A+The+emerging+role+of+BET+inhibitor+treatment+in+malignant+tumors&doi=10.1016%2Fj.pharmthera.2020.107479"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Make your best BET the emerging role of BET inhibitor treatment in malignant tumors</span></div><div class="casAuthors">Bechter, Oliver; Schoffski, Patrick</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacology & Therapeutics</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">208</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">107479</span>CODEN:
                <span class="NLM_cas:coden">PHTHDT</span>;
        ISSN:<span class="NLM_cas:issn">0163-7258</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">Bromodomains are protein-protein interaction modules with a great diversity in terms of no. of proteins and their function.  The bromodomain and extraterminal protein (BET) represents a distinct subclass of bromodomain proteins mainly involved in transcriptional regulation via their interaction with acetylated chromatin.  In cancer cells BET proteins are found to be altered in many ways such as overexpression, mutations and fusions of BET proteins or their interference with cancer relevant signaling pathways and transcriptional programs in order to sustain cancer growth and viability.  Blocking BET protein function with small mols. is assocd. with therapeutic activity.  Consequently, a variety of small mols. have been developed and a no. of phase I clin. trials have explored their tolerability and efficacy in patients with solid tumors and hematol. malignancies.  We will review the rational for applying BET inhibitors in the clinic and we will discuss the toxicity profile as well as efficacy of this new class of protein inhibitors.  We will also highlight the emerging problem of treatment resistance and the potential these drugs might have when combined with other anti-cancer therapies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqiNKo5rStKX7Vg90H21EOLACvtfcHk0lhYZNW_ab238A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXotFyksQ%253D%253D&md5=0f19577c9c572057afcb09414e02886e</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1016%2Fj.pharmthera.2020.107479&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.pharmthera.2020.107479%26sid%3Dliteratum%253Aachs%26aulast%3DBechter%26aufirst%3DO.%26aulast%3DSchoffski%26aufirst%3DP.%26atitle%3DMake%2520your%2520best%2520BET%253A%2520The%2520emerging%2520role%2520of%2520BET%2520inhibitor%2520treatment%2520in%2520malignant%2520tumors%26jtitle%3DPharmacol.%2520Ther.%26date%3D2020%26volume%3D208%26spage%3D107479%26doi%3D10.1016%2Fj.pharmthera.2020.107479" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bolden, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tasdemir, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dow, L. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Es, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilkinson, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clevers, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lowe, S. W.</span></span> <span> </span><span class="NLM_article-title">Inducible in vivo silencing of BRD4 identifies potential toxicities of sustained BET protein inhibition</span>. <i>Cell Rep.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">1919</span>– <span class="NLM_lpage">1929</span>, <span class="refDoi"> DOI: 10.1016/j.celrep.2014.08.025</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1016%2Fj.celrep.2014.08.025" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=25242322" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsFylur7E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2014&pages=1919-1929&author=J.+E.+Boldenauthor=N.+Tasdemirauthor=L.+E.+Dowauthor=J.+H.+van+Esauthor=J.+E.+Wilkinsonauthor=Z.+Zhaoauthor=H.+Cleversauthor=S.+W.+Lowe&title=Inducible+in+vivo+silencing+of+BRD4+identifies+potential+toxicities+of+sustained+BET+protein+inhibition&doi=10.1016%2Fj.celrep.2014.08.025"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Inducible In Vivo Silencing of Brd4 Identifies Potential Toxicities of Sustained BET Protein Inhibition</span></div><div class="casAuthors">Bolden, Jessica E.; Tasdemir, Nilgun; Dow, Lukas E.; van Es, Johan H.; Wilkinson, John E.; Zhao, Zhen; Clevers, Hans; Lowe, Scott W.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Reports</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1919-1929</span>CODEN:
                <span class="NLM_cas:coden">CREED8</span>;
        ISSN:<span class="NLM_cas:issn">2211-1247</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">BET family proteins are novel therapeutic targets for cancer and inflammation and represent the first chromatin readers against which small-mol. inhibitors have been developed.  First-generation BET inhibitors have shown therapeutic efficacy in preclin. models, but the consequences of sustained BET protein inhibition in normal tissues remain poorly characterized.  Using an inducible and reversible transgenic RNAi mouse model, we show that strong suppression of the BET protein Brd4 in adult animals has dramatic effects in multiple tissues.  Brd4-depleted mice display reversible epidermal hyperplasia, alopecia, and decreased cellular diversity and stem cell depletion in the small intestine.  Furthermore, Brd4-suppressed intestines are sensitive to organ stress and show impaired regeneration following irradn., suggesting that concurrent Brd4 suppression and certain cytotoxic therapies may induce undesirable synergistic effects.  These findings provide important insight into Brd4 function in normal tissues and, importantly, predict several potential outcomes assocd. with potent and sustained BET protein inhibition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqGFqmSiDt98LVg90H21EOLACvtfcHk0lgFlrIkTwiNwQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsFylur7E&md5=9de00d3d76679ca7997cb245300a19af</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1016%2Fj.celrep.2014.08.025&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.celrep.2014.08.025%26sid%3Dliteratum%253Aachs%26aulast%3DBolden%26aufirst%3DJ.%2BE.%26aulast%3DTasdemir%26aufirst%3DN.%26aulast%3DDow%26aufirst%3DL.%2BE.%26aulast%3Dvan%2BEs%26aufirst%3DJ.%2BH.%26aulast%3DWilkinson%26aufirst%3DJ.%2BE.%26aulast%3DZhao%26aufirst%3DZ.%26aulast%3DClevers%26aufirst%3DH.%26aulast%3DLowe%26aufirst%3DS.%2BW.%26atitle%3DInducible%2520in%2520vivo%2520silencing%2520of%2520BRD4%2520identifies%2520potential%2520toxicities%2520of%2520sustained%2520BET%2520protein%2520inhibition%26jtitle%3DCell%2520Rep.%26date%3D2014%26volume%3D8%26spage%3D1919%26epage%3D1929%26doi%3D10.1016%2Fj.celrep.2014.08.025" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fong, C. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilan, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lam, E. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rubin, A. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ftouni, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tyler, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stanley, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sinha, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yeh, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morison, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giotopoulos, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lugo, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeffrey, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carpenter, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gregory, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramsay, R. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lane, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abdel-Wahab, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kouzarides, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnstone, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dawson, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huntly, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Papenfuss, A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dawson, M. A.</span></span> <span> </span><span class="NLM_article-title">BET inhibitor resistance emerges from leukaemia stem cells</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>525</i></span>,  <span class="NLM_fpage">538</span>– <span class="NLM_lpage">542</span>, <span class="refDoi"> DOI: 10.1038/nature14888</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1038%2Fnature14888" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=26367796" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsFeiu7fL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=525&publication_year=2015&pages=538-542&author=C.+Y.+Fongauthor=O.+Gilanauthor=E.+Y.+Lamauthor=A.+F.+Rubinauthor=S.+Ftouniauthor=D.+Tylerauthor=K.+Stanleyauthor=D.+Sinhaauthor=P.+Yehauthor=J.+Morisonauthor=G.+Giotopoulosauthor=D.+Lugoauthor=P.+Jeffreyauthor=S.+C.+Leeauthor=C.+Carpenterauthor=R.+Gregoryauthor=R.+G.+Ramsayauthor=S.+W.+Laneauthor=O.+Abdel-Wahabauthor=T.+Kouzaridesauthor=R.+W.+Johnstoneauthor=S.+J.+Dawsonauthor=B.+J.+Huntlyauthor=R.+K.+Prinjhaauthor=A.+T.+Papenfussauthor=M.+A.+Dawson&title=BET+inhibitor+resistance+emerges+from+leukaemia+stem+cells&doi=10.1038%2Fnature14888"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">BET inhibitor resistance emerges from leukaemia stem cells</span></div><div class="casAuthors">Fong, Chun Yew; Gilan, Omer; Lam, Enid Y. N.; Rubin, Alan F.; Ftouni, Sarah; Tyler, Dean; Stanley, Kym; Sinha, Devbarna; Yeh, Paul; Morison, Jessica; Giotopoulos, George; Lugo, Dave; Jeffrey, Philip; Lee, Stanley Chun-Wei; Carpenter, Christopher; Gregory, Richard; Ramsay, Robert G.; Lane, Steven W.; Abdel-Wahab, Omar; Kouzarides, Tony; Johnstone, Ricky W.; Dawson, Sarah-Jane; Huntly, Brian J. P.; Prinjha, Rab K.; Papenfuss, Anthony T.; Dawson, Mark A.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">525</span>
        (<span class="NLM_cas:issue">7570</span>),
    <span class="NLM_cas:pages">538-542</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Bromodomain and extra terminal protein (BET) inhibitors are first-in-class targeted therapies that deliver a new therapeutic opportunity by directly targeting bromodomain proteins that bind acetylated chromatin marks.  Early clin. trials have shown promise, esp. in acute myeloid leukemia, and therefore the evaluation of resistance mechanisms is crucial to optimize the clin. efficacy of these drugs.  Here we use primary mouse hematopoietic stem and progenitor cells immortalized with the fusion protein MLL-AF9 to generate several single-cell clones that demonstrate resistance, in vitro and in vivo, to the prototypical BET inhibitor, I-BET.  Resistance to I-BET confers cross-resistance to chem. distinct BET inhibitors such as JQ1, as well as resistance to genetic knockdown of BET proteins.  Resistance is not mediated through increased drug efflux or metab., but is shown to emerge from leukemia stem cells both ex vivo and in vivo.  Chromatin-bound BRD4 is globally reduced in resistant cells, whereas the expression of key target genes such as Myc remains unaltered, highlighting the existence of alternative mechanisms to regulate transcription.  We demonstrate that resistance to BET inhibitors, in human and mouse leukemia cells, is in part a consequence of increased Wnt/β-catenin signaling, and neg. regulation of this pathway results in restoration of sensitivity to I-BET in vitro and in vivo.  Together, these findings provide new insights into the biol. of acute myeloid leukemia, highlight potential therapeutic limitations of BET inhibitors, and identify strategies that may enhance the clin. utility of these unique targeted therapies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq7Lg1n64SCWrVg90H21EOLACvtfcHk0lgFlrIkTwiNwQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsFeiu7fL&md5=47d1769e077b1c6c92ff52a051368404</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1038%2Fnature14888&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature14888%26sid%3Dliteratum%253Aachs%26aulast%3DFong%26aufirst%3DC.%2BY.%26aulast%3DGilan%26aufirst%3DO.%26aulast%3DLam%26aufirst%3DE.%2BY.%26aulast%3DRubin%26aufirst%3DA.%2BF.%26aulast%3DFtouni%26aufirst%3DS.%26aulast%3DTyler%26aufirst%3DD.%26aulast%3DStanley%26aufirst%3DK.%26aulast%3DSinha%26aufirst%3DD.%26aulast%3DYeh%26aufirst%3DP.%26aulast%3DMorison%26aufirst%3DJ.%26aulast%3DGiotopoulos%26aufirst%3DG.%26aulast%3DLugo%26aufirst%3DD.%26aulast%3DJeffrey%26aufirst%3DP.%26aulast%3DLee%26aufirst%3DS.%2BC.%26aulast%3DCarpenter%26aufirst%3DC.%26aulast%3DGregory%26aufirst%3DR.%26aulast%3DRamsay%26aufirst%3DR.%2BG.%26aulast%3DLane%26aufirst%3DS.%2BW.%26aulast%3DAbdel-Wahab%26aufirst%3DO.%26aulast%3DKouzarides%26aufirst%3DT.%26aulast%3DJohnstone%26aufirst%3DR.%2BW.%26aulast%3DDawson%26aufirst%3DS.%2BJ.%26aulast%3DHuntly%26aufirst%3DB.%2BJ.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DPapenfuss%26aufirst%3DA.%2BT.%26aulast%3DDawson%26aufirst%3DM.%2BA.%26atitle%3DBET%2520inhibitor%2520resistance%2520emerges%2520from%2520leukaemia%2520stem%2520cells%26jtitle%3DNature%26date%3D2015%26volume%3D525%26spage%3D538%26epage%3D542%26doi%3D10.1038%2Fnature14888" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rathert, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roth, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neumann, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muerdter, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roe, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muhar, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deswal, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cerny-Reiterer, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peter, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jude, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoffmann, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boryn, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Axelsson, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schweifer, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tontsch-Grunt, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dow, L. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gianni, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pearson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valent, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stark, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kraut, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vakoc, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zuber, J.</span></span> <span> </span><span class="NLM_article-title">Transcriptional plasticity promotes primary and acquired resistance to BET inhibition</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>525</i></span>,  <span class="NLM_fpage">543</span>– <span class="NLM_lpage">547</span>, <span class="refDoi"> DOI: 10.1038/nature14898</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1038%2Fnature14898" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=26367798" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsFeiu7fO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=525&publication_year=2015&pages=543-547&author=P.+Rathertauthor=M.+Rothauthor=T.+Neumannauthor=F.+Muerdterauthor=J.+S.+Roeauthor=M.+Muharauthor=S.+Deswalauthor=S.+Cerny-Reitererauthor=B.+Peterauthor=J.+Judeauthor=T.+Hoffmannauthor=L.+M.+Borynauthor=E.+Axelssonauthor=N.+Schweiferauthor=U.+Tontsch-Gruntauthor=L.+E.+Dowauthor=D.+Gianniauthor=M.+Pearsonauthor=P.+Valentauthor=A.+Starkauthor=N.+Krautauthor=C.+R.+Vakocauthor=J.+Zuber&title=Transcriptional+plasticity+promotes+primary+and+acquired+resistance+to+BET+inhibition&doi=10.1038%2Fnature14898"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Transcriptional plasticity promotes primary and acquired resistance to BET inhibition</span></div><div class="casAuthors">Rathert, Philipp; Roth, Mareike; Neumann, Tobias; Muerdter, Felix; Roe, Jae-Seok; Muhar, Matthias; Deswal, Sumit; Cerny-Reiterer, Sabine; Peter, Barbara; Jude, Julian; Hoffmann, Thomas; Boryn, Lukasz M.; Axelsson, Elin; Schweifer, Norbert; Tontsch-Grunt, Ulrike; Dow, Lukas E.; Gianni, Davide; Pearson, Mark; Valent, Peter; Stark, Alexander; Kraut, Norbert; Vakoc, Christopher R.; Zuber, Johannes</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">525</span>
        (<span class="NLM_cas:issue">7570</span>),
    <span class="NLM_cas:pages">543-547</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Following the discovery of BRD4 as a non-oncogene addiction target in acute myeloid leukemia (AML), bromodomain and extra terminal protein (BET) inhibitors are being explored as a promising therapeutic avenue in numerous cancers.  While clin. trials have reported single-agent activity in advanced hematol. malignancies, mechanisms detg. the response to BET inhibition remain poorly understood.  To identify factors involved in primary and acquired BET resistance in leukemia, here we perform a chromatin-focused RNAi screen in a sensitive MLL-AF9;NrasG12D-driven AML mouse model, and investigate dynamic transcriptional profiles in sensitive and resistant mouse and human leukemias.  Our screen shows that suppression of the PRC2 complex, contrary to effects in other contexts, promotes BET inhibitor resistance in AML.  PRC2 suppression does not directly affect the regulation of Brd4-dependent transcripts, but facilitates the remodelling of regulatory pathways that restore the transcription of key targets such as Myc.  Similarly, while BET inhibition triggers acute MYC repression in human leukemias regardless of their sensitivity, resistant leukemias are uniformly characterized by their ability to rapidly restore MYC transcription.  This process involves the activation and recruitment of WNT signaling components, which compensate for the loss of BRD4 and drive resistance in various cancer models.  Dynamic chromatin immunopptn. sequencing and self-transcribing active regulatory region sequencing of enhancer profiles reveal that BET-resistant states are characterized by remodelled regulatory landscapes, involving the activation of a focal MYC enhancer that recruits WNT machinery in response to BET inhibition.  Together, our results identify and validate WNT signaling as a driver and candidate biomarker of primary and acquired BET resistance in leukemia, and implicate the rewiring of transcriptional programs as an important mechanism promoting resistance to BET inhibitors and, potentially, other chromatin-targeted therapies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrDgeY3nFr4GrVg90H21EOLACvtfcHk0lg6KJlQcoSZpw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsFeiu7fO&md5=6d6cf4c4123ad6a698b88f02f95a2ebe</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1038%2Fnature14898&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature14898%26sid%3Dliteratum%253Aachs%26aulast%3DRathert%26aufirst%3DP.%26aulast%3DRoth%26aufirst%3DM.%26aulast%3DNeumann%26aufirst%3DT.%26aulast%3DMuerdter%26aufirst%3DF.%26aulast%3DRoe%26aufirst%3DJ.%2BS.%26aulast%3DMuhar%26aufirst%3DM.%26aulast%3DDeswal%26aufirst%3DS.%26aulast%3DCerny-Reiterer%26aufirst%3DS.%26aulast%3DPeter%26aufirst%3DB.%26aulast%3DJude%26aufirst%3DJ.%26aulast%3DHoffmann%26aufirst%3DT.%26aulast%3DBoryn%26aufirst%3DL.%2BM.%26aulast%3DAxelsson%26aufirst%3DE.%26aulast%3DSchweifer%26aufirst%3DN.%26aulast%3DTontsch-Grunt%26aufirst%3DU.%26aulast%3DDow%26aufirst%3DL.%2BE.%26aulast%3DGianni%26aufirst%3DD.%26aulast%3DPearson%26aufirst%3DM.%26aulast%3DValent%26aufirst%3DP.%26aulast%3DStark%26aufirst%3DA.%26aulast%3DKraut%26aufirst%3DN.%26aulast%3DVakoc%26aufirst%3DC.%2BR.%26aulast%3DZuber%26aufirst%3DJ.%26atitle%3DTranscriptional%2520plasticity%2520promotes%2520primary%2520and%2520acquired%2520resistance%2520to%2520BET%2520inhibition%26jtitle%3DNature%26date%3D2015%26volume%3D525%26spage%3D543%26epage%3D547%26doi%3D10.1038%2Fnature14898" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kumar, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raza, S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knab, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chow, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kwok, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bentrem, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Popovic, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ebine, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Licht, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Munshi, H. G.</span></span> <span> </span><span class="NLM_article-title">GLI2-dependent c-MYC upregulation mediates resistance of pancreatic cancer cells to the BET bromodomain inhibitor JQ1</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">9489</span>, <span class="refDoi"> DOI: 10.1038/srep09489</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1038%2Fsrep09489" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=25807524" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtFOjurjO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2015&pages=9489&author=K.+Kumarauthor=S.+S.+Razaauthor=L.+M.+Knabauthor=C.+R.+Chowauthor=B.+Kwokauthor=D.+J.+Bentremauthor=R.+Popovicauthor=K.+Ebineauthor=J.+D.+Lichtauthor=H.+G.+Munshi&title=GLI2-dependent+c-MYC+upregulation+mediates+resistance+of+pancreatic+cancer+cells+to+the+BET+bromodomain+inhibitor+JQ1&doi=10.1038%2Fsrep09489"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">GLI2-dependent c-MYC upregulation mediates resistance of pancreatic cancer cells to the BET bromodomain inhibitor JQ1</span></div><div class="casAuthors">Kumar, Krishan; Raza, Sania S.; Knab, Lawrence M.; Chow, Christina R.; Kwok, Benjamin; Bentrem, David J.; Popovic, Relja; Ebine, Kazumi; Licht, Jonathan D.; Munshi, Hidayatullah G.</div><div class="citationInfo"><span class="NLM_cas:title">Scientific Reports</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">9489</span>CODEN:
                <span class="NLM_cas:coden">SRCEC3</span>;
        ISSN:<span class="NLM_cas:issn">2045-2322</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">JQ1 and I-BET151 are selective inhibitors of BET bromodomain proteins that have efficacy against a no. of different cancers.  Since the effectiveness of targeted therapies is often limited by development of resistance, we examd. whether it was possible for cancer cells to develop resistance to the BET inhibitor JQ1.  Here we show that pancreatic cancer cells developing resistance to JQ1 demonstrate cross-resistance to I-BET151 and insensitivity to BRD4 downregulation.  The resistant cells maintain expression of c-MYC, increase expression of JQ1-target genes FOSL1 and HMGA2, and demonstrate evidence of epithelial-mesenchymal transition (EMT).  However, reverting EMT fails to sensitize the resistant cells to JQ1 treatment.  Importantly, the JQ1-resistant cells remain dependent on c-MYC that now becomes co-regulated by high levels of GLI2.  Furthermore, downregulating GLI2 re-sensitizes the resistant cells to JQ1.  Overall, these results identify a mechanism by which cancer cells develop resistance to BET inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp6mUgUS37czbVg90H21EOLACvtfcHk0lg6KJlQcoSZpw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtFOjurjO&md5=c5f9dad7134441f6c7edbd6dcc7b40e8</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1038%2Fsrep09489&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsrep09489%26sid%3Dliteratum%253Aachs%26aulast%3DKumar%26aufirst%3DK.%26aulast%3DRaza%26aufirst%3DS.%2BS.%26aulast%3DKnab%26aufirst%3DL.%2BM.%26aulast%3DChow%26aufirst%3DC.%2BR.%26aulast%3DKwok%26aufirst%3DB.%26aulast%3DBentrem%26aufirst%3DD.%2BJ.%26aulast%3DPopovic%26aufirst%3DR.%26aulast%3DEbine%26aufirst%3DK.%26aulast%3DLicht%26aufirst%3DJ.%2BD.%26aulast%3DMunshi%26aufirst%3DH.%2BG.%26atitle%3DGLI2-dependent%2520c-MYC%2520upregulation%2520mediates%2520resistance%2520of%2520pancreatic%2520cancer%2520cells%2520to%2520the%2520BET%2520bromodomain%2520inhibitor%2520JQ1%26jtitle%3DSci.%2520Rep.%26date%3D2015%26volume%3D5%26spage%3D9489%26doi%3D10.1038%2Fsrep09489" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Settleman, J.</span></span> <span> </span><span class="NLM_article-title">Cancer: BET on drug resistance</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>529</i></span>,  <span class="NLM_fpage">289</span>– <span class="NLM_lpage">290</span>, <span class="refDoi"> DOI: 10.1038/nature16863</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1038%2Fnature16863" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=26735017" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BC28Xns1GlsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=529&publication_year=2016&pages=289-290&author=J.+Settleman&title=Cancer%3A+BET+on+drug+resistance&doi=10.1038%2Fnature16863"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Cancer Bet on drug resistance</span></div><div class="casAuthors">Settleman, Jeff</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">529</span>
        (<span class="NLM_cas:issue">7586</span>),
    <span class="NLM_cas:pages">289-290</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Inhibitors of the BET bromodomain proteins are promising cancer therapeutics, but tumor cells are likely to become resistant to these drugs.  Anticipated mechanisms of resistance have now been described.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq3IC8VaowJQLVg90H21EOLACvtfcHk0liVM1XGfv45cQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xns1GlsA%253D%253D&md5=9ce8b053ecf769ab9117983eb80b749f</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1038%2Fnature16863&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature16863%26sid%3Dliteratum%253Aachs%26aulast%3DSettleman%26aufirst%3DJ.%26atitle%3DCancer%253A%2520BET%2520on%2520drug%2520resistance%26jtitle%3DNature%26date%3D2016%26volume%3D529%26spage%3D289%26epage%3D290%26doi%3D10.1038%2Fnature16863" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jang, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eom, J. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeung, H. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheong, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Min, Y. H.</span></span> <span> </span><span class="NLM_article-title">AMPK-ULK1-mediated autophagy confers resistance to BET Inhibitor JQ1 in acute myeloid leukemia stem cells</span>. <i>Clin. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">2781</span>– <span class="NLM_lpage">2794</span>, <span class="refDoi"> DOI: 10.1158/1078-0432.CCR-16-1903</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1158%2F1078-0432.CCR-16-1903" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=27864418" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BC2sXpt1Sms7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2017&pages=2781-2794&author=J.+E.+Jangauthor=J.+I.+Eomauthor=H.+K.+Jeungauthor=J.+W.+Cheongauthor=J.+Y.+Leeauthor=J.+S.+Kimauthor=Y.+H.+Min&title=AMPK-ULK1-mediated+autophagy+confers+resistance+to+BET+Inhibitor+JQ1+in+acute+myeloid+leukemia+stem+cells&doi=10.1158%2F1078-0432.CCR-16-1903"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">AMPK-ULK1-Mediated Autophagy Confers Resistance to BET Inhibitor JQ1 in Acute Myeloid Leukemia Stem Cells</span></div><div class="casAuthors">Jang, Ji Eun; Eom, Ju-In; Jeung, Hoi-Kyung; Cheong, June-Won; Lee, Jung Yeon; Kim, Jin Seok; Min, Yoo Hong</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2781-2794</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Purpose: Bromodomain and extraterminal domain (BET) inhibitors are promising epigenetic agents for the treatment of various subsets of acute myeloid leukemia (AML).  However, the resistance of leukemia stem cells (LSC) to BET inhibitors remains a major challenge.  In this study, we evaluated the mechanisms underlying LSC resistance to the BET inhibitor JQ1.  Exptl. Design: We evaluated the levels of apoptosis and autophagy induced by JQ1 in LSC-like leukemia cell lines and primary CD34+CD38- leukemic blasts obtained from AML cases with normal karyotype without recurrent mutations.  Results: JQ1 effectively induced apoptosis in a concn.-dependent manner in JQ1-sensitive AML cells.  However, in JQ1-resistant AML LSCs, JQ1 induced little apoptosis and led to upregulation of beclin-1, increased LC3-II lipidation, formation of autophagosomes, and downregulation of p62/SQSTM1.  Inhibition of autophagy by pharmacol. inhibitors or knockdown of beclin-1 using specific siRNA enhanced JQ1-induced apoptosis in resistant cells, indicating that prosurvival autophagy occurred in these cells.  Independent of mTOR signaling, activation of the AMPK (pThr172)/ULK1 (pSer555) pathway was found to be assocd. with JQ1-induced autophagy in resistant cells.  AMPK inhibition using the pharmacol. inhibitor compd. C or by knockdown of AMPKα suppressed autophagy and promoted JQ1-induced apoptosis in AML LSCs.  Conclusions: These findings revealed that prosurvival autophagy was one of the mechanisms involved in the resistance AML LSCs to JQ1.  Targeting the AMPK/ULK1 pathway or inhibition of autophagy could be an effective therapeutic strategy for combating resistance to BET inhibitors in AML and other types of cancer.  Clin Cancer Res; 23(11); 2781-94. 2016 AACR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqHOzsCo6fOvLVg90H21EOLACvtfcHk0liVM1XGfv45cQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXpt1Sms7o%253D&md5=f15df13e0addda969161b31973857176</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1158%2F1078-0432.CCR-16-1903&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1078-0432.CCR-16-1903%26sid%3Dliteratum%253Aachs%26aulast%3DJang%26aufirst%3DJ.%2BE.%26aulast%3DEom%26aufirst%3DJ.%2BI.%26aulast%3DJeung%26aufirst%3DH.%2BK.%26aulast%3DCheong%26aufirst%3DJ.%2BW.%26aulast%3DLee%26aufirst%3DJ.%2BY.%26aulast%3DKim%26aufirst%3DJ.%2BS.%26aulast%3DMin%26aufirst%3DY.%2BH.%26atitle%3DAMPK-ULK1-mediated%2520autophagy%2520confers%2520resistance%2520to%2520BET%2520Inhibitor%2520JQ1%2520in%2520acute%2520myeloid%2520leukemia%2520stem%2520cells%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D2017%26volume%3D23%26spage%3D2781%26epage%3D2794%26doi%3D10.1158%2F1078-0432.CCR-16-1903" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ren, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ye, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pan, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, H.</span></span> <span> </span><span class="NLM_article-title">Intrinsic BET inhibitor resistance in SPOP-mutated prostate cancer is mediated by BET protein stabilization and AKT-mTORC1 activation</span>. <i>Nat. Med.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">1055</span>– <span class="NLM_lpage">1062</span>, <span class="refDoi"> DOI: 10.1038/nm.4379</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1038%2Fnm.4379" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=28805822" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtlKjtbbO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2017&pages=1055-1062&author=P.+Zhangauthor=D.+Wangauthor=Y.+Zhaoauthor=S.+Renauthor=K.+Gaoauthor=Z.+Yeauthor=S.+Wangauthor=C.+W.+Panauthor=Y.+Zhuauthor=Y.+Yanauthor=Y.+Yangauthor=D.+Wuauthor=Y.+Heauthor=J.+Zhangauthor=D.+Luauthor=X.+Liuauthor=L.+Yuauthor=S.+Zhaoauthor=Y.+Liauthor=D.+Linauthor=Y.+Wangauthor=L.+Wangauthor=Y.+Chenauthor=Y.+Sunauthor=C.+Wangauthor=H.+Huang&title=Intrinsic+BET+inhibitor+resistance+in+SPOP-mutated+prostate+cancer+is+mediated+by+BET+protein+stabilization+and+AKT-mTORC1+activation&doi=10.1038%2Fnm.4379"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Intrinsic BET inhibitor resistance in SPOP-mutated prostate cancer is mediated by BET protein stabilization and AKT-mTORC1 activation</span></div><div class="casAuthors">Zhang, Pingzhao; Wang, Dejie; Zhao, Yu; Ren, Shancheng; Gao, Kun; Ye, Zhenqing; Wang, Shangqian; Pan, Chun-Wu; Zhu, Yasheng; Yan, Yuqian; Yang, Yinhui; Wu, Di; He, Yundong; Zhang, Jun; Lu, Daru; Liu, Xiuping; Yu, Long; Zhao, Shimin; Li, Yao; Lin, Dong; Wang, Yuzhuo; Wang, Liguo; Chen, Yu; Sun, Yinghao; Wang, Chenji; Huang, Haojie</div><div class="citationInfo"><span class="NLM_cas:title">Nature Medicine (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1055-1062</span>CODEN:
                <span class="NLM_cas:coden">NAMEFI</span>;
        ISSN:<span class="NLM_cas:issn">1078-8956</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Bromodomain and extraterminal domain (BET) protein inhibitors are emerging as promising anticancer therapies.  The gene encoding the E3 ubiquitin ligase substrate-binding adaptor speckle-type POZ protein (SPOP) is the most frequently mutated in primary prostate cancer.  Here we demonstrate that wild-type SPOP binds to and induces ubiquitination and proteasomal degrdn. of BET proteins (BRD2, BRD3 and BRD4) by recognizing a degron motif common among them.  In contrast, prostate cancer-assocd. SPOP mutants show impaired binding to BET proteins, resulting in decreased proteasomal degrdn. and accumulation of these proteins in prostate cancer cell lines and patient specimens and causing resistance to BET inhibitors.  Transcriptome and BRD4 cistrome analyses reveal enhanced expression of the GTPase RAC1 and cholesterol-biosynthesis-assocd. genes together with activation of AKT-mTORC1 signaling as a consequence of BRD4 stabilization.  Our data show that resistance to BET inhibitors in SPOP-mutant prostate cancer can be overcome by combination with AKT inhibitors and further support the evaluation of SPOP mutations as biomarkers to guide BET-inhibitor-oriented therapy in patients with prostate cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoJ-PBm8f3OarVg90H21EOLACvtfcHk0lilR0hO-xkxYQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtlKjtbbO&md5=01e4c46e7e5334055a81205c5dac9057</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1038%2Fnm.4379&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnm.4379%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DP.%26aulast%3DWang%26aufirst%3DD.%26aulast%3DZhao%26aufirst%3DY.%26aulast%3DRen%26aufirst%3DS.%26aulast%3DGao%26aufirst%3DK.%26aulast%3DYe%26aufirst%3DZ.%26aulast%3DWang%26aufirst%3DS.%26aulast%3DPan%26aufirst%3DC.%2BW.%26aulast%3DZhu%26aufirst%3DY.%26aulast%3DYan%26aufirst%3DY.%26aulast%3DYang%26aufirst%3DY.%26aulast%3DWu%26aufirst%3DD.%26aulast%3DHe%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DLu%26aufirst%3DD.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DYu%26aufirst%3DL.%26aulast%3DZhao%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DLin%26aufirst%3DD.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DSun%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DC.%26aulast%3DHuang%26aufirst%3DH.%26atitle%3DIntrinsic%2520BET%2520inhibitor%2520resistance%2520in%2520SPOP-mutated%2520prostate%2520cancer%2520is%2520mediated%2520by%2520BET%2520protein%2520stabilization%2520and%2520AKT-mTORC1%2520activation%26jtitle%3DNat.%2520Med.%26date%3D2017%26volume%3D23%26spage%3D1055%26epage%3D1062%26doi%3D10.1038%2Fnm.4379" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref54"><div class="reference"><strong class="refLabel"><a href="#ref54" class="rightTabRefNumLink">54</a></strong><div class="NLM_citation" id="rightTab-cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dai, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gan, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhong, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shimizu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beca, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blattner, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vasudevan, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buckley, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buser, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Inuzuka, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beck, A. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wild, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garraway, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rubin, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barbieri, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, K. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muthuswamy, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, W.</span></span> <span> </span><span class="NLM_article-title">Prostate cancer-associated SPOP mutations confer resistance to BET inhibitors through stabilization of BRD4</span>. <i>Nat. Med.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">1063</span>– <span class="NLM_lpage">1071</span>, <span class="refDoi"> DOI: 10.1038/nm.4378</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1038%2Fnm.4378" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=28805820" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtlKjtbnE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2017&pages=1063-1071&author=X.+Daiauthor=W.+Ganauthor=X.+Liauthor=S.+Wangauthor=W.+Zhangauthor=L.+Huangauthor=S.+Liuauthor=Q.+Zhongauthor=J.+Guoauthor=J.+Zhangauthor=T.+Chenauthor=K.+Shimizuauthor=F.+Becaauthor=M.+Blattnerauthor=D.+Vasudevanauthor=D.+L.+Buckleyauthor=J.+Qiauthor=L.+Buserauthor=P.+Liuauthor=H.+Inuzukaauthor=A.+H.+Beckauthor=L.+Wangauthor=P.+J.+Wildauthor=L.+A.+Garrawayauthor=M.+A.+Rubinauthor=C.+E.+Barbieriauthor=K.+K.+Wongauthor=S.+K.+Muthuswamyauthor=J.+Huangauthor=Y.+Chenauthor=J.+E.+Bradnerauthor=W.+Wei&title=Prostate+cancer-associated+SPOP+mutations+confer+resistance+to+BET+inhibitors+through+stabilization+of+BRD4&doi=10.1038%2Fnm.4378"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Prostate cancer-associated SPOP mutations confer resistance to BET inhibitors through stabilization of BRD4</span></div><div class="casAuthors">Dai, Xiangpeng; Gan, Wenjian; Li, Xiaoning; Wang, Shangqian; Zhang, Wei; Huang, Ling; Liu, Shengwu; Zhong, Qing; Guo, Jianping; Zhang, Jinfang; Chen, Ting; Shimizu, Kouhei; Beca, Francisco; Blattner, Mirjam; Vasudevan, Divya; Buckley, Dennis L.; Qi, Jun; Buser, Lorenz; Liu, Pengda; Inuzuka, Hiroyuki; Beck, Andrew H.; Wang, Liewei; Wild, Peter J.; Garraway, Levi A.; Rubin, Mark A.; Barbieri, Christopher E.; Wong, Kwok-Kin; Muthuswamy, Senthil K.; Huang, Jiaoti; Chen, Yu; Bradner, James E.; Wei, Wenyi</div><div class="citationInfo"><span class="NLM_cas:title">Nature Medicine (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1063-1071</span>CODEN:
                <span class="NLM_cas:coden">NAMEFI</span>;
        ISSN:<span class="NLM_cas:issn">1078-8956</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">The bromodomain and extraterminal (BET) family of proteins comprises four members-BRD2, BRD3, BRD4 and the testis-specific isoform BRDT-that largely function as transcriptional coactivators and play crit. roles in various cellular processes, including the cell cycle, apoptosis, migration and invasion.  BET proteins enhance the oncogenic functions of major cancer drivers by elevating the expression of these drivers, such as c-Myc in leukemia, or by promoting the transcriptional activities of oncogenic factors, such as AR and ERG in prostate cancer.  Pathol., BET proteins are frequently overexpressed and are clin. linked to various types of human cancer; they are therefore being pursued as attractive therapeutic targets for selective inhibition in patients with cancer.  To this end, a no. of bromodomain inhibitors, including JQ1 and I-BET, have been developed and have shown promising outcomes in early clin. trials.  Although resistance to BET inhibitors has been documented in preclin. models, the mol. mechanisms underlying acquired resistance are largely unknown.  Here we report that cullin-3SPOP earmarks BET proteins, including BRD2, BRD3 and BRD4, for ubiquitination-mediated degrdn.  Pathol., prostate cancer-assocd. SPOP mutants fail to interact with and promote the degrdn. of BET proteins, leading to their elevated abundance in SPOP-mutant prostate cancer.  As a result, prostate cancer cell lines and organoids derived from individuals harboring SPOP mutations are more resistant to BET-inhibitor-induced cell growth arrest and apoptosis.  Therefore, our results elucidate the tumor-suppressor role of SPOP in prostate cancer in which it acts as a neg. regulator of BET protein stability and also provide a mol. mechanism for resistance to BET inhibitors in individuals with prostate cancer bearing SPOP mutations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqDt_5oFy4JlLVg90H21EOLACvtfcHk0liBEi37lb4Ykw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtlKjtbnE&md5=dbbb73a496d18b8310d675cd2503853f</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1038%2Fnm.4378&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnm.4378%26sid%3Dliteratum%253Aachs%26aulast%3DDai%26aufirst%3DX.%26aulast%3DGan%26aufirst%3DW.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DS.%26aulast%3DZhang%26aufirst%3DW.%26aulast%3DHuang%26aufirst%3DL.%26aulast%3DLiu%26aufirst%3DS.%26aulast%3DZhong%26aufirst%3DQ.%26aulast%3DGuo%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DT.%26aulast%3DShimizu%26aufirst%3DK.%26aulast%3DBeca%26aufirst%3DF.%26aulast%3DBlattner%26aufirst%3DM.%26aulast%3DVasudevan%26aufirst%3DD.%26aulast%3DBuckley%26aufirst%3DD.%2BL.%26aulast%3DQi%26aufirst%3DJ.%26aulast%3DBuser%26aufirst%3DL.%26aulast%3DLiu%26aufirst%3DP.%26aulast%3DInuzuka%26aufirst%3DH.%26aulast%3DBeck%26aufirst%3DA.%2BH.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DWild%26aufirst%3DP.%2BJ.%26aulast%3DGarraway%26aufirst%3DL.%2BA.%26aulast%3DRubin%26aufirst%3DM.%2BA.%26aulast%3DBarbieri%26aufirst%3DC.%2BE.%26aulast%3DWong%26aufirst%3DK.%2BK.%26aulast%3DMuthuswamy%26aufirst%3DS.%2BK.%26aulast%3DHuang%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26aulast%3DWei%26aufirst%3DW.%26atitle%3DProstate%2520cancer-associated%2520SPOP%2520mutations%2520confer%2520resistance%2520to%2520BET%2520inhibitors%2520through%2520stabilization%2520of%2520BRD4%26jtitle%3DNat.%2520Med.%26date%3D2017%26volume%3D23%26spage%3D1063%26epage%3D1071%26doi%3D10.1038%2Fnm.4378" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref55"><div class="reference"><strong class="refLabel"><a href="#ref55" class="rightTabRefNumLink">55</a></strong><div class="NLM_citation" id="rightTab-cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Janouskova, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">El Tekle, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bellini, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Udeshi, N. D</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rinaldi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ulbricht, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernasocchi, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Civenni, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Losa, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Svinkina, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bielski, C. M</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kryukov, G. V</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cascione, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Napoli, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Enchev, R. I</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mutch, D. G</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carney, M. E</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berchuck, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winterhoff, B. J N</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Broaddus, R. R</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schraml, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moch, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bertoni, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Catapano, C. V</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peter, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carr, S. A</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garraway, L. A</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wild, P. J</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Theurillat, J.-P. P</span></span> <span> </span><span class="NLM_article-title">Opposing effects of cancer-type-specific SPOP mutants on BET protein degradation and sensitivity to BET inhibitors</span>. <i>Nat. Med.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">1046</span>– <span class="NLM_lpage">1054</span>, <span class="refDoi"> DOI: 10.1038/nm.4372</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1038%2Fnm.4372" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=28805821" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtlKjtbbN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2017&pages=1046-1054&author=H.+Janouskovaauthor=G.+El+Tekleauthor=E.+Belliniauthor=N.+D+Udeshiauthor=A.+Rinaldiauthor=A.+Ulbrichtauthor=T.+Bernasocchiauthor=G.+Civenniauthor=M.+Losaauthor=T.+Svinkinaauthor=C.+M+Bielskiauthor=G.+V+Kryukovauthor=L.+Cascioneauthor=S.+Napoliauthor=R.+I+Enchevauthor=D.+G+Mutchauthor=M.+E+Carneyauthor=A.+Berchuckauthor=B.+J+N+Winterhoffauthor=R.+R+Broaddusauthor=P.+Schramlauthor=H.+Mochauthor=F.+Bertoniauthor=C.+V+Catapanoauthor=M.+Peterauthor=S.+A+Carrauthor=L.+A+Garrawayauthor=P.+J+Wildauthor=J.-P.+P+Theurillat&title=Opposing+effects+of+cancer-type-specific+SPOP+mutants+on+BET+protein+degradation+and+sensitivity+to+BET+inhibitors&doi=10.1038%2Fnm.4372"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Opposing effects of cancer-type-specific SPOP mutants on BET protein degradation and sensitivity to BET inhibitors</span></div><div class="casAuthors">Janouskova, Hana; El Tekle, Geniver; Bellini, Elisa; Udeshi, Namrata D.; Rinaldi, Anna; Ulbricht, Anna; Bernasocchi, Tiziano; Civenni, Gianluca; Losa, Marco; Svinkina, Tanya; Bielski, Craig M.; Kryukov, Gregory V.; Cascione, Luciano; Napoli, Sara; Enchev, Radoslav I.; Mutch, David G.; Carney, Michael E.; Berchuck, Andrew; Winterhoff, Boris J. N.; Broaddus, Russell R.; Schraml, Peter; Moch, Holger; Bertoni, Francesco; Catapano, Carlo V.; Peter, Matthias; Carr, Steven A.; Garraway, Levi A.; Wild, Peter J.; Theurillat, Jean-Philippe P.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Medicine (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1046-1054</span>CODEN:
                <span class="NLM_cas:coden">NAMEFI</span>;
        ISSN:<span class="NLM_cas:issn">1078-8956</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">It is generally assumed that recurrent mutations within a given cancer driver gene elicit similar drug responses.  Cancer genome studies have identified recurrent but divergent missense mutations affecting the substrate-recognition domain of the ubiquitin ligase adaptor SPOP in endometrial and prostate cancers.  The therapeutic implications of these mutations remain incompletely understood.  Here we analyzed changes in the ubiquitin landscape induced by endometrial cancer-assocd. SPOP mutations and identified BRD2, BRD3 and BRD4 proteins (BETs) as SPOP-CUL3 substrates that are preferentially degraded by endometrial cancer-assocd. SPOP mutants.  The resulting redn. of BET protein levels sensitized cancer cells to BET inhibitors.  Conversely, prostate cancer-specific SPOP mutations resulted in impaired degrdn. of BETs, promoting their resistance to pharmacol. inhibition.  These results uncover an oncogenomics paradox, whereby mutations mapping to the same domain evoke opposing drug susceptibilities.  Specifically, we provide a mol. rationale for the use of BET inhibitors to treat patients with endometrial but not prostate cancer who harbor SPOP mutations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq_yEfPkA6U9rVg90H21EOLACvtfcHk0liBEi37lb4Ykw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtlKjtbbN&md5=cf43e77dd3d6ed7c3a2fb4efca9167cb</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1038%2Fnm.4372&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnm.4372%26sid%3Dliteratum%253Aachs%26aulast%3DJanouskova%26aufirst%3DH.%26aulast%3DEl%2BTekle%26aufirst%3DG.%26aulast%3DBellini%26aufirst%3DE.%26aulast%3DUdeshi%26aufirst%3DN.%2BD%26aulast%3DRinaldi%26aufirst%3DA.%26aulast%3DUlbricht%26aufirst%3DA.%26aulast%3DBernasocchi%26aufirst%3DT.%26aulast%3DCivenni%26aufirst%3DG.%26aulast%3DLosa%26aufirst%3DM.%26aulast%3DSvinkina%26aufirst%3DT.%26aulast%3DBielski%26aufirst%3DC.%2BM%26aulast%3DKryukov%26aufirst%3DG.%2BV%26aulast%3DCascione%26aufirst%3DL.%26aulast%3DNapoli%26aufirst%3DS.%26aulast%3DEnchev%26aufirst%3DR.%2BI%26aulast%3DMutch%26aufirst%3DD.%2BG%26aulast%3DCarney%26aufirst%3DM.%2BE%26aulast%3DBerchuck%26aufirst%3DA.%26aulast%3DWinterhoff%26aufirst%3DB.%2BJ%2BN%26aulast%3DBroaddus%26aufirst%3DR.%2BR%26aulast%3DSchraml%26aufirst%3DP.%26aulast%3DMoch%26aufirst%3DH.%26aulast%3DBertoni%26aufirst%3DF.%26aulast%3DCatapano%26aufirst%3DC.%2BV%26aulast%3DPeter%26aufirst%3DM.%26aulast%3DCarr%26aufirst%3DS.%2BA%26aulast%3DGarraway%26aufirst%3DL.%2BA%26aulast%3DWild%26aufirst%3DP.%2BJ%26aulast%3DTheurillat%26aufirst%3DJ.-P.%2BP%26atitle%3DOpposing%2520effects%2520of%2520cancer-type-specific%2520SPOP%2520mutants%2520on%2520BET%2520protein%2520degradation%2520and%2520sensitivity%2520to%2520BET%2520inhibitors%26jtitle%3DNat.%2520Med.%26date%3D2017%26volume%3D23%26spage%3D1046%26epage%3D1054%26doi%3D10.1038%2Fnm.4372" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref56"><div class="reference"><strong class="refLabel"><a href="#ref56" class="rightTabRefNumLink">56</a></strong><div class="NLM_citation" id="rightTab-cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jin, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ye, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jimenez, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, H.</span></span> <span> </span><span class="NLM_article-title">DUB3 promotes BET inhibitor resistance and cancer progression by deubiquitinating BRD4</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>71</i></span>,  <span class="NLM_fpage">592</span>– <span class="NLM_lpage">605</span>, <span class="refDoi"> DOI: 10.1016/j.molcel.2018.06.036</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1016%2Fj.molcel.2018.06.036" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=30057199" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhsVSmsb%252FO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=71&publication_year=2018&pages=592-605&author=X.+Jinauthor=Y.+Yanauthor=D.+Wangauthor=D.+Dingauthor=T.+Maauthor=Z.+Yeauthor=R.+Jimenezauthor=L.+Wangauthor=H.+Wuauthor=H.+Huang&title=DUB3+promotes+BET+inhibitor+resistance+and+cancer+progression+by+deubiquitinating+BRD4&doi=10.1016%2Fj.molcel.2018.06.036"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">DUB3 Promotes BET Inhibitor Resistance and Cancer Progression by Deubiquitinating BRD4</span></div><div class="casAuthors">Jin, Xin; Yan, Yuqian; Wang, Dejie; Ding, Donglin; Ma, Tao; Ye, Zhenqing; Jimenez, Rafael; Wang, Liguo; Wu, Heshui; Huang, Haojie</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">71</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">592-605.e4</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">The bromodomain and extra-terminal domain (BET) protein BRD4 is emerging as a promising anticancer therapeutic target.  However, resistance to BET inhibitors often occurs, and it has been linked to aberrant degrdn. of BRD4 protein in cancer.  Here, we demonstrate that the deubiquitinase DUB3 binds to BRD4 and promotes its deubiquitination and stabilization.  Expression of DUB3 is transcriptionally repressed by the NCOR2-HDAC10 complex.  The NCOR2 gene is frequently deleted in castration-resistant prostate cancer patient specimens, and loss of NCOR2 induces elevation of DUB3 and BRD4 proteins in cancer cells.  DUB3-proficient prostate cancer cells are resistant to the BET inhibitor JQ1 in vitro and in mice, but this effect is diminished by DUB3 inhibitory agents such as CDK4/6 inhibitor in a RB-independent manner.  Our findings identify a previously unrecognized mechanism causing BRD4 upregulation and drug resistance, suggesting that DUB3 is a viable therapeutic target to overcome BET inhibitor resistance in cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGozU4LZMIj7-LVg90H21EOLACvtfcHk0lg-8tKEGLj6bA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhsVSmsb%252FO&md5=45ae817ec1b9ac71dc2402b4618a1794</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2018.06.036&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2018.06.036%26sid%3Dliteratum%253Aachs%26aulast%3DJin%26aufirst%3DX.%26aulast%3DYan%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DD.%26aulast%3DDing%26aufirst%3DD.%26aulast%3DMa%26aufirst%3DT.%26aulast%3DYe%26aufirst%3DZ.%26aulast%3DJimenez%26aufirst%3DR.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DWu%26aufirst%3DH.%26aulast%3DHuang%26aufirst%3DH.%26atitle%3DDUB3%2520promotes%2520BET%2520inhibitor%2520resistance%2520and%2520cancer%2520progression%2520by%2520deubiquitinating%2520BRD4%26jtitle%3DMol.%2520Cell%26date%3D2018%26volume%3D71%26spage%3D592%26epage%3D605%26doi%3D10.1016%2Fj.molcel.2018.06.036" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref57"><div class="reference"><strong class="refLabel"><a href="#ref57" class="rightTabRefNumLink">57</a></strong><div class="NLM_citation" id="rightTab-cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Marcotte, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sayad, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanchez-Garcia, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reimand, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haider, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Virtanen, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bader, G. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mills, G. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pe’Er, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moffat, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neel, B. G.</span></span> <span> </span><span class="NLM_article-title">Functional genomic landscape of human breast cancer drivers, vulnerabilities, and resistance</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>164</i></span>,  <span class="NLM_fpage">293</span>– <span class="NLM_lpage">309</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2015.11.062</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1016%2Fj.cell.2015.11.062" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=26771497" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtFShtr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=164&publication_year=2016&pages=293-309&author=R.+Marcotteauthor=A.+Sayadauthor=K.+R.+Brownauthor=F.+Sanchez-Garciaauthor=J.+Reimandauthor=M.+Haiderauthor=C.+Virtanenauthor=J.+E.+Bradnerauthor=G.+D.+Baderauthor=G.+B.+Millsauthor=D.+Pe%E2%80%99Erauthor=J.+Moffatauthor=B.+G.+Neel&title=Functional+genomic+landscape+of+human+breast+cancer+drivers%2C+vulnerabilities%2C+and+resistance&doi=10.1016%2Fj.cell.2015.11.062"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">Functional Genomic Landscape of Human Breast Cancer Drivers, Vulnerabilities, and Resistance</span></div><div class="casAuthors">Marcotte, Richard; Sayad, Azin; Brown, Kevin R.; Sanchez-Garcia, Felix; Reimand, Juri; Haider, Maliha; Virtanen, Carl; Bradner, James E.; Bader, Gary D.; Mills, Gordon B.; Pe'er, Dana; Moffat, Jason; Neel, Benjamin G.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">164</span>
        (<span class="NLM_cas:issue">1-2</span>),
    <span class="NLM_cas:pages">293-309</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Large-scale genomic studies have identified multiple somatic aberrations in breast cancer, including copy no. alterations and point mutations.  Still, identifying causal variants and emergent vulnerabilities that arise as a consequence of genetic alterations remain major challenges.  We performed whole-genome small hairpin RNA (shRNA) "dropout screens" on 77 breast cancer cell lines.  Using a hierarchical linear regression algorithm to score our screen results and integrate them with accompanying detailed genetic and proteomic information, we identify vulnerabilities in breast cancer, including candidate "drivers," and reveal general functional genomic properties of cancer cells.  Comparisons of gene essentiality with drug sensitivity data suggest potential resistance mechanisms, effects of existing anti-cancer drugs, and opportunities for combination therapy.  Finally, we demonstrate the utility of this large dataset by identifying BRD4 as a potential target in luminal breast cancer and PIK3CA mutations as a resistance determinant for BET-inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrc0920t-4Go7Vg90H21EOLACvtfcHk0lg-8tKEGLj6bA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtFShtr0%253D&md5=ebb0a807e744c8fefea76bb4f100fbca</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2015.11.062&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2015.11.062%26sid%3Dliteratum%253Aachs%26aulast%3DMarcotte%26aufirst%3DR.%26aulast%3DSayad%26aufirst%3DA.%26aulast%3DBrown%26aufirst%3DK.%2BR.%26aulast%3DSanchez-Garcia%26aufirst%3DF.%26aulast%3DReimand%26aufirst%3DJ.%26aulast%3DHaider%26aufirst%3DM.%26aulast%3DVirtanen%26aufirst%3DC.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26aulast%3DBader%26aufirst%3DG.%2BD.%26aulast%3DMills%26aufirst%3DG.%2BB.%26aulast%3DPe%25E2%2580%2599Er%26aufirst%3DD.%26aulast%3DMoffat%26aufirst%3DJ.%26aulast%3DNeel%26aufirst%3DB.%2BG.%26atitle%3DFunctional%2520genomic%2520landscape%2520of%2520human%2520breast%2520cancer%2520drivers%252C%2520vulnerabilities%252C%2520and%2520resistance%26jtitle%3DCell%26date%3D2016%26volume%3D164%26spage%3D293%26epage%3D309%26doi%3D10.1016%2Fj.cell.2015.11.062" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref58"><div class="reference"><strong class="refLabel"><a href="#ref58" class="rightTabRefNumLink">58</a></strong><div class="NLM_citation" id="rightTab-cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cochran, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Conery, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sims, R. R.</span></span> <span> </span><span class="NLM_article-title">Bromodomains: a new target class for drug development</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">609</span>– <span class="NLM_lpage">628</span>, <span class="refDoi"> DOI: 10.1038/s41573-019-0030-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1038%2Fs41573-019-0030-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=31273347" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhtlWitbnL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2019&pages=609-628&author=A.+G.+Cochranauthor=A.+R.+Coneryauthor=R.+R.+Sims&title=Bromodomains%3A+a+new+target+class+for+drug+development&doi=10.1038%2Fs41573-019-0030-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">Bromodomains: a new target class for drug development</span></div><div class="casAuthors">Cochran, Andrea G.; Conery, Andrew R.; Sims, III, Robert J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">609-628</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Less than a decade ago, it was shown that bromodomains, acetyl lysine 'reader' modules found in proteins with varied functions, were highly tractable small-mol. targets.  This is an unusual property for protein-protein or protein-peptide interaction domains, and it prompted a wave of chem. probe discovery to understand the biol. potential of new agents that targeted bromodomains.  The original examples, inhibitors of the bromodomain and extra-terminal (BET) class of bromodomains, showed enticing anti-inflammatory and anticancer activities, and several compds. have since advanced to human clin. trials.  Here, we review the current state of BET inhibitor biol. in relation to clin. development, and we discuss the next wave of bromodomain inhibitors with clin. potential in oncol. and non-oncol. indications.  The lessons learned from BET inhibitor programs should affect efforts to develop drugs that target non-BET bromodomains and other epigenetic readers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr6QwxqowW9NbVg90H21EOLACvtfcHk0lhKLzzkCb5LyQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhtlWitbnL&md5=88e56eec8edeb7c325d28156ec131ed8</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1038%2Fs41573-019-0030-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41573-019-0030-7%26sid%3Dliteratum%253Aachs%26aulast%3DCochran%26aufirst%3DA.%2BG.%26aulast%3DConery%26aufirst%3DA.%2BR.%26aulast%3DSims%26aufirst%3DR.%2BR.%26atitle%3DBromodomains%253A%2520a%2520new%2520target%2520class%2520for%2520drug%2520development%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2019%26volume%3D18%26spage%3D609%26epage%3D628%26doi%3D10.1038%2Fs41573-019-0030-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref59"><div class="reference"><strong class="refLabel"><a href="#ref59" class="rightTabRefNumLink">59</a></strong><div class="NLM_citation" id="rightTab-cit59"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rodriguez, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerona-Navarro, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Osman, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, M. M.</span></span> <span> </span><span class="NLM_article-title">In silico design and molecular basis for the selectivity of Olinone toward the first over the second bromodomain of BRD4</span>. <i>Proteins: Struct., Funct., Genet.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>88</i></span>,  <span class="NLM_fpage">414</span>– <span class="NLM_lpage">430</span>, <span class="refDoi"> DOI: 10.1002/prot.25818</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1002%2Fprot.25818" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=31587361" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=88&publication_year=2020&pages=414-430&author=Y.+Rodriguezauthor=G.+Gerona-Navarroauthor=R.+Osmanauthor=M.+M.+Zhou&title=In+silico+design+and+molecular+basis+for+the+selectivity+of+Olinone+toward+the+first+over+the+second+bromodomain+of+BRD4&doi=10.1002%2Fprot.25818"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1002%2Fprot.25818&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fprot.25818%26sid%3Dliteratum%253Aachs%26aulast%3DRodriguez%26aufirst%3DY.%26aulast%3DGerona-Navarro%26aufirst%3DG.%26aulast%3DOsman%26aufirst%3DR.%26aulast%3DZhou%26aufirst%3DM.%2BM.%26atitle%3DIn%2520silico%2520design%2520and%2520molecular%2520basis%2520for%2520the%2520selectivity%2520of%2520Olinone%2520toward%2520the%2520first%2520over%2520the%2520second%2520bromodomain%2520of%2520BRD4%26jtitle%3DProteins%253A%2520Struct.%252C%2520Funct.%252C%2520Genet.%26date%3D2020%26volume%3D88%26spage%3D414%26epage%3D430%26doi%3D10.1002%2Fprot.25818" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref60"><div class="reference"><strong class="refLabel"><a href="#ref60" class="rightTabRefNumLink">60</a></strong><div class="NLM_citation" id="rightTab-cit60"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Divakaran, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Talluri, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ayoub, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mishra, N. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cui, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Widen, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berndt, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, J. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carlson, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Topczewski, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schonbrunn, E. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harki, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pomerantz, W.</span></span> <span> </span><span class="NLM_article-title">Molecular basis for the N-terminal bromodomain-and-extra-terminal-family selectivity of a dual kinase-bromodomain inhibitor</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">9316</span>– <span class="NLM_lpage">9334</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b01248</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b01248" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhvVSis7fM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=9316-9334&author=A.+Divakaranauthor=S.+K.+Talluriauthor=A.+M.+Ayoubauthor=N.+K.+Mishraauthor=H.+Cuiauthor=J.+C.+Widenauthor=N.+Berndtauthor=J.+Y.+Zhuauthor=A.+S.+Carlsonauthor=J.+J.+Topczewskiauthor=E.+K.+Schonbrunnauthor=D.+A.+Harkiauthor=W.+Pomerantz&title=Molecular+basis+for+the+N-terminal+bromodomain-and-extra-terminal-family+selectivity+of+a+dual+kinase-bromodomain+inhibitor&doi=10.1021%2Facs.jmedchem.8b01248"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular Basis for the N-Terminal Bromodomain-and-Extra-Terminal-Family Selectivity of a Dual Kinase-Bromodomain Inhibitor</span></div><div class="casAuthors">Divakaran, Anand; Talluri, Siva K.; Ayoub, Alex M.; Mishra, Neeraj K.; Cui, Huarui; Widen, John C.; Berndt, Norbert; Zhu, Jin-Yi; Carlson, Angela S.; Topczewski, Joseph J.; Schonbrunn, Ernst K.; Harki, Daniel A.; Pomerantz, William C. K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">9316-9334</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">As regulators of transcription, epigenetic proteins that interpret post-translational modifications to N-terminal histone tails are essential for maintaining cellular homeostasis.  When dysregulated, "reader" proteins become drivers of disease.  In the case of bromodomains, which recognize N-ε-acetylated lysine, selective inhibition of individual bromodomain-and-extra-terminal (BET)-family bromodomains has proven challenging.  We describe the >55-fold N-terminal-BET bromodomain selectivity of 1,4,5-trisubstituted-imidazole dual kinase-bromodomain inhibitors.  Selectivity for the BRD4 N-terminal bromodomain (BRD4(1)) over its second bromodomain (BRD4(2)) arises from the displacement of ordered waters and the conformational flexibility of lysine-141 in BRD4(1).  Cellular efficacy was demonstrated via redn. of c-Myc expression, inhibition of NF-κB signaling, and suppression of IL-8 prodn. through potential synergistic inhibition of BRD4(1) and p38α.  These dual inhibitors provide a new scaffold for domain-selective inhibition of BRD4, the aberrant function of which plays a key role in cancer and inflammatory signaling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpZhnL9cPdDi7Vg90H21EOLACvtfcHk0ljT83-06wgi6w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhvVSis7fM&md5=5176ef754a898b3e03fc56e2662ab399</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b01248&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b01248%26sid%3Dliteratum%253Aachs%26aulast%3DDivakaran%26aufirst%3DA.%26aulast%3DTalluri%26aufirst%3DS.%2BK.%26aulast%3DAyoub%26aufirst%3DA.%2BM.%26aulast%3DMishra%26aufirst%3DN.%2BK.%26aulast%3DCui%26aufirst%3DH.%26aulast%3DWiden%26aufirst%3DJ.%2BC.%26aulast%3DBerndt%26aufirst%3DN.%26aulast%3DZhu%26aufirst%3DJ.%2BY.%26aulast%3DCarlson%26aufirst%3DA.%2BS.%26aulast%3DTopczewski%26aufirst%3DJ.%2BJ.%26aulast%3DSchonbrunn%26aufirst%3DE.%2BK.%26aulast%3DHarki%26aufirst%3DD.%2BA.%26aulast%3DPomerantz%26aufirst%3DW.%26atitle%3DMolecular%2520basis%2520for%2520the%2520N-terminal%2520bromodomain-and-extra-terminal-family%2520selectivity%2520of%2520a%2520dual%2520kinase-bromodomain%2520inhibitor%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D9316%26epage%3D9334%26doi%3D10.1021%2Facs.jmedchem.8b01248" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref61"><div class="reference"><strong class="refLabel"><a href="#ref61" class="rightTabRefNumLink">61</a></strong><div class="NLM_citation" id="rightTab-cit61"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Niu, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alamer, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fan, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Endsley, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gelman, B. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tian, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michael, N. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robb, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ananworanich, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, H.</span></span> <span> </span><span class="NLM_article-title">Structure-guided drug design identifies a BRD4-selective small molecule that suppresses HIV.</span>. <i>J. Clin. Invest.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>129</i></span>,  <span class="NLM_fpage">3361</span>– <span class="NLM_lpage">3373</span>, <span class="refDoi"> DOI: 10.1172/JCI120633</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1172%2FJCI120633" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=31329163" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A280%3ADC%252BB3Mzps1ynsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=129&publication_year=2019&pages=3361-3373&author=Q.+Niuauthor=Z.+Liuauthor=E.+Alamerauthor=X.+Fanauthor=H.+Chenauthor=J.+Endsleyauthor=B.+B.+Gelmanauthor=B.+Tianauthor=J.+H.+Kimauthor=N.+L.+Michaelauthor=M.+L.+Robbauthor=J.+Ananworanichauthor=J.+Zhouauthor=H.+Hu&title=Structure-guided+drug+design+identifies+a+BRD4-selective+small+molecule+that+suppresses+HIV.&doi=10.1172%2FJCI120633"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-guided drug design identifies a BRD4-selective small molecule that suppresses HIV</span></div><div class="casAuthors">Niu Qingli; Alamer Edrous; Fan Xiuzhen; Endsley Janice; Hu Haitao; Niu Qingli; Alamer Edrous; Fan Xiuzhen; Endsley Janice; Hu Haitao; Liu Zhiqing; Chen Haiying; Zhou Jia; Gelman Benjamin B; Tian Bing; Kim Jerome H; Michael Nelson L; Robb Merlin L; Ananworanich Jintanat; Michael Nelson L; Robb Merlin L; Ananworanich Jintanat; Ananworanich Jintanat</div><div class="citationInfo"><span class="NLM_cas:title">The Journal of clinical investigation</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">129</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">3361-3373</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">HIV integrates its provirus into the host genome and establishes latent infection.  Antiretroviral therapy (ART) can control HIV viremia, but cannot eradicate or cure the virus.  Approaches targeting host epigenetic machinery to repress HIV, leading to an aviremic state free of ART, are needed.  Bromodomain and extraterminal (BET) family protein BRD4 is an epigenetic reader involved in HIV transcriptional regulation.  Using structure-guided drug design, we identified a small molecule (ZL0580) that induced epigenetic suppression of HIV via BRD4.  We showed that ZL0580 induced HIV suppression in multiple in vitro and ex vivo cell models.  Combination treatment of cells of aviremic HIV-infected individuals with ART and ZL0580 revealed that ZL0580 accelerated HIV suppression during ART and delayed viral rebound after ART cessation.  Mechanistically different from the BET/BRD4 pan-inhibitor JQ1, which nonselectively binds to BD1 and BD2 domains of all BET proteins, ZL0580 selectively bound to BD1 domain of BRD4.  We further demonstrate that ZL0580 induced HIV suppression by inhibiting Tat transactivation and transcription elongation as well as by inducing repressive chromatin structure at the HIV promoter.  Our findings establish a proof of concept for modulation of BRD4 to epigenetically suppress HIV and provide a promising chemical scaffold for the development of probes and/or therapeutic agents for HIV epigenetic silencing.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcS-v5rtAsoTqZ0d-O_J5C0AfW6udTcc2eb1qxJYv_H7bbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3Mzps1ynsg%253D%253D&md5=b6b2d6b47419a88ecdf646cf5d3f859b</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1172%2FJCI120633&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1172%252FJCI120633%26sid%3Dliteratum%253Aachs%26aulast%3DNiu%26aufirst%3DQ.%26aulast%3DLiu%26aufirst%3DZ.%26aulast%3DAlamer%26aufirst%3DE.%26aulast%3DFan%26aufirst%3DX.%26aulast%3DChen%26aufirst%3DH.%26aulast%3DEndsley%26aufirst%3DJ.%26aulast%3DGelman%26aufirst%3DB.%2BB.%26aulast%3DTian%26aufirst%3DB.%26aulast%3DKim%26aufirst%3DJ.%2BH.%26aulast%3DMichael%26aufirst%3DN.%2BL.%26aulast%3DRobb%26aufirst%3DM.%2BL.%26aulast%3DAnanworanich%26aufirst%3DJ.%26aulast%3DZhou%26aufirst%3DJ.%26aulast%3DHu%26aufirst%3DH.%26atitle%3DStructure-guided%2520drug%2520design%2520identifies%2520a%2520BRD4-selective%2520small%2520molecule%2520that%2520suppresses%2520HIV.%26jtitle%3DJ.%2520Clin.%2520Invest.%26date%3D2019%26volume%3D129%26spage%3D3361%26epage%3D3373%26doi%3D10.1172%2FJCI120633" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref62"><div class="reference"><strong class="refLabel"><a href="#ref62" class="rightTabRefNumLink">62</a></strong><div class="NLM_citation" id="rightTab-cit62"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cui, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span> <span> </span><span class="NLM_article-title">Discovery of benzo[cd]indol-2(1H)-ones and pyrrolo[4,3,2-de] quinolin-2(1H)-ones as bromodomain and extra-terminal domain (BET) inhibitors with selectivity for the first bromodomain with potential high efficiency against acute gouty arthritis</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">11080</span>– <span class="NLM_lpage">11107</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b01010</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b01010" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BC1MXit1yqur7E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=11080-11107&author=F.+Jiangauthor=Q.+Huauthor=Z.+Zhangauthor=H.+Liauthor=H.+Liauthor=D.+Zhangauthor=H.+Liauthor=Y.+Maauthor=J.+Xuauthor=H.+Chenauthor=Y.+Cuiauthor=Y.+Zhiauthor=Y.+Zhangauthor=J.+Xuauthor=J.+Zhuauthor=T.+Luauthor=Y.+Chen&title=Discovery+of+benzo%5Bcd%5Dindol-2%281H%29-ones+and+pyrrolo%5B4%2C3%2C2-de%5D+quinolin-2%281H%29-ones+as+bromodomain+and+extra-terminal+domain+%28BET%29+inhibitors+with+selectivity+for+the+first+bromodomain+with+potential+high+efficiency+against+acute+gouty+arthritis&doi=10.1021%2Facs.jmedchem.9b01010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit62R"><div class="casContent"><span class="casTitleNuber">62</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Benzo[cd]indol-2(1H)-ones and Pyrrolo[4,3,2-de]quinolin-2(1H)-ones as Bromodomain and Extra-Terminal Domain (BET) Inhibitors with Selectivity for the First Bromodomain with Potential High Efficiency against Acute Gouty Arthritis</span></div><div class="casAuthors">Jiang, Fei; Hu, Qinghua; Zhang, Zhimin; Li, Hongmei; Li, Huili; Zhang, Dewei; Li, Hanwen; Ma, Yu; Xu, Jingjing; Chen, Haifang; Cui, Yong; Zhi, Yanle; Zhang, Yanmin; Xu, Junyu; Zhu, Jiapeng; Lu, Tao; Chen, Yadong</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">11080-11107</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The bromodomain and extra-terminal domain (BET) family of proteins are readers which specifically recognize histone-acetylated lysine residues.  Each BET bromodomain protein contains two highly homologous domains: the first bromodomain (BD1) and the second bromodomain (BD2).  Pan-BET bromodomain inhibition is a potential therapy for various cancers and immune-inflammatory diseases, but only few reported inhibitors show selectivity within the BET family.  Herein, we identified a series of benzo[cd]indol-2(1H)-ones and pyrrolo[4,3,2-de]quinolin-2(1H)-ones with good selectivity for BET BD1.  Through structure-based optimization, highly active and selective compds. are ultimately obtained.  The representative compds. are the first reported inhibitors with selectivity more than 100-fold for BRD4(1) over BRD4(2).  Among them, we further show that 68 (LT052) mediates BRD4/NF-κB/NLRP3 signaling inflammatory pathways with comparable protein expression and significantly improves symptoms of gout arthritis in a rat model.  Therefore, selective pharmacol. modulation of individual bromodomains could represent a strategy for the treatment of acute gouty arthritis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrav2e3P7XQ1LVg90H21EOLACvtfcHk0lgZhBOfIHu15Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXit1yqur7E&md5=5cce669e137812d20f8ba2a9785f4666</span></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b01010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b01010%26sid%3Dliteratum%253Aachs%26aulast%3DJiang%26aufirst%3DF.%26aulast%3DHu%26aufirst%3DQ.%26aulast%3DZhang%26aufirst%3DZ.%26aulast%3DLi%26aufirst%3DH.%26aulast%3DLi%26aufirst%3DH.%26aulast%3DZhang%26aufirst%3DD.%26aulast%3DLi%26aufirst%3DH.%26aulast%3DMa%26aufirst%3DY.%26aulast%3DXu%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DH.%26aulast%3DCui%26aufirst%3DY.%26aulast%3DZhi%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DXu%26aufirst%3DJ.%26aulast%3DZhu%26aufirst%3DJ.%26aulast%3DLu%26aufirst%3DT.%26aulast%3DChen%26aufirst%3DY.%26atitle%3DDiscovery%2520of%2520benzo%255Bcd%255Dindol-2%25281H%2529-ones%2520and%2520pyrrolo%255B4%252C3%252C2-de%255D%2520quinolin-2%25281H%2529-ones%2520as%2520bromodomain%2520and%2520extra-terminal%2520domain%2520%2528BET%2529%2520inhibitors%2520with%2520selectivity%2520for%2520the%2520first%2520bromodomain%2520with%2520potential%2520high%2520efficiency%2520against%2520acute%2520gouty%2520arthritis%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D11080%26epage%3D11107%26doi%3D10.1021%2Facs.jmedchem.9b01010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref63"><div class="reference"><strong class="refLabel"><a href="#ref63" class="rightTabRefNumLink">63</a></strong><div class="NLM_citation" id="rightTab-cit63"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wold, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leonard, P. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Joseph, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brasier, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tian, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, J.</span></span> <span> </span><span class="NLM_article-title">Discovery of orally bioavailable chromone derivatives as potent and selective BRD4 inhibitors: scaffold hopping, optimization, and pharmacological evaluation</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">5242</span>– <span class="NLM_lpage">5256</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.0c00035</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.0c00035" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A280%3ADC%252BB38zhslKksA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=5242-5256&author=Z.+Liuauthor=H.+Chenauthor=P.+Wangauthor=Y.+Liauthor=E.+A.+Woldauthor=P.+G.+Leonardauthor=S.+Josephauthor=A.+R.+Brasierauthor=B.+Tianauthor=J.+Zhou&title=Discovery+of+orally+bioavailable+chromone+derivatives+as+potent+and+selective+BRD4+inhibitors%3A+scaffold+hopping%2C+optimization%2C+and+pharmacological+evaluation&doi=10.1021%2Facs.jmedchem.0c00035"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit63R"><div class="casContent"><span class="casTitleNuber">63</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Orally Bioavailable Chromone Derivatives as Potent and Selective BRD4 Inhibitors: Scaffold Hopping, Optimization, and Pharmacological Evaluation</span></div><div class="casAuthors">Liu Zhiqing; Chen Haiying; Wang Pingyuan; Li Yi; Wold Eric A; Zhou Jia; Leonard Paul G; Joseph Sarah; Brasier Allan R; Tian Bing; Tian Bing; Zhou Jia; Zhou Jia</div><div class="citationInfo"><span class="NLM_cas:title">Journal of medicinal chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">5242-5256</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Bromodomain-containing protein 4 (BRD4) represents a promising drug target for anti-inflammatory therapeutics.  Herein, we report the design, synthesis, and pharmacological evaluation of novel chromone derivatives via scaffold hopping to discover a new class of orally bioavailable BRD4-selective inhibitors.  Two potent BRD4 bromodomain 1 (BD1)-selective inhibitors 44 (ZL0513) and 45 (ZL0516) have been discovered with high binding affinity (IC50 values of 67-84 nM) and good selectivity over other BRD family proteins and distant BD-containing proteins.  Both compounds significantly inhibited the expression of Toll-like receptor-induced inflammatory genes in vitro and airway inflammation in murine models.  The cocrystal structure of 45 in complex with human BRD4 BD1 at a high resolution of 2.0 ÅA has been solved, offering a solid structural basis for its binding validation and further structure-based optimization.  These BRD4 BD1 inhibitors demonstrated impressive in vivo efficacy and overall promising pharmacokinetic properties, indicating their therapeutic potential for the treatment of inflammatory diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcR1QLaeKXMGxJ0d-O_J5C0AfW6udTcc2ea6XRjYUk94mrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB38zhslKksA%253D%253D&md5=d65e4298cbfff545c6c333713e4db210</span></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.0c00035&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.0c00035%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DZ.%26aulast%3DChen%26aufirst%3DH.%26aulast%3DWang%26aufirst%3DP.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DWold%26aufirst%3DE.%2BA.%26aulast%3DLeonard%26aufirst%3DP.%2BG.%26aulast%3DJoseph%26aufirst%3DS.%26aulast%3DBrasier%26aufirst%3DA.%2BR.%26aulast%3DTian%26aufirst%3DB.%26aulast%3DZhou%26aufirst%3DJ.%26atitle%3DDiscovery%2520of%2520orally%2520bioavailable%2520chromone%2520derivatives%2520as%2520potent%2520and%2520selective%2520BRD4%2520inhibitors%253A%2520scaffold%2520hopping%252C%2520optimization%252C%2520and%2520pharmacological%2520evaluation%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D5242%26epage%3D5256%26doi%3D10.1021%2Facs.jmedchem.0c00035" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref64"><div class="reference"><strong class="refLabel"><a href="#ref64" class="rightTabRefNumLink">64</a></strong><div class="NLM_citation" id="rightTab-cit64"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Watson, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bamborough, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barnett, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C.-w.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grandi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petretich, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phillipou, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rioja, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soden, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Werner, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Demont, E. H.</span></span> <span> </span><span class="NLM_article-title">Demont E. H. GSK789: a selective inhibitor of the first bromodomains (BD1) of the bromo and extra terminal domain (BET) proteins</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">9045</span>– <span class="NLM_lpage">9069</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.0c00614</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.0c00614" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhsVWrsLzJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=9045-9069&author=R.+J.+Watsonauthor=P.+Bamboroughauthor=H.+Barnettauthor=C.-w.+Chungauthor=R.+Davisauthor=L.+Gordonauthor=P.+Grandiauthor=M.+Petretichauthor=A.+Phillipouauthor=R.+K.+Prinjhaauthor=I.+Riojaauthor=P.+Sodenauthor=T.+Wernerauthor=E.+H.+Demont&title=Demont+E.+H.+GSK789%3A+a+selective+inhibitor+of+the+first+bromodomains+%28BD1%29+of+the+bromo+and+extra+terminal+domain+%28BET%29+proteins&doi=10.1021%2Facs.jmedchem.0c00614"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit64R"><div class="casContent"><span class="casTitleNuber">64</span><div class="casTitle"><span class="NLM_cas:atitle">GSK789: A Selective Inhibitor of the First Bromodomains (BD1) of the Bromo and Extra Terminal Domain (BET) Proteins</span></div><div class="casAuthors">Watson, Robert J.; Bamborough, Paul; Barnett, Heather; Chung, Chun-wa; Davis, Rob; Gordon, Laurie; Grandi, Paola; Petretich, Massimo; Phillipou, Alex; Prinjha, Rab K.; Rioja, Inmaculada; Soden, Peter; Werner, Thilo; Demont, Emmanuel H.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">9045-9069</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Pan-bromodomain and extra terminal (BET) inhibitors interact equipotently with all eight bromodomains of the BET family of proteins.  They have shown profound efficacy in vitro and in vivo in oncol. and immunomodulatory models, and a no. of them are currently in clin. trials where significant safety signals have been reported.  It is therefore important to understand the functional contribution of each bromodomain to assess the opportunity to tease apart efficacy and toxicity.  This article discloses the in vitro and cellular activity profiles of GSK789(I), a potent, cell-permeable, and highly selective inhibitor of the first bromodomains of the BET family.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGozid6PMjhvvbVg90H21EOLACvtfcHk0lgOle72Cxsmzw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhsVWrsLzJ&md5=c86b4d2c85dcfa62854debf225c8076c</span></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.0c00614&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.0c00614%26sid%3Dliteratum%253Aachs%26aulast%3DWatson%26aufirst%3DR.%2BJ.%26aulast%3DBamborough%26aufirst%3DP.%26aulast%3DBarnett%26aufirst%3DH.%26aulast%3DChung%26aufirst%3DC.-w.%26aulast%3DDavis%26aufirst%3DR.%26aulast%3DGordon%26aufirst%3DL.%26aulast%3DGrandi%26aufirst%3DP.%26aulast%3DPetretich%26aufirst%3DM.%26aulast%3DPhillipou%26aufirst%3DA.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DRioja%26aufirst%3DI.%26aulast%3DSoden%26aufirst%3DP.%26aulast%3DWerner%26aufirst%3DT.%26aulast%3DDemont%26aufirst%3DE.%2BH.%26atitle%3DDemont%2520E.%2520H.%2520GSK789%253A%2520a%2520selective%2520inhibitor%2520of%2520the%2520first%2520bromodomains%2520%2528BD1%2529%2520of%2520the%2520bromo%2520and%2520extra%2520terminal%2520domain%2520%2528BET%2529%2520proteins%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D9045%26epage%3D9069%26doi%3D10.1021%2Facs.jmedchem.0c00614" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref65"><div class="reference"><strong class="refLabel"><a href="#ref65" class="rightTabRefNumLink">65</a></strong><div class="NLM_citation" id="rightTab-cit65"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wellaway, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bamborough, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernard, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Craggs, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cutler, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Demont, E. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evans, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karamshi, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lewis, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindon, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitchell, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rioja, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soden, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watson, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Willis, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woolven, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wyspianska, B. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kerr, W. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span> <span> </span><span class="NLM_article-title">Structure-based design of a bromodomain and extraterminal domain (BET) inhibitor selective for the N-terminal bromodomains that retains an anti-inflammatory and antiproliferative phenotype</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">9020</span>– <span class="NLM_lpage">9044</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.0c00566</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.0c00566" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhsV2lt77M" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=9020-9044&author=C.+R.+Wellawayauthor=P.+Bamboroughauthor=S.+G.+Bernardauthor=C.+W.+Chungauthor=P.+D.+Craggsauthor=L.+Cutlerauthor=E.+H.+Demontauthor=J.+P.+Evansauthor=L.+Gordonauthor=B.+Karamshiauthor=A.+J.+Lewisauthor=M.+J.+Lindonauthor=D.+J.+Mitchellauthor=I.+Riojaauthor=P.+E.+Sodenauthor=S.+Taylorauthor=R.+J.+Watsonauthor=R.+Willisauthor=J.+M.+Woolvenauthor=B.+S.+Wyspianskaauthor=W.+J.+Kerrauthor=R.+K.+Prinjha&title=Structure-based+design+of+a+bromodomain+and+extraterminal+domain+%28BET%29+inhibitor+selective+for+the+N-terminal+bromodomains+that+retains+an+anti-inflammatory+and+antiproliferative+phenotype&doi=10.1021%2Facs.jmedchem.0c00566"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit65R"><div class="casContent"><span class="casTitleNuber">65</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Based Design of a Bromodomain and Extraterminal Domain (BET) Inhibitor Selective for the N-Terminal Bromodomains That Retains an Anti-inflammatory and Antiproliferative Phenotype</span></div><div class="casAuthors">Wellaway, Christopher R.; Bamborough, Paul; Bernard, Sharon G.; Chung, Chun-wa; Craggs, Peter D.; Cutler, Leanne; Demont, Emmanuel H.; Evans, John P.; Gordon, Laurie; Karamshi, Bhumika; Lewis, Antonia J.; Lindon, Matthew J.; Mitchell, Darren J.; Rioja, Inmaculada; Soden, Peter E.; Taylor, Simon; Watson, Robert J.; Willis, Rob; Woolven, James M.; Wyspianska, Beata S.; Kerr, William J.; Prinjha, Rab K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">9020-9044</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The bromodomain and extraterminal domain (BET) family of epigenetic regulators comprises four proteins (BRD2, BRD3, BRD4, BRDT), each contg. tandem bromodomains.  To date, small mol. inhibitors of these proteins typically bind all eight bromodomains of the family with similar affinity, resulting in a diverse range of biol. effects.  To enable further understanding of the broad phenotype characteristic of pan-BET inhibition, the development of inhibitors selective for individual, or sets of, bromodomains within the family is required.  In this regard, we report the discovery of a potent probe mol. possessing up to 150-fold selectivity for the N-terminal bromodomains (BD1s) over the C-terminal bromodomains (BD2s) of the BETs.  Guided by structural information, a specific amino acid difference between BD1 and BD2 domains was targeted for selective interaction with chem. functionality appended to the previously developed I-BET151 scaffold.  Data presented herein demonstrate that selective inhibition of BD1 domains is sufficient to drive anti-inflammatory and antiproliferative effects.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpkUREcHUhFfbVg90H21EOLACvtfcHk0lgIB66gej4DiA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhsV2lt77M&md5=52b2505bb0a36f526a08bb9bad1f2c26</span></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.0c00566&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.0c00566%26sid%3Dliteratum%253Aachs%26aulast%3DWellaway%26aufirst%3DC.%2BR.%26aulast%3DBamborough%26aufirst%3DP.%26aulast%3DBernard%26aufirst%3DS.%2BG.%26aulast%3DChung%26aufirst%3DC.%2BW.%26aulast%3DCraggs%26aufirst%3DP.%2BD.%26aulast%3DCutler%26aufirst%3DL.%26aulast%3DDemont%26aufirst%3DE.%2BH.%26aulast%3DEvans%26aufirst%3DJ.%2BP.%26aulast%3DGordon%26aufirst%3DL.%26aulast%3DKaramshi%26aufirst%3DB.%26aulast%3DLewis%26aufirst%3DA.%2BJ.%26aulast%3DLindon%26aufirst%3DM.%2BJ.%26aulast%3DMitchell%26aufirst%3DD.%2BJ.%26aulast%3DRioja%26aufirst%3DI.%26aulast%3DSoden%26aufirst%3DP.%2BE.%26aulast%3DTaylor%26aufirst%3DS.%26aulast%3DWatson%26aufirst%3DR.%2BJ.%26aulast%3DWillis%26aufirst%3DR.%26aulast%3DWoolven%26aufirst%3DJ.%2BM.%26aulast%3DWyspianska%26aufirst%3DB.%2BS.%26aulast%3DKerr%26aufirst%3DW.%2BJ.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26atitle%3DStructure-based%2520design%2520of%2520a%2520bromodomain%2520and%2520extraterminal%2520domain%2520%2528BET%2529%2520inhibitor%2520selective%2520for%2520the%2520N-terminal%2520bromodomains%2520that%2520retains%2520an%2520anti-inflammatory%2520and%2520antiproliferative%2520phenotype%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D9020%26epage%3D9044%26doi%3D10.1021%2Facs.jmedchem.0c00566" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref66"><div class="reference"><strong class="refLabel"><a href="#ref66" class="rightTabRefNumLink">66</a></strong><div class="NLM_citation" id="rightTab-cit66"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cui, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Divakaran, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pandey, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zahid, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoell, Z. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ellingson, M. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aihara, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harki, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pomerantz, W. C. K.</span></span> <span> </span><span class="NLM_article-title">Selective N-terminal BRD4 bromodomain inhibitors by targeting non-conserved residues and structured water displacement</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2021</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">1220</span>– <span class="NLM_lpage">1226</span>, <span class="refDoi"> DOI: 10.1002/anie.202008625</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1002%2Fanie.202008625" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BB3cXitlOqtLjI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2021&pages=1220-1226&author=H.+Cuiauthor=A.+Divakaranauthor=A.+K.+Pandeyauthor=J.+A.+Johnsonauthor=H.+Zahidauthor=Z.+J.+Hoellauthor=M.+O.+Ellingsonauthor=K.+Shiauthor=H.+Aiharaauthor=D.+A.+Harkiauthor=W.+C.+K.+Pomerantz&title=Selective+N-terminal+BRD4+bromodomain+inhibitors+by+targeting+non-conserved+residues+and+structured+water+displacement&doi=10.1002%2Fanie.202008625"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit66R"><div class="casContent"><span class="casTitleNuber">66</span><div class="casTitle"><span class="NLM_cas:atitle">Selective N-Terminal BET Bromodomain Inhibitors by Targeting Non-Conserved Residues and Structured Water Displacement**</span></div><div class="casAuthors">Cui, Huarui; Divakaran, Anand; Pandey, Anil K.; Johnson, Jorden A.; Zahid, Huda; Hoell, Zachariah J.; Ellingson, Mikael O.; Shi, Ke; Aihara, Hideki; Harki, Daniel A.; Pomerantz, William C. K.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2021</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1220-1226</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Bromodomain and extra-terminal (BET) family proteins, BRD2-4 and T, are important drug targets; however, the biol. functions of each bromodomain remain ill-defined.  Chem. probes that selectively inhibit a single BET bromodomain are lacking, although pan inhibitors of the first (D1), and second (D2), bromodomain are known.  Here, we develop selective BET D1 inhibitors with preferred binding to BRD4 D1.  In competitive inhibition assays, we show that our lead compd. is 9-33 fold selective for BRD4 D1 over the other BET bromodomains.  X-ray crystallog. supports a role for the selectivity based on reorganization of a non-conserved lysine and displacement of an addnl. structured water in the BRD4 D1 binding site relative to our prior lead.  Whereas pan-D1 inhibitors displace BRD4 from MYC enhancers, BRD4 D1 inhibition in MM.1S cells is insufficient for stopping Myc expression and may lead to its upregulation.  Future anal. of BRD4 D1 gene regulation may shed light on differential BET bromodomain functions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr19lACXxcpUrVg90H21EOLACvtfcHk0lgIB66gej4DiA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXitlOqtLjI&md5=0f0379a17c98e5c1b9c436dc0952e9a7</span></div><a href="/servlet/linkout?suffix=cit66&amp;dbid=16384&amp;doi=10.1002%2Fanie.202008625&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.202008625%26sid%3Dliteratum%253Aachs%26aulast%3DCui%26aufirst%3DH.%26aulast%3DDivakaran%26aufirst%3DA.%26aulast%3DPandey%26aufirst%3DA.%2BK.%26aulast%3DJohnson%26aufirst%3DJ.%2BA.%26aulast%3DZahid%26aufirst%3DH.%26aulast%3DHoell%26aufirst%3DZ.%2BJ.%26aulast%3DEllingson%26aufirst%3DM.%2BO.%26aulast%3DShi%26aufirst%3DK.%26aulast%3DAihara%26aufirst%3DH.%26aulast%3DHarki%26aufirst%3DD.%2BA.%26aulast%3DPomerantz%26aufirst%3DW.%2BC.%2BK.%26atitle%3DSelective%2520N-terminal%2520BRD4%2520bromodomain%2520inhibitors%2520by%2520targeting%2520non-conserved%2520residues%2520and%2520structured%2520water%2520displacement%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2021%26volume%3D60%26spage%3D1220%26epage%3D1226%26doi%3D10.1002%2Fanie.202008625" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref67"><div class="reference"><strong class="refLabel"><a href="#ref67" class="rightTabRefNumLink">67</a></strong><div class="NLM_citation" id="rightTab-cit67"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Law, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Atkinson, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bamborough, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Demont, E. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindon, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watson, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hirst, D. J.</span></span> <span> </span><span class="NLM_article-title">Discovery of tetrahydroquinoxalines as bromodomain and extra-terminal domain (BET) inhibitors with selectivity for the second bromodomain</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">4317</span>– <span class="NLM_lpage">4334</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b01666</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b01666" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BC1cXns1Sjtbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=4317-4334&author=R.+P.+Lawauthor=S.+J.+Atkinsonauthor=P.+Bamboroughauthor=C.+W.+Chungauthor=E.+H.+Demontauthor=L.+J.+Gordonauthor=M.+Lindonauthor=R.+K.+Prinjhaauthor=A.+Watsonauthor=D.+J.+Hirst&title=Discovery+of+tetrahydroquinoxalines+as+bromodomain+and+extra-terminal+domain+%28BET%29+inhibitors+with+selectivity+for+the+second+bromodomain&doi=10.1021%2Facs.jmedchem.7b01666"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit67R"><div class="casContent"><span class="casTitleNuber">67</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Tetrahydroquinoxalines as Bromodomain and Extra-Terminal Domain (BET) Inhibitors with Selectivity for the Second Bromodomain</span></div><div class="casAuthors">Law, Robert P.; Atkinson, Stephen J.; Bamborough, Paul; Chung, Chun-wa; Demont, Emmanuel H.; Gordon, Laurie J.; Lindon, Matthew; Prinjha, Rab K.; Watson, Allan J. B.; Hirst, David J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">4317-4334</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The bromodomain and extra-terminal domain (BET) family of proteins bind acetylated lysine residues on histone proteins.  The four BET bromodomains-BRD2, BRD3, BRD4, and BRDT-each contain two bromodomain modules.  BET bromodomain inhibition is a potential therapy for various cancers and immunoinflammatory diseases, but few reported inhibitors show selectivity within the BET family.  Inhibitors with selectivity for the first or second bromodomain are desired to aid investigation of the biol. function of these domains.  Focused library screening identified a series of tetrahydroquinoxalines with selectivity for the second bromodomains of the BET family (BD2).  Structure-guided optimization of the template improved potency, selectivity, and physicochem. properties, culminating in potent BET inhibitors with BD2 selectivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGobBKYwu7NyTbVg90H21EOLACvtfcHk0lgIB66gej4DiA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXns1Sjtbc%253D&md5=078db91b35feaf1958fa8c33d0544387</span></div><a href="/servlet/linkout?suffix=cit67&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b01666&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b01666%26sid%3Dliteratum%253Aachs%26aulast%3DLaw%26aufirst%3DR.%2BP.%26aulast%3DAtkinson%26aufirst%3DS.%2BJ.%26aulast%3DBamborough%26aufirst%3DP.%26aulast%3DChung%26aufirst%3DC.%2BW.%26aulast%3DDemont%26aufirst%3DE.%2BH.%26aulast%3DGordon%26aufirst%3DL.%2BJ.%26aulast%3DLindon%26aufirst%3DM.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DWatson%26aufirst%3DA.%26aulast%3DHirst%26aufirst%3DD.%2BJ.%26atitle%3DDiscovery%2520of%2520tetrahydroquinoxalines%2520as%2520bromodomain%2520and%2520extra-terminal%2520domain%2520%2528BET%2529%2520inhibitors%2520with%2520selectivity%2520for%2520the%2520second%2520bromodomain%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D4317%26epage%3D4334%26doi%3D10.1021%2Facs.jmedchem.7b01666" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref68"><div class="reference"><strong class="refLabel"><a href="#ref68" class="rightTabRefNumLink">68</a></strong><div class="NLM_citation" id="rightTab-cit68"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lyu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y.</span></span> <span> </span><span class="NLM_article-title">Discovery, structural insight, and bioactivities of BY27 as a selective inhibitor of the second bromodomains of BET proteins</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>182</i></span>,  <span class="NLM_fpage">111633</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2019.111633</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1016%2Fj.ejmech.2019.111633" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=31461688" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhs1Kktb%252FJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=182&publication_year=2019&pages=111633&author=D.+Chenauthor=T.+Luauthor=Z.+Yanauthor=W.+Luauthor=F.+Zhouauthor=X.+Lyuauthor=B.+Xuauthor=H.+Jiangauthor=K.+Chenauthor=C.+Luoauthor=Y.+Zhao&title=Discovery%2C+structural+insight%2C+and+bioactivities+of+BY27+as+a+selective+inhibitor+of+the+second+bromodomains+of+BET+proteins&doi=10.1016%2Fj.ejmech.2019.111633"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit68R"><div class="casContent"><span class="casTitleNuber">68</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery, structural insight, and bioactivities of BY27 as a selective inhibitor of the second bromodomains of BET proteins</span></div><div class="casAuthors">Chen, Deheng; Lu, Tian; Yan, Ziqin; Lu, Wenchao; Zhou, Feilong; Lyu, Xilin; Xu, Biling; Jiang, Hualiang; Chen, Kaixian; Luo, Cheng; Zhao, Yujun</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">182</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">111633</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">Recently, selective inhibition of BET BD2 is emerging as a promising strategy for drug discovery.  Despite significant progress in this area, systematic studies of selective BET BD2 inhibitors are still few.  In this study, we report the discovery of a potent and selective BET BD2 inhibitor BY27 (47).  Our high resoln. co-crystal structures of 47/BRD2 BD1 and BD2 showed that the triazole group of 47, water mols., H433 and N429 in BRD2 BD2 established a water-bridged H-bonding network, which is responsible for the obsd. selectivities.  DNA microarray anal. of HepG2 cells treated with 47 or OTX015 demonstrated the transcriptome impact differences between a BET BD2 selective inhibitor and a pan BET inhibitor.  In a MV4-11 mouse xenograft model, 47 caused 67% of tumor growth inhibition and was less toxic than a pan BET inhibitor 1 at high doses.  We conclude that the improved safety profile of selective BET BD2 inhibitors warrant future studies in BET assocd. diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGre_6szzmO9ZbVg90H21EOLACvtfcHk0lgueq5i3RfAIw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhs1Kktb%252FJ&md5=e3a21e4e692b25a94e0035a2e75d33f7</span></div><a href="/servlet/linkout?suffix=cit68&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2019.111633&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2019.111633%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DD.%26aulast%3DLu%26aufirst%3DT.%26aulast%3DYan%26aufirst%3DZ.%26aulast%3DLu%26aufirst%3DW.%26aulast%3DZhou%26aufirst%3DF.%26aulast%3DLyu%26aufirst%3DX.%26aulast%3DXu%26aufirst%3DB.%26aulast%3DJiang%26aufirst%3DH.%26aulast%3DChen%26aufirst%3DK.%26aulast%3DLuo%26aufirst%3DC.%26aulast%3DZhao%26aufirst%3DY.%26atitle%3DDiscovery%252C%2520structural%2520insight%252C%2520and%2520bioactivities%2520of%2520BY27%2520as%2520a%2520selective%2520inhibitor%2520of%2520the%2520second%2520bromodomains%2520of%2520BET%2520proteins%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2019%26volume%3D182%26spage%3D111633%26doi%3D10.1016%2Fj.ejmech.2019.111633" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref69"><div class="reference"><strong class="refLabel"><a href="#ref69" class="rightTabRefNumLink">69</a></strong><div class="NLM_citation" id="rightTab-cit69"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Faivre, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mcdaniel, K. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Albert, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mantena, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plotnik, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilcox, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bui, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheppard, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sehgal, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uziel, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hessler, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lam, L. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bellin, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mehta, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fidanze, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pratt, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hasvold, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Panchal, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nicolette, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fossey, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Longenecker, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bigelow, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Torrent, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosenberg, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kati, W. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, Y.</span></span> <span> </span><span class="NLM_article-title">Selective inhibition of the BD2 bromodomain of BET proteins in prostate cancer</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>578</i></span>,  <span class="NLM_fpage">306</span>– <span class="NLM_lpage">310</span>, <span class="refDoi"> DOI: 10.1038/s41586-020-1930-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1038%2Fs41586-020-1930-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=31969702" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BB3cXis1Smsr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=578&publication_year=2020&pages=306-310&author=E.+J.+Faivreauthor=K.+F.+Mcdanielauthor=D.+H.+Albertauthor=S.+R.+Mantenaauthor=J.+P.+Plotnikauthor=D.+Wilcoxauthor=L.+Zhangauthor=M.+H.+Buiauthor=G.+S.+Sheppardauthor=L.+Wangauthor=V.+Sehgalauthor=X.+Linauthor=X.+Huangauthor=X.+Luauthor=T.+Uzielauthor=P.+Hesslerauthor=L.+T.+Lamauthor=R.+J.+Bellinauthor=G.+Mehtaauthor=S.+Fidanzeauthor=J.+K.+Prattauthor=D.+Liuauthor=L.+A.+Hasvoldauthor=C.+Sunauthor=S.+C.+Panchalauthor=J.+J.+Nicoletteauthor=S.+L.+Fosseyauthor=C.+H.+Parkauthor=K.+Longeneckerauthor=L.+Bigelowauthor=M.+Torrentauthor=S.+H.+Rosenbergauthor=W.+M.+Katiauthor=Y.+Shen&title=Selective+inhibition+of+the+BD2+bromodomain+of+BET+proteins+in+prostate+cancer&doi=10.1038%2Fs41586-020-1930-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit69R"><div class="casContent"><span class="casTitleNuber">69</span><div class="casTitle"><span class="NLM_cas:atitle">Selective inhibition of the BD2 bromodomain of BET proteins in prostate cancer</span></div><div class="casAuthors">Faivre, Emily J.; McDaniel, Keith F.; Albert, Daniel H.; Mantena, Srinivasa R.; Plotnik, Joshua P.; Wilcox, Denise; Zhang, Lu; Bui, Mai H.; Sheppard, George S.; Wang, Le; Sehgal, Vasudha; Lin, Xiaoyu; Huang, Xiaoli; Lu, Xin; Uziel, Tamar; Hessler, Paul; Lam, Lloyd T.; Bellin, Richard J.; Mehta, Gaurav; Fidanze, Steve; Pratt, John K.; Liu, Dachun; Hasvold, Lisa A.; Sun, Chaohong; Panchal, Sanjay C.; Nicolette, John J.; Fossey, Stacey L.; Park, Chang H.; Longenecker, Kenton; Bigelow, Lance; Torrent, Maricel; Rosenberg, Saul H.; Kati, Warren M.; Shen, Yu</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">578</span>
        (<span class="NLM_cas:issue">7794</span>),
    <span class="NLM_cas:pages">306-310</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Abstr.: Proteins of the bromodomain and extra-terminal (BET) domain family are epigenetic readers that bind acetylated histones through their bromodomains to regulate gene transcription.  Dual-bromodomain BET inhibitors (DbBi) that bind with similar affinities to the first (BD1) and second (BD2) bromodomains of BRD2, BRD3, BRD4 and BRDt have displayed modest clin. activity in monotherapy cancer trials.  A reduced no. of thrombocytes in the blood (thrombocytopenia) as well as symptoms of gastrointestinal toxicity are dose-limiting adverse events for some types of DbBi1-5.  Given that similar haematol. and gastrointestinal defects were obsd. after genetic silencing of Brd4 in mice6, the platelet and gastrointestinal toxicities may represent on-target activities assocd. with BET inhibition.  The two individual bromodomains in BET family proteins may have distinct functions7-9 and different cellular phenotypes after pharmacol. inhibition of one or both bromodomains have been reported10,11, suggesting that selectively targeting one of the bromodomains may result in a different efficacy and tolerability profile compared with DbBi.  Available compds. that are selective to individual domains lack sufficient potency and the pharmacokinetics properties that are required for in vivo efficacy and tolerability assessment10-13.  Here we carried out a medicinal chem. campaign that led to the discovery of ABBV-744, a highly potent and selective inhibitor of the BD2 domain of BET family proteins with drug-like properties.  In contrast to the broad range of cell growth inhibition induced by DbBi, the antiproliferative activity of ABBV-744 was largely, but not exclusively, restricted to cell lines of acute myeloid leukemia and prostate cancer that expressed the full-length androgen receptor (AR).  ABBV-744 retained robust activity in prostate cancer xenografts, and showed fewer platelet and gastrointestinal toxicities than the DbBi ABBV-07514.  Analyses of RNA expression and chromatin immunopptn. followed by sequencing revealed that ABBV-744 displaced BRD4 from AR-contg. super-enhancers and inhibited AR-dependent transcription, with less impact on global transcription compared with ABBV-075.  These results underscore the potential value of selectively targeting the BD2 domain of BET family proteins for cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrF98kER1y9pLVg90H21EOLACvtfcHk0lgueq5i3RfAIw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXis1Smsr0%253D&md5=800d9106c33084390fd8cbf1ac7f1766</span></div><a href="/servlet/linkout?suffix=cit69&amp;dbid=16384&amp;doi=10.1038%2Fs41586-020-1930-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41586-020-1930-8%26sid%3Dliteratum%253Aachs%26aulast%3DFaivre%26aufirst%3DE.%2BJ.%26aulast%3DMcdaniel%26aufirst%3DK.%2BF.%26aulast%3DAlbert%26aufirst%3DD.%2BH.%26aulast%3DMantena%26aufirst%3DS.%2BR.%26aulast%3DPlotnik%26aufirst%3DJ.%2BP.%26aulast%3DWilcox%26aufirst%3DD.%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DBui%26aufirst%3DM.%2BH.%26aulast%3DSheppard%26aufirst%3DG.%2BS.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DSehgal%26aufirst%3DV.%26aulast%3DLin%26aufirst%3DX.%26aulast%3DHuang%26aufirst%3DX.%26aulast%3DLu%26aufirst%3DX.%26aulast%3DUziel%26aufirst%3DT.%26aulast%3DHessler%26aufirst%3DP.%26aulast%3DLam%26aufirst%3DL.%2BT.%26aulast%3DBellin%26aufirst%3DR.%2BJ.%26aulast%3DMehta%26aufirst%3DG.%26aulast%3DFidanze%26aufirst%3DS.%26aulast%3DPratt%26aufirst%3DJ.%2BK.%26aulast%3DLiu%26aufirst%3DD.%26aulast%3DHasvold%26aufirst%3DL.%2BA.%26aulast%3DSun%26aufirst%3DC.%26aulast%3DPanchal%26aufirst%3DS.%2BC.%26aulast%3DNicolette%26aufirst%3DJ.%2BJ.%26aulast%3DFossey%26aufirst%3DS.%2BL.%26aulast%3DPark%26aufirst%3DC.%2BH.%26aulast%3DLongenecker%26aufirst%3DK.%26aulast%3DBigelow%26aufirst%3DL.%26aulast%3DTorrent%26aufirst%3DM.%26aulast%3DRosenberg%26aufirst%3DS.%2BH.%26aulast%3DKati%26aufirst%3DW.%2BM.%26aulast%3DShen%26aufirst%3DY.%26atitle%3DSelective%2520inhibition%2520of%2520the%2520BD2%2520bromodomain%2520of%2520BET%2520proteins%2520in%2520prostate%2520cancer%26jtitle%3DNature%26date%3D2020%26volume%3D578%26spage%3D306%26epage%3D310%26doi%3D10.1038%2Fs41586-020-1930-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref70"><div class="reference"><strong class="refLabel"><a href="#ref70" class="rightTabRefNumLink">70</a></strong><div class="NLM_citation" id="rightTab-cit70"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sheppard, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fidanze, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hasvold, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pratt, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Longenecker, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qiu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Torrent, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kovar, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bui, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Faivre, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilcox, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Albert, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Magoc, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rajaraman, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kati, W. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mcdaniel, K. F.</span></span> <span> </span><span class="NLM_article-title">Discovery of N-Ethyl-4-[2-(4-fluoro-2,6-dimethyl-phenoxy)-5-(1-hydroxy-1-methyl-ethyl)phenyl]- 6-methyl-7-oxo-1H-pyrrolo[2,3-c]pyridine-2-carboxamide (ABBV-744), a BET bromodomain inhibitor with selectivity for the second bromodomain</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">5585</span>– <span class="NLM_lpage">5623</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.0c00628</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.0c00628" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BB3cXnvVeisrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=5585-5623&author=G.+S.+Sheppardauthor=L.+Wangauthor=S.+D.+Fidanzeauthor=L.+A.+Hasvoldauthor=D.+Liuauthor=J.+K.+Prattauthor=C.+H.+Parkauthor=K.+Longeneckerauthor=W.+Qiuauthor=M.+Torrentauthor=P.+J.+Kovarauthor=M.+Buiauthor=E.+Faivreauthor=X.+Huangauthor=X.+Linauthor=D.+Wilcoxauthor=L.+Zhangauthor=Y.+Shenauthor=D.+H.+Albertauthor=T.+J.+Magocauthor=G.+Rajaramanauthor=W.+M.+Katiauthor=K.+F.+Mcdaniel&title=Discovery+of+N-Ethyl-4-%5B2-%284-fluoro-2%2C6-dimethyl-phenoxy%29-5-%281-hydroxy-1-methyl-ethyl%29phenyl%5D-+6-methyl-7-oxo-1H-pyrrolo%5B2%2C3-c%5Dpyridine-2-carboxamide+%28ABBV-744%29%2C+a+BET+bromodomain+inhibitor+with+selectivity+for+the+second+bromodomain&doi=10.1021%2Facs.jmedchem.0c00628"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit70R"><div class="casContent"><span class="casTitleNuber">70</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of N-Ethyl-4-[2-(4-fluoro-2,6-dimethyl-phenoxy)-5-(1-hydroxy-1-methyl-ethyl)phenyl]-6-methyl-7-oxo-1H-pyrrolo[2,3-c]pyridine-2-carboxamide (ABBV-744), a BET Bromodomain Inhibitor with Selectivity for the Second Bromodomain</span></div><div class="casAuthors">Sheppard, George S.; Wang, Le; Fidanze, Steven D.; Hasvold, Lisa A.; Liu, Dachun; Pratt, John K.; Park, Chang H.; Longenecker, Kenton; Qiu, Wei; Torrent, Maricel; Kovar, Peter J.; Bui, Mai; Faivre, Emily; Huang, Xiaoli; Lin, Xiaoyu; Wilcox, Denise; Zhang, Lu; Shen, Yu; Albert, Daniel H.; Magoc, Terrance J.; Rajaraman, Ganesh; Kati, Warren M.; McDaniel, Keith F.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">5585-5623</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The BET family of proteins consists of BRD2, BRD3, BRD4, and BRDt.  Each protein contains two distinct bromodomains (BD1 and BD2).  BET family bromodomain inhibitors under clin. development for oncol. bind to each of the eight bromodomains with similar affinities.  We hypothesized that it may be possible to achieve an improved therapeutic index by selectively targeting subsets of the BET bromodomains.  Both BD1 and BD2 are highly conserved across family members (>70% identity), whereas BD1 and BD2 from the same protein exhibit a larger degree of divergence (~ 40% identity), suggesting selectivity between BD1 and BD2 of all family members would be more straightforward to achieve.  Exploiting the Asp144/His437 and Ile146/Val439 sequence differences (BRD4 BD1/BD2 numbering) allowed the identification of compd. 27 demonstrating greater than 100-fold selectivity for BRD4 BD2 over BRD4 BD1.  Further optimization to improve BD2 selectivity and oral bioavailability resulted in the clin. development compd. 46 (ABBV-744).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoJtZIcWro0xLVg90H21EOLACvtfcHk0liyBlJErRP8nA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXnvVeisrk%253D&md5=675827659c3c7b6099ccd667f4c60f95</span></div><a href="/servlet/linkout?suffix=cit70&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.0c00628&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.0c00628%26sid%3Dliteratum%253Aachs%26aulast%3DSheppard%26aufirst%3DG.%2BS.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DFidanze%26aufirst%3DS.%2BD.%26aulast%3DHasvold%26aufirst%3DL.%2BA.%26aulast%3DLiu%26aufirst%3DD.%26aulast%3DPratt%26aufirst%3DJ.%2BK.%26aulast%3DPark%26aufirst%3DC.%2BH.%26aulast%3DLongenecker%26aufirst%3DK.%26aulast%3DQiu%26aufirst%3DW.%26aulast%3DTorrent%26aufirst%3DM.%26aulast%3DKovar%26aufirst%3DP.%2BJ.%26aulast%3DBui%26aufirst%3DM.%26aulast%3DFaivre%26aufirst%3DE.%26aulast%3DHuang%26aufirst%3DX.%26aulast%3DLin%26aufirst%3DX.%26aulast%3DWilcox%26aufirst%3DD.%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DShen%26aufirst%3DY.%26aulast%3DAlbert%26aufirst%3DD.%2BH.%26aulast%3DMagoc%26aufirst%3DT.%2BJ.%26aulast%3DRajaraman%26aufirst%3DG.%26aulast%3DKati%26aufirst%3DW.%2BM.%26aulast%3DMcdaniel%26aufirst%3DK.%2BF.%26atitle%3DDiscovery%2520of%2520N-Ethyl-4-%255B2-%25284-fluoro-2%252C6-dimethyl-phenoxy%2529-5-%25281-hydroxy-1-methyl-ethyl%2529phenyl%255D-%25206-methyl-7-oxo-1H-pyrrolo%255B2%252C3-c%255Dpyridine-2-carboxamide%2520%2528ABBV-744%2529%252C%2520a%2520BET%2520bromodomain%2520inhibitor%2520with%2520selectivity%2520for%2520the%2520second%2520bromodomain%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D5585%26epage%3D5623%26doi%3D10.1021%2Facs.jmedchem.0c00628" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref71"><div class="reference"><strong class="refLabel"><a href="#ref71" class="rightTabRefNumLink">71</a></strong><div class="NLM_citation" id="rightTab-cit71"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Preston, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Atkinson, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bamborough, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Craggs, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grandi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindon, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michon, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitchell, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rianjongdee, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rioja, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seal, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wall, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watson, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woolven, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Demont, E. H.</span></span> <span> </span><span class="NLM_article-title">Design and synthesis of a highly selective and in vivo-capable inhibitor of the second bromodomain of the bromodomain and extra terminal domain family of proteins</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">9070</span>– <span class="NLM_lpage">9092</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.0c00605</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.0c00605" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhsVWrsLrK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=9070-9092&author=A.+Prestonauthor=S.+Atkinsonauthor=P.+Bamboroughauthor=C.+W.+Chungauthor=P.+D.+Craggsauthor=L.+Gordonauthor=P.+Grandiauthor=J.+Grayauthor=E.+J.+Jonesauthor=M.+Lindonauthor=A.+M.+Michonauthor=D.+J.+Mitchellauthor=R.+K.+Prinjhaauthor=F.+Rianjongdeeauthor=I.+Riojaauthor=J.+Sealauthor=S.+Taylorauthor=I.+Wallauthor=R.+J.+Watsonauthor=J.+Woolvenauthor=E.+H.+Demont&title=Design+and+synthesis+of+a+highly+selective+and+in+vivo-capable+inhibitor+of+the+second+bromodomain+of+the+bromodomain+and+extra+terminal+domain+family+of+proteins&doi=10.1021%2Facs.jmedchem.0c00605"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit71R"><div class="casContent"><span class="casTitleNuber">71</span><div class="casTitle"><span class="NLM_cas:atitle">Design and Synthesis of a Highly Selective and In Vivo-Capable Inhibitor of the Second Bromodomain of the Bromodomain and Extra Terminal Domain Family of Proteins</span></div><div class="casAuthors">Preston, Alex; Atkinson, Stephen; Bamborough, Paul; Chung, Chun-wa; Craggs, Peter D.; Gordon, Laurie; Grandi, Paola; Gray, James R. J.; Jones, Emma J.; Lindon, Matthew; Michon, Anne-Marie; Mitchell, Darren J.; Prinjha, Rab K.; Rianjongdee, Francesco; Rioja, Inmaculada; Seal, Jonathan; Taylor, Simon; Wall, Ian; Watson, Robert J.; Woolven, James; Demont, Emmanuel H.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">9070-9092</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Pan-bromodomain and extra terminal domain (BET) inhibitors interact equipotently with the eight bromodomains of the BET family of proteins and have shown profound efficacy in a no. of in vitro phenotypic assays and in vivo pre-clin. models in inflammation or oncol.  A no. of these inhibitors have progressed to the clinic where pharmacol.-driven adverse events have been reported.  To better understand the contribution of each domain to their efficacy and improve their safety profile, selective inhibitors are required.  This article discloses the profile of GSK046, also known as iBET-BD2(I), a highly selective inhibitor of the second bromodomains of the BET proteins that has undergone extensive pre-clin. in vitro and in vivo characterization.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpXn2qUNkpo27Vg90H21EOLACvtfcHk0liUBgAXjcIKsw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhsVWrsLrK&md5=08f4c47868e0a5acd5d443adba06d016</span></div><a href="/servlet/linkout?suffix=cit71&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.0c00605&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.0c00605%26sid%3Dliteratum%253Aachs%26aulast%3DPreston%26aufirst%3DA.%26aulast%3DAtkinson%26aufirst%3DS.%26aulast%3DBamborough%26aufirst%3DP.%26aulast%3DChung%26aufirst%3DC.%2BW.%26aulast%3DCraggs%26aufirst%3DP.%2BD.%26aulast%3DGordon%26aufirst%3DL.%26aulast%3DGrandi%26aufirst%3DP.%26aulast%3DGray%26aufirst%3DJ.%26aulast%3DJones%26aufirst%3DE.%2BJ.%26aulast%3DLindon%26aufirst%3DM.%26aulast%3DMichon%26aufirst%3DA.%2BM.%26aulast%3DMitchell%26aufirst%3DD.%2BJ.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DRianjongdee%26aufirst%3DF.%26aulast%3DRioja%26aufirst%3DI.%26aulast%3DSeal%26aufirst%3DJ.%26aulast%3DTaylor%26aufirst%3DS.%26aulast%3DWall%26aufirst%3DI.%26aulast%3DWatson%26aufirst%3DR.%2BJ.%26aulast%3DWoolven%26aufirst%3DJ.%26aulast%3DDemont%26aufirst%3DE.%2BH.%26atitle%3DDesign%2520and%2520synthesis%2520of%2520a%2520highly%2520selective%2520and%2520in%2520vivo-capable%2520inhibitor%2520of%2520the%2520second%2520bromodomain%2520of%2520the%2520bromodomain%2520and%2520extra%2520terminal%2520domain%2520family%2520of%2520proteins%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D9070%26epage%3D9092%26doi%3D10.1021%2Facs.jmedchem.0c00605" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref72"><div class="reference"><strong class="refLabel"><a href="#ref72" class="rightTabRefNumLink">72</a></strong><div class="NLM_citation" id="rightTab-cit72"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Seal, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Atkinson, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aylott, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bamborough, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Copley, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grandi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harrison, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayhow, T. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindon, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Messenger, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michon, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitchell, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Preston, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prinjha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rioja, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wall, I. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watson, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woolven, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Demont, E. H.</span></span> <span> </span><span class="NLM_article-title">The optimization of a novel, weak bromo and extra terminal domain (BET) bromodomain fragment ligand to a potent and selective second bromodomain (BD2) inhibitor</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">9093</span>– <span class="NLM_lpage">9126</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.0c00796</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.0c00796" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhsVCmurzI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=9093-9126&author=J.+T.+Sealauthor=S.+J.+Atkinsonauthor=H.+Aylottauthor=P.+Bamboroughauthor=C.+W.+Chungauthor=R.+Copleyauthor=L.+Gordonauthor=P.+Grandiauthor=J.+Grayauthor=L.+A.+Harrisonauthor=T.+G.+Hayhowauthor=M.+Lindonauthor=C.+Messengerauthor=A.+M.+Michonauthor=D.+Mitchellauthor=A.+Prestonauthor=R.+K.+Prinjhaauthor=I.+Riojaauthor=S.+Taylorauthor=I.+D.+Wallauthor=R.+J.+Watsonauthor=J.+M.+Woolvenauthor=E.+H.+Demont&title=The+optimization+of+a+novel%2C+weak+bromo+and+extra+terminal+domain+%28BET%29+bromodomain+fragment+ligand+to+a+potent+and+selective+second+bromodomain+%28BD2%29+inhibitor&doi=10.1021%2Facs.jmedchem.0c00796"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit72R"><div class="casContent"><span class="casTitleNuber">72</span><div class="casTitle"><span class="NLM_cas:atitle">The Optimization of a Novel, Weak Bromo and Extra Terminal Domain (BET) Bromodomain Fragment Ligand to a Potent and Selective Second Bromodomain (BD2) Inhibitor</span></div><div class="casAuthors">Seal, Jonathan T.; Atkinson, Stephen J.; Aylott, Helen; Bamborough, Paul; Chung, Chun-wa; Copley, Royston C. B.; Gordon, Laurie; Grandi, Paola; Gray, James R. J.; Harrison, Lee A.; Hayhow, Thomas G.; Lindon, Matthew; Messenger, Cassie; Michon, Anne-Marie; Mitchell, Darren; Preston, Alex; Prinjha, Rab K.; Rioja, Inmaculada; Taylor, Simon; Wall, Ian D.; Watson, Robert J.; Woolven, James M.; Demont, Emmanuel H.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">9093-9126</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The profound efficacy, yet assocd. toxicity of pan-BET inhibitors is well documented.  The possibility of an ameliorated safety profile driven by significantly selective (>100-fold) inhibition of a subset of the eight bromodomains is enticing, but challenging given the close homol.  Herein, we describe the X-ray crystal structure-directed optimization of a novel weak fragment ligand with a pan-second bromodomain (BD2) bias, to potent and highly BD2 selective inhibitors.  A template hopping approach, enabled by our parallel research into an orthogonal template (15, GSK046, I), was the basis for the high selectivity obsd.  This culminated in two tool mols., 20 (GSK620) and 56 (GSK549), which showed an anti-inflammatory phenotype in human whole blood, confirming their cellular target engagement.  Excellent broad selectivity, developability, and in vivo oral pharmacokinetics characterize these tools, which we hope will be of broad utility to the field of epigenetics research.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoS04XaSgoDyLVg90H21EOLACvtfcHk0lj2FpqwTcaO2Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhsVCmurzI&md5=c562a8c9e4c38abac04149ee96e2f594</span></div><a href="/servlet/linkout?suffix=cit72&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.0c00796&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.0c00796%26sid%3Dliteratum%253Aachs%26aulast%3DSeal%26aufirst%3DJ.%2BT.%26aulast%3DAtkinson%26aufirst%3DS.%2BJ.%26aulast%3DAylott%26aufirst%3DH.%26aulast%3DBamborough%26aufirst%3DP.%26aulast%3DChung%26aufirst%3DC.%2BW.%26aulast%3DCopley%26aufirst%3DR.%26aulast%3DGordon%26aufirst%3DL.%26aulast%3DGrandi%26aufirst%3DP.%26aulast%3DGray%26aufirst%3DJ.%26aulast%3DHarrison%26aufirst%3DL.%2BA.%26aulast%3DHayhow%26aufirst%3DT.%2BG.%26aulast%3DLindon%26aufirst%3DM.%26aulast%3DMessenger%26aufirst%3DC.%26aulast%3DMichon%26aufirst%3DA.%2BM.%26aulast%3DMitchell%26aufirst%3DD.%26aulast%3DPreston%26aufirst%3DA.%26aulast%3DPrinjha%26aufirst%3DR.%2BK.%26aulast%3DRioja%26aufirst%3DI.%26aulast%3DTaylor%26aufirst%3DS.%26aulast%3DWall%26aufirst%3DI.%2BD.%26aulast%3DWatson%26aufirst%3DR.%2BJ.%26aulast%3DWoolven%26aufirst%3DJ.%2BM.%26aulast%3DDemont%26aufirst%3DE.%2BH.%26atitle%3DThe%2520optimization%2520of%2520a%2520novel%252C%2520weak%2520bromo%2520and%2520extra%2520terminal%2520domain%2520%2528BET%2529%2520bromodomain%2520fragment%2520ligand%2520to%2520a%2520potent%2520and%2520selective%2520second%2520bromodomain%2520%2528BD2%2529%2520inhibitor%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D9093%26epage%3D9126%26doi%3D10.1021%2Facs.jmedchem.0c00796" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref73"><div class="reference"><strong class="refLabel"><a href="#ref73" class="rightTabRefNumLink">73</a></strong><div class="NLM_citation" id="rightTab-cit73"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Slavish, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chi, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yun, M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsurkan, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martinez, N. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jonchere, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chai, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Connelly, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waddell, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Das, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neale, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shadrick, W. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olsen, R. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Freeman, K. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Low, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Price, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, B. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bharatham, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boyd, V. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morfouace, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roussel, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Savic, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guy, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">White, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shelat, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Potter, P. M.</span></span> <span> </span><span class="NLM_article-title">Bromodomain-selective BET inhibitors are potent antitumor agents against MYC-driven pediatric cancer</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>80</i></span>,  <span class="NLM_fpage">3507</span>– <span class="NLM_lpage">3518</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.CAN-19-3934</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1158%2F0008-5472.CAN-19-3934" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=32651255" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BB3cXitF2nu7vP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=80&publication_year=2020&pages=3507-3518&author=P.+J.+Slavishauthor=L.+Chiauthor=M.+K.+Yunauthor=L.+Tsurkanauthor=N.+E.+Martinezauthor=B.+Jonchereauthor=S.+C.+Chaiauthor=M.+Connellyauthor=M.+B.+Waddellauthor=S.+Dasauthor=G.+Nealeauthor=Z.+Liauthor=W.+R.+Shadrickauthor=R.+R.+Olsenauthor=K.+W.+Freemanauthor=J.+A.+Lowauthor=J.+E.+Priceauthor=B.+M.+Youngauthor=N.+Bharathamauthor=V.+A.+Boydauthor=J.+Yangauthor=R.+E.+Leeauthor=M.+Morfouaceauthor=M.+F.+Rousselauthor=T.+Chenauthor=D.+Savicauthor=R.+K.+Guyauthor=S.+W.+Whiteauthor=A.+A.+Shelatauthor=P.+M.+Potter&title=Bromodomain-selective+BET+inhibitors+are+potent+antitumor+agents+against+MYC-driven+pediatric+cancer&doi=10.1158%2F0008-5472.CAN-19-3934"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit73R"><div class="casContent"><span class="casTitleNuber">73</span><div class="casTitle"><span class="NLM_cas:atitle">Bromodomain-selective BET inhibitors are potent antitumor agents against MYC-driven pediatric cancer</span></div><div class="casAuthors">Slavish, P. Jake; Chi, Liying; Yun, Mi-Kyung; Tsurkan, Lyudmila; Martinez, Nancy E.; Jonchere, Barbara; Chai, Sergio C.; Connelly, Michele; Waddell, M. Brett; Das, Sourav; Neale, Geoffrey; Li, Zhenmei; Shadrick, William R.; Olsen, Rachelle R.; Freeman, Kevin W.; Low, Jonathan A.; Price, Jeanine E.; Young, Brandon M.; Bharatham, Nagakumar; Boyd, Vincent A.; Yang, Jun; Lee, Richard E.; Morfouace, Marie; Roussel, Martine F.; Chen, Taosheng; Savic, Daniel; Guy, R. Kiplin; White, Stephen W.; Shelat, Anang A.; Potter, Philip M.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">80</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">3507-3518</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Inhibition of members of the bromodomain and extraterminal (BET) family of proteins has proven a valid strategy for cancer chemotherapy.  All BET identified to date contain two bromodomains (BD; BD1 and BD2) that are necessary for recognition of acetylated lysine residues in the N-terminal regions of histones.  Chem. matter that targets BET (BETi) also interact via these domains.  Mol. and cellular data indicate that BD1 and BD2 have different biol. roles depending upon their cellular context, with BD2 particularly assocd. with cancer.  We have therefore pursued the development of BD2-selective mols. both as chem. probes and as potential leads for drug development.  Here we report the structure-based generation of a novel series of tetrahydroquinoline analogs that exhibit >50-fold selectivity for BD2 vs. BD1.  This selective targeting resulted in engagement with BD-contg. proteins in cells, resulting in modulation of MYC proteins and downstream targets.  These compds. were potent cytotoxins toward numerous pediatric cancer cell lines and were minimally toxic to nontumorigenic cells.  In addn., unlike the pan BETi (+)-JQ1, these BD2-selective inhibitors demonstrated no rebound expression effects.  Finally, we report a pharmacokinetic-optimized, metabolically stable deriv. that induced growth delay in a neuroblastoma xenograft model with minimal toxicity.  We conclude that BD2-selective agents are valid candidates for antitumor drug design for pediatric malignancies driven by the MYC oncogene.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGphIgrDNGwlb7Vg90H21EOLACvtfcHk0lj2FpqwTcaO2Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXitF2nu7vP&md5=297abd22381a331adb3d68cd0b2aedea</span></div><a href="/servlet/linkout?suffix=cit73&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-19-3934&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-19-3934%26sid%3Dliteratum%253Aachs%26aulast%3DSlavish%26aufirst%3DP.%2BJ.%26aulast%3DChi%26aufirst%3DL.%26aulast%3DYun%26aufirst%3DM.%2BK.%26aulast%3DTsurkan%26aufirst%3DL.%26aulast%3DMartinez%26aufirst%3DN.%2BE.%26aulast%3DJonchere%26aufirst%3DB.%26aulast%3DChai%26aufirst%3DS.%2BC.%26aulast%3DConnelly%26aufirst%3DM.%26aulast%3DWaddell%26aufirst%3DM.%2BB.%26aulast%3DDas%26aufirst%3DS.%26aulast%3DNeale%26aufirst%3DG.%26aulast%3DLi%26aufirst%3DZ.%26aulast%3DShadrick%26aufirst%3DW.%2BR.%26aulast%3DOlsen%26aufirst%3DR.%2BR.%26aulast%3DFreeman%26aufirst%3DK.%2BW.%26aulast%3DLow%26aufirst%3DJ.%2BA.%26aulast%3DPrice%26aufirst%3DJ.%2BE.%26aulast%3DYoung%26aufirst%3DB.%2BM.%26aulast%3DBharatham%26aufirst%3DN.%26aulast%3DBoyd%26aufirst%3DV.%2BA.%26aulast%3DYang%26aufirst%3DJ.%26aulast%3DLee%26aufirst%3DR.%2BE.%26aulast%3DMorfouace%26aufirst%3DM.%26aulast%3DRoussel%26aufirst%3DM.%2BF.%26aulast%3DChen%26aufirst%3DT.%26aulast%3DSavic%26aufirst%3DD.%26aulast%3DGuy%26aufirst%3DR.%2BK.%26aulast%3DWhite%26aufirst%3DS.%2BW.%26aulast%3DShelat%26aufirst%3DA.%2BA.%26aulast%3DPotter%26aufirst%3DP.%2BM.%26atitle%3DBromodomain-selective%2520BET%2520inhibitors%2520are%2520potent%2520antitumor%2520agents%2520against%2520MYC-driven%2520pediatric%2520cancer%26jtitle%3DCancer%2520Res.%26date%3D2020%26volume%3D80%26spage%3D3507%26epage%3D3518%26doi%3D10.1158%2F0008-5472.CAN-19-3934" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref74"><div class="reference"><strong class="refLabel"><a href="#ref74" class="rightTabRefNumLink">74</a></strong><div class="NLM_citation" id="rightTab-cit74"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Leroy, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rickards, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flint, S. J.</span></span> <span> </span><span class="NLM_article-title">The double bromodomain proteins BRD2 and BRD3 couple histone acetylation to transcription</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">51</span>– <span class="NLM_lpage">60</span>, <span class="refDoi"> DOI: 10.1016/j.molcel.2008.01.018</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1016%2Fj.molcel.2008.01.018" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=18406326" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BD1cXkvFegurk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2008&pages=51-60&author=G.+Leroyauthor=B.+Rickardsauthor=S.+J.+Flint&title=The+double+bromodomain+proteins+BRD2+and+BRD3+couple+histone+acetylation+to+transcription&doi=10.1016%2Fj.molcel.2008.01.018"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit74R"><div class="casContent"><span class="casTitleNuber">74</span><div class="casTitle"><span class="NLM_cas:atitle">The double bromodomain proteins Brd2 and Brd3 couple histone acetylation to transcription</span></div><div class="casAuthors">LeRoy, Gary; Rickards, Brenden; Flint, S. J.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">51-60</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Posttranslational histone modifications are crucial for the modulation of chromatin structure and regulation of transcription.  Bromodomains present in many chromatin-assocd. proteins recognize acetylated lysines in the unstructured N-terminal regions of histones.  Here, we report that the double bromodomain proteins Brd2 and Brd3 assoc. preferentially in vivo with hyperacetylated chromatin along the entire lengths of transcribed genes.  Brd2- and Brd3-assocd. chromatin is significantly enriched in H4K5, H4K12, and H3K14 acetylation and contains relatively little dimethylated H3K9.  Both Brd2 and Brd3 allowed RNA polymerase II to transcribe through nucleosomes in a defined transcription system.  Such activity depended on specific histone H4 modifications known to be recognized by the Brd proteins.  We also demonstrate that Brd2 has intrinsic histone chaperone activity and is required for transcription of the cyclin D1 gene in vivo.  These data identify proteins that render nucleosomes marked by acetylation permissive to the passage of elongating RNA polymerase II.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoNyXwkpnVoo7Vg90H21EOLACvtfcHk0lgx4Vd0lYyhVw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXkvFegurk%253D&md5=63c9910684a1e43fe4ab1f6841443fc7</span></div><a href="/servlet/linkout?suffix=cit74&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2008.01.018&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2008.01.018%26sid%3Dliteratum%253Aachs%26aulast%3DLeroy%26aufirst%3DG.%26aulast%3DRickards%26aufirst%3DB.%26aulast%3DFlint%26aufirst%3DS.%2BJ.%26atitle%3DThe%2520double%2520bromodomain%2520proteins%2520BRD2%2520and%2520BRD3%2520couple%2520histone%2520acetylation%2520to%2520transcription%26jtitle%3DMol.%2520Cell%26date%3D2008%26volume%3D30%26spage%3D51%26epage%3D60%26doi%3D10.1016%2Fj.molcel.2008.01.018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref75"><div class="reference"><strong class="refLabel"><a href="#ref75" class="rightTabRefNumLink">75</a></strong><div class="NLM_citation" id="rightTab-cit75"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Loven, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoke, H. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, C. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lau, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Orlando, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vakoc, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, T. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, R. A.</span></span> <span> </span><span class="NLM_article-title">Selective inhibition of tumor oncogenes by disruption of super-enhancers</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>153</i></span>,  <span class="NLM_fpage">320</span>– <span class="NLM_lpage">334</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2013.03.036</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1016%2Fj.cell.2013.03.036" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=23582323" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BC3sXlvVCgsrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=153&publication_year=2013&pages=320-334&author=J.+Lovenauthor=H.+A.+Hokeauthor=C.+Y.+Linauthor=A.+Lauauthor=D.+A.+Orlandoauthor=C.+R.+Vakocauthor=J.+E.+Bradnerauthor=T.+I.+Leeauthor=R.+A.+Young&title=Selective+inhibition+of+tumor+oncogenes+by+disruption+of+super-enhancers&doi=10.1016%2Fj.cell.2013.03.036"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit75R"><div class="casContent"><span class="casTitleNuber">75</span><div class="casTitle"><span class="NLM_cas:atitle">Selective Inhibition of Tumor Oncogenes by Disruption of Super-Enhancers</span></div><div class="casAuthors">Loven, Jakob; Hoke, Heather A.; Lin, Charles Y.; Lau, Ashley; Orlando, David A.; Vakoc, Christopher R.; Bradner, James E.; Lee, Tong Ihn; Young, Richard A.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">153</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">320-334</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Chromatin regulators have become attractive targets for cancer therapy, but it is unclear why inhibition of these ubiquitous regulators should have gene-specific effects in tumor cells.  Here, we investigate how inhibition of the widely expressed transcriptional coactivator BRD4 leads to selective inhibition of the MYC oncogene in multiple myeloma (MM).  BRD4 and Mediator were found to co-occupy thousands of enhancers assocd. with active genes.  They also co-occupied a small set of exceptionally large super-enhancers assocd. with genes that feature prominently in MM biol., including the MYC oncogene.  Treatment of MM tumor cells with the BET-bromodomain inhibitor JQ1 led to preferential loss of BRD4 at super-enhancers and consequent transcription elongation defects that preferentially impacted genes with super-enhancers, including MYC.  Super-enhancers were found at key oncogenic drivers in many other tumor cells.  These observations have implications for the discovery of cancer therapeutics directed at components of super-enhancers in diverse tumor types.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrM-NqFYuGMqrVg90H21EOLACvtfcHk0lgx4Vd0lYyhVw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXlvVCgsrY%253D&md5=66b6ccb5b8a6039832910443fd8b4664</span></div><a href="/servlet/linkout?suffix=cit75&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2013.03.036&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2013.03.036%26sid%3Dliteratum%253Aachs%26aulast%3DLoven%26aufirst%3DJ.%26aulast%3DHoke%26aufirst%3DH.%2BA.%26aulast%3DLin%26aufirst%3DC.%2BY.%26aulast%3DLau%26aufirst%3DA.%26aulast%3DOrlando%26aufirst%3DD.%2BA.%26aulast%3DVakoc%26aufirst%3DC.%2BR.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26aulast%3DLee%26aufirst%3DT.%2BI.%26aulast%3DYoung%26aufirst%3DR.%2BA.%26atitle%3DSelective%2520inhibition%2520of%2520tumor%2520oncogenes%2520by%2520disruption%2520of%2520super-enhancers%26jtitle%3DCell%26date%3D2013%26volume%3D153%26spage%3D320%26epage%3D334%26doi%3D10.1016%2Fj.cell.2013.03.036" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref76"><div class="reference"><strong class="refLabel"><a href="#ref76" class="rightTabRefNumLink">76</a></strong><div class="NLM_citation" id="rightTab-cit76"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Denis, G. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mccomb, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Faller, D. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sinha, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Romesser, P. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Costello, C. E.</span></span> <span> </span><span class="NLM_article-title">Identification of transcription complexes that contain the double bromodomain protein BRD2 and chromatin remodeling machines</span>. <i>J. Proteome Res.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">502</span>– <span class="NLM_lpage">511</span>, <span class="refDoi"> DOI: 10.1021/pr050430u</span> </span><div class="citationLinks">[<a href="/doi/10.1021/pr050430u" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtlSru7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2006&pages=502-511&author=G.+V.+Denisauthor=M.+E.+Mccombauthor=D.+V.+Fallerauthor=A.+Sinhaauthor=P.+B.+Romesserauthor=C.+E.+Costello&title=Identification+of+transcription+complexes+that+contain+the+double+bromodomain+protein+BRD2+and+chromatin+remodeling+machines&doi=10.1021%2Fpr050430u"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit76R"><div class="casContent"><span class="casTitleNuber">76</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of Transcription Complexes that Contain the Double Bromodomain Protein Brd2 and Chromatin Remodeling Machines</span></div><div class="casAuthors">Denis, Gerald V.; McComb, Mark E.; Faller, Douglas V.; Sinha, Anupama; Romesser, Paul B.; Costello, Catherine E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Proteome Research</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">502-511</span>CODEN:
                <span class="NLM_cas:coden">JPROBS</span>;
        ISSN:<span class="NLM_cas:issn">1535-3893</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We use affinity purifn. of the double bromodomain protein Brd2 to isolate a multicomponent nuclear complex from cultured cells, and apply mass spectrometry/proteomics methods to identify the participants.  We then confirm by immunoblot several transcription co-activators and co-repressors, proteins of the Swi/Snf chromatin remodeling complex, which regulate transcription control of cyclin A.  This multiprotein complex is likely to contribute to cell cycle control and play a role in proliferation and cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGooVmPLcNO2ILVg90H21EOLACvtfcHk0lgx4Vd0lYyhVw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtlSru7Y%253D&md5=42184d6213946e9110843b36709a3c81</span></div><a href="/servlet/linkout?suffix=cit76&amp;dbid=16384&amp;doi=10.1021%2Fpr050430u&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fpr050430u%26sid%3Dliteratum%253Aachs%26aulast%3DDenis%26aufirst%3DG.%2BV.%26aulast%3DMccomb%26aufirst%3DM.%2BE.%26aulast%3DFaller%26aufirst%3DD.%2BV.%26aulast%3DSinha%26aufirst%3DA.%26aulast%3DRomesser%26aufirst%3DP.%2BB.%26aulast%3DCostello%26aufirst%3DC.%2BE.%26atitle%3DIdentification%2520of%2520transcription%2520complexes%2520that%2520contain%2520the%2520double%2520bromodomain%2520protein%2520BRD2%2520and%2520chromatin%2520remodeling%2520machines%26jtitle%3DJ.%2520Proteome%2520Res.%26date%3D2006%26volume%3D5%26spage%3D502%26epage%3D511%26doi%3D10.1021%2Fpr050430u" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref77"><div class="reference"><strong class="refLabel"><a href="#ref77" class="rightTabRefNumLink">77</a></strong><div class="NLM_citation" id="rightTab-cit77"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lambert, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Picaud, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fujisawa, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hou, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Savitsky, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uuskula-Reimand, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gupta, G. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abdouni, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, Z. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tucholska, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knight, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gonzalez-Badillo, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">St-Denis, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Newman, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stucki, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pelletier, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bandeira, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gingras, A. C.</span></span> <span> </span><span class="NLM_article-title">Interactome rewiring following pharmacological targeting of BET bromodomains</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>73</i></span>,  <span class="NLM_fpage">621</span>– <span class="NLM_lpage">638</span>, <span class="refDoi"> DOI: 10.1016/j.molcel.2018.11.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1016%2Fj.molcel.2018.11.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=30554943" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BC1cXisFWqsbjO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=73&publication_year=2019&pages=621-638&author=J.+P.+Lambertauthor=S.+Picaudauthor=T.+Fujisawaauthor=H.+Houauthor=P.+Savitskyauthor=L.+Uuskula-Reimandauthor=G.+D.+Guptaauthor=H.+Abdouniauthor=Z.+Y.+Linauthor=M.+Tucholskaauthor=J.+Knightauthor=B.+Gonzalez-Badilloauthor=N.+St-Denisauthor=J.+A.+Newmanauthor=M.+Stuckiauthor=L.+Pelletierauthor=N.+Bandeiraauthor=M.+D.+Wilsonauthor=P.+Filippakopoulosauthor=A.+C.+Gingras&title=Interactome+rewiring+following+pharmacological+targeting+of+BET+bromodomains&doi=10.1016%2Fj.molcel.2018.11.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit77R"><div class="casContent"><span class="casTitleNuber">77</span><div class="casTitle"><span class="NLM_cas:atitle">Interactome Rewiring Following Pharmacological Targeting of BET Bromodomains</span></div><div class="casAuthors">Lambert, Jean-Philippe; Picaud, Sarah; Fujisawa, Takao; Hou, Huayun; Savitsky, Pavel; Uuskula-Reimand, Liis; Gupta, Gagan D.; Abdouni, Hala; Lin, Zhen-Yuan; Tucholska, Monika; Knight, James D. R.; Gonzalez-Badillo, Beatriz; St-Denis, Nicole; Newman, Joseph A.; Stucki, Manuel; Pelletier, Laurence; Bandeira, Nuno; Wilson, Michael D.; Filippakopoulos, Panagis; Gingras, Anne-Claude</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">73</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">621-638.e17</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Targeting bromodomains (BRDs) of the bromo-and-extra-terminal (BET) family offers opportunities for therapeutic intervention in cancer and other diseases.  Here, we profile the interactomes of BRD2, BRD3, BRD4, and BRDT following treatment with the pan-BET BRD inhibitor JQ1, revealing broad rewiring of the interaction landscape, with three distinct classes of behavior for the 603 unique interactors identified.  A group of proteins assoc. in a JQ1-sensitive manner with BET BRDs through canonical and new binding modes, while two classes of extra-terminal (ET)-domain binding motifs mediate acetylation-independent interactions.  Last, we identify an unexpected increase in several interactions following JQ1 treatment that define neg. functions for BRD3 in the regulation of rRNA synthesis and potentially RNAPII-dependent gene expression that result in decreased cell proliferation.  Together, our data highlight the contributions of BET protein modules to their interactomes allowing for a better understanding of pharmacol. rewiring in response to JQ1.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGojeAO2NIyKGLVg90H21EOLACvtfcHk0ljPPrmYhAYmpw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXisFWqsbjO&md5=70895d9ff04263449c67fbea254140b5</span></div><a href="/servlet/linkout?suffix=cit77&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2018.11.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2018.11.006%26sid%3Dliteratum%253Aachs%26aulast%3DLambert%26aufirst%3DJ.%2BP.%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DFujisawa%26aufirst%3DT.%26aulast%3DHou%26aufirst%3DH.%26aulast%3DSavitsky%26aufirst%3DP.%26aulast%3DUuskula-Reimand%26aufirst%3DL.%26aulast%3DGupta%26aufirst%3DG.%2BD.%26aulast%3DAbdouni%26aufirst%3DH.%26aulast%3DLin%26aufirst%3DZ.%2BY.%26aulast%3DTucholska%26aufirst%3DM.%26aulast%3DKnight%26aufirst%3DJ.%26aulast%3DGonzalez-Badillo%26aufirst%3DB.%26aulast%3DSt-Denis%26aufirst%3DN.%26aulast%3DNewman%26aufirst%3DJ.%2BA.%26aulast%3DStucki%26aufirst%3DM.%26aulast%3DPelletier%26aufirst%3DL.%26aulast%3DBandeira%26aufirst%3DN.%26aulast%3DWilson%26aufirst%3DM.%2BD.%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DGingras%26aufirst%3DA.%2BC.%26atitle%3DInteractome%2520rewiring%2520following%2520pharmacological%2520targeting%2520of%2520BET%2520bromodomains%26jtitle%3DMol.%2520Cell%26date%3D2019%26volume%3D73%26spage%3D621%26epage%3D638%26doi%3D10.1016%2Fj.molcel.2018.11.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref78"><div class="reference"><strong class="refLabel"><a href="#ref78" class="rightTabRefNumLink">78</a></strong><div class="NLM_citation" id="rightTab-cit78"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Leroy, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rickards, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flint, S. J.</span></span> <span> </span><span class="NLM_article-title">The double bromodomain proteins BRD2 and BRD3 couple histone acetylation to transcription</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">51</span>– <span class="NLM_lpage">60</span>, <span class="refDoi"> DOI: 10.1016/j.molcel.2008.01.018</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1016%2Fj.molcel.2008.01.018" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=18406326" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BD1cXkvFegurk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2008&pages=51-60&author=G.+Leroyauthor=B.+Rickardsauthor=S.+J.+Flint&title=The+double+bromodomain+proteins+BRD2+and+BRD3+couple+histone+acetylation+to+transcription&doi=10.1016%2Fj.molcel.2008.01.018"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit78R"><div class="casContent"><span class="casTitleNuber">78</span><div class="casTitle"><span class="NLM_cas:atitle">The double bromodomain proteins Brd2 and Brd3 couple histone acetylation to transcription</span></div><div class="casAuthors">LeRoy, Gary; Rickards, Brenden; Flint, S. J.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">51-60</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Posttranslational histone modifications are crucial for the modulation of chromatin structure and regulation of transcription.  Bromodomains present in many chromatin-assocd. proteins recognize acetylated lysines in the unstructured N-terminal regions of histones.  Here, we report that the double bromodomain proteins Brd2 and Brd3 assoc. preferentially in vivo with hyperacetylated chromatin along the entire lengths of transcribed genes.  Brd2- and Brd3-assocd. chromatin is significantly enriched in H4K5, H4K12, and H3K14 acetylation and contains relatively little dimethylated H3K9.  Both Brd2 and Brd3 allowed RNA polymerase II to transcribe through nucleosomes in a defined transcription system.  Such activity depended on specific histone H4 modifications known to be recognized by the Brd proteins.  We also demonstrate that Brd2 has intrinsic histone chaperone activity and is required for transcription of the cyclin D1 gene in vivo.  These data identify proteins that render nucleosomes marked by acetylation permissive to the passage of elongating RNA polymerase II.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoNyXwkpnVoo7Vg90H21EOLACvtfcHk0li-E3E4KN2dFw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXkvFegurk%253D&md5=63c9910684a1e43fe4ab1f6841443fc7</span></div><a href="/servlet/linkout?suffix=cit78&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2008.01.018&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2008.01.018%26sid%3Dliteratum%253Aachs%26aulast%3DLeroy%26aufirst%3DG.%26aulast%3DRickards%26aufirst%3DB.%26aulast%3DFlint%26aufirst%3DS.%2BJ.%26atitle%3DThe%2520double%2520bromodomain%2520proteins%2520BRD2%2520and%2520BRD3%2520couple%2520histone%2520acetylation%2520to%2520transcription%26jtitle%3DMol.%2520Cell%26date%3D2008%26volume%3D30%26spage%3D51%26epage%3D60%26doi%3D10.1016%2Fj.molcel.2008.01.018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref79"><div class="reference"><strong class="refLabel"><a href="#ref79" class="rightTabRefNumLink">79</a></strong><div class="NLM_citation" id="rightTab-cit79"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lamonica, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kadauke, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campbell, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gamsjaeger, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Billin, A. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hardison, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mackay, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blobel, G. A.</span></span> <span> </span><span class="NLM_article-title">Bromodomain protein BRD3 associates with acetylated GATA1 to promote its chromatin occupancy at erythroid target genes</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>108</i></span>,  <span class="NLM_fpage">E159</span>– <span class="NLM_lpage">E168</span>, <span class="refDoi"> DOI: 10.1073/pnas.1102140108</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1073%2Fpnas.1102140108" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=21536911" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A280%3ADC%252BC3MrmtlGgtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=108&publication_year=2011&pages=E159-E168&author=J.+M.+Lamonicaauthor=W.+Dengauthor=S.+Kadaukeauthor=A.+E.+Campbellauthor=R.+Gamsjaegerauthor=H.+Wangauthor=Y.+Chengauthor=A.+N.+Billinauthor=R.+C.+Hardisonauthor=J.+P.+Mackayauthor=G.+A.+Blobel&title=Bromodomain+protein+BRD3+associates+with+acetylated+GATA1+to+promote+its+chromatin+occupancy+at+erythroid+target+genes&doi=10.1073%2Fpnas.1102140108"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit79R"><div class="casContent"><span class="casTitleNuber">79</span><div class="casTitle"><span class="NLM_cas:atitle">Bromodomain protein Brd3 associates with acetylated GATA1 to promote its chromatin occupancy at erythroid target genes</span></div><div class="casAuthors">Lamonica Janine M; Deng Wulan; Kadauke Stephan; Campbell Amy E; Gamsjaeger Roland; Wang Hongxin; Cheng Yong; Billin Andrew N; Hardison Ross C; Mackay Joel P; Blobel Gerd A</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">108</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">E159-68</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Acetylation of histones triggers association with bromodomain-containing proteins that regulate diverse chromatin-related processes.  Although acetylation of transcription factors has been appreciated for some time, the mechanistic consequences are less well understood.  The hematopoietic transcription factor GATA1 is acetylated at conserved lysines that are required for its stable association with chromatin.  We show that the BET family protein Brd3 binds via its first bromodomain (BD1) to GATA1 in an acetylation-dependent manner in vitro and in vivo.  Mutation of a single residue in BD1 that is involved in acetyl-lysine binding abrogated recruitment of Brd3 by GATA1, demonstrating that acetylation of GATA1 is essential for Brd3 association with chromatin.  Notably, Brd3 is recruited by GATA1 to both active and repressed target genes in a fashion seemingly independent of histone acetylation.  Anti-Brd3 ChIP followed by massively parallel sequencing in GATA1-deficient erythroid precursor cells and those that are GATA1 replete revealed that GATA1 is a major determinant of Brd3 recruitment to genomic targets within chromatin.  A pharmacologic compound that occupies the acetyl-lysine binding pockets of Brd3 bromodomains disrupts the Brd3-GATA1 interaction, diminishes the chromatin occupancy of both proteins, and inhibits erythroid maturation.  Together these findings provide a mechanism for GATA1 acetylation and suggest that Brd3 "reads" acetyl marks on nuclear factors to promote their stable association with chromatin.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQeZ8VSdvvG7p6KlWarZTAzfW6udTcc2eYtRWrzPLfh7bntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3MrmtlGgtQ%253D%253D&md5=b81056e4aee1351dce3899fb8d4e86e8</span></div><a href="/servlet/linkout?suffix=cit79&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1102140108&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1102140108%26sid%3Dliteratum%253Aachs%26aulast%3DLamonica%26aufirst%3DJ.%2BM.%26aulast%3DDeng%26aufirst%3DW.%26aulast%3DKadauke%26aufirst%3DS.%26aulast%3DCampbell%26aufirst%3DA.%2BE.%26aulast%3DGamsjaeger%26aufirst%3DR.%26aulast%3DWang%26aufirst%3DH.%26aulast%3DCheng%26aufirst%3DY.%26aulast%3DBillin%26aufirst%3DA.%2BN.%26aulast%3DHardison%26aufirst%3DR.%2BC.%26aulast%3DMackay%26aufirst%3DJ.%2BP.%26aulast%3DBlobel%26aufirst%3DG.%2BA.%26atitle%3DBromodomain%2520protein%2520BRD3%2520associates%2520with%2520acetylated%2520GATA1%2520to%2520promote%2520its%2520chromatin%2520occupancy%2520at%2520erythroid%2520target%2520genes%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2011%26volume%3D108%26spage%3DE159%26epage%3DE168%26doi%3D10.1073%2Fpnas.1102140108" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref80"><div class="reference"><strong class="refLabel"><a href="#ref80" class="rightTabRefNumLink">80</a></strong><div class="NLM_citation" id="rightTab-cit80"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Matzuk, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mckeown, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Agno, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lemieux, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Picaud, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, R. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span> <span> </span><span class="NLM_article-title">Small-molecule inhibition of BRDT for male contraception</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>150</i></span>,  <span class="NLM_fpage">673</span>– <span class="NLM_lpage">684</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2012.06.045</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1016%2Fj.cell.2012.06.045" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=22901802" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BC38Xht1aktrbO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=150&publication_year=2012&pages=673-684&author=M.+M.+Matzukauthor=M.+R.+Mckeownauthor=P.+Filippakopoulosauthor=Q.+Liauthor=L.+Maauthor=J.+E.+Agnoauthor=M.+E.+Lemieuxauthor=S.+Picaudauthor=R.+N.+Yuauthor=J.+Qiauthor=S.+Knappauthor=J.+E.+Bradner&title=Small-molecule+inhibition+of+BRDT+for+male+contraception&doi=10.1016%2Fj.cell.2012.06.045"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit80R"><div class="casContent"><span class="casTitleNuber">80</span><div class="casTitle"><span class="NLM_cas:atitle">Small-Molecule Inhibition of BRDT for Male Contraception</span></div><div class="casAuthors">Matzuk, Martin M.; McKeown, Michael R.; Filippakopoulos, Panagis; Li, Qinglei; Ma, Lang; Agno, Julio E.; Lemieux, Madeleine E.; Picaud, Sarah; Yu, Richard N.; Qi, Jun; Knapp, Stefan; Bradner, James E.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">150</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">673-684</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A pharmacol. approach to male contraception remains a longstanding challenge in medicine.  Toward this objective, the authors explored the spermatogenic effects of a selective small-mol. inhibitor (JQ1) of the bromodomain and extraterminal (BET) subfamily of epigenetic reader proteins.  Here, the authors report potent inhibition of the testis-specific member BRDT, which is essential for chromatin remodeling during spermatogenesis.  Biochem. and crystallog. studies confirm that occupancy of the BRDT acetyl-lysine binding pocket by JQ1 prevents recognition of acetylated histone H4.  Treatment of mice with JQ1 reduced seminiferous tubule area, testis size, and spermatozoa no. and motility without affecting hormone levels.  Although JQ1-treated males mate normally, inhibitory effects of JQ1 evident at the spermatocyte and round spermatid stages cause a complete and reversible contraceptive effect.  These data establish a new contraceptive that can cross the blood:testis boundary and inhibit bromodomain activity during spermatogenesis, providing a lead compd. targeting the male germ cell for contraception.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqlxVWUgXTC77Vg90H21EOLACvtfcHk0lj8HmfPMfQh3w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xht1aktrbO&md5=ccfa1b443c156612714e4b7f88590041</span></div><a href="/servlet/linkout?suffix=cit80&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2012.06.045&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2012.06.045%26sid%3Dliteratum%253Aachs%26aulast%3DMatzuk%26aufirst%3DM.%2BM.%26aulast%3DMckeown%26aufirst%3DM.%2BR.%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DLi%26aufirst%3DQ.%26aulast%3DMa%26aufirst%3DL.%26aulast%3DAgno%26aufirst%3DJ.%2BE.%26aulast%3DLemieux%26aufirst%3DM.%2BE.%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DYu%26aufirst%3DR.%2BN.%26aulast%3DQi%26aufirst%3DJ.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26atitle%3DSmall-molecule%2520inhibition%2520of%2520BRDT%2520for%2520male%2520contraception%26jtitle%3DCell%26date%3D2012%26volume%3D150%26spage%3D673%26epage%3D684%26doi%3D10.1016%2Fj.cell.2012.06.045" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref81"><div class="reference"><strong class="refLabel"><a href="#ref81" class="rightTabRefNumLink">81</a></strong><div class="NLM_citation" id="rightTab-cit81"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bryant, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berger, S. L.</span></span> <span> </span><span class="NLM_article-title">Low-hanging fruit: targeting BRDT in the testes</span>. <i>EMBO J.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>31</i></span>,  <span class="NLM_fpage">3788</span>– <span class="NLM_lpage">3789</span>, <span class="refDoi"> DOI: 10.1038/emboj.2012.259</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1038%2Femboj.2012.259" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=22960636" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtlWgu7rP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2012&pages=3788-3789&author=J.+M.+Bryantauthor=S.+L.+Berger&title=Low-hanging+fruit%3A+targeting+BRDT+in+the+testes&doi=10.1038%2Femboj.2012.259"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit81R"><div class="casContent"><span class="casTitleNuber">81</span><div class="casTitle"><span class="NLM_cas:atitle">Low-hanging fruit: targeting Brdt in the testes</span></div><div class="casAuthors">Bryant, Jessica M.; Berger, Shelley L.</div><div class="citationInfo"><span class="NLM_cas:title">EMBO Journal</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">3788-3789</span>CODEN:
                <span class="NLM_cas:coden">EMJODG</span>;
        ISSN:<span class="NLM_cas:issn">0261-4189</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The title research of Gaucher, J. et al. (ibid, advance online publication, 24 August 2012; doi:10.1038/emboj.2012.233) and Matzuk, M. M. et al. (Cell, 2012, 150:pg. 673-684) is reviewed with commentary.  In this issue of The EMBO Journal, Gaucher et al. (2012) demonstrate that Brdt, a testes-specific member of the BET family of double bromodomain proteins, is intricately involved in the transcriptional regulation and genome reorganization required for the prodn. of viable, fertile sperm.  These findings are complimented by a recent publication in Cell (Matzuk et al, 2012) where JQ1, a small mol. BET inhibitor, is effective as a male contraceptive in mice.  Both studies establish Brdt as an integral factor in sperm development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGouzkYXqg4y87Vg90H21EOLACvtfcHk0lj8HmfPMfQh3w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtlWgu7rP&md5=0d779b87988844846f2549e375a16230</span></div><a href="/servlet/linkout?suffix=cit81&amp;dbid=16384&amp;doi=10.1038%2Femboj.2012.259&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Femboj.2012.259%26sid%3Dliteratum%253Aachs%26aulast%3DBryant%26aufirst%3DJ.%2BM.%26aulast%3DBerger%26aufirst%3DS.%2BL.%26atitle%3DLow-hanging%2520fruit%253A%2520targeting%2520BRDT%2520in%2520the%2520testes%26jtitle%3DEMBO%2520J.%26date%3D2012%26volume%3D31%26spage%3D3788%26epage%3D3789%26doi%3D10.1038%2Femboj.2012.259" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref82"><div class="reference"><strong class="refLabel"><a href="#ref82" class="rightTabRefNumLink">82</a></strong><div class="NLM_citation" id="rightTab-cit82"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tian, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brasier, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, J.</span></span> <span> </span><span class="NLM_article-title">Discovery of potent and selective BRD4 inhibitors capable of blocking TLR3-induced acute airway inflammation</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>151</i></span>,  <span class="NLM_fpage">450</span>– <span class="NLM_lpage">461</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2018.04.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1016%2Fj.ejmech.2018.04.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=29649741" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BC1cXnt1KgsLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=151&publication_year=2018&pages=450-461&author=Z.+Liuauthor=B.+Tianauthor=H.+Chenauthor=P.+Wangauthor=A.+R.+Brasierauthor=J.+Zhou&title=Discovery+of+potent+and+selective+BRD4+inhibitors+capable+of+blocking+TLR3-induced+acute+airway+inflammation&doi=10.1016%2Fj.ejmech.2018.04.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit82R"><div class="casContent"><span class="casTitleNuber">82</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of potent and selective BRD4 inhibitors capable of blocking TLR3-induced acute airway inflammation</span></div><div class="casAuthors">Liu, Zhiqing; Tian, Bing; Chen, Haiying; Wang, Pingyuan; Brasier, Allan R.; Zhou, Jia</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">151</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">450-461</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">A series of diverse small mols. have been designed and synthesized through structure-based drug design by taking advantage of fragment merging and elaboration approaches.  Compds. ZL0420 (28) and ZL0454 (35) were identified as potent and selective BRD4 inhibitors with nanomolar binding affinities to bromodomains (BDs) of BRD4.  Both of them can be well docked into the acetyl-lysine (KAc) binding pocket of BRD4, forming key interactions including the crit. hydrogen bonds with Asn140 directly and Tyr97 indirectly via a H2O mol.  Both compds. 28 and 35 exhibited submicromolar potency of inhibiting the TLR3-dependent innate immune gene program, including ISG54, ISG56, IL-8, and Groβ genes in cultured human small airway epithelial cells (hSAECs).  More importantly, they also demonstrated potent efficacy reducing airway inflammation in a mouse model with low toxicity, indicating a proof of concept that BRD4 inhibitors may offer the therapeutic potential to block the viral-induced airway inflammation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqLyejdKuiGi7Vg90H21EOLACvtfcHk0lj8HmfPMfQh3w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXnt1KgsLc%253D&md5=564b088a350abef9964d97bbecbb52b1</span></div><a href="/servlet/linkout?suffix=cit82&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2018.04.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2018.04.006%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DZ.%26aulast%3DTian%26aufirst%3DB.%26aulast%3DChen%26aufirst%3DH.%26aulast%3DWang%26aufirst%3DP.%26aulast%3DBrasier%26aufirst%3DA.%2BR.%26aulast%3DZhou%26aufirst%3DJ.%26atitle%3DDiscovery%2520of%2520potent%2520and%2520selective%2520BRD4%2520inhibitors%2520capable%2520of%2520blocking%2520TLR3-induced%2520acute%2520airway%2520inflammation%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2018%26volume%3D151%26spage%3D450%26epage%3D461%26doi%3D10.1016%2Fj.ejmech.2018.04.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref83"><div class="reference"><strong class="refLabel"><a href="#ref83" class="rightTabRefNumLink">83</a></strong><div class="NLM_citation" id="rightTab-cit83"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lv, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mou, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fan, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pei, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiong, B.</span></span> <span> </span><span class="NLM_article-title">Rational design and evaluation of 6-(Pyrimidin-2-ylamino)-3,4-dihydroquinoxalin-2(1H)-ones as polypharmacological inhibitors of BET and kinases</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">9787</span>– <span class="NLM_lpage">9802</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.0c00962</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.0c00962" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref83/cit83&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhsFOisLvN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=9787-9802&author=K.+Lvauthor=W.+Chenauthor=D.+Chenauthor=J.+Mouauthor=H.+Zhangauthor=T.+Fanauthor=Y.+Liauthor=D.+Caoauthor=X.+Wangauthor=L.+Chenauthor=J.+Shenauthor=D.+Peiauthor=B.+Xiong&title=Rational+design+and+evaluation+of+6-%28Pyrimidin-2-ylamino%29-3%2C4-dihydroquinoxalin-2%281H%29-ones+as+polypharmacological+inhibitors+of+BET+and+kinases&doi=10.1021%2Facs.jmedchem.0c00962"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit83R"><div class="casContent"><span class="casTitleNuber">83</span><div class="casTitle"><span class="NLM_cas:atitle">Rational Design and Evaluation of 6-(Pyrimidin-2-ylamino)-3,4-dihydroquinoxalin-2(1H)-ones as Polypharmacological Inhibitors of BET and Kinases</span></div><div class="casAuthors">Lv, Kaikai; Chen, Weicong; Chen, Danqi; Mou, Jie; Zhang, Huijie; Fan, Tiantian; Li, Yanlian; Cao, Danyan; Wang, Xin; Chen, Lin; Shen, Jingkang; Pei, Dongsheng; Xiong, Bing</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">9787-9802</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Cancer exhibits diverse heterogeneity with a complicated mol. basis that usually harbors genetic and epigenetic abnormality, which poses a big challenge for single-target agents.  In the current work, we proposed a hybrid strategy by incorporating pharmacophores that bind to the acetylated lysine binding pocket of BET proteins with a typical kinase hinge binder to generate novel polypharmacol. inhibitors of BET and kinases.  Through elaborating the core structure of 6-(pyrimidin-2-ylamino)-3,4-dihydroquinoxalin-2(1H)-one, we demonstrated that this rational design can produce high potent inhibitors of CDK9 and BET proteins.  In this series, compd. 40(I) was identified as the potential lead compd. with balanced activities of BRD4 (IC50 = 12.7 nM) and CDK9 (IC50 = 22.4 nM), as well as good antiproliferative activities on a small cancer cell panel.  Together, the current study provided a new method for the discovery of bromodomain and kinase dual inhibitors rather than only being discovered by serendipity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGquGRr7R1VS9LVg90H21EOLACvtfcHk0lhLTx95vCgyNA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhsFOisLvN&md5=4305f1e5c45c8e0dd994b042a4ee3bb7</span></div><a href="/servlet/linkout?suffix=cit83&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.0c00962&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.0c00962%26sid%3Dliteratum%253Aachs%26aulast%3DLv%26aufirst%3DK.%26aulast%3DChen%26aufirst%3DW.%26aulast%3DChen%26aufirst%3DD.%26aulast%3DMou%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DH.%26aulast%3DFan%26aufirst%3DT.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DCao%26aufirst%3DD.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DChen%26aufirst%3DL.%26aulast%3DShen%26aufirst%3DJ.%26aulast%3DPei%26aufirst%3DD.%26aulast%3DXiong%26aufirst%3DB.%26atitle%3DRational%2520design%2520and%2520evaluation%2520of%25206-%2528Pyrimidin-2-ylamino%2529-3%252C4-dihydroquinoxalin-2%25281H%2529-ones%2520as%2520polypharmacological%2520inhibitors%2520of%2520BET%2520and%2520kinases%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D9787%26epage%3D9802%26doi%3D10.1021%2Facs.jmedchem.0c00962" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref84"><div class="reference"><strong class="refLabel"><a href="#ref84" class="rightTabRefNumLink">84</a></strong><div class="NLM_citation" id="rightTab-cit84"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Martin, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olesen, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Georg, G. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schonbrunn, E.</span></span> <span> </span><span class="NLM_article-title">Cyclin-dependent kinase inhibitor dinaciclib interacts with the acetyl-lysine recognition site of bromodomains</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">2360</span>– <span class="NLM_lpage">2365</span>, <span class="refDoi"> DOI: 10.1021/cb4003283</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb4003283" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref84/cit84&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtl2rtbvN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&pages=2360-2365&author=M.+P.+Martinauthor=S.+H.+Olesenauthor=G.+I.+Georgauthor=E.+Schonbrunn&title=Cyclin-dependent+kinase+inhibitor+dinaciclib+interacts+with+the+acetyl-lysine+recognition+site+of+bromodomains&doi=10.1021%2Fcb4003283"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit84R"><div class="casContent"><span class="casTitleNuber">84</span><div class="casTitle"><span class="NLM_cas:atitle">Cyclin-Dependent Kinase Inhibitor Dinaciclib Interacts with the Acetyl-Lysine Recognition Site of Bromodomains</span></div><div class="casAuthors">Martin, Mathew P.; Olesen, Sanne H.; Georg, Gunda I.; Schonbrunn, Ernst</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2360-2365</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Bromodomain-contg. proteins are considered atypical kinases, but their potential to interact with kinase inhibitors is unknown.  Dinaciclib is a potent inhibitor of cyclin-dependent kinases (CDKs), which recently advanced to Phase III clin. trials for the treatment of leukemia.  The authors detd. the crystal structure of dinaciclib in complex with CDK2 at 1.7 Å resoln., revealing an elaborate network of binding interactions in the ATP site, which explains the extraordinary potency and selectivity of this inhibitor.  Remarkably, dinaciclib also interacted with the acetyl-lysine recognition site of the bromodomain testis-specific protein BRDT, a member of the BET family of bromodomains.  The binding mode of dinaciclib to BRDT at 2.0 Å resoln. suggests that general kinase inhibitors (hinge binders) possess a previously unrecognized potential to act as protein-protein inhibitors of bromodomains.  The findings may provide a new structural framework for the design of next-generation bromodomain inhibitors using the vast chem. space of kinase inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGonrxl9QUxkE7Vg90H21EOLACvtfcHk0liv7fu6qGq81g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtl2rtbvN&md5=1ae089e8e571ad11aadcdc4a1a5769b5</span></div><a href="/servlet/linkout?suffix=cit84&amp;dbid=16384&amp;doi=10.1021%2Fcb4003283&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb4003283%26sid%3Dliteratum%253Aachs%26aulast%3DMartin%26aufirst%3DM.%2BP.%26aulast%3DOlesen%26aufirst%3DS.%2BH.%26aulast%3DGeorg%26aufirst%3DG.%2BI.%26aulast%3DSchonbrunn%26aufirst%3DE.%26atitle%3DCyclin-dependent%2520kinase%2520inhibitor%2520dinaciclib%2520interacts%2520with%2520the%2520acetyl-lysine%2520recognition%2520site%2520of%2520bromodomains%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2013%26volume%3D8%26spage%3D2360%26epage%3D2365%26doi%3D10.1021%2Fcb4003283" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref85"><div class="reference"><strong class="refLabel"><a href="#ref85" class="rightTabRefNumLink">85</a></strong><div class="NLM_citation" id="rightTab-cit85"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Carlino, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rastelli, G.</span></span> <span> </span><span class="NLM_article-title">Dual kinase-bromodomain inhibitors in anticancer drug discovery: a structural and pharmacological perspective</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">9305</span>– <span class="NLM_lpage">9320</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b00438</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b00438" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref85/cit85&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsVWnt7rP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=9305-9320&author=L.+Carlinoauthor=G.+Rastelli&title=Dual+kinase-bromodomain+inhibitors+in+anticancer+drug+discovery%3A+a+structural+and+pharmacological+perspective&doi=10.1021%2Facs.jmedchem.6b00438"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit85R"><div class="casContent"><span class="casTitleNuber">85</span><div class="casTitle"><span class="NLM_cas:atitle">Dual Kinase-Bromodomain Inhibitors in Anticancer Drug Discovery: A Structural and Pharmacological Perspective</span></div><div class="casAuthors">Carlino, Luca; Rastelli, Giulio</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">9305-9320</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Protein kinases play crucial roles in several cell transformation processes and are validated drug targets for many human diseases, including cancer.  Nevertheless, most tumors have eluded the effects of inhibition of a single kinase by activating resistance mechanisms and/or alternative pathways and escape mechanisms.  In recent years, multitarget approaches directed toward inhibition of kinases and targets of different families have received increasing attention.  In particular, co-targeting kinases and bromodomain epigenetic reader proteins has rapidly emerged as a promising approach to cancer drug development.  In this manuscript, we will review the recent discoveries that led to the identification and optimization of dual kinase/bromodomain inhibitors.  We will analyze and compare the structural features required for dual inhibition and comment on the potential of this approach in anticancer drug discovery.  Moreover, we will introduce computational approaches useful for the identification of dual kinase/bromodomain inhibitors and generate ad hoc pharmacophore and docking models.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqTkHDSBWCGdbVg90H21EOLACvtfcHk0liv7fu6qGq81g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsVWnt7rP&md5=1351a235bba09782160a86ae3f904833</span></div><a href="/servlet/linkout?suffix=cit85&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b00438&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b00438%26sid%3Dliteratum%253Aachs%26aulast%3DCarlino%26aufirst%3DL.%26aulast%3DRastelli%26aufirst%3DG.%26atitle%3DDual%2520kinase-bromodomain%2520inhibitors%2520in%2520anticancer%2520drug%2520discovery%253A%2520a%2520structural%2520and%2520pharmacological%2520perspective%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D9305%26epage%3D9320%26doi%3D10.1021%2Facs.jmedchem.6b00438" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref86"><div class="reference"><strong class="refLabel"><a href="#ref86" class="rightTabRefNumLink">86</a></strong><div class="NLM_citation" id="rightTab-cit86"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sun, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ouyang, L.</span></span> <span> </span><span class="NLM_article-title">Dual-target kinase drug design: current strategies and future directions in cancer therapy</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>188</i></span>,  <span class="NLM_fpage">112025</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2019.112025</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref86/cit86&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1016%2Fj.ejmech.2019.112025" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref86/cit86&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=31931340" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref86/cit86&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhtFCjtLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=188&publication_year=2020&pages=112025&author=D.+Sunauthor=Y.+Zhaoauthor=S.+Zhangauthor=L.+Zhangauthor=B.+Liuauthor=L.+Ouyang&title=Dual-target+kinase+drug+design%3A+current+strategies+and+future+directions+in+cancer+therapy&doi=10.1016%2Fj.ejmech.2019.112025"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit86R"><div class="casContent"><span class="casTitleNuber">86</span><div class="casTitle"><span class="NLM_cas:atitle">Dual-target kinase drug design: Current strategies and future directions in cancer therapy</span></div><div class="casAuthors">Sun, Dejuan; Zhao, Yuqian; Zhang, Shouyue; Zhang, Lan; Liu, Bo; Ouyang, Liang</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">188</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">112025</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">A review.  Protein kinases are well-known to orchestrate the activation of signaling cascades in response to extracellular and intracellular stimuli to control cell proliferation and survival.  The perturbation of such kinases by some mutation or abnormal protein expressions has been closely linked to cancer.  Drug development aimed at several targetable kinases may alter their participated pathways that are able to trigger carcinogenesis.  A series of small-mol. drugs have been approved for the current cancer therapy.  However, their complicated inherent mechanisms may lead to the resistance to such small mols.  Consequently, development of new dual-target kinase drugs simultaneously aimed at two targetable kinases may improve their anti-tumor efficiency and solve resistant mechanism problems.  In this review, we focus on summarizing an overview of the current strategies of dual-target kinase drug design, including medicinal chem. strategies and computational approaches.  Taken together, we believe the above-mentioned strategies will provide a new insight into future directions of dual-target kinase drug design to improve potential cancer therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoL4dbUqruISrVg90H21EOLACvtfcHk0liv7fu6qGq81g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhtFCjtLg%253D&md5=7b4830183b0b7045331f6d62ded69b1d</span></div><a href="/servlet/linkout?suffix=cit86&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2019.112025&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2019.112025%26sid%3Dliteratum%253Aachs%26aulast%3DSun%26aufirst%3DD.%26aulast%3DZhao%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DS.%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DLiu%26aufirst%3DB.%26aulast%3DOuyang%26aufirst%3DL.%26atitle%3DDual-target%2520kinase%2520drug%2520design%253A%2520current%2520strategies%2520and%2520future%2520directions%2520in%2520cancer%2520therapy%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2020%26volume%3D188%26spage%3D112025%26doi%3D10.1016%2Fj.ejmech.2019.112025" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref87"><div class="reference"><strong class="refLabel"><a href="#ref87" class="rightTabRefNumLink">87</a></strong><div class="NLM_citation" id="rightTab-cit87"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mazur, P. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herner, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mello, S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wirth, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hausmann, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanchez-Rivera, F. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lofgren, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuschma, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hahn, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vangala, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trajkovic-Arsic, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gupta, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heid, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Noel, P. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Braren, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Erkan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kleeff, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sipos, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sayles, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heikenwalder, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hessmann, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ellenrieder, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Esposito, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacks, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khatri, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sweet-Cordero, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Attardi, L. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmid, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schneider, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sage, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siveke, J. T.</span></span> <span> </span><span class="NLM_article-title">Combined inhibition of BET family proteins and histone deacetylases as a potential epigenetics-based therapy for pancreatic ductal adenocarcinoma</span>. <i>Nat. Med.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">1163</span>– <span class="NLM_lpage">1171</span>, <span class="refDoi"> DOI: 10.1038/nm.3952</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref87/cit87&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1038%2Fnm.3952" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref87/cit87&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=26390243" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref87/cit87&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsFersL%252FE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2015&pages=1163-1171&author=P.+K.+Mazurauthor=A.+Hernerauthor=S.+S.+Melloauthor=M.+Wirthauthor=S.+Hausmannauthor=F.+J.+Sanchez-Riveraauthor=S.+M.+Lofgrenauthor=T.+Kuschmaauthor=S.+A.+Hahnauthor=D.+Vangalaauthor=M.+Trajkovic-Arsicauthor=A.+Guptaauthor=I.+Heidauthor=P.+B.+Noelauthor=R.+Brarenauthor=M.+Erkanauthor=J.+Kleeffauthor=B.+Siposauthor=L.+C.+Saylesauthor=M.+Heikenwalderauthor=E.+Hessmannauthor=V.+Ellenriederauthor=I.+Espositoauthor=T.+Jacksauthor=J.+E.+Bradnerauthor=P.+Khatriauthor=E.+A.+Sweet-Corderoauthor=L.+D.+Attardiauthor=R.+M.+Schmidauthor=G.+Schneiderauthor=J.+Sageauthor=J.+T.+Siveke&title=Combined+inhibition+of+BET+family+proteins+and+histone+deacetylases+as+a+potential+epigenetics-based+therapy+for+pancreatic+ductal+adenocarcinoma&doi=10.1038%2Fnm.3952"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit87R"><div class="casContent"><span class="casTitleNuber">87</span><div class="casTitle"><span class="NLM_cas:atitle">Combined inhibition of BET family proteins and histone deacetylases as a potential epigenetics-based therapy for pancreatic ductal adenocarcinoma</span></div><div class="casAuthors">Mazur, Pawel K.; Herner, Alexander; Mello, Stephano S.; Wirth, Matthias; Hausmann, Simone; Sanchez-Rivera, Francisco J.; Lofgren, Shane M.; Kuschma, Timo; Hahn, Stephan A.; Vangala, Deepak; Trajkovic-Arsic, Marija; Gupta, Aayush; Heid, Irina; Noel, Peter B.; Braren, Rickmer; Erkan, Mert; Kleeff, Jorg; Sipos, Bence; Sayles, Leanne C.; Heikenwalder, Mathias; Hessmann, Elisabeth; Ellenrieder, Volker; Esposito, Irene; Jacks, Tyler; Bradner, James E.; Khatri, Purvesh; Sweet-Cordero, E. Alejandro; Attardi, Laura D.; Schmid, Roland M.; Schneider, Guenter; Sage, Julien; Siveke, Jens T.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Medicine (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1163-1171</span>CODEN:
                <span class="NLM_cas:coden">NAMEFI</span>;
        ISSN:<span class="NLM_cas:issn">1078-8956</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Pancreatic ductal adenocarcinoma (PDAC) is one of the most lethal human cancers and shows resistance to any therapeutic strategy used.  Here the authors tested small-mol. inhibitors targeting chromatin regulators as possible therapeutic agents in PDAC.  The authors show that JQ1, an inhibitor of the bromodomain and extraterminal (BET) family of proteins, suppresses PDAC development in mice by inhibiting both MYC activity and inflammatory signals.  The histone deacetylase (HDAC) inhibitor SAHA synergizes with JQ1 to augment cell death and more potently suppress advanced PDAC.  Finally, using a CRISPR-Cas9-based method for gene editing directly in the mouse adult pancreas, the authors show that de-repression of p57 (also known as KIP2 or CDKN1C) upon combined BET and HDAC inhibition is required for the induction of combination therapy-induced cell death in PDAC.  SAHA is approved for human use, and mols. similar to JQ1 are being tested in clin. trials.  Thus, these studies identify a promising epigenetic-based therapeutic strategy that may be rapidly implemented in fatal human tumors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqmoDwfBKhuTrVg90H21EOLACvtfcHk0lhAbU4FJxDByA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsFersL%252FE&md5=d9acebe4751ac1fd33f0297e5134ee40</span></div><a href="/servlet/linkout?suffix=cit87&amp;dbid=16384&amp;doi=10.1038%2Fnm.3952&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnm.3952%26sid%3Dliteratum%253Aachs%26aulast%3DMazur%26aufirst%3DP.%2BK.%26aulast%3DHerner%26aufirst%3DA.%26aulast%3DMello%26aufirst%3DS.%2BS.%26aulast%3DWirth%26aufirst%3DM.%26aulast%3DHausmann%26aufirst%3DS.%26aulast%3DSanchez-Rivera%26aufirst%3DF.%2BJ.%26aulast%3DLofgren%26aufirst%3DS.%2BM.%26aulast%3DKuschma%26aufirst%3DT.%26aulast%3DHahn%26aufirst%3DS.%2BA.%26aulast%3DVangala%26aufirst%3DD.%26aulast%3DTrajkovic-Arsic%26aufirst%3DM.%26aulast%3DGupta%26aufirst%3DA.%26aulast%3DHeid%26aufirst%3DI.%26aulast%3DNoel%26aufirst%3DP.%2BB.%26aulast%3DBraren%26aufirst%3DR.%26aulast%3DErkan%26aufirst%3DM.%26aulast%3DKleeff%26aufirst%3DJ.%26aulast%3DSipos%26aufirst%3DB.%26aulast%3DSayles%26aufirst%3DL.%2BC.%26aulast%3DHeikenwalder%26aufirst%3DM.%26aulast%3DHessmann%26aufirst%3DE.%26aulast%3DEllenrieder%26aufirst%3DV.%26aulast%3DEsposito%26aufirst%3DI.%26aulast%3DJacks%26aufirst%3DT.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26aulast%3DKhatri%26aufirst%3DP.%26aulast%3DSweet-Cordero%26aufirst%3DE.%2BA.%26aulast%3DAttardi%26aufirst%3DL.%2BD.%26aulast%3DSchmid%26aufirst%3DR.%2BM.%26aulast%3DSchneider%26aufirst%3DG.%26aulast%3DSage%26aufirst%3DJ.%26aulast%3DSiveke%26aufirst%3DJ.%2BT.%26atitle%3DCombined%2520inhibition%2520of%2520BET%2520family%2520proteins%2520and%2520histone%2520deacetylases%2520as%2520a%2520potential%2520epigenetics-based%2520therapy%2520for%2520pancreatic%2520ductal%2520adenocarcinoma%26jtitle%3DNat.%2520Med.%26date%3D2015%26volume%3D21%26spage%3D1163%26epage%3D1171%26doi%3D10.1038%2Fnm.3952" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref88"><div class="reference"><strong class="refLabel"><a href="#ref88" class="rightTabRefNumLink">88</a></strong><div class="NLM_citation" id="rightTab-cit88"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vakoc, C. R.</span></span> <span> </span><span class="NLM_article-title">The mechanisms behind the therapeutic activity of BET bromodomain inhibition</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">728</span>– <span class="NLM_lpage">736</span>, <span class="refDoi"> DOI: 10.1016/j.molcel.2014.05.016</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref88/cit88&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1016%2Fj.molcel.2014.05.016" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref88/cit88&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=24905006" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref88/cit88&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BC2cXpslCrtrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2014&pages=728-736&author=J.+Shiauthor=C.+R.+Vakoc&title=The+mechanisms+behind+the+therapeutic+activity+of+BET+bromodomain+inhibition&doi=10.1016%2Fj.molcel.2014.05.016"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit88R"><div class="casContent"><span class="casTitleNuber">88</span><div class="casTitle"><span class="NLM_cas:atitle">The Mechanisms behind the Therapeutic Activity of BET Bromodomain Inhibition</span></div><div class="casAuthors">Shi, Junwei; Vakoc, Christopher R.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">728-736</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">A review.  The bromodomain and extraterminal (BET) protein Brd4 recruits transcriptional regulatory complexes to acetylated chromatin.  While Brd4 is considered to be a general transcriptional regulator, pharmacol. inhibition of BET proteins shows therapeutic activity in a variety of different pathologies, particularly in models of cancer and inflammation.  Such effects have been attributed to a specific set of downstream target genes whose expression is disproportionately sensitive to pharmacol. targeting of BET proteins.  Emerging evidence links the transcriptional consequences of BET inhibition to the assocn. of Brd4 with enhancer elements, which tend to be involved in lineage-specific gene regulation.  Furthermore, Brd4 engages in direct regulatory interactions with several DNA-binding transcription factors to influence their disease-relevant functions.  Here we review the current understanding of mol. mechanisms that underlie the promising therapeutic effects of BET bromodomain inhibition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpJpSFRzwcR1rVg90H21EOLACvtfcHk0ljvQjBrG31gVA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXpslCrtrk%253D&md5=f49bf3524ec378576fb1a25c64a464ae</span></div><a href="/servlet/linkout?suffix=cit88&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2014.05.016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2014.05.016%26sid%3Dliteratum%253Aachs%26aulast%3DShi%26aufirst%3DJ.%26aulast%3DVakoc%26aufirst%3DC.%2BR.%26atitle%3DThe%2520mechanisms%2520behind%2520the%2520therapeutic%2520activity%2520of%2520BET%2520bromodomain%2520inhibition%26jtitle%3DMol.%2520Cell%26date%3D2014%26volume%3D54%26spage%3D728%26epage%3D736%26doi%3D10.1016%2Fj.molcel.2014.05.016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref89"><div class="reference"><strong class="refLabel"><a href="#ref89" class="rightTabRefNumLink">89</a></strong><div class="NLM_citation" id="rightTab-cit89"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Donati, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lorenzini, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciarrocchi, A.</span></span> <span> </span><span class="NLM_article-title">Ciarrocchi A. BRD4 and cancer: going beyond transcriptional regulation</span>. <i>Mol. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">164</span>, <span class="refDoi"> DOI: 10.1186/s12943-018-0915-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref89/cit89&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1186%2Fs12943-018-0915-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref89/cit89&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=30466442" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref89/cit89&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BC1MXptlWgu78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2018&pages=164&author=B.+Donatiauthor=E.+Lorenziniauthor=A.+Ciarrocchi&title=Ciarrocchi+A.+BRD4+and+cancer%3A+going+beyond+transcriptional+regulation&doi=10.1186%2Fs12943-018-0915-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit89R"><div class="casContent"><span class="casTitleNuber">89</span><div class="casTitle"><span class="NLM_cas:atitle">BRD4 and Cancer: going beyond transcriptional regulation</span></div><div class="casAuthors">Donati, Benedetta; Lorenzini, Eugenia; Ciarrocchi, Alessia</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cancer</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">164</span>CODEN:
                <span class="NLM_cas:coden">MCOACG</span>;
        ISSN:<span class="NLM_cas:issn">1476-4598</span>.
    
            (<span class="NLM_cas:orgname">BioMed Central Ltd.</span>)
        </div><div class="casAbstract">A review.  BRD4, member of the Bromodomain and Extraterminal (BET) protein family, is largely acknowledged in cancer for its role in super-enhancers (SEs) organization and oncogenes expression regulation.  Inhibition of BRD4 shortcuts the communication between SEs and target promoters with a subsequent cell-specific repression of oncogenes to which cancer cells are addicted and cell death.  To date, this is the most credited mechanism of action of BET inhibitors, a class of small mols. targeting BET proteins which are currently in clin. trials in several cancer settings.  However, recent evidence indicates that BRD4 relevance in cancer goes beyond its role in transcription regulation and identifies this protein as a keeper of genome stability.  Indeed, a non-transcriptional role of BRD4 in controlling DNA damage checkpoint activation and repair as well as telomere maintenance has been proposed, throwing new lights into the multiple functions of this protein and opening new perspectives on the use of BETi in cancer.  Here we discuss the current available information on non-canonical, non-transcriptional functions of BRD4 and on their implications in cancer biol.  Integrating this information with the already known BRD4 role in gene expression regulation, we propose a "common" model to explain BRD4 genomic function.  Furthermore, in light of the transversal function of BRD4, we provide new interpretation for the cytotoxic activity of BETi and we discuss new possibilities for a wide and focused employment of these drugs in clin. settings.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpbYn0mnQW7ELVg90H21EOLACvtfcHk0ljvQjBrG31gVA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXptlWgu78%253D&md5=2ddfde1363d430aa67e15a52fdc2335e</span></div><a href="/servlet/linkout?suffix=cit89&amp;dbid=16384&amp;doi=10.1186%2Fs12943-018-0915-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs12943-018-0915-9%26sid%3Dliteratum%253Aachs%26aulast%3DDonati%26aufirst%3DB.%26aulast%3DLorenzini%26aufirst%3DE.%26aulast%3DCiarrocchi%26aufirst%3DA.%26atitle%3DCiarrocchi%2520A.%2520BRD4%2520and%2520cancer%253A%2520going%2520beyond%2520transcriptional%2520regulation%26jtitle%3DMol.%2520Cancer%26date%3D2018%26volume%3D17%26spage%3D164%26doi%3D10.1186%2Fs12943-018-0915-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref90"><div class="reference"><strong class="refLabel"><a href="#ref90" class="rightTabRefNumLink">90</a></strong><div class="NLM_citation" id="rightTab-cit90"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pan, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, G.</span></span> <span> </span><span class="NLM_article-title">Discovery of thieno[2,3-d] pyrimidine-based hydroxamic acid derivatives as bromodomain-containing protein 4/histone deacetylase dual inhibitors induce autophagic cell death in colorectal carcinoma cells</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">3678</span>– <span class="NLM_lpage">3700</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b02178</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b02178" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref90/cit90&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BB3cXksFajtLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=3678-3700&author=Z.+Panauthor=X.+Liauthor=Y.+Wangauthor=Q.+Jiangauthor=L.+Jiangauthor=M.+Zhangauthor=N.+Zhangauthor=F.+Wuauthor=B.+Liuauthor=G.+He&title=Discovery+of+thieno%5B2%2C3-d%5D+pyrimidine-based+hydroxamic+acid+derivatives+as+bromodomain-containing+protein+4%2Fhistone+deacetylase+dual+inhibitors+induce+autophagic+cell+death+in+colorectal+carcinoma+cells&doi=10.1021%2Facs.jmedchem.9b02178"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit90R"><div class="casContent"><span class="casTitleNuber">90</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Thieno[2,3-d]pyrimidine-Based Hydroxamic Acid Derivatives as Bromodomain-Containing Protein 4/Histone Deacetylase Dual Inhibitors Induce Autophagic Cell Death in Colorectal Carcinoma Cells</span></div><div class="casAuthors">Pan, Zhaoping; Li, Xiang; Wang, Yujia; Jiang, Qinglin; Jiang, Li; Zhang, Min; Zhang, Nan; Wu, Fengbo; Liu, Bo; He, Gu</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">3678-3700</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Bromodomain-contg. protein 4 (BRD4) and histone deacetylases (HDAC) are both attractive epigenetic targets in cancer and other chronic diseases.  Based on the integrated fragment-based drug design, synthesis, and in vitro and in vivo evaluations, a series of novel thieno[2,3-d]pyrimidine-based hydroxamic acid derivs. are discovered as selective BRD4-HDAC dual inhibitors.  Compd. 17c is the most potent inhibitor for BRD4 and HDAC with IC50 values at nanomolar levels, as well as the expression level of c-Myc, and increases the acetylation of histone H3.  Moreover, 17c presents inhibitory effects on the proliferation of colorectal carcinoma (CRC) cells via inducing autophagic cell death.  It also has a good pharmacokinetic profile in rats and oral bioavailability of 40.5%.  In the HCT-116 xenograft in vivo models, 17c displays potent inhibitory efficiency on tumor growth by inducing autophagic cell death and suppressing IL6-JAK-STAT signaling pathways.  Our results suggest that the BRD4-HDAC dual inhibition might be an attractive therapeutic strategy for CRC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr2FHKdnuD3QLVg90H21EOLACvtfcHk0lgyJKcUVCL6GQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXksFajtLs%253D&md5=ce6446afe48f80165d2b21aee04571f6</span></div><a href="/servlet/linkout?suffix=cit90&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b02178&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b02178%26sid%3Dliteratum%253Aachs%26aulast%3DPan%26aufirst%3DZ.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DJiang%26aufirst%3DQ.%26aulast%3DJiang%26aufirst%3DL.%26aulast%3DZhang%26aufirst%3DM.%26aulast%3DZhang%26aufirst%3DN.%26aulast%3DWu%26aufirst%3DF.%26aulast%3DLiu%26aufirst%3DB.%26aulast%3DHe%26aufirst%3DG.%26atitle%3DDiscovery%2520of%2520thieno%255B2%252C3-d%255D%2520pyrimidine-based%2520hydroxamic%2520acid%2520derivatives%2520as%2520bromodomain-containing%2520protein%25204%252Fhistone%2520deacetylase%2520dual%2520inhibitors%2520induce%2520autophagic%2520cell%2520death%2520in%2520colorectal%2520carcinoma%2520cells%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D3678%26epage%3D3700%26doi%3D10.1021%2Facs.jmedchem.9b02178" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref91"><div class="reference"><strong class="refLabel"><a href="#ref91" class="rightTabRefNumLink">91</a></strong><div class="NLM_citation" id="rightTab-cit91"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lan, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geng, M.</span></span> <span> </span><span class="NLM_article-title">Feedback activation of leukemia inhibitory factor receptor limits response to histone deacetylase inhibitors in breast cancer</span>. <i>Cancer Cell</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">459</span>– <span class="NLM_lpage">473</span>, <span class="refDoi"> DOI: 10.1016/j.ccell.2016.08.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref91/cit91&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1016%2Fj.ccell.2016.08.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref91/cit91&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=27622335" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref91/cit91&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsFWitrnI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2016&pages=459-473&author=H.+Zengauthor=J.+Quauthor=N.+Jinauthor=J.+Xuauthor=C.+Linauthor=Y.+Chenauthor=X.+Yangauthor=X.+Heauthor=S.+Tangauthor=X.+Lanauthor=X.+Yangauthor=Z.+Chenauthor=M.+Huangauthor=J.+Dingauthor=M.+Geng&title=Feedback+activation+of+leukemia+inhibitory+factor+receptor+limits+response+to+histone+deacetylase+inhibitors+in+breast+cancer&doi=10.1016%2Fj.ccell.2016.08.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit91R"><div class="casContent"><span class="casTitleNuber">91</span><div class="casTitle"><span class="NLM_cas:atitle">Feedback Activation of Leukemia Inhibitory Factor Receptor Limits Response to Histone Deacetylase Inhibitors in Breast Cancer</span></div><div class="casAuthors">Zeng, Hanlin; Qu, Jia; Jin, Nan; Xu, Jun; Lin, Chenchu; Chen, Yi; Yang, Xinying; He, Xiang; Tang, Shuai; Lan, Xiaojing; Yang, Xiaotong; Chen, Ziqi; Huang, Min; Ding, Jian; Geng, Meiyu</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">459-473</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Histone deacetylase (HDAC) inhibitors have demonstrated clin. benefits in subtypes of hematol. malignancies.  However, the efficacy of HDAC inhibitors in solid tumors remains uncertain.  This study takes breast cancer as a model to understand mechanisms accounting for limited response of HDAC inhibitors in solid tumors and to seek combination solns.  We discover that feedback activation of leukemia inhibitory factor receptor (LIFR) signaling in breast cancer limits the response to HDAC inhibition.  Mechanistically, HDAC inhibition increases histone acetylation at the LIFR gene promoter, which recruits bromodomain protein BRD4, upregulates LIFR expression, and activates JAK1-STAT3 signaling.  Importantly, JAK1 or BRD4 inhibition sensitizes breast cancer to HDAC inhibitors, implicating combination inhibition of HDAC with JAK1 or BRD4 as potential therapies for breast cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrL24sYbhFYeLVg90H21EOLACvtfcHk0lgyJKcUVCL6GQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsFWitrnI&md5=9ce620c7a19519a95edd714f8012518c</span></div><a href="/servlet/linkout?suffix=cit91&amp;dbid=16384&amp;doi=10.1016%2Fj.ccell.2016.08.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccell.2016.08.001%26sid%3Dliteratum%253Aachs%26aulast%3DZeng%26aufirst%3DH.%26aulast%3DQu%26aufirst%3DJ.%26aulast%3DJin%26aufirst%3DN.%26aulast%3DXu%26aufirst%3DJ.%26aulast%3DLin%26aufirst%3DC.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DYang%26aufirst%3DX.%26aulast%3DHe%26aufirst%3DX.%26aulast%3DTang%26aufirst%3DS.%26aulast%3DLan%26aufirst%3DX.%26aulast%3DYang%26aufirst%3DX.%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DHuang%26aufirst%3DM.%26aulast%3DDing%26aufirst%3DJ.%26aulast%3DGeng%26aufirst%3DM.%26atitle%3DFeedback%2520activation%2520of%2520leukemia%2520inhibitory%2520factor%2520receptor%2520limits%2520response%2520to%2520histone%2520deacetylase%2520inhibitors%2520in%2520breast%2520cancer%26jtitle%3DCancer%2520Cell%26date%3D2016%26volume%3D30%26spage%3D459%26epage%3D473%26doi%3D10.1016%2Fj.ccell.2016.08.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref92"><div class="reference"><strong class="refLabel"><a href="#ref92" class="rightTabRefNumLink">92</a></strong><div class="NLM_citation" id="rightTab-cit92"><span><span class="NLM_contrib-group"><span class="NLM_string-name">He, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheng, C.</span></span> <span> </span><span class="NLM_article-title">Potent Dual BET/HDAC Inhibitors for Efficient Treatment of Pancreatic Cancer</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">3028</span>– <span class="NLM_lpage">3032</span>, <span class="refDoi"> DOI: 10.1002/anie.201915896</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref92/cit92&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1002%2Fanie.201915896" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref92/cit92&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BB3cXitFegtLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2020&pages=3028-3032&author=S.+Heauthor=G.+Dongauthor=Y.+Liauthor=S.+Wuauthor=W.+Wangauthor=C.+Sheng&title=Potent+Dual+BET%2FHDAC+Inhibitors+for+Efficient+Treatment+of+Pancreatic+Cancer&doi=10.1002%2Fanie.201915896"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit92R"><div class="casContent"><span class="casTitleNuber">92</span><div class="casTitle"><span class="NLM_cas:atitle">Potent Dual BET/HDAC Inhibitors for Efficient Treatment of Pancreatic Cancer</span></div><div class="casAuthors">He, Shipeng; Dong, Guoqiang; Li, Yu; Wu, Shanchao; Wang, Wei; Sheng, Chunquan</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">3028-3032</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">As one of the most aggressive and lethal human malignancies with extremely poor prognosis, there is an urgent demand of more effective therapy for the treatment of pancreatic cancer.  Reported here is a new, effective therapeutic strategy and the design of small-mol. inhibitors that simultaneously target bromodomain and extra-terminal (BET) and histone deacetylase (HDAC), potentially serving as promising therapeutic agents for pancreatic cancer.  A highly potent dual inhibitor (13 a) is identified to possess excellent and balanced activities against BRD4 BD1 (IC50=11 nM) and HDAC1 (IC50=21 nM).  Notably, this compd. shows higher in vitro and in vivo antitumor potency than the BET inhibitor (+)-JQ1 and the HDAC inhibitor vorinostat, either alone or and in combination, highlighting the advantages of BET/HDAC dual inhibitors for more effective treatment of pancreatic cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp9BFNcLfkA4bVg90H21EOLACvtfcHk0lhCLfFbKJ7YKQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXitFegtLw%253D&md5=78db62cb533611939d9abd0e654ff5f3</span></div><a href="/servlet/linkout?suffix=cit92&amp;dbid=16384&amp;doi=10.1002%2Fanie.201915896&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201915896%26sid%3Dliteratum%253Aachs%26aulast%3DHe%26aufirst%3DS.%26aulast%3DDong%26aufirst%3DG.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DWu%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DW.%26aulast%3DSheng%26aufirst%3DC.%26atitle%3DPotent%2520Dual%2520BET%252FHDAC%2520Inhibitors%2520for%2520Efficient%2520Treatment%2520of%2520Pancreatic%2520Cancer%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2020%26volume%3D59%26spage%3D3028%26epage%3D3032%26doi%3D10.1002%2Fanie.201915896" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref93"><div class="reference"><strong class="refLabel"><a href="#ref93" class="rightTabRefNumLink">93</a></strong><div class="NLM_citation" id="rightTab-cit93"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, D.</span></span> <span> </span><span class="NLM_article-title">Design, synthesis and biological evaluation of novel indole derivatives as potential HDAC/BRD4 dual inhibitors and anti-leukemia agents</span>. <i>Bioorg. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>84</i></span>,  <span class="NLM_fpage">410</span>– <span class="NLM_lpage">417</span>, <span class="refDoi"> DOI: 10.1016/j.bioorg.2018.12.011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref93/cit93&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1016%2Fj.bioorg.2018.12.011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref93/cit93&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=30554080" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref93/cit93&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BC1cXisFaqtb%252FI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=84&publication_year=2019&pages=410-417&author=G.+Chengauthor=Z.+Wangauthor=J.+Yangauthor=Y.+Baoauthor=Q.+Xuauthor=L.+Zhaoauthor=D.+Liu&title=Design%2C+synthesis+and+biological+evaluation+of+novel+indole+derivatives+as+potential+HDAC%2FBRD4+dual+inhibitors+and+anti-leukemia+agents&doi=10.1016%2Fj.bioorg.2018.12.011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit93R"><div class="casContent"><span class="casTitleNuber">93</span><div class="casTitle"><span class="NLM_cas:atitle">Design, synthesis and biological evaluation of novel indole derivatives as potential HDAC/BRD4 dual inhibitors and anti-leukemia agents</span></div><div class="casAuthors">Cheng, Gaoliang; Wang, Zhi; Yang, Jinyu; Bao, Yu; Xu, Qihao; Zhao, Linxiang; Liu, Dan</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">84</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">410-417</span>CODEN:
                <span class="NLM_cas:coden">BOCMBM</span>;
        ISSN:<span class="NLM_cas:issn">0045-2068</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">HDAC inhibitors and BRD4 inhibitors were considered to be potent anti-cancer agents.  Recent studies have demonstrated that HDAC and BRD4 participate in the regulation of some signal paths like PI3K-AKT.  In this work, a series of indole derivs. that combine the inhibitory activities of BRD4 and HDAC into one mol. were designed and synthesized through the structure-based design method.  Most compds. showed potent HDAC inhibitory activity and moderate BRD4 inhibitory activity.  In vitro anti-proliferation activities of the synthesized compds. were also evaluated.  Among them, 19f was the most potent inhibitor against HDAC3 with IC50 value of 5 nM and BRD4 inhibition rate of 88% at 10 μM.  It was confirmed that 19f could up-regulate the expression of Ac-H3 and reduce the expression of c-Myc by western blot anal.  These results indicated that 19f was a potent dual HDAC/BRD4 inhibitor and deserved further investigation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqkPM3yaQ5gE7Vg90H21EOLACvtfcHk0lhCLfFbKJ7YKQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXisFaqtb%252FI&md5=e2c81b5d67fd9f49e6805c6b544438b4</span></div><a href="/servlet/linkout?suffix=cit93&amp;dbid=16384&amp;doi=10.1016%2Fj.bioorg.2018.12.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bioorg.2018.12.011%26sid%3Dliteratum%253Aachs%26aulast%3DCheng%26aufirst%3DG.%26aulast%3DWang%26aufirst%3DZ.%26aulast%3DYang%26aufirst%3DJ.%26aulast%3DBao%26aufirst%3DY.%26aulast%3DXu%26aufirst%3DQ.%26aulast%3DZhao%26aufirst%3DL.%26aulast%3DLiu%26aufirst%3DD.%26atitle%3DDesign%252C%2520synthesis%2520and%2520biological%2520evaluation%2520of%2520novel%2520indole%2520derivatives%2520as%2520potential%2520HDAC%252FBRD4%2520dual%2520inhibitors%2520and%2520anti-leukemia%2520agents%26jtitle%3DBioorg.%2520Chem.%26date%3D2019%26volume%3D84%26spage%3D410%26epage%3D417%26doi%3D10.1016%2Fj.bioorg.2018.12.011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref94"><div class="reference"><strong class="refLabel"><a href="#ref94" class="rightTabRefNumLink">94</a></strong><div class="NLM_citation" id="rightTab-cit94"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Amemiya, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamaguchi, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hashimoto, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Noguchi-Yachide, T.</span></span> <span> </span><span class="NLM_article-title">Synthesis and evaluation of novel dual BRD4/HDAC inhibitors</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">3677</span>– <span class="NLM_lpage">3684</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2017.04.043</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref94/cit94&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1016%2Fj.bmc.2017.04.043" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref94/cit94&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=28549889" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref94/cit94&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BC2sXotlyitL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2017&pages=3677-3684&author=S.+Amemiyaauthor=T.+Yamaguchiauthor=Y.+Hashimotoauthor=T.+Noguchi-Yachide&title=Synthesis+and+evaluation+of+novel+dual+BRD4%2FHDAC+inhibitors&doi=10.1016%2Fj.bmc.2017.04.043"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit94R"><div class="casContent"><span class="casTitleNuber">94</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis and evaluation of novel dual BRD4/HDAC inhibitors</span></div><div class="casAuthors">Amemiya, Seika; Yamaguchi, Takao; Hashimoto, Yuichi; Noguchi-Yachide, Tomomi</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">3677-3684</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Epigenetic regulation of gene expression via histone acetylation modulates many cellular processes, including apoptosis, the cell cycle, cell growth and differentiation, and inhibitors are promising drug candidates.  We have previously developed inhibitors of BRD4, which recognizes acetylated lysine residue on histones and recruits transcription elongation factor to the transcription start site, while inhibitors of histone deacetylase (HDAC), which catalyzes the removal of acetyl groups on histones, are already in clin. use for cancer treatment.  Based on the idea that polypharmacol. agents with multiple targets would have a more robust action, we set out to develop dual BRD4/HDAC inhibitors.  Here, we describe the design and synthesis of N6-[2-(7-hydroxyamino-7-oxoheptyloxy)benzoyl]adenine (I) as a BRD4/HDAC dual inhibitor.  This compd. showed HL-60 cell growth-inhibitory and apoptosis-inducing activity, as well as all-trans retinoic acid (ATRA)-induced HL-60 cell differentiation-enhancing activity, and c-MYC prodn.-inhibitory activity.  Interestingly, it also showed growth-inhibitory activity towards BRD4 inhibitor-resistant cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqw5w5uPYmY37Vg90H21EOLACvtfcHk0linxSGVNYyA7w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXotlyitL4%253D&md5=c9099dc3d40c966d150a13fab28c2b9f</span></div><a href="/servlet/linkout?suffix=cit94&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2017.04.043&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2017.04.043%26sid%3Dliteratum%253Aachs%26aulast%3DAmemiya%26aufirst%3DS.%26aulast%3DYamaguchi%26aufirst%3DT.%26aulast%3DHashimoto%26aufirst%3DY.%26aulast%3DNoguchi-Yachide%26aufirst%3DT.%26atitle%3DSynthesis%2520and%2520evaluation%2520of%2520novel%2520dual%2520BRD4%252FHDAC%2520inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2017%26volume%3D25%26spage%3D3677%26epage%3D3684%26doi%3D10.1016%2Fj.bmc.2017.04.043" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref95"><div class="reference"><strong class="refLabel"><a href="#ref95" class="rightTabRefNumLink">95</a></strong><div class="NLM_citation" id="rightTab-cit95"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hou, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ran, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bian, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span> <span> </span><span class="NLM_article-title">Targeting epigenetic reader and eraser: Rational design, synthesis and in vitro evaluation of dimethylisoxazoles derivatives as BRD4/HDAC dual inhibitors</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">2931</span>– <span class="NLM_lpage">2935</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2016.04.034</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref95/cit95&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1016%2Fj.bmcl.2016.04.034" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref95/cit95&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=27142751" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref95/cit95&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BC28XntFCmt7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2016&pages=2931-2935&author=Z.+Zhangauthor=S.+Houauthor=H.+Chenauthor=T.+Ranauthor=F.+Jiangauthor=Y.+Bianauthor=D.+Zhangauthor=Y.+Zhiauthor=L.+Wangauthor=L.+Zhangauthor=H.+Liauthor=Y.+Zhangauthor=W.+Tangauthor=T.+Luauthor=Y.+Chen&title=Targeting+epigenetic+reader+and+eraser%3A+Rational+design%2C+synthesis+and+in+vitro+evaluation+of+dimethylisoxazoles+derivatives+as+BRD4%2FHDAC+dual+inhibitors&doi=10.1016%2Fj.bmcl.2016.04.034"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit95R"><div class="casContent"><span class="casTitleNuber">95</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting epigenetic reader and eraser: Rational design, synthesis and in vitro evaluation of dimethylisoxazoles derivatives as BRD4/HDAC dual inhibitors</span></div><div class="casAuthors">Zhang, Zhimin; Hou, Shaohua; Chen, Hongli; Ran, Ting; Jiang, Fei; Bian, Yuanyuan; Zhang, Dewei; Zhi, Yanle; Wang, Lu; Zhang, Li; Li, Hongmei; Zhang, Yanmin; Tang, Weifang; Lu, Tao; Chen, Yadong</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2931-2935</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The bromodomain protein module and histone deacetylase (HDAC), which recognize and remove acetylated lysine, resp., have emerged as important epigenetic therapeutic targets in cancer treatments.  Herein the authors presented a novel design approach for cancer drug development by combination of bromodomain and HDAC inhibitory activity in one mol.  The designed compds. were synthesized which showed inhibitory activity against bromodomain 4 and HDAC1.  The representative dual bromodomain/HDAC inhibitors, compds. N1-(3-benzoyl-5-(3,5-dimethylisoxazol-4-yl)phenyl)-N7-hydroxyheptanediamide and N1-(3-benzoyl-5-(3,5-dimethylisoxazol-4-yl)phenyl)-N8-hydroxyoctanediamide showed potent antiproliferative activities against human leukemia cell line K562 and MV4-11 in cellular assays.  This work may lay the foundation for developing dual bromodomain/HDAC inhibitors as potential anticancer therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGriS0VB1G-QoLVg90H21EOLACvtfcHk0lifZ4uzDahQGA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XntFCmt7o%253D&md5=a6ebdbd6402e3117a2f9d96ed071790c</span></div><a href="/servlet/linkout?suffix=cit95&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2016.04.034&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2016.04.034%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DZ.%26aulast%3DHou%26aufirst%3DS.%26aulast%3DChen%26aufirst%3DH.%26aulast%3DRan%26aufirst%3DT.%26aulast%3DJiang%26aufirst%3DF.%26aulast%3DBian%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DD.%26aulast%3DZhi%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DLi%26aufirst%3DH.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DTang%26aufirst%3DW.%26aulast%3DLu%26aufirst%3DT.%26aulast%3DChen%26aufirst%3DY.%26atitle%3DTargeting%2520epigenetic%2520reader%2520and%2520eraser%253A%2520Rational%2520design%252C%2520synthesis%2520and%2520in%2520vitro%2520evaluation%2520of%2520dimethylisoxazoles%2520derivatives%2520as%2520BRD4%252FHDAC%2520dual%2520inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2016%26volume%3D26%26spage%3D2931%26epage%3D2935%26doi%3D10.1016%2Fj.bmcl.2016.04.034" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref96"><div class="reference"><strong class="refLabel"><a href="#ref96" class="rightTabRefNumLink">96</a></strong><div class="NLM_citation" id="rightTab-cit96"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shao, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L.</span></span> <span> </span><span class="NLM_article-title">Structure-based design, synthesis and in vitro antiproliferative effects studies of novel dual BRD4/HDAC inhibitors</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">4051</span>– <span class="NLM_lpage">4055</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2017.07.054</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref96/cit96&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1016%2Fj.bmcl.2017.07.054" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref96/cit96&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=28765013" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref96/cit96&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BC2sXht1KqsLrP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2017&pages=4051-4055&author=M.+Shaoauthor=L.+Heauthor=L.+Zhengauthor=L.+Huangauthor=Y.+Zhouauthor=T.+Wangauthor=Y.+Chenauthor=M.+Shenauthor=F.+Wangauthor=Z.+Yangauthor=L.+Chen&title=Structure-based+design%2C+synthesis+and+in+vitro+antiproliferative+effects+studies+of+novel+dual+BRD4%2FHDAC+inhibitors&doi=10.1016%2Fj.bmcl.2017.07.054"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit96R"><div class="casContent"><span class="casTitleNuber">96</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-based design, synthesis and in vitro antiproliferative effects studies of novel dual BRD4/HDAC inhibitors</span></div><div class="casAuthors">Shao, Mingfeng; He, Linhong; Zheng, Li; Huang, Lingxiao; Zhou, Yuanyuan; Wang, Taijing; Chen, Yong; Shen, Mingsheng; Wang, Fang; Yang, Zhuang; Chen, Lijuan</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">4051-4055</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Histone acetylation marks play important roles in controlling gene expressions and are removed by histone deacetylases (HDACs).  These marks are read by bromodomain and extra-terminal (BET) proteins, whose targeted inhibitors are under clin. investigation.  BET and HDAC inhibitors have been demonstrated to be synergistically killing in Myc-induced murine lymphoma.  Herein, the authors combine the inhibitory activities of BET and HDAC into one mol. through structure-based design method and evaluate its function.  The majority of these synthesized compds. showed inhibitory activity against second bromodomains(BRD) of BRD4 and HDAC1.  Among them, I presented antiproliferative effects against human acute myelogenous leukemia (AML) cell lines in vitro, and I is confirmed to reduce the expression of Myc by Western blot anal.  Those results indicated that I is a potent dual BRD4/HDAC inhibitor and deserves further investigation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpkMaHqVu_LM7Vg90H21EOLACvtfcHk0lifZ4uzDahQGA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXht1KqsLrP&md5=c525f1a2083699dd5f6f6a262f477f24</span></div><a href="/servlet/linkout?suffix=cit96&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2017.07.054&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2017.07.054%26sid%3Dliteratum%253Aachs%26aulast%3DShao%26aufirst%3DM.%26aulast%3DHe%26aufirst%3DL.%26aulast%3DZheng%26aufirst%3DL.%26aulast%3DHuang%26aufirst%3DL.%26aulast%3DZhou%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DT.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DShen%26aufirst%3DM.%26aulast%3DWang%26aufirst%3DF.%26aulast%3DYang%26aufirst%3DZ.%26aulast%3DChen%26aufirst%3DL.%26atitle%3DStructure-based%2520design%252C%2520synthesis%2520and%2520in%2520vitro%2520antiproliferative%2520effects%2520studies%2520of%2520novel%2520dual%2520BRD4%252FHDAC%2520inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2017%26volume%3D27%26spage%3D4051%26epage%3D4055%26doi%3D10.1016%2Fj.bmcl.2017.07.054" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref97"><div class="reference"><strong class="refLabel"><a href="#ref97" class="rightTabRefNumLink">97</a></strong><div class="NLM_citation" id="rightTab-cit97"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shan, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fan, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanyi, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fan, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Montone, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dang, C. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span> <span> </span><span class="NLM_article-title">Repression of BET activity sensitizes homologous recombination-proficient cancers to PARP inhibition</span>. <i>Sci. Transl. Med.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">eaal1645</span>, <span class="refDoi"> DOI: 10.1126/scitranslmed.aal1645</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref97/cit97&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1126%2Fscitranslmed.aal1645" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref97/cit97&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=28747513" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2017&pages=eaal1645&author=L.+Yangauthor=Y.+Zhangauthor=W.+Shanauthor=Z.+Huauthor=J.+Yuanauthor=J.+Piauthor=Y.+Wangauthor=L.+Fanauthor=Z.+Tangauthor=C.+Liauthor=X.+Huauthor=J.+L.+Tanyiauthor=Y.+Fanauthor=Q.+Huangauthor=K.+Montoneauthor=C.+V.+Dangauthor=L.+Zhang&title=Repression+of+BET+activity+sensitizes+homologous+recombination-proficient+cancers+to+PARP+inhibition&doi=10.1126%2Fscitranslmed.aal1645"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit97&amp;dbid=16384&amp;doi=10.1126%2Fscitranslmed.aal1645&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscitranslmed.aal1645%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DL.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DShan%26aufirst%3DW.%26aulast%3DHu%26aufirst%3DZ.%26aulast%3DYuan%26aufirst%3DJ.%26aulast%3DPi%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DFan%26aufirst%3DL.%26aulast%3DTang%26aufirst%3DZ.%26aulast%3DLi%26aufirst%3DC.%26aulast%3DHu%26aufirst%3DX.%26aulast%3DTanyi%26aufirst%3DJ.%2BL.%26aulast%3DFan%26aufirst%3DY.%26aulast%3DHuang%26aufirst%3DQ.%26aulast%3DMontone%26aufirst%3DK.%26aulast%3DDang%26aufirst%3DC.%2BV.%26aulast%3DZhang%26aufirst%3DL.%26atitle%3DRepression%2520of%2520BET%2520activity%2520sensitizes%2520homologous%2520recombination-proficient%2520cancers%2520to%2520PARP%2520inhibition%26jtitle%3DSci.%2520Transl.%2520Med.%26date%3D2017%26volume%3D9%26spage%3Deaal1645%26doi%3D10.1126%2Fscitranslmed.aal1645" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref98"><div class="reference"><strong class="refLabel"><a href="#ref98" class="rightTabRefNumLink">98</a></strong><div class="NLM_citation" id="rightTab-cit98"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ouyang, L.</span></span> <span> </span><span class="NLM_article-title">Design, synthesis, and biological evaluation of quinazolin-4(3H)-one derivatives co-targeting poly(ADP-ribose) polymerase-1 and bromodomain containing protein 4 for breast cancer therapy</span>. <i>Acta Pharm. Sin. B</i> <span class="NLM_year" style="font-weight: bold;">2021</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">156</span>– <span class="NLM_lpage">180</span>, <span class="refDoi"> DOI: 10.1016/j.apsb.2020.06.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref98/cit98&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1016%2Fj.apsb.2020.06.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref98/cit98&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=33532187" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref98/cit98&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A280%3ADC%252BB3srovVGlsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2021&pages=156-180&author=X.+Changauthor=D.+Sunauthor=F.+Shiauthor=G.+Wangauthor=Y.+Chenauthor=K.+Zhangauthor=H.+Tanauthor=J.+Liuauthor=B.+Liuauthor=L.+Ouyang&title=Design%2C+synthesis%2C+and+biological+evaluation+of+quinazolin-4%283H%29-one+derivatives+co-targeting+poly%28ADP-ribose%29+polymerase-1+and+bromodomain+containing+protein+4+for+breast+cancer+therapy&doi=10.1016%2Fj.apsb.2020.06.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit98R"><div class="casContent"><span class="casTitleNuber">98</span><div class="casTitle"><span class="NLM_cas:atitle">Design, synthesis, and biological evaluation of quinazolin-4(3H)-one derivatives co-targeting poly(ADP-ribose) polymerase-1 and bromodomain containing protein 4 for breast cancer therapy</span></div><div class="casAuthors">Chang Xiaosa; Sun Dejuan; Shi Danfeng; Wang Guan; Chen Yanmei; Zhang Kai; Tan Huidan; Liu Jie; Liu Bo; Ouyang Liang</div><div class="citationInfo"><span class="NLM_cas:title">Acta pharmaceutica Sinica. B</span>
        (<span class="NLM_cas:date">2021</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">156-180</span>
        ISSN:<span class="NLM_cas:issn">2211-3835</span>.
    </div><div class="casAbstract">This study was aimed to design the first dual-target small-molecule inhibitor co-targeting poly (ADP-ribose) polymerase-1 (PARP1) and bromodomain containing protein 4 (BRD4), which had important cross relation in the global network of breast cancer, reflecting the synthetic lethal effect.  A series of new BRD4 and PARP1 dual-target inhibitors were discovered and synthesized by fragment-based combinatorial screening and activity assays that together led to the chemical optimization.  Among these compounds, 19d was selected and exhibited micromole enzymatic potencies against BRD4 and PARP1, respectively.  Compound 19d was further shown to efficiently modulate the expression of BRD4 and PARP1.  Subsequently, compound 19d was found to induce breast cancer cell apoptosis and stimulate cell cycle arrest at G1 phase.  Following pharmacokinetic studies, compound 19d showed its antitumor activity in breast cancer susceptibility gene 1/2 (BRCA1/2) wild-type MDA-MB-468 and MCF-7 xenograft models without apparent toxicity and loss of body weight.  These results together demonstrated that a highly potent dual-targeted inhibitor was successfully synthesized and indicated that co-targeting of BRD4 and PARP1 based on the concept of synthetic lethality would be a promising therapeutic strategy for breast cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTHS4WGcbBgy2HOrQ6qeX-mfW6udTcc2eYqHKOEQzsY0Lntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3srovVGlsQ%253D%253D&md5=7108868f2fa08b8bcd5131ec8dd1d281</span></div><a href="/servlet/linkout?suffix=cit98&amp;dbid=16384&amp;doi=10.1016%2Fj.apsb.2020.06.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.apsb.2020.06.003%26sid%3Dliteratum%253Aachs%26aulast%3DChang%26aufirst%3DX.%26aulast%3DSun%26aufirst%3DD.%26aulast%3DShi%26aufirst%3DF.%26aulast%3DWang%26aufirst%3DG.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DK.%26aulast%3DTan%26aufirst%3DH.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DLiu%26aufirst%3DB.%26aulast%3DOuyang%26aufirst%3DL.%26atitle%3DDesign%252C%2520synthesis%252C%2520and%2520biological%2520evaluation%2520of%2520quinazolin-4%25283H%2529-one%2520derivatives%2520co-targeting%2520poly%2528ADP-ribose%2529%2520polymerase-1%2520and%2520bromodomain%2520containing%2520protein%25204%2520for%2520breast%2520cancer%2520therapy%26jtitle%3DActa%2520Pharm.%2520Sin.%2520B%26date%3D2021%26volume%3D11%26spage%3D156%26epage%3D180%26doi%3D10.1016%2Fj.apsb.2020.06.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref99"><div class="reference"><strong class="refLabel"><a href="#ref99" class="rightTabRefNumLink">99</a></strong><div class="NLM_citation" id="rightTab-cit99"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ren, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bi, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lwin, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Silva, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shah, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koomen, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chavez, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pham, L. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sudalagunta, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wan, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dalton, W. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moscinski, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shain, K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vose, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cleveland, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sotomayor, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tao, J.</span></span> <span> </span><span class="NLM_article-title">PLK1 stabilizes a MYC-dependent kinase network in aggressive B cell lymphomas</span>. <i>J. Clin. Invest.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>128</i></span>,  <span class="NLM_fpage">5517</span>– <span class="NLM_lpage">5530</span>, <span class="refDoi"> DOI: 10.1172/JCI122533</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref99/cit99&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1172%2FJCI122533" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref99/cit99&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=30260324" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref99/cit99&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A280%3ADC%252BB3czjt1yntw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=128&publication_year=2018&pages=5517-5530&author=Y.+Renauthor=C.+Biauthor=X.+Zhaoauthor=T.+Lwinauthor=C.+Wangauthor=J.+Yuanauthor=A.+S.+Silvaauthor=B.+D.+Shahauthor=B.+Fangauthor=T.+Liauthor=J.+M.+Koomenauthor=H.+Jiangauthor=J.+C.+Chavezauthor=L.+V.+Phamauthor=P.+R.+Sudalaguntaauthor=L.+Wanauthor=X.+Wangauthor=W.+S.+Daltonauthor=L.+C.+Moscinskiauthor=K.+H.+Shainauthor=J.+Voseauthor=J.+L.+Clevelandauthor=E.+M.+Sotomayorauthor=K.+Fuauthor=J.+Tao&title=PLK1+stabilizes+a+MYC-dependent+kinase+network+in+aggressive+B+cell+lymphomas&doi=10.1172%2FJCI122533"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit99R"><div class="casContent"><span class="casTitleNuber">99</span><div class="casTitle"><span class="NLM_cas:atitle">PLK1 stabilizes a MYC-dependent kinase network in aggressive B cell lymphomas</span></div><div class="casAuthors">Ren Yuan; Zhao Xiaohong; Lwin Tint; Li Tao; Jiang Huijuan; Moscinski Lynn C; Tao Jianguo; Bi Chengfeng; Wang Cheng; Yuan Ji; Fu Kai; Silva Ariosto S; Sudalagunta Praneeth R; Shah Bijal D; Chavez Julio C; Dalton William S; Shain Kenneth H; Tao Jianguo; Fang Bin; Koomen John M; Jiang Huijuan; Pham Lan V; Wan Lixin; Wang Xuefeng; Vose Julie; Cleveland John L; Sotomayor Eduardo M</div><div class="citationInfo"><span class="NLM_cas:title">The Journal of clinical investigation</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">128</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">5517-5530</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Concordant activation of MYC and BCL-2 oncoproteins in double-hit lymphoma (DHL) results in aggressive disease that is refractory to treatment.  By integrating activity-based proteomic profiling and drug screens, polo-like kinase-1 (PLK1) was identified as an essential regulator of the MYC-dependent kinome in DHL.  Notably, PLK1 was expressed at high levels in DHL, correlated with MYC expression, and connoted poor outcome.  Further, PLK1 signaling augmented MYC protein stability, and in turn, MYC directly induced PLK1 transcription, establishing a feed-forward MYC-PLK1 circuit in DHL.  Finally, inhibition of PLK1 triggered degradation of MYC and of the antiapoptotic protein MCL-1, and PLK1 inhibitors showed synergy with BCL-2 antagonists in blocking DHL cell growth, survival, and tumorigenicity, supporting clinical targeting of PLK1 in DHL.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRyRm5q5xX4CfxQvRxYU81tfW6udTcc2eZLlZxD4-ELtrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3czjt1yntw%253D%253D&md5=fec6af4dad461a48831839f52913b704</span></div><a href="/servlet/linkout?suffix=cit99&amp;dbid=16384&amp;doi=10.1172%2FJCI122533&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1172%252FJCI122533%26sid%3Dliteratum%253Aachs%26aulast%3DRen%26aufirst%3DY.%26aulast%3DBi%26aufirst%3DC.%26aulast%3DZhao%26aufirst%3DX.%26aulast%3DLwin%26aufirst%3DT.%26aulast%3DWang%26aufirst%3DC.%26aulast%3DYuan%26aufirst%3DJ.%26aulast%3DSilva%26aufirst%3DA.%2BS.%26aulast%3DShah%26aufirst%3DB.%2BD.%26aulast%3DFang%26aufirst%3DB.%26aulast%3DLi%26aufirst%3DT.%26aulast%3DKoomen%26aufirst%3DJ.%2BM.%26aulast%3DJiang%26aufirst%3DH.%26aulast%3DChavez%26aufirst%3DJ.%2BC.%26aulast%3DPham%26aufirst%3DL.%2BV.%26aulast%3DSudalagunta%26aufirst%3DP.%2BR.%26aulast%3DWan%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DDalton%26aufirst%3DW.%2BS.%26aulast%3DMoscinski%26aufirst%3DL.%2BC.%26aulast%3DShain%26aufirst%3DK.%2BH.%26aulast%3DVose%26aufirst%3DJ.%26aulast%3DCleveland%26aufirst%3DJ.%2BL.%26aulast%3DSotomayor%26aufirst%3DE.%2BM.%26aulast%3DFu%26aufirst%3DK.%26aulast%3DTao%26aufirst%3DJ.%26atitle%3DPLK1%2520stabilizes%2520a%2520MYC-dependent%2520kinase%2520network%2520in%2520aggressive%2520B%2520cell%2520lymphomas%26jtitle%3DJ.%2520Clin.%2520Invest.%26date%3D2018%26volume%3D128%26spage%3D5517%26epage%3D5530%26doi%3D10.1172%2FJCI122533" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref100"><div class="reference"><strong class="refLabel"><a href="#ref100" class="rightTabRefNumLink">100</a></strong><div class="NLM_citation" id="rightTab-cit100"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiao, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zuo, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, L.</span></span> <span> </span><span class="NLM_article-title">Novel dual BET and PLK1 inhibitor WNY0824 exerts potent antitumor effects in CRPC by inhibiting transcription factor function and inducing mitotic abnormality</span>. <i>Mol. Cancer Ther.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">1221</span>– <span class="NLM_lpage">1231</span>, <span class="refDoi"> DOI: 10.1158/1535-7163.MCT-19-0578</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref100/cit100&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1158%2F1535-7163.MCT-19-0578" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref100/cit100&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=32220972" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref100/cit100&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhvVems7%252FO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2020&pages=1221-1231&author=Y.+Xuauthor=Q.+Wangauthor=K.+Xiaoauthor=Z.+Liuauthor=L.+Zhaoauthor=X.+Songauthor=X.+Huauthor=Z.+Fengauthor=T.+Gaoauthor=W.+Zuoauthor=J.+Zengauthor=N.+Wangauthor=L.+Yu&title=Novel+dual+BET+and+PLK1+inhibitor+WNY0824+exerts+potent+antitumor+effects+in+CRPC+by+inhibiting+transcription+factor+function+and+inducing+mitotic+abnormality&doi=10.1158%2F1535-7163.MCT-19-0578"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit100R"><div class="casContent"><span class="casTitleNuber">100</span><div class="casTitle"><span class="NLM_cas:atitle">Novel dual BET and PLK1 inhibitor WNY0824 exerts potent antitumor effects in crpc by inhibiting transcription factor function and inducing mitotic abnormality</span></div><div class="casAuthors">Xu, Ying; Wang, Qianqian; Xiao, Kunjie; Liu, Zhihao; Zhao, Lifeng; Song, Xuejiao; Hu, Xi; Feng, Zhanzhan; Gao, Tiantao; Zuo, Weiqiong; Zeng, Jun; Wang, Ningyu; Yu, Luoting</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cancer Therapeutics</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1221-1231</span>CODEN:
                <span class="NLM_cas:coden">MCTOCF</span>;
        ISSN:<span class="NLM_cas:issn">1535-7163</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Castration-resistant prostate cancer (CRPC) is a lethal disease with few treatment alternatives once patients become resistant to second-generation antiandrogens.  In CRPC, BET proteins are key regulators of AR- and MYC-mediated transcription, while the PLK1 inhibitor potentially downregulates AR and MYC besides influencing the cell cycle.  Therefore, synchronous inhibition of BET and PLK1 would be a promising approach for CRPC therapy.  This study developed a dual BET and PLK1 inhibitor WNY0824 with nanomolar and equipotent inhibition of BRD4 and PLK1.  In vitro, WNY0824 exhibited excellent antiproliferation activity on AR-pos. CRPC cells and induced apoptosis.  These activities are attributable to its disruption of the AR-transcriptional program and the inhibition of the ETS pathway.  Furthermore, WNY0824 downregulated MYC and induced mitotic abnormality.  In vivo, oral WNY0824 administration suppressed tumor growth in the CRPC xenograft model of enzalutamide resistance.  These findings suggest that WNY0824 is a selective dual BET and PLK1 inhibitor with potent anti-CRPC oncogenic activity and provides insights into the development of other novel dual BET- and PLK1-inhibiting drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpNhuOBmxkBB7Vg90H21EOLACvtfcHk0lhrv-lGvUwMgA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhvVems7%252FO&md5=74c4a259cded1d4cb89a2d2cbcb36442</span></div><a href="/servlet/linkout?suffix=cit100&amp;dbid=16384&amp;doi=10.1158%2F1535-7163.MCT-19-0578&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1535-7163.MCT-19-0578%26sid%3Dliteratum%253Aachs%26aulast%3DXu%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DQ.%26aulast%3DXiao%26aufirst%3DK.%26aulast%3DLiu%26aufirst%3DZ.%26aulast%3DZhao%26aufirst%3DL.%26aulast%3DSong%26aufirst%3DX.%26aulast%3DHu%26aufirst%3DX.%26aulast%3DFeng%26aufirst%3DZ.%26aulast%3DGao%26aufirst%3DT.%26aulast%3DZuo%26aufirst%3DW.%26aulast%3DZeng%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DN.%26aulast%3DYu%26aufirst%3DL.%26atitle%3DNovel%2520dual%2520BET%2520and%2520PLK1%2520inhibitor%2520WNY0824%2520exerts%2520potent%2520antitumor%2520effects%2520in%2520CRPC%2520by%2520inhibiting%2520transcription%2520factor%2520function%2520and%2520inducing%2520mitotic%2520abnormality%26jtitle%3DMol.%2520Cancer%2520Ther.%26date%3D2020%26volume%3D19%26spage%3D1221%26epage%3D1231%26doi%3D10.1158%2F1535-7163.MCT-19-0578" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref101"><div class="reference"><strong class="refLabel"><a href="#ref101" class="rightTabRefNumLink">101</a></strong><div class="NLM_citation" id="rightTab-cit101"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yosief, H. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dai, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dhawan, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muthengi, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buckley, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perry, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, W.</span></span> <span> </span><span class="NLM_article-title">Structure-guided design and development of potent and selective dual bromodomain 4 (BRD4)/polo-like kinase 1 (PLK1) inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">7785</span>– <span class="NLM_lpage">7795</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b00765</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b00765" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref101/cit101&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhsFCksr3L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=7785-7795&author=S.+Liuauthor=H.+O.+Yosiefauthor=L.+Daiauthor=H.+Huangauthor=G.+Dhawanauthor=X.+Zhangauthor=A.+M.+Muthengiauthor=J.+Robertsauthor=D.+L.+Buckleyauthor=J.+A.+Perryauthor=L.+Wuauthor=J.+E.+Bradnerauthor=J.+Qiauthor=W.+Zhang&title=Structure-guided+design+and+development+of+potent+and+selective+dual+bromodomain+4+%28BRD4%29%2Fpolo-like+kinase+1+%28PLK1%29+inhibitors&doi=10.1021%2Facs.jmedchem.8b00765"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit101R"><div class="casContent"><span class="casTitleNuber">101</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Guided Design and Development of Potent and Selective Dual Bromodomain 4 (BRD4)/Polo-like Kinase 1 (PLK1) Inhibitors</span></div><div class="casAuthors">Liu, Shuai; Yosief, Hailemichael O.; Dai, Lingling; Huang, He; Dhawan, Gagan; Zhang, Xiaofeng; Muthengi, Alex M.; Roberts, Justin; Buckley, Dennis L.; Perry, Jennifer A.; Wu, Lei; Bradner, James E.; Qi, Jun; Zhang, Wei</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">7785-7795</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The simultaneous inhibition of polo-like kinase 1 (PLK1) and BRD4 bromodomain by a single mol. could lead to the development of an effective therapeutic strategy for a variety of diseases in which PLK1 and BRD4 are implicated.  Compd. 23 has been found to be a potent dual kinase-bromodomain inhibitor (BRD4-BD1 IC50 = 28 nM, PLK1 IC50 = 40 nM).  Compd. 6 was found to be the most selective PLK1 inhibitor over BRD4 in our series (BRD4-BD1 IC50 = 2579 nM, PLK1 IC50 = 9.9 nM).  Mol. docking studies with 23 and BRD4-BD1/PLK1 as well as with 6 corroborate the biochem. assay results.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoUL2G-DnFzU7Vg90H21EOLACvtfcHk0lhrv-lGvUwMgA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhsFCksr3L&md5=e34a58694f0827d33d31516200e2c0d8</span></div><a href="/servlet/linkout?suffix=cit101&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b00765&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b00765%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DS.%26aulast%3DYosief%26aufirst%3DH.%2BO.%26aulast%3DDai%26aufirst%3DL.%26aulast%3DHuang%26aufirst%3DH.%26aulast%3DDhawan%26aufirst%3DG.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DMuthengi%26aufirst%3DA.%2BM.%26aulast%3DRoberts%26aufirst%3DJ.%26aulast%3DBuckley%26aufirst%3DD.%2BL.%26aulast%3DPerry%26aufirst%3DJ.%2BA.%26aulast%3DWu%26aufirst%3DL.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26aulast%3DQi%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DW.%26atitle%3DStructure-guided%2520design%2520and%2520development%2520of%2520potent%2520and%2520selective%2520dual%2520bromodomain%25204%2520%2528BRD4%2529%252Fpolo-like%2520kinase%25201%2520%2528PLK1%2529%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D7785%26epage%3D7795%26doi%3D10.1021%2Facs.jmedchem.8b00765" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref102"><div class="reference"><strong class="refLabel"><a href="#ref102" class="rightTabRefNumLink">102</a></strong><div class="NLM_citation" id="rightTab-cit102"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Watts, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heidenreich, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tucker, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raab, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Strebhardt, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chesler, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bellenie, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoelder, S.</span></span> <span> </span><span class="NLM_article-title">Designing dual inhibitors of anaplastic lymphoma kinase (ALK) and bromodomain-4 (BRD4) by tuning kinase selectivity</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">2618</span>– <span class="NLM_lpage">2637</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b01947</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b01947" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref102/cit102&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BC1MXjtlGmu74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=2618-2637&author=E.+Wattsauthor=D.+Heidenreichauthor=E.+Tuckerauthor=M.+Raabauthor=K.+Strebhardtauthor=L.+Cheslerauthor=S.+Knappauthor=B.+Bellenieauthor=S.+Hoelder&title=Designing+dual+inhibitors+of+anaplastic+lymphoma+kinase+%28ALK%29+and+bromodomain-4+%28BRD4%29+by+tuning+kinase+selectivity&doi=10.1021%2Facs.jmedchem.8b01947"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit102R"><div class="casContent"><span class="casTitleNuber">102</span><div class="casTitle"><span class="NLM_cas:atitle">Designing Dual Inhibitors of Anaplastic Lymphoma Kinase (ALK) and Bromodomain-4 (BRD4) by Tuning Kinase Selectivity</span></div><div class="casAuthors">Watts, Ellen; Heidenreich, David; Tucker, Elizabeth; Raab, Monika; Strebhardt, Klaus; Chesler, Louis; Knapp, Stefan; Bellenie, Benjamin; Hoelder, Swen</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">2618-2637</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Concomitant inhibition of anaplastic lymphoma kinase (ALK) and bromodomain-4 (BRD4) is a potential therapeutic strategy for targeting two key oncogenic drivers that co-segregate in a significant fraction of high-risk neuroblastoma patients, mutation of ALK and amplification of MYCN.  Starting from known dual polo-like kinase (PLK)-1-BRD4 inhibitor BI-2536, we employed structure-based design to redesign this series toward compds. with a dual ALK-BRD4 profile.  These efforts led to compd. (R)-2-((2-ethoxy-4-(1-methylpiperidin-4-yl)phenyl)amino)-7-ethyl-5-methyl-8-((4-methylthiophen-2-yl)methyl)-7,8-dihydropteridin-6(5H)-one (16k) demonstrating improved ALK activity and significantly reduced PLK-1 activity, while maintaining BRD4 activity and overall kinome selectivity.  We demonstrate the compds.' on-target engagement with ALK and BRD4 in cells as well as favorable broad kinase and bromodomain selectivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGop6qx6PSj2KLVg90H21EOLACvtfcHk0ljxM7whwYEZ6w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXjtlGmu74%253D&md5=e00fdf8eb20acc2ef1c8777b2afb750d</span></div><a href="/servlet/linkout?suffix=cit102&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b01947&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b01947%26sid%3Dliteratum%253Aachs%26aulast%3DWatts%26aufirst%3DE.%26aulast%3DHeidenreich%26aufirst%3DD.%26aulast%3DTucker%26aufirst%3DE.%26aulast%3DRaab%26aufirst%3DM.%26aulast%3DStrebhardt%26aufirst%3DK.%26aulast%3DChesler%26aufirst%3DL.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DBellenie%26aufirst%3DB.%26aulast%3DHoelder%26aufirst%3DS.%26atitle%3DDesigning%2520dual%2520inhibitors%2520of%2520anaplastic%2520lymphoma%2520kinase%2520%2528ALK%2529%2520and%2520bromodomain-4%2520%2528BRD4%2529%2520by%2520tuning%2520kinase%2520selectivity%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D2618%26epage%3D2637%26doi%3D10.1021%2Facs.jmedchem.8b01947" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref103"><div class="reference"><strong class="refLabel"><a href="#ref103" class="rightTabRefNumLink">103</a></strong><div class="NLM_citation" id="rightTab-cit103"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Koblan, L. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buckley, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ott, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fitzgerald, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ember, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, J. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Remillard, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vittori, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schonbrunn, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span> <span> </span><span class="NLM_article-title">Assessment of bromodomain target engagement by a series of BI2536 analogues with miniaturized BET-BRET</span>. <i>ChemMedChem</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">2575</span>– <span class="NLM_lpage">2581</span>, <span class="refDoi"> DOI: 10.1002/cmdc.201600502</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref103/cit103&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1002%2Fcmdc.201600502" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref103/cit103&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=27862999" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref103/cit103&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvVChsbfK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2016&pages=2575-2581&author=L.+W.+Koblanauthor=D.+L.+Buckleyauthor=C.+J.+Ottauthor=M.+E.+Fitzgeraldauthor=S.+W.+Emberauthor=J.+Y.+Zhuauthor=S.+Liuauthor=J.+M.+Robertsauthor=D.+Remillardauthor=S.+Vittoriauthor=W.+Zhangauthor=E.+Schonbrunnauthor=J.+E.+Bradner&title=Assessment+of+bromodomain+target+engagement+by+a+series+of+BI2536+analogues+with+miniaturized+BET-BRET&doi=10.1002%2Fcmdc.201600502"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit103R"><div class="casContent"><span class="casTitleNuber">103</span><div class="casTitle"><span class="NLM_cas:atitle">Assessment of Bromodomain Target Engagement by a Series of BI2536 Analogues with Miniaturized BET-BRET</span></div><div class="casAuthors">Koblan, Luke W.; Buckley, Dennis L.; Ott, Christopher J.; Fitzgerald, Mark E.; Ember, Stuart W. J.; Zhu, Jin-Yi; Liu, Shuai; Roberts, Justin M.; Remillard, David; Vittori, Sarah; Zhang, Wei; Schonbrunn, Ernst; Bradner, James E.</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">2575-2581</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Evaluating the engagement of a small mol. ligand with a protein target in cells provides useful information for chem. probe optimization and pharmaceutical development.  While several techniques exist that can be performed in a low-throughput manner, systematic evaluation of large compd. libraries remains a challenge.  In-cell engagement measurements are esp. useful when evaluating compd. classes suspected to target multiple cellular factors.  In this study we used a bioluminescent resonant energy transfer assay to assess bromodomain engagement by a compd. series contg. bromodomain- and kinase-biasing polypharmacophores based on the known dual BRD4 bromodomain/PLK1 kinase inhibitor BI2536.  With this assay, we discovered several novel agents with bromodomain-selective specificity profiles and cellular activity.  Thus, this platform aids in distinguishing mols. whose cellular activity is difficult to assess due to polypharmacol. effects.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpiVVtnPQvEQrVg90H21EOLACvtfcHk0ljxM7whwYEZ6w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvVChsbfK&md5=a6d7c6a58870eb6208e18eb32068d821</span></div><a href="/servlet/linkout?suffix=cit103&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.201600502&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.201600502%26sid%3Dliteratum%253Aachs%26aulast%3DKoblan%26aufirst%3DL.%2BW.%26aulast%3DBuckley%26aufirst%3DD.%2BL.%26aulast%3DOtt%26aufirst%3DC.%2BJ.%26aulast%3DFitzgerald%26aufirst%3DM.%2BE.%26aulast%3DEmber%26aufirst%3DS.%2BW.%26aulast%3DZhu%26aufirst%3DJ.%2BY.%26aulast%3DLiu%26aufirst%3DS.%26aulast%3DRoberts%26aufirst%3DJ.%2BM.%26aulast%3DRemillard%26aufirst%3DD.%26aulast%3DVittori%26aufirst%3DS.%26aulast%3DZhang%26aufirst%3DW.%26aulast%3DSchonbrunn%26aufirst%3DE.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26atitle%3DAssessment%2520of%2520bromodomain%2520target%2520engagement%2520by%2520a%2520series%2520of%2520BI2536%2520analogues%2520with%2520miniaturized%2520BET-BRET%26jtitle%3DChemMedChem%26date%3D2016%26volume%3D11%26spage%3D2575%26epage%3D2581%26doi%3D10.1002%2Fcmdc.201600502" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref104"><div class="reference"><strong class="refLabel"><a href="#ref104" class="rightTabRefNumLink">104</a></strong><div class="NLM_citation" id="rightTab-cit104"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ciceri, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Mahony, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Filippakopoulos, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hunt, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lasater, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pallares, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Picaud, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wells, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wodicka, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shah, N. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Treiber, D. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span> <span> </span><span class="NLM_article-title">Dual kinase-bromodomain inhibitors for rationally designed polypharmacology</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">305</span>– <span class="NLM_lpage">312</span>, <span class="refDoi"> DOI: 10.1038/nchembio.1471</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref104/cit104&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1038%2Fnchembio.1471" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref104/cit104&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=24584101" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref104/cit104&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BC2cXjtl2itro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2014&pages=305-312&author=P.+Ciceriauthor=S.+Mullerauthor=A.+O%E2%80%99Mahonyauthor=O.+Fedorovauthor=P.+Filippakopoulosauthor=J.+P.+Huntauthor=E.+A.+Lasaterauthor=G.+Pallaresauthor=S.+Picaudauthor=C.+Wellsauthor=S.+Martinauthor=L.+M.+Wodickaauthor=N.+P.+Shahauthor=D.+K.+Treiberauthor=S.+Knapp&title=Dual+kinase-bromodomain+inhibitors+for+rationally+designed+polypharmacology&doi=10.1038%2Fnchembio.1471"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit104R"><div class="casContent"><span class="casTitleNuber">104</span><div class="casTitle"><span class="NLM_cas:atitle">Dual kinase-bromodomain inhibitors for rationally designed polypharmacology</span></div><div class="casAuthors">Ciceri, Pietro; Mueller, Susanne; O'Mahony, Alison; Fedorov, Oleg; Filippakopoulos, Panagis; Hunt, Jeremy P.; Lasater, Elisabeth A.; Pallares, Gabriel; Picaud, Sarah; Wells, Christopher; Martin, Sarah; Wodicka, Lisa M.; Shah, Neil P.; Treiber, Daniel K.; Knapp, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">305-312</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Concomitant inhibition of multiple cancer-driving kinases is an established strategy to improve the durability of clin. responses to targeted therapies.  The difficulty of discovering kinase inhibitors with an appropriate multitarget profile has, however, necessitated the application of combination therapies, which can pose major clin. development challenges.  Epigenetic reader domains of the bromodomain family have recently emerged as new targets for cancer therapy.  Here the authors report that several clin. kinase inhibitors also inhibit bromodomains with therapeutically relevant potencies and are best classified as dual kinase-bromodomain inhibitors.  Nanomolar activity on BRD4 by BI-2536 and TG-101348, which are clin. PLK1 and JAK2-FLT3 kinase inhibitors, resp., is particularly noteworthy as these combinations of activities on independent oncogenic pathways exemplify a new strategy for rational single-agent polypharmacol. targeting.  Furthermore, structure-activity relationships and co-crystal structures identify design features that enable a general platform for the rational design of dual kinase-bromodomain inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGotPedDED4FJLVg90H21EOLACvtfcHk0ljxM7whwYEZ6w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXjtl2itro%253D&md5=a00117db2648ea38dd03750faf3b3cb9</span></div><a href="/servlet/linkout?suffix=cit104&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1471&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1471%26sid%3Dliteratum%253Aachs%26aulast%3DCiceri%26aufirst%3DP.%26aulast%3DMuller%26aufirst%3DS.%26aulast%3DO%25E2%2580%2599Mahony%26aufirst%3DA.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DHunt%26aufirst%3DJ.%2BP.%26aulast%3DLasater%26aufirst%3DE.%2BA.%26aulast%3DPallares%26aufirst%3DG.%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DWells%26aufirst%3DC.%26aulast%3DMartin%26aufirst%3DS.%26aulast%3DWodicka%26aufirst%3DL.%2BM.%26aulast%3DShah%26aufirst%3DN.%2BP.%26aulast%3DTreiber%26aufirst%3DD.%2BK.%26aulast%3DKnapp%26aufirst%3DS.%26atitle%3DDual%2520kinase-bromodomain%2520inhibitors%2520for%2520rationally%2520designed%2520polypharmacology%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2014%26volume%3D10%26spage%3D305%26epage%3D312%26doi%3D10.1038%2Fnchembio.1471" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref105"><div class="reference"><strong class="refLabel"><a href="#ref105" class="rightTabRefNumLink">105</a></strong><div class="NLM_citation" id="rightTab-cit105"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wan, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xia, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duan, G.</span></span> <span> </span><span class="NLM_article-title">Dual-target inhibitors based on HDACs: novel antitumor agents for cancer therapy</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">8977</span>– <span class="NLM_lpage">9002</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.0c00491</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.0c00491" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref105/cit105&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BB3cXnslShtL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=8977-9002&author=T.+Liuauthor=Y.+Wanauthor=Y.+Xiaoauthor=C.+Xiaauthor=G.+Duan&title=Dual-target+inhibitors+based+on+HDACs%3A+novel+antitumor+agents+for+cancer+therapy&doi=10.1021%2Facs.jmedchem.0c00491"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit105R"><div class="casContent"><span class="casTitleNuber">105</span><div class="casTitle"><span class="NLM_cas:atitle">Dual-Target Inhibitors Based on HDACs: Novel Antitumor Agents for Cancer Therapy</span></div><div class="casAuthors">Liu, Tingting; Wan, Yichao; Xiao, Yuliang; Xia, Chengcai; Duan, Guiyun</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">8977-9002</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Histone deacetylases (HDACs) play an important role in regulating target gene expression.  They have been highlighted as a novel category of anticancer targets, and their inhibition can induce apoptosis, differentiation, and growth arrest in cancer cells.  In view of the fact that HDAC inhibitors and other antitumor agents, such as BET inhibitors, topoisomerase inhibitors, and RTK pathway inhibitors, exert a synergistic effect on cellular processes in cancer cells, the combined inhibition of two targets is regarded as a rational strategy to improve the effectiveness of these single-target drugs for cancer treatment.  In this review, we discuss the theor. basis for designing HDAC-involved dual-target drugs and provide insight into the structure-activity relationships of these dual-target agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGohbD4ktOU4jbVg90H21EOLACvtfcHk0lgQPLEMv4osXg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXnslShtL8%253D&md5=fa6764d35879c6a8973d57231854f821</span></div><a href="/servlet/linkout?suffix=cit105&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.0c00491&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.0c00491%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DT.%26aulast%3DWan%26aufirst%3DY.%26aulast%3DXiao%26aufirst%3DY.%26aulast%3DXia%26aufirst%3DC.%26aulast%3DDuan%26aufirst%3DG.%26atitle%3DDual-target%2520inhibitors%2520based%2520on%2520HDACs%253A%2520novel%2520antitumor%2520agents%2520for%2520cancer%2520therapy%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D8977%26epage%3D9002%26doi%3D10.1021%2Facs.jmedchem.0c00491" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref106"><div class="reference"><strong class="refLabel"><a href="#ref106" class="rightTabRefNumLink">106</a></strong><div class="NLM_citation" id="rightTab-cit106"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Carlson, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cui, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Divakaran, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brunner, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pomerantz, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Topczewski, J. J.</span></span> <span> </span><span class="NLM_article-title">Systematically mitigating the p38alpha activity of triazole-based BET inhibitors</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">1296</span>– <span class="NLM_lpage">1301</span>, <span class="refDoi"> DOI: 10.1021/acsmedchemlett.9b00227</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.9b00227" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref106/cit106&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhsFSku7jE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2019&pages=1296-1301&author=A.+S.+Carlsonauthor=H.+Cuiauthor=A.+Divakaranauthor=J.+A.+Johnsonauthor=R.+M.+Brunnerauthor=W.+Pomerantzauthor=J.+J.+Topczewski&title=Systematically+mitigating+the+p38alpha+activity+of+triazole-based+BET+inhibitors&doi=10.1021%2Facsmedchemlett.9b00227"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit106R"><div class="casContent"><span class="casTitleNuber">106</span><div class="casTitle"><span class="NLM_cas:atitle">Systematically Mitigating the p38α Activity of Triazole-based BET Inhibitors</span></div><div class="casAuthors">Carlson, Angela S.; Cui, Huarui; Divakaran, Anand; Johnson, Jorden A.; Brunner, Ryan M.; Pomerantz, William C. K.; Topczewski, Joseph J.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1296-1301</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The Bromodomain and Extra Terminal (BET) family of proteins recognize post-translational N-ε-acetylated lysine modifications and act as "reader" proteins in regulating transcription.  This core function makes bromodomain inhibitors interesting targets for the development of potential cancer, inflammation, and heart disease treatments.  Several dual kinase-bromodomain inhibitors have been identified by screening kinase inhibitor libraries against BET proteins.  Although potentially useful from a polypharmacol. standpoint, multi-target binding complicates deciphering mol. mechanisms.  This report describes a systematic approach to mitigating kinase activity in a dual kinase-bromodomain inhibitor based on a 1,2,3-triazole core.  This SAR study enhanced BET activity while reducing the p38α kinase activity >200,000 fold.  A BRD4-D1 co-crystal structure indicates that the 1,2,3-triazole is acting as a N-ε-acetylated lysine mimic.  A BRD4 sensitive cell line, MM.1S, was used to demonstrate activity in cells, which is further sup-ported by reduced c-Myc expression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo2d-QBOuUZHLVg90H21EOLACvtfcHk0lgQPLEMv4osXg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhsFSku7jE&md5=c4f776357db94fca3bdcd4bdc8f377d8</span></div><a href="/servlet/linkout?suffix=cit106&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.9b00227&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.9b00227%26sid%3Dliteratum%253Aachs%26aulast%3DCarlson%26aufirst%3DA.%2BS.%26aulast%3DCui%26aufirst%3DH.%26aulast%3DDivakaran%26aufirst%3DA.%26aulast%3DJohnson%26aufirst%3DJ.%2BA.%26aulast%3DBrunner%26aufirst%3DR.%2BM.%26aulast%3DPomerantz%26aufirst%3DW.%26aulast%3DTopczewski%26aufirst%3DJ.%2BJ.%26atitle%3DSystematically%2520mitigating%2520the%2520p38alpha%2520activity%2520of%2520triazole-based%2520BET%2520inhibitors%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2019%26volume%3D10%26spage%3D1296%26epage%3D1301%26doi%3D10.1021%2Facsmedchemlett.9b00227" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref107"><div class="reference"><strong class="refLabel"><a href="#ref107" class="rightTabRefNumLink">107</a></strong><div class="NLM_citation" id="rightTab-cit107"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yoon, H. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, J.</span></span> <span> </span><span class="NLM_article-title">Regulation of P-TEFb elongation complex activity by CDK9 acetylation</span>. <i>Mol. Cell. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">4641</span>– <span class="NLM_lpage">4651</span>, <span class="refDoi"> DOI: 10.1128/MCB.00857-06</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref107/cit107&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1128%2FMCB.00857-06" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref107/cit107&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=17452463" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref107/cit107&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BD2sXnsVWrs7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2007&pages=4641-4651&author=J.+Fuauthor=H.+G.+Yoonauthor=J.+Qinauthor=J.+Wong&title=Regulation+of+P-TEFb+elongation+complex+activity+by+CDK9+acetylation&doi=10.1128%2FMCB.00857-06"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit107R"><div class="casContent"><span class="casTitleNuber">107</span><div class="casTitle"><span class="NLM_cas:atitle">Regulation of P-TEFb elongation complex activity by CDK9 acetylation</span></div><div class="casAuthors">Fu, Junjiang; Yoon, Ho-Geun; Qin, Jun; Wong, Jiemin</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Biology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">4641-4651</span>CODEN:
                <span class="NLM_cas:coden">MCEBD4</span>;
        ISSN:<span class="NLM_cas:issn">0270-7306</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">P-TEFb, comprised of CDK9 and a cyclin T subunit, is a global transcriptional elongation factor important for most RNA polymerase II (pol II) transcription.  P-TEFb facilitates transcription elongation in part by phosphorylating Ser2 of the heptapeptide repeat of the carboxy-terminal domain (CTD) of the largest subunit of pol II.  Previous studies have shown that P-TEFb is subjected to neg. regulation by forming an inactive complex with 7SK small RNA and HEXIM1.  In an effort to investigate the mol. mechanism by which corepressor N-CoR mediates transcription repression, we identified HEXIM1 as an N-CoR-interacting protein.  This finding led us to test whether the P-TEFb complex is regulated by acetylation.  We demonstrate that CDK9 is an acetylated protein in cells and can be acetylated by p300 in vitro.  Through both in vitro and in vivo assays, we identified lysine 44 of CDK9 as a major acetylation site.  We present evidence that CDK9 is regulated by N-CoR and its assocd. HDAC3 and that acetylation of CDK9 affects its ability to phosphorylate the CTD of pol II.  These results suggest that acetylation of CDK9 is an important posttranslational modification that is involved in regulating P-TEFb transcriptional elongation function.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo1MWK4pAUTKrVg90H21EOLACvtfcHk0lgQPLEMv4osXg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXnsVWrs7c%253D&md5=24a3b35574a6f126baa8bcd4a1fd62a8</span></div><a href="/servlet/linkout?suffix=cit107&amp;dbid=16384&amp;doi=10.1128%2FMCB.00857-06&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMCB.00857-06%26sid%3Dliteratum%253Aachs%26aulast%3DFu%26aufirst%3DJ.%26aulast%3DYoon%26aufirst%3DH.%2BG.%26aulast%3DQin%26aufirst%3DJ.%26aulast%3DWong%26aufirst%3DJ.%26atitle%3DRegulation%2520of%2520P-TEFb%2520elongation%2520complex%2520activity%2520by%2520CDK9%2520acetylation%26jtitle%3DMol.%2520Cell.%2520Biol.%26date%3D2007%26volume%3D27%26spage%3D4641%26epage%3D4651%26doi%3D10.1128%2FMCB.00857-06" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref108"><div class="reference"><strong class="refLabel"><a href="#ref108" class="rightTabRefNumLink">108</a></strong><div class="NLM_citation" id="rightTab-cit108"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sakamoto, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, K. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumagai, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mercurio, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deshaies, R. J.</span></span> <span> </span><span class="NLM_article-title">Protacs: chimeric molecules that target proteins to the Skp1-Cullin-F box complex for ubiquitination and degradation</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>98</i></span>,  <span class="NLM_fpage">8554</span>– <span class="NLM_lpage">8559</span>, <span class="refDoi"> DOI: 10.1073/pnas.141230798</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref108/cit108&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1073%2Fpnas.141230798" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref108/cit108&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=11438690" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref108/cit108&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BD3MXls1Wisbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=98&publication_year=2001&pages=8554-8559&author=K.+M.+Sakamotoauthor=K.+B.+Kimauthor=A.+Kumagaiauthor=F.+Mercurioauthor=C.+M.+Crewsauthor=R.+J.+Deshaies&title=Protacs%3A+chimeric+molecules+that+target+proteins+to+the+Skp1-Cullin-F+box+complex+for+ubiquitination+and+degradation&doi=10.1073%2Fpnas.141230798"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit108R"><div class="casContent"><span class="casTitleNuber">108</span><div class="casTitle"><span class="NLM_cas:atitle">Protacs: chimeric molecules that target proteins to the Skp1-Cullin-F box complex for ubiquitination and degradation</span></div><div class="casAuthors">Sakamoto, Kathleen M.; Kim, Kyung B.; Kumagai, Akiko; Mercurio, Frank; Crews, Craig M.; Deshaies, Raymond J.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">98</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">8554-8559</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The intracellular levels of many proteins are regulated by ubiquitin-dependent proteolysis.  One of the best-characterized enzymes that catalyzes the attachment of ubiquitin to proteins is a ubiquitin ligase complex, Skp1-Cullin-F box complex contg. Hrt1 (SCF).  We sought to artificially target a protein to the SCF complex for ubiquitination and degrdn.  To this end, we tested methionine aminopeptidase-2 (MetAP-2), which covalently binds the angiogenesis inhibitor ovalicin.  A chimeric compd., protein-targeting chimeric mol. 1 (Protac-1), was synthesized to recruit MetAP-2 to SCF.  One domain of Protac-1 contains the IκBα phosphopeptide that is recognized by the F-box protein β-TRCP, whereas the other domain is composed of ovalicin.  We show that MetAP-2 can be tethered to SCFβ-TRCP, ubiquitinated, and degraded in a Protac-1-dependent manner.  In the future, this approach may be useful for conditional inactivation of proteins, and for targeting disease-causing proteins for destruction.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp49fYtuEVlJrVg90H21EOLACvtfcHk0lhIms1aGXiKKA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXls1Wisbk%253D&md5=540358035222f745f7b6367b38781a21</span></div><a href="/servlet/linkout?suffix=cit108&amp;dbid=16384&amp;doi=10.1073%2Fpnas.141230798&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.141230798%26sid%3Dliteratum%253Aachs%26aulast%3DSakamoto%26aufirst%3DK.%2BM.%26aulast%3DKim%26aufirst%3DK.%2BB.%26aulast%3DKumagai%26aufirst%3DA.%26aulast%3DMercurio%26aufirst%3DF.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26aulast%3DDeshaies%26aufirst%3DR.%2BJ.%26atitle%3DProtacs%253A%2520chimeric%2520molecules%2520that%2520target%2520proteins%2520to%2520the%2520Skp1-Cullin-F%2520box%2520complex%2520for%2520ubiquitination%2520and%2520degradation%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2001%26volume%3D98%26spage%3D8554%26epage%3D8559%26doi%3D10.1073%2Fpnas.141230798" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref109"><div class="reference"><strong class="refLabel"><a href="#ref109" class="rightTabRefNumLink">109</a></strong><div class="NLM_citation" id="rightTab-cit109"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lai, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Induced protein degradation: an emerging drug discovery paradigm</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">101</span>– <span class="NLM_lpage">114</span>, <span class="refDoi"> DOI: 10.1038/nrd.2016.211</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref109/cit109&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1038%2Fnrd.2016.211" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref109/cit109&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=27885283" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref109/cit109&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvFCit7rP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2017&pages=101-114&author=A.+C.+Laiauthor=C.+M.+Crews&title=Induced+protein+degradation%3A+an+emerging+drug+discovery+paradigm&doi=10.1038%2Fnrd.2016.211"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit109R"><div class="casContent"><span class="casTitleNuber">109</span><div class="casTitle"><span class="NLM_cas:atitle">Induced protein degradation: an emerging drug discovery paradigm</span></div><div class="casAuthors">Lai, Ashton C.; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">101-114</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Small-mol. drug discovery has traditionally focused on occupancy of a binding site that directly affects protein function, and this approach typically precludes targeting proteins that lack such amenable sites.  Furthermore, high systemic drug exposures may be needed to maintain sufficient target inhibition in vivo, increasing the risk of undesirable off-target effects.  Induced protein degrdn. is an alternative approach that is event-driven: upon drug binding, the target protein is tagged for elimination.  Emerging technologies based on proteolysis-targeting chimaeras (PROTACs) that exploit cellular quality control machinery to selectively degrade target proteins are attracting considerable attention in the pharmaceutical industry owing to the advantages they could offer over traditional small-mol. strategies.  These advantages include the potential to reduce systemic drug exposure, the ability to counteract increased target protein expression that often accompanies inhibition of protein function and the potential ability to target proteins that are not currently therapeutically tractable, such as transcription factors, scaffolding and regulatory proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoj-O7zXGOpXLVg90H21EOLACvtfcHk0lhIms1aGXiKKA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvFCit7rP&md5=a01ff43e9b696fe4dc03f0815158a98f</span></div><a href="/servlet/linkout?suffix=cit109&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2016.211&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2016.211%26sid%3Dliteratum%253Aachs%26aulast%3DLai%26aufirst%3DA.%2BC.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DInduced%2520protein%2520degradation%253A%2520an%2520emerging%2520drug%2520discovery%2520paradigm%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2017%26volume%3D16%26spage%3D101%26epage%3D114%26doi%3D10.1038%2Fnrd.2016.211" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref110"><div class="reference"><strong class="refLabel"><a href="#ref110" class="rightTabRefNumLink">110</a></strong><div class="NLM_citation" id="rightTab-cit110"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bai, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fernandez-Salas, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mceachern, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Przybranowski, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wen, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span> <span> </span><span class="NLM_article-title">Discovery of a small-molecule degrader of bromodomain and extra-terminal (BET) proteins with picomolar cellular potencies and capable of achieving tumor regression</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">462</span>– <span class="NLM_lpage">481</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b01816</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01816" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref110/cit110&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BC2sXkvVajsb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=462-481&author=B.+Zhouauthor=J.+Huauthor=F.+Xuauthor=Z.+Chenauthor=L.+Baiauthor=E.+Fernandez-Salasauthor=M.+Linauthor=L.+Liuauthor=C.+Y.+Yangauthor=Y.+Zhaoauthor=D.+Mceachernauthor=S.+Przybranowskiauthor=B.+Wenauthor=D.+Sunauthor=S.+Wang&title=Discovery+of+a+small-molecule+degrader+of+bromodomain+and+extra-terminal+%28BET%29+proteins+with+picomolar+cellular+potencies+and+capable+of+achieving+tumor+regression&doi=10.1021%2Facs.jmedchem.6b01816"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit110R"><div class="casContent"><span class="casTitleNuber">110</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a Small-Molecule Degrader of Bromodomain and Extra-Terminal (BET) Proteins with Picomolar Cellular Potencies and Capable of Achieving Tumor Regression</span></div><div class="casAuthors">Zhou, Bing; Hu, Jiantao; Xu, Fuming; Chen, Zhuo; Bai, Longchuan; Fernandez-Salas, Ester; Lin, Mei; Liu, Liu; Yang, Chao-Yie; Zhao, Yujun; McEachern, Donna; Przybranowski, Sally; Wen, Bo; Sun, Duxin; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">462-481</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The bromodomain and extra-terminal (BET) family proteins, consisting of BRD2, BRD3, BRD4, and testis-specific BRDT members, are epigenetic "readers" and play a key role in the regulation of gene transcription.  BET proteins are considered to be attractive therapeutic targets for cancer and other human diseases.  Recently, heterobifunctional small-mol. BET degraders have been designed based upon the proteolysis targeting chimera (PROTAC) concept to induce BET protein degrdn.  Herein, we present our design, synthesis, and evaluation of a new class of PROTAC BET degraders.  One of the most promising compds., 23, effectively degrades BRD4 protein at concns. as low as 30 pM in the RS4;11 leukemia cell line, achieves an IC50 value of 51 pM in inhibition of RS4;11 cell growth and induces rapid tumor regression in vivo against RS4;11 xenograft tumors.  These data establish that compd. 23 (BETd-260/ZBC260) is a highly potent and efficacious BET degrader.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqqz2Lwp52x1rVg90H21EOLACvtfcHk0lhIms1aGXiKKA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXkvVajsb8%253D&md5=6b3a10ecd192a5812ef581dca1fdc82a</span></div><a href="/servlet/linkout?suffix=cit110&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01816&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01816%26sid%3Dliteratum%253Aachs%26aulast%3DZhou%26aufirst%3DB.%26aulast%3DHu%26aufirst%3DJ.%26aulast%3DXu%26aufirst%3DF.%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DBai%26aufirst%3DL.%26aulast%3DFernandez-Salas%26aufirst%3DE.%26aulast%3DLin%26aufirst%3DM.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DYang%26aufirst%3DC.%2BY.%26aulast%3DZhao%26aufirst%3DY.%26aulast%3DMceachern%26aufirst%3DD.%26aulast%3DPrzybranowski%26aufirst%3DS.%26aulast%3DWen%26aufirst%3DB.%26aulast%3DSun%26aufirst%3DD.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DDiscovery%2520of%2520a%2520small-molecule%2520degrader%2520of%2520bromodomain%2520and%2520extra-terminal%2520%2528BET%2529%2520proteins%2520with%2520picomolar%2520cellular%2520potencies%2520and%2520capable%2520of%2520achieving%2520tumor%2520regression%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D462%26epage%3D481%26doi%3D10.1021%2Facs.jmedchem.6b01816" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref111"><div class="reference"><strong class="refLabel"><a href="#ref111" class="rightTabRefNumLink">111</a></strong><div class="NLM_citation" id="rightTab-cit111"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Winter, G. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buckley, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paulk, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Souza, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dhe-Paganon, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span> <span> </span><span class="NLM_article-title">DRUG DEVELOPMENT. Phthalimide conjugation as a strategy for in vivo target protein degradation</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>348</i></span>,  <span class="NLM_fpage">1376</span>– <span class="NLM_lpage">1381</span>, <span class="refDoi"> DOI: 10.1126/science.aab1433</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref111/cit111&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1126%2Fscience.aab1433" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref111/cit111&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=25999370" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref111/cit111&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BC2MXpvFansLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=348&publication_year=2015&pages=1376-1381&author=G.+E.+Winterauthor=D.+L.+Buckleyauthor=J.+Paulkauthor=J.+M.+Robertsauthor=A.+Souzaauthor=S.+Dhe-Paganonauthor=J.+E.+Bradner&title=DRUG+DEVELOPMENT.+Phthalimide+conjugation+as+a+strategy+for+in+vivo+target+protein+degradation&doi=10.1126%2Fscience.aab1433"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit111R"><div class="casContent"><span class="casTitleNuber">111</span><div class="casTitle"><span class="NLM_cas:atitle">Phthalimide conjugation as a strategy for in vivo target protein degradation</span></div><div class="casAuthors">Winter, Georg E.; Buckley, Dennis L.; Paulk, Joshiawa; Roberts, Justin M.; Souza, Amanda; Dhe-Paganon, Sirano; Bradner, James E.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">348</span>
        (<span class="NLM_cas:issue">6241</span>),
    <span class="NLM_cas:pages">1376-1381</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">The development of effective pharmacol. inhibitors of multidomain scaffold proteins, notably transcription factors, is a particularly challenging problem.  In part, this is because many small-mol. antagonists disrupt the activity of only one domain in the target protein.  The authors devised a chem. strategy that promotes ligand-dependent target protein degrdn. using as an example the transcriptional coactivator BRD4, a protein crit. for cancer cell growth and survival.  The authors appended a competitive antagonist of BET bromodomains to a phthalimide moiety to hijack the cereblon E3 ubiquitin ligase complex.  The resultant compd., dBET1, induced highly selective cereblon-dependent BET protein degrdn. in vitro and in vivo and delayed leukemia progression in mice.  A second series of probes resulted in selective degrdn. of the cytosolic protein FKBP12.  This chem. strategy for controlling target protein stability may have implications for therapeutically targeting previously intractable proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp1ZW4zjRjXi7Vg90H21EOLACvtfcHk0licpfjoyO29UQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXpvFansLk%253D&md5=b0277cc95d318d926b96e7a119066e3b</span></div><a href="/servlet/linkout?suffix=cit111&amp;dbid=16384&amp;doi=10.1126%2Fscience.aab1433&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.aab1433%26sid%3Dliteratum%253Aachs%26aulast%3DWinter%26aufirst%3DG.%2BE.%26aulast%3DBuckley%26aufirst%3DD.%2BL.%26aulast%3DPaulk%26aufirst%3DJ.%26aulast%3DRoberts%26aufirst%3DJ.%2BM.%26aulast%3DSouza%26aufirst%3DA.%26aulast%3DDhe-Paganon%26aufirst%3DS.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26atitle%3DDRUG%2520DEVELOPMENT.%2520Phthalimide%2520conjugation%2520as%2520a%2520strategy%2520for%2520in%2520vivo%2520target%2520protein%2520degradation%26jtitle%3DScience%26date%3D2015%26volume%3D348%26spage%3D1376%26epage%3D1381%26doi%3D10.1126%2Fscience.aab1433" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref112"><div class="reference"><strong class="refLabel"><a href="#ref112" class="rightTabRefNumLink">112</a></strong><div class="NLM_citation" id="rightTab-cit112"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qian, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Altieri, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raina, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hines, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winkler, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crew, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coleman, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Hijacking the E3 ubiquitin ligase cereblon to efficiently target BRD4</span>. <i>Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">755</span>– <span class="NLM_lpage">763</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2015.05.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref112/cit112&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1016%2Fj.chembiol.2015.05.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref112/cit112&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=26051217" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref112/cit112&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtVWrtbrM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2015&pages=755-763&author=J.+Luauthor=Y.+Qianauthor=M.+Altieriauthor=H.+Dongauthor=J.+Wangauthor=K.+Rainaauthor=J.+Hinesauthor=J.+D.+Winklerauthor=A.+P.+Crewauthor=K.+Colemanauthor=C.+M.+Crews&title=Hijacking+the+E3+ubiquitin+ligase+cereblon+to+efficiently+target+BRD4&doi=10.1016%2Fj.chembiol.2015.05.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit112R"><div class="casContent"><span class="casTitleNuber">112</span><div class="casTitle"><span class="NLM_cas:atitle">Hijacking the E3 Ubiquitin Ligase Cereblon to Efficiently Target BRD4</span></div><div class="casAuthors">Lu, Jing; Qian, Yimin; Altieri, Martha; Dong, Hanqing; Wang, Jing; Raina, Kanak; Hines, John; Winkler, James D.; Crew, Andrew P.; Coleman, Kevin; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">755-763</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">BRD4, a bromodomain and extraterminal domain (BET) family member, is an attractive target in multiple pathol. settings, particularly cancer.  While BRD4 inhibitors have shown some promise in MYC-driven malignancies such as Burkitt's lymphoma (BL), we show that BRD4 inhibitors lead to robust BRD4 protein accumulation, which may account for their limited suppression of MYC expression, modest antiproliferative activity, and lack of apoptotic induction.  To address these limitations we designed ARV-825, a hetero-bifunctional PROTAC (Proteolysis Targeting Chimera) that recruits BRD4 to the E3 ubiquitin ligase cereblon, leading to fast, efficient, and prolonged degrdn. of BRD4 in all BL cell lines tested.  Consequently, ARV-825 more effectively suppresses c-MYC levels and downstream signaling than small-mol. BRD4 inhibitors, resulting in more effective cell proliferation inhibition and apoptosis induction in BL.  Our findings provide strong evidence that cereblon-based PROTACs provide a better and more efficient strategy in targeting BRD4 than traditional small-mol. inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq6Ol4grCG2RbVg90H21EOLACvtfcHk0licpfjoyO29UQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtVWrtbrM&md5=a64675e7a0cd006a8737be9730f447e3</span></div><a href="/servlet/linkout?suffix=cit112&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2015.05.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2015.05.009%26sid%3Dliteratum%253Aachs%26aulast%3DLu%26aufirst%3DJ.%26aulast%3DQian%26aufirst%3DY.%26aulast%3DAltieri%26aufirst%3DM.%26aulast%3DDong%26aufirst%3DH.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DRaina%26aufirst%3DK.%26aulast%3DHines%26aufirst%3DJ.%26aulast%3DWinkler%26aufirst%3DJ.%2BD.%26aulast%3DCrew%26aufirst%3DA.%2BP.%26aulast%3DColeman%26aufirst%3DK.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DHijacking%2520the%2520E3%2520ubiquitin%2520ligase%2520cereblon%2520to%2520efficiently%2520target%2520BRD4%26jtitle%3DChem.%2520Biol.%26date%3D2015%26volume%3D22%26spage%3D755%26epage%3D763%26doi%3D10.1016%2Fj.chembiol.2015.05.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref113"><div class="reference"><strong class="refLabel"><a href="#ref113" class="rightTabRefNumLink">113</a></strong><div class="NLM_citation" id="rightTab-cit113"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bai, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ji, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mceachern, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Przybranowski, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fernandez-Salas, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dou, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wen, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meagher, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stuckey, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayes, D. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ellis, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span> <span> </span><span class="NLM_article-title">Targeted degradation of BET proteins in triple-negative breast cancer</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>77</i></span>,  <span class="NLM_fpage">2476</span>– <span class="NLM_lpage">2487</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.CAN-16-2622</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref113/cit113&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1158%2F0008-5472.CAN-16-2622" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref113/cit113&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=28209615" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref113/cit113&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BC2sXmvFakurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=77&publication_year=2017&pages=2476-2487&author=L.+Baiauthor=B.+Zhouauthor=C.+Y.+Yangauthor=J.+Jiauthor=D.+Mceachernauthor=S.+Przybranowskiauthor=H.+Jiangauthor=J.+Huauthor=F.+Xuauthor=Y.+Zhaoauthor=L.+Liuauthor=E.+Fernandez-Salasauthor=J.+Xuauthor=Y.+Douauthor=B.+Wenauthor=D.+Sunauthor=J.+Meagherauthor=J.+Stuckeyauthor=D.+F.+Hayesauthor=S.+Liauthor=M.+J.+Ellisauthor=S.+Wang&title=Targeted+degradation+of+BET+proteins+in+triple-negative+breast+cancer&doi=10.1158%2F0008-5472.CAN-16-2622"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit113R"><div class="casContent"><span class="casTitleNuber">113</span><div class="casTitle"><span class="NLM_cas:atitle">Targeted Degradation of BET Proteins in Triple-Negative Breast Cancer</span></div><div class="casAuthors">Bai, Longchuan; Zhou, Bing; Yang, Chao-Yie; Ji, Jiao; McEachern, Donna; Przybranowski, Sally; Jiang, Hui; Hu, Jiantao; Xu, Fuming; Zhao, Yujun; Liu, Liu; Fernandez-Salas, Ester; Xu, Jing; Dou, Yali; Wen, Bo; Sun, Duxin; Meagher, Jennifer; Stuckey, Jeanne; Hayes, Daniel F.; Li, Shunqiang; Ellis, Matthew J.; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">77</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2476-2487</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Triple-neg. breast cancers (TNBC) remain clin. challenging with a lack of options for targeted therapy.  In this study, we report the development of a second-generation BET protein degrader, BETd-246, which exhibits superior selectivity, potency, and antitumor activity.  In human TNBC cells, BETd-246 induced degrdn. of BET proteins at low nanomolar concns. within 1 h of exposure, resulting in robust growth inhibition and apoptosis.  BETd-246 was more potent and effective in TNBC cells than its parental BET inhibitor compd. BETi-211.  RNA-seq anal. revealed predominant downregulation of a large no. of genes involved in proliferation and apoptosis in cells treated with BETd-246, as compared with BETi-211 treatment that upregulated and downregulated a similar no. of genes.  Functional investigations identified the MCL1 gene as a crit. downstream effector for BET degraders, which synergized with small-mol. inhibitors of BCL-xL in triggering apoptosis.  In multiple murine xenograft models of human breast cancer, BETd-246 and a further optimized analog BETd-260 effectively depleted BET proteins in tumors and exhibited strong antitumor activities at well-tolerated dosing schedules.  Overall, our findings show that targeting BET proteins for degrdn. represents an effective therapeutic strategy for TNBC treatment.  Cancer Res; 77(9); 2476-87. ©2017 AACR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq8M1VaSNIBCLVg90H21EOLACvtfcHk0licpfjoyO29UQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXmvFakurg%253D&md5=84c0e0b23e61a5503f8d88385cc88ae5</span></div><a href="/servlet/linkout?suffix=cit113&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-16-2622&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-16-2622%26sid%3Dliteratum%253Aachs%26aulast%3DBai%26aufirst%3DL.%26aulast%3DZhou%26aufirst%3DB.%26aulast%3DYang%26aufirst%3DC.%2BY.%26aulast%3DJi%26aufirst%3DJ.%26aulast%3DMceachern%26aufirst%3DD.%26aulast%3DPrzybranowski%26aufirst%3DS.%26aulast%3DJiang%26aufirst%3DH.%26aulast%3DHu%26aufirst%3DJ.%26aulast%3DXu%26aufirst%3DF.%26aulast%3DZhao%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DFernandez-Salas%26aufirst%3DE.%26aulast%3DXu%26aufirst%3DJ.%26aulast%3DDou%26aufirst%3DY.%26aulast%3DWen%26aufirst%3DB.%26aulast%3DSun%26aufirst%3DD.%26aulast%3DMeagher%26aufirst%3DJ.%26aulast%3DStuckey%26aufirst%3DJ.%26aulast%3DHayes%26aufirst%3DD.%2BF.%26aulast%3DLi%26aufirst%3DS.%26aulast%3DEllis%26aufirst%3DM.%2BJ.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DTargeted%2520degradation%2520of%2520BET%2520proteins%2520in%2520triple-negative%2520breast%2520cancer%26jtitle%3DCancer%2520Res.%26date%3D2017%26volume%3D77%26spage%3D2476%26epage%3D2487%26doi%3D10.1158%2F0008-5472.CAN-16-2622" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref114"><div class="reference"><strong class="refLabel"><a href="#ref114" class="rightTabRefNumLink">114</a></strong><div class="NLM_citation" id="rightTab-cit114"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Qin, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fernandez-Salas, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mceachern, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Przybranowski, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stuckey, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meagher, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bai, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ziazadeh, D. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wen, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span> <span> </span><span class="NLM_article-title">Discovery of QCA570 as an exceptionally potent and efficacious proteolysis targeting chimera (PROTAC) degrader of the bromodomain and extra-terminal (BET)proteins capable of inducing complete and durable tumor regression</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">6685</span>– <span class="NLM_lpage">6704</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b00506</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b00506" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref114/cit114&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtlaqsLrE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=6685-6704&author=C.+Qinauthor=Y.+Huauthor=B.+Zhouauthor=E.+Fernandez-Salasauthor=C.+Y.+Yangauthor=L.+Liuauthor=D.+Mceachernauthor=S.+Przybranowskiauthor=M.+Wangauthor=J.+Stuckeyauthor=J.+Meagherauthor=L.+Baiauthor=Z.+Chenauthor=M.+Linauthor=J.+Yangauthor=D.+N.+Ziazadehauthor=F.+Xuauthor=J.+Huauthor=W.+Xiangauthor=L.+Huangauthor=S.+Liauthor=B.+Wenauthor=D.+Sunauthor=S.+Wang&title=Discovery+of+QCA570+as+an+exceptionally+potent+and+efficacious+proteolysis+targeting+chimera+%28PROTAC%29+degrader+of+the+bromodomain+and+extra-terminal+%28BET%29proteins+capable+of+inducing+complete+and+durable+tumor+regression&doi=10.1021%2Facs.jmedchem.8b00506"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit114R"><div class="casContent"><span class="casTitleNuber">114</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of QCA570 as an Exceptionally Potent and Efficacious Proteolysis Targeting Chimera (PROTAC) Degrader of the Bromodomain and Extra-Terminal (BET) Proteins Capable of Inducing Complete and Durable Tumor Regression</span></div><div class="casAuthors">Qin, Chong; Hu, Yang; Zhou, Bing; Fernandez-Salas, Ester; Yang, Chao-Yie; Liu, Liu; McEachern, Donna; Przybranowski, Sally; Wang, Mi; Stuckey, Jeanne; Meagher, Jennifer; Bai, Longchuan; Chen, Zhuo; Lin, Mei; Yang, Jiuling; Ziazadeh, Danya N.; Xu, Fuming; Hu, Jiantao; Xing, Weiguo; Huang, Liyue; Li, Siwei; Wen, Bo; Sun, Duxin; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">6685-6704</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Proteins of the bromodomain and extra-terminal (BET) family are epigenetics "readers" and promising therapeutic targets for cancer and other human diseases.  We describe herein a structure-guided design of [1,4]oxazepines as a new class of BET inhibitors and our subsequent design, synthesis, and evaluation of proteolysis-targeting chimeric (PROTAC) small-mol. BET degraders.  Our efforts have led to the discovery of extremely potent BET degraders, exemplified by QCA570, which effectively induces degrdn. of BET proteins and inhibits cell growth in human acute leukemia cell lines even at low picomolar concns.  QCA570 achieves complete and durable tumor regression in leukemia xenograft models in mice at well-tolerated dose-schedules.  QCA570 is the most potent and efficacious BET degrader reported to date.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo6ZUIsfX3p6LVg90H21EOLACvtfcHk0lhJLGCYVKlUrw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtlaqsLrE&md5=2a731730641372f4526543e03d9ac76f</span></div><a href="/servlet/linkout?suffix=cit114&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b00506&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b00506%26sid%3Dliteratum%253Aachs%26aulast%3DQin%26aufirst%3DC.%26aulast%3DHu%26aufirst%3DY.%26aulast%3DZhou%26aufirst%3DB.%26aulast%3DFernandez-Salas%26aufirst%3DE.%26aulast%3DYang%26aufirst%3DC.%2BY.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DMceachern%26aufirst%3DD.%26aulast%3DPrzybranowski%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DStuckey%26aufirst%3DJ.%26aulast%3DMeagher%26aufirst%3DJ.%26aulast%3DBai%26aufirst%3DL.%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DLin%26aufirst%3DM.%26aulast%3DYang%26aufirst%3DJ.%26aulast%3DZiazadeh%26aufirst%3DD.%2BN.%26aulast%3DXu%26aufirst%3DF.%26aulast%3DHu%26aufirst%3DJ.%26aulast%3DXiang%26aufirst%3DW.%26aulast%3DHuang%26aufirst%3DL.%26aulast%3DLi%26aufirst%3DS.%26aulast%3DWen%26aufirst%3DB.%26aulast%3DSun%26aufirst%3DD.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DDiscovery%2520of%2520QCA570%2520as%2520an%2520exceptionally%2520potent%2520and%2520efficacious%2520proteolysis%2520targeting%2520chimera%2520%2528PROTAC%2529%2520degrader%2520of%2520the%2520bromodomain%2520and%2520extra-terminal%2520%2528BET%2529proteins%2520capable%2520of%2520inducing%2520complete%2520and%2520durable%2520tumor%2520regression%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D6685%26epage%3D6704%26doi%3D10.1021%2Facs.jmedchem.8b00506" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref115"><div class="reference"><strong class="refLabel"><a href="#ref115" class="rightTabRefNumLink">115</a></strong><div class="NLM_citation" id="rightTab-cit115"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mi, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pei, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duan, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span> <span> </span><span class="NLM_article-title">In vivo target protein degradation induced by PROTACs based on E3 ligase DCAF15</span>. <i>Signal Transduction Targeted Ther.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">129</span>, <span class="refDoi"> DOI: 10.1038/s41392-020-00245-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref115/cit115&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1038%2Fs41392-020-00245-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref115/cit115&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=32713946" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref115/cit115&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhsVGlu77N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2020&pages=129&author=L.+Liauthor=D.+Miauthor=H.+Peiauthor=Q.+Duanauthor=X.+Wangauthor=W.+Zhouauthor=J.+Jinauthor=D.+Liauthor=M.+Liuauthor=Y.+Chen&title=In+vivo+target+protein+degradation+induced+by+PROTACs+based+on+E3+ligase+DCAF15&doi=10.1038%2Fs41392-020-00245-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit115R"><div class="casContent"><span class="casTitleNuber">115</span><div class="casTitle"><span class="NLM_cas:atitle">In vivo target protein degradation induced by PROTACs based on E3 ligase DCAF15</span></div><div class="casAuthors">Li, Liang; Mi, Dazhao; Pei, Haixiang; Duan, Qiuhui; Wang, Xinyue; Zhou, Wenbo; Jin, Jianping; Li, Dali; Liu, Mingyao; Chen, Yihua</div><div class="citationInfo"><span class="NLM_cas:title">Signal Transduction and Targeted Therapy</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">129</span>CODEN:
                <span class="NLM_cas:coden">STTTCB</span>;
        ISSN:<span class="NLM_cas:issn">2059-3635</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Proteolysis targeting chimeras (PROTACs) facilitate ubiquitination and degrdn. of the targets by cellular endogenous proteasome system.  To design PROTACs hijacking the E3 ligase complex CRL4-DCAF15, BRD4 was selected as the target protein since which is well-studied member of BET.  To investigate the mechanism of DP1-induced BRD4 degrdn., we explored whether DP1 induced degrdn. of BRD4 isdependent on binding to both BRD4 and DCAF15 and proteasome functions.  Collectively, our study provides a series of evidence that the DCAF15 can serve as a valuable addn. to the limited E3 ligases used in PROTACs design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqZi2tCZj8aWLVg90H21EOLACvtfcHk0lhJLGCYVKlUrw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhsVGlu77N&md5=a9b7410e4316998f21009da41db6f8c0</span></div><a href="/servlet/linkout?suffix=cit115&amp;dbid=16384&amp;doi=10.1038%2Fs41392-020-00245-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41392-020-00245-0%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DL.%26aulast%3DMi%26aufirst%3DD.%26aulast%3DPei%26aufirst%3DH.%26aulast%3DDuan%26aufirst%3DQ.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DZhou%26aufirst%3DW.%26aulast%3DJin%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DD.%26aulast%3DLiu%26aufirst%3DM.%26aulast%3DChen%26aufirst%3DY.%26atitle%3DIn%2520vivo%2520target%2520protein%2520degradation%2520induced%2520by%2520PROTACs%2520based%2520on%2520E3%2520ligase%2520DCAF15%26jtitle%3DSignal%2520Transduction%2520Targeted%2520Ther.%26date%3D2020%26volume%3D5%26spage%3D129%26doi%3D10.1038%2Fs41392-020-00245-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref116"><div class="reference"><strong class="refLabel"><a href="#ref116" class="rightTabRefNumLink">116</a></strong><div class="NLM_citation" id="rightTab-cit116"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nowak, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deangelo, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buckley, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Donovan, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">An, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Safaee, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jedrychowski, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ponthier, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ishoey, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mancias, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fischer, E. S.</span></span> <span> </span><span class="NLM_article-title">Plasticity in binding confers selectivity in ligand-induced protein degradation</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">706</span>– <span class="NLM_lpage">714</span>, <span class="refDoi"> DOI: 10.1038/s41589-018-0055-y</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref116/cit116&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1038%2Fs41589-018-0055-y" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref116/cit116&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=29892083" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref116/cit116&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtFWlurbK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2018&pages=706-714&author=R.+P.+Nowakauthor=S.+L.+Deangeloauthor=D.+Buckleyauthor=Z.+Heauthor=K.+A.+Donovanauthor=J.+Anauthor=N.+Safaeeauthor=M.+P.+Jedrychowskiauthor=C.+M.+Ponthierauthor=M.+Ishoeyauthor=T.+Zhangauthor=J.+D.+Manciasauthor=N.+S.+Grayauthor=J.+E.+Bradnerauthor=E.+S.+Fischer&title=Plasticity+in+binding+confers+selectivity+in+ligand-induced+protein+degradation&doi=10.1038%2Fs41589-018-0055-y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit116R"><div class="casContent"><span class="casTitleNuber">116</span><div class="casTitle"><span class="NLM_cas:atitle">Plasticity in binding confers selectivity in ligand-induced protein degradation</span></div><div class="casAuthors">Nowak, Radoslaw P.; De Angelo, Stephen L.; Buckley, Dennis; He, Zhixiang; Donovan, Katherine A.; An, Jian; Safaee, Nozhat; Jedrychowski, Mark P.; Ponthier, Charles M.; Ishoey, Mette; Zhang, Tinghu; Mancias, Joseph D.; Gray, Nathanael S.; Bradner, James E.; Fischer, Eric S.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">706-714</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Heterobifunctional small-mol. degraders that induce protein degrdn. through ligase-mediated ubiquitination have shown considerable promise as a new pharmacol. modality.  However, we currently lack a detailed understanding of the mol. basis for target recruitment and selectivity, which is critically required to enable rational design of degraders.  Here we utilized a comprehensive characterization of the ligand-dependent CRBN-BRD4 interaction to demonstrate that binding between proteins that have not evolved to interact is plastic.  Multiple x-ray crystal structures showed that plasticity resulted in several distinct low-energy binding conformations that were selectively bound by ligands.  We demonstrated that computational protein-protein docking can reveal the underlying interprotein contacts and inform the design of a BRD4 selective degrader that can discriminate between highly homologous BET bromodomains.  These findings that plastic interprotein contacts confer selectivity for ligand-induced protein dimerization provide a conceptual framework for the development of heterobifunctional ligands.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGreQasCW8bz2LVg90H21EOLACvtfcHk0lhJLGCYVKlUrw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtFWlurbK&md5=e4876c43b1eeb4558c23121b7a9fac27</span></div><a href="/servlet/linkout?suffix=cit116&amp;dbid=16384&amp;doi=10.1038%2Fs41589-018-0055-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41589-018-0055-y%26sid%3Dliteratum%253Aachs%26aulast%3DNowak%26aufirst%3DR.%2BP.%26aulast%3DDeangelo%26aufirst%3DS.%2BL.%26aulast%3DBuckley%26aufirst%3DD.%26aulast%3DHe%26aufirst%3DZ.%26aulast%3DDonovan%26aufirst%3DK.%2BA.%26aulast%3DAn%26aufirst%3DJ.%26aulast%3DSafaee%26aufirst%3DN.%26aulast%3DJedrychowski%26aufirst%3DM.%2BP.%26aulast%3DPonthier%26aufirst%3DC.%2BM.%26aulast%3DIshoey%26aufirst%3DM.%26aulast%3DZhang%26aufirst%3DT.%26aulast%3DMancias%26aufirst%3DJ.%2BD.%26aulast%3DGray%26aufirst%3DN.%2BS.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26aulast%3DFischer%26aufirst%3DE.%2BS.%26atitle%3DPlasticity%2520in%2520binding%2520confers%2520selectivity%2520in%2520ligand-induced%2520protein%2520degradation%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2018%26volume%3D14%26spage%3D706%26epage%3D714%26doi%3D10.1038%2Fs41589-018-0055-y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref117"><div class="reference"><strong class="refLabel"><a href="#ref117" class="rightTabRefNumLink">117</a></strong><div class="NLM_citation" id="rightTab-cit117"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gadd, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Testa, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lucas, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lamont, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zengerle, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span> <span> </span><span class="NLM_article-title">Structural basis of PROTAC cooperative recognition for selective protein degradation</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">514</span>– <span class="NLM_lpage">521</span>, <span class="refDoi"> DOI: 10.1038/nchembio.2329</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref117/cit117&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1038%2Fnchembio.2329" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref117/cit117&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=28288108" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref117/cit117&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BC2sXkvFahtbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2017&pages=514-521&author=M.+S.+Gaddauthor=A.+Testaauthor=X.+Lucasauthor=K.+H.+Chanauthor=W.+Chenauthor=D.+J.+Lamontauthor=M.+Zengerleauthor=A.+Ciulli&title=Structural+basis+of+PROTAC+cooperative+recognition+for+selective+protein+degradation&doi=10.1038%2Fnchembio.2329"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit117R"><div class="casContent"><span class="casTitleNuber">117</span><div class="casTitle"><span class="NLM_cas:atitle">Structural basis of PROTAC cooperative recognition for selective protein degradation</span></div><div class="casAuthors">Gadd, Morgan S.; Testa, Andrea; Lucas, Xavier; Chan, Kwok-Ho; Chen, Wenzhang; Lamont, Douglas J.; Zengerle, Michael; Ciulli, Alessio</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">514-521</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Inducing macromol. interactions with small mols. to activate cellular signaling is a challenging goal.  PROTACs (proteolysis-targeting chimeras) are bifunctional mols. that recruit a target protein in proximity to an E3 ubiquitin ligase to trigger protein degrdn.  Structural elucidation of the key ternary ligase-PROTAC-target species and its impact on target degrdn. selectivity remain elusive.  Here, we solved the crystal structure of Brd4 degrader MZ1 in complex with human VHL and the Brd4 bromodomain (Brd4BD2).  The ligand folds into itself to allow formation of specific intermol. interactions in the ternary complex.  Isothermal titrn. calorimetry studies, supported by surface mutagenesis and proximity assays, were consistent with pronounced cooperative formation of ternary complexes with Brd4BD2.  Structure-based-designed compd. AT1 exhibited highly selective depletion of Brd4 in cells.  These results elucidate how PROTAC-induced de novo contacts dictate preferential recruitment of a target protein into a stable and cooperative complex with an E3 ligase for selective degrdn.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoJri00jB5YJLVg90H21EOLACvtfcHk0liCgTCE-kySNQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXkvFahtbg%253D&md5=dfee13913106d362d0972d7b1a62d700</span></div><a href="/servlet/linkout?suffix=cit117&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.2329&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.2329%26sid%3Dliteratum%253Aachs%26aulast%3DGadd%26aufirst%3DM.%2BS.%26aulast%3DTesta%26aufirst%3DA.%26aulast%3DLucas%26aufirst%3DX.%26aulast%3DChan%26aufirst%3DK.%2BH.%26aulast%3DChen%26aufirst%3DW.%26aulast%3DLamont%26aufirst%3DD.%2BJ.%26aulast%3DZengerle%26aufirst%3DM.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DStructural%2520basis%2520of%2520PROTAC%2520cooperative%2520recognition%2520for%2520selective%2520protein%2520degradation%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2017%26volume%3D13%26spage%3D514%26epage%3D521%26doi%3D10.1038%2Fnchembio.2329" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref118"><div class="reference"><strong class="refLabel"><a href="#ref118" class="rightTabRefNumLink">118</a></strong><div class="NLM_citation" id="rightTab-cit118"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Raina, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qian, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Altieri, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rossi, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winkler, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crew, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coleman, K. G.</span></span> <span> </span><span class="NLM_article-title">PROTAC-induced BET protein degradation as a therapy for castration-resistant prostate cancer</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>113</i></span>,  <span class="NLM_fpage">7124</span>– <span class="NLM_lpage">7129</span>, <span class="refDoi"> DOI: 10.1073/pnas.1521738113</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref118/cit118&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1073%2Fpnas.1521738113" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref118/cit118&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=27274052" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref118/cit118&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BC28Xptlyqs7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=113&publication_year=2016&pages=7124-7129&author=K.+Rainaauthor=J.+Luauthor=Y.+Qianauthor=M.+Altieriauthor=D.+Gordonauthor=A.+M.+Rossiauthor=J.+Wangauthor=X.+Chenauthor=H.+Dongauthor=K.+Siuauthor=J.+D.+Winklerauthor=A.+P.+Crewauthor=C.+M.+Crewsauthor=K.+G.+Coleman&title=PROTAC-induced+BET+protein+degradation+as+a+therapy+for+castration-resistant+prostate+cancer&doi=10.1073%2Fpnas.1521738113"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit118R"><div class="casContent"><span class="casTitleNuber">118</span><div class="casTitle"><span class="NLM_cas:atitle">PROTAC-induced BET protein degradation as a therapy for castration-resistant prostate cancer</span></div><div class="casAuthors">Raina, Kanak; Lu, Jing; Qian, Yimin; Altieri, Martha; Gordon, Deborah; Rossi, Ann Marie K.; Wang, Jing; Chen, Xin; Dong, Hanqing; Siu, Kam; Winkler, James D.; Crew, Andrew P.; Crews, Craig M.; Coleman, Kevin G.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">113</span>
        (<span class="NLM_cas:issue">26</span>),
    <span class="NLM_cas:pages">7124-7129</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Prostate cancer has the second highest incidence among cancers in men worldwide and is the second leading cause of cancer deaths of men in the United States.  Although androgen deprivation can initially lead to remission, the disease often progresses to castration-resistant prostate cancer (CRPC), which is still reliant on androgen receptor (AR) signaling and is assocd. with a poor prognosis.  Some success against CRPC has been achieved by drugs that target AR signaling, but secondary resistance invariably emerges, and new therapies are urgently needed.  Recently, inhibitors of bromodomain and extra-terminal (BET) family proteins have shown growth-inhibitory activity in preclin. models of CRPC.  Here, we demonstrate that ARV-771, a small-mol. pan-BET degrader based on proteolysis-targeting chimera (PROTAC) technol., demonstrates dramatically improved efficacy in cellular models of CRPC as compared with BET inhibition.  Unlike BET inhibitors, ARV-771 results in suppression of both AR signaling and AR levels and leads to tumor regression in a CRPC mouse xenograft model.  This study is, to our knowledge, the first to demonstrate efficacy with a small-mol. BET degrader in a solid-tumor malignancy and potentially represents an important therapeutic advance in the treatment of CRPC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrD7QtYgZVig7Vg90H21EOLACvtfcHk0liCgTCE-kySNQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xptlyqs7s%253D&md5=34218e685ad72950f3eb85586110c43d</span></div><a href="/servlet/linkout?suffix=cit118&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1521738113&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1521738113%26sid%3Dliteratum%253Aachs%26aulast%3DRaina%26aufirst%3DK.%26aulast%3DLu%26aufirst%3DJ.%26aulast%3DQian%26aufirst%3DY.%26aulast%3DAltieri%26aufirst%3DM.%26aulast%3DGordon%26aufirst%3DD.%26aulast%3DRossi%26aufirst%3DA.%2BM.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DDong%26aufirst%3DH.%26aulast%3DSiu%26aufirst%3DK.%26aulast%3DWinkler%26aufirst%3DJ.%2BD.%26aulast%3DCrew%26aufirst%3DA.%2BP.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26aulast%3DColeman%26aufirst%3DK.%2BG.%26atitle%3DPROTAC-induced%2520BET%2520protein%2520degradation%2520as%2520a%2520therapy%2520for%2520castration-resistant%2520prostate%2520cancer%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2016%26volume%3D113%26spage%3D7124%26epage%3D7129%26doi%3D10.1073%2Fpnas.1521738113" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref119"><div class="reference"><strong class="refLabel"><a href="#ref119" class="rightTabRefNumLink">119</a></strong><div class="NLM_citation" id="rightTab-cit119"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sun, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fiskus, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qian, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rajapakshe, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raina, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coleman, K. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crew, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saenz, D. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mill, C. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nowak, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jain, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khoury, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coarfa, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhalla, K. N.</span></span> <span> </span><span class="NLM_article-title">BET protein proteolysis targeting chimera (PROTAC) exerts potent lethal activity against mantle cell lymphoma cells</span>. <i>Leukemia</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>32</i></span>,  <span class="NLM_fpage">343</span>– <span class="NLM_lpage">352</span>, <span class="refDoi"> DOI: 10.1038/leu.2017.207</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref119/cit119&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1038%2Fleu.2017.207" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref119/cit119&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=28663582" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref119/cit119&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BC2sXht1eqt77J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2018&pages=343-352&author=B.+Sunauthor=W.+Fiskusauthor=Y.+Qianauthor=K.+Rajapaksheauthor=K.+Rainaauthor=K.+G.+Colemanauthor=A.+P.+Crewauthor=A.+Shenauthor=D.+T.+Saenzauthor=C.+P.+Millauthor=A.+J.+Nowakauthor=N.+Jainauthor=L.+Zhangauthor=M.+Wangauthor=J.+D.+Khouryauthor=C.+Coarfaauthor=C.+M.+Crewsauthor=K.+N.+Bhalla&title=BET+protein+proteolysis+targeting+chimera+%28PROTAC%29+exerts+potent+lethal+activity+against+mantle+cell+lymphoma+cells&doi=10.1038%2Fleu.2017.207"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit119R"><div class="casContent"><span class="casTitleNuber">119</span><div class="casTitle"><span class="NLM_cas:atitle">Bet protein proteolysis targeting chimera (protac) exerts potent lethal activity against mantle cell lymphoma cells</span></div><div class="casAuthors">Sun, B.; Fiskus, W.; Qian, Y.; Rajapakshe, K.; Raina, K.; Coleman, K. G.; Crew, A. P.; Shen, A.; Saenz, D. T.; Mill, C. P.; Nowak, A. J.; Jain, N.; Zhang, L.; Wang, M.; Khoury, J. D.; Coarfa, C.; Crews, C. M.; Bhalla, K. N.</div><div class="citationInfo"><span class="NLM_cas:title">Leukemia</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">32</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">343-352</span>CODEN:
                <span class="NLM_cas:coden">LEUKED</span>;
        ISSN:<span class="NLM_cas:issn">0887-6924</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Bromodomain extraterminal protein (BETP) inhibitors transcriptionally repress oncoproteins and nuclear factor-κB (NF-κB) target genes that undermines the growth and survival of mantle cell lymphoma (MCL) cells.  However, BET bromodomain inhibitor (BETi) treatment causes accumulation of BETPs, assocd. with reversible binding and incomplete inhibition of BRD4 that potentially compromises the activity of BETi in MCL cells.  Unlike BETi, BET-PROTACs (proteolysis-targeting chimera) ARV-825 and ARV-771 (Arvinas, Inc.) recruit and utilize an E3-ubiquitin ligase to effectively degrade BETPs in MCL cells.  BET-PROTACs induce more apoptosis than BETi of MCL cells, including those resistant to ibrutinib.  BET-PROTAC treatment induced more perturbations in the mRNA and protein expressions than BETi, with depletion of c-Myc, CDK4, cyclin D1 and the NF-κB transcriptional targets Bcl-xL, XIAP and BTK, while inducing the levels of HEXIM1, NOXA and CDKN1A/p21.  Treatment with ARV-771, which possesses superior pharmacol. properties compared with ARV-825, inhibited the in vivo growth and induced greater survival improvement than the BETi OTX015 of immune-depleted mice engrafted with MCL cells.  Cotreatment of ARV-771 with ibrutinib or the BCL2 antagonist venetoclax or CDK4/6 inhibitor palbociclib synergistically induced apoptosis of MCL cells.  These studies highlight promising and superior preclin. activity of BET-PROTAC than BETi, requiring further in vivo evaluation of BET-PROTAC as a therapy for ibrutinib-sensitive or -resistant MCL.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAZxihoDZILrVg90H21EOLACvtfcHk0liCgTCE-kySNQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXht1eqt77J&md5=f771eee574daf11546b93866ea314381</span></div><a href="/servlet/linkout?suffix=cit119&amp;dbid=16384&amp;doi=10.1038%2Fleu.2017.207&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fleu.2017.207%26sid%3Dliteratum%253Aachs%26aulast%3DSun%26aufirst%3DB.%26aulast%3DFiskus%26aufirst%3DW.%26aulast%3DQian%26aufirst%3DY.%26aulast%3DRajapakshe%26aufirst%3DK.%26aulast%3DRaina%26aufirst%3DK.%26aulast%3DColeman%26aufirst%3DK.%2BG.%26aulast%3DCrew%26aufirst%3DA.%2BP.%26aulast%3DShen%26aufirst%3DA.%26aulast%3DSaenz%26aufirst%3DD.%2BT.%26aulast%3DMill%26aufirst%3DC.%2BP.%26aulast%3DNowak%26aufirst%3DA.%2BJ.%26aulast%3DJain%26aufirst%3DN.%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DKhoury%26aufirst%3DJ.%2BD.%26aulast%3DCoarfa%26aufirst%3DC.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26aulast%3DBhalla%26aufirst%3DK.%2BN.%26atitle%3DBET%2520protein%2520proteolysis%2520targeting%2520chimera%2520%2528PROTAC%2529%2520exerts%2520potent%2520lethal%2520activity%2520against%2520mantle%2520cell%2520lymphoma%2520cells%26jtitle%3DLeukemia%26date%3D2018%26volume%3D32%26spage%3D343%26epage%3D352%26doi%3D10.1038%2Fleu.2017.207" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref120"><div class="reference"><strong class="refLabel"><a href="#ref120" class="rightTabRefNumLink">120</a></strong><div class="NLM_citation" id="rightTab-cit120"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zengerle, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span> <span> </span><span class="NLM_article-title">Selective small molecule induced degradation of the BET bromodomain protein BRD4</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">1770</span>– <span class="NLM_lpage">1777</span>, <span class="refDoi"> DOI: 10.1021/acschembio.5b00216</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.5b00216" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref120/cit120&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BC2MXptlOgu70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=1770-1777&author=M.+Zengerleauthor=K.+H.+Chanauthor=A.+Ciulli&title=Selective+small+molecule+induced+degradation+of+the+BET+bromodomain+protein+BRD4&doi=10.1021%2Facschembio.5b00216"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit120R"><div class="casContent"><span class="casTitleNuber">120</span><div class="casTitle"><span class="NLM_cas:atitle">Selective Small Molecule Induced Degradation of the BET Bromodomain Protein BRD4</span></div><div class="casAuthors">Zengerle, Michael; Chan, Kwok-Ho; Ciulli, Alessio</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1770-1777</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The Bromo- and Extra-Terminal (BET) proteins BRD2, BRD3, and BRD4 play important roles in transcriptional regulation, epigenetics, and cancer and are the targets of pan-BET selective bromodomain inhibitor JQ1.  However, the lack of intra-BET selectivity limits the scope of current inhibitors as probes for target validation and could lead to unwanted side effects or toxicity in a therapeutic setting.  We designed Proteolysis Targeted Chimeras (PROTACs) that tether JQ1 to a ligand for the E3 ubiquitin ligase VHL, aimed at triggering the intracellular destruction of BET proteins.  Compd. MZ1 potently and rapidly induces reversible, long-lasting, and unexpectedly selective removal of BRD4 over BRD2 and BRD3.  The activity of MZ1 is dependent on binding to VHL but is achieved at a sufficiently low concn. not to induce stabilization of HIF-1α.  Gene expression profiles of selected cancer-related genes responsive to JQ1 reveal distinct and more limited transcriptional responses induced by MZ1, consistent with selective suppression of BRD4.  Our discovery opens up new opportunities to elucidate the cellular phenotypes and therapeutic implications assocd. with selective targeting of BRD4.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrEmSv7p9MZOrVg90H21EOLACvtfcHk0li7H0FznR1wSg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXptlOgu70%253D&md5=b7514fd274eff8e4bb6a771f5aaf9ab5</span></div><a href="/servlet/linkout?suffix=cit120&amp;dbid=16384&amp;doi=10.1021%2Facschembio.5b00216&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.5b00216%26sid%3Dliteratum%253Aachs%26aulast%3DZengerle%26aufirst%3DM.%26aulast%3DChan%26aufirst%3DK.%2BH.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DSelective%2520small%2520molecule%2520induced%2520degradation%2520of%2520the%2520BET%2520bromodomain%2520protein%2520BRD4%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2015%26volume%3D10%26spage%3D1770%26epage%3D1777%26doi%3D10.1021%2Facschembio.5b00216" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref121"><div class="reference"><strong class="refLabel"><a href="#ref121" class="rightTabRefNumLink">121</a></strong><div class="NLM_citation" id="rightTab-cit121"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Testa, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hughes, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lucas, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wright, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span> <span> </span><span class="NLM_article-title">Structure-based design of a macrocyclic PROTAC</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">1727</span>– <span class="NLM_lpage">1734</span>, <span class="refDoi"> DOI: 10.1002/anie.201914396</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref121/cit121&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1002%2Fanie.201914396" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref121/cit121&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitlyitbjK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2020&pages=1727-1734&author=A.+Testaauthor=S.+J.+Hughesauthor=X.+Lucasauthor=J.+E.+Wrightauthor=A.+Ciulli&title=Structure-based+design+of+a+macrocyclic+PROTAC&doi=10.1002%2Fanie.201914396"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit121R"><div class="casContent"><span class="casTitleNuber">121</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Based Design of a Macrocyclic PROTAC</span></div><div class="casAuthors">Testa, Andrea; Hughes, Scott J.; Lucas, Xavier; Wright, Jane E.; Ciulli, Alessio</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1727-1734</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Constraining a mol. in its bioactive conformation via macrocyclization represents an attractive strategy to rationally design functional chem. probes.  While this approach has been applied to enzyme inhibitors or receptor antagonists, to date it remains unprecedented for bifunctional mols. that bring proteins together, such as PROTAC degraders.  Herein, the authors report the design and synthesis of a macrocyclic PROTAC by adding a cyclizing linker to the BET degrader MZ1.  A cocrystal structure of macroPROTAC-1 bound in a ternary complex with VHL and the second bromodomain of Brd4 validated the rational design.  Biophys. studies revealed enhanced discrimination between the second and the first bromodomains of BET proteins.  Despite a 12-fold loss of binary binding affinity for Brd4, macroPROTAC-1 exhibited cellular activity comparable to MZ1.  The findings support macrocyclization as an advantageous strategy to enhance PROTAC degrdn. potency and selectivity between homologous targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqZjLgy_V3_abVg90H21EOLACvtfcHk0li7H0FznR1wSg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitlyitbjK&md5=6f3f279f6220ec30b30ca3d2c931da95</span></div><a href="/servlet/linkout?suffix=cit121&amp;dbid=16384&amp;doi=10.1002%2Fanie.201914396&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201914396%26sid%3Dliteratum%253Aachs%26aulast%3DTesta%26aufirst%3DA.%26aulast%3DHughes%26aufirst%3DS.%2BJ.%26aulast%3DLucas%26aufirst%3DX.%26aulast%3DWright%26aufirst%3DJ.%2BE.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DStructure-based%2520design%2520of%2520a%2520macrocyclic%2520PROTAC%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2020%26volume%3D59%26spage%3D1727%26epage%3D1734%26doi%3D10.1002%2Fanie.201914396" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref122"><div class="reference"><strong class="refLabel"><a href="#ref122" class="rightTabRefNumLink">122</a></strong><div class="NLM_citation" id="rightTab-cit122"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chan, K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zengerle, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Testa, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciulli, A.</span></span> <span> </span><span class="NLM_article-title">Impact of target warhead and linkage vector on inducing protein degradation: comparison of bromodomain and extra-terminal (BET) degraders derived from triazolodiazepine (JQ1) and tetrahydroquinoline (I-BET726) BET inhibitor scaffolds</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">504</span>– <span class="NLM_lpage">513</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b01912</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01912" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref122/cit122&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BC2sXpslSktL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=504-513&author=K.+H.+Chanauthor=M.+Zengerleauthor=A.+Testaauthor=A.+Ciulli&title=Impact+of+target+warhead+and+linkage+vector+on+inducing+protein+degradation%3A+comparison+of+bromodomain+and+extra-terminal+%28BET%29+degraders+derived+from+triazolodiazepine+%28JQ1%29+and+tetrahydroquinoline+%28I-BET726%29+BET+inhibitor+scaffolds&doi=10.1021%2Facs.jmedchem.6b01912"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit122R"><div class="casContent"><span class="casTitleNuber">122</span><div class="casTitle"><span class="NLM_cas:atitle">Impact of Target Warhead and Linkage Vector on Inducing Protein Degradation: Comparison of Bromodomain and Extra-Terminal (BET) Degraders Derived from Triazolodiazepine (JQ1) and Tetrahydroquinoline (I-BET726) BET Inhibitor Scaffolds</span></div><div class="casAuthors">Chan, Kwok-Ho; Zengerle, Michael; Testa, Andrea; Ciulli, Alessio</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">504-513</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The design of proteolysis-targeting chimeras (PROTACs) is a powerful small-mol. approach for inducing protein degrdn.  PROTACs conjugate a target warhead to an E3 ubiquitin ligase ligand via a linker.  Here the authors examd. the impact of derivatizing two different BET bromodomain inhibitors, triazolodiazepine JQ1 and the more potent tetrahydroquinoline I-BET726, via distinct exit vectors, using different polyethylene glycol linkers to VHL ligand VH032.  Triazolodiazepine PROTACs exhibited pos. cooperativities of ternary complex formation and were more potent degraders than tetrahydroquinoline compds., which showed neg. cooperativities instead.  Marked dependency on linker length was obsd. for BET-degrading and cMyc-driven antiproliferative activities in acute myeloid leukemia cell lines.  This work exemplifies as a cautionary tale how a more potent inhibitor does not necessarily generate more potent PROTACs and underscores the key roles played by the conjugation.  The provided insights and framework for structure-activity relationships of bivalent degraders are anticipated to have wide future applicability.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqvEL41TefwTbVg90H21EOLACvtfcHk0li7H0FznR1wSg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXpslSktL4%253D&md5=c3bfd9377d8e7805ba962cd0180431af</span></div><a href="/servlet/linkout?suffix=cit122&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01912&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01912%26sid%3Dliteratum%253Aachs%26aulast%3DChan%26aufirst%3DK.%2BH.%26aulast%3DZengerle%26aufirst%3DM.%26aulast%3DTesta%26aufirst%3DA.%26aulast%3DCiulli%26aufirst%3DA.%26atitle%3DImpact%2520of%2520target%2520warhead%2520and%2520linkage%2520vector%2520on%2520inducing%2520protein%2520degradation%253A%2520comparison%2520of%2520bromodomain%2520and%2520extra-terminal%2520%2528BET%2529%2520degraders%2520derived%2520from%2520triazolodiazepine%2520%2528JQ1%2529%2520and%2520tetrahydroquinoline%2520%2528I-BET726%2529%2520BET%2520inhibitor%2520scaffolds%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D504%26epage%3D513%26doi%3D10.1021%2Facs.jmedchem.6b01912" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref123"><div class="reference"><strong class="refLabel"><a href="#ref123" class="rightTabRefNumLink">123</a></strong><div class="NLM_citation" id="rightTab-cit123"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bondeson, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, B. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burslem, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buhimschi, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hines, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jaime-Figueroa, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamman, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ishchenko, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Lessons in PROTAC design from selective degradation with a promiscuous warhead</span>. <i>Cell Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">78</span>– <span class="NLM_lpage">87</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2017.09.010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref123/cit123&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1016%2Fj.chembiol.2017.09.010" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref123/cit123&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=29129718" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref123/cit123&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsl2lsrjN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2018&pages=78-87&author=D.+P.+Bondesonauthor=B.+E.+Smithauthor=G.+M.+Burslemauthor=A.+D.+Buhimschiauthor=J.+Hinesauthor=S.+Jaime-Figueroaauthor=J.+Wangauthor=B.+D.+Hammanauthor=A.+Ishchenkoauthor=C.+M.+Crews&title=Lessons+in+PROTAC+design+from+selective+degradation+with+a+promiscuous+warhead&doi=10.1016%2Fj.chembiol.2017.09.010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit123R"><div class="casContent"><span class="casTitleNuber">123</span><div class="casTitle"><span class="NLM_cas:atitle">Lessons in PROTAC Design from Selective Degradation with a Promiscuous Warhead</span></div><div class="casAuthors">Bondeson, Daniel P.; Smith, Blake E.; Burslem, George M.; Buhimschi, Alexandru D.; Hines, John; Jaime-Figueroa, Saul; Wang, Jing; Hamman, Brian D.; Ishchenko, Alexey; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">78-87.e5</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Inhibiting protein function selectively is a major goal of modern drug discovery.  Here, we report a previously understudied benefit of small mol. proteolysis-targeting chimeras (PROTACs) that recruit E3 ubiquitin ligases to target proteins for their ubiquitination and subsequent proteasome-mediated degrdn.  Using promiscuous CRBN- and VHL-recruiting PROTACs that bind >50 kinases, we show that only a subset of bound targets is degraded.  The basis of this selectivity relies on protein-protein interactions between the E3 ubiquitin ligase and the target protein, as illustrated by engaged proteins that are not degraded as a result of unstable ternary complexes with PROTAC-recruited E3 ligases.  In contrast, weak PROTAC:target protein affinity can be stabilized by high-affinity target:PROTAC:ligase trimer interactions, leading to efficient degrdn.  This study highlights design guidelines for generating potent PROTACs as well as possibilities for degrading undruggable proteins immune to traditional small-mol. inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoISOBudOLyFrVg90H21EOLACvtfcHk0ljUz1EOIM6HUg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsl2lsrjN&md5=30fac502705a2344c4261cf531e6bf9d</span></div><a href="/servlet/linkout?suffix=cit123&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2017.09.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2017.09.010%26sid%3Dliteratum%253Aachs%26aulast%3DBondeson%26aufirst%3DD.%2BP.%26aulast%3DSmith%26aufirst%3DB.%2BE.%26aulast%3DBurslem%26aufirst%3DG.%2BM.%26aulast%3DBuhimschi%26aufirst%3DA.%2BD.%26aulast%3DHines%26aufirst%3DJ.%26aulast%3DJaime-Figueroa%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DHamman%26aufirst%3DB.%2BD.%26aulast%3DIshchenko%26aufirst%3DA.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DLessons%2520in%2520PROTAC%2520design%2520from%2520selective%2520degradation%2520with%2520a%2520promiscuous%2520warhead%26jtitle%3DCell%2520Chem.%2520Biol.%26date%3D2018%26volume%3D25%26spage%3D78%26epage%3D87%26doi%3D10.1016%2Fj.chembiol.2017.09.010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref124"><div class="reference"><strong class="refLabel"><a href="#ref124" class="rightTabRefNumLink">124</a></strong><div class="NLM_citation" id="rightTab-cit124"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qin, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bai, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span> <span> </span><span class="NLM_article-title">Small-molecule PROTAC degraders of the Bromodomain and Extra Terminal (BET) proteins - A review</span>. <i>Drug Discovery Today: Technol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>31</i></span>,  <span class="NLM_fpage">43</span>– <span class="NLM_lpage">51</span>, <span class="refDoi"> DOI: 10.1016/j.ddtec.2019.04.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref124/cit124&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1016%2Fj.ddtec.2019.04.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref124/cit124&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=31200858" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref124/cit124&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A280%3ADC%252BB3M3mvFWrsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2019&pages=43-51&author=C.+Y.+Yangauthor=C.+Qinauthor=L.+Baiauthor=S.+Wang&title=Small-molecule+PROTAC+degraders+of+the+Bromodomain+and+Extra+Terminal+%28BET%29+proteins+-+A+review&doi=10.1016%2Fj.ddtec.2019.04.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit124R"><div class="casContent"><span class="casTitleNuber">124</span><div class="casTitle"><span class="NLM_cas:atitle">Small-molecule PROTAC degraders of the Bromodomain and Extra Terminal (BET) proteins - A review</span></div><div class="casAuthors">Yang Chao-Yie; Qin Chong; Bai Longchuan; Wang Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Drug discovery today. Technologies</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">43-51</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The PROteolysis TArgeting Chimeric (PROTAC) concept has provided an opportunity for the discovery and development of a completely new type of therapy involving induction of protein degradation.  The BET proteins, comprised of BRD2, BRD3, BRD4 and the testis-specific BRDT protein, are epigenetic readers and master transcription coactivators.  Extremely potent and efficacious small-molecule PROTAC degraders of the BET proteins, based on available, potent and selective BET inhibitors, have been reported.  BET degraders differ from BET inhibitors in their cellular potency, phenotypic effects, pharmacokinetic properties and toxicity profiles.  Herein, we provide a review of BET degraders and the differential outcome observed in the cellular and animal models for BET degraders in comparison to BET inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQCRHissLLXoQvupmecNw_lfW6udTcc2eaDZlIICXYNLLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3M3mvFWrsA%253D%253D&md5=de5790c1db28d56717ff76374072ad81</span></div><a href="/servlet/linkout?suffix=cit124&amp;dbid=16384&amp;doi=10.1016%2Fj.ddtec.2019.04.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ddtec.2019.04.001%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DC.%2BY.%26aulast%3DQin%26aufirst%3DC.%26aulast%3DBai%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DSmall-molecule%2520PROTAC%2520degraders%2520of%2520the%2520Bromodomain%2520and%2520Extra%2520Terminal%2520%2528BET%2529%2520proteins%2520-%2520A%2520review%26jtitle%3DDrug%2520Discovery%2520Today%253A%2520Technol.%26date%3D2019%26volume%3D31%26spage%3D43%26epage%3D51%26doi%3D10.1016%2Fj.ddtec.2019.04.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref125"><div class="reference"><strong class="refLabel"><a href="#ref125" class="rightTabRefNumLink">125</a></strong><div class="NLM_citation" id="rightTab-cit125"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Maneiro, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Forte, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shchepinova, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kounde, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chudasama, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baker, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tate, E. W.</span></span> <span> </span><span class="NLM_article-title">Antibody-PROTAC conjugates enable HER2-dependent targeted protein degradation of BRD4</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">1306</span>– <span class="NLM_lpage">1312</span>, <span class="refDoi"> DOI: 10.1021/acschembio.0c00285</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.0c00285" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref125/cit125&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BB3cXotFehsLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2020&pages=1306-1312&author=M.+A.+Maneiroauthor=N.+Forteauthor=M.+M.+Shchepinovaauthor=C.+S.+Koundeauthor=V.+Chudasamaauthor=J.+R.+Bakerauthor=E.+W.+Tate&title=Antibody-PROTAC+conjugates+enable+HER2-dependent+targeted+protein+degradation+of+BRD4&doi=10.1021%2Facschembio.0c00285"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit125R"><div class="casContent"><span class="casTitleNuber">125</span><div class="casTitle"><span class="NLM_cas:atitle">Antibody-PROTAC Conjugates Enable HER2-Dependent Targeted Protein Degradation of BRD4</span></div><div class="casAuthors">Maneiro, Maria; Forte, Nafsika; Shchepinova, Maria M.; Kounde, Cyrille S.; Chudasama, Vijay; Baker, James Richard; Tate, Edward W.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1306-1312</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Targeting protein degrdn. with Proteolysis-Targeting Chimeras (PROTACs) is an area of great current interest in drug discovery.  Nevertheless, although the high effectiveness of PROTACs against a wide variety of targets has been established, most degraders reported to date display limited intrinsic tissue selectivity and do not discriminate between cells of different types.  Here, we describe a strategy for selective protein degrdn. in a specific cell type.  We report the design and synthesis of a trastuzumab-PROTAC conjugate (Ab-PROTAC 3) in which E3 ligase-directed degrader activity is caged with an antibody linker which can be hydrolyzed following antibody-PROTAC internalization, releasing the active PROTAC and inducing catalytic protein degrdn.  We show that 3 selectively targets bromodomain-contg. protein 4 (BRD4) for degrdn. only in HER2 pos. breast cancer cell lines, while sparing HER2 neg. cells.  Using live cell confocal microscopy, we show internalization and lysosomal trafficking of the conjugate specifically in HER2 pos. cells, leading to the release of active PROTAC in quantities sufficient to induce potent BRD4 degrdn.  These studies demonstrate proof-of-concept for tissue-specific BRD4 degrdn., overcoming limitations of PROTAC selectivity, with significant potential for application to novel targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPwGNLaGmRbbVg90H21EOLACvtfcHk0ljG0SIJi2sKng"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXotFehsLo%253D&md5=544b53e54d3f0696f0b425c3f6e48e46</span></div><a href="/servlet/linkout?suffix=cit125&amp;dbid=16384&amp;doi=10.1021%2Facschembio.0c00285&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.0c00285%26sid%3Dliteratum%253Aachs%26aulast%3DManeiro%26aufirst%3DM.%2BA.%26aulast%3DForte%26aufirst%3DN.%26aulast%3DShchepinova%26aufirst%3DM.%2BM.%26aulast%3DKounde%26aufirst%3DC.%2BS.%26aulast%3DChudasama%26aufirst%3DV.%26aulast%3DBaker%26aufirst%3DJ.%2BR.%26aulast%3DTate%26aufirst%3DE.%2BW.%26atitle%3DAntibody-PROTAC%2520conjugates%2520enable%2520HER2-dependent%2520targeted%2520protein%2520degradation%2520of%2520BRD4%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2020%26volume%3D15%26spage%3D1306%26epage%3D1312%26doi%3D10.1021%2Facschembio.0c00285" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref126"><div class="reference"><strong class="refLabel"><a href="#ref126" class="rightTabRefNumLink">126</a></strong><div class="NLM_citation" id="rightTab-cit126"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Takahashi, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moriyama, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakamura, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miki, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takahashi, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sato, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Akaike, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Itto-Nakama, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arimoto, H.</span></span> <span> </span><span class="NLM_article-title">AUTACs: cargo-specific degraders using selective autophagy</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>76</i></span>,  <span class="NLM_fpage">797</span>– <span class="NLM_lpage">810</span>, <span class="refDoi"> DOI: 10.1016/j.molcel.2019.09.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref126/cit126&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1016%2Fj.molcel.2019.09.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref126/cit126&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=31606272" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref126/cit126&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhvFKjsr%252FE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=76&publication_year=2019&pages=797-810&author=D.+Takahashiauthor=J.+Moriyamaauthor=T.+Nakamuraauthor=E.+Mikiauthor=E.+Takahashiauthor=A.+Satoauthor=T.+Akaikeauthor=K.+Itto-Nakamaauthor=H.+Arimoto&title=AUTACs%3A+cargo-specific+degraders+using+selective+autophagy&doi=10.1016%2Fj.molcel.2019.09.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit126R"><div class="casContent"><span class="casTitleNuber">126</span><div class="casTitle"><span class="NLM_cas:atitle">AUTACs: Cargo-Specific Degraders Using Selective Autophagy</span></div><div class="casAuthors">Takahashi, Daiki; Moriyama, Jun; Nakamura, Tomoe; Miki, Erika; Takahashi, Eriko; Sato, Ayami; Akaike, Takaaki; Itto-Nakama, Kaori; Arimoto, Hirokazu</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">76</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">797-810.e10</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Protein silencing represents an essential tool in biomedical research.  Targeted protein degrdn. (TPD) strategies exemplified by PROTACs are rapidly emerging as modalities in drug discovery.  However, the scope of current TPD techniques is limited because many intracellular materials are not substrates of proteasomal clearance.  Here, we described a novel targeted-clearance strategy (autophagy-targeting chimera [AUTAC]) that contains a degrdn. tag (guanine derivs.) and a warhead to provide target specificity.  As expected from the substrate scope of autophagy, AUTAC degraded fragmented mitochondria as well as proteins.  Mitochondria-targeted AUTAC accelerated both the removal of dysfunctional fragmented mitochondria and the biogenesis of functionally normal mitochondria in patient-derived fibroblast cells.  Cytoprotective effects against acute mitochondrial injuries were also seen.  Canonical autophagy is viewed as a nonselective bulk decompn. system, and none of the available autophagy-inducing agents exhibit useful cargo selectivity.  With its target specificity, AUTAC provides a new modality for research on autophagy-based drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqEB3iYVVOYJbVg90H21EOLACvtfcHk0ljG0SIJi2sKng"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhvFKjsr%252FE&md5=2eb738f4a51d83e7aee6d69e47bf2e00</span></div><a href="/servlet/linkout?suffix=cit126&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2019.09.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2019.09.009%26sid%3Dliteratum%253Aachs%26aulast%3DTakahashi%26aufirst%3DD.%26aulast%3DMoriyama%26aufirst%3DJ.%26aulast%3DNakamura%26aufirst%3DT.%26aulast%3DMiki%26aufirst%3DE.%26aulast%3DTakahashi%26aufirst%3DE.%26aulast%3DSato%26aufirst%3DA.%26aulast%3DAkaike%26aufirst%3DT.%26aulast%3DItto-Nakama%26aufirst%3DK.%26aulast%3DArimoto%26aufirst%3DH.%26atitle%3DAUTACs%253A%2520cargo-specific%2520degraders%2520using%2520selective%2520autophagy%26jtitle%3DMol.%2520Cell%26date%3D2019%26volume%3D76%26spage%3D797%26epage%3D810%26doi%3D10.1016%2Fj.molcel.2019.09.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref127"><div class="reference"><strong class="refLabel"><a href="#ref127" class="rightTabRefNumLink">127</a></strong><div class="NLM_citation" id="rightTab-cit127"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ivanov, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khuri, F. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fu, H.</span></span> <span> </span><span class="NLM_article-title">Targeting protein-protein interactions as an anticancer strategy</span>. <i>Trends Pharmacol. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>34</i></span>,  <span class="NLM_fpage">393</span>– <span class="NLM_lpage">400</span>, <span class="refDoi"> DOI: 10.1016/j.tips.2013.04.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref127/cit127&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1016%2Fj.tips.2013.04.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref127/cit127&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=23725674" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref127/cit127&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BC3sXosVGjt7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=2013&pages=393-400&author=A.+A.+Ivanovauthor=F.+R.+Khuriauthor=H.+Fu&title=Targeting+protein-protein+interactions+as+an+anticancer+strategy&doi=10.1016%2Fj.tips.2013.04.007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit127R"><div class="casContent"><span class="casTitleNuber">127</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting protein-protein interactions as an anticancer strategy</span></div><div class="casAuthors">Ivanov, Andrei A.; Khuri, Fadlo R.; Fu, Haian</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Pharmacological Sciences</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">34</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">393-400</span>CODEN:
                <span class="NLM_cas:coden">TPHSDY</span>;
        ISSN:<span class="NLM_cas:issn">0165-6147</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  The emergence and convergence of cancer genomics, targeted therapies, and network oncol. have significantly expanded the landscape of protein-protein interaction (PPI) networks in cancer for therapeutic discovery.  Extensive biol. and clin. investigations have led to the identification of protein interaction hubs and nodes that are crit. for the acquisition and maintenance of characteristics of cancer essential for cell transformation.  Such cancer-enabling PPIs have become promising therapeutic targets.  With technol. advances in PPI modulator discovery and validation of PPI-targeting agents in clin. settings, targeting of PPI interfaces as an anticancer strategy has become a reality.  Future research directed at genomics-based PPI target discovery, PPI interface characterization, PPI-focused chem. library design, and patient-genomic subpopulation-driven clin. studies is expected to accelerate the development of the next generation of PPI-based anticancer agents for personalized precision medicine.  Here we briefly review prominent PPIs that mediate cancer-acquired properties, highlight recognized challenges and promising clin. results in targeting PPIs, and outline emerging opportunities.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqKeFT8Id1vhbVg90H21EOLACvtfcHk0ljG0SIJi2sKng"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXosVGjt7g%253D&md5=2f4772a4abfa6640b6af922671834fa3</span></div><a href="/servlet/linkout?suffix=cit127&amp;dbid=16384&amp;doi=10.1016%2Fj.tips.2013.04.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tips.2013.04.007%26sid%3Dliteratum%253Aachs%26aulast%3DIvanov%26aufirst%3DA.%2BA.%26aulast%3DKhuri%26aufirst%3DF.%2BR.%26aulast%3DFu%26aufirst%3DH.%26atitle%3DTargeting%2520protein-protein%2520interactions%2520as%2520an%2520anticancer%2520strategy%26jtitle%3DTrends%2520Pharmacol.%2520Sci.%26date%3D2013%26volume%3D34%26spage%3D393%26epage%3D400%26doi%3D10.1016%2Fj.tips.2013.04.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref128"><div class="reference"><strong class="refLabel"><a href="#ref128" class="rightTabRefNumLink">128</a></strong><div class="NLM_citation" id="rightTab-cit128"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cai, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, A. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiang, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kodadek, T.</span></span> <span> </span><span class="NLM_article-title">Peptoid ligands that bind selectively to phosphoproteins</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">4960</span>– <span class="NLM_lpage">4964</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2011.06.011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref128/cit128&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1016%2Fj.bmcl.2011.06.011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref128/cit128&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=21742492" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref128/cit128&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtVaisL%252FJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2011&pages=4960-4964&author=D.+Caiauthor=A.+Y.+Leeauthor=C.+M.+Chiangauthor=T.+Kodadek&title=Peptoid+ligands+that+bind+selectively+to+phosphoproteins&doi=10.1016%2Fj.bmcl.2011.06.011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit128R"><div class="casContent"><span class="casTitleNuber">128</span><div class="casTitle"><span class="NLM_cas:atitle">Peptoid ligands that bind selectively to phosphoproteins</span></div><div class="casAuthors">Cai, Di; Lee, A.-Young; Chiang, Cheng-Ming; Kodadek, Thomas</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">4960-4964</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Synthetic equiv. of phosphoprotein-specific antibodies would be valuable reagents for biol. research, since these antibodies can often be difficult to produce.  Protein phosphorylation is thought to result in significant conformational changes in most substrate proteins.  Therefore, one approach might be to simply screen combinatorial libraries for ligands to the phosphorylated state in the hope of isolating a ligand that binds to a pocket created by the conformational shift.  In this study, the authors probe this strategy by screening a peptoid library for ligands to the phosphorylated form of the Brd4 chromatin adaptor and transcriptional coactivator protein.  The authors find that peptoids with high selectivity for binding to the phosphorylation form of Brd4 can indeed be isolated in this screen.  Moreover, these ligands do not bind promiscuously to other phospho-proteins.  However, attempts to employ these reagents as antibody substitutes in an immunoaffinity purifn.-like application showed that they do not perform as well as bona fide antibodies and that significant optimization will be required.  This study highlights the potential and current limitations of a naive library screening strategy for phosphoprotein-specific antibody surrogates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqf0autNVBtLLVg90H21EOLACvtfcHk0ljG0SIJi2sKng"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtVaisL%252FJ&md5=d91d1c1daa186fd809289d528760ce79</span></div><a href="/servlet/linkout?suffix=cit128&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2011.06.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2011.06.011%26sid%3Dliteratum%253Aachs%26aulast%3DCai%26aufirst%3DD.%26aulast%3DLee%26aufirst%3DA.%2BY.%26aulast%3DChiang%26aufirst%3DC.%2BM.%26aulast%3DKodadek%26aufirst%3DT.%26atitle%3DPeptoid%2520ligands%2520that%2520bind%2520selectively%2520to%2520phosphoproteins%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2011%26volume%3D21%26spage%3D4960%26epage%3D4964%26doi%3D10.1016%2Fj.bmcl.2011.06.011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref129"><div class="reference"><strong class="refLabel"><a href="#ref129" class="rightTabRefNumLink">129</a></strong><div class="NLM_citation" id="rightTab-cit129"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhong, H.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leung, K.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, C. C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peng, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, S.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, D.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">David Wang, H.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leung, C.-H.</span></span> <span> </span><span class="NLM_article-title">An iridium(iii)-based irreversible protein-protein interaction inhibitor of BRD4 as a potent anticancer agent</span>. <i>Chem. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">5400</span>– <span class="NLM_lpage">5408</span>, <span class="refDoi"> DOI: 10.1039/C5SC02321A</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref129/cit129&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1039%2FC5SC02321A" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref129/cit129&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=28757943" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref129/cit129&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht1Grtr7E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=5400-5408&author=H.-J.+Zhongauthor=L.+Luauthor=K.-H.+Leungauthor=C.+C.+L.+Wongauthor=C.+Pengauthor=S.-C.+Yanauthor=D.-L.+Maauthor=Z.+Caiauthor=H.-M.+David+Wangauthor=C.-H.+Leung&title=An+iridium%28iii%29-based+irreversible+protein-protein+interaction+inhibitor+of+BRD4+as+a+potent+anticancer+agent&doi=10.1039%2FC5SC02321A"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit129R"><div class="casContent"><span class="casTitleNuber">129</span><div class="casTitle"><span class="NLM_cas:atitle">An iridium(III)-based irreversible protein-protein interaction inhibitor of BRD4 as a potent anticancer agent</span></div><div class="casAuthors">Zhong, Hai-Jing; Lu, Lihua; Leung, Ka-Ho; Wong, Catherine C. L.; Peng, Chao; Yan, Siu-Cheong; Ma, Dik-Lung; Cai, Zongwei; David Wang, Hui-Min; Leung, Chung-Hang</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Science</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">5400-5408</span>CODEN:
                <span class="NLM_cas:coden">CSHCCN</span>;
        ISSN:<span class="NLM_cas:issn">2041-6520</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Bromodomain-contg. protein 4 (BRD4) has recently emerged as an attractive epigenetic target for anticancer therapy.  In this study, an iridium(III) complex is reported as the first metal-based, irreversible inhibitor of BRD4.  Complex 1a is able to antagonize the BRD4-acetylated histone protein-protein interaction (PPI) in vitro, and to bind BRD4 and down-regulate c-myc oncogenic expression in cellulo.  Chromatin immunopptn. (ChIP) anal. revealed that 1a could modulate the interaction between BRD4 and chromatin in melanoma cells, particular at the MYC promoter.  Finally, the complex showed potent activity against melanoma xenografts in an in vivo mouse model.  To our knowledge, this is the first report of a Group 9 metal complex inhibiting the PPI of a member of the bromodomain and extraterminal domain (BET) family.  We envision that complex 1a may serve as a useful scaffold for the development of more potent epigenetic agents against cancers such as melanoma.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpqMXspNfXFlbVg90H21EOLACvtfcHk0ljSxWpdJHhk9w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXht1Grtr7E&md5=ae176cbfd3621dd7939c234d643db84e</span></div><a href="/servlet/linkout?suffix=cit129&amp;dbid=16384&amp;doi=10.1039%2FC5SC02321A&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC5SC02321A%26sid%3Dliteratum%253Aachs%26aulast%3DZhong%26aufirst%3DH.-J.%26aulast%3DLu%26aufirst%3DL.%26aulast%3DLeung%26aufirst%3DK.-H.%26aulast%3DWong%26aufirst%3DC.%2BC.%2BL.%26aulast%3DPeng%26aufirst%3DC.%26aulast%3DYan%26aufirst%3DS.-C.%26aulast%3DMa%26aufirst%3DD.-L.%26aulast%3DCai%26aufirst%3DZ.%26aulast%3DDavid%2BWang%26aufirst%3DH.-M.%26aulast%3DLeung%26aufirst%3DC.-H.%26atitle%3DAn%2520iridium%2528iii%2529-based%2520irreversible%2520protein-protein%2520interaction%2520inhibitor%2520of%2520BRD4%2520as%2520a%2520potent%2520anticancer%2520agent%26jtitle%3DChem.%2520Sci.%26date%3D2015%26volume%3D6%26spage%3D5400%26epage%3D5408%26doi%3D10.1039%2FC5SC02321A" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref130"><div class="reference"><strong class="refLabel"><a href="#ref130" class="rightTabRefNumLink">130</a></strong><div class="NLM_citation" id="rightTab-cit130"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Leung, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, Q. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, D. L.</span></span> <span> </span><span class="NLM_article-title">Emerging screening approaches in the development of Nrf2-Keap1 protein-protein interaction inhibitors</span>. <i>Int. J. Mol. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">4445</span>, <span class="refDoi"> DOI: 10.3390/ijms20184445</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref130/cit130&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.3390%2Fijms20184445" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref130/cit130&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BB3cXovFSgsbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2019&pages=4445&author=C.+H.+Leungauthor=J.+T.+Zhangauthor=G.+J.+Yangauthor=H.+Liuauthor=Q.+B.+Hanauthor=D.+L.+Ma&title=Emerging+screening+approaches+in+the+development+of+Nrf2-Keap1+protein-protein+interaction+inhibitors&doi=10.3390%2Fijms20184445"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit130R"><div class="casContent"><span class="casTitleNuber">130</span><div class="casTitle"><span class="NLM_cas:atitle">Emerging screening approaches in the development of Nrf2-Keap1 protein-protein interaction inhibitors</span></div><div class="casAuthors">Leung, Chung-Hang; Zhang, Jia-Tong; Yang, Guan-Jun; Liu, Hao; Han, Quan-Bin; Ma, Dik-Lung</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Molecular Sciences</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">4445</span>CODEN:
                <span class="NLM_cas:coden">IJMCFK</span>;
        ISSN:<span class="NLM_cas:issn">1422-0067</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">A review.  Due to role of the Keap1-Nrf2 protein-protein interaction (PPI) in protecting cells from oxidative stress, the development of small mol. inhibitors that inhibit this interaction has arisen as a viable approach to combat maladies caused by oxidative stress, such as cancers, neurodegenerative disease and diabetes.  To obtain specific and genuine Keap1-Nrf2 inhibitors, many efforts have been made towards developing new screening approaches.  However, there is no inhibitor for this target entering the clinic for the treatment of human diseases.  New strategies to identify novel bioactive compds. from large mol. databases and accelerate the developmental process of the clin. application of Keap1-Nrf2 protein-protein interaction inhibitors are greatly needed.  In this review, we have summarized virtual screening and other methods for discovering new lead compds. against the Keap1-Nrf2 protein-protein interaction.  We also discuss the advantages and limitations of different strategies, and the potential of this PPI as a drug target in disease therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqjIj82HYmo1bVg90H21EOLACvtfcHk0ljSxWpdJHhk9w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXovFSgsbo%253D&md5=bf4681884449ccb168a226d399b21747</span></div><a href="/servlet/linkout?suffix=cit130&amp;dbid=16384&amp;doi=10.3390%2Fijms20184445&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fijms20184445%26sid%3Dliteratum%253Aachs%26aulast%3DLeung%26aufirst%3DC.%2BH.%26aulast%3DZhang%26aufirst%3DJ.%2BT.%26aulast%3DYang%26aufirst%3DG.%2BJ.%26aulast%3DLiu%26aufirst%3DH.%26aulast%3DHan%26aufirst%3DQ.%2BB.%26aulast%3DMa%26aufirst%3DD.%2BL.%26atitle%3DEmerging%2520screening%2520approaches%2520in%2520the%2520development%2520of%2520Nrf2-Keap1%2520protein-protein%2520interaction%2520inhibitors%26jtitle%3DInt.%2520J.%2520Mol.%2520Sci.%26date%3D2019%26volume%3D20%26spage%3D4445%26doi%3D10.3390%2Fijms20184445" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref131"><div class="reference"><strong class="refLabel"><a href="#ref131" class="rightTabRefNumLink">131</a></strong><div class="NLM_citation" id="rightTab-cit131"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Padmanabhan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alexanian, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Linares-Saldana, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gonzalez-Teran, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andreoletti, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Connolly, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hsu, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duan, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winchester, S. A.B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Felix, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perez-Bermejo, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shah, P. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haldar, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jain, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Srivastava, D.</span></span> <span> </span><span class="NLM_article-title">Srivastava D. BRD4 interacts with GATA4 to govern mitochondrial homeostasis in adult cardiomyocytes</span>. <i>Circulation</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>142</i></span>,  <span class="NLM_fpage">2338</span>– <span class="NLM_lpage">2355</span>, <span class="refDoi"> DOI: 10.1161/CIRCULATIONAHA.120.047753</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref131/cit131&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1161%2FCIRCULATIONAHA.120.047753" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref131/cit131&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=33094644" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref131/cit131&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BB3MXms1Snuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=142&publication_year=2020&pages=2338-2355&author=A.+Padmanabhanauthor=M.+Alexanianauthor=R.+Linares-Saldanaauthor=B.+Gonzalez-Teranauthor=G.+Andreolettiauthor=Y.+Huangauthor=A.+J.+Connollyauthor=W.+Kimauthor=A.+Hsuauthor=Q.+Duanauthor=S.+A.B.+Winchesterauthor=F.+Felixauthor=J.+A.+Perez-Bermejoauthor=Q.+Wangauthor=L.+Liauthor=P.+P.+Shahauthor=S.+M.+Haldarauthor=R.+Jainauthor=D.+Srivastava&title=Srivastava+D.+BRD4+interacts+with+GATA4+to+govern+mitochondrial+homeostasis+in+adult+cardiomyocytes&doi=10.1161%2FCIRCULATIONAHA.120.047753"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit131R"><div class="casContent"><span class="casTitleNuber">131</span><div class="casTitle"><span class="NLM_cas:atitle">BRD4 (bromodomain-containing protein 4) interacts with GATA4 (GATA binding protein 4) to govern mitochondrial homeostasis in adult cardiomyocytes</span></div><div class="casAuthors">Padmanabhan, Arun; Alexanian, Michael; Linares-Saldana, Ricardo; Gonzalez-Teran, Barbara; Andreoletti, Gaia; Huang, Yu; Connolly, Andrew J.; Kim, Wonho; Hsu, Austin; Duan, Qiming; Winchester, Sarah A. B.; Felix, Franco; Perez-Bermejo, Juan A.; Wang, Qiaohong; Li, Li; Shah, Parisha P.; Haldar, Saptarsi M.; Jain, Rajan; Srivastava, Deepak</div><div class="citationInfo"><span class="NLM_cas:title">Circulation</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">142</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">2338-2355</span>CODEN:
                <span class="NLM_cas:coden">CIRCAZ</span>;
        ISSN:<span class="NLM_cas:issn">0009-7322</span>.
    
            (<span class="NLM_cas:orgname">Lippincott Williams & Wilkins</span>)
        </div><div class="casAbstract">Gene regulatory networks control tissue homeostasis and disease progression in a cell type-specific manner.  Ubiquitously expressed chromatin regulators modulate these networks, yet the mechanisms governing how tissue specificity of their function is achieved are poorly understood.  BRD4 (bromodomain-contg. protein 4), a member of the BET (bromo- and extraterminal domain) family of ubiquitously expressed acetyl-lysine reader proteins, plays a pivotal role as a coactivator of enhancer signaling across diverse tissue types in both health and disease and has been implicated as a pharmacol. target in heart failure.  However, the cell-specific role of BRD4 in adult cardiomyocytes remains unknown.  We combined conditional mouse genetics, unbiased transcriptomic and epigenomic analyses, and classic mol. biol. and biochem. approaches to understand the mechanism by which BRD4 in adult cardiomyocyte homeostasis.  Here, we show that cardiomyocyte-specific deletion of Brd4 in adult mice leads to acute deterioration of cardiac contractile function with mutant animals demonstrating a transcriptomic signature characterized by decreased expression of genes crit. for mitochondrial energy prodn.  Genome-wide occupancy data show that BRD4 enriches at many downregulated genes (including the master coactivators Ppargc1a, Ppargc1b, and their downstream targets) and preferentially colocalizes with GATA4 (GATA binding protein 4), a lineage-detg. cardiac transcription factor not previously implicated in regulation of adult cardiac metab.  BRD4 and GATA4 form an endogenous complex in cardiomyocytes and interact in a bromodomain-independent manner, revealing a new functional interaction partner for BRD4 that can direct its locus and tissue specificity.  These results highlight a novel role for a BRD4-GATA4 module in cooperative regulation of a cardiomyocyte-specific gene program governing bioenergetic homeostasis in the adult heart.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqdl4WwdZyHlbVg90H21EOLACvtfcHk0lgb4Ds64-ZzZA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3MXms1Snuw%253D%253D&md5=f3e0c0725b70e82f10a9fbfd78cf03b1</span></div><a href="/servlet/linkout?suffix=cit131&amp;dbid=16384&amp;doi=10.1161%2FCIRCULATIONAHA.120.047753&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1161%252FCIRCULATIONAHA.120.047753%26sid%3Dliteratum%253Aachs%26aulast%3DPadmanabhan%26aufirst%3DA.%26aulast%3DAlexanian%26aufirst%3DM.%26aulast%3DLinares-Saldana%26aufirst%3DR.%26aulast%3DGonzalez-Teran%26aufirst%3DB.%26aulast%3DAndreoletti%26aufirst%3DG.%26aulast%3DHuang%26aufirst%3DY.%26aulast%3DConnolly%26aufirst%3DA.%2BJ.%26aulast%3DKim%26aufirst%3DW.%26aulast%3DHsu%26aufirst%3DA.%26aulast%3DDuan%26aufirst%3DQ.%26aulast%3DWinchester%26aufirst%3DS.%2BA.B.%26aulast%3DFelix%26aufirst%3DF.%26aulast%3DPerez-Bermejo%26aufirst%3DJ.%2BA.%26aulast%3DWang%26aufirst%3DQ.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DShah%26aufirst%3DP.%2BP.%26aulast%3DHaldar%26aufirst%3DS.%2BM.%26aulast%3DJain%26aufirst%3DR.%26aulast%3DSrivastava%26aufirst%3DD.%26atitle%3DSrivastava%2520D.%2520BRD4%2520interacts%2520with%2520GATA4%2520to%2520govern%2520mitochondrial%2520homeostasis%2520in%2520adult%2520cardiomyocytes%26jtitle%3DCirculation%26date%3D2020%26volume%3D142%26spage%3D2338%26epage%3D2355%26doi%3D10.1161%2FCIRCULATIONAHA.120.047753" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref132"><div class="reference"><strong class="refLabel"><a href="#ref132" class="rightTabRefNumLink">132</a></strong><div class="NLM_citation" id="rightTab-cit132"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kulikowski, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rakai, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, N.</span></span> <span> </span><span class="NLM_article-title">Inhibitors of bromodomain and extra-terminal proteins for treating multiple human diseases</span>. <i>Med. Res. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2021</span>,  <span class="NLM_volume"><i>41</i></span>,  <span class="NLM_fpage">223</span>– <span class="NLM_lpage">245</span>, <span class="refDoi"> DOI: 10.1002/med.21730</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref132/cit132&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1002%2Fmed.21730" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref132/cit132&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=32926459" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref132/cit132&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=1%3ACAS%3A528%3ADC%252BB3MXhsVagtro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2021&pages=223-245&author=E.+Kulikowskiauthor=B.+D.+Rakaiauthor=N.+Wong&title=Inhibitors+of+bromodomain+and+extra-terminal+proteins+for+treating+multiple+human+diseases&doi=10.1002%2Fmed.21730"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit132R"><div class="casContent"><span class="casTitleNuber">132</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibitors of bromodomain and extra-terminal proteins for treating multiple human diseases</span></div><div class="casAuthors">Kulikowski, Ewelina; Rakai, Brooke D.; Wong, Norman C. W.</div><div class="citationInfo"><span class="NLM_cas:title">Medicinal Research Reviews</span>
        (<span class="NLM_cas:date">2021</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">223-245</span>CODEN:
                <span class="NLM_cas:coden">MRREDD</span>;
        ISSN:<span class="NLM_cas:issn">0198-6325</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">Clin. development of bromodomain and extra-terminal (BET) protein inhibitors differs from the traditional course of drug development.  These drugs are simultaneously being evaluated for treating a wide spectrum of human diseases due to their novel mechanism of action.  BET proteins are epigenetic "readers," which play a primary role in transcription.  Here, we briefly describe the BET family of proteins, of which BRD4 has been studied most extensively.  We discuss BRD4 activity at latent enhancers as an example of BET protein function.  We examine BRD4 redistribution and enhancer reprogramming in embryonic development, cancer, cardiovascular, autoimmune, and metabolic diseases, presenting hallmark studies that highlight BET proteins as attractive targets for therapeutic intervention.  We review the currently available approaches to targeting BET proteins, methods of selectively targeting individual bromodomains, and review studies that compare the effects of selective BET inhibition to those of pan-BET inhibition.  Lastly, we examine the current clin. landscape of BET inhibitor development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqiAZK3p5-347Vg90H21EOLACvtfcHk0lgb4Ds64-ZzZA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3MXhsVagtro%253D&md5=a63df8a1142c18a4c3e02d4567ded365</span></div><a href="/servlet/linkout?suffix=cit132&amp;dbid=16384&amp;doi=10.1002%2Fmed.21730&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fmed.21730%26sid%3Dliteratum%253Aachs%26aulast%3DKulikowski%26aufirst%3DE.%26aulast%3DRakai%26aufirst%3DB.%2BD.%26aulast%3DWong%26aufirst%3DN.%26atitle%3DInhibitors%2520of%2520bromodomain%2520and%2520extra-terminal%2520proteins%2520for%2520treating%2520multiple%2520human%2520diseases%26jtitle%3DMed.%2520Res.%2520Rev.%26date%3D2021%26volume%3D41%26spage%3D223%26epage%3D245%26doi%3D10.1002%2Fmed.21730" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref133"><div class="reference"><strong class="refLabel"><a href="#ref133" class="rightTabRefNumLink">133</a></strong><div class="NLM_citation" id="rightTab-cit133"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Feig, M.</span></span> <span> </span><span class="NLM_article-title">Computational protein structure refinement: almost there, yet still so far to go</span>. <i>Wiley Interdiscip Rev. Comput. Mol. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">e1307</span>, <span class="refDoi"> DOI: 10.1002/wcms.1307</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref133/cit133&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1002%2Fwcms.1307" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref133/cit133&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=30613211" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2017&pages=e1307&author=M.+Feig&title=Computational+protein+structure+refinement%3A+almost+there%2C+yet+still+so+far+to+go&doi=10.1002%2Fwcms.1307"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit133&amp;dbid=16384&amp;doi=10.1002%2Fwcms.1307&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fwcms.1307%26sid%3Dliteratum%253Aachs%26aulast%3DFeig%26aufirst%3DM.%26atitle%3DComputational%2520protein%2520structure%2520refinement%253A%2520almost%2520there%252C%2520yet%2520still%2520so%2520far%2520to%2520go%26jtitle%3DWiley%2520Interdiscip%2520Rev.%2520Comput.%2520Mol.%2520Sci.%26date%3D2017%26volume%3D7%26spage%3De1307%26doi%3D10.1002%2Fwcms.1307" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref134"><div class="reference"><strong class="refLabel"><a href="#ref134" class="rightTabRefNumLink">134</a></strong><div class="NLM_citation" id="rightTab-cit134"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cerutti, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Case, D. A.</span></span> <span> </span><span class="NLM_article-title">Molecular dynamics simulations of macromolecular crystals</span>. <i>Wiley Interdiscip. Rev.: Comput. Mol. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">e1402</span>, <span class="refDoi"> DOI: 10.1002/wcms.1402</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref134/cit134&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=10.1002%2Fwcms.1402" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref134/cit134&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;key=31662799" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2019&pages=e1402&author=D.+S.+Ceruttiauthor=D.+A.+Case&title=Molecular+dynamics+simulations+of+macromolecular+crystals&doi=10.1002%2Fwcms.1402"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit134&amp;dbid=16384&amp;doi=10.1002%2Fwcms.1402&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fwcms.1402%26sid%3Dliteratum%253Aachs%26aulast%3DCerutti%26aufirst%3DD.%2BS.%26aulast%3DCase%26aufirst%3DD.%2BA.%26atitle%3DMolecular%2520dynamics%2520simulations%2520of%2520macromolecular%2520crystals%26jtitle%3DWiley%2520Interdiscip.%2520Rev.%253A%2520Comput.%2520Mol.%2520Sci.%26date%3D2019%26volume%3D9%26spage%3De1402%26doi%3D10.1002%2Fwcms.1402" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.0c01487&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.0c01487%3Bctype%3Astring%3AJournal+Content%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2021.64.issue-5%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.0c01487" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                5MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"6799237e48bfd952","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
