
<html lang="en"     class="pb-page"  data-request-id="1c86e803-2114-412e-8a82-565d9ff1f301"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;article:article:10.1021/jm800791f;wgroup:string:ACHS website Group;issue:issue:10.1021/jmcmar.2009.52.issue-4;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="dc.Title" content="Directed Discovery of Agents Targeting the Met Tyrosine Kinase Domain by Virtual Screening" /></meta><meta name="dc.Creator" content="Megan L.  Peach" /></meta><meta name="dc.Creator" content="Nelly  Tan" /></meta><meta name="dc.Creator" content="Sarah J.  Choyke" /></meta><meta name="dc.Creator" content="Alessio  Giubellino" /></meta><meta name="dc.Creator" content="Gagani  Athauda" /></meta><meta name="dc.Creator" content="Terrence R.  Burke, Jr." /></meta><meta name="dc.Creator" content="Marc C.  Nicklaus" /></meta><meta name="dc.Creator" content="Donald P.  Bottaro" /></meta><meta name="dc.Description" content="Hepatocyte growth factor (HGF) is an important regulator of normal development and homeostasis, and dysregulated signaling through the HGF receptor, Met, contributes to tumorigenesis, tumor progres..." /></meta><meta name="Description" content="Hepatocyte growth factor (HGF) is an important regulator of normal development and homeostasis, and dysregulated signaling through the HGF receptor, Met, contributes to tumorigenesis, tumor progres..." /></meta><meta name="dc.Publisher" content=" American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="January 15, 2009" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/jm800791f" /></meta><meta name="dc.Language" content="EN" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="Â© 2009 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Fjm800791f" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Fjm800791f" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Fjm800791f" /></link>
        
    
    

<title>Directed Discovery of Agents Targeting the Met Tyrosine Kinase Domain by Virtual Screening | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/jm800791f" /></meta><meta property="og:title" content="Directed Discovery of Agents Targeting the Met Tyrosine Kinase Domain by Virtual Screening" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-4/jm800791f/production/images/large/jm-2008-00791f_0005.jpeg" /></meta><meta property="og:description" content=" Hepatocyte growth factor (HGF) is an important regulator of normal development and homeostasis, and dysregulated signaling through the HGF receptor, Met, contributes to tumorigenesis, tumor progression, and metastasis in numerous human malignancies. The development of selective small-molecule inhibitors of oncogenic tyrosine kinases (TK) has led to well-tolerated, targeted therapies for a growing number of cancer types. To identify selective Met TK inhibitors, we used a high-throughput virtual screen of the 13.5 million compound ChemNavigator database to find compounds most likely to bind to the Met ATP binding site and to form several critical interactions with binding site residues predicted to stabilize the kinase domain in its inactive conformation. Subsequent biological screening of 70 in silico hit structures using cell-free and intact cell assays identified three active compounds with micromolar IC50 values. The predicted binding modes and target selectivity of these compounds are discussed and compared to other known Met TK inhibitors." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/jm800791f"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/jm800791f">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=âtrueâ></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=âtrueâ></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=âtrueâ></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=âtrueâ></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Fjm800791f&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Fjm800791f&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/jm800791f&amp;href=/doi/10.1021/jm800791f" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2009</span><span class="cit-fg-volume">, 52</span><span class="cit-fg-issue">, 4</span><span class="cit-fg-pageRange">, 943-951</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/52/4" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/jm800724t" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/jm8008103" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Directed Discovery of Agents Targeting the Met Tyrosine Kinase Domain by Virtual Screening</span></h1><ul class="loa non-jats-loa"><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Megan+L.++Peach">Megan L. Peach</a></span><span class="author-xref-symbol "><sup>â </sup></span></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Nelly++Tan">Nelly Tan</a></span><span class="author-xref-symbol "><sup>â¡</sup></span></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Sarah+J.++Choyke">Sarah J. Choyke</a></span><span class="author-xref-symbol "><sup>â¡</sup></span></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Alessio++Giubellino">Alessio Giubellino</a></span><span class="author-xref-symbol "><sup>â¡</sup></span></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Gagani++Athauda">Gagani Athauda</a></span><span class="author-xref-symbol "><sup>â¡</sup></span></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Terrence+R.++Burke%2C++Jr.">Terrence R. Burke, Jr.</a></span><span class="author-xref-symbol "><sup>Â§</sup></span></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Marc+C.++Nicklaus">Marc C. Nicklaus</a></span><span class="author-xref-symbol "><sup>Â§</sup></span></span></li><span class="comma-separator">,Â andÂ </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Donald+P.++Bottaro">Donald P. Bottaro</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol "><sup>â¡</sup></span></span></li><span class="comma-separator"></span></ul><div class="affiliations"><div class="view-affs"><span class="title">View Author Information</span><i class="caret-icon icon-angle-down"></i></div><div class="all-aff-infos hlFld-Affiliation"><div class="aff-info"><span class="aff-text">Basic Research Program, SAICâFrederick, Inc., NCIâFrederick, Frederick, Maryland 21702, Urologic Oncology Branch, Center for Cancer Research, National Cancer Institute, Bethesda, Maryland 20892, and Laboratory of Medicinal Chemistry, Center for Cancer Research, National Cancer Institute, Frederick, Maryland 21702</span></div><div class="corresp-info"><strong>*</strong> To whom correspondence should be addressed. Address: Urologic Oncology Branch, National Cancer Institute, 10 Center Drive MSC 1107, Building 10, CRC, Room 1-3961, Bethesda, Maryland 20892-1107. Phone: 301-402-6499. Fax: 301-402-0922. E-mail: <a href="/cdn-cgi/l/email-protection#f4969b808095869b90b499959d98da9a9d9cda939b82"><span class="__cf_email__" data-cfemail="6d0f0219190c1f02092d000c040143030405430a021b">[emailÂ protected]</span></a></div><div class="corresp-fn" id="afn1">â SAICâFrederick, Inc.</div><div class="corresp-fn" id="afn2">â¡Urologic Oncology Branch, Center for Cancer Research, National Cancer Institute.</div><div class="corresp-fn" id="afn3">Â§Laboratory of Medicinal Chemistry, Center for Cancer Research, National Cancer Institute.</div></div></div><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Fjm800791f&amp;href=/doi/10.1021%2Fjm800791f" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2009</span></span><span class="cit-volume">, 52</span><span class="cit-issue">, 4</span><span class="cit-pageRange">, 943â951</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">January 15, 2009</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>30 June 2008</li><li><span class="item_label"><b>Published</b> online</span>15 January 2009</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 26 February 2009</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/jm800791f" title="DOI URL">https://doi.org/10.1021/jm800791f</a></div><div class="article_header-article-copyright"><strong>Copyright Â© 2009 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D943%26pageCount%3D9%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DMegan%2BL.%2BPeach%252C%2BNelly%2BTan%252C%2BSarah%2BJ.%2BChoyke%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D52%26issueNum%3D4%26contentID%3Djm800791f%26title%3DDirected%2BDiscovery%2Bof%2BAgents%2BTargeting%2Bthe%2BMet%2BTyrosine%2BKinase%2BDomain%2Bby%2BVirtual%2BScreening%26numPages%3D9%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D951%26publicationDate%3DFebruary%2B2009">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/jm800791f"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">2375</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">45</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/jm800791f" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Directed Discovery of Agents Targeting the Met Tyrosine Kinase Domain by Virtual Screening&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Megan&quot;,&quot;last_name&quot;:&quot;L. Peach&quot;},{&quot;first_name&quot;:&quot;Nelly&quot;,&quot;last_name&quot;:&quot;Tan&quot;},{&quot;first_name&quot;:&quot;Sarah&quot;,&quot;last_name&quot;:&quot;J. Choyke&quot;},{&quot;first_name&quot;:&quot;Alessio&quot;,&quot;last_name&quot;:&quot;Giubellino&quot;},{&quot;first_name&quot;:&quot;Gagani&quot;,&quot;last_name&quot;:&quot;Athauda&quot;},{&quot;first_name&quot;:&quot;Terrence&quot;,&quot;last_name&quot;:&quot;R. Burke&quot;},{&quot;first_name&quot;:&quot;Marc&quot;,&quot;last_name&quot;:&quot;C. Nicklaus&quot;},{&quot;first_name&quot;:&quot;Donald&quot;,&quot;last_name&quot;:&quot;P. Bottaro&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2009&quot;,&quot;month&quot;:&quot;01&quot;,&quot;day&quot;:&quot;15&quot;,&quot;issue&quot;:&quot;4&quot;,&quot;volume&quot;:&quot;52&quot;,&quot;pages&quot;:&quot;943-951&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/jm800791f&quot;},&quot;abstract&quot;:&quot;Hepatocyte growth factor (HGF) is an important regulator of normal development and homeostasis, and dysregulated signaling through the HGF receptor, Met, contributes to tumorigenesis, tumor progression, and metastasis in numerous human malignancies. The development of selective small-molecule inhibitors of oncogenic tyrosine kinases (TK) has led to well-tolerated, targeted therapies for a growing number of cancer types. To identify selective Met TK inhibitors, we used a high-throughput virtual screen of the 13.5 million compound ChemNavigator database to find compounds most likely to bind to the Met ATP binding site and to form several critical interactions with binding site residues predicted to stabilize the kinase domain in its inactive conformation. Subsequent biological screening of 70 in silico hit structures using cell-free and intact cell assays identified three active compounds with micromolar IC50 values. The predicted binding modes and target selectivity of these compounds are discussed and compare&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Fjm800791f&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=jm800791f" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Fjm800791f&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=jm800791f" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Fjm800791f&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=jm800791f" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Fjm800791f&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=jm800791f&amp;href=/doi/10.1021/jm800791f" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/jm800791f" /></input><a href="/doi/pdf/10.1021/jm800791f" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (2 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/jm800791f&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Fjm800791f%26sid%3Dliteratum%253Aachs%26pmid%3D19199650%26genre%3Darticle%26aulast%3DPeach%26date%3D2009%26atitle%3DDirected%2BDiscovery%2Bof%2BAgents%2BTargeting%2Bthe%2BMet%2BTyrosine%2BKinase%2BDomain%2Bby%2BVirtual%2BScreening%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D52%26issue%3D4%26spage%3D943%26epage%3D951%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=291445" title="Chemical structure">Chemical structure</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=291352" title="Screening assays">Screening assays</a>,</li><li><a href="/action/doSearch?ConceptID=292186" title="Receptors">Receptors</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/52/4" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="f_tgr1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-4/jm800791f/production/images/medium/jm-2008-00791f_0005.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-4/jm800791f/production/images/large/jm-2008-00791f_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm800791f&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Hepatocyte growth factor (HGF) is an important regulator of normal development and homeostasis, and dysregulated signaling through the HGF receptor, Met, contributes to tumorigenesis, tumor progression, and metastasis in numerous human malignancies. The development of selective small-molecule inhibitors of oncogenic tyrosine kinases (TK) has led to well-tolerated, targeted therapies for a growing number of cancer types. To identify selective Met TK inhibitors, we used a high-throughput virtual screen of the 13.5 million compound ChemNavigator database to find compounds most likely to bind to the Met ATP binding site and to form several critical interactions with binding site residues predicted to stabilize the kinase domain in its inactive conformation. Subsequent biological screening of 70 in silico hit structures using cell-free and intact cell assays identified three active compounds with micromolar IC<sub>50</sub> values. The predicted binding modes and target selectivity of these compounds are discussed and compared to other known Met TK inhibitors.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i1">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_13724" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_13724" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Hepatocyte growth factor (HGF<a class="ref internalNav" href="#fnabbrev" aria-label="a">a</a>) <div class="NLM_fn" id="fnabbrev">a<p class="last">Abbreviations: HGF, hepatocyte growth factor; TK, tyrosine kinase; RTK, receptor tyrosine kinase; EGFR, epidermal growth factor receptor; PDGFRÎ², platelet-derived growth factor receptor Î²; FGFR1, fibroblast growth factor receptor 1; ABL1, Abelson murine leukemia viral oncogene homologue 1; IGF1R, insulin-like growth factor receptor 1; VEGFR2, vascular endothelial cell growth factor receptor 2; MST1R, macrophage stimulating 1 receptor; ROS1, v-ros avian UR2 sarcoma virus oncogene homologue 1; ALK, anaplastic lymphoma receptor tyrosine kinase; ELISA, enzyme-linked immunosorbent assay; AMP-PNP, adenosine 5â²-(Î²,Î³-imido)triphosphate; PBS, phosphate-buffered saline; rmsd, root mean square deviation; SD, self-describing; MOE, molecular operating environment; PCH, polarity-charged-hydrophobicity; GB/SA, generalized Born/surface area; OPLS-AA, optimized potential for liquid simulationsâall atom; ANOVA, analysis of variance.</p></div> is a secreted, heparin-binding protein that stimulates mitogenesis, motogenesis, and morphogenesis in a wide spectrum of cellular targets. Its receptor is the receptor tyrosine kinase (RTK) Met. Activation of the HGF/Met signaling pathway leads to a variety of cellular responses, including proliferation and survival, angiogenesis, and motility and invasion.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> Overexpression of Met and/or uncontrolled activation of its signaling pathway occurs in many human cancers. The presence of increased expression of either Met or HGF in tumor cell lines has been shown to correlate with tumor aggressiveness and decreased survival rates in several types of cancer.<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a> Germline and somatic missense mutations in the kinase domain of Met, leading to increased kinase activity, have been found in papillary renal cell carcinomas. This suggests that selective inhibition of the kinase domain may be a viable therapeutic strategy for the treatment of papillary renal carcinoma and possibly several other human cancers.</div><div class="NLM_p">The overall structure of the Met receptor is that of a typical RTK, with an extracellular ligand binding domain, a transmembrane helix, and an intracellular kinase domain. HGF binding to the extracellular domain promotes receptor clustering and the autophosphorylation of several tyrosine residues in the kinase domain, leading to kinase activation.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> The intracellular domain has the standard kinase fold, with an amino-terminal Î²-sheet-containing lobe and a carboxyl-terminal helical lobe connected through a hinge region. The ATP binding site is in a deep, narrow, coin-slot-like cleft between the two lobes.<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> Most existing kinase domain inhibitors target the ATP binding site. It was originally thought that identifying inhibitors selective to only one kinase domain would be difficult, since there are many kinases, all of which bind ATP, and the sequence of residues in the ATP binding site is highly conserved.<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> However, in recent years many selective kinase inhibitors have been developed. One method for achieving selectivity is to target an inactive conformation of the binding site.<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> This is a useful strategy for Met because in the crystal structure complexed with the staurosporine analogue K-252a, the activation loop adopts a unique inhibitory conformation such that ATP and substrate peptides cannot bind.<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a></div><div class="NLM_p last">Here, we describe a virtual screen to identify new compounds that inhibit the Met kinase and specifically its conformation in the inactive state. The general objective of virtual screening is to select a small subset of compounds predicted to have activity against a given biological target out of a large database of commercially available samples. In conventional high-throughput screening, thousands to hundreds of thousands of compounds are physically tested in parallel. The goal of virtual high-throughput screening is to test compounds computationally in order to reduce the number of compounds that are tested experimentally. The number of compounds in the final set can be adjusted according to the resources available for assaying. A variety of computational methods can be used for virtual screening depending on the desired size of the final subset and on the amount of information known about the target, its natural ligands, and any known inhibitors. The screening methods used here included filtering of a large database of commercially available compounds based on physicochemical properties, receptorâligand docking and scoring, and pharmacophore searches within the docking results. This produced an initial subset of approximately 600â000 compounds, which was reduced to a final set of 175 molecules. This set had very little structural similarity to any known kinase inhibitors. The set was ranked using detailed force field calculations, and the top 70 compounds were purchased for testing in a cell-free system as well as in intact cells using a two site electrochemiluminescent immunoassay of Met activation. Three of the compounds tested showed inhibition of Met at micromolar or submicromolar levels.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_02585" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_02585" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i3" class="anchor-spacer"></div><h3 class="article-section__title" id="_i3"> Virtual Screen</h3><div class="NLM_p">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a> shows a schematic summary of the overall virtual screening procedure followed in this study. The ChemNavigator database (November 2004 release) consisted of a compilation of 13.5 million commercially available chemical samples from 154 international chemistry suppliers. During preliminary processing of the database, we added explicit hydrogens and calculated three-dimensional coordinates for each molecule. The first stage of processing was designed to remove generally unsuitable and undesirable compounds: very large and very small molecules, inorganic compounds, molecules whose lipophilicity was considered too high or too low, molecules with more than 15 rotatable bonds (which are not handled well by the docking program), and molecules with more than one undefined stereocenter, whose three-dimensional structures are therefore partially unknown. The processed database was further filtered to choose compounds whose physicochemical properties were within the ranges found in known kinase inhibitors to eliminate from the outset compounds that were unlikely to show any kinase-binding ability. These filtering criteria included molecular weight, number of aromatic rings and rotatable bonds, polar and nonpolar surface area, logâ<i>P</i>, and number of hydrogen bond donors and acceptors.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-4/jm800791f/production/images/medium/jm-2008-00791f_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-4/jm800791f/production/images/large/jm-2008-00791f_0004.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Flowchart of the virtual screening procedure. The number of compounds at each stage, from the starting point of a large database of small molecules, a crystal structure, and a set of known inhibitors to the end point of a small set of compounds proposed for biological screening, is listed.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-4/jm800791f/production/images/large/jm-2008-00791f_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm800791f&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The primary target crystal structure used was that of the Met TK domain cocrystallized with the staurosporine analogue K-252a.<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> The structure was prepared for docking, and a test run was performed using a series of 40 known kinase inhibitors from the literature that possessed a variety of core structures.<a onclick="showRef(event, 'ref5 ref6'); return false;" href="javascript:void(0);" class="ref ref5 ref6">(5, 6)</a> The filtered database was then docked using GOLD,<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> saving up to 10 poses for each compound. The majority of the compounds from prior filtering were docked successfully. As described in more detail in <a class="ref internalNav" href="#sec4" aria-label="Experimental Section">Experimental Section</a>, this was desirable because the scoring function of the docking program had very little predictive ability to discern binders from nonbinders.</div><div class="NLM_p">We used the structural interactions between Met and its ligands for analysis of the docked poses with pharmacophore-based filtering. This step is not typical of virtual screening strategies. Rather, the process generally moves directly from docking to scoring, since the docking program output is a score for each docked pose. However, we have found that filtering the docked poses with a series of pharmacophore queries to remove poses that do not form certain essential interactions with the target binding site significantly improves the quality of the results. Large-scale analysis of docking program performance has shown that, while existing scoring functions are generally good at producing reasonable docked poses of a molecule in a binding site, they are not necessarily good at discriminating between good binders and poor binders.<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a></div><div class="NLM_p">Docking results were filtered to enforce the presence of the following four interactions (Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>): (1) a hydrogen bond to residues in the hinge region, which is an interaction that is highly characteristic of all compounds bound to the ATP binding site in kinase domains; (2) a hydrophobic or aromatic interaction filling the central region of the pocket; (3) an additional hydrophobic or aromatic interaction in one of two smaller subpockets; (4) either a hydrogen bond to the backbone of residue Y1230 in the activation loop or an aromatic Ï-stacking interaction with its ring. The substantial movement of the portion of the activation loop surrounding Y1230 upon ligand binding, together with the hydrogen bonding between this tyrosine and the inhibitor K-252a seen in the crystal structure,<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> suggests that an interaction with Y1230 may be key to inducing and stabilizing the inhibitory conformation of the activation loop. The docked molecules that satisfied all four interaction criteria gave a final set of 175 compounds.</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-4/jm800791f/production/images/medium/jm-2008-00791f_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-4/jm800791f/production/images/large/jm-2008-00791f_0003.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Topographical features of the Met ATP binding site used to filter docked poses. An interaction with each defined region of the binding site was required for a successfully docked compound.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-4/jm800791f/production/images/large/jm-2008-00791f_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm800791f&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">Of 175 molecules in the final set, 70 were available for purchase. The available compounds were ranked for priority of testing according to the predicted strength of their interactions with binding site, using eMBrAcE in MacroModel.<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> Each docked ligand was energy-minimized in the binding site, and the total force field interaction energy (the sum of the van der Waals, electrostatic, and solvation energies) was calculated. The previously calculated set of physicochemical properties was used to establish that all the compounds followed the Lipinski<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> and Veber<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> rules for druglikeness and oral bioavailability, with the exception of a few where logâ<i>P</i> was modestly higher than typically accepted. Veberâs work suggests that polar surface area is a better predictor of membrane permeability than logâ<i>P</i>, so these compounds were not eliminated from consideration.</div></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> Biological Assays Measuring Met Activation</h3><div class="NLM_p">The 70 commercially available compounds were purchased and screened using cell-free and intact-cell assays developed to measure Met TK activation as represented by receptor autophosphorylation. Both assays were two-site immunoassays that utilize electrochemiluminescent detection, which provides significantly greater sensitivity and dynamic range relative to conventional enzyme-linked immunosorbent assay (ELISA) methods. Assay results were normalized to standard curves prepared using recombinantly expressed, purified Met proteins to maximize reproducibility and quantitation of kinase inhibition. Biological testing yielded three preliminary hit compounds that inhibited Met TK autophosphorylation induced by the addition of ATP and metal ions to unstimulated receptor in detergent solution and by ligand treatment of intact, quiescent normal human mammary epithelial cells (Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a>). Compound <b>1</b> inhibited kinase activity in the cell-free assay with an IC<sub>50</sub> of 0.675 Î¼M and inhibited HGF-induced Met activation in intact cells with an IC<sub>50</sub> of 30 Î¼M. Compound <b>2</b> inhibited kinase activity in the cell-free assay with an IC<sub>50</sub> of 43 Î¼M and Met activation in intact cells with an IC<sub>50</sub> of 50 Î¼M, while compound <b>3</b> inhibited Met activation in intact cells with an IC<sub>50</sub> of 30 Î¼M. Each compound showed greater than 95% inhibition at or less than 100 Î¼M.</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Met TK IC<sub>50</sub> and Maximal Inhibition Values of Hit Compounds<a class="ref internalNav" href="#tbl1-fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table frame_topbot"><colgroup><col align="left" /></col><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead valign="middle"><tr valign="top"><th align="center" valign="middle">Â </th><th colspan="2" align="center" valign="middle">Met TK IC<sub>50</sub> (Î¼M)</th><th align="center" valign="middle" char=".">Â </th></tr><tr valign="top"><th align="center" valign="middle">hit compd</th><th align="center" valign="middle">cell-free</th><th align="center" valign="middle" char=".">intact cells</th><th align="center" valign="middle" char=".">max inhibition (%), intact cells</th></tr></thead><tbody><tr valign="top"><td align="left"><b>1</b></td><td align="left">0.675</td><td align="char" char=".">30</td><td align="char" char=".">98</td></tr><tr valign="top"><td align="left"><b>2</b></td><td align="left">43</td><td align="char" char=".">50</td><td align="char" char=".">95</td></tr><tr valign="top"><td align="left"><b>3</b></td><td align="left">ND</td><td align="char" char=".">30</td><td align="char" char=".">98</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-tbl1-fn1"><div class="footnote" id="tbl1-fn1"><sup>a</sup><p class="last">ND = not determined.</p></div></div></div></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i6" class="anchor-spacer"></div><h3 class="article-section__title" id="_i6"> Docked Structures of Preliminary Hits</h3><div class="NLM_p">The structures of compounds <b>2</b> and <b>3</b>, along with their binding modes as predicted by docking, are shown in Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a>. Both compounds have a three-ring structure in which the central ring forms a hydrogen bonding interaction with the backbone of residue M1160 in the hinge region. Interestingly, in both cases these hydrogen bonds are somewhat unusual, occurring via a cyano group in the case of compound <b>2</b> and a thione in the case of compound <b>3</b>. A second, hydrophobic ring is buried in the binding pocket and interacts with hydrophobic residues I1084, V1092, M1211, and M1229. The third ring is oriented along the surface edge of the binding site and makes an aromatic ring-stacking interaction with Y1230, along with a hydrogen bond to its backbone in the case of compound <b>2</b>. The lower binding affinity of compound <b>3</b> relative to <b>2</b> indicated in cell-free kinase assays can be explained by the lack of hydrogen bonding to Y1230 and by a weaker hydrogen bond to the hinge via the thione compared to the cyano group.</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-4/jm800791f/production/images/medium/jm-2008-00791f_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-4/jm800791f/production/images/large/jm-2008-00791f_0002.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. (A) Chemical structure of compound <b>2</b>. (B) Predicted binding orientation and residue interactions of compound <b>2</b>. (C) Chemical structure of compound <b>3</b>. (D) Predicted binding orientation and residue interactions of compound <b>3</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-4/jm800791f/production/images/large/jm-2008-00791f_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm800791f&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The structure of compound <b>1</b> with its predicted binding site orientation is shown in Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a>. This compound has a central structure of three linked rings, one side of which hydrogen-bonds to hinge residues M1160 and P1158 while the other side interacts with Y1230 in the activation loop. Like compounds <b>2</b> and <b>3</b>, compound <b>1</b> also has a hydrophobic ring buried in the binding pocket and interacting with hydrophobic residues I1084, V1092, F1089, and M1229. The chlorine atoms on this phenyl ring form somewhat unfavorable van der Waals interactions with one side of the central linked ring structure. This is due to the fact that the docking was done with a static receptor in which the binding site is not quite wide enough for the rings to adopt a more orthogonal orientation, no doubt because the cocrystallized ligand is planar.<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> The range of existing crystal structures of Met<a onclick="showRef(event, 'ref3 ref12 ref13'); return false;" href="javascript:void(0);" class="ref ref3 ref12 ref13">(3, 12, 13)</a> confirm the plasticity of this kinase in molding the activation and nucleotide-binding loops to the shape of the ligand in the ATP binding site, so we anticipate that this compound could easily be accommodated with a small conformational shift.</div><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-4/jm800791f/production/images/medium/jm-2008-00791f_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-4/jm800791f/production/images/large/jm-2008-00791f_0001.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. (A) Chemical structure of compound <b>1</b>. (B) Predicted binding orientation and residue interactions of compound <b>1</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-4/jm800791f/production/images/large/jm-2008-00791f_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm800791f&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i7" class="anchor-spacer"></div><h3 class="article-section__title" id="_i7"> Comparison with Other Met Inhibitors</h3><div class="NLM_p">Compound <b>1</b> is generally similar in shape to the cocrystal structure ligand K-252a.<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> An overlay of its docked pose to the binding mode of K-252a shows the central three linked rings aligned across one axis of the planar K-252a structure, with the chlorophenyl ring partially overlapping one of the nonpolar side rings (Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a>A,C). PF-02341066, a potent Met inhibitor whose preclinical evaluation was recently published,<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> is structurally similar to compound <b>2</b> in that it has a central ring that hydrogen-bonds to the hinge backbone and an adjacent hydrophobic ring that is buried in the binding pocket (Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a>B,D). PF-02341066 does not, however, form any interactions with Y1230 or surrounding regions of the activation loop.</div><figure id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-4/jm800791f/production/images/medium/jm-2008-00791f_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-4/jm800791f/production/images/large/jm-2008-00791f_0007.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. (A, C) Overlay of the docked position of compound <b>1</b> (gray) with the crystal structure of Met inhibitor K252-a (orange). (B, D) Overlay of the docked positions of compound <b>2</b> (gray) with the Met inhibitor PF-02341066 (blue). The orientation of the compounds in views A and B is the same as that shown in Figures <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a> and <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a>, whereas views C and D are rotated by 90Â° about the <i>X</i> axis for clarity in visualizing the overlay.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-4/jm800791f/production/images/large/jm-2008-00791f_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm800791f&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">PHA665752 and SU11274 are potent and selective inhibitors for Met from a family of rationally designed pyrrole indolinones.<a onclick="showRef(event, 'ref2 ref15 ref16'); return false;" href="javascript:void(0);" class="ref ref2 ref15 ref16">(2, 15, 16)</a> However, we discovered as we began docking calculations that these indolinone compounds, which were designed using a homology model of Met built from the structure of the closely related fibroblast growth factor receptor-1 (FGFR1) kinase,<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a> do not fit into the Met/K-252a cocrystal structure<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> binding site because of the conformation of the activation loop. The moiety at the 5-position of the indolinone ring, which was designed to displace a water molecule in the FGFR1 structure, is the portion of the structure that does not fit. The sulfoxide clashes with M1229, and the chlorinated phenyl ring clashes with V1092 and F1089 in the hydrophobic region of the binding site (Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>). The recently released structure of Met bound to SU11274<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> suggests that these compounds are likely to be accommodated by a shift in the position of the activation loop and a flip of the Y1230 side chain to pack against the chlorinated phenyl ring. This shift in loop conformation to maintain the interaction with Y1230 confirms the importance of interactions with this residue in conferring binding affinity. Similarly, in the structures of Met bound to triazolopyridazines, Y1230 is observed to form a Ï-stacking interaction with the heteroaromatic ring system in those compounds.<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a></div><div class="NLM_p last">In contrast to the indolinone and triazolopyridazine compounds, both <b>1</b> and <b>2</b> form hydrogen bonds to the backbone of residue Y1230 rather than simply making an aromatic ring-stacking interaction. This suggests that these compounds might be effective inhibitors of the mutated variants Y1230H/C/D (SwissProt P08581, also known as Y1248<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> in the context of GenBank <a href="https://www.ncbi.nlm.nih.gov/gquery?Term=J02958">J02958</a>), unlike SU11274.<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a></div></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i8" class="anchor-spacer"></div><h3 class="article-section__title" id="_i8"> Similarity to Other Kinase Inhibitors</h3><div class="NLM_p last">To examine the chemical novelty of the three hits, we looked for any similarity to compounds in a commercial database of 170â000 known kinase inhibitors (Kinase ChemBioBase from Jubilant Biosys), as well as a set of 7944 compounds with reported binding affinities against kinase targets from BindingDB.<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> The databases were searched using SciTegic extended connectivity functional class fingerprints<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> for compounds similar to any of the three hits. No compounds were found in either database with a Tanimoto similarity<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> higher than 0.4 to compound <b>1</b> or higher than 0.5 to compounds <b>2</b> and <b>3</b>, suggesting that these hits do not have any significant substructural similarity to known kinase inhibitors.</div></div><div id="sec2_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i9" class="anchor-spacer"></div><h3 class="article-section__title" id="_i9"> Selectivity Assessment of Hit Compound <b>1</b></h3><div class="NLM_p">The submicromolar potency of compound <b>1</b> for inhibition of Met ATP binding and autophosphorylation prompted us to investigate the activity of this compound against other structurally related, oncogenically relevant TKs. A panel of 10 TKs, Abelson murine leukemia viral oncogene homologue 1 (ABL1), anaplastic lymphoma receptor tyrosine kinase (ALK), epidermal growth factor receptor (EGFR), FGFR1, insulin-like growth factor receptor 1 (IGF1R), vascular endothelial cell growth factor receptor 2 (VEGFR2), Met, macrophage stimulating 1 receptor (MST1R), platelet-derived growth factor receptor Î² (PDGFR Î²), and v-ros avian UR2 sarcoma virus oncogene homologue 1 (ROS1), was selected for analysis through the SelectScreen Kinase Profiling Service (Invitrogen, Carlsbad, CA). The assay method employs a fluorescence-coupled enzyme format and is based on the differential sensitivity of phosphorylated and nonphosphorylated peptides to proteolytic cleavage. A significant benefit of this ratiometric method for quantitating reaction progress is the elimination of well-to-well variation in peptide concentration and signal intensities. The results of this analysis are summarized in Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a>. Significant inhibition (>50%) at 10 Î¼M for compound <b>1</b> was observed for Met, VEGFR2, ABL1, and FGFR1, with greatest relative inhibition of FGFR1. Very little (<10%) inhibition was observed for IGF1R or PDGFRÎ².</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Initial TK Selectivity Profile of Compound <b>1</b></div></div></div><div class="scrollable-table-wrap"><table class="table frame_topbot"><colgroup><col align="left" /></col><col align="char" char="." /></col></colgroup><thead valign="middle"><tr valign="top"><th align="center" valign="middle">kinase tested</th><th align="center" valign="middle" char=".">mean inhibition at 10 Î¼M (%)</th></tr></thead><tbody><tr valign="top"><td align="left">ABL1</td><td align="char" char=".">60</td></tr><tr valign="top"><td align="left">ALK</td><td align="char" char=".">36</td></tr><tr valign="top"><td align="left">EGFRâ(ErbB1)</td><td align="char" char=".">14</td></tr><tr valign="top"><td align="left">FGFR1</td><td align="char" char=".">85</td></tr><tr valign="top"><td align="left">IGF1R</td><td align="char" char=".">â7</td></tr><tr valign="top"><td align="left">KDRâ(VEGFR2)</td><td align="char" char=".">73</td></tr><tr valign="top"><td align="left">METâ(cMet)</td><td align="char" char=".">55</td></tr><tr valign="top"><td align="left">MST1Râ(RON)</td><td align="char" char=".">45</td></tr><tr valign="top"><td align="left">PDGFRÎ²</td><td align="char" char=".">8</td></tr><tr valign="top"><td align="left">ROS1</td><td align="char" char=".">27</td></tr></tbody></table></div></div><div class="NLM_p">To identify the structural basis for these differences in affinity, we performed additional docking studies with GOLD<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> to examine the interactions of compound <b>1</b> with those TKs for which it had the highest and lowest affinities: FGFR1 (85% inhibition) and IGF1R (no observed inhibition). We used the crystal structures of FGFR1 bound to the inhibitor PD 173074<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> and of IGF1R bound to a nonhydrolyzable ATP analogue.<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> The volume of each binding site was visualized (Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a>) using the program PASS,<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> which fills the cavities in a structure with a set of solvent-excluded probe spheres.</div><figure id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-4/jm800791f/production/images/medium/jm-2008-00791f_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-4/jm800791f/production/images/large/jm-2008-00791f_0006.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Peptide backbone structural models of the ATP binding sites of (A) Met, (B) FGFR1, and (C) IGFR1 with compound <b>1</b> docked in the ATP binding site. The hinge region is shown in red. The solvent-excluded volume in the binding site, as calculated by the program PASS,<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> is outlined in yellow mesh. This figure was created using PyMOL.<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-4/jm800791f/production/images/large/jm-2008-00791f_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm800791f&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">As expected from the selectivity screen, <b>1</b> binds well to the FGFR1 structure, with its chlorinated phenyl ring buried more deeply in the binding site than observed for Met. The hydrophobic subpocket where this ring fits is deeper and larger in FGFR1 than in Met, as shown in Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a>A,B, and the ring fits in much the same position as the dimethoxyphenyl ring of PD 173074, which has been shown to confer FGFR selectivity to that class of compounds.<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> For IGF1R, in contrast, this subpocket is essentially nonexistent (Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a>C), and <b>1</b> is misdocked âupside downâ with the chlorinated phenyl ring outside the binding site, consistent with the low observed selectivity for that kinase. These docking results suggest that the selectivity of analogues of <b>1</b> could be tuned to favor either FGFR1 or Met by adjusting the size and nature of substituents on the phenyl ring. Such modifications could presumably also affect binding affinity to the clinically relevant Met mutant V1092I (V1110<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> in GenBank <a href="https://www.ncbi.nlm.nih.gov/gquery?Term=J02958">J02958</a>), in which the size of the hydrophobic subpocket is likely to be reduced.</div><div class="NLM_p">While the strategy of developing TK inhibitors with selectivity for a single kinase offers theoretical advantages in limiting potential off-target toxicity, it is also possible that many such compounds will have limited efficacy as single agent anticancer therapies, due in part to the development of drug-resistant mutations in tumor cell lines. The number of selective, molecularly targeted therapies being tested in combinations in human clinical trials is growing rapidly, particularly for cancers where multiple oncogenic determinants have been identified. Thus, engineering a TK inhibitor with selectivity for a small group of known, disease-critical kinases may offer the advantages of broad action without the development cost of singularly targeted agents or the potential toxicity of broad-acting agents with poorly defined selectivity. A potent inhibitory effect on FGFR1, an important mediator of angiogenesis, could prove to be a very desirable feature of a small molecule Met antagonist. FGFR1-mediated signaling has been implicated in the progression of a wide variety of solid tumor types where Met activation also occurs, including pancreatic and thyroid cancers.<a onclick="showRef(event, 'ref25 ref26'); return false;" href="javascript:void(0);" class="ref ref25 ref26">(25, 26)</a><i>FGFR1</i> gene amplification occurs in breast cancer<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> and overexpression occurs in the majority of high-grade prostate cancers.<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a></div><div class="NLM_p last">Inhibitors with combined activity against VEGFR2 and FGFR1 are known; for example, brivanib alaninate (Bristol-Myers Squibb) is currently in clinical trials for a variety of solid tumors.<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> Similarly, inhibitory activity against VEGFR2 has been reported for other Met TK inhibitors such as XL880,<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> PHA665752, and SU11274.<a onclick="showRef(event, 'ref1 ref31 ref32'); return false;" href="javascript:void(0);" class="ref ref1 ref31 ref32">(1, 31, 32)</a> However, a Met/FGFR1 or Met/FGFR1/VEGFR2 inhibitor would be novel and a logical target combination for a variety of human cancers.</div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i11">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_14017" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_14017" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">The HGF/Met signaling pathway contributes to tumorigenesis, tumor progression, and metastasis in numerous human malignancies. By use of a high-throughput virtual screening approach to identify potential competitive inhibitors of the Met TK ATP binding site, a set of 70 compounds was selected from the 13.5 million compound ChemNavigator database for biological testing. This set was found to contain three compounds with IC<sub>50</sub> values for inhibition of Met TK activation in the submicromolar to 50 Î¼M range. This hit rate (4%) is comparable to those of other combined virtual and physical screens, particularly those of very large compound libraries not enriched for specific structure types.<a onclick="showRef(event, 'ref33 ref34'); return false;" href="javascript:void(0);" class="ref ref33 ref34">(33, 34)</a> Our results offer another illustration of the efficiency of virtual screening compared to physical high-throughput screening approaches that frequently process 100â000 or more compounds. Here, with the modest effort and expense involved in screening only 70 compounds, we have identified three interesting new hits. The properties of these hit compounds warrant further biological and computational development, which is already underway.</div><div class="NLM_p last">Importantly, the hits identified in our study had little structural similarity to other known Met TK inhibitors, highlighting an important feature of the virtual screening approach. Despite this limited similarity, the active ligands were capable of supporting several key binding interactions common to those known compounds. Outside of these common critical interactions, the novel structures provide new opportunities to chemically improve affinity, selectivity, bioavailability, and toxicity profiles. The selectivity assessment of the active compounds reported here against a panel of sequence-related and oncogenically relevant TKs, combined with additional docking studies, reinforced key Met binding interactions and revealed interactions likely to impart high affinity binding for FGFR1, along with interactions that may confer or remove selectivity toward clinically relevant Met kinase domain mutations. A better understanding of the structural and chemical determinants of selectivity is the first step toward the rational design of potent Met TK antagonists with activity against strategically chosen TK combinations.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i12">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_64655" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_64655" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i13" class="anchor-spacer"></div><h3 class="article-section__title" id="_i13"> Database Processing</h3><div class="NLM_p last">The ChemNavigator iResearch Library (August 2004 release plus the November 2004 update) was processed using the chemoinformatics toolkit CACTVS<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> to add explicit hydrogens to the chemical structures, standardize the encoding of certain functional groups, and generate three-dimensional coordinates with CORINA.<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> In preliminary filtering using the program Pipeline Pilot,<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> we removed any salts and solvents, keeping only the largest fragment in each molecular record, and filtered out compounds containing atoms other than H, C, N, O, P, S, F, Cl, Br, and I, compounds with a molecular weight less than 100 or greater than 800, compounds with more than 15 rotatable bonds (excluding terminal rotors), compounds with a logâ<i>P</i> of less than â3.0 or greater than 8.0, and compounds with more than one undefined stereocenter. Any duplicate structures were then eliminated.</div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i14" class="anchor-spacer"></div><h3 class="article-section__title" id="_i14"> Preparation of Crystal Structures</h3><div class="NLM_p">For the virtual screening run, the crystal structure (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1R0P">1R0P</a>) of the kinase domain of Met crystallized with the inhibitor K-252a<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> was prepared for docking by deleting the crystal waters, capping the terminal and loop ends (where regions of the sequence were disordered in the crystal) with neutral amine or aldehyde groups, and adding explicit hydrogens. The binding site was defined as a sphere with radius 10 Ã, centered at the midpoint of the bond between atoms C3 and C4 in K-252a.</div><div class="NLM_p last">For selectivity analysis of compound <b>1</b>, the crystal structure of FGFR1 bound to the inhibitor PD 173074<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2FGI">2FGI</a>) and the crystal structure of IGF1R bound to adenosine 5â²-(Î²,Î³-imido)triphosphate (AMP-PNP)<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1JQH">1JQH</a>) were used. As with Met, these structures were prepared for docking by deleting crystal waters (and the B chain in the case of 2FGI), capping chain ends with neutral groups, and adding explicit hydrogens. Coordinates were also built for all incompletely resolved surface side chains, none of which were near the binding site. The 10 Ã binding site sphere was centered at the position of ligand atom C31 for <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2FGI">2FGI</a> and of ligand atom N9 for <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1JQH">1JQH</a>.</div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i15" class="anchor-spacer"></div><h3 class="article-section__title" id="_i15"> Validation of Docking Protocol</h3><div class="NLM_p last">A small database of four known Met inhibitors<a onclick="showRef(event, 'ref2 ref16'); return false;" href="javascript:void(0);" class="ref ref2 ref16">(2, 16)</a> along with K-252a<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> and staurosporine was constructed to be used as a test case for determining optimal docking parameters. As discussed above, these inhibitors do not fit in the 1R0P crystal structure binding site, so a second small database composed of a series of 40 known kinase inhibitors with a variety of core structures from the literature<a onclick="showRef(event, 'ref5 ref6'); return false;" href="javascript:void(0);" class="ref ref5 ref6">(5, 6)</a> was also constructed. Two GOLD<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> docking runs to compare the GoldScore fitness function to the ChemScore fitness function were executed. We had found in prior work that the âlibrary screeningâ genetic algorithm settings performed poorly, so the â7â8 times speedupâ settings were used and the docking results were analyzed to choose a scoring function and a reasonable score cutoff value. There are no <i>K</i><sub>i</sub> or binding affinity data for this set of kinase inhibitors against Met, so we were unable to compare the scores from the two different scoring functions to experimental data. However, we found that ChemScore generated poses that were consistent with experimentally determined binding modes of the known inhibitors, although the value of the score itself had no predictive ability to distinguish between high- and low-affinity compounds. The ChemScore scoring function was therefore used with a generous score cutoff for keeping poses, since a low score did not necessarily mean that a pose was bad in this case.</div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i16" class="anchor-spacer"></div><h3 class="article-section__title" id="_i16"> Met-Specific Filtering of Processed ChemNavigator Database</h3><div class="NLM_p last">The test database of kinase inhibitors was also used to look for reasonable ranges for logâ<i>P</i>, polar surface area, molecular weight, and other properties for filtering potential new inhibitors. A set of 32 properties for the known inhibitors was calculated using MOE<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> and Pipeline Pilot,<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> including several versions of logâ<i>P</i>, logâ<i>D</i> and solubility, and various estimations of polar/nonpolar surface area. On the basis of these results, the processed ChemNavigator database was filtered to keep only compounds with molecular weights between 250 and 500, more than two aromatic rings, fewer than four rotatable bonds (not counting terminal rotors), between two and five hydrogen bond acceptors, between one and three hydrogen bond donors, a logâ<i>P</i> value between 1.0 and 6.0, no phosphate or sulfate groups, a polar surface area less than 100 Ã<sup>2</sup>, and a nonpolar surface area of at least 200 Ã<sup>2</sup>.</div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17"> Docking</h3><div class="NLM_p last">Docking runs were performed using the program GOLD 2.1<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> with the â7â8 times speedupâ genetic algorithm settings for virtual screening and the default settings for follow-up analysis. The ChemScore fitness function was used with the âflip ring corners,â âflip planar N,â and âinternal H-bondsâ flags set. Early termination was allowed if the top five solutions were within 1.5 Ã root-mean-square deviation (rmsd). The 10 highest-scoring poses were saved for each compound, and poses with ChemScore fitness less than 20.0 were rejected. The results and scores were saved in a single self-describing (SD) file for each run.</div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18"> Pharmacophore-Based Filtering</h3><div class="NLM_p last">A series of four receptor-based pharmacophore filters were created in MOE<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> by defining required hydrogen bond or hydrophobic features for ligands based on the positions of atoms in the binding site. The first filter eliminated all poses that did not have a hydrogen bond to the backbone of hinge residue P1158 or M1160. The second filter eliminated all poses that did not have a hydrophobic or aromatic interaction with the central hydrophobic residues I1084, V1092, M1211, and M1229. The third filter defined a point at the centroid of atoms V1092 CÎ³1, A1108 CÎ², and L1157 CÎ´1 and a second point at the centroid of atoms F1089 CÎµ2, and CÎ¶ and K1110 CÎ³, CÎ´, and CÎµ. This filter then eliminated all poses that did not have a hydrophobic or aromatic atom within 2.5 Ã of at least one of these two points. The fourth filter eliminated all poses that did not have either a hydrogen bond to Y1230 N or an aromatic Ï-stacking interaction with the tyrosine ring. Successful docking required that a molecule passed all four filters. The SD file of docked poses was imported into a MOE database, and each pose was annotated with the polarity-charged-hydrophobicity (PCH) pharmacophore scheme. The database was then searched with the âuse absolute positionsâ option in each query to test the poses in the frame of reference of the binding site as they were docked by the docking program rather than rotating each molecule to best match the query.</div></div><div id="sec4_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i19" class="anchor-spacer"></div><h3 class="article-section__title" id="_i19"> Ranking Commercially Available Compounds</h3><div class="NLM_p last">The set of physicochemical properties (molecular weight, number of hydrogen bond acceptors and donors, logâ<i>P</i>, polar surface area, and number of rotatable bonds) calculated previously was used to check the compounds for violations of Lipinskiâs rule of five<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> and Veberâs rules for oral bioavailability.<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> To evaluate binding site interactions, each docked ligand was minimized in the binding site and the force field interaction energy between protein and ligand was calculated using eMBrAcE in MacroModel.<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> The energy minimization used the PolakâRibiere conjugate gradient method, with convergence set to a gradient of 0.05, and the optimized potential for liquid simulationsâall atom (OPLS-AA) force field with implicit generalized Born/surface area (GB/SA) water solvent and extended nonbonded cutoffs. Residues I1084, G1085, F1089, V1092, K1110, L1157, P1158, Y1159, M1160, G1163, M1211, M1229, and Y1230 were allowed to move, and a shell of residues within 5 Ã of these was restrained with a force constant of 100 kcal/mol; all other residues were frozen. The compounds were then sorted according to the total interaction energy, equal to the sum of the van der Waals, electrostatic, and solvation energies.</div></div><div id="sec4_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i20" class="anchor-spacer"></div><h3 class="article-section__title" id="_i20"> Electrochemiluminescent Two-Site Immunoassays of Met TK Activity and Content</h3><div class="NLM_p last">The human mammary epithelial cell line B5/589 was cultured to 90% confluence in multiwell plates and serum deprived for 48 h prior to use. For intact cell assays only, test compound over a range of concentrations was added for the final 16 h of the serum deprivation period. After drug treatment, cells were rinsed briefly with phosphate-buffered saline (PBS), stimulated with HGF (180 ng/mL), or left unstimulated for 20 min at 37 Â°C before lysis and protein extraction with ice cold buffer (pH 7.4) containing nonionic detergent, protease, and phosphatase inhibitors. Protein extracts were clarified by centrifugation, total protein concentration in the supernatant was determined, and aliquots of 500 Î¼g were applied to each well of 96-well streptavidin-coated immunoassay plates (MA2400 96 plate, Meso Scale Discovery, Gaithersburg, MD, catalog no. L15SA-1) that had been coated with a biotinylated anti-Met capture antibody (R&D Systems, Minneapolis, MN, catalog no. BAF 358) and blocked against non-specific binding with I-Block (Applied Biosystems, Foster City, CA, catalog no. A1300). For cell-free assays, clarified cell lysates prepared from serum-deprived, untreated cells were applied to the similarly prepared plates. Test compounds at various concentrations were incubated with immobilized Met in the assay plate in a lysis buffer containing 5 mM MgCl<sub>2</sub>, 5 mM MnCl<sub>2</sub>, and 20 Î¼M ATP for 1 h at room temperature. Met activation, as reflected by receptor autophosphorylation, was detected for both intact cell and cell-free samples using a ruthenium (Ru)-tagged antiphosphotyrosine antibody (Clone 4G10, Millipore, Billerica, MA, catalog no. 05-1050). The amount of Met protein captured was detected in parallel using a second Met-specific antibody (R&D Systems, catalog no. AF276). A tripropylamine-based buffer was added to all wells, and electrochemiluminescent light measurements were made using a Sector Imager 2400 plate reader (Meso Scale Discovery). Met protein content was calculated from raw signal values using a standard curve obtained from purified recombinant Met protein (R&D Systems, catalog no. 358-MT). Met autophosphorylation measurements were normalized to Met protein content; Met content, in turn, was normalized to total protein concentrations to detect effects on Met expression level in intact cell samples. A previously characterized Met ATP binding antagonist, PHA665752 (gift from Dr. James Christensen, Pfizer, Inc.) was used as a positive control, and extracts prepared from resting, non-HGF-stimulated cells were used as a negative control. Vehicle controls were included as needed at the highest concentration present in samples.</div></div><div id="sec4_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i21" class="anchor-spacer"></div><h3 class="article-section__title" id="_i21"> Calculations and Statistical Analysis</h3><div class="NLM_p last">Drug treatment of intact cells was performed in duplicate wells; all electrochemiluminescence assay samples were performed in triplicate. Mean values from empty wells were subtracted from all other raw values. The standard curve of Met protein content was analyzed by nonlinear regression curve fitting algorithms (Microsoft Excel or GraphPad Prism software) to generate an equation from which sample values for Met content were derived. Mean values among groups were compared for statistically significant differences using unpaired Studentâs <i>t</i> test (<i>p</i> < 0.01) or ANOVA.</div></div></div><div class="NLM_back"><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_50664" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_50664" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Donald P. Bottaro</span> - <span class="hlFld-Affiliation affiliation">Basic Research Program, SAICâFrederick, Inc., NCIâFrederick, Frederick, Maryland 21702, Urologic Oncology Branch, Center for Cancer Research, National Cancer Institute, Bethesda, Maryland 20892, and Laboratory of Medicinal Chemistry, Center for Cancer Research, National Cancer Institute, Frederick, Maryland 21702</span>;Â 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#c7a5a8b3b3a6b5a8a387aaa6aeabe9a9aeafe9a0a8b1"><span class="__cf_email__" data-cfemail="bbd9d4cfcfdac9d4dffbd6dad2d795d5d2d395dcd4cd">[emailÂ protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Megan L. Peach</span> - <span class="hlFld-Affiliation affiliation">Basic Research Program, SAICâFrederick, Inc., NCIâFrederick, Frederick, Maryland 21702, Urologic Oncology Branch, Center for Cancer Research, National Cancer Institute, Bethesda, Maryland 20892, and Laboratory of Medicinal Chemistry, Center for Cancer Research, National Cancer Institute, Frederick, Maryland 21702</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Nelly Tan</span> - <span class="hlFld-Affiliation affiliation">Basic Research Program, SAICâFrederick, Inc., NCIâFrederick, Frederick, Maryland 21702, Urologic Oncology Branch, Center for Cancer Research, National Cancer Institute, Bethesda, Maryland 20892, and Laboratory of Medicinal Chemistry, Center for Cancer Research, National Cancer Institute, Frederick, Maryland 21702</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Sarah J. Choyke</span> - <span class="hlFld-Affiliation affiliation">Basic Research Program, SAICâFrederick, Inc., NCIâFrederick, Frederick, Maryland 21702, Urologic Oncology Branch, Center for Cancer Research, National Cancer Institute, Bethesda, Maryland 20892, and Laboratory of Medicinal Chemistry, Center for Cancer Research, National Cancer Institute, Frederick, Maryland 21702</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Alessio Giubellino</span> - <span class="hlFld-Affiliation affiliation">Basic Research Program, SAICâFrederick, Inc., NCIâFrederick, Frederick, Maryland 21702, Urologic Oncology Branch, Center for Cancer Research, National Cancer Institute, Bethesda, Maryland 20892, and Laboratory of Medicinal Chemistry, Center for Cancer Research, National Cancer Institute, Frederick, Maryland 21702</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Gagani Athauda</span> - <span class="hlFld-Affiliation affiliation">Basic Research Program, SAICâFrederick, Inc., NCIâFrederick, Frederick, Maryland 21702, Urologic Oncology Branch, Center for Cancer Research, National Cancer Institute, Bethesda, Maryland 20892, and Laboratory of Medicinal Chemistry, Center for Cancer Research, National Cancer Institute, Frederick, Maryland 21702</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Terrence R. Burke</span> - <span class="hlFld-Affiliation affiliation">Basic Research Program, SAICâFrederick, Inc., NCIâFrederick, Frederick, Maryland 21702, Urologic Oncology Branch, Center for Cancer Research, National Cancer Institute, Bethesda, Maryland 20892, and Laboratory of Medicinal Chemistry, Center for Cancer Research, National Cancer Institute, Frederick, Maryland 21702</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Marc C. Nicklaus</span> - <span class="hlFld-Affiliation affiliation">Basic Research Program, SAICâFrederick, Inc., NCIâFrederick, Frederick, Maryland 21702, Urologic Oncology Branch, Center for Cancer Research, National Cancer Institute, Bethesda, Maryland 20892, and Laboratory of Medicinal Chemistry, Center for Cancer Research, National Cancer Institute, Frederick, Maryland 21702</span></div></li></ul></li><li></li><li></li><li></li></ul></div><div class="testing" data-doi="10.1021/jm800791f" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="ack"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i22">Acknowledgment</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_10618" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_10618" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">This research was supported by the Intramural Research Program of the NIH, National Cancer Institute, Center for Cancer Research. This project has been funded in part with federal funds from the National Cancer Institute, National Institutes of Health, under Contract N01-CO-12400. We thank Dr. James Christensen and Pfizer for the generous gift of PHA665752.</p></div><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i23">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_22578" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_22578" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 40 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group">Ma, P. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maulik, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Christensen, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Salgia, R.</span><span> </span><span class="NLM_article-title">c-Met: structure, functions and potential for therapeutic inhibition</span> <span class="citation_source-journal">Cancer Metastasis Rev.</span><span class="NLM_year" style="font-weight: bold;">2003</span> <span class="NLM_volume">22</span> <span class="NLM_fpage">309</span> <span class="NLM_lpage">325</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Fjm800791f&amp;key=10.1023%2FA%3A1023768811842" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Fjm800791f&amp;key=12884908" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Fjm800791f&amp;key=1%3ACAS%3A528%3ADC%252BD3sXjslyiurY%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2003&pages=309-325&author=P.+C.+Maauthor=G.+Maulikauthor=J.+Christensenauthor=R.+Salgia&title=c-Met%3A+structure%2C+functions+and+potential+for+therapeutic+inhibition"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">c-Met: Structure, functions and potential for therapeutic inhibition</span></div><div class="casAuthors">Ma, Patrick C.; Maulik, Gautam; Christensen, James; Salgia, Ravi</div><div class="citationInfo"><span class="NLM_cas:title">Cancer and Metastasis Reviews</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">309-325</span>CODEN:
                <span class="NLM_cas:coden">CMRED4</span>;
        ISSN:<span class="NLM_cas:issn">0167-7659</span>.
    
            (<span class="NLM_cas:orgname">Kluwer Academic Publishers</span>)
        </div><div class="casAbstract">A review.  Studies on signal transduction pathways have generated various promising mol. targets for therapeutic inhibition in cancer therapy.  Receptor tyrosine kinases represent an important class of such therapeutic targets.  C-Met is a receptor tyrosine kinase that has been shown to be overexpressed and/or mutated in a variety of malignancies.  A no. of c-Met activating mutations, many of which are located in the tyrosine kinase domain, have been detected in various solid tumors and have been implicated in invasion and metastasis of tumor cells.  It is known that stimulation of c-Met via its natural ligand, hepatocyte growth factor (also known as scatter factor, HGF/SF) results in a plethora of biol. and biochem. effects in the cell.  Activation of c-Met signaling can lead to scattering, angiogenesis, proliferation, enhanced cell motility, invasion, and eventual metastasis.  In this review, the role of c-Met dysregulation in tumor progression and metastasis is discussed in detail with particular emphasis on c-Met mutations.  Moreover, we summarize current knowledge on various pathways of c-Met signal transduction, highlighting the central role in the cytoskeletal functions.  In this summary is included recent data in our lab. indicating that phosphorylation of focal adhesion proteins, such as paxillin, p125FAK, and PYK2, occurs in response to c-Met stimulation in lung cancer cells.  Most importantly, current data on c-Met suggest that when mutated or overexpressed in malignant cells, c-Met would serve as an important therapeutic target.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo9rIdb5Ao4a7Vg90H21EOLACvtfcHk0lhqdLoFmX4n3A"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXjslyiurY%253D&md5=7b7bf60da872071a39ce313efd32fd1f</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1023%2FA%3A1023768811842&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1023%252FA%253A1023768811842%26sid%3Dliteratum%253Aachs%26aulast%3DMa%26aufirst%3DP.%2BC.%26aulast%3DMaulik%26aufirst%3DG.%26aulast%3DChristensen%26aufirst%3DJ.%26aulast%3DSalgia%26aufirst%3DR.%26atitle%3Dc-Met%253A%2520structure%252C%2520functions%2520and%2520potential%2520for%2520therapeutic%2520inhibition%26jtitle%3DCancer%2520Metastasis%2520Rev.%26date%3D2003%26volume%3D22%26spage%3D309%26epage%3D325" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group">Wang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Le, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liang, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chan, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kiewlich, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miller, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Harris, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sun, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rice, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vasile, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blake, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Howlett, A. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Patel, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McMahon, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lipson, K. E.</span><span> </span><span class="NLM_article-title">Potent and selective inhibitors of the Met [hepatocyte growth factor/scatter factor (HGF/SF) receptor] tyrosine kinase block HGF/SF-induced tumor cell growth and invasion</span> <span class="citation_source-journal">Mol. Cancer Ther.</span><span class="NLM_year" style="font-weight: bold;">2003</span> <span class="NLM_volume">2</span> <span class="NLM_fpage">1085</span> <span class="NLM_lpage">1092</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Fjm800791f&amp;key=14617781" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Fjm800791f&amp;key=1%3ACAS%3A528%3ADC%252BD3sXovFCqtL8%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2003&pages=1085-1092&author=X.+Wangauthor=P.+Leauthor=C.+Liangauthor=J.+Chanauthor=D.+Kiewlichauthor=T.+Millerauthor=D.+Harrisauthor=L.+Sunauthor=A.+Riceauthor=S.+Vasileauthor=R.+A.+Blakeauthor=A.+R.+Howlettauthor=N.+Patelauthor=G.+McMahonauthor=K.+E.+Lipson&title=Potent+and+selective+inhibitors+of+the+Met+%5Bhepatocyte+growth+factor%2Fscatter+factor+%28HGF%2FSF%29+receptor%5D+tyrosine+kinase+block+HGF%2FSF-induced+tumor+cell+growth+and+invasion"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Potent and selective inhibitors of the Met [hepatocyte growth factor/scatter factor (HGF/SF) receptor] tyrosine kinase block HGF/SF-induced tumor cell growth and invasion</span></div><div class="casAuthors">Wang, Xueyan; Le, Phuong; Liang, Congxin; Chan, Julie; Kiewlich, David; Miller, Todd; Harris, Dave; Sun, Li; Rice, Audie; Vasile, Stefan; Blake, Robert A.; Howlett, Anthony R.; Patel, Neela; McMahon, Gerald; Lipson, Kenneth E.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cancer Therapeutics</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1085-1092</span>CODEN:
                <span class="NLM_cas:coden">MCTOCF</span>;
        ISSN:<span class="NLM_cas:issn">1535-7163</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">The hepatocyte growth factor/scatter factor (HGF/SF) receptor, Met, mediates various cellular responses on activation with its ligand, including proliferation, survival, motility, invasion, and tubular morphogenesis.  Met expression is frequently up-regulated in sarcomas and carcinomas.  Exptl. evidence suggests that Met activation correlates with poor clin. outcome and the likelihood of metastasis.  Therefore, inhibitors of Met tyrosine kinase may be useful for the treatment of a wide variety of cancers that have spread from the primary site.  We have discovered potent and selective pyrrole-indolinone Met kinase inhibitors and characterized them for their ability to inhibit HGF/SF-induced cellular responses in vitro.  These compds. inhibit HGF/SF-induced receptor phosphorylation in a dose-dependent manner.  They also inhibit the HGF/SF-induced motility and invasion of epithelial and carcinoma cells.  Therefore, these compds. represent a class of prototype small mols. that selectively inhibit the Met kinase and could lead to identification of compds. with potential therapeutic utility in treatment of cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoVAQRRxT_J1rVg90H21EOLACvtfcHk0livZNXSE1QlFw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXovFCqtL8%253D&md5=3b3141600b15ed4f9c97d86fb7ade81d</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DX.%26aulast%3DLe%26aufirst%3DP.%26aulast%3DLiang%26aufirst%3DC.%26aulast%3DChan%26aufirst%3DJ.%26aulast%3DKiewlich%26aufirst%3DD.%26aulast%3DMiller%26aufirst%3DT.%26aulast%3DHarris%26aufirst%3DD.%26aulast%3DSun%26aufirst%3DL.%26aulast%3DRice%26aufirst%3DA.%26aulast%3DVasile%26aufirst%3DS.%26aulast%3DBlake%26aufirst%3DR.%2BA.%26aulast%3DHowlett%26aufirst%3DA.%2BR.%26aulast%3DPatel%26aufirst%3DN.%26aulast%3DMcMahon%26aufirst%3DG.%26aulast%3DLipson%26aufirst%3DK.%2BE.%26atitle%3DPotent%2520and%2520selective%2520inhibitors%2520of%2520the%2520Met%2520%255Bhepatocyte%2520growth%2520factor%252Fscatter%2520factor%2520%2528HGF%252FSF%2529%2520receptor%255D%2520tyrosine%2520kinase%2520block%2520HGF%252FSF-induced%2520tumor%2520cell%2520growth%2520and%2520invasion%26jtitle%3DMol.%2520Cancer%2520Ther.%26date%3D2003%26volume%3D2%26spage%3D1085%26epage%3D1092" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group">Schiering, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knapp, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marconi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Flocco, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cui, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perego, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rusconi, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cristiani, C.</span><span> </span><span class="NLM_article-title">Crystal structure of the tyrosine kinase domain of the hepatocyte growth factor receptor c-Met and its complex with the microbial alkaloid K-252a</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U.S.A.</span><span class="NLM_year" style="font-weight: bold;">2003</span> <span class="NLM_volume">100</span> <span class="NLM_fpage">12654</span> <span class="NLM_lpage">12659</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Fjm800791f&amp;key=10.1073%2Fpnas.1734128100" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Fjm800791f&amp;key=14559966" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Fjm800791f&amp;key=1%3ACAS%3A528%3ADC%252BD3sXoslKnt74%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=100&publication_year=2003&pages=12654-12659&author=N.+Schieringauthor=S.+Knappauthor=M.+Marconiauthor=M.+M.+Floccoauthor=J.+Cuiauthor=R.+Peregoauthor=L.+Rusconiauthor=C.+Cristiani&title=Crystal+structure+of+the+tyrosine+kinase+domain+of+the+hepatocyte+growth+factor+receptor+c-Met+and+its+complex+with+the+microbial+alkaloid+K-252a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal structure of the tyrosine kinase domain of the hepatocyte growth factor receptor c-Met and its complex with the microbial alkaloid K-252a</span></div><div class="casAuthors">Schiering, Nikolaus; Knapp, Stefan; Marconi, Marina; Flocco, Maria M.; Cui, Jean; Perego, Rita; Rusconi, Luisa; Cristiani, Cinzia</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">100</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">12654-12659</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The protooncogene c-met codes for the hepatocyte growth factor receptor tyrosine kinase.  Binding of its ligand, hepatocyte growth factor/scatter factor, stimulates receptor autophosphorylation, which leads to pleiotropic downstream signaling events in epithelial cells, including cell growth, motility, and invasion.  These events are mediated by interaction of cytoplasmic effectors, generally through Src homol. 2 (SH2) domains, with two phosphotyrosine-contg. sequence motifs in the unique C-terminal tail of c-Met (supersite).  2 (SH2) domains, with two phosphotyrosine-contg. sequence motifs in the unique C-terminal tail of c-Met (supersite).  There is a strong link between aberrant c-Met activity and oncogenesis, which makes this kinase an important cancer drug target.  The furanosylated indolocarbazole K-252a belongs to a family of microbial alkaloids that also includes staurosporine.  It was recently shown to be a potent inhibitor of c-Met.  Here we report the crystal structures of an unphosphorylated c-Met kinase domain harboring a human cancer mutation and its complex with K-252a at 1.8-Ã resoln.  The structure follows the well established architecture of protein kinases.  It adopts a unique, inhibitory conformation of the activation loop, a catalytically noncompetent orientation of helix Î±C, and reveals the complete C-terminal docking site.  The first SH2-binding motif (1349YVHV) adopts an extended conformation, whereas the second motif (1356YVNV), a binding site for Grb2-SH2, folds as a type II Î²-turn.  The intermediate portion of the supersite (1353NATY) assumes a type I Î²-turn conformation as in an Shc-phosphotyrosine binding domain peptide complex.  K-252a is bound in the adenosine pocket with an analogous binding mode to those obsd. in previously reported structures of protein kinases in complex with staurosporine.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo13Yf3cKB-ybVg90H21EOLACvtfcHk0livZNXSE1QlFw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXoslKnt74%253D&md5=d8cd3494f2583dfcb11efb6862aff607</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1734128100&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1734128100%26sid%3Dliteratum%253Aachs%26aulast%3DSchiering%26aufirst%3DN.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DMarconi%26aufirst%3DM.%26aulast%3DFlocco%26aufirst%3DM.%2BM.%26aulast%3DCui%26aufirst%3DJ.%26aulast%3DPerego%26aufirst%3DR.%26aulast%3DRusconi%26aufirst%3DL.%26aulast%3DCristiani%26aufirst%3DC.%26atitle%3DCrystal%2520structure%2520of%2520the%2520tyrosine%2520kinase%2520domain%2520of%2520the%2520hepatocyte%2520growth%2520factor%2520receptor%2520c-Met%2520and%2520its%2520complex%2520with%2520the%2520microbial%2520alkaloid%2520K-252a%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2003%26volume%3D100%26spage%3D12654%26epage%3D12659" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group">Gould, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wong, C. F.</span><span> </span><span class="NLM_article-title">Designing specific protein kinase inhibitors: insights from computer simulations and comparative sequence/structure analysis</span> <span class="citation_source-journal">Pharmacol. Ther.</span><span class="NLM_year" style="font-weight: bold;">2002</span> <span class="NLM_volume">93</span> <span class="NLM_fpage">169</span> <span class="NLM_lpage">178</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Fjm800791f&amp;key=10.1016%2FS0163-7258%2802%2900186-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Fjm800791f&amp;key=12191609" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Fjm800791f&amp;key=1%3ACAS%3A528%3ADC%252BD38XksVShtbc%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=93&publication_year=2002&pages=169-178&author=C.+Gouldauthor=C.+F.+Wong&title=Designing+specific+protein+kinase+inhibitors%3A+insights+from+computer+simulations+and+comparative+sequence%2Fstructure+analysis"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Designing specific protein kinase inhibitors: insights from computer simulations and comparative sequence/structure analysis</span></div><div class="casAuthors">Gould, Christine; Wong, Chung F.</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacology & Therapeutics</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">93</span>
        (<span class="NLM_cas:issue">2-3</span>),
    <span class="NLM_cas:pages">169-178</span>CODEN:
                <span class="NLM_cas:coden">PHTHDT</span>;
        ISSN:<span class="NLM_cas:issn">0163-7258</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Inc.</span>)
        </div><div class="casAbstract">A review.  Protein kinases are important targets for designing therapeutic drugs.  We describe here a computational approach to extend the usefulness of a single protein-inhibitor structure in aiding the design of protein kinase inhibitors.  This approach is based on using sensitivity anal. to identify the most significant functional groups of a lead compd. in accounting for binding affinity and on using comparative sequence/structure anal. to examine whether these functional groups would present specificity.  A sensitivity anal. study is similar to genetic or chem. modification expts. in which specific features of a lead compd. are modified to examine whether they affect properties such as binding affinity.  In this study, the binding affinity was estd. by using an implicit-solvent model in which the electrostatic contributions were obtained by solving the Poisson equation, and the hydrophobic effects were accounted for by using surface-area-dependent terms.  The comparative sequence/structure anal. involves the study of the amino acid distributions of a large no. of protein kinases (384 in this study) near the ligand-binding sites.  This anal. provides useful guiding principles for designing specific inhibitors targeted towards a particular kinase.  Here, we illustrate the utility of these computational approaches by applying them to identify the determinants of the recognition between the protein kinase A and two of its inhibitors.  One inhibitor, balanol, binds to the ATP-binding pocket.  The other, protein kinase inhibitor, binds to the substrate-binding site.  These analyses have helped to construct pharmacophore models for mining new drug leads from small-mol. libraries and for suggesting how a lead compd. or a peptide inhibitor may be modified to generate selective inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpD4cNyidIWgLVg90H21EOLACvtfcHk0livZNXSE1QlFw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XksVShtbc%253D&md5=81c5c5809f3bedad54f9bc0559aca972</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1016%2FS0163-7258%2802%2900186-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0163-7258%252802%252900186-9%26sid%3Dliteratum%253Aachs%26aulast%3DGould%26aufirst%3DC.%26aulast%3DWong%26aufirst%3DC.%2BF.%26atitle%3DDesigning%2520specific%2520protein%2520kinase%2520inhibitors%253A%2520insights%2520from%2520computer%2520simulations%2520and%2520comparative%2520sequence%252Fstructure%2520analysis%26jtitle%3DPharmacol.%2520Ther.%26date%3D2002%26volume%3D93%26spage%3D169%26epage%3D178" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group">Noble, M. E. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Endicott, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johnson, L. N.</span><span> </span><span class="NLM_article-title">Protein kinase inhibitors: insights into drug design from structure</span> <span class="citation_source-journal">Science</span><span class="NLM_year" style="font-weight: bold;">2004</span> <span class="NLM_volume">303</span> <span class="NLM_fpage">1800</span> <span class="NLM_lpage">1805</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Fjm800791f&amp;key=10.1126%2Fscience.1095920" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Fjm800791f&amp;key=15031492" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Fjm800791f&amp;key=1%3ACAS%3A528%3ADC%252BD2cXitFehurc%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=303&publication_year=2004&pages=1800-1805&author=M.+E.+M.+Nobleauthor=J.+A.+Endicottauthor=L.+N.+Johnson&title=Protein+kinase+inhibitors%3A+insights+into+drug+design+from+structure"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Protein Kinase Inhibitors: Insights into Drug Design from Structure</span></div><div class="casAuthors">Noble, Martin E. M.; Endicott, Jane A.; Johnson, Louise N.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">303</span>
        (<span class="NLM_cas:issue">5665</span>),
    <span class="NLM_cas:pages">1800-1805</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">A review.  Protein kinases are targets for treatment of a no. of diseases.  This review focuses on kinase inhibitors that are in the clinic or in clin. trials and for which structural information is available.  Structures have informed drug design and have illuminated the mechanism of inhibition.  We review progress with the receptor tyrosine kinases (growth factor receptors EGFR, VEGFR, and FGFR) and nonreceptor tyrosine kinases (Bcr-Abl), where advances have been made with cancer therapeutic agents such as Herceptin and Gleevec.  Among the serine-threonine kinases, p38, Rho-kinase, cyclin-dependent kinases, and Chk1 have been targeted with productive results for inflammation and cancer.  Structures have provided insights into targeting the inactive or active form of the kinase, for targeting the global constellation of residues at the ATP site or less conserved addnl. pockets or single residues, and into targeting noncatalytic domains.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWqtKz7cvN_rVg90H21EOLACvtfcHk0lhR7IkNOzw8RA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXitFehurc%253D&md5=7fd3f60bee46e59307436eeed285d6ae</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1126%2Fscience.1095920&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1095920%26sid%3Dliteratum%253Aachs%26aulast%3DNoble%26aufirst%3DM.%2BE.%2BM.%26aulast%3DEndicott%26aufirst%3DJ.%2BA.%26aulast%3DJohnson%26aufirst%3DL.%2BN.%26atitle%3DProtein%2520kinase%2520inhibitors%253A%2520insights%2520into%2520drug%2520design%2520from%2520structure%26jtitle%3DScience%26date%3D2004%26volume%3D303%26spage%3D1800%26epage%3D1805" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group">GarcÃ­a-EcheverrÃ­a, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Traxler, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Evans, D. B.</span><span> </span><span class="NLM_article-title">ATP site-directed competitive and irreversible inhibitors of protein kinases</span> <span class="citation_source-journal">Med. Res. Rev.</span><span class="NLM_year" style="font-weight: bold;">2000</span> <span class="NLM_volume">20</span> <span class="NLM_fpage">28</span> <span class="NLM_lpage">57</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Fjm800791f&amp;key=10.1002%2F%28SICI%291098-1128%28200001%2920%3A1%3C28%3A%3AAID-MED2%3E3.0.CO%3B2-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Fjm800791f&amp;key=10608920" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Fjm800791f&amp;key=1%3ACAS%3A528%3ADC%252BD3cXltFWruw%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2000&pages=28-57&author=C.+Garc%C3%ADa-Echeverr%C3%ADaauthor=P.+Traxlerauthor=D.+B.+Evans&title=ATP+site-directed+competitive+and+irreversible+inhibitors+of+protein+kinases"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">ATP site-directed competitive and irreversible inhibitors of protein kinases</span></div><div class="casAuthors">Garcia-Echeverria, Carlos; Traxler, Peter; Evans, Dean B.</div><div class="citationInfo"><span class="NLM_cas:title">Medicinal Research Reviews</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">28-57</span>CODEN:
                <span class="NLM_cas:coden">MRREDD</span>;
        ISSN:<span class="NLM_cas:issn">0198-6325</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">A review with 220 refs.  Several tyrosine and serine/threonine protein kinases have emerged in the last few years as attractive targets in the search for new therapeutic agents being applicable in many different disease indications.  Initially, inhibition of these protein kinases by ATP site-directed inhibitors was considered less prone to success, but medicinal chemists from both academia and industry have been able to impart potency and selectivity to a limited no. of scaffolds by modulating and fine-tuning the interactions of the modified template with the ATP binding site of the selected kinase.  The chem. templates that have been used in the synthesis of ATP site-directed protein kinase inhibitors are reviewed with emphasis on the kinase inhibitors that have entered or are about to enter clin. trials.  Examples have been selected to illustrate how structure-based design approaches and new methods to increase compd. diversity have had an impact on this area of research.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqRnYxd5Xp0UrVg90H21EOLACvtfcHk0lhR7IkNOzw8RA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXltFWruw%253D%253D&md5=f610b09ddf97aeb2cc9f4b930d7c27d7</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1002%2F%28SICI%291098-1128%28200001%2920%3A1%3C28%3A%3AAID-MED2%3E3.0.CO%3B2-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F%2528SICI%25291098-1128%2528200001%252920%253A1%253C28%253A%253AAID-MED2%253E3.0.CO%253B2-2%26sid%3Dliteratum%253Aachs%26aulast%3DGarc%25C3%25ADa-Echeverr%25C3%25ADa%26aufirst%3DC.%26aulast%3DTraxler%26aufirst%3DP.%26aulast%3DEvans%26aufirst%3DD.%2BB.%26atitle%3DATP%2520site-directed%2520competitive%2520and%2520irreversible%2520inhibitors%2520of%2520protein%2520kinases%26jtitle%3DMed.%2520Res.%2520Rev.%26date%3D2000%26volume%3D20%26spage%3D28%26epage%3D57" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group">Jones, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Willett, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Glen, R. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leach, A. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taylor, R.</span><span> </span><span class="NLM_article-title">Development and validation of a genetic algorithm for flexible docking</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_year" style="font-weight: bold;">1997</span> <span class="NLM_volume">267</span> <span class="NLM_fpage">727</span> <span class="NLM_lpage">748</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Fjm800791f&amp;key=10.1006%2Fjmbi.1996.0897" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Fjm800791f&amp;key=9126849" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Fjm800791f&amp;key=1%3ACAS%3A528%3ADyaK2sXis1KntLo%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=267&publication_year=1997&pages=727-748&author=G.+Jonesauthor=P.+Willettauthor=R.+C.+Glenauthor=A.+R.+Leachauthor=R.+Taylor&title=Development+and+validation+of+a+genetic+algorithm+for+flexible+docking"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Development and validation of a genetic algorithm for flexible docking</span></div><div class="casAuthors">Jones, Gareth; Willett, Peter; Glen, Robert C.; Leach, Andrew R.; Taylor, Robin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">267</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">727-748</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Academic</span>)
        </div><div class="casAbstract">Prediction of small mol. binding modes to macromols. of known three-dimensional structure is a problem of paramount importance in rational drug design (the "docking" problem).  We report the development and validation of the program GOLD (Genetic Optimization for Ligand Docking).  GOLD is an automated ligand docking program that uses a genetic algorithm to explore the full range of ligand conformational flexibility with partial flexibility of the protein and satisfies the fundamental requirement that the ligand must displace loosely bound water on binding.  Numerous enhancements and modifications have been applied to the original technique resulting in a substantial increase in the reliability and the applicability of the algorithm.  The advanced algorithm has been tested on a dataset of 100 complexes extd. from the Brookhaven Protein Data Bank.  When used to dock the ligand back into the binding site, GOLD achieved a 71% success rate in identifying the exptl. binding mode.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrP3I60Rl2uorVg90H21EOLACvtfcHk0lhR7IkNOzw8RA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXis1KntLo%253D&md5=476a2b1d8f80f3ba418052fe29d735ca</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1006%2Fjmbi.1996.0897&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1006%252Fjmbi.1996.0897%26sid%3Dliteratum%253Aachs%26aulast%3DJones%26aufirst%3DG.%26aulast%3DWillett%26aufirst%3DP.%26aulast%3DGlen%26aufirst%3DR.%2BC.%26aulast%3DLeach%26aufirst%3DA.%2BR.%26aulast%3DTaylor%26aufirst%3DR.%26atitle%3DDevelopment%2520and%2520validation%2520of%2520a%2520genetic%2520algorithm%2520for%2520flexible%2520docking%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D1997%26volume%3D267%26spage%3D727%26epage%3D748" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group">Warren, G. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Andrews, C. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Capelli, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clarke, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">LaLonde, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lambert, M. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lindvall, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nevins, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Semus, S. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Senger, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tedesco, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wall, I. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Woolven, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peishoff, C. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Head, M. S.</span><span> </span><span class="NLM_article-title">A critical assessment of docking programs and scoring functions</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_year" style="font-weight: bold;">2006</span> <span class="NLM_volume">49</span> <span class="NLM_fpage">5912</span> <span class="NLM_lpage">5931</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm050362n" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Fjm800791f&amp;key=1%3ACAS%3A528%3ADC%252BD2MXnslOrtrY%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2006&pages=5912-5931&author=G.+L.+Warrenauthor=C.+W.+Andrewsauthor=A.+M.+Capelliauthor=B.+Clarkeauthor=J.+LaLondeauthor=M.+H.+Lambertauthor=M.+Lindvallauthor=N.+Nevinsauthor=S.+F.+Semusauthor=S.+Sengerauthor=G.+Tedescoauthor=I.+D.+Wallauthor=J.+M.+Woolvenauthor=C.+E.+Peishoffauthor=M.+S.+Head&title=A+critical+assessment+of+docking+programs+and+scoring+functions"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">A Critical Assessment of Docking Programs and Scoring Functions</span></div><div class="casAuthors">Warren, Gregory L.; Andrews, C. Webster; Capelli, Anna-Maria; Clarke, Brian; LaLonde, Judith; Lambert, Millard H.; Lindvall, Mika; Nevins, Neysa; Semus, Simon F.; Senger, Stefan; Tedesco, Giovanna; Wall, Ian D.; Woolven, James M.; Peishoff, Catherine E.; Head, Martha S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">5912-5931</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Docking is a computational technique that samples conformations of small mols. in protein binding sites; scoring functions are used to assess which of these conformations best complements the protein binding site.  An evaluation of 10 docking programs and 37 scoring functions was conducted against eight proteins of seven protein types for three tasks: binding mode prediction, virtual screening for lead identification, and rank-ordering by affinity for lead optimization.  All of the docking programs were able to generate ligand conformations similar to crystallog. detd. protein/ligand complex structures for at least one of the targets.  However, scoring functions were less successful at distinguishing the crystallog. conformation from the set of docked poses.  Docking programs identified active compds. from a pharmaceutically relevant pool of decoy compds.; however, no single program performed well for all of the targets.  For prediction of compd. affinity, none of the docking programs or scoring functions made a useful prediction of ligand binding affinity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpfJn8_WHSjurVg90H21EOLACvtfcHk0ljalChIVWo_4A"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXnslOrtrY%253D&md5=463ab40c3d23e67b62df408fc5e7f08d</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1021%2Fjm050362n&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm050362n%26sid%3Dliteratum%253Aachs%26aulast%3DWarren%26aufirst%3DG.%2BL.%26aulast%3DAndrews%26aufirst%3DC.%2BW.%26aulast%3DCapelli%26aufirst%3DA.%2BM.%26aulast%3DClarke%26aufirst%3DB.%26aulast%3DLaLonde%26aufirst%3DJ.%26aulast%3DLambert%26aufirst%3DM.%2BH.%26aulast%3DLindvall%26aufirst%3DM.%26aulast%3DNevins%26aufirst%3DN.%26aulast%3DSemus%26aufirst%3DS.%2BF.%26aulast%3DSenger%26aufirst%3DS.%26aulast%3DTedesco%26aufirst%3DG.%26aulast%3DWall%26aufirst%3DI.%2BD.%26aulast%3DWoolven%26aufirst%3DJ.%2BM.%26aulast%3DPeishoff%26aufirst%3DC.%2BE.%26aulast%3DHead%26aufirst%3DM.%2BS.%26atitle%3DA%2520critical%2520assessment%2520of%2520docking%2520programs%2520and%2520scoring%2520functions%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2006%26volume%3D49%26spage%3D5912%26epage%3D5931" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group">Mohamadi, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richards, N. G. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guida, W. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liskamp, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lipton, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Caufield, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chang, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hendrickson, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Still, W. C.</span><span> </span><span class="NLM_article-title">Macromodel, an integrated software system for modeling organic and bioorganic molecules using molecular mechanics</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_year" style="font-weight: bold;">1990</span> <span class="NLM_volume">11</span> <span class="NLM_fpage">440</span> <span class="NLM_lpage">467</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=1990&pages=440-467&author=F.+Mohamadiauthor=N.+G.+J.+Richardsauthor=W.+C.+Guidaauthor=R.+Liskampauthor=M.+Liptonauthor=C.+Caufieldauthor=G.+Changauthor=T.+Hendricksonauthor=W.+C.+Still&title=Macromodel%2C+an+integrated+software+system+for+modeling+organic+and+bioorganic+molecules+using+molecular+mechanics"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMohamadi%26aufirst%3DF.%26aulast%3DRichards%26aufirst%3DN.%2BG.%2BJ.%26aulast%3DGuida%26aufirst%3DW.%2BC.%26aulast%3DLiskamp%26aufirst%3DR.%26aulast%3DLipton%26aufirst%3DM.%26aulast%3DCaufield%26aufirst%3DC.%26aulast%3DChang%26aufirst%3DG.%26aulast%3DHendrickson%26aufirst%3DT.%26aulast%3DStill%26aufirst%3DW.%2BC.%26atitle%3DMacromodel%252C%2520an%2520integrated%2520software%2520system%2520for%2520modeling%2520organic%2520and%2520bioorganic%2520molecules%2520using%2520molecular%2520mechanics%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D1990%26volume%3D11%26spage%3D440%26epage%3D467" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group">Lipinski, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lombardo, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dominy, B. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feeney, P. J.</span><span> </span><span class="NLM_article-title">Experimental and computational approaches to estimate solubility and permeability in drug discovery and development settings</span> <span class="citation_source-journal">Adv. Drug Delivery Rev.</span><span class="NLM_year" style="font-weight: bold;">1997</span> <span class="NLM_volume">23</span> <span class="NLM_fpage">3</span> <span class="NLM_lpage">25</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Fjm800791f&amp;key=10.1016%2FS0169-409X%2896%2900423-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Fjm800791f&amp;key=1%3ACAS%3A528%3ADyaK2sXktlKlsQ%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=1997&pages=3-25&author=C.+A.+Lipinskiauthor=F.+Lombardoauthor=B.+W.+Dominyauthor=P.+J.+Feeney&title=Experimental+and+computational+approaches+to+estimate+solubility+and+permeability+in+drug+discovery+and+development+settings"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Experimental and computational approaches to estimate solubility and permeability in drug discovery and development settings</span></div><div class="casAuthors">Lipinski, Christopher A.; Lombardo, Franco; Dominy, Beryl W.; Feeney, Paul J.</div><div class="citationInfo"><span class="NLM_cas:title">Advanced Drug Delivery Reviews</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">1-3</span>),
    <span class="NLM_cas:pages">3-25</span>CODEN:
                <span class="NLM_cas:coden">ADDREP</span>;
        ISSN:<span class="NLM_cas:issn">0169-409X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">A review with 50 refs.  Exptl. and computational approaches to est. soly. and permeability in discovery and development settings are described.  In the discovery setting 'the rule of 5' predicts that poor absorption or permeation is more likely when there are more than 5 H-bond donors, 10 H-bond acceptors, the mol. wt. (MWT) is >500 and the calcd. Log P (CLogP) is >5.  Computational methodol. for the rule-based Moriguchi Log P (MLogP) calcn. is described.  Turbidimetric soly. measurement is described and applied to known drugs.  High throughput screening (HTS) leads tend to have higher MWT and Log P and lower turbidimetric soly. than leads in the pre-HTS era.  In the development setting, soly. calcns. focus on exact value prediction and are difficult because of polymorphism.  Recent work on linear free energy relationships and Log P approaches are critically reviewed.  Useful predictions are possible in closely related analog series when coupled with exptl. thermodn. soly. measurements.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGon9gEeZ7cboLVg90H21EOLACvtfcHk0ljalChIVWo_4A"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXktlKlsQ%253D%253D&md5=405f70b0594d428f1275e1d56642cd3a</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1016%2FS0169-409X%2896%2900423-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0169-409X%252896%252900423-1%26sid%3Dliteratum%253Aachs%26aulast%3DLipinski%26aufirst%3DC.%2BA.%26aulast%3DLombardo%26aufirst%3DF.%26aulast%3DDominy%26aufirst%3DB.%2BW.%26aulast%3DFeeney%26aufirst%3DP.%2BJ.%26atitle%3DExperimental%2520and%2520computational%2520approaches%2520to%2520estimate%2520solubility%2520and%2520permeability%2520in%2520drug%2520discovery%2520and%2520development%2520settings%26jtitle%3DAdv.%2520Drug%2520Delivery%2520Rev.%26date%3D1997%26volume%3D23%26spage%3D3%26epage%3D25" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group">Veber, D. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johnson, S. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheng, H. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, B. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ward, K. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kopple, K. D.</span><span> </span><span class="NLM_article-title">Molecular properties that influence the oral bioavailability of drug candidates</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_year" style="font-weight: bold;">2002</span> <span class="NLM_volume">45</span> <span class="NLM_fpage">2615</span> <span class="NLM_lpage">2623</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm020017n" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Fjm800791f&amp;key=1%3ACAS%3A528%3ADC%252BD38XjsFCmt7g%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2002&pages=2615-2623&author=D.+F.+Veberauthor=S.+R.+Johnsonauthor=H.+Y.+Chengauthor=B.+R.+Smithauthor=K.+W.+Wardauthor=K.+D.+Kopple&title=Molecular+properties+that+influence+the+oral+bioavailability+of+drug+candidates"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular Properties That Influence the Oral Bioavailability of Drug Candidates</span></div><div class="casAuthors">Veber, Daniel F.; Johnson, Stephen R.; Cheng, Hung-Yuan; Smith, Brian R.; Ward, Keith W.; Kopple, Kenneth D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2615-2623</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Oral bioavailability measurements in rats for over 1100 drug candidates studied at Smith-Kline Beecham Pharmaceuticals (now Glaxo Smith-Kline) have allowed us to analyze the relative importance of mol. properties considered to influence that drug property.  Reduced mol. flexibility, as measured by the no. of rotatable bonds, and low polar surface area or total hydrogen bond count (sum of donors and acceptors) are found to be important predictors of good oral bioavailability, independent of mol. wt.  That on av. both the no. of rotatable bonds and polar surface area or hydrogen bond count tend to increase with mol. wt. may in part explain the success of the mol. wt. parameter in predicting oral bioavailability.  The commonly applied mol. wt. cutoff at 500 does not itself significantly sep. compds. with poor oral bioavailability from those with acceptable values in this extensive data set.  Our observations suggest that compds. which meet only the 2 criteria of (1) 10 or fewer rotatable bonds and (2) polar surface area â¤140 Ã2 (or 12 or fewer H-bond donors and acceptors) will have a high probability of good oral bioavailability in the rat.  Data sets for the artificial membrane permeation rate and for clearance in the rat were also examd.  Reduced polar surface area correlates better with increased permeation rate than does lipophilicity (C log P), and increased rotatable bond count has a neg. effect on the permeation rate.  A threshold permeation rate is a prerequisite of oral bioavailability.  The rotatable bond count does not correlate with the data examd. here for the in vivo clearance rate in the rat.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr3JtJEkO7yVLVg90H21EOLACvtfcHk0lh4kNNbuICWUA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XjsFCmt7g%253D&md5=eaad26ed6a259de82ad65a8834fc397d</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1021%2Fjm020017n&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm020017n%26sid%3Dliteratum%253Aachs%26aulast%3DVeber%26aufirst%3DD.%2BF.%26aulast%3DJohnson%26aufirst%3DS.%2BR.%26aulast%3DCheng%26aufirst%3DH.%2BY.%26aulast%3DSmith%26aufirst%3DB.%2BR.%26aulast%3DWard%26aufirst%3DK.%2BW.%26aulast%3DKopple%26aufirst%3DK.%2BD.%26atitle%3DMolecular%2520properties%2520that%2520influence%2520the%2520oral%2520bioavailability%2520of%2520drug%2520candidates%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2002%26volume%3D45%26spage%3D2615%26epage%3D2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group">Bellon, S. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kaplan-Lefko, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moriguchi, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rex, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johnson, C. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rose, P. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Long, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">OâConnor, A. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Coxon, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, T. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tasker, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Burgess, T. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dussault, I.</span><span> </span><span class="NLM_article-title">c-Met inhibitors with novel binding mode show activity against several hereditary papillary renal cell carcinoma related mutations</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_year" style="font-weight: bold;">2008</span> <span class="NLM_volume">283</span> <span class="NLM_fpage">2675</span> <span class="NLM_lpage">2683</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=283&publication_year=2008&pages=2675-2683&author=S.+F.+Bellonauthor=P.+Kaplan-Lefkoauthor=Y.+Yangauthor=Y.+Zhangauthor=J.+Moriguchiauthor=K.+Rexauthor=C.+W.+Johnsonauthor=P.+E.+Roseauthor=A.+M.+Longauthor=A.+B.+O%E2%80%99Connorauthor=Y.+Guauthor=A.+Coxonauthor=T.+S.+Kimauthor=A.+Taskerauthor=T.+L.+Burgessauthor=I.+Dussault&title=c-Met+inhibitors+with+novel+binding+mode+show+activity+against+several+hereditary+papillary+renal+cell+carcinoma+related+mutations"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBellon%26aufirst%3DS.%2BF.%26aulast%3DKaplan-Lefko%26aufirst%3DP.%26aulast%3DYang%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DMoriguchi%26aufirst%3DJ.%26aulast%3DRex%26aufirst%3DK.%26aulast%3DJohnson%26aufirst%3DC.%2BW.%26aulast%3DRose%26aufirst%3DP.%2BE.%26aulast%3DLong%26aufirst%3DA.%2BM.%26aulast%3DO%25E2%2580%2599Connor%26aufirst%3DA.%2BB.%26aulast%3DGu%26aufirst%3DY.%26aulast%3DCoxon%26aufirst%3DA.%26aulast%3DKim%26aufirst%3DT.%2BS.%26aulast%3DTasker%26aufirst%3DA.%26aulast%3DBurgess%26aufirst%3DT.%2BL.%26aulast%3DDussault%26aufirst%3DI.%26atitle%3Dc-Met%2520inhibitors%2520with%2520novel%2520binding%2520mode%2520show%2520activity%2520against%2520several%2520hereditary%2520papillary%2520renal%2520cell%2520carcinoma%2520related%2520mutations%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2008%26volume%3D283%26spage%3D2675%26epage%3D2683" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group">Albrecht, B. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Harmange, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bauer, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Berry, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bode, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boezio, A. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Choquette, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dussault, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fridrich, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hirai, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hoffman, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Larrow, J. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kaplan-Lefko, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lohman, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Long, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moriguchi, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">OâConnor, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Potashman, M. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reese, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rex, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Siegmund, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shah, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shimanovich, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Springer, S. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Teffera, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bellon, S. F.</span><span> </span><span class="NLM_article-title">Discovery and optimization of triazolopyridazines as potent and selective inhibitors of the c-Met kinase</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_year" style="font-weight: bold;">2008</span> <span class="NLM_volume">51</span> <span class="NLM_fpage">2879</span> <span class="NLM_lpage">2882</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm800043g" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Fjm800791f&amp;key=1%3ACAS%3A528%3ADC%252BD1cXkvFShtLk%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2008&pages=2879-2882&author=B.+K.+Albrechtauthor=J.+C.+Harmangeauthor=D.+Bauerauthor=L.+Berryauthor=C.+Bodeauthor=A.+A.+Boezioauthor=A.+Chenauthor=D.+Choquetteauthor=I.+Dussaultauthor=C.+Fridrichauthor=S.+Hiraiauthor=D.+Hoffmanauthor=J.+F.+Larrowauthor=P.+Kaplan-Lefkoauthor=J.+Linauthor=J.+Lohmanauthor=A.+M.+Longauthor=J.+Moriguchiauthor=A.+O%E2%80%98Connorauthor=M.+H.+Potashmanauthor=M.+Reeseauthor=K.+Rexauthor=A.+Siegmundauthor=K.+Shahauthor=R.+Shimanovichauthor=S.+K.+Springerauthor=Y.+Tefferaauthor=Y.+Yangauthor=Y.+Zhangauthor=S.+F.+Bellon&title=Discovery+and+optimization+of+triazolopyridazines+as+potent+and+selective+inhibitors+of+the+c-Met+kinase"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery and Optimization of Triazolopyridazines as Potent and Selective Inhibitors of the c-Met Kinase</span></div><div class="casAuthors">Albrecht, Brian K.; Harmange, Jean-Christophe; Bauer, David; Berry, Loren; Bode, Christiane; Boezio, Alessandro A.; Chen, April; Choquette, Deborah; Dussault, Isabelle; Fridrich, Cary; Hirai, Satoko; Hoffman, Doug; Larrow, Jay F.; Kaplan-Lefko, Paula; Lin, Jasmine; Lohman, Julia; Long, Alexander M.; Moriguchi, Jodi; O'Connor, Anne; Potashman, Michele H.; Reese, Monica; Rex, Karen; Siegmund, Aaron; Shah, Kavita; Shimanovich, Roman; Springer, Stephanie K.; Teffera, Yohannes; Yang, Yajing; Zhang, Yihong; Bellon, Steven F.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2879-2882</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Tumorigenesis is a multistep process in which oncogenes play a key role in tumor formation, growth, and maintenance.  MET was discovered as an oncogene that is activated by its ligand, hepatocyte growth factor.  Deregulated signaling in the c-Met pathway has been obsd. in multiple tumor types.  Herein we report the discovery of potent and selective triazolopyridazine small mols. that inhibit c-Met activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoVeQ8sYpasHLVg90H21EOLACvtfcHk0lh4kNNbuICWUA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXkvFShtLk%253D&md5=c0aec3ca102d90869884c139372f60a0</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1021%2Fjm800043g&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm800043g%26sid%3Dliteratum%253Aachs%26aulast%3DAlbrecht%26aufirst%3DB.%2BK.%26aulast%3DHarmange%26aufirst%3DJ.%2BC.%26aulast%3DBauer%26aufirst%3DD.%26aulast%3DBerry%26aufirst%3DL.%26aulast%3DBode%26aufirst%3DC.%26aulast%3DBoezio%26aufirst%3DA.%2BA.%26aulast%3DChen%26aufirst%3DA.%26aulast%3DChoquette%26aufirst%3DD.%26aulast%3DDussault%26aufirst%3DI.%26aulast%3DFridrich%26aufirst%3DC.%26aulast%3DHirai%26aufirst%3DS.%26aulast%3DHoffman%26aufirst%3DD.%26aulast%3DLarrow%26aufirst%3DJ.%2BF.%26aulast%3DKaplan-Lefko%26aufirst%3DP.%26aulast%3DLin%26aufirst%3DJ.%26aulast%3DLohman%26aufirst%3DJ.%26aulast%3DLong%26aufirst%3DA.%2BM.%26aulast%3DMoriguchi%26aufirst%3DJ.%26aulast%3DO%25E2%2580%2598Connor%26aufirst%3DA.%26aulast%3DPotashman%26aufirst%3DM.%2BH.%26aulast%3DReese%26aufirst%3DM.%26aulast%3DRex%26aufirst%3DK.%26aulast%3DSiegmund%26aufirst%3DA.%26aulast%3DShah%26aufirst%3DK.%26aulast%3DShimanovich%26aufirst%3DR.%26aulast%3DSpringer%26aufirst%3DS.%2BK.%26aulast%3DTeffera%26aufirst%3DY.%26aulast%3DYang%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DBellon%26aufirst%3DS.%2BF.%26atitle%3DDiscovery%2520and%2520optimization%2520of%2520triazolopyridazines%2520as%2520potent%2520and%2520selective%2520inhibitors%2520of%2520the%2520c-Met%2520kinase%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2008%26volume%3D51%26spage%3D2879%26epage%3D2882" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group">Zou, H. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, J. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arango, M. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McDonnell, S. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yamazaki, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Koudriakova, T. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Alton, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cui, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kung, P. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nambu, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Los, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bender, S. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mroczkowski, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Christensen, J. G.</span><span> </span><span class="NLM_article-title">An orally available small-molecule inhibitor of c-Met, PF-2341066, exhibits cytoreductive antitumor efficacy through antiproliferative and antiangiogenic mechanisms</span> <span class="citation_source-journal">Cancer Res.</span><span class="NLM_year" style="font-weight: bold;">2007</span> <span class="NLM_volume">67</span> <span class="NLM_fpage">4408</span> <span class="NLM_lpage">4417</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Fjm800791f&amp;key=10.1158%2F0008-5472.CAN-06-4443" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Fjm800791f&amp;key=17483355" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Fjm800791f&amp;key=1%3ACAS%3A528%3ADC%252BD2sXltVyrsbg%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2007&pages=4408-4417&author=H.+Y.+Zouauthor=Q.+Liauthor=J.+H.+Leeauthor=M.+E.+Arangoauthor=S.+R.+McDonnellauthor=S.+Yamazakiauthor=T.+B.+Koudriakovaauthor=G.+Altonauthor=J.+J.+Cuiauthor=P.+P.+Kungauthor=M.+D.+Nambuauthor=G.+Losauthor=S.+L.+Benderauthor=B.+Mroczkowskiauthor=J.+G.+Christensen&title=An+orally+available+small-molecule+inhibitor+of+c-Met%2C+PF-2341066%2C+exhibits+cytoreductive+antitumor+efficacy+through+antiproliferative+and+antiangiogenic+mechanisms"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">An Orally Available Small-Molecule Inhibitor of c-Met, PF-2341066, Exhibits Cytoreductive Antitumor Efficacy through Antiproliferative and Antiangiogenic Mechanisms</span></div><div class="casAuthors">Zou, Helen Y.; Li, Qiuhua; Lee, Joseph H.; Arango, Maria E.; McDonnell, Scott R.; Yamazaki, Shinji; Koudriakova, Tatiana B.; Alton, Gordon; Cui, Jingrong J.; Kung, Pei-Pei; Nambu, Mitchell D.; Los, Gerrit; Bender, Steven L.; Mroczkowski, Barbara; Christensen, James G.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">4408-4417</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">The c-Met receptor tyrosine kinase and its ligand, hepatocyte growth factor (HGF), have been implicated in the progression of several human cancers and are attractive therapeutic targets.  PF-2341066 was identified as a potent, orally bioavailable, ATP-competitive small-mol. inhibitor of the catalytic activity of c-Met kinase.  PF-2341066 was selective for c-Met (and anaplastic lymphoma kinase) compared with a panel of >120 diverse tyrosine and serine-threonine kinases.  PF-2341066 potently inhibited c-Met phosphorylation and c-Met-dependent proliferation, migration, or invasion of human tumor cells in vitro (IC50 values, 5-20 nmol/L).  In addn., PF-2341066 potently inhibited HGF-stimulated endothelial cell survival or invasion and serum-stimulated tubulogenesis in vitro, suggesting that this agent also exhibits antiangiogenic properties.  PF-2341066 showed efficacy at well-tolerated doses, including marked cytoreductive antitumor activity, in several tumor models that expressed activated c-Met.  The antitumor efficacy of PF-2341066 was dose dependent and showed a strong correlation to inhibition of c-Met phosphorylation in vivo.  Near-maximal inhibition of c-Met activity for the full dosing interval was necessary to maximize the efficacy of PF-2341066.  Addnl. mechanism-of-action studies showed dose-dependent inhibition of c-Met-dependent signal transduction, tumor cell proliferation (Ki67), induction of apoptosis (caspase-3), and redn. of microvessel d. (CD31).  These results indicated that the antitumor activity of PF-2341066 may be mediated by direct effects on tumor cell growth or survival as well as antiangiogenic mechanisms.  Collectively, these results show the therapeutic potential of targeting c-Met with selective small-mol. inhibitors for the treatment of human cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrEAWtGPOBHC7Vg90H21EOLACvtfcHk0lj9n_5h8rosnw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXltVyrsbg%253D&md5=798bf93e6cf8d5e9897e27485cba9002</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-06-4443&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-06-4443%26sid%3Dliteratum%253Aachs%26aulast%3DZou%26aufirst%3DH.%2BY.%26aulast%3DLi%26aufirst%3DQ.%26aulast%3DLee%26aufirst%3DJ.%2BH.%26aulast%3DArango%26aufirst%3DM.%2BE.%26aulast%3DMcDonnell%26aufirst%3DS.%2BR.%26aulast%3DYamazaki%26aufirst%3DS.%26aulast%3DKoudriakova%26aufirst%3DT.%2BB.%26aulast%3DAlton%26aufirst%3DG.%26aulast%3DCui%26aufirst%3DJ.%2BJ.%26aulast%3DKung%26aufirst%3DP.%2BP.%26aulast%3DNambu%26aufirst%3DM.%2BD.%26aulast%3DLos%26aufirst%3DG.%26aulast%3DBender%26aufirst%3DS.%2BL.%26aulast%3DMroczkowski%26aufirst%3DB.%26aulast%3DChristensen%26aufirst%3DJ.%2BG.%26atitle%3DAn%2520orally%2520available%2520small-molecule%2520inhibitor%2520of%2520c-Met%252C%2520PF-2341066%252C%2520exhibits%2520cytoreductive%2520antitumor%2520efficacy%2520through%2520antiproliferative%2520and%2520antiangiogenic%2520mechanisms%26jtitle%3DCancer%2520Res.%26date%3D2007%26volume%3D67%26spage%3D4408%26epage%3D4417" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group">Sattler, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pride, Y. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ma, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gramlich, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chu, S. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Quinnan, L. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shirazian, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liang, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Podar, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Christensen, J. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Salgia, R.</span><span> </span><span class="NLM_article-title">A novel small molecule met inhibitor induces apoptosis in cells transformed by the oncogenic TPR-MET tyrosine kinase</span> <span class="citation_source-journal">Cancer Res.</span><span class="NLM_year" style="font-weight: bold;">2003</span> <span class="NLM_volume">63</span> <span class="NLM_fpage">5462</span> <span class="NLM_lpage">5469</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Fjm800791f&amp;key=14500382" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Fjm800791f&amp;key=1%3ACAS%3A528%3ADC%252BD3sXntleku7s%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2003&pages=5462-5469&author=M.+Sattlerauthor=Y.+B.+Prideauthor=P.+Maauthor=J.+L.+Gramlichauthor=S.+C.+Chuauthor=L.+A.+Quinnanauthor=S.+Shirazianauthor=C.+Liangauthor=K.+Podarauthor=J.+G.+Christensenauthor=R.+Salgia&title=A+novel+small+molecule+met+inhibitor+induces+apoptosis+in+cells+transformed+by+the+oncogenic+TPR-MET+tyrosine+kinase"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">A Novel Small Molecule Met Inhibitor Induces Apoptosis in Cells Transformed by the Oncogenic TPR-MET Tyrosine Kinase</span></div><div class="casAuthors">Sattler, Martin; Pride, Yuri B.; Ma, Patrick; Gramlich, Jessica L.; Chu, Stephanie C.; Quinnan, Laura A.; Shirazian, Sheri; Liang, Congxin; Podar, Klaus; Christensen, James G.; Salgia, Ravi</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">5462-5469</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">The Met receptor tyrosine kinase has been shown to be overexpressed or mutated in a variety of solid tumors and has, therefore, been identified as a good candidate for molecularly targeted therapy.  Activation of the Met tyrosine kinase by the TPR gene was originally described in vitro through carcinogen-induced rearrangement.  The TPR-MET fusion protein contains constitutively elevated Met tyrosine kinase activity and constitutes an ideal model to study the transforming activity of the Met kinase.  We found, when introduced into an interleukin 3-dependent cell line, TPR-MET induces factor independence and constitutive tyrosine phosphorylation of several cellular proteins.  One major tyrosine phosphorylated protein was identified as the TPR-MET oncoprotein itself.  Inhibition of the Met kinase activity by the novel small mol. drug SU11274 [(3Z)-N-(3-chlorophenyl)-3-({3,5-dimethyl-4-[(4-methylpiperazin-1-yl)carbonyl]-1H-pyrrol-2-yl}methylene)-N-methyl-2-oxo-2,3-dihydro-1H-indole-5-sulfonamide] led to time- and dose-dependent reduced cell growth.  The inhibitor did not affect other tyrosine kinase oncoproteins, including BCR-ABL, TEL-JAK2, TEL-PDGFÎ²R, or TEL-ABL.  The Met inhibitor induced G1 cell cycle arrest and apoptosis with increased Annexin V staining and caspase 3 activity.  The autophosphorylation of the Met kinase was reduced on sites that have been shown previously to be important for activation of pathways involved in cell growth and survival, esp. the phosphatidylinositol-3'-kinase and the Ras pathway.  In particular, we found that the inhibitor blocked phosphorylation of AKT, GSK-3Î², and the pro-apoptotic transcription factor FKHR.  The characterization of SU11274 as an effective inhibitor of Met tyrosine kinase activity illustrates the potential of targeting for Met therapeutic use in cancers assocd. with activated forms of this kinase.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrGvIwRT4z1IbVg90H21EOLACvtfcHk0lj9n_5h8rosnw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXntleku7s%253D&md5=3223841b665e6b95f3dab0f18ce64dbe</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DSattler%26aufirst%3DM.%26aulast%3DPride%26aufirst%3DY.%2BB.%26aulast%3DMa%26aufirst%3DP.%26aulast%3DGramlich%26aufirst%3DJ.%2BL.%26aulast%3DChu%26aufirst%3DS.%2BC.%26aulast%3DQuinnan%26aufirst%3DL.%2BA.%26aulast%3DShirazian%26aufirst%3DS.%26aulast%3DLiang%26aufirst%3DC.%26aulast%3DPodar%26aufirst%3DK.%26aulast%3DChristensen%26aufirst%3DJ.%2BG.%26aulast%3DSalgia%26aufirst%3DR.%26atitle%3DA%2520novel%2520small%2520molecule%2520met%2520inhibitor%2520induces%2520apoptosis%2520in%2520cells%2520transformed%2520by%2520the%2520oncogenic%2520TPR-MET%2520tyrosine%2520kinase%26jtitle%3DCancer%2520Res.%26date%3D2003%26volume%3D63%26spage%3D5462%26epage%3D5469" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group">Christensen, J. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schreck, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Burrows, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuruganti, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chan, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Le, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ruslim, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blake, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lipson, K. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ramphal, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Do, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cui, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cherrington, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mendel, D. B.</span><span> </span><span class="NLM_article-title">A selective small molecule inhibitor of c-Met kinase inhibits c-Met-dependent phenotypes in vitro and exhibits cytoreductive antitumor activity in vivo</span> <span class="citation_source-journal">Cancer Res.</span><span class="NLM_year" style="font-weight: bold;">2003</span> <span class="NLM_volume">63</span> <span class="NLM_fpage">7345</span> <span class="NLM_lpage">7355</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2003&pages=7345-7355&author=J.+G.+Christensenauthor=R.+Schreckauthor=J.+Burrowsauthor=P.+Kurugantiauthor=E.+Chanauthor=P.+Leauthor=J.+Chenauthor=X.+Wangauthor=L.+Ruslimauthor=R.+Blakeauthor=K.+E.+Lipsonauthor=J.+Ramphalauthor=S.+Doauthor=J.+J.+Cuiauthor=J.+M.+Cherringtonauthor=D.+B.+Mendel&title=A+selective+small+molecule+inhibitor+of+c-Met+kinase+inhibits+c-Met-dependent+phenotypes+in+vitro+and+exhibits+cytoreductive+antitumor+activity+in+vivo"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DChristensen%26aufirst%3DJ.%2BG.%26aulast%3DSchreck%26aufirst%3DR.%26aulast%3DBurrows%26aufirst%3DJ.%26aulast%3DKuruganti%26aufirst%3DP.%26aulast%3DChan%26aufirst%3DE.%26aulast%3DLe%26aufirst%3DP.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DRuslim%26aufirst%3DL.%26aulast%3DBlake%26aufirst%3DR.%26aulast%3DLipson%26aufirst%3DK.%2BE.%26aulast%3DRamphal%26aufirst%3DJ.%26aulast%3DDo%26aufirst%3DS.%26aulast%3DCui%26aufirst%3DJ.%2BJ.%26aulast%3DCherrington%26aufirst%3DJ.%2BM.%26aulast%3DMendel%26aufirst%3DD.%2BB.%26atitle%3DA%2520selective%2520small%2520molecule%2520inhibitor%2520of%2520c-Met%2520kinase%2520inhibits%2520c-Met-dependent%2520phenotypes%2520in%2520vitro%2520and%2520exhibits%2520cytoreductive%2520antitumor%2520activity%2520in%2520vivo%26jtitle%3DCancer%2520Res.%26date%3D2003%26volume%3D63%26spage%3D7345%26epage%3D7355" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group">Dharmawardana, P. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Giubellino, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bottaro, D. P.</span><span> </span><span class="NLM_article-title">Hereditary papillary renal carcinoma type I</span> <span class="citation_source-journal">Curr. Mol. Med.</span><span class="NLM_year" style="font-weight: bold;">2004</span> <span class="NLM_volume">4</span> <span class="NLM_fpage">855</span> <span class="NLM_lpage">868</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Fjm800791f&amp;key=10.2174%2F1566524043359674" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Fjm800791f&amp;key=15579033" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Fjm800791f&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVGlsbzF" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2004&pages=855-868&author=P.+G.+Dharmawardanaauthor=A.+Giubellinoauthor=D.+P.+Bottaro&title=Hereditary+papillary+renal+carcinoma+type+I"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Hereditary papillary renal carcinoma type I</span></div><div class="casAuthors">Dharmawardana, Pathirage G.; Giubellino, Alessio; Bottaro, Donald P.</div><div class="citationInfo"><span class="NLM_cas:title">Current Molecular Medicine</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">855-868</span>CODEN:
                <span class="NLM_cas:coden">CMMUBP</span>;
        ISSN:<span class="NLM_cas:issn">1566-5240</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Germline missense mutations in the tyrosine kinase domain of the hepatocyte growth factor/scatter factor (HGF/SF) receptor, c-Met, are thought to be responsible for hereditary papillary renal carcinoma (HPRC) type 1, a form of human kidney cancer.  In addn. to extensive linkage anal. of HPRC families localizing the HPRC type 1 gene within chromosome 7, the demonstration that individual c-Met mutations reconstituted in cultured cells display enhanced and dysregulated kinase activity, and confer cell transformation and tumorigenicity in mice, solidifies this conclusion.  Our prior knowledge of HGF/SF biol. and c-Met signaling enabled rapid progress in unraveling the mol. pathogenesis of HPRC type 1, and in laying the framework for the development of novel therapeutics for the treatment of this cancer.  At the same time, the study of HPRC type 1 has refined our appreciation of the oncogenic potential of c-Met signaling, and challenges our current understanding of HGF/SF and c-Met function in health and disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrY89l4mqksLLVg90H21EOLACvtfcHk0lhyy4zya8WdCw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVGlsbzF&md5=cba15709d949ccfddcd52dccb7e88ffc</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.2174%2F1566524043359674&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F1566524043359674%26sid%3Dliteratum%253Aachs%26aulast%3DDharmawardana%26aufirst%3DP.%2BG.%26aulast%3DGiubellino%26aufirst%3DA.%26aulast%3DBottaro%26aufirst%3DD.%2BP.%26atitle%3DHereditary%2520papillary%2520renal%2520carcinoma%2520type%2520I%26jtitle%3DCurr.%2520Mol.%2520Med.%26date%3D2004%26volume%3D4%26spage%3D855%26epage%3D868" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group">Berthou, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aebersold, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schmidt, L. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stroka, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heigl, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Streit, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stalder, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gruber, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liang, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Howlett, A. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Candinas, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Greiner, R. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lipson, K. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zimmer, Y.</span><span> </span><span class="NLM_article-title">The Met kinase inhibitor SU11274 exhibits a selective inhibition pattern toward different receptor mutated variants</span> <span class="citation_source-journal">Oncogene</span><span class="NLM_year" style="font-weight: bold;">2004</span> <span class="NLM_volume">23</span> <span class="NLM_fpage">5387</span> <span class="NLM_lpage">5393</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2004&pages=5387-5393&author=S.+Berthouauthor=D.+M.+Aebersoldauthor=L.+S.+Schmidtauthor=D.+Strokaauthor=C.+Heiglauthor=B.+Streitauthor=D.+Stalderauthor=G.+Gruberauthor=C.+Liangauthor=A.+R.+Howlettauthor=D.+Candinasauthor=R.+H.+Greinerauthor=K.+E.+Lipsonauthor=Y.+Zimmer&title=The+Met+kinase+inhibitor+SU11274+exhibits+a+selective+inhibition+pattern+toward+different+receptor+mutated+variants"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBerthou%26aufirst%3DS.%26aulast%3DAebersold%26aufirst%3DD.%2BM.%26aulast%3DSchmidt%26aufirst%3DL.%2BS.%26aulast%3DStroka%26aufirst%3DD.%26aulast%3DHeigl%26aufirst%3DC.%26aulast%3DStreit%26aufirst%3DB.%26aulast%3DStalder%26aufirst%3DD.%26aulast%3DGruber%26aufirst%3DG.%26aulast%3DLiang%26aufirst%3DC.%26aulast%3DHowlett%26aufirst%3DA.%2BR.%26aulast%3DCandinas%26aufirst%3DD.%26aulast%3DGreiner%26aufirst%3DR.%2BH.%26aulast%3DLipson%26aufirst%3DK.%2BE.%26aulast%3DZimmer%26aufirst%3DY.%26atitle%3DThe%2520Met%2520kinase%2520inhibitor%2520SU11274%2520exhibits%2520a%2520selective%2520inhibition%2520pattern%2520toward%2520different%2520receptor%2520mutated%2520variants%26jtitle%3DOncogene%26date%3D2004%26volume%3D23%26spage%3D5387%26epage%3D5393" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group">Liu, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wen, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jorissen, R. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gilson, M. K.</span><span> </span><span class="NLM_article-title">BindingDB: a Web-accessible database of experimentally determined proteinâligand binding affinities</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_year" style="font-weight: bold;">2007</span> <span class="NLM_volume">35</span> <span class="NLM_fpage">D198</span> <span class="NLM_lpage">D201</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2007&pages=D198-D201&author=T.+Liuauthor=Y.+Linauthor=X.+Wenauthor=R.+N.+Jorissenauthor=M.+K.+Gilson&title=BindingDB%3A+a+Web-accessible+database+of+experimentally+determined+protein%E2%88%92ligand+binding+affinities"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DT.%26aulast%3DLin%26aufirst%3DY.%26aulast%3DWen%26aufirst%3DX.%26aulast%3DJorissen%26aufirst%3DR.%2BN.%26aulast%3DGilson%26aufirst%3DM.%2BK.%26atitle%3DBindingDB%253A%2520a%2520Web-accessible%2520database%2520of%2520experimentally%2520determined%2520protein%25E2%2588%2592ligand%2520binding%2520affinities%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2007%26volume%3D35%26spage%3DD198%26epage%3DD201" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group">Rogers, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, R. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hahn, M.</span><span> </span><span class="NLM_article-title">Using extended-connectivity fingerprints with Laplacian-modified Bayesian analysis in high-throughput screening follow-up</span> <span class="citation_source-journal">J. Biomol. Screening</span><span class="NLM_year" style="font-weight: bold;">2005</span> <span class="NLM_volume">10</span> <span class="NLM_fpage">682</span> <span class="NLM_lpage">686</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Fjm800791f&amp;key=10.1177%2F1087057105281365" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Fjm800791f&amp;key=16170046" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Fjm800791f&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtF2qs77L" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2005&pages=682-686&author=D.+Rogersauthor=R.+D.+Brownauthor=M.+Hahn&title=Using+extended-connectivity+fingerprints+with+Laplacian-modified+Bayesian+analysis+in+high-throughput+screening+follow-up"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Using extended-connectivity fingerprints with Laplacian-modified Bayesian analysis in high-throughput screening follow-up</span></div><div class="casAuthors">Rogers, David; Brown, Robert D.; Hahn, Mathew</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular Screening</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">682-686</span>CODEN:
                <span class="NLM_cas:coden">JBISF3</span>;
        ISSN:<span class="NLM_cas:issn">1087-0571</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">This article describes the use of a combination of extended-connectivity fingerprints (ECFPs) and Laplacian-modified Bayesian anal. in a study of the inhibition of Escherichia coli dihydrofolate reductase.  The McMaster High-Throughput Screening Lab at McMaster University proposed a competition to predict the hits in a sep. test set of 50,000 compds.  Although the problem seemed best approached with 3D methods, the authors show that 2D methods offer surprisingly competitive results with a low computational cost.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq9tK9Lo95V-7Vg90H21EOLACvtfcHk0ljiyECTc4D2Tw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtF2qs77L&md5=b8600cd46275811e81d40dcc2c438c99</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1177%2F1087057105281365&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1087057105281365%26sid%3Dliteratum%253Aachs%26aulast%3DRogers%26aufirst%3DD.%26aulast%3DBrown%26aufirst%3DR.%2BD.%26aulast%3DHahn%26aufirst%3DM.%26atitle%3DUsing%2520extended-connectivity%2520fingerprints%2520with%2520Laplacian-modified%2520Bayesian%2520analysis%2520in%2520high-throughput%2520screening%2520follow-up%26jtitle%3DJ.%2520Biomol.%2520Screening%26date%3D2005%26volume%3D10%26spage%3D682%26epage%3D686" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group">Willett, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barnard, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Downs, G. M.</span><span> </span><span class="NLM_article-title">Chemical similarity searching</span> <span class="citation_source-journal">J. Chem. Inf. Comput. Sci.</span><span class="NLM_year" style="font-weight: bold;">1998</span> <span class="NLM_volume">38</span> <span class="NLM_fpage">983</span> <span class="NLM_lpage">996</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci9800211" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Fjm800791f&amp;key=1%3ACAS%3A528%3ADyaK1cXks1ynu74%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=1998&pages=983-996&author=P.+Willettauthor=J.+M.+Barnardauthor=G.+M.+Downs&title=Chemical+similarity+searching"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Chemical similarity searching</span></div><div class="casAuthors">Willett, Peter; Barnard, John M.; Downs, Geoffrey M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Computer Sciences</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">983-996</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">0095-2338</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">This paper reviews the use of similarity searching in chem. databases.  It begins by introducing the concept of similarity searching, differentiating it from the more common substructure searching, and then discusses the current generation of fragment-based measures that are used for searching chem. structure databases.  The next sections focus upon two of the principal characteristics of a similarity measure: the coeff. that is used to quantify the degree of structural resemblance between pairs of mols. and the structural representations that are used to characterize mols. that are being compared in a similarity calcn.  New types of similarity measure are then compared with current approaches, and examples are given of several applications that are related to similarity searching.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpxjjSk407VCLVg90H21EOLACvtfcHk0ljDQxSKvKSChw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXks1ynu74%253D&md5=4233b0ea4edc052f0186cccbd5eae427</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1021%2Fci9800211&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci9800211%26sid%3Dliteratum%253Aachs%26aulast%3DWillett%26aufirst%3DP.%26aulast%3DBarnard%26aufirst%3DJ.%2BM.%26aulast%3DDowns%26aufirst%3DG.%2BM.%26atitle%3DChemical%2520similarity%2520searching%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Comput.%2520Sci.%26date%3D1998%26volume%3D38%26spage%3D983%26epage%3D996" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group">Mohammadi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Froum, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hamby, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schroeder, M. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Panek, R. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lu, G. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eliseenkova, A. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Green, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schlessinger, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hubbard, S. R.</span><span> </span><span class="NLM_article-title">Crystal structure of an angiogenesis inhibitor bound to the FGF receptor tyrosine kinase domain</span> <span class="citation_source-journal">EMBO J.</span><span class="NLM_year" style="font-weight: bold;">1998</span> <span class="NLM_volume">17</span> <span class="NLM_fpage">6896</span> <span class="NLM_lpage">5904</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=1998&pages=6896-5904&author=M.+Mohammadiauthor=S.+Froumauthor=J.+M.+Hambyauthor=M.+C.+Schroederauthor=R.+L.+Panekauthor=G.+H.+Luauthor=A.+V.+Eliseenkovaauthor=D.+Greenauthor=J.+Schlessingerauthor=S.+R.+Hubbard&title=Crystal+structure+of+an+angiogenesis+inhibitor+bound+to+the+FGF+receptor+tyrosine+kinase+domain"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMohammadi%26aufirst%3DM.%26aulast%3DFroum%26aufirst%3DS.%26aulast%3DHamby%26aufirst%3DJ.%2BM.%26aulast%3DSchroeder%26aufirst%3DM.%2BC.%26aulast%3DPanek%26aufirst%3DR.%2BL.%26aulast%3DLu%26aufirst%3DG.%2BH.%26aulast%3DEliseenkova%26aufirst%3DA.%2BV.%26aulast%3DGreen%26aufirst%3DD.%26aulast%3DSchlessinger%26aufirst%3DJ.%26aulast%3DHubbard%26aufirst%3DS.%2BR.%26atitle%3DCrystal%2520structure%2520of%2520an%2520angiogenesis%2520inhibitor%2520bound%2520to%2520the%2520FGF%2520receptor%2520tyrosine%2520kinase%2520domain%26jtitle%3DEMBO%2520J.%26date%3D1998%26volume%3D17%26spage%3D6896%26epage%3D5904" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group">Pautsch, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zoephel, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ahorn, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Spevak, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hauptmann, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nar, H.</span><span> </span><span class="NLM_article-title">Crystal structure of bisphosphorylated IGF-1 receptor kinase: insight into domain movements upon kinase activation</span> <span class="citation_source-journal">Structure</span><span class="NLM_year" style="font-weight: bold;">2001</span> <span class="NLM_volume">9</span> <span class="NLM_fpage">955</span> <span class="NLM_lpage">965</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2001&pages=955-965&author=A.+Pautschauthor=A.+Zoephelauthor=H.+Ahornauthor=W.+Spevakauthor=R.+Hauptmannauthor=H.+Nar&title=Crystal+structure+of+bisphosphorylated+IGF-1+receptor+kinase%3A+insight+into+domain+movements+upon+kinase+activation"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DPautsch%26aufirst%3DA.%26aulast%3DZoephel%26aufirst%3DA.%26aulast%3DAhorn%26aufirst%3DH.%26aulast%3DSpevak%26aufirst%3DW.%26aulast%3DHauptmann%26aufirst%3DR.%26aulast%3DNar%26aufirst%3DH.%26atitle%3DCrystal%2520structure%2520of%2520bisphosphorylated%2520IGF-1%2520receptor%2520kinase%253A%2520insight%2520into%2520domain%2520movements%2520upon%2520kinase%2520activation%26jtitle%3DStructure%26date%3D2001%26volume%3D9%26spage%3D955%26epage%3D965" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group">Brady, G. P.,  Jr.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stouten, P. F. W.</span><span> </span><span class="NLM_article-title">Fast prediction and visualization of protein binding pockets with PASS</span> <span class="citation_source-journal">J. Comput.-Aided. Mol. Des.</span><span class="NLM_year" style="font-weight: bold;">2000</span> <span class="NLM_volume">14</span> <span class="NLM_fpage">383</span> <span class="NLM_lpage">401</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Fjm800791f&amp;key=10.1023%2FA%3A1008124202956" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Fjm800791f&amp;key=10815774" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Fjm800791f&amp;key=1%3ACAS%3A528%3ADC%252BD3cXjt1GrtLs%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2000&pages=383-401&author=G.+P.+Bradyauthor=P.+F.+W.+Stouten&title=Fast+prediction+and+visualization+of+protein+binding+pockets+with+PASS"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Fast prediction and visualization of protein binding pockets with PASS</span></div><div class="casAuthors">Brady, G. Patrick, Jr.; Stouten, Pieter F. W.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer-Aided Molecular Design</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">383-401</span>CODEN:
                <span class="NLM_cas:coden">JCADEQ</span>;
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    
            (<span class="NLM_cas:orgname">Kluwer Academic Publishers</span>)
        </div><div class="casAbstract">PASS (Putative Active Sites with Spheres) is a simple computational tool that uses geometry to characterize regions of buried vol. in proteins and to identify positions likely to represent binding sites based upon the size, shape, and burial extent of these vols.  Its utility as a predictive tool for binding site identification is tested by predicting known binding sites of proteins in the PDB using both complexed macromols. and their corresponding apo-protein structures.  The results indicate that PASS can serve as a front-end to fast docking.  The main utility of PASS lies in the fact that it can analyze a moderate-size protein (â¼30 kDa) in under 20 s, which makes it suitable for interactive mol. modeling, protein database anal., and aggressive virtual screening efforts.  As a modeling tool, PASS (i) rapidly identifies favorable regions of the protein surface, (ii) simplifies visualization of residues modulating binding in these regions, and (iii) provides a means of directly visualizing buried vol., which is often inferred indirectly from curvature in a surface representation.  PASS produces output in the form of std. PDB files, which are suitable for any modeling package, and provides script files to simplify visualization in Cerius2, InsightII, MOE, Quanta, RasMol, and Sybyl.  PASS is freely available to all.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq6J6I5HqOUY7Vg90H21EOLACvtfcHk0ljp3wfSo5P2cg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXjt1GrtLs%253D&md5=8a444f703004aa07f5258e919068eb03</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1023%2FA%3A1008124202956&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1023%252FA%253A1008124202956%26sid%3Dliteratum%253Aachs%26aulast%3DBrady%26aufirst%3DG.%2BP.%26aulast%3DStouten%26aufirst%3DP.%2BF.%2BW.%26atitle%3DFast%2520prediction%2520and%2520visualization%2520of%2520protein%2520binding%2520pockets%2520with%2520PASS%26jtitle%3DJ.%2520Comput.-Aided.%2520Mol.%2520Des.%26date%3D2000%26volume%3D14%26spage%3D383%26epage%3D401" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group">Liu, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Neiss, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Henne-Bruns, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Korc, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bachem, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kornmann, M.</span><span> </span><span class="NLM_article-title">Identification of a fibroblast growth factor receptor 1 splice variant that inhibits pancreatic cancer cell growth</span> <span class="citation_source-journal">Cancer Res.</span><span class="NLM_year" style="font-weight: bold;">2007</span> <span class="NLM_volume">67</span> <span class="NLM_fpage">2712</span> <span class="NLM_lpage">2719</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2007&pages=2712-2719&author=Z.+Liuauthor=N.+Neissauthor=S.+Zhouauthor=D.+Henne-Brunsauthor=M.+Korcauthor=M.+Bachemauthor=M.+Kornmann&title=Identification+of+a+fibroblast+growth+factor+receptor+1+splice+variant+that+inhibits+pancreatic+cancer+cell+growth"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DZ.%26aulast%3DNeiss%26aufirst%3DN.%26aulast%3DZhou%26aufirst%3DS.%26aulast%3DHenne-Bruns%26aufirst%3DD.%26aulast%3DKorc%26aufirst%3DM.%26aulast%3DBachem%26aufirst%3DM.%26aulast%3DKornmann%26aufirst%3DM.%26atitle%3DIdentification%2520of%2520a%2520fibroblast%2520growth%2520factor%2520receptor%25201%2520splice%2520variant%2520that%2520inhibits%2520pancreatic%2520cancer%2520cell%2520growth%26jtitle%3DCancer%2520Res.%26date%3D2007%26volume%3D67%26spage%3D2712%26epage%3D2719" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group">Kondo, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zheng, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kurebayashi, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Asa, S. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ezzat, S.</span><span> </span><span class="NLM_article-title">Epigenetically controlled fibroblast growth factor receptor 2 signaling imposes on the RAS/BRAF/mitogen-activated protein kinase pathway to modulate thyroid cancer progression</span> <span class="citation_source-journal">Cancer Res.</span><span class="NLM_year" style="font-weight: bold;">2007</span> <span class="NLM_volume">67</span> <span class="NLM_fpage">5461</span> <span class="NLM_lpage">5470</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2007&pages=5461-5470&author=T.+Kondoauthor=L.+Zhengauthor=W.+Liuauthor=J.+Kurebayashiauthor=S.+L.+Asaauthor=S.+Ezzat&title=Epigenetically+controlled+fibroblast+growth+factor+receptor+2+signaling+imposes+on+the+RAS%2FBRAF%2Fmitogen-activated+protein+kinase+pathway+to+modulate+thyroid+cancer+progression"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DKondo%26aufirst%3DT.%26aulast%3DZheng%26aufirst%3DL.%26aulast%3DLiu%26aufirst%3DW.%26aulast%3DKurebayashi%26aufirst%3DJ.%26aulast%3DAsa%26aufirst%3DS.%2BL.%26aulast%3DEzzat%26aufirst%3DS.%26atitle%3DEpigenetically%2520controlled%2520fibroblast%2520growth%2520factor%2520receptor%25202%2520signaling%2520imposes%2520on%2520the%2520RAS%252FBRAF%252Fmitogen-activated%2520protein%2520kinase%2520pathway%2520to%2520modulate%2520thyroid%2520cancer%2520progression%26jtitle%3DCancer%2520Res.%26date%3D2007%26volume%3D67%26spage%3D5461%26epage%3D5470" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group">Elbauomy, E. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Green, A. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lambros, M. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Turner, N. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grainge, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Powe, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ellis, I. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reis-Filho, J. S.</span><span> </span><span class="NLM_article-title">FGFR1 amplification in breast carcinomas: a chromogenic in situ hybridisation analysis</span> <span class="citation_source-journal">Breast Cancer Res.</span><span class="NLM_year" style="font-weight: bold;">2007</span> <span class="NLM_volume">9</span> <span class="NLM_fpage">R23</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2007&pages=R23&author=E.+S.+Elbauomyauthor=A.+R.+Greenauthor=M.+B.+Lambrosauthor=N.+C.+Turnerauthor=M.+J.+Graingeauthor=D.+Poweauthor=I.+O.+Ellisauthor=J.+S.+Reis-Filho&title=FGFR1+amplification+in+breast+carcinomas%3A+a+chromogenic+in+situ+hybridisation+analysis"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DElbauomy%26aufirst%3DE.%2BS.%26aulast%3DGreen%26aufirst%3DA.%2BR.%26aulast%3DLambros%26aufirst%3DM.%2BB.%26aulast%3DTurner%26aufirst%3DN.%2BC.%26aulast%3DGrainge%26aufirst%3DM.%2BJ.%26aulast%3DPowe%26aufirst%3DD.%26aulast%3DEllis%26aufirst%3DI.%2BO.%26aulast%3DReis-Filho%26aufirst%3DJ.%2BS.%26atitle%3DFGFR1%2520amplification%2520in%2520breast%2520carcinomas%253A%2520a%2520chromogenic%2520in%2520situ%2520hybridisation%2520analysis%26jtitle%3DBreast%2520Cancer%2520Res.%26date%3D2007%26volume%3D9%26spage%3DR23" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group">Sahadevan, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Darby, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leung, H. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mathers, M. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Robson, C. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gnanapragasam, V. J.</span><span> </span><span class="NLM_article-title">Selective over-expression of fibroblast growth factor receptors 1 and 4 in clinical prostate cancer</span> <span class="citation_source-journal">J. Pathol.</span><span class="NLM_year" style="font-weight: bold;">2007</span> <span class="NLM_volume">213</span> <span class="NLM_fpage">82</span> <span class="NLM_lpage">90</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Fjm800791f&amp;key=10.1002%2Fpath.2205" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Fjm800791f&amp;key=17607666" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Fjm800791f&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtFSksLvF" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=213&publication_year=2007&pages=82-90&author=K.+Sahadevanauthor=S.+Darbyauthor=H.+Y.+Leungauthor=M.+E.+Mathersauthor=C.+N.+Robsonauthor=V.+J.+Gnanapragasam&title=Selective+over-expression+of+fibroblast+growth+factor+receptors+1+and+4+in+clinical+prostate+cancer"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Selective over-expression of fibroblast growth factor receptors 1 and 4 in clinical prostate cancer</span></div><div class="casAuthors">Sahadevan, K.; Darby, S.; Leung, H. Y.; Mathers, M. E.; Robson, C. N.; Gnanapragasam, V. J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Pathology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">213</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">82-90</span>CODEN:
                <span class="NLM_cas:coden">JPTLAS</span>;
        ISSN:<span class="NLM_cas:issn">0022-3417</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons Ltd.</span>)
        </div><div class="casAbstract">Fibroblast growth factor receptors (FGFRs) mediate the tumorigenic effects of FGFs in prostate cancer.  These receptors are therefore potential therapeutic targets in the development of inhibitors to this pathway.  To identify the most relevant targets, we simultaneously investigated FGFR1-4 expression using a prostate cancer tissue microarray (TMA) and in laser capture microdissected (LCM) prostate epithelial cells.  In malignant prostates (n = 138) we obsd. significant FGFR1 and FGFR4 protein over-expression in comparison with benign prostates (n = 58; p < 0.0001).  FGFR1 was expressed at high levels in the majority of tumors (69% of grade 3 or less, 74% of grade 4 and 70% of grade 5), while FGFR4 was strongly expressed in 83% of grade 5 cancers but in only 25% of grade 1-3 cancers (p < 0.0001).  At the transcript level we obsd. a similar pattern, with FGFR1 and FGFR4 mRNA over-expressed in malignant epithelial cells compared to benign cells (p < 0.0005 and p < 0.05, resp.).  While total FGFR2 was increased in some cancers, there was no assocn. between expression and tumor grade or stage.  Transcript anal., however, revealed a switch in the predominant isoform expressed from FGFR2IIIb to FGFR2IIIc among malignant epithelial cells.  In contrast, protein and transcript expression of FGFR3 was very similar between benign and cancer biopsies.  The functional effect of targeting FGFR4 in prostate cancer cells has not previously been investigated.  In in vitro expts., suppression of FGFR4 by RNA interference effectively blocked prostate cancer cell proliferation (p < 0.0001) and invasion (p < 0.001) in response to exogenous stimulation.  This effect was evident regardless of whether the cells expressed the FGFR4 Arg388 or Gly388 allele.  In parallel expts., FGFR3 suppression had no discernible effect on cancer cell behavior.  These results suggest evidence of selective over-expression of FGFR1 and FGFR4 in clin. prostate cancer and support the notion of targeted inhibition of these receptors to disrupt FGF signalling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrfun6Hyw60Z7Vg90H21EOLACvtfcHk0lhyRz7OjtUwBw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtFSksLvF&md5=3729c3a83c8c3fdf634220c5fe1a5992</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1002%2Fpath.2205&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fpath.2205%26sid%3Dliteratum%253Aachs%26aulast%3DSahadevan%26aufirst%3DK.%26aulast%3DDarby%26aufirst%3DS.%26aulast%3DLeung%26aufirst%3DH.%2BY.%26aulast%3DMathers%26aufirst%3DM.%2BE.%26aulast%3DRobson%26aufirst%3DC.%2BN.%26aulast%3DGnanapragasam%26aufirst%3DV.%2BJ.%26atitle%3DSelective%2520over-expression%2520of%2520fibroblast%2520growth%2520factor%2520receptors%25201%2520and%25204%2520in%2520clinical%2520prostate%2520cancer%26jtitle%3DJ.%2520Pathol.%26date%3D2007%26volume%3D213%26spage%3D82%26epage%3D90" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group">Ayers, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fargnoli, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lewin, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Platero, J. S.</span><span> </span><span class="NLM_article-title">Discovery and validation of biomarkers that respond to treatment with brivanib alaninate, a small-molecule VEGFR-2/FGFR-1 antagonist</span> <span class="citation_source-journal">Cancer Res.</span><span class="NLM_year" style="font-weight: bold;">2007</span> <span class="NLM_volume">67</span> <span class="NLM_fpage">6899</span> <span class="NLM_lpage">6906</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2007&pages=6899-6906&author=M.+Ayersauthor=J.+Fargnoliauthor=A.+Lewinauthor=Q.+Wuauthor=J.+S.+Platero&title=Discovery+and+validation+of+biomarkers+that+respond+to+treatment+with+brivanib+alaninate%2C+a+small-molecule+VEGFR-2%2FFGFR-1+antagonist"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DAyers%26aufirst%3DM.%26aulast%3DFargnoli%26aufirst%3DJ.%26aulast%3DLewin%26aufirst%3DA.%26aulast%3DWu%26aufirst%3DQ.%26aulast%3DPlatero%26aufirst%3DJ.%2BS.%26atitle%3DDiscovery%2520and%2520validation%2520of%2520biomarkers%2520that%2520respond%2520to%2520treatment%2520with%2520brivanib%2520alaninate%252C%2520a%2520small-molecule%2520VEGFR-2%252FFGFR-1%2520antagonist%26jtitle%3DCancer%2520Res.%26date%3D2007%26volume%3D67%26spage%3D6899%26epage%3D6906" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group">Bean, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brennan, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shih, J. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Riely, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Viale, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chitale, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Motoi, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Szoke, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Broderick, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Balak, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chang, W. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yu, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gazdar, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pass, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rusch, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gerald, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, S. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, P. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miller, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ladanyi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pao, W.</span><span> </span><span class="NLM_article-title">MET amplification occurs with or without T790M mutations in EGFR mutant lung tumors with acquired resistance to gefitinib or erlotinib</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U.S.A.</span><span class="NLM_year" style="font-weight: bold;">2007</span> <span class="NLM_volume">104</span> <span class="NLM_fpage">20932</span> <span class="NLM_lpage">20937</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=104&publication_year=2007&pages=20932-20937&author=J.+Beanauthor=C.+Brennanauthor=J.+Y.+Shihauthor=G.+Rielyauthor=A.+Vialeauthor=L.+Wangauthor=D.+Chitaleauthor=N.+Motoiauthor=J.+Szokeauthor=S.+Broderickauthor=M.+Balakauthor=W.+C.+Changauthor=C.+J.+Yuauthor=A.+Gazdarauthor=H.+Passauthor=V.+Ruschauthor=W.+Geraldauthor=S.+F.+Huangauthor=P.+C.+Yangauthor=V.+Millerauthor=M.+Ladanyiauthor=C.+H.+Yangauthor=W.+Pao&title=MET+amplification+occurs+with+or+without+T790M+mutations+in+EGFR+mutant+lung+tumors+with+acquired+resistance+to+gefitinib+or+erlotinib"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBean%26aufirst%3DJ.%26aulast%3DBrennan%26aufirst%3DC.%26aulast%3DShih%26aufirst%3DJ.%2BY.%26aulast%3DRiely%26aufirst%3DG.%26aulast%3DViale%26aufirst%3DA.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DChitale%26aufirst%3DD.%26aulast%3DMotoi%26aufirst%3DN.%26aulast%3DSzoke%26aufirst%3DJ.%26aulast%3DBroderick%26aufirst%3DS.%26aulast%3DBalak%26aufirst%3DM.%26aulast%3DChang%26aufirst%3DW.%2BC.%26aulast%3DYu%26aufirst%3DC.%2BJ.%26aulast%3DGazdar%26aufirst%3DA.%26aulast%3DPass%26aufirst%3DH.%26aulast%3DRusch%26aufirst%3DV.%26aulast%3DGerald%26aufirst%3DW.%26aulast%3DHuang%26aufirst%3DS.%2BF.%26aulast%3DYang%26aufirst%3DP.%2BC.%26aulast%3DMiller%26aufirst%3DV.%26aulast%3DLadanyi%26aufirst%3DM.%26aulast%3DYang%26aufirst%3DC.%2BH.%26aulast%3DPao%26aufirst%3DW.%26atitle%3DMET%2520amplification%2520occurs%2520with%2520or%2520without%2520T790M%2520mutations%2520in%2520EGFR%2520mutant%2520lung%2520tumors%2520with%2520acquired%2520resistance%2520to%2520gefitinib%2520or%2520erlotinib%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2007%26volume%3D104%26spage%3D20932%26epage%3D20937" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group">Ma, P. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schaefer, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Christensen, J. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Salgia, R.</span><span> </span><span class="NLM_article-title">A selective small molecule c-MET Inhibitor, PHA665752, cooperates with rapamycin</span> <span class="citation_source-journal">Clin. Cancer Res.</span><span class="NLM_year" style="font-weight: bold;">2005</span> <span class="NLM_volume">11</span> <span class="NLM_fpage">2312</span> <span class="NLM_lpage">2319</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Fjm800791f&amp;key=10.1158%2F1078-0432.CCR-04-1708" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Fjm800791f&amp;key=15788682" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Fjm800791f&amp;key=1%3ACAS%3A528%3ADC%252BD2MXisFaju7c%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2005&pages=2312-2319&author=P.+C.+Maauthor=E.+Schaeferauthor=J.+G.+Christensenauthor=R.+Salgia&title=A+selective+small+molecule+c-MET+Inhibitor%2C+PHA665752%2C+cooperates+with+rapamycin"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">A selective small molecule c-MET inhibitor, PHA665752, cooperates with rapamycin</span></div><div class="casAuthors">Ma, Patrick C.; Schaefer, Erik; Christensen, James G.; Salgia, Ravi</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2312-2319</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">c-MET is believed to be an attractive receptor target for mol. therapeutic inhibition.  TPR-MET, a constitutively active oncogenic variant of MET, serves as excellent model for testing c-MET inhibitors.  Here, we characterized a small mol. c-MET inhibitor, PHA665752, and tested its cooperation with the mammalian target of rapamycin inhibitor as potential targeted therapy.  The effect of PHA665752 treatment was detd. on cell growth, motility and migration, apoptosis, and cell-cycle arrest of TPR-MET-transformed cells.  Moreover, the effect of PHA665752 on the phosphorylation on MET, as well as its downstream effectors, p-AKT and p-S6K, was also detd.  Finally, growth of TPR-MET-transformed cells was tested in the presence of PHA665752 and rapamycin.  H441 non-small cell lung cancer (NSCLC) cells (with activated c-Met) were also tested against both PHA665752 and rapamycin.  PHA665752 specifically inhibited cell growth in BaF3.  TPR-MET cells (IC50 < 0.06 Î¼mol/L), induced apoptosis and cell cycle arrest.  Constitutive cell motility and migration of the BaF3.  TPR-MET cells was also inhibited.  PHA665752 inhibited specific phosphorylation of TPR-MET as well as phosphorylation of downstream targets of the mammalian target of rapamycin pathway.  When combined with PHA665752, rapamycin showed cooperative inhibition to reduce growth of BaF3.  TPR-MET- and c-MET-expressing H441 NSCLC cells.  PHA665752 is a potent small mol.-selective c-MET inhibitor and is highly active against TPR-MET-transformed cells both biol. and biochem.  PHA665752 is also active against H441 NSCLC cells.  The c-MET inhibitor can cooperate with rapamycin in therapeutic inhibition of NSCLC, and in vivo studies of this combination against c-MET expressing cancers would be merited.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrQ6fVYSbU8ibVg90H21EOLACvtfcHk0lgspEftM6KjBQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXisFaju7c%253D&md5=9e69f0253f675f4a1aeb96c115e2e420</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1158%2F1078-0432.CCR-04-1708&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1078-0432.CCR-04-1708%26sid%3Dliteratum%253Aachs%26aulast%3DMa%26aufirst%3DP.%2BC.%26aulast%3DSchaefer%26aufirst%3DE.%26aulast%3DChristensen%26aufirst%3DJ.%2BG.%26aulast%3DSalgia%26aufirst%3DR.%26atitle%3DA%2520selective%2520small%2520molecule%2520c-MET%2520Inhibitor%252C%2520PHA665752%252C%2520cooperates%2520with%2520rapamycin%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D2005%26volume%3D11%26spage%3D2312%26epage%3D2319" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group">Ma, P. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jagadeeswaran, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jagadeesh, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tretiakova, M. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nallasura, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fox, E. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hansen, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schaefer, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Naoki, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lader, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richards, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sugarbaker, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Husain, A. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Christensen, J. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Salgia, R.</span><span> </span><span class="NLM_article-title">Functional expression and mutations of c-Met and its therapeutic inhibition with SU11274 and small interfering RNA in non-small cell lung cancer</span> <span class="citation_source-journal">Cancer Res.</span><span class="NLM_year" style="font-weight: bold;">2005</span> <span class="NLM_volume">65</span> <span class="NLM_fpage">1479</span> <span class="NLM_lpage">1488</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=65&publication_year=2005&pages=1479-1488&author=P.+C.+Maauthor=R.+Jagadeeswaranauthor=S.+Jagadeeshauthor=M.+S.+Tretiakovaauthor=V.+Nallasuraauthor=E.+A.+Foxauthor=M.+Hansenauthor=E.+Schaeferauthor=K.+Naokiauthor=A.+Laderauthor=W.+Richardsauthor=D.+Sugarbakerauthor=A.+N.+Husainauthor=J.+G.+Christensenauthor=R.+Salgia&title=Functional+expression+and+mutations+of+c-Met+and+its+therapeutic+inhibition+with+SU11274+and+small+interfering+RNA+in+non-small+cell+lung+cancer"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMa%26aufirst%3DP.%2BC.%26aulast%3DJagadeeswaran%26aufirst%3DR.%26aulast%3DJagadeesh%26aufirst%3DS.%26aulast%3DTretiakova%26aufirst%3DM.%2BS.%26aulast%3DNallasura%26aufirst%3DV.%26aulast%3DFox%26aufirst%3DE.%2BA.%26aulast%3DHansen%26aufirst%3DM.%26aulast%3DSchaefer%26aufirst%3DE.%26aulast%3DNaoki%26aufirst%3DK.%26aulast%3DLader%26aufirst%3DA.%26aulast%3DRichards%26aufirst%3DW.%26aulast%3DSugarbaker%26aufirst%3DD.%26aulast%3DHusain%26aufirst%3DA.%2BN.%26aulast%3DChristensen%26aufirst%3DJ.%2BG.%26aulast%3DSalgia%26aufirst%3DR.%26atitle%3DFunctional%2520expression%2520and%2520mutations%2520of%2520c-Met%2520and%2520its%2520therapeutic%2520inhibition%2520with%2520SU11274%2520and%2520small%2520interfering%2520RNA%2520in%2520non-small%2520cell%2520lung%2520cancer%26jtitle%3DCancer%2520Res.%26date%3D2005%26volume%3D65%26spage%3D1479%26epage%3D1488" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">Virtual screening of chemical libraries</span> <span class="citation_source-journal">Nature</span><span class="NLM_year" style="font-weight: bold;">2004</span> <span class="NLM_volume">432</span> <span class="NLM_fpage">862</span> <span class="NLM_lpage">865</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Fjm800791f&amp;key=10.1038%2Fnature03197" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Fjm800791f&amp;key=15602552" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Fjm800791f&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVOht7jK" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=432&publication_year=2004&pages=862-865&author=B.+K.+Shoichet&title=Virtual+screening+of+chemical+libraries"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Virtual screening of chemical libraries</span></div><div class="casAuthors">Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">432</span>
        (<span class="NLM_cas:issue">7019</span>),
    <span class="NLM_cas:pages">862-865</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Virtual screening uses computer-based methods to discover new ligands on the basis of biol. structures.  Although widely heralded in the 1970s and 1980s, the technique has since struggled to meet its initial promise, and drug discovery remains dominated by empirical screening.  Recent successes in predicting new ligands and their receptor-bound structures, and better rates of ligand discovery compared to empirical screening, have reignited interest in virtual screening, which is now widely used in drug discovery, albeit on a more limited scale than empirical screening.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrSY9IbbFldkrVg90H21EOLACvtfcHk0lizJLdi5bqtwQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVOht7jK&md5=e351e226e8985d7c304a05fb67dfdc60</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1038%2Fnature03197&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature03197%26sid%3Dliteratum%253Aachs%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DVirtual%2520screening%2520of%2520chemical%2520libraries%26jtitle%3DNature%26date%3D2004%26volume%3D432%26spage%3D862%26epage%3D865" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group">Klebe, G.</span><span> </span><span class="NLM_article-title">Virtual ligand screening: strategies, perspectives and limitations</span> <span class="citation_source-journal">Drug Discovery Today</span><span class="NLM_year" style="font-weight: bold;">2006</span> <span class="NLM_volume">11</span> <span class="NLM_fpage">580</span> <span class="NLM_lpage">594</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Fjm800791f&amp;key=10.1016%2Fj.drudis.2006.05.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Fjm800791f&amp;key=16793526" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Fjm800791f&amp;key=1%3ACAS%3A528%3ADC%252BD28XlvFGqtLo%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2006&pages=580-594&author=G.+Klebe&title=Virtual+ligand+screening%3A+strategies%2C+perspectives+and+limitations"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Virtual ligand screening: strategies, perspectives and limitations</span></div><div class="casAuthors">Klebe, Gerhard</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">13 & 14</span>),
    <span class="NLM_cas:pages">580-594</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  In contrast to high-throughput screening, in virtual ligand screening (VS), compds. are selected using computer programs to predict their binding to a target receptor.  A key prerequisite is knowledge about the spatial and energetic criteria responsible for protein-ligand binding.  The concepts and prerequisites to perform VS are summarized here, and explanations are sought for the enduring limitations of the technol.  Target selection, anal. and prepn. are discussed, as well as considerations about the compilation of candidate ligand libraries.  The tools and strategies of a VS campaign, and the accuracy of scoring and ranking of the results, are also considered.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrN3dWlP-WDY7Vg90H21EOLACvtfcHk0lgQxBLQ2kicJg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XlvFGqtLo%253D&md5=5adfd125d48082238a6c5ad0e8343c59</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2006.05.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2006.05.012%26sid%3Dliteratum%253Aachs%26aulast%3DKlebe%26aufirst%3DG.%26atitle%3DVirtual%2520ligand%2520screening%253A%2520strategies%252C%2520perspectives%2520and%2520limitations%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2006%26volume%3D11%26spage%3D580%26epage%3D594" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group">Ihlenfeldt, W. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Takahashi, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abe, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sasaki, S.</span><span> </span><span class="NLM_article-title">Computation and management of chemical properties in CACTVS: an extensible networked approach toward modularity and compatibility</span> <span class="citation_source-journal">J. Chem. Inf. Comput. Sci.</span><span class="NLM_year" style="font-weight: bold;">1994</span> <span class="NLM_volume">34</span> <span class="NLM_fpage">109</span> <span class="NLM_lpage">116</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci00017a013" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Fjm800791f&amp;key=1%3ACAS%3A528%3ADyaK2cXns1GrtQ%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=1994&pages=109-116&author=W.+D.+Ihlenfeldtauthor=Y.+Takahashiauthor=H.+Abeauthor=S.+Sasaki&title=Computation+and+management+of+chemical+properties+in+CACTVS%3A+an+extensible+networked+approach+toward+modularity+and+compatibility"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Computation and management of chemical properties in CACTVS: An extensible networked approach toward modularity and compatibility</span></div><div class="casAuthors">Ihlenfeldt, Wolf Dietrich; Takahashi, Yoshimasa; Abe, Hidetsugu; Sasaki, S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Computer Sciences</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">34</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">109-16</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">0095-2338</span>.
    </div><div class="casAbstract">The data model of the CACTVS program suite (Chem. Algorithms Construction, Threading, and Verification System) is presented.  CACTVS is an open environment which readily integrates new computational modules and data format descriptions in the domain of computational chem.  These chem. information (property) computation modules are provided with requested input data, typically transparently computed by other modules.  Computational results are managed appropriately with regard to file and data base I/O, transport to display servers, input for high-level routines, and report generation.  Definitions of property characteristics which include optionally precompiled objects and source plus addnl. documentation may be stored in networked data bases for global access.  This facilitates module reuse and the exchange of modules between interested parties.  The elementary data format handling routines and the property computation routines need not to be part of the core program.  They are retrieved from and loaded dynamically at run time from local files or data bases which can be reached on the network.  This look-up process can be made completely transparent, resulting in a comparably small core program which grows only on demand to incorporate procedures for the evaluation of requested information.  A model for asynchronous distributed client-server computations is supported as an alternative to dynamic linking for expensive calcns.  CACTVS is intended to make algorithm development in chem. more productive and to help in collecting the synergetic benefits from the concerted application of compatible algorithm sets which derive chem. information in all its variety.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpSgwAre-5hnbVg90H21EOLACvtfcHk0lgQxBLQ2kicJg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2cXns1GrtQ%253D%253D&md5=32a50d1b7997840849d7697fa0a603e8</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1021%2Fci00017a013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci00017a013%26sid%3Dliteratum%253Aachs%26aulast%3DIhlenfeldt%26aufirst%3DW.%2BD.%26aulast%3DTakahashi%26aufirst%3DY.%26aulast%3DAbe%26aufirst%3DH.%26aulast%3DSasaki%26aufirst%3DS.%26atitle%3DComputation%2520and%2520management%2520of%2520chemical%2520properties%2520in%2520CACTVS%253A%2520an%2520extensible%2520networked%2520approach%2520toward%2520modularity%2520and%2520compatibility%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Comput.%2520Sci.%26date%3D1994%26volume%3D34%26spage%3D109%26epage%3D116" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group">Gasteiger, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rudolph, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sadowski, J.</span><span> </span><span class="NLM_article-title">Automatic generation of 3D-atomic coordinates for organic molecules</span> <span class="citation_source-journal">Tetrahedron Comput. Methodol.</span><span class="NLM_year" style="font-weight: bold;">1990</span> <span class="NLM_volume">3</span> <span class="NLM_fpage">547</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Fjm800791f&amp;key=10.1016%2F0898-5529%2890%2990156-3" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=1990&pages=547&author=J.+Gasteigerauthor=C.+Rudolphauthor=J.+Sadowski&title=Automatic+generation+of+3D-atomic+coordinates+for+organic+molecules"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1016%2F0898-5529%2890%2990156-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0898-5529%252890%252990156-3%26sid%3Dliteratum%253Aachs%26aulast%3DGasteiger%26aufirst%3DJ.%26aulast%3DRudolph%26aufirst%3DC.%26aulast%3DSadowski%26aufirst%3DJ.%26atitle%3DAutomatic%2520generation%2520of%25203D-atomic%2520coordinates%2520for%2520organic%2520molecules%26jtitle%3DTetrahedron%2520Comput.%2520Methodol.%26date%3D1990%26volume%3D3%26spage%3D547" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span> <span class="citation_source-book">Pipeline Pilot</span>, version 4.1.1; SciTegic, Inc.: San Diego, CA, <span class="NLM_year">2004</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Pipeline+Pilot%2C+version+4.1.1%3B+SciTegic%2C+Inc.%3A+San+Diego%2C+CA%2C+2004."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26btitle%3DPipeline%2520Pilot%26date%3D2004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group">Morotti, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mila, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Accornero, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taglibue, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ponzetto, C.</span><span> </span><span class="NLM_article-title">K252a inhibits the oncogenic properties of Met, the HGF receptor</span> <span class="citation_source-journal">Oncogene</span><span class="NLM_year" style="font-weight: bold;">2002</span> <span class="NLM_volume">21</span> <span class="NLM_fpage">4885</span> <span class="NLM_lpage">4893</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2002&pages=4885-4893&author=A.+Morottiauthor=S.+Milaauthor=P.+Accorneroauthor=E.+Taglibueauthor=C.+Ponzetto&title=K252a+inhibits+the+oncogenic+properties+of+Met%2C+the+HGF+receptor"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMorotti%26aufirst%3DA.%26aulast%3DMila%26aufirst%3DS.%26aulast%3DAccornero%26aufirst%3DP.%26aulast%3DTaglibue%26aufirst%3DE.%26aulast%3DPonzetto%26aufirst%3DC.%26atitle%3DK252a%2520inhibits%2520the%2520oncogenic%2520properties%2520of%2520Met%252C%2520the%2520HGF%2520receptor%26jtitle%3DOncogene%26date%3D2002%26volume%3D21%26spage%3D4885%26epage%3D4893" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span> <span class="citation_source-book">MOE: Molecular Operating Environment</span>, version 2004.03; Chemical Computing Group, Inc.: Montreal, Canada, <span class="NLM_year">2004</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=MOE%3A+Molecular+Operating+Environment%2C+version+2004.03%3B+Chemical+Computing+Group%2C+Inc.%3A+Montreal%2C+Canada%2C+2004."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26btitle%3DMOE%253A%2520Molecular%2520Operating%2520Environment%26date%3D2004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group">Delano, W. L.</span> <span class="citation_source-book">The PyMOL Molecular Graphics System</span>, version 0.99; DeLano Scientific: Palo Alto, CA, <span class="NLM_year">2002</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2002&author=W.+L.+Delano&title=The+PyMOL+Molecular+Graphics+System"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DDelano%26aufirst%3DW.%2BL.%26btitle%3DThe%2520PyMOL%2520Molecular%2520Graphics%2520System%26date%3D2002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 45 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Gaochan Wu, Tong Zhao, Dongwei Kang, Jian Zhang, Yuning Song, Vigneshwaran Namasivayam, Jacob Kongsted, Christophe Pannecouque, Erik De Clercq, Vasanthanathan Poongavanam, Xinyong Liu, <span class="NLM_string-name hlFld-ContribAuthor">Peng Zhan</span>. </span><span class="cited-content_cbyCitation_article-title">Overview of Recent Strategic Advances in Medicinal Chemistry. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2019,</strong> <em>62 </em>
                                    (21)
                                     , 9375-9414. <a href="https://doi.org/10.1021/acs.jmedchem.9b00359" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b00359</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b00359&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.9b00359%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DOverview%252Bof%252BRecent%252BStrategic%252BAdvances%252Bin%252BMedicinal%252BChemistry%26aulast%3DWu%26aufirst%3DGaochan%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D26022019%26date%3D28052019%26date%3D03052019%26volume%3D62%26issue%3D21%26spage%3D9375%26epage%3D9414" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jinrong  Zhang</span>, <span class="hlFld-ContribAuthor ">Xuli  Tang</span>, <span class="hlFld-ContribAuthor ">Jing  Li</span>, <span class="hlFld-ContribAuthor ">Peifeng  Li</span>, <span class="hlFld-ContribAuthor ">Nicole J.  de Voogd</span>, <span class="hlFld-ContribAuthor ">Xiaoqin  Ni</span>, <span class="hlFld-ContribAuthor ">Xiaojie  Jin</span>, <span class="hlFld-ContribAuthor ">Xiaojun  Yao</span>, <span class="hlFld-ContribAuthor ">Pinglin  Li</span>, and <span class="hlFld-ContribAuthor ">Guoqiang  Li</span>  . </span><span class="cited-content_cbyCitation_article-title">Cytotoxic Polyketide Derivatives from the South China Sea Sponge Plakortis simplex. </span><span class="cited-content_cbyCitation_journal-name">Journal of Natural Products</span><span> <strong>2013,</strong> <em>76 </em>
                                    (4)
                                     , 600-606. <a href="https://doi.org/10.1021/np300771p" title="DOI URL">https://doi.org/10.1021/np300771p</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/np300771p&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fnp300771p%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Natural%2520Products%26atitle%3DCytotoxic%252BPolyketide%252BDerivatives%252Bfrom%252Bthe%252BSouth%252BChina%252BSea%252BSponge%252BPlakortis%252Bsimplex%26aulast%3DZhang%26aufirst%3DJinrong%26date%3D2013%26date%3D2013%26date%3D2012%26date%3D04112012%26date%3D25032013%26date%3D26042013%26volume%3D76%26issue%3D4%26spage%3D600%26epage%3D606" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yuanxiang  Wang</span>, <span class="hlFld-ContribAuthor ">Jing  Ai</span>, <span class="hlFld-ContribAuthor ">Ying  Wang</span>, <span class="hlFld-ContribAuthor ">Yi  Chen</span>, <span class="hlFld-ContribAuthor ">Lu  Wang</span>, <span class="hlFld-ContribAuthor ">Gang  Liu</span>, <span class="hlFld-ContribAuthor ">Meiyu  Geng</span>, and <span class="hlFld-ContribAuthor ">Ao  Zhang</span>  . </span><span class="cited-content_cbyCitation_article-title">Synthesis and c-Met Kinase Inhibition of 3,5-Disubstituted and 3,5,7-Trisubstituted Quinolines: Identification of 3-(4-Acetylpiperazin-1-yl)-5-(3-nitrobenzylamino)-7- (trifluoromethyl)quinoline as a Novel Anticancer Agent. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2011,</strong> <em>54 </em>
                                    (7)
                                     , 2127-2142. <a href="https://doi.org/10.1021/jm101340q" title="DOI URL">https://doi.org/10.1021/jm101340q</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/jm101340q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fjm101340q%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DSynthesis%252Band%252Bc-Met%252BKinase%252BInhibition%252Bof%252B3%25252C5-Disubstituted%252Band%252B3%25252C5%25252C7-Trisubstituted%252BQuinolines%25253A%252BIdentification%252Bof%252B3-%2525284-Acetylpiperazin-1-yl%252529-5-%2525283-nitrobenzylamino%252529-7-%252B%252528trifluoromethyl%252529quinoline%252Bas%252Ba%252BNovel%252BAnticancer%252BAgent%26aulast%3DWang%26aufirst%3DYuanxiang%26date%3D2011%26date%3D2011%26date%3D2010%26date%3D16102010%26date%3D15032011%26date%3D14042011%26volume%3D54%26issue%3D7%26spage%3D2127%26epage%3D2142" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Michal  Brylinski</span> and <span class="hlFld-ContribAuthor ">Jeffrey  Skolnick</span>. </span><span class="cited-content_cbyCitation_article-title">Comprehensive Structural and Functional Characterization of the Human Kinome by Protein Structure Modeling and Ligand Virtual Screening. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2010,</strong> <em>50 </em>
                                    (10)
                                     , 1839-1854. <a href="https://doi.org/10.1021/ci100235n" title="DOI URL">https://doi.org/10.1021/ci100235n</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/ci100235n&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fci100235n%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DComprehensive%252BStructural%252Band%252BFunctional%252BCharacterization%252Bof%252Bthe%252BHuman%252BKinome%252Bby%252BProtein%252BStructure%252BModeling%252Band%252BLigand%252BVirtual%252BScreening%26aulast%3DBrylinski%26aufirst%3DMichal%26date%3D2010%26date%3D2010%26date%3D2010%26date%3D17%252006%25202010%26date%3D20092010%26date%3D25102010%26volume%3D50%26issue%3D10%26spage%3D1839%26epage%3D1854" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Christine A.  Larsen</span>, <span class="hlFld-ContribAuthor ">William H.  Bisson</span> and <span class="hlFld-ContribAuthor ">Roderick H.  Dashwood</span>. </span><span class="cited-content_cbyCitation_article-title">Tea Catechins Inhibit Hepatocyte Growth Factor Receptor (MET Kinase) Activity in Human Colon Cancer Cells: Kinetic and Molecular Docking Studies. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2009,</strong> <em>52 </em>
                                    (21)
                                     , 6543-6545. <a href="https://doi.org/10.1021/jm901330e" title="DOI URL">https://doi.org/10.1021/jm901330e</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/jm901330e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fjm901330e%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DTea%252BCatechins%252BInhibit%252BHepatocyte%252BGrowth%252BFactor%252BReceptor%252B%252528MET%252BKinase%252529%252BActivity%252Bin%252BHuman%252BColon%252BCancer%252BCells%25253A%252BKinetic%252Band%252BMolecular%252BDocking%252BStudies%26aulast%3DLarsen%26aufirst%3DChristine%2BA.%26date%3D2009%26date%3D2009%26date%3D2009%26date%3D07092009%26date%3D19102009%26date%3D12112009%26volume%3D52%26issue%3D21%26spage%3D6543%26epage%3D6545" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yuâjuan  Feng</span>, <span class="hlFld-ContribAuthor ">YuâLin  Ren</span>, <span class="hlFld-ContribAuthor ">LiâMing  Zhao</span>, <span class="hlFld-ContribAuthor ">GuoâQiang  Xue</span>, <span class="hlFld-ContribAuthor ">WenâHao  Yu</span>, <span class="hlFld-ContribAuthor ">JiaâQi  Yang</span>, <span class="hlFld-ContribAuthor ">JunâWei  Liu</span>. </span><span class="cited-content_cbyCitation_article-title">Design, Synthesis and Biological Evaluation of Novel
              Î±âAcyloxycarboxamideâBased
              Derivatives as
              câMet
              Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Chinese Journal of Chemistry</span><span> <strong>2021,</strong> <em>39 </em>
                                    (8)
                                     , 2241-2250. <a href="https://doi.org/10.1002/cjoc.202100106" title="DOI URL">https://doi.org/10.1002/cjoc.202100106</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/cjoc.202100106&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fcjoc.202100106%26sid%3Dliteratum%253Aachs%26jtitle%3DChinese%2520Journal%2520of%2520Chemistry%26atitle%3DDesign%25252C%252BSynthesis%252Band%252BBiological%252BEvaluation%252Bof%252BNovel%252B%2525CE%2525B1%2525E2%252580%252590Acyloxycarboxamide%2525E2%252580%252590Based%252BDerivatives%252Bas%252Bc%2525E2%252580%252590Met%252BInhibitors%26aulast%3DFeng%26aufirst%3DYu%25E2%2580%2590juan%26date%3D2021%26date%3D2021%26volume%3D39%26issue%3D8%26spage%3D2241%26epage%3D2250" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jianqing  Zhang</span>, <span class="hlFld-ContribAuthor ">Pengqin  Chen</span>, <span class="hlFld-ContribAuthor ">Yongli  Duan</span>, <span class="hlFld-ContribAuthor ">Hehua  Xiong</span>, <span class="hlFld-ContribAuthor ">Hongmin  Li</span>, <span class="hlFld-ContribAuthor ">Yao  Zeng</span>, <span class="hlFld-ContribAuthor ">Guang  Liang</span>, <span class="hlFld-ContribAuthor ">Qidong  Tang</span>, <span class="hlFld-ContribAuthor ">Di  Wu</span>. </span><span class="cited-content_cbyCitation_article-title">Design, synthesis and biological evaluation of 7H-pyrrolo[2,3-d]pyrimidine derivatives containing 1,8-naphthyridine-4-one fragment. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>215 </em>, 113273. <a href="https://doi.org/10.1016/j.ejmech.2021.113273" title="DOI URL">https://doi.org/10.1016/j.ejmech.2021.113273</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2021.113273&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2021.113273%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDesign%25252C%252Bsynthesis%252Band%252Bbiological%252Bevaluation%252Bof%252B7H-pyrrolo%25255B2%25252C3-d%25255Dpyrimidine%252Bderivatives%252Bcontaining%252B1%25252C8-naphthyridine-4-one%252Bfragment%26aulast%3DZhang%26aufirst%3DJianqing%26date%3D2021%26volume%3D215%26spage%3D113273" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Eleonora  Gianti</span>, <span class="hlFld-ContribAuthor ">Randy J.  Zauhar</span>. </span><span class="cited-content_cbyCitation_article-title">Structureâactivity relationships and drug design. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2021,</strong>,, 129-153. <a href="https://doi.org/10.1016/B978-0-12-820007-0.00007-6" title="DOI URL">https://doi.org/10.1016/B978-0-12-820007-0.00007-6</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/B978-0-12-820007-0.00007-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2FB978-0-12-820007-0.00007-6%26sid%3Dliteratum%253Aachs%26atitle%3DStructure%2525E2%252580%252593activity%252Brelationships%252Band%252Bdrug%252Bdesign%26aulast%3DGianti%26aufirst%3DEleonora%26date%3D2021%26spage%3D129%26epage%3D153%26pub%3DElsevier%26atitle%3DRemington%26date%3D2021%26volume%3D50" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Rafat M.  Mohareb</span>, <span class="hlFld-ContribAuthor ">Ensaf S.  Alwan</span>. </span><span class="cited-content_cbyCitation_article-title">Heterocyclization of 2-(2-phenylhydrazono)cyclohexane-1,3-dione to Synthesis Thiophene, Pyrazole and 1,2,4-triazine Derivatives with Anti-Tumor and Tyrosine Kinase Inhibitions. </span><span class="cited-content_cbyCitation_journal-name">Anti-Cancer Agents in Medicinal Chemistry</span><span> <strong>2020,</strong> <em>20 </em>
                                    (10)
                                     , 1209-1220. <a href="https://doi.org/10.2174/1871520620666200310093911" title="DOI URL">https://doi.org/10.2174/1871520620666200310093911</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.2174/1871520620666200310093911&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.2174%2F1871520620666200310093911%26sid%3Dliteratum%253Aachs%26jtitle%3DAnti-Cancer%2520Agents%2520in%2520Medicinal%2520Chemistry%26atitle%3DHeterocyclization%252Bof%252B2-%2525282-phenylhydrazono%252529cyclohexane-1%25252C3-dione%252Bto%252BSynthesis%252BThiophene%25252C%252BPyrazole%252Band%252B1%25252C2%25252C4-triazine%252BDerivatives%252Bwith%252BAnti-Tumor%252Band%252BTyrosine%252BKinase%252BInhibitions%26aulast%3DMohareb%26aufirst%3DRafat%2BM.%26date%3D2020%26volume%3D20%26issue%3D10%26spage%3D1209%26epage%3D1220" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xiang  Nan</span>, <span class="hlFld-ContribAuthor ">Jing  Zhang</span>, <span class="hlFld-ContribAuthor ">Hui-Jing  Li</span>, <span class="hlFld-ContribAuthor ">Rui  Wu</span>, <span class="hlFld-ContribAuthor ">Sen-Biao  Fang</span>, <span class="hlFld-ContribAuthor ">Zhi-Zhou  Zhang</span>, <span class="hlFld-ContribAuthor ">Yan-Chao  Wu</span>. </span><span class="cited-content_cbyCitation_article-title">Design, synthesis and biological evaluation of novel N-sulfonylamidine-based derivatives as c-Met inhibitors via Cu-catalyzed three-component reaction. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>200 </em>, 112470. <a href="https://doi.org/10.1016/j.ejmech.2020.112470" title="DOI URL">https://doi.org/10.1016/j.ejmech.2020.112470</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2020.112470&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2020.112470%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDesign%25252C%252Bsynthesis%252Band%252Bbiological%252Bevaluation%252Bof%252Bnovel%252BN-sulfonylamidine-based%252Bderivatives%252Bas%252Bc-Met%252Binhibitors%252Bvia%252BCu-catalyzed%252Bthree-component%252Breaction%26aulast%3DNan%26aufirst%3DXiang%26date%3D2020%26volume%3D200%26spage%3D112470" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Rafat M.  Mohareb</span>, <span class="hlFld-ContribAuthor ">Nadia Y.  Megally Abdo</span>, <span class="hlFld-ContribAuthor ">Marwa S.  Gamaan</span>. </span><span class="cited-content_cbyCitation_article-title">Uses of cyclohexanâ1,3âdione for the synthesis of tetrahydrochromeno[3,4â
              c
              ]chromen derivatives with antiâtumor activities. </span><span class="cited-content_cbyCitation_journal-name">Journal of Heterocyclic Chemistry</span><span> <strong>2020,</strong> <em>57 </em>
                                    (6)
                                     , 2512-2527. <a href="https://doi.org/10.1002/jhet.3966" title="DOI URL">https://doi.org/10.1002/jhet.3966</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/jhet.3966&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fjhet.3966%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Heterocyclic%2520Chemistry%26atitle%3DUses%252Bof%252Bcyclohexan%2525E2%252580%2525901%25252C3%2525E2%252580%252590dione%252Bfor%252Bthe%252Bsynthesis%252Bof%252Btetrahydrochromeno%25255B3%25252C4%2525E2%252580%252590%252Bc%252B%25255Dchromen%252Bderivatives%252Bwith%252Banti%2525E2%252580%252590tumor%252Bactivities%26aulast%3DMohareb%26aufirst%3DRafat%2BM.%26date%3D2020%26date%3D2020%26volume%3D57%26issue%3D6%26spage%3D2512%26epage%3D2527" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Rafat Milad  Mohareb</span>, <span class="hlFld-ContribAuthor ">Nadia Youssef  Megally Abdo</span>, <span class="hlFld-ContribAuthor ">Waleed Nabeel  Al-darkazali</span>. </span><span class="cited-content_cbyCitation_article-title">Uses of Cyclohexan-1,3-dione for the Synthesis of Thiazole, Pyrazole, Thiophene, Isoxazole and Pyran Derivatives with Antitumor Activities. </span><span class="cited-content_cbyCitation_journal-name">Letters in Drug Design & Discovery</span><span> <strong>2020,</strong> <em>17 </em>
                                    (5)
                                     , 597-609. <a href="https://doi.org/10.2174/1570180816666190618115128" title="DOI URL">https://doi.org/10.2174/1570180816666190618115128</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.2174/1570180816666190618115128&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.2174%2F1570180816666190618115128%26sid%3Dliteratum%253Aachs%26jtitle%3DLetters%2520in%2520Drug%2520Design%2520%2526%2520Discovery%26atitle%3DUses%252Bof%252BCyclohexan-1%25252C3-dione%252Bfor%252Bthe%252BSynthesis%252Bof%252BThiazole%25252C%252BPyrazole%25252C%252BThiophene%25252C%252BIsoxazole%252Band%252BPyran%252BDerivatives%252Bwith%252BAntitumor%252BActivities%26aulast%3DMohareb%26aufirst%3DRafat%2BMilad%26date%3D2020%26volume%3D17%26issue%3D5%26spage%3D597%26epage%3D609" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xiang  Nan</span>, <span class="hlFld-ContribAuthor ">Hui-Jing  Li</span>, <span class="hlFld-ContribAuthor ">Sen-Biao  Fang</span>, <span class="hlFld-ContribAuthor ">Qin-Ying  Li</span>, <span class="hlFld-ContribAuthor ">Yan-Chao  Wu</span>. </span><span class="cited-content_cbyCitation_article-title">Structure-based discovery of novel 4-(2-fluorophenoxy)quinoline derivatives as c-Met inhibitors using isocyanide-involved multicomponent reactions. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>193 </em>, 112241. <a href="https://doi.org/10.1016/j.ejmech.2020.112241" title="DOI URL">https://doi.org/10.1016/j.ejmech.2020.112241</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2020.112241&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2020.112241%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DStructure-based%252Bdiscovery%252Bof%252Bnovel%252B4-%2525282-fluorophenoxy%252529quinoline%252Bderivatives%252Bas%252Bc-Met%252Binhibitors%252Busing%252Bisocyanide-involved%252Bmulticomponent%252Breactions%26aulast%3DNan%26aufirst%3DXiang%26date%3D2020%26volume%3D193%26spage%3D112241" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Nadia Y. Megally  Abdo</span>, <span class="hlFld-ContribAuthor ">Rafat M.  Mohareb</span>, <span class="hlFld-ContribAuthor ">Waleed N.  Al-darkazali</span>. </span><span class="cited-content_cbyCitation_article-title">Heterocyclization of 2-Arylidenecyclohexan-1,3-dione: Synthesis of Thiophene, Thiazole, and Isoxazole Derivatives with Potential Antitumor Activities. </span><span class="cited-content_cbyCitation_journal-name">Anti-Cancer Agents in Medicinal Chemistry</span><span> <strong>2020,</strong> <em>20 </em>
                                    (3)
                                     , 335-345. <a href="https://doi.org/10.2174/1871520619666190730103425" title="DOI URL">https://doi.org/10.2174/1871520619666190730103425</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.2174/1871520619666190730103425&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.2174%2F1871520619666190730103425%26sid%3Dliteratum%253Aachs%26jtitle%3DAnti-Cancer%2520Agents%2520in%2520Medicinal%2520Chemistry%26atitle%3DHeterocyclization%252Bof%252B2-Arylidenecyclohexan-1%25252C3-dione%25253A%252BSynthesis%252Bof%252BThiophene%25252C%252BThiazole%25252C%252Band%252BIsoxazole%252BDerivatives%252Bwith%252BPotential%252BAntitumor%252BActivities%26aulast%3DAbdo%26aufirst%3DNadia%2BY.%2BMegally%26date%3D2020%26volume%3D20%26issue%3D3%26spage%3D335%26epage%3D345" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Nadia Y.  Megally Abdo</span>, <span class="hlFld-ContribAuthor ">Eman M.  Samir</span>, <span class="hlFld-ContribAuthor ">Rafat M.  Mohareb</span>. </span><span class="cited-content_cbyCitation_article-title">Synthesis and evaluation of novel 4
              H
              âpyrazole and thiophene derivatives derived from chalcone as potential antiâproliferative agents, Pimâ1 kinase inhibitors, and PAINS. </span><span class="cited-content_cbyCitation_journal-name">Journal of Heterocyclic Chemistry</span><span> <strong>2020,</strong> <em>57 </em>
                                    (4)
                                     , 1993-2009. <a href="https://doi.org/10.1002/jhet.3928" title="DOI URL">https://doi.org/10.1002/jhet.3928</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/jhet.3928&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fjhet.3928%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Heterocyclic%2520Chemistry%26atitle%3DSynthesis%252Band%252Bevaluation%252Bof%252Bnovel%252B4%252BH%252B%2525E2%252580%252590pyrazole%252Band%252Bthiophene%252Bderivatives%252Bderived%252Bfrom%252Bchalcone%252Bas%252Bpotential%252Banti%2525E2%252580%252590proliferative%252Bagents%25252C%252BPim%2525E2%252580%2525901%252Bkinase%252Binhibitors%25252C%252Band%252BPAINS%26aulast%3DMegally%2BAbdo%26aufirst%3DNadia%2BY.%26date%3D2020%26volume%3D57%26issue%3D4%26spage%3D1993%26epage%3D2009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Rafat M.  Mohareb</span>, <span class="hlFld-ContribAuthor ">Wagnat W.  Wardakhan</span>, <span class="hlFld-ContribAuthor ">Nermeen S.  Abbas</span>. </span><span class="cited-content_cbyCitation_article-title">Synthesis of Tetrahydrobenzo[b]thiophene-3-carbohydrazide Derivatives as Potential Anti-cancer Agents and Pim-1 Kinase Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Anti-Cancer Agents in Medicinal Chemistry</span><span> <strong>2019,</strong> <em>19 </em>
                                    (14)
                                     , 1737-1753. <a href="https://doi.org/10.2174/1871520619666190402153429" title="DOI URL">https://doi.org/10.2174/1871520619666190402153429</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.2174/1871520619666190402153429&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.2174%2F1871520619666190402153429%26sid%3Dliteratum%253Aachs%26jtitle%3DAnti-Cancer%2520Agents%2520in%2520Medicinal%2520Chemistry%26atitle%3DSynthesis%252Bof%252BTetrahydrobenzo%25255Bb%25255Dthiophene-3-carbohydrazide%252BDerivatives%252Bas%252BPotential%252BAnti-cancer%252BAgents%252Band%252BPim-1%252BKinase%252BInhibitors%26aulast%3DMohareb%26aufirst%3DRafat%2BM.%26date%3D2019%26volume%3D19%26issue%3D14%26spage%3D1737%26epage%3D1753" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Rafat M.  Mohareb</span>, <span class="hlFld-ContribAuthor ">Amr S.  Abouzied</span>, <span class="hlFld-ContribAuthor ">Nermeen S.  Abbas</span>. </span><span class="cited-content_cbyCitation_article-title">Synthesis and Biological Evaluation of Novel 4,5,6,7-Tetrahydrobenzo[D]-Thiazol-2- Yl Derivatives Derived from Dimedone with Anti-Tumor, C-Met, Tyrosine Kinase and Pim-1 Inhibitions. </span><span class="cited-content_cbyCitation_journal-name">Anti-Cancer Agents in Medicinal Chemistry</span><span> <strong>2019,</strong> <em>19 </em>
                                    (12)
                                     , 1438-1453. <a href="https://doi.org/10.2174/1871520619666190416102144" title="DOI URL">https://doi.org/10.2174/1871520619666190416102144</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.2174/1871520619666190416102144&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.2174%2F1871520619666190416102144%26sid%3Dliteratum%253Aachs%26jtitle%3DAnti-Cancer%2520Agents%2520in%2520Medicinal%2520Chemistry%26atitle%3DSynthesis%252Band%252BBiological%252BEvaluation%252Bof%252BNovel%252B4%25252C5%25252C6%25252C7-Tetrahydrobenzo%25255BD%25255D-Thiazol-2-%252BYl%252BDerivatives%252BDerived%252Bfrom%252BDimedone%252Bwith%252BAnti-Tumor%25252C%252BC-Met%25252C%252BTyrosine%252BKinase%252Band%252BPim-1%252BInhibitions%26aulast%3DMohareb%26aufirst%3DRafat%2BM.%26date%3D2019%26volume%3D19%26issue%3D12%26spage%3D1438%26epage%3D1453" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xiang  Nan</span>, <span class="hlFld-ContribAuthor ">Yi-Fan  Jiang</span>, <span class="hlFld-ContribAuthor ">Hui-Jing  Li</span>, <span class="hlFld-ContribAuthor ">Jun-Hu  Wang</span>, <span class="hlFld-ContribAuthor ">Yan-Chao  Wu</span>. </span><span class="cited-content_cbyCitation_article-title">Design, synthesis and evaluation of sulfonylurea-containing 4-phenoxyquinolines as highly selective c-Met kinase inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry</span><span> <strong>2019,</strong> <em>27 </em>
                                    (13)
                                     , 2801-2812. <a href="https://doi.org/10.1016/j.bmc.2019.05.007" title="DOI URL">https://doi.org/10.1016/j.bmc.2019.05.007</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmc.2019.05.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmc.2019.05.007%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%26atitle%3DDesign%25252C%252Bsynthesis%252Band%252Bevaluation%252Bof%252Bsulfonylurea-containing%252B4-phenoxyquinolines%252Bas%252Bhighly%252Bselective%252Bc-Met%252Bkinase%252Binhibitors%26aulast%3DNan%26aufirst%3DXiang%26date%3D2019%26volume%3D27%26issue%3D13%26spage%3D2801%26epage%3D2812" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Rafat M.  Mohareb</span>, <span class="hlFld-ContribAuthor ">Eman M.  Samir</span>, <span class="hlFld-ContribAuthor ">Peter A.  Halim</span>. </span><span class="cited-content_cbyCitation_article-title">Synthesis, and anti-proliferative, Pim-1 kinase inhibitors and molecular docking of thiophenes derived from estrone. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic Chemistry</span><span> <strong>2019,</strong> <em>83 </em>, 402-413. <a href="https://doi.org/10.1016/j.bioorg.2018.10.067" title="DOI URL">https://doi.org/10.1016/j.bioorg.2018.10.067</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bioorg.2018.10.067&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bioorg.2018.10.067%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520Chemistry%26atitle%3DSynthesis%25252C%252Band%252Banti-proliferative%25252C%252BPim-1%252Bkinase%252Binhibitors%252Band%252Bmolecular%252Bdocking%252Bof%252Bthiophenes%252Bderived%252Bfrom%252Bestrone%26aulast%3DMohareb%26aufirst%3DRafat%2BM.%26date%3D2019%26volume%3D83%26spage%3D402%26epage%3D413" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Rafat M.  Mohareb</span>, <span class="hlFld-ContribAuthor ">Thomas M.  KlapoÌtke</span>, <span class="hlFld-ContribAuthor ">Elena  Reinhardt</span>. </span><span class="cited-content_cbyCitation_article-title">Uses of dimedone for the synthesis of thiazole derivatives as new anti-tumor, c-Met, tyrosine kinase, and Pim-1 inhibitions. </span><span class="cited-content_cbyCitation_journal-name">Medicinal Chemistry Research</span><span> <strong>2018,</strong> <em>27 </em>
                                    (11-12)
                                     , 2494-2511. <a href="https://doi.org/10.1007/s00044-018-2252-3" title="DOI URL">https://doi.org/10.1007/s00044-018-2252-3</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s00044-018-2252-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs00044-018-2252-3%26sid%3Dliteratum%253Aachs%26jtitle%3DMedicinal%2520Chemistry%2520Research%26atitle%3DUses%252Bof%252Bdimedone%252Bfor%252Bthe%252Bsynthesis%252Bof%252Bthiazole%252Bderivatives%252Bas%252Bnew%252Banti-tumor%25252C%252Bc-Met%25252C%252Btyrosine%252Bkinase%25252C%252Band%252BPim-1%252Binhibitions%26aulast%3DMohareb%26aufirst%3DRafat%2BM.%26date%3D2018%26date%3D2018%26volume%3D27%26issue%3D11-12%26spage%3D2494%26epage%3D2511" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Shima  Aliebrahimi</span>, <span class="hlFld-ContribAuthor ">Shideh  Montasser Kouhsari</span>, <span class="hlFld-ContribAuthor ">Seyed Nasser  Ostad</span>, <span class="hlFld-ContribAuthor ">Seyed Shahriar  Arab</span>, <span class="hlFld-ContribAuthor ">Leila  Karami</span>. </span><span class="cited-content_cbyCitation_article-title">Identification of Phytochemicals Targeting c-Met Kinase Domain using Consensus Docking and Molecular Dynamics Simulation Studies. </span><span class="cited-content_cbyCitation_journal-name">Cell Biochemistry and Biophysics</span><span> <strong>2018,</strong> <em>76 </em>
                                    (1-2)
                                     , 135-145. <a href="https://doi.org/10.1007/s12013-017-0821-6" title="DOI URL">https://doi.org/10.1007/s12013-017-0821-6</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s12013-017-0821-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs12013-017-0821-6%26sid%3Dliteratum%253Aachs%26jtitle%3DCell%2520Biochemistry%2520and%2520Biophysics%26atitle%3DIdentification%252Bof%252BPhytochemicals%252BTargeting%252Bc-Met%252BKinase%252BDomain%252Busing%252BConsensus%252BDocking%252Band%252BMolecular%252BDynamics%252BSimulation%252BStudies%26aulast%3DAliebrahimi%26aufirst%3DShima%26date%3D2018%26date%3D2017%26volume%3D76%26issue%3D1-2%26spage%3D135%26epage%3D145" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Qidong  Tang</span>, <span class="hlFld-ContribAuthor ">Yongli  Duan</span>, <span class="hlFld-ContribAuthor ">Linxiao  Wang</span>, <span class="hlFld-ContribAuthor ">Min  Wang</span>, <span class="hlFld-ContribAuthor ">Yiqiang  Ouyang</span>, <span class="hlFld-ContribAuthor ">Caolin  Wang</span>, <span class="hlFld-ContribAuthor ">Han  Mei</span>, <span class="hlFld-ContribAuthor ">Sheng  Tang</span>, <span class="hlFld-ContribAuthor ">Yinhua  Xiong</span>, <span class="hlFld-ContribAuthor ">Pengwu  Zheng</span>, <span class="hlFld-ContribAuthor ">Ping  Gong</span>, <span class="hlFld-ContribAuthor ">Wufu  Zhu</span>. </span><span class="cited-content_cbyCitation_article-title">Synthesis and antiproliferative activity of pyrrolo[2,3-b]pyridine derivatives bearing the 1,8-naphthyridin-2-one moiety. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2018,</strong> <em>143 </em>, 266-275. <a href="https://doi.org/10.1016/j.ejmech.2017.11.034" title="DOI URL">https://doi.org/10.1016/j.ejmech.2017.11.034</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2017.11.034&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2017.11.034%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DSynthesis%252Band%252Bantiproliferative%252Bactivity%252Bof%252Bpyrrolo%25255B2%25252C3-b%25255Dpyridine%252Bderivatives%252Bbearing%252Bthe%252B1%25252C8-naphthyridin-2-one%252Bmoiety%26aulast%3DTang%26aufirst%3DQidong%26date%3D2018%26volume%3D143%26spage%3D266%26epage%3D275" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xinyuan  Lu</span>, <span class="hlFld-ContribAuthor ">Nir  Peled</span>, <span class="hlFld-ContribAuthor ">John  Greer</span>, <span class="hlFld-ContribAuthor ">Wei  Wu</span>, <span class="hlFld-ContribAuthor ">Peter  Choi</span>, <span class="hlFld-ContribAuthor ">Alice H.  Berger</span>, <span class="hlFld-ContribAuthor ">Sergio  Wong</span>, <span class="hlFld-ContribAuthor ">Kuang-Yu  Jen</span>, <span class="hlFld-ContribAuthor ">Youngho  Seo</span>, <span class="hlFld-ContribAuthor ">Byron  Hann</span>, <span class="hlFld-ContribAuthor ">Angela  Brooks</span>, <span class="hlFld-ContribAuthor ">Matthew  Meyerson</span>, <span class="hlFld-ContribAuthor ">Eric A.  Collisson</span>. </span><span class="cited-content_cbyCitation_article-title">MET
              Exon 14 Mutation Encodes an Actionable Therapeutic Target in Lung Adenocarcinoma. </span><span class="cited-content_cbyCitation_journal-name">Cancer Research</span><span> <strong>2017,</strong> <em>77 </em>
                                    (16)
                                     , 4498-4505. <a href="https://doi.org/10.1158/0008-5472.CAN-16-1944" title="DOI URL">https://doi.org/10.1158/0008-5472.CAN-16-1944</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1158/0008-5472.CAN-16-1944&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1158%2F0008-5472.CAN-16-1944%26sid%3Dliteratum%253Aachs%26jtitle%3DCancer%2520Research%26atitle%3DMET%252BExon%252B14%252BMutation%252BEncodes%252Ban%252BActionable%252BTherapeutic%252BTarget%252Bin%252BLung%252BAdenocarcinoma%26aulast%3DLu%26aufirst%3DXinyuan%26date%3D2017%26date%3D2017%26volume%3D77%26issue%3D16%26spage%3D4498%26epage%3D4505" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Qidong  Tang</span>, <span class="hlFld-ContribAuthor ">Linxiao  Wang</span>, <span class="hlFld-ContribAuthor ">Yongli  Duan</span>, <span class="hlFld-ContribAuthor ">Wenhui  Wang</span>, <span class="hlFld-ContribAuthor ">Shunmin  Huang</span>, <span class="hlFld-ContribAuthor ">Jia  Zhi</span>, <span class="hlFld-ContribAuthor ">Shuang  Jia</span>, <span class="hlFld-ContribAuthor ">Wufu  Zhu</span>, <span class="hlFld-ContribAuthor ">Ping  Wang</span>, <span class="hlFld-ContribAuthor ">Rong  Luo</span>, <span class="hlFld-ContribAuthor ">Pengwu  Zheng</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of novel 7-azaindole derivatives bearing dihydropyridazine moiety as c-Met kinase inhibitors. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2017,</strong> <em>133 </em>, 97-106. <a href="https://doi.org/10.1016/j.ejmech.2017.03.045" title="DOI URL">https://doi.org/10.1016/j.ejmech.2017.03.045</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2017.03.045&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2017.03.045%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252Bnovel%252B7-azaindole%252Bderivatives%252Bbearing%252Bdihydropyridazine%252Bmoiety%252Bas%252Bc-Met%252Bkinase%252Binhibitors%26aulast%3DTang%26aufirst%3DQidong%26date%3D2017%26volume%3D133%26spage%3D97%26epage%3D106" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Hongrui  Lei</span>, <span class="hlFld-ContribAuthor ">Gang  Hu</span>, <span class="hlFld-ContribAuthor ">Yu  Wang</span>, <span class="hlFld-ContribAuthor ">Pei  Han</span>, <span class="hlFld-ContribAuthor ">Zijian  Liu</span>, <span class="hlFld-ContribAuthor ">Yanfang  Zhao</span>, <span class="hlFld-ContribAuthor ">Ping  Gong</span>. </span><span class="cited-content_cbyCitation_article-title">Design, Synthesis, and Biological Evaluation of 4-Phenoxyquinoline Derivatives Containing Benzo[
              d
              ]thiazole-2-yl Urea as c-Met Kinase Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Archiv der Pharmazie</span><span> <strong>2016,</strong> <em>349 </em>
                                    (8)
                                     , 651-661. <a href="https://doi.org/10.1002/ardp.201600003" title="DOI URL">https://doi.org/10.1002/ardp.201600003</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/ardp.201600003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fardp.201600003%26sid%3Dliteratum%253Aachs%26jtitle%3DArchiv%2520der%2520Pharmazie%26atitle%3DDesign%25252C%252BSynthesis%25252C%252Band%252BBiological%252BEvaluation%252Bof%252B4-Phenoxyquinoline%252BDerivatives%252BContaining%252BBenzo%25255B%252Bd%252B%25255Dthiazole-2-yl%252BUrea%252Bas%252Bc-Met%252BKinase%252BInhibitors%26aulast%3DLei%26aufirst%3DHongrui%26date%3D2016%26date%3D2016%26volume%3D349%26issue%3D8%26spage%3D651%26epage%3D661" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Mohamed S.  Mady</span>, <span class="hlFld-ContribAuthor ">Mohamed M.  Mohyeldin</span>, <span class="hlFld-ContribAuthor ">Hassan Y.  Ebrahim</span>, <span class="hlFld-ContribAuthor ">Heba E.  Elsayed</span>, <span class="hlFld-ContribAuthor ">Wael E.  Houssen</span>, <span class="hlFld-ContribAuthor ">Eman G.  Haggag</span>, <span class="hlFld-ContribAuthor ">Randa F.  Soliman</span>, <span class="hlFld-ContribAuthor ">Khalid A.  El Sayed</span>. </span><span class="cited-content_cbyCitation_article-title">The indole alkaloid meleagrin, from the olive tree endophytic fungus Penicillium chrysogenum, as a novel lead for the control of c-Met-dependent breast cancer proliferation, migration and invasion. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry</span><span> <strong>2016,</strong> <em>24 </em>
                                    (2)
                                     , 113-122. <a href="https://doi.org/10.1016/j.bmc.2015.11.038" title="DOI URL">https://doi.org/10.1016/j.bmc.2015.11.038</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmc.2015.11.038&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmc.2015.11.038%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%26atitle%3DThe%252Bindole%252Balkaloid%252Bmeleagrin%25252C%252Bfrom%252Bthe%252Bolive%252Btree%252Bendophytic%252Bfungus%252BPenicillium%252Bchrysogenum%25252C%252Bas%252Ba%252Bnovel%252Blead%252Bfor%252Bthe%252Bcontrol%252Bof%252Bc-Met-dependent%252Bbreast%252Bcancer%252Bproliferation%25252C%252Bmigration%252Band%252Binvasion%26aulast%3DMady%26aufirst%3DMohamed%2BS.%26date%3D2016%26volume%3D24%26issue%3D2%26spage%3D113%26epage%3D122" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Weike  Liao</span>, <span class="hlFld-ContribAuthor ">Gang  Hu</span>, <span class="hlFld-ContribAuthor ">Zhuang  Guo</span>, <span class="hlFld-ContribAuthor ">Deyu  Sun</span>, <span class="hlFld-ContribAuthor ">Lixia  Zhang</span>, <span class="hlFld-ContribAuthor ">Yanxin  Bu</span>, <span class="hlFld-ContribAuthor ">Yingxiu  Li</span>, <span class="hlFld-ContribAuthor ">Yajing  Liu</span>, <span class="hlFld-ContribAuthor ">Ping  Gong</span>. </span><span class="cited-content_cbyCitation_article-title">Design and biological evaluation of novel 4-(2-fluorophenoxy)quinoline derivatives bearing an imidazolone moiety as c-Met kinase inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry</span><span> <strong>2015,</strong> <em>23 </em>
                                    (15)
                                     , 4410-4422. <a href="https://doi.org/10.1016/j.bmc.2015.06.026" title="DOI URL">https://doi.org/10.1016/j.bmc.2015.06.026</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmc.2015.06.026&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmc.2015.06.026%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%26atitle%3DDesign%252Band%252Bbiological%252Bevaluation%252Bof%252Bnovel%252B4-%2525282-fluorophenoxy%252529quinoline%252Bderivatives%252Bbearing%252Ban%252Bimidazolone%252Bmoiety%252Bas%252Bc-Met%252Bkinase%252Binhibitors%26aulast%3DLiao%26aufirst%3DWeike%26date%3D2015%26volume%3D23%26issue%3D15%26spage%3D4410%26epage%3D4422" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ashutosh  Kumar</span>, <span class="hlFld-ContribAuthor ">Kam Y.J.  Zhang</span>. </span><span class="cited-content_cbyCitation_article-title">Hierarchical virtual screening approaches in small molecule drug discovery. </span><span class="cited-content_cbyCitation_journal-name">Methods</span><span> <strong>2015,</strong> <em>71 </em>, 26-37. <a href="https://doi.org/10.1016/j.ymeth.2014.07.007" title="DOI URL">https://doi.org/10.1016/j.ymeth.2014.07.007</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ymeth.2014.07.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ymeth.2014.07.007%26sid%3Dliteratum%253Aachs%26jtitle%3DMethods%26atitle%3DHierarchical%252Bvirtual%252Bscreening%252Bapproaches%252Bin%252Bsmall%252Bmolecule%252Bdrug%252Bdiscovery%26aulast%3DKumar%26aufirst%3DAshutosh%26date%3D2015%26volume%3D71%26spage%3D26%26epage%3D37" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Shunguang  Zhou</span>, <span class="hlFld-ContribAuthor ">Huimin  Liao</span>, <span class="hlFld-ContribAuthor ">Mingmei  Liu</span>, <span class="hlFld-ContribAuthor ">Guobing  Feng</span>, <span class="hlFld-ContribAuthor ">Baolin  Fu</span>, <span class="hlFld-ContribAuthor ">Ruijuan  Li</span>, <span class="hlFld-ContribAuthor ">Maosheng  Cheng</span>, <span class="hlFld-ContribAuthor ">Yanfang  Zhao</span>, <span class="hlFld-ContribAuthor ">Ping  Gong</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery andw biological evaluation of novel 6,7-disubstituted-4-(2-fluorophenoxy)quinoline derivatives possessing 1,2,3-triazole-4-carboxamide moiety as c-Met kinase inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry</span><span> <strong>2014,</strong> <em>22 </em>
                                    (22)
                                     , 6438-6452. <a href="https://doi.org/10.1016/j.bmc.2014.09.037" title="DOI URL">https://doi.org/10.1016/j.bmc.2014.09.037</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmc.2014.09.037&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmc.2014.09.037%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bandw%252Bbiological%252Bevaluation%252Bof%252Bnovel%252B6%25252C7-disubstituted-4-%2525282-fluorophenoxy%252529quinoline%252Bderivatives%252Bpossessing%252B1%25252C2%25252C3-triazole-4-carboxamide%252Bmoiety%252Bas%252Bc-Met%252Bkinase%252Binhibitors%26aulast%3DZhou%26aufirst%3DShunguang%26date%3D2014%26volume%3D22%26issue%3D22%26spage%3D6438%26epage%3D6452" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Zijian  Liu</span>, <span class="hlFld-ContribAuthor ">Rui  Wang</span>, <span class="hlFld-ContribAuthor ">Ruiming  Guo</span>, <span class="hlFld-ContribAuthor ">Jinxing  Hu</span>, <span class="hlFld-ContribAuthor ">Ruijuan  Li</span>, <span class="hlFld-ContribAuthor ">Yanfang  Zhao</span>, <span class="hlFld-ContribAuthor ">Ping  Gong</span>. </span><span class="cited-content_cbyCitation_article-title">Design, synthesis and biological evaluation of novel 6,7-disubstituted-4-phenoxyquinoline derivatives bearing 4-oxo-3,4-dihydrophthalazine-1-carboxamide moieties as c-Met kinase inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry</span><span> <strong>2014,</strong> <em>22 </em>
                                    (14)
                                     , 3642-3653. <a href="https://doi.org/10.1016/j.bmc.2014.05.013" title="DOI URL">https://doi.org/10.1016/j.bmc.2014.05.013</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmc.2014.05.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmc.2014.05.013%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%26atitle%3DDesign%25252C%252Bsynthesis%252Band%252Bbiological%252Bevaluation%252Bof%252Bnovel%252B6%25252C7-disubstituted-4-phenoxyquinoline%252Bderivatives%252Bbearing%252B4-oxo-3%25252C4-dihydrophthalazine-1-carboxamide%252Bmoieties%252Bas%252Bc-Met%252Bkinase%252Binhibitors%26aulast%3DLiu%26aufirst%3DZijian%26date%3D2014%26volume%3D22%26issue%3D14%26spage%3D3642%26epage%3D3653" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Qidong  Tang</span>, <span class="hlFld-ContribAuthor ">Guogang  Zhang</span>, <span class="hlFld-ContribAuthor ">Xinming  Du</span>, <span class="hlFld-ContribAuthor ">Wufu  Zhu</span>, <span class="hlFld-ContribAuthor ">Ruijuan  Li</span>, <span class="hlFld-ContribAuthor ">Huafang  Lin</span>, <span class="hlFld-ContribAuthor ">Pengcheng  Li</span>, <span class="hlFld-ContribAuthor ">Maosheng  Cheng</span>, <span class="hlFld-ContribAuthor ">Ping  Gong</span>, <span class="hlFld-ContribAuthor ">Yanfang  Zhao</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of novel 6,7-disubstituted-4-phenoxyquinoline derivatives bearing 5-(aminomethylene)pyrimidine-2,4,6-trione moiety as c-Met kinase inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry</span><span> <strong>2014,</strong> <em>22 </em>
                                    (4)
                                     , 1236-1249. <a href="https://doi.org/10.1016/j.bmc.2014.01.014" title="DOI URL">https://doi.org/10.1016/j.bmc.2014.01.014</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmc.2014.01.014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmc.2014.01.014%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252Bnovel%252B6%25252C7-disubstituted-4-phenoxyquinoline%252Bderivatives%252Bbearing%252B5-%252528aminomethylene%252529pyrimidine-2%25252C4%25252C6-trione%252Bmoiety%252Bas%252Bc-Met%252Bkinase%252Binhibitors%26aulast%3DTang%26aufirst%3DQidong%26date%3D2014%26volume%3D22%26issue%3D4%26spage%3D1236%26epage%3D1249" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Baohui  Qi</span>, <span class="hlFld-ContribAuthor ">Bin  Mi</span>, <span class="hlFld-ContribAuthor ">Xin  Zhai</span>, <span class="hlFld-ContribAuthor ">Ziyi  Xu</span>, <span class="hlFld-ContribAuthor ">Xiaolong  Zhang</span>, <span class="hlFld-ContribAuthor ">Zeru  Tian</span>, <span class="hlFld-ContribAuthor ">Ping  Gong</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery and optimization of novel 4-phenoxy-6,7-disubstituted quinolines possessing semicarbazones as c-Met kinase inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry</span><span> <strong>2013,</strong> <em>21 </em>
                                    (17)
                                     , 5246-5260. <a href="https://doi.org/10.1016/j.bmc.2013.06.026" title="DOI URL">https://doi.org/10.1016/j.bmc.2013.06.026</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmc.2013.06.026&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmc.2013.06.026%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Band%252Boptimization%252Bof%252Bnovel%252B4-phenoxy-6%25252C7-disubstituted%252Bquinolines%252Bpossessing%252Bsemicarbazones%252Bas%252Bc-Met%252Bkinase%252Binhibitors%26aulast%3DQi%26aufirst%3DBaohui%26date%3D2013%26volume%3D21%26issue%3D17%26spage%3D5246%26epage%3D5260" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Sai  Li</span>, <span class="hlFld-ContribAuthor ">Yanfang  Zhao</span>, <span class="hlFld-ContribAuthor ">Kewen  Wang</span>, <span class="hlFld-ContribAuthor ">Yali  Gao</span>, <span class="hlFld-ContribAuthor ">Jianming  Han</span>, <span class="hlFld-ContribAuthor ">Bingbing  Cui</span>, <span class="hlFld-ContribAuthor ">Ping  Gong</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of novel 4-(2-fluorophenoxy)quinoline derivatives bearing 4-oxo-1,4-dihydrocinnoline-3-carboxamide moiety as c-Met kinase inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry</span><span> <strong>2013,</strong> <em>21 </em>
                                    (11)
                                     , 2843-2855. <a href="https://doi.org/10.1016/j.bmc.2013.04.013" title="DOI URL">https://doi.org/10.1016/j.bmc.2013.04.013</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmc.2013.04.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmc.2013.04.013%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252Bnovel%252B4-%2525282-fluorophenoxy%252529quinoline%252Bderivatives%252Bbearing%252B4-oxo-1%25252C4-dihydrocinnoline-3-carboxamide%252Bmoiety%252Bas%252Bc-Met%252Bkinase%252Binhibitors%26aulast%3DLi%26aufirst%3DSai%26date%3D2013%26volume%3D21%26issue%3D11%26spage%3D2843%26epage%3D2855" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Belnaser A.  Busnena</span>, <span class="hlFld-ContribAuthor ">Ahmed I.  Foudah</span>, <span class="hlFld-ContribAuthor ">Tina  Melancon</span>, <span class="hlFld-ContribAuthor ">Khalid A.  El Sayed</span>. </span><span class="cited-content_cbyCitation_article-title">Olive secoiridoids and semisynthetic bioisostere analogues for the control of metastatic breast cancer. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry</span><span> <strong>2013,</strong> <em>21 </em>
                                    (7)
                                     , 2117-2127. <a href="https://doi.org/10.1016/j.bmc.2012.12.050" title="DOI URL">https://doi.org/10.1016/j.bmc.2012.12.050</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmc.2012.12.050&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmc.2012.12.050%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%26atitle%3DOlive%252Bsecoiridoids%252Band%252Bsemisynthetic%252Bbioisostere%252Banalogues%252Bfor%252Bthe%252Bcontrol%252Bof%252Bmetastatic%252Bbreast%252Bcancer%26aulast%3DBusnena%26aufirst%3DBelnaser%2BA.%26date%3D2013%26volume%3D21%26issue%3D7%26spage%3D2117%26epage%3D2127" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Neeta K.  Venepalli</span>, <span class="hlFld-ContribAuthor ">Laura  Goff</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting the HGF-cMET Axis in Hepatocellular Carcinoma. </span><span class="cited-content_cbyCitation_journal-name">International Journal of Hepatology</span><span> <strong>2013,</strong> <em>2013 </em>, 1-11. <a href="https://doi.org/10.1155/2013/341636" title="DOI URL">https://doi.org/10.1155/2013/341636</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1155/2013/341636&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1155%2F2013%2F341636%26sid%3Dliteratum%253Aachs%26jtitle%3DInternational%2520Journal%2520of%2520Hepatology%26atitle%3DTargeting%252Bthe%252BHGF-cMET%252BAxis%252Bin%252BHepatocellular%252BCarcinoma%26aulast%3DVenepalli%26aufirst%3DNeeta%2BK.%26date%3D2013%26volume%3D2013%26spage%3D1%26epage%3D11" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Fabiola  Cecchi</span>, <span class="hlFld-ContribAuthor ">Daniel C  Rabe</span>, <span class="hlFld-ContribAuthor ">Donald P  Bottaro</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting the HGF/Met signaling pathway in cancer therapy. </span><span class="cited-content_cbyCitation_journal-name">Expert Opinion on Therapeutic Targets</span><span> <strong>2012,</strong> <em>16 </em>
                                    (6)
                                     , 553-572. <a href="https://doi.org/10.1517/14728222.2012.680957" title="DOI URL">https://doi.org/10.1517/14728222.2012.680957</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1517/14728222.2012.680957&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1517%2F14728222.2012.680957%26sid%3Dliteratum%253Aachs%26jtitle%3DExpert%2520Opinion%2520on%2520Therapeutic%2520Targets%26atitle%3DTargeting%252Bthe%252BHGF%25252FMet%252Bsignaling%252Bpathway%252Bin%252Bcancer%252Btherapy%26aulast%3DCecchi%26aufirst%3DFabiola%26date%3D2012%26date%3D2012%26volume%3D16%26issue%3D6%26spage%3D553%26epage%3D572" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Peter  Ripphausen</span>, <span class="hlFld-ContribAuthor ">Dagmar  Stumpfe</span>, <span class="hlFld-ContribAuthor ">JÃ¼rgen  Bajorath</span>. </span><span class="cited-content_cbyCitation_article-title">Analysis of structure-based virtual screening studies and characterization of identified active compounds. </span><span class="cited-content_cbyCitation_journal-name">Future Medicinal Chemistry</span><span> <strong>2012,</strong> <em>4 </em>
                                    (5)
                                     , 603-613. <a href="https://doi.org/10.4155/fmc.12.18" title="DOI URL">https://doi.org/10.4155/fmc.12.18</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.4155/fmc.12.18&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.4155%2Ffmc.12.18%26sid%3Dliteratum%253Aachs%26jtitle%3DFuture%2520Medicinal%2520Chemistry%26atitle%3DAnalysis%252Bof%252Bstructure-based%252Bvirtual%252Bscreening%252Bstudies%252Band%252Bcharacterization%252Bof%252Bidentified%252Bactive%252Bcompounds%26aulast%3DRipphausen%26aufirst%3DPeter%26date%3D2012%26volume%3D4%26issue%3D5%26spage%3D603%26epage%3D613" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ailing  Zhao</span>, <span class="hlFld-ContribAuthor ">Xin  Gao</span>, <span class="hlFld-ContribAuthor ">Yuanxiang  Wang</span>, <span class="hlFld-ContribAuthor ">Jing  Ai</span>, <span class="hlFld-ContribAuthor ">Ying  Wang</span>, <span class="hlFld-ContribAuthor ">Yi  Chen</span>, <span class="hlFld-ContribAuthor ">Meiyu  Geng</span>, <span class="hlFld-ContribAuthor ">Ao  Zhang</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of novel c-Met kinase inhibitors bearing a thieno[2,3-d]pyrimidine or furo[2,3-d]pyrimidine scaffold. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry</span><span> <strong>2011,</strong> <em>19 </em>
                                    (13)
                                     , 3906-3918. <a href="https://doi.org/10.1016/j.bmc.2011.05.038" title="DOI URL">https://doi.org/10.1016/j.bmc.2011.05.038</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmc.2011.05.038&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmc.2011.05.038%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252Bnovel%252Bc-Met%252Bkinase%252Binhibitors%252Bbearing%252Ba%252Bthieno%25255B2%25252C3-d%25255Dpyrimidine%252Bor%252Bfuro%25255B2%25252C3-d%25255Dpyrimidine%252Bscaffold%26aulast%3DZhao%26aufirst%3DAiling%26date%3D2011%26volume%3D19%26issue%3D13%26spage%3D3906%26epage%3D3918" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Chenzhong  Liao</span>, <span class="hlFld-ContribAuthor ">Markus  Sitzmann</span>, <span class="hlFld-ContribAuthor ">Angelo  Pugliese</span>, <span class="hlFld-ContribAuthor ">Marc C  Nicklaus</span>. </span><span class="cited-content_cbyCitation_article-title">Software and resources for computational medicinal chemistry. </span><span class="cited-content_cbyCitation_journal-name">Future Medicinal Chemistry</span><span> <strong>2011,</strong> <em>3 </em>
                                    (8)
                                     , 1057-1085. <a href="https://doi.org/10.4155/fmc.11.63" title="DOI URL">https://doi.org/10.4155/fmc.11.63</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.4155/fmc.11.63&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.4155%2Ffmc.11.63%26sid%3Dliteratum%253Aachs%26jtitle%3DFuture%2520Medicinal%2520Chemistry%26atitle%3DSoftware%252Band%252Bresources%252Bfor%252Bcomputational%252Bmedicinal%252Bchemistry%26aulast%3DLiao%26aufirst%3DChenzhong%26date%3D2011%26volume%3D3%26issue%3D8%26spage%3D1057%26epage%3D1085" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Thomas  Lemcke</span>, <span class="hlFld-ContribAuthor ">Jan  Dreher</span>, <span class="hlFld-ContribAuthor ">Matthias  Rarey</span>, <span class="hlFld-ContribAuthor ">Frank  Totzke</span>, <span class="hlFld-ContribAuthor ">Christoph  SchÃ¤chtele</span>, <span class="hlFld-ContribAuthor ">Michael H. G.  Kubbutat</span>, <span class="hlFld-ContribAuthor ">Conrad  Kunick</span>. </span><span class="cited-content_cbyCitation_article-title">Identification of Inhibitors of the Tyrosine Kinase c-Met by Structure-Based Virtual Screening. </span><span class="cited-content_cbyCitation_journal-name">Molecular Informatics</span><span> <strong>2011,</strong> <em>30 </em>
                                    (2-3)
                                     , 145-150. <a href="https://doi.org/10.1002/minf.201000138" title="DOI URL">https://doi.org/10.1002/minf.201000138</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/minf.201000138&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fminf.201000138%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecular%2520Informatics%26atitle%3DIdentification%252Bof%252BInhibitors%252Bof%252Bthe%252BTyrosine%252BKinase%252Bc-Met%252Bby%252BStructure-Based%252BVirtual%252BScreening%26aulast%3DLemcke%26aufirst%3DThomas%26date%3D2011%26date%3D2011%26volume%3D30%26issue%3D2-3%26spage%3D145%26epage%3D150" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Natasja  Brooijmans</span>, <span class="hlFld-ContribAuthor ">Jason B.  Cross</span>, <span class="hlFld-ContribAuthor ">Christine  Humblet</span>. </span><span class="cited-content_cbyCitation_article-title">Biased retrieval of chemical series in receptor-based virtual screening. </span><span class="cited-content_cbyCitation_journal-name">Journal of Computer-Aided Molecular Design</span><span> <strong>2010,</strong> <em>24 </em>
                                    (12)
                                     , 1053-1062. <a href="https://doi.org/10.1007/s10822-010-9394-9" title="DOI URL">https://doi.org/10.1007/s10822-010-9394-9</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s10822-010-9394-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs10822-010-9394-9%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Computer-Aided%2520Molecular%2520Design%26atitle%3DBiased%252Bretrieval%252Bof%252Bchemical%252Bseries%252Bin%252Breceptor-based%252Bvirtual%252Bscreening%26aulast%3DBrooijmans%26aufirst%3DNatasja%26date%3D2010%26date%3D2010%26volume%3D24%26issue%3D12%26spage%3D1053%26epage%3D1062" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ciro Leonardo  Pierri</span>, <span class="hlFld-ContribAuthor ">Giovanni  Parisi</span>, <span class="hlFld-ContribAuthor ">Vito  Porcelli</span>. </span><span class="cited-content_cbyCitation_article-title">Computational approaches for protein function prediction: A combined strategy from multiple sequence alignment to molecular docking-based virtual screening. </span><span class="cited-content_cbyCitation_journal-name">Biochimica et Biophysica Acta (BBA) - Proteins and Proteomics</span><span> <strong>2010,</strong> <em>1804 </em>
                                    (9)
                                     , 1695-1712. <a href="https://doi.org/10.1016/j.bbapap.2010.04.008" title="DOI URL">https://doi.org/10.1016/j.bbapap.2010.04.008</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bbapap.2010.04.008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bbapap.2010.04.008%26sid%3Dliteratum%253Aachs%26jtitle%3DBiochimica%2520et%2520Biophysica%2520Acta%2520%2528BBA%2529%2520-%2520Proteins%2520and%2520Proteomics%26atitle%3DComputational%252Bapproaches%252Bfor%252Bprotein%252Bfunction%252Bprediction%25253A%252BA%252Bcombined%252Bstrategy%252Bfrom%252Bmultiple%252Bsequence%252Balignment%252Bto%252Bmolecular%252Bdocking-based%252Bvirtual%252Bscreening%26aulast%3DPierri%26aufirst%3DCiro%2BLeonardo%26date%3D2010%26volume%3D1804%26issue%3D9%26spage%3D1695%26epage%3D1712" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Natasja  Brooijmans</span>, <span class="hlFld-ContribAuthor ">Christine  Humblet</span>. </span><span class="cited-content_cbyCitation_article-title">Chemical space sampling by different scoring functions and crystal structures. </span><span class="cited-content_cbyCitation_journal-name">Journal of Computer-Aided Molecular Design</span><span> <strong>2010,</strong> <em>24 </em>
                                    (5)
                                     , 433-447. <a href="https://doi.org/10.1007/s10822-010-9356-2" title="DOI URL">https://doi.org/10.1007/s10822-010-9356-2</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s10822-010-9356-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs10822-010-9356-2%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Computer-Aided%2520Molecular%2520Design%26atitle%3DChemical%252Bspace%252Bsampling%252Bby%252Bdifferent%252Bscoring%252Bfunctions%252Band%252Bcrystal%252Bstructures%26aulast%3DBrooijmans%26aufirst%3DNatasja%26date%3D2010%26date%3D2010%26volume%3D24%26issue%3D5%26spage%3D433%26epage%3D447" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Megan L  Peach</span>, <span class="hlFld-ContribAuthor ">Marc C  Nicklaus</span>. </span><span class="cited-content_cbyCitation_article-title">Combining docking with pharmacophore filtering for improved virtual screening. </span><span class="cited-content_cbyCitation_journal-name">Journal of Cheminformatics</span><span> <strong>2009,</strong> <em>1 </em>
                                    (1)
                                     <a href="https://doi.org/10.1186/1758-2946-1-6" title="DOI URL">https://doi.org/10.1186/1758-2946-1-6</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1186/1758-2946-1-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1186%2F1758-2946-1-6%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Cheminformatics%26atitle%3DCombining%252Bdocking%252Bwith%252Bpharmacophore%252Bfiltering%252Bfor%252Bimproved%252Bvirtual%252Bscreening%26aulast%3DPeach%26aufirst%3DMegan%2BL%26date%3D2009%26date%3D2009%26volume%3D1%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Lynette A.  Smyth</span>, <span class="hlFld-ContribAuthor ">Ian  Collins</span>. </span><span class="cited-content_cbyCitation_article-title">Measuring and interpreting the selectivity of protein kinase inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Biology</span><span> <strong>2009,</strong> <em>2 </em>
                                    (3)
                                     , 131-151. <a href="https://doi.org/10.1007/s12154-009-0023-9" title="DOI URL">https://doi.org/10.1007/s12154-009-0023-9</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s12154-009-0023-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs12154-009-0023-9%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Biology%26atitle%3DMeasuring%252Band%252Binterpreting%252Bthe%252Bselectivity%252Bof%252Bprotein%252Bkinase%252Binhibitors%26aulast%3DSmyth%26aufirst%3DLynette%2BA.%26date%3D2009%26date%3D2009%26volume%3D2%26issue%3D3%26spage%3D131%26epage%3D151" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="f_tgr1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-4/jm800791f/production/images/medium/jm-2008-00791f_0005.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-4/jm800791f/production/images/large/jm-2008-00791f_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm800791f&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-4/jm800791f/production/images/medium/jm-2008-00791f_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-4/jm800791f/production/images/large/jm-2008-00791f_0004.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Flowchart of the virtual screening procedure. The number of compounds at each stage, from the starting point of a large database of small molecules, a crystal structure, and a set of known inhibitors to the end point of a small set of compounds proposed for biological screening, is listed.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-4/jm800791f/production/images/large/jm-2008-00791f_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm800791f&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-4/jm800791f/production/images/medium/jm-2008-00791f_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-4/jm800791f/production/images/large/jm-2008-00791f_0003.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Topographical features of the Met ATP binding site used to filter docked poses. An interaction with each defined region of the binding site was required for a successfully docked compound.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-4/jm800791f/production/images/large/jm-2008-00791f_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm800791f&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-4/jm800791f/production/images/medium/jm-2008-00791f_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-4/jm800791f/production/images/large/jm-2008-00791f_0002.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. (A) Chemical structure of compound <b>2</b>. (B) Predicted binding orientation and residue interactions of compound <b>2</b>. (C) Chemical structure of compound <b>3</b>. (D) Predicted binding orientation and residue interactions of compound <b>3</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-4/jm800791f/production/images/large/jm-2008-00791f_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm800791f&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-4/jm800791f/production/images/medium/jm-2008-00791f_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-4/jm800791f/production/images/large/jm-2008-00791f_0001.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. (A) Chemical structure of compound <b>1</b>. (B) Predicted binding orientation and residue interactions of compound <b>1</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-4/jm800791f/production/images/large/jm-2008-00791f_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm800791f&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-4/jm800791f/production/images/medium/jm-2008-00791f_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-4/jm800791f/production/images/large/jm-2008-00791f_0007.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. (A, C) Overlay of the docked position of compound <b>1</b> (gray) with the crystal structure of Met inhibitor K252-a (orange). (B, D) Overlay of the docked positions of compound <b>2</b> (gray) with the Met inhibitor PF-02341066 (blue). The orientation of the compounds in views A and B is the same as that shown in Figures <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a> and <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a>, whereas views C and D are rotated by 90Â° about the <i>X</i> axis for clarity in visualizing the overlay.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-4/jm800791f/production/images/large/jm-2008-00791f_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm800791f&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-4/jm800791f/production/images/medium/jm-2008-00791f_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-4/jm800791f/production/images/large/jm-2008-00791f_0006.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Peptide backbone structural models of the ATP binding sites of (A) Met, (B) FGFR1, and (C) IGFR1 with compound <b>1</b> docked in the ATP binding site. The hinge region is shown in red. The solvent-excluded volume in the binding site, as calculated by the program PASS,<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> is outlined in yellow mesh. This figure was created using PyMOL.<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2009/jmcmar.2009.52.issue-4/jm800791f/production/images/large/jm-2008-00791f_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm800791f&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i23">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_22336" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_22336" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 40 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group">Ma, P. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maulik, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Christensen, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Salgia, R.</span><span> </span><span class="NLM_article-title">c-Met: structure, functions and potential for therapeutic inhibition</span> <span class="citation_source-journal">Cancer Metastasis Rev.</span><span class="NLM_year" style="font-weight: bold;">2003</span> <span class="NLM_volume">22</span> <span class="NLM_fpage">309</span> <span class="NLM_lpage">325</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Fjm800791f&amp;key=10.1023%2FA%3A1023768811842" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Fjm800791f&amp;key=12884908" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Fjm800791f&amp;key=1%3ACAS%3A528%3ADC%252BD3sXjslyiurY%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2003&pages=309-325&author=P.+C.+Maauthor=G.+Maulikauthor=J.+Christensenauthor=R.+Salgia&title=c-Met%3A+structure%2C+functions+and+potential+for+therapeutic+inhibition"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">c-Met: Structure, functions and potential for therapeutic inhibition</span></div><div class="casAuthors">Ma, Patrick C.; Maulik, Gautam; Christensen, James; Salgia, Ravi</div><div class="citationInfo"><span class="NLM_cas:title">Cancer and Metastasis Reviews</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">309-325</span>CODEN:
                <span class="NLM_cas:coden">CMRED4</span>;
        ISSN:<span class="NLM_cas:issn">0167-7659</span>.
    
            (<span class="NLM_cas:orgname">Kluwer Academic Publishers</span>)
        </div><div class="casAbstract">A review.  Studies on signal transduction pathways have generated various promising mol. targets for therapeutic inhibition in cancer therapy.  Receptor tyrosine kinases represent an important class of such therapeutic targets.  C-Met is a receptor tyrosine kinase that has been shown to be overexpressed and/or mutated in a variety of malignancies.  A no. of c-Met activating mutations, many of which are located in the tyrosine kinase domain, have been detected in various solid tumors and have been implicated in invasion and metastasis of tumor cells.  It is known that stimulation of c-Met via its natural ligand, hepatocyte growth factor (also known as scatter factor, HGF/SF) results in a plethora of biol. and biochem. effects in the cell.  Activation of c-Met signaling can lead to scattering, angiogenesis, proliferation, enhanced cell motility, invasion, and eventual metastasis.  In this review, the role of c-Met dysregulation in tumor progression and metastasis is discussed in detail with particular emphasis on c-Met mutations.  Moreover, we summarize current knowledge on various pathways of c-Met signal transduction, highlighting the central role in the cytoskeletal functions.  In this summary is included recent data in our lab. indicating that phosphorylation of focal adhesion proteins, such as paxillin, p125FAK, and PYK2, occurs in response to c-Met stimulation in lung cancer cells.  Most importantly, current data on c-Met suggest that when mutated or overexpressed in malignant cells, c-Met would serve as an important therapeutic target.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo9rIdb5Ao4a7Vg90H21EOLACvtfcHk0lgdYs_DZx_vTw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXjslyiurY%253D&md5=7b7bf60da872071a39ce313efd32fd1f</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1023%2FA%3A1023768811842&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1023%252FA%253A1023768811842%26sid%3Dliteratum%253Aachs%26aulast%3DMa%26aufirst%3DP.%2BC.%26aulast%3DMaulik%26aufirst%3DG.%26aulast%3DChristensen%26aufirst%3DJ.%26aulast%3DSalgia%26aufirst%3DR.%26atitle%3Dc-Met%253A%2520structure%252C%2520functions%2520and%2520potential%2520for%2520therapeutic%2520inhibition%26jtitle%3DCancer%2520Metastasis%2520Rev.%26date%3D2003%26volume%3D22%26spage%3D309%26epage%3D325" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group">Wang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Le, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liang, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chan, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kiewlich, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miller, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Harris, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sun, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rice, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vasile, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blake, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Howlett, A. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Patel, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McMahon, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lipson, K. E.</span><span> </span><span class="NLM_article-title">Potent and selective inhibitors of the Met [hepatocyte growth factor/scatter factor (HGF/SF) receptor] tyrosine kinase block HGF/SF-induced tumor cell growth and invasion</span> <span class="citation_source-journal">Mol. Cancer Ther.</span><span class="NLM_year" style="font-weight: bold;">2003</span> <span class="NLM_volume">2</span> <span class="NLM_fpage">1085</span> <span class="NLM_lpage">1092</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Fjm800791f&amp;key=14617781" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Fjm800791f&amp;key=1%3ACAS%3A528%3ADC%252BD3sXovFCqtL8%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2003&pages=1085-1092&author=X.+Wangauthor=P.+Leauthor=C.+Liangauthor=J.+Chanauthor=D.+Kiewlichauthor=T.+Millerauthor=D.+Harrisauthor=L.+Sunauthor=A.+Riceauthor=S.+Vasileauthor=R.+A.+Blakeauthor=A.+R.+Howlettauthor=N.+Patelauthor=G.+McMahonauthor=K.+E.+Lipson&title=Potent+and+selective+inhibitors+of+the+Met+%5Bhepatocyte+growth+factor%2Fscatter+factor+%28HGF%2FSF%29+receptor%5D+tyrosine+kinase+block+HGF%2FSF-induced+tumor+cell+growth+and+invasion"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Potent and selective inhibitors of the Met [hepatocyte growth factor/scatter factor (HGF/SF) receptor] tyrosine kinase block HGF/SF-induced tumor cell growth and invasion</span></div><div class="casAuthors">Wang, Xueyan; Le, Phuong; Liang, Congxin; Chan, Julie; Kiewlich, David; Miller, Todd; Harris, Dave; Sun, Li; Rice, Audie; Vasile, Stefan; Blake, Robert A.; Howlett, Anthony R.; Patel, Neela; McMahon, Gerald; Lipson, Kenneth E.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cancer Therapeutics</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1085-1092</span>CODEN:
                <span class="NLM_cas:coden">MCTOCF</span>;
        ISSN:<span class="NLM_cas:issn">1535-7163</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">The hepatocyte growth factor/scatter factor (HGF/SF) receptor, Met, mediates various cellular responses on activation with its ligand, including proliferation, survival, motility, invasion, and tubular morphogenesis.  Met expression is frequently up-regulated in sarcomas and carcinomas.  Exptl. evidence suggests that Met activation correlates with poor clin. outcome and the likelihood of metastasis.  Therefore, inhibitors of Met tyrosine kinase may be useful for the treatment of a wide variety of cancers that have spread from the primary site.  We have discovered potent and selective pyrrole-indolinone Met kinase inhibitors and characterized them for their ability to inhibit HGF/SF-induced cellular responses in vitro.  These compds. inhibit HGF/SF-induced receptor phosphorylation in a dose-dependent manner.  They also inhibit the HGF/SF-induced motility and invasion of epithelial and carcinoma cells.  Therefore, these compds. represent a class of prototype small mols. that selectively inhibit the Met kinase and could lead to identification of compds. with potential therapeutic utility in treatment of cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoVAQRRxT_J1rVg90H21EOLACvtfcHk0lgWM1uV3rsKSw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXovFCqtL8%253D&md5=3b3141600b15ed4f9c97d86fb7ade81d</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DX.%26aulast%3DLe%26aufirst%3DP.%26aulast%3DLiang%26aufirst%3DC.%26aulast%3DChan%26aufirst%3DJ.%26aulast%3DKiewlich%26aufirst%3DD.%26aulast%3DMiller%26aufirst%3DT.%26aulast%3DHarris%26aufirst%3DD.%26aulast%3DSun%26aufirst%3DL.%26aulast%3DRice%26aufirst%3DA.%26aulast%3DVasile%26aufirst%3DS.%26aulast%3DBlake%26aufirst%3DR.%2BA.%26aulast%3DHowlett%26aufirst%3DA.%2BR.%26aulast%3DPatel%26aufirst%3DN.%26aulast%3DMcMahon%26aufirst%3DG.%26aulast%3DLipson%26aufirst%3DK.%2BE.%26atitle%3DPotent%2520and%2520selective%2520inhibitors%2520of%2520the%2520Met%2520%255Bhepatocyte%2520growth%2520factor%252Fscatter%2520factor%2520%2528HGF%252FSF%2529%2520receptor%255D%2520tyrosine%2520kinase%2520block%2520HGF%252FSF-induced%2520tumor%2520cell%2520growth%2520and%2520invasion%26jtitle%3DMol.%2520Cancer%2520Ther.%26date%3D2003%26volume%3D2%26spage%3D1085%26epage%3D1092" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group">Schiering, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knapp, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marconi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Flocco, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cui, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perego, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rusconi, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cristiani, C.</span><span> </span><span class="NLM_article-title">Crystal structure of the tyrosine kinase domain of the hepatocyte growth factor receptor c-Met and its complex with the microbial alkaloid K-252a</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U.S.A.</span><span class="NLM_year" style="font-weight: bold;">2003</span> <span class="NLM_volume">100</span> <span class="NLM_fpage">12654</span> <span class="NLM_lpage">12659</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Fjm800791f&amp;key=10.1073%2Fpnas.1734128100" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Fjm800791f&amp;key=14559966" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Fjm800791f&amp;key=1%3ACAS%3A528%3ADC%252BD3sXoslKnt74%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=100&publication_year=2003&pages=12654-12659&author=N.+Schieringauthor=S.+Knappauthor=M.+Marconiauthor=M.+M.+Floccoauthor=J.+Cuiauthor=R.+Peregoauthor=L.+Rusconiauthor=C.+Cristiani&title=Crystal+structure+of+the+tyrosine+kinase+domain+of+the+hepatocyte+growth+factor+receptor+c-Met+and+its+complex+with+the+microbial+alkaloid+K-252a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal structure of the tyrosine kinase domain of the hepatocyte growth factor receptor c-Met and its complex with the microbial alkaloid K-252a</span></div><div class="casAuthors">Schiering, Nikolaus; Knapp, Stefan; Marconi, Marina; Flocco, Maria M.; Cui, Jean; Perego, Rita; Rusconi, Luisa; Cristiani, Cinzia</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">100</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">12654-12659</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The protooncogene c-met codes for the hepatocyte growth factor receptor tyrosine kinase.  Binding of its ligand, hepatocyte growth factor/scatter factor, stimulates receptor autophosphorylation, which leads to pleiotropic downstream signaling events in epithelial cells, including cell growth, motility, and invasion.  These events are mediated by interaction of cytoplasmic effectors, generally through Src homol. 2 (SH2) domains, with two phosphotyrosine-contg. sequence motifs in the unique C-terminal tail of c-Met (supersite).  2 (SH2) domains, with two phosphotyrosine-contg. sequence motifs in the unique C-terminal tail of c-Met (supersite).  There is a strong link between aberrant c-Met activity and oncogenesis, which makes this kinase an important cancer drug target.  The furanosylated indolocarbazole K-252a belongs to a family of microbial alkaloids that also includes staurosporine.  It was recently shown to be a potent inhibitor of c-Met.  Here we report the crystal structures of an unphosphorylated c-Met kinase domain harboring a human cancer mutation and its complex with K-252a at 1.8-Ã resoln.  The structure follows the well established architecture of protein kinases.  It adopts a unique, inhibitory conformation of the activation loop, a catalytically noncompetent orientation of helix Î±C, and reveals the complete C-terminal docking site.  The first SH2-binding motif (1349YVHV) adopts an extended conformation, whereas the second motif (1356YVNV), a binding site for Grb2-SH2, folds as a type II Î²-turn.  The intermediate portion of the supersite (1353NATY) assumes a type I Î²-turn conformation as in an Shc-phosphotyrosine binding domain peptide complex.  K-252a is bound in the adenosine pocket with an analogous binding mode to those obsd. in previously reported structures of protein kinases in complex with staurosporine.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo13Yf3cKB-ybVg90H21EOLACvtfcHk0lgWM1uV3rsKSw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXoslKnt74%253D&md5=d8cd3494f2583dfcb11efb6862aff607</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1734128100&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1734128100%26sid%3Dliteratum%253Aachs%26aulast%3DSchiering%26aufirst%3DN.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DMarconi%26aufirst%3DM.%26aulast%3DFlocco%26aufirst%3DM.%2BM.%26aulast%3DCui%26aufirst%3DJ.%26aulast%3DPerego%26aufirst%3DR.%26aulast%3DRusconi%26aufirst%3DL.%26aulast%3DCristiani%26aufirst%3DC.%26atitle%3DCrystal%2520structure%2520of%2520the%2520tyrosine%2520kinase%2520domain%2520of%2520the%2520hepatocyte%2520growth%2520factor%2520receptor%2520c-Met%2520and%2520its%2520complex%2520with%2520the%2520microbial%2520alkaloid%2520K-252a%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2003%26volume%3D100%26spage%3D12654%26epage%3D12659" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group">Gould, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wong, C. F.</span><span> </span><span class="NLM_article-title">Designing specific protein kinase inhibitors: insights from computer simulations and comparative sequence/structure analysis</span> <span class="citation_source-journal">Pharmacol. Ther.</span><span class="NLM_year" style="font-weight: bold;">2002</span> <span class="NLM_volume">93</span> <span class="NLM_fpage">169</span> <span class="NLM_lpage">178</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Fjm800791f&amp;key=10.1016%2FS0163-7258%2802%2900186-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Fjm800791f&amp;key=12191609" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Fjm800791f&amp;key=1%3ACAS%3A528%3ADC%252BD38XksVShtbc%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=93&publication_year=2002&pages=169-178&author=C.+Gouldauthor=C.+F.+Wong&title=Designing+specific+protein+kinase+inhibitors%3A+insights+from+computer+simulations+and+comparative+sequence%2Fstructure+analysis"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Designing specific protein kinase inhibitors: insights from computer simulations and comparative sequence/structure analysis</span></div><div class="casAuthors">Gould, Christine; Wong, Chung F.</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacology & Therapeutics</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">93</span>
        (<span class="NLM_cas:issue">2-3</span>),
    <span class="NLM_cas:pages">169-178</span>CODEN:
                <span class="NLM_cas:coden">PHTHDT</span>;
        ISSN:<span class="NLM_cas:issn">0163-7258</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Inc.</span>)
        </div><div class="casAbstract">A review.  Protein kinases are important targets for designing therapeutic drugs.  We describe here a computational approach to extend the usefulness of a single protein-inhibitor structure in aiding the design of protein kinase inhibitors.  This approach is based on using sensitivity anal. to identify the most significant functional groups of a lead compd. in accounting for binding affinity and on using comparative sequence/structure anal. to examine whether these functional groups would present specificity.  A sensitivity anal. study is similar to genetic or chem. modification expts. in which specific features of a lead compd. are modified to examine whether they affect properties such as binding affinity.  In this study, the binding affinity was estd. by using an implicit-solvent model in which the electrostatic contributions were obtained by solving the Poisson equation, and the hydrophobic effects were accounted for by using surface-area-dependent terms.  The comparative sequence/structure anal. involves the study of the amino acid distributions of a large no. of protein kinases (384 in this study) near the ligand-binding sites.  This anal. provides useful guiding principles for designing specific inhibitors targeted towards a particular kinase.  Here, we illustrate the utility of these computational approaches by applying them to identify the determinants of the recognition between the protein kinase A and two of its inhibitors.  One inhibitor, balanol, binds to the ATP-binding pocket.  The other, protein kinase inhibitor, binds to the substrate-binding site.  These analyses have helped to construct pharmacophore models for mining new drug leads from small-mol. libraries and for suggesting how a lead compd. or a peptide inhibitor may be modified to generate selective inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpD4cNyidIWgLVg90H21EOLACvtfcHk0lgWM1uV3rsKSw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XksVShtbc%253D&md5=81c5c5809f3bedad54f9bc0559aca972</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1016%2FS0163-7258%2802%2900186-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0163-7258%252802%252900186-9%26sid%3Dliteratum%253Aachs%26aulast%3DGould%26aufirst%3DC.%26aulast%3DWong%26aufirst%3DC.%2BF.%26atitle%3DDesigning%2520specific%2520protein%2520kinase%2520inhibitors%253A%2520insights%2520from%2520computer%2520simulations%2520and%2520comparative%2520sequence%252Fstructure%2520analysis%26jtitle%3DPharmacol.%2520Ther.%26date%3D2002%26volume%3D93%26spage%3D169%26epage%3D178" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group">Noble, M. E. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Endicott, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johnson, L. N.</span><span> </span><span class="NLM_article-title">Protein kinase inhibitors: insights into drug design from structure</span> <span class="citation_source-journal">Science</span><span class="NLM_year" style="font-weight: bold;">2004</span> <span class="NLM_volume">303</span> <span class="NLM_fpage">1800</span> <span class="NLM_lpage">1805</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Fjm800791f&amp;key=10.1126%2Fscience.1095920" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Fjm800791f&amp;key=15031492" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Fjm800791f&amp;key=1%3ACAS%3A528%3ADC%252BD2cXitFehurc%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=303&publication_year=2004&pages=1800-1805&author=M.+E.+M.+Nobleauthor=J.+A.+Endicottauthor=L.+N.+Johnson&title=Protein+kinase+inhibitors%3A+insights+into+drug+design+from+structure"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Protein Kinase Inhibitors: Insights into Drug Design from Structure</span></div><div class="casAuthors">Noble, Martin E. M.; Endicott, Jane A.; Johnson, Louise N.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">303</span>
        (<span class="NLM_cas:issue">5665</span>),
    <span class="NLM_cas:pages">1800-1805</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">A review.  Protein kinases are targets for treatment of a no. of diseases.  This review focuses on kinase inhibitors that are in the clinic or in clin. trials and for which structural information is available.  Structures have informed drug design and have illuminated the mechanism of inhibition.  We review progress with the receptor tyrosine kinases (growth factor receptors EGFR, VEGFR, and FGFR) and nonreceptor tyrosine kinases (Bcr-Abl), where advances have been made with cancer therapeutic agents such as Herceptin and Gleevec.  Among the serine-threonine kinases, p38, Rho-kinase, cyclin-dependent kinases, and Chk1 have been targeted with productive results for inflammation and cancer.  Structures have provided insights into targeting the inactive or active form of the kinase, for targeting the global constellation of residues at the ATP site or less conserved addnl. pockets or single residues, and into targeting noncatalytic domains.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWqtKz7cvN_rVg90H21EOLACvtfcHk0lh-HPLA8tDSNw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXitFehurc%253D&md5=7fd3f60bee46e59307436eeed285d6ae</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1126%2Fscience.1095920&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1095920%26sid%3Dliteratum%253Aachs%26aulast%3DNoble%26aufirst%3DM.%2BE.%2BM.%26aulast%3DEndicott%26aufirst%3DJ.%2BA.%26aulast%3DJohnson%26aufirst%3DL.%2BN.%26atitle%3DProtein%2520kinase%2520inhibitors%253A%2520insights%2520into%2520drug%2520design%2520from%2520structure%26jtitle%3DScience%26date%3D2004%26volume%3D303%26spage%3D1800%26epage%3D1805" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group">GarcÃ­a-EcheverrÃ­a, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Traxler, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Evans, D. B.</span><span> </span><span class="NLM_article-title">ATP site-directed competitive and irreversible inhibitors of protein kinases</span> <span class="citation_source-journal">Med. Res. Rev.</span><span class="NLM_year" style="font-weight: bold;">2000</span> <span class="NLM_volume">20</span> <span class="NLM_fpage">28</span> <span class="NLM_lpage">57</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Fjm800791f&amp;key=10.1002%2F%28SICI%291098-1128%28200001%2920%3A1%3C28%3A%3AAID-MED2%3E3.0.CO%3B2-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Fjm800791f&amp;key=10608920" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Fjm800791f&amp;key=1%3ACAS%3A528%3ADC%252BD3cXltFWruw%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2000&pages=28-57&author=C.+Garc%C3%ADa-Echeverr%C3%ADaauthor=P.+Traxlerauthor=D.+B.+Evans&title=ATP+site-directed+competitive+and+irreversible+inhibitors+of+protein+kinases"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">ATP site-directed competitive and irreversible inhibitors of protein kinases</span></div><div class="casAuthors">Garcia-Echeverria, Carlos; Traxler, Peter; Evans, Dean B.</div><div class="citationInfo"><span class="NLM_cas:title">Medicinal Research Reviews</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">28-57</span>CODEN:
                <span class="NLM_cas:coden">MRREDD</span>;
        ISSN:<span class="NLM_cas:issn">0198-6325</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">A review with 220 refs.  Several tyrosine and serine/threonine protein kinases have emerged in the last few years as attractive targets in the search for new therapeutic agents being applicable in many different disease indications.  Initially, inhibition of these protein kinases by ATP site-directed inhibitors was considered less prone to success, but medicinal chemists from both academia and industry have been able to impart potency and selectivity to a limited no. of scaffolds by modulating and fine-tuning the interactions of the modified template with the ATP binding site of the selected kinase.  The chem. templates that have been used in the synthesis of ATP site-directed protein kinase inhibitors are reviewed with emphasis on the kinase inhibitors that have entered or are about to enter clin. trials.  Examples have been selected to illustrate how structure-based design approaches and new methods to increase compd. diversity have had an impact on this area of research.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqRnYxd5Xp0UrVg90H21EOLACvtfcHk0lh-HPLA8tDSNw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXltFWruw%253D%253D&md5=f610b09ddf97aeb2cc9f4b930d7c27d7</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1002%2F%28SICI%291098-1128%28200001%2920%3A1%3C28%3A%3AAID-MED2%3E3.0.CO%3B2-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F%2528SICI%25291098-1128%2528200001%252920%253A1%253C28%253A%253AAID-MED2%253E3.0.CO%253B2-2%26sid%3Dliteratum%253Aachs%26aulast%3DGarc%25C3%25ADa-Echeverr%25C3%25ADa%26aufirst%3DC.%26aulast%3DTraxler%26aufirst%3DP.%26aulast%3DEvans%26aufirst%3DD.%2BB.%26atitle%3DATP%2520site-directed%2520competitive%2520and%2520irreversible%2520inhibitors%2520of%2520protein%2520kinases%26jtitle%3DMed.%2520Res.%2520Rev.%26date%3D2000%26volume%3D20%26spage%3D28%26epage%3D57" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group">Jones, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Willett, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Glen, R. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leach, A. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taylor, R.</span><span> </span><span class="NLM_article-title">Development and validation of a genetic algorithm for flexible docking</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_year" style="font-weight: bold;">1997</span> <span class="NLM_volume">267</span> <span class="NLM_fpage">727</span> <span class="NLM_lpage">748</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Fjm800791f&amp;key=10.1006%2Fjmbi.1996.0897" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Fjm800791f&amp;key=9126849" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Fjm800791f&amp;key=1%3ACAS%3A528%3ADyaK2sXis1KntLo%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=267&publication_year=1997&pages=727-748&author=G.+Jonesauthor=P.+Willettauthor=R.+C.+Glenauthor=A.+R.+Leachauthor=R.+Taylor&title=Development+and+validation+of+a+genetic+algorithm+for+flexible+docking"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Development and validation of a genetic algorithm for flexible docking</span></div><div class="casAuthors">Jones, Gareth; Willett, Peter; Glen, Robert C.; Leach, Andrew R.; Taylor, Robin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">267</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">727-748</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Academic</span>)
        </div><div class="casAbstract">Prediction of small mol. binding modes to macromols. of known three-dimensional structure is a problem of paramount importance in rational drug design (the "docking" problem).  We report the development and validation of the program GOLD (Genetic Optimization for Ligand Docking).  GOLD is an automated ligand docking program that uses a genetic algorithm to explore the full range of ligand conformational flexibility with partial flexibility of the protein and satisfies the fundamental requirement that the ligand must displace loosely bound water on binding.  Numerous enhancements and modifications have been applied to the original technique resulting in a substantial increase in the reliability and the applicability of the algorithm.  The advanced algorithm has been tested on a dataset of 100 complexes extd. from the Brookhaven Protein Data Bank.  When used to dock the ligand back into the binding site, GOLD achieved a 71% success rate in identifying the exptl. binding mode.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrP3I60Rl2uorVg90H21EOLACvtfcHk0lh-HPLA8tDSNw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXis1KntLo%253D&md5=476a2b1d8f80f3ba418052fe29d735ca</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1006%2Fjmbi.1996.0897&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1006%252Fjmbi.1996.0897%26sid%3Dliteratum%253Aachs%26aulast%3DJones%26aufirst%3DG.%26aulast%3DWillett%26aufirst%3DP.%26aulast%3DGlen%26aufirst%3DR.%2BC.%26aulast%3DLeach%26aufirst%3DA.%2BR.%26aulast%3DTaylor%26aufirst%3DR.%26atitle%3DDevelopment%2520and%2520validation%2520of%2520a%2520genetic%2520algorithm%2520for%2520flexible%2520docking%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D1997%26volume%3D267%26spage%3D727%26epage%3D748" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group">Warren, G. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Andrews, C. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Capelli, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clarke, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">LaLonde, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lambert, M. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lindvall, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nevins, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Semus, S. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Senger, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tedesco, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wall, I. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Woolven, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peishoff, C. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Head, M. S.</span><span> </span><span class="NLM_article-title">A critical assessment of docking programs and scoring functions</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_year" style="font-weight: bold;">2006</span> <span class="NLM_volume">49</span> <span class="NLM_fpage">5912</span> <span class="NLM_lpage">5931</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm050362n" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Fjm800791f&amp;key=1%3ACAS%3A528%3ADC%252BD2MXnslOrtrY%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2006&pages=5912-5931&author=G.+L.+Warrenauthor=C.+W.+Andrewsauthor=A.+M.+Capelliauthor=B.+Clarkeauthor=J.+LaLondeauthor=M.+H.+Lambertauthor=M.+Lindvallauthor=N.+Nevinsauthor=S.+F.+Semusauthor=S.+Sengerauthor=G.+Tedescoauthor=I.+D.+Wallauthor=J.+M.+Woolvenauthor=C.+E.+Peishoffauthor=M.+S.+Head&title=A+critical+assessment+of+docking+programs+and+scoring+functions"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">A Critical Assessment of Docking Programs and Scoring Functions</span></div><div class="casAuthors">Warren, Gregory L.; Andrews, C. Webster; Capelli, Anna-Maria; Clarke, Brian; LaLonde, Judith; Lambert, Millard H.; Lindvall, Mika; Nevins, Neysa; Semus, Simon F.; Senger, Stefan; Tedesco, Giovanna; Wall, Ian D.; Woolven, James M.; Peishoff, Catherine E.; Head, Martha S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">5912-5931</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Docking is a computational technique that samples conformations of small mols. in protein binding sites; scoring functions are used to assess which of these conformations best complements the protein binding site.  An evaluation of 10 docking programs and 37 scoring functions was conducted against eight proteins of seven protein types for three tasks: binding mode prediction, virtual screening for lead identification, and rank-ordering by affinity for lead optimization.  All of the docking programs were able to generate ligand conformations similar to crystallog. detd. protein/ligand complex structures for at least one of the targets.  However, scoring functions were less successful at distinguishing the crystallog. conformation from the set of docked poses.  Docking programs identified active compds. from a pharmaceutically relevant pool of decoy compds.; however, no single program performed well for all of the targets.  For prediction of compd. affinity, none of the docking programs or scoring functions made a useful prediction of ligand binding affinity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpfJn8_WHSjurVg90H21EOLACvtfcHk0li2vORDXQ0S5A"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXnslOrtrY%253D&md5=463ab40c3d23e67b62df408fc5e7f08d</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1021%2Fjm050362n&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm050362n%26sid%3Dliteratum%253Aachs%26aulast%3DWarren%26aufirst%3DG.%2BL.%26aulast%3DAndrews%26aufirst%3DC.%2BW.%26aulast%3DCapelli%26aufirst%3DA.%2BM.%26aulast%3DClarke%26aufirst%3DB.%26aulast%3DLaLonde%26aufirst%3DJ.%26aulast%3DLambert%26aufirst%3DM.%2BH.%26aulast%3DLindvall%26aufirst%3DM.%26aulast%3DNevins%26aufirst%3DN.%26aulast%3DSemus%26aufirst%3DS.%2BF.%26aulast%3DSenger%26aufirst%3DS.%26aulast%3DTedesco%26aufirst%3DG.%26aulast%3DWall%26aufirst%3DI.%2BD.%26aulast%3DWoolven%26aufirst%3DJ.%2BM.%26aulast%3DPeishoff%26aufirst%3DC.%2BE.%26aulast%3DHead%26aufirst%3DM.%2BS.%26atitle%3DA%2520critical%2520assessment%2520of%2520docking%2520programs%2520and%2520scoring%2520functions%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2006%26volume%3D49%26spage%3D5912%26epage%3D5931" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group">Mohamadi, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richards, N. G. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guida, W. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liskamp, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lipton, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Caufield, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chang, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hendrickson, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Still, W. C.</span><span> </span><span class="NLM_article-title">Macromodel, an integrated software system for modeling organic and bioorganic molecules using molecular mechanics</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_year" style="font-weight: bold;">1990</span> <span class="NLM_volume">11</span> <span class="NLM_fpage">440</span> <span class="NLM_lpage">467</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=1990&pages=440-467&author=F.+Mohamadiauthor=N.+G.+J.+Richardsauthor=W.+C.+Guidaauthor=R.+Liskampauthor=M.+Liptonauthor=C.+Caufieldauthor=G.+Changauthor=T.+Hendricksonauthor=W.+C.+Still&title=Macromodel%2C+an+integrated+software+system+for+modeling+organic+and+bioorganic+molecules+using+molecular+mechanics"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMohamadi%26aufirst%3DF.%26aulast%3DRichards%26aufirst%3DN.%2BG.%2BJ.%26aulast%3DGuida%26aufirst%3DW.%2BC.%26aulast%3DLiskamp%26aufirst%3DR.%26aulast%3DLipton%26aufirst%3DM.%26aulast%3DCaufield%26aufirst%3DC.%26aulast%3DChang%26aufirst%3DG.%26aulast%3DHendrickson%26aufirst%3DT.%26aulast%3DStill%26aufirst%3DW.%2BC.%26atitle%3DMacromodel%252C%2520an%2520integrated%2520software%2520system%2520for%2520modeling%2520organic%2520and%2520bioorganic%2520molecules%2520using%2520molecular%2520mechanics%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D1990%26volume%3D11%26spage%3D440%26epage%3D467" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group">Lipinski, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lombardo, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dominy, B. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feeney, P. J.</span><span> </span><span class="NLM_article-title">Experimental and computational approaches to estimate solubility and permeability in drug discovery and development settings</span> <span class="citation_source-journal">Adv. Drug Delivery Rev.</span><span class="NLM_year" style="font-weight: bold;">1997</span> <span class="NLM_volume">23</span> <span class="NLM_fpage">3</span> <span class="NLM_lpage">25</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Fjm800791f&amp;key=10.1016%2FS0169-409X%2896%2900423-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Fjm800791f&amp;key=1%3ACAS%3A528%3ADyaK2sXktlKlsQ%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=1997&pages=3-25&author=C.+A.+Lipinskiauthor=F.+Lombardoauthor=B.+W.+Dominyauthor=P.+J.+Feeney&title=Experimental+and+computational+approaches+to+estimate+solubility+and+permeability+in+drug+discovery+and+development+settings"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Experimental and computational approaches to estimate solubility and permeability in drug discovery and development settings</span></div><div class="casAuthors">Lipinski, Christopher A.; Lombardo, Franco; Dominy, Beryl W.; Feeney, Paul J.</div><div class="citationInfo"><span class="NLM_cas:title">Advanced Drug Delivery Reviews</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">1-3</span>),
    <span class="NLM_cas:pages">3-25</span>CODEN:
                <span class="NLM_cas:coden">ADDREP</span>;
        ISSN:<span class="NLM_cas:issn">0169-409X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">A review with 50 refs.  Exptl. and computational approaches to est. soly. and permeability in discovery and development settings are described.  In the discovery setting 'the rule of 5' predicts that poor absorption or permeation is more likely when there are more than 5 H-bond donors, 10 H-bond acceptors, the mol. wt. (MWT) is >500 and the calcd. Log P (CLogP) is >5.  Computational methodol. for the rule-based Moriguchi Log P (MLogP) calcn. is described.  Turbidimetric soly. measurement is described and applied to known drugs.  High throughput screening (HTS) leads tend to have higher MWT and Log P and lower turbidimetric soly. than leads in the pre-HTS era.  In the development setting, soly. calcns. focus on exact value prediction and are difficult because of polymorphism.  Recent work on linear free energy relationships and Log P approaches are critically reviewed.  Useful predictions are possible in closely related analog series when coupled with exptl. thermodn. soly. measurements.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGon9gEeZ7cboLVg90H21EOLACvtfcHk0li2vORDXQ0S5A"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXktlKlsQ%253D%253D&md5=405f70b0594d428f1275e1d56642cd3a</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1016%2FS0169-409X%2896%2900423-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0169-409X%252896%252900423-1%26sid%3Dliteratum%253Aachs%26aulast%3DLipinski%26aufirst%3DC.%2BA.%26aulast%3DLombardo%26aufirst%3DF.%26aulast%3DDominy%26aufirst%3DB.%2BW.%26aulast%3DFeeney%26aufirst%3DP.%2BJ.%26atitle%3DExperimental%2520and%2520computational%2520approaches%2520to%2520estimate%2520solubility%2520and%2520permeability%2520in%2520drug%2520discovery%2520and%2520development%2520settings%26jtitle%3DAdv.%2520Drug%2520Delivery%2520Rev.%26date%3D1997%26volume%3D23%26spage%3D3%26epage%3D25" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group">Veber, D. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johnson, S. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheng, H. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, B. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ward, K. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kopple, K. D.</span><span> </span><span class="NLM_article-title">Molecular properties that influence the oral bioavailability of drug candidates</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_year" style="font-weight: bold;">2002</span> <span class="NLM_volume">45</span> <span class="NLM_fpage">2615</span> <span class="NLM_lpage">2623</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm020017n" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Fjm800791f&amp;key=1%3ACAS%3A528%3ADC%252BD38XjsFCmt7g%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2002&pages=2615-2623&author=D.+F.+Veberauthor=S.+R.+Johnsonauthor=H.+Y.+Chengauthor=B.+R.+Smithauthor=K.+W.+Wardauthor=K.+D.+Kopple&title=Molecular+properties+that+influence+the+oral+bioavailability+of+drug+candidates"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular Properties That Influence the Oral Bioavailability of Drug Candidates</span></div><div class="casAuthors">Veber, Daniel F.; Johnson, Stephen R.; Cheng, Hung-Yuan; Smith, Brian R.; Ward, Keith W.; Kopple, Kenneth D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2615-2623</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Oral bioavailability measurements in rats for over 1100 drug candidates studied at Smith-Kline Beecham Pharmaceuticals (now Glaxo Smith-Kline) have allowed us to analyze the relative importance of mol. properties considered to influence that drug property.  Reduced mol. flexibility, as measured by the no. of rotatable bonds, and low polar surface area or total hydrogen bond count (sum of donors and acceptors) are found to be important predictors of good oral bioavailability, independent of mol. wt.  That on av. both the no. of rotatable bonds and polar surface area or hydrogen bond count tend to increase with mol. wt. may in part explain the success of the mol. wt. parameter in predicting oral bioavailability.  The commonly applied mol. wt. cutoff at 500 does not itself significantly sep. compds. with poor oral bioavailability from those with acceptable values in this extensive data set.  Our observations suggest that compds. which meet only the 2 criteria of (1) 10 or fewer rotatable bonds and (2) polar surface area â¤140 Ã2 (or 12 or fewer H-bond donors and acceptors) will have a high probability of good oral bioavailability in the rat.  Data sets for the artificial membrane permeation rate and for clearance in the rat were also examd.  Reduced polar surface area correlates better with increased permeation rate than does lipophilicity (C log P), and increased rotatable bond count has a neg. effect on the permeation rate.  A threshold permeation rate is a prerequisite of oral bioavailability.  The rotatable bond count does not correlate with the data examd. here for the in vivo clearance rate in the rat.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr3JtJEkO7yVLVg90H21EOLACvtfcHk0li2vORDXQ0S5A"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XjsFCmt7g%253D&md5=eaad26ed6a259de82ad65a8834fc397d</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1021%2Fjm020017n&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm020017n%26sid%3Dliteratum%253Aachs%26aulast%3DVeber%26aufirst%3DD.%2BF.%26aulast%3DJohnson%26aufirst%3DS.%2BR.%26aulast%3DCheng%26aufirst%3DH.%2BY.%26aulast%3DSmith%26aufirst%3DB.%2BR.%26aulast%3DWard%26aufirst%3DK.%2BW.%26aulast%3DKopple%26aufirst%3DK.%2BD.%26atitle%3DMolecular%2520properties%2520that%2520influence%2520the%2520oral%2520bioavailability%2520of%2520drug%2520candidates%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2002%26volume%3D45%26spage%3D2615%26epage%3D2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group">Bellon, S. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kaplan-Lefko, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moriguchi, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rex, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johnson, C. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rose, P. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Long, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">OâConnor, A. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Coxon, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, T. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tasker, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Burgess, T. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dussault, I.</span><span> </span><span class="NLM_article-title">c-Met inhibitors with novel binding mode show activity against several hereditary papillary renal cell carcinoma related mutations</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_year" style="font-weight: bold;">2008</span> <span class="NLM_volume">283</span> <span class="NLM_fpage">2675</span> <span class="NLM_lpage">2683</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=283&publication_year=2008&pages=2675-2683&author=S.+F.+Bellonauthor=P.+Kaplan-Lefkoauthor=Y.+Yangauthor=Y.+Zhangauthor=J.+Moriguchiauthor=K.+Rexauthor=C.+W.+Johnsonauthor=P.+E.+Roseauthor=A.+M.+Longauthor=A.+B.+O%E2%80%99Connorauthor=Y.+Guauthor=A.+Coxonauthor=T.+S.+Kimauthor=A.+Taskerauthor=T.+L.+Burgessauthor=I.+Dussault&title=c-Met+inhibitors+with+novel+binding+mode+show+activity+against+several+hereditary+papillary+renal+cell+carcinoma+related+mutations"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBellon%26aufirst%3DS.%2BF.%26aulast%3DKaplan-Lefko%26aufirst%3DP.%26aulast%3DYang%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DMoriguchi%26aufirst%3DJ.%26aulast%3DRex%26aufirst%3DK.%26aulast%3DJohnson%26aufirst%3DC.%2BW.%26aulast%3DRose%26aufirst%3DP.%2BE.%26aulast%3DLong%26aufirst%3DA.%2BM.%26aulast%3DO%25E2%2580%2599Connor%26aufirst%3DA.%2BB.%26aulast%3DGu%26aufirst%3DY.%26aulast%3DCoxon%26aufirst%3DA.%26aulast%3DKim%26aufirst%3DT.%2BS.%26aulast%3DTasker%26aufirst%3DA.%26aulast%3DBurgess%26aufirst%3DT.%2BL.%26aulast%3DDussault%26aufirst%3DI.%26atitle%3Dc-Met%2520inhibitors%2520with%2520novel%2520binding%2520mode%2520show%2520activity%2520against%2520several%2520hereditary%2520papillary%2520renal%2520cell%2520carcinoma%2520related%2520mutations%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2008%26volume%3D283%26spage%3D2675%26epage%3D2683" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group">Albrecht, B. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Harmange, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bauer, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Berry, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bode, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boezio, A. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Choquette, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dussault, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fridrich, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hirai, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hoffman, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Larrow, J. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kaplan-Lefko, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lohman, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Long, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moriguchi, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">OâConnor, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Potashman, M. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reese, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rex, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Siegmund, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shah, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shimanovich, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Springer, S. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Teffera, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bellon, S. F.</span><span> </span><span class="NLM_article-title">Discovery and optimization of triazolopyridazines as potent and selective inhibitors of the c-Met kinase</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_year" style="font-weight: bold;">2008</span> <span class="NLM_volume">51</span> <span class="NLM_fpage">2879</span> <span class="NLM_lpage">2882</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm800043g" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Fjm800791f&amp;key=1%3ACAS%3A528%3ADC%252BD1cXkvFShtLk%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2008&pages=2879-2882&author=B.+K.+Albrechtauthor=J.+C.+Harmangeauthor=D.+Bauerauthor=L.+Berryauthor=C.+Bodeauthor=A.+A.+Boezioauthor=A.+Chenauthor=D.+Choquetteauthor=I.+Dussaultauthor=C.+Fridrichauthor=S.+Hiraiauthor=D.+Hoffmanauthor=J.+F.+Larrowauthor=P.+Kaplan-Lefkoauthor=J.+Linauthor=J.+Lohmanauthor=A.+M.+Longauthor=J.+Moriguchiauthor=A.+O%E2%80%98Connorauthor=M.+H.+Potashmanauthor=M.+Reeseauthor=K.+Rexauthor=A.+Siegmundauthor=K.+Shahauthor=R.+Shimanovichauthor=S.+K.+Springerauthor=Y.+Tefferaauthor=Y.+Yangauthor=Y.+Zhangauthor=S.+F.+Bellon&title=Discovery+and+optimization+of+triazolopyridazines+as+potent+and+selective+inhibitors+of+the+c-Met+kinase"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery and Optimization of Triazolopyridazines as Potent and Selective Inhibitors of the c-Met Kinase</span></div><div class="casAuthors">Albrecht, Brian K.; Harmange, Jean-Christophe; Bauer, David; Berry, Loren; Bode, Christiane; Boezio, Alessandro A.; Chen, April; Choquette, Deborah; Dussault, Isabelle; Fridrich, Cary; Hirai, Satoko; Hoffman, Doug; Larrow, Jay F.; Kaplan-Lefko, Paula; Lin, Jasmine; Lohman, Julia; Long, Alexander M.; Moriguchi, Jodi; O'Connor, Anne; Potashman, Michele H.; Reese, Monica; Rex, Karen; Siegmund, Aaron; Shah, Kavita; Shimanovich, Roman; Springer, Stephanie K.; Teffera, Yohannes; Yang, Yajing; Zhang, Yihong; Bellon, Steven F.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2879-2882</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Tumorigenesis is a multistep process in which oncogenes play a key role in tumor formation, growth, and maintenance.  MET was discovered as an oncogene that is activated by its ligand, hepatocyte growth factor.  Deregulated signaling in the c-Met pathway has been obsd. in multiple tumor types.  Herein we report the discovery of potent and selective triazolopyridazine small mols. that inhibit c-Met activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoVeQ8sYpasHLVg90H21EOLACvtfcHk0lgqjGOwnR3Img"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXkvFShtLk%253D&md5=c0aec3ca102d90869884c139372f60a0</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1021%2Fjm800043g&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm800043g%26sid%3Dliteratum%253Aachs%26aulast%3DAlbrecht%26aufirst%3DB.%2BK.%26aulast%3DHarmange%26aufirst%3DJ.%2BC.%26aulast%3DBauer%26aufirst%3DD.%26aulast%3DBerry%26aufirst%3DL.%26aulast%3DBode%26aufirst%3DC.%26aulast%3DBoezio%26aufirst%3DA.%2BA.%26aulast%3DChen%26aufirst%3DA.%26aulast%3DChoquette%26aufirst%3DD.%26aulast%3DDussault%26aufirst%3DI.%26aulast%3DFridrich%26aufirst%3DC.%26aulast%3DHirai%26aufirst%3DS.%26aulast%3DHoffman%26aufirst%3DD.%26aulast%3DLarrow%26aufirst%3DJ.%2BF.%26aulast%3DKaplan-Lefko%26aufirst%3DP.%26aulast%3DLin%26aufirst%3DJ.%26aulast%3DLohman%26aufirst%3DJ.%26aulast%3DLong%26aufirst%3DA.%2BM.%26aulast%3DMoriguchi%26aufirst%3DJ.%26aulast%3DO%25E2%2580%2598Connor%26aufirst%3DA.%26aulast%3DPotashman%26aufirst%3DM.%2BH.%26aulast%3DReese%26aufirst%3DM.%26aulast%3DRex%26aufirst%3DK.%26aulast%3DSiegmund%26aufirst%3DA.%26aulast%3DShah%26aufirst%3DK.%26aulast%3DShimanovich%26aufirst%3DR.%26aulast%3DSpringer%26aufirst%3DS.%2BK.%26aulast%3DTeffera%26aufirst%3DY.%26aulast%3DYang%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DBellon%26aufirst%3DS.%2BF.%26atitle%3DDiscovery%2520and%2520optimization%2520of%2520triazolopyridazines%2520as%2520potent%2520and%2520selective%2520inhibitors%2520of%2520the%2520c-Met%2520kinase%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2008%26volume%3D51%26spage%3D2879%26epage%3D2882" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group">Zou, H. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, J. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arango, M. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McDonnell, S. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yamazaki, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Koudriakova, T. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Alton, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cui, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kung, P. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nambu, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Los, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bender, S. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mroczkowski, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Christensen, J. G.</span><span> </span><span class="NLM_article-title">An orally available small-molecule inhibitor of c-Met, PF-2341066, exhibits cytoreductive antitumor efficacy through antiproliferative and antiangiogenic mechanisms</span> <span class="citation_source-journal">Cancer Res.</span><span class="NLM_year" style="font-weight: bold;">2007</span> <span class="NLM_volume">67</span> <span class="NLM_fpage">4408</span> <span class="NLM_lpage">4417</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Fjm800791f&amp;key=10.1158%2F0008-5472.CAN-06-4443" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Fjm800791f&amp;key=17483355" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Fjm800791f&amp;key=1%3ACAS%3A528%3ADC%252BD2sXltVyrsbg%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2007&pages=4408-4417&author=H.+Y.+Zouauthor=Q.+Liauthor=J.+H.+Leeauthor=M.+E.+Arangoauthor=S.+R.+McDonnellauthor=S.+Yamazakiauthor=T.+B.+Koudriakovaauthor=G.+Altonauthor=J.+J.+Cuiauthor=P.+P.+Kungauthor=M.+D.+Nambuauthor=G.+Losauthor=S.+L.+Benderauthor=B.+Mroczkowskiauthor=J.+G.+Christensen&title=An+orally+available+small-molecule+inhibitor+of+c-Met%2C+PF-2341066%2C+exhibits+cytoreductive+antitumor+efficacy+through+antiproliferative+and+antiangiogenic+mechanisms"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">An Orally Available Small-Molecule Inhibitor of c-Met, PF-2341066, Exhibits Cytoreductive Antitumor Efficacy through Antiproliferative and Antiangiogenic Mechanisms</span></div><div class="casAuthors">Zou, Helen Y.; Li, Qiuhua; Lee, Joseph H.; Arango, Maria E.; McDonnell, Scott R.; Yamazaki, Shinji; Koudriakova, Tatiana B.; Alton, Gordon; Cui, Jingrong J.; Kung, Pei-Pei; Nambu, Mitchell D.; Los, Gerrit; Bender, Steven L.; Mroczkowski, Barbara; Christensen, James G.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">4408-4417</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">The c-Met receptor tyrosine kinase and its ligand, hepatocyte growth factor (HGF), have been implicated in the progression of several human cancers and are attractive therapeutic targets.  PF-2341066 was identified as a potent, orally bioavailable, ATP-competitive small-mol. inhibitor of the catalytic activity of c-Met kinase.  PF-2341066 was selective for c-Met (and anaplastic lymphoma kinase) compared with a panel of >120 diverse tyrosine and serine-threonine kinases.  PF-2341066 potently inhibited c-Met phosphorylation and c-Met-dependent proliferation, migration, or invasion of human tumor cells in vitro (IC50 values, 5-20 nmol/L).  In addn., PF-2341066 potently inhibited HGF-stimulated endothelial cell survival or invasion and serum-stimulated tubulogenesis in vitro, suggesting that this agent also exhibits antiangiogenic properties.  PF-2341066 showed efficacy at well-tolerated doses, including marked cytoreductive antitumor activity, in several tumor models that expressed activated c-Met.  The antitumor efficacy of PF-2341066 was dose dependent and showed a strong correlation to inhibition of c-Met phosphorylation in vivo.  Near-maximal inhibition of c-Met activity for the full dosing interval was necessary to maximize the efficacy of PF-2341066.  Addnl. mechanism-of-action studies showed dose-dependent inhibition of c-Met-dependent signal transduction, tumor cell proliferation (Ki67), induction of apoptosis (caspase-3), and redn. of microvessel d. (CD31).  These results indicated that the antitumor activity of PF-2341066 may be mediated by direct effects on tumor cell growth or survival as well as antiangiogenic mechanisms.  Collectively, these results show the therapeutic potential of targeting c-Met with selective small-mol. inhibitors for the treatment of human cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrEAWtGPOBHC7Vg90H21EOLACvtfcHk0lgqjGOwnR3Img"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXltVyrsbg%253D&md5=798bf93e6cf8d5e9897e27485cba9002</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-06-4443&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-06-4443%26sid%3Dliteratum%253Aachs%26aulast%3DZou%26aufirst%3DH.%2BY.%26aulast%3DLi%26aufirst%3DQ.%26aulast%3DLee%26aufirst%3DJ.%2BH.%26aulast%3DArango%26aufirst%3DM.%2BE.%26aulast%3DMcDonnell%26aufirst%3DS.%2BR.%26aulast%3DYamazaki%26aufirst%3DS.%26aulast%3DKoudriakova%26aufirst%3DT.%2BB.%26aulast%3DAlton%26aufirst%3DG.%26aulast%3DCui%26aufirst%3DJ.%2BJ.%26aulast%3DKung%26aufirst%3DP.%2BP.%26aulast%3DNambu%26aufirst%3DM.%2BD.%26aulast%3DLos%26aufirst%3DG.%26aulast%3DBender%26aufirst%3DS.%2BL.%26aulast%3DMroczkowski%26aufirst%3DB.%26aulast%3DChristensen%26aufirst%3DJ.%2BG.%26atitle%3DAn%2520orally%2520available%2520small-molecule%2520inhibitor%2520of%2520c-Met%252C%2520PF-2341066%252C%2520exhibits%2520cytoreductive%2520antitumor%2520efficacy%2520through%2520antiproliferative%2520and%2520antiangiogenic%2520mechanisms%26jtitle%3DCancer%2520Res.%26date%3D2007%26volume%3D67%26spage%3D4408%26epage%3D4417" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group">Sattler, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pride, Y. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ma, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gramlich, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chu, S. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Quinnan, L. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shirazian, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liang, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Podar, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Christensen, J. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Salgia, R.</span><span> </span><span class="NLM_article-title">A novel small molecule met inhibitor induces apoptosis in cells transformed by the oncogenic TPR-MET tyrosine kinase</span> <span class="citation_source-journal">Cancer Res.</span><span class="NLM_year" style="font-weight: bold;">2003</span> <span class="NLM_volume">63</span> <span class="NLM_fpage">5462</span> <span class="NLM_lpage">5469</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Fjm800791f&amp;key=14500382" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Fjm800791f&amp;key=1%3ACAS%3A528%3ADC%252BD3sXntleku7s%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2003&pages=5462-5469&author=M.+Sattlerauthor=Y.+B.+Prideauthor=P.+Maauthor=J.+L.+Gramlichauthor=S.+C.+Chuauthor=L.+A.+Quinnanauthor=S.+Shirazianauthor=C.+Liangauthor=K.+Podarauthor=J.+G.+Christensenauthor=R.+Salgia&title=A+novel+small+molecule+met+inhibitor+induces+apoptosis+in+cells+transformed+by+the+oncogenic+TPR-MET+tyrosine+kinase"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">A Novel Small Molecule Met Inhibitor Induces Apoptosis in Cells Transformed by the Oncogenic TPR-MET Tyrosine Kinase</span></div><div class="casAuthors">Sattler, Martin; Pride, Yuri B.; Ma, Patrick; Gramlich, Jessica L.; Chu, Stephanie C.; Quinnan, Laura A.; Shirazian, Sheri; Liang, Congxin; Podar, Klaus; Christensen, James G.; Salgia, Ravi</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">5462-5469</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">The Met receptor tyrosine kinase has been shown to be overexpressed or mutated in a variety of solid tumors and has, therefore, been identified as a good candidate for molecularly targeted therapy.  Activation of the Met tyrosine kinase by the TPR gene was originally described in vitro through carcinogen-induced rearrangement.  The TPR-MET fusion protein contains constitutively elevated Met tyrosine kinase activity and constitutes an ideal model to study the transforming activity of the Met kinase.  We found, when introduced into an interleukin 3-dependent cell line, TPR-MET induces factor independence and constitutive tyrosine phosphorylation of several cellular proteins.  One major tyrosine phosphorylated protein was identified as the TPR-MET oncoprotein itself.  Inhibition of the Met kinase activity by the novel small mol. drug SU11274 [(3Z)-N-(3-chlorophenyl)-3-({3,5-dimethyl-4-[(4-methylpiperazin-1-yl)carbonyl]-1H-pyrrol-2-yl}methylene)-N-methyl-2-oxo-2,3-dihydro-1H-indole-5-sulfonamide] led to time- and dose-dependent reduced cell growth.  The inhibitor did not affect other tyrosine kinase oncoproteins, including BCR-ABL, TEL-JAK2, TEL-PDGFÎ²R, or TEL-ABL.  The Met inhibitor induced G1 cell cycle arrest and apoptosis with increased Annexin V staining and caspase 3 activity.  The autophosphorylation of the Met kinase was reduced on sites that have been shown previously to be important for activation of pathways involved in cell growth and survival, esp. the phosphatidylinositol-3'-kinase and the Ras pathway.  In particular, we found that the inhibitor blocked phosphorylation of AKT, GSK-3Î², and the pro-apoptotic transcription factor FKHR.  The characterization of SU11274 as an effective inhibitor of Met tyrosine kinase activity illustrates the potential of targeting for Met therapeutic use in cancers assocd. with activated forms of this kinase.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrGvIwRT4z1IbVg90H21EOLACvtfcHk0lh7LobcVPIIYQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXntleku7s%253D&md5=3223841b665e6b95f3dab0f18ce64dbe</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DSattler%26aufirst%3DM.%26aulast%3DPride%26aufirst%3DY.%2BB.%26aulast%3DMa%26aufirst%3DP.%26aulast%3DGramlich%26aufirst%3DJ.%2BL.%26aulast%3DChu%26aufirst%3DS.%2BC.%26aulast%3DQuinnan%26aufirst%3DL.%2BA.%26aulast%3DShirazian%26aufirst%3DS.%26aulast%3DLiang%26aufirst%3DC.%26aulast%3DPodar%26aufirst%3DK.%26aulast%3DChristensen%26aufirst%3DJ.%2BG.%26aulast%3DSalgia%26aufirst%3DR.%26atitle%3DA%2520novel%2520small%2520molecule%2520met%2520inhibitor%2520induces%2520apoptosis%2520in%2520cells%2520transformed%2520by%2520the%2520oncogenic%2520TPR-MET%2520tyrosine%2520kinase%26jtitle%3DCancer%2520Res.%26date%3D2003%26volume%3D63%26spage%3D5462%26epage%3D5469" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group">Christensen, J. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schreck, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Burrows, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuruganti, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chan, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Le, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ruslim, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blake, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lipson, K. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ramphal, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Do, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cui, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cherrington, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mendel, D. B.</span><span> </span><span class="NLM_article-title">A selective small molecule inhibitor of c-Met kinase inhibits c-Met-dependent phenotypes in vitro and exhibits cytoreductive antitumor activity in vivo</span> <span class="citation_source-journal">Cancer Res.</span><span class="NLM_year" style="font-weight: bold;">2003</span> <span class="NLM_volume">63</span> <span class="NLM_fpage">7345</span> <span class="NLM_lpage">7355</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2003&pages=7345-7355&author=J.+G.+Christensenauthor=R.+Schreckauthor=J.+Burrowsauthor=P.+Kurugantiauthor=E.+Chanauthor=P.+Leauthor=J.+Chenauthor=X.+Wangauthor=L.+Ruslimauthor=R.+Blakeauthor=K.+E.+Lipsonauthor=J.+Ramphalauthor=S.+Doauthor=J.+J.+Cuiauthor=J.+M.+Cherringtonauthor=D.+B.+Mendel&title=A+selective+small+molecule+inhibitor+of+c-Met+kinase+inhibits+c-Met-dependent+phenotypes+in+vitro+and+exhibits+cytoreductive+antitumor+activity+in+vivo"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DChristensen%26aufirst%3DJ.%2BG.%26aulast%3DSchreck%26aufirst%3DR.%26aulast%3DBurrows%26aufirst%3DJ.%26aulast%3DKuruganti%26aufirst%3DP.%26aulast%3DChan%26aufirst%3DE.%26aulast%3DLe%26aufirst%3DP.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DRuslim%26aufirst%3DL.%26aulast%3DBlake%26aufirst%3DR.%26aulast%3DLipson%26aufirst%3DK.%2BE.%26aulast%3DRamphal%26aufirst%3DJ.%26aulast%3DDo%26aufirst%3DS.%26aulast%3DCui%26aufirst%3DJ.%2BJ.%26aulast%3DCherrington%26aufirst%3DJ.%2BM.%26aulast%3DMendel%26aufirst%3DD.%2BB.%26atitle%3DA%2520selective%2520small%2520molecule%2520inhibitor%2520of%2520c-Met%2520kinase%2520inhibits%2520c-Met-dependent%2520phenotypes%2520in%2520vitro%2520and%2520exhibits%2520cytoreductive%2520antitumor%2520activity%2520in%2520vivo%26jtitle%3DCancer%2520Res.%26date%3D2003%26volume%3D63%26spage%3D7345%26epage%3D7355" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group">Dharmawardana, P. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Giubellino, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bottaro, D. P.</span><span> </span><span class="NLM_article-title">Hereditary papillary renal carcinoma type I</span> <span class="citation_source-journal">Curr. Mol. Med.</span><span class="NLM_year" style="font-weight: bold;">2004</span> <span class="NLM_volume">4</span> <span class="NLM_fpage">855</span> <span class="NLM_lpage">868</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Fjm800791f&amp;key=10.2174%2F1566524043359674" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Fjm800791f&amp;key=15579033" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Fjm800791f&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVGlsbzF" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2004&pages=855-868&author=P.+G.+Dharmawardanaauthor=A.+Giubellinoauthor=D.+P.+Bottaro&title=Hereditary+papillary+renal+carcinoma+type+I"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Hereditary papillary renal carcinoma type I</span></div><div class="casAuthors">Dharmawardana, Pathirage G.; Giubellino, Alessio; Bottaro, Donald P.</div><div class="citationInfo"><span class="NLM_cas:title">Current Molecular Medicine</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">855-868</span>CODEN:
                <span class="NLM_cas:coden">CMMUBP</span>;
        ISSN:<span class="NLM_cas:issn">1566-5240</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Germline missense mutations in the tyrosine kinase domain of the hepatocyte growth factor/scatter factor (HGF/SF) receptor, c-Met, are thought to be responsible for hereditary papillary renal carcinoma (HPRC) type 1, a form of human kidney cancer.  In addn. to extensive linkage anal. of HPRC families localizing the HPRC type 1 gene within chromosome 7, the demonstration that individual c-Met mutations reconstituted in cultured cells display enhanced and dysregulated kinase activity, and confer cell transformation and tumorigenicity in mice, solidifies this conclusion.  Our prior knowledge of HGF/SF biol. and c-Met signaling enabled rapid progress in unraveling the mol. pathogenesis of HPRC type 1, and in laying the framework for the development of novel therapeutics for the treatment of this cancer.  At the same time, the study of HPRC type 1 has refined our appreciation of the oncogenic potential of c-Met signaling, and challenges our current understanding of HGF/SF and c-Met function in health and disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrY89l4mqksLLVg90H21EOLACvtfcHk0lh7LobcVPIIYQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVGlsbzF&md5=cba15709d949ccfddcd52dccb7e88ffc</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.2174%2F1566524043359674&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F1566524043359674%26sid%3Dliteratum%253Aachs%26aulast%3DDharmawardana%26aufirst%3DP.%2BG.%26aulast%3DGiubellino%26aufirst%3DA.%26aulast%3DBottaro%26aufirst%3DD.%2BP.%26atitle%3DHereditary%2520papillary%2520renal%2520carcinoma%2520type%2520I%26jtitle%3DCurr.%2520Mol.%2520Med.%26date%3D2004%26volume%3D4%26spage%3D855%26epage%3D868" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group">Berthou, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aebersold, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schmidt, L. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stroka, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heigl, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Streit, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stalder, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gruber, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liang, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Howlett, A. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Candinas, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Greiner, R. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lipson, K. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zimmer, Y.</span><span> </span><span class="NLM_article-title">The Met kinase inhibitor SU11274 exhibits a selective inhibition pattern toward different receptor mutated variants</span> <span class="citation_source-journal">Oncogene</span><span class="NLM_year" style="font-weight: bold;">2004</span> <span class="NLM_volume">23</span> <span class="NLM_fpage">5387</span> <span class="NLM_lpage">5393</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2004&pages=5387-5393&author=S.+Berthouauthor=D.+M.+Aebersoldauthor=L.+S.+Schmidtauthor=D.+Strokaauthor=C.+Heiglauthor=B.+Streitauthor=D.+Stalderauthor=G.+Gruberauthor=C.+Liangauthor=A.+R.+Howlettauthor=D.+Candinasauthor=R.+H.+Greinerauthor=K.+E.+Lipsonauthor=Y.+Zimmer&title=The+Met+kinase+inhibitor+SU11274+exhibits+a+selective+inhibition+pattern+toward+different+receptor+mutated+variants"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBerthou%26aufirst%3DS.%26aulast%3DAebersold%26aufirst%3DD.%2BM.%26aulast%3DSchmidt%26aufirst%3DL.%2BS.%26aulast%3DStroka%26aufirst%3DD.%26aulast%3DHeigl%26aufirst%3DC.%26aulast%3DStreit%26aufirst%3DB.%26aulast%3DStalder%26aufirst%3DD.%26aulast%3DGruber%26aufirst%3DG.%26aulast%3DLiang%26aufirst%3DC.%26aulast%3DHowlett%26aufirst%3DA.%2BR.%26aulast%3DCandinas%26aufirst%3DD.%26aulast%3DGreiner%26aufirst%3DR.%2BH.%26aulast%3DLipson%26aufirst%3DK.%2BE.%26aulast%3DZimmer%26aufirst%3DY.%26atitle%3DThe%2520Met%2520kinase%2520inhibitor%2520SU11274%2520exhibits%2520a%2520selective%2520inhibition%2520pattern%2520toward%2520different%2520receptor%2520mutated%2520variants%26jtitle%3DOncogene%26date%3D2004%26volume%3D23%26spage%3D5387%26epage%3D5393" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group">Liu, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wen, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jorissen, R. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gilson, M. K.</span><span> </span><span class="NLM_article-title">BindingDB: a Web-accessible database of experimentally determined proteinâligand binding affinities</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_year" style="font-weight: bold;">2007</span> <span class="NLM_volume">35</span> <span class="NLM_fpage">D198</span> <span class="NLM_lpage">D201</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2007&pages=D198-D201&author=T.+Liuauthor=Y.+Linauthor=X.+Wenauthor=R.+N.+Jorissenauthor=M.+K.+Gilson&title=BindingDB%3A+a+Web-accessible+database+of+experimentally+determined+protein%E2%88%92ligand+binding+affinities"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DT.%26aulast%3DLin%26aufirst%3DY.%26aulast%3DWen%26aufirst%3DX.%26aulast%3DJorissen%26aufirst%3DR.%2BN.%26aulast%3DGilson%26aufirst%3DM.%2BK.%26atitle%3DBindingDB%253A%2520a%2520Web-accessible%2520database%2520of%2520experimentally%2520determined%2520protein%25E2%2588%2592ligand%2520binding%2520affinities%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2007%26volume%3D35%26spage%3DD198%26epage%3DD201" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group">Rogers, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, R. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hahn, M.</span><span> </span><span class="NLM_article-title">Using extended-connectivity fingerprints with Laplacian-modified Bayesian analysis in high-throughput screening follow-up</span> <span class="citation_source-journal">J. Biomol. Screening</span><span class="NLM_year" style="font-weight: bold;">2005</span> <span class="NLM_volume">10</span> <span class="NLM_fpage">682</span> <span class="NLM_lpage">686</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Fjm800791f&amp;key=10.1177%2F1087057105281365" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Fjm800791f&amp;key=16170046" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Fjm800791f&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtF2qs77L" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2005&pages=682-686&author=D.+Rogersauthor=R.+D.+Brownauthor=M.+Hahn&title=Using+extended-connectivity+fingerprints+with+Laplacian-modified+Bayesian+analysis+in+high-throughput+screening+follow-up"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Using extended-connectivity fingerprints with Laplacian-modified Bayesian analysis in high-throughput screening follow-up</span></div><div class="casAuthors">Rogers, David; Brown, Robert D.; Hahn, Mathew</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biomolecular Screening</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">682-686</span>CODEN:
                <span class="NLM_cas:coden">JBISF3</span>;
        ISSN:<span class="NLM_cas:issn">1087-0571</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">This article describes the use of a combination of extended-connectivity fingerprints (ECFPs) and Laplacian-modified Bayesian anal. in a study of the inhibition of Escherichia coli dihydrofolate reductase.  The McMaster High-Throughput Screening Lab at McMaster University proposed a competition to predict the hits in a sep. test set of 50,000 compds.  Although the problem seemed best approached with 3D methods, the authors show that 2D methods offer surprisingly competitive results with a low computational cost.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq9tK9Lo95V-7Vg90H21EOLACvtfcHk0lgjWoFyMIT9gQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtF2qs77L&md5=b8600cd46275811e81d40dcc2c438c99</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1177%2F1087057105281365&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1087057105281365%26sid%3Dliteratum%253Aachs%26aulast%3DRogers%26aufirst%3DD.%26aulast%3DBrown%26aufirst%3DR.%2BD.%26aulast%3DHahn%26aufirst%3DM.%26atitle%3DUsing%2520extended-connectivity%2520fingerprints%2520with%2520Laplacian-modified%2520Bayesian%2520analysis%2520in%2520high-throughput%2520screening%2520follow-up%26jtitle%3DJ.%2520Biomol.%2520Screening%26date%3D2005%26volume%3D10%26spage%3D682%26epage%3D686" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group">Willett, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barnard, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Downs, G. M.</span><span> </span><span class="NLM_article-title">Chemical similarity searching</span> <span class="citation_source-journal">J. Chem. Inf. Comput. Sci.</span><span class="NLM_year" style="font-weight: bold;">1998</span> <span class="NLM_volume">38</span> <span class="NLM_fpage">983</span> <span class="NLM_lpage">996</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci9800211" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Fjm800791f&amp;key=1%3ACAS%3A528%3ADyaK1cXks1ynu74%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=1998&pages=983-996&author=P.+Willettauthor=J.+M.+Barnardauthor=G.+M.+Downs&title=Chemical+similarity+searching"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Chemical similarity searching</span></div><div class="casAuthors">Willett, Peter; Barnard, John M.; Downs, Geoffrey M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Computer Sciences</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">983-996</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">0095-2338</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">This paper reviews the use of similarity searching in chem. databases.  It begins by introducing the concept of similarity searching, differentiating it from the more common substructure searching, and then discusses the current generation of fragment-based measures that are used for searching chem. structure databases.  The next sections focus upon two of the principal characteristics of a similarity measure: the coeff. that is used to quantify the degree of structural resemblance between pairs of mols. and the structural representations that are used to characterize mols. that are being compared in a similarity calcn.  New types of similarity measure are then compared with current approaches, and examples are given of several applications that are related to similarity searching.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpxjjSk407VCLVg90H21EOLACvtfcHk0lgjWoFyMIT9gQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXks1ynu74%253D&md5=4233b0ea4edc052f0186cccbd5eae427</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1021%2Fci9800211&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci9800211%26sid%3Dliteratum%253Aachs%26aulast%3DWillett%26aufirst%3DP.%26aulast%3DBarnard%26aufirst%3DJ.%2BM.%26aulast%3DDowns%26aufirst%3DG.%2BM.%26atitle%3DChemical%2520similarity%2520searching%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Comput.%2520Sci.%26date%3D1998%26volume%3D38%26spage%3D983%26epage%3D996" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group">Mohammadi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Froum, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hamby, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schroeder, M. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Panek, R. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lu, G. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eliseenkova, A. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Green, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schlessinger, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hubbard, S. R.</span><span> </span><span class="NLM_article-title">Crystal structure of an angiogenesis inhibitor bound to the FGF receptor tyrosine kinase domain</span> <span class="citation_source-journal">EMBO J.</span><span class="NLM_year" style="font-weight: bold;">1998</span> <span class="NLM_volume">17</span> <span class="NLM_fpage">6896</span> <span class="NLM_lpage">5904</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=1998&pages=6896-5904&author=M.+Mohammadiauthor=S.+Froumauthor=J.+M.+Hambyauthor=M.+C.+Schroederauthor=R.+L.+Panekauthor=G.+H.+Luauthor=A.+V.+Eliseenkovaauthor=D.+Greenauthor=J.+Schlessingerauthor=S.+R.+Hubbard&title=Crystal+structure+of+an+angiogenesis+inhibitor+bound+to+the+FGF+receptor+tyrosine+kinase+domain"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMohammadi%26aufirst%3DM.%26aulast%3DFroum%26aufirst%3DS.%26aulast%3DHamby%26aufirst%3DJ.%2BM.%26aulast%3DSchroeder%26aufirst%3DM.%2BC.%26aulast%3DPanek%26aufirst%3DR.%2BL.%26aulast%3DLu%26aufirst%3DG.%2BH.%26aulast%3DEliseenkova%26aufirst%3DA.%2BV.%26aulast%3DGreen%26aufirst%3DD.%26aulast%3DSchlessinger%26aufirst%3DJ.%26aulast%3DHubbard%26aufirst%3DS.%2BR.%26atitle%3DCrystal%2520structure%2520of%2520an%2520angiogenesis%2520inhibitor%2520bound%2520to%2520the%2520FGF%2520receptor%2520tyrosine%2520kinase%2520domain%26jtitle%3DEMBO%2520J.%26date%3D1998%26volume%3D17%26spage%3D6896%26epage%3D5904" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group">Pautsch, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zoephel, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ahorn, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Spevak, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hauptmann, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nar, H.</span><span> </span><span class="NLM_article-title">Crystal structure of bisphosphorylated IGF-1 receptor kinase: insight into domain movements upon kinase activation</span> <span class="citation_source-journal">Structure</span><span class="NLM_year" style="font-weight: bold;">2001</span> <span class="NLM_volume">9</span> <span class="NLM_fpage">955</span> <span class="NLM_lpage">965</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2001&pages=955-965&author=A.+Pautschauthor=A.+Zoephelauthor=H.+Ahornauthor=W.+Spevakauthor=R.+Hauptmannauthor=H.+Nar&title=Crystal+structure+of+bisphosphorylated+IGF-1+receptor+kinase%3A+insight+into+domain+movements+upon+kinase+activation"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DPautsch%26aufirst%3DA.%26aulast%3DZoephel%26aufirst%3DA.%26aulast%3DAhorn%26aufirst%3DH.%26aulast%3DSpevak%26aufirst%3DW.%26aulast%3DHauptmann%26aufirst%3DR.%26aulast%3DNar%26aufirst%3DH.%26atitle%3DCrystal%2520structure%2520of%2520bisphosphorylated%2520IGF-1%2520receptor%2520kinase%253A%2520insight%2520into%2520domain%2520movements%2520upon%2520kinase%2520activation%26jtitle%3DStructure%26date%3D2001%26volume%3D9%26spage%3D955%26epage%3D965" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group">Brady, G. P.,  Jr.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stouten, P. F. W.</span><span> </span><span class="NLM_article-title">Fast prediction and visualization of protein binding pockets with PASS</span> <span class="citation_source-journal">J. Comput.-Aided. Mol. Des.</span><span class="NLM_year" style="font-weight: bold;">2000</span> <span class="NLM_volume">14</span> <span class="NLM_fpage">383</span> <span class="NLM_lpage">401</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Fjm800791f&amp;key=10.1023%2FA%3A1008124202956" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Fjm800791f&amp;key=10815774" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Fjm800791f&amp;key=1%3ACAS%3A528%3ADC%252BD3cXjt1GrtLs%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2000&pages=383-401&author=G.+P.+Bradyauthor=P.+F.+W.+Stouten&title=Fast+prediction+and+visualization+of+protein+binding+pockets+with+PASS"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Fast prediction and visualization of protein binding pockets with PASS</span></div><div class="casAuthors">Brady, G. Patrick, Jr.; Stouten, Pieter F. W.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer-Aided Molecular Design</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">383-401</span>CODEN:
                <span class="NLM_cas:coden">JCADEQ</span>;
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    
            (<span class="NLM_cas:orgname">Kluwer Academic Publishers</span>)
        </div><div class="casAbstract">PASS (Putative Active Sites with Spheres) is a simple computational tool that uses geometry to characterize regions of buried vol. in proteins and to identify positions likely to represent binding sites based upon the size, shape, and burial extent of these vols.  Its utility as a predictive tool for binding site identification is tested by predicting known binding sites of proteins in the PDB using both complexed macromols. and their corresponding apo-protein structures.  The results indicate that PASS can serve as a front-end to fast docking.  The main utility of PASS lies in the fact that it can analyze a moderate-size protein (â¼30 kDa) in under 20 s, which makes it suitable for interactive mol. modeling, protein database anal., and aggressive virtual screening efforts.  As a modeling tool, PASS (i) rapidly identifies favorable regions of the protein surface, (ii) simplifies visualization of residues modulating binding in these regions, and (iii) provides a means of directly visualizing buried vol., which is often inferred indirectly from curvature in a surface representation.  PASS produces output in the form of std. PDB files, which are suitable for any modeling package, and provides script files to simplify visualization in Cerius2, InsightII, MOE, Quanta, RasMol, and Sybyl.  PASS is freely available to all.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq6J6I5HqOUY7Vg90H21EOLACvtfcHk0lgjWoFyMIT9gQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXjt1GrtLs%253D&md5=8a444f703004aa07f5258e919068eb03</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1023%2FA%3A1008124202956&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1023%252FA%253A1008124202956%26sid%3Dliteratum%253Aachs%26aulast%3DBrady%26aufirst%3DG.%2BP.%26aulast%3DStouten%26aufirst%3DP.%2BF.%2BW.%26atitle%3DFast%2520prediction%2520and%2520visualization%2520of%2520protein%2520binding%2520pockets%2520with%2520PASS%26jtitle%3DJ.%2520Comput.-Aided.%2520Mol.%2520Des.%26date%3D2000%26volume%3D14%26spage%3D383%26epage%3D401" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group">Liu, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Neiss, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Henne-Bruns, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Korc, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bachem, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kornmann, M.</span><span> </span><span class="NLM_article-title">Identification of a fibroblast growth factor receptor 1 splice variant that inhibits pancreatic cancer cell growth</span> <span class="citation_source-journal">Cancer Res.</span><span class="NLM_year" style="font-weight: bold;">2007</span> <span class="NLM_volume">67</span> <span class="NLM_fpage">2712</span> <span class="NLM_lpage">2719</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2007&pages=2712-2719&author=Z.+Liuauthor=N.+Neissauthor=S.+Zhouauthor=D.+Henne-Brunsauthor=M.+Korcauthor=M.+Bachemauthor=M.+Kornmann&title=Identification+of+a+fibroblast+growth+factor+receptor+1+splice+variant+that+inhibits+pancreatic+cancer+cell+growth"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DZ.%26aulast%3DNeiss%26aufirst%3DN.%26aulast%3DZhou%26aufirst%3DS.%26aulast%3DHenne-Bruns%26aufirst%3DD.%26aulast%3DKorc%26aufirst%3DM.%26aulast%3DBachem%26aufirst%3DM.%26aulast%3DKornmann%26aufirst%3DM.%26atitle%3DIdentification%2520of%2520a%2520fibroblast%2520growth%2520factor%2520receptor%25201%2520splice%2520variant%2520that%2520inhibits%2520pancreatic%2520cancer%2520cell%2520growth%26jtitle%3DCancer%2520Res.%26date%3D2007%26volume%3D67%26spage%3D2712%26epage%3D2719" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group">Kondo, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zheng, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kurebayashi, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Asa, S. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ezzat, S.</span><span> </span><span class="NLM_article-title">Epigenetically controlled fibroblast growth factor receptor 2 signaling imposes on the RAS/BRAF/mitogen-activated protein kinase pathway to modulate thyroid cancer progression</span> <span class="citation_source-journal">Cancer Res.</span><span class="NLM_year" style="font-weight: bold;">2007</span> <span class="NLM_volume">67</span> <span class="NLM_fpage">5461</span> <span class="NLM_lpage">5470</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2007&pages=5461-5470&author=T.+Kondoauthor=L.+Zhengauthor=W.+Liuauthor=J.+Kurebayashiauthor=S.+L.+Asaauthor=S.+Ezzat&title=Epigenetically+controlled+fibroblast+growth+factor+receptor+2+signaling+imposes+on+the+RAS%2FBRAF%2Fmitogen-activated+protein+kinase+pathway+to+modulate+thyroid+cancer+progression"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DKondo%26aufirst%3DT.%26aulast%3DZheng%26aufirst%3DL.%26aulast%3DLiu%26aufirst%3DW.%26aulast%3DKurebayashi%26aufirst%3DJ.%26aulast%3DAsa%26aufirst%3DS.%2BL.%26aulast%3DEzzat%26aufirst%3DS.%26atitle%3DEpigenetically%2520controlled%2520fibroblast%2520growth%2520factor%2520receptor%25202%2520signaling%2520imposes%2520on%2520the%2520RAS%252FBRAF%252Fmitogen-activated%2520protein%2520kinase%2520pathway%2520to%2520modulate%2520thyroid%2520cancer%2520progression%26jtitle%3DCancer%2520Res.%26date%3D2007%26volume%3D67%26spage%3D5461%26epage%3D5470" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group">Elbauomy, E. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Green, A. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lambros, M. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Turner, N. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grainge, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Powe, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ellis, I. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reis-Filho, J. S.</span><span> </span><span class="NLM_article-title">FGFR1 amplification in breast carcinomas: a chromogenic in situ hybridisation analysis</span> <span class="citation_source-journal">Breast Cancer Res.</span><span class="NLM_year" style="font-weight: bold;">2007</span> <span class="NLM_volume">9</span> <span class="NLM_fpage">R23</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2007&pages=R23&author=E.+S.+Elbauomyauthor=A.+R.+Greenauthor=M.+B.+Lambrosauthor=N.+C.+Turnerauthor=M.+J.+Graingeauthor=D.+Poweauthor=I.+O.+Ellisauthor=J.+S.+Reis-Filho&title=FGFR1+amplification+in+breast+carcinomas%3A+a+chromogenic+in+situ+hybridisation+analysis"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DElbauomy%26aufirst%3DE.%2BS.%26aulast%3DGreen%26aufirst%3DA.%2BR.%26aulast%3DLambros%26aufirst%3DM.%2BB.%26aulast%3DTurner%26aufirst%3DN.%2BC.%26aulast%3DGrainge%26aufirst%3DM.%2BJ.%26aulast%3DPowe%26aufirst%3DD.%26aulast%3DEllis%26aufirst%3DI.%2BO.%26aulast%3DReis-Filho%26aufirst%3DJ.%2BS.%26atitle%3DFGFR1%2520amplification%2520in%2520breast%2520carcinomas%253A%2520a%2520chromogenic%2520in%2520situ%2520hybridisation%2520analysis%26jtitle%3DBreast%2520Cancer%2520Res.%26date%3D2007%26volume%3D9%26spage%3DR23" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group">Sahadevan, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Darby, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leung, H. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mathers, M. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Robson, C. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gnanapragasam, V. J.</span><span> </span><span class="NLM_article-title">Selective over-expression of fibroblast growth factor receptors 1 and 4 in clinical prostate cancer</span> <span class="citation_source-journal">J. Pathol.</span><span class="NLM_year" style="font-weight: bold;">2007</span> <span class="NLM_volume">213</span> <span class="NLM_fpage">82</span> <span class="NLM_lpage">90</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Fjm800791f&amp;key=10.1002%2Fpath.2205" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Fjm800791f&amp;key=17607666" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Fjm800791f&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtFSksLvF" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=213&publication_year=2007&pages=82-90&author=K.+Sahadevanauthor=S.+Darbyauthor=H.+Y.+Leungauthor=M.+E.+Mathersauthor=C.+N.+Robsonauthor=V.+J.+Gnanapragasam&title=Selective+over-expression+of+fibroblast+growth+factor+receptors+1+and+4+in+clinical+prostate+cancer"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Selective over-expression of fibroblast growth factor receptors 1 and 4 in clinical prostate cancer</span></div><div class="casAuthors">Sahadevan, K.; Darby, S.; Leung, H. Y.; Mathers, M. E.; Robson, C. N.; Gnanapragasam, V. J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Pathology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">213</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">82-90</span>CODEN:
                <span class="NLM_cas:coden">JPTLAS</span>;
        ISSN:<span class="NLM_cas:issn">0022-3417</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons Ltd.</span>)
        </div><div class="casAbstract">Fibroblast growth factor receptors (FGFRs) mediate the tumorigenic effects of FGFs in prostate cancer.  These receptors are therefore potential therapeutic targets in the development of inhibitors to this pathway.  To identify the most relevant targets, we simultaneously investigated FGFR1-4 expression using a prostate cancer tissue microarray (TMA) and in laser capture microdissected (LCM) prostate epithelial cells.  In malignant prostates (n = 138) we obsd. significant FGFR1 and FGFR4 protein over-expression in comparison with benign prostates (n = 58; p < 0.0001).  FGFR1 was expressed at high levels in the majority of tumors (69% of grade 3 or less, 74% of grade 4 and 70% of grade 5), while FGFR4 was strongly expressed in 83% of grade 5 cancers but in only 25% of grade 1-3 cancers (p < 0.0001).  At the transcript level we obsd. a similar pattern, with FGFR1 and FGFR4 mRNA over-expressed in malignant epithelial cells compared to benign cells (p < 0.0005 and p < 0.05, resp.).  While total FGFR2 was increased in some cancers, there was no assocn. between expression and tumor grade or stage.  Transcript anal., however, revealed a switch in the predominant isoform expressed from FGFR2IIIb to FGFR2IIIc among malignant epithelial cells.  In contrast, protein and transcript expression of FGFR3 was very similar between benign and cancer biopsies.  The functional effect of targeting FGFR4 in prostate cancer cells has not previously been investigated.  In in vitro expts., suppression of FGFR4 by RNA interference effectively blocked prostate cancer cell proliferation (p < 0.0001) and invasion (p < 0.001) in response to exogenous stimulation.  This effect was evident regardless of whether the cells expressed the FGFR4 Arg388 or Gly388 allele.  In parallel expts., FGFR3 suppression had no discernible effect on cancer cell behavior.  These results suggest evidence of selective over-expression of FGFR1 and FGFR4 in clin. prostate cancer and support the notion of targeted inhibition of these receptors to disrupt FGF signalling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrfun6Hyw60Z7Vg90H21EOLACvtfcHk0lhmih7G-prUPA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtFSksLvF&md5=3729c3a83c8c3fdf634220c5fe1a5992</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1002%2Fpath.2205&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fpath.2205%26sid%3Dliteratum%253Aachs%26aulast%3DSahadevan%26aufirst%3DK.%26aulast%3DDarby%26aufirst%3DS.%26aulast%3DLeung%26aufirst%3DH.%2BY.%26aulast%3DMathers%26aufirst%3DM.%2BE.%26aulast%3DRobson%26aufirst%3DC.%2BN.%26aulast%3DGnanapragasam%26aufirst%3DV.%2BJ.%26atitle%3DSelective%2520over-expression%2520of%2520fibroblast%2520growth%2520factor%2520receptors%25201%2520and%25204%2520in%2520clinical%2520prostate%2520cancer%26jtitle%3DJ.%2520Pathol.%26date%3D2007%26volume%3D213%26spage%3D82%26epage%3D90" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group">Ayers, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fargnoli, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lewin, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Platero, J. S.</span><span> </span><span class="NLM_article-title">Discovery and validation of biomarkers that respond to treatment with brivanib alaninate, a small-molecule VEGFR-2/FGFR-1 antagonist</span> <span class="citation_source-journal">Cancer Res.</span><span class="NLM_year" style="font-weight: bold;">2007</span> <span class="NLM_volume">67</span> <span class="NLM_fpage">6899</span> <span class="NLM_lpage">6906</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2007&pages=6899-6906&author=M.+Ayersauthor=J.+Fargnoliauthor=A.+Lewinauthor=Q.+Wuauthor=J.+S.+Platero&title=Discovery+and+validation+of+biomarkers+that+respond+to+treatment+with+brivanib+alaninate%2C+a+small-molecule+VEGFR-2%2FFGFR-1+antagonist"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DAyers%26aufirst%3DM.%26aulast%3DFargnoli%26aufirst%3DJ.%26aulast%3DLewin%26aufirst%3DA.%26aulast%3DWu%26aufirst%3DQ.%26aulast%3DPlatero%26aufirst%3DJ.%2BS.%26atitle%3DDiscovery%2520and%2520validation%2520of%2520biomarkers%2520that%2520respond%2520to%2520treatment%2520with%2520brivanib%2520alaninate%252C%2520a%2520small-molecule%2520VEGFR-2%252FFGFR-1%2520antagonist%26jtitle%3DCancer%2520Res.%26date%3D2007%26volume%3D67%26spage%3D6899%26epage%3D6906" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group">Bean, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brennan, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shih, J. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Riely, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Viale, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chitale, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Motoi, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Szoke, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Broderick, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Balak, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chang, W. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yu, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gazdar, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pass, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rusch, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gerald, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, S. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, P. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miller, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ladanyi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pao, W.</span><span> </span><span class="NLM_article-title">MET amplification occurs with or without T790M mutations in EGFR mutant lung tumors with acquired resistance to gefitinib or erlotinib</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U.S.A.</span><span class="NLM_year" style="font-weight: bold;">2007</span> <span class="NLM_volume">104</span> <span class="NLM_fpage">20932</span> <span class="NLM_lpage">20937</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=104&publication_year=2007&pages=20932-20937&author=J.+Beanauthor=C.+Brennanauthor=J.+Y.+Shihauthor=G.+Rielyauthor=A.+Vialeauthor=L.+Wangauthor=D.+Chitaleauthor=N.+Motoiauthor=J.+Szokeauthor=S.+Broderickauthor=M.+Balakauthor=W.+C.+Changauthor=C.+J.+Yuauthor=A.+Gazdarauthor=H.+Passauthor=V.+Ruschauthor=W.+Geraldauthor=S.+F.+Huangauthor=P.+C.+Yangauthor=V.+Millerauthor=M.+Ladanyiauthor=C.+H.+Yangauthor=W.+Pao&title=MET+amplification+occurs+with+or+without+T790M+mutations+in+EGFR+mutant+lung+tumors+with+acquired+resistance+to+gefitinib+or+erlotinib"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBean%26aufirst%3DJ.%26aulast%3DBrennan%26aufirst%3DC.%26aulast%3DShih%26aufirst%3DJ.%2BY.%26aulast%3DRiely%26aufirst%3DG.%26aulast%3DViale%26aufirst%3DA.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DChitale%26aufirst%3DD.%26aulast%3DMotoi%26aufirst%3DN.%26aulast%3DSzoke%26aufirst%3DJ.%26aulast%3DBroderick%26aufirst%3DS.%26aulast%3DBalak%26aufirst%3DM.%26aulast%3DChang%26aufirst%3DW.%2BC.%26aulast%3DYu%26aufirst%3DC.%2BJ.%26aulast%3DGazdar%26aufirst%3DA.%26aulast%3DPass%26aufirst%3DH.%26aulast%3DRusch%26aufirst%3DV.%26aulast%3DGerald%26aufirst%3DW.%26aulast%3DHuang%26aufirst%3DS.%2BF.%26aulast%3DYang%26aufirst%3DP.%2BC.%26aulast%3DMiller%26aufirst%3DV.%26aulast%3DLadanyi%26aufirst%3DM.%26aulast%3DYang%26aufirst%3DC.%2BH.%26aulast%3DPao%26aufirst%3DW.%26atitle%3DMET%2520amplification%2520occurs%2520with%2520or%2520without%2520T790M%2520mutations%2520in%2520EGFR%2520mutant%2520lung%2520tumors%2520with%2520acquired%2520resistance%2520to%2520gefitinib%2520or%2520erlotinib%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2007%26volume%3D104%26spage%3D20932%26epage%3D20937" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group">Ma, P. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schaefer, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Christensen, J. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Salgia, R.</span><span> </span><span class="NLM_article-title">A selective small molecule c-MET Inhibitor, PHA665752, cooperates with rapamycin</span> <span class="citation_source-journal">Clin. Cancer Res.</span><span class="NLM_year" style="font-weight: bold;">2005</span> <span class="NLM_volume">11</span> <span class="NLM_fpage">2312</span> <span class="NLM_lpage">2319</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Fjm800791f&amp;key=10.1158%2F1078-0432.CCR-04-1708" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Fjm800791f&amp;key=15788682" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Fjm800791f&amp;key=1%3ACAS%3A528%3ADC%252BD2MXisFaju7c%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2005&pages=2312-2319&author=P.+C.+Maauthor=E.+Schaeferauthor=J.+G.+Christensenauthor=R.+Salgia&title=A+selective+small+molecule+c-MET+Inhibitor%2C+PHA665752%2C+cooperates+with+rapamycin"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">A selective small molecule c-MET inhibitor, PHA665752, cooperates with rapamycin</span></div><div class="casAuthors">Ma, Patrick C.; Schaefer, Erik; Christensen, James G.; Salgia, Ravi</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2312-2319</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">c-MET is believed to be an attractive receptor target for mol. therapeutic inhibition.  TPR-MET, a constitutively active oncogenic variant of MET, serves as excellent model for testing c-MET inhibitors.  Here, we characterized a small mol. c-MET inhibitor, PHA665752, and tested its cooperation with the mammalian target of rapamycin inhibitor as potential targeted therapy.  The effect of PHA665752 treatment was detd. on cell growth, motility and migration, apoptosis, and cell-cycle arrest of TPR-MET-transformed cells.  Moreover, the effect of PHA665752 on the phosphorylation on MET, as well as its downstream effectors, p-AKT and p-S6K, was also detd.  Finally, growth of TPR-MET-transformed cells was tested in the presence of PHA665752 and rapamycin.  H441 non-small cell lung cancer (NSCLC) cells (with activated c-Met) were also tested against both PHA665752 and rapamycin.  PHA665752 specifically inhibited cell growth in BaF3.  TPR-MET cells (IC50 < 0.06 Î¼mol/L), induced apoptosis and cell cycle arrest.  Constitutive cell motility and migration of the BaF3.  TPR-MET cells was also inhibited.  PHA665752 inhibited specific phosphorylation of TPR-MET as well as phosphorylation of downstream targets of the mammalian target of rapamycin pathway.  When combined with PHA665752, rapamycin showed cooperative inhibition to reduce growth of BaF3.  TPR-MET- and c-MET-expressing H441 NSCLC cells.  PHA665752 is a potent small mol.-selective c-MET inhibitor and is highly active against TPR-MET-transformed cells both biol. and biochem.  PHA665752 is also active against H441 NSCLC cells.  The c-MET inhibitor can cooperate with rapamycin in therapeutic inhibition of NSCLC, and in vivo studies of this combination against c-MET expressing cancers would be merited.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrQ6fVYSbU8ibVg90H21EOLACvtfcHk0lhmih7G-prUPA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXisFaju7c%253D&md5=9e69f0253f675f4a1aeb96c115e2e420</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1158%2F1078-0432.CCR-04-1708&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1078-0432.CCR-04-1708%26sid%3Dliteratum%253Aachs%26aulast%3DMa%26aufirst%3DP.%2BC.%26aulast%3DSchaefer%26aufirst%3DE.%26aulast%3DChristensen%26aufirst%3DJ.%2BG.%26aulast%3DSalgia%26aufirst%3DR.%26atitle%3DA%2520selective%2520small%2520molecule%2520c-MET%2520Inhibitor%252C%2520PHA665752%252C%2520cooperates%2520with%2520rapamycin%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D2005%26volume%3D11%26spage%3D2312%26epage%3D2319" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group">Ma, P. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jagadeeswaran, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jagadeesh, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tretiakova, M. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nallasura, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fox, E. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hansen, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schaefer, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Naoki, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lader, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richards, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sugarbaker, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Husain, A. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Christensen, J. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Salgia, R.</span><span> </span><span class="NLM_article-title">Functional expression and mutations of c-Met and its therapeutic inhibition with SU11274 and small interfering RNA in non-small cell lung cancer</span> <span class="citation_source-journal">Cancer Res.</span><span class="NLM_year" style="font-weight: bold;">2005</span> <span class="NLM_volume">65</span> <span class="NLM_fpage">1479</span> <span class="NLM_lpage">1488</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=65&publication_year=2005&pages=1479-1488&author=P.+C.+Maauthor=R.+Jagadeeswaranauthor=S.+Jagadeeshauthor=M.+S.+Tretiakovaauthor=V.+Nallasuraauthor=E.+A.+Foxauthor=M.+Hansenauthor=E.+Schaeferauthor=K.+Naokiauthor=A.+Laderauthor=W.+Richardsauthor=D.+Sugarbakerauthor=A.+N.+Husainauthor=J.+G.+Christensenauthor=R.+Salgia&title=Functional+expression+and+mutations+of+c-Met+and+its+therapeutic+inhibition+with+SU11274+and+small+interfering+RNA+in+non-small+cell+lung+cancer"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMa%26aufirst%3DP.%2BC.%26aulast%3DJagadeeswaran%26aufirst%3DR.%26aulast%3DJagadeesh%26aufirst%3DS.%26aulast%3DTretiakova%26aufirst%3DM.%2BS.%26aulast%3DNallasura%26aufirst%3DV.%26aulast%3DFox%26aufirst%3DE.%2BA.%26aulast%3DHansen%26aufirst%3DM.%26aulast%3DSchaefer%26aufirst%3DE.%26aulast%3DNaoki%26aufirst%3DK.%26aulast%3DLader%26aufirst%3DA.%26aulast%3DRichards%26aufirst%3DW.%26aulast%3DSugarbaker%26aufirst%3DD.%26aulast%3DHusain%26aufirst%3DA.%2BN.%26aulast%3DChristensen%26aufirst%3DJ.%2BG.%26aulast%3DSalgia%26aufirst%3DR.%26atitle%3DFunctional%2520expression%2520and%2520mutations%2520of%2520c-Met%2520and%2520its%2520therapeutic%2520inhibition%2520with%2520SU11274%2520and%2520small%2520interfering%2520RNA%2520in%2520non-small%2520cell%2520lung%2520cancer%26jtitle%3DCancer%2520Res.%26date%3D2005%26volume%3D65%26spage%3D1479%26epage%3D1488" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group">Shoichet, B. K.</span><span> </span><span class="NLM_article-title">Virtual screening of chemical libraries</span> <span class="citation_source-journal">Nature</span><span class="NLM_year" style="font-weight: bold;">2004</span> <span class="NLM_volume">432</span> <span class="NLM_fpage">862</span> <span class="NLM_lpage">865</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Fjm800791f&amp;key=10.1038%2Fnature03197" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Fjm800791f&amp;key=15602552" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Fjm800791f&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVOht7jK" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=432&publication_year=2004&pages=862-865&author=B.+K.+Shoichet&title=Virtual+screening+of+chemical+libraries"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Virtual screening of chemical libraries</span></div><div class="casAuthors">Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">432</span>
        (<span class="NLM_cas:issue">7019</span>),
    <span class="NLM_cas:pages">862-865</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Virtual screening uses computer-based methods to discover new ligands on the basis of biol. structures.  Although widely heralded in the 1970s and 1980s, the technique has since struggled to meet its initial promise, and drug discovery remains dominated by empirical screening.  Recent successes in predicting new ligands and their receptor-bound structures, and better rates of ligand discovery compared to empirical screening, have reignited interest in virtual screening, which is now widely used in drug discovery, albeit on a more limited scale than empirical screening.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrSY9IbbFldkrVg90H21EOLACvtfcHk0lg4TO29PSLBlg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVOht7jK&md5=e351e226e8985d7c304a05fb67dfdc60</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1038%2Fnature03197&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature03197%26sid%3Dliteratum%253Aachs%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DVirtual%2520screening%2520of%2520chemical%2520libraries%26jtitle%3DNature%26date%3D2004%26volume%3D432%26spage%3D862%26epage%3D865" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group">Klebe, G.</span><span> </span><span class="NLM_article-title">Virtual ligand screening: strategies, perspectives and limitations</span> <span class="citation_source-journal">Drug Discovery Today</span><span class="NLM_year" style="font-weight: bold;">2006</span> <span class="NLM_volume">11</span> <span class="NLM_fpage">580</span> <span class="NLM_lpage">594</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Fjm800791f&amp;key=10.1016%2Fj.drudis.2006.05.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Fjm800791f&amp;key=16793526" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Fjm800791f&amp;key=1%3ACAS%3A528%3ADC%252BD28XlvFGqtLo%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2006&pages=580-594&author=G.+Klebe&title=Virtual+ligand+screening%3A+strategies%2C+perspectives+and+limitations"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Virtual ligand screening: strategies, perspectives and limitations</span></div><div class="casAuthors">Klebe, Gerhard</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">13 & 14</span>),
    <span class="NLM_cas:pages">580-594</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  In contrast to high-throughput screening, in virtual ligand screening (VS), compds. are selected using computer programs to predict their binding to a target receptor.  A key prerequisite is knowledge about the spatial and energetic criteria responsible for protein-ligand binding.  The concepts and prerequisites to perform VS are summarized here, and explanations are sought for the enduring limitations of the technol.  Target selection, anal. and prepn. are discussed, as well as considerations about the compilation of candidate ligand libraries.  The tools and strategies of a VS campaign, and the accuracy of scoring and ranking of the results, are also considered.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrN3dWlP-WDY7Vg90H21EOLACvtfcHk0lg4TO29PSLBlg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XlvFGqtLo%253D&md5=5adfd125d48082238a6c5ad0e8343c59</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2006.05.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2006.05.012%26sid%3Dliteratum%253Aachs%26aulast%3DKlebe%26aufirst%3DG.%26atitle%3DVirtual%2520ligand%2520screening%253A%2520strategies%252C%2520perspectives%2520and%2520limitations%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2006%26volume%3D11%26spage%3D580%26epage%3D594" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group">Ihlenfeldt, W. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Takahashi, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abe, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sasaki, S.</span><span> </span><span class="NLM_article-title">Computation and management of chemical properties in CACTVS: an extensible networked approach toward modularity and compatibility</span> <span class="citation_source-journal">J. Chem. Inf. Comput. Sci.</span><span class="NLM_year" style="font-weight: bold;">1994</span> <span class="NLM_volume">34</span> <span class="NLM_fpage">109</span> <span class="NLM_lpage">116</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci00017a013" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Fjm800791f&amp;key=1%3ACAS%3A528%3ADyaK2cXns1GrtQ%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=1994&pages=109-116&author=W.+D.+Ihlenfeldtauthor=Y.+Takahashiauthor=H.+Abeauthor=S.+Sasaki&title=Computation+and+management+of+chemical+properties+in+CACTVS%3A+an+extensible+networked+approach+toward+modularity+and+compatibility"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Computation and management of chemical properties in CACTVS: An extensible networked approach toward modularity and compatibility</span></div><div class="casAuthors">Ihlenfeldt, Wolf Dietrich; Takahashi, Yoshimasa; Abe, Hidetsugu; Sasaki, S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Computer Sciences</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">34</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">109-16</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">0095-2338</span>.
    </div><div class="casAbstract">The data model of the CACTVS program suite (Chem. Algorithms Construction, Threading, and Verification System) is presented.  CACTVS is an open environment which readily integrates new computational modules and data format descriptions in the domain of computational chem.  These chem. information (property) computation modules are provided with requested input data, typically transparently computed by other modules.  Computational results are managed appropriately with regard to file and data base I/O, transport to display servers, input for high-level routines, and report generation.  Definitions of property characteristics which include optionally precompiled objects and source plus addnl. documentation may be stored in networked data bases for global access.  This facilitates module reuse and the exchange of modules between interested parties.  The elementary data format handling routines and the property computation routines need not to be part of the core program.  They are retrieved from and loaded dynamically at run time from local files or data bases which can be reached on the network.  This look-up process can be made completely transparent, resulting in a comparably small core program which grows only on demand to incorporate procedures for the evaluation of requested information.  A model for asynchronous distributed client-server computations is supported as an alternative to dynamic linking for expensive calcns.  CACTVS is intended to make algorithm development in chem. more productive and to help in collecting the synergetic benefits from the concerted application of compatible algorithm sets which derive chem. information in all its variety.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpSgwAre-5hnbVg90H21EOLACvtfcHk0lg4TO29PSLBlg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2cXns1GrtQ%253D%253D&md5=32a50d1b7997840849d7697fa0a603e8</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1021%2Fci00017a013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci00017a013%26sid%3Dliteratum%253Aachs%26aulast%3DIhlenfeldt%26aufirst%3DW.%2BD.%26aulast%3DTakahashi%26aufirst%3DY.%26aulast%3DAbe%26aufirst%3DH.%26aulast%3DSasaki%26aufirst%3DS.%26atitle%3DComputation%2520and%2520management%2520of%2520chemical%2520properties%2520in%2520CACTVS%253A%2520an%2520extensible%2520networked%2520approach%2520toward%2520modularity%2520and%2520compatibility%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Comput.%2520Sci.%26date%3D1994%26volume%3D34%26spage%3D109%26epage%3D116" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group">Gasteiger, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rudolph, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sadowski, J.</span><span> </span><span class="NLM_article-title">Automatic generation of 3D-atomic coordinates for organic molecules</span> <span class="citation_source-journal">Tetrahedron Comput. Methodol.</span><span class="NLM_year" style="font-weight: bold;">1990</span> <span class="NLM_volume">3</span> <span class="NLM_fpage">547</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Fjm800791f&amp;key=10.1016%2F0898-5529%2890%2990156-3" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=1990&pages=547&author=J.+Gasteigerauthor=C.+Rudolphauthor=J.+Sadowski&title=Automatic+generation+of+3D-atomic+coordinates+for+organic+molecules"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1016%2F0898-5529%2890%2990156-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0898-5529%252890%252990156-3%26sid%3Dliteratum%253Aachs%26aulast%3DGasteiger%26aufirst%3DJ.%26aulast%3DRudolph%26aufirst%3DC.%26aulast%3DSadowski%26aufirst%3DJ.%26atitle%3DAutomatic%2520generation%2520of%25203D-atomic%2520coordinates%2520for%2520organic%2520molecules%26jtitle%3DTetrahedron%2520Comput.%2520Methodol.%26date%3D1990%26volume%3D3%26spage%3D547" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span> <span class="citation_source-book">Pipeline Pilot</span>, version 4.1.1; SciTegic, Inc.: San Diego, CA, <span class="NLM_year">2004</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Pipeline+Pilot%2C+version+4.1.1%3B+SciTegic%2C+Inc.%3A+San+Diego%2C+CA%2C+2004."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26btitle%3DPipeline%2520Pilot%26date%3D2004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group">Morotti, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mila, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Accornero, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taglibue, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ponzetto, C.</span><span> </span><span class="NLM_article-title">K252a inhibits the oncogenic properties of Met, the HGF receptor</span> <span class="citation_source-journal">Oncogene</span><span class="NLM_year" style="font-weight: bold;">2002</span> <span class="NLM_volume">21</span> <span class="NLM_fpage">4885</span> <span class="NLM_lpage">4893</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2002&pages=4885-4893&author=A.+Morottiauthor=S.+Milaauthor=P.+Accorneroauthor=E.+Taglibueauthor=C.+Ponzetto&title=K252a+inhibits+the+oncogenic+properties+of+Met%2C+the+HGF+receptor"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMorotti%26aufirst%3DA.%26aulast%3DMila%26aufirst%3DS.%26aulast%3DAccornero%26aufirst%3DP.%26aulast%3DTaglibue%26aufirst%3DE.%26aulast%3DPonzetto%26aufirst%3DC.%26atitle%3DK252a%2520inhibits%2520the%2520oncogenic%2520properties%2520of%2520Met%252C%2520the%2520HGF%2520receptor%26jtitle%3DOncogene%26date%3D2002%26volume%3D21%26spage%3D4885%26epage%3D4893" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span> <span class="citation_source-book">MOE: Molecular Operating Environment</span>, version 2004.03; Chemical Computing Group, Inc.: Montreal, Canada, <span class="NLM_year">2004</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=MOE%3A+Molecular+Operating+Environment%2C+version+2004.03%3B+Chemical+Computing+Group%2C+Inc.%3A+Montreal%2C+Canada%2C+2004."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26btitle%3DMOE%253A%2520Molecular%2520Operating%2520Environment%26date%3D2004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group">Delano, W. L.</span> <span class="citation_source-book">The PyMOL Molecular Graphics System</span>, version 0.99; DeLano Scientific: Palo Alto, CA, <span class="NLM_year">2002</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2002&author=W.+L.+Delano&title=The+PyMOL+Molecular+Graphics+System"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DDelano%26aufirst%3DW.%2BL.%26btitle%3DThe%2520PyMOL%2520Molecular%2520Graphics%2520System%26date%3D2002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1R0P" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1R0P','PDB','1R0P'); return false;">PDB: 1R0P</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2FGI" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2FGI','PDB','2FGI'); return false;">PDB: 2FGI</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1JQH" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1JQH','PDB','1JQH'); return false;">PDB: 1JQH</a></li><li><a href="http://www.ncbi.nlm.nih.gov/gquery?Term=J02958" onclick="trackOutboundLink('http://www.ncbi.nlm.nih.gov/gquery?Term=J02958','Genbank','J02958'); return false;">Genbank: J02958</a></li></ul></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Fjm800791f&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Barticle%3Aarticle%3A10.1021%2Fjm800791f%3Bwgroup%3Astring%3AACHS+website+Group%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2009.52.issue-4%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/jm800791f" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                2MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright Â© 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">Youâve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"67991575ce623c0a","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
