
<html lang="en"     class="pb-page"  data-request-id="7b96a6a2-f331-4bfe-8192-627328eb266c"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";article:article:10.1021/acs.jmedchem.8b00256;requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;issue:issue:10.1021/jmcmar.2018.61.issue-14;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Structural Insights on Fragment Binding Mode Conservation" /></meta><meta name="dc.Creator" content="Malgorzata&#xA;N.  Drwal" /></meta><meta name="dc.Creator" content="Guillaume  Bret" /></meta><meta name="dc.Creator" content="Carlos  Perez" /></meta><meta name="dc.Creator" content="Célien  Jacquemard" /></meta><meta name="dc.Creator" content="Jérémy  Desaphy" /></meta><meta name="dc.Creator" content="Esther  Kellenberger" /></meta><meta name="dc.Description" content="Aiming at a deep understanding of fragment binding to ligandable targets, we performed a large scale analysis of the Protein Data Bank. Binding modes of 1832 drug-like ligands and 1079 fragments to..." /></meta><meta name="Description" content="Aiming at a deep understanding of fragment binding to ligandable targets, we performed a large scale analysis of the Protein Data Bank. Binding modes of 1832 drug-like ligands and 1079 fragments to..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="June 15, 2018" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.8b00256" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="Copyright © 2018 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.8b00256" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.8b00256" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.8b00256" /></link>
        
    
    

<title>Structural Insights on Fragment Binding Mode Conservation | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.8b00256" /></meta><meta property="og:title" content="Structural Insights on Fragment Binding Mode Conservation" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-14/acs.jmedchem.8b00256/20180719/images/large/jm-2018-002562_0003.jpeg" /></meta><meta property="og:description" content="Aiming at a deep understanding of fragment binding to ligandable targets, we performed a large scale analysis of the Protein Data Bank. Binding modes of 1832 drug-like ligands and 1079 fragments to 235 proteins were compared. We observed that the binding modes of fragments and their drug-like superstructures binding to the same protein are mostly conserved, thereby providing experimental evidence for the preservation of fragment binding modes during molecular growing. Furthermore, small chemical changes in the fragment are tolerated without alteration of the fragment binding mode. The exceptions to this observation generally involve conformational variability of the molecules. Our data analysis also suggests that, provided enough fragments have been crystallized within a protein, good interaction coverage of the binding pocket is achieved. Last, we extended our study to 126 crystallization additives and discuss in which cases they provide information relevant to structure-based drug design." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.8b00256"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.8b00256">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.8b00256&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.8b00256&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.8b00256&amp;href=/doi/10.1021/acs.jmedchem.8b00256" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2018</span><span class="cit-fg-volume">, 61</span><span class="cit-fg-issue">, 14</span><span class="cit-fg-pageRange">, 5963-5973</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/61/14" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.8b00182" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.8b00322" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Structural Insights on Fragment Binding Mode Conservation</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Malgorzata N. Drwal</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Malgorzata N. Drwal</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Laboratoire d’Innovation Thérapeutique, UMR7200, Université de Strasbourg, 74 Route du Rhin, 67401 Illkirch, France</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Malgorzata%0AN.++Drwal">Malgorzata N. Drwal</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Guillaume Bret</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Guillaume Bret</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Laboratoire d’Innovation Thérapeutique, UMR7200, Université de Strasbourg, 74 Route du Rhin, 67401 Illkirch, France</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Guillaume++Bret">Guillaume Bret</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Carlos Perez</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Carlos Perez</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Eli Lilly Research Laboratories, Avenida de la Industria, 30, 28108 Alcobendas, Madrid, Spain</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Carlos++Perez">Carlos Perez</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Célien Jacquemard</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Célien Jacquemard</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Laboratoire d’Innovation Thérapeutique, UMR7200, Université de Strasbourg, 74 Route du Rhin, 67401 Illkirch, France</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=C%C3%A9lien++Jacquemard">Célien Jacquemard</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Jérémy Desaphy</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jérémy Desaphy</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Lilly Research Laboratories, Eli Lilly and Company, Lilly Corporate Center, Indianapolis, Indiana 46285, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=J%C3%A9r%C3%A9my++Desaphy">Jérémy Desaphy</a></span></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Esther Kellenberger</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Esther Kellenberger</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Laboratoire d’Innovation Thérapeutique, UMR7200, Université de Strasbourg, 74 Route du Rhin, 67401 Illkirch, France</div></div><span class="conrtib-corresp"><strong>*</strong>E-mail: <a href="/cdn-cgi/l/email-protection#eb8e808e87878e85ab9e8582989f998ac58d99"><span class="__cf_email__" data-cfemail="9bfef0fef7f7fef5dbeef5f2e8efe9fab5fde9">[email protected]</span></a>. Phone: +33 368854221.</span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Esther++Kellenberger">Esther Kellenberger</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-9320-4840" title="Orcid link">http://orcid.org/0000-0002-9320-4840</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.8b00256&amp;href=/doi/10.1021%2Facs.jmedchem.8b00256" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2018</span></span><span class="cit-volume">, 61</span><span class="cit-issue">, 14</span><span class="cit-pageRange">, 5963–5973</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">June 15, 2018</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>15 February 2018</li><li><span class="item_label"><b>Published</b> online</span>15 June 2018</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 26 July 2018</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.8b00256" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.8b00256</a></div><div class="article_header-article-copyright"><strong>Copyright © 2018 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D5963%26pageCount%3D11%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DMalgorzata%250AN.%2BDrwal%252C%2BGuillaume%2BBret%252C%2BCarlos%2BPerez%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D61%26issueNum%3D14%26contentID%3Dacs.jmedchem.8b00256%26title%3DStructural%2BInsights%2Bon%2BFragment%2BBinding%2BMode%2BConservation%26numPages%3D11%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D5973%26publicationDate%3DJuly%2B2018">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.8b00256"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">2125</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">15</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.8b00256" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Structural Insights on Fragment Binding Mode Conservation&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Malgorzata\nN.&quot;,&quot;last_name&quot;:&quot;Drwal&quot;},{&quot;first_name&quot;:&quot;Guillaume&quot;,&quot;last_name&quot;:&quot;Bret&quot;},{&quot;first_name&quot;:&quot;Carlos&quot;,&quot;last_name&quot;:&quot;Perez&quot;},{&quot;first_name&quot;:&quot;Célien&quot;,&quot;last_name&quot;:&quot;Jacquemard&quot;},{&quot;first_name&quot;:&quot;Jérémy&quot;,&quot;last_name&quot;:&quot;Desaphy&quot;},{&quot;first_name&quot;:&quot;Esther&quot;,&quot;last_name&quot;:&quot;Kellenberger&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2018&quot;,&quot;month&quot;:&quot;06&quot;,&quot;day&quot;:&quot;15&quot;,&quot;issue&quot;:&quot;14&quot;,&quot;volume&quot;:&quot;61&quot;,&quot;pages&quot;:&quot;5963-5973&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.8b00256&quot;},&quot;abstract&quot;:&quot;Aiming at a deep understanding of fragment binding to ligandable targets, we performed a large scale analysis of the Protein Data Bank. Binding modes of 1832 drug-like ligands and 1079 fragments to 235 proteins were compared. We observed that the binding modes of fragments and their drug-like superstructures binding to the same protein are mostly conserved, thereby providing experimental evidence for the preservation of fragment binding modes during molecular growing. Furthermore, small chemical changes in the fragment are tolerated without alteration of the fragment binding mode. The exceptions to this observation generally involve conformational variability of the molecules. Our data analysis also suggests that, provided enough fragments have been crystallized within a protein, good interaction coverage of the binding pocket is achieved. Last, we extended our study to 126 crystallization additives and discuss in which cases they provide information relevant to structure-based drug design.&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.8b00256&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.8b00256" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.8b00256&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.8b00256" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.8b00256&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.8b00256" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.8b00256&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.8b00256&amp;href=/doi/10.1021/acs.jmedchem.8b00256" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.8b00256" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.8b00256" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (4 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.8b00256&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.8b00256%26sid%3Dliteratum%253Aachs%26pmid%3D29906118%26genre%3Darticle%26aulast%3DDrwal%26date%3D2018%26atitle%3DStructural%2BInsights%2Bon%2BFragment%2BBinding%2BMode%2BConservation%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D61%26issue%3D14%26spage%3D5963%26epage%3D5973%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (2)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=292580" title="Binding modes">Binding modes</a>,</li><li><a href="/action/doSearch?ConceptID=292197" title="Additives">Additives</a>,</li><li><a href="/action/doSearch?ConceptID=292366" title="Ligands">Ligands</a>,</li><li><a href="/action/doSearch?ConceptID=290901" title="Protein structure">Protein structure</a>,</li><li><a href="/action/doSearch?ConceptID=291352" title="Screening assays">Screening assays</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/61/14" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-14/jmcmar.2018.61.issue-14/20180726/jmcmar.2018.61.issue-14.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-14/acs.jmedchem.8b00256/20180719/images/medium/jm-2018-002562_0003.gif" alt="" id="_i2" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-14/acs.jmedchem.8b00256/20180719/images/large/jm-2018-002562_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00256&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Aiming at a deep understanding of fragment binding to ligandable targets, we performed a large scale analysis of the Protein Data Bank. Binding modes of 1832 drug-like ligands and 1079 fragments to 235 proteins were compared. We observed that the binding modes of fragments and their drug-like superstructures binding to the same protein are mostly conserved, thereby providing experimental evidence for the preservation of fragment binding modes during molecular growing. Furthermore, small chemical changes in the fragment are tolerated without alteration of the fragment binding mode. The exceptions to this observation generally involve conformational variability of the molecules. Our data analysis also suggests that, provided enough fragments have been crystallized within a protein, good interaction coverage of the binding pocket is achieved. Last, we extended our study to 126 crystallization additives and discuss in which cases they provide information relevant to structure-based drug design.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_00858" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_00858" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Fragment-based drug design (FBDD) is a well-established method to find new drug candidates by optimizing small chemical fragments into larger molecules.<a onclick="showRef(event, 'ref1 ref2'); return false;" href="javascript:void(0);" class="ref ref1 ref2">(1,2)</a> As compared to larger ligands, fragments have in principle a higher proportion of functional groups involved in protein binding and many of them precisely fit the target subpockets. Moreover, due to their reduced size and complexity, fragments allow an efficient exploration of protein binding sites.<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> Thus, FBDD usually results in higher hit rates than high-throughput screening (HTS) with large molecules, providing good starting points for drug discovery programs.<a onclick="showRef(event, 'ref4 ref5'); return false;" href="javascript:void(0);" class="ref ref4 ref5">(4,5)</a> Examples have shown that FBDD can be successful where other drug discovery programs have failed, e.g., for difficult protein targets or protein–protein interfaces.<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a></div><div class="NLM_p">FBDD generally begins with the experimental screening of fragment libraries to determine possible hits. Once their 3D structure with the protein has been determined, they are optimized into larger molecules by growing them into molecules occupying the entire binding pocket or linking fragments binding to different subpockets. At this stage, computational chemists can support the FBDD project by predicting how to grow or link fragments to develop a high-affinity drug-like ligand. Usually, the predictions rely on the assumption that the binding mode of a fragment is unique and that the binding mode of a fragment and its drug-like counterpart will be conserved. However, several studies on ligand deconstruction have shown that this is not always the case. On the basis of eight examples, Kozakov and colleagues showed that fragments coinciding with low-energy hot spots tend to have conserved binding modes.<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> The detection of protein hot spot has been proposed to assist fragment selection and elaboration by prioritizing protein subpockets.<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a></div><div class="NLM_p last">In the current study, we have performed a large-scale analysis of fragment binding modes in crystallographic complexes obtained from the RCSB Protein Data Bank (PDB).<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> The first question we asked is how often and why fragments crystallized multiple times in the same protein cavity have variable binding modes. We have evaluated the degree of binding mode conservation between drug-like ligands and their root fragments bound to the same protein target and the influencing parameters. In particular, we have systematically compared interactions made by fragments and their drug-like superstructures bound to the same proteins. Considering all the fragments and drug-like ligands bound to the same protein binding site, we have investigated if fragments cover all the interactions made by the drug-like ligands or if, by contrast, they have specific recognition subsites. We have extended our analyses to small crystallization additives and have characterized which information is relevant for the design of drug-like ligands. Overall, our results provide guidelines for the computational support of an FBDD project where one or more fragment–protein complex structures have been solved.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i4">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_46145" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_46145" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">In the current study, we explore and compare the binding modes of fragments and drug-like ligands that have been crystallized with the same protein target and within the same ligandable binding site (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>A). For this purpose, we have processed protein complexes from the PDB as described in detail in the <a class="ref internalNav" href="#sec4" aria-label="Experimental Section">Experimental Section</a>. Special care was taken to ensure relevant description of the binding mode by removing complexes with protein pockets containing mutated or missing residues as well as removing small molecules with missing atoms. The quality of 3D-structures was assessed using the EDIA approach recently proposed by Meyder et al.<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> On average, the 3D-structure is well covered by the electron density in more than 80% of the studied complexes. Furthermore, in none of them, the structure of the ligand or protein residues poorly fits to the corresponding electron density (more statistics are given in the <a href="/doi/suppl/10.1021/acs.jmedchem.8b00256/suppl_file/jm8b00256_si_002.pdf" class="ext-link">Supporting Information Table S1</a>).</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-14/acs.jmedchem.8b00256/20180719/images/medium/jm-2018-002562_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-14/acs.jmedchem.8b00256/20180719/images/large/jm-2018-002562_0006.jpeg" id="GRAPHIC-d7e266-autogenerated" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Overview of the study. (A) Selection of 3D-structures in the PDB. (B) Numerical representation of binding mode used in this study. For the sake of illustration, depicted is the fragment X76 bound to human CDK2 ATP-binding pocket in the PDB structure <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3R1Y">3R1Y</a>. (C) Similarity between the binding modes of two small molecules bound to the same pocket. For the sake of illustration, displayed are the fragment X76 and drug-like ligand Z04 bound to human CDK2 ATP-binding pocket in the PDB structures <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3R1Y">3R1Y</a> and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3R7Y">3R7Y</a> (IFP sim = 1). (D) Similarity between the interactions coverage by all fragments and by all drug-like ligands bound to the same pocket. For the sake of illustration, displayed are all the ligands of human CDK2 ATP-binding pocket (cIFP sim = 0.895).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-14/acs.jmedchem.8b00256/20180719/images/large/jm-2018-002562_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00256&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The studied data set contains 1079 fragments and 1832 drug-like ligands found in 1404 and 2268 PDB files, respectively. In total, 240 binding sites in 235 different protein targets are considered (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>A). The average molecular weight of fragments and drug-like ligands is 204.5 ± 45.6 and 398.0 ± 79.2 Da. Binding modes are compared by investigating the noncovalent interactions between the small molecule and the protein pocket residues. These include nondirectional (hydrophobic) as well as directional (polar) interactions like hydrogen bonds (H-bonds), aromatic and ionic interactions, and also metal interactions with divalent cation. The numerical representation of interactions involves the definition of a binding site which is common to all the ligands bound to a pocket (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>B). We present here the study of two different aspects of binding modes: <i>binding mode conservation</i> and <i>interaction pattern similarity</i>. Focusing on binding mode conservation, we investigate the binding mode similarity between a single fragment and a single drug-like ligand (one-to-one comparisons scored using IFP similarity, <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>C). Focusing on interaction pattern similarity, we study the interactions of all fragments and compare them to the interactions of all drug-like ligands to answer the question of whether the two types of ligands display the same coverage of the pocket interactions (comparisons of consensus binding mode score using cIFP similarity, <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>D).</div><div class="NLM_p">In the last part of the manuscript, we extend the analysis to small crystallization additives and ask the question of whether they contain useful information for FBDD studies. The parsing of the PDB selected 3287 files allowing the comparison of the binding mode of additives with the binding mode of drug-like ligands bound to the same protein pocket (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>). We investigate here 126 additives found in 319 binding sites of 309 proteins. We especially focus on additives bound to a free protein (or <i>apo</i> additives). On average, <i>apo</i> additives have more accurate structures than other additives. About three-quarters of them show good local fit to the electron density (<a href="/doi/suppl/10.1021/acs.jmedchem.8b00256/suppl_file/jm8b00256_si_002.pdf" class="ext-link">Supporting Information Table S1</a>).</div><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5"> Part I: Fragment Binding Mode Conservation</h3><div class="NLM_p">The general assumption in FBDD when a fragment is extended into a drug-like ligand is that the binding mode of the shared substructure will be conserved. In this section, we focus on the binding mode conservation between a single fragment and a single drug-like ligand on the PDB scale. In particular, we aim at finding answers to these four different questions: (1) Is the binding mode of a single fragment conserved in multiple complexes with the same protein? (2) Is the binding mode conserved when extending the fragment into a drug-like ligand superstructure? (3) Is the binding mode conserved when extending the fragment into a structurally similar drug-like ligand? (4) Is there a correlation between fragment size and binding mode conservation?</div><div id="sec2_1_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i6" class="anchor-spacer"></div><h4 class="article-section__title" id="_i6"> Binding Mode Conservation of Fragments within the Same Protein Pocket</h4><div class="NLM_p">We investigated the binding mode of 453 fragments that have been crystallized multiple times with the same protein and within the same, ligandable, binding cavity. These fragments are found in 501 complexes, involving 152 binding sites in 149 proteins. In total, 1502 3D-structures are considered. Of note, a single PDB file can contain several biounits, generally distinguished by different chain names, and therefore can provide more than one 3D-structure for the same complex. More than two-thirds of the 501 complexes have only two 3D-structures, while a few of them have more than 10 copies in our data set (<a href="/doi/suppl/10.1021/acs.jmedchem.8b00256/suppl_file/jm8b00256_si_002.pdf" class="ext-link">Supporting Information Figure S1</a>).</div><div class="NLM_p">For each complex, we evaluate the degree of binding mode conservation by considering the minimum IFP similarity value obtained for the comparisons of all pairs of their 3D-structures (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>C). IFP similarity values range from zero (no common interactions) to 1 (exactly the same interactions). IFP similarity values in the 0.6–1 range correspond to low of root-mean-square deviation (rmsd) of the fragment atom coordinates with a mean value of 0.39 ± 0.36 Å (<a href="/doi/suppl/10.1021/acs.jmedchem.8b00256/suppl_file/jm8b00256_si_002.pdf" class="ext-link">Supporting Information Figure 2A</a>). Conversely, low IFP similarity values correspond to high rmsd of the fragment atom coordinates (mean rmsd = 7.62 ± 2.68 Å if 0 ≤ IFP sim < 0.2). A strict conservation of interactions is observed in about one-third of the cases (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>A). An example of lower binding mode conservation is the 3-isobutyl-1-methylxanthine (IBM) bound to human phosphodiesterase 9A2 (IFP sim = 0.62). In the two biounits of the PDB structure <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2HD1">2HD1</a>, IBM adopts a different position and shows variable torsion angles in its aliphatic part (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>B). The rmsd values of the fragment atom coordinates are respectively 3.8 and 1.3 Å before and after optimal superposition of the IBM poses in the two structures. In the two biounits, the binding sites are well aligned and no side chain flexibility can be observed. However, inspection of symmetry contacts in the crystal reveals significant changes in the environment of the fragment. Cases where rigid fragments have low binding mode conservation provide further evidence for the influence of the protein structure and crystallographic conditions. In the example of camphor (CAM) bound to <i>Pseudomonas putida</i> Camphor 5-monooxygenase, different protein backbone conformations correspond to different positions of camphor in the pocket (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>C). Noteworthy, in the two examples discussed above, the electron density well defines the position and orientation of the fragment in its binding pocket (<a href="/doi/suppl/10.1021/acs.jmedchem.8b00256/suppl_file/jm8b00256_si_002.pdf" class="ext-link">Supporting Information Table S2</a>).</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-14/acs.jmedchem.8b00256/20180719/images/medium/jm-2018-002562_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-14/acs.jmedchem.8b00256/20180719/images/large/jm-2018-002562_0007.jpeg" id="GRAPHIC-d7e369-autogenerated" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Binding mode conservation of the same fragment bound to multiple structures of the same protein site. (A) Histogram shows the frequency of the observed minimum binding mode similarity values. (B) 3-Isobutyl-1-methylxanthine (IBM) bound to human phosphodiesterase 9A2 in two biounits of the PDB structure <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2HD1">2HD1</a> (biounit 1 in magenta and biounit 2 in yellow). (C) Camphor (CAM) bound to <i>Pseudomonas putida</i> Camphor 5-monooxygenase in the PDB structures <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1DZ4">1DZ4</a> (green) and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4JX1">4JX1</a> (orange). In (B) and (C) protein backbone is displayed as cartoon and binding site residues are displayed as lines and fragments as sticks. Fragment chemical structures are inserted in the two images.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-14/acs.jmedchem.8b00256/20180719/images/large/jm-2018-002562_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00256&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Considering all the studied complexes, many nonconserved binding modes correspond to cases where the bound-conformation of a fragment is different in the multiple crystallographic structures of the same complex or when structural change occurs in the protein site. The removal of fragments with variable conformations in the same protein site from the data set results in an increased binding mode conservation (<a href="/doi/suppl/10.1021/acs.jmedchem.8b00256/suppl_file/jm8b00256_si_002.pdf" class="ext-link">Supporting Information Figure S2B</a>). The filtering of fragment structures where atoms are poorly resolved also discards many cases of low binding mode conservation. We found only 12 cases where very well-defined fragments have low binding mode conservation while no structural changes occur in the binding site and the fragments exhibit the same conformation and orientation in site. In these cases binding mode similarity is underestimated due to missed interactions caused by differential protonation or a threshold effect in the detection of interactions.</div><div class="NLM_p last">Overall, PDB fragments tend to exhibit the same position and orientation when crystallized multiple times within the same pocket (rmsd max of <0.5 Å in 73.8% of cases). Accordingly, the binding mode is generally conserved (min IFP sim ≥ 0.6 in 74.3% of cases). Noteworthy changes in fragment or protein conformation can be observed in a single PDB file.</div></div><div id="sec2_1_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i7" class="anchor-spacer"></div><h4 class="article-section__title" id="_i7"> Binding Mode Conservation of Fragments in Substructure Pairs</h4><div class="NLM_p">The conservation of a binding mode between a fragment and the same substructure incorporated into a larger, drug-like ligand has been mainly studied with examples, whereby different conclusions about binding mode conservation have been reached (see ref <a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> and references therein). As a recent example, the deconstruction of an allosteric inhibitor of the methyltransferase PRC2 has yielded a fragment that retains binding mode and has been optimized into larger ligand that also retains critical interactions.<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> In another study, crystallographic structures of the TGFBR1 kinase in complex with five small ligands have shown different binding modes of the common hinge-binding substructure, depending on the position and the type of the substitution at this core fragment.<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a></div><div class="NLM_p">In a previous study, we systematically analyzed the binding mode of fragments and drug-like ligands of four targets over-represented in the PDB (human β secretase 1, human cyclin-dependent kinase 2, human carbonic anhydrase 2, and bovine trypsin).<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> For the four targets, we have shown that the binding modes of fragments and their drug-like superstructures are highly similar. Here, we have extended the analysis to the PDB and can confirm the previous observations. Our data set includes 359 fragment–ligand substructure pairs bound to 51 proteins. When considering all or only hydrophilic interactions, fragments have on average highly conserved binding modes (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>A). Binding mode similarity is here computed as the proportion of fragment interactions that are retained in the drug-like ligand complex, and thus similarity values range from zero (no common interactions) to 1 (all interactions preserved). In almost 75% of the pairs, more than 80% of the interactions made by a fragment are conserved in the complex with its drug-like superstructure. For only a few of the substructure pairs, the median binding mode similarity from all complexes lies below 0.4, and interestingly most of them contain adenine as the substructure fragment. In a recent analysis of multitarget fragments in the PDB, we found that adenine has versatile binding modes and is indeed able to use different functional groups to recognize different proteins, including close homologs.<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a></div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-14/acs.jmedchem.8b00256/20180719/images/medium/jm-2018-002562_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-14/acs.jmedchem.8b00256/20180719/images/large/jm-2018-002562_0008.jpeg" id="GRAPHIC-d7e411-autogenerated" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Analysis of 359 fragment–ligand substructure pairs. (A) Overall binding mode similarity considering all interactions (left) and polar interactions (right). The median for each substructure pair is shown. (B) Effect of fragment binding mode variability on the binding mode similarity of fragment–ligand substructure pairs. Fragment CK2 (PDB structures <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1PXJ">1PXJ</a> and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2C5O">2C5O</a>) and its drug-like superstructure CK8 (PDB structure <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2C5N">2C5N</a>) bound to human cyclin dependent kinase 2. The protein in its active form is displayed as ribbon, the fragment (orange) and ligand (green) are displayed as sticks, and hydrogen bonds in the hinge region of the protein are in yellow. For the sake of comparison, the inactive form of the protein is shown as a light orange ribbon on the right panel. The proportion of interactions conserved is high (0.89) when considering fragment and ligand binding to the active form of the protein (left) and decreases (0.57) when considering fragment bound to the inactive form and ligand bound to the active one (right). (C) Overall pose similarity considering shape overlap (ROCS shape, left) and chemistry match (ROCS color, right). The median for each substructure pair is shown.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-14/acs.jmedchem.8b00256/20180719/images/large/jm-2018-002562_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00256&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">A variable binding mode of a fragment in its target site, although not frequent in our data set, is likely to decrease the median similarity computed from all 3D-structures of a pair. As shown in the previous part, variations in the protein structure often explain inconsistent binding modes. Classifying pairs in the data set according to the rmsd of the site Cα atomic coordinates indeed reveals that the ability of fragments to maintain their binding modes is inversely correlated to protein site flexibility (<a href="/doi/suppl/10.1021/acs.jmedchem.8b00256/suppl_file/jm8b00256_si_002.pdf" class="ext-link">Supporting Information Figure S3</a>). As an example, the fragment CK2 was crystallized in the active and inactive forms of human cyclin-dependent kinase 2 where it adopts two different poses to fit to changes in the ATP-binding pocket.<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> CK2 is a substructure of the drug-like ligand CK8, which was crystallized in the active form of the protein. Comparing the binding modes of CK2 and CK8 thus yields inconsistent similarity values (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>B). This result stresses again the importance of considering protein flexibility during fragment elaboration. It also suggests that interactions are better preserved than binding poses. Therefore, we also determined for all the 359 fragment–ligand substructure pairs whether the fragment pose is conserved (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>C). Pose similarity is here computed as shape overlap and chemistry match, and thus similarity values range from zero (no overlap/chemical match) to 1 (perfect overlap/chemical match). In almost all pairs, the shape overlap score is very high, thus demonstrating that a fragment and its related drug-like ligand occupy the same spot in the protein pocket. By contrast, chemistry match scores are significantly lower. Variable fragment poses are therefore frequent in the compared pairs. Noteworthy, chemistry match scores are very sensitive to the atomic coordinates and therefore detect ring flips in the fragment and shifts of the entire fragment within the protein pocket. Recalling <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>B which compares the binding of the fragment CK2 and the drug-like ligand CK8 to human CDK2, flip of the thioazole substructure (left image) causes the chemistry match score decrease to 0.638 while it marginally affects the binding mode (IFP sim = 0.89). In the case of the entire fragment tilt in the pocket (right image), the chemistry match score drops to 0.144 while some key interactions are preserved (IFP sim = 0.57). Density maps clearly support the relocation of CK2 fragment upon protein conformational change, yet they do not define a single placement of the thiazole substructure.</div><div class="NLM_p">In a FBDD project, fragment growing is not restricted to exact substructure pairs. For example the expansion of fragments in definite directions may require fragment modifications for the sake of synthesis. Small changes in the chemical scaffold may also be beneficial to druglikeness, as assessed by ligand efficiency and other related metrics.<a onclick="showRef(event, 'ref16 ref17'); return false;" href="javascript:void(0);" class="ref ref16 ref17">(16,17)</a> When considering fragment–ligand pairs with high chemical similarity (ECFP2 similarity of ≥0.7; 1533 pairs in 3300 comparisons), high binding mode conservation (median conservation above 80%) is still reached (<a href="/doi/suppl/10.1021/acs.jmedchem.8b00256/suppl_file/jm8b00256_si_002.pdf" class="ext-link">Supporting Information Figure S4</a>). Conservation of polar interactions is especially high (median conservation of 100%), but there is a large variance in binding mode conservation levels. Discarding pairs with changes in the binding site structure drastically increases the proportion of pairs with identical or very similar polar interactions.</div><div class="NLM_p last">Overall, PDB fragments tend to make the same interactions with a protein pocket when alone and included in a larger drug-like ligand. Moreover, variations in the fragment pose were more frequently observed but have limited effect on the interactions made with the protein pocket, especially the polar ones which are highly preserved.</div></div><div id="sec2_1_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i8" class="anchor-spacer"></div><h4 class="article-section__title" id="_i8"> Binding Mode Conservation and Fragment Size</h4><div class="NLM_p">Is it possible to predict which fragments are most likely to change their binding mode during growing? This question was recently addressed by Malhotra and Karanicolas, who analyzed protein complexes from the PDBbind database.<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> They considered 297 pairs of molecules bound to the same protein where one ligand is a substructure of the other ligand, both molecules having between 6 and 55 non-hydrogen atoms and the substructure–ligand ratio of molecular weight (MW) being at least 1.3. Binding mode conservation was observed in 86% of pairs. Examination of ligand and protein descriptors revealed no statistically significant correlations with changes in the binding mode. Binding mode is however better conserved for the smaller ligands which comply the rule-of-three.<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> Noteworthy, only 68 pairs analyzed by Malhotra and Karanicolas pass our selection protocol (pairs were discarded because the smaller molecule has a MW of >300 Da, because the larger molecule does not comply the rule-of-five, or because of mutations in binding site).</div><div class="NLM_p">In our previous study on four targets which are over-represented in the PDB,<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> we also observed that there is a relationship between the fragment size and the binding mode conservation. In particular, in 90% of the studied complexes with conserved binding modes, the number of additive/fragment heavy atoms was higher than 8 and the MW was higher than 110 Da. We here extend our analysis to the 359 substructure pairs and the 1533 chemically similar pairs in the PDB (<a href="/doi/suppl/10.1021/acs.jmedchem.8b00256/suppl_file/jm8b00256_si_002.pdf" class="ext-link">Supporting Information Figure S5</a>). We confirm that binding mode is overall conserved if the fragment MW is high enough, with a threshold around 150 Da. The same trend is observed for the fragment binding pose, although the level of conservation is generally lower, as previously mentioned in the analysis of substructure pairs (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>).</div><div class="NLM_p last">Are there other reasons why a fragment binding pose varies? Case studies are more suitable to unravel the cause of change in binding mode than a global analysis. For example, Schauperl et al. used molecular dynamics simulations to provide a thermodynamical understanding of the variable binding mode of fragments in complex with the TGFBR1 kinase.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a></div></div></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i9" class="anchor-spacer"></div><h3 class="article-section__title" id="_i9"> Part II: Similarity between Fragment and Drug-like Interaction Patterns</h3><div class="NLM_p">In this section, we examine the coverage of protein binding site by fragments and drug-like ligands. Therefore, all available fragment- and drug-like ligand–protein complexes of the same target are considered and global interaction patterns of the fragment and drug-like ligand sets are compared.</div><div id="sec2_2_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i10" class="anchor-spacer"></div><h4 class="article-section__title" id="_i10"> Only a Few Ligandable Proteins in the PDB Have High Interaction Coverage Data Sets</h4><div class="NLM_p">In the entire PDB we found 240 ligandable sites, in 235 proteins, in complex with both fragment and drug-like ligands. When only one fragment and one drug-like ligand complex are available, different levels of interaction pattern similarity are observed. Therefore, we hypothesized that the more fragment and drug-like ligands are available for a given target, the more chances of mapping the entire pocket and thus the more chances of high interaction coverage. Indeed, our hypothesis was confirmed, as indicated in <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>. As a rule of thumb, data suggest that at least nine different fragments and nine different drug-like ligands in complex with the same target are necessary to observe good interaction coverage between the two sets. In our PDB data set, there are only 11 proteins which fulfill this rule (“ligand- and fragment-rich targets”; <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). They however belong to diverse functional classes.</div><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-14/acs.jmedchem.8b00256/20180719/images/medium/jm-2018-002562_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-14/acs.jmedchem.8b00256/20180719/images/large/jm-2018-002562_0009.jpeg" id="GRAPHIC-d7e493-autogenerated" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Similarity between fragment and drug-like interaction patterns. Dependence of fragment–ligand interaction pattern similarity on the number of different molecules considered. In each step, all targets with at least <i>X</i> (1, 2, 3, etc.) HET codes of both fragments and drug-like ligands are considered. Number of distinct binding sites is shown in red.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-14/acs.jmedchem.8b00256/20180719/images/large/jm-2018-002562_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00256&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Ligand- and Fragment-Rich Targets<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-14/acs.jmedchem.8b00256/20180719/images/medium/jm-2018-002562_0001.gif" alt="" id="GRAPHIC-d7e501-autogenerated" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup><sup>a</sup></sup><p class="last">The colors indicate the diversity within the fragment or ligand sets. Green: high diversity (average(1 – similarity) > 0.75). Light green: medium diversity (average(1 – similarity) > 0.6). Tanimoto similarity between molecules is calculated using ECFP2 fingerprints.</p></div></div><div></div></div></div><div id="sec2_2_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i12" class="anchor-spacer"></div><h4 class="article-section__title" id="_i12"> Conditions for Similar Coverage of the Pocket Interactions by Fragment and Drug-like Ligands</h4><div class="NLM_p">When investigating the pocket properties of the 11 ligand- and fragment-rich targets, we observe large variability in size, polarity, and flexibility (<a href="/doi/suppl/10.1021/acs.jmedchem.8b00256/suppl_file/jm8b00256_si_002.pdf" class="ext-link">Supporting Information Table S3</a>). The size of the pocket, for instance, varies between 25 and 60 residues and the average volume between 327 and 739 Å<sup>3</sup>. It is interesting to note that the two largest pockets when considering the residue count (BACE1 and LTA4H) have the lowest interaction coverage between ligands and fragments, while the smallest pocket (BRD4) has the highest interaction pattern similarity. For the other ligand- and fragment-rich target properties, however, no correlation with the interaction pattern similarity is apparent. When extending the analysis to all targets with good interaction coverage (≥0.6; 81 targets), the pocket properties are even more variable. The count of pocket residues in those targets lies between 20 and 78 residues, the average volume between 150 and 1037 Å<sup>3</sup>, the volume variation between 155 and 1121 Å<sup>3</sup>, the average Cα rmsd to the reference structure between 0.1 and 4.3 Å, and the average percentage of polar points between 27% and 94%. Taken together, the results suggest that high coverage of fragment and drug-like ligand interactions can be observed independent of the pocket properties.</div><div class="NLM_p">We then verified whether the similarity of the interaction patterns is not overestimated due to low chemical diversity of the ligands. The structural diversity within the fragment and drug-like ligand sets of the ligand- and fragment-rich targets ranges between medium and high (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>, <a href="/doi/suppl/10.1021/acs.jmedchem.8b00256/suppl_file/jm8b00256_si_002.pdf" class="ext-link">Supporting Information Table S3</a>). In most of the cases, the diversity within the fragment set is higher than the diversity of the drug-like ligands. By comparison of the fragments and the drug-like ligands, interset similarity is relatively low, although there are, for 10 of the 11 targets, a few pairs where the fragment is an exact or a close substructure of the drug-like ligand (<a href="/doi/suppl/10.1021/acs.jmedchem.8b00256/suppl_file/jm8b00256_si_002.pdf" class="ext-link">Supporting Information Figure S6</a>). The exception is LTA4H, where the maximal similarity of fragments and ligands is 0.63 and the interaction pattern similarity is relatively low (0.425). The low similarity is however not due to fragment and drug-like ligands interaction with mainly specific residues, yet to an unbalanced distribution of fragments and ligands within the binding site (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>). Whereas drug-like ligands (green) are evenly distributed over the entire LTA4H pocket, all but one fragment (magenta) are found on the left site of the pocket. Correspondingly, the distribution of fragments and drug-like ligands in the BACE1 pocket (<a href="/doi/suppl/10.1021/acs.jmedchem.8b00256/suppl_file/jm8b00256_si_002.pdf" class="ext-link">Supporting Information Figure S7</a>) also explains the rather low interaction pattern similarity observed for this target, stressing that our computing approach can underestimate interaction coverage in the cases of poor spatial coverage.</div><figure id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-14/acs.jmedchem.8b00256/20180719/images/medium/jm-2018-002562_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-14/acs.jmedchem.8b00256/20180719/images/large/jm-2018-002562_0010.jpeg" id="GRAPHIC-d7e556-autogenerated" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Coverage of human leukotriene A-4 hydrolase pocket. (A) Drug-like ligand (left) and fragment (right) interaction heatmaps. Different types of interactions (hydrophobic, HYD; aromatic, AROM; hydrogen bonding, HB; ionic, IONIC; metal, METAL) are displayed on the <i>X</i>-axis. The binding site residues (one-letter code, residue number, chain) are displayed on the <i>y</i>-axis. The color intensity describes the frequency of the observed interaction in all complexes for this set (e.g., fragments). Residue number 1 is zinc. (B) Overlay of all fragments and drug-like ligands in the reference 3D-structure. Drug-like ligands (green) and fragments (magenta) are shown as sticks, the binding site is shown as surface, and the residues of the binding site are shown as lines.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-14/acs.jmedchem.8b00256/20180719/images/large/jm-2018-002562_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00256&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">Lastly, we asked ourselves whether some polar interactions are specific to fragment or drug-like ligands. We observe that on average 20% of polar interactions found in fragment complexes are unique and thus not found in drug-like ligand complexes. For example, fragments form two specific H-bonds with LTA4H (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>A). Considering the 11 ligand- and fragment-rich proteins, there are target-dependent differences in the percentage of unique polar interactions (<a href="/doi/suppl/10.1021/acs.jmedchem.8b00256/suppl_file/jm8b00256_si_002.pdf" class="ext-link">Supporting Information Table S4</a>). The values can vary between 0% and 39% for unique H-bonds and between 0% and 100% for unique aromatic bonds. It could be assumed that a factor that influences the results is the nature of the fragments and drug-like ligands in the data set. However, this is not confirmed for the ligand- and fragment-rich targets, where neither the overall fragment–ligand set similarity (<a href="/doi/suppl/10.1021/acs.jmedchem.8b00256/suppl_file/jm8b00256_si_002.pdf" class="ext-link">Supporting Information Table S3 and Figure S6</a>) nor the count of H-bond donors or acceptors (data not shown) correlates with the observed unique interaction ratio. The differences cannot be simply explained by the diverse polarities of the corresponding binding sites (<a href="/doi/suppl/10.1021/acs.jmedchem.8b00256/suppl_file/jm8b00256_si_002.pdf" class="ext-link">Supporting Information Table S4</a>).</div></div></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i13" class="anchor-spacer"></div><h3 class="article-section__title" id="_i13"> Part III: Can Crystallization Additives Be Regarded as Fragments?</h3><div class="NLM_p">In contrast to fragments that are crystallized with the target protein on purpose, other small molecules identified in PDB structures had been unintentionally incorporated into the protein crystals.<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> These compounds are buffers, reducing agents, ions, detergents, or cryoprotectants added to the experimental sample for solubilizing and stabilizing the protein. They can also be precipitants and additives of various chemical nature and size added to the experimental sample for promoting crystal formation. A study of additives interaction in crystals grown in different experimental conditions has emphasized the important role of additives in the crystal formation, showing that additives directly mediate intermolecular crystal contacts or induce conformational changes at the protein surface.<a onclick="showRef(event, 'ref22 ref23'); return false;" href="javascript:void(0);" class="ref ref22 ref23">(22,23)</a> An interesting case is the additive benzamidine, which enhances the crystallization of trypsin when it is bound to the active site.<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> Benzamidine is actually also a substructure of drug-like competitive inhibitors of trypsin, and its binding mode to the enzyme active site is indeed preserved in the drug-like superstructures;<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> thereby benzamidine can also be regarded as fragment. We were here wondering whether we can find in the PDB other examples of protein–additive complexes containing useful information for computational drug design.</div><div class="NLM_p">Focusing on small additives with a MW below 300 Da, we repeated the analyses made for fragments (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>): we first checked whether additives have a consistent binding mode in a protein site, next we compared additives and drug-like ligands binding modes, pair-by-pair for each protein, and last, we considered the coverage of binding site interactions by additives and by drug-like ligands. We distinguished between additives bound to a free protein (<i>apo</i> additives) and those present in a protein in complex between another molecule, e.g., a fragment or drug-like ligand. This distinction is important, since additives can hardly compete with a larger and stronger ligand for targeting protein hot spots. When considering the additives as a whole, we found that additives and fragments behave differently, while when focusing on <i>apo</i> additives, we observed common trends (see <a href="/doi/suppl/10.1021/acs.jmedchem.8b00256/suppl_file/jm8b00256_si_002.pdf" class="ext-link">Supporting Information Figures S8, S9, and S10</a>).</div><div class="NLM_p">An example of a target with many fragment– and additive–drug-like ligand substructure pairs is human macrophage metalloelastase (MMP12). As shown in <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>, both structures of the additive HAE, one being an <i>apo</i> additive structure, show a perfect conservation of the binding mode to the drug-like ligand superstructure CGS. Apart from hydrophobic interactions, the metal interactions with the zinc ion as well as a H-bond to Ala182 are conserved. Another example of well conserved binding modes is displayed in <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>. Bacillus thermoproteolyticus thermolysin is one of the targets with multiple <i>apo</i> additive complexes. In a previous study, the benzene ring of the fragment <i>N</i>-(phenylcarbonyl)-β-alanine (BYA) has been identified as a hot spot because several ligands place this ring at the given position.<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> Interestingly, several <i>apo</i> additives map the directional interactions of BYA (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>) and show good binding mode conservation. In this case, all of the additives are very small (four or five heavy atoms), indicating that a certain molecular complexity is not necessary to observe binding mode conservation.</div><figure id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-14/acs.jmedchem.8b00256/20180719/images/medium/jm-2018-002562_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-14/acs.jmedchem.8b00256/20180719/images/large/jm-2018-002562_0011.jpeg" id="GRAPHIC-d7e669-autogenerated" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Examples of additive binding mode conservation. (Left panel) Example of the binding mode conservation of an additive/drug-like ligand substructure pair. Example of the drug-like ligand CGS (green, PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1JIZ">1JIZ</a>) and its additive substructure HAE bound alone (white, PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1OS2">1OS2</a>) and in the presence of the ligand EEG (light pink, PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3LIK">3LIK</a>) in the human macrophage metalloelastase (MMP12) pocket. The protein is shown as green cartoon, and important protein residues are shown as sticks. Polar interactions are indicated as dotted lines. The zinc ion is shown as small sphere. (Right panel) Example of <i>apo</i> additives matching the polar interactions of a fragment. Bacillus thermoproteolyticus thermolysin bound to the fragment BYA (green, PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3FGD">3FGD</a>) and three examples of <i>apo</i> additives: 2-bromoacetate (cyan; PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3NN7">3NN7</a>), <i>S</i>-1,2-propanediol (blue, PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3N21">3N21</a>), and acetic acid (white, PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2A7G">2A7G</a>). The small molecules are shown are sticks, the protein is shown as cartoon, the interacting protein residues are shown as lines, and the zinc ion is shown as a small sphere. Polar interactions are indicated as dotted lines: hydrogen bonds in green and metal interaction in yellow. All electron densities have been deposited and correspond with the shown structures.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-14/acs.jmedchem.8b00256/20180719/images/large/jm-2018-002562_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00256&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">When considering chemical structure, the set of additives is very heterogeneous. There are small inorganic and organic ions (e.g., phosphate and Tris buffer), long straight-chain compounds (e.g., fatty acid and polyethylene glycol), small polyols (e.g., 1,2-ethanediol), small aromatic compounds (e.g., benzoic acid), and other polar cyclic or linear compounds of various sizes (e.g., DMSO, panthothenic acid, or MES buffer). We thus questioned whether binding mode conservation depends on the chemical nature of additives. When we investigated the variability of additive binding modes within the same protein, we observed that although there are high discrepancies in the classes containing the smallest compounds (small anions, small aromatic compounds, small inorganic compounds, and small polyols), more than half of the additives cluster in three or less spots of a protein site (<a href="/doi/suppl/10.1021/acs.jmedchem.8b00256/suppl_file/jm8b00256_si_002.pdf" class="ext-link">Supporting Information Figure S11</a>). For example, the 10 poses of malonate bound to <i>E. coli</i> aminopeptidase N pocket cluster into three groups in which the orientation and interactions are well preserved (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>, left). By contrast, the 18 poses of glycerol bound to the same protein pocket define a continuous shape overlapping the hydrophobic parts of the drug-like ligand actinonin (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>, right).</div><figure id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-14/acs.jmedchem.8b00256/20180719/images/medium/jm-2018-002562_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-14/acs.jmedchem.8b00256/20180719/images/large/jm-2018-002562_0012.jpeg" id="GRAPHIC-d7e704-autogenerated" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Examples of multiple additive poses in a protein site. (Left panel) Malonate (MLI, pink) bound to <i>E. coli</i> aminopeptidase N pocket (UniProt code <a href="https://P04825" class="ext-link">P04825</a>). The protein is shown as green cartoon and zinc ion as sphere. (Right panel) Glycerol (GOL, pink) and actinonin (BB2, green, PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4Q4E">4Q4E</a>) bound to <i>E. coli</i> aminopeptidase N pocket. The protein is shown as green cartoon, and hydrophobic protein residues in site are shown as lines.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-14/acs.jmedchem.8b00256/20180719/images/large/jm-2018-002562_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00256&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Exploring the interactions made by additives revealed that the small anions can act as molecular probes to reveal charged regions in the protein pocket and that the long straight-chain compounds can act as molecular probes to reveal hydrophobic regions. More surprisingly, we found that polar additives such as small polyols or DMSO are largely engaged in hydrophobic contacts (<a href="/doi/suppl/10.1021/acs.jmedchem.8b00256/suppl_file/jm8b00256_si_002.pdf" class="ext-link">Supporting Information Figure S12</a>). The above-described example of glycerol bound to <i>E. coli</i> aminopeptidase N pocket well illustrates that small polyols tend to bind efficiently to hydrophobic regions of the protein. In the 18 structures available for glycerol bound to <i>E. coli</i> aminopeptidase N, we identified 86 interactions between the additive and protein, including 56 hydrophobic contacts with 10 protein residues and 30 H-bonds with 14 protein residues. Two additional examples of polar additives engaged in an H-bond on one side and in hydrophobic contact on the other side are shown in <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>. Glycerol makes H-bonds with the hinge region of the ALK tyrosine kinase receptor, like most inhibitors of kinase ATP binding sites. The carbon atoms of glycerol face hydrophobic side chains in the protein pocket. Another example is 1,2-ethanediol which is engaged in two H-bonds with a hydrophobic site of the bromodomain-containing protein 2. Of note, the additive was crystallized alone in the protein in the two examples (<i>apo</i> additive).</div><figure id="fig8" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-14/acs.jmedchem.8b00256/20180719/images/medium/jm-2018-002562_0013.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-14/acs.jmedchem.8b00256/20180719/images/large/jm-2018-002562_0013.jpeg" id="GRAPHIC-d7e735-autogenerated" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Examples of additive interactions. (Left panel) Glycerol (GOL, pink) and drug-like inhibitor (OFG, green, PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4CCB">4CCB</a>) bound to the hinge region of the ALK tyrosin kinase receptor ATP binding site. (Right panel) 1,2-Ethanediol (EDO, pink) and the inhibitor GSK525762 (EAM, green, PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2YEK">2YEK</a>) bound to bromodomain-containing protein 2. The protein is shown as green cartoon, and hydrophobic protein residues in site are shown as lines. Hydrogen bonds are indicated as yellow dotted lines.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-14/acs.jmedchem.8b00256/20180719/images/large/jm-2018-002562_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00256&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i14">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_48376" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_48376" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">The PDB-wide analysis of fragment binding modes has answered several questions related to FBDD projects (see <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>). The study revealed that a fragment crystallized multiple times in the same protein cavity tends to occupy the same position and form the same interactions in the binding pocket. In addition, interactions made by fragments are well conserved in the related drug-like ligands, provided fragment MW is larger than 150 Da. Directional interactions, especially H-bonds, are highly conserved. Examples of variable binding modes drew our attention to protein flexibility and to the ability of some ligands to adopt multiple bound conformations. Growing fragments into structurally similar ligands results in good conservation of polar interactions provided the chemical modification does not induce a change in the protein structure.</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Main Conclusions for FBDD</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">question</th><th class="colsep0 rowsep0" align="center">answer</th><th class="colsep0 rowsep0" align="center">cases</th><th class="colsep0 rowsep0" align="center">comments</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Does a fragment always bind a protein pocket in a similar way?</td><td class="colsep0 rowsep0" align="left">Yes</td><td class="colsep0 rowsep0" align="left">74% (IFP sim ≥ 0.6)</td><td class="colsep0 rowsep0" align="left">Exceptions: protein flexibility, multiple conformations of fragment, multiple tautomeric states, multiple molecules within the binding pocket.</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Is the binding mode between a fragment and its drug-like superstructure conserved?</td><td class="colsep0 rowsep0" align="left">Yes</td><td class="colsep0 rowsep0" align="left">75% (IFP sim ≥ 0.8)</td><td class="colsep0 rowsep0" align="left">Exceptions: protein flexibility.</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">Interactions are better conserved than pose.</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Is the binding mode between a fragment and a similar drug-like ligand conserved?</td><td class="colsep0 rowsep0" align="left">Yes</td><td class="colsep0 rowsep0" align="left">62.5% (ECFP2 sim ≥ 0.7, IFP sim ≥ 0.8)</td><td class="colsep0 rowsep0" align="left">Exceptions: protein flexibility.</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">Polar interactions are better conserved.</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Is there an influence of fragment size on binding mode conservation?</td><td class="colsep0 rowsep0" align="left">Yes</td><td class="colsep0 rowsep0" align="left">Inferred from 623 similar fragment–ligand pairs</td><td class="colsep0 rowsep0" align="left">Binding mode is generally conserved if MW > 150 Da. Same trend is observed for binding pose.</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Is there a minimal data set size to observe interaction pattern similarity between fragment and ligand complexes?</td><td class="colsep0 rowsep0" align="left">Yes</td><td class="colsep0 rowsep0" align="left">Inferred from 235 proteins binding 1079 fragments and 1832 drug-like ligands</td><td class="colsep0 rowsep0" align="left">>9 different fragments and ligands should be crystallized in the protein pocket (fulfilled by 11 proteins only).</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Can crystallization additives be regarded as fragments in drug design?</td><td class="colsep0 rowsep0" align="left">No</td><td class="colsep0 rowsep0" align="left">Inferred from 309 proteins binding 1079 fragments and 1832 drug-like ligands</td><td class="colsep0 rowsep0" align="left">Additive binding modes are more variable than those of fragments and rarely bound alone in the pocket.</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"><i>apo</i> additives however reveal interactions made by drug-like ligands.</td></tr></tbody></table></div></div><div class="NLM_p">When comparing interaction coverage of fragments and drug-like ligands, we observed that the more fragments and drug-like ligands have been crystallized with a protein, the better is the sampling of interactions in its binding site. We evaluated that nine different fragments are enough to achieve good interaction coverage for diverse pockets. Besides, fragments tend to have unique polar interactions (not seen in complexes with drug-like ligands) in polar protein pockets.</div><div class="NLM_p">The study of small additives binding to ligandable PDB proteins showed that clusters of <i>apo</i> additives usually reveal interactions made by drug-like ligands. Interestingly, binding of small polyols and small polar compounds (e.g., glycerol or DMSO) involves both H-bonds and hydrophobic contacts. In conclusion, from a purely structural perspective, small additives cannot be regarded as fragments because their binding modes are too variable; nevertheless they provide key information on the protein binding properties, especially revealing pharmacophoric anchor points in the protein and supporting target ligandability.</div><div class="NLM_p last">The main conclusions of the study are summarized in <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i16">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_57348" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_57348" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17"> Data Set and General Procedure</h3><div class="NLM_p last">The analysis was performed on publicly available data from the RCSB PDB Web site.<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> Crystal structures of additive, fragment, or ligand–protein complexes were filtered as described previously.<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> Briefly, structures with a resolution of ≤3 Å that were deposited between January 2000 and August 2016 and containing at least one protein chain were analyzed. Fragments were defined as small molecules with a molecular weight (MW) below 300 Da and between 2 and 18 heavy atoms, whereas ligands were defined as nonfragments following the scPDB<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> ligand rules as well as the rule of 5<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> with up to one exception. Crystallization additives were removed from the list of fragments and ligands and treated separately if they were found to agree with the fragment rules (small additives).</div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18"> Preparation of PDB Files</h3><div class="NLM_p">All PDB files were downloaded from the RCSB PDB Web site and prepared in a number of steps, similar to the previously described procedure.<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> First, protein complexes were protonated using Protoss<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> and their binding pockets were identified as all residues having at least one atom within 6.5 Å around the ligand. Protein residues were renumbered to match the Universal Protein Resource (UniProt)<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> residue numbering. This was achieved using a protein chain sequence alignment with the EMBOSS needle package<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> or, in cases with many gaps and mismatches, a structure-based sequence alignment with MOE (Molecular Operating Environment, 2016.08; Chemical Computing Group ULC, Montreal, Canada). For each target with drug-like ligands, one or multiple “common” binding sites were defined as follows. A first distinction was made between pockets containing one (“monomeric” pockets) or multiple protein chains (“dimeric” pockets, etc.). Because the study focuses on the comparison of binding modes within one protein target, heteromultimeric pockets were not considered in the analysis. Second, drug-like ligand binding sites were clustered based on their residue overlap to distinguish between spatially separated sites of a target. This was achieved using a hierarchical clustering with average linkage. When two site clusters show a residue overlap of at least 5%, the clusters are merged. For each binding site cluster, residues occurring in more than 10% of sites were defined as the “common” site.</div><div class="NLM_p last">All PDB files containing mutations or missing residues in the common site were removed from the analysis. The remaining PDB files for each target and common site were superimposed to a reference structure using CE.<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> The reference was chosen as the centroid according to binding site similarity calculated with the Shaper software.<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> Finally, binding modes of all molecules overlapping with the drug-like ligand cavity were investigated in terms of interaction fingerprints (IFPs)<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> calculated using the in-house IChem software.<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> Interaction fingerprints represent the presence and absence of specific interaction types with the binding site residues. Default IChem settings were used to detect H-bonding (distance between H-bond donor atom and H-bond acceptor atom is lower the 3.5 Å, and H-bond angle is 180° ± 60°), ionic interaction (distance between positively and negatively charged atoms is lower than 4.0 Å), and interaction between a H-bond acceptor and a metallic cation (distance is lower than 2.8 Å). Hydrophobic contacts were detected between carbon atoms if the interatomic distance is lower than 5 Å. Aromatic interactions were detected as follows: in π–π interactions, the distance between π ring centers is lower than 5.0 Å and angle between π ring planes is 180° ± 30 or 90° ± 60; in π–cation interaction, the distance between π ring center and aromatic cation is lower than 5.0 Å and the angle between planes is 180° ± 30. Water molecules, which are not built in all the studied PDB structures, were not considered in the analysis.</div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i19" class="anchor-spacer"></div><h3 class="article-section__title" id="_i19"> Comparison of Binding Modes and Interaction Patterns</h3><div class="NLM_p">To compare the binding mode conservation of the same fragment within a protein pocket, the IFP similarity was calculated using the Tanimoto metric (Tc) and the minimal observed similarity for a given molecule and target was investigated. It was also determined whether the fragment in different PDB structures binds to the protein pocket with a similar pose and with a similar conformation. To that purpose, the rmsd of non-hydrogen atom coordinates of the fragment was computed using a script in python 2.7.14, respectively, before and after optimal superposition of the compared fragment poses using the maximum common substructure alignment from oechem library (OpenEye Scientific, USA).</div><div class="NLM_p">To compare the binding mode conservation of a fragment (or an additive) with the binding mode of drug-like ligand within the same protein pocket, IFP similarity was calculated as a proportion (Pr) of the common interactions divided by the smaller molecule’s interactions, as described by the following formula:<span class="NLM_disp-formula" id="ueq1"><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-14/acs.jmedchem.8b00256/20180719/images/medium/jm-2018-002562_m001.gif" alt="" id="_i20" /></img></span>where molecule A is smaller than molecule B (i.e., A is a fragment or an additive and B is a drug-like ligand), <i>o</i> is the count of overlapping interactions, and <i>u</i> is the unique molecular interactions.</div><div class="NLM_p">For the one-to-one comparisons between a fragment and its drug-like superstructure within the same protein pocket (i.e., the substructure pairs), it was also determined whether the fragment alone and included in a larger ligand binds to the protein pocket with a similar pose. To that purpose, shape overlap and chemistry match were computed using ROCS v3.2.0.4 (OpenEye Scientific, USA). Importantly, scoring calculation only was performed since all the structures were 3D-aligned onto the reference structure of the protein pocket beforehand. The shape overlap (ROCS shape) and chemistry match (ROCS color) were evaluated using the Tversky and ColorTversky scores, respectively.</div><div class="NLM_p">To compare the interaction patterns of all the fragments and all the ligands bound to the same protein pocket, consensus interaction fingerprints (cIFPs) were generated. These are numerical fingerprints describing the frequency of each interaction within the given set (fragments or drug-like ligands). To describe the similarity between two cIFPs, the Tanimoto coefficient, defined for continuous variables,<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> was used.</div><div class="NLM_p">In addition, numerical cIFPs were converted into binary fingerprints to study the ratios of unique and shared interactions between two molecule sets. Different thresholds to convert the fingerprints were tested, but the clearest results were generally obtained when a threshold of >0 was set. Using this threshold, every interaction observed at least once is converted into an on-bit in the binary fingerprint.</div><div class="NLM_p last">Importantly, all interaction similarities (as expressed by IFP Tc, IFP Pr, or cIFP similarity) were calculated separately for hydrophobic and polar interactions and both weighted with 0.5.</div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i21" class="anchor-spacer"></div><h3 class="article-section__title" id="_i21"> Molecular Properties</h3><div class="NLM_p last">To investigate the similarity/diversity within a molecule set as well as between sets, extended-connectivity fingerprints with the radius 2 (ECFP2) were calculated using Pipeline Pilot 2016 (BIOVIA, Dassault Systèmes, France). Furthermore, several molecular properties were tested for their correlation with binding mode similarity: molecular weight and the count of heavy atoms. Both of them were also calculated with Pipeline Pilot 2016.</div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i22" class="anchor-spacer"></div><h3 class="article-section__title" id="_i22"> Pocket Properties</h3><div class="NLM_p last">Properties of binding sites were calculated using the in-house IChem software (Volsite<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a>). For each common binding site, the average descriptors of all pockets of this target were determined. Apart from calculating the pocket volume, the software determines all possible interaction points describing different molecular interactions (e.g., hydrophobic points, hydrogen bonding points, etc.). For the pocket volume, the pocket volume variation was also determined by subtracting the minimum from the maximum volume.</div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i23" class="anchor-spacer"></div><h3 class="article-section__title" id="_i23"> Box Plot</h3><div class="NLM_p">Box plots were generated by matplotlib 2.1.0 in python 2.7.14. Only <a href="/doi/suppl/10.1021/acs.jmedchem.8b00256/suppl_file/jm8b00256_si_002.pdf" class="ext-link">Figure S12</a> use the display obtained using the default settings. In the other figures, the box gives the first and third quartiles, the median is shown in red, the whiskers indicate the first and ninth deciles, and outliers are not represented.</div><div class="NLM_p last">A Web interface has been designed to query the data presented in this article. It is freely available at <a href="http://bioinfo-pharma.u-strasbg.fr/PDBmob" class="extLink">bioinfo-pharma.u-strasbg.fr/PDBmob</a>.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i24"><a href="/doi/suppl/10.1021/acs.jmedchem.8b00256" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_46872" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_46872" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a href="/" class="ext-link">ACS Publications website</a> at DOI: <a href="/doi/abs/10.1021/acs.jmedchem.8b00256" class="ext-link">10.1021/acs.jmedchem.8b00256</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Molecular formula strings and PDB and HET codes (<a href="/doi/suppl/10.1021/acs.jmedchem.8b00256/suppl_file/jm8b00256_si_001.xlsx" class="ext-link">XLSX</a>)</p></li><li><p class="inline">Number of structures available for fragment–protein complexes (Figure S1), effect of conformational change and structure quality on the binding mode conservation of the same fragment within the same binding pocket (Figure S2), effect on protein flexibility on binding modes conservation (Figure S3), dependence of fragment–ligand binding mode conservation on chemical similarity (Figure S4), fragment size and conservation of binding mode and binding pose (Figure S5), chemical similarity between fragment and ligand sets for the fragment and ligand-rich targets (Figure S6), coverage of human β secretase pocket (Figure S7), binding mode conservation of the same additive bound to multiple structures of the same protein site (Figure S8), binding mode conservation of additive–ligand substructure pairs (Figure S9), dependence of additive–drug-like ligand interaction pattern similarity on the number of different molecules (Figure S10), variability of additive binding mode in multiple structures of the same protein site (Figure S11), nature of interaction made by additives (Figure S12), 3D-structure quality evaluated using the electron density support for individual atoms (Table S1), EDIA score of the complexes described in the figures (Table S2), properties of ligand and fragment-rich targets (Table S3), and unique polar interactions of targets with many fragments and ligands (Table S4) (<a href="/doi/suppl/10.1021/acs.jmedchem.8b00256/suppl_file/jm8b00256_si_002.pdf" class="ext-link">PDF</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.8b00256/suppl_file/jm8b00256_si_001.xlsx">jm8b00256_si_001.xlsx (11.93 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.8b00256/suppl_file/jm8b00256_si_002.pdf">jm8b00256_si_002.pdf (1.84 MB)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.8b00256" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_50784" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_50784" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Esther Kellenberger</span> - <span class="hlFld-Affiliation affiliation">Laboratoire
d’Innovation Thérapeutique, UMR7200, Université de Strasbourg, 74 Route du Rhin, 67401 Illkirch, France</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-9320-4840" title="Orcid link">http://orcid.org/0000-0002-9320-4840</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#07626c626b6b62694772696e74737566296175"><span class="__cf_email__" data-cfemail="84e1efe1e8e8e1eac4f1eaedf7f0f6e5aae2f6">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Malgorzata
N. Drwal</span> - <span class="hlFld-Affiliation affiliation">Laboratoire
d’Innovation Thérapeutique, UMR7200, Université de Strasbourg, 74 Route du Rhin, 67401 Illkirch, France</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Guillaume Bret</span> - <span class="hlFld-Affiliation affiliation">Laboratoire
d’Innovation Thérapeutique, UMR7200, Université de Strasbourg, 74 Route du Rhin, 67401 Illkirch, France</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Carlos Perez</span> - <span class="hlFld-Affiliation affiliation">Eli
Lilly Research Laboratories, Avenida de la Industria, 30, 28108 Alcobendas, Madrid, Spain</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Célien Jacquemard</span> - <span class="hlFld-Affiliation affiliation">Laboratoire
d’Innovation Thérapeutique, UMR7200, Université de Strasbourg, 74 Route du Rhin, 67401 Illkirch, France</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jérémy Desaphy</span> - <span class="hlFld-Affiliation affiliation">Lilly
Research Laboratories, Eli Lilly and Company, Lilly Corporate Center, Indianapolis, Indiana 46285, United States</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>Project coordination: E.K. and J.D. Design of protocol: E.K. and M.N.D. Molecule classification: M.N.D., C.P., and E.K. Implementation of protocol: M.N.D. Data preparation: M.N.D. and G.B. Data analysis: M.N.D., E.K., and C.J. Preparation of manuscript: M.N.D., E.K., C.J., C.P., and J.D. All authors have given approval to the final version of the manuscript.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br><p class="inlineNote">Other protein name abbreviations not in the section <a class="ref internalNav" href="#dl1" aria-label="Abbreviations Used">Abbreviations Used</a> are given in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>.</p></div></li></ul></div><div class="ack" id="ACK-d7e1053-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i27">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_61057" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_61057" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The authors thank the LRAP funding program. This work was supported by Eli Lilly and Company through the Lilly Research Award Program (LRAP).</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations Used</td></tr><tr><td class="NLM_term">CAM</td><td class="NLM_def"><p class="first last">camphor</p></td></tr><tr><td class="NLM_term">DMSO</td><td class="NLM_def"><p class="first last">dimethyl sulfoxide</p></td></tr><tr><td class="NLM_term">FBDD</td><td class="NLM_def"><p class="first last">fragment-based drug design</p></td></tr><tr><td class="NLM_term">IBM</td><td class="NLM_def"><p class="first last">3-isobutyl-1-methylxanthine</p></td></tr><tr><td class="NLM_term">IFP</td><td class="NLM_def"><p class="first last">interaction fingerprint</p></td></tr><tr><td class="NLM_term">MES</td><td class="NLM_def"><p class="first last">2-(<i>N</i>-morpholino)ethanesulfonic acid</p></td></tr><tr><td class="NLM_term">MW</td><td class="NLM_def"><p class="first last">molecular weight</p></td></tr><tr><td class="NLM_term">MYI</td><td class="NLM_def"><p class="first last">5-methoxyindole acetate</p></td></tr><tr><td class="NLM_term">PDB</td><td class="NLM_def"><p class="first last">Protein Data Bank</p></td></tr><tr><td class="NLM_term">PPARγ</td><td class="NLM_def"><p class="first last">human peroxisome proliferator-activated protein γ</p></td></tr><tr><td class="NLM_term">rmsd</td><td class="NLM_def"><p class="first last">root-mean-square deviation</p></td></tr><tr><td class="NLM_term">TGFBR1</td><td class="NLM_def"><p class="first last">transforming growth factor β receptor type 1</p></td></tr><tr><td class="NLM_term">Tris</td><td class="NLM_def"><p class="first last">tris(hydroxymethyl)aminomethane</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i29">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_83206" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_83206" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 33 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Doak, B. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Norton, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scanlon, M. J.</span></span> <span> </span><span class="NLM_article-title">The Ways and Means of Fragment-Based Drug Design</span>. <i>Pharmacol. Ther.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>167</i></span>,  <span class="NLM_fpage">28</span>– <span class="NLM_lpage">37</span>, <span class="refDoi"> DOI: 10.1016/j.pharmthera.2016.07.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00256&amp;key=10.1016%2Fj.pharmthera.2016.07.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00256&amp;key=27452339" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00256&amp;key=1%3ACAS%3A528%3ADC%252BC28Xht1eku77E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=167&publication_year=2016&pages=28-37&author=B.+C.+Doakauthor=R.+S.+Nortonauthor=M.+J.+Scanlon&title=The+Ways+and+Means+of+Fragment-Based+Drug+Design&doi=10.1016%2Fj.pharmthera.2016.07.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">The ways and means of fragment-based drug design</span></div><div class="casAuthors">Doak, Bradley C.; Norton, Raymond S.; Scanlon, Martin J.</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacology & Therapeutics</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">167</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">28-37</span>CODEN:
                <span class="NLM_cas:coden">PHTHDT</span>;
        ISSN:<span class="NLM_cas:issn">0163-7258</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">Fragment-based drug design (FBDD) has emerged as a mainstream approach for the rapid and efficient identification of building blocks that can be used to develop high-affinity ligands against protein targets.  One of the strengths of FBDD is the relative ease and low cost of the primary screen to identify fragments that bind.  However, the fragments that emerge from primary screens often have low affinities, with KD values in the high μM to mM range, and a significant challenge for FBDD is to develop the initial fragments into more potent ligands.  Successful fragment elaboration often requires co-structures of the fragments bound to their target proteins, as well as a range of biophys. and biochem. assays to track potency and efficacy.  These challenges have led to the development of specific chem. strategies for the elaboration of weakly-binding fragments into more potent "hits" and lead compds.  In this article we review different approaches that have been employed to meet these challenges and describe some of the strategies that have resulted in several fragment-derived compds. entering clin. trials.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGohQqashtjg-rVg90H21EOLACvtfcHk0ljcfnBuV8eruA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xht1eku77E&md5=f2d1425d4f3870b57d2dd99994252ddc</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1016%2Fj.pharmthera.2016.07.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.pharmthera.2016.07.003%26sid%3Dliteratum%253Aachs%26aulast%3DDoak%26aufirst%3DB.%2BC.%26aulast%3DNorton%26aufirst%3DR.%2BS.%26aulast%3DScanlon%26aufirst%3DM.%2BJ.%26atitle%3DThe%2520Ways%2520and%2520Means%2520of%2520Fragment-Based%2520Drug%2520Design%26jtitle%3DPharmacol.%2520Ther.%26date%3D2016%26volume%3D167%26spage%3D28%26epage%3D37%26doi%3D10.1016%2Fj.pharmthera.2016.07.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rees, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Congreve, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carr, R.</span></span> <span> </span><span class="NLM_article-title">Fragment-Based Lead Discovery</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>3</i></span> (<span class="NLM_issue">8</span>),  <span class="NLM_fpage">660</span>– <span class="NLM_lpage">672</span>, <span class="refDoi"> DOI: 10.1038/nrd1467</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00256&amp;key=10.1038%2Fnrd1467" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00256&amp;key=15286733" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00256&amp;key=1%3ACAS%3A528%3ADC%252BD2cXmtVOht7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2004&pages=660-672&issue=8&author=D.+C.+Reesauthor=M.+Congreveauthor=C.+W.+Murrayauthor=R.+Carr&title=Fragment-Based+Lead+Discovery&doi=10.1038%2Fnrd1467"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment-based lead discovery</span></div><div class="casAuthors">Rees, David C.; Congreve, Miles; Murray, Christopher W.; Carr, Robin</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">660-672</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Fragment-based lead discovery is gaining momentum in both large pharmaceutical companies and biotechnol. labs. as a complementary approach to traditional screening.  This is because fragment-based approaches require significantly fewer compds. to be screened and synthesized, and are showing a high success rate in generating chem. series with lead-like properties.  Compared with traditional screening hits, the starting fragments have considerably lower mol. mass, and although the binding interactions of these fragments with a target protein are weak, they are structurally understood through x-ray crystallog. or NMR, and they exhibit high 'ligand efficiency'.  Here, the authors use examples from 25 different protein targets to describe chem. strategies that exploit this structural knowledge to rapidly develop fragments into high-affinity leads.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrbKeUzFFIlhbVg90H21EOLACvtfcHk0lh4TTZWHPlIyw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXmtVOht7Y%253D&md5=8d49ac7f698c9b9d573ee3c3485eced8</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1038%2Fnrd1467&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd1467%26sid%3Dliteratum%253Aachs%26aulast%3DRees%26aufirst%3DD.%2BC.%26aulast%3DCongreve%26aufirst%3DM.%26aulast%3DMurray%26aufirst%3DC.%2BW.%26aulast%3DCarr%26aufirst%3DR.%26atitle%3DFragment-Based%2520Lead%2520Discovery%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2004%26volume%3D3%26issue%3D8%26spage%3D660%26epage%3D672%26doi%3D10.1038%2Fnrd1467" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kalliokoski, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olsson, T. S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vulpetti, A.</span></span> <span> </span><span class="NLM_article-title">Subpocket Analysis Method for Fragment-Based Drug Discovery</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>53</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">131</span>– <span class="NLM_lpage">141</span>, <span class="refDoi"> DOI: 10.1021/ci300523r</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci300523r" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00256&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtFWlu7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2013&pages=131-141&issue=1&author=T.+Kalliokoskiauthor=T.+S.+G.+Olssonauthor=A.+Vulpetti&title=Subpocket+Analysis+Method+for+Fragment-Based+Drug+Discovery&doi=10.1021%2Fci300523r"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Subpocket Analysis Method for Fragment-Based Drug Discovery</span></div><div class="casAuthors">Kalliokoski, Tuomo; Olsson, Tjelvar S. G.; Vulpetti, Anna</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">131-141</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Although two binding sites might be dissimilar overall, they might still bind the same fragments if they share suitable subpockets.  Information about shared subpockets can be therefore used in fragment-based drug design to suggest new fragments or to replace existing fragments within an already known compd.  A novel computational method called SubCav is described which allows the similarity searching and alignment of subpockets from a PDB-wide database against a user-defined query.  The method is based on pharmacophoric fingerprints combined with a subpocket alignment algorithm.  SubCav was shown to be effective in producing reasonable alignments for subpockets with low sequence similarity and be able to retrieve relevant subpockets from a large database of structures including those with different folds.  It can also be used to analyze subpockets inside a protein family to facilitate drug design and to rationalize compd. selectivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp5ULmbpoNBtLVg90H21EOLACvtfcHk0lh4TTZWHPlIyw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtFWlu7c%253D&md5=49216111d0f500b6fefb715287fc315c</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1021%2Fci300523r&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci300523r%26sid%3Dliteratum%253Aachs%26aulast%3DKalliokoski%26aufirst%3DT.%26aulast%3DOlsson%26aufirst%3DT.%2BS.%2BG.%26aulast%3DVulpetti%26aufirst%3DA.%26atitle%3DSubpocket%2520Analysis%2520Method%2520for%2520Fragment-Based%2520Drug%2520Discovery%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2013%26volume%3D53%26issue%3D1%26spage%3D131%26epage%3D141%26doi%3D10.1021%2Fci300523r" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Carr, R. A.
E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Congreve, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rees, D. C.</span></span> <span> </span><span class="NLM_article-title">Fragment-Based Lead Discovery: Leads by Design</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>10</i></span> (<span class="NLM_issue">14</span>),  <span class="NLM_fpage">987</span>– <span class="NLM_lpage">992</span>, <span class="refDoi"> DOI: 10.1016/S1359-6446(05)03511-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00256&amp;key=10.1016%2FS1359-6446%2805%2903511-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00256&amp;key=16023057" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00256&amp;key=1%3ACAS%3A528%3ADC%252BD2MXmt1Whtr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2005&pages=987-992&issue=14&author=R.+A.%0AE.+Carrauthor=M.+Congreveauthor=C.+W.+Murrayauthor=D.+C.+Rees&title=Fragment-Based+Lead+Discovery%3A+Leads+by+Design&doi=10.1016%2FS1359-6446%2805%2903511-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment-based lead discovery: leads by design</span></div><div class="casAuthors">Carr, Robin A. E.; Congreve, Miles; Murray, Christopher W.; Rees, David C.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">987-992</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">A review.  Fragment-based lead discovery (also referred to as needles, shapes, binding elements, seed templates or scaffolds) is a new lead discovery approach in which much lower mol. wt. (120-250Da) compds. are screened relative to HTS campaigns.  Fragment-based hits are typically weak inhibitors (10μM-mM), and therefore need to be screened at higher concn. using very sensitive biophys. detection techniques such as protein crystallog. and NMR as the primary screening techniques, rather than bioassays.  Compared with HTS hits, these fragments are simpler, less functionalized compds. with correspondingly lower affinity.  However, fragment hits typically possess high ligand efficiency' (binding affinity per heavy atom) and so are highly suitable for optimization into clin. candidates with good drug-like properties.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq15FNhjT6Qw7Vg90H21EOLACvtfcHk0lhEkN21dV_FcA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXmt1Whtr0%253D&md5=1bf7dd97ec056af9ed5143f96b34bb53</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1016%2FS1359-6446%2805%2903511-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1359-6446%252805%252903511-7%26sid%3Dliteratum%253Aachs%26aulast%3DCarr%26aufirst%3DR.%2BA.%2BE.%26aulast%3DCongreve%26aufirst%3DM.%26aulast%3DMurray%26aufirst%3DC.%2BW.%26aulast%3DRees%26aufirst%3DD.%2BC.%26atitle%3DFragment-Based%2520Lead%2520Discovery%253A%2520Leads%2520by%2520Design%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2005%26volume%3D10%26issue%3D14%26spage%3D987%26epage%3D992%26doi%3D10.1016%2FS1359-6446%2805%2903511-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hajduk, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greer, J.</span></span> <span> </span><span class="NLM_article-title">A Decade of Fragment-Based Drug Design: Strategic Advances and Lessons Learned</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>6</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">211</span>– <span class="NLM_lpage">219</span>, <span class="refDoi"> DOI: 10.1038/nrd2220</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00256&amp;key=10.1038%2Fnrd2220" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00256&amp;key=17290284" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00256&amp;key=1%3ACAS%3A528%3ADC%252BD2sXit12rtr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2007&pages=211-219&issue=3&author=P.+J.+Hajdukauthor=J.+Greer&title=A+Decade+of+Fragment-Based+Drug+Design%3A+Strategic+Advances+and+Lessons+Learned&doi=10.1038%2Fnrd2220"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">A decade of fragment-based drug design: strategic advances and lessons learned</span></div><div class="casAuthors">Hajduk, Philip J.; Greer, Jonathan</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">211-219</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Since the early 1990s, several technol. and scientific advances - such as combinatorial chem., high-throughput screening and the sequencing of the human genome - have been heralded as remedies to the problems facing the pharmaceutical industry.  The use of these technologies in some form is now well established at most pharmaceutical companies; however, the return on investment in terms of marketed products has not met expectations.  Fragment-based drug design is another tool for drug discovery that has emerged in the past decade.  Here, we describe the development and evolution of fragment-based drug design, analyze the role that this approach can have in combination with other discovery technologies and highlight the impact that fragment-based methods have made in progressing new medicines into the clinic.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpOB8K3AESqTbVg90H21EOLACvtfcHk0lhEkN21dV_FcA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXit12rtr4%253D&md5=bae574da3671475d8488615c3bfa9849</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1038%2Fnrd2220&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd2220%26sid%3Dliteratum%253Aachs%26aulast%3DHajduk%26aufirst%3DP.%2BJ.%26aulast%3DGreer%26aufirst%3DJ.%26atitle%3DA%2520Decade%2520of%2520Fragment-Based%2520Drug%2520Design%253A%2520Strategic%2520Advances%2520and%2520Lessons%2520Learned%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2007%26volume%3D6%26issue%3D3%26spage%3D211%26epage%3D219%26doi%3D10.1038%2Fnrd2220" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Price, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Howard, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cons, B. D.</span></span> <span> </span><span class="NLM_article-title">Fragment-Based Drug Discovery and Its Application to Challenging Drug Targets</span>. <i>Essays Biochem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>61</i></span> (<span class="NLM_issue">5</span>),  <span class="NLM_fpage">475</span>– <span class="NLM_lpage">484</span>, <span class="refDoi"> DOI: 10.1042/EBC20170029</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00256&amp;key=10.1042%2FEBC20170029" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00256&amp;key=29118094" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00256&amp;key=1%3ACAS%3A280%3ADC%252BC1M7pt1GmtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2017&pages=475-484&issue=5&author=A.+J.+Priceauthor=S.+Howardauthor=B.+D.+Cons&title=Fragment-Based+Drug+Discovery+and+Its+Application+to+Challenging+Drug+Targets&doi=10.1042%2FEBC20170029"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment-based drug discovery and its application to challenging drug targets</span></div><div class="casAuthors">Price Amanda J; Howard Steven; Cons Benjamin D</div><div class="citationInfo"><span class="NLM_cas:title">Essays in biochemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">475-484</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Fragment-based drug discovery (FBDD) is a technique for identifying low molecular weight chemical starting points for drug discovery.  Since its inception 20 years ago, FBDD has grown in popularity to the point where it is now an established technique in industry and academia.  The approach involves the biophysical screening of proteins against collections of low molecular weight compounds (fragments).  Although fragments bind to proteins with relatively low affinity, they form efficient, high quality binding interactions with the protein architecture as they have to overcome a significant entropy barrier to bind.  Of the biophysical methods available for fragment screening, X-ray protein crystallography is one of the most sensitive and least prone to false positives.  It also provides detailed structural information of the protein-fragment complex at the atomic level.  Fragment-based screening using X-ray crystallography is therefore an efficient method for identifying binding hotspots on proteins, which can then be exploited by chemists and biologists for the discovery of new drugs.  The use of FBDD is illustrated here with a recently published case study of a drug discovery programme targeting the challenging protein-protein interaction Kelch-like ECH-associated protein 1:nuclear factor erythroid 2-related factor 2.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTqZGQ3bfUY0VHFYivNCdPRfW6udTcc2eaxuqnZsNeHvbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1M7pt1GmtA%253D%253D&md5=f8572516bfb1ecc4a0ef57b0c5ba1bc0</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1042%2FEBC20170029&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252FEBC20170029%26sid%3Dliteratum%253Aachs%26aulast%3DPrice%26aufirst%3DA.%2BJ.%26aulast%3DHoward%26aufirst%3DS.%26aulast%3DCons%26aufirst%3DB.%2BD.%26atitle%3DFragment-Based%2520Drug%2520Discovery%2520and%2520Its%2520Application%2520to%2520Challenging%2520Drug%2520Targets%26jtitle%3DEssays%2520Biochem.%26date%3D2017%26volume%3D61%26issue%3D5%26spage%3D475%26epage%3D484%26doi%3D10.1042%2FEBC20170029" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kozakov, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hall, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jehle, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ochiana, S. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, E. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pollastri, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allen, K. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whitty, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vajda, S.</span></span> <span> </span><span class="NLM_article-title">Ligand Deconstruction: Why Some Fragment Binding Positions Are Conserved and Others Are Not</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>112</i></span> (<span class="NLM_issue">20</span>),  <span class="NLM_fpage">E2585</span>– <span class="NLM_lpage">E2594</span>, <span class="refDoi"> DOI: 10.1073/pnas.1501567112</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00256&amp;key=10.1073%2Fpnas.1501567112" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=112&publication_year=2015&pages=E2585-E2594&issue=20&author=D.+Kozakovauthor=D.+R.+Hallauthor=S.+Jehleauthor=L.+Luoauthor=S.+O.+Ochianaauthor=E.+V.+Jonesauthor=M.+Pollastriauthor=K.+N.+Allenauthor=A.+Whittyauthor=S.+Vajda&title=Ligand+Deconstruction%3A+Why+Some+Fragment+Binding+Positions+Are+Conserved+and+Others+Are+Not&doi=10.1073%2Fpnas.1501567112"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1501567112&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1501567112%26sid%3Dliteratum%253Aachs%26aulast%3DKozakov%26aufirst%3DD.%26aulast%3DHall%26aufirst%3DD.%2BR.%26aulast%3DJehle%26aufirst%3DS.%26aulast%3DLuo%26aufirst%3DL.%26aulast%3DOchiana%26aufirst%3DS.%2BO.%26aulast%3DJones%26aufirst%3DE.%2BV.%26aulast%3DPollastri%26aufirst%3DM.%26aulast%3DAllen%26aufirst%3DK.%2BN.%26aulast%3DWhitty%26aufirst%3DA.%26aulast%3DVajda%26aufirst%3DS.%26atitle%3DLigand%2520Deconstruction%253A%2520Why%2520Some%2520Fragment%2520Binding%2520Positions%2520Are%2520Conserved%2520and%2520Others%2520Are%2520Not%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2015%26volume%3D112%26issue%3D20%26spage%3DE2585%26epage%3DE2594%26doi%3D10.1073%2Fpnas.1501567112" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rathi, P. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ludlow, R. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hall, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mortenson, P. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verdonk, M. L.</span></span> <span> </span><span class="NLM_article-title">Predicting “Hot” and “Warm” Spots for Fragment Binding</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span> (<span class="NLM_issue">9</span>),  <span class="NLM_fpage">4036</span>– <span class="NLM_lpage">4046</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b00366</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b00366" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=4036-4046&issue=9&author=P.+C.+Rathiauthor=R.+F.+Ludlowauthor=R.+J.+Hallauthor=C.+W.+Murrayauthor=P.+N.+Mortensonauthor=M.+L.+Verdonk&title=Predicting+%E2%80%9CHot%E2%80%9D+and+%E2%80%9CWarm%E2%80%9D+Spots+for+Fragment+Binding&doi=10.1021%2Facs.jmedchem.7b00366"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b00366%26sid%3Dliteratum%253Aachs%26aulast%3DRathi%26aufirst%3DP.%2BC.%26aulast%3DLudlow%26aufirst%3DR.%2BF.%26aulast%3DHall%26aufirst%3DR.%2BJ.%26aulast%3DMurray%26aufirst%3DC.%2BW.%26aulast%3DMortenson%26aufirst%3DP.%2BN.%26aulast%3DVerdonk%26aufirst%3DM.%2BL.%26atitle%3DPredicting%2520%25E2%2580%259CHot%25E2%2580%259D%2520and%2520%25E2%2580%259CWarm%25E2%2580%259D%2520Spots%2520for%2520Fragment%2520Binding%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26issue%3D9%26spage%3D4036%26epage%3D4046%26doi%3D10.1021%2Facs.jmedchem.7b00366" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Berman, H. M.</span></span> <span> </span><span class="NLM_article-title">The Protein Data Bank</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>28</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">235</span>– <span class="NLM_lpage">242</span>, <span class="refDoi"> DOI: 10.1093/nar/28.1.235</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00256&amp;key=10.1093%2Fnar%2F28.1.235" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00256&amp;key=10592235" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00256&amp;key=1%3ACAS%3A528%3ADC%252BD3cXhvVKjt7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2000&pages=235-242&issue=1&author=H.+M.+Berman&title=The+Protein+Data+Bank&doi=10.1093%2Fnar%2F28.1.235"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">The Protein Data Bank</span></div><div class="casAuthors">Berman, Helen M.; Westbrook, John; Feng, Zukang; Gilliland, Gary; Bhat, T. N.; Weissig, Helge; Shindyalov, Ilya N.; Bourne, Philip E.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">235-242</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The Protein Data Bank (PDB; http://www.rcsb.org/pdb/)is the single worldwide archive of structural data of biol. macromols.  This paper describes the goals of the PDB, the systems in place for data deposition and access, how to obtain further information, and near-term plans for the future development of the resource.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpStGCxT8QzirVg90H21EOLACvtfcHk0liZgzWryamCBQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXhvVKjt7w%253D&md5=227fb393f754be2be375ab727bfd05dc</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1093%2Fnar%2F28.1.235&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252F28.1.235%26sid%3Dliteratum%253Aachs%26aulast%3DBerman%26aufirst%3DH.%2BM.%26atitle%3DThe%2520Protein%2520Data%2520Bank%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2000%26volume%3D28%26issue%3D1%26spage%3D235%26epage%3D242%26doi%3D10.1093%2Fnar%2F28.1.235" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Meyder, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nittinger, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lange, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klein, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rarey, M.</span></span> <span> </span><span class="NLM_article-title">Estimating Electron Density Support for Individual Atoms and Molecular Fragments in X-ray Structures</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>57</i></span> (<span class="NLM_issue">10</span>),  <span class="NLM_fpage">2437</span>– <span class="NLM_lpage">2447</span>, <span class="refDoi"> DOI: 10.1021/acs.jcim.7b00391</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.7b00391" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00256&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhs1Shs7rN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2017&pages=2437-2447&issue=10&author=A.+Meyderauthor=E.+Nittingerauthor=G.+Langeauthor=R.+Kleinauthor=M.+Rarey&title=Estimating+Electron+Density+Support+for+Individual+Atoms+and+Molecular+Fragments+in+X-ray+Structures&doi=10.1021%2Facs.jcim.7b00391"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Estimating Electron Density Support for Individual Atoms and Molecular Fragments in X-ray Structures</span></div><div class="casAuthors">Meyder, Agnes; Nittinger, Eva; Lange, Gudrun; Klein, Robert; Rarey, Matthias</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2437-2447</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Macromol. structures resolved by x-ray crystallog. are essential for life science research.  While some methods exist to automatically quantify the quality of the electron d. fit, none of them is without flaws.  Esp. the question of how well individual parts like atoms, small fragments, or mols. are supported by electron d. is difficult to quantify.  While taking exptl. uncertainties correctly into account, they do not offer an answer on how reliable an individual atom position is.  A rapid quantification of this at. position reliability would be highly valuable in structure-based mol. design.  To overcome this limitation, the authors introduce the electron d. score EDIA for individual atoms and mol. fragments.  EDIA assesses rapidly, automatically, and intuitively the fit of individual as well as multiple atoms (EDIAm) into electron d. accompanied by an integrated error anal.  The computation is based on the std. 2fo - fc electron d. map in combination with the model of the mol. structure.  For evaluating partial structures, EDIAm shows significant advantages compared to the real-space R correlation coeff. (RSCC) and the real-space difference d. Z score (RSZD) from the mol. modeler's point of view.  Thus, EDIA abolishes the time-consuming step of visually inspecting the electron d. during structure selection and curation.  It supports daily modeling tasks of medicinal and computational chemists and enables a fully automated assembly of large-scale, high-quality structure data sets.  Furthermore, EDIA scores can be applied for model validation and method development in computer-aided mol. design.  In contrast to measuring the deviation from the structure model by root-mean-squared deviation, EDIA scores allow comparison to the underlying exptl. data taking its uncertainty into account.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr9gkygdo2Is7Vg90H21EOLACvtfcHk0liZgzWryamCBQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhs1Shs7rN&md5=124b9ee097f58bb95b60412d9b730ca6</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.7b00391&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.7b00391%26sid%3Dliteratum%253Aachs%26aulast%3DMeyder%26aufirst%3DA.%26aulast%3DNittinger%26aufirst%3DE.%26aulast%3DLange%26aufirst%3DG.%26aulast%3DKlein%26aufirst%3DR.%26aulast%3DRarey%26aufirst%3DM.%26atitle%3DEstimating%2520Electron%2520Density%2520Support%2520for%2520Individual%2520Atoms%2520and%2520Molecular%2520Fragments%2520in%2520X-ray%2520Structures%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2017%26volume%3D57%26issue%3D10%26spage%3D2437%26epage%3D2447%26doi%3D10.1021%2Facs.jcim.7b00391" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lingel, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sendzik, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shultz, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cantwell, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dillon, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fuller, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gabriel, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McKenna, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rao, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sutton, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taft, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindvall, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bussiere, D. E.</span></span> <span> </span><span class="NLM_article-title">Structure-Guided Design of EED Binders Allosterically Inhibiting the Epigenetic Polycomb Repressive Complex 2 (PRC2) Methyltransferase</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">415</span>– <span class="NLM_lpage">427</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b01473</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01473" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00256&amp;key=1%3ACAS%3A528%3ADC%252BC28XitFaiurrF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=415-427&issue=1&author=A.+Lingelauthor=M.+Sendzikauthor=Y.+Huangauthor=M.+D.+Shultzauthor=J.+Cantwellauthor=M.+P.+Dillonauthor=X.+Fuauthor=J.+Fullerauthor=T.+Gabrielauthor=J.+Guauthor=X.+Jiangauthor=L.+Liauthor=F.+Liangauthor=M.+McKennaauthor=W.+Qiauthor=W.+Raoauthor=X.+Shengauthor=W.+Shuauthor=J.+Suttonauthor=B.+Taftauthor=L.+Wangauthor=J.+Zengauthor=H.+Zhangauthor=M.+Zhangauthor=K.+Zhaoauthor=M.+Lindvallauthor=D.+E.+Bussiere&title=Structure-Guided+Design+of+EED+Binders+Allosterically+Inhibiting+the+Epigenetic+Polycomb+Repressive+Complex+2+%28PRC2%29+Methyltransferase&doi=10.1021%2Facs.jmedchem.6b01473"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Guided Design of EED Binders Allosterically Inhibiting the Epigenetic Polycomb Repressive Complex 2 (PRC2) Methyltransferase</span></div><div class="casAuthors">Lingel, Andreas; Sendzik, Martin; Huang, Ying; Shultz, Michael D.; Cantwell, John; Dillon, Michael P.; Fu, Xingnian; Fuller, John; Gabriel, Tobias; Gu, Justin; Jiang, Xiangqing; Li, Ling; Liang, Fang; McKenna, Maureen; Qi, Wei; Rao, Weijun; Sheng, Xijun; Shu, Wei; Sutton, James; Taft, Benjamin; Wang, Long; Zeng, Jue; Zhang, Hailong; Zhang, Maya; Zhao, Kehao; Lindvall, Mika; Bussiere, Dirksen E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">415-427</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">PRC2 is a multisubunit methyltransferase involved in epigenetic regulation of early embryonic development and cell growth.  The catalytic subunit EZH2 methylates primarily lysine 27 of histone H3, leading to chromatin compaction and repression of tumor suppressor genes.  Inhibiting this activity by small mols. targeting EZH2 was shown to result in antitumor efficacy.  Here, the authors describe for the first time the optimization of a chem. series representing a new class of PRC2 inhibitors which acts allosterically via the trimethyllysine pocket of the noncatalytic EED subunit.  Deconstruction of a larger and complex screening hit to a simple fragment-sized mol. followed by structure-guided regrowth and careful property modulation were employed to yield compds. which achieve sub-micromolar inhibition in functional assays and cellular activity.  The resulting mols. can serve as a simplified entry point for lead optimization and can be utilized to study this new mechanism of PRC2 inhibition and the assocd. biol. in detail.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpRo2rkeTDLibVg90H21EOLACvtfcHk0liZgzWryamCBQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitFaiurrF&md5=3aec108471439f4ac99728a57b4eaf8e</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01473&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01473%26sid%3Dliteratum%253Aachs%26aulast%3DLingel%26aufirst%3DA.%26aulast%3DSendzik%26aufirst%3DM.%26aulast%3DHuang%26aufirst%3DY.%26aulast%3DShultz%26aufirst%3DM.%2BD.%26aulast%3DCantwell%26aufirst%3DJ.%26aulast%3DDillon%26aufirst%3DM.%2BP.%26aulast%3DFu%26aufirst%3DX.%26aulast%3DFuller%26aufirst%3DJ.%26aulast%3DGabriel%26aufirst%3DT.%26aulast%3DGu%26aufirst%3DJ.%26aulast%3DJiang%26aufirst%3DX.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DLiang%26aufirst%3DF.%26aulast%3DMcKenna%26aufirst%3DM.%26aulast%3DQi%26aufirst%3DW.%26aulast%3DRao%26aufirst%3DW.%26aulast%3DSheng%26aufirst%3DX.%26aulast%3DShu%26aufirst%3DW.%26aulast%3DSutton%26aufirst%3DJ.%26aulast%3DTaft%26aufirst%3DB.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DZeng%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DH.%26aulast%3DZhang%26aufirst%3DM.%26aulast%3DZhao%26aufirst%3DK.%26aulast%3DLindvall%26aufirst%3DM.%26aulast%3DBussiere%26aufirst%3DD.%2BE.%26atitle%3DStructure-Guided%2520Design%2520of%2520EED%2520Binders%2520Allosterically%2520Inhibiting%2520the%2520Epigenetic%2520Polycomb%2520Repressive%2520Complex%25202%2520%2528PRC2%2529%2520Methyltransferase%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26issue%3D1%26spage%3D415%26epage%3D427%26doi%3D10.1021%2Facs.jmedchem.6b01473" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Czodrowski, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hölzemann, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barnickel, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greiner, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Musil, D.</span></span> <span> </span><span class="NLM_article-title">Selection of Fragments for Kinase Inhibitor Design: Decoration Is Key</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">457</span>– <span class="NLM_lpage">465</span>, <span class="refDoi"> DOI: 10.1021/jm501597j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm501597j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=457-465&issue=1&author=P.+Czodrowskiauthor=G.+H%C3%B6lzemannauthor=G.+Barnickelauthor=H.+Greinerauthor=D.+Musil&title=Selection+of+Fragments+for+Kinase+Inhibitor+Design%3A+Decoration+Is+Key&doi=10.1021%2Fjm501597j"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1021%2Fjm501597j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm501597j%26sid%3Dliteratum%253Aachs%26aulast%3DCzodrowski%26aufirst%3DP.%26aulast%3DH%25C3%25B6lzemann%26aufirst%3DG.%26aulast%3DBarnickel%26aufirst%3DG.%26aulast%3DGreiner%26aufirst%3DH.%26aulast%3DMusil%26aufirst%3DD.%26atitle%3DSelection%2520of%2520Fragments%2520for%2520Kinase%2520Inhibitor%2520Design%253A%2520Decoration%2520Is%2520Key%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26issue%3D1%26spage%3D457%26epage%3D465%26doi%3D10.1021%2Fjm501597j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Drwal, M. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacquemard, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perez, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Desaphy, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kellenberger, E.</span></span> <span> </span><span class="NLM_article-title">Do Fragments and Crystallization Additives Bind Similarly to Drug-like Ligands?</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>57</i></span> (<span class="NLM_issue">5</span>),  <span class="NLM_fpage">1197</span>– <span class="NLM_lpage">1209</span>, <span class="refDoi"> DOI: 10.1021/acs.jcim.6b00769</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.6b00769" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00256&amp;key=1%3ACAS%3A528%3ADC%252BC2sXmtFCgt7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2017&pages=1197-1209&issue=5&author=M.+N.+Drwalauthor=C.+Jacquemardauthor=C.+Perezauthor=J.+Desaphyauthor=E.+Kellenberger&title=Do+Fragments+and+Crystallization+Additives+Bind+Similarly+to+Drug-like+Ligands%3F&doi=10.1021%2Facs.jcim.6b00769"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Do Fragments and Crystallization Additives Bind Similarly to Drug-like Ligands?</span></div><div class="casAuthors">Drwal, Malgorzata N.; Jacquemard, Celien; Perez, Carlos; Desaphy, Jeremy; Kellenberger, Esther</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1197-1209</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The success of fragment-based drug design (FBDD) hinges upon the optimization of low-mol.-wt. compds. (MW < 300 Da) with weak binding affinities to lead compds. with high affinity and selectivity.  Usually, structural information from fragment-protein complexes is used to develop ideas about the binding mode of similar but drug-like mols.  In this regard, crystn. additives such as cryoprotectants or buffer components, which are highly abundant in crystal structures, are frequently ignored.  Thus, the aim of this study was to investigate the information present in protein complexes with fragments as well as those with additives and how they relate to the binding modes of their drug-like counterparts.  We present a thorough anal. of the binding modes of crystallog. additives, fragments, and drug-like ligands bound to four diverse targets of wide interest in drug discovery and highly represented in the Protein Data Bank: cyclin-dependent kinase 2, β-secretase 1, carbonic anhydrase 2, and trypsin.  We identified a total of 630 unique mols. bound to the catalytic binding sites, among them 31 additives, 222 fragments, and 377 drug-like ligands.  In general, we obsd. that, independent of the target, protein-fragment interaction patterns are highly similar to those of drug-like ligands and mostly cover the residues crucial for binding.  Crystallog. additives are also able to show conserved binding modes and recover the residues important for binding in some of the cases.  Moreover, we show evidence that the information from fragments and drug-like ligands can be applied to rescore docking poses in order to improve the prediction of binding modes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoVxqwFETp6zbVg90H21EOLACvtfcHk0lhN1KQI-g6XcQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXmtFCgt7k%253D&md5=56294da65813c14a2586719faca2da46</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.6b00769&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.6b00769%26sid%3Dliteratum%253Aachs%26aulast%3DDrwal%26aufirst%3DM.%2BN.%26aulast%3DJacquemard%26aufirst%3DC.%26aulast%3DPerez%26aufirst%3DC.%26aulast%3DDesaphy%26aufirst%3DJ.%26aulast%3DKellenberger%26aufirst%3DE.%26atitle%3DDo%2520Fragments%2520and%2520Crystallization%2520Additives%2520Bind%2520Similarly%2520to%2520Drug-like%2520Ligands%253F%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2017%26volume%3D57%26issue%3D5%26spage%3D1197%26epage%3D1209%26doi%3D10.1021%2Facs.jcim.6b00769" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Drwal, M. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bret, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kellenberger, E.</span></span> <span> </span><span class="NLM_article-title">Multi-Target Fragments Display Versatile Binding Modes</span>. <i>Mol. Inf.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>36</i></span> (<span class="NLM_issue">10</span>),  <span class="NLM_fpage">1700042</span>– <span class="NLM_lpage">1700042</span>, <span class="refDoi"> DOI: 10.1002/minf.201700042</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00256&amp;key=10.1002%2Fminf.201700042" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2017&pages=1700042-1700042&issue=10&author=M.+N.+Drwalauthor=G.+Bretauthor=E.+Kellenberger&title=Multi-Target+Fragments+Display+Versatile+Binding+Modes&doi=10.1002%2Fminf.201700042"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1002%2Fminf.201700042&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fminf.201700042%26sid%3Dliteratum%253Aachs%26aulast%3DDrwal%26aufirst%3DM.%2BN.%26aulast%3DBret%26aufirst%3DG.%26aulast%3DKellenberger%26aufirst%3DE.%26atitle%3DMulti-Target%2520Fragments%2520Display%2520Versatile%2520Binding%2520Modes%26jtitle%3DMol.%2520Inf.%26date%3D2017%26volume%3D36%26issue%3D10%26spage%3D1700042%26epage%3D1700042%26doi%3D10.1002%2Fminf.201700042" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kontopidis, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McInnes, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pandalaneni, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McNae, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gibson, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mezna, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wood, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walkinshaw, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fischer, P. M.</span></span> <span> </span><span class="NLM_article-title">Differential Binding of Inhibitors to Active and Inactive CDK2 Provides Insights for Drug Design</span>. <i>Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>13</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">201</span>– <span class="NLM_lpage">211</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2005.11.011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00256&amp;key=10.1016%2Fj.chembiol.2005.11.011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00256&amp;key=16492568" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00256&amp;key=1%3ACAS%3A528%3ADC%252BD28XhvVemsr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2006&pages=201-211&issue=2&author=G.+Kontopidisauthor=C.+McInnesauthor=S.+R.+Pandalaneniauthor=I.+McNaeauthor=D.+Gibsonauthor=M.+Meznaauthor=M.+Thomasauthor=G.+Woodauthor=S.+Wangauthor=M.+D.+Walkinshawauthor=P.+M.+Fischer&title=Differential+Binding+of+Inhibitors+to+Active+and+Inactive+CDK2+Provides+Insights+for+Drug+Design&doi=10.1016%2Fj.chembiol.2005.11.011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Differential Binding of Inhibitors to Active and Inactive CDK2 Provides Insights for Drug Design</span></div><div class="casAuthors">Kontopidis, George; McInnes, Campbell; Pandalaneni, Sravan R.; McNae, Iain; Gibson, Darren; Mezna, Mokdad; Thomas, Mark; Wood, Gavin; Wang, Shudong; Walkinshaw, Malcolm D.; Fischer, Peter M.</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">201-211</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The cyclin-dependent kinases (CDKs) have been characterized in complex with a variety of inhibitors, but the majority of structures solved are in the inactive form.  We have solved the structures of six inhibitors in both the monomeric CDK2 and binary CDK2/cyclinA complexes and demonstrate that significant differences in ligand binding occur depending on the activation state.  The binding mode of two ligands in particular varies substantially in active and inactive CDK2.  Furthermore, energetic anal. of CDK2/cyclin/inhibitors demonstrates that a good correlation exists between the in vitro potency and the calcd. energies of interaction, but no such relationship exists for CDK2/inhibitor structures.  These results confirm that monomeric CDK2 ligand complexes do not fully reflect active conformations, revealing significant implications for inhibitor design while also suggesting that the monomeric CDK2 conformation can be selectively inhibited.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpjllZrMBoaHbVg90H21EOLACvtfcHk0lhN1KQI-g6XcQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhvVemsr8%253D&md5=3f5dce248d66e7478a6f6661a47ca388</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2005.11.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2005.11.011%26sid%3Dliteratum%253Aachs%26aulast%3DKontopidis%26aufirst%3DG.%26aulast%3DMcInnes%26aufirst%3DC.%26aulast%3DPandalaneni%26aufirst%3DS.%2BR.%26aulast%3DMcNae%26aufirst%3DI.%26aulast%3DGibson%26aufirst%3DD.%26aulast%3DMezna%26aufirst%3DM.%26aulast%3DThomas%26aufirst%3DM.%26aulast%3DWood%26aufirst%3DG.%26aulast%3DWang%26aufirst%3DS.%26aulast%3DWalkinshaw%26aufirst%3DM.%2BD.%26aulast%3DFischer%26aufirst%3DP.%2BM.%26atitle%3DDifferential%2520Binding%2520of%2520Inhibitors%2520to%2520Active%2520and%2520Inactive%2520CDK2%2520Provides%2520Insights%2520for%2520Drug%2520Design%26jtitle%3DChem.%2520Biol.%26date%3D2006%26volume%3D13%26issue%3D2%26spage%3D201%26epage%3D211%26doi%3D10.1016%2Fj.chembiol.2005.11.011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tarcsay, Á.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nyíri, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keserű, G. M.</span></span> <span> </span><span class="NLM_article-title">Impact of Lipophilic Efficiency on Compound Quality</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>55</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">1252</span>– <span class="NLM_lpage">1260</span>, <span class="refDoi"> DOI: 10.1021/jm201388p</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm201388p" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=1252-1260&issue=3&author=%C3%81.+Tarcsayauthor=K.+Ny%C3%ADriauthor=G.+M.+Keser%C5%B1&title=Impact+of+Lipophilic+Efficiency+on+Compound+Quality&doi=10.1021%2Fjm201388p"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1021%2Fjm201388p&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm201388p%26sid%3Dliteratum%253Aachs%26aulast%3DTarcsay%26aufirst%3D%25C3%2581.%26aulast%3DNy%25C3%25ADri%26aufirst%3DK.%26aulast%3DKeser%25C5%25B1%26aufirst%3DG.%2BM.%26atitle%3DImpact%2520of%2520Lipophilic%2520Efficiency%2520on%2520Compound%2520Quality%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26issue%3D3%26spage%3D1252%26epage%3D1260%26doi%3D10.1021%2Fjm201388p" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Smith, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dougan, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Komandla, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kanouni, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knight, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lawson, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sabat, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, E. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vu, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wyrick, C.</span></span> <span> </span><span class="NLM_article-title">Fragment-Based Discovery of a Small Molecule Inhibitor of Bruton’s Tyrosine Kinase</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span> (<span class="NLM_issue">14</span>),  <span class="NLM_fpage">5437</span>– <span class="NLM_lpage">5444</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b00734</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b00734" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=5437-5444&issue=14&author=C.+R.+Smithauthor=D.+R.+Douganauthor=M.+Komandlaauthor=T.+Kanouniauthor=B.+Knightauthor=J.+D.+Lawsonauthor=M.+Sabatauthor=E.+R.+Taylorauthor=P.+Vuauthor=C.+Wyrick&title=Fragment-Based+Discovery+of+a+Small+Molecule+Inhibitor+of+Bruton%E2%80%99s+Tyrosine+Kinase&doi=10.1021%2Facs.jmedchem.5b00734"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b00734&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b00734%26sid%3Dliteratum%253Aachs%26aulast%3DSmith%26aufirst%3DC.%2BR.%26aulast%3DDougan%26aufirst%3DD.%2BR.%26aulast%3DKomandla%26aufirst%3DM.%26aulast%3DKanouni%26aufirst%3DT.%26aulast%3DKnight%26aufirst%3DB.%26aulast%3DLawson%26aufirst%3DJ.%2BD.%26aulast%3DSabat%26aufirst%3DM.%26aulast%3DTaylor%26aufirst%3DE.%2BR.%26aulast%3DVu%26aufirst%3DP.%26aulast%3DWyrick%26aufirst%3DC.%26atitle%3DFragment-Based%2520Discovery%2520of%2520a%2520Small%2520Molecule%2520Inhibitor%2520of%2520Bruton%25E2%2580%2599s%2520Tyrosine%2520Kinase%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26issue%3D14%26spage%3D5437%26epage%3D5444%26doi%3D10.1021%2Facs.jmedchem.5b00734" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Malhotra, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karanicolas, J.</span></span> <span> </span><span class="NLM_article-title">When Does Chemical Elaboration Induce a Ligand To Change Its Binding Mode?</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">128</span>– <span class="NLM_lpage">145</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b00725</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b00725" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=128-145&issue=1&author=S.+Malhotraauthor=J.+Karanicolas&title=When+Does+Chemical+Elaboration+Induce+a+Ligand+To+Change+Its+Binding+Mode%3F&doi=10.1021%2Facs.jmedchem.6b00725"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b00725&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b00725%26sid%3Dliteratum%253Aachs%26aulast%3DMalhotra%26aufirst%3DS.%26aulast%3DKaranicolas%26aufirst%3DJ.%26atitle%3DWhen%2520Does%2520Chemical%2520Elaboration%2520Induce%2520a%2520Ligand%2520To%2520Change%2520Its%2520Binding%2520Mode%253F%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26issue%3D1%26spage%3D128%26epage%3D145%26doi%3D10.1021%2Facs.jmedchem.6b00725" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Congreve, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carr, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jhoti, H.</span></span> <span> </span><span class="NLM_article-title">A “Rule of Three” for Fragment-Based Lead Discovery?</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>8</i></span> (<span class="NLM_issue">19</span>),  <span class="NLM_fpage">876</span>– <span class="NLM_lpage">877</span>, <span class="refDoi"> DOI: 10.1016/S1359-6446(03)02831-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00256&amp;key=10.1016%2FS1359-6446%2803%2902831-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00256&amp;key=14554012" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00256&amp;key=1%3ACAS%3A280%3ADC%252BD3svns1Cnsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2003&pages=876-877&issue=19&author=M.+Congreveauthor=R.+Carrauthor=C.+Murrayauthor=H.+Jhoti&title=A+%E2%80%9CRule+of+Three%E2%80%9D+for+Fragment-Based+Lead+Discovery%3F&doi=10.1016%2FS1359-6446%2803%2902831-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">A 'rule of three' for fragment-based lead discovery?</span></div><div class="casAuthors">Congreve Miles; Carr Robin; Murray Chris; Jhoti Harren</div><div class="citationInfo"><span class="NLM_cas:title">Drug discovery today</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">876-7</span>
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRfFswFSn5vb43yKPtYViAGfW6udTcc2eZyXtMwdNwnk7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD3svns1Cnsw%253D%253D&md5=137886dbb2062f7a3ad39277dd888df3</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1016%2FS1359-6446%2803%2902831-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1359-6446%252803%252902831-9%26sid%3Dliteratum%253Aachs%26aulast%3DCongreve%26aufirst%3DM.%26aulast%3DCarr%26aufirst%3DR.%26aulast%3DMurray%26aufirst%3DC.%26aulast%3DJhoti%26aufirst%3DH.%26atitle%3DA%2520%25E2%2580%259CRule%2520of%2520Three%25E2%2580%259D%2520for%2520Fragment-Based%2520Lead%2520Discovery%253F%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2003%26volume%3D8%26issue%3D19%26spage%3D876%26epage%3D877%26doi%3D10.1016%2FS1359-6446%2803%2902831-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schauperl, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Czodrowski, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fuchs, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huber, R. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waldner, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Podewitz, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kramer, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liedl, K. R.</span></span> <span> </span><span class="NLM_article-title">Binding Pose Flip Explained via Enthalpic and Entropic Contributions</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>57</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">345</span>– <span class="NLM_lpage">354</span>, <span class="refDoi"> DOI: 10.1021/acs.jcim.6b00483</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.6b00483" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00256&amp;key=1%3ACAS%3A528%3ADC%252BC2sXosF2huw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2017&pages=345-354&issue=2&author=M.+Schauperlauthor=P.+Czodrowskiauthor=J.+E.+Fuchsauthor=R.+G.+Huberauthor=B.+J.+Waldnerauthor=M.+Podewitzauthor=C.+Kramerauthor=K.+R.+Liedl&title=Binding+Pose+Flip+Explained+via+Enthalpic+and+Entropic+Contributions&doi=10.1021%2Facs.jcim.6b00483"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Binding Pose Flip Explained via Enthalpic and Entropic Contributions</span></div><div class="casAuthors">Schauperl, Michael; Czodrowski, Paul; Fuchs, Julian E.; Huber, Roland G.; Waldner, Birgit J.; Podewitz, Maren; Kramer, Christian; Liedl, Klaus R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">345-354</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The anomalous binding modes of 5 highly similar fragments of angiopoietin receptor TIE2 inhibitors, showing 3 distinct binding poses were investigated.  The authors report a quant. rationalization for the changes in binding pose based on mol. dynamics (MD) simulations.  The authors investigated 5 fragments in complex with the transforming growth factor-β receptor type 1 kinase domain.  Analyses of these MD simulations using Grid Inhomogeneous Solvation Theory (GIST), pKa calcns., and a tool to investigate enthalpic differences upon binding unraveled the various thermodn. contributions to the different binding modes.  While one binding mode flip could be rationalized by steric repulsion, the 2nd binding pose flip revealed a different protonation state for one of the ligands, leading to different enthalpic and entropic contributions to the binding free energy.  While one binding pose was stabilized by the displacement of entropically unfavored water mols. (binding pose detd. by solvation entropy), ligands in the other binding pose were stabilized by strong enthalpic interactions, overcompensating the unfavorable water entropy in this pose (binding pose detd. by enthalpic interactions).  This anal. elucidated unprecedented details detg. the flipping of the binding modes, which can elegantly explain the exptl. findings for this system.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpyksMbMzYXP7Vg90H21EOLACvtfcHk0limx5eRUAgiYg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXosF2huw%253D%253D&md5=9fa0737411901c4d27ae9954dca37ecf</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.6b00483&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.6b00483%26sid%3Dliteratum%253Aachs%26aulast%3DSchauperl%26aufirst%3DM.%26aulast%3DCzodrowski%26aufirst%3DP.%26aulast%3DFuchs%26aufirst%3DJ.%2BE.%26aulast%3DHuber%26aufirst%3DR.%2BG.%26aulast%3DWaldner%26aufirst%3DB.%2BJ.%26aulast%3DPodewitz%26aufirst%3DM.%26aulast%3DKramer%26aufirst%3DC.%26aulast%3DLiedl%26aufirst%3DK.%2BR.%26atitle%3DBinding%2520Pose%2520Flip%2520Explained%2520via%2520Enthalpic%2520and%2520Entropic%2520Contributions%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2017%26volume%3D57%26issue%3D2%26spage%3D345%26epage%3D354%26doi%3D10.1021%2Facs.jcim.6b00483" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kirkwood, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hargreaves, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Keefe, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, J.</span></span> <span> </span><span class="NLM_article-title">Analysis of Crystallization Data in the Protein Data Bank</span>. <i>Acta Crystallogr., Sect. F: Struct. Biol. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>71</i></span> (<span class="NLM_issue">10</span>),  <span class="NLM_fpage">1228</span>– <span class="NLM_lpage">1234</span>, <span class="refDoi"> DOI: 10.1107/S2053230X15014892</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00256&amp;key=10.1107%2FS2053230X15014892" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00256&amp;key=26457511" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00256&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhs1eksLnL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=71&publication_year=2015&pages=1228-1234&issue=10&author=J.+Kirkwoodauthor=D.+Hargreavesauthor=S.+O%E2%80%99Keefeauthor=J.+Wilson&title=Analysis+of+Crystallization+Data+in+the+Protein+Data+Bank&doi=10.1107%2FS2053230X15014892"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Analysis of crystallization data in the Protein Data Bank</span></div><div class="casAuthors">Kirkwood, Jobie; Hargreaves, David; O'Keefe, Simon; Wilson, Julie</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section F: Structural Biology Communications</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">71</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1228-1234</span>CODEN:
                <span class="NLM_cas:coden">ACSFEN</span>;
        ISSN:<span class="NLM_cas:issn">2053-230X</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">The Protein Data Bank (PDB) is the largest available repository of solved protein structures and contains a wealth of information on successful crystn.  Many centers have used their own exptl. data to draw conclusions about proteins and the conditions in which they crystallize.  Here, data from the PDB were used to reanalyze some of these results.  The most successful crystn. reagents were identified, the link between soln. pH and the isoelec. point of the protein was investigated and the possibility of predicting whether a protein will crystallize was explored.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq5dWh2xg2hXLVg90H21EOLACvtfcHk0limx5eRUAgiYg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhs1eksLnL&md5=aad3de27009436f443e1f47e71a70099</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1107%2FS2053230X15014892&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS2053230X15014892%26sid%3Dliteratum%253Aachs%26aulast%3DKirkwood%26aufirst%3DJ.%26aulast%3DHargreaves%26aufirst%3DD.%26aulast%3DO%25E2%2580%2599Keefe%26aufirst%3DS.%26aulast%3DWilson%26aufirst%3DJ.%26atitle%3DAnalysis%2520of%2520Crystallization%2520Data%2520in%2520the%2520Protein%2520Data%2520Bank%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520F%253A%2520Struct.%2520Biol.%2520Commun.%26date%3D2015%26volume%3D71%26issue%3D10%26spage%3D1228%26epage%3D1234%26doi%3D10.1107%2FS2053230X15014892" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McPherson, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cudney, B.</span></span> <span> </span><span class="NLM_article-title">Searching for Silver Bullets: An Alternative Strategy for Crystallizing Macromolecules</span>. <i>J. Struct. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>156</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">387</span>– <span class="NLM_lpage">406</span>, <span class="refDoi"> DOI: 10.1016/j.jsb.2006.09.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00256&amp;key=10.1016%2Fj.jsb.2006.09.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00256&amp;key=17101277" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00256&amp;key=1%3ACAS%3A528%3ADC%252BD28Xht1CgsbzM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=156&publication_year=2006&pages=387-406&issue=3&author=A.+McPhersonauthor=B.+Cudney&title=Searching+for+Silver+Bullets%3A+An+Alternative+Strategy+for+Crystallizing+Macromolecules&doi=10.1016%2Fj.jsb.2006.09.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Searching for silver bullets: An alternative strategy for crystallizing macromolecules</span></div><div class="casAuthors">McPherson, Alexander; Cudney, Bob</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Structural Biology</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">156</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">387-406</span>CODEN:
                <span class="NLM_cas:coden">JSBIEM</span>;
        ISSN:<span class="NLM_cas:issn">1047-8477</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">Based on a hypothesis that various small mols. might establish stabilizing, intermol., non covalent crosslinks in protein crystals and thereby promote lattice formation, the authors carried out three sep. expts.  The authors assessed the impact of 200 chems. on the propensity of 81 different proteins and viruses to crystallize.  The expts. were comprised of 18 240 vapor diffusion trials.  A salient feature of the expts. was that, aside from the inclusion of the reagent mixes, only two fundamental crystn. conditions were used, 30% PEG 3350, and 50% Tacsimate at pH 7.  Overall, 65 proteins (85%) were crystd.  Most significant was that 35 of the 65 (54%) crystd. only in the presence of one or more reagent mixes, but not in control samples lacking any additives.  Among the most promising types of reagent mixes were those composed of polyvalent, charged groups, such as di and tri carboxylic acids, diamino compds., mols. bearing one or more sulfonyl or phosphate groups, and a broad range of common biochems., coenzymes, biol. effectors, and ligands.  The authors propose that an alternate approach to crystg. proteins might be developed, which employs a limited set of fundamental crystn. conditions combined with a broad screen of potentially useful small mol. additives.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGopSz9TqLH0_rVg90H21EOLACvtfcHk0limx5eRUAgiYg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xht1CgsbzM&md5=f64b256b085e6fff737f501dc028b740</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1016%2Fj.jsb.2006.09.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jsb.2006.09.006%26sid%3Dliteratum%253Aachs%26aulast%3DMcPherson%26aufirst%3DA.%26aulast%3DCudney%26aufirst%3DB.%26atitle%3DSearching%2520for%2520Silver%2520Bullets%253A%2520An%2520Alternative%2520Strategy%2520for%2520Crystallizing%2520Macromolecules%26jtitle%3DJ.%2520Struct.%2520Biol.%26date%3D2006%26volume%3D156%26issue%3D3%26spage%3D387%26epage%3D406%26doi%3D10.1016%2Fj.jsb.2006.09.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Larson, S. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Day, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cudney, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McPherson, A.</span></span> <span> </span><span class="NLM_article-title">A Novel Strategy for the Crystallization of Proteins: X-Ray Diffraction Validation</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>63</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">310</span>– <span class="NLM_lpage">318</span>, <span class="refDoi"> DOI: 10.1107/S0907444906053303</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00256&amp;key=10.1107%2FS0907444906053303" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2007&pages=310-318&issue=3&author=S.+B.+Larsonauthor=J.+S.+Dayauthor=R.+Cudneyauthor=A.+McPherson&title=A+Novel+Strategy+for+the+Crystallization+of+Proteins%3A+X-Ray+Diffraction+Validation&doi=10.1107%2FS0907444906053303"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1107%2FS0907444906053303&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444906053303%26sid%3Dliteratum%253Aachs%26aulast%3DLarson%26aufirst%3DS.%2BB.%26aulast%3DDay%26aufirst%3DJ.%2BS.%26aulast%3DCudney%26aufirst%3DR.%26aulast%3DMcPherson%26aufirst%3DA.%26atitle%3DA%2520Novel%2520Strategy%2520for%2520the%2520Crystallization%2520of%2520Proteins%253A%2520X-Ray%2520Diffraction%2520Validation%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2007%26volume%3D63%26issue%3D3%26spage%3D310%26epage%3D318%26doi%3D10.1107%2FS0907444906053303" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Desaphy, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bret, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rognan, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kellenberger, E.</span></span> <span> </span><span class="NLM_article-title">Sc-PDB: A 3D-Database of Ligandable Binding Sites--10 Years On</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>43</i></span> (<span class="NLM_issue">D1</span>),  <span class="NLM_fpage">D399</span>– <span class="NLM_lpage">D404</span>, <span class="refDoi"> DOI: 10.1093/nar/gku928</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00256&amp;key=10.1093%2Fnar%2Fgku928" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2015&pages=D399-D404&issue=D1&author=J.+Desaphyauthor=G.+Bretauthor=D.+Rognanauthor=E.+Kellenberger&title=Sc-PDB%3A+A+3D-Database+of+Ligandable+Binding+Sites%2D%2D10+Years+On&doi=10.1093%2Fnar%2Fgku928"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgku928&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgku928%26sid%3Dliteratum%253Aachs%26aulast%3DDesaphy%26aufirst%3DJ.%26aulast%3DBret%26aufirst%3DG.%26aulast%3DRognan%26aufirst%3DD.%26aulast%3DKellenberger%26aufirst%3DE.%26atitle%3DSc-PDB%253A%2520A%25203D-Database%2520of%2520Ligandable%2520Binding%2520Sites--10%2520Years%2520On%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2015%26volume%3D43%26issue%3DD1%26spage%3DD399%26epage%3DD404%26doi%3D10.1093%2Fnar%2Fgku928" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lipinski, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lombardo, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dominy, B. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feeney, P. J.</span></span> <span> </span><span class="NLM_article-title">Experimental and Computational Approaches to Estimate Solubility and Permeability in Drug Discovery and Development Settings</span>. <i>Adv. Drug Delivery Rev.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>46</i></span> (<span class="NLM_issue">1–3</span>),  <span class="NLM_fpage">3</span>– <span class="NLM_lpage">26</span>, <span class="refDoi"> DOI: 10.1016/S0169-409X(00)00129-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00256&amp;key=10.1016%2FS0169-409X%2800%2900129-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00256&amp;key=11259830" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00256&amp;key=1%3ACAS%3A528%3ADC%252BD3MXitVOhs7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2001&pages=3-26&issue=1%E2%80%933&author=C.+A.+Lipinskiauthor=F.+Lombardoauthor=B.+W.+Dominyauthor=P.+J.+Feeney&title=Experimental+and+Computational+Approaches+to+Estimate+Solubility+and+Permeability+in+Drug+Discovery+and+Development+Settings&doi=10.1016%2FS0169-409X%2800%2900129-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Experimental and computational approaches to estimate solubility and permeability in drug discovery and development settings</span></div><div class="casAuthors">Lipinski, C. A.; Lombardo, F.; Dominy, B. W.; Feeney, P. J.</div><div class="citationInfo"><span class="NLM_cas:title">Advanced Drug Delivery Reviews</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">1-3</span>),
    <span class="NLM_cas:pages">3-26</span>CODEN:
                <span class="NLM_cas:coden">ADDREP</span>;
        ISSN:<span class="NLM_cas:issn">0169-409X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ireland Ltd.</span>)
        </div><div class="casAbstract">A review with 50 refs.  Exptl. and computational approaches to est. soly. and permeability in discovery and development settings are described.  In the discovery setting 'the rule of 5' predicts that poor absorption or permeation is more likely when there are more than 5 H-bond donors, 10 H-bond acceptors, the mol. wt. (MWT) is greater than 500 and the calcd. Log P (CLogP) is greater than 5 (or MlogP >4.15).  Computational methodol. for the rule-based Moriguchi Log P (MLogP) calcn. is described.  Turbidimetric soly. measurement is described and applied to known drugs.  High throughput screening (HTS) leads tend to have higher MWT and Log P and lower turbidimetric soly. than leads in the pre-HTS era.  In the development setting, soly. calcns. focus on exact value prediction and are difficult because of polymorphism.  Recent work on linear free energy relationships and Log P approaches are critically reviewed.  Useful predictions are possible in closely related analog series when coupled with exptl. thermodn. soly. measurements.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoYDqa02g16urVg90H21EOLACvtfcHk0lhA-ThgALHQQg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXitVOhs7o%253D&md5=c60bb89da68f051c0ee7ac4c0468a0e4</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1016%2FS0169-409X%2800%2900129-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0169-409X%252800%252900129-0%26sid%3Dliteratum%253Aachs%26aulast%3DLipinski%26aufirst%3DC.%2BA.%26aulast%3DLombardo%26aufirst%3DF.%26aulast%3DDominy%26aufirst%3DB.%2BW.%26aulast%3DFeeney%26aufirst%3DP.%2BJ.%26atitle%3DExperimental%2520and%2520Computational%2520Approaches%2520to%2520Estimate%2520Solubility%2520and%2520Permeability%2520in%2520Drug%2520Discovery%2520and%2520Development%2520Settings%26jtitle%3DAdv.%2520Drug%2520Delivery%2520Rev.%26date%3D2001%26volume%3D46%26issue%3D1%25E2%2580%25933%26spage%3D3%26epage%3D26%26doi%3D10.1016%2FS0169-409X%2800%2900129-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bietz, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Urbaczek, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schulz, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rarey, M.</span></span> <span> </span><span class="NLM_article-title">Protoss: A Holistic Approach to Predict Tautomers and Protonation States in Protein-Ligand Complexes</span>. <i>J. Cheminf.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">12</span>, <span class="refDoi"> DOI: 10.1186/1758-2946-6-12</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00256&amp;key=10.1186%2F1758-2946-6-12" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00256&amp;key=1%3ACAS%3A528%3ADC%252BC2MXivFSktrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2014&pages=12&author=S.+Bietzauthor=S.+Urbaczekauthor=B.+Schulzauthor=M.+Rarey&title=Protoss%3A+A+Holistic+Approach+to+Predict+Tautomers+and+Protonation+States+in+Protein-Ligand+Complexes&doi=10.1186%2F1758-2946-6-12"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Protoss: a holistic approach to predict tautomers and protonation states in protein-ligand complexes</span></div><div class="casAuthors">Bietz, Stefan; Urbaczek, Sascha; Schulz, Benjamin; Rarey, Matthias</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Cheminformatics</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">12/1-12/12, 12 pp.</span>CODEN:
                <span class="NLM_cas:coden">JCOHB3</span>;
        ISSN:<span class="NLM_cas:issn">1758-2946</span>.
    
            (<span class="NLM_cas:orgname">Chemistry Central Ltd.</span>)
        </div><div class="casAbstract">The calcn. of hydrogen positions is a common preprocessing step when working with crystal structures of protein-ligand complexes.  An explicit description of hydrogen atoms is generally needed in order to analyze the binding mode of particular ligands or to calc. the assocd. binding energies.  Due to the large no. of degrees of freedom resulting from different chem. moieties and the high degree of mutual dependence this problem is anything but trivial.  In addn. to an efficient algorithm to take care of the complexity resulting from complicated hydrogen bonding networks, a robust chem. model is needed to describe effects such as tautomerism and ionization consistently.  We present a novel method for the placement of hydrogen coordinates in protein-ligand complexes which takes tautomers and protonation states of both protein and ligand into account.  Our method generates the most probable hydrogen positions on the basis of an optimal hydrogen bonding network using an empirical scoring function.  The high quality of our results could be verified by comparison to the manually adjusted Astex diverse set and a remarkably low rate of undesirable hydrogen contacts compared to other tools.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGouik3ZwZigrrVg90H21EOLACvtfcHk0lhA-ThgALHQQg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXivFSktrw%253D&md5=457b696089a3340ee9bb8e44609d3d4e</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1186%2F1758-2946-6-12&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1758-2946-6-12%26sid%3Dliteratum%253Aachs%26aulast%3DBietz%26aufirst%3DS.%26aulast%3DUrbaczek%26aufirst%3DS.%26aulast%3DSchulz%26aufirst%3DB.%26aulast%3DRarey%26aufirst%3DM.%26atitle%3DProtoss%253A%2520A%2520Holistic%2520Approach%2520to%2520Predict%2520Tautomers%2520and%2520Protonation%2520States%2520in%2520Protein-Ligand%2520Complexes%26jtitle%3DJ.%2520Cheminf.%26date%3D2014%26volume%3D6%26spage%3D12%26doi%3D10.1186%2F1758-2946-6-12" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group">The
UniProt Consortium.</span> <span> </span><span class="NLM_article-title">UniProt:
The Universal Protein Knowledgebase</span>. <i>Nucleic
Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>45</i></span> (<span class="NLM_issue">D1</span>),  <span class="NLM_fpage">D158</span>– <span class="NLM_lpage">D169</span>, <span class="refDoi"> DOI: 10.1093/nar/gkw1099</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00256&amp;key=10.1093%2Fnar%2Fgkw1099" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00256&amp;key=27899622" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2017&pages=D158-D169&issue=D1&author=The%0AUniProt+Consortium.&title=UniProt%3A%0AThe+Universal+Protein+Knowledgebase&doi=10.1093%2Fnar%2Fgkw1099"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkw1099&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkw1099%26sid%3Dliteratum%253Aachs%26aulast%3D%26atitle%3DUniProt%253A%250AThe%2520Universal%2520Protein%2520Knowledgebase%26jtitle%3DNucleic%250AAcids%2520Res.%26date%3D2017%26volume%3D45%26issue%3DD1%26spage%3DD158%26epage%3DD169%26doi%3D10.1093%2Fnar%2Fgkw1099" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rice, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Longden, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bleasby, A.</span></span> <span> </span><span class="NLM_article-title">EMBOSS: The European Molecular Biology Open Software Suite</span>. <i>Trends Genet.</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>16</i></span> (<span class="NLM_issue">6</span>),  <span class="NLM_fpage">276</span>– <span class="NLM_lpage">277</span>, <span class="refDoi"> DOI: 10.1016/S0168-9525(00)02024-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00256&amp;key=10.1016%2FS0168-9525%2800%2902024-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00256&amp;key=10827456" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00256&amp;key=1%3ACAS%3A528%3ADC%252BD3cXjvVygsbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2000&pages=276-277&issue=6&author=P.+Riceauthor=I.+Longdenauthor=A.+Bleasby&title=EMBOSS%3A+The+European+Molecular+Biology+Open+Software+Suite&doi=10.1016%2FS0168-9525%2800%2902024-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">EMBOSS: the european molecular biology open software suite</span></div><div class="casAuthors">Rice, Peter; Longden, Ian; Bleasby, Alan</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Genetics</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">276-277</span>CODEN:
                <span class="NLM_cas:coden">TRGEE2</span>;
        ISSN:<span class="NLM_cas:issn">0168-9525</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ltd.</span>)
        </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqqX8TJNeY34rVg90H21EOLACvtfcHk0lhA-ThgALHQQg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXjvVygsbs%253D&md5=6608aa9c93ff3740ca8af20578774ebe</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1016%2FS0168-9525%2800%2902024-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0168-9525%252800%252902024-2%26sid%3Dliteratum%253Aachs%26aulast%3DRice%26aufirst%3DP.%26aulast%3DLongden%26aufirst%3DI.%26aulast%3DBleasby%26aufirst%3DA.%26atitle%3DEMBOSS%253A%2520The%2520European%2520Molecular%2520Biology%2520Open%2520Software%2520Suite%26jtitle%3DTrends%2520Genet.%26date%3D2000%26volume%3D16%26issue%3D6%26spage%3D276%26epage%3D277%26doi%3D10.1016%2FS0168-9525%2800%2902024-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shindyalov, I. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bourne, P. E.</span></span> <span> </span><span class="NLM_article-title">Protein Structure Alignment by Incremental Combinatorial Extension (CE) of the Optimal Path</span>. <i>Protein Eng., Des. Sel.</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>11</i></span> (<span class="NLM_issue">9</span>),  <span class="NLM_fpage">739</span>– <span class="NLM_lpage">747</span>, <span class="refDoi"> DOI: 10.1093/protein/11.9.739</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00256&amp;key=10.1093%2Fprotein%2F11.9.739" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=1998&pages=739-747&issue=9&author=I.+N.+Shindyalovauthor=P.+E.+Bourne&title=Protein+Structure+Alignment+by+Incremental+Combinatorial+Extension+%28CE%29+of+the+Optimal+Path&doi=10.1093%2Fprotein%2F11.9.739"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1093%2Fprotein%2F11.9.739&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fprotein%252F11.9.739%26sid%3Dliteratum%253Aachs%26aulast%3DShindyalov%26aufirst%3DI.%2BN.%26aulast%3DBourne%26aufirst%3DP.%2BE.%26atitle%3DProtein%2520Structure%2520Alignment%2520by%2520Incremental%2520Combinatorial%2520Extension%2520%2528CE%2529%2520of%2520the%2520Optimal%2520Path%26jtitle%3DProtein%2520Eng.%252C%2520Des.%2520Sel.%26date%3D1998%26volume%3D11%26issue%3D9%26spage%3D739%26epage%3D747%26doi%3D10.1093%2Fprotein%2F11.9.739" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Desaphy, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Azdimousa, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kellenberger, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rognan, D.</span></span> <span> </span><span class="NLM_article-title">Comparison and Druggability Prediction of Protein–Ligand Binding Sites from Pharmacophore-Annotated Cavity Shapes</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>52</i></span> (<span class="NLM_issue">8</span>),  <span class="NLM_fpage">2287</span>– <span class="NLM_lpage">2299</span>, <span class="refDoi"> DOI: 10.1021/ci300184x</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci300184x" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00256&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtV2isb7I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2012&pages=2287-2299&issue=8&author=J.+Desaphyauthor=K.+Azdimousaauthor=E.+Kellenbergerauthor=D.+Rognan&title=Comparison+and+Druggability+Prediction+of+Protein%E2%80%93Ligand+Binding+Sites+from+Pharmacophore-Annotated+Cavity+Shapes&doi=10.1021%2Fci300184x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Comparison and Druggability Prediction of Protein-Ligand Binding Sites from Pharmacophore-Annotated Cavity Shapes</span></div><div class="casAuthors">Desaphy, Jeremy; Azdimousa, Karima; Kellenberger, Esther; Rognan, Didier</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2287-2299</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Estg. the pairwise similarity of protein-ligand binding sites is a fast and efficient way of predicting cross-reactivity and putative side effects of drug candidates.  Among the many tools available, three-dimensional (3D) alignment-dependent methods are usually slow and based on simplified representations of binding site atoms or surfaces.  On the other hand, fast and efficient alignment-free methods have recently been described but suffer from a lack of interpretability.  We herewith present a novel binding site description (VolSite), coupled to an alignment and comparison tool (Shaper) combining the speed of alignment-free methods with the interpretability of alignment-dependent approaches.  It is based on the comparison of neg. images of binding cavities encoding both shape and pharmacophoric properties at regularly spaced grid points.  Shaper approximates the resulting mol. shape with a smooth Gaussian function and aligns protein binding sites by optimizing their vol. overlap.  Volsite and Shaper were successfully applied to compare protein-ligand binding sites and to predict their structural druggability.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrjUMu2riLDLbVg90H21EOLACvtfcHk0ljURJ9GTvz7EQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtV2isb7I&md5=d019f05891c51085323078e5a880e30b</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Fci300184x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci300184x%26sid%3Dliteratum%253Aachs%26aulast%3DDesaphy%26aufirst%3DJ.%26aulast%3DAzdimousa%26aufirst%3DK.%26aulast%3DKellenberger%26aufirst%3DE.%26aulast%3DRognan%26aufirst%3DD.%26atitle%3DComparison%2520and%2520Druggability%2520Prediction%2520of%2520Protein%25E2%2580%2593Ligand%2520Binding%2520Sites%2520from%2520Pharmacophore-Annotated%2520Cavity%2520Shapes%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2012%26volume%3D52%26issue%3D8%26spage%3D2287%26epage%3D2299%26doi%3D10.1021%2Fci300184x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Marcou, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rognan, D.</span></span> <span> </span><span class="NLM_article-title">Optimizing Fragment and Scaffold Docking by Use of Molecular Interaction Fingerprints</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>47</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">195</span>– <span class="NLM_lpage">207</span>, <span class="refDoi"> DOI: 10.1021/ci600342e</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci600342e" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00256&amp;key=1%3ACAS%3A528%3ADC%252BD28Xht12iurzL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2007&pages=195-207&issue=1&author=G.+Marcouauthor=D.+Rognan&title=Optimizing+Fragment+and+Scaffold+Docking+by+Use+of+Molecular+Interaction+Fingerprints&doi=10.1021%2Fci600342e"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Optimizing Fragment and Scaffold Docking by Use of Molecular Interaction Fingerprints</span></div><div class="casAuthors">Marcou, Gilles; Rognan, Didier</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">195-207</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Protein-ligand interaction fingerprints have been used to postprocess docking poses of three ligand data sets: a set of 40 low-mol.-wt. compds. from the Protein Data Bank, a collection of 40 scaffolds from pharmaceutically relevant protein ligands, and a database of 19 scaffolds extd. from true cdk2 inhibitors seeded in 2230 scaffold decoys.  Four popular docking tools (FlexX, Glide, Gold, and Surflex) were used to generate poses for ligands of the three data sets.  In all cases, scoring by the similarity of interaction fingerprints to a given ref. was statistically superior to conventional scoring functions in posing low-mol.-wt. fragments, predicting protein-bound scaffold coordinates according to the known binding mode of related ligands, and screening a scaffold library to enrich a hit list in true cdk2-targeted scaffolds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpHLbVKxK-BfLVg90H21EOLACvtfcHk0ljURJ9GTvz7EQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xht12iurzL&md5=0c9137a39d40fbcc83546aec17b595ba</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Fci600342e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci600342e%26sid%3Dliteratum%253Aachs%26aulast%3DMarcou%26aufirst%3DG.%26aulast%3DRognan%26aufirst%3DD.%26atitle%3DOptimizing%2520Fragment%2520and%2520Scaffold%2520Docking%2520by%2520Use%2520of%2520Molecular%2520Interaction%2520Fingerprints%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2007%26volume%3D47%26issue%3D1%26spage%3D195%26epage%3D207%26doi%3D10.1021%2Fci600342e" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Desaphy, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raimbaud, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ducrot, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rognan, D.</span></span> <span> </span><span class="NLM_article-title">Encoding Protein-Ligand Interaction Patterns in Fingerprints and Graphs</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>53</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">623</span>– <span class="NLM_lpage">637</span>, <span class="refDoi"> DOI: 10.1021/ci300566n</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci300566n" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00256&amp;key=1%3ACAS%3A528%3ADC%252BC3sXivFCgtrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2013&pages=623-637&issue=3&author=J.+Desaphyauthor=E.+Raimbaudauthor=P.+Ducrotauthor=D.+Rognan&title=Encoding+Protein-Ligand+Interaction+Patterns+in+Fingerprints+and+Graphs&doi=10.1021%2Fci300566n"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Encoding Protein-Ligand Interaction Patterns in Fingerprints and Graphs</span></div><div class="casAuthors">Desaphy, Jeremy; Raimbaud, Eric; Ducrot, Pierre; Rognan, Didier</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">623-637</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We herewith present a novel and universal method to convert protein-ligand coordinates into a simple fingerprint of 210 integers registering the corresponding mol. interaction pattern.  Each interaction (hydrophobic, arom., hydrogen bond, ionic bond, metal complexation) is detected on the fly and phys. described by a pseudoatom centered either on the interacting ligand atom, the interacting protein atom, or the geometric center of both interacting atoms.  Counting all possible triplets of interaction pseudoatoms within six distance ranges, and pruning the full integer vector to keep the most frequent triplets enables the definition of a simple (210 integers) and coordinate frame-invariant interaction pattern descriptor (TIFP) that can be applied to compare any pair of protein-ligand complexes.  TIFP fingerprints have been calcd. for ca. 10 000 druggable protein-ligand complexes therefore enabling a wide comparison of relationships between interaction pattern similarity and ligand or binding site pairwise similarity.  We notably show that interaction pattern similarity strongly depends on binding site similarity.  In addn. to the TIFP fingerprint which registers intermol. interactions between a ligand and its target protein, we developed two tools (Ishape, Grim) to align protein-ligand complexes from their interaction patterns.  Ishape is based on the overlap of interaction pseudoatoms using a smooth Gaussian function, whereas Grim utilizes a std. clique detection algorithm to match interaction pattern graphs.  Both tools are complementary and enable protein-ligand complex alignments capitalizing on both global and local pattern similarities.  The new fingerprint and companion alignment tools have been successfully used in three scenarios: (i) interaction-biased alignment of protein-ligand complexes, (ii) postprocessing docking poses according to known interaction patterns for a particular target, and (iii) virtual screening for bioisosteric scaffolds sharing similar interaction patterns.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoPJWN4w2G1UbVg90H21EOLACvtfcHk0ljURJ9GTvz7EQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXivFCgtrc%253D&md5=48809ef8ecd09a323f8956b5f0fa3068</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1021%2Fci300566n&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci300566n%26sid%3Dliteratum%253Aachs%26aulast%3DDesaphy%26aufirst%3DJ.%26aulast%3DRaimbaud%26aufirst%3DE.%26aulast%3DDucrot%26aufirst%3DP.%26aulast%3DRognan%26aufirst%3DD.%26atitle%3DEncoding%2520Protein-Ligand%2520Interaction%2520Patterns%2520in%2520Fingerprints%2520and%2520Graphs%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2013%26volume%3D53%26issue%3D3%26spage%3D623%26epage%3D637%26doi%3D10.1021%2Fci300566n" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bajusz, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rácz, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Héberger, K.</span></span> <span> </span><span class="NLM_article-title">Why Is Tanimoto Index an Appropriate Choice for Fingerprint-Based Similarity Calculations?</span>. <i>J. Cheminf.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>7</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">20</span>, <span class="refDoi"> DOI: 10.1186/s13321-015-0069-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00256&amp;key=10.1186%2Fs13321-015-0069-3" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2015&pages=20&issue=1&author=D.+Bajuszauthor=A.+R%C3%A1czauthor=K.+H%C3%A9berger&title=Why+Is+Tanimoto+Index+an+Appropriate+Choice+for+Fingerprint-Based+Similarity+Calculations%3F&doi=10.1186%2Fs13321-015-0069-3"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1186%2Fs13321-015-0069-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs13321-015-0069-3%26sid%3Dliteratum%253Aachs%26aulast%3DBajusz%26aufirst%3DD.%26aulast%3DR%25C3%25A1cz%26aufirst%3DA.%26aulast%3DH%25C3%25A9berger%26aufirst%3DK.%26atitle%3DWhy%2520Is%2520Tanimoto%2520Index%2520an%2520Appropriate%2520Choice%2520for%2520Fingerprint-Based%2520Similarity%2520Calculations%253F%26jtitle%3DJ.%2520Cheminf.%26date%3D2015%26volume%3D7%26issue%3D1%26spage%3D20%26doi%3D10.1186%2Fs13321-015-0069-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 15 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Amanda E. Wakefield, Christine Yueh, Dmitri Beglov, Marcelo S. Castilho, Dima Kozakov, György M. Keserű, Adrian Whitty, <span class="NLM_string-name hlFld-ContribAuthor">Sandor Vajda</span>. </span><span class="cited-content_cbyCitation_article-title">Benchmark Sets for Binding Hot Spot Identification in Fragment-Based Ligand Discovery. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2020,</strong> <em>60 </em>
                                    (12)
                                     , 6612-6623. <a href="https://doi.org/10.1021/acs.jcim.0c00877" title="DOI URL">https://doi.org/10.1021/acs.jcim.0c00877</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.0c00877&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.0c00877%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DBenchmark%252BSets%252Bfor%252BBinding%252BHot%252BSpot%252BIdentification%252Bin%252BFragment-Based%252BLigand%252BDiscovery%26aulast%3DWakefield%26aufirst%3DAmanda%2BE.%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D31072020%26date%3D08122020%26volume%3D60%26issue%3D12%26spage%3D6612%26epage%3D6623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Merveille Eguida, <span class="NLM_string-name hlFld-ContribAuthor">Didier Rognan</span>. </span><span class="cited-content_cbyCitation_article-title">A Computer Vision Approach to Align and Compare Protein Cavities: Application to Fragment-Based Drug Design. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (13)
                                     , 7127-7142. <a href="https://doi.org/10.1021/acs.jmedchem.0c00422" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c00422</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c00422&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c00422%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DA%252BComputer%252BVision%252BApproach%252Bto%252BAlign%252Band%252BCompare%252BProtein%252BCavities%25253A%252BApplication%252Bto%252BFragment-Based%252BDrug%252BDesign%26aulast%3DEguida%26aufirst%3DMerveille%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D12032020%26date%3D18062020%26date%3D04062020%26volume%3D63%26issue%3D13%26spage%3D7127%26epage%3D7142" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Daniel A. Erlanson, Iwan J. P. de Esch, Wolfgang Jahnke, Christopher N. Johnson, <span class="NLM_string-name hlFld-ContribAuthor">Paul N. Mortenson</span>. </span><span class="cited-content_cbyCitation_article-title">Fragment-to-Lead Medicinal Chemistry Publications in 2018. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (9)
                                     , 4430-4444. <a href="https://doi.org/10.1021/acs.jmedchem.9b01581" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b01581</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b01581&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.9b01581%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DFragment-to-Lead%252BMedicinal%252BChemistry%252BPublications%252Bin%252B2018%26aulast%3DErlanson%26aufirst%3DDaniel%2BA.%26date%3D2020%26date%3D2020%26date%3D2019%26date%3D22092019%26date%3D22012020%26date%3D08012020%26volume%3D63%26issue%3D9%26spage%3D4430%26epage%3D4444" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Jade Fogha, Julien Diharce, Alan Obled, Samia Aci-Sèche, <span class="NLM_string-name hlFld-ContribAuthor">Pascal Bonnet</span>. </span><span class="cited-content_cbyCitation_article-title">Computational Analysis of Crystallization Additives for the Identification of New Allosteric Sites. </span><span class="cited-content_cbyCitation_journal-name">ACS Omega</span><span> <strong>2020,</strong> <em>5 </em>
                                    (5)
                                     , 2114-2122. <a href="https://doi.org/10.1021/acsomega.9b02697" title="DOI URL">https://doi.org/10.1021/acsomega.9b02697</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsomega.9b02697&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsomega.9b02697%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Omega%26atitle%3DComputational%252BAnalysis%252Bof%252BCrystallization%252BAdditives%252Bfor%252Bthe%252BIdentification%252Bof%252BNew%252BAllosteric%252BSites%26aulast%3DFogha%26aufirst%3DJade%26date%3D2020%26date%3D2020%26date%3D2019%26date%3D2019%26date%3D21082019%26date%3D21112019%26date%3D03022020%26volume%3D5%26issue%3D5%26spage%3D2114%26epage%3D2122" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Natalie
J. Tatum, Fernanda Duarte, Shina C. L. Kamerlin, <span class="NLM_string-name hlFld-ContribAuthor">Ehmke Pohl</span>. </span><span class="cited-content_cbyCitation_article-title">Relative Binding Energies Predict Crystallographic Binding Modes of Ethionamide Booster Lead Compounds. </span><span class="cited-content_cbyCitation_journal-name">The Journal of Physical Chemistry Letters</span><span> <strong>2019,</strong> <em>10 </em>
                                    (9)
                                     , 2244-2249. <a href="https://doi.org/10.1021/acs.jpclett.9b00741" title="DOI URL">https://doi.org/10.1021/acs.jpclett.9b00741</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jpclett.9b00741&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jpclett.9b00741%26sid%3Dliteratum%253Aachs%26jtitle%3DThe%2520Journal%2520of%2520Physical%2520Chemistry%2520Letters%26atitle%3DRelative%252BBinding%252BEnergies%252BPredict%252BCrystallographic%252BBinding%252BModes%252Bof%252BEthionamide%252BBooster%252BLead%252BCompounds%26aulast%3DTatum%26aufirst%3DNatalie%2BJ.%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D14032019%26date%3D09042019%26date%3D23042019%26date%3D09042019%26volume%3D10%26issue%3D9%26spage%3D2244%26epage%3D2249" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Zhi-Zheng  Wang</span>, <span class="hlFld-ContribAuthor ">Xing-Xing  Shi</span>, <span class="hlFld-ContribAuthor ">Guang-Yi  Huang</span>, <span class="hlFld-ContribAuthor ">Ge-Fei  Hao</span>, <span class="hlFld-ContribAuthor ">Guang-Fu  Yang</span>. </span><span class="cited-content_cbyCitation_article-title">Fragment-based drug design facilitates selective kinase inhibitor discovery. </span><span class="cited-content_cbyCitation_journal-name">Trends in Pharmacological Sciences</span><span> <strong>2021,</strong> <em>42 </em>
                                    (7)
                                     , 551-565. <a href="https://doi.org/10.1016/j.tips.2021.04.001" title="DOI URL">https://doi.org/10.1016/j.tips.2021.04.001</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.tips.2021.04.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.tips.2021.04.001%26sid%3Dliteratum%253Aachs%26jtitle%3DTrends%2520in%2520Pharmacological%2520Sciences%26atitle%3DFragment-based%252Bdrug%252Bdesign%252Bfacilitates%252Bselective%252Bkinase%252Binhibitor%252Bdiscovery%26aulast%3DWang%26aufirst%3DZhi-Zheng%26date%3D2021%26volume%3D42%26issue%3D7%26spage%3D551%26epage%3D565" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">José T.  Moreira-Filho</span>, <span class="hlFld-ContribAuthor ">Arthur C.  Silva</span>, <span class="hlFld-ContribAuthor ">Rafael F.  Dantas</span>, <span class="hlFld-ContribAuthor ">Barbara F.  Gomes</span>, <span class="hlFld-ContribAuthor ">Lauro R.  Souza Neto</span>, <span class="hlFld-ContribAuthor ">Jose  Brandao-Neto</span>, <span class="hlFld-ContribAuthor ">Raymond J.  Owens</span>, <span class="hlFld-ContribAuthor ">Nicholas  Furnham</span>, <span class="hlFld-ContribAuthor ">Bruno J.  Neves</span>, <span class="hlFld-ContribAuthor ">Floriano P.  Silva-Junior</span>, <span class="hlFld-ContribAuthor ">Carolina H.  Andrade</span>. </span><span class="cited-content_cbyCitation_article-title">Schistosomiasis Drug Discovery in the Era of Automation and Artificial Intelligence. </span><span class="cited-content_cbyCitation_journal-name">Frontiers in Immunology</span><span> <strong>2021,</strong> <em>12 </em><a href="https://doi.org/10.3389/fimmu.2021.642383" title="DOI URL">https://doi.org/10.3389/fimmu.2021.642383</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3389/fimmu.2021.642383&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3389%2Ffimmu.2021.642383%26sid%3Dliteratum%253Aachs%26jtitle%3DFrontiers%2520in%2520Immunology%26atitle%3DSchistosomiasis%252BDrug%252BDiscovery%252Bin%252Bthe%252BEra%252Bof%252BAutomation%252Band%252BArtificial%252BIntelligence%26aulast%3DMoreira-Filho%26aufirst%3DJos%25C3%25A9%2BT.%26date%3D2021%26date%3D2021%26volume%3D12" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Lena  Münzker</span>, <span class="hlFld-ContribAuthor ">Joy Kristin  Petrick</span>, <span class="hlFld-ContribAuthor ">Christian  Schleberger</span>, <span class="hlFld-ContribAuthor ">Damien  Clavel</span>, <span class="hlFld-ContribAuthor ">Irina  Cornaciu</span>, <span class="hlFld-ContribAuthor ">Rainer  Wilcken</span>, <span class="hlFld-ContribAuthor ">José A.  Márquez</span>, <span class="hlFld-ContribAuthor ">Gerhard  Klebe</span>, <span class="hlFld-ContribAuthor ">Andreas  Marzinzik</span>, <span class="hlFld-ContribAuthor ">Wolfgang  Jahnke</span>. </span><span class="cited-content_cbyCitation_article-title">Fragment‐Based Discovery of Non‐bisphosphonate Binders of
              Trypanosoma brucei
              Farnesyl Pyrophosphate Synthase. </span><span class="cited-content_cbyCitation_journal-name">ChemBioChem</span><span> <strong>2020,</strong> <em>21 </em>
                                    (21)
                                     , 3096-3111. <a href="https://doi.org/10.1002/cbic.202000246" title="DOI URL">https://doi.org/10.1002/cbic.202000246</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/cbic.202000246&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fcbic.202000246%26sid%3Dliteratum%253Aachs%26jtitle%3DChemBioChem%26atitle%3DFragment%2525E2%252580%252590Based%252BDiscovery%252Bof%252BNon%2525E2%252580%252590bisphosphonate%252BBinders%252Bof%252BTrypanosoma%252Bbrucei%252BFarnesyl%252BPyrophosphate%252BSynthase%26aulast%3DM%25C3%25BCnzker%26aufirst%3DLena%26date%3D2020%26date%3D2020%26volume%3D21%26issue%3D21%26spage%3D3096%26epage%3D3111" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Deborah E.A.  Lockhart</span>, <span class="hlFld-ContribAuthor ">Mathew  Stanley</span>, <span class="hlFld-ContribAuthor ">Olawale G.  Raimi</span>, <span class="hlFld-ContribAuthor ">David A.  Robinson</span>, <span class="hlFld-ContribAuthor ">Dominika  Boldovjakova</span>, <span class="hlFld-ContribAuthor ">Daniel R.  Squair</span>, <span class="hlFld-ContribAuthor ">Andrew T.  Ferenbach</span>, <span class="hlFld-ContribAuthor ">Wenxia  Fang</span>, <span class="hlFld-ContribAuthor ">Daan M.F.  van Aalten</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting a critical step in fungal hexosamine biosynthesis. </span><span class="cited-content_cbyCitation_journal-name">Journal of Biological Chemistry</span><span> <strong>2020,</strong> <em>295 </em>
                                    (26)
                                     , 8678-8691. <a href="https://doi.org/10.1074/jbc.RA120.012985" title="DOI URL">https://doi.org/10.1074/jbc.RA120.012985</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1074/jbc.RA120.012985&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1074%2Fjbc.RA120.012985%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Biological%2520Chemistry%26atitle%3DTargeting%252Ba%252Bcritical%252Bstep%252Bin%252Bfungal%252Bhexosamine%252Bbiosynthesis%26aulast%3DLockhart%26aufirst%3DDeborah%2BE.A.%26date%3D2020%26volume%3D295%26issue%3D26%26spage%3D8678%26epage%3D8691" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Moira  Rachman</span>, <span class="hlFld-ContribAuthor ">Dávid  Bajusz</span>, <span class="hlFld-ContribAuthor ">Anasztázia  Hetényi</span>, <span class="hlFld-ContribAuthor ">Andrea  Scarpino</span>, <span class="hlFld-ContribAuthor ">Balázs  Merő</span>, <span class="hlFld-ContribAuthor ">Attila  Egyed</span>, <span class="hlFld-ContribAuthor ">László  Buday</span>, <span class="hlFld-ContribAuthor ">Xavier  Barril</span>, <span class="hlFld-ContribAuthor ">György M.  Keserű</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of a novel kinase hinge binder fragment by dynamic undocking. </span><span class="cited-content_cbyCitation_journal-name">RSC Medicinal Chemistry</span><span> <strong>2020,</strong> <em>11 </em>
                                    (5)
                                     , 552-558. <a href="https://doi.org/10.1039/C9MD00519F" title="DOI URL">https://doi.org/10.1039/C9MD00519F</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/C9MD00519F&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FC9MD00519F%26sid%3Dliteratum%253Aachs%26jtitle%3DRSC%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252Ba%252Bnovel%252Bkinase%252Bhinge%252Bbinder%252Bfragment%252Bby%252Bdynamic%252Bundocking%26aulast%3DRachman%26aufirst%3DMoira%26date%3D2020%26date%3D2020%26volume%3D11%26issue%3D5%26spage%3D552%26epage%3D558" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Nicolas K.  Shinada</span>, <span class="hlFld-ContribAuthor ">Peter  Schmidtke</span>, <span class="hlFld-ContribAuthor ">Alexandre G.  de Brevern</span>. </span><span class="cited-content_cbyCitation_article-title">Accurate Representation of Protein-Ligand Structural Diversity in the Protein Data Bank (PDB). </span><span class="cited-content_cbyCitation_journal-name">International Journal of Molecular Sciences</span><span> <strong>2020,</strong> <em>21 </em>
                                    (6)
                                     , 2243. <a href="https://doi.org/10.3390/ijms21062243" title="DOI URL">https://doi.org/10.3390/ijms21062243</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/ijms21062243&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fijms21062243%26sid%3Dliteratum%253Aachs%26jtitle%3DInternational%2520Journal%2520of%2520Molecular%2520Sciences%26atitle%3DAccurate%252BRepresentation%252Bof%252BProtein-Ligand%252BStructural%252BDiversity%252Bin%252Bthe%252BProtein%252BData%252BBank%252B%252528PDB%252529%26aulast%3DShinada%26aufirst%3DNicolas%2BK.%26date%3D2020%26date%3D2020%26volume%3D21%26issue%3D6%26spage%3D2243" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Maciej  Majewski</span>, <span class="hlFld-ContribAuthor ">Sergio  Ruiz-Carmona</span>, <span class="hlFld-ContribAuthor ">Xavier  Barril</span>. </span><span class="cited-content_cbyCitation_article-title">An investigation of structural stability in protein-ligand complexes reveals the balance between order and disorder. </span><span class="cited-content_cbyCitation_journal-name">Communications Chemistry</span><span> <strong>2019,</strong> <em>2 </em>
                                    (1)
                                     <a href="https://doi.org/10.1038/s42004-019-0205-5" title="DOI URL">https://doi.org/10.1038/s42004-019-0205-5</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s42004-019-0205-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs42004-019-0205-5%26sid%3Dliteratum%253Aachs%26jtitle%3DCommunications%2520Chemistry%26atitle%3DAn%252Binvestigation%252Bof%252Bstructural%252Bstability%252Bin%252Bprotein-ligand%252Bcomplexes%252Breveals%252Bthe%252Bbalance%252Bbetween%252Border%252Band%252Bdisorder%26aulast%3DMajewski%26aufirst%3DMaciej%26date%3D2019%26date%3D2019%26volume%3D2%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Célien  Jacquemard</span>, <span class="hlFld-ContribAuthor ">Malgorzata N.  Drwal</span>, <span class="hlFld-ContribAuthor ">Jérémy  Desaphy</span>, <span class="hlFld-ContribAuthor ">Esther  Kellenberger</span>. </span><span class="cited-content_cbyCitation_article-title">Binding mode information improves fragment docking. </span><span class="cited-content_cbyCitation_journal-name">Journal of Cheminformatics</span><span> <strong>2019,</strong> <em>11 </em>
                                    (1)
                                     <a href="https://doi.org/10.1186/s13321-019-0346-7" title="DOI URL">https://doi.org/10.1186/s13321-019-0346-7</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1186/s13321-019-0346-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1186%2Fs13321-019-0346-7%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Cheminformatics%26atitle%3DBinding%252Bmode%252Binformation%252Bimproves%252Bfragment%252Bdocking%26aulast%3DJacquemard%26aufirst%3DC%25C3%25A9lien%26date%3D2019%26date%3D2019%26volume%3D11%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Célien  Jacquemard</span>, <span class="hlFld-ContribAuthor ">Viet-Khoa  Tran-Nguyen</span>, <span class="hlFld-ContribAuthor ">Malgorzata N.  Drwal</span>, <span class="hlFld-ContribAuthor ">Didier  Rognan</span>, <span class="hlFld-ContribAuthor ">Esther  Kellenberger</span>. </span><span class="cited-content_cbyCitation_article-title">Local Interaction Density (LID), a Fast and Efficient Tool to Prioritize Docking Poses. </span><span class="cited-content_cbyCitation_journal-name">Molecules</span><span> <strong>2019,</strong> <em>24 </em>
                                    (14)
                                     , 2610. <a href="https://doi.org/10.3390/molecules24142610" title="DOI URL">https://doi.org/10.3390/molecules24142610</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/molecules24142610&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fmolecules24142610%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecules%26atitle%3DLocal%252BInteraction%252BDensity%252B%252528LID%252529%25252C%252Ba%252BFast%252Band%252BEfficient%252BTool%252Bto%252BPrioritize%252BDocking%252BPoses%26aulast%3DJacquemard%26aufirst%3DC%25C3%25A9lien%26date%3D2019%26date%3D2019%26volume%3D24%26issue%3D14%26spage%3D2610" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Célien  Jacquemard</span>, <span class="hlFld-ContribAuthor ">Esther  Kellenberger</span>. </span><span class="cited-content_cbyCitation_article-title">A bright future for fragment-based drug discovery: what does it hold?. </span><span class="cited-content_cbyCitation_journal-name">Expert Opinion on Drug Discovery</span><span> <strong>2019,</strong> <em>14 </em>
                                    (5)
                                     , 413-416. <a href="https://doi.org/10.1080/17460441.2019.1583643" title="DOI URL">https://doi.org/10.1080/17460441.2019.1583643</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/17460441.2019.1583643&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F17460441.2019.1583643%26sid%3Dliteratum%253Aachs%26jtitle%3DExpert%2520Opinion%2520on%2520Drug%2520Discovery%26atitle%3DA%252Bbright%252Bfuture%252Bfor%252Bfragment-based%252Bdrug%252Bdiscovery%25253A%252Bwhat%252Bdoes%252Bit%252Bhold%25253F%26aulast%3DJacquemard%26aufirst%3DC%25C3%25A9lien%26date%3D2019%26date%3D2019%26volume%3D14%26issue%3D5%26spage%3D413%26epage%3D416" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-14/acs.jmedchem.8b00256/20180719/images/medium/jm-2018-002562_0003.gif" alt="" id="_i2" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-14/acs.jmedchem.8b00256/20180719/images/large/jm-2018-002562_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00256&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-14/acs.jmedchem.8b00256/20180719/images/medium/jm-2018-002562_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-14/acs.jmedchem.8b00256/20180719/images/large/jm-2018-002562_0006.jpeg" id="rightTab-GRAPHIC-d7e266-autogenerated" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Overview of the study. (A) Selection of 3D-structures in the PDB. (B) Numerical representation of binding mode used in this study. For the sake of illustration, depicted is the fragment X76 bound to human CDK2 ATP-binding pocket in the PDB structure <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3R1Y">3R1Y</a>. (C) Similarity between the binding modes of two small molecules bound to the same pocket. For the sake of illustration, displayed are the fragment X76 and drug-like ligand Z04 bound to human CDK2 ATP-binding pocket in the PDB structures <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3R1Y">3R1Y</a> and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3R7Y">3R7Y</a> (IFP sim = 1). (D) Similarity between the interactions coverage by all fragments and by all drug-like ligands bound to the same pocket. For the sake of illustration, displayed are all the ligands of human CDK2 ATP-binding pocket (cIFP sim = 0.895).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-14/acs.jmedchem.8b00256/20180719/images/large/jm-2018-002562_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00256&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-14/acs.jmedchem.8b00256/20180719/images/medium/jm-2018-002562_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-14/acs.jmedchem.8b00256/20180719/images/large/jm-2018-002562_0007.jpeg" id="rightTab-GRAPHIC-d7e369-autogenerated" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Binding mode conservation of the same fragment bound to multiple structures of the same protein site. (A) Histogram shows the frequency of the observed minimum binding mode similarity values. (B) 3-Isobutyl-1-methylxanthine (IBM) bound to human phosphodiesterase 9A2 in two biounits of the PDB structure <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2HD1">2HD1</a> (biounit 1 in magenta and biounit 2 in yellow). (C) Camphor (CAM) bound to <i>Pseudomonas putida</i> Camphor 5-monooxygenase in the PDB structures <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1DZ4">1DZ4</a> (green) and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4JX1">4JX1</a> (orange). In (B) and (C) protein backbone is displayed as cartoon and binding site residues are displayed as lines and fragments as sticks. Fragment chemical structures are inserted in the two images.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-14/acs.jmedchem.8b00256/20180719/images/large/jm-2018-002562_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00256&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-14/acs.jmedchem.8b00256/20180719/images/medium/jm-2018-002562_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-14/acs.jmedchem.8b00256/20180719/images/large/jm-2018-002562_0008.jpeg" id="rightTab-GRAPHIC-d7e411-autogenerated" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Analysis of 359 fragment–ligand substructure pairs. (A) Overall binding mode similarity considering all interactions (left) and polar interactions (right). The median for each substructure pair is shown. (B) Effect of fragment binding mode variability on the binding mode similarity of fragment–ligand substructure pairs. Fragment CK2 (PDB structures <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1PXJ">1PXJ</a> and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2C5O">2C5O</a>) and its drug-like superstructure CK8 (PDB structure <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2C5N">2C5N</a>) bound to human cyclin dependent kinase 2. The protein in its active form is displayed as ribbon, the fragment (orange) and ligand (green) are displayed as sticks, and hydrogen bonds in the hinge region of the protein are in yellow. For the sake of comparison, the inactive form of the protein is shown as a light orange ribbon on the right panel. The proportion of interactions conserved is high (0.89) when considering fragment and ligand binding to the active form of the protein (left) and decreases (0.57) when considering fragment bound to the inactive form and ligand bound to the active one (right). (C) Overall pose similarity considering shape overlap (ROCS shape, left) and chemistry match (ROCS color, right). The median for each substructure pair is shown.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-14/acs.jmedchem.8b00256/20180719/images/large/jm-2018-002562_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00256&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-14/acs.jmedchem.8b00256/20180719/images/medium/jm-2018-002562_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-14/acs.jmedchem.8b00256/20180719/images/large/jm-2018-002562_0009.jpeg" id="rightTab-GRAPHIC-d7e493-autogenerated" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Similarity between fragment and drug-like interaction patterns. Dependence of fragment–ligand interaction pattern similarity on the number of different molecules considered. In each step, all targets with at least <i>X</i> (1, 2, 3, etc.) HET codes of both fragments and drug-like ligands are considered. Number of distinct binding sites is shown in red.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-14/acs.jmedchem.8b00256/20180719/images/large/jm-2018-002562_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00256&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-14/acs.jmedchem.8b00256/20180719/images/medium/jm-2018-002562_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-14/acs.jmedchem.8b00256/20180719/images/large/jm-2018-002562_0010.jpeg" id="rightTab-GRAPHIC-d7e556-autogenerated" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Coverage of human leukotriene A-4 hydrolase pocket. (A) Drug-like ligand (left) and fragment (right) interaction heatmaps. Different types of interactions (hydrophobic, HYD; aromatic, AROM; hydrogen bonding, HB; ionic, IONIC; metal, METAL) are displayed on the <i>X</i>-axis. The binding site residues (one-letter code, residue number, chain) are displayed on the <i>y</i>-axis. The color intensity describes the frequency of the observed interaction in all complexes for this set (e.g., fragments). Residue number 1 is zinc. (B) Overlay of all fragments and drug-like ligands in the reference 3D-structure. Drug-like ligands (green) and fragments (magenta) are shown as sticks, the binding site is shown as surface, and the residues of the binding site are shown as lines.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-14/acs.jmedchem.8b00256/20180719/images/large/jm-2018-002562_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00256&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-14/acs.jmedchem.8b00256/20180719/images/medium/jm-2018-002562_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-14/acs.jmedchem.8b00256/20180719/images/large/jm-2018-002562_0011.jpeg" id="rightTab-GRAPHIC-d7e669-autogenerated" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Examples of additive binding mode conservation. (Left panel) Example of the binding mode conservation of an additive/drug-like ligand substructure pair. Example of the drug-like ligand CGS (green, PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1JIZ">1JIZ</a>) and its additive substructure HAE bound alone (white, PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1OS2">1OS2</a>) and in the presence of the ligand EEG (light pink, PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3LIK">3LIK</a>) in the human macrophage metalloelastase (MMP12) pocket. The protein is shown as green cartoon, and important protein residues are shown as sticks. Polar interactions are indicated as dotted lines. The zinc ion is shown as small sphere. (Right panel) Example of <i>apo</i> additives matching the polar interactions of a fragment. Bacillus thermoproteolyticus thermolysin bound to the fragment BYA (green, PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3FGD">3FGD</a>) and three examples of <i>apo</i> additives: 2-bromoacetate (cyan; PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3NN7">3NN7</a>), <i>S</i>-1,2-propanediol (blue, PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3N21">3N21</a>), and acetic acid (white, PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2A7G">2A7G</a>). The small molecules are shown are sticks, the protein is shown as cartoon, the interacting protein residues are shown as lines, and the zinc ion is shown as a small sphere. Polar interactions are indicated as dotted lines: hydrogen bonds in green and metal interaction in yellow. All electron densities have been deposited and correspond with the shown structures.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-14/acs.jmedchem.8b00256/20180719/images/large/jm-2018-002562_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00256&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-14/acs.jmedchem.8b00256/20180719/images/medium/jm-2018-002562_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-14/acs.jmedchem.8b00256/20180719/images/large/jm-2018-002562_0012.jpeg" id="rightTab-GRAPHIC-d7e704-autogenerated" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Examples of multiple additive poses in a protein site. (Left panel) Malonate (MLI, pink) bound to <i>E. coli</i> aminopeptidase N pocket (UniProt code <a href="https://P04825" class="ext-link">P04825</a>). The protein is shown as green cartoon and zinc ion as sphere. (Right panel) Glycerol (GOL, pink) and actinonin (BB2, green, PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4Q4E">4Q4E</a>) bound to <i>E. coli</i> aminopeptidase N pocket. The protein is shown as green cartoon, and hydrophobic protein residues in site are shown as lines.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-14/acs.jmedchem.8b00256/20180719/images/large/jm-2018-002562_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00256&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig8" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-14/acs.jmedchem.8b00256/20180719/images/medium/jm-2018-002562_0013.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-14/acs.jmedchem.8b00256/20180719/images/large/jm-2018-002562_0013.jpeg" id="rightTab-GRAPHIC-d7e735-autogenerated" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Examples of additive interactions. (Left panel) Glycerol (GOL, pink) and drug-like inhibitor (OFG, green, PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4CCB">4CCB</a>) bound to the hinge region of the ALK tyrosin kinase receptor ATP binding site. (Right panel) 1,2-Ethanediol (EDO, pink) and the inhibitor GSK525762 (EAM, green, PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2YEK">2YEK</a>) bound to bromodomain-containing protein 2. The protein is shown as green cartoon, and hydrophobic protein residues in site are shown as lines. Hydrogen bonds are indicated as yellow dotted lines.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-14/acs.jmedchem.8b00256/20180719/images/large/jm-2018-002562_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00256&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i29">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_54503" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_54503" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 33 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Doak, B. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Norton, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scanlon, M. J.</span></span> <span> </span><span class="NLM_article-title">The Ways and Means of Fragment-Based Drug Design</span>. <i>Pharmacol. Ther.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>167</i></span>,  <span class="NLM_fpage">28</span>– <span class="NLM_lpage">37</span>, <span class="refDoi"> DOI: 10.1016/j.pharmthera.2016.07.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00256&amp;key=10.1016%2Fj.pharmthera.2016.07.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00256&amp;key=27452339" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00256&amp;key=1%3ACAS%3A528%3ADC%252BC28Xht1eku77E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=167&publication_year=2016&pages=28-37&author=B.+C.+Doakauthor=R.+S.+Nortonauthor=M.+J.+Scanlon&title=The+Ways+and+Means+of+Fragment-Based+Drug+Design&doi=10.1016%2Fj.pharmthera.2016.07.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">The ways and means of fragment-based drug design</span></div><div class="casAuthors">Doak, Bradley C.; Norton, Raymond S.; Scanlon, Martin J.</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacology & Therapeutics</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">167</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">28-37</span>CODEN:
                <span class="NLM_cas:coden">PHTHDT</span>;
        ISSN:<span class="NLM_cas:issn">0163-7258</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">Fragment-based drug design (FBDD) has emerged as a mainstream approach for the rapid and efficient identification of building blocks that can be used to develop high-affinity ligands against protein targets.  One of the strengths of FBDD is the relative ease and low cost of the primary screen to identify fragments that bind.  However, the fragments that emerge from primary screens often have low affinities, with KD values in the high μM to mM range, and a significant challenge for FBDD is to develop the initial fragments into more potent ligands.  Successful fragment elaboration often requires co-structures of the fragments bound to their target proteins, as well as a range of biophys. and biochem. assays to track potency and efficacy.  These challenges have led to the development of specific chem. strategies for the elaboration of weakly-binding fragments into more potent "hits" and lead compds.  In this article we review different approaches that have been employed to meet these challenges and describe some of the strategies that have resulted in several fragment-derived compds. entering clin. trials.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGohQqashtjg-rVg90H21EOLACvtfcHk0ligg5MCBrHe1g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xht1eku77E&md5=f2d1425d4f3870b57d2dd99994252ddc</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1016%2Fj.pharmthera.2016.07.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.pharmthera.2016.07.003%26sid%3Dliteratum%253Aachs%26aulast%3DDoak%26aufirst%3DB.%2BC.%26aulast%3DNorton%26aufirst%3DR.%2BS.%26aulast%3DScanlon%26aufirst%3DM.%2BJ.%26atitle%3DThe%2520Ways%2520and%2520Means%2520of%2520Fragment-Based%2520Drug%2520Design%26jtitle%3DPharmacol.%2520Ther.%26date%3D2016%26volume%3D167%26spage%3D28%26epage%3D37%26doi%3D10.1016%2Fj.pharmthera.2016.07.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rees, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Congreve, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carr, R.</span></span> <span> </span><span class="NLM_article-title">Fragment-Based Lead Discovery</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>3</i></span> (<span class="NLM_issue">8</span>),  <span class="NLM_fpage">660</span>– <span class="NLM_lpage">672</span>, <span class="refDoi"> DOI: 10.1038/nrd1467</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00256&amp;key=10.1038%2Fnrd1467" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00256&amp;key=15286733" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00256&amp;key=1%3ACAS%3A528%3ADC%252BD2cXmtVOht7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2004&pages=660-672&issue=8&author=D.+C.+Reesauthor=M.+Congreveauthor=C.+W.+Murrayauthor=R.+Carr&title=Fragment-Based+Lead+Discovery&doi=10.1038%2Fnrd1467"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment-based lead discovery</span></div><div class="casAuthors">Rees, David C.; Congreve, Miles; Murray, Christopher W.; Carr, Robin</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">660-672</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Fragment-based lead discovery is gaining momentum in both large pharmaceutical companies and biotechnol. labs. as a complementary approach to traditional screening.  This is because fragment-based approaches require significantly fewer compds. to be screened and synthesized, and are showing a high success rate in generating chem. series with lead-like properties.  Compared with traditional screening hits, the starting fragments have considerably lower mol. mass, and although the binding interactions of these fragments with a target protein are weak, they are structurally understood through x-ray crystallog. or NMR, and they exhibit high 'ligand efficiency'.  Here, the authors use examples from 25 different protein targets to describe chem. strategies that exploit this structural knowledge to rapidly develop fragments into high-affinity leads.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrbKeUzFFIlhbVg90H21EOLACvtfcHk0ligg5MCBrHe1g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXmtVOht7Y%253D&md5=8d49ac7f698c9b9d573ee3c3485eced8</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1038%2Fnrd1467&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd1467%26sid%3Dliteratum%253Aachs%26aulast%3DRees%26aufirst%3DD.%2BC.%26aulast%3DCongreve%26aufirst%3DM.%26aulast%3DMurray%26aufirst%3DC.%2BW.%26aulast%3DCarr%26aufirst%3DR.%26atitle%3DFragment-Based%2520Lead%2520Discovery%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2004%26volume%3D3%26issue%3D8%26spage%3D660%26epage%3D672%26doi%3D10.1038%2Fnrd1467" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kalliokoski, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olsson, T. S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vulpetti, A.</span></span> <span> </span><span class="NLM_article-title">Subpocket Analysis Method for Fragment-Based Drug Discovery</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>53</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">131</span>– <span class="NLM_lpage">141</span>, <span class="refDoi"> DOI: 10.1021/ci300523r</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci300523r" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00256&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtFWlu7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2013&pages=131-141&issue=1&author=T.+Kalliokoskiauthor=T.+S.+G.+Olssonauthor=A.+Vulpetti&title=Subpocket+Analysis+Method+for+Fragment-Based+Drug+Discovery&doi=10.1021%2Fci300523r"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Subpocket Analysis Method for Fragment-Based Drug Discovery</span></div><div class="casAuthors">Kalliokoski, Tuomo; Olsson, Tjelvar S. G.; Vulpetti, Anna</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">131-141</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Although two binding sites might be dissimilar overall, they might still bind the same fragments if they share suitable subpockets.  Information about shared subpockets can be therefore used in fragment-based drug design to suggest new fragments or to replace existing fragments within an already known compd.  A novel computational method called SubCav is described which allows the similarity searching and alignment of subpockets from a PDB-wide database against a user-defined query.  The method is based on pharmacophoric fingerprints combined with a subpocket alignment algorithm.  SubCav was shown to be effective in producing reasonable alignments for subpockets with low sequence similarity and be able to retrieve relevant subpockets from a large database of structures including those with different folds.  It can also be used to analyze subpockets inside a protein family to facilitate drug design and to rationalize compd. selectivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp5ULmbpoNBtLVg90H21EOLACvtfcHk0liC4ob7abJCPw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtFWlu7c%253D&md5=49216111d0f500b6fefb715287fc315c</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1021%2Fci300523r&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci300523r%26sid%3Dliteratum%253Aachs%26aulast%3DKalliokoski%26aufirst%3DT.%26aulast%3DOlsson%26aufirst%3DT.%2BS.%2BG.%26aulast%3DVulpetti%26aufirst%3DA.%26atitle%3DSubpocket%2520Analysis%2520Method%2520for%2520Fragment-Based%2520Drug%2520Discovery%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2013%26volume%3D53%26issue%3D1%26spage%3D131%26epage%3D141%26doi%3D10.1021%2Fci300523r" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Carr, R. A.
E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Congreve, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rees, D. C.</span></span> <span> </span><span class="NLM_article-title">Fragment-Based Lead Discovery: Leads by Design</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>10</i></span> (<span class="NLM_issue">14</span>),  <span class="NLM_fpage">987</span>– <span class="NLM_lpage">992</span>, <span class="refDoi"> DOI: 10.1016/S1359-6446(05)03511-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00256&amp;key=10.1016%2FS1359-6446%2805%2903511-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00256&amp;key=16023057" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00256&amp;key=1%3ACAS%3A528%3ADC%252BD2MXmt1Whtr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2005&pages=987-992&issue=14&author=R.+A.%0AE.+Carrauthor=M.+Congreveauthor=C.+W.+Murrayauthor=D.+C.+Rees&title=Fragment-Based+Lead+Discovery%3A+Leads+by+Design&doi=10.1016%2FS1359-6446%2805%2903511-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment-based lead discovery: leads by design</span></div><div class="casAuthors">Carr, Robin A. E.; Congreve, Miles; Murray, Christopher W.; Rees, David C.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">987-992</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">A review.  Fragment-based lead discovery (also referred to as needles, shapes, binding elements, seed templates or scaffolds) is a new lead discovery approach in which much lower mol. wt. (120-250Da) compds. are screened relative to HTS campaigns.  Fragment-based hits are typically weak inhibitors (10μM-mM), and therefore need to be screened at higher concn. using very sensitive biophys. detection techniques such as protein crystallog. and NMR as the primary screening techniques, rather than bioassays.  Compared with HTS hits, these fragments are simpler, less functionalized compds. with correspondingly lower affinity.  However, fragment hits typically possess high ligand efficiency' (binding affinity per heavy atom) and so are highly suitable for optimization into clin. candidates with good drug-like properties.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq15FNhjT6Qw7Vg90H21EOLACvtfcHk0liC4ob7abJCPw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXmt1Whtr0%253D&md5=1bf7dd97ec056af9ed5143f96b34bb53</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1016%2FS1359-6446%2805%2903511-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1359-6446%252805%252903511-7%26sid%3Dliteratum%253Aachs%26aulast%3DCarr%26aufirst%3DR.%2BA.%2BE.%26aulast%3DCongreve%26aufirst%3DM.%26aulast%3DMurray%26aufirst%3DC.%2BW.%26aulast%3DRees%26aufirst%3DD.%2BC.%26atitle%3DFragment-Based%2520Lead%2520Discovery%253A%2520Leads%2520by%2520Design%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2005%26volume%3D10%26issue%3D14%26spage%3D987%26epage%3D992%26doi%3D10.1016%2FS1359-6446%2805%2903511-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hajduk, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greer, J.</span></span> <span> </span><span class="NLM_article-title">A Decade of Fragment-Based Drug Design: Strategic Advances and Lessons Learned</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>6</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">211</span>– <span class="NLM_lpage">219</span>, <span class="refDoi"> DOI: 10.1038/nrd2220</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00256&amp;key=10.1038%2Fnrd2220" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00256&amp;key=17290284" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00256&amp;key=1%3ACAS%3A528%3ADC%252BD2sXit12rtr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2007&pages=211-219&issue=3&author=P.+J.+Hajdukauthor=J.+Greer&title=A+Decade+of+Fragment-Based+Drug+Design%3A+Strategic+Advances+and+Lessons+Learned&doi=10.1038%2Fnrd2220"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">A decade of fragment-based drug design: strategic advances and lessons learned</span></div><div class="casAuthors">Hajduk, Philip J.; Greer, Jonathan</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">211-219</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Since the early 1990s, several technol. and scientific advances - such as combinatorial chem., high-throughput screening and the sequencing of the human genome - have been heralded as remedies to the problems facing the pharmaceutical industry.  The use of these technologies in some form is now well established at most pharmaceutical companies; however, the return on investment in terms of marketed products has not met expectations.  Fragment-based drug design is another tool for drug discovery that has emerged in the past decade.  Here, we describe the development and evolution of fragment-based drug design, analyze the role that this approach can have in combination with other discovery technologies and highlight the impact that fragment-based methods have made in progressing new medicines into the clinic.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpOB8K3AESqTbVg90H21EOLACvtfcHk0lh1J599Ht49HA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXit12rtr4%253D&md5=bae574da3671475d8488615c3bfa9849</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1038%2Fnrd2220&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd2220%26sid%3Dliteratum%253Aachs%26aulast%3DHajduk%26aufirst%3DP.%2BJ.%26aulast%3DGreer%26aufirst%3DJ.%26atitle%3DA%2520Decade%2520of%2520Fragment-Based%2520Drug%2520Design%253A%2520Strategic%2520Advances%2520and%2520Lessons%2520Learned%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2007%26volume%3D6%26issue%3D3%26spage%3D211%26epage%3D219%26doi%3D10.1038%2Fnrd2220" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Price, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Howard, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cons, B. D.</span></span> <span> </span><span class="NLM_article-title">Fragment-Based Drug Discovery and Its Application to Challenging Drug Targets</span>. <i>Essays Biochem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>61</i></span> (<span class="NLM_issue">5</span>),  <span class="NLM_fpage">475</span>– <span class="NLM_lpage">484</span>, <span class="refDoi"> DOI: 10.1042/EBC20170029</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00256&amp;key=10.1042%2FEBC20170029" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00256&amp;key=29118094" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00256&amp;key=1%3ACAS%3A280%3ADC%252BC1M7pt1GmtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2017&pages=475-484&issue=5&author=A.+J.+Priceauthor=S.+Howardauthor=B.+D.+Cons&title=Fragment-Based+Drug+Discovery+and+Its+Application+to+Challenging+Drug+Targets&doi=10.1042%2FEBC20170029"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment-based drug discovery and its application to challenging drug targets</span></div><div class="casAuthors">Price Amanda J; Howard Steven; Cons Benjamin D</div><div class="citationInfo"><span class="NLM_cas:title">Essays in biochemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">475-484</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Fragment-based drug discovery (FBDD) is a technique for identifying low molecular weight chemical starting points for drug discovery.  Since its inception 20 years ago, FBDD has grown in popularity to the point where it is now an established technique in industry and academia.  The approach involves the biophysical screening of proteins against collections of low molecular weight compounds (fragments).  Although fragments bind to proteins with relatively low affinity, they form efficient, high quality binding interactions with the protein architecture as they have to overcome a significant entropy barrier to bind.  Of the biophysical methods available for fragment screening, X-ray protein crystallography is one of the most sensitive and least prone to false positives.  It also provides detailed structural information of the protein-fragment complex at the atomic level.  Fragment-based screening using X-ray crystallography is therefore an efficient method for identifying binding hotspots on proteins, which can then be exploited by chemists and biologists for the discovery of new drugs.  The use of FBDD is illustrated here with a recently published case study of a drug discovery programme targeting the challenging protein-protein interaction Kelch-like ECH-associated protein 1:nuclear factor erythroid 2-related factor 2.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTqZGQ3bfUY0VHFYivNCdPRfW6udTcc2eaJ0tI0By-4prntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1M7pt1GmtA%253D%253D&md5=f8572516bfb1ecc4a0ef57b0c5ba1bc0</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1042%2FEBC20170029&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252FEBC20170029%26sid%3Dliteratum%253Aachs%26aulast%3DPrice%26aufirst%3DA.%2BJ.%26aulast%3DHoward%26aufirst%3DS.%26aulast%3DCons%26aufirst%3DB.%2BD.%26atitle%3DFragment-Based%2520Drug%2520Discovery%2520and%2520Its%2520Application%2520to%2520Challenging%2520Drug%2520Targets%26jtitle%3DEssays%2520Biochem.%26date%3D2017%26volume%3D61%26issue%3D5%26spage%3D475%26epage%3D484%26doi%3D10.1042%2FEBC20170029" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kozakov, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hall, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jehle, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ochiana, S. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, E. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pollastri, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allen, K. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whitty, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vajda, S.</span></span> <span> </span><span class="NLM_article-title">Ligand Deconstruction: Why Some Fragment Binding Positions Are Conserved and Others Are Not</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>112</i></span> (<span class="NLM_issue">20</span>),  <span class="NLM_fpage">E2585</span>– <span class="NLM_lpage">E2594</span>, <span class="refDoi"> DOI: 10.1073/pnas.1501567112</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00256&amp;key=10.1073%2Fpnas.1501567112" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=112&publication_year=2015&pages=E2585-E2594&issue=20&author=D.+Kozakovauthor=D.+R.+Hallauthor=S.+Jehleauthor=L.+Luoauthor=S.+O.+Ochianaauthor=E.+V.+Jonesauthor=M.+Pollastriauthor=K.+N.+Allenauthor=A.+Whittyauthor=S.+Vajda&title=Ligand+Deconstruction%3A+Why+Some+Fragment+Binding+Positions+Are+Conserved+and+Others+Are+Not&doi=10.1073%2Fpnas.1501567112"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1501567112&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1501567112%26sid%3Dliteratum%253Aachs%26aulast%3DKozakov%26aufirst%3DD.%26aulast%3DHall%26aufirst%3DD.%2BR.%26aulast%3DJehle%26aufirst%3DS.%26aulast%3DLuo%26aufirst%3DL.%26aulast%3DOchiana%26aufirst%3DS.%2BO.%26aulast%3DJones%26aufirst%3DE.%2BV.%26aulast%3DPollastri%26aufirst%3DM.%26aulast%3DAllen%26aufirst%3DK.%2BN.%26aulast%3DWhitty%26aufirst%3DA.%26aulast%3DVajda%26aufirst%3DS.%26atitle%3DLigand%2520Deconstruction%253A%2520Why%2520Some%2520Fragment%2520Binding%2520Positions%2520Are%2520Conserved%2520and%2520Others%2520Are%2520Not%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2015%26volume%3D112%26issue%3D20%26spage%3DE2585%26epage%3DE2594%26doi%3D10.1073%2Fpnas.1501567112" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rathi, P. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ludlow, R. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hall, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mortenson, P. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verdonk, M. L.</span></span> <span> </span><span class="NLM_article-title">Predicting “Hot” and “Warm” Spots for Fragment Binding</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span> (<span class="NLM_issue">9</span>),  <span class="NLM_fpage">4036</span>– <span class="NLM_lpage">4046</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b00366</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b00366" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=4036-4046&issue=9&author=P.+C.+Rathiauthor=R.+F.+Ludlowauthor=R.+J.+Hallauthor=C.+W.+Murrayauthor=P.+N.+Mortensonauthor=M.+L.+Verdonk&title=Predicting+%E2%80%9CHot%E2%80%9D+and+%E2%80%9CWarm%E2%80%9D+Spots+for+Fragment+Binding&doi=10.1021%2Facs.jmedchem.7b00366"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b00366&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b00366%26sid%3Dliteratum%253Aachs%26aulast%3DRathi%26aufirst%3DP.%2BC.%26aulast%3DLudlow%26aufirst%3DR.%2BF.%26aulast%3DHall%26aufirst%3DR.%2BJ.%26aulast%3DMurray%26aufirst%3DC.%2BW.%26aulast%3DMortenson%26aufirst%3DP.%2BN.%26aulast%3DVerdonk%26aufirst%3DM.%2BL.%26atitle%3DPredicting%2520%25E2%2580%259CHot%25E2%2580%259D%2520and%2520%25E2%2580%259CWarm%25E2%2580%259D%2520Spots%2520for%2520Fragment%2520Binding%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26issue%3D9%26spage%3D4036%26epage%3D4046%26doi%3D10.1021%2Facs.jmedchem.7b00366" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Berman, H. M.</span></span> <span> </span><span class="NLM_article-title">The Protein Data Bank</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>28</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">235</span>– <span class="NLM_lpage">242</span>, <span class="refDoi"> DOI: 10.1093/nar/28.1.235</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00256&amp;key=10.1093%2Fnar%2F28.1.235" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00256&amp;key=10592235" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00256&amp;key=1%3ACAS%3A528%3ADC%252BD3cXhvVKjt7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2000&pages=235-242&issue=1&author=H.+M.+Berman&title=The+Protein+Data+Bank&doi=10.1093%2Fnar%2F28.1.235"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">The Protein Data Bank</span></div><div class="casAuthors">Berman, Helen M.; Westbrook, John; Feng, Zukang; Gilliland, Gary; Bhat, T. N.; Weissig, Helge; Shindyalov, Ilya N.; Bourne, Philip E.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">235-242</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The Protein Data Bank (PDB; http://www.rcsb.org/pdb/)is the single worldwide archive of structural data of biol. macromols.  This paper describes the goals of the PDB, the systems in place for data deposition and access, how to obtain further information, and near-term plans for the future development of the resource.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpStGCxT8QzirVg90H21EOLACvtfcHk0lhoYWGynDKjkA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXhvVKjt7w%253D&md5=227fb393f754be2be375ab727bfd05dc</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1093%2Fnar%2F28.1.235&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252F28.1.235%26sid%3Dliteratum%253Aachs%26aulast%3DBerman%26aufirst%3DH.%2BM.%26atitle%3DThe%2520Protein%2520Data%2520Bank%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2000%26volume%3D28%26issue%3D1%26spage%3D235%26epage%3D242%26doi%3D10.1093%2Fnar%2F28.1.235" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Meyder, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nittinger, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lange, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klein, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rarey, M.</span></span> <span> </span><span class="NLM_article-title">Estimating Electron Density Support for Individual Atoms and Molecular Fragments in X-ray Structures</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>57</i></span> (<span class="NLM_issue">10</span>),  <span class="NLM_fpage">2437</span>– <span class="NLM_lpage">2447</span>, <span class="refDoi"> DOI: 10.1021/acs.jcim.7b00391</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.7b00391" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00256&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhs1Shs7rN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2017&pages=2437-2447&issue=10&author=A.+Meyderauthor=E.+Nittingerauthor=G.+Langeauthor=R.+Kleinauthor=M.+Rarey&title=Estimating+Electron+Density+Support+for+Individual+Atoms+and+Molecular+Fragments+in+X-ray+Structures&doi=10.1021%2Facs.jcim.7b00391"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Estimating Electron Density Support for Individual Atoms and Molecular Fragments in X-ray Structures</span></div><div class="casAuthors">Meyder, Agnes; Nittinger, Eva; Lange, Gudrun; Klein, Robert; Rarey, Matthias</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2437-2447</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Macromol. structures resolved by x-ray crystallog. are essential for life science research.  While some methods exist to automatically quantify the quality of the electron d. fit, none of them is without flaws.  Esp. the question of how well individual parts like atoms, small fragments, or mols. are supported by electron d. is difficult to quantify.  While taking exptl. uncertainties correctly into account, they do not offer an answer on how reliable an individual atom position is.  A rapid quantification of this at. position reliability would be highly valuable in structure-based mol. design.  To overcome this limitation, the authors introduce the electron d. score EDIA for individual atoms and mol. fragments.  EDIA assesses rapidly, automatically, and intuitively the fit of individual as well as multiple atoms (EDIAm) into electron d. accompanied by an integrated error anal.  The computation is based on the std. 2fo - fc electron d. map in combination with the model of the mol. structure.  For evaluating partial structures, EDIAm shows significant advantages compared to the real-space R correlation coeff. (RSCC) and the real-space difference d. Z score (RSZD) from the mol. modeler's point of view.  Thus, EDIA abolishes the time-consuming step of visually inspecting the electron d. during structure selection and curation.  It supports daily modeling tasks of medicinal and computational chemists and enables a fully automated assembly of large-scale, high-quality structure data sets.  Furthermore, EDIA scores can be applied for model validation and method development in computer-aided mol. design.  In contrast to measuring the deviation from the structure model by root-mean-squared deviation, EDIA scores allow comparison to the underlying exptl. data taking its uncertainty into account.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr9gkygdo2Is7Vg90H21EOLACvtfcHk0lhoYWGynDKjkA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhs1Shs7rN&md5=124b9ee097f58bb95b60412d9b730ca6</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.7b00391&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.7b00391%26sid%3Dliteratum%253Aachs%26aulast%3DMeyder%26aufirst%3DA.%26aulast%3DNittinger%26aufirst%3DE.%26aulast%3DLange%26aufirst%3DG.%26aulast%3DKlein%26aufirst%3DR.%26aulast%3DRarey%26aufirst%3DM.%26atitle%3DEstimating%2520Electron%2520Density%2520Support%2520for%2520Individual%2520Atoms%2520and%2520Molecular%2520Fragments%2520in%2520X-ray%2520Structures%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2017%26volume%3D57%26issue%3D10%26spage%3D2437%26epage%3D2447%26doi%3D10.1021%2Facs.jcim.7b00391" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lingel, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sendzik, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shultz, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cantwell, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dillon, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fuller, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gabriel, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McKenna, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rao, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sutton, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taft, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindvall, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bussiere, D. E.</span></span> <span> </span><span class="NLM_article-title">Structure-Guided Design of EED Binders Allosterically Inhibiting the Epigenetic Polycomb Repressive Complex 2 (PRC2) Methyltransferase</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">415</span>– <span class="NLM_lpage">427</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b01473</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01473" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00256&amp;key=1%3ACAS%3A528%3ADC%252BC28XitFaiurrF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=415-427&issue=1&author=A.+Lingelauthor=M.+Sendzikauthor=Y.+Huangauthor=M.+D.+Shultzauthor=J.+Cantwellauthor=M.+P.+Dillonauthor=X.+Fuauthor=J.+Fullerauthor=T.+Gabrielauthor=J.+Guauthor=X.+Jiangauthor=L.+Liauthor=F.+Liangauthor=M.+McKennaauthor=W.+Qiauthor=W.+Raoauthor=X.+Shengauthor=W.+Shuauthor=J.+Suttonauthor=B.+Taftauthor=L.+Wangauthor=J.+Zengauthor=H.+Zhangauthor=M.+Zhangauthor=K.+Zhaoauthor=M.+Lindvallauthor=D.+E.+Bussiere&title=Structure-Guided+Design+of+EED+Binders+Allosterically+Inhibiting+the+Epigenetic+Polycomb+Repressive+Complex+2+%28PRC2%29+Methyltransferase&doi=10.1021%2Facs.jmedchem.6b01473"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Guided Design of EED Binders Allosterically Inhibiting the Epigenetic Polycomb Repressive Complex 2 (PRC2) Methyltransferase</span></div><div class="casAuthors">Lingel, Andreas; Sendzik, Martin; Huang, Ying; Shultz, Michael D.; Cantwell, John; Dillon, Michael P.; Fu, Xingnian; Fuller, John; Gabriel, Tobias; Gu, Justin; Jiang, Xiangqing; Li, Ling; Liang, Fang; McKenna, Maureen; Qi, Wei; Rao, Weijun; Sheng, Xijun; Shu, Wei; Sutton, James; Taft, Benjamin; Wang, Long; Zeng, Jue; Zhang, Hailong; Zhang, Maya; Zhao, Kehao; Lindvall, Mika; Bussiere, Dirksen E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">415-427</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">PRC2 is a multisubunit methyltransferase involved in epigenetic regulation of early embryonic development and cell growth.  The catalytic subunit EZH2 methylates primarily lysine 27 of histone H3, leading to chromatin compaction and repression of tumor suppressor genes.  Inhibiting this activity by small mols. targeting EZH2 was shown to result in antitumor efficacy.  Here, the authors describe for the first time the optimization of a chem. series representing a new class of PRC2 inhibitors which acts allosterically via the trimethyllysine pocket of the noncatalytic EED subunit.  Deconstruction of a larger and complex screening hit to a simple fragment-sized mol. followed by structure-guided regrowth and careful property modulation were employed to yield compds. which achieve sub-micromolar inhibition in functional assays and cellular activity.  The resulting mols. can serve as a simplified entry point for lead optimization and can be utilized to study this new mechanism of PRC2 inhibition and the assocd. biol. in detail.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpRo2rkeTDLibVg90H21EOLACvtfcHk0ljPZWRNr6AzVw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitFaiurrF&md5=3aec108471439f4ac99728a57b4eaf8e</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01473&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01473%26sid%3Dliteratum%253Aachs%26aulast%3DLingel%26aufirst%3DA.%26aulast%3DSendzik%26aufirst%3DM.%26aulast%3DHuang%26aufirst%3DY.%26aulast%3DShultz%26aufirst%3DM.%2BD.%26aulast%3DCantwell%26aufirst%3DJ.%26aulast%3DDillon%26aufirst%3DM.%2BP.%26aulast%3DFu%26aufirst%3DX.%26aulast%3DFuller%26aufirst%3DJ.%26aulast%3DGabriel%26aufirst%3DT.%26aulast%3DGu%26aufirst%3DJ.%26aulast%3DJiang%26aufirst%3DX.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DLiang%26aufirst%3DF.%26aulast%3DMcKenna%26aufirst%3DM.%26aulast%3DQi%26aufirst%3DW.%26aulast%3DRao%26aufirst%3DW.%26aulast%3DSheng%26aufirst%3DX.%26aulast%3DShu%26aufirst%3DW.%26aulast%3DSutton%26aufirst%3DJ.%26aulast%3DTaft%26aufirst%3DB.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DZeng%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DH.%26aulast%3DZhang%26aufirst%3DM.%26aulast%3DZhao%26aufirst%3DK.%26aulast%3DLindvall%26aufirst%3DM.%26aulast%3DBussiere%26aufirst%3DD.%2BE.%26atitle%3DStructure-Guided%2520Design%2520of%2520EED%2520Binders%2520Allosterically%2520Inhibiting%2520the%2520Epigenetic%2520Polycomb%2520Repressive%2520Complex%25202%2520%2528PRC2%2529%2520Methyltransferase%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26issue%3D1%26spage%3D415%26epage%3D427%26doi%3D10.1021%2Facs.jmedchem.6b01473" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Czodrowski, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hölzemann, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barnickel, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greiner, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Musil, D.</span></span> <span> </span><span class="NLM_article-title">Selection of Fragments for Kinase Inhibitor Design: Decoration Is Key</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">457</span>– <span class="NLM_lpage">465</span>, <span class="refDoi"> DOI: 10.1021/jm501597j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm501597j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=457-465&issue=1&author=P.+Czodrowskiauthor=G.+H%C3%B6lzemannauthor=G.+Barnickelauthor=H.+Greinerauthor=D.+Musil&title=Selection+of+Fragments+for+Kinase+Inhibitor+Design%3A+Decoration+Is+Key&doi=10.1021%2Fjm501597j"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1021%2Fjm501597j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm501597j%26sid%3Dliteratum%253Aachs%26aulast%3DCzodrowski%26aufirst%3DP.%26aulast%3DH%25C3%25B6lzemann%26aufirst%3DG.%26aulast%3DBarnickel%26aufirst%3DG.%26aulast%3DGreiner%26aufirst%3DH.%26aulast%3DMusil%26aufirst%3DD.%26atitle%3DSelection%2520of%2520Fragments%2520for%2520Kinase%2520Inhibitor%2520Design%253A%2520Decoration%2520Is%2520Key%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26issue%3D1%26spage%3D457%26epage%3D465%26doi%3D10.1021%2Fjm501597j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Drwal, M. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacquemard, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perez, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Desaphy, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kellenberger, E.</span></span> <span> </span><span class="NLM_article-title">Do Fragments and Crystallization Additives Bind Similarly to Drug-like Ligands?</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>57</i></span> (<span class="NLM_issue">5</span>),  <span class="NLM_fpage">1197</span>– <span class="NLM_lpage">1209</span>, <span class="refDoi"> DOI: 10.1021/acs.jcim.6b00769</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.6b00769" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00256&amp;key=1%3ACAS%3A528%3ADC%252BC2sXmtFCgt7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2017&pages=1197-1209&issue=5&author=M.+N.+Drwalauthor=C.+Jacquemardauthor=C.+Perezauthor=J.+Desaphyauthor=E.+Kellenberger&title=Do+Fragments+and+Crystallization+Additives+Bind+Similarly+to+Drug-like+Ligands%3F&doi=10.1021%2Facs.jcim.6b00769"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Do Fragments and Crystallization Additives Bind Similarly to Drug-like Ligands?</span></div><div class="casAuthors">Drwal, Malgorzata N.; Jacquemard, Celien; Perez, Carlos; Desaphy, Jeremy; Kellenberger, Esther</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1197-1209</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The success of fragment-based drug design (FBDD) hinges upon the optimization of low-mol.-wt. compds. (MW < 300 Da) with weak binding affinities to lead compds. with high affinity and selectivity.  Usually, structural information from fragment-protein complexes is used to develop ideas about the binding mode of similar but drug-like mols.  In this regard, crystn. additives such as cryoprotectants or buffer components, which are highly abundant in crystal structures, are frequently ignored.  Thus, the aim of this study was to investigate the information present in protein complexes with fragments as well as those with additives and how they relate to the binding modes of their drug-like counterparts.  We present a thorough anal. of the binding modes of crystallog. additives, fragments, and drug-like ligands bound to four diverse targets of wide interest in drug discovery and highly represented in the Protein Data Bank: cyclin-dependent kinase 2, β-secretase 1, carbonic anhydrase 2, and trypsin.  We identified a total of 630 unique mols. bound to the catalytic binding sites, among them 31 additives, 222 fragments, and 377 drug-like ligands.  In general, we obsd. that, independent of the target, protein-fragment interaction patterns are highly similar to those of drug-like ligands and mostly cover the residues crucial for binding.  Crystallog. additives are also able to show conserved binding modes and recover the residues important for binding in some of the cases.  Moreover, we show evidence that the information from fragments and drug-like ligands can be applied to rescore docking poses in order to improve the prediction of binding modes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoVxqwFETp6zbVg90H21EOLACvtfcHk0ljPZWRNr6AzVw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXmtFCgt7k%253D&md5=56294da65813c14a2586719faca2da46</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.6b00769&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.6b00769%26sid%3Dliteratum%253Aachs%26aulast%3DDrwal%26aufirst%3DM.%2BN.%26aulast%3DJacquemard%26aufirst%3DC.%26aulast%3DPerez%26aufirst%3DC.%26aulast%3DDesaphy%26aufirst%3DJ.%26aulast%3DKellenberger%26aufirst%3DE.%26atitle%3DDo%2520Fragments%2520and%2520Crystallization%2520Additives%2520Bind%2520Similarly%2520to%2520Drug-like%2520Ligands%253F%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2017%26volume%3D57%26issue%3D5%26spage%3D1197%26epage%3D1209%26doi%3D10.1021%2Facs.jcim.6b00769" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Drwal, M. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bret, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kellenberger, E.</span></span> <span> </span><span class="NLM_article-title">Multi-Target Fragments Display Versatile Binding Modes</span>. <i>Mol. Inf.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>36</i></span> (<span class="NLM_issue">10</span>),  <span class="NLM_fpage">1700042</span>– <span class="NLM_lpage">1700042</span>, <span class="refDoi"> DOI: 10.1002/minf.201700042</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00256&amp;key=10.1002%2Fminf.201700042" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2017&pages=1700042-1700042&issue=10&author=M.+N.+Drwalauthor=G.+Bretauthor=E.+Kellenberger&title=Multi-Target+Fragments+Display+Versatile+Binding+Modes&doi=10.1002%2Fminf.201700042"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1002%2Fminf.201700042&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fminf.201700042%26sid%3Dliteratum%253Aachs%26aulast%3DDrwal%26aufirst%3DM.%2BN.%26aulast%3DBret%26aufirst%3DG.%26aulast%3DKellenberger%26aufirst%3DE.%26atitle%3DMulti-Target%2520Fragments%2520Display%2520Versatile%2520Binding%2520Modes%26jtitle%3DMol.%2520Inf.%26date%3D2017%26volume%3D36%26issue%3D10%26spage%3D1700042%26epage%3D1700042%26doi%3D10.1002%2Fminf.201700042" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kontopidis, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McInnes, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pandalaneni, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McNae, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gibson, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mezna, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wood, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walkinshaw, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fischer, P. M.</span></span> <span> </span><span class="NLM_article-title">Differential Binding of Inhibitors to Active and Inactive CDK2 Provides Insights for Drug Design</span>. <i>Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>13</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">201</span>– <span class="NLM_lpage">211</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2005.11.011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00256&amp;key=10.1016%2Fj.chembiol.2005.11.011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00256&amp;key=16492568" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00256&amp;key=1%3ACAS%3A528%3ADC%252BD28XhvVemsr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2006&pages=201-211&issue=2&author=G.+Kontopidisauthor=C.+McInnesauthor=S.+R.+Pandalaneniauthor=I.+McNaeauthor=D.+Gibsonauthor=M.+Meznaauthor=M.+Thomasauthor=G.+Woodauthor=S.+Wangauthor=M.+D.+Walkinshawauthor=P.+M.+Fischer&title=Differential+Binding+of+Inhibitors+to+Active+and+Inactive+CDK2+Provides+Insights+for+Drug+Design&doi=10.1016%2Fj.chembiol.2005.11.011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Differential Binding of Inhibitors to Active and Inactive CDK2 Provides Insights for Drug Design</span></div><div class="casAuthors">Kontopidis, George; McInnes, Campbell; Pandalaneni, Sravan R.; McNae, Iain; Gibson, Darren; Mezna, Mokdad; Thomas, Mark; Wood, Gavin; Wang, Shudong; Walkinshaw, Malcolm D.; Fischer, Peter M.</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">201-211</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The cyclin-dependent kinases (CDKs) have been characterized in complex with a variety of inhibitors, but the majority of structures solved are in the inactive form.  We have solved the structures of six inhibitors in both the monomeric CDK2 and binary CDK2/cyclinA complexes and demonstrate that significant differences in ligand binding occur depending on the activation state.  The binding mode of two ligands in particular varies substantially in active and inactive CDK2.  Furthermore, energetic anal. of CDK2/cyclin/inhibitors demonstrates that a good correlation exists between the in vitro potency and the calcd. energies of interaction, but no such relationship exists for CDK2/inhibitor structures.  These results confirm that monomeric CDK2 ligand complexes do not fully reflect active conformations, revealing significant implications for inhibitor design while also suggesting that the monomeric CDK2 conformation can be selectively inhibited.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpjllZrMBoaHbVg90H21EOLACvtfcHk0lil3Mek9GELEA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhvVemsr8%253D&md5=3f5dce248d66e7478a6f6661a47ca388</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2005.11.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2005.11.011%26sid%3Dliteratum%253Aachs%26aulast%3DKontopidis%26aufirst%3DG.%26aulast%3DMcInnes%26aufirst%3DC.%26aulast%3DPandalaneni%26aufirst%3DS.%2BR.%26aulast%3DMcNae%26aufirst%3DI.%26aulast%3DGibson%26aufirst%3DD.%26aulast%3DMezna%26aufirst%3DM.%26aulast%3DThomas%26aufirst%3DM.%26aulast%3DWood%26aufirst%3DG.%26aulast%3DWang%26aufirst%3DS.%26aulast%3DWalkinshaw%26aufirst%3DM.%2BD.%26aulast%3DFischer%26aufirst%3DP.%2BM.%26atitle%3DDifferential%2520Binding%2520of%2520Inhibitors%2520to%2520Active%2520and%2520Inactive%2520CDK2%2520Provides%2520Insights%2520for%2520Drug%2520Design%26jtitle%3DChem.%2520Biol.%26date%3D2006%26volume%3D13%26issue%3D2%26spage%3D201%26epage%3D211%26doi%3D10.1016%2Fj.chembiol.2005.11.011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tarcsay, Á.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nyíri, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keserű, G. M.</span></span> <span> </span><span class="NLM_article-title">Impact of Lipophilic Efficiency on Compound Quality</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>55</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">1252</span>– <span class="NLM_lpage">1260</span>, <span class="refDoi"> DOI: 10.1021/jm201388p</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm201388p" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=1252-1260&issue=3&author=%C3%81.+Tarcsayauthor=K.+Ny%C3%ADriauthor=G.+M.+Keser%C5%B1&title=Impact+of+Lipophilic+Efficiency+on+Compound+Quality&doi=10.1021%2Fjm201388p"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1021%2Fjm201388p&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm201388p%26sid%3Dliteratum%253Aachs%26aulast%3DTarcsay%26aufirst%3D%25C3%2581.%26aulast%3DNy%25C3%25ADri%26aufirst%3DK.%26aulast%3DKeser%25C5%25B1%26aufirst%3DG.%2BM.%26atitle%3DImpact%2520of%2520Lipophilic%2520Efficiency%2520on%2520Compound%2520Quality%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26issue%3D3%26spage%3D1252%26epage%3D1260%26doi%3D10.1021%2Fjm201388p" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Smith, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dougan, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Komandla, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kanouni, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knight, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lawson, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sabat, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, E. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vu, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wyrick, C.</span></span> <span> </span><span class="NLM_article-title">Fragment-Based Discovery of a Small Molecule Inhibitor of Bruton’s Tyrosine Kinase</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span> (<span class="NLM_issue">14</span>),  <span class="NLM_fpage">5437</span>– <span class="NLM_lpage">5444</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b00734</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b00734" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=5437-5444&issue=14&author=C.+R.+Smithauthor=D.+R.+Douganauthor=M.+Komandlaauthor=T.+Kanouniauthor=B.+Knightauthor=J.+D.+Lawsonauthor=M.+Sabatauthor=E.+R.+Taylorauthor=P.+Vuauthor=C.+Wyrick&title=Fragment-Based+Discovery+of+a+Small+Molecule+Inhibitor+of+Bruton%E2%80%99s+Tyrosine+Kinase&doi=10.1021%2Facs.jmedchem.5b00734"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b00734&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b00734%26sid%3Dliteratum%253Aachs%26aulast%3DSmith%26aufirst%3DC.%2BR.%26aulast%3DDougan%26aufirst%3DD.%2BR.%26aulast%3DKomandla%26aufirst%3DM.%26aulast%3DKanouni%26aufirst%3DT.%26aulast%3DKnight%26aufirst%3DB.%26aulast%3DLawson%26aufirst%3DJ.%2BD.%26aulast%3DSabat%26aufirst%3DM.%26aulast%3DTaylor%26aufirst%3DE.%2BR.%26aulast%3DVu%26aufirst%3DP.%26aulast%3DWyrick%26aufirst%3DC.%26atitle%3DFragment-Based%2520Discovery%2520of%2520a%2520Small%2520Molecule%2520Inhibitor%2520of%2520Bruton%25E2%2580%2599s%2520Tyrosine%2520Kinase%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26issue%3D14%26spage%3D5437%26epage%3D5444%26doi%3D10.1021%2Facs.jmedchem.5b00734" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Malhotra, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karanicolas, J.</span></span> <span> </span><span class="NLM_article-title">When Does Chemical Elaboration Induce a Ligand To Change Its Binding Mode?</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">128</span>– <span class="NLM_lpage">145</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b00725</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b00725" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=128-145&issue=1&author=S.+Malhotraauthor=J.+Karanicolas&title=When+Does+Chemical+Elaboration+Induce+a+Ligand+To+Change+Its+Binding+Mode%3F&doi=10.1021%2Facs.jmedchem.6b00725"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b00725&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b00725%26sid%3Dliteratum%253Aachs%26aulast%3DMalhotra%26aufirst%3DS.%26aulast%3DKaranicolas%26aufirst%3DJ.%26atitle%3DWhen%2520Does%2520Chemical%2520Elaboration%2520Induce%2520a%2520Ligand%2520To%2520Change%2520Its%2520Binding%2520Mode%253F%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26issue%3D1%26spage%3D128%26epage%3D145%26doi%3D10.1021%2Facs.jmedchem.6b00725" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Congreve, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carr, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jhoti, H.</span></span> <span> </span><span class="NLM_article-title">A “Rule of Three” for Fragment-Based Lead Discovery?</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>8</i></span> (<span class="NLM_issue">19</span>),  <span class="NLM_fpage">876</span>– <span class="NLM_lpage">877</span>, <span class="refDoi"> DOI: 10.1016/S1359-6446(03)02831-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00256&amp;key=10.1016%2FS1359-6446%2803%2902831-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00256&amp;key=14554012" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00256&amp;key=1%3ACAS%3A280%3ADC%252BD3svns1Cnsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2003&pages=876-877&issue=19&author=M.+Congreveauthor=R.+Carrauthor=C.+Murrayauthor=H.+Jhoti&title=A+%E2%80%9CRule+of+Three%E2%80%9D+for+Fragment-Based+Lead+Discovery%3F&doi=10.1016%2FS1359-6446%2803%2902831-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">A 'rule of three' for fragment-based lead discovery?</span></div><div class="casAuthors">Congreve Miles; Carr Robin; Murray Chris; Jhoti Harren</div><div class="citationInfo"><span class="NLM_cas:title">Drug discovery today</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">876-7</span>
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRfFswFSn5vb43yKPtYViAGfW6udTcc2ebuXCPkwJybh7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD3svns1Cnsw%253D%253D&md5=137886dbb2062f7a3ad39277dd888df3</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1016%2FS1359-6446%2803%2902831-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1359-6446%252803%252902831-9%26sid%3Dliteratum%253Aachs%26aulast%3DCongreve%26aufirst%3DM.%26aulast%3DCarr%26aufirst%3DR.%26aulast%3DMurray%26aufirst%3DC.%26aulast%3DJhoti%26aufirst%3DH.%26atitle%3DA%2520%25E2%2580%259CRule%2520of%2520Three%25E2%2580%259D%2520for%2520Fragment-Based%2520Lead%2520Discovery%253F%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2003%26volume%3D8%26issue%3D19%26spage%3D876%26epage%3D877%26doi%3D10.1016%2FS1359-6446%2803%2902831-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schauperl, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Czodrowski, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fuchs, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huber, R. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waldner, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Podewitz, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kramer, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liedl, K. R.</span></span> <span> </span><span class="NLM_article-title">Binding Pose Flip Explained via Enthalpic and Entropic Contributions</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>57</i></span> (<span class="NLM_issue">2</span>),  <span class="NLM_fpage">345</span>– <span class="NLM_lpage">354</span>, <span class="refDoi"> DOI: 10.1021/acs.jcim.6b00483</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.6b00483" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00256&amp;key=1%3ACAS%3A528%3ADC%252BC2sXosF2huw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2017&pages=345-354&issue=2&author=M.+Schauperlauthor=P.+Czodrowskiauthor=J.+E.+Fuchsauthor=R.+G.+Huberauthor=B.+J.+Waldnerauthor=M.+Podewitzauthor=C.+Kramerauthor=K.+R.+Liedl&title=Binding+Pose+Flip+Explained+via+Enthalpic+and+Entropic+Contributions&doi=10.1021%2Facs.jcim.6b00483"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Binding Pose Flip Explained via Enthalpic and Entropic Contributions</span></div><div class="casAuthors">Schauperl, Michael; Czodrowski, Paul; Fuchs, Julian E.; Huber, Roland G.; Waldner, Birgit J.; Podewitz, Maren; Kramer, Christian; Liedl, Klaus R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">345-354</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The anomalous binding modes of 5 highly similar fragments of angiopoietin receptor TIE2 inhibitors, showing 3 distinct binding poses were investigated.  The authors report a quant. rationalization for the changes in binding pose based on mol. dynamics (MD) simulations.  The authors investigated 5 fragments in complex with the transforming growth factor-β receptor type 1 kinase domain.  Analyses of these MD simulations using Grid Inhomogeneous Solvation Theory (GIST), pKa calcns., and a tool to investigate enthalpic differences upon binding unraveled the various thermodn. contributions to the different binding modes.  While one binding mode flip could be rationalized by steric repulsion, the 2nd binding pose flip revealed a different protonation state for one of the ligands, leading to different enthalpic and entropic contributions to the binding free energy.  While one binding pose was stabilized by the displacement of entropically unfavored water mols. (binding pose detd. by solvation entropy), ligands in the other binding pose were stabilized by strong enthalpic interactions, overcompensating the unfavorable water entropy in this pose (binding pose detd. by enthalpic interactions).  This anal. elucidated unprecedented details detg. the flipping of the binding modes, which can elegantly explain the exptl. findings for this system.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpyksMbMzYXP7Vg90H21EOLACvtfcHk0liL6I709QRDsw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXosF2huw%253D%253D&md5=9fa0737411901c4d27ae9954dca37ecf</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.6b00483&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.6b00483%26sid%3Dliteratum%253Aachs%26aulast%3DSchauperl%26aufirst%3DM.%26aulast%3DCzodrowski%26aufirst%3DP.%26aulast%3DFuchs%26aufirst%3DJ.%2BE.%26aulast%3DHuber%26aufirst%3DR.%2BG.%26aulast%3DWaldner%26aufirst%3DB.%2BJ.%26aulast%3DPodewitz%26aufirst%3DM.%26aulast%3DKramer%26aufirst%3DC.%26aulast%3DLiedl%26aufirst%3DK.%2BR.%26atitle%3DBinding%2520Pose%2520Flip%2520Explained%2520via%2520Enthalpic%2520and%2520Entropic%2520Contributions%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2017%26volume%3D57%26issue%3D2%26spage%3D345%26epage%3D354%26doi%3D10.1021%2Facs.jcim.6b00483" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kirkwood, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hargreaves, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Keefe, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, J.</span></span> <span> </span><span class="NLM_article-title">Analysis of Crystallization Data in the Protein Data Bank</span>. <i>Acta Crystallogr., Sect. F: Struct. Biol. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>71</i></span> (<span class="NLM_issue">10</span>),  <span class="NLM_fpage">1228</span>– <span class="NLM_lpage">1234</span>, <span class="refDoi"> DOI: 10.1107/S2053230X15014892</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00256&amp;key=10.1107%2FS2053230X15014892" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00256&amp;key=26457511" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00256&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhs1eksLnL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=71&publication_year=2015&pages=1228-1234&issue=10&author=J.+Kirkwoodauthor=D.+Hargreavesauthor=S.+O%E2%80%99Keefeauthor=J.+Wilson&title=Analysis+of+Crystallization+Data+in+the+Protein+Data+Bank&doi=10.1107%2FS2053230X15014892"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Analysis of crystallization data in the Protein Data Bank</span></div><div class="casAuthors">Kirkwood, Jobie; Hargreaves, David; O'Keefe, Simon; Wilson, Julie</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section F: Structural Biology Communications</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">71</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1228-1234</span>CODEN:
                <span class="NLM_cas:coden">ACSFEN</span>;
        ISSN:<span class="NLM_cas:issn">2053-230X</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">The Protein Data Bank (PDB) is the largest available repository of solved protein structures and contains a wealth of information on successful crystn.  Many centers have used their own exptl. data to draw conclusions about proteins and the conditions in which they crystallize.  Here, data from the PDB were used to reanalyze some of these results.  The most successful crystn. reagents were identified, the link between soln. pH and the isoelec. point of the protein was investigated and the possibility of predicting whether a protein will crystallize was explored.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq5dWh2xg2hXLVg90H21EOLACvtfcHk0liL6I709QRDsw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhs1eksLnL&md5=aad3de27009436f443e1f47e71a70099</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1107%2FS2053230X15014892&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS2053230X15014892%26sid%3Dliteratum%253Aachs%26aulast%3DKirkwood%26aufirst%3DJ.%26aulast%3DHargreaves%26aufirst%3DD.%26aulast%3DO%25E2%2580%2599Keefe%26aufirst%3DS.%26aulast%3DWilson%26aufirst%3DJ.%26atitle%3DAnalysis%2520of%2520Crystallization%2520Data%2520in%2520the%2520Protein%2520Data%2520Bank%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520F%253A%2520Struct.%2520Biol.%2520Commun.%26date%3D2015%26volume%3D71%26issue%3D10%26spage%3D1228%26epage%3D1234%26doi%3D10.1107%2FS2053230X15014892" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McPherson, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cudney, B.</span></span> <span> </span><span class="NLM_article-title">Searching for Silver Bullets: An Alternative Strategy for Crystallizing Macromolecules</span>. <i>J. Struct. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>156</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">387</span>– <span class="NLM_lpage">406</span>, <span class="refDoi"> DOI: 10.1016/j.jsb.2006.09.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00256&amp;key=10.1016%2Fj.jsb.2006.09.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00256&amp;key=17101277" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00256&amp;key=1%3ACAS%3A528%3ADC%252BD28Xht1CgsbzM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=156&publication_year=2006&pages=387-406&issue=3&author=A.+McPhersonauthor=B.+Cudney&title=Searching+for+Silver+Bullets%3A+An+Alternative+Strategy+for+Crystallizing+Macromolecules&doi=10.1016%2Fj.jsb.2006.09.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Searching for silver bullets: An alternative strategy for crystallizing macromolecules</span></div><div class="casAuthors">McPherson, Alexander; Cudney, Bob</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Structural Biology</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">156</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">387-406</span>CODEN:
                <span class="NLM_cas:coden">JSBIEM</span>;
        ISSN:<span class="NLM_cas:issn">1047-8477</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">Based on a hypothesis that various small mols. might establish stabilizing, intermol., non covalent crosslinks in protein crystals and thereby promote lattice formation, the authors carried out three sep. expts.  The authors assessed the impact of 200 chems. on the propensity of 81 different proteins and viruses to crystallize.  The expts. were comprised of 18 240 vapor diffusion trials.  A salient feature of the expts. was that, aside from the inclusion of the reagent mixes, only two fundamental crystn. conditions were used, 30% PEG 3350, and 50% Tacsimate at pH 7.  Overall, 65 proteins (85%) were crystd.  Most significant was that 35 of the 65 (54%) crystd. only in the presence of one or more reagent mixes, but not in control samples lacking any additives.  Among the most promising types of reagent mixes were those composed of polyvalent, charged groups, such as di and tri carboxylic acids, diamino compds., mols. bearing one or more sulfonyl or phosphate groups, and a broad range of common biochems., coenzymes, biol. effectors, and ligands.  The authors propose that an alternate approach to crystg. proteins might be developed, which employs a limited set of fundamental crystn. conditions combined with a broad screen of potentially useful small mol. additives.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGopSz9TqLH0_rVg90H21EOLACvtfcHk0liL6I709QRDsw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xht1CgsbzM&md5=f64b256b085e6fff737f501dc028b740</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1016%2Fj.jsb.2006.09.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jsb.2006.09.006%26sid%3Dliteratum%253Aachs%26aulast%3DMcPherson%26aufirst%3DA.%26aulast%3DCudney%26aufirst%3DB.%26atitle%3DSearching%2520for%2520Silver%2520Bullets%253A%2520An%2520Alternative%2520Strategy%2520for%2520Crystallizing%2520Macromolecules%26jtitle%3DJ.%2520Struct.%2520Biol.%26date%3D2006%26volume%3D156%26issue%3D3%26spage%3D387%26epage%3D406%26doi%3D10.1016%2Fj.jsb.2006.09.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Larson, S. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Day, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cudney, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McPherson, A.</span></span> <span> </span><span class="NLM_article-title">A Novel Strategy for the Crystallization of Proteins: X-Ray Diffraction Validation</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>63</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">310</span>– <span class="NLM_lpage">318</span>, <span class="refDoi"> DOI: 10.1107/S0907444906053303</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00256&amp;key=10.1107%2FS0907444906053303" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2007&pages=310-318&issue=3&author=S.+B.+Larsonauthor=J.+S.+Dayauthor=R.+Cudneyauthor=A.+McPherson&title=A+Novel+Strategy+for+the+Crystallization+of+Proteins%3A+X-Ray+Diffraction+Validation&doi=10.1107%2FS0907444906053303"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1107%2FS0907444906053303&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444906053303%26sid%3Dliteratum%253Aachs%26aulast%3DLarson%26aufirst%3DS.%2BB.%26aulast%3DDay%26aufirst%3DJ.%2BS.%26aulast%3DCudney%26aufirst%3DR.%26aulast%3DMcPherson%26aufirst%3DA.%26atitle%3DA%2520Novel%2520Strategy%2520for%2520the%2520Crystallization%2520of%2520Proteins%253A%2520X-Ray%2520Diffraction%2520Validation%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2007%26volume%3D63%26issue%3D3%26spage%3D310%26epage%3D318%26doi%3D10.1107%2FS0907444906053303" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Desaphy, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bret, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rognan, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kellenberger, E.</span></span> <span> </span><span class="NLM_article-title">Sc-PDB: A 3D-Database of Ligandable Binding Sites--10 Years On</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>43</i></span> (<span class="NLM_issue">D1</span>),  <span class="NLM_fpage">D399</span>– <span class="NLM_lpage">D404</span>, <span class="refDoi"> DOI: 10.1093/nar/gku928</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00256&amp;key=10.1093%2Fnar%2Fgku928" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2015&pages=D399-D404&issue=D1&author=J.+Desaphyauthor=G.+Bretauthor=D.+Rognanauthor=E.+Kellenberger&title=Sc-PDB%3A+A+3D-Database+of+Ligandable+Binding+Sites%2D%2D10+Years+On&doi=10.1093%2Fnar%2Fgku928"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgku928&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgku928%26sid%3Dliteratum%253Aachs%26aulast%3DDesaphy%26aufirst%3DJ.%26aulast%3DBret%26aufirst%3DG.%26aulast%3DRognan%26aufirst%3DD.%26aulast%3DKellenberger%26aufirst%3DE.%26atitle%3DSc-PDB%253A%2520A%25203D-Database%2520of%2520Ligandable%2520Binding%2520Sites--10%2520Years%2520On%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2015%26volume%3D43%26issue%3DD1%26spage%3DD399%26epage%3DD404%26doi%3D10.1093%2Fnar%2Fgku928" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lipinski, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lombardo, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dominy, B. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feeney, P. J.</span></span> <span> </span><span class="NLM_article-title">Experimental and Computational Approaches to Estimate Solubility and Permeability in Drug Discovery and Development Settings</span>. <i>Adv. Drug Delivery Rev.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>46</i></span> (<span class="NLM_issue">1–3</span>),  <span class="NLM_fpage">3</span>– <span class="NLM_lpage">26</span>, <span class="refDoi"> DOI: 10.1016/S0169-409X(00)00129-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00256&amp;key=10.1016%2FS0169-409X%2800%2900129-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00256&amp;key=11259830" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00256&amp;key=1%3ACAS%3A528%3ADC%252BD3MXitVOhs7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2001&pages=3-26&issue=1%E2%80%933&author=C.+A.+Lipinskiauthor=F.+Lombardoauthor=B.+W.+Dominyauthor=P.+J.+Feeney&title=Experimental+and+Computational+Approaches+to+Estimate+Solubility+and+Permeability+in+Drug+Discovery+and+Development+Settings&doi=10.1016%2FS0169-409X%2800%2900129-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Experimental and computational approaches to estimate solubility and permeability in drug discovery and development settings</span></div><div class="casAuthors">Lipinski, C. A.; Lombardo, F.; Dominy, B. W.; Feeney, P. J.</div><div class="citationInfo"><span class="NLM_cas:title">Advanced Drug Delivery Reviews</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">1-3</span>),
    <span class="NLM_cas:pages">3-26</span>CODEN:
                <span class="NLM_cas:coden">ADDREP</span>;
        ISSN:<span class="NLM_cas:issn">0169-409X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ireland Ltd.</span>)
        </div><div class="casAbstract">A review with 50 refs.  Exptl. and computational approaches to est. soly. and permeability in discovery and development settings are described.  In the discovery setting 'the rule of 5' predicts that poor absorption or permeation is more likely when there are more than 5 H-bond donors, 10 H-bond acceptors, the mol. wt. (MWT) is greater than 500 and the calcd. Log P (CLogP) is greater than 5 (or MlogP >4.15).  Computational methodol. for the rule-based Moriguchi Log P (MLogP) calcn. is described.  Turbidimetric soly. measurement is described and applied to known drugs.  High throughput screening (HTS) leads tend to have higher MWT and Log P and lower turbidimetric soly. than leads in the pre-HTS era.  In the development setting, soly. calcns. focus on exact value prediction and are difficult because of polymorphism.  Recent work on linear free energy relationships and Log P approaches are critically reviewed.  Useful predictions are possible in closely related analog series when coupled with exptl. thermodn. soly. measurements.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoYDqa02g16urVg90H21EOLACvtfcHk0ljZ65jwBHTKBw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXitVOhs7o%253D&md5=c60bb89da68f051c0ee7ac4c0468a0e4</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1016%2FS0169-409X%2800%2900129-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0169-409X%252800%252900129-0%26sid%3Dliteratum%253Aachs%26aulast%3DLipinski%26aufirst%3DC.%2BA.%26aulast%3DLombardo%26aufirst%3DF.%26aulast%3DDominy%26aufirst%3DB.%2BW.%26aulast%3DFeeney%26aufirst%3DP.%2BJ.%26atitle%3DExperimental%2520and%2520Computational%2520Approaches%2520to%2520Estimate%2520Solubility%2520and%2520Permeability%2520in%2520Drug%2520Discovery%2520and%2520Development%2520Settings%26jtitle%3DAdv.%2520Drug%2520Delivery%2520Rev.%26date%3D2001%26volume%3D46%26issue%3D1%25E2%2580%25933%26spage%3D3%26epage%3D26%26doi%3D10.1016%2FS0169-409X%2800%2900129-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bietz, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Urbaczek, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schulz, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rarey, M.</span></span> <span> </span><span class="NLM_article-title">Protoss: A Holistic Approach to Predict Tautomers and Protonation States in Protein-Ligand Complexes</span>. <i>J. Cheminf.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">12</span>, <span class="refDoi"> DOI: 10.1186/1758-2946-6-12</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00256&amp;key=10.1186%2F1758-2946-6-12" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00256&amp;key=1%3ACAS%3A528%3ADC%252BC2MXivFSktrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2014&pages=12&author=S.+Bietzauthor=S.+Urbaczekauthor=B.+Schulzauthor=M.+Rarey&title=Protoss%3A+A+Holistic+Approach+to+Predict+Tautomers+and+Protonation+States+in+Protein-Ligand+Complexes&doi=10.1186%2F1758-2946-6-12"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Protoss: a holistic approach to predict tautomers and protonation states in protein-ligand complexes</span></div><div class="casAuthors">Bietz, Stefan; Urbaczek, Sascha; Schulz, Benjamin; Rarey, Matthias</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Cheminformatics</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">12/1-12/12, 12 pp.</span>CODEN:
                <span class="NLM_cas:coden">JCOHB3</span>;
        ISSN:<span class="NLM_cas:issn">1758-2946</span>.
    
            (<span class="NLM_cas:orgname">Chemistry Central Ltd.</span>)
        </div><div class="casAbstract">The calcn. of hydrogen positions is a common preprocessing step when working with crystal structures of protein-ligand complexes.  An explicit description of hydrogen atoms is generally needed in order to analyze the binding mode of particular ligands or to calc. the assocd. binding energies.  Due to the large no. of degrees of freedom resulting from different chem. moieties and the high degree of mutual dependence this problem is anything but trivial.  In addn. to an efficient algorithm to take care of the complexity resulting from complicated hydrogen bonding networks, a robust chem. model is needed to describe effects such as tautomerism and ionization consistently.  We present a novel method for the placement of hydrogen coordinates in protein-ligand complexes which takes tautomers and protonation states of both protein and ligand into account.  Our method generates the most probable hydrogen positions on the basis of an optimal hydrogen bonding network using an empirical scoring function.  The high quality of our results could be verified by comparison to the manually adjusted Astex diverse set and a remarkably low rate of undesirable hydrogen contacts compared to other tools.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGouik3ZwZigrrVg90H21EOLACvtfcHk0ljZ65jwBHTKBw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXivFSktrw%253D&md5=457b696089a3340ee9bb8e44609d3d4e</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1186%2F1758-2946-6-12&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1758-2946-6-12%26sid%3Dliteratum%253Aachs%26aulast%3DBietz%26aufirst%3DS.%26aulast%3DUrbaczek%26aufirst%3DS.%26aulast%3DSchulz%26aufirst%3DB.%26aulast%3DRarey%26aufirst%3DM.%26atitle%3DProtoss%253A%2520A%2520Holistic%2520Approach%2520to%2520Predict%2520Tautomers%2520and%2520Protonation%2520States%2520in%2520Protein-Ligand%2520Complexes%26jtitle%3DJ.%2520Cheminf.%26date%3D2014%26volume%3D6%26spage%3D12%26doi%3D10.1186%2F1758-2946-6-12" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group">The
UniProt Consortium.</span> <span> </span><span class="NLM_article-title">UniProt:
The Universal Protein Knowledgebase</span>. <i>Nucleic
Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>45</i></span> (<span class="NLM_issue">D1</span>),  <span class="NLM_fpage">D158</span>– <span class="NLM_lpage">D169</span>, <span class="refDoi"> DOI: 10.1093/nar/gkw1099</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00256&amp;key=10.1093%2Fnar%2Fgkw1099" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00256&amp;key=27899622" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2017&pages=D158-D169&issue=D1&author=The%0AUniProt+Consortium.&title=UniProt%3A%0AThe+Universal+Protein+Knowledgebase&doi=10.1093%2Fnar%2Fgkw1099"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkw1099&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkw1099%26sid%3Dliteratum%253Aachs%26aulast%3D%26atitle%3DUniProt%253A%250AThe%2520Universal%2520Protein%2520Knowledgebase%26jtitle%3DNucleic%250AAcids%2520Res.%26date%3D2017%26volume%3D45%26issue%3DD1%26spage%3DD158%26epage%3DD169%26doi%3D10.1093%2Fnar%2Fgkw1099" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rice, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Longden, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bleasby, A.</span></span> <span> </span><span class="NLM_article-title">EMBOSS: The European Molecular Biology Open Software Suite</span>. <i>Trends Genet.</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>16</i></span> (<span class="NLM_issue">6</span>),  <span class="NLM_fpage">276</span>– <span class="NLM_lpage">277</span>, <span class="refDoi"> DOI: 10.1016/S0168-9525(00)02024-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00256&amp;key=10.1016%2FS0168-9525%2800%2902024-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00256&amp;key=10827456" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00256&amp;key=1%3ACAS%3A528%3ADC%252BD3cXjvVygsbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2000&pages=276-277&issue=6&author=P.+Riceauthor=I.+Longdenauthor=A.+Bleasby&title=EMBOSS%3A+The+European+Molecular+Biology+Open+Software+Suite&doi=10.1016%2FS0168-9525%2800%2902024-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">EMBOSS: the european molecular biology open software suite</span></div><div class="casAuthors">Rice, Peter; Longden, Ian; Bleasby, Alan</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Genetics</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">276-277</span>CODEN:
                <span class="NLM_cas:coden">TRGEE2</span>;
        ISSN:<span class="NLM_cas:issn">0168-9525</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ltd.</span>)
        </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqqX8TJNeY34rVg90H21EOLACvtfcHk0ljgvWZz5QkD0g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXjvVygsbs%253D&md5=6608aa9c93ff3740ca8af20578774ebe</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1016%2FS0168-9525%2800%2902024-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0168-9525%252800%252902024-2%26sid%3Dliteratum%253Aachs%26aulast%3DRice%26aufirst%3DP.%26aulast%3DLongden%26aufirst%3DI.%26aulast%3DBleasby%26aufirst%3DA.%26atitle%3DEMBOSS%253A%2520The%2520European%2520Molecular%2520Biology%2520Open%2520Software%2520Suite%26jtitle%3DTrends%2520Genet.%26date%3D2000%26volume%3D16%26issue%3D6%26spage%3D276%26epage%3D277%26doi%3D10.1016%2FS0168-9525%2800%2902024-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shindyalov, I. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bourne, P. E.</span></span> <span> </span><span class="NLM_article-title">Protein Structure Alignment by Incremental Combinatorial Extension (CE) of the Optimal Path</span>. <i>Protein Eng., Des. Sel.</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>11</i></span> (<span class="NLM_issue">9</span>),  <span class="NLM_fpage">739</span>– <span class="NLM_lpage">747</span>, <span class="refDoi"> DOI: 10.1093/protein/11.9.739</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00256&amp;key=10.1093%2Fprotein%2F11.9.739" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=1998&pages=739-747&issue=9&author=I.+N.+Shindyalovauthor=P.+E.+Bourne&title=Protein+Structure+Alignment+by+Incremental+Combinatorial+Extension+%28CE%29+of+the+Optimal+Path&doi=10.1093%2Fprotein%2F11.9.739"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1093%2Fprotein%2F11.9.739&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fprotein%252F11.9.739%26sid%3Dliteratum%253Aachs%26aulast%3DShindyalov%26aufirst%3DI.%2BN.%26aulast%3DBourne%26aufirst%3DP.%2BE.%26atitle%3DProtein%2520Structure%2520Alignment%2520by%2520Incremental%2520Combinatorial%2520Extension%2520%2528CE%2529%2520of%2520the%2520Optimal%2520Path%26jtitle%3DProtein%2520Eng.%252C%2520Des.%2520Sel.%26date%3D1998%26volume%3D11%26issue%3D9%26spage%3D739%26epage%3D747%26doi%3D10.1093%2Fprotein%2F11.9.739" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Desaphy, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Azdimousa, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kellenberger, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rognan, D.</span></span> <span> </span><span class="NLM_article-title">Comparison and Druggability Prediction of Protein–Ligand Binding Sites from Pharmacophore-Annotated Cavity Shapes</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>52</i></span> (<span class="NLM_issue">8</span>),  <span class="NLM_fpage">2287</span>– <span class="NLM_lpage">2299</span>, <span class="refDoi"> DOI: 10.1021/ci300184x</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci300184x" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00256&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtV2isb7I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2012&pages=2287-2299&issue=8&author=J.+Desaphyauthor=K.+Azdimousaauthor=E.+Kellenbergerauthor=D.+Rognan&title=Comparison+and+Druggability+Prediction+of+Protein%E2%80%93Ligand+Binding+Sites+from+Pharmacophore-Annotated+Cavity+Shapes&doi=10.1021%2Fci300184x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Comparison and Druggability Prediction of Protein-Ligand Binding Sites from Pharmacophore-Annotated Cavity Shapes</span></div><div class="casAuthors">Desaphy, Jeremy; Azdimousa, Karima; Kellenberger, Esther; Rognan, Didier</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2287-2299</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Estg. the pairwise similarity of protein-ligand binding sites is a fast and efficient way of predicting cross-reactivity and putative side effects of drug candidates.  Among the many tools available, three-dimensional (3D) alignment-dependent methods are usually slow and based on simplified representations of binding site atoms or surfaces.  On the other hand, fast and efficient alignment-free methods have recently been described but suffer from a lack of interpretability.  We herewith present a novel binding site description (VolSite), coupled to an alignment and comparison tool (Shaper) combining the speed of alignment-free methods with the interpretability of alignment-dependent approaches.  It is based on the comparison of neg. images of binding cavities encoding both shape and pharmacophoric properties at regularly spaced grid points.  Shaper approximates the resulting mol. shape with a smooth Gaussian function and aligns protein binding sites by optimizing their vol. overlap.  Volsite and Shaper were successfully applied to compare protein-ligand binding sites and to predict their structural druggability.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrjUMu2riLDLbVg90H21EOLACvtfcHk0ljgvWZz5QkD0g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtV2isb7I&md5=d019f05891c51085323078e5a880e30b</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Fci300184x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci300184x%26sid%3Dliteratum%253Aachs%26aulast%3DDesaphy%26aufirst%3DJ.%26aulast%3DAzdimousa%26aufirst%3DK.%26aulast%3DKellenberger%26aufirst%3DE.%26aulast%3DRognan%26aufirst%3DD.%26atitle%3DComparison%2520and%2520Druggability%2520Prediction%2520of%2520Protein%25E2%2580%2593Ligand%2520Binding%2520Sites%2520from%2520Pharmacophore-Annotated%2520Cavity%2520Shapes%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2012%26volume%3D52%26issue%3D8%26spage%3D2287%26epage%3D2299%26doi%3D10.1021%2Fci300184x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Marcou, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rognan, D.</span></span> <span> </span><span class="NLM_article-title">Optimizing Fragment and Scaffold Docking by Use of Molecular Interaction Fingerprints</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>47</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">195</span>– <span class="NLM_lpage">207</span>, <span class="refDoi"> DOI: 10.1021/ci600342e</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci600342e" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00256&amp;key=1%3ACAS%3A528%3ADC%252BD28Xht12iurzL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2007&pages=195-207&issue=1&author=G.+Marcouauthor=D.+Rognan&title=Optimizing+Fragment+and+Scaffold+Docking+by+Use+of+Molecular+Interaction+Fingerprints&doi=10.1021%2Fci600342e"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Optimizing Fragment and Scaffold Docking by Use of Molecular Interaction Fingerprints</span></div><div class="casAuthors">Marcou, Gilles; Rognan, Didier</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">195-207</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Protein-ligand interaction fingerprints have been used to postprocess docking poses of three ligand data sets: a set of 40 low-mol.-wt. compds. from the Protein Data Bank, a collection of 40 scaffolds from pharmaceutically relevant protein ligands, and a database of 19 scaffolds extd. from true cdk2 inhibitors seeded in 2230 scaffold decoys.  Four popular docking tools (FlexX, Glide, Gold, and Surflex) were used to generate poses for ligands of the three data sets.  In all cases, scoring by the similarity of interaction fingerprints to a given ref. was statistically superior to conventional scoring functions in posing low-mol.-wt. fragments, predicting protein-bound scaffold coordinates according to the known binding mode of related ligands, and screening a scaffold library to enrich a hit list in true cdk2-targeted scaffolds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpHLbVKxK-BfLVg90H21EOLACvtfcHk0ljgvWZz5QkD0g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xht12iurzL&md5=0c9137a39d40fbcc83546aec17b595ba</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Fci600342e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci600342e%26sid%3Dliteratum%253Aachs%26aulast%3DMarcou%26aufirst%3DG.%26aulast%3DRognan%26aufirst%3DD.%26atitle%3DOptimizing%2520Fragment%2520and%2520Scaffold%2520Docking%2520by%2520Use%2520of%2520Molecular%2520Interaction%2520Fingerprints%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2007%26volume%3D47%26issue%3D1%26spage%3D195%26epage%3D207%26doi%3D10.1021%2Fci600342e" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Desaphy, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raimbaud, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ducrot, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rognan, D.</span></span> <span> </span><span class="NLM_article-title">Encoding Protein-Ligand Interaction Patterns in Fingerprints and Graphs</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>53</i></span> (<span class="NLM_issue">3</span>),  <span class="NLM_fpage">623</span>– <span class="NLM_lpage">637</span>, <span class="refDoi"> DOI: 10.1021/ci300566n</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci300566n" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00256&amp;key=1%3ACAS%3A528%3ADC%252BC3sXivFCgtrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2013&pages=623-637&issue=3&author=J.+Desaphyauthor=E.+Raimbaudauthor=P.+Ducrotauthor=D.+Rognan&title=Encoding+Protein-Ligand+Interaction+Patterns+in+Fingerprints+and+Graphs&doi=10.1021%2Fci300566n"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Encoding Protein-Ligand Interaction Patterns in Fingerprints and Graphs</span></div><div class="casAuthors">Desaphy, Jeremy; Raimbaud, Eric; Ducrot, Pierre; Rognan, Didier</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">623-637</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We herewith present a novel and universal method to convert protein-ligand coordinates into a simple fingerprint of 210 integers registering the corresponding mol. interaction pattern.  Each interaction (hydrophobic, arom., hydrogen bond, ionic bond, metal complexation) is detected on the fly and phys. described by a pseudoatom centered either on the interacting ligand atom, the interacting protein atom, or the geometric center of both interacting atoms.  Counting all possible triplets of interaction pseudoatoms within six distance ranges, and pruning the full integer vector to keep the most frequent triplets enables the definition of a simple (210 integers) and coordinate frame-invariant interaction pattern descriptor (TIFP) that can be applied to compare any pair of protein-ligand complexes.  TIFP fingerprints have been calcd. for ca. 10 000 druggable protein-ligand complexes therefore enabling a wide comparison of relationships between interaction pattern similarity and ligand or binding site pairwise similarity.  We notably show that interaction pattern similarity strongly depends on binding site similarity.  In addn. to the TIFP fingerprint which registers intermol. interactions between a ligand and its target protein, we developed two tools (Ishape, Grim) to align protein-ligand complexes from their interaction patterns.  Ishape is based on the overlap of interaction pseudoatoms using a smooth Gaussian function, whereas Grim utilizes a std. clique detection algorithm to match interaction pattern graphs.  Both tools are complementary and enable protein-ligand complex alignments capitalizing on both global and local pattern similarities.  The new fingerprint and companion alignment tools have been successfully used in three scenarios: (i) interaction-biased alignment of protein-ligand complexes, (ii) postprocessing docking poses according to known interaction patterns for a particular target, and (iii) virtual screening for bioisosteric scaffolds sharing similar interaction patterns.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoPJWN4w2G1UbVg90H21EOLACvtfcHk0ljHdK7cy7R3Qw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXivFCgtrc%253D&md5=48809ef8ecd09a323f8956b5f0fa3068</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1021%2Fci300566n&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci300566n%26sid%3Dliteratum%253Aachs%26aulast%3DDesaphy%26aufirst%3DJ.%26aulast%3DRaimbaud%26aufirst%3DE.%26aulast%3DDucrot%26aufirst%3DP.%26aulast%3DRognan%26aufirst%3DD.%26atitle%3DEncoding%2520Protein-Ligand%2520Interaction%2520Patterns%2520in%2520Fingerprints%2520and%2520Graphs%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2013%26volume%3D53%26issue%3D3%26spage%3D623%26epage%3D637%26doi%3D10.1021%2Fci300566n" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bajusz, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rácz, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Héberger, K.</span></span> <span> </span><span class="NLM_article-title">Why Is Tanimoto Index an Appropriate Choice for Fingerprint-Based Similarity Calculations?</span>. <i>J. Cheminf.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>7</i></span> (<span class="NLM_issue">1</span>),  <span class="NLM_fpage">20</span>, <span class="refDoi"> DOI: 10.1186/s13321-015-0069-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00256&amp;key=10.1186%2Fs13321-015-0069-3" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2015&pages=20&issue=1&author=D.+Bajuszauthor=A.+R%C3%A1czauthor=K.+H%C3%A9berger&title=Why+Is+Tanimoto+Index+an+Appropriate+Choice+for+Fingerprint-Based+Similarity+Calculations%3F&doi=10.1186%2Fs13321-015-0069-3"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1186%2Fs13321-015-0069-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs13321-015-0069-3%26sid%3Dliteratum%253Aachs%26aulast%3DBajusz%26aufirst%3DD.%26aulast%3DR%25C3%25A1cz%26aufirst%3DA.%26aulast%3DH%25C3%25A9berger%26aufirst%3DK.%26atitle%3DWhy%2520Is%2520Tanimoto%2520Index%2520an%2520Appropriate%2520Choice%2520for%2520Fingerprint-Based%2520Similarity%2520Calculations%253F%26jtitle%3DJ.%2520Cheminf.%26date%3D2015%26volume%3D7%26issue%3D1%26spage%3D20%26doi%3D10.1186%2Fs13321-015-0069-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3R1Y" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3R1Y','PDB','3R1Y'); return false;">PDB: 3R1Y</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3R7Y" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3R7Y','PDB','3R7Y'); return false;">PDB: 3R7Y</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2HD1" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2HD1','PDB','2HD1'); return false;">PDB: 2HD1</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1DZ4" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1DZ4','PDB','1DZ4'); return false;">PDB: 1DZ4</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4JX1" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4JX1','PDB','4JX1'); return false;">PDB: 4JX1</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1PXJ" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1PXJ','PDB','1PXJ'); return false;">PDB: 1PXJ</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2C5O" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2C5O','PDB','2C5O'); return false;">PDB: 2C5O</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2C5N" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2C5N','PDB','2C5N'); return false;">PDB: 2C5N</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1JIZ" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1JIZ','PDB','1JIZ'); return false;">PDB: 1JIZ</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1OS2" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1OS2','PDB','1OS2'); return false;">PDB: 1OS2</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3LIK" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3LIK','PDB','3LIK'); return false;">PDB: 3LIK</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3FGD" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3FGD','PDB','3FGD'); return false;">PDB: 3FGD</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3NN7" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3NN7','PDB','3NN7'); return false;">PDB: 3NN7</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3N21" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3N21','PDB','3N21'); return false;">PDB: 3N21</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2A7G" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2A7G','PDB','2A7G'); return false;">PDB: 2A7G</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4Q4E" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4Q4E','PDB','4Q4E'); return false;">PDB: 4Q4E</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4CCB" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4CCB','PDB','4CCB'); return false;">PDB: 4CCB</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2YEK" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2YEK','PDB','2YEK'); return false;">PDB: 2YEK</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i24"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.8b00256">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_45718"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a class="ext-link" href="/">ACS Publications website</a> at DOI: <a class="ext-link" href="/doi/abs/10.1021/acs.jmedchem.8b00256">10.1021/acs.jmedchem.8b00256</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Molecular formula strings and PDB and HET codes (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.8b00256/suppl_file/jm8b00256_si_001.xlsx">XLSX</a>)</p></li><li><p class="inline">Number of structures available for fragment–protein complexes (Figure S1), effect of conformational change and structure quality on the binding mode conservation of the same fragment within the same binding pocket (Figure S2), effect on protein flexibility on binding modes conservation (Figure S3), dependence of fragment–ligand binding mode conservation on chemical similarity (Figure S4), fragment size and conservation of binding mode and binding pose (Figure S5), chemical similarity between fragment and ligand sets for the fragment and ligand-rich targets (Figure S6), coverage of human β secretase pocket (Figure S7), binding mode conservation of the same additive bound to multiple structures of the same protein site (Figure S8), binding mode conservation of additive–ligand substructure pairs (Figure S9), dependence of additive–drug-like ligand interaction pattern similarity on the number of different molecules (Figure S10), variability of additive binding mode in multiple structures of the same protein site (Figure S11), nature of interaction made by additives (Figure S12), 3D-structure quality evaluated using the electron density support for individual atoms (Table S1), EDIA score of the complexes described in the figures (Table S2), properties of ligand and fragment-rich targets (Table S3), and unique polar interactions of targets with many fragments and ligands (Table S4) (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.8b00256/suppl_file/jm8b00256_si_002.pdf">PDF</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.8b00256/suppl_file/jm8b00256_si_001.xlsx">jm8b00256_si_001.xlsx (11.93 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.8b00256/suppl_file/jm8b00256_si_002.pdf">jm8b00256_si_002.pdf (1.84 MB)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.8b00256&amp;pbContext=%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.8b00256%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2018.61.issue-14%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.8b00256" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                4MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"679906752885196f","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
