
<html lang="en"     class="pb-page"  data-request-id="2ced33f9-98bc-489b-a193-27443c87325f"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;article:article:10.1021/acs.jmedchem.0c00870;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;website:website:acspubs;issue:issue:10.1021/jmcmar.2020.63.issue-20;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Targeting Her2-insYVMA with Covalent Inhibitors—A Focused Compound Screening and Structure-Based Design Approach" /></meta><meta name="dc.Creator" content="Jonas  Lategahn" /></meta><meta name="dc.Creator" content="Julia  Hardick" /></meta><meta name="dc.Creator" content="Tobias  Grabe" /></meta><meta name="dc.Creator" content="Janina  Niggenaber" /></meta><meta name="dc.Creator" content="Kirujan  Jeyakumar" /></meta><meta name="dc.Creator" content="Marina  Keul" /></meta><meta name="dc.Creator" content="Hannah L.  Tumbrink" /></meta><meta name="dc.Creator" content="Christian  Becker" /></meta><meta name="dc.Creator" content="Luke  Hodson" /></meta><meta name="dc.Creator" content="Tonia  Kirschner" /></meta><meta name="dc.Creator" content="Philip  Klövekorn" /></meta><meta name="dc.Creator" content="Julia  Ketzer" /></meta><meta name="dc.Creator" content="Matthias  Baumann" /></meta><meta name="dc.Creator" content="Susanne  Terheyden" /></meta><meta name="dc.Creator" content="Anke  Unger" /></meta><meta name="dc.Creator" content="Jörn  Weisner" /></meta><meta name="dc.Creator" content="Matthias P.  Müller" /></meta><meta name="dc.Creator" content="Willem A. L.  van Otterlo" /></meta><meta name="dc.Creator" content="Sebastian  Bauer" /></meta><meta name="dc.Creator" content="Daniel  Rauh" /></meta><meta name="dc.Description" content="Mutated or amplified Her2 serves as a driver of non-small cell lung cancer or mediates resistance toward the inhibition of its family member epidermal growth factor receptor with small-molecule inh..." /></meta><meta name="Description" content="Mutated or amplified Her2 serves as a driver of non-small cell lung cancer or mediates resistance toward the inhibition of its family member epidermal growth factor receptor with small-molecule inh..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="September 15, 2020" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.0c00870" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2020 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c00870" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.0c00870" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c00870" /></link>
        
    
    

<title>Targeting Her2-insYVMA with Covalent Inhibitors—A Focused Compound Screening and Structure-Based Design Approach | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.0c00870" /></meta><meta property="og:title" content="Targeting Her2-insYVMA with Covalent Inhibitors—A Focused Compound Screening and Structure-Based Design Approach" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c00870/20201019/images/large/jm0c00870_0014.jpeg" /></meta><meta property="og:description" content="Mutated or amplified Her2 serves as a driver of non-small cell lung cancer or mediates resistance toward the inhibition of its family member epidermal growth factor receptor with small-molecule inhibitors. To date, small-molecule inhibitors targeting Her2 which can be used in clinical routine are lacking, and therefore, the development of novel inhibitors was undertaken. In this study, the well-established pyrrolopyrimidine scaffold was modified with structural motifs identified from a screening campaign with more than 1600 compounds, which were applied against wild-type Her2 and its mutant variant Her2-A775_G776insYVMA. The resulting inhibitors were designed to covalently target a reactive cysteine in the binding site of Her2 and were further optimized by means of structure-based drug design utilizing a set of obtained complex crystal structures. In addition, the analysis of binding kinetics and absorption, distribution, metabolism, and excretion parameters as well as mass spectrometry experiments and western blot analysis substantiated our approach." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.0c00870"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c00870">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.0c00870&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.0c00870&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.0c00870&amp;href=/doi/10.1021/acs.jmedchem.0c00870" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2020</span><span class="cit-fg-volume">, 63</span><span class="cit-fg-issue">, 20</span><span class="cit-fg-pageRange">, 11725-11755</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/63/20" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.0c00856" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.0c00858" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Targeting Her2-insYVMA with Covalent Inhibitors—A Focused Compound Screening and Structure-Based Design Approach</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Jonas Lategahn</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jonas Lategahn</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Faculty of Chemistry and Chemical Biology, TU Dortmund University, Otto-Hahn-Strasse 4a, 44227 Dortmund, Germany</div><div class="loa-info-affiliations-info">Drug Discovery Hub Dortmund (DDHD) am Zentrum für Integrierte Wirkstoffforschung (ZIW), 44227 Dortmund, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jonas++Lategahn">Jonas Lategahn</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-5993-7082" title="Orcid link">http://orcid.org/0000-0001-5993-7082</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Julia Hardick</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Julia Hardick</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Faculty of Chemistry and Chemical Biology, TU Dortmund University, Otto-Hahn-Strasse 4a, 44227 Dortmund, Germany</div><div class="loa-info-affiliations-info">Drug Discovery Hub Dortmund (DDHD) am Zentrum für Integrierte Wirkstoffforschung (ZIW), 44227 Dortmund, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Julia++Hardick">Julia Hardick</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Tobias Grabe</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Tobias Grabe</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Faculty of Chemistry and Chemical Biology, TU Dortmund University, Otto-Hahn-Strasse 4a, 44227 Dortmund, Germany</div><div class="loa-info-affiliations-info">Drug Discovery Hub Dortmund (DDHD) am Zentrum für Integrierte Wirkstoffforschung (ZIW), 44227 Dortmund, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Tobias++Grabe">Tobias Grabe</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Janina Niggenaber</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Janina Niggenaber</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Faculty of Chemistry and Chemical Biology, TU Dortmund University, Otto-Hahn-Strasse 4a, 44227 Dortmund, Germany</div><div class="loa-info-affiliations-info">Drug Discovery Hub Dortmund (DDHD) am Zentrum für Integrierte Wirkstoffforschung (ZIW), 44227 Dortmund, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Janina++Niggenaber">Janina Niggenaber</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-2942-263X" title="Orcid link">http://orcid.org/0000-0003-2942-263X</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Kirujan Jeyakumar</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Kirujan Jeyakumar</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Faculty of Chemistry and Chemical Biology, TU Dortmund University, Otto-Hahn-Strasse 4a, 44227 Dortmund, Germany</div><div class="loa-info-affiliations-info">Drug Discovery Hub Dortmund (DDHD) am Zentrum für Integrierte Wirkstoffforschung (ZIW), 44227 Dortmund, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Kirujan++Jeyakumar">Kirujan Jeyakumar</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-0184-6775" title="Orcid link">http://orcid.org/0000-0002-0184-6775</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Marina Keul</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Marina Keul</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Faculty of Chemistry and Chemical Biology, TU Dortmund University, Otto-Hahn-Strasse 4a, 44227 Dortmund, Germany</div><div class="loa-info-affiliations-info">Drug Discovery Hub Dortmund (DDHD) am Zentrum für Integrierte Wirkstoffforschung (ZIW), 44227 Dortmund, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Marina++Keul">Marina Keul</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Hannah L. Tumbrink</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Hannah L. Tumbrink</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Faculty of Chemistry and Chemical Biology, TU Dortmund University, Otto-Hahn-Strasse 4a, 44227 Dortmund, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Hannah+L.++Tumbrink">Hannah L. Tumbrink</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Christian Becker</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Christian Becker</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Faculty of Chemistry and Chemical Biology, TU Dortmund University, Otto-Hahn-Strasse 4a, 44227 Dortmund, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Christian++Becker">Christian Becker</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Luke Hodson</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Luke Hodson</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Faculty of Chemistry and Chemical Biology, TU Dortmund University, Otto-Hahn-Strasse 4a, 44227 Dortmund, Germany</div><div class="loa-info-affiliations-info">Department of Chemistry and Polymer Science, Stellenbosch University, Private Bag X1, 7602 Matieland, South Africa</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Luke++Hodson">Luke Hodson</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Tonia Kirschner</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Tonia Kirschner</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Faculty of Chemistry and Chemical Biology, TU Dortmund University, Otto-Hahn-Strasse 4a, 44227 Dortmund, Germany</div><div class="loa-info-affiliations-info">Drug Discovery Hub Dortmund (DDHD) am Zentrum für Integrierte Wirkstoffforschung (ZIW), 44227 Dortmund, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Tonia++Kirschner">Tonia Kirschner</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Philip Klövekorn</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Philip Klövekorn</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Faculty of Chemistry and Chemical Biology, TU Dortmund University, Otto-Hahn-Strasse 4a, 44227 Dortmund, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Philip++Kl%C3%B6vekorn">Philip Klövekorn</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Julia Ketzer</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Julia Ketzer</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Medical Oncology, Sarcoma Center, West German Cancer Center, Medical School, University Duisburg-Essen, Hufelandstrasse 55, 45122 Essen, Germany</div><div class="loa-info-affiliations-info">German Cancer Consortium (DKTK), 69120 Heidelberg, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Julia++Ketzer">Julia Ketzer</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Matthias Baumann</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Matthias Baumann</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Lead Discovery Center GmbH, Otto-Hahn-Strasse 15, 44227 Dortmund, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Matthias++Baumann">Matthias Baumann</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Susanne Terheyden</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Susanne Terheyden</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Lead Discovery Center GmbH, Otto-Hahn-Strasse 15, 44227 Dortmund, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Susanne++Terheyden">Susanne Terheyden</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Anke Unger</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Anke Unger</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Lead Discovery Center GmbH, Otto-Hahn-Strasse 15, 44227 Dortmund, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Anke++Unger">Anke Unger</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Jörn Weisner</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jörn Weisner</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Faculty of Chemistry and Chemical Biology, TU Dortmund University, Otto-Hahn-Strasse 4a, 44227 Dortmund, Germany</div><div class="loa-info-affiliations-info">Drug Discovery Hub Dortmund (DDHD) am Zentrum für Integrierte Wirkstoffforschung (ZIW), 44227 Dortmund, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=J%C3%B6rn++Weisner">Jörn Weisner</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Matthias P. Müller</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Matthias P. Müller</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Faculty of Chemistry and Chemical Biology, TU Dortmund University, Otto-Hahn-Strasse 4a, 44227 Dortmund, Germany</div><div class="loa-info-affiliations-info">Drug Discovery Hub Dortmund (DDHD) am Zentrum für Integrierte Wirkstoffforschung (ZIW), 44227 Dortmund, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Matthias+P.++M%C3%BCller">Matthias P. Müller</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-1529-8933" title="Orcid link">http://orcid.org/0000-0002-1529-8933</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Willem A. L. van Otterlo</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Willem A. L. van Otterlo</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemistry and Polymer Science, Stellenbosch University, Private Bag X1, 7602 Matieland, South Africa</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Willem+A.+L.++van+Otterlo">Willem A. L. van Otterlo</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-3300-6463" title="Orcid link">http://orcid.org/0000-0002-3300-6463</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Sebastian Bauer</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Sebastian Bauer</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Medical Oncology, Sarcoma Center, West German Cancer Center, Medical School, University Duisburg-Essen, Hufelandstrasse 55, 45122 Essen, Germany</div><div class="loa-info-affiliations-info">German Cancer Consortium (DKTK), 69120 Heidelberg, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Sebastian++Bauer">Sebastian Bauer</a></span></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Daniel Rauh</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Daniel Rauh</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Faculty of Chemistry and Chemical Biology, TU Dortmund University, Otto-Hahn-Strasse 4a, 44227 Dortmund, Germany</div><div class="loa-info-affiliations-info">Drug Discovery Hub Dortmund (DDHD) am Zentrum für Integrierte Wirkstoffforschung (ZIW), 44227 Dortmund, Germany</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#f99d9897909c95d78b988c91b98d8cd49d968b8d948c979dd79d9c"><span class="__cf_email__" data-cfemail="bfdbded1d6dad391cddecad7ffcbca92dbd0cdcbd2cad1db91dbda">[email protected]</span></a>. Phone: +49(0)231/755-7080. Twitter: @DDHDortmund.</span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Daniel++Rauh">Daniel Rauh</a></span></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c00870&amp;href=/doi/10.1021%2Facs.jmedchem.0c00870" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2020</span></span><span class="cit-volume">, 63</span><span class="cit-issue">, 20</span><span class="cit-pageRange">, 11725–11755</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">September 15, 2020</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>22 May 2020</li><li><span class="item_label"><b>Published</b> online</span>15 September 2020</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 22 October 2020</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.0c00870" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c00870</a></div><div class="article_header-article-copyright"><strong>Copyright © 2020 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D11725%26pageCount%3D31%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DJonas%2BLategahn%252C%2BJulia%2BHardick%252C%2BTobias%2BGrabe%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D63%26issueNum%3D20%26contentID%3Dacs.jmedchem.0c00870%26title%3DTargeting%2BHer2-insYVMA%2Bwith%2BCovalent%2BInhibitors%25E2%2580%2594A%2BFocused%2BCompound%2BScreening%2Band%2BStructure-Based%2BDesign%2BApproach%26numPages%3D31%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D11755%26publicationDate%3DOctober%2B2020">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.0c00870"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">1340</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">2</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.0c00870" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Targeting Her2-insYVMA with Covalent Inhibitors—A Focused Compound Screening and Structure-Based Design Approach&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Jonas&quot;,&quot;last_name&quot;:&quot;Lategahn&quot;},{&quot;first_name&quot;:&quot;Julia&quot;,&quot;last_name&quot;:&quot;Hardick&quot;},{&quot;first_name&quot;:&quot;Tobias&quot;,&quot;last_name&quot;:&quot;Grabe&quot;},{&quot;first_name&quot;:&quot;Janina&quot;,&quot;last_name&quot;:&quot;Niggenaber&quot;},{&quot;first_name&quot;:&quot;Kirujan&quot;,&quot;last_name&quot;:&quot;Jeyakumar&quot;},{&quot;first_name&quot;:&quot;Marina&quot;,&quot;last_name&quot;:&quot;Keul&quot;},{&quot;first_name&quot;:&quot;Hannah&quot;,&quot;last_name&quot;:&quot;L. Tumbrink&quot;},{&quot;first_name&quot;:&quot;Christian&quot;,&quot;last_name&quot;:&quot;Becker&quot;},{&quot;first_name&quot;:&quot;Luke&quot;,&quot;last_name&quot;:&quot;Hodson&quot;},{&quot;first_name&quot;:&quot;Tonia&quot;,&quot;last_name&quot;:&quot;Kirschner&quot;},{&quot;first_name&quot;:&quot;Philip&quot;,&quot;last_name&quot;:&quot;Klövekorn&quot;},{&quot;first_name&quot;:&quot;Julia&quot;,&quot;last_name&quot;:&quot;Ketzer&quot;},{&quot;first_name&quot;:&quot;Matthias&quot;,&quot;last_name&quot;:&quot;Baumann&quot;},{&quot;first_name&quot;:&quot;Susanne&quot;,&quot;last_name&quot;:&quot;Terheyden&quot;},{&quot;first_name&quot;:&quot;Anke&quot;,&quot;last_name&quot;:&quot;Unger&quot;},{&quot;first_name&quot;:&quot;Jörn&quot;,&quot;last_name&quot;:&quot;Weisner&quot;},{&quot;first_name&quot;:&quot;Matthias&quot;,&quot;last_name&quot;:&quot;P. Müller&quot;},{&quot;first_name&quot;:&quot;Willem&quot;,&quot;last_name&quot;:&quot;A. L. van Otterlo&quot;},{&quot;first_name&quot;:&quot;Sebastian&quot;,&quot;last_name&quot;:&quot;Bauer&quot;},{&quot;first_name&quot;:&quot;Daniel&quot;,&quot;last_name&quot;:&quot;Rauh&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2020&quot;,&quot;month&quot;:&quot;09&quot;,&quot;day&quot;:&quot;15&quot;,&quot;issue&quot;:&quot;20&quot;,&quot;volume&quot;:&quot;63&quot;,&quot;pages&quot;:&quot;11725-11755&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.0c00870&quot;},&quot;abstract&quot;:&quot;Mutated or amplified Her2 serves as a driver of non-small cell lung cancer or mediates resistance toward the inhibition of its family member epidermal growth factor receptor with small-molecule inhibitors. To date, small-molecule inhibitors targeting Her2 which can be used in clinical routine are lacking, and therefore, the development of novel inhibitors was undertaken. In this study, the well-established pyrrolopyrimidine scaffold was modified with structural motifs identified from a screening campaign with more than 1600 compounds, which were applied against wild-type Her2 and its mutant variant Her2-A775_G776insYVMA. The resulting inhibitors were designed to covalently target a reactive cysteine in the binding site of Her2 and were further optimized by means of structure-based drug design utilizing a set of obtained complex crystal structures. In addition, the analysis of binding kinetics and absorption, distribution, metabolism, and excretion parameters as well as mass spectrometry experiments and wester&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c00870&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c00870" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c00870&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c00870" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c00870&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c00870" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c00870&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c00870&amp;href=/doi/10.1021/acs.jmedchem.0c00870" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.0c00870" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.0c00870" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (6 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c00870&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.0c00870%26sid%3Dliteratum%253Aachs%26pmid%3D32931277%26genre%3Darticle%26aulast%3DLategahn%26date%3D2020%26atitle%3DTargeting%2BHer2-insYVMA%2Bwith%2BCovalent%2BInhibitors%25E2%2580%2594A%2BFocused%2BCompound%2BScreening%2Band%2BStructure-Based%2BDesign%2BApproach%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D63%26issue%3D20%26spage%3D11725%26epage%3D11755%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (2)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=290692" title="Elements">Elements</a>,</li><li><a href="/action/doSearch?ConceptID=291350" title="Assays">Assays</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=291793" title="Inhibition">Inhibition</a>,</li><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/63/20" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/jmcmar.2020.63.issue-20/20201022/jmcmar.2020.63.issue-20.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c00870/20201019/images/medium/jm0c00870_0014.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c00870/20201019/images/large/jm0c00870_0014.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00870&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Mutated or amplified Her2 serves as a driver of non-small cell lung cancer or mediates resistance toward the inhibition of its family member epidermal growth factor receptor with small-molecule inhibitors. To date, small-molecule inhibitors targeting Her2 which can be used in clinical routine are lacking, and therefore, the development of novel inhibitors was undertaken. In this study, the well-established pyrrolopyrimidine scaffold was modified with structural motifs identified from a screening campaign with more than 1600 compounds, which were applied against wild-type Her2 and its mutant variant Her2-A775_G776insYVMA. The resulting inhibitors were designed to covalently target a reactive cysteine in the binding site of Her2 and were further optimized by means of structure-based drug design utilizing a set of obtained complex crystal structures. In addition, the analysis of binding kinetics and absorption, distribution, metabolism, and excretion parameters as well as mass spectrometry experiments and western blot analysis substantiated our approach.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_27725" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_27725" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Mutant Her2 (Neu/ErbB2) is a driver of non-small cell lung cancer (NSCLC) in 4% of the patients.<a onclick="showRef(event, 'ref1 ref2'); return false;" href="javascript:void(0);" class="ref ref1 ref2">(1,2)</a> It is one of the four ErbB receptor tyrosine kinases with close homology to the ErbB family member epidermal growth factor receptor (EGFR). In contrast to its family members, no endogenous ligand for Her2 has been identified, and therefore, it exerts its biological function mainly as a hetero-dimerization partner with EGFR, Her3, or Her4 to enhance the downstream signaling.<a onclick="showRef(event, 'ref2 ref3 ref4'); return false;" href="javascript:void(0);" class="ref ref2 ref3 ref4">(2−4)</a> Her2 exhibits only low intrinsic kinase activity, which was proposed to be related to a GVG sequence, which is unique to Her2 among the family and provides flexibility to the adjacent regulatory helix αC in exon 20 (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>A and <a href="/doi/suppl/10.1021/acs.jmedchem.0c00870/suppl_file/jm0c00870_si_001.pdf" class="ext-link">S1</a>).<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> Because the orientation of this helix controls the activated or inhibited state of the kinase, it is believed that because of its increased flexibility the active state is less persistent. About 80% of mutations that occur in Her2 are insertions that affect the beginning of the helix αC.<a onclick="showRef(event, 'ref5 ref6'); return false;" href="javascript:void(0);" class="ref ref5 ref6">(5,6)</a> Most frequently found was the A775_G776insYVMA mutation that results in duplication of the YVMA motif (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>A and <a href="/doi/suppl/10.1021/acs.jmedchem.0c00870/suppl_file/jm0c00870_si_001.pdf" class="ext-link">S1</a>).<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a> An accepted theory is that mutations in exon 20 induce a stabilized active conformation of the regulatory helix, thereby increasing the kinase activity which finally results in the onset of the RAS–RAF–MEK–ERK pathway that regulates cell proliferation and survival.<a onclick="showRef(event, 'ref5 ref7'); return false;" href="javascript:void(0);" class="ref ref5 ref7">(5,7)</a> Moreover, signaling through amplified Her2 has been reported to be a mechanism of resistance toward the treatment of mutant EGFR with small-molecule tyrosine kinase inhibitors (TKIs).<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> Afatinib (<b>1</b>),<a onclick="showRef(event, 'ref9 ref10'); return false;" href="javascript:void(0);" class="ref ref9 ref10">(9,10)</a> poziotinib,<a onclick="showRef(event, 'ref11 ref12 ref13'); return false;" href="javascript:void(0);" class="ref ref11 ref12 ref13">(11−13)</a> dacomitinib,<a onclick="showRef(event, 'ref14 ref15'); return false;" href="javascript:void(0);" class="ref ref14 ref15">(14,15)</a> neratinib (<b>2</b>),<a onclick="showRef(event, 'ref16 ref17 ref18'); return false;" href="javascript:void(0);" class="ref ref16 ref17 ref18">(16−18)</a> and lapatinib (<b>3</b>)<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> have been evaluated as TKIs targeting Her2. Among these, afatinib (<b>1</b>), poziotinib, dacomitinib, and neratinib (<b>2</b>) bear an acrylamide to form a covalent bond with Cys805 (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>B, highlighted in yellow) that is located in the binding site of Her2. However, they had limited activity in patients with Her2-mutant tumors.<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a> Objective response rates of less than 40% were reported from clinical trials, although some activity was observed with afatinib in preclinical mouse models bearing mutated Her2.<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> Poziotinib was proposed as an effective inhibitor of Her2 in preclinical studies, but results from clinical stage are pending.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> Altogether, to date no TKI for the treatment of Her2-mutant NSCLC is approved, and therefore, there is a need to develop novel inhibitors that effectively target aberrant Her2 signaling.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c00870/20201019/images/medium/jm0c00870_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c00870/20201019/images/large/jm0c00870_0002.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. (A) Mutations found in Her2 are primarily insertion mutations adjacent to the regulatory helix αC (yellow; PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3PP0">3PP0</a>). Among these, A775_G776insYVMA is most predominant in Her2-mutated patients harboring NSCLC.<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a> (B) Type I and type II inhibitors were exemplified by afatinib (<b>1</b>) and neratinib (<b>2</b>), respectively, have been evaluated as Her2 inhibitors.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c00870/20201019/images/large/jm0c00870_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00870&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">Here, we present the development of covalent inhibitors based on the pyrrolopyrimidine scaffold that target mutant Her2-A775_G776insYVMA. These compounds were modified based on results from a biochemical screening campaign by adapting identified structural motives. Further derivatization was guided by X-ray complex crystal structures obtained from the close family member EGFR and yielded compounds with high potency on the cellular level. The resulting inhibitors were further examined with respect to their binding kinetics and <i>in vitro</i> absorption, distribution, metabolism, and excretion properties.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Results</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_08514" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_08514" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> Evaluation of Pyrrolopyrimidine-Based Compounds as Her2 Inhibitors</h3><div class="NLM_p">We recently described pyrrolopyrimidine-based inhibitors that revealed high potency against EGFR and its mutant variants. They possessed a reactive acrylamide that formed a covalent bond with Cys797 in the active site of EGFR.<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> Her2 exhibits Cys805 in a position isostructural to that in EGFR, which was believed to be similarly alkylated by these inhibitors. Given the high similarity between the ErbB family members (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00870/suppl_file/jm0c00870_si_001.pdf" class="ext-link">Figure S1</a>), we tested representative inhibitors <b>4–5</b> from previous works<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> against Her2 wild-type and the insertion YVMA-mutated kinase in biochemical activity-based assays (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). Indeed, especially the compounds <b>5a–d</b> exhibited potent inhibition of the kinase activity with IC<sub>50</sub> values between 3 and 18 nM against Her2-insYVMA. Next, we evaluated their potential against the cancer cell line H1781 that harbors an Her2-G776delinsVC mutation at a similar position as found in A775_G776insYVMA, which was inserted into Ba/F3 cells to obtain another model system. Despite the high biochemical activity observed with compounds <b>5a–d</b>, we detected only a moderate influence on the cellular viability with EC<sub>50</sub> values greater than 350 nM in H1781 cells and above 320 nM in the Ba/F3 model.</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Covalent Pyrrozolopyrimidine-Based Inhibitors <b>4–5</b> and Corresponding IC<sub>50</sub> and EC<sub>50</sub> Evaluations<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c00870/20201019/images/medium/jm0c00870_0009.gif" alt="" id="GRAPHIC-d7e581-autogenerated" /></img><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="char" char="±" /></col><col align="left" /></col><col align="char" char="±" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="2" align="center" char="±">Her2 HTRF IC<sub>50</sub> [nM]</th><th class="rowsep1 colsep0" colspan="2" align="center" char="±">Her2 CTG EC<sub>50</sub> [nM]</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">Cpd</th><th class="colsep0 rowsep0" align="center">R</th><th class="colsep0 rowsep0" align="center" char="±">wt</th><th class="colsep0 rowsep0" align="center">insYVMA</th><th class="colsep0 rowsep0" align="center" char="±">H1781 (GdelinsVC)</th><th class="colsep0 rowsep0" align="center">Ba/F3 (insYVMA)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>4a</b><a class="ref internalNav" href="#t1fn3" aria-label="c">c</a></td><td class="colsep0 rowsep0" align="left">Me</td><td class="colsep0 rowsep0" align="char" char="±">146 ± 84</td><td class="colsep0 rowsep0" align="left">36 ± 8</td><td class="colsep0 rowsep0" align="char" char="±">1231 ± 81</td><td class="colsep0 rowsep0" align="left">3302 ± 547</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>4b</b><a class="ref internalNav" href="#t1fn3" aria-label="c">c</a></td><td class="colsep0 rowsep0" align="left">Et</td><td class="colsep0 rowsep0" align="char" char="±">84 ± 21</td><td class="colsep0 rowsep0" align="left">59 ± 22</td><td class="colsep0 rowsep0" align="char" char="±">2339 ± 292</td><td class="colsep0 rowsep0" align="left">2813 ± 1035</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>4c</b><a class="ref internalNav" href="#t1fn3" aria-label="c">c</a></td><td class="colsep0 rowsep0" align="left"><i>i</i>Pr</td><td class="colsep0 rowsep0" align="char" char="±">592 ± 99</td><td class="colsep0 rowsep0" align="left">418 ± 82</td><td class="colsep0 rowsep0" align="char" char="±">2048 ± 214</td><td class="colsep0 rowsep0" align="left">3602 ± 1061</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>4d</b><a class="ref internalNav" href="#t1fn3" aria-label="c">c</a></td><td class="colsep0 rowsep0" align="left"><i>i</i>Bu</td><td class="colsep0 rowsep0" align="char" char="±">321 ± 138</td><td class="colsep0 rowsep0" align="left">33 ± 5</td><td class="colsep0 rowsep0" align="char" char="±">988 ± 44</td><td class="colsep0 rowsep0" align="left">2416 ± 386</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>5a</b><a class="ref internalNav" href="#t1fn3" aria-label="c">c</a></td><td class="colsep0 rowsep0" align="left">Me</td><td class="colsep0 rowsep0" align="char" char="±">50 ± 3</td><td class="colsep0 rowsep0" align="left">11 ± 2</td><td class="colsep0 rowsep0" align="char" char="±">357 ± 92</td><td class="colsep0 rowsep0" align="left">910 ± 71</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>5b</b><a class="ref internalNav" href="#t1fn3" aria-label="c">c</a></td><td class="colsep0 rowsep0" align="left">Et</td><td class="colsep0 rowsep0" align="char" char="±">14 ± 1</td><td class="colsep0 rowsep0" align="left">9.2<a class="ref internalNav" href="#t1fn2" aria-label="b">b</a></td><td class="colsep0 rowsep0" align="char" char="±">417 ± 60</td><td class="colsep0 rowsep0" align="left">327 ± 160</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>5c</b><a class="ref internalNav" href="#t1fn3" aria-label="c">c</a></td><td class="colsep0 rowsep0" align="left"><i>i</i>Pr</td><td class="colsep0 rowsep0" align="char" char="±">21 ± 3</td><td class="colsep0 rowsep0" align="left">18 ± 7</td><td class="colsep0 rowsep0" align="char" char="±">1175 ± 436</td><td class="colsep0 rowsep0" align="left">1445 ± 430</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>5d</b><a class="ref internalNav" href="#t1fn3" aria-label="c">c</a></td><td class="colsep0 rowsep0" align="left"><i>i</i>Bu</td><td class="colsep0 rowsep0" align="char" char="±">26 ± 14</td><td class="colsep0 rowsep0" align="left">3.5 ± 0.6</td><td class="colsep0 rowsep0" align="char" char="±">2310 ± 654</td><td class="colsep0 rowsep0" align="left">1117 ± 288</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>neratinib</b></td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="char" char="±">9.7 ± 0.3</td><td class="colsep0 rowsep0" align="left">20 ± 6</td><td class="colsep0 rowsep0" align="char" char="±">148 ± 30</td><td class="colsep0 rowsep0" align="left"><14</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>lapatinib</b></td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="char" char="±">2.8 ± 0.3</td><td class="colsep0 rowsep0" align="left">54 ± 14</td><td class="colsep0 rowsep0" align="char" char="±">3941 ± 505</td><td class="colsep0 rowsep0" align="left">2761 ± 348</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>TAK-285</b></td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="char" char="±">9.1 ± 2.4</td><td class="colsep0 rowsep0" align="left">311 ± 3</td><td class="colsep0 rowsep0" align="char" char="±">3984 ± 917</td><td class="colsep0 rowsep0" align="left">3218 ± 518</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>afatinib</b></td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="char" char="±">7.7 ± 4.5</td><td class="colsep0 rowsep0" align="left">2.1 ± 0.2</td><td class="colsep0 rowsep0" align="char" char="±">151 ± 46</td><td class="colsep0 rowsep0" align="left">58 ± 16</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>poziotinib</b></td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="char" char="±">12 ± 1</td><td class="colsep0 rowsep0" align="left">0.3 ± 0.2</td><td class="colsep0 rowsep0" align="char" char="±">105 ± 62</td><td class="colsep0 rowsep0" align="left"><14</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>gefitinib</b></td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="char" char="±">97 ± 10</td><td class="colsep0 rowsep0" align="left">15 ± 4</td><td class="colsep0 rowsep0" align="char" char="±">1794 ± 698</td><td class="colsep0 rowsep0" align="left">2882 ± 1230</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>osimertinib</b></td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="char" char="±">4.8 ± 0.1</td><td class="colsep0 rowsep0" align="left">45 ± 5</td><td class="colsep0 rowsep0" align="char" char="±">808 ± 151</td><td class="colsep0 rowsep0" align="left">1389 ± 452</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup><sup>a</sup></sup><p class="last">Values are the mean ± standard deviation (SD) of three independent measurements in duplicates.</p></div><div class="footnote" id="t1fn2"><sup><sup>b</sup></sup><p class="last">Single measurement.</p></div><div class="footnote" id="t1fn3"><sup><sup>c</sup></sup><p class="last">Compounds <b>4–5</b> were previously reported as EGFR inhibitors.<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a></p></div></div></div></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i6" class="anchor-spacer"></div><h3 class="article-section__title" id="_i6"> Compound Screening To Identify Novel Starting Points and the Subsequent Inhibitor Design</h3><div class="NLM_p">To identify novel starting points for the development of Her2 inhibitors with improved activity on the cellular level, we performed a screening campaign within compound libraries that were enriched with kinase-directed inhibitors. The screen comprised four arms, testing 1635 compounds in a biochemical assay at concentrations of 1 and 10 μM against Her2-wt and Her2-insYVMA (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>A). <i>Z</i>′ values, commonly referred to as a measure for the assay robustness, were only moderate (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00870/suppl_file/jm0c00870_si_001.pdf" class="ext-link">Figure S2</a>), especially for assay plate #4. Our laboratory now employs a semiautomated screening unit Robotics-Assisted Screening Platform for Efficient Ligand Discovery (RASPELD)<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> for screening and compound optimization purposes, which provides improved assay robustness and will be utilized in future screening projects. Compounds that displayed ≤50% residual kinase activity at 1 μM and/or ≤30% residual activity at 10 μM were considered as hits. Of note, some hits within one arm of the screening campaign did not meet the hit criteria in other arms (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>B and <a href="/doi/suppl/10.1021/acs.jmedchem.0c00870/suppl_file/jm0c00870_si_001.pdf" class="ext-link">S3</a>) but were taken into account in the subsequent hit validation. Hits detected on assay plate #4 were particularly carefully validated in this step. In addition, structurally similar compounds were included in the validation process and IC<sub>50</sub> values were recorded for total 132 compounds, resulting in a hit rate of 8.1% which is reasonable for a kinase-focused compound library. <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a> shows the compounds clustered by chemical similarity using SARanea,<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> with color coding with respect to the determined IC<sub>50</sub> values against Her2-insYVMA. Representative compound structures <b>i–viii</b> for clusters that contained valid hits are shown (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>) and an extended compound panel with the respective values is summarized in <a href="/doi/suppl/10.1021/acs.jmedchem.0c00870/suppl_file/jm0c00870_si_001.pdf" class="ext-link">Table S1</a>. Several compounds (<b>i–v</b>) containing hydrophobic moieties meant to occupy a back-pocket that emerges in the inactive kinase conformation in ErbB members were identified, which is not surprising because these moieties are widespread in established Her2-targeted inhibitors such as neratinib (<b>2</b>) or lapatinib (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>B, highlighted in blue). Interestingly, hybrid compound <b>vi</b> (type II) revealed an IC<sub>50</sub> of 20 nM and the allosteric inhibitor <b>vii</b> (type III), originally developed to target the p38 MAP kinase,<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> exhibited an IC<sub>50</sub> value in the low micromolar range against the YVMA insertion mutated form of Her2. Considering the recently described binding mode of the allosteric EGFR inhibitor EAI001 (<b>6</b>),<a onclick="showRef(event, 'ref25 ref26'); return false;" href="javascript:void(0);" class="ref ref25 ref26">(25,26)</a> which binds comparably as the back-pocket moieties of lapatinib or neratinib, we reasoned that the herein identified inhibitors <b>vi–viii</b> might occupy an allosteric site in EGFR and Her2 in a similar fashion (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>A,B). We therefore set out to design, synthesize, and evaluate covalent type II inhibitors based on the promising pyrrolopyrimidine core scaffold which exhibited hydrophobic elements that would target the allosteric back-pocket to yield novel inhibitors of Her2 (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>C).</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c00870/20201019/images/medium/jm0c00870_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c00870/20201019/images/large/jm0c00870_0003.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Initial focused biochemical screening against Her2 with 1635 compounds and neratinib as a reference compound. (A) Screen comprised four arms: Her2-wt and Her2-insYVMA were each tested at concentrations of 1 and 10 μM. (B) Compounds that displayed ≤50% residual kinase activity at 1 μM and/or ≤30% residual activity at 10 μM were considered as hits (see <a href="/doi/suppl/10.1021/acs.jmedchem.0c00870/suppl_file/jm0c00870_si_001.pdf" class="ext-link">Figure S3</a>). Some hits within one arm of the screening campaign did not meet the hit criteria in other arms but were taken into account in the subsequent hit validation.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c00870/20201019/images/large/jm0c00870_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00870&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c00870/20201019/images/medium/jm0c00870_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c00870/20201019/images/large/jm0c00870_0004.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. IC<sub>50</sub> values were recorded for 132 compounds and were clustered by chemical similarity using SARanea. The color code indicates the determined IC<sub>50</sub> values against Her2-insYVMA. Representative compound structures <b>i–viii</b> for clusters that contained valid hits are shown.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c00870/20201019/images/large/jm0c00870_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00870&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c00870/20201019/images/medium/jm0c00870_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c00870/20201019/images/large/jm0c00870_0005.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. (A) Alignment of X-ray complex crystal structures of compounds bearing structural motifs found in the focused screening campaign: EAI001 bound to EGFR (green; PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5D41">5D41</a>), <b>vii</b> bound to p38 MAPK (red; PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3LFB">3LFB</a>), and lapatinib bound to EGFR (yellow; PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1XKK">1XKK</a>). (B) Chemical structures of EAI001 (green), <b>vii</b> (red), and lapatinib (yellow). (C) Design of Her2 inhibitors based on the covalent pyrrolopyrimidine core equipped with structural motifs found in the focused screen.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c00870/20201019/images/large/jm0c00870_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00870&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i7" class="anchor-spacer"></div><h3 class="article-section__title" id="_i7"> Design, Synthesis, and Evaluation of the First Set of Pyrrolopyrimidines</h3><div class="NLM_p">We designed hybrid compounds consisting of the pyrrolopyrimidine core equipped with a reactive warhead for covalently targeting Her2 and hydrophobic moieties in the 4-position, which were identified in a focused screening campaign. The corresponding compounds <b>7a</b>,<b>b</b> and <b>7h–m</b> bearing moieties such as <i>N</i>-benzyl-indazoles or indoles, sterically demanding phthalimide substituents, or back-pocket elements of established TKIs, such as lapatinib (<b>3</b>) and neratinib (<b>2</b>), were subsequently synthesized. The synthesis scheme started with the iodination of commercially available 4-chloro-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidine (<b>8</b>) in the 5-position and subsequent [2-(trimethylsilyl)ethoxy]methyl (SEM)-protection of the cyclic amine. The resulting central building block <b>10</b> was utilized in a Pd-catalyzed Suzuki coupling with 3-nitrophenyl boronic acid with simultaneous hydroxylation of the 4-position to yield intermediate <b>11</b>. Bromination followed by a secondary Suzuki coupling in the 6-position resulted in compound <b>12</b>. Chlorination of the 4-position followed by nitro group reduction and subsequent acylation of the amine group with acryloyl chloride afforded intermediate <b>13</b>. Finally, nucleophilic substitution with different back-pocket anilines in the 4-position and SEM deprotection led to the desired series of covalent inhibitors <b>7</b> (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>).</div><figure id="sch1" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c00870/20201019/images/medium/jm0c00870_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c00870/20201019/images/large/jm0c00870_0008.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of Pyrrolopyrimidine-Based Inhibitors <b>7</b>, <b>14</b>, <b>18–21</b>, and <b>25</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c00870/20201019/images/large/jm0c00870_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00870&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (i) <i>N</i>-iodosuccinimide, DMF, rt; (ii) SEM-Cl, NaH, THF, 0 °C, then rt; (iii) (3-nitrophenyl)boronic acid, Pd(PPh<sub>3</sub>)<sub>4</sub>, K<sub>2</sub>CO<sub>3</sub>, MeCN/H<sub>2</sub>O (2:1), 150 °C, 90 min, μW; (iv) <i>N</i>-bromosuccinimide, DMF, rt; (v) 1-methyl-4-[4-(4,4,5,5-tetramethyl-1,3,2-dioxaborolan-2-yl)phenyl]piperazine, Pd(PPh<sub>3</sub>)<sub>4</sub>, K<sub>2</sub>CO<sub>3</sub>, DME/H<sub>2</sub>O (3:1), 150 °C, 90 min, μW; (vi) benzyltriethylammonium chloride, <i>N</i>,<i>N</i>-dimethylaniline, POCl<sub>3</sub>, MeCN, 85 °C; (vii) iron powder, NH<sub>4</sub>Cl, EtOH:H<sub>2</sub>O (9:1), 60 °C; (viii) acryloyl chloride, <i>N</i>,<i>N</i>-diisopropylethylamine (DIPEA), THF, 0 °C then rt; (ix) RNH<sub>2</sub>, HCl/dioxane (4 M), <sup><i>n</i></sup>BuOH, 80 °C; (x) TFA/ dichloromethane (DCM) (1:3), rt, then 1 M KOH, THF, rt; (xi) (3-nitrophenyl)boronic acid, Pd(PPh<sub>3</sub>)<sub>4</sub>, K<sub>2</sub>CO<sub>3</sub>, MeCN/H<sub>2</sub>O (2:1), 80 °C; (xii) (4-fluoro-3-nitrophenyl)boronic acid, Na<sub>2</sub>CO<sub>3</sub>, Pd(PPh<sub>3</sub>)<sub>4</sub>, dioxane/H<sub>2</sub>O (3:1), Ar atm, 80 °C; (xiii) HOR′, NaH, dry THF, rt; (xiv) K<sub>2</sub>CO<sub>3</sub>, MeOH, rt.</p></p></figure><div class="NLM_p">The resulting pyrrolopyrimidine-based inhibitors were tested in an activity-based biochemical assay to determine the inhibitory activity on wild-type and the Her2-insYVMA mutant variant and further, on the cellular level, utilizing the CellTiter-Glo assay (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>). Compounds <b>7a</b>,<b>b</b> and <b>7h–m</b> showed high potency on the mutant kinase Her2-insYVMA with IC<sub>50</sub> values below 20 nM. Wild-type Her2 generally gave higher IC<sub>50</sub> values in our assay, and compounds <b>7</b> showed half-maximal inhibition below 55 nM (<b>7a</b> being the only exception with 95 nM). Heteroatom-rich compounds <b>7b</b> and <b>7i</b> were more potent than their respective lipophilic counterparts <b>7a</b> and <b>7h</b>. This trend translated into cellular inhibition because H1781 cells exhibited EC<sub>50</sub> values of >30,000 nM and 252 nM for <b>7a</b> and <b>7b</b>, respectively. Furthermore, the molecules <b>7j</b>, <b>7k</b>, and <b>7l</b>, displayed only moderate cellular inhibition (2629, 699, and 334 nM, respectively). The ester-bridged inhibitor <b>7l</b> showed cellular inhibition properties; however, these structural motifs are prone to hydrolyzation and might have resulted in the <i>para</i>-phenol <b>7m</b>, which represents a motif of pan-assay interference compounds (PAINS).<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> Together, these results demonstrate that the introduction of back-pocket elements is well tolerated but that they only partially and moderately increase the potency as compared to the initial compounds <b>5a–d</b>.</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Covalent Pyrrolopyrimidine-Based Inhibitors <b>7</b> and <b>14</b> and Corresponding IC<sub>50</sub> and EC<sub>50</sub> Evaluations<a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c00870/20201019/images/medium/jm0c00870_0010.gif" alt="" id="GRAPHIC-d7e1442-autogenerated" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c00870/20201019/images/medium/jm0c00870_0011.gif" alt="" id="gr8" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup><sup>a</sup></sup><p class="last">Values are the mean ± SD of three independent measurements performed in duplicate.</p></div></div><div></div></div><div class="NLM_p">Along with the observed impact of lipophilicity, we assumed the high molecular weight of compounds <b>7</b> of about 670–800 Da (excluding <b>7m</b>) might result in poor permeability through cellular membranes. It was assumed that in the presence of a back-pocket element, the bulky group in the 6-position could be removed, which led us to the design and synthesis of compounds <b>14</b> (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>), which resulted in a reduced molecular weight of 450–620 Da. Here, we used building block <b>10</b> for the Pd-catalyzed Suzuki coupling with 3-nitrophenyl boronic acid, followed by the nitro group reduction to introduce the Michael acceptor using acryloyl chloride. As previously described, the so-obtained intermediate <b>17</b> underwent nucleophilic substitution with different back-pocket amines in the 4-position followed by SEM deprotection with trifluoroacetic acid (TFA) in dichloromethane to afford the desired series of covalent inhibitors <b>14</b>.</div><div class="NLM_p">When evaluated in the biochemical assays and compared to compounds <b>7</b>, compounds <b>14</b> showed more pronounced biochemical activity against wild-type Her2 and reduced IC<sub>50</sub> values on insYVMA mutant Her2 (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>). However, compounds <b>14</b> exhibited an intense gain in cellular potency toward H1781- and Her2-mutated Ba/F3 cells. While in the insYVMA Ba/F3 model system moderate EC<sub>50</sub> values of 2216 and 285 nM were observed for <b>7a</b> and 7<b>b</b>, remarkable EC<sub>50</sub> values of 135 and 83 nM were determined for their counterparts <b>14a</b> and <b>14b</b>, respectively. Compounds <b>14b</b> and <b>14d</b> set the lead and revealed EC<sub>50</sub> values below 200 nM in both cell lines. Reduced solubility and potency due to the removal of the piperazine moiety might be counteracted by an increase in cell membrane penetration gained by lower molecular weight, resulting in the high cellular potency observed for compounds <b>14</b>. The previously described observation of a positive effect mediated by the introduction of heteroatoms was confirmed within this compound series, when comparing, for example, inhibitors <b>14a</b> and <b>14b</b> or <b>14c</b> and <b>14d</b>. Poor cellular inhibition was observed with nonpolar inhibitors <b>14f–g</b> and <b>14j</b> (>3000, >3000, and 6390 nM, respectively, with H1781), indicating that polarity and solubility are key parameters affecting biochemical and cellular inhibition properties.</div><div class="NLM_p">Because the neratinib back-pocket substituent bearing compound <b>14b</b> exhibited the most promising biochemical and cellular inhibitory properties, we evaluated it further in western blot studies to determine its effects on the phosphorylation of Her2 and the signaling pathway. We included <b>14h</b> with an isoindoline-1,3-dione moiety, a slightly less potent compound, in these studies. Treatment of H1781 cells with <b>14b</b> (100 nM) and <b>14h</b> (100–500 nM) resulted in a notable dose-dependent reduction of Her2 activity and downstream phosphorylation. The inhibitory doses of these compounds were in the same range of the EC<sub>50</sub> values of 161 and 419 nM, respectively, determined in the CellTiter-Glo assays. These studies revealed that compound <b>14b</b> exhibited comparable potency as the clinically used inhibitor neratinib (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a> and <a href="/doi/suppl/10.1021/acs.jmedchem.0c00870/suppl_file/jm0c00870_si_001.pdf" class="ext-link">S4</a>).</div><figure id="fig5" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c00870/20201019/images/medium/jm0c00870_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c00870/20201019/images/large/jm0c00870_0006.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Western blot analysis of Her2 and downstream cascade phosphorylation inhibition by representative compounds in H1781 cells.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c00870/20201019/images/large/jm0c00870_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00870&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">We set out to solve the crystal structure of selected inhibitors in complex with Her2. Our attempts were, however, not successful to date. The challenge to obtain diffracting complex crystals of Her2 is also reflected by the limited number of only two structures of the Her2 wild-type kinase domain published in the Protein Data Bank (PDB; PDB IDs: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1PP0">1PP0</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3RCD">3RCD</a>). To our delight, compound <b>14d</b> gave diffraction-quality cocrystals within our EGFR crystallization program, and the respective structure could be refined to a resolution of 2.0 Å. EGFR is a close relative of Her2 within the ErbB receptor tyrosine kinase family and shares high homology within the binding site (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00870/suppl_file/jm0c00870_si_001.pdf" class="ext-link">Figure S1</a>). We, therefore, believe that the structure obtained can serve as an appropriate surrogate. This structure revealed the pyrrolopyrimidine core to form two hydrogen bonds with Met793 within the hinge region and a covalent bond with the reactive Cys797 in T790M variant of EGFR. The <i>N</i>-benzyl-indazole moiety occupies a pocket between the Met790 gatekeeper, the DFG motif, and the helix αC in its inactive state. The acrylamide was found to form a water-mediated contact with the aniline-NH of the back-pocket element (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>A, <a href="/doi/suppl/10.1021/acs.jmedchem.0c00870/suppl_file/jm0c00870_si_001.pdf" class="ext-link">S1</a> and <a href="/doi/suppl/10.1021/acs.jmedchem.0c00870/suppl_file/jm0c00870_si_001.pdf" class="ext-link">S5A,B</a>). Thereby, we were able to confirm the anticipated binding mode within ErbB family kinases.</div><figure id="fig6" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c00870/20201019/images/medium/jm0c00870_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c00870/20201019/images/large/jm0c00870_0007.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Cocrystal structures of covalent pyrrolopyrimidines in complex with the EGFR-T790M/V948R mutant. Diagrams of the experimental electron densities of (A) <b>14d</b>/EGFR-T790M/V948R at 2.0 Å, chain A (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TFU">6TFU</a>), (B) <b>18b</b>/EGFR-T790M/V948R at 1.5 Å, chain A (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TFV">6TFV</a>), (C) <b>18c</b>/EGFR-T790M/V948R at 1.7 Å, chain A (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TFY">6TFY</a>), (D) <b>18d</b>/EGFR-T790M/V948R at 2.0 Å, chain A (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TFW">6TFW</a>), (E) <b>19</b>/EGFR-T790M/V948R at 1.8 Å, chain B (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TFZ">6TFZ</a>), (F) <b>21a</b>/EGFR-T790M/V948R at 1.5 Å, chain B (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TG0">6TG0</a>), and (G) <b>21b</b>/EGFR-T790M/V948R at 1.6 Å, chain B (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TG1">6TG1</a>); 2Fo-Fc map contoured at an root-mean-square deviation of 1.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c00870/20201019/images/large/jm0c00870_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00870&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">Altogether, 18 compounds were synthesized, exhibiting different back-pocket substituents in the 4-position of the pyrrolopyrimidine core. Compounds <b>7</b> bearing a methylpiperazine solubilizing element showed a reasonable biochemical activity on Her2-insYVMA but only moderate cellular activity. The 6-position unsubstituted inhibitors <b>14</b> were successfully optimized to exhibit remarkable EC<sub>50</sub> values of below 200 nM against H1781 (as exemplified with compounds <b>14b</b> and <b>14d–e</b>) because of reduced molecular weight accompanied with improved cell permeability. Thereby, cellular activity was improved as compared to the initial starting point <b>5a–d</b>, as well as the neratinib reference compound.</div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i10" class="anchor-spacer"></div><h3 class="article-section__title" id="_i10"> Improving Solubility and the Inhibitory Effect of Pyrrolopyrimidines against Her2-Mutated Cell Lines</h3><div class="NLM_p">During the previous optimization efforts, the molecular weight was successfully decreased by the removal of the solubilizing group. However, we set out to restore the solubility for improved inhibitory properties. We revisited the obtained complex crystal structure of <b>14d</b> to identify the phenyl linker exhibiting the acrylamide as solvent-exposed and well suited for the introduction of solubilizing groups (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>A). Because the reduced molecular weight should be maintained, small alkyl chains were introduced in the ortho-position of the Michael acceptor exhibiting alcohol, amine, or ether functionalities, resulting in compounds <b>18</b>–<b>21</b>. Therefore, the synthetic route was adapted (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>) and the building block <b>10</b> was utilized in a Pd-catalyzed Suzuki coupling with 4-fluoro-3-nitrophenyl boronic acid to yield intermediate <b>22</b>, which underwent nucleophilic substitution with different back-pocket amines in the 4-position (<b>23</b>). Nucleophilic aromatic substitution and following nitro-to-amine reduction afforded intermediate <b>24</b>. Finally, acylation of the amine group with acryloyl chloride and SEM deprotection led to the desired series of covalent inhibitors <b>18–21</b>.</div><div class="NLM_p">First, pyrrolopyrimidine-based inhibitors <b>18a–d</b> bearing the neratinib back-pocket element in the 4-position and an ether moiety, alcohol substituents with different linker lengths, or an amine linker as solubilizing groups were tested (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>). Methyl ether-bearing inhibitor <b>18a</b> showed reduced activity in biochemical and cellular settings. <b>18b</b> and <b>18c</b>, which exhibited C2- and C3-alkyl chains with terminal hydroxy groups, displayed biochemical activities below 10 nM and cellular potency below 270 nM and below 85 nM on H1781 and Ba/F3 cell lines, respectively. Compound <b>18d</b> equipped with a tertiary amine moiety showed similarly intense inhibition of the Her2-insYVMA kinase with an IC<sub>50</sub> of 7.4 nM and cellular potencies of 164 nM and 120 nM on the Her2-mutated cell lines. The introduction of other back-pocket elements reduced the potency of respective compounds <b>19–21</b> and, thus, inhibitors <b>18</b> represented the most potent inhibitors within this series and revealed increased biochemical potency while maintaining cellular efficacy as compared to compounds <b>14b</b> and <b>14d–e</b>. To confirm these results, <b>18d</b> and <b>19</b> were included in western blot experiments. In line with the results from CellTiter-Glo assays, treatment of H1781 cells revealed inhibition of Her2 phosphorylation in the concentration range between 100 and 500 nM (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>). To test our hypothesis that the methylpiperazine solubilizing group or a hydrophobic back-pocket element is needed to achieve potency, inhibitors <b>25</b> without either motif were synthesized (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>). Further biochemical and cellular evaluation revealed IC<sub>50</sub> values in the micromolar range (<b>25b</b> being the only exception with IC<sub>50</sub> values of about 750 nM) and cellular EC<sub>50</sub> levels >3000 nM (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00870/suppl_file/jm0c00870_si_001.pdf" class="ext-link">Table S2</a>). These finding suggest that these inhibitors lack a moiety that properly occupies the binding site.</div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. Covalent Pyrrolopyrimidine-Based Inhibitors <b>18–21</b> and Corresponding IC<sub>50</sub> and EC<sub>50</sub> Evaluations<a class="ref internalNav" href="#t3fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c00870/20201019/images/medium/jm0c00870_0012.gif" alt="" id="GRAPHIC-d7e1805-autogenerated" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c00870/20201019/images/medium/jm0c00870_0013.gif" alt="" id="gr9" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t3fn1"><div class="footnote" id="t3fn1"><sup><sup>a</sup></sup><p class="last">Values are the mean ± SD of three independent measurements performed in duplicate.</p></div></div><div></div></div><div class="NLM_p last">In addition, we obtained a second set of six complex crystal structures with EGFR-T790M (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>, <a href="/doi/suppl/10.1021/acs.jmedchem.0c00870/suppl_file/jm0c00870_si_001.pdf" class="ext-link">S1</a> and <a href="/doi/suppl/10.1021/acs.jmedchem.0c00870/suppl_file/jm0c00870_si_001.pdf" class="ext-link">S5</a>). Compounds <b>18b</b>, <b>18c</b>, <b>18d</b>, <b>19</b>, <b>21a</b>, and <b>21b</b> gave high-resolution datasets (1.5–2.0 Å) and allowed insight into their binding modes. These structures revealed (i) the effect of heteroatoms within the back-pocket element, which were found to make polar contacts with the side chain of Thr854 and the carbonyl of Asp855 within the DFG motif in the case of compounds <b>18</b>, while the isoindoline-dione moiety of compounds <b>21</b> formed an additional interaction with Phe856. (ii) The glycol-based solubilizing groups tended to form a bridged interaction with a water molecule involving the ether oxygen and the terminal heteroatom. Thereby, these chains are constrained and could be resolved in these crystal structures. Interestingly, <b>19</b> and <b>21a</b> showed an interaction of the C2-chain alcohol with one rotamer of Arg841. Together, these results revealed the rational design for introduction of solubilizing groups in this position; however, future compound optimizations should involve nonether bridged groups to avoid water contacts and foster protein contacts. (iii) Back-pocket elements of compounds <b>18c</b> and <b>19</b> could bind with different rotamers. Interestingly, the neratinib-derived back-pocket element of <b>18c</b> was found in an additional orientation that allows interaction with the catalytic Lys745, and both rotamers together resembled the assumed binding mode of compound <b>14j</b>. Therefore, we propose that back-pocket elements with three aromatic rings should be equipped with additional heteroatoms in future medicinal chemistry approaches to target the inactive conformation of ErbB kinases.</div></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i12" class="anchor-spacer"></div><h3 class="article-section__title" id="_i12"> Mass Spectrometry-Based and Kinetic Characterization of Covalent Bond Formation</h3><div class="NLM_p">We aimed to design compounds capable of covalently alkylating Cys805 in Her2. X-ray crystal structures indicated a covalent mode of action in EGFR by clear electron density corresponding to the inhibitor–protein adduct formed with Cys797. We validated this finding by incubation of representative inhibitors with mutant EGFR, which resulted in a mass shift corresponding to single labeling with the inhibitors. The analogous experiment was performed with the Her2 kinase domain and similarly confirmed the formation of a covalent adduct between the inhibitors and Her2 (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00870/suppl_file/jm0c00870_si_001.pdf" class="ext-link">Figure S6</a>).</div><div class="NLM_p">Of note, IC<sub>50</sub> values can only give an approximation of the potency of covalent inhibitors because these values depend on the preincubation time of the protein and inhibitor. Therefore, we investigated the time dependency of the IC<sub>50</sub> values in detail. Covalent bond formation relies on the initial reversible arrangement of the inhibitor in the protein binding site and the resulting proximity of the desired cysteine side chain with the reactive acrylamide linker. If this complex is given enough time to react, covalent binding can occur. These two characteristic steps can be described by determining the parameters <i>K</i><sub>i</sub> and <i>k</i><sub>inact</sub>, representing the reversible binding event and the rate of covalent bond formation, respectively. The newly synthesized inhibitors <b>14b</b>, <b>14h</b>, <b>18d</b>, and <b>19</b> were tested along with reference compounds afatinib and neratinib in insYVMA-mutated Her2 (<a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>). The reversible affinities represented by the inhibition constant <i>K</i><sub>i</sub> showed only minor differences within the panel of compounds ranging from 49 to 143 nM. In comparison, reference inhibitors afatinib and neratinib were demonstrated to bind with significantly higher affinity to Her2-insYVMA with <i>K</i><sub>i</sub> values of 0.76 and 6.3 nM, respectively. In contrast, the maximum potential rates of inactivation (<i>k</i><sub>inact</sub>) were 2.4-fold to 6.6-fold higher (0.132–0.212 min<sup>–1</sup>) for the compounds presented in this study as compared to afatinib and neratinib (0.054 and 0.032 min<sup>–1</sup>, respectively). This observation may hint toward an elevated intrinsic reactivity of the unsubstituted acrylamide (<b>14b</b>, <b>14h</b>, <b>18d</b>, and <b>19</b>) as opposed to the 4-(dimethylamino)-but-2-enamide warhead present in afatinib and neratinib. Alternatively, the orientation of the acrylamide group linked to the pyrrolopyrimidine scaffold may be favored over the spatial arrangement of the 4-(dimethylamino)-but-2-enamide warhead attached to the quinolin/quinazolin scaffold toward the nucleophilic Cys805. However, the superior <i>k</i><sub>inact</sub> values for compounds <b>14b</b>, <b>14h</b>, <b>18d</b>, and <b>19</b> cannot compensate for the comparably weaker reversible affinities, resulting in generally lower efficiencies of covalent bond formation as indicated by the ratio <i>k</i><sub>inact</sub>/<i>K</i><sub>i</sub>. Therefore, future efforts should be specifically directed toward the optimization of the reversible binding affinity of the compound class described herein while retaining the improved maximum potential rate of inactivation. To the best of our knowledge, this is the first evaluation of binding kinetics of covalent inhibitors with Her2 and the findings validate the IC<sub>50</sub> determinations as a valid approximation of inhibitory potency.</div><div class="NLM_table-wrap" id="tbl4"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 4. Kinetic Parameters <i>K</i><sub>i</sub>, <i>k</i><sub>inact</sub>, and <i>k</i><sub>inact</sub>/<i>K</i><sub>i</sub> Determined for Representative Inhibitors against Mutated Her2-insYVMA<a class="ref internalNav" href="#t4fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="char" char="±" /></col><col align="char" char="±" /></col><col align="char" char="±" /></col><col align="char" char="±" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">Cpd</th><th class="colsep0 rowsep0" align="center" char="±">HTRF IC<sub>50</sub> [nM]</th><th class="colsep0 rowsep0" align="center" char="±"><i>K</i><sub>i</sub> [nM]</th><th class="colsep0 rowsep0" align="center" char="±"><i>k</i><sub>inact</sub> [min<sup>–1</sup>]</th><th class="colsep0 rowsep0" align="center" char="±"><i>k</i><sub>inact</sub>/<i>K</i><sub>i</sub> [μM<sup>–1</sup> s<sup>–1</sup>]</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>14b</b></td><td class="colsep0 rowsep0" align="char" char="±">17 ± 3</td><td class="colsep0 rowsep0" align="char" char="±">54 ± 5</td><td class="colsep0 rowsep0" align="char" char="±">0.136 ± 0.027</td><td class="colsep0 rowsep0" align="char" char="±">0.042 ± 0.006</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>14h</b></td><td class="colsep0 rowsep0" align="char" char="±">97 ± 48</td><td class="colsep0 rowsep0" align="char" char="±">143 ± 22</td><td class="colsep0 rowsep0" align="char" char="±">0.132 ± 0.020</td><td class="colsep0 rowsep0" align="char" char="±">0.015 ± 0.001</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>18d</b></td><td class="colsep0 rowsep0" align="char" char="±">7.4 ± 1.2</td><td class="colsep0 rowsep0" align="char" char="±">49 ± 4</td><td class="colsep0 rowsep0" align="char" char="±">0.212 ± 0.031</td><td class="colsep0 rowsep0" align="char" char="±">0.073 ± 0.016</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>19</b></td><td class="colsep0 rowsep0" align="char" char="±">46 ± 2</td><td class="colsep0 rowsep0" align="char" char="±">67 ± 8</td><td class="colsep0 rowsep0" align="char" char="±">0.138 ± 0.036</td><td class="colsep0 rowsep0" align="char" char="±">0.034 ± 0.006</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>afatinib</b></td><td class="colsep0 rowsep0" align="char" char="±">2.1 ± 0.2</td><td class="colsep0 rowsep0" align="char" char="±">0.76 ± 0.24</td><td class="colsep0 rowsep0" align="char" char="±">0.054 ± 0.029</td><td class="colsep0 rowsep0" align="char" char="±">1.113 ± 0.426</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>neratinib</b></td><td class="colsep0 rowsep0" align="char" char="±">20 ± 6</td><td class="colsep0 rowsep0" align="char" char="±">6.3 ± 0.8</td><td class="colsep0 rowsep0" align="char" char="±">0.032 ± 0.013</td><td class="colsep0 rowsep0" align="char" char="±">0.081 ± 0.024</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t4fn1"><div class="footnote" id="t4fn1"><sup><sup>a</sup></sup><p class="last">Values are the mean ± SD of three independent measurements performed in duplicate.</p></div></div></div></div><div id="sec2_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i14" class="anchor-spacer"></div><h3 class="article-section__title" id="_i14"> Analysis of Selectivity over the ErbB Tyrosine Kinase Family Member EGFR</h3><div class="NLM_p">Given the high similarity within the ErbB kinases Her2 and EGFR (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00870/suppl_file/jm0c00870_si_001.pdf" class="ext-link">Figure S1</a>), we set out to analyze the off-target inhibitory potency of the pyrrolopyrimidine-based inhibitors. We, therefore, determined the 50% inhibitory concentration against the kinase activity of wild-type EGFR, as well as clinically relevant mutant variants L858R and L858R/T790M (LR/TM) in cell-free assays and against cell lines A431 and H1975 harboring wild-type and LR/TM-mutated EGFR, respectively (<a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">Table <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">5</a></a>). Biochemical analysis revealed intense inhibitory properties for key compounds <b>14b</b>, <b>14h</b>, <b>18d</b>, and <b>19</b> with IC<sub>50</sub> values of 0.07–0.4 nM against wild-type EGFR similar to the reference compounds neratinib, lapatinb, afatinib, and osimertinib, which showed IC<sub>50</sub>’s in the range of 0.01–0.2 nM. Of note, on the cellular level, pyrrolopyrimidine-based inhibitors <b>14b</b>, <b>14h</b>, and <b>19</b> revealed low potency in the micromolar range against A431 cells (EC<sub>50</sub>’s of 1.6–2.2 μM), while <b>18d</b> showed moderate potency (EC<sub>50</sub> = 0.9 μM). Interestingly, the threonine-to-methionine mutation of the gatekeeper residue (T790M) reduced the potency of all Her2-targeted inhibitors but affected <b>14h</b> harboring an isoindoline-dione residue to a great extent. Together, the compounds developed herein exhibited similar selectivity over the kinase family member EGFR, as compared to clinically used inhibitors.</div><div class="NLM_table-wrap" id="tbl5"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 5. IC<sub>50</sub> and EC<sub>50</sub> Evaluations for Representative Inhibitors against Wild-Type and Mutated EGFR<a class="ref internalNav" href="#t5fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="char" char="±" /></col><col align="char" char="±" /></col><col align="char" char="±" /></col><col align="char" char="±" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="3" align="center" char="±">EGFR HTRF IC<sub>50</sub> [nM]</th><th class="rowsep1 colsep0" colspan="2" align="center" char="±">EGFR CTG EC<sub>50</sub> [nM]</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">Cpd</th><th class="colsep0 rowsep0" align="center" char="±">wt</th><th class="colsep0 rowsep0" align="center" char="±">L858R</th><th class="colsep0 rowsep0" align="center" char="±">L858R/T790M</th><th class="colsep0 rowsep0" align="center" char="±">A431 (wt)</th><th class="colsep0 rowsep0" align="center">H1975 (LR/TM)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>14b</b></td><td class="colsep0 rowsep0" align="char" char="±">0.08 ± 0.03</td><td class="colsep0 rowsep0" align="char" char="±">0.1 ± 0.01</td><td class="colsep0 rowsep0" align="char" char="±">3.9 ± 3.6</td><td class="colsep0 rowsep0" align="char" char="±">1675 ± 284</td><td class="colsep0 rowsep0" align="left">739 ± 55</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>14h</b></td><td class="colsep0 rowsep0" align="char" char="±">0.4 ± 0.01</td><td class="colsep0 rowsep0" align="char" char="±">0.2 ± 0.2</td><td class="colsep0 rowsep0" align="char" char="±">33 ± 12</td><td class="colsep0 rowsep0" align="char" char="±">1570 ± 635</td><td class="colsep0 rowsep0" align="left">2664 ± 715</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>18d</b></td><td class="colsep0 rowsep0" align="char" char="±">0.09 ± 0.03</td><td class="colsep0 rowsep0" align="char" char="±">0.1 ± 0.04</td><td class="colsep0 rowsep0" align="char" char="±">2.0 ± 0.08</td><td class="colsep0 rowsep0" align="char" char="±">862 ± 206</td><td class="colsep0 rowsep0" align="left">807 ± 82</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>19</b></td><td class="colsep0 rowsep0" align="char" char="±">0.07 ± 0.01</td><td class="colsep0 rowsep0" align="char" char="±">0.1 ± 0.03</td><td class="colsep0 rowsep0" align="char" char="±">6.0 ± 0.2<a class="ref internalNav" href="#t5fn2" aria-label="b">b</a></td><td class="colsep0 rowsep0" align="char" char="±">2236 ± 1185</td><td class="colsep0 rowsep0" align="left">n.d.</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>neratinib</b></td><td class="colsep0 rowsep0" align="char" char="±">0.02 ± 0.009</td><td class="colsep0 rowsep0" align="char" char="±">0.02 ± 0.01</td><td class="colsep0 rowsep0" align="char" char="±">0.8 ± 0.16</td><td class="colsep0 rowsep0" align="char" char="±">10117 ± 8826</td><td class="colsep0 rowsep0" align="left"><14<a class="ref internalNav" href="#t5fn2" aria-label="b">b</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>lapatinib</b></td><td class="colsep0 rowsep0" align="char" char="±">0.06 ± 0.02</td><td class="colsep0 rowsep0" align="char" char="±">0.2 ± 0.01</td><td class="colsep0 rowsep0" align="char" char="±">24 ± 4<a class="ref internalNav" href="#t5fn2" aria-label="b">b</a></td><td class="colsep0 rowsep0" align="char" char="±">2685 ± 1165</td><td class="colsep0 rowsep0" align="left">14781 ± 4062</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>afatinib</b></td><td class="colsep0 rowsep0" align="char" char="±">0.01 ± 0.006</td><td class="colsep0 rowsep0" align="char" char="±">0.01 ± 0.001</td><td class="colsep0 rowsep0" align="char" char="±">0.3 ± 0.2</td><td class="colsep0 rowsep0" align="char" char="±">1401 ± 413</td><td class="colsep0 rowsep0" align="left">714 ± 153</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>osimertinib</b></td><td class="colsep0 rowsep0" align="char" char="±">0.2 ± 0.02</td><td class="colsep0 rowsep0" align="char" char="±">0.06 ± 0.01</td><td class="colsep0 rowsep0" align="char" char="±">0.02 ± 0.004</td><td class="colsep0 rowsep0" align="char" char="±">911 ± 419</td><td class="colsep0 rowsep0" align="left">22 ± 14</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t5fn1"><div class="footnote" id="t5fn1"><sup><sup>a</sup></sup><p class="last">Values are the mean ± SD of three independent measurements in duplicates.</p></div><div class="footnote" id="t5fn2"><sup><sup>b</sup></sup><p class="last">Double measurement. n.d. = not determined.</p></div></div></div></div><div id="sec2_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i16" class="anchor-spacer"></div><h3 class="article-section__title" id="_i16"> Physicochemical and Pharmacokinetic Analysis of Pyrrolopyrimidines</h3><div class="NLM_p">The high potency of the developed inhibitors prompted us to further analyze them regarding their <i>in silico</i> physicochemical and <i>in vitro</i> pharmacokinetic parameters. Representative inhibitors <b>14b</b>, <b>14h</b>, <b>18d</b>, and <b>19</b> were evaluated and compared to the references neratinib, lapatinib, afatinib, and poziotinib (<a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">Table <a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">6</a></a>). Of note, type II inhibitors such as neratinib or lapatinib exhibit increased molecular weight and lipophilicity (560–580 Da; clog <i>P</i> = 4.99–5.82) as compared to type I inhibitors afatinib or poziotinib (480–490 Da; clog <i>P</i> = 4.34–5.33). The type II inhibitors <b>18d</b> and <b>19</b> which resulted from compound optimization revealed a similar molecular weight (550–580 Da) but were more polar because of the inserted solubilizing groups (clog <i>P</i> = 4.32–4.55). The determined ligand-lipophilicity efficiency (LLE), which is a benchmark of the lipophilicity with respect to resulting potency in H1781 cells, was only slightly increased as compared to neratinib. The high lipophilicity of the compounds was accompanied with a low or moderate aqueous kinetic solubility (SolRank). Additionally, a TurbiSol assay, in which the aggregation with increasing compound concentration is measured via optical density, showed a good correlation compared with the SolRank assay, validating the results obtained. The developed compounds showed good stability in human plasma (>74% remaining) and low clearance rates in human liver microsomes (HLMs) (CL<sub>int</sub> = 28–66 μL min<sup>–1</sup> mg<sup>–1</sup>). However, they exhibited shortcomings with respect to stability in mouse plasma (<b>14b</b> with 49% and <b>18d</b> with 0.1% remaining) and in mouse liver microsomes (MLMs) (CL<sub>int</sub> = 125–870 μL min<sup>–1</sup> mg<sup>–1</sup>). <b>18d</b> showed pronounced clearance which might result from the dimethylamine solubilizing group, which is prone to demethylation reactions. Compound <b>14b</b> showed equal clearance rates as compared to the reference neratinib (28/125 and 42/126 μL min<sup>–1</sup> mg<sup>–1</sup> in HLM/MLM, respectively) because of the high structural similarity. Thus, it can be concluded that the back-pocket elements confer some degree of instability, which will be addressed during future optimizations. Next, intestinal drug absorption was examined using the Caco-2 cell permeability assay as a cellular model system. Both neratinib and lapatinib showed low cellular apical (A) to basolateral (B) permeability with 0.09 to 0.7 × 10<sup>–6</sup> cm s<sup>–1</sup>, predicting a poor oral absorption. Of note, the herein developed compounds showed improved absorption properties with A–B rates of 0.5–2.1 × 10<sup>–6</sup> cm s<sup>–1</sup> and, in particular, <b>14b</b> exhibited a high influx (A–B) rate and a low efflux (B–A) rate, resulting in a favorable ratio of 0.3, indicating superior oral absorption as compared to the clinically used inhibitors, with poziotinib being the only exception. Altogether, compound <b>14b</b> showed a profile comparable with neratinib and established the pyrrolopyrimidine core as suitable for covalent inhibition of Her2.</div><div class="NLM_table-wrap" id="tbl6"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 6. Physicochemical and <i>in Vitro</i> Pharmacokinetic Parameters for <b>14b</b>, <b>14h</b>, <b>18d</b>, <b>19</b> and Neratinib, Lapatinib, Afatinib, and Poziotinib</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">pharmacokinetic parameter human/mouse</th><th class="colsep0 rowsep0" align="center">neratinib</th><th class="colsep0 rowsep0" align="center">lapatinib</th><th class="colsep0 rowsep0" align="center">afatinib</th><th class="colsep0 rowsep0" align="center">poziotinib</th><th class="colsep0 rowsep0" align="center"><b>14b</b></th><th class="colsep0 rowsep0" align="center"><b>14h</b></th><th class="colsep0 rowsep0" align="center"><b>18d</b></th><th class="colsep0 rowsep0" align="center"><b>19</b></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">MW</td><td class="colsep0 rowsep0" align="left">557.0</td><td class="colsep0 rowsep0" align="left">581.0</td><td class="colsep0 rowsep0" align="left">485.9</td><td class="colsep0 rowsep0" align="left">491.3</td><td class="colsep0 rowsep0" align="left">496.9</td><td class="colsep0 rowsep0" align="left">514.5</td><td class="colsep0 rowsep0" align="left">584.0</td><td class="colsep0 rowsep0" align="left">545.6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">clog <i>P</i><a class="ref internalNav" href="#t6fn1" aria-label="a">a</a></td><td class="colsep0 rowsep0" align="left">4.99</td><td class="colsep0 rowsep0" align="left">5.82</td><td class="colsep0 rowsep0" align="left">4.34</td><td class="colsep0 rowsep0" align="left">5.33</td><td class="colsep0 rowsep0" align="left">4.97</td><td class="colsep0 rowsep0" align="left">5.35</td><td class="colsep0 rowsep0" align="left">4.55</td><td class="colsep0 rowsep0" align="left">4.32</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">LLE<a class="ref internalNav" href="#t6fn2" aria-label="b">b</a></td><td class="colsep0 rowsep0" align="left">1.84</td><td class="colsep0 rowsep0" align="left">–0.42</td><td class="colsep0 rowsep0" align="left">2.48</td><td class="colsep0 rowsep0" align="left">1.65</td><td class="colsep0 rowsep0" align="left">1.82</td><td class="colsep0 rowsep0" align="left">1.03</td><td class="colsep0 rowsep0" align="left">2.24</td><td class="colsep0 rowsep0" align="left">1.97</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">SolRank [μM]</td><td class="colsep0 rowsep0" align="left">1.20</td><td class="colsep0 rowsep0" align="left">4.10</td><td class="colsep0 rowsep0" align="left">281.16</td><td class="colsep0 rowsep0" align="left">39.34</td><td class="colsep0 rowsep0" align="left">n.d.</td><td class="colsep0 rowsep0" align="left">8.30</td><td class="colsep0 rowsep0" align="left">60.60</td><td class="colsep0 rowsep0" align="left">16.80</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">turbidimetric solubility range [μM]</td><td class="colsep0 rowsep0" align="left">>10</td><td class="colsep0 rowsep0" align="left">3–10</td><td class="colsep0 rowsep0" align="left">n.d.</td><td class="colsep0 rowsep0" align="left">n.d.</td><td class="colsep0 rowsep0" align="left">30–100</td><td class="colsep0 rowsep0" align="left">∼30</td><td class="colsep0 rowsep0" align="left">∼100</td><td class="colsep0 rowsep0" align="left">30–100</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">plasma Stability [% remain]</td><td class="colsep0 rowsep0" align="left">80.7/97.4</td><td class="colsep0 rowsep0" align="left">99.7/95.9</td><td class="colsep0 rowsep0" align="left">96.5/100.6</td><td class="colsep0 rowsep0" align="left">86.9/120.4</td><td class="colsep0 rowsep0" align="left">79.8/49.0</td><td class="colsep0 rowsep0" align="left">78.1/81.3</td><td class="colsep0 rowsep0" align="left">74.4/0.8</td><td class="colsep0 rowsep0" align="left">90.9/88.4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">microsomal stability phase I CL<sub>int</sub> [μL min<sup>–1</sup> mg<sup>–1</sup>]</td><td class="colsep0 rowsep0" align="left">42.0/126.0</td><td class="colsep0 rowsep0" align="left">16.7/73.7</td><td class="colsep0 rowsep0" align="left">1.4/9.4</td><td class="colsep0 rowsep0" align="left">22.0/35.4</td><td class="colsep0 rowsep0" align="left">27.7/124.9</td><td class="colsep0 rowsep0" align="left">96.3<a class="ref internalNav" href="#t6fn3" aria-label="c">c</a>/770.2<a class="ref internalNav" href="#t6fn3" aria-label="c">c</a></td><td class="colsep0 rowsep0" align="left">64.2<a class="ref internalNav" href="#t6fn3" aria-label="c">c</a>/866.4<a class="ref internalNav" href="#t6fn3" aria-label="c">c</a></td><td class="colsep0 rowsep0" align="left">66.0<a class="ref internalNav" href="#t6fn3" aria-label="c">c</a>/602.7<a class="ref internalNav" href="#t6fn3" aria-label="c">c</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Caco-2: <i>P</i><sub>app</sub>(A–B) [×10<sup>–6</sup> cm s<sup>–1</sup>]</td><td class="colsep0 rowsep0" align="left">0.09</td><td class="colsep0 rowsep0" align="left">0.7</td><td class="colsep0 rowsep0" align="left">2.38</td><td class="colsep0 rowsep0" align="left">21.1</td><td class="colsep0 rowsep0" align="left">1.9</td><td class="colsep0 rowsep0" align="left">0.5</td><td class="colsep0 rowsep0" align="left">2.1</td><td class="colsep0 rowsep0" align="left">1.2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Caco-2: <i>P</i><sub>app</sub>(B–A) [×10<sup>–6</sup> cm s<sup>–1</sup>]</td><td class="colsep0 rowsep0" align="left">0.4</td><td class="colsep0 rowsep0" align="left">0.2</td><td class="colsep0 rowsep0" align="left">55.4</td><td class="colsep0 rowsep0" align="left">7.5</td><td class="colsep0 rowsep0" align="left">0.6</td><td class="colsep0 rowsep0" align="left">0.1</td><td class="colsep0 rowsep0" align="left">23.9</td><td class="colsep0 rowsep0" align="left">5.2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Caco-2: ratio (B–A)/(A–B)</td><td class="colsep0 rowsep0" align="left">4.34</td><td class="colsep0 rowsep0" align="left">0.2</td><td class="colsep0 rowsep0" align="left">23.2</td><td class="colsep0 rowsep0" align="left">0.36</td><td class="colsep0 rowsep0" align="left">0.3</td><td class="colsep0 rowsep0" align="left">0.2</td><td class="colsep0 rowsep0" align="left">11.4</td><td class="colsep0 rowsep0" align="left">4.5</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t6fn1"><div class="footnote" id="t6fn1"><sup><sup>a</sup></sup><p class="last">clog <i>P</i> was calculated using ChemDraw, PerkinElmer.</p></div><div class="footnote" id="t6fn2"><sup><sup>b</sup></sup><p class="last">LLE was calculated from pEC<sub>50</sub>(H1781) – clog <i>P</i>.</p></div><div class="footnote" id="t6fn3"><sup><sup>c</sup></sup><p class="last">Measured at 1 μM. n.d. = not determined.</p></div></div></div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i18">Discussion and Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_81268" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_81268" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">Given the need for Her2-targeted inhibitors, we herein evaluated the pyrrolopyrimidine scaffold for its potential in yielding covalent inhibitors of insYVMA-mutated Her2. Utilizing novel scaffolds has been a successful strategy in targeting mutant kinases including EGFR and ALK in the past.<a onclick="showRef(event, 'ref6 ref28'); return false;" href="javascript:void(0);" class="ref ref6 ref28">(6,28)</a> Focused screening revealed established back-pocket elements that resulted in high biochemical potency and prompted us to combine these moieties with the core scaffold. Further studies were performed related to the structure–activity relationship of solubilizing groups, a line of inquiry that arose from a set of complex crystal structures from the closely related family member EGFR. Western blot analyses further substantiated our approach. We found that reduced molecular weight and the introduction of heteroatoms improved the cellular potency of the inhibitors against cancer cell lines and we proposed optimized solubilizing groups attached to the linker moiety as a future approach to develop more potent inhibitors. We, thereby, highlighted critical features for pyrrolopyrimidine-based Her2 inhibitors. Pharmacokinetic profiling revealed stability issues in some back-pocket elements which will be addressed in future works. However, the oral absorption as predicted by the Caco-2 assay was revealed to be optimized as compared to type II inhibitors neratinib and lapatinib. Altogether, these studies confirmed compound <b>14b</b> as a potent inhibitor of mutated Her2 with promising <i>in vitro</i> pharmacokinetic properties. The findings described here will stimulate and guide further medicinal chemistry approaches toward type II Her2 inhibitors.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i19">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_47757" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_47757" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i20" class="anchor-spacer"></div><h3 class="article-section__title" id="_i20"> Construct Design of Her2-wt and A775_G776insYVMA</h3><div class="NLM_p">For mass experiments as well as for assays, the Her2 constructs (wt and A775_G776insYVMA) encoding residues 703–1029 of the kinase domain was synthesized by <i>GeneArt</i> (life technologies). The construct was cloned into pIEX/Bac3 expression vector (MerckMillipore) including a C-terminal His<sub>6</sub>-tag. The A775_G776insYVMA-mutation was introduced by site-directed mutagenesis (QuikChange, Stratagene/Agilent Technologies). Transfection, virus generation, and amplification were carried out in <i>Spodoptera frugiperda</i> cell line <i>Sf9</i> following the <i>flash</i>BAC protocol.</div><div class="NLM_p last">For crystallographic studies, codon-optimized DNA-encoding residues 696–1022 of the human EGFR with a N-terminal His<sub>6</sub>-tag and thrombin cleavage site were cloned into a pIEX/Bac-3 vector and expressed in <i>Sf9</i> insect cells. Point mutations T790M and V948R were introduced by site-directed mutagenesis (QuikChange, Stratagene/Agilent Technologies). Transfection, virus generation, and amplification were carried out in <i>S. frugiperda</i> cell line <i>Sf9</i> following the <i>flash</i>BAC protocol.</div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i21" class="anchor-spacer"></div><h3 class="article-section__title" id="_i21"> Protein Expression and Purification</h3><div class="NLM_p">EGFR-T790M/V948R, Her2-wt, and Her2-A775_G776insYVMA were expressed in <i>Sf9</i> cells using the <i>flash</i>BAC system. The EGFR mutant was purified as follows. After 3 days of expression (27 °C, 110 rpm), the cells were harvested (3000 g, 20 min), resuspended in lysis buffer [50 mM Tris-HCl, 600 mM NaCl, 15% glycerol, 1 mM tris(2-carboxyethyl)phosphine (TCEP), pH 8.0], supplemented with a protease-inhibitor cocktail (cOmplete EDTA-free), homogenized, lysed, and centrifuged at 20,500 rpm for 1 h at 4 °C. The filtered supernatant was loaded onto a nickel-affinity column. The protein was eluted with 500 mM NaCl, 25 mM Tris-HCl pH 8, 250 mM imidazole, 10% glycerol, and 1 mM TCEP, followed by a cleavage with thrombin to remove the His<sub>6</sub>-tag and a second nickel-affinity chromatography capturing the flow through. Finally, the protein was purified by size-exclusion chromatography (100 mM NaCl, 25 mM Tris-HCl, 10% glycerol, and 1 mM TCEP, pH 8.0) and concentrated to 7.0 mg/mL. Protein identity was confirmed by electrospray ionization-mass spectrometry (ESI-MS) analysis.</div><div class="NLM_p last">The Her2-wt and the insYVMA mutant were purified as follows. After 3 days of expression (27 °C, 110 rpm), the cells were harvested (3000 g, 20 min), homogenized, and lysated in buffer A [500 mM NaCl, 50 mM Tris-HCl, 10 mM imidazole, 10% glycerol, 1 mM dithiothreitol (DTT), pH 8.0 supplemented with a protease-inhibitor cocktail (cOmplete EDTA-free)], followed by centrifugation (60 min, 40,000<i>g</i>, 4 °C) twice. The filtered lysate was loaded onto a nickel-affinity column. The protein was eluted with buffer B (500 mM NaCl, 50 mM Tris-HCl, 500 mM imidazole, 10% glycerol, and 1 mM TCEP, pH 8.0), followed by a final size-exclusion chromatography (300 mM NaCl, 30 mM Tris-HCl pH 8, 10% glycerol and 1 mM TCEP) and concentrated to 2.0 and 0.109 mg/mL for Her2 and the insYVMA mutant, respectively. The protein identity was confirmed by ESI-MS analysis.</div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i22" class="anchor-spacer"></div><h3 class="article-section__title" id="_i22"> MS Experiments</h3><div class="NLM_p last">The concentrated proteins EGFR-T790M/V948R and Her2-wt (1.0 mg/mL) were incubated with a three-fold excess of representative inhibitors [10 mM dimethyl sulfoxide (DMSO) stock] for 1 h on ice in a buffer (100 mM NaCl, 25 mM Tris-HCl, 10% glycerol and 1 mM TCEP, pH 8.0). We analyzed the aliquots by ESI-MS using a Thermo Fisher Scientific Dionex UltiMate 3000 HPLC system connected to a Thermo Fisher Scientific Velos Pro (2d ion trap). Therefore, 2 μL of the sample was injected and separated using a AdvanceBio Desalting-RP cartridge starting at 5% of solvent B for 5 min, followed by a gradient up to 80% of solvent B over 2.5 min with a flow rate of 400 μL/min with 0.1% formic acid in water as solvent A and 0.1% formic acid in MeCN as solvent B. A mass range of 700–2000 <i>m</i>/<i>z</i> was scanned, and raw data were deconvoluted and analyzed with MagTran software (Z.Q. Zhang, A.G. Marshall, <i>J. Am. Soc. Mass. Spectr.</i><b>1998,</b><i>9,</i> 225–233).</div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i23" class="anchor-spacer"></div><h3 class="article-section__title" id="_i23"> Crystallization of EGFR-T790M/V948R with Her2 Inhibitors</h3><div class="NLM_p last">The concentrated protein EGFR-T790M/V948R (7.0 mg/mL) was incubated with a three-fold excess of Her2 inhibitors (10 mM DMSO stock) for 1 h on ice. Crystals were grown at 20 °C by the hanging drop vapor diffusion method. The protein–compound cocktail was mixed in a 1:1 ratio (1 μL protein: 1 μL reservoir solution containing 15–37.5% PEG3350, 100 mM MgSO<sub>4</sub>, 2–4% ethylene glycol). Crystals were grown after 1 to 2 weeks and were flash-frozen in liquid nitrogen. Diffraction data were recorded using the PXII X10SA beamline at the Swiss Light Source (PSI, Villigen, Switzerland). The data were processed and merged using XDS and scaled using XSCALE.</div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i24" class="anchor-spacer"></div><h3 class="article-section__title" id="_i24"> Structure Determination and Refinement</h3><div class="NLM_p last">The complex crystal structures of EGFR T790M/V948R were solved by molecular replacement with PHASER (R.J. Read, <i>Acta Crystallogr D Biol Crystallogr</i><b>2001,</b><i>57,</i> 1373–1382) using structure PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5J9Z">5J9Z</a> as the template. The molecules in the asymmetric unit were manually adjusted using the program COOT (P. Emsley, K. Cowtan, <i>Acta Crystallogr D Biol Crystallogr</i><b>2004,</b><i>60,</i> 2126–2132). The refinement was performed with Phenix 1.10 (P.D. Adams, P.V. Afonine, G. Bunkoczi, V.B. Chen, I.W. Davis, N. Echols, J.J. Headd, L.W. Hung, G.J. Kapral, R.W. Grosse-Kunstleve, A.J. McCoy, N.W. Moriarty, R. Oeffner, R.J. Read, D.C. Richardson, J.S. Richardson, T.C. Terwilliger, P.H. Zwart, <i>Acta Cryst</i><b>2010,</b><i>D66,</i> 213–221). Inhibitor topology files were generated using the Dundee PRODRG2 server (R.A. Laskowski, M.W. Macarthur, D.S. Moss, J.M. Thornton, <i>J Appl Crystallogr</i><b>1993,</b><i>26,</i> 283–291). Refined structures were validated with the PDB validation server. Data collection, structure refinement statistics, PDB ID codes, and further details for data collection are provided in <a href="/doi/suppl/10.1021/acs.jmedchem.0c00870/suppl_file/jm0c00870_si_001.pdf" class="ext-link">Table S3</a>. PyMOL (W.L. DeLano, <i>The PyMOL Molecular Graphics System</i>) was used for generating the figures.</div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i25" class="anchor-spacer"></div><h3 class="article-section__title" id="_i25"> Activity-Based Assay for IC<sub>50</sub> Determination and Kinetic Characterization</h3><div class="NLM_p">The focused screening and subsequent hit validation (kinases expressed and purified as stated above) and following IC<sub>50</sub> determinations for Her2-wt and its mutant insYVMA (kinase expressed and purified as stated above for Her2-wt and ProQinase, lot1525-0000-1/003 for Her2-insYVMA) were performed with the HTRF KinEASE-TK assay from Cisbio according to the manufacturer’s instructions. Briefly, the amount of Her2 in each reaction well was set to 20 ng Her2-wt (21.28 nM) or 10 ng Her2-A775_G776insYVMA (10.6 nM) for screen and hit validation and following IC<sub>50</sub> determinations, respectively. IC<sub>50</sub> determinations for EGFR and its mutants (Carna Biosciences, lot13CBS-0005K for EGFR-wt; Carna, lot13CBS-0537B for EGFR-L858R and Carna, lot12CBS-0765B for EGFR-L858R/T790M) were performed with the HTRF KinEASE-TK assay from Cisbio according to the manufacturer’s instructions. Briefly, the amount of EGFR in each reaction well was set to 0.60 ng EGFR-wt (0.67 nM), 0.10 ng EGFR-L858R (0.11 nM), or 0.07 ng EGFR-T790M/L858R (0.08 nM), respectively. An artificial substrate peptide (TK-substrate from Cisbio) was phosphorylated by Her2 or EGFR. After completion of the reaction, the reaction was stopped by the addition of buffer containing ethylenediaminetetraacetic acid (EDTA), an anti-phosphotyrosine antibody labeled with europium cryptate and streptavidin labeled with the fluorophore XL665. FRET between europium cryptate and XL665 was measured after an additional hour of incubation to quantify the phosphorylation of the substrate peptide. ATP concentrations were set at their respective <i>K</i><sub>m</sub> values (3 μM for Her2-wt, 6 μM for Her2-insYVMA, 9.5 μM for EGFR-wt, 9 μM for EGFR-L858R, and 4 μM for EGFR-L858R/T790M), while substrate concentrations of 1 μM, 1 μM, 1 μM, 225 nM, and 200 nM, respectively, were used. After kinase and inhibitor were preincubated for 30 min, the reaction (reaction time: 60 min for Her2-wt, 40 min for Her2-insYVMA, 25 min for EGFR-wt, 15 min for EGFR-L858R, and 20 min for EGFR-T790M/L858R) was started by the addition of ATP and the substrate peptide. An EnVision multimode plate reader (PerkinElmer) was used to measure the fluorescence of the samples at 620 nm (Eu<sup>3+</sup>-labeled antibody) and 665 nm (XL665-labeled streptavidin) 50 μs after excitation at 320 nm. The quotient of both intensities for reactions made with eight different inhibitor concentrations was then analyzed using the Quattro Software Suite for IC<sub>50</sub>-determination. Each reaction was performed in duplicate, and at least three independent determinations of each IC<sub>50</sub> were made.</div><div class="NLM_p last">For kinetic characterization (<i>k</i><sub>inact</sub>/<i>K</i><sub>i</sub>), Her2-insYVMA kinase was added to a mixture of ATP, TK-substrate, and inhibitor, and the enzymatic reaction was run for 2–40 min. Afterward, the reactions were stopped as described above. Six-fold dilution series (eight data points per IC<sub>50</sub> curve) starting at 2 μM (afatinib, neratinib) or 20 μM (<b>14b</b>, <b>14h</b>, <b>18d</b>, <b>19</b>) final compound concentrations were used for time-dependent IC<sub>50</sub> determination. Compound dilutions were generated using the acoustic dispensing system “ECHO 520 Liquid Handler” from Labcyte (Sunnyvale, California, USA) and the according dose–response software “Echo Dose-Response v1.5.4”. IC<sub>50</sub> values were calculated using the Quattro Software Suite, and the obtained data were analyzed using XLfit 5 (IDBS, Guildford, UK) as described in the literature to determine <i>k</i><sub>inact</sub> and <i>K</i><sub>i</sub> (B.F. Krippendorff, R. Neuhaus, P. Lienau, A. Reichel, W. Huisinga, <i>J. Biomol. Screen</i><b>2009,</b><i>14,</i> 913–923).</div></div><div id="sec4_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i26" class="anchor-spacer"></div><h3 class="article-section__title" id="_i26"> Cell Culture</h3><div class="NLM_p">H1781, A431, and H1975 cells were obtained from the American Type Culture Collection (ATCC), and Ba/F3(Her2-insYVMA) cells were obtained from Prof. Roman K. Thomas’ Lab (University of Cologne, Germany). H1781, H1975, and Ba/F3 cells were cultured in RPMI-1640 GlutaMAX (Life Technologies, Germany) supplemented with 10% FBS Good (PanBiotech, Germany) and 1% PenStrep (Life Technologies, Germany) in a humidified incubator at 37 °C and 5% CO<sub>2</sub>. A431 was cultivated in Dulbecco’s modified Eagle’s medium high-glucose GlutaMAX (Life Technologies, Germany) supplemented with 10% FBS Good (PanBiotech, Germany) and 1% PenStrep (Life Technologies, Germany) in a humidified incubator at 37 °C and 5% CO<sub>2</sub>.</div><div class="NLM_p last">Cell line authentication of H1781, A431, and H1975 cells has been performed last in August 2017 by STR profiling of 16 alleles.</div></div><div id="sec4_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i27" class="anchor-spacer"></div><h3 class="article-section__title" id="_i27"> Viability Assay</h3><div class="NLM_p last">Cells were seeded at cell numbers that assure linearity and optimal signal intensity (150–300 cells/well, 25 μL) and cultured for 24 h in serum- and antibiotics-containing media in humidified chambers at 37 °C/5% CO<sub>2</sub>. The cells were then treated with EGFR inhibitors in serial dilutions (14 nM to 30 μM) with DMSO and Staurosporine as control and incubated for 96 h. Afterward, viability studies were carried out using CellTiter-Glo Assay (Promega, USA) that is a homogeneous method of determining the number of viable cells in culture. It is based on quantification of ATP, indicating the presence of metabolically active cells. For these studies, the CellTiter-Glo reagent was prepared according to the instructions of the kit and diluted 1:1 with the complete growth medium suitable for the corresponding cell line. Thereon, reagent and assay plates were equilibrated at room temperature for 20 min. Equal volumes of the reagent were added to the volume of culture medium present in each well (25 μL). The plates were mixed for 2 min on an orbital shaker to induce cell lysis. The microplates were then incubated at room temperature for 20 min for stabilization of the luminescent signal. Following incubation, the luminescence was recorded on an EnVision microplate reader (PerkinElmer) using 500 ms integration time. The data were then analyzed using the Quattro Research Software Suite for EC<sub>50</sub>-determination. As quality control, the <i>Z</i>′-factor was calculated from 16 positive and negative control values. Only assay results showing a <i>Z</i>′-factor ≥ 0.5 were used for further analysis. All data points were measured in duplicates for each plate and were replicated in at least two plates.</div></div><div id="sec4_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i28" class="anchor-spacer"></div><h3 class="article-section__title" id="_i28"> Western Blotting</h3><div class="NLM_p last">The cells were plated in six-well plates on day 1 and incubated at 37 °C and 5% CO<sub>2</sub>. On day 2, the cells were starved over night with media containing 0.5% fetal bovine serum (FBS) to be incubated on day 3 for 1 h with inhibitors. The media was discarded, and cells were washed once with Dulbecco’s phosphate-buffered saline 1×. Using 100 μL per well of lysis buffer, the cells were then scraped off and lysed while rotating (Stuart Scientific, Staffordshire, UK) for 1 h at 4 °C. The lysis buffer was containing 1% (v/v) NP-40, 50 mM (v/v) Tris-HCl pH 8.0, 100 mM (m/v) sodium fluoride, 30 mM (m/v) sodium pyrophosphate, 2 mM (m/v) sodium molybdate, 5 mM (v/v) EDTA, and 2 mM (m/v) sodium vanadate; 1% 10 mg/mL aprotinin and leupeptin as well as 10% 100 mM phenylmethylsulfonyl fluoride and 200 mM sodium vanadate were added shortly before use. Cells were centrifuged at 4 °C for 30 min at 14,000 rpm. The supernatant was transferred into a new tube, and protein concentrations were determined with the Bio-Rad Protein Assay (Bio-Rad Laboratories, Hercules, CA). Loading buffer containing 0.5 M (v/v) Tris-HCl pH 6.7, 10% (v/v) SDS, (w/v) DTT, (v/v) glycerol, and 0.05% (w/v) bromophenol blue was added, and lysates were incubated for 5 min at 95 °C. Proteins were separated by sodium dodecyl sulfate-polyacrylamide gel electrophoresis and blotted onto NC membranes (GE Healthcare/Amersham-Biosciences, Freiburg, Germany) using X Cell ll Blot module according to manufacturer’s manual (Thermo Fisher Scientific, Waltham, MA). After blocking with 1x Net-G buffer [10× buffer containing 1.5 M (w/v) NaCl, 50 mM (w/v) EDTA, 500 mM (w/v) Tris 0.5% (v/v) Tween 20 and 0.4% (w/v) gelatine], membranes were incubated at 4 °C overnight with the respective primary antibody (1:5.000 for β-actin, 1:1000 for all other antibodies). After washing, the membranes were incubated for 2 h at room temperature with the secondary antibody (anti-IgG-HRP 1:2000, CST, Frankfurt, Germany) and washed again. Changes in protein expression and phosphorylation as visualized by chemiluminescence using Amersham ECL Prime (GE Healthcare/Amersham-Biosciences, Freiburg, Germany) were captured and quantified using a FUJI LAS3000 system with Science Lab 2001 ImageGauge 4.0 software (Fujifilm Medial Systems, Stamford CT, USA).</div></div><div id="sec4_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i29" class="anchor-spacer"></div><h3 class="article-section__title" id="_i29"> Antibodies</h3><div class="NLM_p last">Rabbit antibodies specific for Her2/ErbB2, phospho-Her2/ErbB2 (Tyr1221/1222), phospho-Her2/ErbB2 (Tyr1248), AKT, phospho-AKT (Ser473), p42/44 mitogen-activated protein kinase (MAPK), phospho-p44/42 MAPK (Thr202/Tyr204), S6 ribosomal protein, phospho-S6 ribosomal protein (Ser235/236), 4E-BP1, phospho-4E-BP1 (Ser65), mouse anti-β-actin antibody, and HRP-conjugated anti-rabbit and anti-mouse secondary antibodies were purchased from Cell Signaling (Frankfurt, Germany; Beverly, MA).</div></div><div id="sec4_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i30" class="anchor-spacer"></div><h3 class="article-section__title" id="_i30"> Microsomal Stability (Phase I) Assay</h3><div class="NLM_p last">Metabolic stability under oxidative conditions was measured in liver microsomes from different species by LC/MS-based measuring of depletion of the compound at a concentration of 3 μM over time up to 50 min (HLM) and 60 min (MLM) at 37 °C. Based on the compound half-life <i>t</i><sub>1/2</sub>, the <i>in vitro</i> intrinsic clearance CL<sub>int</sub> was calculated.</div></div><div id="sec4_12" class="NLM_sec NLM_sec_level_2"><div id="ac_i31" class="anchor-spacer"></div><h3 class="article-section__title" id="_i31"> Plasma Stability</h3><div class="NLM_p last">Plasma stability was measured by LC/MS-based determination of the percentage of remaining compounds at a concentration of 5 μM after incubation in plasma obtained from different species for 1 h at 37 °C.</div></div><div id="sec4_13" class="NLM_sec NLM_sec_level_2"><div id="ac_i32" class="anchor-spacer"></div><h3 class="article-section__title" id="_i32"> Caco-2 Assay</h3><div class="NLM_p last">For the Caco-2 cell assay, a 10 mM DMSO stock of the compound was diluted to a final concentration of 5 μM in HBSS buffer pH 7.4 and incubated for 2 h at 37 °C and 5% CO<sub>2</sub> on a monolayer of Caco-2 cells (ATCC) that had been grown on a transwell membrane (Millipore, Schwalbach, Germany) for 21 days. The compound concentration was measured in the receiver as well as the donor well. Apparent permeability (<i>P</i><sub>app</sub>) from either the apical to basolateral direction or vice versa was calculated by the equation: <i>P</i><sub>app</sub> = 1/<i>AC</i><sub>0</sub> (d<i>Q</i>/d<i>t</i>), where <i>A</i> is the membrane surface area, <i>C</i><sub>0</sub> is the donor drug concentration at <i>t</i> = 0, and d<i>Q</i>/d<i>t</i> is the amount of drug transported within the given time period of 2 h.</div></div><div id="sec4_14" class="NLM_sec NLM_sec_level_2"><div id="ac_i33" class="anchor-spacer"></div><h3 class="article-section__title" id="_i33"> Turbidimetric Solubility</h3><div class="NLM_p last">Turbidimetric solubility of compounds was determined by diluting a DMSO solution of the test compound into aqueous buffer at pH 7.4 and measuring the absorbance at 620 nm after 120 min of incubation at room temperature.</div></div><div id="sec4_15" class="NLM_sec NLM_sec_level_2"><div id="ac_i34" class="anchor-spacer"></div><h3 class="article-section__title" id="_i34"> SolRank</h3><div class="NLM_p last">For determining the kinetic solubility, the compound was diluted from a 10 mM stock in DMSO to a final concentration of 500 μM in 50 mM Hepes buffer, pH 7.4. Following an incubation of 90 min at room temperature on a shaker, the aqueous dilution was filtered through a 0.2 μm poly(vinylidene difluoride) filter, and the optical density between 250 and 500 nm was measured at an interval of 10 nm. The kinetic solubility was calculated from the area under the curve between 250 and 500 nm and normalized to absorption of a dilution of the compound in acetonitrile.</div></div><div id="sec4_16" class="NLM_sec NLM_sec_level_2"><div id="ac_i35" class="anchor-spacer"></div><h3 class="article-section__title" id="_i35"> Chemistry</h3><div class="NLM_p">Unless otherwise noted, all commercially available compounds, reagents, solvents, and anhydrous solvents were used as provided without further purification. <sup>1</sup>H and <sup>13</sup>C NMR spectra were recorded on a Bruker AVANCE DRX 300, DRX 400, or DRX 500 spectrometer or 400 MHz AVANCE-III HD and 600 MHz AVANCE-III HD (Bruker BioSpin GmbH) equipped with a 5 mm helium-cooled broadband BBFO cryo probe from Bruker BioSpin GmbH 500 MHz DD2-500 (Agilent Technologies). Chemical shifts (δ) are reported in parts per million (ppm) and are referenced to the residual solvent signal DMSO-<i>d</i><sub>6</sub> (2.50 or 39.52 ppm), CDCl<sub>3</sub> (7.26 or 77.16 ppm), or CD<sub>3</sub>OD (1.94 or 49.00 ppm, respectively). High-resolution ESI mass spectra (ESI-Fourier transform mass spectrometry) were recorded on a Thermo LTQ Orbitrap (high-resolution mass spectrometer from Thermo Electron) coupled to an “Accela” HPLC system supplied with a “Hypersil GOLD” column (Thermo Electron). Microwave reactions were performed using the Monowave 300 reactor by Anton Paar. Analytical TLC was carried out on Merck 60 F254 aluminum-backed silica gel plates and monitored by UV at λ = 254 and 366 nm. Compounds were purified by column chromatography using Baker silica gel (40–70 μm particle size) or flash chromatography on a Biotage Isolera One using Biotage SNAP, SNAP Ultra, ZIP Sphere, or ZIP KP-Sil columns (25, 10, 5, or 120 g, respectively) or Grace Reveleris C18 40 μm columns (4 or 12 g) and monitored by UV at λ = 254 and 280 nm. Preparative HPLC was conducted on an Agilent HPLC system (1200 series) with a VP 125/21 Nucleodur C18 column from Macherey-Nagel and monitored by UV at λ = 210, 254, and 280 nm. All final compounds were purified to ≥95% purity confirmed by NMR analysis as well as LC/MS analysis on LCQ Advantage MAX (1200 series, Agilent) with Eclipse XDB-C18-column (5 μM 150 × 1.6 mm, Phenomenex).</div><div id="sec4_16_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i36" class="anchor-spacer"></div><h4 class="article-section__title" id="_i36"> General Method for Back-Pocket Aniline Coupling</h4><div class="NLM_p last">4-Chloro-pyrrolopyrimidine (1.00 equiv) was dissolved in butan-1-ol, and to it was added HCl (4 M in dioxane, 1.25 equiv). Subsequently, the respective aniline (1.50 equiv) was added, and the resulting solution was stirred for 72 h at 80 °C. The solution was allowed to cool to room temperature, and to it was added an aqueous solution of NaHCO<sub>3</sub> after which it was extracted with CH<sub>2</sub>Cl<sub>2</sub>. The combined organic layers were dried over Na<sub>2</sub>SO<sub>4</sub>, and the solvent was removed <i>in vacuo</i>. The crude product was purified by repeated flash chromatography (SiO<sub>2</sub>, followed by C<sub>18</sub> column). Where appropriate, the SEM protecting group was removed following the <a class="ref internalNav" href="#sec4_16_6" aria-label="General Method for SEM Deprotection">General Method for SEM Deprotection</a> as described below.</div></div><div id="sec4_16_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i37" class="anchor-spacer"></div><h4 class="article-section__title" id="_i37"> General Method for Suzuki–Miyaura Coupling</h4><div class="NLM_p last">The aryl halide (1 equiv), boronic acid (1.2 equiv), Pd(PPh<sub>3</sub>)<sub>4</sub> (5 mol %), and sodium carbonate (2.5 equiv) were dissolved in a mixture of dioxane/water (3/1) (5 mL/mmol aryl halide), the atmosphere was evacuated, and the flask was filled with Ar three times. The sealed reaction mixture was then heated to 80 °C and allowed to stir overnight (16 h). The reaction mixture was cooled to room temperature, filtered through Celite, diluted with EtOAc and water, and the organic layer was separated. The aqueous layer was extracted with aliquots of EtOAc (3×), and the combined organic layers were then washed with a saturated solution of brine, dried over MgSO<sub>4</sub>, and filtered. After the removal of the solvent <i>in vacuo</i>, the crude product was adsorbed onto silica and purified by flash column chromatography.</div></div><div id="sec4_16_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i38" class="anchor-spacer"></div><h4 class="article-section__title" id="_i38"> General Method for Nucleophilic Aromatic Substitution</h4><div class="NLM_p last">To a suspension of sodium hydride (60% dispersion in oil, 2.4 equiv) in tetrahydrofuran (THF) was added a solution of alkyl alcohol (10 equiv) in THF dropwise over 5 min. After allowing the reaction mixture to stir for 30 min at room temperature, it was subsequently cooled to 0 °C (ice) and a solution of the pyrrolopyrimidine (1 equiv) in THF was added dropwise over 10 min (10 mL/mmol alkyl alcohol). The reaction mixture was allowed to warm to room temperature and left to stir overnight (16 h). The reaction mixture was then diluted with water and EtOAC, and the organic layer was separated. The aqueous layer was extracted with aliquots of EtOAc (3×), and the combined organic layers were then washed with a saturated solution of sodium bicarbonate and a saturated solution of brine, dried over MgSO<sub>4</sub>, and filtered. After removal of the solvent <i>in vacuo</i>, the crude product was adsorbed onto silica and purified by flash column chromatography.</div></div><div id="sec4_16_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i39" class="anchor-spacer"></div><h4 class="article-section__title" id="_i39"> General Method for Aromatic Nitro Reduction</h4><div class="NLM_p last">A suspension of aromatic nitro derivative (1 equiv), iron (5 equiv), and ammonium chloride (10 equiv) in a mixture of EtOH/water (9/1, 10 mL/mmol aromatic nitro) was stirred at 60 °C for 3 h. The reaction mixture was cooled to room temperature, filtered through Celite, and washed with MeOH. After removal of the solvent <i>in vacuo</i>, the residue was diluted with DCM and water and the organic layer was separated. The aqueous layer was extracted with aliquots of DCM (3×), and the combined organic layers were then washed with a saturated solution of brine, dried over MgSO<sub>4</sub>, and filtered. After removal of the solvent <i>in vacuo</i>, the crude product was adsorbed onto silica and purified by flash column chromatography.</div></div><div id="sec4_16_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i40" class="anchor-spacer"></div><h4 class="article-section__title" id="_i40"> General Method for Aniline Derivative Acylation</h4><div class="NLM_p last">To a suspension of aniline derivative (1 equiv) and DIPEA (2 equiv) in dry THF, cooled to 0 °C, a solution of acryloyl chloride (1.1 equiv) in THF was added dropwise over 15 min (10 mL/mmol aniline derivative). The reaction was allowed to warm to room temperature and left to stir until full conversion. The reaction mixture was then diluted with DCM and water, and the organic layer was separated. The aqueous layer was extracted with aliquots of DCM (3×), and the combined organic layers were then washed with 1 M solution of HCl, a saturated solution of sodium bicarbonate, and a saturated solution of brine, dried over MgSO<sub>4</sub>, and filtered. After the removal of the solvent <i>in vacuo</i>, the crude product was adsorbed onto silica and purified by flash column chromatography. Where appropriate, the SEM protection group was removed following the <a class="ref internalNav" href="#sec4_16_6" aria-label="General Method for SEM Deprotection">General Method for SEM Deprotection</a> as described below.</div></div><div id="sec4_16_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i41" class="anchor-spacer"></div><h4 class="article-section__title" id="_i41"> General Method for SEM Deprotection</h4><div class="NLM_p">To a solution of SEM-protected pyrrolopyrimidine derivative (1 equiv) in DCM (15 mL) was added trifluoroacetic acid (5 mL). The reaction mixture was allowed to stir for 3 h, and after the removal of the solvent <i>in vacuo</i>, the crude residue was redissolved in THF (10 mL) and an aqueous solution of 1 M KOH (10 mL) and allowed to stir for further 2 h. The reaction mixture was then neutralized by the addition of an aqueous solution of 1 M HCl (10 mL) and diluted with DCM (30 mL) and water (20 mL), and the organic layer was separated. The aqueous layer was extracted with aliquots of DCM (3×), and the combined organic layers were then washed with a saturated solution of sodium bicarbonate and a saturated solution of brine, dried over MgSO<sub>4</sub>, and filtered. After the removal of the solvent <i>in vacuo</i>, the crude product was adsorbed onto silica and purified by flash column chromatography.</div><div id="sec4_16_6_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i42" class="anchor-spacer"></div><h5 class="article-section__title" id="_i42"> 4-Chloro-5-iodo-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidine (<b>9</b>)</h5><div class="NLM_p last">Over a period of 15 min, a solution of <i>N</i>-iodosuccinimide (12.30 g, 54.67 mmol, 1.05 equiv) in 20 mL of dimethylformamide (DMF) was added dropwise to a solution of 4-chloro-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidine (<b>8</b>) (8.00 g, 52.10 mmol, 1 equiv) in 80 mL of DMF. The reaction flask was covered in foil and allowed to stir in darkness at room temperature overnight (16 h). The reaction mixture developed a deep orange color, which reverted to colorless upon the addition of a sufficient amount of aqueous sodium thiosulfate (10 mol %). The reaction mixture was then diluted with EtOAc (50 mL) and water (30 mL), and the organic layer was separated. The aqueous layer was extracted with aliquots of EtOAc (3 × 30 mL), and the combined organic layers were then washed with successive aliquots of water (3 × 30 mL) and a saturated solution of brine (50 mL), dried over MgSO<sub>4</sub>, and filtered. After the removal of the solvent <i>in vacuo</i>, the pure product was afforded as an off-white solid and no further purification was needed. Yield: 97% (14.12 g, 50.5 mmol). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>): δ 12.95 (s, 1H), 8.59 (s, 1H), 7.93 (d, <i>J</i> = 2.2 Hz, 1H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>): δ 151.5, 150.8, 150.5, 133.9, 115.8, 51.7. LC/MS: <i>m</i>/<i>z</i> calcd for [C<sub>6</sub>H<sub>3</sub>N<sub>3</sub>ICl + H]<sup>+</sup>, 280.47; found, 280.18.</div></div><div id="sec4_16_6_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i43" class="anchor-spacer"></div><h5 class="article-section__title" id="_i43"> 4-Chloro-5-iodo-7-{[2-(trimethylsilyl)ethoxy]methyl}-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidine (<b>10</b>)</h5><div class="NLM_p last">A solution of 4-chloro-5-iodo-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidine (<b>9</b>) (7.21 g, 25.80 mmol, 1 equiv) in 15 mL of DMF was added dropwise to a suspension of sodium hydride (1.20 g, 60% dispersion in oil, 30.08 mmol, 1.17 equiv) in 90 mL of DMF on ice over 15 min and stirred for 30 min. This was followed by the addition of a solution of 2-[(trimethylsilyl)ethoxy]methyl chloride (5.29 g, 31.74 mmol, 1.23 equiv) in 15 mL of DMF over 15 min. The reaction mixture was allowed to warm to room temperature and left to stir overnight (16 h). Water (50 mL) was added to the reaction, and the resulting white precipitate was collected by filtration, washed with water (3 × 30 mL), and dried <i>in vacuo</i>. The compound was obtained as pure white crystals, and no further purification was needed. Yield: 76% (8.03 g, 19.6 mmol). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.68 (s, 1H), 8.12 (s, 1H), 5.60 (s, 2H), 3.50 (t, <i>J</i> = 8.0 Hz, 2H), 0.81 (t, <i>J</i> = 8.0 Hz, 2H), −0.11 (s, 9H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>): δ 151.2, 151.1, 150.9, 136.6, 116.2, 72.9, 66.1, 52.9, 17.1, −1.43. HRMS: <i>m</i>/<i>z</i> calcd for [C<sub>12</sub>H<sub>17</sub>ON<sub>3</sub>ClISi + H]<sup>+</sup>, 409.9947; found, 409.9944, calcd for [C<sub>12</sub>H<sub>17</sub>ON<sub>3</sub><sup>37</sup>ClISi + H]<sup>+</sup>, 411.9917; found, 411.9910.</div></div><div id="sec4_16_6_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i44" class="anchor-spacer"></div><h5 class="article-section__title" id="_i44"> 5-(3-Nitrophenyl)-7-{[2-(trimethylsilyl)ethoxy]methyl}-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-4-ol (<b>11</b>)</h5><div class="NLM_p last">A solution of 4-chloro-5-iodo-7-{[2-(trimethylsilyl)ethoxy]methyl}-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidine (<b>10</b>) (2.9 g, 7.1 mmol, 1.0 equiv), K<sub>2</sub>CO<sub>3</sub> (2.0 g, 14.2 mmol, 2.0 equiv) and <i>m</i>-nitrobenzeneboronic acid (1.4 g, 8.5 mmol, 1.2 equiv) in MeCN/H<sub>2</sub>O (30:15 mL) was degassed with Ar for 15 min. Pd(PPh<sub>3</sub>)<sub>4</sub> (820.5 mg, 0.7 mmol, 0.1 equiv) was then added, and the sealed vial was heated to 150 °C for 90 min in a microwave reactor. The reaction mixture was filtered through Celite, concentrated <i>in vacuo</i>, taken in EtOAc, washed with a saturated solution of NaCl, and dried over Na<sub>2</sub>SO<sub>4</sub>. After the removal of the solvent <i>in vacuo</i>, the crude product was adsorbed onto silica and purified by column chromatography. Pure fractions were evaporated to dryness to afford the desired compound as a yellow solid. Yield: 69% (1.9 g, 4.9 mmol). <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>): δ 12.54 (s, 1H), 9.03 (s, 1H), 8.21 (d, <i>J</i> = 7.7 Hz, 1H), 8.14 (dd, <i>J</i> = 8.0, 1.5 Hz, 1H), 8.04 (s, 1H), 7.57 (t, <i>J</i> = 7.7 Hz, 1H), 7.37 (s, 1H), 5.62 (s, 2H), 3.61 (t, <i>J</i> = 8.4 Hz, 2H), 0.97 (t, <i>J</i> = 8.4 Hz, 2H), −0.01 ppm (s, 9H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>): δ 158.5, 149.2, 144.7, 135.3, 133.8, 133.1, 132.0, 129.5, 123.4, 122.4, 120.7, 104.8, 73.0, 65.7, 17.1, −1.4 ppm. HRMS: <i>m</i>/<i>z</i> calcd for [C<sub>18</sub>H<sub>22</sub>O<sub>4</sub>N<sub>4</sub>Si + H]<sup>+</sup>, 387.1483; found, 387.1495.</div></div><div id="sec4_16_6_4" class="NLM_sec NLM_sec_level_4"><div id="ac_i45" class="anchor-spacer"></div><h5 class="article-section__title" id="_i45"> 6-Bromo-5-(3-nitrophenyl)-7-{[2-(trimethylsilyl)ethoxy]methyl}-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-4-ol</h5><div class="NLM_p last">5-(3-Nitrophenyl)-7-{[2-(trimethylsilyl)ethoxy]methyl}-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-4-ol (<b>11</b>) (726 mg, 1.9 mmol) was dissolved in DMF (40 mL), and NBS (669 mg, 3.8 mmol) was added. The reaction was stirred at room temperature for 4 h. Afterward, brine and EtOAc were added and the aqueous phase is extracted twice with EtOAc. The combined organic layers were washed with a saturated aqueous solution of Na<sub>2</sub>S<sub>2</sub>O<sub>3</sub> and brine, dried over Na<sub>2</sub>SO<sub>4</sub>, and concentrated <i>in vacuo</i>. The crude product was purified by flash chromatography (SiO<sub>2</sub>, DCM/MeOH) to yield the desired product. Yield: 96% (841 mg, 1.8 mmol). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>): δ 12.28 (s, 1H), 8.40–8.35 (m, 1H), 8.21 (ddd, <i>J</i> = 8.3, 2.3, 1.0 Hz, 1H), 8.06 (d, <i>J</i> = 3.8 Hz, 1H), 8.02–7.98 (m, 1H), 7.74 (app. t, dd, <i>J</i> = 8.0, 8.0 Hz, 1H), 5.61 (s, 2H), 3.60 (t, <i>J</i> = 7.9 Hz, 2H), 0.86 (t, <i>J</i> = 7.9 Hz, 2H), −0.06 ppm (s, 9H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>): δ 179.32, 156.94, 147.22, 145.29, 136.79, 133.48, 133.12, 131.98, 129.18, 124.78, 121.87, 117.13, 107.84, 105.68, 71.95, 65.84, 17.07, −1.42 ppm. HRMS: <i>m</i>/<i>z</i> calcd for [C<sub>18</sub>H<sub>21</sub>O<sub>4</sub>N<sub>4</sub>BrSi + H]<sup>+</sup>, 465.0588; found, 465.0588, calcd for [C<sub>18</sub>H<sub>21</sub>O<sub>4</sub>N<sub>4</sub><sup>81</sup>BrSi + H]<sup>+</sup>, 467.0568; found, 467.0567.</div></div><div id="sec4_16_6_5" class="NLM_sec NLM_sec_level_4"><div id="ac_i46" class="anchor-spacer"></div><h5 class="article-section__title" id="_i46"> 6-[4-(4-Methylpiperazin-1-yl)phenyl]-5-(3-nitrophenyl)-7-{[2-(trimethylsilyl)ethoxy]methyl}-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-4-ol (<b>12</b>)</h5><div class="NLM_p last">6-Bromo-5-(3-nitrophenyl)-7-{[2-(trimethylsilyl)ethoxy]methyl}-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-4-ol (500 mg, 1.1 mmol) and 1-methyl-4-[4-(4,4,5,5-tetramethyl-1,3,2-dioxaborolan-2-yl)phenyl]piperazine (390 mg, 1.3 mmol) were dissolved in 1,2-dimethoxyethane (DME)/water (3/1) (10 mL/mmol aryl bromide) and degassed with Argon for 30 min. K<sub>2</sub>CO<sub>3</sub> (223 mg, 1.6 mmol) and Pd(PPh<sub>3</sub>)<sub>4</sub> (10 mol%) were added, and the reaction was stirred in the microwave for 3 h at 105 °C. The reaction mixture was cooled to room temperature, filtered through Celite, diluted with EtOAc and water, and the organic layer was separated. The aqueous layer was extracted with aliquots of EtOAc, and the combined organic layers were dried over MgSO<sub>4</sub> and filtered. After the removal of the solvent <i>in vacuo</i>, the crude product was adsorbed onto silica and purified by flash column chromatography (SiO<sub>2</sub>, DCM/MeOH) to yield the desired product. Yield: 63% (377 mg, 0.67 mmol). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>): δ 12.13 (s, 1H), 8.23–8.11 (m, 1H), 8.04 (s, 1H), 8.03–7.98 (m, 1H), 7.61 (d, <i>J</i> = 7.8 Hz, 1H), 7.47 (app. t, dd, <i>J</i> = 8.0, 8.0 Hz, 1H), 7.18 (d, <i>J</i> = 8.8 Hz, 2H), 6.93 (d, <i>J</i> = 8.8 Hz, 2H), 5.38 (s, 2H), 3.48 (t, <i>J</i> = 8.0, 2H), 3.20–3.16 (m, 4H), 2.45–2.40 (m, 4H), 2.21 (s, 3H), 0.79 (t, <i>J</i> = 8.0 Hz, 2H), −0.09 ppm (s, 9H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>): δ 157.90, 150.68, 148.42, 147.04, 144.37, 136.91, 135.25, 134.22, 131.79, 128.60, 125.24, 120.62, 118.44, 114.71, 114.49, 105.13, 70.53, 65.70, 54.41, 47.13, 45.69, 24.91, 24.64, 17.28, −1.46 ppm. HRMS: <i>m</i>/<i>z</i> calcd for [C<sub>29</sub>H<sub>36</sub>O<sub>4</sub>N<sub>6</sub>Si + H]<sup>+</sup>, 561.2640; found, 561.2650.</div></div><div id="sec4_16_6_6" class="NLM_sec NLM_sec_level_4"><div id="ac_i47" class="anchor-spacer"></div><h5 class="article-section__title" id="_i47"> 4-Chloro-6-[4-(4-methylpiperazin-1-yl)phenyl]-5-(3-nitrophenyl)-7-{[2-(trimethylsilyl)ethoxy]methyl}-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidine</h5><div class="NLM_p last">6-[4-(4-Methylpiperazin-1-yl)phenyl]-5-(3-nitrophenyl)-7-{[2-(trimethylsilyl)ethoxy]methyl}-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-4-ol (<b>12</b>) (357 mg, 0.64 mmol) was dissolved in MeCN (25 mL), and benzyltriethylammonium chloride (290 mg, 1.27 mmol), <i>N</i>,<i>N</i>-dimethylaniline (161 μL, 1.27 mmol), and phosphoryl trichloride (1.2 mL, 12.8 mmol) were added. The reaction was stirred for 2 h at 85 °C. The reaction mixture was cooled to room temperature and quenched with water and an aqueous saturated solution of NaHCO<sub>3</sub>. The aqueous layer was extracted with aliquots of EtOAc, and the combined organic layers were then dried over MgSO<sub>4</sub> and filtered. After the removal of the solvent <i>in vacuo</i>, the crude product was adsorbed onto silica and purified by flash column chromatography (SiO<sub>2</sub>, DCM/MeOH) to yield the desired product. Yield: 92% (340 mg, 0.59 mmol). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.75 (s, 1H), 8.15–8.18 (m, 2H), 7.70–7.72 (m, 1H), 7.59–7.63 (m, 1H), 7.23–7.26 (m, 2H), 6.91–6.94 (m, 2H), 5.56 (s, 2H), 3.52 (t, <i>J</i> = 7.9 Hz, 2H), 3.16–3.20 (m, 4H), 2.39–2.43 (m 4H), 2.20 (s, 3H), 0.81 (t, <i>J</i> = 7.9 Hz, 2H), −0.09 ppm (s, 9H). LC/MS: <i>m</i>/<i>z</i> calcd for [C<sub>29</sub>H<sub>35</sub>ClN<sub>6</sub>O<sub>3</sub>Si + H]<sup>+</sup>, 579.23; found, 579.59.</div></div><div id="sec4_16_6_7" class="NLM_sec NLM_sec_level_4"><div id="ac_i48" class="anchor-spacer"></div><h5 class="article-section__title" id="_i48"> 3-{4-Chloro-6-[4-(4-methylpiperazin-1-yl)phenyl]-7-{[2-(trimethylsilyl)ethoxy]methyl}-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-5-yl}aniline</h5><div class="NLM_p last">The reaction was performed according to the <a class="ref internalNav" href="#sec4_16_4" aria-label="General Method for Aromatic Nitro Reduction">General Method for Aromatic Nitro Reduction</a>. 4-Chloro-6-[4-(4-methylpiperazin-1-yl)phenyl]-5-(3-nitrophenyl)-7-{[2-(trimethylsilyl)ethoxy]methyl}-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidine (535 mg, 0.92 mmol) was reduced and purified by flash column chromatography, and this yielded the desired product. Yield: 100% (507 mg, 0.92 mmol). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.67 (s, 1H), 7.27 (d, <i>J</i> = 8.8 Hz, 2H), 6.90 (d, <i>J</i> = 8.8 Hz, 3H), 6.47 (d, <i>J</i> = 1.0 Hz, 2H), 6.39 (d, <i>J</i> = 7.3 Hz, 1H), 5.53 (s, 2H), 4.99 (s, 2H), 3.52 (t, <i>J</i> = 7.9 Hz, 2H), 3.14–3.22 (m, 4H), 2.38–2.45 (m, 4H), 2.21 (s, 3H), 0.79 (t, <i>J</i> = 7.9 Hz, 2H), −0.10 ppm (s, 9H). LC/MS: <i>m</i>/<i>z</i> calcd for [C<sub>29</sub>H<sub>37</sub>ClN<sub>6</sub>OSi + H]<sup>+</sup>, 549.26; found, 549.19.</div></div><div id="sec4_16_6_8" class="NLM_sec NLM_sec_level_4"><div id="ac_i49" class="anchor-spacer"></div><h5 class="article-section__title" id="_i49"> <i>N</i>-{3-{4-Chloro-6-[4-(4-methylpiperazin-1-yl)phenyl]-7-{[2-(trimethylsilyl)ethoxy]methyl}-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-5-yl}phenyl}acrylamide (<b>13</b>)</h5><div class="NLM_p last">The reaction was performed according to the <a class="ref internalNav" href="#sec4_16_5" aria-label="General Method for Aniline Derivative Acylation">General Method for Aniline Derivative Acylation</a>. 3-{4-Chloro-6-[4-(4-methylpiperazin-1-yl)phenyl]-7-{[2-(trimethylsilyl)ethoxy]methyl}-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-5-yl}aniline (412 mg, 0.75 mmol, 1.0 equiv) was acylated, purified by flash column chromatography, and yielded the desired product as a yellow solid. Yield: 95% (432 mg, 0.72 mmol). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>): δ 10.14 (s, 1H), 8.71 (s, 1H), 7.66 (d, <i>J</i> = 8.2 Hz, 1H), 7.61 (s, 1H), 7.20–7.30 (m, 3H), 6.87–6.97 (m, 3H), 6.41 (dd, <i>J</i> = 16.9, 10.2 Hz, 1H), 6.22 (dd, <i>J</i> = 16.9, 2.0 Hz, 1H), 5.68–5.77 (m, 1H), 5.55 (s, 2H), 3.52 (t, <i>J</i> = 7.9 Hz, 2H), 3.14–3.22 (m, 4H), 2.39–2.42 (m, 4H), 2.19 (s, 3H), 0.79 (t, <i>J</i> = 7.9 Hz, 2H), −0.10 ppm (s, 9H). LC/MS: <i>m</i>/<i>z</i> calcd for [C<sub>32</sub>H<sub>39</sub>ClN<sub>6</sub>O<sub>2</sub>Si + H]<sup>+</sup>, 603.27; found, 603.13.</div></div><div id="sec4_16_6_9" class="NLM_sec NLM_sec_level_4"><div id="ac_i50" class="anchor-spacer"></div><h5 class="article-section__title" id="_i50"> <i>N</i>-{3-{4-Chloro-6-[4-(4-methylpiperazin-1-yl)phenyl]-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-5-yl}phenyl}acrylamide</h5><div class="NLM_p last">The reaction was performed according to the <a class="ref internalNav" href="#sec4_16_6" aria-label="General Method for SEM Deprotection">General Method for SEM Deprotection</a>. <i>N</i>-{3-{4-Chloro-6-[4-(4-methylpiperazin-1-yl)phenyl]-7-{[2-(trimethylsilyl)ethoxy]methyl}-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-5-yl}phenyl}acrylamide (<b>13</b>) (181 mg, 0.30 mmol) was deprotected, purified by flash column chromatography, and yielded the desired product. Yield: 86% (122 mg, 0.26 mmol). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>): δ 12.86 (s, 1H), 10.19 (s, 1H), 8.56 (s, 1H), 7.76–7.81 (m, 1H), 7.61 (s, 1H), 7.36 (app. t, dd, <i>J</i> = 7.9, 7.9 Hz, 1H), 7.27–7.32 (m, 2H), 7.06 (d, <i>J</i> = 7.9 Hz, 1H), 6.85–6.89 (m, 2H), 6.42 (dd, <i>J</i> = 16.9, 10.2 Hz, 1H), 6.23 (dd, <i>J</i> = 16.9, 2.0 Hz, 1H), 5.72–5.76 (m, 1H), 3.13–3.21 (m, 4H), 2.35–2.42 (m, 4H), 2.19 ppm (s, 3H). LC/MS: <i>m</i>/<i>z</i> calcd for [C<sub>26</sub>H<sub>25</sub>ClN<sub>6</sub>O + H]<sup>+</sup>, 473.19; found, 473.53.</div></div><div id="sec4_16_6_10" class="NLM_sec NLM_sec_level_4"><div id="ac_i51" class="anchor-spacer"></div><h5 class="article-section__title" id="_i51"> 2-Chloro-1-[(3-fluorobenzyl)oxy]-4-nitrobenzene</h5><div class="NLM_p last">2-Chloro-4-nitrophenol (4 g, 23.1 mmol, 1.0 equiv) and K<sub>2</sub>CO<sub>3</sub> (9.6 g, 69.2 mmol, 3.0 equiv) were dissolved in DMF (120 mL), and the suspension was heated to 75 °C. Following that, a solution of 3-fluorobenzylchloride (5 g, 34.6 mmol, 1.5 equiv) in DMF (80 mL) was added, and the reaction mixture was stirred at 75 °C until full conversion. The suspension was allowed to cool to room temperature and was poured into ice water (200 mL). The precipitate was filtered and washed with MeCN/H<sub>2</sub>O (1:1) and <i>n</i>-hexane. The product was dried further <i>in vacuo</i> and appeared as a white solid, which was utilized without further purification. Yield: 39% (2.54 g, 9.0 mmol). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.30–8.32 (d, <i>J</i> = 2.7 Hz, 1H), 8.24 (dd, <i>J</i> = 9.2, 2.7 Hz, 1H), 7.42–7.52 (m, 2H), 7.29–7.35 (m, 2H), 7.20 (td, <i>J</i> = 8.6, 1.9 Hz, 1H), 5.40 ppm (s, 2H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>): δ 163.1, 161.2, 158.6, 140.9, 138.4, 130.7, 130.6, 125.4, 124.5, 123.5, 122.0, 113.8, 70.1 ppm. LC/MS: <i>m</i>/<i>z</i> calcd for [C<sub>13</sub>H<sub>9</sub>ClFNO<sub>3</sub> + H]<sup>+</sup>, 282.03; found, n.d. (not ionizable).</div></div><div id="sec4_16_6_11" class="NLM_sec NLM_sec_level_4"><div id="ac_i52" class="anchor-spacer"></div><h5 class="article-section__title" id="_i52"> 3-Chloro-4-[(3-fluorbenzyl)oxy]aniline</h5><div class="NLM_p last">The reaction was performed according to the <a class="ref internalNav" href="#sec4_16_4" aria-label="General Method for Aromatic Nitro Reduction">General Method for Aromatic Nitro Reduction</a>. 2-Chloro-1-[(3-fluorobenzyl)oxy]-4-nitrobenzene (2.5 g, 8.9 mmol, 1.0 equiv), NH<sub>4</sub>Cl (4.3 g, 79.9 mmol, 9.0 equiv), and iron powder (1.5 g, 26.6 mmol, 3.0 equiv) were used, and all were dissolved in EtOH/H<sub>2</sub>O (4:1, 47 mL). The product was obtained without further purification as a beige solid. Yield: 96% (2.1 g, 8.5 mmol). <sup>1</sup>H NMR (500 MHz DMSO-<i>d</i><sub>6</sub>): δ 7.40–7.45 (m, 1H), 7.22–7.29 (m, 2H), 7.12–7.16 (m, 1H), 6.91 (d, <i>J</i> = 8.8 Hz, 1H), 6.66 (d, <i>J</i> = 2.3 Hz, 1H), 6.47 (dd, <i>J</i> = 8.8, 2.7 Hz, 1H), 5.03 (s, 2H), 4.94 ppm (s, 2H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>): δ 163.1, 161.1, 144.1, 140.3, 140.2, 130.3, 130.3, 123.2, 122.5, 116.9, 114.9, 113.2, 70.3 ppm. HRMS: <i>m</i>/<i>z</i> calcd for [C<sub>13</sub>H<sub>11</sub>ClFNO + H]<sup>+</sup>, 252.0586; found, 252.0592.</div></div><div id="sec4_16_6_12" class="NLM_sec NLM_sec_level_4"><div id="ac_i53" class="anchor-spacer"></div><h5 class="article-section__title" id="_i53"> 2-[(2-Chloro-4-nitrophenoxy)methyl]pyridine</h5><div class="NLM_p last">2-(Chloromethyl)pyridine (5.0 g, 30.5 mmol, 1.10 equiv) and K<sub>2</sub>CO<sub>3</sub> (7.6 g, 55.4 mmol, 2.00 equiv) were dissolved in DMF (40 mL), and the suspension was stirred for 30 min. Following that, 2-chloro-4-nitrophenol (4.8 g, 27.7 mmol, 1.00 equiv) and KI (232 mg, 1.4 mmol, 0.05 equiv) were added and the reaction mixture was stirred at 60 °C until full conversion. The suspension was allowed to cool to room temperature and was poured into ice water (120 mL). The precipitate was filtered and then washed with water. The product was dried further <i>in vacuo</i> and appeared as a white solid and was utilized without further purification. Yield: 97% (7.1 g, 26.8 mmol). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.60 (d, <i>J</i> = 4.2 Hz, 1H), 8.34 (d, <i>J</i> = 2.7 Hz, 1H), 8.23 (dd, <i>J</i> = 9.2, 2.7 Hz, 1H), 7.87–7.91 (m, 1H), 7.57 (d, <i>J</i> = 7.7 Hz, 1H), 7.47 (d, <i>J</i> = 9.2 Hz, 1H), 7.38 (dd, <i>J</i> = 6.9, 5.4 Hz, 1H), 5.46 ppm (s, 2H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>): δ 158.7, 155.1, 149.2, 140.9, 137.2, 125.4, 124.5, 123.3, 121.9, 121.6, 113.9, 71.7 ppm. HRMS: <i>m</i>/<i>z</i> calcd for [C<sub>12</sub>H<sub>9</sub>ClN<sub>2</sub>O<sub>3</sub> + H]<sup>+</sup>, 265.0375; found, 265.0365.</div></div><div id="sec4_16_6_13" class="NLM_sec NLM_sec_level_4"><div id="ac_i54" class="anchor-spacer"></div><h5 class="article-section__title" id="_i54"> 3-Chloro-4-(pyridin-2-ylmethoxy)aniline</h5><div class="NLM_p last">The reaction was performed according to the <a class="ref internalNav" href="#sec4_16_4" aria-label="General Method for Aromatic Nitro Reduction">General Method for Aromatic Nitro Reduction</a>. It used 2-[(2-chloro-4-nitrophenoxy)methyl]pyridine (7.0 g, 26.4 mmol, 1.0 equiv), NH<sub>4</sub>Cl (12.7 g, 238 mmol, 9.0 equiv), and iron powder (4.4 g, 79.3 mmol, 3.0 equiv), and all were dissolved in EtOH/H<sub>2</sub>O (4:1, 135 mL). The product was obtained without further purification as a brownish solid. Yield: 90% (5.6 g, 23.8 mmol). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.55 (dd, <i>J</i> = 3.8, 1.2 Hz, 1H), 7.82–7.86 (m, 1H), 7.54 (d, <i>J</i> = 7.7 Hz, 1H), 7.32 (dd, <i>J</i> = 6.1, 1.5 Hz, 1H), 6.91 (d, <i>J</i> = 8.4 Hz, 1H), 6.67 (d, <i>J</i> = 2.7 Hz, 1H), 6.47 (dd, <i>J</i> = 8.4, 2.3 Hz, 1H), 5.08 (s, 2H), 4.94 ppm (s, 2H). <sup>13</sup>C NMR (126 MHz DMSO-<i>d</i><sub>6</sub>): δ 157.0, 148.9, 144.3, 144.1, 136.9, 122.8, 122.3, 121.3, 116.5, 115.0, 113.2, 72.0 ppm. HRMS: <i>m</i>/<i>z</i> calcd for [C<sub>12</sub>H<sub>11</sub>ClN<sub>2</sub>O + H]<sup>+</sup>, 235.0633; found, 235.0644.</div></div><div id="sec4_16_6_14" class="NLM_sec NLM_sec_level_4"><div id="ac_i55" class="anchor-spacer"></div><h5 class="article-section__title" id="_i55"> 1-Benzyl-5-nitro-1<i>H</i>-indole</h5><div class="NLM_p last">Sodium hydride, 60% in mineral oil (148.0 mg, 3.7 mmol, 1.2 equiv), was dissolved in DMF (5 mL), and to it was added 5-nitro-1<i>H</i>-indole (500.0 mg, 3.1 mmol, 1.0 equiv) and stirred for 30 min at 0 °C. Following that, benzyl chloride (425.8 μL, 3.7 mmol, 1.2 equiv) was added, and the reaction mixture was stirred at room temperature until full conversion (2 h). The mixture was then poured into water and extracted with EtOAc. The combined organic layers were dried over Na<sub>2</sub>SO<sub>4</sub>, and the solvent was removed <i>in vacuo</i>. Column chromatography subsequently afforded the desired product. Yield: 98% (763.6 mg, 3.0 mmol). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.58 (d, <i>J</i> = 1.9 Hz, 1H), 8.00 (dd, <i>J</i> = 9.1, 2.1 Hz, 1H), 7.78 (d, <i>J</i> = 3.1 Hz, 1H), 7.68 (d, <i>J</i> = 9.1 Hz, 1H), 7.29–7.34 (m, 2H), 7.29–7.19 (m, 3H), 6.81 (d, <i>J</i> = 2.9 Hz, 1H), 5.53 ppm (s, 2H). <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>): δ 140.85, 138.68, 137.45, 132.98, 128.69, 127.63, 127.56, 127.07, 117.65, 116.56, 110.69, 104.03, 49.47, 39.52 ppm. LC/MS: <i>m</i>/<i>z</i> calcd for [C<sub>15</sub>H<sub>12</sub>O<sub>2</sub>N<sub>2</sub> + H]<sup>+</sup>, 253.10; found, 253.12.</div></div><div id="sec4_16_6_15" class="NLM_sec NLM_sec_level_4"><div id="ac_i56" class="anchor-spacer"></div><h5 class="article-section__title" id="_i56"> 1-Benzyl-1<i>H</i>-indol-5-amine</h5><div class="NLM_p last">The reaction was performed according to the <a class="ref internalNav" href="#sec4_16_4" aria-label="General Method for Aromatic Nitro Reduction">General Method for Aromatic Nitro Reduction</a>. It used 1-benzyl-5-nitro-1<i>H</i>-indole (740.0 mg, 2.9 mmol, 1.0 equiv), NH<sub>4</sub>Cl (1.42 g, 26.4 mmol, 9.0 equiv), and iron powder (491.4 mg, 8.8 mmol, 3.0 equiv), and all were dissolved in EtOH/H<sub>2</sub>O (4:1, 47 mL). Yield: 93% (606.4 mg, 2.73 mmol). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>): δ 7.28 (dd, <i>J</i> = 9.3, 5.1 Hz, 3H), 7.20–7.24 (m, 1H), 7.16 (s, 1H), 7.14 (s, 1H), 7.08 (d, <i>J</i> = 8.6 Hz, 1H), 6.69 (d, <i>J</i> = 2.1 Hz, 1H), 6.48 (dd, <i>J</i> = 8.6, 2.1 Hz, 1H), 6.17 (d, <i>J</i> = 3.0 Hz, 1H), 5.27 (s, 2H), 4.50 ppm (s, 2H). <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>): δ 141.44, 138.68, 129.73, 129.28, 128.59, 128.41, 127.15, 126.90, 111.81, 110.19, 103.53, 99.38, 49.10, 39.52 ppm. LC/MS: <i>m</i>/<i>z</i> calcd for [C<sub>15</sub>H<sub>14</sub>N<sub>2</sub> + H]<sup>+</sup>, 223.12; found, 223.18.</div></div><div id="sec4_16_6_16" class="NLM_sec NLM_sec_level_4"><div id="ac_i57" class="anchor-spacer"></div><h5 class="article-section__title" id="_i57"> 1-Benzyl-5-nitro-1<i>H</i>-indazole</h5><div class="NLM_p last">5-Nitroindazole (326 mg, 2.0 mmol) was dissolved in DMF (6 mL), and K<sub>2</sub>CO<sub>3</sub> (553 mg, 4.0 mmol) was added. Benzyl bromide (0.29 mL, 2.4 mmol) was added, and the reaction was stirred at room temperature for 5 h. The reaction mixture was quenched with water and an aqueous saturated solution of NaHCO<sub>3</sub>. The aqueous layer was extracted with aliquots of EtOAc, and the combined organic layers were then dried over MgSO<sub>4</sub> and filtered. After the removal of the solvent <i>in vacuo</i>, the crude product was adsorbed onto silica and purified by flash column chromatography (SiO<sub>2</sub>, DCM/MeOH) to yield the desired product. Yield: 41% (210 mg, 0.83 mmol). <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): δ 8.82 (d, <i>J</i> = 2.2 Hz, 1H), 8.44 (d, <i>J</i> = 0.7 Hz, 1H), 8.21 (dd, <i>J</i> = 9.2, 2.2 Hz, 1H), 7.94 (d, <i>J</i> = 9.2 Hz, 1H), 7.29–7.33 (m, 2H), 7.23–7.28 (m, 3H), 5.75 ppm (s, 2H), <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>): δ 141.73, 141.06, 136.76, 136.54, 128.62, 128.60, 127.74, 127.41, 127.39, 122.88, 121.09, 119.13, 110.64, 52.17 ppm. LC/MS: <i>m</i>/<i>z</i> calcd for [C<sub>14</sub>H<sub>11</sub>N<sub>3</sub>O<sub>2</sub> + H]<sup>+</sup>, 254.09; found, 254.12.</div></div><div id="sec4_16_6_17" class="NLM_sec NLM_sec_level_4"><div id="ac_i58" class="anchor-spacer"></div><h5 class="article-section__title" id="_i58"> 1-Benzyl-1<i>H</i>-indazol-5-amine</h5><div class="NLM_p last">The reaction was performed according to the <a class="ref internalNav" href="#sec4_16_4" aria-label="General Method for Aromatic Nitro Reduction">General Method for Aromatic Nitro Reduction</a>. It used 1-benzyl-5-nitro-1<i>H</i>-indazole (140 mg, 0.55 mmol) as the starting material. Yield: 98% (120 mg, 0.54 mmol). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>): δ 7.75 (s, 1H), 7.16–7.35 (m, 6H), 6.74–6.80 (m, 2H), 5.51 (s, 2H), 4.81 ppm (s, 2H). LC/MS: <i>m</i>/<i>z</i> calcd for [C<sub>14</sub>H<sub>13</sub>N<sub>3</sub> + H]<sup>+</sup>, 224.12; found, 224.21.</div></div><div id="sec4_16_6_18" class="NLM_sec NLM_sec_level_4"><div id="ac_i59" class="anchor-spacer"></div><h5 class="article-section__title" id="_i59"> 2-Benzyl-5-nitro-1<i>H</i>-benzo[<i>d</i>]imidazole</h5><div class="NLM_p last">2-Phenylacetic acid (1.36 g, 10.0 mmol) and 4-nitrobenzene-1,2-diamine (1.53 g, 10.0 mmol) were stirred in the microwave at 180 °C for 1 h. The crude mixture was dissolved in 100 mL of water and extracted with aliquots of EtOAc, and the combined organic layers were then dried over MgSO<sub>4</sub> and filtered. After the removal of the solvent <i>in vacuo</i>, the crude product was adsorbed onto silica and purified by flash column chromatography (C<sub>18</sub>-column, water/MeCN) to yield the desired product as an orange crystalline solid. Yield: 20% (507 mg, 2.0 mmol). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>): δ 13.00 (br s, 1H), 8.44 (br s, 1H), 8.07 (br s, 1H), 7.61 (br s, 1H), 7.29–7.40 (m, 4H), 7.21–7.29 (m, 1H), 4.26 ppm (s, 2H). LC/MS: <i>m</i>/<i>z</i> calcd for [C<sub>14</sub>H<sub>11</sub>N<sub>3</sub>O<sub>2</sub> + H]<sup>+</sup>, 254.09; found, 254.12.</div></div><div id="sec4_16_6_19" class="NLM_sec NLM_sec_level_4"><div id="ac_i60" class="anchor-spacer"></div><h5 class="article-section__title" id="_i60"> 2-Benzyl-1<i>H</i>-benzo[<i>d</i>]imidazol-5-amine</h5><div class="NLM_p last">The reaction was performed according to the <a class="ref internalNav" href="#sec4_16_4" aria-label="General Method for Aromatic Nitro Reduction">General Method for Aromatic Nitro Reduction</a>. 2-Benzyl-5-nitro-1<i>H</i>-benzo[<i>d</i>]imidazole (368 mg, 1.45 mmol) was reduced and purified by flash column chromatography to yield the desired product. Yield: 79% (255 mg, 1.14 mmol). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>): δ 7.26–7.33 (m, 4H), 7.18–7.25 (m, 1H), 7.14 (d, <i>J</i> = 8.5 Hz, 1H), 6.59 (d, <i>J</i> = 1.8 Hz, 1H), 6.45 (dd, <i>J</i> = 8.2, 2.1 Hz, 1H), 4.05 ppm (s, 2H). LC/MS: <i>m</i>/<i>z</i> calcd for [C<sub>14</sub>H<sub>13</sub>N<sub>3</sub> + H]<sup>+</sup>, 224.12; found, 224.16.</div></div><div id="sec4_16_6_20" class="NLM_sec NLM_sec_level_4"><div id="ac_i61" class="anchor-spacer"></div><h5 class="article-section__title" id="_i61"> 1-Nitro-4-phenoxybenzene</h5><div class="NLM_p last">Sodium hydride, 60% in mineral oil (170.1 mg, 4.25 mmol, 1.2 equiv), was dissolved in DMF (5 mL), and to it was added phenol (400.2 mg, 4.25 mmol, 1.2 equiv) and 1-fluoro-4-nitrobenzene (376 μL, 3.54 mmol, 1.0 equiv). The reaction mixture was then stirred at room temperature until full conversion. The mixture was poured into water and extracted with EtOAc. The combined organic layers were dried over Na<sub>2</sub>SO<sub>4</sub>, and the solvent removed <i>in vacuo</i>. Column chromatography afforded the desired product as a yellow oil. Yield: 66% (505.6 mg, 2.3 mmol). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.28–8.23 (m, 2H), 7.53–7.48 (m, 2H), 7.28–7.33 (m, 1H), 7.20 (dd, <i>J</i> = 8.6, 0.9 Hz, 2H), 7.15–7.10 ppm (m, 2H). <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>): δ 163.37, 154.74, 142.70, 131.04, 126.68, 126.00, 120.97, 117.81 ppm. LC/MS: <i>m</i>/<i>z</i> calcd for [C<sub>12</sub>H<sub>9</sub>NO<sub>3</sub> + H]<sup>+</sup>, 216.07; found, n.d. (not ionizable).</div></div><div id="sec4_16_6_21" class="NLM_sec NLM_sec_level_4"><div id="ac_i62" class="anchor-spacer"></div><h5 class="article-section__title" id="_i62"> 4-Phenoxyaniline</h5><div class="NLM_p last">The reaction was performed according to the <a class="ref internalNav" href="#sec4_16_4" aria-label="General Method for Aromatic Nitro Reduction">General Method for Aromatic Nitro Reduction</a>. It used 1-nitro-4-phenoxybenzene (480.0 mg, 2.23 mmol, 1.0 equiv), NH<sub>4</sub>Cl (1074 mg, 20.07 mmol, 9.0 equiv), and iron powder (373.7 mg, 6.69 mmol, 3.0 equiv), all dissolved in EtOH/H<sub>2</sub>O (16:4 mL). Column chromatography yielded the desired product as a dark oil. Yield: quant. (448.4 mg, 2.4 mmol). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>): δ 7.29 (dd, <i>J</i> = 8.5, 7.5 Hz, 2H), 7.01–6.96 (m, 1H), 6.84 (d, <i>J</i> = 7.9 Hz, 2H), 6.77–6.74 (m, 2H), 6.61–6.57 (m, 2H), 4.97 ppm (s, 2H). <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>): δ 158.99, 145.50, 145.42, 129.66, 121.69, 120.93, 116.38, 114.85, 39.52 ppm. LC/MS: <i>m</i>/<i>z</i> calcd for [C<sub>12</sub>H<sub>11</sub>NO + H]<sup>+</sup>, 186.09; found, 186.10.</div></div><div id="sec4_16_6_22" class="NLM_sec NLM_sec_level_4"><div id="ac_i63" class="anchor-spacer"></div><h5 class="article-section__title" id="_i63"> 2-(4-Nitrobenzyl)isoindoline-1,3-dione</h5><div class="NLM_p last">Isoindoline-1,3-dione (1000 mg, 6.8 mmol, 1 equiv), 1-(chloromethyl)-4-nitrobenzene (1283 mg, 7.5 mmol, 1.1 equiv), and K<sub>2</sub>CO<sub>3</sub> (1880 mg, 13.6 mmol, 2 equiv) were dissolved in DMF and stirred at room temperature until full conversion (16 h). The reaction mixture was washed with water and brine and extracted with EtOAc. The combined organic layers were dried over Na<sub>2</sub>SO<sub>4</sub>, and the solvent removed <i>in vacuo</i>. Column chromatography afforded the desired product. Yield: 89% (1700 mg, 6.02 mmol). <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.19 (d, <i>J</i> = 8.8 Hz, 2H), 7.93–7.91 (m, 2H), 7.89–7.86 (m, 2H), 7.60 (d, <i>J</i> = 8.9 Hz, 2H), 4.91 (s, 2H). <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>): δ 167.66, 146.82, 144.31, 134.65, 131.60, 128.47, 123.72, 123.33, 40.38. LC/MS: <i>m</i>/<i>z</i> calcd for [C<sub>15</sub>H<sub>10</sub>N<sub>2</sub>O<sub>4</sub> + H]<sup>+</sup>, 283.06; found, 282.16.</div></div><div id="sec4_16_6_23" class="NLM_sec NLM_sec_level_4"><div id="ac_i64" class="anchor-spacer"></div><h5 class="article-section__title" id="_i64"> 2-(4-Aminobenzyl)isoindoline-1,3-dione</h5><div class="NLM_p last">The reaction was performed according to the <a class="ref internalNav" href="#sec4_16_4" aria-label="General Method for Aromatic Nitro Reduction">General Method for Aromatic Nitro Reduction</a>. It used 2-(4-nitrobenzyl)isoindoline-1,3-dione (1700 mg, 6.02 mmol, 1.0 equiv), NH<sub>4</sub>Cl (3220 mg, 60.20 mmol, 10.0 equiv), and iron powder (1681 mg, 30.10 mmol, 5.0 equiv), all dissolved in EtOH/H<sub>2</sub>O (1:1). Column chromatography yielded the desired product. Yield: 75% (1136 mg, 4.50 mmol). <sup>1</sup>H NMR (700 MHz, DMSO-<i>d</i><sub>6</sub>): δ 7.88–7.86 (m, 2H), 7.85–7.82 (m, 2H), 6.98 (d, <i>J</i> = 8.4 Hz, 2H), 6.48 (d, 2H), 5.03 (s, 2H), 4.57 (s, 2H). <sup>13</sup>C NMR (176 MHz, DMSO-<i>d</i><sub>6</sub>): δ 167.69, 148.11, 134.48, 131.56, 128.69, 123.55, 123.09, 113.68, 40.65. TLC/MS: <i>m</i>/<i>z</i> calcd for [C<sub>15</sub>H<sub>12</sub>N<sub>2</sub>O<sub>2</sub> + H]<sup>+</sup>, 253.10; found, 253.00.</div></div><div id="sec4_16_6_24" class="NLM_sec NLM_sec_level_4"><div id="ac_i65" class="anchor-spacer"></div><h5 class="article-section__title" id="_i65"> 2-[(4-Nitrophenoxy)methyl]isoindoline-1,3-dione</h5><div class="NLM_p last">2-(Chloromethyl)isoindoline-1,3-dione (233 mg, 1.2 mmol) was dissolved in 1 mL of acetone, and NaI (180 mg, 1.2 mmol) and K<sub>2</sub>CO<sub>3</sub> (166 mg, 1.2 mmol) were added. After stirring for 15 min, 4-nitrophenol (166 mg, 1.2 mmol) was added. The reaction mixture was stirred at rt for 24 h and quenched with water and an aqueous saturated solution of NaHCO<sub>3</sub>. The aqueous phase was extracted with aliquots of EtOAc, and the combined organic layers were then dried over MgSO<sub>4</sub> and filtered. After removal of the solvent <i>in vacuo</i>, the crude product was adsorbed onto silica and purified by flash column chromatography (SiO<sub>2</sub>, petrolether/EtOAc) to yield the desired product as a yellow solid. Yield: 75% (270 mg, 0.91 mmol). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.23 (d, <i>J</i> = 9.3 Hz, 2H), 7.90–8.00 (m, 4H), 7.34 (d, <i>J</i> = 9.3 Hz, 2H), 5.73 ppm (s, 2H). LC/MS: <i>m</i>/<i>z</i> calcd for [C<sub>15</sub>H<sub>10</sub>N<sub>2</sub>O<sub>5</sub> + H]<sup>+</sup>, 299.07; found, n.d. (not ionizable).</div></div><div id="sec4_16_6_25" class="NLM_sec NLM_sec_level_4"><div id="ac_i66" class="anchor-spacer"></div><h5 class="article-section__title" id="_i66"> 2-[(4-Aminophenoxy)methyl]isoindoline-1,3-dione</h5><div class="NLM_p last">The reaction was performed according to the <a class="ref internalNav" href="#sec4_16_4" aria-label="General Method for Aromatic Nitro Reduction">General Method for Aromatic Nitro Reduction</a>. 2-[(4-nitrophenoxy)methyl]isoindoline-1,3-dione (100 mg, 0.34 mmol) was reduced and purified by flash column chromatography, and this yielded the desired product. Yield: 79% (72 mg, 0.27 mmol). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>): δ 7.85–7.99 (m, 4H), 6.74 (d, <i>J</i> = 8.8 Hz, 2H), 6.47 (d, <i>J</i> = 8.8 Hz, 2H), 5.39 (s, 2H), 4.77 ppm (br s, 2H). LC/MS: <i>m</i>/<i>z</i> calcd for [C<sub>15</sub>H<sub>12</sub>N<sub>2</sub>O<sub>3</sub> + H]<sup>+</sup>, 269.09; found, 269.17.</div></div><div id="sec4_16_6_26" class="NLM_sec NLM_sec_level_4"><div id="ac_i67" class="anchor-spacer"></div><h5 class="article-section__title" id="_i67"> [(4-Nitrophenoxy)methylene]dibenzene</h5><div class="NLM_p last">Sodium hydride (416 mg, 60% in mineral oil, 6.6 mmol) was dissolved in DMF, and diphenylmethanol (1011 mg, 5.5 mmol) was added at 0 °C. After 30 min, 1-fluoro-4-nitrobenzene (774 mg, 5.5 mmol) was added and the reaction was stirred for 2 h. The reaction was quenched with water and an aqueous saturated solution of NaHCO<sub>3</sub>. The aqueous phase was extracted with aliquots of EtOAc (3×), and the combined organic layers were then dried over MgSO<sub>4</sub> and filtered. After the removal of the solvent <i>in vacuo</i>, the crude product was adsorbed onto silica and purified by flash column chromatography (C<sub>18</sub>-column, water/MeCN) to yield the desired product as a yellow solid. Yield: 25% (420 mg, 1.4 mmol). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.16 (d, <i>J</i> = 9.3 Hz, 2H), 7.51 (d, <i>J</i> = 7.8 Hz, 4H), 7.37 (t, <i>J</i> = 7.8 Hz, 4H), 7.28 (app. t, dd, <i>J</i> = 7.3, 7.3 Hz, 2H), 7.23 (d, <i>J</i> = 9.3 Hz, 2H), 6.77 ppm (s, 1H). LC/MS: <i>m</i>/<i>z</i> calcd for [C<sub>19</sub>H<sub>15</sub>NO<sub>3</sub> + H]<sup>+</sup>, 306.11; found, n.d. (not ionizable).</div></div><div id="sec4_16_6_27" class="NLM_sec NLM_sec_level_4"><div id="ac_i68" class="anchor-spacer"></div><h5 class="article-section__title" id="_i68"> 4-(Benzhydryloxy)aniline</h5><div class="NLM_p last">The reaction was performed according to the <a class="ref internalNav" href="#sec4_16_4" aria-label="General Method for Aromatic Nitro Reduction">General Method for Aromatic Nitro Reduction</a>. [(4-Nitrophenoxy)methylene]dibenzene (420 mg, 1.4 mmol) was reduced and purified by flash column chromatography, and this yielded the desired product. Yield: 85% (323 mg, 1.2 mmol). <sup>1</sup>H NMR (700 MHz, DMSO-<i>d</i><sub>6</sub>): δ 7.51 (d, <i>J</i> = 8.0 Hz, 4H), 7.34 (app. t, dd, <i>J</i> = 7.9, 7.9 Hz, 4H), 7.24 (app. t, dd, <i>J</i> = 7.4, 7.4 Hz, 2H), 6.81 (d, <i>J</i> = 9.0 Hz, 2H), 6.54 (d, <i>J</i> = 9.0 Hz, 2H), 6.31 (s, 1H), 4.66 ppm (br s, 2H), <sup>13</sup>C NMR (176 MHz, DMSO-<i>d</i><sub>6</sub>): δ 148.7, 142.8, 142.2, 128.4, 127.3, 126.6, 117.2, 114.8, 81.0 ppm. LC/MS: <i>m</i>/<i>z</i> calcd for [C<sub>19</sub>H<sub>17</sub>NO + H]<sup>+</sup>, 276.14; found, 276.25.</div></div><div id="sec4_16_6_28" class="NLM_sec NLM_sec_level_4"><div id="ac_i69" class="anchor-spacer"></div><h5 class="article-section__title" id="_i69"> (<i>R</i>)-2-(1,3-Dioxoisoindolin-2-yl)-2-phenylacetic Acid</h5><div class="NLM_p last">(<i>R</i>)-2-Amino-2-phenylacetic acid (2.00 g, 13.2 mmol) and phthalic anhydride (1.96 g, 13.2 mmol) were dissolved in toluene (30 mL) and a catalytic amount of NEt<sub>3</sub> (91 μL, 0.66 mmol) was added. The reaction was stirred for 3 h at 130 °C using a Dean–Stark apparatus. The solvent was then removed under reduced pressure, and the crude product was washed with water and petroleum ether to yield the desired product as a white crystalline solid. Yield: 96% (3.56 g, 12.6 mmol). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>): δ 13.47 (br s, 1H), 7.79–8.03 (m, 4H), 7.42–7.56 (m, 2H), 7.24–7.42 (m, 3H), 6.04 ppm (s, 1H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>): δ 169.5, 167.5, 135.5, 131.5, 129.7, 128.6, 128.4, 124.0, 55.6 ppm. LC/MS: <i>m</i>/<i>z</i> calcd for [C<sub>16</sub>H<sub>11</sub>NO<sub>4</sub> + H]<sup>+</sup>, 282.08; found, 282.00.</div></div><div id="sec4_16_6_29" class="NLM_sec NLM_sec_level_4"><div id="ac_i70" class="anchor-spacer"></div><h5 class="article-section__title" id="_i70"> <i>N</i>-(4-Aminophenyl)-2-(1,3-dioxoisoindolin-2-yl)-2-phenylacetamide</h5><div class="NLM_p last">(<i>R</i>)-2-(1,3-Dioxoisoindolin-2-yl)-2-phenylacetic acid (300 mg, 1.0 mmol) was dissolved in DCM (3 mL) and stirred with thionyl chloride (725 μL, 10.0 mmol) for 16 h. The solvent and excess thionyl chloride were removed under reduced pressure, and the acid chloride was redissolved in DCM (3 mL). 4-Nitroaniline (207 mg, 1.5 mmol) and NEt<sub>3</sub> (1.25 mmol) were then added. The reaction mixture was subsequently stirred at rt for 16 h. After solvent removal (rotary evaporator), the crude product was purified by flash chromatography (C<sub>18</sub>-RP, water/MeCN), before it was reduced to the desired amine by the <a class="ref internalNav" href="#sec4_16_4" aria-label="General Method for Aromatic Nitro Reduction">General Method for Aromatic Nitro Reduction</a>. The crude product was used without further purification. Yield: 36% over two steps (143 mg, 0.36 mmol). LC/MS: <i>m</i>/<i>z</i> calcd for [C<sub>22</sub>H<sub>17</sub>N<sub>3</sub>O<sub>3</sub> + H]<sup>+</sup>, 372.13; found, 372.20.</div></div><div id="sec4_16_6_30" class="NLM_sec NLM_sec_level_4"><div id="ac_i71" class="anchor-spacer"></div><h5 class="article-section__title" id="_i71"> 4-Aminophenyl 2-(1,3-Dioxoisoindolin-2-yl)-2-phenylacetate</h5><div class="NLM_p last">(<i>R</i>)-2-(1,3-Dioxoisoindolin-2-yl)-2-phenylacetic acid (300 mg, 1.0 mmol) was dissolved in DCM (3 mL) and stirred with thionyl chloride (725 μL, 10.0 mmol) for 16 h. The solvent and excess thionyl chloride were removed under reduced pressure, and the acid chloride was redissolved in DCM (3 mL). 4-Nitrophenol (209 mg, 1.5 mmol) and NEt<sub>3</sub> (1.25 mmol) were then added. The reaction mixture was stirred at rt for 16 h. After solvent removal (rotary evaporator), the crude product was purified by flash chromatography on silica gel (petroleum ether/ethyl acetate) before it was reduced to the desired amine by the <a class="ref internalNav" href="#sec4_16_4" aria-label="General Method for Aromatic Nitro Reduction">General Method for Aromatic Nitro Reduction</a>. The crude product was purified by flash chromatography on silica gel (DCM/MeOH) and subsequent reverse phase chromatography (C<sub>18</sub>-RP, water/MeCN) to yield the desired product as an off-white solid. Yield: 15% over two steps (56 mg, 0.15 mmol). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>): δ 7.95–8.01 (m, 2H), 7.87–7.94 (m, 2H), 7.53 (d, <i>J</i> = 7.3 Hz, 2H), 7.33–7.42 (m, 3 H), 6.76 (d, <i>J</i> = 8.9 Hz, 2H), 6.54 (d, <i>J</i> = 8.9 Hz, 2H), 6.42 (s, 1H), 5.12 ppm (br s, 2H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>): δ 167.8, 167.4, 147.3, 140.8, 135.7, 135.0, 131.4, 129.5, 128.9, 128.8, 124.3, 122.0, 114.5, 55.4 ppm. HRMS: <i>m</i>/<i>z</i> calcd for [C<sub>22</sub>H<sub>16</sub>N<sub>2</sub>O<sub>4</sub> + H]<sup>+</sup>, 373.1188; found, 373.1183.</div></div><div id="sec4_16_6_31" class="NLM_sec NLM_sec_level_4"><div id="ac_i72" class="anchor-spacer"></div><h5 class="article-section__title" id="_i72"> <i>N</i>-{3-{4-{{3-Chloro-4-[(3-fluorobenzyl)oxy]phenyl}amino}-6-[4-(4-methylpiperazin-1-yl)phenyl]-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-5-yl}phenyl}acrylamide (<b>7a</b>)</h5><div class="NLM_p last">The reaction was performed according to the <a class="ref internalNav" href="#sec4_16_1" aria-label="General Method for Back-Pocket Aniline Coupling">General Method for Back-Pocket Aniline Coupling</a>. <i>N</i>-{3-{4-chloro-6-[4-(4-methylpiperazin-1-yl)phenyl]-7-{[2-(trimethylsilyl)ethoxy]methyl}-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-5-yl}phenyl}acrylamide (50 mg, 0.08 mmol) was substituted, deprotected, and purified by flash column chromatography, and this yielded the desired product. Yield: 30% (17 mg, 0.03 mmol). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>): δ 12.30 (s, 1H), 10.35 (s, 1H), 8.34 (s, 1H), 7.75–7.88 (m, 3H), 7.48–7.52 (m, 1H), 7.42–7.46 (m, 1H), 7.23–7.32 (m, 4H), 7.09–7.20 (m, 3H), 7.02 (dd, <i>J</i> = 8.9, 2.4 Hz, 1H), 6.88 (d, <i>J</i> = 8.9 Hz, 2H), 6.81 (s, 1H), 6.43 (dd, <i>J</i> = 17.0, 10.2 Hz, 1H), 6.25 (dd, <i>J</i> = 17.0, 1.8 Hz, 1H), 5.73–5.80 (m, 1H), 5.17 (s, 2H), 3.20 (br s, 4H), 2.56 (br s, 4H), 2.31 (br s, 3H). HRMS: <i>m</i>/<i>z</i> calcd for [C<sub>39</sub>H<sub>35</sub>ClFN<sub>7</sub>O<sub>2</sub> + H]<sup>+</sup>, 688.2598; found, 688.2594.</div></div><div id="sec4_16_6_32" class="NLM_sec NLM_sec_level_4"><div id="ac_i73" class="anchor-spacer"></div><h5 class="article-section__title" id="_i73"> <i>N</i>-{3-{4-{[3-Chloro-4-(pyridin-2-ylmethoxy)phenyl]amino}-6-[4-(4-methylpiperazin-1-yl)phenyl]-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-5-yl}phenyl}acrylamide (<b>7b</b>)</h5><div class="NLM_p last">The reaction was performed according to the <a class="ref internalNav" href="#sec4_16_1" aria-label="General Method for Back-Pocket Aniline Coupling">General Method for Back-Pocket Aniline Coupling</a>. <i>N</i>-{3-{4-chloro-6-[4-(4-methylpiperazin-1-yl)phenyl]-7-{[2-(trimethylsilyl)ethoxy]methyl}-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-5-yl}phenyl}acrylamide (95 mg, 0.16 mmol) was substituted, deprotected, and purified by flash column chromatography, and this yielded the desired product. Yield: 55% (57 mg, 0.09 mmol). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>): δ 12.31 (s, 1H), 10.76 (s, 1H), 8.54–8.58 (m, 1H), 8.31–8.34 (m, 1H), 7.91 (s, 1H), 7.80–7.88 (m, 3H), 7.45–7.54 (m, 2H), 7.27–7.37 (m, 3H), 7.10–7.17 (m, 2H), 6.99 (dd, <i>J</i> = 8.9, 2.4 Hz, 1H), 6.88 (d, <i>J</i> = 9.2 Hz, 2H), 6.80 (s, 1H), 6.56 (dd, <i>J</i> = 16.9, 10.2 Hz, 1H), 6.24 (dd, <i>J</i> = 16.9, 1.7 Hz, 1H), 5.72–5.77 (m, 1H), 5.19 (s, 2H), 3.25 (br s, 4H), 2.66 (br s, 4H), 2.37 ppm (br s, 3H). HRMS: <i>m</i>/<i>z</i> calcd for [C<sub>38</sub>H<sub>35</sub>ClN<sub>8</sub>O<sub>2</sub> + H]<sup>+</sup>, 671.2644; found, 671.2653.</div></div><div id="sec4_16_6_33" class="NLM_sec NLM_sec_level_4"><div id="ac_i74" class="anchor-spacer"></div><h5 class="article-section__title" id="_i74"> <i>N</i>-{3-{4-{{4-[(1,3-Dioxoisoindolin-2-yl)methyl]phenyl}amino}-6-[4-(4-methylpiperazin-1-yl)phenyl]-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-5-yl}phenyl}acrylamide (<b>7h</b>)</h5><div class="NLM_p last">The reaction was performed according to the <a class="ref internalNav" href="#sec4_16_1" aria-label="General Method for Back-Pocket Aniline Coupling">General Method for Back-Pocket Aniline Coupling</a>. <i>N</i>-{3-{4-chloro-6-[4-(4-methylpiperazin-1-yl)phenyl]-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-5-yl}phenyl}acrylamide (30 mg, 0.06 mmol) was substituted and purified by flash column chromatography, and this yielded the desired product. Yield: 30% (13 mg, 0.02 mmol). <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>): δ 12.27 (s, 1H), 10.29 (s, 1H), 8.32 (s, 1H), 7.81–7.90 (m, 5H), 7.77 (s, 1H), 7.51 (app. t, dd, <i>J</i> = 7.9 Hz, <i>J</i> = 7.9 Hz 1H), 7.33 (d, <i>J</i> = 8.4 Hz, 2H), 7.29 (d, <i>J</i> = 8.8 Hz, 2H), 7.17–7.22 (m, 3H), 6.87 (d, <i>J</i> = 9.2 Hz, 2H), 6.79 (s, 1H), 6.40 (dd, <i>J</i> = 16.9, 10.2 Hz, 1H), 6.23 (dd, <i>J</i> = 16.9, 1.8 Hz, 1H), 5.73 (dd, <i>J</i> = 10.2, 1.8 Hz, 1H), 4.68 (s, 2H), 3.17 (br s, 4H), 2.45 (br s, 4H), 2.24 ppm (br s, 3H). <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>): δ 167.65, 163.39, 152.60, 150.62, 150.49, 150.01, 139.81, 138.57, 135.81, 134.51, 132.12, 131.66, 131.56, 130.48, 129.94, 128.38, 128.16, 127.23, 125.76, 123.16, 121.10, 120.82, 119.07, 118.76, 114.63, 109.28, 103.69, 54.28, 47.03, 45.44, 40.42 ppm. HRMS: <i>m</i>/<i>z</i> calcd for [C<sub>41</sub>H<sub>36</sub>N<sub>8</sub>O<sub>3</sub> + H]<sup>+</sup>, 689.2983; found, 689.2990.</div></div><div id="sec4_16_6_34" class="NLM_sec NLM_sec_level_4"><div id="ac_i75" class="anchor-spacer"></div><h5 class="article-section__title" id="_i75"> <i>N</i>-{3-{4-{{4-[(1,3-Dioxoisoindolin-2-yl)methoxy]phenyl}amino}-6-[4-(4-methylpiperazin-1-yl)phenyl]-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-5-yl}phenyl}acrylamide (<b>7i</b>)</h5><div class="NLM_p last">The reaction was performed according to the <a class="ref internalNav" href="#sec4_16_1" aria-label="General Method for Back-Pocket Aniline Coupling">General Method for Back-Pocket Aniline Coupling</a>. <i>N</i>-{3-{4-chloro-6-[4-(4-methylpiperazin-1-yl)phenyl]-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-5-yl}phenyl}acrylamide (33 mg, 0.07 mmol) was substituted and purified by flash column chromatography, and this yielded the desired product. Yield: 20% (10 mg, 0.01 mmol). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 12.24 (s, 1H), 10.32 (s, 1H), 8.30 (s, 1H), 7.83–7.97 (m, 5H), 7.78 (s, 1H), 7.53 (app. t, dd, <i>J</i> = 7.82, 7.82 Hz, 1H), 7.31 (m, 4H), 7.21 (d, <i>J</i> = 7.34 Hz, 1H), 7.00 (d, <i>J</i> = 8.80 Hz, 2H), 6.88 (d, <i>J</i> = 8.80 Hz, 2H), 6.71 (s, 1H), 6.42 (dd, <i>J</i> = 17.00, 9.90 Hz, 1H), 6.24 (dd, <i>J</i> = 17.00, 1.96 Hz, 1H), 5.76 (dd, <i>J</i> = 9.80, 2.00 Hz, 1H), 5.50 (s, 2H), 3.20 (br s, 4H), 2.55 (br s, 4H), 2.31 ppm (br s, 3H). HRMS: <i>m</i>/<i>z</i> calcd for [C<sub>41</sub>H<sub>36</sub>N<sub>8</sub>O<sub>4</sub> + H]<sup>+</sup>, 705.2932; found, 705.2941.</div></div><div id="sec4_16_6_35" class="NLM_sec NLM_sec_level_4"><div id="ac_i76" class="anchor-spacer"></div><h5 class="article-section__title" id="_i76"> <i>N</i>-{3-{4-{[4-(Benzhydryloxy)phenyl]amino}-6-[4-(4-methylpiperazin-1-yl)phenyl]-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-5-yl}phenyl}acrylamide (<b>7j</b>)</h5><div class="NLM_p last">The reaction was performed according to the <a class="ref internalNav" href="#sec4_16_1" aria-label="General Method for Back-Pocket Aniline Coupling">General Method for Back-Pocket Aniline Coupling</a>. <i>N</i>-{3-{4-Chloro-6-[4-(4-methylpiperazin-1-yl)phenyl]-7-{[2-(trimethylsilyl)ethoxy]methyl}-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-5-yl}phenyl}acrylamide (50 mg, 0.08 mmol) was substituted and purified by flash column chromatography. The SEM-protected intermediate (26 mg, 0.03 mmol) was dissolved in DMF (1 mL), and ethane-1,2-diamine (6.0 μL, 0.09 mmol) and tetra-<i>n</i>-butylammonium fluoride (0.3 mL, 1 M in THF, 0.31 mmol) were added. The reaction mixture was stirred at room temperature, 0.1 mbar for 16 h. The reaction mixture was quenched with water and an aqueous saturated solution of NaHCO<sub>3</sub>. The aqueous layer was extracted with aliquots of DCM (3×), and the combined organic layers were then dried over MgSO<sub>4</sub> and filtered. After removal of the solvent <i>in vacuo</i>, the crude product was adsorbed onto silica and purified by flash column chromatography (SiO<sub>2</sub>, DCM/MeOH) to yield the desired product. Yield: 30% (18 mg, 0.03 mmol). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>): δ 12.24 (s, 1H), 10.32 (s, 1H), 8.26 (s, 1H), 7.83 (d, <i>J</i> = 8.2 Hz, 1H), 7.76 (s, 1H), 7.40–7.54 (m, 5H), 7.16–7.39 (m, 10H), 6.84–6.96 (m, 3H), 6.62 (s, 1H), 6.34–6.47 (m, 2H), 6.19–6.27 (m, 1H), 5.72–5.81 (m, 1H), 3.22 (br s, 4H), 2.63 (br s, 3H), 2.36 ppm (br s, 4H). HRMS: <i>m</i>/<i>z</i> calcd for [C<sub>45</sub>H<sub>41</sub>N<sub>7</sub>O<sub>2</sub> + H]<sup>+</sup>, 712.3395; found, 712.3399.</div></div><div id="sec4_16_6_36" class="NLM_sec NLM_sec_level_4"><div id="ac_i77" class="anchor-spacer"></div><h5 class="article-section__title" id="_i77"> <i>N</i>-{3-{6-[4-(4-Methylpiperazin-1-yl)phenyl]-4-{{4-[2-(1-oxoisoindolin-2-yl)-2-phenylacetamido]phenyl}amino}-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-5-yl}phenyl}acrylamide (<b>7k</b>)</h5><div class="NLM_p last">The reaction was performed according to the <a class="ref internalNav" href="#sec4_16_1" aria-label="General Method for Back-Pocket Aniline Coupling">General Method for Back-Pocket Aniline Coupling</a>. <i>N</i>-{3-{4-chloro-6-[4-(4-methylpiperazin-1-yl)phenyl]-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-5-yl}phenyl}acrylamide (50 mg, 0.11 mmol) was substituted and purified by flash column chromatography, and this yielded the desired product. Yield: 27% (23 mg, 0.03 mmol). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>): δ 12.28 (s, 1H), 10.49 (s, 1H), 10.35 (s, 1H), 8.33 (s, 1H), 7.86 (d, <i>J</i> = 8.2 Hz, 1H), 7.80 (s, 1H), 7.75 (d, <i>J</i> = 7.6 Hz, 1H), 7.48–7.61 (m, 6H), 7.43 (d, <i>J</i> = 7.0 Hz, 2H), 7.34–7.41 (m, 5H), 7.26–7.31 (m, 2H), 7.21 (d, <i>J</i> = 7.6 Hz, 1H), 6.83–6.89 (m, 2H), 6.79 (s, 1H), 6.40 (d, <i>J</i> = 10.1 Hz, 1H), 6.18–6.29 (m, 2H), 5.75 (dd, <i>J</i> = 10.2, 1.7 Hz, 1H), 4.83 (d, <i>J</i> = 18.0 Hz, 1H), 3.95 (d, <i>J</i> = 18.0 Hz, 1H), 3.11–3.19 (m, 4H), 2.42 (br s, 4H), 2.20 ppm (s, 3H), <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>): δ 168.2, 168.1, 163.9, 153.1, 151.2, 150.9, 150.6, 142.9, 140.3, 136.4, 136.0, 135.7, 133.7, 132.5, 132.2, 132.1, 132.0, 130.5, 129.5, 129.0, 128.9, 128.4, 127.8, 126.3, 124.1, 123.4, 121.5, 121.3, 120.3, 119.9, 119.2, 115.1, 109.8, 104.0, 58.7, 54.9, 48.8, 47.6, 46.1 ppm. HRMS: <i>m</i>/<i>z</i> calcd for [C<sub>48</sub>H<sub>43</sub>N<sub>9</sub>O<sub>3</sub> + H]<sup>+</sup>, 794.3567; found, 794.3567.</div></div><div id="sec4_16_6_37" class="NLM_sec NLM_sec_level_4"><div id="ac_i78" class="anchor-spacer"></div><h5 class="article-section__title" id="_i78"> 4-{{5-(3-Acrylamidophenyl)-6-[4-(4-methylpiperazin-1-yl)phenyl]-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-4-yl}amino}phenyl-2-(1-oxoisoindolin-2-yl)-2-phenylacetate (<b>7l</b>)</h5><div class="NLM_p last">The reaction was performed according to the <a class="ref internalNav" href="#sec4_16_1" aria-label="General Method for Back-Pocket Aniline Coupling">General Method for Back-Pocket Aniline Coupling</a>. <i>N</i>-{3-{4-chloro-6-[4-(4-methylpiperazin-1-yl)phenyl]-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-5-yl}phenyl}acrylamide (30 mg, 0.06 mmol) was substituted and purified by flash column chromatography, and this yielded the desired product. Yield: 23% (11 mg, 0.01 mmol). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>): δ 12.35 (s, 1H), 10.65 (s, 1H), 8.34 (s, 1H), 7.89 (d, <i>J</i> = 8.2 Hz, 1H), 7.85 (s, 1H), 7.77 (d, <i>J</i> = 7.6 Hz, 1H), 7.62 (td, <i>J</i> = 7.4, 1.1 Hz, 1H), 7.57 (d, <i>J</i> = 7.3 Hz, 1H), 7.42–7.54 (m, 9H), 7.30–7.34 (m, 2H), 7.19 (d, <i>J</i> = 7.6 Hz, 1H), 7.06–7.10 (m, 2H), 6.89–6.93 (m, 2H), 6.87 (s, 1H), 6.52 (dd, <i>J</i> = 16.9, 10.2 Hz, 1H), 6.31 (s, 1H), 6.23 (dd, <i>J</i> = 16.9, 1.7 Hz, 1H), 5.72–5.76 (m, 1H), 4.62 (d, <i>J</i> = 17.4 Hz, 1H), 4.06 (d, <i>J</i> = 17.7 Hz, 1H), 2.88 ppm (br s, 3H). HRMS: <i>m</i>/<i>z</i> calcd for [C<sub>48</sub>H<sub>43</sub>N<sub>8</sub>O<sub>4</sub> + H]<sup>+</sup>, 795.3402; found, 795.3407.</div></div><div id="sec4_16_6_38" class="NLM_sec NLM_sec_level_4"><div id="ac_i79" class="anchor-spacer"></div><h5 class="article-section__title" id="_i79"> <i>N</i>-{3-{4-[(4-Hydroxyphenyl)amino]-6-[4-(4-methylpiperazin-1-yl)phenyl]-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-5-yl}phenyl}acrylamide (<b>7m</b>)</h5><div class="NLM_p last">The reaction was performed according to the <a class="ref internalNav" href="#sec4_16_1" aria-label="General Method for Back-Pocket Aniline Coupling">General Method for Back-Pocket Aniline Coupling</a>. <i>N</i>-{3-{4-chloro-6-[4-(4-methylpiperazin-1-yl)phenyl]-7-{[2-(trimethylsilyl)ethoxy]methyl}-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-5-yl}phenyl}acrylamide (50 mg, 0.08 mmol) was substituted, deprotected, and purified by flash column chromatography, and this yielded the desired product. Yield: 18% (8 mg, 0.02 mmol). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>): δ 12.17 (s, 1H), 10.31 (s, 1H), 9.14 (s, 1H), 8.26 (s, 1H), 7.85 (d, <i>J</i> = 8.3 Hz, 1H), 7.78 (s, 1H), 7.52 (app. t, dd, <i>J</i> = 7.8, 7.8 Hz, 1H), 7.25–7.34 (m, 2H), 7.14–7.23 (m, 3H), 6.82–6.90 (m, 2H), 6.62–6.69 (m, 2H), 6.56 (s, 1H), 6.42 (dd, <i>J</i> = 17.1, 10.3 Hz, 1H), 6.25 (dd, <i>J</i> = 17.1, 2.0 Hz, 1H), 5.76 (dd, <i>J</i> = 10.0, 1.7 Hz, 1H), 3.17 (br s, 4H), 2.25 (s, 4H), 1.23 ppm (br s, 4H). HRMS: <i>m</i>/<i>z</i> calcd for [C<sub>32</sub>H<sub>31</sub>N<sub>7</sub>O<sub>2</sub> + H]<sup>+</sup>, 546.2618; found, 546.2613.</div></div><div id="sec4_16_6_39" class="NLM_sec NLM_sec_level_4"><div id="ac_i80" class="anchor-spacer"></div><h5 class="article-section__title" id="_i80"> 4-Chloro-5-(3-nitrophenyl)-7-{[2-(trimethylsilyl)ethoxy]methyl}-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidine (<b>15</b>)</h5><div class="NLM_p last">4-Chloro-5-iodo-7-{[2-(trimethylsilyl)ethoxy]methyl}-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidine (<b>10</b>) (200 mg, 490 μmol, 1.00 equiv), K<sub>2</sub>CO<sub>3</sub> (135 mg, 980 μmol, 2.00 equiv) and (3-nitrophenyl)boronic acid (98 mg, 588 μmol, 1.20 equiv) were dissolved in MeCN/H<sub>2</sub>O (2:1, 2 mL) and degassed with argon for 15 min in a pressure tube. Following that, Pd(PPh<sub>3</sub>)<sub>4</sub> (85 mg, 73.5 μmol, 0.15 equiv) was added and the reaction mixture was heated at 80 °C overnight. After cooling, the solution was filtered through Celite, and the filtrate was added to EtOAc (15 mL). The organic layer was washed with an aqueous saturated solution of NaCl and dried over Na<sub>2</sub>SO<sub>4</sub>, and the solvent was removed <i>in vacuo</i>. Column chromatography yielded the desired product. Yield: 51% (101 mg, 250 mmol). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.77 (s, 1H), 8.38 (t, <i>J</i> = 1.9 Hz, 1H), 8.23–8.27 (m, 1H), 8.21 (s, 1H), 8.03 (dt, <i>J</i> = 8.0, 1.2 Hz, 1H), 7.77 (app.t, dd, <i>J</i> = 8.0, 8.0 Hz, 1H), 5.72 (s, 2H), 3.61 (t, <i>J</i> = 8.0 Hz, 2H), 0.87 (t, <i>J</i> = 8.0 Hz, 2H), −0.07 ppm (s, 9H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>): δ 151.9, 150.9, 150.7, 147.4, 136.5, 134.0, 130.9, 129.5, 124.3, 122.0, 113.6, 73.1, 66.1, 17.1, −1.4 ppm. HRMS: <i>m</i>/<i>z</i> calcd for [C<sub>18</sub>H<sub>21</sub>ClN<sub>4</sub>O<sub>3</sub>Si + H]<sup>+</sup>, 405.1144; found, 405.1163.</div></div><div id="sec4_16_6_40" class="NLM_sec NLM_sec_level_4"><div id="ac_i81" class="anchor-spacer"></div><h5 class="article-section__title" id="_i81"> 3-{4-Chloro-7-{[2-(trimethylsilyl)ethoxy]methyl}-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-5-yl}aniline (<b>16</b>)</h5><div class="NLM_p last">4-Chloro-5-(3-nitrophenyl)-7-{[2-(trimethylsilyl)ethoxy]methyl}-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidine (<b>15</b>) (100 mg, 0.3 mmol, 1.0 equiv), NH<sub>4</sub>Cl (118 mg, 2.3 mmol, 9.0 equiv), and iron powder (41.4 mg, 0.8 mmol, 3.0 equiv) were added to a flask under an argon atmosphere and were subsequently taken in EtOH/H<sub>2</sub>O (4:1, 1.6 mL) after which the reaction mixture was stirred at 80 °C until full conversion. The suspension was allowed to cool to room temperature, filtered through Celite, and concentrated <i>in vacuo</i>. The resulting residue was extracted with CH<sub>2</sub>Cl<sub>2</sub>, after which the solvent was removed <i>in vacuo</i> and to afford the desired product as a yellow solid without further purification required. Yield: quant. (93 mg, 0.3 mmol). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.69 (s, 1H), 7.84 (s, 1H), 7.07 (app. t, dd, <i>J</i> = 7.6, 7.6 Hz, 1H), 6.71 (s, 1H), 6.64 (d, <i>J</i> = 7.6 Hz, 1H), 6.58 (dd, <i>J</i> = 8.0, 1.1 Hz, 1H), 5.68 (s, 2H), 5.13 (br s, 2H), 3.57 (t, <i>J</i> = 7.7 Hz, 2H), 0.84 (t, <i>J</i> = 7.7 Hz, 2H), −0.09 ppm (s, 9H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>): δ 151.6, 150.8, 150.4, 148.1, 132.7, 129.2, 128.3, 117.8, 116.9, 115.7, 114.2, 112.9, 72.8, 66.0, 17.1, −1.5 ppm. HRMS: <i>m</i>/<i>z</i> calcd for [C<sub>18</sub>H<sub>23</sub>ClN<sub>4</sub>OSi + H]<sup>+</sup>, 375.1402; found, 375.1417.</div></div><div id="sec4_16_6_41" class="NLM_sec NLM_sec_level_4"><div id="ac_i82" class="anchor-spacer"></div><h5 class="article-section__title" id="_i82"> <i>N</i>-{3-{4-Chloro-7-{[2-(trimethylsilyl)ethoxy]methyl}-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-5-yl}phenyl}acrylamide (<b>17</b>)</h5><div class="NLM_p last">3-{4-Chloro-7-{[2-(trimethylsilyl)ethoxy]methyl}-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-5-yl}aniline (<b>16</b>) (100 mg, 267 μmol, 1.0 equiv) was taken up in dry THF (2 mL), and to it was added DIPEA (465 μL, 2.7 mmol, 10.0 equiv). At 0 °C, a solution of acryloyl chloride (33 μL, 400 μmol, 1.5 equiv) in THF (1 mL) was added and the reaction mixture was stirred until full conversion. Subsequently, to it was added an aqueous saturated solution of NaHCO<sub>3</sub>. The aqueous solution was extracted with CH<sub>2</sub>Cl<sub>2</sub>, the solvent was dried over Na<sub>2</sub>SO<sub>4</sub> and removed <i>in vacuo</i>. Column chromatography of the residue afforded the desired product as a yellow solid. Yield: 79% (90 mg, 211 μmol). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>): δ 10.22 (s, 1H), 8.72 (s, 1H), 7.96 (s, 1H), 7.90 (s, 1H), 7.67 (d, <i>J</i> = 8.0 Hz, 1H), 7.40 (app. t, dd, <i>J</i> = 8.0, 8.0 Hz, 1H), 7.22 (d, <i>J</i> = 7.7 Hz, 1H), 6.46 (dd, <i>J</i> = 17.2, 10.3 Hz, 1H), 6.27 (dd, <i>J</i> = 17.2, 1.9 Hz, 1H), 5.76 (dd, <i>J</i> = 10.3, 1.9 Hz, 1H), 5.70 (s, 2H), 3.59 (t, <i>J</i> = 8.0 Hz, 2H), 0.85 (t, <i>J</i> = 8.0 Hz, 2H), −0.09 ppm (s, 9H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>): δ 163.2, 151.7, 150.8, 150.6, 138.7, 132.8, 131.8, 129.7, 128.3, 126.9, 125.3, 121.0, 118.2, 115.9, 114.1, 72.9, 66.0, 17.1, −1.5 ppm. HRMS: <i>m</i>/<i>z</i> calcd for [C<sub>21</sub>H<sub>25</sub>ClN<sub>4</sub>O<sub>2</sub>Si + H]<sup>+</sup>, 429.1508; found, 429.1520.</div></div><div id="sec4_16_6_42" class="NLM_sec NLM_sec_level_4"><div id="ac_i83" class="anchor-spacer"></div><h5 class="article-section__title" id="_i83"> <i>N</i>-[3-(4-Chloro-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-5-yl)phenyl]acrylamide</h5><div class="NLM_p last">The reaction was performed according to the <a class="ref internalNav" href="#sec4_16_6" aria-label="General Method for SEM Deprotection">General Method for SEM Deprotection</a>. <i>N</i>-{3-{4-Chloro-7-{[2-(trimethylsilyl)ethoxy]methyl}-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-5-yl}phenyl}acrylamide (<b>17</b>) (565.0 mg. 1.3 mmol) was utilized as starting material, and repeated flash chromatography (SiO<sub>2</sub>, followed by C<sub>18</sub> column) yielded the final product. Yield: 90% (353 mg, 1.18 mmol). <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>): δ 12.79 (s, 1H), 10.22 (s, 1H), 8.64 (s, 1H), 7.86 (s, 1H), 7.78 (d, <i>J</i> = 2.2 Hz, 1H), 7.68 (d, <i>J</i> = 8.1 Hz, 1H), 7.39 (app. t, dd, <i>J</i> = 7.9 Hz, 1H), 7.23 (d, <i>J</i> = 7.7 Hz, 1H), 6.47 (dd, <i>J</i> = 17.2, 10.1 Hz, 1H), 6.27 (dd, <i>J</i> = 17.2, 1.8 Hz, 1H), 5.77 ppm (dd, <i>J</i> = 10.1, 2.0 Hz, 1H). LC/MS: <i>m</i>/<i>z</i> calcd for [C<sub>15</sub>H<sub>11</sub>ClN<sub>4</sub>O + H]<sup>+</sup>, 299.07; found, 299.33.</div></div><div id="sec4_16_6_43" class="NLM_sec NLM_sec_level_4"><div id="ac_i84" class="anchor-spacer"></div><h5 class="article-section__title" id="_i84"> <i>N</i>-{3-{4-{{3-Chloro-4-{[3-fluorbenzyl]oxy}phenyl}amino}-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-5-yl}phenyl}acrylamide (<b>14a</b>)</h5><div class="NLM_p last">The reaction was performed according to the <a class="ref internalNav" href="#sec4_16_1" aria-label="General Method for Back-Pocket Aniline Coupling">General Method for Back-Pocket Aniline Coupling</a>. It used <i>N</i>-{3-{4-chloro-7-{[2-(trimethylsilyl)ethoxy]methyl}-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-5-yl}phenyl}acrylamide (60 mg, 139.9 μmol, 1.0 equiv), conc. HCl (37%) (1.7 μL, 21.0 μmol, 0.15 equiv), and 3-chloro-4-[(3-fluorobenzyl)oxy]aniline (52.8 mg, 209.8 μmol, 1.50 equiv). Flash chromatography yielded the desired product. Yield: 44% (32 mg, 61.8 μmol). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>): δ 12.10 (br s, 1H), 10.30 (s, 1H), 8.34 (s, 1H), 7.98 (s, 1H), 7.86 (d, <i>J</i> = 2.3 Hz, 1H), 7.63 (d, <i>J</i> = 8.0 Hz, 1H), 7.53 (s, 1H), 7.42–7.48 (m, 2H), 7.40 (d, <i>J</i> = 2.3 Hz, 1H), 7.34 (dd, <i>J</i> = 8.8, 2.3 Hz, 1H), 7.24–7.32 (m, 3H), 7.17 (dd, <i>J</i> = 8.8, 1.9 Hz, 1H), 7.13 (d, <i>J</i> = 8.8 Hz, 1H), 6.47 (dd, <i>J</i> = 16.8, 10.3 Hz, 1H), 6.28 (dd, <i>J</i> = 16.8, 1.5 Hz, 1H), 5.78 (dd, <i>J</i> = 10.3, 1.5 Hz, 1H), 5.19 ppm (s, 2H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>): δ 163.4, 153.4, 151.5, 148.8, 139.7, 139.6, 135.4, 133.7, 131.7, 130.5, 130.4, 129.5, 127.0, 123.8, 123.2, 122.3, 121.6, 121.2, 120.4, 119.2, 115.3, 114.7, 114.6, 114.5, 114.0, 113.8, 100.7, 69.4 ppm. HRMS: <i>m</i>/<i>z</i> calcd for [C<sub>28</sub>H<sub>21</sub>ClFN<sub>5</sub>O<sub>2</sub> + H]<sup>+</sup>, 514.1441; found, 514.1449.</div></div><div id="sec4_16_6_44" class="NLM_sec NLM_sec_level_4"><div id="ac_i85" class="anchor-spacer"></div><h5 class="article-section__title" id="_i85"> <i>N</i>-{3-{4-{[3-Chloro-4-(pyridin-2-ylmethoxy)phenyl]amino}-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-5-yl}phenyl}acrylamid (<b>14b</b>)</h5><div class="NLM_p last">The reaction was performed according to the <a class="ref internalNav" href="#sec4_16_1" aria-label="General Method for Back-Pocket Aniline Coupling">General Method for Back-Pocket Aniline Coupling</a>. It used <i>N</i>-{3-{4-chloro-7-{[2-(trimethylsilyl)ethoxy]methyl}-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-5-yl}phenyl}acrylamide (60 mg, 139.9 μmol, 1.0 equiv), conc. HCl (37%) (1.7 μL, 21.0 μmol, 0.15 equiv), and 3-chloro-4-(pyridin-2-ylmethoxy)aniline (49.2 mg, 209.8 μmol, 1.50 equiv). Flash chromatography yielded the desired product. Yield: 62% (43 mg, 86.5 μmol). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>): δ 12.11 (br s, 1H), 10.31 (s, 1H), 8.58 (d, <i>J</i> = 4.2 Hz, 1H), 8.34 (s, 1H), 7.98 (s, 1H), 7.87 (d, <i>J</i> = 2.3 Hz, 1H), 7.85 (dd, <i>J</i> = 7.7, 1.5 Hz, 1H), 7.63 (d, <i>J</i> = 8.0 Hz, 1H), 7.56 (d, <i>J</i> = 8.0 Hz 1H), 7.54 (s, 1H), 7.45 (app. t, dd, <i>J</i> = 7.7, 7.7 Hz, 1H), 7.40 (d, <i>J</i> = 2.3 Hz, 1H), 7.32–7.37 (m, 2H), 7.26 (d, <i>J</i> = 7.3 Hz, 1H), 7.14 (d, <i>J</i> = 9.2 Hz, 1H), 6.47 (dd, <i>J</i> = 16.8, 10.3 Hz, 1H), 6.27 (dd, <i>J</i> = 16.8, 1.5 Hz, 1H), 5.77 (dd, <i>J</i> = 10.3, 1.5 Hz, 1H), 5.23 ppm (s, 2H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>): δ 163.4, 156.4, 153.4, 151.5, 150.9, 149.0, 148.9, 139.6, 137.0, 135.4, 133.6, 131.7, 129.5, 127.0, 123.8, 122.9, 122.3, 121.6, 121.3, 121.0, 120.4, 119.2, 117.7, 115.3, 114.4, 100.7, 71.2 ppm. HRMS: <i>m</i>/<i>z</i> calcd for [C<sub>27</sub>H<sub>21</sub>ClN<sub>6</sub>O<sub>2</sub> + H]<sup>+</sup>, 497.1487; found, 497.1499.</div></div><div id="sec4_16_6_45" class="NLM_sec NLM_sec_level_4"><div id="ac_i86" class="anchor-spacer"></div><h5 class="article-section__title" id="_i86"> <i>N</i>-{3-{4-[(1-Benzyl-1<i>H</i>-indol-5-yl)amino]-7-{[2-(trimethylsilyl)ethoxy]methyl}-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-5-yl}phenyl}acrylamide</h5><div class="NLM_p last">The reaction was performed according to the <a class="ref internalNav" href="#sec4_16_1" aria-label="General Method for Back-Pocket Aniline Coupling">General Method for Back-Pocket Aniline Coupling</a>. It used <i>N</i>-{3-{4-chloro-7-{[2-(trimethylsilyl)ethoxy]methyl}-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-5-yl}phenyl}acrylamide (<b>17</b>) (80.0 mg, 0.19 mmol, 1.0 equiv), conc. HCl (37%) (2.3 μL, 0.03 mmol, 0.15 equiv), and 1-benzyl-1<i>H</i>-indol-5-amine (62.2 mg, 0.28 mmol, 1.50 equiv). The product was isolated by column chromatography (EtOAc/PE) as a white solid. Yield: 72% (82.7 mg, 0.13 mmol). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>): δ 10.33 (s, 1H), 8.34 (s, 1H), 8.04 (s, 1H), 7.91 (d, <i>J</i> = 2.0 Hz, 1H), 7.67 (dd, <i>J</i> = 8.2, 1.1 Hz, 1H), 7.54 (s, 1H), 7.51 (d, <i>J</i> = 7.8 Hz, 1H), 7.49–7.47 (m, 1H), 7.36 (s, 1H), 7.33 (d, <i>J</i> = 8.8 Hz, 1H), 7.29 (t, <i>J</i> = 7.3 Hz, 3H), 7.23 (t, <i>J</i> = 7.3 Hz, 1H), 7.18 (d, <i>J</i> = 1.4 Hz, 1H), 7.16 (s, 1H), 7.04 (dd, <i>J</i> = 8.8, 2.0 Hz, 1H), 6.46 (dd, <i>J</i> = 17.0, 10.2 Hz, 1H), 6.42 (dd, <i>J</i> = 3.1, 0.6 Hz, 1H), 6.28 (dd, <i>J</i> = 17.0, 1.9 Hz, 1H), 5.78 (dd, <i>J</i> = 10.1, 1.9 Hz, 1H), 5.60 (s, 2H), 5.39 (s, 2H), 3.62–3.56 (m, 2H), 0.89–0.84 (m, 2H), −0.06 ppm (s, 9H). LC/MS: <i>m</i>/<i>z</i> calcd for [C<sub>36</sub>H<sub>38</sub>N<sub>6</sub>O<sub>2</sub>Si + H]<sup>+</sup>, 615.29; found, 615.47.</div></div><div id="sec4_16_6_46" class="NLM_sec NLM_sec_level_4"><div id="ac_i87" class="anchor-spacer"></div><h5 class="article-section__title" id="_i87"> <i>N</i>-{3-{4-[(1-Benzyl-1<i>H</i>-indol-5-yl)amino]-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-5-yl}phenyl}acrylamide (<b>14c</b>)</h5><div class="NLM_p last">The reaction was performed according to the <a class="ref internalNav" href="#sec4_16_6" aria-label="General Method for SEM Deprotection">General Method for SEM Deprotection</a>. It used <i>N</i>-{3-{4-[(1-benzyl-1<i>H</i>-indol-5-yl)amino]-7-{[2-(trimethylsilyl)ethoxy]methyl}-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-5-yl}phenyl}acrylamide (80.0 mg. 0.13 mmol), and HPLC purification yielded the final product. Yield: 10% (6.9 mg, 0.01 mmol). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>): δ 12.02 (s, 1H), 10.32 (s, 1H), 8.28 (s, 1H), 8.00 (s, 1H), 7.93 (s, 1H), 7.67 (d, <i>J</i> = 8.1 Hz, 1H), 7.52–7.46 (m, 2H), 7.37–7.26 (m, 6H), 7.24 (d, <i>J</i> = 6.8 Hz, 1H), 7.17 (d, <i>J</i> = 7.5 Hz, 2H), 7.04 (d, <i>J</i> = 8.8 Hz, 1H), 6.46 (dd, <i>J</i> = 16.9, 10.1 Hz, 1H), 6.42 (s, 1H), 6.28 (d, <i>J</i> = 17.0 Hz, 1H), 5.77 (d, <i>J</i> = 10.2 Hz, 1H), 5.38 ppm (s, 2H). HRMS: <i>m</i>/<i>z</i> calcd for [C<sub>30</sub>H<sub>24</sub>ON<sub>6</sub> + H]<sup>+</sup>, 485.2084; found, 485.2080.</div></div><div id="sec4_16_6_47" class="NLM_sec NLM_sec_level_4"><div id="ac_i88" class="anchor-spacer"></div><h5 class="article-section__title" id="_i88"> <i>N</i>-{3-{4-[(1-Benzyl-1<i>H</i>-indazol-5-yl)amino]-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-5-yl}phenyl}acrylamide (<b>14d</b>)</h5><div class="NLM_p last">The reaction was performed according to the <a class="ref internalNav" href="#sec4_16_1" aria-label="General Method for Back-Pocket Aniline Coupling">General Method for Back-Pocket Aniline Coupling</a>. <i>N</i>-{3-{4-Chloro-6-[4-(4-methylpiperazin-1-yl)phenyl]-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-5-yl}phenyl}acrylamide (50 mg, 0.12 mmol) was substituted, deprotected, and purified by flash column chromatography to yield the desired product. Yield: 66% (37 mg, 0.08 mmol). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>): δ 12.05–12.18 (m, 1H), 10.34 (s, 1H), 8.33 (s, 1H), 8.23 (d, <i>J</i> = 1.5 Hz, 1H), 8.05 (s, 1H), 8.01 (s, 1H), 7.67 (d, <i>J</i> = 8.2 Hz, 1H), 7.59 (d, <i>J</i> = 8.9 Hz, 1H), 7.52 (s, 1H), 7.48 (t, <i>J</i> = 7.8 Hz, 1H), 7.40 (d, <i>J</i> = 2.4 Hz, 1H), 7.22–7.32 (m, 5H), 7.17–7.21 (m, 2H), 6.47 (dd, <i>J</i> = 16.9, 10.2 Hz, 1H), 6.28 (dd, <i>J</i> = 17.1, 1.8 Hz, 1H), 5.75–5.80 (m, 1H), 5.62 ppm (s, 2H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>): δ 163.9, 154.5, 151.8, 151.6, 140.1, 138.1, 136.8, 136.1, 133.3, 133.2, 132.2, 130.1, 129.0, 127.9, 127.7, 127.7, 124.4, 124.3, 122.7, 121.9, 119.8, 118.2, 115.8, 111.7, 110.3, 101.2, 52.4 ppm. HRMS: <i>m</i>/<i>z</i> calcd for [C<sub>29</sub>H<sub>23</sub>N<sub>7</sub>O + H]<sup>+</sup>, 486.2042; found, 486.2036.</div></div><div id="sec4_16_6_48" class="NLM_sec NLM_sec_level_4"><div id="ac_i89" class="anchor-spacer"></div><h5 class="article-section__title" id="_i89"> <i>N</i>-{3-{4-[(2-Benzyl-1<i>H</i>-benzo[<i>d</i>]imidazol-5-yl)amino]-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-5-yl}phenyl}acrylamide (<b>14e</b>)</h5><div class="NLM_p last">The reaction was performed according to the <a class="ref internalNav" href="#sec4_16_1" aria-label="General Method for Back-Pocket Aniline Coupling">General Method for Back-Pocket Aniline Coupling</a>. <i>N</i>-{3-{4-Chloro-6-[4-(4-methylpiperazin-1-yl)phenyl]-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-5-yl}phenyl}acrylamide (50 mg, 0.12 mmol) was substituted, deprotected, and purified by flash column chromatography, and this yielded the desired product. Yield: 75% (43 mg, 0.09 mmol). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>): δ 12.11 (d, <i>J</i> = 2.1 Hz, 1H), 10.35 (s, 1H), 8.39–8.42 (m, 1H), 8.36 (s, 1H), 8.26 (d, <i>J</i> = 1.5 Hz, 1H), 8.01 (s, 1H), 7.67–7.71 (m, 1H), 7.47–7.52 (m, 2H), 7.44 (s, 1H), 7.40 (d, <i>J</i> = 2.4 Hz, 1H), 7.27–7.36 (m, 6H), 7.02 (dd, <i>J</i> = 9.3, 2.0 Hz, 1H), 6.47 (dd, <i>J</i> = 16.9, 10.2 Hz, 1H), 6.28 (dd, <i>J</i> = 16.9, 2.0 Hz, 1H), 5.75–5.80 (m, 1H), 5.60 ppm (s, 2H), <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>): δ 163.9, 154.3, 151.7, 151.6, 146.1, 140.2, 137.6, 136.1, 133.3, 132.2, 130.2, 129.0, 128.3, 128.2, 127.7, 124.4, 124.1, 123.1, 121.9, 121.8, 119.8, 118.3, 117.8, 115.7, 109.3, 101.3, 56.7 ppm. HRMS: <i>m</i>/<i>z</i> calcd for [C<sub>29</sub>H<sub>23</sub>N<sub>7</sub>O + H]<sup>+</sup>, 486.2042; found, 486.2035.</div></div><div id="sec4_16_6_49" class="NLM_sec NLM_sec_level_4"><div id="ac_i90" class="anchor-spacer"></div><h5 class="article-section__title" id="_i90"> <i>N</i>-{3-{4-[(4-Benzylphenyl)amino]-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-5-yl}phenyl}acrylamid (<b>14f</b>)</h5><div class="NLM_p last">The reaction was performed according to the <a class="ref internalNav" href="#sec4_16_1" aria-label="General Method for Back-Pocket Aniline Coupling">General Method for Back-Pocket Aniline Coupling</a>. It used <i>N</i>-{3-{4-chloro-7-{[2-(trimethylsilyl)ethoxy]methyl}-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-5-yl}phenyl}acrylamide (80.0 mg, 0.19 mmol, 1.0 equiv), HCl (25%) (3.4 μL, 0.03 mmol, 0.15 equiv), and commercially available 4-benzylaniline (61.5 mg, 0.34 mmol, 1.80 equiv). The product was isolated by HPLC purification as a white solid. Yield: 3% (2.8 mg, 6.3 nmol). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>): δ 12.11–12.06 (m, 1H), 10.29 (s, 1H), 8.32 (s, 1H), 7.99–7.92 (m, 1H), 7.69–7.65 (m, 1H), 7.47 (d, <i>J</i> = 8.5 Hz, 3H), 7.40–7.37 (m, 2H), 7.26 (d, <i>J</i> = 7.2 Hz, 3H), 7.22–7.19 (m, 2H), 7.19–7.15 (m, 1H), 7.13 (d, <i>J</i> = 8.5 Hz, 2H), 6.49–6.42 (m, 1H), 6.29–6.22 (m, 1H), 5.79–5.74 (m, 1H), 3.87 ppm (s, 2H). HRMS: <i>m</i>/<i>z</i> calcd for [C<sub>28</sub>H<sub>23</sub>ON<sub>5</sub> + H]<sup>+</sup>, 446.1975; found, 446.1962.</div></div><div id="sec4_16_6_50" class="NLM_sec NLM_sec_level_4"><div id="ac_i91" class="anchor-spacer"></div><h5 class="article-section__title" id="_i91"> <i>N</i>-{3-{4-[(4-Phenoxyphenyl)amino]-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-5-yl}phenyl}acrylamide (<b>14g</b>)</h5><div class="NLM_p last">The reaction was performed according to the <a class="ref internalNav" href="#sec4_16_1" aria-label="General Method for Back-Pocket Aniline Coupling">General Method for Back-Pocket Aniline Coupling</a>. It used <i>N</i>-[3-(4-chloro-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-5-yl)phenyl]acrylamide (80.0 mg, 0.19 mmol, 1.0 equiv), HCl (25%) (3.4 μL, 0.03 mmol, 0.15 equiv), and 4-phenoxyaniline (62.2 mg, 0.34 mmol, 1.80 equiv). The product was isolated by HPLC purification as a white solid. Yield: 2% (1.9 mg, 4.3 μmol). <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>): δ 12.14 (s, 1H), 10.30 (s, 1H), 8.34 (s, 1H), 7.97 (s, 1H), 7.65 (d, <i>J</i> = 7.8 Hz, 1H), 7.58 (d, <i>J</i> = 8.9 Hz, 2H), 7.47 (app. t, dd, <i>J</i> = 7.9, 7.9 Hz, 1H), 7.42 (s, 1H), 7.36 (dd, <i>J</i> = 8.4, 7.5 Hz, 2H), 7.28 (d, <i>J</i> = 7.6 Hz, 1H), 7.09 (t, <i>J</i> = 7.4 Hz, 1H), 6.96 (dd, <i>J</i> = 15.6, 8.4 Hz, 4H), 6.46 (dd, <i>J</i> = 17.0, 10.2 Hz, 1H), 6.26 (dd, <i>J</i> = 17.0, 1.8 Hz, 1H), 5.77 (dd, <i>J</i> = 10.2, 1.8 Hz, 1H). HRMS: <i>m</i>/<i>z</i> calcd for [C<sub>27</sub>H<sub>22</sub>O<sub>2</sub>N<sub>5</sub> + H]<sup>+</sup>, 448.1768; found, 448.1753.</div></div><div id="sec4_16_6_51" class="NLM_sec NLM_sec_level_4"><div id="ac_i92" class="anchor-spacer"></div><h5 class="article-section__title" id="_i92"> <i>N</i>-{3-{4-{{4-[(1,3-Dioxoisoindolin-2-yl)methyl]phenyl}amino}-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-5-yl}phenyl}acrylamide (<b>14h</b>)</h5><div class="NLM_p last">The reaction was performed according to the <a class="ref internalNav" href="#sec4_16_1" aria-label="General Method for Back-Pocket Aniline Coupling">General Method for Back-Pocket Aniline Coupling</a>. <i>N</i>-[3-(4-chloro-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-5-yl)phenyl]acrylamide (40 mg, 0.13 mmol) was substituted and purified by flash column chromatography, and this yielded the desired product. Yield: 67% (47 mg, 0.09 mmol). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>): δ 12.13 (s, 1H), 10.31 (s, 1H), 8.33 (s, 1H), 7.94 (s, 1H), 7.81–7.90 (m, 4H), 7.63–7.68 (m, 1H), 7.53 (s, 1H), 7.51 (d, <i>J</i> = 2.14 Hz, 2H), 7.45 (app. t, dd, <i>J</i> = 7.93, 7.93 Hz, 1H), 7.41 (d, <i>J</i> = 2.44 Hz, 1H), 7.20–7.26 (m, 3H), 6.44 (dd, <i>J</i> = 16.94, 10.22 Hz, 1H), 6.25 (dd, <i>J</i> = 17.09, 1.83 Hz, 1H), 5.73–5.76 (m, 1H), 4.70 ppm (s, 2H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>): δ 167.71, 163.38, 153.45, 151.52, 151.00, 139.68, 138.81, 135.48, 134.56, 131.78, 131.58, 130.64, 129.62, 128.00, 127.16, 123.90, 123.22, 121.68, 120.10, 119.29, 117.81, 115.24, 101.01, 40.51 ppm. HRMS: <i>m</i>/<i>z</i> calcd for [C<sub>30</sub>H<sub>22</sub>N<sub>6</sub>O<sub>3</sub> + H]<sup>+</sup>, 515.1832; found, 515.1825.</div></div><div id="sec4_16_6_52" class="NLM_sec NLM_sec_level_4"><div id="ac_i93" class="anchor-spacer"></div><h5 class="article-section__title" id="_i93"> <i>N</i>-{3-{4-{{4-[(1,3-Dioxoisoindolin-2-yl)methoxy]phenyl}amino}-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-5-yl}phenyl}acrylamide (<b>14i</b>)</h5><div class="NLM_p last">The reaction was performed according to the <a class="ref internalNav" href="#sec4_16_1" aria-label="General Method for Back-Pocket Aniline Coupling">General Method for Back-Pocket Aniline Coupling</a>. <i>N</i>-[3-(4-Chloro-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-5-yl)phenyl]acrylamide (40 mg, 0.13 mmol) was substituted and purified by flash column chromatography, and this yielded the desired product. Yield: 83% (59 mg, 0.11 mmol). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 12.11 (br s, 1H), 10.33 (s, 1H), 8.31 (s, 1H), 7.88–7.97 (m, 5H), 7.64–7.68 (m, 1H), 7.45–7.52 (m, 3H), 7.41 (br s, 1H), 7.39 (d, <i>J</i> = 2.44 Hz, 1H), 7.26 (d, <i>J</i> = 7.63 Hz, 1H), 7.00–7.05 (m, 2H), 6.46 (dd, <i>J</i> = 17.09, 10.07 Hz, 1H), 6.27 (dd, <i>J</i> = 17.09, 1.83 Hz, 1H), 5.77 (dd, <i>J</i> = 10.10, 1.80 Hz, 1H), 5.52 ppm (s, 2H). HRMS: <i>m</i>/<i>z</i> calcd for [C<sub>30</sub>H<sub>22</sub>N<sub>6</sub>O<sub>4</sub> + H]<sup>+</sup>, 531.1781; found, 531.1774.</div></div><div id="sec4_16_6_53" class="NLM_sec NLM_sec_level_4"><div id="ac_i94" class="anchor-spacer"></div><h5 class="article-section__title" id="_i94"> <i>N</i>-{3-{4-{[4-(Benzhydryloxy)phenyl]amino}-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-5-yl}phenyl}acrylamide (<b>14j</b>)</h5><div class="NLM_p last">The reaction was performed according to the <a class="ref internalNav" href="#sec4_16_1" aria-label="General Method for Back-Pocket Aniline Coupling">General Method for Back-Pocket Aniline Coupling</a>. <i>N</i>-[3-(4-Chloro-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-5-yl)phenyl]acrylamide (65 mg, 0.22 mmol) was substituted and purified by flash column chromatography, and this yielded the desired product. Yield: 43% (50 mg, 0.09 mmol). <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>): δ 12.05 (br s, 1H), 10.30 (s, 1H), 8.27 (s, 1H), 7.94 (br s, 1H), 7.66 (d, <i>J</i> = 7.7 Hz, 1H), 7.48 (d, <i>J</i> = 7.7 Hz, 4H), 7.41 (d, <i>J</i> = 9.2 Hz, 2H), 7.31–7.37 (m, 6H), 7.28 (s, 1H), 7.23–7.26 (m, 3H), 6.95 (d, <i>J</i> = 8.8 Hz, 2H), 6.42–6.50 (m, 2H), 6.27 (d, <i>J</i> = 16.9 Hz, 1H), 5.77 ppm (d, <i>J</i> = 11.0 Hz, 1H). <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>): δ 163.37, 153.68, 152.96, 151.30, 151.09, 141.67, 139.65, 135.54, 132.72, 131.76, 129.63, 128.49, 127.49, 127.14, 126.54, 123.84, 121.92, 121.25, 119.27, 117.78, 116.08, 115.23, 100.54, 80.12 ppm. HRMS: <i>m</i>/<i>z</i> calcd for [C<sub>34</sub>H<sub>27</sub>N<sub>5</sub>O<sub>2</sub> + H]<sup>+</sup>, 538.2243; found, 538.2242.</div></div><div id="sec4_16_6_54" class="NLM_sec NLM_sec_level_4"><div id="ac_i95" class="anchor-spacer"></div><h5 class="article-section__title" id="_i95"> 4-Chloro-5-(4-fluoro-3-nitrophenyl)-7-{[2-(trimethylsilyl)ethoxy]methyl}-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidine (<b>22</b>)</h5><div class="NLM_p last">The reaction was performed according to the <a class="ref internalNav" href="#sec4_16_2" aria-label="General Method for Suzuki–Miyaura Coupling">General Method for Suzuki–Miyaura Coupling</a>. It used 4-chloro-5-iodo-7-{[2-(trimethylsilyl)ethoxy]methyl}-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidine (<b>10</b>) (500 mg, 1.22 mmol) and (4-fluoro-3-nitrophenyl)boronic acid (271 mg, 1.5 mmol) that were coupled and purified by flash column chromatography. This afforded the desired compound. Yield: 84% (434 mg, 1.03 mmol). <sup>1</sup>H NMR (700 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.76 (s, 1H), 8.30 (dd, <i>J</i> = 7.1, 2.2 Hz, 1H), 8.17 (s, 1H), 8.01–7.98 (m, 1H), 7.71 (dd, <i>J</i> = 11.2, 8.6 Hz, 1H), 5.71 (s, 2H), 3.60 (t, <i>J</i> = 8.0 Hz, 2H), 0.86 (t, <i>J</i> = 8.0 Hz, 2H), −0.08 (s, 9H). <sup>13</sup>C NMR (176 MHz, DMSO-<i>d</i><sub>6</sub>): δ 153.16, 151.78, 150.96, 150.73, 137.74, 136.61, 130.93, 129.64, 127.03, 118.28, 114.15, 112.73, 73.08, 66.33, 17.07, −1.43. LC/MS: <i>m</i>/<i>z</i> calcd for [C<sub>18</sub>H<sub>20</sub>ClFN<sub>4</sub>O<sub>3</sub>Si + H]<sup>+</sup>, 423.09; found, 423.14.</div></div><div id="sec4_16_6_55" class="NLM_sec NLM_sec_level_4"><div id="ac_i96" class="anchor-spacer"></div><h5 class="article-section__title" id="_i96"> <i>N</i>-[3-Chloro-4-(pyridin-2-ylmethoxy)phenyl]-5-(4-fluoro-3-nitrophenyl)-7-{[2-(trimethylsilyl)ethoxy]methyl}-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-4-amine (<b>23a</b>)</h5><div class="NLM_p last">The reaction was performed according to the <a class="ref internalNav" href="#sec4_16_1" aria-label="General Method for Back-Pocket Aniline Coupling">General Method for Back-Pocket Aniline Coupling</a>. It used 4-chloro-5-(4-fluoro-3-nitrophenyl)-7-{[2-(trimethylsilyl)ethoxy]methyl}-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidine (<b>22</b>) (426 mg, 1.01 mmol, 1.0 equiv) and 3-chloro-4-(pyridin-2-ylmethoxy)aniline (354.6 mg, 1.51 mmol, 1.5 equiv). Purification by flash column chromatography yielded the desired product. Yield: 61% (381 mg, 0.61 mmol). <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.60–8.58 (m, 1H), 8.41 (s, 1H), 8.22–8.18 (m, 2H), 7.90–7.86 (m, 2H), 7.81 (s, 1H), 7.71 (d, <i>J</i> = 2.6 Hz, 1H), 7.66 (dd, <i>J</i> = 11.2, 8.7 Hz, 1H), 7.55 (d, <i>J</i> = 5.9 Hz, 1H), 7.38–7.36 (m, 1H), 7.34 (dd, <i>J</i> = 8.9, 2.6 Hz, 1H), 7.13 (d, <i>J</i> = 9.0 Hz, 1H), 5.61 (s, 2H), 5.23 (s, 2H), 3.59 (t, <i>J</i> = 8.0 Hz, 2H), 0.86 (t, <i>J</i> = 8.0 Hz, 2H), −0.07 (s, 9H). <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>): δ 156.58, 153.99, 151.88, 149.31, 149.23, 137.25, 136.97, 136.03, 134.07, 132.99, 131.88, 131.33, 126.25, 125.37, 123.33, 123.17, 121.59, 121.47, 120.98, 118.77, 114.33, 113.32, 101.11, 72.63, 71.32, 65.90, 17.27, −1.23. LC/MS: <i>m</i>/<i>z</i> calcd for [C<sub>30</sub>H<sub>30</sub>ClFN<sub>6</sub>O<sub>4</sub>Si + H]<sup>+</sup>, 621.18; found, 621.29.</div></div><div id="sec4_16_6_56" class="NLM_sec NLM_sec_level_4"><div id="ac_i97" class="anchor-spacer"></div><h5 class="article-section__title" id="_i97"> <i>N</i>-(1-Benzyl-1<i>H</i>-indazol-5-yl)-5-(4-fluoro-3-nitrophenyl)-7-{[2-(trimethylsilyl)ethoxy]methyl}-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-4-amine (<b>23b</b>)</h5><div class="NLM_p last">The reaction was performed according to the <a class="ref internalNav" href="#sec4_16_1" aria-label="General Method for Back-Pocket Aniline Coupling">General Method for Back-Pocket Aniline Coupling</a>. It used 4-chloro-5-(4-fluoro-3-nitrophenyl)-7-{[2-(trimethylsilyl)ethoxy]methyl}-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidine (<b>22</b>) (300 mg, 0.79 mmol, 1.0 equiv) and 1-benzyl-1<i>H</i>-indazol-5-amine (235.6 mg, 1.06 mmol, 1.5 equiv). Purification by flash column chromatography yielded the desired product. Yield: 45% (218 mg, 0.38 mmol). <sup>1</sup>H NMR (700 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.37 (s, 1H), 8.22 (d, <i>J</i> = 2.2 Hz, 1H), 8.21 (d, <i>J</i> = 2.4 Hz, 1H), 8.02 (d, <i>J</i> = 0.7 Hz, 1H), 7.93 (d, <i>J</i> = 1.5 Hz, 1H), 7.90–7.87 (m, 1H), 7.78 (s, 1H), 7.61–7.58 (m, 1H), 7.57 (d, <i>J</i> = 8.9 Hz, 1H), 7.36 (dd, 1H), 7.32–7.29 (m, 2H), 7.26 (d, <i>J</i> = 7.4 Hz, 1H), 7.20 (d, <i>J</i> = 7.2 Hz, 2H), 5.62 (s, 2H), 5.61 (s, 2H), 3.60 (t, <i>J</i> = 8.0 Hz, 2H), 0.87 (t, <i>J</i> = 8.0 Hz, 2H), −0.06 (s, 9H). <sup>13</sup>C NMR (176 MHz, DMSO-<i>d</i><sub>6</sub>): δ 154.49, 154.16, 152.68, 151.66, 137.92, 137.61, 136.49, 133.03, 132.72, 130.90, 128.48, 127.21, 125.77, 125.25, 123.69, 123.34, 118.12, 113.27, 112.50, 109.97, 109.48, 100.83, 72.46, 65.73, 51.87, 17.13, −1.38. LC/MS: <i>m</i>/<i>z</i> calcd for [C<sub>32</sub>H<sub>32</sub>FN<sub>7</sub>O<sub>3</sub>Si + H]<sup>+</sup>, 610.23; found, 610.43.</div></div><div id="sec4_16_6_57" class="NLM_sec NLM_sec_level_4"><div id="ac_i98" class="anchor-spacer"></div><h5 class="article-section__title" id="_i98"> <i>N</i>-(2-Benzyl-1<i>H</i>-benzo[<i>d</i>]imidazol-5-yl)-5-iodo-7-{[2-(trimethylsilyl)ethoxy]methyl}-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-4-amine</h5><div class="NLM_p last">The reaction was performed according to the <a class="ref internalNav" href="#sec4_16_1" aria-label="General Method for Back-Pocket Aniline Coupling">General Method for Back-Pocket Aniline Coupling</a>. 4-Chloro-5-iodo-7-{[2-(trimethylsilyl)ethoxy]methyl}-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidine (<b>10</b>) (450 mg, 1.09 mmol, 1.0 equiv) and 2-benzyl-1<i>H</i>-benzo[<i>d</i>]imidazol-5-amine (369.0 mg, 1.65 mmol, 1.5 equiv) were used. Purification by flash column chromatography yielded the desired product. Yield: 42% (274 mg, 0.46 mmol). <sup>1</sup>H NMR (700 MHz, DMSO-<i>d</i><sub>6</sub>): δ 12.25 (s, 1H), 8.35 (s, 1H), 8.27 (s, 1H), 8.13 (d, <i>J</i> = 1.9 Hz, 1H), 7.73 (s, 1H), 7.52 (d, <i>J</i> = 8.5 Hz, 1H), 7.41 (d, <i>J</i> = 8.5 Hz, 1H), 7.34–7.33 (m, 4H), 7.24 (d, <i>J</i> = 3.8 Hz, 1H), 5.52 (s, 2H), 4.17 (d, <i>J</i> = 6.9 Hz, 2H), 3.52 (t, <i>J</i> = 8.1 Hz, 2H), 0.83 (t, <i>J</i> = 7.2 Hz, 2H), −0.08 (s, 9H). <sup>13</sup>C NMR (176 MHz, DMSO-<i>d</i><sub>6</sub>): δ 158.21, 153.61, 151.67, 150.10, 143.88, 139.87, 137.66, 133.13, 130.80, 128.73, 128.46, 126.50, 118.18, 115.27, 110.67, 102.14, 72.23, 65.61, 51.03, 35.00, 17.08, −1.39. LC/MS: <i>m</i>/<i>z</i> calcd for [C<sub>26</sub>H<sub>29</sub>IN<sub>6</sub>OSi + H]<sup>+</sup>, 597.12; found, 597.16.</div></div><div id="sec4_16_6_58" class="NLM_sec NLM_sec_level_4"><div id="ac_i99" class="anchor-spacer"></div><h5 class="article-section__title" id="_i99"> <i>N</i>-(2-Benzyl-1<i>H</i>-benzo[<i>d</i>]imidazol-5-yl)-5-(4-fluoro-3-nitrophenyl)-7-{[2-(trimethylsilyl)ethoxy]methyl}-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-4-amine (<b>23c</b>)</h5><div class="NLM_p last">The reaction was performed according to the <a class="ref internalNav" href="#sec4_16_2" aria-label="General Method for Suzuki–Miyaura Coupling">General Method for Suzuki–Miyaura Coupling</a>. It used <i>N</i>-(2-benzyl-1<i>H</i>-benzo[<i>d</i>]imidazol-5-yl)-5-iodo-7-{[2-(trimethylsilyl)ethoxy]methyl}-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-4-amine (274 mg, 0.46 mmol) that was coupled and purified by flash column chromatography. This afforded the desired compound. Yield: 66% (185 mg, 0.30 mmol). <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>): δ 12.12 (s, 1H), 8.37 (s, 1H), 8.18 (s, 1H), 7.89–7.85 (m, 1H), 7.77 (s, 1H), 7.64–7.63 (m, 2H), 7.62 (t, <i>J</i> = 1.4 Hz, 1H), 7.61–7.60 (m, 2H), 7.56 (d, <i>J</i> = 3.0 Hz, 2H), 7.54 (d, <i>J</i> = 2.9 Hz, 1H), 7.36 (d, <i>J</i> = 6.3 Hz, 1H), 7.31 (s, 1H), 5.61 (d, <i>J</i> = 1.8 Hz, 2H), 4.13 (d, <i>J</i> = 3.5 Hz, 2H), 3.59 (t, <i>J</i> = 8.0 Hz, 2H), 0.87 (t, <i>J</i> = 8.0 Hz, 5H), −0.06 (s, 9H). LC/MS: <i>m</i>/<i>z</i> calcd for [C<sub>32</sub>H<sub>32</sub>FN<sub>7</sub>O<sub>3</sub>Si + H]<sup>+</sup>, 610.23; found, 610.30.</div></div><div id="sec4_16_6_59" class="NLM_sec NLM_sec_level_4"><div id="ac_i100" class="anchor-spacer"></div><h5 class="article-section__title" id="_i100"> 2-{4-{{5-(4-Fluoro-3-nitrophenyl)-7-{[2-(trimethylsilyl)ethoxy]methyl}-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-4-yl}amino}benzyl}isoindoline-1,3-dione (<b>23d</b>)</h5><div class="NLM_p last">The reaction was performed according to the <a class="ref internalNav" href="#sec4_16_1" aria-label="General Method for Back-Pocket Aniline Coupling">General Method for Back-Pocket Aniline Coupling</a>. It used 4-chloro-5-(4-fluoro-3-nitrophenyl)-7-{[2-(trimethylsilyl)ethoxy]methyl}-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidine (<b>22</b>) (670 mg, 1.58 mmol, 1.0 equiv) and 2-(4-aminobenzyl)isoindoline-1,3-dione (599.3 mg, 2.4 mmol, 1.5 equiv). Purification by flash column chromatography yielded the desired product. Yield: 76% (770 mg, 1.2 mmol). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.39 (s, 1H), 8.27 (s, 1H), 8.19 (dd, <i>J</i> = 7.0, 2.2 Hz, 1H), 7.90 (d, <i>J</i> = 2.8 Hz, 1H), 7.87–7.85 (m, 2H), 7.82 (s, 1H), 7.59 (dd, <i>J</i> = 11.2, 8.7 Hz, 1H), 7.45 (d, <i>J</i> = 8.5 Hz, 2H), 7.20 (d, <i>J</i> = 8.5 Hz, 2H), 6.98 (d, <i>J</i> = 8.4 Hz, 1H), 6.48 (d, <i>J</i> = 8.4 Hz, 1H), 5.60 (s, 2H), 4.70 (s, 2H), 3.58 (t, <i>J</i> = 8.0 Hz, 2H), 0.85 (t, <i>J</i> = 8.1 Hz, 2H), −0.08 (s, 9H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>): δ 167.73, 153.84, 151.83, 151.52, 148.16, 139.21, 134.58, 131.60, 130.85, 128.74, 127.74, 126.18, 123.24, 121.06, 118.57, 113.70, 113.23, 101.33, 72.50, 65.77, 40.52, 17.13, −1.35. LC/MS: <i>m</i>/<i>z</i> calcd for [C<sub>33</sub>H<sub>31</sub>FN<sub>6</sub>O<sub>5</sub>Si + H]<sup>+</sup>, 639.21; found, 639.31.</div></div><div id="sec4_16_6_60" class="NLM_sec NLM_sec_level_4"><div id="ac_i101" class="anchor-spacer"></div><h5 class="article-section__title" id="_i101"> <i>N</i>-[3-Chloro-4-(pyridin-2-ylmethoxy)phenyl]-5-{4-[(1-methoxypropan-2-yl)oxy]-3-nitrophenyl}-7-{[2-(trimethylsilyl)ethoxy]methyl}-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-4-amine</h5><div class="NLM_p last">The reaction was performed according to the <a class="ref internalNav" href="#sec4_16_3" aria-label="General Method for Nucleophilic Aromatic Substitution">General Method for Nucleophilic Aromatic Substitution</a>. <i>N</i>-[3-Chloro-4-(pyridin-2-ylmethoxy)phenyl]-5-(4-fluoro-3-nitrophenyl)-7-{[2-(trimethylsilyl)ethoxy]methyl}-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-4-amine (<b>23a</b>) (300 mg, 0.48 mmol) was reacted with 1-methoxypropan-2-ol (433 mg, 4.8 mmol), and purification by flash column chromatography afforded the pure compound. Yield: 46% (154 mg, 0.22 mmol). <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.63 (d, <i>J</i> = 4.6 Hz, 1H), 8.40 (d, <i>J</i> = 3.3 Hz, 1H), 8.00 (d, <i>J</i> = 2.2 Hz, 1H), 7.97 (d, <i>J</i> = 7.7 Hz, 1H), 7.95 (d, <i>J</i> = 2.1 Hz, 1H), 7.74 (dd, <i>J</i> = 5.0, 2.5 Hz, 2H), 7.73–7.70 (m, 1H), 7.68 (dd, <i>J</i> = 8.6, 2.2 Hz, 1H), 7.63 (d, <i>J</i> = 7.8 Hz, 1H), 7.50 (d, <i>J</i> = 8.9 Hz, 1H), 7.44 (t, <i>J</i> = 6.2 Hz, 1H), 7.41–7.37 (m, 1H), 7.22–7.17 (m, 2H), 5.61 (s, 2H), 5.28 (s, 2H), 3.60–3.56 (m, 2H), 3.51 (dd, <i>J</i> = 4.9, 2.7 Hz, 1H), 3.29 (s, 3H), 1.27 (d, <i>J</i> = 6.3 Hz, 3H), 0.86 (t, <i>J</i> = 8.0 Hz, 2H), −0.06 (s, 9H). <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>): δ 158.61, 158.36, 158.12, 157.87, 155.81, 149.38, 148.42, 140.55, 138.19, 136.75, 135.26, 133.76, 126.60, 124.58, 124.29, 123.44, 121.84, 121.13, 119.62, 117.00, 116.38, 114.44, 100.99, 75.13, 74.84, 72.55, 70.75, 65.79, 58.57, 17.14, 16.43, −1.36. LC/MS: <i>m</i>/<i>z</i> calcd for [C<sub>34</sub>H<sub>39</sub>ClN<sub>6</sub>O<sub>6</sub>Si + H]<sup>+</sup>, 691.24; found, 621.14.</div></div><div id="sec4_16_6_61" class="NLM_sec NLM_sec_level_4"><div id="ac_i102" class="anchor-spacer"></div><h5 class="article-section__title" id="_i102"> 2-{4-{4-{[3-Chloro-4-(pyridin-2-ylmethoxy)phenyl]amino}-7-{[2-(trimethylsilyl)ethoxy]methyl}-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-5-yl}-2-nitrophenoxy}ethan-1-ol</h5><div class="NLM_p last">The reaction was performed according to the <a class="ref internalNav" href="#sec4_16_3" aria-label="General Method for Nucleophilic Aromatic Substitution">General Method for Nucleophilic Aromatic Substitution</a>. <i>N</i>-[3-Chloro-4-(pyridin-2-ylmethoxy)phenyl]-5-(4-fluoro-3-nitrophenyl)-7-{[2-(trimethylsilyl)ethoxy]methyl}-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-4-amine (<b>23a</b>) (323 mg, 0.52 mmol) was reacted with ethane-1,2-diol (322.8 mg, 5.2 mmol), and purification by flash column chromatography afforded the pure compound. Yield: 87% (300 mg, 0.45 mmol). <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.59 (d, <i>J</i> = 4.4 Hz, 1H), 8.39 (s, 1H), 8.05 (s, 1H), 7.97 (d, <i>J</i> = 2.3 Hz, 1H), 7.87 (ddd, <i>J</i> = 7.7, 7.7, 1.7 Hz, 1H), 7.77 (d, <i>J</i> = 2.5 Hz, 1H), 7.73 (dd, <i>J</i> = 8.7, 2.3 Hz, 1H), 7.70 (s, 1H), 7.56 (d, <i>J</i> = 7.8 Hz, 1H), 7.46 (d, <i>J</i> = 8.8 Hz, 1H), 7.39–7.34 (m, 2H), 7.14 (d, <i>J</i> = 9.0 Hz, 1H), 5.60 (s, 2H), 5.23 (d, <i>J</i> = 7.9 Hz, 2H), 4.92 (s, 1H), 4.21 (t, <i>J</i> = 4.9 Hz, 2H), 3.75 (d, <i>J</i> = 3.4 Hz, 2H), 3.58 (t, <i>J</i> = 8.0 Hz, 2H), 0.86 (t, <i>J</i> = 8.0 Hz, 2H), −0.06 (s, 9H). <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>): δ 156.48, 153.82, 151.48, 151.37, 150.32, 149.10, 149.06, 139.65, 137.09, 133.98, 133.92, 126.77, 125.34, 124.29, 123.01, 122.91, 121.35, 121.14, 120.88, 115.79, 114.22, 113.78, 101.11, 72.39, 71.41, 71.19, 65.69, 59.31, 17.13, −1.37. LC/MS: <i>m</i>/<i>z</i> calcd for [C<sub>32</sub>H<sub>35</sub>ClN<sub>6</sub>O<sub>6</sub>Si + H]<sup>+</sup>, 663.21; found, 663.41.</div></div><div id="sec4_16_6_62" class="NLM_sec NLM_sec_level_4"><div id="ac_i103" class="anchor-spacer"></div><h5 class="article-section__title" id="_i103"> 3-{4-{4-{[3-Chloro-4-(pyridin-2-ylmethoxy)phenyl]amino}-7-{[2-(trimethylsilyl)ethoxy]methyl}-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-5-yl}-2-nitrophenoxy}propan-1-ol</h5><div class="NLM_p last">The reaction was performed according to the <a class="ref internalNav" href="#sec4_16_3" aria-label="General Method for Nucleophilic Aromatic Substitution">General Method for Nucleophilic Aromatic Substitution</a>. <i>N</i>-[3-Chloro-4-(pyridin-2-ylmethoxy)phenyl]-5-(4-fluoro-3-nitrophenyl)-7-{[2-(trimethylsilyl)ethoxy]methyl}-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-4-amine (<b>23a</b>) (300 mg, 0.48 mmol) was reacted with propane-1,3-diol (365.2 mg, 4.8 mmol). Purification by flash column chromatography afforded the pure compound. Yield: 86% (278 mg, 0.41 mmol). <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.59 (d, <i>J</i> = 4.7 Hz, 1H), 8.39 (s, 1H), 8.06 (s, 1H), 7.98 (d, <i>J</i> = 2.3 Hz, 1H), 7.87 (ddd, <i>J</i> = 7.7, 7.7, 1.7 Hz, 1H), 7.75 (ddd, <i>J</i> = 2.6, 2.3, 2.3 Hz, 2H), 7.70 (s, 1H), 7.56 (d, <i>J</i> = 6.2 Hz, 1H), 7.44 (d, <i>J</i> = 8.8 Hz, 1H), 7.38–7.34 (m, 2H), 7.14 (d, <i>J</i> = 9.0 Hz, 1H), 5.60 (s, 2H), 5.23 (s, 2H), 4.57 (s, 1H), 4.25 (t, <i>J</i> = 6.1 Hz, 2H), 3.59 (t, <i>J</i> = 4.2 Hz, 2H), 3.56 (t, <i>J</i> = 4.8 Hz, 2H), 1.89 (app. p, <i>J</i> = 6.1 Hz, 2H), 0.86 (t, <i>J</i> = 6.6 Hz, 2H), −0.07 (s, 9H). <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>): δ 156.48, 153.86, 151.50, 151.37, 150.29, 149.10, 139.42, 137.08, 136.97, 134.13, 133.92, 126.65, 125.34, 124.35, 123.02, 122.99, 121.33, 121.21, 120.87, 115.45, 114.21, 113.81, 101.10, 72.36, 71.16, 66.37, 65.67, 57.00, 31.78, 17.11, −1.45. LC/MS: <i>m</i>/<i>z</i> calcd for [C<sub>33</sub>H<sub>37</sub>ClN<sub>6</sub>O<sub>6</sub>Si + H]<sup>+</sup>, 677.22; found, 677.21.</div></div><div id="sec4_16_6_63" class="NLM_sec NLM_sec_level_4"><div id="ac_i104" class="anchor-spacer"></div><h5 class="article-section__title" id="_i104"> <i>N</i>-[3-Chloro-4-(pyridin-2-ylmethoxy)phenyl]-5-{4-[2-(dimethylamino)ethoxy]-3-nitrophenyl}-7-{[2-(trimethylsilyl)ethoxy]methyl}-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-4-amine</h5><div class="NLM_p last">The reaction was performed according to the <a class="ref internalNav" href="#sec4_16_3" aria-label="General Method for Nucleophilic Aromatic Substitution">General Method for Nucleophilic Aromatic Substitution</a>. <i>N</i>-[3-Chloro-4-(pyridin-2-ylmethoxy)phenyl]-5-(4-fluoro-3-nitrophenyl)-7-{[2-(trimethylsilyl)ethoxy]methyl}-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-4-amine (<b>23a</b>) (300 mg, 0.48 mmol) was reacted with 2-(dimethylamino)ethan-1-ol (427.8 mg, 4.8 mmol). Purification by repeated flash column chromatography (SiO<sub>2</sub>, followed by C<sub>18</sub> column) afforded the pure compound. Yield: 37% (122 mg, 0.18 mmol). <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.58 (d, <i>J</i> = 4.7 Hz, 1H), 8.39 (s, 1H), 8.07 (s, 1H), 7.97 (d, <i>J</i> = 2.2 Hz, 1H), 7.87 (ddd, <i>J</i> = 7.7, 7.7, 1.6 Hz, 1H), 7.75–7.72 (m, 2H), 7.70 (s, 1H), 7.56 (d, <i>J</i> = 7.8 Hz, 1H), 7.46 (d, <i>J</i> = 8.8 Hz, 1H), 7.38–7.34 (m, 2H), 7.13 (d, <i>J</i> = 9.0 Hz, 1H), 5.60 (s, 2H), 5.23 (s, 2H), 4.27 (t, <i>J</i> = 5.6 Hz, 2H), 3.58 (t, <i>J</i> = 8.0 Hz, 2H), 2.67 (t, <i>J</i> = 5.6 Hz, 2H), 2.23 (s, 6H), 0.86 (t, <i>J</i> = 8.0 Hz, 2H), −0.07 (s, 9H). <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>): δ 156.48, 153.87, 151.52, 151.38, 150.05, 149.10, 149.06, 139.55, 137.08, 134.05, 133.96, 126.84, 125.36, 124.31, 123.01, 122.93, 121.33, 121.14, 120.88, 115.58, 114.21, 113.79, 101.12, 72.41, 71.20, 68.10, 65.66, 57.25, 45.56, 17.17, −1.43. LC/MS: <i>m</i>/<i>z</i> calcd for [C<sub>34</sub>H<sub>40</sub>ClN<sub>7</sub>O<sub>5</sub>Si + H]<sup>+</sup>, 690.26; found, 690.20.</div></div><div id="sec4_16_6_64" class="NLM_sec NLM_sec_level_4"><div id="ac_i105" class="anchor-spacer"></div><h5 class="article-section__title" id="_i105"> 2-{4-{4-[(1-Benzyl-1<i>H</i>-indazol-5-yl)amino]-7-{[2-(trimethylsilyl)ethoxy]methyl}-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-5-yl}-2-nitrophenoxy}ethan-1-ol</h5><div class="NLM_p last">The reaction was performed according to the <a class="ref internalNav" href="#sec4_16_3" aria-label="General Method for Nucleophilic Aromatic Substitution">General Method for Nucleophilic Aromatic Substitution</a>. <i>N</i>-(1-Benzyl-1<i>H</i>-indazol-5-yl)-5-(4-fluoro-3-nitrophenyl)-7-{[2-(trimethylsilyl)ethoxy]methyl}-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-4-amine (<b>23b</b>) (218 mg, 0.36 mmol) was reacted with ethane-1,2-diol (271.64 mg, 3.57 mmol). Purification by flash column chromatography afforded the pure compound. Yield: 54% (125 mg, 0.19 mmol). <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.36 (s, 1H), 8.04 (d, <i>J</i> = 0.8 Hz, 2H), 8.01 (d, <i>J</i> = 2.3 Hz, 1H), 7.98 (s, 1H), 7.75 (dd, <i>J</i> = 8.7, 2.3 Hz, 1H), 7.68 (s, 1H), 7.59 (d, <i>J</i> = 9.0 Hz, 1H), 7.44 (d, <i>J</i> = 8.8 Hz, 1H), 7.37 (dd, <i>J</i> = 9.0, 1.9 Hz, 1H), 7.31 (t, <i>J</i> = 7.3 Hz, 2H), 7.25 (dd, <i>J</i> = 8.4, 6.2 Hz, 1H), 7.21 (d, <i>J</i> = 7.1 Hz, 2H), 5.63 (s, 2H), 5.60 (s, 2H), 4.91 (t, <i>J</i> = 5.4 Hz, 1H), 4.19 (t, <i>J</i> = 4.9 Hz, 2H), 3.74 (dd, <i>J</i> = 10.0, 5.2 Hz, 2H), 3.58 (t, <i>J</i> = 8.0 Hz, 2H), 0.86 (t, <i>J</i> = 8.0 Hz, 2H), −0.06 (s, 9H). <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>): δ 154.43, 151.51, 151.32, 150.33, 139.66, 137.65, 136.44, 134.02, 132.93, 132.77, 128.50, 127.46, 127.25, 126.82, 125.06, 124.38, 123.75, 123.09, 115.80, 113.81, 112.17, 109.54, 100.96, 72.35, 71.36, 65.63, 59.30, 51.86, 17.13, −1.35. LC/MS: <i>m</i>/<i>z</i> calcd for [C<sub>34</sub>H<sub>37</sub>N<sub>7</sub>O<sub>5</sub>Si + H]<sup>+</sup>, 652.26; found, 652.23.</div></div><div id="sec4_16_6_65" class="NLM_sec NLM_sec_level_4"><div id="ac_i106" class="anchor-spacer"></div><h5 class="article-section__title" id="_i106"> 2-{4-{4-[(2-Benzyl-1<i>H</i>-benzo[<i>d</i>]imidazole-5-yl)amino]-7-{[2-(trimethylsilyl)ethoxy]methyl}-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-5-yl}-2-nitrophenoxy}ethan-1-ol</h5><div class="NLM_p last">The reaction was performed according to the <a class="ref internalNav" href="#sec4_16_3" aria-label="General Method for Nucleophilic Aromatic Substitution">General Method for Nucleophilic Aromatic Substitution</a>. <i>N</i>-(2-Benzyl-1<i>H</i>-benzo[<i>d</i>]imidazol-5-yl)-5-(4-fluoro-3-nitrophenyl)-7-{[2-(trimethylsilyl)ethoxy]methyl}-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-4-amine (<b>23c</b>) (185 mg, 0.30 mmol) was reacted with ethane-1,2-diol (229.79 mg, 3.02 mmol). Purification by flash column chromatography afforded the pure compound. Yield: 36% (70 mg, 0.11 mmol). <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>): δ 12.16 (s, 1H), 8.37 (s, 1H), 8.00 (t, <i>J</i> = 2.2 Hz, 1H), 7.90–7.87 (m, 1H), 7.80 (d, <i>J</i> = 13.9 Hz, 1H), 7.77–7.74 (m, 1H), 7.66 (d, <i>J</i> = 12.6 Hz, 1H), 7.45 (d, <i>J</i> = 8.7 Hz, 1H), 7.40 (d, <i>J</i> = 8.6 Hz, 1H), 7.32 (d, <i>J</i> = 4.5 Hz, 4H), 7.29 (t, <i>J</i> = 8.6 Hz, 1H), 7.24 (dd, <i>J</i> = 8.7, 4.4 Hz, 1H), 5.60 (s, 2H), 4.93 (t, <i>J</i> = 5.3 Hz, 1H), 4.20 (t, <i>J</i> = 4.9 Hz, 2H), 4.14 (t, <i>J</i> = 5.1 Hz, 2H), 3.58 (t, <i>J</i> = 8.0 Hz, 2H), 2.01 (t, <i>J</i> = 7.5 Hz, 2H), 0.85 (t, <i>J</i> = 7.0 Hz, 2H), −0.06 (s, 9H). LC/MS: <i>m</i>/<i>z</i> calcd for [C<sub>34</sub>H<sub>37</sub>N<sub>7</sub>O<sub>5</sub>Si + H]<sup>+</sup>, 652.26; found, 652.31.</div></div><div id="sec4_16_6_66" class="NLM_sec NLM_sec_level_4"><div id="ac_i107" class="anchor-spacer"></div><h5 class="article-section__title" id="_i107"> 2-{4-{{5-[4-(2-Hydroxyethoxy)-3-nitrophenyl]-7-{[2-(trimethylsilyl)ethoxy]methyl}-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-4-yl}amino}benzyl}isoindoline-1,3-dione</h5><div class="NLM_p last">The reaction was performed according to the <a class="ref internalNav" href="#sec4_16_3" aria-label="General Method for Nucleophilic Aromatic Substitution">General Method for Nucleophilic Aromatic Substitution</a>. 2-{4-{{5-(4-Fluoro-3-nitrophenyl)-7-{[2-(trimethylsilyl)ethoxy]methyl}-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-4-yl}amino}benzyl}isoindoline-1,3-dione (<b>23d</b>) (360 mg, 0.56 mmol) was reacted with ethane-1,2-diol (347.59 mg, 5.60 mmol). Purification by flash column chromatography afforded the pure compound. Yield: 9% (37 mg, 0.05 mmol). <sup>1</sup>H NMR (700 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.37 (s, 1H), 7.98 (s, 2H), 7.91–7.89 (m, 2H), 7.87–7.84 (m, 2H), 7.72 (dd, <i>J</i> = 8.6, 1.5 Hz, 1H), 7.70 (s, 1H), 7.52 (d, <i>J</i> = 8.3 Hz, 2H), 7.45 (d, <i>J</i> = 8.7 Hz, 1H), 7.23 (d, <i>J</i> = 8.3 Hz, 2H), 5.59 (s, 2H), 4.89 (t, <i>J</i> = 5.3 Hz, 1H), 4.72 (s, 2H), 4.20 (t, <i>J</i> = 4.8 Hz, 2H), 3.73 (dd, <i>J</i> = 10.0, 5.0 Hz, 2H), 3.57 (t, <i>J</i> = 8.0 Hz, 2H), 0.85 (t, <i>J</i> = 7.9 Hz, 2H), −0.07 (s, 9H). <sup>13</sup>C NMR (176 MHz, DMSO-<i>d</i><sub>6</sub>): δ 167.69, 153.78, 151.44, 151.32, 150.31, 139.70, 138.99, 134.54, 133.97, 131.57, 130.75, 127.73, 126.74, 125.35, 124.27, 123.19, 120.84, 115.81, 113.71, 101.39, 72.39, 71.40, 65.68, 59.32, 40.52, 20.75, −1.38. LC/MS: <i>m</i>/<i>z</i> calcd for [C<sub>35</sub>H<sub>36</sub>N<sub>6</sub>O<sub>7</sub>Si + H]<sup>+</sup>, 681.24; found, 681.33.</div></div><div id="sec4_16_6_67" class="NLM_sec NLM_sec_level_4"><div id="ac_i108" class="anchor-spacer"></div><h5 class="article-section__title" id="_i108"> 2-{4-{{5-{4-[2-(Dimethylamino)ethoxy]-3-nitrophenyl}-7-{[2-(trimethylsilyl)ethoxy]methyl}-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-4-yl}amino}benzyl}isoindoline-1,3-dione</h5><div class="NLM_p last">The reaction was performed according to the <a class="ref internalNav" href="#sec4_16_3" aria-label="General Method for Nucleophilic Aromatic Substitution">General Method for Nucleophilic Aromatic Substitution</a>. 2-{4-{{5-(4-Fluoro-3-nitrophenyl)-7-{[2-(trimethylsilyl)ethoxy]methyl}-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-4-yl}amino}benzyl}isoindoline-1,3-dione (<b>23d</b>) (360 mg, 0.56 mmol) was reacted with 2-(dimethylamino)ethan-1-ol (516.95 mg, 5.80 mmol). Purification by flash column chromatography afforded the pure compound. Yield: 9% (38 mg, 0.05 mmol). <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.37 (s, 1H), 8.01 (s, 1H), 7.99 (d, <i>J</i> = 2.3 Hz, 1H), 7.91–7.89 (m, 2H), 7.87–7.84 (m, 2H), 7.72 (dd, <i>J</i> = 8.7, 2.3 Hz, 1H), 7.71 (s, 1H), 7.51 (dd, <i>J</i> = 7.4, 1.8 Hz, 2H), 7.45 (d, <i>J</i> = 8.8 Hz, 1H), 7.23 (d, <i>J</i> = 8.6 Hz, 2H), 5.59 (s, 2H), 4.72 (s, 2H), 4.26 (t, <i>J</i> = 5.6 Hz, 2H), 3.57 (t, <i>J</i> = 8.0 Hz, 2H), 2.66 (t, <i>J</i> = 5.5 Hz, 2H), 0.85 (t, <i>J</i> = 8.0 Hz, 2H), −0.08 (s, 9H). <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>): δ 167.71, 153.81, 151.47, 151.34, 150.05, 139.61, 139.03, 134.57, 134.04, 131.57, 130.75, 127.74, 126.78, 125.40, 124.28, 123.21, 120.88, 115.60, 113.71, 101.40, 72.40, 68.08, 65.69, 57.29, 45.60, 40.06, 17.12, −1.37. LC/MS: <i>m</i>/<i>z</i> calcd for [C<sub>37</sub>H<sub>41</sub>N<sub>7</sub>O<sub>6</sub>Si + H]<sup>+</sup>, 708.29; found, 708.30.</div></div><div id="sec4_16_6_68" class="NLM_sec NLM_sec_level_4"><div id="ac_i109" class="anchor-spacer"></div><h5 class="article-section__title" id="_i109"> 5-{3-Amino-4-[(1-methoxypropan-2-yl)oxy]phenyl}-<i>N</i>-[3-chloro-4-(pyridin-2-ylmethoxy)phenyl]-7-{[2-(trimethylsilyl)ethoxy]methyl}-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-4-amine (<b>24a</b>)</h5><div class="NLM_p last">The reaction was performed according to the <a class="ref internalNav" href="#sec4_16_4" aria-label="General Method for Aromatic Nitro Reduction">General Method for Aromatic Nitro Reduction</a>. <i>N</i>-[3-Chloro-4-(pyridin-2-ylmethoxy)phenyl]-5-{4-[(1-methoxypropan-2-yl)oxy]-3-nitrophenyl}-7-{[2-(trimethylsilyl)ethoxy]methyl}-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-4-amine (152 mg, 0.22 mmol) was reduced and purified by repeated flash column chromatography (SiO<sub>2</sub>, followed by C<sub>18</sub> column). This afforded the pure compound. Yield: 20% (29 mg, 0.04 mmol). <sup>1</sup>H NMR (600 MHz, MeOD): δ 8.64 (s, 1H), 8.34 (s, 1H), 8.12–8.06 (m, 1H), 7.83 (d, <i>J</i> = 7.1 Hz, 1H), 7.67 (s, 1H), 7.59–7.54 (m, 1H), 7.54 (s, 1H), 7.47–7.38 (m, 2H), 7.34 (dd, <i>J</i> = 8.8, 2.6 Hz, 1H), 7.31 (d, <i>J</i> = 8.6 Hz, 1H), 7.20 (d, <i>J</i> = 8.9 Hz, 1H), 5.70 (s, 2H), 5.34 (s, 2H), 4.81–4.76 (m, 1H), 3.68–3.63 (m, 3H), 3.61 (dd, <i>J</i> = 10.6, 3.4 Hz, 1H), 3.41 (s, 3H), 1.38 (d, <i>J</i> = 6.3 Hz, 3H), 0.93 (t, <i>J</i> = 8.0 Hz, 2H), −0.03 (s, 9H). <sup>13</sup>C NMR (151 MHz, MeOD): δ 156.47, 153.61, 153.47, 151.28, 150.94, 149.89, 148.43, 148.09, 141.25, 132.25, 128.98, 127.97, 127.23, 127.14, 125.42, 124.88, 124.85, 124.22, 123.42, 118.56, 116.36, 116.17, 102.31, 76.62, 75.79, 74.60, 71.51, 67.85, 59.47, 18.60, 16.75, −1.40. LC/MS: <i>m</i>/<i>z</i> calcd for [C<sub>34</sub>H<sub>41</sub>ClN<sub>6</sub>O<sub>4</sub>Si + H]<sup>+</sup>, 661.26; found, 661.22.</div></div><div id="sec4_16_6_69" class="NLM_sec NLM_sec_level_4"><div id="ac_i110" class="anchor-spacer"></div><h5 class="article-section__title" id="_i110"> 2-{2-Amino-4-{4-{[3-chloro-4-(pyridin-2-ylmethoxy)phenyl]amino}-7-{[2-(trimethylsilyl)ethoxy]methyl}-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-5-yl}phenoxy}ethan-1-ol (<b>24b</b>)</h5><div class="NLM_p last">The reaction was performed according to the <a class="ref internalNav" href="#sec4_16_4" aria-label="General Method for Aromatic Nitro Reduction">General Method for Aromatic Nitro Reduction</a>. 2-{4-{4-{[3-Chloro-4-(pyridin-2-ylmethoxy)phenyl]amino}-7-{[2-(trimethylsilyl)ethoxy]methyl}-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-5-yl}-2-nitrophenoxy}ethan-1-ol (293 mg, 0.44 mmol) was reduced and purified by column chromatography. This afforded the pure compound. Yield: 71% (197 mg, 0.31 mmol). <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.58 (d, <i>J</i> = 4.7 Hz, 1H), 8.40 (s, 1H), 7.98 (d, <i>J</i> = 2.5 Hz, 1H), 7.86 (ddd, <i>J</i> = 7.7, 7.7, 1.6 Hz, 1H), 7.55 (d, <i>J</i> = 9.1 Hz, 2H), 7.42 (s, 1H), 7.35 (dd, <i>J</i> = 7.1, 5.2 Hz, 1H), 7.25 (dd, <i>J</i> = 8.9, 2.6 Hz, 1H), 7.19 (d, <i>J</i> = 9.0 Hz, 1H), 6.94 (d, <i>J</i> = 8.1 Hz, 1H), 6.83 (d, <i>J</i> = 2.1 Hz, 1H), 6.65 (dd, <i>J</i> = 8.0, 2.1 Hz, 1H), 5.58 (s, 2H), 5.24 (s, 2H), 5.17 (s, 2H), 4.94 (t, <i>J</i> = 6.2 Hz, 1H), 4.00 (t, <i>J</i> = 4.7 Hz, 2H), 3.76 (dd, <i>J</i> = 10.3, 5.3 Hz, 2H), 3.57 (t, <i>J</i> = 8.0 Hz, 2H), 0.84 (t, <i>J</i> = 5.8 Hz, 2H), −0.07 (s, 9H). <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>): δ 156.38, 153.42, 151.22, 150.47, 149.11, 148.96, 145.18, 138.98, 137.04, 133.31, 126.70, 123.50, 122.99, 121.68, 121.35, 121.23, 119.74, 116.26, 116.11, 114.60, 113.46, 112.38, 101.17, 72.11, 71.16, 70.11, 65.53, 59.72, 17.05, −1.45. LC/MS: <i>m</i>/<i>z</i> calcd for [C<sub>32</sub>H<sub>37</sub>ClN<sub>6</sub>O<sub>4</sub>Si + H]<sup>+</sup>, 633.23; found, 633.34.</div></div><div id="sec4_16_6_70" class="NLM_sec NLM_sec_level_4"><div id="ac_i111" class="anchor-spacer"></div><h5 class="article-section__title" id="_i111"> 3-{2-Amino-4-{4-{[3-chloro-4-(pyridin-2-ylmethoxy)phenyl]amino}-7-{[2-(trimethylsilyl)ethoxy]methyl}-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-5-yl}phenoxy}propan-1-ol (<b>24c</b>)</h5><div class="NLM_p last">The reaction was performed according to the <a class="ref internalNav" href="#sec4_16_4" aria-label="General Method for Aromatic Nitro Reduction">General Method for Aromatic Nitro Reduction</a>. 3-{4-{4-{[3-Chloro-4-(pyridin-2-ylmethoxy)phenyl]amino}-7-{[2-(trimethylsilyl)ethoxy]methyl}-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-5-yl}-2-nitrophenoxy}propan-1-ol (271 mg, 0.4 mmol) was reduced and purified by flash column chromatography. This afforded the pure compound. Yield: 53% (138 mg, 0.21 mmol). <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.58 (d, <i>J</i> = 4.3 Hz, 1H), 8.39 (s, 1H), 7.98 (d, <i>J</i> = 2.5 Hz, 1H), 7.86 (ddd, <i>J</i> = 7.7, 7.7, 1.6 Hz, 1H), 7.55 (d, <i>J</i> = 6.9 Hz, 2H), 7.41 (s, 1H), 7.35 (dd, <i>J</i> = 7.0, 5.1 Hz, 1H), 7.25 (dd, <i>J</i> = 8.9, 2.5 Hz, 1H), 7.19 (d, <i>J</i> = 9.0 Hz, 1H), 6.95 (d, <i>J</i> = 8.1 Hz, 1H), 6.85 (d, <i>J</i> = 2.1 Hz, 1H), 6.67 (dd, <i>J</i> = 8.0, 2.1 Hz, 1H), 5.58 (s, 2H), 5.24 (s, 2H), 5.01 (s, 1H), 4.55 (t, <i>J</i> = 5.1 Hz, 1H), 4.08 (t, <i>J</i> = 6.3 Hz, 2H), 3.62 (dd, <i>J</i> = 11.5, 6.0 Hz, 2H), 3.57 (t, <i>J</i> = 7.9 Hz, 2H), 1.92 (app. p, <i>J</i> = 6.2 Hz, 2H), 0.84 (t, <i>J</i> = 5.9 Hz, 2H), −0.07 (s, 9H). <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>): δ 156.44, 153.45, 151.22, 150.49, 149.11, 148.95, 145.19, 138.75, 137.08, 133.32, 126.51, 123.49, 123.01, 121.77, 121.35, 121.24, 119.83, 116.29, 116.26, 114.63, 113.59, 112.22, 101.23, 72.22, 71.21, 65.55, 65.01, 57.44, 32.23, 17.09, −1.36. LC/MS: <i>m</i>/<i>z</i> calcd for [C<sub>33</sub>H<sub>39</sub>ClN<sub>6</sub>O<sub>4</sub>Si + H]<sup>+</sup>, 647.25; found, 647.21.</div></div><div id="sec4_16_6_71" class="NLM_sec NLM_sec_level_4"><div id="ac_i112" class="anchor-spacer"></div><h5 class="article-section__title" id="_i112"> 5-{3-Amino-4-[2-(dimethylamino)ethoxy]phenyl}-<i>N</i>-[3-chloro-4-(pyridin-2-ylmethoxy)phenyl]-7-{[2-(trimethylsilyl)ethoxy]methyl}-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-4-amine (<b>24d</b>)</h5><div class="NLM_p last"><i>N</i>-(3-Chloro-4-(pyridin-2-ylmethoxy)phenyl)-5-(4-(2-(dimethylamino)ethoxy)-3-nitrophenyl)-7-((2-(trimethylsilyl)ethoxy)methyl)-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-4-amine (115 mg, 0.17 mmol) was dissolved in MeOH and reduced using a H-Cube Pro flow reactor (ThalesNano Inc.; Budapest, Hungary) with a 10% Pd/C-cartridge and a flow rate of 1 mL/min at 50 °C. After evaporation of the solvent, the resulting residue was purified by flash column chromatography, and this afforded the pure compound. Yield: 85% (93 mg, 0.14 mmol). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.58 (d, <i>J</i> = 4.0 Hz, 1H), 8.40 (d, <i>J</i> = 8.2 Hz, 1H), 8.19 (s, 1H), 7.96 (d, <i>J</i> = 2.1 Hz, 1H), 7.86 (d, <i>J</i> = 7.9 Hz, 1H), 7.56 (d, <i>J</i> = 3.0 Hz, 1H), 7.43 (d, <i>J</i> = 5.5 Hz, 1H), 7.35 (t, <i>J</i> = 6.0 Hz, 1H), 7.27 (d, <i>J</i> = 7.1 Hz, 1H), 7.18 (d, <i>J</i> = 8.9 Hz, 1H), 6.98 (d, <i>J</i> = 8.0 Hz, 1H), 6.85 (s, 1H), 6.67 (t, <i>J</i> = 7.9 Hz, 1H), 5.58 (s, 2H), 5.25 (s, 2H), 4.13 (s, 2H), 3.55 (t, <i>J</i> = 8.0 Hz, 2H), 2.84 (s, 2H), 2.37 (s, 6H), 2.12 (s, 2H), 0.85 (t, <i>J</i> = 7.8 Hz, 2H), −0.08 (s, 9H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>): δ 156.43, 153.46, 151.27, 150.53, 149.16, 148.99, 144.85, 139.06, 137.14, 136.99, 133.34, 127.06, 123.07, 121.78, 121.38, 121.23, 121.13, 119.86, 116.23, 114.59, 113.68, 112.96, 101.24, 72.24, 71.19, 65.61, 57.34, 54.97, 44.92, 17.14, −1.33. LC/MS: <i>m</i>/<i>z</i> calcd for [C<sub>34</sub>H<sub>42</sub>ClN<sub>7</sub>O<sub>3</sub>Si + H]<sup>+</sup>, 660.28; found, 660.28.</div></div><div id="sec4_16_6_72" class="NLM_sec NLM_sec_level_4"><div id="ac_i113" class="anchor-spacer"></div><h5 class="article-section__title" id="_i113"> 2-{2-Amino-4-{4-[(1-benzyl-1<i>H</i>-indazol-5-yl)amino]-7-{[2-(trimethylsilyl)ethoxy]methyl}-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-5-yl}phenoxy}ethan-1-ol (<b>24e</b>)</h5><div class="NLM_p last">The reaction was performed according to the <a class="ref internalNav" href="#sec4_16_4" aria-label="General Method for Aromatic Nitro Reduction">General Method for Aromatic Nitro Reduction</a>. 2-{4-{4-[(1-Benzyl-1<i>H</i>-indazol-5-yl)amino]-7-{[2-(trimethylsilyl)ethoxy]methyl}-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-5-yl}-2-nitrophenoxy}ethan-1-ol (125 mg, 0.19 mmol) was reduced and purified by flash column chromatography to afford the pure compound. Yield: 88% (105 mg, 0.17 mmol). <sup>1</sup>H NMR (700 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.38 (s, 1H), 8.29 (d, 1H), 8.07 (s, 1H), 7.64 (d, <i>J</i> = 9.0 Hz, 1H), 7.61 (s, 1H), 7.41 (s, 1H), 7.30 (t, <i>J</i> = 7.4 Hz, 2H), 7.25 (d, <i>J</i> = 7.3 Hz, 1H), 7.24–7.21 (m, 1H), 7.20 (d, <i>J</i> = 7.3 Hz, 2H), 6.96 (d, <i>J</i> = 8.1 Hz, 1H), 6.86 (d, <i>J</i> = 2.1 Hz, 1H), 6.68 (dd, <i>J</i> = 8.0, 2.1 Hz, 1H), 5.63 (s, 2H), 5.58 (s, 2H), 5.17 (s, 2H), 4.93 (t, <i>J</i> = 6.2 Hz, 1H), 4.01 (t, <i>J</i> = 4.8 Hz, 2H), 3.76 (dd, <i>J</i> = 10.3, 5.3 Hz, 2H), 3.58 (t, <i>J</i> = 8.0 Hz, 2H), 0.86 (t, <i>J</i> = 8.0 Hz, 2H), −0.06 (s, 9H). LC/MS: <i>m</i>/<i>z</i> calcd for [C<sub>34</sub>H<sub>39</sub>N<sub>7</sub>O<sub>3</sub>Si + H]<sup>+</sup>, 622.29; found, 622.26.</div></div><div id="sec4_16_6_73" class="NLM_sec NLM_sec_level_4"><div id="ac_i114" class="anchor-spacer"></div><h5 class="article-section__title" id="_i114"> 2-{2-Amino-4-{4-[(2-benzyl-1<i>H</i>-benzo[<i>d</i>]imidazol-5-yl)amino]-7-{[2-(trimethylsilyl)ethoxy]methyl}-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-5-yl}phenoxy}ethan-1-ol (<b>24f</b>)</h5><div class="NLM_p last">The reaction was performed according to the <a class="ref internalNav" href="#sec4_16_4" aria-label="General Method for Aromatic Nitro Reduction">General Method for Aromatic Nitro Reduction</a>. 2-{4-{4-[(2-Benzyl-1<i>H</i>-benzo[<i>d</i>]imidazol-5-yl)amino]-7-{[2-(trimethylsilyl)ethoxy]methyl}-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-5-yl}-2-nitrophenoxy}ethan-1-ol (70 mg, 0.11 mmol) was reduced and purified by flash column chromatography. This afforded the pure compound. Yield: 72% (49 mg, 0.08 mmol). <sup>1</sup>H NMR (700 MHz, DMSO-<i>d</i><sub>6</sub>): δ 12.24 (s, 1H), 8.37 (s, 1H), 8.06 (s, 1H), 7.57 (s, 1H), 7.54–7.42 (m, 1H), 7.39 (s, 1H), 7.30 (d, <i>J</i> = 7.0 Hz, 1H), 7.25–7.20 (m, 1H), 6.97 (d, <i>J</i> = 8.1 Hz, 1H), 6.86 (d, <i>J</i> = 2.0 Hz, 1H), 6.67 (dd, <i>J</i> = 8.0, 2.0 Hz, 1H), 5.58 (s, 2H), 5.19 (s, 2H), 4.13 (s, 2H), 4.02 (t, <i>J</i> = 4.7 Hz, 2H), 3.77 (t, <i>J</i> = 4.6 Hz, 2H), 3.58 (t, <i>J</i> = 7.9 Hz, 2H), 0.86 (t, <i>J</i> = 7.9 Hz, 2H), −0.07 (s, 9H). LC/MS: <i>m</i>/<i>z</i> calcd for [C<sub>34</sub>H<sub>39</sub>N<sub>7</sub>O<sub>3</sub>Si + H]<sup>+</sup>, 622.29; found, 622.29.</div></div><div id="sec4_16_6_74" class="NLM_sec NLM_sec_level_4"><div id="ac_i115" class="anchor-spacer"></div><h5 class="article-section__title" id="_i115"> 2-{4-{{5-[3-Amino-4-(2-hydroxyethoxy)phenyl]-7-{[2-(trimethylsilyl)ethoxy]methyl}-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-4-yl}amino}benzyl}isoindoline-1,3-dione (<b>24g</b>)</h5><div class="NLM_p last">The reaction was performed according to the <a class="ref internalNav" href="#sec4_16_4" aria-label="General Method for Aromatic Nitro Reduction">General Method for Aromatic Nitro Reduction</a>. 2-{4-{{5-[4-(2-Hydroxyethoxy)-3-nitrophenyl]-7-{[2-(trimethylsilyl)ethoxy]methyl}-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-4-yl}amino}benzyl}isoindoline-1,3-dione (37 mg, 0.05 mmol) was reduced and purified by flash column chromatography. This afforded the pure compound. Yield: 68% (22 mg, 0.03 mmol). <sup>1</sup>H NMR (700 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.38 (s, 1H), 7.95 (s, 1H), 7.91–7.88 (m, 2H), 7.87–7.84 (m, 2H), 7.59 (s, 1H), 7.54 (d, <i>J</i> = 8.3 Hz, 2H), 7.41 (s, 1H), 7.27 (d, <i>J</i> = 8.3 Hz, 2H), 6.93 (d, <i>J</i> = 8.0 Hz, 1H), 6.81 (s, 1H), 5.58 (s, 2H), 5.14 (s, 2H), 4.92 (t, <i>J</i> = 6.2 Hz, 1H), 4.72 (s, 2H), 4.00 (t, <i>J</i> = 4.6 Hz, 2H), 3.75 (dd, <i>J</i> = 10.2, 5.2 Hz, 2H), 3.56 (t, <i>J</i> = 7.9 Hz, 2H), 0.84 (t, <i>J</i> = 7.8 Hz, 2H), −0.08 (s, 9H). <sup>13</sup>C NMR (176 MHz, DMSO-<i>d</i><sub>6</sub>): δ 168.56, 163.17, 154.33, 152.08, 151.34, 146.12, 139.89, 139.34, 135.43, 132.45, 131.67, 129.03, 127.54, 124.35, 124.08, 120.57, 117.05, 114.47, 113.31, 102.32, 70.98, 66.38, 60.63, 55.70, 41.31, 17.99, −0.50. TLC/MS: <i>m</i>/<i>z</i> calcd for [C<sub>35</sub>H<sub>38</sub>N<sub>6</sub>O<sub>5</sub>Si + H]<sup>+</sup>, 651.27; found, 650.60.</div></div><div id="sec4_16_6_75" class="NLM_sec NLM_sec_level_4"><div id="ac_i116" class="anchor-spacer"></div><h5 class="article-section__title" id="_i116"> 2-{4-{{5-{3-Amino-4-[2-(dimethylamino)ethoxy]phenyl}-7-{[2-(trimethylsilyl)ethoxy]methyl}-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-4-yl}amino}benzyl}isoindoline-1,3-dione (<b>24h</b>)</h5><div class="NLM_p last">2-{4-{{5-{4-[2-(Dimethylamino)ethoxy]-3-nitrophenyl}-7-{[2-(trimethylsilyl)ethoxy]methyl}-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-4-yl}amino}benzyl}isoindoline-1,3-dione (38 mg, 0.05 mmol) was dissolved in MeOH and reduced using a H-Cube Pro flow reactor (ThalesNano Inc.; Budapest, Hungary) with a 20% Pd(OH)<sub>2</sub>-cartridge and a flow rate of 1 mL/min at 80 °C. After evaporation of the solvent, the resulting residue was purified by flash column chromatography to afford the pure compound. Yield: 16% (6 mg, 0.009 mmol). <sup>1</sup>H NMR (600 MHz, MeOD): δ 8.32 (s, 1H), 7.86–7.83 (m, 2H), 7.80–7.77 (m, 2H), 7.48 (d, <i>J</i> = 8.6 Hz, 2H), 7.30 (d, <i>J</i> = 8.5 Hz, 2H), 7.23 (s, 1H), 6.98 (d, <i>J</i> = 8.1 Hz, 1H), 6.91 (d, <i>J</i> = 2.1 Hz, 1H), 6.78 (dd, <i>J</i> = 8.0, 2.1 Hz, 1H), 5.60 (s, 2H), 4.76 (s, 2H), 4.19 (t, <i>J</i> = 5.3 Hz, 2H), 3.61 (t, <i>J</i> = 8.0 Hz, 2H), 2.89 (t, <i>J</i> = 5.1 Hz, 2H), 2.43 (s, 6H), 0.89 (t, <i>J</i> = 8.0 Hz, 2H), −0.06 (s, 9H). <sup>13</sup>C NMR (151 MHz, MeOD): δ 169.43, 155.51, 152.45, 151.60, 147.67, 139.82, 139.56, 135.42, 133.34, 132.92, 129.92, 128.48, 124.47, 124.19, 121.37, 119.46, 118.41, 116.62, 113.09, 103.52, 73.99, 67.33, 66.21, 59.07, 45.45, 41.91, 18.58, −1.36. TLC/MS: <i>m</i>/<i>z</i> calcd for [C<sub>37</sub>H<sub>43</sub>N<sub>7</sub>O<sub>4</sub>Si + H]<sup>+</sup>, 678.31; found, 677.70.</div></div><div id="sec4_16_6_76" class="NLM_sec NLM_sec_level_4"><div id="ac_i117" class="anchor-spacer"></div><h5 class="article-section__title" id="_i117"> <i>N</i>-{5-{4-{[3-Chloro-4-(pyridin-2-ylmethoxy)phenyl]amino}-7-{[2-(trimethylsilyl)ethoxy]methyl}-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-5-yl}-2-[(1-methoxypropan-2-yl)oxy]phenyl}acrylamide</h5><div class="NLM_p last">The reaction was performed according to the <a class="ref internalNav" href="#sec4_16_5" aria-label="General Method for Aniline Derivative Acylation">General Method for Aniline Derivative Acylation</a>. 5-{3-Amino-4-[(1-methoxypropan-2-yl)oxy]phenyl}-<i>N</i>-[3-chloro-4-(pyridin-2-ylmethoxy)phenyl]-7-{[2-(trimethylsilyl)ethoxy]methyl}-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-4-amine (<b>24a</b>) (16 mg, 0.02 mmol) was acylated and purified by flash column chromatography to yield the desired product. Yield: 64% (11 mg, 0.02 mmol). <sup>1</sup>H NMR (500 MHz, MeOD): δ 8.49 (d, <i>J</i> = 4.6 Hz, 1H), 8.43 (s, 1H), 8.27 (s, 1H), 7.83 (ddd, <i>J</i> = 7.7, 7.7, 1.6 Hz, 1H), 7.72 (d, <i>J</i> = 2.6 Hz, 1H), 7.63 (d, <i>J</i> = 7.9 Hz, 1H), 7.32 (dd, <i>J</i> = 7.0, 5.3 Hz, 1H), 7.28 (s, 1H), 7.26 (dd, <i>J</i> = 8.9, 2.6 Hz, 1H), 7.18 (s, 2H), 6.95 (d, <i>J</i> = 8.9 Hz, 1H), 6.51 (dd, <i>J</i> = 17.0, 10.2 Hz, 1H), 6.33 (dd, <i>J</i> = 17.0, 1.4 Hz, 1H), 5.76 (dd, <i>J</i> = 10.3, 1.1 Hz, 1H), 5.57 (s, 2H), 5.14 (s, 2H), 4.52–4.56 (m, 1H), 3.63–3.58 (m, 2H), 3.57 (d, <i>J</i> = 8.4 Hz, 2H), 3.41 (s, 3H), 1.34 (d, <i>J</i> = 6.3 Hz, 3H), 0.86 (t, <i>J</i> = 8.0 Hz, 2H), −0.09 (s, 9H). <sup>13</sup>C NMR (126 MHz, MeOD): δ 166.11, 157.94, 155.66, 152.42, 152.02, 151.45, 149.74, 148.82, 139.04, 134.57, 132.41, 130.86, 129.07, 128.29, 126.47, 125.19, 124.85, 124.47, 123.79, 123.16, 122.27, 117.95, 117.62, 115.25, 108.89, 103.12, 76.94, 76.63, 74.13, 72.21, 67.43, 59.56, 30.12, 16.84, −1.36. LC/MS: <i>m</i>/<i>z</i> calcd for [C<sub>37</sub>H<sub>43</sub>ClN<sub>6</sub>O<sub>5</sub>Si + H]<sup>+</sup>, 715.28; found, 715.23.</div></div><div id="sec4_16_6_77" class="NLM_sec NLM_sec_level_4"><div id="ac_i118" class="anchor-spacer"></div><h5 class="article-section__title" id="_i118"> <i>N</i>-{5-{4-{[3-Chloro-4-(pyridin-2-ylmethoxy)phenyl]amino}-7-{[2-(trimethylsilyl)ethoxy]methyl}-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-5-yl}-2-(2-hydroxyethoxy)phenyl}acrylamide</h5><div class="NLM_p last">The reaction was performed according to the <a class="ref internalNav" href="#sec4_16_5" aria-label="General Method for Aniline Derivative Acylation">General Method for Aniline Derivative Acylation</a>. 2-{2-Amino-4-{4-{[3-chloro-4-(pyridin-2-ylmethoxy)phenyl]amino}-7-{[2-(trimethylsilyl)ethoxy]methyl}-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-5-yl}phenoxy}ethan-1-ol (<b>24b</b>) (113 mg, 0.18 mmol) was acylated and purified by flash column chromatography to yield the desired product. Yield: 69% (84 mg, 0.12 mmol). <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.38 (s, 1H), 8.58 (d, <i>J</i> = 4.3 Hz, 1H), 8.44 (s, 1H), 8.38 (s, 1H), 7.88–7.84 (m, 2H), 7.60 (s, 1H), 7.54 (d, <i>J</i> = 7.8 Hz, 1H), 7.51 (s, 1H), 7.35 (ddd, <i>J</i> = 5.6, 2.9, 2.9 Hz, 2H), 7.22 (s, 2H), 7.13 (d, <i>J</i> = 9.0 Hz, 1H), 6.68 (dd, <i>J</i> = 16.9, 10.2 Hz, 1H), 6.26 (dd, <i>J</i> = 17.0, 1.5 Hz, 1H), 5.79 (dd, <i>J</i> = 10.2, 1.3 Hz, 1H), 5.60 (s, 2H), 5.23 (s, 2H), 5.07 (t, <i>J</i> = 6.1 Hz, 1H), 4.11 (t, <i>J</i> = 4.7 Hz, 2H), 3.80 (dd, <i>J</i> = 10.1, 5.3 Hz, 2H), 3.57 (t, <i>J</i> = 8.0 Hz, 2H), 0.85 (t, <i>J</i> = 13.2, 5.3 Hz, 2H), −0.07 (s, 9H). <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>): δ 163.46, 156.44, 153.73, 151.25, 150.97, 149.15 (s, <i>J</i> = 3.6 Hz), 149.10, 147.79, 137.07, 133.38, 131.80, 128.07, 127.23, 126.51, 124.59, 124.25, 123.01, 122.84, 121.33, 121.17, 121.08, 120.94, 115.67, 114.24, 113.45, 101.15, 72.28, 71.17, 71.01, 65.62, 59.52, 17.13, −1.36. LC/MS: <i>m</i>/<i>z</i> calcd for [C<sub>35</sub>H<sub>39</sub>ClN<sub>6</sub>O<sub>5</sub>Si + H]<sup>+</sup>, 687.24; found, 687.38.</div></div><div id="sec4_16_6_78" class="NLM_sec NLM_sec_level_4"><div id="ac_i119" class="anchor-spacer"></div><h5 class="article-section__title" id="_i119"> <i>N</i>-{5-{4-{[3-Chloro-4-(pyridin-2-ylmethoxy)phenyl]amino}-7-{[2-(trimethylsilyl)ethoxy]methyl}-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-5-yl}-2-(3-hydroxypropoxy)phenyl}acrylamide</h5><div class="NLM_p last">The reaction was performed according to the <a class="ref internalNav" href="#sec4_16_5" aria-label="General Method for Aniline Derivative Acylation">General Method for Aniline Derivative Acylation</a>. 3-{2-Amino-4-{4-{[3-chloro-4-(pyridin-2-ylmethoxy)phenyl]amino}-7-{[2-(trimethylsilyl)ethoxy]methyl}-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-5-yl}phenoxy}propan-1-ol (<b>24c</b>) (133 mg, 0.21 mmol) was acylated and purified by flash column chromatography to yield the desired product. Yield: 73% (105 mg, 0.15 mmol). <sup>1</sup>H NMR (700 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.41 (s, 1H), 8.58 (d, <i>J</i> = 4.8 Hz, 1H), 8.38 (s, 1H), 8.33 (s, 1H), 7.88 (d, <i>J</i> = 2.6 Hz, 1H), 7.86 (dd, <i>J</i> = 7.7, 1.7 Hz, 1H), 7.60 (s, 1H), 7.55 (d, <i>J</i> = 7.8 Hz, 1H), 7.51 (s, 1H), 7.36 (dd, <i>J</i> = 8.8, 2.5 Hz, 1H), 7.24 (dd, <i>J</i> = 8.3, 2.0 Hz, 1H), 7.22 (d, <i>J</i> = 8.4 Hz, 1H), 7.14 (d, <i>J</i> = 9.0 Hz, 1H), 6.71 (dd, <i>J</i> = 16.9, 10.2 Hz, 1H), 6.25 (dd, <i>J</i> = 17.0, 1.8 Hz, 1H), 5.76 (dd, <i>J</i> = 10.4, 1.3 Hz, 1H), 5.60 (s, 1H), 5.24 (s, 2H), 4.19 (t, <i>J</i> = 6.3 Hz, 2H), 3.63 (t, <i>J</i> = 6.1 Hz, 2H), 3.58 (t, <i>J</i> = 8.0 Hz, 2H), 3.42–3.46 (m, 1H), 1.96 (app. p, <i>J</i> = 6.2 Hz, 2H), 0.86 (t, <i>J</i> = 8.0 Hz, 2H), −0.06 (s, 9H). <sup>13</sup>C NMR (176 MHz, DMSO-<i>d</i><sub>6</sub>): δ 163.53, 156.42, 153.74, 151.19, 150.94, 149.19, 149.07, 148.33, 137.09, 133.35, 131.89, 127.83, 126.96, 126.17, 124.88, 124.23, 123.01, 122.94, 121.85, 121.35, 121.16, 120.96, 115.66, 114.25, 113.22, 101.17, 71.17, 65.61, 57.66, 56.07, 48.58, 31.83, 17.13, −1.38. LC/MS: <i>m</i>/<i>z</i> calcd for [C<sub>36</sub>H<sub>41</sub>ClN<sub>6</sub>O<sub>5</sub>Si + H]<sup>+</sup>, 701.26; found, 701.24.</div></div><div id="sec4_16_6_79" class="NLM_sec NLM_sec_level_4"><div id="ac_i120" class="anchor-spacer"></div><h5 class="article-section__title" id="_i120"> <i>N</i>-{5-{4-{[3-Chloro-4-(pyridin-2-ylmethoxy)phenyl]amino}-7-{[2-(trimethylsilyl)ethoxy]methyl}-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-5-yl}-2-[2-(dimethylamino)ethoxy]phenyl}acrylamide</h5><div class="NLM_p last">The reaction was performed according to the <a class="ref internalNav" href="#sec4_16_5" aria-label="General Method for Aniline Derivative Acylation">General Method for Aniline Derivative Acylation</a>. 5-{3-Amino-4-[2-(dimethylamino)ethoxy]phenyl}-<i>N</i>-[3-chloro-4-(pyridin-2-ylmethoxy)phenyl]-7-{[2-(trimethylsilyl)ethoxy]methyl}-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-4-amine (<b>24d</b>) (86 mg, 0.13 mmol) was acylated and purified by flash column chromatography to yield the desired product. Yield: 30% (28 mg, 0.04 mmol). <sup>1</sup>H NMR (700 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.81 (s, 1H), 8.58 (d, <i>J</i> = 4.8 Hz, 1H), 8.38 (s, 1H), 8.37 (s, 1H), 7.86 (ddd, <i>J</i> = 7.8, 7.8, 1.6 Hz, 2H), 7.61 (s, 1H), 7.55 (d, <i>J</i> = 7.8 Hz, 1H), 7.53 (s, 1H), 7.38 (dd, <i>J</i> = 9.0, 2.6 Hz, 1H), 7.35 (dd, <i>J</i> = 6.9, 5.2 Hz, 1H), 7.28 (d, <i>J</i> = 8.4 Hz, 1H), 7.23 (dd, <i>J</i> = 8.3, 2.2 Hz, 1H), 7.13 (d, <i>J</i> = 9.0 Hz, 1H), 6.57 (dd, <i>J</i> = 16.9, 10.2 Hz, 1H), 6.25 (dd, <i>J</i> = 16.9, 1.7 Hz, 1H), 5.78 (d, <i>J</i> = 11.7 Hz, 1H), 5.60 (s, 2H), 5.23 (s, 2H), 4.21 (t, <i>J</i> = 5.7 Hz, 2H), 3.58 (t, <i>J</i> = 8.0 Hz, 2H), 2.66 (t, <i>J</i> = 5.6 Hz, 2H), 2.27 (s, 6H), 0.86 (t, <i>J</i> = 8.0 Hz, 2H), −0.07 (s, 9H). <sup>13</sup>C NMR (176 MHz, DMSO-<i>d</i><sub>6</sub>): δ 163.39, 156.43, 153.70, 151.25, 150.99, 149.11, 149.08, 147.85, 137.05, 133.43, 131.78, 129.33, 127.35, 127.10, 124.81, 124.30, 123.00, 122.73, 121.53, 121.33, 120.95, 115.87, 115.62, 114.25, 101.14, 97.14, 72.28, 71.11, 65.67, 65.61, 57.25, 45.18, 17.02, −1.46. LC/MS: <i>m</i>/<i>z</i> calcd for [C<sub>37</sub>H<sub>44</sub>ClN<sub>7</sub>O<sub>4</sub>Si + H]<sup>+</sup>, 714.29; found, 714.28.</div></div><div id="sec4_16_6_80" class="NLM_sec NLM_sec_level_4"><div id="ac_i121" class="anchor-spacer"></div><h5 class="article-section__title" id="_i121"> <i>N</i>-{5-{4-[(1-Benzyl-1<i>H</i>-indazol-5-yl)amino]-7-{[2-(trimethylsilyl)ethoxy]methyl}-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-5-yl}-2-(2-hydroxyethoxy)phenyl}acrylamide</h5><div class="NLM_p last">The reaction was performed according to the <a class="ref internalNav" href="#sec4_16_5" aria-label="General Method for Aniline Derivative Acylation">General Method for Aniline Derivative Acylation</a>. 2-{2-Amino-4-{4-[(1-benzyl-1<i>H</i>-indazol-5-yl)amino]-7-{[2-(trimethylsilyl)ethoxy]methyl}-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-5-yl}phenoxy}ethan-1-ol (<b>24e</b>) (105 mg, 0.17 mmol) was acylated and purified by flash column chromatography to yield the desired product. Yield: 63% (72 mg, 0.11 mmol). <sup>1</sup>H NMR (700 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.52 (s, 1H), 8.48 (s, 1H), 8.36 (s, 1H), 8.18 (s, 1H), 8.04 (s, 1H), 7.58 (d, <i>J</i> = 6.5 Hz, 2H), 7.49 (s, 1H), 7.29 (t, <i>J</i> = 7.5 Hz, 4H), 7.25 (dd, <i>J</i> = 10.8, 4.7 Hz, 2H), 7.22 (d, <i>J</i> = 8.4 Hz, 1H), 7.19 (d, <i>J</i> = 7.7 Hz, 2H), 6.78 (dd, <i>J</i> = 16.6, 10.3 Hz, 1H), 6.24 (dd, <i>J</i> = 16.9 Hz, 1H), 5.76 (d, <i>J</i> = 10.1 Hz, 1H), 5.62 (s, 2H), 5.60 (s, 2H), 4.11 (t, <i>J</i> = 4.5 Hz, 2H), 3.80 (t, <i>J</i> = 4.2 Hz, 2H), 3.59 (t, <i>J</i> = 7.9 Hz, 2H), 0.86 (t, <i>J</i> = 8.0 Hz, 2H), −0.07 (s, 9H). <sup>13</sup>C NMR (176 MHz, DMSO-<i>d</i><sub>6</sub>): δ 163.55, 154.23, 151.41, 150.77, 147.92, 137.60, 136.42, 132.81, 132.46, 131.99, 128.48, 128.19, 127.44, 127.24, 127.02, 126.48, 124.57, 123.97, 123.80, 122.60, 121.33, 115.68, 113.33, 111.84, 109.72, 101.11, 72.26, 71.04, 65.60, 59.46, 51.90, 17.14, −1.37. LC/MS: <i>m</i>/<i>z</i> calcd for [C<sub>37</sub>H<sub>41</sub>N<sub>7</sub>O<sub>4</sub>Si + H]<sup>+</sup>, 676.29; found, 676.29.</div></div><div id="sec4_16_6_81" class="NLM_sec NLM_sec_level_4"><div id="ac_i122" class="anchor-spacer"></div><h5 class="article-section__title" id="_i122"> <i>N</i>-{5-{4-{{4-[(1,3-Dioxoisoindolin-2-yl)methyl]phenyl}amino}-7-{[2-(trimethylsilyl)ethoxy]methyl}-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-5-yl}-2-(2-hydroxyethoxy)phenyl}acrylamide</h5><div class="NLM_p last">The reaction was performed according to the <a class="ref internalNav" href="#sec4_16_5" aria-label="General Method for Aniline Derivative Acylation">General Method for Aniline Derivative Acylation</a>. 2-{4-{{5-[3-Amino-4-(2-hydroxyethoxy)phenyl]-7-{[2-(trimethylsilyl)ethoxy]methyl}-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-4-yl}amino}benzyl}isoindoline-1,3-dione (<b>24g</b>) (20 mg, 0.03 mmol) was acylated and purified by flash column chromatography to yield the desired product. Yield: 95% (20 mg, 0.03 mmol). <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.35 (s, 1H), 8.43 (s, 1H), 8.37 (s, 1H), 7.91–7.88 (m, 2H), 7.86–7.84 (m, 2H), 7.57 (s, 1H), 7.53 (s, 1H), 7.51 (d, <i>J</i> = 2.3 Hz, 2H), 7.22 (d, <i>J</i> = 9.6 Hz, 4H), 6.65 (dd, <i>J</i> = 17.0, 10.2 Hz, 1H), 6.22 (dd, <i>J</i> = 17.0, 1.7 Hz, 1H), 5.76 (s, 1H), 5.60 (s, 2H), 5.05 (t, <i>J</i> = 6.1 Hz, 1H), 4.71 (s, 2H), 4.11 (t, <i>J</i> = 4.7 Hz, 2H), 3.79 (dd, <i>J</i> = 10.1, 5.4 Hz, 2H), 3.57 (t, <i>J</i> = 8.0 Hz, 2H), 0.85 (t, <i>J</i> = 8.0 Hz, 2H), −0.08 (s, 9H). TLC/MS: <i>m</i>/<i>z</i> calcd for [C<sub>38</sub>H<sub>40</sub>N<sub>6</sub>O<sub>6</sub>Si + H]<sup>+</sup>, 705.28; found, 704.60.</div></div><div id="sec4_16_6_82" class="NLM_sec NLM_sec_level_4"><div id="ac_i123" class="anchor-spacer"></div><h5 class="article-section__title" id="_i123"> <i>N</i>-{2-[2-(Dimethylamino)ethoxy]-5-{4-{{4-[(1,3-dioxoisoindolin-2-yl)methyl]phenyl}amino}-7-{[2-(trimethylsilyl)ethoxy]methyl}-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-5-yl}phenyl}acrylamide</h5><div class="NLM_p last">The reaction was performed according to the <a class="ref internalNav" href="#sec4_16_5" aria-label="General Method for Aniline Derivative Acylation">General Method for Aniline Derivative Acylation</a>. 2-{4-{{5-{3-Amino-4-[2-(dimethylamino)ethoxy]phenyl}-7-{[2-(trimethylsilyl)ethoxy]methyl}-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-4-yl}amino}benzyl}isoindoline-1,3-dione (<b>24h</b>) (6 mg, 0.009 mmol) was acylated and purified by flash column chromatography to yield the desired product. Yield: 62% (4 mg, 0.006 mmol). <sup>1</sup>H NMR (600 MHz, MeOD): δ 8.49 (d, <i>J</i> = 2.0 Hz, 1H), 8.33 (s, 1H), 7.87–7.84 (m, 2H), 7.80–7.78 (m, 2H), 7.50–7.48 (m, 2H), 7.34 (s, 1H), 7.29 (d, <i>J</i> = 8.6 Hz, 2H), 7.25 (dd, <i>J</i> = 8.3, 2.2 Hz, 1H), 7.18 (d, <i>J</i> = 8.4 Hz, 1H), 6.61 (dd, <i>J</i> = 17.0, 10.3 Hz, 1H), 6.33 (dd, <i>J</i> = 17.0, 1.5 Hz, 1H), 5.76 (dd, <i>J</i> = 10.3, 1.5 Hz, 1H), 5.63 (s, 2H), 4.77 (s, 2H), 4.23 (t, <i>J</i> = 5.0 Hz, 2H), 3.62 (t, <i>J</i> = 8.0 Hz, 2H), 2.86 (t, <i>J</i> = 5.0 Hz, 2H), 2.44 (s, 6H), 0.90 (t, <i>J</i> = 7.9 Hz, 2H), −0.05 (s, 9H). <sup>13</sup>C NMR (151 MHz, MeOD): δ 169.44, 166.44, 155.67, 152.52, 152.02, 149.84, 139.81, 135.45, 133.37, 133.15, 132.40, 130.80, 129.83, 129.36, 128.11, 126.59, 125.13, 124.21, 123.51, 122.08, 117.89, 113.48, 103.45, 74.12, 67.42, 66.20, 58.86, 45.17, 41.92, 18.58, −1.39. TLC/MS: <i>m</i>/<i>z</i> calcd for [C<sub>40</sub>H<sub>45</sub>N<sub>7</sub>O<sub>5</sub>Si + H]<sup>+</sup>, 732.32; found, 731.6.</div></div><div id="sec4_16_6_83" class="NLM_sec NLM_sec_level_4"><div id="ac_i124" class="anchor-spacer"></div><h5 class="article-section__title" id="_i124"> <i>N</i>-{5-{4-{[3-Chloro-4-(pyridin-2-ylmethoxy)phenyl]amino}-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-5-yl}-2-[(1-methoxypropan-2-yl)oxy]phenyl}acrylamide (<b>18a</b>)</h5><div class="NLM_p last">The reaction was performed according to the <a class="ref internalNav" href="#sec4_16_6" aria-label="General Method for SEM Deprotection">General Method for SEM Deprotection</a>. <i>N</i>-{5-{4-{[3-Chloro-4-(pyridin-2-ylmethoxy)phenyl]amino}-7-{[2-(trimethylsilyl)ethoxy]methyl}-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-5-yl}-2-[(1-methoxypropan-2-yl)oxy]phenyl}acrylamide (11 mg, 0.02 mmol) was deprotected and purified by flash column chromatography to afford the pure compound. Yield: 71% (7 mg, 0.01 mmol). <sup>1</sup>H NMR (600 MHz, MeOD): δ 8.60 (d, <i>J</i> = 4.6 Hz, 1H), 8.43 (s, 1H), 8.24 (s, 1H), 8.01–7.96 (m, 1H), 7.76 (d, <i>J</i> = 8.1 Hz, 2H), 7.50–7.45 (m, 2H), 7.41 (dd, <i>J</i> = 8.7, 1.5 Hz, 1H), 7.32 (d, <i>J</i> = 8.2 Hz, 1H), 7.24 (d, <i>J</i> = 7.8 Hz, 2H), 6.54 (dd, <i>J</i> = 16.9, 10.2 Hz, 1H), 6.38 (d, <i>J</i> = 16.6 Hz, 1H), 5.81 (d, <i>J</i> = 10.3 Hz, 1H), 5.32 (s, 2H), 4.63–4.56 (m, 1H), 3.67–3.58 (m, 2H), 3.44 (s, 3H), 1.37 (d, <i>J</i> = 6.3 Hz, 3H). <sup>13</sup>C NMR (151 MHz, MeOD): δ 166.17, 157.09, 154.71, 152.54, 150.22, 149.41, 149.32, 144.74, 139.89, 132.33, 130.66, 130.37, 129.27, 128.24, 127.85, 126.77, 126.39, 124.94, 124.87, 124.47, 123.62, 123.49, 120.30, 117.67, 115.87, 101.31, 76.83, 76.67, 71.97, 59.53, 16.80. HRMS: <i>m</i>/<i>z</i> calcd for [C<sub>31</sub>H<sub>29</sub>ClN<sub>6</sub>O<sub>4</sub> + H<sup>+</sup>], 585.2011; found, 585.2012, calcd for [C<sub>31</sub>H<sub>30</sub>N<sub>4</sub>O<sub>6</sub> + <sup>37</sup>Cl]<sup>+</sup>, 587.1991; found, 587.1982.</div></div><div id="sec4_16_6_84" class="NLM_sec NLM_sec_level_4"><div id="ac_i125" class="anchor-spacer"></div><h5 class="article-section__title" id="_i125"> <i>N</i>-{5-{4-{[3-Chloro-4-(pyridin-2-ylmethoxy)phenyl]amino}-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-5-yl}-2-(2-hydroxyethoxy)phenyl}acrylamide (<b>18b</b>)</h5><div class="NLM_p last">The reaction was performed according to the <a class="ref internalNav" href="#sec4_16_6" aria-label="General Method for SEM Deprotection">General Method for SEM Deprotection</a>. <i>N</i>-{5-{4-{[3-Chloro-4-(pyridin-2-ylmethoxy)phenyl]amino}-7-{[2-(trimethylsilyl)ethoxy]methyl}-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-5-yl}-2-(2-hydroxyethoxy)phenyl}acrylamide (77 mg, 0.11 mmol) was deprotected and purified by flash column chromatography to afford the pure compound. Yield: 55% (35 mg, 0.06 mmol). <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>): δ 12.03 (s, 1H), 9.39 (s, 1H), 8.58 (d, <i>J</i> = 4.5 Hz, 1H), 8.42 (s, 1H), 8.32 (s, 1H), 7.89 (d, <i>J</i> = 3.0 Hz, 1H), 7.86 (ddd, <i>J</i> = 7.7, 7.7, 1.5 Hz, 1H), 7.55 (d, <i>J</i> = 7.8 Hz, 1H), 7.50 (s, 1H), 7.38–7.34 (m, 2H), 7.32 (d, <i>J</i> = 2.3 Hz, 1H), 7.23 (dd, <i>J</i> = 5.3, 2.7 Hz, 1H), 7.20 (d, <i>J</i> = 8.4 Hz, 1H), 7.13 (d, <i>J</i> = 9.0 Hz, 1H), 6.68 (dd, <i>J</i> = 17.0, 10.2 Hz, 1H), 6.27 (dd, <i>J</i> = 17.0, 1.5 Hz, 1H), 5.79 (d, <i>J</i> = 10.2 Hz, 1H), 5.23 (s, 2H), 5.07 (t, <i>J</i> = 6.1 Hz, 1H), 4.11 (t, <i>J</i> = 4.7 Hz, 2H), 3.80 (dd, <i>J</i> = 10.1, 5.3 Hz, 2H). <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>): δ 163.46, 156.46, 153.57, 151.39, 150.87, 149.08, 148.94, 147.63, 137.07, 133.62, 131.81, 128.01, 127.29, 127.21, 124.74, 123.00, 122.56, 121.32, 121.29, 121.17, 120.95, 120.78, 115.14, 114.28, 113.48, 100.92, 71.18, 71.00, 59.49. HRMS: <i>m</i>/<i>z</i> calcd for [C<sub>29</sub>H<sub>25</sub>ClN<sub>6</sub>O<sub>4</sub> + H<sup>+</sup>], 557.1684; found, 557.1699, calcd for [C<sub>29</sub>H<sub>25</sub><sup>37</sup>ClN<sub>6</sub>O<sub>4</sub>]<sup>+</sup>, 559.1656; found, 559.1669.</div></div><div id="sec4_16_6_85" class="NLM_sec NLM_sec_level_4"><div id="ac_i126" class="anchor-spacer"></div><h5 class="article-section__title" id="_i126"> <i>N</i>-{5-{4-{[3-Chloro-4-(pyridin-2-ylmethoxy)phenyl]amino}-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-5-yl}-2-(3-hydroxypropoxy)phenyl}acrylamide (<b>18c</b>)</h5><div class="NLM_p last">The reaction was performed according to the <a class="ref internalNav" href="#sec4_16_6" aria-label="General Method for SEM Deprotection">General Method for SEM Deprotection</a>. <i>N</i>-{5-{4-{[3-Chloro-4-(pyridin-2-ylmethoxy)phenyl]amino}-7-{[2-(trimethylsilyl)ethoxy]methyl}-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-5-yl}-2-(3-hydroxypropoxy)phenyl}acrylamide (100 mg, 0.14 mmol) was deprotected and purified by flash column chromatography to afford the pure compound. Yield: 59% (47 mg, 0.08 mmol). <sup>1</sup>H NMR (700 MHz, DMSO-<i>d</i><sub>6</sub>): δ 12.03 (s, 1H), 9.41 (s, 1H), 8.58 (d, <i>J</i> = 4.7 Hz, 1H), 8.31 (s, 1H), 8.30 (s, 1H), 7.90 (d, <i>J</i> = 2.6 Hz, 1H), 7.86 (ddd, <i>J</i> = 7.7, 7.7, 1.7 Hz, 1H), 7.55 (d, <i>J</i> = 7.8 Hz, 1H), 7.50 (s, 1H), 7.36 (dd, <i>J</i> = 8.8, 2.4 Hz, 2H), 7.32 (d, <i>J</i> = 2.4 Hz, 1H), 7.24 (dd, <i>J</i> = 8.3, 2.1 Hz, 1H), 7.20 (d, <i>J</i> = 8.4 Hz, 1H), 7.13 (d, <i>J</i> = 9.0 Hz, 1H), 6.70 (dd, <i>J</i> = 17.0, 10.2 Hz, 1H), 6.25 (dd, <i>J</i> = 17.0, 1.7 Hz, 1H), 5.75 (dd, <i>J</i> = 8.0, 3.6 Hz, 1H), 5.23 (s, 2H), 4.19 (t, <i>J</i> = 6.3 Hz, 2H), 3.63 (t, <i>J</i> = 6.0 Hz, 2H), 1.95 (app. p, <i>J</i> = 6.2 Hz, 2H). <sup>13</sup>C NMR (176 MHz, DMSO-<i>d</i><sub>6</sub>): δ 163.50, 156.41, 153.51, 151.30, 150.77, 149.08, 149.02, 148.17, 137.03, 133.53, 131.86, 127.73, 126.90, 124.96, 123.00, 122.67, 121.96, 121.34, 121.16, 120.94, 120.85, 115.11, 114.29, 113.24, 100.92, 71.18, 66.27, 57.58, 31.84. HRMS: <i>m</i>/<i>z</i> calcd for [C<sub>30</sub>H<sub>27</sub>ClN<sub>6</sub>O<sub>4</sub> + H<sup>+</sup>], 571.1851; found, 571.1855, calcd for [C<sub>30</sub>H<sub>27</sub><sup>37</sup>ClN<sub>6</sub>O<sub>4</sub>]<sup>+</sup>, 573.1824; found, 573.1826.</div></div><div id="sec4_16_6_86" class="NLM_sec NLM_sec_level_4"><div id="ac_i127" class="anchor-spacer"></div><h5 class="article-section__title" id="_i127"> <i>N</i>-{5-{4-{[3-Chloro-4-(pyridin-2-ylmethoxy)phenyl]amino}-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-5-yl}-2-[2-(dimethylamino)ethoxy]phenyl}acrylamide (<b>18d</b>)</h5><div class="NLM_p last">The reaction was performed according to the <a class="ref internalNav" href="#sec4_16_6" aria-label="General Method for SEM Deprotection">General Method for SEM Deprotection</a>. <i>N</i>-{5-{4-{[3-Chloro-4-(pyridin-2-ylmethoxy)phenyl]amino}-7-{[2-(trimethylsilyl)ethoxy]methyl}-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-5-yl}-2-[2-(dimethylamino)ethoxy]phenyl}acrylamide (26 mg, 0.04 mmol) was deprotected and purified by flash column chromatography to afford the pure compound. Yield: 58% (12 mg, 0.02 mmol). <sup>1</sup>H NMR (600 MHz, MeOD): δ 8.53 (d, <i>J</i> = 4.6 Hz, 1H), 8.44 (s, 1H), 8.26 (s, 1H), 7.88 (ddd, <i>J</i> = 7.7, 7.7, 1.4 Hz, 1H), 7.78 (d, <i>J</i> = 2.6 Hz, 1H), 7.67 (d, <i>J</i> = 7.9 Hz, 1H), 7.37 (dd, <i>J</i> = 6.9, 5.0 Hz, 1H), 7.31 (dd, <i>J</i> = 8.8, 2.6 Hz, 1H), 7.26 (dd, <i>J</i> = 8.3, 2.0 Hz, 1H), 7.20 (s, 1H), 7.17 (d, <i>J</i> = 8.4 Hz, 1H), 7.00 (d, <i>J</i> = 8.9 Hz, 1H), 6.64 (dd, <i>J</i> = 17.0, 10.3 Hz, 1H), 6.39 (dd, <i>J</i> = 17.0, 1.3 Hz, 1H), 5.80 (dd, <i>J</i> = 10.4, 1.1 Hz, 1H), 5.19 (s, 2H), 4.27 (t, <i>J</i> = 4.8 Hz, 2H), 3.00 (t, <i>J</i> = 3.7 Hz, 2H), 2.54 (s, 6H). <sup>13</sup>C NMR (151 MHz, MeOD): δ 166.43, 157.96, 155.71, 152.09, 151.92, 151.44, 149.76, 149.55, 139.00, 134.75, 132.39, 129.13, 129.06, 128.19, 126.76, 124.88, 124.47, 123.87, 123.65, 123.19, 122.29, 122.08, 117.34, 115.36, 113.58, 102.82, 72.35, 65.74, 58.64, 44.93. HRMS: <i>m</i>/<i>z</i> calcd for [C<sub>31</sub>H<sub>30</sub>ClN<sub>7</sub>O<sub>3</sub> + H<sup>+</sup>], 584.2169; found, 584.2171, calcd for [C<sub>31</sub>H<sub>30</sub><sup>37</sup>ClN<sub>7</sub>O<sub>3</sub>]<sup>+</sup>, 586.2142; found, 586.2142.</div></div><div id="sec4_16_6_87" class="NLM_sec NLM_sec_level_4"><div id="ac_i128" class="anchor-spacer"></div><h5 class="article-section__title" id="_i128"> <i>N</i>-{5-{4-[(1-Benzyl-1<i>H</i>-indazol-5-yl)amino]-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-5-yl}-2-(2-hydroxyethoxy)phenyl}acrylamide (<b>19</b>)</h5><div class="NLM_p last">The reaction was performed according to the <a class="ref internalNav" href="#sec4_16_6" aria-label="General Method for SEM Deprotection">General Method for SEM Deprotection</a>. <i>N</i>-{5-{4-[(1-Benzyl-1<i>H</i>-indazol-5-yl)amino]-7-{[2-(trimethylsilyl)ethoxy]methyl}-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-5-yl}-2-(2-hydroxyethoxy)phenyl}acrylamide (72 mg, 0.11 mmol) was deprotected and purified by flash column chromatography to afford the pure compound. Yield: 52% (30 mg, 0.05 mmol). <sup>1</sup>H NMR (700 MHz, DMSO-<i>d</i><sub>6</sub>): δ 11.99 (d, <i>J</i> = 1.8 Hz, 1H), 9.37 (s, 1H), 8.45 (s, 1H), 8.30 (s, 1H), 8.20 (d, <i>J</i> = 1.6 Hz, 1H), 8.03 (d, <i>J</i> = 0.6 Hz, 1H), 7.57 (d, <i>J</i> = 9.0 Hz, 1H), 7.48 (s, 1H), 7.32–7.27 (m, 4H), 7.27–7.24 (m, 2H), 7.22 (d, <i>J</i> = 5.8 Hz, 1H), 7.19 (d, <i>J</i> = 7.2 Hz, 2H), 6.67 (dd, <i>J</i> = 17.0, 10.2 Hz, 1H), 6.25 (dd, <i>J</i> = 17.0, 1.7 Hz, 1H), 5.78 (d, <i>J</i> = 11.6 Hz, 1H), 5.62 (s, 2H), 5.06 (t, <i>J</i> = 6.1 Hz, 1H), 4.12 (t, <i>J</i> = 4.7 Hz, 2H), 3.80 (dd, <i>J</i> = 10.0, 5.3 Hz, 2H). <sup>13</sup>C NMR (176 MHz, DMSO-<i>d</i><sub>6</sub>): δ 163.42, 154.07, 151.16, 151.03, 147.67, 137.61, 136.31, 132.75, 132.68, 131.83, 128.47, 128.10, 127.43, 127.37, 127.22, 127.17, 124.73, 123.81, 122.43, 121.38, 120.86, 115.11, 113.50, 111.46, 109.68, 100.86, 71.04, 59.52, 51.88. HRMS: <i>m</i>/<i>z</i> calcd for [C<sub>31</sub>H<sub>27</sub>N<sub>7</sub>O<sub>3</sub> + H<sup>+</sup>], 546.2241; found, 546.2248.</div></div><div id="sec4_16_6_88" class="NLM_sec NLM_sec_level_4"><div id="ac_i129" class="anchor-spacer"></div><h5 class="article-section__title" id="_i129"> <i>N</i>-{5-{4-[(2-Benzyl-1<i>H</i>-benzo[<i>d</i>]imidazol-5-yl)amino]-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-5-yl}-2-(2-hydroxyethoxy)phenyl}acrylamide (<b>20</b>)</h5><div class="NLM_p last">The reaction was performed according to the <a class="ref internalNav" href="#sec4_16_5" aria-label="General Method for Aniline Derivative Acylation">General Method for Aniline Derivative Acylation</a>, followed by the <a class="ref internalNav" href="#sec4_16_6" aria-label="General Method for SEM Deprotection">General Method for SEM Deprotection</a>. 2-{2-Amino-4-{4-[(2-benzyl-1<i>H</i>-benzo[<i>d</i>]imidazol-5-yl)amino]-7-{[2-(trimethylsilyl)ethoxy]methyl}-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-5-yl}phenoxy}ethan-1-ol (<b>24f</b>) (49 mg, 0.08 mmol) was acylated and purified by flash column chromatography. Subsequently, <i>N</i>-{5-{4-[(2-benzyl-1<i>H</i>-benzo[<i>d</i>]imidazol-5-yl)amino]-7-{[2-(trimethylsilyl)ethoxy]methyl}-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-5-yl}-2-(2-hydroxyethoxy)phenyl}acrylamide (31 mg, 0.05 mmol) was deprotected and purified by flash column chromatography to afford the pure compound. Yield: 12% (5 mg, 0.01 mmol) over two steps. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>): δ 11.98 (s, 1H), 9.38 (s, 1H), 8.43 (s, 1H), 8.34 (s, 1H), 7.73–7.69 (m, 1H), 7.69–7.64 (m, 1H), 7.48 (d, <i>J</i> = 8.9 Hz, 1H), 7.39–7.34 (m, 5H), 7.33–7.28 (m, 2H), 7.26 (dd, <i>J</i> = 8.3, 2.1 Hz, 1H), 7.22 (s, 1H), 7.20 (s, 1H), 6.67 (dd, <i>J</i> = 16.9, 10.3 Hz, 1H), 6.22 (dd, <i>J</i> = 17.0, 1.7 Hz, 1H), 5.77 (dd, <i>J</i> = 10.1, 1.6 Hz, 1H), 5.32 (t, <i>J</i> = 4.4 Hz, 2H), 4.32 (s, 1H), 4.12 (t, <i>J</i> = 4.7 Hz, 2H), 3.80 (s, 2H). HRMS: <i>m</i>/<i>z</i> calcd for [C<sub>31</sub>H<sub>27</sub>N<sub>7</sub>O<sub>3</sub> + H<sup>+</sup>], 546.2234; found, 546.2248.</div></div><div id="sec4_16_6_89" class="NLM_sec NLM_sec_level_4"><div id="ac_i130" class="anchor-spacer"></div><h5 class="article-section__title" id="_i130"> <i>N</i>-{5-{4-{{4-[(1,3-Dioxoisoindolin-2-yl)methyl]phenyl}amino}-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-5-yl}-2-(2-hydroxyethoxy)phenyl}acrylamide (<b>21a</b>)</h5><div class="NLM_p last">The reaction was performed according to the <a class="ref internalNav" href="#sec4_16_6" aria-label="General Method for SEM Deprotection">General Method for SEM Deprotection</a>. <i>N</i>-{5-{4-{{4-[(1,3-Dioxoisoindolin-2-yl)methyl]phenyl}amino}-7-{[2-(trimethylsilyl)ethoxy]methyl}-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-5-yl}-2-(2-hydroxyethoxy)phenyl}acrylamide (20 mg, 0.03 mmol) was deprotected and purified by flash column chromatography to afford the pure compound. Yield: 29% (5 mg, 0.009 mmol). <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.35 (s, 1H), 8.44 (s, 1H), 8.37 (s, 1H), 7.90–7.88 (m, 2H), 7.86–7.84 (m, 2H), 7.55 (s, 1H), 7.53 (dd, <i>J</i> = 7.9, 1.9 Hz, 2H), 7.43 (s, 1H), 7.23 (s, 2H), 7.21 (s, 2H) 6.63 (dd, <i>J</i> = 16.4, 8.9 Hz, 2H), 6.22 (dd, <i>J</i> = 17.0, 1.7 Hz, 1H), 5.75 (dd, <i>J</i> = 10.2, 1.6 Hz, 1H), 5.58 (s, 1H), 5.06 (t, <i>J</i> = 6.1 Hz, 1H), 4.71 (s, 2H), 4.11 (t, <i>J</i> = 4.7 Hz, 2H), 3.79 (dd, <i>J</i> = 10.3, 5.2 Hz, 2H). <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>): δ 167.68, 163.35, 153.55, 150.98, 149.97, 147.77, 138.63, 134.56, 131.84, 131.57, 130.78, 128.14, 127.89, 127.15, 126.60, 124.50, 123.68, 123.20, 120.35, 120.19, 115.16, 113.41, 101.49, 70.99, 59.49, 40.06. HRMS: <i>m</i>/<i>z</i> calcd for [C<sub>32</sub>H<sub>26</sub>N<sub>6</sub>O<sub>5</sub> + H<sup>+</sup>], 575.2035; found, 575.2037.</div></div><div id="sec4_16_6_90" class="NLM_sec NLM_sec_level_4"><div id="ac_i131" class="anchor-spacer"></div><h5 class="article-section__title" id="_i131"> <i>N</i>-{2-[2-(Dimethylamino)ethoxy]-5-{4-{{4-[(1,3-dioxoisoindolin-2-yl)methyl]phenyl}amino}-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-5-yl}phenyl}acrylamide (<b>21b</b>)</h5><div class="NLM_p last">The reaction was performed according to the <a class="ref internalNav" href="#sec4_16_6" aria-label="General Method for SEM Deprotection">General Method for SEM Deprotection</a>. 2-{4-{{5-{3-Amino-4-[2-(dimethylamino)ethoxy]phenyl}-7-{[2-(trimethylsilyl)ethoxy]methyl}-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-4-yl}amino}benzyl}isoindoline-1,3-dione (4 mg, 0.006 mmol) was deprotected and purified by flash column chromatography to afford the pure compound. Yield: 75% (2.47 mg, 0.004 mmol). <sup>1</sup>H NMR (600 MHz, DMSO): δ 8.37 (s, 4H), 7.91–7.88 (m, 2H), 7.87–7.84 (m, 2H), 7.56–7.51 (m, 2H), 7.45 (s, 1H), 7.27 (s, 1H), 7.22 (d, <i>J</i> = 8.6 Hz, 1H), 6.63 (dd, <i>J</i> = 18.6, 11.2 Hz, 1H), 6.24 (dd, <i>J</i> = 17.0, 1.5 Hz, 1H), 5.76 (s, 1H), 5.59 (d, <i>J</i> = 7.3 Hz, 1H), 4.71 (s, 2H), 4.39–4.31 (m, 2H), 3.17 (t, <i>J</i> = 5.1 Hz, 2H). <sup>13</sup>C NMR (151 MHz, DMSO): δ 174.50, 170.34, 167.69, 163.42, 153.58, 151.02, 150.00, 138.64, 134.59, 131.78, 131.56, 130.84, 129.65, 127.92, 127.38, 127.27, 124.76, 123.78, 123.21, 121.81, 120.40, 115.00, 101.50, 66.40, 59.73, 54.90, 48.58. HRMS: <i>m</i>/<i>z</i> calcd for [C<sub>34</sub>H<sub>31</sub>N<sub>7</sub>O<sub>4</sub> + H<sup>+</sup>], 602.2510; found, 602.2510.</div></div><div id="sec4_16_6_91" class="NLM_sec NLM_sec_level_4"><div id="ac_i132" class="anchor-spacer"></div><h5 class="article-section__title" id="_i132"> 5-Iodo-4-methoxy-7-{[2-(trimethylsilyl)ethoxy]methyl}-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidine (<b>26</b>)</h5><div class="NLM_p last">4-Chloro-5-iodo-7-{[2-(trimethylsilyl)ethoxy]methyl}-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidine (<b>10</b>) (3.35 g, 8.18 mmol, 1 equiv) and potassium carbonate (2.49 g, 17.99 mmol, 2.2 equiv) in MeOH (40 mL) were allowed to stir at room temperature overnight (20 h). The solvent was removed <i>in vacuo</i> and diluted with EtOAc (50 mL) and water (50 mL), and the organic layer was separated. The aqueous layer was extracted with aliquots of EtOAc (3 × 30 mL), and the combined organic layers were then washed with a saturated solution of brine (50 mL), dried over MgSO<sub>4</sub>, and filtered. After removal of the solvent <i>in vacuo</i>, the crude product was adsorbed onto silica, and purification by flash column chromatography afforded the desired compound. Yield: 92% (3.05 g, 7.51 mmol). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.45 (s, 1H), 7.75 (s, 1H), 5.53 (s, 2H), 4.05 (s, 3H), 3.48 (t, <i>J</i> = 8.0 Hz, 2H), 0.80 (t, <i>J</i> = 8.0 Hz, 2H), −0.11 (s, 9H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>): δ 162.3, 151.9, 151.3, 132.3, 106.3, 72.5, 65.7, 53.7, 50.9, 17.1, −1.42. HRMS: <i>m</i>/<i>z</i> calcd for [C<sub>13</sub>H<sub>20</sub>IN<sub>3</sub>O<sub>2</sub>Si + H<sup>+</sup>], 406.0442; found, 406.0432.</div></div><div id="sec4_16_6_92" class="NLM_sec NLM_sec_level_4"><div id="ac_i133" class="anchor-spacer"></div><h5 class="article-section__title" id="_i133"> 5-(4-Fluoro-3-nitrophenyl)-4-methoxy-7-((2-(trimethylsilyl)ethoxy)methyl)-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidine (<b>27</b>)</h5><div class="NLM_p last">The reaction was performed according to the <a class="ref internalNav" href="#sec4_16_2" aria-label="General Method for Suzuki–Miyaura Coupling">General Method for Suzuki–Miyaura Coupling</a>. It used 5-iodo-4-methoxy-7-{[2-(trimethylsilyl)ethoxy]methyl}-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidine (<b>26</b>) (1 g, 2.47 mmol) and (4-fluoro-3-nitrophenyl)boronic acid (548 mg, 2.96 mmol) that were coupled and purified by flash column chromatography. This afforded the desired compound. Yield: 86% (890 mg, 2.13 mmol). <sup>1</sup>H NMR (700 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.50 (s, 1H), 8.49 (dd, <i>J</i> = 7.3, 2.2 Hz, 1H), 8.09 (ddd, <i>J</i> = 8.8, 3.9, 2.4 Hz, 1H), 8.03 (s, 1H), 7.62 (dd, <i>J</i> = 11.1, 8.8 Hz, 1H), 5.61 (s, 2H), 4.05 (s, 3H), 3.56 (t, <i>J</i> = 8.0 Hz, 2H), 0.84 (t, <i>J</i> = 8.0, 2H), −0.09 (s, 9H). <sup>13</sup>C NMR (176 MHz, DMSO-<i>d</i><sub>6</sub>): δ 162.5, 152.7, 151.3, 136.6, 135.5, 130.9, 126.7, 125.2, 118.4, 118.3, 112.9, 102.1, 72.8, 65.9, 53.8, 17.1, −1.45. LC/MS: <i>m</i>/<i>z</i> calcd for [C<sub>19</sub>H<sub>23</sub>FN<sub>4</sub>O<sub>4</sub>Si + H<sup>+</sup>], 419.15; found, 419.15.</div></div><div id="sec4_16_6_93" class="NLM_sec NLM_sec_level_4"><div id="ac_i134" class="anchor-spacer"></div><h5 class="article-section__title" id="_i134"> 4-Methoxy-5-[4-(2-methoxyethoxy)-3-nitrophenyl]-7-{[2-(trimethylsilyl)ethoxy]methyl}-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidine</h5><div class="NLM_p last">The reaction was performed according to the <a class="ref internalNav" href="#sec4_16_3" aria-label="General Method for Nucleophilic Aromatic Substitution">General Method for Nucleophilic Aromatic Substitution</a>. 5-(4-Fluoro-3-nitrophenyl)-4-methoxy-7-{[2-(trimethylsilyl)ethoxy]methyl}-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidine (<b>27</b>) (440 mg, 1.05 mmol) was alkylated with 2-methoxyethan-1-ol (800 mg, 10.5 mmol), and purification by flash column chromatography afforded the pure compound. Yield: 71% (354 mg, 0.75 mmol). <sup>1</sup>H NMR (500 MHz, CDCl<sub>3</sub>): δ 8.51 (s, 1H), 8.23 (d, <i>J</i> = 2.2 Hz, 1H), 7.78 (dd, <i>J</i> = 8.7, 2.2 Hz, 1H), 7.30 (s, 1H), 7.14 (d, <i>J</i> = 8.7 Hz, 1H), 5.65 (s, 2H), 4.29 (t, <i>J</i> = 4.9, 2H), 4.10 (s, 3H), 3.82 (t, <i>J</i> = 5.0, 2H), 3.56 (t, <i>J</i> = 8.2 2H), 3.46 (s, 3H), 0.92 (t, <i>J</i> = 8.0, 2H), −0.06 (s, 9H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>): δ 163.41, 152.95, 151.59, 151.18, 139.89, 133.99, 127.05, 125.97, 123.69, 115.49, 115.05, 103.21, 73.24, 70.79, 69.82, 66.76, 59.62, 54.05, 17.80, −1.34. LC/MS: <i>m</i>/<i>z</i> calcd for [C<sub>22</sub>H<sub>30</sub>N<sub>4</sub>O<sub>6</sub>Si + H<sup>+</sup>], 475.20; found, 474.95.</div></div><div id="sec4_16_6_94" class="NLM_sec NLM_sec_level_4"><div id="ac_i135" class="anchor-spacer"></div><h5 class="article-section__title" id="_i135"> 4-Methoxy-5-{4-[(1-methoxypropan-2-yl)oxy]-3-nitrophenyl}-7-{[2-(trimethylsilyl)ethoxy]methyl}-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidine</h5><div class="NLM_p last">The reaction was performed according to the <a class="ref internalNav" href="#sec4_16_3" aria-label="General Method for Nucleophilic Aromatic Substitution">General Method for Nucleophilic Aromatic Substitution</a>. 5-(4-Fluoro-3-nitrophenyl)-4-methoxy-7-{[2-(trimethylsilyl)ethoxy]methyl}-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidine (<b>27</b>) (500 mg, 1.19 mmol) was reacted with 1-methoxypropan-2-ol (129 mg, 1.43 mmol), and purification by flash column chromatography afforded the pure compound. Yield: 73% (425 mg, 0.87 mmol). <sup>1</sup>H NMR (700 MHz, CDCl<sub>3</sub>): δ 8.52 (s, 1H), 8.18 (d, <i>J</i> = 2.1 Hz, 1H), 7.76 (dd, <i>J</i> = 8.7, 2.1 Hz, 1H), 7.31 (s, 1H), 7.20 (d, <i>J</i> = 8.7 Hz, 1H), 5.66 (s, 2H), 4.71–4.68 (m, 1H), 4.12 (s, 3H), 3.68–3.64 (m, 1H), 3.57 (t, <i>J</i> = 8.4 Hz, 2H), 3.56–3.54 (m, 1H), 3.42 (s, 3H), 1.40 (d, <i>J</i> = 6.3 Hz, 3H), 0.93 (t, <i>J</i> = 8.4 Hz, 2H), −0.05 (s, 9H). <sup>13</sup>C NMR (176 MHz, CDCl<sub>3</sub>): δ 163.48, 152.93, 151.54, 150.48, 141.04, 133.65, 127.00, 125.77, 123.73, 116.80, 115.70, 103.29, 76.52, 76.07, 73.34, 66.82, 59.60, 54.13, 17.87, 17.12, −1.31. LC/MS: <i>m</i>/<i>z</i> calcd for [C<sub>23</sub>H<sub>32</sub>N<sub>4</sub>O<sub>6</sub>Si + H]<sup>+</sup>, 489.22; found, 488.99.</div></div><div id="sec4_16_6_95" class="NLM_sec NLM_sec_level_4"><div id="ac_i136" class="anchor-spacer"></div><h5 class="article-section__title" id="_i136"> 5-[4-(2-Ethoxyethoxy)-3-nitrophenyl]-4-methoxy-7-{[2-(trimethylsilyl)ethoxy]methyl}-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidine</h5><div class="NLM_p last">The reaction was performed according to the <a class="ref internalNav" href="#sec4_16_3" aria-label="General Method for Nucleophilic Aromatic Substitution">General Method for Nucleophilic Aromatic Substitution</a>. 5-(4-Fluoro-3-nitrophenyl)-4-methoxy-7-{[2-(trimethylsilyl)ethoxy]methyl}-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidine (<b>27</b>) (500 mg, 1.19 mmol) was alkylated with 2-ethoxyethan-1-ol (1.08 g, 11.94 mmol), and purification by flash column chromatography afforded the pure compound. Yield: 74% (431 mg, 0.88 mmol). <sup>1</sup>H NMR (700 MHz, CDCl<sub>3</sub>): δ 8.53 (s, 1H), 8.23 (d, <i>J</i> = 2.2 Hz, 1H), 7.78 (dd, <i>J</i> = 8.6, 2.2 Hz, 1H), 7.32 (s, 1H), 7.16 (d, <i>J</i> = 8.7 Hz, 1H), 5.67 (s, 2H), 4.32–4.29 (m, 2H), 4.13 (s, 3H), 3.87 (t, <i>J</i> = 4.9 Hz, 2H), 3.64 (q, <i>J</i> = 7.0 Hz, 2H), 3.58 (t, <i>J</i> = 8.4 Hz, 2H), 1.24 (t, <i>J</i> = 7.0 Hz, 3H), 0.94 (t, <i>J</i> = 8.4 Hz, 2H), −0.04 (s, 9H). <sup>13</sup>C NMR (176 MHz, CDCl<sub>3</sub>): δ 163.49, 152.73, 151.38, 151.33, 140.04, 133.98, 126.94, 125.99, 123.81, 115.72, 115.18, 103.34, 73.43, 70.01, 68.76, 67.32, 66.87, 54.28, 17.88, 15.32, −1.30. LC/MS: <i>m</i>/<i>z</i> calcd for [C<sub>23</sub>H<sub>32</sub>N<sub>4</sub>O<sub>6</sub>Si + H]<sup>+</sup>, 489.22; found, 489.05.</div></div><div id="sec4_16_6_96" class="NLM_sec NLM_sec_level_4"><div id="ac_i137" class="anchor-spacer"></div><h5 class="article-section__title" id="_i137"> 2-{4-{4-Methoxy-7-{[2-(trimethylsilyl)ethoxy]methyl}-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-5-yl}-2-nitrophenoxy}ethan-1-ol</h5><div class="NLM_p last">The reaction was performed according to the <a class="ref internalNav" href="#sec4_16_3" aria-label="General Method for Nucleophilic Aromatic Substitution">General Method for Nucleophilic Aromatic Substitution</a>. 5-(4-Fluoro-3-nitrophenyl)-4-methoxy-7-{[2-(trimethylsilyl)ethoxy]methyl}-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidine (<b>27</b>) (500 mg, 1.19 mmol) was alkylated with ethane-1,2-diol (738.6 mg, 11.9 mmol), and purification by flash column chromatography afforded the pure compound. Yield: 68% (363 mg, 0.8 mmol). <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.51 (s, 1H), 8.21 (d, <i>J</i> = 2.3 Hz, 1H), 7.94 (d, 1H), 7.93 (d, <i>J</i> = 2.5 Hz, 1H), 7.44 (d, <i>J</i> = 8.9 Hz, 1H), 5.62 (s, 2H), 4.92 (t, <i>J</i> = 5.4 Hz, 1H), 4.23 (t, <i>J</i> = 4.9 Hz, 2H), 4.04 (s, 3H), 3.75 (dd, <i>J</i> = 10.0, 5.2 Hz, 2H), 3.56 (t, <i>J</i> = 8.0 Hz, 2H), 0.85 (t, <i>J</i> = 8.0 Hz, 2H), −0.09 (s, 9H). <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>): δ 170.33, 162.59, 152.61, 151.19, 150.12, 139.46, 133.67, 126.11, 125.80, 124.40, 115.39, 113.60, 102.22, 71.37, 65.79, 59.34, 53.72, 17.09, −1.40. LC/MS: <i>m</i>/<i>z</i> calcd for [C<sub>21</sub>H<sub>28</sub>N<sub>4</sub>O<sub>6</sub>Si + H]<sup>+</sup>, 461.18; found, 461.02.</div></div><div id="sec4_16_6_97" class="NLM_sec NLM_sec_level_4"><div id="ac_i138" class="anchor-spacer"></div><h5 class="article-section__title" id="_i138"> 2-{4-{4-Methoxy-7-{[2-(trimethylsilyl)ethoxy]methyl}-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-5-yl}-2-nitrophenoxy}-<i>N,N</i>-dimethylethan-1-amine</h5><div class="NLM_p last">The reaction was performed according to the <a class="ref internalNav" href="#sec4_16_3" aria-label="General Method for Nucleophilic Aromatic Substitution">General Method for Nucleophilic Aromatic Substitution</a>. 5-(4-Fluoro-3-nitrophenyl)-4-methoxy-7-{[2-(trimethylsilyl)ethoxy]methyl}-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidine (<b>27</b>) (600 mg, 1.43 mmol) was alkylated with 2-(dimethylamino)ethan-1-ol (1.27 g, 14.3 mmol), and purification by flash column chromatography afforded the pure compound. Yield: 88% (610 mg, 1.25 mmol). <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.51 (s, 1H), 8.21 (d, <i>J</i> = 2.3 Hz, 1H), 7.95–7.92 (m, 2H), 7.45 (d, <i>J</i> = 7.0 Hz, 1H), 5.62 (s, 2H), 4.28 (t, <i>J</i> = 5.6 Hz, 2H), 4.04 (s, 3H), 3.56 (t, <i>J</i> = 8.0 Hz, 2H), 2.67 (t, <i>J</i> = 5.7 Hz, 2H), 2.23 (s, 6H), 0.85 (t, <i>J</i> = 8.0 Hz, 2H), −0.09 (s, 9H). <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>): δ 162.58, 152.59, 151.15, 149.84, 139.34, 133.70, 126.43, 126.12, 125.81, 124.38, 115.18, 113.57, 102.22, 68.09, 65.79, 57.34, 53.72, 45.60, 17.09, −1.41. LC/MS: <i>m</i>/<i>z</i> calcd for [C<sub>23</sub>H<sub>33</sub>N<sub>5</sub>O<sub>5</sub>Si + H]<sup>+</sup>, 488.22; found, 488.24.</div></div><div id="sec4_16_6_98" class="NLM_sec NLM_sec_level_4"><div id="ac_i139" class="anchor-spacer"></div><h5 class="article-section__title" id="_i139"> 5-{4-Methoxy-7-{[2-(trimethylsilyl)ethoxy]methyl}-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-5-yl}-2-(2-methoxyethoxy)aniline (<b>28a</b>)</h5><div class="NLM_p last">The reaction was performed according to the <a class="ref internalNav" href="#sec4_16_4" aria-label="General Method for Aromatic Nitro Reduction">General Method for Aromatic Nitro Reduction</a>. 4-Methoxy-5-[4-(2-methoxyethoxy)-3-nitrophenyl]-7-{[2-(trimethylsilyl)ethoxy]methyl}-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidine (125 mg, 0.26 mmol) was reduced and purified by flash column chromatography and yielded the desired product. Yield: 96% (112 mg, 0.25 mmol). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 8.47 (s, 1H), 7.21 (d, <i>J</i> = 2.1 Hz, 1H), 7.20 (s, 1H), 7.10 (dd, <i>J</i> = 8.3, 2.1 Hz, 1H), 6.88 (d, <i>J</i> = 8.4 Hz, 1H), 5.62 (s, 2H), 4.39 (br s, 2H), 4.20 (t, <i>J</i> = 4.8 Hz, 2H), 4.08 (s, 3H), 3.78 (t, <i>J</i> = 4.8 Hz, 2H), 3.55 (t, <i>J</i> = 8.4 Hz, 2H), 3.43 (s, 3H), 0.91 (t, <i>J</i> = 8.4 Hz, 2H), −0.06 (s, 9H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>): δ 163.58, 152.99, 151.28, 146.63, 133.71, 127.69, 123.25, 121.05, 117.70, 117.66, 112.78, 103.48, 73.12, 71.22, 68.50, 66.56, 59.19, 53.56, 17.84, −1.33. LC/MS: <i>m</i>/<i>z</i> calcd for [C<sub>22</sub>H<sub>32</sub>N<sub>4</sub>O<sub>4</sub>Si + H]<sup>+</sup>, 445.23; found, 445.14.</div></div><div id="sec4_16_6_99" class="NLM_sec NLM_sec_level_4"><div id="ac_i140" class="anchor-spacer"></div><h5 class="article-section__title" id="_i140"> 5-{4-Methoxy-7-{[2-(trimethylsilyl)ethoxy]methyl}-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-5-yl}-2-[(1-methoxypropan-2-yl)oxy]aniline (<b>28b</b>)</h5><div class="NLM_p last">The reaction was performed according to the <a class="ref internalNav" href="#sec4_16_4" aria-label="General Method for Aromatic Nitro Reduction">General Method for Aromatic Nitro Reduction</a>. 4-Methoxy-5-[4-(3-methoxypropoxy)-3-nitrophenyl]-7-{[2-(trimethylsilyl)ethoxy]methyl}-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidine (102 mg, 0.21 mmol) was reduced and purified by flash column chromatography and yielded the desired product. Yield: 92% (89 mg, 0.19 mmol). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 8.55 (s, 1H), 7.27 (s, 1H), 7.15 (d, <i>J</i> = 2.1 Hz, 1H), 7.08 (dd, <i>J</i> = 8.3, 2.1 Hz, 1H), 6.99 (d, <i>J</i> = 8.3 Hz, 1H), 5.69 (s, 2H), 4.61–4.43 (m, 1H), 4.20 (br s, 2H), 4.16 (s, 3H), 3.72–3.66 (m, 1H), 3.63 (t, <i>J</i> = 8.2 Hz, 2H), 3.60–3.55 (m, 1H), 3.50 (s, 3H), 1.43 (d, <i>J</i> = 6.3 Hz, 3H), 0.99 (t, <i>J</i> = 8.2 Hz, 2H), 0.02 (s, 9H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>): δ 163.51, 152.96, 151.19, 144.77, 137.29, 128.03, 123.14, 119.25, 117.93, 116.51, 115.76, 103.42, 76.13, 74.96, 73.05, 66.46, 59.32, 53.73, 17.78, 17.33, −1.37. LC/MS: <i>m</i>/<i>z</i> calcd for [C<sub>23</sub>H<sub>34</sub>N<sub>4</sub>O<sub>4</sub>Si + H]<sup>+</sup>, 459.24; found, 459.10.</div></div><div id="sec4_16_6_100" class="NLM_sec NLM_sec_level_4"><div id="ac_i141" class="anchor-spacer"></div><h5 class="article-section__title" id="_i141"> 2-(2-Ethoxyethoxy)-5-{4-methoxy-7-{[2-(trimethylsilyl)ethoxy]methyl}-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-5-yl}aniline (<b>28c</b>)</h5><div class="NLM_p last">The reaction was performed according to the <a class="ref internalNav" href="#sec4_16_4" aria-label="General Method for Aromatic Nitro Reduction">General Method for Aromatic Nitro Reduction</a>. 5-[4-(2-Ethoxyethoxy)-3-nitrophenyl]-4-methoxy-7-{[2-(trimethylsilyl)ethoxy]methyl}-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidine (118 mg, 0.24 mmol) was reduced and purified by flash column chromatography. This afforded the pure compound. Yield: 89% (98 mg, 0.21 mmol). <sup>1</sup>H NMR (400 MHz, CDCl<sub>3</sub>): δ 8.47 (s, 1H), 7.17 (s, 1H), 7.09 (d, <i>J</i> = 2.1 Hz, 1H), 7.03 (dd, <i>J</i> = 8.3, 2.1 Hz, 1H), 6.85 (d, <i>J</i> = 8.3 Hz, 1H), 5.60 (s, 2H), 4.50 (br s, 2H), 4.18 (t, <i>J</i> = 5.0 Hz, 2H), 4.06 (s, 3H), 3.80 (t, <i>J</i> = 5.0 Hz, 2H), 3.59 (q, <i>J</i> = 7.0 Hz, 2H), 3.54 (t, <i>J</i> = 8.0 Hz, 2H), 1.23 (t, <i>J</i> = 7.0 Hz, 3H), 0.90 (t, <i>J</i> = 8.4 Hz, 2H), −0.07 (s, 9H). <sup>13</sup>C NMR (101 MHz, CDCl<sub>3</sub>): δ 163.51, 152.92, 151.18, 146.10, 135.45, 127.54, 123.12, 119.82, 117.84, 116.66, 112.74, 103.43, 73.04, 69.12, 68.60, 66.81, 66.46, 53.73, 17.78, 15.26, −1.38. LC/MS: <i>m</i>/<i>z</i> calcd for [C<sub>23</sub>H<sub>34</sub>N<sub>4</sub>O<sub>4</sub>Si + H]<sup>+</sup>, 459.24; found, 459.13.</div></div><div id="sec4_16_6_101" class="NLM_sec NLM_sec_level_4"><div id="ac_i142" class="anchor-spacer"></div><h5 class="article-section__title" id="_i142"> 2-{2-Amino-4-{4-methoxy-7-{[2-(trimethylsilyl)ethoxy]methyl}-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-5-yl}phenoxy}ethan-1-ol (<b>28d</b>)</h5><div class="NLM_p last">The reaction was performed according to the <a class="ref internalNav" href="#sec4_16_4" aria-label="General Method for Aromatic Nitro Reduction">General Method for Aromatic Nitro Reduction</a>. 2-{4-{4-Methoxy-7-{[2-(trimethylsilyl)ethoxy]methyl}-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-5-yl}-2-nitrophenoxy}ethan-1-ol (363 mg, 0.79 mmol) was reduced and purified by flash column chromatography. This afforded the pure compound. Yield: 98% (331 mg, 0.77 mmol). <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.45 (s, 1H), 7.53 (s, 1H), 6.95 (d, <i>J</i> = 1.8 Hz, 1H), 6.80 (s, 1H), 6.79 (d, <i>J</i> = 1.9 Hz, 1H), 5.76 (s, 1H), 5.60 (s, 2H), 4.87 (s, 2H), 4.02 (s, 3H), 3.96 (t, <i>J</i> = 4.8 Hz, 2H), 3.73 (dd, <i>J</i> = 10.1, 5.4 Hz, 2H), 3.54 (t, <i>J</i> = 8.0 Hz, 2H), 0.84 (t, <i>J</i> = 8.0 Hz, 2H), −0.09 (s, 9H). <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>): δ 162.73, 152.39, 150.70, 144.78, 128.69, 126.52, 126.06, 124.22, 116.89, 116.46, 114.37, 111.81, 102.46, 70.19, 65.60, 59.86, 53.53, 17.10, −1.39. LC/MS: <i>m</i>/<i>z</i> calcd for [C<sub>21</sub>H<sub>30</sub>N<sub>4</sub>O<sub>4</sub>Si + H]<sup>+</sup>, 431.20; found, 431.13.</div></div><div id="sec4_16_6_102" class="NLM_sec NLM_sec_level_4"><div id="ac_i143" class="anchor-spacer"></div><h5 class="article-section__title" id="_i143"> 3-{2-Amino-4-{4-methoxy-7-{[2-(trimethylsilyl)ethoxy]methyl}-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-5-yl}phenoxy}propan-1-ol (<b>28e</b>)</h5><div class="NLM_p last">The reaction was performed according to the <a class="ref internalNav" href="#sec4_16_3" aria-label="General Method for Nucleophilic Aromatic Substitution">General Method for Nucleophilic Aromatic Substitution</a>, followed by the <a class="ref internalNav" href="#sec4_16_4" aria-label="General Method for Aromatic Nitro Reduction">General Method for Aromatic Nitro Reduction</a>. 5-(4-Fluoro-3-nitrophenyl)-4-methoxy-7-{[2-(trimethylsilyl)ethoxy]methyl}-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidine (482 mg, 1.15 mmol) was alkylated with propane-1,3-diol (875.1 mg, 11.5 mmol) and purified by flash column chromatography. Subsequently, 3-{4-{4-methoxy-7-{[2-(trimethylsilyl)ethoxy]methyl}-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-5-yl}-2-nitrophenoxy}propan-1-ol (361 mg, 0.76 mmol) was reduced and purified by flash column chromatography to yield the desired compound. Yield: 7% (44 mg, 0.1 mmol) over two steps. <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.45 (s, 1H), 7.53 (s, 1H), 6.96 (d, <i>J</i> = 1.7 Hz, 1H), 6.81 (d, <i>J</i> = 5.0 Hz, 1H), 6.80 (dd, <i>J</i> = 8.2, 1.8 Hz, 1H), 5.60 (s, 2H), 4.70 (s, 2H), 4.53 (t, <i>J</i> = 5.1 Hz, 1H), 4.04 (t, <i>J</i> = 3.2 Hz, 2H), 4.01 (s, 3H), 3.61 (dd, <i>J</i> = 11.3, 6.1 Hz, 2H), 3.54 (t, <i>J</i> = 8.0 Hz, 2H), 1.89 (app. p, <i>J</i> = 6.2 Hz, 2H), 0.84 (t, <i>J</i> = 8.0 Hz, 2H), −0.09 (s, 9H). <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>): δ 162.72, 152.39, 150.70, 144.79, 131.51, 128.78, 126.28, 124.19, 116.88, 114.46, 111.50, 102.46, 72.51, 65.58, 64.97, 57.51, 53.50, 32.28, 17.10, −1.39. LC/MS: <i>m</i>/<i>z</i> calcd for [C<sub>22</sub>H<sub>32</sub>N<sub>4</sub>O<sub>4</sub>Si + H]<sup>+</sup>, 445.22; found, 445.16.</div></div><div id="sec4_16_6_103" class="NLM_sec NLM_sec_level_4"><div id="ac_i144" class="anchor-spacer"></div><h5 class="article-section__title" id="_i144"> 2-[2-(Dimethylamino)ethoxy]-5-{4-methoxy-7-{[2-(trimethylsilyl)ethoxy]methyl}-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-5-yl}aniline (<b>28f</b>)</h5><div class="NLM_p last">2-{4-{4-Methoxy-7-{[2-(trimethylsilyl)ethoxy]methyl}-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-5-yl}-2-nitrophenoxy}-<i>N</i>,<i>N</i>-dimethylethan-1-amine (600 mg, 1.23 mmol) was dissolved in MeOH and reduced using a H-Cube Pro flow reactor (ThalesNano Inc.; Budapest, Hungary) with a 20% Pd(OH)<sub>2</sub> cartridge and a flow rate of 1 mL/min at 60 °C. After evaporation of the solvent, it was purified by flash column chromatography, and this afforded the pure compound. Yield: 74% (483 mg, 1.06 mmol). <sup>1</sup>H NMR (600 MHz, MeOD): δ 8.44 (s, 1H), 7.60 (d, <i>J</i> = 12.1 Hz, 1H), 7.57 (s, 1H), 7.52 (s, 1H), 7.22 (d, <i>J</i> = 8.5 Hz, 1H), 5.67 (s, 2H), 4.52 (t, <i>J</i> = 4.1 Hz, 2H), 4.09 (s, 3H), 3.68 (t, <i>J</i> = 4.8 Hz, 2H), 3.60 (t, <i>J</i> = 8.0 Hz, 2H), 3.03 (s, 6H), 0.89 (t, <i>J</i> = 8.0 Hz, 2H), −0.08 (s, 9H). <sup>13</sup>C NMR (151 MHz, MeOD): δ 164.81, 154.09, 152.35, 149.65, 129.87, 128.58, 125.84, 125.06, 123.76, 117.40, 113.64, 104.41, 74.30, 67.54, 63.68, 57.62, 54.32, 43.66, 18.55, −1.52. LC/MS: <i>m</i>/<i>z</i> calcd for [C<sub>23</sub>H<sub>35</sub>N<sub>5</sub>O<sub>3</sub>Si + H]<sup>+</sup>, 458.25; found, 458.20.</div></div><div id="sec4_16_6_104" class="NLM_sec NLM_sec_level_4"><div id="ac_i145" class="anchor-spacer"></div><h5 class="article-section__title" id="_i145"> <i>N</i>-{5-{4-Methoxy-7-{[2-(trimethylsilyl)ethoxy]methyl}-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-5-yl}-2-(2-methoxyethoxy)phenyl}acrylamide</h5><div class="NLM_p last">The reaction was performed according to the <a class="ref internalNav" href="#sec4_16_5" aria-label="General Method for Aniline Derivative Acylation">General Method for Aniline Derivative Acylation</a>. 5-{4-Methoxy-7-{[2-(trimethylsilyl)ethoxy]methyl}-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-5-yl}-2-(2-methoxyethoxy)aniline (<b>28a</b>) (49 mg, 0.11 mmol) was acylated and purified by flash column chromatography, and this yielded the desired product. Yield: 89% (49 mg, 0.1 mmol). <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): δ 8.91 (s, 1H), 8.55 (s, 1H), 8.38 (s, 1H), 7.43 (dd, <i>J</i> = 8.4, 2.1 Hz, 1H), 7.38 (s, 1H), 7.05 (d, <i>J</i> = 8.4 Hz, 1H), 6.48 (d, <i>J</i> = 16.9 Hz, 1H), 6.36 (dd, <i>J</i> = 16.9, 10.2 Hz, 1H), 5.81 (dd, 10.2, 1.0 Hz, 1H), 5.69 (s, 2H), 4.28 (dd, <i>J</i> = 5.3, 3.7 Hz, 2H), 4.19 (s, 3H), 3.80 (dd, <i>J</i> = 5.3, 3.7 Hz, 2H), 3.61 (t, <i>J</i> = 8.4 Hz, 2H), 3.52 (s, 3H), 0.98 (t, <i>J</i> = 8.4 Hz, 2H), 0.00 (s, 9H). <sup>13</sup>C NMR (151 MHz, CDCl<sub>3</sub>): δ 163.60, 163.46, 152.73, 150.97, 146.38, 131.86, 129.27, 128.37, 127.21, 124.42, 123.85, 120.62, 117.75, 114.22, 103.53, 73.27, 70.98, 70.07, 66.61, 59.18, 54.07, 17.84, −1.32. LC/MS: <i>m</i>/<i>z</i> calcd for [C<sub>25</sub>H<sub>34</sub>N<sub>4</sub>O<sub>5</sub>Si + H]<sup>+</sup>, 499.24; found, 499.30.</div></div><div id="sec4_16_6_105" class="NLM_sec NLM_sec_level_4"><div id="ac_i146" class="anchor-spacer"></div><h5 class="article-section__title" id="_i146"> <i>N</i>-{5-{4-Methoxy-7-{[2-(trimethylsilyl)ethoxy]methyl}-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-5-yl}-2-[(1-methoxypropan-2-yl)oxy]phenyl}acrylamide</h5><div class="NLM_p last">The reaction was performed according to the <a class="ref internalNav" href="#sec4_16_5" aria-label="General Method for Aniline Derivative Acylation">General Method for Aniline Derivative Acylation</a>. 5-{4-Methoxy-7-{[2-(trimethylsilyl)ethoxy]methyl}-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-5-yl}-2-(3-methoxypropoxy)aniline (<b>28b</b>) (55 mg, 0.12 mmol) was acylated and purified by flash column chromatography. This afforded the pure compound. Yield: 88% (54 mg, 0.11 mmol). <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): δ 8.96 (s, 1H), 8.76 (s, 1H), 8.56 (s, 1H), 7.42 (dd, <i>J</i> = 8.3, 2.1 Hz, 1H), 7.39 (s, 1H), 7.11 (d, <i>J</i> = 8.4 Hz, 1H), 6.49 (d, <i>J</i> = 16.9 Hz, 1H), 6.36 (dd, <i>J</i> = 16.9, 10.2 Hz, 1H), 5.80 (dd, <i>J</i> = 10.2, 1.2 Hz, 1H), 5.70 (s, 2H), 4.38–4.34 (m, 1H), 4.20 (s, 3H), 3.68–3.63 (m, 1H), 3.61 (dtd, <i>J</i> = 13.3, 10.3, 5.1 Hz, 2H), 3.59–3.56 (m, 1H), 3.52 (s, 3H), 1.44 (d, <i>J</i> = 6.4 Hz, 3H), 0.98 (t, <i>J</i> = 8.4 Hz, 2H), 0.00 (s, 9H). <sup>13</sup>C NMR (151 MHz, CDCl<sub>3</sub>): δ 163.59, 163.55, 152.71, 150.94, 145.68, 131.82, 130.78, 129.21, 127.17, 124.25, 123.96, 120.74, 118.20, 117.79, 103.51, 77.66, 76.02, 73.29, 66.60, 59.35, 54.10, 17.84, 17.14, −1.32. LC/MS: <i>m</i>/<i>z</i> calcd for [C<sub>26</sub>H<sub>36</sub>N<sub>4</sub>O<sub>5</sub>Si + H]<sup>+</sup>, 513.25; found, 513.29.</div></div><div id="sec4_16_6_106" class="NLM_sec NLM_sec_level_4"><div id="ac_i147" class="anchor-spacer"></div><h5 class="article-section__title" id="_i147"> <i>N</i>-{2-(2-Ethoxyethoxy)-5-{4-methoxy-7-{[2-(trimethylsilyl)ethoxy]methyl}-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-5-yl}phenyl}acrylamide</h5><div class="NLM_p last">The reaction was performed according to the <a class="ref internalNav" href="#sec4_16_5" aria-label="General Method for Aniline Derivative Acylation">General Method for Aniline Derivative Acylation</a>. 2-(2-Ethoxyethoxy)-5-{4-methoxy-7-{[2-(trimethylsilyl)ethoxy]methyl}-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-5-yl}aniline (<b>28c</b>) (56 mg, 0.12 mmol) was acylated and purified by flash column chromatography. This afforded the pure compound. Yield: 91% (56 mg, 0.11 mmol). <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): δ 8.90 (s, 1H), 8.54 (s, 1H), 8.30 (s, 1H), 7.43 (dd, <i>J</i> = 8.4, 2.1 Hz, 1H), 7.36 (s, 1H), 7.04 (d, <i>J</i> = 8.4 Hz, 1H), 6.46 (d, <i>J</i> = 16.9 Hz, 1H), 6.36 (dd, <i>J</i> = 16.9, 10.1 Hz, 1H), 5.80 (d, <i>J</i> = 10.5 Hz, 1H), 5.68 (s, 2H), 4.29 (t, <i>J</i> = 4.5 Hz, 2H), 4.17 (s, 3H), 3.84 (t, <i>J</i> = 5.6 Hz, 2H), 3.66 (q, <i>J</i> = 7.1 Hz, 2H), 3.60 (t, <i>J</i> = 8.0 Hz, 2H), 1.31 (t, <i>J</i> = 7.1 Hz, 3H), 0.97 (t, <i>J</i> = 8.4 Hz, 2H), −0.01 (s, 9H). <sup>13</sup>C NMR (151 MHz, CDCl<sub>3</sub>): δ 167.94, 164.95, 155.94, 154.27, 152.52, 134.50, 133.21, 129.51, 128.50, 125.04, 124.57, 121.92, 118.98, 114.97, 104.84, 86.93, 74.50, 71.20, 70.13, 68.21, 55.19, 19.15, 16.63, 0.00. LC/MS: <i>m</i>/<i>z</i> calcd for [C<sub>26</sub>H<sub>36</sub>N<sub>4</sub>O<sub>5</sub>Si + H]<sup>+</sup>, 513.25; found, 513.30.</div></div><div id="sec4_16_6_107" class="NLM_sec NLM_sec_level_4"><div id="ac_i148" class="anchor-spacer"></div><h5 class="article-section__title" id="_i148"> <i>N</i>-{2-(2-Hydroxyethoxy)-5-{4-methoxy-7-{[2-(trimethylsilyl)ethoxy]methyl}-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-5-yl}phenyl}acrylamide</h5><div class="NLM_p last">The reaction was performed according to the <a class="ref internalNav" href="#sec4_16_5" aria-label="General Method for Aniline Derivative Acylation">General Method for Aniline Derivative Acylation</a>. 2-{2-Amino-4-{4-methoxy-7-{[2-(trimethylsilyl)ethoxy]methyl}-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-5-yl}phenoxy}ethan-1-ol (<b>28d</b>) (285 mg, 0.66 mmol) was acylated and purified by flash column chromatography. This afforded the pure compound. Yield: 75% (240 mg, 0.49 mmol). <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.23 (s, 1H), 8.48 (s, <i>J</i> = 5.8 Hz, 1H), 7.66 (s, 1H), 7.37 (dd, <i>J</i> = 8.5, 2.2 Hz, 1H), 7.11 (d, <i>J</i> = 8.5 Hz, 1H), 6.65 (dd, <i>J</i> = 16.9, 10.2 Hz, 1H), 6.27 (dd, <i>J</i> = 17.0, 1.8 Hz, 1H), 5.78 (dd, <i>J</i> = 10.2, 1.8 Hz, 1H), 5.62 (s, 2H), 5.06 (t, <i>J</i> = 6.1 Hz, 1H), 4.09 (t, <i>J</i> = 4.7 Hz, 2H), 4.03 (s, 3H), 3.78 (dd, <i>J</i> = 10.2, 5.2 Hz, 2H), 3.56 (t, <i>J</i> = 8.0 Hz, 2H), 0.85 (t, <i>J</i> = 8.0 Hz, 2H), −0.09 (s, 9H). <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>): δ 163.10, 162.76, 152.48, 150.92, 132.03, 128.90, 127.43, 126.79, 126.22, 125.46, 124.72, 124.39, 116.00, 112.63, 102.36, 70.98, 65.61, 59.49, 54.89, 53.49, 17.03, −1.44. LC/MS: <i>m</i>/<i>z</i> calcd for [C<sub>24</sub>H<sub>32</sub>N<sub>4</sub>O<sub>5</sub>Si + H]<sup>+</sup>, 485.21; found, 485.13.</div></div><div id="sec4_16_6_108" class="NLM_sec NLM_sec_level_4"><div id="ac_i149" class="anchor-spacer"></div><h5 class="article-section__title" id="_i149"> <i>N</i>-{2-(3-Hydroxypropoxy)-5-{4-methoxy-7-{[2-(trimethylsilyl)ethoxy]methyl}-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-5-yl}phenyl}acrylamide</h5><div class="NLM_p last">The reaction was performed according to the <a class="ref internalNav" href="#sec4_16_5" aria-label="General Method for Aniline Derivative Acylation">General Method for Aniline Derivative Acylation</a>. 3-{2-Amino-4-{4-methoxy-7-{[2-(trimethylsilyl)ethoxy]methyl}-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-5-yl}phenoxy}propan-1-ol (<b>28e</b>) (21 mg, 0.05 mmol) was acylated and purified by flash column chromatography. This afforded the pure compound. Yield: 40% (10 mg, 0.02 mmol). <sup>1</sup>H NMR (600 MHz, MeOD): δ 8.42 (s, 1H), 7.65 (dd, <i>J</i> = 7.1, 3.0 Hz, 1H), 7.57 (dd, <i>J</i> = 7.4, 2.8 Hz, 1H), 7.47 (s, 1H), 7.09 (d, <i>J</i> = 8.5, 3.4 Hz, 1H), 6.58 (dd, <i>J</i> = 17.0, 10.3 Hz, 1H), 6.38 (dd, <i>J</i> = 17.0, 1.5 Hz, 1H), 5.79 (dd, <i>J</i> = 10.3, 1.4 Hz, 1H), 5.66 (s, 2H), 4.25 (d, <i>J</i> = 6.1 Hz, 2H), 4.10 (s, 3H), 3.81 (d, <i>J</i> = 6.0 Hz, 2H), 3.61 (t, <i>J</i> = 8.0 Hz, 2H), 2.08 (t, <i>J</i> = 6.0 Hz, 2H), 0.90 (t, <i>J</i> = 8.0 Hz, 2H), −0.06 ppm (s, 9H). <sup>13</sup>C NMR (151 MHz, MeOD): δ 164.97, 153.93, 152.06, 133.78, 133.12, 133.05, 130.02, 129.94, 129.79, 127.84 (d, <i>J</i> = 4.6 Hz), 127.65, 126.92, 125.44, 113.05, 104.64, 99.22, 74.26, 67.44, 59.93, 54.24, 33.06, 18.55, −1.41. LC/MS: <i>m</i>/<i>z</i> calcd for [C<sub>25</sub>H<sub>34</sub>N<sub>4</sub>O<sub>5</sub>Si + H]<sup>+</sup>, 499.23; found, 499.08.</div></div><div id="sec4_16_6_109" class="NLM_sec NLM_sec_level_4"><div id="ac_i150" class="anchor-spacer"></div><h5 class="article-section__title" id="_i150"> <i>N</i>-{2-[2-(Dimethylamino)ethoxy]-5-{4-methoxy-7-{[2-(trimethylsilyl)ethoxy]methyl}-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-5-yl}phenyl}acrylamide</h5><div class="NLM_p last">The reaction was performed according to the <a class="ref internalNav" href="#sec4_16_5" aria-label="General Method for Aniline Derivative Acylation">General Method for Aniline Derivative Acylation</a>. 2-[2-(Dimethylamino)ethoxy]-5-{4-methoxy-7-{[2-(trimethylsilyl)ethoxy]methyl}-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-5-yl}aniline (<b>28f</b>) (460 mg, 1.00 mmol) was acylated and purified by flash column chromatography. This afforded the pure compound. Yield: 44% (225 mg, 0.44 mmol). <sup>1</sup>H NMR (600 MHz, MeOD): δ 8.58 (d, <i>J</i> = 1.9 Hz, 1H), 8.43 (s, 1H), 7.49 (s, 1H), 7.42 (dd, <i>J</i> = 8.5, 2.2 Hz, 1H), 7.08 (d, <i>J</i> = 8.5 Hz, 1H), 6.64 (dd, <i>J</i> = 17.0, 10.3 Hz, 1H), 6.40 (dd, <i>J</i> = 17.0, 1.6 Hz, 1H), 5.79 (dd, <i>J</i> = 10.3, 1.6 Hz, 1H), 5.67 (s, 2H), 4.21 (t, <i>J</i> = 5.1 Hz, 2H), 4.11 (s, 3H), 3.61 (t, <i>J</i> = 8.0 Hz, 2H), 2.82 (t, <i>J</i> = 5.1 Hz, 2H), 2.41 (s, 6H), 0.90 (t, <i>J</i> = 8.0 Hz, 2H), −0.06 (s, 9H). <sup>13</sup>C NMR (151 MHz, MeOD): δ 166.25, 164.99, 153.94, 152.07, 149.26, 132.73, 128.33, 128.06, 127.61, 126.47, 125.48, 123.77, 118.77, 112.48, 104.66, 74.27, 67.44, 66.29, 59.00, 54.24, 45.26, 18.55, −1.41. LC/MS: <i>m</i>/<i>z</i> calcd for [C<sub>26</sub>H<sub>37</sub>N<sub>5</sub>O<sub>4</sub>Si + H]<sup>+</sup>, 512.26; found, 512.20.</div></div><div id="sec4_16_6_110" class="NLM_sec NLM_sec_level_4"><div id="ac_i151" class="anchor-spacer"></div><h5 class="article-section__title" id="_i151"> <i>N</i>-[5-(4-Methoxy-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-5-yl)-2-(2-methoxyethoxy)phenyl]acrylamide (<b>25a</b>)</h5><div class="NLM_p last">The reaction was performed according to the <a class="ref internalNav" href="#sec4_16_6" aria-label="General Method for SEM Deprotection">General Method for SEM Deprotection</a>. <i>N</i>-{5-{4-Methoxy-7-{[2-(trimethylsilyl)ethoxy]methyl}-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-5-yl}-2-(2-methoxyethoxy)phenyl}acrylamide (40 mg, 0.08 mmol) was deprotected and purified by flash column chromatography. This afforded the pure compound. Yield: 81% (24 mg, 0.07 mmol). <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): δ 11.26 (s, 1H), 8.85 (s, 1H), 8.50 (s, 1H), 8.34 (s, 1H), 7.37 (dd, <i>J</i> = 8.3, 2.0 Hz, 1H), 7.34 (s, 1H), 7.00 (d, <i>J</i> = 8.4 Hz, 1H), 6.44 (d, <i>J</i> = 16.9 Hz, 1H), 6.32 (dd, <i>J</i> = 16.9, 10.3 Hz, 1H), 5.76 (d, <i>J</i> = 10.3 Hz, 1H), 4.23 (t, <i>J</i> = 4.5 Hz, 2H), 4.15 (s, 3H), 3.75 (t, <i>J</i> = 4.5 Hz, 2H), 3.47 (s, 3H). <sup>13</sup>C NMR (151 MHz, CDCl<sub>3</sub>): δ 163.97, 163.53, 151.88, 150.00, 146.34, 131.87, 129.23, 128.56, 127.29, 124.48, 121.26, 120.72, 117.49, 114.26, 103.58, 71.00, 70.07, 59.20, 54.13. HRMS: <i>m</i>/<i>z</i> calcd for [C<sub>19</sub>H<sub>20</sub>N<sub>4</sub>O<sub>4</sub> + H]<sup>+</sup>, 369.15573; found, 369.15627, calcd for [C<sub>19</sub>H<sub>20</sub>N<sub>4</sub>O<sub>4</sub> + Na]<sup>+</sup>, 391.1377; found, 391.1381.</div></div><div id="sec4_16_6_111" class="NLM_sec NLM_sec_level_4"><div id="ac_i152" class="anchor-spacer"></div><h5 class="article-section__title" id="_i152"> <i>N</i>-{5-(4-Methoxy-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-5-yl)-2-[(1-methoxypropan-2-yl)oxy]phenyl}acrylamide (<b>25b</b>)</h5><div class="NLM_p last">The reaction was performed according to the <a class="ref internalNav" href="#sec4_16_6" aria-label="General Method for SEM Deprotection">General Method for SEM Deprotection</a>. <i>N</i>-{5-{4-Methoxy-7-{[2-(trimethylsilyl)ethoxy]methyl}-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-5-yl}-2-(3-methoxypropoxy)phenyl}acrylamide (42 mg, 0.09 mmol) was deprotected and purified by flash column chromatography, and this afforded the pure compound. Yield: 78% (27 mg, 0.07 mmol). <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): δ 11.46 (s, 1H), 8.90 (s, 1H), 8.72 (s, 1H), 8.50 (s, 1H), 7.37 (dd, <i>J</i> = 8.5, 2.1 Hz, 1H), 7.36 (s, 1H), 7.05 (d, <i>J</i> = 8.4 Hz, 1H), 6.44 (dd, <i>J</i> = 16.6, 9.1 Hz, 1H), 6.31 (dd, <i>J</i> = 16.9, 10.3 Hz, 1H), 5.75 (d, <i>J</i> = 10.3 Hz, 1H), 4.32–4.29 (m, 1H), 4.15 (s, 3H), 3.62–3.58 (m, 1H), 3.54–3.51 (m, 1H), 3.47 (s, 3H), 1.39 (d, <i>J</i> = 6.3 Hz, 3H). <sup>13</sup>C NMR (151 MHz, CDCl<sub>3</sub>): δ 163.91, 163.62, 152.19, 150.12, 145.59, 131.82, 130.74, 129.51, 127.24, 124.32, 121.37, 120.79, 118.25, 117.43, 103.52, 77.67, 76.03, 59.36, 54.06, 17.15. HRMS: <i>m</i>/<i>z</i> calcd for [C<sub>20</sub>H<sub>22</sub>N<sub>4</sub>O<sub>4</sub> + H]<sup>+</sup>, 383.1714; found, 383.1719, calcd for [C<sub>20</sub>H<sub>22</sub>N<sub>4</sub>O<sub>4</sub> + Na]<sup>+</sup>, 405.1533; found, 405.1537.</div></div><div id="sec4_16_6_112" class="NLM_sec NLM_sec_level_4"><div id="ac_i153" class="anchor-spacer"></div><h5 class="article-section__title" id="_i153"> <i>N</i>-[2-(2-Ethoxyethoxy)-5-(4-methoxy-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-5-yl)phenyl]acrylamide (<b>25c</b>)</h5><div class="NLM_p last">The reaction was performed according to the <a class="ref internalNav" href="#sec4_16_6" aria-label="General Method for SEM Deprotection">General Method for SEM Deprotection</a>. <i>N</i>-{2-(2-Ethoxyethoxy)-5-{4-methoxy-7-{[2-(trimethylsilyl)ethoxy]methyl}-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-5-yl}phenyl}acrylamide (46 mg, 0.10 mmol) was deprotected and purified by flash column chromatography to afford the pure compound. Yield: 79% (30 mg, 0.08 mmol). <sup>1</sup>H NMR (600 MHz, CDCl<sub>3</sub>): δ 11.12 (s, 1H), 8.84 (s, 1H), 8.50 (s, 1H), 8.26 (s, 1H), 7.36 (dd, <i>J</i> = 8.4, 1.9 Hz, 1H), 7.33 (s, 1H), 6.99 (d, <i>J</i> = 8.4 Hz, 1H), 6.43 (d, <i>J</i> = 16.9 Hz, 1H), 6.33 (dd, <i>J</i> = 16.9, 10.2 Hz, 1H), 5.76 (d, <i>J</i> = 10.2 Hz, 1H), 4.25 (t, <i>J</i> = 4.5 Hz, 2H), 4.16 (s, 3H), 3.80 (t, <i>J</i> = 4.8 Hz, 2H), 3.62 (q, <i>J</i> = 7.0 Hz, 2H), 1.27 (t, <i>J</i> = 7.0 Hz, 3H). HRMS: <i>m</i>/<i>z</i> calcd for [C<sub>20</sub>H<sub>22</sub>N<sub>4</sub>O<sub>4</sub> + H]<sup>+</sup>, 383.1714; found, 383.1717, calcd for [C<sub>20</sub>H<sub>22</sub>N<sub>4</sub>O<sub>4</sub> + Na]<sup>+</sup>, 405.1533; found, 405.1535.</div></div><div id="sec4_16_6_113" class="NLM_sec NLM_sec_level_4"><div id="ac_i154" class="anchor-spacer"></div><h5 class="article-section__title" id="_i154"> <i>N</i>-[2-(2-Hydroxyethoxy)-5-(4-methoxy-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-5-yl)phenyl]acrylamide (<b>25d</b>)</h5><div class="NLM_p last">The reaction was performed according to the <a class="ref internalNav" href="#sec4_16_6" aria-label="General Method for SEM Deprotection">General Method for SEM Deprotection</a>. <i>N</i>-{2-(2-Hydroxyethoxy)-5-{4-methoxy-7-{[2-(trimethylsilyl)ethoxy]methyl}-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-5-yl}phenyl}acrylamide (230 mg, 0.65 mmol) was deprotected and purified by flash column chromatography to afford the pure compound. Yield: 11% (18 mg, 0.05 mmol). <sup>1</sup>H NMR (700 MHz, MeOD): δ 8.56 (s, 1H), 8.37 (s, 1H), 7.43 (d, <i>J</i> = 8.4 Hz, 1H), 7.36 (s, 1H), 7.08 (d, <i>J</i> = 8.4 Hz, 1H), 6.62 (dd, <i>J</i> = 17.0, 10.3 Hz, 1H), 6.41 (dd, <i>J</i> = 17.0, 1.3 Hz, 1H), 5.83–5.80 (m, 1H), 4.19 (t, <i>J</i> = 4.2 Hz, 2H), 4.11 (s, 3H), 3.97 (t, <i>J</i> = 4.3 Hz, 2H). <sup>13</sup>C NMR (176 MHz, MeOD): δ 166.13, 164.94, 153.86, 151.60, 149.11, 132.70, 128.82, 128.30, 127.70, 126.50, 123.55, 122.44, 118.24, 112.96, 104.16, 71.73, 61.54, 54.09. HRMS: <i>m</i>/<i>z</i> calcd for [C<sub>18</sub>H<sub>18</sub>N<sub>4</sub>O<sub>4</sub> + H<sup>+</sup>], 355.1401; found, 355.1401, calcd for [C<sub>18</sub>H<sub>18</sub>N<sub>4</sub>O<sub>4</sub> + Na], 377.1220; found, 377.1220.</div></div><div id="sec4_16_6_114" class="NLM_sec NLM_sec_level_4"><div id="ac_i155" class="anchor-spacer"></div><h5 class="article-section__title" id="_i155"> <i>N</i>-[2-(3-Hydroxypropoxy)-5-(4-methoxy-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-5-yl)phenyl]acrylamide (<b>25e</b>)</h5><div class="NLM_p last">The reaction was performed according to the <a class="ref internalNav" href="#sec4_16_6" aria-label="General Method for SEM Deprotection">General Method for SEM Deprotection</a>. <i>N</i>-{2-(3-Hydroxypropoxy)-5-{4-methoxy-7-{[2-(trimethylsilyl)ethoxy]methyl}-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-5-yl}phenyl}acrylamide (9 mg, 0.02 mmol) was deprotected and purified by flash column chromatography, and this afforded the pure compound. Yield: 45% (3 mg, 8.14 μmol). <sup>1</sup>H NMR (700 MHz, MeOD): δ 8.40 (s, 1H), 8.37 (s, 1H), 7.46 (dd, <i>J</i> = 8.3, 1.6 Hz, 1H), 7.36 (s, 1H), 7.10 (d, <i>J</i> = 8.5 Hz, 1H), 6.59 (dd, <i>J</i> = 16.9, 10.3 Hz, 1H), 6.39 (dd, <i>J</i> = 17.0, 1.3 Hz, 1H), 5.81 (d, <i>J</i> = 10.3 Hz, 1H), 4.26 (t, <i>J</i> = 6.1 Hz, 2H), 3.82 (t, <i>J</i> = 6.0 Hz, 2H), 2.07–2.11 (m, 2H). <sup>13</sup>C NMR (176 MHz, MeOD): δ 166.20, 164.94, 153.86, 151.63, 149.95, 132.68, 128.53, 127.95, 127.65, 126.94, 124.30, 122.39, 118.22, 113.10, 104.18, 67.50, 59.96, 54.09, 33.08. HRMS: <i>m</i>/<i>z</i> calcd for [C<sub>19</sub>H<sub>20</sub>N<sub>4</sub>O<sub>4</sub> + H<sup>+</sup>], 369.1558; found, 369.1558, calcd for [C<sub>19</sub>H<sub>20</sub>N<sub>4</sub>O<sub>4</sub> + Na], 391.1377; found, 391.1377.</div></div><div id="sec4_16_6_115" class="NLM_sec NLM_sec_level_4"><div id="ac_i156" class="anchor-spacer"></div><h5 class="article-section__title" id="_i156"> <i>N</i>-{2-[2-(Dimethylamino)ethoxy]-5-(4-methoxy-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-5-yl)phenyl}acrylamide (<b>25f</b>)</h5><div class="NLM_p last">The reaction was performed according to the <a class="ref internalNav" href="#sec4_16_6" aria-label="General Method for SEM Deprotection">General Method for SEM Deprotection</a>. <i>N</i>-{2-[2-(Dimethylamino)ethoxy]-5-{4-methoxy-7-{[2-(trimethylsilyl)ethoxy]methyl}-7<i>H</i>-pyrrolo[2,3-<i>d</i>]pyrimidin-5-yl}phenyl}acrylamide (100 mg, 0.19 mmol) was deprotected and purified by flash column chromatography, and this afforded the pure compound. Yield: 92% (69 mg, 0.18 mmol). <sup>1</sup>H NMR (500 MHz, MeOD): δ 8.34 (s, 1H), 8.10 (s, 1H), 7.49 (d, <i>J</i> = 7.7 Hz, 1H), 7.34 (s, 1H), 7.13 (d, <i>J</i> = 8.6 Hz, 1H), 6.54 (dd, <i>J</i> = 16.9, 10.2 Hz, 1H), 6.39 (dd, <i>J</i> = 16.9, 1.5 Hz, 1H), 5.80 (dd, <i>J</i> = 10.3, 1.2 Hz, 1H), 4.46 (t, <i>J</i> = 4.8 Hz, 2H), 3.61 (t, <i>J</i> = 4.8 Hz, 2H), 2.99 (s, 6H). <sup>13</sup>C NMR (126 MHz, MeOD): δ 166.61, 164.91, 153.56, 151.51, 149.47, 132.20, 129.79, 128.39, 127.88, 127.27, 125.80, 122.77, 117.65, 113.22, 104.14, 63.59, 57.65, 54.24, 43.83. HRMS: <i>m</i>/<i>z</i> calcd for [C<sub>20</sub>H<sub>23</sub>N<sub>5</sub>O<sub>3</sub> + H<sup>+</sup>], 382.1875; found, 382.1874.</div></div></div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i157"><a href="/doi/suppl/10.1021/acs.jmedchem.0c00870" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_84024" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_84024" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a href="/doi/10.1021/acs.jmedchem.0c00870?goto=supporting-info" class="ext-link">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c00870</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Sequence alignment of EGFR and Her2 kinases; <i>Z</i>′ values obtained within a focused screen; analysis of activity data obtained within a focused screen; Western blot analysis of reference compounds in H1781 cells; cocrystal structures of covalent pyrrolopyrimidines in complex with mutant EGFR-T790M/V948R; mass spectrometry-based analysis of covalent bond formation; hit validation for representative inhibitors; IC50 and EC50 evaluation of pyrrolopyrimidine-based inhibitors; data collection and refinement statistics of complex crystal structures; and LC/MS spectra of key compounds <b>14b</b>, <b>14h</b>, <b>18d</b>, and <b>19</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00870/suppl_file/jm0c00870_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Molecular formula strings of <b>1</b>–<b>5, poziotinib</b>, <b>TAK-285</b>, <b>gefitinib</b>, <b>osimertinib</b>, <b>7, 14, 18–21, 25</b>, <b>i–viii</b>, and so forth (<a href="/doi/suppl/10.1021/acs.jmedchem.0c00870/suppl_file/jm0c00870_si_002.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00870/suppl_file/jm0c00870_si_001.pdf">jm0c00870_si_001.pdf (0.99 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00870/suppl_file/jm0c00870_si_002.csv">jm0c00870_si_002.csv (16.09 kb)</a></li></ul></div></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">PDB IDs are <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TFU">6TFU</a> (<b>14d</b>/EGFR-T790M/V948R), <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TFV">6TFV</a> (<b>18b</b>/EGFR-T790M/V948R), <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TFY">6TFY</a> (<b>18c</b>/EGFR-T790M/V948R), <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TFW">6TFW</a> (<b>18d</b>/EGFR-T790M/V948R), <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TFZ">6TFZ</a> (<b>19</b>/EGFR-T790M/V948R), and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TG0">6TG0</a> (<b>21a</b>/EGFR-T790M/V948R), <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TG1">6TG1</a> (<b>21b</b>/EGFR-T790M/V948R).</p><div class="testing" data-doi="10.1021/acs.jmedchem.0c00870" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_82654" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_82654" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Daniel Rauh</span> - <span class="hlFld-Affiliation affiliation">Faculty
of Chemistry and Chemical Biology, TU Dortmund
University, Otto-Hahn-Strasse 4a, 44227 Dortmund, Germany</span>; 
    <span class="hlFld-Affiliation affiliation">Drug
Discovery Hub Dortmund (DDHD) am Zentrum für Integrierte Wirkstoffforschung
(ZIW), 44227 Dortmund, Germany</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#d4b0b5babdb1b8faa6b5a1bc94a0a1f9b0bba6a0b9a1bab0fab0b1"><span class="__cf_email__" data-cfemail="5531343b3c30397b2734203d15212078313a272138203b317b3130">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jonas Lategahn</span> - <span class="hlFld-Affiliation affiliation">Faculty
of Chemistry and Chemical Biology, TU Dortmund
University, Otto-Hahn-Strasse 4a, 44227 Dortmund, Germany</span>; 
    <span class="hlFld-Affiliation affiliation">Drug
Discovery Hub Dortmund (DDHD) am Zentrum für Integrierte Wirkstoffforschung
(ZIW), 44227 Dortmund, Germany</span>; 
    <span>Present Address:
                        PearlRiver Bio GmbH, Otto-Hahn-Strasse 15, 44227 Dortmund, Germany</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-5993-7082" title="Orcid link">http://orcid.org/0000-0001-5993-7082</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Julia Hardick</span> - <span class="hlFld-Affiliation affiliation">Faculty
of Chemistry and Chemical Biology, TU Dortmund
University, Otto-Hahn-Strasse 4a, 44227 Dortmund, Germany</span>; 
    <span class="hlFld-Affiliation affiliation">Drug
Discovery Hub Dortmund (DDHD) am Zentrum für Integrierte Wirkstoffforschung
(ZIW), 44227 Dortmund, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Tobias Grabe</span> - <span class="hlFld-Affiliation affiliation">Faculty
of Chemistry and Chemical Biology, TU Dortmund
University, Otto-Hahn-Strasse 4a, 44227 Dortmund, Germany</span>; 
    <span class="hlFld-Affiliation affiliation">Drug
Discovery Hub Dortmund (DDHD) am Zentrum für Integrierte Wirkstoffforschung
(ZIW), 44227 Dortmund, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Janina Niggenaber</span> - <span class="hlFld-Affiliation affiliation">Faculty
of Chemistry and Chemical Biology, TU Dortmund
University, Otto-Hahn-Strasse 4a, 44227 Dortmund, Germany</span>; 
    <span class="hlFld-Affiliation affiliation">Drug
Discovery Hub Dortmund (DDHD) am Zentrum für Integrierte Wirkstoffforschung
(ZIW), 44227 Dortmund, Germany</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-2942-263X" title="Orcid link">http://orcid.org/0000-0003-2942-263X</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Kirujan Jeyakumar</span> - <span class="hlFld-Affiliation affiliation">Faculty
of Chemistry and Chemical Biology, TU Dortmund
University, Otto-Hahn-Strasse 4a, 44227 Dortmund, Germany</span>; 
    <span class="hlFld-Affiliation affiliation">Drug
Discovery Hub Dortmund (DDHD) am Zentrum für Integrierte Wirkstoffforschung
(ZIW), 44227 Dortmund, Germany</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-0184-6775" title="Orcid link">http://orcid.org/0000-0002-0184-6775</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Marina Keul</span> - <span class="hlFld-Affiliation affiliation">Faculty
of Chemistry and Chemical Biology, TU Dortmund
University, Otto-Hahn-Strasse 4a, 44227 Dortmund, Germany</span>; 
    <span class="hlFld-Affiliation affiliation">Drug
Discovery Hub Dortmund (DDHD) am Zentrum für Integrierte Wirkstoffforschung
(ZIW), 44227 Dortmund, Germany</span>; 
    <span>Present Address:
                        Evotec SE, Manfred Eigen Campus, Essener Bogen 7, 22419 Hamburg, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Hannah L. Tumbrink</span> - <span class="hlFld-Affiliation affiliation">Faculty
of Chemistry and Chemical Biology, TU Dortmund
University, Otto-Hahn-Strasse 4a, 44227 Dortmund, Germany</span>; 
    <span>Present Address:
                        Department of Translational Genomics, University of Cologne, Weyertal 115b, 50931 Cologne, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Christian Becker</span> - <span class="hlFld-Affiliation affiliation">Faculty
of Chemistry and Chemical Biology, TU Dortmund
University, Otto-Hahn-Strasse 4a, 44227 Dortmund, Germany</span>; 
    <span>Present Address:
                        Institute of Pharmaceutical Sciences, Pharmaceutical and Medicinal Chemistry, Eberhard Karls University Tübingen, Auf der Morgenstelle 8, 72076 Tübingen, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Luke Hodson</span> - <span class="hlFld-Affiliation affiliation">Faculty
of Chemistry and Chemical Biology, TU Dortmund
University, Otto-Hahn-Strasse 4a, 44227 Dortmund, Germany</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Chemistry and Polymer Science, Stellenbosch
University, Private Bag
X1, 7602 Matieland, South Africa</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Tonia Kirschner</span> - <span class="hlFld-Affiliation affiliation">Faculty
of Chemistry and Chemical Biology, TU Dortmund
University, Otto-Hahn-Strasse 4a, 44227 Dortmund, Germany</span>; 
    <span class="hlFld-Affiliation affiliation">Drug
Discovery Hub Dortmund (DDHD) am Zentrum für Integrierte Wirkstoffforschung
(ZIW), 44227 Dortmund, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Philip Klövekorn</span> - <span class="hlFld-Affiliation affiliation">Faculty
of Chemistry and Chemical Biology, TU Dortmund
University, Otto-Hahn-Strasse 4a, 44227 Dortmund, Germany</span>; 
    <span>Present Address:
                        Institute of Pharmaceutical Sciences, Pharmaceutical and Medicinal Chemistry, Eberhard Karls University Tübingen, Auf der Morgenstelle 8, 72076 Tübingen, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Julia Ketzer</span> - <span class="hlFld-Affiliation affiliation">Department
of Medical Oncology, Sarcoma Center, West German Cancer Center, Medical
School, University Duisburg-Essen, Hufelandstrasse 55, 45122 Essen, Germany</span>; 
    <span class="hlFld-Affiliation affiliation">German
Cancer Consortium (DKTK), 69120 Heidelberg, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Matthias Baumann</span> - <span class="hlFld-Affiliation affiliation">Lead
Discovery
Center GmbH, Otto-Hahn-Strasse
15, 44227 Dortmund, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Susanne Terheyden</span> - <span class="hlFld-Affiliation affiliation">Lead
Discovery
Center GmbH, Otto-Hahn-Strasse
15, 44227 Dortmund, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Anke Unger</span> - <span class="hlFld-Affiliation affiliation">Lead
Discovery
Center GmbH, Otto-Hahn-Strasse
15, 44227 Dortmund, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jörn Weisner</span> - <span class="hlFld-Affiliation affiliation">Faculty
of Chemistry and Chemical Biology, TU Dortmund
University, Otto-Hahn-Strasse 4a, 44227 Dortmund, Germany</span>; 
    <span class="hlFld-Affiliation affiliation">Drug
Discovery Hub Dortmund (DDHD) am Zentrum für Integrierte Wirkstoffforschung
(ZIW), 44227 Dortmund, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Matthias P. Müller</span> - <span class="hlFld-Affiliation affiliation">Faculty
of Chemistry and Chemical Biology, TU Dortmund
University, Otto-Hahn-Strasse 4a, 44227 Dortmund, Germany</span>; 
    <span class="hlFld-Affiliation affiliation">Drug
Discovery Hub Dortmund (DDHD) am Zentrum für Integrierte Wirkstoffforschung
(ZIW), 44227 Dortmund, Germany</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-1529-8933" title="Orcid link">http://orcid.org/0000-0002-1529-8933</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Willem A. L. van Otterlo</span> - <span class="hlFld-Affiliation affiliation">Department
of Chemistry and Polymer Science, Stellenbosch
University, Private Bag
X1, 7602 Matieland, South Africa</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-3300-6463" title="Orcid link">http://orcid.org/0000-0002-3300-6463</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Sebastian Bauer</span> - <span class="hlFld-Affiliation affiliation">Department
of Medical Oncology, Sarcoma Center, West German Cancer Center, Medical
School, University Duisburg-Essen, Hufelandstrasse 55, 45122 Essen, Germany</span>; 
    <span class="hlFld-Affiliation affiliation">German
Cancer Consortium (DKTK), 69120 Heidelberg, Germany</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>J.L., J.H., T.G., and J.N. contributed equally. The manuscript was written through contributions of all authors. All authors have given approval to the final version of the manuscript.</p></li><li><span class="author-information-subsection-header">Funding</span><p>This work was cofunded by the German Federal Ministry for Education and Research (NGFNPlus and e:Med) (grant nos. BMBF 01GS08104, 01ZX1303C, 01ZX1901B), the Deutsche Forschungsgemeinschaft (DFG), the German federal state North Rhine-Westphalia (NRW), and the European Union (European Regional Development Fund: Investing In Your Future) (EFRE-800400), NEGECA (PerMed NRW), EMODI, and Drug Discovery Hub Dortmund (DDHD). The research visit by L.H. was supported by the South African Medical Research Council (MRC) under a Self-Initiated Research Grant (W.A.L.v.O.). We thank the staff at the PXII X10SA beamline for the possibility to collect diffraction data.</p></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare the following competing financial interest(s): J.L. is shareholder and full-time employee of PearlRiver Bio GmbH. D.R. is shareholder and consultant of PearlRiver Bio GmbH. The Lead Discovery Center GmbH is shareholder of PearlRiver Bio GmbH. M.B., S.T., and A.U. are full-time employees of the Lead Discovery Center GmbH.<br /></br></div></li></ul></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations</td></tr><tr><td class="NLM_term">CTG</td><td class="NLM_def"><p class="first last">CellTiter-Glo</p></td></tr><tr><td class="NLM_term">EGFR</td><td class="NLM_def"><p class="first last">epidermal growth factor receptor</p></td></tr><tr><td class="NLM_term">HLM/MLM</td><td class="NLM_def"><p class="first last">human/mouse liver microsomes</p></td></tr><tr><td class="NLM_term">HTRF</td><td class="NLM_def"><p class="first last">homogeneous time resolved fluorescence</p></td></tr><tr><td class="NLM_term">LLE</td><td class="NLM_def"><p class="first last">ligand-lipophilicity efficiency</p></td></tr><tr><td class="NLM_term">NSCLC</td><td class="NLM_def"><p class="first last">non-small cell lung cancer</p></td></tr><tr><td class="NLM_term">PAINS</td><td class="NLM_def"><p class="first last">pan-assay interference compounds</p></td></tr><tr><td class="NLM_term">RASPELD</td><td class="NLM_def"><p class="first last">Robotics Assisted Screening Platform for Efficient Ligand Discovery</p></td></tr><tr><td class="NLM_term">SEM</td><td class="NLM_def"><p class="first last">[2-(trimethylsilyl)ethoxy]methyl</p></td></tr><tr><td class="NLM_term">TKIs</td><td class="NLM_def"><p class="first last">tyrosine kinase inhibitors</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i165">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_72068" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_72068" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 28 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pillai, R. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Behera, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berry, L. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rossi, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kris, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, B. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bunn, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramalingam, S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khuri, F. R.</span></span> <span> </span><span class="NLM_article-title">HER2 mutations in lung adenocarcinomas: a report from the Lung Cancer Mutation Consortium</span>. <i>Cancer</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>123</i></span>,  <span class="NLM_fpage">4099</span>– <span class="NLM_lpage">4105</span>, <span class="refDoi"> DOI: 10.1002/cncr.30869</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00870&amp;key=10.1002%2Fcncr.30869" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00870&amp;key=28743157" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00870&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhs12mtLzL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=123&publication_year=2017&pages=4099-4105&author=R.+N.+Pillaiauthor=M.+Beheraauthor=L.+D.+Berryauthor=M.+R.+Rossiauthor=M.+G.+Krisauthor=B.+E.+Johnsonauthor=P.+A.+Bunnauthor=S.+S.+Ramalingamauthor=F.+R.+Khuri&title=HER2+mutations+in+lung+adenocarcinomas%3A+a+report+from+the+Lung+Cancer+Mutation+Consortium&doi=10.1002%2Fcncr.30869"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">HER2 mutations in lung adenocarcinomas: A report from the Lung Cancer Mutation Consortium</span></div><div class="casAuthors">Pillai, Rathi N.; Behera, Madhusmita; Berry, Lynne D.; Rossi, Mike R.; Kris, Mark G.; Johnson, Bruce E.; Bunn, Paul A.; Ramalingam, Suresh S.; Khuri, Fadlo R.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer (Hoboken, NJ, United States)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">123</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">4099-4105</span>CODEN:
                <span class="NLM_cas:coden">CANCAR</span>;
        ISSN:<span class="NLM_cas:issn">0008-543X</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">BACKGROUND : Human epidermal growth factor receptor 2 (HER2) mutations have been reported in lung adenocarcinomas.  Herein, the authors describe the prevalence, clin. features, and outcomes assocd. with HER2 mutations in 1007 patients in the Lung Cancer Mutation Consortium (LCMC).  METHODS : Patients with advanced-stage lung adenocarcinomas were enrolled to the LCMC.  Tumor specimens were assessed for diagnosis and adequacy; multiplexed genotyping was performed in Clin. Lab. Improvement Amendments (CLIA)-certified labs. to examine 10 oncogenic drivers.  The LCMC database was queried for patients with HER2 mutations to access demog. data, treatment history, and vital status.  An exploratory anal. was performed to evaluate the survival of patients with HER2 mutations who were treated with HER2-directed therapies.  RESULTS : A total of 920 patients were tested for HER2 mutations; 24 patients (3%) harbored exon 20 insertion mutations (95% confidence interval, 2%-4%).  One patient had a concurrent mesenchymal-epithelial transition factor (MET) amplification.  The median age of the patients was 62 years, with a slight predominance of females over males (14 females vs 10 males).  The majority of the patients were never-smokers (71%) and presented with advanced disease at the time of diagnosis.  The median survival for patients who received HER2-targeted therapies (12 patients) was 2.1 years compared with 1.4 years for those who did not (12 patients) (P = .48).  Patients with HER2 mutations were found to have inferior survival compared with the rest of the LCMC cohort with other mutations: the median survival was 3.5 years in the LCMC population receiving targeted therapy and 2.4 years for patients not receiving targeted therapy.  CONCLUSIONS : HER2 mutations were detected in 3% of patients with lung adenocarcinoma in the LCMC.  HER2-directed therapies should be investigated in this subgroup of patients.  Cancer 2017;123:4099-4105. 2017 American Cancer Society.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo2KOE5lpETgLVg90H21EOLACvtfcHk0lgAcdlxP1tWIA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhs12mtLzL&md5=f9df8f1ac0e5053795c2b7b2d3bac2b8</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1002%2Fcncr.30869&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcncr.30869%26sid%3Dliteratum%253Aachs%26aulast%3DPillai%26aufirst%3DR.%2BN.%26aulast%3DBehera%26aufirst%3DM.%26aulast%3DBerry%26aufirst%3DL.%2BD.%26aulast%3DRossi%26aufirst%3DM.%2BR.%26aulast%3DKris%26aufirst%3DM.%2BG.%26aulast%3DJohnson%26aufirst%3DB.%2BE.%26aulast%3DBunn%26aufirst%3DP.%2BA.%26aulast%3DRamalingam%26aufirst%3DS.%2BS.%26aulast%3DKhuri%26aufirst%3DF.%2BR.%26atitle%3DHER2%2520mutations%2520in%2520lung%2520adenocarcinomas%253A%2520a%2520report%2520from%2520the%2520Lung%2520Cancer%2520Mutation%2520Consortium%26jtitle%3DCancer%26date%3D2017%26volume%3D123%26spage%3D4099%26epage%3D4105%26doi%3D10.1002%2Fcncr.30869" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Arcila, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chaft, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nafa, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roy-Chowdhuri, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lau, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zaidinski, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paik, P. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zakowski, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kris, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ladanyi, M.</span></span> <span> </span><span class="NLM_article-title">Prevalence, clinicopathologic associations, and molecular spectrum of ERBB2 (HER2) tyrosine kinase mutations in lung adenocarcinomas</span>. <i>Clin. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">4910</span>– <span class="NLM_lpage">4918</span>, <span class="refDoi"> DOI: 10.1158/1078-0432.ccr-12-0912</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00870&amp;key=10.1158%2F1078-0432.CCR-12-0912" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00870&amp;key=22761469" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00870&amp;key=1%3ACAS%3A528%3ADC%252BC38Xhtl2lt7%252FM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2012&pages=4910-4918&author=M.+E.+Arcilaauthor=J.+E.+Chaftauthor=K.+Nafaauthor=S.+Roy-Chowdhuriauthor=C.+Lauauthor=M.+Zaidinskiauthor=P.+K.+Paikauthor=M.+F.+Zakowskiauthor=M.+G.+Krisauthor=M.+Ladanyi&title=Prevalence%2C+clinicopathologic+associations%2C+and+molecular+spectrum+of+ERBB2+%28HER2%29+tyrosine+kinase+mutations+in+lung+adenocarcinomas&doi=10.1158%2F1078-0432.ccr-12-0912"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Prevalence, Clinicopathologic Associations, and Molecular Spectrum of ERBB2 (HER2) Tyrosine Kinase Mutations in Lung Adenocarcinomas</span></div><div class="casAuthors">Arcila, Maria E.; Chaft, Jamie E.; Nafa, Khedoudja; Roy-Chowdhuri, Sinchita; Lau, Christopher; Zaidinski, Michael; Paik, Paul K.; Zakowski, Maureen F.; Kris, Mark G.; Ladanyi, Marc</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">4910-4918</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Purpose: Activating mutations in the tyrosine kinase domain of HER2 (ERBB2) have been described in a subset of lung adenocarcinomas (ADCs) and are mutually exclusive with EGFR and KRAS mutations.  The prevalence, clinicopathol. characteristics, prognostic implications, and mol. heterogeneity of HER2-mutated lung ADCs are not well established in U.S. patients.  Exptl. Design: Lung ADC samples (N = 1,478) were first screened for mutations in EGFR (exons 19 and 21) and KRAS (exon 2), and neg. cases were then assessed for HER2 mutations (exons 19-20) using a sizing assay and mass spectrometry.  Testing for addnl. recurrent point mutations in EGFR, KRAS, BRAF, NRAS, PIK3CA, MEK1, and AKT was conducted by mass spectrometry.  ALK rearrangements and HER2 amplification were assessed by FISH.  Results: We identified 25 cases with HER2 mutations, representing 6% of EGFR/KRAS/ALK-neg. specimens.  Small insertions in exon 20 accounted for 96% (24/25) of the cases.  Compared with insertions in EGFR exon 20, there was less variability, with 83% (20/24) being a 12 bp insertion causing duplication of amino acids YVMA at codon 775.  Morphol., 92% (23/25) were moderately or poorly differentiated ADC.  HER2 mutation was not assocd. with concurrent HER2 amplification in 11 cases tested for both.  HER2 mutations were more frequent among never-smokers (P < 0.0001) but there were no assocns. with sex, race, or stage.  Conclusions: HER2 mutations identify a distinct subset of lung ADCs.  Given the high prevalence of lung cancer worldwide and the availability of std. and investigational therapies targeting HER2, routine clin. genotyping of lung ADC should include HER2.  Clin Cancer Res; 18(18); 4910-8. ©2012 AACR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGql2xQKmrAL7bVg90H21EOLACvtfcHk0lgZB5anLfVBdQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xhtl2lt7%252FM&md5=683086d6c06b3fb76b94f7c4d8bfed40</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1158%2F1078-0432.CCR-12-0912&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1078-0432.CCR-12-0912%26sid%3Dliteratum%253Aachs%26aulast%3DArcila%26aufirst%3DM.%2BE.%26aulast%3DChaft%26aufirst%3DJ.%2BE.%26aulast%3DNafa%26aufirst%3DK.%26aulast%3DRoy-Chowdhuri%26aufirst%3DS.%26aulast%3DLau%26aufirst%3DC.%26aulast%3DZaidinski%26aufirst%3DM.%26aulast%3DPaik%26aufirst%3DP.%2BK.%26aulast%3DZakowski%26aufirst%3DM.%2BF.%26aulast%3DKris%26aufirst%3DM.%2BG.%26aulast%3DLadanyi%26aufirst%3DM.%26atitle%3DPrevalence%252C%2520clinicopathologic%2520associations%252C%2520and%2520molecular%2520spectrum%2520of%2520ERBB2%2520%2528HER2%2529%2520tyrosine%2520kinase%2520mutations%2520in%2520lung%2520adenocarcinomas%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D2012%26volume%3D18%26spage%3D4910%26epage%3D4918%26doi%3D10.1158%2F1078-0432.ccr-12-0912" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ishikawa, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seto, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Banno, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kawakita, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oorui, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taniguchi, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohta, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tamura, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakayama, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miki, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kamiguchi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanaka, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Habuka, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sogabe, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yano, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aertgeerts, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kamiyama, K.</span></span> <span> </span><span class="NLM_article-title">Design and synthesis of novel human epidermal growth factor receptor 2 (HER2)/epidermal growth factor receptor (EGFR) dual inhibitors bearing a pyrrolo[3,2-d]pyrimidine scaffold</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">8030</span>– <span class="NLM_lpage">8050</span>, <span class="refDoi"> DOI: 10.1021/jm2008634</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm2008634" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00870&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtlajur7M" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=8030-8050&author=T.+Ishikawaauthor=M.+Setoauthor=H.+Bannoauthor=Y.+Kawakitaauthor=M.+Ooruiauthor=T.+Taniguchiauthor=Y.+Ohtaauthor=T.+Tamuraauthor=A.+Nakayamaauthor=H.+Mikiauthor=H.+Kamiguchiauthor=T.+Tanakaauthor=N.+Habukaauthor=S.+Sogabeauthor=J.+Yanoauthor=K.+Aertgeertsauthor=K.+Kamiyama&title=Design+and+synthesis+of+novel+human+epidermal+growth+factor+receptor+2+%28HER2%29%2Fepidermal+growth+factor+receptor+%28EGFR%29+dual+inhibitors+bearing+a+pyrrolo%5B3%2C2-d%5Dpyrimidine+scaffold&doi=10.1021%2Fjm2008634"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Design and Synthesis of Novel Human Epidermal Growth Factor Receptor 2 (HER2)/Epidermal Growth Factor Receptor (EGFR) Dual Inhibitors Bearing a Pyrrolo[3,2-d]pyrimidine Scaffold</span></div><div class="casAuthors">Ishikawa, Tomoyasu; Seto, Masaki; Banno, Hiroshi; Kawakita, Youichi; Oorui, Mami; Taniguchi, Takahiko; Ohta, Yoshikazu; Tamura, Toshiya; Nakayama, Akiko; Miki, Hiroshi; Kamiguchi, Hidenori; Tanaka, Toshimasa; Habuka, Noriyuki; Sogabe, Satoshi; Yano, Jason; Aertgeerts, Kathleen; Kamiyama, Keiji</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">8030-8050</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Dual inhibitors of human epidermal growth factor receptor 2 (HER2) and epidermal growth factor receptor (EGFR) have been investigated for breast, lung, gastric, prostate, and other cancers; one, lapatinib, is currently approved for breast cancer.  To develop novel HER2/EGFR dual kinase inhibitors, we designed and synthesized pyrrolo[3,2-d]pyrimidine derivs., e.g. I, capable of fitting into the receptors' ATP binding site.  Among the prepd. compds., I showed potent HER2 and EGFR (HER1) inhibitory activities as well as tumor growth inhibitory activity.  The X-ray cocrystal structures of I with both HER2 and EGFR demonstrated that I interacts with the expected residues in their resp. ATP pockets.  Furthermore, reflecting its good oral bioavailability, I exhibited potent in vivo efficacy in HER2-overexpressing tumor xenograft models.  On the basis of these findings, we report I (TAK-285) as a promising candidate for clin. development as a novel HER2/EGFR dual kinase inhibitor.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq9QtqAj6idYrVg90H21EOLACvtfcHk0lhu46xJ_yVEBQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtlajur7M&md5=ce030ac133a1c0ff7cb78b269ff85987</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1021%2Fjm2008634&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm2008634%26sid%3Dliteratum%253Aachs%26aulast%3DIshikawa%26aufirst%3DT.%26aulast%3DSeto%26aufirst%3DM.%26aulast%3DBanno%26aufirst%3DH.%26aulast%3DKawakita%26aufirst%3DY.%26aulast%3DOorui%26aufirst%3DM.%26aulast%3DTaniguchi%26aufirst%3DT.%26aulast%3DOhta%26aufirst%3DY.%26aulast%3DTamura%26aufirst%3DT.%26aulast%3DNakayama%26aufirst%3DA.%26aulast%3DMiki%26aufirst%3DH.%26aulast%3DKamiguchi%26aufirst%3DH.%26aulast%3DTanaka%26aufirst%3DT.%26aulast%3DHabuka%26aufirst%3DN.%26aulast%3DSogabe%26aufirst%3DS.%26aulast%3DYano%26aufirst%3DJ.%26aulast%3DAertgeerts%26aufirst%3DK.%26aulast%3DKamiyama%26aufirst%3DK.%26atitle%3DDesign%2520and%2520synthesis%2520of%2520novel%2520human%2520epidermal%2520growth%2520factor%2520receptor%25202%2520%2528HER2%2529%252Fepidermal%2520growth%2520factor%2520receptor%2520%2528EGFR%2529%2520dual%2520inhibitors%2520bearing%2520a%2520pyrrolo%255B3%252C2-d%255Dpyrimidine%2520scaffold%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26spage%3D8030%26epage%3D8050%26doi%3D10.1021%2Fjm2008634" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gschwind, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fischer, O. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ullrich, A.</span></span> <span> </span><span class="NLM_article-title">The discovery of receptor tyrosine kinases: targets for cancer therapy</span>. <i>Nat. Rev. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">361</span>– <span class="NLM_lpage">370</span>, <span class="refDoi"> DOI: 10.1038/nrc1360</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00870&amp;key=10.1038%2Fnrc1360" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00870&amp;key=15122207" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00870&amp;key=1%3ACAS%3A528%3ADC%252BD2cXjsFSlu7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2004&pages=361-370&author=A.+Gschwindauthor=O.+M.+Fischerauthor=A.+Ullrich&title=The+discovery+of+receptor+tyrosine+kinases%3A+targets+for+cancer+therapy&doi=10.1038%2Fnrc1360"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Timeline: The discovery of receptor tyrosine kinases: targets for cancer therapy</span></div><div class="casAuthors">Gschwind, Andreas; Fischer, Oliver M.; Ullrich, Axel</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">361-370</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Receptor tyrosine kinases are a subclass of cell-surface growth-factor receptors with an intrinsic, ligand-controlled tyrosine-kinase activity.  They regulate diverse functions in normal cells and have a crucial role in oncogenesis.  Twenty years ago, the 1st primary structure of a receptor Tyr kinase, the epidermal growth factor receptor, was elucidated.  The characterization of both the mol. architecture of receptor Tyr kinases and the main functions of these proteins and their ligands in tumorigenesis opened the door to a new era in mol. oncol. and paved the way to the development of the 1st target-specific cancer therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo1NLVNn4dw37Vg90H21EOLACvtfcHk0lhu46xJ_yVEBQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXjsFSlu7g%253D&md5=ea24c0ccaea68b743b5da9587ed19406</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1038%2Fnrc1360&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc1360%26sid%3Dliteratum%253Aachs%26aulast%3DGschwind%26aufirst%3DA.%26aulast%3DFischer%26aufirst%3DO.%2BM.%26aulast%3DUllrich%26aufirst%3DA.%26atitle%3DThe%2520discovery%2520of%2520receptor%2520tyrosine%2520kinases%253A%2520targets%2520for%2520cancer%2520therapy%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2004%26volume%3D4%26spage%3D361%26epage%3D370%26doi%3D10.1038%2Fnrc1360" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Aertgeerts, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skene, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yano, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sang, B.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zou, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Snell, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jennings, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iwamoto, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Habuka, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hirokawa, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ishikawa, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanaka, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miki, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohta, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sogabe, S.</span></span> <span> </span><span class="NLM_article-title">Structural analysis of the mechanism of inhibition and allosteric activation of the kinase domain of HER2 protein</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>286</i></span>,  <span class="NLM_fpage">18756</span>– <span class="NLM_lpage">18765</span>, <span class="refDoi"> DOI: 10.1074/jbc.m110.206193</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00870&amp;key=10.1074%2Fjbc.m110.206193" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00870&amp;key=21454582" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00870&amp;key=1%3ACAS%3A528%3ADC%252BC3MXmtlGltL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=286&publication_year=2011&pages=18756-18765&author=K.+Aertgeertsauthor=R.+Skeneauthor=J.+Yanoauthor=B.-C.+Sangauthor=H.+Zouauthor=G.+Snellauthor=A.+Jenningsauthor=K.+Iwamotoauthor=N.+Habukaauthor=A.+Hirokawaauthor=T.+Ishikawaauthor=T.+Tanakaauthor=H.+Mikiauthor=Y.+Ohtaauthor=S.+Sogabe&title=Structural+analysis+of+the+mechanism+of+inhibition+and+allosteric+activation+of+the+kinase+domain+of+HER2+protein&doi=10.1074%2Fjbc.m110.206193"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Structural Analysis of the Mechanism of Inhibition and Allosteric Activation of the Kinase Domain of HER2 Protein</span></div><div class="casAuthors">Aertgeerts, Kathleen; Skene, Robert; Yano, Jason; Sang, Bi-Ching; Zou, Hua; Snell, Gyorgy; Jennings, Andy; Iwamoto, Keiji; Habuka, Noriyuki; Hirokawa, Aki; Ishikawa, Tomoyasu; Tanaka, Toshimasa; Miki, Hiroshi; Ohta, Yoshikazu; Sogabe, Satoshi</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">286</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">18756-18765</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Aberrant signaling of ErbB family members human epidermal growth factor 2 (HER2) and epidermal growth factor receptor (EGFR) is implicated in many human cancers, and HER2 expression is predictive of human disease recurrence and prognosis.  Small mol. kinase inhibitors of EGFR and of both HER2 and EGFR have received approval for the treatment of cancer.  We present the first high resoln. crystal structure of the kinase domain of HER2 in complex with a selective inhibitor to understand protein activation, inhibition, and function at the mol. level.  HER2 kinase domain crystallizes as a dimer and suggests evidence for an allosteric mechanism of activation comparable with previously reported activation mechanisms for EGFR and HER4.  A unique Gly-rich region in HER2 following the α-helix C is responsible for increased conformational flexibility within the active site and could explain the low intrinsic catalytic activity previously reported for HER2.  In addn., we solved the crystal structure of the kinase domain of EGFR in complex with a HER2/EGFR dual inhibitor (TAK-285).  Comparison with previously reported inactive and active EGFR kinase domain structures gave insight into the mechanism of HER2 and EGFR inhibition and may help guide the design and development of new cancer drugs with improved potency and selectivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGomtAlJ0FGUX7Vg90H21EOLACvtfcHk0lhx0KTK4rTOMQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXmtlGltL0%253D&md5=c7897f5363d6452f7b60a8ced0f2bc30</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1074%2Fjbc.m110.206193&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.m110.206193%26sid%3Dliteratum%253Aachs%26aulast%3DAertgeerts%26aufirst%3DK.%26aulast%3DSkene%26aufirst%3DR.%26aulast%3DYano%26aufirst%3DJ.%26aulast%3DSang%26aufirst%3DB.-C.%26aulast%3DZou%26aufirst%3DH.%26aulast%3DSnell%26aufirst%3DG.%26aulast%3DJennings%26aufirst%3DA.%26aulast%3DIwamoto%26aufirst%3DK.%26aulast%3DHabuka%26aufirst%3DN.%26aulast%3DHirokawa%26aufirst%3DA.%26aulast%3DIshikawa%26aufirst%3DT.%26aulast%3DTanaka%26aufirst%3DT.%26aulast%3DMiki%26aufirst%3DH.%26aulast%3DOhta%26aufirst%3DY.%26aulast%3DSogabe%26aufirst%3DS.%26atitle%3DStructural%2520analysis%2520of%2520the%2520mechanism%2520of%2520inhibition%2520and%2520allosteric%2520activation%2520of%2520the%2520kinase%2520domain%2520of%2520HER2%2520protein%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2011%26volume%3D286%26spage%3D18756%26epage%3D18765%26doi%3D10.1074%2Fjbc.m110.206193" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lategahn, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keul, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rauh, D.</span></span> <span> </span><span class="NLM_article-title">Lessons to be learned: the molecular basis of kinase-targeted therapies and drug resistance in non-small cell lung cancer</span>. <i>Angew. Chem., Int. Ed. Engl.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">2307</span>– <span class="NLM_lpage">2313</span>, <span class="refDoi"> DOI: 10.1002/anie.201710398</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00870&amp;key=10.1002%2Fanie.201710398" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00870&amp;key=29178586" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00870&amp;key=1%3ACAS%3A280%3ADC%252BC1M3lt1yrsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2018&pages=2307-2313&author=J.+Lategahnauthor=M.+Keulauthor=D.+Rauh&title=Lessons+to+be+learned%3A+the+molecular+basis+of+kinase-targeted+therapies+and+drug+resistance+in+non-small+cell+lung+cancer&doi=10.1002%2Fanie.201710398"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Lessons To Be Learned: The Molecular Basis of Kinase-Targeted Therapies and Drug Resistance in Non-Small Cell Lung Cancer</span></div><div class="casAuthors">Lategahn Jonas; Keul Marina; Rauh Daniel</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie (International ed. in English)</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2307-2313</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The treatment of non-small cell lung cancer (NSCLC) is currently experiencing a revolution.  Over the last decade, the knowledge gained about the biochemical features of biomarkers and their predictive abilities has led to the development of targeted small-molecule inhibitors that present an alternative to harsh chemotherapy.  The use of these new therapies has improved the quality of life and increased the survival of patients.  The occurrence of inevitable drug resistance requires the constant development of precision medicine.  The detailed understanding of the target biology and the search for innovative chemical approaches has encouraged investigations in this field.  Herein, we review selected aspects of the molecular targets and present an overview of current topics and challenges in the rational development of small molecules to target NSCLC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQ1En_K7f1V4brIbt-tGZmxfW6udTcc2ebFKaTBGKZnmbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1M3lt1yrsA%253D%253D&md5=634b74865cb6382392e8846e02de3810</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1002%2Fanie.201710398&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201710398%26sid%3Dliteratum%253Aachs%26aulast%3DLategahn%26aufirst%3DJ.%26aulast%3DKeul%26aufirst%3DM.%26aulast%3DRauh%26aufirst%3DD.%26atitle%3DLessons%2520to%2520be%2520learned%253A%2520the%2520molecular%2520basis%2520of%2520kinase-targeted%2520therapies%2520and%2520drug%2520resistance%2520in%2520non-small%2520cell%2520lung%2520cancer%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%2520Engl.%26date%3D2018%26volume%3D57%26spage%3D2307%26epage%3D2313%26doi%3D10.1002%2Fanie.201710398" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Niggenaber, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hardick, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lategahn, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rauh, D.</span></span> <span> </span><span class="NLM_article-title">Structure defines function: clinically relevant mutations in ErbB kinases</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">40</span>– <span class="NLM_lpage">51</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b00964</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b00964" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00870&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhsFymtLbK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=40-51&author=J.+Niggenaberauthor=J.+Hardickauthor=J.+Lategahnauthor=D.+Rauh&title=Structure+defines+function%3A+clinically+relevant+mutations+in+ErbB+kinases&doi=10.1021%2Facs.jmedchem.9b00964"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Structure Defines Function: Clinically Relevant Mutations in ErbB Kinases</span></div><div class="casAuthors">Niggenaber, Janina; Hardick, Julia; Lategahn, Jonas; Rauh, Daniel</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">40-51</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The ErbB receptor tyrosine kinase family members EGFR (epidermal growth factor receptor) and Her2 are among the prominent mutated oncogenic drivers of non-small cell lung cancer (NSCLC).  Their importance in proliferation, apoptosis, and cell death ultimately renders them hot targets in cancer therapy.  Small-mol. tyrosine kinase inhibitors seem well suited to be tailor-made therapeutics for EGFR mutant NSCLC; however, drug resistance mutations limit their success.  Against this background, the elucidation and visualization of the three-dimensional structure of cancer-related kinases provide valuable insights into their mol. functions.  This field has undergone a revolution because X-ray crystal structure detns. aided structure-based drug design approaches and clarified the effect of activating and resistance-conferring mutations.  Here, we present an overview of important mutations affecting EGFR and Her2 and highlight their influence on the kinase domain conformations and active site accessibility.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqgcJQfrfEznLVg90H21EOLACvtfcHk0ljIW-Hd76n0dQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhsFymtLbK&md5=e9f13c582aeeca169e66ae119aadf600</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b00964&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b00964%26sid%3Dliteratum%253Aachs%26aulast%3DNiggenaber%26aufirst%3DJ.%26aulast%3DHardick%26aufirst%3DJ.%26aulast%3DLategahn%26aufirst%3DJ.%26aulast%3DRauh%26aufirst%3DD.%26atitle%3DStructure%2520defines%2520function%253A%2520clinically%2520relevant%2520mutations%2520in%2520ErbB%2520kinases%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D40%26epage%3D51%26doi%3D10.1021%2Facs.jmedchem.9b00964" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hata, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yoshioka, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fujita, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kunimasa, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaji, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Imai, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomii, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iwasaku, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nishiyama, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ishida, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Katakami, N.</span></span> <span> </span><span class="NLM_article-title">Complex mutations in the epidermal growth factor receptor gene in non-small cell lung cancer</span>. <i>J. Thorac. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">1524</span>– <span class="NLM_lpage">1528</span>, <span class="refDoi"> DOI: 10.1097/jto.0b013e3181e8b3c5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00870&amp;key=10.1097%2FJTO.0b013e3181e8b3c5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00870&amp;key=20808254" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00870&amp;key=1%3ACAS%3A280%3ADC%252BC3cfkvFSisA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2010&pages=1524-1528&author=A.+Hataauthor=H.+Yoshiokaauthor=S.+Fujitaauthor=K.+Kunimasaauthor=R.+Kajiauthor=Y.+Imaiauthor=K.+Tomiiauthor=M.+Iwasakuauthor=A.+Nishiyamaauthor=T.+Ishidaauthor=N.+Katakami&title=Complex+mutations+in+the+epidermal+growth+factor+receptor+gene+in+non-small+cell+lung+cancer&doi=10.1097%2Fjto.0b013e3181e8b3c5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Complex mutations in the epidermal growth factor receptor gene in non-small cell lung cancer</span></div><div class="casAuthors">Hata Akito; Yoshioka Hiroshige; Fujita Shiro; Kunimasa Kei; Kaji Reiko; Imai Yukihiro; Tomii Keisuke; Iwasaku Masahiro; Nishiyama Akihiro; Ishida Tadashi; Katakami Nobuyuki</div><div class="citationInfo"><span class="NLM_cas:title">Journal of thoracic oncology : official publication of the International Association for the Study of Lung Cancer</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1524-8</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">INTRODUCTION:  Mutation of the epidermal growth factor receptor (EGFR) gene can predict the efficacy of EGFR-tyrosine kinase inhibitors.  Different mutations have been shown to co-occur in a single tumor.  However, the frequency of these so-called "complex mutations" and the efficacy of gefitinib in treating patients with these mutations are unclear.  METHODS:  We investigated the frequency of complex mutations in 783 patients with non-small cell lung cancer seen at our institutes between April 2006 and May 2009.  Mutational analysis was performed using the peptide nucleic acid-locked nucleic acid polymerase chain reaction clamp method.  Gefitinib efficacy was evaluated in patients found to have complex mutations.  RESULTS:  EGFR: mutations were detected in 318 (41%) patients, with 21 (6.6%) of these individuals having complex mutations.  Sixteen of these 21 patients received gefitinib.  The response rate (RR) was 67% (95% confidence interval [CI], 35-90%) and median progression-free survival was 12.2 months (95% CI, 1.3 months to undeterminable).  Analysis of RR according to mutation type revealed that patients with deletional mutation in exon 19 (Del-19) and a point mutation in exon 21 (L858R) had a better RR (86%, 6 of 7) than those with other complex mutation patterns such as a point mutation in exon 18 (G719S) + L858R (40%, 2 of 5) (p = 0.2222).  The median progression-free survival was also longer in these patients (16.5 months; 95% CI, 1.1 months to undeterminable versus 3.8 months; 95% CI, 0.7-10.0 months) (p = 0.0459).  CONCLUSIONS:  Complex EGFR mutations are not rare.  Gefitinib has different efficacy according to the type of complex EGFR mutations.  Patients with Del-19 and L858R mutations may benefit more from gefitinib than other types of complex mutations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTZq7EL0Eg8p7HHuJhbyicZfW6udTcc2eaipZEPD3AzcLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3cfkvFSisA%253D%253D&md5=68353796c0df45267d25c4e4be03a305</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1097%2FJTO.0b013e3181e8b3c5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1097%252FJTO.0b013e3181e8b3c5%26sid%3Dliteratum%253Aachs%26aulast%3DHata%26aufirst%3DA.%26aulast%3DYoshioka%26aufirst%3DH.%26aulast%3DFujita%26aufirst%3DS.%26aulast%3DKunimasa%26aufirst%3DK.%26aulast%3DKaji%26aufirst%3DR.%26aulast%3DImai%26aufirst%3DY.%26aulast%3DTomii%26aufirst%3DK.%26aulast%3DIwasaku%26aufirst%3DM.%26aulast%3DNishiyama%26aufirst%3DA.%26aulast%3DIshida%26aufirst%3DT.%26aulast%3DKatakami%26aufirst%3DN.%26atitle%3DComplex%2520mutations%2520in%2520the%2520epidermal%2520growth%2520factor%2520receptor%2520gene%2520in%2520non-small%2520cell%2520lung%2520cancer%26jtitle%3DJ.%2520Thorac.%2520Oncol.%26date%3D2010%26volume%3D5%26spage%3D1524%26epage%3D1528%26doi%3D10.1097%2Fjto.0b013e3181e8b3c5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ambrogio, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shimamura, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kubo, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takahashi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chirieac, L. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Padera, R. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shapiro, G. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baum, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Himmelsbach, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rettig, W. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meyerson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Solca, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greulich, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, K.-K.</span></span> <span> </span><span class="NLM_article-title">BIBW2992, an irreversible EGFR/HER2 inhibitor highly effective in preclinical lung cancer models</span>. <i>Oncogene</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">4702</span>– <span class="NLM_lpage">4711</span>, <span class="refDoi"> DOI: 10.1038/onc.2008.109</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00870&amp;key=10.1038%2Fonc.2008.109" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00870&amp;key=18408761" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00870&amp;key=1%3ACAS%3A528%3ADC%252BD1cXptlKnsbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2008&pages=4702-4711&author=D.+Liauthor=L.+Ambrogioauthor=T.+Shimamuraauthor=S.+Kuboauthor=M.+Takahashiauthor=L.+R.+Chirieacauthor=R.+F.+Paderaauthor=G.+I.+Shapiroauthor=A.+Baumauthor=F.+Himmelsbachauthor=W.+J.+Rettigauthor=M.+Meyersonauthor=F.+Solcaauthor=H.+Greulichauthor=K.-K.+Wong&title=BIBW2992%2C+an+irreversible+EGFR%2FHER2+inhibitor+highly+effective+in+preclinical+lung+cancer+models&doi=10.1038%2Fonc.2008.109"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">BIBW2992, an irreversible EGFR/HER2 inhibitor highly effective in preclinical lung cancer models</span></div><div class="casAuthors">Li, D.; Ambrogio, L.; Shimamura, T.; Kubo, S.; Takahashi, M.; Chirieac, L. R.; Padera, R. F.; Shapiro, G. I.; Baum, A.; Himmelsbach, F.; Rettig, W. J.; Meyerson, M.; Solca, F.; Greulich, H.; Wong, K-K.</div><div class="citationInfo"><span class="NLM_cas:title">Oncogene</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">34</span>),
    <span class="NLM_cas:pages">4702-4711</span>CODEN:
                <span class="NLM_cas:coden">ONCNES</span>;
        ISSN:<span class="NLM_cas:issn">0950-9232</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Genetic alterations in the kinase domain of the epidermal growth factor receptor (EGFR) in non-small cell lung cancer (NSCLC) patients are assocd. with sensitivity to treatment with small mol. tyrosine kinase inhibitors.  Although first-generation reversible, ATP-competitive inhibitors showed encouraging clin. responses in lung adenocarcinoma tumors harboring such EGFR mutations, almost all patients developed resistance to these inhibitors over time.  Such resistance to first-generation EGFR inhibitors was frequently linked to an acquired T790M point mutation in the kinase domain of EGFR, or upregulation of signaling pathways downstream of HER3.  Overcoming these mechanisms of resistance, as well as primary resistance to reversible EGFR inhibitors driven by a subset of EGFR mutations, will be necessary for development of an effective targeted therapy regimen.  Here, we show that BIBW2992, an anilino-quinazoline designed to irreversibly bind EGFR and HER2, potently suppresses the kinase activity of wild-type and activated EGFR and HER2 mutants, including erlotinib-resistant isoforms.  Consistent with this activity, BIBW2992 suppresses transformation in isogenic cell-based assays, inhibits survival of cancer cell lines and induces tumor regression in xenograft and transgenic lung cancer models, with superior activity over erlotinib.  These findings encourage further testing of BIBW2992 in lung cancer patients harboring EGFR or HER2 oncogenes.  Oncogene (2008) 27, 4702-4711; doi:10.1038/onc.2008.109; published online 14 Apr. 2008.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo5PQdKep-Sp7Vg90H21EOLACvtfcHk0ljIW-Hd76n0dQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXptlKnsbk%253D&md5=24945893e83b4e3d01c267100fdaf57c</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1038%2Fonc.2008.109&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fonc.2008.109%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DD.%26aulast%3DAmbrogio%26aufirst%3DL.%26aulast%3DShimamura%26aufirst%3DT.%26aulast%3DKubo%26aufirst%3DS.%26aulast%3DTakahashi%26aufirst%3DM.%26aulast%3DChirieac%26aufirst%3DL.%2BR.%26aulast%3DPadera%26aufirst%3DR.%2BF.%26aulast%3DShapiro%26aufirst%3DG.%2BI.%26aulast%3DBaum%26aufirst%3DA.%26aulast%3DHimmelsbach%26aufirst%3DF.%26aulast%3DRettig%26aufirst%3DW.%2BJ.%26aulast%3DMeyerson%26aufirst%3DM.%26aulast%3DSolca%26aufirst%3DF.%26aulast%3DGreulich%26aufirst%3DH.%26aulast%3DWong%26aufirst%3DK.-K.%26atitle%3DBIBW2992%252C%2520an%2520irreversible%2520EGFR%252FHER2%2520inhibitor%2520highly%2520effective%2520in%2520preclinical%2520lung%2520cancer%2520models%26jtitle%3DOncogene%26date%3D2008%26volume%3D27%26spage%3D4702%26epage%3D4711%26doi%3D10.1038%2Fonc.2008.109" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dungo, R. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keating, G. M.</span></span> <span> </span><span class="NLM_article-title">Afatinib: first global approval</span>. <i>Drugs</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>73</i></span>,  <span class="NLM_fpage">1503</span>– <span class="NLM_lpage">1515</span>, <span class="refDoi"> DOI: 10.1007/s40265-013-0111-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00870&amp;key=10.1007%2Fs40265-013-0111-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00870&amp;key=23982599" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00870&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvVWmurnM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=73&publication_year=2013&pages=1503-1515&author=R.+T.+Dungoauthor=G.+M.+Keating&title=Afatinib%3A+first+global+approval&doi=10.1007%2Fs40265-013-0111-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Afatinib: First Global Approval</span></div><div class="casAuthors">Dungo, Rosselle T.; Keating, Gillian M.</div><div class="citationInfo"><span class="NLM_cas:title">Drugs</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">73</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">1503-1515</span>CODEN:
                <span class="NLM_cas:coden">DRUGAY</span>;
        ISSN:<span class="NLM_cas:issn">0012-6667</span>.
    
            (<span class="NLM_cas:orgname">Springer International Publishing AG</span>)
        </div><div class="casAbstract">A review.  Afatinib, an irreversible inhibitor of the ErbB family of tyrosine kinases, is under development with Boehringer Ingelheim for the once-daily, oral treatment of cancer.  Afatinib downregulates ErbB signalling by covalently binding to epidermal growth factor receptor (EGFR), human epidermal growth factor receptor (HER) 2 and HER4, irreversibly inhibiting tyrosine kinase autophosphorylation.  It also inhibits transphosphorylation of HER3.  Oral afatinib (Gilotrif®) has been approved in the US for the first-line treatment of patients with metastatic non-small-cell lung cancer (NSCLC) who have tumors with EGFR exon 19 deletions or exon 21 (L858R) substitution mutations as detected by a US FDA-approved test.  Afatinib has also been approved in Taiwan for the first-line treatment of patients with EGFR mutation-pos. NSCLC.  In addn., the European Medicines Agency's Committee for Medicinal Products for Human Use has recommended the approval of afatinib (Giotrif) for the treatment of patients with locally advanced or metastatic NSCLC with activating EGFR mutations who are EGFR tyrosine kinase inhibitor naive.  Afatinib is also under regulatory review in Canada, Japan and other Asian countries.  This article summarizes the milestones in the development of afatinib, leading to this first approval in patients with metastatic NSCLC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrdPcU5aIqrY7Vg90H21EOLACvtfcHk0lj4mcT0FDNQ6g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvVWmurnM&md5=21851cbfac56e2fffcba6c731f123cff</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1007%2Fs40265-013-0111-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs40265-013-0111-6%26sid%3Dliteratum%253Aachs%26aulast%3DDungo%26aufirst%3DR.%2BT.%26aulast%3DKeating%26aufirst%3DG.%2BM.%26atitle%3DAfatinib%253A%2520first%2520global%2520approval%26jtitle%3DDrugs%26date%3D2013%26volume%3D73%26spage%3D1503%26epage%3D1515%26doi%3D10.1007%2Fs40265-013-0111-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cha, M. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, K.-O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, J. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chae, Y. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, Y. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suh, K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, S. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, M. S.</span></span> <span> </span><span class="NLM_article-title">Antitumor activity of HM781-36B, a highly effective pan-HER inhibitor in erlotinib-resistant NSCLC and other EGFR-dependent cancer models</span>. <i>Int. J. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>130</i></span>,  <span class="NLM_fpage">2445</span>– <span class="NLM_lpage">2454</span>, <span class="refDoi"> DOI: 10.1002/ijc.26276</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00870&amp;key=10.1002%2Fijc.26276" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00870&amp;key=21732342" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00870&amp;key=1%3ACAS%3A528%3ADC%252BC38XjvVajtr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=130&publication_year=2012&pages=2445-2454&author=M.+Y.+Chaauthor=K.-O.+Leeauthor=M.+Kimauthor=J.+Y.+Songauthor=K.+H.+Leeauthor=J.+Parkauthor=Y.+J.+Chaeauthor=Y.+H.+Kimauthor=K.+H.+Suhauthor=G.+S.+Leeauthor=S.+B.+Parkauthor=M.+S.+Kim&title=Antitumor+activity+of+HM781-36B%2C+a+highly+effective+pan-HER+inhibitor+in+erlotinib-resistant+NSCLC+and+other+EGFR-dependent+cancer+models&doi=10.1002%2Fijc.26276"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Antitumor activity of HM781-36B, a highly effective pan-HER inhibitor in erlotinib-resistant NSCLC and other EGFR-dependent cancer models</span></div><div class="casAuthors">Cha, Mi Young; Lee, Kwang-Ok; Kim, Mira; Song, Ji Yeon; Lee, Kyu Hang; Park, Jongmin; Chae, Yun Jung; Kim, Young Hoon; Suh, Kwee Hyun; Lee, Gwan Sun; Park, Seung Bum; Kim, Maeng Sup</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Cancer</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">130</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2445-2454</span>CODEN:
                <span class="NLM_cas:coden">IJCNAW</span>;
        ISSN:<span class="NLM_cas:issn">0020-7136</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">The epidermal growth factor receptor (EGFR) family of receptor tyrosine kinases has been implicated in a variety of cancers.  In particular, activating mutations such as the L858R point mutation in exon 21 and the small in-frame deletions in exon 19 of the EGFR tyrosine kinase domain are correlated with sensitivity to EGFR tyrosine kinase inhibitors in non-small cell lung cancer (NSCLC) patients.  Clin. treatment of patients is limited by the development of drug resistance resulting mainly from a gatekeeper mutation (T790M).  In this study, we evaluated the therapeutic potential of a novel, irreversible pan-HER inhibitor, HM781-36B.  The results from this study show that HM781-36B is a potent inhibitor of EGFR in vitro, including the EGFR-acquired resistance mutation (T790M), as well as HER-2 and HER-4, compared with other EGFR tyrosine kinases inhibitors (erlotinib, lapatinib and BIBW2992).  HM781-36B treatment of EGFR DelE746A750-harboring erlotinib-sensitive HCC827 and EGFR L858R/T790M-harboring erlotinib-resistant NCI-H1975 NSCLC cells results in the inhibition of EGFR phosphorylation and the subsequent deactivation of downstream signaling proteins.  Addnl., HM781-36B shows an excellent efficacy in a variety of EGFR- and HER-2-dependent tumor xenograft models, including erlotinib-sensitive HCC827 NSCLC cells, erlotinib-resistant NCI-H1975 NSCLC cells, HER-2 overexpressing Calu-3 NSCLC cells, NCI-N87 gastric cancer cells, SK-Ov3 ovarian cancer cells and EGFR-overexpressing A431 epidermoid carcinoma cancer cells.  On the basis of these preclin. results, HM781-36B is the most potent pan-HER inhibitor, which will be advantageous for the treatment of patients with NSCLC including clin. limitation caused by acquired mutation (EGFR T790M), breast cancer and gastric cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrT8jGQHJHfWrVg90H21EOLACvtfcHk0lj4mcT0FDNQ6g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XjvVajtr8%253D&md5=004f3a840f97464644b3a7466d636748</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1002%2Fijc.26276&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fijc.26276%26sid%3Dliteratum%253Aachs%26aulast%3DCha%26aufirst%3DM.%2BY.%26aulast%3DLee%26aufirst%3DK.-O.%26aulast%3DKim%26aufirst%3DM.%26aulast%3DSong%26aufirst%3DJ.%2BY.%26aulast%3DLee%26aufirst%3DK.%2BH.%26aulast%3DPark%26aufirst%3DJ.%26aulast%3DChae%26aufirst%3DY.%2BJ.%26aulast%3DKim%26aufirst%3DY.%2BH.%26aulast%3DSuh%26aufirst%3DK.%2BH.%26aulast%3DLee%26aufirst%3DG.%2BS.%26aulast%3DPark%26aufirst%3DS.%2BB.%26aulast%3DKim%26aufirst%3DM.%2BS.%26atitle%3DAntitumor%2520activity%2520of%2520HM781-36B%252C%2520a%2520highly%2520effective%2520pan-HER%2520inhibitor%2520in%2520erlotinib-resistant%2520NSCLC%2520and%2520other%2520EGFR-dependent%2520cancer%2520models%26jtitle%3DInt.%2520J.%2520Cancer%26date%3D2012%26volume%3D130%26spage%3D2445%26epage%3D2454%26doi%3D10.1002%2Fijc.26276" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Robichaux, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elamin, Y. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carter, B. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poteete, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Estrada-Bernal, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Le, A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Truini, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nilsson, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roarty, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goldberg, S. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brahmer, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Altan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Papadimitrakopoulou, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Politi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doebele, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, K.-K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heymach, J. V.</span></span> <span> </span><span class="NLM_article-title">Mechanisms and clinical activity of an EGFR and HER2 exon 20-selective kinase inhibitor in non-small cell lung cancer</span>. <i>Nat. Med.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">638</span>– <span class="NLM_lpage">646</span>, <span class="refDoi"> DOI: 10.1038/s41591-018-0007-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00870&amp;key=10.1038%2Fs41591-018-0007-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00870&amp;key=29686424" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00870&amp;key=1%3ACAS%3A528%3ADC%252BC1cXosVCnsLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2018&pages=638-646&author=J.+P.+Robichauxauthor=Y.+Y.+Elaminauthor=Z.+Tanauthor=B.+W.+Carterauthor=S.+Zhangauthor=S.+Liuauthor=S.+Liauthor=T.+Chenauthor=A.+Poteeteauthor=A.+Estrada-Bernalauthor=A.+T.+Leauthor=A.+Truiniauthor=M.+B.+Nilssonauthor=H.+Sunauthor=E.+Roartyauthor=S.+B.+Goldbergauthor=J.+R.+Brahmerauthor=M.+Altanauthor=C.+Luauthor=V.+Papadimitrakopoulouauthor=K.+Politiauthor=R.+C.+Doebeleauthor=K.-K.+Wongauthor=J.+V.+Heymach&title=Mechanisms+and+clinical+activity+of+an+EGFR+and+HER2+exon+20-selective+kinase+inhibitor+in+non-small+cell+lung+cancer&doi=10.1038%2Fs41591-018-0007-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Mechanisms and clinical activity of an EGFR and HER2 exon 20-selective kinase inhibitor in non-small cell lung cancer</span></div><div class="casAuthors">Robichaux, Jacqulyne P.; Elamin, Yasir Y.; Tan, Zhi; Carter, Brett W.; Zhang, Shuxing; Liu, Shengwu; Li, Shuai; Chen, Ting; Poteete, Alissa; Estrada-Bernal, Adriana; Le, Anh T.; Truini, Anna; Nilsson, Monique B.; Sun, Huiying; Roarty, Emily; Goldberg, Sarah B.; Brahmer, Julie R.; Altan, Mehmet; Lu, Charles; Papadimitrakopoulou, Vassiliki; Politi, Katerina; Doebele, Robert C.; Wong, Kwok-Kin; Heymach, John V.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Medicine (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">638-646</span>CODEN:
                <span class="NLM_cas:coden">NAMEFI</span>;
        ISSN:<span class="NLM_cas:issn">1078-8956</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Although most activating mutations of epidermal growth factor receptor (EGFR)-mutant non-small cell lung cancers (NSCLCs) are sensitive to available EGFR tyrosine kinase inhibitors (TKIs), a subset with alterations in exon 20 of EGFR and HER2 are intrinsically resistant and lack an effective therapy.  We used in silico, in vitro, and in vivo testing to model structural alterations induced by exon 20 mutations and to identify effective inhibitors. 3D modeling indicated alterations restricted the size of the drug-binding pocket, limiting the binding of large, rigid inhibitors.  We found that poziotinib, owing to its small size and flexibility, can circumvent these steric changes and is a potent inhibitor of the most common EGFR and HER2 exon 20 mutants.  Poziotinib demonstrated greater activity than approved EGFR TKIs in vitro and in patient-derived xenograft models of EGFR or HER2 exon 20 mutant NSCLC and in genetically engineered mouse models of NSCLC.  In a phase 2 trial, the first 11 patients with NSCLC with EGFR exon 20 mutations receiving poziotinib had a confirmed objective response rate of 64%.  These data identify poziotinib as a potent, clin. active inhibitor of EGFR and HER2 exon 20 mutations and illuminate the mol. features of TKIs that may circumvent steric changes induced by these mutations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrk6UyVpv0h47Vg90H21EOLACvtfcHk0lj_P5wJoGssQQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXosVCnsLs%253D&md5=316a0a0765ebc9eec9b6460d27b28bb5</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1038%2Fs41591-018-0007-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41591-018-0007-9%26sid%3Dliteratum%253Aachs%26aulast%3DRobichaux%26aufirst%3DJ.%2BP.%26aulast%3DElamin%26aufirst%3DY.%2BY.%26aulast%3DTan%26aufirst%3DZ.%26aulast%3DCarter%26aufirst%3DB.%2BW.%26aulast%3DZhang%26aufirst%3DS.%26aulast%3DLiu%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DS.%26aulast%3DChen%26aufirst%3DT.%26aulast%3DPoteete%26aufirst%3DA.%26aulast%3DEstrada-Bernal%26aufirst%3DA.%26aulast%3DLe%26aufirst%3DA.%2BT.%26aulast%3DTruini%26aufirst%3DA.%26aulast%3DNilsson%26aufirst%3DM.%2BB.%26aulast%3DSun%26aufirst%3DH.%26aulast%3DRoarty%26aufirst%3DE.%26aulast%3DGoldberg%26aufirst%3DS.%2BB.%26aulast%3DBrahmer%26aufirst%3DJ.%2BR.%26aulast%3DAltan%26aufirst%3DM.%26aulast%3DLu%26aufirst%3DC.%26aulast%3DPapadimitrakopoulou%26aufirst%3DV.%26aulast%3DPoliti%26aufirst%3DK.%26aulast%3DDoebele%26aufirst%3DR.%2BC.%26aulast%3DWong%26aufirst%3DK.-K.%26aulast%3DHeymach%26aufirst%3DJ.%2BV.%26atitle%3DMechanisms%2520and%2520clinical%2520activity%2520of%2520an%2520EGFR%2520and%2520HER2%2520exon%252020-selective%2520kinase%2520inhibitor%2520in%2520non-small%2520cell%2520lung%2520cancer%26jtitle%3DNat.%2520Med.%26date%3D2018%26volume%3D24%26spage%3D638%26epage%3D646%26doi%3D10.1038%2Fs41591-018-0007-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Robichaux, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elamin, Y. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vijayan, R. S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nilsson, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pisegna, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poteete, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Negrao, M. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ahnert, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diao, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Le, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meric-Bernstam, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Routbort, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roeck, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raymond, V. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lanman, R. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frampton, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, V. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schrock, A. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Albacker, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, K.-k.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cross, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heymach, J. V.</span></span> <span> </span><span class="NLM_article-title">Pan-cancer landscape and analysis of ERBB2 mutations identifies poziotinib as a clinically active inhibitor and enhancer of T-DM1 activity</span>. <i>Cancer Cell</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>36</i></span>,  <span class="NLM_fpage">444</span>– <span class="NLM_lpage">457</span>, <span class="refDoi"> DOI: 10.1016/j.ccell.2019.09.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00870&amp;key=10.1016%2Fj.ccell.2019.09.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00870&amp;key=31588020" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00870&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhvFersr%252FP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2019&pages=444-457&author=J.+P.+Robichauxauthor=Y.+Y.+Elaminauthor=R.+S.+K.+Vijayanauthor=M.+B.+Nilssonauthor=L.+Huauthor=J.+Heauthor=F.+Zhangauthor=M.+Pisegnaauthor=A.+Poteeteauthor=H.+Sunauthor=S.+Liauthor=T.+Chenauthor=H.+Hanauthor=M.+V.+Negraoauthor=J.+R.+Ahnertauthor=L.+Diaoauthor=J.+Wangauthor=X.+Leauthor=F.+Meric-Bernstamauthor=M.+Routbortauthor=B.+Roeckauthor=Z.+Yangauthor=V.+M.+Raymondauthor=R.+B.+Lanmanauthor=G.+M.+Framptonauthor=V.+A.+Millerauthor=A.+B.+Schrockauthor=L.+A.+Albackerauthor=K.-k.+Wongauthor=J.+B.+Crossauthor=J.+V.+Heymach&title=Pan-cancer+landscape+and+analysis+of+ERBB2+mutations+identifies+poziotinib+as+a+clinically+active+inhibitor+and+enhancer+of+T-DM1+activity&doi=10.1016%2Fj.ccell.2019.09.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Pan-Cancer Landscape and Analysis of ERBB2 Mutations Identifies Poziotinib as a Clinically Active Inhibitor and Enhancer of T-DM1 Activity</span></div><div class="casAuthors">Robichaux, Jacqulyne P.; Elamin, Yasir Y.; Vijayan, R. S. K.; Nilsson, Monique B.; Hu, Lemei; He, Junqin; Zhang, Fahao; Pisegna, Marlese; Poteete, Alissa; Sun, Huiying; Li, Shuai; Chen, Ting; Han, Han; Vailati Negrao, Marcelo; Ahnert, Jordi Rodon; Diao, Lixia; Wang, Jing; Le, Xiuning; Meric-Bernstam, Funda; Routbort, Mark; Roeck, Brent; Yang, Zane; Raymond, Victoria M.; Lanman, Richard B.; Frampton, Garrett M.; Miller, Vincent A.; Schrock, Alexa B.; Albacker, Lee A.; Wong, Kwok-kin; Cross, Jason B.; Heymach, John V.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">444-457.e7</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">We characterized the landscape and drug sensitivity of ERBB2 (HER2) mutations in cancers.  In 11 datasets (n = 211,726), ERBB2 mutational hotspots varied across 25 tumor types.  Common HER2 mutants yielded differential sensitivities to eleven EGFR/HER2 tyrosine kinase inhibitors (TKIs) in vitro, and mol. dynamics simulations revealed that mutants with a reduced drug-binding pocket vol. were assocd. with decreased affinity for larger TKIs.  Overall, poziotinib was the most potent HER2 mutant-selective TKI tested.  Phase II clin. testing in ERBB2 exon 20-mutant non-small cell lung cancer resulted in a confirmed objective response rate of 42% in the first 12 evaluable patients.  In pre-clin. models, poziotinib upregulated HER2 cell-surface expression and potentiated the activity of T-DM1, resulting in complete tumor regression with combination treatment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpiTO9bz4Pd_rVg90H21EOLACvtfcHk0lj_P5wJoGssQQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhvFersr%252FP&md5=cdcb488a797546e1c0f4ccac85e9e13c</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1016%2Fj.ccell.2019.09.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccell.2019.09.001%26sid%3Dliteratum%253Aachs%26aulast%3DRobichaux%26aufirst%3DJ.%2BP.%26aulast%3DElamin%26aufirst%3DY.%2BY.%26aulast%3DVijayan%26aufirst%3DR.%2BS.%2BK.%26aulast%3DNilsson%26aufirst%3DM.%2BB.%26aulast%3DHu%26aufirst%3DL.%26aulast%3DHe%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DF.%26aulast%3DPisegna%26aufirst%3DM.%26aulast%3DPoteete%26aufirst%3DA.%26aulast%3DSun%26aufirst%3DH.%26aulast%3DLi%26aufirst%3DS.%26aulast%3DChen%26aufirst%3DT.%26aulast%3DHan%26aufirst%3DH.%26aulast%3DNegrao%26aufirst%3DM.%2BV.%26aulast%3DAhnert%26aufirst%3DJ.%2BR.%26aulast%3DDiao%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DLe%26aufirst%3DX.%26aulast%3DMeric-Bernstam%26aufirst%3DF.%26aulast%3DRoutbort%26aufirst%3DM.%26aulast%3DRoeck%26aufirst%3DB.%26aulast%3DYang%26aufirst%3DZ.%26aulast%3DRaymond%26aufirst%3DV.%2BM.%26aulast%3DLanman%26aufirst%3DR.%2BB.%26aulast%3DFrampton%26aufirst%3DG.%2BM.%26aulast%3DMiller%26aufirst%3DV.%2BA.%26aulast%3DSchrock%26aufirst%3DA.%2BB.%26aulast%3DAlbacker%26aufirst%3DL.%2BA.%26aulast%3DWong%26aufirst%3DK.-k.%26aulast%3DCross%26aufirst%3DJ.%2BB.%26aulast%3DHeymach%26aufirst%3DJ.%2BV.%26atitle%3DPan-cancer%2520landscape%2520and%2520analysis%2520of%2520ERBB2%2520mutations%2520identifies%2520poziotinib%2520as%2520a%2520clinically%2520active%2520inhibitor%2520and%2520enhancer%2520of%2520T-DM1%2520activity%26jtitle%3DCancer%2520Cell%26date%3D2019%26volume%3D36%26spage%3D444%26epage%3D457%26doi%3D10.1016%2Fj.ccell.2019.09.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Smaill, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gonzales, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spicer, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reed, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sexton, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Althaus, I. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Black, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blaser, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Denny, W. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ellis, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fakhoury, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harvey, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hook, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCarthy, F. O. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Palmer, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rivault, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schlosser, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ellis, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thompson, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trachet, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winters, R. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tecle, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bridges, A.</span></span> <span> </span><span class="NLM_article-title">Tyrosine kinase kihibitors. 20. Optimization of substituted quinazoline and pyrido[3,4-d]pyrimidine derivatives as orally active, irreversible inhibitors of the epidermal growth factor receptor family</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">8103</span>– <span class="NLM_lpage">8124</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b00883</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b00883" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00870&amp;key=1%3ACAS%3A528%3ADC%252BC28Xht12gtbvM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=8103-8124&author=J.+B.+Smaillauthor=A.+J.+Gonzalesauthor=J.+A.+Spicerauthor=H.+Leeauthor=J.+E.+Reedauthor=K.+Sextonauthor=I.+W.+Althausauthor=T.+Zhuauthor=S.+L.+Blackauthor=A.+Blaserauthor=W.+A.+Dennyauthor=P.+A.+Ellisauthor=S.+Fakhouryauthor=P.+J.+Harveyauthor=K.+Hookauthor=F.+O.+J.+McCarthyauthor=B.+D.+Palmerauthor=F.+Rivaultauthor=K.+Schlosserauthor=T.+Ellisauthor=A.+M.+Thompsonauthor=E.+Trachetauthor=R.+T.+Wintersauthor=H.+Tecleauthor=A.+Bridges&title=Tyrosine+kinase+kihibitors.+20.+Optimization+of+substituted+quinazoline+and+pyrido%5B3%2C4-d%5Dpyrimidine+derivatives+as+orally+active%2C+irreversible+inhibitors+of+the+epidermal+growth+factor+receptor+family&doi=10.1021%2Facs.jmedchem.6b00883"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Tyrosine Kinase Inhibitors. 20. Optimization of Substituted Quinazoline and Pyrido[3,4-d]pyrimidine Derivatives as Orally Active, Irreversible Inhibitors of the Epidermal Growth Factor Receptor Family</span></div><div class="casAuthors">Smaill, Jeff B.; Gonzales, Andrea J.; Spicer, Julie A.; Lee, Helen; Reed, Jessica E.; Sexton, Karen; Althaus, Irene W.; Zhu, Tong; Black, Shannon L.; Blaser, Adrian; Denny, William A.; Ellis, Paul A.; Fakhoury, Stephen; Harvey, Patricia J.; Hook, Ken; McCarthy, Florence O. J.; Palmer, Brian D.; Rivault, Freddy; Schlosser, Kevin; Ellis, Teresa; Thompson, Andrew M.; Trachet, Erin; Winters, R. Thomas; Tecle, Haile; Bridges, Alexander</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">8103-8124</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Structure-activity relationships for inhibition of erbB1, erbB2 and erbB4 were detd. for a series of quinazoline- and pyrido[3,4-d]pyrimidine-based analogs of the irreversible pan-erbB inhibitor, canertinib.  Cyclic amine bearing crotonamides were detd. to provide rapid inhibition of cellular erbB1 autophosphorylation and good metabolic stability in liver microsome and hepatocyte assays.  The influence of 4-anilino substitution on pan-erbB inhibitory potency was investigated.  Several anilines were identified as providing potent, reversible pan-erbB inhibition.  Optimum 4- and 6-substituents with known 7-substituents provided preferred irreversible inhibitors for pharmacodynamic testing in vivo.  Quinazoline I and pyrido[3,4-d]pyrimidine II were identified as clearly superior to canertinib.  Both compds. possess a piperidinyl crotonamide Michael acceptor and a 3-chloro-4-fluoroaniline, indicating these as optimized 6- and 4-substituents resp.  Pharmacokinetic comparison of compds. I and II across three species selected compd. I as the preferred candidate.  Compd. I (PF-00299804) has been assigned the nomenclature of dacomitinib and is currently under clin. evaluation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo0-NpOmVq10bVg90H21EOLACvtfcHk0lhN7QuJOmV6Kw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xht12gtbvM&md5=442afda669a5c1b7a190cd82b77bee33</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b00883&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b00883%26sid%3Dliteratum%253Aachs%26aulast%3DSmaill%26aufirst%3DJ.%2BB.%26aulast%3DGonzales%26aufirst%3DA.%2BJ.%26aulast%3DSpicer%26aufirst%3DJ.%2BA.%26aulast%3DLee%26aufirst%3DH.%26aulast%3DReed%26aufirst%3DJ.%2BE.%26aulast%3DSexton%26aufirst%3DK.%26aulast%3DAlthaus%26aufirst%3DI.%2BW.%26aulast%3DZhu%26aufirst%3DT.%26aulast%3DBlack%26aufirst%3DS.%2BL.%26aulast%3DBlaser%26aufirst%3DA.%26aulast%3DDenny%26aufirst%3DW.%2BA.%26aulast%3DEllis%26aufirst%3DP.%2BA.%26aulast%3DFakhoury%26aufirst%3DS.%26aulast%3DHarvey%26aufirst%3DP.%2BJ.%26aulast%3DHook%26aufirst%3DK.%26aulast%3DMcCarthy%26aufirst%3DF.%2BO.%2BJ.%26aulast%3DPalmer%26aufirst%3DB.%2BD.%26aulast%3DRivault%26aufirst%3DF.%26aulast%3DSchlosser%26aufirst%3DK.%26aulast%3DEllis%26aufirst%3DT.%26aulast%3DThompson%26aufirst%3DA.%2BM.%26aulast%3DTrachet%26aufirst%3DE.%26aulast%3DWinters%26aufirst%3DR.%2BT.%26aulast%3DTecle%26aufirst%3DH.%26aulast%3DBridges%26aufirst%3DA.%26atitle%3DTyrosine%2520kinase%2520kihibitors.%252020.%2520Optimization%2520of%2520substituted%2520quinazoline%2520and%2520pyrido%255B3%252C4-d%255Dpyrimidine%2520derivatives%2520as%2520orally%2520active%252C%2520irreversible%2520inhibitors%2520of%2520the%2520epidermal%2520growth%2520factor%2520receptor%2520family%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D8103%26epage%3D8124%26doi%3D10.1021%2Facs.jmedchem.6b00883" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gonzales, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hook, K. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Althaus, I. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ellis, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trachet, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Delaney, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harvey, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ellis, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amato, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nelson, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fry, D. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loi, C.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fakhoury, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schlosser, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sexton, K. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winters, R. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reed, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bridges, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lettiere, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baker, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, H. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tecle, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vincent, P. W.</span></span> <span> </span><span class="NLM_article-title">Antitumor activity and pharmacokinetic properties of PF-00299804, a second-generation irreversible pan-erbB receptor tyrosine kinase inhibitor</span>. <i>Mol. Cancer Ther.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">1880</span>– <span class="NLM_lpage">1889</span>, <span class="refDoi"> DOI: 10.1158/1535-7163.mct-07-2232</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00870&amp;key=10.1158%2F1535-7163.MCT-07-2232" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00870&amp;key=18606718" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00870&amp;key=1%3ACAS%3A528%3ADC%252BD1cXoslCgtb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2008&pages=1880-1889&author=A.+J.+Gonzalesauthor=K.+E.+Hookauthor=I.+W.+Althausauthor=P.+A.+Ellisauthor=E.+Trachetauthor=A.+M.+Delaneyauthor=P.+J.+Harveyauthor=T.+A.+Ellisauthor=D.+M.+Amatoauthor=J.+M.+Nelsonauthor=D.+W.+Fryauthor=T.+Zhuauthor=C.-M.+Loiauthor=S.+A.+Fakhouryauthor=K.+M.+Schlosserauthor=K.+E.+Sextonauthor=R.+T.+Wintersauthor=J.+E.+Reedauthor=A.+J.+Bridgesauthor=D.+J.+Lettiereauthor=D.+A.+Bakerauthor=J.+Yangauthor=H.+T.+Leeauthor=H.+Tecleauthor=P.+W.+Vincent&title=Antitumor+activity+and+pharmacokinetic+properties+of+PF-00299804%2C+a+second-generation+irreversible+pan-erbB+receptor+tyrosine+kinase+inhibitor&doi=10.1158%2F1535-7163.mct-07-2232"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Antitumor activity and pharmacokinetic properties of PF-00299804, a second-generation irreversible pan-erbB receptor tyrosine kinase inhibitor</span></div><div class="casAuthors">Gonzales, Andrea J.; Hook, Kenneth E.; Althaus, Irene W.; Ellis, Paul A.; Trachet, Erin; Delaney, Amy M.; Harvey, Patricia J.; Ellis, Teresa A.; Amato, Danielle M.; Nelson, James M.; Fry, David W.; Zhu, Tong; Loi, Cho-Ming; Fakhoury, Stephen A.; Schlosser, Kevin M.; Sexton, Karen E.; Winters, R. Thomas; Reed, Jessica E.; Bridges, Alex J.; Lettiere, Daniel J.; Baker, Deborah A.; Yang, Jianxin; Lee, Helen T.; Tecle, Haile; Vincent, Patrick W.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cancer Therapeutics</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1880-1889</span>CODEN:
                <span class="NLM_cas:coden">MCTOCF</span>;
        ISSN:<span class="NLM_cas:issn">1535-7163</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Signaling through the erbB receptor family of tyrosine kinases contributes to the proliferation, differentiation, migration, and survival of a variety of cell types.  Abnormalities in members of this receptor family have been shown to play a role in oncogenesis, thus making them attractive targets for anticancer treatments.  PF-00299804 is a second-generation irreversible pan-erbB receptor tyrosine kinase inhibitor currently in phase I clin. trials.  PF-00299804 is believed to irreversibly inhibit erbB tyrosine kinase activity through binding at the ATP site and covalent modification of nucleophilic cysteine residues in the catalytic domains of erbB family members.  Oral administration of PF-00299804 causes significant antitumor activity, including marked tumor regressions in a variety of human tumor xenograft models that express and/or overexpress erbB family members or contain the double mutation (L858R/T790M) in erbB1 (EGFR) assocd. with resistance to gefitinib and erlotinib.  Furthermore, PF-00299804 shows exceptional distribution to human tumor xenografts and excellent pharmacokinetic properties across species. [Mol Cancer Ther 2008;7(7):1880-9].</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpbL0hP8K2G37Vg90H21EOLACvtfcHk0lhN7QuJOmV6Kw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXoslCgtb0%253D&md5=3392275d1e4f4d5d70e3b22d4893e3c8</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1158%2F1535-7163.MCT-07-2232&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1535-7163.MCT-07-2232%26sid%3Dliteratum%253Aachs%26aulast%3DGonzales%26aufirst%3DA.%2BJ.%26aulast%3DHook%26aufirst%3DK.%2BE.%26aulast%3DAlthaus%26aufirst%3DI.%2BW.%26aulast%3DEllis%26aufirst%3DP.%2BA.%26aulast%3DTrachet%26aufirst%3DE.%26aulast%3DDelaney%26aufirst%3DA.%2BM.%26aulast%3DHarvey%26aufirst%3DP.%2BJ.%26aulast%3DEllis%26aufirst%3DT.%2BA.%26aulast%3DAmato%26aufirst%3DD.%2BM.%26aulast%3DNelson%26aufirst%3DJ.%2BM.%26aulast%3DFry%26aufirst%3DD.%2BW.%26aulast%3DZhu%26aufirst%3DT.%26aulast%3DLoi%26aufirst%3DC.-M.%26aulast%3DFakhoury%26aufirst%3DS.%2BA.%26aulast%3DSchlosser%26aufirst%3DK.%2BM.%26aulast%3DSexton%26aufirst%3DK.%2BE.%26aulast%3DWinters%26aufirst%3DR.%2BT.%26aulast%3DReed%26aufirst%3DJ.%2BE.%26aulast%3DBridges%26aufirst%3DA.%2BJ.%26aulast%3DLettiere%26aufirst%3DD.%2BJ.%26aulast%3DBaker%26aufirst%3DD.%2BA.%26aulast%3DYang%26aufirst%3DJ.%26aulast%3DLee%26aufirst%3DH.%2BT.%26aulast%3DTecle%26aufirst%3DH.%26aulast%3DVincent%26aufirst%3DP.%2BW.%26atitle%3DAntitumor%2520activity%2520and%2520pharmacokinetic%2520properties%2520of%2520PF-00299804%252C%2520a%2520second-generation%2520irreversible%2520pan-erbB%2520receptor%2520tyrosine%2520kinase%2520inhibitor%26jtitle%3DMol.%2520Cancer%2520Ther.%26date%3D2008%26volume%3D7%26spage%3D1880%26epage%3D1889%26doi%3D10.1158%2F1535-7163.mct-07-2232" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rabindran, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Discafani, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosfjord, E. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baxter, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Floyd, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Golas, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hallett, W. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nilakantan, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Overbeek, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reich, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsou, H.-R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.-F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wissner, A.</span></span> <span> </span><span class="NLM_article-title">Antitumor activity of HKI-272, an orally active, irreversible inhibitor of the HER-2 tyrosine kinase</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>64</i></span>,  <span class="NLM_fpage">3958</span>– <span class="NLM_lpage">3965</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.can-03-2868</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00870&amp;key=10.1158%2F0008-5472.CAN-03-2868" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00870&amp;key=15173008" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00870&amp;key=1%3ACAS%3A528%3ADC%252BD2cXksVKns7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=64&publication_year=2004&pages=3958-3965&author=S.+K.+Rabindranauthor=C.+M.+Discafaniauthor=E.+C.+Rosfjordauthor=M.+Baxterauthor=M.+B.+Floydauthor=J.+Golasauthor=W.+A.+Hallettauthor=B.+D.+Johnsonauthor=R.+Nilakantanauthor=E.+Overbeekauthor=M.+F.+Reichauthor=R.+Shenauthor=X.+Shiauthor=H.-R.+Tsouauthor=Y.-F.+Wangauthor=A.+Wissner&title=Antitumor+activity+of+HKI-272%2C+an+orally+active%2C+irreversible+inhibitor+of+the+HER-2+tyrosine+kinase&doi=10.1158%2F0008-5472.can-03-2868"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Antitumor Activity of HKI-272, an Orally Active, Irreversible Inhibitor of the HER-2 Tyrosine Kinase</span></div><div class="casAuthors">Rabindran, Sridhar K.; Discafani, Carolyn M.; Rosfjord, Edward C.; Baxter, Michelle; Floyd, M. Brawner; Golas, Jonathan; Hallett, William A.; Johnson, Bernard D.; Nilakantan, Ramaswamy; Overbeek, Elsebe; Reich, Marvin F.; Shen, Ru; Shi, Xiaoqing; Tsou, Hwei-Ru; Wang, Yu-Fen; Wissner, Allan</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">64</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">3958-3965</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">HER-2 belongs to the ErbB family of receptor tyrosine kinases, which has been implicated in a variety of cancers.  Overexpression of HER-2 is seen in 25-30% of breast cancer patients and predicts a poor outcome in patients with primary disease.  Trastuzumab (Herceptin), a monoclonal antibody to HER-2, is specifically approved for HER-2-pos. breast cancer but is active only in a subset of these tumors.  Blocking HER-2 function by a small mol. kinase inhibitor, therefore, represents an attractive alternate strategy to inhibit the growth of HER-2-pos. tumors.  HKI-272 is a potent inhibitor of HER-2 and is highly active against HER-2-overexpressing human breast cancer cell lines in vitro.  It also inhibits the epidermal growth factor receptor (EGFR) kinase and the proliferation of EGFR-dependent cells.  HKI-272 reduces HER-2 receptor autophosphorylation in cells at doses consistent with inhibition of cell proliferation and functions as an irreversible binding inhibitor, most likely by targeting a cysteine residue in the ATP-binding pocket of the receptor.  In agreement with the predicted effects of HER-2 inactivation, HKI-272 treatment of cells results in inhibition of downstream signal transduction events and cell cycle regulatory pathways.  This leads to arrest at the G1-S (Gap 1/DNA synthesis)-phase transition of the cell division cycle, ultimately resulting in decreased cell proliferation.  In vivo, HKI-272 is active in HER-2- and EGFR-dependent tumor xenograft models when dosed orally on a once daily schedule.  On the basis of its favorable preclin. pharmacol. profile, HKI-272 has been selected as a candidate for addnl. development as an antitumor agent in breast and other HER-2-dependent cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo2xeLXlAwWvbVg90H21EOLACvtfcHk0ljwsbFmRVZnEA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXksVKns7o%253D&md5=60accd0599b7c489e4c3753d4f39cab4</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-03-2868&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-03-2868%26sid%3Dliteratum%253Aachs%26aulast%3DRabindran%26aufirst%3DS.%2BK.%26aulast%3DDiscafani%26aufirst%3DC.%2BM.%26aulast%3DRosfjord%26aufirst%3DE.%2BC.%26aulast%3DBaxter%26aufirst%3DM.%26aulast%3DFloyd%26aufirst%3DM.%2BB.%26aulast%3DGolas%26aufirst%3DJ.%26aulast%3DHallett%26aufirst%3DW.%2BA.%26aulast%3DJohnson%26aufirst%3DB.%2BD.%26aulast%3DNilakantan%26aufirst%3DR.%26aulast%3DOverbeek%26aufirst%3DE.%26aulast%3DReich%26aufirst%3DM.%2BF.%26aulast%3DShen%26aufirst%3DR.%26aulast%3DShi%26aufirst%3DX.%26aulast%3DTsou%26aufirst%3DH.-R.%26aulast%3DWang%26aufirst%3DY.-F.%26aulast%3DWissner%26aufirst%3DA.%26atitle%3DAntitumor%2520activity%2520of%2520HKI-272%252C%2520an%2520orally%2520active%252C%2520irreversible%2520inhibitor%2520of%2520the%2520HER-2%2520tyrosine%2520kinase%26jtitle%3DCancer%2520Res.%26date%3D2004%26volume%3D64%26spage%3D3958%26epage%3D3965%26doi%3D10.1158%2F0008-5472.can-03-2868" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tsou, H.-R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Overbeek-Klumpers, E. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hallett, W. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reich, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Floyd, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michalak, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nilakantan, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Discafani, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Golas, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rabindran, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.-F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Upeslacis, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wissner, A.</span></span> <span> </span><span class="NLM_article-title">Optimization of 6,7-disubstituted-4-(arylamino)quinoline-3-carbonitriles as orally active, irreversible inhibitors of human epidermal growth factor receptor-2 kinase activity</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>48</i></span>,  <span class="NLM_fpage">1107</span>– <span class="NLM_lpage">1131</span>, <span class="refDoi"> DOI: 10.1021/jm040159c</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm040159c" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00870&amp;key=1%3ACAS%3A528%3ADC%252BD2MXntFKjsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2005&pages=1107-1131&author=H.-R.+Tsouauthor=E.+G.+Overbeek-Klumpersauthor=W.+A.+Hallettauthor=M.+F.+Reichauthor=M.+B.+Floydauthor=B.+D.+Johnsonauthor=R.+S.+Michalakauthor=R.+Nilakantanauthor=C.+Discafaniauthor=J.+Golasauthor=S.+K.+Rabindranauthor=R.+Shenauthor=X.+Shiauthor=Y.-F.+Wangauthor=J.+Upeslacisauthor=A.+Wissner&title=Optimization+of+6%2C7-disubstituted-4-%28arylamino%29quinoline-3-carbonitriles+as+orally+active%2C+irreversible+inhibitors+of+human+epidermal+growth+factor+receptor-2+kinase+activity&doi=10.1021%2Fjm040159c"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Optimization of 6,7-Disubstituted-4-(arylamino)quinoline-3-carbonitriles as Orally Active, Irreversible Inhibitors of Human Epidermal Growth Factor Receptor-2 Kinase Activity</span></div><div class="casAuthors">Tsou, Hwei-Ru; Overbeek-Klumpers, Elsebe G.; Hallett, William A.; Reich, Marvin F.; Floyd, M. Brawner; Johnson, Bernard D.; Michalak, Ronald S.; Nilakantan, Ramaswamy; Discafani, Carolyn; Golas, Jonathan; Rabindran, Sridhar K.; Shen, Ru; Shi, Xiaoqing; Wang, Yu-Fen; Upeslacis, Janis; Wissner, Allan</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1107-1131</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A series of new 6,7-disubstituted-4-(arylamino)quinoline-3-carbonitriles, e.g. I (R1 = H, Cl; R2 = PhCH2O, 1-imidazolyl, 2-furylmethoxy, etc.; R3 = Cl, CN, PhCH2O; R4 = Me, Et; R5 = Me, R6 = Me, HOCH2CH2; R5R6N = azetidinyl, piperidinyl, thiomorpholinyl, etc.) that function as irreversible inhibitors of human epidermal growth factor receptor-2 (HER-2) and epidermal growth factor receptor (EGFR) kinases have been prepd.  These compds. demonstrated enhanced activities for inhibiting HER-2 kinase and the growth of HER-2 pos. cells compared to the EGFR kinase inhibitor I [R1 = H; R2 = F; R3 = Cl; R4 = Et; R5 = R6 = Me; (EKB-569)].  Three synthetic routes were used to prep. these compds.  They were prepd. mostly by acylation of 6-amino-4-(arylamino)quinoline-3-carbonitriles with unsatd. acid chlorides or by amination of 4-chloro-6-(crotonamido)quinoline-3-carbonitriles with monocyclic or bicyclic anilines.  The third route was developed to prep. a key intermediate, 6-acetamido-4-chloroquinoline-3-carbonitrile, that involved a safer cyclization step.  It was shown that attaching a large lipophilic group at the para position of the 4-(arylamino) ring results in improved potency for inhibiting HER-2 kinase.  The importance of a basic dialkylamino group at the end of the Michael acceptor for activity, due to intramol. catalysis of the Michael addn. has also been demonstrated.  This, along with improved water soly., resulted in compds. with enhanced biol. properties.  The mol. modeling results consistent with the proposed mechanism of inhibition are presented.  Binding studies of one compd., I [R1 = H; R2 = 2-pyridylmethoxy; R3 = Cl; R4 = Et; R5 = R6 = Me; (HKI-272)] (C-14 radiolabeled), showed that it binds irreversibly to HER-2 protein in BT474 cells.  Furthermore, it demonstrated excellent oral activity, esp. in HER-2 overexpressing xenografts.  Compd. HKI-272 was selected for further studies and is currently in phase I clin. trials for the treatment of cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqD0QL4-qQZCbVg90H21EOLACvtfcHk0lgXKO244HdBvg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXntFKjsA%253D%253D&md5=55f77e41bf920f738ae2750e7ca0eb22</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1021%2Fjm040159c&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm040159c%26sid%3Dliteratum%253Aachs%26aulast%3DTsou%26aufirst%3DH.-R.%26aulast%3DOverbeek-Klumpers%26aufirst%3DE.%2BG.%26aulast%3DHallett%26aufirst%3DW.%2BA.%26aulast%3DReich%26aufirst%3DM.%2BF.%26aulast%3DFloyd%26aufirst%3DM.%2BB.%26aulast%3DJohnson%26aufirst%3DB.%2BD.%26aulast%3DMichalak%26aufirst%3DR.%2BS.%26aulast%3DNilakantan%26aufirst%3DR.%26aulast%3DDiscafani%26aufirst%3DC.%26aulast%3DGolas%26aufirst%3DJ.%26aulast%3DRabindran%26aufirst%3DS.%2BK.%26aulast%3DShen%26aufirst%3DR.%26aulast%3DShi%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DY.-F.%26aulast%3DUpeslacis%26aufirst%3DJ.%26aulast%3DWissner%26aufirst%3DA.%26atitle%3DOptimization%2520of%25206%252C7-disubstituted-4-%2528arylamino%2529quinoline-3-carbonitriles%2520as%2520orally%2520active%252C%2520irreversible%2520inhibitors%2520of%2520human%2520epidermal%2520growth%2520factor%2520receptor-2%2520kinase%2520activity%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2005%26volume%3D48%26spage%3D1107%26epage%3D1131%26doi%3D10.1021%2Fjm040159c" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wissner, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Overbeek, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reich, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Floyd, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mamuya, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosfjord, E. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Discafani, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rabindran, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gruber, B. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ye, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hallett, W. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nilakantan, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.-F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greenberger, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsou, H.-R.</span></span> <span> </span><span class="NLM_article-title">Synthesis and structure-activity relationships of 6,7-disubstituted 4-anilinoquinoline-3-carbonitriles. The design of an orally active, irreversible inhibitor of the tyrosine kinase activity of the epidermal growth factor receptor (EGFR) and the human epidermal growth factor receptor-2 (HER-2)</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>46</i></span>,  <span class="NLM_fpage">49</span>– <span class="NLM_lpage">63</span>, <span class="refDoi"> DOI: 10.1021/jm020241c</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm020241c" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00870&amp;key=1%3ACAS%3A528%3ADC%252BD38XptFOlsbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2003&pages=49-63&author=A.+Wissnerauthor=E.+Overbeekauthor=M.+F.+Reichauthor=M.+B.+Floydauthor=B.+D.+Johnsonauthor=N.+Mamuyaauthor=E.+C.+Rosfjordauthor=C.+Discafaniauthor=R.+Davisauthor=X.+Shiauthor=S.+K.+Rabindranauthor=B.+C.+Gruberauthor=F.+Yeauthor=W.+A.+Hallettauthor=R.+Nilakantanauthor=R.+Shenauthor=Y.-F.+Wangauthor=L.+M.+Greenbergerauthor=H.-R.+Tsou&title=Synthesis+and+structure-activity+relationships+of+6%2C7-disubstituted+4-anilinoquinoline-3-carbonitriles.+The+design+of+an+orally+active%2C+irreversible+inhibitor+of+the+tyrosine+kinase+activity+of+the+epidermal+growth+factor+receptor+%28EGFR%29+and+the+human+epidermal+growth+factor+receptor-2+%28HER-2%29&doi=10.1021%2Fjm020241c"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis and Structure-Activity Relationships of 6,7-Disubstituted 4-Anilinoquinoline-3-carbonitriles. The Design of an Orally Active, Irreversible Inhibitor of the Tyrosine Kinase Activity of the Epidermal Growth Factor Receptor (EGFR) and the Human Epidermal Growth Factor Receptor-2 (HER-2)</span></div><div class="casAuthors">Wissner, Allan; Overbeek, Elsebe; Reich, Marvin F.; Floyd, M. Brawner; Johnson, Bernard D.; Mamuya, Nellie; Rosfjord, Edward C.; Discafani, Carolyn; Davis, Rachel; Shi, Xiaoqing; Rabindran, Sridhar K.; Gruber, Brian C.; Ye, Fei; Hallett, William A.; Nilakantan, Ramaswamy; Shen, Ru; Wang, Yu-Fen; Greenberger, Lee M.; Tsou, Hwei-Ru</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">49-63</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A series of 6,7-disubstituted 4-anilinoquinoline-3-carbonitriles, e.g., I, that function as irreversible inhibitors of EGFR and HER-2 kinases have been prepd.  These inhibitors have, at the 6-position, butynamide, crotonamide, and methacrylamide Michael acceptors bearing water-solubilizing substituents.  These compds. were prepd. by acylation of 6-amino-4-(arylamino)quinoline-3-carbonitriles, e.g., II, with unsatd. acid chlorides or mixed anhydrides.  Competitive reactivity studies were performed, showing that attaching a dialkylamino group onto the end of the Michael acceptor results in compds. with greater reactivity due to intramol. catalysis of the Michael addn.  This, along with improved water-soly. results in compds. with enhanced biol. properties.  Mol. modeling results consistent with the proposed mechanism of inhibition are presented.  One compd., I (EKB-569), which shows excellent oral in vivo activity, was selected for further studies and is currently in phase I clin. trials for the treatment of cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq317fAvBYUW7Vg90H21EOLACvtfcHk0lhl_DuQ8uIJpA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XptFOlsbk%253D&md5=db70c2f78c62824810854295d06ef6a4</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1021%2Fjm020241c&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm020241c%26sid%3Dliteratum%253Aachs%26aulast%3DWissner%26aufirst%3DA.%26aulast%3DOverbeek%26aufirst%3DE.%26aulast%3DReich%26aufirst%3DM.%2BF.%26aulast%3DFloyd%26aufirst%3DM.%2BB.%26aulast%3DJohnson%26aufirst%3DB.%2BD.%26aulast%3DMamuya%26aufirst%3DN.%26aulast%3DRosfjord%26aufirst%3DE.%2BC.%26aulast%3DDiscafani%26aufirst%3DC.%26aulast%3DDavis%26aufirst%3DR.%26aulast%3DShi%26aufirst%3DX.%26aulast%3DRabindran%26aufirst%3DS.%2BK.%26aulast%3DGruber%26aufirst%3DB.%2BC.%26aulast%3DYe%26aufirst%3DF.%26aulast%3DHallett%26aufirst%3DW.%2BA.%26aulast%3DNilakantan%26aufirst%3DR.%26aulast%3DShen%26aufirst%3DR.%26aulast%3DWang%26aufirst%3DY.-F.%26aulast%3DGreenberger%26aufirst%3DL.%2BM.%26aulast%3DTsou%26aufirst%3DH.-R.%26atitle%3DSynthesis%2520and%2520structure-activity%2520relationships%2520of%25206%252C7-disubstituted%25204-anilinoquinoline-3-carbonitriles.%2520The%2520design%2520of%2520an%2520orally%2520active%252C%2520irreversible%2520inhibitor%2520of%2520the%2520tyrosine%2520kinase%2520activity%2520of%2520the%2520epidermal%2520growth%2520factor%2520receptor%2520%2528EGFR%2529%2520and%2520the%2520human%2520epidermal%2520growth%2520factor%2520receptor-2%2520%2528HER-2%2529%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2003%26volume%3D46%26spage%3D49%26epage%3D63%26doi%3D10.1021%2Fjm020241c" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xia, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mullin, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keith, B. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rusnak, D. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Owens, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alligood, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spector, N. L.</span></span> <span> </span><span class="NLM_article-title">Anti-tumor activity of GW572016: a dual tyrosine kinase inhibitor blocks EGF activation of EGFR/erbB2 and downstream Erk1/2 and AKT pathways</span>. <i>Oncogene</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">6255</span>– <span class="NLM_lpage">6263</span>, <span class="refDoi"> DOI: 10.1038/sj.onc.1205794</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00870&amp;key=10.1038%2Fsj.onc.1205794" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00870&amp;key=12214266" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00870&amp;key=1%3ACAS%3A528%3ADC%252BD38Xms1yrs7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2002&pages=6255-6263&author=W.+Xiaauthor=R.+J.+Mullinauthor=B.+R.+Keithauthor=L.-H.+Liuauthor=H.+Maauthor=D.+W.+Rusnakauthor=G.+Owensauthor=K.+J.+Alligoodauthor=N.+L.+Spector&title=Anti-tumor+activity+of+GW572016%3A+a+dual+tyrosine+kinase+inhibitor+blocks+EGF+activation+of+EGFR%2FerbB2+and+downstream+Erk1%2F2+and+AKT+pathways&doi=10.1038%2Fsj.onc.1205794"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Anti-tumor activity of GW572016: a dual tyrosine kinase inhibitor blocks EGF activation of EGFR/erbB2 and downstream Erk1/2 and AKT pathways</span></div><div class="casAuthors">Xia, Wenle; Mullin, Robert J.; Keith, Barry R.; Liu, Lei-Hua; Ma, Hong; Rusnak, David W.; Owens, Gary; Alligood, Krystal J.; Spector, Neil L.</div><div class="citationInfo"><span class="NLM_cas:title">Oncogene</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">41</span>),
    <span class="NLM_cas:pages">6255-6263</span>CODEN:
                <span class="NLM_cas:coden">ONCNES</span>;
        ISSN:<span class="NLM_cas:issn">0950-9232</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Dual EGFR/erbB2 inhibition is an attractive therapeutic strategy for epithelial tumors, as ligand-induced erbB2/EGFR heterodimerization triggers potent proliferative and survival signals.  Here we show that a small mol., GW572016, potently inhibits both EGFR and erbB2 tyrosine kinases leading to growth arrest and/or apoptosis in EGFR and erbB2-dependent tumor cell lines.  GW572016 markedly reduced tyrosine phosphorylation of EGFR and erbB2, and inhibited activation of Erk1/2 and AKT, downstream effectors of proliferation and cell survival, resp.  Complete inhibition of activated AKT in erbB2 overexpressing cells correlated with a 23-fold increase in apoptosis compared with vehicle controls.  EGF, often elevated in cancer patients, did not reverse the inhibitory effects of GW572016.  These observations were reproduced in vivo, where GW572016 treatment inhibited activation of EGFR, erbB2, Erk1/2 and AKT in human tumor xenografts.  Erk1/2 and AKT represent potential biomarkers to assess the clin. activity of GW572016.  Inhibition of activated AKT in EGFR or erbB2-dependent tumors by GW572016 may lead to tumor regressions when used as a monotherapy, or may enhance the anti-tumor activity of chemotherapeutics, since constitutive activation of AKT has been linked to chemo-resistance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqYPkZ4sa88irVg90H21EOLACvtfcHk0lhl_DuQ8uIJpA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38Xms1yrs7w%253D&md5=38d6e25a7cfa1fdfe77fa2fea2c0aa0c</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1038%2Fsj.onc.1205794&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.onc.1205794%26sid%3Dliteratum%253Aachs%26aulast%3DXia%26aufirst%3DW.%26aulast%3DMullin%26aufirst%3DR.%2BJ.%26aulast%3DKeith%26aufirst%3DB.%2BR.%26aulast%3DLiu%26aufirst%3DL.-H.%26aulast%3DMa%26aufirst%3DH.%26aulast%3DRusnak%26aufirst%3DD.%2BW.%26aulast%3DOwens%26aufirst%3DG.%26aulast%3DAlligood%26aufirst%3DK.%2BJ.%26aulast%3DSpector%26aufirst%3DN.%2BL.%26atitle%3DAnti-tumor%2520activity%2520of%2520GW572016%253A%2520a%2520dual%2520tyrosine%2520kinase%2520inhibitor%2520blocks%2520EGF%2520activation%2520of%2520EGFR%252FerbB2%2520and%2520downstream%2520Erk1%252F2%2520and%2520AKT%2520pathways%26jtitle%3DOncogene%26date%3D2002%26volume%3D21%26spage%3D6255%26epage%3D6263%26doi%3D10.1038%2Fsj.onc.1205794" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Heymach, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Negrao, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robichaux, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carter, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Altan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gibbons, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fossella, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simon, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lam, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blumenschein, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsao, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kurie, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mott, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jenkins, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mack, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roeck, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Papadimitrakopoulou, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elamin, Y.</span></span> <span> </span><span class="NLM_article-title">OA02.06 A phase II trial of poziotinib in EGFR and HER2 exon 20 mutant non-small cell lung cancer (NSCLC)</span>. <i>J. Thorac. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">S323</span>– <span class="NLM_lpage">S324</span>, <span class="refDoi"> DOI: 10.1016/j.jtho.2018.08.243</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00870&amp;key=10.1016%2Fj.jtho.2018.08.243" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00870&amp;key=29396253" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2018&pages=S323-S324&author=J.+Heymachauthor=M.+Negraoauthor=J.+Robichauxauthor=B.+Carterauthor=A.+Patelauthor=M.+Altanauthor=D.+Gibbonsauthor=F.+Fossellaauthor=G.+Simonauthor=V.+Lamauthor=G.+Blumenscheinauthor=A.+Tsaoauthor=J.+Kurieauthor=F.+Mottauthor=D.+Jenkinsauthor=D.+Mackauthor=L.+Fengauthor=B.+Roeckauthor=Z.+Yangauthor=V.+Papadimitrakopoulouauthor=Y.+Elamin&title=OA02.06+A+phase+II+trial+of+poziotinib+in+EGFR+and+HER2+exon+20+mutant+non-small+cell+lung+cancer+%28NSCLC%29&doi=10.1016%2Fj.jtho.2018.08.243"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1016%2Fj.jtho.2018.08.243&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jtho.2018.08.243%26sid%3Dliteratum%253Aachs%26aulast%3DHeymach%26aufirst%3DJ.%26aulast%3DNegrao%26aufirst%3DM.%26aulast%3DRobichaux%26aufirst%3DJ.%26aulast%3DCarter%26aufirst%3DB.%26aulast%3DPatel%26aufirst%3DA.%26aulast%3DAltan%26aufirst%3DM.%26aulast%3DGibbons%26aufirst%3DD.%26aulast%3DFossella%26aufirst%3DF.%26aulast%3DSimon%26aufirst%3DG.%26aulast%3DLam%26aufirst%3DV.%26aulast%3DBlumenschein%26aufirst%3DG.%26aulast%3DTsao%26aufirst%3DA.%26aulast%3DKurie%26aufirst%3DJ.%26aulast%3DMott%26aufirst%3DF.%26aulast%3DJenkins%26aufirst%3DD.%26aulast%3DMack%26aufirst%3DD.%26aulast%3DFeng%26aufirst%3DL.%26aulast%3DRoeck%26aufirst%3DB.%26aulast%3DYang%26aufirst%3DZ.%26aulast%3DPapadimitrakopoulou%26aufirst%3DV.%26aulast%3DElamin%26aufirst%3DY.%26atitle%3DOA02.06%2520A%2520phase%2520II%2520trial%2520of%2520poziotinib%2520in%2520EGFR%2520and%2520HER2%2520exon%252020%2520mutant%2520non-small%2520cell%2520lung%2520cancer%2520%2528NSCLC%2529%26jtitle%3DJ.%2520Thorac.%2520Oncol.%26date%3D2018%26volume%3D13%26spage%3DS323%26epage%3DS324%26doi%3D10.1016%2Fj.jtho.2018.08.243" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lategahn, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keul, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klövekorn, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tumbrink, H. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Niggenaber, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hodson, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flasshoff, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hardick, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grabe, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Engel, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schultz-Fademrecht, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baumann, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ketzer, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mühlenberg, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hiller, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Günther, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Unger, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heimsoeth, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Golz, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blank-Landeshammer, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kollipara, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zahedi, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Strohmann, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hengstler, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Otterlo, W. A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bauer, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rauh, D.</span></span> <span> </span><span class="NLM_article-title">Inhibition of osimertinib-resistant epidermal growth factor receptor EGFR-T790M/C797S</span>. <i>Chem. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">10789</span>– <span class="NLM_lpage">10801</span>, <span class="refDoi"> DOI: 10.1039/c9sc03445e</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00870&amp;key=10.1039%2FC9SC03445E" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00870&amp;key=31857889" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00870&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhvFSntr7N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2019&pages=10789-10801&author=J.+Lategahnauthor=M.+Keulauthor=P.+Kl%C3%B6vekornauthor=H.+L.+Tumbrinkauthor=J.+Niggenaberauthor=M.+P.+M%C3%BCllerauthor=L.+Hodsonauthor=M.+Flasshoffauthor=J.+Hardickauthor=T.+Grabeauthor=J.+Engelauthor=C.+Schultz-Fademrechtauthor=M.+Baumannauthor=J.+Ketzerauthor=T.+M%C3%BChlenbergauthor=W.+Hillerauthor=G.+G%C3%BCntherauthor=A.+Ungerauthor=H.+M%C3%BCllerauthor=A.+Heimsoethauthor=C.+Golzauthor=B.+Blank-Landeshammerauthor=L.+Kolliparaauthor=R.+P.+Zahediauthor=C.+Strohmannauthor=J.+G.+Hengstlerauthor=W.+A.+L.+van+Otterloauthor=S.+Bauerauthor=D.+Rauh&title=Inhibition+of+osimertinib-resistant+epidermal+growth+factor+receptor+EGFR-T790M%2FC797S&doi=10.1039%2Fc9sc03445e"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of osimertinib-resistant epidermal growth factor receptor EGFR-T790M/C797S</span></div><div class="casAuthors">Lategahn, Jonas; Keul, Marina; Kloevekorn, Philip; Tumbrink, Hannah L.; Niggenaber, Janina; Mueller, Matthias P.; Hodson, Luke; Flasshoff, Maren; Hardick, Julia; Grabe, Tobias; Engel, Julian; Schultz-Fademrecht, Carsten; Baumann, Matthias; Ketzer, Julia; Muehlenberg, Thomas; Hiller, Wolf; Guenther, Georgia; Unger, Anke; Mueller, Heiko; Heimsoeth, Alena; Golz, Christopher; Blank-Landeshammer, Bernhard; Kollipara, Laxmikanth; Zahedi, Rene P.; Strohmann, Carsten; Hengstler, Jan G.; van Otterlo, Willem A. L.; Bauer, Sebastian; Rauh, Daniel</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Science</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">46</span>),
    <span class="NLM_cas:pages">10789-10801</span>CODEN:
                <span class="NLM_cas:coden">CSHCCN</span>;
        ISSN:<span class="NLM_cas:issn">2041-6520</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Precision medicine has revolutionized the treatment of patients in EGFR driven non-small cell lung cancer (NSCLC).  Targeted drugs show high response rates in genetically defined subsets of cancer patients and markedly increase their progression-free survival as compared to conventional chemotherapy.  However, recurrent acquired drug resistance limits the success of targeted drugs in long-term treatment and requires the const. development of novel efficient inhibitors of drug resistant cancer subtypes.  Herein, we present covalent inhibitors of the drug resistant gatekeeper mutant EGFR-L858R/T790M based on the pyrrolopyrimidine scaffold.  Biochem. and cellular characterization, as well as kinase selectivity profiling and western blot anal., substantiate our approach.  Moreover, the developed compds. possess high activity against multi drug resistant EGFR-L858R/T790M/C797S in biochem. assays due to their highly reversible binding character, that was revealed by characterization of the binding kinetics.  In addn., we present the first X-ray crystal structures of covalent inhibitors in complex with C797S-mutated EGFR which provide detailed insight into their binding mode.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqpOILrK_TpebVg90H21EOLACvtfcHk0lht-Bj5SNcVWw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhvFSntr7N&md5=0dd60e9ade21e33ae5656d50175e39ba</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1039%2FC9SC03445E&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC9SC03445E%26sid%3Dliteratum%253Aachs%26aulast%3DLategahn%26aufirst%3DJ.%26aulast%3DKeul%26aufirst%3DM.%26aulast%3DKl%25C3%25B6vekorn%26aufirst%3DP.%26aulast%3DTumbrink%26aufirst%3DH.%2BL.%26aulast%3DNiggenaber%26aufirst%3DJ.%26aulast%3DM%25C3%25BCller%26aufirst%3DM.%2BP.%26aulast%3DHodson%26aufirst%3DL.%26aulast%3DFlasshoff%26aufirst%3DM.%26aulast%3DHardick%26aufirst%3DJ.%26aulast%3DGrabe%26aufirst%3DT.%26aulast%3DEngel%26aufirst%3DJ.%26aulast%3DSchultz-Fademrecht%26aufirst%3DC.%26aulast%3DBaumann%26aufirst%3DM.%26aulast%3DKetzer%26aufirst%3DJ.%26aulast%3DM%25C3%25BChlenberg%26aufirst%3DT.%26aulast%3DHiller%26aufirst%3DW.%26aulast%3DG%25C3%25BCnther%26aufirst%3DG.%26aulast%3DUnger%26aufirst%3DA.%26aulast%3DM%25C3%25BCller%26aufirst%3DH.%26aulast%3DHeimsoeth%26aufirst%3DA.%26aulast%3DGolz%26aufirst%3DC.%26aulast%3DBlank-Landeshammer%26aufirst%3DB.%26aulast%3DKollipara%26aufirst%3DL.%26aulast%3DZahedi%26aufirst%3DR.%2BP.%26aulast%3DStrohmann%26aufirst%3DC.%26aulast%3DHengstler%26aufirst%3DJ.%2BG.%26aulast%3Dvan%2BOtterlo%26aufirst%3DW.%2BA.%2BL.%26aulast%3DBauer%26aufirst%3DS.%26aulast%3DRauh%26aufirst%3DD.%26atitle%3DInhibition%2520of%2520osimertinib-resistant%2520epidermal%2520growth%2520factor%2520receptor%2520EGFR-T790M%252FC797S%26jtitle%3DChem.%2520Sci.%26date%3D2019%26volume%3D10%26spage%3D10789%26epage%3D10801%26doi%3D10.1039%2Fc9sc03445e" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wolle, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weisner, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keul, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Landel, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lategahn, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rauh, D.</span></span> <span> </span><span class="NLM_article-title">RASPELD to perform high-end screening in an academic environment toward the development of cancer therapeutics</span>. <i>ChemMedChem</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">2065</span>– <span class="NLM_lpage">2072</span>, <span class="refDoi"> DOI: 10.1002/cmdc.201800477</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00870&amp;key=10.1002%2Fcmdc.201800477" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00870&amp;key=30079978" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00870&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhsFKqu7%252FL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2018&pages=2065-2072&author=P.+Wolleauthor=J.+Weisnerauthor=M.+Keulauthor=I.+Landelauthor=J.+Lategahnauthor=D.+Rauh&title=RASPELD+to+perform+high-end+screening+in+an+academic+environment+toward+the+development+of+cancer+therapeutics&doi=10.1002%2Fcmdc.201800477"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">RASPELD to Perform High-End Screening in an Academic Environment toward the Development of Cancer Therapeutics</span></div><div class="casAuthors">Wolle, Patrik; Weisner, Joern; Keul, Marina; Landel, Ina; Lategahn, Jonas; Rauh, Daniel</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">2065-2072</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">The identification of compds. for dissecting biol. functions and the development of novel drug mols. are central tasks that often require screening campaigns.  However, the required architecture is cost- and time-intensive.  Herein the authors describe the devices and technologies that comprise a Robotics-Assisted Screening Platform for Efficient Ligand Discovery (RASPELD), which the authors set up in an academic lab.  RASPELD provides semi-automated high-end screening, and it can be maintained by graduate students.  The authors demonstrate its successful application in biochem. and cellular screens for the identification and validation of bioactive chem. entities as candidate cancer-relevant inhibitors.  Specifically, the authors examd. the interaction between a transcription factor, Nrf2, and its key regulator, Keap1.  The authors also examd. drug-resistant mutants of the epidermal growth factor receptor (EGFR).  Screening campaigns with >30,000 compds. were performed in a reasonable period of time.  The authors identified the mol. RSL6586 as a starting point for hit optimization, which is currently ongoing.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGphaUbPTnBjCbVg90H21EOLACvtfcHk0lj0X9aIIBhhaw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhsFKqu7%252FL&md5=947a778e507c4d6c34905bcd40fd20e5</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.201800477&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.201800477%26sid%3Dliteratum%253Aachs%26aulast%3DWolle%26aufirst%3DP.%26aulast%3DWeisner%26aufirst%3DJ.%26aulast%3DKeul%26aufirst%3DM.%26aulast%3DLandel%26aufirst%3DI.%26aulast%3DLategahn%26aufirst%3DJ.%26aulast%3DRauh%26aufirst%3DD.%26atitle%3DRASPELD%2520to%2520perform%2520high-end%2520screening%2520in%2520an%2520academic%2520environment%2520toward%2520the%2520development%2520of%2520cancer%2520therapeutics%26jtitle%3DChemMedChem%26date%3D2018%26volume%3D13%26spage%3D2065%26epage%3D2072%26doi%3D10.1002%2Fcmdc.201800477" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lounkine, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wawer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wassermann, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bajorath, J.</span></span> <span> </span><span class="NLM_article-title">SARANEA: a freely available program to mine structure-activity and structure-selectivity relationship information in compound data sets</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>50</i></span>,  <span class="NLM_fpage">68</span>– <span class="NLM_lpage">78</span>, <span class="refDoi"> DOI: 10.1021/ci900416a</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci900416a" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00870&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs12jtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2010&pages=68-78&author=E.+Lounkineauthor=M.+Wawerauthor=A.+M.+Wassermannauthor=J.+Bajorath&title=SARANEA%3A+a+freely+available+program+to+mine+structure-activity+and+structure-selectivity+relationship+information+in+compound+data+sets&doi=10.1021%2Fci900416a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">SARANEA: A Freely Available Program To Mine Structure-Activity and Structure-Selectivity Relationship Information in Compound Data Sets</span></div><div class="casAuthors">Lounkine, Eugen; Wawer, Mathias; Wassermann, Anne Mai; Bajorath, Jurgen</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">68-78</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We introduce SARANEA, an open-source Java application for interactive exploration of structure-activity relationship (SAR) and structure-selectivity relationship (SSR) information in compd. sets of any source.  SARANEA integrates various SAR and SSR anal. functions and utilizes a network-like similarity graph data structure for visualization.  The program enables the systematic detection of activity and selectivity cliffs and corresponding key compds. across multiple targets.  Advanced SAR anal. functions implemented in SARANEA include, among others, layered chem. neighborhood graphs, cliff indexes, selectivity trees, editing functions for mol. networks and pathways, bioactivity summaries of key compds., and markers for bioactive compds. having potential side effects.  We report the application of SARANEA to identify SAR and SSR determinants in different sets of serine protease inhibitors.  It is found that key compds. can influence SARs and SSRs in rather different ways.  Such compds. and their SAR/SSR characteristics can be systematically identified and explored using SARANEA.  The program and source code are made freely available under the GNU General Public License.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrWTKafem6ttbVg90H21EOLACvtfcHk0lj0X9aIIBhhaw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs12jtQ%253D%253D&md5=936edf696d3ae7bb8196c81158528369</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1021%2Fci900416a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci900416a%26sid%3Dliteratum%253Aachs%26aulast%3DLounkine%26aufirst%3DE.%26aulast%3DWawer%26aufirst%3DM.%26aulast%3DWassermann%26aufirst%3DA.%2BM.%26aulast%3DBajorath%26aufirst%3DJ.%26atitle%3DSARANEA%253A%2520a%2520freely%2520available%2520program%2520to%2520mine%2520structure-activity%2520and%2520structure-selectivity%2520relationship%2520information%2520in%2520compound%2520data%2520sets%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2010%26volume%3D50%26spage%3D68%26epage%3D78%26doi%3D10.1021%2Fci900416a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Getlik, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grütter, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simard, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen, H. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robubi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aust, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Otterlo, W. A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rauh, D.</span></span> <span> </span><span class="NLM_article-title">Structure-based design, synthesis and biological evaluation of N-pyrazole, N’-thiazole urea inhibitors of MAP kinase p38alpha</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>48</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">15</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2011.11.019</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00870&amp;key=10.1016%2Fj.ejmech.2011.11.019" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00870&amp;key=22154891" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00870&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtFKmsbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2012&pages=1-15&author=M.+Getlikauthor=C.+Gr%C3%BCtterauthor=J.+R.+Simardauthor=H.+D.+Nguyenauthor=A.+Robubiauthor=B.+Austauthor=W.+A.+L.+van+Otterloauthor=D.+Rauh&title=Structure-based+design%2C+synthesis+and+biological+evaluation+of+N-pyrazole%2C+N%E2%80%99-thiazole+urea+inhibitors+of+MAP+kinase+p38alpha&doi=10.1016%2Fj.ejmech.2011.11.019"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-based design, synthesis and biological evaluation of N-pyrazole, N'-thiazole urea inhibitors of MAP kinase p38α</span></div><div class="casAuthors">Getlik, Matthaeus; Gruetter, Christian; Simard, Jeffrey R.; Nguyen, Hoang D.; Robubi, Armin; Aust, Beate; van Otterlo, Willem A. L.; Rauh, Daniel</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">1-15</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">The structure-based design, synthesis and biol. activity of N-(1-aryl-1H-pyrazol-5-yl)-N'-(2 -thiazolyl)ureas as potent inhibitors of p38α mitogen-activated protein kinase (p38α MAPK).  Guided by complex crystal structures, key structural elements were introduced into the initially identified N-aryl-N'-thiazolylurea scaffold that allowed the formation of novel hydrogen bonding interactions within the allosteric site of p38α, resulting in potent type III inhibitors.  [4-(3-Tert-Butyl-5-{[(1,3-thiazol-2-ylamino)carbonyl]amino}-1H-pyrazol-1-yl)-phenyl]acetic acid was found to be the most potent compd. within this series and inhibited p38α activity with an IC 50 of 135 ± 21 nM.  Its closest analog, the Et ester, effectively inhibited p38α mediated phosphorylation of the mitogen activated protein kinase activated protein kinase 2 (MK2) in HeLa cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpxt0EyIJuA17Vg90H21EOLACvtfcHk0ljKD4L8y3BK-Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtFKmsbc%253D&md5=3b6adebcdb5fbede2909c596e1d7901a</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2011.11.019&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2011.11.019%26sid%3Dliteratum%253Aachs%26aulast%3DGetlik%26aufirst%3DM.%26aulast%3DGr%25C3%25BCtter%26aufirst%3DC.%26aulast%3DSimard%26aufirst%3DJ.%2BR.%26aulast%3DNguyen%26aufirst%3DH.%2BD.%26aulast%3DRobubi%26aufirst%3DA.%26aulast%3DAust%26aufirst%3DB.%26aulast%3Dvan%2BOtterlo%26aufirst%3DW.%2BA.%2BL.%26aulast%3DRauh%26aufirst%3DD.%26atitle%3DStructure-based%2520design%252C%2520synthesis%2520and%2520biological%2520evaluation%2520of%2520N-pyrazole%252C%2520N%25E2%2580%2599-thiazole%2520urea%2520inhibitors%2520of%2520MAP%2520kinase%2520p38alpha%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2012%26volume%3D48%26spage%3D1%26epage%3D15%26doi%3D10.1016%2Fj.ejmech.2011.11.019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Spellmon, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Z.</span></span> <span> </span><span class="NLM_article-title">Allosterically targeting EGFR drug-resistance gatekeeper mutations</span>. <i>J. Thorac. Dis.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">1756</span>– <span class="NLM_lpage">1758</span>, <span class="refDoi"> DOI: 10.21037/jtd.2017.06.43</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00870&amp;key=10.21037%2Fjtd.2017.06.43" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00870&amp;key=28839955" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00870&amp;key=1%3ACAS%3A280%3ADC%252BC1cbhsFCmsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2017&pages=1756-1758&author=N.+Spellmonauthor=C.+Liauthor=Z.+Yang&title=Allosterically+targeting+EGFR+drug-resistance+gatekeeper+mutations&doi=10.21037%2Fjtd.2017.06.43"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Allosterically targeting EGFR drug-resistance gatekeeper mutations</span></div><div class="casAuthors">Spellmon Nicholas; Yang Zhe; Li Chunying</div><div class="citationInfo"><span class="NLM_cas:title">Journal of thoracic disease</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1756-1758</span>
        ISSN:<span class="NLM_cas:issn">2072-1439</span>.
    </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQMer61q-2iHLZpSb_XWfMrfW6udTcc2eZHWm8SlP7htbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1cbhsFCmsg%253D%253D&md5=32e7407f858050a7ddbb026c5477d183</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.21037%2Fjtd.2017.06.43&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.21037%252Fjtd.2017.06.43%26sid%3Dliteratum%253Aachs%26aulast%3DSpellmon%26aufirst%3DN.%26aulast%3DLi%26aufirst%3DC.%26aulast%3DYang%26aufirst%3DZ.%26atitle%3DAllosterically%2520targeting%2520EGFR%2520drug-resistance%2520gatekeeper%2520mutations%26jtitle%3DJ.%2520Thorac.%2520Dis.%26date%3D2017%26volume%3D9%26spage%3D1756%26epage%3D1758%26doi%3D10.21037%2Fjtd.2017.06.43" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jia, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yun, C.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ercan, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manuia, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Juarez, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rhee, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Palakurthi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lelais, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DiDonato, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bursulaya, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michellys, P.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Epple, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marsilje, T. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McNeill, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harris, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bender, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, K.-K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jänne, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eck, M. J.</span></span> <span> </span><span class="NLM_article-title">Overcoming EGFR(T790M) and EGFR(C797S) resistance with mutant-selective allosteric inhibitors</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>534</i></span>,  <span class="NLM_fpage">129</span>– <span class="NLM_lpage">132</span>, <span class="refDoi"> DOI: 10.1038/nature17960</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00870&amp;key=10.1038%2Fnature17960" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00870&amp;key=27251290" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00870&amp;key=1%3ACAS%3A528%3ADC%252BC28XosFWht7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=534&publication_year=2016&pages=129-132&author=Y.+Jiaauthor=C.-H.+Yunauthor=E.+Parkauthor=D.+Ercanauthor=M.+Manuiaauthor=J.+Juarezauthor=C.+Xuauthor=K.+Rheeauthor=T.+Chenauthor=H.+Zhangauthor=S.+Palakurthiauthor=J.+Jangauthor=G.+Lelaisauthor=M.+DiDonatoauthor=B.+Bursulayaauthor=P.-Y.+Michellysauthor=R.+Eppleauthor=T.+H.+Marsiljeauthor=M.+McNeillauthor=W.+Luauthor=J.+Harrisauthor=S.+Benderauthor=K.-K.+Wongauthor=P.+A.+J%C3%A4nneauthor=M.+J.+Eck&title=Overcoming+EGFR%28T790M%29+and+EGFR%28C797S%29+resistance+with+mutant-selective+allosteric+inhibitors&doi=10.1038%2Fnature17960"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Overcoming EGFR(T790M) and EGFR(C797S) resistance with mutant-selective allosteric inhibitors</span></div><div class="casAuthors">Jia, Yong; Yun, Cai-Hong; Park, Eunyoung; Ercan, Dalia; Manuia, Mari; Juarez, Jose; Xu, Chunxiao; Rhee, Kevin; Chen, Ting; Zhang, Haikuo; Palakurthi, Sangeetha; Jang, Jaebong; Lelais, Gerald; DiDonato, Michael; Bursulaya, Badry; Michellys, Pierre-Yves; Epple, Robert; Marsilje, Thomas H.; McNeill, Matthew; Lu, Wenshuo; Harris, Jennifer; Bender, Steven; Wong, Kwok-Kin; Janne, Pasi A.; Eck, Michael J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">534</span>
        (<span class="NLM_cas:issue">7605</span>),
    <span class="NLM_cas:pages">129-132</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The epidermal growth factor receptor (EGFR)-directed tyrosine kinase inhibitors (TKIs) gefitinib, erlotinib and afatinib are approved treatments for non-small cell lung cancers harbouring activating mutations in the EGFR kinase, but resistance arises rapidly, most frequently owing to the secondary T790M mutation within the ATP site of the receptor.  Recently developed mutant-selective irreversible inhibitors are highly active against the T790M mutant, but their efficacy can be compromised by acquired mutation of C797, the cysteine residue with which they form a key covalent bond.  All current EGFR TKIs target the ATP-site of the kinase, highlighting the need for therapeutic agents with alternative mechanisms of action.  Here we describe the rational discovery of EAI045, an allosteric inhibitor that targets selected drug-resistant EGFR mutants but spares the wild-type receptor.  The crystal structure shows that the compd. binds an allosteric site created by the displacement of the regulatory C-helix in an inactive conformation of the kinase.  The compd. inhibits L858R/T790M-mutant EGFR with low-nanomolar potency in biochem. assays.  However, as a single agent it is not effective in blocking EGFR-driven proliferation in cells owing to differential potency on the two subunits of the dimeric receptor, which interact in an asym. manner in the active state.  We observe marked synergy of EAI045 with cetuximab, an antibody therapeutic that blocks EGFR dimerization, rendering the kinase uniformly susceptible to the allosteric agent.  EAI045 in combination with cetuximab is effective in mouse models of lung cancer driven by EGFR(L858R/T790M) and by EGFR(L858R/T790M/C797S), a mutant that is resistant to all currently available EGFR TKIs.  More generally, our findings illustrate the utility of purposefully targeting allosteric sites to obtain mutant-selective inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqQkr363w1dMLVg90H21EOLACvtfcHk0ljKD4L8y3BK-Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XosFWht7k%253D&md5=f4c2d2362eb900c64bad8550b9da0872</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1038%2Fnature17960&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature17960%26sid%3Dliteratum%253Aachs%26aulast%3DJia%26aufirst%3DY.%26aulast%3DYun%26aufirst%3DC.-H.%26aulast%3DPark%26aufirst%3DE.%26aulast%3DErcan%26aufirst%3DD.%26aulast%3DManuia%26aufirst%3DM.%26aulast%3DJuarez%26aufirst%3DJ.%26aulast%3DXu%26aufirst%3DC.%26aulast%3DRhee%26aufirst%3DK.%26aulast%3DChen%26aufirst%3DT.%26aulast%3DZhang%26aufirst%3DH.%26aulast%3DPalakurthi%26aufirst%3DS.%26aulast%3DJang%26aufirst%3DJ.%26aulast%3DLelais%26aufirst%3DG.%26aulast%3DDiDonato%26aufirst%3DM.%26aulast%3DBursulaya%26aufirst%3DB.%26aulast%3DMichellys%26aufirst%3DP.-Y.%26aulast%3DEpple%26aufirst%3DR.%26aulast%3DMarsilje%26aufirst%3DT.%2BH.%26aulast%3DMcNeill%26aufirst%3DM.%26aulast%3DLu%26aufirst%3DW.%26aulast%3DHarris%26aufirst%3DJ.%26aulast%3DBender%26aufirst%3DS.%26aulast%3DWong%26aufirst%3DK.-K.%26aulast%3DJ%25C3%25A4nne%26aufirst%3DP.%2BA.%26aulast%3DEck%26aufirst%3DM.%2BJ.%26atitle%3DOvercoming%2520EGFR%2528T790M%2529%2520and%2520EGFR%2528C797S%2529%2520resistance%2520with%2520mutant-selective%2520allosteric%2520inhibitors%26jtitle%3DNature%26date%3D2016%26volume%3D534%26spage%3D129%26epage%3D132%26doi%3D10.1038%2Fnature17960" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Baell, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holloway, G. A.</span></span> <span> </span><span class="NLM_article-title">New substructure filters for removal of pan assay interference compounds (PAINS) from screening libraries and for their exclusion in bioassays</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">2719</span>– <span class="NLM_lpage">2740</span>, <span class="refDoi"> DOI: 10.1021/jm901137j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm901137j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00870&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsF2qsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=2719-2740&author=J.+B.+Baellauthor=G.+A.+Holloway&title=New+substructure+filters+for+removal+of+pan+assay+interference+compounds+%28PAINS%29+from+screening+libraries+and+for+their+exclusion+in+bioassays&doi=10.1021%2Fjm901137j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">New Substructure Filters for Removal of Pan Assay Interference Compounds (PAINS) from Screening Libraries and for Their Exclusion in Bioassays</span></div><div class="casAuthors">Baell, Jonathan B.; Holloway, Georgina A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2719-2740</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">This report describes a no. of substructural features which can help to identify compds. that appear as frequent hitters (promiscuous compds.) in many biochem. high throughput screens.  The compds. identified by such substructural features are not recognized by filters commonly used to identify reactive compds.  Even though these substructural features were identified using only one assay detection technol., such compds. have been reported to be active from many different assays.  In fact, these compds. are increasingly prevalent in the literature as potential starting points for further exploration, whereas they may not be.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGro_JARcceWjLVg90H21EOLACvtfcHk0lh2DeSE1wUdrA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsF2qsLw%253D&md5=fbf397aa4910753c550425708c866fd2</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1021%2Fjm901137j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm901137j%26sid%3Dliteratum%253Aachs%26aulast%3DBaell%26aufirst%3DJ.%2BB.%26aulast%3DHolloway%26aufirst%3DG.%2BA.%26atitle%3DNew%2520substructure%2520filters%2520for%2520removal%2520of%2520pan%2520assay%2520interference%2520compounds%2520%2528PAINS%2529%2520from%2520screening%2520libraries%2520and%2520for%2520their%2520exclusion%2520in%2520bioassays%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D2719%26epage%3D2740%26doi%3D10.1021%2Fjm901137j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Engel, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lategahn, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rauh, D.</span></span> <span> </span><span class="NLM_article-title">Hope and disappointment: covalent inhibitors to overcome drug resistance in non-small cell lung cancer</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">2</span>– <span class="NLM_lpage">5</span>, <span class="refDoi"> DOI: 10.1021/acsmedchemlett.5b00475</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.5b00475" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00870&amp;key=1%3ACAS%3A528%3ADC%252BC28XktVOluw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=2-5&author=J.+Engelauthor=J.+Lategahnauthor=D.+Rauh&title=Hope+and+disappointment%3A+covalent+inhibitors+to+overcome+drug+resistance+in+non-small+cell+lung+cancer&doi=10.1021%2Facsmedchemlett.5b00475"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Hope and Disappointment: Covalent Inhibitors to Overcome Drug Resistance in Non-Small Cell Lung Cancer</span></div><div class="casAuthors">Engel, Julian; Lategahn, Jonas; Rauh, Daniel</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">2-5</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">In the last five years, the detailed understanding of how to overcome T790M drug resistance in non-small cell lung cancer (NSCLC) has culminated in the development of a third-generation of covalent EGFR inhibitors with excellent clin. outcomes.  However, the emergence of a newly discovered acquired drug resistance challenges the concept of small mol. targeted cancer therapy in NSCLC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqcxNkfyBm4vrVg90H21EOLACvtfcHk0lh2DeSE1wUdrA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XktVOluw%253D%253D&md5=738e3286dd1b1d4110249e33cceec9ac</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.5b00475&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.5b00475%26sid%3Dliteratum%253Aachs%26aulast%3DEngel%26aufirst%3DJ.%26aulast%3DLategahn%26aufirst%3DJ.%26aulast%3DRauh%26aufirst%3DD.%26atitle%3DHope%2520and%2520disappointment%253A%2520covalent%2520inhibitors%2520to%2520overcome%2520drug%2520resistance%2520in%2520non-small%2520cell%2520lung%2520cancer%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2016%26volume%3D7%26spage%3D2%26epage%3D5%26doi%3D10.1021%2Facsmedchemlett.5b00475" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 2 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Youqin  Xu</span>, <span class="hlFld-ContribAuthor ">Wancheng  Chen</span>, <span class="hlFld-ContribAuthor ">Jing  Liang</span>, <span class="hlFld-ContribAuthor ">Xiaoqi  Zeng</span>, <span class="hlFld-ContribAuthor ">Kaiyuan  Ji</span>, <span class="hlFld-ContribAuthor ">Jianlong  Zhou</span>, <span class="hlFld-ContribAuthor ">Shijun  Liao</span>, <span class="hlFld-ContribAuthor ">Jiexian  Wu</span>, <span class="hlFld-ContribAuthor ">Kongyang  Xing</span>, <span class="hlFld-ContribAuthor ">Zilong  He</span>, <span class="hlFld-ContribAuthor ">Yang  Yang</span>, <span class="hlFld-ContribAuthor ">Qianzhen  Liu</span>, <span class="hlFld-ContribAuthor ">Pingyi  Zhu</span>, <span class="hlFld-ContribAuthor ">Yuchang  Liu</span>, <span class="hlFld-ContribAuthor ">Li  Li</span>, <span class="hlFld-ContribAuthor ">Minfeng  Liu</span>, <span class="hlFld-ContribAuthor ">Wenxiao  Chen</span>, <span class="hlFld-ContribAuthor ">Wenhua  Huang</span>. </span><span class="cited-content_cbyCitation_article-title">The miR-1185-2-3p—GOLPH3L pathway promotes glucose metabolism in breast cancer by stabilizing p53-induced SERPINE1. </span><span class="cited-content_cbyCitation_journal-name">Journal of Experimental & Clinical Cancer Research</span><span> <strong>2021,</strong> <em>40 </em>
                                    (1)
                                     <a href="https://doi.org/10.1186/s13046-020-01767-9" title="DOI URL">https://doi.org/10.1186/s13046-020-01767-9</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1186/s13046-020-01767-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1186%2Fs13046-020-01767-9%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Experimental%2520%2526%2520Clinical%2520Cancer%2520Research%26atitle%3DThe%252BmiR-1185-2-3p%2525E2%252580%252594GOLPH3L%252Bpathway%252Bpromotes%252Bglucose%252Bmetabolism%252Bin%252Bbreast%252Bcancer%252Bby%252Bstabilizing%252Bp53-induced%252BSERPINE1%26aulast%3DXu%26aufirst%3DYouqin%26date%3D2021%26date%3D2021%26volume%3D40%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Andrea  Bistrović Popov</span>, <span class="hlFld-ContribAuthor ">Robert  Vianelo</span>, <span class="hlFld-ContribAuthor ">Petra  Grbčić</span>, <span class="hlFld-ContribAuthor ">Mirela  Sedić</span>, <span class="hlFld-ContribAuthor ">Sandra Kraljević  Pavelić</span>, <span class="hlFld-ContribAuthor ">Krešimir  Pavelić</span>, <span class="hlFld-ContribAuthor ">Silvana  Raić-Malić</span>. </span><span class="cited-content_cbyCitation_article-title">Novel Bis- and Mono-Pyrrolo[2,3-d]pyrimidine and Purine Derivatives: Synthesis, Computational Analysis and Antiproliferative Evaluation. </span><span class="cited-content_cbyCitation_journal-name">Molecules</span><span> <strong>2021,</strong> <em>26 </em>
                                    (11)
                                     , 3334. <a href="https://doi.org/10.3390/molecules26113334" title="DOI URL">https://doi.org/10.3390/molecules26113334</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/molecules26113334&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fmolecules26113334%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecules%26atitle%3DNovel%252BBis-%252Band%252BMono-Pyrrolo%25255B2%25252C3-d%25255Dpyrimidine%252Band%252BPurine%252BDerivatives%25253A%252BSynthesis%25252C%252BComputational%252BAnalysis%252Band%252BAntiproliferative%252BEvaluation%26aulast%3DBistrovi%25C4%2587%2BPopov%26aufirst%3DAndrea%26date%3D2021%26date%3D2021%26volume%3D26%26issue%3D11%26spage%3D3334" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c00870/20201019/images/medium/jm0c00870_0014.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c00870/20201019/images/large/jm0c00870_0014.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00870&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c00870/20201019/images/medium/jm0c00870_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c00870/20201019/images/large/jm0c00870_0002.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. (A) Mutations found in Her2 are primarily insertion mutations adjacent to the regulatory helix αC (yellow; PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3PP0">3PP0</a>). Among these, A775_G776insYVMA is most predominant in Her2-mutated patients harboring NSCLC.<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a> (B) Type I and type II inhibitors were exemplified by afatinib (<b>1</b>) and neratinib (<b>2</b>), respectively, have been evaluated as Her2 inhibitors.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c00870/20201019/images/large/jm0c00870_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00870&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c00870/20201019/images/medium/jm0c00870_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c00870/20201019/images/large/jm0c00870_0003.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Initial focused biochemical screening against Her2 with 1635 compounds and neratinib as a reference compound. (A) Screen comprised four arms: Her2-wt and Her2-insYVMA were each tested at concentrations of 1 and 10 μM. (B) Compounds that displayed ≤50% residual kinase activity at 1 μM and/or ≤30% residual activity at 10 μM were considered as hits (see <a href="/doi/suppl/10.1021/acs.jmedchem.0c00870/suppl_file/jm0c00870_si_001.pdf" class="ext-link">Figure S3</a>). Some hits within one arm of the screening campaign did not meet the hit criteria in other arms but were taken into account in the subsequent hit validation.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c00870/20201019/images/large/jm0c00870_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00870&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c00870/20201019/images/medium/jm0c00870_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c00870/20201019/images/large/jm0c00870_0004.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. IC<sub>50</sub> values were recorded for 132 compounds and were clustered by chemical similarity using SARanea. The color code indicates the determined IC<sub>50</sub> values against Her2-insYVMA. Representative compound structures <b>i–viii</b> for clusters that contained valid hits are shown.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c00870/20201019/images/large/jm0c00870_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00870&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c00870/20201019/images/medium/jm0c00870_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c00870/20201019/images/large/jm0c00870_0005.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. (A) Alignment of X-ray complex crystal structures of compounds bearing structural motifs found in the focused screening campaign: EAI001 bound to EGFR (green; PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5D41">5D41</a>), <b>vii</b> bound to p38 MAPK (red; PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3LFB">3LFB</a>), and lapatinib bound to EGFR (yellow; PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1XKK">1XKK</a>). (B) Chemical structures of EAI001 (green), <b>vii</b> (red), and lapatinib (yellow). (C) Design of Her2 inhibitors based on the covalent pyrrolopyrimidine core equipped with structural motifs found in the focused screen.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c00870/20201019/images/large/jm0c00870_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00870&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c00870/20201019/images/medium/jm0c00870_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c00870/20201019/images/large/jm0c00870_0008.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of Pyrrolopyrimidine-Based Inhibitors <b>7</b>, <b>14</b>, <b>18–21</b>, and <b>25</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c00870/20201019/images/large/jm0c00870_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00870&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (i) <i>N</i>-iodosuccinimide, DMF, rt; (ii) SEM-Cl, NaH, THF, 0 °C, then rt; (iii) (3-nitrophenyl)boronic acid, Pd(PPh<sub>3</sub>)<sub>4</sub>, K<sub>2</sub>CO<sub>3</sub>, MeCN/H<sub>2</sub>O (2:1), 150 °C, 90 min, μW; (iv) <i>N</i>-bromosuccinimide, DMF, rt; (v) 1-methyl-4-[4-(4,4,5,5-tetramethyl-1,3,2-dioxaborolan-2-yl)phenyl]piperazine, Pd(PPh<sub>3</sub>)<sub>4</sub>, K<sub>2</sub>CO<sub>3</sub>, DME/H<sub>2</sub>O (3:1), 150 °C, 90 min, μW; (vi) benzyltriethylammonium chloride, <i>N</i>,<i>N</i>-dimethylaniline, POCl<sub>3</sub>, MeCN, 85 °C; (vii) iron powder, NH<sub>4</sub>Cl, EtOH:H<sub>2</sub>O (9:1), 60 °C; (viii) acryloyl chloride, <i>N</i>,<i>N</i>-diisopropylethylamine (DIPEA), THF, 0 °C then rt; (ix) RNH<sub>2</sub>, HCl/dioxane (4 M), <sup><i>n</i></sup>BuOH, 80 °C; (x) TFA/ dichloromethane (DCM) (1:3), rt, then 1 M KOH, THF, rt; (xi) (3-nitrophenyl)boronic acid, Pd(PPh<sub>3</sub>)<sub>4</sub>, K<sub>2</sub>CO<sub>3</sub>, MeCN/H<sub>2</sub>O (2:1), 80 °C; (xii) (4-fluoro-3-nitrophenyl)boronic acid, Na<sub>2</sub>CO<sub>3</sub>, Pd(PPh<sub>3</sub>)<sub>4</sub>, dioxane/H<sub>2</sub>O (3:1), Ar atm, 80 °C; (xiii) HOR′, NaH, dry THF, rt; (xiv) K<sub>2</sub>CO<sub>3</sub>, MeOH, rt.</p></p></figure><figure data-id="fig5" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c00870/20201019/images/medium/jm0c00870_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c00870/20201019/images/large/jm0c00870_0006.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Western blot analysis of Her2 and downstream cascade phosphorylation inhibition by representative compounds in H1781 cells.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c00870/20201019/images/large/jm0c00870_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00870&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c00870/20201019/images/medium/jm0c00870_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c00870/20201019/images/large/jm0c00870_0007.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Cocrystal structures of covalent pyrrolopyrimidines in complex with the EGFR-T790M/V948R mutant. Diagrams of the experimental electron densities of (A) <b>14d</b>/EGFR-T790M/V948R at 2.0 Å, chain A (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TFU">6TFU</a>), (B) <b>18b</b>/EGFR-T790M/V948R at 1.5 Å, chain A (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TFV">6TFV</a>), (C) <b>18c</b>/EGFR-T790M/V948R at 1.7 Å, chain A (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TFY">6TFY</a>), (D) <b>18d</b>/EGFR-T790M/V948R at 2.0 Å, chain A (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TFW">6TFW</a>), (E) <b>19</b>/EGFR-T790M/V948R at 1.8 Å, chain B (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TFZ">6TFZ</a>), (F) <b>21a</b>/EGFR-T790M/V948R at 1.5 Å, chain B (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TG0">6TG0</a>), and (G) <b>21b</b>/EGFR-T790M/V948R at 1.6 Å, chain B (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TG1">6TG1</a>); 2Fo-Fc map contoured at an root-mean-square deviation of 1.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-20/acs.jmedchem.0c00870/20201019/images/large/jm0c00870_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c00870&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i165">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_73545" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_73545" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 28 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pillai, R. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Behera, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berry, L. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rossi, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kris, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, B. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bunn, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramalingam, S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khuri, F. R.</span></span> <span> </span><span class="NLM_article-title">HER2 mutations in lung adenocarcinomas: a report from the Lung Cancer Mutation Consortium</span>. <i>Cancer</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>123</i></span>,  <span class="NLM_fpage">4099</span>– <span class="NLM_lpage">4105</span>, <span class="refDoi"> DOI: 10.1002/cncr.30869</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00870&amp;key=10.1002%2Fcncr.30869" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00870&amp;key=28743157" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00870&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhs12mtLzL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=123&publication_year=2017&pages=4099-4105&author=R.+N.+Pillaiauthor=M.+Beheraauthor=L.+D.+Berryauthor=M.+R.+Rossiauthor=M.+G.+Krisauthor=B.+E.+Johnsonauthor=P.+A.+Bunnauthor=S.+S.+Ramalingamauthor=F.+R.+Khuri&title=HER2+mutations+in+lung+adenocarcinomas%3A+a+report+from+the+Lung+Cancer+Mutation+Consortium&doi=10.1002%2Fcncr.30869"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">HER2 mutations in lung adenocarcinomas: A report from the Lung Cancer Mutation Consortium</span></div><div class="casAuthors">Pillai, Rathi N.; Behera, Madhusmita; Berry, Lynne D.; Rossi, Mike R.; Kris, Mark G.; Johnson, Bruce E.; Bunn, Paul A.; Ramalingam, Suresh S.; Khuri, Fadlo R.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer (Hoboken, NJ, United States)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">123</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">4099-4105</span>CODEN:
                <span class="NLM_cas:coden">CANCAR</span>;
        ISSN:<span class="NLM_cas:issn">0008-543X</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">BACKGROUND : Human epidermal growth factor receptor 2 (HER2) mutations have been reported in lung adenocarcinomas.  Herein, the authors describe the prevalence, clin. features, and outcomes assocd. with HER2 mutations in 1007 patients in the Lung Cancer Mutation Consortium (LCMC).  METHODS : Patients with advanced-stage lung adenocarcinomas were enrolled to the LCMC.  Tumor specimens were assessed for diagnosis and adequacy; multiplexed genotyping was performed in Clin. Lab. Improvement Amendments (CLIA)-certified labs. to examine 10 oncogenic drivers.  The LCMC database was queried for patients with HER2 mutations to access demog. data, treatment history, and vital status.  An exploratory anal. was performed to evaluate the survival of patients with HER2 mutations who were treated with HER2-directed therapies.  RESULTS : A total of 920 patients were tested for HER2 mutations; 24 patients (3%) harbored exon 20 insertion mutations (95% confidence interval, 2%-4%).  One patient had a concurrent mesenchymal-epithelial transition factor (MET) amplification.  The median age of the patients was 62 years, with a slight predominance of females over males (14 females vs 10 males).  The majority of the patients were never-smokers (71%) and presented with advanced disease at the time of diagnosis.  The median survival for patients who received HER2-targeted therapies (12 patients) was 2.1 years compared with 1.4 years for those who did not (12 patients) (P = .48).  Patients with HER2 mutations were found to have inferior survival compared with the rest of the LCMC cohort with other mutations: the median survival was 3.5 years in the LCMC population receiving targeted therapy and 2.4 years for patients not receiving targeted therapy.  CONCLUSIONS : HER2 mutations were detected in 3% of patients with lung adenocarcinoma in the LCMC.  HER2-directed therapies should be investigated in this subgroup of patients.  Cancer 2017;123:4099-4105. 2017 American Cancer Society.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo2KOE5lpETgLVg90H21EOLACvtfcHk0liPh_CHII5Jcw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhs12mtLzL&md5=f9df8f1ac0e5053795c2b7b2d3bac2b8</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1002%2Fcncr.30869&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcncr.30869%26sid%3Dliteratum%253Aachs%26aulast%3DPillai%26aufirst%3DR.%2BN.%26aulast%3DBehera%26aufirst%3DM.%26aulast%3DBerry%26aufirst%3DL.%2BD.%26aulast%3DRossi%26aufirst%3DM.%2BR.%26aulast%3DKris%26aufirst%3DM.%2BG.%26aulast%3DJohnson%26aufirst%3DB.%2BE.%26aulast%3DBunn%26aufirst%3DP.%2BA.%26aulast%3DRamalingam%26aufirst%3DS.%2BS.%26aulast%3DKhuri%26aufirst%3DF.%2BR.%26atitle%3DHER2%2520mutations%2520in%2520lung%2520adenocarcinomas%253A%2520a%2520report%2520from%2520the%2520Lung%2520Cancer%2520Mutation%2520Consortium%26jtitle%3DCancer%26date%3D2017%26volume%3D123%26spage%3D4099%26epage%3D4105%26doi%3D10.1002%2Fcncr.30869" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Arcila, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chaft, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nafa, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roy-Chowdhuri, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lau, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zaidinski, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paik, P. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zakowski, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kris, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ladanyi, M.</span></span> <span> </span><span class="NLM_article-title">Prevalence, clinicopathologic associations, and molecular spectrum of ERBB2 (HER2) tyrosine kinase mutations in lung adenocarcinomas</span>. <i>Clin. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">4910</span>– <span class="NLM_lpage">4918</span>, <span class="refDoi"> DOI: 10.1158/1078-0432.ccr-12-0912</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00870&amp;key=10.1158%2F1078-0432.CCR-12-0912" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00870&amp;key=22761469" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00870&amp;key=1%3ACAS%3A528%3ADC%252BC38Xhtl2lt7%252FM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2012&pages=4910-4918&author=M.+E.+Arcilaauthor=J.+E.+Chaftauthor=K.+Nafaauthor=S.+Roy-Chowdhuriauthor=C.+Lauauthor=M.+Zaidinskiauthor=P.+K.+Paikauthor=M.+F.+Zakowskiauthor=M.+G.+Krisauthor=M.+Ladanyi&title=Prevalence%2C+clinicopathologic+associations%2C+and+molecular+spectrum+of+ERBB2+%28HER2%29+tyrosine+kinase+mutations+in+lung+adenocarcinomas&doi=10.1158%2F1078-0432.ccr-12-0912"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Prevalence, Clinicopathologic Associations, and Molecular Spectrum of ERBB2 (HER2) Tyrosine Kinase Mutations in Lung Adenocarcinomas</span></div><div class="casAuthors">Arcila, Maria E.; Chaft, Jamie E.; Nafa, Khedoudja; Roy-Chowdhuri, Sinchita; Lau, Christopher; Zaidinski, Michael; Paik, Paul K.; Zakowski, Maureen F.; Kris, Mark G.; Ladanyi, Marc</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">4910-4918</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Purpose: Activating mutations in the tyrosine kinase domain of HER2 (ERBB2) have been described in a subset of lung adenocarcinomas (ADCs) and are mutually exclusive with EGFR and KRAS mutations.  The prevalence, clinicopathol. characteristics, prognostic implications, and mol. heterogeneity of HER2-mutated lung ADCs are not well established in U.S. patients.  Exptl. Design: Lung ADC samples (N = 1,478) were first screened for mutations in EGFR (exons 19 and 21) and KRAS (exon 2), and neg. cases were then assessed for HER2 mutations (exons 19-20) using a sizing assay and mass spectrometry.  Testing for addnl. recurrent point mutations in EGFR, KRAS, BRAF, NRAS, PIK3CA, MEK1, and AKT was conducted by mass spectrometry.  ALK rearrangements and HER2 amplification were assessed by FISH.  Results: We identified 25 cases with HER2 mutations, representing 6% of EGFR/KRAS/ALK-neg. specimens.  Small insertions in exon 20 accounted for 96% (24/25) of the cases.  Compared with insertions in EGFR exon 20, there was less variability, with 83% (20/24) being a 12 bp insertion causing duplication of amino acids YVMA at codon 775.  Morphol., 92% (23/25) were moderately or poorly differentiated ADC.  HER2 mutation was not assocd. with concurrent HER2 amplification in 11 cases tested for both.  HER2 mutations were more frequent among never-smokers (P < 0.0001) but there were no assocns. with sex, race, or stage.  Conclusions: HER2 mutations identify a distinct subset of lung ADCs.  Given the high prevalence of lung cancer worldwide and the availability of std. and investigational therapies targeting HER2, routine clin. genotyping of lung ADC should include HER2.  Clin Cancer Res; 18(18); 4910-8. ©2012 AACR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGql2xQKmrAL7bVg90H21EOLACvtfcHk0liPh_CHII5Jcw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xhtl2lt7%252FM&md5=683086d6c06b3fb76b94f7c4d8bfed40</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1158%2F1078-0432.CCR-12-0912&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1078-0432.CCR-12-0912%26sid%3Dliteratum%253Aachs%26aulast%3DArcila%26aufirst%3DM.%2BE.%26aulast%3DChaft%26aufirst%3DJ.%2BE.%26aulast%3DNafa%26aufirst%3DK.%26aulast%3DRoy-Chowdhuri%26aufirst%3DS.%26aulast%3DLau%26aufirst%3DC.%26aulast%3DZaidinski%26aufirst%3DM.%26aulast%3DPaik%26aufirst%3DP.%2BK.%26aulast%3DZakowski%26aufirst%3DM.%2BF.%26aulast%3DKris%26aufirst%3DM.%2BG.%26aulast%3DLadanyi%26aufirst%3DM.%26atitle%3DPrevalence%252C%2520clinicopathologic%2520associations%252C%2520and%2520molecular%2520spectrum%2520of%2520ERBB2%2520%2528HER2%2529%2520tyrosine%2520kinase%2520mutations%2520in%2520lung%2520adenocarcinomas%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D2012%26volume%3D18%26spage%3D4910%26epage%3D4918%26doi%3D10.1158%2F1078-0432.ccr-12-0912" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ishikawa, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seto, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Banno, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kawakita, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oorui, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taniguchi, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohta, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tamura, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakayama, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miki, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kamiguchi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanaka, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Habuka, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sogabe, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yano, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aertgeerts, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kamiyama, K.</span></span> <span> </span><span class="NLM_article-title">Design and synthesis of novel human epidermal growth factor receptor 2 (HER2)/epidermal growth factor receptor (EGFR) dual inhibitors bearing a pyrrolo[3,2-d]pyrimidine scaffold</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">8030</span>– <span class="NLM_lpage">8050</span>, <span class="refDoi"> DOI: 10.1021/jm2008634</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm2008634" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00870&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtlajur7M" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=8030-8050&author=T.+Ishikawaauthor=M.+Setoauthor=H.+Bannoauthor=Y.+Kawakitaauthor=M.+Ooruiauthor=T.+Taniguchiauthor=Y.+Ohtaauthor=T.+Tamuraauthor=A.+Nakayamaauthor=H.+Mikiauthor=H.+Kamiguchiauthor=T.+Tanakaauthor=N.+Habukaauthor=S.+Sogabeauthor=J.+Yanoauthor=K.+Aertgeertsauthor=K.+Kamiyama&title=Design+and+synthesis+of+novel+human+epidermal+growth+factor+receptor+2+%28HER2%29%2Fepidermal+growth+factor+receptor+%28EGFR%29+dual+inhibitors+bearing+a+pyrrolo%5B3%2C2-d%5Dpyrimidine+scaffold&doi=10.1021%2Fjm2008634"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Design and Synthesis of Novel Human Epidermal Growth Factor Receptor 2 (HER2)/Epidermal Growth Factor Receptor (EGFR) Dual Inhibitors Bearing a Pyrrolo[3,2-d]pyrimidine Scaffold</span></div><div class="casAuthors">Ishikawa, Tomoyasu; Seto, Masaki; Banno, Hiroshi; Kawakita, Youichi; Oorui, Mami; Taniguchi, Takahiko; Ohta, Yoshikazu; Tamura, Toshiya; Nakayama, Akiko; Miki, Hiroshi; Kamiguchi, Hidenori; Tanaka, Toshimasa; Habuka, Noriyuki; Sogabe, Satoshi; Yano, Jason; Aertgeerts, Kathleen; Kamiyama, Keiji</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">8030-8050</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Dual inhibitors of human epidermal growth factor receptor 2 (HER2) and epidermal growth factor receptor (EGFR) have been investigated for breast, lung, gastric, prostate, and other cancers; one, lapatinib, is currently approved for breast cancer.  To develop novel HER2/EGFR dual kinase inhibitors, we designed and synthesized pyrrolo[3,2-d]pyrimidine derivs., e.g. I, capable of fitting into the receptors' ATP binding site.  Among the prepd. compds., I showed potent HER2 and EGFR (HER1) inhibitory activities as well as tumor growth inhibitory activity.  The X-ray cocrystal structures of I with both HER2 and EGFR demonstrated that I interacts with the expected residues in their resp. ATP pockets.  Furthermore, reflecting its good oral bioavailability, I exhibited potent in vivo efficacy in HER2-overexpressing tumor xenograft models.  On the basis of these findings, we report I (TAK-285) as a promising candidate for clin. development as a novel HER2/EGFR dual kinase inhibitor.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq9QtqAj6idYrVg90H21EOLACvtfcHk0ljIsvcZJV2D_Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtlajur7M&md5=ce030ac133a1c0ff7cb78b269ff85987</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1021%2Fjm2008634&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm2008634%26sid%3Dliteratum%253Aachs%26aulast%3DIshikawa%26aufirst%3DT.%26aulast%3DSeto%26aufirst%3DM.%26aulast%3DBanno%26aufirst%3DH.%26aulast%3DKawakita%26aufirst%3DY.%26aulast%3DOorui%26aufirst%3DM.%26aulast%3DTaniguchi%26aufirst%3DT.%26aulast%3DOhta%26aufirst%3DY.%26aulast%3DTamura%26aufirst%3DT.%26aulast%3DNakayama%26aufirst%3DA.%26aulast%3DMiki%26aufirst%3DH.%26aulast%3DKamiguchi%26aufirst%3DH.%26aulast%3DTanaka%26aufirst%3DT.%26aulast%3DHabuka%26aufirst%3DN.%26aulast%3DSogabe%26aufirst%3DS.%26aulast%3DYano%26aufirst%3DJ.%26aulast%3DAertgeerts%26aufirst%3DK.%26aulast%3DKamiyama%26aufirst%3DK.%26atitle%3DDesign%2520and%2520synthesis%2520of%2520novel%2520human%2520epidermal%2520growth%2520factor%2520receptor%25202%2520%2528HER2%2529%252Fepidermal%2520growth%2520factor%2520receptor%2520%2528EGFR%2529%2520dual%2520inhibitors%2520bearing%2520a%2520pyrrolo%255B3%252C2-d%255Dpyrimidine%2520scaffold%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26spage%3D8030%26epage%3D8050%26doi%3D10.1021%2Fjm2008634" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gschwind, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fischer, O. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ullrich, A.</span></span> <span> </span><span class="NLM_article-title">The discovery of receptor tyrosine kinases: targets for cancer therapy</span>. <i>Nat. Rev. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">361</span>– <span class="NLM_lpage">370</span>, <span class="refDoi"> DOI: 10.1038/nrc1360</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00870&amp;key=10.1038%2Fnrc1360" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00870&amp;key=15122207" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00870&amp;key=1%3ACAS%3A528%3ADC%252BD2cXjsFSlu7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2004&pages=361-370&author=A.+Gschwindauthor=O.+M.+Fischerauthor=A.+Ullrich&title=The+discovery+of+receptor+tyrosine+kinases%3A+targets+for+cancer+therapy&doi=10.1038%2Fnrc1360"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Timeline: The discovery of receptor tyrosine kinases: targets for cancer therapy</span></div><div class="casAuthors">Gschwind, Andreas; Fischer, Oliver M.; Ullrich, Axel</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">361-370</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Receptor tyrosine kinases are a subclass of cell-surface growth-factor receptors with an intrinsic, ligand-controlled tyrosine-kinase activity.  They regulate diverse functions in normal cells and have a crucial role in oncogenesis.  Twenty years ago, the 1st primary structure of a receptor Tyr kinase, the epidermal growth factor receptor, was elucidated.  The characterization of both the mol. architecture of receptor Tyr kinases and the main functions of these proteins and their ligands in tumorigenesis opened the door to a new era in mol. oncol. and paved the way to the development of the 1st target-specific cancer therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo1NLVNn4dw37Vg90H21EOLACvtfcHk0ljIsvcZJV2D_Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXjsFSlu7g%253D&md5=ea24c0ccaea68b743b5da9587ed19406</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1038%2Fnrc1360&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc1360%26sid%3Dliteratum%253Aachs%26aulast%3DGschwind%26aufirst%3DA.%26aulast%3DFischer%26aufirst%3DO.%2BM.%26aulast%3DUllrich%26aufirst%3DA.%26atitle%3DThe%2520discovery%2520of%2520receptor%2520tyrosine%2520kinases%253A%2520targets%2520for%2520cancer%2520therapy%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2004%26volume%3D4%26spage%3D361%26epage%3D370%26doi%3D10.1038%2Fnrc1360" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Aertgeerts, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skene, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yano, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sang, B.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zou, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Snell, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jennings, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iwamoto, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Habuka, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hirokawa, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ishikawa, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanaka, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miki, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohta, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sogabe, S.</span></span> <span> </span><span class="NLM_article-title">Structural analysis of the mechanism of inhibition and allosteric activation of the kinase domain of HER2 protein</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>286</i></span>,  <span class="NLM_fpage">18756</span>– <span class="NLM_lpage">18765</span>, <span class="refDoi"> DOI: 10.1074/jbc.m110.206193</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00870&amp;key=10.1074%2Fjbc.m110.206193" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00870&amp;key=21454582" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00870&amp;key=1%3ACAS%3A528%3ADC%252BC3MXmtlGltL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=286&publication_year=2011&pages=18756-18765&author=K.+Aertgeertsauthor=R.+Skeneauthor=J.+Yanoauthor=B.-C.+Sangauthor=H.+Zouauthor=G.+Snellauthor=A.+Jenningsauthor=K.+Iwamotoauthor=N.+Habukaauthor=A.+Hirokawaauthor=T.+Ishikawaauthor=T.+Tanakaauthor=H.+Mikiauthor=Y.+Ohtaauthor=S.+Sogabe&title=Structural+analysis+of+the+mechanism+of+inhibition+and+allosteric+activation+of+the+kinase+domain+of+HER2+protein&doi=10.1074%2Fjbc.m110.206193"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Structural Analysis of the Mechanism of Inhibition and Allosteric Activation of the Kinase Domain of HER2 Protein</span></div><div class="casAuthors">Aertgeerts, Kathleen; Skene, Robert; Yano, Jason; Sang, Bi-Ching; Zou, Hua; Snell, Gyorgy; Jennings, Andy; Iwamoto, Keiji; Habuka, Noriyuki; Hirokawa, Aki; Ishikawa, Tomoyasu; Tanaka, Toshimasa; Miki, Hiroshi; Ohta, Yoshikazu; Sogabe, Satoshi</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">286</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">18756-18765</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Aberrant signaling of ErbB family members human epidermal growth factor 2 (HER2) and epidermal growth factor receptor (EGFR) is implicated in many human cancers, and HER2 expression is predictive of human disease recurrence and prognosis.  Small mol. kinase inhibitors of EGFR and of both HER2 and EGFR have received approval for the treatment of cancer.  We present the first high resoln. crystal structure of the kinase domain of HER2 in complex with a selective inhibitor to understand protein activation, inhibition, and function at the mol. level.  HER2 kinase domain crystallizes as a dimer and suggests evidence for an allosteric mechanism of activation comparable with previously reported activation mechanisms for EGFR and HER4.  A unique Gly-rich region in HER2 following the α-helix C is responsible for increased conformational flexibility within the active site and could explain the low intrinsic catalytic activity previously reported for HER2.  In addn., we solved the crystal structure of the kinase domain of EGFR in complex with a HER2/EGFR dual inhibitor (TAK-285).  Comparison with previously reported inactive and active EGFR kinase domain structures gave insight into the mechanism of HER2 and EGFR inhibition and may help guide the design and development of new cancer drugs with improved potency and selectivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGomtAlJ0FGUX7Vg90H21EOLACvtfcHk0lizykfL51wGCQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXmtlGltL0%253D&md5=c7897f5363d6452f7b60a8ced0f2bc30</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1074%2Fjbc.m110.206193&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.m110.206193%26sid%3Dliteratum%253Aachs%26aulast%3DAertgeerts%26aufirst%3DK.%26aulast%3DSkene%26aufirst%3DR.%26aulast%3DYano%26aufirst%3DJ.%26aulast%3DSang%26aufirst%3DB.-C.%26aulast%3DZou%26aufirst%3DH.%26aulast%3DSnell%26aufirst%3DG.%26aulast%3DJennings%26aufirst%3DA.%26aulast%3DIwamoto%26aufirst%3DK.%26aulast%3DHabuka%26aufirst%3DN.%26aulast%3DHirokawa%26aufirst%3DA.%26aulast%3DIshikawa%26aufirst%3DT.%26aulast%3DTanaka%26aufirst%3DT.%26aulast%3DMiki%26aufirst%3DH.%26aulast%3DOhta%26aufirst%3DY.%26aulast%3DSogabe%26aufirst%3DS.%26atitle%3DStructural%2520analysis%2520of%2520the%2520mechanism%2520of%2520inhibition%2520and%2520allosteric%2520activation%2520of%2520the%2520kinase%2520domain%2520of%2520HER2%2520protein%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2011%26volume%3D286%26spage%3D18756%26epage%3D18765%26doi%3D10.1074%2Fjbc.m110.206193" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lategahn, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keul, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rauh, D.</span></span> <span> </span><span class="NLM_article-title">Lessons to be learned: the molecular basis of kinase-targeted therapies and drug resistance in non-small cell lung cancer</span>. <i>Angew. Chem., Int. Ed. Engl.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">2307</span>– <span class="NLM_lpage">2313</span>, <span class="refDoi"> DOI: 10.1002/anie.201710398</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00870&amp;key=10.1002%2Fanie.201710398" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00870&amp;key=29178586" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00870&amp;key=1%3ACAS%3A280%3ADC%252BC1M3lt1yrsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2018&pages=2307-2313&author=J.+Lategahnauthor=M.+Keulauthor=D.+Rauh&title=Lessons+to+be+learned%3A+the+molecular+basis+of+kinase-targeted+therapies+and+drug+resistance+in+non-small+cell+lung+cancer&doi=10.1002%2Fanie.201710398"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Lessons To Be Learned: The Molecular Basis of Kinase-Targeted Therapies and Drug Resistance in Non-Small Cell Lung Cancer</span></div><div class="casAuthors">Lategahn Jonas; Keul Marina; Rauh Daniel</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie (International ed. in English)</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2307-2313</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The treatment of non-small cell lung cancer (NSCLC) is currently experiencing a revolution.  Over the last decade, the knowledge gained about the biochemical features of biomarkers and their predictive abilities has led to the development of targeted small-molecule inhibitors that present an alternative to harsh chemotherapy.  The use of these new therapies has improved the quality of life and increased the survival of patients.  The occurrence of inevitable drug resistance requires the constant development of precision medicine.  The detailed understanding of the target biology and the search for innovative chemical approaches has encouraged investigations in this field.  Herein, we review selected aspects of the molecular targets and present an overview of current topics and challenges in the rational development of small molecules to target NSCLC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQ1En_K7f1V4brIbt-tGZmxfW6udTcc2eaQqxJ6sYdgJLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1M3lt1yrsA%253D%253D&md5=634b74865cb6382392e8846e02de3810</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1002%2Fanie.201710398&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201710398%26sid%3Dliteratum%253Aachs%26aulast%3DLategahn%26aufirst%3DJ.%26aulast%3DKeul%26aufirst%3DM.%26aulast%3DRauh%26aufirst%3DD.%26atitle%3DLessons%2520to%2520be%2520learned%253A%2520the%2520molecular%2520basis%2520of%2520kinase-targeted%2520therapies%2520and%2520drug%2520resistance%2520in%2520non-small%2520cell%2520lung%2520cancer%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%2520Engl.%26date%3D2018%26volume%3D57%26spage%3D2307%26epage%3D2313%26doi%3D10.1002%2Fanie.201710398" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Niggenaber, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hardick, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lategahn, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rauh, D.</span></span> <span> </span><span class="NLM_article-title">Structure defines function: clinically relevant mutations in ErbB kinases</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">40</span>– <span class="NLM_lpage">51</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b00964</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b00964" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00870&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhsFymtLbK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=40-51&author=J.+Niggenaberauthor=J.+Hardickauthor=J.+Lategahnauthor=D.+Rauh&title=Structure+defines+function%3A+clinically+relevant+mutations+in+ErbB+kinases&doi=10.1021%2Facs.jmedchem.9b00964"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Structure Defines Function: Clinically Relevant Mutations in ErbB Kinases</span></div><div class="casAuthors">Niggenaber, Janina; Hardick, Julia; Lategahn, Jonas; Rauh, Daniel</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">40-51</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The ErbB receptor tyrosine kinase family members EGFR (epidermal growth factor receptor) and Her2 are among the prominent mutated oncogenic drivers of non-small cell lung cancer (NSCLC).  Their importance in proliferation, apoptosis, and cell death ultimately renders them hot targets in cancer therapy.  Small-mol. tyrosine kinase inhibitors seem well suited to be tailor-made therapeutics for EGFR mutant NSCLC; however, drug resistance mutations limit their success.  Against this background, the elucidation and visualization of the three-dimensional structure of cancer-related kinases provide valuable insights into their mol. functions.  This field has undergone a revolution because X-ray crystal structure detns. aided structure-based drug design approaches and clarified the effect of activating and resistance-conferring mutations.  Here, we present an overview of important mutations affecting EGFR and Her2 and highlight their influence on the kinase domain conformations and active site accessibility.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqgcJQfrfEznLVg90H21EOLACvtfcHk0lizykfL51wGCQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhsFymtLbK&md5=e9f13c582aeeca169e66ae119aadf600</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b00964&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b00964%26sid%3Dliteratum%253Aachs%26aulast%3DNiggenaber%26aufirst%3DJ.%26aulast%3DHardick%26aufirst%3DJ.%26aulast%3DLategahn%26aufirst%3DJ.%26aulast%3DRauh%26aufirst%3DD.%26atitle%3DStructure%2520defines%2520function%253A%2520clinically%2520relevant%2520mutations%2520in%2520ErbB%2520kinases%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D40%26epage%3D51%26doi%3D10.1021%2Facs.jmedchem.9b00964" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hata, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yoshioka, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fujita, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kunimasa, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaji, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Imai, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomii, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iwasaku, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nishiyama, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ishida, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Katakami, N.</span></span> <span> </span><span class="NLM_article-title">Complex mutations in the epidermal growth factor receptor gene in non-small cell lung cancer</span>. <i>J. Thorac. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">1524</span>– <span class="NLM_lpage">1528</span>, <span class="refDoi"> DOI: 10.1097/jto.0b013e3181e8b3c5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00870&amp;key=10.1097%2FJTO.0b013e3181e8b3c5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00870&amp;key=20808254" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00870&amp;key=1%3ACAS%3A280%3ADC%252BC3cfkvFSisA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2010&pages=1524-1528&author=A.+Hataauthor=H.+Yoshiokaauthor=S.+Fujitaauthor=K.+Kunimasaauthor=R.+Kajiauthor=Y.+Imaiauthor=K.+Tomiiauthor=M.+Iwasakuauthor=A.+Nishiyamaauthor=T.+Ishidaauthor=N.+Katakami&title=Complex+mutations+in+the+epidermal+growth+factor+receptor+gene+in+non-small+cell+lung+cancer&doi=10.1097%2Fjto.0b013e3181e8b3c5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Complex mutations in the epidermal growth factor receptor gene in non-small cell lung cancer</span></div><div class="casAuthors">Hata Akito; Yoshioka Hiroshige; Fujita Shiro; Kunimasa Kei; Kaji Reiko; Imai Yukihiro; Tomii Keisuke; Iwasaku Masahiro; Nishiyama Akihiro; Ishida Tadashi; Katakami Nobuyuki</div><div class="citationInfo"><span class="NLM_cas:title">Journal of thoracic oncology : official publication of the International Association for the Study of Lung Cancer</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1524-8</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">INTRODUCTION:  Mutation of the epidermal growth factor receptor (EGFR) gene can predict the efficacy of EGFR-tyrosine kinase inhibitors.  Different mutations have been shown to co-occur in a single tumor.  However, the frequency of these so-called "complex mutations" and the efficacy of gefitinib in treating patients with these mutations are unclear.  METHODS:  We investigated the frequency of complex mutations in 783 patients with non-small cell lung cancer seen at our institutes between April 2006 and May 2009.  Mutational analysis was performed using the peptide nucleic acid-locked nucleic acid polymerase chain reaction clamp method.  Gefitinib efficacy was evaluated in patients found to have complex mutations.  RESULTS:  EGFR: mutations were detected in 318 (41%) patients, with 21 (6.6%) of these individuals having complex mutations.  Sixteen of these 21 patients received gefitinib.  The response rate (RR) was 67% (95% confidence interval [CI], 35-90%) and median progression-free survival was 12.2 months (95% CI, 1.3 months to undeterminable).  Analysis of RR according to mutation type revealed that patients with deletional mutation in exon 19 (Del-19) and a point mutation in exon 21 (L858R) had a better RR (86%, 6 of 7) than those with other complex mutation patterns such as a point mutation in exon 18 (G719S) + L858R (40%, 2 of 5) (p = 0.2222).  The median progression-free survival was also longer in these patients (16.5 months; 95% CI, 1.1 months to undeterminable versus 3.8 months; 95% CI, 0.7-10.0 months) (p = 0.0459).  CONCLUSIONS:  Complex EGFR mutations are not rare.  Gefitinib has different efficacy according to the type of complex EGFR mutations.  Patients with Del-19 and L858R mutations may benefit more from gefitinib than other types of complex mutations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTZq7EL0Eg8p7HHuJhbyicZfW6udTcc2ebYT3ZoNDlPSLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3cfkvFSisA%253D%253D&md5=68353796c0df45267d25c4e4be03a305</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1097%2FJTO.0b013e3181e8b3c5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1097%252FJTO.0b013e3181e8b3c5%26sid%3Dliteratum%253Aachs%26aulast%3DHata%26aufirst%3DA.%26aulast%3DYoshioka%26aufirst%3DH.%26aulast%3DFujita%26aufirst%3DS.%26aulast%3DKunimasa%26aufirst%3DK.%26aulast%3DKaji%26aufirst%3DR.%26aulast%3DImai%26aufirst%3DY.%26aulast%3DTomii%26aufirst%3DK.%26aulast%3DIwasaku%26aufirst%3DM.%26aulast%3DNishiyama%26aufirst%3DA.%26aulast%3DIshida%26aufirst%3DT.%26aulast%3DKatakami%26aufirst%3DN.%26atitle%3DComplex%2520mutations%2520in%2520the%2520epidermal%2520growth%2520factor%2520receptor%2520gene%2520in%2520non-small%2520cell%2520lung%2520cancer%26jtitle%3DJ.%2520Thorac.%2520Oncol.%26date%3D2010%26volume%3D5%26spage%3D1524%26epage%3D1528%26doi%3D10.1097%2Fjto.0b013e3181e8b3c5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ambrogio, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shimamura, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kubo, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takahashi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chirieac, L. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Padera, R. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shapiro, G. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baum, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Himmelsbach, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rettig, W. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meyerson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Solca, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greulich, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, K.-K.</span></span> <span> </span><span class="NLM_article-title">BIBW2992, an irreversible EGFR/HER2 inhibitor highly effective in preclinical lung cancer models</span>. <i>Oncogene</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">4702</span>– <span class="NLM_lpage">4711</span>, <span class="refDoi"> DOI: 10.1038/onc.2008.109</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00870&amp;key=10.1038%2Fonc.2008.109" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00870&amp;key=18408761" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00870&amp;key=1%3ACAS%3A528%3ADC%252BD1cXptlKnsbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2008&pages=4702-4711&author=D.+Liauthor=L.+Ambrogioauthor=T.+Shimamuraauthor=S.+Kuboauthor=M.+Takahashiauthor=L.+R.+Chirieacauthor=R.+F.+Paderaauthor=G.+I.+Shapiroauthor=A.+Baumauthor=F.+Himmelsbachauthor=W.+J.+Rettigauthor=M.+Meyersonauthor=F.+Solcaauthor=H.+Greulichauthor=K.-K.+Wong&title=BIBW2992%2C+an+irreversible+EGFR%2FHER2+inhibitor+highly+effective+in+preclinical+lung+cancer+models&doi=10.1038%2Fonc.2008.109"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">BIBW2992, an irreversible EGFR/HER2 inhibitor highly effective in preclinical lung cancer models</span></div><div class="casAuthors">Li, D.; Ambrogio, L.; Shimamura, T.; Kubo, S.; Takahashi, M.; Chirieac, L. R.; Padera, R. F.; Shapiro, G. I.; Baum, A.; Himmelsbach, F.; Rettig, W. J.; Meyerson, M.; Solca, F.; Greulich, H.; Wong, K-K.</div><div class="citationInfo"><span class="NLM_cas:title">Oncogene</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">34</span>),
    <span class="NLM_cas:pages">4702-4711</span>CODEN:
                <span class="NLM_cas:coden">ONCNES</span>;
        ISSN:<span class="NLM_cas:issn">0950-9232</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Genetic alterations in the kinase domain of the epidermal growth factor receptor (EGFR) in non-small cell lung cancer (NSCLC) patients are assocd. with sensitivity to treatment with small mol. tyrosine kinase inhibitors.  Although first-generation reversible, ATP-competitive inhibitors showed encouraging clin. responses in lung adenocarcinoma tumors harboring such EGFR mutations, almost all patients developed resistance to these inhibitors over time.  Such resistance to first-generation EGFR inhibitors was frequently linked to an acquired T790M point mutation in the kinase domain of EGFR, or upregulation of signaling pathways downstream of HER3.  Overcoming these mechanisms of resistance, as well as primary resistance to reversible EGFR inhibitors driven by a subset of EGFR mutations, will be necessary for development of an effective targeted therapy regimen.  Here, we show that BIBW2992, an anilino-quinazoline designed to irreversibly bind EGFR and HER2, potently suppresses the kinase activity of wild-type and activated EGFR and HER2 mutants, including erlotinib-resistant isoforms.  Consistent with this activity, BIBW2992 suppresses transformation in isogenic cell-based assays, inhibits survival of cancer cell lines and induces tumor regression in xenograft and transgenic lung cancer models, with superior activity over erlotinib.  These findings encourage further testing of BIBW2992 in lung cancer patients harboring EGFR or HER2 oncogenes.  Oncogene (2008) 27, 4702-4711; doi:10.1038/onc.2008.109; published online 14 Apr. 2008.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo5PQdKep-Sp7Vg90H21EOLACvtfcHk0lh5jsHMtGYb4Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXptlKnsbk%253D&md5=24945893e83b4e3d01c267100fdaf57c</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1038%2Fonc.2008.109&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fonc.2008.109%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DD.%26aulast%3DAmbrogio%26aufirst%3DL.%26aulast%3DShimamura%26aufirst%3DT.%26aulast%3DKubo%26aufirst%3DS.%26aulast%3DTakahashi%26aufirst%3DM.%26aulast%3DChirieac%26aufirst%3DL.%2BR.%26aulast%3DPadera%26aufirst%3DR.%2BF.%26aulast%3DShapiro%26aufirst%3DG.%2BI.%26aulast%3DBaum%26aufirst%3DA.%26aulast%3DHimmelsbach%26aufirst%3DF.%26aulast%3DRettig%26aufirst%3DW.%2BJ.%26aulast%3DMeyerson%26aufirst%3DM.%26aulast%3DSolca%26aufirst%3DF.%26aulast%3DGreulich%26aufirst%3DH.%26aulast%3DWong%26aufirst%3DK.-K.%26atitle%3DBIBW2992%252C%2520an%2520irreversible%2520EGFR%252FHER2%2520inhibitor%2520highly%2520effective%2520in%2520preclinical%2520lung%2520cancer%2520models%26jtitle%3DOncogene%26date%3D2008%26volume%3D27%26spage%3D4702%26epage%3D4711%26doi%3D10.1038%2Fonc.2008.109" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dungo, R. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keating, G. M.</span></span> <span> </span><span class="NLM_article-title">Afatinib: first global approval</span>. <i>Drugs</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>73</i></span>,  <span class="NLM_fpage">1503</span>– <span class="NLM_lpage">1515</span>, <span class="refDoi"> DOI: 10.1007/s40265-013-0111-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00870&amp;key=10.1007%2Fs40265-013-0111-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00870&amp;key=23982599" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00870&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvVWmurnM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=73&publication_year=2013&pages=1503-1515&author=R.+T.+Dungoauthor=G.+M.+Keating&title=Afatinib%3A+first+global+approval&doi=10.1007%2Fs40265-013-0111-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Afatinib: First Global Approval</span></div><div class="casAuthors">Dungo, Rosselle T.; Keating, Gillian M.</div><div class="citationInfo"><span class="NLM_cas:title">Drugs</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">73</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">1503-1515</span>CODEN:
                <span class="NLM_cas:coden">DRUGAY</span>;
        ISSN:<span class="NLM_cas:issn">0012-6667</span>.
    
            (<span class="NLM_cas:orgname">Springer International Publishing AG</span>)
        </div><div class="casAbstract">A review.  Afatinib, an irreversible inhibitor of the ErbB family of tyrosine kinases, is under development with Boehringer Ingelheim for the once-daily, oral treatment of cancer.  Afatinib downregulates ErbB signalling by covalently binding to epidermal growth factor receptor (EGFR), human epidermal growth factor receptor (HER) 2 and HER4, irreversibly inhibiting tyrosine kinase autophosphorylation.  It also inhibits transphosphorylation of HER3.  Oral afatinib (Gilotrif®) has been approved in the US for the first-line treatment of patients with metastatic non-small-cell lung cancer (NSCLC) who have tumors with EGFR exon 19 deletions or exon 21 (L858R) substitution mutations as detected by a US FDA-approved test.  Afatinib has also been approved in Taiwan for the first-line treatment of patients with EGFR mutation-pos. NSCLC.  In addn., the European Medicines Agency's Committee for Medicinal Products for Human Use has recommended the approval of afatinib (Giotrif) for the treatment of patients with locally advanced or metastatic NSCLC with activating EGFR mutations who are EGFR tyrosine kinase inhibitor naive.  Afatinib is also under regulatory review in Canada, Japan and other Asian countries.  This article summarizes the milestones in the development of afatinib, leading to this first approval in patients with metastatic NSCLC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrdPcU5aIqrY7Vg90H21EOLACvtfcHk0lh5jsHMtGYb4Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvVWmurnM&md5=21851cbfac56e2fffcba6c731f123cff</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1007%2Fs40265-013-0111-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs40265-013-0111-6%26sid%3Dliteratum%253Aachs%26aulast%3DDungo%26aufirst%3DR.%2BT.%26aulast%3DKeating%26aufirst%3DG.%2BM.%26atitle%3DAfatinib%253A%2520first%2520global%2520approval%26jtitle%3DDrugs%26date%3D2013%26volume%3D73%26spage%3D1503%26epage%3D1515%26doi%3D10.1007%2Fs40265-013-0111-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cha, M. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, K.-O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, J. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chae, Y. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, Y. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suh, K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, S. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, M. S.</span></span> <span> </span><span class="NLM_article-title">Antitumor activity of HM781-36B, a highly effective pan-HER inhibitor in erlotinib-resistant NSCLC and other EGFR-dependent cancer models</span>. <i>Int. J. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>130</i></span>,  <span class="NLM_fpage">2445</span>– <span class="NLM_lpage">2454</span>, <span class="refDoi"> DOI: 10.1002/ijc.26276</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00870&amp;key=10.1002%2Fijc.26276" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00870&amp;key=21732342" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00870&amp;key=1%3ACAS%3A528%3ADC%252BC38XjvVajtr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=130&publication_year=2012&pages=2445-2454&author=M.+Y.+Chaauthor=K.-O.+Leeauthor=M.+Kimauthor=J.+Y.+Songauthor=K.+H.+Leeauthor=J.+Parkauthor=Y.+J.+Chaeauthor=Y.+H.+Kimauthor=K.+H.+Suhauthor=G.+S.+Leeauthor=S.+B.+Parkauthor=M.+S.+Kim&title=Antitumor+activity+of+HM781-36B%2C+a+highly+effective+pan-HER+inhibitor+in+erlotinib-resistant+NSCLC+and+other+EGFR-dependent+cancer+models&doi=10.1002%2Fijc.26276"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Antitumor activity of HM781-36B, a highly effective pan-HER inhibitor in erlotinib-resistant NSCLC and other EGFR-dependent cancer models</span></div><div class="casAuthors">Cha, Mi Young; Lee, Kwang-Ok; Kim, Mira; Song, Ji Yeon; Lee, Kyu Hang; Park, Jongmin; Chae, Yun Jung; Kim, Young Hoon; Suh, Kwee Hyun; Lee, Gwan Sun; Park, Seung Bum; Kim, Maeng Sup</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Cancer</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">130</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2445-2454</span>CODEN:
                <span class="NLM_cas:coden">IJCNAW</span>;
        ISSN:<span class="NLM_cas:issn">0020-7136</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">The epidermal growth factor receptor (EGFR) family of receptor tyrosine kinases has been implicated in a variety of cancers.  In particular, activating mutations such as the L858R point mutation in exon 21 and the small in-frame deletions in exon 19 of the EGFR tyrosine kinase domain are correlated with sensitivity to EGFR tyrosine kinase inhibitors in non-small cell lung cancer (NSCLC) patients.  Clin. treatment of patients is limited by the development of drug resistance resulting mainly from a gatekeeper mutation (T790M).  In this study, we evaluated the therapeutic potential of a novel, irreversible pan-HER inhibitor, HM781-36B.  The results from this study show that HM781-36B is a potent inhibitor of EGFR in vitro, including the EGFR-acquired resistance mutation (T790M), as well as HER-2 and HER-4, compared with other EGFR tyrosine kinases inhibitors (erlotinib, lapatinib and BIBW2992).  HM781-36B treatment of EGFR DelE746A750-harboring erlotinib-sensitive HCC827 and EGFR L858R/T790M-harboring erlotinib-resistant NCI-H1975 NSCLC cells results in the inhibition of EGFR phosphorylation and the subsequent deactivation of downstream signaling proteins.  Addnl., HM781-36B shows an excellent efficacy in a variety of EGFR- and HER-2-dependent tumor xenograft models, including erlotinib-sensitive HCC827 NSCLC cells, erlotinib-resistant NCI-H1975 NSCLC cells, HER-2 overexpressing Calu-3 NSCLC cells, NCI-N87 gastric cancer cells, SK-Ov3 ovarian cancer cells and EGFR-overexpressing A431 epidermoid carcinoma cancer cells.  On the basis of these preclin. results, HM781-36B is the most potent pan-HER inhibitor, which will be advantageous for the treatment of patients with NSCLC including clin. limitation caused by acquired mutation (EGFR T790M), breast cancer and gastric cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrT8jGQHJHfWrVg90H21EOLACvtfcHk0lik7dZD5PzYNA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XjvVajtr8%253D&md5=004f3a840f97464644b3a7466d636748</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1002%2Fijc.26276&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fijc.26276%26sid%3Dliteratum%253Aachs%26aulast%3DCha%26aufirst%3DM.%2BY.%26aulast%3DLee%26aufirst%3DK.-O.%26aulast%3DKim%26aufirst%3DM.%26aulast%3DSong%26aufirst%3DJ.%2BY.%26aulast%3DLee%26aufirst%3DK.%2BH.%26aulast%3DPark%26aufirst%3DJ.%26aulast%3DChae%26aufirst%3DY.%2BJ.%26aulast%3DKim%26aufirst%3DY.%2BH.%26aulast%3DSuh%26aufirst%3DK.%2BH.%26aulast%3DLee%26aufirst%3DG.%2BS.%26aulast%3DPark%26aufirst%3DS.%2BB.%26aulast%3DKim%26aufirst%3DM.%2BS.%26atitle%3DAntitumor%2520activity%2520of%2520HM781-36B%252C%2520a%2520highly%2520effective%2520pan-HER%2520inhibitor%2520in%2520erlotinib-resistant%2520NSCLC%2520and%2520other%2520EGFR-dependent%2520cancer%2520models%26jtitle%3DInt.%2520J.%2520Cancer%26date%3D2012%26volume%3D130%26spage%3D2445%26epage%3D2454%26doi%3D10.1002%2Fijc.26276" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Robichaux, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elamin, Y. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carter, B. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poteete, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Estrada-Bernal, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Le, A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Truini, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nilsson, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roarty, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goldberg, S. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brahmer, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Altan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Papadimitrakopoulou, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Politi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doebele, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, K.-K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heymach, J. V.</span></span> <span> </span><span class="NLM_article-title">Mechanisms and clinical activity of an EGFR and HER2 exon 20-selective kinase inhibitor in non-small cell lung cancer</span>. <i>Nat. Med.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">638</span>– <span class="NLM_lpage">646</span>, <span class="refDoi"> DOI: 10.1038/s41591-018-0007-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00870&amp;key=10.1038%2Fs41591-018-0007-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00870&amp;key=29686424" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00870&amp;key=1%3ACAS%3A528%3ADC%252BC1cXosVCnsLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2018&pages=638-646&author=J.+P.+Robichauxauthor=Y.+Y.+Elaminauthor=Z.+Tanauthor=B.+W.+Carterauthor=S.+Zhangauthor=S.+Liuauthor=S.+Liauthor=T.+Chenauthor=A.+Poteeteauthor=A.+Estrada-Bernalauthor=A.+T.+Leauthor=A.+Truiniauthor=M.+B.+Nilssonauthor=H.+Sunauthor=E.+Roartyauthor=S.+B.+Goldbergauthor=J.+R.+Brahmerauthor=M.+Altanauthor=C.+Luauthor=V.+Papadimitrakopoulouauthor=K.+Politiauthor=R.+C.+Doebeleauthor=K.-K.+Wongauthor=J.+V.+Heymach&title=Mechanisms+and+clinical+activity+of+an+EGFR+and+HER2+exon+20-selective+kinase+inhibitor+in+non-small+cell+lung+cancer&doi=10.1038%2Fs41591-018-0007-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Mechanisms and clinical activity of an EGFR and HER2 exon 20-selective kinase inhibitor in non-small cell lung cancer</span></div><div class="casAuthors">Robichaux, Jacqulyne P.; Elamin, Yasir Y.; Tan, Zhi; Carter, Brett W.; Zhang, Shuxing; Liu, Shengwu; Li, Shuai; Chen, Ting; Poteete, Alissa; Estrada-Bernal, Adriana; Le, Anh T.; Truini, Anna; Nilsson, Monique B.; Sun, Huiying; Roarty, Emily; Goldberg, Sarah B.; Brahmer, Julie R.; Altan, Mehmet; Lu, Charles; Papadimitrakopoulou, Vassiliki; Politi, Katerina; Doebele, Robert C.; Wong, Kwok-Kin; Heymach, John V.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Medicine (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">638-646</span>CODEN:
                <span class="NLM_cas:coden">NAMEFI</span>;
        ISSN:<span class="NLM_cas:issn">1078-8956</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Although most activating mutations of epidermal growth factor receptor (EGFR)-mutant non-small cell lung cancers (NSCLCs) are sensitive to available EGFR tyrosine kinase inhibitors (TKIs), a subset with alterations in exon 20 of EGFR and HER2 are intrinsically resistant and lack an effective therapy.  We used in silico, in vitro, and in vivo testing to model structural alterations induced by exon 20 mutations and to identify effective inhibitors. 3D modeling indicated alterations restricted the size of the drug-binding pocket, limiting the binding of large, rigid inhibitors.  We found that poziotinib, owing to its small size and flexibility, can circumvent these steric changes and is a potent inhibitor of the most common EGFR and HER2 exon 20 mutants.  Poziotinib demonstrated greater activity than approved EGFR TKIs in vitro and in patient-derived xenograft models of EGFR or HER2 exon 20 mutant NSCLC and in genetically engineered mouse models of NSCLC.  In a phase 2 trial, the first 11 patients with NSCLC with EGFR exon 20 mutations receiving poziotinib had a confirmed objective response rate of 64%.  These data identify poziotinib as a potent, clin. active inhibitor of EGFR and HER2 exon 20 mutations and illuminate the mol. features of TKIs that may circumvent steric changes induced by these mutations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrk6UyVpv0h47Vg90H21EOLACvtfcHk0lhKIssL6EKXNg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXosVCnsLs%253D&md5=316a0a0765ebc9eec9b6460d27b28bb5</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1038%2Fs41591-018-0007-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41591-018-0007-9%26sid%3Dliteratum%253Aachs%26aulast%3DRobichaux%26aufirst%3DJ.%2BP.%26aulast%3DElamin%26aufirst%3DY.%2BY.%26aulast%3DTan%26aufirst%3DZ.%26aulast%3DCarter%26aufirst%3DB.%2BW.%26aulast%3DZhang%26aufirst%3DS.%26aulast%3DLiu%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DS.%26aulast%3DChen%26aufirst%3DT.%26aulast%3DPoteete%26aufirst%3DA.%26aulast%3DEstrada-Bernal%26aufirst%3DA.%26aulast%3DLe%26aufirst%3DA.%2BT.%26aulast%3DTruini%26aufirst%3DA.%26aulast%3DNilsson%26aufirst%3DM.%2BB.%26aulast%3DSun%26aufirst%3DH.%26aulast%3DRoarty%26aufirst%3DE.%26aulast%3DGoldberg%26aufirst%3DS.%2BB.%26aulast%3DBrahmer%26aufirst%3DJ.%2BR.%26aulast%3DAltan%26aufirst%3DM.%26aulast%3DLu%26aufirst%3DC.%26aulast%3DPapadimitrakopoulou%26aufirst%3DV.%26aulast%3DPoliti%26aufirst%3DK.%26aulast%3DDoebele%26aufirst%3DR.%2BC.%26aulast%3DWong%26aufirst%3DK.-K.%26aulast%3DHeymach%26aufirst%3DJ.%2BV.%26atitle%3DMechanisms%2520and%2520clinical%2520activity%2520of%2520an%2520EGFR%2520and%2520HER2%2520exon%252020-selective%2520kinase%2520inhibitor%2520in%2520non-small%2520cell%2520lung%2520cancer%26jtitle%3DNat.%2520Med.%26date%3D2018%26volume%3D24%26spage%3D638%26epage%3D646%26doi%3D10.1038%2Fs41591-018-0007-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Robichaux, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elamin, Y. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vijayan, R. S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nilsson, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pisegna, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poteete, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Negrao, M. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ahnert, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diao, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Le, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meric-Bernstam, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Routbort, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roeck, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raymond, V. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lanman, R. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frampton, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, V. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schrock, A. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Albacker, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, K.-k.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cross, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heymach, J. V.</span></span> <span> </span><span class="NLM_article-title">Pan-cancer landscape and analysis of ERBB2 mutations identifies poziotinib as a clinically active inhibitor and enhancer of T-DM1 activity</span>. <i>Cancer Cell</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>36</i></span>,  <span class="NLM_fpage">444</span>– <span class="NLM_lpage">457</span>, <span class="refDoi"> DOI: 10.1016/j.ccell.2019.09.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00870&amp;key=10.1016%2Fj.ccell.2019.09.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00870&amp;key=31588020" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00870&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhvFersr%252FP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2019&pages=444-457&author=J.+P.+Robichauxauthor=Y.+Y.+Elaminauthor=R.+S.+K.+Vijayanauthor=M.+B.+Nilssonauthor=L.+Huauthor=J.+Heauthor=F.+Zhangauthor=M.+Pisegnaauthor=A.+Poteeteauthor=H.+Sunauthor=S.+Liauthor=T.+Chenauthor=H.+Hanauthor=M.+V.+Negraoauthor=J.+R.+Ahnertauthor=L.+Diaoauthor=J.+Wangauthor=X.+Leauthor=F.+Meric-Bernstamauthor=M.+Routbortauthor=B.+Roeckauthor=Z.+Yangauthor=V.+M.+Raymondauthor=R.+B.+Lanmanauthor=G.+M.+Framptonauthor=V.+A.+Millerauthor=A.+B.+Schrockauthor=L.+A.+Albackerauthor=K.-k.+Wongauthor=J.+B.+Crossauthor=J.+V.+Heymach&title=Pan-cancer+landscape+and+analysis+of+ERBB2+mutations+identifies+poziotinib+as+a+clinically+active+inhibitor+and+enhancer+of+T-DM1+activity&doi=10.1016%2Fj.ccell.2019.09.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Pan-Cancer Landscape and Analysis of ERBB2 Mutations Identifies Poziotinib as a Clinically Active Inhibitor and Enhancer of T-DM1 Activity</span></div><div class="casAuthors">Robichaux, Jacqulyne P.; Elamin, Yasir Y.; Vijayan, R. S. K.; Nilsson, Monique B.; Hu, Lemei; He, Junqin; Zhang, Fahao; Pisegna, Marlese; Poteete, Alissa; Sun, Huiying; Li, Shuai; Chen, Ting; Han, Han; Vailati Negrao, Marcelo; Ahnert, Jordi Rodon; Diao, Lixia; Wang, Jing; Le, Xiuning; Meric-Bernstam, Funda; Routbort, Mark; Roeck, Brent; Yang, Zane; Raymond, Victoria M.; Lanman, Richard B.; Frampton, Garrett M.; Miller, Vincent A.; Schrock, Alexa B.; Albacker, Lee A.; Wong, Kwok-kin; Cross, Jason B.; Heymach, John V.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">444-457.e7</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">We characterized the landscape and drug sensitivity of ERBB2 (HER2) mutations in cancers.  In 11 datasets (n = 211,726), ERBB2 mutational hotspots varied across 25 tumor types.  Common HER2 mutants yielded differential sensitivities to eleven EGFR/HER2 tyrosine kinase inhibitors (TKIs) in vitro, and mol. dynamics simulations revealed that mutants with a reduced drug-binding pocket vol. were assocd. with decreased affinity for larger TKIs.  Overall, poziotinib was the most potent HER2 mutant-selective TKI tested.  Phase II clin. testing in ERBB2 exon 20-mutant non-small cell lung cancer resulted in a confirmed objective response rate of 42% in the first 12 evaluable patients.  In pre-clin. models, poziotinib upregulated HER2 cell-surface expression and potentiated the activity of T-DM1, resulting in complete tumor regression with combination treatment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpiTO9bz4Pd_rVg90H21EOLACvtfcHk0lj861QFE3CwMw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhvFersr%252FP&md5=cdcb488a797546e1c0f4ccac85e9e13c</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1016%2Fj.ccell.2019.09.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccell.2019.09.001%26sid%3Dliteratum%253Aachs%26aulast%3DRobichaux%26aufirst%3DJ.%2BP.%26aulast%3DElamin%26aufirst%3DY.%2BY.%26aulast%3DVijayan%26aufirst%3DR.%2BS.%2BK.%26aulast%3DNilsson%26aufirst%3DM.%2BB.%26aulast%3DHu%26aufirst%3DL.%26aulast%3DHe%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DF.%26aulast%3DPisegna%26aufirst%3DM.%26aulast%3DPoteete%26aufirst%3DA.%26aulast%3DSun%26aufirst%3DH.%26aulast%3DLi%26aufirst%3DS.%26aulast%3DChen%26aufirst%3DT.%26aulast%3DHan%26aufirst%3DH.%26aulast%3DNegrao%26aufirst%3DM.%2BV.%26aulast%3DAhnert%26aufirst%3DJ.%2BR.%26aulast%3DDiao%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DLe%26aufirst%3DX.%26aulast%3DMeric-Bernstam%26aufirst%3DF.%26aulast%3DRoutbort%26aufirst%3DM.%26aulast%3DRoeck%26aufirst%3DB.%26aulast%3DYang%26aufirst%3DZ.%26aulast%3DRaymond%26aufirst%3DV.%2BM.%26aulast%3DLanman%26aufirst%3DR.%2BB.%26aulast%3DFrampton%26aufirst%3DG.%2BM.%26aulast%3DMiller%26aufirst%3DV.%2BA.%26aulast%3DSchrock%26aufirst%3DA.%2BB.%26aulast%3DAlbacker%26aufirst%3DL.%2BA.%26aulast%3DWong%26aufirst%3DK.-k.%26aulast%3DCross%26aufirst%3DJ.%2BB.%26aulast%3DHeymach%26aufirst%3DJ.%2BV.%26atitle%3DPan-cancer%2520landscape%2520and%2520analysis%2520of%2520ERBB2%2520mutations%2520identifies%2520poziotinib%2520as%2520a%2520clinically%2520active%2520inhibitor%2520and%2520enhancer%2520of%2520T-DM1%2520activity%26jtitle%3DCancer%2520Cell%26date%3D2019%26volume%3D36%26spage%3D444%26epage%3D457%26doi%3D10.1016%2Fj.ccell.2019.09.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Smaill, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gonzales, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spicer, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reed, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sexton, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Althaus, I. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Black, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blaser, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Denny, W. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ellis, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fakhoury, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harvey, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hook, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCarthy, F. O. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Palmer, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rivault, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schlosser, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ellis, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thompson, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trachet, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winters, R. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tecle, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bridges, A.</span></span> <span> </span><span class="NLM_article-title">Tyrosine kinase kihibitors. 20. Optimization of substituted quinazoline and pyrido[3,4-d]pyrimidine derivatives as orally active, irreversible inhibitors of the epidermal growth factor receptor family</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">8103</span>– <span class="NLM_lpage">8124</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b00883</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b00883" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00870&amp;key=1%3ACAS%3A528%3ADC%252BC28Xht12gtbvM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=8103-8124&author=J.+B.+Smaillauthor=A.+J.+Gonzalesauthor=J.+A.+Spicerauthor=H.+Leeauthor=J.+E.+Reedauthor=K.+Sextonauthor=I.+W.+Althausauthor=T.+Zhuauthor=S.+L.+Blackauthor=A.+Blaserauthor=W.+A.+Dennyauthor=P.+A.+Ellisauthor=S.+Fakhouryauthor=P.+J.+Harveyauthor=K.+Hookauthor=F.+O.+J.+McCarthyauthor=B.+D.+Palmerauthor=F.+Rivaultauthor=K.+Schlosserauthor=T.+Ellisauthor=A.+M.+Thompsonauthor=E.+Trachetauthor=R.+T.+Wintersauthor=H.+Tecleauthor=A.+Bridges&title=Tyrosine+kinase+kihibitors.+20.+Optimization+of+substituted+quinazoline+and+pyrido%5B3%2C4-d%5Dpyrimidine+derivatives+as+orally+active%2C+irreversible+inhibitors+of+the+epidermal+growth+factor+receptor+family&doi=10.1021%2Facs.jmedchem.6b00883"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Tyrosine Kinase Inhibitors. 20. Optimization of Substituted Quinazoline and Pyrido[3,4-d]pyrimidine Derivatives as Orally Active, Irreversible Inhibitors of the Epidermal Growth Factor Receptor Family</span></div><div class="casAuthors">Smaill, Jeff B.; Gonzales, Andrea J.; Spicer, Julie A.; Lee, Helen; Reed, Jessica E.; Sexton, Karen; Althaus, Irene W.; Zhu, Tong; Black, Shannon L.; Blaser, Adrian; Denny, William A.; Ellis, Paul A.; Fakhoury, Stephen; Harvey, Patricia J.; Hook, Ken; McCarthy, Florence O. J.; Palmer, Brian D.; Rivault, Freddy; Schlosser, Kevin; Ellis, Teresa; Thompson, Andrew M.; Trachet, Erin; Winters, R. Thomas; Tecle, Haile; Bridges, Alexander</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">8103-8124</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Structure-activity relationships for inhibition of erbB1, erbB2 and erbB4 were detd. for a series of quinazoline- and pyrido[3,4-d]pyrimidine-based analogs of the irreversible pan-erbB inhibitor, canertinib.  Cyclic amine bearing crotonamides were detd. to provide rapid inhibition of cellular erbB1 autophosphorylation and good metabolic stability in liver microsome and hepatocyte assays.  The influence of 4-anilino substitution on pan-erbB inhibitory potency was investigated.  Several anilines were identified as providing potent, reversible pan-erbB inhibition.  Optimum 4- and 6-substituents with known 7-substituents provided preferred irreversible inhibitors for pharmacodynamic testing in vivo.  Quinazoline I and pyrido[3,4-d]pyrimidine II were identified as clearly superior to canertinib.  Both compds. possess a piperidinyl crotonamide Michael acceptor and a 3-chloro-4-fluoroaniline, indicating these as optimized 6- and 4-substituents resp.  Pharmacokinetic comparison of compds. I and II across three species selected compd. I as the preferred candidate.  Compd. I (PF-00299804) has been assigned the nomenclature of dacomitinib and is currently under clin. evaluation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo0-NpOmVq10bVg90H21EOLACvtfcHk0lge1FI_6VJD3w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xht12gtbvM&md5=442afda669a5c1b7a190cd82b77bee33</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b00883&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b00883%26sid%3Dliteratum%253Aachs%26aulast%3DSmaill%26aufirst%3DJ.%2BB.%26aulast%3DGonzales%26aufirst%3DA.%2BJ.%26aulast%3DSpicer%26aufirst%3DJ.%2BA.%26aulast%3DLee%26aufirst%3DH.%26aulast%3DReed%26aufirst%3DJ.%2BE.%26aulast%3DSexton%26aufirst%3DK.%26aulast%3DAlthaus%26aufirst%3DI.%2BW.%26aulast%3DZhu%26aufirst%3DT.%26aulast%3DBlack%26aufirst%3DS.%2BL.%26aulast%3DBlaser%26aufirst%3DA.%26aulast%3DDenny%26aufirst%3DW.%2BA.%26aulast%3DEllis%26aufirst%3DP.%2BA.%26aulast%3DFakhoury%26aufirst%3DS.%26aulast%3DHarvey%26aufirst%3DP.%2BJ.%26aulast%3DHook%26aufirst%3DK.%26aulast%3DMcCarthy%26aufirst%3DF.%2BO.%2BJ.%26aulast%3DPalmer%26aufirst%3DB.%2BD.%26aulast%3DRivault%26aufirst%3DF.%26aulast%3DSchlosser%26aufirst%3DK.%26aulast%3DEllis%26aufirst%3DT.%26aulast%3DThompson%26aufirst%3DA.%2BM.%26aulast%3DTrachet%26aufirst%3DE.%26aulast%3DWinters%26aufirst%3DR.%2BT.%26aulast%3DTecle%26aufirst%3DH.%26aulast%3DBridges%26aufirst%3DA.%26atitle%3DTyrosine%2520kinase%2520kihibitors.%252020.%2520Optimization%2520of%2520substituted%2520quinazoline%2520and%2520pyrido%255B3%252C4-d%255Dpyrimidine%2520derivatives%2520as%2520orally%2520active%252C%2520irreversible%2520inhibitors%2520of%2520the%2520epidermal%2520growth%2520factor%2520receptor%2520family%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D8103%26epage%3D8124%26doi%3D10.1021%2Facs.jmedchem.6b00883" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gonzales, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hook, K. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Althaus, I. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ellis, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trachet, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Delaney, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harvey, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ellis, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amato, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nelson, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fry, D. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loi, C.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fakhoury, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schlosser, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sexton, K. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winters, R. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reed, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bridges, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lettiere, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baker, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, H. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tecle, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vincent, P. W.</span></span> <span> </span><span class="NLM_article-title">Antitumor activity and pharmacokinetic properties of PF-00299804, a second-generation irreversible pan-erbB receptor tyrosine kinase inhibitor</span>. <i>Mol. Cancer Ther.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">1880</span>– <span class="NLM_lpage">1889</span>, <span class="refDoi"> DOI: 10.1158/1535-7163.mct-07-2232</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00870&amp;key=10.1158%2F1535-7163.MCT-07-2232" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00870&amp;key=18606718" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00870&amp;key=1%3ACAS%3A528%3ADC%252BD1cXoslCgtb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2008&pages=1880-1889&author=A.+J.+Gonzalesauthor=K.+E.+Hookauthor=I.+W.+Althausauthor=P.+A.+Ellisauthor=E.+Trachetauthor=A.+M.+Delaneyauthor=P.+J.+Harveyauthor=T.+A.+Ellisauthor=D.+M.+Amatoauthor=J.+M.+Nelsonauthor=D.+W.+Fryauthor=T.+Zhuauthor=C.-M.+Loiauthor=S.+A.+Fakhouryauthor=K.+M.+Schlosserauthor=K.+E.+Sextonauthor=R.+T.+Wintersauthor=J.+E.+Reedauthor=A.+J.+Bridgesauthor=D.+J.+Lettiereauthor=D.+A.+Bakerauthor=J.+Yangauthor=H.+T.+Leeauthor=H.+Tecleauthor=P.+W.+Vincent&title=Antitumor+activity+and+pharmacokinetic+properties+of+PF-00299804%2C+a+second-generation+irreversible+pan-erbB+receptor+tyrosine+kinase+inhibitor&doi=10.1158%2F1535-7163.mct-07-2232"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Antitumor activity and pharmacokinetic properties of PF-00299804, a second-generation irreversible pan-erbB receptor tyrosine kinase inhibitor</span></div><div class="casAuthors">Gonzales, Andrea J.; Hook, Kenneth E.; Althaus, Irene W.; Ellis, Paul A.; Trachet, Erin; Delaney, Amy M.; Harvey, Patricia J.; Ellis, Teresa A.; Amato, Danielle M.; Nelson, James M.; Fry, David W.; Zhu, Tong; Loi, Cho-Ming; Fakhoury, Stephen A.; Schlosser, Kevin M.; Sexton, Karen E.; Winters, R. Thomas; Reed, Jessica E.; Bridges, Alex J.; Lettiere, Daniel J.; Baker, Deborah A.; Yang, Jianxin; Lee, Helen T.; Tecle, Haile; Vincent, Patrick W.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cancer Therapeutics</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1880-1889</span>CODEN:
                <span class="NLM_cas:coden">MCTOCF</span>;
        ISSN:<span class="NLM_cas:issn">1535-7163</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Signaling through the erbB receptor family of tyrosine kinases contributes to the proliferation, differentiation, migration, and survival of a variety of cell types.  Abnormalities in members of this receptor family have been shown to play a role in oncogenesis, thus making them attractive targets for anticancer treatments.  PF-00299804 is a second-generation irreversible pan-erbB receptor tyrosine kinase inhibitor currently in phase I clin. trials.  PF-00299804 is believed to irreversibly inhibit erbB tyrosine kinase activity through binding at the ATP site and covalent modification of nucleophilic cysteine residues in the catalytic domains of erbB family members.  Oral administration of PF-00299804 causes significant antitumor activity, including marked tumor regressions in a variety of human tumor xenograft models that express and/or overexpress erbB family members or contain the double mutation (L858R/T790M) in erbB1 (EGFR) assocd. with resistance to gefitinib and erlotinib.  Furthermore, PF-00299804 shows exceptional distribution to human tumor xenografts and excellent pharmacokinetic properties across species. [Mol Cancer Ther 2008;7(7):1880-9].</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpbL0hP8K2G37Vg90H21EOLACvtfcHk0ljU9yr7pWBZAg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXoslCgtb0%253D&md5=3392275d1e4f4d5d70e3b22d4893e3c8</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1158%2F1535-7163.MCT-07-2232&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1535-7163.MCT-07-2232%26sid%3Dliteratum%253Aachs%26aulast%3DGonzales%26aufirst%3DA.%2BJ.%26aulast%3DHook%26aufirst%3DK.%2BE.%26aulast%3DAlthaus%26aufirst%3DI.%2BW.%26aulast%3DEllis%26aufirst%3DP.%2BA.%26aulast%3DTrachet%26aufirst%3DE.%26aulast%3DDelaney%26aufirst%3DA.%2BM.%26aulast%3DHarvey%26aufirst%3DP.%2BJ.%26aulast%3DEllis%26aufirst%3DT.%2BA.%26aulast%3DAmato%26aufirst%3DD.%2BM.%26aulast%3DNelson%26aufirst%3DJ.%2BM.%26aulast%3DFry%26aufirst%3DD.%2BW.%26aulast%3DZhu%26aufirst%3DT.%26aulast%3DLoi%26aufirst%3DC.-M.%26aulast%3DFakhoury%26aufirst%3DS.%2BA.%26aulast%3DSchlosser%26aufirst%3DK.%2BM.%26aulast%3DSexton%26aufirst%3DK.%2BE.%26aulast%3DWinters%26aufirst%3DR.%2BT.%26aulast%3DReed%26aufirst%3DJ.%2BE.%26aulast%3DBridges%26aufirst%3DA.%2BJ.%26aulast%3DLettiere%26aufirst%3DD.%2BJ.%26aulast%3DBaker%26aufirst%3DD.%2BA.%26aulast%3DYang%26aufirst%3DJ.%26aulast%3DLee%26aufirst%3DH.%2BT.%26aulast%3DTecle%26aufirst%3DH.%26aulast%3DVincent%26aufirst%3DP.%2BW.%26atitle%3DAntitumor%2520activity%2520and%2520pharmacokinetic%2520properties%2520of%2520PF-00299804%252C%2520a%2520second-generation%2520irreversible%2520pan-erbB%2520receptor%2520tyrosine%2520kinase%2520inhibitor%26jtitle%3DMol.%2520Cancer%2520Ther.%26date%3D2008%26volume%3D7%26spage%3D1880%26epage%3D1889%26doi%3D10.1158%2F1535-7163.mct-07-2232" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rabindran, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Discafani, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosfjord, E. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baxter, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Floyd, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Golas, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hallett, W. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nilakantan, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Overbeek, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reich, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsou, H.-R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.-F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wissner, A.</span></span> <span> </span><span class="NLM_article-title">Antitumor activity of HKI-272, an orally active, irreversible inhibitor of the HER-2 tyrosine kinase</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>64</i></span>,  <span class="NLM_fpage">3958</span>– <span class="NLM_lpage">3965</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.can-03-2868</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00870&amp;key=10.1158%2F0008-5472.CAN-03-2868" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00870&amp;key=15173008" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00870&amp;key=1%3ACAS%3A528%3ADC%252BD2cXksVKns7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=64&publication_year=2004&pages=3958-3965&author=S.+K.+Rabindranauthor=C.+M.+Discafaniauthor=E.+C.+Rosfjordauthor=M.+Baxterauthor=M.+B.+Floydauthor=J.+Golasauthor=W.+A.+Hallettauthor=B.+D.+Johnsonauthor=R.+Nilakantanauthor=E.+Overbeekauthor=M.+F.+Reichauthor=R.+Shenauthor=X.+Shiauthor=H.-R.+Tsouauthor=Y.-F.+Wangauthor=A.+Wissner&title=Antitumor+activity+of+HKI-272%2C+an+orally+active%2C+irreversible+inhibitor+of+the+HER-2+tyrosine+kinase&doi=10.1158%2F0008-5472.can-03-2868"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Antitumor Activity of HKI-272, an Orally Active, Irreversible Inhibitor of the HER-2 Tyrosine Kinase</span></div><div class="casAuthors">Rabindran, Sridhar K.; Discafani, Carolyn M.; Rosfjord, Edward C.; Baxter, Michelle; Floyd, M. Brawner; Golas, Jonathan; Hallett, William A.; Johnson, Bernard D.; Nilakantan, Ramaswamy; Overbeek, Elsebe; Reich, Marvin F.; Shen, Ru; Shi, Xiaoqing; Tsou, Hwei-Ru; Wang, Yu-Fen; Wissner, Allan</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">64</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">3958-3965</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">HER-2 belongs to the ErbB family of receptor tyrosine kinases, which has been implicated in a variety of cancers.  Overexpression of HER-2 is seen in 25-30% of breast cancer patients and predicts a poor outcome in patients with primary disease.  Trastuzumab (Herceptin), a monoclonal antibody to HER-2, is specifically approved for HER-2-pos. breast cancer but is active only in a subset of these tumors.  Blocking HER-2 function by a small mol. kinase inhibitor, therefore, represents an attractive alternate strategy to inhibit the growth of HER-2-pos. tumors.  HKI-272 is a potent inhibitor of HER-2 and is highly active against HER-2-overexpressing human breast cancer cell lines in vitro.  It also inhibits the epidermal growth factor receptor (EGFR) kinase and the proliferation of EGFR-dependent cells.  HKI-272 reduces HER-2 receptor autophosphorylation in cells at doses consistent with inhibition of cell proliferation and functions as an irreversible binding inhibitor, most likely by targeting a cysteine residue in the ATP-binding pocket of the receptor.  In agreement with the predicted effects of HER-2 inactivation, HKI-272 treatment of cells results in inhibition of downstream signal transduction events and cell cycle regulatory pathways.  This leads to arrest at the G1-S (Gap 1/DNA synthesis)-phase transition of the cell division cycle, ultimately resulting in decreased cell proliferation.  In vivo, HKI-272 is active in HER-2- and EGFR-dependent tumor xenograft models when dosed orally on a once daily schedule.  On the basis of its favorable preclin. pharmacol. profile, HKI-272 has been selected as a candidate for addnl. development as an antitumor agent in breast and other HER-2-dependent cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo2xeLXlAwWvbVg90H21EOLACvtfcHk0ljU9yr7pWBZAg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXksVKns7o%253D&md5=60accd0599b7c489e4c3753d4f39cab4</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-03-2868&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-03-2868%26sid%3Dliteratum%253Aachs%26aulast%3DRabindran%26aufirst%3DS.%2BK.%26aulast%3DDiscafani%26aufirst%3DC.%2BM.%26aulast%3DRosfjord%26aufirst%3DE.%2BC.%26aulast%3DBaxter%26aufirst%3DM.%26aulast%3DFloyd%26aufirst%3DM.%2BB.%26aulast%3DGolas%26aufirst%3DJ.%26aulast%3DHallett%26aufirst%3DW.%2BA.%26aulast%3DJohnson%26aufirst%3DB.%2BD.%26aulast%3DNilakantan%26aufirst%3DR.%26aulast%3DOverbeek%26aufirst%3DE.%26aulast%3DReich%26aufirst%3DM.%2BF.%26aulast%3DShen%26aufirst%3DR.%26aulast%3DShi%26aufirst%3DX.%26aulast%3DTsou%26aufirst%3DH.-R.%26aulast%3DWang%26aufirst%3DY.-F.%26aulast%3DWissner%26aufirst%3DA.%26atitle%3DAntitumor%2520activity%2520of%2520HKI-272%252C%2520an%2520orally%2520active%252C%2520irreversible%2520inhibitor%2520of%2520the%2520HER-2%2520tyrosine%2520kinase%26jtitle%3DCancer%2520Res.%26date%3D2004%26volume%3D64%26spage%3D3958%26epage%3D3965%26doi%3D10.1158%2F0008-5472.can-03-2868" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tsou, H.-R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Overbeek-Klumpers, E. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hallett, W. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reich, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Floyd, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michalak, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nilakantan, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Discafani, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Golas, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rabindran, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.-F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Upeslacis, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wissner, A.</span></span> <span> </span><span class="NLM_article-title">Optimization of 6,7-disubstituted-4-(arylamino)quinoline-3-carbonitriles as orally active, irreversible inhibitors of human epidermal growth factor receptor-2 kinase activity</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>48</i></span>,  <span class="NLM_fpage">1107</span>– <span class="NLM_lpage">1131</span>, <span class="refDoi"> DOI: 10.1021/jm040159c</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm040159c" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00870&amp;key=1%3ACAS%3A528%3ADC%252BD2MXntFKjsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2005&pages=1107-1131&author=H.-R.+Tsouauthor=E.+G.+Overbeek-Klumpersauthor=W.+A.+Hallettauthor=M.+F.+Reichauthor=M.+B.+Floydauthor=B.+D.+Johnsonauthor=R.+S.+Michalakauthor=R.+Nilakantanauthor=C.+Discafaniauthor=J.+Golasauthor=S.+K.+Rabindranauthor=R.+Shenauthor=X.+Shiauthor=Y.-F.+Wangauthor=J.+Upeslacisauthor=A.+Wissner&title=Optimization+of+6%2C7-disubstituted-4-%28arylamino%29quinoline-3-carbonitriles+as+orally+active%2C+irreversible+inhibitors+of+human+epidermal+growth+factor+receptor-2+kinase+activity&doi=10.1021%2Fjm040159c"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Optimization of 6,7-Disubstituted-4-(arylamino)quinoline-3-carbonitriles as Orally Active, Irreversible Inhibitors of Human Epidermal Growth Factor Receptor-2 Kinase Activity</span></div><div class="casAuthors">Tsou, Hwei-Ru; Overbeek-Klumpers, Elsebe G.; Hallett, William A.; Reich, Marvin F.; Floyd, M. Brawner; Johnson, Bernard D.; Michalak, Ronald S.; Nilakantan, Ramaswamy; Discafani, Carolyn; Golas, Jonathan; Rabindran, Sridhar K.; Shen, Ru; Shi, Xiaoqing; Wang, Yu-Fen; Upeslacis, Janis; Wissner, Allan</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1107-1131</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A series of new 6,7-disubstituted-4-(arylamino)quinoline-3-carbonitriles, e.g. I (R1 = H, Cl; R2 = PhCH2O, 1-imidazolyl, 2-furylmethoxy, etc.; R3 = Cl, CN, PhCH2O; R4 = Me, Et; R5 = Me, R6 = Me, HOCH2CH2; R5R6N = azetidinyl, piperidinyl, thiomorpholinyl, etc.) that function as irreversible inhibitors of human epidermal growth factor receptor-2 (HER-2) and epidermal growth factor receptor (EGFR) kinases have been prepd.  These compds. demonstrated enhanced activities for inhibiting HER-2 kinase and the growth of HER-2 pos. cells compared to the EGFR kinase inhibitor I [R1 = H; R2 = F; R3 = Cl; R4 = Et; R5 = R6 = Me; (EKB-569)].  Three synthetic routes were used to prep. these compds.  They were prepd. mostly by acylation of 6-amino-4-(arylamino)quinoline-3-carbonitriles with unsatd. acid chlorides or by amination of 4-chloro-6-(crotonamido)quinoline-3-carbonitriles with monocyclic or bicyclic anilines.  The third route was developed to prep. a key intermediate, 6-acetamido-4-chloroquinoline-3-carbonitrile, that involved a safer cyclization step.  It was shown that attaching a large lipophilic group at the para position of the 4-(arylamino) ring results in improved potency for inhibiting HER-2 kinase.  The importance of a basic dialkylamino group at the end of the Michael acceptor for activity, due to intramol. catalysis of the Michael addn. has also been demonstrated.  This, along with improved water soly., resulted in compds. with enhanced biol. properties.  The mol. modeling results consistent with the proposed mechanism of inhibition are presented.  Binding studies of one compd., I [R1 = H; R2 = 2-pyridylmethoxy; R3 = Cl; R4 = Et; R5 = R6 = Me; (HKI-272)] (C-14 radiolabeled), showed that it binds irreversibly to HER-2 protein in BT474 cells.  Furthermore, it demonstrated excellent oral activity, esp. in HER-2 overexpressing xenografts.  Compd. HKI-272 was selected for further studies and is currently in phase I clin. trials for the treatment of cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqD0QL4-qQZCbVg90H21EOLACvtfcHk0lg1Bc930_e0mA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXntFKjsA%253D%253D&md5=55f77e41bf920f738ae2750e7ca0eb22</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1021%2Fjm040159c&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm040159c%26sid%3Dliteratum%253Aachs%26aulast%3DTsou%26aufirst%3DH.-R.%26aulast%3DOverbeek-Klumpers%26aufirst%3DE.%2BG.%26aulast%3DHallett%26aufirst%3DW.%2BA.%26aulast%3DReich%26aufirst%3DM.%2BF.%26aulast%3DFloyd%26aufirst%3DM.%2BB.%26aulast%3DJohnson%26aufirst%3DB.%2BD.%26aulast%3DMichalak%26aufirst%3DR.%2BS.%26aulast%3DNilakantan%26aufirst%3DR.%26aulast%3DDiscafani%26aufirst%3DC.%26aulast%3DGolas%26aufirst%3DJ.%26aulast%3DRabindran%26aufirst%3DS.%2BK.%26aulast%3DShen%26aufirst%3DR.%26aulast%3DShi%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DY.-F.%26aulast%3DUpeslacis%26aufirst%3DJ.%26aulast%3DWissner%26aufirst%3DA.%26atitle%3DOptimization%2520of%25206%252C7-disubstituted-4-%2528arylamino%2529quinoline-3-carbonitriles%2520as%2520orally%2520active%252C%2520irreversible%2520inhibitors%2520of%2520human%2520epidermal%2520growth%2520factor%2520receptor-2%2520kinase%2520activity%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2005%26volume%3D48%26spage%3D1107%26epage%3D1131%26doi%3D10.1021%2Fjm040159c" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wissner, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Overbeek, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reich, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Floyd, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mamuya, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosfjord, E. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Discafani, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rabindran, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gruber, B. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ye, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hallett, W. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nilakantan, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.-F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greenberger, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsou, H.-R.</span></span> <span> </span><span class="NLM_article-title">Synthesis and structure-activity relationships of 6,7-disubstituted 4-anilinoquinoline-3-carbonitriles. The design of an orally active, irreversible inhibitor of the tyrosine kinase activity of the epidermal growth factor receptor (EGFR) and the human epidermal growth factor receptor-2 (HER-2)</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>46</i></span>,  <span class="NLM_fpage">49</span>– <span class="NLM_lpage">63</span>, <span class="refDoi"> DOI: 10.1021/jm020241c</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm020241c" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00870&amp;key=1%3ACAS%3A528%3ADC%252BD38XptFOlsbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2003&pages=49-63&author=A.+Wissnerauthor=E.+Overbeekauthor=M.+F.+Reichauthor=M.+B.+Floydauthor=B.+D.+Johnsonauthor=N.+Mamuyaauthor=E.+C.+Rosfjordauthor=C.+Discafaniauthor=R.+Davisauthor=X.+Shiauthor=S.+K.+Rabindranauthor=B.+C.+Gruberauthor=F.+Yeauthor=W.+A.+Hallettauthor=R.+Nilakantanauthor=R.+Shenauthor=Y.-F.+Wangauthor=L.+M.+Greenbergerauthor=H.-R.+Tsou&title=Synthesis+and+structure-activity+relationships+of+6%2C7-disubstituted+4-anilinoquinoline-3-carbonitriles.+The+design+of+an+orally+active%2C+irreversible+inhibitor+of+the+tyrosine+kinase+activity+of+the+epidermal+growth+factor+receptor+%28EGFR%29+and+the+human+epidermal+growth+factor+receptor-2+%28HER-2%29&doi=10.1021%2Fjm020241c"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis and Structure-Activity Relationships of 6,7-Disubstituted 4-Anilinoquinoline-3-carbonitriles. The Design of an Orally Active, Irreversible Inhibitor of the Tyrosine Kinase Activity of the Epidermal Growth Factor Receptor (EGFR) and the Human Epidermal Growth Factor Receptor-2 (HER-2)</span></div><div class="casAuthors">Wissner, Allan; Overbeek, Elsebe; Reich, Marvin F.; Floyd, M. Brawner; Johnson, Bernard D.; Mamuya, Nellie; Rosfjord, Edward C.; Discafani, Carolyn; Davis, Rachel; Shi, Xiaoqing; Rabindran, Sridhar K.; Gruber, Brian C.; Ye, Fei; Hallett, William A.; Nilakantan, Ramaswamy; Shen, Ru; Wang, Yu-Fen; Greenberger, Lee M.; Tsou, Hwei-Ru</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">49-63</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A series of 6,7-disubstituted 4-anilinoquinoline-3-carbonitriles, e.g., I, that function as irreversible inhibitors of EGFR and HER-2 kinases have been prepd.  These inhibitors have, at the 6-position, butynamide, crotonamide, and methacrylamide Michael acceptors bearing water-solubilizing substituents.  These compds. were prepd. by acylation of 6-amino-4-(arylamino)quinoline-3-carbonitriles, e.g., II, with unsatd. acid chlorides or mixed anhydrides.  Competitive reactivity studies were performed, showing that attaching a dialkylamino group onto the end of the Michael acceptor results in compds. with greater reactivity due to intramol. catalysis of the Michael addn.  This, along with improved water-soly. results in compds. with enhanced biol. properties.  Mol. modeling results consistent with the proposed mechanism of inhibition are presented.  One compd., I (EKB-569), which shows excellent oral in vivo activity, was selected for further studies and is currently in phase I clin. trials for the treatment of cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq317fAvBYUW7Vg90H21EOLACvtfcHk0lg1Bc930_e0mA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XptFOlsbk%253D&md5=db70c2f78c62824810854295d06ef6a4</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1021%2Fjm020241c&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm020241c%26sid%3Dliteratum%253Aachs%26aulast%3DWissner%26aufirst%3DA.%26aulast%3DOverbeek%26aufirst%3DE.%26aulast%3DReich%26aufirst%3DM.%2BF.%26aulast%3DFloyd%26aufirst%3DM.%2BB.%26aulast%3DJohnson%26aufirst%3DB.%2BD.%26aulast%3DMamuya%26aufirst%3DN.%26aulast%3DRosfjord%26aufirst%3DE.%2BC.%26aulast%3DDiscafani%26aufirst%3DC.%26aulast%3DDavis%26aufirst%3DR.%26aulast%3DShi%26aufirst%3DX.%26aulast%3DRabindran%26aufirst%3DS.%2BK.%26aulast%3DGruber%26aufirst%3DB.%2BC.%26aulast%3DYe%26aufirst%3DF.%26aulast%3DHallett%26aufirst%3DW.%2BA.%26aulast%3DNilakantan%26aufirst%3DR.%26aulast%3DShen%26aufirst%3DR.%26aulast%3DWang%26aufirst%3DY.-F.%26aulast%3DGreenberger%26aufirst%3DL.%2BM.%26aulast%3DTsou%26aufirst%3DH.-R.%26atitle%3DSynthesis%2520and%2520structure-activity%2520relationships%2520of%25206%252C7-disubstituted%25204-anilinoquinoline-3-carbonitriles.%2520The%2520design%2520of%2520an%2520orally%2520active%252C%2520irreversible%2520inhibitor%2520of%2520the%2520tyrosine%2520kinase%2520activity%2520of%2520the%2520epidermal%2520growth%2520factor%2520receptor%2520%2528EGFR%2529%2520and%2520the%2520human%2520epidermal%2520growth%2520factor%2520receptor-2%2520%2528HER-2%2529%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2003%26volume%3D46%26spage%3D49%26epage%3D63%26doi%3D10.1021%2Fjm020241c" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xia, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mullin, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keith, B. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rusnak, D. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Owens, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alligood, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spector, N. L.</span></span> <span> </span><span class="NLM_article-title">Anti-tumor activity of GW572016: a dual tyrosine kinase inhibitor blocks EGF activation of EGFR/erbB2 and downstream Erk1/2 and AKT pathways</span>. <i>Oncogene</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">6255</span>– <span class="NLM_lpage">6263</span>, <span class="refDoi"> DOI: 10.1038/sj.onc.1205794</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00870&amp;key=10.1038%2Fsj.onc.1205794" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00870&amp;key=12214266" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00870&amp;key=1%3ACAS%3A528%3ADC%252BD38Xms1yrs7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2002&pages=6255-6263&author=W.+Xiaauthor=R.+J.+Mullinauthor=B.+R.+Keithauthor=L.-H.+Liuauthor=H.+Maauthor=D.+W.+Rusnakauthor=G.+Owensauthor=K.+J.+Alligoodauthor=N.+L.+Spector&title=Anti-tumor+activity+of+GW572016%3A+a+dual+tyrosine+kinase+inhibitor+blocks+EGF+activation+of+EGFR%2FerbB2+and+downstream+Erk1%2F2+and+AKT+pathways&doi=10.1038%2Fsj.onc.1205794"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Anti-tumor activity of GW572016: a dual tyrosine kinase inhibitor blocks EGF activation of EGFR/erbB2 and downstream Erk1/2 and AKT pathways</span></div><div class="casAuthors">Xia, Wenle; Mullin, Robert J.; Keith, Barry R.; Liu, Lei-Hua; Ma, Hong; Rusnak, David W.; Owens, Gary; Alligood, Krystal J.; Spector, Neil L.</div><div class="citationInfo"><span class="NLM_cas:title">Oncogene</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">41</span>),
    <span class="NLM_cas:pages">6255-6263</span>CODEN:
                <span class="NLM_cas:coden">ONCNES</span>;
        ISSN:<span class="NLM_cas:issn">0950-9232</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Dual EGFR/erbB2 inhibition is an attractive therapeutic strategy for epithelial tumors, as ligand-induced erbB2/EGFR heterodimerization triggers potent proliferative and survival signals.  Here we show that a small mol., GW572016, potently inhibits both EGFR and erbB2 tyrosine kinases leading to growth arrest and/or apoptosis in EGFR and erbB2-dependent tumor cell lines.  GW572016 markedly reduced tyrosine phosphorylation of EGFR and erbB2, and inhibited activation of Erk1/2 and AKT, downstream effectors of proliferation and cell survival, resp.  Complete inhibition of activated AKT in erbB2 overexpressing cells correlated with a 23-fold increase in apoptosis compared with vehicle controls.  EGF, often elevated in cancer patients, did not reverse the inhibitory effects of GW572016.  These observations were reproduced in vivo, where GW572016 treatment inhibited activation of EGFR, erbB2, Erk1/2 and AKT in human tumor xenografts.  Erk1/2 and AKT represent potential biomarkers to assess the clin. activity of GW572016.  Inhibition of activated AKT in EGFR or erbB2-dependent tumors by GW572016 may lead to tumor regressions when used as a monotherapy, or may enhance the anti-tumor activity of chemotherapeutics, since constitutive activation of AKT has been linked to chemo-resistance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqYPkZ4sa88irVg90H21EOLACvtfcHk0lgVEzb7f7AMxQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38Xms1yrs7w%253D&md5=38d6e25a7cfa1fdfe77fa2fea2c0aa0c</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1038%2Fsj.onc.1205794&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.onc.1205794%26sid%3Dliteratum%253Aachs%26aulast%3DXia%26aufirst%3DW.%26aulast%3DMullin%26aufirst%3DR.%2BJ.%26aulast%3DKeith%26aufirst%3DB.%2BR.%26aulast%3DLiu%26aufirst%3DL.-H.%26aulast%3DMa%26aufirst%3DH.%26aulast%3DRusnak%26aufirst%3DD.%2BW.%26aulast%3DOwens%26aufirst%3DG.%26aulast%3DAlligood%26aufirst%3DK.%2BJ.%26aulast%3DSpector%26aufirst%3DN.%2BL.%26atitle%3DAnti-tumor%2520activity%2520of%2520GW572016%253A%2520a%2520dual%2520tyrosine%2520kinase%2520inhibitor%2520blocks%2520EGF%2520activation%2520of%2520EGFR%252FerbB2%2520and%2520downstream%2520Erk1%252F2%2520and%2520AKT%2520pathways%26jtitle%3DOncogene%26date%3D2002%26volume%3D21%26spage%3D6255%26epage%3D6263%26doi%3D10.1038%2Fsj.onc.1205794" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Heymach, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Negrao, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robichaux, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carter, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Altan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gibbons, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fossella, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simon, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lam, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blumenschein, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsao, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kurie, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mott, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jenkins, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mack, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roeck, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Papadimitrakopoulou, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elamin, Y.</span></span> <span> </span><span class="NLM_article-title">OA02.06 A phase II trial of poziotinib in EGFR and HER2 exon 20 mutant non-small cell lung cancer (NSCLC)</span>. <i>J. Thorac. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">S323</span>– <span class="NLM_lpage">S324</span>, <span class="refDoi"> DOI: 10.1016/j.jtho.2018.08.243</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00870&amp;key=10.1016%2Fj.jtho.2018.08.243" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00870&amp;key=29396253" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2018&pages=S323-S324&author=J.+Heymachauthor=M.+Negraoauthor=J.+Robichauxauthor=B.+Carterauthor=A.+Patelauthor=M.+Altanauthor=D.+Gibbonsauthor=F.+Fossellaauthor=G.+Simonauthor=V.+Lamauthor=G.+Blumenscheinauthor=A.+Tsaoauthor=J.+Kurieauthor=F.+Mottauthor=D.+Jenkinsauthor=D.+Mackauthor=L.+Fengauthor=B.+Roeckauthor=Z.+Yangauthor=V.+Papadimitrakopoulouauthor=Y.+Elamin&title=OA02.06+A+phase+II+trial+of+poziotinib+in+EGFR+and+HER2+exon+20+mutant+non-small+cell+lung+cancer+%28NSCLC%29&doi=10.1016%2Fj.jtho.2018.08.243"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1016%2Fj.jtho.2018.08.243&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jtho.2018.08.243%26sid%3Dliteratum%253Aachs%26aulast%3DHeymach%26aufirst%3DJ.%26aulast%3DNegrao%26aufirst%3DM.%26aulast%3DRobichaux%26aufirst%3DJ.%26aulast%3DCarter%26aufirst%3DB.%26aulast%3DPatel%26aufirst%3DA.%26aulast%3DAltan%26aufirst%3DM.%26aulast%3DGibbons%26aufirst%3DD.%26aulast%3DFossella%26aufirst%3DF.%26aulast%3DSimon%26aufirst%3DG.%26aulast%3DLam%26aufirst%3DV.%26aulast%3DBlumenschein%26aufirst%3DG.%26aulast%3DTsao%26aufirst%3DA.%26aulast%3DKurie%26aufirst%3DJ.%26aulast%3DMott%26aufirst%3DF.%26aulast%3DJenkins%26aufirst%3DD.%26aulast%3DMack%26aufirst%3DD.%26aulast%3DFeng%26aufirst%3DL.%26aulast%3DRoeck%26aufirst%3DB.%26aulast%3DYang%26aufirst%3DZ.%26aulast%3DPapadimitrakopoulou%26aufirst%3DV.%26aulast%3DElamin%26aufirst%3DY.%26atitle%3DOA02.06%2520A%2520phase%2520II%2520trial%2520of%2520poziotinib%2520in%2520EGFR%2520and%2520HER2%2520exon%252020%2520mutant%2520non-small%2520cell%2520lung%2520cancer%2520%2528NSCLC%2529%26jtitle%3DJ.%2520Thorac.%2520Oncol.%26date%3D2018%26volume%3D13%26spage%3DS323%26epage%3DS324%26doi%3D10.1016%2Fj.jtho.2018.08.243" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lategahn, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keul, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klövekorn, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tumbrink, H. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Niggenaber, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hodson, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flasshoff, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hardick, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grabe, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Engel, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schultz-Fademrecht, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baumann, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ketzer, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mühlenberg, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hiller, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Günther, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Unger, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heimsoeth, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Golz, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blank-Landeshammer, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kollipara, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zahedi, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Strohmann, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hengstler, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Otterlo, W. A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bauer, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rauh, D.</span></span> <span> </span><span class="NLM_article-title">Inhibition of osimertinib-resistant epidermal growth factor receptor EGFR-T790M/C797S</span>. <i>Chem. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">10789</span>– <span class="NLM_lpage">10801</span>, <span class="refDoi"> DOI: 10.1039/c9sc03445e</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00870&amp;key=10.1039%2FC9SC03445E" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00870&amp;key=31857889" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00870&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhvFSntr7N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2019&pages=10789-10801&author=J.+Lategahnauthor=M.+Keulauthor=P.+Kl%C3%B6vekornauthor=H.+L.+Tumbrinkauthor=J.+Niggenaberauthor=M.+P.+M%C3%BCllerauthor=L.+Hodsonauthor=M.+Flasshoffauthor=J.+Hardickauthor=T.+Grabeauthor=J.+Engelauthor=C.+Schultz-Fademrechtauthor=M.+Baumannauthor=J.+Ketzerauthor=T.+M%C3%BChlenbergauthor=W.+Hillerauthor=G.+G%C3%BCntherauthor=A.+Ungerauthor=H.+M%C3%BCllerauthor=A.+Heimsoethauthor=C.+Golzauthor=B.+Blank-Landeshammerauthor=L.+Kolliparaauthor=R.+P.+Zahediauthor=C.+Strohmannauthor=J.+G.+Hengstlerauthor=W.+A.+L.+van+Otterloauthor=S.+Bauerauthor=D.+Rauh&title=Inhibition+of+osimertinib-resistant+epidermal+growth+factor+receptor+EGFR-T790M%2FC797S&doi=10.1039%2Fc9sc03445e"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of osimertinib-resistant epidermal growth factor receptor EGFR-T790M/C797S</span></div><div class="casAuthors">Lategahn, Jonas; Keul, Marina; Kloevekorn, Philip; Tumbrink, Hannah L.; Niggenaber, Janina; Mueller, Matthias P.; Hodson, Luke; Flasshoff, Maren; Hardick, Julia; Grabe, Tobias; Engel, Julian; Schultz-Fademrecht, Carsten; Baumann, Matthias; Ketzer, Julia; Muehlenberg, Thomas; Hiller, Wolf; Guenther, Georgia; Unger, Anke; Mueller, Heiko; Heimsoeth, Alena; Golz, Christopher; Blank-Landeshammer, Bernhard; Kollipara, Laxmikanth; Zahedi, Rene P.; Strohmann, Carsten; Hengstler, Jan G.; van Otterlo, Willem A. L.; Bauer, Sebastian; Rauh, Daniel</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Science</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">46</span>),
    <span class="NLM_cas:pages">10789-10801</span>CODEN:
                <span class="NLM_cas:coden">CSHCCN</span>;
        ISSN:<span class="NLM_cas:issn">2041-6520</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Precision medicine has revolutionized the treatment of patients in EGFR driven non-small cell lung cancer (NSCLC).  Targeted drugs show high response rates in genetically defined subsets of cancer patients and markedly increase their progression-free survival as compared to conventional chemotherapy.  However, recurrent acquired drug resistance limits the success of targeted drugs in long-term treatment and requires the const. development of novel efficient inhibitors of drug resistant cancer subtypes.  Herein, we present covalent inhibitors of the drug resistant gatekeeper mutant EGFR-L858R/T790M based on the pyrrolopyrimidine scaffold.  Biochem. and cellular characterization, as well as kinase selectivity profiling and western blot anal., substantiate our approach.  Moreover, the developed compds. possess high activity against multi drug resistant EGFR-L858R/T790M/C797S in biochem. assays due to their highly reversible binding character, that was revealed by characterization of the binding kinetics.  In addn., we present the first X-ray crystal structures of covalent inhibitors in complex with C797S-mutated EGFR which provide detailed insight into their binding mode.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqpOILrK_TpebVg90H21EOLACvtfcHk0lgVEzb7f7AMxQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhvFSntr7N&md5=0dd60e9ade21e33ae5656d50175e39ba</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1039%2FC9SC03445E&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC9SC03445E%26sid%3Dliteratum%253Aachs%26aulast%3DLategahn%26aufirst%3DJ.%26aulast%3DKeul%26aufirst%3DM.%26aulast%3DKl%25C3%25B6vekorn%26aufirst%3DP.%26aulast%3DTumbrink%26aufirst%3DH.%2BL.%26aulast%3DNiggenaber%26aufirst%3DJ.%26aulast%3DM%25C3%25BCller%26aufirst%3DM.%2BP.%26aulast%3DHodson%26aufirst%3DL.%26aulast%3DFlasshoff%26aufirst%3DM.%26aulast%3DHardick%26aufirst%3DJ.%26aulast%3DGrabe%26aufirst%3DT.%26aulast%3DEngel%26aufirst%3DJ.%26aulast%3DSchultz-Fademrecht%26aufirst%3DC.%26aulast%3DBaumann%26aufirst%3DM.%26aulast%3DKetzer%26aufirst%3DJ.%26aulast%3DM%25C3%25BChlenberg%26aufirst%3DT.%26aulast%3DHiller%26aufirst%3DW.%26aulast%3DG%25C3%25BCnther%26aufirst%3DG.%26aulast%3DUnger%26aufirst%3DA.%26aulast%3DM%25C3%25BCller%26aufirst%3DH.%26aulast%3DHeimsoeth%26aufirst%3DA.%26aulast%3DGolz%26aufirst%3DC.%26aulast%3DBlank-Landeshammer%26aufirst%3DB.%26aulast%3DKollipara%26aufirst%3DL.%26aulast%3DZahedi%26aufirst%3DR.%2BP.%26aulast%3DStrohmann%26aufirst%3DC.%26aulast%3DHengstler%26aufirst%3DJ.%2BG.%26aulast%3Dvan%2BOtterlo%26aufirst%3DW.%2BA.%2BL.%26aulast%3DBauer%26aufirst%3DS.%26aulast%3DRauh%26aufirst%3DD.%26atitle%3DInhibition%2520of%2520osimertinib-resistant%2520epidermal%2520growth%2520factor%2520receptor%2520EGFR-T790M%252FC797S%26jtitle%3DChem.%2520Sci.%26date%3D2019%26volume%3D10%26spage%3D10789%26epage%3D10801%26doi%3D10.1039%2Fc9sc03445e" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wolle, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weisner, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keul, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Landel, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lategahn, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rauh, D.</span></span> <span> </span><span class="NLM_article-title">RASPELD to perform high-end screening in an academic environment toward the development of cancer therapeutics</span>. <i>ChemMedChem</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">2065</span>– <span class="NLM_lpage">2072</span>, <span class="refDoi"> DOI: 10.1002/cmdc.201800477</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00870&amp;key=10.1002%2Fcmdc.201800477" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00870&amp;key=30079978" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00870&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhsFKqu7%252FL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2018&pages=2065-2072&author=P.+Wolleauthor=J.+Weisnerauthor=M.+Keulauthor=I.+Landelauthor=J.+Lategahnauthor=D.+Rauh&title=RASPELD+to+perform+high-end+screening+in+an+academic+environment+toward+the+development+of+cancer+therapeutics&doi=10.1002%2Fcmdc.201800477"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">RASPELD to Perform High-End Screening in an Academic Environment toward the Development of Cancer Therapeutics</span></div><div class="casAuthors">Wolle, Patrik; Weisner, Joern; Keul, Marina; Landel, Ina; Lategahn, Jonas; Rauh, Daniel</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">2065-2072</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">The identification of compds. for dissecting biol. functions and the development of novel drug mols. are central tasks that often require screening campaigns.  However, the required architecture is cost- and time-intensive.  Herein the authors describe the devices and technologies that comprise a Robotics-Assisted Screening Platform for Efficient Ligand Discovery (RASPELD), which the authors set up in an academic lab.  RASPELD provides semi-automated high-end screening, and it can be maintained by graduate students.  The authors demonstrate its successful application in biochem. and cellular screens for the identification and validation of bioactive chem. entities as candidate cancer-relevant inhibitors.  Specifically, the authors examd. the interaction between a transcription factor, Nrf2, and its key regulator, Keap1.  The authors also examd. drug-resistant mutants of the epidermal growth factor receptor (EGFR).  Screening campaigns with >30,000 compds. were performed in a reasonable period of time.  The authors identified the mol. RSL6586 as a starting point for hit optimization, which is currently ongoing.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGphaUbPTnBjCbVg90H21EOLACvtfcHk0ljIU_1-yamQAA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhsFKqu7%252FL&md5=947a778e507c4d6c34905bcd40fd20e5</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.201800477&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.201800477%26sid%3Dliteratum%253Aachs%26aulast%3DWolle%26aufirst%3DP.%26aulast%3DWeisner%26aufirst%3DJ.%26aulast%3DKeul%26aufirst%3DM.%26aulast%3DLandel%26aufirst%3DI.%26aulast%3DLategahn%26aufirst%3DJ.%26aulast%3DRauh%26aufirst%3DD.%26atitle%3DRASPELD%2520to%2520perform%2520high-end%2520screening%2520in%2520an%2520academic%2520environment%2520toward%2520the%2520development%2520of%2520cancer%2520therapeutics%26jtitle%3DChemMedChem%26date%3D2018%26volume%3D13%26spage%3D2065%26epage%3D2072%26doi%3D10.1002%2Fcmdc.201800477" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lounkine, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wawer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wassermann, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bajorath, J.</span></span> <span> </span><span class="NLM_article-title">SARANEA: a freely available program to mine structure-activity and structure-selectivity relationship information in compound data sets</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>50</i></span>,  <span class="NLM_fpage">68</span>– <span class="NLM_lpage">78</span>, <span class="refDoi"> DOI: 10.1021/ci900416a</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci900416a" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00870&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs12jtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2010&pages=68-78&author=E.+Lounkineauthor=M.+Wawerauthor=A.+M.+Wassermannauthor=J.+Bajorath&title=SARANEA%3A+a+freely+available+program+to+mine+structure-activity+and+structure-selectivity+relationship+information+in+compound+data+sets&doi=10.1021%2Fci900416a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">SARANEA: A Freely Available Program To Mine Structure-Activity and Structure-Selectivity Relationship Information in Compound Data Sets</span></div><div class="casAuthors">Lounkine, Eugen; Wawer, Mathias; Wassermann, Anne Mai; Bajorath, Jurgen</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">68-78</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We introduce SARANEA, an open-source Java application for interactive exploration of structure-activity relationship (SAR) and structure-selectivity relationship (SSR) information in compd. sets of any source.  SARANEA integrates various SAR and SSR anal. functions and utilizes a network-like similarity graph data structure for visualization.  The program enables the systematic detection of activity and selectivity cliffs and corresponding key compds. across multiple targets.  Advanced SAR anal. functions implemented in SARANEA include, among others, layered chem. neighborhood graphs, cliff indexes, selectivity trees, editing functions for mol. networks and pathways, bioactivity summaries of key compds., and markers for bioactive compds. having potential side effects.  We report the application of SARANEA to identify SAR and SSR determinants in different sets of serine protease inhibitors.  It is found that key compds. can influence SARs and SSRs in rather different ways.  Such compds. and their SAR/SSR characteristics can be systematically identified and explored using SARANEA.  The program and source code are made freely available under the GNU General Public License.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrWTKafem6ttbVg90H21EOLACvtfcHk0ljgOKPApfUKqg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs12jtQ%253D%253D&md5=936edf696d3ae7bb8196c81158528369</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1021%2Fci900416a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci900416a%26sid%3Dliteratum%253Aachs%26aulast%3DLounkine%26aufirst%3DE.%26aulast%3DWawer%26aufirst%3DM.%26aulast%3DWassermann%26aufirst%3DA.%2BM.%26aulast%3DBajorath%26aufirst%3DJ.%26atitle%3DSARANEA%253A%2520a%2520freely%2520available%2520program%2520to%2520mine%2520structure-activity%2520and%2520structure-selectivity%2520relationship%2520information%2520in%2520compound%2520data%2520sets%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2010%26volume%3D50%26spage%3D68%26epage%3D78%26doi%3D10.1021%2Fci900416a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Getlik, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grütter, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simard, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen, H. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robubi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aust, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Otterlo, W. A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rauh, D.</span></span> <span> </span><span class="NLM_article-title">Structure-based design, synthesis and biological evaluation of N-pyrazole, N’-thiazole urea inhibitors of MAP kinase p38alpha</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>48</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">15</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2011.11.019</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00870&amp;key=10.1016%2Fj.ejmech.2011.11.019" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00870&amp;key=22154891" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00870&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtFKmsbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2012&pages=1-15&author=M.+Getlikauthor=C.+Gr%C3%BCtterauthor=J.+R.+Simardauthor=H.+D.+Nguyenauthor=A.+Robubiauthor=B.+Austauthor=W.+A.+L.+van+Otterloauthor=D.+Rauh&title=Structure-based+design%2C+synthesis+and+biological+evaluation+of+N-pyrazole%2C+N%E2%80%99-thiazole+urea+inhibitors+of+MAP+kinase+p38alpha&doi=10.1016%2Fj.ejmech.2011.11.019"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-based design, synthesis and biological evaluation of N-pyrazole, N'-thiazole urea inhibitors of MAP kinase p38α</span></div><div class="casAuthors">Getlik, Matthaeus; Gruetter, Christian; Simard, Jeffrey R.; Nguyen, Hoang D.; Robubi, Armin; Aust, Beate; van Otterlo, Willem A. L.; Rauh, Daniel</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">1-15</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">The structure-based design, synthesis and biol. activity of N-(1-aryl-1H-pyrazol-5-yl)-N'-(2 -thiazolyl)ureas as potent inhibitors of p38α mitogen-activated protein kinase (p38α MAPK).  Guided by complex crystal structures, key structural elements were introduced into the initially identified N-aryl-N'-thiazolylurea scaffold that allowed the formation of novel hydrogen bonding interactions within the allosteric site of p38α, resulting in potent type III inhibitors.  [4-(3-Tert-Butyl-5-{[(1,3-thiazol-2-ylamino)carbonyl]amino}-1H-pyrazol-1-yl)-phenyl]acetic acid was found to be the most potent compd. within this series and inhibited p38α activity with an IC 50 of 135 ± 21 nM.  Its closest analog, the Et ester, effectively inhibited p38α mediated phosphorylation of the mitogen activated protein kinase activated protein kinase 2 (MK2) in HeLa cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpxt0EyIJuA17Vg90H21EOLACvtfcHk0ljgOKPApfUKqg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtFKmsbc%253D&md5=3b6adebcdb5fbede2909c596e1d7901a</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2011.11.019&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2011.11.019%26sid%3Dliteratum%253Aachs%26aulast%3DGetlik%26aufirst%3DM.%26aulast%3DGr%25C3%25BCtter%26aufirst%3DC.%26aulast%3DSimard%26aufirst%3DJ.%2BR.%26aulast%3DNguyen%26aufirst%3DH.%2BD.%26aulast%3DRobubi%26aufirst%3DA.%26aulast%3DAust%26aufirst%3DB.%26aulast%3Dvan%2BOtterlo%26aufirst%3DW.%2BA.%2BL.%26aulast%3DRauh%26aufirst%3DD.%26atitle%3DStructure-based%2520design%252C%2520synthesis%2520and%2520biological%2520evaluation%2520of%2520N-pyrazole%252C%2520N%25E2%2580%2599-thiazole%2520urea%2520inhibitors%2520of%2520MAP%2520kinase%2520p38alpha%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2012%26volume%3D48%26spage%3D1%26epage%3D15%26doi%3D10.1016%2Fj.ejmech.2011.11.019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Spellmon, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Z.</span></span> <span> </span><span class="NLM_article-title">Allosterically targeting EGFR drug-resistance gatekeeper mutations</span>. <i>J. Thorac. Dis.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">1756</span>– <span class="NLM_lpage">1758</span>, <span class="refDoi"> DOI: 10.21037/jtd.2017.06.43</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00870&amp;key=10.21037%2Fjtd.2017.06.43" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00870&amp;key=28839955" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00870&amp;key=1%3ACAS%3A280%3ADC%252BC1cbhsFCmsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2017&pages=1756-1758&author=N.+Spellmonauthor=C.+Liauthor=Z.+Yang&title=Allosterically+targeting+EGFR+drug-resistance+gatekeeper+mutations&doi=10.21037%2Fjtd.2017.06.43"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Allosterically targeting EGFR drug-resistance gatekeeper mutations</span></div><div class="casAuthors">Spellmon Nicholas; Yang Zhe; Li Chunying</div><div class="citationInfo"><span class="NLM_cas:title">Journal of thoracic disease</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1756-1758</span>
        ISSN:<span class="NLM_cas:issn">2072-1439</span>.
    </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQMer61q-2iHLZpSb_XWfMrfW6udTcc2eZRpuKdMq0-1bntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1cbhsFCmsg%253D%253D&md5=32e7407f858050a7ddbb026c5477d183</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.21037%2Fjtd.2017.06.43&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.21037%252Fjtd.2017.06.43%26sid%3Dliteratum%253Aachs%26aulast%3DSpellmon%26aufirst%3DN.%26aulast%3DLi%26aufirst%3DC.%26aulast%3DYang%26aufirst%3DZ.%26atitle%3DAllosterically%2520targeting%2520EGFR%2520drug-resistance%2520gatekeeper%2520mutations%26jtitle%3DJ.%2520Thorac.%2520Dis.%26date%3D2017%26volume%3D9%26spage%3D1756%26epage%3D1758%26doi%3D10.21037%2Fjtd.2017.06.43" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jia, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yun, C.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ercan, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manuia, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Juarez, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rhee, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Palakurthi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lelais, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DiDonato, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bursulaya, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michellys, P.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Epple, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marsilje, T. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McNeill, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harris, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bender, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, K.-K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jänne, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eck, M. J.</span></span> <span> </span><span class="NLM_article-title">Overcoming EGFR(T790M) and EGFR(C797S) resistance with mutant-selective allosteric inhibitors</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>534</i></span>,  <span class="NLM_fpage">129</span>– <span class="NLM_lpage">132</span>, <span class="refDoi"> DOI: 10.1038/nature17960</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c00870&amp;key=10.1038%2Fnature17960" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c00870&amp;key=27251290" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00870&amp;key=1%3ACAS%3A528%3ADC%252BC28XosFWht7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=534&publication_year=2016&pages=129-132&author=Y.+Jiaauthor=C.-H.+Yunauthor=E.+Parkauthor=D.+Ercanauthor=M.+Manuiaauthor=J.+Juarezauthor=C.+Xuauthor=K.+Rheeauthor=T.+Chenauthor=H.+Zhangauthor=S.+Palakurthiauthor=J.+Jangauthor=G.+Lelaisauthor=M.+DiDonatoauthor=B.+Bursulayaauthor=P.-Y.+Michellysauthor=R.+Eppleauthor=T.+H.+Marsiljeauthor=M.+McNeillauthor=W.+Luauthor=J.+Harrisauthor=S.+Benderauthor=K.-K.+Wongauthor=P.+A.+J%C3%A4nneauthor=M.+J.+Eck&title=Overcoming+EGFR%28T790M%29+and+EGFR%28C797S%29+resistance+with+mutant-selective+allosteric+inhibitors&doi=10.1038%2Fnature17960"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Overcoming EGFR(T790M) and EGFR(C797S) resistance with mutant-selective allosteric inhibitors</span></div><div class="casAuthors">Jia, Yong; Yun, Cai-Hong; Park, Eunyoung; Ercan, Dalia; Manuia, Mari; Juarez, Jose; Xu, Chunxiao; Rhee, Kevin; Chen, Ting; Zhang, Haikuo; Palakurthi, Sangeetha; Jang, Jaebong; Lelais, Gerald; DiDonato, Michael; Bursulaya, Badry; Michellys, Pierre-Yves; Epple, Robert; Marsilje, Thomas H.; McNeill, Matthew; Lu, Wenshuo; Harris, Jennifer; Bender, Steven; Wong, Kwok-Kin; Janne, Pasi A.; Eck, Michael J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">534</span>
        (<span class="NLM_cas:issue">7605</span>),
    <span class="NLM_cas:pages">129-132</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The epidermal growth factor receptor (EGFR)-directed tyrosine kinase inhibitors (TKIs) gefitinib, erlotinib and afatinib are approved treatments for non-small cell lung cancers harbouring activating mutations in the EGFR kinase, but resistance arises rapidly, most frequently owing to the secondary T790M mutation within the ATP site of the receptor.  Recently developed mutant-selective irreversible inhibitors are highly active against the T790M mutant, but their efficacy can be compromised by acquired mutation of C797, the cysteine residue with which they form a key covalent bond.  All current EGFR TKIs target the ATP-site of the kinase, highlighting the need for therapeutic agents with alternative mechanisms of action.  Here we describe the rational discovery of EAI045, an allosteric inhibitor that targets selected drug-resistant EGFR mutants but spares the wild-type receptor.  The crystal structure shows that the compd. binds an allosteric site created by the displacement of the regulatory C-helix in an inactive conformation of the kinase.  The compd. inhibits L858R/T790M-mutant EGFR with low-nanomolar potency in biochem. assays.  However, as a single agent it is not effective in blocking EGFR-driven proliferation in cells owing to differential potency on the two subunits of the dimeric receptor, which interact in an asym. manner in the active state.  We observe marked synergy of EAI045 with cetuximab, an antibody therapeutic that blocks EGFR dimerization, rendering the kinase uniformly susceptible to the allosteric agent.  EAI045 in combination with cetuximab is effective in mouse models of lung cancer driven by EGFR(L858R/T790M) and by EGFR(L858R/T790M/C797S), a mutant that is resistant to all currently available EGFR TKIs.  More generally, our findings illustrate the utility of purposefully targeting allosteric sites to obtain mutant-selective inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqQkr363w1dMLVg90H21EOLACvtfcHk0lib3e98X2r2Dw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XosFWht7k%253D&md5=f4c2d2362eb900c64bad8550b9da0872</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1038%2Fnature17960&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature17960%26sid%3Dliteratum%253Aachs%26aulast%3DJia%26aufirst%3DY.%26aulast%3DYun%26aufirst%3DC.-H.%26aulast%3DPark%26aufirst%3DE.%26aulast%3DErcan%26aufirst%3DD.%26aulast%3DManuia%26aufirst%3DM.%26aulast%3DJuarez%26aufirst%3DJ.%26aulast%3DXu%26aufirst%3DC.%26aulast%3DRhee%26aufirst%3DK.%26aulast%3DChen%26aufirst%3DT.%26aulast%3DZhang%26aufirst%3DH.%26aulast%3DPalakurthi%26aufirst%3DS.%26aulast%3DJang%26aufirst%3DJ.%26aulast%3DLelais%26aufirst%3DG.%26aulast%3DDiDonato%26aufirst%3DM.%26aulast%3DBursulaya%26aufirst%3DB.%26aulast%3DMichellys%26aufirst%3DP.-Y.%26aulast%3DEpple%26aufirst%3DR.%26aulast%3DMarsilje%26aufirst%3DT.%2BH.%26aulast%3DMcNeill%26aufirst%3DM.%26aulast%3DLu%26aufirst%3DW.%26aulast%3DHarris%26aufirst%3DJ.%26aulast%3DBender%26aufirst%3DS.%26aulast%3DWong%26aufirst%3DK.-K.%26aulast%3DJ%25C3%25A4nne%26aufirst%3DP.%2BA.%26aulast%3DEck%26aufirst%3DM.%2BJ.%26atitle%3DOvercoming%2520EGFR%2528T790M%2529%2520and%2520EGFR%2528C797S%2529%2520resistance%2520with%2520mutant-selective%2520allosteric%2520inhibitors%26jtitle%3DNature%26date%3D2016%26volume%3D534%26spage%3D129%26epage%3D132%26doi%3D10.1038%2Fnature17960" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Baell, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holloway, G. A.</span></span> <span> </span><span class="NLM_article-title">New substructure filters for removal of pan assay interference compounds (PAINS) from screening libraries and for their exclusion in bioassays</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">2719</span>– <span class="NLM_lpage">2740</span>, <span class="refDoi"> DOI: 10.1021/jm901137j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm901137j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00870&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsF2qsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=2719-2740&author=J.+B.+Baellauthor=G.+A.+Holloway&title=New+substructure+filters+for+removal+of+pan+assay+interference+compounds+%28PAINS%29+from+screening+libraries+and+for+their+exclusion+in+bioassays&doi=10.1021%2Fjm901137j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">New Substructure Filters for Removal of Pan Assay Interference Compounds (PAINS) from Screening Libraries and for Their Exclusion in Bioassays</span></div><div class="casAuthors">Baell, Jonathan B.; Holloway, Georgina A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2719-2740</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">This report describes a no. of substructural features which can help to identify compds. that appear as frequent hitters (promiscuous compds.) in many biochem. high throughput screens.  The compds. identified by such substructural features are not recognized by filters commonly used to identify reactive compds.  Even though these substructural features were identified using only one assay detection technol., such compds. have been reported to be active from many different assays.  In fact, these compds. are increasingly prevalent in the literature as potential starting points for further exploration, whereas they may not be.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGro_JARcceWjLVg90H21EOLACvtfcHk0lgq7HsrJq66mA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsF2qsLw%253D&md5=fbf397aa4910753c550425708c866fd2</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1021%2Fjm901137j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm901137j%26sid%3Dliteratum%253Aachs%26aulast%3DBaell%26aufirst%3DJ.%2BB.%26aulast%3DHolloway%26aufirst%3DG.%2BA.%26atitle%3DNew%2520substructure%2520filters%2520for%2520removal%2520of%2520pan%2520assay%2520interference%2520compounds%2520%2528PAINS%2529%2520from%2520screening%2520libraries%2520and%2520for%2520their%2520exclusion%2520in%2520bioassays%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D2719%26epage%3D2740%26doi%3D10.1021%2Fjm901137j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Engel, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lategahn, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rauh, D.</span></span> <span> </span><span class="NLM_article-title">Hope and disappointment: covalent inhibitors to overcome drug resistance in non-small cell lung cancer</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">2</span>– <span class="NLM_lpage">5</span>, <span class="refDoi"> DOI: 10.1021/acsmedchemlett.5b00475</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.5b00475" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c00870&amp;key=1%3ACAS%3A528%3ADC%252BC28XktVOluw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=2-5&author=J.+Engelauthor=J.+Lategahnauthor=D.+Rauh&title=Hope+and+disappointment%3A+covalent+inhibitors+to+overcome+drug+resistance+in+non-small+cell+lung+cancer&doi=10.1021%2Facsmedchemlett.5b00475"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Hope and Disappointment: Covalent Inhibitors to Overcome Drug Resistance in Non-Small Cell Lung Cancer</span></div><div class="casAuthors">Engel, Julian; Lategahn, Jonas; Rauh, Daniel</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">2-5</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">In the last five years, the detailed understanding of how to overcome T790M drug resistance in non-small cell lung cancer (NSCLC) has culminated in the development of a third-generation of covalent EGFR inhibitors with excellent clin. outcomes.  However, the emergence of a newly discovered acquired drug resistance challenges the concept of small mol. targeted cancer therapy in NSCLC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqcxNkfyBm4vrVg90H21EOLACvtfcHk0lgq7HsrJq66mA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XktVOluw%253D%253D&md5=738e3286dd1b1d4110249e33cceec9ac</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.5b00475&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.5b00475%26sid%3Dliteratum%253Aachs%26aulast%3DEngel%26aufirst%3DJ.%26aulast%3DLategahn%26aufirst%3DJ.%26aulast%3DRauh%26aufirst%3DD.%26atitle%3DHope%2520and%2520disappointment%253A%2520covalent%2520inhibitors%2520to%2520overcome%2520drug%2520resistance%2520in%2520non-small%2520cell%2520lung%2520cancer%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2016%26volume%3D7%26spage%3D2%26epage%3D5%26doi%3D10.1021%2Facsmedchemlett.5b00475" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3PP0" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3PP0','PDB','3PP0'); return false;">PDB: 3PP0</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5D41" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5D41','PDB','5D41'); return false;">PDB: 5D41</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3LFB" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3LFB','PDB','3LFB'); return false;">PDB: 3LFB</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1XKK" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1XKK','PDB','1XKK'); return false;">PDB: 1XKK</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1PP0" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1PP0','PDB','1PP0'); return false;">PDB: 1PP0</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3RCD" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3RCD','PDB','3RCD'); return false;">PDB: 3RCD</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TFU" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TFU','PDB','6TFU'); return false;">PDB: 6TFU</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TFV" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TFV','PDB','6TFV'); return false;">PDB: 6TFV</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TFY" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TFY','PDB','6TFY'); return false;">PDB: 6TFY</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TFW" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TFW','PDB','6TFW'); return false;">PDB: 6TFW</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TFZ" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TFZ','PDB','6TFZ'); return false;">PDB: 6TFZ</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TG0" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TG0','PDB','6TG0'); return false;">PDB: 6TG0</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TG1" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TG1','PDB','6TG1'); return false;">PDB: 6TG1</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5J9Z" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5J9Z','PDB','5J9Z'); return false;">PDB: 5J9Z</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i157"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c00870">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_22085"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a class="ext-link" href="/doi/10.1021/acs.jmedchem.0c00870?goto=supporting-info">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c00870</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Sequence alignment of EGFR and Her2 kinases; <i>Z</i>′ values obtained within a focused screen; analysis of activity data obtained within a focused screen; Western blot analysis of reference compounds in H1781 cells; cocrystal structures of covalent pyrrolopyrimidines in complex with mutant EGFR-T790M/V948R; mass spectrometry-based analysis of covalent bond formation; hit validation for representative inhibitors; IC50 and EC50 evaluation of pyrrolopyrimidine-based inhibitors; data collection and refinement statistics of complex crystal structures; and LC/MS spectra of key compounds <b>14b</b>, <b>14h</b>, <b>18d</b>, and <b>19</b> (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c00870/suppl_file/jm0c00870_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Molecular formula strings of <b>1</b>–<b>5, poziotinib</b>, <b>TAK-285</b>, <b>gefitinib</b>, <b>osimertinib</b>, <b>7, 14, 18–21, 25</b>, <b>i–viii</b>, and so forth (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c00870/suppl_file/jm0c00870_si_002.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00870/suppl_file/jm0c00870_si_001.pdf">jm0c00870_si_001.pdf (0.99 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c00870/suppl_file/jm0c00870_si_002.csv">jm0c00870_si_002.csv (16.09 kb)</a></li></ul></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">PDB IDs are <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TFU">6TFU</a> (<b>14d</b>/EGFR-T790M/V948R), <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TFV">6TFV</a> (<b>18b</b>/EGFR-T790M/V948R), <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TFY">6TFY</a> (<b>18c</b>/EGFR-T790M/V948R), <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TFW">6TFW</a> (<b>18d</b>/EGFR-T790M/V948R), <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TFZ">6TFZ</a> (<b>19</b>/EGFR-T790M/V948R), and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TG0">6TG0</a> (<b>21a</b>/EGFR-T790M/V948R), <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6TG1">6TG1</a> (<b>21b</b>/EGFR-T790M/V948R).</p><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.0c00870&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.0c00870%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2020.63.issue-20%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.0c00870" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                6MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"67996079dd6a23f8","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
