
<html lang="en"     class="pb-page"  data-request-id="6002da06-daba-4815-9da7-c267362d58a9"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";issue:issue:10.1021/jmcmar.2020.63.issue-23;article:article:10.1021/acs.jmedchem.0c01342;requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Discovery of First-In-Class Potent and Selective Tropomyosin Receptor Kinase Degraders" /></meta><meta name="dc.Creator" content="Liqun  Chen" /></meta><meta name="dc.Creator" content="Yanke  Chen" /></meta><meta name="dc.Creator" content="Chunyan  Zhang" /></meta><meta name="dc.Creator" content="Bingyang  Jiao" /></meta><meta name="dc.Creator" content="Sheng  Liang" /></meta><meta name="dc.Creator" content="Qiong  Tan" /></meta><meta name="dc.Creator" content="Hongyu  Chai" /></meta><meta name="dc.Creator" content="Weihua  Yu" /></meta><meta name="dc.Creator" content="Yongzheng  Qian" /></meta><meta name="dc.Creator" content="Hui  Yang" /></meta><meta name="dc.Creator" content="Wuyi  Yao" /></meta><meta name="dc.Creator" content="Jianguo  Yu" /></meta><meta name="dc.Creator" content="Ying  Luo" /></meta><meta name="dc.Creator" content="Michael  Plewe" /></meta><meta name="dc.Creator" content="Jialiang  Wang" /></meta><meta name="dc.Creator" content="Xiao-Ran  Han" /></meta><meta name="dc.Creator" content="Jing  Liu" /></meta><meta name="dc.Description" content="We report compounds 5 (CG416) and 6 (CG428) as two first-in-class tropomyosin receptor kinase (TRK) degraders that target the intracellular kinase domain of TRK. Degraders 5 and 6 reduced levels of..." /></meta><meta name="Description" content="We report compounds 5 (CG416) and 6 (CG428) as two first-in-class tropomyosin receptor kinase (TRK) degraders that target the intracellular kinase domain of TRK. Degraders 5 and 6 reduced levels of..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="October 15, 2020" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.0c01342" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2020 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c01342" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.0c01342" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c01342" /></link>
        
    
    

<title>Discovery of First-In-Class Potent and Selective Tropomyosin Receptor Kinase Degraders | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.0c01342" /></meta><meta property="og:title" content="Discovery of First-In-Class Potent and Selective Tropomyosin Receptor Kinase Degraders" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01342/20201203/images/large/jm0c01342_0013.jpeg" /></meta><meta property="og:description" content="We report compounds 5 (CG416) and 6 (CG428) as two first-in-class tropomyosin receptor kinase (TRK) degraders that target the intracellular kinase domain of TRK. Degraders 5 and 6 reduced levels of the tropomyosin 3 (TPM3)-TRKA fusion protein in KM12 colorectal carcinoma cells and inhibited downstream PLCγ1 signaling at sub-nanomolar concentrations. Both degraders also degraded human wild-type TRKA with similar potency. Interestingly, both degraders, especially 6, showed selectivity for the degradation of endogenous TPM3-TRKA over ectopically expressed ATP/GTP binding protein-like 4 (AGBL4)-TRKB or ETS variant transcription factor 6 (ETV6)-TRKC fusion proteins in KM12 cells. Global proteomic profiling assays demonstrated that 5 is highly selective for the intended target. TPM3-TRKA protein degradation induced by 5 and 6 was further confirmed to be mediated through cereblon and the ubiquitin-proteasome system. Compared with the parental TRK kinase inhibitor, both degraders exhibited higher potency for inhibiting growth of KM12 cells. Moreover, both 5 and 6 showed good plasma exposure levels in mice. Therefore, 5 and 6 are valuable chemical tool compounds for investigating the in vivo function of TRK fusion during tumorigenesis. Our study also paves the way for pharmacological degradation of TRK." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.0c01342"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c01342">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.0c01342&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.0c01342&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.0c01342&amp;href=/doi/10.1021/acs.jmedchem.0c01342" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2020</span><span class="cit-fg-volume">, 63</span><span class="cit-fg-issue">, 23</span><span class="cit-fg-pageRange">, 14562-14575</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/63/23" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.0c01947" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Featured Article</span><a href="/doi/10.1021/acs.jmedchem.0c01245" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Discovery of First-In-Class Potent and Selective Tropomyosin Receptor Kinase Degraders</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Liqun Chen</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Liqun Chen</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Cullgen (Shanghai), Inc., 230 ChuanHong Road, Building 6, Chuansha, Pudong New Area, Shanghai 201202, People’s Republic of China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Liqun++Chen">Liqun Chen</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Yanke Chen</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Yanke Chen</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Cullgen (Shanghai), Inc., 230 ChuanHong Road, Building 6, Chuansha, Pudong New Area, Shanghai 201202, People’s Republic of China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Yanke++Chen">Yanke Chen</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Chunyan Zhang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Chunyan Zhang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Cullgen (Shanghai), Inc., 230 ChuanHong Road, Building 6, Chuansha, Pudong New Area, Shanghai 201202, People’s Republic of China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Chunyan++Zhang">Chunyan Zhang</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Bingyang Jiao</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Bingyang Jiao</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Cullgen (Shanghai), Inc., 230 ChuanHong Road, Building 6, Chuansha, Pudong New Area, Shanghai 201202, People’s Republic of China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Bingyang++Jiao">Bingyang Jiao</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Sheng Liang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Sheng Liang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Cullgen (Shanghai), Inc., 230 ChuanHong Road, Building 6, Chuansha, Pudong New Area, Shanghai 201202, People’s Republic of China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Sheng++Liang">Sheng Liang</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Qiong Tan</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Qiong Tan</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Cullgen (Shanghai), Inc., 230 ChuanHong Road, Building 6, Chuansha, Pudong New Area, Shanghai 201202, People’s Republic of China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Qiong++Tan">Qiong Tan</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Hongyu Chai</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Hongyu Chai</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Cullgen (Shanghai), Inc., 230 ChuanHong Road, Building 6, Chuansha, Pudong New Area, Shanghai 201202, People’s Republic of China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Hongyu++Chai">Hongyu Chai</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Weihua Yu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Weihua Yu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Cullgen (Shanghai), Inc., 230 ChuanHong Road, Building 6, Chuansha, Pudong New Area, Shanghai 201202, People’s Republic of China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Weihua++Yu">Weihua Yu</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Yongzheng Qian</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Yongzheng Qian</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Cullgen (Shanghai), Inc., 230 ChuanHong Road, Building 6, Chuansha, Pudong New Area, Shanghai 201202, People’s Republic of China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Yongzheng++Qian">Yongzheng Qian</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Hui Yang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Hui Yang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Cullgen (Shanghai), Inc., 230 ChuanHong Road, Building 6, Chuansha, Pudong New Area, Shanghai 201202, People’s Republic of China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Hui++Yang">Hui Yang</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Wuyi Yao</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Wuyi Yao</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Cullgen (Shanghai), Inc., 230 ChuanHong Road, Building 6, Chuansha, Pudong New Area, Shanghai 201202, People’s Republic of China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Wuyi++Yao">Wuyi Yao</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Jianguo Yu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jianguo Yu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Cullgen (Shanghai), Inc., 230 ChuanHong Road, Building 6, Chuansha, Pudong New Area, Shanghai 201202, People’s Republic of China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jianguo++Yu">Jianguo Yu</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Ying Luo</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Ying Luo</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Cullgen Inc., 12671 High Bluff Drive Suite 130, San Diego, California 92130, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Ying++Luo">Ying Luo</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Michael Plewe</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Michael Plewe</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Cullgen Inc., 12671 High Bluff Drive Suite 130, San Diego, California 92130, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Michael++Plewe">Michael Plewe</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Jialiang Wang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jialiang Wang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Cullgen Inc., 12671 High Bluff Drive Suite 130, San Diego, California 92130, United States</div><div class="loa-info-affiliations-info">Cullgen (Shanghai), Inc., 230 ChuanHong Road, Building 6, Chuansha, Pudong New Area, Shanghai 201202, People’s Republic of China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jialiang++Wang">Jialiang Wang</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Xiao-Ran Han</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Xiao-Ran Han</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Cullgen Inc., 12671 High Bluff Drive Suite 130, San Diego, California 92130, United States</div><div class="loa-info-affiliations-info">Cullgen (Shanghai), Inc., 230 ChuanHong Road, Building 6, Chuansha, Pudong New Area, Shanghai 201202, People’s Republic of China</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#136b7a727c61727d3d7b727d5370667f7f74767d3d707c7e"><span class="__cf_email__" data-cfemail="add5c4ccc2dfccc383c5ccc3edced8c1c1cac8c383cec2c0">[email protected]</span></a></span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Xiao-Ran++Han">Xiao-Ran Han</a></span></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Jing Liu</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jing Liu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Cullgen Inc., 12671 High Bluff Drive Suite 130, San Diego, California 92130, United States</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#6b0102050c4507021e2b081e07070c0e0545080406"><span class="__cf_email__" data-cfemail="640e0d0a034a080d11240711080803010a4a070b09">[email protected]</span></a></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jing++Liu">Jing Liu</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-4740-243X" title="Orcid link">http://orcid.org/0000-0003-4740-243X</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c01342&amp;href=/doi/10.1021%2Facs.jmedchem.0c01342" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2020</span></span><span class="cit-volume">, 63</span><span class="cit-issue">, 23</span><span class="cit-pageRange">, 14562–14575</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">October 15, 2020</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>7 August 2020</li><li><span class="item_label"><b>Published</b> online</span>15 October 2020</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 10 December 2020</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.0c01342" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c01342</a></div><div class="article_header-article-copyright"><strong>Copyright © 2020 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D14562%26pageCount%3D14%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DLiqun%2BChen%252C%2BYanke%2BChen%252C%2BChunyan%2BZhang%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D63%26issueNum%3D23%26contentID%3Dacs.jmedchem.0c01342%26title%3DDiscovery%2Bof%2BFirst-In-Class%2BPotent%2Band%2BSelective%2BTropomyosin%2BReceptor%2BKinase%2BDegraders%26numPages%3D14%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D14575%26publicationDate%3DDecember%2B2020">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.0c01342"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">2850</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">3</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.0c01342" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Discovery of First-In-Class Potent and Selective Tropomyosin Receptor Kinase Degraders&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Liqun&quot;,&quot;last_name&quot;:&quot;Chen&quot;},{&quot;first_name&quot;:&quot;Yanke&quot;,&quot;last_name&quot;:&quot;Chen&quot;},{&quot;first_name&quot;:&quot;Chunyan&quot;,&quot;last_name&quot;:&quot;Zhang&quot;},{&quot;first_name&quot;:&quot;Bingyang&quot;,&quot;last_name&quot;:&quot;Jiao&quot;},{&quot;first_name&quot;:&quot;Sheng&quot;,&quot;last_name&quot;:&quot;Liang&quot;},{&quot;first_name&quot;:&quot;Qiong&quot;,&quot;last_name&quot;:&quot;Tan&quot;},{&quot;first_name&quot;:&quot;Hongyu&quot;,&quot;last_name&quot;:&quot;Chai&quot;},{&quot;first_name&quot;:&quot;Weihua&quot;,&quot;last_name&quot;:&quot;Yu&quot;},{&quot;first_name&quot;:&quot;Yongzheng&quot;,&quot;last_name&quot;:&quot;Qian&quot;},{&quot;first_name&quot;:&quot;Hui&quot;,&quot;last_name&quot;:&quot;Yang&quot;},{&quot;first_name&quot;:&quot;Wuyi&quot;,&quot;last_name&quot;:&quot;Yao&quot;},{&quot;first_name&quot;:&quot;Jianguo&quot;,&quot;last_name&quot;:&quot;Yu&quot;},{&quot;first_name&quot;:&quot;Ying&quot;,&quot;last_name&quot;:&quot;Luo&quot;},{&quot;first_name&quot;:&quot;Michael&quot;,&quot;last_name&quot;:&quot;Plewe&quot;},{&quot;first_name&quot;:&quot;Jialiang&quot;,&quot;last_name&quot;:&quot;Wang&quot;},{&quot;first_name&quot;:&quot;Xiao-Ran&quot;,&quot;last_name&quot;:&quot;Han&quot;},{&quot;first_name&quot;:&quot;Jing&quot;,&quot;last_name&quot;:&quot;Liu&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2020&quot;,&quot;month&quot;:&quot;10&quot;,&quot;day&quot;:&quot;15&quot;,&quot;issue&quot;:&quot;23&quot;,&quot;volume&quot;:&quot;63&quot;,&quot;pages&quot;:&quot;14562-14575&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.0c01342&quot;},&quot;abstract&quot;:&quot;We report compounds 5 (CG416) and 6 (CG428) as two first-in-class tropomyosin receptor kinase (TRK) degraders that target the intracellular kinase domain of TRK. Degraders 5 and 6 reduced levels of the tropomyosin 3 (TPM3)-TRKA fusion protein in KM12 colorectal carcinoma cells and inhibited downstream PLCγ1 signaling at sub-nanomolar concentrations. Both degraders also degraded human wild-type TRKA with similar potency. Interestingly, both degraders, especially 6, showed selectivity for the degradation of endogenous TPM3-TRKA over ectopically expressed ATP/GTP binding protein-like 4 (AGBL4)-TRKB or ETS variant transcription factor 6 (ETV6)-TRKC fusion proteins in KM12 cells. Global proteomic profiling assays demonstrated that 5 is highly selective for the intended target. TPM3-TRKA protein degradation induced by 5 and 6 was further confirmed to be mediated through cereblon and the ubiquitin-proteasome system. Compared with the parental TRK kinase inhibitor, both degraders exhibited higher potency for inhibiti&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c01342&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c01342" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c01342&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c01342" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c01342&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c01342" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c01342&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c01342&amp;href=/doi/10.1021/acs.jmedchem.0c01342" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.0c01342" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.0c01342" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (7 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c01342&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.0c01342%26sid%3Dliteratum%253Aachs%26pmid%3D33058680%26genre%3Darticle%26aulast%3DChen%26date%3D2020%26atitle%3DDiscovery%2Bof%2BFirst-In-Class%2BPotent%2Band%2BSelective%2BTropomyosin%2BReceptor%2BKinase%2BDegraders%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D63%26issue%3D23%26spage%3D14562%26epage%3D14575%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (2)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=291781" title="Degradation">Degradation</a>,</li><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=291316" title="Mixtures">Mixtures</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=291352" title="Screening assays">Screening assays</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/63/23" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/jmcmar.2020.63.issue-23/20201210/jmcmar.2020.63.issue-23.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01342/20201203/images/medium/jm0c01342_0013.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01342/20201203/images/large/jm0c01342_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01342&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">We report compounds <b>5</b> (CG416) and <b>6</b> (CG428) as two first-in-class tropomyosin receptor kinase (TRK) degraders that target the intracellular kinase domain of TRK. Degraders <b>5</b> and <b>6</b> reduced levels of the tropomyosin 3 (TPM3)-TRKA fusion protein in KM12 colorectal carcinoma cells and inhibited downstream PLCγ1 signaling at sub-nanomolar concentrations. Both degraders also degraded human wild-type TRKA with similar potency. Interestingly, both degraders, especially <b>6</b>, showed selectivity for the degradation of endogenous TPM3-TRKA over ectopically expressed ATP/GTP binding protein-like 4 (AGBL4)-TRKB or ETS variant transcription factor 6 (ETV6)-TRKC fusion proteins in KM12 cells. Global proteomic profiling assays demonstrated that <b>5</b> is highly selective for the intended target. TPM3-TRKA protein degradation induced by <b>5</b> and <b>6</b> was further confirmed to be mediated through cereblon and the ubiquitin-proteasome system. Compared with the parental TRK kinase inhibitor, both degraders exhibited higher potency for inhibiting growth of KM12 cells. Moreover, both <b>5</b> and <b>6</b> showed good plasma exposure levels in mice. Therefore, <b>5</b> and <b>6</b> are valuable chemical tool compounds for investigating the <i>in vivo</i> function of TRK fusion during tumorigenesis. Our study also paves the way for pharmacological degradation of TRK.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_22213" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_22213" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">The tropomyosin receptor kinase (TRK) receptor family comprises three members: TRKA, TRKB, and TRKC that are encoded by the <i>NTRK1</i>, <i>NTRK2</i>, and <i>NTRK3</i> genes, respectively.<a onclick="showRef(event, 'ref1 ref2 ref3 ref4 ref5'); return false;" href="javascript:void(0);" class="ref ref1 ref2 ref3 ref4 ref5">(1−5)</a> TRKs are receptor tyrosine kinases primarily implicated in the development and function of the neuronal tissues. The main ligands of TRKs are the nerve growth factor (NGF) for TRKA,<a onclick="showRef(event, 'ref6 ref7'); return false;" href="javascript:void(0);" class="ref ref6 ref7">(6,7)</a> brain-derived neurotrophic factor (BDNF) for TRKB,<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> and neurotrophins for TRKC.<a onclick="showRef(event, 'ref9 ref10'); return false;" href="javascript:void(0);" class="ref ref9 ref10">(9,10)</a> The binding of ligands to the extracellular domains of TRKs induces dimerization and activation of the intracellular kinase domains, which provide signaling to downstream pathways, primarily PI3K/AKT, RAF/MEK/ERK, and phospholipase C gamma (PLCγ).<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> Various genetic aberrations of <i>NTRK</i> genes have been reported, including chromosomal translocation, amplification, point mutations, and alternative splicing. These changes lead to constitutive activation of the TRK pathway that promotes cellular proliferation, survival, and malignant transformation.<a onclick="showRef(event, 'ref12 ref13 ref14 ref15'); return false;" href="javascript:void(0);" class="ref ref12 ref13 ref14 ref15">(12−15)</a> Among the different molecular mechanisms of TRK activation, <i>NTRK</i> gene chromosomal translocation has been the most frequently studied.<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> The most common TRK fusions include tropomyosin 3 (TPM3)-TRKA, lamin A/C-TRKA, and ETS variant transcription factor 6 (ETV6)-TRKC.<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> Typically, the 3′ regions of the <i>NTRK</i> genes are joined with the 5′ regions of a partner gene because of chromosomal rearrangement. The resulting chimeric proteins lead to constitutive activation of the kinase domain and downstream oncogenic signaling.<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> Additionally, expression of the chimeric proteins is driven by the promoters of the fusion partners, which often result in overexpression.<a onclick="showRef(event, 'ref18 ref19'); return false;" href="javascript:void(0);" class="ref ref18 ref19">(18,19)</a> TRK fusions have been found across a wide range of human malignancies, including lung cancer,<a onclick="showRef(event, 'ref20 ref21'); return false;" href="javascript:void(0);" class="ref ref20 ref21">(20,21)</a> colorectal cancer,<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> and soft tissue sarcoma.<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> In addition, TRK fusions appear to be the primary oncological drivers of some rare cancers, such as infantile fibrosarcoma,<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> secretory breast carcinoma,<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> and mammary analogue secretory carcinoma (MASC).<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a></div><div class="NLM_p">In the past decade, significant efforts have been devoted toward the development of TRK inhibitors that can antagonize TRK’s catalytic function.<a onclick="showRef(event, 'ref14 ref15 ref27'); return false;" href="javascript:void(0);" class="ref ref14 ref15 ref27">(14,15,27)</a> As a result, two TRK inhibitors, <b>1</b> (larotrectinib) and <b>2</b> (entrectinib), have been approved by FDA to treat certain cancer patients carrying <i>NTRK</i> fusion genes (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>A).<a onclick="showRef(event, 'ref28 ref29'); return false;" href="javascript:void(0);" class="ref ref28 ref29">(28,29)</a> Although both <b>1</b> and <b>2</b> showed good safety profiles in the clinic, modest on-target adverse effects, such as dizziness/ataxia, paresthesia, and weight gain, have been observed.<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> Some of these adverse effects are believed to result from inhibition of wild-type TRK signaling in the brain. In addition, acquired drug resistance has been reported in patients after treatment with these FDA-approved drugs.<a onclick="showRef(event, 'ref14 ref27'); return false;" href="javascript:void(0);" class="ref ref14 ref27">(14,27)</a> Hence, therapeutic approaches with a new mechanism of action (MOA) are desired to provide more durable inhibition of the TRK pathway with fewer adverse effects.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01342/20201203/images/medium/jm0c01342_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01342/20201203/images/large/jm0c01342_0002.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Structures of TRK inhibitors. (A) Chemical structures. (B) X-ray cocrystal structure of compound <b>3</b> (cyan) in the complex with TRKA (gray) (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4YNE">4YNE</a>). The loop formed by GLY605 and GLY623 residues is highlighted in light blue.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01342/20201203/images/large/jm0c01342_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01342&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Molecules capable of chemical downregulation of target protein levels using proteolysis-targeting chimeras have been called PROTACs (PROteolysis TArgeting Chimeras), uSMITEs (ubiquitin-mediated, small molecule-induced target elimination), SNIPERs (specific and nongenetic IAP-dependent protein ERasers), DEGRONIMIDs, and other acronyms. In this paper, we will simply refer to these molecules as degraders. Numerous degraders have exhibited significant advantages over enzymatic inhibitors.<a onclick="showRef(event, 'ref30 ref31 ref32 ref33 ref34'); return false;" href="javascript:void(0);" class="ref ref30 ref31 ref32 ref33 ref34">(30−34)</a> For example, protein degraders not only suppress the enzymatic activity of the intended targets but also deplete the scaffolding and other noncatalytic functions of the targeted proteins.<a onclick="showRef(event, 'ref32 ref35 ref36'); return false;" href="javascript:void(0);" class="ref ref32 ref35 ref36">(32,35,36)</a> Degraders also have the potential to overcome the acquired resistance induced by traditional small-molecule drugs.<a onclick="showRef(event, 'ref31 ref37 ref38 ref39'); return false;" href="javascript:void(0);" class="ref ref31 ref37 ref38 ref39">(31,37−39)</a> Moreover, because of the catalytic nature of degraders, high exposure may be avoided to ameliorate potential adverse effects.<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a></div><div class="NLM_p last">A growing number of protein degraders have been developed to selectively degrade disease-associated proteins, of which many are kinases, including RIPK2,<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a> BCR-ABL,<a onclick="showRef(event, 'ref42 ref43 ref44'); return false;" href="javascript:void(0);" class="ref ref42 ref43 ref44">(42−44)</a> CDK9,<a onclick="showRef(event, 'ref45 ref46'); return false;" href="javascript:void(0);" class="ref ref45 ref46">(45,46)</a> FLT3,<a onclick="showRef(event, 'ref47'); return false;" href="javascript:void(0);" class="ref ref47">(47)</a> BTK,<a onclick="showRef(event, 'ref37 ref47'); return false;" href="javascript:void(0);" class="ref ref37 ref47">(37,47)</a> EGFR,<a onclick="showRef(event, 'ref48 ref49'); return false;" href="javascript:void(0);" class="ref ref48 ref49">(48,49)</a> ALK,<a onclick="showRef(event, 'ref50 ref51 ref52'); return false;" href="javascript:void(0);" class="ref ref50 ref51 ref52">(50−52)</a> CDK4/6,<a onclick="showRef(event, 'ref53 ref54 ref55'); return false;" href="javascript:void(0);" class="ref ref53 ref54 ref55">(53−55)</a> MEK,<a onclick="showRef(event, 'ref56 ref57'); return false;" href="javascript:void(0);" class="ref ref56 ref57">(56,57)</a> and BRAF.<a onclick="showRef(event, 'ref58'); return false;" href="javascript:void(0);" class="ref ref58">(58)</a> Recently, a wild-type TRKC degrader has been reported using a bivalent dipeptidomimetic which binds wild-type TRKC at the ectodomain.<a onclick="showRef(event, 'ref59'); return false;" href="javascript:void(0);" class="ref ref59">(59)</a> Because TRK activation in human cancers typically includes fusions, mutations, and deletions at the extracellular domain, TRK degraders targeting the ectodomain may have limited application to treat cancers. In this study, we designed, synthesized, and evaluated a series of TRK degraders derived from an analogue (<b>3</b>) of a pan TRK inhibitor <b>4</b> (GNF-8625)<a onclick="showRef(event, 'ref60'); return false;" href="javascript:void(0);" class="ref ref60">(60)</a> and a cereblon (CRBN) E3 ligase ligand, pomalidomide.<a onclick="showRef(event, 'ref61'); return false;" href="javascript:void(0);" class="ref ref61">(61)</a> Through a structure–activity relationship (SAR) study, we identified two potent and selective TRK degraders, <b>5</b> (CG416) and 6 (CG428). Here, we present the discovery and detailed biological characterization of these two first-in-class TRK degraders that target the intracellular domain of the kinase.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_75368" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_75368" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> Design and Synthesis of Putative TRK Degraders</h3><div class="NLM_p">GNF-8625 is a type I pan TRK kinase inhibitor competitively occupying the ATP binding site.<a onclick="showRef(event, 'ref60'); return false;" href="javascript:void(0);" class="ref ref60">(60)</a> Although an X-ray cocrystal structure of GNF-8625 in complex with TRK does not currently exist, a cocrystal structure is available for its truncated analogue <b>3</b> in complex with the TRKA kinase domain (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4YNE">4YNE</a>) (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>B).<a onclick="showRef(event, 'ref60'); return false;" href="javascript:void(0);" class="ref ref60">(60)</a> It is worth mentioning that a loop between GLY605 and GLY623 in the TRKA C-lobe (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>B) together with the N-lobe of the kinase forms a concave surface at the incoming E3 complex binding site. This surface shape may be unfavorable for the protein–protein interactions between TRK and the recruited E3 ligase. However, we speculated that this loop could be flexible enough to adapt to another geometry to accommodate the incoming E3 complex. This hypothesis fueled our curiosity to initiate this TRK degrader project. Based on the cocrystal structure and the published cellular activity data obtained from the Ba/F3 assay (TRKA IC<sub>50</sub> = 3 nM for <b>3</b> and GNF-8625), we anticipate that both <b>3</b> and GNF-8625 should bind to TRKA.<a onclick="showRef(event, 'ref60'); return false;" href="javascript:void(0);" class="ref ref60">(60)</a> Considering the significant difference in the molecular weights between the two compounds (MW = 359 for <b>3</b> and 536 for GNF-8625), we decided to choose <b>3</b> as the TRK binding moiety for our TRK degraders. From the X-ray crystal structure, the pyridinyl group of <b>3</b> is at the solvent front and suitable to attach linkers. To facilitate the synthesis of degraders, we attached a piperazinyl moiety at the 2-position of the pyridinyl group (compound <b>7</b> in <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>). This newly installed piperazinyl group can serve as a bridge between <b>3</b> and the linker region. It is well documented that the linker region is critical for the development of degraders.<a onclick="showRef(event, 'ref62'); return false;" href="javascript:void(0);" class="ref ref62">(62)</a> Therefore, we designed degraders bearing a variety of carbon linkers or polyethylene glycol (PEG) linkers (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a> and <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). In this study, we chose pomalidomide as the E3 recruiting ligand because of its relatively low molecular weight (MW = 273) and potentially favorable PK properties compared with Von Hippel–Lindau (VHL) recruiting ligands.<a onclick="showRef(event, 'ref63 ref64 ref65'); return false;" href="javascript:void(0);" class="ref ref63 ref64 ref65">(63−65)</a></div><figure id="sch1" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01342/20201203/images/medium/jm0c01342_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01342/20201203/images/large/jm0c01342_0010.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthetic Routes for TRK Degraders <b>5</b> and <b>6</b> and Their Control Analogues <b>25</b> and <b>26</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01342/20201203/images/large/jm0c01342_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01342&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) KF, DMSO, 100 °C, 12 h; (b) Pd(Ph<sub>3</sub>P)<sub>4</sub>, toluene, 110 °C, 12 h; (c) piperazine, KF, DMSO, 130 °C; (d) DIEA, NMP, microwave, 85 °C, 50 min; (e) formic acid, 30 h; (f) EDCI, HOAt, NMM, DMSO, rt, overnight; and (g) MeI, K<sub>2</sub>CO<sub>3</sub>, DMSO, rt, 2 h.</p></p></figure><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. SAR of Synthesized Putative TRK Degraders<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01342/20201203/images/medium/jm0c01342_0011.gif" alt="" id="GRAPHIC-d7e669-autogenerated" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01342/20201203/images/medium/jm0c01342_0012.gif" alt="" id="gr10" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup><sup>a</sup></sup><p class="last">KM12 cells were treated with indicated compounds at 0.1, 1, and 10 nM for 6 h. TPM3-TRKA protein levels were determined by immunoblotting and normalized against α-tubulin. The experiments were performed in triplicate, and the values are presented as mean ± SD.</p></div></div><div></div></div><div class="NLM_p last">We developed a convergent synthetic route to prepare the designed putative TRK degraders. The synthesis of two key degraders, <b>5</b> and <b>6</b>, and their corresponding control compounds, <b>25</b> and <b>26</b>, are outlined in <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>. Briefly, 3-bromo-6-chloroimidazo[1,2-<i>b</i>]pyridazine <b>8</b> reacted with (<i>R</i>)-2-(3-fluorophenyl)pyrrolidine <b>9</b> through a nucleophilic aromatic substitution reaction to yield intermediate <b>10</b>.<a onclick="showRef(event, 'ref60'); return false;" href="javascript:void(0);" class="ref ref60">(60)</a> Palladium-catalyzed Stille coupling reaction between <b>10</b> and 2-fluoro-6-(tributylstannyl)pyridine <b>11</b> provided intermediate <b>12</b>,<a onclick="showRef(event, 'ref60'); return false;" href="javascript:void(0);" class="ref ref60">(60)</a> which reacted with piperazine through a nucleophilic substitution reaction to provide the common intermediate <b>7</b>. The linker-attached pomalidomide analogues <b>15a</b> and <b>15b</b> were prepared from 2-fluoro-substituted thalidomide <b>13</b> through nucleophilic aromatic substitution reactions with amino esters <b>14a</b> and <b>14b</b>, respectively, followed by acid-mediated <i>tert</i>-butyl ester deprotection.<a onclick="showRef(event, 'ref58'); return false;" href="javascript:void(0);" class="ref ref58">(58)</a> Amide coupling of the common intermediate <b>7</b> with <b>15a</b> and <b>15b</b> yielded degraders <b>5</b> and <b>6</b>, respectively. Other putative degraders (<b>16</b>–<b>24</b>) were synthesized similarly using different linker-attached pomalidomide analogues. It has been well documented that N-methylation of the glutarimide moiety could impair the binding of pomalidomide-based degraders to CRBN, thus diminishing the target protein degradation capability.<a onclick="showRef(event, 'ref50 ref58'); return false;" href="javascript:void(0);" class="ref ref50 ref58">(50,58)</a> Therefore, the <i>N</i>-methyl glutarimide analogues <b>25</b> and <b>26</b> were prepared from <b>5</b> and <b>6</b>, respectively, by <i>N</i>-alkylation reactions with methyl iodide (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>).</div></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i7" class="anchor-spacer"></div><h3 class="article-section__title" id="_i7"> SAR Study of the Synthesized Putative TRK Degraders</h3><div class="NLM_p last">The TRK degradation capability of the synthesized compounds was evaluated using an immunoblotting assay in KM12 cells (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>, and <a href="/doi/suppl/10.1021/acs.jmedchem.0c01342/suppl_file/jm0c01342_si_001.pdf" class="ext-link">Figure S3</a>), which are human colorectal carcinoma cells expressing the TPM3-TRKA fusion protein.<a onclick="showRef(event, 'ref21 ref66'); return false;" href="javascript:void(0);" class="ref ref21 ref66">(21,66)</a> Our data showed that all of the synthesized compounds significantly reduced TPM3-TRKA levels at 10 nM after 6 h of treatment. Interestingly, the pan-kinase inhibitor GNF-8625 also decreased TPM3-TRKA protein levels at 10 nM, albeit to a lesser degree (52 ± 5% degradation). Compound <b>17</b>, bearing an ethylene linker, showed a similar effect as GNF-8625 and reduced protein levels by 65 ± 5%. The remaining designed compounds appear to be much more effective than GNF-8625 and <b>17</b> at this concentration with at least 80% degradation at 10 nM. At 1 nM compound concentration, GNF-8625, <b>17</b>, <b>20</b>, <b>21</b>, and <b>23</b> did not significantly reduce TPM3-TRKA levels (<30% degradation). Compounds <b>18</b>, <b>19</b>, and <b>5</b>, bearing propylene, butylene, and pentylene linkers, respectively, were the most potent compounds carrying alkyl linkers and led to over 60% TPM3-TRKA fusion protein downregulation at 1 nM. Among compounds carrying PEG linkers, <b>6</b>, bearing a short PEG linker, showed the best degradation potency with 82 ± 2% TPM3-TRKA reduction. At 0.1 nM compound concentration, only compound <b>6</b> significantly degraded TPM3-TRKA (46 ± 6%). Compounds <b>5</b> and <b>6</b> are close analogues with only one atom difference on the linker moiety. Hence, they are an excellent pair of degraders for investigating the effects of the subtle linker chemical structure changes. Therefore, we selected these two compounds for further biological characterizations.</div></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i8" class="anchor-spacer"></div><h3 class="article-section__title" id="_i8"> Compounds <b>5</b> and <b>6</b> Bind TRKA, TRKB, and TRKC with High Affinity</h3><div class="NLM_p">In order to design our degraders, we significantly modified the chemical structure of the pan TRK inhibitor, GNF-8625, by replacing the 4-hydroxypiperidinyl-substituted pyridinyl group with a piperazinyl moiety and adding the linker and E3 binder moieties. To determine whether these structural changes at the solvent exposed area affected the binding to TRK proteins, we evaluated the binding affinities of inhibitor GNF-8625, degraders <b>5</b> and <b>6</b>, and the control compounds <b>25</b> and <b>26</b> to TRKA, TRKB, and TRKC using the DiscoverX KINOMEscan platform (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>). Compared with GNF-8625, the binding affinities of the carbon linker-bearing degrader <b>5</b> were slightly decreased by about three-, five-, and threefold for TRKA, TRKB, and TRKC, respectively. The PEG linker-derived degrader <b>6</b>, however, showed identical binding affinities to all three TRK family members. These binding affinity results validated our hypothesis that modifications of solvent-exposed groups of GNF-8625 are well tolerated. Importantly, the degrader and control pairs showed similar binding affinities to all three TRKs. These data confirmed that N-methylation of the glutarimide on the pomalidomide moiety does not affect the binding affinities to TRKs. Considering the close chemical structural similarities and near-identical target protein binding affinities to their corresponding degraders, <b>25</b> and <b>26</b> are excellent control compounds for <b>5</b> and <b>6</b>, respectively.</div><figure id="fig2" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01342/20201203/images/medium/jm0c01342_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01342/20201203/images/large/jm0c01342_0003.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Binding affinities of GNF-8625, degraders (<b>5</b> and <b>6</b>), and negative controls (<b>25</b> and <b>26</b>) to TRKA, TRKB, or TRKC. Binding affinities of individual compounds were determined using the KINOMEscan assay (DiscoveryX). The lowest concentration points represent data from the DMSO samples. Data are shown as mean ± SD derived from duplicated independent experiments.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01342/20201203/images/large/jm0c01342_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01342&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i9" class="anchor-spacer"></div><h3 class="article-section__title" id="_i9"> Compounds <b>5</b> and <b>6</b> are Potent and Selective TRK Degraders</h3><div class="NLM_p">To quantitatively determine the TRK degradation capability of compounds <b>5</b> and <b>6</b>, KM12 cells were incubated with these degraders at various concentrations for 6 h. As illustrated in <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>A, compounds <b>5</b> and <b>6</b> potently induced TPM3-TRKA degradation (DC<sub>50</sub> = 0.48 ± 0.06 nM for <b>5</b> and 0.36 ± 0.06 nM for <b>6</b>) and inhibited downstream PLCγ1 phosphorylation (IC<sub>50</sub> = 0.36 ± 0.06 nM for <b>5</b> 0.33 ± 0.06 nM for <b>6</b>) in a concentration-dependent manner. Similar to GNF-8625 (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01342/suppl_file/jm0c01342_si_001.pdf" class="ext-link">Figure S3</a>), the control compounds <b>25</b> and <b>26</b> also reduced TPM3-TRKA levels at 30 nM. Although we are still investigating the degradation mechanisms of GNF-8625 and the control compounds, we are certain that degraders <b>5</b> and <b>6</b> are significantly more potent than these compounds for the degradation of the TPM3-TRKA fusion protein. We further confirmed that inhibition of PLCγ1 phosphorylation is not due to the modulation of PLCγ1 protein levels (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>A). It was recently reported that immunomodulatory imide drug (IMiD)-derived degraders degrade G1 to S phase transition 1 (GSPT1) protein, a neo-substrate for CC-885,<a onclick="showRef(event, 'ref67'); return false;" href="javascript:void(0);" class="ref ref67">(67)</a> and lead to unexpected toxicity to certain types of cells.<a onclick="showRef(event, 'ref68 ref69 ref70'); return false;" href="javascript:void(0);" class="ref ref68 ref69 ref70">(68−70)</a> Therefore, we assessed GSPT1 protein levels in our experiments and determined that neither degraders affected GSPT1 levels (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>A).</div><figure id="fig3" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01342/20201203/images/medium/jm0c01342_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01342/20201203/images/large/jm0c01342_0004.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Compounds <b>5</b> and <b>6</b> induce degradation of TRKA fusion and wild-type TRKA. (A) KM12 cells harboring TPM3-TRKA fusion were treated with DMSO or indicated compounds following a threefold serial dilution for 6 h. (B) KM12 cells were treated with 10 nM compound <b>5</b> or <b>6</b> for indicated time points. (C) HEL cells harboring wild-type TRKA were treated with DMSO or indicated compounds following a threefold serial dilution for 6 h. Levels of indicated proteins were determined by immunoblotting and normalized against loading controls.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01342/20201203/images/large/jm0c01342_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01342&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">We further evaluated the kinetics of the induced TPM3-TRKA degradation process in KM12 cells. As illustrated in <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>B, significant TPM3-TRKA protein degradation and PLCγ1 phosphorylation inhibition were observed for both <b>5</b> and <b>6</b> within 0.3 h, and near complete TPM3-TRKA degradation and PLCγ1 phosphorylation inhibition were achieved within 4 h post 10 nM compound treatment. Therefore, both <b>5</b> and <b>6</b> rapidly induced TPM3-TRKA protein reduction and inhibited downstream signaling in KM12 cells.</div><div class="NLM_p">It has been reported that some protein degraders, such as EGFR degraders<a onclick="showRef(event, 'ref48 ref49'); return false;" href="javascript:void(0);" class="ref ref48 ref49">(48,49)</a> and BRAF degraders,<a onclick="showRef(event, 'ref58'); return false;" href="javascript:void(0);" class="ref ref58">(58)</a> have the potential to selectively downregulate mutated proteins and spare the wild-type proteins. We evaluated compounds <b>5</b> and <b>6</b> in HEL cells expressing endogenous wild-type TRKA (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>C). In contrast to EGFR degraders and BRAF degraders, our TRK degraders <b>5</b> and <b>6</b> did not spare the wild-type TRKA. Instead, both degraders <b>5</b> and <b>6</b> degraded wild-type TRKA in HEL cells with high potency (DC<sub>50</sub> values of 1.26 ± 0.25 and 2.23 ± 0.42 nM, respectively). Conversely, the control compounds <b>25</b> and <b>26</b> did not degrade wild-type TRKA in HEL cells at 30 nM.</div><div class="NLM_p">To assess the degradation potency of compounds <b>5</b> and <b>6</b> on other TRK family members, we ectopically expressed ATP/GTP binding protein-like 4 (AGBL4)-TRKB or ETV6-TRKC fusion proteins in KM12 cells. After incubating the cells with various concentrations of <b>5</b> and <b>6</b> for 6 h, we assessed the TRK fusion protein levels (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>). Interestingly, compared with the TPM3-TRKA degradation results shown in <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>A, compounds <b>5</b> and <b>6</b> demonstrated much weaker degradation potency for overexpressed AGBL4-TRKB or ETV6-TRKC in KM12 cells. This selectivity is more significant for the PEG linker-derived degrader <b>6</b>. We believe that such a selectivity profile within the same TRK family members may be due to the decreased binding affinities of <b>5</b> and <b>6</b> for TRKB (<i>K</i><sub>d</sub> = 112 ± 21 nM for <b>5</b> and 28 ± 5 nM for <b>6</b>) and TRKC (<i>K</i><sub>d</sub> = 18 ± 4 nM for <b>5</b> and 4.2 ± 1.0 nM for <b>6</b>) compared with TRKA (<i>K</i><sub>d</sub> = 2.1 ± 0.3 nM for <b>5</b> and 1.0 ± 0.2 nM for <b>6</b>) (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>). However, we do not rule out the possible contributions from subcellular locations, microenvironments, protein surfaces, and protein expression levels of these TRK fusion proteins. In addition, with respect to the degrader-induced AGBL4-TRKB downregulation experiments, a “hook effect” was observed for both compounds <b>5</b> and <b>6</b> at 500 nM compound concentration.<a onclick="showRef(event, 'ref48'); return false;" href="javascript:void(0);" class="ref ref48">(48)</a> We also found that the weaker TRKB binder <b>5</b> exhibited a superior degradation capability for AGBL4-TRKB when compared to the stronger TRKB binder <b>6</b> (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>A). This unexpected result highlights the fact that binding affinity is not the only factor that determines the degradation potency of degrader molecules. As illustrated in <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>B, both <b>5</b> and <b>6</b> are weak degraders for the overexpressed ETV6-TRKC fusion proteins.</div><figure id="fig4" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01342/20201203/images/medium/jm0c01342_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01342/20201203/images/large/jm0c01342_0005.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Compounds <b>5</b> and <b>6</b> induce degradation of TRKB and TRKC fusion protein. (A,B) KM12 cells stably expressing AGBL4-TRKB or ETV6-TRKC fusion proteins were treated with DMSO or indicated compounds following a threefold serial dilution for 6 h. Levels of indicated proteins were determined by immunoblotting and normalized against loading controls.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01342/20201203/images/large/jm0c01342_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01342&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">We further evaluated the selectivity of compound <b>5</b> across the global proteome using tandem mass tag (TMT)-based quantitative proteomics technology. The samples were prepared by treating KM12 cells with 100 nM (200-fold of the DC<sub>50</sub> value) of compound <b>5</b> or dimethyl sulfoxide (DMSO) for 6 h. This relatively high compound concentration was selected for exploring possible off-targets. TPM3-TRKA downregulation was confirmed by immunoblotting (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01342/suppl_file/jm0c01342_si_001.pdf" class="ext-link">Figure S4</a>). Out of the 7903 identified proteins, only 5 proteins, TRKA, GDF15, ERRFI1, DUSP4, and AREG, were significantly downregulated (<i>p</i>-value <0.01, fold change <0.7) and 2 proteins, SOX4 and FZD6, were upregulated (<i>p</i>-value < 0.01, fold change >1.5), as indicated in the volcano plot (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>, <a href="/doi/suppl/10.1021/acs.jmedchem.0c01342/suppl_file/jm0c01342_si_001.pdf" class="ext-link">Table S1</a>). Although we are still investigating whether these proteins are directly or indirectly targeted by degrader <b>5</b>, the proteomic study results suggest that compound <b>5</b> is a highly selective TRK degrader.</div><figure id="fig5" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01342/20201203/images/medium/jm0c01342_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01342/20201203/images/large/jm0c01342_0006.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Compound <b>5</b> selectively degrades TPM3-TRKA. The volcano plot of proteins with significantly differential abundance for compound <b>5</b>-treated samples <i>vs</i> DMSO-treated control samples. KM12 cells were treated with DMSO or compound <b>5</b> at 100 nM for 6 h and subsequently analyzed using TMT quantitative proteomic analyses. The negative log (base 10) of the <i>P</i>-values is plotted on the <i>y</i>-axis, and the log (base 2) of the fold change is plotted on the <i>x</i>-axis. The green and red data points indicate proteins with a <i>p</i>-value of <0.01 and fold changes of <0.7 and >1.5, respectively. Data were representative of three independent experiments.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01342/20201203/images/large/jm0c01342_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01342&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i10" class="anchor-spacer"></div><h3 class="article-section__title" id="_i10"> Degrader-Induced TRK Downregulation is Mediated by the UPS and CRL4<sup>CRBN</sup></h3><div class="NLM_p">In an effort to verify that the observed TPM3-TRKA degradation induced by degraders <b>5</b> and <b>6</b> is mediated through the ubiquitin-proteasome system (UPS), we performed a series of rescue experiments. KM12 cells were pretreated with an excess amount of pomalidomide (10 μM), MG-132 (20 μM), bortezomib (200 nM), or MLN4924 (5 μM), prior to the treatment with degraders <b>5</b> and <b>6</b> for 6 h (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>A,B). Pomalidomide is a CRBN binder and competes with degraders <b>5</b> and <b>6</b> at the same binding site.<a onclick="showRef(event, 'ref61'); return false;" href="javascript:void(0);" class="ref ref61">(61)</a> MG-132 and bortezomib are proteasome inhibitors. MLN4924 is an inhibitor of the NEDD8-activating enzyme (NAE).<a onclick="showRef(event, 'ref71'); return false;" href="javascript:void(0);" class="ref ref71">(71)</a> It inhibits neddylation of the cullin RING family E3 ubiquitin ligases (CRL) and suppresses their activity. As illustrated in <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>A,B, the addition of these inhibitors significantly impaired TPM3-TRKA degradation induced by degraders <b>5</b> and <b>6</b>, indicating that targeted degradation of TPM3-TRKA was mediated by CRBN, proteasome, and CRL. To further confirm that degrader-induced TPM3-TRKA reduction is mediated by CRBN, we performed another rescue assay by knocking down <i>CRBN</i> in KM12 cells (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>C). As expected, knockdown of <i>CRBN</i> also significantly impaired degrader <b>5</b>-induced TPM3-TRKA protein reduction. Based on these results, we can conclude that <b>5</b> and <b>6</b> degrade TPM3-TRKA fusion protein through UPS and CRL4<sup>CRBN</sup>.</div><figure id="fig6" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01342/20201203/images/medium/jm0c01342_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01342/20201203/images/large/jm0c01342_0007.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Degradation of TPM3-TRKA fusion is mediated by E3 ligase cereblon and the UPS. (A, B) KM12 cells were pretreated with DMSO, pomalidomide (POM, 10 μM), MG-132 (20 μM), bortezomib (BTZ, 200 nM), or MLN4924 (5 μM) and subsequently incubated with 10 nM compounds <b>5</b> or <b>6</b> for 6 h prior to immunoblotting. (C) KM12 cells were transfected with two distinct siRNAs targeting <i>CRBN</i> gene and then treated with compound <b>5</b> following a threefold serial dilution for 6 h prior to immunoblotting.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01342/20201203/images/large/jm0c01342_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01342&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i11" class="anchor-spacer"></div><h3 class="article-section__title" id="_i11"> Compounds <b>5</b> and <b>6</b> Potently Inhibit Cell Growth</h3><div class="NLM_p">We also investigated the impact of degraders on the growth of KM12 cells using GNF-8625 and the <i>N</i>-methyl glutarimide analogues <b>25</b> and <b>26</b> as controls (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>). Interestingly, although compound <b>5</b> displayed a threefold weaker binding affinity to TRKA than GNF-8625, compound <b>5</b> (IC<sub>50</sub> = 5.4 ± 0.4 nM) showed threefold higher potency to inhibit the growth of KM12 cells than GNF-8625 (IC<sub>50</sub> = 16 ± 1.7 nM) and a tenfold higher potency than its control compound <b>25</b> (IC<sub>50</sub> = 52 ± 5.2 nM). Similarly, compound <b>6</b> (IC<sub>50</sub> = 2.9 ± 0.3 nM), an equally potent TRKA binder as GNF-8625, was fivefold more potent at the inhibition of KM12 cell growth than GNF-8625 and 10-fold more potent than its control compound <b>26</b> (IC<sub>50</sub> = 28 ± 2.9 nM). These results suggest that event-driven degraders could exhibit superior cellular activities compared with the occupancy-driven small-molecule kinase inhibitors, despite weaker binding affinities.</div><figure id="fig7" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01342/20201203/images/medium/jm0c01342_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01342/20201203/images/large/jm0c01342_0008.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Degraders <b>5</b> and <b>6</b> are more potent than GNF-8625, <b>25</b>, and <b>26</b> to inhibit KM12 cell viability. KM12 cells were seeded in 96-well plates and treated with test compounds following 11-point serial dilution for 3 days. Cell viability was determined using the CellTiter-Lumi assay. Error bars represent ± SD in three independent experiments.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01342/20201203/images/large/jm0c01342_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01342&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i12" class="anchor-spacer"></div><h3 class="article-section__title" id="_i12"> Compounds <b>5</b> and <b>6</b> are Bioavailable in Mice</h3><div class="NLM_p">We next evaluated the pharmacokinetic (PK) properties of compounds <b>5</b> and <b>6</b> in ICR mice (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>, <a href="/doi/suppl/10.1021/acs.jmedchem.0c01342/suppl_file/jm0c01342_si_001.pdf" class="ext-link">Table S2A–C</a>). The plasma concentrations of compounds <b>5</b> and <b>6</b> were monitored over 8 h after a single intraperitoneal (IP) administration at 10 mg/kg. Compounds <b>5</b> and <b>6</b> shared similar PK profiles. Both compounds reached maximum plasma concentrations (<i>C</i><sub>max</sub> = 2050 ± 120 nM for <b>5</b> and 1640 ± 120 nM for <b>6</b>) at 1 h post dosing. Although the half-lives of both compounds were not optimal (<i>T</i><sub>1/2</sub> = 0.9 h), compound plasma concentrations (14.8 ± 3.3 nM for <b>5</b> and 12.4 ± 13.6 nM for <b>6</b>) at the last time point (8 h) were still well above their DC<sub>50</sub> values (DC<sub>50</sub> = 0.48 ± 0.06 nM for <b>5</b> and 0.36 ± 0.06 nM for <b>6</b>) in KM12 cells. In addition, no clinical signs were observed during the PK study, and both compounds were well-tolerated by the treated mice. These results indicated that both <b>5</b> and <b>6</b> could be valuable tool compounds for <i>in vivo</i> study.</div><figure id="fig8" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01342/20201203/images/medium/jm0c01342_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01342/20201203/images/large/jm0c01342_0009.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Plasma concentrations of compounds <b>5</b> and <b>6</b> following a single 10 mg/kg IP injection in male ICR mice. Plasma concentrations reported at each of the eight time points are the average values from four test animals. Error bars represent + SD.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01342/20201203/images/large/jm0c01342_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01342&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i13">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_01231" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_01231" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Based on the X-ray cocrystal structure of TRKA in complex with an analogue of GNF-8625, we have designed and synthesized a series of CRBN-recruiting putative protein degrader molecules with different types of linkers. Initially, we were concerned about the geometry of the loop between GLY605 and GLY623 residues on TRKA. The data collected here reveal that the loop did not undermine interaction with TRK degraders. Indeed, most of the designed degraders effectively reduced TPM3-TRKA fusion protein levels at low nanomolar concentrations in KM12 cells. We selected two structurally similar degraders <b>5</b> and <b>6</b> for detailed biological investigation. Although the overall degradation profiles of these two degraders were very similar, the weaker TRKB binder compound <b>5</b> turned out to be a better AGBL4-TRKB degrader than the stronger TRKB binder <b>6</b>.</div><div class="NLM_p last">The common features of these two key TRK degraders are summarized as follows: both <b>5</b> and <b>6</b> exhibited high binding affinities to TRK family members and degraded TPM3-TRKA fusion protein levels in KM12 cells in a concentration- and time-dependent manner with sub-nanomolar DC<sub>50</sub> values. Both <b>5</b> and <b>6</b> also exhibited single-digit nanomolar potencies for the degradation of wild-type TRKA in HEL cells. Interestingly, both degraders, especially <b>6</b>, showed excellent selectivity for endogenous TPM3-TRKA over ectopically expressed AGBL4-TRKB and ETV6-TRKC fusion proteins in KM12 cells. Global proteomic studies further revealed that <b>5</b> was highly selective for TRKA. Next, we confirmed that <b>5</b> and <b>6</b> induced the degradation of TPM3-TRKA through the UPS and CRL4<sup>CRBN</sup>. Moreover, both <b>5</b> and <b>6</b> showed better antiproliferation activity than their parental kinase inhibitor GNF-8625 in KM12 cells, indicating possible advantages of TRK degraders over TRK kinase inhibitors. In mouse PK studies, both <b>5</b> and <b>6</b> showed good plasma exposure levels <i>via</i> IP injection, suggesting potential use as <i>in vivo</i> tool compounds. In summary, <b>5</b> and <b>6</b> are highly potent and selective TRK degraders. Further optimization is ongoing to improve their drug properties.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i14">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_50814" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_50814" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i15" class="anchor-spacer"></div><h3 class="article-section__title" id="_i15"> Chemistry General Procedure</h3><div class="NLM_p last">All chemicals and reagents were purchased from commercial suppliers and used without further purification. LCMS spectra for all compounds were acquired using a Waters LC–MS AcQuity H UPLC class system. The Waters LC–MS AcQuity H UPLC class system comprised a pump (Quaternary Solvent Manager) with a degasser, an autosampler (FTN), a column oven (40 °C, unless otherwise indicated), and a photodiode array PDA detector. Chromatography was performed on an AcQuity UPLC BEH C18 (1.7 μm, 2.1 × 50 mm) with water containing 0.1% formic acid as solvent A and acetonitrile containing 0.1% formic acid as solvent B at a flow rate of 0.6 mL/min. Flow from the column was split to a mass spectrometry (MS) spectrometer. The MS detector was configured with an electrospray ionization source. Nitrogen was used as the nebulizer gas. Data acquisition was performed with a MassLynx data system. Nuclear magnetic resonance spectra were recorded on a Bruker Avance III400 spectrometer. Chemical shifts are expressed in parts per million (ppm) and reported as the δ value (chemical shift δ). Coupling constants are reported in units of hertz (<i>J</i> value, Hz; integration and splitting patterns: where s = singlet, d = double, t = triplet, q = quartet, brs = broad singlet, and m = multiple). The purification of intermediates or final products was performed on Agilent Prep 1260 series with a UV detector set to 254 or 220 nm. Samples were injected onto a Phenomenex Luna C18 column (5 μm, 30 × 75 mm) at room temperature. The flow rate was 40 mL/min. A linear gradient was used with either 10 or 50% MeOH in H<sub>2</sub>O containing 0.1% TFA as solvent A and 100% of MeOH as solvent B. Alternatively, the products were purified on a CombiFlash NextGen 300 system with a UV detector set to 254, 220, or 280 nm. The flow rate was 40 mL/min. A linear gradient was used with H<sub>2</sub>O containing 0.05% TFA as solvent A and 100% of MeOH containing 0.05% TFA as solvent B. All compounds showed >95% purity using the LCMS methods described above.</div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i16" class="anchor-spacer"></div><h3 class="article-section__title" id="_i16"> (<i>R</i>)-3-Bromo-6-(2-(3-fluorophenyl)pyrrolidin-1-yl)imidazo[1,2-<i>b</i>]pyridazine (<b>10</b>)</h3><div class="NLM_p last">To a solution of 3-bromo-6-chloroimidazo[1,2-<i>b</i>]pyridazine (4.6 g, 20.0 mmol) in DMSO (40 mL) were added potassium fluoride (20 g, 362 mmol) and (<i>R</i>)-2-(3-fluorophenyl)pyrrolidine (3 g, 18.2 mmol). The resulting mixture was stirred at 100 °C for 12 h, before the mixture was diluted with EtOAc, and washed with brine. The organic layer was dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated. The resulting residue was passed through a short silica gel column (100% EtOAc) to give the title compound (1.8 g, crude yield: 28%) as a yellow solid, which was used in the next step without further purification.</div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17"> (<i>R</i>)-6-(2-(3-Fluorophenyl)pyrrolidin-1-yl)-3-(6-fluoropyridin-2-yl)imidazo[1,2-<i>b</i>]pyridazine (<b>12</b>)</h3><div class="NLM_p last">To a solution of (<i>R</i>)-3-bromo-6-(2-(3-fluorophenyl)pyrrolidin-1-yl)imidazo[1,2-<i>b</i>]pyridazine <b>10</b> (2.17 g, 6.03 mmol) in toluene (50 mL) were added 2-fluoro-6-(tributylstannyl)pyridine <b>11</b> (3.5 g, 9.04 mmol) and tetrakis(triphenylphosphine)palladium (566 mg, 0.49 mmol). The resulting mixture was stirred at 110 °C for 12 h under a nitrogen atmosphere before being poured into a mixture of EtOAc and aqueous solution of 1 M potassium fluoride. After stirring at rt for 2 h, the mixture was extracted with EtOAc (3×). The combined organic layers were dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated. The resulting residue was purified by silica gel column chromatography (50–100% EtOAc in hexanes) to provide the title compound (2.2 g, yield: 97%) as yellow oil. MS (ESI) <i>m</i>/<i>z</i>: 378.0 [M + 1]<sup>+</sup>. <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD): δ 8.09 (s, 1H), 8.01 (br s, 1H), 7.74–7.79 (m, 2H), 7.35–7.41 (m, 1H), 7.15 (d, <i>J</i> = 7.6 Hz, 1H), 7.07 (d, <i>J</i> = 10.0 Hz, 1H), 6.86–6.99 (m, 3H), 5.16 (dd, <i>J</i> = 8.0 Hz, 2.4 Hz, 1H), 3.98–4.03 (m, 1H), 3.71–3.77 (m, 1H), 2.55–2.60 (m, 1H), 2.12–2.17 (m, 2H), 1.99–2.04 (m, 1H).</div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18"> (<i>R</i>)-6-(2-(3-Fluorophenyl)pyrrolidin-1-yl)-3-(6-(piperazin-1-yl)pyridin-2-yl)imidazo[1,2-<i>b</i>]pyridazine (<b>7</b>)</h3><div class="NLM_p last">To a solution of (<i>R</i>)-6-(2-(3-fluorophenyl)pyrrolidin-1-yl)-3-(6-fluoropyridin-2-yl)imidazo [1,2-<i>b</i>]pyridazine (1.4 g, 3.7 mmol) <b>12</b> in DMSO (40 mL) was added piperazine (6.4 g, 74 mmol), followed by potassium fluoride (8.6 g, 148 mmol). The resulting mixture was stirred at 130 °C for 12 h before being poured into water and extracted with EtOAc. The combined organic layers were washed with brine, dried over Na<sub>2</sub>SO<sub>4</sub>, and concentrated. The resulting residue was purified by silica gel column chromatography (10–20% MeOH in DCM) to give the desired product as yellow oil. The oil was dissolved in a HCl solution in EtOAc (4 M) and stirred for 1 h, at which time the reaction mixture was concentrated to give the title compound as HCl salt (1.17 g, yield: 66%) as a yellow solid. MS (ESI) <i>m</i>/<i>z</i>: 444.2 [M + 1]<sup>+</sup>. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.62 (s, 2H), 8.63 (s, 1H), 8.21 (s, 1H), 7.62–7.19 (m, 6H), 7.06–7.01 (m, 2H), 5.26–5.25 (m, 1H), 4.07–4.02 (m, 1H), 3.86–3.85 (m, 4H), 3.74–3.72 (m, 1H), 3.16–3.15 (m, 4H), 2.08–2.07 (m, 2H), 1.92–1.91 (m, 2H).</div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i19" class="anchor-spacer"></div><h3 class="article-section__title" id="_i19"> 6-((2-(2,6-Dioxopiperidin-3-yl)-1,3-dioxoisoindolin-4-yl)amino)hexanoic Acid (<b>15a</b>)</h3><div class="NLM_p last">A solution of 2-(2,6-dioxopiperidin-3-yl)-4-fluoroisoindoline-1,3-dione <b>13</b>(1.4 g, 5.0 mmol), <i>tert</i>-butyl 6-aminohexanoate <b>14a</b> (1.1 g, 5.9 mmol) and <i>N</i>,<i>N</i>-diisopropylethylamine (DIEA, 1.94 g, 15 mmol) in <i>N</i>-methyl-2-pyrrolidone (NMP, 12 mL) was heated at 85 °C in a microwave reactor for 50 min. Four batches of the reaction mixture were combined and diluted with EtOAc (200 mL). The mixture was washed with water and brine, dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated. The resulting residue was purified by silica gel chromatography (30% EtOAc in hexanes) to give <i>tert</i>-butyl 6-((2-(2,6-dioxopiperidin-3-yl)-1,3-dioxoisoindolin-4-yl)amino)hexanoate (1.8 g, yield: 20%) as a yellow solid. A solution of this intermediate (1.8 g, 4.06 mmol) in formic acid (15 mL) was stirred at rt for 30 h. The reaction was concentrated and triturated with DCM. After the resulting mixture was filtered, the solid was washed with DCM and MTBE and dried to give the title compound (1.43 g, yield: 91%) as a yellow solid. MS (ESI) <i>m</i>/<i>z</i>: 388.1 [M + 1]<sup>+</sup>. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 11.97 (s, 1H), 11.08 (s, 1H), 7.57 (dd, <i>J</i> = 7.2, 8.8 Hz, 1H), 7.08 (d, <i>J</i> = 8.8 Hz, 1H), 7.02 (d, <i>J</i> = 7.2 Hz, 1H), 6.52 (t, <i>J</i> = 6.0 Hz, 1H), 5.05 (dd, <i>J</i> = 5.6, 12.8 Hz, 1H), 3.30 (q, <i>J</i> = 6.8 Hz, 2H), 2.93–2.83 (m, 1H), 2.61–2.50 (m, 2H), 2.32 (t, <i>J</i> = 7.2 Hz, 2H), 2.07–2.00 (m, 1H), 1.61–1.50 (m, 4H), 1.39–1.33 (m, 2H).</div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i20" class="anchor-spacer"></div><h3 class="article-section__title" id="_i20"> 3-(2-((2-(2,6-Dioxopiperidin-3-yl)-1,3-dioxoisoindolin-4-yl)amino)ethoxy)propanoic Acid (<b>15b</b>)</h3><div class="NLM_p last">The title compound (18% yield over two steps) was synthesized according to procedures for the preparation of <b>15a</b>. MS (ESI) <i>m</i>/<i>z</i>: 390.1 [M + 1]<sup>+</sup>. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 12.18 (s, 1H), 11.08 (s, 1H), 7.58 (dd, <i>J</i> = 7.2 Hz, 8.8 Hz, 1H), 7.13 (d, <i>J</i> = 8.4 Hz, 1H), 7.04 (d, <i>J</i> = 7.2 Hz, 1H), 6.58 (t, <i>J</i> = 5.6 Hz 1H), 5.05 (dd, <i>J</i> = 6.4 Hz, 12.8 Hz, 1H), 3.67–3.58 (m, 4H), 3.47–3.43 (m, 2H), 2.93–2.84 (m, 1H), 2.61–2.45 (m, 4H), 2.07–2.01 (m, 1H).</div></div><div id="sec4_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i21" class="anchor-spacer"></div><h3 class="article-section__title" id="_i21"> 2-(2,6-Dioxopiperidin-3-yl)-4-((6-(4-(6-(6-((<i>R</i>)-2-(3-fluorophenyl)pyrrolidin-1-yl)imidazo[1,2-<i>b</i>]pyridazin-3-yl)pyridin-2-yl)piperazin-1-yl)-6-oxohexyl)amino)isoindoline-1,3-dione (<b>5</b>)</h3><div class="NLM_p last">A mixture of (<i>R</i>)-6-(2-(3-fluorophenyl)pyrrolidin-1-yl)-3-(6-(piperazin-1-yl)pyridin-2-yl)imidazo[1,2-<i>b</i>]pyridazine <b>7</b> (50.0 mg, 0.113 mmol), 6-((2-(2,6-dioxopiperidin-3-yl)-1,3-dioxoisoindolin-4-yl)amino)hexanoic acid <b>15a</b> (43.6 mg, 0.113 mmol), 1-hydroxy-7-aza-benzo-triazole (HOAt, 45.7 mg, 0.339 mmol), and 1-ethyl-3-(3- dimethylaminopropyl)carbodiimide (EDCI, 65.1 mg, 0.339 mmol) in DMSO (2 mL) was added to <i>N</i>-methylmorpholine (0.5 mL). After the reaction was stirred at rt for 12 h, it was diluted with EtOAc (20 mL) and brine (30 mL). The organic layer was washed with brine, dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated. The resulting residue was purified by preparative TLC (10% MeOH in DCM) to give the title compound (60 mg, 65% yield) as a yellow solid. MS (ESI) <i>m</i>/<i>z</i>: 813.7 [M + 1]<sup>+</sup>. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 11.03 (s, 1H), 8.40 (s, 1H), 8.06 (d, <i>J</i> = 9.6 Hz, 1H), 7.49 (t, <i>J</i> = 8.0 Hz, 2H), 7.30–7.36 (m, 1H), 7.10–7.21 (m, 3H), 6.97–7.03 (m, 2H), 6.93 (d, <i>J</i> = 6.8 Hz, 1H), 6.81 (d, <i>J</i> = 8.8 Hz, 1H), 6.47 (br s, 1H), 5.15 (d, <i>J</i> = 6.8 Hz, 1H), 4.95–5.00 (m, 1H), 3.95–3.99 (m, 1H), 3.61–3.68 (m, 1H), 3.50 (br s, 7H), 3.23 (br s, 2H), 2.76–2.82 (m, 1H), 2.40–2.53 (m, 5H), 2.31 (t, <i>J</i> = 7.2 Hz, 2H), 1.92–2.22 (m, 3H), 1.79–1.88 (m, 1H), 1.47–1.57 (m, 4H), 1.28–1.35 (m, 2H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): δ 173.3, 171.2, 170.6, 169.4, 164.1, 161.7, 158.4, 153.4, 146.9, 146.7, 146.6, 144.0, 138.7, 136.7, 132.7, 131.0, 128.5, 124.8, 123.1, 122.2, 117.6, 115.5, 114.2, 114.0, 113.1, 112.9, 110.8, 109.5, 108.0, 62.0, 49.0, 45.1, 45.0, 44.7, 42.2, 41.1, 35.6, 32.7, 31.5, 29.0, 26.6, 25.0, 23.1, 26.6.</div></div><div id="sec4_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i22" class="anchor-spacer"></div><h3 class="article-section__title" id="_i22"> 2-(2,6-Dioxopiperidin-3-yl)-4-((2-(3-(4-(6-(6-((<i>R</i>)-2-(3-fluorophenyl)pyrrolidin-1-yl)imidazo[1,2-<i>b</i>]pyridazin-3-yl)pyridin-2-yl)piperazin-1-yl)-3-oxopropoxy)ethyl)amino)isoindoline-1,3-dione (<b>6</b>)</h3><div class="NLM_p last">The title compound (83% yield) was synthesized according to procedures for the preparation of <b>5</b>. MS (ESI) <i>m</i>/<i>z</i>: 815.8 [M + 1]<sup>+</sup>. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 11.04 (s, 1H), 8.38 (s, 1H), 8.06 (d, <i>J</i> = 10.0 Hz, 1H), 7.45–7.49 (m, 2H), 7.31–7.36 (m, 1H), 7.10–7.24 (m, 3H), 6.97–7.05 (m, 2H), 6.91–6.93 (m, 1H), 6.76 (d, <i>J</i> = 8.0 Hz, 1H), 6.50 (br s, 1H), 5.15 (d, <i>J</i> = 6.8 Hz, 1H), 4.94–4.99 (m, 1H), 3.95–4.02 (m, 1H), 3.65 (t, <i>J</i> = 6.4 Hz, 2H), 3.50–3.56 (m, 10H), 3.39 (br s, 2H), 2.71–2.83 (m, 1H), 2.58 (t, <i>J</i> = 6.8 Hz, 2H), 2.38–2.50 (m, 5H), 1.91–2.02 (m, 3H), 1.79–1.88 (m, 1H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): δ 173.2, 170.5, 169.4, 167.7, 164.1, 161.7, 158.4, 153.4, 146.8, 146.7, 146.6, 143.9, 138.6, 136.7, 134.1, 132.5, 131.0, 128.5, 123.0, 122.2, 117.8, 116.8, 115.7, 114.2, 113.1, 112.9, 111.1, 110.9, 109.7, 108.0, 69.2, 67.2, 62.0, 49.0, 45.0, 44.6, 42.2, 41.2, 35.6, 33.3, 31.4, 23.1, 22.6.</div></div><div id="sec4_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i23" class="anchor-spacer"></div><h3 class="article-section__title" id="_i23"> 4-((6-(4-(6-(6-((<i>R</i>)-2-(3-Fluorophenyl)pyrrolidin-1-yl)imidazo[1,2-<i>b</i>]pyridazin-3-yl)pyridin-2-yl)piperazin-1-yl)-6-oxohexyl)amino)-2-(1-methyl-2,6-dioxopiperidin-3-yl)isoindoline-1,3-dione (<b>25</b>)</h3><div class="NLM_p last">To a solution of 2-(2,6-dioxopiperidin-3-yl)-4-((6-(4-(6-(6-((<i>R</i>)-2-(3-fluorophenyl)pyrrolidin-1-yl)imidazo[1,2-<i>b</i>]pyridazin-3-yl)pyridin-2-yl)piperazin-1-yl)-6-oxohexyl)amino)isoindoline-1,3-dione <b>5</b> (25 mg, 0.03 mmol) in DMSO were added K<sub>2</sub>CO<sub>3</sub> (12.24 mg, 0.09 mmol) and MeI (5.07 mg, 0.036 mmol). The reaction mixture was stirred at rt for 30 min, before it was diluted with EtOAc (10 mL) and brine (20 mL). The organic layer was washed with brine, dried over Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated. The resulting residue was purified by Prep-TLC (5% MeOH in DCM) to give the title compound (24 mg, 97% yield) as a yellow solid. MS (ESI) <i>m</i>/<i>z</i>: 827.8 [M + 1]<sup>+</sup>. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.02 (s, 1H), 7.84 (d, <i>J</i> = 9.2 Hz, 1H), 7.49 (t, <i>J</i> = 8.0 Hz, 1H), 7.41 (br s, 1H), 7.27–7.33 (m, 1H), 6.92–7.09 (m, 5H), 6.77 (br s, 1H), 6.68 (d, <i>J</i> = 8.4 Hz, 1H), 6.47 (t, <i>J</i> = 6.0 Hz, 1H), 5.02–5.08 (m, 2H), 3.88–3.93 (m, 1H), 3.55–3.61 (m, 1H), 3.50 (br s, 5H), 3.44 (br s, 2H), 3.19–3.24 (m, 2H), 2.94 (s, 3H), 2.82–2.92 (m, 1H), 2.65–2.70 (m, 1H), 2.35–2.51 (m, 4H), 2.30 (t, <i>J</i> = 11.2 Hz, 2H), 1.91–2.01 (m, 3H), 1.77–1.81 (m, 1H), 1.46–1.56 (m, 4H), 1.26–1.34 (m, 2H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): δ 172.3, 171.2, 170.3, 169.4, 167.8, 164.1, 161.7, 158.4, 152.3, 147.7, 146.9, 145.9, 138.5, 137.8, 136.8, 132.6, 130.9, 127.7, 126.3, 122.1, 117.7, 114.0, 113.8, 113.0, 112.8, 110.9, 109.9, 109.4, 106.5, 61.8, 49.6, 48.9, 45.2, 44.9, 42.2, 41.2, 35.7, 32.7, 31.6, 29.0, 27.1, 26.6, 25.0, 23.2, 21.8.</div></div><div id="sec4_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i24" class="anchor-spacer"></div><h3 class="article-section__title" id="_i24"> 4-((2-(3-(4-(6-(6-((<i>R</i>)-2-(3-Fluorophenyl)pyrrolidin-1-yl)imidazo[1,2-<i>b</i>]pyridazin-3-yl)pyridin-2-yl)piperazin-1-yl)-3-oxopropoxy)ethyl)amino)-2-(1-methyl-2,6-dioxopiperidin-3-yl)isoindoline-1,3-dione (<b>26</b>)</h3><div class="NLM_p last">The title compound (74% yield) was synthesized according to procedures for the preparation of <b>25</b>. MS (ESI) <i>m</i>/<i>z</i>: 829.7 [M + 1]<sup>+</sup>. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.45 (br s, 1H), 8.12 (br s, 1H), 7.54 (t, <i>J</i> = 8.0 Hz, 2H), 7.37–7.43 (m, 1H), 7.17–7.31 (m, 3H), 7.11 (d, <i>J</i> = 8.4 Hz, 1H), 7.06 (t, <i>J</i> = 8.0 Hz, 1H), 6.99 (d, <i>J</i> = 6.8 Hz, 1H), 6.83 (d, <i>J</i> = 8.4 Hz, 1H), 6.57 (br s, 1H), 5.21 (d, <i>J</i> = 6.8 Hz, 1H), 5.07–5.12 (m, 1H), 4.02–4.07 (m, 1H), 3.72 (t, <i>J</i> = 6.4 Hz, 3H), 3.57–3.64 (m, 9H), 3.46 (br s, 2H), 2.99 (s, 3H), 2.87–2.96 (m, 1H), 2.63–2.74 (m, 3H), 2.45–2.56 (m, 4H), 2.00–2.09 (m, 3H), 1.86–1.96 (m, 1H). <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>): δ 172.2, 170.3, 169.4, 167.7, 164.1, 161.7, 158.4, 153.4, 146.9, 146.8, 146.6, 144.0, 138.6, 136.8, 132.5, 131.0, 130.9, 123.1, 122.2, 117.9, 115.6, 114.2, 114.0, 113.1, 112.9, 111.1, 110.9, 109.7, 107.9, 69.1, 67.2, 62.0, 49.6, 49.1, 45.0, 44.6, 42.2, 41.1, 35.6, 33.3, 31.6, 27.0, 23.0, 21.8.</div></div><div id="sec4_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i25" class="anchor-spacer"></div><h3 class="article-section__title" id="_i25"> 2-(2,6-Dioxopiperidin-3-yl)-4-((2-(4-(6-(6-((<i>R</i>)-2-(3-fluorophenyl)pyrrolidin-1-yl)imidazo[1,2-<i>b</i>]pyridazin-3-yl)pyridin-2-yl)piperazin-1-yl)-2-oxoethyl)amino)isoindoline-1,3-dione (<b>16</b>)</h3><div class="NLM_p last">The title compound (59% yield) was synthesized according to procedures for the preparation of <b>5</b>. MS (ESI) <i>m</i>/<i>z</i>: 757.6 [M + 1]<sup>+</sup>. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 11.06 (s, 1H), 8.38 (s, 1H), 8.04 (d, <i>J</i> = 8.8 Hz, 1H), 7.45–7.56 (m, 2H), 7.30–7.36 (m, 1H), 6.96–7.11 (m, 6H), 6.84 (d, <i>J</i> = 8.8 Hz, 1H), 5.13 (d, <i>J</i> = 7.2 Hz, 1H), 4.99–5.04 (m, 1H), 4.18 (s, 2H), 3.94–3.97 (m, 1H), 3.46–3.67 (m, 8H), 2.79–2.88 (m, 1H), 2.38–2.56 (m, 5H), 1.93–2.01 (m, 3H), 1.79–1.85 (m, 1H).</div></div><div id="sec4_12" class="NLM_sec NLM_sec_level_2"><div id="ac_i26" class="anchor-spacer"></div><h3 class="article-section__title" id="_i26"> 2-(2,6-Dioxopiperidin-3-yl)-4-((3-(4-(6-(6-((<i>R</i>)-2-(3-fluorophenyl)pyrrolidin-1-yl)imidazo[1,2-<i>b</i>]pyridazin-3-yl)pyridin-2-yl)piperazin-1-yl)-3-oxopropyl)amino)isoindoline-1,3-dione (<b>17</b>)</h3><div class="NLM_p last">The title compound (53% yield) was synthesized according to procedures for the preparation of <b>5</b>. MS (ESI) <i>m</i>/<i>z</i>: 771.6 [M + 1]<sup>+</sup>. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 11.03 (s, 1H), 8.36 (s, 1H), 8.03 (d, <i>J</i> = 9.6 Hz, 1H), 7.47–7.54 (m, 2H), 7.30–7.35 (m, 1H), 7.09–7.15 (m, 4H), 6.95–7.00 (m, 2H), 6.80 (d, <i>J</i> = 8.4 Hz, 1H), 6.72 (s, 1H), 5.14 (d, <i>J</i> = 7.2 Hz, 1H), 4.95–4.99 (m, 1H), 3.94–3.99 (m, 1H), 3.60–3.66 (m, 1H), 3.51 (br s, 9H), 2.75–2.84 (m, 1H), 2.66 (t, <i>J</i> = 6.0 Hz, 2H), 2.37–2.52 (m, 5H), 1.91–2.02 (m, 3H), 1.80–1.85 (m, 1H).</div></div><div id="sec4_13" class="NLM_sec NLM_sec_level_2"><div id="ac_i27" class="anchor-spacer"></div><h3 class="article-section__title" id="_i27"> 2-(2,6-Dioxopiperidin-3-yl)-4-((4-(4-(6-(6-((<i>R</i>)-2-(3-fluorophenyl)pyrrolidin-1-yl)imidazo[1,2-<i>b</i>]pyridazin-3-yl)pyridin-2-yl)piperazin-1-yl)-4-oxobutyl)amino)isoindoline-1,3-dione (<b>18</b>)</h3><div class="NLM_p last">The title compound (54% yield) was synthesized according to procedures for the preparation of <b>5</b>. MS (ESI) <i>m</i>/<i>z</i>: 785.6 [M + 1]<sup>+</sup>. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 11.03 (s, 1H), 8.40 (s, 1H), 8.06 (d, <i>J</i> = 10.0 Hz, 1H), 7.42–7.53 (m, 2H), 7.30–7.36 (m, 1H), 7.10–7.19 (m, 4H), 6.99 (t, <i>J</i> = 8.4 Hz, 1H), 6.94 (d, <i>J</i> = 6.8 Hz, 1H), 6.82 (d, <i>J</i> = 8.0 Hz, 1H), 6.61 (s, 1H), 5.15 (d, <i>J</i> = 7.6 Hz, 1H), 4.96–5.01 (m, 1H), 3.96–4.01 (m, 1H), 3.61–3.67 (m, 1H), 3.50–3.52 (m, 7H), 3.28 (s, 2H), 2.77–2.86 (m, 1H), 2.38–2.53 (m, 7H), 1.94–2.02 (m, 3H), 1.73–1.85 (m, 3H).</div></div><div id="sec4_14" class="NLM_sec NLM_sec_level_2"><div id="ac_i28" class="anchor-spacer"></div><h3 class="article-section__title" id="_i28"> 2-(2,6-Dioxopiperidin-3-yl)-4-((5-(4-(6-(6-((<i>R</i>)-2-(3-fluorophenyl)pyrrolidin-1-yl)imidazo[1,2-<i>b</i>]pyridazin-3-yl)pyridin-2-yl)piperazin-1-yl)-5-oxopentyl)amino)isoindoline-1,3-dione (<b>19</b>)</h3><div class="NLM_p last">The title compound (53% yield) was synthesized according to procedures for the preparation of <b>5</b>. MS (ESI) <i>m</i>/<i>z</i>: 799.7 [M + 1]<sup>+</sup>. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 11.10 (s, 1H), 8.48 (s, 1H), 8.13 (d, <i>J</i> = 8.8 Hz, 1H), 7.49–7.60 (m, 2H), 7.38–7.43 (m, 1H), 7.17–7.29 (m, 3H), 7.00–7.12 (m, 3H), 6.89 (d, <i>J</i> = 8.4 Hz, 1H), 6.58 (s, 1H), 5.22 (d, <i>J</i> = 7.2 Hz, 1H), 5.03–5.07 (m, 1H), 4.01–4.08 (m, 1H), 3.68–3.75 (m, 1H), 3.58 (s, 7H), 3.33 (s, 2H), 2.83–2.91 (m, 1H), 2.43–2.60 (m, 7H), 2.00–2.10 (m, 3H), 1.88–1.96 (m, 1H), 1.62 (s, 4H).</div></div><div id="sec4_15" class="NLM_sec NLM_sec_level_2"><div id="ac_i29" class="anchor-spacer"></div><h3 class="article-section__title" id="_i29"> 2-(2,6-Dioxopiperidin-3-yl)-4-((7-(4-(6-(6-((<i>R</i>)-2-(3-fluorophenyl)pyrrolidin-1-yl)imidazo[1,2-<i>b</i>]pyridazin-3-yl)pyridin-2-yl)piperazin-1-yl)-7-oxoheptyl)amino)isoindoline-1,3-dione (<b>20</b>)</h3><div class="NLM_p last">The title compound (53% yield) was synthesized according to procedures for the preparation of <b>5</b>. MS (ESI) <i>m</i>/<i>z</i>: 827.7 [M + 1]<sup>+</sup>. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 11.00 (s, 1H), 8.37 (s, 1H), 8.04 (d, <i>J</i> = 9.6 Hz, 1H), 7.48 (d, <i>J</i> = 8.4 Hz, 2H), 7.30–7.36 (m, 1H), 7.10–7.20 (m, 4H), 6.98 (t, <i>J</i> = 8.0 Hz, 1H), 6.87 (d, <i>J</i> = 1.2 Hz, 1H), 6.76–6.82 (m, 2H), 5.15 (d, <i>J</i> = 7.2 Hz, 1H), 4.94–4.98 (m, 1H), 3.95–3.98 (m, 1H), 3.61–3.67 (m, 1H), 3.50 (s, 7H), 3.08 (t, <i>J</i> = 6.8 Hz, 2H), 2.77–2.81 (m, 1H), 2.40–2.52 (m, 5H), 2.30 (t, <i>J</i> = 7.6 Hz, 2H), 1.82–2.22 (m, 4H), 1.44–1.52 (m, 4H), 1.24–1.36 (m, 4H).</div></div><div id="sec4_16" class="NLM_sec NLM_sec_level_2"><div id="ac_i30" class="anchor-spacer"></div><h3 class="article-section__title" id="_i30"> 2-(2,6-Dioxopiperidin-3-yl)-4-((8-(4-(6-(6-((<i>R</i>)-2-(3-fluorophenyl)pyrrolidin-1-yl)imidazo[1,2-<i>b</i>]pyridazin-3-yl)pyridin-2-yl)piperazin-1-yl)-8-oxooctyl)amino)isoindoline-1,3-dione (<b>21</b>)</h3><div class="NLM_p last">The title compound (54% yield) was synthesized according to procedures for the preparation of <b>5</b>. MS (ESI) <i>m</i>/<i>z</i>: 841.7 [M + 1]<sup>+</sup>. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 11.10 (s, 1H), 8.46 (s, 1H), 8.13 (d, <i>J</i> = 10.4 Hz, 1H), 7.48–7.58 (m, 2H), 7.37–7.43 (m, 1H), 7.17–7.27 (m, 3H), 7.00–7.08 (m, 3H), 6.89 (d, <i>J</i> = 7.6 Hz, 1H), 6.52 (s, 1H), 5.22 (d, <i>J</i> = 7.2 Hz, 1H), 5.03–5.07 (m, 1H), 3.99–4.08 (m, 1H), 3.69–3.75 (m, 1H), 3.57 (s, 7H), 3.29 (s, 2H), 2.84–2.89 (m, 1H), 2.47–2.61 (m, 5H), 2.35 (t, <i>J</i> = 7.2 Hz, 2H), 2.01–2.08 (m, 3H), 1.90–1.92 (m, 1H), 1.52–1.57 (m, 4H), 1.33 (s, 6H).</div></div><div id="sec4_17" class="NLM_sec NLM_sec_level_2"><div id="ac_i31" class="anchor-spacer"></div><h3 class="article-section__title" id="_i31"> 2-(2,6-Dioxopiperidin-3-yl)-4-((2-(2-(3-(4-(6-(6-((<i>R</i>)-2-(3-fluorophenyl)pyrrolidin-1-yl)imidazo[1,2-<i>b</i>]pyridazin-3-yl)pyridin-2-yl)piperazin-1-yl)-3-oxopropoxy)ethoxy)ethyl)amino)isoindoline-1,3-dione (<b>22</b>)</h3><div class="NLM_p last">The title compound (72% yield) was synthesized according to procedures for the preparation of <b>5</b>. MS (ESI) <i>m</i>/<i>z</i>: 859.9 [M + 1]<sup>+</sup>. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 11.11 (s, 1H), 8.45 (s, 1H), 8.11 (d, <i>J</i> = 9.6 Hz, 1H), 7.50–7.54 (m, 2H), 7.38–7.43 (m, 1H), 7.17–7.27 (m, 3H), 7.04–7.08 (m, 2H), 6.98–7.00 (m, 1H), 6.86 (d, <i>J</i> = 8.8 Hz, 1H), 6.57 (s, 1H), 5.22 (d, <i>J</i> = 8.0 Hz, 1H), 5.03–5.08 (m, 1H), 4.02–4.06 (m, 1H), 3.64–3.72 (m, 3H), 3.54–3.61 (s, 13H), 3.42 (s, 2H), 2.85–2.89 (m, 1H), 2.46–2.62 (m, 7H), 2.00–2.10 (m, 3H), 1.90–1.93 (m, 1H).</div></div><div id="sec4_18" class="NLM_sec NLM_sec_level_2"><div id="ac_i32" class="anchor-spacer"></div><h3 class="article-section__title" id="_i32"> 2-(2,6-Dioxopiperidin-3-yl)-4-((2-(2-(2-(3-(4-(6-(6-((<i>R</i>)-2-(3-fluorophenyl)pyrrolidin-1-yl)imidazo[1,2-<i>b</i>]pyridazin-3-yl)pyridin-2-yl)piperazin-1-yl)-3-oxopropoxy)ethoxy)ethoxy)ethyl)amino)isoindoline-1,3-dione (<b>23</b>)</h3><div class="NLM_p last">The title compound (49% yield) was synthesized according to procedures for the preparation of <b>5</b>. MS (ESI) <i>m</i>/<i>z</i>: 903.7 [M + 1]<sup>+</sup>. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 11.11 (s, 1H), 8.46 (s, 1H), 8.12 (d, <i>J</i> = 10.0 Hz, 1H), 7.53 (t, <i>J</i> = 8.0 Hz, 2H), 7.37–7.43 (m, 1H), 7.17–7.27 (m, 3H), 7.00–7.09 (m, 3H), 6.88 (d, <i>J</i> = 7.6 Hz, 1H), 6.56 (s, 1H), 5.21 (d, <i>J</i> = 7.2 Hz, 1H), 5.03–5.08 (m, 1H), 3.99–4.09 (m, 1H), 3.50–3.72 (m, 19H), 3.42 (s, 2H), 2.84–2.93 (m, 1H), 2.43–2.63 (m, 8H), 2.01–2.09 (m, 3H), 1.89–1.92 (m, 1H).</div></div><div id="sec4_19" class="NLM_sec NLM_sec_level_2"><div id="ac_i33" class="anchor-spacer"></div><h3 class="article-section__title" id="_i33"> 2-(2,6-Dioxopiperidin-3-yl)-4-((15-(4-(6-(6-((<i>R</i>)-2-(3-fluorophenyl)pyrrolidin-1-yl)imidazo[1,2-<i>b</i>]pyridazin-3-yl)pyridin-2-yl)piperazin-1-yl)-15-oxo-3,6,9,12-tetraoxapentadecyl)amino)isoindoline-1,3-dione (<b>24</b>)</h3><div class="NLM_p last">The title compound (56% yield) was synthesized according to procedures for the preparation of <b>5</b>. MS (ESI) <i>m</i>/<i>z</i>: 947.6 [M + 1]<sup>+</sup>. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 11.16 (s, 1H), 8.58 (s, 1H), 8.21 (d, <i>J</i> = 9.6 Hz, 1H), 7.59 (t, <i>J</i> = 8.0 Hz, 2H), 7.42–7.48 (m, 1H), 7.22–7.35 (m, 3H), 7.05–7.15 (m, 3H), 6.95 (d, <i>J</i> = 8.4 Hz, 1H), 6.62 (s, 1H), 5.27 (d, <i>J</i> = 7.2 Hz, 1H), 5.08–5.13 (m, 1H), 4.07–4.11 (m, 1H), 3.48–3.79 (m, 26H), 2.89–2.98 (m, 1H), 2.67 (t, <i>J</i> = 6.4 Hz, 2H), 2.50–2.62 (m, 5H), 2.06–2.15 (m, 3H), 1.95–1.98 (m, 1H).</div></div><div id="sec4_20" class="NLM_sec NLM_sec_level_2"><div id="ac_i34" class="anchor-spacer"></div><h3 class="article-section__title" id="_i34"> Binding Affinity Assays</h3><div class="NLM_p last">Binding affinities were determined by DiscoverX (San Diego, USA) using the KINOMEscan platform. KINOMEscan is based on a competition binding assay that quantitatively measures the ability of a compound to compete with an immobilized, active site-directed ligand. The assay is performed by combining three components: DNA-tagged TRK, an immobilized ligand, and a test compound. The ability of the test compound to compete with the immobilized ligand is measured <i>via</i> quantitative PCR of the DNA tag. <i>K</i><sub>d</sub> values were determined using an 11-point threefold compound dilution series (the top concentration of 1 μM for degraders and 0.3 μM for GNF-8625) with three DMSO control points in duplicate. Some outlier data points were subtracted.</div></div><div id="sec4_21" class="NLM_sec NLM_sec_level_2"><div id="ac_i35" class="anchor-spacer"></div><h3 class="article-section__title" id="_i35"> Cell Culture and Transfection</h3><div class="NLM_p last">KM12 and HEL cells were cultured at 37 °C with 5% CO<sub>2</sub> in DMEM or RPMI 1640 medium supplemented with 10% fetal bovine serum (Gibco, Thermo Fisher). Cells were authenticated using the short tandem repeat (STR) assays by BioWing Applied Biotechnology, Shanghai. Mycoplasma contamination was excluded following a PCR-based method. Cell transfection was performed using Lipofectamine 2000 (Invitrogen) following the manufacturer’s instructions. Stable cell lines were established by lentivirus transduction, selected, and maintained in a medium containing 1 μg/mL puromycin (Beyotime Biotechnology).</div></div><div id="sec4_22" class="NLM_sec NLM_sec_level_2"><div id="ac_i36" class="anchor-spacer"></div><h3 class="article-section__title" id="_i36"> Plasmids and Reagents</h3><div class="NLM_p last">Human AGBL4-TRKB and ETV6-TRKC fusions were constructed into the pLVX-EF1a-mCherry-C1 lentiviral expression vector for stable transduction. Antibodies against TRK (pan) (92991S), PLCγ2 (3872S), p-PLCγ2 (Tyr759, 3874S), and CRBN (71810S) were purchased from Cell Signaling Technology. Antibodies against eRF3/GSPT1 (ab126090) and mCherry (ab167453) were purchase from Abcam. HRP-conjugated anti-α-Tubulin (GNI4310-AT) and anti-GAPDH (GNI4310-GH) antibodies were purchased from GNI. The CellTiter-Lumi assay kit was purchased from Beyotime Biotechnology, Beijing. The small interference RNAs (siRNAs) targeting human <i>CRBN</i> were purchased from GenePharma, Shanghai. The sequences of all siRNAs used in this study are as follow: siControl, UUCUCCGAACGUGUCACGUTT; siCRBN-1, CCAGCAAGCUAAAGUGCAA; siCRBN-2, GCGACUUCGCUGUGAAUUA.</div></div><div id="sec4_23" class="NLM_sec NLM_sec_level_2"><div id="ac_i37" class="anchor-spacer"></div><h3 class="article-section__title" id="_i37"> Immunoblotting</h3><div class="NLM_p last">Cultured cells were washed with cold PBS once and lysed in cold RIPA buffer supplemented with protease inhibitors and phosphatase inhibitors (Beyotime Biotechnology). The solutions were then incubated at 4 °C for 30 min with gentle agitation to fully lyse cells. Cell lysates were centrifuged at 13,000 rpm for 10 min at 4 °C, and pellets were discarded. Total protein concentrations in the lysates were determined following BCA assays (Beyotime Biotechnology). Cell lysates were mixed with Laemmli loading buffer to 1× and heated at 99 °C for 5 min. Proteins were resolved on SDS-PAGE and visualized using Western ECL Substrate kits on a ChemiDoc MP Imaging system (Bio-Rad). Protein bands were quantitated using Image Lab software provided by Bio-Rad.</div></div><div id="sec4_24" class="NLM_sec NLM_sec_level_2"><div id="ac_i38" class="anchor-spacer"></div><h3 class="article-section__title" id="_i38"> Cell Viability Assay</h3><div class="NLM_p last">Cells were seeded at a density of 5000 cells per well in 96-well assay plates and treated with test compounds following a 11-point serial dilution for 3 days. Cell viability was determined using the CellTiter-Lumi assay kit according to the manufacturer’s instructions.</div></div><div id="sec4_25" class="NLM_sec NLM_sec_level_2"><div id="ac_i39" class="anchor-spacer"></div><h3 class="article-section__title" id="_i39"> Statistical Analyses</h3><div class="NLM_p last">Data were analyzed using GraphPad Prism software. <i>K</i><sub>d</sub> (binding affinities), DC<sub>50</sub> (concentration that resulted in 50% of protein degradation), and IC<sub>50</sub> (concentration that led to 50% of maximal cell growth inhibition) were calculated using a nonlinear regression (least squares fit) method. Data presented were mean ± standard deviation (SD).</div></div><div id="sec4_26" class="NLM_sec NLM_sec_level_2"><div id="ac_i40" class="anchor-spacer"></div><h3 class="article-section__title" id="_i40"> TMT-Based Quantitative Proteomic Analyses</h3><div class="NLM_p">The selectivity of degraders was determined by MajorBio (Shanghai, China) using the TMT quantitative proteomic analysis. In brief, total protein was extracted from treated KM12 cells using a urea lysis buffer (8 M urea and 1% SDS) supplemented with protease inhibitors. Protein concentrations were determined following BCA assays. Protein digestion was performed according to the standard procedure, and the resulting peptide mixture was labeled using the 16-plex TMT reagent (Thermo fisher, A44522) according to the manufacturer’s instructions. Digested peptides were pooled, desalted, and vacuum-dried. Pooled peptides were fractionated into fractions by ACQUITY Ultra Performance liquid chromatography (Waters, USA) with an ACQUITY UPLC BEH C18 column (1.7 μm, 2.1 mm × 150 mm, Waters, USA) to increase proteomic depth.</div><div class="NLM_p last">The labeled peptide mixture was separated with a nano reverse-phase C18 column (75 μm × 25 cm, Thermo, USA) connected to EASY-nLC 1200 (Thermo, USA) and analyzed with a Q Exactive HF-X Hybrid Quadrupole-Orbitrap mass spectrometer (Thermo, USA). The RAW data files were analyzed using Proteome Discover software version 2.4 (Thermo Scientific) against the UniProt human protein sequence database (UP000005640). The MS/MS search criteria were as follows: mass tolerance of 20 ppm for MS and 0.02 Da for MS/MS, trypsin as the enzyme with two missed cleavage allowed, carbamido methylation of cysteine and the TMT of N- terminus and lysine side chains of peptides as fixed modification, and methionine oxidation as dynamic modifications. The false discovery rate (FDR) of peptide identification was set as FDR ≤0.01. A minimum of one unique peptide identification was used to support protein identification.</div></div><div id="sec4_27" class="NLM_sec NLM_sec_level_2"><div id="ac_i41" class="anchor-spacer"></div><h3 class="article-section__title" id="_i41"> Mouse PK Studies</h3><div class="NLM_p last">Standard PK studies were conducted using male ICR mice. Compounds <b>5</b> and <b>6</b> were dissolved in 5% NMP, 10% solutol, 30% PEG 400, and 55% saline as solution formulation. A single 10 mg/kg IP injection of degraders was evaluated. Plasma concentrations of degraders reported at each of the eight time points (5, 15, 30 min, 1, 2, 4, 6, and 8 h post dosing) are the mean values from four experimental animals. Error bars represent ± SD.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i42"><a href="/doi/suppl/10.1021/acs.jmedchem.0c01342" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_67614" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_67614" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a href="/doi/10.1021/acs.jmedchem.0c01342?goto=supporting-info" class="ext-link">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c01342</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline"><sup>1</sup>H NMR and <sup>13</sup>C NMR spectra, and UPLC-MS traces of compounds <b>5</b> and <b>6</b>; screening of the synthesized putative TRK degraders; immunoblotting analysis of samples for proteomic profiling; significantly differential proteins from proteomic profiling; and raw data  of compounds <b>5</b> and <b>6</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01342/suppl_file/jm0c01342_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Molecular formula strings and biological data (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01342/suppl_file/jm0c01342_si_002.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01342/suppl_file/jm0c01342_si_001.pdf">jm0c01342_si_001.pdf (1.02 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01342/suppl_file/jm0c01342_si_002.csv">jm0c01342_si_002.csv (2.66 kb)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.0c01342" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_12374" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_12374" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Xiao-Ran Han</span> - <span class="hlFld-Affiliation affiliation">Cullgen Inc., 12671 High Bluff Drive Suite 130, San Diego, California 92130, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Cullgen
(Shanghai), Inc., 230
ChuanHong Road, Building 6, Chuansha, Pudong New
Area, Shanghai 201202, People’s Republic of China</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#5d25343c322f3c3373353c331d3e2831313a3833733e3230"><span class="__cf_email__" data-cfemail="cab2a3aba5b8aba4e4a2aba48aa9bfa6a6adafa4e4a9a5a7">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jing Liu</span> - <span class="hlFld-Affiliation affiliation">Cullgen Inc., 12671 High Bluff Drive Suite 130, San Diego, California 92130, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-4740-243X" title="Orcid link">http://orcid.org/0000-0003-4740-243X</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#94fefdfaf3baf8fde1d4f7e1f8f8f3f1fabaf7fbf9"><span class="__cf_email__" data-cfemail="7f151611185113160a3f1c0a1313181a11511c1012">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Liqun Chen</span> - <span class="hlFld-Affiliation affiliation">Cullgen
(Shanghai), Inc., 230
ChuanHong Road, Building 6, Chuansha, Pudong New
Area, Shanghai 201202, People’s Republic of China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yanke Chen</span> - <span class="hlFld-Affiliation affiliation">Cullgen
(Shanghai), Inc., 230
ChuanHong Road, Building 6, Chuansha, Pudong New
Area, Shanghai 201202, People’s Republic of China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Chunyan Zhang</span> - <span class="hlFld-Affiliation affiliation">Cullgen
(Shanghai), Inc., 230
ChuanHong Road, Building 6, Chuansha, Pudong New
Area, Shanghai 201202, People’s Republic of China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Bingyang Jiao</span> - <span class="hlFld-Affiliation affiliation">Cullgen
(Shanghai), Inc., 230
ChuanHong Road, Building 6, Chuansha, Pudong New
Area, Shanghai 201202, People’s Republic of China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Sheng Liang</span> - <span class="hlFld-Affiliation affiliation">Cullgen
(Shanghai), Inc., 230
ChuanHong Road, Building 6, Chuansha, Pudong New
Area, Shanghai 201202, People’s Republic of China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Qiong Tan</span> - <span class="hlFld-Affiliation affiliation">Cullgen
(Shanghai), Inc., 230
ChuanHong Road, Building 6, Chuansha, Pudong New
Area, Shanghai 201202, People’s Republic of China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Hongyu Chai</span> - <span class="hlFld-Affiliation affiliation">Cullgen
(Shanghai), Inc., 230
ChuanHong Road, Building 6, Chuansha, Pudong New
Area, Shanghai 201202, People’s Republic of China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Weihua Yu</span> - <span class="hlFld-Affiliation affiliation">Cullgen
(Shanghai), Inc., 230
ChuanHong Road, Building 6, Chuansha, Pudong New
Area, Shanghai 201202, People’s Republic of China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yongzheng Qian</span> - <span class="hlFld-Affiliation affiliation">Cullgen
(Shanghai), Inc., 230
ChuanHong Road, Building 6, Chuansha, Pudong New
Area, Shanghai 201202, People’s Republic of China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Hui Yang</span> - <span class="hlFld-Affiliation affiliation">Cullgen
(Shanghai), Inc., 230
ChuanHong Road, Building 6, Chuansha, Pudong New
Area, Shanghai 201202, People’s Republic of China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Wuyi Yao</span> - <span class="hlFld-Affiliation affiliation">Cullgen
(Shanghai), Inc., 230
ChuanHong Road, Building 6, Chuansha, Pudong New
Area, Shanghai 201202, People’s Republic of China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jianguo Yu</span> - <span class="hlFld-Affiliation affiliation">Cullgen
(Shanghai), Inc., 230
ChuanHong Road, Building 6, Chuansha, Pudong New
Area, Shanghai 201202, People’s Republic of China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ying Luo</span> - <span class="hlFld-Affiliation affiliation">Cullgen Inc., 12671 High Bluff Drive Suite 130, San Diego, California 92130, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Michael Plewe</span> - <span class="hlFld-Affiliation affiliation">Cullgen Inc., 12671 High Bluff Drive Suite 130, San Diego, California 92130, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jialiang Wang</span> - <span class="hlFld-Affiliation affiliation">Cullgen Inc., 12671 High Bluff Drive Suite 130, San Diego, California 92130, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Cullgen
(Shanghai), Inc., 230
ChuanHong Road, Building 6, Chuansha, Pudong New
Area, Shanghai 201202, People’s Republic of China</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>The manuscript was written based on contributions of all authors. All authors have given approval to the final version of the manuscript.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare the following competing financial interest(s): All authors are employees of Cullgen. Y. L. is the president and CEO of Cullgen and GNI Group. GNI group has a significant financial equity interest in Cullgen.<br /></br></div></li></ul></div><div class="ack" id="ack1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i44">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_25204" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_25204" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">We thank Seth Goldblum at Cullgen for proof-reading the manuscript. The design, study conduct, and financial support for the research were provided by Cullgen.</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations</td></tr><tr><td class="NLM_term">TRK</td><td class="NLM_def"><p class="first last">tropomyosin receptor kinase</p></td></tr><tr><td class="NLM_term">PROTAC</td><td class="NLM_def"><p class="first last">proteolysis targeting chimera</p></td></tr><tr><td class="NLM_term">uSMITE</td><td class="NLM_def"><p class="first last">ubiquitin-mediated, small molecule-induced target elimination</p></td></tr><tr><td class="NLM_term">SNIPER</td><td class="NLM_def"><p class="first last">specific and nongenetic IAP-dependent protein erasers</p></td></tr><tr><td class="NLM_term">DEGRONIMID</td><td class="NLM_def"><p class="first last">acronym/neologism from C4 Therapeutics used to describe a protein degrader</p></td></tr><tr><td class="NLM_term">TPM3</td><td class="NLM_def"><p class="first last">tropomyosin 3</p></td></tr><tr><td class="NLM_term">AGBL4</td><td class="NLM_def"><p class="first last">ATP/GTP binding protein like 4</p></td></tr><tr><td class="NLM_term">ETV6</td><td class="NLM_def"><p class="first last">ETS variant transcription factor 6</p></td></tr><tr><td class="NLM_term">NGF</td><td class="NLM_def"><p class="first last">nerve growth factor</p></td></tr><tr><td class="NLM_term">BDNF</td><td class="NLM_def"><p class="first last">brain-derived neurotrophic factor</p></td></tr><tr><td class="NLM_term">PLCγ</td><td class="NLM_def"><p class="first last">phospholipase C gamma</p></td></tr><tr><td class="NLM_term">LMNA</td><td class="NLM_def"><p class="first last">lamin A/C</p></td></tr><tr><td class="NLM_term">MOA</td><td class="NLM_def"><p class="first last">mechanism of action</p></td></tr><tr><td class="NLM_term">UPS</td><td class="NLM_def"><p class="first last">ubiquitin–proteasome system</p></td></tr><tr><td class="NLM_term">BBB</td><td class="NLM_def"><p class="first last">blood–brain–barrier</p></td></tr><tr><td class="NLM_term">CNS</td><td class="NLM_def"><p class="first last">central nervous system</p></td></tr><tr><td class="NLM_term">CRBN</td><td class="NLM_def"><p class="first last">cereblon</p></td></tr><tr><td class="NLM_term">MS</td><td class="NLM_def"><p class="first last">mass spectrometry</p></td></tr><tr><td class="NLM_term">VHL</td><td class="NLM_def"><p class="first last">Von Hippel–Lindau</p></td></tr><tr><td class="NLM_term">SAR</td><td class="NLM_def"><p class="first last">structure–activity relationship</p></td></tr><tr><td class="NLM_term">PEG</td><td class="NLM_def"><p class="first last">polyethylene glycol</p></td></tr><tr><td class="NLM_term">GSPT1</td><td class="NLM_def"><p class="first last">G1 to S phase transition 1</p></td></tr><tr><td class="NLM_term">TMT</td><td class="NLM_def"><p class="first last">tandem mass tag</p></td></tr><tr><td class="NLM_term">CRL</td><td class="NLM_def"><p class="first last">cullin-RING E3 ubiquitin ligase</p></td></tr><tr><td class="NLM_term">PK</td><td class="NLM_def"><p class="first last">pharmacokinetic</p></td></tr><tr><td class="NLM_term">IP</td><td class="NLM_def"><p class="first last">intraperitoneal</p></td></tr><tr><td class="NLM_term">DMSO</td><td class="NLM_def"><p class="first last">dimethyl sulfoxide</p></td></tr><tr><td class="NLM_term">DIEA</td><td class="NLM_def"><p class="first last"><i>N</i>,<i>N</i>-diisopropylethylamine</p></td></tr><tr><td class="NLM_term">NMP</td><td class="NLM_def"><p class="first last"><i>N</i>-methyl-2-pyrrolidone</p></td></tr><tr><td class="NLM_term">EDCI</td><td class="NLM_def"><p class="first last">1-ethyl-3-(3- dimethylaminopropyl)carbodiimide</p></td></tr><tr><td class="NLM_term">HOAt</td><td class="NLM_def"><p class="first last">1-hydroxy-7-aza-benzo-triazole</p></td></tr><tr><td class="NLM_term">NMM</td><td class="NLM_def"><p class="first last"><i>N</i>-methylmorpholine</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i46">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_13886" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_13886" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 71 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pulciani, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Santos, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lauver, A. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Long, L. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aaronson, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barbacid, M.</span></span> <span> </span><span class="NLM_article-title">Oncogenes in solid human tumours</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">1982</span>,  <span class="NLM_volume"><i>300</i></span>,  <span class="NLM_fpage">539</span>– <span class="NLM_lpage">542</span>, <span class="refDoi"> DOI: 10.1038/300539a0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=10.1038%2F300539a0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=7144906" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=1%3ACAS%3A528%3ADyaL3sXlslekurs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=300&publication_year=1982&pages=539-542&author=S.+Pulcianiauthor=E.+Santosauthor=A.+V.+Lauverauthor=L.+K.+Longauthor=S.+A.+Aaronsonauthor=M.+Barbacid&title=Oncogenes+in+solid+human+tumours&doi=10.1038%2F300539a0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Oncogenes in solid human tumors</span></div><div class="casAuthors">Pulciani, Simonetta; Santos, Eugenio; Lauver, Anne V.; Long, Linda K.; Aaronson, Stuart A.; Barbacid, Mariano</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">1982</span>),
    <span class="NLM_cas:volume">300</span>
        (<span class="NLM_cas:issue">5892</span>),
    <span class="NLM_cas:pages">539-42</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    </div><div class="casAbstract">Dominant transforming genes were found in unmanipulated solid human tumors, including carcinomas of the colon (2), lung, and pancreas, and an embryonal rhabdomyosarcoma.  The carcinomas of the lung and pancreas and the embryonal rhabdomyosarcoma possessed a common oncogene which, like that isolated from human LX-1 lung carcinoma cells, shared sequences with the onc gene of the Kirsten strain of murine sarcoma virus (MSV).  The same oncogene was found in several other human tumor cell lines, including those established from carcinomas of the colon (A2233), lung (A427 and A2182), gall bladder (A1604), and urinary bladder.  Hence, a variety of human tumors, regardless of their clin. manifestations, contain a common transforming gene.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp_-WRvbEuIdrVg90H21EOLACvtfcHk0limLkAgYxDxRg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL3sXlslekurs%253D&md5=9f226d0b79d522381e696f5cdfb84ac5</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2F300539a0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F300539a0%26sid%3Dliteratum%253Aachs%26aulast%3DPulciani%26aufirst%3DS.%26aulast%3DSantos%26aufirst%3DE.%26aulast%3DLauver%26aufirst%3DA.%2BV.%26aulast%3DLong%26aufirst%3DL.%2BK.%26aulast%3DAaronson%26aufirst%3DS.%2BA.%26aulast%3DBarbacid%26aufirst%3DM.%26atitle%3DOncogenes%2520in%2520solid%2520human%2520tumours%26jtitle%3DNature%26date%3D1982%26volume%3D300%26spage%3D539%26epage%3D542%26doi%3D10.1038%2F300539a0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Martin-Zanca, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hughes, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barbacid, M.</span></span> <span> </span><span class="NLM_article-title">A human oncogene formed by the fusion of truncated tropomyosin and protein tyrosine kinase sequences</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">1986</span>,  <span class="NLM_volume"><i>319</i></span>,  <span class="NLM_fpage">743</span>– <span class="NLM_lpage">748</span>, <span class="refDoi"> DOI: 10.1038/319743a0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=10.1038%2F319743a0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=2869410" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=1%3ACAS%3A528%3ADyaL28XhvVSqt7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=319&publication_year=1986&pages=743-748&author=D.+Martin-Zancaauthor=S.+H.+Hughesauthor=M.+Barbacid&title=A+human+oncogene+formed+by+the+fusion+of+truncated+tropomyosin+and+protein+tyrosine+kinase+sequences&doi=10.1038%2F319743a0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">A human oncogene formed by the fusion of truncated tropomyosin and protein tyrosine kinase sequences</span></div><div class="casAuthors">Martin-Zanca, Dionisio; Hughes, Stephen H.; Barbacid, Mariano</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">1986</span>),
    <span class="NLM_cas:volume">319</span>
        (<span class="NLM_cas:issue">6056</span>),
    <span class="NLM_cas:pages">743-8</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    </div><div class="casAbstract">A biol. active cDNA clone of a transforming gene present in a human colon carcinoma contains gene sequences of both tropomyosin and a previously unknown tyrosine kinase  [80449-02-1].  The predicted protein (641 amino acids) encoded by this oncogene seems to have been formed by a somatic rearrangement that replaced the extracellular domain of a putative transmembrane receptor by the 1st 221 amino acids of a non-muscle tropomyosin mol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqA0AjUZGDew7Vg90H21EOLACvtfcHk0lh82xd1is26BA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL28XhvVSqt7g%253D&md5=da5002349ed602cac199727be7149d40</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1038%2F319743a0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F319743a0%26sid%3Dliteratum%253Aachs%26aulast%3DMartin-Zanca%26aufirst%3DD.%26aulast%3DHughes%26aufirst%3DS.%2BH.%26aulast%3DBarbacid%26aufirst%3DM.%26atitle%3DA%2520human%2520oncogene%2520formed%2520by%2520the%2520fusion%2520of%2520truncated%2520tropomyosin%2520and%2520protein%2520tyrosine%2520kinase%2520sequences%26jtitle%3DNature%26date%3D1986%26volume%3D319%26spage%3D743%26epage%3D748%26doi%3D10.1038%2F319743a0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Martin-Zanca, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oskam, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitra, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Copeland, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barbacid, M.</span></span> <span> </span><span class="NLM_article-title">Molecular and biochemical characterization of the human trk proto-oncogene</span>. <i>Mol. Cell. Biol.</i> <span class="NLM_year" style="font-weight: bold;">1989</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">24</span>– <span class="NLM_lpage">33</span>, <span class="refDoi"> DOI: 10.1128/mcb.9.1.24</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=10.1128%2FMCB.9.1.24" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=2927393" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=1%3ACAS%3A528%3ADyaL1MXmtV2ru78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=1989&pages=24-33&author=D.+Martin-Zancaauthor=R.+Oskamauthor=G.+Mitraauthor=T.+Copelandauthor=M.+Barbacid&title=Molecular+and+biochemical+characterization+of+the+human+trk+proto-oncogene&doi=10.1128%2Fmcb.9.1.24"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular and biochemical characterization of the human trk proto-oncogene</span></div><div class="casAuthors">Martin-Zanca, Dionisio; Oskam, Ralph; Mitra, Gopa; Copeland, Terry; Barbacid, Mariano</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Biology</span>
        (<span class="NLM_cas:date">1989</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">24-33</span>CODEN:
                <span class="NLM_cas:coden">MCEBD4</span>;
        ISSN:<span class="NLM_cas:issn">0270-7306</span>.
    </div><div class="casAbstract">Mol. anal. of the human trk oncogene, a transforming gene isolated from a colon biopsy, revealed the existence of a novel member of the tyrosine kinase gene family.  This locus codes for a protein of 790 amino acid residues that has several features characteristic of cell surface receptors.  They include (i) a 32-amino-acid-long putative signal peptide, (ii) an amino-terminal moiety (residues 33 to 407) rich in consensus sites for N-glycosylation, (iii) a transmembrane domain, (iv) a kinase catalytic region highly related to that of other tyrosine kinases, and (v) a very short (15 residue) carboxy-terminal tail.  Residues 1 to 392 were absent in the trk oncogene, as they were replaced by tropomyosin sequences.  However, no other differences were found between the transforming and nontransforming trk alleles (residues 392 to 790), suggesting that no addnl. mutations are required to activate the transforming potential of this gene.  The human trk proto-oncogene codes for a 140,000-dalton glycoprotein, designated gp140proto-trk.  However, its primary translational product is a 110,000-dalton glycoprotein which becomes immediately glycosylated, presumably during its translocation into the endoplasmic reticulum.  This mol., designated gp110proto-trk, is further glycosylated to yield the mature form, gp140proto-trk.  Both gp110proto-trk and gp140proto-trk proteins possess in vitro kinase activity specific for tyrosine residues.  Finally, iodination of intact NIH 3T3 cells expressing trk proto-oncogene products indicated that only the mature form, gp140proto-trk, cross the plasma membrane, becoming exposed to the outside of the cell.  These results indicate that the product of the human trk locus is a novel tyrosine kinase cell surface receptor for an as yet unknown ligand.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGru0x_s3mmlXrVg90H21EOLACvtfcHk0lh82xd1is26BA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL1MXmtV2ru78%253D&md5=6b98f810db97ba934de4c5e6a848abd9</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1128%2FMCB.9.1.24&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMCB.9.1.24%26sid%3Dliteratum%253Aachs%26aulast%3DMartin-Zanca%26aufirst%3DD.%26aulast%3DOskam%26aufirst%3DR.%26aulast%3DMitra%26aufirst%3DG.%26aulast%3DCopeland%26aufirst%3DT.%26aulast%3DBarbacid%26aufirst%3DM.%26atitle%3DMolecular%2520and%2520biochemical%2520characterization%2520of%2520the%2520human%2520trk%2520proto-oncogene%26jtitle%3DMol.%2520Cell.%2520Biol.%26date%3D1989%26volume%3D9%26spage%3D24%26epage%3D33%26doi%3D10.1128%2Fmcb.9.1.24" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Klein, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parada, L. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coulier, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barbacid, M.</span></span> <span> </span><span class="NLM_article-title">TrkB, a novel tyrosine protein kinase receptor expressed during mouse neural development</span>. <i>EMBO J.</i> <span class="NLM_year" style="font-weight: bold;">1989</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">3701</span>– <span class="NLM_lpage">3709</span>, <span class="refDoi"> DOI: 10.1002/j.1460-2075.1989.tb08545.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=10.1002%2Fj.1460-2075.1989.tb08545.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=2555172" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=1%3ACAS%3A528%3ADyaK3cXktlGlsbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=1989&pages=3701-3709&author=R.+Kleinauthor=L.+F.+Paradaauthor=F.+Coulierauthor=M.+Barbacid&title=TrkB%2C+a+novel+tyrosine+protein+kinase+receptor+expressed+during+mouse+neural+development&doi=10.1002%2Fj.1460-2075.1989.tb08545.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">trkB, a novel tyrosine protein kinase receptor expressed during mouse neural development</span></div><div class="casAuthors">Klein, Rudiger; Parada, Luis F.; Coulier, Francois; Barbacid, Mariano</div><div class="citationInfo"><span class="NLM_cas:title">EMBO Journal</span>
        (<span class="NLM_cas:date">1989</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">3701-9</span>CODEN:
                <span class="NLM_cas:coden">EMJODG</span>;
        ISSN:<span class="NLM_cas:issn">0261-4189</span>.
    </div><div class="casAbstract">The authors isolated a novel member of the tyrosine protein kinase family of cell surface receptors.  This gene, designated trkB, is highly related to the human trk protoncogene.  At the amino acid level, their resp. products share a 57% homol. in their extracellular regions, including 9 of the 11 cysteines present in the trk proto-oncogene.  This homol. increases to 88% within their resp. tyrosine kinase catalytic domains.  Both trk and trkB are equally distantly related to the other members of this gene family of receptors.  A biol. active cDNA clone of trkB can direct the synthesis of gp145trkB, a glycoprotein of 145 kd of which only 93 kd correspond to its polypeptide backbone.  In adult mice, trkB is preferentially expressed in brain tissue, although significant levels of trkB RNA have also been obsd. in lung, muscle, and ovaries.  In addn., trkB transcripts can be detected in mid and late gestation embryos.  The trkB locus exhibits a complex pattern of transcription.  At least seven RNA species ranging in size from ∼9 kb to 2 kb have been identified in brain.  However, only a subset of these transcripts appears to be expressed in the other tissues.  In situ hybridization anal. of 14 and 18 day old mouse embryos indicates that trkB transcripts are localized in the central (CNS) and peripheral (PNS) nervous systems, including brain, spinal cord, spinal and cranial ganglia, paravertebral trunk of the sympathetic nervous system, and various innervation pathways.  These results suggest that trkB may code for a novel cell surface receptor involved in neurogenesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpZMlwtDc9XqLVg90H21EOLACvtfcHk0lh82xd1is26BA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3cXktlGlsbg%253D&md5=2fb4ec51e64c20ac0d721ce13a85f592</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1002%2Fj.1460-2075.1989.tb08545.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fj.1460-2075.1989.tb08545.x%26sid%3Dliteratum%253Aachs%26aulast%3DKlein%26aufirst%3DR.%26aulast%3DParada%26aufirst%3DL.%2BF.%26aulast%3DCoulier%26aufirst%3DF.%26aulast%3DBarbacid%26aufirst%3DM.%26atitle%3DTrkB%252C%2520a%2520novel%2520tyrosine%2520protein%2520kinase%2520receptor%2520expressed%2520during%2520mouse%2520neural%2520development%26jtitle%3DEMBO%2520J.%26date%3D1989%26volume%3D8%26spage%3D3701%26epage%3D3709%26doi%3D10.1002%2Fj.1460-2075.1989.tb08545.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lamballe, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klein, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barbacid, M.</span></span> <span> </span><span class="NLM_article-title">TrkC, a new member of the trk family of tyrosine protein kinases, is a receptor for neurotrophin-3</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">1991</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">967</span>– <span class="NLM_lpage">979</span>, <span class="refDoi"> DOI: 10.1016/0092-8674(91)90442-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=10.1016%2F0092-8674%2891%2990442-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=1653651" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=1%3ACAS%3A528%3ADyaK38Xmt12jsro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=1991&pages=967-979&author=F.+Lamballeauthor=R.+Kleinauthor=M.+Barbacid&title=TrkC%2C+a+new+member+of+the+trk+family+of+tyrosine+protein+kinases%2C+is+a+receptor+for+neurotrophin-3&doi=10.1016%2F0092-8674%2891%2990442-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">TrkC, a new member of the trk family of tyrosine protein kinases, is a receptor for neurotrophin-3</span></div><div class="casAuthors">Lamballe, Fabienne; Klein, Rudiger; Barbacid, Mariano</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">1991</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">967-79</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    </div><div class="casAbstract">The isolation and mol. characterization of trkC, a new member of the trk family of tyrosine protein kinase genes is reported.  Gene trkC is preferentially expressed in the brain.  In situ hybridization studies revealed trkC transcripts in the hippocampus, cerebral cortex, and the granular cell layer of the cerebellum.  The product of the trkC gene was identified as a glycoprotein of 145,000 daltons, gp145trkC, which is equally related to the previously characterized gp140trk and gp145trkB tyrosine kinases.  Glycoprotein gp145trkC is a functional receptor for neurotrophin-3 (NT-3).  However, gp145trkC does not bind the highly related neurotrophic factors, NGF or BDNF.  In proliferating cells, the interaction between gp145trkC and NT-3 elicits a more efficient biol. response than when NT-3 binds to its other receptors, gp140trk and gp145trkB.  These results indicate that gp145trkC may play an important role in mediating the neurotrophic effects of NT-3.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrp1C6bvPklNLVg90H21EOLACvtfcHk0lhHSSQmRrnWYg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK38Xmt12jsro%253D&md5=8aa49b5b94d3f1f337ae097d246e8e9e</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1016%2F0092-8674%2891%2990442-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0092-8674%252891%252990442-2%26sid%3Dliteratum%253Aachs%26aulast%3DLamballe%26aufirst%3DF.%26aulast%3DKlein%26aufirst%3DR.%26aulast%3DBarbacid%26aufirst%3DM.%26atitle%3DTrkC%252C%2520a%2520new%2520member%2520of%2520the%2520trk%2520family%2520of%2520tyrosine%2520protein%2520kinases%252C%2520is%2520a%2520receptor%2520for%2520neurotrophin-3%26jtitle%3DCell%26date%3D1991%26volume%3D66%26spage%3D967%26epage%3D979%26doi%3D10.1016%2F0092-8674%2891%2990442-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Klein, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jing, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nanduri, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Rourke, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barbacid, M.</span></span> <span> </span><span class="NLM_article-title">The trk proto-oncogene encodes a receptor for nerve growth factor</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">1991</span>,  <span class="NLM_volume"><i>65</i></span>,  <span class="NLM_fpage">189</span>– <span class="NLM_lpage">197</span>, <span class="refDoi"> DOI: 10.1016/0092-8674(91)90419-y</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=10.1016%2F0092-8674%2891%2990419-Y" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=1849459" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=1%3ACAS%3A528%3ADyaK3MXisVWgs7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=65&publication_year=1991&pages=189-197&author=R.+Kleinauthor=S.+Jingauthor=V.+Nanduriauthor=E.+O%E2%80%99Rourkeauthor=M.+Barbacid&title=The+trk+proto-oncogene+encodes+a+receptor+for+nerve+growth+factor&doi=10.1016%2F0092-8674%2891%2990419-y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">The trk proto-oncogene encodes a receptor for nerve growth factor</span></div><div class="casAuthors">Klein, Ruediger; Jing, Shuqian; Nanduri, Venkata; O'Rourke, Edward; Barbacid, Mariano</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">1991</span>),
    <span class="NLM_cas:volume">65</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">189-97</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    </div><div class="casAbstract">Two classes of receptors with distinct affinities for nerve growth factor (NGF) have been identified.  The low affinity receptor (Kd ≈ 10-9-10-8M) is a cysteine-rich glycoprotein encoded by the previously characterized LNGFR gene.  The structural nature of the high affinity receptor (Kd ≈ 10-10M) has yet to be established.  The product of the human trk proto-oncogene (gp140trk) bound NGF with high affinity.  Moreover, NGF could be chem. cross-linked to the endogenous gp140trk present in rat PC12 pheochromocytoma cells, as well as to gp140trk ectopically expressed in mouse fibroblasts and in insect Sf9 cells.  High affinity binding of NGF to gp140trk could occur in the absence of low affinity LNGFR receptors, at least in nonneural cells.  Addn. of NGF to PC12 cells elicited rapid phosphorylation of gp140trk on tyrosine residues and stimulated its tyrosine kinase activity.  Apparently, gp14-trk is a functional NGF receptor that mediates at least some of the signal transduction processes initiated by this neurotropic factor.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqlUBY2nUNMtLVg90H21EOLACvtfcHk0lhHSSQmRrnWYg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3MXisVWgs7Y%253D&md5=362cd01ed7fdf258cfe7f59169b6b5ee</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1016%2F0092-8674%2891%2990419-Y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0092-8674%252891%252990419-Y%26sid%3Dliteratum%253Aachs%26aulast%3DKlein%26aufirst%3DR.%26aulast%3DJing%26aufirst%3DS.%26aulast%3DNanduri%26aufirst%3DV.%26aulast%3DO%25E2%2580%2599Rourke%26aufirst%3DE.%26aulast%3DBarbacid%26aufirst%3DM.%26atitle%3DThe%2520trk%2520proto-oncogene%2520encodes%2520a%2520receptor%2520for%2520nerve%2520growth%2520factor%26jtitle%3DCell%26date%3D1991%26volume%3D65%26spage%3D189%26epage%3D197%26doi%3D10.1016%2F0092-8674%2891%2990419-y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kaplan, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hempstead, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin-Zanca, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chao, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parada, L.</span></span> <span> </span><span class="NLM_article-title">The trk proto-oncogene product: a signal transducing receptor for nerve growth factor</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">1991</span>,  <span class="NLM_volume"><i>252</i></span>,  <span class="NLM_fpage">554</span>– <span class="NLM_lpage">558</span>, <span class="refDoi"> DOI: 10.1126/science.1850549</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=10.1126%2Fscience.1850549" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=1850549" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=1%3ACAS%3A528%3ADyaK3MXisVWgtrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=252&publication_year=1991&pages=554-558&author=D.+Kaplanauthor=B.+Hempsteadauthor=D.+Martin-Zancaauthor=M.+Chaoauthor=L.+Parada&title=The+trk+proto-oncogene+product%3A+a+signal+transducing+receptor+for+nerve+growth+factor&doi=10.1126%2Fscience.1850549"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">The trk proto-oncogene product:  a signal transducing receptor for nerve growth factor</span></div><div class="casAuthors">Kaplan, David R.; Hempstead, Barbara L.; Martin-Zanca, Dionisio; Chao, Moses V.; Parada, Luis F.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">1991</span>),
    <span class="NLM_cas:volume">252</span>
        (<span class="NLM_cas:issue">5005</span>),
    <span class="NLM_cas:pages">554-8</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    </div><div class="casAbstract">The trk proto-oncogene encodes a 140-kilodalton, membrane-spanning protein tyrosine kinase (p140prototrk) that is expressed only in neural tissues.  Nerve growth factor (NGF) stimulates phosphorylation of p140prototrk in neural cell lines and in embσyonic dorsal root ganglia.  Affinity crosslinking and equil. binding expts. with 125I-labeled NGF indicate that p140prototrk binds NGF specifically in cultured cells with a dissocn. const. of 10-9 M.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoOsZ0C6-OsXrVg90H21EOLACvtfcHk0lhHSSQmRrnWYg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3MXisVWgtrg%253D&md5=5daf16d6938c40bbeb4c44fc58a5df87</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1126%2Fscience.1850549&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1850549%26sid%3Dliteratum%253Aachs%26aulast%3DKaplan%26aufirst%3DD.%26aulast%3DHempstead%26aufirst%3DB.%26aulast%3DMartin-Zanca%26aufirst%3DD.%26aulast%3DChao%26aufirst%3DM.%26aulast%3DParada%26aufirst%3DL.%26atitle%3DThe%2520trk%2520proto-oncogene%2520product%253A%2520a%2520signal%2520transducing%2520receptor%2520for%2520nerve%2520growth%2520factor%26jtitle%3DScience%26date%3D1991%26volume%3D252%26spage%3D554%26epage%3D558%26doi%3D10.1126%2Fscience.1850549" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Soppet, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Escandon, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maragos, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Middlemas, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raid, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blair, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burton, L. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stanton, B. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaplan, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hunter, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nikolics, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parade, L. F.</span></span> <span> </span><span class="NLM_article-title">The neurotrophic factors brain-derived neurotrophic factor and neurotrophin-3 are ligands for the trkB tyrosine kinase receptor</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">1991</span>,  <span class="NLM_volume"><i>65</i></span>,  <span class="NLM_fpage">895</span>– <span class="NLM_lpage">903</span>, <span class="refDoi"> DOI: 10.1016/0092-8674(91)90396-g</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=10.1016%2F0092-8674%2891%2990396-G" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=1645620" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=1%3ACAS%3A528%3ADyaK3MXks1Gnur8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=65&publication_year=1991&pages=895-903&author=D.+Soppetauthor=E.+Escandonauthor=J.+Maragosauthor=D.+S.+Middlemasauthor=S.+W.+Raidauthor=J.+Blairauthor=L.+E.+Burtonauthor=B.+R.+Stantonauthor=D.+R.+Kaplanauthor=T.+Hunterauthor=K.+Nikolicsauthor=L.+F.+Parade&title=The+neurotrophic+factors+brain-derived+neurotrophic+factor+and+neurotrophin-3+are+ligands+for+the+trkB+tyrosine+kinase+receptor&doi=10.1016%2F0092-8674%2891%2990396-g"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">The neurotropic factors brain-derived neurotropic factor and neurotrophin-3 are ligands for the trkB tyrosine kinase receptor</span></div><div class="casAuthors">Soppet, Dan; Escandon, Enrique; Maragos, Johnna; Middlemas, David S.; Reid, Susan W.; Blair, Janet; Burton, Louis E.; Stanton, Brian R.; Kaplan, David R.; et al.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">1991</span>),
    <span class="NLM_cas:volume">65</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">895-903</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    </div><div class="casAbstract">Neurotropic factors are essential for neuronal survival and function.  Recent data have demonstrated that the product of the tyrosine kinase trk proto-oncogene binds (nerve growth factor) NGF and is a component of the high affinity NGF receptor.  Anal. of the trkB gene product, gp145trkB, in NIH 3T3 cells indicates that this tyrosine kinase receptor is rapidly phosphorylated on tyrosine residues upon exposure to the NGF-related neurotropic factors BDNF and NT-3.  Furthermore, gp145trkB specifically binds BDNF and NT-3 in NIH 3T3 cells and in hippocampal cells, but does not bind NGF.  Thus, the trk family of receptors are likely to be important signal transducers of NGF-related trophic signals in the formation and maintenance of neuronal circuits.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGocw0gcgp1GxrVg90H21EOLACvtfcHk0ljpiv4PgLDm8A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3MXks1Gnur8%253D&md5=c8ac31331087f978d5b36a3d8bb84152</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1016%2F0092-8674%2891%2990396-G&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0092-8674%252891%252990396-G%26sid%3Dliteratum%253Aachs%26aulast%3DSoppet%26aufirst%3DD.%26aulast%3DEscandon%26aufirst%3DE.%26aulast%3DMaragos%26aufirst%3DJ.%26aulast%3DMiddlemas%26aufirst%3DD.%2BS.%26aulast%3DRaid%26aufirst%3DS.%2BW.%26aulast%3DBlair%26aufirst%3DJ.%26aulast%3DBurton%26aufirst%3DL.%2BE.%26aulast%3DStanton%26aufirst%3DB.%2BR.%26aulast%3DKaplan%26aufirst%3DD.%2BR.%26aulast%3DHunter%26aufirst%3DT.%26aulast%3DNikolics%26aufirst%3DK.%26aulast%3DParade%26aufirst%3DL.%2BF.%26atitle%3DThe%2520neurotrophic%2520factors%2520brain-derived%2520neurotrophic%2520factor%2520and%2520neurotrophin-3%2520are%2520ligands%2520for%2520the%2520trkB%2520tyrosine%2520kinase%2520receptor%26jtitle%3DCell%26date%3D1991%26volume%3D65%26spage%3D895%26epage%3D903%26doi%3D10.1016%2F0092-8674%2891%2990396-g" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vaishnavi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Le, A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doebele, R. C.</span></span> <span> </span><span class="NLM_article-title">TRKing down an old oncogene in a new era of targeted therapy</span>. <i>Canc. Discov.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">25</span>– <span class="NLM_lpage">34</span>, <span class="refDoi"> DOI: 10.1158/2159-8290.cd-14-0765</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=10.1158%2F2159-8290.CD-14-0765" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=25527197" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtFCntbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2015&pages=25-34&author=A.+Vaishnaviauthor=A.+T.+Leauthor=R.+C.+Doebele&title=TRKing+down+an+old+oncogene+in+a+new+era+of+targeted+therapy&doi=10.1158%2F2159-8290.cd-14-0765"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">TRKing Down an Old Oncogene in a New Era of Targeted Therapy</span></div><div class="casAuthors">Vaishnavi, Aria; Le, Anh T.; Doebele, Robert C.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Discovery</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">25-34</span>CODEN:
                <span class="NLM_cas:coden">CDAIB2</span>;
        ISSN:<span class="NLM_cas:issn">2159-8274</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">A review.  The use of high-throughput next-generation sequencing techniques in multiple tumor types during the last few years has identified NTRK1, 2, and 3 gene rearrangements encoding novel oncogenic fusions in 19 different tumor types to date.  These recent developments have led us to revisit an old oncogene, Trk (originally identified as OncD), which encodes the TPM3-NTRK1 gene fusion and was one of the first transforming chromosomal rearrangements identified 32 years ago.  However, no drug has yet been approved by the FDA for cancers harboring this oncogene.  This review will discuss the biol. of the TRK family of receptors, their role in human cancer, the types of oncogenic alterations, and drugs that are currently in development for this family of oncogene targets.  Significance: Precision oncol. approaches have accelerated recently due to advancements in our ability to detect oncogenic mutations in tumor samples.  Oncogenic alterations, most commonly gene fusions, have now been detected for the genes encoding the TRKA, TRKB, and TRKC receptor tyrosine kinases across multiple tumor types.  The scientific rationale for the targeting of the TRK oncogene family will be discussed here.  Cancer Discov; 5(1); 25-34. ©2014 AACR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqKUH4ooHY0TbVg90H21EOLACvtfcHk0ljpiv4PgLDm8A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtFCntbw%253D&md5=02103d46d85b6dc948dba58d32ffba9f</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1158%2F2159-8290.CD-14-0765&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F2159-8290.CD-14-0765%26sid%3Dliteratum%253Aachs%26aulast%3DVaishnavi%26aufirst%3DA.%26aulast%3DLe%26aufirst%3DA.%2BT.%26aulast%3DDoebele%26aufirst%3DR.%2BC.%26atitle%3DTRKing%2520down%2520an%2520old%2520oncogene%2520in%2520a%2520new%2520era%2520of%2520targeted%2520therapy%26jtitle%3DCanc.%2520Discov.%26date%3D2015%26volume%3D5%26spage%3D25%26epage%3D34%26doi%3D10.1158%2F2159-8290.cd-14-0765" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Davies, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Horton, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burton, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmelzer, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vandlen, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosenthal, A.</span></span> <span> </span><span class="NLM_article-title">Neurotrophin-4/5 is a mammalian-specific survival factor for distinct populations of sensory neurons</span>. <i>J. Neurosci.</i> <span class="NLM_year" style="font-weight: bold;">1993</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">4961</span>– <span class="NLM_lpage">4967</span>, <span class="refDoi"> DOI: 10.1523/jneurosci.13-11-04961.1993</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=10.1523%2FJNEUROSCI.13-11-04961.1993" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=8229208" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=1%3ACAS%3A528%3ADyaK2cXks1GltA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=1993&pages=4961-4967&author=A.+Daviesauthor=A.+Hortonauthor=L.+Burtonauthor=C.+Schmelzerauthor=R.+Vandlenauthor=A.+Rosenthal&title=Neurotrophin-4%2F5+is+a+mammalian-specific+survival+factor+for+distinct+populations+of+sensory+neurons&doi=10.1523%2Fjneurosci.13-11-04961.1993"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Neurotrophin-4/5 is a mammalian-specific survival factor for distinct populations of sensory neurons</span></div><div class="casAuthors">Davies, Alun M.; Horton, Antony; Burton, Louis E.; Schmelzer, Charles; Vandlen, Richard; Rosenthal, Arnon</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Neuroscience</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">4961-7</span>CODEN:
                <span class="NLM_cas:coden">JNRSDS</span>;
        ISSN:<span class="NLM_cas:issn">0270-6474</span>.
    </div><div class="casAbstract">The authors have studied the effect of human recombinant neurotrophin-4/5 (NT-4/5) on the survival of developing peripheral nervous system neurons from embryonic mice and chickens.  NT-4/5 transiently supported mouse NGF-dependent trigeminal and jugular neurons at early stages of target field innervation and mouse brain-derived neurotrophic factor (BDNF)-dependent nodose neurons during the phase of naturally occurring cell death.  NT-4/5 was as potent as BDNF in supporting the survival of these neuronal populations.  Surprisingly, NT-4/5 was 3 orders of magnitude less potent than BDNF as a survival factor for early chick dorsomedial trigeminal sensory neurons and did not support the survival of chick BDNF-dependent trigeminal mesencephalic or ventrolateral trigeminal sensory neurons at any of the developmental stages tested.  Thus, NT-4/5 is a survival factor for certain embryonic mouse cranial sensory neurons.  It is the first species-specific neurotrophin to be identified and it can discriminate at high concns. between different BDNF-responsive chick neurons.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGod0Sjz6gWpk7Vg90H21EOLACvtfcHk0ljpiv4PgLDm8A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2cXks1GltA%253D%253D&md5=296f7b429fa95d629d5b65ed44d57a6b</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1523%2FJNEUROSCI.13-11-04961.1993&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1523%252FJNEUROSCI.13-11-04961.1993%26sid%3Dliteratum%253Aachs%26aulast%3DDavies%26aufirst%3DA.%26aulast%3DHorton%26aufirst%3DA.%26aulast%3DBurton%26aufirst%3DL.%26aulast%3DSchmelzer%26aufirst%3DC.%26aulast%3DVandlen%26aufirst%3DR.%26aulast%3DRosenthal%26aufirst%3DA.%26atitle%3DNeurotrophin-4%252F5%2520is%2520a%2520mammalian-specific%2520survival%2520factor%2520for%2520distinct%2520populations%2520of%2520sensory%2520neurons%26jtitle%3DJ.%2520Neurosci.%26date%3D1993%26volume%3D13%26spage%3D4961%26epage%3D4967%26doi%3D10.1523%2Fjneurosci.13-11-04961.1993" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Segal, R. A.</span></span> <span> </span><span class="NLM_article-title">Selectivity in neurotrophin signaling: theme and variations</span>. <i>Annu. Rev. Neurosci.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">299</span>– <span class="NLM_lpage">330</span>, <span class="refDoi"> DOI: 10.1146/annurev.neuro.26.041002.131421</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=10.1146%2Fannurev.neuro.26.041002.131421" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=12598680" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=1%3ACAS%3A528%3ADC%252BD3sXntFSisLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2003&pages=299-330&author=R.+A.+Segal&title=Selectivity+in+neurotrophin+signaling%3A+theme+and+variations&doi=10.1146%2Fannurev.neuro.26.041002.131421"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Selectivity in neurotrophin signaling: theme and variations</span></div><div class="casAuthors">Segal, Rosalind A.</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Neuroscience</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">299-330</span>CODEN:
                <span class="NLM_cas:coden">ARNSD5</span>;
        ISSN:<span class="NLM_cas:issn">0147-006X</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews Inc.</span>)
        </div><div class="casAbstract">A review.  Neurotrophins are a family of growth factors crit. for the development and functioning of the nervous system.  Although originally identified as neuronal survival factors, neurotrophins elicit many biol. effects, ranging from proliferation to synaptic modulation to axonal pathfinding.  Recent data indicate that the nature of the signaling cascades activated by neurotrophins, and the biol. responses that ensue, are specified not only by the ligand itself but also by the temporal pattern and spatial location of stimulation.  Studies on neurotrophin signaling have revealed variations in the Ras/MAP kinase, phosphatidylinositol 3-kinase, and phospholipase C pathways, which transmit spatial and temporal information.  The anatomy of neurons makes them particularly appropriate for studying how the location and tempo of stimulation det. the signal cascades that are activated by receptor tyrosine kinases such as the Trk receptors.  These signaling variations may represent a general mechanism eliciting specificity in growth factor responses.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpjZGcPFtywOLVg90H21EOLACvtfcHk0ljpiv4PgLDm8A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXntFSisLo%253D&md5=f585bf86e97bbcf7467d21533b62cda2</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1146%2Fannurev.neuro.26.041002.131421&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev.neuro.26.041002.131421%26sid%3Dliteratum%253Aachs%26aulast%3DSegal%26aufirst%3DR.%2BA.%26atitle%3DSelectivity%2520in%2520neurotrophin%2520signaling%253A%2520theme%2520and%2520variations%26jtitle%3DAnnu.%2520Rev.%2520Neurosci.%26date%3D2003%26volume%3D26%26spage%3D299%26epage%3D330%26doi%3D10.1146%2Fannurev.neuro.26.041002.131421" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hanahan, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weinberg, R. A.</span></span> <span> </span><span class="NLM_article-title">Hallmarks of cancer: the next generation</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>144</i></span>,  <span class="NLM_fpage">646</span>– <span class="NLM_lpage">674</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2011.02.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=10.1016%2Fj.cell.2011.02.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=21376230" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=1%3ACAS%3A528%3ADC%252BC3MXjsFeqtrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=144&publication_year=2011&pages=646-674&author=D.+Hanahanauthor=R.+A.+Weinberg&title=Hallmarks+of+cancer%3A+the+next+generation&doi=10.1016%2Fj.cell.2011.02.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Hallmarks of cancer: the next generation</span></div><div class="casAuthors">Hanahan, Douglas; Weinberg, Robert A.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">144</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">646-674</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  The hallmarks of cancer comprise six biol. capabilities acquired during the multistep development of human tumors.  The hallmarks constitute an organizing principle for rationalizing the complexities of neoplastic disease.  They include sustaining proliferative signaling, evading growth suppressors, resisting cell death, enabling replicative immortality, inducing angiogenesis, and activating invasion and metastasis.  Underlying these hallmarks are genome instability, which generates the genetic diversity that expedites their acquisition, and inflammation, which fosters multiple hallmark functions.  Conceptual progress in the last decade has added two emerging hallmarks of potential generality to this list-reprogramming of energy metab. and evading immune destruction.  In addn. to cancer cells, tumors exhibit another dimension of complexity: they contain a repertoire of recruited, ostensibly normal cells that contribute to the acquisition of hallmark traits by creating the "tumor microenvironment.".  Recognition of the widespread applicability of these concepts will increasingly affect the development of new means to treat human cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqTcpqXALcb6LVg90H21EOLACvtfcHk0ljXIbsawrNEJw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXjsFeqtrk%253D&md5=b36f160c417a712e83cb83c577f0018e</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2011.02.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2011.02.013%26sid%3Dliteratum%253Aachs%26aulast%3DHanahan%26aufirst%3DD.%26aulast%3DWeinberg%26aufirst%3DR.%2BA.%26atitle%3DHallmarks%2520of%2520cancer%253A%2520the%2520next%2520generation%26jtitle%3DCell%26date%3D2011%26volume%3D144%26spage%3D646%26epage%3D674%26doi%3D10.1016%2Fj.cell.2011.02.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cocco, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scaltriti, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drilon, A.</span></span> <span> </span><span class="NLM_article-title">NTRK fusion-positive cancers and TRK inhibitor therapy</span>. <i>Nat. Rev. Clin. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">731</span>– <span class="NLM_lpage">747</span>, <span class="refDoi"> DOI: 10.1038/s41571-018-0113-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=10.1038%2Fs41571-018-0113-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=30333516" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhvFCrsLnO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2018&pages=731-747&author=E.+Coccoauthor=M.+Scaltritiauthor=A.+Drilon&title=NTRK+fusion-positive+cancers+and+TRK+inhibitor+therapy&doi=10.1038%2Fs41571-018-0113-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">NTRK fusion-positive cancers and TRK inhibitor therapy</span></div><div class="casAuthors">Cocco, Emiliano; Scaltriti, Maurizio; Drilon, Alexander</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Clinical Oncology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">731-747</span>CODEN:
                <span class="NLM_cas:coden">NRCOAA</span>;
        ISSN:<span class="NLM_cas:issn">1759-4774</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">A review.  NTRK gene fusions involving either NTRK1, NTRK2 or NTRK3 (encoding the neurotrophin receptors TRKA, TRKB and TRKC, resp.) are oncogenic drivers of various adult and paediatric tumor types.  These fusions can be detected in the clinic using a variety of methods, including tumor DNA and RNA sequencing and plasma cell-free DNA profiling.  The treatment of patients with NTRK fusion-pos. cancers with a first-generation TRK inhibitor, such as larotrectinib or entrectinib, is assocd. with high response rates (>75%), regardless of tumor histol.  First-generation TRK inhibitors are well tolerated by most patients, with toxicity profiles characterized by occasional off-tumor, on-target adverse events (attributable to TRK inhibition in non-malignant tissues).  Despite durable disease control in many patients, advanced-stage NTRK fusion-pos. cancers eventually become refractory to TRK inhibition; resistance can be mediated by the acquisition of NTRK kinase domain mutations.  Fortunately, certain resistance mutations can be overcome by second-generation TRK inhibitors, including LOXO-195 and TPX-0005 that are being explored in clin. trials.  In this Review, we discuss the biol. of NTRK fusions, strategies to target these drivers in the treatment-naive and acquired-resistance disease settings, and the unique safety profile of TRK inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo2tGShkjX-5LVg90H21EOLACvtfcHk0ljXIbsawrNEJw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhvFCrsLnO&md5=fbc1f5b0de3614008f1b1f1e2e6167ce</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1038%2Fs41571-018-0113-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41571-018-0113-0%26sid%3Dliteratum%253Aachs%26aulast%3DCocco%26aufirst%3DE.%26aulast%3DScaltriti%26aufirst%3DM.%26aulast%3DDrilon%26aufirst%3DA.%26atitle%3DNTRK%2520fusion-positive%2520cancers%2520and%2520TRK%2520inhibitor%2520therapy%26jtitle%3DNat.%2520Rev.%2520Clin.%2520Oncol.%26date%3D2018%26volume%3D15%26spage%3D731%26epage%3D747%26doi%3D10.1038%2Fs41571-018-0113-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ouyang, L.</span></span> <span> </span><span class="NLM_article-title">Development of small-molecule tropomyosin receptor kinase (TRK) inhibitors for NTRK fusion cancers</span>. <i>Acta Pharm. Sin. B</i> <span class="NLM_year" style="font-weight: bold;">2020</span>, <span class="refDoi"> DOI: 10.1016/j.apsb.2020.05.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=10.1016%2Fj.apsb.2020.05.004" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2020&author=T.+Jiangauthor=G.+Wangauthor=Y.+Liuauthor=L.+Fengauthor=M.+Wangauthor=J.+Liuauthor=Y.+Chenauthor=L.+Ouyang&title=Development+of+small-molecule+tropomyosin+receptor+kinase+%28TRK%29+inhibitors+for+NTRK+fusion+cancers&doi=10.1016%2Fj.apsb.2020.05.004"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1016%2Fj.apsb.2020.05.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.apsb.2020.05.004%26sid%3Dliteratum%253Aachs%26aulast%3DJiang%26aufirst%3DT.%26aulast%3DWang%26aufirst%3DG.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DFeng%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DOuyang%26aufirst%3DL.%26atitle%3DDevelopment%2520of%2520small-molecule%2520tropomyosin%2520receptor%2520kinase%2520%2528TRK%2529%2520inhibitors%2520for%2520NTRK%2520fusion%2520cancers%26jtitle%3DActa%2520Pharm.%2520Sin.%2520B%26date%3D2020%26doi%3D10.1016%2Fj.apsb.2020.05.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yan, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lakkaniga, N. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carlomagno, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Santoro, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McDonald, N. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lv, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gunaganti, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frett, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, H.-y.</span></span> <span> </span><span class="NLM_article-title">Insights into current tropomyosin receptor kinase (TRK) inhibitors: development and clinical application</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">1731</span>– <span class="NLM_lpage">1760</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b01092</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b01092" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhs1Khur%252FO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=1731-1760&author=W.+Yanauthor=N.+R.+Lakkanigaauthor=F.+Carlomagnoauthor=M.+Santoroauthor=N.+Q.+McDonaldauthor=F.+Lvauthor=N.+Gunagantiauthor=B.+Frettauthor=H.-y.+Li&title=Insights+into+current+tropomyosin+receptor+kinase+%28TRK%29+inhibitors%3A+development+and+clinical+application&doi=10.1021%2Facs.jmedchem.8b01092"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Insights into Current Tropomyosin Receptor Kinase (TRK) Inhibitors: Development and Clinical Application</span></div><div class="casAuthors">Yan, Wei; Lakkaniga, Naga Rajiv; Carlomagno, Francesca; Santoro, Massimo; McDonald, Neil Q.; Lv, Fengping; Gunaganti, Naresh; Frett, Brendan; Li, Hong-yu</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1731-1760</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The use of kinase-directed precision medicine has been heavily pursued since the discovery and development of imatinib.  Annually, it is estd. that around ∼20 000 new cases of tropomyosin receptor kinase (TRK) cancers are diagnosed, with the majority of cases exhibiting a TRK genomic rearrangement.  In this Perspective, we discuss current development and clin. applications for TRK precision medicine by providing the following: (1) the biol. background and significance of the TRK kinase family, (2) a compilation of known TRK inhibitors and anal. of their cocrystal structures, (3) an overview of TRK clin. trials, and (4) future perspectives for drug discovery and development of TRK inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrwUthQyJEzFrVg90H21EOLACvtfcHk0lgur0hx_xDapw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhs1Khur%252FO&md5=5ce2f8d295cf46f84b46f5890dad0160</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b01092&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b01092%26sid%3Dliteratum%253Aachs%26aulast%3DYan%26aufirst%3DW.%26aulast%3DLakkaniga%26aufirst%3DN.%2BR.%26aulast%3DCarlomagno%26aufirst%3DF.%26aulast%3DSantoro%26aufirst%3DM.%26aulast%3DMcDonald%26aufirst%3DN.%2BQ.%26aulast%3DLv%26aufirst%3DF.%26aulast%3DGunaganti%26aufirst%3DN.%26aulast%3DFrett%26aufirst%3DB.%26aulast%3DLi%26aufirst%3DH.-y.%26atitle%3DInsights%2520into%2520current%2520tropomyosin%2520receptor%2520kinase%2520%2528TRK%2529%2520inhibitors%253A%2520development%2520and%2520clinical%2520application%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D1731%26epage%3D1760%26doi%3D10.1021%2Facs.jmedchem.8b01092" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Amatu, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sartore-Bianchi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siena, S.</span></span> <span> </span><span class="NLM_article-title">NTRKgene fusions as novel targets of cancer therapy across multiple tumour types</span>. <i>ESMO Open</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>1</i></span>, <span class="NLM_elocation-id">e000023</span> <span class="refDoi"> DOI: 10.1136/esmoopen-2015-000023</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=10.1136%2Fesmoopen-2015-000023" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=27843590" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2016&author=A.+Amatuauthor=A.+Sartore-Bianchiauthor=S.+Siena&title=NTRKgene+fusions+as+novel+targets+of+cancer+therapy+across+multiple+tumour+types&doi=10.1136%2Fesmoopen-2015-000023"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1136%2Fesmoopen-2015-000023&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1136%252Fesmoopen-2015-000023%26sid%3Dliteratum%253Aachs%26aulast%3DAmatu%26aufirst%3DA.%26aulast%3DSartore-Bianchi%26aufirst%3DA.%26aulast%3DSiena%26aufirst%3DS.%26atitle%3DNTRKgene%2520fusions%2520as%2520novel%2520targets%2520of%2520cancer%2520therapy%2520across%2520multiple%2520tumour%2520types%26jtitle%3DESMO%2520Open%26date%3D2016%26volume%3D1%26doi%3D10.1136%2Fesmoopen-2015-000023" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Khotskaya, Y. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holla, V. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farago, A. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mills Shaw, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meric-Bernstam, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hong, D. S.</span></span> <span> </span><span class="NLM_article-title">Targeting TRK family proteins in cancer</span>. <i>Pharmacol. Ther.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>173</i></span>,  <span class="NLM_fpage">58</span>– <span class="NLM_lpage">66</span>, <span class="refDoi"> DOI: 10.1016/j.pharmthera.2017.02.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=10.1016%2Fj.pharmthera.2017.02.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=28174090" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=1%3ACAS%3A528%3ADC%252BC2sXisF2jtL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=173&publication_year=2017&pages=58-66&author=Y.+B.+Khotskayaauthor=V.+R.+Hollaauthor=A.+F.+Faragoauthor=K.+R.+Mills+Shawauthor=F.+Meric-Bernstamauthor=D.+S.+Hong&title=Targeting+TRK+family+proteins+in+cancer&doi=10.1016%2Fj.pharmthera.2017.02.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting TRK family proteins in cancer</span></div><div class="casAuthors">Khotskaya, Yekaterina B.; Holla, Vijaykumar R.; Farago, Anna F.; Mills Shaw, Kenna R.; Meric-Bernstam, Funda; Hong, David S.</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacology & Therapeutics</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">173</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">58-66</span>CODEN:
                <span class="NLM_cas:coden">PHTHDT</span>;
        ISSN:<span class="NLM_cas:issn">0163-7258</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">The tropomyosin receptor kinase (TRK) family includes TRKA, TRKB, and TRKC proteins, which are encoded by NTRK1, NTRK2 and NTRK3 genes, resp.  Binding of neurotrophins to TRK proteins induces receptor dimerization, phosphorylation, and activation of the downstream signaling cascades via PI3K, RAS/MAPK/ERK, and PLC-gamma.  TRK pathway aberrations, including gene fusions, protein overexpression, and single nucleotide alterations, have been implicated in the pathogenesis of many cancer types, with NTRK gene fusions being the most well validated oncogenic events to date.  Although the NTRK gene fusions are infrequent in most cancer types, certain rare tumor types are predominately driven by these events.  Conversely, in more common histologies, such as lung and colorectal cancers, prevalence of the NTRK fusions is well below 5%.  Selective inhibition of TRK signaling may therefore be beneficial among patients whose tumors vary in histologies, but share underlying oncogenic NTRK gene alterations.  Currently, several TRK-targeting compds. are in clin. development.  The ongoing Phase 2 trials with entrectinib and LOXO-101, two of the leading TRK inhibitors, are designed as 'basket trials', inclusive of patients whose tumors harbor NTRK gene fusions, independent of histol.  Addnl. Phase 1 studies of other TRK inhibitors, including MGCD516, PLX7486, DS-6051b, and TSR-011, are underway.  Interim data examg. NTRK-rearranged tumors treated with entrectinib or LOXO-101 demonstrate encouraging activity, with patients achieving rapid and durable responses.  Consequently, both drugs have achieved orphan designation from regulatory agencies, and efforts are underway to further expedite their development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGozoy77YmUc_LVg90H21EOLACvtfcHk0lgur0hx_xDapw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXisF2jtL8%253D&md5=f18f232e6d3c4c9a5d2e398fe60ae7e4</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1016%2Fj.pharmthera.2017.02.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.pharmthera.2017.02.006%26sid%3Dliteratum%253Aachs%26aulast%3DKhotskaya%26aufirst%3DY.%2BB.%26aulast%3DHolla%26aufirst%3DV.%2BR.%26aulast%3DFarago%26aufirst%3DA.%2BF.%26aulast%3DMills%2BShaw%26aufirst%3DK.%2BR.%26aulast%3DMeric-Bernstam%26aufirst%3DF.%26aulast%3DHong%26aufirst%3DD.%2BS.%26atitle%3DTargeting%2520TRK%2520family%2520proteins%2520in%2520cancer%26jtitle%3DPharmacol.%2520Ther.%26date%3D2017%26volume%3D173%26spage%3D58%26epage%3D66%26doi%3D10.1016%2Fj.pharmthera.2017.02.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Greco, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miranda, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pierotti, M. A.</span></span> <span> </span><span class="NLM_article-title">Rearrangements of NTRK1 gene in papillary thyroid carcinoma</span>. <i>Mol. Cell. Endocrinol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>321</i></span>,  <span class="NLM_fpage">44</span>– <span class="NLM_lpage">49</span>, <span class="refDoi"> DOI: 10.1016/j.mce.2009.10.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=10.1016%2Fj.mce.2009.10.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=19883730" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=1%3ACAS%3A528%3ADC%252BC3cXksVamu7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=321&publication_year=2010&pages=44-49&author=A.+Grecoauthor=C.+Mirandaauthor=M.+A.+Pierotti&title=Rearrangements+of+NTRK1+gene+in+papillary+thyroid+carcinoma&doi=10.1016%2Fj.mce.2009.10.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Rearrangements of NTRK1 gene in papillary thyroid carcinoma</span></div><div class="casAuthors">Greco, A.; Miranda, C.; Pierotti, M. A.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Endocrinology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">321</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">44-49</span>CODEN:
                <span class="NLM_cas:coden">MCEND6</span>;
        ISSN:<span class="NLM_cas:issn">0303-7207</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ireland Ltd.</span>)
        </div><div class="casAbstract">A review.  TRK oncogenes are obsd. in a consistent fraction of papillary thyroid carcinoma (PTC); they arise from the fusion of the 3' terminal sequences of the NTRK1/NGF receptor gene with 5' terminal sequences of various activating genes, such as TPM3, TPR and TFG.  TRK oncoproteins display constitutive tyrosine-kinase activity, leading to in vitro and in vivo transformation.  In this review studies performed during the last 20 years will be summarized.  The following topics will be illustrated: (a) frequency of TRK oncogenes and correlation with radiation and tumor histopathol. features; (b) mol. mechanisms underlying NTRK1 oncogenic rearrangements; (c) mol. and biochem. characterization of TRK oncoproteins, and their mechanism of action; (d) role of activating sequences in the activation of TRK oncoproteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrnGkkW9wYpTLVg90H21EOLACvtfcHk0lgur0hx_xDapw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXksVamu7g%253D&md5=ea9abddd4191599672958ac3875bf9ae</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1016%2Fj.mce.2009.10.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.mce.2009.10.009%26sid%3Dliteratum%253Aachs%26aulast%3DGreco%26aufirst%3DA.%26aulast%3DMiranda%26aufirst%3DC.%26aulast%3DPierotti%26aufirst%3DM.%2BA.%26atitle%3DRearrangements%2520of%2520NTRK1%2520gene%2520in%2520papillary%2520thyroid%2520carcinoma%26jtitle%3DMol.%2520Cell.%2520Endocrinol.%26date%3D2010%26volume%3D321%26spage%3D44%26epage%3D49%26doi%3D10.1016%2Fj.mce.2009.10.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Greco, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miranda, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pierotti, M. A.</span></span> <span> </span><span class="NLM_article-title">Rearrangements of NTRK1 gene in papillary thyroid carcinoma</span>. <i>Mol. Cell. Endocrinol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>321</i></span>,  <span class="NLM_fpage">44</span>– <span class="NLM_lpage">49</span>, <span class="refDoi"> DOI: 10.1016/j.mce.2009.10.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=10.1016%2Fj.mce.2009.10.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=19883730" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=1%3ACAS%3A528%3ADC%252BC3cXksVamu7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=321&publication_year=2010&pages=44-49&author=A.+Grecoauthor=C.+Mirandaauthor=M.+A.+Pierotti&title=Rearrangements+of+NTRK1+gene+in+papillary+thyroid+carcinoma&doi=10.1016%2Fj.mce.2009.10.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Rearrangements of NTRK1 gene in papillary thyroid carcinoma</span></div><div class="casAuthors">Greco, A.; Miranda, C.; Pierotti, M. A.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Endocrinology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">321</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">44-49</span>CODEN:
                <span class="NLM_cas:coden">MCEND6</span>;
        ISSN:<span class="NLM_cas:issn">0303-7207</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ireland Ltd.</span>)
        </div><div class="casAbstract">A review.  TRK oncogenes are obsd. in a consistent fraction of papillary thyroid carcinoma (PTC); they arise from the fusion of the 3' terminal sequences of the NTRK1/NGF receptor gene with 5' terminal sequences of various activating genes, such as TPM3, TPR and TFG.  TRK oncoproteins display constitutive tyrosine-kinase activity, leading to in vitro and in vivo transformation.  In this review studies performed during the last 20 years will be summarized.  The following topics will be illustrated: (a) frequency of TRK oncogenes and correlation with radiation and tumor histopathol. features; (b) mol. mechanisms underlying NTRK1 oncogenic rearrangements; (c) mol. and biochem. characterization of TRK oncoproteins, and their mechanism of action; (d) role of activating sequences in the activation of TRK oncoproteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrnGkkW9wYpTLVg90H21EOLACvtfcHk0ljxIHUV_w0Rtw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXksVamu7g%253D&md5=ea9abddd4191599672958ac3875bf9ae</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1016%2Fj.mce.2009.10.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.mce.2009.10.009%26sid%3Dliteratum%253Aachs%26aulast%3DGreco%26aufirst%3DA.%26aulast%3DMiranda%26aufirst%3DC.%26aulast%3DPierotti%26aufirst%3DM.%2BA.%26atitle%3DRearrangements%2520of%2520NTRK1%2520gene%2520in%2520papillary%2520thyroid%2520carcinoma%26jtitle%3DMol.%2520Cell.%2520Endocrinol.%26date%3D2010%26volume%3D321%26spage%3D44%26epage%3D49%26doi%3D10.1016%2Fj.mce.2009.10.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liebers, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhelyazkova, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Panditi, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lynch, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robinson, H. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shim, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chmielecki, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pao, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Engelman, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iafrate, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Le, L. P.</span></span> <span> </span><span class="NLM_article-title">Anchored multiplex PCR for targeted next-generation sequencing</span>. <i>Nat. Med.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">1479</span>– <span class="NLM_lpage">1484</span>, <span class="refDoi"> DOI: 10.1038/nm.3729</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=10.1038%2Fnm.3729" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=25384085" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvF2js7jF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2014&pages=1479-1484&author=Z.+Zhengauthor=M.+Liebersauthor=B.+Zhelyazkovaauthor=Y.+Caoauthor=D.+Panditiauthor=K.+D.+Lynchauthor=J.+Chenauthor=H.+E.+Robinsonauthor=H.+S.+Shimauthor=J.+Chmieleckiauthor=W.+Paoauthor=J.+A.+Engelmanauthor=A.+J.+Iafrateauthor=L.+P.+Le&title=Anchored+multiplex+PCR+for+targeted+next-generation+sequencing&doi=10.1038%2Fnm.3729"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Anchored multiplex PCR for targeted next-generation sequencing</span></div><div class="casAuthors">Zheng, Zongli; Liebers, Matthew; Zhelyazkova, Boryana; Cao, Yi; Panditi, Divya; Lynch, Kerry D.; Chen, Juxiang; Robinson, Hayley E.; Shim, Hyo Sup; Chmielecki, Juliann; Pao, William; Engelman, Jeffrey A.; Iafrate, A. John; Le, Long Phi</div><div class="citationInfo"><span class="NLM_cas:title">Nature Medicine (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1479-1484</span>CODEN:
                <span class="NLM_cas:coden">NAMEFI</span>;
        ISSN:<span class="NLM_cas:issn">1078-8956</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">We describe a rapid target enrichment method for next-generation sequencing, termed anchored multiplex PCR (AMP), that is compatible with low nucleic acid input from formalin-fixed paraffin-embedded (FFPE) specimens.  AMP is effective in detecting gene rearrangements (without prior knowledge of the fusion partners), single nucleotide variants, insertions, deletions and copy no. changes.  Validation of a gene rearrangement panel using 319 FFPE samples showed 100% sensitivity (95% confidence limit: 96.5-100%) and 100% specificity (95% confidence limit: 99.3-100%) compared with ref. assays.  On the basis of our experience with performing AMP on 986 clin. FFPE samples, we show its potential as both a robust clin. assay and a powerful discovery tool, which we used to identify new therapeutically important gene fusions: ARHGEF2-NTRK1 and CHTOP-NTRK1 in glioblastoma, MSN-ROS1, TRIM4-BRAF, VAMP2-NRG1, TPM3-NTRK1 and RUFY2-RET in lung cancer, FGFR2-CREB5 in cholangiocarcinoma and PPL-NTRK1 in thyroid carcinoma.  AMP is a scalable and efficient next-generation sequencing target enrichment method for research and clin. applications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoBvfrWVSNX4rVg90H21EOLACvtfcHk0ljxIHUV_w0Rtw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvF2js7jF&md5=77c01e9735858185681607b148ad8192</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1038%2Fnm.3729&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnm.3729%26sid%3Dliteratum%253Aachs%26aulast%3DZheng%26aufirst%3DZ.%26aulast%3DLiebers%26aufirst%3DM.%26aulast%3DZhelyazkova%26aufirst%3DB.%26aulast%3DCao%26aufirst%3DY.%26aulast%3DPanditi%26aufirst%3DD.%26aulast%3DLynch%26aufirst%3DK.%2BD.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DRobinson%26aufirst%3DH.%2BE.%26aulast%3DShim%26aufirst%3DH.%2BS.%26aulast%3DChmielecki%26aufirst%3DJ.%26aulast%3DPao%26aufirst%3DW.%26aulast%3DEngelman%26aufirst%3DJ.%2BA.%26aulast%3DIafrate%26aufirst%3DA.%2BJ.%26aulast%3DLe%26aufirst%3DL.%2BP.%26atitle%3DAnchored%2520multiplex%2520PCR%2520for%2520targeted%2520next-generation%2520sequencing%26jtitle%3DNat.%2520Med.%26date%3D2014%26volume%3D20%26spage%3D1479%26epage%3D1484%26doi%3D10.1038%2Fnm.3729" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vaishnavi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Capelletti, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Le, A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kako, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Butaney, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ercan, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mahale, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davies, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aisner, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pilling, A. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berge, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sasaki, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, S.-i.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kryukov, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garraway, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hammerman, P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haas, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andrews, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lipson, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stephens, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, V. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Varella-Garcia, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jänne, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doebele, R. C.</span></span> <span> </span><span class="NLM_article-title">Oncogenic and drug-sensitive NTRK1 rearrangements in lung cancer</span>. <i>Nat. Med.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">1469</span>– <span class="NLM_lpage">1472</span>, <span class="refDoi"> DOI: 10.1038/nm.3352</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=10.1038%2Fnm.3352" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=24162815" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhs1yiurrI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2013&pages=1469-1472&author=A.+Vaishnaviauthor=M.+Capellettiauthor=A.+T.+Leauthor=S.+Kakoauthor=M.+Butaneyauthor=D.+Ercanauthor=S.+Mahaleauthor=K.+D.+Daviesauthor=D.+L.+Aisnerauthor=A.+B.+Pillingauthor=E.+M.+Bergeauthor=J.+Kimauthor=H.+Sasakiauthor=S.-i.+Parkauthor=G.+Kryukovauthor=L.+A.+Garrawayauthor=P.+S.+Hammermanauthor=J.+Haasauthor=S.+W.+Andrewsauthor=D.+Lipsonauthor=P.+J.+Stephensauthor=V.+A.+Millerauthor=M.+Varella-Garciaauthor=P.+A.+J%C3%A4nneauthor=R.+C.+Doebele&title=Oncogenic+and+drug-sensitive+NTRK1+rearrangements+in+lung+cancer&doi=10.1038%2Fnm.3352"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Oncogenic and drug-sensitive NTRK1 rearrangements in lung cancer</span></div><div class="casAuthors">Vaishnavi, Aria; Capelletti, Marzia; Le, Anh T.; Kako, Severine; Butaney, Mohit; Ercan, Dalia; Mahale, Sakshi; Davies, Kurtis D.; Aisner, Dara L.; Pilling, Amanda B.; Berge, Eamon M.; Kim, Jhingook; Sasaki, Hidefumi; Park, Seung-il; Kryukov, Gregory; Garraway, Levi A.; Hammerman, Peter S.; Haas, Julia; Andrews, Steven W.; Lipson, Doron; Stephens, Philip J.; Miller, Vince A.; Varella-Garcia, Marileila; Jaenne, Pasi A.; Doebele, Robert C.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Medicine (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1469-1472</span>CODEN:
                <span class="NLM_cas:coden">NAMEFI</span>;
        ISSN:<span class="NLM_cas:issn">1078-8956</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">We identified new gene fusions in patients with lung cancer harboring the kinase domain of the NTRK1 gene that encodes the high-affinity nerve growth factor receptor (TRKA protein).  Both the MPRIP-NTRK1 and CD74-NTRK1 fusions lead to constitutive TRKA kinase activity and are oncogenic.  Treatment of cells expressing NTRK1 fusions with inhibitors of TRKA kinase activity inhibited autophosphorylation of TRKA and cell growth.  Tumor samples from 3 of 91 patients with lung cancer (3.3%) without known oncogenic alterations assayed by next-generation sequencing or fluorescence in situ hybridization demonstrated evidence of NTRK1 gene fusions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpBHNoZGMQLMrVg90H21EOLACvtfcHk0ljxIHUV_w0Rtw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhs1yiurrI&md5=23b413826ea700bd831eadbfde270615</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1038%2Fnm.3352&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnm.3352%26sid%3Dliteratum%253Aachs%26aulast%3DVaishnavi%26aufirst%3DA.%26aulast%3DCapelletti%26aufirst%3DM.%26aulast%3DLe%26aufirst%3DA.%2BT.%26aulast%3DKako%26aufirst%3DS.%26aulast%3DButaney%26aufirst%3DM.%26aulast%3DErcan%26aufirst%3DD.%26aulast%3DMahale%26aufirst%3DS.%26aulast%3DDavies%26aufirst%3DK.%2BD.%26aulast%3DAisner%26aufirst%3DD.%2BL.%26aulast%3DPilling%26aufirst%3DA.%2BB.%26aulast%3DBerge%26aufirst%3DE.%2BM.%26aulast%3DKim%26aufirst%3DJ.%26aulast%3DSasaki%26aufirst%3DH.%26aulast%3DPark%26aufirst%3DS.-i.%26aulast%3DKryukov%26aufirst%3DG.%26aulast%3DGarraway%26aufirst%3DL.%2BA.%26aulast%3DHammerman%26aufirst%3DP.%2BS.%26aulast%3DHaas%26aufirst%3DJ.%26aulast%3DAndrews%26aufirst%3DS.%2BW.%26aulast%3DLipson%26aufirst%3DD.%26aulast%3DStephens%26aufirst%3DP.%2BJ.%26aulast%3DMiller%26aufirst%3DV.%2BA.%26aulast%3DVarella-Garcia%26aufirst%3DM.%26aulast%3DJ%25C3%25A4nne%26aufirst%3DP.%2BA.%26aulast%3DDoebele%26aufirst%3DR.%2BC.%26atitle%3DOncogenic%2520and%2520drug-sensitive%2520NTRK1%2520rearrangements%2520in%2520lung%2520cancer%26jtitle%3DNat.%2520Med.%26date%3D2013%26volume%3D19%26spage%3D1469%26epage%3D1472%26doi%3D10.1038%2Fnm.3352" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sartore-Bianchi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ardini, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bosotti, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amatu, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valtorta, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Somaschini, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raddrizzani, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Palmeri, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Banfi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bonazzina, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Misale, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marrapese, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leone, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alzani, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hornby, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lim, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Veronese, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vanzulli, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bardelli, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martignoni, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davite, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galvani, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Isacchi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siena, S.</span></span> <span> </span><span class="NLM_article-title">Sensitivity to entrectinib associated with a novel LMNA-NTRK1 gene fusion in metastatic colorectal cancer</span>. <i>J. Natl. Cancer Inst.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>108</i></span>,  <span class="NLM_fpage">djv306</span>, <span class="refDoi"> DOI: 10.1093/jnci/djv306</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=10.1093%2Fjnci%2Fdjv306" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=26563355" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=108&publication_year=2016&pages=djv306&author=A.+Sartore-Bianchiauthor=E.+Ardiniauthor=R.+Bosottiauthor=A.+Amatuauthor=E.+Valtortaauthor=A.+Somaschiniauthor=L.+Raddrizzaniauthor=L.+Palmeriauthor=P.+Banfiauthor=E.+Bonazzinaauthor=S.+Misaleauthor=G.+Marrapeseauthor=A.+Leoneauthor=R.+Alzaniauthor=D.+Luoauthor=Z.+Hornbyauthor=J.+Limauthor=S.+Veroneseauthor=A.+Vanzulliauthor=A.+Bardelliauthor=M.+Martignoniauthor=C.+Daviteauthor=A.+Galvaniauthor=A.+Isacchiauthor=S.+Siena&title=Sensitivity+to+entrectinib+associated+with+a+novel+LMNA-NTRK1+gene+fusion+in+metastatic+colorectal+cancer&doi=10.1093%2Fjnci%2Fdjv306"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1093%2Fjnci%2Fdjv306&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fjnci%252Fdjv306%26sid%3Dliteratum%253Aachs%26aulast%3DSartore-Bianchi%26aufirst%3DA.%26aulast%3DArdini%26aufirst%3DE.%26aulast%3DBosotti%26aufirst%3DR.%26aulast%3DAmatu%26aufirst%3DA.%26aulast%3DValtorta%26aufirst%3DE.%26aulast%3DSomaschini%26aufirst%3DA.%26aulast%3DRaddrizzani%26aufirst%3DL.%26aulast%3DPalmeri%26aufirst%3DL.%26aulast%3DBanfi%26aufirst%3DP.%26aulast%3DBonazzina%26aufirst%3DE.%26aulast%3DMisale%26aufirst%3DS.%26aulast%3DMarrapese%26aufirst%3DG.%26aulast%3DLeone%26aufirst%3DA.%26aulast%3DAlzani%26aufirst%3DR.%26aulast%3DLuo%26aufirst%3DD.%26aulast%3DHornby%26aufirst%3DZ.%26aulast%3DLim%26aufirst%3DJ.%26aulast%3DVeronese%26aufirst%3DS.%26aulast%3DVanzulli%26aufirst%3DA.%26aulast%3DBardelli%26aufirst%3DA.%26aulast%3DMartignoni%26aufirst%3DM.%26aulast%3DDavite%26aufirst%3DC.%26aulast%3DGalvani%26aufirst%3DA.%26aulast%3DIsacchi%26aufirst%3DA.%26aulast%3DSiena%26aufirst%3DS.%26atitle%3DSensitivity%2520to%2520entrectinib%2520associated%2520with%2520a%2520novel%2520LMNA-NTRK1%2520gene%2520fusion%2520in%2520metastatic%2520colorectal%2520cancer%26jtitle%3DJ.%2520Natl.%2520Cancer%2520Inst.%26date%3D2016%26volume%3D108%26spage%3Ddjv306%26doi%3D10.1093%2Fjnci%2Fdjv306" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Doebele, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, L. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vaishnavi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Le, A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Estrada-Bernal, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keysar, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jimeno, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Varella-Garcia, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aisner, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stephens, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morosini, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tuch, B. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fernandes, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nanda, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Low, J. A.</span></span> <span> </span><span class="NLM_article-title">An oncogenic NTRK fusion in a patient with soft-tissue sarcoma with response to the tropomyosin-related kinase inhibitor LOXO-101</span>. <i>Canc. Discov.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">1049</span>– <span class="NLM_lpage">1057</span>, <span class="refDoi"> DOI: 10.1158/2159-8290.cd-15-0443</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=10.1158%2F2159-8290.CD-15-0443" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=26216294" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhs1Wqs7fF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2015&pages=1049-1057&author=R.+C.+Doebeleauthor=L.+E.+Davisauthor=A.+Vaishnaviauthor=A.+T.+Leauthor=A.+Estrada-Bernalauthor=S.+Keysarauthor=A.+Jimenoauthor=M.+Varella-Garciaauthor=D.+L.+Aisnerauthor=Y.+Liauthor=P.+J.+Stephensauthor=D.+Morosiniauthor=B.+B.+Tuchauthor=M.+Fernandesauthor=N.+Nandaauthor=J.+A.+Low&title=An+oncogenic+NTRK+fusion+in+a+patient+with+soft-tissue+sarcoma+with+response+to+the+tropomyosin-related+kinase+inhibitor+LOXO-101&doi=10.1158%2F2159-8290.cd-15-0443"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">An Oncogenic NTRK Fusion in a Patient with Soft-Tissue Sarcoma with Response to the Tropomyosin-Related Kinase Inhibitor LOXO-101</span></div><div class="casAuthors">Doebele, Robert C.; Davis, Lara E.; Vaishnavi, Aria; Le, Anh T.; Estrada-Bernal, Adriana; Keysar, Stephen; Jimeno, Antonio; Varella-Garcia, Marileila; Aisner, Dara L.; Li, Yali; Stephens, Philip J.; Morosini, Deborah; Tuch, Brian B.; Fernandes, Michele; Nanda, Nisha; Low, Jennifer A.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Discovery</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1049-1057</span>CODEN:
                <span class="NLM_cas:coden">CDAIB2</span>;
        ISSN:<span class="NLM_cas:issn">2159-8274</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Oncogenic TRK fusions induce cancer cell proliferation and engage crit. cancer-related downstream signaling pathways.  These TRK fusions occur rarely, but in a diverse spectrum of tumor histologies.  LOXO-101 is an orally administered inhibitor of the TRK kinase and is highly selective only for the TRK family of receptors.  Preclin. models of LOXO-101 using TRK-fusion-bearing human-derived cancer cell lines demonstrate inhibition of the fusion oncoprotein and cellular proliferation in vitro, and tumor growth in vivo.  The tumor of a 41-yr-old woman with soft-tissue sarcoma metastatic to the lung was found to harbor an LMNA-NTRK1 gene fusion encoding a functional LMNA-TRKA fusion oncoprotein as detd. by an in situ proximity ligation assay.  In a phase I study of LOXO-101 (ClinicalTrials.gov no. NCT02122913), this patient's tumors underwent rapid and substantial tumor regression, with an accompanying improvement in pulmonary dyspnea, oxygen satn., and plasma tumor markers.  Significance: TRK fusions have been deemed putative oncogenic drivers, but their clin. significance remained unclear.  A patient with a metastatic soft-tissue sarcoma with an LMNA-NTRK1 fusion had rapid and substantial tumor regression with a novel, highly selective TRK inhibitor, LOXO-101, providing the first clin. evidence of benefit from inhibiting TRK fusions.  Cancer Discov; 5(10); 1049-57. ©2015 AACR.  This article is highlighted in the In This Issue feature, p.  1005.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqOOz1IuW1mjrVg90H21EOLACvtfcHk0lhmFefV4uJODQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhs1Wqs7fF&md5=52fcc72687bcd256dc54e77936d3ebee</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1158%2F2159-8290.CD-15-0443&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F2159-8290.CD-15-0443%26sid%3Dliteratum%253Aachs%26aulast%3DDoebele%26aufirst%3DR.%2BC.%26aulast%3DDavis%26aufirst%3DL.%2BE.%26aulast%3DVaishnavi%26aufirst%3DA.%26aulast%3DLe%26aufirst%3DA.%2BT.%26aulast%3DEstrada-Bernal%26aufirst%3DA.%26aulast%3DKeysar%26aufirst%3DS.%26aulast%3DJimeno%26aufirst%3DA.%26aulast%3DVarella-Garcia%26aufirst%3DM.%26aulast%3DAisner%26aufirst%3DD.%2BL.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DStephens%26aufirst%3DP.%2BJ.%26aulast%3DMorosini%26aufirst%3DD.%26aulast%3DTuch%26aufirst%3DB.%2BB.%26aulast%3DFernandes%26aufirst%3DM.%26aulast%3DNanda%26aufirst%3DN.%26aulast%3DLow%26aufirst%3DJ.%2BA.%26atitle%3DAn%2520oncogenic%2520NTRK%2520fusion%2520in%2520a%2520patient%2520with%2520soft-tissue%2520sarcoma%2520with%2520response%2520to%2520the%2520tropomyosin-related%2520kinase%2520inhibitor%2520LOXO-101%26jtitle%3DCanc.%2520Discov.%26date%3D2015%26volume%3D5%26spage%3D1049%26epage%3D1057%26doi%3D10.1158%2F2159-8290.cd-15-0443" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hung, Y. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fletcher, C. D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hornick, J. L.</span></span> <span> </span><span class="NLM_article-title">Evaluation of pan-TRK immunohistochemistry in infantile fibrosarcoma, lipofibromatosis-like neural tumour and histological mimics</span>. <i>Histopathology</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>73</i></span>,  <span class="NLM_fpage">634</span>– <span class="NLM_lpage">644</span>, <span class="refDoi"> DOI: 10.1111/his.13666</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=10.1111%2Fhis.13666" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=29863809" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=1%3ACAS%3A280%3ADC%252BC1MbjsVKqsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=73&publication_year=2018&pages=634-644&author=Y.+P.+Hungauthor=C.+D.+M.+Fletcherauthor=J.+L.+Hornick&title=Evaluation+of+pan-TRK+immunohistochemistry+in+infantile+fibrosarcoma%2C+lipofibromatosis-like+neural+tumour+and+histological+mimics&doi=10.1111%2Fhis.13666"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Evaluation of pan-TRK immunohistochemistry in infantile fibrosarcoma, lipofibromatosis-like neural tumour and histological mimics</span></div><div class="casAuthors">Hung Yin P; Fletcher Christopher D M; Hornick Jason L</div><div class="citationInfo"><span class="NLM_cas:title">Histopathology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">73</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">634-644</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">AIMS:  Infantile fibrosarcoma is characterised by intersecting fascicles of spindle cells and ETV6-NTRK3 gene fusion in most cases.  Given histological overlap with other spindle-cell tumours, the diagnosis can be challenging and often requires molecular confirmation.  A recently developed pan-TRK antibody shows promise for identifying tumours with NTRK fusions.  The purpose of this study was to evaluate the potential diagnostic utility of pan-TRK immunohistochemistry for infantile fibrosarcoma.  METHODS AND RESULTS:  We evaluated whole-tissue sections from 210 cases, including 15 infantile fibrosarcomas; five each lipofibromatosis-like neural tumour and lipofibromatosis; 10 each primitive myxoid mesenchymal tumour of infancy (PMMTI) and low-grade myofibroblastic sarcoma; 15 each fibrous hamartoma of infancy (FHI), myofibroma/myofibromatosis and desmoid-type fibromatosis; and 20 each low-grade fibromyxoid sarcoma, synovial sarcoma, spindle-cell rhabdomyosarcoma, malignant peripheral nerve sheath tumour, fibrosarcomatous dermatofibrosarcoma protuberans (F-DFSP) and nodular fasciitis.  Immunohistochemistry was performed using a rabbit monoclonal pan-TRK antibody.  Immunoreactivity for pan-TRK was observed in all 15 (100%) infantile fibrosarcomas, including diffuse immunoreactivity (>50% of cells) in 14 (93%) cases.  Pan-TRK was positive in all five (100%) lipofibromatosis-like neural tumours.  Of the 190 histological mimics, diffuse pan-TRK immunoreactivity was noted in 16 (8%) cases, including five PMMTI, five FHI (highlighting predominantly the primitive myxoid spindle-cell components), three F-DFSP, one low-grade myofibroblastic sarcoma, one myofibroma and one spindle-cell rhabdomyosarcoma.  CONCLUSIONS:  Diffuse pan-TRK immunoreactivity is a highly sensitive but not entirely specific diagnostic marker for infantile fibrosarcoma, and may be helpful in selecting patients for TRK-targeted therapy.  As expected, lipofibromatosis-like neural tumours, which harbour NTRK1 fusions, also show diffuse pan-TRK immunoreactivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQOm9XON-5CyinGiHuhFGv8fW6udTcc2eaiE4Yb70hPZLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1MbjsVKqsg%253D%253D&md5=4e22b595fd2cb8e304991c4922c0f380</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1111%2Fhis.13666&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fhis.13666%26sid%3Dliteratum%253Aachs%26aulast%3DHung%26aufirst%3DY.%2BP.%26aulast%3DFletcher%26aufirst%3DC.%2BD.%2BM.%26aulast%3DHornick%26aufirst%3DJ.%2BL.%26atitle%3DEvaluation%2520of%2520pan-TRK%2520immunohistochemistry%2520in%2520infantile%2520fibrosarcoma%252C%2520lipofibromatosis-like%2520neural%2520tumour%2520and%2520histological%2520mimics%26jtitle%3DHistopathology%26date%3D2018%26volume%3D73%26spage%3D634%26epage%3D644%26doi%3D10.1111%2Fhis.13666" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tognon, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knezevich, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huntsman, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roskelley, C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Melnyk, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mathers, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Becker, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carneiro, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MacPherson, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Horsman, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poremba, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sorensen, P. H. B.</span></span> <span> </span><span class="NLM_article-title">Expression of the ETV6-NTRK3 gene fusion as a primary event in human secretory breast carcinoma</span>. <i>Cancer Cell</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>2</i></span>,  <span class="NLM_fpage">367</span>– <span class="NLM_lpage">376</span>, <span class="refDoi"> DOI: 10.1016/s1535-6108(02)00180-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=10.1016%2FS1535-6108%2802%2900180-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=12450792" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=1%3ACAS%3A528%3ADC%252BD38Xptlalurw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2002&pages=367-376&author=C.+Tognonauthor=S.+R.+Knezevichauthor=D.+Huntsmanauthor=C.+D.+Roskelleyauthor=N.+Melnykauthor=J.+A.+Mathersauthor=L.+Beckerauthor=F.+Carneiroauthor=N.+MacPhersonauthor=D.+Horsmanauthor=C.+Porembaauthor=P.+H.+B.+Sorensen&title=Expression+of+the+ETV6-NTRK3+gene+fusion+as+a+primary+event+in+human+secretory+breast+carcinoma&doi=10.1016%2Fs1535-6108%2802%2900180-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Expression of the ETV6-NTRK3 gene fusion as a primary event in human secretory breast carcinoma</span></div><div class="casAuthors">Tognon, Cristina; Knezevich, Stevan R.; Huntsman, David; Roskelley, Calvin D.; Melnyk, Natalya; Mathers, Joan A.; Becker, Laurence; Carneiro, Fatima; MacPherson, Nicol; Horsman, Doug; Poremba, Christopher; Sorensen, Poul H. B.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">367-376</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">We report that human secretory breast carcinoma (SBC), a rare subtype of infiltrating ductal carcinoma, expresses the ETV6-NTRK3 gene fusion previously cloned in pediatric mesenchymal cancers.  This gene fusion encodes a chimeric tyrosine kinase with potent transforming activity in fibroblasts.  ETV6-NTRK3 expression was confirmed in 12 (92%) of 13 SBC cases, but not in other ductal carcinomas.  Retroviral transfer of ETV6-NTRK3 (EN) into murine mammary epithelial cells resulted in transformed cells that readily formed tumors in nude mice.  Phenotypically, tumors produced glands and expressed epithelial antigens, confirming that EN transformation is compatible with epithelial differentiation.  This represents a recurrent chromosomal rearrangement and expression of a dominantly acting oncogene as a primary event in human breast carcinoma.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp9UOXU8lUATrVg90H21EOLACvtfcHk0lhq7Ciugi84-g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38Xptlalurw%253D&md5=0b6abd134a44218e8ce03ffd25effe2d</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1016%2FS1535-6108%2802%2900180-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1535-6108%252802%252900180-0%26sid%3Dliteratum%253Aachs%26aulast%3DTognon%26aufirst%3DC.%26aulast%3DKnezevich%26aufirst%3DS.%2BR.%26aulast%3DHuntsman%26aufirst%3DD.%26aulast%3DRoskelley%26aufirst%3DC.%2BD.%26aulast%3DMelnyk%26aufirst%3DN.%26aulast%3DMathers%26aufirst%3DJ.%2BA.%26aulast%3DBecker%26aufirst%3DL.%26aulast%3DCarneiro%26aufirst%3DF.%26aulast%3DMacPherson%26aufirst%3DN.%26aulast%3DHorsman%26aufirst%3DD.%26aulast%3DPoremba%26aufirst%3DC.%26aulast%3DSorensen%26aufirst%3DP.%2BH.%2BB.%26atitle%3DExpression%2520of%2520the%2520ETV6-NTRK3%2520gene%2520fusion%2520as%2520a%2520primary%2520event%2520in%2520human%2520secretory%2520breast%2520carcinoma%26jtitle%3DCancer%2520Cell%26date%3D2002%26volume%3D2%26spage%3D367%26epage%3D376%26doi%3D10.1016%2Fs1535-6108%2802%2900180-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Skálová, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vanecek, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simpson, R. H. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laco, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Majewska, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baneckova, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steiner, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michal, M.</span></span> <span> </span><span class="NLM_article-title">Mammary Analogue Secretory Carcinoma of Salivary Glands</span>. <i>Am. J. Surg. Pathol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>40</i></span>,  <span class="NLM_fpage">3</span>– <span class="NLM_lpage">13</span>, <span class="refDoi"> DOI: 10.1097/pas.0000000000000537</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=10.1097%2FPAS.0000000000000537" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=26492182" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=1%3ACAS%3A280%3ADC%252BC28zkt1SmsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2016&pages=3-13&author=A.+Sk%C3%A1lov%C3%A1author=T.+Vanecekauthor=R.+H.+W.+Simpsonauthor=J.+Lacoauthor=H.+Majewskaauthor=M.+Baneckovaauthor=P.+Steinerauthor=M.+Michal&title=Mammary+Analogue+Secretory+Carcinoma+of+Salivary+Glands&doi=10.1097%2Fpas.0000000000000537"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Mammary Analogue Secretory Carcinoma of Salivary Glands: Molecular Analysis of 25 ETV6 Gene Rearranged Tumors With Lack of Detection of Classical ETV6-NTRK3 Fusion Transcript by Standard RT-PCR: Report of 4 Cases Harboring ETV6-X Gene Fusion</span></div><div class="casAuthors">Skalova Alena; Vanecek Tomas; Simpson Roderick H W; Laco Jan; Majewska Hanna; Baneckova Martina; Steiner Petr; Michal Michal</div><div class="citationInfo"><span class="NLM_cas:title">The American journal of surgical pathology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">3-13</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">ETV6 gene abnormalities are well described in tumor pathology.  Many fusion partners of ETV6 have been reported in a variety of epithelial and hematological malignancies.  In salivary gland tumor pathology, however, the ETV6-NTRK3 translocation is specific for mammary analogue secretory carcinoma (MASC), and has not been documented in any other salivary tumor type.  The present study comprised a clinical and molecular analysis of 25 cases morphologically and immunohistochemically typical of MASC.  They all also displayed the ETV6 rearrangement as visualized by fluorescent in situ hybridization but lacked the classical ETV6-NTRK3 fusion transcript by standard reverse-transcriptase-polymerase chain reaction.  In 4 cases, the classical fusion transcript was found by more sensitive, nested reverse-transcription-polymerase chain reaction.  Five other cases harbored atypical fusion transcripts as detected by both standard and nested reverse-transcription-polymerase chain reaction.  In addition, fluorescent in situ hybridization with an NTRK3 break-apart probe was also performed; rearrangement of NTRK3 gene was detected in 16 of 25 cases.  In 3 other cases, the tissue was not analyzable, and in 2 further cases analysis could not be performed because of a lack of appropriate tissue material.  Finally, in the 4 remaining cases whose profile was NTRK3 split-negative and ETV6 split-positive, unknown (non-NTRK) genes appeared to fuse with ETV6 (ETV6-X fusion).  In looking for possible fusion partners, analysis of rearrangement of other kinase genes known to fuse with ETV6 was also performed, but without positive results.  Although numbers were small, correlating the clinico-pathologic features of the 4 ETV6-X fusion tumors and 5 MASC cases with atypical fusion transcripts raises the possibility of that they may behave more aggressively.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRCDWAmzOrORQxyojvcjibhfW6udTcc2eaJVAXbgLIU97ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC28zkt1SmsQ%253D%253D&md5=a2e9b88b4c578008679b67647ef18763</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1097%2FPAS.0000000000000537&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1097%252FPAS.0000000000000537%26sid%3Dliteratum%253Aachs%26aulast%3DSk%25C3%25A1lov%25C3%25A1%26aufirst%3DA.%26aulast%3DVanecek%26aufirst%3DT.%26aulast%3DSimpson%26aufirst%3DR.%2BH.%2BW.%26aulast%3DLaco%26aufirst%3DJ.%26aulast%3DMajewska%26aufirst%3DH.%26aulast%3DBaneckova%26aufirst%3DM.%26aulast%3DSteiner%26aufirst%3DP.%26aulast%3DMichal%26aufirst%3DM.%26atitle%3DMammary%2520Analogue%2520Secretory%2520Carcinoma%2520of%2520Salivary%2520Glands%26jtitle%3DAm.%2520J.%2520Surg.%2520Pathol.%26date%3D2016%26volume%3D40%26spage%3D3%26epage%3D13%26doi%3D10.1097%2Fpas.0000000000000537" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Drilon, A.</span></span> <span> </span><span class="NLM_article-title">TRK inhibitors in TRK fusion-positive cancers</span>. <i>Ann. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">viii23</span>– <span class="NLM_lpage">viii30</span>, <span class="refDoi"> DOI: 10.1093/annonc/mdz282</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=10.1093%2Fannonc%2Fmdz282" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2019&pages=viii23-viii30&author=A.+Drilon&title=TRK+inhibitors+in+TRK+fusion-positive+cancers&doi=10.1093%2Fannonc%2Fmdz282"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1093%2Fannonc%2Fmdz282&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fannonc%252Fmdz282%26sid%3Dliteratum%253Aachs%26aulast%3DDrilon%26aufirst%3DA.%26atitle%3DTRK%2520inhibitors%2520in%2520TRK%2520fusion-positive%2520cancers%26jtitle%3DAnn.%2520Oncol.%26date%3D2019%26volume%3D30%26spage%3Dviii23%26epage%3Dviii30%26doi%3D10.1093%2Fannonc%2Fmdz282" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Scott, L. J.</span></span> <span> </span><span class="NLM_article-title">Larotrectinib: first global approval</span>. <i>Drugs</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>79</i></span>,  <span class="NLM_fpage">201</span>– <span class="NLM_lpage">206</span>, <span class="refDoi"> DOI: 10.1007/s40265-018-1044-x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=10.1007%2Fs40265-018-1044-x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=30635837" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=1%3ACAS%3A528%3ADC%252BC1MXnvFOqsb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=79&publication_year=2019&pages=201-206&author=L.+J.+Scott&title=Larotrectinib%3A+first+global+approval&doi=10.1007%2Fs40265-018-1044-x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Larotrectinib: First Global Approval</span></div><div class="casAuthors">Scott, Lesley J.</div><div class="citationInfo"><span class="NLM_cas:title">Drugs</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">79</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">201-206</span>CODEN:
                <span class="NLM_cas:coden">DRUGAY</span>;
        ISSN:<span class="NLM_cas:issn">0012-6667</span>.
    
            (<span class="NLM_cas:orgname">Springer International Publishing AG</span>)
        </div><div class="casAbstract">A review.  Larotrectinib (Vitrakvi) is an orally administered, small mol., highly-selective, tropomyosin receptor kinase (TRK) inhibitor that was developed by Loxo Oncol. in collaboration with Bayer AG as a treatment for adult and paediatric patients whose cancers harbor neurotrophic receptor tyrosine kinase (NTRK) gene fusions.  In Nov. 2018 larotrectinib received its first global approval in the USA for the treatment of adult and paediatric patients with solid tumors that have a NTRK gene fusion without a known acquired resistance mutation, are metastatic or where surgical resection is likely to result in severe morbidity, and have no satisfactory alternative treatments or that have progressed following treatment.  A registration application for the use of larotrectinib in pediatric and adult patients with locally advanced or metastatic solid tumors with NTRK gene fusion proteins has been submitted in the EU.  This article summarizes the milestones in the development of larotrectinib leading to its first approval for the treatment of adult and pediatric patients with solid tumors that have NTRK gene fusion.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpMpwdBG0H7ELVg90H21EOLACvtfcHk0lhq7Ciugi84-g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXnvFOqsb8%253D&md5=8c3d442665ece609cc09422052903696</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1007%2Fs40265-018-1044-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs40265-018-1044-x%26sid%3Dliteratum%253Aachs%26aulast%3DScott%26aufirst%3DL.%2BJ.%26atitle%3DLarotrectinib%253A%2520first%2520global%2520approval%26jtitle%3DDrugs%26date%3D2019%26volume%3D79%26spage%3D201%26epage%3D206%26doi%3D10.1007%2Fs40265-018-1044-x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Al-Salama, Z. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keam, S. J.</span></span> <span> </span><span class="NLM_article-title">Entrectinib: first global approval</span>. <i>Drugs</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>79</i></span>,  <span class="NLM_fpage">1477</span>– <span class="NLM_lpage">1483</span>, <span class="refDoi"> DOI: 10.1007/s40265-019-01177-y</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=10.1007%2Fs40265-019-01177-y" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=31372957" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=1%3ACAS%3A280%3ADC%252BB3MvktVemsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=79&publication_year=2019&pages=1477-1483&author=Z.+T.+Al-Salamaauthor=S.+J.+Keam&title=Entrectinib%3A+first+global+approval&doi=10.1007%2Fs40265-019-01177-y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Entrectinib: First Global Approval</span></div><div class="casAuthors">Al-Salama Zaina T; Keam Susan J</div><div class="citationInfo"><span class="NLM_cas:title">Drugs</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">79</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">1477-1483</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Entrectinib (Rozlytrek(®)) is an oral selective inhibitor of the tyrosine kinases tropomyosin receptor kinases (Trk)A/B/C [encoded by the genes neurotrophic tyrosine receptor kinase (NTRK) 1, 2 and 3, respectively], c-ros oncogene 1 (ROS1) and anaplastic lymphoma kinase (ALK) with central nervous system (CNS) activity developed by Roche for the treatment of various solid tumours harbouring NTRK1/2/3 or ROS1 gene fusions.  In June 2019, entrectinib received its first global approval in Japan, for the treatment of adult and paediatric patients with NTRK fusion-positive, advanced or recurrent solid tumours and is under regulatory review for the treatment of adult patients with ROS1-positive non-small cell lung cancer (NSCLC).  Entrectinib is also under regulatory review in the USA (PDUFA date 18 August 2019) and EU [Priority Medicines (PRIME) designation] for NTRK-positive solid tumours and ROS1-positive NSCLC.  This article summarizes the milestones in the development of entrectinib leading to this first global approval for solid tumours in Japan.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRqIyGZmvQlpEkUaC_kzhvyfW6udTcc2ebawdBiKIB5nLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3MvktVemsg%253D%253D&md5=750c86d20c43273cf07f505988ac8dbd</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1007%2Fs40265-019-01177-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs40265-019-01177-y%26sid%3Dliteratum%253Aachs%26aulast%3DAl-Salama%26aufirst%3DZ.%2BT.%26aulast%3DKeam%26aufirst%3DS.%2BJ.%26atitle%3DEntrectinib%253A%2520first%2520global%2520approval%26jtitle%3DDrugs%26date%3D2019%26volume%3D79%26spage%3D1477%26epage%3D1483%26doi%3D10.1007%2Fs40265-019-01177-y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sun, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tong, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rao, Y.</span></span> <span> </span><span class="NLM_article-title">PROTACs: great opportunities for academia and industry</span>. <i>Signal Transduction Targeted Ther.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">64</span>, <span class="refDoi"> DOI: 10.1038/s41392-019-0101-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=10.1038%2Fs41392-019-0101-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=31885879" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=1%3ACAS%3A280%3ADC%252BB3MbkvFCrtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2019&pages=64&author=X.+Sunauthor=H.+Gaoauthor=Y.+Yangauthor=M.+Heauthor=Y.+Wuauthor=Y.+Songauthor=Y.+Tongauthor=Y.+Rao&title=PROTACs%3A+great+opportunities+for+academia+and+industry&doi=10.1038%2Fs41392-019-0101-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">PROTACs: great opportunities for academia and industry</span></div><div class="casAuthors">Sun Xiuyun; Gao Hongying; Yang Yiqing; He Ming; Wu Yue; Song Yugang; Tong Yan; Rao Yu; Sun Xiuyun; Gao Hongying; Yang Yiqing; Rao Yu</div><div class="citationInfo"><span class="NLM_cas:title">Signal transduction and targeted therapy</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">64</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Although many kinds of therapies are applied in the clinic, drug-resistance is a major and unavoidable problem.  Another disturbing statistic is the limited number of drug targets, which are presently only 20-25% of all protein targets that are currently being studied.  Moreover, the focus of current explorations of targets are their enzymatic functions, which ignores the functions from their scaffold moiety.  As a promising and appealing technology, PROteolysis TArgeting Chimeras (PROTACs) have attracted great attention both from academia and industry for finding available approaches to solve the above problems.  PROTACs regulate protein function by degrading target proteins instead of inhibiting them, providing more sensitivity to drug-resistant targets and a greater chance to affect the nonenzymatic functions.  PROTACs have been proven to show better selectivity compared to classic inhibitors.  PROTACs can be described as a chemical knockdown approach with rapidity and reversibility, which presents new and different biology compared to other gene editing tools by avoiding misinterpretations that arise from potential genetic compensation and/or spontaneous mutations.  PRTOACs have been widely explored throughout the world and have outperformed not only in cancer diseases, but also in immune disorders, viral infections and neurodegenerative diseases.  Although PROTACs present a very promising and powerful approach for crossing the hurdles of present drug discovery and tool development in biology, more efforts are needed to gain to get deeper insight into the efficacy and safety of PROTACs in the clinic.  More target binders and more E3 ligases applicable for developing PROTACs are waiting for exploration.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQretvERU2xT-_Q9URG6Dx8fW6udTcc2ebawdBiKIB5nLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3MbkvFCrtA%253D%253D&md5=7d9545a21e3a78cededf2977a31e8f7d</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1038%2Fs41392-019-0101-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41392-019-0101-6%26sid%3Dliteratum%253Aachs%26aulast%3DSun%26aufirst%3DX.%26aulast%3DGao%26aufirst%3DH.%26aulast%3DYang%26aufirst%3DY.%26aulast%3DHe%26aufirst%3DM.%26aulast%3DWu%26aufirst%3DY.%26aulast%3DSong%26aufirst%3DY.%26aulast%3DTong%26aufirst%3DY.%26aulast%3DRao%26aufirst%3DY.%26atitle%3DPROTACs%253A%2520great%2520opportunities%2520for%2520academia%2520and%2520industry%26jtitle%3DSignal%2520Transduction%2520Targeted%2520Ther.%26date%3D2019%26volume%3D4%26spage%3D64%26doi%3D10.1038%2Fs41392-019-0101-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rao, Y.</span></span> <span> </span><span class="NLM_article-title">PROTAC technology: opportunities and challenges</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">237</span>– <span class="NLM_lpage">240</span>, <span class="refDoi"> DOI: 10.1021/acsmedchemlett.9b00597</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.9b00597" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=1%3ACAS%3A280%3ADC%252BB383kslaitA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2020&pages=237-240&author=H.+Gaoauthor=X.+Sunauthor=Y.+Rao&title=PROTAC+technology%3A+opportunities+and+challenges&doi=10.1021%2Facsmedchemlett.9b00597"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">PROTAC Technology: Opportunities and Challenges</span></div><div class="casAuthors">Gao Hongying; Sun Xiuyun; Rao Yu; Gao Hongying; Sun Xiuyun</div><div class="citationInfo"><span class="NLM_cas:title">ACS medicinal chemistry letters</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">237-240</span>
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    </div><div class="casAbstract">PROTACs-induced targeted protein degradation has emerged as a novel therapeutic strategy in drug development and attracted the favor of academic institutions, large pharmaceutical enterprises (e.g., AstraZeneca, Bayer, Novartis, Amgen, Pfizer, GlaxoSmithKline, Merck, and Boehringer Ingelheim, etc.), and biotechnology companies.  PROTACs opened a new chapter for novel drug development.  However, any new technology will face many new problems and challenges.  Perspectives on the potential opportunities and challenges of PROTACs will contribute to the research and development of new protein degradation drugs and degrader tools.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSj8qlWczrRKmx8TEN9K18QfW6udTcc2eYFiW4nDW67trntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB383kslaitA%253D%253D&md5=68988d6477e83f4ebc26d11e3b8ce5b1</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.9b00597&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.9b00597%26sid%3Dliteratum%253Aachs%26aulast%3DGao%26aufirst%3DH.%26aulast%3DSun%26aufirst%3DX.%26aulast%3DRao%26aufirst%3DY.%26atitle%3DPROTAC%2520technology%253A%2520opportunities%2520and%2520challenges%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2020%26volume%3D11%26spage%3D237%26epage%3D240%26doi%3D10.1021%2Facsmedchemlett.9b00597" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lai, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Induced protein degradation: an emerging drug discovery paradigm</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">101</span>– <span class="NLM_lpage">114</span>, <span class="refDoi"> DOI: 10.1038/nrd.2016.211</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=10.1038%2Fnrd.2016.211" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=27885283" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvFCit7rP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2017&pages=101-114&author=A.+C.+Laiauthor=C.+M.+Crews&title=Induced+protein+degradation%3A+an+emerging+drug+discovery+paradigm&doi=10.1038%2Fnrd.2016.211"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Induced protein degradation: an emerging drug discovery paradigm</span></div><div class="casAuthors">Lai, Ashton C.; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">101-114</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Small-mol. drug discovery has traditionally focused on occupancy of a binding site that directly affects protein function, and this approach typically precludes targeting proteins that lack such amenable sites.  Furthermore, high systemic drug exposures may be needed to maintain sufficient target inhibition in vivo, increasing the risk of undesirable off-target effects.  Induced protein degrdn. is an alternative approach that is event-driven: upon drug binding, the target protein is tagged for elimination.  Emerging technologies based on proteolysis-targeting chimaeras (PROTACs) that exploit cellular quality control machinery to selectively degrade target proteins are attracting considerable attention in the pharmaceutical industry owing to the advantages they could offer over traditional small-mol. strategies.  These advantages include the potential to reduce systemic drug exposure, the ability to counteract increased target protein expression that often accompanies inhibition of protein function and the potential ability to target proteins that are not currently therapeutically tractable, such as transcription factors, scaffolding and regulatory proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoj-O7zXGOpXLVg90H21EOLACvtfcHk0lgGtoDE2irpIw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvFCit7rP&md5=a01ff43e9b696fe4dc03f0815158a98f</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2016.211&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2016.211%26sid%3Dliteratum%253Aachs%26aulast%3DLai%26aufirst%3DA.%2BC.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DInduced%2520protein%2520degradation%253A%2520an%2520emerging%2520drug%2520discovery%2520paradigm%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2017%26volume%3D16%26spage%3D101%26epage%3D114%26doi%3D10.1038%2Fnrd.2016.211" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schapira, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Calabrese, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bullock, A. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Targeted protein degradation: expanding the toolbox</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">949</span>– <span class="NLM_lpage">963</span>, <span class="refDoi"> DOI: 10.1038/s41573-019-0047-y</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=10.1038%2Fs41573-019-0047-y" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=31666732" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitVKrtLrM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2019&pages=949-963&author=M.+Schapiraauthor=M.+F.+Calabreseauthor=A.+N.+Bullockauthor=C.+M.+Crews&title=Targeted+protein+degradation%3A+expanding+the+toolbox&doi=10.1038%2Fs41573-019-0047-y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Targeted protein degradation: expanding the toolbox</span></div><div class="casAuthors">Schapira, Matthieu; Calabrese, Matthew F.; Bullock, Alex N.; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">949-963</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">A review. Proteolysis-targeting chimeras (PROTACs) and related mols. that induce targeted protein degrdn. by the ubiquitin-proteasome system represent a new therapeutic modality and are the focus of great interest, owing to potential advantages over traditional occupancy-based inhibitors with respect to dosing, side effects, drug resistance and modulating 'undruggable' targets.  However, the technol. is still maturing, and the design elements for successful PROTAC-based drugs are currently being elucidated.  Importantly, fewer than 10 of the more than 600 E3 ubiquitin ligases have so far been exploited for targeted protein degrdn., and expansion of knowledge in this area is a key opportunity.  Here, we briefly discuss lessons learned about targeted protein degrdn. in chem. biol. and drug discovery and systematically review the expression profile, domain architecture and chem. tractability of human E3 ligases that could expand the toolbox for PROTAC discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpwUvM27MU3WrVg90H21EOLACvtfcHk0lgGtoDE2irpIw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitVKrtLrM&md5=9bfd48c6c47fc1cce15129495fea7079</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1038%2Fs41573-019-0047-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41573-019-0047-y%26sid%3Dliteratum%253Aachs%26aulast%3DSchapira%26aufirst%3DM.%26aulast%3DCalabrese%26aufirst%3DM.%2BF.%26aulast%3DBullock%26aufirst%3DA.%2BN.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DTargeted%2520protein%2520degradation%253A%2520expanding%2520the%2520toolbox%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2019%26volume%3D18%26spage%3D949%26epage%3D963%26doi%3D10.1038%2Fs41573-019-0047-y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ohoka, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shibata, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hattori, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Naito, M.</span></span> <span> </span><span class="NLM_article-title">Protein knockdown technology: application of ubiquitin ligase to cancer therapy</span>. <i>Curr. Cancer Drug Targets</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">136</span>– <span class="NLM_lpage">146</span>, <span class="refDoi"> DOI: 10.2174/1568009616666151112122502</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=10.2174%2F1568009616666151112122502" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=26560118" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=1%3ACAS%3A528%3ADC%252BC28XnvFOqtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2016&pages=136-146&author=N.+Ohokaauthor=N.+Shibataauthor=T.+Hattoriauthor=M.+Naito&title=Protein+knockdown+technology%3A+application+of+ubiquitin+ligase+to+cancer+therapy&doi=10.2174%2F1568009616666151112122502"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Protein Knockdown Technology: Application of Ubiquitin Ligase to Cancer Therapy</span></div><div class="casAuthors">Ohoka, Nobumichi; Shibata, Norihito; Hattori, Takayuki; Naito, Mikihiko</div><div class="citationInfo"><span class="NLM_cas:title">Current Cancer Drug Targets</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">136-146</span>CODEN:
                <span class="NLM_cas:coden">CCDTB9</span>;
        ISSN:<span class="NLM_cas:issn">1568-0096</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">Selective degrdn. of pathogenic proteins by small mols. in cells is a novel approach for development of therapeutic agents against various diseases, including cancer.  We and others have developed a protein knockdown technol. with a series of hybrid small compds., called SNIPERs (Specific and Nongenetic IAP-dependent Protein ERasers); and peptidic chimeric mols., called PROTACs (proteolysis-targeting chimeric mols.), which induce selective degrdn. of target proteins via the ubiquitin-proteasome pathway.  These compds. include two different ligands connected by a linker; one is a ligand for a ubiquitin ligase and the other is a ligand for the target protein, which are expected to crosslink these proteins in cells.  Theor., any cytosolic protein can be targeted for degrdn. by this technol.  To date, several SNIPERs and PROTACs against various oncogenic proteins have been developed, which specifically induce polyubiquitylation and proteasomal degrdn. of the oncogenic proteins, resulting in cell death, growth arrest, or impaired migration of cancer cells.  Thus, this protein knockdown technol. has a great potential for cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrmS_kGOWfeuLVg90H21EOLACvtfcHk0lgGtoDE2irpIw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XnvFOqtw%253D%253D&md5=d78f1ae70ea89843d9f946454cc95058</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.2174%2F1568009616666151112122502&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F1568009616666151112122502%26sid%3Dliteratum%253Aachs%26aulast%3DOhoka%26aufirst%3DN.%26aulast%3DShibata%26aufirst%3DN.%26aulast%3DHattori%26aufirst%3DT.%26aulast%3DNaito%26aufirst%3DM.%26atitle%3DProtein%2520knockdown%2520technology%253A%2520application%2520of%2520ubiquitin%2520ligase%2520to%2520cancer%2520therapy%26jtitle%3DCurr.%2520Cancer%2520Drug%2520Targets%26date%3D2016%26volume%3D16%26spage%3D136%26epage%3D146%26doi%3D10.2174%2F1568009616666151112122502" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cromm, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Samarasinghe, K. T. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hines, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Addressing kinase-independent functions of Fak via PROTAC-mediated degradation</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>140</i></span>,  <span class="NLM_fpage">17019</span>– <span class="NLM_lpage">17026</span>, <span class="refDoi"> DOI: 10.1021/jacs.8b08008</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.8b08008" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=1%3ACAS%3A528%3ADC%252BC1cXit1CitLnF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=140&publication_year=2018&pages=17019-17026&author=P.+M.+Crommauthor=K.+T.+G.+Samarasingheauthor=J.+Hinesauthor=C.+M.+Crews&title=Addressing+kinase-independent+functions+of+Fak+via+PROTAC-mediated+degradation&doi=10.1021%2Fjacs.8b08008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Addressing Kinase-Independent Functions of Fak via PROTAC-Mediated Degradation</span></div><div class="casAuthors">Cromm, Philipp M.; Samarasinghe, Kusal T. G.; Hines, John; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">140</span>
        (<span class="NLM_cas:issue">49</span>),
    <span class="NLM_cas:pages">17019-17026</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Enzymic inhibition proved to be a successful modality for the development of many small-mol. drugs.  In recent years, small-mol.-induced protein degrdn. has emerged as an orthogonal therapeutic strategy that has the potential to expand the druggable target space.  Focal adhesion kinase (Fak) is a key player in tumor invasion and metastasis, acting simultaneously as a kinase and a scaffold for several signaling proteins.  While previous efforts to modulate Fak activity were limited to kinase inhibitors with low success in clin. studies, protein degrdn. offers a possibility to simultaneously block Fak's kinase signaling and scaffolding capabilities.  Here, the authors report the development of a selective and potent Fak degrader, PROTAC-3, which outperforms a clin. candidate, defactinib, with respect to Fak activation as well as Fak-mediated cell migration and invasion.  These results underline the potential that PROTACs offer in expanding the druggable space and controlling protein functions that are not easily addressed by traditional small-mol. therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqTn8r4NmDDObVg90H21EOLACvtfcHk0lhkxrLxmgcWCw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXit1CitLnF&md5=b97b1a646a8f9dbf7a2bed640d0f001d</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1021%2Fjacs.8b08008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.8b08008%26sid%3Dliteratum%253Aachs%26aulast%3DCromm%26aufirst%3DP.%2BM.%26aulast%3DSamarasinghe%26aufirst%3DK.%2BT.%2BG.%26aulast%3DHines%26aufirst%3DJ.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DAddressing%2520kinase-independent%2520functions%2520of%2520Fak%2520via%2520PROTAC-mediated%2520degradation%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2018%26volume%3D140%26spage%3D17019%26epage%3D17026%26doi%3D10.1021%2Fjacs.8b08008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rao, Y.</span></span> <span> </span><span class="NLM_article-title">Design, synthesis, and evaluation of highly potent FAK-targeting PROTACs</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">1855</span>– <span class="NLM_lpage">1862</span>, <span class="refDoi"> DOI: 10.1021/acsmedchemlett.9b00372</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.9b00372" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitVajtrfJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2020&pages=1855-1862&author=H.+Gaoauthor=Y.+Wuauthor=Y.+Sunauthor=Y.+Yangauthor=G.+Zhouauthor=Y.+Rao&title=Design%2C+synthesis%2C+and+evaluation+of+highly+potent+FAK-targeting+PROTACs&doi=10.1021%2Facsmedchemlett.9b00372"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Design, Synthesis, and Evaluation of Highly Potent FAK-Targeting PROTACs</span></div><div class="casAuthors">Gao, Hongying; Wu, Yue; Sun, Yonghui; Yang, Yiqing; Zhou, Guangbiao; Rao, Yu</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1855-1862</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Focal adhesion kinase (FAK), a cytoplasmic protein tyrosine kinase, exerts kinase-dependent enzymic functions and kinase-independent scaffolding functions, both of which are crucial in cancer development, early embryonic development, and reprodn.  However, previous efforts for FAK blocking mainly focus on kinase inhibitors.  Proteolysis targeting chimeras (PROTACs) are heterobifunctional mols. that allow direct post-translational knockdown of proteins via ubiquitination of a target protein by E3 ubiquitin ligase and subsequent proteasomal degrdn.  Here, we designed and synthesized a FAK PROTAC library with FAK inhibitor (PF562271 or VS6063) and CRBN E3 ligand.  A novel FAK-targeting PROTAC, FC-11, showed a rapid and reversible FAK degrdn. with a picomolar of DC50 in various cell lines in vitro, which imply that FAK-PROTACs could be useful as expand tools for studying functions of FAK in biol. system and as potential therapeutic agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoDJ4SVK5xRNrVg90H21EOLACvtfcHk0lhkxrLxmgcWCw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitVajtrfJ&md5=79cacdc66925d21441732c26c9b268d4</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.9b00372&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.9b00372%26sid%3Dliteratum%253Aachs%26aulast%3DGao%26aufirst%3DH.%26aulast%3DWu%26aufirst%3DY.%26aulast%3DSun%26aufirst%3DY.%26aulast%3DYang%26aufirst%3DY.%26aulast%3DZhou%26aufirst%3DG.%26aulast%3DRao%26aufirst%3DY.%26atitle%3DDesign%252C%2520synthesis%252C%2520and%2520evaluation%2520of%2520highly%2520potent%2520FAK-targeting%2520PROTACs%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2020%26volume%3D11%26spage%3D1855%26epage%3D1862%26doi%3D10.1021%2Facsmedchemlett.9b00372" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sun, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hwang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rao, Y.</span></span> <span> </span><span class="NLM_article-title">PROTAC-induced BTK degradation as a novel therapy for mutated BTK C481S induced ibrutinib-resistant B-cell malignancies</span>. <i>Cell Res.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>28</i></span>,  <span class="NLM_fpage">779</span>– <span class="NLM_lpage">781</span>, <span class="refDoi"> DOI: 10.1038/s41422-018-0055-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=10.1038%2Fs41422-018-0055-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=29875397" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=1%3ACAS%3A528%3ADC%252BC1cXht1ektbzO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2018&pages=779-781&author=Y.+Sunauthor=X.+Zhaoauthor=N.+Dingauthor=H.+Gaoauthor=Y.+Wuauthor=Y.+Yangauthor=M.+Zhaoauthor=J.+Hwangauthor=Y.+Songauthor=W.+Liuauthor=Y.+Rao&title=PROTAC-induced+BTK+degradation+as+a+novel+therapy+for+mutated+BTK+C481S+induced+ibrutinib-resistant+B-cell+malignancies&doi=10.1038%2Fs41422-018-0055-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">PROTAC-induced BTK degradation as a novel therapy for mutated BTK C481S induced ibrutinib-resistant B-cell malignancies</span></div><div class="casAuthors">Sun, Yonghui; Zhao, Xingwang; Ding, Ning; Gao, Hongying; Wu, Yue; Yang, Yiqing; Zhao, Meng; Hwang, Jinseok; Song, Yuqin; Liu, Wanli; Rao, Yu</div><div class="citationInfo"><span class="NLM_cas:title">Cell Research</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">779-781</span>CODEN:
                <span class="NLM_cas:coden">CREEB6</span>;
        ISSN:<span class="NLM_cas:issn">1001-0602</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">In this study, for the first time, we report the development of BTK-targeting degraders using the PROTAC strategy.  These PROTACs could ef fi ciently degrade ibrutinib-sensitive BTK-WT (wild type).  More importantly, our newly designed PROTACs also significantly induced the degrdn. of ibrutinib resistant BTK-C481S (50% degrdn. efficiency at 30 nM).  Furthermore, our PROTAC mols. ef fi ciently inhibited cell proliferation and colony formation, while exhibited no obvious inhibition (>1000 nM) of ITK, EGFR, and TEC, which are major off-targets of ibrutinib.  These data demonstrate the strong potential for developing PROTAC-based therapeutic mols.  The results indicate that the PROTAC strategy (protein degrdn. instead of inhibition) could be adopted as a general and powerful therapeutic treatment for drug-resistant cancers in the future.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqg_FLdaUP_HLVg90H21EOLACvtfcHk0lhkxrLxmgcWCw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXht1ektbzO&md5=1cca51ac76ecb3b491da0117b21271fb</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1038%2Fs41422-018-0055-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41422-018-0055-1%26sid%3Dliteratum%253Aachs%26aulast%3DSun%26aufirst%3DY.%26aulast%3DZhao%26aufirst%3DX.%26aulast%3DDing%26aufirst%3DN.%26aulast%3DGao%26aufirst%3DH.%26aulast%3DWu%26aufirst%3DY.%26aulast%3DYang%26aufirst%3DY.%26aulast%3DZhao%26aufirst%3DM.%26aulast%3DHwang%26aufirst%3DJ.%26aulast%3DSong%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DW.%26aulast%3DRao%26aufirst%3DY.%26atitle%3DPROTAC-induced%2520BTK%2520degradation%2520as%2520a%2520novel%2520therapy%2520for%2520mutated%2520BTK%2520C481S%2520induced%2520ibrutinib-resistant%2520B-cell%2520malignancies%26jtitle%3DCell%2520Res.%26date%3D2018%26volume%3D28%26spage%3D779%26epage%3D781%26doi%3D10.1038%2Fs41422-018-0055-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Salami, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alabi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Willard, R. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vitale, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McDonnell, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crew, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neklesa, T. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Androgen receptor degradation by the proteolysis-targeting chimera ARCC-4 outperforms enzalutamide in cellular models of prostate cancer drug resistance</span>. <i>Commun. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">100</span>, <span class="refDoi"> DOI: 10.1038/s42003-018-0105-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=10.1038%2Fs42003-018-0105-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=30271980" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=1%3ACAS%3A280%3ADC%252BB3czksVCksA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2018&pages=100&author=J.+Salamiauthor=S.+Alabiauthor=R.+R.+Willardauthor=N.+J.+Vitaleauthor=J.+Wangauthor=H.+Dongauthor=M.+Jinauthor=D.+P.+McDonnellauthor=A.+P.+Crewauthor=T.+K.+Neklesaauthor=C.+M.+Crews&title=Androgen+receptor+degradation+by+the+proteolysis-targeting+chimera+ARCC-4+outperforms+enzalutamide+in+cellular+models+of+prostate+cancer+drug+resistance&doi=10.1038%2Fs42003-018-0105-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Androgen receptor degradation by the proteolysis-targeting chimera ARCC-4 outperforms enzalutamide in cellular models of prostate cancer drug resistance</span></div><div class="casAuthors">Salami Jemilat; Crews Craig M; Alabi Shanique; Crews Craig M; Willard Ryan R; Vitale Nick J; Wang Jing; Dong Hanqing; Jin Meizhong; Crew Andrew P; Neklesa Taavi K; McDonnell Donald P; Crews Craig M</div><div class="citationInfo"><span class="NLM_cas:title">Communications biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">100</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The androgen receptor is a major driver of prostate cancer and inhibition of its transcriptional activity using competitive antagonists, such as enzalutamide remains a frontline therapy for prostate cancer management.  However, the majority of patients eventually develop drug resistance.  We propose that targeting the androgen receptor for degradation via Proteolysis Targeting Chimeras (PROTACs) will be a better therapeutic strategy for targeting androgen receptor signaling in prostate cancer cells.  Here we perform a head-to-head comparison between a currently approved androgen receptor antagonist enzalutamide, and its PROTAC derivative, ARCC-4, across different cellular models of prostate cancer drug resistance.  ARCC-4 is a low-nanomolar androgen receptor degrader able to degrade about 95% of cellular androgen receptors.  ARCC-4 inhibits prostate tumor cell proliferation, degrades clinically relevant androgen receptor point mutants and unlike enzalutamide, retains antiproliferative effect in a high androgen environment.  Thus, ARCC-4 exemplifies how protein degradation can address the drug resistance hurdles of enzalutamide.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQilKiHIro9_JI8NvIb08rbfW6udTcc2eYAosnGWfzAILntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3czksVCksA%253D%253D&md5=b8b7a81226dcf37d9ff430244b21f952</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1038%2Fs42003-018-0105-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs42003-018-0105-8%26sid%3Dliteratum%253Aachs%26aulast%3DSalami%26aufirst%3DJ.%26aulast%3DAlabi%26aufirst%3DS.%26aulast%3DWillard%26aufirst%3DR.%2BR.%26aulast%3DVitale%26aufirst%3DN.%2BJ.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DDong%26aufirst%3DH.%26aulast%3DJin%26aufirst%3DM.%26aulast%3DMcDonnell%26aufirst%3DD.%2BP.%26aulast%3DCrew%26aufirst%3DA.%2BP.%26aulast%3DNeklesa%26aufirst%3DT.%2BK.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DAndrogen%2520receptor%2520degradation%2520by%2520the%2520proteolysis-targeting%2520chimera%2520ARCC-4%2520outperforms%2520enzalutamide%2520in%2520cellular%2520models%2520of%2520prostate%2520cancer%2520drug%2520resistance%26jtitle%3DCommun.%2520Biol.%26date%3D2018%26volume%3D1%26spage%3D100%26doi%3D10.1038%2Fs42003-018-0105-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kregel, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiao, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fernandez-Salas, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bawa, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCollum, B. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilder-Romans, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Apel, I. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Speers, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chinnaiyan, A. M.</span></span> <span> </span><span class="NLM_article-title">Androgen receptor degraders overcome common resistance mechanisms developed during prostate cancer treatment</span>. <i>Neoplasia</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">111</span>– <span class="NLM_lpage">119</span>, <span class="refDoi"> DOI: 10.1016/j.neo.2019.12.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=10.1016%2Fj.neo.2019.12.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=31931431" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=1%3ACAS%3A528%3ADC%252BB3cXht1Ogsbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2020&pages=111-119&author=S.+Kregelauthor=C.+Wangauthor=X.+Hanauthor=L.+Xiaoauthor=E.+Fernandez-Salasauthor=P.+Bawaauthor=B.+L.+McCollumauthor=K.+Wilder-Romansauthor=I.+J.+Apelauthor=X.+Caoauthor=C.+Speersauthor=S.+Wangauthor=A.+M.+Chinnaiyan&title=Androgen+receptor+degraders+overcome+common+resistance+mechanisms+developed+during+prostate+cancer+treatment&doi=10.1016%2Fj.neo.2019.12.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Androgen receptor degraders overcome common resistance mechanisms developed during prostate cancer treatment</span></div><div class="casAuthors">Kregel, Steven; Wang, Chao; Han, Xin; Xiao, Lanbo; Fernandez-Salas, Ester; Bawa, Pushpinder; McCollum, Brooke L.; Wilder-Romans, Kari; Apel, Ingrid J.; Cao, Xuhong; Speers, Corey; Wang, Shaomeng; Chinnaiyan, Arul M.</div><div class="citationInfo"><span class="NLM_cas:title">Neoplasia (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">111-119</span>CODEN:
                <span class="NLM_cas:coden">NEOPFL</span>;
        ISSN:<span class="NLM_cas:issn">1476-5586</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">Androgen receptor (AR) antagonists, such as enzalutamide, have had a major impact on the treatment of metastatic castration-resistant prostate cancer (CRPC).  However, even with the advent of AR antagonist therapies, patients continue to develop resistance, and new strategies to combat continued AR signalling are needed.  Here, we develop AR degraders using PROteolysis TArgeting Chimeric (PROTAC) technol. in order to det. whether depletion of AR protein can overcome mechanisms of resistance commonly assocd. with current AR-targeting therapies.  ARD-61 is the most potent of the AR degraders and effectively induces on-target AR degrdn. with a mechanism consistent with the PROTAC design.  Compared to clin.-approved AR antagonists, administration of ARD-61 in vitro and in vivo results in more potent anti-proliferative, pro-apoptotic effects and attenuation of downstream AR target gene expression in prostate cancer cells.  Importantly, we demonstrate that ARD-61 functions in enzalutamide-resistant model systems, characterized by diverse proposed mechanisms of resistance that include AR amplification/overexpression, AR mutation, and expression of AR splice variants, such as AR-V7.  While AR degraders are unable to bind and degrade AR-V7, they continue to inhibit tumor cell growth in models overexpressing AR-V7.  To further explore this, we developed several isogenic prostate cell line models in which AR-V7 is highly expressed, which also failed to influence the cell inhibitory effects of AR degraders, suggesting that AR-V7 is not a functional resistance mechanism for AR antagonism.  These data provide compelling evidence that full-length AR remains a prominent oncogenic driver of prostate cancers which have developed resistance to AR antagonists and highlight the clin. potential of AR degraders for treatment of CRPC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoPKragCvCdqrVg90H21EOLACvtfcHk0ljvIXhKM_giVg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXht1Ogsbc%253D&md5=dbc33b022029430baaf2a8b6a85b886f</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1016%2Fj.neo.2019.12.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.neo.2019.12.003%26sid%3Dliteratum%253Aachs%26aulast%3DKregel%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DC.%26aulast%3DHan%26aufirst%3DX.%26aulast%3DXiao%26aufirst%3DL.%26aulast%3DFernandez-Salas%26aufirst%3DE.%26aulast%3DBawa%26aufirst%3DP.%26aulast%3DMcCollum%26aufirst%3DB.%2BL.%26aulast%3DWilder-Romans%26aufirst%3DK.%26aulast%3DApel%26aufirst%3DI.%2BJ.%26aulast%3DCao%26aufirst%3DX.%26aulast%3DSpeers%26aufirst%3DC.%26aulast%3DWang%26aufirst%3DS.%26aulast%3DChinnaiyan%26aufirst%3DA.%2BM.%26atitle%3DAndrogen%2520receptor%2520degraders%2520overcome%2520common%2520resistance%2520mechanisms%2520developed%2520during%2520prostate%2520cancer%2520treatment%26jtitle%3DNeoplasia%26date%3D2020%26volume%3D22%26spage%3D111%26epage%3D119%26doi%3D10.1016%2Fj.neo.2019.12.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Salami, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Waste disposal-An attractive strategy for cancer therapy</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>355</i></span>,  <span class="NLM_fpage">1163</span>– <span class="NLM_lpage">1167</span>, <span class="refDoi"> DOI: 10.1126/science.aam7340</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=10.1126%2Fscience.aam7340" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=28302825" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=1%3ACAS%3A528%3ADC%252BC2sXktFOmt74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=355&publication_year=2017&pages=1163-1167&author=J.+Salamiauthor=C.+M.+Crews&title=Waste+disposal-An+attractive+strategy+for+cancer+therapy&doi=10.1126%2Fscience.aam7340"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Waste disposal-An attractive strategy for cancer therapy</span></div><div class="casAuthors">Salami, Jemilat; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">355</span>
        (<span class="NLM_cas:issue">6330</span>),
    <span class="NLM_cas:pages">1163-1167</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">A review.  Targeted therapies for cancer are typically small mols. or monoclonal antibodies that act by inhibiting the activity of specific proteins that drive tumor growth.  Although many of these drugs are effective in cancer patients, the response is often not durable because tumor cells develop resistance to the drugs.  Another limitation of this strategy is that not all oncogenic driver proteins are "druggable" enzymes or receptors with activities that can be inhibited.  Here the authors describe an alternative approach to targeted therapy that is based on co-opting the cellular quality-control machinery-the ubiquitin-proteasome system-to remove specific cancer-causing proteins from the cell.  The authors first discuss examples of existing cancer drugs that work by degrading specific proteins and then review recent progress in the rational design and preclin. testing of small mols. that induce selective degrdn. of specific target proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpVoZn-YpfXmbVg90H21EOLACvtfcHk0ljvIXhKM_giVg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXktFOmt74%253D&md5=918621d50a82b7797aa0300537a4486e</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1126%2Fscience.aam7340&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.aam7340%26sid%3Dliteratum%253Aachs%26aulast%3DSalami%26aufirst%3DJ.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DWaste%2520disposal-An%2520attractive%2520strategy%2520for%2520cancer%2520therapy%26jtitle%3DScience%26date%3D2017%26volume%3D355%26spage%3D1163%26epage%3D1167%26doi%3D10.1126%2Fscience.aam7340" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bondeson, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mares, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, I. E. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ko, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campos, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miah, A. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mulholland, K. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Routly, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buckley, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gustafson, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zinn, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grandi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shimamura, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bergamini, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Faelth-Savitski, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bantscheff, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cox, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Willard, R. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flanagan, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Casillas, L. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Votta, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">den Besten, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Famm, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kruidenier, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carter, P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harling, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Churcher, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Catalytic in vivo protein knockdown by small-molecule PROTACs</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">611</span>– <span class="NLM_lpage">617</span>, <span class="refDoi"> DOI: 10.1038/nchembio.1858</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=10.1038%2Fnchembio.1858" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=26075522" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtFeju7rP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2015&pages=611-617&author=D.+P.+Bondesonauthor=A.+Maresauthor=I.+E.+D.+Smithauthor=E.+Koauthor=S.+Camposauthor=A.+H.+Miahauthor=K.+E.+Mulhollandauthor=N.+Routlyauthor=D.+L.+Buckleyauthor=J.+L.+Gustafsonauthor=N.+Zinnauthor=P.+Grandiauthor=S.+Shimamuraauthor=G.+Bergaminiauthor=M.+Faelth-Savitskiauthor=M.+Bantscheffauthor=C.+Coxauthor=D.+A.+Gordonauthor=R.+R.+Willardauthor=J.+J.+Flanaganauthor=L.+N.+Casillasauthor=B.+J.+Vottaauthor=W.+den+Bestenauthor=K.+Fammauthor=L.+Kruidenierauthor=P.+S.+Carterauthor=J.+D.+Harlingauthor=I.+Churcherauthor=C.+M.+Crews&title=Catalytic+in+vivo+protein+knockdown+by+small-molecule+PROTACs&doi=10.1038%2Fnchembio.1858"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Catalytic in vivo protein knockdown by small-molecule PROTACs</span></div><div class="casAuthors">Bondeson, Daniel P.; Mares, Alina; Smith, Ian E. D.; Ko, Eunhwa; Campos, Sebastien; Miah, Afjal H.; Mulholland, Katie E.; Routly, Natasha; Buckley, Dennis L.; Gustafson, Jeffrey L.; Zinn, Nico; Grandi, Paola; Shimamura, Satoko; Bergamini, Giovanna; Faelth-Savitski, Maria; Bantscheff, Marcus; Cox, Carly; Gordon, Deborah A.; Willard, Ryan R.; Flanagan, John J.; Casillas, Linda N.; Votta, Bartholomew J.; den Besten, Willem; Famm, Kristoffer; Kruidenier, Laurens; Carter, Paul S.; Harling, John D.; Churcher, Ian; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">611-617</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The current predominant therapeutic paradigm is based on maximizing drug-receptor occupancy to achieve clin. benefit.  This strategy, however, generally requires excessive drug concns. to ensure sufficient occupancy, often leading to adverse side effects.  Here, we describe major improvements to the proteolysis targeting chimeras (PROTACs) method, a chem. knockdown strategy in which a heterobifunctional mol. recruits a specific protein target to an E3 ubiquitin ligase, resulting in the target's ubiquitination and degrdn.  These compds. behave catalytically in their ability to induce the ubiquitination of super-stoichiometric quantities of proteins, providing efficacy that is not limited by equil. occupancy.  We present two PROTACs that are capable of specifically reducing protein levels by >90% at nanomolar concns.  In addn., mouse studies indicate that they provide broad tissue distribution and knockdown of the targeted protein in tumor xenografts.  Together, these data demonstrate a protein knockdown system combining many of the favorable properties of small-mol. agents with the potent protein knockdown of RNAi and CRISPR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGonhnajD3CxS7Vg90H21EOLACvtfcHk0liXQqxq_TTK5g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtFeju7rP&md5=31eb8756c5661f78ab4527a50dc1e251</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1858&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1858%26sid%3Dliteratum%253Aachs%26aulast%3DBondeson%26aufirst%3DD.%2BP.%26aulast%3DMares%26aufirst%3DA.%26aulast%3DSmith%26aufirst%3DI.%2BE.%2BD.%26aulast%3DKo%26aufirst%3DE.%26aulast%3DCampos%26aufirst%3DS.%26aulast%3DMiah%26aufirst%3DA.%2BH.%26aulast%3DMulholland%26aufirst%3DK.%2BE.%26aulast%3DRoutly%26aufirst%3DN.%26aulast%3DBuckley%26aufirst%3DD.%2BL.%26aulast%3DGustafson%26aufirst%3DJ.%2BL.%26aulast%3DZinn%26aufirst%3DN.%26aulast%3DGrandi%26aufirst%3DP.%26aulast%3DShimamura%26aufirst%3DS.%26aulast%3DBergamini%26aufirst%3DG.%26aulast%3DFaelth-Savitski%26aufirst%3DM.%26aulast%3DBantscheff%26aufirst%3DM.%26aulast%3DCox%26aufirst%3DC.%26aulast%3DGordon%26aufirst%3DD.%2BA.%26aulast%3DWillard%26aufirst%3DR.%2BR.%26aulast%3DFlanagan%26aufirst%3DJ.%2BJ.%26aulast%3DCasillas%26aufirst%3DL.%2BN.%26aulast%3DVotta%26aufirst%3DB.%2BJ.%26aulast%3Dden%2BBesten%26aufirst%3DW.%26aulast%3DFamm%26aufirst%3DK.%26aulast%3DKruidenier%26aufirst%3DL.%26aulast%3DCarter%26aufirst%3DP.%2BS.%26aulast%3DHarling%26aufirst%3DJ.%2BD.%26aulast%3DChurcher%26aufirst%3DI.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DCatalytic%2520in%2520vivo%2520protein%2520knockdown%2520by%2520small-molecule%2520PROTACs%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2015%26volume%3D11%26spage%3D611%26epage%3D617%26doi%3D10.1038%2Fnchembio.1858" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lai, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toure, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hellerschmied, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salami, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jaime-Figueroa, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ko, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hines, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Modular PROTAC design for the degradation of oncogenic BCR-ABL</span>. <i>Angew. Chem., Int. Ed. Engl.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">807</span>– <span class="NLM_lpage">810</span>, <span class="refDoi"> DOI: 10.1002/anie.201507634</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=10.1002%2Fanie.201507634" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=26593377" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=1%3ACAS%3A280%3ADC%252BC28vktlanug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2016&pages=807-810&author=A.+C.+Laiauthor=M.+Toureauthor=D.+Hellerschmiedauthor=J.+Salamiauthor=S.+Jaime-Figueroaauthor=E.+Koauthor=J.+Hinesauthor=C.+M.+Crews&title=Modular+PROTAC+design+for+the+degradation+of+oncogenic+BCR-ABL&doi=10.1002%2Fanie.201507634"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Modular PROTAC Design for the Degradation of Oncogenic BCR-ABL</span></div><div class="casAuthors">Lai Ashton C; Toure Momar; Hellerschmied Doris; Salami Jemilat; Jaime-Figueroa Saul; Ko Eunhwa; Hines John; Crews Craig M</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie (International ed. in English)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">807-10</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Proteolysis Targeting Chimera (PROTAC) technology is a rapidly emerging alternative therapeutic strategy with the potential to address many of the challenges currently faced in modern drug development programs.  PROTAC technology employs small molecules that recruit target proteins for ubiquitination and removal by the proteasome.  The synthesis of PROTAC compounds that mediate the degradation of c-ABL and BCR-ABL by recruiting either Cereblon or Von Hippel Lindau E3 ligases is reported.  During the course of their development, we discovered that the capacity of a PROTAC to induce degradation involves more than just target binding: the identity of the inhibitor warhead and the recruited E3 ligase largely determine the degradation profiles of the compounds; thus, as a starting point for PROTAC development, both the target ligand and the recruited E3 ligase should be varied to rapidly generate a PROTAC with the desired degradation profile.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSu-9RPU8iMpUodMGTatQ7LfW6udTcc2eaarPxssaxw77ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC28vktlanug%253D%253D&md5=5796ba6cff3b26c00c88ad700385f37a</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1002%2Fanie.201507634&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201507634%26sid%3Dliteratum%253Aachs%26aulast%3DLai%26aufirst%3DA.%2BC.%26aulast%3DToure%26aufirst%3DM.%26aulast%3DHellerschmied%26aufirst%3DD.%26aulast%3DSalami%26aufirst%3DJ.%26aulast%3DJaime-Figueroa%26aufirst%3DS.%26aulast%3DKo%26aufirst%3DE.%26aulast%3DHines%26aufirst%3DJ.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DModular%2520PROTAC%2520design%2520for%2520the%2520degradation%2520of%2520oncogenic%2520BCR-ABL%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%2520Engl.%26date%3D2016%26volume%3D55%26spage%3D807%26epage%3D810%26doi%3D10.1002%2Fanie.201507634" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Demizu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shibata, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hattori, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohoka, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Motoi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Misawa, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shoda, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Naito, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kurihara, M.</span></span> <span> </span><span class="NLM_article-title">Development of BCR-ABL degradation inducers via the conjugation of an imatinib derivative and a cIAP1 ligand</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">4865</span>– <span class="NLM_lpage">4869</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2016.09.041</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=10.1016%2Fj.bmcl.2016.09.041" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=27666635" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsFOntbnK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2016&pages=4865-4869&author=Y.+Demizuauthor=N.+Shibataauthor=T.+Hattoriauthor=N.+Ohokaauthor=H.+Motoiauthor=T.+Misawaauthor=T.+Shodaauthor=M.+Naitoauthor=M.+Kurihara&title=Development+of+BCR-ABL+degradation+inducers+via+the+conjugation+of+an+imatinib+derivative+and+a+cIAP1+ligand&doi=10.1016%2Fj.bmcl.2016.09.041"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Development of BCR-ABL degradation inducers via the conjugation of an imatinib derivative and a cIAP1 ligand</span></div><div class="casAuthors">Demizu, Yosuke; Shibata, Norihito; Hattori, Takayuki; Ohoka, Nobumichi; Motoi, Hiromi; Misawa, Takashi; Shoda, Takuji; Naito, Mikihiko; Kurihara, Masaaki</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">4865-4869</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The manipulation of protein stability with small mols. has great potential as a technique for aiding the development of clin. therapies, including treatments for cancer.  In this study, BCR-ABL protein degrdn. inducers called SNIPER(ABL) (Specific and Nongenetic inhibitors of apoptosis protein [IAP]-dependent Protein Erasers) were developed.  The designed mols. contained two biol. active scaffolds: one was an imatinib deriv. that binds to BCL-ABL and the other was a Me bestatin that binds to cellular IAP 1 (cIAP1).  The hybrid mols., SNIPER(ABL), were expected to recruit BCR-ABL to cIAP1 for removal by proteasomes.  In fact, SNIPER(ABL) induced the degrdn. of BCR-ABL protein and a subsequent redn. in cell growth.  Thus, the degrdn. of BCR-ABL by SNIPER(ABL) is one potential strategy for treating BCR-ABL driven chronic myelogenous leukemia.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpLT6dtmVPL47Vg90H21EOLACvtfcHk0liXQqxq_TTK5g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsFOntbnK&md5=31c045087293acf60932a9d6c4f38504</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2016.09.041&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2016.09.041%26sid%3Dliteratum%253Aachs%26aulast%3DDemizu%26aufirst%3DY.%26aulast%3DShibata%26aufirst%3DN.%26aulast%3DHattori%26aufirst%3DT.%26aulast%3DOhoka%26aufirst%3DN.%26aulast%3DMotoi%26aufirst%3DH.%26aulast%3DMisawa%26aufirst%3DT.%26aulast%3DShoda%26aufirst%3DT.%26aulast%3DNaito%26aufirst%3DM.%26aulast%3DKurihara%26aufirst%3DM.%26atitle%3DDevelopment%2520of%2520BCR-ABL%2520degradation%2520inducers%2520via%2520the%2520conjugation%2520of%2520an%2520imatinib%2520derivative%2520and%2520a%2520cIAP1%2520ligand%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2016%26volume%3D26%26spage%3D4865%26epage%3D4869%26doi%3D10.1016%2Fj.bmcl.2016.09.041" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ren, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kong, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yin, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, B.</span></span> <span> </span><span class="NLM_article-title">Discovery of SIAIS178 as an effective BCR-ABL degrader by recruiting von hippel-lindau (VHL) E3 ubiquitin ligase</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">9281</span>– <span class="NLM_lpage">9298</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b01264</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b01264" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhvVantrnN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=9281-9298&author=Q.+Zhaoauthor=C.+Renauthor=L.+Liuauthor=J.+Chenauthor=Y.+Shaoauthor=N.+Sunauthor=R.+Sunauthor=Y.+Kongauthor=X.+Dingauthor=X.+Zhangauthor=Y.+Xuauthor=B.+Yangauthor=Q.+Yinauthor=X.+Yangauthor=B.+Jiang&title=Discovery+of+SIAIS178+as+an+effective+BCR-ABL+degrader+by+recruiting+von+hippel-lindau+%28VHL%29+E3+ubiquitin+ligase&doi=10.1021%2Facs.jmedchem.9b01264"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of SIAIS178 as an Effective BCR-ABL Degrader by Recruiting Von Hippel-Lindau (VHL) E3 Ubiquitin Ligase</span></div><div class="casAuthors">Zhao, Quanju; Ren, Chaowei; Liu, Linyi; Chen, Jinju; Shao, Yubao; Sun, Ning; Sun, Renhong; Kong, Ying; Ding, Xinyu; Zhang, Xianfang; Xu, Youwei; Yang, Bei; Yin, Qianqian; Yang, Xiaobao; Jiang, Biao</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">9281-9298</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The oncogenic fusion protein BCR-ABL is the driving force of leukemogenesis in chronic myeloid leukemia (CML).  Despite great progress for CML treatment through application of tyrosine kinase inhibitors (TKIs) against BCR-ABL, long-term drug administration and clin. resistance continue to be an issue.  Herein, we described the design, synthesis, and evaluation of novel proteolysis-targeting chimeric (PROTAC) small mols. targeting BCR-ABL which connect dasatinib and VHL E3 ubiquitin ligase ligand by extensive optimization of linkers.  Our efforts have yielded SIAIS178 (19), which induces proper interaction between BCR-ABL and VHL ligase leading to effective degrdn. of BCR-ABL protein, achieves significant growth inhibition of BCR-ABL+ leukemic cells in vitro, and induces substantial tumor regression against K562 xenograft tumors in vivo.  In addn., SIAIS178 also degrades several clin. relevant resistance-conferring mutations.  Our data indicate that SIAIS178 as efficacious BCR-ABL degrader warrants extensive further investigation for the treatment of BCR-ABL+ leukemia.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrLBFfujSx6ZbVg90H21EOLACvtfcHk0lizg1-UrFLnVA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhvVantrnN&md5=50ace274dbc6ef3931f86504a1be93a7</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b01264&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b01264%26sid%3Dliteratum%253Aachs%26aulast%3DZhao%26aufirst%3DQ.%26aulast%3DRen%26aufirst%3DC.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DShao%26aufirst%3DY.%26aulast%3DSun%26aufirst%3DN.%26aulast%3DSun%26aufirst%3DR.%26aulast%3DKong%26aufirst%3DY.%26aulast%3DDing%26aufirst%3DX.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DXu%26aufirst%3DY.%26aulast%3DYang%26aufirst%3DB.%26aulast%3DYin%26aufirst%3DQ.%26aulast%3DYang%26aufirst%3DX.%26aulast%3DJiang%26aufirst%3DB.%26atitle%3DDiscovery%2520of%2520SIAIS178%2520as%2520an%2520effective%2520BCR-ABL%2520degrader%2520by%2520recruiting%2520von%2520hippel-lindau%2520%2528VHL%2529%2520E3%2520ubiquitin%2520ligase%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D9281%26epage%3D9298%26doi%3D10.1021%2Facs.jmedchem.9b01264" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Robb, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Contreras, J. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kour, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abid, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sonawane, Y. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zahid, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murry, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Natarajan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rana, S.</span></span> <span> </span><span class="NLM_article-title">Chemically induced degradation of CDK9 by a proteolysis targeting chimera (PROTAC)</span>. <i>Chem. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">7577</span>– <span class="NLM_lpage">7580</span>, <span class="refDoi"> DOI: 10.1039/c7cc03879h</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=10.1039%2Fc7cc03879h" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=28636052" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=1%3ACAS%3A528%3ADC%252BC2sXpvFyjsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2017&pages=7577-7580&author=C.+M.+Robbauthor=J.+I.+Contrerasauthor=S.+Kourauthor=M.+A.+Taylorauthor=M.+Abidauthor=Y.+A.+Sonawaneauthor=M.+Zahidauthor=D.+J.+Murryauthor=A.+Natarajanauthor=S.+Rana&title=Chemically+induced+degradation+of+CDK9+by+a+proteolysis+targeting+chimera+%28PROTAC%29&doi=10.1039%2Fc7cc03879h"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Chemically induced degradation of CDK9 by a proteolysis targeting chimera (PROTAC)</span></div><div class="casAuthors">Robb, Caroline M.; Contreras, Jacob I.; Kour, Smit; Taylor, Margaret A.; Abid, Mohammad; Sonawane, Yogesh A.; Zahid, Muhammad; Murry, Daryl J.; Natarajan, Amarnath; Rana, Sandeep</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Communications (Cambridge, United Kingdom)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">54</span>),
    <span class="NLM_cas:pages">7577-7580</span>CODEN:
                <span class="NLM_cas:coden">CHCOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-7345</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Cyclin-dependent kinase 9 (CDK9), a member of the cyclin-dependent protein kinase (CDK) family, is involved in transcriptional elongation of several target genes.  CDK9 is ubiquitously expressed and has been shown to contribute to a variety of malignancies such as pancreatic, prostate and breast cancers.  Here we report the development of a heterobifunctional small mol. proteolysis targeting chimera (PROTAC) capable of cereblon (CRBN)-mediated proteasomal degrdn. of CDK9.  In HCT116 cells, it selectively degrades CDK9 while sparing other CDK family members.  This is the first example of a PROTAC that selectively degrades CDK9.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpXgLtOTwvlnbVg90H21EOLACvtfcHk0lizg1-UrFLnVA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXpvFyjsLw%253D&md5=a9aa8ebf8109f11206740efd1b5e09a2</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1039%2Fc7cc03879h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc7cc03879h%26sid%3Dliteratum%253Aachs%26aulast%3DRobb%26aufirst%3DC.%2BM.%26aulast%3DContreras%26aufirst%3DJ.%2BI.%26aulast%3DKour%26aufirst%3DS.%26aulast%3DTaylor%26aufirst%3DM.%2BA.%26aulast%3DAbid%26aufirst%3DM.%26aulast%3DSonawane%26aufirst%3DY.%2BA.%26aulast%3DZahid%26aufirst%3DM.%26aulast%3DMurry%26aufirst%3DD.%2BJ.%26aulast%3DNatarajan%26aufirst%3DA.%26aulast%3DRana%26aufirst%3DS.%26atitle%3DChemically%2520induced%2520degradation%2520of%2520CDK9%2520by%2520a%2520proteolysis%2520targeting%2520chimera%2520%2528PROTAC%2529%26jtitle%3DChem.%2520Commun.%26date%3D2017%26volume%3D53%26spage%3D7577%26epage%3D7580%26doi%3D10.1039%2Fc7cc03879h" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Olson, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Erb, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doctor, Z. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kwiatkowski, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boukhali, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Green, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haas, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nomanbhoy, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fischer, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winter, G. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span> <span> </span><span class="NLM_article-title">Pharmacological perturbation of CDK9 using selective CDK9 inhibition or degradation</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">163</span>– <span class="NLM_lpage">170</span>, <span class="refDoi"> DOI: 10.1038/nchembio.2538</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=10.1038%2Fnchembio.2538" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=29251720" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2017&pages=163-170&author=C.+M.+Olsonauthor=B.+Jiangauthor=M.+A.+Erbauthor=Y.+Liangauthor=Z.+M.+Doctorauthor=Z.+Zhangauthor=T.+Zhangauthor=N.+Kwiatkowskiauthor=M.+Boukhaliauthor=J.+L.+Greenauthor=W.+Haasauthor=T.+Nomanbhoyauthor=E.+S.+Fischerauthor=R.+A.+Youngauthor=J.+E.+Bradnerauthor=G.+E.+Winterauthor=N.+S.+Gray&title=Pharmacological+perturbation+of+CDK9+using+selective+CDK9+inhibition+or+degradation&doi=10.1038%2Fnchembio.2538"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.2538&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.2538%26sid%3Dliteratum%253Aachs%26aulast%3DOlson%26aufirst%3DC.%2BM.%26aulast%3DJiang%26aufirst%3DB.%26aulast%3DErb%26aufirst%3DM.%2BA.%26aulast%3DLiang%26aufirst%3DY.%26aulast%3DDoctor%26aufirst%3DZ.%2BM.%26aulast%3DZhang%26aufirst%3DZ.%26aulast%3DZhang%26aufirst%3DT.%26aulast%3DKwiatkowski%26aufirst%3DN.%26aulast%3DBoukhali%26aufirst%3DM.%26aulast%3DGreen%26aufirst%3DJ.%2BL.%26aulast%3DHaas%26aufirst%3DW.%26aulast%3DNomanbhoy%26aufirst%3DT.%26aulast%3DFischer%26aufirst%3DE.%2BS.%26aulast%3DYoung%26aufirst%3DR.%2BA.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26aulast%3DWinter%26aufirst%3DG.%2BE.%26aulast%3DGray%26aufirst%3DN.%2BS.%26atitle%3DPharmacological%2520perturbation%2520of%2520CDK9%2520using%2520selective%2520CDK9%2520inhibition%2520or%2520degradation%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2017%26volume%3D14%26spage%3D163%26epage%3D170%26doi%3D10.1038%2Fnchembio.2538" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huang, H.-T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dobrovolsky, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paulk, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weisberg, E. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doctor, Z. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buckley, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cho, J.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ko, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choi, H. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Griffin, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Treon, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fischer, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span> <span> </span><span class="NLM_article-title">A chemoproteomic approach to query the degradable kinome using a multi-kinase degrader</span>. <i>Cell Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">88</span>– <span class="NLM_lpage">99.e6</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2017.10.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=10.1016%2Fj.chembiol.2017.10.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=29129717" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsl2lsrjO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2018&pages=88-99.e6&author=H.-T.+Huangauthor=D.+Dobrovolskyauthor=J.+Paulkauthor=G.+Yangauthor=E.+L.+Weisbergauthor=Z.+M.+Doctorauthor=D.+L.+Buckleyauthor=J.-H.+Choauthor=E.+Koauthor=J.+Jangauthor=K.+Shiauthor=H.+G.+Choiauthor=J.+D.+Griffinauthor=Y.+Liauthor=S.+P.+Treonauthor=E.+S.+Fischerauthor=J.+E.+Bradnerauthor=L.+Tanauthor=N.+S.+Gray&title=A+chemoproteomic+approach+to+query+the+degradable+kinome+using+a+multi-kinase+degrader&doi=10.1016%2Fj.chembiol.2017.10.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">A Chemoproteomic Approach to Query the Degradable Kinome Using a Multi-kinase Degrader</span></div><div class="casAuthors">Huang, Hai-Tsang; Dobrovolsky, Dennis; Paulk, Joshiawa; Yang, Guang; Weisberg, Ellen L.; Doctor, Zainab M.; Buckley, Dennis L.; Cho, Joong-Heui; Ko, Eunhwa; Jang, Jaebong; Shi, Kun; Choi, Hwan Geun; Griffin, James D.; Li, Ying; Treon, Steven P.; Fischer, Eric S.; Bradner, James E.; Tan, Li; Gray, Nathanael S.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">88-99.e6</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Heterobifunctional mols. that recruit E3 ubiquitin ligases, such as cereblon, for targeted protein degrdn. represent an emerging pharmacol. strategy.  A major unanswered question is how generally applicable this strategy is to all protein targets.  In this study, we designed a multi-kinase degrader by conjugating a highly promiscuous kinase inhibitor with a cereblon-binding ligand, and used quant. proteomics to discover 28 kinases, including BTK, PTK2, PTK2B, FLT3, AURKA, AURKB, TEC, ULK1, ITK, and nine members of the CDK family, as degradable.  This set of kinases is only a fraction of the intracellular targets bound by the degrader, demonstrating that successful degrdn. requires more than target engagement.  The results guided us to develop selective degraders for FLT3 and BTK, with potentials to improve disease treatment.  Together, this study demonstrates an efficient approach to triage a gene family of interest to identify readily degradable targets for further studies and pre-clin. developments.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrYNxhoPV3127Vg90H21EOLACvtfcHk0lha7JC2VLo-TQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsl2lsrjO&md5=703cfbaca3edd24176472158563f895f</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2017.10.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2017.10.005%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DH.-T.%26aulast%3DDobrovolsky%26aufirst%3DD.%26aulast%3DPaulk%26aufirst%3DJ.%26aulast%3DYang%26aufirst%3DG.%26aulast%3DWeisberg%26aufirst%3DE.%2BL.%26aulast%3DDoctor%26aufirst%3DZ.%2BM.%26aulast%3DBuckley%26aufirst%3DD.%2BL.%26aulast%3DCho%26aufirst%3DJ.-H.%26aulast%3DKo%26aufirst%3DE.%26aulast%3DJang%26aufirst%3DJ.%26aulast%3DShi%26aufirst%3DK.%26aulast%3DChoi%26aufirst%3DH.%2BG.%26aulast%3DGriffin%26aufirst%3DJ.%2BD.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DTreon%26aufirst%3DS.%2BP.%26aulast%3DFischer%26aufirst%3DE.%2BS.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26aulast%3DTan%26aufirst%3DL.%26aulast%3DGray%26aufirst%3DN.%2BS.%26atitle%3DA%2520chemoproteomic%2520approach%2520to%2520query%2520the%2520degradable%2520kinome%2520using%2520a%2520multi-kinase%2520degrader%26jtitle%3DCell%2520Chem.%2520Biol.%26date%3D2018%26volume%3D25%26spage%3D88%26epage%3D99.e6%26doi%3D10.1016%2Fj.chembiol.2017.10.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Burslem, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, B. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jaime-Figueroa, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McQuaid, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bondeson, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toure, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qian, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crew, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hines, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">The advantages of targeted protein degradation over inhibition: an RTK case study</span>. <i>Cell Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">67</span>– <span class="NLM_lpage">77.e3</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2017.09.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=10.1016%2Fj.chembiol.2017.09.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=29129716" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsl2lsrjM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2018&pages=67-77.e3&author=G.+M.+Burslemauthor=B.+E.+Smithauthor=A.+C.+Laiauthor=S.+Jaime-Figueroaauthor=D.+C.+McQuaidauthor=D.+P.+Bondesonauthor=M.+Toureauthor=H.+Dongauthor=Y.+Qianauthor=J.+Wangauthor=A.+P.+Crewauthor=J.+Hinesauthor=C.+M.+Crews&title=The+advantages+of+targeted+protein+degradation+over+inhibition%3A+an+RTK+case+study&doi=10.1016%2Fj.chembiol.2017.09.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">The Advantages of Targeted Protein Degradation Over Inhibition: An RTK Case Study</span></div><div class="casAuthors">Burslem, George M.; Smith, Blake E.; Lai, Ashton C.; Jaime-Figueroa, Saul; McQuaid, Daniel C.; Bondeson, Daniel P.; Toure, Momar; Dong, Hanqing; Qian, Yimin; Wang, Jing; Crew, Andrew P.; Hines, John; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">67-77.e3</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Proteolysis targeting chimera (PROTAC) technol. has emerged over the last two decades as a powerful tool for targeted degrdn. of endogenous proteins.  Herein we describe the development of PROTACs for receptor tyrosine kinases, a protein family yet to be targeted for induced protein degrdn.  The use of VHL-recruiting PROTACs against this protein family reveals several advantages of degrdn. over inhibition alone: direct comparisons of fully functional, target-degrading PROTACs with target-inhibiting variants that contain an inactivated E3 ligase-recruiting ligand show that degrdn. leads to more potent inhibition of cell proliferation and a more durable and sustained downstream signaling response, and thus addresses the kinome rewiring challenge seen with many receptor tyrosine kinase inhibitors.  Combined, these findings demonstrate the ability to target receptor tyrosine kinases for degrdn. using the PROTAC technol. and outline the advantages of this degrdn.-based approach.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr4yDhRDWqNv7Vg90H21EOLACvtfcHk0lha7JC2VLo-TQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsl2lsrjM&md5=0ac39a6f5836ea3772a0396356efb630</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2017.09.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2017.09.009%26sid%3Dliteratum%253Aachs%26aulast%3DBurslem%26aufirst%3DG.%2BM.%26aulast%3DSmith%26aufirst%3DB.%2BE.%26aulast%3DLai%26aufirst%3DA.%2BC.%26aulast%3DJaime-Figueroa%26aufirst%3DS.%26aulast%3DMcQuaid%26aufirst%3DD.%2BC.%26aulast%3DBondeson%26aufirst%3DD.%2BP.%26aulast%3DToure%26aufirst%3DM.%26aulast%3DDong%26aufirst%3DH.%26aulast%3DQian%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DCrew%26aufirst%3DA.%2BP.%26aulast%3DHines%26aufirst%3DJ.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DThe%2520advantages%2520of%2520targeted%2520protein%2520degradation%2520over%2520inhibition%253A%2520an%2520RTK%2520case%2520study%26jtitle%3DCell%2520Chem.%2520Biol.%26date%3D2018%26volume%3D25%26spage%3D67%26epage%3D77.e3%26doi%3D10.1016%2Fj.chembiol.2017.09.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meng, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiong, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span> <span> </span><span class="NLM_article-title">Discovery of potent and selective epidermal growth factor receptor (EGFR) bifunctional small-molecule degraders</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">1216</span>– <span class="NLM_lpage">1232</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b01566</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b01566" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=1%3ACAS%3A528%3ADC%252BB3cXisFWksA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=1216-1232&author=M.+Chengauthor=X.+Yuauthor=K.+Luauthor=L.+Xieauthor=L.+Wangauthor=F.+Mengauthor=X.+Hanauthor=X.+Chenauthor=J.+Liuauthor=Y.+Xiongauthor=J.+Jin&title=Discovery+of+potent+and+selective+epidermal+growth+factor+receptor+%28EGFR%29+bifunctional+small-molecule+degraders&doi=10.1021%2Facs.jmedchem.9b01566"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Potent and Selective Epidermal Growth Factor Receptor (EGFR) Bifunctional Small-Molecule Degraders</span></div><div class="casAuthors">Cheng, Meng; Yu, Xufen; Lu, Kaylene; Xie, Ling; Wang, Li; Meng, Fanye; Han, Xiaoran; Chen, Xian; Liu, Jing; Xiong, Yue; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1216-1232</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Several epidermal growth factor receptor (EGFR) tyrosine kinase inhibitors have been developed and approved by Food and Drug Administration for the treatment of non-small-cell lung cancers, but their efficacy can be compromised by acquired drug resistance conferred by EGFR-mutant variants.  Here, we described the discovery of a novel E3 ligase von Hippel-Lindau-recruiting EGFR degrader, MS39 (compd. 6), and a first-in-class E3 ligase cereblon-recruiting EGFR degrader, MS154 (compd. 10), using the proteolysis targeting chimera technol.  These compds. potently induced the degrdn. of mutant but not wild-type EGFR in an E3 ligase-dependent manner in cancer cell lines and effectively suppressed the growth of lung cancer cells compared with the corresponding neg. controls.  The global proteomic analyses revealed that the compds. were highly selective for EGFR.  Furthermore, both compds. were bioavailable in mouse pharmacokinetic studies, and compd. 6 is the first EGFR degrader suitable for in vivo efficacy studies.  Overall, we provide a set of well-characterized chem. tools to the research community.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGraFLSjUGE-6rVg90H21EOLACvtfcHk0lha7JC2VLo-TQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXisFWksA%253D%253D&md5=ec831897a4258f1968e254aa433426c3</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b01566%26sid%3Dliteratum%253Aachs%26aulast%3DCheng%26aufirst%3DM.%26aulast%3DYu%26aufirst%3DX.%26aulast%3DLu%26aufirst%3DK.%26aulast%3DXie%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DMeng%26aufirst%3DF.%26aulast%3DHan%26aufirst%3DX.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DXiong%26aufirst%3DY.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DDiscovery%2520of%2520potent%2520and%2520selective%2520epidermal%2520growth%2520factor%2520receptor%2520%2528EGFR%2529%2520bifunctional%2520small-molecule%2520degraders%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D1216%26epage%3D1232%26doi%3D10.1021%2Facs.jmedchem.9b01566" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, X.-R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiong, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span> <span> </span><span class="NLM_article-title">Proteolysis targeting chimeras (PROTACs) of anaplastic lymphoma kinase (ALK)</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>151</i></span>,  <span class="NLM_fpage">304</span>– <span class="NLM_lpage">314</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2018.03.071</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=10.1016%2Fj.ejmech.2018.03.071" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=29627725" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=1%3ACAS%3A528%3ADC%252BC1cXnt1ejs7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=151&publication_year=2018&pages=304-314&author=C.+Zhangauthor=X.-R.+Hanauthor=X.+Yangauthor=B.+Jiangauthor=J.+Liuauthor=Y.+Xiongauthor=J.+Jin&title=Proteolysis+targeting+chimeras+%28PROTACs%29+of+anaplastic+lymphoma+kinase+%28ALK%29&doi=10.1016%2Fj.ejmech.2018.03.071"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Proteolysis Targeting Chimeras (PROTACs) of Anaplastic Lymphoma Kinase (ALK)</span></div><div class="casAuthors">Zhang, Chengwei; Han, Xiao-Ran; Yang, Xiaobao; Jiang, Biao; Liu, Jing; Xiong, Yue; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">151</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">304-314</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">Anaplastic lymphoma kinase (ALK) activation has been assocd. with many types of human cancer.  Significant efforts have been devoted to the development of ALK inhibitors to antagonize the kinase activity of ALK.  Four ALK inhibitors have been approved by the FDA to date for treating patients with ALK-pos. nonsmall cell lung cancers (NSCLC).  However, drug resistance has been obsd. in the majority of patients treated with these inhibitors.  New therapeutic strategies (e.g., compds. with novel mechanisms of action) are needed to overcome the drug resistance issue.  The emerging PROTAC (Proteolysis Targeting Chimera) technol. has been successfully applied to selective degrdn. of multiple protein targets, but not ALK.  Since ALK protein levels are not important for viability in mammals, ALK PROTACs could lead to novel therapeutics with minimal toxicity.  Here the authors report the design, synthesis and biol. evaluation of novel PROTACs (degraders) of ALK.  MS4077 (I) and MS4078 (II) potently decreased cellular levels of oncogenic active ALK fusion proteins in a concn.- and time-dependent manner in SU-DHL-1 lymphoma and NCI-H2228 lung cancer cells.  The ALK protein degrdn. induced by compds. I and II was cereblon and proteasome dependent.  In addn., compds. I and II potently inhibited proliferation of SU-DHL-1 cells.  Furthermore, compd. II displayed good plasma exposure in a mouse pharmacokinetic study, thus is suitable for in vivo efficacy studies.  The authors also developed MS4748 and MS4740 , very close analogs of I and II resp., which are incapable to degrade the ALK fusion proteins, as neg. controls.  All 4 compds. are valuable chem. tools for investigating effects of ALK pharmacol. degrdn.  The study paved the way for developing the next generation of ALK PROTACs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPGM3aAZZ6h7Vg90H21EOLACvtfcHk0ljw-IRNEngOqQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXnt1ejs7Y%253D&md5=804324b95034f38b46e45f89e18e55be</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2018.03.071&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2018.03.071%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DC.%26aulast%3DHan%26aufirst%3DX.-R.%26aulast%3DYang%26aufirst%3DX.%26aulast%3DJiang%26aufirst%3DB.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DXiong%26aufirst%3DY.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DProteolysis%2520targeting%2520chimeras%2520%2528PROTACs%2529%2520of%2520anaplastic%2520lymphoma%2520kinase%2520%2528ALK%2529%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2018%26volume%3D151%26spage%3D304%26epage%3D314%26doi%3D10.1016%2Fj.ejmech.2018.03.071" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Powell, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Donovan, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nowak, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loehr, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bahcall, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fischer, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jänne, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">George, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span> <span> </span><span class="NLM_article-title">Chemically induced degradation of anaplastic lymphoma kinase (ALK)</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">4249</span>– <span class="NLM_lpage">4255</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b01655</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b01655" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=1%3ACAS%3A528%3ADC%252BC1cXns1OitrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=4249-4255&author=C.+E.+Powellauthor=Y.+Gaoauthor=L.+Tanauthor=K.+A.+Donovanauthor=R.+P.+Nowakauthor=A.+Loehrauthor=M.+Bahcallauthor=E.+S.+Fischerauthor=P.+A.+J%C3%A4nneauthor=R.+E.+Georgeauthor=N.+S.+Gray&title=Chemically+induced+degradation+of+anaplastic+lymphoma+kinase+%28ALK%29&doi=10.1021%2Facs.jmedchem.7b01655"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Chemically Induced Degradation of Anaplastic Lymphoma Kinase (ALK)</span></div><div class="casAuthors">Powell, Chelsea E.; Gao, Yang; Tan, Li; Donovan, Katherine A.; Nowak, Radoslaw P.; Loehr, Amanda; Bahcall, Magda; Fischer, Eric S.; Janne, Pasi A.; George, Rani E.; Gray, Nathanael S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">4249-4255</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We present the development of the first small mol. degraders that can induce anaplastic lymphoma kinase (ALK) degrdn., including in non-small-cell lung cancer (NSCLC), anaplastic large-cell lymphoma (ALCL), and neuroblastoma (NB) cell lines.  These degraders were developed through conjugation of known pyrimidine-based ALK inhibitors, TAE684 or LDK378, and the cereblon ligand pomalidomide.  We demonstrate that in some cell types degrader potency is compromised by expression of drug transporter ABCB1.  In addn., proteomic profiling demonstrated that these compds. also promote the degrdn. of addnl. kinases including PTK2 (FAK), Aurora A, FER, and RPS6KA1 (RSK1).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp7mDH7Sj8EqbVg90H21EOLACvtfcHk0ljw-IRNEngOqQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXns1OitrY%253D&md5=486db11b3fbc2283700f38b09cc39bd7</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b01655&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b01655%26sid%3Dliteratum%253Aachs%26aulast%3DPowell%26aufirst%3DC.%2BE.%26aulast%3DGao%26aufirst%3DY.%26aulast%3DTan%26aufirst%3DL.%26aulast%3DDonovan%26aufirst%3DK.%2BA.%26aulast%3DNowak%26aufirst%3DR.%2BP.%26aulast%3DLoehr%26aufirst%3DA.%26aulast%3DBahcall%26aufirst%3DM.%26aulast%3DFischer%26aufirst%3DE.%2BS.%26aulast%3DJ%25C3%25A4nne%26aufirst%3DP.%2BA.%26aulast%3DGeorge%26aufirst%3DR.%2BE.%26aulast%3DGray%26aufirst%3DN.%2BS.%26atitle%3DChemically%2520induced%2520degradation%2520of%2520anaplastic%2520lymphoma%2520kinase%2520%2528ALK%2529%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D4249%26epage%3D4255%26doi%3D10.1021%2Facs.jmedchem.7b01655" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sun, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ren, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kong, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhong, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qiu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, B.</span></span> <span> </span><span class="NLM_article-title">Development of a Brigatinib degrader (SIAIS117) as a potential treatment for ALK positive cancer resistance</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>193</i></span>,  <span class="NLM_fpage">112190</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2020.112190</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=10.1016%2Fj.ejmech.2020.112190" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=32179332" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=1%3ACAS%3A528%3ADC%252BB3cXkvF2qu70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=193&publication_year=2020&pages=112190&author=N.+Sunauthor=C.+Renauthor=Y.+Kongauthor=H.+Zhongauthor=J.+Chenauthor=Y.+Liauthor=J.+Zhangauthor=Y.+Zhouauthor=X.+Qiuauthor=H.+Linauthor=X.+Songauthor=X.+Yangauthor=B.+Jiang&title=Development+of+a+Brigatinib+degrader+%28SIAIS117%29+as+a+potential+treatment+for+ALK+positive+cancer+resistance&doi=10.1016%2Fj.ejmech.2020.112190"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Development of a Brigatinib degrader (SIAIS117) as a potential treatment for ALK positive cancer resistance</span></div><div class="casAuthors">Sun, Ning; Ren, Chaowei; Kong, Ying; Zhong, Hui; Chen, Jinju; Li, Yan; Zhang, Jianshui; Zhou, Yuedong; Qiu, Xing; Lin, Haifan; Song, Xiaoling; Yang, Xiaobao; Jiang, Biao</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">193</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">112190</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">EML4-ALK and NPM-ALK fusion proteins possess constitutively activated ALK (anaplastic lymphoma kinase) activity, which in turn leads to the development of non-small cell lung cancer and anaplastic large-cell lymphomas (ALCLs).  FDA-approved ALK inhibitor drugs cause significant cancer regression.  However, drug resistance eventually occurs and it becomes a big obstacle in clinic.  Novel proteolysis targeting chimera (PROTAC) technol. platform provides a potential therapeutic strategy for drug resistance.  Herein, we designed and synthesized a series of ALK PROTACs based on Brigatinib and VHL-1 conjunction, and screened SIAIS117 as the best degrader which not only blocked the growth of SR and H2228 cancer cell lines, but also degraded ALK protein.  In addn., SIAIS117 also showed much better growth inhibition effect than Brigatinib on 293T cell line that exogenously expressed G1202R-resistant ALK proteins.  Furthermore, it also degraded G1202R mutant ALK protein in vitro.  At last, it has the potentially anti-proliferation ability of small cell lung cancer.  Thus, we have successfully generated the degrader SIAIS117 that can potentially overcome resistance in cancer targeted therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrX9rzk88HMe7Vg90H21EOLACvtfcHk0li4tWEgu_qQUA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXkvF2qu70%253D&md5=61f0228111c806ae62473fc450fce276</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2020.112190&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2020.112190%26sid%3Dliteratum%253Aachs%26aulast%3DSun%26aufirst%3DN.%26aulast%3DRen%26aufirst%3DC.%26aulast%3DKong%26aufirst%3DY.%26aulast%3DZhong%26aufirst%3DH.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DZhou%26aufirst%3DY.%26aulast%3DQiu%26aufirst%3DX.%26aulast%3DLin%26aufirst%3DH.%26aulast%3DSong%26aufirst%3DX.%26aulast%3DYang%26aufirst%3DX.%26aulast%3DJiang%26aufirst%3DB.%26atitle%3DDevelopment%2520of%2520a%2520Brigatinib%2520degrader%2520%2528SIAIS117%2529%2520as%2520a%2520potential%2520treatment%2520for%2520ALK%2520positive%2520cancer%2520resistance%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2020%26volume%3D193%26spage%3D112190%26doi%3D10.1016%2Fj.ejmech.2020.112190" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brand, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bauer, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Donovan, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nowak, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kwiatkowski, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fischer, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winter, G. E.</span></span> <span> </span><span class="NLM_article-title">Homolog-selective degradation as a strategy to probe the function of CDK6 in AML</span>. <i>Cell Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">300</span>– <span class="NLM_lpage">306.e9</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2018.11.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=10.1016%2Fj.chembiol.2018.11.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=30595531" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=1%3ACAS%3A528%3ADC%252BC1MXnsVSj" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2019&pages=300-306.e9&author=M.+Brandauthor=B.+Jiangauthor=S.+Bauerauthor=K.+A.+Donovanauthor=Y.+Liangauthor=E.+S.+Wangauthor=R.+P.+Nowakauthor=J.+C.+Yuanauthor=T.+Zhangauthor=N.+Kwiatkowskiauthor=A.+C.+M%C3%BCllerauthor=E.+S.+Fischerauthor=N.+S.+Grayauthor=G.+E.+Winter&title=Homolog-selective+degradation+as+a+strategy+to+probe+the+function+of+CDK6+in+AML&doi=10.1016%2Fj.chembiol.2018.11.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Homolog-Selective Degradation as a Strategy to Probe the Function of CDK6 in AML</span></div><div class="casAuthors">Brand, Matthias; Jiang, Baishan; Bauer, Sophie; Donovan, Katherine A.; Liang, Yanke; Wang, Eric S.; Nowak, Radoslaw P.; Yuan, Jingting C.; Zhang, Tinghu; Kwiatkowski, Nicholas; Mueller, Andre C.; Fischer, Eric S.; Gray, Nathanael S.; Winter, Georg E.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">300-306.e9</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The design of selective small mols. is often stymied by similar ligand binding pockets.  Here, we report BSJ-03-123, a phthalimide-based degrader that exploits protein-interface determinants to achieve proteome-wide selectivity for the degrdn. of cyclin-dependent kinase 6 (CDK6).  Pharmacol. CDK6 degrdn. targets a selective dependency of acute myeloid leukemia cells, and transcriptomics and phosphoproteomics profiling of acute degrdn. of CDK6 enabled dynamic mapping of its immediate role in coordinating signaling and transcription.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGognoz8FFoR9LVg90H21EOLACvtfcHk0li4tWEgu_qQUA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXnsVSj&md5=f8c1c3520c6c0218763d8769c59b4342</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2018.11.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2018.11.006%26sid%3Dliteratum%253Aachs%26aulast%3DBrand%26aufirst%3DM.%26aulast%3DJiang%26aufirst%3DB.%26aulast%3DBauer%26aufirst%3DS.%26aulast%3DDonovan%26aufirst%3DK.%2BA.%26aulast%3DLiang%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DE.%2BS.%26aulast%3DNowak%26aufirst%3DR.%2BP.%26aulast%3DYuan%26aufirst%3DJ.%2BC.%26aulast%3DZhang%26aufirst%3DT.%26aulast%3DKwiatkowski%26aufirst%3DN.%26aulast%3DM%25C3%25BCller%26aufirst%3DA.%2BC.%26aulast%3DFischer%26aufirst%3DE.%2BS.%26aulast%3DGray%26aufirst%3DN.%2BS.%26aulast%3DWinter%26aufirst%3DG.%2BE.%26atitle%3DHomolog-selective%2520degradation%2520as%2520a%2520strategy%2520to%2520probe%2520the%2520function%2520of%2520CDK6%2520in%2520AML%26jtitle%3DCell%2520Chem.%2520Biol.%26date%3D2019%26volume%3D26%26spage%3D300%26epage%3D306.e9%26doi%3D10.1016%2Fj.chembiol.2018.11.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref54"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref54'); return false;" data-citation="" class="refNumLink">54</a></strong><div class="NLM_citation" id="cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Donovan, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fischer, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span> <span> </span><span class="NLM_article-title">Development of Dual and Selective Degraders of Cyclin-Dependent Kinases 4 and 6</span>. <i>Angew. Chem., Int. Ed. Engl.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">6321</span>– <span class="NLM_lpage">6326</span>, <span class="refDoi"> DOI: 10.1002/anie.201901336</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=10.1002%2Fanie.201901336" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=30802347" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=1%3ACAS%3A280%3ADC%252BB3cfnt1OqtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2019&pages=6321-6326&author=B.+Jiangauthor=E.+S.+Wangauthor=K.+A.+Donovanauthor=Y.+Liangauthor=E.+S.+Fischerauthor=T.+Zhangauthor=N.+S.+Gray&title=Development+of+Dual+and+Selective+Degraders+of+Cyclin-Dependent+Kinases+4+and+6&doi=10.1002%2Fanie.201901336"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Development of Dual and Selective Degraders of Cyclin-Dependent Kinases 4 and 6</span></div><div class="casAuthors">Jiang Baishan; Wang Eric S; Donovan Katherine A; Liang Yanke; Fischer Eric S; Zhang Tinghu; Gray Nathanael S</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie (International ed. in English)</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">6321-6326</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Cyclin-dependent kinases 4 and 6 (CDK4/6) are key regulators of the cell cycle, and there are FDA-approved CDK4/6 inhibitors for treating patients with metastatic breast cancer.  However, due to conservation of their ATP-binding sites, development of selective agents has remained elusive.  Here, we report imide-based degrader molecules capable of degrading both CDK4/6, or selectively degrading either CDK4 or CDK6.  We were also able to tune the activity of these molecules against Ikaros (IKZF1) and Aiolos (IKZF3), which are well-established targets of imide-based degraders.  We found that in mantle cell lymphoma cell lines, combined IKZF1/3 degradation with dual CDK4/6 degradation produced enhanced anti-proliferative effects compared to CDK4/6 inhibition, CDK4/6 degradation, or IKZF1/3 degradation.  In summary, we report here the first compounds capable of inducing selective degradation of CDK4 and CDK6 as tools to pharmacologically dissect their distinct biological functions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQvjF82FsSWX6qcneEtCi92fW6udTcc2eZZjJmYDb8dsbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3cfnt1OqtA%253D%253D&md5=fa27938a9ec50a89644eb87812048329</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1002%2Fanie.201901336&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201901336%26sid%3Dliteratum%253Aachs%26aulast%3DJiang%26aufirst%3DB.%26aulast%3DWang%26aufirst%3DE.%2BS.%26aulast%3DDonovan%26aufirst%3DK.%2BA.%26aulast%3DLiang%26aufirst%3DY.%26aulast%3DFischer%26aufirst%3DE.%2BS.%26aulast%3DZhang%26aufirst%3DT.%26aulast%3DGray%26aufirst%3DN.%2BS.%26atitle%3DDevelopment%2520of%2520Dual%2520and%2520Selective%2520Degraders%2520of%2520Cyclin-Dependent%2520Kinases%25204%2520and%25206%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%2520Engl.%26date%3D2019%26volume%3D58%26spage%3D6321%26epage%3D6326%26doi%3D10.1002%2Fanie.201901336" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref55"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref55'); return false;" data-citation="" class="refNumLink">55</a></strong><div class="NLM_citation" id="cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Su, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">An, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chandarlapaty, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rao, Y.</span></span> <span> </span><span class="NLM_article-title">Potent and preferential degradation of CDK6 via proteolysis targeting chimera degraders</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">7575</span>– <span class="NLM_lpage">7582</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b00871</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b00871" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhsVWls7zF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=7575-7582&author=S.+Suauthor=Z.+Yangauthor=H.+Gaoauthor=H.+Yangauthor=S.+Zhuauthor=Z.+Anauthor=J.+Wangauthor=Q.+Liauthor=S.+Chandarlapatyauthor=H.+Dengauthor=W.+Wuauthor=Y.+Rao&title=Potent+and+preferential+degradation+of+CDK6+via+proteolysis+targeting+chimera+degraders&doi=10.1021%2Facs.jmedchem.9b00871"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Potent and Preferential Degradation of CDK6 via Proteolysis Targeting Chimera Degraders</span></div><div class="casAuthors">Su, Shang; Yang, Zimo; Gao, Hongying; Yang, Haiyan; Zhu, Songbiao; An, Zixuan; Wang, Juanjuan; Li, Qing; Chandarlapaty, Sarat; Deng, Haiteng; Wu, Wei; Rao, Yu</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">7575-7582</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A focused PROTAC library hijacking cancer therapeutic target CDK6 was developed.  A design principle as "match/mismatch" was proposed for understanding the degrdn. profile differences in these PROTACs.  Notably, potent PROTACs with specific and remarkable CDK6 degrdn. potential were generated by linking CDK6 inhibitor palbociclib and E3 ligase CRBN recruiter pomalidomide.  The PROTAC strongly inhibited proliferation of hematopoietic cancer cells including multiple myeloma and robustly degraded copy-amplified/mutated forms of CDK6, indicating future potential clin. applications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqQhCa_z8uvA7Vg90H21EOLACvtfcHk0lguQ4Eqmj6kBw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhsVWls7zF&md5=ed100530898c2e4c1754ca48e7295134</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b00871&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b00871%26sid%3Dliteratum%253Aachs%26aulast%3DSu%26aufirst%3DS.%26aulast%3DYang%26aufirst%3DZ.%26aulast%3DGao%26aufirst%3DH.%26aulast%3DYang%26aufirst%3DH.%26aulast%3DZhu%26aufirst%3DS.%26aulast%3DAn%26aufirst%3DZ.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DQ.%26aulast%3DChandarlapaty%26aufirst%3DS.%26aulast%3DDeng%26aufirst%3DH.%26aulast%3DWu%26aufirst%3DW.%26aulast%3DRao%26aufirst%3DY.%26atitle%3DPotent%2520and%2520preferential%2520degradation%2520of%2520CDK6%2520via%2520proteolysis%2520targeting%2520chimera%2520degraders%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D7575%26epage%3D7582%26doi%3D10.1021%2Facs.jmedchem.9b00871" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref56"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref56'); return false;" data-citation="" class="refNumLink">56</a></strong><div class="NLM_citation" id="cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wei, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span> <span> </span><span class="NLM_article-title">Discovery of a first-in-class mitogen-activated protein kinase kinase 1/2 degrader</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">10897</span>– <span class="NLM_lpage">10911</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b01528</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b01528" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitFOmt7bK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=10897-10911&author=J.+Weiauthor=J.+Huauthor=L.+Wangauthor=L.+Xieauthor=M.+S.+Jinauthor=X.+Chenauthor=J.+Liuauthor=J.+Jin&title=Discovery+of+a+first-in-class+mitogen-activated+protein+kinase+kinase+1%2F2+degrader&doi=10.1021%2Facs.jmedchem.9b01528"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a First-in-Class Mitogen-Activated Protein Kinase Kinase 1/2 Degrader</span></div><div class="casAuthors">Wei, Jieli; Hu, Jianping; Wang, Li; Xie, Ling; Jin, Margaret S.; Chen, Xian; Liu, Jing; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">10897-10911</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">MEK1 and MEK2 (also known as MAP2K1 and MAP2K2) are the "gatekeepers" of the ERK signaling output with redundant roles in controlling ERK activity.  Numerous inhibitors targeting MEK1/2 have been developed including three FDA-approved drugs.  However, acquired resistance to MEK1/2 inhibitors has been obsd. in patients and new therapeutic strategies are needed to overcome the resistance.  Here, the authors report a first-in-class degrader of MEK1/2, MS432 (I), which potently and selectively degraded MEK1 and MEK2 in a VHL E3 ligase- and proteasome-dependent manner and suppressed ERK phosphorylation in cells.  It inhibited colorectal cancer and melanoma cell proliferation much more effectively than its neg. control MS432N and its effect was phenocopied by MEK1/2 knockdown.  Compd. I was highly selective for MEK1/2 in global proteomic profiling studies.  It was also bioavailable in mice and can be used for in vivo efficacy studies.  The authors provide two well-characterized chem. tools to the biomedical community.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq1e5sg33oDYrVg90H21EOLACvtfcHk0lguQ4Eqmj6kBw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitFOmt7bK&md5=2c293b6b87990d6b814ec1dc88034f19</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b01528%26sid%3Dliteratum%253Aachs%26aulast%3DWei%26aufirst%3DJ.%26aulast%3DHu%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DXie%26aufirst%3DL.%26aulast%3DJin%26aufirst%3DM.%2BS.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DDiscovery%2520of%2520a%2520first-in-class%2520mitogen-activated%2520protein%2520kinase%2520kinase%25201%252F2%2520degrader%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D10897%26epage%3D10911%26doi%3D10.1021%2Facs.jmedchem.9b01528" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref57"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref57'); return false;" data-citation="" class="refNumLink">57</a></strong><div class="NLM_citation" id="cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vollmer, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cunoosamy, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lv, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nan, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perry, M. W. D.</span></span> <span> </span><span class="NLM_article-title">Design, synthesis, and biological evaluation of MEK PROTACs</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">157</span>– <span class="NLM_lpage">162</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b00810</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b00810" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitlWkt77E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=157-162&author=S.+Vollmerauthor=D.+Cunoosamyauthor=H.+Lvauthor=H.+Fengauthor=X.+Liauthor=Z.+Nanauthor=W.+Yangauthor=M.+W.+D.+Perry&title=Design%2C+synthesis%2C+and+biological+evaluation+of+MEK+PROTACs&doi=10.1021%2Facs.jmedchem.9b00810"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">Design, Synthesis, and Biological Evaluation of MEK PROTACs</span></div><div class="casAuthors">Vollmer, Stefan; Cunoosamy, Danen; Lv, Huafei; Feng, Huanxi; Li, Xia; Nan, Ziyang; Yang, Wenzhen; Perry, Matthew W. D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">157-162</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">PROteolysis TArgeting Chimeras (PROTACs) targeting the degrdn. of MEK have been designed based on allosteric MEK inhibitors.  Inhibition of the phosphorylation of ERK1/2 was less effective with the PROTACs than a small-mol. inhibitor; the best PROTACs, however, were more effective in inhibiting proliferation of A375 cells than an inhibitor.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGof7TjD-lFjAbVg90H21EOLACvtfcHk0lguQ4Eqmj6kBw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitlWkt77E&md5=c83d63e23d436e0ef1a3c98e994f1073</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b00810&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b00810%26sid%3Dliteratum%253Aachs%26aulast%3DVollmer%26aufirst%3DS.%26aulast%3DCunoosamy%26aufirst%3DD.%26aulast%3DLv%26aufirst%3DH.%26aulast%3DFeng%26aufirst%3DH.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DNan%26aufirst%3DZ.%26aulast%3DYang%26aufirst%3DW.%26aulast%3DPerry%26aufirst%3DM.%2BW.%2BD.%26atitle%3DDesign%252C%2520synthesis%252C%2520and%2520biological%2520evaluation%2520of%2520MEK%2520PROTACs%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D157%26epage%3D162%26doi%3D10.1021%2Facs.jmedchem.9b00810" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref58"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref58'); return false;" data-citation="" class="refNumLink">58</a></strong><div class="NLM_citation" id="cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Han, X.-R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiao, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plewe, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span> <span> </span><span class="NLM_article-title">Discovery of selective small molecule degraders of BRAF-V600E</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">4069</span>– <span class="NLM_lpage">4080</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b02083</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b02083" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=1%3ACAS%3A528%3ADC%252BB3cXlvFCjtbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=4069-4080&author=X.-R.+Hanauthor=L.+Chenauthor=Y.+Weiauthor=W.+Yuauthor=Y.+Chenauthor=C.+Zhangauthor=B.+Jiaoauthor=T.+Shiauthor=L.+Sunauthor=C.+Zhangauthor=Y.+Xuauthor=M.+R.+Leeauthor=Y.+Luoauthor=M.+B.+Pleweauthor=J.+Wang&title=Discovery+of+selective+small+molecule+degraders+of+BRAF-V600E&doi=10.1021%2Facs.jmedchem.9b02083"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Selective Small Molecule Degraders of BRAF-V600E</span></div><div class="casAuthors">Han, Xiao-Ran; Chen, Liqun; Wei, Yuanqi; Yu, Weihua; Chen, Yanke; Zhang, Chunyan; Jiao, Bingyang; Shi, Tingting; Sun, Lei; Zhang, Chao; Xu, Yang; Lee, Matthew R.; Luo, Ying; Plewe, Michael B.; Wang, Jialiang</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">4069-4080</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">BRAF is among the most frequently mutated oncogenes in human cancers.  Multiple small mol. BRAF kinase inhibitors have been approved for treating melanoma carrying BRAF-V600 mutations.  However, the benefits of BRAF kinase inhibitors are generally short-lived.  Small mol.-mediated targeted protein degrdn. has recently emerged as a novel pharmaceutical strategy to remove disease proteins through hijacking the cellular ubiquitin proteasome system (UPS).  In this study, we developed thalidomide-based heterobifunctional compds. that induced selective degrdn. of BRAF-V600E, but not the wild-type BRAF.  Downregulation of BRAF-V600E suppressed the MEK/ERK kinase cascade in melanoma cells and impaired cell growth in culture.  Abolishing the interaction between degraders and cereblon or blocking the UPS significantly impaired the activities of these degraders, validating a mechanistic role of UPS in mediating targeted degrdn. of BRAF-V600E.  These findings highlight a new approach to modulate the functions of oncogenic BRAF mutants and provide a framework to treat BRAF-dependent human cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoCRmOwotGfh7Vg90H21EOLACvtfcHk0lguQ4Eqmj6kBw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXlvFCjtbg%253D&md5=db67b3ec42b8a3eb9641c278c8cd17b6</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b02083&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b02083%26sid%3Dliteratum%253Aachs%26aulast%3DHan%26aufirst%3DX.-R.%26aulast%3DChen%26aufirst%3DL.%26aulast%3DWei%26aufirst%3DY.%26aulast%3DYu%26aufirst%3DW.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DC.%26aulast%3DJiao%26aufirst%3DB.%26aulast%3DShi%26aufirst%3DT.%26aulast%3DSun%26aufirst%3DL.%26aulast%3DZhang%26aufirst%3DC.%26aulast%3DXu%26aufirst%3DY.%26aulast%3DLee%26aufirst%3DM.%2BR.%26aulast%3DLuo%26aufirst%3DY.%26aulast%3DPlewe%26aufirst%3DM.%2BB.%26aulast%3DWang%26aufirst%3DJ.%26atitle%3DDiscovery%2520of%2520selective%2520small%2520molecule%2520degraders%2520of%2520BRAF-V600E%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D4069%26epage%3D4080%26doi%3D10.1021%2Facs.jmedchem.9b02083" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref59"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref59'); return false;" data-citation="" class="refNumLink">59</a></strong><div class="NLM_citation" id="cit59"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burgess, K.</span></span> <span> </span><span class="NLM_article-title">TrkC-targeted kinase inhibitors and PROTACs</span>. <i>Mol. Pharm.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">4313</span>– <span class="NLM_lpage">4318</span>, <span class="refDoi"> DOI: 10.1021/acs.molpharmaceut.9b00673</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.molpharmaceut.9b00673" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhslCntb%252FI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2019&pages=4313-4318&author=B.+Zhaoauthor=K.+Burgess&title=TrkC-targeted+kinase+inhibitors+and+PROTACs&doi=10.1021%2Facs.molpharmaceut.9b00673"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">TrkC-Targeted Kinase Inhibitors And PROTACs</span></div><div class="casAuthors">Zhao, Bosheng; Burgess, Kevin</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Pharmaceutics</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">4313-4318</span>CODEN:
                <span class="NLM_cas:coden">MPOHBP</span>;
        ISSN:<span class="NLM_cas:issn">1543-8384</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A small mol. motif (IY-IY), which binds the tropomyosin receptor kinase C (TrkC), was used to deliver the promiscuous kinase inhibitor (KI) dasatinib into breast cancer.  Conjugates with noncleavable (1) and cleavable (2) linkers were compared in cellular assays and shown to have more impact on the cell viabilities of TrkC+ breast cancer cells over TrkC- epithelial cells.  The IY-IY fragment was also used to recruit the E3 ligase cereblon, giving a potent proteolysis targeting chimeric (PROTAC) for TrkC degrdn. in metastatic breast cancer cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoBUrtHpr2asrVg90H21EOLACvtfcHk0liz-uayyv03yA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhslCntb%252FI&md5=15fb3168872fe9d91bad6f288b92e87b</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1021%2Facs.molpharmaceut.9b00673&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.molpharmaceut.9b00673%26sid%3Dliteratum%253Aachs%26aulast%3DZhao%26aufirst%3DB.%26aulast%3DBurgess%26aufirst%3DK.%26atitle%3DTrkC-targeted%2520kinase%2520inhibitors%2520and%2520PROTACs%26jtitle%3DMol.%2520Pharm.%26date%3D2019%26volume%3D16%26spage%3D4313%26epage%3D4318%26doi%3D10.1021%2Facs.molpharmaceut.9b00673" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref60"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref60'); return false;" data-citation="" class="refNumLink">60</a></strong><div class="NLM_citation" id="cit60"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Choi, H.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rucker, P. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fan, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Albaugh, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chopiuk, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gessier, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adrian, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hood, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jia, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Che, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCormack, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steffy, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Culazzo, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tompkins, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phung, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kreusch, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chaudhary, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prashad, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tuntland, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harris, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seidel, H. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loren, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Molteni, V.</span></span> <span> </span><span class="NLM_article-title">(R)-2-Phenylpyrrolidine substituted imidazopyridazines: a new class of potent and selective pan-TRK inhibitors</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">562</span>– <span class="NLM_lpage">567</span>, <span class="refDoi"> DOI: 10.1021/acsmedchemlett.5b00050</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.5b00050" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=1%3ACAS%3A528%3ADC%252BC2MXksVOrsbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=562-567&author=H.-S.+Choiauthor=P.+V.+Ruckerauthor=Z.+Wangauthor=Y.+Fanauthor=P.+Albaughauthor=G.+Chopiukauthor=F.+Gessierauthor=F.+Sunauthor=F.+Adrianauthor=G.+Liuauthor=T.+Hoodauthor=N.+Liauthor=Y.+Jiaauthor=J.+Cheauthor=S.+McCormackauthor=A.+Liauthor=J.+Liauthor=A.+Steffyauthor=A.+Culazzoauthor=C.+Tompkinsauthor=V.+Phungauthor=A.+Kreuschauthor=M.+Luauthor=B.+Huauthor=A.+Chaudharyauthor=M.+Prashadauthor=T.+Tuntlandauthor=B.+Liuauthor=J.+Harrisauthor=H.+M.+Seidelauthor=J.+Lorenauthor=V.+Molteni&title=%28R%29-2-Phenylpyrrolidine+substituted+imidazopyridazines%3A+a+new+class+of+potent+and+selective+pan-TRK+inhibitors&doi=10.1021%2Facsmedchemlett.5b00050"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">(R)-2-Phenylpyrrolidine Substituted Imidazopyridazines: A New Class of Potent and Selective Pan-TRK Inhibitors</span></div><div class="casAuthors">Choi, Ha-Soon; Rucker, Paul V.; Wang, Zhicheng; Fan, Yi; Albaugh, Pamela; Chopiuk, Greg; Gessier, Francois; Sun, Fangxian; Adrian, Francisco; Liu, Guoxun; Hood, Tami; Li, Nanxin; Jia, Yong; Che, Jianwei; McCormack, Susan; Li, Allen; Li, Jie; Steffy, Auzon; Culazzo, AnneMarie; Tompkins, Celine; Phung, Van; Kreusch, Andreas; Lu, Min; Hu, Bin; Chaudhary, Apurva; Prashad, Mahavir; Tuntland, Tove; Liu, Bo; Harris, Jennifer; Seidel, H. Martin; Loren, Jon; Molteni, Valentina</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">562-567</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Deregulated kinase activities of tropomyosin receptor kinase (TRK) family members have been shown to be assocd. with tumorigenesis and poor prognosis in a variety of cancer types.  In particular, several chromosomal rearrangements involving TRKA have been reported in colorectal, papillary thyroid, glioblastoma, melanoma, and lung tissue that are believed to be the key oncogenic driver in these tumors.  By screening the Novartis compd. collection, a novel imidazopyridazine TRK inhibitor was identified that served as a launching point for drug optimization.  Structure guided drug design led to the identification of (R)-2-phenylpyrrolidine substituted imidazopyridazines, e.g., I, as a series of potent, selective, orally bioavailable pan-TRK inhibitors achieving tumor regression in rats bearing KM12 xenografts.  From this work the (R)-2-phenylpyrrolidine has emerged as an ideal moiety to incorporate in bicyclic TRK inhibitors by virtue of its shape complementarity to the hydrophobic pocket of TRKs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrBOGdLZcZ2HbVg90H21EOLACvtfcHk0liz-uayyv03yA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXksVOrsbo%253D&md5=e2eacf85a3e762e8f2474a568e6d1b39</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.5b00050&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.5b00050%26sid%3Dliteratum%253Aachs%26aulast%3DChoi%26aufirst%3DH.-S.%26aulast%3DRucker%26aufirst%3DP.%2BV.%26aulast%3DWang%26aufirst%3DZ.%26aulast%3DFan%26aufirst%3DY.%26aulast%3DAlbaugh%26aufirst%3DP.%26aulast%3DChopiuk%26aufirst%3DG.%26aulast%3DGessier%26aufirst%3DF.%26aulast%3DSun%26aufirst%3DF.%26aulast%3DAdrian%26aufirst%3DF.%26aulast%3DLiu%26aufirst%3DG.%26aulast%3DHood%26aufirst%3DT.%26aulast%3DLi%26aufirst%3DN.%26aulast%3DJia%26aufirst%3DY.%26aulast%3DChe%26aufirst%3DJ.%26aulast%3DMcCormack%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DA.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DSteffy%26aufirst%3DA.%26aulast%3DCulazzo%26aufirst%3DA.%26aulast%3DTompkins%26aufirst%3DC.%26aulast%3DPhung%26aufirst%3DV.%26aulast%3DKreusch%26aufirst%3DA.%26aulast%3DLu%26aufirst%3DM.%26aulast%3DHu%26aufirst%3DB.%26aulast%3DChaudhary%26aufirst%3DA.%26aulast%3DPrashad%26aufirst%3DM.%26aulast%3DTuntland%26aufirst%3DT.%26aulast%3DLiu%26aufirst%3DB.%26aulast%3DHarris%26aufirst%3DJ.%26aulast%3DSeidel%26aufirst%3DH.%2BM.%26aulast%3DLoren%26aufirst%3DJ.%26aulast%3DMolteni%26aufirst%3DV.%26atitle%3D%2528R%2529-2-Phenylpyrrolidine%2520substituted%2520imidazopyridazines%253A%2520a%2520new%2520class%2520of%2520potent%2520and%2520selective%2520pan-TRK%2520inhibitors%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2015%26volume%3D6%26spage%3D562%26epage%3D567%26doi%3D10.1021%2Facsmedchemlett.5b00050" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref61"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref61'); return false;" data-citation="" class="refNumLink">61</a></strong><div class="NLM_citation" id="cit61"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fischer, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Böhm, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lydeard, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stadler, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cavadini, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nagel, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Serluca, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Acker, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lingaraju, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tichkule, R. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schebesta, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Forrester, W. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schirle, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hassiepen, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ottl, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hild, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beckwith, R. E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harper, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jenkins, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomä, N. H.</span></span> <span> </span><span class="NLM_article-title">Structure of the DDB1-CRBN E3 ubiquitin ligase in complex with thalidomide</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>512</i></span>,  <span class="NLM_fpage">49</span>– <span class="NLM_lpage">53</span>, <span class="refDoi"> DOI: 10.1038/nature13527</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=10.1038%2Fnature13527" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=25043012" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=1%3ACAS%3A528%3ADC%252BC2cXht12ms7%252FP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=512&publication_year=2014&pages=49-53&author=E.+S.+Fischerauthor=K.+B%C3%B6hmauthor=J.+R.+Lydeardauthor=H.+Yangauthor=M.+B.+Stadlerauthor=S.+Cavadiniauthor=J.+Nagelauthor=F.+Serlucaauthor=V.+Ackerauthor=G.+M.+Lingarajuauthor=R.+B.+Tichkuleauthor=M.+Schebestaauthor=W.+C.+Forresterauthor=M.+Schirleauthor=U.+Hassiepenauthor=J.+Ottlauthor=M.+Hildauthor=R.+E.+J.+Beckwithauthor=J.+W.+Harperauthor=J.+L.+Jenkinsauthor=N.+H.+Thom%C3%A4&title=Structure+of+the+DDB1-CRBN+E3+ubiquitin+ligase+in+complex+with+thalidomide&doi=10.1038%2Fnature13527"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">Structure of the DDB1-CRBN E3 ubiquitin ligase in complex with thalidomide</span></div><div class="casAuthors">Fischer, Eric S.; Bohm, Kerstin; Lydeard, John R.; Yang, Haidi; Stadler, Michael B.; Cavadini, Simone; Nagel, Jane; Serluca, Fabrizio; Acker, Vincent; Lingaraju, Gondichatnahalli M.; Tichkule, Ritesh B.; Schebesta, Michael; Forrester, William C.; Schirle, Markus; Hassiepen, Ulrich; Ottl, Johannes; Hild, Marc; Beckwith, Rohan E. J.; Harper, J. Wade; Jenkins, Jeremy L.; Thoma, Nicolas H.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">512</span>
        (<span class="NLM_cas:issue">7512</span>),
    <span class="NLM_cas:pages">49-53</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">In the 1950s, the drug thalidomide, administered as a sedative to pregnant women, led to the birth of thousands of children with multiple defects.  Despite the teratogenicity of thalidomide and its derivs. lenalidomide and pomalidomide, these immunomodulatory drugs (IMiDs) recently emerged as effective treatments for multiple myeloma and 5q-deletion-assocd. dysplasia.  IMiDs target the E3 ubiquitin ligase CUL4-RBX1-DDB1-CRBN (known as CRL4CRBN) and promote the ubiquitination of the IKAROS family transcription factors IKZF1 and IKZF3 by CRL4CRBN.  Here we present crystal structures of the DDB1-CRBN complex bound to thalidomide, lenalidomide and pomalidomide.  The structure establishes that CRBN is a substrate receptor within CRL4CRBN and enantioselectively binds IMiDs.  Using an unbiased screen, we identified the homeobox transcription factor MEIS2 as an endogenous substrate of CRL4CRBN.  Our studies suggest that IMiDs block endogenous substrates (MEIS2) from binding to CRL4CRBN while the ligase complex is recruiting IKZF1 or IKZF3 for degrdn.  This dual activity implies that small mols. can modulate an E3 ubiquitin ligase and thereby upregulate or downregulate the ubiquitination of proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpGt2Y9GecKr7Vg90H21EOLACvtfcHk0ljaU59YxBvBuw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXht12ms7%252FP&md5=1c1eaa48bef87463cdd3f2b1e1bd400d</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1038%2Fnature13527&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature13527%26sid%3Dliteratum%253Aachs%26aulast%3DFischer%26aufirst%3DE.%2BS.%26aulast%3DB%25C3%25B6hm%26aufirst%3DK.%26aulast%3DLydeard%26aufirst%3DJ.%2BR.%26aulast%3DYang%26aufirst%3DH.%26aulast%3DStadler%26aufirst%3DM.%2BB.%26aulast%3DCavadini%26aufirst%3DS.%26aulast%3DNagel%26aufirst%3DJ.%26aulast%3DSerluca%26aufirst%3DF.%26aulast%3DAcker%26aufirst%3DV.%26aulast%3DLingaraju%26aufirst%3DG.%2BM.%26aulast%3DTichkule%26aufirst%3DR.%2BB.%26aulast%3DSchebesta%26aufirst%3DM.%26aulast%3DForrester%26aufirst%3DW.%2BC.%26aulast%3DSchirle%26aufirst%3DM.%26aulast%3DHassiepen%26aufirst%3DU.%26aulast%3DOttl%26aufirst%3DJ.%26aulast%3DHild%26aufirst%3DM.%26aulast%3DBeckwith%26aufirst%3DR.%2BE.%2BJ.%26aulast%3DHarper%26aufirst%3DJ.%2BW.%26aulast%3DJenkins%26aufirst%3DJ.%2BL.%26aulast%3DThom%25C3%25A4%26aufirst%3DN.%2BH.%26atitle%3DStructure%2520of%2520the%2520DDB1-CRBN%2520E3%2520ubiquitin%2520ligase%2520in%2520complex%2520with%2520thalidomide%26jtitle%3DNature%26date%3D2014%26volume%3D512%26spage%3D49%26epage%3D53%26doi%3D10.1038%2Fnature13527" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref62"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref62'); return false;" data-citation="" class="refNumLink">62</a></strong><div class="NLM_citation" id="cit62"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Churcher, I.</span></span> <span> </span><span class="NLM_article-title">Protac-induced protein degradation in drug discovery: breaking the rules or just making new ones?</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">444</span>– <span class="NLM_lpage">452</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b01272</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b01272" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvVWqtbvJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=444-452&author=I.+Churcher&title=Protac-induced+protein+degradation+in+drug+discovery%3A+breaking+the+rules+or+just+making+new+ones%3F&doi=10.1021%2Facs.jmedchem.7b01272"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit62R"><div class="casContent"><span class="casTitleNuber">62</span><div class="casTitle"><span class="NLM_cas:atitle">Protac-Induced Protein Degradation in Drug Discovery: Breaking the Rules or Just Making New Ones?</span></div><div class="casAuthors">Churcher, Ian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">444-452</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Targeted protein degrdn., using bifunctional small mols. (Protacs) to remove specific proteins from within cells, has emerged as a novel drug discovery strategy with the potential to offer therapeutic interventions not achievable with existing approaches.  In this perspective, the brief history of the field is surveyed from a drug discovery perspective with a focus on the key advances in knowledge which have led to the definition and exemplification of protein degrdn. concepts, and their resulting applications to medicine discovery.  The approach has the potential to bring disruptive change to drug discovery - the many potential advantages and outstanding challenges which lie ahead of this technol. are discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr9n8I_0MhtkbVg90H21EOLACvtfcHk0ljaU59YxBvBuw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvVWqtbvJ&md5=e60f1777de8bd860c5001137d8583f1f</span></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b01272&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b01272%26sid%3Dliteratum%253Aachs%26aulast%3DChurcher%26aufirst%3DI.%26atitle%3DProtac-induced%2520protein%2520degradation%2520in%2520drug%2520discovery%253A%2520breaking%2520the%2520rules%2520or%2520just%2520making%2520new%2520ones%253F%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D444%26epage%3D452%26doi%3D10.1021%2Facs.jmedchem.7b01272" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref63"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref63'); return false;" data-citation="" class="refNumLink">63</a></strong><div class="NLM_citation" id="cit63"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Scott, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rooney, T. P. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bayle, E. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mirza, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Willems, H. M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clarke, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andrews, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skidmore, J.</span></span> <span> </span><span class="NLM_article-title">Systematic investigation of the permeability of androgen receptor PROTACs</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">1539</span>, <span class="refDoi"> DOI: 10.1021/acsmedchemlett.0c00194</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.0c00194" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhtFWmt7%252FO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2020&pages=1539&author=D.+E.+Scottauthor=T.+P.+C.+Rooneyauthor=E.+D.+Bayleauthor=T.+Mirzaauthor=H.+M.+G.+Willemsauthor=J.+H.+Clarkeauthor=S.+P.+Andrewsauthor=J.+Skidmore&title=Systematic+investigation+of+the+permeability+of+androgen+receptor+PROTACs&doi=10.1021%2Facsmedchemlett.0c00194"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit63R"><div class="casContent"><span class="casTitleNuber">63</span><div class="casTitle"><span class="NLM_cas:atitle">Systematic Investigation of the Permeability of Androgen Receptor PROTACs</span></div><div class="casAuthors">Scott, Duncan E.; Rooney, Timothy P. C.; Bayle, Elliott D.; Mirza, Tashfina; Willems, Henriette M. G.; Clarke, Jonathan H.; Andrews, Stephen P.; Skidmore, John</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1539-1547</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Bifunctional mols. known as PROTACs simultaneously bind an E3 ligase and a protein of interest to direct ubiquitination and clearance of that protein, and they have emerged in the past decade as an exciting new paradigm in drug discovery.  In order to investigate the permeability and properties of these large mols., we synthesized two panels of PROTAC mols., constructed from a range of protein-target ligands, linkers, and E3 ligase ligands.  The androgen receptor, which is a well-studied protein in the PROTAC field was used as a model system.  The physicochem. properties and permeability of PROTACs are discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoG6Q_chyrzDLVg90H21EOLACvtfcHk0ljaU59YxBvBuw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhtFWmt7%252FO&md5=26c8ea71901a57a09c03f7fb79bbe2c9</span></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.0c00194&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.0c00194%26sid%3Dliteratum%253Aachs%26aulast%3DScott%26aufirst%3DD.%2BE.%26aulast%3DRooney%26aufirst%3DT.%2BP.%2BC.%26aulast%3DBayle%26aufirst%3DE.%2BD.%26aulast%3DMirza%26aufirst%3DT.%26aulast%3DWillems%26aufirst%3DH.%2BM.%2BG.%26aulast%3DClarke%26aufirst%3DJ.%2BH.%26aulast%3DAndrews%26aufirst%3DS.%2BP.%26aulast%3DSkidmore%26aufirst%3DJ.%26atitle%3DSystematic%2520investigation%2520of%2520the%2520permeability%2520of%2520androgen%2520receptor%2520PROTACs%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2020%26volume%3D11%26spage%3D1539%26doi%3D10.1021%2Facsmedchemlett.0c00194" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref64"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref64'); return false;" data-citation="" class="refNumLink">64</a></strong><div class="NLM_citation" id="cit64"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Edmondson, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fallan, C.</span></span> <span> </span><span class="NLM_article-title">Proteolysis targeting chimeras (PROTACs) in ‘beyond rule-of-five’ chemical space: Recent progress and future challenges</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">1555</span>– <span class="NLM_lpage">1564</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2019.04.030</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=10.1016%2Fj.bmcl.2019.04.030" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=31047748" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=1%3ACAS%3A528%3ADC%252BC1MXos12gs74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2019&pages=1555-1564&author=S.+D.+Edmondsonauthor=B.+Yangauthor=C.+Fallan&title=Proteolysis+targeting+chimeras+%28PROTACs%29+in+%E2%80%98beyond+rule-of-five%E2%80%99+chemical+space%3A+Recent+progress+and+future+challenges&doi=10.1016%2Fj.bmcl.2019.04.030"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit64R"><div class="casContent"><span class="casTitleNuber">64</span><div class="casTitle"><span class="NLM_cas:atitle">Proteolysis targeting chimeras (PROTACs) in 'beyond rule-of-five' chemical space: Recent progress and future challenges</span></div><div class="casAuthors">Edmondson, Scott D.; Yang, Bin; Fallan, Charlene</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">1555-1564</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Proteolysis targeting chimeras (PROTACs) are heterobifunctional compds. with mol. wts. and other properties that lie outside the classic 'rule-of-five' space.  Consequently, PROTACs have unique challenges assocd. with their development as potential therapeutic agents.  This review summarizes and analyzes a representative set of recent PROTACs and highlights some of the potential future challenges facing this promising modality.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqDcS0gIul-K7Vg90H21EOLACvtfcHk0lhD-ynvR42O7Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXos12gs74%253D&md5=62a74500178bd711b1ddaa33d731b960</span></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2019.04.030&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2019.04.030%26sid%3Dliteratum%253Aachs%26aulast%3DEdmondson%26aufirst%3DS.%2BD.%26aulast%3DYang%26aufirst%3DB.%26aulast%3DFallan%26aufirst%3DC.%26atitle%3DProteolysis%2520targeting%2520chimeras%2520%2528PROTACs%2529%2520in%2520%25E2%2580%2598beyond%2520rule-of-five%25E2%2580%2599%2520chemical%2520space%253A%2520Recent%2520progress%2520and%2520future%2520challenges%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2019%26volume%3D29%26spage%3D1555%26epage%3D1564%26doi%3D10.1016%2Fj.bmcl.2019.04.030" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref65"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref65'); return false;" data-citation="" class="refNumLink">65</a></strong><div class="NLM_citation" id="cit65"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pike, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williamson, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harlfinger, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McGinnity, D. F.</span></span> <span> </span><span class="NLM_article-title">Optimising proteolysis-targeting chimeras (PROTACs) for oral drug delivery: a drug metabolism and pharmacokinetics perspective</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2020</span>, <span class="refDoi"> DOI: 10.1016/j.drudis.2020.07.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=10.1016%2Fj.drudis.2020.07.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=32693163" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2020&author=A.+Pikeauthor=B.+Williamsonauthor=S.+Harlfingerauthor=S.+Martinauthor=D.+F.+McGinnity&title=Optimising+proteolysis-targeting+chimeras+%28PROTACs%29+for+oral+drug+delivery%3A+a+drug+metabolism+and+pharmacokinetics+perspective&doi=10.1016%2Fj.drudis.2020.07.013"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2020.07.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2020.07.013%26sid%3Dliteratum%253Aachs%26aulast%3DPike%26aufirst%3DA.%26aulast%3DWilliamson%26aufirst%3DB.%26aulast%3DHarlfinger%26aufirst%3DS.%26aulast%3DMartin%26aufirst%3DS.%26aulast%3DMcGinnity%26aufirst%3DD.%2BF.%26atitle%3DOptimising%2520proteolysis-targeting%2520chimeras%2520%2528PROTACs%2529%2520for%2520oral%2520drug%2520delivery%253A%2520a%2520drug%2520metabolism%2520and%2520pharmacokinetics%2520perspective%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2020%26doi%3D10.1016%2Fj.drudis.2020.07.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref66"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref66'); return false;" data-citation="" class="refNumLink">66</a></strong><div class="NLM_citation" id="cit66"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ardini, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bosotti, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Borgia, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Ponti, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Somaschini, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cammarota, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amboldi, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raddrizzani, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Milani, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Magnaghi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ballinari, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Casero, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gasparri, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Banfi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Avanzi, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saccardo, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alzani, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bandiera, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Felder, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Donati, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pesenti, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sartore-Bianchi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gambacorta, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pierotti, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siena, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Veronese, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galvani, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Isacchi, A.</span></span> <span> </span><span class="NLM_article-title">The TPM3-NTRK1 rearrangement is a recurring event in colorectal carcinoma and is associated with tumor sensitivity to TRKA kinase inhibition</span>. <i>Mol. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">1495</span>– <span class="NLM_lpage">1507</span>, <span class="refDoi"> DOI: 10.1016/j.molonc.2014.06.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=10.1016%2Fj.molonc.2014.06.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=24962792" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtVaktb%252FI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2014&pages=1495-1507&author=E.+Ardiniauthor=R.+Bosottiauthor=A.+L.+Borgiaauthor=C.+De+Pontiauthor=A.+Somaschiniauthor=R.+Cammarotaauthor=N.+Amboldiauthor=L.+Raddrizzaniauthor=A.+Milaniauthor=P.+Magnaghiauthor=D.+Ballinariauthor=D.+Caseroauthor=F.+Gasparriauthor=P.+Banfiauthor=N.+Avanziauthor=M.+B.+Saccardoauthor=R.+Alzaniauthor=T.+Bandieraauthor=E.+Felderauthor=D.+Donatiauthor=E.+Pesentiauthor=A.+Sartore-Bianchiauthor=M.+Gambacortaauthor=M.+A.+Pierottiauthor=S.+Sienaauthor=S.+Veroneseauthor=A.+Galvaniauthor=A.+Isacchi&title=The+TPM3-NTRK1+rearrangement+is+a+recurring+event+in+colorectal+carcinoma+and+is+associated+with+tumor+sensitivity+to+TRKA+kinase+inhibition&doi=10.1016%2Fj.molonc.2014.06.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit66R"><div class="casContent"><span class="casTitleNuber">66</span><div class="casTitle"><span class="NLM_cas:atitle">The TPM3-NTRK1 rearrangement is a recurring event in colorectal carcinoma and is associated with tumor sensitivity to TRKA kinase inhibition</span></div><div class="casAuthors">Ardini, Elena; Bosotti, Roberta; Borgia, Andrea Lombardi; De Ponti, Cristina; Somaschini, Alessio; Cammarota, Rosaria; Amboldi, Nadia; Raddrizzani, Laura; Milani, Andrea; Magnaghi, Paola; Ballinari, Dario; Casero, Daniele; Gasparri, Fabio; Banfi, Patrizia; Avanzi, Nilla; Saccardo, Maria B.; Alzani, Rachele; Bandiera, Tiziano; Felder, Eduard; Donati, Daniele; Pesenti, Enrico; Sartore-Bianchi, Andrea; Gambacorta, Marcello; Pierotti, Marco A.; Siena, Salvatore; Veronese, Silvio; Galvani, Arturo; Isacchi, Antonella</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Oncology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1495-1507</span>CODEN:
                <span class="NLM_cas:coden">MOONC3</span>;
        ISSN:<span class="NLM_cas:issn">1574-7891</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The NTRK1 gene encodes Tropomyosin-related kinase A (TRKA), the high-affinity Nerve Growth Factor Receptor.  NTRK1 was originally isolated from a colorectal carcinoma (CRC) sample as component of a somatic rearrangement (TPM3-NTRK1) resulting in expression of the oncogenic chimeric protein TPM3-TRKA, but there has been no subsequent report regarding the relevance of this oncogene in CRC.  The KM12 human CRC cell line expresses the chimeric TPM3-TRKA protein and is hypersensitive to TRKA kinase inhibition.  We report the detailed characterization of the TPM3-NTRK1 genomic rearrangement in KM12 cells and through a cellular screening approach, the identification of NMS-P626, a novel highly potent and selective TRKA inhibitor.  NMS-P626 suppressed TPM3-TRKA phosphorylation and downstream signaling in KM12 cells and showed remarkable antitumor activity in mice bearing KM12 tumors.  Finally, using quant. reverse transcriptase PCR and immunohistochem. (IHC) we identified the TPM3-NTRK1 rearrangement in a CRC clin. sample, therefore suggesting that this chromosomal translocation is indeed a low frequency recurring event in CRC and that such patients might benefit from therapy with TRKA kinase inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo-EhukkQ24OrVg90H21EOLACvtfcHk0lhD-ynvR42O7Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtVaktb%252FI&md5=2a9de2410f69e2eb8ff84c9516828984</span></div><a href="/servlet/linkout?suffix=cit66&amp;dbid=16384&amp;doi=10.1016%2Fj.molonc.2014.06.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molonc.2014.06.001%26sid%3Dliteratum%253Aachs%26aulast%3DArdini%26aufirst%3DE.%26aulast%3DBosotti%26aufirst%3DR.%26aulast%3DBorgia%26aufirst%3DA.%2BL.%26aulast%3DDe%2BPonti%26aufirst%3DC.%26aulast%3DSomaschini%26aufirst%3DA.%26aulast%3DCammarota%26aufirst%3DR.%26aulast%3DAmboldi%26aufirst%3DN.%26aulast%3DRaddrizzani%26aufirst%3DL.%26aulast%3DMilani%26aufirst%3DA.%26aulast%3DMagnaghi%26aufirst%3DP.%26aulast%3DBallinari%26aufirst%3DD.%26aulast%3DCasero%26aufirst%3DD.%26aulast%3DGasparri%26aufirst%3DF.%26aulast%3DBanfi%26aufirst%3DP.%26aulast%3DAvanzi%26aufirst%3DN.%26aulast%3DSaccardo%26aufirst%3DM.%2BB.%26aulast%3DAlzani%26aufirst%3DR.%26aulast%3DBandiera%26aufirst%3DT.%26aulast%3DFelder%26aufirst%3DE.%26aulast%3DDonati%26aufirst%3DD.%26aulast%3DPesenti%26aufirst%3DE.%26aulast%3DSartore-Bianchi%26aufirst%3DA.%26aulast%3DGambacorta%26aufirst%3DM.%26aulast%3DPierotti%26aufirst%3DM.%2BA.%26aulast%3DSiena%26aufirst%3DS.%26aulast%3DVeronese%26aufirst%3DS.%26aulast%3DGalvani%26aufirst%3DA.%26aulast%3DIsacchi%26aufirst%3DA.%26atitle%3DThe%2520TPM3-NTRK1%2520rearrangement%2520is%2520a%2520recurring%2520event%2520in%2520colorectal%2520carcinoma%2520and%2520is%2520associated%2520with%2520tumor%2520sensitivity%2520to%2520TRKA%2520kinase%2520inhibition%26jtitle%3DMol.%2520Oncol.%26date%3D2014%26volume%3D8%26spage%3D1495%26epage%3D1507%26doi%3D10.1016%2Fj.molonc.2014.06.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref67"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref67'); return false;" data-citation="" class="refNumLink">67</a></strong><div class="NLM_citation" id="cit67"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Matyskiela, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ito, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pagarigan, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, C.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, N.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Houston, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carmel, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tran, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riley, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nosaka, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lander, G. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gaidarova, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruchelman, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Handa, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carmichael, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daniel, T. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cathers, B. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lopez-Girona, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chamberlain, P. P.</span></span> <span> </span><span class="NLM_article-title">A novel cereblon modulator recruits GSPT1 to the CRL4CRBN ubiquitin ligase</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>535</i></span>,  <span class="NLM_fpage">252</span>– <span class="NLM_lpage">257</span>, <span class="refDoi"> DOI: 10.1038/nature18611</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=10.1038%2Fnature18611" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=27338790" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtVCltbnI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=535&publication_year=2016&pages=252-257&author=M.+E.+Matyskielaauthor=G.+Luauthor=T.+Itoauthor=B.+Pagariganauthor=C.-C.+Luauthor=K.+Millerauthor=W.+Fangauthor=N.-Y.+Wangauthor=D.+Nguyenauthor=J.+Houstonauthor=G.+Carmelauthor=T.+Tranauthor=M.+Rileyauthor=L.+A.+Nosakaauthor=G.+C.+Landerauthor=S.+Gaidarovaauthor=S.+Xuauthor=A.+L.+Ruchelmanauthor=H.+Handaauthor=J.+Carmichaelauthor=T.+O.+Danielauthor=B.+E.+Cathersauthor=A.+Lopez-Gironaauthor=P.+P.+Chamberlain&title=A+novel+cereblon+modulator+recruits+GSPT1+to+the+CRL4CRBN+ubiquitin+ligase&doi=10.1038%2Fnature18611"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit67R"><div class="casContent"><span class="casTitleNuber">67</span><div class="casTitle"><span class="NLM_cas:atitle">A novel cereblon modulator recruits GSPT1 to the CRL4CRBN ubiquitin ligase</span></div><div class="casAuthors">Matyskiela, Mary E.; Lu, Gang; Ito, Takumi; Pagarigan, Barbra; Lu, Chin-Chun; Miller, Karen; Fang, Wei; Wang, Nai-Yu; Nguyen, Derek; Houston, Jack; Carmel, Gilles; Tran, Tam; Riley, Mariko; Nosaka, Lyn'Al; Lander, Gabriel C.; Gaidarova, Svetlana; Xu, Shuichan; Ruchelman, Alexander L.; Handa, Hiroshi; Carmichael, James; Daniel, Thomas O.; Cathers, Brian E.; Lopez-Girona, Antonia; Chamberlain, Philip P.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">535</span>
        (<span class="NLM_cas:issue">7611</span>),
    <span class="NLM_cas:pages">252-257</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Immunomodulatory drugs bind to cereblon (CRBN) to confer differentiated substrate specificity on the CRL4CRBN E3 ubiquitin ligase.  Here we report the identification of a new cereblon modulator, CC-885, with potent anti-tumor activity.  The anti-tumor activity of CC-885 is mediated through the cereblon-dependent ubiquitination and degrdn. of the translation termination factor GSPT1.  Patient-derived acute myeloid leukemia tumor cells exhibit high sensitivity to CC-885, indicating the clin. potential of this mechanism.  Crystallog. studies of the CRBN-DDB1-CC-885-GSPT1 complex reveal that GSPT1 binds to cereblon through a surface turn contg. a glycine residue at a key position, interacting with both CC-885 and a 'hotspot' on the cereblon surface.  Although GSPT1 possesses no obvious structural, sequence or functional homol. to previously known cereblon substrates, mutational anal. and modeling indicate that the cereblon substrate Ikaros uses a similar structural feature to bind cereblon, suggesting a common motif for substrate recruitment.  These findings define a structural degron underlying cereblon 'neosubstrate' selectivity, and identify an anti-tumor target rendered druggable by cereblon modulation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqgAWiXdlJgYLVg90H21EOLACvtfcHk0lhTbJKN_nJ23Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtVCltbnI&md5=1d4e340416aa0b08fc0f123faf638c3f</span></div><a href="/servlet/linkout?suffix=cit67&amp;dbid=16384&amp;doi=10.1038%2Fnature18611&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature18611%26sid%3Dliteratum%253Aachs%26aulast%3DMatyskiela%26aufirst%3DM.%2BE.%26aulast%3DLu%26aufirst%3DG.%26aulast%3DIto%26aufirst%3DT.%26aulast%3DPagarigan%26aufirst%3DB.%26aulast%3DLu%26aufirst%3DC.-C.%26aulast%3DMiller%26aufirst%3DK.%26aulast%3DFang%26aufirst%3DW.%26aulast%3DWang%26aufirst%3DN.-Y.%26aulast%3DNguyen%26aufirst%3DD.%26aulast%3DHouston%26aufirst%3DJ.%26aulast%3DCarmel%26aufirst%3DG.%26aulast%3DTran%26aufirst%3DT.%26aulast%3DRiley%26aufirst%3DM.%26aulast%3DNosaka%26aufirst%3DL.%2BA.%26aulast%3DLander%26aufirst%3DG.%2BC.%26aulast%3DGaidarova%26aufirst%3DS.%26aulast%3DXu%26aufirst%3DS.%26aulast%3DRuchelman%26aufirst%3DA.%2BL.%26aulast%3DHanda%26aufirst%3DH.%26aulast%3DCarmichael%26aufirst%3DJ.%26aulast%3DDaniel%26aufirst%3DT.%2BO.%26aulast%3DCathers%26aufirst%3DB.%2BE.%26aulast%3DLopez-Girona%26aufirst%3DA.%26aulast%3DChamberlain%26aufirst%3DP.%2BP.%26atitle%3DA%2520novel%2520cereblon%2520modulator%2520recruits%2520GSPT1%2520to%2520the%2520CRL4CRBN%2520ubiquitin%2520ligase%26jtitle%3DNature%26date%3D2016%26volume%3D535%26spage%3D252%26epage%3D257%26doi%3D10.1038%2Fnature18611" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref68"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref68'); return false;" data-citation="" class="refNumLink">68</a></strong><div class="NLM_citation" id="cit68"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bondeson, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, B. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burslem, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buhimschi, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hines, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jaime-Figueroa, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamman, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ishchenko, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Lessons in PROTAC design from selective degradation with a promiscuous warhead</span>. <i>Cell Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">78</span>– <span class="NLM_lpage">87.e5</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2017.09.010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=10.1016%2Fj.chembiol.2017.09.010" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=29129718" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsl2lsrjN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2018&pages=78-87.e5&author=D.+P.+Bondesonauthor=B.+E.+Smithauthor=G.+M.+Burslemauthor=A.+D.+Buhimschiauthor=J.+Hinesauthor=S.+Jaime-Figueroaauthor=J.+Wangauthor=B.+D.+Hammanauthor=A.+Ishchenkoauthor=C.+M.+Crews&title=Lessons+in+PROTAC+design+from+selective+degradation+with+a+promiscuous+warhead&doi=10.1016%2Fj.chembiol.2017.09.010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit68R"><div class="casContent"><span class="casTitleNuber">68</span><div class="casTitle"><span class="NLM_cas:atitle">Lessons in PROTAC Design from Selective Degradation with a Promiscuous Warhead</span></div><div class="casAuthors">Bondeson, Daniel P.; Smith, Blake E.; Burslem, George M.; Buhimschi, Alexandru D.; Hines, John; Jaime-Figueroa, Saul; Wang, Jing; Hamman, Brian D.; Ishchenko, Alexey; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">78-87.e5</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Inhibiting protein function selectively is a major goal of modern drug discovery.  Here, we report a previously understudied benefit of small mol. proteolysis-targeting chimeras (PROTACs) that recruit E3 ubiquitin ligases to target proteins for their ubiquitination and subsequent proteasome-mediated degrdn.  Using promiscuous CRBN- and VHL-recruiting PROTACs that bind >50 kinases, we show that only a subset of bound targets is degraded.  The basis of this selectivity relies on protein-protein interactions between the E3 ubiquitin ligase and the target protein, as illustrated by engaged proteins that are not degraded as a result of unstable ternary complexes with PROTAC-recruited E3 ligases.  In contrast, weak PROTAC:target protein affinity can be stabilized by high-affinity target:PROTAC:ligase trimer interactions, leading to efficient degrdn.  This study highlights design guidelines for generating potent PROTACs as well as possibilities for degrading undruggable proteins immune to traditional small-mol. inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoISOBudOLyFrVg90H21EOLACvtfcHk0lhTbJKN_nJ23Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsl2lsrjN&md5=30fac502705a2344c4261cf531e6bf9d</span></div><a href="/servlet/linkout?suffix=cit68&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2017.09.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2017.09.010%26sid%3Dliteratum%253Aachs%26aulast%3DBondeson%26aufirst%3DD.%2BP.%26aulast%3DSmith%26aufirst%3DB.%2BE.%26aulast%3DBurslem%26aufirst%3DG.%2BM.%26aulast%3DBuhimschi%26aufirst%3DA.%2BD.%26aulast%3DHines%26aufirst%3DJ.%26aulast%3DJaime-Figueroa%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DHamman%26aufirst%3DB.%2BD.%26aulast%3DIshchenko%26aufirst%3DA.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DLessons%2520in%2520PROTAC%2520design%2520from%2520selective%2520degradation%2520with%2520a%2520promiscuous%2520warhead%26jtitle%3DCell%2520Chem.%2520Biol.%26date%3D2018%26volume%3D25%26spage%3D78%26epage%3D87.e5%26doi%3D10.1016%2Fj.chembiol.2017.09.010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref69"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref69'); return false;" data-citation="" class="refNumLink">69</a></strong><div class="NLM_citation" id="cit69"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ishoey, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chorn, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jaeger, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brand, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paulk, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bauer, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Erb, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parapatics, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bennett, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ecker, G. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winter, G. E.</span></span> <span> </span><span class="NLM_article-title">Translation termination factor GSPT1 is a phenotypically relevant off-target of heterobifunctional phthalimide degraders</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">553</span>– <span class="NLM_lpage">560</span>, <span class="refDoi"> DOI: 10.1021/acschembio.7b00969</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.7b00969" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtlyqu7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2018&pages=553-560&author=M.+Ishoeyauthor=S.+Chornauthor=N.+Singhauthor=M.+G.+Jaegerauthor=M.+Brandauthor=J.+Paulkauthor=S.+Bauerauthor=M.+A.+Erbauthor=K.+Parapaticsauthor=A.+C.+M%C3%BCllerauthor=K.+L.+Bennettauthor=G.+F.+Eckerauthor=J.+E.+Bradnerauthor=G.+E.+Winter&title=Translation+termination+factor+GSPT1+is+a+phenotypically+relevant+off-target+of+heterobifunctional+phthalimide+degraders&doi=10.1021%2Facschembio.7b00969"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit69R"><div class="casContent"><span class="casTitleNuber">69</span><div class="casTitle"><span class="NLM_cas:atitle">Translation Termination Factor GSPT1 Is a Phenotypically Relevant Off-Target of Heterobifunctional Phthalimide Degraders</span></div><div class="casAuthors">Ishoey, Mette; Chorn, Someth; Singh, Natesh; Jaeger, Martin G.; Brand, Matthias; Paulk, Joshiawa; Bauer, Sophie; Erb, Michael A.; Parapatics, Katja; Muller, Andre C.; Bennett, Keiryn L.; Ecker, Gerhard F.; Bradner, James E.; Winter, Georg E.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">553-560</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Protein degrdn. is an emerging therapeutic strategy with a unique mol. pharmacol. that enables the disruption of all functions assocd. with a target.  This is particularly relevant for proteins depending on mol. scaffolding, such as transcription factors or receptor tyrosine kinases (RTKs).  To address tractability of multiple RTKs for chem. degrdn. by the E3 ligase CUL4-RBX1-DDB1-CRBN (CRL4CRBN), we synthesized a series of phthalimide degraders based on the promiscuous kinase inhibitors sunitinib and PHA665752.  While both series failed to induce degrdn. of their consensus targets, individual mols. displayed pronounced efficacy in leukemia cell lines.  Orthogonal target identification supported by mol. docking led us to identify the translation termination factor G1 to S phase transition 1 (GSPT1) as a converging off-target, resulting from inadvertent E3 ligase modulation.  This research highlights the importance of monitoring degrdn. events that are independent of the resp. targeting ligand as a unique feature of small-mol. degraders.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq_ZxnB5rKvvLVg90H21EOLACvtfcHk0lhTbJKN_nJ23Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtlyqu7o%253D&md5=02a70959e8a8929a3c70846c1ed0200f</span></div><a href="/servlet/linkout?suffix=cit69&amp;dbid=16384&amp;doi=10.1021%2Facschembio.7b00969&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.7b00969%26sid%3Dliteratum%253Aachs%26aulast%3DIshoey%26aufirst%3DM.%26aulast%3DChorn%26aufirst%3DS.%26aulast%3DSingh%26aufirst%3DN.%26aulast%3DJaeger%26aufirst%3DM.%2BG.%26aulast%3DBrand%26aufirst%3DM.%26aulast%3DPaulk%26aufirst%3DJ.%26aulast%3DBauer%26aufirst%3DS.%26aulast%3DErb%26aufirst%3DM.%2BA.%26aulast%3DParapatics%26aufirst%3DK.%26aulast%3DM%25C3%25BCller%26aufirst%3DA.%2BC.%26aulast%3DBennett%26aufirst%3DK.%2BL.%26aulast%3DEcker%26aufirst%3DG.%2BF.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26aulast%3DWinter%26aufirst%3DG.%2BE.%26atitle%3DTranslation%2520termination%2520factor%2520GSPT1%2520is%2520a%2520phenotypically%2520relevant%2520off-target%2520of%2520heterobifunctional%2520phthalimide%2520degraders%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2018%26volume%3D13%26spage%3D553%26epage%3D560%26doi%3D10.1021%2Facschembio.7b00969" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref70"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref70'); return false;" data-citation="" class="refNumLink">70</a></strong><div class="NLM_citation" id="cit70"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aguilar, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span> <span> </span><span class="NLM_article-title">Simple structural modifications converting a bona fide MDM2 PROTAC degrader into a molecular glue molecule: a cautionary tale in the design of PROTAC degraders</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">9471</span>– <span class="NLM_lpage">9487</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b00846</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b00846" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhvVGgtb7K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=9471-9487&author=J.+Yangauthor=Y.+Liauthor=A.+Aguilarauthor=Z.+Liuauthor=C.-Y.+Yangauthor=S.+Wang&title=Simple+structural+modifications+converting+a+bona+fide+MDM2+PROTAC+degrader+into+a+molecular+glue+molecule%3A+a+cautionary+tale+in+the+design+of+PROTAC+degraders&doi=10.1021%2Facs.jmedchem.9b00846"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit70R"><div class="casContent"><span class="casTitleNuber">70</span><div class="casTitle"><span class="NLM_cas:atitle">Simple Structural Modifications Converting a Bona fide MDM2 PROTAC Degrader into a Molecular Glue Molecule: A Cautionary Tale in the Design of PROTAC Degraders</span></div><div class="casAuthors">Yang, Jiuling; Li, Yangbing; Aguilar, Angelo; Liu, Zhaomin; Yang, Chao-Yie; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">9471-9487</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Inducing protein degrdn. by proteolysis targeting chimeras (PROTACs) has gained tremendous momentum for its promise to discover and develop new therapies.  Based upon our previously reported PROTAC MDM2 degraders, we have designed and synthesized addnl. analogs.  Surprisingly, we found that simple structural modifications of MD-222, a bona fide MDM2 PROTAC degrader, converts it into a "mol. glue", as exemplified by MG-277.  MG-277 induces only moderate MDM2 degrdn. and fails to activate wild-type p53 but is highly potent in inhibition of tumor cell growth in a p53-independent manner.  Our mechanistic investigation established that MG-277 is not a PROTAC MDM2 degrader but instead works as a mol. glue, inducing degrdn. of a translation termination factor, GSPT1 to achieve its potent anticancer activity.  Our study provides the first example that simple structural modifications can convert a bona fide PROTAC degrader into a mol. glue compd., which has a completely different mechanism of action.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrH5W2xSSGs6rVg90H21EOLACvtfcHk0ljrD0A7S8XLQQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhvVGgtb7K&md5=8a3e36a4fc3110c798d379b70d6203fc</span></div><a href="/servlet/linkout?suffix=cit70&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b00846%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DAguilar%26aufirst%3DA.%26aulast%3DLiu%26aufirst%3DZ.%26aulast%3DYang%26aufirst%3DC.-Y.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DSimple%2520structural%2520modifications%2520converting%2520a%2520bona%2520fide%2520MDM2%2520PROTAC%2520degrader%2520into%2520a%2520molecular%2520glue%2520molecule%253A%2520a%2520cautionary%2520tale%2520in%2520the%2520design%2520of%2520PROTAC%2520degraders%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D9471%26epage%3D9487%26doi%3D10.1021%2Facs.jmedchem.9b00846" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref71"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref71'); return false;" data-citation="" class="refNumLink">71</a></strong><div class="NLM_citation" id="cit71"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Soucy, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, P. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Milhollen, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berger, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gavin, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adhikari, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brownell, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burke, K. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cardin, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Critchley, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cullis, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doucette, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garnsey, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gaulin, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gershman, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lublinsky, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McDonald, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mizutani, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Narayanan, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olhava, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peluso, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rezaei, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sintchak, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Talreja, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Traore, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vyskocil, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weatherhead, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dick, L. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Claiborne, C. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rolfe, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bolen, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Langston, S. P.</span></span> <span> </span><span class="NLM_article-title">An inhibitor of NEDD8-activating enzyme as a new approach to treat cancer</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>458</i></span>,  <span class="NLM_fpage">732</span>– <span class="NLM_lpage">736</span>, <span class="refDoi"> DOI: 10.1038/nature07884</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=10.1038%2Fnature07884" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=19360080" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=1%3ACAS%3A528%3ADC%252BD1MXksVenu7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=458&publication_year=2009&pages=732-736&author=T.+A.+Soucyauthor=P.+G.+Smithauthor=M.+A.+Milhollenauthor=A.+J.+Bergerauthor=J.+M.+Gavinauthor=S.+Adhikariauthor=J.+E.+Brownellauthor=K.+E.+Burkeauthor=D.+P.+Cardinauthor=S.+Critchleyauthor=C.+A.+Cullisauthor=A.+Doucetteauthor=J.+J.+Garnseyauthor=J.+L.+Gaulinauthor=R.+E.+Gershmanauthor=A.+R.+Lublinskyauthor=A.+McDonaldauthor=H.+Mizutaniauthor=U.+Narayananauthor=E.+J.+Olhavaauthor=S.+Pelusoauthor=M.+Rezaeiauthor=M.+D.+Sintchakauthor=T.+Talrejaauthor=M.+P.+Thomasauthor=T.+Traoreauthor=S.+Vyskocilauthor=G.+S.+Weatherheadauthor=J.+Yuauthor=J.+Zhangauthor=L.+R.+Dickauthor=C.+F.+Claiborneauthor=M.+Rolfeauthor=J.+B.+Bolenauthor=S.+P.+Langston&title=An+inhibitor+of+NEDD8-activating+enzyme+as+a+new+approach+to+treat+cancer&doi=10.1038%2Fnature07884"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit71R"><div class="casContent"><span class="casTitleNuber">71</span><div class="casTitle"><span class="NLM_cas:atitle">An inhibitor of NEDD8-activating enzyme as a new approach to treat cancer</span></div><div class="casAuthors">Soucy, Teresa A.; Smith, Peter G.; Milhollen, Michael A.; Berger, Allison J.; Gavin, James M.; Adhikari, Sharmila; Brownell, James E.; Burke, Kristine E.; Cardin, David P.; Critchley, Stephen; Cullis, Courtney A.; Doucette, Amanda; Garnsey, James J.; Gaulin, Jeffrey L.; Gershman, Rachel E.; Lublinsky, Anna R.; McDonald, Alice; Mizutani, Hirotake; Narayanan, Usha; Olhava, Edward J.; Peluso, Stephane; Rezaei, Mansoureh; Sintchak, Michael D.; Talreja, Tina; Thomas, Michael P.; Traore, Tary; Vyskocil, Stepan; Weatherhead, Gabriel S.; Yu, Jie; Zhang, Julie; Dick, Lawrence R.; Claiborne, Christopher F.; Rolfe, Mark; Bolen, Joseph B.; Langston, Steven P.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">458</span>
        (<span class="NLM_cas:issue">7239</span>),
    <span class="NLM_cas:pages">732-736</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The clin. development of an inhibitor of cellular proteasome function suggests that compds. targeting other components of the ubiquitin-proteasome system might prove useful for the treatment of human malignancies.  NEDD8-activating enzyme (NAE) is an essential component of the NEDD8 conjugation pathway that controls the activity of the cullin-RING subtype of ubiquitin ligases, thereby regulating the turnover of a subset of proteins upstream of the proteasome.  Substrates of cullin-RING ligases have important roles in cellular processes assocd. with cancer cell growth and survival pathways.  Here we describe MLN4924, a potent and selective inhibitor of NAE.  MLN4924 disrupts cullin-RING ligase-mediated protein turnover leading to apoptotic death in human tumor cells by a new mechanism of action, the deregulation of S-phase DNA synthesis.  MLN4924 suppressed the growth of human tumor xenografts in mice at compd. exposures that were well tolerated.  Our data suggest that NAE inhibitors may hold promise for the treatment of cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr8fx7OnZtYjbVg90H21EOLACvtfcHk0ljrD0A7S8XLQQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXksVenu7Y%253D&md5=365d4b4feac3994849050e00f1aae5e0</span></div><a href="/servlet/linkout?suffix=cit71&amp;dbid=16384&amp;doi=10.1038%2Fnature07884&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature07884%26sid%3Dliteratum%253Aachs%26aulast%3DSoucy%26aufirst%3DT.%2BA.%26aulast%3DSmith%26aufirst%3DP.%2BG.%26aulast%3DMilhollen%26aufirst%3DM.%2BA.%26aulast%3DBerger%26aufirst%3DA.%2BJ.%26aulast%3DGavin%26aufirst%3DJ.%2BM.%26aulast%3DAdhikari%26aufirst%3DS.%26aulast%3DBrownell%26aufirst%3DJ.%2BE.%26aulast%3DBurke%26aufirst%3DK.%2BE.%26aulast%3DCardin%26aufirst%3DD.%2BP.%26aulast%3DCritchley%26aufirst%3DS.%26aulast%3DCullis%26aufirst%3DC.%2BA.%26aulast%3DDoucette%26aufirst%3DA.%26aulast%3DGarnsey%26aufirst%3DJ.%2BJ.%26aulast%3DGaulin%26aufirst%3DJ.%2BL.%26aulast%3DGershman%26aufirst%3DR.%2BE.%26aulast%3DLublinsky%26aufirst%3DA.%2BR.%26aulast%3DMcDonald%26aufirst%3DA.%26aulast%3DMizutani%26aufirst%3DH.%26aulast%3DNarayanan%26aufirst%3DU.%26aulast%3DOlhava%26aufirst%3DE.%2BJ.%26aulast%3DPeluso%26aufirst%3DS.%26aulast%3DRezaei%26aufirst%3DM.%26aulast%3DSintchak%26aufirst%3DM.%2BD.%26aulast%3DTalreja%26aufirst%3DT.%26aulast%3DThomas%26aufirst%3DM.%2BP.%26aulast%3DTraore%26aufirst%3DT.%26aulast%3DVyskocil%26aufirst%3DS.%26aulast%3DWeatherhead%26aufirst%3DG.%2BS.%26aulast%3DYu%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DDick%26aufirst%3DL.%2BR.%26aulast%3DClaiborne%26aufirst%3DC.%2BF.%26aulast%3DRolfe%26aufirst%3DM.%26aulast%3DBolen%26aufirst%3DJ.%2BB.%26aulast%3DLangston%26aufirst%3DS.%2BP.%26atitle%3DAn%2520inhibitor%2520of%2520NEDD8-activating%2520enzyme%2520as%2520a%2520new%2520approach%2520to%2520treat%2520cancer%26jtitle%3DNature%26date%3D2009%26volume%3D458%26spage%3D732%26epage%3D736%26doi%3D10.1038%2Fnature07884" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52'],'ref53':['cit53'],'ref54':['cit54'],'ref55':['cit55'],'ref56':['cit56'],'ref57':['cit57'],'ref58':['cit58'],'ref59':['cit59'],'ref60':['cit60'],'ref61':['cit61'],'ref62':['cit62'],'ref63':['cit63'],'ref64':['cit64'],'ref65':['cit65'],'ref66':['cit66'],'ref67':['cit67'],'ref68':['cit68'],'ref69':['cit69'],'ref70':['cit70'],'ref71':['cit71']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 3 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Weiguo Xiang, <span class="NLM_string-name hlFld-ContribAuthor">Shaomeng Wang</span>. </span><span class="cited-content_cbyCitation_article-title">Selectively Targeting Tropomyosin Receptor Kinase A (TRKA) via PROTACs. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (23)
                                     , 14560-14561. <a href="https://doi.org/10.1021/acs.jmedchem.0c01947" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c01947</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c01947&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c01947%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26doi%3D10.1021%2Facs.jmedchem.0c01342%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DSelectively%252BTargeting%252BTropomyosin%252BReceptor%252BKinase%252BA%252B%252528TRKA%252529%252Bvia%252BPROTACs%26aulast%3DXiang%26aufirst%3DWeiguo%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D10112020%26date%3D18112020%26volume%3D63%26issue%3D23%26spage%3D14560%26epage%3D14561" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Aimo  Kannt</span>, <span class="hlFld-ContribAuthor ">Ivan  Đikić</span>. </span><span class="cited-content_cbyCitation_article-title">Expanding the arsenal of E3 ubiquitin ligases for proximity-induced protein degradation. </span><span class="cited-content_cbyCitation_journal-name">Cell Chemical Biology</span><span> <strong>2021,</strong> <em>28 </em>
                                    (7)
                                     , 1014-1031. <a href="https://doi.org/10.1016/j.chembiol.2021.04.007" title="DOI URL">https://doi.org/10.1016/j.chembiol.2021.04.007</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.chembiol.2021.04.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.chembiol.2021.04.007%26sid%3Dliteratum%253Aachs%26jtitle%3DCell%2520Chemical%2520Biology%26atitle%3DExpanding%252Bthe%252Barsenal%252Bof%252BE3%252Bubiquitin%252Bligases%252Bfor%252Bproximity-induced%252Bprotein%252Bdegradation%26aulast%3DKannt%26aufirst%3DAimo%26date%3D2021%26volume%3D28%26issue%3D7%26spage%3D1014%26epage%3D1031" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Brandon  Dale</span>, <span class="hlFld-ContribAuthor ">Meng  Cheng</span>, <span class="hlFld-ContribAuthor ">Kwang-Su  Park</span>, <span class="hlFld-ContribAuthor ">H. Ümit  Kaniskan</span>, <span class="hlFld-ContribAuthor ">Yue  Xiong</span>, <span class="hlFld-ContribAuthor ">Jian  Jin</span>. </span><span class="cited-content_cbyCitation_article-title">Advancing targeted protein degradation for cancer therapy. </span><span class="cited-content_cbyCitation_journal-name">Nature Reviews Cancer</span><span> <strong>2021,</strong> <em>170 </em><a href="https://doi.org/10.1038/s41568-021-00365-x" title="DOI URL">https://doi.org/10.1038/s41568-021-00365-x</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41568-021-00365-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41568-021-00365-x%26sid%3Dliteratum%253Aachs%26jtitle%3DNature%2520Reviews%2520Cancer%26atitle%3DAdvancing%252Btargeted%252Bprotein%252Bdegradation%252Bfor%252Bcancer%252Btherapy%26aulast%3DDale%26aufirst%3DBrandon%26date%3D2021%26date%3D2021%26volume%3D170" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01342/20201203/images/medium/jm0c01342_0013.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01342/20201203/images/large/jm0c01342_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01342&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01342/20201203/images/medium/jm0c01342_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01342/20201203/images/large/jm0c01342_0002.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Structures of TRK inhibitors. (A) Chemical structures. (B) X-ray cocrystal structure of compound <b>3</b> (cyan) in the complex with TRKA (gray) (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4YNE">4YNE</a>). The loop formed by GLY605 and GLY623 residues is highlighted in light blue.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01342/20201203/images/large/jm0c01342_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01342&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01342/20201203/images/medium/jm0c01342_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01342/20201203/images/large/jm0c01342_0010.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthetic Routes for TRK Degraders <b>5</b> and <b>6</b> and Their Control Analogues <b>25</b> and <b>26</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01342/20201203/images/large/jm0c01342_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01342&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) KF, DMSO, 100 °C, 12 h; (b) Pd(Ph<sub>3</sub>P)<sub>4</sub>, toluene, 110 °C, 12 h; (c) piperazine, KF, DMSO, 130 °C; (d) DIEA, NMP, microwave, 85 °C, 50 min; (e) formic acid, 30 h; (f) EDCI, HOAt, NMM, DMSO, rt, overnight; and (g) MeI, K<sub>2</sub>CO<sub>3</sub>, DMSO, rt, 2 h.</p></p></figure><figure data-id="fig2" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01342/20201203/images/medium/jm0c01342_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01342/20201203/images/large/jm0c01342_0003.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Binding affinities of GNF-8625, degraders (<b>5</b> and <b>6</b>), and negative controls (<b>25</b> and <b>26</b>) to TRKA, TRKB, or TRKC. Binding affinities of individual compounds were determined using the KINOMEscan assay (DiscoveryX). The lowest concentration points represent data from the DMSO samples. Data are shown as mean ± SD derived from duplicated independent experiments.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01342/20201203/images/large/jm0c01342_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01342&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01342/20201203/images/medium/jm0c01342_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01342/20201203/images/large/jm0c01342_0004.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Compounds <b>5</b> and <b>6</b> induce degradation of TRKA fusion and wild-type TRKA. (A) KM12 cells harboring TPM3-TRKA fusion were treated with DMSO or indicated compounds following a threefold serial dilution for 6 h. (B) KM12 cells were treated with 10 nM compound <b>5</b> or <b>6</b> for indicated time points. (C) HEL cells harboring wild-type TRKA were treated with DMSO or indicated compounds following a threefold serial dilution for 6 h. Levels of indicated proteins were determined by immunoblotting and normalized against loading controls.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01342/20201203/images/large/jm0c01342_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01342&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01342/20201203/images/medium/jm0c01342_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01342/20201203/images/large/jm0c01342_0005.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Compounds <b>5</b> and <b>6</b> induce degradation of TRKB and TRKC fusion protein. (A,B) KM12 cells stably expressing AGBL4-TRKB or ETV6-TRKC fusion proteins were treated with DMSO or indicated compounds following a threefold serial dilution for 6 h. Levels of indicated proteins were determined by immunoblotting and normalized against loading controls.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01342/20201203/images/large/jm0c01342_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01342&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01342/20201203/images/medium/jm0c01342_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01342/20201203/images/large/jm0c01342_0006.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Compound <b>5</b> selectively degrades TPM3-TRKA. The volcano plot of proteins with significantly differential abundance for compound <b>5</b>-treated samples <i>vs</i> DMSO-treated control samples. KM12 cells were treated with DMSO or compound <b>5</b> at 100 nM for 6 h and subsequently analyzed using TMT quantitative proteomic analyses. The negative log (base 10) of the <i>P</i>-values is plotted on the <i>y</i>-axis, and the log (base 2) of the fold change is plotted on the <i>x</i>-axis. The green and red data points indicate proteins with a <i>p</i>-value of <0.01 and fold changes of <0.7 and >1.5, respectively. Data were representative of three independent experiments.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01342/20201203/images/large/jm0c01342_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01342&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01342/20201203/images/medium/jm0c01342_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01342/20201203/images/large/jm0c01342_0007.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Degradation of TPM3-TRKA fusion is mediated by E3 ligase cereblon and the UPS. (A, B) KM12 cells were pretreated with DMSO, pomalidomide (POM, 10 μM), MG-132 (20 μM), bortezomib (BTZ, 200 nM), or MLN4924 (5 μM) and subsequently incubated with 10 nM compounds <b>5</b> or <b>6</b> for 6 h prior to immunoblotting. (C) KM12 cells were transfected with two distinct siRNAs targeting <i>CRBN</i> gene and then treated with compound <b>5</b> following a threefold serial dilution for 6 h prior to immunoblotting.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01342/20201203/images/large/jm0c01342_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01342&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01342/20201203/images/medium/jm0c01342_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01342/20201203/images/large/jm0c01342_0008.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Degraders <b>5</b> and <b>6</b> are more potent than GNF-8625, <b>25</b>, and <b>26</b> to inhibit KM12 cell viability. KM12 cells were seeded in 96-well plates and treated with test compounds following 11-point serial dilution for 3 days. Cell viability was determined using the CellTiter-Lumi assay. Error bars represent ± SD in three independent experiments.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01342/20201203/images/large/jm0c01342_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01342&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig8" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01342/20201203/images/medium/jm0c01342_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01342/20201203/images/large/jm0c01342_0009.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Plasma concentrations of compounds <b>5</b> and <b>6</b> following a single 10 mg/kg IP injection in male ICR mice. Plasma concentrations reported at each of the eight time points are the average values from four test animals. Error bars represent + SD.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-23/acs.jmedchem.0c01342/20201203/images/large/jm0c01342_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01342&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i46">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_18521" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_18521" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 71 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pulciani, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Santos, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lauver, A. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Long, L. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aaronson, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barbacid, M.</span></span> <span> </span><span class="NLM_article-title">Oncogenes in solid human tumours</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">1982</span>,  <span class="NLM_volume"><i>300</i></span>,  <span class="NLM_fpage">539</span>– <span class="NLM_lpage">542</span>, <span class="refDoi"> DOI: 10.1038/300539a0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=10.1038%2F300539a0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=7144906" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=1%3ACAS%3A528%3ADyaL3sXlslekurs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=300&publication_year=1982&pages=539-542&author=S.+Pulcianiauthor=E.+Santosauthor=A.+V.+Lauverauthor=L.+K.+Longauthor=S.+A.+Aaronsonauthor=M.+Barbacid&title=Oncogenes+in+solid+human+tumours&doi=10.1038%2F300539a0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Oncogenes in solid human tumors</span></div><div class="casAuthors">Pulciani, Simonetta; Santos, Eugenio; Lauver, Anne V.; Long, Linda K.; Aaronson, Stuart A.; Barbacid, Mariano</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">1982</span>),
    <span class="NLM_cas:volume">300</span>
        (<span class="NLM_cas:issue">5892</span>),
    <span class="NLM_cas:pages">539-42</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    </div><div class="casAbstract">Dominant transforming genes were found in unmanipulated solid human tumors, including carcinomas of the colon (2), lung, and pancreas, and an embryonal rhabdomyosarcoma.  The carcinomas of the lung and pancreas and the embryonal rhabdomyosarcoma possessed a common oncogene which, like that isolated from human LX-1 lung carcinoma cells, shared sequences with the onc gene of the Kirsten strain of murine sarcoma virus (MSV).  The same oncogene was found in several other human tumor cell lines, including those established from carcinomas of the colon (A2233), lung (A427 and A2182), gall bladder (A1604), and urinary bladder.  Hence, a variety of human tumors, regardless of their clin. manifestations, contain a common transforming gene.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp_-WRvbEuIdrVg90H21EOLACvtfcHk0lhRwuhNgbovdQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL3sXlslekurs%253D&md5=9f226d0b79d522381e696f5cdfb84ac5</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2F300539a0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F300539a0%26sid%3Dliteratum%253Aachs%26aulast%3DPulciani%26aufirst%3DS.%26aulast%3DSantos%26aufirst%3DE.%26aulast%3DLauver%26aufirst%3DA.%2BV.%26aulast%3DLong%26aufirst%3DL.%2BK.%26aulast%3DAaronson%26aufirst%3DS.%2BA.%26aulast%3DBarbacid%26aufirst%3DM.%26atitle%3DOncogenes%2520in%2520solid%2520human%2520tumours%26jtitle%3DNature%26date%3D1982%26volume%3D300%26spage%3D539%26epage%3D542%26doi%3D10.1038%2F300539a0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Martin-Zanca, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hughes, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barbacid, M.</span></span> <span> </span><span class="NLM_article-title">A human oncogene formed by the fusion of truncated tropomyosin and protein tyrosine kinase sequences</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">1986</span>,  <span class="NLM_volume"><i>319</i></span>,  <span class="NLM_fpage">743</span>– <span class="NLM_lpage">748</span>, <span class="refDoi"> DOI: 10.1038/319743a0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=10.1038%2F319743a0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=2869410" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=1%3ACAS%3A528%3ADyaL28XhvVSqt7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=319&publication_year=1986&pages=743-748&author=D.+Martin-Zancaauthor=S.+H.+Hughesauthor=M.+Barbacid&title=A+human+oncogene+formed+by+the+fusion+of+truncated+tropomyosin+and+protein+tyrosine+kinase+sequences&doi=10.1038%2F319743a0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">A human oncogene formed by the fusion of truncated tropomyosin and protein tyrosine kinase sequences</span></div><div class="casAuthors">Martin-Zanca, Dionisio; Hughes, Stephen H.; Barbacid, Mariano</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">1986</span>),
    <span class="NLM_cas:volume">319</span>
        (<span class="NLM_cas:issue">6056</span>),
    <span class="NLM_cas:pages">743-8</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    </div><div class="casAbstract">A biol. active cDNA clone of a transforming gene present in a human colon carcinoma contains gene sequences of both tropomyosin and a previously unknown tyrosine kinase  [80449-02-1].  The predicted protein (641 amino acids) encoded by this oncogene seems to have been formed by a somatic rearrangement that replaced the extracellular domain of a putative transmembrane receptor by the 1st 221 amino acids of a non-muscle tropomyosin mol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqA0AjUZGDew7Vg90H21EOLACvtfcHk0lhRVyREGZDeuw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL28XhvVSqt7g%253D&md5=da5002349ed602cac199727be7149d40</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1038%2F319743a0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F319743a0%26sid%3Dliteratum%253Aachs%26aulast%3DMartin-Zanca%26aufirst%3DD.%26aulast%3DHughes%26aufirst%3DS.%2BH.%26aulast%3DBarbacid%26aufirst%3DM.%26atitle%3DA%2520human%2520oncogene%2520formed%2520by%2520the%2520fusion%2520of%2520truncated%2520tropomyosin%2520and%2520protein%2520tyrosine%2520kinase%2520sequences%26jtitle%3DNature%26date%3D1986%26volume%3D319%26spage%3D743%26epage%3D748%26doi%3D10.1038%2F319743a0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Martin-Zanca, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oskam, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitra, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Copeland, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barbacid, M.</span></span> <span> </span><span class="NLM_article-title">Molecular and biochemical characterization of the human trk proto-oncogene</span>. <i>Mol. Cell. Biol.</i> <span class="NLM_year" style="font-weight: bold;">1989</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">24</span>– <span class="NLM_lpage">33</span>, <span class="refDoi"> DOI: 10.1128/mcb.9.1.24</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=10.1128%2FMCB.9.1.24" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=2927393" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=1%3ACAS%3A528%3ADyaL1MXmtV2ru78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=1989&pages=24-33&author=D.+Martin-Zancaauthor=R.+Oskamauthor=G.+Mitraauthor=T.+Copelandauthor=M.+Barbacid&title=Molecular+and+biochemical+characterization+of+the+human+trk+proto-oncogene&doi=10.1128%2Fmcb.9.1.24"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular and biochemical characterization of the human trk proto-oncogene</span></div><div class="casAuthors">Martin-Zanca, Dionisio; Oskam, Ralph; Mitra, Gopa; Copeland, Terry; Barbacid, Mariano</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Biology</span>
        (<span class="NLM_cas:date">1989</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">24-33</span>CODEN:
                <span class="NLM_cas:coden">MCEBD4</span>;
        ISSN:<span class="NLM_cas:issn">0270-7306</span>.
    </div><div class="casAbstract">Mol. anal. of the human trk oncogene, a transforming gene isolated from a colon biopsy, revealed the existence of a novel member of the tyrosine kinase gene family.  This locus codes for a protein of 790 amino acid residues that has several features characteristic of cell surface receptors.  They include (i) a 32-amino-acid-long putative signal peptide, (ii) an amino-terminal moiety (residues 33 to 407) rich in consensus sites for N-glycosylation, (iii) a transmembrane domain, (iv) a kinase catalytic region highly related to that of other tyrosine kinases, and (v) a very short (15 residue) carboxy-terminal tail.  Residues 1 to 392 were absent in the trk oncogene, as they were replaced by tropomyosin sequences.  However, no other differences were found between the transforming and nontransforming trk alleles (residues 392 to 790), suggesting that no addnl. mutations are required to activate the transforming potential of this gene.  The human trk proto-oncogene codes for a 140,000-dalton glycoprotein, designated gp140proto-trk.  However, its primary translational product is a 110,000-dalton glycoprotein which becomes immediately glycosylated, presumably during its translocation into the endoplasmic reticulum.  This mol., designated gp110proto-trk, is further glycosylated to yield the mature form, gp140proto-trk.  Both gp110proto-trk and gp140proto-trk proteins possess in vitro kinase activity specific for tyrosine residues.  Finally, iodination of intact NIH 3T3 cells expressing trk proto-oncogene products indicated that only the mature form, gp140proto-trk, cross the plasma membrane, becoming exposed to the outside of the cell.  These results indicate that the product of the human trk locus is a novel tyrosine kinase cell surface receptor for an as yet unknown ligand.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGru0x_s3mmlXrVg90H21EOLACvtfcHk0lhRVyREGZDeuw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL1MXmtV2ru78%253D&md5=6b98f810db97ba934de4c5e6a848abd9</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1128%2FMCB.9.1.24&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMCB.9.1.24%26sid%3Dliteratum%253Aachs%26aulast%3DMartin-Zanca%26aufirst%3DD.%26aulast%3DOskam%26aufirst%3DR.%26aulast%3DMitra%26aufirst%3DG.%26aulast%3DCopeland%26aufirst%3DT.%26aulast%3DBarbacid%26aufirst%3DM.%26atitle%3DMolecular%2520and%2520biochemical%2520characterization%2520of%2520the%2520human%2520trk%2520proto-oncogene%26jtitle%3DMol.%2520Cell.%2520Biol.%26date%3D1989%26volume%3D9%26spage%3D24%26epage%3D33%26doi%3D10.1128%2Fmcb.9.1.24" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Klein, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parada, L. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coulier, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barbacid, M.</span></span> <span> </span><span class="NLM_article-title">TrkB, a novel tyrosine protein kinase receptor expressed during mouse neural development</span>. <i>EMBO J.</i> <span class="NLM_year" style="font-weight: bold;">1989</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">3701</span>– <span class="NLM_lpage">3709</span>, <span class="refDoi"> DOI: 10.1002/j.1460-2075.1989.tb08545.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=10.1002%2Fj.1460-2075.1989.tb08545.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=2555172" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=1%3ACAS%3A528%3ADyaK3cXktlGlsbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=1989&pages=3701-3709&author=R.+Kleinauthor=L.+F.+Paradaauthor=F.+Coulierauthor=M.+Barbacid&title=TrkB%2C+a+novel+tyrosine+protein+kinase+receptor+expressed+during+mouse+neural+development&doi=10.1002%2Fj.1460-2075.1989.tb08545.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">trkB, a novel tyrosine protein kinase receptor expressed during mouse neural development</span></div><div class="casAuthors">Klein, Rudiger; Parada, Luis F.; Coulier, Francois; Barbacid, Mariano</div><div class="citationInfo"><span class="NLM_cas:title">EMBO Journal</span>
        (<span class="NLM_cas:date">1989</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">3701-9</span>CODEN:
                <span class="NLM_cas:coden">EMJODG</span>;
        ISSN:<span class="NLM_cas:issn">0261-4189</span>.
    </div><div class="casAbstract">The authors isolated a novel member of the tyrosine protein kinase family of cell surface receptors.  This gene, designated trkB, is highly related to the human trk protoncogene.  At the amino acid level, their resp. products share a 57% homol. in their extracellular regions, including 9 of the 11 cysteines present in the trk proto-oncogene.  This homol. increases to 88% within their resp. tyrosine kinase catalytic domains.  Both trk and trkB are equally distantly related to the other members of this gene family of receptors.  A biol. active cDNA clone of trkB can direct the synthesis of gp145trkB, a glycoprotein of 145 kd of which only 93 kd correspond to its polypeptide backbone.  In adult mice, trkB is preferentially expressed in brain tissue, although significant levels of trkB RNA have also been obsd. in lung, muscle, and ovaries.  In addn., trkB transcripts can be detected in mid and late gestation embryos.  The trkB locus exhibits a complex pattern of transcription.  At least seven RNA species ranging in size from ∼9 kb to 2 kb have been identified in brain.  However, only a subset of these transcripts appears to be expressed in the other tissues.  In situ hybridization anal. of 14 and 18 day old mouse embryos indicates that trkB transcripts are localized in the central (CNS) and peripheral (PNS) nervous systems, including brain, spinal cord, spinal and cranial ganglia, paravertebral trunk of the sympathetic nervous system, and various innervation pathways.  These results suggest that trkB may code for a novel cell surface receptor involved in neurogenesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpZMlwtDc9XqLVg90H21EOLACvtfcHk0lhRVyREGZDeuw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3cXktlGlsbg%253D&md5=2fb4ec51e64c20ac0d721ce13a85f592</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1002%2Fj.1460-2075.1989.tb08545.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fj.1460-2075.1989.tb08545.x%26sid%3Dliteratum%253Aachs%26aulast%3DKlein%26aufirst%3DR.%26aulast%3DParada%26aufirst%3DL.%2BF.%26aulast%3DCoulier%26aufirst%3DF.%26aulast%3DBarbacid%26aufirst%3DM.%26atitle%3DTrkB%252C%2520a%2520novel%2520tyrosine%2520protein%2520kinase%2520receptor%2520expressed%2520during%2520mouse%2520neural%2520development%26jtitle%3DEMBO%2520J.%26date%3D1989%26volume%3D8%26spage%3D3701%26epage%3D3709%26doi%3D10.1002%2Fj.1460-2075.1989.tb08545.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lamballe, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klein, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barbacid, M.</span></span> <span> </span><span class="NLM_article-title">TrkC, a new member of the trk family of tyrosine protein kinases, is a receptor for neurotrophin-3</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">1991</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">967</span>– <span class="NLM_lpage">979</span>, <span class="refDoi"> DOI: 10.1016/0092-8674(91)90442-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=10.1016%2F0092-8674%2891%2990442-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=1653651" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=1%3ACAS%3A528%3ADyaK38Xmt12jsro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=1991&pages=967-979&author=F.+Lamballeauthor=R.+Kleinauthor=M.+Barbacid&title=TrkC%2C+a+new+member+of+the+trk+family+of+tyrosine+protein+kinases%2C+is+a+receptor+for+neurotrophin-3&doi=10.1016%2F0092-8674%2891%2990442-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">TrkC, a new member of the trk family of tyrosine protein kinases, is a receptor for neurotrophin-3</span></div><div class="casAuthors">Lamballe, Fabienne; Klein, Rudiger; Barbacid, Mariano</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">1991</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">967-79</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    </div><div class="casAbstract">The isolation and mol. characterization of trkC, a new member of the trk family of tyrosine protein kinase genes is reported.  Gene trkC is preferentially expressed in the brain.  In situ hybridization studies revealed trkC transcripts in the hippocampus, cerebral cortex, and the granular cell layer of the cerebellum.  The product of the trkC gene was identified as a glycoprotein of 145,000 daltons, gp145trkC, which is equally related to the previously characterized gp140trk and gp145trkB tyrosine kinases.  Glycoprotein gp145trkC is a functional receptor for neurotrophin-3 (NT-3).  However, gp145trkC does not bind the highly related neurotrophic factors, NGF or BDNF.  In proliferating cells, the interaction between gp145trkC and NT-3 elicits a more efficient biol. response than when NT-3 binds to its other receptors, gp140trk and gp145trkB.  These results indicate that gp145trkC may play an important role in mediating the neurotrophic effects of NT-3.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrp1C6bvPklNLVg90H21EOLACvtfcHk0liUwMtUCuyePw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK38Xmt12jsro%253D&md5=8aa49b5b94d3f1f337ae097d246e8e9e</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1016%2F0092-8674%2891%2990442-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0092-8674%252891%252990442-2%26sid%3Dliteratum%253Aachs%26aulast%3DLamballe%26aufirst%3DF.%26aulast%3DKlein%26aufirst%3DR.%26aulast%3DBarbacid%26aufirst%3DM.%26atitle%3DTrkC%252C%2520a%2520new%2520member%2520of%2520the%2520trk%2520family%2520of%2520tyrosine%2520protein%2520kinases%252C%2520is%2520a%2520receptor%2520for%2520neurotrophin-3%26jtitle%3DCell%26date%3D1991%26volume%3D66%26spage%3D967%26epage%3D979%26doi%3D10.1016%2F0092-8674%2891%2990442-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Klein, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jing, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nanduri, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Rourke, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barbacid, M.</span></span> <span> </span><span class="NLM_article-title">The trk proto-oncogene encodes a receptor for nerve growth factor</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">1991</span>,  <span class="NLM_volume"><i>65</i></span>,  <span class="NLM_fpage">189</span>– <span class="NLM_lpage">197</span>, <span class="refDoi"> DOI: 10.1016/0092-8674(91)90419-y</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=10.1016%2F0092-8674%2891%2990419-Y" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=1849459" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=1%3ACAS%3A528%3ADyaK3MXisVWgs7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=65&publication_year=1991&pages=189-197&author=R.+Kleinauthor=S.+Jingauthor=V.+Nanduriauthor=E.+O%E2%80%99Rourkeauthor=M.+Barbacid&title=The+trk+proto-oncogene+encodes+a+receptor+for+nerve+growth+factor&doi=10.1016%2F0092-8674%2891%2990419-y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">The trk proto-oncogene encodes a receptor for nerve growth factor</span></div><div class="casAuthors">Klein, Ruediger; Jing, Shuqian; Nanduri, Venkata; O'Rourke, Edward; Barbacid, Mariano</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">1991</span>),
    <span class="NLM_cas:volume">65</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">189-97</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    </div><div class="casAbstract">Two classes of receptors with distinct affinities for nerve growth factor (NGF) have been identified.  The low affinity receptor (Kd ≈ 10-9-10-8M) is a cysteine-rich glycoprotein encoded by the previously characterized LNGFR gene.  The structural nature of the high affinity receptor (Kd ≈ 10-10M) has yet to be established.  The product of the human trk proto-oncogene (gp140trk) bound NGF with high affinity.  Moreover, NGF could be chem. cross-linked to the endogenous gp140trk present in rat PC12 pheochromocytoma cells, as well as to gp140trk ectopically expressed in mouse fibroblasts and in insect Sf9 cells.  High affinity binding of NGF to gp140trk could occur in the absence of low affinity LNGFR receptors, at least in nonneural cells.  Addn. of NGF to PC12 cells elicited rapid phosphorylation of gp140trk on tyrosine residues and stimulated its tyrosine kinase activity.  Apparently, gp14-trk is a functional NGF receptor that mediates at least some of the signal transduction processes initiated by this neurotropic factor.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqlUBY2nUNMtLVg90H21EOLACvtfcHk0liUwMtUCuyePw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3MXisVWgs7Y%253D&md5=362cd01ed7fdf258cfe7f59169b6b5ee</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1016%2F0092-8674%2891%2990419-Y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0092-8674%252891%252990419-Y%26sid%3Dliteratum%253Aachs%26aulast%3DKlein%26aufirst%3DR.%26aulast%3DJing%26aufirst%3DS.%26aulast%3DNanduri%26aufirst%3DV.%26aulast%3DO%25E2%2580%2599Rourke%26aufirst%3DE.%26aulast%3DBarbacid%26aufirst%3DM.%26atitle%3DThe%2520trk%2520proto-oncogene%2520encodes%2520a%2520receptor%2520for%2520nerve%2520growth%2520factor%26jtitle%3DCell%26date%3D1991%26volume%3D65%26spage%3D189%26epage%3D197%26doi%3D10.1016%2F0092-8674%2891%2990419-y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kaplan, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hempstead, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin-Zanca, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chao, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parada, L.</span></span> <span> </span><span class="NLM_article-title">The trk proto-oncogene product: a signal transducing receptor for nerve growth factor</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">1991</span>,  <span class="NLM_volume"><i>252</i></span>,  <span class="NLM_fpage">554</span>– <span class="NLM_lpage">558</span>, <span class="refDoi"> DOI: 10.1126/science.1850549</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=10.1126%2Fscience.1850549" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=1850549" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=1%3ACAS%3A528%3ADyaK3MXisVWgtrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=252&publication_year=1991&pages=554-558&author=D.+Kaplanauthor=B.+Hempsteadauthor=D.+Martin-Zancaauthor=M.+Chaoauthor=L.+Parada&title=The+trk+proto-oncogene+product%3A+a+signal+transducing+receptor+for+nerve+growth+factor&doi=10.1126%2Fscience.1850549"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">The trk proto-oncogene product:  a signal transducing receptor for nerve growth factor</span></div><div class="casAuthors">Kaplan, David R.; Hempstead, Barbara L.; Martin-Zanca, Dionisio; Chao, Moses V.; Parada, Luis F.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">1991</span>),
    <span class="NLM_cas:volume">252</span>
        (<span class="NLM_cas:issue">5005</span>),
    <span class="NLM_cas:pages">554-8</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    </div><div class="casAbstract">The trk proto-oncogene encodes a 140-kilodalton, membrane-spanning protein tyrosine kinase (p140prototrk) that is expressed only in neural tissues.  Nerve growth factor (NGF) stimulates phosphorylation of p140prototrk in neural cell lines and in embσyonic dorsal root ganglia.  Affinity crosslinking and equil. binding expts. with 125I-labeled NGF indicate that p140prototrk binds NGF specifically in cultured cells with a dissocn. const. of 10-9 M.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoOsZ0C6-OsXrVg90H21EOLACvtfcHk0liUwMtUCuyePw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3MXisVWgtrg%253D&md5=5daf16d6938c40bbeb4c44fc58a5df87</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1126%2Fscience.1850549&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1850549%26sid%3Dliteratum%253Aachs%26aulast%3DKaplan%26aufirst%3DD.%26aulast%3DHempstead%26aufirst%3DB.%26aulast%3DMartin-Zanca%26aufirst%3DD.%26aulast%3DChao%26aufirst%3DM.%26aulast%3DParada%26aufirst%3DL.%26atitle%3DThe%2520trk%2520proto-oncogene%2520product%253A%2520a%2520signal%2520transducing%2520receptor%2520for%2520nerve%2520growth%2520factor%26jtitle%3DScience%26date%3D1991%26volume%3D252%26spage%3D554%26epage%3D558%26doi%3D10.1126%2Fscience.1850549" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Soppet, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Escandon, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maragos, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Middlemas, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raid, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blair, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burton, L. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stanton, B. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaplan, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hunter, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nikolics, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parade, L. F.</span></span> <span> </span><span class="NLM_article-title">The neurotrophic factors brain-derived neurotrophic factor and neurotrophin-3 are ligands for the trkB tyrosine kinase receptor</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">1991</span>,  <span class="NLM_volume"><i>65</i></span>,  <span class="NLM_fpage">895</span>– <span class="NLM_lpage">903</span>, <span class="refDoi"> DOI: 10.1016/0092-8674(91)90396-g</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=10.1016%2F0092-8674%2891%2990396-G" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=1645620" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=1%3ACAS%3A528%3ADyaK3MXks1Gnur8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=65&publication_year=1991&pages=895-903&author=D.+Soppetauthor=E.+Escandonauthor=J.+Maragosauthor=D.+S.+Middlemasauthor=S.+W.+Raidauthor=J.+Blairauthor=L.+E.+Burtonauthor=B.+R.+Stantonauthor=D.+R.+Kaplanauthor=T.+Hunterauthor=K.+Nikolicsauthor=L.+F.+Parade&title=The+neurotrophic+factors+brain-derived+neurotrophic+factor+and+neurotrophin-3+are+ligands+for+the+trkB+tyrosine+kinase+receptor&doi=10.1016%2F0092-8674%2891%2990396-g"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">The neurotropic factors brain-derived neurotropic factor and neurotrophin-3 are ligands for the trkB tyrosine kinase receptor</span></div><div class="casAuthors">Soppet, Dan; Escandon, Enrique; Maragos, Johnna; Middlemas, David S.; Reid, Susan W.; Blair, Janet; Burton, Louis E.; Stanton, Brian R.; Kaplan, David R.; et al.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">1991</span>),
    <span class="NLM_cas:volume">65</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">895-903</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    </div><div class="casAbstract">Neurotropic factors are essential for neuronal survival and function.  Recent data have demonstrated that the product of the tyrosine kinase trk proto-oncogene binds (nerve growth factor) NGF and is a component of the high affinity NGF receptor.  Anal. of the trkB gene product, gp145trkB, in NIH 3T3 cells indicates that this tyrosine kinase receptor is rapidly phosphorylated on tyrosine residues upon exposure to the NGF-related neurotropic factors BDNF and NT-3.  Furthermore, gp145trkB specifically binds BDNF and NT-3 in NIH 3T3 cells and in hippocampal cells, but does not bind NGF.  Thus, the trk family of receptors are likely to be important signal transducers of NGF-related trophic signals in the formation and maintenance of neuronal circuits.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGocw0gcgp1GxrVg90H21EOLACvtfcHk0lhcdK67ORS68w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3MXks1Gnur8%253D&md5=c8ac31331087f978d5b36a3d8bb84152</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1016%2F0092-8674%2891%2990396-G&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0092-8674%252891%252990396-G%26sid%3Dliteratum%253Aachs%26aulast%3DSoppet%26aufirst%3DD.%26aulast%3DEscandon%26aufirst%3DE.%26aulast%3DMaragos%26aufirst%3DJ.%26aulast%3DMiddlemas%26aufirst%3DD.%2BS.%26aulast%3DRaid%26aufirst%3DS.%2BW.%26aulast%3DBlair%26aufirst%3DJ.%26aulast%3DBurton%26aufirst%3DL.%2BE.%26aulast%3DStanton%26aufirst%3DB.%2BR.%26aulast%3DKaplan%26aufirst%3DD.%2BR.%26aulast%3DHunter%26aufirst%3DT.%26aulast%3DNikolics%26aufirst%3DK.%26aulast%3DParade%26aufirst%3DL.%2BF.%26atitle%3DThe%2520neurotrophic%2520factors%2520brain-derived%2520neurotrophic%2520factor%2520and%2520neurotrophin-3%2520are%2520ligands%2520for%2520the%2520trkB%2520tyrosine%2520kinase%2520receptor%26jtitle%3DCell%26date%3D1991%26volume%3D65%26spage%3D895%26epage%3D903%26doi%3D10.1016%2F0092-8674%2891%2990396-g" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vaishnavi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Le, A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doebele, R. C.</span></span> <span> </span><span class="NLM_article-title">TRKing down an old oncogene in a new era of targeted therapy</span>. <i>Canc. Discov.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">25</span>– <span class="NLM_lpage">34</span>, <span class="refDoi"> DOI: 10.1158/2159-8290.cd-14-0765</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=10.1158%2F2159-8290.CD-14-0765" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=25527197" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtFCntbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2015&pages=25-34&author=A.+Vaishnaviauthor=A.+T.+Leauthor=R.+C.+Doebele&title=TRKing+down+an+old+oncogene+in+a+new+era+of+targeted+therapy&doi=10.1158%2F2159-8290.cd-14-0765"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">TRKing Down an Old Oncogene in a New Era of Targeted Therapy</span></div><div class="casAuthors">Vaishnavi, Aria; Le, Anh T.; Doebele, Robert C.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Discovery</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">25-34</span>CODEN:
                <span class="NLM_cas:coden">CDAIB2</span>;
        ISSN:<span class="NLM_cas:issn">2159-8274</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">A review.  The use of high-throughput next-generation sequencing techniques in multiple tumor types during the last few years has identified NTRK1, 2, and 3 gene rearrangements encoding novel oncogenic fusions in 19 different tumor types to date.  These recent developments have led us to revisit an old oncogene, Trk (originally identified as OncD), which encodes the TPM3-NTRK1 gene fusion and was one of the first transforming chromosomal rearrangements identified 32 years ago.  However, no drug has yet been approved by the FDA for cancers harboring this oncogene.  This review will discuss the biol. of the TRK family of receptors, their role in human cancer, the types of oncogenic alterations, and drugs that are currently in development for this family of oncogene targets.  Significance: Precision oncol. approaches have accelerated recently due to advancements in our ability to detect oncogenic mutations in tumor samples.  Oncogenic alterations, most commonly gene fusions, have now been detected for the genes encoding the TRKA, TRKB, and TRKC receptor tyrosine kinases across multiple tumor types.  The scientific rationale for the targeting of the TRK oncogene family will be discussed here.  Cancer Discov; 5(1); 25-34. ©2014 AACR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqKUH4ooHY0TbVg90H21EOLACvtfcHk0lhcdK67ORS68w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtFCntbw%253D&md5=02103d46d85b6dc948dba58d32ffba9f</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1158%2F2159-8290.CD-14-0765&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F2159-8290.CD-14-0765%26sid%3Dliteratum%253Aachs%26aulast%3DVaishnavi%26aufirst%3DA.%26aulast%3DLe%26aufirst%3DA.%2BT.%26aulast%3DDoebele%26aufirst%3DR.%2BC.%26atitle%3DTRKing%2520down%2520an%2520old%2520oncogene%2520in%2520a%2520new%2520era%2520of%2520targeted%2520therapy%26jtitle%3DCanc.%2520Discov.%26date%3D2015%26volume%3D5%26spage%3D25%26epage%3D34%26doi%3D10.1158%2F2159-8290.cd-14-0765" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Davies, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Horton, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burton, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmelzer, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vandlen, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosenthal, A.</span></span> <span> </span><span class="NLM_article-title">Neurotrophin-4/5 is a mammalian-specific survival factor for distinct populations of sensory neurons</span>. <i>J. Neurosci.</i> <span class="NLM_year" style="font-weight: bold;">1993</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">4961</span>– <span class="NLM_lpage">4967</span>, <span class="refDoi"> DOI: 10.1523/jneurosci.13-11-04961.1993</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=10.1523%2FJNEUROSCI.13-11-04961.1993" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=8229208" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=1%3ACAS%3A528%3ADyaK2cXks1GltA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=1993&pages=4961-4967&author=A.+Daviesauthor=A.+Hortonauthor=L.+Burtonauthor=C.+Schmelzerauthor=R.+Vandlenauthor=A.+Rosenthal&title=Neurotrophin-4%2F5+is+a+mammalian-specific+survival+factor+for+distinct+populations+of+sensory+neurons&doi=10.1523%2Fjneurosci.13-11-04961.1993"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Neurotrophin-4/5 is a mammalian-specific survival factor for distinct populations of sensory neurons</span></div><div class="casAuthors">Davies, Alun M.; Horton, Antony; Burton, Louis E.; Schmelzer, Charles; Vandlen, Richard; Rosenthal, Arnon</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Neuroscience</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">4961-7</span>CODEN:
                <span class="NLM_cas:coden">JNRSDS</span>;
        ISSN:<span class="NLM_cas:issn">0270-6474</span>.
    </div><div class="casAbstract">The authors have studied the effect of human recombinant neurotrophin-4/5 (NT-4/5) on the survival of developing peripheral nervous system neurons from embryonic mice and chickens.  NT-4/5 transiently supported mouse NGF-dependent trigeminal and jugular neurons at early stages of target field innervation and mouse brain-derived neurotrophic factor (BDNF)-dependent nodose neurons during the phase of naturally occurring cell death.  NT-4/5 was as potent as BDNF in supporting the survival of these neuronal populations.  Surprisingly, NT-4/5 was 3 orders of magnitude less potent than BDNF as a survival factor for early chick dorsomedial trigeminal sensory neurons and did not support the survival of chick BDNF-dependent trigeminal mesencephalic or ventrolateral trigeminal sensory neurons at any of the developmental stages tested.  Thus, NT-4/5 is a survival factor for certain embryonic mouse cranial sensory neurons.  It is the first species-specific neurotrophin to be identified and it can discriminate at high concns. between different BDNF-responsive chick neurons.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGod0Sjz6gWpk7Vg90H21EOLACvtfcHk0lhcdK67ORS68w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2cXks1GltA%253D%253D&md5=296f7b429fa95d629d5b65ed44d57a6b</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1523%2FJNEUROSCI.13-11-04961.1993&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1523%252FJNEUROSCI.13-11-04961.1993%26sid%3Dliteratum%253Aachs%26aulast%3DDavies%26aufirst%3DA.%26aulast%3DHorton%26aufirst%3DA.%26aulast%3DBurton%26aufirst%3DL.%26aulast%3DSchmelzer%26aufirst%3DC.%26aulast%3DVandlen%26aufirst%3DR.%26aulast%3DRosenthal%26aufirst%3DA.%26atitle%3DNeurotrophin-4%252F5%2520is%2520a%2520mammalian-specific%2520survival%2520factor%2520for%2520distinct%2520populations%2520of%2520sensory%2520neurons%26jtitle%3DJ.%2520Neurosci.%26date%3D1993%26volume%3D13%26spage%3D4961%26epage%3D4967%26doi%3D10.1523%2Fjneurosci.13-11-04961.1993" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Segal, R. A.</span></span> <span> </span><span class="NLM_article-title">Selectivity in neurotrophin signaling: theme and variations</span>. <i>Annu. Rev. Neurosci.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">299</span>– <span class="NLM_lpage">330</span>, <span class="refDoi"> DOI: 10.1146/annurev.neuro.26.041002.131421</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=10.1146%2Fannurev.neuro.26.041002.131421" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=12598680" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=1%3ACAS%3A528%3ADC%252BD3sXntFSisLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2003&pages=299-330&author=R.+A.+Segal&title=Selectivity+in+neurotrophin+signaling%3A+theme+and+variations&doi=10.1146%2Fannurev.neuro.26.041002.131421"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Selectivity in neurotrophin signaling: theme and variations</span></div><div class="casAuthors">Segal, Rosalind A.</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Neuroscience</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">299-330</span>CODEN:
                <span class="NLM_cas:coden">ARNSD5</span>;
        ISSN:<span class="NLM_cas:issn">0147-006X</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews Inc.</span>)
        </div><div class="casAbstract">A review.  Neurotrophins are a family of growth factors crit. for the development and functioning of the nervous system.  Although originally identified as neuronal survival factors, neurotrophins elicit many biol. effects, ranging from proliferation to synaptic modulation to axonal pathfinding.  Recent data indicate that the nature of the signaling cascades activated by neurotrophins, and the biol. responses that ensue, are specified not only by the ligand itself but also by the temporal pattern and spatial location of stimulation.  Studies on neurotrophin signaling have revealed variations in the Ras/MAP kinase, phosphatidylinositol 3-kinase, and phospholipase C pathways, which transmit spatial and temporal information.  The anatomy of neurons makes them particularly appropriate for studying how the location and tempo of stimulation det. the signal cascades that are activated by receptor tyrosine kinases such as the Trk receptors.  These signaling variations may represent a general mechanism eliciting specificity in growth factor responses.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpjZGcPFtywOLVg90H21EOLACvtfcHk0lhKhZvQcUN93g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXntFSisLo%253D&md5=f585bf86e97bbcf7467d21533b62cda2</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1146%2Fannurev.neuro.26.041002.131421&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev.neuro.26.041002.131421%26sid%3Dliteratum%253Aachs%26aulast%3DSegal%26aufirst%3DR.%2BA.%26atitle%3DSelectivity%2520in%2520neurotrophin%2520signaling%253A%2520theme%2520and%2520variations%26jtitle%3DAnnu.%2520Rev.%2520Neurosci.%26date%3D2003%26volume%3D26%26spage%3D299%26epage%3D330%26doi%3D10.1146%2Fannurev.neuro.26.041002.131421" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hanahan, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weinberg, R. A.</span></span> <span> </span><span class="NLM_article-title">Hallmarks of cancer: the next generation</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>144</i></span>,  <span class="NLM_fpage">646</span>– <span class="NLM_lpage">674</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2011.02.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=10.1016%2Fj.cell.2011.02.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=21376230" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=1%3ACAS%3A528%3ADC%252BC3MXjsFeqtrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=144&publication_year=2011&pages=646-674&author=D.+Hanahanauthor=R.+A.+Weinberg&title=Hallmarks+of+cancer%3A+the+next+generation&doi=10.1016%2Fj.cell.2011.02.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Hallmarks of cancer: the next generation</span></div><div class="casAuthors">Hanahan, Douglas; Weinberg, Robert A.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">144</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">646-674</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  The hallmarks of cancer comprise six biol. capabilities acquired during the multistep development of human tumors.  The hallmarks constitute an organizing principle for rationalizing the complexities of neoplastic disease.  They include sustaining proliferative signaling, evading growth suppressors, resisting cell death, enabling replicative immortality, inducing angiogenesis, and activating invasion and metastasis.  Underlying these hallmarks are genome instability, which generates the genetic diversity that expedites their acquisition, and inflammation, which fosters multiple hallmark functions.  Conceptual progress in the last decade has added two emerging hallmarks of potential generality to this list-reprogramming of energy metab. and evading immune destruction.  In addn. to cancer cells, tumors exhibit another dimension of complexity: they contain a repertoire of recruited, ostensibly normal cells that contribute to the acquisition of hallmark traits by creating the "tumor microenvironment.".  Recognition of the widespread applicability of these concepts will increasingly affect the development of new means to treat human cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqTcpqXALcb6LVg90H21EOLACvtfcHk0lhKhZvQcUN93g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXjsFeqtrk%253D&md5=b36f160c417a712e83cb83c577f0018e</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2011.02.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2011.02.013%26sid%3Dliteratum%253Aachs%26aulast%3DHanahan%26aufirst%3DD.%26aulast%3DWeinberg%26aufirst%3DR.%2BA.%26atitle%3DHallmarks%2520of%2520cancer%253A%2520the%2520next%2520generation%26jtitle%3DCell%26date%3D2011%26volume%3D144%26spage%3D646%26epage%3D674%26doi%3D10.1016%2Fj.cell.2011.02.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cocco, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scaltriti, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drilon, A.</span></span> <span> </span><span class="NLM_article-title">NTRK fusion-positive cancers and TRK inhibitor therapy</span>. <i>Nat. Rev. Clin. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">731</span>– <span class="NLM_lpage">747</span>, <span class="refDoi"> DOI: 10.1038/s41571-018-0113-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=10.1038%2Fs41571-018-0113-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=30333516" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhvFCrsLnO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2018&pages=731-747&author=E.+Coccoauthor=M.+Scaltritiauthor=A.+Drilon&title=NTRK+fusion-positive+cancers+and+TRK+inhibitor+therapy&doi=10.1038%2Fs41571-018-0113-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">NTRK fusion-positive cancers and TRK inhibitor therapy</span></div><div class="casAuthors">Cocco, Emiliano; Scaltriti, Maurizio; Drilon, Alexander</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Clinical Oncology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">731-747</span>CODEN:
                <span class="NLM_cas:coden">NRCOAA</span>;
        ISSN:<span class="NLM_cas:issn">1759-4774</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">A review.  NTRK gene fusions involving either NTRK1, NTRK2 or NTRK3 (encoding the neurotrophin receptors TRKA, TRKB and TRKC, resp.) are oncogenic drivers of various adult and paediatric tumor types.  These fusions can be detected in the clinic using a variety of methods, including tumor DNA and RNA sequencing and plasma cell-free DNA profiling.  The treatment of patients with NTRK fusion-pos. cancers with a first-generation TRK inhibitor, such as larotrectinib or entrectinib, is assocd. with high response rates (>75%), regardless of tumor histol.  First-generation TRK inhibitors are well tolerated by most patients, with toxicity profiles characterized by occasional off-tumor, on-target adverse events (attributable to TRK inhibition in non-malignant tissues).  Despite durable disease control in many patients, advanced-stage NTRK fusion-pos. cancers eventually become refractory to TRK inhibition; resistance can be mediated by the acquisition of NTRK kinase domain mutations.  Fortunately, certain resistance mutations can be overcome by second-generation TRK inhibitors, including LOXO-195 and TPX-0005 that are being explored in clin. trials.  In this Review, we discuss the biol. of NTRK fusions, strategies to target these drivers in the treatment-naive and acquired-resistance disease settings, and the unique safety profile of TRK inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo2tGShkjX-5LVg90H21EOLACvtfcHk0lhKhZvQcUN93g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhvFCrsLnO&md5=fbc1f5b0de3614008f1b1f1e2e6167ce</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1038%2Fs41571-018-0113-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41571-018-0113-0%26sid%3Dliteratum%253Aachs%26aulast%3DCocco%26aufirst%3DE.%26aulast%3DScaltriti%26aufirst%3DM.%26aulast%3DDrilon%26aufirst%3DA.%26atitle%3DNTRK%2520fusion-positive%2520cancers%2520and%2520TRK%2520inhibitor%2520therapy%26jtitle%3DNat.%2520Rev.%2520Clin.%2520Oncol.%26date%3D2018%26volume%3D15%26spage%3D731%26epage%3D747%26doi%3D10.1038%2Fs41571-018-0113-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ouyang, L.</span></span> <span> </span><span class="NLM_article-title">Development of small-molecule tropomyosin receptor kinase (TRK) inhibitors for NTRK fusion cancers</span>. <i>Acta Pharm. Sin. B</i> <span class="NLM_year" style="font-weight: bold;">2020</span>, <span class="refDoi"> DOI: 10.1016/j.apsb.2020.05.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=10.1016%2Fj.apsb.2020.05.004" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2020&author=T.+Jiangauthor=G.+Wangauthor=Y.+Liuauthor=L.+Fengauthor=M.+Wangauthor=J.+Liuauthor=Y.+Chenauthor=L.+Ouyang&title=Development+of+small-molecule+tropomyosin+receptor+kinase+%28TRK%29+inhibitors+for+NTRK+fusion+cancers&doi=10.1016%2Fj.apsb.2020.05.004"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1016%2Fj.apsb.2020.05.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.apsb.2020.05.004%26sid%3Dliteratum%253Aachs%26aulast%3DJiang%26aufirst%3DT.%26aulast%3DWang%26aufirst%3DG.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DFeng%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DOuyang%26aufirst%3DL.%26atitle%3DDevelopment%2520of%2520small-molecule%2520tropomyosin%2520receptor%2520kinase%2520%2528TRK%2529%2520inhibitors%2520for%2520NTRK%2520fusion%2520cancers%26jtitle%3DActa%2520Pharm.%2520Sin.%2520B%26date%3D2020%26doi%3D10.1016%2Fj.apsb.2020.05.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yan, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lakkaniga, N. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carlomagno, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Santoro, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McDonald, N. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lv, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gunaganti, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frett, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, H.-y.</span></span> <span> </span><span class="NLM_article-title">Insights into current tropomyosin receptor kinase (TRK) inhibitors: development and clinical application</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">1731</span>– <span class="NLM_lpage">1760</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b01092</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b01092" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhs1Khur%252FO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=1731-1760&author=W.+Yanauthor=N.+R.+Lakkanigaauthor=F.+Carlomagnoauthor=M.+Santoroauthor=N.+Q.+McDonaldauthor=F.+Lvauthor=N.+Gunagantiauthor=B.+Frettauthor=H.-y.+Li&title=Insights+into+current+tropomyosin+receptor+kinase+%28TRK%29+inhibitors%3A+development+and+clinical+application&doi=10.1021%2Facs.jmedchem.8b01092"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Insights into Current Tropomyosin Receptor Kinase (TRK) Inhibitors: Development and Clinical Application</span></div><div class="casAuthors">Yan, Wei; Lakkaniga, Naga Rajiv; Carlomagno, Francesca; Santoro, Massimo; McDonald, Neil Q.; Lv, Fengping; Gunaganti, Naresh; Frett, Brendan; Li, Hong-yu</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1731-1760</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The use of kinase-directed precision medicine has been heavily pursued since the discovery and development of imatinib.  Annually, it is estd. that around ∼20 000 new cases of tropomyosin receptor kinase (TRK) cancers are diagnosed, with the majority of cases exhibiting a TRK genomic rearrangement.  In this Perspective, we discuss current development and clin. applications for TRK precision medicine by providing the following: (1) the biol. background and significance of the TRK kinase family, (2) a compilation of known TRK inhibitors and anal. of their cocrystal structures, (3) an overview of TRK clin. trials, and (4) future perspectives for drug discovery and development of TRK inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrwUthQyJEzFrVg90H21EOLACvtfcHk0lhtYogPqp3jSQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhs1Khur%252FO&md5=5ce2f8d295cf46f84b46f5890dad0160</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b01092&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b01092%26sid%3Dliteratum%253Aachs%26aulast%3DYan%26aufirst%3DW.%26aulast%3DLakkaniga%26aufirst%3DN.%2BR.%26aulast%3DCarlomagno%26aufirst%3DF.%26aulast%3DSantoro%26aufirst%3DM.%26aulast%3DMcDonald%26aufirst%3DN.%2BQ.%26aulast%3DLv%26aufirst%3DF.%26aulast%3DGunaganti%26aufirst%3DN.%26aulast%3DFrett%26aufirst%3DB.%26aulast%3DLi%26aufirst%3DH.-y.%26atitle%3DInsights%2520into%2520current%2520tropomyosin%2520receptor%2520kinase%2520%2528TRK%2529%2520inhibitors%253A%2520development%2520and%2520clinical%2520application%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D1731%26epage%3D1760%26doi%3D10.1021%2Facs.jmedchem.8b01092" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Amatu, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sartore-Bianchi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siena, S.</span></span> <span> </span><span class="NLM_article-title">NTRKgene fusions as novel targets of cancer therapy across multiple tumour types</span>. <i>ESMO Open</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>1</i></span>, <span class="NLM_elocation-id">e000023</span> <span class="refDoi"> DOI: 10.1136/esmoopen-2015-000023</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=10.1136%2Fesmoopen-2015-000023" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=27843590" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2016&author=A.+Amatuauthor=A.+Sartore-Bianchiauthor=S.+Siena&title=NTRKgene+fusions+as+novel+targets+of+cancer+therapy+across+multiple+tumour+types&doi=10.1136%2Fesmoopen-2015-000023"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1136%2Fesmoopen-2015-000023&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1136%252Fesmoopen-2015-000023%26sid%3Dliteratum%253Aachs%26aulast%3DAmatu%26aufirst%3DA.%26aulast%3DSartore-Bianchi%26aufirst%3DA.%26aulast%3DSiena%26aufirst%3DS.%26atitle%3DNTRKgene%2520fusions%2520as%2520novel%2520targets%2520of%2520cancer%2520therapy%2520across%2520multiple%2520tumour%2520types%26jtitle%3DESMO%2520Open%26date%3D2016%26volume%3D1%26doi%3D10.1136%2Fesmoopen-2015-000023" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Khotskaya, Y. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holla, V. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farago, A. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mills Shaw, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meric-Bernstam, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hong, D. S.</span></span> <span> </span><span class="NLM_article-title">Targeting TRK family proteins in cancer</span>. <i>Pharmacol. Ther.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>173</i></span>,  <span class="NLM_fpage">58</span>– <span class="NLM_lpage">66</span>, <span class="refDoi"> DOI: 10.1016/j.pharmthera.2017.02.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=10.1016%2Fj.pharmthera.2017.02.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=28174090" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=1%3ACAS%3A528%3ADC%252BC2sXisF2jtL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=173&publication_year=2017&pages=58-66&author=Y.+B.+Khotskayaauthor=V.+R.+Hollaauthor=A.+F.+Faragoauthor=K.+R.+Mills+Shawauthor=F.+Meric-Bernstamauthor=D.+S.+Hong&title=Targeting+TRK+family+proteins+in+cancer&doi=10.1016%2Fj.pharmthera.2017.02.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting TRK family proteins in cancer</span></div><div class="casAuthors">Khotskaya, Yekaterina B.; Holla, Vijaykumar R.; Farago, Anna F.; Mills Shaw, Kenna R.; Meric-Bernstam, Funda; Hong, David S.</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacology & Therapeutics</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">173</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">58-66</span>CODEN:
                <span class="NLM_cas:coden">PHTHDT</span>;
        ISSN:<span class="NLM_cas:issn">0163-7258</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">The tropomyosin receptor kinase (TRK) family includes TRKA, TRKB, and TRKC proteins, which are encoded by NTRK1, NTRK2 and NTRK3 genes, resp.  Binding of neurotrophins to TRK proteins induces receptor dimerization, phosphorylation, and activation of the downstream signaling cascades via PI3K, RAS/MAPK/ERK, and PLC-gamma.  TRK pathway aberrations, including gene fusions, protein overexpression, and single nucleotide alterations, have been implicated in the pathogenesis of many cancer types, with NTRK gene fusions being the most well validated oncogenic events to date.  Although the NTRK gene fusions are infrequent in most cancer types, certain rare tumor types are predominately driven by these events.  Conversely, in more common histologies, such as lung and colorectal cancers, prevalence of the NTRK fusions is well below 5%.  Selective inhibition of TRK signaling may therefore be beneficial among patients whose tumors vary in histologies, but share underlying oncogenic NTRK gene alterations.  Currently, several TRK-targeting compds. are in clin. development.  The ongoing Phase 2 trials with entrectinib and LOXO-101, two of the leading TRK inhibitors, are designed as 'basket trials', inclusive of patients whose tumors harbor NTRK gene fusions, independent of histol.  Addnl. Phase 1 studies of other TRK inhibitors, including MGCD516, PLX7486, DS-6051b, and TSR-011, are underway.  Interim data examg. NTRK-rearranged tumors treated with entrectinib or LOXO-101 demonstrate encouraging activity, with patients achieving rapid and durable responses.  Consequently, both drugs have achieved orphan designation from regulatory agencies, and efforts are underway to further expedite their development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGozoy77YmUc_LVg90H21EOLACvtfcHk0lhtYogPqp3jSQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXisF2jtL8%253D&md5=f18f232e6d3c4c9a5d2e398fe60ae7e4</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1016%2Fj.pharmthera.2017.02.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.pharmthera.2017.02.006%26sid%3Dliteratum%253Aachs%26aulast%3DKhotskaya%26aufirst%3DY.%2BB.%26aulast%3DHolla%26aufirst%3DV.%2BR.%26aulast%3DFarago%26aufirst%3DA.%2BF.%26aulast%3DMills%2BShaw%26aufirst%3DK.%2BR.%26aulast%3DMeric-Bernstam%26aufirst%3DF.%26aulast%3DHong%26aufirst%3DD.%2BS.%26atitle%3DTargeting%2520TRK%2520family%2520proteins%2520in%2520cancer%26jtitle%3DPharmacol.%2520Ther.%26date%3D2017%26volume%3D173%26spage%3D58%26epage%3D66%26doi%3D10.1016%2Fj.pharmthera.2017.02.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Greco, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miranda, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pierotti, M. A.</span></span> <span> </span><span class="NLM_article-title">Rearrangements of NTRK1 gene in papillary thyroid carcinoma</span>. <i>Mol. Cell. Endocrinol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>321</i></span>,  <span class="NLM_fpage">44</span>– <span class="NLM_lpage">49</span>, <span class="refDoi"> DOI: 10.1016/j.mce.2009.10.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=10.1016%2Fj.mce.2009.10.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=19883730" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=1%3ACAS%3A528%3ADC%252BC3cXksVamu7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=321&publication_year=2010&pages=44-49&author=A.+Grecoauthor=C.+Mirandaauthor=M.+A.+Pierotti&title=Rearrangements+of+NTRK1+gene+in+papillary+thyroid+carcinoma&doi=10.1016%2Fj.mce.2009.10.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Rearrangements of NTRK1 gene in papillary thyroid carcinoma</span></div><div class="casAuthors">Greco, A.; Miranda, C.; Pierotti, M. A.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Endocrinology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">321</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">44-49</span>CODEN:
                <span class="NLM_cas:coden">MCEND6</span>;
        ISSN:<span class="NLM_cas:issn">0303-7207</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ireland Ltd.</span>)
        </div><div class="casAbstract">A review.  TRK oncogenes are obsd. in a consistent fraction of papillary thyroid carcinoma (PTC); they arise from the fusion of the 3' terminal sequences of the NTRK1/NGF receptor gene with 5' terminal sequences of various activating genes, such as TPM3, TPR and TFG.  TRK oncoproteins display constitutive tyrosine-kinase activity, leading to in vitro and in vivo transformation.  In this review studies performed during the last 20 years will be summarized.  The following topics will be illustrated: (a) frequency of TRK oncogenes and correlation with radiation and tumor histopathol. features; (b) mol. mechanisms underlying NTRK1 oncogenic rearrangements; (c) mol. and biochem. characterization of TRK oncoproteins, and their mechanism of action; (d) role of activating sequences in the activation of TRK oncoproteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrnGkkW9wYpTLVg90H21EOLACvtfcHk0lhtYogPqp3jSQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXksVamu7g%253D&md5=ea9abddd4191599672958ac3875bf9ae</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1016%2Fj.mce.2009.10.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.mce.2009.10.009%26sid%3Dliteratum%253Aachs%26aulast%3DGreco%26aufirst%3DA.%26aulast%3DMiranda%26aufirst%3DC.%26aulast%3DPierotti%26aufirst%3DM.%2BA.%26atitle%3DRearrangements%2520of%2520NTRK1%2520gene%2520in%2520papillary%2520thyroid%2520carcinoma%26jtitle%3DMol.%2520Cell.%2520Endocrinol.%26date%3D2010%26volume%3D321%26spage%3D44%26epage%3D49%26doi%3D10.1016%2Fj.mce.2009.10.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Greco, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miranda, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pierotti, M. A.</span></span> <span> </span><span class="NLM_article-title">Rearrangements of NTRK1 gene in papillary thyroid carcinoma</span>. <i>Mol. Cell. Endocrinol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>321</i></span>,  <span class="NLM_fpage">44</span>– <span class="NLM_lpage">49</span>, <span class="refDoi"> DOI: 10.1016/j.mce.2009.10.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=10.1016%2Fj.mce.2009.10.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=19883730" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=1%3ACAS%3A528%3ADC%252BC3cXksVamu7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=321&publication_year=2010&pages=44-49&author=A.+Grecoauthor=C.+Mirandaauthor=M.+A.+Pierotti&title=Rearrangements+of+NTRK1+gene+in+papillary+thyroid+carcinoma&doi=10.1016%2Fj.mce.2009.10.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Rearrangements of NTRK1 gene in papillary thyroid carcinoma</span></div><div class="casAuthors">Greco, A.; Miranda, C.; Pierotti, M. A.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Endocrinology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">321</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">44-49</span>CODEN:
                <span class="NLM_cas:coden">MCEND6</span>;
        ISSN:<span class="NLM_cas:issn">0303-7207</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ireland Ltd.</span>)
        </div><div class="casAbstract">A review.  TRK oncogenes are obsd. in a consistent fraction of papillary thyroid carcinoma (PTC); they arise from the fusion of the 3' terminal sequences of the NTRK1/NGF receptor gene with 5' terminal sequences of various activating genes, such as TPM3, TPR and TFG.  TRK oncoproteins display constitutive tyrosine-kinase activity, leading to in vitro and in vivo transformation.  In this review studies performed during the last 20 years will be summarized.  The following topics will be illustrated: (a) frequency of TRK oncogenes and correlation with radiation and tumor histopathol. features; (b) mol. mechanisms underlying NTRK1 oncogenic rearrangements; (c) mol. and biochem. characterization of TRK oncoproteins, and their mechanism of action; (d) role of activating sequences in the activation of TRK oncoproteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrnGkkW9wYpTLVg90H21EOLACvtfcHk0lhtYogPqp3jSQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXksVamu7g%253D&md5=ea9abddd4191599672958ac3875bf9ae</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1016%2Fj.mce.2009.10.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.mce.2009.10.009%26sid%3Dliteratum%253Aachs%26aulast%3DGreco%26aufirst%3DA.%26aulast%3DMiranda%26aufirst%3DC.%26aulast%3DPierotti%26aufirst%3DM.%2BA.%26atitle%3DRearrangements%2520of%2520NTRK1%2520gene%2520in%2520papillary%2520thyroid%2520carcinoma%26jtitle%3DMol.%2520Cell.%2520Endocrinol.%26date%3D2010%26volume%3D321%26spage%3D44%26epage%3D49%26doi%3D10.1016%2Fj.mce.2009.10.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liebers, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhelyazkova, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Panditi, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lynch, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robinson, H. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shim, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chmielecki, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pao, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Engelman, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iafrate, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Le, L. P.</span></span> <span> </span><span class="NLM_article-title">Anchored multiplex PCR for targeted next-generation sequencing</span>. <i>Nat. Med.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">1479</span>– <span class="NLM_lpage">1484</span>, <span class="refDoi"> DOI: 10.1038/nm.3729</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=10.1038%2Fnm.3729" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=25384085" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvF2js7jF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2014&pages=1479-1484&author=Z.+Zhengauthor=M.+Liebersauthor=B.+Zhelyazkovaauthor=Y.+Caoauthor=D.+Panditiauthor=K.+D.+Lynchauthor=J.+Chenauthor=H.+E.+Robinsonauthor=H.+S.+Shimauthor=J.+Chmieleckiauthor=W.+Paoauthor=J.+A.+Engelmanauthor=A.+J.+Iafrateauthor=L.+P.+Le&title=Anchored+multiplex+PCR+for+targeted+next-generation+sequencing&doi=10.1038%2Fnm.3729"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Anchored multiplex PCR for targeted next-generation sequencing</span></div><div class="casAuthors">Zheng, Zongli; Liebers, Matthew; Zhelyazkova, Boryana; Cao, Yi; Panditi, Divya; Lynch, Kerry D.; Chen, Juxiang; Robinson, Hayley E.; Shim, Hyo Sup; Chmielecki, Juliann; Pao, William; Engelman, Jeffrey A.; Iafrate, A. John; Le, Long Phi</div><div class="citationInfo"><span class="NLM_cas:title">Nature Medicine (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1479-1484</span>CODEN:
                <span class="NLM_cas:coden">NAMEFI</span>;
        ISSN:<span class="NLM_cas:issn">1078-8956</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">We describe a rapid target enrichment method for next-generation sequencing, termed anchored multiplex PCR (AMP), that is compatible with low nucleic acid input from formalin-fixed paraffin-embedded (FFPE) specimens.  AMP is effective in detecting gene rearrangements (without prior knowledge of the fusion partners), single nucleotide variants, insertions, deletions and copy no. changes.  Validation of a gene rearrangement panel using 319 FFPE samples showed 100% sensitivity (95% confidence limit: 96.5-100%) and 100% specificity (95% confidence limit: 99.3-100%) compared with ref. assays.  On the basis of our experience with performing AMP on 986 clin. FFPE samples, we show its potential as both a robust clin. assay and a powerful discovery tool, which we used to identify new therapeutically important gene fusions: ARHGEF2-NTRK1 and CHTOP-NTRK1 in glioblastoma, MSN-ROS1, TRIM4-BRAF, VAMP2-NRG1, TPM3-NTRK1 and RUFY2-RET in lung cancer, FGFR2-CREB5 in cholangiocarcinoma and PPL-NTRK1 in thyroid carcinoma.  AMP is a scalable and efficient next-generation sequencing target enrichment method for research and clin. applications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoBvfrWVSNX4rVg90H21EOLACvtfcHk0ljtfh9nXlLiuQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvF2js7jF&md5=77c01e9735858185681607b148ad8192</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1038%2Fnm.3729&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnm.3729%26sid%3Dliteratum%253Aachs%26aulast%3DZheng%26aufirst%3DZ.%26aulast%3DLiebers%26aufirst%3DM.%26aulast%3DZhelyazkova%26aufirst%3DB.%26aulast%3DCao%26aufirst%3DY.%26aulast%3DPanditi%26aufirst%3DD.%26aulast%3DLynch%26aufirst%3DK.%2BD.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DRobinson%26aufirst%3DH.%2BE.%26aulast%3DShim%26aufirst%3DH.%2BS.%26aulast%3DChmielecki%26aufirst%3DJ.%26aulast%3DPao%26aufirst%3DW.%26aulast%3DEngelman%26aufirst%3DJ.%2BA.%26aulast%3DIafrate%26aufirst%3DA.%2BJ.%26aulast%3DLe%26aufirst%3DL.%2BP.%26atitle%3DAnchored%2520multiplex%2520PCR%2520for%2520targeted%2520next-generation%2520sequencing%26jtitle%3DNat.%2520Med.%26date%3D2014%26volume%3D20%26spage%3D1479%26epage%3D1484%26doi%3D10.1038%2Fnm.3729" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vaishnavi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Capelletti, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Le, A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kako, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Butaney, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ercan, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mahale, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davies, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aisner, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pilling, A. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berge, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sasaki, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, S.-i.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kryukov, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garraway, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hammerman, P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haas, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andrews, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lipson, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stephens, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, V. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Varella-Garcia, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jänne, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doebele, R. C.</span></span> <span> </span><span class="NLM_article-title">Oncogenic and drug-sensitive NTRK1 rearrangements in lung cancer</span>. <i>Nat. Med.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">1469</span>– <span class="NLM_lpage">1472</span>, <span class="refDoi"> DOI: 10.1038/nm.3352</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=10.1038%2Fnm.3352" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=24162815" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhs1yiurrI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2013&pages=1469-1472&author=A.+Vaishnaviauthor=M.+Capellettiauthor=A.+T.+Leauthor=S.+Kakoauthor=M.+Butaneyauthor=D.+Ercanauthor=S.+Mahaleauthor=K.+D.+Daviesauthor=D.+L.+Aisnerauthor=A.+B.+Pillingauthor=E.+M.+Bergeauthor=J.+Kimauthor=H.+Sasakiauthor=S.-i.+Parkauthor=G.+Kryukovauthor=L.+A.+Garrawayauthor=P.+S.+Hammermanauthor=J.+Haasauthor=S.+W.+Andrewsauthor=D.+Lipsonauthor=P.+J.+Stephensauthor=V.+A.+Millerauthor=M.+Varella-Garciaauthor=P.+A.+J%C3%A4nneauthor=R.+C.+Doebele&title=Oncogenic+and+drug-sensitive+NTRK1+rearrangements+in+lung+cancer&doi=10.1038%2Fnm.3352"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Oncogenic and drug-sensitive NTRK1 rearrangements in lung cancer</span></div><div class="casAuthors">Vaishnavi, Aria; Capelletti, Marzia; Le, Anh T.; Kako, Severine; Butaney, Mohit; Ercan, Dalia; Mahale, Sakshi; Davies, Kurtis D.; Aisner, Dara L.; Pilling, Amanda B.; Berge, Eamon M.; Kim, Jhingook; Sasaki, Hidefumi; Park, Seung-il; Kryukov, Gregory; Garraway, Levi A.; Hammerman, Peter S.; Haas, Julia; Andrews, Steven W.; Lipson, Doron; Stephens, Philip J.; Miller, Vince A.; Varella-Garcia, Marileila; Jaenne, Pasi A.; Doebele, Robert C.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Medicine (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1469-1472</span>CODEN:
                <span class="NLM_cas:coden">NAMEFI</span>;
        ISSN:<span class="NLM_cas:issn">1078-8956</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">We identified new gene fusions in patients with lung cancer harboring the kinase domain of the NTRK1 gene that encodes the high-affinity nerve growth factor receptor (TRKA protein).  Both the MPRIP-NTRK1 and CD74-NTRK1 fusions lead to constitutive TRKA kinase activity and are oncogenic.  Treatment of cells expressing NTRK1 fusions with inhibitors of TRKA kinase activity inhibited autophosphorylation of TRKA and cell growth.  Tumor samples from 3 of 91 patients with lung cancer (3.3%) without known oncogenic alterations assayed by next-generation sequencing or fluorescence in situ hybridization demonstrated evidence of NTRK1 gene fusions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpBHNoZGMQLMrVg90H21EOLACvtfcHk0ljtfh9nXlLiuQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhs1yiurrI&md5=23b413826ea700bd831eadbfde270615</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1038%2Fnm.3352&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnm.3352%26sid%3Dliteratum%253Aachs%26aulast%3DVaishnavi%26aufirst%3DA.%26aulast%3DCapelletti%26aufirst%3DM.%26aulast%3DLe%26aufirst%3DA.%2BT.%26aulast%3DKako%26aufirst%3DS.%26aulast%3DButaney%26aufirst%3DM.%26aulast%3DErcan%26aufirst%3DD.%26aulast%3DMahale%26aufirst%3DS.%26aulast%3DDavies%26aufirst%3DK.%2BD.%26aulast%3DAisner%26aufirst%3DD.%2BL.%26aulast%3DPilling%26aufirst%3DA.%2BB.%26aulast%3DBerge%26aufirst%3DE.%2BM.%26aulast%3DKim%26aufirst%3DJ.%26aulast%3DSasaki%26aufirst%3DH.%26aulast%3DPark%26aufirst%3DS.-i.%26aulast%3DKryukov%26aufirst%3DG.%26aulast%3DGarraway%26aufirst%3DL.%2BA.%26aulast%3DHammerman%26aufirst%3DP.%2BS.%26aulast%3DHaas%26aufirst%3DJ.%26aulast%3DAndrews%26aufirst%3DS.%2BW.%26aulast%3DLipson%26aufirst%3DD.%26aulast%3DStephens%26aufirst%3DP.%2BJ.%26aulast%3DMiller%26aufirst%3DV.%2BA.%26aulast%3DVarella-Garcia%26aufirst%3DM.%26aulast%3DJ%25C3%25A4nne%26aufirst%3DP.%2BA.%26aulast%3DDoebele%26aufirst%3DR.%2BC.%26atitle%3DOncogenic%2520and%2520drug-sensitive%2520NTRK1%2520rearrangements%2520in%2520lung%2520cancer%26jtitle%3DNat.%2520Med.%26date%3D2013%26volume%3D19%26spage%3D1469%26epage%3D1472%26doi%3D10.1038%2Fnm.3352" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sartore-Bianchi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ardini, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bosotti, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amatu, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valtorta, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Somaschini, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raddrizzani, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Palmeri, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Banfi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bonazzina, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Misale, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marrapese, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leone, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alzani, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hornby, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lim, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Veronese, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vanzulli, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bardelli, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martignoni, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davite, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galvani, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Isacchi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siena, S.</span></span> <span> </span><span class="NLM_article-title">Sensitivity to entrectinib associated with a novel LMNA-NTRK1 gene fusion in metastatic colorectal cancer</span>. <i>J. Natl. Cancer Inst.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>108</i></span>,  <span class="NLM_fpage">djv306</span>, <span class="refDoi"> DOI: 10.1093/jnci/djv306</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=10.1093%2Fjnci%2Fdjv306" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=26563355" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=108&publication_year=2016&pages=djv306&author=A.+Sartore-Bianchiauthor=E.+Ardiniauthor=R.+Bosottiauthor=A.+Amatuauthor=E.+Valtortaauthor=A.+Somaschiniauthor=L.+Raddrizzaniauthor=L.+Palmeriauthor=P.+Banfiauthor=E.+Bonazzinaauthor=S.+Misaleauthor=G.+Marrapeseauthor=A.+Leoneauthor=R.+Alzaniauthor=D.+Luoauthor=Z.+Hornbyauthor=J.+Limauthor=S.+Veroneseauthor=A.+Vanzulliauthor=A.+Bardelliauthor=M.+Martignoniauthor=C.+Daviteauthor=A.+Galvaniauthor=A.+Isacchiauthor=S.+Siena&title=Sensitivity+to+entrectinib+associated+with+a+novel+LMNA-NTRK1+gene+fusion+in+metastatic+colorectal+cancer&doi=10.1093%2Fjnci%2Fdjv306"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1093%2Fjnci%2Fdjv306&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fjnci%252Fdjv306%26sid%3Dliteratum%253Aachs%26aulast%3DSartore-Bianchi%26aufirst%3DA.%26aulast%3DArdini%26aufirst%3DE.%26aulast%3DBosotti%26aufirst%3DR.%26aulast%3DAmatu%26aufirst%3DA.%26aulast%3DValtorta%26aufirst%3DE.%26aulast%3DSomaschini%26aufirst%3DA.%26aulast%3DRaddrizzani%26aufirst%3DL.%26aulast%3DPalmeri%26aufirst%3DL.%26aulast%3DBanfi%26aufirst%3DP.%26aulast%3DBonazzina%26aufirst%3DE.%26aulast%3DMisale%26aufirst%3DS.%26aulast%3DMarrapese%26aufirst%3DG.%26aulast%3DLeone%26aufirst%3DA.%26aulast%3DAlzani%26aufirst%3DR.%26aulast%3DLuo%26aufirst%3DD.%26aulast%3DHornby%26aufirst%3DZ.%26aulast%3DLim%26aufirst%3DJ.%26aulast%3DVeronese%26aufirst%3DS.%26aulast%3DVanzulli%26aufirst%3DA.%26aulast%3DBardelli%26aufirst%3DA.%26aulast%3DMartignoni%26aufirst%3DM.%26aulast%3DDavite%26aufirst%3DC.%26aulast%3DGalvani%26aufirst%3DA.%26aulast%3DIsacchi%26aufirst%3DA.%26aulast%3DSiena%26aufirst%3DS.%26atitle%3DSensitivity%2520to%2520entrectinib%2520associated%2520with%2520a%2520novel%2520LMNA-NTRK1%2520gene%2520fusion%2520in%2520metastatic%2520colorectal%2520cancer%26jtitle%3DJ.%2520Natl.%2520Cancer%2520Inst.%26date%3D2016%26volume%3D108%26spage%3Ddjv306%26doi%3D10.1093%2Fjnci%2Fdjv306" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Doebele, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, L. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vaishnavi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Le, A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Estrada-Bernal, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keysar, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jimeno, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Varella-Garcia, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aisner, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stephens, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morosini, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tuch, B. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fernandes, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nanda, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Low, J. A.</span></span> <span> </span><span class="NLM_article-title">An oncogenic NTRK fusion in a patient with soft-tissue sarcoma with response to the tropomyosin-related kinase inhibitor LOXO-101</span>. <i>Canc. Discov.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">1049</span>– <span class="NLM_lpage">1057</span>, <span class="refDoi"> DOI: 10.1158/2159-8290.cd-15-0443</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=10.1158%2F2159-8290.CD-15-0443" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=26216294" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhs1Wqs7fF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2015&pages=1049-1057&author=R.+C.+Doebeleauthor=L.+E.+Davisauthor=A.+Vaishnaviauthor=A.+T.+Leauthor=A.+Estrada-Bernalauthor=S.+Keysarauthor=A.+Jimenoauthor=M.+Varella-Garciaauthor=D.+L.+Aisnerauthor=Y.+Liauthor=P.+J.+Stephensauthor=D.+Morosiniauthor=B.+B.+Tuchauthor=M.+Fernandesauthor=N.+Nandaauthor=J.+A.+Low&title=An+oncogenic+NTRK+fusion+in+a+patient+with+soft-tissue+sarcoma+with+response+to+the+tropomyosin-related+kinase+inhibitor+LOXO-101&doi=10.1158%2F2159-8290.cd-15-0443"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">An Oncogenic NTRK Fusion in a Patient with Soft-Tissue Sarcoma with Response to the Tropomyosin-Related Kinase Inhibitor LOXO-101</span></div><div class="casAuthors">Doebele, Robert C.; Davis, Lara E.; Vaishnavi, Aria; Le, Anh T.; Estrada-Bernal, Adriana; Keysar, Stephen; Jimeno, Antonio; Varella-Garcia, Marileila; Aisner, Dara L.; Li, Yali; Stephens, Philip J.; Morosini, Deborah; Tuch, Brian B.; Fernandes, Michele; Nanda, Nisha; Low, Jennifer A.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Discovery</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1049-1057</span>CODEN:
                <span class="NLM_cas:coden">CDAIB2</span>;
        ISSN:<span class="NLM_cas:issn">2159-8274</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Oncogenic TRK fusions induce cancer cell proliferation and engage crit. cancer-related downstream signaling pathways.  These TRK fusions occur rarely, but in a diverse spectrum of tumor histologies.  LOXO-101 is an orally administered inhibitor of the TRK kinase and is highly selective only for the TRK family of receptors.  Preclin. models of LOXO-101 using TRK-fusion-bearing human-derived cancer cell lines demonstrate inhibition of the fusion oncoprotein and cellular proliferation in vitro, and tumor growth in vivo.  The tumor of a 41-yr-old woman with soft-tissue sarcoma metastatic to the lung was found to harbor an LMNA-NTRK1 gene fusion encoding a functional LMNA-TRKA fusion oncoprotein as detd. by an in situ proximity ligation assay.  In a phase I study of LOXO-101 (ClinicalTrials.gov no. NCT02122913), this patient's tumors underwent rapid and substantial tumor regression, with an accompanying improvement in pulmonary dyspnea, oxygen satn., and plasma tumor markers.  Significance: TRK fusions have been deemed putative oncogenic drivers, but their clin. significance remained unclear.  A patient with a metastatic soft-tissue sarcoma with an LMNA-NTRK1 fusion had rapid and substantial tumor regression with a novel, highly selective TRK inhibitor, LOXO-101, providing the first clin. evidence of benefit from inhibiting TRK fusions.  Cancer Discov; 5(10); 1049-57. ©2015 AACR.  This article is highlighted in the In This Issue feature, p.  1005.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqOOz1IuW1mjrVg90H21EOLACvtfcHk0lhz2OQBHmHA_g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhs1Wqs7fF&md5=52fcc72687bcd256dc54e77936d3ebee</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1158%2F2159-8290.CD-15-0443&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F2159-8290.CD-15-0443%26sid%3Dliteratum%253Aachs%26aulast%3DDoebele%26aufirst%3DR.%2BC.%26aulast%3DDavis%26aufirst%3DL.%2BE.%26aulast%3DVaishnavi%26aufirst%3DA.%26aulast%3DLe%26aufirst%3DA.%2BT.%26aulast%3DEstrada-Bernal%26aufirst%3DA.%26aulast%3DKeysar%26aufirst%3DS.%26aulast%3DJimeno%26aufirst%3DA.%26aulast%3DVarella-Garcia%26aufirst%3DM.%26aulast%3DAisner%26aufirst%3DD.%2BL.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DStephens%26aufirst%3DP.%2BJ.%26aulast%3DMorosini%26aufirst%3DD.%26aulast%3DTuch%26aufirst%3DB.%2BB.%26aulast%3DFernandes%26aufirst%3DM.%26aulast%3DNanda%26aufirst%3DN.%26aulast%3DLow%26aufirst%3DJ.%2BA.%26atitle%3DAn%2520oncogenic%2520NTRK%2520fusion%2520in%2520a%2520patient%2520with%2520soft-tissue%2520sarcoma%2520with%2520response%2520to%2520the%2520tropomyosin-related%2520kinase%2520inhibitor%2520LOXO-101%26jtitle%3DCanc.%2520Discov.%26date%3D2015%26volume%3D5%26spage%3D1049%26epage%3D1057%26doi%3D10.1158%2F2159-8290.cd-15-0443" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hung, Y. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fletcher, C. D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hornick, J. L.</span></span> <span> </span><span class="NLM_article-title">Evaluation of pan-TRK immunohistochemistry in infantile fibrosarcoma, lipofibromatosis-like neural tumour and histological mimics</span>. <i>Histopathology</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>73</i></span>,  <span class="NLM_fpage">634</span>– <span class="NLM_lpage">644</span>, <span class="refDoi"> DOI: 10.1111/his.13666</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=10.1111%2Fhis.13666" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=29863809" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=1%3ACAS%3A280%3ADC%252BC1MbjsVKqsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=73&publication_year=2018&pages=634-644&author=Y.+P.+Hungauthor=C.+D.+M.+Fletcherauthor=J.+L.+Hornick&title=Evaluation+of+pan-TRK+immunohistochemistry+in+infantile+fibrosarcoma%2C+lipofibromatosis-like+neural+tumour+and+histological+mimics&doi=10.1111%2Fhis.13666"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Evaluation of pan-TRK immunohistochemistry in infantile fibrosarcoma, lipofibromatosis-like neural tumour and histological mimics</span></div><div class="casAuthors">Hung Yin P; Fletcher Christopher D M; Hornick Jason L</div><div class="citationInfo"><span class="NLM_cas:title">Histopathology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">73</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">634-644</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">AIMS:  Infantile fibrosarcoma is characterised by intersecting fascicles of spindle cells and ETV6-NTRK3 gene fusion in most cases.  Given histological overlap with other spindle-cell tumours, the diagnosis can be challenging and often requires molecular confirmation.  A recently developed pan-TRK antibody shows promise for identifying tumours with NTRK fusions.  The purpose of this study was to evaluate the potential diagnostic utility of pan-TRK immunohistochemistry for infantile fibrosarcoma.  METHODS AND RESULTS:  We evaluated whole-tissue sections from 210 cases, including 15 infantile fibrosarcomas; five each lipofibromatosis-like neural tumour and lipofibromatosis; 10 each primitive myxoid mesenchymal tumour of infancy (PMMTI) and low-grade myofibroblastic sarcoma; 15 each fibrous hamartoma of infancy (FHI), myofibroma/myofibromatosis and desmoid-type fibromatosis; and 20 each low-grade fibromyxoid sarcoma, synovial sarcoma, spindle-cell rhabdomyosarcoma, malignant peripheral nerve sheath tumour, fibrosarcomatous dermatofibrosarcoma protuberans (F-DFSP) and nodular fasciitis.  Immunohistochemistry was performed using a rabbit monoclonal pan-TRK antibody.  Immunoreactivity for pan-TRK was observed in all 15 (100%) infantile fibrosarcomas, including diffuse immunoreactivity (>50% of cells) in 14 (93%) cases.  Pan-TRK was positive in all five (100%) lipofibromatosis-like neural tumours.  Of the 190 histological mimics, diffuse pan-TRK immunoreactivity was noted in 16 (8%) cases, including five PMMTI, five FHI (highlighting predominantly the primitive myxoid spindle-cell components), three F-DFSP, one low-grade myofibroblastic sarcoma, one myofibroma and one spindle-cell rhabdomyosarcoma.  CONCLUSIONS:  Diffuse pan-TRK immunoreactivity is a highly sensitive but not entirely specific diagnostic marker for infantile fibrosarcoma, and may be helpful in selecting patients for TRK-targeted therapy.  As expected, lipofibromatosis-like neural tumours, which harbour NTRK1 fusions, also show diffuse pan-TRK immunoreactivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQOm9XON-5CyinGiHuhFGv8fW6udTcc2eYdO5Y6DRwJe7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1MbjsVKqsg%253D%253D&md5=4e22b595fd2cb8e304991c4922c0f380</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1111%2Fhis.13666&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fhis.13666%26sid%3Dliteratum%253Aachs%26aulast%3DHung%26aufirst%3DY.%2BP.%26aulast%3DFletcher%26aufirst%3DC.%2BD.%2BM.%26aulast%3DHornick%26aufirst%3DJ.%2BL.%26atitle%3DEvaluation%2520of%2520pan-TRK%2520immunohistochemistry%2520in%2520infantile%2520fibrosarcoma%252C%2520lipofibromatosis-like%2520neural%2520tumour%2520and%2520histological%2520mimics%26jtitle%3DHistopathology%26date%3D2018%26volume%3D73%26spage%3D634%26epage%3D644%26doi%3D10.1111%2Fhis.13666" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tognon, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knezevich, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huntsman, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roskelley, C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Melnyk, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mathers, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Becker, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carneiro, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MacPherson, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Horsman, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poremba, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sorensen, P. H. B.</span></span> <span> </span><span class="NLM_article-title">Expression of the ETV6-NTRK3 gene fusion as a primary event in human secretory breast carcinoma</span>. <i>Cancer Cell</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>2</i></span>,  <span class="NLM_fpage">367</span>– <span class="NLM_lpage">376</span>, <span class="refDoi"> DOI: 10.1016/s1535-6108(02)00180-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=10.1016%2FS1535-6108%2802%2900180-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=12450792" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=1%3ACAS%3A528%3ADC%252BD38Xptlalurw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2002&pages=367-376&author=C.+Tognonauthor=S.+R.+Knezevichauthor=D.+Huntsmanauthor=C.+D.+Roskelleyauthor=N.+Melnykauthor=J.+A.+Mathersauthor=L.+Beckerauthor=F.+Carneiroauthor=N.+MacPhersonauthor=D.+Horsmanauthor=C.+Porembaauthor=P.+H.+B.+Sorensen&title=Expression+of+the+ETV6-NTRK3+gene+fusion+as+a+primary+event+in+human+secretory+breast+carcinoma&doi=10.1016%2Fs1535-6108%2802%2900180-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Expression of the ETV6-NTRK3 gene fusion as a primary event in human secretory breast carcinoma</span></div><div class="casAuthors">Tognon, Cristina; Knezevich, Stevan R.; Huntsman, David; Roskelley, Calvin D.; Melnyk, Natalya; Mathers, Joan A.; Becker, Laurence; Carneiro, Fatima; MacPherson, Nicol; Horsman, Doug; Poremba, Christopher; Sorensen, Poul H. B.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">367-376</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">We report that human secretory breast carcinoma (SBC), a rare subtype of infiltrating ductal carcinoma, expresses the ETV6-NTRK3 gene fusion previously cloned in pediatric mesenchymal cancers.  This gene fusion encodes a chimeric tyrosine kinase with potent transforming activity in fibroblasts.  ETV6-NTRK3 expression was confirmed in 12 (92%) of 13 SBC cases, but not in other ductal carcinomas.  Retroviral transfer of ETV6-NTRK3 (EN) into murine mammary epithelial cells resulted in transformed cells that readily formed tumors in nude mice.  Phenotypically, tumors produced glands and expressed epithelial antigens, confirming that EN transformation is compatible with epithelial differentiation.  This represents a recurrent chromosomal rearrangement and expression of a dominantly acting oncogene as a primary event in human breast carcinoma.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp9UOXU8lUATrVg90H21EOLACvtfcHk0ljHVDkGjWQuuQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38Xptlalurw%253D&md5=0b6abd134a44218e8ce03ffd25effe2d</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1016%2FS1535-6108%2802%2900180-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1535-6108%252802%252900180-0%26sid%3Dliteratum%253Aachs%26aulast%3DTognon%26aufirst%3DC.%26aulast%3DKnezevich%26aufirst%3DS.%2BR.%26aulast%3DHuntsman%26aufirst%3DD.%26aulast%3DRoskelley%26aufirst%3DC.%2BD.%26aulast%3DMelnyk%26aufirst%3DN.%26aulast%3DMathers%26aufirst%3DJ.%2BA.%26aulast%3DBecker%26aufirst%3DL.%26aulast%3DCarneiro%26aufirst%3DF.%26aulast%3DMacPherson%26aufirst%3DN.%26aulast%3DHorsman%26aufirst%3DD.%26aulast%3DPoremba%26aufirst%3DC.%26aulast%3DSorensen%26aufirst%3DP.%2BH.%2BB.%26atitle%3DExpression%2520of%2520the%2520ETV6-NTRK3%2520gene%2520fusion%2520as%2520a%2520primary%2520event%2520in%2520human%2520secretory%2520breast%2520carcinoma%26jtitle%3DCancer%2520Cell%26date%3D2002%26volume%3D2%26spage%3D367%26epage%3D376%26doi%3D10.1016%2Fs1535-6108%2802%2900180-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Skálová, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vanecek, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simpson, R. H. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laco, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Majewska, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baneckova, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steiner, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michal, M.</span></span> <span> </span><span class="NLM_article-title">Mammary Analogue Secretory Carcinoma of Salivary Glands</span>. <i>Am. J. Surg. Pathol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>40</i></span>,  <span class="NLM_fpage">3</span>– <span class="NLM_lpage">13</span>, <span class="refDoi"> DOI: 10.1097/pas.0000000000000537</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=10.1097%2FPAS.0000000000000537" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=26492182" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=1%3ACAS%3A280%3ADC%252BC28zkt1SmsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2016&pages=3-13&author=A.+Sk%C3%A1lov%C3%A1author=T.+Vanecekauthor=R.+H.+W.+Simpsonauthor=J.+Lacoauthor=H.+Majewskaauthor=M.+Baneckovaauthor=P.+Steinerauthor=M.+Michal&title=Mammary+Analogue+Secretory+Carcinoma+of+Salivary+Glands&doi=10.1097%2Fpas.0000000000000537"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Mammary Analogue Secretory Carcinoma of Salivary Glands: Molecular Analysis of 25 ETV6 Gene Rearranged Tumors With Lack of Detection of Classical ETV6-NTRK3 Fusion Transcript by Standard RT-PCR: Report of 4 Cases Harboring ETV6-X Gene Fusion</span></div><div class="casAuthors">Skalova Alena; Vanecek Tomas; Simpson Roderick H W; Laco Jan; Majewska Hanna; Baneckova Martina; Steiner Petr; Michal Michal</div><div class="citationInfo"><span class="NLM_cas:title">The American journal of surgical pathology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">3-13</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">ETV6 gene abnormalities are well described in tumor pathology.  Many fusion partners of ETV6 have been reported in a variety of epithelial and hematological malignancies.  In salivary gland tumor pathology, however, the ETV6-NTRK3 translocation is specific for mammary analogue secretory carcinoma (MASC), and has not been documented in any other salivary tumor type.  The present study comprised a clinical and molecular analysis of 25 cases morphologically and immunohistochemically typical of MASC.  They all also displayed the ETV6 rearrangement as visualized by fluorescent in situ hybridization but lacked the classical ETV6-NTRK3 fusion transcript by standard reverse-transcriptase-polymerase chain reaction.  In 4 cases, the classical fusion transcript was found by more sensitive, nested reverse-transcription-polymerase chain reaction.  Five other cases harbored atypical fusion transcripts as detected by both standard and nested reverse-transcription-polymerase chain reaction.  In addition, fluorescent in situ hybridization with an NTRK3 break-apart probe was also performed; rearrangement of NTRK3 gene was detected in 16 of 25 cases.  In 3 other cases, the tissue was not analyzable, and in 2 further cases analysis could not be performed because of a lack of appropriate tissue material.  Finally, in the 4 remaining cases whose profile was NTRK3 split-negative and ETV6 split-positive, unknown (non-NTRK) genes appeared to fuse with ETV6 (ETV6-X fusion).  In looking for possible fusion partners, analysis of rearrangement of other kinase genes known to fuse with ETV6 was also performed, but without positive results.  Although numbers were small, correlating the clinico-pathologic features of the 4 ETV6-X fusion tumors and 5 MASC cases with atypical fusion transcripts raises the possibility of that they may behave more aggressively.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRCDWAmzOrORQxyojvcjibhfW6udTcc2eYjtXb9W6jaqLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC28zkt1SmsQ%253D%253D&md5=a2e9b88b4c578008679b67647ef18763</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1097%2FPAS.0000000000000537&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1097%252FPAS.0000000000000537%26sid%3Dliteratum%253Aachs%26aulast%3DSk%25C3%25A1lov%25C3%25A1%26aufirst%3DA.%26aulast%3DVanecek%26aufirst%3DT.%26aulast%3DSimpson%26aufirst%3DR.%2BH.%2BW.%26aulast%3DLaco%26aufirst%3DJ.%26aulast%3DMajewska%26aufirst%3DH.%26aulast%3DBaneckova%26aufirst%3DM.%26aulast%3DSteiner%26aufirst%3DP.%26aulast%3DMichal%26aufirst%3DM.%26atitle%3DMammary%2520Analogue%2520Secretory%2520Carcinoma%2520of%2520Salivary%2520Glands%26jtitle%3DAm.%2520J.%2520Surg.%2520Pathol.%26date%3D2016%26volume%3D40%26spage%3D3%26epage%3D13%26doi%3D10.1097%2Fpas.0000000000000537" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Drilon, A.</span></span> <span> </span><span class="NLM_article-title">TRK inhibitors in TRK fusion-positive cancers</span>. <i>Ann. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">viii23</span>– <span class="NLM_lpage">viii30</span>, <span class="refDoi"> DOI: 10.1093/annonc/mdz282</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=10.1093%2Fannonc%2Fmdz282" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2019&pages=viii23-viii30&author=A.+Drilon&title=TRK+inhibitors+in+TRK+fusion-positive+cancers&doi=10.1093%2Fannonc%2Fmdz282"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1093%2Fannonc%2Fmdz282&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fannonc%252Fmdz282%26sid%3Dliteratum%253Aachs%26aulast%3DDrilon%26aufirst%3DA.%26atitle%3DTRK%2520inhibitors%2520in%2520TRK%2520fusion-positive%2520cancers%26jtitle%3DAnn.%2520Oncol.%26date%3D2019%26volume%3D30%26spage%3Dviii23%26epage%3Dviii30%26doi%3D10.1093%2Fannonc%2Fmdz282" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Scott, L. J.</span></span> <span> </span><span class="NLM_article-title">Larotrectinib: first global approval</span>. <i>Drugs</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>79</i></span>,  <span class="NLM_fpage">201</span>– <span class="NLM_lpage">206</span>, <span class="refDoi"> DOI: 10.1007/s40265-018-1044-x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=10.1007%2Fs40265-018-1044-x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=30635837" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=1%3ACAS%3A528%3ADC%252BC1MXnvFOqsb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=79&publication_year=2019&pages=201-206&author=L.+J.+Scott&title=Larotrectinib%3A+first+global+approval&doi=10.1007%2Fs40265-018-1044-x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Larotrectinib: First Global Approval</span></div><div class="casAuthors">Scott, Lesley J.</div><div class="citationInfo"><span class="NLM_cas:title">Drugs</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">79</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">201-206</span>CODEN:
                <span class="NLM_cas:coden">DRUGAY</span>;
        ISSN:<span class="NLM_cas:issn">0012-6667</span>.
    
            (<span class="NLM_cas:orgname">Springer International Publishing AG</span>)
        </div><div class="casAbstract">A review.  Larotrectinib (Vitrakvi) is an orally administered, small mol., highly-selective, tropomyosin receptor kinase (TRK) inhibitor that was developed by Loxo Oncol. in collaboration with Bayer AG as a treatment for adult and paediatric patients whose cancers harbor neurotrophic receptor tyrosine kinase (NTRK) gene fusions.  In Nov. 2018 larotrectinib received its first global approval in the USA for the treatment of adult and paediatric patients with solid tumors that have a NTRK gene fusion without a known acquired resistance mutation, are metastatic or where surgical resection is likely to result in severe morbidity, and have no satisfactory alternative treatments or that have progressed following treatment.  A registration application for the use of larotrectinib in pediatric and adult patients with locally advanced or metastatic solid tumors with NTRK gene fusion proteins has been submitted in the EU.  This article summarizes the milestones in the development of larotrectinib leading to its first approval for the treatment of adult and pediatric patients with solid tumors that have NTRK gene fusion.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpMpwdBG0H7ELVg90H21EOLACvtfcHk0ljHVDkGjWQuuQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXnvFOqsb8%253D&md5=8c3d442665ece609cc09422052903696</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1007%2Fs40265-018-1044-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs40265-018-1044-x%26sid%3Dliteratum%253Aachs%26aulast%3DScott%26aufirst%3DL.%2BJ.%26atitle%3DLarotrectinib%253A%2520first%2520global%2520approval%26jtitle%3DDrugs%26date%3D2019%26volume%3D79%26spage%3D201%26epage%3D206%26doi%3D10.1007%2Fs40265-018-1044-x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Al-Salama, Z. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keam, S. J.</span></span> <span> </span><span class="NLM_article-title">Entrectinib: first global approval</span>. <i>Drugs</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>79</i></span>,  <span class="NLM_fpage">1477</span>– <span class="NLM_lpage">1483</span>, <span class="refDoi"> DOI: 10.1007/s40265-019-01177-y</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=10.1007%2Fs40265-019-01177-y" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=31372957" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=1%3ACAS%3A280%3ADC%252BB3MvktVemsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=79&publication_year=2019&pages=1477-1483&author=Z.+T.+Al-Salamaauthor=S.+J.+Keam&title=Entrectinib%3A+first+global+approval&doi=10.1007%2Fs40265-019-01177-y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Entrectinib: First Global Approval</span></div><div class="casAuthors">Al-Salama Zaina T; Keam Susan J</div><div class="citationInfo"><span class="NLM_cas:title">Drugs</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">79</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">1477-1483</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Entrectinib (Rozlytrek(®)) is an oral selective inhibitor of the tyrosine kinases tropomyosin receptor kinases (Trk)A/B/C [encoded by the genes neurotrophic tyrosine receptor kinase (NTRK) 1, 2 and 3, respectively], c-ros oncogene 1 (ROS1) and anaplastic lymphoma kinase (ALK) with central nervous system (CNS) activity developed by Roche for the treatment of various solid tumours harbouring NTRK1/2/3 or ROS1 gene fusions.  In June 2019, entrectinib received its first global approval in Japan, for the treatment of adult and paediatric patients with NTRK fusion-positive, advanced or recurrent solid tumours and is under regulatory review for the treatment of adult patients with ROS1-positive non-small cell lung cancer (NSCLC).  Entrectinib is also under regulatory review in the USA (PDUFA date 18 August 2019) and EU [Priority Medicines (PRIME) designation] for NTRK-positive solid tumours and ROS1-positive NSCLC.  This article summarizes the milestones in the development of entrectinib leading to this first global approval for solid tumours in Japan.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRqIyGZmvQlpEkUaC_kzhvyfW6udTcc2eYo6ou19iO0ebntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3MvktVemsg%253D%253D&md5=750c86d20c43273cf07f505988ac8dbd</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1007%2Fs40265-019-01177-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs40265-019-01177-y%26sid%3Dliteratum%253Aachs%26aulast%3DAl-Salama%26aufirst%3DZ.%2BT.%26aulast%3DKeam%26aufirst%3DS.%2BJ.%26atitle%3DEntrectinib%253A%2520first%2520global%2520approval%26jtitle%3DDrugs%26date%3D2019%26volume%3D79%26spage%3D1477%26epage%3D1483%26doi%3D10.1007%2Fs40265-019-01177-y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sun, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tong, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rao, Y.</span></span> <span> </span><span class="NLM_article-title">PROTACs: great opportunities for academia and industry</span>. <i>Signal Transduction Targeted Ther.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">64</span>, <span class="refDoi"> DOI: 10.1038/s41392-019-0101-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=10.1038%2Fs41392-019-0101-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=31885879" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=1%3ACAS%3A280%3ADC%252BB3MbkvFCrtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2019&pages=64&author=X.+Sunauthor=H.+Gaoauthor=Y.+Yangauthor=M.+Heauthor=Y.+Wuauthor=Y.+Songauthor=Y.+Tongauthor=Y.+Rao&title=PROTACs%3A+great+opportunities+for+academia+and+industry&doi=10.1038%2Fs41392-019-0101-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">PROTACs: great opportunities for academia and industry</span></div><div class="casAuthors">Sun Xiuyun; Gao Hongying; Yang Yiqing; He Ming; Wu Yue; Song Yugang; Tong Yan; Rao Yu; Sun Xiuyun; Gao Hongying; Yang Yiqing; Rao Yu</div><div class="citationInfo"><span class="NLM_cas:title">Signal transduction and targeted therapy</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">64</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Although many kinds of therapies are applied in the clinic, drug-resistance is a major and unavoidable problem.  Another disturbing statistic is the limited number of drug targets, which are presently only 20-25% of all protein targets that are currently being studied.  Moreover, the focus of current explorations of targets are their enzymatic functions, which ignores the functions from their scaffold moiety.  As a promising and appealing technology, PROteolysis TArgeting Chimeras (PROTACs) have attracted great attention both from academia and industry for finding available approaches to solve the above problems.  PROTACs regulate protein function by degrading target proteins instead of inhibiting them, providing more sensitivity to drug-resistant targets and a greater chance to affect the nonenzymatic functions.  PROTACs have been proven to show better selectivity compared to classic inhibitors.  PROTACs can be described as a chemical knockdown approach with rapidity and reversibility, which presents new and different biology compared to other gene editing tools by avoiding misinterpretations that arise from potential genetic compensation and/or spontaneous mutations.  PRTOACs have been widely explored throughout the world and have outperformed not only in cancer diseases, but also in immune disorders, viral infections and neurodegenerative diseases.  Although PROTACs present a very promising and powerful approach for crossing the hurdles of present drug discovery and tool development in biology, more efforts are needed to gain to get deeper insight into the efficacy and safety of PROTACs in the clinic.  More target binders and more E3 ligases applicable for developing PROTACs are waiting for exploration.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQretvERU2xT-_Q9URG6Dx8fW6udTcc2eYo6ou19iO0ebntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3MbkvFCrtA%253D%253D&md5=7d9545a21e3a78cededf2977a31e8f7d</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1038%2Fs41392-019-0101-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41392-019-0101-6%26sid%3Dliteratum%253Aachs%26aulast%3DSun%26aufirst%3DX.%26aulast%3DGao%26aufirst%3DH.%26aulast%3DYang%26aufirst%3DY.%26aulast%3DHe%26aufirst%3DM.%26aulast%3DWu%26aufirst%3DY.%26aulast%3DSong%26aufirst%3DY.%26aulast%3DTong%26aufirst%3DY.%26aulast%3DRao%26aufirst%3DY.%26atitle%3DPROTACs%253A%2520great%2520opportunities%2520for%2520academia%2520and%2520industry%26jtitle%3DSignal%2520Transduction%2520Targeted%2520Ther.%26date%3D2019%26volume%3D4%26spage%3D64%26doi%3D10.1038%2Fs41392-019-0101-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rao, Y.</span></span> <span> </span><span class="NLM_article-title">PROTAC technology: opportunities and challenges</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">237</span>– <span class="NLM_lpage">240</span>, <span class="refDoi"> DOI: 10.1021/acsmedchemlett.9b00597</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.9b00597" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=1%3ACAS%3A280%3ADC%252BB383kslaitA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2020&pages=237-240&author=H.+Gaoauthor=X.+Sunauthor=Y.+Rao&title=PROTAC+technology%3A+opportunities+and+challenges&doi=10.1021%2Facsmedchemlett.9b00597"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">PROTAC Technology: Opportunities and Challenges</span></div><div class="casAuthors">Gao Hongying; Sun Xiuyun; Rao Yu; Gao Hongying; Sun Xiuyun</div><div class="citationInfo"><span class="NLM_cas:title">ACS medicinal chemistry letters</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">237-240</span>
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    </div><div class="casAbstract">PROTACs-induced targeted protein degradation has emerged as a novel therapeutic strategy in drug development and attracted the favor of academic institutions, large pharmaceutical enterprises (e.g., AstraZeneca, Bayer, Novartis, Amgen, Pfizer, GlaxoSmithKline, Merck, and Boehringer Ingelheim, etc.), and biotechnology companies.  PROTACs opened a new chapter for novel drug development.  However, any new technology will face many new problems and challenges.  Perspectives on the potential opportunities and challenges of PROTACs will contribute to the research and development of new protein degradation drugs and degrader tools.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSj8qlWczrRKmx8TEN9K18QfW6udTcc2eYo6ou19iO0ebntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB383kslaitA%253D%253D&md5=68988d6477e83f4ebc26d11e3b8ce5b1</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.9b00597&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.9b00597%26sid%3Dliteratum%253Aachs%26aulast%3DGao%26aufirst%3DH.%26aulast%3DSun%26aufirst%3DX.%26aulast%3DRao%26aufirst%3DY.%26atitle%3DPROTAC%2520technology%253A%2520opportunities%2520and%2520challenges%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2020%26volume%3D11%26spage%3D237%26epage%3D240%26doi%3D10.1021%2Facsmedchemlett.9b00597" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lai, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Induced protein degradation: an emerging drug discovery paradigm</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">101</span>– <span class="NLM_lpage">114</span>, <span class="refDoi"> DOI: 10.1038/nrd.2016.211</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=10.1038%2Fnrd.2016.211" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=27885283" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvFCit7rP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2017&pages=101-114&author=A.+C.+Laiauthor=C.+M.+Crews&title=Induced+protein+degradation%3A+an+emerging+drug+discovery+paradigm&doi=10.1038%2Fnrd.2016.211"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Induced protein degradation: an emerging drug discovery paradigm</span></div><div class="casAuthors">Lai, Ashton C.; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">101-114</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Small-mol. drug discovery has traditionally focused on occupancy of a binding site that directly affects protein function, and this approach typically precludes targeting proteins that lack such amenable sites.  Furthermore, high systemic drug exposures may be needed to maintain sufficient target inhibition in vivo, increasing the risk of undesirable off-target effects.  Induced protein degrdn. is an alternative approach that is event-driven: upon drug binding, the target protein is tagged for elimination.  Emerging technologies based on proteolysis-targeting chimaeras (PROTACs) that exploit cellular quality control machinery to selectively degrade target proteins are attracting considerable attention in the pharmaceutical industry owing to the advantages they could offer over traditional small-mol. strategies.  These advantages include the potential to reduce systemic drug exposure, the ability to counteract increased target protein expression that often accompanies inhibition of protein function and the potential ability to target proteins that are not currently therapeutically tractable, such as transcription factors, scaffolding and regulatory proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoj-O7zXGOpXLVg90H21EOLACvtfcHk0lilcqOHWwTYFg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvFCit7rP&md5=a01ff43e9b696fe4dc03f0815158a98f</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2016.211&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2016.211%26sid%3Dliteratum%253Aachs%26aulast%3DLai%26aufirst%3DA.%2BC.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DInduced%2520protein%2520degradation%253A%2520an%2520emerging%2520drug%2520discovery%2520paradigm%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2017%26volume%3D16%26spage%3D101%26epage%3D114%26doi%3D10.1038%2Fnrd.2016.211" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schapira, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Calabrese, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bullock, A. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Targeted protein degradation: expanding the toolbox</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">949</span>– <span class="NLM_lpage">963</span>, <span class="refDoi"> DOI: 10.1038/s41573-019-0047-y</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=10.1038%2Fs41573-019-0047-y" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=31666732" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitVKrtLrM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2019&pages=949-963&author=M.+Schapiraauthor=M.+F.+Calabreseauthor=A.+N.+Bullockauthor=C.+M.+Crews&title=Targeted+protein+degradation%3A+expanding+the+toolbox&doi=10.1038%2Fs41573-019-0047-y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Targeted protein degradation: expanding the toolbox</span></div><div class="casAuthors">Schapira, Matthieu; Calabrese, Matthew F.; Bullock, Alex N.; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">949-963</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">A review. Proteolysis-targeting chimeras (PROTACs) and related mols. that induce targeted protein degrdn. by the ubiquitin-proteasome system represent a new therapeutic modality and are the focus of great interest, owing to potential advantages over traditional occupancy-based inhibitors with respect to dosing, side effects, drug resistance and modulating 'undruggable' targets.  However, the technol. is still maturing, and the design elements for successful PROTAC-based drugs are currently being elucidated.  Importantly, fewer than 10 of the more than 600 E3 ubiquitin ligases have so far been exploited for targeted protein degrdn., and expansion of knowledge in this area is a key opportunity.  Here, we briefly discuss lessons learned about targeted protein degrdn. in chem. biol. and drug discovery and systematically review the expression profile, domain architecture and chem. tractability of human E3 ligases that could expand the toolbox for PROTAC discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpwUvM27MU3WrVg90H21EOLACvtfcHk0lhkSnDdQEMsbQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitVKrtLrM&md5=9bfd48c6c47fc1cce15129495fea7079</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1038%2Fs41573-019-0047-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41573-019-0047-y%26sid%3Dliteratum%253Aachs%26aulast%3DSchapira%26aufirst%3DM.%26aulast%3DCalabrese%26aufirst%3DM.%2BF.%26aulast%3DBullock%26aufirst%3DA.%2BN.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DTargeted%2520protein%2520degradation%253A%2520expanding%2520the%2520toolbox%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2019%26volume%3D18%26spage%3D949%26epage%3D963%26doi%3D10.1038%2Fs41573-019-0047-y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ohoka, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shibata, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hattori, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Naito, M.</span></span> <span> </span><span class="NLM_article-title">Protein knockdown technology: application of ubiquitin ligase to cancer therapy</span>. <i>Curr. Cancer Drug Targets</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">136</span>– <span class="NLM_lpage">146</span>, <span class="refDoi"> DOI: 10.2174/1568009616666151112122502</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=10.2174%2F1568009616666151112122502" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=26560118" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=1%3ACAS%3A528%3ADC%252BC28XnvFOqtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2016&pages=136-146&author=N.+Ohokaauthor=N.+Shibataauthor=T.+Hattoriauthor=M.+Naito&title=Protein+knockdown+technology%3A+application+of+ubiquitin+ligase+to+cancer+therapy&doi=10.2174%2F1568009616666151112122502"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Protein Knockdown Technology: Application of Ubiquitin Ligase to Cancer Therapy</span></div><div class="casAuthors">Ohoka, Nobumichi; Shibata, Norihito; Hattori, Takayuki; Naito, Mikihiko</div><div class="citationInfo"><span class="NLM_cas:title">Current Cancer Drug Targets</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">136-146</span>CODEN:
                <span class="NLM_cas:coden">CCDTB9</span>;
        ISSN:<span class="NLM_cas:issn">1568-0096</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">Selective degrdn. of pathogenic proteins by small mols. in cells is a novel approach for development of therapeutic agents against various diseases, including cancer.  We and others have developed a protein knockdown technol. with a series of hybrid small compds., called SNIPERs (Specific and Nongenetic IAP-dependent Protein ERasers); and peptidic chimeric mols., called PROTACs (proteolysis-targeting chimeric mols.), which induce selective degrdn. of target proteins via the ubiquitin-proteasome pathway.  These compds. include two different ligands connected by a linker; one is a ligand for a ubiquitin ligase and the other is a ligand for the target protein, which are expected to crosslink these proteins in cells.  Theor., any cytosolic protein can be targeted for degrdn. by this technol.  To date, several SNIPERs and PROTACs against various oncogenic proteins have been developed, which specifically induce polyubiquitylation and proteasomal degrdn. of the oncogenic proteins, resulting in cell death, growth arrest, or impaired migration of cancer cells.  Thus, this protein knockdown technol. has a great potential for cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrmS_kGOWfeuLVg90H21EOLACvtfcHk0lhkSnDdQEMsbQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XnvFOqtw%253D%253D&md5=d78f1ae70ea89843d9f946454cc95058</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.2174%2F1568009616666151112122502&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F1568009616666151112122502%26sid%3Dliteratum%253Aachs%26aulast%3DOhoka%26aufirst%3DN.%26aulast%3DShibata%26aufirst%3DN.%26aulast%3DHattori%26aufirst%3DT.%26aulast%3DNaito%26aufirst%3DM.%26atitle%3DProtein%2520knockdown%2520technology%253A%2520application%2520of%2520ubiquitin%2520ligase%2520to%2520cancer%2520therapy%26jtitle%3DCurr.%2520Cancer%2520Drug%2520Targets%26date%3D2016%26volume%3D16%26spage%3D136%26epage%3D146%26doi%3D10.2174%2F1568009616666151112122502" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cromm, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Samarasinghe, K. T. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hines, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Addressing kinase-independent functions of Fak via PROTAC-mediated degradation</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>140</i></span>,  <span class="NLM_fpage">17019</span>– <span class="NLM_lpage">17026</span>, <span class="refDoi"> DOI: 10.1021/jacs.8b08008</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.8b08008" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=1%3ACAS%3A528%3ADC%252BC1cXit1CitLnF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=140&publication_year=2018&pages=17019-17026&author=P.+M.+Crommauthor=K.+T.+G.+Samarasingheauthor=J.+Hinesauthor=C.+M.+Crews&title=Addressing+kinase-independent+functions+of+Fak+via+PROTAC-mediated+degradation&doi=10.1021%2Fjacs.8b08008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Addressing Kinase-Independent Functions of Fak via PROTAC-Mediated Degradation</span></div><div class="casAuthors">Cromm, Philipp M.; Samarasinghe, Kusal T. G.; Hines, John; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">140</span>
        (<span class="NLM_cas:issue">49</span>),
    <span class="NLM_cas:pages">17019-17026</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Enzymic inhibition proved to be a successful modality for the development of many small-mol. drugs.  In recent years, small-mol.-induced protein degrdn. has emerged as an orthogonal therapeutic strategy that has the potential to expand the druggable target space.  Focal adhesion kinase (Fak) is a key player in tumor invasion and metastasis, acting simultaneously as a kinase and a scaffold for several signaling proteins.  While previous efforts to modulate Fak activity were limited to kinase inhibitors with low success in clin. studies, protein degrdn. offers a possibility to simultaneously block Fak's kinase signaling and scaffolding capabilities.  Here, the authors report the development of a selective and potent Fak degrader, PROTAC-3, which outperforms a clin. candidate, defactinib, with respect to Fak activation as well as Fak-mediated cell migration and invasion.  These results underline the potential that PROTACs offer in expanding the druggable space and controlling protein functions that are not easily addressed by traditional small-mol. therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqTn8r4NmDDObVg90H21EOLACvtfcHk0lhkSnDdQEMsbQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXit1CitLnF&md5=b97b1a646a8f9dbf7a2bed640d0f001d</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1021%2Fjacs.8b08008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.8b08008%26sid%3Dliteratum%253Aachs%26aulast%3DCromm%26aufirst%3DP.%2BM.%26aulast%3DSamarasinghe%26aufirst%3DK.%2BT.%2BG.%26aulast%3DHines%26aufirst%3DJ.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DAddressing%2520kinase-independent%2520functions%2520of%2520Fak%2520via%2520PROTAC-mediated%2520degradation%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2018%26volume%3D140%26spage%3D17019%26epage%3D17026%26doi%3D10.1021%2Fjacs.8b08008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rao, Y.</span></span> <span> </span><span class="NLM_article-title">Design, synthesis, and evaluation of highly potent FAK-targeting PROTACs</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">1855</span>– <span class="NLM_lpage">1862</span>, <span class="refDoi"> DOI: 10.1021/acsmedchemlett.9b00372</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.9b00372" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitVajtrfJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2020&pages=1855-1862&author=H.+Gaoauthor=Y.+Wuauthor=Y.+Sunauthor=Y.+Yangauthor=G.+Zhouauthor=Y.+Rao&title=Design%2C+synthesis%2C+and+evaluation+of+highly+potent+FAK-targeting+PROTACs&doi=10.1021%2Facsmedchemlett.9b00372"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Design, Synthesis, and Evaluation of Highly Potent FAK-Targeting PROTACs</span></div><div class="casAuthors">Gao, Hongying; Wu, Yue; Sun, Yonghui; Yang, Yiqing; Zhou, Guangbiao; Rao, Yu</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1855-1862</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Focal adhesion kinase (FAK), a cytoplasmic protein tyrosine kinase, exerts kinase-dependent enzymic functions and kinase-independent scaffolding functions, both of which are crucial in cancer development, early embryonic development, and reprodn.  However, previous efforts for FAK blocking mainly focus on kinase inhibitors.  Proteolysis targeting chimeras (PROTACs) are heterobifunctional mols. that allow direct post-translational knockdown of proteins via ubiquitination of a target protein by E3 ubiquitin ligase and subsequent proteasomal degrdn.  Here, we designed and synthesized a FAK PROTAC library with FAK inhibitor (PF562271 or VS6063) and CRBN E3 ligand.  A novel FAK-targeting PROTAC, FC-11, showed a rapid and reversible FAK degrdn. with a picomolar of DC50 in various cell lines in vitro, which imply that FAK-PROTACs could be useful as expand tools for studying functions of FAK in biol. system and as potential therapeutic agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoDJ4SVK5xRNrVg90H21EOLACvtfcHk0liceMo2MxBaEQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitVajtrfJ&md5=79cacdc66925d21441732c26c9b268d4</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.9b00372&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.9b00372%26sid%3Dliteratum%253Aachs%26aulast%3DGao%26aufirst%3DH.%26aulast%3DWu%26aufirst%3DY.%26aulast%3DSun%26aufirst%3DY.%26aulast%3DYang%26aufirst%3DY.%26aulast%3DZhou%26aufirst%3DG.%26aulast%3DRao%26aufirst%3DY.%26atitle%3DDesign%252C%2520synthesis%252C%2520and%2520evaluation%2520of%2520highly%2520potent%2520FAK-targeting%2520PROTACs%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2020%26volume%3D11%26spage%3D1855%26epage%3D1862%26doi%3D10.1021%2Facsmedchemlett.9b00372" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sun, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hwang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rao, Y.</span></span> <span> </span><span class="NLM_article-title">PROTAC-induced BTK degradation as a novel therapy for mutated BTK C481S induced ibrutinib-resistant B-cell malignancies</span>. <i>Cell Res.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>28</i></span>,  <span class="NLM_fpage">779</span>– <span class="NLM_lpage">781</span>, <span class="refDoi"> DOI: 10.1038/s41422-018-0055-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=10.1038%2Fs41422-018-0055-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=29875397" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=1%3ACAS%3A528%3ADC%252BC1cXht1ektbzO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2018&pages=779-781&author=Y.+Sunauthor=X.+Zhaoauthor=N.+Dingauthor=H.+Gaoauthor=Y.+Wuauthor=Y.+Yangauthor=M.+Zhaoauthor=J.+Hwangauthor=Y.+Songauthor=W.+Liuauthor=Y.+Rao&title=PROTAC-induced+BTK+degradation+as+a+novel+therapy+for+mutated+BTK+C481S+induced+ibrutinib-resistant+B-cell+malignancies&doi=10.1038%2Fs41422-018-0055-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">PROTAC-induced BTK degradation as a novel therapy for mutated BTK C481S induced ibrutinib-resistant B-cell malignancies</span></div><div class="casAuthors">Sun, Yonghui; Zhao, Xingwang; Ding, Ning; Gao, Hongying; Wu, Yue; Yang, Yiqing; Zhao, Meng; Hwang, Jinseok; Song, Yuqin; Liu, Wanli; Rao, Yu</div><div class="citationInfo"><span class="NLM_cas:title">Cell Research</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">779-781</span>CODEN:
                <span class="NLM_cas:coden">CREEB6</span>;
        ISSN:<span class="NLM_cas:issn">1001-0602</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">In this study, for the first time, we report the development of BTK-targeting degraders using the PROTAC strategy.  These PROTACs could ef fi ciently degrade ibrutinib-sensitive BTK-WT (wild type).  More importantly, our newly designed PROTACs also significantly induced the degrdn. of ibrutinib resistant BTK-C481S (50% degrdn. efficiency at 30 nM).  Furthermore, our PROTAC mols. ef fi ciently inhibited cell proliferation and colony formation, while exhibited no obvious inhibition (>1000 nM) of ITK, EGFR, and TEC, which are major off-targets of ibrutinib.  These data demonstrate the strong potential for developing PROTAC-based therapeutic mols.  The results indicate that the PROTAC strategy (protein degrdn. instead of inhibition) could be adopted as a general and powerful therapeutic treatment for drug-resistant cancers in the future.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqg_FLdaUP_HLVg90H21EOLACvtfcHk0liceMo2MxBaEQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXht1ektbzO&md5=1cca51ac76ecb3b491da0117b21271fb</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1038%2Fs41422-018-0055-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41422-018-0055-1%26sid%3Dliteratum%253Aachs%26aulast%3DSun%26aufirst%3DY.%26aulast%3DZhao%26aufirst%3DX.%26aulast%3DDing%26aufirst%3DN.%26aulast%3DGao%26aufirst%3DH.%26aulast%3DWu%26aufirst%3DY.%26aulast%3DYang%26aufirst%3DY.%26aulast%3DZhao%26aufirst%3DM.%26aulast%3DHwang%26aufirst%3DJ.%26aulast%3DSong%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DW.%26aulast%3DRao%26aufirst%3DY.%26atitle%3DPROTAC-induced%2520BTK%2520degradation%2520as%2520a%2520novel%2520therapy%2520for%2520mutated%2520BTK%2520C481S%2520induced%2520ibrutinib-resistant%2520B-cell%2520malignancies%26jtitle%3DCell%2520Res.%26date%3D2018%26volume%3D28%26spage%3D779%26epage%3D781%26doi%3D10.1038%2Fs41422-018-0055-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Salami, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alabi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Willard, R. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vitale, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McDonnell, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crew, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neklesa, T. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Androgen receptor degradation by the proteolysis-targeting chimera ARCC-4 outperforms enzalutamide in cellular models of prostate cancer drug resistance</span>. <i>Commun. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">100</span>, <span class="refDoi"> DOI: 10.1038/s42003-018-0105-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=10.1038%2Fs42003-018-0105-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=30271980" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=1%3ACAS%3A280%3ADC%252BB3czksVCksA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2018&pages=100&author=J.+Salamiauthor=S.+Alabiauthor=R.+R.+Willardauthor=N.+J.+Vitaleauthor=J.+Wangauthor=H.+Dongauthor=M.+Jinauthor=D.+P.+McDonnellauthor=A.+P.+Crewauthor=T.+K.+Neklesaauthor=C.+M.+Crews&title=Androgen+receptor+degradation+by+the+proteolysis-targeting+chimera+ARCC-4+outperforms+enzalutamide+in+cellular+models+of+prostate+cancer+drug+resistance&doi=10.1038%2Fs42003-018-0105-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Androgen receptor degradation by the proteolysis-targeting chimera ARCC-4 outperforms enzalutamide in cellular models of prostate cancer drug resistance</span></div><div class="casAuthors">Salami Jemilat; Crews Craig M; Alabi Shanique; Crews Craig M; Willard Ryan R; Vitale Nick J; Wang Jing; Dong Hanqing; Jin Meizhong; Crew Andrew P; Neklesa Taavi K; McDonnell Donald P; Crews Craig M</div><div class="citationInfo"><span class="NLM_cas:title">Communications biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">100</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The androgen receptor is a major driver of prostate cancer and inhibition of its transcriptional activity using competitive antagonists, such as enzalutamide remains a frontline therapy for prostate cancer management.  However, the majority of patients eventually develop drug resistance.  We propose that targeting the androgen receptor for degradation via Proteolysis Targeting Chimeras (PROTACs) will be a better therapeutic strategy for targeting androgen receptor signaling in prostate cancer cells.  Here we perform a head-to-head comparison between a currently approved androgen receptor antagonist enzalutamide, and its PROTAC derivative, ARCC-4, across different cellular models of prostate cancer drug resistance.  ARCC-4 is a low-nanomolar androgen receptor degrader able to degrade about 95% of cellular androgen receptors.  ARCC-4 inhibits prostate tumor cell proliferation, degrades clinically relevant androgen receptor point mutants and unlike enzalutamide, retains antiproliferative effect in a high androgen environment.  Thus, ARCC-4 exemplifies how protein degradation can address the drug resistance hurdles of enzalutamide.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQilKiHIro9_JI8NvIb08rbfW6udTcc2eaO4MevQdVL8bntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3czksVCksA%253D%253D&md5=b8b7a81226dcf37d9ff430244b21f952</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1038%2Fs42003-018-0105-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs42003-018-0105-8%26sid%3Dliteratum%253Aachs%26aulast%3DSalami%26aufirst%3DJ.%26aulast%3DAlabi%26aufirst%3DS.%26aulast%3DWillard%26aufirst%3DR.%2BR.%26aulast%3DVitale%26aufirst%3DN.%2BJ.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DDong%26aufirst%3DH.%26aulast%3DJin%26aufirst%3DM.%26aulast%3DMcDonnell%26aufirst%3DD.%2BP.%26aulast%3DCrew%26aufirst%3DA.%2BP.%26aulast%3DNeklesa%26aufirst%3DT.%2BK.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DAndrogen%2520receptor%2520degradation%2520by%2520the%2520proteolysis-targeting%2520chimera%2520ARCC-4%2520outperforms%2520enzalutamide%2520in%2520cellular%2520models%2520of%2520prostate%2520cancer%2520drug%2520resistance%26jtitle%3DCommun.%2520Biol.%26date%3D2018%26volume%3D1%26spage%3D100%26doi%3D10.1038%2Fs42003-018-0105-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kregel, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiao, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fernandez-Salas, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bawa, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCollum, B. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilder-Romans, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Apel, I. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Speers, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chinnaiyan, A. M.</span></span> <span> </span><span class="NLM_article-title">Androgen receptor degraders overcome common resistance mechanisms developed during prostate cancer treatment</span>. <i>Neoplasia</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">111</span>– <span class="NLM_lpage">119</span>, <span class="refDoi"> DOI: 10.1016/j.neo.2019.12.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=10.1016%2Fj.neo.2019.12.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=31931431" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=1%3ACAS%3A528%3ADC%252BB3cXht1Ogsbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2020&pages=111-119&author=S.+Kregelauthor=C.+Wangauthor=X.+Hanauthor=L.+Xiaoauthor=E.+Fernandez-Salasauthor=P.+Bawaauthor=B.+L.+McCollumauthor=K.+Wilder-Romansauthor=I.+J.+Apelauthor=X.+Caoauthor=C.+Speersauthor=S.+Wangauthor=A.+M.+Chinnaiyan&title=Androgen+receptor+degraders+overcome+common+resistance+mechanisms+developed+during+prostate+cancer+treatment&doi=10.1016%2Fj.neo.2019.12.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Androgen receptor degraders overcome common resistance mechanisms developed during prostate cancer treatment</span></div><div class="casAuthors">Kregel, Steven; Wang, Chao; Han, Xin; Xiao, Lanbo; Fernandez-Salas, Ester; Bawa, Pushpinder; McCollum, Brooke L.; Wilder-Romans, Kari; Apel, Ingrid J.; Cao, Xuhong; Speers, Corey; Wang, Shaomeng; Chinnaiyan, Arul M.</div><div class="citationInfo"><span class="NLM_cas:title">Neoplasia (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">111-119</span>CODEN:
                <span class="NLM_cas:coden">NEOPFL</span>;
        ISSN:<span class="NLM_cas:issn">1476-5586</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">Androgen receptor (AR) antagonists, such as enzalutamide, have had a major impact on the treatment of metastatic castration-resistant prostate cancer (CRPC).  However, even with the advent of AR antagonist therapies, patients continue to develop resistance, and new strategies to combat continued AR signalling are needed.  Here, we develop AR degraders using PROteolysis TArgeting Chimeric (PROTAC) technol. in order to det. whether depletion of AR protein can overcome mechanisms of resistance commonly assocd. with current AR-targeting therapies.  ARD-61 is the most potent of the AR degraders and effectively induces on-target AR degrdn. with a mechanism consistent with the PROTAC design.  Compared to clin.-approved AR antagonists, administration of ARD-61 in vitro and in vivo results in more potent anti-proliferative, pro-apoptotic effects and attenuation of downstream AR target gene expression in prostate cancer cells.  Importantly, we demonstrate that ARD-61 functions in enzalutamide-resistant model systems, characterized by diverse proposed mechanisms of resistance that include AR amplification/overexpression, AR mutation, and expression of AR splice variants, such as AR-V7.  While AR degraders are unable to bind and degrade AR-V7, they continue to inhibit tumor cell growth in models overexpressing AR-V7.  To further explore this, we developed several isogenic prostate cell line models in which AR-V7 is highly expressed, which also failed to influence the cell inhibitory effects of AR degraders, suggesting that AR-V7 is not a functional resistance mechanism for AR antagonism.  These data provide compelling evidence that full-length AR remains a prominent oncogenic driver of prostate cancers which have developed resistance to AR antagonists and highlight the clin. potential of AR degraders for treatment of CRPC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoPKragCvCdqrVg90H21EOLACvtfcHk0litS6oUU2eDpg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXht1Ogsbc%253D&md5=dbc33b022029430baaf2a8b6a85b886f</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1016%2Fj.neo.2019.12.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.neo.2019.12.003%26sid%3Dliteratum%253Aachs%26aulast%3DKregel%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DC.%26aulast%3DHan%26aufirst%3DX.%26aulast%3DXiao%26aufirst%3DL.%26aulast%3DFernandez-Salas%26aufirst%3DE.%26aulast%3DBawa%26aufirst%3DP.%26aulast%3DMcCollum%26aufirst%3DB.%2BL.%26aulast%3DWilder-Romans%26aufirst%3DK.%26aulast%3DApel%26aufirst%3DI.%2BJ.%26aulast%3DCao%26aufirst%3DX.%26aulast%3DSpeers%26aufirst%3DC.%26aulast%3DWang%26aufirst%3DS.%26aulast%3DChinnaiyan%26aufirst%3DA.%2BM.%26atitle%3DAndrogen%2520receptor%2520degraders%2520overcome%2520common%2520resistance%2520mechanisms%2520developed%2520during%2520prostate%2520cancer%2520treatment%26jtitle%3DNeoplasia%26date%3D2020%26volume%3D22%26spage%3D111%26epage%3D119%26doi%3D10.1016%2Fj.neo.2019.12.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Salami, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Waste disposal-An attractive strategy for cancer therapy</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>355</i></span>,  <span class="NLM_fpage">1163</span>– <span class="NLM_lpage">1167</span>, <span class="refDoi"> DOI: 10.1126/science.aam7340</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=10.1126%2Fscience.aam7340" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=28302825" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=1%3ACAS%3A528%3ADC%252BC2sXktFOmt74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=355&publication_year=2017&pages=1163-1167&author=J.+Salamiauthor=C.+M.+Crews&title=Waste+disposal-An+attractive+strategy+for+cancer+therapy&doi=10.1126%2Fscience.aam7340"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Waste disposal-An attractive strategy for cancer therapy</span></div><div class="casAuthors">Salami, Jemilat; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">355</span>
        (<span class="NLM_cas:issue">6330</span>),
    <span class="NLM_cas:pages">1163-1167</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">A review.  Targeted therapies for cancer are typically small mols. or monoclonal antibodies that act by inhibiting the activity of specific proteins that drive tumor growth.  Although many of these drugs are effective in cancer patients, the response is often not durable because tumor cells develop resistance to the drugs.  Another limitation of this strategy is that not all oncogenic driver proteins are "druggable" enzymes or receptors with activities that can be inhibited.  Here the authors describe an alternative approach to targeted therapy that is based on co-opting the cellular quality-control machinery-the ubiquitin-proteasome system-to remove specific cancer-causing proteins from the cell.  The authors first discuss examples of existing cancer drugs that work by degrading specific proteins and then review recent progress in the rational design and preclin. testing of small mols. that induce selective degrdn. of specific target proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpVoZn-YpfXmbVg90H21EOLACvtfcHk0litS6oUU2eDpg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXktFOmt74%253D&md5=918621d50a82b7797aa0300537a4486e</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1126%2Fscience.aam7340&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.aam7340%26sid%3Dliteratum%253Aachs%26aulast%3DSalami%26aufirst%3DJ.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DWaste%2520disposal-An%2520attractive%2520strategy%2520for%2520cancer%2520therapy%26jtitle%3DScience%26date%3D2017%26volume%3D355%26spage%3D1163%26epage%3D1167%26doi%3D10.1126%2Fscience.aam7340" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bondeson, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mares, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, I. E. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ko, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campos, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miah, A. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mulholland, K. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Routly, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buckley, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gustafson, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zinn, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grandi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shimamura, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bergamini, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Faelth-Savitski, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bantscheff, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cox, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gordon, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Willard, R. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flanagan, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Casillas, L. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Votta, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">den Besten, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Famm, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kruidenier, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carter, P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harling, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Churcher, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Catalytic in vivo protein knockdown by small-molecule PROTACs</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">611</span>– <span class="NLM_lpage">617</span>, <span class="refDoi"> DOI: 10.1038/nchembio.1858</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=10.1038%2Fnchembio.1858" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=26075522" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtFeju7rP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2015&pages=611-617&author=D.+P.+Bondesonauthor=A.+Maresauthor=I.+E.+D.+Smithauthor=E.+Koauthor=S.+Camposauthor=A.+H.+Miahauthor=K.+E.+Mulhollandauthor=N.+Routlyauthor=D.+L.+Buckleyauthor=J.+L.+Gustafsonauthor=N.+Zinnauthor=P.+Grandiauthor=S.+Shimamuraauthor=G.+Bergaminiauthor=M.+Faelth-Savitskiauthor=M.+Bantscheffauthor=C.+Coxauthor=D.+A.+Gordonauthor=R.+R.+Willardauthor=J.+J.+Flanaganauthor=L.+N.+Casillasauthor=B.+J.+Vottaauthor=W.+den+Bestenauthor=K.+Fammauthor=L.+Kruidenierauthor=P.+S.+Carterauthor=J.+D.+Harlingauthor=I.+Churcherauthor=C.+M.+Crews&title=Catalytic+in+vivo+protein+knockdown+by+small-molecule+PROTACs&doi=10.1038%2Fnchembio.1858"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Catalytic in vivo protein knockdown by small-molecule PROTACs</span></div><div class="casAuthors">Bondeson, Daniel P.; Mares, Alina; Smith, Ian E. D.; Ko, Eunhwa; Campos, Sebastien; Miah, Afjal H.; Mulholland, Katie E.; Routly, Natasha; Buckley, Dennis L.; Gustafson, Jeffrey L.; Zinn, Nico; Grandi, Paola; Shimamura, Satoko; Bergamini, Giovanna; Faelth-Savitski, Maria; Bantscheff, Marcus; Cox, Carly; Gordon, Deborah A.; Willard, Ryan R.; Flanagan, John J.; Casillas, Linda N.; Votta, Bartholomew J.; den Besten, Willem; Famm, Kristoffer; Kruidenier, Laurens; Carter, Paul S.; Harling, John D.; Churcher, Ian; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">611-617</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The current predominant therapeutic paradigm is based on maximizing drug-receptor occupancy to achieve clin. benefit.  This strategy, however, generally requires excessive drug concns. to ensure sufficient occupancy, often leading to adverse side effects.  Here, we describe major improvements to the proteolysis targeting chimeras (PROTACs) method, a chem. knockdown strategy in which a heterobifunctional mol. recruits a specific protein target to an E3 ubiquitin ligase, resulting in the target's ubiquitination and degrdn.  These compds. behave catalytically in their ability to induce the ubiquitination of super-stoichiometric quantities of proteins, providing efficacy that is not limited by equil. occupancy.  We present two PROTACs that are capable of specifically reducing protein levels by >90% at nanomolar concns.  In addn., mouse studies indicate that they provide broad tissue distribution and knockdown of the targeted protein in tumor xenografts.  Together, these data demonstrate a protein knockdown system combining many of the favorable properties of small-mol. agents with the potent protein knockdown of RNAi and CRISPR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGonhnajD3CxS7Vg90H21EOLACvtfcHk0litS6oUU2eDpg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtFeju7rP&md5=31eb8756c5661f78ab4527a50dc1e251</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1858&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1858%26sid%3Dliteratum%253Aachs%26aulast%3DBondeson%26aufirst%3DD.%2BP.%26aulast%3DMares%26aufirst%3DA.%26aulast%3DSmith%26aufirst%3DI.%2BE.%2BD.%26aulast%3DKo%26aufirst%3DE.%26aulast%3DCampos%26aufirst%3DS.%26aulast%3DMiah%26aufirst%3DA.%2BH.%26aulast%3DMulholland%26aufirst%3DK.%2BE.%26aulast%3DRoutly%26aufirst%3DN.%26aulast%3DBuckley%26aufirst%3DD.%2BL.%26aulast%3DGustafson%26aufirst%3DJ.%2BL.%26aulast%3DZinn%26aufirst%3DN.%26aulast%3DGrandi%26aufirst%3DP.%26aulast%3DShimamura%26aufirst%3DS.%26aulast%3DBergamini%26aufirst%3DG.%26aulast%3DFaelth-Savitski%26aufirst%3DM.%26aulast%3DBantscheff%26aufirst%3DM.%26aulast%3DCox%26aufirst%3DC.%26aulast%3DGordon%26aufirst%3DD.%2BA.%26aulast%3DWillard%26aufirst%3DR.%2BR.%26aulast%3DFlanagan%26aufirst%3DJ.%2BJ.%26aulast%3DCasillas%26aufirst%3DL.%2BN.%26aulast%3DVotta%26aufirst%3DB.%2BJ.%26aulast%3Dden%2BBesten%26aufirst%3DW.%26aulast%3DFamm%26aufirst%3DK.%26aulast%3DKruidenier%26aufirst%3DL.%26aulast%3DCarter%26aufirst%3DP.%2BS.%26aulast%3DHarling%26aufirst%3DJ.%2BD.%26aulast%3DChurcher%26aufirst%3DI.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DCatalytic%2520in%2520vivo%2520protein%2520knockdown%2520by%2520small-molecule%2520PROTACs%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2015%26volume%3D11%26spage%3D611%26epage%3D617%26doi%3D10.1038%2Fnchembio.1858" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lai, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toure, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hellerschmied, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salami, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jaime-Figueroa, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ko, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hines, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Modular PROTAC design for the degradation of oncogenic BCR-ABL</span>. <i>Angew. Chem., Int. Ed. Engl.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">807</span>– <span class="NLM_lpage">810</span>, <span class="refDoi"> DOI: 10.1002/anie.201507634</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=10.1002%2Fanie.201507634" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=26593377" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=1%3ACAS%3A280%3ADC%252BC28vktlanug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2016&pages=807-810&author=A.+C.+Laiauthor=M.+Toureauthor=D.+Hellerschmiedauthor=J.+Salamiauthor=S.+Jaime-Figueroaauthor=E.+Koauthor=J.+Hinesauthor=C.+M.+Crews&title=Modular+PROTAC+design+for+the+degradation+of+oncogenic+BCR-ABL&doi=10.1002%2Fanie.201507634"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Modular PROTAC Design for the Degradation of Oncogenic BCR-ABL</span></div><div class="casAuthors">Lai Ashton C; Toure Momar; Hellerschmied Doris; Salami Jemilat; Jaime-Figueroa Saul; Ko Eunhwa; Hines John; Crews Craig M</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie (International ed. in English)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">807-10</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Proteolysis Targeting Chimera (PROTAC) technology is a rapidly emerging alternative therapeutic strategy with the potential to address many of the challenges currently faced in modern drug development programs.  PROTAC technology employs small molecules that recruit target proteins for ubiquitination and removal by the proteasome.  The synthesis of PROTAC compounds that mediate the degradation of c-ABL and BCR-ABL by recruiting either Cereblon or Von Hippel Lindau E3 ligases is reported.  During the course of their development, we discovered that the capacity of a PROTAC to induce degradation involves more than just target binding: the identity of the inhibitor warhead and the recruited E3 ligase largely determine the degradation profiles of the compounds; thus, as a starting point for PROTAC development, both the target ligand and the recruited E3 ligase should be varied to rapidly generate a PROTAC with the desired degradation profile.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSu-9RPU8iMpUodMGTatQ7LfW6udTcc2eY-TtqOJW0B2Lntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC28vktlanug%253D%253D&md5=5796ba6cff3b26c00c88ad700385f37a</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1002%2Fanie.201507634&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201507634%26sid%3Dliteratum%253Aachs%26aulast%3DLai%26aufirst%3DA.%2BC.%26aulast%3DToure%26aufirst%3DM.%26aulast%3DHellerschmied%26aufirst%3DD.%26aulast%3DSalami%26aufirst%3DJ.%26aulast%3DJaime-Figueroa%26aufirst%3DS.%26aulast%3DKo%26aufirst%3DE.%26aulast%3DHines%26aufirst%3DJ.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DModular%2520PROTAC%2520design%2520for%2520the%2520degradation%2520of%2520oncogenic%2520BCR-ABL%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%2520Engl.%26date%3D2016%26volume%3D55%26spage%3D807%26epage%3D810%26doi%3D10.1002%2Fanie.201507634" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Demizu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shibata, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hattori, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohoka, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Motoi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Misawa, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shoda, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Naito, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kurihara, M.</span></span> <span> </span><span class="NLM_article-title">Development of BCR-ABL degradation inducers via the conjugation of an imatinib derivative and a cIAP1 ligand</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">4865</span>– <span class="NLM_lpage">4869</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2016.09.041</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=10.1016%2Fj.bmcl.2016.09.041" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=27666635" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsFOntbnK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2016&pages=4865-4869&author=Y.+Demizuauthor=N.+Shibataauthor=T.+Hattoriauthor=N.+Ohokaauthor=H.+Motoiauthor=T.+Misawaauthor=T.+Shodaauthor=M.+Naitoauthor=M.+Kurihara&title=Development+of+BCR-ABL+degradation+inducers+via+the+conjugation+of+an+imatinib+derivative+and+a+cIAP1+ligand&doi=10.1016%2Fj.bmcl.2016.09.041"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Development of BCR-ABL degradation inducers via the conjugation of an imatinib derivative and a cIAP1 ligand</span></div><div class="casAuthors">Demizu, Yosuke; Shibata, Norihito; Hattori, Takayuki; Ohoka, Nobumichi; Motoi, Hiromi; Misawa, Takashi; Shoda, Takuji; Naito, Mikihiko; Kurihara, Masaaki</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">4865-4869</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The manipulation of protein stability with small mols. has great potential as a technique for aiding the development of clin. therapies, including treatments for cancer.  In this study, BCR-ABL protein degrdn. inducers called SNIPER(ABL) (Specific and Nongenetic inhibitors of apoptosis protein [IAP]-dependent Protein Erasers) were developed.  The designed mols. contained two biol. active scaffolds: one was an imatinib deriv. that binds to BCL-ABL and the other was a Me bestatin that binds to cellular IAP 1 (cIAP1).  The hybrid mols., SNIPER(ABL), were expected to recruit BCR-ABL to cIAP1 for removal by proteasomes.  In fact, SNIPER(ABL) induced the degrdn. of BCR-ABL protein and a subsequent redn. in cell growth.  Thus, the degrdn. of BCR-ABL by SNIPER(ABL) is one potential strategy for treating BCR-ABL driven chronic myelogenous leukemia.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpLT6dtmVPL47Vg90H21EOLACvtfcHk0lgtcskKrkzRPA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsFOntbnK&md5=31c045087293acf60932a9d6c4f38504</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2016.09.041&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2016.09.041%26sid%3Dliteratum%253Aachs%26aulast%3DDemizu%26aufirst%3DY.%26aulast%3DShibata%26aufirst%3DN.%26aulast%3DHattori%26aufirst%3DT.%26aulast%3DOhoka%26aufirst%3DN.%26aulast%3DMotoi%26aufirst%3DH.%26aulast%3DMisawa%26aufirst%3DT.%26aulast%3DShoda%26aufirst%3DT.%26aulast%3DNaito%26aufirst%3DM.%26aulast%3DKurihara%26aufirst%3DM.%26atitle%3DDevelopment%2520of%2520BCR-ABL%2520degradation%2520inducers%2520via%2520the%2520conjugation%2520of%2520an%2520imatinib%2520derivative%2520and%2520a%2520cIAP1%2520ligand%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2016%26volume%3D26%26spage%3D4865%26epage%3D4869%26doi%3D10.1016%2Fj.bmcl.2016.09.041" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ren, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kong, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yin, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, B.</span></span> <span> </span><span class="NLM_article-title">Discovery of SIAIS178 as an effective BCR-ABL degrader by recruiting von hippel-lindau (VHL) E3 ubiquitin ligase</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">9281</span>– <span class="NLM_lpage">9298</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b01264</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b01264" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhvVantrnN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=9281-9298&author=Q.+Zhaoauthor=C.+Renauthor=L.+Liuauthor=J.+Chenauthor=Y.+Shaoauthor=N.+Sunauthor=R.+Sunauthor=Y.+Kongauthor=X.+Dingauthor=X.+Zhangauthor=Y.+Xuauthor=B.+Yangauthor=Q.+Yinauthor=X.+Yangauthor=B.+Jiang&title=Discovery+of+SIAIS178+as+an+effective+BCR-ABL+degrader+by+recruiting+von+hippel-lindau+%28VHL%29+E3+ubiquitin+ligase&doi=10.1021%2Facs.jmedchem.9b01264"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of SIAIS178 as an Effective BCR-ABL Degrader by Recruiting Von Hippel-Lindau (VHL) E3 Ubiquitin Ligase</span></div><div class="casAuthors">Zhao, Quanju; Ren, Chaowei; Liu, Linyi; Chen, Jinju; Shao, Yubao; Sun, Ning; Sun, Renhong; Kong, Ying; Ding, Xinyu; Zhang, Xianfang; Xu, Youwei; Yang, Bei; Yin, Qianqian; Yang, Xiaobao; Jiang, Biao</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">9281-9298</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The oncogenic fusion protein BCR-ABL is the driving force of leukemogenesis in chronic myeloid leukemia (CML).  Despite great progress for CML treatment through application of tyrosine kinase inhibitors (TKIs) against BCR-ABL, long-term drug administration and clin. resistance continue to be an issue.  Herein, we described the design, synthesis, and evaluation of novel proteolysis-targeting chimeric (PROTAC) small mols. targeting BCR-ABL which connect dasatinib and VHL E3 ubiquitin ligase ligand by extensive optimization of linkers.  Our efforts have yielded SIAIS178 (19), which induces proper interaction between BCR-ABL and VHL ligase leading to effective degrdn. of BCR-ABL protein, achieves significant growth inhibition of BCR-ABL+ leukemic cells in vitro, and induces substantial tumor regression against K562 xenograft tumors in vivo.  In addn., SIAIS178 also degrades several clin. relevant resistance-conferring mutations.  Our data indicate that SIAIS178 as efficacious BCR-ABL degrader warrants extensive further investigation for the treatment of BCR-ABL+ leukemia.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrLBFfujSx6ZbVg90H21EOLACvtfcHk0lgtcskKrkzRPA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhvVantrnN&md5=50ace274dbc6ef3931f86504a1be93a7</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b01264&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b01264%26sid%3Dliteratum%253Aachs%26aulast%3DZhao%26aufirst%3DQ.%26aulast%3DRen%26aufirst%3DC.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DShao%26aufirst%3DY.%26aulast%3DSun%26aufirst%3DN.%26aulast%3DSun%26aufirst%3DR.%26aulast%3DKong%26aufirst%3DY.%26aulast%3DDing%26aufirst%3DX.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DXu%26aufirst%3DY.%26aulast%3DYang%26aufirst%3DB.%26aulast%3DYin%26aufirst%3DQ.%26aulast%3DYang%26aufirst%3DX.%26aulast%3DJiang%26aufirst%3DB.%26atitle%3DDiscovery%2520of%2520SIAIS178%2520as%2520an%2520effective%2520BCR-ABL%2520degrader%2520by%2520recruiting%2520von%2520hippel-lindau%2520%2528VHL%2529%2520E3%2520ubiquitin%2520ligase%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D9281%26epage%3D9298%26doi%3D10.1021%2Facs.jmedchem.9b01264" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Robb, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Contreras, J. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kour, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abid, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sonawane, Y. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zahid, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murry, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Natarajan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rana, S.</span></span> <span> </span><span class="NLM_article-title">Chemically induced degradation of CDK9 by a proteolysis targeting chimera (PROTAC)</span>. <i>Chem. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">7577</span>– <span class="NLM_lpage">7580</span>, <span class="refDoi"> DOI: 10.1039/c7cc03879h</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=10.1039%2Fc7cc03879h" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=28636052" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=1%3ACAS%3A528%3ADC%252BC2sXpvFyjsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2017&pages=7577-7580&author=C.+M.+Robbauthor=J.+I.+Contrerasauthor=S.+Kourauthor=M.+A.+Taylorauthor=M.+Abidauthor=Y.+A.+Sonawaneauthor=M.+Zahidauthor=D.+J.+Murryauthor=A.+Natarajanauthor=S.+Rana&title=Chemically+induced+degradation+of+CDK9+by+a+proteolysis+targeting+chimera+%28PROTAC%29&doi=10.1039%2Fc7cc03879h"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Chemically induced degradation of CDK9 by a proteolysis targeting chimera (PROTAC)</span></div><div class="casAuthors">Robb, Caroline M.; Contreras, Jacob I.; Kour, Smit; Taylor, Margaret A.; Abid, Mohammad; Sonawane, Yogesh A.; Zahid, Muhammad; Murry, Daryl J.; Natarajan, Amarnath; Rana, Sandeep</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Communications (Cambridge, United Kingdom)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">54</span>),
    <span class="NLM_cas:pages">7577-7580</span>CODEN:
                <span class="NLM_cas:coden">CHCOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-7345</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Cyclin-dependent kinase 9 (CDK9), a member of the cyclin-dependent protein kinase (CDK) family, is involved in transcriptional elongation of several target genes.  CDK9 is ubiquitously expressed and has been shown to contribute to a variety of malignancies such as pancreatic, prostate and breast cancers.  Here we report the development of a heterobifunctional small mol. proteolysis targeting chimera (PROTAC) capable of cereblon (CRBN)-mediated proteasomal degrdn. of CDK9.  In HCT116 cells, it selectively degrades CDK9 while sparing other CDK family members.  This is the first example of a PROTAC that selectively degrades CDK9.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpXgLtOTwvlnbVg90H21EOLACvtfcHk0lhGnzIUyrr2qg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXpvFyjsLw%253D&md5=a9aa8ebf8109f11206740efd1b5e09a2</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1039%2Fc7cc03879h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc7cc03879h%26sid%3Dliteratum%253Aachs%26aulast%3DRobb%26aufirst%3DC.%2BM.%26aulast%3DContreras%26aufirst%3DJ.%2BI.%26aulast%3DKour%26aufirst%3DS.%26aulast%3DTaylor%26aufirst%3DM.%2BA.%26aulast%3DAbid%26aufirst%3DM.%26aulast%3DSonawane%26aufirst%3DY.%2BA.%26aulast%3DZahid%26aufirst%3DM.%26aulast%3DMurry%26aufirst%3DD.%2BJ.%26aulast%3DNatarajan%26aufirst%3DA.%26aulast%3DRana%26aufirst%3DS.%26atitle%3DChemically%2520induced%2520degradation%2520of%2520CDK9%2520by%2520a%2520proteolysis%2520targeting%2520chimera%2520%2528PROTAC%2529%26jtitle%3DChem.%2520Commun.%26date%3D2017%26volume%3D53%26spage%3D7577%26epage%3D7580%26doi%3D10.1039%2Fc7cc03879h" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Olson, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Erb, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doctor, Z. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kwiatkowski, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boukhali, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Green, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haas, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nomanbhoy, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fischer, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winter, G. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span> <span> </span><span class="NLM_article-title">Pharmacological perturbation of CDK9 using selective CDK9 inhibition or degradation</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">163</span>– <span class="NLM_lpage">170</span>, <span class="refDoi"> DOI: 10.1038/nchembio.2538</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=10.1038%2Fnchembio.2538" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=29251720" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2017&pages=163-170&author=C.+M.+Olsonauthor=B.+Jiangauthor=M.+A.+Erbauthor=Y.+Liangauthor=Z.+M.+Doctorauthor=Z.+Zhangauthor=T.+Zhangauthor=N.+Kwiatkowskiauthor=M.+Boukhaliauthor=J.+L.+Greenauthor=W.+Haasauthor=T.+Nomanbhoyauthor=E.+S.+Fischerauthor=R.+A.+Youngauthor=J.+E.+Bradnerauthor=G.+E.+Winterauthor=N.+S.+Gray&title=Pharmacological+perturbation+of+CDK9+using+selective+CDK9+inhibition+or+degradation&doi=10.1038%2Fnchembio.2538"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.2538&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.2538%26sid%3Dliteratum%253Aachs%26aulast%3DOlson%26aufirst%3DC.%2BM.%26aulast%3DJiang%26aufirst%3DB.%26aulast%3DErb%26aufirst%3DM.%2BA.%26aulast%3DLiang%26aufirst%3DY.%26aulast%3DDoctor%26aufirst%3DZ.%2BM.%26aulast%3DZhang%26aufirst%3DZ.%26aulast%3DZhang%26aufirst%3DT.%26aulast%3DKwiatkowski%26aufirst%3DN.%26aulast%3DBoukhali%26aufirst%3DM.%26aulast%3DGreen%26aufirst%3DJ.%2BL.%26aulast%3DHaas%26aufirst%3DW.%26aulast%3DNomanbhoy%26aufirst%3DT.%26aulast%3DFischer%26aufirst%3DE.%2BS.%26aulast%3DYoung%26aufirst%3DR.%2BA.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26aulast%3DWinter%26aufirst%3DG.%2BE.%26aulast%3DGray%26aufirst%3DN.%2BS.%26atitle%3DPharmacological%2520perturbation%2520of%2520CDK9%2520using%2520selective%2520CDK9%2520inhibition%2520or%2520degradation%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2017%26volume%3D14%26spage%3D163%26epage%3D170%26doi%3D10.1038%2Fnchembio.2538" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huang, H.-T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dobrovolsky, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paulk, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weisberg, E. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doctor, Z. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buckley, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cho, J.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ko, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choi, H. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Griffin, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Treon, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fischer, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span> <span> </span><span class="NLM_article-title">A chemoproteomic approach to query the degradable kinome using a multi-kinase degrader</span>. <i>Cell Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">88</span>– <span class="NLM_lpage">99.e6</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2017.10.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=10.1016%2Fj.chembiol.2017.10.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=29129717" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsl2lsrjO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2018&pages=88-99.e6&author=H.-T.+Huangauthor=D.+Dobrovolskyauthor=J.+Paulkauthor=G.+Yangauthor=E.+L.+Weisbergauthor=Z.+M.+Doctorauthor=D.+L.+Buckleyauthor=J.-H.+Choauthor=E.+Koauthor=J.+Jangauthor=K.+Shiauthor=H.+G.+Choiauthor=J.+D.+Griffinauthor=Y.+Liauthor=S.+P.+Treonauthor=E.+S.+Fischerauthor=J.+E.+Bradnerauthor=L.+Tanauthor=N.+S.+Gray&title=A+chemoproteomic+approach+to+query+the+degradable+kinome+using+a+multi-kinase+degrader&doi=10.1016%2Fj.chembiol.2017.10.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">A Chemoproteomic Approach to Query the Degradable Kinome Using a Multi-kinase Degrader</span></div><div class="casAuthors">Huang, Hai-Tsang; Dobrovolsky, Dennis; Paulk, Joshiawa; Yang, Guang; Weisberg, Ellen L.; Doctor, Zainab M.; Buckley, Dennis L.; Cho, Joong-Heui; Ko, Eunhwa; Jang, Jaebong; Shi, Kun; Choi, Hwan Geun; Griffin, James D.; Li, Ying; Treon, Steven P.; Fischer, Eric S.; Bradner, James E.; Tan, Li; Gray, Nathanael S.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">88-99.e6</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Heterobifunctional mols. that recruit E3 ubiquitin ligases, such as cereblon, for targeted protein degrdn. represent an emerging pharmacol. strategy.  A major unanswered question is how generally applicable this strategy is to all protein targets.  In this study, we designed a multi-kinase degrader by conjugating a highly promiscuous kinase inhibitor with a cereblon-binding ligand, and used quant. proteomics to discover 28 kinases, including BTK, PTK2, PTK2B, FLT3, AURKA, AURKB, TEC, ULK1, ITK, and nine members of the CDK family, as degradable.  This set of kinases is only a fraction of the intracellular targets bound by the degrader, demonstrating that successful degrdn. requires more than target engagement.  The results guided us to develop selective degraders for FLT3 and BTK, with potentials to improve disease treatment.  Together, this study demonstrates an efficient approach to triage a gene family of interest to identify readily degradable targets for further studies and pre-clin. developments.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrYNxhoPV3127Vg90H21EOLACvtfcHk0lhGnzIUyrr2qg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsl2lsrjO&md5=703cfbaca3edd24176472158563f895f</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2017.10.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2017.10.005%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DH.-T.%26aulast%3DDobrovolsky%26aufirst%3DD.%26aulast%3DPaulk%26aufirst%3DJ.%26aulast%3DYang%26aufirst%3DG.%26aulast%3DWeisberg%26aufirst%3DE.%2BL.%26aulast%3DDoctor%26aufirst%3DZ.%2BM.%26aulast%3DBuckley%26aufirst%3DD.%2BL.%26aulast%3DCho%26aufirst%3DJ.-H.%26aulast%3DKo%26aufirst%3DE.%26aulast%3DJang%26aufirst%3DJ.%26aulast%3DShi%26aufirst%3DK.%26aulast%3DChoi%26aufirst%3DH.%2BG.%26aulast%3DGriffin%26aufirst%3DJ.%2BD.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DTreon%26aufirst%3DS.%2BP.%26aulast%3DFischer%26aufirst%3DE.%2BS.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26aulast%3DTan%26aufirst%3DL.%26aulast%3DGray%26aufirst%3DN.%2BS.%26atitle%3DA%2520chemoproteomic%2520approach%2520to%2520query%2520the%2520degradable%2520kinome%2520using%2520a%2520multi-kinase%2520degrader%26jtitle%3DCell%2520Chem.%2520Biol.%26date%3D2018%26volume%3D25%26spage%3D88%26epage%3D99.e6%26doi%3D10.1016%2Fj.chembiol.2017.10.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Burslem, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, B. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jaime-Figueroa, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McQuaid, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bondeson, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toure, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qian, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crew, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hines, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">The advantages of targeted protein degradation over inhibition: an RTK case study</span>. <i>Cell Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">67</span>– <span class="NLM_lpage">77.e3</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2017.09.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=10.1016%2Fj.chembiol.2017.09.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=29129716" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsl2lsrjM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2018&pages=67-77.e3&author=G.+M.+Burslemauthor=B.+E.+Smithauthor=A.+C.+Laiauthor=S.+Jaime-Figueroaauthor=D.+C.+McQuaidauthor=D.+P.+Bondesonauthor=M.+Toureauthor=H.+Dongauthor=Y.+Qianauthor=J.+Wangauthor=A.+P.+Crewauthor=J.+Hinesauthor=C.+M.+Crews&title=The+advantages+of+targeted+protein+degradation+over+inhibition%3A+an+RTK+case+study&doi=10.1016%2Fj.chembiol.2017.09.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">The Advantages of Targeted Protein Degradation Over Inhibition: An RTK Case Study</span></div><div class="casAuthors">Burslem, George M.; Smith, Blake E.; Lai, Ashton C.; Jaime-Figueroa, Saul; McQuaid, Daniel C.; Bondeson, Daniel P.; Toure, Momar; Dong, Hanqing; Qian, Yimin; Wang, Jing; Crew, Andrew P.; Hines, John; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">67-77.e3</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Proteolysis targeting chimera (PROTAC) technol. has emerged over the last two decades as a powerful tool for targeted degrdn. of endogenous proteins.  Herein we describe the development of PROTACs for receptor tyrosine kinases, a protein family yet to be targeted for induced protein degrdn.  The use of VHL-recruiting PROTACs against this protein family reveals several advantages of degrdn. over inhibition alone: direct comparisons of fully functional, target-degrading PROTACs with target-inhibiting variants that contain an inactivated E3 ligase-recruiting ligand show that degrdn. leads to more potent inhibition of cell proliferation and a more durable and sustained downstream signaling response, and thus addresses the kinome rewiring challenge seen with many receptor tyrosine kinase inhibitors.  Combined, these findings demonstrate the ability to target receptor tyrosine kinases for degrdn. using the PROTAC technol. and outline the advantages of this degrdn.-based approach.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr4yDhRDWqNv7Vg90H21EOLACvtfcHk0lifp4HdbInnjw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsl2lsrjM&md5=0ac39a6f5836ea3772a0396356efb630</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2017.09.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2017.09.009%26sid%3Dliteratum%253Aachs%26aulast%3DBurslem%26aufirst%3DG.%2BM.%26aulast%3DSmith%26aufirst%3DB.%2BE.%26aulast%3DLai%26aufirst%3DA.%2BC.%26aulast%3DJaime-Figueroa%26aufirst%3DS.%26aulast%3DMcQuaid%26aufirst%3DD.%2BC.%26aulast%3DBondeson%26aufirst%3DD.%2BP.%26aulast%3DToure%26aufirst%3DM.%26aulast%3DDong%26aufirst%3DH.%26aulast%3DQian%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DCrew%26aufirst%3DA.%2BP.%26aulast%3DHines%26aufirst%3DJ.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DThe%2520advantages%2520of%2520targeted%2520protein%2520degradation%2520over%2520inhibition%253A%2520an%2520RTK%2520case%2520study%26jtitle%3DCell%2520Chem.%2520Biol.%26date%3D2018%26volume%3D25%26spage%3D67%26epage%3D77.e3%26doi%3D10.1016%2Fj.chembiol.2017.09.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meng, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiong, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span> <span> </span><span class="NLM_article-title">Discovery of potent and selective epidermal growth factor receptor (EGFR) bifunctional small-molecule degraders</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">1216</span>– <span class="NLM_lpage">1232</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b01566</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b01566" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=1%3ACAS%3A528%3ADC%252BB3cXisFWksA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=1216-1232&author=M.+Chengauthor=X.+Yuauthor=K.+Luauthor=L.+Xieauthor=L.+Wangauthor=F.+Mengauthor=X.+Hanauthor=X.+Chenauthor=J.+Liuauthor=Y.+Xiongauthor=J.+Jin&title=Discovery+of+potent+and+selective+epidermal+growth+factor+receptor+%28EGFR%29+bifunctional+small-molecule+degraders&doi=10.1021%2Facs.jmedchem.9b01566"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Potent and Selective Epidermal Growth Factor Receptor (EGFR) Bifunctional Small-Molecule Degraders</span></div><div class="casAuthors">Cheng, Meng; Yu, Xufen; Lu, Kaylene; Xie, Ling; Wang, Li; Meng, Fanye; Han, Xiaoran; Chen, Xian; Liu, Jing; Xiong, Yue; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1216-1232</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Several epidermal growth factor receptor (EGFR) tyrosine kinase inhibitors have been developed and approved by Food and Drug Administration for the treatment of non-small-cell lung cancers, but their efficacy can be compromised by acquired drug resistance conferred by EGFR-mutant variants.  Here, we described the discovery of a novel E3 ligase von Hippel-Lindau-recruiting EGFR degrader, MS39 (compd. 6), and a first-in-class E3 ligase cereblon-recruiting EGFR degrader, MS154 (compd. 10), using the proteolysis targeting chimera technol.  These compds. potently induced the degrdn. of mutant but not wild-type EGFR in an E3 ligase-dependent manner in cancer cell lines and effectively suppressed the growth of lung cancer cells compared with the corresponding neg. controls.  The global proteomic analyses revealed that the compds. were highly selective for EGFR.  Furthermore, both compds. were bioavailable in mouse pharmacokinetic studies, and compd. 6 is the first EGFR degrader suitable for in vivo efficacy studies.  Overall, we provide a set of well-characterized chem. tools to the research community.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGraFLSjUGE-6rVg90H21EOLACvtfcHk0lifp4HdbInnjw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXisFWksA%253D%253D&md5=ec831897a4258f1968e254aa433426c3</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b01566&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b01566%26sid%3Dliteratum%253Aachs%26aulast%3DCheng%26aufirst%3DM.%26aulast%3DYu%26aufirst%3DX.%26aulast%3DLu%26aufirst%3DK.%26aulast%3DXie%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DMeng%26aufirst%3DF.%26aulast%3DHan%26aufirst%3DX.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DXiong%26aufirst%3DY.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DDiscovery%2520of%2520potent%2520and%2520selective%2520epidermal%2520growth%2520factor%2520receptor%2520%2528EGFR%2529%2520bifunctional%2520small-molecule%2520degraders%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D1216%26epage%3D1232%26doi%3D10.1021%2Facs.jmedchem.9b01566" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, X.-R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiong, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span> <span> </span><span class="NLM_article-title">Proteolysis targeting chimeras (PROTACs) of anaplastic lymphoma kinase (ALK)</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>151</i></span>,  <span class="NLM_fpage">304</span>– <span class="NLM_lpage">314</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2018.03.071</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=10.1016%2Fj.ejmech.2018.03.071" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=29627725" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=1%3ACAS%3A528%3ADC%252BC1cXnt1ejs7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=151&publication_year=2018&pages=304-314&author=C.+Zhangauthor=X.-R.+Hanauthor=X.+Yangauthor=B.+Jiangauthor=J.+Liuauthor=Y.+Xiongauthor=J.+Jin&title=Proteolysis+targeting+chimeras+%28PROTACs%29+of+anaplastic+lymphoma+kinase+%28ALK%29&doi=10.1016%2Fj.ejmech.2018.03.071"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Proteolysis Targeting Chimeras (PROTACs) of Anaplastic Lymphoma Kinase (ALK)</span></div><div class="casAuthors">Zhang, Chengwei; Han, Xiao-Ran; Yang, Xiaobao; Jiang, Biao; Liu, Jing; Xiong, Yue; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">151</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">304-314</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">Anaplastic lymphoma kinase (ALK) activation has been assocd. with many types of human cancer.  Significant efforts have been devoted to the development of ALK inhibitors to antagonize the kinase activity of ALK.  Four ALK inhibitors have been approved by the FDA to date for treating patients with ALK-pos. nonsmall cell lung cancers (NSCLC).  However, drug resistance has been obsd. in the majority of patients treated with these inhibitors.  New therapeutic strategies (e.g., compds. with novel mechanisms of action) are needed to overcome the drug resistance issue.  The emerging PROTAC (Proteolysis Targeting Chimera) technol. has been successfully applied to selective degrdn. of multiple protein targets, but not ALK.  Since ALK protein levels are not important for viability in mammals, ALK PROTACs could lead to novel therapeutics with minimal toxicity.  Here the authors report the design, synthesis and biol. evaluation of novel PROTACs (degraders) of ALK.  MS4077 (I) and MS4078 (II) potently decreased cellular levels of oncogenic active ALK fusion proteins in a concn.- and time-dependent manner in SU-DHL-1 lymphoma and NCI-H2228 lung cancer cells.  The ALK protein degrdn. induced by compds. I and II was cereblon and proteasome dependent.  In addn., compds. I and II potently inhibited proliferation of SU-DHL-1 cells.  Furthermore, compd. II displayed good plasma exposure in a mouse pharmacokinetic study, thus is suitable for in vivo efficacy studies.  The authors also developed MS4748 and MS4740 , very close analogs of I and II resp., which are incapable to degrade the ALK fusion proteins, as neg. controls.  All 4 compds. are valuable chem. tools for investigating effects of ALK pharmacol. degrdn.  The study paved the way for developing the next generation of ALK PROTACs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPGM3aAZZ6h7Vg90H21EOLACvtfcHk0lifp4HdbInnjw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXnt1ejs7Y%253D&md5=804324b95034f38b46e45f89e18e55be</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2018.03.071&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2018.03.071%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DC.%26aulast%3DHan%26aufirst%3DX.-R.%26aulast%3DYang%26aufirst%3DX.%26aulast%3DJiang%26aufirst%3DB.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DXiong%26aufirst%3DY.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DProteolysis%2520targeting%2520chimeras%2520%2528PROTACs%2529%2520of%2520anaplastic%2520lymphoma%2520kinase%2520%2528ALK%2529%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2018%26volume%3D151%26spage%3D304%26epage%3D314%26doi%3D10.1016%2Fj.ejmech.2018.03.071" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Powell, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Donovan, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nowak, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loehr, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bahcall, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fischer, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jänne, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">George, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span> <span> </span><span class="NLM_article-title">Chemically induced degradation of anaplastic lymphoma kinase (ALK)</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">4249</span>– <span class="NLM_lpage">4255</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b01655</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b01655" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=1%3ACAS%3A528%3ADC%252BC1cXns1OitrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=4249-4255&author=C.+E.+Powellauthor=Y.+Gaoauthor=L.+Tanauthor=K.+A.+Donovanauthor=R.+P.+Nowakauthor=A.+Loehrauthor=M.+Bahcallauthor=E.+S.+Fischerauthor=P.+A.+J%C3%A4nneauthor=R.+E.+Georgeauthor=N.+S.+Gray&title=Chemically+induced+degradation+of+anaplastic+lymphoma+kinase+%28ALK%29&doi=10.1021%2Facs.jmedchem.7b01655"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Chemically Induced Degradation of Anaplastic Lymphoma Kinase (ALK)</span></div><div class="casAuthors">Powell, Chelsea E.; Gao, Yang; Tan, Li; Donovan, Katherine A.; Nowak, Radoslaw P.; Loehr, Amanda; Bahcall, Magda; Fischer, Eric S.; Janne, Pasi A.; George, Rani E.; Gray, Nathanael S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">4249-4255</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We present the development of the first small mol. degraders that can induce anaplastic lymphoma kinase (ALK) degrdn., including in non-small-cell lung cancer (NSCLC), anaplastic large-cell lymphoma (ALCL), and neuroblastoma (NB) cell lines.  These degraders were developed through conjugation of known pyrimidine-based ALK inhibitors, TAE684 or LDK378, and the cereblon ligand pomalidomide.  We demonstrate that in some cell types degrader potency is compromised by expression of drug transporter ABCB1.  In addn., proteomic profiling demonstrated that these compds. also promote the degrdn. of addnl. kinases including PTK2 (FAK), Aurora A, FER, and RPS6KA1 (RSK1).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp7mDH7Sj8EqbVg90H21EOLACvtfcHk0lgDWc3375u8hQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXns1OitrY%253D&md5=486db11b3fbc2283700f38b09cc39bd7</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b01655&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b01655%26sid%3Dliteratum%253Aachs%26aulast%3DPowell%26aufirst%3DC.%2BE.%26aulast%3DGao%26aufirst%3DY.%26aulast%3DTan%26aufirst%3DL.%26aulast%3DDonovan%26aufirst%3DK.%2BA.%26aulast%3DNowak%26aufirst%3DR.%2BP.%26aulast%3DLoehr%26aufirst%3DA.%26aulast%3DBahcall%26aufirst%3DM.%26aulast%3DFischer%26aufirst%3DE.%2BS.%26aulast%3DJ%25C3%25A4nne%26aufirst%3DP.%2BA.%26aulast%3DGeorge%26aufirst%3DR.%2BE.%26aulast%3DGray%26aufirst%3DN.%2BS.%26atitle%3DChemically%2520induced%2520degradation%2520of%2520anaplastic%2520lymphoma%2520kinase%2520%2528ALK%2529%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D4249%26epage%3D4255%26doi%3D10.1021%2Facs.jmedchem.7b01655" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sun, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ren, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kong, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhong, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qiu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, B.</span></span> <span> </span><span class="NLM_article-title">Development of a Brigatinib degrader (SIAIS117) as a potential treatment for ALK positive cancer resistance</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>193</i></span>,  <span class="NLM_fpage">112190</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2020.112190</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=10.1016%2Fj.ejmech.2020.112190" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=32179332" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=1%3ACAS%3A528%3ADC%252BB3cXkvF2qu70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=193&publication_year=2020&pages=112190&author=N.+Sunauthor=C.+Renauthor=Y.+Kongauthor=H.+Zhongauthor=J.+Chenauthor=Y.+Liauthor=J.+Zhangauthor=Y.+Zhouauthor=X.+Qiuauthor=H.+Linauthor=X.+Songauthor=X.+Yangauthor=B.+Jiang&title=Development+of+a+Brigatinib+degrader+%28SIAIS117%29+as+a+potential+treatment+for+ALK+positive+cancer+resistance&doi=10.1016%2Fj.ejmech.2020.112190"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Development of a Brigatinib degrader (SIAIS117) as a potential treatment for ALK positive cancer resistance</span></div><div class="casAuthors">Sun, Ning; Ren, Chaowei; Kong, Ying; Zhong, Hui; Chen, Jinju; Li, Yan; Zhang, Jianshui; Zhou, Yuedong; Qiu, Xing; Lin, Haifan; Song, Xiaoling; Yang, Xiaobao; Jiang, Biao</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">193</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">112190</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">EML4-ALK and NPM-ALK fusion proteins possess constitutively activated ALK (anaplastic lymphoma kinase) activity, which in turn leads to the development of non-small cell lung cancer and anaplastic large-cell lymphomas (ALCLs).  FDA-approved ALK inhibitor drugs cause significant cancer regression.  However, drug resistance eventually occurs and it becomes a big obstacle in clinic.  Novel proteolysis targeting chimera (PROTAC) technol. platform provides a potential therapeutic strategy for drug resistance.  Herein, we designed and synthesized a series of ALK PROTACs based on Brigatinib and VHL-1 conjunction, and screened SIAIS117 as the best degrader which not only blocked the growth of SR and H2228 cancer cell lines, but also degraded ALK protein.  In addn., SIAIS117 also showed much better growth inhibition effect than Brigatinib on 293T cell line that exogenously expressed G1202R-resistant ALK proteins.  Furthermore, it also degraded G1202R mutant ALK protein in vitro.  At last, it has the potentially anti-proliferation ability of small cell lung cancer.  Thus, we have successfully generated the degrader SIAIS117 that can potentially overcome resistance in cancer targeted therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrX9rzk88HMe7Vg90H21EOLACvtfcHk0lgDWc3375u8hQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXkvF2qu70%253D&md5=61f0228111c806ae62473fc450fce276</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2020.112190&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2020.112190%26sid%3Dliteratum%253Aachs%26aulast%3DSun%26aufirst%3DN.%26aulast%3DRen%26aufirst%3DC.%26aulast%3DKong%26aufirst%3DY.%26aulast%3DZhong%26aufirst%3DH.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DZhou%26aufirst%3DY.%26aulast%3DQiu%26aufirst%3DX.%26aulast%3DLin%26aufirst%3DH.%26aulast%3DSong%26aufirst%3DX.%26aulast%3DYang%26aufirst%3DX.%26aulast%3DJiang%26aufirst%3DB.%26atitle%3DDevelopment%2520of%2520a%2520Brigatinib%2520degrader%2520%2528SIAIS117%2529%2520as%2520a%2520potential%2520treatment%2520for%2520ALK%2520positive%2520cancer%2520resistance%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2020%26volume%3D193%26spage%3D112190%26doi%3D10.1016%2Fj.ejmech.2020.112190" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brand, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bauer, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Donovan, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nowak, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kwiatkowski, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fischer, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winter, G. E.</span></span> <span> </span><span class="NLM_article-title">Homolog-selective degradation as a strategy to probe the function of CDK6 in AML</span>. <i>Cell Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">300</span>– <span class="NLM_lpage">306.e9</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2018.11.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=10.1016%2Fj.chembiol.2018.11.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=30595531" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=1%3ACAS%3A528%3ADC%252BC1MXnsVSj" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2019&pages=300-306.e9&author=M.+Brandauthor=B.+Jiangauthor=S.+Bauerauthor=K.+A.+Donovanauthor=Y.+Liangauthor=E.+S.+Wangauthor=R.+P.+Nowakauthor=J.+C.+Yuanauthor=T.+Zhangauthor=N.+Kwiatkowskiauthor=A.+C.+M%C3%BCllerauthor=E.+S.+Fischerauthor=N.+S.+Grayauthor=G.+E.+Winter&title=Homolog-selective+degradation+as+a+strategy+to+probe+the+function+of+CDK6+in+AML&doi=10.1016%2Fj.chembiol.2018.11.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Homolog-Selective Degradation as a Strategy to Probe the Function of CDK6 in AML</span></div><div class="casAuthors">Brand, Matthias; Jiang, Baishan; Bauer, Sophie; Donovan, Katherine A.; Liang, Yanke; Wang, Eric S.; Nowak, Radoslaw P.; Yuan, Jingting C.; Zhang, Tinghu; Kwiatkowski, Nicholas; Mueller, Andre C.; Fischer, Eric S.; Gray, Nathanael S.; Winter, Georg E.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">300-306.e9</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The design of selective small mols. is often stymied by similar ligand binding pockets.  Here, we report BSJ-03-123, a phthalimide-based degrader that exploits protein-interface determinants to achieve proteome-wide selectivity for the degrdn. of cyclin-dependent kinase 6 (CDK6).  Pharmacol. CDK6 degrdn. targets a selective dependency of acute myeloid leukemia cells, and transcriptomics and phosphoproteomics profiling of acute degrdn. of CDK6 enabled dynamic mapping of its immediate role in coordinating signaling and transcription.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGognoz8FFoR9LVg90H21EOLACvtfcHk0lhfGoOXYrIKJA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXnsVSj&md5=f8c1c3520c6c0218763d8769c59b4342</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2018.11.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2018.11.006%26sid%3Dliteratum%253Aachs%26aulast%3DBrand%26aufirst%3DM.%26aulast%3DJiang%26aufirst%3DB.%26aulast%3DBauer%26aufirst%3DS.%26aulast%3DDonovan%26aufirst%3DK.%2BA.%26aulast%3DLiang%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DE.%2BS.%26aulast%3DNowak%26aufirst%3DR.%2BP.%26aulast%3DYuan%26aufirst%3DJ.%2BC.%26aulast%3DZhang%26aufirst%3DT.%26aulast%3DKwiatkowski%26aufirst%3DN.%26aulast%3DM%25C3%25BCller%26aufirst%3DA.%2BC.%26aulast%3DFischer%26aufirst%3DE.%2BS.%26aulast%3DGray%26aufirst%3DN.%2BS.%26aulast%3DWinter%26aufirst%3DG.%2BE.%26atitle%3DHomolog-selective%2520degradation%2520as%2520a%2520strategy%2520to%2520probe%2520the%2520function%2520of%2520CDK6%2520in%2520AML%26jtitle%3DCell%2520Chem.%2520Biol.%26date%3D2019%26volume%3D26%26spage%3D300%26epage%3D306.e9%26doi%3D10.1016%2Fj.chembiol.2018.11.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref54"><div class="reference"><strong class="refLabel"><a href="#ref54" class="rightTabRefNumLink">54</a></strong><div class="NLM_citation" id="rightTab-cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Donovan, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fischer, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span> <span> </span><span class="NLM_article-title">Development of Dual and Selective Degraders of Cyclin-Dependent Kinases 4 and 6</span>. <i>Angew. Chem., Int. Ed. Engl.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">6321</span>– <span class="NLM_lpage">6326</span>, <span class="refDoi"> DOI: 10.1002/anie.201901336</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=10.1002%2Fanie.201901336" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=30802347" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=1%3ACAS%3A280%3ADC%252BB3cfnt1OqtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2019&pages=6321-6326&author=B.+Jiangauthor=E.+S.+Wangauthor=K.+A.+Donovanauthor=Y.+Liangauthor=E.+S.+Fischerauthor=T.+Zhangauthor=N.+S.+Gray&title=Development+of+Dual+and+Selective+Degraders+of+Cyclin-Dependent+Kinases+4+and+6&doi=10.1002%2Fanie.201901336"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Development of Dual and Selective Degraders of Cyclin-Dependent Kinases 4 and 6</span></div><div class="casAuthors">Jiang Baishan; Wang Eric S; Donovan Katherine A; Liang Yanke; Fischer Eric S; Zhang Tinghu; Gray Nathanael S</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie (International ed. in English)</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">6321-6326</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Cyclin-dependent kinases 4 and 6 (CDK4/6) are key regulators of the cell cycle, and there are FDA-approved CDK4/6 inhibitors for treating patients with metastatic breast cancer.  However, due to conservation of their ATP-binding sites, development of selective agents has remained elusive.  Here, we report imide-based degrader molecules capable of degrading both CDK4/6, or selectively degrading either CDK4 or CDK6.  We were also able to tune the activity of these molecules against Ikaros (IKZF1) and Aiolos (IKZF3), which are well-established targets of imide-based degraders.  We found that in mantle cell lymphoma cell lines, combined IKZF1/3 degradation with dual CDK4/6 degradation produced enhanced anti-proliferative effects compared to CDK4/6 inhibition, CDK4/6 degradation, or IKZF1/3 degradation.  In summary, we report here the first compounds capable of inducing selective degradation of CDK4 and CDK6 as tools to pharmacologically dissect their distinct biological functions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQvjF82FsSWX6qcneEtCi92fW6udTcc2ea6N1L85lpaorntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3cfnt1OqtA%253D%253D&md5=fa27938a9ec50a89644eb87812048329</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1002%2Fanie.201901336&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201901336%26sid%3Dliteratum%253Aachs%26aulast%3DJiang%26aufirst%3DB.%26aulast%3DWang%26aufirst%3DE.%2BS.%26aulast%3DDonovan%26aufirst%3DK.%2BA.%26aulast%3DLiang%26aufirst%3DY.%26aulast%3DFischer%26aufirst%3DE.%2BS.%26aulast%3DZhang%26aufirst%3DT.%26aulast%3DGray%26aufirst%3DN.%2BS.%26atitle%3DDevelopment%2520of%2520Dual%2520and%2520Selective%2520Degraders%2520of%2520Cyclin-Dependent%2520Kinases%25204%2520and%25206%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%2520Engl.%26date%3D2019%26volume%3D58%26spage%3D6321%26epage%3D6326%26doi%3D10.1002%2Fanie.201901336" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref55"><div class="reference"><strong class="refLabel"><a href="#ref55" class="rightTabRefNumLink">55</a></strong><div class="NLM_citation" id="rightTab-cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Su, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">An, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chandarlapaty, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rao, Y.</span></span> <span> </span><span class="NLM_article-title">Potent and preferential degradation of CDK6 via proteolysis targeting chimera degraders</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">7575</span>– <span class="NLM_lpage">7582</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b00871</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b00871" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhsVWls7zF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=7575-7582&author=S.+Suauthor=Z.+Yangauthor=H.+Gaoauthor=H.+Yangauthor=S.+Zhuauthor=Z.+Anauthor=J.+Wangauthor=Q.+Liauthor=S.+Chandarlapatyauthor=H.+Dengauthor=W.+Wuauthor=Y.+Rao&title=Potent+and+preferential+degradation+of+CDK6+via+proteolysis+targeting+chimera+degraders&doi=10.1021%2Facs.jmedchem.9b00871"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Potent and Preferential Degradation of CDK6 via Proteolysis Targeting Chimera Degraders</span></div><div class="casAuthors">Su, Shang; Yang, Zimo; Gao, Hongying; Yang, Haiyan; Zhu, Songbiao; An, Zixuan; Wang, Juanjuan; Li, Qing; Chandarlapaty, Sarat; Deng, Haiteng; Wu, Wei; Rao, Yu</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">7575-7582</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A focused PROTAC library hijacking cancer therapeutic target CDK6 was developed.  A design principle as "match/mismatch" was proposed for understanding the degrdn. profile differences in these PROTACs.  Notably, potent PROTACs with specific and remarkable CDK6 degrdn. potential were generated by linking CDK6 inhibitor palbociclib and E3 ligase CRBN recruiter pomalidomide.  The PROTAC strongly inhibited proliferation of hematopoietic cancer cells including multiple myeloma and robustly degraded copy-amplified/mutated forms of CDK6, indicating future potential clin. applications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqQhCa_z8uvA7Vg90H21EOLACvtfcHk0lihSwWyNIKNMA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhsVWls7zF&md5=ed100530898c2e4c1754ca48e7295134</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b00871&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b00871%26sid%3Dliteratum%253Aachs%26aulast%3DSu%26aufirst%3DS.%26aulast%3DYang%26aufirst%3DZ.%26aulast%3DGao%26aufirst%3DH.%26aulast%3DYang%26aufirst%3DH.%26aulast%3DZhu%26aufirst%3DS.%26aulast%3DAn%26aufirst%3DZ.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DQ.%26aulast%3DChandarlapaty%26aufirst%3DS.%26aulast%3DDeng%26aufirst%3DH.%26aulast%3DWu%26aufirst%3DW.%26aulast%3DRao%26aufirst%3DY.%26atitle%3DPotent%2520and%2520preferential%2520degradation%2520of%2520CDK6%2520via%2520proteolysis%2520targeting%2520chimera%2520degraders%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D7575%26epage%3D7582%26doi%3D10.1021%2Facs.jmedchem.9b00871" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref56"><div class="reference"><strong class="refLabel"><a href="#ref56" class="rightTabRefNumLink">56</a></strong><div class="NLM_citation" id="rightTab-cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wei, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span> <span> </span><span class="NLM_article-title">Discovery of a first-in-class mitogen-activated protein kinase kinase 1/2 degrader</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">10897</span>– <span class="NLM_lpage">10911</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b01528</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b01528" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitFOmt7bK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=10897-10911&author=J.+Weiauthor=J.+Huauthor=L.+Wangauthor=L.+Xieauthor=M.+S.+Jinauthor=X.+Chenauthor=J.+Liuauthor=J.+Jin&title=Discovery+of+a+first-in-class+mitogen-activated+protein+kinase+kinase+1%2F2+degrader&doi=10.1021%2Facs.jmedchem.9b01528"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a First-in-Class Mitogen-Activated Protein Kinase Kinase 1/2 Degrader</span></div><div class="casAuthors">Wei, Jieli; Hu, Jianping; Wang, Li; Xie, Ling; Jin, Margaret S.; Chen, Xian; Liu, Jing; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">10897-10911</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">MEK1 and MEK2 (also known as MAP2K1 and MAP2K2) are the "gatekeepers" of the ERK signaling output with redundant roles in controlling ERK activity.  Numerous inhibitors targeting MEK1/2 have been developed including three FDA-approved drugs.  However, acquired resistance to MEK1/2 inhibitors has been obsd. in patients and new therapeutic strategies are needed to overcome the resistance.  Here, the authors report a first-in-class degrader of MEK1/2, MS432 (I), which potently and selectively degraded MEK1 and MEK2 in a VHL E3 ligase- and proteasome-dependent manner and suppressed ERK phosphorylation in cells.  It inhibited colorectal cancer and melanoma cell proliferation much more effectively than its neg. control MS432N and its effect was phenocopied by MEK1/2 knockdown.  Compd. I was highly selective for MEK1/2 in global proteomic profiling studies.  It was also bioavailable in mice and can be used for in vivo efficacy studies.  The authors provide two well-characterized chem. tools to the biomedical community.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq1e5sg33oDYrVg90H21EOLACvtfcHk0lihSwWyNIKNMA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitFOmt7bK&md5=2c293b6b87990d6b814ec1dc88034f19</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b01528&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b01528%26sid%3Dliteratum%253Aachs%26aulast%3DWei%26aufirst%3DJ.%26aulast%3DHu%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DXie%26aufirst%3DL.%26aulast%3DJin%26aufirst%3DM.%2BS.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DDiscovery%2520of%2520a%2520first-in-class%2520mitogen-activated%2520protein%2520kinase%2520kinase%25201%252F2%2520degrader%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D10897%26epage%3D10911%26doi%3D10.1021%2Facs.jmedchem.9b01528" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref57"><div class="reference"><strong class="refLabel"><a href="#ref57" class="rightTabRefNumLink">57</a></strong><div class="NLM_citation" id="rightTab-cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vollmer, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cunoosamy, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lv, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nan, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perry, M. W. D.</span></span> <span> </span><span class="NLM_article-title">Design, synthesis, and biological evaluation of MEK PROTACs</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">157</span>– <span class="NLM_lpage">162</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b00810</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b00810" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitlWkt77E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=157-162&author=S.+Vollmerauthor=D.+Cunoosamyauthor=H.+Lvauthor=H.+Fengauthor=X.+Liauthor=Z.+Nanauthor=W.+Yangauthor=M.+W.+D.+Perry&title=Design%2C+synthesis%2C+and+biological+evaluation+of+MEK+PROTACs&doi=10.1021%2Facs.jmedchem.9b00810"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">Design, Synthesis, and Biological Evaluation of MEK PROTACs</span></div><div class="casAuthors">Vollmer, Stefan; Cunoosamy, Danen; Lv, Huafei; Feng, Huanxi; Li, Xia; Nan, Ziyang; Yang, Wenzhen; Perry, Matthew W. D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">157-162</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">PROteolysis TArgeting Chimeras (PROTACs) targeting the degrdn. of MEK have been designed based on allosteric MEK inhibitors.  Inhibition of the phosphorylation of ERK1/2 was less effective with the PROTACs than a small-mol. inhibitor; the best PROTACs, however, were more effective in inhibiting proliferation of A375 cells than an inhibitor.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGof7TjD-lFjAbVg90H21EOLACvtfcHk0lioZ1UH5IgdKw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitlWkt77E&md5=c83d63e23d436e0ef1a3c98e994f1073</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b00810&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b00810%26sid%3Dliteratum%253Aachs%26aulast%3DVollmer%26aufirst%3DS.%26aulast%3DCunoosamy%26aufirst%3DD.%26aulast%3DLv%26aufirst%3DH.%26aulast%3DFeng%26aufirst%3DH.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DNan%26aufirst%3DZ.%26aulast%3DYang%26aufirst%3DW.%26aulast%3DPerry%26aufirst%3DM.%2BW.%2BD.%26atitle%3DDesign%252C%2520synthesis%252C%2520and%2520biological%2520evaluation%2520of%2520MEK%2520PROTACs%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D157%26epage%3D162%26doi%3D10.1021%2Facs.jmedchem.9b00810" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref58"><div class="reference"><strong class="refLabel"><a href="#ref58" class="rightTabRefNumLink">58</a></strong><div class="NLM_citation" id="rightTab-cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Han, X.-R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiao, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plewe, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span> <span> </span><span class="NLM_article-title">Discovery of selective small molecule degraders of BRAF-V600E</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">4069</span>– <span class="NLM_lpage">4080</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b02083</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b02083" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=1%3ACAS%3A528%3ADC%252BB3cXlvFCjtbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=4069-4080&author=X.-R.+Hanauthor=L.+Chenauthor=Y.+Weiauthor=W.+Yuauthor=Y.+Chenauthor=C.+Zhangauthor=B.+Jiaoauthor=T.+Shiauthor=L.+Sunauthor=C.+Zhangauthor=Y.+Xuauthor=M.+R.+Leeauthor=Y.+Luoauthor=M.+B.+Pleweauthor=J.+Wang&title=Discovery+of+selective+small+molecule+degraders+of+BRAF-V600E&doi=10.1021%2Facs.jmedchem.9b02083"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Selective Small Molecule Degraders of BRAF-V600E</span></div><div class="casAuthors">Han, Xiao-Ran; Chen, Liqun; Wei, Yuanqi; Yu, Weihua; Chen, Yanke; Zhang, Chunyan; Jiao, Bingyang; Shi, Tingting; Sun, Lei; Zhang, Chao; Xu, Yang; Lee, Matthew R.; Luo, Ying; Plewe, Michael B.; Wang, Jialiang</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">4069-4080</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">BRAF is among the most frequently mutated oncogenes in human cancers.  Multiple small mol. BRAF kinase inhibitors have been approved for treating melanoma carrying BRAF-V600 mutations.  However, the benefits of BRAF kinase inhibitors are generally short-lived.  Small mol.-mediated targeted protein degrdn. has recently emerged as a novel pharmaceutical strategy to remove disease proteins through hijacking the cellular ubiquitin proteasome system (UPS).  In this study, we developed thalidomide-based heterobifunctional compds. that induced selective degrdn. of BRAF-V600E, but not the wild-type BRAF.  Downregulation of BRAF-V600E suppressed the MEK/ERK kinase cascade in melanoma cells and impaired cell growth in culture.  Abolishing the interaction between degraders and cereblon or blocking the UPS significantly impaired the activities of these degraders, validating a mechanistic role of UPS in mediating targeted degrdn. of BRAF-V600E.  These findings highlight a new approach to modulate the functions of oncogenic BRAF mutants and provide a framework to treat BRAF-dependent human cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoCRmOwotGfh7Vg90H21EOLACvtfcHk0lihJndo9mVjOw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXlvFCjtbg%253D&md5=db67b3ec42b8a3eb9641c278c8cd17b6</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b02083&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b02083%26sid%3Dliteratum%253Aachs%26aulast%3DHan%26aufirst%3DX.-R.%26aulast%3DChen%26aufirst%3DL.%26aulast%3DWei%26aufirst%3DY.%26aulast%3DYu%26aufirst%3DW.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DC.%26aulast%3DJiao%26aufirst%3DB.%26aulast%3DShi%26aufirst%3DT.%26aulast%3DSun%26aufirst%3DL.%26aulast%3DZhang%26aufirst%3DC.%26aulast%3DXu%26aufirst%3DY.%26aulast%3DLee%26aufirst%3DM.%2BR.%26aulast%3DLuo%26aufirst%3DY.%26aulast%3DPlewe%26aufirst%3DM.%2BB.%26aulast%3DWang%26aufirst%3DJ.%26atitle%3DDiscovery%2520of%2520selective%2520small%2520molecule%2520degraders%2520of%2520BRAF-V600E%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D4069%26epage%3D4080%26doi%3D10.1021%2Facs.jmedchem.9b02083" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref59"><div class="reference"><strong class="refLabel"><a href="#ref59" class="rightTabRefNumLink">59</a></strong><div class="NLM_citation" id="rightTab-cit59"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burgess, K.</span></span> <span> </span><span class="NLM_article-title">TrkC-targeted kinase inhibitors and PROTACs</span>. <i>Mol. Pharm.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">4313</span>– <span class="NLM_lpage">4318</span>, <span class="refDoi"> DOI: 10.1021/acs.molpharmaceut.9b00673</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.molpharmaceut.9b00673" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhslCntb%252FI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2019&pages=4313-4318&author=B.+Zhaoauthor=K.+Burgess&title=TrkC-targeted+kinase+inhibitors+and+PROTACs&doi=10.1021%2Facs.molpharmaceut.9b00673"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">TrkC-Targeted Kinase Inhibitors And PROTACs</span></div><div class="casAuthors">Zhao, Bosheng; Burgess, Kevin</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Pharmaceutics</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">4313-4318</span>CODEN:
                <span class="NLM_cas:coden">MPOHBP</span>;
        ISSN:<span class="NLM_cas:issn">1543-8384</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A small mol. motif (IY-IY), which binds the tropomyosin receptor kinase C (TrkC), was used to deliver the promiscuous kinase inhibitor (KI) dasatinib into breast cancer.  Conjugates with noncleavable (1) and cleavable (2) linkers were compared in cellular assays and shown to have more impact on the cell viabilities of TrkC+ breast cancer cells over TrkC- epithelial cells.  The IY-IY fragment was also used to recruit the E3 ligase cereblon, giving a potent proteolysis targeting chimeric (PROTAC) for TrkC degrdn. in metastatic breast cancer cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoBUrtHpr2asrVg90H21EOLACvtfcHk0lihJndo9mVjOw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhslCntb%252FI&md5=15fb3168872fe9d91bad6f288b92e87b</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1021%2Facs.molpharmaceut.9b00673&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.molpharmaceut.9b00673%26sid%3Dliteratum%253Aachs%26aulast%3DZhao%26aufirst%3DB.%26aulast%3DBurgess%26aufirst%3DK.%26atitle%3DTrkC-targeted%2520kinase%2520inhibitors%2520and%2520PROTACs%26jtitle%3DMol.%2520Pharm.%26date%3D2019%26volume%3D16%26spage%3D4313%26epage%3D4318%26doi%3D10.1021%2Facs.molpharmaceut.9b00673" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref60"><div class="reference"><strong class="refLabel"><a href="#ref60" class="rightTabRefNumLink">60</a></strong><div class="NLM_citation" id="rightTab-cit60"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Choi, H.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rucker, P. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fan, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Albaugh, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chopiuk, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gessier, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adrian, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hood, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jia, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Che, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCormack, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steffy, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Culazzo, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tompkins, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phung, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kreusch, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chaudhary, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prashad, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tuntland, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harris, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seidel, H. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loren, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Molteni, V.</span></span> <span> </span><span class="NLM_article-title">(R)-2-Phenylpyrrolidine substituted imidazopyridazines: a new class of potent and selective pan-TRK inhibitors</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">562</span>– <span class="NLM_lpage">567</span>, <span class="refDoi"> DOI: 10.1021/acsmedchemlett.5b00050</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.5b00050" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=1%3ACAS%3A528%3ADC%252BC2MXksVOrsbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=562-567&author=H.-S.+Choiauthor=P.+V.+Ruckerauthor=Z.+Wangauthor=Y.+Fanauthor=P.+Albaughauthor=G.+Chopiukauthor=F.+Gessierauthor=F.+Sunauthor=F.+Adrianauthor=G.+Liuauthor=T.+Hoodauthor=N.+Liauthor=Y.+Jiaauthor=J.+Cheauthor=S.+McCormackauthor=A.+Liauthor=J.+Liauthor=A.+Steffyauthor=A.+Culazzoauthor=C.+Tompkinsauthor=V.+Phungauthor=A.+Kreuschauthor=M.+Luauthor=B.+Huauthor=A.+Chaudharyauthor=M.+Prashadauthor=T.+Tuntlandauthor=B.+Liuauthor=J.+Harrisauthor=H.+M.+Seidelauthor=J.+Lorenauthor=V.+Molteni&title=%28R%29-2-Phenylpyrrolidine+substituted+imidazopyridazines%3A+a+new+class+of+potent+and+selective+pan-TRK+inhibitors&doi=10.1021%2Facsmedchemlett.5b00050"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">(R)-2-Phenylpyrrolidine Substituted Imidazopyridazines: A New Class of Potent and Selective Pan-TRK Inhibitors</span></div><div class="casAuthors">Choi, Ha-Soon; Rucker, Paul V.; Wang, Zhicheng; Fan, Yi; Albaugh, Pamela; Chopiuk, Greg; Gessier, Francois; Sun, Fangxian; Adrian, Francisco; Liu, Guoxun; Hood, Tami; Li, Nanxin; Jia, Yong; Che, Jianwei; McCormack, Susan; Li, Allen; Li, Jie; Steffy, Auzon; Culazzo, AnneMarie; Tompkins, Celine; Phung, Van; Kreusch, Andreas; Lu, Min; Hu, Bin; Chaudhary, Apurva; Prashad, Mahavir; Tuntland, Tove; Liu, Bo; Harris, Jennifer; Seidel, H. Martin; Loren, Jon; Molteni, Valentina</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">562-567</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Deregulated kinase activities of tropomyosin receptor kinase (TRK) family members have been shown to be assocd. with tumorigenesis and poor prognosis in a variety of cancer types.  In particular, several chromosomal rearrangements involving TRKA have been reported in colorectal, papillary thyroid, glioblastoma, melanoma, and lung tissue that are believed to be the key oncogenic driver in these tumors.  By screening the Novartis compd. collection, a novel imidazopyridazine TRK inhibitor was identified that served as a launching point for drug optimization.  Structure guided drug design led to the identification of (R)-2-phenylpyrrolidine substituted imidazopyridazines, e.g., I, as a series of potent, selective, orally bioavailable pan-TRK inhibitors achieving tumor regression in rats bearing KM12 xenografts.  From this work the (R)-2-phenylpyrrolidine has emerged as an ideal moiety to incorporate in bicyclic TRK inhibitors by virtue of its shape complementarity to the hydrophobic pocket of TRKs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrBOGdLZcZ2HbVg90H21EOLACvtfcHk0lh6ig5NggPnSg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXksVOrsbo%253D&md5=e2eacf85a3e762e8f2474a568e6d1b39</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.5b00050&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.5b00050%26sid%3Dliteratum%253Aachs%26aulast%3DChoi%26aufirst%3DH.-S.%26aulast%3DRucker%26aufirst%3DP.%2BV.%26aulast%3DWang%26aufirst%3DZ.%26aulast%3DFan%26aufirst%3DY.%26aulast%3DAlbaugh%26aufirst%3DP.%26aulast%3DChopiuk%26aufirst%3DG.%26aulast%3DGessier%26aufirst%3DF.%26aulast%3DSun%26aufirst%3DF.%26aulast%3DAdrian%26aufirst%3DF.%26aulast%3DLiu%26aufirst%3DG.%26aulast%3DHood%26aufirst%3DT.%26aulast%3DLi%26aufirst%3DN.%26aulast%3DJia%26aufirst%3DY.%26aulast%3DChe%26aufirst%3DJ.%26aulast%3DMcCormack%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DA.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DSteffy%26aufirst%3DA.%26aulast%3DCulazzo%26aufirst%3DA.%26aulast%3DTompkins%26aufirst%3DC.%26aulast%3DPhung%26aufirst%3DV.%26aulast%3DKreusch%26aufirst%3DA.%26aulast%3DLu%26aufirst%3DM.%26aulast%3DHu%26aufirst%3DB.%26aulast%3DChaudhary%26aufirst%3DA.%26aulast%3DPrashad%26aufirst%3DM.%26aulast%3DTuntland%26aufirst%3DT.%26aulast%3DLiu%26aufirst%3DB.%26aulast%3DHarris%26aufirst%3DJ.%26aulast%3DSeidel%26aufirst%3DH.%2BM.%26aulast%3DLoren%26aufirst%3DJ.%26aulast%3DMolteni%26aufirst%3DV.%26atitle%3D%2528R%2529-2-Phenylpyrrolidine%2520substituted%2520imidazopyridazines%253A%2520a%2520new%2520class%2520of%2520potent%2520and%2520selective%2520pan-TRK%2520inhibitors%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2015%26volume%3D6%26spage%3D562%26epage%3D567%26doi%3D10.1021%2Facsmedchemlett.5b00050" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref61"><div class="reference"><strong class="refLabel"><a href="#ref61" class="rightTabRefNumLink">61</a></strong><div class="NLM_citation" id="rightTab-cit61"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fischer, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Böhm, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lydeard, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stadler, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cavadini, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nagel, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Serluca, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Acker, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lingaraju, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tichkule, R. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schebesta, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Forrester, W. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schirle, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hassiepen, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ottl, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hild, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beckwith, R. E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harper, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jenkins, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomä, N. H.</span></span> <span> </span><span class="NLM_article-title">Structure of the DDB1-CRBN E3 ubiquitin ligase in complex with thalidomide</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>512</i></span>,  <span class="NLM_fpage">49</span>– <span class="NLM_lpage">53</span>, <span class="refDoi"> DOI: 10.1038/nature13527</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=10.1038%2Fnature13527" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=25043012" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=1%3ACAS%3A528%3ADC%252BC2cXht12ms7%252FP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=512&publication_year=2014&pages=49-53&author=E.+S.+Fischerauthor=K.+B%C3%B6hmauthor=J.+R.+Lydeardauthor=H.+Yangauthor=M.+B.+Stadlerauthor=S.+Cavadiniauthor=J.+Nagelauthor=F.+Serlucaauthor=V.+Ackerauthor=G.+M.+Lingarajuauthor=R.+B.+Tichkuleauthor=M.+Schebestaauthor=W.+C.+Forresterauthor=M.+Schirleauthor=U.+Hassiepenauthor=J.+Ottlauthor=M.+Hildauthor=R.+E.+J.+Beckwithauthor=J.+W.+Harperauthor=J.+L.+Jenkinsauthor=N.+H.+Thom%C3%A4&title=Structure+of+the+DDB1-CRBN+E3+ubiquitin+ligase+in+complex+with+thalidomide&doi=10.1038%2Fnature13527"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">Structure of the DDB1-CRBN E3 ubiquitin ligase in complex with thalidomide</span></div><div class="casAuthors">Fischer, Eric S.; Bohm, Kerstin; Lydeard, John R.; Yang, Haidi; Stadler, Michael B.; Cavadini, Simone; Nagel, Jane; Serluca, Fabrizio; Acker, Vincent; Lingaraju, Gondichatnahalli M.; Tichkule, Ritesh B.; Schebesta, Michael; Forrester, William C.; Schirle, Markus; Hassiepen, Ulrich; Ottl, Johannes; Hild, Marc; Beckwith, Rohan E. J.; Harper, J. Wade; Jenkins, Jeremy L.; Thoma, Nicolas H.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">512</span>
        (<span class="NLM_cas:issue">7512</span>),
    <span class="NLM_cas:pages">49-53</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">In the 1950s, the drug thalidomide, administered as a sedative to pregnant women, led to the birth of thousands of children with multiple defects.  Despite the teratogenicity of thalidomide and its derivs. lenalidomide and pomalidomide, these immunomodulatory drugs (IMiDs) recently emerged as effective treatments for multiple myeloma and 5q-deletion-assocd. dysplasia.  IMiDs target the E3 ubiquitin ligase CUL4-RBX1-DDB1-CRBN (known as CRL4CRBN) and promote the ubiquitination of the IKAROS family transcription factors IKZF1 and IKZF3 by CRL4CRBN.  Here we present crystal structures of the DDB1-CRBN complex bound to thalidomide, lenalidomide and pomalidomide.  The structure establishes that CRBN is a substrate receptor within CRL4CRBN and enantioselectively binds IMiDs.  Using an unbiased screen, we identified the homeobox transcription factor MEIS2 as an endogenous substrate of CRL4CRBN.  Our studies suggest that IMiDs block endogenous substrates (MEIS2) from binding to CRL4CRBN while the ligase complex is recruiting IKZF1 or IKZF3 for degrdn.  This dual activity implies that small mols. can modulate an E3 ubiquitin ligase and thereby upregulate or downregulate the ubiquitination of proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpGt2Y9GecKr7Vg90H21EOLACvtfcHk0lh6ig5NggPnSg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXht12ms7%252FP&md5=1c1eaa48bef87463cdd3f2b1e1bd400d</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1038%2Fnature13527&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature13527%26sid%3Dliteratum%253Aachs%26aulast%3DFischer%26aufirst%3DE.%2BS.%26aulast%3DB%25C3%25B6hm%26aufirst%3DK.%26aulast%3DLydeard%26aufirst%3DJ.%2BR.%26aulast%3DYang%26aufirst%3DH.%26aulast%3DStadler%26aufirst%3DM.%2BB.%26aulast%3DCavadini%26aufirst%3DS.%26aulast%3DNagel%26aufirst%3DJ.%26aulast%3DSerluca%26aufirst%3DF.%26aulast%3DAcker%26aufirst%3DV.%26aulast%3DLingaraju%26aufirst%3DG.%2BM.%26aulast%3DTichkule%26aufirst%3DR.%2BB.%26aulast%3DSchebesta%26aufirst%3DM.%26aulast%3DForrester%26aufirst%3DW.%2BC.%26aulast%3DSchirle%26aufirst%3DM.%26aulast%3DHassiepen%26aufirst%3DU.%26aulast%3DOttl%26aufirst%3DJ.%26aulast%3DHild%26aufirst%3DM.%26aulast%3DBeckwith%26aufirst%3DR.%2BE.%2BJ.%26aulast%3DHarper%26aufirst%3DJ.%2BW.%26aulast%3DJenkins%26aufirst%3DJ.%2BL.%26aulast%3DThom%25C3%25A4%26aufirst%3DN.%2BH.%26atitle%3DStructure%2520of%2520the%2520DDB1-CRBN%2520E3%2520ubiquitin%2520ligase%2520in%2520complex%2520with%2520thalidomide%26jtitle%3DNature%26date%3D2014%26volume%3D512%26spage%3D49%26epage%3D53%26doi%3D10.1038%2Fnature13527" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref62"><div class="reference"><strong class="refLabel"><a href="#ref62" class="rightTabRefNumLink">62</a></strong><div class="NLM_citation" id="rightTab-cit62"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Churcher, I.</span></span> <span> </span><span class="NLM_article-title">Protac-induced protein degradation in drug discovery: breaking the rules or just making new ones?</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">444</span>– <span class="NLM_lpage">452</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b01272</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b01272" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvVWqtbvJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=444-452&author=I.+Churcher&title=Protac-induced+protein+degradation+in+drug+discovery%3A+breaking+the+rules+or+just+making+new+ones%3F&doi=10.1021%2Facs.jmedchem.7b01272"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit62R"><div class="casContent"><span class="casTitleNuber">62</span><div class="casTitle"><span class="NLM_cas:atitle">Protac-Induced Protein Degradation in Drug Discovery: Breaking the Rules or Just Making New Ones?</span></div><div class="casAuthors">Churcher, Ian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">444-452</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Targeted protein degrdn., using bifunctional small mols. (Protacs) to remove specific proteins from within cells, has emerged as a novel drug discovery strategy with the potential to offer therapeutic interventions not achievable with existing approaches.  In this perspective, the brief history of the field is surveyed from a drug discovery perspective with a focus on the key advances in knowledge which have led to the definition and exemplification of protein degrdn. concepts, and their resulting applications to medicine discovery.  The approach has the potential to bring disruptive change to drug discovery - the many potential advantages and outstanding challenges which lie ahead of this technol. are discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr9n8I_0MhtkbVg90H21EOLACvtfcHk0lgE2PayIsMqGg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvVWqtbvJ&md5=e60f1777de8bd860c5001137d8583f1f</span></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b01272&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b01272%26sid%3Dliteratum%253Aachs%26aulast%3DChurcher%26aufirst%3DI.%26atitle%3DProtac-induced%2520protein%2520degradation%2520in%2520drug%2520discovery%253A%2520breaking%2520the%2520rules%2520or%2520just%2520making%2520new%2520ones%253F%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D444%26epage%3D452%26doi%3D10.1021%2Facs.jmedchem.7b01272" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref63"><div class="reference"><strong class="refLabel"><a href="#ref63" class="rightTabRefNumLink">63</a></strong><div class="NLM_citation" id="rightTab-cit63"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Scott, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rooney, T. P. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bayle, E. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mirza, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Willems, H. M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clarke, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andrews, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skidmore, J.</span></span> <span> </span><span class="NLM_article-title">Systematic investigation of the permeability of androgen receptor PROTACs</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">1539</span>, <span class="refDoi"> DOI: 10.1021/acsmedchemlett.0c00194</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.0c00194" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhtFWmt7%252FO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2020&pages=1539&author=D.+E.+Scottauthor=T.+P.+C.+Rooneyauthor=E.+D.+Bayleauthor=T.+Mirzaauthor=H.+M.+G.+Willemsauthor=J.+H.+Clarkeauthor=S.+P.+Andrewsauthor=J.+Skidmore&title=Systematic+investigation+of+the+permeability+of+androgen+receptor+PROTACs&doi=10.1021%2Facsmedchemlett.0c00194"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit63R"><div class="casContent"><span class="casTitleNuber">63</span><div class="casTitle"><span class="NLM_cas:atitle">Systematic Investigation of the Permeability of Androgen Receptor PROTACs</span></div><div class="casAuthors">Scott, Duncan E.; Rooney, Timothy P. C.; Bayle, Elliott D.; Mirza, Tashfina; Willems, Henriette M. G.; Clarke, Jonathan H.; Andrews, Stephen P.; Skidmore, John</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1539-1547</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Bifunctional mols. known as PROTACs simultaneously bind an E3 ligase and a protein of interest to direct ubiquitination and clearance of that protein, and they have emerged in the past decade as an exciting new paradigm in drug discovery.  In order to investigate the permeability and properties of these large mols., we synthesized two panels of PROTAC mols., constructed from a range of protein-target ligands, linkers, and E3 ligase ligands.  The androgen receptor, which is a well-studied protein in the PROTAC field was used as a model system.  The physicochem. properties and permeability of PROTACs are discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoG6Q_chyrzDLVg90H21EOLACvtfcHk0lgdYMP1oW1vsA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhtFWmt7%252FO&md5=26c8ea71901a57a09c03f7fb79bbe2c9</span></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.0c00194&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.0c00194%26sid%3Dliteratum%253Aachs%26aulast%3DScott%26aufirst%3DD.%2BE.%26aulast%3DRooney%26aufirst%3DT.%2BP.%2BC.%26aulast%3DBayle%26aufirst%3DE.%2BD.%26aulast%3DMirza%26aufirst%3DT.%26aulast%3DWillems%26aufirst%3DH.%2BM.%2BG.%26aulast%3DClarke%26aufirst%3DJ.%2BH.%26aulast%3DAndrews%26aufirst%3DS.%2BP.%26aulast%3DSkidmore%26aufirst%3DJ.%26atitle%3DSystematic%2520investigation%2520of%2520the%2520permeability%2520of%2520androgen%2520receptor%2520PROTACs%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2020%26volume%3D11%26spage%3D1539%26doi%3D10.1021%2Facsmedchemlett.0c00194" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref64"><div class="reference"><strong class="refLabel"><a href="#ref64" class="rightTabRefNumLink">64</a></strong><div class="NLM_citation" id="rightTab-cit64"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Edmondson, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fallan, C.</span></span> <span> </span><span class="NLM_article-title">Proteolysis targeting chimeras (PROTACs) in ‘beyond rule-of-five’ chemical space: Recent progress and future challenges</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">1555</span>– <span class="NLM_lpage">1564</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2019.04.030</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=10.1016%2Fj.bmcl.2019.04.030" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=31047748" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=1%3ACAS%3A528%3ADC%252BC1MXos12gs74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2019&pages=1555-1564&author=S.+D.+Edmondsonauthor=B.+Yangauthor=C.+Fallan&title=Proteolysis+targeting+chimeras+%28PROTACs%29+in+%E2%80%98beyond+rule-of-five%E2%80%99+chemical+space%3A+Recent+progress+and+future+challenges&doi=10.1016%2Fj.bmcl.2019.04.030"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit64R"><div class="casContent"><span class="casTitleNuber">64</span><div class="casTitle"><span class="NLM_cas:atitle">Proteolysis targeting chimeras (PROTACs) in 'beyond rule-of-five' chemical space: Recent progress and future challenges</span></div><div class="casAuthors">Edmondson, Scott D.; Yang, Bin; Fallan, Charlene</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">1555-1564</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Proteolysis targeting chimeras (PROTACs) are heterobifunctional compds. with mol. wts. and other properties that lie outside the classic 'rule-of-five' space.  Consequently, PROTACs have unique challenges assocd. with their development as potential therapeutic agents.  This review summarizes and analyzes a representative set of recent PROTACs and highlights some of the potential future challenges facing this promising modality.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqDcS0gIul-K7Vg90H21EOLACvtfcHk0lgdYMP1oW1vsA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXos12gs74%253D&md5=62a74500178bd711b1ddaa33d731b960</span></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2019.04.030&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2019.04.030%26sid%3Dliteratum%253Aachs%26aulast%3DEdmondson%26aufirst%3DS.%2BD.%26aulast%3DYang%26aufirst%3DB.%26aulast%3DFallan%26aufirst%3DC.%26atitle%3DProteolysis%2520targeting%2520chimeras%2520%2528PROTACs%2529%2520in%2520%25E2%2580%2598beyond%2520rule-of-five%25E2%2580%2599%2520chemical%2520space%253A%2520Recent%2520progress%2520and%2520future%2520challenges%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2019%26volume%3D29%26spage%3D1555%26epage%3D1564%26doi%3D10.1016%2Fj.bmcl.2019.04.030" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref65"><div class="reference"><strong class="refLabel"><a href="#ref65" class="rightTabRefNumLink">65</a></strong><div class="NLM_citation" id="rightTab-cit65"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pike, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williamson, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harlfinger, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McGinnity, D. F.</span></span> <span> </span><span class="NLM_article-title">Optimising proteolysis-targeting chimeras (PROTACs) for oral drug delivery: a drug metabolism and pharmacokinetics perspective</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2020</span>, <span class="refDoi"> DOI: 10.1016/j.drudis.2020.07.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=10.1016%2Fj.drudis.2020.07.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=32693163" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2020&author=A.+Pikeauthor=B.+Williamsonauthor=S.+Harlfingerauthor=S.+Martinauthor=D.+F.+McGinnity&title=Optimising+proteolysis-targeting+chimeras+%28PROTACs%29+for+oral+drug+delivery%3A+a+drug+metabolism+and+pharmacokinetics+perspective&doi=10.1016%2Fj.drudis.2020.07.013"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2020.07.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2020.07.013%26sid%3Dliteratum%253Aachs%26aulast%3DPike%26aufirst%3DA.%26aulast%3DWilliamson%26aufirst%3DB.%26aulast%3DHarlfinger%26aufirst%3DS.%26aulast%3DMartin%26aufirst%3DS.%26aulast%3DMcGinnity%26aufirst%3DD.%2BF.%26atitle%3DOptimising%2520proteolysis-targeting%2520chimeras%2520%2528PROTACs%2529%2520for%2520oral%2520drug%2520delivery%253A%2520a%2520drug%2520metabolism%2520and%2520pharmacokinetics%2520perspective%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2020%26doi%3D10.1016%2Fj.drudis.2020.07.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref66"><div class="reference"><strong class="refLabel"><a href="#ref66" class="rightTabRefNumLink">66</a></strong><div class="NLM_citation" id="rightTab-cit66"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ardini, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bosotti, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Borgia, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Ponti, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Somaschini, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cammarota, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amboldi, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raddrizzani, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Milani, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Magnaghi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ballinari, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Casero, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gasparri, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Banfi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Avanzi, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saccardo, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alzani, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bandiera, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Felder, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Donati, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pesenti, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sartore-Bianchi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gambacorta, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pierotti, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siena, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Veronese, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galvani, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Isacchi, A.</span></span> <span> </span><span class="NLM_article-title">The TPM3-NTRK1 rearrangement is a recurring event in colorectal carcinoma and is associated with tumor sensitivity to TRKA kinase inhibition</span>. <i>Mol. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">1495</span>– <span class="NLM_lpage">1507</span>, <span class="refDoi"> DOI: 10.1016/j.molonc.2014.06.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=10.1016%2Fj.molonc.2014.06.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=24962792" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtVaktb%252FI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2014&pages=1495-1507&author=E.+Ardiniauthor=R.+Bosottiauthor=A.+L.+Borgiaauthor=C.+De+Pontiauthor=A.+Somaschiniauthor=R.+Cammarotaauthor=N.+Amboldiauthor=L.+Raddrizzaniauthor=A.+Milaniauthor=P.+Magnaghiauthor=D.+Ballinariauthor=D.+Caseroauthor=F.+Gasparriauthor=P.+Banfiauthor=N.+Avanziauthor=M.+B.+Saccardoauthor=R.+Alzaniauthor=T.+Bandieraauthor=E.+Felderauthor=D.+Donatiauthor=E.+Pesentiauthor=A.+Sartore-Bianchiauthor=M.+Gambacortaauthor=M.+A.+Pierottiauthor=S.+Sienaauthor=S.+Veroneseauthor=A.+Galvaniauthor=A.+Isacchi&title=The+TPM3-NTRK1+rearrangement+is+a+recurring+event+in+colorectal+carcinoma+and+is+associated+with+tumor+sensitivity+to+TRKA+kinase+inhibition&doi=10.1016%2Fj.molonc.2014.06.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit66R"><div class="casContent"><span class="casTitleNuber">66</span><div class="casTitle"><span class="NLM_cas:atitle">The TPM3-NTRK1 rearrangement is a recurring event in colorectal carcinoma and is associated with tumor sensitivity to TRKA kinase inhibition</span></div><div class="casAuthors">Ardini, Elena; Bosotti, Roberta; Borgia, Andrea Lombardi; De Ponti, Cristina; Somaschini, Alessio; Cammarota, Rosaria; Amboldi, Nadia; Raddrizzani, Laura; Milani, Andrea; Magnaghi, Paola; Ballinari, Dario; Casero, Daniele; Gasparri, Fabio; Banfi, Patrizia; Avanzi, Nilla; Saccardo, Maria B.; Alzani, Rachele; Bandiera, Tiziano; Felder, Eduard; Donati, Daniele; Pesenti, Enrico; Sartore-Bianchi, Andrea; Gambacorta, Marcello; Pierotti, Marco A.; Siena, Salvatore; Veronese, Silvio; Galvani, Arturo; Isacchi, Antonella</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Oncology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1495-1507</span>CODEN:
                <span class="NLM_cas:coden">MOONC3</span>;
        ISSN:<span class="NLM_cas:issn">1574-7891</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The NTRK1 gene encodes Tropomyosin-related kinase A (TRKA), the high-affinity Nerve Growth Factor Receptor.  NTRK1 was originally isolated from a colorectal carcinoma (CRC) sample as component of a somatic rearrangement (TPM3-NTRK1) resulting in expression of the oncogenic chimeric protein TPM3-TRKA, but there has been no subsequent report regarding the relevance of this oncogene in CRC.  The KM12 human CRC cell line expresses the chimeric TPM3-TRKA protein and is hypersensitive to TRKA kinase inhibition.  We report the detailed characterization of the TPM3-NTRK1 genomic rearrangement in KM12 cells and through a cellular screening approach, the identification of NMS-P626, a novel highly potent and selective TRKA inhibitor.  NMS-P626 suppressed TPM3-TRKA phosphorylation and downstream signaling in KM12 cells and showed remarkable antitumor activity in mice bearing KM12 tumors.  Finally, using quant. reverse transcriptase PCR and immunohistochem. (IHC) we identified the TPM3-NTRK1 rearrangement in a CRC clin. sample, therefore suggesting that this chromosomal translocation is indeed a low frequency recurring event in CRC and that such patients might benefit from therapy with TRKA kinase inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo-EhukkQ24OrVg90H21EOLACvtfcHk0lhje7TYAkt2HQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtVaktb%252FI&md5=2a9de2410f69e2eb8ff84c9516828984</span></div><a href="/servlet/linkout?suffix=cit66&amp;dbid=16384&amp;doi=10.1016%2Fj.molonc.2014.06.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molonc.2014.06.001%26sid%3Dliteratum%253Aachs%26aulast%3DArdini%26aufirst%3DE.%26aulast%3DBosotti%26aufirst%3DR.%26aulast%3DBorgia%26aufirst%3DA.%2BL.%26aulast%3DDe%2BPonti%26aufirst%3DC.%26aulast%3DSomaschini%26aufirst%3DA.%26aulast%3DCammarota%26aufirst%3DR.%26aulast%3DAmboldi%26aufirst%3DN.%26aulast%3DRaddrizzani%26aufirst%3DL.%26aulast%3DMilani%26aufirst%3DA.%26aulast%3DMagnaghi%26aufirst%3DP.%26aulast%3DBallinari%26aufirst%3DD.%26aulast%3DCasero%26aufirst%3DD.%26aulast%3DGasparri%26aufirst%3DF.%26aulast%3DBanfi%26aufirst%3DP.%26aulast%3DAvanzi%26aufirst%3DN.%26aulast%3DSaccardo%26aufirst%3DM.%2BB.%26aulast%3DAlzani%26aufirst%3DR.%26aulast%3DBandiera%26aufirst%3DT.%26aulast%3DFelder%26aufirst%3DE.%26aulast%3DDonati%26aufirst%3DD.%26aulast%3DPesenti%26aufirst%3DE.%26aulast%3DSartore-Bianchi%26aufirst%3DA.%26aulast%3DGambacorta%26aufirst%3DM.%26aulast%3DPierotti%26aufirst%3DM.%2BA.%26aulast%3DSiena%26aufirst%3DS.%26aulast%3DVeronese%26aufirst%3DS.%26aulast%3DGalvani%26aufirst%3DA.%26aulast%3DIsacchi%26aufirst%3DA.%26atitle%3DThe%2520TPM3-NTRK1%2520rearrangement%2520is%2520a%2520recurring%2520event%2520in%2520colorectal%2520carcinoma%2520and%2520is%2520associated%2520with%2520tumor%2520sensitivity%2520to%2520TRKA%2520kinase%2520inhibition%26jtitle%3DMol.%2520Oncol.%26date%3D2014%26volume%3D8%26spage%3D1495%26epage%3D1507%26doi%3D10.1016%2Fj.molonc.2014.06.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref67"><div class="reference"><strong class="refLabel"><a href="#ref67" class="rightTabRefNumLink">67</a></strong><div class="NLM_citation" id="rightTab-cit67"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Matyskiela, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ito, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pagarigan, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, C.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, N.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Houston, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carmel, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tran, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riley, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nosaka, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lander, G. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gaidarova, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruchelman, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Handa, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carmichael, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daniel, T. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cathers, B. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lopez-Girona, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chamberlain, P. P.</span></span> <span> </span><span class="NLM_article-title">A novel cereblon modulator recruits GSPT1 to the CRL4CRBN ubiquitin ligase</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>535</i></span>,  <span class="NLM_fpage">252</span>– <span class="NLM_lpage">257</span>, <span class="refDoi"> DOI: 10.1038/nature18611</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=10.1038%2Fnature18611" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=27338790" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtVCltbnI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=535&publication_year=2016&pages=252-257&author=M.+E.+Matyskielaauthor=G.+Luauthor=T.+Itoauthor=B.+Pagariganauthor=C.-C.+Luauthor=K.+Millerauthor=W.+Fangauthor=N.-Y.+Wangauthor=D.+Nguyenauthor=J.+Houstonauthor=G.+Carmelauthor=T.+Tranauthor=M.+Rileyauthor=L.+A.+Nosakaauthor=G.+C.+Landerauthor=S.+Gaidarovaauthor=S.+Xuauthor=A.+L.+Ruchelmanauthor=H.+Handaauthor=J.+Carmichaelauthor=T.+O.+Danielauthor=B.+E.+Cathersauthor=A.+Lopez-Gironaauthor=P.+P.+Chamberlain&title=A+novel+cereblon+modulator+recruits+GSPT1+to+the+CRL4CRBN+ubiquitin+ligase&doi=10.1038%2Fnature18611"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit67R"><div class="casContent"><span class="casTitleNuber">67</span><div class="casTitle"><span class="NLM_cas:atitle">A novel cereblon modulator recruits GSPT1 to the CRL4CRBN ubiquitin ligase</span></div><div class="casAuthors">Matyskiela, Mary E.; Lu, Gang; Ito, Takumi; Pagarigan, Barbra; Lu, Chin-Chun; Miller, Karen; Fang, Wei; Wang, Nai-Yu; Nguyen, Derek; Houston, Jack; Carmel, Gilles; Tran, Tam; Riley, Mariko; Nosaka, Lyn'Al; Lander, Gabriel C.; Gaidarova, Svetlana; Xu, Shuichan; Ruchelman, Alexander L.; Handa, Hiroshi; Carmichael, James; Daniel, Thomas O.; Cathers, Brian E.; Lopez-Girona, Antonia; Chamberlain, Philip P.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">535</span>
        (<span class="NLM_cas:issue">7611</span>),
    <span class="NLM_cas:pages">252-257</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Immunomodulatory drugs bind to cereblon (CRBN) to confer differentiated substrate specificity on the CRL4CRBN E3 ubiquitin ligase.  Here we report the identification of a new cereblon modulator, CC-885, with potent anti-tumor activity.  The anti-tumor activity of CC-885 is mediated through the cereblon-dependent ubiquitination and degrdn. of the translation termination factor GSPT1.  Patient-derived acute myeloid leukemia tumor cells exhibit high sensitivity to CC-885, indicating the clin. potential of this mechanism.  Crystallog. studies of the CRBN-DDB1-CC-885-GSPT1 complex reveal that GSPT1 binds to cereblon through a surface turn contg. a glycine residue at a key position, interacting with both CC-885 and a 'hotspot' on the cereblon surface.  Although GSPT1 possesses no obvious structural, sequence or functional homol. to previously known cereblon substrates, mutational anal. and modeling indicate that the cereblon substrate Ikaros uses a similar structural feature to bind cereblon, suggesting a common motif for substrate recruitment.  These findings define a structural degron underlying cereblon 'neosubstrate' selectivity, and identify an anti-tumor target rendered druggable by cereblon modulation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqgAWiXdlJgYLVg90H21EOLACvtfcHk0lj29l5qwNCBTg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtVCltbnI&md5=1d4e340416aa0b08fc0f123faf638c3f</span></div><a href="/servlet/linkout?suffix=cit67&amp;dbid=16384&amp;doi=10.1038%2Fnature18611&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature18611%26sid%3Dliteratum%253Aachs%26aulast%3DMatyskiela%26aufirst%3DM.%2BE.%26aulast%3DLu%26aufirst%3DG.%26aulast%3DIto%26aufirst%3DT.%26aulast%3DPagarigan%26aufirst%3DB.%26aulast%3DLu%26aufirst%3DC.-C.%26aulast%3DMiller%26aufirst%3DK.%26aulast%3DFang%26aufirst%3DW.%26aulast%3DWang%26aufirst%3DN.-Y.%26aulast%3DNguyen%26aufirst%3DD.%26aulast%3DHouston%26aufirst%3DJ.%26aulast%3DCarmel%26aufirst%3DG.%26aulast%3DTran%26aufirst%3DT.%26aulast%3DRiley%26aufirst%3DM.%26aulast%3DNosaka%26aufirst%3DL.%2BA.%26aulast%3DLander%26aufirst%3DG.%2BC.%26aulast%3DGaidarova%26aufirst%3DS.%26aulast%3DXu%26aufirst%3DS.%26aulast%3DRuchelman%26aufirst%3DA.%2BL.%26aulast%3DHanda%26aufirst%3DH.%26aulast%3DCarmichael%26aufirst%3DJ.%26aulast%3DDaniel%26aufirst%3DT.%2BO.%26aulast%3DCathers%26aufirst%3DB.%2BE.%26aulast%3DLopez-Girona%26aufirst%3DA.%26aulast%3DChamberlain%26aufirst%3DP.%2BP.%26atitle%3DA%2520novel%2520cereblon%2520modulator%2520recruits%2520GSPT1%2520to%2520the%2520CRL4CRBN%2520ubiquitin%2520ligase%26jtitle%3DNature%26date%3D2016%26volume%3D535%26spage%3D252%26epage%3D257%26doi%3D10.1038%2Fnature18611" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref68"><div class="reference"><strong class="refLabel"><a href="#ref68" class="rightTabRefNumLink">68</a></strong><div class="NLM_citation" id="rightTab-cit68"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bondeson, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, B. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burslem, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buhimschi, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hines, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jaime-Figueroa, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamman, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ishchenko, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Lessons in PROTAC design from selective degradation with a promiscuous warhead</span>. <i>Cell Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">78</span>– <span class="NLM_lpage">87.e5</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2017.09.010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=10.1016%2Fj.chembiol.2017.09.010" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=29129718" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsl2lsrjN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2018&pages=78-87.e5&author=D.+P.+Bondesonauthor=B.+E.+Smithauthor=G.+M.+Burslemauthor=A.+D.+Buhimschiauthor=J.+Hinesauthor=S.+Jaime-Figueroaauthor=J.+Wangauthor=B.+D.+Hammanauthor=A.+Ishchenkoauthor=C.+M.+Crews&title=Lessons+in+PROTAC+design+from+selective+degradation+with+a+promiscuous+warhead&doi=10.1016%2Fj.chembiol.2017.09.010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit68R"><div class="casContent"><span class="casTitleNuber">68</span><div class="casTitle"><span class="NLM_cas:atitle">Lessons in PROTAC Design from Selective Degradation with a Promiscuous Warhead</span></div><div class="casAuthors">Bondeson, Daniel P.; Smith, Blake E.; Burslem, George M.; Buhimschi, Alexandru D.; Hines, John; Jaime-Figueroa, Saul; Wang, Jing; Hamman, Brian D.; Ishchenko, Alexey; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">78-87.e5</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Inhibiting protein function selectively is a major goal of modern drug discovery.  Here, we report a previously understudied benefit of small mol. proteolysis-targeting chimeras (PROTACs) that recruit E3 ubiquitin ligases to target proteins for their ubiquitination and subsequent proteasome-mediated degrdn.  Using promiscuous CRBN- and VHL-recruiting PROTACs that bind >50 kinases, we show that only a subset of bound targets is degraded.  The basis of this selectivity relies on protein-protein interactions between the E3 ubiquitin ligase and the target protein, as illustrated by engaged proteins that are not degraded as a result of unstable ternary complexes with PROTAC-recruited E3 ligases.  In contrast, weak PROTAC:target protein affinity can be stabilized by high-affinity target:PROTAC:ligase trimer interactions, leading to efficient degrdn.  This study highlights design guidelines for generating potent PROTACs as well as possibilities for degrading undruggable proteins immune to traditional small-mol. inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoISOBudOLyFrVg90H21EOLACvtfcHk0ljMDvx6yN-Xkw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsl2lsrjN&md5=30fac502705a2344c4261cf531e6bf9d</span></div><a href="/servlet/linkout?suffix=cit68&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2017.09.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2017.09.010%26sid%3Dliteratum%253Aachs%26aulast%3DBondeson%26aufirst%3DD.%2BP.%26aulast%3DSmith%26aufirst%3DB.%2BE.%26aulast%3DBurslem%26aufirst%3DG.%2BM.%26aulast%3DBuhimschi%26aufirst%3DA.%2BD.%26aulast%3DHines%26aufirst%3DJ.%26aulast%3DJaime-Figueroa%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DHamman%26aufirst%3DB.%2BD.%26aulast%3DIshchenko%26aufirst%3DA.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DLessons%2520in%2520PROTAC%2520design%2520from%2520selective%2520degradation%2520with%2520a%2520promiscuous%2520warhead%26jtitle%3DCell%2520Chem.%2520Biol.%26date%3D2018%26volume%3D25%26spage%3D78%26epage%3D87.e5%26doi%3D10.1016%2Fj.chembiol.2017.09.010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref69"><div class="reference"><strong class="refLabel"><a href="#ref69" class="rightTabRefNumLink">69</a></strong><div class="NLM_citation" id="rightTab-cit69"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ishoey, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chorn, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jaeger, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brand, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paulk, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bauer, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Erb, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parapatics, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bennett, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ecker, G. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winter, G. E.</span></span> <span> </span><span class="NLM_article-title">Translation termination factor GSPT1 is a phenotypically relevant off-target of heterobifunctional phthalimide degraders</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">553</span>– <span class="NLM_lpage">560</span>, <span class="refDoi"> DOI: 10.1021/acschembio.7b00969</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.7b00969" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtlyqu7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2018&pages=553-560&author=M.+Ishoeyauthor=S.+Chornauthor=N.+Singhauthor=M.+G.+Jaegerauthor=M.+Brandauthor=J.+Paulkauthor=S.+Bauerauthor=M.+A.+Erbauthor=K.+Parapaticsauthor=A.+C.+M%C3%BCllerauthor=K.+L.+Bennettauthor=G.+F.+Eckerauthor=J.+E.+Bradnerauthor=G.+E.+Winter&title=Translation+termination+factor+GSPT1+is+a+phenotypically+relevant+off-target+of+heterobifunctional+phthalimide+degraders&doi=10.1021%2Facschembio.7b00969"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit69R"><div class="casContent"><span class="casTitleNuber">69</span><div class="casTitle"><span class="NLM_cas:atitle">Translation Termination Factor GSPT1 Is a Phenotypically Relevant Off-Target of Heterobifunctional Phthalimide Degraders</span></div><div class="casAuthors">Ishoey, Mette; Chorn, Someth; Singh, Natesh; Jaeger, Martin G.; Brand, Matthias; Paulk, Joshiawa; Bauer, Sophie; Erb, Michael A.; Parapatics, Katja; Muller, Andre C.; Bennett, Keiryn L.; Ecker, Gerhard F.; Bradner, James E.; Winter, Georg E.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">553-560</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Protein degrdn. is an emerging therapeutic strategy with a unique mol. pharmacol. that enables the disruption of all functions assocd. with a target.  This is particularly relevant for proteins depending on mol. scaffolding, such as transcription factors or receptor tyrosine kinases (RTKs).  To address tractability of multiple RTKs for chem. degrdn. by the E3 ligase CUL4-RBX1-DDB1-CRBN (CRL4CRBN), we synthesized a series of phthalimide degraders based on the promiscuous kinase inhibitors sunitinib and PHA665752.  While both series failed to induce degrdn. of their consensus targets, individual mols. displayed pronounced efficacy in leukemia cell lines.  Orthogonal target identification supported by mol. docking led us to identify the translation termination factor G1 to S phase transition 1 (GSPT1) as a converging off-target, resulting from inadvertent E3 ligase modulation.  This research highlights the importance of monitoring degrdn. events that are independent of the resp. targeting ligand as a unique feature of small-mol. degraders.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq_ZxnB5rKvvLVg90H21EOLACvtfcHk0ljMDvx6yN-Xkw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtlyqu7o%253D&md5=02a70959e8a8929a3c70846c1ed0200f</span></div><a href="/servlet/linkout?suffix=cit69&amp;dbid=16384&amp;doi=10.1021%2Facschembio.7b00969&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.7b00969%26sid%3Dliteratum%253Aachs%26aulast%3DIshoey%26aufirst%3DM.%26aulast%3DChorn%26aufirst%3DS.%26aulast%3DSingh%26aufirst%3DN.%26aulast%3DJaeger%26aufirst%3DM.%2BG.%26aulast%3DBrand%26aufirst%3DM.%26aulast%3DPaulk%26aufirst%3DJ.%26aulast%3DBauer%26aufirst%3DS.%26aulast%3DErb%26aufirst%3DM.%2BA.%26aulast%3DParapatics%26aufirst%3DK.%26aulast%3DM%25C3%25BCller%26aufirst%3DA.%2BC.%26aulast%3DBennett%26aufirst%3DK.%2BL.%26aulast%3DEcker%26aufirst%3DG.%2BF.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26aulast%3DWinter%26aufirst%3DG.%2BE.%26atitle%3DTranslation%2520termination%2520factor%2520GSPT1%2520is%2520a%2520phenotypically%2520relevant%2520off-target%2520of%2520heterobifunctional%2520phthalimide%2520degraders%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2018%26volume%3D13%26spage%3D553%26epage%3D560%26doi%3D10.1021%2Facschembio.7b00969" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref70"><div class="reference"><strong class="refLabel"><a href="#ref70" class="rightTabRefNumLink">70</a></strong><div class="NLM_citation" id="rightTab-cit70"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aguilar, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span> <span> </span><span class="NLM_article-title">Simple structural modifications converting a bona fide MDM2 PROTAC degrader into a molecular glue molecule: a cautionary tale in the design of PROTAC degraders</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">9471</span>– <span class="NLM_lpage">9487</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.9b00846</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.9b00846" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhvVGgtb7K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=9471-9487&author=J.+Yangauthor=Y.+Liauthor=A.+Aguilarauthor=Z.+Liuauthor=C.-Y.+Yangauthor=S.+Wang&title=Simple+structural+modifications+converting+a+bona+fide+MDM2+PROTAC+degrader+into+a+molecular+glue+molecule%3A+a+cautionary+tale+in+the+design+of+PROTAC+degraders&doi=10.1021%2Facs.jmedchem.9b00846"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit70R"><div class="casContent"><span class="casTitleNuber">70</span><div class="casTitle"><span class="NLM_cas:atitle">Simple Structural Modifications Converting a Bona fide MDM2 PROTAC Degrader into a Molecular Glue Molecule: A Cautionary Tale in the Design of PROTAC Degraders</span></div><div class="casAuthors">Yang, Jiuling; Li, Yangbing; Aguilar, Angelo; Liu, Zhaomin; Yang, Chao-Yie; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">9471-9487</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Inducing protein degrdn. by proteolysis targeting chimeras (PROTACs) has gained tremendous momentum for its promise to discover and develop new therapies.  Based upon our previously reported PROTAC MDM2 degraders, we have designed and synthesized addnl. analogs.  Surprisingly, we found that simple structural modifications of MD-222, a bona fide MDM2 PROTAC degrader, converts it into a "mol. glue", as exemplified by MG-277.  MG-277 induces only moderate MDM2 degrdn. and fails to activate wild-type p53 but is highly potent in inhibition of tumor cell growth in a p53-independent manner.  Our mechanistic investigation established that MG-277 is not a PROTAC MDM2 degrader but instead works as a mol. glue, inducing degrdn. of a translation termination factor, GSPT1 to achieve its potent anticancer activity.  Our study provides the first example that simple structural modifications can convert a bona fide PROTAC degrader into a mol. glue compd., which has a completely different mechanism of action.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrH5W2xSSGs6rVg90H21EOLACvtfcHk0lgRpcNxUWNLKQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhvVGgtb7K&md5=8a3e36a4fc3110c798d379b70d6203fc</span></div><a href="/servlet/linkout?suffix=cit70&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.9b00846&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.9b00846%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DAguilar%26aufirst%3DA.%26aulast%3DLiu%26aufirst%3DZ.%26aulast%3DYang%26aufirst%3DC.-Y.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DSimple%2520structural%2520modifications%2520converting%2520a%2520bona%2520fide%2520MDM2%2520PROTAC%2520degrader%2520into%2520a%2520molecular%2520glue%2520molecule%253A%2520a%2520cautionary%2520tale%2520in%2520the%2520design%2520of%2520PROTAC%2520degraders%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D9471%26epage%3D9487%26doi%3D10.1021%2Facs.jmedchem.9b00846" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref71"><div class="reference"><strong class="refLabel"><a href="#ref71" class="rightTabRefNumLink">71</a></strong><div class="NLM_citation" id="rightTab-cit71"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Soucy, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, P. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Milhollen, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berger, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gavin, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adhikari, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brownell, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burke, K. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cardin, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Critchley, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cullis, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doucette, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garnsey, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gaulin, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gershman, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lublinsky, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McDonald, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mizutani, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Narayanan, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olhava, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peluso, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rezaei, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sintchak, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Talreja, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Traore, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vyskocil, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weatherhead, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dick, L. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Claiborne, C. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rolfe, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bolen, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Langston, S. P.</span></span> <span> </span><span class="NLM_article-title">An inhibitor of NEDD8-activating enzyme as a new approach to treat cancer</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>458</i></span>,  <span class="NLM_fpage">732</span>– <span class="NLM_lpage">736</span>, <span class="refDoi"> DOI: 10.1038/nature07884</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=10.1038%2Fnature07884" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=19360080" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;key=1%3ACAS%3A528%3ADC%252BD1MXksVenu7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=458&publication_year=2009&pages=732-736&author=T.+A.+Soucyauthor=P.+G.+Smithauthor=M.+A.+Milhollenauthor=A.+J.+Bergerauthor=J.+M.+Gavinauthor=S.+Adhikariauthor=J.+E.+Brownellauthor=K.+E.+Burkeauthor=D.+P.+Cardinauthor=S.+Critchleyauthor=C.+A.+Cullisauthor=A.+Doucetteauthor=J.+J.+Garnseyauthor=J.+L.+Gaulinauthor=R.+E.+Gershmanauthor=A.+R.+Lublinskyauthor=A.+McDonaldauthor=H.+Mizutaniauthor=U.+Narayananauthor=E.+J.+Olhavaauthor=S.+Pelusoauthor=M.+Rezaeiauthor=M.+D.+Sintchakauthor=T.+Talrejaauthor=M.+P.+Thomasauthor=T.+Traoreauthor=S.+Vyskocilauthor=G.+S.+Weatherheadauthor=J.+Yuauthor=J.+Zhangauthor=L.+R.+Dickauthor=C.+F.+Claiborneauthor=M.+Rolfeauthor=J.+B.+Bolenauthor=S.+P.+Langston&title=An+inhibitor+of+NEDD8-activating+enzyme+as+a+new+approach+to+treat+cancer&doi=10.1038%2Fnature07884"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit71R"><div class="casContent"><span class="casTitleNuber">71</span><div class="casTitle"><span class="NLM_cas:atitle">An inhibitor of NEDD8-activating enzyme as a new approach to treat cancer</span></div><div class="casAuthors">Soucy, Teresa A.; Smith, Peter G.; Milhollen, Michael A.; Berger, Allison J.; Gavin, James M.; Adhikari, Sharmila; Brownell, James E.; Burke, Kristine E.; Cardin, David P.; Critchley, Stephen; Cullis, Courtney A.; Doucette, Amanda; Garnsey, James J.; Gaulin, Jeffrey L.; Gershman, Rachel E.; Lublinsky, Anna R.; McDonald, Alice; Mizutani, Hirotake; Narayanan, Usha; Olhava, Edward J.; Peluso, Stephane; Rezaei, Mansoureh; Sintchak, Michael D.; Talreja, Tina; Thomas, Michael P.; Traore, Tary; Vyskocil, Stepan; Weatherhead, Gabriel S.; Yu, Jie; Zhang, Julie; Dick, Lawrence R.; Claiborne, Christopher F.; Rolfe, Mark; Bolen, Joseph B.; Langston, Steven P.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">458</span>
        (<span class="NLM_cas:issue">7239</span>),
    <span class="NLM_cas:pages">732-736</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The clin. development of an inhibitor of cellular proteasome function suggests that compds. targeting other components of the ubiquitin-proteasome system might prove useful for the treatment of human malignancies.  NEDD8-activating enzyme (NAE) is an essential component of the NEDD8 conjugation pathway that controls the activity of the cullin-RING subtype of ubiquitin ligases, thereby regulating the turnover of a subset of proteins upstream of the proteasome.  Substrates of cullin-RING ligases have important roles in cellular processes assocd. with cancer cell growth and survival pathways.  Here we describe MLN4924, a potent and selective inhibitor of NAE.  MLN4924 disrupts cullin-RING ligase-mediated protein turnover leading to apoptotic death in human tumor cells by a new mechanism of action, the deregulation of S-phase DNA synthesis.  MLN4924 suppressed the growth of human tumor xenografts in mice at compd. exposures that were well tolerated.  Our data suggest that NAE inhibitors may hold promise for the treatment of cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr8fx7OnZtYjbVg90H21EOLACvtfcHk0lgRpcNxUWNLKQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXksVenu7Y%253D&md5=365d4b4feac3994849050e00f1aae5e0</span></div><a href="/servlet/linkout?suffix=cit71&amp;dbid=16384&amp;doi=10.1038%2Fnature07884&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature07884%26sid%3Dliteratum%253Aachs%26aulast%3DSoucy%26aufirst%3DT.%2BA.%26aulast%3DSmith%26aufirst%3DP.%2BG.%26aulast%3DMilhollen%26aufirst%3DM.%2BA.%26aulast%3DBerger%26aufirst%3DA.%2BJ.%26aulast%3DGavin%26aufirst%3DJ.%2BM.%26aulast%3DAdhikari%26aufirst%3DS.%26aulast%3DBrownell%26aufirst%3DJ.%2BE.%26aulast%3DBurke%26aufirst%3DK.%2BE.%26aulast%3DCardin%26aufirst%3DD.%2BP.%26aulast%3DCritchley%26aufirst%3DS.%26aulast%3DCullis%26aufirst%3DC.%2BA.%26aulast%3DDoucette%26aufirst%3DA.%26aulast%3DGarnsey%26aufirst%3DJ.%2BJ.%26aulast%3DGaulin%26aufirst%3DJ.%2BL.%26aulast%3DGershman%26aufirst%3DR.%2BE.%26aulast%3DLublinsky%26aufirst%3DA.%2BR.%26aulast%3DMcDonald%26aufirst%3DA.%26aulast%3DMizutani%26aufirst%3DH.%26aulast%3DNarayanan%26aufirst%3DU.%26aulast%3DOlhava%26aufirst%3DE.%2BJ.%26aulast%3DPeluso%26aufirst%3DS.%26aulast%3DRezaei%26aufirst%3DM.%26aulast%3DSintchak%26aufirst%3DM.%2BD.%26aulast%3DTalreja%26aufirst%3DT.%26aulast%3DThomas%26aufirst%3DM.%2BP.%26aulast%3DTraore%26aufirst%3DT.%26aulast%3DVyskocil%26aufirst%3DS.%26aulast%3DWeatherhead%26aufirst%3DG.%2BS.%26aulast%3DYu%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DDick%26aufirst%3DL.%2BR.%26aulast%3DClaiborne%26aufirst%3DC.%2BF.%26aulast%3DRolfe%26aufirst%3DM.%26aulast%3DBolen%26aufirst%3DJ.%2BB.%26aulast%3DLangston%26aufirst%3DS.%2BP.%26atitle%3DAn%2520inhibitor%2520of%2520NEDD8-activating%2520enzyme%2520as%2520a%2520new%2520approach%2520to%2520treat%2520cancer%26jtitle%3DNature%26date%3D2009%26volume%3D458%26spage%3D732%26epage%3D736%26doi%3D10.1038%2Fnature07884" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4YNE" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4YNE','PDB','4YNE'); return false;">PDB: 4YNE</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i42"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c01342">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_10073"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a class="ext-link" href="/doi/10.1021/acs.jmedchem.0c01342?goto=supporting-info">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c01342</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline"><sup>1</sup>H NMR and <sup>13</sup>C NMR spectra, and UPLC-MS traces of compounds <b>5</b> and <b>6</b>; screening of the synthesized putative TRK degraders; immunoblotting analysis of samples for proteomic profiling; significantly differential proteins from proteomic profiling; and raw data  of compounds <b>5</b> and <b>6</b> (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c01342/suppl_file/jm0c01342_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Molecular formula strings and biological data (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c01342/suppl_file/jm0c01342_si_002.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01342/suppl_file/jm0c01342_si_001.pdf">jm0c01342_si_001.pdf (1.02 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01342/suppl_file/jm0c01342_si_002.csv">jm0c01342_si_002.csv (2.66 kb)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.0c01342&amp;pbContext=%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2020.63.issue-23%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.0c01342%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.0c01342" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                7MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"67993edc48721923","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
