
<html lang="en"     class="pb-page"  data-request-id="796be179-0a39-4f5c-8040-306a2f483305"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";issue:issue:10.1021/jmcmar.2018.61.issue-20;requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;article:article:10.1021/acs.jmedchem.8b00378;ctype:string:Journal Content;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="New Modalities, Technologies, and Partnerships in Probe and Lead Generation: Enabling a Mode-of-Action Centric Paradigm" /></meta><meta name="dc.Creator" content="Eric  Valeur" /></meta><meta name="dc.Creator" content="Patrick  Jimonet" /></meta><meta name="dc.Description" content="With the rise of novel biology and high potential target identification technologies originating from advances in genomics, medicinal chemists are progressively facing targets of increasing complex..." /></meta><meta name="Description" content="With the rise of novel biology and high potential target identification technologies originating from advances in genomics, medicinal chemists are progressively facing targets of increasing complex..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="May 31, 2018" /></meta><meta name="dc.Type" content="review-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.8b00378" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="Copyright © 2018 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.8b00378" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.8b00378" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.8b00378" /></link>
        
    
    

<title>New Modalities, Technologies, and Partnerships in Probe and Lead Generation: Enabling a Mode-of-Action Centric Paradigm | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.8b00378" /></meta><meta property="og:title" content="New Modalities, Technologies, and Partnerships in Probe and Lead Generation: Enabling a Mode-of-Action Centric Paradigm" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b00378/20181019/images/large/jm-2018-00378e_0022.jpeg" /></meta><meta property="og:description" content="With the rise of novel biology and high potential target identification technologies originating from advances in genomics, medicinal chemists are progressively facing targets of increasing complexity and often unprecedented. Novel hit finding technologies, combined with a wider choice of drug modalities, has resulted in a unique repertoire of options to address these challenging targets and to identify suitable starting points for optimization. Furthermore, innovative solutions originating from a range of academic groups and biotech companies require new types of collaborative models to leverage and integrate them in the drug discovery process. This perspective provides a guide for medicinal chemists covering contemporary probe and lead generation approaches and discusses the strengths and limitations of each strategy. Moreover, the expansion of strategies to modulate proteins creates the opportunity of a modality-agnostic and mode-of-action centric hit finding paradigm." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.8b00378"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.8b00378">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.8b00378&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.8b00378&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.8b00378&amp;href=/doi/10.1021/acs.jmedchem.8b00378" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2018</span><span class="cit-fg-volume">, 61</span><span class="cit-fg-issue">, 20</span><span class="cit-fg-pageRange">, 9004-9029</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/61/20" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.8b00159" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Perspective</span><a href="/doi/10.1021/acs.jmedchem.8b00667" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">New Modalities, Technologies, and Partnerships in Probe and Lead Generation: Enabling a Mode-of-Action Centric Paradigm</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Eric Valeur</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Eric Valeur</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Medicinal Chemistry, Cardiovascular, Renal and Metabolism, IMED Biotech Unit, AstraZeneca, Pepparedsleden 1, Mölndal 431 83, Sweden</div></div><span class="conrtib-corresp"><strong>*</strong>For E.V.: phone, +46 706384056; E-mail, <a href="/cdn-cgi/l/email-protection#9bfee9f2f8b5edfaf7feeee9dbfae8efe9fae1fef5fef8fab5f8f4f6"><span class="__cf_email__" data-cfemail="a9ccdbc0ca87dfc8c5ccdcdbe9c8dadddbc8d3ccc7cccac887cac6c4">[email protected]</span></a></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Eric++Valeur">Eric Valeur</a></span></div><div class="loa-info-bio"><span><a class="internalNav" href="#ath1">View Biography</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-8270-9432" title="Orcid link">http://orcid.org/0000-0001-8270-9432</a></div></div></span></li><span class="comma-separator"> and </span><li><span><span class="hlFld-ContribAuthor">Patrick Jimonet</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Patrick Jimonet</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">External Innovation Drug Discovery, Global Business Development & Licensing, Sanofi, 13 quai Jules Guesde, 94400 Vitry-sur-Seine, France</div></div><span class="conrtib-corresp"><strong>*</strong>For P.J.: phone, +33158938738; E-mail, <a href="/cdn-cgi/l/email-protection#42122336302b21296c282b2f2d2c27360231232c2d242b6c212d2f"><span class="__cf_email__" data-cfemail="114170656378727a3f7b787c7e7f74655162707f7e77783f727e7c">[email protected]</span></a></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Patrick++Jimonet">Patrick Jimonet</a></span></div><div class="loa-info-bio"><span><a class="internalNav" href="#ath2">View Biography</a></span></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.8b00378&amp;href=/doi/10.1021%2Facs.jmedchem.8b00378" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2018</span></span><span class="cit-volume">, 61</span><span class="cit-issue">, 20</span><span class="cit-pageRange">, 9004–9029</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">May 31, 2018</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>9 March 2018</li><li><span class="item_label"><b>Published</b> online</span>31 May 2018</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 25 October 2018</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.8b00378" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.8b00378</a></div><div class="article_header-article-copyright"><strong>Copyright © 2018 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D9004%26pageCount%3D26%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DEric%2BValeur%252C%2BPatrick%2BJimonet%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D61%26issueNum%3D20%26contentID%3Dacs.jmedchem.8b00378%26title%3DNew%2BModalities%252C%2BTechnologies%252C%2Band%2BPartnerships%2Bin%2BProbe%2Band%2BLead%2BGeneration%253A%2BEnabling%2Ba%2BMode-of-Action%2BCentric%2BParadigm%26numPages%3D26%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D9029%26publicationDate%3DOctober%2B2018">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.8b00378"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">5045</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">24</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.8b00378" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;New Modalities, Technologies, and Partnerships in Probe and Lead Generation: Enabling a Mode-of-Action Centric Paradigm&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Eric&quot;,&quot;last_name&quot;:&quot;Valeur&quot;},{&quot;first_name&quot;:&quot;Patrick&quot;,&quot;last_name&quot;:&quot;Jimonet&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2018&quot;,&quot;month&quot;:&quot;05&quot;,&quot;day&quot;:&quot;31&quot;,&quot;issue&quot;:&quot;20&quot;,&quot;volume&quot;:&quot;61&quot;,&quot;pages&quot;:&quot;9004-9029&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.8b00378&quot;},&quot;abstract&quot;:&quot;With the rise of novel biology and high potential target identification technologies originating from advances in genomics, medicinal chemists are progressively facing targets of increasing complexity and often unprecedented. Novel hit finding technologies, combined with a wider choice of drug modalities, has resulted in a unique repertoire of options to address these challenging targets and to identify suitable starting points for optimization. Furthermore, innovative solutions originating from a range of academic groups and biotech companies require new types of collaborative models to leverage and integrate them in the drug discovery process. This perspective provides a guide for medicinal chemists covering contemporary probe and lead generation approaches and discusses the strengths and limitations of each strategy. Moreover, the expansion of strategies to modulate proteins creates the opportunity of a modality-agnostic and mode-of-action centric hit finding paradigm.&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.8b00378&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.8b00378" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.8b00378&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.8b00378" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.8b00378&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.8b00378" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.8b00378&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.8b00378&amp;href=/doi/10.1021/acs.jmedchem.8b00378" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.8b00378" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.8b00378" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (7 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.8b00378&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.8b00378%26sid%3Dliteratum%253Aachs%26pmid%3D29851477%26genre%3Darticle%26aulast%3DValeur%26date%3D2018%26atitle%3DNew%2BModalities%252C%2BTechnologies%252C%2Band%2BPartnerships%2Bin%2BProbe%2Band%2BLead%2BGeneration%253A%2BEnabling%2Ba%2BMode-of-Action%2BCentric%2BParadigm%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D61%26issue%3D20%26spage%3D9004%26epage%3D9029%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=291439" title="Genetics">Genetics</a>,</li><li><a href="/action/doSearch?ConceptID=292202" title="Drug discovery">Drug discovery</a>,</li><li><a href="/action/doSearch?ConceptID=292276" title="Small molecules">Small molecules</a>,</li><li><a href="/action/doSearch?ConceptID=291352" title="Screening assays">Screening assays</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/61/20" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/jmcmar.2018.61.issue-20/20181025/jmcmar.2018.61.issue-20.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b00378/20181019/images/medium/jm-2018-00378e_0022.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b00378/20181019/images/large/jm-2018-00378e_0022.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00378&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">With the rise of novel biology and high potential target identification technologies originating from advances in genomics, medicinal chemists are progressively facing targets of increasing complexity and often unprecedented. Novel hit finding technologies, combined with a wider choice of drug modalities, has resulted in a unique repertoire of options to address these challenging targets and to identify suitable starting points for optimization. Furthermore, innovative solutions originating from a range of academic groups and biotech companies require new types of collaborative models to leverage and integrate them in the drug discovery process. This perspective provides a guide for medicinal chemists covering contemporary probe and lead generation approaches and discusses the strengths and limitations of each strategy. Moreover, the expansion of strategies to modulate proteins creates the opportunity of a modality-agnostic and mode-of-action centric hit finding paradigm.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">1.  Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_24466" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_24466" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Over the past decade, the target landscape has dramatically changed. Many protein targets, now considered as the “low hanging fruits”, have been successfully drugged, progressively shifting focus to more challenging, often labeled as “undruggable”, target classes such as protein–protein interactions (PPIs).<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> The rise of novel target identification strategies, such as genomics<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a> and genome-wide CRISPR screens,<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> is also resulting in an increasing number of unprecedented targets. Furthermore, progress in the understanding of cell biology and of mechanisms involved in the regulation of protein levels is also further extending the target scope. Indeed, less than 2% of the human genome encodes protein, and noncoding RNAs play fundamental roles in regulating gene expression and other cellular processes, therefore representing a further reservoir of potential drug targets.<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a></div><div class="NLM_p">To address this ever-changing target landscape, a broader variety of drug modalities is required. In addition to the opportunities presented by small molecules and antibody/protein therapeutics, a range of so-called new modalities,<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> including precise genome editing, modified peptides, oligonucleotides, macrocycles, and various conjugates, have opened up novel ways to modulate targets.</div><div class="NLM_p">Medicinal chemists face the demanding responsibility of identifying starting points for these challenging and novel targets to either generate probe or lead molecules. Fundamentally, chemists also have a major role to play in selecting the right modalities. Interestingly, these various modalities can modulate proteins at different levels, including at the transcriptional and translational level (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>), which triggers the opportunity to place the selection of the mode-of-action (MOA) at the center of any drug discovery project, and apply the appropriate hit finding strategy based on this selection. Such a paradigm also fosters the possibility to be truly modality-agnostic by shifting the selection of modality after the choice of MOAs and reinforces the step of building strong knowledge on targets for decision-making.<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a></div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b00378/20181019/images/medium/jm-2018-00378e_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b00378/20181019/images/large/jm-2018-00378e_0001.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. MOAs affecting direct protein function and protein levels</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b00378/20181019/images/large/jm-2018-00378e_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00378&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Within a MOA-centric paradigm, DNA can be modified through precise genome editing such as CRISPR/CAS9.<a onclick="showRef(event, 'ref7 ref8 ref9'); return false;" href="javascript:void(0);" class="ref ref7 ref8 ref9">(7−9)</a> Many challenges exist with this emerging technology, likely restricting this MOA to few cases, such as rare genetic disorders, for now. Another MOA is to modulate proteins levels through up- or down- regulation of RNA, including to affect splicing, using either oligonucleotides or small molecule approaches or perhaps CRISPR.<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> Oligonucleotides, in the form of small interfering RNA (siRNA) or antisense oligonucleotides (ASOs) can silence RNA targets and hence reduce protein levels but also increase levels of downregulated proteins when used in the form of modified RNA.<a onclick="showRef(event, 'ref11 ref12'); return false;" href="javascript:void(0);" class="ref ref11 ref12">(11,12)</a> Small molecules that downregulate proteins at the translational level are also becoming a credible option, as exemplified with the PCSK9 protein.<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> The next stage of intervention is to modulate protein levels through induction of degradation, for example with the so-called proteolysis targeting chimera (PROTAC).<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> Finally, the classical MOA of modulating protein functions through agonism/antagonism is enabled through a choice of small molecule, peptide, antibody, or even aptamer modalities.<a onclick="showRef(event, 'ref5 ref15 ref16'); return false;" href="javascript:void(0);" class="ref ref5 ref15 ref16">(5,15,16)</a></div><div class="NLM_p">To enable probe and lead generation against these MOAs, a range of new hit finding technologies have been developed across modalities. For example, phage and RNA display have allowed production of large libraries of cyclic peptides containing natural and noncanonical amino acids.<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> Progress in structural technologies is expanding the possibilities of structure-based lead generation through both protein structure mimetics and a range of small molecules diversity approaches such as virtual screening.<a onclick="showRef(event, 'ref18 ref19 ref20'); return false;" href="javascript:void(0);" class="ref ref18 ref19 ref20">(18−20)</a> The success of high throughput screening (HTS) using discrete screening collections can be optimized by new collaboration models, increasing the diversity and quality of such libraries.<a onclick="showRef(event, 'ref21 ref22 ref23'); return false;" href="javascript:void(0);" class="ref ref21 ref22 ref23">(21−23)</a></div><div class="NLM_p">To enable a MOA-centric paradigm and with such a vast repertoire, the selection of the best approaches with their pros and cons, their issues, and potential solutions, should be carefully made, knowing that access to innovation through external partners will often be necessary to succeed. With this prospect in mind, this review provides a contemporary guide to novel hit finding opportunities.</div><div class="NLM_p last">Herein, the range of new hit finding strategies for directly modulating protein functions are broadly divided into structure-agnostic screens (<a class="ref internalNav" href="#sec2" aria-label="section 2">section 2</a>), including through the extension of compound collections with partnerships (<a class="ref internalNav" href="#sec3" aria-label="section 3">section 3</a>) and structure-enabled approaches (<a class="ref internalNav" href="#sec4" aria-label="section 4">section 4</a>). The modulation of protein levels through RNA interference with oligonucleotides and small molecules and through protein degradation is then covered (<a class="ref internalNav" href="#sec5" aria-label="section 5">section 5</a>). All of these new approaches are brought into context with a guide to select the most suitable hit finding strategies within a MOA-centric paradigm (<a class="ref internalNav" href="#sec6" aria-label="section 6">section 6</a>). Finally, an outlook on emerging hit finding technologies is provided (<a class="ref internalNav" href="#sec7" aria-label="section 7">section 7</a>). All sections may be read independently, with the recommendation that medicinal chemists should first apply a MOA-centric selection for their target of interest before diving back into the relevant approaches. Due to the broad scope of this review, only representative examples of each strategy are presented.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">2.  Structure-Agnostic Probe and Lead Generation for Direct Modulation of Protein Functions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_75835" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_75835" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">In the context of modulating protein functions through classical agonism or antagonism, medicinal chemists have access to a range of established and new technologies and modalities regardless of the availability of structural information. Indeed, while the approaches described in this section are most relevant for proteins lacking structural data, these may still be applied to the cases where crystal structures are available, with the mindset of maximizing chances of identifying a starting point. This section focuses on novel approaches and modalities. Within the category of structure-agnostic technologies, the discovery of antibodies, in particular through phage display,<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> is well-established and particularly well suited for challenging extracellular targets. Similarly, the so-called systematic evolution of ligands by exponential enrichment (SELEX) display and select method can robustly identify aptamers.<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> The foundation and prospect for these two modalities are covered in details in selected reviews.<a onclick="showRef(event, 'ref15 ref16'); return false;" href="javascript:void(0);" class="ref ref15 ref16">(15,16)</a></div><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4">2.1.  Identification of Stabilized Peptides As Starting Points</h3><div class="NLM_p">Phage display of peptides was developed several decades ago and represents a robust method to identify peptidic starting points against any protein. This strategy has historically only delivered linear peptides, with the associated challenges of limited stability and cell permeability.<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> With the development and interest in next generation peptides to address challenging targets, novel technologies or adaptation of current technologies have emerged. In particular, phage display has now been extended to the generation of constrained cyclic peptides. After displaying random sequences containing three cysteine residues, the phage is exposed to a reactive small molecule, resulting in the formation of a bicyclic peptide (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>a).<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> The small molecule core and the length of the different loops (typically 3–6 amino acids) can be varied to induce conformational changes in the loop regions, resulting in increased library diversity.<a onclick="showRef(event, 'ref28 ref29'); return false;" href="javascript:void(0);" class="ref ref28 ref29">(28,29)</a> The constraint induces increased stability as well as higher affinity and specificity. For example, a library of around 10<sup>9</sup> bicyclic peptides underwent three rounds of selection against human plasma kallikrein (PKL).<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> From the identified hits with binding affinities in the region of 100–400 nM, consensus sequences were identified and utilized for new library design which was applied to new rounds of selection. The best analogue <b>1</b> displayed a binding affinity of 20 nM for human PKL (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>). Interestingly, the hit was selective over the mouse orthologue, highlighting the very high selectivity that bicyclic structures can achieve. Overall, this approach has been applied to several target classes beyond proteases,<a onclick="showRef(event, 'ref30 ref31'); return false;" href="javascript:void(0);" class="ref ref30 ref31">(30,31)</a> including targets “undruggable” to small molecules such as NOTCH1<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> and β-catenin,<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> and is being pursued commercially by Bicycle Therapeutics.</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b00378/20181019/images/medium/jm-2018-00378e_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b00378/20181019/images/large/jm-2018-00378e_0002.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. (a) modified phage display for the identification of bicyclic peptides. (b) Modified mRNA display for cyclic peptides containing unnatural amino acids.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b00378/20181019/images/large/jm-2018-00378e_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00378&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b00378/20181019/images/medium/jm-2018-00378e_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b00378/20181019/images/large/jm-2018-00378e_0003.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Examples of stabilized peptides. Color coding for amino acids: purple = <span class="smallcaps smallerCapital">d</span>-amino acid</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b00378/20181019/images/large/jm-2018-00378e_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00378&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Another modification consists in displaying β-hairpins, in essence via fusion of the two peptidic ends with the phage surface. Although the approach has been commercially advertised by Polyphor, explicit published examples have not been reported.<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> Modified phage display approaches have also been developed from the inspiration of naturally occurring stabilized peptides. Lanthipeptides are a class of peptides produced by a range of bacteria and contain multiple thioether cycles. In an elegant approach, phage display was combined with modifying enzymes to generate post-translationally modified peptides, and this de novo lanthipeptide library was applied to the identification of novel uPA inhibitors.<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a></div><div class="NLM_p">Another strategy has been aimed at enabling the incorporation of unnatural amino acids to increase diversity and stability. To generate any meaningful library size, display methods remain the approach of choice. Thus, the genetic code for natural amino acids and the translation machinery has been altered and reconstituted in vitro, significantly expanding the scope of mRNA display.<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> To achieve this, tRNA are loaded with unnatural amino acids, switching the assigned codons normally encoding for a given natural amino acid to their nonproteinogenic replacement. Both modified and canonical tRNA are then mixed with ribosomes and with mRNA encoding for the peptides to be generated. Beyond the advantage of merely introducing unnatural amino acids, the major advantage of the system is its compatibility with chemoreactive unnatural amino acids. Thus, a chloroacetyl moiety can be post-translationally cyclized via reaction with a cysteine. This so-called RaPID system can generate cyclic peptides containing typically 4–25 residues with a library size of approximatively 10<sup>12</sup>, which are selected in an affinity setting, followed by amplification (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>b).<a onclick="showRef(event, 'ref17 ref36'); return false;" href="javascript:void(0);" class="ref ref17 ref36">(17,36)</a> The technology has been applied to a range of targets including enzymes,<a onclick="showRef(event, 'ref37 ref38 ref39'); return false;" href="javascript:void(0);" class="ref ref37 ref38 ref39">(37−39)</a> and PPIs,<a onclick="showRef(event, 'ref40 ref41'); return false;" href="javascript:void(0);" class="ref ref40 ref41">(40,41)</a> and is commercialized by Peptidream. A recent example is the identification of selective histone demethylases inhibitors, where potent hits inhibited KDM4 family members in excess of 100 fold.<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a> In particular, peptide <b>2</b> bound the KDM4A substrate binding pocket leading to double-digit nanomolar activity (IC<sub>50</sub> = 42 nM, <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>), and selectivity could be attributed to a specific interaction with an arginine residue. Further optimization led to peptides with increased stability and target engagement in cells, as demonstrated by cellular thermal shift assay (CETSA). Another significant example is the activation of the Met signaling pathway. Traditional drug discovery approaches are often aimed at inhibiting pathways and strategies to induce direct activation are scarce. Using the RaPID approach, monomeric Met binders where identified and turned into dimeric binders through cross-linking.<a onclick="showRef(event, 'ref43'); return false;" href="javascript:void(0);" class="ref ref43">(43)</a> This approach resulted in the recruitment of two Met proteins leading to formation of the homodimer and subsequent activation of the signaling pathway.</div><div class="NLM_p">Finally, a platform developed at Roche Nimblegen enables the production of high density peptides microarrays. Almost three millions peptides, including cyclic ones, can be synthesized on one single slide with high precision through digital, light-directed, controlled synthesis.<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a> The platform tolerates formation of cyclic peptides and incorporation of unnatural amino acids providing a broad diversity, which can be screened by surface plasmon resonance (SPR). As a proof of principle for the technology, novel binders to streptavidin were identified. While this approach provides far less diversity than display, the rapid synthesis and incorporation of a range of amino acids is attractive for hit optimization.</div><div class="NLM_p last">Overall, these strategies are most relevant for challenging extracellular targets. In addition, when selectivity is deemed challenging to achieve, these stabilized peptides approaches may be more suitable than small molecules. Examples of intracellular targets remain scarce, which still places this field as a less favorable option for these targets, although further understanding of cellular uptake may eventually change this limitation.<a onclick="showRef(event, 'ref45 ref46 ref47 ref48'); return false;" href="javascript:void(0);" class="ref ref45 ref46 ref47 ref48">(45−48)</a> The process typically takes 3–4 months and requires building partnerships with specialized companies such as Bicycle Therapeutics, Polyphor, or Peptidream. Importantly, follow-on functional assays are required to confirm the relevance of hits identified through these affinity-based assays. From an optimization standpoint, the cyclic nature of these peptides provides them with greater stability than linear peptides. However, when limited stability is observed during hit and lead optimization, in particular in vivo, several strategies are possible, including the introduction of <span class="smallcaps smallerCapital">d</span>-amino acids and other noncanonical derivatives.<a onclick="showRef(event, 'ref49'); return false;" href="javascript:void(0);" class="ref ref49">(49)</a> Half-life extension through, for example, lipidation is an additional possibility,<a onclick="showRef(event, 'ref50'); return false;" href="javascript:void(0);" class="ref ref50">(50)</a> which has demonstrated impact up to market with semaglutide.</div></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5">2.2.  Screening Libraries of Peptidic Natural Products</h3><div class="NLM_p">Compounds of peptidic nature are largely represented in natural products from various sources. These include the immunosuppressant cyclosporine originating from soil fungi and Kalata B1, a cyclotide isolated from plant extracts that has been used for labor acceleration through its uterotonic effect in Africa. Peptidic natural products possess complex properties often very difficult to mimic with synthetic analogues. Animal venoms are a representative example of such valuable peptides in the context of probe and lead generation. Venoms represent an enormous reservoir of bioactive compounds: 170000 venomous species, each venom containing 100–1000 compounds, and only a very small number already characterized (∼3000 compounds).<a onclick="showRef(event, 'ref51'); return false;" href="javascript:void(0);" class="ref ref51">(51)</a> This set of millions of natural products may be considered as a unique library of peptidic natural products of high interest for challenging targets but also as valuable chemical tools to study biological mechanisms. Venom peptides have been the source of interesting new drugs for many years.<a onclick="showRef(event, 'ref52'); return false;" href="javascript:void(0);" class="ref ref52">(52)</a> From captopril originating from pit viper for hypertension,<a onclick="showRef(event, 'ref53'); return false;" href="javascript:void(0);" class="ref ref53">(53)</a> to more recently ziconotide from cone snail for chronic pain<a onclick="showRef(event, 'ref54'); return false;" href="javascript:void(0);" class="ref ref54">(54)</a> and exenatide from gila monster lizard for type 2 diabetes,<a onclick="showRef(event, 'ref55'); return false;" href="javascript:void(0);" class="ref ref55">(55)</a> venoms have confirmed their value as source of drugs targeting various protein target types, as the three above-mentioned compounds are respectively targeting an enzyme, an ion channel, and a GPCR.</div><div class="NLM_p">Peptides found in venoms are generally highly structured with a diversity of tridimensional folds favorable for interaction with target proteins.<a onclick="showRef(event, 'ref56'); return false;" href="javascript:void(0);" class="ref ref56">(56)</a> The main target class of these compounds is clearly ion channels, as evolution has optimized venoms to immobilize preys generally through ion channel blockade. Importantly, this bioactive effect is however only observed for a very minor fraction of the peptide pool in venoms. The diversity of folds in venom peptides can therefore be considered as a library of cyclic folded peptides suitable for opportunistic interaction with any protein target, particularly when small molecules or antibodies are not a good option, especially for extracellular or membrane-bound targets, and when oral administration is not an absolute requirement.</div><div class="NLM_p">To perform their screening, venoms are fractionated into solutions containing about 10 peptides (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>). Active fractions are then further fractionated, until subfractions containing only one peptide are obtained for the next round of screening. The active peptides are then sequenced and confirmed by chemical synthesis and folding (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>). This process can be handled in parallel to classical compound collection screening but requires specialist knowledge, often sourced through an expert collaboration partner, to efficiently drive the whole process. One example of a specialized company in this field is Smartox.</div><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b00378/20181019/images/medium/jm-2018-00378e_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b00378/20181019/images/large/jm-2018-00378e_0004.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Workflow for screening venoms.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b00378/20181019/images/large/jm-2018-00378e_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00378&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Limitations for the use of venoms in the hit finding process stem mainly from the process duration and the difficulty to confirm active venom fractions when single peptides are isolated and synthesized. However, these potential drawbacks are directly correlated with the quality of venom fractions that should contain a limited number of peptides within mixtures, devoid of any unwanted or reactive material that could negatively affect screening results. A comparison of head-to-head crude venoms and venom fractions in a primary screening underlined the significant added value of careful fractionation of crude venoms as an initial step. Many false positives are removed and false negatives using crude venoms revealed, therefore focusing efforts on high value active fractions to proceed toward peptide identification and synthesis for confirmation. The whole process is manageable within a few months, but the outcome depends on the quality of the biological assays validating target engagement and on the peptide hit structures, especially when considering potential folding difficulty if multiple disulfide bridges are present. Ion channels, GPCRs, and other membrane proteins are of particular interest for this approach, as several successful peptide drugs have been reported recently.<a onclick="showRef(event, 'ref57 ref58 ref59'); return false;" href="javascript:void(0);" class="ref ref57 ref58 ref59">(57−59)</a></div><div class="NLM_p">Interestingly, access to such venom-containing folded peptides, which may be considered as “mini-proteins” covering a diversity of tridimensional structures, has recently been published using a very innovative approach. Knowledge from natural peptides originating from animal venoms, microorganisms, and plants has been coupled with high throughput de novo design of hyperstable constrained peptides, leading to a large number of novel and structurally constrained peptidic structures.<a onclick="showRef(event, 'ref60'); return false;" href="javascript:void(0);" class="ref ref60">(60)</a></div><div class="NLM_p last">Overall, the strategy of screening collections of constrained peptides offers great potential toward the design of folded peptidic drugs for difficult targets with the significant advantage of chemical accessibility. It remains best suited for extracellular or membrane-bound targets as cell penetration of such cyclic peptides is still often difficult, even if examples of cell-penetrating peptides originating from venoms have been reported.<a onclick="showRef(event, 'ref61'); return false;" href="javascript:void(0);" class="ref ref61">(61)</a></div></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i6" class="anchor-spacer"></div><h3 class="article-section__title" id="_i6">2.3.  Screening Large Number of Compounds with DNA-Encoded Libraries</h3><div class="NLM_p">The concept of screening large libraries of compounds, particularly through mixtures, has been the basis of the split-and-pool methodology and combinatorial synthesis developed in the 1990s. One of the main issues with this strategy was the identification of the active component(s) within a mixture. Various encoding methods were evaluated, but practical application of the generally required solid-phase synthesis was quite cumbersome, including long library synthesis development and an uncertain hit identification process.<a onclick="showRef(event, 'ref62'); return false;" href="javascript:void(0);" class="ref ref62">(62)</a> The development of in vitro directed evolution methods for the synthesis of peptides is at the origin of the use of DNA sequences as encoding methodology associated with the combinatorial split-and-mix synthesis (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>).<a onclick="showRef(event, 'ref63'); return false;" href="javascript:void(0);" class="ref ref63">(63)</a></div><figure id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b00378/20181019/images/medium/jm-2018-00378e_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b00378/20181019/images/large/jm-2018-00378e_0005.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Principle of DELs.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b00378/20181019/images/large/jm-2018-00378e_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00378&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Compared to other encoding methods, combinatorial synthesis is performed in solution and its use is typically limited to aqueous media and mild pH due to the limitations associated with DNA solubility and stability in reaction conditions and solvents. The design and synthesis of large DNA-encoded libraries (DEL) of 10<sup>6</sup>–10<sup>9</sup> members have been described using a rather limited set of reactions tolerated by the encoding tag.<a onclick="showRef(event, 'ref64 ref65'); return false;" href="javascript:void(0);" class="ref ref64 ref65">(64,65)</a> However, despite this limitation, several examples of successful hit finding, sometimes for challenging biological targets, have been published.<a onclick="showRef(event, 'ref66'); return false;" href="javascript:void(0);" class="ref ref66">(66)</a> As an example, DEL libraries were screened against the receptor interaction protein 1 (RIP1) kinase and led to nanomolar benzoxazepinone hits that were optimized into the clinical candidate <b>3</b> for inflammatory diseases (GSK2982772) (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>).<a onclick="showRef(event, 'ref67'); return false;" href="javascript:void(0);" class="ref ref67">(67)</a> It is noteworthy that an HTS was also performed in parallel to the DEL screening and provided another lead series possessing a different binding mode but which was deprioritized due to lack of oral exposure. This example therefore demonstrates the complementarity of the DEL strategy.</div><figure id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b00378/20181019/images/medium/jm-2018-00378e_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b00378/20181019/images/large/jm-2018-00378e_0006.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Examples of hits and probes identified by DEL.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b00378/20181019/images/large/jm-2018-00378e_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00378&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The benefit of DEL relies primarily on the size of the libraries compared to regular HTS screening of about 10<sup>6</sup> diverse chemical structures. Enhancing diversity of DNA-compatible reaction types would strengthen the added value of DEL, but library diversity may be increased with proprietary scaffolds and building blocks to build and decorate libraries. Novel strategies, such as the use of hexathymidine oligonucleotides, are also being developed to enable acidic catalysts and therefore increase diversity.<a onclick="showRef(event, 'ref68'); return false;" href="javascript:void(0);" class="ref ref68">(68)</a></div><div class="NLM_p">Another limitation of DEL is the need to immobilize the biological target in order to identify potential binders within a mixture. Consequently, only soluble proteins can robustly be screened using affinity selection. Membrane proteins, such as GPCRs, known for their low expression, are less suited, although cellular systems have been reported.<a onclick="showRef(event, 'ref69'); return false;" href="javascript:void(0);" class="ref ref69">(69)</a> One strategy to circumvent this challenge is to produce membrane proteins in soluble formats by using either cell-free production systems or by cell lysis. In both cases, detergents, liposomes, nanodiscs, or lipoparticles may be used to stabilize the protein in the absence of its membrane-stabilizing environment. Several technologies and know-how have been under development in academia and biotechnology companies with some success, including for GPCRs.<a onclick="showRef(event, 'ref70 ref71 ref72'); return false;" href="javascript:void(0);" class="ref ref70 ref71 ref72">(70−72)</a> Combination of cutting-edge technologies in affinity selection and protein production is therefore a winning combination to identify valuable binders for difficult proteins.</div><div class="NLM_p">Several biotechnology companies offer the DEL technology, including X-Chem, Nuevolution, Philochem, Vipergen, Dice, or HitGen, using slightly different methods of encoding and chemical synthesis but all with the objective to build large and diverse combinatorial libraries. The majority of large pharmaceutical companies have signed deals with at least one of the specialized DEL companies. Some have even internalized the technology, either by company acquisition (GSK with Praecis)<a onclick="showRef(event, 'ref73'); return false;" href="javascript:void(0);" class="ref ref73">(73)</a> or through technology transfer (Novartis with Nuevolution, and recently AstraZeneca with X-Chem), which highlights the recognized value of such a technology as part of the hit finding toolbox.</div><div class="NLM_p">DELs also offer the additional opportunity to identify allosteric sites through exposure to a large number of compounds. Several examples of new binding modes to known targets have been reported,<a onclick="showRef(event, 'ref74 ref75 ref76 ref77 ref78'); return false;" href="javascript:void(0);" class="ref ref74 ref75 ref76 ref77 ref78">(74−78)</a> which represents a particularly attractive way to identify functional effectors quite impossible to rationally design in the absence of useable structural knowledge of the target. An additional opportunity is the design of libraries of reactive molecules, often fragments, designed to covalently bind label free cysteines.<a onclick="showRef(event, 'ref79'); return false;" href="javascript:void(0);" class="ref ref79">(79)</a> In addition, the scope of DELs expands beyond lead generation and may be used in chemical biology for target validation and study of biological mechanisms.<a onclick="showRef(event, 'ref80'); return false;" href="javascript:void(0);" class="ref ref80">(80)</a> A recent example illustrating this use was the discovery of isoform-selective ATAD2 inhibitors and the chemical probe <b>4</b> (BAY-850) showing the unexpected property of inducing ATAD2 bromodomain dimerization (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>).<a onclick="showRef(event, 'ref75'); return false;" href="javascript:void(0);" class="ref ref75">(75)</a> This chemical probe represents an invaluable tool to better understand ATAD2 biological role.</div><div class="NLM_p last">Overall, every protein target difficult to drug using classical small molecule HTS represents a favorable application of DEL, particularly when dealing with a soluble protein and which can be effectively tagged and made suitable for immobilization on beads. Application is not limited to challenging targets, and the approach is also valuable for classical targets, representing a complementary strategy to other hit finding strategies. Importantly, this technology aims at identifying binders, and functional assays are therefore always needed to profile the hits. The process takes a few months from inception to the first hits. Historically, access to the technology has been restricted to multitarget partnerships, but recently some contract research organizations (CROs) have started to offer the approach as a fee-for-service, which will undoubtedly be attractive for small companies.</div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i7" class="anchor-spacer"></div><h3 class="article-section__title" id="_i7">2.4.  Screening Libraries of Nonpeptidic Macrocycles</h3><div class="NLM_p">The size and molecular weight of macrocycles generally extend beyond the Lipinski rule-of-five boundary of 500 Da. This is clearly an advantage as it increases the possibility to bind a larger protein binding site compared to smaller size molecules, particularly considering the general flexibility of those large macrocycles favoring an induced-fit interaction.<a onclick="showRef(event, 'ref81'); return false;" href="javascript:void(0);" class="ref ref81">(81)</a> However, a balance between binding efficacy and favorable physicochemical and pharmacokinetic properties needs to be found to select a suitable molecule for clinical development. A recent analysis of macrocyclic compounds on the market or in development has shown that oral bioavailability can be obtained in the molecular weight range of 500–1000.<a onclick="showRef(event, 'ref82'); return false;" href="javascript:void(0);" class="ref ref82">(82)</a> However, a careful management of properties, such as lipophilicity and solubility, is necessary to identify a balanced profile. Furthermore, and importantly, a continuous progress in the knowledge of property- and structure-based modulation of membrane penetration is critical to produce new macrocyclic drugs for difficult targets that are not reachable or druggable with classical small molecules or antibodies.<a onclick="showRef(event, 'ref83'); return false;" href="javascript:void(0);" class="ref ref83">(83)</a></div><div class="NLM_p">Access to macrocyclic structures has been enabled by DEL-specialized biotechnology companies and academic groups,<a onclick="showRef(event, 'ref84 ref85 ref86 ref87'); return false;" href="javascript:void(0);" class="ref ref84 ref85 ref86 ref87">(84−87)</a> but screening collections of discrete macrocycles is also offered by a range of providers. Despite generally challenging chemistry, libraries of novel macrocyclic compounds have been proposed by specialized companies such as Asinex, Analyticon, Fidelta, and Polyphor or developed in house in major pharmaceutical companies as part of library enhancement initiatives to extend the chemical space of their screening sets. The promise of these libraries has already been shown and much more is expected,<a onclick="showRef(event, 'ref88 ref89 ref90'); return false;" href="javascript:void(0);" class="ref ref88 ref89 ref90">(88−90)</a> with several macrocyclic compounds targeting challenging proteins moving to the clinic, either originating from macrocyclic hits or designed during optimization. As a striking example, the macrocyclic inhibitor <b>5</b> (AMG-176) targeting the induced myeloid leukemia cell differentiation protein Mcl-1, previously known as an undruggable protein of the Bcl-2 family, has been reported to be in clinical phase 1 for hematologic malignancies (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>). It has previously shown efficacy at slowing down tumor growth following oral administration in multiple myeloma xenografts.<a onclick="showRef(event, 'ref91'); return false;" href="javascript:void(0);" class="ref ref91">(91)</a></div><figure id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b00378/20181019/images/medium/jm-2018-00378e_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b00378/20181019/images/large/jm-2018-00378e_0007.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Examples of macrocycles in clinical phase.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b00378/20181019/images/large/jm-2018-00378e_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00378&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Another example of macrocycle in clinical development is the third-generation ALK inhibitor lorlatinib <b>6</b> (PF-06463922), which displayed impressive efficacy in reversing resistance against crizotinib and other ALK inhibitors (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>).<a onclick="showRef(event, 'ref92'); return false;" href="javascript:void(0);" class="ref ref92">(92)</a></div><div class="NLM_p last">Overall, macrocycles libraries are well suited for challenging targets, especially PPIs, and many examples of cell permeable macrocycles demonstrate they are an appropriate choice for intracellular targets. While macrocyclization restricts the number of conformations, probing in theory a smaller portion of the chemical space, this is mitigated by the adaptability of targets upon binding.<a onclick="showRef(event, 'ref81'); return false;" href="javascript:void(0);" class="ref ref81">(81)</a> For most companies, this screening strategy will have to rely on establishing partnerships or fee-for-service screening with specialized providers.</div></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i8" class="anchor-spacer"></div><h3 class="article-section__title" id="_i8">2.5.  Identifying Modulators of Protein Functions through Phenotypic Screening</h3><div class="NLM_p">With lack of translation of target-based drug discovery still representing a major reason for attrition from preclinical to clinical phases,<a onclick="showRef(event, 'ref93'); return false;" href="javascript:void(0);" class="ref ref93">(93)</a> a renaissance of phenotypic approaches has been witnessed over the last years.<a onclick="showRef(event, 'ref94 ref95'); return false;" href="javascript:void(0);" class="ref ref94 ref95">(94,95)</a> Indeed, increased access to patient tissue samples offers the opportunity to screen libraries of compounds for new phenotypes. This lead generation strategy is in theory fully target-agnostic, although few companies tend to be prepared to optimize hits and eventually leads without target identification.<a onclick="showRef(event, 'ref94'); return false;" href="javascript:void(0);" class="ref ref94">(94)</a> Nevertheless, the approach can enable identification of novel targets as well as molecules driving a desired effect through polypharmacology. Phenotypic screening is crucial in the field of anti-infectives<a onclick="showRef(event, 'ref96 ref97'); return false;" href="javascript:void(0);" class="ref ref96 ref97">(96,97)</a> and particularly relevant to the challenges faced with regenerative medicine, where biology is complex and often not well understood. In this respect, development of high content assays is enabling phenotypic assays, with multiple end points providing the opportunity to dissect phenotypes of interest.<a onclick="showRef(event, 'ref98'); return false;" href="javascript:void(0);" class="ref ref98">(98)</a> The increasing use of induced pluripotent stem (iPS) cells enables screening in a setting that may more accurately replicate specific disease phenotypes compared to, for example, immortalized cell lines,<a onclick="showRef(event, 'ref99'); return false;" href="javascript:void(0);" class="ref ref99">(99)</a> a consideration that can be further expanded to organoids, as illustrated with iPS-derived cardiac organoids.<a onclick="showRef(event, 'ref100'); return false;" href="javascript:void(0);" class="ref ref100">(100)</a> Moreover, more complex readouts such as quantitative polymerase chain reaction (qPCR) can also be incorporated.<a onclick="showRef(event, 'ref101'); return false;" href="javascript:void(0);" class="ref ref101">(101)</a></div><div class="NLM_p">From a compound collection standpoint, the more translational approaches are typically lower throughput and consequently dictate screening a limited number of compounds, typically up to a few thousand. Thus, chemogenomic libraries consisting of annotated compounds known to interact with specific targets are often used.<a onclick="showRef(event, 'ref102 ref103'); return false;" href="javascript:void(0);" class="ref ref102 ref103">(102,103)</a> However, these approaches inherently reduce the opportunity of identifying novel targets since current knowledge in terms of probes represent only a few percent of the human proteome.<a onclick="showRef(event, 'ref104'); return false;" href="javascript:void(0);" class="ref ref104">(104)</a> A balance between translatability and throughput has therefore to be struck. In particular, the reliance of artificial stimuli to mimic disease state (for example chemically induced) is often misleading and results in the identification of stimulus-specific hits which do not translate to patients. In addition, target deconvolution remains a major endeavor that hinders the relevance of screening large number of compounds to identify novel targets. Thus, acceptance of the lower throughput but more translational screens should still be embraced. Nevertheless, efforts have been directed at expanding the chemical space used in phenotypic screening including secreted proteins and natural products.<a onclick="showRef(event, 'ref105'); return false;" href="javascript:void(0);" class="ref ref105">(105)</a> A significantly different approach to identifying novel targets through phenotypic screening was developed by leveraging the ability of fragments to inherently have a higher probability of binding to proteins compared to larger entities.<a onclick="showRef(event, 'ref106'); return false;" href="javascript:void(0);" class="ref ref106">(106)</a> By functionalizing fragments with photoaffinity probes and analyzing cells with proteomics, novel binders to proteins could be identified and the approach was in particular applied to adipocyte differentiation.<a onclick="showRef(event, 'ref106'); return false;" href="javascript:void(0);" class="ref ref106">(106)</a></div><div class="NLM_p last">Overall, phenotypic screening offers many opportunities to identify modulators of protein function, but a focus on the most relevant approaches drives high costs. Importantly, the recognition that the scope of targets prosecuted through phenotypic screening is intrinsically wider is growing with the identification of target modulation at the RNA level, as will be discussed in <a class="ref internalNav" href="#sec5_2" aria-label="section 5.2">section 5.2</a>.</div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i9">3.  Expanding Diversity for Structure-Agnostic Screening through New Partnerships</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_48355" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_48355" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">High throughput screening remains a very valid and robust strategy to identify suitable starting points for drug discovery.<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> Increasing diversity and quality of small molecules in screening collections is however required to offer the best target space coverage and the best lead-like compounds.<a onclick="showRef(event, 'ref23 ref90 ref107 ref108'); return false;" href="javascript:void(0);" class="ref ref23 ref90 ref107 ref108">(23,90,107,108)</a> Historically, the enhancement of screening collections has been performed through compound acquisition from chemical suppliers and brokers.<a onclick="showRef(event, 'ref109 ref110'); return false;" href="javascript:void(0);" class="ref ref109 ref110">(109,110)</a> However, this model requires a large budget to cover a sufficiently large and diverse number of compounds.<a onclick="showRef(event, 'ref111'); return false;" href="javascript:void(0);" class="ref ref111">(111)</a> Another model exploited by several major pharmaceutical companies has been to outsource the synthesis of small molecule libraries designed by chemists of the ordering company. Here again, a very large budget is required to cover the cost of synthesis. Nevertheless, it allows internalization of compounds specifically designed for underexplored target classes or molecules that are mimetics of natural molecules, e.g., carbohydrates and nucleosides, generally poorly represented in screening collections, as well as in vendors’ offers. AstraZeneca exploited this model over a period of 10 years in order to add more than 600000 novel compounds to the compound collection.<a onclick="showRef(event, 'ref93 ref112'); return false;" href="javascript:void(0);" class="ref ref93 ref112">(93,112)</a></div><div class="NLM_p">New business models aiming at accessing valuable chemicals bringing novelty and complementarity to internal compound collections have been developed recently. The objective has been to internalize large sets of novel compounds for screening at a much lower cost and possibly more efficiently than classical purchasing. Several approaches, broadly covered under the umbrella of “open innovation”, have been evaluated and implemented as is illustrated in the rest of this section (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>).</div><figure id="fig8" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b00378/20181019/images/medium/jm-2018-00378e_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b00378/20181019/images/large/jm-2018-00378e_0008.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Overview of historical and new business models for expanding compound collections.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b00378/20181019/images/large/jm-2018-00378e_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00378&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div id="sec3_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i10" class="anchor-spacer"></div><h3 class="article-section__title" id="_i10">3.1.  Accessing Larger Collections through Consortia</h3><div class="NLM_p last">The European Lead Factory (ELF) is a public-private partnership initiated in 2013 and involves seven pharmaceutical companies, five CROs, and 10 academic institutes over a 5-year period.<a onclick="showRef(event, 'ref113'); return false;" href="javascript:void(0);" class="ref ref113">(113)</a> The project is aimed at screening 300000 industrial compounds and 200000 newly designed academic compounds against academic and industry member targets.<a onclick="showRef(event, 'ref113'); return false;" href="javascript:void(0);" class="ref ref113">(113)</a> The first compound library was built from the pharmaceutical companies’ proprietary collections by selecting up to 50000 molecules each, covering complementary diversity and lead-like properties.<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> This initial 300000 collection may be screened against up to 24 academic targets per year and against an agreed number of internal targets for each pharmaceutical member. A second collection designed by academic and CRO members of the consortium has been implemented progressively during the five years of the project life thanks to an ambitious synthetic program at the CROs and with the end-goal of 200000 compounds.<a onclick="showRef(event, 'ref114'); return false;" href="javascript:void(0);" class="ref ref114">(114)</a> The advantage for the pharmaceutical members has been 2-fold: a “give one–get nine” chemicals to screen against internal targets and an access to novel academic targets when public target programs are proposed to industrial partners following successful hit identification. For academics, the value of the consortium has been the access to a high quality screening collection and the potential to identify an industrial partner for their innovative biology. Importantly, all screenings are structure-blinded and a structure-clearance process was defined at the start of the initiative to contractually frame IP rights on disclosed structures. The consortium is also beneficial to CROs, including through the development of a strong network with academic partners, the design and implementation of innovative chemistries, as well as an enhancement of the production of compounds necessary to build the new compound collection. The benefits gained by each partner therefore demonstrate the value of such a consortium.</div></div><div id="sec3_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i11" class="anchor-spacer"></div><h3 class="article-section__title" id="_i11">3.2.  Increasing the Size of Screening Collections through Compound Swaps</h3><div class="NLM_p">A next level of open innovation has been to swap a defined number of compounds and amount of material between two partners. This type of deal has been undertaken between two companies working in two different business areas, e.g., human health and crop protection, with each receiving freedom to patent and develop in their own field when interesting molecules originating from the partner are identified. This type of deal involves mainly chemoinformatic and logistic resources in order to select the best complementary chemical space and compounds adding value to the collection of the receiving party.<a onclick="showRef(event, 'ref115'); return false;" href="javascript:void(0);" class="ref ref115">(115)</a> Furthermore, and contrary to the previous consortium model, chemical structures are disclosed from the start and the amount of material to exchange is negotiated.</div><div class="NLM_p">This compound swap can reach an even higher stage of “openness” by performing it between two competitors within the same business area. Such a deal was effectively signed in 2015 by Sanofi and AstraZeneca with the exchange of 200000 compounds.<a onclick="showRef(event, 'ref116'); return false;" href="javascript:void(0);" class="ref ref116">(116)</a> The two partners accepted the risk of potential competition on the same targets as well as potential loss of IP for the compounds in the exchanged set. This apparent weakness is however outweighed by the opportunity to identify valuable hits from a collection dramatically enhanced by the partner’s set. This deal demonstrates the fundamental principles of open innovation: “to give to better receive”. Clearly, appreciating the level of risk and the benefit/risk ratio are cornerstones to the initiation of innovative business models.</div><div class="NLM_p last">Another approach has been the reciprocal blinded access to the compound collection of two competitors. In this so-called “boomerang” model, Bayer and AstraZeneca can screen a proprietary target on the compound collection of the partner in exchange for the same in the opposite direction, essentially representing a “target swap”.<a onclick="showRef(event, 'ref117'); return false;" href="javascript:void(0);" class="ref ref117">(117)</a> A prerequisite for a screen to be endorsed, is for the nominated target not to be relevant to the partner. Considering the very small overlap between the two collections, approximatively 3%, this innovative model creates a pool of greater than 4.2 million compounds.<a onclick="showRef(event, 'ref118'); return false;" href="javascript:void(0);" class="ref ref118">(118)</a></div></div><div id="sec3_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i12" class="anchor-spacer"></div><h3 class="article-section__title" id="_i12">3.3.  Relaxed Boundaries for Compound Sharing</h3><div class="NLM_p last">Systemic pressure to optimize R&D cost has pushed some pharmaceutical companies to outsource the management of their collection to specialized CROs possessing cutting-edge robotic equipment and facilities.<a onclick="showRef(event, 'ref119'); return false;" href="javascript:void(0);" class="ref ref119">(119)</a> This strategic evolution is facilitating the implementation of innovative business models by streamlining the flow of compounds between collaboration partners. A typical example is the deal between Sanofi and the academic French National Compound Library (Chimiothèque Nationale).<a onclick="showRef(event, 'ref120'); return false;" href="javascript:void(0);" class="ref ref120">(120)</a> Compounds from this library originate from over 40 academic groups are now housed by Evotec, who manages and distributes the compounds for the academic members. In exchange, Sanofi has access to compounds from this library for structure-blinded screening under a risk-sharing contract.<a onclick="showRef(event, 'ref121'); return false;" href="javascript:void(0);" class="ref ref121">(121)</a></div></div><div id="sec3_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i13" class="anchor-spacer"></div><h3 class="article-section__title" id="_i13">3.4.  Risk-Sharing and Collection Leasing Partnerships</h3><div class="NLM_p">Risk-sharing and library leasing partnerships represent another model of compound access, which usually relies on success-based milestones and redefines the traditional value chain. This approach rewards the success related to the identification of high value hits coming from structure-blinded screening against portfolio targets. Negotiated access fees are paid to chemical providers for a very small quantity of selected compounds bringing value to existing collections. These compounds are screened for a given period of time during which process and prenegotiated milestones related to high quality hit identification are executed following contractually agreed terms. The benefit and value for chemical providers depends on an estimation of the number of screenings, which will include the provider’s compounds and the expected hit rate. Screening of these external collections is usually conducted in parallel to internal collections depending on target types and capacities. An analysis of the outcome of seven partnerships at Sanofi has shown a similar hit rate to the internal diversity-based collection but with the great advantage of identifying chemotypes complementary to internal ones. Importantly, competition between the various hit sources has required education of project teams to consider all the sources at a comparable level of potential value. The role of the “firewall” scientists between partners is then critical to an efficient and successful implementation, as this risk-sharing model of compound access is definitely more a research collaboration than pure procurement.</div><div class="NLM_p last">Importantly, this model is not limited to small molecules, and collections of other modalities can be considered. For example, natural products and mimetics, macrocycles, glycomimetics, glycoconjugates, nucleosides, and others may be involved in such a model for screening against relevant biological targets, as is illustrated with AstraZeneca screening libraries of glycomimetics from Alchemia (now VAST Bioscience).</div></div><div id="sec3_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i14" class="anchor-spacer"></div><h3 class="article-section__title" id="_i14">3.5.  Crowdsourcing</h3><div class="NLM_p">Another creative way to access innovation is to ask “the crowd” for proposals around a specific need. A few companies focusing on crowdsourcing strategies have been developing dedicated portals or expert networks over the past decade, including Innocentive, NineSigma, and Biowebspin. The objective is to make a community aware of a requester’s need in order to get proposals from solvers unknown to the requester or unexpected. Such a type of challenge can either be focused on a specific solution to solve a problem, generally financially rewarded, or a request for a partner possessing expertise, tools, and/or technologies to initiate collaboration. This call for partners can be dedicated to a specific class of modalities such as macrocycles, as published on the Innocentive portal in April 2016.<a onclick="showRef(event, 'ref122'); return false;" href="javascript:void(0);" class="ref ref122">(122)</a> This innovative way to access expertise and solutions presents several advantages: (1) a quite short and simple process, (2) a complementary approach to other sourcing methods based on scouting techniques such as the use of personal networks, participation to conferences and business meetings, or in silico queries, and (3) access to potential partners really willing to collaborate as they respond positively to a call. In this specific case of a call for partners on macrocycles, it was possible to identify innovation in academia, only partly published, but more importantly ready for industrialization through the creation of start-ups looking for pharma partners.</div><div class="NLM_p last">Overall, the innovative partnerships presented in this section are aimed at accessing novel chemical matter and represent a complementary approach to the classical screening of internal collections. To select which collection to screen, the potential added-value of each set and their complementarity, as well as the capacity and cost of the screening, have to be considered in the context of each target.</div></div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i15">4.  Structure-Enabled Probe and Lead Generation for Direct Modulation of Protein Functions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_45615" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_45615" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Possessing structural information on a target considerably widens the range of possible hit finding approaches. For example, the structure of ligand–receptor or more generally PPI interfaces can be used to quickly derive a peptidic starting point. More generally, structural information can be used for diversity-based approaches, in particular fragment-based screening, which has been a major pillar of structure-based lead generation, principally using NMR or X-ray. Chemists are directed to detailed reviews in this field,<a onclick="showRef(event, 'ref123 ref124'); return false;" href="javascript:void(0);" class="ref ref123 ref124">(123,124)</a> and this approach is particularly well suited for classical targets. Virtual screening is also enabled through structure, and progress in in silico technologies is increasing its relevance as a potential systematic hit finding strategy. The need for structure generation is implicit and obvious for structure-enabled approaches. Therefore, developments in emerging technologies in this field, such as cryo-electron microscopy (cryo-EM) and free electron laser (FEL) are also discussed in this section due to their implication for challenging targets.</div><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i16" class="anchor-spacer"></div><h3 class="article-section__title" id="_i16">4.1.  Generation of Starting Points through Protein Structure Mimetics</h3><div class="NLM_p">PPIs offer the unique opportunity to derive a starting point by directly mimicking one of the binding partners and stabilizing it.<a onclick="showRef(event, 'ref125'); return false;" href="javascript:void(0);" class="ref ref125">(125)</a> The principle can leverage either secondary structures such as α-helices and β-sheets or tertiary structures by combining several units of secondary structures mimetics. In particular, motifs such as helix–loop–helix, loop-strand–helix–loop, loop–strand–helix, or coiled-coil are being increasingly investigated,<a onclick="showRef(event, 'ref126 ref127 ref128'); return false;" href="javascript:void(0);" class="ref ref126 ref127 ref128">(126−128)</a> although the approach may remain reduced to one helix of these tertiary motifs.<a onclick="showRef(event, 'ref129'); return false;" href="javascript:void(0);" class="ref ref129">(129)</a></div><div class="NLM_p">α-Helices represent about 60% of the key interactions at protein–protein interfaces, making them a preferred motif to mimic.<a onclick="showRef(event, 'ref130'); return false;" href="javascript:void(0);" class="ref ref130">(130)</a> Two main approaches have been developed to stabilize such helices. Stapled peptides consist in cross-linking two amino acid residues located on the same side of the helix, usually through ring-closing metathesis performed on an α-methylated olefin located on the side chain of amino acids (<a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a></a>a). The cross-link can either constitute a simple α-helix stabilizer, make additional contacts at the interface, or be leveraged for significant interaction at the interface.<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> For example, starting from a fragment of the MAML1 protein, a known antagonist of NOTCH signaling,<a onclick="showRef(event, 'ref131'); return false;" href="javascript:void(0);" class="ref ref131">(131)</a> the stapled peptide 7 displaying 94% helicity bound the NOTCH transactivation complex with a <i>K</i><sub>D</sub> of 120 nM and abolished NOTCH signaling, leading to inhibition of cell proliferation in T-ALL cells (<a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">10</a></a>). Furthermore, in vivo efficacy through dose-dependent tumor regression was demonstrated in mice in a model of leukemia. Other examples of probes developed against notoriously difficult targets include β-catenin<a onclick="showRef(event, 'ref132'); return false;" href="javascript:void(0);" class="ref ref132">(132)</a> and a range of PPIs.<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> Further stabilization of α-helices can be achieved by a double cross-link<a onclick="showRef(event, 'ref133'); return false;" href="javascript:void(0);" class="ref ref133">(133)</a> and are sometimes referred as “stitched” peptides (<a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a></a>a).<a onclick="showRef(event, 'ref134'); return false;" href="javascript:void(0);" class="ref ref134">(134)</a></div><figure id="fig9" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b00378/20181019/images/medium/jm-2018-00378e_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b00378/20181019/images/large/jm-2018-00378e_0009.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. Protein structure mimetics strategies. (a) stabilization of α-helices, (b) stabilization of β-hairpins, (c) stabilization of loops.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b00378/20181019/images/large/jm-2018-00378e_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00378&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig10" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 10</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b00378/20181019/images/medium/jm-2018-00378e_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b00378/20181019/images/large/jm-2018-00378e_0010.jpeg" id="gr10" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 10. Examples of protein-structure mimetics. Color coding for amino acids: light blue = β-amino acid, purple = <span class="smallcaps smallerCapital">d</span>-amino acid, and orange = unnatural amino acid used for ring closure metathesis. X indicates an amino acid whose identity has not been released.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b00378/20181019/images/large/jm-2018-00378e_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00378&amp;id=fig10"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Another approach utilizes α-helices inducers such as proline. Other modifications relying, for example, on β-amino acids can achieve similar success (<a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a></a>a).<a onclick="showRef(event, 'ref135'); return false;" href="javascript:void(0);" class="ref ref135">(135)</a> In this particular case, the approach can be systematic, progressively scanning a sequence with repeats of an ααβ or αααβ pattern. For example, peptide hormones can be turned into so-called β-peptide foldamers. The repeat of an αααβ pattern was used in the sequence of parathyroid hormone PTH1(1–34) to generate a novel agonist of PTH1-R.<a onclick="showRef(event, 'ref136'); return false;" href="javascript:void(0);" class="ref ref136">(136)</a> This modification stabilized the α-helical structure of PTH1(1–34) and retained cAMP activity as exemplified by peptide <b>9</b> (<a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">10</a></a>). In addition, the αααβ repeat resulted in increased proteolytic resistance and duration of action. Interestingly, in the context of GPCRs, introduction of β-amino acids can induce signaling bias as was shown in the case of the glucagon-like receptor 1 GLP1R.<a onclick="showRef(event, 'ref137'); return false;" href="javascript:void(0);" class="ref ref137">(137)</a></div><div class="NLM_p">β-Sheets are another secondary structure which can be effectively stabilized. The β-strand–turn−β-strand can be cyclized through interchain cross-links or through N-to-C terminus cyclization with a turn motif such as a <span class="smallcaps smallerCapital">d</span>-proline–<span class="smallcaps smallerCapital">l</span>-proline-, biaryl-, and diketopiperazine-based systems (<a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a></a>b) and is sometimes referred as protein epitope mimetic (PEM).<a onclick="showRef(event, 'ref138'); return false;" href="javascript:void(0);" class="ref ref138">(138)</a> One prominent example is the mimicking of the antibacterial peptide protegrin I.<a onclick="showRef(event, 'ref139'); return false;" href="javascript:void(0);" class="ref ref139">(139)</a> The macrocyclization strategy followed by optimization led to potent peptides against several strains of <i>Pseudomonas</i> with strong selectivity against other bacteria as illustrated by peptide <b>8</b> (<a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">10</a></a>). The overall profile enabled clinical evaluation, with a phase III clinical study due to commence in 2018, demonstrating the strong potential of the approach. The β-hairpin stabilization has been applied to a range of biological targets, including proteases and chemokine receptors.<a onclick="showRef(event, 'ref140 ref141'); return false;" href="javascript:void(0);" class="ref ref140 ref141">(140,141)</a></div><div class="NLM_p">While α-helices and β-sheets have been extensively pursued, loops represent an underexplored secondary motif. An analysis of interfaces revealed a propensity of loop motifs at several PPIs, which now offers the possibly to publicly search loop motifs with the LoopFinder algorithm.<a onclick="showRef(event, 'ref142 ref143'); return false;" href="javascript:void(0);" class="ref ref142 ref143">(142,143)</a> Loop motifs can be classically cyclized by forming a disulfide bridge (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>c). One example is a cyclic peptide aimed at interacting with the TEAD transcription factor in the Hippo pathway, at the interface with one of its coactivator, YAP. Transcription factors are notorious for being “undruggable”, which calls for the use of different modalities. The PPI structure reveals the presence of an omega loop in YAP at the interface with TEAD, and this loop was selected for cyclization and optimization.<a onclick="showRef(event, 'ref144 ref145'); return false;" href="javascript:void(0);" class="ref ref144 ref145">(144,145)</a> Mutation of three key residues with noncanonical amino acids and disulfide cyclization afforded the lead peptide <b>10</b> with an IC<sub>50</sub> of 25 nM (<a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">10</a></a>). However, such cyclization induces an inherent metabolic soft spot through the disulfide. Therefore, an alternative strategy consists of cyclizing with reactive small molecule scaffolds via covalent formation with cysteine and analogues such as penicillamine (<a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a></a>c). The approach was applied to disrupt the interaction between stonin-2 and the epidermal growth factor substrate 15 (Ebs15) protein, which may reduce the Ebola virus infectivity. A hot loop from the stonin-2 containing 10 amino acids was flanked with cysteine analogues and cyclized with a set of dibromomethyl aromatic systems to generate a range of macrocycles, including peptide <b>11</b>.<a onclick="showRef(event, 'ref143'); return false;" href="javascript:void(0);" class="ref ref143">(143)</a> The best analogues bound Ebs15 with submicrolar affinity (compared with 18 μM for the uncyclized loop) (<a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">10</a></a>).</div><div class="NLM_p">Pushing the concept of secondary structure stabilization further, emerging approaches are directed toward stabilizing tertiary structures. Such strategies are extremely attractive as they may deliver mini-proteins with antibody-like affinity but with much reduced molecular weight allowing for deeper tissue penetration and even potentially cell permeability in some cases. Using a computational approach, hyperstable peptides containing cross-links were designed to afford a range of tertiary protein structures.<a onclick="showRef(event, 'ref146'); return false;" href="javascript:void(0);" class="ref ref146">(146)</a> Remarkably, X-ray and NMR structures were accurately predicted by the in silico workflow, which will enable the design of tailored tertiary structures directed at specific biological targets.</div><div class="NLM_p last">Overall, these approaches can systematically and rapidly deliver starting points and are best suited for challenging extracellular and membrane-bound targets. They can also be effectively combined with other hit finding strategies.<a onclick="showRef(event, 'ref147'); return false;" href="javascript:void(0);" class="ref ref147">(147)</a> While intracellular delivery can in some cases be achieved, in particular with stapled peptides,<a onclick="showRef(event, 'ref148'); return false;" href="javascript:void(0);" class="ref ref148">(148)</a> this generally remains a challenge which still limits the potential for intracellular targets. As mentioned in <a class="ref internalNav" href="#sec2_1" aria-label="section 2.1">section 2.1</a>, an acceleration in the understanding of cellular uptake may progressively change this paradigm.<a onclick="showRef(event, 'ref45 ref46 ref47 ref48'); return false;" href="javascript:void(0);" class="ref ref45 ref46 ref47 ref48">(45−48)</a> Nevertheless, one fundamental point to consider in the context of probe and lead generation is the opportunity to use these protein structure mimetics as reporters for larger screens. Peptides can, for example, be tagged with a fluorophore and then used in high throughput screening in a competition fluorescence polarization (FP) assay, potentially leading to novel small molecule hits with more optimizable properties.</div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17">4.2.  Generation of Stabilized Peptidic Starting Points through Scaffold Grafting</h3><div class="NLM_p">Alongside secondary and tertiary structure, unstructured peptide epitopes can also constitute starting points and be cyclized through a disulfide bridge as mentioned in <a class="ref internalNav" href="#sec4_1" aria-label="section 4.1">section 4.1</a>. An original alternative is to combine the binding affinity of the peptide epitope with a larger scaffold displaying high stability. This category of scaffolds is represented by cysteine-rich peptides present in many plants and organisms and which display chemical, serum, and proteolytic stability. Several scaffolds exist including cyclotides,<a onclick="showRef(event, 'ref149'); return false;" href="javascript:void(0);" class="ref ref149">(149)</a> sunflower trypsin inhibitor,<a onclick="showRef(event, 'ref150'); return false;" href="javascript:void(0);" class="ref ref150">(150)</a> and conotoxins.<a onclick="showRef(event, 'ref151'); return false;" href="javascript:void(0);" class="ref ref151">(151)</a> To leverage these scaffolds, parts of a sequence of one of the loops of a cysteine-rich peptide is replaced by a peptide epitope sequence (<a data-tab="pane-pcw-Figures" href="#fig11" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig11" id="" class=" internalNav">11</a></a>).<a onclick="showRef(event, 'ref152 ref153 ref154'); return false;" href="javascript:void(0);" class="ref ref152 ref153 ref154">(152−154)</a> This grafting process enables novel lead structures for a given biological target with both affinity and suitable properties. Typically, up to 10–15 amino acids can be grafted into a single loop. Beyond developing novel and stable binders, this strategy can also enable selectivity across isoforms. For example, a pentapeptide binder to the melanocortin receptor (MC-R) family was grafted into the sixth loop of the cyclotide kalata B1 scaffold.<a onclick="showRef(event, 'ref155'); return false;" href="javascript:void(0);" class="ref ref155">(155)</a> Remarkably, analogue <b>12</b> displayed high affinity for the MC4 receptor with significant selectivity against the other isoforms 1, 2, and 5 (100–300-fold) (<a data-tab="pane-pcw-Figures" href="#fig12" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig12" id="" class=" internalNav">12</a></a>). Cellular penetration remains a challenge, but grafting of a p53 peptide epitope into a cyclotide scaffold led to a 2 nM binder to HDM2 and displayed cellular activity in the micromolar range in a p53 deficient cell line,<a onclick="showRef(event, 'ref156'); return false;" href="javascript:void(0);" class="ref ref156">(156)</a> demonstrating the potential of the grafting concept for intracellular PPIs.</div><figure id="fig11" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Figure 11</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b00378/20181019/images/medium/jm-2018-00378e_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b00378/20181019/images/large/jm-2018-00378e_0011.jpeg" id="gr11" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 11. Principle of scaffold grafting</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b00378/20181019/images/large/jm-2018-00378e_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00378&amp;id=fig11"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig12" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Figure 12</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b00378/20181019/images/medium/jm-2018-00378e_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b00378/20181019/images/large/jm-2018-00378e_0012.jpeg" id="gr12" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 12. Example of cyclotide (Kalata B1) with a grafted MCR4 sequence.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b00378/20181019/images/large/jm-2018-00378e_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00378&amp;id=fig12"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">Overall, this approach is well suited to stabilize a known peptide epitope, derived for example from a ligand or binding partner. Due to the current limitation in cell permeability, the approach is best suited for challenging extracellular and membrane-bound targets, particularly when achieving selectivity is a key requirement. The approach can be demanding from a synthetic standpoint and requires specialist knowledge, which mostly comes from expert academic groups.</div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18">4.3.  Massive Virtual Screening</h3><div class="NLM_p">As a complementary strategy to HTS, virtual screening (VS) offers the promise of rapidly identifying potential de novo small molecule binders to a protein. VS can be broadly divided into structure-based VS (SBVS), which relies on docking putative ligands, and ligand-based VS (LBVS), which focuses on two-dimensional fingerprint or on three-dimensional pharmacophores and shape matching. For LBVS, only a ligand is required to perform the search. In the context of unprecedented targets, it therefore mainly applies to cases where a natural ligand, including cofactors and peptide, is already known.</div><div class="NLM_p">Historically, SBVS has suffered from limited throughput (virtual screens in the region of 10<sup>6</sup> molecules maximum), partially as a consequence of limited computing power, as well as challenges in reliable scoring functions. With computing limitations progressively vanishing and the rise of machine learning algorithms,<a onclick="showRef(event, 'ref157'); return false;" href="javascript:void(0);" class="ref ref157">(157)</a> in addition to novel scoring methodologies,<a onclick="showRef(event, 'ref158 ref159'); return false;" href="javascript:void(0);" class="ref ref158 ref159">(158,159)</a> the question of library size and composition is becoming central. Since the size of small molecule chemical space is vast at over 10<sup>63</sup> drug-like small molecules,<a onclick="showRef(event, 'ref160'); return false;" href="javascript:void(0);" class="ref ref160">(160)</a> rational selection approaches have to be considered,<a onclick="showRef(event, 'ref161'); return false;" href="javascript:void(0);" class="ref ref161">(161)</a> in particular since hits have to be synthesized to confirm their predicted activity. One approach consists in selecting commercially available compounds. For example, a large VS was performed on millions of molecules to identify inhibitors of EGFR kinase and BRD4.<a onclick="showRef(event, 'ref162'); return false;" href="javascript:void(0);" class="ref ref162">(162)</a> Another strategy consists in enumerating synthetically accessible de novo molecules<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> and libraries up to 10<sup>20</sup>, but typically more in the region of 10<sup>12</sup>–10<sup>15</sup> can be generated today.<a onclick="showRef(event, 'ref163 ref164 ref165 ref166'); return false;" href="javascript:void(0);" class="ref ref163 ref164 ref165 ref166">(163−166)</a> On the basis of this strategy, scientists at Lilly designed a “proximal” collection of molecules representing the in-house rapidly synthesizable chemical space.<a onclick="showRef(event, 'ref167'); return false;" href="javascript:void(0);" class="ref ref167">(167)</a> The focus was put on robustness of chemical reactions and access from trusted vendors, leading to libraries typically in the region of 10<sup>7</sup>–10<sup>9</sup> and up to 10<sup>11</sup>. Using this approach, selective inhibitors of the RIO2 kinase were identified. Specialized CROs and biotechnologies companies in the VS area include Exscientia, Relay Therapeutics, and Nimbus Therapeutics.</div><div class="NLM_p last">Overall, the VS strategy is still best suited for classical targets, with well-defined binding sites. In theory, a large screen can be performed in days to generate virtual hits. An important consideration with increased throughput and library sizes is the rapid access to hits from a synthetic standpoint. The power of the strategy can only be truly leveraged if many hits can be synthesized to confirm them. Therefore, synthetic technologies become an integral part of a massive VS paradigm. For example, establishing an expedited synthesis-purification platform can, beyond reducing hit optimization cycle time,<a onclick="showRef(event, 'ref168 ref169'); return false;" href="javascript:void(0);" class="ref ref168 ref169">(168,169)</a> be leveraged as a versatile approach to synthesize large numbers of hits generated from massive VS.<a onclick="showRef(event, 'ref170'); return false;" href="javascript:void(0);" class="ref ref170">(170)</a> While the ultimate development of VS coupled to machine learning algorithms should progressively reduce the number of false virtual hits, a burden on synthesis will remain as not every hit will likely be accessible through an automated platform.</div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i19" class="anchor-spacer"></div><h3 class="article-section__title" id="_i19">4.4.  Emerging Structural Technologies Enabling Structure-Enabled Lead Generation</h3><div class="NLM_p">As mentioned previously, the need for structure generation is implicit for all structure-enabled hit finding approaches. However, for numerous proteins, especially membrane proteins, generating structural information has proven challenging. Perhaps as a consequence, and despite the large numbers of GPCRs in the human genome, the number of GPCRs targets in the pipeline of pharmaceutical companies has tended to decrease over the past decade. While class A GPCRs have seen many successes with screening strategies for small molecules, delivering drugs, for example, for the adrenergic and opioid receptors, prosecuting class B and C GPCRs have resulted in limited success with small molecules. These receptors are often either orphan or in many cases their native ligands are large peptides, often calling for allosteric modulation when small molecules are chosen as a modality. Targeting allosteric modulation is extremely challenging, especially in absence of structure, but the development of new technologies over the past few years is shedding light on structural characteristics of such targets. For example, stabilization of the receptor with the StaR technology from Heptares Therapeutics has enabled the first crystal structure of prominent targets bound to agonists or antagonists such as the GLP1 class B GPCR.<a onclick="showRef(event, 'ref171 ref172'); return false;" href="javascript:void(0);" class="ref ref171 ref172">(171,172)</a> In a fascinating example, a previously known antagonist of the glucagon receptor was shown to bind to a novel allosteric binding site. The MK-0893 small molecule bound the receptor between the sixth and seventh transmembrane helix, likely resulting in prevention of G-protein coupling and an antagonistic effect.<a onclick="showRef(event, 'ref173'); return false;" href="javascript:void(0);" class="ref ref173">(173)</a> The technology has brought stronger structural understanding on several other receptors, including CCR9,<a onclick="showRef(event, 'ref174'); return false;" href="javascript:void(0);" class="ref ref174">(174)</a> metabotropic glutamate receptors,<a onclick="showRef(event, 'ref175'); return false;" href="javascript:void(0);" class="ref ref175">(175)</a> the corticotropin releasing factor 1,<a onclick="showRef(event, 'ref176'); return false;" href="javascript:void(0);" class="ref ref176">(176)</a> and protease activated receptor 2.<a onclick="showRef(event, 'ref177'); return false;" href="javascript:void(0);" class="ref ref177">(177)</a> Beyond developing understanding, the approach opens up the possibility for fragment-based lead generation, although examples are yet to be published.</div><div class="NLM_p">Another strategy consists in stabilizing conformations linked to specific signaling pathways. The use of nanobodies has proven particularly successful to stabilize, for example, an active state of the β2 adrenergic receptor,<a onclick="showRef(event, 'ref178'); return false;" href="javascript:void(0);" class="ref ref178">(178)</a> and of its complex with the Gs protein,<a onclick="showRef(event, 'ref179'); return false;" href="javascript:void(0);" class="ref ref179">(179)</a> or for studying conformational changes during μ-opioid receptor activation.<a onclick="showRef(event, 'ref180'); return false;" href="javascript:void(0);" class="ref ref180">(180)</a> The approach of stabilizing pathway-specific conformations with nanobodies is being pursued by Confo Therapeutics to enable fragment-based screening of cryptic pockets, as exemplified with the β2-adrenergic receptor.<a onclick="showRef(event, 'ref181'); return false;" href="javascript:void(0);" class="ref ref181">(181)</a> In general, these structural approaches, aimed at generating starting points with bias toward a given signaling pathway through either experimental or virtual screening, may expand therapeutic opportunities of drugging GPCRs.<a onclick="showRef(event, 'ref182'); return false;" href="javascript:void(0);" class="ref ref182">(182)</a></div><div class="NLM_p">With the pursuit of novel structures to enable fragment-based lead generation and more generally structure-based drug design (SBDD), novel technologies have emerged which expand the scope of tractable proteins from a structural standpoint. Notably, advances in cryo-EM over the past decade have increased the relevance of this technology to drug discovery. Structures of increasingly smaller protein size and better resolution have been enabled<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> for a range of biologically relevant targets including enzymes and ion channels.<a onclick="showRef(event, 'ref183 ref184 ref185 ref186 ref187'); return false;" href="javascript:void(0);" class="ref ref183 ref184 ref185 ref186 ref187">(183−187)</a> Typically, proteins with a lower limit of 64 kDa and a resolution of around 3 Å can be generated,<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> which, however, remains insufficient to enable SBDD in most cases. Going beyond the isolated examples of high resolution structures such as glutamate dehydrogenase,<a onclick="showRef(event, 'ref184'); return false;" href="javascript:void(0);" class="ref ref184">(184)</a> and higher throughput, will transform the relevance of cryo-EM to lead generation for virtual screening, epitope mimetics, and even potential fragment screening. In particular, the technique being particularly well suited for membrane proteins, promises of opening new opportunities in this field are high.</div><div class="NLM_p">Free electron X-ray laser (FEL) is another emerging technology.<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> The principle leverages short and intense flashes of X-ray radiation at room temperature and may enable not only high resolution structure for proteins but also study of binding events. Indeed, the brilliance, which relates to the concentration of photons, is significantly higher than the current most advanced synchrotron by a factor of up to 10<sup>9</sup>.<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> Moreover, the technology enables data acquisition on small and weaker diffracting crystals compared to other X-ray methodologies. Overall, data is acquired rapidly (femtoseconds) on tens of thousands of crystals, allowing time-resolved investigation of conformational changes, which is not possible with cryogenic methods, which tend to freeze conformations.<a onclick="showRef(event, 'ref188'); return false;" href="javascript:void(0);" class="ref ref188">(188)</a> Examples of structures generated with this technology include the angiotensin II receptor with crystals almost 3000 times smaller than those used to acquire a structure with similar resolution with cryo methods,<a onclick="showRef(event, 'ref189'); return false;" href="javascript:void(0);" class="ref ref189">(189)</a> the CPV17 polyhedrin,<a onclick="showRef(event, 'ref190'); return false;" href="javascript:void(0);" class="ref ref190">(190)</a> as well as membrane proteins such as ATPases,<a onclick="showRef(event, 'ref191'); return false;" href="javascript:void(0);" class="ref ref191">(191)</a> proton channels,<a onclick="showRef(event, 'ref192'); return false;" href="javascript:void(0);" class="ref ref192">(192)</a> the smoothened receptor,<a onclick="showRef(event, 'ref193'); return false;" href="javascript:void(0);" class="ref ref193">(193)</a> and the serotonin 5-HT2B receptor.<a onclick="showRef(event, 'ref194'); return false;" href="javascript:void(0);" class="ref ref194">(194)</a> Since access to FELs remains very limited, this promising technology has yet to demonstrate concrete examples in lead generation, although it is being pursued commercially, with for example LeadXPro.</div><div class="NLM_p last">Overall, these structural technologies are a field to monitor for medicinal chemists as advances against the current limitations are likely to progressively enable relevant structure and therefore expand the scope of structure-enabled targets and the associated lead generation strategies.</div></div></div><div id="sec5" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i20">5.  Probe and Lead Generation to Modulate Protein Levels</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_44363" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_44363" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Modulating proteins levels can be achieved through three different MOAs. At the earlier stage of protein production, DNA may now be targeted with precise genome editing, in particular through the CRISPS/CAS9 system (<a data-tab="pane-pcw-Figures" href="#fig13" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig13" id="" class=" internalNav">13</a></a>).<a onclick="showRef(event, 'ref7 ref8 ref9'); return false;" href="javascript:void(0);" class="ref ref7 ref8 ref9">(7−9)</a> In short, an RNA sequence usually referred as “guide” recognizes the gene to be modified which is then cleaved off by the CAS9 endonuclease leading to a knockdown. Novel development in the technology has also enabled subsequent insertion of a gene. Two key challenges need to be addressed for this modality: delivery/distribution and safety (specificity). In addition, concerns related to the innate immune response against the CAS protein have been raised recently.<a onclick="showRef(event, 'ref195'); return false;" href="javascript:void(0);" class="ref ref195">(195)</a> Delivery usually relies on viral vectors which have a capacity limitation and may therefore trigger immune response from the resulting high quantity of virus needed. Furthermore, this approach is to date limited to few tissues: liver, muscle, and eye. Despite being considered as cell therapy, medicinal chemists are encouraged to monitor developments and build awareness in this area since this modality may deliver very relevant probes for target validation.</div><figure id="fig13" data-index="13" class="article__inlineFigure"><h2 class="fig-label">Figure 13</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b00378/20181019/images/medium/jm-2018-00378e_0013.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b00378/20181019/images/large/jm-2018-00378e_0013.jpeg" id="gr13" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 13. MOAs of modalities regulating protein levels.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b00378/20181019/images/large/jm-2018-00378e_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00378&amp;id=fig13"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">At the second level, protein levels may be regulated through modulation of transcription and though up- or down-regulation of translation by interfering at different points of RNA regulation processes and modulate splicing (<a data-tab="pane-pcw-Figures" href="#fig13" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig13" id="" class=" internalNav">13</a></a>). This may be achieved through several oligonucleotides approaches but also with classical small molecules and more recently with CRISPR.<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> At the third and last level, once the protein has actually been expressed, modulation of protein levels may be achieved by inducing protein degradation (<a data-tab="pane-pcw-Figures" href="#fig13" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig13" id="" class=" internalNav">13</a></a>). Hit finding for these different MOAs is described in this section.</div><div id="sec5_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i21" class="anchor-spacer"></div><h3 class="article-section__title" id="_i21">5.1.  Modulation of Transcription and Translation with Oligonucleotides</h3><div class="NLM_p">From a lead generation perspective, oligonucleotides differ significantly from other modalities since developing a hit or lead originates from a targeted RNA sequence, incorporating several modified nucleotides to increase their stability (<a data-tab="pane-pcw-Figures" href="#fig14" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig14" id="" class=" internalNav">14</a></a>). Oligonucleotides can be divided in several categories. Importantly, different types of oligonucleotides enable targeting of RNA at different points of the regulation machinery (<a data-tab="pane-pcw-Figures" href="#fig13" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig13" id="" class=" internalNav">13</a></a>). Briefly, antisense oligonucleotides (ASOs) and small interfering RNA (siRNA) can be aimed at splicing correction by blocking a specific sequence, but also downregulation of a protein at the mRNA level, either via direct interaction with mRNA or via the RISC complex. ASOs and siRNA differ significantly: ASOs are single-stranded oligonucleotides which are typically composed of 16–20 nucleotides, while siRNA are made of 20–25 oligonucleotides and are double stranded. In addition, siRNA act specifically via the RISC complex. The majority of ASOs target directly mRNA, but this modality may also be directed at or mimic the so-called micro RNAs (miRNAs). miRNA are noncoding RNA species which bind to the argonaut protein to form the RISC complex, which then recognizes a complementary mRNA sequence and is followed by degradation of the mRNA by RNases (<a data-tab="pane-pcw-Figures" href="#fig13" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig13" id="" class=" internalNav">13</a></a>). On one hand, when mimicking a miRNA, ASOs form part of the RISC complex in place of the miRNA and therefore promote mRNA degradation. On the other hand, and very interestingly, developing an ASO with a complementary sequence to a miRNA, blocks its function, and therefore results in an up-regulation of mRNA and consequently of the encoded protein. Essentially, this means that ASOs can be used for both down- and up-regulating protein levels, although intervention at the miRNA remains an underdeveloped area of research. Several ASOs are either marketed, including fomivirsen, mipomersen, eteplirsen, and nusinersen, or in clinical development. siRNA has experienced less success, although patisiran is due to become the first RNA inference drug to be approved.</div><figure id="fig14" data-index="14" class="article__inlineFigure"><h2 class="fig-label">Figure 14</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b00378/20181019/images/medium/jm-2018-00378e_0014.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b00378/20181019/images/large/jm-2018-00378e_0014.jpeg" id="gr14" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 14. Examples of modified nucleotides. Modification of the carbohydrate: MOE = methoxyethyl; cEt = constrained ethyl; LNA = locked nucleic acid. The nucleobase is represented in green, and the replacement of the phosphate group by a thiophosphate is highlighted in light blue.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b00378/20181019/images/large/jm-2018-00378e_0014.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00378&amp;id=fig14"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Separately, modified RNA (modRNA) brings a different paradigm to drug discovery by increasing levels of downregulated proteins. An oligonucleotide composed of thousands of modified nucleotides is produced biosynthetically with RNA polymerases. The resulting RNA can be transfected and then be translated into a native or modified protein of interest intracellularly. The technology is still emerging although several programs have entered the clinic, and the most advanced AZD8601 is due to enter phase IIa in 2018 in heart failure patients undergoing cardiac bypass grafting (CABG) surgery.</div><div class="NLM_p">From a lead generation point of view, oligonucleotides are very different from modalities directly modulating protein functions. Being derived from the sequence of the targeted mRNA, they indeed afford systematically lead sequences through an established workflow, which aims at either accomplishing sequence complementarity via base-pairing or sequence mimicry. Essentially, a rule-based approach is used for ASOs to design compounds that must target unstructured regions. Indeed, the potential formation of secondary and tertiary RNA structure may render a particular sequence inaccessible to a complementary oligonucleotide sequence. For this reason, hundreds of sequences are typically screened, albeit also looking at safety markers. Interestingly, the approach is very fast, with a typical target selection to clinical candidate cycle time of 18 months in the case of ASOs.</div><div class="NLM_p last">Overall, oligonucleotide approaches enable modulation of protein levels through different points of action. Lead generation follows a rule-based strategy, derived from sequences, and these approaches are particularly well suited for challenging intracellular targets. Importantly, the MOA means that any scaffolding function is removed through the induced protein knockdown. Modified RNA is uniquely positioned for up-regulation of protein levels as well as for the expression of intracellular antibodies. Importantly, these technologies are highly complex and access is only possible through partnerships with specialized companies such as Ionis Pharmaceuticals (formerly known as Isis Pharmaceuticals), Alnylam, Moderna, Ethris, or Curevac. One important limitation is tissue access through systemic delivery (typically liver, spleen, and kidney), implying in some cases a need for local delivery. In addition, some cell types are refractory to oligonucleotide uptake.</div></div><div id="sec5_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i22" class="anchor-spacer"></div><h3 class="article-section__title" id="_i22">5.2.  Small Molecule Phenotypic Approaches to Modulate Translation</h3><div class="NLM_p">Historically, several small molecules have been known to interfere with DNA, including intercalants and alkylating agents. However, this approach is rather gene-unspecific and leads to general cytotoxicity or genotoxicity. A more precise field of intervention, which can directly affect protein levels is at the mRNA and noncoding RNA level. RNA can indeed form secondary, tertiary, and quaternary structures, including loops, hairpins, and bulges, which can be seen as pockets, much like proteins. In addition, RNA interacts with protein creating protein–RNA interfaces and creating small molecule binding sites. Consequently, small molecules may well be a viable modality for interacting with RNA to control biological function.<a onclick="showRef(event, 'ref196 ref197 ref198'); return false;" href="javascript:void(0);" class="ref ref196 ref197 ref198">(196−198)</a></div><div class="NLM_p">The first examples of lead generation for targeting RNA have predominantly been the results of phenotypic assays directed to specific cellular events. For example, targeting the splicing process of the survival of motor neurons (SMN) protein in the context of spinal muscular atrophy (SMA), a cell-based HTS was performed using a SMN2 gene reporter assay.<a onclick="showRef(event, 'ref199'); return false;" href="javascript:void(0);" class="ref ref199">(199)</a> Hits were confirmed by dose response and qPCR to confirm splicing activity, together with an ELISA assay to assess SMN protein levels. One scaffold, based on pyridazines, was optimized and led to compounds with a cellular activity of 5–20 nM, as exemplified by compound <b>13</b> (<a data-tab="pane-pcw-Figures" href="#fig15" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig15" id="" class=" internalNav">15</a></a>). Further evaluation in vivo, in a model of SMA, demonstrated dose-dependent increase in the levels of the SMN protein both in the brain and spinal cord. Using the most efficacious compound, the mode-of-action could be elucidated and was shown to involve specific splice variants. More specifically, the small molecule stabilized a protein–RNA complex consisting of the U1 small nuclear ribonucleic protein (snRNP) from the spliceosome and the SMN2 pre-mRNA. Interestingly, another small molecule scaffold was also reported to modify SMN2 splicing, although the mode of action was not elucidated at the time.<a onclick="showRef(event, 'ref200'); return false;" href="javascript:void(0);" class="ref ref200">(200)</a></div><figure id="fig15" data-index="15" class="article__inlineFigure"><h2 class="fig-label">Figure 15</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b00378/20181019/images/medium/jm-2018-00378e_0015.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b00378/20181019/images/large/jm-2018-00378e_0015.jpeg" id="gr15" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 15. Examples of small molecules binders to RNA.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b00378/20181019/images/large/jm-2018-00378e_0015.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00378&amp;id=fig15"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">In another example of phenotypic screening, a cell-based HTS was performed to identify modulators of PCSK9 secretion from CHO cells, a key target involved in the regulation of cholesterol levels with implication in cardiovascular diseases and obesity.<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> While displaying only micromolar activity, the reported hit <b>14</b> was not cytotoxic and did not affect secretion of a control protein, pointing to a specific effect (<a data-tab="pane-pcw-Figures" href="#fig15" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig15" id="" class=" internalNav">15</a></a>). The inhibitor was subsequently shown to affect the level of the PCSK9 protein by reducing its synthesis at the translational level and could be linked to inhibition of the 80S ribosome. Interestingly, the inhibitor was also shown to be orally active in vivo in rats, reducing plasma PCSK9 and cholesterol levels.<a onclick="showRef(event, 'ref201'); return false;" href="javascript:void(0);" class="ref ref201">(201)</a></div><div class="NLM_p last">Overall, the PCKS9 example is particularly exciting in the context of novel projects. Efforts to identify small molecule direct binders and effectors of PCSK9 have repeatedly failed, and the development of now marketed antibodies have been a more logical and successful approach. However, aiming at phenotypically modulating PCSK9 levels as was performed by the team at Pfizer opens up the thinking that PPIs including secreted proteins may be targetable with small molecules via modulation of translation. Consequently, a paradigm-shifting hit finding strategy would be to regularly implement phenotypic screening for identifying different modulators of protein levels perhaps in parallel to target-based screening on that specific protein.</div></div><div id="sec5_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i23" class="anchor-spacer"></div><h3 class="article-section__title" id="_i23">5.3.  Modulation of Translation with Small Molecules through Direct RNA Targeting</h3><div class="NLM_p">The previous examples highlight the potential of interacting with some RNA species or complexes as a result of phenotypic approaches. A range of approaches and assays has been developed to identify interactions of molecules with RNA, but this has to-date not being truly amenable to HTS of small molecules.<a onclick="showRef(event, 'ref202 ref203'); return false;" href="javascript:void(0);" class="ref ref202 ref203">(202,203)</a> Rationally targeting RNA requires novel approaches as robust HTS assays are lacking, although a new strategy has recently been published. One approach relies on so-called two-dimensional combinatorial screen (2DCS) in a library-versus-library selection strategy (<a data-tab="pane-pcw-Figures" href="#fig16" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig16" id="" class=" internalNav">16</a></a>).<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> Agarose-coated small molecules microarrays are screened against libraries of RNA, and bound RNA can be isolated by gel excision and analyzed by reverse transcription PCR and sequencing. To deconvolute RNA sequences, a statistical method, called structure–activity relationships through sequencing (StARTS) is applied, leading to small molecule–RNA secondary structure relationships. Importantly, this screening strategy is target-agnostic and can be seen as knowledge generation, which can be mined once a specific RNA sequence is selected as a target. The resulting database becomes a hit finding strategy dubbed Inforna.<a onclick="showRef(event, 'ref204'); return false;" href="javascript:void(0);" class="ref ref204">(204)</a> For example, the approach was leveraged to identify small molecule binders of a toxic CUG RNA repeat responsible for myotonic dystrophy type 1.<a onclick="showRef(event, 'ref205'); return false;" href="javascript:void(0);" class="ref ref205">(205)</a> A broader application has been developed to target miRNA, which are key regulators of coding RNA (<a data-tab="pane-pcw-Figures" href="#fig13" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig13" id="" class=" internalNav">13</a></a>). Mature miRNAs are generated after processing by the Dicer and Drosha enzymes and pri- and pre-miRNA exhibit structures targetable by small molecules at the Dicer and Drosha cleaving sites. Following the Inforna approach, binders of the oncogenic miRNA18a resulted in upregulation of the STK4 kinase and subsequent apoptosis.<a onclick="showRef(event, 'ref206'); return false;" href="javascript:void(0);" class="ref ref206">(206)</a> Similarly, a miRNA210 binder could be identified and was shown to up-regulate the GDP1L enzyme and resulted in reduction in HI1a levels and induction of apoptosis in a breast cancer cell line.<a onclick="showRef(event, 'ref207'); return false;" href="javascript:void(0);" class="ref ref207">(207)</a></div><figure id="fig16" data-index="16" class="article__inlineFigure"><h2 class="fig-label">Figure 16</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b00378/20181019/images/medium/jm-2018-00378e_0016.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b00378/20181019/images/large/jm-2018-00378e_0016.jpeg" id="gr16" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 16. Principle of 2D combinatorial libraries and inforna.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b00378/20181019/images/large/jm-2018-00378e_0016.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00378&amp;id=fig16"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Overall, the field of hit finding for direct RNA targeting is in its infancy. Other examples exist in the field for antibiotics, as illustrated by a specific screening strategy to target riboswitches.<a onclick="showRef(event, 'ref208'); return false;" href="javascript:void(0);" class="ref ref208">(208)</a></div><div class="NLM_p last">Considering the presumed highly flexible and dynamic structure of RNA, structure-based VS approaches are likely to remain challenging, although examples exist.<a onclick="showRef(event, 'ref209'); return false;" href="javascript:void(0);" class="ref ref209">(209)</a> There is a clear need for novel, especially high throughput, screening and structural approaches. Very recently, an assay based on affinity-selection mass spectrometry was developed specifically for RNA.<a onclick="showRef(event, 'ref210'); return false;" href="javascript:void(0);" class="ref ref210">(210)</a> Using this so-called automated ligand detection system (ALIS) approach, 53000 compounds were screened against a range of bacterial riboswitches and led to the identification of selective small molecules binders. More developments like this latest example are needed. In fact, more basic research is needed to leverage the potential of targeting RNA with synthetic molecules and develop novel lead generation approaches. Novel collaborations will therefore likely be required, as illustrated by the Chemical Genomics Centre of the Max Planck Society in Dortmund (Germany), which plans to investigate the druggability of RNA and RNA–protein complexes from 2018 onward. Beyond the interest in large pharmaceutical companies, several biotechnology companies have been created around this field including PTC Therapeutics, Arrakis Therapeutics, Nymirium, and very recently Expansion Therapeutics. Notably, the nature of the interactions with RNA may provide opportunities for small peptides and hybrid molecules to achieve higher level of selectivity.</div></div><div id="sec5_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i24" class="anchor-spacer"></div><h3 class="article-section__title" id="_i24">5.4.  Direct Modulation of Protein Levels through Induction of Degradation</h3><div class="NLM_p">Inducing protein degradation to modulate protein levels represents an emerging MOA,<a onclick="showRef(event, 'ref211 ref212 ref213'); return false;" href="javascript:void(0);" class="ref ref211 ref212 ref213">(211−213)</a> and the main approach is based on small molecules forming a new modality called proteolysis targeting chimera (PROTAC).<a onclick="showRef(event, 'ref214 ref215'); return false;" href="javascript:void(0);" class="ref ref214 ref215">(214,215)</a> Two ligands, one aimed at binding an E3 ligase and the other aimed at binding a protein of interest, are connected via a linker. Consequently, a PROTAC essentially brings together the two proteins, leading to ubiquitination of the targeted protein and its subsequent degradation by the proteasome (<a data-tab="pane-pcw-Figures" href="#fig17" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig17" id="" class=" internalNav">17</a></a>). Two E3 ligase binders have repeatedly been used with thalidomide derivatives targeting the cereblon (CRBN) E3 ubiquitin ligase and small molecules ligands targeting von Hippel–Lindau (VHL) E3 ligase. For example, the VHL ligand was connected to binders of the TBK1 kinase, varying the length of the linker between the two entities.<a onclick="showRef(event, 'ref216'); return false;" href="javascript:void(0);" class="ref ref216">(216)</a> A 12 nM PROTAC <b>15</b> could be identified, which achieved degradation of TBK1 while achieving selectivity against related kinases (<a data-tab="pane-pcw-Figures" href="#fig18" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig18" id="" class=" internalNav">18</a></a>). Considering the size of the chimera, cell permeability could be perceived as a challenge, but cellular activity has repeatedly been observed for many PROTACs. Since the degradation process is catalytic, only low intracellular concentrations are required to achieve degradation. One important consideration to keep in mind when selecting this MOA is the associated alleviation of any noncanonical and scaffolding function of the protein which is degraded.</div><figure id="fig17" data-index="17" class="article__inlineFigure"><h2 class="fig-label">Figure 17</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b00378/20181019/images/medium/jm-2018-00378e_0017.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b00378/20181019/images/large/jm-2018-00378e_0017.jpeg" id="gr17" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 17. Principle of PROTAC.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b00378/20181019/images/large/jm-2018-00378e_0017.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00378&amp;id=fig17"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig18" data-index="18" class="article__inlineFigure"><h2 class="fig-label">Figure 18</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b00378/20181019/images/medium/jm-2018-00378e_0018.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b00378/20181019/images/large/jm-2018-00378e_0018.jpeg" id="gr18" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 18. Examples of PROTACs. Color coding for amino acids: purple = <span class="smallcaps smallerCapital">d</span>-amino acid.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b00378/20181019/images/large/jm-2018-00378e_0018.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00378&amp;id=fig18"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The attractive feature of PROTAC lies in the mere requirement of a binder rather than a functional inhibitor. Thus, the complete panel of structure-agnostic and structure-enabled lead generation strategies, especially DELs, can be considered to quickly identify a binder. Interestingly, the ligand to the protein of interest does not have to be limited to small molecules, and stabilized peptides were recently used to degrade the estrogen receptor α.<a onclick="showRef(event, 'ref217'); return false;" href="javascript:void(0);" class="ref ref217">(217)</a> Furthermore, PROTAC <b>16</b> based on a peptidic binding ligand to the KEAP1 E3 ligase has also been reported (<a data-tab="pane-pcw-Figures" href="#fig18" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig18" id="" class=" internalNav">18</a></a>).<a onclick="showRef(event, 'ref218'); return false;" href="javascript:void(0);" class="ref ref218">(218)</a></div><div class="NLM_p last">Overall, when considering direct protein degradation as MOA and PROTAC as the corresponding modality, the whole range of hit finding technologies described in <a class="ref internalNav" href="#sec2" aria-label="sections 2">sections 2</a>, <a class="ref internalNav" href="#sec3" aria-label="3">3</a>, and <a class="ref internalNav" href="#sec4" aria-label="4">4</a> can be applied to identify binders to the protein of interest. It is noteworthy that several biotechnology companies have been founded around this concept, including Arvinas, C4 Therapeutics, and Kymera Therapeutics. An important additional step for identifying a probe or lead is however to successfully link the binder to an E3 ligase ligand. In this field, more research is required to robustly, rather than empirically, select the most effective linkers, and synergies with the field of drug conjugates for targeted delivery represent a vast opportunity to accelerate knowledge.</div></div></div><div id="sec6" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i25">6.  Selecting the “Right” Probe and Lead Generation Strategies</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_05457" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_05457" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec6_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i26" class="anchor-spacer"></div><h3 class="article-section__title" id="_i26">6.1.  A MOA-centric Framework</h3><div class="NLM_p">The various structure-agnostic and structure-enabled hit finding strategies together with sequence-based approaches for oligonucleotides provide a vast repertoire of techniques to identify starting points against many targets. As mentioned at the beginning of this perspective, medicinal chemists are presented with the opportunity to place MOA at the center of any strategy. We believe that this modality-agnostic, MOA-centric paradigm maximizes the relevance and rationale of the chosen hit finding strategies.</div><div class="NLM_p">Within this paradigm and the framework summarized on <a data-tab="pane-pcw-Figures" href="#fig11" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig11" id="" class=" internalNav">11</a></a>, biological knowledge on the target, pathway, and disease has to drive selection of the MOA(s) by first considering at which level the target should/could be modulated: DNA, RNA, or protein. In parallel, phenotypic approaches may be run in a target-agnostic or target-biased strategy. An associated consideration is whether the target is located intra- or extracellularly and if the target product profile (TPP) exclusively requires oral administration. Furthermore, the tissue where the target is located has to be considered since delivery of some modalities can be challenging (for example, ASOs predominantly distribute to the liver, spleen, and kidney). These parameters drive selection of the MOA(s), followed by selection of modalities, which clearly must be the result of a dialogue across the project team. Medicinal chemists must however be familiar with the different considerations in order to effectively influence decision-making.</div><div class="NLM_p">The next level of potential triaging is to determine whether structural information is available on the target or can be readily generated, either through homology models or through straightforward protein crystallization or NMR. This is of course most relevant for the MOAs based on protein binders. This stepwise approach unravels the range of possible hit finding strategies (<a data-tab="pane-pcw-Figures" href="#fig19" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig19" id="" class=" internalNav">19</a></a>). It is noteworthy that, and as mentioned previously, structure-enabled targets may leverage all hit finding approaches including those that do not require structural information such as HTS or DELs. The decision on which strategy(ies) to select out of the hit finding tool kit is then project- and institution-specific. Nevertheless, an educated choice can be made based on the strengths and limitations of each approach, and these are summarized in the next subsection.</div><figure id="fig19" data-index="19" class="article__inlineFigure"><h2 class="fig-label">Figure 19</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b00378/20181019/images/medium/jm-2018-00378e_0019.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b00378/20181019/images/large/jm-2018-00378e_0019.jpeg" id="gr19" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 19. Mode-of-action centric hit finding. DNA is transcribed into RNA, which is then translated into either an extracellular/membrane-bound protein or an intracellular protein. When the target modulation is agnostic and/or the target is unknown, phenotypic screening may be used directly as illustrated within the first level oval.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b00378/20181019/images/large/jm-2018-00378e_0019.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00378&amp;id=fig19"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">Importantly, the stepwise guide presented above should only be seen as a pragmatic and modular framework rather than a “one size fits all” process since specific targets, corporate resources, and capabilities, and competitive landscape are all factors influencing decision-making. Other factors driven by the disease to treat are also important for modality selection, including patient compliance and unmet medical need.</div></div><div id="sec6_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i27" class="anchor-spacer"></div><h3 class="article-section__title" id="_i27">6.2.  Strengths and Limitations of Hit Finding Strategies</h3><div class="NLM_p">The pros and cons of each probe and lead generation approach have been discussed in their respective sections. To summarize these, a heat map to guide selection of the most suited strategies of a target-specific hit finding toolkit is provided on <a data-tab="pane-pcw-Figures" href="#fig20" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig20" id="" class=" internalNav">20</a></a>. Some of the key criteria to take into account are the time from project inception to the first hits, the cost of running the approach, and/or to access it externally. In addition, the other criteria are directly aligned with the MOA-centric framework discussed in the previous subsection, in particular target localization (intra-, extracellular, or membrane-bound) and suitability for oral administration. Furthermore, proteins may be split into two categories: classical (many enzymes including kinases, to some extent ion channels, proteases, and class A GPCRs) and challenging (PPIs including E3 ligases and transcription factors, phosphatases, etc.).</div><figure id="fig20" data-index="20" class="article__inlineFigure"><h2 class="fig-label">Figure 20</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b00378/20181019/images/medium/jm-2018-00378e_0020.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b00378/20181019/images/large/jm-2018-00378e_0020.jpeg" id="gr20" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 20. Heat map for the strength and limitations of hit finding strategies and their associated modalities. *: Cell and tissue specific. P: Requires partnership with specialized company.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b00378/20181019/images/large/jm-2018-00378e_0020.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00378&amp;id=fig20"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">Taking DELs as an example (fifth row), the strengths and limitations are found from left to right on <a data-tab="pane-pcw-Figures" href="#fig20" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig20" id="" class=" internalNav">20</a></a>. DELs take a few months for hit identification (orange). In terms of costs, it usually requires a partnership (red P), although fee-for-service is emerging (orange). It is well suited for most targets, both intracellular (green) and extracellular (green). Membrane proteins are less desirable due to technical challenges (red). Challenging targets are a viable option (orange) depending on suitability of larger small molecules. It generates molecules that should be optimizable for oral delivery (green).</div></div><div id="sec6_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i28" class="anchor-spacer"></div><h3 class="article-section__title" id="_i28">6.3.  Phasing Hit Finding Approaches</h3><div class="NLM_p">Notably, several hit finding technologies and several modalities may be relevant for a given target. In some companies, there has been a tendency to either run multiple approaches in parallel or to only run one single approach. Rather than considering a once-for-all process selection but also a single modality track, there is an opportunity for a more agile and dynamic strategy, running multiple waves of hit finding at different time points. This may foster speed and maximize impact from knowledge generation, with each wave of hit finding informing the next one.<a onclick="showRef(event, 'ref219'); return false;" href="javascript:void(0);" class="ref ref219">(219)</a> Furthermore, one initial modality, for example, a modified peptide, may represent a chance to build further knowledge on a target by affording a probe for target validation and then be pursued as a front runner track. It may also be used to generate a reporter molecule for screening another modality such as small molecules. This overall integrated hit finding and integrated modality landscape approach maximizes the chances of finding relevant and multiple chemical series. To illustrate this concept, an example is presented on <a data-tab="pane-pcw-Figures" href="#fig21" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig21" id="" class=" internalNav">21</a></a>.</div><figure id="fig21" data-index="21" class="article__inlineFigure"><h2 class="fig-label">Figure 21</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b00378/20181019/images/medium/jm-2018-00378e_0021.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b00378/20181019/images/large/jm-2018-00378e_0021.jpeg" id="gr21" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 21. Example of phased hit finding waves integrating multiple strategies and modalities.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b00378/20181019/images/large/jm-2018-00378e_0021.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00378&amp;id=fig21"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">Overall, following the guiding principles of this modality-agnostic MOA-centric paradigm, novice and experienced medicinal chemists have the opportunity to make an informed decision for the right approaches and to enable robust probe and lead generation. Importantly, with knowledge across science expanding fast, regular updating of the state-of-the-art for each approach and modality has also to be captured and implemented to constantly adopt the most relevant strategies.</div></div></div><div id="sec7" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i29">7.  Outlook: Additional Opportunities for Probe and Lead Generation</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_81428" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_81428" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Armed with the knowledge from the previous sections, medicinal chemists may further expand their range of opportunities by considering the upcoming wave of novel hit finding approaches and technologies. In this respect, this final section illustrates the continuous flow of novel ideas to systematically find more and better starting points. As was discussed in <a class="ref internalNav" href="#sec3" aria-label="section 3">section 3</a>, the question of the ideal proprietary compound collection size resurfaces regularly. An HTS can typically screen 10<sup>6</sup> compounds and DNA-encoded libraries 10<sup>9</sup>. In comparison to the estimated drug-like chemical space of 10<sup>63</sup>, this does not even “scratch the surface”. Accessing different chemical space is therefore a recurrent theme, and progress in synthetic biology is enabling novel molecules to be generated. For example, nature is able to synthesize complex molecules including peptides without relying on the classical ribosomal machinery. The field of natural products typically results from the expression of enzymes carrying synthetic transformations, which are combined in a sequence. These so-called biosynthetic gene clusters can be mined to either exploit novel compounds synthesis from organisms or reshuffled and recombined to generate novel entities. For example, LifeMine Therapeutics is exploiting fungi and Lodo Therapeutics bacteria from soil. GyreOx is also generating collections of macrocyclic compounds by leveraging specific enzymes based on academic research.<a onclick="showRef(event, 'ref220'); return false;" href="javascript:void(0);" class="ref ref220">(220)</a> Excitingly, compound synthesis can be combined with a readout assay (linked to survival), also called a biosensor. One specific example is the discovery of novel Brome mosaic virus inhibitors at the company Evolva.<a onclick="showRef(event, 'ref221'); return false;" href="javascript:void(0);" class="ref ref221">(221)</a> Selection and recombination of biosynthetic gene clusters including alkaloid, flavonoid, and polyunsaturated fatty acid was performed to generate artificial chromosomes in yeast. Following transfection, yeast cells were subjected to the virus and survival clones were selected for deconvolution. Upon characterization, 74 new lead-like compounds could be identified and 28 showed activity in a secondary assay.</div><div class="NLM_p">Another powerful example for the discovery of cyclic peptides starting points is the intracellular, traceless method referred as SICLOPPS (split intein synthesis of proteins and peptides), which can display in the region of 10<sup>6</sup> cyclic peptides, typically 5- to 7-mers. SICLOPPS can be associated with the so-called reverse two-hybrid system (RTHS), which links cell survival to inhibition of a PPI of interest.<a onclick="showRef(event, 'ref222'); return false;" href="javascript:void(0);" class="ref ref222">(222)</a> The principle, which can also be extended to unnatural amino acids,<a onclick="showRef(event, 'ref223'); return false;" href="javascript:void(0);" class="ref ref223">(223)</a> has been applied to several PPIs, including to the hypoxia-inducible factor 1 (HIF-1) transcription factor heterodimer with the identification of a 124 nM binding affinity inhibitor.<a onclick="showRef(event, 'ref224'); return false;" href="javascript:void(0);" class="ref ref224">(224)</a></div><div class="NLM_p">To broadly explore chemical space governed by a biological target of interest, one proposal is to link mutation to the synthesis and assaying of the compound, in essence leveraging directed evolution.<a onclick="showRef(event, 'ref225'); return false;" href="javascript:void(0);" class="ref ref225">(225)</a> In such a paradigm, cells such as yeast are engineered to synthesize small molecules or peptides and are subjected to a selection pressure linked to a biological target of interest. Surviving cells are selected and subjected to mutations to improve the synthesized inhibitor, and this cycle of pressure, selection, and mutation is repeated until a potent inhibitor can be identified. While major advances will have to be seen, emerging examples in the field of proteins are illustrating the potential of the approach.<a onclick="showRef(event, 'ref226'); return false;" href="javascript:void(0);" class="ref ref226">(226)</a> Such a paradigm is likely to revolutionize hit finding for challenging targets.</div><div class="NLM_p">On the screening side, HTS is still widely used but presents the disadvantage of a labor intensive and generally over reductionist assay systems, typically not representative of the complexity of a pathological three-dimensional cell environment. Microfluidic cell chips are under development to mimic those environments at the microscale level. Several technical challenges on material, miniaturized devices still need to be overcome to make this technology the future screening process closer to the targeted disease.<a onclick="showRef(event, 'ref227'); return false;" href="javascript:void(0);" class="ref ref227">(227)</a></div><div class="NLM_p last">Finally, artificial intelligence (AI) is likely to significantly impact probe and lead generation in the coming years. Beyond the hype of the technology, the opportunity to broadly mine data to identify hits and also the possibility to further leverage virtual screening should provide a further extended set of hit finding strategies. AI is also being increasingly leveraged for synthesis planning, which should facilitate rapid synthesis of virtual hits.<a onclick="showRef(event, 'ref228 ref229'); return false;" href="javascript:void(0);" class="ref ref228 ref229">(228,229)</a> In addition, machine learning may enable the design of bifunctional molecules of great value for many diseases. The company Exscientia, in collaboration with Evotec, has recently presented the discovery of a small molecule simultaneously antagonizing the adenosine 2A receptor and inhibiting CD73.<a onclick="showRef(event, 'ref230'); return false;" href="javascript:void(0);" class="ref ref230">(230)</a></div></div><div id="sec8" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i30">8.  Conclusion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_51148" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_51148" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">With the rise of genomics and other target identification strategies, the need for probes and leads to validate and drug these targets is increasing dramatically. In this respect, the expanding range of approaches both for random screening and rational design are providing medicinal chemists with the opportunity to effectively identify starting points. Acceleration of knowledge across disciplines is also further expanding the hit finding toolkit, illustrated, for example, with the recently published expansion of the genetic code with two additional nucleobases which would further increase the high diversity of genetic encoding technologies including phage and mRNA display.<a onclick="showRef(event, 'ref231'); return false;" href="javascript:void(0);" class="ref ref231">(231)</a> Additional opportunities exist, including bringing together modalities, as illustrated by drug conjugates.<a onclick="showRef(event, 'ref232'); return false;" href="javascript:void(0);" class="ref ref232">(232)</a> From a decision-making standpoint, the opportunities to leverage protein degradation, to stabilize protein–protein interactions, and to activate pathways also trigger a need to revise screening strategies against novel targets. The possibility to address these additional MOAs, together with the opportunity to use other new modalities, calls indeed for a shift from an almost “all inhibition” mindset around targets with systematic hit finding strategies, to a modality-agnostic, MOA-centric hit finding paradigm. To exploit these expanded probe and lead generation opportunities, new models of collaboration and partnerships are required. Apart from very large pharmaceutical companies, it is unlikely that many companies can develop all capabilities in-house. Even in large pharmaceutical companies, boundaries are relaxing, with a redefinition of the precompetitive concept, illustrated by the exchange of compounds between AstraZeneca and Sanofi, an approach still inconceivable in company cultures focused on inward innovation. Enabling this MOA-centric paradigm requires overcoming several challenges, predominantly cultural. First, an outward-looking and networked innovation corporate culture is necessary, embracing the fact that expertise in specific areas likely sits outside the company and represents an opportunity to combine internal and external knowledge. Second, the expansion of drug modalities may require some level of re-skilling for drug discovery scientists to become fluent with these new types of molecules. Finally, the broadening of MOAs further blurs the boundaries between medicinal chemistry and chemical biology with the associated need of building a broader skillset beyond drug design. Seeing these challenges as an exciting opportunity to foster creativity, with new modalities, new technologies, and new partnerships, medicinal chemists are equipped like never before to tackle the challenges of answering increasingly complex biological questions and developing starting points toward new drugs.</div></div><div class="NLM_back"><div class="article_supporting-info" id=""></div><div class="testing" data-doi="10.1021/acs.jmedchem.8b00378" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_88106" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_88106" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Eric Valeur</span> - <span class="hlFld-Affiliation affiliation">Medicinal
Chemistry, Cardiovascular, Renal and Metabolism, IMED Biotech Unit, AstraZeneca, Pepparedsleden 1, Mölndal 431 83, Sweden</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-8270-9432" title="Orcid link">http://orcid.org/0000-0001-8270-9432</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#3b5e495258154d5a575e4e497b5a484f495a415e555e585a15585456"><span class="__cf_email__" data-cfemail="83e6f1eae0adf5e2efe6f6f1c3e2f0f7f1e2f9e6ede6e0e2ade0ecee">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Patrick Jimonet</span> - <span class="hlFld-Affiliation affiliation">External
Innovation Drug Discovery, Global Business Development & Licensing, Sanofi, 13 quai Jules Guesde, 94400 Vitry-sur-Seine, France</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#500031242239333b7e3a393d3f3e35241023313e3f36397e333f3d"><span class="__cf_email__" data-cfemail="acfccdd8dec5cfc782c6c5c1c3c2c9d8ecdfcdc2c3cac582cfc3c1">[email protected]</span></a></span></div></li></ul></li><li><ul></ul></li><li></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="bio" rid="ath1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="biographies">Biographies</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_62326" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_62326" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><figure id="ath1" data-index="22" class="article__inlineFigure"><h2 class="fig-label">Eric Valeur</h2><div class="figure-bottom-links"><div class="download-hi-res-img"></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00378&amp;id=BIO-d7e1531-autogenerated"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><b>Eric Valeur</b> is Director of New Modalities & Innovation, Medicinal Chemistry, at AstraZeneca, Sweden. Over his 15 years of drug discovery experience gained at Merck, Novartis, and AstraZeneca, he has led medicinal chemistry teams across several therapeutic areas and across modalities, including small molecules, macrocycles, peptides, oligonucleotides, drug conjugates, and PROTACs, delivering clinical candidates and novel technologies. He is passionate about challenging targets and disruptive drug discovery, including outwards-looking innovation partnerships, such as the AstraZeneca–Max Planck Institute Satellite Unit he currently leads and which is based in the group of Prof. Herbert Waldmann. Eric is coauthor/coinventor of over 50 publications, patents, posters, and oral presentations and holds a Ph.D. from the University of Edinburgh (under supervision of Prof. Mark Bradley) and an MBA from Imperial College London.</p></figure></div><div class="bio" rid="ath2"><figure id="ath2" data-index="23" class="article__inlineFigure"><h2 class="fig-label">Patrick Jimonet</h2><div class="figure-bottom-links"><div class="download-hi-res-img"></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00378&amp;id=BIO-d7e1536-autogenerated"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><b>Patrick Jimonet</b>’s present role at Sanofi is to bring external innovation to the early portfolio and the lead generation process; his particular focus is on modalities, technologies, and innovative biology through various and novel business models and partnerships. Previous assignments were the identification of key areas for improvement in early phases of discovery, leadership of a global interdisciplinary team responsible for the identification of novel GPCR targets and validated tool compounds, and manager of multicultural teams of Ph.Ds, project leader of projects from hit finding to clinical candidate. All activities have been performed at Sanofi and predecessor companies in France and in the US. Patrick received his Ph.D. in chemistry at the Institut de Chimie des Substances Naturelles of the CNRS in Gif-sur-Yvette, France.</p></figure></div><div class="ack" id="ack1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i33">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_37418" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_37418" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">We are grateful to Dr. Alleyn Plowright, Dr. Peter Hamley, and Dr. Andrew Davis for critical review of the manuscript and to Dr. Graham Smith and Dr. Samantha Hughes for providing critical inputs for the RNA and VS sections, respectively.</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations Used</td></tr><tr><td class="NLM_term">ALIS</td><td class="NLM_def"><p class="first last">automated ligand detection system</p></td></tr><tr><td class="NLM_term">ASO</td><td class="NLM_def"><p class="first last">antisense oligonucleotide</p></td></tr><tr><td class="NLM_term">cEt</td><td class="NLM_def"><p class="first last">constrained ethyl</p></td></tr><tr><td class="NLM_term">CETSA</td><td class="NLM_def"><p class="first last">cellular thermal shift assay</p></td></tr><tr><td class="NLM_term">CRO</td><td class="NLM_def"><p class="first last">contract research organization</p></td></tr><tr><td class="NLM_term">CRISPR</td><td class="NLM_def"><p class="first last">clustered regularly interspaced short palindromic repeats</p></td></tr><tr><td class="NLM_term">DEL</td><td class="NLM_def"><p class="first last">DNA-encoded library</p></td></tr><tr><td class="NLM_term">EM</td><td class="NLM_def"><p class="first last">electron microscopy</p></td></tr><tr><td class="NLM_term">FEL</td><td class="NLM_def"><p class="first last">free electron laser</p></td></tr><tr><td class="NLM_term">HTS</td><td class="NLM_def"><p class="first last">high throughput screening</p></td></tr><tr><td class="NLM_term">iPS</td><td class="NLM_def"><p class="first last">induced pluripotent stem (cell)</p></td></tr><tr><td class="NLM_term">MOA</td><td class="NLM_def"><p class="first last">mode-of-action</p></td></tr><tr><td class="NLM_term">modRNA</td><td class="NLM_def"><p class="first last">modified RNA</p></td></tr><tr><td class="NLM_term">PEM</td><td class="NLM_def"><p class="first last">protein epitope mimetic</p></td></tr><tr><td class="NLM_term">PPI</td><td class="NLM_def"><p class="first last">protein–protein interaction</p></td></tr><tr><td class="NLM_term">PROTAC</td><td class="NLM_def"><p class="first last">proteolysis targeting chimera</p></td></tr><tr><td class="NLM_term">qPCR</td><td class="NLM_def"><p class="first last">quantitative polymerase chain reaction</p></td></tr><tr><td class="NLM_term">RaPID</td><td class="NLM_def"><p class="first last">random nonstandard peptide integrated discover</p></td></tr><tr><td class="NLM_term">RTHS</td><td class="NLM_def"><p class="first last">reverse two-hybrid system</p></td></tr><tr><td class="NLM_term">SBDD</td><td class="NLM_def"><p class="first last">structure-based drug design</p></td></tr><tr><td class="NLM_term">SELEX</td><td class="NLM_def"><p class="first last">systematic evolution of ligands by exponential enrichment</p></td></tr><tr><td class="NLM_term">SICLOPPS</td><td class="NLM_def"><p class="first last">split intein synthesis of proteins and peptides</p></td></tr><tr><td class="NLM_term">siRNA</td><td class="NLM_def"><p class="first last">silencing RNA</p></td></tr><tr><td class="NLM_term">SMA</td><td class="NLM_def"><p class="first last">spinal muscular atrophy</p></td></tr><tr><td class="NLM_term">SPR</td><td class="NLM_def"><p class="first last">surface plasmon resonance</p></td></tr><tr><td class="NLM_term">StARTS</td><td class="NLM_def"><p class="first last">structure–activity relationships through sequencing</p></td></tr><tr><td class="NLM_term">TPP</td><td class="NLM_def"><p class="first last">target product profile</p></td></tr><tr><td class="NLM_term">VS</td><td class="NLM_def"><p class="first last">virtual screening</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i35">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_22731" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_22731" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 232 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Verdine, G. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walensky, L. D.</span></span> <span> </span><span class="NLM_article-title">The Challenge of Drugging Undruggable Targets in Cancer: Lessons Learned from Targeting BCL-2 Family Members</span>. <i>Clin. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">7264</span>– <span class="NLM_lpage">7270</span>, <span class="refDoi"> DOI: 10.1158/1078-0432.CCR-07-2184</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1158%2F1078-0432.CCR-07-2184" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=18094406" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhsVCktr%252FK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2007&pages=7264-7270&author=G.+L.+Verdineauthor=L.+D.+Walensky&title=The+Challenge+of+Drugging+Undruggable+Targets+in+Cancer%3A+Lessons+Learned+from+Targeting+BCL-2+Family+Members&doi=10.1158%2F1078-0432.CCR-07-2184"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">The Challenge of Drugging Undruggable Targets in Cancer: Lessons Learned from Targeting BCL-2 Family Members</span></div><div class="casAuthors">Verdine, Gregory L.; Walensky, Loren D.</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">7264-7270</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">A review.  The genomic and proteomic revolutions have provided us with an ever-increasing no. of mechanistic insights into cancer pathogenesis.  Mutated genes and pathol. protein products have emerged as the basis for modern anticancer drug development.  With the increasing realization of the importance of disrupting oncogenic protein-protein interaction, new challenges have emerged for classical small mol. and protein-based drug modalities, i.e., the crit. need to target flat and extended protein surfaces.  Here, we highlight two distinct technologies that are being used to bridge the pharmacol. gap between small mols. and protein therapeutics.  With the BCL-2 family of survival proteins as their substrate for intracellular targeting, we conclude that peptide stapling and fragment-based drug discovery show promise to traverse the crit. surface features of proteins that drive human cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrgYQX3c_4_qLVg90H21EOLACvtfcHk0ljmUao4WWSfDw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhsVCktr%252FK&md5=1b7fcbe41063e04470f7c34b50410a42</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1158%2F1078-0432.CCR-07-2184&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1078-0432.CCR-07-2184%26sid%3Dliteratum%253Aachs%26aulast%3DVerdine%26aufirst%3DG.%2BL.%26aulast%3DWalensky%26aufirst%3DL.%2BD.%26atitle%3DThe%2520Challenge%2520of%2520Drugging%2520Undruggable%2520Targets%2520in%2520Cancer%253A%2520Lessons%2520Learned%2520from%2520Targeting%2520BCL-2%2520Family%2520Members%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D2007%26volume%3D13%26spage%3D7264%26epage%3D7270%26doi%3D10.1158%2F1078-0432.CCR-07-2184" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dugger, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Platt, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goldstein, D. B.</span></span> <span> </span><span class="NLM_article-title">Drug Development in the Era of Precision Medicine</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">183</span>– <span class="NLM_lpage">196</span>, <span class="refDoi"> DOI: 10.1038/nrd.2017.226</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1038%2Fnrd.2017.226" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=29217837" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvFCjur%252FM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2018&pages=183-196&author=S.+A.+Duggerauthor=A.+Plattauthor=D.+B.+Goldstein&title=Drug+Development+in+the+Era+of+Precision+Medicine&doi=10.1038%2Fnrd.2017.226"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Drug development in the era of precision medicine</span></div><div class="casAuthors">Dugger, Sarah A.; Platt, Adam; Goldstein, David B.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">183-196</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">A review.  For the past three decades, the use of genomics to inform drug discovery and development pipelines has generated both excitement and scepticism.  Although earlier efforts successfully identified some new drug targets, the overall clin. efficacy of developed drugs has remained unimpressive, owing in large part to the heterogeneous causes of disease.  Recent technol. and anal. advances in genomics, however, have now made it possible to rapidly identify and interpret the genetic variation underlying a single patient's disease, thereby providing a window into patient-specific mechanisms that cause or contribute to disease, which could ultimately enable the 'precise' targeting of these mechanisms.  Here, we first examine and highlight the successes and limitations of the earlier phases of genomics in drug discovery and development.  We then review the current major efforts in precision medicine and discuss the potential broader utility of mechanistically guided treatments going forward.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpshWBfUfoWf7Vg90H21EOLACvtfcHk0ljmUao4WWSfDw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvFCjur%252FM&md5=e06762106391d7646c686e6ef773398a</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2017.226&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2017.226%26sid%3Dliteratum%253Aachs%26aulast%3DDugger%26aufirst%3DS.%2BA.%26aulast%3DPlatt%26aufirst%3DA.%26aulast%3DGoldstein%26aufirst%3DD.%2BB.%26atitle%3DDrug%2520Development%2520in%2520the%2520Era%2520of%2520Precision%2520Medicine%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2018%26volume%3D17%26spage%3D183%26epage%3D196%26doi%3D10.1038%2Fnrd.2017.226" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hart, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tong, A. H. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Leeuwen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seetharaman, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aregger, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chandrashekhar, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hustedt, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seth, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Noonan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Habsid, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sizova, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nedyalkova, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Climie, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tworzyanski, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lawson, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sartori, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alibeh, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tieu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Masud, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mero, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weiss, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Usaj, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Billmann, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rahman, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Costanzo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Myers, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andrews, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boone, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Durocher, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moffat, J.</span></span> <span> </span><span class="NLM_article-title">Evaluation and Design of Genome-Wide CRISPR/SpCas9 Knockout Screens</span>. <i>G3: Genes, Genomes, Genet.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">2719</span>– <span class="NLM_lpage">2727</span>, <span class="refDoi"> DOI: 10.1534/g3.117.041277</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1534%2Fg3.117.041277" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2017&pages=2719-2727&author=T.+Hartauthor=A.+H.+Y.+Tongauthor=K.+Chanauthor=J.+Van+Leeuwenauthor=A.+Seetharamanauthor=M.+Areggerauthor=M.+Chandrashekharauthor=N.+Hustedtauthor=S.+Sethauthor=A.+Noonanauthor=A.+Habsidauthor=O.+Sizovaauthor=L.+Nedyalkovaauthor=R.+Climieauthor=L.+Tworzyanskiauthor=K.+Lawsonauthor=M.+A.+Sartoriauthor=S.+Alibehauthor=D.+Tieuauthor=S.+Masudauthor=P.+Meroauthor=A.+Weissauthor=K.+R.+Brownauthor=M.+Usajauthor=M.+Billmannauthor=M.+Rahmanauthor=M.+Costanzoauthor=C.+L.+Myersauthor=B.+J.+Andrewsauthor=C.+Booneauthor=D.+Durocherauthor=J.+Moffat&title=Evaluation+and+Design+of+Genome-Wide+CRISPR%2FSpCas9+Knockout+Screens&doi=10.1534%2Fg3.117.041277"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1534%2Fg3.117.041277&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1534%252Fg3.117.041277%26sid%3Dliteratum%253Aachs%26aulast%3DHart%26aufirst%3DT.%26aulast%3DTong%26aufirst%3DA.%2BH.%2BY.%26aulast%3DChan%26aufirst%3DK.%26aulast%3DVan%2BLeeuwen%26aufirst%3DJ.%26aulast%3DSeetharaman%26aufirst%3DA.%26aulast%3DAregger%26aufirst%3DM.%26aulast%3DChandrashekhar%26aufirst%3DM.%26aulast%3DHustedt%26aufirst%3DN.%26aulast%3DSeth%26aufirst%3DS.%26aulast%3DNoonan%26aufirst%3DA.%26aulast%3DHabsid%26aufirst%3DA.%26aulast%3DSizova%26aufirst%3DO.%26aulast%3DNedyalkova%26aufirst%3DL.%26aulast%3DClimie%26aufirst%3DR.%26aulast%3DTworzyanski%26aufirst%3DL.%26aulast%3DLawson%26aufirst%3DK.%26aulast%3DSartori%26aufirst%3DM.%2BA.%26aulast%3DAlibeh%26aufirst%3DS.%26aulast%3DTieu%26aufirst%3DD.%26aulast%3DMasud%26aufirst%3DS.%26aulast%3DMero%26aufirst%3DP.%26aulast%3DWeiss%26aufirst%3DA.%26aulast%3DBrown%26aufirst%3DK.%2BR.%26aulast%3DUsaj%26aufirst%3DM.%26aulast%3DBillmann%26aufirst%3DM.%26aulast%3DRahman%26aufirst%3DM.%26aulast%3DCostanzo%26aufirst%3DM.%26aulast%3DMyers%26aufirst%3DC.%2BL.%26aulast%3DAndrews%26aufirst%3DB.%2BJ.%26aulast%3DBoone%26aufirst%3DC.%26aulast%3DDurocher%26aufirst%3DD.%26aulast%3DMoffat%26aufirst%3DJ.%26atitle%3DEvaluation%2520and%2520Design%2520of%2520Genome-Wide%2520CRISPR%252FSpCas9%2520Knockout%2520Screens%26jtitle%3DG3%253A%2520Genes%252C%2520Genomes%252C%2520Genet.%26date%3D2017%26volume%3D7%26spage%3D2719%26epage%3D2727%26doi%3D10.1534%2Fg3.117.041277" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Disney, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Angelbello, A. J.</span></span> <span> </span><span class="NLM_article-title">Rational Design of Small Molecules Targeting Oncogenic Noncoding RNAs from Sequence</span>. <i>Acc. Chem. Res.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>49</i></span>,  <span class="NLM_fpage">2698</span>– <span class="NLM_lpage">2704</span>, <span class="refDoi"> DOI: 10.1021/acs.accounts.6b00326</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.accounts.6b00326" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvV2ht7nJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2016&pages=2698-2704&author=M.+D.+Disneyauthor=A.+J.+Angelbello&title=Rational+Design+of+Small+Molecules+Targeting+Oncogenic+Noncoding+RNAs+from+Sequence&doi=10.1021%2Facs.accounts.6b00326"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Rational Design of Small Molecules Targeting Oncogenic Noncoding RNAs from Sequence</span></div><div class="casAuthors">Disney, Matthew D.; Angelbello, Alicia J.</div><div class="citationInfo"><span class="NLM_cas:title">Accounts of Chemical Research</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2698-2704</span>CODEN:
                <span class="NLM_cas:coden">ACHRE4</span>;
        ISSN:<span class="NLM_cas:issn">0001-4842</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The discovery of RNA catalysis in the 1980s and the dissemination of the human genome sequence at the start of this century inspired investigations of the regulatory roles of noncoding RNAs in biol.  In fact, the Encyclopedia of DNA Elements (ENCODE) project has shown that only 1-2% of the human genome encodes protein, yet 75% is transcribed into RNA.  Functional studies both preceding and following the ENCODE project have shown that these noncoding RNAs have important roles that regulate gene expression, developmental timing, and other crit. functions.  RNA's diverse roles are often a consequence of the various folds that it adopts.  The single stranded nature of the biopolymer enables it to adopt intramol. folds with noncanonical pairings to lower its free energy (Figure ).  These folds can be scaffolds to bind proteins or to form frameworks to interact with other RNAs.  Not surprisingly, dysregulation of certain noncoding RNAs has been shown to be causative of disease.  Given this as a background, it is easy to see why it would be useful to develop methods to target RNA and manipulate its biol. in rational and predictable ways.  The antisense approach has afforded strategies to target RNAs via Watson-Crick based pairing and has typically focused on targeting partially unstructured regions of RNA.  Small mol. strategies to target RNA would be desirable, not only because compds. could be lead optimized via medicinal chem., but also because structured regions within an RNA of interest could be targeted in order to directly interfere with RNA folds that contribute to disease.  Addnl., small mols. have historically been the most successful drug candidates.  Until recently, the ability to design small mols. that target nonrRNAs has been elusive, creating the perception that they are "undruggable".  In this Account, approaches to demystify targeting RNA with small mols. are described.  Rather than bulk screening for compds. that bind to singular targets, which is the purview of the pharmaceutical industry and academic institutions with high throughput screening facilities, we focus on methods that allow for the rational design of small mols. toward biol. RNAs.  Enabling and foundational technologies that have been developed include two-dimensional combinatorial screening (2DCS), a library-vs.-library selection approach that allows the identification of the RNA motif binding preferences of small mols. from millions of combinations.  A landscape map of the 2DCS-defined and annotated RNA motif-small mol. interactions is then placed into Inforna, a computational tool that allows one to mine these interactions against an RNA of interest or an entire transcriptome.  Indeed, this approach has been enabled by tools to annotate RNA structure from sequence-an invaluable asset to the RNA community and this work-and has allowed for the rational identification of "druggable" RNAs in a target agnostic fashion.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpU-8j6q0xYYrVg90H21EOLACvtfcHk0lg8d5k7wspDMA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvV2ht7nJ&md5=14a75becddc19fd360e9b5dbd94eec48</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1021%2Facs.accounts.6b00326&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.accounts.6b00326%26sid%3Dliteratum%253Aachs%26aulast%3DDisney%26aufirst%3DM.%2BD.%26aulast%3DAngelbello%26aufirst%3DA.%2BJ.%26atitle%3DRational%2520Design%2520of%2520Small%2520Molecules%2520Targeting%2520Oncogenic%2520Noncoding%2520RNAs%2520from%2520Sequence%26jtitle%3DAcc.%2520Chem.%2520Res.%26date%3D2016%26volume%3D49%26spage%3D2698%26epage%3D2704%26doi%3D10.1021%2Facs.accounts.6b00326" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Valeur, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guéret, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adihou, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gopalakrishnan, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lemurell, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waldmann, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grossmann, T. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plowright, A. T.</span></span> <span> </span><span class="NLM_article-title">New Modalities for Challenging Targets in Drug Discovery</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">10294</span>– <span class="NLM_lpage">10323</span>, <span class="refDoi"> DOI: 10.1002/anie.201611914</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1002%2Fanie.201611914" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC2sXht1WlsLvP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2017&pages=10294-10323&author=E.+Valeurauthor=S.+M.+Gu%C3%A9retauthor=H.+Adihouauthor=R.+Gopalakrishnanauthor=M.+Lemurellauthor=H.+Waldmannauthor=T.+N.+Grossmannauthor=A.+T.+Plowright&title=New+Modalities+for+Challenging+Targets+in+Drug+Discovery&doi=10.1002%2Fanie.201611914"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">New Modalities for Challenging Targets in Drug Discovery</span></div><div class="casAuthors">Valeur, Eric; Gueret, Stephanie M.; Adihou, Helene; Gopalakrishnan, Ranganath; Lemurell, Malin; Waldmann, Herbert; Grossmann, Tom N.; Plowright, Alleyn T.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">35</span>),
    <span class="NLM_cas:pages">10294-10323</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A review.  The ever-increasing understanding of biol. systems is providing a range of exciting novel biol. targets, whose modulation may enable novel therapeutic options for many diseases.  These targets include protein-protein and protein-nucleic acid interactions, which are, however, often refractory to classical small-mol. approaches.  Other types of mols., or modalities, are therefore required to address these targets, which has led several academic research groups and pharmaceutical companies to increasingly use the concept of so-called "new modalities".  This review defines for the first time the scope of this term, which includes novel peptidic scaffolds, oligonucleotides, hybrids, mol. conjugates, as well as new uses of classical small mols.  The authors provide the most representative examples of these modalities to target large binding surface areas such as those found in protein-protein interactions and for biol. processes at the center of cell regulation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp3BpAx5oKYa7Vg90H21EOLACvtfcHk0lg8d5k7wspDMA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXht1WlsLvP&md5=e09ef5bb8dbbf09c05b1a2340aa431ec</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1002%2Fanie.201611914&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201611914%26sid%3Dliteratum%253Aachs%26aulast%3DValeur%26aufirst%3DE.%26aulast%3DGu%25C3%25A9ret%26aufirst%3DS.%2BM.%26aulast%3DAdihou%26aufirst%3DH.%26aulast%3DGopalakrishnan%26aufirst%3DR.%26aulast%3DLemurell%26aufirst%3DM.%26aulast%3DWaldmann%26aufirst%3DH.%26aulast%3DGrossmann%26aufirst%3DT.%2BN.%26aulast%3DPlowright%26aufirst%3DA.%2BT.%26atitle%3DNew%2520Modalities%2520for%2520Challenging%2520Targets%2520in%2520Drug%2520Discovery%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2017%26volume%3D56%26spage%3D10294%26epage%3D10323%26doi%3D10.1002%2Fanie.201611914" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bunnage, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilbert, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, L. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hett, E. C.</span></span> <span> </span><span class="NLM_article-title">Know Your Target, Know Your Molecule</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">368</span>– <span class="NLM_lpage">372</span>, <span class="refDoi"> DOI: 10.1038/nchembio.1813</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1038%2Fnchembio.1813" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=25978985" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtFemtrvK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2015&pages=368-372&author=M.+E.+Bunnageauthor=A.+M.+Gilbertauthor=L.+H.+Jonesauthor=E.+C.+Hett&title=Know+Your+Target%2C+Know+Your+Molecule&doi=10.1038%2Fnchembio.1813"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Know your target, know your molecule</span></div><div class="casAuthors">Bunnage, Mark E.; Gilbert, Adam M.; Jones, Lyn H.; Hett, Erik C.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">368-372</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review. The pharmaceutical industry continues to experience significant attrition of drug candidates during phase 2 proof-of-concept clin. studies.  We describe some questions about the characteristics of protein targets and small-mol. drugs that may be important to consider in drug-discovery projects and could improve prospects for future clin. success.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpRthCFSFvOXrVg90H21EOLACvtfcHk0lg8d5k7wspDMA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtFemtrvK&md5=f9606135ccea6af432040d8afb630ede</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1813&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1813%26sid%3Dliteratum%253Aachs%26aulast%3DBunnage%26aufirst%3DM.%2BE.%26aulast%3DGilbert%26aufirst%3DA.%2BM.%26aulast%3DJones%26aufirst%3DL.%2BH.%26aulast%3DHett%26aufirst%3DE.%2BC.%26atitle%3DKnow%2520Your%2520Target%252C%2520Know%2520Your%2520Molecule%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2015%26volume%3D11%26spage%3D368%26epage%3D372%26doi%3D10.1038%2Fnchembio.1813" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fellmann, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gowen, B. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, P.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doudna, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Corn, J. E.</span></span> <span> </span><span class="NLM_article-title">Cornerstones of CRISPR–Cas in Drug Discovery and Therapy</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">89</span>– <span class="NLM_lpage">100</span>, <span class="refDoi"> DOI: 10.1038/nrd.2016.238</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1038%2Fnrd.2016.238" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2017&pages=89-100&author=C.+Fellmannauthor=B.+G.+Gowenauthor=P.-C.+Linauthor=J.+A.+Doudnaauthor=J.+E.+Corn&title=Cornerstones+of+CRISPR%E2%80%93Cas+in+Drug+Discovery+and+Therapy&doi=10.1038%2Fnrd.2016.238"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2016.238&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2016.238%26sid%3Dliteratum%253Aachs%26aulast%3DFellmann%26aufirst%3DC.%26aulast%3DGowen%26aufirst%3DB.%2BG.%26aulast%3DLin%26aufirst%3DP.-C.%26aulast%3DDoudna%26aufirst%3DJ.%2BA.%26aulast%3DCorn%26aufirst%3DJ.%2BE.%26atitle%3DCornerstones%2520of%2520CRISPR%25E2%2580%2593Cas%2520in%2520Drug%2520Discovery%2520and%2520Therapy%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2017%26volume%3D16%26spage%3D89%26epage%3D100%26doi%3D10.1038%2Fnrd.2016.238" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sander, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Joung, J. K.</span></span> <span> </span><span class="NLM_article-title">CRISPR-Cas Systems for Editing, Regulating and Targeting Genomes</span>. <i>Nat. Biotechnol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>32</i></span>,  <span class="NLM_fpage">347</span>– <span class="NLM_lpage">355</span>, <span class="refDoi"> DOI: 10.1038/nbt.2842</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1038%2Fnbt.2842" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=24584096" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC2cXjtlyrsLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2014&pages=347-355&author=J.+D.+Sanderauthor=J.+K.+Joung&title=CRISPR-Cas+Systems+for+Editing%2C+Regulating+and+Targeting+Genomes&doi=10.1038%2Fnbt.2842"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">CRISPR-Cas systems for editing, regulating and targeting genomes</span></div><div class="casAuthors">Sander, Jeffry D.; Joung, J. Keith</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">32</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">347-355</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Targeted genome editing using engineered nucleases has rapidly gone from being a niche technol. to a mainstream method used by many biol. researchers.  This widespread adoption has been largely fueled by the emergence of the clustered, regularly interspaced, short palindromic repeat (CRISPR) technol., an important new approach for generating RNA-guided nucleases, such as Cas9, with customizable specificities.  Genome editing mediated by these nucleases has been used to rapidly, easily and efficiently modify endogenous genes in a wide variety of biomedically important cell types and in organisms that have traditionally been challenging to manipulate genetically.  Furthermore, a modified version of the CRISPR-Cas9 system was developed to recruit heterologous domains that can regulate endogenous gene expression or label specific genomic loci in living cells.  Although the genome-wide specificities of CRISPR-Cas9 systems remain to be fully defined, the power of these systems to perform targeted, highly efficient alterations of genome sequence and gene expression will undoubtedly transform biol. research and spur the development of novel mol. therapeutics for human disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp4VCoNVtj1fLVg90H21EOLACvtfcHk0ljK537ApR5KRg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXjtlyrsLo%253D&md5=7be62d2d37b419010fb8bec273731c07</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1038%2Fnbt.2842&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt.2842%26sid%3Dliteratum%253Aachs%26aulast%3DSander%26aufirst%3DJ.%2BD.%26aulast%3DJoung%26aufirst%3DJ.%2BK.%26atitle%3DCRISPR-Cas%2520Systems%2520for%2520Editing%252C%2520Regulating%2520and%2520Targeting%2520Genomes%26jtitle%3DNat.%2520Biotechnol.%26date%3D2014%26volume%3D32%26spage%3D347%26epage%3D355%26doi%3D10.1038%2Fnbt.2842" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cox, D. B. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Platt, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, F.</span></span> <span> </span><span class="NLM_article-title">Therapeutic Genome Editing: Prospects and Challenges</span>. <i>Nat. Med.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">121</span>– <span class="NLM_lpage">131</span>, <span class="refDoi"> DOI: 10.1038/nm.3793</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1038%2Fnm.3793" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=25654603" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC2MXit1artbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2015&pages=121-131&author=D.+B.+T.+Coxauthor=R.+J.+Plattauthor=F.+Zhang&title=Therapeutic+Genome+Editing%3A+Prospects+and+Challenges&doi=10.1038%2Fnm.3793"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Therapeutic genome editing: prospects and challenges</span></div><div class="casAuthors">Cox, David Benjamin Turitz; Platt, Randall Jeffrey; Zhang, Feng</div><div class="citationInfo"><span class="NLM_cas:title">Nature Medicine (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">121-131</span>CODEN:
                <span class="NLM_cas:coden">NAMEFI</span>;
        ISSN:<span class="NLM_cas:issn">1078-8956</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Recent advances in the development of genome editing technologies based on programmable nucleases have substantially improved our ability to make precise changes in the genomes of eukaryotic cells.  Genome editing is already broadening our ability to elucidate the contribution of genetics to disease by facilitating the creation of more accurate cellular and animal models of pathol. processes.  A particularly tantalizing application of programmable nucleases is the potential to directly correct genetic mutations in affected tissues and cells to treat diseases that are refractory to traditional therapies.  Here we discuss current progress toward developing programmable nuclease-based therapies as well as future prospects and challenges.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp0EN1KkwY_dLVg90H21EOLACvtfcHk0ljK537ApR5KRg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXit1artbk%253D&md5=352473152693168e492dbdc6410cebb0</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1038%2Fnm.3793&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnm.3793%26sid%3Dliteratum%253Aachs%26aulast%3DCox%26aufirst%3DD.%2BB.%2BT.%26aulast%3DPlatt%26aufirst%3DR.%2BJ.%26aulast%3DZhang%26aufirst%3DF.%26atitle%3DTherapeutic%2520Genome%2520Editing%253A%2520Prospects%2520and%2520Challenges%26jtitle%3DNat.%2520Med.%26date%3D2015%26volume%3D21%26spage%3D121%26epage%3D131%26doi%3D10.1038%2Fnm.3793" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cox, D. B. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gootenberg, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abudayyeh, O. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Franklin, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kellner, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Joung, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, F.</span></span> <span> </span><span class="NLM_article-title">RNA Editing with CRISPR-Cas13</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>358</i></span>,  <span class="NLM_fpage">1019</span>, <span class="refDoi"> DOI: 10.1126/science.aaq0180</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1126%2Fscience.aaq0180" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=29070703" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvFahsb3J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=358&publication_year=2017&pages=1019&author=D.+B.+T.+Coxauthor=J.+S.+Gootenbergauthor=O.+O.+Abudayyehauthor=B.+Franklinauthor=M.+J.+Kellnerauthor=J.+Joungauthor=F.+Zhang&title=RNA+Editing+with+CRISPR-Cas13&doi=10.1126%2Fscience.aaq0180"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">RNA editing with CRISPR-Cas13</span></div><div class="casAuthors">Cox, David B. T.; Gootenberg, Jonathan S.; Abudayyeh, Omar O.; Franklin, Brian; Kellner, Max J.; Joung, Julia; Zhang, Feng</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">358</span>
        (<span class="NLM_cas:issue">6366</span>),
    <span class="NLM_cas:pages">1019-1027</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Nucleic acid editing holds promise for treating genetic disease, particularly at the RNA level, where disease-relevant sequences can be rescued to yield functional protein products.  Type VI CRISPR-Cas systems contain the programmable single-effector RNA-guided RNase Cas13.  We profiled type VI systems in order to engineer a Cas13 ortholog capable of robust knockdown and demonstrated RNA editing by using catalytically inactive Cas13 (dCas13) to direct adenosine-to-inosine deaminase activity by ADAR2 (adenosine deaminase acting on RNA type 2) to transcripts in mammalian cells.  This system, referred to as RNA editing for programmable A to I replacement (REPAIR), which has no strict sequence constraints, can be used to edit full-length transcripts contg. pathogenic mutations.  We further engineered this system to create a high-specificity variant and minimized the system to facilitate viral delivery.  REPAIR presents a promising RNA-editing platform with broad applicability for research, therapeutics, and biotechnol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrlQ-Cv1-eMdbVg90H21EOLACvtfcHk0ljK537ApR5KRg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvFahsb3J&md5=21ed8ebbd2cc334be0911129196e6b6e</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1126%2Fscience.aaq0180&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.aaq0180%26sid%3Dliteratum%253Aachs%26aulast%3DCox%26aufirst%3DD.%2BB.%2BT.%26aulast%3DGootenberg%26aufirst%3DJ.%2BS.%26aulast%3DAbudayyeh%26aufirst%3DO.%2BO.%26aulast%3DFranklin%26aufirst%3DB.%26aulast%3DKellner%26aufirst%3DM.%2BJ.%26aulast%3DJoung%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DF.%26atitle%3DRNA%2520Editing%2520with%2520CRISPR-Cas13%26jtitle%3DScience%26date%3D2017%26volume%3D358%26spage%3D1019%26doi%3D10.1126%2Fscience.aaq0180" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">An, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schneller, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frassetto, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, J.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Theisen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hong, S.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rajendran, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Levy, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Howell, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Besin, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Presnyak, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sabnis, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murphy-Benenato, K. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumarasinghe, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salerno, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mihai, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lukacs, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chandler, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guey, L. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Venditti, C. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martini, P. G. V.</span></span> <span> </span><span class="NLM_article-title">Systemic Messenger RNA Therapy as a Treatment for Methylmalonic Acidemia</span>. <i>Cell Rep.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">3548</span>– <span class="NLM_lpage">3558</span>, <span class="refDoi"> DOI: 10.1016/j.celrep.2017.11.081</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1016%2Fj.celrep.2017.11.081" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=29262333" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC2sXitVWlsrbP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2017&pages=3548-3558&author=D.+Anauthor=J.+L.+Schnellerauthor=A.+Frassettoauthor=S.+Liangauthor=X.+Zhuauthor=J.-S.+Parkauthor=M.+Theisenauthor=S.-J.+Hongauthor=J.+Zhouauthor=R.+Rajendranauthor=B.+Levyauthor=R.+Howellauthor=G.+Besinauthor=V.+Presnyakauthor=S.+Sabnisauthor=K.+E.+Murphy-Benenatoauthor=E.+S.+Kumarasingheauthor=T.+Salernoauthor=C.+Mihaiauthor=C.+M.+Lukacsauthor=R.+J.+Chandlerauthor=L.+T.+Gueyauthor=C.+P.+Vendittiauthor=P.+G.+V.+Martini&title=Systemic+Messenger+RNA+Therapy+as+a+Treatment+for+Methylmalonic+Acidemia&doi=10.1016%2Fj.celrep.2017.11.081"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Systemic Messenger RNA Therapy as a Treatment for Methylmalonic Acidemia</span></div><div class="casAuthors">An, Ding; Schneller, Jessica L.; Frassetto, Andrea; Liang, Shi; Zhu, Xuling; Park, Ji-Sun; Theisen, Matt; Hong, Sue-Jean; Zhou, Jenny; Rajendran, Raj; Levy, Becca; Howell, Rebecca; Besin, Gilles; Presnyak, Vladimir; Sabnis, Staci; Murphy-Benenato, Kerry E.; Kumarasinghe, E. Sathyajith; Salerno, Timothy; Mihai, Cosmin; Lukacs, Christine M.; Chandler, Randy J.; Guey, Lin T.; Venditti, Charles P.; Martini, Paolo G. V.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Reports</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">3548-3558</span>CODEN:
                <span class="NLM_cas:coden">CREED8</span>;
        ISSN:<span class="NLM_cas:issn">2211-1247</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Isolated methylmalonic acidemia/aciduria (MMA) is a devastating metabolic disorder with poor outcomes despite current medical treatments.  Like other mitochondrial enzymopathies, enzyme replacement therapy (ERT) is not available, and although promising, AAV gene therapy can be limited by pre-existing immunity and has been assocd. with genotoxicity in mice.  To develop a new class of therapy for MMA, we generated a pseudoU-modified codon-optimized mRNA encoding human methylmalonyl-CoA mutase (hMUT), the enzyme most frequently mutated in MMA, and encapsulated it into biodegradable lipid nanoparticles (LNPs).  I.v. (i.v.) administration of hMUT mRNA in two different mouse models of MMA resulted in a 75%-85% redn. in plasma methylmalonic acid and was assocd. with increased hMUT protein expression and activity in liver.  Repeat dosing of hMUT mRNA reduced circulating metabolites and dramatically improved survival and wt. gain.  Addnl., repeat i.v. dosing did not increase markers of liver toxicity or inflammation in heterozygote MMA mice.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqf0QTenUCp_rVg90H21EOLACvtfcHk0lgQztKQ5wj31w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXitVWlsrbP&md5=1bd1d194653ce58f22f92e8aa77c7004</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1016%2Fj.celrep.2017.11.081&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.celrep.2017.11.081%26sid%3Dliteratum%253Aachs%26aulast%3DAn%26aufirst%3DD.%26aulast%3DSchneller%26aufirst%3DJ.%2BL.%26aulast%3DFrassetto%26aufirst%3DA.%26aulast%3DLiang%26aufirst%3DS.%26aulast%3DZhu%26aufirst%3DX.%26aulast%3DPark%26aufirst%3DJ.-S.%26aulast%3DTheisen%26aufirst%3DM.%26aulast%3DHong%26aufirst%3DS.-J.%26aulast%3DZhou%26aufirst%3DJ.%26aulast%3DRajendran%26aufirst%3DR.%26aulast%3DLevy%26aufirst%3DB.%26aulast%3DHowell%26aufirst%3DR.%26aulast%3DBesin%26aufirst%3DG.%26aulast%3DPresnyak%26aufirst%3DV.%26aulast%3DSabnis%26aufirst%3DS.%26aulast%3DMurphy-Benenato%26aufirst%3DK.%2BE.%26aulast%3DKumarasinghe%26aufirst%3DE.%2BS.%26aulast%3DSalerno%26aufirst%3DT.%26aulast%3DMihai%26aufirst%3DC.%26aulast%3DLukacs%26aufirst%3DC.%2BM.%26aulast%3DChandler%26aufirst%3DR.%2BJ.%26aulast%3DGuey%26aufirst%3DL.%2BT.%26aulast%3DVenditti%26aufirst%3DC.%2BP.%26aulast%3DMartini%26aufirst%3DP.%2BG.%2BV.%26atitle%3DSystemic%2520Messenger%2520RNA%2520Therapy%2520as%2520a%2520Treatment%2520for%2520Methylmalonic%2520Acidemia%26jtitle%3DCell%2520Rep.%26date%3D2017%26volume%3D21%26spage%3D3548%26epage%3D3558%26doi%3D10.1016%2Fj.celrep.2017.11.081" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zangi, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lui, K. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">von Gise, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ebina, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ptaszek, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Später, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tabebordbar, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gorbatov, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sena, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nahrendorf, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Briscoe, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wagers, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rossi, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pu, W. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chien, K. R.</span></span> <span> </span><span class="NLM_article-title">Modified mRNA Directs the Fate of Heart Progenitor Cells and Induces Vascular Regeneration after Myocardial Infarction</span>. <i>Nat. Biotechnol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>31</i></span>,  <span class="NLM_fpage">898</span>– <span class="NLM_lpage">907</span>, <span class="refDoi"> DOI: 10.1038/nbt.2682</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1038%2Fnbt.2682" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=24013197" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtl2ksbbF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2013&pages=898-907&author=L.+Zangiauthor=K.+O.+Luiauthor=A.+von+Giseauthor=Q.+Maauthor=W.+Ebinaauthor=L.+M.+Ptaszekauthor=D.+Sp%C3%A4terauthor=H.+Xuauthor=M.+Tabebordbarauthor=R.+Gorbatovauthor=B.+Senaauthor=M.+Nahrendorfauthor=D.+M.+Briscoeauthor=R.+A.+Liauthor=A.+J.+Wagersauthor=D.+J.+Rossiauthor=W.+T.+Puauthor=K.+R.+Chien&title=Modified+mRNA+Directs+the+Fate+of+Heart+Progenitor+Cells+and+Induces+Vascular+Regeneration+after+Myocardial+Infarction&doi=10.1038%2Fnbt.2682"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Modified mRNA directs the fate of heart progenitor cells and induces vascular regeneration after myocardial infarction</span></div><div class="casAuthors">Zangi, Lior; Lui, Kathy O.; von Gise, Alexander; Ma, Qing; Ebina, Wataru; Ptaszek, Leon M.; Spaeter, Daniela; Xu, Huansheng; Tabebordbar, Mohammadsharif; Gorbatov, Rostic; Sena, Brena; Nahrendorf, Matthias; Briscoe, David M.; Li, Ronald A.; Wagers, Amy J.; Rossi, Derrick J.; Pu, William T.; Chien, Kenneth R.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">898-907</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">In a cell-free approach to regenerative therapeutics, transient application of paracrine factors in vivo could be used to alter the behavior and fate of progenitor cells to achieve sustained clin. benefits.  Here we show that intramyocardial injection of synthetic modified RNA (modRNA) encoding human vascular endothelial growth factor-A (VEGF-A) results in the expansion and directed differentiation of endogenous heart progenitors in a mouse myocardial infarction model.  VEGF-A modRNA markedly improved heart function and enhanced long-term survival of recipients.  This improvement was in part due to mobilization of epicardial progenitor cells and redirection of their differentiation toward cardiovascular cell types.  Direct in vivo comparison with DNA vectors and temporal control with VEGF inhibitors revealed the greatly increased efficacy of pulse-like delivery of VEGF-A.  Our results suggest that modRNA is a versatile approach for expressing paracrine factors as cell fate switches to control progenitor cell fate and thereby enhance long-term organ repair.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrmPXkQ6XhLk7Vg90H21EOLACvtfcHk0lgQztKQ5wj31w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtl2ksbbF&md5=911543ad4989377e233ec2f0dcbc900d</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1038%2Fnbt.2682&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt.2682%26sid%3Dliteratum%253Aachs%26aulast%3DZangi%26aufirst%3DL.%26aulast%3DLui%26aufirst%3DK.%2BO.%26aulast%3Dvon%2BGise%26aufirst%3DA.%26aulast%3DMa%26aufirst%3DQ.%26aulast%3DEbina%26aufirst%3DW.%26aulast%3DPtaszek%26aufirst%3DL.%2BM.%26aulast%3DSp%25C3%25A4ter%26aufirst%3DD.%26aulast%3DXu%26aufirst%3DH.%26aulast%3DTabebordbar%26aufirst%3DM.%26aulast%3DGorbatov%26aufirst%3DR.%26aulast%3DSena%26aufirst%3DB.%26aulast%3DNahrendorf%26aufirst%3DM.%26aulast%3DBriscoe%26aufirst%3DD.%2BM.%26aulast%3DLi%26aufirst%3DR.%2BA.%26aulast%3DWagers%26aufirst%3DA.%2BJ.%26aulast%3DRossi%26aufirst%3DD.%2BJ.%26aulast%3DPu%26aufirst%3DW.%2BT.%26aulast%3DChien%26aufirst%3DK.%2BR.%26atitle%3DModified%2520mRNA%2520Directs%2520the%2520Fate%2520of%2520Heart%2520Progenitor%2520Cells%2520and%2520Induces%2520Vascular%2520Regeneration%2520after%2520Myocardial%2520Infarction%26jtitle%3DNat.%2520Biotechnol.%26date%3D2013%26volume%3D31%26spage%3D898%26epage%3D907%26doi%3D10.1038%2Fnbt.2682" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Petersen, D. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hawkins, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruangsiriluk, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stevens, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maguire, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Connell, T. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rocke, B. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boehm, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruggeri, R. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rolph, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hepworth, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loria, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carpino, P. A.</span></span> <span> </span><span class="NLM_article-title">A Small-Molecule Anti-Secretagogue of PCSK9 Targets the 80S Ribosome to Inhibit PCSK9 Protein Translation</span>. <i>Cell Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">1362</span>– <span class="NLM_lpage">1371</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2016.08.016</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1016%2Fj.chembiol.2016.08.016" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=27746128" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC28Xhs1yqs7%252FJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2016&pages=1362-1371&author=D.+N.+Petersenauthor=J.+Hawkinsauthor=W.+Ruangsirilukauthor=K.+A.+Stevensauthor=B.+A.+Maguireauthor=T.+N.+O%E2%80%99Connellauthor=B.+N.+Rockeauthor=M.+Boehmauthor=R.+B.+Ruggeriauthor=T.+Rolphauthor=D.+Hepworthauthor=P.+M.+Loriaauthor=P.+A.+Carpino&title=A+Small-Molecule+Anti-Secretagogue+of+PCSK9+Targets+the+80S+Ribosome+to+Inhibit+PCSK9+Protein+Translation&doi=10.1016%2Fj.chembiol.2016.08.016"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">A Small-Molecule Anti-secretagogue of PCSK9 Targets the 80S Ribosome to Inhibit PCSK9 Protein Translation</span></div><div class="casAuthors">Petersen, Donna N.; Hawkins, Julie; Ruangsiriluk, Wanida; Stevens, Kimberly A.; Maguire, Bruce A.; O'Connell, Thomas N.; Rocke, Benjamin N.; Boehm, Markus; Ruggeri, Roger B.; Rolph, Tim; Hepworth, David; Loria, Paula M.; Carpino, Philip A.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1362-1371</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Proprotein convertase subtilisin/kexin type 9 (PCSK9) is a secreted protein that downregulates low-d. lipoprotein (LDL) receptor (LDL-R) levels on the surface of hepatocytes, resulting in decreased clearance of LDL-cholesterol (LDL-C).  Phenotypic screening of a small-mol. compd. collection was used to identify an inhibitor of PCSK9 secretion, (R)-N-(isoquinolin-1-yl)-3-(4-methoxyphenyl)-N-(piperidin-3-yl)propanamide (R-IMPP), which was shown to stimulate uptake of LDL-C in hepatoma cells by increasing LDL-R levels, without altering levels of secreted transferrin.  Systematic investigation of the mode of action revealed that R-IMPP did not decrease PCSK9 transcription or increase PCSK9 degrdn., but instead caused transcript-dependent inhibition of PCSK9 translation.  In support of this surprising mechanism of action, we found that R-IMPP was able to selectively bind to human, but not E. coli, ribosomes.  This study opens a new avenue for the development of drugs that modulate the activity of target proteins by mechanisms involving inhibition of eukaryotic translation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqbEnbBYD6NKbVg90H21EOLACvtfcHk0lgQztKQ5wj31w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xhs1yqs7%252FJ&md5=c375439e6ed7c49daa4625baff8e4e48</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2016.08.016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2016.08.016%26sid%3Dliteratum%253Aachs%26aulast%3DPetersen%26aufirst%3DD.%2BN.%26aulast%3DHawkins%26aufirst%3DJ.%26aulast%3DRuangsiriluk%26aufirst%3DW.%26aulast%3DStevens%26aufirst%3DK.%2BA.%26aulast%3DMaguire%26aufirst%3DB.%2BA.%26aulast%3DO%25E2%2580%2599Connell%26aufirst%3DT.%2BN.%26aulast%3DRocke%26aufirst%3DB.%2BN.%26aulast%3DBoehm%26aufirst%3DM.%26aulast%3DRuggeri%26aufirst%3DR.%2BB.%26aulast%3DRolph%26aufirst%3DT.%26aulast%3DHepworth%26aufirst%3DD.%26aulast%3DLoria%26aufirst%3DP.%2BM.%26aulast%3DCarpino%26aufirst%3DP.%2BA.%26atitle%3DA%2520Small-Molecule%2520Anti-Secretagogue%2520of%2520PCSK9%2520Targets%2520the%252080S%2520Ribosome%2520to%2520Inhibit%2520PCSK9%2520Protein%2520Translation%26jtitle%3DCell%2520Chem.%2520Biol.%26date%3D2016%26volume%3D23%26spage%3D1362%26epage%3D1371%26doi%3D10.1016%2Fj.chembiol.2016.08.016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Churcher, I.</span></span> <span> </span><span class="NLM_article-title">Protac-Induced Protein Degradation in Drug Discovery: Breaking the Rules or Just Making New Ones?</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">444</span>– <span class="NLM_lpage">452</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b01272</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b01272" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvVWqtbvJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=444-452&author=I.+Churcher&title=Protac-Induced+Protein+Degradation+in+Drug+Discovery%3A+Breaking+the+Rules+or+Just+Making+New+Ones%3F&doi=10.1021%2Facs.jmedchem.7b01272"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Protac-Induced Protein Degradation in Drug Discovery: Breaking the Rules or Just Making New Ones?</span></div><div class="casAuthors">Churcher, Ian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">444-452</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Targeted protein degrdn., using bifunctional small mols. (Protacs) to remove specific proteins from within cells, has emerged as a novel drug discovery strategy with the potential to offer therapeutic interventions not achievable with existing approaches.  In this perspective, the brief history of the field is surveyed from a drug discovery perspective with a focus on the key advances in knowledge which have led to the definition and exemplification of protein degrdn. concepts, and their resulting applications to medicine discovery.  The approach has the potential to bring disruptive change to drug discovery - the many potential advantages and outstanding challenges which lie ahead of this technol. are discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr9n8I_0MhtkbVg90H21EOLACvtfcHk0lisVWCWXNu90A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvVWqtbvJ&md5=e60f1777de8bd860c5001137d8583f1f</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b01272&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b01272%26sid%3Dliteratum%253Aachs%26aulast%3DChurcher%26aufirst%3DI.%26atitle%3DProtac-Induced%2520Protein%2520Degradation%2520in%2520Drug%2520Discovery%253A%2520Breaking%2520the%2520Rules%2520or%2520Just%2520Making%2520New%2520Ones%253F%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D444%26epage%3D452%26doi%3D10.1021%2Facs.jmedchem.7b01272" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rossi, J.</span></span> <span> </span><span class="NLM_article-title">Aptamers as Targeted Therapeutics: Current Potential and Challenges</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">181</span>– <span class="NLM_lpage">202</span>, <span class="refDoi"> DOI: 10.1038/nrd.2016.199</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1038%2Fnrd.2016.199" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=27807347" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC28XhslOqtr3I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2017&pages=181-202&author=J.+Zhouauthor=J.+Rossi&title=Aptamers+as+Targeted+Therapeutics%3A+Current+Potential+and+Challenges&doi=10.1038%2Fnrd.2016.199"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Aptamers as targeted therapeutics: current potential and challenges</span></div><div class="casAuthors">Zhou, Jiehua; Rossi, John</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">181-202</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Nucleic acid aptamers, often termed 'chem. antibodies', are functionally comparable to traditional antibodies, but offer several advantages, including their relatively small phys. size, flexible structure, quick chem. prodn., versatile chem. modification, high stability and lack of immunogenicity.  In addn., many aptamers are internalized upon binding to cellular receptors, making them useful targeted delivery agents for small interfering RNAs (siRNAs), microRNAs and conventional drugs.  However, several crucial factors have delayed the clin. translation of therapeutic aptamers, such as their inherent physicochem. characteristics and lack of safety data.  This Review discusses these challenges, highlighting recent clin. developments and technol. advances that have revived the impetus for this promising class of therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr4KSEhC4dW0LVg90H21EOLACvtfcHk0lisVWCWXNu90A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhslOqtr3I&md5=3d79942589e20829b3e8de582a08d6e1</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2016.199&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2016.199%26sid%3Dliteratum%253Aachs%26aulast%3DZhou%26aufirst%3DJ.%26aulast%3DRossi%26aufirst%3DJ.%26atitle%3DAptamers%2520as%2520Targeted%2520Therapeutics%253A%2520Current%2520Potential%2520and%2520Challenges%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2017%26volume%3D16%26spage%3D181%26epage%3D202%26doi%3D10.1038%2Fnrd.2016.199" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Carter, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lazar, G. A.</span></span> <span> </span><span class="NLM_article-title">Next Generation Antibody Drugs: Pursuit of the “High-Hanging Fruit"</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">197</span>– <span class="NLM_lpage">223</span>, <span class="refDoi"> DOI: 10.1038/nrd.2017.227</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1038%2Fnrd.2017.227" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2018&pages=197-223&author=P.+J.+Carterauthor=G.+A.+Lazar&title=Next+Generation+Antibody+Drugs%3A+Pursuit+of+the+%E2%80%9CHigh-Hanging+Fruit%22&doi=10.1038%2Fnrd.2017.227"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2017.227&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2017.227%26sid%3Dliteratum%253Aachs%26aulast%3DCarter%26aufirst%3DP.%2BJ.%26aulast%3DLazar%26aufirst%3DG.%2BA.%26atitle%3DNext%2520Generation%2520Antibody%2520Drugs%253A%2520Pursuit%2520of%2520the%2520%25E2%2580%259CHigh-Hanging%2520Fruit%2522%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2018%26volume%3D17%26spage%3D197%26epage%3D223%26doi%3D10.1038%2Fnrd.2017.227" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Passioura, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suga, H.</span></span> <span> </span><span class="NLM_article-title">A RaPID Way to Discover Nonstandard Macrocyclic Peptide Modulators of Drug Targets</span>. <i>Chem. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">1931</span>– <span class="NLM_lpage">1940</span>, <span class="refDoi"> DOI: 10.1039/C6CC06951G</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1039%2FC6CC06951G" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=28091672" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC2sXnslCnsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2017&pages=1931-1940&author=T.+Passiouraauthor=H.+Suga&title=A+RaPID+Way+to+Discover+Nonstandard+Macrocyclic+Peptide+Modulators+of+Drug+Targets&doi=10.1039%2FC6CC06951G"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">A RaPID way to discover nonstandard macrocyclic peptide modulators of drug targets</span></div><div class="casAuthors">Passioura, Toby; Suga, Hiroaki</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Communications (Cambridge, United Kingdom)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1931-1940</span>CODEN:
                <span class="NLM_cas:coden">CHCOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-7345</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">A review.  Studies of the fundamental nature of RNA catalysis and the potential mechanism of a shift from the "RNA world" to proteinaceous life lead the authors to identify a set of ribozymes (flexizymes) capable of promiscuous tRNA acylation.  While theor. and mechanistically interesting in their own right, flexizymes have turned out to have immense practical value for the simple synthesis of tRNAs acylated with unusual amino acids, which in turn can be used for the ribosomal synthesis of peptides contg. non-canonical residues.  Using this technique, it is possible to synthesize peptides contg. a range of structural features (macrocyclic backbones, backbone N-methylation, D-stereochem., etc.) commonly obsd. in natural product secondary metabolites, a chem. class that has historically been a rich source of drug-like mols.  Moreover, when combined with biochem. display screening technologies, this synthetic approach can be used to generate (and screen for target affinity) extremely diverse (in excess of 1012 compd.) chem. libraries, making it an extraordinary tool for drug discovery.  The current review charts the history of flexizyme technol. and its use for non-canonical peptide synthesis and screening.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo9yxKNW7clerVg90H21EOLACvtfcHk0lisVWCWXNu90A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXnslCnsg%253D%253D&md5=987ef1038fb1ed65a88e07db3623a039</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1039%2FC6CC06951G&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC6CC06951G%26sid%3Dliteratum%253Aachs%26aulast%3DPassioura%26aufirst%3DT.%26aulast%3DSuga%26aufirst%3DH.%26atitle%3DA%2520RaPID%2520Way%2520to%2520Discover%2520Nonstandard%2520Macrocyclic%2520Peptide%2520Modulators%2520of%2520Drug%2520Targets%26jtitle%3DChem.%2520Commun.%26date%3D2017%26volume%3D53%26spage%3D1931%26epage%3D1940%26doi%3D10.1039%2FC6CC06951G" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, R. K. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abela, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hennig, M.</span></span> <span> </span><span class="NLM_article-title">X-Ray Free Electron Laser: Opportunities for Drug Discovery</span>. <i>Essays Biochem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">529</span>– <span class="NLM_lpage">542</span>, <span class="refDoi"> DOI: 10.1042/EBC20170031</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1042%2FEBC20170031" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=29118098" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A280%3ADC%252BC1M7pt1Glsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2017&pages=529-542&author=R.+K.+Y.+Chengauthor=R.+Abelaauthor=M.+Hennig&title=X-Ray+Free+Electron+Laser%3A+Opportunities+for+Drug+Discovery&doi=10.1042%2FEBC20170031"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">X-ray free electron laser: opportunities for drug discovery</span></div><div class="casAuthors">Cheng Robert K Y; Abela Rafael; Hennig Michael</div><div class="citationInfo"><span class="NLM_cas:title">Essays in biochemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">529-542</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Past decades have shown the impact of structural information derived from complexes of drug candidates with their protein targets to facilitate the discovery of safe and effective medicines.  Despite recent developments in single particle cryo-electron microscopy, X-ray crystallography has been the main method to derive structural information.  The unique properties of X-ray free electron laser (XFEL) with unmet peak brilliance and beam focus allow X-ray diffraction data recording and successful structure determination from smaller and weaker diffracting crystals shortening timelines in crystal optimization.  To further capitalize on the XFEL advantage, innovations in crystal sample delivery for the X-ray experiment, data collection and processing methods are required.  This development was a key contributor to serial crystallography allowing structure determination at room temperature yielding physiologically more relevant structures.  Adding the time resolution provided by the femtosecond X-ray pulse will enable monitoring and capturing of dynamic processes of ligand binding and associated conformational changes with great impact to the design of candidate drug compounds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTqZGQ3bfUY0bxanA7vODO6fW6udTcc2eYovyIYFm-JxLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1M7pt1Glsg%253D%253D&md5=1b74caa7a7bbf89ca30736d76a02588e</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1042%2FEBC20170031&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252FEBC20170031%26sid%3Dliteratum%253Aachs%26aulast%3DCheng%26aufirst%3DR.%2BK.%2BY.%26aulast%3DAbela%26aufirst%3DR.%26aulast%3DHennig%26aufirst%3DM.%26atitle%3DX-Ray%2520Free%2520Electron%2520Laser%253A%2520Opportunities%2520for%2520Drug%2520Discovery%26jtitle%3DEssays%2520Biochem.%26date%3D2017%26volume%3D61%26spage%3D529%26epage%3D542%26doi%3D10.1042%2FEBC20170031" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Boland, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barford, D.</span></span> <span> </span><span class="NLM_article-title">The Potential of Cryo-Electron Microscopy for Structure-Based Drug Design</span>. <i>Essays Biochem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">543</span>– <span class="NLM_lpage">560</span>, <span class="refDoi"> DOI: 10.1042/EBC20170032</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1042%2FEBC20170032" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=29118099" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A280%3ADC%252BC1M7pt1Glsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2017&pages=543-560&author=A.+Bolandauthor=L.+Changauthor=D.+Barford&title=The+Potential+of+Cryo-Electron+Microscopy+for+Structure-Based+Drug+Design&doi=10.1042%2FEBC20170032"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">The potential of cryo-electron microscopy for structure-based drug design</span></div><div class="casAuthors">Boland Andreas; Chang Leifu; Barford David</div><div class="citationInfo"><span class="NLM_cas:title">Essays in biochemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">543-560</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Structure-based drug design plays a central role in therapeutic development.  Until recently, protein crystallography and NMR have dominated experimental approaches to obtain structural information of biological molecules.  However, in recent years rapid technical developments in single particle cryo-electron microscopy (cryo-EM) have enabled the determination to near-atomic resolution of macromolecules ranging from large multi-subunit molecular machines to proteins as small as 64 kDa.  These advances have revolutionized structural biology by hugely expanding both the range of macromolecules whose structures can be determined, and by providing a description of macromolecular dynamics.  Cryo-EM is now poised to similarly transform the discipline of structure-based drug discovery.  This article reviews the potential of cryo-EM for drug discovery with reference to protein ligand complex structures determined using this technique.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTqZGQ3bfUY0c-crCu0nI_sfW6udTcc2eYovyIYFm-JxLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1M7pt1Glsw%253D%253D&md5=a5bdc609a7330f1b01dd7ef921c4afd4</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1042%2FEBC20170032&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252FEBC20170032%26sid%3Dliteratum%253Aachs%26aulast%3DBoland%26aufirst%3DA.%26aulast%3DChang%26aufirst%3DL.%26aulast%3DBarford%26aufirst%3DD.%26atitle%3DThe%2520Potential%2520of%2520Cryo-Electron%2520Microscopy%2520for%2520Structure-Based%2520Drug%2520Design%26jtitle%3DEssays%2520Biochem.%26date%3D2017%26volume%3D61%26spage%3D543%26epage%3D560%26doi%3D10.1042%2FEBC20170032" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cramer, R. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soltanshahi, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jilek, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campbell, B.</span></span> <span> </span><span class="NLM_article-title">AllChem: Generating and Searching 1020 Synthetically Accessible Structures</span>. <i>J. Comput.-Aided Mol. Des.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">341</span>– <span class="NLM_lpage">350</span>, <span class="refDoi"> DOI: 10.1007/s10822-006-9093-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1007%2Fs10822-006-9093-8" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2007&pages=341-350&author=R.+D.+Cramerauthor=F.+Soltanshahiauthor=R.+Jilekauthor=B.+Campbell&title=AllChem%3A+Generating+and+Searching+1020+Synthetically+Accessible+Structures&doi=10.1007%2Fs10822-006-9093-8"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1007%2Fs10822-006-9093-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-006-9093-8%26sid%3Dliteratum%253Aachs%26aulast%3DCramer%26aufirst%3DR.%2BD.%26aulast%3DSoltanshahi%26aufirst%3DF.%26aulast%3DJilek%26aufirst%3DR.%26aulast%3DCampbell%26aufirst%3DB.%26atitle%3DAllChem%253A%2520Generating%2520and%2520Searching%25201020%2520Synthetically%2520Accessible%2520Structures%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2007%26volume%3D21%26spage%3D341%26epage%3D350%26doi%3D10.1007%2Fs10822-006-9093-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Macarron, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Banks, M. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bojanic, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burns, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cirovic, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garyantes, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Green, D. V. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hertzberg, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Janzen, W. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paslay, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schopfer, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sittampalam, G. S.</span></span> <span> </span><span class="NLM_article-title">Impact of High-Throughput Screening in Biomedical Research</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">188</span>– <span class="NLM_lpage">195</span>, <span class="refDoi"> DOI: 10.1038/nrd3368</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1038%2Fnrd3368" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=21358738" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC3MXisFensbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2011&pages=188-195&author=R.+Macarronauthor=M.+N.+Banksauthor=D.+Bojanicauthor=D.+J.+Burnsauthor=D.+A.+Cirovicauthor=T.+Garyantesauthor=D.+V.+S.+Greenauthor=R.+P.+Hertzbergauthor=W.+P.+Janzenauthor=J.+W.+Paslayauthor=U.+Schopferauthor=G.+S.+Sittampalam&title=Impact+of+High-Throughput+Screening+in+Biomedical+Research&doi=10.1038%2Fnrd3368"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Impact of high-throughput screening in biomedical research</span></div><div class="casAuthors">Macarron, Ricardo; Banks, Martyn N.; Bojanic, Dejan; Burns, David J.; Cirovic, Dragan A.; Garyantes, Tina; Green, Darren V. S.; Hertzberg, Robert P.; Janzen, William P.; Paslay, Jeff W.; Schopfer, Ulrich; Sittampalam, G. Sitta</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">188-195</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  High-throughput screening (HTS) has been postulated in several quarters to be a contributory factor to the decline in productivity in the pharmaceutical industry.  Moreover, it has been blamed for stifling the creativity that drug discovery demands.  In this article, we aim to dispel these myths and present the case for the use of HTS as part of a proven scientific tool kit, the wider use of which is essential for the discovery of new chemotypes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpLnrSVehADbLVg90H21EOLACvtfcHk0lgywIuTSIf8rw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXisFensbY%253D&md5=cb9853227506f3a3d774791cff9fd79d</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1038%2Fnrd3368&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd3368%26sid%3Dliteratum%253Aachs%26aulast%3DMacarron%26aufirst%3DR.%26aulast%3DBanks%26aufirst%3DM.%2BN.%26aulast%3DBojanic%26aufirst%3DD.%26aulast%3DBurns%26aufirst%3DD.%2BJ.%26aulast%3DCirovic%26aufirst%3DD.%2BA.%26aulast%3DGaryantes%26aufirst%3DT.%26aulast%3DGreen%26aufirst%3DD.%2BV.%2BS.%26aulast%3DHertzberg%26aufirst%3DR.%2BP.%26aulast%3DJanzen%26aufirst%3DW.%2BP.%26aulast%3DPaslay%26aufirst%3DJ.%2BW.%26aulast%3DSchopfer%26aufirst%3DU.%26aulast%3DSittampalam%26aufirst%3DG.%2BS.%26atitle%3DImpact%2520of%2520High-Throughput%2520Screening%2520in%2520Biomedical%2520Research%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2011%26volume%3D10%26spage%3D188%26epage%3D195%26doi%3D10.1038%2Fnrd3368" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Besnard, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hopkins, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pannifer, A. D.</span></span> <span> </span><span class="NLM_article-title">The Joint European Compound Library: Boosting Precompetitive Research</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">181</span>– <span class="NLM_lpage">186</span>, <span class="refDoi"> DOI: 10.1016/j.drudis.2014.08.014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1016%2Fj.drudis.2014.08.014" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=25205347" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsFGgsLbE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2015&pages=181-186&author=J.+Besnardauthor=P.+S.+Jonesauthor=A.+L.+Hopkinsauthor=A.+D.+Pannifer&title=The+Joint+European+Compound+Library%3A+Boosting+Precompetitive+Research&doi=10.1016%2Fj.drudis.2014.08.014"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">The Joint European Compound Library: boosting precompetitive research</span></div><div class="casAuthors">Besnard, Jeremy; Jones, Philip S.; Hopkins, Andrew L.; Pannifer, Andrew D.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">181-186</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">The Joint European Compd. Library (JECL) is a new high-throughput screening collection aimed at driving precompetitive drug discovery and target validation.  The JECL has been established with a core of over 321 000 compds. from the proprietary collections of seven pharmaceutical companies and will expand to around 500 000 compds.  Here, we analyze the physicochem. profile and chem. diversity of the core collection, showing that the collection is diverse and has a broad spectrum of predicted biol. activity.  We also describe a model for sharing compd. information from multiple proprietary collections, enabling diversity and quality anal. without disclosing structures.  The JECL is available for screening at no cost to European academic labs. and SMEs through the IMI European Lead Factory (http://www.europeanleadfactory.eu/).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrYUIRI3z5DAbVg90H21EOLACvtfcHk0linLr2X0D9bhQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsFGgsLbE&md5=cbb46ec6091685a0f9908e5100c46c9c</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2014.08.014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2014.08.014%26sid%3Dliteratum%253Aachs%26aulast%3DBesnard%26aufirst%3DJ.%26aulast%3DJones%26aufirst%3DP.%2BS.%26aulast%3DHopkins%26aufirst%3DA.%2BL.%26aulast%3DPannifer%26aufirst%3DA.%2BD.%26atitle%3DThe%2520Joint%2520European%2520Compound%2520Library%253A%2520Boosting%2520Precompetitive%2520Research%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2015%26volume%3D20%26spage%3D181%26epage%3D186%26doi%3D10.1016%2Fj.drudis.2014.08.014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hamley, P.</span>; <span class="NLM_string-name">Jimonet, P.</span></span> <span> </span><span class="NLM_article-title">Advances in the Understanding of Drug Properties in Medicinal Chemistry</span>. In  <i>Small Molecule Medicinal Chemistry</i>; <span class="NLM_contrib-group"><span class="NLM_string-name">Czechtizky, W.</span>, <span class="NLM_string-name">Hamley, P.</span></span>, Eds.; <span class="NLM_publisher-name">John Wiley & Sons, Inc</span>: <span class="NLM_publisher-loc">Hoboken, NJ</span>, <span class="NLM_year">2015</span>; pp  <span class="NLM_fpage">305</span>– <span class="NLM_lpage">322</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1002%2F9781118771723.ch11" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2015&pages=305-322&author=P.+Hamley&author=P.+Jimonetauthor=W.+Czechtizky&author=P.+Hamley&title=Small+Molecule+Medicinal+Chemistry"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1002%2F9781118771723.ch11&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1002%252F9781118771723.ch11%26sid%3Dliteratum%253Aachs%26aulast%3DHamley%26aufirst%3DP.%26atitle%3DAdvances%2520in%2520the%2520Understanding%2520of%2520Drug%2520Properties%2520in%2520Medicinal%2520Chemistry%26btitle%3DSmall%2520Molecule%2520Medicinal%2520Chemistry%26aulast%3DCzechtizky%26aufirst%3DW.%26pub%3DJohn%2520Wiley%2520%2526%2520Sons%252C%2520Inc%26date%3D2015%26spage%3D305%26epage%3D322" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lerner, R. A.</span></span> <span> </span><span class="NLM_article-title">Combinatorial Antibody Libraries: New Advances, New Immunological Insights</span>. <i>Nat. Rev. Immunol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">498</span>– <span class="NLM_lpage">508</span>, <span class="refDoi"> DOI: 10.1038/nri.2016.67</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1038%2Fnri.2016.67" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=27374636" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtV2ntLvK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2016&pages=498-508&author=R.+A.+Lerner&title=Combinatorial+Antibody+Libraries%3A+New+Advances%2C+New+Immunological+Insights&doi=10.1038%2Fnri.2016.67"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Combinatorial antibody libraries: new advances, new immunological insights</span></div><div class="casAuthors">Lerner, Richard A.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Immunology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">498-508</span>CODEN:
                <span class="NLM_cas:coden">NRIABX</span>;
        ISSN:<span class="NLM_cas:issn">1474-1733</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Immunochemists have become quite proficient in engineering existing antibody mols. to control their pharmacol. properties.  However, in terms of generating new antibodies, the combinatorial antibody library has become a central feature of modern immunochem.  These libraries are essentially an immune system in a test tube and enable the selection of antibodies without the constraints of whole animal or cell-based systems.  This Review provides an overview of how antibody libraries are constructed and discusses what can be learnt from these synthetic systems.  In particular, the Review focuses on new biol. insights from antibody libraries - such as the concept of 'SOS antibodies' - and the growing use of intracellular antibodies to perturb cellular functions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoEEd11DiD6mLVg90H21EOLACvtfcHk0linLr2X0D9bhQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtV2ntLvK&md5=5674760f51d0ded0d280b6f0758d1409</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1038%2Fnri.2016.67&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnri.2016.67%26sid%3Dliteratum%253Aachs%26aulast%3DLerner%26aufirst%3DR.%2BA.%26atitle%3DCombinatorial%2520Antibody%2520Libraries%253A%2520New%2520Advances%252C%2520New%2520Immunological%2520Insights%26jtitle%3DNat.%2520Rev.%2520Immunol.%26date%3D2016%26volume%3D16%26spage%3D498%26epage%3D508%26doi%3D10.1038%2Fnri.2016.67" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Darmostuk, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rimpelova, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gbelcova, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruml, T.</span></span> <span> </span><span class="NLM_article-title">Current Approaches in SELEX: An Update to Aptamer Selection Technology</span>. <i>Biotechnol. Adv.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>33</i></span>,  <span class="NLM_fpage">1141</span>– <span class="NLM_lpage">1161</span>, <span class="refDoi"> DOI: 10.1016/j.biotechadv.2015.02.008</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1016%2Fj.biotechadv.2015.02.008" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=25708387" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC2MXjslylsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2015&pages=1141-1161&author=M.+Darmostukauthor=S.+Rimpelovaauthor=H.+Gbelcovaauthor=T.+Ruml&title=Current+Approaches+in+SELEX%3A+An+Update+to+Aptamer+Selection+Technology&doi=10.1016%2Fj.biotechadv.2015.02.008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Current approaches in SELEX: An update to aptamer selection technology</span></div><div class="casAuthors">Darmostuk, Mariia; Rimpelova, Silvie; Gbelcova, Helena; Ruml, Tomas</div><div class="citationInfo"><span class="NLM_cas:title">Biotechnology Advances</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">6_Part_2</span>),
    <span class="NLM_cas:pages">1141-1161</span>CODEN:
                <span class="NLM_cas:coden">BIADDD</span>;
        ISSN:<span class="NLM_cas:issn">0734-9750</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">Systematic evolution of ligands by exponential enrichment (SELEX) is a well-established and efficient technol. for the generation of oligonucleotides with a high target affinity.  These SELEX-derived single stranded DNA and RNA mols., called aptamers, were selected against various targets, such as proteins, cells, microorganisms, chem. compds. etc.  They have a great potential in the use as novel antibodies, in cancer theragnostics and in biomedical research.  Vast interest in aptamers stimulated continuous development of SELEX, which underwent numerous modifications since its first application in 1990.  Novel modifications made the selection process more efficient, cost-effective and significantly less time-consuming.  This article brings a comprehensive and up-to-date review of recent advances in SELEX methods and pinpoints advantages, main obstacles and limitations.  The post-SELEX strategies and examples of application are also briefly outlined in this review.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpFs2pUhH9mnrVg90H21EOLACvtfcHk0linLr2X0D9bhQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXjslylsb4%253D&md5=dc2fdb1a71c8f5eb7a77ce6d14bf4f28</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1016%2Fj.biotechadv.2015.02.008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.biotechadv.2015.02.008%26sid%3Dliteratum%253Aachs%26aulast%3DDarmostuk%26aufirst%3DM.%26aulast%3DRimpelova%26aufirst%3DS.%26aulast%3DGbelcova%26aufirst%3DH.%26aulast%3DRuml%26aufirst%3DT.%26atitle%3DCurrent%2520Approaches%2520in%2520SELEX%253A%2520An%2520Update%2520to%2520Aptamer%2520Selection%2520Technology%26jtitle%3DBiotechnol.%2520Adv.%26date%3D2015%26volume%3D33%26spage%3D1141%26epage%3D1161%26doi%3D10.1016%2Fj.biotechadv.2015.02.008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, R.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thapa, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Espiritu, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Menon, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bingham, J.-P.</span></span> <span> </span><span class="NLM_article-title">From Nature to Creation: Going around in Circles, the Art of Peptide Cyclization</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">1135</span>– <span class="NLM_lpage">1150</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2017.11.017</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1016%2Fj.bmc.2017.11.017" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=29295762" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitlWhtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2018&pages=1135-1150&author=R.-Y.+Zhangauthor=P.+Thapaauthor=M.+J.+Espirituauthor=V.+Menonauthor=J.-P.+Bingham&title=From+Nature+to+Creation%3A+Going+around+in+Circles%2C+the+Art+of+Peptide+Cyclization&doi=10.1016%2Fj.bmc.2017.11.017"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">From Nature to creation: Going around in circles, the art of peptide cyclization</span></div><div class="casAuthors">Zhang, Rui-Yang; Thapa, Parashar; Espiritu, Michael J.; Menon, Vinay; Bingham, Jon-Paul</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1135-1150</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Cyclic peptides and cyclotides are becoming common identities within present efforts seen in peptide engineering as we seek approaches to achieve potent biol. activity, pharmacol. selectivity, structurally stability, and oral bioavailability.  Yet, this unique family of peptides has faced uncommon hurdles in their discovery, synthesis, and bioengineering, retaining characteristics that truly deviate these from their linear counterparts.  Here, we take a board spectrum approach to introduce this novel family of biomols. and the troubles that they face in their sequence and disulfide connectivity assignment, together highlighting the present combined strategies involved in cyclic peptide/cyclotide synthesis and modification.  These efforts have circumvented otherwise impossible hurdles in their manipulation and prodn. that are only now advancing cyclic peptides/cyclotides as research probes and future pharmaceutical templates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqmjnZmnb594LVg90H21EOLACvtfcHk0ljLXo4hNBs9Ew"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitlWhtw%253D%253D&md5=45e70431f57bc151655776f103acd39c</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2017.11.017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2017.11.017%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DR.-Y.%26aulast%3DThapa%26aufirst%3DP.%26aulast%3DEspiritu%26aufirst%3DM.%2BJ.%26aulast%3DMenon%26aufirst%3DV.%26aulast%3DBingham%26aufirst%3DJ.-P.%26atitle%3DFrom%2520Nature%2520to%2520Creation%253A%2520Going%2520around%2520in%2520Circles%252C%2520the%2520Art%2520of%2520Peptide%2520Cyclization%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2018%26volume%3D26%26spage%3D1135%26epage%3D1150%26doi%3D10.1016%2Fj.bmc.2017.11.017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Heinis, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rutherford, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Freund, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winter, G.</span></span> <span> </span><span class="NLM_article-title">Phage-Encoded Combinatorial Chemical Libraries Based on Bicyclic Peptides</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">502</span>– <span class="NLM_lpage">507</span>, <span class="refDoi"> DOI: 10.1038/nchembio.184</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1038%2Fnchembio.184" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=19483697" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BD1MXms1altbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2009&pages=502-507&author=C.+Heinisauthor=T.+Rutherfordauthor=S.+Freundauthor=G.+Winter&title=Phage-Encoded+Combinatorial+Chemical+Libraries+Based+on+Bicyclic+Peptides&doi=10.1038%2Fnchembio.184"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Phage-encoded combinatorial chemical libraries based on bicyclic peptides</span></div><div class="casAuthors">Heinis, Christian; Rutherford, Trevor; Freund, Stephan; Winter, Greg</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">502-507</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Here we describe a phage strategy for the selection of ligands based on bicyclic or linear peptides attached covalently to an org. core.  We designed peptide repertoires with three reactive cysteine residues, each spaced apart by several random amino acid residues, and we fused the repertoires to the phage gene-3-protein.  Conjugation with tris-(bromomethyl)benzene via the reactive cysteines generated repertoires of peptide conjugates with two peptide loops anchored to a mesitylene core.  Iterative affinity selections yielded several enzyme inhibitors; after further mutagenesis and selection, we were able to chem. synthesize a lead inhibitor (PK15; Ki = 1.5 nM) specific to human plasma kallikrein that efficiently interrupted the intrinsic coagulation pathway in human plasma tested ex vivo.  This approach offers a powerful means of generating and selecting bicyclic macrocycles (or if cleaved, linear derivs. thereof) as ligands poised at the interface of small-mol. drugs and biologics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrZhJRZyldisLVg90H21EOLACvtfcHk0ljLXo4hNBs9Ew"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXms1altbs%253D&md5=0634460f77b185fbcf85309c6dbf93e3</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.184&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.184%26sid%3Dliteratum%253Aachs%26aulast%3DHeinis%26aufirst%3DC.%26aulast%3DRutherford%26aufirst%3DT.%26aulast%3DFreund%26aufirst%3DS.%26aulast%3DWinter%26aufirst%3DG.%26atitle%3DPhage-Encoded%2520Combinatorial%2520Chemical%2520Libraries%2520Based%2520on%2520Bicyclic%2520Peptides%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2009%26volume%3D5%26spage%3D502%26epage%3D507%26doi%3D10.1038%2Fnchembio.184" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morales-Sanfrutos, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Angelini, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cutting, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heinis, C.</span></span> <span> </span><span class="NLM_article-title">Structurally Diverse Cyclisation Linkers Impose Different Backbone Conformations in Bicyclic Peptides</span>. <i>ChemBioChem</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">1032</span>– <span class="NLM_lpage">1038</span>, <span class="refDoi"> DOI: 10.1002/cbic.201200049</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1002%2Fcbic.201200049" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=22492661" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC38Xlt1Kltb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2012&pages=1032-1038&author=S.+Chenauthor=J.+Morales-Sanfrutosauthor=A.+Angeliniauthor=B.+Cuttingauthor=C.+Heinis&title=Structurally+Diverse+Cyclisation+Linkers+Impose+Different+Backbone+Conformations+in+Bicyclic+Peptides&doi=10.1002%2Fcbic.201200049"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Structurally Diverse Cyclisation Linkers Impose Different Backbone Conformations in Bicyclic Peptides</span></div><div class="casAuthors">Chen, Shiyu; Morales-Sanfrutos, Julia; Angelini, Alessandro; Cutting, Brian; Heinis, Christian</div><div class="citationInfo"><span class="NLM_cas:title">ChemBioChem</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1032-1038</span>CODEN:
                <span class="NLM_cas:coden">CBCHFX</span>;
        ISSN:<span class="NLM_cas:issn">1439-4227</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Combinatorial libraries of structurally diverse peptide macrocycles offer a rich source for the development of high-affinity ligands to targets of interest.  In this work we have developed linkers for the generation of genetically encoded bicyclic peptides and tested whether the peptides cyclised by them have significant variations in their backbone conformations.  Two new cyclisation reagents, each contg. three thiol-reactive groups, efficiently and selectively cyclised linear peptides contg. three cysteine moieties.  When the mesitylene linker of the bicyclic peptide PK15, a potent inhibitor of plasma kallikrein (Ki=2 nM), was replaced by the new linkers, its inhibitory activity dropped by a factor of more than 1000, suggesting that the linkers impose different conformations on the peptide.  Indeed, structural anal. by soln.-state NMR revealed different NOE constraints in the three bicyclic peptides, indicating that these relatively small linkers at the centers of bicyclic peptide structures significantly influence the conformations of the peptides.  These results demonstrate the prominent structural role of linkers in peptide macrocycles and suggest that application of different cyclisation linkers in a combinatorial fashion could be an attractive means to generate topol. diverse macrocycle libraries.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGphdCp8KV5HeLVg90H21EOLACvtfcHk0ljLXo4hNBs9Ew"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xlt1Kltb4%253D&md5=ddb3f7e338a22ea1a1e3ced71624781d</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1002%2Fcbic.201200049&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcbic.201200049%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DS.%26aulast%3DMorales-Sanfrutos%26aufirst%3DJ.%26aulast%3DAngelini%26aufirst%3DA.%26aulast%3DCutting%26aufirst%3DB.%26aulast%3DHeinis%26aufirst%3DC.%26atitle%3DStructurally%2520Diverse%2520Cyclisation%2520Linkers%2520Impose%2520Different%2520Backbone%2520Conformations%2520in%2520Bicyclic%2520Peptides%26jtitle%3DChemBioChem%26date%3D2012%26volume%3D13%26spage%3D1032%26epage%3D1038%26doi%3D10.1002%2Fcbic.201200049" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rebollo, I. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Angelini, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heinis, C.</span></span> <span> </span><span class="NLM_article-title">Phage Display Libraries of Differently Sized Bicyclic Peptides</span>. <i>MedChemComm</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">145</span>– <span class="NLM_lpage">150</span>, <span class="refDoi"> DOI: 10.1039/C2MD20171B</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1039%2FC2MD20171B" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvVymsr%252FO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2013&pages=145-150&author=I.+R.+Rebolloauthor=A.+Angeliniauthor=C.+Heinis&title=Phage+Display+Libraries+of+Differently+Sized+Bicyclic+Peptides&doi=10.1039%2FC2MD20171B"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Phage display libraries of differently sized bicyclic peptides</span></div><div class="casAuthors">Rebollo, Inmaculada Rentero; Angelini, Alessandro; Heinis, Christian</div><div class="citationInfo"><span class="NLM_cas:title">MedChemComm</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">145-150</span>CODEN:
                <span class="NLM_cas:coden">MCCEAY</span>;
        ISSN:<span class="NLM_cas:issn">2040-2503</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Phage selections with combinatorial libraries of uniformly sized bicyclic peptides have recently yielded potent and selective binders of several protein targets.  In this work we varied in a combinatorial fashion the ring sizes of bicyclic peptides in phage libraries, expecting that they would yield binders with higher affinities and/or more diverse binding motifs that could be affinity matured. 14 new phage peptide libraries of the format Cys-(Xaa)m-Cys-(Xaa)n-Cys (Xaa are random amino acids, m and n = 3, 4, 5 or 6) were generated and cyclized with tris-(bromomethyl)benzene.  Affinity selections against the tumor-assocd. serine protease urokinase-type plasminogen activator yielded bicyclic peptide inhibitors with a large variety of consensus sequences.  Several of the identified consensus sequences were exclusively found in bicyclic peptides having defined ring size combinations.  Some of these peptides may bind in orientations that allow affinity maturation of non-conserved regions, while others do not.  Having available multiple leads isolated from such bicyclic peptide libraries with variable ring sizes could therefore be a great asset for the generation of high affinity binders.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrYZtLlWd3zQbVg90H21EOLACvtfcHk0lh4POYM_A7MWw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvVymsr%252FO&md5=26f43d83d89ab90d12331b3614dd2dad</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1039%2FC2MD20171B&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC2MD20171B%26sid%3Dliteratum%253Aachs%26aulast%3DRebollo%26aufirst%3DI.%2BR.%26aulast%3DAngelini%26aufirst%3DA.%26aulast%3DHeinis%26aufirst%3DC.%26atitle%3DPhage%2520Display%2520Libraries%2520of%2520Differently%2520Sized%2520Bicyclic%2520Peptides%26jtitle%3DMedChemComm%26date%3D2013%26volume%3D4%26spage%3D145%26epage%3D150%26doi%3D10.1039%2FC2MD20171B" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rentero
Rebollo, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCallin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bertoldo, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Entenza, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moreillon, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heinis, C.</span></span> <span> </span><span class="NLM_article-title">Development of Potent and Selective S. Aureus Sortase A Inhibitors Based on Peptide Macrocycles</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">606</span>– <span class="NLM_lpage">611</span>, <span class="refDoi"> DOI: 10.1021/acsmedchemlett.6b00045</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.6b00045" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC28XmtFShu78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=606-611&author=I.+Rentero%0ARebolloauthor=S.+McCallinauthor=D.+Bertoldoauthor=J.+M.+Entenzaauthor=P.+Moreillonauthor=C.+Heinis&title=Development+of+Potent+and+Selective+S.+Aureus+Sortase+A+Inhibitors+Based+on+Peptide+Macrocycles&doi=10.1021%2Facsmedchemlett.6b00045"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Development of Potent and Selective S. aureus Sortase A Inhibitors Based on Peptide Macrocycles</span></div><div class="casAuthors">Rentero Rebollo, Inmaculada; McCallin, Shawna; Bertoldo, Davide; Entenza, Jose Manuel; Moreillon, Philippe; Heinis, Christian</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">606-611</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Sortases are transpeptidase enzymes that anchor surface proteins, including virulence factors, to the cell wall of Gram-pos. bacteria, and they are potential targets for the development of anti-infective agents.  While several large compd. libraries were searched by high-throughput screening, no high-affinity inhibitors of sortases could be developed to date.  Here, we applied phage display to screen billions of peptide macrocycles against sortase A (SrtA) of Staphylococcus aureus (S. aureus).  We were able to identify potent and selective inhibitors of SrtA that blocked SrtA-mediated anchoring of synthetic substrates to the surface of live S. aureus cells.  A region present in all inhibitory peptides (Leu-Pro-Pro) resembled the natural substrates of SrtA (Leu-Pro-Xaa-Thr-Gly), suggesting that the macrocycles bind to the enzyme's active site and that they form similar mol. contacts as natural substrates.  The evolved peptide macrocycles may be used as lead structures for the development of potent peptidomimetic SrtA inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqYzcM3aIF9WrVg90H21EOLACvtfcHk0lh4POYM_A7MWw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XmtFShu78%253D&md5=0768ae2e5722bac33b68e99272be06e4</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.6b00045&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.6b00045%26sid%3Dliteratum%253Aachs%26aulast%3DRentero%2BRebollo%26aufirst%3DI.%26aulast%3DMcCallin%26aufirst%3DS.%26aulast%3DBertoldo%26aufirst%3DD.%26aulast%3DEntenza%26aufirst%3DJ.%2BM.%26aulast%3DMoreillon%26aufirst%3DP.%26aulast%3DHeinis%26aufirst%3DC.%26atitle%3DDevelopment%2520of%2520Potent%2520and%2520Selective%2520S.%2520Aureus%2520Sortase%2520A%2520Inhibitors%2520Based%2520on%2520Peptide%2520Macrocycles%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2016%26volume%3D7%26spage%3D606%26epage%3D611%26doi%3D10.1021%2Facsmedchemlett.6b00045" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Diderich, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heinis, C.</span></span> <span> </span><span class="NLM_article-title">Phage Selection of Bicyclic Peptides Binding Her2</span>. <i>Tetrahedron</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>70</i></span>,  <span class="NLM_fpage">7733</span>– <span class="NLM_lpage">7739</span>, <span class="refDoi"> DOI: 10.1016/j.tet.2014.05.106</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1016%2Fj.tet.2014.05.106" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtVWgsr%252FE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=70&publication_year=2014&pages=7733-7739&author=P.+Diderichauthor=C.+Heinis&title=Phage+Selection+of+Bicyclic+Peptides+Binding+Her2&doi=10.1016%2Fj.tet.2014.05.106"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Phage selection of bicyclic peptides binding Her2</span></div><div class="casAuthors">Diderich, Philippe; Heinis, Christian</div><div class="citationInfo"><span class="NLM_cas:title">Tetrahedron</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">70</span>
        (<span class="NLM_cas:issue">42</span>),
    <span class="NLM_cas:pages">7733-7739</span>CODEN:
                <span class="NLM_cas:coden">TETRAB</span>;
        ISSN:<span class="NLM_cas:issn">0040-4020</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Aberrant expression of the epidermal growth factor receptor Her2 has been implicated in various malignancies including breast cancer.  Monoclonal antibodies and an antibody-drug conjugate targeting Her2 have found wide clin. application.  Herein, we aimed at developing Her2-specifc ligands based on peptides that have a 100-fold smaller mol. wt. than antibodies.  Such peptides could potentially offer advantages in the development of ligand-drug conjugates, such as ease of synthesis and conjugation, higher mol.-per-mass ratios, and better tumor penetration.  Panning of large bicyclic peptide phage display libraries against Her2 yielded a range of Her2-specific ligands having different formats and binding motifs.  Strong sequence similarities among several of the isolated peptides indicated that they interact with Her2 in a specific manner.  The best bicyclic peptide obtained after affinity maturation bound Her2 with a KD of 304 nM.  The diverse peptide ligands may offer valuable starting points for the development of high-affinity Her2 binders with potential application for tumor imaging and therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrBqi6pJaeyNLVg90H21EOLACvtfcHk0lh4POYM_A7MWw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtVWgsr%252FE&md5=33a503998bb2006f44f277b8d7e425ce</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1016%2Fj.tet.2014.05.106&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tet.2014.05.106%26sid%3Dliteratum%253Aachs%26aulast%3DDiderich%26aufirst%3DP.%26aulast%3DHeinis%26aufirst%3DC.%26atitle%3DPhage%2520Selection%2520of%2520Bicyclic%2520Peptides%2520Binding%2520Her2%26jtitle%3DTetrahedron%26date%3D2014%26volume%3D70%26spage%3D7733%26epage%3D7739%26doi%3D10.1016%2Fj.tet.2014.05.106" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Urech-Varenne, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Radtke, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heinis, C.</span></span> <span> </span><span class="NLM_article-title">Phage Selection of Bicyclic Peptide Ligands of the Notch1 Receptor</span>. <i>ChemMedChem</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">1754</span>– <span class="NLM_lpage">1761</span>, <span class="refDoi"> DOI: 10.1002/cmdc.201500261</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1002%2Fcmdc.201500261" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=26381087" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsFWrtLnO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=1754-1761&author=C.+Urech-Varenneauthor=F.+Radtkeauthor=C.+Heinis&title=Phage+Selection+of+Bicyclic+Peptide+Ligands+of+the+Notch1+Receptor&doi=10.1002%2Fcmdc.201500261"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Phage Selection of Bicyclic Peptide Ligands of the Notch1 Receptor</span></div><div class="casAuthors">Urech-Varenne, Charlotte; Radtke, Freddy; Heinis, Christian</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1754-1761</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Precise regulation of the Notch signaling pathway is crucial, as increases or deficiencies in signaling are assocd. with diseases, including a wide range of cancers.  Recent studies have demonstrated that monoclonal antibodies that bind and stabilize the structure of the neg. regulatory region (NRR) in the extracellular domain of the Notch receptor can inhibit Notch signaling.  In this work, we posed the question whether bicyclic peptides, being around 100-fold smaller than antibodies, can also stabilize the NRR and inhibit Notch signaling.  Bicyclic peptides that bind the NRR of human Notch1 were isolated from combinatorial libraries by phage display.  Affinity maturation yielded ligands with dissocn. const. (Kd) values as low as 150 nΜ.  The bicyclic peptides increased the melting temp. of the NRR by up to 8 °C, thus substantially stabilizing the protein structure, but they did not inhibit Notch signaling in cellular assays.  Although Notch signaling could not be inhibited, this work demonstrates that phage-selected bicyclic peptides can stabilize proteins; this capacity of bicyclic peptides may be exploited for modulating the conformation of other disease targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqVL32Vw08kRLVg90H21EOLACvtfcHk0lh4POYM_A7MWw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsFWrtLnO&md5=13300060cc9bb51acd1ed15e4cceb56d</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.201500261&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.201500261%26sid%3Dliteratum%253Aachs%26aulast%3DUrech-Varenne%26aufirst%3DC.%26aulast%3DRadtke%26aufirst%3DF.%26aulast%3DHeinis%26aufirst%3DC.%26atitle%3DPhage%2520Selection%2520of%2520Bicyclic%2520Peptide%2520Ligands%2520of%2520the%2520Notch1%2520Receptor%26jtitle%3DChemMedChem%26date%3D2015%26volume%3D10%26spage%3D1754%26epage%3D1761%26doi%3D10.1002%2Fcmdc.201500261" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bertoldo, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khan, M. M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dessen, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Held, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huelsken, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heinis, C.</span></span> <span> </span><span class="NLM_article-title">Phage Selection of Peptide Macrocycles against Β-Catenin To Interfere with Wnt Signaling</span>. <i>ChemMedChem</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">834</span>– <span class="NLM_lpage">839</span>, <span class="refDoi"> DOI: 10.1002/cmdc.201500557</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1002%2Fcmdc.201500557" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2016&pages=834-839&author=D.+Bertoldoauthor=M.+M.+G.+Khanauthor=P.+Dessenauthor=W.+Heldauthor=J.+Huelskenauthor=C.+Heinis&title=Phage+Selection+of+Peptide+Macrocycles+against+%CE%92-Catenin+To+Interfere+with+Wnt+Signaling&doi=10.1002%2Fcmdc.201500557"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.201500557&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.201500557%26sid%3Dliteratum%253Aachs%26aulast%3DBertoldo%26aufirst%3DD.%26aulast%3DKhan%26aufirst%3DM.%2BM.%2BG.%26aulast%3DDessen%26aufirst%3DP.%26aulast%3DHeld%26aufirst%3DW.%26aulast%3DHuelsken%26aufirst%3DJ.%26aulast%3DHeinis%26aufirst%3DC.%26atitle%3DPhage%2520Selection%2520of%2520Peptide%2520Macrocycles%2520against%2520%25CE%2592-Catenin%2520To%2520Interfere%2520with%2520Wnt%2520Signaling%26jtitle%3DChemMedChem%26date%3D2016%26volume%3D11%26spage%3D834%26epage%3D839%26doi%3D10.1002%2Fcmdc.201500557" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Luther, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moehle, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chevalier, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dale, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Obrecht, D.</span></span> <span> </span><span class="NLM_article-title">Protein Epitope Mimetic Macrocycles as Biopharmaceuticals</span>. <i>Curr. Opin. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>38</i></span>,  <span class="NLM_fpage">45</span>– <span class="NLM_lpage">51</span>, <span class="refDoi"> DOI: 10.1016/j.cbpa.2017.02.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1016%2Fj.cbpa.2017.02.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=28319811" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC2sXksFSiu7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=2017&pages=45-51&author=A.+Lutherauthor=K.+Moehleauthor=E.+Chevalierauthor=G.+Daleauthor=D.+Obrecht&title=Protein+Epitope+Mimetic+Macrocycles+as+Biopharmaceuticals&doi=10.1016%2Fj.cbpa.2017.02.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Protein epitope mimetic macrocycles as biopharmaceuticals</span></div><div class="casAuthors">Luther, Anatol; Moehle, Kerstin; Chevalier, Eric; Dale, Glenn; Obrecht, Daniel</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Chemical Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">45-51</span>CODEN:
                <span class="NLM_cas:coden">COCBF4</span>;
        ISSN:<span class="NLM_cas:issn">1367-5931</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Fully synthetic medium-sized macrocyclic peptides mimicking the key β-hairpin and α-helical protein epitopes relevant in many protein-protein interactions have emerged as a novel class of drugs with the potential to fill an important gap between small mols. and proteins.  Conformationally stabilized macrocyclic scaffolds represent ideal templates for medicinal chemists to incorporate bioactive peptide and protein pharmacophores in order to generate novel drugs to treat diseases with high unmet medical need.  This review describes recent approaches to design and generate large libraries of such macrocycles, for hit identification, and for their efficient optimization.  Finally, this review describes some of the most advanced protein epitope mimetic (PEM) macrocycles in clin. development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpKLA10A3orAbVg90H21EOLACvtfcHk0lhB_c-D9cx_8A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXksFSiu7Y%253D&md5=623f351d0e915d58f1ebd698da850a37</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1016%2Fj.cbpa.2017.02.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cbpa.2017.02.004%26sid%3Dliteratum%253Aachs%26aulast%3DLuther%26aufirst%3DA.%26aulast%3DMoehle%26aufirst%3DK.%26aulast%3DChevalier%26aufirst%3DE.%26aulast%3DDale%26aufirst%3DG.%26aulast%3DObrecht%26aufirst%3DD.%26atitle%3DProtein%2520Epitope%2520Mimetic%2520Macrocycles%2520as%2520Biopharmaceuticals%26jtitle%3DCurr.%2520Opin.%2520Chem.%2520Biol.%26date%3D2017%26volume%3D38%26spage%3D45%26epage%3D51%26doi%3D10.1016%2Fj.cbpa.2017.02.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Urban, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moosmeier, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aumüller, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thein, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bosma, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rink, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Groth, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zulley, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siegers, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tissot, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moll, G. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prassler, J.</span></span> <span> </span><span class="NLM_article-title">Phage Display and Selection of Lanthipeptides on the Carboxy-Terminus of the Gene-3 minor Coat Protein</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">1500</span>, <span class="refDoi"> DOI: 10.1038/s41467-017-01413-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1038%2Fs41467-017-01413-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=29138389" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A280%3ADC%252BC1M3ht1yisw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=1500&author=J.+H.+Urbanauthor=M.+A.+Moosmeierauthor=T.+Aum%C3%BCllerauthor=M.+Theinauthor=T.+Bosmaauthor=R.+Rinkauthor=K.+Grothauthor=M.+Zulleyauthor=K.+Siegersauthor=K.+Tissotauthor=G.+N.+Mollauthor=J.+Prassler&title=Phage+Display+and+Selection+of+Lanthipeptides+on+the+Carboxy-Terminus+of+the+Gene-3+minor+Coat+Protein&doi=10.1038%2Fs41467-017-01413-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Phage display and selection of lanthipeptides on the carboxy-terminus of the gene-3 minor coat protein</span></div><div class="casAuthors">Urban Johannes H; Moosmeier Markus A; Aumuller Tobias; Thein Marcus; Groth Katharina; Zulley Moritz; Siegers Katja; Tissot Kathrin; Prassler Josef; Bosma Tjibbe; Rink Rick; Moll Gert N</div><div class="citationInfo"><span class="NLM_cas:title">Nature communications</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1500</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Ribosomally synthesized and post-translationally modified peptides (RiPPs) are an emerging class of natural products with drug-like properties.  To fully exploit the potential of RiPPs as peptide drug candidates, tools for their systematic engineering are required.  Here we report the engineering of lanthipeptides, a subclass of RiPPs characterized by multiple thioether cycles that are enzymatically introduced in a regio- and stereospecific manner, by phage display.  This was achieved by heterologous co-expression of linear lanthipeptide precursors fused to the widely neglected C-terminus of the bacteriophage M13 minor coat protein pIII, rather than the conventionally used N-terminus, along with the modifying enzymes from distantly related bacteria.  We observe that C-terminal precursor peptide fusions to pIII are enzymatically modified in the cytoplasm of the producing cell and subsequently displayed as mature cyclic peptides on the phage surface.  Biopanning of large C-terminal display libraries readily identifies artificial lanthipeptide ligands specific to urokinase plasminogen activator (uPA) and streptavidin.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQsuQSi4xSarvh3YQQtAhNufW6udTcc2ealM4gLR7wmv7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1M3ht1yisw%253D%253D&md5=8e6f2d606a659541adcd2a28aea4a953</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1038%2Fs41467-017-01413-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41467-017-01413-7%26sid%3Dliteratum%253Aachs%26aulast%3DUrban%26aufirst%3DJ.%2BH.%26aulast%3DMoosmeier%26aufirst%3DM.%2BA.%26aulast%3DAum%25C3%25BCller%26aufirst%3DT.%26aulast%3DThein%26aufirst%3DM.%26aulast%3DBosma%26aufirst%3DT.%26aulast%3DRink%26aufirst%3DR.%26aulast%3DGroth%26aufirst%3DK.%26aulast%3DZulley%26aufirst%3DM.%26aulast%3DSiegers%26aufirst%3DK.%26aulast%3DTissot%26aufirst%3DK.%26aulast%3DMoll%26aufirst%3DG.%2BN.%26aulast%3DPrassler%26aufirst%3DJ.%26atitle%3DPhage%2520Display%2520and%2520Selection%2520of%2520Lanthipeptides%2520on%2520the%2520Carboxy-Terminus%2520of%2520the%2520Gene-3%2520minor%2520Coat%2520Protein%26jtitle%3DNat.%2520Commun.%26date%3D2017%26volume%3D8%26spage%3D1500%26doi%3D10.1038%2Fs41467-017-01413-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ito, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Passioura, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suga, H.</span></span> <span> </span><span class="NLM_article-title">Technologies for the Synthesis of mRNA-Encoding Libraries and Discovery of Bioactive Natural Product-Inspired Non-Traditional Macrocyclic Peptides</span>. <i>Molecules</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">3502</span>– <span class="NLM_lpage">3528</span>, <span class="refDoi"> DOI: 10.3390/molecules18033502</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.3390%2Fmolecules18033502" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=23507778" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC3sXlt1GitLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2013&pages=3502-3528&author=K.+Itoauthor=T.+Passiouraauthor=H.+Suga&title=Technologies+for+the+Synthesis+of+mRNA-Encoding+Libraries+and+Discovery+of+Bioactive+Natural+Product-Inspired+Non-Traditional+Macrocyclic+Peptides&doi=10.3390%2Fmolecules18033502"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Technologies for the synthesis of mRNA-encoding libraries and discovery of bioactive natural product-inspired non-traditional macrocyclic peptides</span></div><div class="casAuthors">Ito, Kenichiro; Passioura, Toby; Suga, Hiroaki</div><div class="citationInfo"><span class="NLM_cas:title">Molecules</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">3502-3528</span>CODEN:
                <span class="NLM_cas:coden">MOLEFW</span>;
        ISSN:<span class="NLM_cas:issn">1420-3049</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">In this review, we discuss emerging technologies for drug discovery, which yields novel mol. scaffolds based on natural product-inspired non-traditional peptides expressed using the translation machinery.  Unlike natural products, these technologies allow for constructing mRNA-encoding libraries of macrocyclic peptides contg. non-canonical sidechains and N-methyl-modified backbones.  The complexity of sequence space in such libraries reaches as high as a trillion (>1012), affording initial hits of high affinity ligands against protein targets.  Although this article comprehensively covers several related technologies, we discuss in greater detail the tech. development and advantages of the Random non-std. Peptide Integration Discovery (RaPID) system, including the recent identification of inhibitors against various therapeutic targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq3aPtwP6VYNrVg90H21EOLACvtfcHk0lhB_c-D9cx_8A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXlt1GitLw%253D&md5=71b8bd1771f6d01a7fce404ec6de8db5</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.3390%2Fmolecules18033502&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fmolecules18033502%26sid%3Dliteratum%253Aachs%26aulast%3DIto%26aufirst%3DK.%26aulast%3DPassioura%26aufirst%3DT.%26aulast%3DSuga%26aufirst%3DH.%26atitle%3DTechnologies%2520for%2520the%2520Synthesis%2520of%2520mRNA-Encoding%2520Libraries%2520and%2520Discovery%2520of%2520Bioactive%2520Natural%2520Product-Inspired%2520Non-Traditional%2520Macrocyclic%2520Peptides%26jtitle%3DMolecules%26date%3D2013%26volume%3D18%26spage%3D3502%26epage%3D3528%26doi%3D10.3390%2Fmolecules18033502" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jongkees, S. A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caner, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tysoe, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brayer, G. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Withers, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suga, H.</span></span> <span> </span><span class="NLM_article-title">Rapid Discovery of Potent and Selective Glycosidase-Inhibiting De Novo Peptides</span>. <i>Cell Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">381</span>– <span class="NLM_lpage">390</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2017.02.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1016%2Fj.chembiol.2017.02.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=28262556" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC2sXjvVOktr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2017&pages=381-390&author=S.+A.+K.+Jongkeesauthor=S.+Canerauthor=C.+Tysoeauthor=G.+D.+Brayerauthor=S.+G.+Withersauthor=H.+Suga&title=Rapid+Discovery+of+Potent+and+Selective+Glycosidase-Inhibiting+De+Novo+Peptides&doi=10.1016%2Fj.chembiol.2017.02.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Rapid Discovery of Potent and Selective Glycosidase-Inhibiting De Novo Peptides</span></div><div class="casAuthors">Jongkees, Seino A. K.; Caner, Sami; Tysoe, Christina; Brayer, Gary D.; Withers, Stephen G.; Suga, Hiroaki</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">381-390</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Human pancreatic α-amylase (HPA) is responsible for degrading starch to malto-oligosaccharides, thence to glucose, and is therefore an attractive therapeutic target for the treatment of diabetes and obesity.  Here we report the discovery of a unique lariat nonapeptide, by means of the RaPID (Random non-std. Peptides Integrated Discovery) system, composed of five amino acids in a head-to-side-chain thioether macrocycle and a further four amino acids in a 310 helical C terminus.  This is a potent inhibitor of HPA (Ki = 7 nM) yet exhibits selectivity for the target over other glycosidases tested.  Structural studies show that this nonapeptide forms a compact tertiary structure, and illustrate that a general inhibitory motif involving two phenolic groups is often accessed for tight binding of inhibitors to HPA.  Furthermore, the work reported here demonstrates the potential of this methodol. for the discovery of de novo peptide inhibitors against other glycosidases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqu0khtrOxH6bVg90H21EOLACvtfcHk0lj21QAHwU0-0w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXjvVOktr0%253D&md5=68dff6c2cd4c8fba0387a316802fc67b</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2017.02.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2017.02.001%26sid%3Dliteratum%253Aachs%26aulast%3DJongkees%26aufirst%3DS.%2BA.%2BK.%26aulast%3DCaner%26aufirst%3DS.%26aulast%3DTysoe%26aufirst%3DC.%26aulast%3DBrayer%26aufirst%3DG.%2BD.%26aulast%3DWithers%26aufirst%3DS.%2BG.%26aulast%3DSuga%26aufirst%3DH.%26atitle%3DRapid%2520Discovery%2520of%2520Potent%2520and%2520Selective%2520Glycosidase-Inhibiting%2520De%2520Novo%2520Peptides%26jtitle%3DCell%2520Chem.%2520Biol.%26date%3D2017%26volume%3D24%26spage%3D381%26epage%3D390%26doi%3D10.1016%2Fj.chembiol.2017.02.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yamagishi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shoji, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miyagawa, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kawakami, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Katoh, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goto, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suga, H.</span></span> <span> </span><span class="NLM_article-title">Natural Product-Like Macrocyclic N-Methyl-Peptide Inhibitors against a Ubiquitin Ligase Uncovered from a Ribosome-Expressed De Novo Library</span>. <i>Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">1562</span>– <span class="NLM_lpage">1570</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2011.09.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1016%2Fj.chembiol.2011.09.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=22195558" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhs12htrnI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2011&pages=1562-1570&author=Y.+Yamagishiauthor=I.+Shojiauthor=S.+Miyagawaauthor=T.+Kawakamiauthor=T.+Katohauthor=Y.+Gotoauthor=H.+Suga&title=Natural+Product-Like+Macrocyclic+N-Methyl-Peptide+Inhibitors+against+a+Ubiquitin+Ligase+Uncovered+from+a+Ribosome-Expressed+De+Novo+Library&doi=10.1016%2Fj.chembiol.2011.09.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Natural Product-Like Macrocyclic N-Methyl-Peptide Inhibitors against a Ubiquitin Ligase Uncovered from a Ribosome-Expressed De Novo Library</span></div><div class="casAuthors">Yamagishi, Yusuke; Shoji, Ikuo; Miyagawa, Shoji; Kawakami, Takashi; Katoh, Takayuki; Goto, Yuki; Suga, Hiroaki</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1562-1570</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Summary: Naturally occurring peptides often possess macrocyclic and N-methylated backbone.  These features grant them structural rigidity, high affinity to targets, proteolytic resistance, and occasionally membrane permeability.  Because such peptides are produced by either nonribosomal peptide synthetases or enzymic posttranslational modifications, it is yet a formidable challenge in degenerating sequence or length and prepg. libraries for screening bioactive mols.  Here, we report a new means of synthesizing a de novo library of "natural product-like" macrocyclic N-methyl-peptides using translation machinery under the reprogrammed genetic code, which is coupled with an in vitro display technique, referred to as RaPID (random nonstandard peptides integrated discovery) system.  This system allows for rapid selection of strong binders against an arbitrarily chosen therapeutic target.  Here, we have demonstrated the selection of anti-E6AP macrocyclic N-methyl-peptides, one of which strongly inhibits polyubiquitination of proteins such as p53.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq5j7qHsUITV7Vg90H21EOLACvtfcHk0lj21QAHwU0-0w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhs12htrnI&md5=14dd193429cd72bfb8dc87a9be5a7cf9</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2011.09.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2011.09.013%26sid%3Dliteratum%253Aachs%26aulast%3DYamagishi%26aufirst%3DY.%26aulast%3DShoji%26aufirst%3DI.%26aulast%3DMiyagawa%26aufirst%3DS.%26aulast%3DKawakami%26aufirst%3DT.%26aulast%3DKatoh%26aufirst%3DT.%26aulast%3DGoto%26aufirst%3DY.%26aulast%3DSuga%26aufirst%3DH.%26atitle%3DNatural%2520Product-Like%2520Macrocyclic%2520N-Methyl-Peptide%2520Inhibitors%2520against%2520a%2520Ubiquitin%2520Ligase%2520Uncovered%2520from%2520a%2520Ribosome-Expressed%2520De%2520Novo%2520Library%26jtitle%3DChem.%2520Biol.%26date%3D2011%26volume%3D18%26spage%3D1562%26epage%3D1570%26doi%3D10.1016%2Fj.chembiol.2011.09.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hayashi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morimoto, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suga, H.</span></span> <span> </span><span class="NLM_article-title">In Vitro Selection of Anti-Akt2 Thioether-Macrocyclic Peptides Leading to Isoform-Selective Inhibitors</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">607</span>– <span class="NLM_lpage">613</span>, <span class="refDoi"> DOI: 10.1021/cb200388k</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb200388k" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtFemtrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2012&pages=607-613&author=Y.+Hayashiauthor=J.+Morimotoauthor=H.+Suga&title=In+Vitro+Selection+of+Anti-Akt2+Thioether-Macrocyclic+Peptides+Leading+to+Isoform-Selective+Inhibitors&doi=10.1021%2Fcb200388k"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">In Vitro Selection of Anti-Akt2 Thioether-Macrocyclic Peptides Leading to Isoform-Selective Inhibitors</span></div><div class="casAuthors">Hayashi, Yuuki; Morimoto, Jumpei; Suga, Hiroaki</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">607-613</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The Akt kinase family, consisting of three isoforms in humans, is a well-validated class of drug target.  Through various screening campaigns in academics and pharmaceutical industries, several promising inhibitors have been developed to date.  However, due to the mechanistic and structural similarities of Akt kinases, it is yet a challenging task to discover selective inhibitors against a specific Akt isoform.  We here report Akt-selective and also Akt2 isoform-selective inhibitors based on a thioether-macrocyclic peptide scaffold.  Several anti-Akt2 peptides have been selected from a library by means of an in vitro display system, referred to as the RaPID (Random nonstandard Peptide Integrated Discovery) system.  Remarkably, the majority of these "binding-active" anti-Akt2 peptides turned out to be "inhibitory active", exhibiting IC50 values of approx. 100 nM.  Moreover, these peptides are not only selective to the Akt kinase family but also isoform-selective to Akt2.  Particularly, one referred to as Pakti-L1 is able to discriminate Akt2 250- and 40-fold over Akt1 and Akt3, resp.  This proof-of-concept case study suggests that the RaPID system has a tremendous potential for the discovery of unique inhibitors with high family- and isoform-selectivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrR40cWg05Nh7Vg90H21EOLACvtfcHk0lj21QAHwU0-0w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtFemtrs%253D&md5=2d4b48ab2653f7eace723a59f206af37</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1021%2Fcb200388k&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb200388k%26sid%3Dliteratum%253Aachs%26aulast%3DHayashi%26aufirst%3DY.%26aulast%3DMorimoto%26aufirst%3DJ.%26aulast%3DSuga%26aufirst%3DH.%26atitle%3DIn%2520Vitro%2520Selection%2520of%2520Anti-Akt2%2520Thioether-Macrocyclic%2520Peptides%2520Leading%2520to%2520Isoform-Selective%2520Inhibitors%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2012%26volume%3D7%26spage%3D607%26epage%3D613%26doi%3D10.1021%2Fcb200388k" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Song, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, L.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Passioura, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suga, H.</span></span> <span> </span><span class="NLM_article-title">Macrocyclic Peptide Inhibitors for the Protein-Protein Interaction of Zaire Ebola Virus Protein 24 and Karyopherin Alpha 5</span>. <i>Org. Biomol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">5155</span>– <span class="NLM_lpage">5160</span>, <span class="refDoi"> DOI: 10.1039/C7OB00012J</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1039%2FC7OB00012J" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2017&pages=5155-5160&author=X.+Songauthor=L.-Y.+Luauthor=T.+Passiouraauthor=H.+Suga&title=Macrocyclic+Peptide+Inhibitors+for+the+Protein-Protein+Interaction+of+Zaire+Ebola+Virus+Protein+24+and+Karyopherin+Alpha+5&doi=10.1039%2FC7OB00012J"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1039%2FC7OB00012J&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC7OB00012J%26sid%3Dliteratum%253Aachs%26aulast%3DSong%26aufirst%3DX.%26aulast%3DLu%26aufirst%3DL.-Y.%26aulast%3DPassioura%26aufirst%3DT.%26aulast%3DSuga%26aufirst%3DH.%26atitle%3DMacrocyclic%2520Peptide%2520Inhibitors%2520for%2520the%2520Protein-Protein%2520Interaction%2520of%2520Zaire%2520Ebola%2520Virus%2520Protein%252024%2520and%2520Karyopherin%2520Alpha%25205%26jtitle%3DOrg.%2520Biomol.%2520Chem.%26date%3D2017%26volume%3D15%26spage%3D5155%26epage%3D5160%26doi%3D10.1039%2FC7OB00012J" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Miller, M. M.</span>; <span class="NLM_string-name">Allen, M. P.</span>; <span class="NLM_string-name">Li, L.</span>; <span class="NLM_string-name">Bowsher, M. S.</span>; <span class="NLM_string-name">Gillis, E. P.</span>; <span class="NLM_string-name">Mull, E.</span>; <span class="NLM_string-name">Zhao, Q.</span>; <span class="NLM_string-name">Sun, L.-Q.</span>; <span class="NLM_string-name">Langley, D. R.</span>; <span class="NLM_string-name">Scola, P. M.</span></span> <span> </span><span class="NLM_article-title">Macrocyclic Inhibitors of the Pd-1/Pd-L1 and Cd80/Pd-L1 Protein/Protein Interactions</span>. <span class="NLM_patent">WO2017176608 (A1)</span>, <span class="NLM_year">2017</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2017&author=M.+M.+Miller&author=M.+P.+Allen&author=L.+Li&author=M.+S.+Bowsher&author=E.+P.+Gillis&author=E.+Mull&author=Q.+Zhao&author=L.-Q.+Sun&author=D.+R.+Langley&author=P.+M.+Scola&title=Macrocyclic+Inhibitors+of+the+Pd-1%2FPd-L1+and+Cd80%2FPd-L1+Protein%2FProtein+Interactions"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DMiller%26aufirst%3DM.%2BM.%26atitle%3DMacrocyclic%2520Inhibitors%2520of%2520the%2520Pd-1%252FPd-L1%2520and%2520Cd80%252FPd-L1%2520Protein%252FProtein%2520Interactions%26date%3D2017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kawamura, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Münzel, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kojima, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yapp, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhushan, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goto, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tumber, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Katoh, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">King, O. N. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Passioura, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walport, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hatch, S. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Madden, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brennan, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chowdhury, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hopkinson, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suga, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schofield, C. J.</span></span> <span> </span><span class="NLM_article-title">Highly Selective Inhibition of Histone Demethylases by de Novo Macrocyclic Peptides</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">14773</span>, <span class="refDoi"> DOI: 10.1038/ncomms14773</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1038%2Fncomms14773" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=28382930" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC2sXlvVersro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=14773&author=A.+Kawamuraauthor=M.+M%C3%BCnzelauthor=T.+Kojimaauthor=C.+Yappauthor=B.+Bhushanauthor=Y.+Gotoauthor=A.+Tumberauthor=T.+Katohauthor=O.+N.+F.+Kingauthor=T.+Passiouraauthor=L.+J.+Walportauthor=S.+B.+Hatchauthor=S.+Maddenauthor=S.+M%C3%BCllerauthor=P.+E.+Brennanauthor=R.+Chowdhuryauthor=R.+J.+Hopkinsonauthor=H.+Sugaauthor=C.+J.+Schofield&title=Highly+Selective+Inhibition+of+Histone+Demethylases+by+de+Novo+Macrocyclic+Peptides&doi=10.1038%2Fncomms14773"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Highly selective inhibition of histone demethylases by de novo macrocyclic peptides</span></div><div class="casAuthors">Kawamura, Akane; Munzel, Martin; Kojima, Tatsuya; Yapp, Clarence; Bhushan, Bhaskar; Goto, Yuki; Tumber, Anthony; Katoh, Takayuki; King, Oliver N. F.; Passioura, Toby; Walport, Louise J.; Hatch, Stephanie B.; Madden, Sarah; Muller, Susanne; Brennan, Paul E.; Chowdhury, Rasheduzzaman; Hopkinson, Richard J.; Suga, Hiroaki; Schofield, Christopher J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Communications</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">14773</span>CODEN:
                <span class="NLM_cas:coden">NCAOBW</span>;
        ISSN:<span class="NLM_cas:issn">2041-1723</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The JmjC histone demethylases (KDMs) are linked to tumor cell proliferation and are current cancer targets; however, very few highly selective inhibitors for these are available.  Here we report cyclic peptide inhibitors of the KDM4A-C with selectivity over other KDMs/2OG oxygenases, including closely related KDM4D/E isoforms.  Crystal structures and biochem. analyses of one of the inhibitors (CP2) with KDM4A reveals that CP2 binds differently to, but competes with, histone substrates in the active site.  Substitution of the active site binding arginine of CP2 to N-ε-trimethyl-lysine or methylated arginine results in cyclic peptide substrates, indicating that KDM4s may act on non-histone substrates.  Targeted modifications to CP2 based on crystallog. and mass spectrometry analyses results in variants with greater proteolytic robustness.  Peptide dosing in cells manifests KDM4A target stabilization.  Although further development is required to optimize cellular activity, the results reveal the feasibility of highly selective non-metal chelating, substrate-competitive inhibitors of the JmjC KDMs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrsOYIV1M7RVLVg90H21EOLACvtfcHk0lheubR6oKyy0Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXlvVersro%253D&md5=3e84efae6c9715ad9ecd33f42de87911</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1038%2Fncomms14773&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms14773%26sid%3Dliteratum%253Aachs%26aulast%3DKawamura%26aufirst%3DA.%26aulast%3DM%25C3%25BCnzel%26aufirst%3DM.%26aulast%3DKojima%26aufirst%3DT.%26aulast%3DYapp%26aufirst%3DC.%26aulast%3DBhushan%26aufirst%3DB.%26aulast%3DGoto%26aufirst%3DY.%26aulast%3DTumber%26aufirst%3DA.%26aulast%3DKatoh%26aufirst%3DT.%26aulast%3DKing%26aufirst%3DO.%2BN.%2BF.%26aulast%3DPassioura%26aufirst%3DT.%26aulast%3DWalport%26aufirst%3DL.%2BJ.%26aulast%3DHatch%26aufirst%3DS.%2BB.%26aulast%3DMadden%26aufirst%3DS.%26aulast%3DM%25C3%25BCller%26aufirst%3DS.%26aulast%3DBrennan%26aufirst%3DP.%2BE.%26aulast%3DChowdhury%26aufirst%3DR.%26aulast%3DHopkinson%26aufirst%3DR.%2BJ.%26aulast%3DSuga%26aufirst%3DH.%26aulast%3DSchofield%26aufirst%3DC.%2BJ.%26atitle%3DHighly%2520Selective%2520Inhibition%2520of%2520Histone%2520Demethylases%2520by%2520de%2520Novo%2520Macrocyclic%2520Peptides%26jtitle%3DNat.%2520Commun.%26date%3D2017%26volume%3D8%26spage%3D14773%26doi%3D10.1038%2Fncomms14773" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ito, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sakai, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suzuki, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ozawa, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hatta, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Natsume, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsumoto, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suga, H.</span></span> <span> </span><span class="NLM_article-title">Artificial Human Met Agonists Based on Macrocycle Scaffolds</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">6373</span>, <span class="refDoi"> DOI: 10.1038/ncomms7373</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1038%2Fncomms7373" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=25758345" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC2MXosFegu74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=6373&author=K.+Itoauthor=K.+Sakaiauthor=Y.+Suzukiauthor=N.+Ozawaauthor=T.+Hattaauthor=T.+Natsumeauthor=K.+Matsumotoauthor=H.+Suga&title=Artificial+Human+Met+Agonists+Based+on+Macrocycle+Scaffolds&doi=10.1038%2Fncomms7373"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Artificial human Met agonists based on macrocycle scaffolds</span></div><div class="casAuthors">Ito, Kenichiro; Sakai, Katsuya; Suzuki, Yoshinori; Ozawa, Naoya; Hatta, Tomohisa; Natsume, Tohru; Matsumoto, Kunio; Suga, Hiroaki</div><div class="citationInfo"><span class="NLM_cas:title">Nature Communications</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">6373</span>CODEN:
                <span class="NLM_cas:coden">NCAOBW</span>;
        ISSN:<span class="NLM_cas:issn">2041-1723</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Hepatocyte growth factor (HGF) receptor, also known as Met, is a member of the receptor tyrosine kinase family.  The Met-HGF interaction regulates various signalling pathways involving downstream kinases, such as Akt and Erk.  Met activation is implicated in wound healing of tissues via multiple biol. responses triggered by the above-mentioned signalling cascade.  Here we report the development of artificial Met-activating dimeric macrocycles.  We identify Met-binding monomeric macrocyclic peptides by means of the RaPID (random non-std. peptide integrated discovery) system, and dimerize the resp. monomers through rational design.  These dimeric macrocycles specifically and strongly activate Met signalling pathways through receptor dimerization and induce various HGF-like cellular responses, such as branching morphogenesis, in human cells.  This work suggests our approach for generating dimeric macrocycles as non-protein ligands for cell surface receptors can be useful for developing potential therapeutics with a broad range of potential applications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrAtMddCIHHmLVg90H21EOLACvtfcHk0lheubR6oKyy0Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXosFegu74%253D&md5=6b83f797358b402d7472f989b5913d0f</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1038%2Fncomms7373&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms7373%26sid%3Dliteratum%253Aachs%26aulast%3DIto%26aufirst%3DK.%26aulast%3DSakai%26aufirst%3DK.%26aulast%3DSuzuki%26aufirst%3DY.%26aulast%3DOzawa%26aufirst%3DN.%26aulast%3DHatta%26aufirst%3DT.%26aulast%3DNatsume%26aufirst%3DT.%26aulast%3DMatsumoto%26aufirst%3DK.%26aulast%3DSuga%26aufirst%3DH.%26atitle%3DArtificial%2520Human%2520Met%2520Agonists%2520Based%2520on%2520Macrocycle%2520Scaffolds%26jtitle%3DNat.%2520Commun.%26date%3D2015%26volume%3D6%26spage%3D6373%26doi%3D10.1038%2Fncomms7373" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lyamichev, V. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goodrich, L. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sullivan, E. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bannen, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Benz, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Albert, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, J. J.</span></span> <span> </span><span class="NLM_article-title">Stepwise Evolution Improves Identification of Diverse Peptides Binding to a Protein Target</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">12116</span>, <span class="refDoi"> DOI: 10.1038/s41598-017-12440-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1038%2Fs41598-017-12440-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=28935886" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A280%3ADC%252BC1M%252FhtFSguw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2017&pages=12116&author=V.+I.+Lyamichevauthor=L.+E.+Goodrichauthor=E.+H.+Sullivanauthor=R.+M.+Bannenauthor=J.+Benzauthor=T.+J.+Albertauthor=J.+J.+Patel&title=Stepwise+Evolution+Improves+Identification+of+Diverse+Peptides+Binding+to+a+Protein+Target&doi=10.1038%2Fs41598-017-12440-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Stepwise Evolution Improves Identification of Diverse Peptides Binding to a Protein Target</span></div><div class="casAuthors">Lyamichev Victor I; Goodrich Lauren E; Sullivan Eric H; Bannen Ryan M; Albert Thomas J; Patel Jigar J; Benz Joerg</div><div class="citationInfo"><span class="NLM_cas:title">Scientific reports</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">12116</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Considerable efforts have been made to develop technologies for selection of peptidic molecules that act as substrates or binders to a protein of interest.  Here we demonstrate the combination of rational peptide array library design, parallel screening and stepwise evolution, to discover novel peptide hotspots.  These hotspots can be systematically evolved to create high-affinity, high-specificity binding peptides to a protein target in a reproducible and digitally controlled process.  The method can be applied to synthesize both linear and cyclic peptides, as well as peptides composed of natural and non-natural amino acid analogs, thereby enabling screens in a much diverse chemical space.  We apply this method to stepwise evolve peptide binders to streptavidin, a protein studied for over two decades and report novel peptides that mimic key interactions of biotin to streptavidin.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSMFn84w0HfN3thwrmtH4OBfW6udTcc2ea3Tkl0bzDBgLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1M%252FhtFSguw%253D%253D&md5=22bc53b44626b80450bf0dad2b2c1976</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1038%2Fs41598-017-12440-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41598-017-12440-1%26sid%3Dliteratum%253Aachs%26aulast%3DLyamichev%26aufirst%3DV.%2BI.%26aulast%3DGoodrich%26aufirst%3DL.%2BE.%26aulast%3DSullivan%26aufirst%3DE.%2BH.%26aulast%3DBannen%26aufirst%3DR.%2BM.%26aulast%3DBenz%26aufirst%3DJ.%26aulast%3DAlbert%26aufirst%3DT.%2BJ.%26aulast%3DPatel%26aufirst%3DJ.%2BJ.%26atitle%3DStepwise%2520Evolution%2520Improves%2520Identification%2520of%2520Diverse%2520Peptides%2520Binding%2520to%2520a%2520Protein%2520Target%26jtitle%3DSci.%2520Rep.%26date%3D2017%26volume%3D7%26spage%3D12116%26doi%3D10.1038%2Fs41598-017-12440-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hewitt, W. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leung, S. S. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pye, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ponkey, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bednarek, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacobson, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lokey, R. S.</span></span> <span> </span><span class="NLM_article-title">Cell-Permeable Cyclic Peptides from Synthetic Libraries Inspired by Natural Products</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>137</i></span>,  <span class="NLM_fpage">715</span>– <span class="NLM_lpage">721</span>, <span class="refDoi"> DOI: 10.1021/ja508766b</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja508766b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC2cXitFOhu7zF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=137&publication_year=2015&pages=715-721&author=W.+M.+Hewittauthor=S.+S.+F.+Leungauthor=C.+R.+Pyeauthor=A.+R.+Ponkeyauthor=M.+Bednarekauthor=M.+P.+Jacobsonauthor=R.+S.+Lokey&title=Cell-Permeable+Cyclic+Peptides+from+Synthetic+Libraries+Inspired+by+Natural+Products&doi=10.1021%2Fja508766b"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Cell-Permeable Cyclic Peptides from Synthetic Libraries Inspired by Natural Products</span></div><div class="casAuthors">Hewitt, William M.; Leung, Siegfried S. F.; Pye, Cameron R.; Ponkey, Alexandra R.; Bednarek, Maria; Jacobson, Matthew P.; Lokey, R. Scott</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">137</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">715-721</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Drug design efforts are turning to a new generation of therapeutic targets, such as protein-protein interactions (PPIs), that had previously been considered "undruggable" by typical small mols.  There is an emerging view that accessing these targets will require mols. that are larger and more complex than typical small mol. drugs.  Here, the authors present a methodol. for the discovery of geometrically diverse, membrane permeable cyclic peptide scaffolds based on the synthesis and permeability screening of a combinatorial library, followed by deconvolution of membrane-permeable scaffolds to identify cyclic peptides with good to excellent passive cell permeabilities.  The authors use a combination of exptl. and computational approaches to investigate structure-permeability relationships in one of these scaffolds, and uncover structural and conformational factors that govern passive membrane diffusion in a related set of cyclic peptide diastereomers.  Further, the authors investigate the dependency of permeability on side-chain identity of one of these scaffolds through single-point diversifications to show the adaptability of these scaffolds toward development of permeability-biased libraries suitable for bioactivity screens.  Overall, the results demonstrate that many novel, cell permeable scaffolds exist beyond those found in extant natural products, and that such scaffolds can be rapidly identified using a combination of synthesis and deconvolution which can, in principle, be applied to any type of macrocyclic template.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq5wbqbKaUG9rVg90H21EOLACvtfcHk0lhkTtH0BtA9-Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXitFOhu7zF&md5=0d112bf0da119b6a50b55997f64ac214</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1021%2Fja508766b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja508766b%26sid%3Dliteratum%253Aachs%26aulast%3DHewitt%26aufirst%3DW.%2BM.%26aulast%3DLeung%26aufirst%3DS.%2BS.%2BF.%26aulast%3DPye%26aufirst%3DC.%2BR.%26aulast%3DPonkey%26aufirst%3DA.%2BR.%26aulast%3DBednarek%26aufirst%3DM.%26aulast%3DJacobson%26aufirst%3DM.%2BP.%26aulast%3DLokey%26aufirst%3DR.%2BS.%26atitle%3DCell-Permeable%2520Cyclic%2520Peptides%2520from%2520Synthetic%2520Libraries%2520Inspired%2520by%2520Natural%2520Products%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2015%26volume%3D137%26spage%3D715%26epage%3D721%26doi%3D10.1021%2Fja508766b" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bockus, A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lexa, K. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pye, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kalgutkar, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gardner, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hund, K. C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hewitt, W. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwochert, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glassey, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Price, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mathiowetz, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liras, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacobson, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lokey, R. S.</span></span> <span> </span><span class="NLM_article-title">Probing the Physicochemical Boundaries of Cell Permeability and Oral Bioavailability in Lipophilic Macrocycles Inspired by Natural Products</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">4581</span>– <span class="NLM_lpage">4589</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b00128</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b00128" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC2MXnvV2ku7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=4581-4589&author=A.+T.+Bockusauthor=K.+W.+Lexaauthor=C.+R.+Pyeauthor=A.+S.+Kalgutkarauthor=J.+W.+Gardnerauthor=K.+C.+R.+Hundauthor=W.+M.+Hewittauthor=J.+A.+Schwochertauthor=E.+Glasseyauthor=D.+A.+Priceauthor=A.+M.+Mathiowetzauthor=S.+Lirasauthor=M.+P.+Jacobsonauthor=R.+S.+Lokey&title=Probing+the+Physicochemical+Boundaries+of+Cell+Permeability+and+Oral+Bioavailability+in+Lipophilic+Macrocycles+Inspired+by+Natural+Products&doi=10.1021%2Facs.jmedchem.5b00128"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Probing the Physicochemical Boundaries of Cell Permeability and Oral Bioavailability in Lipophilic Macrocycles Inspired by Natural Products</span></div><div class="casAuthors">Bockus, Andrew T.; Lexa, Katrina W.; Pye, Cameron R.; Kalgutkar, Amit S.; Gardner, Jarret W.; Hund, Kathryn C. R.; Hewitt, William M.; Schwochert, Joshua A.; Glassey, Emerson; Price, David A.; Mathiowetz, Alan M.; Liras, Spiros; Jacobson, Matthew P.; Lokey, R. Scott</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">4581-4589</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Cyclic peptide natural products contain a variety of conserved, nonproteinogenic structural elements such as D-amino acids and amide N-methylation.  In addn., many cyclic peptides incorporate γ-amino acids and other elements derived from polyketide synthases.  We hypothesized that the position and orientation of these extended backbone elements impact the ADME properties of these hybrid mols., esp. their ability to cross cell membranes and avoid metabolic degrdn.  Here we report the synthesis of cyclic hexapeptide diastereomers contg. γ-amino acids (e.g., statines) and systematically investigate their structure-permeability relationships.  These compds. were much more water-sol. and, in many cases, were both more membrane permeable and more stable to liver microsomes than a similar non-statine-contg. deriv.  Permeability correlated well with the extent of intramol. hydrogen bonding obsd. in the soln. structures detd. in the low-dielec. solvent CDCl3, and one compd. showed an oral bioavailability of 21% in rat.  Thus, the incorporation of γ-amino acids offers a route to increase backbone diversity and improve ADME properties in cyclic peptide scaffolds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoXlzvMDhSrerVg90H21EOLACvtfcHk0lhkTtH0BtA9-Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXnvV2ku7s%253D&md5=b50bfd139a4abcae5ede988a5b095ce1</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b00128&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b00128%26sid%3Dliteratum%253Aachs%26aulast%3DBockus%26aufirst%3DA.%2BT.%26aulast%3DLexa%26aufirst%3DK.%2BW.%26aulast%3DPye%26aufirst%3DC.%2BR.%26aulast%3DKalgutkar%26aufirst%3DA.%2BS.%26aulast%3DGardner%26aufirst%3DJ.%2BW.%26aulast%3DHund%26aufirst%3DK.%2BC.%2BR.%26aulast%3DHewitt%26aufirst%3DW.%2BM.%26aulast%3DSchwochert%26aufirst%3DJ.%2BA.%26aulast%3DGlassey%26aufirst%3DE.%26aulast%3DPrice%26aufirst%3DD.%2BA.%26aulast%3DMathiowetz%26aufirst%3DA.%2BM.%26aulast%3DLiras%26aufirst%3DS.%26aulast%3DJacobson%26aufirst%3DM.%2BP.%26aulast%3DLokey%26aufirst%3DR.%2BS.%26atitle%3DProbing%2520the%2520Physicochemical%2520Boundaries%2520of%2520Cell%2520Permeability%2520and%2520Oral%2520Bioavailability%2520in%2520Lipophilic%2520Macrocycles%2520Inspired%2520by%2520Natural%2520Products%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D4581%26epage%3D4589%26doi%3D10.1021%2Facs.jmedchem.5b00128" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schwochert, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turner, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berkeley, R. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ponkey, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rodriguez, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leung, S. S. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khunte, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goetz, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Limberakis, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kalgutkar, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eng, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shapiro, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mathiowetz, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Price, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liras, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacobson, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lokey, R. S.</span></span> <span> </span><span class="NLM_article-title">Peptide to Peptoid Substitutions Increase Cell Permeability in Cyclic Hexapeptides</span>. <i>Org. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">2928</span>– <span class="NLM_lpage">2931</span>, <span class="refDoi"> DOI: 10.1021/acs.orglett.5b01162</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.orglett.5b01162" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC2MXps1arsrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2015&pages=2928-2931&author=J.+Schwochertauthor=R.+Turnerauthor=M.+Thangauthor=R.+F.+Berkeleyauthor=A.+R.+Ponkeyauthor=K.+M.+Rodriguezauthor=S.+S.+F.+Leungauthor=B.+Khunteauthor=G.+Goetzauthor=C.+Limberakisauthor=A.+S.+Kalgutkarauthor=H.+Engauthor=M.+J.+Shapiroauthor=A.+M.+Mathiowetzauthor=D.+A.+Priceauthor=S.+Lirasauthor=M.+P.+Jacobsonauthor=R.+S.+Lokey&title=Peptide+to+Peptoid+Substitutions+Increase+Cell+Permeability+in+Cyclic+Hexapeptides&doi=10.1021%2Facs.orglett.5b01162"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Peptide to Peptoid Substitutions Increase Cell Permeability in Cyclic Hexapeptides</span></div><div class="casAuthors">Schwochert, Joshua; Turner, Rushia; Thang, Melissa; Berkeley, Ray F.; Ponkey, Alexandra R.; Rodriguez, Kelsie M.; Leung, Siegfried S. F.; Khunte, Bhagyashree; Goetz, Gilles; Limberakis, Chris; Kalgutkar, Amit S.; Eng, Heather; Shapiro, Michael J.; Mathiowetz, Alan M.; Price, David A.; Liras, Spiros; Jacobson, Matthew P.; Lokey, R. Scott</div><div class="citationInfo"><span class="NLM_cas:title">Organic Letters</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2928-2931</span>CODEN:
                <span class="NLM_cas:coden">ORLEF7</span>;
        ISSN:<span class="NLM_cas:issn">1523-7052</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The effect of peptide-to-peptoid substitutions on the passive membrane permeability of an N-methylated cyclic hexapeptide is examd.  In general, substitutions maintained permeability but increased conformational heterogeneity.  Diversification with nonproteinogenic side chains increased permeability up to 3-fold.  Addnl., the conformational impact of peptoid substitutions within a β-turn are explored.  Based on these results, the strategic incorporation of peptoid residues into cyclic peptides can maintain or improve cell permeability, while increasing access to diverse side-chain functionality.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGofUG1_hYdVn7Vg90H21EOLACvtfcHk0ljsKw5vh2V6kQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXps1arsrs%253D&md5=156985175ddcafd34a67e81713934040</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1021%2Facs.orglett.5b01162&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.orglett.5b01162%26sid%3Dliteratum%253Aachs%26aulast%3DSchwochert%26aufirst%3DJ.%26aulast%3DTurner%26aufirst%3DR.%26aulast%3DThang%26aufirst%3DM.%26aulast%3DBerkeley%26aufirst%3DR.%2BF.%26aulast%3DPonkey%26aufirst%3DA.%2BR.%26aulast%3DRodriguez%26aufirst%3DK.%2BM.%26aulast%3DLeung%26aufirst%3DS.%2BS.%2BF.%26aulast%3DKhunte%26aufirst%3DB.%26aulast%3DGoetz%26aufirst%3DG.%26aulast%3DLimberakis%26aufirst%3DC.%26aulast%3DKalgutkar%26aufirst%3DA.%2BS.%26aulast%3DEng%26aufirst%3DH.%26aulast%3DShapiro%26aufirst%3DM.%2BJ.%26aulast%3DMathiowetz%26aufirst%3DA.%2BM.%26aulast%3DPrice%26aufirst%3DD.%2BA.%26aulast%3DLiras%26aufirst%3DS.%26aulast%3DJacobson%26aufirst%3DM.%2BP.%26aulast%3DLokey%26aufirst%3DR.%2BS.%26atitle%3DPeptide%2520to%2520Peptoid%2520Substitutions%2520Increase%2520Cell%2520Permeability%2520in%2520Cyclic%2520Hexapeptides%26jtitle%3DOrg.%2520Lett.%26date%3D2015%26volume%3D17%26spage%3D2928%26epage%3D2931%26doi%3D10.1021%2Facs.orglett.5b01162" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Weinmüller, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rechenmacher, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kiran Marelli, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reichart, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kapp, T. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Räder, A. F. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Di Leva, F. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marinelli, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Novellino, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muñoz-Félix, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hodivala-Dilke, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schumacher, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fanous, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilon, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoffman, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kessler, H.</span></span> <span> </span><span class="NLM_article-title">Overcoming the Lack of Oral Availability of Cyclic Hexapeptides: Design of a Selective and Orally Available Ligand for the Integrin αvβ3</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">16405</span>– <span class="NLM_lpage">16409</span>, <span class="refDoi"> DOI: 10.1002/anie.201709709</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1002%2Fanie.201709709" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvVKqtLjF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2017&pages=16405-16409&author=M.+Weinm%C3%BCllerauthor=F.+Rechenmacherauthor=U.+Kiran+Marelliauthor=F.+Reichartauthor=T.+G.+Kappauthor=A.+F.+B.+R%C3%A4derauthor=F.+S.+Di+Levaauthor=L.+Marinelliauthor=E.+Novellinoauthor=J.+M.+Mu%C3%B1oz-F%C3%A9lixauthor=K.+Hodivala-Dilkeauthor=A.+Schumacherauthor=J.+Fanousauthor=C.+Gilonauthor=A.+Hoffmanauthor=H.+Kessler&title=Overcoming+the+Lack+of+Oral+Availability+of+Cyclic+Hexapeptides%3A+Design+of+a+Selective+and+Orally+Available+Ligand+for+the+Integrin+%CE%B1v%CE%B23&doi=10.1002%2Fanie.201709709"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Overcoming the Lack of Oral Availability of Cyclic Hexapeptides: Design of a Selective and Orally Available Ligand for the Integrin αvβ3</span></div><div class="casAuthors">Weinmueller, Michael; Rechenmacher, Florian; Kiran Marelli, Udaya; Reichart, Florian; Kapp, Tobias G.; Raeder, Andreas F. B.; Di Leva, Francesco Saverio; Marinelli, Luciana; Novellino, Ettore; Munoz-Felix, Jose M.; Hodivala-Dilke, Kairbaan; Schumacher, Adi; Fanous, Joseph; Gilon, Chaim; Hoffman, Amnon; Kessler, Horst</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">51</span>),
    <span class="NLM_cas:pages">16405-16409</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A highly systematic approach for the development of both orally bioavailable and bioactive cyclic N-methylated hexapeptides as high affinity ligands for the integrin αvβ3 is based on two concepts: (a) screening of systematically designed libraries with spatial diversity and (b) masking of the peptide charge with a lipophilic protecting group.  The key steps of the method are (1) initial design of a combinatorial library of N-methylated analogs of the stem peptide cyclo(D-Ala-Ala5); (2) selection of cyclic peptides with the highest intestinal permeability; (3) design of sublibraries with the bioactive RGD sequence in all possible positions; (4) selection of the best ligands for RGD-recognizing integrin subtypes; (5) fine-tuning of the affinity and selectivity by addnl. Ala to Xaa substitutions; (6) protection of the charged functional groups according to the prodrug concept to regain intestinal and oral permeability; (7) proof of biol. effects in mice after oral administration.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpUvvbSmX8C4rVg90H21EOLACvtfcHk0ljsKw5vh2V6kQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvVKqtLjF&md5=acd048e94a0525991c7019a2e334b60b</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1002%2Fanie.201709709&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201709709%26sid%3Dliteratum%253Aachs%26aulast%3DWeinm%25C3%25BCller%26aufirst%3DM.%26aulast%3DRechenmacher%26aufirst%3DF.%26aulast%3DKiran%2BMarelli%26aufirst%3DU.%26aulast%3DReichart%26aufirst%3DF.%26aulast%3DKapp%26aufirst%3DT.%2BG.%26aulast%3DR%25C3%25A4der%26aufirst%3DA.%2BF.%2BB.%26aulast%3DDi%2BLeva%26aufirst%3DF.%2BS.%26aulast%3DMarinelli%26aufirst%3DL.%26aulast%3DNovellino%26aufirst%3DE.%26aulast%3DMu%25C3%25B1oz-F%25C3%25A9lix%26aufirst%3DJ.%2BM.%26aulast%3DHodivala-Dilke%26aufirst%3DK.%26aulast%3DSchumacher%26aufirst%3DA.%26aulast%3DFanous%26aufirst%3DJ.%26aulast%3DGilon%26aufirst%3DC.%26aulast%3DHoffman%26aufirst%3DA.%26aulast%3DKessler%26aufirst%3DH.%26atitle%3DOvercoming%2520the%2520Lack%2520of%2520Oral%2520Availability%2520of%2520Cyclic%2520Hexapeptides%253A%2520Design%2520of%2520a%2520Selective%2520and%2520Orally%2520Available%2520Ligand%2520for%2520the%2520Integrin%2520%25CE%25B1v%25CE%25B23%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2017%26volume%3D56%26spage%3D16405%26epage%3D16409%26doi%3D10.1002%2Fanie.201709709" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Di, L.</span></span> <span> </span><span class="NLM_article-title">Strategic Approaches to Optimizing Peptide ADME Properties</span>. <i>AAPS J.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">134</span>– <span class="NLM_lpage">143</span>, <span class="refDoi"> DOI: 10.1208/s12248-014-9687-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1208%2Fs12248-014-9687-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=25366889" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvFSnurvJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2015&pages=134-143&author=L.+Di&title=Strategic+Approaches+to+Optimizing+Peptide+ADME+Properties&doi=10.1208%2Fs12248-014-9687-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Strategic Approaches to Optimizing Peptide ADME Properties</span></div><div class="casAuthors">Di, Li</div><div class="citationInfo"><span class="NLM_cas:title">AAPS Journal</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">134-143</span>CODEN:
                <span class="NLM_cas:coden">AJAOB6</span>;
        ISSN:<span class="NLM_cas:issn">1550-7416</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Development of peptide drugs is challenging but also quite rewarding.  Five blockbuster peptide drugs are currently on the market, and six new peptides received first marketing approval as new mol. entities in 2012.  Although peptides only represent 2% of the drug market, the market is growing twice as quickly and might soon occupy a larger niche.  Natural peptides typically have poor absorption, distribution, metab., and excretion (ADME) properties with rapid clearance, short half-life, low permeability, and sometimes low soly.  Strategies have been developed to improve peptide drugability through enhancing permeability, reducing proteolysis and renal clearance, and prolonging half-life.  In vivo, in vitro, and in silico tools are available to evaluate ADME properties of peptides, and structural modification strategies are in place to improve peptide developability.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpDLDsR-Zk0OLVg90H21EOLACvtfcHk0ljsKw5vh2V6kQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvFSnurvJ&md5=530597d3719d65509105096f433fee0a</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1208%2Fs12248-014-9687-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1208%252Fs12248-014-9687-3%26sid%3Dliteratum%253Aachs%26aulast%3DDi%26aufirst%3DL.%26atitle%3DStrategic%2520Approaches%2520to%2520Optimizing%2520Peptide%2520ADME%2520Properties%26jtitle%3DAAPS%2520J.%26date%3D2015%26volume%3D17%26spage%3D134%26epage%3D143%26doi%3D10.1208%2Fs12248-014-9687-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">van
Witteloostuijn, S. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pedersen, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jensen, K. J.</span></span> <span> </span><span class="NLM_article-title">Half-Life Extension of Biopharmaceuticals Using Chemical Methods: Alternatives to PEGylation</span>. <i>ChemMedChem</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">2474</span>– <span class="NLM_lpage">2495</span>, <span class="refDoi"> DOI: 10.1002/cmdc.201600374</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1002%2Fcmdc.201600374" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=27775236" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC28XhslSqtrbJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2016&pages=2474-2495&author=S.+B.+van%0AWitteloostuijnauthor=S.+L.+Pedersenauthor=K.+J.+Jensen&title=Half-Life+Extension+of+Biopharmaceuticals+Using+Chemical+Methods%3A+Alternatives+to+PEGylation&doi=10.1002%2Fcmdc.201600374"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Half-Life Extension of Biopharmaceuticals using Chemical Methods: Alternatives to PEGylation</span></div><div class="casAuthors">van Witteloostuijn, Soren B.; Pedersen, Soren L.; Jensen, Knud J.</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">2474-2495</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A review.  Peptides and proteins constitute a vast pool of excellent drug candidates.  Evolution has equipped these mols. with superior drug-like properties such as high specificity and potency.  However, native peptides and proteins suffer from an inadequate pharmacokinetic profile, and their outstanding pharmacol. potential can only be realized if this issue is addressed during drug development.  To overcome this challenge, a variety of half-life extension techniques relying on covalent chem. modification have been developed.  These methods include PEGylation, fusion to unstructured polypeptide-based PEG mimetics, conjugation of large polysaccharides, native-like glycosylation, lipidation, fusion to albumin or the Fc domain of IgG, and derivatization with bio-orthogonal moieties that direct self-assembly.  This review provides an overview of available conjugation chemistries, biophys. properties, and safety data assocd. with these concepts.  Moreover, the effects of these modifications on peptide and protein pharmacokinetics are demonstrated through key examples.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGodYDhxsEiYZrVg90H21EOLACvtfcHk0li0hrlMLQiw0Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhslSqtrbJ&md5=b3811dad7e9a100fa235d8eebf7272c6</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.201600374&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.201600374%26sid%3Dliteratum%253Aachs%26aulast%3Dvan%2BWitteloostuijn%26aufirst%3DS.%2BB.%26aulast%3DPedersen%26aufirst%3DS.%2BL.%26aulast%3DJensen%26aufirst%3DK.%2BJ.%26atitle%3DHalf-Life%2520Extension%2520of%2520Biopharmaceuticals%2520Using%2520Chemical%2520Methods%253A%2520Alternatives%2520to%2520PEGylation%26jtitle%3DChemMedChem%26date%3D2016%26volume%3D11%26spage%3D2474%26epage%3D2495%26doi%3D10.1002%2Fcmdc.201600374" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">King, G. F.</span></span> <span> </span><span class="NLM_article-title">Venoms as a Platform for Human Drugs: Translating Toxins into Therapeutics</span>. <i>Expert Opin. Biol. Ther.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">1469</span>– <span class="NLM_lpage">1484</span>, <span class="refDoi"> DOI: 10.1517/14712598.2011.621940</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1517%2F14712598.2011.621940" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=21939428" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC3MXht1ylsrjF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2011&pages=1469-1484&author=G.+F.+King&title=Venoms+as+a+Platform+for+Human+Drugs%3A+Translating+Toxins+into+Therapeutics&doi=10.1517%2F14712598.2011.621940"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Venoms as a platform for human drugs: translating toxins into therapeutics</span></div><div class="casAuthors">King, Glenn F.</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Biological Therapy</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1469-1484</span>CODEN:
                <span class="NLM_cas:coden">EOBTA2</span>;
        ISSN:<span class="NLM_cas:issn">1471-2598</span>.
    
            (<span class="NLM_cas:orgname">Informa Healthcare</span>)
        </div><div class="casAbstract">A review.  Introduction:An extraordinarily diverse range of animals have evolved venoms for predation, defense, or competitor deterrence.  The major components of most venoms are peptides and proteins that are often protease-resistant due to their disulfide-rich architectures.  Some of these toxins have become valuable as pharmacol. tools and/or therapeutics due to their extremely high specificity and potency for particular mol. targets.  There are currently six FDA-approved drugs derived from venom peptides or proteins.  Areas covered:This article surveys the current pipeline of venom-derived therapeutics and critically examines the potential of peptide and protein drugs derived from venoms.  Emerging trends are identified, including an increasing industry focus on disulfide-rich venom peptides and the use of a broader array of mol. targets in order to develop venom-based therapeutics for treating a wider range of clin. conditions.  Expert opinion:Key tech. advances in combination with a renewed industry-wide focus on biologics have converged to provide a larger than ever pipeline of venom-derived therapeutics.  Disulfide-rich venom peptides obviate some of the traditional disadvantages of therapeutic peptides and some may be suitable for oral administration.  Moreover, some venom peptides can breach the blood brain barrier and translocate across cell membranes, which opens up the possibility of exploiting mol. targets not previously accessible to peptide drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpoYClUIGUrZbVg90H21EOLACvtfcHk0li0hrlMLQiw0Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXht1ylsrjF&md5=4983d10289108fac346231191e6169f7</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1517%2F14712598.2011.621940&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1517%252F14712598.2011.621940%26sid%3Dliteratum%253Aachs%26aulast%3DKing%26aufirst%3DG.%2BF.%26atitle%3DVenoms%2520as%2520a%2520Platform%2520for%2520Human%2520Drugs%253A%2520Translating%2520Toxins%2520into%2520Therapeutics%26jtitle%3DExpert%2520Opin.%2520Biol.%2520Ther.%26date%3D2011%26volume%3D11%26spage%3D1469%26epage%3D1484%26doi%3D10.1517%2F14712598.2011.621940" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pennington, M. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Czerwinski, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Norton, R. S.</span></span> <span> </span><span class="NLM_article-title">Peptide Therapeutics from Venom: Current Status and Potential</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">2738</span>– <span class="NLM_lpage">2758</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2017.09.029</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1016%2Fj.bmc.2017.09.029" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=28988749" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhs1aitrzI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2018&pages=2738-2758&author=M.+W.+Penningtonauthor=A.+Czerwinskiauthor=R.+S.+Norton&title=Peptide+Therapeutics+from+Venom%3A+Current+Status+and+Potential&doi=10.1016%2Fj.bmc.2017.09.029"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Peptide therapeutics from venom: Current status and potential</span></div><div class="casAuthors">Pennington, Michael W.; Czerwinski, Andrzej; Norton, Raymond S.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2738-2758</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Peptides are recognized as being highly selective, potent and relatively safe as potential therapeutics.  Peptides isolated from the venom of different animals satisfy most of these criteria with the possible exception of safety, but when isolated as single compds. and used at appropriate concns., venom-derived peptides can become useful drugs.  Although the no. of venom-derived peptides that have successfully progressed to the clinic is currently limited, the prospects for venom-derived peptides look very optimistic.  As proteomic and transcriptomic approaches continue to identify new sequences, the potential of venom-derived peptides to find applications as therapeutics, cosmetics and insecticides grows accordingly.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp8haCt_E4Tu7Vg90H21EOLACvtfcHk0li0hrlMLQiw0Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhs1aitrzI&md5=253b75540d1a2117338259c087b119f7</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2017.09.029&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2017.09.029%26sid%3Dliteratum%253Aachs%26aulast%3DPennington%26aufirst%3DM.%2BW.%26aulast%3DCzerwinski%26aufirst%3DA.%26aulast%3DNorton%26aufirst%3DR.%2BS.%26atitle%3DPeptide%2520Therapeutics%2520from%2520Venom%253A%2520Current%2520Status%2520and%2520Potential%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2018%26volume%3D26%26spage%3D2738%26epage%3D2758%26doi%3D10.1016%2Fj.bmc.2017.09.029" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Campolina
Rebello Horta, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chatzaki, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Almeida Rezende, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Freitas Magalhães, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guerra Duarte, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Figueiredo Felicori, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ribeiro Oliveira-Mendes, B. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oliveira
do Carmo, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chávez-Olórtegui, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kalapothakis, E.</span></span> <span> </span><span class="NLM_article-title">Cardiovascular-Active Venom Toxins: An Overview</span>. <i>Curr. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">603</span>– <span class="NLM_lpage">622</span>, <span class="refDoi"> DOI: 10.2174/0929867323666160126142837</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.2174%2F0929867323666160126142837" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=26812904" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2016&pages=603-622&author=C.+Campolina%0ARebello+Hortaauthor=M.+Chatzakiauthor=B.+Almeida+Rezendeauthor=B.+de+Freitas+Magalh%C3%A3esauthor=C.+Guerra+Duarteauthor=L.+Figueiredo+Felicoriauthor=B.+B.+Ribeiro+Oliveira-Mendesauthor=A.+Oliveira%0Ado+Carmoauthor=C.+Ch%C3%A1vez-Ol%C3%B3rteguiauthor=E.+Kalapothakis&title=Cardiovascular-Active+Venom+Toxins%3A+An+Overview&doi=10.2174%2F0929867323666160126142837"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.2174%2F0929867323666160126142837&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F0929867323666160126142837%26sid%3Dliteratum%253Aachs%26aulast%3DCampolina%2BRebello%2BHorta%26aufirst%3DC.%26aulast%3DChatzaki%26aufirst%3DM.%26aulast%3DAlmeida%2BRezende%26aufirst%3DB.%26aulast%3Dde%2BFreitas%2BMagalh%25C3%25A3es%26aufirst%3DB.%26aulast%3DGuerra%2BDuarte%26aufirst%3DC.%26aulast%3DFigueiredo%2BFelicori%26aufirst%3DL.%26aulast%3DRibeiro%2BOliveira-Mendes%26aufirst%3DB.%2BB.%26aulast%3DOliveira%2Bdo%2BCarmo%26aufirst%3DA.%26aulast%3DCh%25C3%25A1vez-Ol%25C3%25B3rtegui%26aufirst%3DC.%26aulast%3DKalapothakis%26aufirst%3DE.%26atitle%3DCardiovascular-Active%2520Venom%2520Toxins%253A%2520An%2520Overview%26jtitle%3DCurr.%2520Med.%2520Chem.%26date%3D2016%26volume%3D23%26spage%3D603%26epage%3D622%26doi%3D10.2174%2F0929867323666160126142837" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref54"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref54'); return false;" data-citation="" class="refNumLink">54</a></strong><div class="NLM_citation" id="cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lewis, R. J.</span></span> <span> </span><span class="NLM_article-title">Case Study 1: Development of the Analgesic Drugs Prialt® and Xen2174 from Cone Snail Venoms</span>. In  <i>Venoms to Drugs</i>; <span class="NLM_contrib-group"><span class="NLM_string-name">King, G. F.</span></span>, Ed.; <span class="NLM_publisher-name">The Royal Society of Chemistry</span>: <span class="NLM_publisher-loc">Cambridge, U.K.</span>, <span class="NLM_year">2015</span>; pp  <span class="NLM_fpage">245</span>– <span class="NLM_lpage">254</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1039%2F9781849737876-00245" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2015&pages=245-254&author=R.+J.+Lewisauthor=G.+F.+King&title=Venoms+to+Drugs"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1039%2F9781849737876-00245&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1039%252F9781849737876-00245%26sid%3Dliteratum%253Aachs%26aulast%3DLewis%26aufirst%3DR.%2BJ.%26atitle%3DCase%2520Study%25201%253A%2520Development%2520of%2520the%2520Analgesic%2520Drugs%2520Prialt%25C2%25AE%2520and%2520Xen2174%2520from%2520Cone%2520Snail%2520Venoms%26btitle%3DVenoms%2520to%2520Drugs%26aulast%3DKing%26aufirst%3DG.%2BF.%26pub%3DThe%2520Royal%2520Society%2520of%2520Chemistry%26date%3D2015%26spage%3D245%26epage%3D254" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref55"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref55'); return false;" data-citation="" class="refNumLink">55</a></strong><div class="NLM_citation" id="cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sanggaard, K. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dyrlund, T. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomsen, L. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nielsen, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brøndum, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thøgersen, I. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Enghild, J. J.</span></span> <span> </span><span class="NLM_article-title">Characterization of the Gila Monster (Heloderma Suspectum Suspectum) Venom Proteome</span>. <i>J. Proteomics</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>117</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">11</span>, <span class="refDoi"> DOI: 10.1016/j.jprot.2015.01.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1016%2Fj.jprot.2015.01.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=25603280" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtlWgu7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=117&publication_year=2015&pages=1-11&author=K.+W.+Sanggaardauthor=T.+F.+Dyrlundauthor=L.+R.+Thomsenauthor=T.+A.+Nielsenauthor=L.+Br%C3%B8ndumauthor=T.+Wangauthor=I.+B.+Th%C3%B8gersenauthor=J.+J.+Enghild&title=Characterization+of+the+Gila+Monster+%28Heloderma+Suspectum+Suspectum%29+Venom+Proteome&doi=10.1016%2Fj.jprot.2015.01.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Characterization of the gila monster (Heloderma suspectum suspectum) venom proteome</span></div><div class="casAuthors">Sanggaard, Kristian W.; Dyrlund, Thomas F.; Thomsen, Line R.; Nielsen, Tania A.; Broendum, Lars; Wang, Tobias; Thoegersen, Ida B.; Enghild, Jan J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Proteomics</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">117</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">1-11</span>CODEN:
                <span class="NLM_cas:coden">JPORFQ</span>;
        ISSN:<span class="NLM_cas:issn">1874-3919</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The archetypical venomous lizard species are the helodermatids, the gila monsters (Heloderma suspectum) and the beaded lizards (Heloderma horridum).  In the present study, the gila monster venom proteome was characterized using 2D-gel electrophoresis and tandem mass spectrometry-based de novo peptide sequencing followed by protein identification based on sequence homol.  A total of 39 different proteins were identified out of the 58 selected spots that represent the major constituents of venom.  Of these proteins, 19 have not previously been identified in helodermatid venom.  The data showed that helodermatid venom is complex and that this complexity is caused by genetic isoforms and post-translational modifications including proteolytic processing.  In addn., the venom proteome anal. revealed that the major constituents of the gila monster venom are kallikrein-like serine proteinases (EC 3.4.21) and phospholipase A2 (type III) enzymes (EC 3.1.1.4).  A neuroendocrine convertase 1 homolog that most likely converts the proforms of the previously identified bioactive exendins into the mature and active forms was identified suggesting that these peptide toxins are secreted as proforms that are activated by proteolytic cleavage following secretion as opposed to being activated intracellularly.  The presented global protein identification-anal. provides the first overview of the helodermatid venom compn.The helodermatid lizards are the classical venomous lizards, and the pharmacol. potential of the venom from these species has been known for years; best illustrated by the identification of exendin-4, which is now used in the treatment of type 2 diabetes.  Despite the potential, no global analyses of the protein components in the venom exist.  A hindrance is the lack of a genome sequence because it prevents protein identification using a conventional approach where MS data are searched against predicted protein sequences based on the genome sequence.  However, in the recent years the development of software tools for de novo sequencing and homol. searches have improved significantly facilitating the first global anal. of the major protein components of helodermatid venom presented in this study.  We have used a 2D-gel approach and detd. the protein components in the 58 major spots resulting in the identification of 39 unique proteins.  Of these, 19 have not previously been identified in helodermatid venom.  The anal. provides results with impact on our understanding of the function and evolution of venom proteins, and serves as a basis for further unraveling of the pharmaceutical potential of the venom components.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq0jT-qNadT-rVg90H21EOLACvtfcHk0ljX_qKH8BCQMQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtlWgu7g%253D&md5=7b20f196b8301fa4d769001e876ea332</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1016%2Fj.jprot.2015.01.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jprot.2015.01.004%26sid%3Dliteratum%253Aachs%26aulast%3DSanggaard%26aufirst%3DK.%2BW.%26aulast%3DDyrlund%26aufirst%3DT.%2BF.%26aulast%3DThomsen%26aufirst%3DL.%2BR.%26aulast%3DNielsen%26aufirst%3DT.%2BA.%26aulast%3DBr%25C3%25B8ndum%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DT.%26aulast%3DTh%25C3%25B8gersen%26aufirst%3DI.%2BB.%26aulast%3DEnghild%26aufirst%3DJ.%2BJ.%26atitle%3DCharacterization%2520of%2520the%2520Gila%2520Monster%2520%2528Heloderma%2520Suspectum%2520Suspectum%2529%2520Venom%2520Proteome%26jtitle%3DJ.%2520Proteomics%26date%3D2015%26volume%3D117%26spage%3D1%26epage%3D11%26doi%3D10.1016%2Fj.jprot.2015.01.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref56"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref56'); return false;" data-citation="" class="refNumLink">56</a></strong><div class="NLM_citation" id="cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Akondi, K. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muttenthaler, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dutertre, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaas, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Craik, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lewis, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alewood, P. F.</span></span> <span> </span><span class="NLM_article-title">Discovery, Synthesis, and Structure-Activity Relationships of Conotoxins</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>114</i></span>,  <span class="NLM_fpage">5815</span>– <span class="NLM_lpage">5847</span>, <span class="refDoi"> DOI: 10.1021/cr400401e</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cr400401e" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC2cXmtVegt78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=114&publication_year=2014&pages=5815-5847&author=K.+B.+Akondiauthor=M.+Muttenthalerauthor=S.+Dutertreauthor=Q.+Kaasauthor=D.+J.+Craikauthor=R.+J.+Lewisauthor=P.+F.+Alewood&title=Discovery%2C+Synthesis%2C+and+Structure-Activity+Relationships+of+Conotoxins&doi=10.1021%2Fcr400401e"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery, Synthesis, and Structure-Activity Relationships of Conotoxins</span></div><div class="casAuthors">Akondi, Kalyana B.; Muttenthaler, Markus; Dutertre, Sebastien; Kaas, Quentin; Craik, David J.; Lewis, Richard J.; Alewood, Paul F.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">114</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">5815-5847</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Venom peptides, esp. conotoxins isolated from the venom of predatory marine snails, hold enormous promise for the development of peptide-based drugs.  Conotoxin classification, nomenclature, synthesis, structure activity relationships, and as candidates for drugs are reviewed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqBcE_sYX4b2LVg90H21EOLACvtfcHk0ljX_qKH8BCQMQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXmtVegt78%253D&md5=02f1253f5605757259436d2ecefd42a6</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1021%2Fcr400401e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcr400401e%26sid%3Dliteratum%253Aachs%26aulast%3DAkondi%26aufirst%3DK.%2BB.%26aulast%3DMuttenthaler%26aufirst%3DM.%26aulast%3DDutertre%26aufirst%3DS.%26aulast%3DKaas%26aufirst%3DQ.%26aulast%3DCraik%26aufirst%3DD.%2BJ.%26aulast%3DLewis%26aufirst%3DR.%2BJ.%26aulast%3DAlewood%26aufirst%3DP.%2BF.%26atitle%3DDiscovery%252C%2520Synthesis%252C%2520and%2520Structure-Activity%2520Relationships%2520of%2520Conotoxins%26jtitle%3DChem.%2520Rev.%26date%3D2014%26volume%3D114%26spage%3D5815%26epage%3D5847%26doi%3D10.1021%2Fcr400401e" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref57"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref57'); return false;" data-citation="" class="refNumLink">57</a></strong><div class="NLM_citation" id="cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ciolek, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reinfrank, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quinton, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Viengchareun, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stura, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vera, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sigismeau, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mouillac, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Orcel, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peigneur, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tytgat, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Droctové, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beau, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nevoux, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lombès, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mourier, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Pauw, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Servent, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mendre, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Witzgall, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilles, N.</span></span> <span> </span><span class="NLM_article-title">Green Mamba Peptide Targets Type-2 Vasopressin Receptor against Polycystic Kidney Disease</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>114</i></span>,  <span class="NLM_fpage">7154</span>– <span class="NLM_lpage">7159</span>, <span class="refDoi"> DOI: 10.1073/pnas.1620454114</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1073%2Fpnas.1620454114" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=28630289" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtVWmtL3N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=114&publication_year=2017&pages=7154-7159&author=J.+Ciolekauthor=H.+Reinfrankauthor=L.+Quintonauthor=S.+Viengchareunauthor=E.+A.+Sturaauthor=L.+Veraauthor=S.+Sigismeauauthor=B.+Mouillacauthor=H.+Orcelauthor=S.+Peigneurauthor=J.+Tytgatauthor=L.+Droctov%C3%A9author=F.+Beauauthor=J.+Nevouxauthor=M.+Lomb%C3%A8sauthor=G.+Mourierauthor=E.+De+Pauwauthor=D.+Serventauthor=C.+Mendreauthor=R.+Witzgallauthor=N.+Gilles&title=Green+Mamba+Peptide+Targets+Type-2+Vasopressin+Receptor+against+Polycystic+Kidney+Disease&doi=10.1073%2Fpnas.1620454114"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">Green mamba peptide targets type-2 vasopressin receptor against polycystic kidney disease</span></div><div class="casAuthors">Ciolek, Justyna; Reinfrank, Helen; Quinton, Loic; Viengchareun, Say; Stura, Enrico A.; Vera, Laura; Sigismeau, Sabrina; Mouillac, Bernard; Orcel, Helene; Peigneur, Steve; Tytgat, Jan; Droctove, Laura; Beau, Fabrice; Nevoux, Jerome; Lombes, Marc; Mourier, Gilles; De Pauw, Edwin; Servent, Denis; Mendre, Christiane; Witzgall, Ralph; Gilles, Nicolas</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">114</span>
        (<span class="NLM_cas:issue">27</span>),
    <span class="NLM_cas:pages">7154-7159</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Polycystic kidney diseases (PKDs) are genetic disorders that can cause renal failure and death in children and adults.  Lowering cAMP in cystic tissues through the inhibition of the type-2 vasopressin receptor (V2R) constitutes a validated strategy to reduce disease progression.  The authors identified a peptide from green mamba venom that exhibits nanomolar affinity for the V2R without any activity on 155 other G-protein-coupled receptors or on 15 ionic channels.  Mambaquaretin-1 is a full antagonist of the V2R activation pathways studied: cAMP prodn., beta-arrestin interaction, and MAP kinase activity.  This peptide adopts the Kunitz fold known to mostly act on potassium channels and serine proteases.  Mambaquaretin-1 interacts selectively with the V2R through its first loop, in the same manner that aprotinin inhibits trypsin.  Injected in mice, mambaquaretin-1 increases in a dose-dependent manner urine outflow with concomitant redn. of urine osmolality, indicating a purely aquaretic effect assocd. with the in vivo blockade of V2R.  CD1-pcy/pcy mice, a juvenile model of PKD, daily treated with 13 μg of mambaquaretin-1 for 99 d, developed less abundant (by 33%) and smaller (by 47%) cysts than control mice.  Neither tachyphylaxis nor apparent toxicity has been noted.  Mambaquaretin-1 represents a promising therapeutic agent against PKDs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqI-J7A636tYrVg90H21EOLACvtfcHk0ljX_qKH8BCQMQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtVWmtL3N&md5=2cad6950c107df021baf5a66c6b086b0</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1620454114&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1620454114%26sid%3Dliteratum%253Aachs%26aulast%3DCiolek%26aufirst%3DJ.%26aulast%3DReinfrank%26aufirst%3DH.%26aulast%3DQuinton%26aufirst%3DL.%26aulast%3DViengchareun%26aufirst%3DS.%26aulast%3DStura%26aufirst%3DE.%2BA.%26aulast%3DVera%26aufirst%3DL.%26aulast%3DSigismeau%26aufirst%3DS.%26aulast%3DMouillac%26aufirst%3DB.%26aulast%3DOrcel%26aufirst%3DH.%26aulast%3DPeigneur%26aufirst%3DS.%26aulast%3DTytgat%26aufirst%3DJ.%26aulast%3DDroctov%25C3%25A9%26aufirst%3DL.%26aulast%3DBeau%26aufirst%3DF.%26aulast%3DNevoux%26aufirst%3DJ.%26aulast%3DLomb%25C3%25A8s%26aufirst%3DM.%26aulast%3DMourier%26aufirst%3DG.%26aulast%3DDe%2BPauw%26aufirst%3DE.%26aulast%3DServent%26aufirst%3DD.%26aulast%3DMendre%26aufirst%3DC.%26aulast%3DWitzgall%26aufirst%3DR.%26aulast%3DGilles%26aufirst%3DN.%26atitle%3DGreen%2520Mamba%2520Peptide%2520Targets%2520Type-2%2520Vasopressin%2520Receptor%2520against%2520Polycystic%2520Kidney%2520Disease%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2017%26volume%3D114%26spage%3D7154%26epage%3D7159%26doi%3D10.1073%2Fpnas.1620454114" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref58"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref58'); return false;" data-citation="" class="refNumLink">58</a></strong><div class="NLM_citation" id="cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Diochot, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baron, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salinas, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Douguet, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scarzello, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dabert-Gay, A.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Debayle, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friend, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alloui, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lazdunski, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lingueglia, E.</span></span> <span> </span><span class="NLM_article-title">Black Mamba Venom Peptides Target Acid-Sensing Ion Channels to Abolish Pain</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>490</i></span>,  <span class="NLM_fpage">552</span>– <span class="NLM_lpage">555</span>, <span class="refDoi"> DOI: 10.1038/nature11494</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1038%2Fnature11494" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=23034652" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsVGjsrnL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=490&publication_year=2012&pages=552-555&author=S.+Diochotauthor=A.+Baronauthor=M.+Salinasauthor=D.+Douguetauthor=S.+Scarzelloauthor=A.-S.+Dabert-Gayauthor=D.+Debayleauthor=V.+Friendauthor=A.+Allouiauthor=M.+Lazdunskiauthor=E.+Lingueglia&title=Black+Mamba+Venom+Peptides+Target+Acid-Sensing+Ion+Channels+to+Abolish+Pain&doi=10.1038%2Fnature11494"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">Black mamba venom peptides target acid-sensing ion channels to abolish pain</span></div><div class="casAuthors">Diochot, Sylvie; Baron, Anne; Salinas, Miguel; Douguet, Dominique; Scarzello, Sabine; Dabert-Gay, Anne-Sophie; Debayle, Delphine; Friend, Valerie; Alloui, Abdelkrim; Lazdunski, Michel; Lingueglia, Eric</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">490</span>
        (<span class="NLM_cas:issue">7421</span>),
    <span class="NLM_cas:pages">552-555</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Polypeptide toxins have played a central part in understanding physiol. and physiopathol. functions of ion channels.  In the field of pain, they led to important advances in basic research and even to clin. applications.  Acid-sensing ion channels (ASICs) are generally considered principal players in the pain pathway, including in humans.  A snake toxin activating peripheral ASICs in nociceptive neurons has been recently shown to evoke pain.  Here we show that a new class of three-finger peptides from another snake, the black mamba, is able to abolish pain through inhibition of ASICs expressed either in central or peripheral neurons.  These peptides, which we call mambalgins, are not toxic in mice but show a potent analgesic effect upon central and peripheral injection that can be as strong as morphine.  This effect is, however, resistant to naloxone, and mambalgins cause much less tolerance than morphine and no respiratory distress.  Pharmacol. inhibition by mambalgins combined with the use of knockdown and knockout animals indicates that blockade of heteromeric channels made of ASIC1a and ASIC2a subunits in central neurons and of ASIC1b-contg. channels in nociceptors is involved in the analgesic effect of mambalgins.  These findings identify new potential therapeutic targets for pain and introduce natural peptides that block them to produce a potent analgesia.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqtE5BL138DF7Vg90H21EOLACvtfcHk0lgs_Trj8jStuA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsVGjsrnL&md5=35cb1982de34af4d45935b3106f17787</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1038%2Fnature11494&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature11494%26sid%3Dliteratum%253Aachs%26aulast%3DDiochot%26aufirst%3DS.%26aulast%3DBaron%26aufirst%3DA.%26aulast%3DSalinas%26aufirst%3DM.%26aulast%3DDouguet%26aufirst%3DD.%26aulast%3DScarzello%26aufirst%3DS.%26aulast%3DDabert-Gay%26aufirst%3DA.-S.%26aulast%3DDebayle%26aufirst%3DD.%26aulast%3DFriend%26aufirst%3DV.%26aulast%3DAlloui%26aufirst%3DA.%26aulast%3DLazdunski%26aufirst%3DM.%26aulast%3DLingueglia%26aufirst%3DE.%26atitle%3DBlack%2520Mamba%2520Venom%2520Peptides%2520Target%2520Acid-Sensing%2520Ion%2520Channels%2520to%2520Abolish%2520Pain%26jtitle%3DNature%26date%3D2012%26volume%3D490%26spage%3D552%26epage%3D555%26doi%3D10.1038%2Fnature11494" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref59"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref59'); return false;" data-citation="" class="refNumLink">59</a></strong><div class="NLM_citation" id="cit59"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Er, S. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cristofori-Armstrong, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Escoubas, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rash, L. D.</span></span> <span> </span><span class="NLM_article-title">Discovery and Molecular Interaction Studies of a Highly Stable, Tarantula Peptide Modulator of Acid-Sensing Ion Channel 1</span>. <i>Neuropharmacology</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>127</i></span>,  <span class="NLM_fpage">185</span>– <span class="NLM_lpage">195</span>, <span class="refDoi"> DOI: 10.1016/j.neuropharm.2017.03.020</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1016%2Fj.neuropharm.2017.03.020" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=127&publication_year=2017&pages=185-195&author=S.+Y.+Erauthor=B.+Cristofori-Armstrongauthor=P.+Escoubasauthor=L.+D.+Rash&title=Discovery+and+Molecular+Interaction+Studies+of+a+Highly+Stable%2C+Tarantula+Peptide+Modulator+of+Acid-Sensing+Ion+Channel+1&doi=10.1016%2Fj.neuropharm.2017.03.020"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1016%2Fj.neuropharm.2017.03.020&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.neuropharm.2017.03.020%26sid%3Dliteratum%253Aachs%26aulast%3DEr%26aufirst%3DS.%2BY.%26aulast%3DCristofori-Armstrong%26aufirst%3DB.%26aulast%3DEscoubas%26aufirst%3DP.%26aulast%3DRash%26aufirst%3DL.%2BD.%26atitle%3DDiscovery%2520and%2520Molecular%2520Interaction%2520Studies%2520of%2520a%2520Highly%2520Stable%252C%2520Tarantula%2520Peptide%2520Modulator%2520of%2520Acid-Sensing%2520Ion%2520Channel%25201%26jtitle%3DNeuropharmacology%26date%3D2017%26volume%3D127%26spage%3D185%26epage%3D195%26doi%3D10.1016%2Fj.neuropharm.2017.03.020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref60"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref60'); return false;" data-citation="" class="refNumLink">60</a></strong><div class="NLM_citation" id="cit60"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chevalier, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Silva, D.-A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rocklin, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hicks, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vergara, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murapa, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernard, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lam, K.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bahl, C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miyashita, S.-I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goreshnik, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fuller, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koday, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jenkins, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Colvin, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carter, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bohn, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bryan, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fernández-Velasco, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stewart, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, I. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fuller, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baker, D.</span></span> <span> </span><span class="NLM_article-title">Massively Parallel <i>de Novo</i> Protein Design for Targeted Therapeutics</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>550</i></span>,  <span class="NLM_fpage">74</span>– <span class="NLM_lpage">79</span>, <span class="refDoi"> DOI: 10.1038/nature23912</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1038%2Fnature23912" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=28953867" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsFKmsL%252FL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=550&publication_year=2017&pages=74-79&author=A.+Chevalierauthor=D.-A.+Silvaauthor=G.+J.+Rocklinauthor=D.+R.+Hicksauthor=R.+Vergaraauthor=P.+Murapaauthor=S.+M.+Bernardauthor=L.+Zhangauthor=K.-H.+Lamauthor=G.+Yaoauthor=C.+D.+Bahlauthor=S.-I.+Miyashitaauthor=I.+Goreshnikauthor=J.+T.+Fullerauthor=M.+T.+Kodayauthor=C.+M.+Jenkinsauthor=T.+Colvinauthor=L.+Carterauthor=A.+Bohnauthor=C.+M.+Bryanauthor=D.+A.+Fern%C3%A1ndez-Velascoauthor=L.+Stewartauthor=M.+Dongauthor=X.+Huangauthor=R.+Jinauthor=I.+A.+Wilsonauthor=D.+H.+Fullerauthor=D.+Baker&title=Massively+Parallel+de+Novo+Protein+Design+for+Targeted+Therapeutics&doi=10.1038%2Fnature23912"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">Massively parallel de novo protein design for targeted therapeutics</span></div><div class="casAuthors">Chevalier, Aaron; Silva, Daniel-Adriano; Rocklin, Gabriel J.; Hicks, Derrick R.; Vergara, Renan; Murapa, Patience; Bernard, Steffen M.; Zhang, Lu; Lam, Kwok-Ho; Yao, Guorui; Bahl, Christopher D.; Miyashita, Shin-ichiro; Goreshnik, Inna; Fuller, James T.; Koday, Merika T.; Jenkins, Cody M.; Colvin, Tom; Carter, Lauren; Bohn, Alan; Bryan, Cassie M.; Fernandez-Velasco, D. Alejandro; Stewart, Lance; Dong, Min; Huang, Xuhui; Jin, Rongsheng; Wilson, Ian A.; Fuller, Deborah H.; Baker, David</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">550</span>
        (<span class="NLM_cas:issue">7674</span>),
    <span class="NLM_cas:pages">74-79</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">De novo protein design holds promise for creating small stable proteins with shapes customized to bind therapeutic targets.  We describe a massively parallel approach for designing, manufg. and screening mini-protein binders, integrating large-scale computational design, oligonucleotide synthesis, yeast display screening and next-generation sequencing.  We designed and tested 22,660 mini-proteins of 37-43 residues that target influenza hemagglutinin and botulinum neurotoxin B, along with 6286 control sequences to probe contributions to folding and binding, and identified 2618 high-affinity binders.  Comparison of the binding and non-binding design sets, which are two orders of magnitude larger than any previously investigated, enabled the evaluation and improvement of the computational model.  Biophys. characterization of a subset of the binder designs showed that they are extremely stable and, unlike antibodies, do not lose activity after exposure to high temps.  The designs elicit little or no immune response and provide potent prophylactic and therapeutic protection against influenza, even after extensive repeated dosing.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq_7ufRgBkWxLVg90H21EOLACvtfcHk0lgs_Trj8jStuA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsFKmsL%252FL&md5=c9f52079bed31a8dd4b09f0c84df0b97</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1038%2Fnature23912&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature23912%26sid%3Dliteratum%253Aachs%26aulast%3DChevalier%26aufirst%3DA.%26aulast%3DSilva%26aufirst%3DD.-A.%26aulast%3DRocklin%26aufirst%3DG.%2BJ.%26aulast%3DHicks%26aufirst%3DD.%2BR.%26aulast%3DVergara%26aufirst%3DR.%26aulast%3DMurapa%26aufirst%3DP.%26aulast%3DBernard%26aufirst%3DS.%2BM.%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DLam%26aufirst%3DK.-H.%26aulast%3DYao%26aufirst%3DG.%26aulast%3DBahl%26aufirst%3DC.%2BD.%26aulast%3DMiyashita%26aufirst%3DS.-I.%26aulast%3DGoreshnik%26aufirst%3DI.%26aulast%3DFuller%26aufirst%3DJ.%2BT.%26aulast%3DKoday%26aufirst%3DM.%2BT.%26aulast%3DJenkins%26aufirst%3DC.%2BM.%26aulast%3DColvin%26aufirst%3DT.%26aulast%3DCarter%26aufirst%3DL.%26aulast%3DBohn%26aufirst%3DA.%26aulast%3DBryan%26aufirst%3DC.%2BM.%26aulast%3DFern%25C3%25A1ndez-Velasco%26aufirst%3DD.%2BA.%26aulast%3DStewart%26aufirst%3DL.%26aulast%3DDong%26aufirst%3DM.%26aulast%3DHuang%26aufirst%3DX.%26aulast%3DJin%26aufirst%3DR.%26aulast%3DWilson%26aufirst%3DI.%2BA.%26aulast%3DFuller%26aufirst%3DD.%2BH.%26aulast%3DBaker%26aufirst%3DD.%26atitle%3DMassively%2520Parallel%2520de%2520Novo%2520Protein%2520Design%2520for%2520Targeted%2520Therapeutics%26jtitle%3DNature%26date%3D2017%26volume%3D550%26spage%3D74%26epage%3D79%26doi%3D10.1038%2Fnature23912" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref61"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref61'); return false;" data-citation="" class="refNumLink">61</a></strong><div class="NLM_citation" id="cit61"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Poillot, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bichraoui, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tisseyre, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bahemberae, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andreotti, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sabatier, J.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ronjat, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Waard, M.</span></span> <span> </span><span class="NLM_article-title">Small Efficient Cell-Penetrating Peptides Derived from Scorpion Toxin Maurocalcine</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>287</i></span>,  <span class="NLM_fpage">17331</span>– <span class="NLM_lpage">17342</span>, <span class="refDoi"> DOI: 10.1074/jbc.M112.360628</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1074%2Fjbc.M112.360628" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=287&publication_year=2012&pages=17331-17342&author=C.+Poillotauthor=H.+Bichraouiauthor=C.+Tisseyreauthor=E.+Bahemberaeauthor=N.+Andreottiauthor=J.-M.+Sabatierauthor=M.+Ronjatauthor=M.+De+Waard&title=Small+Efficient+Cell-Penetrating+Peptides+Derived+from+Scorpion+Toxin+Maurocalcine&doi=10.1074%2Fjbc.M112.360628"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M112.360628&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M112.360628%26sid%3Dliteratum%253Aachs%26aulast%3DPoillot%26aufirst%3DC.%26aulast%3DBichraoui%26aufirst%3DH.%26aulast%3DTisseyre%26aufirst%3DC.%26aulast%3DBahemberae%26aufirst%3DE.%26aulast%3DAndreotti%26aufirst%3DN.%26aulast%3DSabatier%26aufirst%3DJ.-M.%26aulast%3DRonjat%26aufirst%3DM.%26aulast%3DDe%2BWaard%26aufirst%3DM.%26atitle%3DSmall%2520Efficient%2520Cell-Penetrating%2520Peptides%2520Derived%2520from%2520Scorpion%2520Toxin%2520Maurocalcine%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2012%26volume%3D287%26spage%3D17331%26epage%3D17342%26doi%3D10.1074%2Fjbc.M112.360628" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref62"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref62'); return false;" data-citation="" class="refNumLink">62</a></strong><div class="NLM_citation" id="cit62"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Goodnow, R. A.,  Jr.</span></span> <span> </span><span class="NLM_article-title">A Brief History of the Development of Combinatorial Chemistry and the Emerging Need for DNA-Encoded Chemistry</span>. In  <i>A Handbook for DNA-Encoded Chemistry</i>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goodnow, R. A.,  Jr.</span></span>, Ed.; <span class="NLM_publisher-name">John Wiley & Sons, Inc.</span>: <span class="NLM_publisher-loc">New York</span>, <span class="NLM_year">2014</span>; pp  <span class="NLM_fpage">19</span>– <span class="NLM_lpage">43</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1002%2F9781118832738.ch2" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2014&pages=19-43&author=R.+A.+Goodnowauthor=R.+A.+Goodnow&title=A+Handbook+for+DNA-Encoded+Chemistry"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1002%2F9781118832738.ch2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1002%252F9781118832738.ch2%26sid%3Dliteratum%253Aachs%26aulast%3DGoodnow%26aufirst%3DR.%2BA.%26atitle%3DA%2520Brief%2520History%2520of%2520the%2520Development%2520of%2520Combinatorial%2520Chemistry%2520and%2520the%2520Emerging%2520Need%2520for%2520DNA-Encoded%2520Chemistry%26btitle%3DA%2520Handbook%2520for%2520DNA-Encoded%2520Chemistry%26aulast%3DGoodnow%26aufirst%3DR.%2BA.%26pub%3DJohn%2520Wiley%2520%2526%2520Sons%252C%2520Inc%26date%3D2014%26spage%3D19%26epage%3D43" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref63"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref63'); return false;" data-citation="" class="refNumLink">63</a></strong><div class="NLM_citation" id="cit63"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brenner, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lerner, R. A.</span></span> <span> </span><span class="NLM_article-title">Encoded Combinatorial Chemistry</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">1992</span>,  <span class="NLM_volume"><i>89</i></span>,  <span class="NLM_fpage">5381</span>– <span class="NLM_lpage">5383</span>, <span class="refDoi"> DOI: 10.1073/pnas.89.12.5381</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1073%2Fpnas.89.12.5381" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1608946" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADyaK38XksVeqsrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=89&publication_year=1992&pages=5381-5383&author=S.+Brennerauthor=R.+A.+Lerner&title=Encoded+Combinatorial+Chemistry&doi=10.1073%2Fpnas.89.12.5381"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit63R"><div class="casContent"><span class="casTitleNuber">63</span><div class="casTitle"><span class="NLM_cas:atitle">Encoded combinatorial chemistry</span></div><div class="casAuthors">Brenner, Sydney; Lerner, Richard A.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">1992</span>),
    <span class="NLM_cas:volume">89</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">5381-3</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    </div><div class="casAbstract">The diversity of chem. synthesis and the power of genetics are linked to provide a powerful, versatile method for drug screening.  A process of alternating parallel combinatorial synthesis is used to encode individual members of a large library of chems. with unique nucleotide sequences.  After the chem. entity is bound to a target, the genetic tag can be amplified by replication and utilized for enrichment of the bound mols. by serial hybridization to a subset of the library.  The library of the tagged chems. is termed an encoded combinatorial chem. library.  The nature of the chem. structure bound to the receptor is decoded by sequencing the nucleotide tag.  For example, with a library of peptides, each peptide was linked to a specifically designed oligodeoxynucleotide (genetic tag).  Each of these oligodeoxynucleotides had a signature (unique) sequence corresponding to the linked peptide and was flanked by sequences recognizable by designed DNA primers of the polymerase chain reaction (PCR).  After a desired member of this library of peptide-oligonucleotides was selected by screening (e.g., selected by binding to a target), this desired member was then identified by PCR amplification of the signature nucleotide sequence.  Detn. of the signature sequence from PCR amplified DNA gave the identity of the desired peptide.  Thus, this method can be used to identify a few members in a large chem. library.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoxFs-UqY_2JbVg90H21EOLACvtfcHk0liTP02v6WJ3sg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK38XksVeqsrs%253D&md5=3f03fc103c039ce7985cd7b25142c7ed</span></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1073%2Fpnas.89.12.5381&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.89.12.5381%26sid%3Dliteratum%253Aachs%26aulast%3DBrenner%26aufirst%3DS.%26aulast%3DLerner%26aufirst%3DR.%2BA.%26atitle%3DEncoded%2520Combinatorial%2520Chemistry%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D1992%26volume%3D89%26spage%3D5381%26epage%3D5383%26doi%3D10.1073%2Fpnas.89.12.5381" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref64"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref64'); return false;" data-citation="" class="refNumLink">64</a></strong><div class="NLM_citation" id="cit64"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Neri, D.</span></span> <span> </span><span class="NLM_article-title">Twenty-Five Years of DNA-Encoded Chemical Libraries</span>. <i>ChemBioChem</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">827</span>– <span class="NLM_lpage">828</span>, <span class="refDoi"> DOI: 10.1002/cbic.201700130</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1002%2Fcbic.201700130" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=28318088" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC2sXksV2lur0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2017&pages=827-828&author=D.+Neri&title=Twenty-Five+Years+of+DNA-Encoded+Chemical+Libraries&doi=10.1002%2Fcbic.201700130"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit64R"><div class="casContent"><span class="casTitleNuber">64</span><div class="casTitle"><span class="NLM_cas:atitle">Twenty-five Years of DNA-Encoded Chemical Libraries</span></div><div class="casAuthors">Neri, Dario</div><div class="citationInfo"><span class="NLM_cas:title">ChemBioChem</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">827-828</span>CODEN:
                <span class="NLM_cas:coden">CBCHFX</span>;
        ISSN:<span class="NLM_cas:issn">1439-4227</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr_MwQL5lTsnbVg90H21EOLACvtfcHk0liTP02v6WJ3sg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXksV2lur0%253D&md5=02b1b7f3ce7e3f49d572c1166b7bc049</span></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=10.1002%2Fcbic.201700130&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcbic.201700130%26sid%3Dliteratum%253Aachs%26aulast%3DNeri%26aufirst%3DD.%26atitle%3DTwenty-Five%2520Years%2520of%2520DNA-Encoded%2520Chemical%2520Libraries%26jtitle%3DChemBioChem%26date%3D2017%26volume%3D18%26spage%3D827%26epage%3D828%26doi%3D10.1002%2Fcbic.201700130" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref65"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref65'); return false;" data-citation="" class="refNumLink">65</a></strong><div class="NLM_citation" id="cit65"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Malone, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paegel, B. M.</span></span> <span> </span><span class="NLM_article-title">What Is a “DNA-Compatible” Reaction?</span>. <i>ACS Comb. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">182</span>– <span class="NLM_lpage">187</span>, <span class="refDoi"> DOI: 10.1021/acscombsci.5b00198</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acscombsci.5b00198" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC28XktFaqu7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2016&pages=182-187&author=M.+L.+Maloneauthor=B.+M.+Paegel&title=What+Is+a+%E2%80%9CDNA-Compatible%E2%80%9D+Reaction%3F&doi=10.1021%2Facscombsci.5b00198"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit65R"><div class="casContent"><span class="casTitleNuber">65</span><div class="casTitle"><span class="NLM_cas:atitle">What is a "DNA-Compatible" Reaction?</span></div><div class="casAuthors">Malone, Marie L.; Paegel, Brian M.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Combinatorial Science</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">182-187</span>CODEN:
                <span class="NLM_cas:coden">ACSCCC</span>;
        ISSN:<span class="NLM_cas:issn">2156-8944</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">DNA-encoded reaction rehearsal was developed as a method to assay the yields of a potential DNA-encoded reaction and the stability of the DNA strand in one pot under the same conditions.  Magnetic DNA-functionalized sensor beads quant. report the % DNA template mols. remaining viable for PCR amplification after exposure to test reaction conditions; resin-bound peptides with the appropriate functionality were used to det. the chem. yields of functionalized products.  Anal. of solid-phase bond forming (e.g., Suzuki-Miyaura cross-coupling, reductive amination) and deprotection reactions (e.g., allyl esters, silyl ethers) guided the definition and optimization of DNA-compatible reaction conditions (>90% yield, >30% viable DNA mols.), most notably in cases that involved known (H+, Pd) and more obscure (Δ, DMF) hazards to DNA integrity.  The data provide an empirical yet mechanistically consistent and predictive framework for designing successful DNA-encoded reaction sequences for combinatorial library synthesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGocOsuE1mW_x7Vg90H21EOLACvtfcHk0liTP02v6WJ3sg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XktFaqu7c%253D&md5=991cbb9fc51fe1de0160b79029fe325a</span></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=10.1021%2Facscombsci.5b00198&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facscombsci.5b00198%26sid%3Dliteratum%253Aachs%26aulast%3DMalone%26aufirst%3DM.%2BL.%26aulast%3DPaegel%26aufirst%3DB.%2BM.%26atitle%3DWhat%2520Is%2520a%2520%25E2%2580%259CDNA-Compatible%25E2%2580%259D%2520Reaction%253F%26jtitle%3DACS%2520Comb.%2520Sci.%26date%3D2016%26volume%3D18%26spage%3D182%26epage%3D187%26doi%3D10.1021%2Facscombsci.5b00198" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref66"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref66'); return false;" data-citation="" class="refNumLink">66</a></strong><div class="NLM_citation" id="cit66"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Goodnow, R. A.,  Jr</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dumelin, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keefe, A. D.</span></span> <span> </span><span class="NLM_article-title">DNA-Encoded Chemistry: Enabling the Deeper Sampling of Chemical Space</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">131</span>– <span class="NLM_lpage">147</span>, <span class="refDoi"> DOI: 10.1038/nrd.2016.213</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1038%2Fnrd.2016.213" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=27932801" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC28XitVKkurjM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2017&pages=131-147&author=R.+A.+Goodnowauthor=C.+E.+Dumelinauthor=A.+D.+Keefe&title=DNA-Encoded+Chemistry%3A+Enabling+the+Deeper+Sampling+of+Chemical+Space&doi=10.1038%2Fnrd.2016.213"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit66R"><div class="casContent"><span class="casTitleNuber">66</span><div class="casTitle"><span class="NLM_cas:atitle">DNA-encoded chemistry: enabling the deeper sampling of chemical space</span></div><div class="casAuthors">Goodnow, Robert A. Jr; Dumelin, Christoph E.; Keefe, Anthony D.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">131-147</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  DNA-encoded chem. library technologies are increasingly being adopted in drug discovery for hit and lead generation.  DNA-encoded chem. enables the exploration of chem. spaces four to five orders of magnitude more deeply than is achievable by traditional high-throughput screening methods.  Operation of this technol. requires developing a range of capabilities including aq. synthetic chem., building block acquisition, oligonucleotide conjugation, large-scale mol. biol. transformations, selection methodologies, PCR, sequencing, sequence data anal. and the anal. of large chem. spaces.  This Review provides an overview of the development and applications of DNA-encoded chem., highlighting the challenges and future directions for the use of this technol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrdEd70MeexerVg90H21EOLACvtfcHk0liTP02v6WJ3sg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitVKkurjM&md5=7ca97b01d75e600378799baf77665c07</span></div><a href="/servlet/linkout?suffix=cit66&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2016.213&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2016.213%26sid%3Dliteratum%253Aachs%26aulast%3DGoodnow%26aufirst%3DR.%2BA.%26aulast%3DDumelin%26aufirst%3DC.%2BE.%26aulast%3DKeefe%26aufirst%3DA.%2BD.%26atitle%3DDNA-Encoded%2520Chemistry%253A%2520Enabling%2520the%2520Deeper%2520Sampling%2520of%2520Chemical%2520Space%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2017%26volume%3D16%26spage%3D131%26epage%3D147%26doi%3D10.1038%2Fnrd.2016.213" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref67"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref67'); return false;" data-citation="" class="refNumLink">67</a></strong><div class="NLM_citation" id="cit67"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Harris, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berger, S. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeong, J. U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nagilla, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bandyopadhyay, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campobasso, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Capriotti, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cox, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dare, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eidam, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Finger, J. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoffman, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kasparcova, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">King, B. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lehr, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lan, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leister, L. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lich, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MacDonald, T. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, N. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ouellette, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pao, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rahman, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reilly, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rendina, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rivera, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schaeffer, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sehon, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singhaus, R. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, H. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Swift, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Totoritis, R. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vossenkämper, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ward, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wisnoski, D. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marquis, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gough, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bertin, J.</span></span> <span> </span><span class="NLM_article-title">Discovery of a First-in-Class Receptor Interacting Protein 1 (RIP1) Kinase Specific Clinical Candidate (GSK2982772) for the Treatment of Inflammatory Diseases</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">1247</span>– <span class="NLM_lpage">1261</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b01751</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01751" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC2sXitVeksLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=1247-1261&author=P.+A.+Harrisauthor=S.+B.+Bergerauthor=J.+U.+Jeongauthor=R.+Nagillaauthor=D.+Bandyopadhyayauthor=N.+Campobassoauthor=C.+A.+Capriottiauthor=J.+A.+Coxauthor=L.+Dareauthor=X.+Dongauthor=P.+M.+Eidamauthor=J.+N.+Fingerauthor=S.+J.+Hoffmanauthor=J.+Kangauthor=V.+Kasparcovaauthor=B.+W.+Kingauthor=R.+Lehrauthor=Y.+Lanauthor=L.+K.+Leisterauthor=J.+D.+Lichauthor=T.+T.+MacDonaldauthor=N.+A.+Millerauthor=M.+T.+Ouelletteauthor=C.+S.+Paoauthor=A.+Rahmanauthor=M.+A.+Reillyauthor=A.+R.+Rendinaauthor=E.+J.+Riveraauthor=M.+C.+Schaefferauthor=C.+A.+Sehonauthor=R.+R.+Singhausauthor=H.+H.+Sunauthor=B.+A.+Swiftauthor=R.+D.+Totoritisauthor=A.+Vossenk%C3%A4mperauthor=P.+Wardauthor=D.+D.+Wisnoskiauthor=D.+Zhangauthor=R.+W.+Marquisauthor=P.+J.+Goughauthor=J.+Bertin&title=Discovery+of+a+First-in-Class+Receptor+Interacting+Protein+1+%28RIP1%29+Kinase+Specific+Clinical+Candidate+%28GSK2982772%29+for+the+Treatment+of+Inflammatory+Diseases&doi=10.1021%2Facs.jmedchem.6b01751"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit67R"><div class="casContent"><span class="casTitleNuber">67</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a First-in-Class Receptor Interacting Protein 1 (RIP1) Kinase Specific Clinical Candidate (GSK2982772) for the Treatment of Inflammatory Diseases</span></div><div class="casAuthors">Harris, Philip A.; Berger, Scott B.; Jeong, Jae U.; Nagilla, Rakesh; Bandyopadhyay, Deepak; Campobasso, Nino; Capriotti, Carol A.; Cox, Julie A.; Dare, Lauren; Dong, Xiaoyang; Eidam, Patrick M.; Finger, Joshua N.; Hoffman, Sandra J.; Kang, James; Kasparcova, Viera; King, Bryan W.; Lehr, Ruth; Lan, Yunfeng; Leister, Lara K.; Lich, John D.; MacDonald, Thomas T.; Miller, Nathan A.; Ouellette, Michael T.; Pao, Christina S.; Rahman, Attiq; Reilly, Michael A.; Rendina, Alan R.; Rivera, Elizabeth J.; Schaeffer, Michelle C.; Sehon, Clark A.; Singhaus, Robert R.; Sun, Helen H.; Swift, Barbara A.; Totoritis, Rachel D.; Vossenkamper, Anna; Ward, Paris; Wisnoski, David D.; Zhang, Daohua; Marquis, Robert W.; Gough, Peter J.; Bertin, John</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1247-1261</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">RIP1 regulates necroptosis and inflammation and may play an important role in contributing to a variety of human pathologies, including immune-mediated inflammatory diseases.  Small-mol. inhibitors of RIP1 kinase that are suitable for advancement into the clinic have yet to be described.  Herein, the authors report the lead optimization of a benzoxazepinone hit from a DNA-encoded library and the discovery and profile of clin. candidate GSK2982772 (compd. I), currently in phase 2a clin. studies for psoriasis, rheumatoid arthritis, and ulcerative colitis.  Compd. I potently binds to RIP1 with exquisite kinase specificity and has excellent activity in blocking many TNF-dependent cellular responses.  Highlighting its potential as a novel anti-inflammatory agent, the inhibitor was also able to reduce spontaneous prodn. of cytokines from human ulcerative colitis explants.  The highly favorable physicochem. and ADMET properties of I, combined with high potency, led to a predicted low oral dose in humans.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqZGRDTQv-vs7Vg90H21EOLACvtfcHk0lgeoEiRtLAq5A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXitVeksLk%253D&md5=ee0c3c60bb1351dff696490ac9edb1cc</span></div><a href="/servlet/linkout?suffix=cit67&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01751&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01751%26sid%3Dliteratum%253Aachs%26aulast%3DHarris%26aufirst%3DP.%2BA.%26aulast%3DBerger%26aufirst%3DS.%2BB.%26aulast%3DJeong%26aufirst%3DJ.%2BU.%26aulast%3DNagilla%26aufirst%3DR.%26aulast%3DBandyopadhyay%26aufirst%3DD.%26aulast%3DCampobasso%26aufirst%3DN.%26aulast%3DCapriotti%26aufirst%3DC.%2BA.%26aulast%3DCox%26aufirst%3DJ.%2BA.%26aulast%3DDare%26aufirst%3DL.%26aulast%3DDong%26aufirst%3DX.%26aulast%3DEidam%26aufirst%3DP.%2BM.%26aulast%3DFinger%26aufirst%3DJ.%2BN.%26aulast%3DHoffman%26aufirst%3DS.%2BJ.%26aulast%3DKang%26aufirst%3DJ.%26aulast%3DKasparcova%26aufirst%3DV.%26aulast%3DKing%26aufirst%3DB.%2BW.%26aulast%3DLehr%26aufirst%3DR.%26aulast%3DLan%26aufirst%3DY.%26aulast%3DLeister%26aufirst%3DL.%2BK.%26aulast%3DLich%26aufirst%3DJ.%2BD.%26aulast%3DMacDonald%26aufirst%3DT.%2BT.%26aulast%3DMiller%26aufirst%3DN.%2BA.%26aulast%3DOuellette%26aufirst%3DM.%2BT.%26aulast%3DPao%26aufirst%3DC.%2BS.%26aulast%3DRahman%26aufirst%3DA.%26aulast%3DReilly%26aufirst%3DM.%2BA.%26aulast%3DRendina%26aufirst%3DA.%2BR.%26aulast%3DRivera%26aufirst%3DE.%2BJ.%26aulast%3DSchaeffer%26aufirst%3DM.%2BC.%26aulast%3DSehon%26aufirst%3DC.%2BA.%26aulast%3DSinghaus%26aufirst%3DR.%2BR.%26aulast%3DSun%26aufirst%3DH.%2BH.%26aulast%3DSwift%26aufirst%3DB.%2BA.%26aulast%3DTotoritis%26aufirst%3DR.%2BD.%26aulast%3DVossenk%25C3%25A4mper%26aufirst%3DA.%26aulast%3DWard%26aufirst%3DP.%26aulast%3DWisnoski%26aufirst%3DD.%2BD.%26aulast%3DZhang%26aufirst%3DD.%26aulast%3DMarquis%26aufirst%3DR.%2BW.%26aulast%3DGough%26aufirst%3DP.%2BJ.%26aulast%3DBertin%26aufirst%3DJ.%26atitle%3DDiscovery%2520of%2520a%2520First-in-Class%2520Receptor%2520Interacting%2520Protein%25201%2520%2528RIP1%2529%2520Kinase%2520Specific%2520Clinical%2520Candidate%2520%2528GSK2982772%2529%2520for%2520the%2520Treatment%2520of%2520Inflammatory%2520Diseases%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D1247%26epage%3D1261%26doi%3D10.1021%2Facs.jmedchem.6b01751" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref68"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref68'); return false;" data-citation="" class="refNumLink">68</a></strong><div class="NLM_citation" id="cit68"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Škopić, M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salamon, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bugain, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jung, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gohla, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doetsch, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">dos Santos, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhat, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wagner, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brunschweiger, A.</span></span> <span> </span><span class="NLM_article-title">Acid- and Au(I)-Mediated Synthesis of Hexathymidine-DNA-Heterocycle Chimeras, an Efficient Entry to DNA-Encoded Libraries Inspired by Drug Structures</span>. <i>Chem. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">3356</span>– <span class="NLM_lpage">3361</span>, <span class="refDoi"> DOI: 10.1039/C7SC00455A</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1039%2FC7SC00455A" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=28507705" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC2sXjsVKmtrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=3356-3361&author=M.+K.+%C5%A0kopi%C4%87author=H.+Salamonauthor=O.+Bugainauthor=K.+Jungauthor=A.+Gohlaauthor=L.+J.+Doetschauthor=D.+dos+Santosauthor=A.+Bhatauthor=B.+Wagnerauthor=A.+Brunschweiger&title=Acid-+and+Au%28I%29-Mediated+Synthesis+of+Hexathymidine-DNA-Heterocycle+Chimeras%2C+an+Efficient+Entry+to+DNA-Encoded+Libraries+Inspired+by+Drug+Structures&doi=10.1039%2FC7SC00455A"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit68R"><div class="casContent"><span class="casTitleNuber">68</span><div class="casTitle"><span class="NLM_cas:atitle">Acid- and Au(I)-mediated synthesis of hexathymidine-DNA-heterocycle chimeras, an efficient entry to DNA-encoded libraries inspired by drug structures</span></div><div class="casAuthors">Skopic, Mateja Klika; Salamon, Hazem; Bugain, Olivia; Jung, Kathrin; Gohla, Anne; Doetsch, Lara J.; Santos, Denise dos; Bhat, Avinash; Wagner, Bernd; Brunschweiger, Andreas</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Science</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">3356-3361</span>CODEN:
                <span class="NLM_cas:coden">CSHCCN</span>;
        ISSN:<span class="NLM_cas:issn">2041-6520</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Libraries of DNA-tagged compds. are a validated screening technol. for drug discovery.  They are synthesized through combinatorial iterations of alternated coding and preparative synthesis steps.  Thus, large chem. space can be accessed for target-based screening.  However, the need to preserve the functionality of the DNA tag severely restricts the choice of chem. methods for library synthesis.  Acidic organocatalysts, transition metals, and oxidants furnish diverse drug-like structures from simple starting materials, but cause loss of genetic information by depurination.  A hexathymidine oligonucleotide, called "hexT" allows the chemist utilizing these classes of catalysts to access a potentially broad variety of structures in the initial step of library synthesis.  We exploited its catalyst tolerance to efficiently synthesize diverse substituted β-carbolines, pyrazolines, and pyrazoles from readily available starting materials as hexT conjugates by acid- and Au(I)-catalysis, resp.  The hexT conjugates were ligated to coding DNA sequences yielding encoded screening libraries inspired by drug structures.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGouSdRpFmyaN7Vg90H21EOLACvtfcHk0lgeoEiRtLAq5A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXjsVKmtrg%253D&md5=cc6a646c1e6ecf3cdfae7d36cca63440</span></div><a href="/servlet/linkout?suffix=cit68&amp;dbid=16384&amp;doi=10.1039%2FC7SC00455A&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC7SC00455A%26sid%3Dliteratum%253Aachs%26aulast%3D%25C5%25A0kopi%25C4%2587%26aufirst%3DM.%2BK.%26aulast%3DSalamon%26aufirst%3DH.%26aulast%3DBugain%26aufirst%3DO.%26aulast%3DJung%26aufirst%3DK.%26aulast%3DGohla%26aufirst%3DA.%26aulast%3DDoetsch%26aufirst%3DL.%2BJ.%26aulast%3Ddos%2BSantos%26aufirst%3DD.%26aulast%3DBhat%26aufirst%3DA.%26aulast%3DWagner%26aufirst%3DB.%26aulast%3DBrunschweiger%26aufirst%3DA.%26atitle%3DAcid-%2520and%2520Au%2528I%2529-Mediated%2520Synthesis%2520of%2520Hexathymidine-DNA-Heterocycle%2520Chimeras%252C%2520an%2520Efficient%2520Entry%2520to%2520DNA-Encoded%2520Libraries%2520Inspired%2520by%2520Drug%2520Structures%26jtitle%3DChem.%2520Sci.%26date%3D2017%26volume%3D8%26spage%3D3356%26epage%3D3361%26doi%3D10.1039%2FC7SC00455A" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref69"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref69'); return false;" data-citation="" class="refNumLink">69</a></strong><div class="NLM_citation" id="cit69"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Graybill, T. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Platchek, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bodmer, V. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wisnoski, D. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coppo, F. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tamburino, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scavello, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Franklin, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mataruse, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bedard, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chai, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Summerfield, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Centrella, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Messer, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pope, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Israel, D. I.</span></span> <span> </span><span class="NLM_article-title">Cell-Based Selection Expands the Utility of DNA-Encoded Small-Molecule Library Technology to Cell Surface Drug Targets: Identification of Novel Antagonists of the NK3 Tachykinin Receptor</span>. <i>ACS Comb. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">722</span>– <span class="NLM_lpage">731</span>, <span class="refDoi"> DOI: 10.1021/acscombsci.5b00124</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acscombsci.5b00124" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvVaht7jM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2015&pages=722-731&author=Z.+Wuauthor=T.+L.+Graybillauthor=X.+Zengauthor=M.+Platchekauthor=J.+Zhangauthor=V.+Q.+Bodmerauthor=D.+D.+Wisnoskiauthor=J.+Dengauthor=F.+T.+Coppoauthor=G.+Yaoauthor=A.+Tamburinoauthor=G.+Scavelloauthor=G.+J.+Franklinauthor=S.+Mataruseauthor=K.+L.+Bedardauthor=Y.+Dingauthor=J.+Chaiauthor=J.+Summerfieldauthor=P.+A.+Centrellaauthor=J.+A.+Messerauthor=A.+J.+Popeauthor=D.+I.+Israel&title=Cell-Based+Selection+Expands+the+Utility+of+DNA-Encoded+Small-Molecule+Library+Technology+to+Cell+Surface+Drug+Targets%3A+Identification+of+Novel+Antagonists+of+the+NK3+Tachykinin+Receptor&doi=10.1021%2Facscombsci.5b00124"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit69R"><div class="casContent"><span class="casTitleNuber">69</span><div class="casTitle"><span class="NLM_cas:atitle">Cell-Based Selection Expands the Utility of DNA-Encoded Small-Molecule Library Technology to Cell Surface Drug Targets: Identification of Novel Antagonists of the NK3 Tachykinin Receptor</span></div><div class="casAuthors">Wu, Zining; Graybill, Todd L.; Zeng, Xin; Platchek, Michael; Zhang, Jean; Bodmer, Vera Q.; Wisnoski, David D.; Deng, Jianghe; Coppo, Frank T.; Yao, Gang; Tamburino, Alex; Scavello, Genaro; Franklin, G. Joseph; Mataruse, Sibongile; Bedard, Katie L.; Ding, Yun; Chai, Jing; Summerfield, Jennifer; Centrella, Paolo A.; Messer, Jeffrey A.; Pope, Andrew J.; Israel, David I.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Combinatorial Science</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">722-731</span>CODEN:
                <span class="NLM_cas:coden">ACSCCC</span>;
        ISSN:<span class="NLM_cas:issn">2156-8944</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">DNA-encoded small-mol. library technol. has recently emerged as a new paradigm for identifying ligands against drug targets.  To date, this technol. has been used with sol. protein targets that are produced and used in a purified state.  Here, we describe a cell-based method for identifying small-mol. ligands from DNA-encoded libraries against integral membrane protein targets.  We use this method to identify novel, potent, and specific inhibitors of NK3, a member of the tachykinin family of G-protein coupled receptors (GPCRs).  The method is simple and broadly applicable to other GPCRs and integral membrane proteins.  We have extended the application of DNA-encoded library technol. to membrane-assocd. targets and demonstrate the feasibility of selecting DNA-tagged, small-mol. ligands from complex combinatorial libraries against targets in a heterogeneous milieu, such as the surface of a cell.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpMG6EaclP47bVg90H21EOLACvtfcHk0lg0Boq-vf3Vpg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvVaht7jM&md5=eb7ebd0e31863dc8d756c14e9248791d</span></div><a href="/servlet/linkout?suffix=cit69&amp;dbid=16384&amp;doi=10.1021%2Facscombsci.5b00124&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facscombsci.5b00124%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DZ.%26aulast%3DGraybill%26aufirst%3DT.%2BL.%26aulast%3DZeng%26aufirst%3DX.%26aulast%3DPlatchek%26aufirst%3DM.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DBodmer%26aufirst%3DV.%2BQ.%26aulast%3DWisnoski%26aufirst%3DD.%2BD.%26aulast%3DDeng%26aufirst%3DJ.%26aulast%3DCoppo%26aufirst%3DF.%2BT.%26aulast%3DYao%26aufirst%3DG.%26aulast%3DTamburino%26aufirst%3DA.%26aulast%3DScavello%26aufirst%3DG.%26aulast%3DFranklin%26aufirst%3DG.%2BJ.%26aulast%3DMataruse%26aufirst%3DS.%26aulast%3DBedard%26aufirst%3DK.%2BL.%26aulast%3DDing%26aufirst%3DY.%26aulast%3DChai%26aufirst%3DJ.%26aulast%3DSummerfield%26aufirst%3DJ.%26aulast%3DCentrella%26aufirst%3DP.%2BA.%26aulast%3DMesser%26aufirst%3DJ.%2BA.%26aulast%3DPope%26aufirst%3DA.%2BJ.%26aulast%3DIsrael%26aufirst%3DD.%2BI.%26atitle%3DCell-Based%2520Selection%2520Expands%2520the%2520Utility%2520of%2520DNA-Encoded%2520Small-Molecule%2520Library%2520Technology%2520to%2520Cell%2520Surface%2520Drug%2520Targets%253A%2520Identification%2520of%2520Novel%2520Antagonists%2520of%2520the%2520NK3%2520Tachykinin%2520Receptor%26jtitle%3DACS%2520Comb.%2520Sci.%26date%3D2015%26volume%3D17%26spage%3D722%26epage%3D731%26doi%3D10.1021%2Facscombsci.5b00124" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref70"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref70'); return false;" data-citation="" class="refNumLink">70</a></strong><div class="NLM_citation" id="cit70"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Henrich, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hein, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dötsch, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernhard, F.</span></span> <span> </span><span class="NLM_article-title">Membrane Protein Production in Escherichia Coli Cell-Free Lysates</span>. <i>FEBS Lett.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>589</i></span>,  <span class="NLM_fpage">1713</span>– <span class="NLM_lpage">1722</span>, <span class="refDoi"> DOI: 10.1016/j.febslet.2015.04.045</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1016%2Fj.febslet.2015.04.045" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=589&publication_year=2015&pages=1713-1722&author=E.+Henrichauthor=C.+Heinauthor=V.+D%C3%B6tschauthor=F.+Bernhard&title=Membrane+Protein+Production+in+Escherichia+Coli+Cell-Free+Lysates&doi=10.1016%2Fj.febslet.2015.04.045"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit70&amp;dbid=16384&amp;doi=10.1016%2Fj.febslet.2015.04.045&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.febslet.2015.04.045%26sid%3Dliteratum%253Aachs%26aulast%3DHenrich%26aufirst%3DE.%26aulast%3DHein%26aufirst%3DC.%26aulast%3DD%25C3%25B6tsch%26aufirst%3DV.%26aulast%3DBernhard%26aufirst%3DF.%26atitle%3DMembrane%2520Protein%2520Production%2520in%2520Escherichia%2520Coli%2520Cell-Free%2520Lysates%26jtitle%3DFEBS%2520Lett.%26date%3D2015%26volume%3D589%26spage%3D1713%26epage%3D1722%26doi%3D10.1016%2Fj.febslet.2015.04.045" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref71"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref71'); return false;" data-citation="" class="refNumLink">71</a></strong><div class="NLM_citation" id="cit71"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shinoda, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shinya, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ito, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ishizuka-Katsura, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohsawa, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Terada, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hirata, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kawano, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamamoto, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomita, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ishibashi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hirabayashi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kimura-Someya, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shirouzu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yokoyama, S.</span></span> <span> </span><span class="NLM_article-title">Cell-Free Methods to Produce Structurally Intact Mammalian Membrane Proteins</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">30442</span>, <span class="refDoi"> DOI: 10.1038/srep30442</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1038%2Fsrep30442" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=27465719" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC2sXksFeltr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2016&pages=30442&author=T.+Shinodaauthor=N.+Shinyaauthor=K.+Itoauthor=Y.+Ishizuka-Katsuraauthor=N.+Ohsawaauthor=T.+Teradaauthor=K.+Hirataauthor=Y.+Kawanoauthor=M.+Yamamotoauthor=T.+Tomitaauthor=Y.+Ishibashiauthor=Y.+Hirabayashiauthor=T.+Kimura-Someyaauthor=M.+Shirouzuauthor=S.+Yokoyama&title=Cell-Free+Methods+to+Produce+Structurally+Intact+Mammalian+Membrane+Proteins&doi=10.1038%2Fsrep30442"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit71R"><div class="casContent"><span class="casTitleNuber">71</span><div class="casTitle"><span class="NLM_cas:atitle">Cell-free methods to produce structurally intact mammalian membrane proteins</span></div><div class="casAuthors">Shinoda, Takehiro; Shinya, Naoko; Ito, Kaori; Ishizuka-Katsura, Yoshiko; Ohsawa, Noboru; Terada, Takaho; Hirata, Kunio; Kawano, Yoshiaki; Yamamoto, Masaki; Tomita, Taisuke; Ishibashi, Yohei; Hirabayashi, Yoshio; Kimura-Someya, Tomomi; Shirouzu, Mikako; Yokoyama, Shigeyuki</div><div class="citationInfo"><span class="NLM_cas:title">Scientific Reports</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">30442</span>CODEN:
                <span class="NLM_cas:coden">SRCEC3</span>;
        ISSN:<span class="NLM_cas:issn">2045-2322</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The crystal structures of four membrane proteins, from bacteria or a unicellular alga, have been solved with samples produced by cell-free protein synthesis.  In this study, for mammalian membrane protein prodn., we established the pptg. and sol. membrane fragment methods: membrane proteins are synthesized with the Escherichia coli cell-free system in the presence of large and small membrane fragments, resp., and are simultaneously integrated into the lipid environments.  We applied the pptg. membrane fragment method to produce various mammalian membrane proteins, including human claudins, glucosylceramide synthase, and the γ-secretase subunits.  These proteins were produced at levels of about 0.1-1.0 mg per mL cell-free reaction under the initial conditions, and were obtained as ppts. by ultracentrifugation.  Larger amts. of membrane proteins were produced by the sol. membrane fragment method, collected in the ultracentrifugation supernatants, and purified directly by column chromatog.  For several proteins, the conditions of the membrane fragment methods were further optimized, such as by the addn. of specific lipids/detergents.  The functional and structural integrities of the purified proteins were confirmed by analyses of their ligand binding activities, size-exclusion chromatog. profiles, and/or thermal stabilities.  We successfully obtained high-quality crystals of the complex of human claudin-4 with an enterotoxin.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGptV1wwXNXR-LVg90H21EOLACvtfcHk0lg0Boq-vf3Vpg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXksFeltr8%253D&md5=4324f644d00ee011808931a4f410f6bf</span></div><a href="/servlet/linkout?suffix=cit71&amp;dbid=16384&amp;doi=10.1038%2Fsrep30442&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsrep30442%26sid%3Dliteratum%253Aachs%26aulast%3DShinoda%26aufirst%3DT.%26aulast%3DShinya%26aufirst%3DN.%26aulast%3DIto%26aufirst%3DK.%26aulast%3DIshizuka-Katsura%26aufirst%3DY.%26aulast%3DOhsawa%26aufirst%3DN.%26aulast%3DTerada%26aufirst%3DT.%26aulast%3DHirata%26aufirst%3DK.%26aulast%3DKawano%26aufirst%3DY.%26aulast%3DYamamoto%26aufirst%3DM.%26aulast%3DTomita%26aufirst%3DT.%26aulast%3DIshibashi%26aufirst%3DY.%26aulast%3DHirabayashi%26aufirst%3DY.%26aulast%3DKimura-Someya%26aufirst%3DT.%26aulast%3DShirouzu%26aufirst%3DM.%26aulast%3DYokoyama%26aufirst%3DS.%26atitle%3DCell-Free%2520Methods%2520to%2520Produce%2520Structurally%2520Intact%2520Mammalian%2520Membrane%2520Proteins%26jtitle%3DSci.%2520Rep.%26date%3D2016%26volume%3D6%26spage%3D30442%26doi%3D10.1038%2Fsrep30442" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref72"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref72'); return false;" data-citation="" class="refNumLink">72</a></strong><div class="NLM_citation" id="cit72"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Desuzinges
Mandon, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Agez, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pellegrin, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Igonet, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jawhari, A.</span></span> <span> </span><span class="NLM_article-title">Novel Systematic Detergent Screening Method for Membrane Proteins Solubilization</span>. <i>Anal. Biochem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>517</i></span>,  <span class="NLM_fpage">40</span>– <span class="NLM_lpage">49</span>, <span class="refDoi"> DOI: 10.1016/j.ab.2016.11.008</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1016%2Fj.ab.2016.11.008" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=27847172" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvFKjs7vF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=517&publication_year=2017&pages=40-49&author=E.+Desuzinges%0AMandonauthor=M.+Agezauthor=R.+Pellegrinauthor=S.+Igonetauthor=A.+Jawhari&title=Novel+Systematic+Detergent+Screening+Method+for+Membrane+Proteins+Solubilization&doi=10.1016%2Fj.ab.2016.11.008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit72R"><div class="casContent"><span class="casTitleNuber">72</span><div class="casTitle"><span class="NLM_cas:atitle">Novel systematic detergent screening method for membrane proteins solubilization</span></div><div class="casAuthors">Desuzinges Mandon, Elodie; Agez, Morgane; Pellegrin, Rebecca; Igonet, Sebastien; Jawhari, Anass</div><div class="citationInfo"><span class="NLM_cas:title">Analytical Biochemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">517</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">40-49</span>CODEN:
                <span class="NLM_cas:coden">ANBCA2</span>;
        ISSN:<span class="NLM_cas:issn">0003-2697</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Membrane proteins play crucial role in many cellular processes including cell adhesion, cell-cell communication, signal transduction and transport.  To better understand the mol. basis of such central biol. machines and to specifically study their biol. and medical role, it is necessary to ext. them from their membrane environment.  To do so, it is challenging to find the best solubilization condition.  Here the authors describe, a systematic screening method called BMSS (Biotinylated Membranes Solubilization and Sepn.) that allow screening 96 conditions at once.  Streptavidin magnetic beads were used to sep. solubilized proteins from remaining biotinylated membranes after solubilization.  Relative quantification of dot blots help to select the best conditions to be confirmed by classical ultra-centrifugation and western blot.  Classical detergents with different phys.-chem. characteristics, novel calixarene based detergents and combination of both, were used for solubilization trials to obtain broad spectrum of conditions.  Here, the authors show the application of BMSS to discover solubilization conditions of a GPCR target (MP-A) and a transporter (MP-B).  The selected conditions allowed the solubilization and purifn. of non-aggregated and homogeneous native membrane proteins A and B.  Taken together, BMSS represent a rapid, reproducible and high throughput assessment of solubilization toward biochem./functional characterization, biophys. screening and structural studies of membrane proteins of high biol. and medical relevance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoaByHSw5OH8rVg90H21EOLACvtfcHk0lhl1DgGsNZXaw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvFKjs7vF&md5=3e2118032081cc3e637147b009d2a7f6</span></div><a href="/servlet/linkout?suffix=cit72&amp;dbid=16384&amp;doi=10.1016%2Fj.ab.2016.11.008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ab.2016.11.008%26sid%3Dliteratum%253Aachs%26aulast%3DDesuzinges%2BMandon%26aufirst%3DE.%26aulast%3DAgez%26aufirst%3DM.%26aulast%3DPellegrin%26aufirst%3DR.%26aulast%3DIgonet%26aufirst%3DS.%26aulast%3DJawhari%26aufirst%3DA.%26atitle%3DNovel%2520Systematic%2520Detergent%2520Screening%2520Method%2520for%2520Membrane%2520Proteins%2520Solubilization%26jtitle%3DAnal.%2520Biochem.%26date%3D2017%26volume%3D517%26spage%3D40%26epage%3D49%26doi%3D10.1016%2Fj.ab.2016.11.008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref73"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref73'); return false;" data-citation="" class="refNumLink">73</a></strong><div class="NLM_citation" id="cit73"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Arico-Muendel, C. C.</span></span> <span> </span><span class="NLM_article-title">From Haystack to Needle: Finding Value with DNA Encoded Library Technology at GSK</span>. <i>MedChemComm</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">1898</span>– <span class="NLM_lpage">1909</span>, <span class="refDoi"> DOI: 10.1039/C6MD00341A</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1039%2FC6MD00341A" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC28Xht1ehu7jL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=1898-1909&author=C.+C.+Arico-Muendel&title=From+Haystack+to+Needle%3A+Finding+Value+with+DNA+Encoded+Library+Technology+at+GSK&doi=10.1039%2FC6MD00341A"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit73R"><div class="casContent"><span class="casTitleNuber">73</span><div class="casTitle"><span class="NLM_cas:atitle">From haystack to needle: finding value with DNA encoded library technology at GSK</span></div><div class="casAuthors">Arico-Muendel, Christopher C.</div><div class="citationInfo"><span class="NLM_cas:title">MedChemComm</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1898-1909</span>CODEN:
                <span class="NLM_cas:coden">MCCEAY</span>;
        ISSN:<span class="NLM_cas:issn">2040-2503</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">DNA encoded library technol. (ELT) provides access to broad chem. diversity through affinity selection.  As an early adopter, GlaxoSmithKline enabled the development of ELT from proof of concept to full fledged contributor to the small mol. therapeutic pipeline.  Unique benefits of the GSK incubation include uptake of large nos. of building blocks and privileged scaffolds, guidance on library design, and opportunities to add value beyond lead discovery.  Sol. epoxide hydrolase and RIP1 kinase represent two case studies of ELT hits that have advanced into clin. studies.  ELT also shows promise as a strategy to forecast target tractability.  As an emerging technol., it benefits from incubation at the interface of industry and academia.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrGhx5inxOj_7Vg90H21EOLACvtfcHk0lhl1DgGsNZXaw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xht1ehu7jL&md5=1862af051a956ff1893143e107cff4f4</span></div><a href="/servlet/linkout?suffix=cit73&amp;dbid=16384&amp;doi=10.1039%2FC6MD00341A&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC6MD00341A%26sid%3Dliteratum%253Aachs%26aulast%3DArico-Muendel%26aufirst%3DC.%2BC.%26atitle%3DFrom%2520Haystack%2520to%2520Needle%253A%2520Finding%2520Value%2520with%2520DNA%2520Encoded%2520Library%2520Technology%2520at%2520GSK%26jtitle%3DMedChemComm%26date%3D2016%26volume%3D7%26spage%3D1898%26epage%3D1909%26doi%3D10.1039%2FC6MD00341A" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref74"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref74'); return false;" data-citation="" class="refNumLink">74</a></strong><div class="NLM_citation" id="cit74"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ahn, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kahsai, A. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pani, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Q.-T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wall, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Strachan, R. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Staus, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wingler, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, L. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sinnaeve, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cho, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, T. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hansen, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burnett, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lamerdin, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bassoni, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gavino, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Husemoen, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olsen, E. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Franch, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Costanzi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lefkowitz, R. J.</span></span> <span> </span><span class="NLM_article-title">Allosteric “beta-Blocker” Isolated from a DNA-Encoded Small Molecule Library</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>114</i></span>,  <span class="NLM_fpage">1708</span>– <span class="NLM_lpage">1713</span>, <span class="refDoi"> DOI: 10.1073/pnas.1620645114</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1073%2Fpnas.1620645114" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=28130548" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhs1Kjsbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=114&publication_year=2017&pages=1708-1713&author=S.+Ahnauthor=A.+W.+Kahsaiauthor=B.+Paniauthor=Q.-T.+Wangauthor=S.+Zhaoauthor=A.+L.+Wallauthor=R.+T.+Strachanauthor=D.+P.+Stausauthor=L.+M.+Winglerauthor=L.+D.+Sunauthor=J.+Sinnaeveauthor=M.+Choiauthor=T.+Choauthor=T.+T.+Xuauthor=G.+M.+Hansenauthor=M.+B.+Burnettauthor=J.+E.+Lamerdinauthor=D.+L.+Bassoniauthor=B.+J.+Gavinoauthor=G.+Husemoenauthor=E.+K.+Olsenauthor=T.+Franchauthor=S.+Costanziauthor=X.+Chenauthor=R.+J.+Lefkowitz&title=Allosteric+%E2%80%9Cbeta-Blocker%E2%80%9D+Isolated+from+a+DNA-Encoded+Small+Molecule+Library&doi=10.1073%2Fpnas.1620645114"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit74R"><div class="casContent"><span class="casTitleNuber">74</span><div class="casTitle"><span class="NLM_cas:atitle">Allosteric "beta-blocker" isolated from a DNA-encoded small molecule library</span></div><div class="casAuthors">Ahn, Seungkirl; Kahsai, Alem W.; Pani, Biswaranjan; Wang, Qin-Ting; Zhao, Shuai; Wall, Alissa L.; Strachan, Ryan T.; Staus, Dean P.; Wingler, Laura M.; Sun, Lillian D.; Sinnaeve, Justine; Choi, Minjung; Cho, Ted; Xu, Thomas T.; Hansen, Gwenn M.; Burnett, Michael B.; Lamerdin, Jane E.; Bassoni, Daniel L.; Gavino, Bryant J.; Husemoen, Gitte; Olsen, Eva K.; Franch, Thomas; Costanzi, Stefano; Chen, Xin; Lefkowitz, Robert J.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">114</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1708-1713</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The β2-adrenergic receptor (β2AR) has been a model system for understanding regulatory mechanisms of G-protein-coupled receptor (GPCR) actions and plays a significant role in cardiovascular and pulmonary diseases.  Because all known β-adrenergic receptor drugs target the orthosteric binding site of the receptor, we set out to isolate allosteric ligands for this receptor by panning DNA-encoded small-mol. libraries comprising 190 million distinct compds. against purified human β2AR.  Here, we report the discovery of a small-mol. neg. allosteric modulator (antagonist), compd. 15 [([4-((2S)-3-(((S)-3-(3-bromophenyl)-1-(methylamino)-1-oxopropan-2-yl)amino)-2-(2-cyclohexyl-2-phenylacetamido)-3-oxopropyl)benzamide], exhibiting a unique chemotype and low micromolar affinity for the β2AR.  Binding of 15 to the receptor cooperatively enhances orthosteric inverse agonist binding while neg. modulating binding of orthosteric agonists.  Studies with a specific antibody that binds to an intracellular region of the β2AR suggest that 15 binds in proximity to the G-protein binding site on the cytosolic surface of the β2AR.  In cell-signaling studies, 15 inhibits cAMP prodn. through the β2AR, but not that mediated by other Gs-coupled receptors.  Compd. 15 also similarly inhibits β-arrestin recruitment to the activated β2AR.  This study presents an allosteric small-mol. ligand for the β2AR and introduces a broadly applicable method for screening DNA-encoded small-mol. libraries against purified GPCR targets.  Importantly, such an approach could facilitate the discovery of GPCR drugs with tailored allosteric effects.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpE5o7Ib4LtSrVg90H21EOLACvtfcHk0lhl1DgGsNZXaw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhs1Kjsbs%253D&md5=c42863673c8b91fcb4a2c42c2b253ebc</span></div><a href="/servlet/linkout?suffix=cit74&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1620645114&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1620645114%26sid%3Dliteratum%253Aachs%26aulast%3DAhn%26aufirst%3DS.%26aulast%3DKahsai%26aufirst%3DA.%2BW.%26aulast%3DPani%26aufirst%3DB.%26aulast%3DWang%26aufirst%3DQ.-T.%26aulast%3DZhao%26aufirst%3DS.%26aulast%3DWall%26aufirst%3DA.%2BL.%26aulast%3DStrachan%26aufirst%3DR.%2BT.%26aulast%3DStaus%26aufirst%3DD.%2BP.%26aulast%3DWingler%26aufirst%3DL.%2BM.%26aulast%3DSun%26aufirst%3DL.%2BD.%26aulast%3DSinnaeve%26aufirst%3DJ.%26aulast%3DChoi%26aufirst%3DM.%26aulast%3DCho%26aufirst%3DT.%26aulast%3DXu%26aufirst%3DT.%2BT.%26aulast%3DHansen%26aufirst%3DG.%2BM.%26aulast%3DBurnett%26aufirst%3DM.%2BB.%26aulast%3DLamerdin%26aufirst%3DJ.%2BE.%26aulast%3DBassoni%26aufirst%3DD.%2BL.%26aulast%3DGavino%26aufirst%3DB.%2BJ.%26aulast%3DHusemoen%26aufirst%3DG.%26aulast%3DOlsen%26aufirst%3DE.%2BK.%26aulast%3DFranch%26aufirst%3DT.%26aulast%3DCostanzi%26aufirst%3DS.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DLefkowitz%26aufirst%3DR.%2BJ.%26atitle%3DAllosteric%2520%25E2%2580%259Cbeta-Blocker%25E2%2580%259D%2520Isolated%2520from%2520a%2520DNA-Encoded%2520Small%2520Molecule%2520Library%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2017%26volume%3D114%26spage%3D1708%26epage%3D1713%26doi%3D10.1073%2Fpnas.1620645114" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref75"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref75'); return false;" data-citation="" class="refNumLink">75</a></strong><div class="NLM_citation" id="cit75"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fernández-Montalván, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berger, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuropka, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koo, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Badock, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weiske, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Puetter, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holton, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stöckigt, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ter Laak, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Centrella, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clark, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dumelin, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sigel, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soutter, H. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Troast, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cuozzo, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keefe, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roche, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rodeschini, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chaikuad, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Díaz-Sáez, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bennett, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huber, K. V. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hübner, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weinmann, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hartung, I. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gorjánácz, M.</span></span> <span> </span><span class="NLM_article-title">Isoform-Selective ATAD2 Chemical Probe with Novel Chemical Structure and Unusual Mode of Action</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">2730</span>– <span class="NLM_lpage">2736</span>, <span class="refDoi"> DOI: 10.1021/acschembio.7b00708</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.7b00708" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhs1KqsrrL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2017&pages=2730-2736&author=A.+E.+Fern%C3%A1ndez-Montalv%C3%A1nauthor=M.+Bergerauthor=B.+Kuropkaauthor=S.+J.+Kooauthor=V.+Badockauthor=J.+Weiskeauthor=V.+Puetterauthor=S.+J.+Holtonauthor=D.+St%C3%B6ckigtauthor=A.+Ter+Laakauthor=P.+A.+Centrellaauthor=M.+A.+Clarkauthor=C.+E.+Dumelinauthor=E.+A.+Sigelauthor=H.+H.+Soutterauthor=D.+M.+Troastauthor=Y.+Zhangauthor=J.+W.+Cuozzoauthor=A.+D.+Keefeauthor=D.+Rocheauthor=V.+Rodeschiniauthor=A.+Chaikuadauthor=L.+D%C3%ADaz-S%C3%A1ezauthor=J.+M.+Bennettauthor=O.+Fedorovauthor=K.+V.+M.+Huberauthor=J.+H%C3%BCbnerauthor=H.+Weinmannauthor=I.+V.+Hartungauthor=M.+Gorj%C3%A1n%C3%A1cz&title=Isoform-Selective+ATAD2+Chemical+Probe+with+Novel+Chemical+Structure+and+Unusual+Mode+of+Action&doi=10.1021%2Facschembio.7b00708"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit75R"><div class="casContent"><span class="casTitleNuber">75</span><div class="casTitle"><span class="NLM_cas:atitle">Isoform-Selective ATAD2 Chemical Probe with Novel Chemical Structure and Unusual Mode of Action</span></div><div class="casAuthors">Fernandez-Montalvan, Amaury E.; Berger, Markus; Kuropka, Benno; Koo, Seong Joo; Badock, Volker; Weiske, Joerg; Puetter, Vera; Holton, Simon J.; Stoeckigt, Detlef; ter Laak, Antonius; Centrella, Paolo A.; Clark, Matthew A.; Dumelin, Christoph E.; Sigel, Eric A.; Soutter, Holly H.; Troast, Dawn M.; Zhang, Ying; Cuozzo, John W.; Keefe, Anthony D.; Roche, Didier; Rodeschini, Vincent; Chaikuad, Apirat; Diaz-Saez, Laura; Bennett, James M.; Fedorov, Oleg; Huber, Kilian V. M.; Huebner, Jan; Weinmann, Hilmar; Hartung, Ingo V.; Gorjanacz, Matyas</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2730-2736</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">ATAD2 (ANCCA) is an epigenetic regulator and transcriptional cofactor, whose overexpression has been linked to the progress of various cancer types.  Here, we report a DNA-encoded library screen leading to the discovery of BAY-850, a potent and isoform selective inhibitor that specifically induces ATAD2 bromodomain dimerization and prevents interactions with acetylated histones in vitro, as well as with chromatin in cells.  These features qualify BAY-850 as a chem. probe to explore ATAD2 biol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrCB3nyG9YvubVg90H21EOLACvtfcHk0lhnYYKgzIcChA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhs1KqsrrL&md5=1f09000a7d7e60e30540fbbae117acfe</span></div><a href="/servlet/linkout?suffix=cit75&amp;dbid=16384&amp;doi=10.1021%2Facschembio.7b00708&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.7b00708%26sid%3Dliteratum%253Aachs%26aulast%3DFern%25C3%25A1ndez-Montalv%25C3%25A1n%26aufirst%3DA.%2BE.%26aulast%3DBerger%26aufirst%3DM.%26aulast%3DKuropka%26aufirst%3DB.%26aulast%3DKoo%26aufirst%3DS.%2BJ.%26aulast%3DBadock%26aufirst%3DV.%26aulast%3DWeiske%26aufirst%3DJ.%26aulast%3DPuetter%26aufirst%3DV.%26aulast%3DHolton%26aufirst%3DS.%2BJ.%26aulast%3DSt%25C3%25B6ckigt%26aufirst%3DD.%26aulast%3DTer%2BLaak%26aufirst%3DA.%26aulast%3DCentrella%26aufirst%3DP.%2BA.%26aulast%3DClark%26aufirst%3DM.%2BA.%26aulast%3DDumelin%26aufirst%3DC.%2BE.%26aulast%3DSigel%26aufirst%3DE.%2BA.%26aulast%3DSoutter%26aufirst%3DH.%2BH.%26aulast%3DTroast%26aufirst%3DD.%2BM.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DCuozzo%26aufirst%3DJ.%2BW.%26aulast%3DKeefe%26aufirst%3DA.%2BD.%26aulast%3DRoche%26aufirst%3DD.%26aulast%3DRodeschini%26aufirst%3DV.%26aulast%3DChaikuad%26aufirst%3DA.%26aulast%3DD%25C3%25ADaz-S%25C3%25A1ez%26aufirst%3DL.%26aulast%3DBennett%26aufirst%3DJ.%2BM.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DHuber%26aufirst%3DK.%2BV.%2BM.%26aulast%3DH%25C3%25BCbner%26aufirst%3DJ.%26aulast%3DWeinmann%26aufirst%3DH.%26aulast%3DHartung%26aufirst%3DI.%2BV.%26aulast%3DGorj%25C3%25A1n%25C3%25A1cz%26aufirst%3DM.%26atitle%3DIsoform-Selective%2520ATAD2%2520Chemical%2520Probe%2520with%2520Novel%2520Chemical%2520Structure%2520and%2520Unusual%2520Mode%2520of%2520Action%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2017%26volume%3D12%26spage%3D2730%26epage%3D2736%26doi%3D10.1021%2Facschembio.7b00708" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref76"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref76'); return false;" data-citation="" class="refNumLink">76</a></strong><div class="NLM_citation" id="cit76"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cuozzo, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Centrella, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gikunju, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Habeshian, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hupp, C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keefe, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sigel, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soutter, H. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomson, H. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clark, M. A.</span></span> <span> </span><span class="NLM_article-title">Discovery of a Potent BTK Inhibitor with a Novel Binding Mode by Using Parallel Selections with a DNA-Encoded Chemical Library</span>. <i>ChemBioChem</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">864</span>– <span class="NLM_lpage">871</span>, <span class="refDoi"> DOI: 10.1002/cbic.201600573</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1002%2Fcbic.201600573" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=28056160" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC2sXit1Chu7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2017&pages=864-871&author=J.+W.+Cuozzoauthor=P.+A.+Centrellaauthor=D.+Gikunjuauthor=S.+Habeshianauthor=C.+D.+Huppauthor=A.+D.+Keefeauthor=E.+A.+Sigelauthor=H.+H.+Soutterauthor=H.+A.+Thomsonauthor=Y.+Zhangauthor=M.+A.+Clark&title=Discovery+of+a+Potent+BTK+Inhibitor+with+a+Novel+Binding+Mode+by+Using+Parallel+Selections+with+a+DNA-Encoded+Chemical+Library&doi=10.1002%2Fcbic.201600573"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit76R"><div class="casContent"><span class="casTitleNuber">76</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a Potent BTK Inhibitor with a Novel Binding Mode by Using Parallel Selections with a DNA-Encoded Chemical Library</span></div><div class="casAuthors">Cuozzo, John W.; Centrella, Paolo A.; Gikunju, Diana; Habeshian, Sevan; Hupp, Christopher D.; Keefe, Anthony D.; Sigel, Eric A.; Soutter, Holly H.; Thomson, Heather A.; Zhang, Ying; Clark, Matthew A.</div><div class="citationInfo"><span class="NLM_cas:title">ChemBioChem</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">864-871</span>CODEN:
                <span class="NLM_cas:coden">CBCHFX</span>;
        ISSN:<span class="NLM_cas:issn">1439-4227</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">We have identified and characterized novel potent inhibitors of Bruton's tyrosine kinase (BTK) from a single DNA-encoded library of over 110 million compds. by using multiple parallel selection conditions, including variation in target concn. and addn. of known binders to provide competition information.  Distinct binding profiles were obsd. by comparing enrichments of library building block combinations under these conditions; one enriched only at high concns. of BTK and was competitive with ATP, and another enriched at both high and low concns. of BTK and was not competitive with ATP.  A compd. representing the latter profile showed low nanomolar potency in biochem. and cellular BTK assays.  Results from kinetic mechanism of action studies were consistent with the selection profiles.  Anal. of the co-crystal structure of the most potent compd. demonstrated a novel binding mode that revealed a new pocket in BTK.  Our results demonstrate that profile-based selection strategies using DNA-encoded libraries form the basis of a new methodol. to rapidly identify small mol. inhibitors with novel binding modes to clin. relevant targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqxaUUXy-jOi7Vg90H21EOLACvtfcHk0lhnYYKgzIcChA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXit1Chu7Y%253D&md5=22423b0496eb68130236292433bc7251</span></div><a href="/servlet/linkout?suffix=cit76&amp;dbid=16384&amp;doi=10.1002%2Fcbic.201600573&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcbic.201600573%26sid%3Dliteratum%253Aachs%26aulast%3DCuozzo%26aufirst%3DJ.%2BW.%26aulast%3DCentrella%26aufirst%3DP.%2BA.%26aulast%3DGikunju%26aufirst%3DD.%26aulast%3DHabeshian%26aufirst%3DS.%26aulast%3DHupp%26aufirst%3DC.%2BD.%26aulast%3DKeefe%26aufirst%3DA.%2BD.%26aulast%3DSigel%26aufirst%3DE.%2BA.%26aulast%3DSoutter%26aufirst%3DH.%2BH.%26aulast%3DThomson%26aufirst%3DH.%2BA.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DClark%26aufirst%3DM.%2BA.%26atitle%3DDiscovery%2520of%2520a%2520Potent%2520BTK%2520Inhibitor%2520with%2520a%2520Novel%2520Binding%2520Mode%2520by%2520Using%2520Parallel%2520Selections%2520with%2520a%2520DNA-Encoded%2520Chemical%2520Library%26jtitle%3DChemBioChem%26date%3D2017%26volume%3D18%26spage%3D864%26epage%3D871%26doi%3D10.1002%2Fcbic.201600573" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref77"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref77'); return false;" data-citation="" class="refNumLink">77</a></strong><div class="NLM_citation" id="cit77"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Petersen, L. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blakskjær, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chaikuad, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Christensen, A. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dietvorst, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holmkvist, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kořínek, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Larsen, L. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pedersen, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Röhm, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sløk, F. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hansen, N. J. V.</span></span> <span> </span><span class="NLM_article-title">Novel p38α MAP Kinase Inhibitors Identified from yoctoReactor DNA-Encoded Small Molecule Library</span>. <i>MedChemComm</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">1332</span>– <span class="NLM_lpage">1339</span>, <span class="refDoi"> DOI: 10.1039/C6MD00241B</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1039%2FC6MD00241B" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtVCis77J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=1332-1339&author=L.+K.+Petersenauthor=P.+Blakskj%C3%A6rauthor=A.+Chaikuadauthor=A.+B.+Christensenauthor=J.+Dietvorstauthor=J.+Holmkvistauthor=S.+Knappauthor=M.+Ko%C5%99%C3%ADnekauthor=L.+K.+Larsenauthor=A.+E.+Pedersenauthor=S.+R%C3%B6hmauthor=F.+A.+Sl%C3%B8kauthor=N.+J.+V.+Hansen&title=Novel+p38%CE%B1+MAP+Kinase+Inhibitors+Identified+from+yoctoReactor+DNA-Encoded+Small+Molecule+Library&doi=10.1039%2FC6MD00241B"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit77R"><div class="casContent"><span class="casTitleNuber">77</span><div class="casTitle"><span class="NLM_cas:atitle">Novel p38α MAP kinase inhibitors identified from yoctoReactor DNA-encoded small molecule library</span></div><div class="casAuthors">Petersen, L. K.; Blakskjaer, P.; Chaikuad, A.; Christensen, A. B.; Dietvorst, J.; Holmkvist, J.; Knapp, S.; Korinek, M.; Larsen, L. K.; Pedersen, A. E.; Rohm, S.; Sloek, F. A.; Hansen, N. J. V.</div><div class="citationInfo"><span class="NLM_cas:title">MedChemComm</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1332-1339</span>CODEN:
                <span class="NLM_cas:coden">MCCEAY</span>;
        ISSN:<span class="NLM_cas:issn">2040-2503</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">A highly specific and potent (7 nM cellular IC50) inhibitor of p38α kinase was identified directly from a 12.6 million membered DNA-encoded small mol. library.  This was achieved using the high fidelity yoctoReactor technol. (yR) for prepg. the DNA-encoded library, and a homogeneous screening technique - the binder trap enrichment technol. (BTE).  Although structurally atypical to other kinase blockers, this inhibitor was found by X-ray crystallog. to interact with the ATP binding site and provide strong distortion of the P-loop.  Remarkably, it assumed an alternative binding mode as it lacks key features of known kinase inhibitors such as typical hinge binding motifs.  Interestingly, the inhibitor bound assuming a canonical type-II ('DFG-out') binding mode by forming hinge hydrogen bonds with the backbone, showed excellent shape complementarity, and formed a no. of specific polar interactions.  Moreover, the crystal structure showed, that although buried in the p38α active site, the original DNA attachment point of the compd. was accessible through a channel created by the distorted P-loop conformation.  This study demonstrates the usability of DNA-encoded library technologies for identifying novel chem. matter with alternative binding modes to provide a good starting point for drug development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpgxosw66vOu7Vg90H21EOLACvtfcHk0lgnVB1a4dZLzQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtVCis77J&md5=8f4feb94165c51dcabc6110ff877dbe4</span></div><a href="/servlet/linkout?suffix=cit77&amp;dbid=16384&amp;doi=10.1039%2FC6MD00241B&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC6MD00241B%26sid%3Dliteratum%253Aachs%26aulast%3DPetersen%26aufirst%3DL.%2BK.%26aulast%3DBlakskj%25C3%25A6r%26aufirst%3DP.%26aulast%3DChaikuad%26aufirst%3DA.%26aulast%3DChristensen%26aufirst%3DA.%2BB.%26aulast%3DDietvorst%26aufirst%3DJ.%26aulast%3DHolmkvist%26aufirst%3DJ.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DKo%25C5%2599%25C3%25ADnek%26aufirst%3DM.%26aulast%3DLarsen%26aufirst%3DL.%2BK.%26aulast%3DPedersen%26aufirst%3DA.%2BE.%26aulast%3DR%25C3%25B6hm%26aufirst%3DS.%26aulast%3DSl%25C3%25B8k%26aufirst%3DF.%2BA.%26aulast%3DHansen%26aufirst%3DN.%2BJ.%2BV.%26atitle%3DNovel%2520p38%25CE%25B1%2520MAP%2520Kinase%2520Inhibitors%2520Identified%2520from%2520yoctoReactor%2520DNA-Encoded%2520Small%2520Molecule%2520Library%26jtitle%3DMedChemComm%26date%3D2016%26volume%3D7%26spage%3D1332%26epage%3D1339%26doi%3D10.1039%2FC6MD00241B" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref78"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref78'); return false;" data-citation="" class="refNumLink">78</a></strong><div class="NLM_citation" id="cit78"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McMillan, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adler, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Auld, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baldwin, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blasko, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Browne, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chelsky, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davey, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dolle, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eagen, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Erickson, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feldman, R. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glaser, C. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mallari, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morrissey, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohlmeyer, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pan, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parkinson, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phillips, G. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Polokoff, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sigal, N. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vergona, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whitlow, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Devlin, J. J.</span></span> <span> </span><span class="NLM_article-title">Allosteric Inhibitors of Inducible Nitric Oxide Synthase Dimerization Discovered via Combinatorial Chemistry</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>97</i></span>,  <span class="NLM_fpage">1506</span>– <span class="NLM_lpage">1511</span>, <span class="refDoi"> DOI: 10.1073/pnas.97.4.1506</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1073%2Fpnas.97.4.1506" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=97&publication_year=2000&pages=1506-1511&author=K.+McMillanauthor=M.+Adlerauthor=D.+S.+Auldauthor=J.+J.+Baldwinauthor=E.+Blaskoauthor=L.+J.+Browneauthor=D.+Chelskyauthor=D.+Daveyauthor=R.+E.+Dolleauthor=K.+A.+Eagenauthor=S.+Ericksonauthor=R.+I.+Feldmanauthor=C.+B.+Glaserauthor=C.+Mallariauthor=M.+M.+Morrisseyauthor=M.+H.+Ohlmeyerauthor=G.+Panauthor=J.+F.+Parkinsonauthor=G.+B.+Phillipsauthor=M.+A.+Polokoffauthor=N.+H.+Sigalauthor=R.+Vergonaauthor=M.+Whitlowauthor=T.+A.+Youngauthor=J.+J.+Devlin&title=Allosteric+Inhibitors+of+Inducible+Nitric+Oxide+Synthase+Dimerization+Discovered+via+Combinatorial+Chemistry&doi=10.1073%2Fpnas.97.4.1506"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit78&amp;dbid=16384&amp;doi=10.1073%2Fpnas.97.4.1506&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.97.4.1506%26sid%3Dliteratum%253Aachs%26aulast%3DMcMillan%26aufirst%3DK.%26aulast%3DAdler%26aufirst%3DM.%26aulast%3DAuld%26aufirst%3DD.%2BS.%26aulast%3DBaldwin%26aufirst%3DJ.%2BJ.%26aulast%3DBlasko%26aufirst%3DE.%26aulast%3DBrowne%26aufirst%3DL.%2BJ.%26aulast%3DChelsky%26aufirst%3DD.%26aulast%3DDavey%26aufirst%3DD.%26aulast%3DDolle%26aufirst%3DR.%2BE.%26aulast%3DEagen%26aufirst%3DK.%2BA.%26aulast%3DErickson%26aufirst%3DS.%26aulast%3DFeldman%26aufirst%3DR.%2BI.%26aulast%3DGlaser%26aufirst%3DC.%2BB.%26aulast%3DMallari%26aufirst%3DC.%26aulast%3DMorrissey%26aufirst%3DM.%2BM.%26aulast%3DOhlmeyer%26aufirst%3DM.%2BH.%26aulast%3DPan%26aufirst%3DG.%26aulast%3DParkinson%26aufirst%3DJ.%2BF.%26aulast%3DPhillips%26aufirst%3DG.%2BB.%26aulast%3DPolokoff%26aufirst%3DM.%2BA.%26aulast%3DSigal%26aufirst%3DN.%2BH.%26aulast%3DVergona%26aufirst%3DR.%26aulast%3DWhitlow%26aufirst%3DM.%26aulast%3DYoung%26aufirst%3DT.%2BA.%26aulast%3DDevlin%26aufirst%3DJ.%2BJ.%26atitle%3DAllosteric%2520Inhibitors%2520of%2520Inducible%2520Nitric%2520Oxide%2520Synthase%2520Dimerization%2520Discovered%2520via%2520Combinatorial%2520Chemistry%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2000%26volume%3D97%26spage%3D1506%26epage%3D1511%26doi%3D10.1073%2Fpnas.97.4.1506" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref79"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref79'); return false;" data-citation="" class="refNumLink">79</a></strong><div class="NLM_citation" id="cit79"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zambaldo, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daguer, J.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saarbach, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barluenga, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winssinger, N.</span></span> <span> </span><span class="NLM_article-title">Screening for Covalent Inhibitors Using DNA-Display of Small Molecule Libraries Functionalized with Cysteine Reactive Moieties</span>. <i>MedChemComm</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">1340</span>– <span class="NLM_lpage">1351</span>, <span class="refDoi"> DOI: 10.1039/C6MD00242K</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1039%2FC6MD00242K" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=1340-1351&author=C.+Zambaldoauthor=J.-P.+Daguerauthor=J.+Saarbachauthor=S.+Barluengaauthor=N.+Winssinger&title=Screening+for+Covalent+Inhibitors+Using+DNA-Display+of+Small+Molecule+Libraries+Functionalized+with+Cysteine+Reactive+Moieties&doi=10.1039%2FC6MD00242K"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit79&amp;dbid=16384&amp;doi=10.1039%2FC6MD00242K&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC6MD00242K%26sid%3Dliteratum%253Aachs%26aulast%3DZambaldo%26aufirst%3DC.%26aulast%3DDaguer%26aufirst%3DJ.-P.%26aulast%3DSaarbach%26aufirst%3DJ.%26aulast%3DBarluenga%26aufirst%3DS.%26aulast%3DWinssinger%26aufirst%3DN.%26atitle%3DScreening%2520for%2520Covalent%2520Inhibitors%2520Using%2520DNA-Display%2520of%2520Small%2520Molecule%2520Libraries%2520Functionalized%2520with%2520Cysteine%2520Reactive%2520Moieties%26jtitle%3DMedChemComm%26date%3D2016%26volume%3D7%26spage%3D1340%26epage%3D1351%26doi%3D10.1039%2FC6MD00242K" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref80"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref80'); return false;" data-citation="" class="refNumLink">80</a></strong><div class="NLM_citation" id="cit80"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Salamon, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klika Škopić, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jung, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bugain, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brunschweiger, A.</span></span> <span> </span><span class="NLM_article-title">Chemical Biology Probes from Advanced DNA-Encoded Libraries</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">296</span>– <span class="NLM_lpage">307</span>, <span class="refDoi"> DOI: 10.1021/acschembio.5b00981</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.5b00981" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC28XitVyhtbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2016&pages=296-307&author=H.+Salamonauthor=M.+Klika+%C5%A0kopi%C4%87author=K.+Jungauthor=O.+Bugainauthor=A.+Brunschweiger&title=Chemical+Biology+Probes+from+Advanced+DNA-Encoded+Libraries&doi=10.1021%2Facschembio.5b00981"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit80R"><div class="casContent"><span class="casTitleNuber">80</span><div class="casTitle"><span class="NLM_cas:atitle">Chemical Biology Probes from Advanced DNA-encoded Libraries</span></div><div class="casAuthors">Salamon, Hazem; Klika Skopic, Mateja; Jung, Kathrin; Bugain, Olivia; Brunschweiger, Andreas</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">296-307</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The identification of bioactive compds. is a crucial step toward development of probes for chem. biol. studies.  Screening of DNA-encoded small mol. libraries (DELs) has emerged as a validated technol. to interrogate vast chem. space.  DELs consist of chimeric mols. composed of a low-mol. wt. compd. that is conjugated to a DNA identifier tag.  They are screened as pooled libraries using selection to identify "hits." Screening of DELs has identified numerous bioactive compds.  Some of these mols. were instrumental in gaining a deeper understanding of biol. systems.  One of the main challenges in the field is the development of synthesis methodol. for DELs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpoCCmTdlAe37Vg90H21EOLACvtfcHk0lgnVB1a4dZLzQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitVyhtbk%253D&md5=a506c420e03fb229c928459b05c23406</span></div><a href="/servlet/linkout?suffix=cit80&amp;dbid=16384&amp;doi=10.1021%2Facschembio.5b00981&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.5b00981%26sid%3Dliteratum%253Aachs%26aulast%3DSalamon%26aufirst%3DH.%26aulast%3DKlika%2B%25C5%25A0kopi%25C4%2587%26aufirst%3DM.%26aulast%3DJung%26aufirst%3DK.%26aulast%3DBugain%26aufirst%3DO.%26aulast%3DBrunschweiger%26aufirst%3DA.%26atitle%3DChemical%2520Biology%2520Probes%2520from%2520Advanced%2520DNA-Encoded%2520Libraries%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2016%26volume%3D11%26spage%3D296%26epage%3D307%26doi%3D10.1021%2Facschembio.5b00981" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref81"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref81'); return false;" data-citation="" class="refNumLink">81</a></strong><div class="NLM_citation" id="cit81"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Villar, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beglov, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chennamadhavuni, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Porco, J. A.,  Jr</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kozakov, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vajda, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whitty, A.</span></span> <span> </span><span class="NLM_article-title">How Proteins Bind Macrocycles</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">723</span>– <span class="NLM_lpage">731</span>, <span class="refDoi"> DOI: 10.1038/nchembio.1584</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1038%2Fnchembio.1584" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=25038790" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtFygs7jF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2014&pages=723-731&author=E.+A.+Villarauthor=D.+Beglovauthor=S.+Chennamadhavuniauthor=J.+A.+Porcoauthor=D.+Kozakovauthor=S.+Vajdaauthor=A.+Whitty&title=How+Proteins+Bind+Macrocycles&doi=10.1038%2Fnchembio.1584"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit81R"><div class="casContent"><span class="casTitleNuber">81</span><div class="casTitle"><span class="NLM_cas:atitle">How proteins bind macrocycles</span></div><div class="casAuthors">Villar, Elizabeth A.; Beglov, Dmitri; Chennamadhavuni, Spandan; Porco, John A. Jr; Kozakov, Dima; Vajda, Sandor; Whitty, Adrian</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">723-731</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The potential utility of synthetic macrocycles (MCs) as drugs, particularly against low-druggability targets such as protein-protein interactions, has been widely discussed.  There is little information, however, to guide the design of MCs for good target protein-binding activity or bioavailability.  To address this knowledge gap, we analyze the binding modes of a representative set of MC-protein complexes.  The results, combined with consideration of the physicochem. properties of approved macrocyclic drugs, allow us to propose specific guidelines for the design of synthetic MC libraries with structural and physicochem. features likely to favor strong binding to protein targets as well as good bioavailability.  We addnl. provide evidence that large, natural product-derived MCs can bind targets that are not druggable by conventional, drug-like compds., supporting the notion that natural product-inspired synthetic MCs can expand the no. of proteins that are druggable by synthetic small mols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqa3yuNCHU5d7Vg90H21EOLACvtfcHk0lj344Q5g5rx9w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtFygs7jF&md5=0d41fba9267abb3a3d0065487937075a</span></div><a href="/servlet/linkout?suffix=cit81&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1584&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1584%26sid%3Dliteratum%253Aachs%26aulast%3DVillar%26aufirst%3DE.%2BA.%26aulast%3DBeglov%26aufirst%3DD.%26aulast%3DChennamadhavuni%26aufirst%3DS.%26aulast%3DPorco%26aufirst%3DJ.%2BA.%26aulast%3DKozakov%26aufirst%3DD.%26aulast%3DVajda%26aufirst%3DS.%26aulast%3DWhitty%26aufirst%3DA.%26atitle%3DHow%2520Proteins%2520Bind%2520Macrocycles%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2014%26volume%3D10%26spage%3D723%26epage%3D731%26doi%3D10.1038%2Fnchembio.1584" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref82"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref82'); return false;" data-citation="" class="refNumLink">82</a></strong><div class="NLM_citation" id="cit82"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Giordanetto, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kihlberg, J.</span></span> <span> </span><span class="NLM_article-title">Macrocyclic Drugs and Clinical Candidates: What Can Medicinal Chemists Learn from Their Properties?</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">278</span>– <span class="NLM_lpage">295</span>, <span class="refDoi"> DOI: 10.1021/jm400887j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm400887j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtlGnurjI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=278-295&author=F.+Giordanettoauthor=J.+Kihlberg&title=Macrocyclic+Drugs+and+Clinical+Candidates%3A+What+Can+Medicinal+Chemists+Learn+from+Their+Properties%3F&doi=10.1021%2Fjm400887j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit82R"><div class="casContent"><span class="casTitleNuber">82</span><div class="casTitle"><span class="NLM_cas:atitle">Macrocyclic Drugs and Clinical Candidates: What Can Medicinal Chemists Learn from Their Properties?</span></div><div class="casAuthors">Giordanetto, Fabrizio; Kihlberg, Jan</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">278-295</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Macrocycles are ideal in efforts to tackle difficult targets, but our understanding of what makes them cell permeable and orally bioavailable is limited.  Anal. of approx. 100 macrocyclic drugs and clin. candidates revealed that macrocycles are predominantly used for infectious disease and in oncol. and that most belong to the macrolide or cyclic peptide class.  A significant no. (N = 34) of these macrocycles are administered orally, revealing that oral bioavailability can be obtained at mol. wts. up to and above 1 kDa and polar surface areas ranging toward 250 Å2.  Moreover, insight from a group of de novo designed oral macrocycles in clin. studies and understanding of how cyclosporin A and model cyclic hexapeptides cross cell membranes may unlock wider opportunities in drug discovery.  However, the no. of oral macrocycles is still low and it remains to be seen if they are outliers or if macrocycles will open up novel oral druggable space.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpaeM2DXwvmiLVg90H21EOLACvtfcHk0lj344Q5g5rx9w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtlGnurjI&md5=a3e1743bd5ba34f73d8e78f8cb1bee5f</span></div><a href="/servlet/linkout?suffix=cit82&amp;dbid=16384&amp;doi=10.1021%2Fjm400887j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm400887j%26sid%3Dliteratum%253Aachs%26aulast%3DGiordanetto%26aufirst%3DF.%26aulast%3DKihlberg%26aufirst%3DJ.%26atitle%3DMacrocyclic%2520Drugs%2520and%2520Clinical%2520Candidates%253A%2520What%2520Can%2520Medicinal%2520Chemists%2520Learn%2520from%2520Their%2520Properties%253F%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D278%26epage%3D295%26doi%3D10.1021%2Fjm400887j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref83"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref83'); return false;" data-citation="" class="refNumLink">83</a></strong><div class="NLM_citation" id="cit83"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Whitty, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Viarengo, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhong, M.</span></span> <span> </span><span class="NLM_article-title">Progress towards the Broad Use of Non-Peptide Synthetic Macrocycles in Drug Discovery</span>. <i>Org. Biomol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">7729</span>– <span class="NLM_lpage">7735</span>, <span class="refDoi"> DOI: 10.1039/C7OB00056A</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref83/cit83&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1039%2FC7OB00056A" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref83/cit83&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=28876025" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref83/cit83&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsVKgsLrO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2017&pages=7729-7735&author=A.+Whittyauthor=L.+A.+Viarengoauthor=M.+Zhong&title=Progress+towards+the+Broad+Use+of+Non-Peptide+Synthetic+Macrocycles+in+Drug+Discovery&doi=10.1039%2FC7OB00056A"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit83R"><div class="casContent"><span class="casTitleNuber">83</span><div class="casTitle"><span class="NLM_cas:atitle">Progress towards the broad use of non-peptide synthetic macrocycles in drug discovery</span></div><div class="casAuthors">Whitty, Adrian; Viarengo, Lauren A.; Zhong, Mengqi</div><div class="citationInfo"><span class="NLM_cas:title">Organic & Biomolecular Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">37</span>),
    <span class="NLM_cas:pages">7729-7735</span>CODEN:
                <span class="NLM_cas:coden">OBCRAK</span>;
        ISSN:<span class="NLM_cas:issn">1477-0520</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">A review.  The authors discuss progress towards addressing three key questions pertaining to the design of screening libraries of synthetic nonpeptidic macrocycles (MCs) for drug discovery: What structural and physicochem. properties of MCs maximize the likelihood of achieving strong and specific binding to protein targets.  What features render a protein target suitable for binding MCs, and can this information be used to identify suitable targets for inhibition by MCs.  What properties of synthetic MCs confer good pharmaceutical properties, and particularly good aq. soly. coupled with passive membrane permeability.  The authors addnl. discuss how the criteria that define a meaningful MC screening hit are linked to the size of the screening library and the synthetic methodol. employed in its prepn.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpeNdUQVdF6FrVg90H21EOLACvtfcHk0lj344Q5g5rx9w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsVKgsLrO&md5=451ddddc4457d5a2ee84b384aa15508e</span></div><a href="/servlet/linkout?suffix=cit83&amp;dbid=16384&amp;doi=10.1039%2FC7OB00056A&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC7OB00056A%26sid%3Dliteratum%253Aachs%26aulast%3DWhitty%26aufirst%3DA.%26aulast%3DViarengo%26aufirst%3DL.%2BA.%26aulast%3DZhong%26aufirst%3DM.%26atitle%3DProgress%2520towards%2520the%2520Broad%2520Use%2520of%2520Non-Peptide%2520Synthetic%2520Macrocycles%2520in%2520Drug%2520Discovery%26jtitle%3DOrg.%2520Biomol.%2520Chem.%26date%3D2017%26volume%3D15%26spage%3D7729%26epage%3D7735%26doi%3D10.1039%2FC7OB00056A" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref84"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref84'); return false;" data-citation="" class="refNumLink">84</a></strong><div class="NLM_citation" id="cit84"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Seigal, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Connors, W. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fraley, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Borzilleri, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carter, P. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Emanuel, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fargnoli, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lei, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Naglich, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pokross, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Posy, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Surti, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Talbott, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Terrett, N. K.</span></span> <span> </span><span class="NLM_article-title">The Discovery of Macrocyclic XIAP Antagonists from a DNA-Programmed Chemistry Library, and Their Optimization to Give Lead Compounds with in Vivo Antitumor Activity</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">2855</span>– <span class="NLM_lpage">2861</span>, <span class="refDoi"> DOI: 10.1021/jm501892g</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm501892g" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref84/cit84&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC2MXivF2qu78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=2855-2861&author=B.+A.+Seigalauthor=W.+H.+Connorsauthor=A.+Fraleyauthor=R.+M.+Borzilleriauthor=P.+H.+Carterauthor=S.+L.+Emanuelauthor=J.+Fargnoliauthor=K.+Kimauthor=M.+Leiauthor=J.+G.+Naglichauthor=M.+E.+Pokrossauthor=S.+L.+Posyauthor=H.+Shenauthor=N.+Surtiauthor=R.+Talbottauthor=Y.+Zhangauthor=N.+K.+Terrett&title=The+Discovery+of+Macrocyclic+XIAP+Antagonists+from+a+DNA-Programmed+Chemistry+Library%2C+and+Their+Optimization+to+Give+Lead+Compounds+with+in+Vivo+Antitumor+Activity&doi=10.1021%2Fjm501892g"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit84R"><div class="casContent"><span class="casTitleNuber">84</span><div class="casTitle"><span class="NLM_cas:atitle">The Discovery of Macrocyclic XIAP Antagonists from a DNA-Programmed Chemistry Library, and Their Optimization To Give Lead Compounds with in Vivo Antitumor Activity</span></div><div class="casAuthors">Seigal, Benjamin A.; Connors, William H.; Fraley, Andrew; Borzilleri, Robert M.; Carter, Percy H.; Emanuel, Stuart L.; Fargnoli, Joseph; Kim, Kyoung; Lei, Ming; Naglich, Joseph G.; Pokross, Matthew E.; Posy, Shana L.; Shen, Henry; Surti, Neha; Talbott, Randy; Zhang, Yong; Terrett, Nicholas K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2855-2861</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Affinity selection screening of macrocycle libraries derived from DNA-programmed chem. identified XIAP BIR2 and BIR3 domain inhibitors that displace bound pro-apoptotic caspases.  X-ray cocrystal structures of key compds. with XIAP BIR2 suggested potency-enhancing structural modifications.  Optimization of dimeric macrocycles with similar affinity for both domains were potent pro-apoptotic agents in cancer cell lines and efficacious in shrinking tumors in a mouse xenograft model.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrlB6i1KOFJjrVg90H21EOLACvtfcHk0lhDH_3HYgCuZw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXivF2qu78%253D&md5=9bd53cc3ed014796248e8048cecea0d1</span></div><a href="/servlet/linkout?suffix=cit84&amp;dbid=16384&amp;doi=10.1021%2Fjm501892g&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm501892g%26sid%3Dliteratum%253Aachs%26aulast%3DSeigal%26aufirst%3DB.%2BA.%26aulast%3DConnors%26aufirst%3DW.%2BH.%26aulast%3DFraley%26aufirst%3DA.%26aulast%3DBorzilleri%26aufirst%3DR.%2BM.%26aulast%3DCarter%26aufirst%3DP.%2BH.%26aulast%3DEmanuel%26aufirst%3DS.%2BL.%26aulast%3DFargnoli%26aufirst%3DJ.%26aulast%3DKim%26aufirst%3DK.%26aulast%3DLei%26aufirst%3DM.%26aulast%3DNaglich%26aufirst%3DJ.%2BG.%26aulast%3DPokross%26aufirst%3DM.%2BE.%26aulast%3DPosy%26aufirst%3DS.%2BL.%26aulast%3DShen%26aufirst%3DH.%26aulast%3DSurti%26aufirst%3DN.%26aulast%3DTalbott%26aufirst%3DR.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DTerrett%26aufirst%3DN.%2BK.%26atitle%3DThe%2520Discovery%2520of%2520Macrocyclic%2520XIAP%2520Antagonists%2520from%2520a%2520DNA-Programmed%2520Chemistry%2520Library%252C%2520and%2520Their%2520Optimization%2520to%2520Give%2520Lead%2520Compounds%2520with%2520in%2520Vivo%2520Antitumor%2520Activity%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D2855%26epage%3D2861%26doi%3D10.1021%2Fjm501892g" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref85"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref85'); return false;" data-citation="" class="refNumLink">85</a></strong><div class="NLM_citation" id="cit85"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Groves, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dömling, A.</span></span> <span> </span><span class="NLM_article-title">Artificial Macrocycles as IL-17A/IL-17RA Antagonists</span>. <i>MedChemComm</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">22</span>– <span class="NLM_lpage">26</span>, <span class="refDoi"> DOI: 10.1039/C7MD00464H</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref85/cit85&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1039%2FC7MD00464H" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref85/cit85&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=29750108" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref85/cit85&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhslagtL3L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2018&pages=22-26&author=W.+Wangauthor=M.+R.+Grovesauthor=A.+D%C3%B6mling&title=Artificial+Macrocycles+as+IL-17A%2FIL-17RA+Antagonists&doi=10.1039%2FC7MD00464H"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit85R"><div class="casContent"><span class="casTitleNuber">85</span><div class="casTitle"><span class="NLM_cas:atitle">Artificial macrocycles as IL-17A/IL-17RA antagonists</span></div><div class="casAuthors">Wang, Wenjia; Groves, Matthew R.; Doemling, Alexander</div><div class="citationInfo"><span class="NLM_cas:title">MedChemComm</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">22-26</span>CODEN:
                <span class="NLM_cas:coden">MCCEAY</span>;
        ISSN:<span class="NLM_cas:issn">2040-2503</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">A review.  Interleukin 17(A) is a pro-inflammatory cytokine involved in several auto-immune and inflammatory diseases.  Current antagonists against IL-17(A) or its receptor (IL-17RA) that show efficacy in clin. trials are monoclonal antibodies (mAbs).  However, recently designed artificial macrocycles are potent IL-17A/IL-17RA antagonists.  Based on co-crystal structures, a better understanding of the biol. activity and SAR of the macrocycles has been gained, demonstrating that they can compete with mAbs for difficult targets such as PPIs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpMncYtypRDh7Vg90H21EOLACvtfcHk0lhDH_3HYgCuZw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhslagtL3L&md5=62b3ef7b447d2367e337dad6e5a3a204</span></div><a href="/servlet/linkout?suffix=cit85&amp;dbid=16384&amp;doi=10.1039%2FC7MD00464H&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC7MD00464H%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DW.%26aulast%3DGroves%26aufirst%3DM.%2BR.%26aulast%3DD%25C3%25B6mling%26aufirst%3DA.%26atitle%3DArtificial%2520Macrocycles%2520as%2520IL-17A%252FIL-17RA%2520Antagonists%26jtitle%3DMedChemComm%26date%3D2018%26volume%3D9%26spage%3D22%26epage%3D26%26doi%3D10.1039%2FC7MD00464H" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref86"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref86'); return false;" data-citation="" class="refNumLink">86</a></strong><div class="NLM_citation" id="cit86"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Maianti, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McFedries, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Foda, Z. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kleiner, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Du, X. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leissring, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, W.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Charron, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seeliger, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saghatelian, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, D. R.</span></span> <span> </span><span class="NLM_article-title">Anti-Diabetic Activity of Insulin-Degrading Enzyme Inhibitors Mediated by Multiple Hormones</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>511</i></span>,  <span class="NLM_fpage">94</span>– <span class="NLM_lpage">98</span>, <span class="refDoi"> DOI: 10.1038/nature13297</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref86/cit86&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1038%2Fnature13297" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref86/cit86&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=24847884" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref86/cit86&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC2cXpslGitbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=511&publication_year=2014&pages=94-98&author=J.+P.+Maiantiauthor=A.+McFedriesauthor=Z.+H.+Fodaauthor=R.+E.+Kleinerauthor=X.+Q.+Duauthor=M.+A.+Leissringauthor=W.-J.+Tangauthor=M.+J.+Charronauthor=M.+A.+Seeligerauthor=A.+Saghatelianauthor=D.+R.+Liu&title=Anti-Diabetic+Activity+of+Insulin-Degrading+Enzyme+Inhibitors+Mediated+by+Multiple+Hormones&doi=10.1038%2Fnature13297"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit86R"><div class="casContent"><span class="casTitleNuber">86</span><div class="casTitle"><span class="NLM_cas:atitle">Anti-diabetic activity of insulin-degrading enzyme inhibitors mediated by multiple hormones</span></div><div class="casAuthors">Maianti, Juan Pablo; McFedries, Amanda; Foda, Zachariah H.; Kleiner, Ralph E.; Du, Xiu Quan; Leissring, Malcolm A.; Tang, Wei-Jen; Charron, Maureen J.; Seeliger, Markus A.; Saghatelian, Alan; Liu, David R.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">511</span>
        (<span class="NLM_cas:issue">7507</span>),
    <span class="NLM_cas:pages">94-98</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Despite decades of speculation that inhibiting endogenous insulin degrdn. might treat type-2 diabetes, and the identification of IDE (insulin-degrading enzyme) as a diabetes susceptibility gene, the relationship between the activity of the zinc metalloprotein IDE and glucose homeostasis remains unclear.  Although Ide-/- mice have elevated insulin levels, they exhibit impaired, rather than improved, glucose tolerance that may arise from compensatory insulin signaling dysfunction.  IDE inhibitors that are active in vivo are therefore needed to elucidate IDE's physiol. roles and to det. its potential to serve as a target for the treatment of diabetes.  Here we report the discovery of a physiol. active IDE inhibitor identified from a DNA-templated macrocycle library.  An X-ray structure of the macrocycle bound to IDE reveals that it engages a binding pocket away from the catalytic site, which explains its remarkable selectivity.  Treatment of lean and obese mice with this inhibitor shows that IDE regulates the abundance and signaling of glucagon and amylin, in addn. to that of insulin.  Under physiol. conditions that augment insulin and amylin levels, such as oral glucose administration, acute IDE inhibition leads to substantially improved glucose tolerance and slower gastric emptying.  These findings demonstrate the feasibility of modulating IDE activity as a new therapeutic strategy to treat type-2 diabetes and expand our understanding of the roles of IDE in glucose and hormone regulation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqaRhLWpG6WfbVg90H21EOLACvtfcHk0lhDH_3HYgCuZw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXpslGitbg%253D&md5=803357a4dc422275db54d15aa7518f89</span></div><a href="/servlet/linkout?suffix=cit86&amp;dbid=16384&amp;doi=10.1038%2Fnature13297&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature13297%26sid%3Dliteratum%253Aachs%26aulast%3DMaianti%26aufirst%3DJ.%2BP.%26aulast%3DMcFedries%26aufirst%3DA.%26aulast%3DFoda%26aufirst%3DZ.%2BH.%26aulast%3DKleiner%26aufirst%3DR.%2BE.%26aulast%3DDu%26aufirst%3DX.%2BQ.%26aulast%3DLeissring%26aufirst%3DM.%2BA.%26aulast%3DTang%26aufirst%3DW.-J.%26aulast%3DCharron%26aufirst%3DM.%2BJ.%26aulast%3DSeeliger%26aufirst%3DM.%2BA.%26aulast%3DSaghatelian%26aufirst%3DA.%26aulast%3DLiu%26aufirst%3DD.%2BR.%26atitle%3DAnti-Diabetic%2520Activity%2520of%2520Insulin-Degrading%2520Enzyme%2520Inhibitors%2520Mediated%2520by%2520Multiple%2520Hormones%26jtitle%3DNature%26date%3D2014%26volume%3D511%26spage%3D94%26epage%3D98%26doi%3D10.1038%2Fnature13297" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref87"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref87'); return false;" data-citation="" class="refNumLink">87</a></strong><div class="NLM_citation" id="cit87"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Usanov, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, A. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maianti, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, D. R.</span></span> <span> </span><span class="NLM_article-title">Second-Generation DNA-Templated Macrocycle Libraries for the Discovery of Bioactive Small Molecules</span>. <i>Nat. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>, <span class="refDoi"> DOI: 10.1038/s41557-018-0033-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref87/cit87&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1038%2Fs41557-018-0033-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref87/cit87&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=29610462" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2018&author=D.+L.+Usanovauthor=A.+I.+Chanauthor=J.+P.+Maiantiauthor=D.+R.+Liu&title=Second-Generation+DNA-Templated+Macrocycle+Libraries+for+the+Discovery+of+Bioactive+Small+Molecules&doi=10.1038%2Fs41557-018-0033-8"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit87&amp;dbid=16384&amp;doi=10.1038%2Fs41557-018-0033-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41557-018-0033-8%26sid%3Dliteratum%253Aachs%26aulast%3DUsanov%26aufirst%3DD.%2BL.%26aulast%3DChan%26aufirst%3DA.%2BI.%26aulast%3DMaianti%26aufirst%3DJ.%2BP.%26aulast%3DLiu%26aufirst%3DD.%2BR.%26atitle%3DSecond-Generation%2520DNA-Templated%2520Macrocycle%2520Libraries%2520for%2520the%2520Discovery%2520of%2520Bioactive%2520Small%2520Molecules%26jtitle%3DNat.%2520Chem.%26date%3D2018%26doi%3D10.1038%2Fs41557-018-0033-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref88"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref88'); return false;" data-citation="" class="refNumLink">88</a></strong><div class="NLM_citation" id="cit88"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Saupe, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kunz, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haustedt, L. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jakupovic, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mang, C.</span></span> <span> </span><span class="NLM_article-title">MacroEvoLution: A New Method for the Rapid Generation of Novel Scaffold-Diverse Macrocyclic Libraries</span>. <i>Chem. - Eur. J.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">11784</span>– <span class="NLM_lpage">11791</span>, <span class="refDoi"> DOI: 10.1002/chem.201703209</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref88/cit88&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1002%2Fchem.201703209" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref88/cit88&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=28715083" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref88/cit88&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC2sXht12hs7jO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2017&pages=11784-11791&author=J.+Saupeauthor=O.+Kunzauthor=L.+O.+Haustedtauthor=S.+Jakupovicauthor=C.+Mang&title=MacroEvoLution%3A+A+New+Method+for+the+Rapid+Generation+of+Novel+Scaffold-Diverse+Macrocyclic+Libraries&doi=10.1002%2Fchem.201703209"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit88R"><div class="casContent"><span class="casTitleNuber">88</span><div class="casTitle"><span class="NLM_cas:atitle">MacroEvoLution: A new method for the rapid generation of novel scaffold-diverse macrocyclic libraries</span></div><div class="casAuthors">Saupe, Joern; Kunz, Oliver; Haustedt, Lars Ole; Jakupovic, Sven; Mang, Christian</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry - A European Journal</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">49</span>),
    <span class="NLM_cas:pages">11784-11791</span>CODEN:
                <span class="NLM_cas:coden">CEUJED</span>;
        ISSN:<span class="NLM_cas:issn">0947-6539</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Macrocycles are a structural class bearing great promise for future challenges in medicinal chem.  Nevertheless, there are few flexible approaches for the rapid generation of structurally diverse macrocyclic compd. collections.  Here, an efficient method for the generation of novel macrocyclic peptide-based scaffolds is reported.  The process, named here as "MacroEvoLution", is based on a cyclization screening approach that gives reliable access to novel macrocyclic architectures.  Classification of building blocks into specific pools ensures that scaffolds with orthogonally addressable functionalities are generated, which can easily be used for the generation of structurally diverse compd. libraries.  The method grants rapid access to novel scaffolds with scalable synthesis (multi gram scale) and the introduction of further diversity at a late stage.  Despite being developed for peptidic systems, the approach can easily be extended for the synthesis of systems with a decreased peptidic character.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrAkGYsnFt95rVg90H21EOLACvtfcHk0lhDzNh1uY_uOg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXht12hs7jO&md5=c266860b10199fde8140d50ec18ec9b6</span></div><a href="/servlet/linkout?suffix=cit88&amp;dbid=16384&amp;doi=10.1002%2Fchem.201703209&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fchem.201703209%26sid%3Dliteratum%253Aachs%26aulast%3DSaupe%26aufirst%3DJ.%26aulast%3DKunz%26aufirst%3DO.%26aulast%3DHaustedt%26aufirst%3DL.%2BO.%26aulast%3DJakupovic%26aufirst%3DS.%26aulast%3DMang%26aufirst%3DC.%26atitle%3DMacroEvoLution%253A%2520A%2520New%2520Method%2520for%2520the%2520Rapid%2520Generation%2520of%2520Novel%2520Scaffold-Diverse%2520Macrocyclic%2520Libraries%26jtitle%3DChem.%2520-%2520Eur.%2520J.%26date%3D2017%26volume%3D23%26spage%3D11784%26epage%3D11791%26doi%3D10.1002%2Fchem.201703209" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref89"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref89'); return false;" data-citation="" class="refNumLink">89</a></strong><div class="NLM_citation" id="cit89"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ermert, P.</span></span> <span> </span><span class="NLM_article-title">Design, Properties and Recent Application of Macrocycles in Medicinal Chemistry</span>. <i>Chimia</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>71</i></span>,  <span class="NLM_fpage">678</span>– <span class="NLM_lpage">702</span>, <span class="refDoi"> DOI: 10.2533/chimia.2017.678</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref89/cit89&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.2533%2Fchimia.2017.678" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref89/cit89&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=29070413" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref89/cit89&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhsFeitbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=71&publication_year=2017&pages=678-702&author=P.+Ermert&title=Design%2C+Properties+and+Recent+Application+of+Macrocycles+in+Medicinal+Chemistry&doi=10.2533%2Fchimia.2017.678"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit89R"><div class="casContent"><span class="casTitleNuber">89</span><div class="casTitle"><span class="NLM_cas:atitle">Design, properties and recent application of macrocycles in medicinal chemistry</span></div><div class="casAuthors">Ermert, Philipp</div><div class="citationInfo"><span class="NLM_cas:title">Chimia</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">71</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">678-702</span>CODEN:
                <span class="NLM_cas:coden">CHIMAD</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    
            (<span class="NLM_cas:orgname">Swiss Chemical Society</span>)
        </div><div class="casAbstract">Macrocyclic compds. have recently received increasing attention in drug discovery as these compds. offer the potential to modulate difficult targets and to access novel chemotypes.  Approaches towards libraries of macrocyclic compds. based on modular org. synthesis and applications of these technol. platforms to find and improve biol. active compds. are introduced in this minireview.  Alternatively, lead compds. may be obtained by truncation and modification of macrocyclic natural products.  Selected medicinal chem. programs are discussed, illustrating a macrocyclization approach toward ligands with improved properties.  The design of such ligands is often informed by X-ray crystal structures of protein-ligand complexes.  Efforts to understand cellular permeability and oral bioavailability of cyclic peptides and non-peptidic macrocycles are summarized.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqo_L_ef2BYLrVg90H21EOLACvtfcHk0lhDzNh1uY_uOg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhsFeitbg%253D&md5=aca0ab87a30863de362d9002c3d9f6cc</span></div><a href="/servlet/linkout?suffix=cit89&amp;dbid=16384&amp;doi=10.2533%2Fchimia.2017.678&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2533%252Fchimia.2017.678%26sid%3Dliteratum%253Aachs%26aulast%3DErmert%26aufirst%3DP.%26atitle%3DDesign%252C%2520Properties%2520and%2520Recent%2520Application%2520of%2520Macrocycles%2520in%2520Medicinal%2520Chemistry%26jtitle%3DChimia%26date%3D2017%26volume%3D71%26spage%3D678%26epage%3D702%26doi%3D10.2533%2Fchimia.2017.678" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref90"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref90'); return false;" data-citation="" class="refNumLink">90</a></strong><div class="NLM_citation" id="cit90"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Boss, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hazemann, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kimmerlin, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">von Korff, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lüthi, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peter, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sander, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siegrist, R.</span></span> <span> </span><span class="NLM_article-title">The Screening Compound Collection: A Key Asset for Drug Discovery</span>. <i>Chimia</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>71</i></span>,  <span class="NLM_fpage">667</span>– <span class="NLM_lpage">677</span>, <span class="refDoi"> DOI: 10.2533/chimia.2017.667</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref90/cit90&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.2533%2Fchimia.2017.667" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=71&publication_year=2017&pages=667-677&author=C.+Bossauthor=J.+Hazemannauthor=T.+Kimmerlinauthor=M.+von+Korffauthor=U.+L%C3%BCthiauthor=O.+Peterauthor=T.+Sanderauthor=R.+Siegrist&title=The+Screening+Compound+Collection%3A+A+Key+Asset+for+Drug+Discovery&doi=10.2533%2Fchimia.2017.667"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit90&amp;dbid=16384&amp;doi=10.2533%2Fchimia.2017.667&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2533%252Fchimia.2017.667%26sid%3Dliteratum%253Aachs%26aulast%3DBoss%26aufirst%3DC.%26aulast%3DHazemann%26aufirst%3DJ.%26aulast%3DKimmerlin%26aufirst%3DT.%26aulast%3Dvon%2BKorff%26aufirst%3DM.%26aulast%3DL%25C3%25BCthi%26aufirst%3DU.%26aulast%3DPeter%26aufirst%3DO.%26aulast%3DSander%26aufirst%3DT.%26aulast%3DSiegrist%26aufirst%3DR.%26atitle%3DThe%2520Screening%2520Compound%2520Collection%253A%2520A%2520Key%2520Asset%2520for%2520Drug%2520Discovery%26jtitle%3DChimia%26date%3D2017%26volume%3D71%26spage%3D667%26epage%3D677%26doi%3D10.2533%2Fchimia.2017.667" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref91"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref91'); return false;" data-citation="" class="refNumLink">91</a></strong><div class="NLM_citation" id="cit91"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Caenepeel, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Belmontes, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coxon, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moody, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hughes, P. E.</span></span> <span> </span><span class="NLM_article-title">Abstract 2027: Preclinical Evaluation of AMG 176, a Novel, Potent and Selective Mcl-1 Inhibitor with Robust Anti-Tumor Activity in Mcl-1 Dependent Cancer Models</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>77</i></span>,  <span class="NLM_fpage">2027</span>, <span class="refDoi"> DOI: 10.1158/1538-7445.AM2017-2027</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref91/cit91&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1158%2F1538-7445.AM2017-2027" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=77&publication_year=2017&pages=2027&author=S.+R.+Caenepeelauthor=B.+Belmontesauthor=J.+Sunauthor=A.+Coxonauthor=G.+Moodyauthor=P.+E.+Hughes&title=Abstract+2027%3A+Preclinical+Evaluation+of+AMG+176%2C+a+Novel%2C+Potent+and+Selective+Mcl-1+Inhibitor+with+Robust+Anti-Tumor+Activity+in+Mcl-1+Dependent+Cancer+Models&doi=10.1158%2F1538-7445.AM2017-2027"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit91&amp;dbid=16384&amp;doi=10.1158%2F1538-7445.AM2017-2027&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1538-7445.AM2017-2027%26sid%3Dliteratum%253Aachs%26aulast%3DCaenepeel%26aufirst%3DS.%2BR.%26aulast%3DBelmontes%26aufirst%3DB.%26aulast%3DSun%26aufirst%3DJ.%26aulast%3DCoxon%26aufirst%3DA.%26aulast%3DMoody%26aufirst%3DG.%26aulast%3DHughes%26aufirst%3DP.%2BE.%26atitle%3DAbstract%25202027%253A%2520Preclinical%2520Evaluation%2520of%2520AMG%2520176%252C%2520a%2520Novel%252C%2520Potent%2520and%2520Selective%2520Mcl-1%2520Inhibitor%2520with%2520Robust%2520Anti-Tumor%2520Activity%2520in%2520Mcl-1%2520Dependent%2520Cancer%2520Models%26jtitle%3DCancer%2520Res.%26date%3D2017%26volume%3D77%26spage%3D2027%26doi%3D10.1158%2F1538-7445.AM2017-2027" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref92"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref92'); return false;" data-citation="" class="refNumLink">92</a></strong><div class="NLM_citation" id="cit92"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Facchinetti, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tiseo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Di Maio, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Graziano, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bria, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rossi, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Novello, S.</span></span> <span> </span><span class="NLM_article-title">Tackling ALK in Non-Small Cell Lung Cancer: The Role of Novel Inhibitors</span>. <i>Transl. Lung Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">301</span>– <span class="NLM_lpage">321</span>, <span class="refDoi"> DOI: 10.21037/tlcr.2016.06.10</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref92/cit92&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.21037%2Ftlcr.2016.06.10" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref92/cit92&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=27413712" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref92/cit92&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsFekt7jI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2016&pages=301-321&author=F.+Facchinettiauthor=M.+Tiseoauthor=M.+Di+Maioauthor=P.+Grazianoauthor=E.+Briaauthor=G.+Rossiauthor=S.+Novello&title=Tackling+ALK+in+Non-Small+Cell+Lung+Cancer%3A+The+Role+of+Novel+Inhibitors&doi=10.21037%2Ftlcr.2016.06.10"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit92R"><div class="casContent"><span class="casTitleNuber">92</span><div class="casTitle"><span class="NLM_cas:atitle">Tackling ALK in non-small cell lung cancer: the role of novel inhibitors</span></div><div class="casAuthors">Facchinetti, Francesco; Tiseo, Marcello; Di Maio, Massimo; Graziano, Paolo; Bria, Emilio; Rossi, Giulio; Novello, Silvia</div><div class="citationInfo"><span class="NLM_cas:title">Translational Lung Cancer Research</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">301-321</span>CODEN:
                <span class="NLM_cas:coden">TLCRA9</span>;
        ISSN:<span class="NLM_cas:issn">2226-4477</span>.
    
            (<span class="NLM_cas:orgname">Pioneer Bioscience Publishing Co.</span>)
        </div><div class="casAbstract">Crizotinib is an oral inhibitor of anaplastic lymphoma kinase (ALK) with remarkable clin. activity in patients suffering from ALK-rearranged non-small cell lung cancer (NSCLC), accounting to its superiority compared to chemotherapy.  Unfortunately, virtually all ALK-rearranged tumors acquire resistance to crizotinib, frequently within one year since the treatment initiation.  To date, therapeutic strategies to overcome crizotinib resistance have focused on the use of more potent and structurally different compds.  Second-generation ALK inhibitors such as ceritinib (LDK378), alectinib (CH5424802/ RO5424802) and brigatinib (AP26113) have shown relevant clin. activity, consequently fostering their rapid clin. development and their approval by health agencies.  The third-generation inhibitor lorlatinib (PF-06463922), selectively active against ALK and ROS1, harbors impressive biol. potency; its efficacy in reversing resistance to crizotinib and to other ALK inhibitors is being proven by early clin. trials.  The NTRK1-3 and ROS1 inhibitor entrectinib (RXDX-101) has been reported to act against NSCLC harboring ALK fusion proteins too.  Despite the quick development of these novel agents, several issues remain to be discussed in the treatment of patients suffering from ALK-rearranged NSCLC.  This position paper will discuss the development, the current evidence and approvals, as long as the future perspectives of new ALK inhibitors beyond crizotinib.  Clin. behaviors of ALK-rearranged NSCLC vary significantly among patients and differential mol. events responsible of crizotinib resistance account for the most important quote of this heterogeneity.  The precious availability of a wide range of active anti-ALK compds. should be approached in a crit. and careful perspective, in order to develop treatment strategies tailored on the disease evolution of every single patient.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGop_lMlX4zg0LVg90H21EOLACvtfcHk0lhDzNh1uY_uOg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsFekt7jI&md5=4612d0c957b51f6407488ddb10ab41e0</span></div><a href="/servlet/linkout?suffix=cit92&amp;dbid=16384&amp;doi=10.21037%2Ftlcr.2016.06.10&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.21037%252Ftlcr.2016.06.10%26sid%3Dliteratum%253Aachs%26aulast%3DFacchinetti%26aufirst%3DF.%26aulast%3DTiseo%26aufirst%3DM.%26aulast%3DDi%2BMaio%26aufirst%3DM.%26aulast%3DGraziano%26aufirst%3DP.%26aulast%3DBria%26aufirst%3DE.%26aulast%3DRossi%26aufirst%3DG.%26aulast%3DNovello%26aufirst%3DS.%26atitle%3DTackling%2520ALK%2520in%2520Non-Small%2520Cell%2520Lung%2520Cancer%253A%2520The%2520Role%2520of%2520Novel%2520Inhibitors%26jtitle%3DTransl.%2520Lung%2520Cancer%2520Res.%26date%3D2016%26volume%3D5%26spage%3D301%26epage%3D321%26doi%3D10.21037%2Ftlcr.2016.06.10" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref93"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref93'); return false;" data-citation="" class="refNumLink">93</a></strong><div class="NLM_citation" id="cit93"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Morgan, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, D. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lennard, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anderton, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barrett, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eriksson, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fidock, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamrén, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">March, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matcham, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mettetal, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nicholls, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Platz, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rees, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Snowden, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pangalos, M. N.</span></span> <span> </span><span class="NLM_article-title">Impact of a Five-Dimensional Framework on R&D Productivity at AstraZeneca</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">167</span>– <span class="NLM_lpage">181</span>, <span class="refDoi"> DOI: 10.1038/nrd.2017.244</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref93/cit93&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1038%2Fnrd.2017.244" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref93/cit93&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=29348681" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref93/cit93&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC1cXht1SitL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2018&pages=167-181&author=P.+Morganauthor=D.+G.+Brownauthor=S.+Lennardauthor=M.+J.+Andertonauthor=J.+C.+Barrettauthor=U.+Erikssonauthor=M.+Fidockauthor=B.+Hamr%C3%A9nauthor=A.+Johnsonauthor=R.+E.+Marchauthor=J.+Matchamauthor=J.+Mettetalauthor=D.+J.+Nichollsauthor=S.+Platzauthor=S.+Reesauthor=M.+A.+Snowdenauthor=M.+N.+Pangalos&title=Impact+of+a+Five-Dimensional+Framework+on+R%26D+Productivity+at+AstraZeneca&doi=10.1038%2Fnrd.2017.244"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit93R"><div class="casContent"><span class="casTitleNuber">93</span><div class="casTitle"><span class="NLM_cas:atitle">Impact of a five-dimensional framework on R&D productivity at AstraZeneca</span></div><div class="casAuthors">Morgan, Paul; Brown, Dean G.; Lennard, Simon; Anderton, Mark J.; Barrett, J. Carl; Eriksson, Ulf; Fidock, Mark; Hamren, Bengt; Johnson, Anthony; March, Ruth E.; Matcham, James; Mettetal, Jerome; Nicholls, David J.; Platz, Stefan; Rees, Steve; Snowden, Michael A.; Pangalos, Menelas N.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">167-181</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">In 2011, AstraZeneca embarked on a major revision of its research and development (R&D) strategy with the aim of improving R&D productivity, which was below industry avs. in 2005-2010.  A cornerstone of the revised strategy was to focus decision-making on five tech. determinants (the right target, right tissue, right safety, right patient and right com. potential).  In this article, we describe the progress made using this '5R framework' in the hope that our experience could be useful to other companies tackling R&D productivity issues.  We focus on the evolution of our approach to target validation, hit and lead optimization, pharmacokinetic/pharmacodynamic modeling and drug safety testing, which have helped improve the quality of candidate drug nomination, as well as the development of the right culture, where 'truth seeking' is encouraged by more rigorous and quant. decision-making.  We also discuss where the approach has failed and the lessons learned.  Overall, the continued evolution and application of the 5R framework are beginning to have an impact, with success rates from candidate drug nomination to phase III completion improving from 4% in 2005-2010 to 19% in 2012-2016.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpwdx6OlRZpzrVg90H21EOLACvtfcHk0lj8_SMuJmxLuA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXht1SitL4%253D&md5=06906e0bc59dc715941998660a514ffe</span></div><a href="/servlet/linkout?suffix=cit93&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2017.244&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2017.244%26sid%3Dliteratum%253Aachs%26aulast%3DMorgan%26aufirst%3DP.%26aulast%3DBrown%26aufirst%3DD.%2BG.%26aulast%3DLennard%26aufirst%3DS.%26aulast%3DAnderton%26aufirst%3DM.%2BJ.%26aulast%3DBarrett%26aufirst%3DJ.%2BC.%26aulast%3DEriksson%26aufirst%3DU.%26aulast%3DFidock%26aufirst%3DM.%26aulast%3DHamr%25C3%25A9n%26aufirst%3DB.%26aulast%3DJohnson%26aufirst%3DA.%26aulast%3DMarch%26aufirst%3DR.%2BE.%26aulast%3DMatcham%26aufirst%3DJ.%26aulast%3DMettetal%26aufirst%3DJ.%26aulast%3DNicholls%26aufirst%3DD.%2BJ.%26aulast%3DPlatz%26aufirst%3DS.%26aulast%3DRees%26aufirst%3DS.%26aulast%3DSnowden%26aufirst%3DM.%2BA.%26aulast%3DPangalos%26aufirst%3DM.%2BN.%26atitle%3DImpact%2520of%2520a%2520Five-Dimensional%2520Framework%2520on%2520R%2526D%2520Productivity%2520at%2520AstraZeneca%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2018%26volume%3D17%26spage%3D167%26epage%3D181%26doi%3D10.1038%2Fnrd.2017.244" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref94"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref94'); return false;" data-citation="" class="refNumLink">94</a></strong><div class="NLM_citation" id="cit94"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Moffat, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vincent, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eder, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prunotto, M.</span></span> <span> </span><span class="NLM_article-title">Opportunities and Challenges in Phenotypic Drug Discovery: An Industry Perspective</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">531</span>– <span class="NLM_lpage">543</span>, <span class="refDoi"> DOI: 10.1038/nrd.2017.111</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref94/cit94&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1038%2Fnrd.2017.111" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref94/cit94&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=28685762" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref94/cit94&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtFelsLrJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2017&pages=531-543&author=J.+G.+Moffatauthor=F.+Vincentauthor=J.+A.+Leeauthor=J.+Ederauthor=M.+Prunotto&title=Opportunities+and+Challenges+in+Phenotypic+Drug+Discovery%3A+An+Industry+Perspective&doi=10.1038%2Fnrd.2017.111"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit94R"><div class="casContent"><span class="casTitleNuber">94</span><div class="casTitle"><span class="NLM_cas:atitle">Opportunities and challenges in phenotypic drug discovery: an industry perspective</span></div><div class="casAuthors">Moffat, John G.; Vincent, Fabien; Lee, Jonathan A.; Eder, Jorg; Prunotto, Marco</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">531-543</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Phenotypic drug discovery (PDD) approaches do not rely on knowledge of the identity of a specific drug target or a hypothesis about its role in disease, in contrast to the target-based strategies that have been widely used in the pharmaceutical industry in the past three decades.  However, in recent years, there has been a resurgence in interest in PDD approaches based on their potential to address the incompletely understood complexity of diseases and their promise of delivering first-in-class drugs, as well as major advances in the tools for cell-based phenotypic screening.  Nevertheless, PDD approaches also have considerable challenges, such as hit validation and target deconvolution.  This article focuses on the lessons learned by researchers engaged in PDD in the pharmaceutical industry and considers the impact of 'omics' knowledge in defining a cellular disease phenotype in the era of precision medicine, introducing the concept of a chain of translatability.  We particularly aim to identify features and areas in which PDD can best deliver value to drug discovery portfolios and can contribute to the identification and the development of novel medicines, and to illustrate the challenges and uncertainties that are assocd. with PDD in order to help set realistic expectations with regard to its benefits and costs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpGrD5TsU8_kLVg90H21EOLACvtfcHk0lj8_SMuJmxLuA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtFelsLrJ&md5=d83196715c879a81d9eabcf544e303db</span></div><a href="/servlet/linkout?suffix=cit94&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2017.111&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2017.111%26sid%3Dliteratum%253Aachs%26aulast%3DMoffat%26aufirst%3DJ.%2BG.%26aulast%3DVincent%26aufirst%3DF.%26aulast%3DLee%26aufirst%3DJ.%2BA.%26aulast%3DEder%26aufirst%3DJ.%26aulast%3DPrunotto%26aufirst%3DM.%26atitle%3DOpportunities%2520and%2520Challenges%2520in%2520Phenotypic%2520Drug%2520Discovery%253A%2520An%2520Industry%2520Perspective%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2017%26volume%3D16%26spage%3D531%26epage%3D543%26doi%3D10.1038%2Fnrd.2017.111" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref95"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref95'); return false;" data-citation="" class="refNumLink">95</a></strong><div class="NLM_citation" id="cit95"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Plowright, A. T.</span>; <span class="NLM_string-name">Drowley, L.</span></span> <span> </span><span class="NLM_article-title">Phenotypic Screening</span>. In  <i>Platform Technologies in Drug Discovery and Validation</i>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goodnow, R. A.,  Jr.</span></span>, Ed.; <span class="NLM_publisher-name">Academic Press</span>: <span class="NLM_publisher-loc">Cambridge, MA</span>, <span class="NLM_year">2017</span>; pp  <span class="NLM_fpage">263</span>– <span class="NLM_lpage">299</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref95/cit95&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1016%2Fbs.armc.2017.07.001" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2017&pages=263-299&author=A.+T.+Plowright&author=L.+Drowleyauthor=R.+A.+Goodnow&title=Platform+Technologies+in+Drug+Discovery+and+Validation"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit95&amp;dbid=16384&amp;doi=10.1016%2Fbs.armc.2017.07.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1016%252Fbs.armc.2017.07.001%26sid%3Dliteratum%253Aachs%26aulast%3DPlowright%26aufirst%3DA.%2BT.%26atitle%3DPhenotypic%2520Screening%26btitle%3DPlatform%2520Technologies%2520in%2520Drug%2520Discovery%2520and%2520Validation%26aulast%3DGoodnow%26aufirst%3DR.%2BA.%26pub%3DAcademic%2520Press%26date%3D2017%26spage%3D263%26epage%3D299" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref96"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref96'); return false;" data-citation="" class="refNumLink">96</a></strong><div class="NLM_citation" id="cit96"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lewis, K.</span></span> <span> </span><span class="NLM_article-title">Platforms for Antibiotic Discovery</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">371</span>– <span class="NLM_lpage">387</span>, <span class="refDoi"> DOI: 10.1038/nrd3975</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref96/cit96&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1038%2Fnrd3975" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref96/cit96&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=23629505" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref96/cit96&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC3sXms1Ojt70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2013&pages=371-387&author=K.+Lewis&title=Platforms+for+Antibiotic+Discovery&doi=10.1038%2Fnrd3975"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit96R"><div class="casContent"><span class="casTitleNuber">96</span><div class="casTitle"><span class="NLM_cas:atitle">Platforms for antibiotic discovery</span></div><div class="casAuthors">Lewis, Kim</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">371-387</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The spread of resistant bacteria, leading to untreatable infections, is a major public health threat but the pace of antibiotic discovery to combat these pathogens has slowed down.  Most antibiotics were originally isolated by screening soil-derived actinomycetes during the golden era of antibiotic discovery in the 1940s to 1960s.  However, diminishing returns from this discovery platform led to its collapse, and efforts to create a new platform based on target-focused screening of large libraries of synthetic compds. failed, in part owing to the lack of penetration of such compds. through the bacterial envelope.  This article considers strategies to re-establish viable platforms for antibiotic discovery.  These include investigating untapped natural product sources such as uncultured bacteria, establishing rules of compd. penetration to enable the development of synthetic antibiotics, developing species-specific antibiotics and identifying prodrugs that have the potential to eradicate dormant persisters, which are often responsible for hard-to-treat infections.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqOyTc7KWJVJrVg90H21EOLACvtfcHk0lj8_SMuJmxLuA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXms1Ojt70%253D&md5=4f69d8c4c1a25e952ae8ed7df2631115</span></div><a href="/servlet/linkout?suffix=cit96&amp;dbid=16384&amp;doi=10.1038%2Fnrd3975&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd3975%26sid%3Dliteratum%253Aachs%26aulast%3DLewis%26aufirst%3DK.%26atitle%3DPlatforms%2520for%2520Antibiotic%2520Discovery%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2013%26volume%3D12%26spage%3D371%26epage%3D387%26doi%3D10.1038%2Fnrd3975" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref97"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref97'); return false;" data-citation="" class="refNumLink">97</a></strong><div class="NLM_citation" id="cit97"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ling, L. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schneider, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peoples, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spoering, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Engels, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Conlon, B. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mueller, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schäberle, T. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hughes, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Epstein, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lazarides, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steadman, V. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cohen, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Felix, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fetterman, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Millett, W. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nitti, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zullo, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lewis, K.</span></span> <span> </span><span class="NLM_article-title">A New Antibiotic Kills Pathogens without Detectable Resistance</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>517</i></span>,  <span class="NLM_fpage">455</span>– <span class="NLM_lpage">459</span>, <span class="refDoi"> DOI: 10.1038/nature14098</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref97/cit97&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1038%2Fnature14098" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref97/cit97&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=25561178" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref97/cit97&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtFOju7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=517&publication_year=2015&pages=455-459&author=L.+L.+Lingauthor=T.+Schneiderauthor=A.+J.+Peoplesauthor=A.+L.+Spoeringauthor=I.+Engelsauthor=B.+P.+Conlonauthor=A.+Muellerauthor=T.+F.+Sch%C3%A4berleauthor=D.+E.+Hughesauthor=S.+Epsteinauthor=M.+Jonesauthor=L.+Lazaridesauthor=V.+A.+Steadmanauthor=D.+R.+Cohenauthor=C.+R.+Felixauthor=K.+A.+Fettermanauthor=W.+P.+Millettauthor=A.+G.+Nittiauthor=A.+M.+Zulloauthor=C.+Chenauthor=K.+Lewis&title=A+New+Antibiotic+Kills+Pathogens+without+Detectable+Resistance&doi=10.1038%2Fnature14098"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit97R"><div class="casContent"><span class="casTitleNuber">97</span><div class="casTitle"><span class="NLM_cas:atitle">A new antibiotic kills pathogens without detectable resistance</span></div><div class="casAuthors">Ling, Losee L.; Schneider, Tanja; Peoples, Aaron J.; Spoering, Amy L.; Engels, Ina; Conlon, Brian P.; Mueller, Anna; Schaberle, Till F.; Hughes, Dallas E.; Epstein, Slava; Jones, Michael; Lazarides, Linos; Steadman, Victoria A.; Cohen, Douglas R.; Felix, Cintia R.; Fetterman, K. Ashley; Millett, William P.; Nitti, Anthony G.; Zullo, Ashley M.; Chen, Chao; Lewis, Kim</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">517</span>
        (<span class="NLM_cas:issue">7535</span>),
    <span class="NLM_cas:pages">455-459</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Antibiotic resistance is spreading faster than the introduction of new compds. into clin. practice, causing a public health crisis.  Most antibiotics were produced by screening soil microorganisms, but this limited resource of cultivable bacteria was overmined by the 1960s.  Synthetic approaches to produce antibiotics have been unable to replace this platform.  Uncultured bacteria make up approx. 99% of all species in external environments, and are an untapped source of new antibiotics.  We developed several methods to grow uncultured organisms by cultivation in situ or by using specific growth factors.  Here we report a new antibiotic that we term teixobactin, discovered in a screen of uncultured bacteria.  Teixobactin inhibits cell wall synthesis by binding to a highly conserved motif of lipid II (precursor of peptidoglycan) and lipid III (precursor of cell wall teichoic acid).  We did not obtain any mutants of Staphylococcus aureus or Mycobacterium tuberculosis resistant to teixobactin.  The properties of this compd. suggest a path towards developing antibiotics that are likely to avoid development of resistance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrV0hZrSZheoLVg90H21EOLACvtfcHk0lgKL_6Bgm8RxQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtFOju7w%253D&md5=27e302d7d44a549a91aa52113c3b4ad8</span></div><a href="/servlet/linkout?suffix=cit97&amp;dbid=16384&amp;doi=10.1038%2Fnature14098&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature14098%26sid%3Dliteratum%253Aachs%26aulast%3DLing%26aufirst%3DL.%2BL.%26aulast%3DSchneider%26aufirst%3DT.%26aulast%3DPeoples%26aufirst%3DA.%2BJ.%26aulast%3DSpoering%26aufirst%3DA.%2BL.%26aulast%3DEngels%26aufirst%3DI.%26aulast%3DConlon%26aufirst%3DB.%2BP.%26aulast%3DMueller%26aufirst%3DA.%26aulast%3DSch%25C3%25A4berle%26aufirst%3DT.%2BF.%26aulast%3DHughes%26aufirst%3DD.%2BE.%26aulast%3DEpstein%26aufirst%3DS.%26aulast%3DJones%26aufirst%3DM.%26aulast%3DLazarides%26aufirst%3DL.%26aulast%3DSteadman%26aufirst%3DV.%2BA.%26aulast%3DCohen%26aufirst%3DD.%2BR.%26aulast%3DFelix%26aufirst%3DC.%2BR.%26aulast%3DFetterman%26aufirst%3DK.%2BA.%26aulast%3DMillett%26aufirst%3DW.%2BP.%26aulast%3DNitti%26aufirst%3DA.%2BG.%26aulast%3DZullo%26aufirst%3DA.%2BM.%26aulast%3DChen%26aufirst%3DC.%26aulast%3DLewis%26aufirst%3DK.%26atitle%3DA%2520New%2520Antibiotic%2520Kills%2520Pathogens%2520without%2520Detectable%2520Resistance%26jtitle%3DNature%26date%3D2015%26volume%3D517%26spage%3D455%26epage%3D459%26doi%3D10.1038%2Fnature14098" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref98"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref98'); return false;" data-citation="" class="refNumLink">98</a></strong><div class="NLM_citation" id="cit98"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Plowright, A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Engkvist, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gill, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knerr, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Q.-D.</span></span> <span> </span><span class="NLM_article-title">Heart Regeneration: Opportunities and Challenges for Drug Discovery with Novel Chemical and Therapeutic Methods or Agents</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">4056</span>– <span class="NLM_lpage">4075</span>, <span class="refDoi"> DOI: 10.1002/anie.201307034</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref98/cit98&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1002%2Fanie.201307034" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2014&pages=4056-4075&author=A.+T.+Plowrightauthor=O.+Engkvistauthor=A.+Gillauthor=L.+Knerrauthor=Q.-D.+Wang&title=Heart+Regeneration%3A+Opportunities+and+Challenges+for+Drug+Discovery+with+Novel+Chemical+and+Therapeutic+Methods+or+Agents&doi=10.1002%2Fanie.201307034"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit98&amp;dbid=16384&amp;doi=10.1002%2Fanie.201307034&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201307034%26sid%3Dliteratum%253Aachs%26aulast%3DPlowright%26aufirst%3DA.%2BT.%26aulast%3DEngkvist%26aufirst%3DO.%26aulast%3DGill%26aufirst%3DA.%26aulast%3DKnerr%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DQ.-D.%26atitle%3DHeart%2520Regeneration%253A%2520Opportunities%2520and%2520Challenges%2520for%2520Drug%2520Discovery%2520with%2520Novel%2520Chemical%2520and%2520Therapeutic%2520Methods%2520or%2520Agents%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2014%26volume%3D53%26spage%3D4056%26epage%3D4075%26doi%3D10.1002%2Fanie.201307034" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref99"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref99'); return false;" data-citation="" class="refNumLink">99</a></strong><div class="NLM_citation" id="cit99"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Avior, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sagi, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Benvenisty, N.</span></span> <span> </span><span class="NLM_article-title">Pluripotent Stem Cells in Disease Modelling and Drug Discovery</span>. <i>Nat. Rev. Mol. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">170</span>– <span class="NLM_lpage">182</span>, <span class="refDoi"> DOI: 10.1038/nrm.2015.27</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref99/cit99&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1038%2Fnrm.2015.27" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref99/cit99&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=26818440" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref99/cit99&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsFygsro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2016&pages=170-182&author=Y.+Aviorauthor=I.+Sagiauthor=N.+Benvenisty&title=Pluripotent+Stem+Cells+in+Disease+Modelling+and+Drug+Discovery&doi=10.1038%2Fnrm.2015.27"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit99R"><div class="casContent"><span class="casTitleNuber">99</span><div class="casTitle"><span class="NLM_cas:atitle">Pluripotent stem cells in disease modelling and drug discovery</span></div><div class="casAuthors">Avior, Yishai; Sagi, Ido; Benvenisty, Nissim</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Molecular Cell Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">170-182</span>CODEN:
                <span class="NLM_cas:coden">NRMCBP</span>;
        ISSN:<span class="NLM_cas:issn">1471-0072</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Exptl. modeling of human disorders enables the definition of the cellular and mol. mechanisms underlying diseases and the development of therapies for treating them.  The availability of human pluripotent stem cells (PSCs), which are capable of self-renewal and have the potential to differentiate into virtually any cell type, can now help to overcome the limitations of animal models for certain disorders.  The ability to model human diseases using cultured PSCs has revolutionized the ways in which we study monogenic, complex and epigenetic disorders, as well as early- and late-onset diseases.  Several strategies are used to generate such disease models using either embryonic stem cells (ES cells) or patient-specific induced PSCs (iPSCs), creating new possibilities for the establishment of models and their use in drug screening.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpHtd9-bDU1BLVg90H21EOLACvtfcHk0lgKL_6Bgm8RxQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsFygsro%253D&md5=4072946787f6f29809eac02e1b2574e5</span></div><a href="/servlet/linkout?suffix=cit99&amp;dbid=16384&amp;doi=10.1038%2Fnrm.2015.27&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrm.2015.27%26sid%3Dliteratum%253Aachs%26aulast%3DAvior%26aufirst%3DY.%26aulast%3DSagi%26aufirst%3DI.%26aulast%3DBenvenisty%26aufirst%3DN.%26atitle%3DPluripotent%2520Stem%2520Cells%2520in%2520Disease%2520Modelling%2520and%2520Drug%2520Discovery%26jtitle%3DNat.%2520Rev.%2520Mol.%2520Cell%2520Biol.%26date%3D2016%26volume%3D17%26spage%3D170%26epage%3D182%26doi%3D10.1038%2Fnrm.2015.27" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref100"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref100'); return false;" data-citation="" class="refNumLink">100</a></strong><div class="NLM_citation" id="cit100"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mills, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Titmarsh, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koenig, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parker, B. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ryall, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quaife-Ryan, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Voges, H. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hodson, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferguson, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drowley, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plowright, A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Needham, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Q.-D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gregorevic, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xin, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, W. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parton, R. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nielsen, L. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Launikonis, B. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">James, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elliott, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Porrello, E. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hudson, J. E.</span></span> <span> </span><span class="NLM_article-title">Functional Screening in Human Cardiac Organoids Reveals a Metabolic Mechanism for Cardiomyocyte Cell Cycle Arrest</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>114</i></span>,  <span class="NLM_fpage">E8372</span>– <span class="NLM_lpage">E8381</span>, <span class="refDoi"> DOI: 10.1073/pnas.1707316114</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref100/cit100&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1073%2Fpnas.1707316114" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=114&publication_year=2017&pages=E8372-E8381&author=R.+J.+Millsauthor=D.+M.+Titmarshauthor=X.+Koenigauthor=B.+L.+Parkerauthor=J.+G.+Ryallauthor=G.+A.+Quaife-Ryanauthor=H.+K.+Vogesauthor=M.+P.+Hodsonauthor=C.+Fergusonauthor=L.+Drowleyauthor=A.+T.+Plowrightauthor=E.+J.+Needhamauthor=Q.-D.+Wangauthor=P.+Gregorevicauthor=M.+Xinauthor=W.+G.+Thomasauthor=R.+G.+Partonauthor=L.+K.+Nielsenauthor=B.+S.+Launikonisauthor=D.+E.+Jamesauthor=D.+A.+Elliottauthor=E.+R.+Porrelloauthor=J.+E.+Hudson&title=Functional+Screening+in+Human+Cardiac+Organoids+Reveals+a+Metabolic+Mechanism+for+Cardiomyocyte+Cell+Cycle+Arrest&doi=10.1073%2Fpnas.1707316114"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit100&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1707316114&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1707316114%26sid%3Dliteratum%253Aachs%26aulast%3DMills%26aufirst%3DR.%2BJ.%26aulast%3DTitmarsh%26aufirst%3DD.%2BM.%26aulast%3DKoenig%26aufirst%3DX.%26aulast%3DParker%26aufirst%3DB.%2BL.%26aulast%3DRyall%26aufirst%3DJ.%2BG.%26aulast%3DQuaife-Ryan%26aufirst%3DG.%2BA.%26aulast%3DVoges%26aufirst%3DH.%2BK.%26aulast%3DHodson%26aufirst%3DM.%2BP.%26aulast%3DFerguson%26aufirst%3DC.%26aulast%3DDrowley%26aufirst%3DL.%26aulast%3DPlowright%26aufirst%3DA.%2BT.%26aulast%3DNeedham%26aufirst%3DE.%2BJ.%26aulast%3DWang%26aufirst%3DQ.-D.%26aulast%3DGregorevic%26aufirst%3DP.%26aulast%3DXin%26aufirst%3DM.%26aulast%3DThomas%26aufirst%3DW.%2BG.%26aulast%3DParton%26aufirst%3DR.%2BG.%26aulast%3DNielsen%26aufirst%3DL.%2BK.%26aulast%3DLaunikonis%26aufirst%3DB.%2BS.%26aulast%3DJames%26aufirst%3DD.%2BE.%26aulast%3DElliott%26aufirst%3DD.%2BA.%26aulast%3DPorrello%26aufirst%3DE.%2BR.%26aulast%3DHudson%26aufirst%3DJ.%2BE.%26atitle%3DFunctional%2520Screening%2520in%2520Human%2520Cardiac%2520Organoids%2520Reveals%2520a%2520Metabolic%2520Mechanism%2520for%2520Cardiomyocyte%2520Cell%2520Cycle%2520Arrest%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2017%26volume%3D114%26spage%3DE8372%26epage%3DE8381%26doi%3D10.1073%2Fpnas.1707316114" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref101"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref101'); return false;" data-citation="" class="refNumLink">101</a></strong><div class="NLM_citation" id="cit101"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lee, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramirez, C. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeltner, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Radu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhinder, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, Y. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choi, I. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mukherjee-Clavin, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Djaballah, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Studer, L.</span></span> <span> </span><span class="NLM_article-title">Large-Scale Screening Using Familial Dysautonomia Induced Pluripotent Stem Cells Identifies Compounds That Rescue <i>IKBKAP</i> Expression</span>. <i>Nat. Biotechnol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">1244</span>– <span class="NLM_lpage">1248</span>, <span class="refDoi"> DOI: 10.1038/nbt.2435</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref101/cit101&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1038%2Fnbt.2435" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2012&pages=1244-1248&author=G.+Leeauthor=C.+N.+Ramirezauthor=H.+Kimauthor=N.+Zeltnerauthor=B.+Liuauthor=C.+Raduauthor=B.+Bhinderauthor=Y.+J.+Kimauthor=I.+Y.+Choiauthor=B.+Mukherjee-Clavinauthor=H.+Djaballahauthor=L.+Studer&title=Large-Scale+Screening+Using+Familial+Dysautonomia+Induced+Pluripotent+Stem+Cells+Identifies+Compounds+That+Rescue+IKBKAP+Expression&doi=10.1038%2Fnbt.2435"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit101&amp;dbid=16384&amp;doi=10.1038%2Fnbt.2435&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt.2435%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DG.%26aulast%3DRamirez%26aufirst%3DC.%2BN.%26aulast%3DKim%26aufirst%3DH.%26aulast%3DZeltner%26aufirst%3DN.%26aulast%3DLiu%26aufirst%3DB.%26aulast%3DRadu%26aufirst%3DC.%26aulast%3DBhinder%26aufirst%3DB.%26aulast%3DKim%26aufirst%3DY.%2BJ.%26aulast%3DChoi%26aufirst%3DI.%2BY.%26aulast%3DMukherjee-Clavin%26aufirst%3DB.%26aulast%3DDjaballah%26aufirst%3DH.%26aulast%3DStuder%26aufirst%3DL.%26atitle%3DLarge-Scale%2520Screening%2520Using%2520Familial%2520Dysautonomia%2520Induced%2520Pluripotent%2520Stem%2520Cells%2520Identifies%2520Compounds%2520That%2520Rescue%2520IKBKAP%2520Expression%26jtitle%3DNat.%2520Biotechnol.%26date%3D2012%26volume%3D30%26spage%3D1244%26epage%3D1248%26doi%3D10.1038%2Fnbt.2435" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref102"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref102'); return false;" data-citation="" class="refNumLink">102</a></strong><div class="NLM_citation" id="cit102"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jones, L. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bunnage, M. E.</span></span> <span> </span><span class="NLM_article-title">Applications of Chemogenomic Library Screening in Drug Discovery</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">285</span>– <span class="NLM_lpage">296</span>, <span class="refDoi"> DOI: 10.1038/nrd.2016.244</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref102/cit102&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1038%2Fnrd.2016.244" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref102/cit102&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=28104905" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref102/cit102&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC2sXht1Ort7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2017&pages=285-296&author=L.+H.+Jonesauthor=M.+E.+Bunnage&title=Applications+of+Chemogenomic+Library+Screening+in+Drug+Discovery&doi=10.1038%2Fnrd.2016.244"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit102R"><div class="casContent"><span class="casTitleNuber">102</span><div class="casTitle"><span class="NLM_cas:atitle">Applications of chemogenomic library screening in drug discovery</span></div><div class="casAuthors">Jones, Lyn H.; Bunnage, Mark E.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">285-296</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The allure of phenotypic screening, combined with the industry preference for target-based approaches, has prompted the development of innovative chem. biol. technologies that facilitate the identification of new therapeutic targets for accelerated drug discovery.  A chemogenomic library is a collection of selective small-mol. pharmacol. agents, and a hit from such a set in a phenotypic screen suggests that the annotated target or targets of that pharmacol. agent may be involved in perturbing the observable phenotype.  In this Review, we describe opportunities for chemogenomic screening to considerably expedite the conversion of phenotypic screening projects into target-based drug discovery approaches.  Other applications are explored, including drug repositioning, predictive toxicol. and the discovery of novel pharmacol. modalities.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGruetMviqci8rVg90H21EOLACvtfcHk0lh5PmguPQW-Pg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXht1Ort7c%253D&md5=a51a882947528be18392170cfa48c554</span></div><a href="/servlet/linkout?suffix=cit102&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2016.244&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2016.244%26sid%3Dliteratum%253Aachs%26aulast%3DJones%26aufirst%3DL.%2BH.%26aulast%3DBunnage%26aufirst%3DM.%2BE.%26atitle%3DApplications%2520of%2520Chemogenomic%2520Library%2520Screening%2520in%2520Drug%2520Discovery%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2017%26volume%3D16%26spage%3D285%26epage%3D296%26doi%3D10.1038%2Fnrd.2016.244" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref103"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref103'); return false;" data-citation="" class="refNumLink">103</a></strong><div class="NLM_citation" id="cit103"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cornett, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">King, F. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nigsch, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paris, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McAllister, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jenkins, J. L.</span></span> <span> </span><span class="NLM_article-title">Evidence-Based and Quantitative Prioritization of Tool Compounds in Phenotypic Drug Discovery</span>. <i>Cell Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">862</span>– <span class="NLM_lpage">874</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2016.05.016</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref103/cit103&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1016%2Fj.chembiol.2016.05.016" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref103/cit103&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=27427232" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref103/cit103&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtFyiur3E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2016&pages=862-874&author=Y.+Wangauthor=A.+Cornettauthor=F.+J.+Kingauthor=Y.+Maoauthor=F.+Nigschauthor=C.+G.+Parisauthor=G.+McAllisterauthor=J.+L.+Jenkins&title=Evidence-Based+and+Quantitative+Prioritization+of+Tool+Compounds+in+Phenotypic+Drug+Discovery&doi=10.1016%2Fj.chembiol.2016.05.016"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit103R"><div class="casContent"><span class="casTitleNuber">103</span><div class="casTitle"><span class="NLM_cas:atitle">Evidence-Based and Quantitative Prioritization of Tool Compounds in Phenotypic Drug Discovery</span></div><div class="casAuthors">Wang, Yuan; Cornett, Allen; King, Fred J.; Mao, Yi; Nigsch, Florian; Paris, C. Gregory; McAllister, Gregory; Jenkins, Jeremy L.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">862-874</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The use of potent and selective chem. tools with well-defined targets can help elucidate biol. processes driving phenotypes in phenotypic screens.  However, identification of selective compds. en masse to create targeted screening sets is non-trivial.  A systematic approach is needed to prioritize probes, which prevents the repeated use of published but unselective compds.  Here we performed a meta-anal. of integrated large-scale, heterogeneous bioactivity data to create an evidence-based, quant. metric to systematically rank tool compds. for targets.  Our tool score (TS) was then tested on hundreds of compds. by assessing their activity profiles in a panel of 41 cell-based pathway assays.  We demonstrate that high-TS tools show more reliably selective phenotypic profiles than lower-TS compds.  Addnl. we highlight frequently tested compds. that are non-selective tools and distinguish target family polypharmacol. from cross-family promiscuity.  TS can therefore be used to prioritize compds. from heterogeneous databases for phenotypic screening.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGquaPu2t_l4ZbVg90H21EOLACvtfcHk0lh5PmguPQW-Pg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtFyiur3E&md5=b990346556ac032b8cecbfc9455f8458</span></div><a href="/servlet/linkout?suffix=cit103&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2016.05.016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2016.05.016%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DY.%26aulast%3DCornett%26aufirst%3DA.%26aulast%3DKing%26aufirst%3DF.%2BJ.%26aulast%3DMao%26aufirst%3DY.%26aulast%3DNigsch%26aufirst%3DF.%26aulast%3DParis%26aufirst%3DC.%2BG.%26aulast%3DMcAllister%26aufirst%3DG.%26aulast%3DJenkins%26aufirst%3DJ.%2BL.%26atitle%3DEvidence-Based%2520and%2520Quantitative%2520Prioritization%2520of%2520Tool%2520Compounds%2520in%2520Phenotypic%2520Drug%2520Discovery%26jtitle%3DCell%2520Chem.%2520Biol.%26date%3D2016%26volume%3D23%26spage%3D862%26epage%3D874%26doi%3D10.1016%2Fj.chembiol.2016.05.016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref104"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref104'); return false;" data-citation="" class="refNumLink">104</a></strong><div class="NLM_citation" id="cit104"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Santos, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ursu, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gaulton, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bento, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Donadi, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bologa, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karlsson, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Al-Lazikani, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hersey, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oprea, T. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Overington, J. P.</span></span> <span> </span><span class="NLM_article-title">A Comprehensive Map of Molecular Drug Targets</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">19</span>– <span class="NLM_lpage">34</span>, <span class="refDoi"> DOI: 10.1038/nrd.2016.230</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref104/cit104&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1038%2Fnrd.2016.230" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref104/cit104&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=27910877" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref104/cit104&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC28XitVSms7rK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2017&pages=19-34&author=R.+Santosauthor=O.+Ursuauthor=A.+Gaultonauthor=A.+P.+Bentoauthor=R.+S.+Donadiauthor=C.+G.+Bologaauthor=A.+Karlssonauthor=B.+Al-Lazikaniauthor=A.+Herseyauthor=T.+I.+Opreaauthor=J.+P.+Overington&title=A+Comprehensive+Map+of+Molecular+Drug+Targets&doi=10.1038%2Fnrd.2016.230"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit104R"><div class="casContent"><span class="casTitleNuber">104</span><div class="casTitle"><span class="NLM_cas:atitle">A comprehensive map of molecular drug targets</span></div><div class="casAuthors">Santos, Rita; Ursu, Oleg; Gaulton, Anna; Bento, A. Patricia; Donadi, Ramesh S.; Bologa, Cristian G.; Karlsson, Anneli; Al-Lazikani, Bissan; Hersey, Anne; Oprea, Tudor I.; Overington, John P.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">19-34</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The success of mechanism-based drug discovery depends on the definition of the drug target.  This definition becomes even more important as we try to link drug response to genetic variation, understand stratified clin. efficacy and safety, rationalize the differences between drugs in the same therapeutic class and predict drug utility in patient subgroups.  However, drug targets are often poorly defined in the literature, both for launched drugs and for potential therapeutic agents in discovery and development.  Here, we present an updated comprehensive map of mol. targets of approved drugs.  We curate a total of 893 human and pathogen-derived biomols. through which 1,578 US FDA-approved drugs act.  These biomols. include 667 human-genome-derived proteins targeted by drugs for human disease.  Anal. of these drug targets indicates the continued dominance of privileged target families across disease areas, but also the growth of novel first-in-class mechanisms, particularly in oncol.  We explore the relationships between bioactivity class and clin. success, as well as the presence of orthologues between human and animal models and between pathogen and human genomes.  Through the collaboration of three independent teams, we highlight some of the ongoing challenges in accurately defining the targets of mol. therapeutics and present conventions for deconvoluting the complexities of mol. pharmacol. and drug efficacy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpTfWwWkoZnbrVg90H21EOLACvtfcHk0lh5PmguPQW-Pg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitVSms7rK&md5=7437d4b678ac75dac0a5e3248df98761</span></div><a href="/servlet/linkout?suffix=cit104&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2016.230&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2016.230%26sid%3Dliteratum%253Aachs%26aulast%3DSantos%26aufirst%3DR.%26aulast%3DUrsu%26aufirst%3DO.%26aulast%3DGaulton%26aufirst%3DA.%26aulast%3DBento%26aufirst%3DA.%2BP.%26aulast%3DDonadi%26aufirst%3DR.%2BS.%26aulast%3DBologa%26aufirst%3DC.%2BG.%26aulast%3DKarlsson%26aufirst%3DA.%26aulast%3DAl-Lazikani%26aufirst%3DB.%26aulast%3DHersey%26aufirst%3DA.%26aulast%3DOprea%26aufirst%3DT.%2BI.%26aulast%3DOverington%26aufirst%3DJ.%2BP.%26atitle%3DA%2520Comprehensive%2520Map%2520of%2520Molecular%2520Drug%2520Targets%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2017%26volume%3D16%26spage%3D19%26epage%3D34%26doi%3D10.1038%2Fnrd.2016.230" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref105"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref105'); return false;" data-citation="" class="refNumLink">105</a></strong><div class="NLM_citation" id="cit105"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kato, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Comer, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sakata-Kato, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharma, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharma, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maetani, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bastien, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brancucci, N. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bittker, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Corey, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clarke, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Derbyshire, E. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dornan, G. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duffy, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eckley, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Itoe, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koolen, K. M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lewis, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lui, P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lukens, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lund, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">March, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meibalan, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meier, B. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McPhail, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitasev, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moss, E. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sayes, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Gessel, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wawer, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yoshinaga, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeeman, A.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Avery, V. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhatia, S. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burke, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Catteruccia, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clardy, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clemons, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dechering, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duvall, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Foley, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gusovsky, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kocken, C. H. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marti, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morningstar, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Munoz, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neafsey, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharma, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winzeler, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wirth, D. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scherer, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schreiber, S. L.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">Diversity-Oriented Synthesis Yields Novel Multistage Antimalarial Inhibitors</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>538</i></span>,  <span class="NLM_fpage">344</span>– <span class="NLM_lpage">349</span>, <span class="refDoi"> DOI: 10.1038/nature19804</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref105/cit105&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1038%2Fnature19804" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref105/cit105&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=27602946" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref105/cit105&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC28XhslWhsLrI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=538&publication_year=2016&pages=344-349&author=N.+Katoauthor=E.+Comerauthor=T.+Sakata-Katoauthor=A.+Sharmaauthor=M.+Sharmaauthor=M.+Maetaniauthor=J.+Bastienauthor=N.+M.+Brancucciauthor=J.+A.+Bittkerauthor=V.+Coreyauthor=D.+Clarkeauthor=E.+R.+Derbyshireauthor=G.+L.+Dornanauthor=S.+Duffyauthor=S.+Eckleyauthor=M.+A.+Itoeauthor=K.+M.+J.+Koolenauthor=T.+A.+Lewisauthor=P.+S.+Luiauthor=A.+K.+Lukensauthor=E.+Lundauthor=S.+Marchauthor=E.+Meibalanauthor=B.+C.+Meierauthor=J.+A.+McPhailauthor=B.+Mitasevauthor=E.+L.+Mossauthor=M.+Sayesauthor=Y.+Van+Gesselauthor=M.+J.+Wawerauthor=T.+Yoshinagaauthor=A.-M.+Zeemanauthor=V.+M.+Averyauthor=S.+N.+Bhatiaauthor=J.+E.+Burkeauthor=F.+Catterucciaauthor=J.+C.+Clardyauthor=P.+A.+Clemonsauthor=K.+J.+Decheringauthor=J.+R.+Duvallauthor=M.+A.+Foleyauthor=F.+Gusovskyauthor=C.+H.+M.+Kockenauthor=M.+Martiauthor=M.+L.+Morningstarauthor=B.+Munozauthor=D.+E.+Neafseyauthor=A.+Sharmaauthor=E.+A.+Winzelerauthor=D.+F.+Wirthauthor=C.+A.+Schererauthor=S.+L.+Schreiber&title=Diversity-Oriented+Synthesis+Yields+Novel+Multistage+Antimalarial+Inhibitors&doi=10.1038%2Fnature19804"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit105R"><div class="casContent"><span class="casTitleNuber">105</span><div class="casTitle"><span class="NLM_cas:atitle">Diversity-oriented synthesis yields novel multistage antimalarial inhibitors</span></div><div class="casAuthors">Kato, Nobutaka; Comer, Eamon; Sakata-Kato, Tomoyo; Sharma, Arvind; Sharma, Manmohan; Maetani, Micah; Bastien, Jessica; Brancucci, Nicolas M.; Bittker, Joshua A.; Corey, Victoria; Clarke, David; Derbyshire, Emily R.; Dornan, Gillian L.; Duffy, Sandra; Eckley, Sean; Itoe, Maurice A.; Koolen, Karin M. J.; Lewis, Timothy A.; Lui, Ping S.; Lukens, Amanda K.; Lund, Emily; March, Sandra; Meibalan, Elamaran; Meier, Bennett C.; McPhail, Jacob A.; Mitasev, Branko; Moss, Eli L.; Sayes, Morgane; Van Gessel, Yvonne; Wawer, Mathias J.; Yoshinaga, Takashi; Zeeman, Anne-Marie; Avery, Vicky M.; Bhatia, Sangeeta N.; Burke, John E.; Catteruccia, Flaminia; Clardy, Jon C.; Clemons, Paul A.; Dechering, Koen J.; Duvall, Jeremy R.; Foley, Michael A.; Gusovsky, Fabian; Kocken, Clemens H. M.; Marti, Matthias; Morningstar, Marshall L.; Munoz, Benito; Neafsey, Daniel E.; Sharma, Amit; Winzeler, Elizabeth A.; Wirth, Dyann F.; Scherer, Christina A.; Schreiber, Stuart L.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">538</span>
        (<span class="NLM_cas:issue">7625</span>),
    <span class="NLM_cas:pages">344-349</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Antimalarial drugs have thus far been chiefly derived from two sources-natural products and synthetic drug-like compds.  Here the authors investigate whether antimalarial agents with novel mechanisms of action could be discovered using a diverse collection of synthetic compds. that have three-dimensional features reminiscent of natural products and are underrepresented in typical screening collections.  The authors report the identification of such compds. with both previously reported and undescribed mechanisms of action, including a series of bicyclic azetidines that inhibit a new antimalarial target, phenylalanyl-tRNA synthetase.  These mols. are curative in mice at a single, low dose and show activity against all parasite life stages in multiple in vivo efficacy models.  The authors' findings identify bicyclic azetidines with the potential to both cure and prevent transmission of the disease as well as protect at-risk populations with a single oral dose, highlighting the strength of diversity-oriented synthesis in revealing promising therapeutic targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp9CdwKMqzmhLVg90H21EOLACvtfcHk0lhaeQsb5R-tUQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhslWhsLrI&md5=68e6e2018f2d49278d255ee02bfc8d16</span></div><a href="/servlet/linkout?suffix=cit105&amp;dbid=16384&amp;doi=10.1038%2Fnature19804&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature19804%26sid%3Dliteratum%253Aachs%26aulast%3DKato%26aufirst%3DN.%26aulast%3DComer%26aufirst%3DE.%26aulast%3DSakata-Kato%26aufirst%3DT.%26aulast%3DSharma%26aufirst%3DA.%26aulast%3DSharma%26aufirst%3DM.%26aulast%3DMaetani%26aufirst%3DM.%26aulast%3DBastien%26aufirst%3DJ.%26aulast%3DBrancucci%26aufirst%3DN.%2BM.%26aulast%3DBittker%26aufirst%3DJ.%2BA.%26aulast%3DCorey%26aufirst%3DV.%26aulast%3DClarke%26aufirst%3DD.%26aulast%3DDerbyshire%26aufirst%3DE.%2BR.%26aulast%3DDornan%26aufirst%3DG.%2BL.%26aulast%3DDuffy%26aufirst%3DS.%26aulast%3DEckley%26aufirst%3DS.%26aulast%3DItoe%26aufirst%3DM.%2BA.%26aulast%3DKoolen%26aufirst%3DK.%2BM.%2BJ.%26aulast%3DLewis%26aufirst%3DT.%2BA.%26aulast%3DLui%26aufirst%3DP.%2BS.%26aulast%3DLukens%26aufirst%3DA.%2BK.%26aulast%3DLund%26aufirst%3DE.%26aulast%3DMarch%26aufirst%3DS.%26aulast%3DMeibalan%26aufirst%3DE.%26aulast%3DMeier%26aufirst%3DB.%2BC.%26aulast%3DMcPhail%26aufirst%3DJ.%2BA.%26aulast%3DMitasev%26aufirst%3DB.%26aulast%3DMoss%26aufirst%3DE.%2BL.%26aulast%3DSayes%26aufirst%3DM.%26aulast%3DVan%2BGessel%26aufirst%3DY.%26aulast%3DWawer%26aufirst%3DM.%2BJ.%26aulast%3DYoshinaga%26aufirst%3DT.%26aulast%3DZeeman%26aufirst%3DA.-M.%26aulast%3DAvery%26aufirst%3DV.%2BM.%26aulast%3DBhatia%26aufirst%3DS.%2BN.%26aulast%3DBurke%26aufirst%3DJ.%2BE.%26aulast%3DCatteruccia%26aufirst%3DF.%26aulast%3DClardy%26aufirst%3DJ.%2BC.%26aulast%3DClemons%26aufirst%3DP.%2BA.%26aulast%3DDechering%26aufirst%3DK.%2BJ.%26aulast%3DDuvall%26aufirst%3DJ.%2BR.%26aulast%3DFoley%26aufirst%3DM.%2BA.%26aulast%3DGusovsky%26aufirst%3DF.%26aulast%3DKocken%26aufirst%3DC.%2BH.%2BM.%26aulast%3DMarti%26aufirst%3DM.%26aulast%3DMorningstar%26aufirst%3DM.%2BL.%26aulast%3DMunoz%26aufirst%3DB.%26aulast%3DNeafsey%26aufirst%3DD.%2BE.%26aulast%3DSharma%26aufirst%3DA.%26aulast%3DWinzeler%26aufirst%3DE.%2BA.%26aulast%3DWirth%26aufirst%3DD.%2BF.%26aulast%3DScherer%26aufirst%3DC.%2BA.%26aulast%3DSchreiber%26aufirst%3DS.%2BL.%26atitle%3DDiversity-Oriented%2520Synthesis%2520Yields%2520Novel%2520Multistage%2520Antimalarial%2520Inhibitors%26jtitle%3DNature%26date%3D2016%26volume%3D538%26spage%3D344%26epage%3D349%26doi%3D10.1038%2Fnature19804" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref106"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref106'); return false;" data-citation="" class="refNumLink">106</a></strong><div class="NLM_citation" id="cit106"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Parker, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galmozzi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Correia, B. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sasaki, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Joslyn, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cavallaro, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lawrence, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Narvaiza, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saez, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cravatt, B. F.</span></span> <span> </span><span class="NLM_article-title">Ligand and Target Discovery by Fragment-Based Screening in Human Cells</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>168</i></span>,  <span class="NLM_fpage">527.e29</span>– <span class="NLM_lpage">541.e29</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2016.12.029</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref106/cit106&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1016%2Fj.cell.2016.12.029" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=168&publication_year=2017&pages=527.e29-541.e29&author=C.+G.+Parkerauthor=A.+Galmozziauthor=Y.+Wangauthor=B.+E.+Correiaauthor=K.+Sasakiauthor=C.+M.+Joslynauthor=A.+S.+Kimauthor=C.+L.+Cavallaroauthor=R.+M.+Lawrenceauthor=S.+R.+Johnsonauthor=I.+Narvaizaauthor=E.+Saezauthor=B.+F.+Cravatt&title=Ligand+and+Target+Discovery+by+Fragment-Based+Screening+in+Human+Cells&doi=10.1016%2Fj.cell.2016.12.029"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit106&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2016.12.029&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2016.12.029%26sid%3Dliteratum%253Aachs%26aulast%3DParker%26aufirst%3DC.%2BG.%26aulast%3DGalmozzi%26aufirst%3DA.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DCorreia%26aufirst%3DB.%2BE.%26aulast%3DSasaki%26aufirst%3DK.%26aulast%3DJoslyn%26aufirst%3DC.%2BM.%26aulast%3DKim%26aufirst%3DA.%2BS.%26aulast%3DCavallaro%26aufirst%3DC.%2BL.%26aulast%3DLawrence%26aufirst%3DR.%2BM.%26aulast%3DJohnson%26aufirst%3DS.%2BR.%26aulast%3DNarvaiza%26aufirst%3DI.%26aulast%3DSaez%26aufirst%3DE.%26aulast%3DCravatt%26aufirst%3DB.%2BF.%26atitle%3DLigand%2520and%2520Target%2520Discovery%2520by%2520Fragment-Based%2520Screening%2520in%2520Human%2520Cells%26jtitle%3DCell%26date%3D2017%26volume%3D168%26spage%3D527.e29%26epage%3D541.e29%26doi%3D10.1016%2Fj.cell.2016.12.029" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref107"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref107'); return false;" data-citation="" class="refNumLink">107</a></strong><div class="NLM_citation" id="cit107"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wenlock, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Austin, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barton, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leeson, P. D.</span></span> <span> </span><span class="NLM_article-title">A Comparison of Physiochemical Property Profiles of Development and Marketed Oral Drugs</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>46</i></span>,  <span class="NLM_fpage">1250</span>– <span class="NLM_lpage">1256</span>, <span class="refDoi"> DOI: 10.1021/jm021053p</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm021053p" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref107/cit107&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BD3sXhslSlsbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2003&pages=1250-1256&author=M.+C.+Wenlockauthor=R.+P.+Austinauthor=P.+Bartonauthor=A.+M.+Davisauthor=P.+D.+Leeson&title=A+Comparison+of+Physiochemical+Property+Profiles+of+Development+and+Marketed+Oral+Drugs&doi=10.1021%2Fjm021053p"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit107R"><div class="casContent"><span class="casTitleNuber">107</span><div class="casTitle"><span class="NLM_cas:atitle">A Comparison of Physiochemical Property Profiles of Development and Marketed Oral Drugs</span></div><div class="casAuthors">Wenlock, Mark C.; Austin, Rupert P.; Barton, Patrick; Davis, Andrew M.; Leeson, Paul D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1250-1256</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The process of drug discovery applies rigorous selection pressures.  Marketed oral drugs will generally possess favorable physiochem. properties with respect to absorption, metab., distribution, and clearance.  This paper describes a study in which the distributions of physiochem. properties of oral drugs in different phases of clin. development are compared to those already marketed.  The aim is to identify the trends in physiochem. properties that favor a drug's successful passage through clin. development and on to the market.  Two libraries were created, one of current development oral drugs and one of marketed oral drugs.  Statistical anal. of the two showed that the mean mol. wt. of orally administered drugs in development decreases on passing through each of the different clin. phases and gradually converges toward the mean mol. wt. of marketed oral drugs.  It is also clear that the most lipophilic compds. are being discontinued from development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoF0PksUFJq0bVg90H21EOLACvtfcHk0lhaeQsb5R-tUQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXhslSlsbY%253D&md5=b9809c803a61ec99a1b4171e97952452</span></div><a href="/servlet/linkout?suffix=cit107&amp;dbid=16384&amp;doi=10.1021%2Fjm021053p&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm021053p%26sid%3Dliteratum%253Aachs%26aulast%3DWenlock%26aufirst%3DM.%2BC.%26aulast%3DAustin%26aufirst%3DR.%2BP.%26aulast%3DBarton%26aufirst%3DP.%26aulast%3DDavis%26aufirst%3DA.%2BM.%26aulast%3DLeeson%26aufirst%3DP.%2BD.%26atitle%3DA%2520Comparison%2520of%2520Physiochemical%2520Property%2520Profiles%2520of%2520Development%2520and%2520Marketed%2520Oral%2520Drugs%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2003%26volume%3D46%26spage%3D1250%26epage%3D1256%26doi%3D10.1021%2Fjm021053p" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref108"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref108'); return false;" data-citation="" class="refNumLink">108</a></strong><div class="NLM_citation" id="cit108"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Leeson, P.
D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">St-Gallay, S. A.</span></span> <span> </span><span class="NLM_article-title">The Influence of the “Organizational Factor” on Compound Quality in Drug Discovery</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">749</span>– <span class="NLM_lpage">765</span>, <span class="refDoi"> DOI: 10.1038/nrd3552</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref108/cit108&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1038%2Fnrd3552" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2011&pages=749-765&author=P.%0AD.+Leesonauthor=S.+A.+St-Gallay&title=The+Influence+of+the+%E2%80%9COrganizational+Factor%E2%80%9D+on+Compound+Quality+in+Drug+Discovery&doi=10.1038%2Fnrd3552"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit108&amp;dbid=16384&amp;doi=10.1038%2Fnrd3552&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd3552%26sid%3Dliteratum%253Aachs%26aulast%3DLeeson%26aufirst%3DP.%2BD.%26aulast%3DSt-Gallay%26aufirst%3DS.%2BA.%26atitle%3DThe%2520Influence%2520of%2520the%2520%25E2%2580%259COrganizational%2520Factor%25E2%2580%259D%2520on%2520Compound%2520Quality%2520in%2520Drug%2520Discovery%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2011%26volume%3D10%26spage%3D749%26epage%3D765%26doi%3D10.1038%2Fnrd3552" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref109"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref109'); return false;" data-citation="" class="refNumLink">109</a></strong><div class="NLM_citation" id="cit109"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cox, P. B.</span>; <span class="NLM_string-name">Vasudevan, A.</span></span> <span> </span><span class="NLM_article-title">Generating a High-Quality Compound Collection</span>. In  <i>Management of Chemical and Biological Samples for Screening Applications</i>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wigglesworth, M.</span>, <span class="NLM_string-name">Wood, T.</span></span>, Eds.; <span class="NLM_publisher-name">Wiley-VCH Verlag GmbH & Co. KGaA</span>: <span class="NLM_publisher-loc">Weinheim, Germany</span>, <span class="NLM_year">2012</span>; pp  <span class="NLM_fpage">9</span>– <span class="NLM_lpage">21</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref109/cit109&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1002%2F9783527645251.ch2" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2012&pages=9-21&author=P.+B.+Cox&author=A.+Vasudevanauthor=M.+Wigglesworth&author=T.+Wood&title=Management+of+Chemical+and+Biological+Samples+for+Screening+Applications"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit109&amp;dbid=16384&amp;doi=10.1002%2F9783527645251.ch2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1002%252F9783527645251.ch2%26sid%3Dliteratum%253Aachs%26aulast%3DCox%26aufirst%3DP.%2BB.%26atitle%3DGenerating%2520a%2520High-Quality%2520Compound%2520Collection%26btitle%3DManagement%2520of%2520Chemical%2520and%2520Biological%2520Samples%2520for%2520Screening%2520Applications%26aulast%3DWigglesworth%26aufirst%3DM.%26pub%3DWiley-VCH%2520Verlag%2520GmbH%2520%2526%2520Co.%2520KGaA%26date%3D2012%26spage%3D9%26epage%3D21" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref110"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref110'); return false;" data-citation="" class="refNumLink">110</a></strong><div class="NLM_citation" id="cit110"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rankovic, Z.</span>; <span class="NLM_string-name">Jamieson, C.</span>; <span class="NLM_string-name">Morphy, R.</span></span> <span> </span><span class="NLM_article-title">High Throughput Screening Approach to Lead Discovery</span>. In  <i>Lead Generation Approaches in Drug Discovery</i>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rankovic, Z.</span>, <span class="NLM_string-name">Morphy, R.</span></span>, Eds.; <span class="NLM_publisher-name">John Wiley & Sons, Inc.</span>: <span class="NLM_publisher-loc">Hoboken, NJ</span>., <span class="NLM_year">2010</span>; pp  <span class="NLM_fpage">21</span>– <span class="NLM_lpage">71</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref110/cit110&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1002%2F9780470584170.ch2" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2010&pages=21-71&author=Z.+Rankovic&author=C.+Jamieson&author=R.+Morphyauthor=Z.+Rankovic&author=R.+Morphy&title=Lead+Generation+Approaches+in+Drug+Discovery"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit110&amp;dbid=16384&amp;doi=10.1002%2F9780470584170.ch2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1002%252F9780470584170.ch2%26sid%3Dliteratum%253Aachs%26aulast%3DRankovic%26aufirst%3DZ.%26atitle%3DHigh%2520Throughput%2520Screening%2520Approach%2520to%2520Lead%2520Discovery%26btitle%3DLead%2520Generation%2520Approaches%2520in%2520Drug%2520Discovery%26aulast%3DRankovic%26aufirst%3DZ.%26pub%3DJohn%2520Wiley%2520%2526%2520Sons%252C%2520Inc%26date%3D2010%26spage%3D21%26epage%3D71" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref111"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref111'); return false;" data-citation="" class="refNumLink">111</a></strong><div class="NLM_citation" id="cit111"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kalliokoski, T.</span></span> <span> </span><span class="NLM_article-title">Price-Focused Analysis of Commercially Available Building Blocks for Combinatorial Library Synthesis</span>. <i>ACS Comb. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">600</span>– <span class="NLM_lpage">607</span>, <span class="refDoi"> DOI: 10.1021/acscombsci.5b00063</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acscombsci.5b00063" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref111/cit111&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsV2mtLrK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2015&pages=600-607&author=T.+Kalliokoski&title=Price-Focused+Analysis+of+Commercially+Available+Building+Blocks+for+Combinatorial+Library+Synthesis&doi=10.1021%2Facscombsci.5b00063"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit111R"><div class="casContent"><span class="casTitleNuber">111</span><div class="casTitle"><span class="NLM_cas:atitle">Price-Focused Analysis of Commercially Available Building Blocks for Combinatorial Library Synthesis</span></div><div class="casAuthors">Kalliokoski, Tuomo</div><div class="citationInfo"><span class="NLM_cas:title">ACS Combinatorial Science</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">600-607</span>CODEN:
                <span class="NLM_cas:coden">ACSCCC</span>;
        ISSN:<span class="NLM_cas:issn">2156-8944</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Combinatorial libraries are synthesized by combining smaller reagents (building blocks), the price of which is an important component of the total costs assocd. with the synthetic exercise.  A significant portion of com. available reagents are too expensive for large scale work.  In this study, 13 commonly used reagent classes in combinatorial library synthesis (primary and secondary amines, carboxylic acids, alcs., ketones, aldehydes, boronic acids, acyl halides, sulfonyl chlorides, isocyanates, isothiocyanates, azides and chloroformates) were analyzed with respect to the cost, physicochem. properties (mol. wt. and calcd. logP), chem. diversity, and 3D-likeness using a large data set.  The results define the chem. space accessible under a constraint of limited financial resources.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrlW1WcWLiiE7Vg90H21EOLACvtfcHk0li8ZeVYVxRKeQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsV2mtLrK&md5=a7f8c2a12a14ecb81a228c478c09ef06</span></div><a href="/servlet/linkout?suffix=cit111&amp;dbid=16384&amp;doi=10.1021%2Facscombsci.5b00063&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facscombsci.5b00063%26sid%3Dliteratum%253Aachs%26aulast%3DKalliokoski%26aufirst%3DT.%26atitle%3DPrice-Focused%2520Analysis%2520of%2520Commercially%2520Available%2520Building%2520Blocks%2520for%2520Combinatorial%2520Library%2520Synthesis%26jtitle%3DACS%2520Comb.%2520Sci.%26date%3D2015%26volume%3D17%26spage%3D600%26epage%3D607%26doi%3D10.1021%2Facscombsci.5b00063" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref112"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref112'); return false;" data-citation="" class="refNumLink">112</a></strong><div class="NLM_citation" id="cit112"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Stocks, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilden, G. R. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pairaudeau, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perry, M. W. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steele, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stonehouse, J. P.</span></span> <span> </span><span class="NLM_article-title">A Practical Method for Targeted Library Design Balancing Lead-like Properties with Diversity</span>. <i>ChemMedChem</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">800</span>– <span class="NLM_lpage">808</span>, <span class="refDoi"> DOI: 10.1002/cmdc.200900050</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref112/cit112&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1002%2Fcmdc.200900050" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2009&pages=800-808&author=M.+J.+Stocksauthor=G.+R.+H.+Wildenauthor=G.+Pairaudeauauthor=M.+W.+D.+Perryauthor=J.+Steeleauthor=J.+P.+Stonehouse&title=A+Practical+Method+for+Targeted+Library+Design+Balancing+Lead-like+Properties+with+Diversity&doi=10.1002%2Fcmdc.200900050"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit112&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.200900050&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.200900050%26sid%3Dliteratum%253Aachs%26aulast%3DStocks%26aufirst%3DM.%2BJ.%26aulast%3DWilden%26aufirst%3DG.%2BR.%2BH.%26aulast%3DPairaudeau%26aufirst%3DG.%26aulast%3DPerry%26aufirst%3DM.%2BW.%2BD.%26aulast%3DSteele%26aufirst%3DJ.%26aulast%3DStonehouse%26aufirst%3DJ.%2BP.%26atitle%3DA%2520Practical%2520Method%2520for%2520Targeted%2520Library%2520Design%2520Balancing%2520Lead-like%2520Properties%2520with%2520Diversity%26jtitle%3DChemMedChem%26date%3D2009%26volume%3D4%26spage%3D800%26epage%3D808%26doi%3D10.1002%2Fcmdc.200900050" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref113"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref113'); return false;" data-citation="" class="refNumLink">113</a></strong><div class="NLM_citation" id="cit113"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kallus, C.</span>; <span class="NLM_string-name">Hüser, J.</span>; <span class="NLM_string-name">Jones, P.
S.</span>; <span class="NLM_string-name">Nelson, A.</span></span> <span> </span><span class="NLM_article-title">The European Lead Factory</span>. In  <i>Small Molecule Medicinal Chemistry</i>; <span class="NLM_contrib-group"><span class="NLM_string-name">Czechtizky, W.</span>, <span class="NLM_string-name">Hamley, P.</span></span>, Eds.; <span class="NLM_publisher-name">John Wiley & Sons, Inc</span>: <span class="NLM_publisher-loc">Hoboken, NJ</span>., <span class="NLM_year">2015</span>; pp  <span class="NLM_fpage">37</span>– <span class="NLM_lpage">60</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref113/cit113&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1002%2F9781118771723.ch2" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2015&pages=37-60&author=C.+Kallus&author=J.+H%C3%BCser&author=P.%0AS.+Jones&author=A.+Nelsonauthor=W.+Czechtizky&author=P.+Hamley&title=Small+Molecule+Medicinal+Chemistry"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit113&amp;dbid=16384&amp;doi=10.1002%2F9781118771723.ch2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1002%252F9781118771723.ch2%26sid%3Dliteratum%253Aachs%26aulast%3DKallus%26aufirst%3DC.%26atitle%3DThe%2520European%2520Lead%2520Factory%26btitle%3DSmall%2520Molecule%2520Medicinal%2520Chemistry%26aulast%3DCzechtizky%26aufirst%3DW.%26pub%3DJohn%2520Wiley%2520%2526%2520Sons%252C%2520Inc%26date%3D2015%26spage%3D37%26epage%3D60" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref114"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref114'); return false;" data-citation="" class="refNumLink">114</a></strong><div class="NLM_citation" id="cit114"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Karawajczyk, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giordanetto, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Benningshof, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamza, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kalliokoski, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pouwer, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morgentin, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nelson, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Piechot, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tzalis, D.</span></span> <span> </span><span class="NLM_article-title">Expansion of Chemical Space for Collaborative Lead Generation and Drug Discovery: The European Lead Factory Perspective. Drug Discov</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">1310</span>– <span class="NLM_lpage">1316</span>, <span class="refDoi"> DOI: 10.1016/j.drudis.2015.09.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref114/cit114&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1016%2Fj.drudis.2015.09.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref114/cit114&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=26429298" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref114/cit114&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhs1aktLnM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2015&pages=1310-1316&author=A.+Karawajczykauthor=F.+Giordanettoauthor=J.+Benningshofauthor=D.+Hamzaauthor=T.+Kalliokoskiauthor=K.+Pouwerauthor=R.+Morgentinauthor=A.+Nelsonauthor=G.+M%C3%BCllerauthor=A.+Piechotauthor=D.+Tzalis&title=Expansion+of+Chemical+Space+for+Collaborative+Lead+Generation+and+Drug+Discovery%3A+The+European+Lead+Factory+Perspective.+Drug+Discov&doi=10.1016%2Fj.drudis.2015.09.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit114R"><div class="casContent"><span class="casTitleNuber">114</span><div class="casTitle"><span class="NLM_cas:atitle">Expansion of chemical space for collaborative lead generation and drug discovery: the European Lead Factory Perspective</span></div><div class="casAuthors">Karawajczyk, Anna; Giordanetto, Fabrizio; Benningshof, Jorg; Hamza, Daniel; Kalliokoski, Tuomo; Pouwer, Kees; Morgentin, Remy; Nelson, Adam; Mueller, Gerhard; Piechot, Alexander; Tzalis, Dimitrios</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1310-1316</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">High-throughput screening (HTS) represents a major cornerstone of drug discovery.  The availability of an innovative, relevant and high-quality compd. collection to be screened often dictates the final fate of a drug discovery campaign.  Given that the chem. space to be sampled in research programs is practically infinite and sparsely populated, significant efforts and resources need to be invested in the generation and maintenance of a competitive compd. collection.  The European Lead Factory (ELF) project is addressing this challenge by leveraging the diverse experience and know-how of academic groups and small and medium enterprises (SMEs) engaged in synthetic and/or medicinal chem.  Here, we describe the novelty, diversity, structural complexity, physicochem. characteristics and overall attractiveness of this first batch of ELF compds. for HTS purposes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpkcY_bnknKv7Vg90H21EOLACvtfcHk0lgVV32Jy-ly0g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhs1aktLnM&md5=7a3bc7e49c3c9ff7d92c238295a62143</span></div><a href="/servlet/linkout?suffix=cit114&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2015.09.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2015.09.009%26sid%3Dliteratum%253Aachs%26aulast%3DKarawajczyk%26aufirst%3DA.%26aulast%3DGiordanetto%26aufirst%3DF.%26aulast%3DBenningshof%26aufirst%3DJ.%26aulast%3DHamza%26aufirst%3DD.%26aulast%3DKalliokoski%26aufirst%3DT.%26aulast%3DPouwer%26aufirst%3DK.%26aulast%3DMorgentin%26aufirst%3DR.%26aulast%3DNelson%26aufirst%3DA.%26aulast%3DM%25C3%25BCller%26aufirst%3DG.%26aulast%3DPiechot%26aufirst%3DA.%26aulast%3DTzalis%26aufirst%3DD.%26atitle%3DExpansion%2520of%2520Chemical%2520Space%2520for%2520Collaborative%2520Lead%2520Generation%2520and%2520Drug%2520Discovery%253A%2520The%2520European%2520Lead%2520Factory%2520Perspective.%2520Drug%2520Discov%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2015%26volume%3D20%26spage%3D1310%26epage%3D1316%26doi%3D10.1016%2Fj.drudis.2015.09.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref115"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref115'); return false;" data-citation="" class="refNumLink">115</a></strong><div class="NLM_citation" id="cit115"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Medina-Franco, J. L.</span></span> <span> </span><span class="NLM_article-title">Chemoinformatic Characterization of the Chemical Space and Molecular Diversity of Compound Libraries</span>. In  <i>Diversity-Oriented Synthesis</i>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trabocchi, A.</span></span>, Ed.; <span class="NLM_publisher-name">John Wiley & Sons, Inc.</span>: <span class="NLM_publisher-loc">Hoboken, NJ</span>, <span class="NLM_year">2013</span>; pp  <span class="NLM_fpage">325</span>– <span class="NLM_lpage">352</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref115/cit115&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1002%2F9781118618110.ch10" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2013&pages=325-352&author=J.+L.+Medina-Francoauthor=A.+Trabocchi&title=Diversity-Oriented+Synthesis"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit115&amp;dbid=16384&amp;doi=10.1002%2F9781118618110.ch10&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1002%252F9781118618110.ch10%26sid%3Dliteratum%253Aachs%26aulast%3DMedina-Franco%26aufirst%3DJ.%2BL.%26atitle%3DChemoinformatic%2520Characterization%2520of%2520the%2520Chemical%2520Space%2520and%2520Molecular%2520Diversity%2520of%2520Compound%2520Libraries%26btitle%3DDiversity-Oriented%2520Synthesis%26aulast%3DTrabocchi%26aufirst%3DA.%26pub%3DJohn%2520Wiley%2520%2526%2520Sons%252C%2520Inc%26date%3D2013%26spage%3D325%26epage%3D352" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref116"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref116'); return false;" data-citation="" class="refNumLink">116</a></strong><div class="NLM_citation" id="cit116"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mullin, R.</span></span> <span> </span><span class="NLM_article-title">AstraZeneca, Sanofi Exchange Compounds</span>. <i>Chem. Eng. News</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>93</i></span>,  <span class="NLM_fpage">4</span>, <span class="refDoi"> DOI: 10.1021/cen-09347-notw3</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cen-09347-notw3" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=93&publication_year=2015&pages=4&author=R.+Mullin&title=AstraZeneca%2C+Sanofi+Exchange+Compounds&doi=10.1021%2Fcen-09347-notw3"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit116&amp;dbid=16384&amp;doi=10.1021%2Fcen-09347-notw3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcen-09347-notw3%26sid%3Dliteratum%253Aachs%26aulast%3DMullin%26aufirst%3DR.%26atitle%3DAstraZeneca%252C%2520Sanofi%2520Exchange%2520Compounds%26jtitle%3DChem.%2520Eng.%2520News%26date%3D2015%26volume%3D93%26spage%3D4%26doi%3D10.1021%2Fcen-09347-notw3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref117"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref117'); return false;" data-citation="" class="refNumLink">117</a></strong><div class="NLM_citation" id="cit117"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ten Holte, P.</span></span> <span> </span><span class="NLM_article-title">Access to Compound Collections</span>. In  <i>Small Molecule Medicinal Chemistry</i>; <span class="NLM_contrib-group"><span class="NLM_string-name">Czechtizky, W.</span>, <span class="NLM_string-name">Hamley, P.</span></span>, Eds.; <span class="NLM_publisher-name">John Wiley & Sons, Inc</span>: <span class="NLM_publisher-loc">Hoboken, NJ</span>, <span class="NLM_year">2015</span>; pp  <span class="NLM_fpage">61</span>– <span class="NLM_lpage">76</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref117/cit117&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1002%2F9781118771723.ch3" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2015&pages=61-76&author=P.+Ten+Holteauthor=W.+Czechtizky&author=P.+Hamley&title=Small+Molecule+Medicinal+Chemistry"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit117&amp;dbid=16384&amp;doi=10.1002%2F9781118771723.ch3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1002%252F9781118771723.ch3%26sid%3Dliteratum%253Aachs%26aulast%3DTen%2BHolte%26aufirst%3DP.%26atitle%3DAccess%2520to%2520Compound%2520Collections%26btitle%3DSmall%2520Molecule%2520Medicinal%2520Chemistry%26aulast%3DCzechtizky%26aufirst%3DW.%26pub%3DJohn%2520Wiley%2520%2526%2520Sons%252C%2520Inc%26date%3D2015%26spage%3D61%26epage%3D76" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref118"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref118'); return false;" data-citation="" class="refNumLink">118</a></strong><div class="NLM_citation" id="cit118"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kogej, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blomberg, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greasley, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mundt, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vainio, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schamberger, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmidt, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hüser, J.</span></span> <span> </span><span class="NLM_article-title">Big Pharma Screening Collections: More of the Same or Unique Libraries? The AstraZeneca–Bayer Pharma AG Case. Drug Discov</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">1014</span>– <span class="NLM_lpage">1024</span>, <span class="refDoi"> DOI: 10.1016/j.drudis.2012.10.011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref118/cit118&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1016%2Fj.drudis.2012.10.011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref118/cit118&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=23127858" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref118/cit118&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A280%3ADC%252BC3s7htVeqtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2013&pages=1014-1024&author=T.+Kogejauthor=N.+Blombergauthor=P.+J.+Greasleyauthor=S.+Mundtauthor=M.+J.+Vainioauthor=J.+Schambergerauthor=G.+Schmidtauthor=J.+H%C3%BCser&title=Big+Pharma+Screening+Collections%3A+More+of+the+Same+or+Unique+Libraries%3F+The+AstraZeneca%E2%80%93Bayer+Pharma+AG+Case.+Drug+Discov&doi=10.1016%2Fj.drudis.2012.10.011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit118R"><div class="casContent"><span class="casTitleNuber">118</span><div class="casTitle"><span class="NLM_cas:atitle">Big pharma screening collections: more of the same or unique libraries? The AstraZeneca-Bayer Pharma AG case</span></div><div class="casAuthors">Kogej Thierry; Blomberg Niklas; Greasley Peter J; Mundt Stefan; Vainio Mikko J; Schamberger Jens; Schmidt Georg; Huser Jorg</div><div class="citationInfo"><span class="NLM_cas:title">Drug discovery today</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">19-20</span>),
    <span class="NLM_cas:pages">1014-24</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">In this study, the screening collections of two major pharmaceutical companies (AstraZeneca and Bayer Pharma AG) have been compared using a 2D molecular fingerprint by a nearest neighborhood approach.  Results revealed a low overlap between both collections in terms of compound identity and similarity.  This emphasizes the value of screening multiple compound collections to expand the chemical space that can be accessed by high-throughput screening (HTS).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQJl8Dg2rrY8XO4gDb7JggCfW6udTcc2ea6vxhSXWT6yLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3s7htVeqtw%253D%253D&md5=86eb921c384773e2a9a6a2017cbef5a1</span></div><a href="/servlet/linkout?suffix=cit118&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2012.10.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2012.10.011%26sid%3Dliteratum%253Aachs%26aulast%3DKogej%26aufirst%3DT.%26aulast%3DBlomberg%26aufirst%3DN.%26aulast%3DGreasley%26aufirst%3DP.%2BJ.%26aulast%3DMundt%26aufirst%3DS.%26aulast%3DVainio%26aufirst%3DM.%2BJ.%26aulast%3DSchamberger%26aufirst%3DJ.%26aulast%3DSchmidt%26aufirst%3DG.%26aulast%3DH%25C3%25BCser%26aufirst%3DJ.%26atitle%3DBig%2520Pharma%2520Screening%2520Collections%253A%2520More%2520of%2520the%2520Same%2520or%2520Unique%2520Libraries%253F%2520The%2520AstraZeneca%25E2%2580%2593Bayer%2520Pharma%2520AG%2520Case.%2520Drug%2520Discov%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2013%26volume%3D18%26spage%3D1014%26epage%3D1024%26doi%3D10.1016%2Fj.drudis.2012.10.011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref119"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref119'); return false;" data-citation="" class="refNumLink">119</a></strong><div class="NLM_citation" id="cit119"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ramsay, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kwasnoski, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caldwell, G. W.</span></span> <span> </span><span class="NLM_article-title">Small Molecule Compound Logistics Outsourcing–Going beyond the “Thought Experiment.</span>. <i>Curr. Top. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">1298</span>– <span class="NLM_lpage">1308</span>, <span class="refDoi"> DOI: 10.2174/156802612800672826</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref119/cit119&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.2174%2F156802612800672826" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref119/cit119&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=22571792" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref119/cit119&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC38XovFers74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2012&pages=1298-1308&author=D.+L.+Ramsayauthor=J.+D.+Kwasnoskiauthor=G.+W.+Caldwell&title=Small+Molecule+Compound+Logistics+Outsourcing%E2%80%93Going+beyond+the+%E2%80%9CThought+Experiment.&doi=10.2174%2F156802612800672826"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit119R"><div class="casContent"><span class="casTitleNuber">119</span><div class="casTitle"><span class="NLM_cas:atitle">Small molecule compound logistics outsourcing - going beyond the "thought experiment"</span></div><div class="casAuthors">Ramsay, Devon L.; Kwasnoski, Joseph D.; Caldwell, Gary W.</div><div class="citationInfo"><span class="NLM_cas:title">Current Topics in Medicinal Chemistry (Sharjah, United Arab Emirates)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1298-1308</span>CODEN:
                <span class="NLM_cas:coden">CTMCCL</span>;
        ISSN:<span class="NLM_cas:issn">1568-0266</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Increasing pressure on the pharmaceutical industry to reduce cost and focus internal resources on "high value" activities is driving a trend to outsource traditionally "inhouse" drug discovery activities.  Compd. collections are typically viewed as drug discovery's "crown jewels"; however, in late 2007, Johnson & Johnson Pharmaceutical Research & Development (J&J PRD) took a bold step to move their entire North American compd. inventory and processing capability to an external third party vendor.  The authors discuss the combination model implemented, that of local compd. logistics site support with an outsourced centralized processing center.  Some of the lessons learned over the past five years were predictable while others were unexpected.  The substantial cost savings, improved local service response and flexible platform to adjust to changing business needs resulted.  Continued sustainable success relies heavily upon maintaining internal headcount dedicated to vendor management, an open collaboration approach and a solid information technol. infrastructure with complete transparency and visibility.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpB941ZIlLXkbVg90H21EOLACvtfcHk0lgVV32Jy-ly0g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XovFers74%253D&md5=b69c0ba5d07fa83ad1ad1a897a1f13a2</span></div><a href="/servlet/linkout?suffix=cit119&amp;dbid=16384&amp;doi=10.2174%2F156802612800672826&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F156802612800672826%26sid%3Dliteratum%253Aachs%26aulast%3DRamsay%26aufirst%3DD.%2BL.%26aulast%3DKwasnoski%26aufirst%3DJ.%2BD.%26aulast%3DCaldwell%26aufirst%3DG.%2BW.%26atitle%3DSmall%2520Molecule%2520Compound%2520Logistics%2520Outsourcing%25E2%2580%2593Going%2520beyond%2520the%2520%25E2%2580%259CThought%2520Experiment.%26jtitle%3DCurr.%2520Top.%2520Med.%2520Chem.%26date%3D2012%26volume%3D12%26spage%3D1298%26epage%3D1308%26doi%3D10.2174%2F156802612800672826" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref120"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref120'); return false;" data-citation="" class="refNumLink">120</a></strong><div class="NLM_citation" id="cit120"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mahuteau-Betzer, F.</span></span> <span> </span><span class="NLM_article-title">The French National Compound Library: advances and future prospects</span>. <i>Med. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>31</i></span>,  <span class="NLM_fpage">417</span>– <span class="NLM_lpage">422</span>, <span class="refDoi"> DOI: 10.1051/medsci/20153104016</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref120/cit120&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1051%2Fmedsci%2F20153104016" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref120/cit120&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A280%3ADC%252BC2Mfis1Cjuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2015&pages=417-422&author=F.+Mahuteau-Betzer&title=The+French+National+Compound+Library%3A+advances+and+future+prospects&doi=10.1051%2Fmedsci%2F20153104016"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit120R"><div class="casContent"><span class="casTitleNuber">120</span><div class="casTitle"><span class="NLM_cas:atitle">The French National Compound Library: advances and future prospects</span></div><div class="casAuthors">Mahuteau-Betzer Florence</div><div class="citationInfo"><span class="NLM_cas:title">Medecine sciences : M/S</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">417-22</span>
        ISSN:<span class="NLM_cas:issn">0767-0974</span>.
    </div><div class="casAbstract">The French National Compound Library (Chimiotheque Nationale) has been created in 2003 and is the federation of local collections.  It contains more than 56 000 small molecules and natural compounds synthesised or isolated in different laboratories over the past years.  This explains the diversity of the collection.  The strength of this initiative is the ability to connect chemists and biologists for the development of hits.  This development involves the synthesis of analogues or/and chemical tools to find new targets.  These collaborations lead to the identification of new chemical probes.  These probes able to modulate a biological function are essential to study biological pathways.  They can also be useful for therapeutic applications.  This article will describe the major achievements and perspectives of the French Chemical Library.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTjzt2Wqi1XWxgd2Wuil24tfW6udTcc2eaHhq5rbkPdJrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2Mfis1Cjuw%253D%253D&md5=db4eaa72173176a4393c72ad8316365e</span></div><a href="/servlet/linkout?suffix=cit120&amp;dbid=16384&amp;doi=10.1051%2Fmedsci%2F20153104016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1051%252Fmedsci%252F20153104016%26sid%3Dliteratum%253Aachs%26aulast%3DMahuteau-Betzer%26aufirst%3DF.%26atitle%3DThe%2520French%2520National%2520Compound%2520Library%253A%2520advances%2520and%2520future%2520prospects%26jtitle%3DMed.%2520Sci.%26date%3D2015%26volume%3D31%26spage%3D417%26epage%3D422%26doi%3D10.1051%2Fmedsci%2F20153104016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref121"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref121'); return false;" data-citation="" class="refNumLink">121</a></strong><div class="NLM_citation" id="cit121"><span> <i>Sanofi Signe un Partenariat avec le CNRS et Vingt-Six
Membres de
la Chimiothèque Nationale [Sanofi Signs a Partnerships with
CNRS and Twenty-Six Members of the French Nationale Compound Bank]</i>; <span class="NLM_publisher-name">Chimiotheque Nationale</span>, October
6, <span class="NLM_year">2015</span>; <a href="http://chimiotheque-nationale.cn.cnrs.fr" class="extLink">http://chimiotheque-nationale.cn.cnrs.fr</a> (accessed February 16, 2018].</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+Sanofi+Signe+un+Partenariat+avec+le+CNRS+et+Vingt-Six%0AMembres+de%0Ala+Chimioth%C3%A8que+Nationale+%5BSanofi+Signs+a+Partnerships+with%0ACNRS+and+Twenty-Six+Members+of+the+French+Nationale+Compound+Bank%5D%3B+Chimiotheque+Nationale%2C+October%0A6%2C+2015%3B+http%3A%2F%2Fchimiotheque-nationale.cn.cnrs.fr+%28accessed+February+16%2C+2018%5D."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit121&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26jtitle%3DSanofi%2520Signe%2520un%2520Partenariat%2520avec%2520le%2520CNRS%2520et%2520Vingt-Six%250AMembres%2520de%250Ala%2520Chimioth%25C3%25A8que%2520Nationale%2520%255BSanofi%2520Signs%2520a%2520Partnerships%2520with%250ACNRS%2520and%2520Twenty-Six%2520Members%2520of%2520the%2520French%2520Nationale%2520Compound%2520Bank%255D%26pub%3DChimiotheque%2520Nationale%26date%3D2015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref122"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref122'); return false;" data-citation="" class="refNumLink">122</a></strong><div class="NLM_citation" id="cit122"><span> <i>Seeking Partners
for Macrocycle Drug Discovery (Challenge ID: 9933642)</i>; <span class="NLM_publisher-name">Innocentive</span>, February 5, <span class="NLM_year">2015</span>; <a href="https://www.innocentive.com/ar/challenge/9933642" class="extLink">https://www.innocentive.com/ar/challenge/9933642</a> (accessed February 16, 2018).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+Seeking+Partners%0Afor+Macrocycle+Drug+Discovery+%28Challenge+ID%3A+9933642%29%3B+Innocentive%2C+February+5%2C+2015%3B+https%3A%2F%2Fwww.innocentive.com%2Far%2Fchallenge%2F9933642+%28accessed+February+16%2C+2018%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit122&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26jtitle%3DSeeking%2520Partners%250Afor%2520Macrocycle%2520Drug%2520Discovery%2520%2528Challenge%2520ID%253A%25209933642%2529%26pub%3DInnocentive%26date%3D2015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref123"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref123'); return false;" data-citation="" class="refNumLink">123</a></strong><div class="NLM_citation" id="cit123"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Price, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Howard, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cons, B. D.</span></span> <span> </span><span class="NLM_article-title">Fragment-Based Drug Discovery and Its Application to Challenging Drug Targets</span>. <i>Essays Biochem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">475</span>– <span class="NLM_lpage">484</span>, <span class="refDoi"> DOI: 10.1042/EBC20170029</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref123/cit123&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1042%2FEBC20170029" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref123/cit123&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=29118094" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref123/cit123&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A280%3ADC%252BC1M7pt1GmtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2017&pages=475-484&author=A.+J.+Priceauthor=S.+Howardauthor=B.+D.+Cons&title=Fragment-Based+Drug+Discovery+and+Its+Application+to+Challenging+Drug+Targets&doi=10.1042%2FEBC20170029"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit123R"><div class="casContent"><span class="casTitleNuber">123</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment-based drug discovery and its application to challenging drug targets</span></div><div class="casAuthors">Price Amanda J; Howard Steven; Cons Benjamin D</div><div class="citationInfo"><span class="NLM_cas:title">Essays in biochemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">475-484</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Fragment-based drug discovery (FBDD) is a technique for identifying low molecular weight chemical starting points for drug discovery.  Since its inception 20 years ago, FBDD has grown in popularity to the point where it is now an established technique in industry and academia.  The approach involves the biophysical screening of proteins against collections of low molecular weight compounds (fragments).  Although fragments bind to proteins with relatively low affinity, they form efficient, high quality binding interactions with the protein architecture as they have to overcome a significant entropy barrier to bind.  Of the biophysical methods available for fragment screening, X-ray protein crystallography is one of the most sensitive and least prone to false positives.  It also provides detailed structural information of the protein-fragment complex at the atomic level.  Fragment-based screening using X-ray crystallography is therefore an efficient method for identifying binding hotspots on proteins, which can then be exploited by chemists and biologists for the discovery of new drugs.  The use of FBDD is illustrated here with a recently published case study of a drug discovery programme targeting the challenging protein-protein interaction Kelch-like ECH-associated protein 1:nuclear factor erythroid 2-related factor 2.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTqZGQ3bfUY0VHFYivNCdPRfW6udTcc2eaHhq5rbkPdJrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1M7pt1GmtA%253D%253D&md5=f8572516bfb1ecc4a0ef57b0c5ba1bc0</span></div><a href="/servlet/linkout?suffix=cit123&amp;dbid=16384&amp;doi=10.1042%2FEBC20170029&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252FEBC20170029%26sid%3Dliteratum%253Aachs%26aulast%3DPrice%26aufirst%3DA.%2BJ.%26aulast%3DHoward%26aufirst%3DS.%26aulast%3DCons%26aufirst%3DB.%2BD.%26atitle%3DFragment-Based%2520Drug%2520Discovery%2520and%2520Its%2520Application%2520to%2520Challenging%2520Drug%2520Targets%26jtitle%3DEssays%2520Biochem.%26date%3D2017%26volume%3D61%26spage%3D475%26epage%3D484%26doi%3D10.1042%2FEBC20170029" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref124"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref124'); return false;" data-citation="" class="refNumLink">124</a></strong><div class="NLM_citation" id="cit124"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lamoree, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hubbard, R. E.</span></span> <span> </span><span class="NLM_article-title">Current Perspectives in Fragment-Based Lead Discovery (FBLD)</span>. <i>Essays Biochem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">453</span>– <span class="NLM_lpage">464</span>, <span class="refDoi"> DOI: 10.1042/EBC20170028</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref124/cit124&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1042%2FEBC20170028" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref124/cit124&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=29118093" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref124/cit124&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A280%3ADC%252BC1M7pt1Gmtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2017&pages=453-464&author=B.+Lamoreeauthor=R.+E.+Hubbard&title=Current+Perspectives+in+Fragment-Based+Lead+Discovery+%28FBLD%29&doi=10.1042%2FEBC20170028"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit124R"><div class="casContent"><span class="casTitleNuber">124</span><div class="casTitle"><span class="NLM_cas:atitle">Current perspectives in fragment-based lead discovery (FBLD)</span></div><div class="casAuthors">Lamoree Bas; Hubbard Roderick E; Hubbard Roderick E</div><div class="citationInfo"><span class="NLM_cas:title">Essays in biochemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">453-464</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">It is over 20 years since the first fragment-based discovery projects were disclosed.  The methods are now mature for most 'conventional' targets in drug discovery such as enzymes (kinases and proteases) but there has also been growing success on more challenging targets, such as disruption of protein-protein interactions.  The main application is to identify tractable chemical startpoints that non-covalently modulate the activity of a biological molecule.  In this essay, we overview current practice in the methods and discuss how they have had an impact in lead discovery - generating a large number of fragment-derived compounds that are in clinical trials and two medicines treating patients.  In addition, we discuss some of the more recent applications of the methods in chemical biology - providing chemical tools to investigate biological molecules, mechanisms and systems.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTqZGQ3bfUY0f6BottYdIAWfW6udTcc2eaHhq5rbkPdJrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1M7pt1Gmtw%253D%253D&md5=35bce3c5978329c6737ffd719135d363</span></div><a href="/servlet/linkout?suffix=cit124&amp;dbid=16384&amp;doi=10.1042%2FEBC20170028&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252FEBC20170028%26sid%3Dliteratum%253Aachs%26aulast%3DLamoree%26aufirst%3DB.%26aulast%3DHubbard%26aufirst%3DR.%2BE.%26atitle%3DCurrent%2520Perspectives%2520in%2520Fragment-Based%2520Lead%2520Discovery%2520%2528FBLD%2529%26jtitle%3DEssays%2520Biochem.%26date%3D2017%26volume%3D61%26spage%3D453%26epage%3D464%26doi%3D10.1042%2FEBC20170028" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref125"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref125'); return false;" data-citation="" class="refNumLink">125</a></strong><div class="NLM_citation" id="cit125"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jochim, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arora, P. S.</span></span> <span> </span><span class="NLM_article-title">Assessment of Helical Interfaces in Protein–protein Interactions</span>. <i>Mol. BioSyst.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">924</span>– <span class="NLM_lpage">926</span>, <span class="refDoi"> DOI: 10.1039/b903202a</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref125/cit125&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1039%2Fb903202a" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref125/cit125&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=19668855" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref125/cit125&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BD1MXps1KksLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2009&pages=924-926&author=A.+L.+Jochimauthor=P.+S.+Arora&title=Assessment+of+Helical+Interfaces+in+Protein%E2%80%93protein+Interactions&doi=10.1039%2Fb903202a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit125R"><div class="casContent"><span class="casTitleNuber">125</span><div class="casTitle"><span class="NLM_cas:atitle">Assessment of helical interfaces in protein-protein interactions</span></div><div class="casAuthors">Jochim, Andrea L.; Arora, Paramjit S.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular BioSystems</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">924-926</span>CODEN:
                <span class="NLM_cas:coden">MBOIBW</span>;
        ISSN:<span class="NLM_cas:issn">1742-206X</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Herein we identify and analyze helical protein interfaces as potential targets for synthetic modulators of protein-protein interactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGptv4ZRquFYk7Vg90H21EOLACvtfcHk0ljGZyeD9WmGzg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXps1KksLw%253D&md5=18d83b22339213b36dbe7df2d0d5cd91</span></div><a href="/servlet/linkout?suffix=cit125&amp;dbid=16384&amp;doi=10.1039%2Fb903202a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fb903202a%26sid%3Dliteratum%253Aachs%26aulast%3DJochim%26aufirst%3DA.%2BL.%26aulast%3DArora%26aufirst%3DP.%2BS.%26atitle%3DAssessment%2520of%2520Helical%2520Interfaces%2520in%2520Protein%25E2%2580%2593protein%2520Interactions%26jtitle%3DMol.%2520BioSyst.%26date%3D2009%26volume%3D5%26spage%3D924%26epage%3D926%26doi%3D10.1039%2Fb903202a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref126"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref126'); return false;" data-citation="" class="refNumLink">126</a></strong><div class="NLM_citation" id="cit126"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Watkins, A.
M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wuo, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arora, P. S.</span></span> <span> </span><span class="NLM_article-title">Protein–Protein Interactions Mediated by Helical Tertiary Structure Motifs</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>137</i></span>,  <span class="NLM_fpage">11622</span>– <span class="NLM_lpage">11630</span>, <span class="refDoi"> DOI: 10.1021/jacs.5b05527</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.5b05527" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref126/cit126&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtlOqu7rK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=137&publication_year=2015&pages=11622-11630&author=A.%0AM.+Watkinsauthor=M.+G.+Wuoauthor=P.+S.+Arora&title=Protein%E2%80%93Protein+Interactions+Mediated+by+Helical+Tertiary+Structure+Motifs&doi=10.1021%2Fjacs.5b05527"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit126R"><div class="casContent"><span class="casTitleNuber">126</span><div class="casTitle"><span class="NLM_cas:atitle">Protein-protein interactions mediated by helical tertiary structure motifs</span></div><div class="casAuthors">Watkins, Andrew M.; Wuo, Michael G.; Arora, Paramjit S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">137</span>
        (<span class="NLM_cas:issue">36</span>),
    <span class="NLM_cas:pages">11622-11630</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The modulation of protein-protein interactions (PPIs) by means of creating or stabilizing secondary structure conformations is a rapidly growing area of research.  Recent success in the inhibition of difficult PPIs by secondary structure mimetics also points to potential limitations, because often, specific cases require tertiary structure mimetics.  To streamline protein structure-based inhibitor design, the authors have previously described the examn. of protein complexes in the Protein Data Bank where α-helixes or β-strands form crit. contacts.  Here, the authors examd. coiled-coils and helix bundles that mediate complex formation to create a platform for the discovery of potential tertiary structure mimetics.  Although there has been extensive anal. of coiled-coil motifs, the interactions between pre-formed coiled-coils and globular proteins have not been systematically analyzed.  Here, the authors identified crit. features of these helical interfaces with respect to coiled-coil and other helical PPIs.  The authors expect the anal. to prove useful for the rational design of modulators of this fundamental class of protein assemblies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrhNUkUL-2YLbVg90H21EOLACvtfcHk0ljGZyeD9WmGzg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtlOqu7rK&md5=440d2e7bda4a4806e0d79721fe899304</span></div><a href="/servlet/linkout?suffix=cit126&amp;dbid=16384&amp;doi=10.1021%2Fjacs.5b05527&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.5b05527%26sid%3Dliteratum%253Aachs%26aulast%3DWatkins%26aufirst%3DA.%2BM.%26aulast%3DWuo%26aufirst%3DM.%2BG.%26aulast%3DArora%26aufirst%3DP.%2BS.%26atitle%3DProtein%25E2%2580%2593Protein%2520Interactions%2520Mediated%2520by%2520Helical%2520Tertiary%2520Structure%2520Motifs%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2015%26volume%3D137%26spage%3D11622%26epage%3D11630%26doi%3D10.1021%2Fjacs.5b05527" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref127"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref127'); return false;" data-citation="" class="refNumLink">127</a></strong><div class="NLM_citation" id="cit127"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Checco, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gellman, S. H.</span></span> <span> </span><span class="NLM_article-title">Iterative Nonproteinogenic Residue Incorporation Yields Α/β-Peptides with a Helix–Loop–Helix Tertiary Structure and High Affinity for VEGF</span>. <i>ChemBioChem</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">291</span>– <span class="NLM_lpage">299</span>, <span class="refDoi"> DOI: 10.1002/cbic.201600545</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref127/cit127&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1002%2Fcbic.201600545" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref127/cit127&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=27897370" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref127/cit127&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC2sXltVanuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2017&pages=291-299&author=J.+W.+Checcoauthor=S.+H.+Gellman&title=Iterative+Nonproteinogenic+Residue+Incorporation+Yields+%CE%91%2F%CE%B2-Peptides+with+a+Helix%E2%80%93Loop%E2%80%93Helix+Tertiary+Structure+and+High+Affinity+for+VEGF&doi=10.1002%2Fcbic.201600545"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit127R"><div class="casContent"><span class="casTitleNuber">127</span><div class="casTitle"><span class="NLM_cas:atitle">Iterative Nonproteinogenic Residue Incorporation Yields α/β-Peptides with a Helix-Loop-Helix Tertiary Structure and High Affinity for VEGF</span></div><div class="casAuthors">Checco, James W.; Gellman, Samuel H.</div><div class="citationInfo"><span class="NLM_cas:title">ChemBioChem</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">291-299</span>CODEN:
                <span class="NLM_cas:coden">CBCHFX</span>;
        ISSN:<span class="NLM_cas:issn">1439-4227</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Inhibition of specific protein-protein interactions is attractive for a range of therapeutic applications, but the large and irregularly shaped contact surfaces involved in many such interactions make it challenging to design synthetic antagonists.  Here, we describe the development of backbone-modified peptides contg. both α- and β-amino acid residues (α/β-peptides) that target the receptor-binding surface of vascular endothelial growth factor (VEGF).  Our approach is based on the Z-domain, which adopts a three-helix bundle tertiary structure.  We show how a two-helix "mini-Z-domain" can be modified to contain β and other nonproteinogenic residues while retaining the target-binding epitope by using iterative unnatural residue incorporation.  The resulting α/β-peptides are less susceptible to proteolysis than is their parent α-peptide, and some of these α/β-peptides match the full-length Z-domain in terms of affinity for receptor-recognition surfaces on the VEGF homodimer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGozhiwqlXcFTrVg90H21EOLACvtfcHk0ljGZyeD9WmGzg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXltVanuw%253D%253D&md5=6c653a8760340c0410d9f598be076a22</span></div><a href="/servlet/linkout?suffix=cit127&amp;dbid=16384&amp;doi=10.1002%2Fcbic.201600545&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcbic.201600545%26sid%3Dliteratum%253Aachs%26aulast%3DChecco%26aufirst%3DJ.%2BW.%26aulast%3DGellman%26aufirst%3DS.%2BH.%26atitle%3DIterative%2520Nonproteinogenic%2520Residue%2520Incorporation%2520Yields%2520%25CE%2591%252F%25CE%25B2-Peptides%2520with%2520a%2520Helix%25E2%2580%2593Loop%25E2%2580%2593Helix%2520Tertiary%2520Structure%2520and%2520High%2520Affinity%2520for%2520VEGF%26jtitle%3DChemBioChem%26date%3D2017%26volume%3D18%26spage%3D291%26epage%3D299%26doi%3D10.1002%2Fcbic.201600545" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref128"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref128'); return false;" data-citation="" class="refNumLink">128</a></strong><div class="NLM_citation" id="cit128"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wuo, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mahon, A. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arora, P. S.</span></span> <span> </span><span class="NLM_article-title">An Effective Strategy for Stabilizing Minimal Coiled Coil Mimetics</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>137</i></span>,  <span class="NLM_fpage">11618</span>– <span class="NLM_lpage">11621</span>, <span class="refDoi"> DOI: 10.1021/jacs.5b05525</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.5b05525" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref128/cit128&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsVKit7fI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=137&publication_year=2015&pages=11618-11621&author=M.+G.+Wuoauthor=A.+B.+Mahonauthor=P.+S.+Arora&title=An+Effective+Strategy+for+Stabilizing+Minimal+Coiled+Coil+Mimetics&doi=10.1021%2Fjacs.5b05525"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit128R"><div class="casContent"><span class="casTitleNuber">128</span><div class="casTitle"><span class="NLM_cas:atitle">An Effective Strategy for Stabilizing Minimal Coiled Coil Mimetics</span></div><div class="casAuthors">Wuo, Michael G.; Mahon, Andrew B.; Arora, Paramjit S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">137</span>
        (<span class="NLM_cas:issue">36</span>),
    <span class="NLM_cas:pages">11618-11621</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Coiled coils are a major motif in proteins and orchestrate multimerization of various complexes important for biol. processes.  Inhibition of coiled coil-mediated interactions has significant biomedical potential.  However, general approaches that afford short peptides with defined coiled coil conformation remain elusive.  The authors evaluated several strategies to stabilize minimal helical bundles, with the dimer motif as the initial focus.  A stable dimeric scaffold was realized in a synthetic sequence by replacing an interhelical ionic bond with a covalent bond.  Application of this strategy to a more challenging native protein-protein interaction (PPI) suggested that an addnl. constraint, a disulfide bond at the internal a/d' position along with a linker at the e/e' position, is required for enhanced conformational stability.  The authors anticipate the coiled coil stabilization methodol. described herein to yield new classes of modulators for PPIs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqb0FEdmteRRLVg90H21EOLACvtfcHk0ljGZyeD9WmGzg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsVKit7fI&md5=8efd6f5dd2576c989ed1542b650ec454</span></div><a href="/servlet/linkout?suffix=cit128&amp;dbid=16384&amp;doi=10.1021%2Fjacs.5b05525&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.5b05525%26sid%3Dliteratum%253Aachs%26aulast%3DWuo%26aufirst%3DM.%2BG.%26aulast%3DMahon%26aufirst%3DA.%2BB.%26aulast%3DArora%26aufirst%3DP.%2BS.%26atitle%3DAn%2520Effective%2520Strategy%2520for%2520Stabilizing%2520Minimal%2520Coiled%2520Coil%2520Mimetics%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2015%26volume%3D137%26spage%3D11618%26epage%3D11621%26doi%3D10.1021%2Fjacs.5b05525" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref129"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref129'); return false;" data-citation="" class="refNumLink">129</a></strong><div class="NLM_citation" id="cit129"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Roschger, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neukirchen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elsässer, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schubert, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maeding, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verwanger, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krammer, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cabrele, C.</span></span> <span> </span><span class="NLM_article-title">Targeting of a Helix-Loop-Helix Transcriptional Regulator by a Short Helical Peptide</span>. <i>ChemMedChem</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">1497</span>– <span class="NLM_lpage">1503</span>, <span class="refDoi"> DOI: 10.1002/cmdc.201700305</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref129/cit129&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1002%2Fcmdc.201700305" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2017&pages=1497-1503&author=C.+Roschgerauthor=S.+Neukirchenauthor=B.+Els%C3%A4sserauthor=M.+Schubertauthor=N.+Maedingauthor=T.+Verwangerauthor=B.+Krammerauthor=C.+Cabrele&title=Targeting+of+a+Helix-Loop-Helix+Transcriptional+Regulator+by+a+Short+Helical+Peptide&doi=10.1002%2Fcmdc.201700305"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit129&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.201700305&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.201700305%26sid%3Dliteratum%253Aachs%26aulast%3DRoschger%26aufirst%3DC.%26aulast%3DNeukirchen%26aufirst%3DS.%26aulast%3DEls%25C3%25A4sser%26aufirst%3DB.%26aulast%3DSchubert%26aufirst%3DM.%26aulast%3DMaeding%26aufirst%3DN.%26aulast%3DVerwanger%26aufirst%3DT.%26aulast%3DKrammer%26aufirst%3DB.%26aulast%3DCabrele%26aufirst%3DC.%26atitle%3DTargeting%2520of%2520a%2520Helix-Loop-Helix%2520Transcriptional%2520Regulator%2520by%2520a%2520Short%2520Helical%2520Peptide%26jtitle%3DChemMedChem%26date%3D2017%26volume%3D12%26spage%3D1497%26epage%3D1503%26doi%3D10.1002%2Fcmdc.201700305" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref130"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref130'); return false;" data-citation="" class="refNumLink">130</a></strong><div class="NLM_citation" id="cit130"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jochim, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arora, P. S.</span></span> <span> </span><span class="NLM_article-title">Systematic Analysis of Helical Protein Interfaces Reveals Targets for Synthetic Inhibitors</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">919</span>– <span class="NLM_lpage">923</span>, <span class="refDoi"> DOI: 10.1021/cb1001747</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb1001747" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref130/cit130&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtVKrsLnO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2010&pages=919-923&author=A.+L.+Jochimauthor=P.+S.+Arora&title=Systematic+Analysis+of+Helical+Protein+Interfaces+Reveals+Targets+for+Synthetic+Inhibitors&doi=10.1021%2Fcb1001747"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit130R"><div class="casContent"><span class="casTitleNuber">130</span><div class="casTitle"><span class="NLM_cas:atitle">Systematic Analysis of Helical Protein Interfaces Reveals Targets for Synthetic Inhibitors</span></div><div class="casAuthors">Jochim, Andrea L.; Arora, Paramjit S.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">919-923</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Synthetic inhibitors of protein-protein interactions are being discovered despite the inherent challenge in targeting large contact surfaces with small mols.  An anal. of available examples identifies common features of complexes that make them tractable for small mols.  We deduced that relative disposition and energetic contributions of "hot spot" residues provide a predictive scale for the potential of protein-protein interactions to be inhibited by small mols.  On the basis of this model, we analyzed the full set of helical protein interfaces in the Protein Data Bank to identify those that are potentially suitable candidates for synthetic ligands.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr3SMHxSvMd7LVg90H21EOLACvtfcHk0ljeREhrPJp9Yw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtVKrsLnO&md5=ea2f564abefa82573fb2320fd70e9fe0</span></div><a href="/servlet/linkout?suffix=cit130&amp;dbid=16384&amp;doi=10.1021%2Fcb1001747&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb1001747%26sid%3Dliteratum%253Aachs%26aulast%3DJochim%26aufirst%3DA.%2BL.%26aulast%3DArora%26aufirst%3DP.%2BS.%26atitle%3DSystematic%2520Analysis%2520of%2520Helical%2520Protein%2520Interfaces%2520Reveals%2520Targets%2520for%2520Synthetic%2520Inhibitors%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2010%26volume%3D5%26spage%3D919%26epage%3D923%26doi%3D10.1021%2Fcb1001747" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref131"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref131'); return false;" data-citation="" class="refNumLink">131</a></strong><div class="NLM_citation" id="cit131"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Moellering, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cornejo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, T. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bianco, C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aster, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blacklow, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kung, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilliland, D. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verdine, G. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span> <span> </span><span class="NLM_article-title">Direct Inhibition of the NOTCH Transcription Factor Complex</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>462</i></span>,  <span class="NLM_fpage">182</span>– <span class="NLM_lpage">188</span>, <span class="refDoi"> DOI: 10.1038/nature08543</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref131/cit131&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1038%2Fnature08543" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref131/cit131&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=19907488" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref131/cit131&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsVSjsr3F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=462&publication_year=2009&pages=182-188&author=R.+E.+Moelleringauthor=M.+Cornejoauthor=T.+N.+Davisauthor=C.+D.+Biancoauthor=J.+C.+Asterauthor=S.+C.+Blacklowauthor=A.+L.+Kungauthor=D.+G.+Gillilandauthor=G.+L.+Verdineauthor=J.+E.+Bradner&title=Direct+Inhibition+of+the+NOTCH+Transcription+Factor+Complex&doi=10.1038%2Fnature08543"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit131R"><div class="casContent"><span class="casTitleNuber">131</span><div class="casTitle"><span class="NLM_cas:atitle">Direct inhibition of the NOTCH transcription factor complex</span></div><div class="casAuthors">Moellering, Raymond E.; Cornejo, Melanie; Davis, Tina N.; Del Bianco, Cristina; Aster, Jon C.; Blacklow, Stephen C.; Kung, Andrew L.; Gilliland, D. Gary; Verdine, Gregory L.; Bradner, James E.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">462</span>
        (<span class="NLM_cas:issue">7270</span>),
    <span class="NLM_cas:pages">182-188</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Direct inhibition of transcription factor complexes remains a central challenge in the discipline of ligand discovery.  In general, these proteins lack surface involutions suitable for high-affinity binding by small mols.  Here the authors report the design of synthetic, cell-permeable, stabilized α-helical peptides that target a crit. protein-protein interface in the NOTCH transactivation complex.  The authors demonstrate that direct, high-affinity binding of the hydrocarbon-stapled peptide SAHM1 prevents assembly of the active transcriptional complex.  Inappropriate NOTCH activation is directly implicated in the pathogenesis of several disease states, including T-cell acute lymphoblastic leukemia (T-ALL).  The treatment of leukemic cells with SAHM1 results in genome-wide suppression of NOTCH-activated genes.  Direct antagonism of the NOTCH transcriptional program causes potent, NOTCH-specific anti-proliferative effects in cultured cells and in a mouse model of NOTCH1-driven T-ALL.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq_hyuFJHX0T7Vg90H21EOLACvtfcHk0ljeREhrPJp9Yw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsVSjsr3F&md5=a12f19dd03303fd054e3052d7bbffd47</span></div><a href="/servlet/linkout?suffix=cit131&amp;dbid=16384&amp;doi=10.1038%2Fnature08543&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature08543%26sid%3Dliteratum%253Aachs%26aulast%3DMoellering%26aufirst%3DR.%2BE.%26aulast%3DCornejo%26aufirst%3DM.%26aulast%3DDavis%26aufirst%3DT.%2BN.%26aulast%3DBianco%26aufirst%3DC.%2BD.%26aulast%3DAster%26aufirst%3DJ.%2BC.%26aulast%3DBlacklow%26aufirst%3DS.%2BC.%26aulast%3DKung%26aufirst%3DA.%2BL.%26aulast%3DGilliland%26aufirst%3DD.%2BG.%26aulast%3DVerdine%26aufirst%3DG.%2BL.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26atitle%3DDirect%2520Inhibition%2520of%2520the%2520NOTCH%2520Transcription%2520Factor%2520Complex%26jtitle%3DNature%26date%3D2009%26volume%3D462%26spage%3D182%26epage%3D188%26doi%3D10.1038%2Fnature08543" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref132"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref132'); return false;" data-citation="" class="refNumLink">132</a></strong><div class="NLM_citation" id="cit132"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Grossmann, T. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yeh, J. T.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bowman, B. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chu, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moellering, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verdine, G. L.</span></span> <span> </span><span class="NLM_article-title">Inhibition of Oncogenic Wnt Signaling through Direct Targeting of Β-Catenin</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>109</i></span>,  <span class="NLM_fpage">17942</span>– <span class="NLM_lpage">17947</span>, <span class="refDoi"> DOI: 10.1073/pnas.1208396109</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref132/cit132&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1073%2Fpnas.1208396109" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref132/cit132&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=23071338" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref132/cit132&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC38Xhsl2ktbvI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=109&publication_year=2012&pages=17942-17947&author=T.+N.+Grossmannauthor=J.+T.-H.+Yehauthor=B.+R.+Bowmanauthor=Q.+Chuauthor=R.+E.+Moelleringauthor=G.+L.+Verdine&title=Inhibition+of+Oncogenic+Wnt+Signaling+through+Direct+Targeting+of+%CE%92-Catenin&doi=10.1073%2Fpnas.1208396109"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit132R"><div class="casContent"><span class="casTitleNuber">132</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of oncogenic Wnt signaling through direct targeting of β-catenin</span></div><div class="casAuthors">Grossmann, Tom N.; Yeh, Johannes T.-H.; Bowman, Brian R.; Chu, Qian; Moellering, Raymond E.; Verdine, Gregory L.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">109</span>
        (<span class="NLM_cas:issue">44</span>),
    <span class="NLM_cas:pages">17942-17947, S17942/1-S17942/10</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Aberrant activation of signaling by the Wnt pathway is strongly implicated in the onset and progression of numerous types of cancer.  Owing to the persistent dependence of these tumors on Wnt signaling for growth and survival, inhibition of this pathway is considered an attractive mechanism-based therapeutic approach.  Oncogenic activation of Wnt signaling can ensue from a variety of distinct aberrations in the signaling pathway, but most share the common feature of causing increased cellular levels of β-catenin by interfering with its constitutive degrdn.  β-Catenin serves as a central hub in Wnt signaling by engaging in crucial protein-protein interactions with both neg. and pos. effectors of the pathway.  Direct interference with these protein-protein interactions is a biol. compelling approach toward suppression of β-catenin hyperactivity, but such interactions have proven intransigent with respect to small-mol. targeting.  Hence β-catenin remains an elusive target for translational cancer therapy.  Here the authors report the discovery of a hydrocarbon-stapled peptide that directly targets β-catenin and interferes with its ability to serve as a transcriptional coactivator for T-cell factor (TCF) proteins, the downstream transcriptional regulators of the Wnt pathway.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqD9Q6Y8-IAI7Vg90H21EOLACvtfcHk0ljeREhrPJp9Yw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xhsl2ktbvI&md5=3f2fc0f4f616ef057317639fe7c465c6</span></div><a href="/servlet/linkout?suffix=cit132&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1208396109&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1208396109%26sid%3Dliteratum%253Aachs%26aulast%3DGrossmann%26aufirst%3DT.%2BN.%26aulast%3DYeh%26aufirst%3DJ.%2BT.-H.%26aulast%3DBowman%26aufirst%3DB.%2BR.%26aulast%3DChu%26aufirst%3DQ.%26aulast%3DMoellering%26aufirst%3DR.%2BE.%26aulast%3DVerdine%26aufirst%3DG.%2BL.%26atitle%3DInhibition%2520of%2520Oncogenic%2520Wnt%2520Signaling%2520through%2520Direct%2520Targeting%2520of%2520%25CE%2592-Catenin%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2012%26volume%3D109%26spage%3D17942%26epage%3D17947%26doi%3D10.1073%2Fpnas.1208396109" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref133"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref133'); return false;" data-citation="" class="refNumLink">133</a></strong><div class="NLM_citation" id="cit133"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lau, Y. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rossmann, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, B. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Andrade, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, Y. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verma, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McKenzie, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Venkitaraman, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hyvönen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spring, D. R.</span></span> <span> </span><span class="NLM_article-title">Double Strain-Promoted Macrocyclization for the Rapid Selection of Cell-Active Stapled Peptides</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">15410</span>– <span class="NLM_lpage">15413</span>, <span class="refDoi"> DOI: 10.1002/anie.201508416</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref133/cit133&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1002%2Fanie.201508416" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref133/cit133&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhslClsrbE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2015&pages=15410-15413&author=Y.+H.+Lauauthor=Y.+Wuauthor=M.+Rossmannauthor=B.+X.+Tanauthor=P.+de+Andradeauthor=Y.+S.+Tanauthor=C.+Vermaauthor=G.+J.+McKenzieauthor=A.+R.+Venkitaramanauthor=M.+Hyv%C3%B6nenauthor=D.+R.+Spring&title=Double+Strain-Promoted+Macrocyclization+for+the+Rapid+Selection+of+Cell-Active+Stapled+Peptides&doi=10.1002%2Fanie.201508416"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit133R"><div class="casContent"><span class="casTitleNuber">133</span><div class="casTitle"><span class="NLM_cas:atitle">Double Strain-Promoted Macrocyclization for the Rapid Selection of Cell-Active Stapled Peptides</span></div><div class="casAuthors">Lau, Yu Heng; Wu, Yuteng; Rossmann, Maxim; Tan, Ban Xiong; de Andrade, Peterson; Tan, Yaw Sing; Verma, Chandra; McKenzie, Grahame J.; Venkitaraman, Ashok R.; Hyvoenen, Marko; Spring, David R.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">51</span>),
    <span class="NLM_cas:pages">15410-15413</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Peptide stapling is a method for designing macrocyclic alpha-helical inhibitors of protein-protein interactions.  However, obtaining a cell-active inhibitor can require significant optimization.  The authors report a novel stapling technique based on a double strain-promoted azide-alkyne reaction, and exploit its biocompatibility to accelerate the discovery of cell-active stapled peptides.  As a proof of concept, MDM2-binding peptides were stapled in parallel, directly in cell culture medium in 96-well plates, and simultaneously evaluated in a p53 reporter assay.  This in situ stapling/screening process gave an optimal candidate that showed improved proteolytic stability and nanomolar binding to MDM2 in subsequent biophys. assays.  α-Helicity was confirmed by a crystal structure of the MDM2-peptide complex.  This work introduces in situ stapling as a versatile biocompatible technique with many other potential high-throughput biol. applications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpbh5ftgHQc1LVg90H21EOLACvtfcHk0lgK4BNKe1PcZw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhslClsrbE&md5=7ce32efbb010623a6ee8247373d6701c</span></div><a href="/servlet/linkout?suffix=cit133&amp;dbid=16384&amp;doi=10.1002%2Fanie.201508416&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201508416%26sid%3Dliteratum%253Aachs%26aulast%3DLau%26aufirst%3DY.%2BH.%26aulast%3DWu%26aufirst%3DY.%26aulast%3DRossmann%26aufirst%3DM.%26aulast%3DTan%26aufirst%3DB.%2BX.%26aulast%3Dde%2BAndrade%26aufirst%3DP.%26aulast%3DTan%26aufirst%3DY.%2BS.%26aulast%3DVerma%26aufirst%3DC.%26aulast%3DMcKenzie%26aufirst%3DG.%2BJ.%26aulast%3DVenkitaraman%26aufirst%3DA.%2BR.%26aulast%3DHyv%25C3%25B6nen%26aufirst%3DM.%26aulast%3DSpring%26aufirst%3DD.%2BR.%26atitle%3DDouble%2520Strain-Promoted%2520Macrocyclization%2520for%2520the%2520Rapid%2520Selection%2520of%2520Cell-Active%2520Stapled%2520Peptides%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2015%26volume%3D54%26spage%3D15410%26epage%3D15413%26doi%3D10.1002%2Fanie.201508416" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref134"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref134'); return false;" data-citation="" class="refNumLink">134</a></strong><div class="NLM_citation" id="cit134"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hilinski, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, Y.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hong, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kutchukian, P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crenshaw, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berkovitch, S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ham, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verdine, G. L.</span></span> <span> </span><span class="NLM_article-title">Stitched Α-Helical Peptides via Bis Ring-Closing Metathesis</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>136</i></span>,  <span class="NLM_fpage">12314</span>– <span class="NLM_lpage">12322</span>, <span class="refDoi"> DOI: 10.1021/ja505141j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja505141j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref134/cit134&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtlaisrfI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=136&publication_year=2014&pages=12314-12322&author=G.+J.+Hilinskiauthor=Y.-W.+Kimauthor=J.+Hongauthor=P.+S.+Kutchukianauthor=C.+M.+Crenshawauthor=S.+S.+Berkovitchauthor=A.+Changauthor=S.+Hamauthor=G.+L.+Verdine&title=Stitched+%CE%91-Helical+Peptides+via+Bis+Ring-Closing+Metathesis&doi=10.1021%2Fja505141j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit134R"><div class="casContent"><span class="casTitleNuber">134</span><div class="casTitle"><span class="NLM_cas:atitle">Stitched α-Helical Peptides via Bis Ring-Closing Metathesis</span></div><div class="casAuthors">Hilinski, Gerard J.; Kim, Young-Woo; Hong, Jooyeon; Kutchukian, Peter S.; Crenshaw, Charisse M.; Berkovitch, Shaunna S.; Chang, Andrew; Ham, Sihyun; Verdine, Gregory L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">136</span>
        (<span class="NLM_cas:issue">35</span>),
    <span class="NLM_cas:pages">12314-12322</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Conformationally stabilized α-helical peptides are capable of inhibiting disease-relevant intracellular or extracellular protein-protein interactions in vivo.  The authors have previously reported that the employment of ring-closing metathesis to introduce a single all-hydrocarbon staple along one face of an α-helical peptide greatly increases α-helical content, binding affinity to a target protein, cell penetration through active transport, and resistance to proteolytic degrdn.  In an effort to improve upon this technol. for stabilizing a peptide in a bioactive α-helical conformation, the authors report the discovery of an efficient and selective bis ring-closing metathesis reaction leading to peptides bearing multiple contiguous staples connected by a central spiro ring junction.  CD spectroscopy, NMR, and computational analyses have been used to study the conformation of these "stitched" peptides, which exhibit remarkable thermal stabilities.  Likewise, trypsin proteolysis assays confirm the achievement of a structural rigidity unmatched by peptides bearing a single staple.  Furthermore, fluorescence-activated cell sorting (FACS) and confocal microscopy assays demonstrate that stitched peptides display superior cell penetrating ability compared to their stapled counterparts, suggesting that this technol. may be useful not only in the context of enhancing the drug-like properties of α-helical peptides but also in producing potent agents for the intracellular delivery of proteins and oligonucleotides.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGob9a_dts0-bLVg90H21EOLACvtfcHk0lgK4BNKe1PcZw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtlaisrfI&md5=45e1542906e2062a3859ace862d0865d</span></div><a href="/servlet/linkout?suffix=cit134&amp;dbid=16384&amp;doi=10.1021%2Fja505141j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja505141j%26sid%3Dliteratum%253Aachs%26aulast%3DHilinski%26aufirst%3DG.%2BJ.%26aulast%3DKim%26aufirst%3DY.-W.%26aulast%3DHong%26aufirst%3DJ.%26aulast%3DKutchukian%26aufirst%3DP.%2BS.%26aulast%3DCrenshaw%26aufirst%3DC.%2BM.%26aulast%3DBerkovitch%26aufirst%3DS.%2BS.%26aulast%3DChang%26aufirst%3DA.%26aulast%3DHam%26aufirst%3DS.%26aulast%3DVerdine%26aufirst%3DG.%2BL.%26atitle%3DStitched%2520%25CE%2591-Helical%2520Peptides%2520via%2520Bis%2520Ring-Closing%2520Metathesis%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2014%26volume%3D136%26spage%3D12314%26epage%3D12322%26doi%3D10.1021%2Fja505141j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref135"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref135'); return false;" data-citation="" class="refNumLink">135</a></strong><div class="NLM_citation" id="cit135"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gopalakrishnan, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frolov, A. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knerr, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drury, W. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valeur, E.</span></span> <span> </span><span class="NLM_article-title">Therapeutic Potential of Foldamers: From Chemical Biology Tools To Drug Candidates?</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">9599</span>– <span class="NLM_lpage">9621</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b00376</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b00376" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref135/cit135&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtV2gsbrE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=9599-9621&author=R.+Gopalakrishnanauthor=A.+I.+Frolovauthor=L.+Knerrauthor=W.+J.+Druryauthor=E.+Valeur&title=Therapeutic+Potential+of+Foldamers%3A+From+Chemical+Biology+Tools+To+Drug+Candidates%3F&doi=10.1021%2Facs.jmedchem.6b00376"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit135R"><div class="casContent"><span class="casTitleNuber">135</span><div class="casTitle"><span class="NLM_cas:atitle">Therapeutic Potential of Foldamers: From Chemical Biology Tools To Drug Candidates?</span></div><div class="casAuthors">Gopalakrishnan, Ranganath; Frolov, Andrey I.; Knerr, Laurent; Drury, William J.; Valeur, Eric</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">9599-9621</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Over the past decade, foldamers have progressively emerged as useful architectures to mimic secondary structures of proteins.  Peptidic foldamers, consisting of various amino acid based backbones, have been the most studied from a therapeutic perspective, while polyarom. foldamers have barely evolved from their nascency and remain perplexing for medicinal chemists due to their poor drug-like nature.  Despite these limitations, this compd. class may still offer opportunities to study challenging targets or provide chem. biol. tools.  The potential of foldamer drug candidates reaching the clinic is still a stretch.  Nevertheless, advances in the field have demonstrated their potential for the discovery of next generation therapeutics.  In this perspective, the current knowledge of foldamers is reviewed in a drug discovery context.  Recent advances in the early phases of drug discovery including hit finding, target validation, and optimization and mol. modeling are discussed.  In addn., challenges and focus areas are debated and gaps highlighted.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoI_-tbQEJhE7Vg90H21EOLACvtfcHk0lgK4BNKe1PcZw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtV2gsbrE&md5=7e6ff6d5e90ef9992e71c17ce30f565e</span></div><a href="/servlet/linkout?suffix=cit135&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b00376&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b00376%26sid%3Dliteratum%253Aachs%26aulast%3DGopalakrishnan%26aufirst%3DR.%26aulast%3DFrolov%26aufirst%3DA.%2BI.%26aulast%3DKnerr%26aufirst%3DL.%26aulast%3DDrury%26aufirst%3DW.%2BJ.%26aulast%3DValeur%26aufirst%3DE.%26atitle%3DTherapeutic%2520Potential%2520of%2520Foldamers%253A%2520From%2520Chemical%2520Biology%2520Tools%2520To%2520Drug%2520Candidates%253F%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D9599%26epage%3D9621%26doi%3D10.1021%2Facs.jmedchem.6b00376" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref136"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref136'); return false;" data-citation="" class="refNumLink">136</a></strong><div class="NLM_citation" id="cit136"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cheloha, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maeda, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dean, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gardella, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gellman, S. H.</span></span> <span> </span><span class="NLM_article-title">Backbone Modification of a Polypeptide Drug Alters Duration of Action in Vivo</span>. <i>Nat. Biotechnol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>32</i></span>,  <span class="NLM_fpage">653</span>– <span class="NLM_lpage">655</span>, <span class="refDoi"> DOI: 10.1038/nbt.2920</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref136/cit136&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1038%2Fnbt.2920" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref136/cit136&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=24929976" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref136/cit136&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC2cXpslGis7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2014&pages=653-655&author=R.+W.+Chelohaauthor=A.+Maedaauthor=T.+Deanauthor=T.+J.+Gardellaauthor=S.+H.+Gellman&title=Backbone+Modification+of+a+Polypeptide+Drug+Alters+Duration+of+Action+in+Vivo&doi=10.1038%2Fnbt.2920"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit136R"><div class="casContent"><span class="casTitleNuber">136</span><div class="casTitle"><span class="NLM_cas:atitle">Backbone modification of a polypeptide drug alters duration of action in vivo</span></div><div class="casAuthors">Cheloha, Ross W.; Maeda, Akira; Dean, Thomas; Gardella, Thomas J.; Gellman, Samuel H.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">32</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">653-655</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Systematic modification of the backbone of bioactive polypeptides through β-amino acid residue incorporation could provide a strategy for generating mols. with improved drug properties, but such alterations can result in lower receptor affinity and potency.  Using an agonist of parathyroid hormone receptor-1 (PTHR1), a G protein-coupled receptor in the B-family, we present an approach for α→β residue replacement that enables both high activity and improved pharmacokinetic properties in vivo.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr8cWxIA-3shbVg90H21EOLACvtfcHk0ljDEjvJO6FvDA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXpslGis7k%253D&md5=100382f0d7e67fe9c11fd39886e7acac</span></div><a href="/servlet/linkout?suffix=cit136&amp;dbid=16384&amp;doi=10.1038%2Fnbt.2920&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt.2920%26sid%3Dliteratum%253Aachs%26aulast%3DCheloha%26aufirst%3DR.%2BW.%26aulast%3DMaeda%26aufirst%3DA.%26aulast%3DDean%26aufirst%3DT.%26aulast%3DGardella%26aufirst%3DT.%2BJ.%26aulast%3DGellman%26aufirst%3DS.%2BH.%26atitle%3DBackbone%2520Modification%2520of%2520a%2520Polypeptide%2520Drug%2520Alters%2520Duration%2520of%2520Action%2520in%2520Vivo%26jtitle%3DNat.%2520Biotechnol.%26date%3D2014%26volume%3D32%26spage%3D653%26epage%3D655%26doi%3D10.1038%2Fnbt.2920" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref137"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref137'); return false;" data-citation="" class="refNumLink">137</a></strong><div class="NLM_citation" id="cit137"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hager, M. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wootten, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sexton, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gellman, S. H.</span></span> <span> </span><span class="NLM_article-title">Β-Arrestin-Biased Agonists of the GLP-1 Receptor from Β-Amino Acid Residue Incorporation into GLP-1 Analogues</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>138</i></span>,  <span class="NLM_fpage">14970</span>– <span class="NLM_lpage">14979</span>, <span class="refDoi"> DOI: 10.1021/jacs.6b08323</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.6b08323" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=138&publication_year=2016&pages=14970-14979&author=M.+V.+Hagerauthor=L.+M.+Johnsonauthor=D.+Woottenauthor=P.+M.+Sextonauthor=S.+H.+Gellman&title=%CE%92-Arrestin-Biased+Agonists+of+the+GLP-1+Receptor+from+%CE%92-Amino+Acid+Residue+Incorporation+into+GLP-1+Analogues&doi=10.1021%2Fjacs.6b08323"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit137&amp;dbid=16384&amp;doi=10.1021%2Fjacs.6b08323&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.6b08323%26sid%3Dliteratum%253Aachs%26aulast%3DHager%26aufirst%3DM.%2BV.%26aulast%3DJohnson%26aufirst%3DL.%2BM.%26aulast%3DWootten%26aufirst%3DD.%26aulast%3DSexton%26aufirst%3DP.%2BM.%26aulast%3DGellman%26aufirst%3DS.%2BH.%26atitle%3D%25CE%2592-Arrestin-Biased%2520Agonists%2520of%2520the%2520GLP-1%2520Receptor%2520from%2520%25CE%2592-Amino%2520Acid%2520Residue%2520Incorporation%2520into%2520GLP-1%2520Analogues%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2016%26volume%3D138%26spage%3D14970%26epage%3D14979%26doi%3D10.1021%2Fjacs.6b08323" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref138"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref138'); return false;" data-citation="" class="refNumLink">138</a></strong><div class="NLM_citation" id="cit138"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Robinson, J. A.</span></span> <span> </span><span class="NLM_article-title">Β-Hairpin Peptidomimetics: Design, Structures and Biological Activities</span>. <i>Acc. Chem. Res.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>41</i></span>,  <span class="NLM_fpage">1278</span>– <span class="NLM_lpage">1288</span>, <span class="refDoi"> DOI: 10.1021/ar700259k</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ar700259k" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref138/cit138&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BD1cXkslChsbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2008&pages=1278-1288&author=J.+A.+Robinson&title=%CE%92-Hairpin+Peptidomimetics%3A+Design%2C+Structures+and+Biological+Activities&doi=10.1021%2Far700259k"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit138R"><div class="casContent"><span class="casTitleNuber">138</span><div class="casTitle"><span class="NLM_cas:atitle">β-Hairpin Peptidomimetics: Design, Structures and Biological Activities</span></div><div class="casAuthors">Robinson, John A.</div><div class="citationInfo"><span class="NLM_cas:title">Accounts of Chemical Research</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1278-1288</span>CODEN:
                <span class="NLM_cas:coden">ACHRE4</span>;
        ISSN:<span class="NLM_cas:issn">0001-4842</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The folded 3D structures of peptides and proteins provide excellent starting points for the design of synthetic mols. that mimic key epitopes (or surface patches) involved in protein-protein and protein-nucleic acid interactions.  Protein epitope mimetics (PEMs) may recapitulate not only the structural and conformational properties of the target epitope but also their biol. activities.  By transferring the epitope from a recombinant to a synthetic scaffold that can be produced by parallel combinatorial methods, it is possible to optimize properties through iterative cycles of library synthesis and screening, and even to evolve new biol. activities.  One very interesting scaffold is found in β-hairpin motifs, which are used by many proteins to mediate mol. recognition events.  This motif is readily amenable to PEM design, for example, by transplanting hairpin loop sequences from folded proteins onto hairpin-stabilizing templates, such as the dipeptide D-Pro-L-Pro.  In addn., β-hairpin peptidomimetics can also be exploited to mimic other types of epitopes, such as those based on α-helical secondary structures.  The size and shape of β-hairpin PEMs appear well suited for the design of inhibitors of both protein-protein and protein-nucleic acid interactions, endeavors that have so far proven difficult using small "drug-like" mols.  In recent work, it was shown that β-hairpin PEMs can be designed that mimic the canonical conformations of antibody hypervariable loops, suggesting that novel small-mol. antibody mimics may be feasible.  Using naturally occurring peptides as starting points, β-hairpin mimetics have been discovered that possess antimicrobial activity, while others are potent inhibitors of the chemokine receptor CXCR4. β-Hairpin PEMs have also been designed and optimized that mimic an α-helical epitope in p53 and so block its interaction with HDM2.  A crystal structure of one HDM2-mimetic complex revealed how the surface of the protein had adapted to the shape of the hairpin, thereby enhancing inhibitor affinity.  Small folded RNA motifs also make interesting targets for inhibitor design.  For example, β-hairpin mimetics have been designed and optimized that bind with high affinity and good selectivity to the TAR and RRE RNA motifs from HIV-1.  Soln. structures of the mimetics both free and bound to the RNA target provided some surprises, as well as an improved understanding of the mechanisms of binding.  These mimetics represent still a relatively new family of RNA-binding mols., but clearly one with potential for development into novel antiviral agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqJpSiBOoiIQLVg90H21EOLACvtfcHk0ljDEjvJO6FvDA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXkslChsbc%253D&md5=083014216e0fc0918cf096d5fcf446d4</span></div><a href="/servlet/linkout?suffix=cit138&amp;dbid=16384&amp;doi=10.1021%2Far700259k&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Far700259k%26sid%3Dliteratum%253Aachs%26aulast%3DRobinson%26aufirst%3DJ.%2BA.%26atitle%3D%25CE%2592-Hairpin%2520Peptidomimetics%253A%2520Design%252C%2520Structures%2520and%2520Biological%2520Activities%26jtitle%3DAcc.%2520Chem.%2520Res.%26date%3D2008%26volume%3D41%26spage%3D1278%26epage%3D1288%26doi%3D10.1021%2Far700259k" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref139"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref139'); return false;" data-citation="" class="refNumLink">139</a></strong><div class="NLM_citation" id="cit139"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Srinivas, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jetter, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ueberbacher, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Werneburg, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zerbe, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steinmann, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van der Meijden, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernardini, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lederer, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dias, R. L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Misson, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Henze, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zumbrunn, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gombert, F. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Obrecht, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hunziker, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schauer, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ziegler, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Käch, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eberl, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riedel, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DeMarco, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robinson, J. A.</span></span> <span> </span><span class="NLM_article-title">Peptidomimetic Antibiotics Target Outer-Membrane Biogenesis in Pseudomonas Aeruginosa</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>327</i></span>,  <span class="NLM_fpage">1010</span>– <span class="NLM_lpage">1013</span>, <span class="refDoi"> DOI: 10.1126/science.1182749</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref139/cit139&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1126%2Fscience.1182749" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref139/cit139&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=20167788" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref139/cit139&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC3cXitVSjtbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=327&publication_year=2010&pages=1010-1013&author=N.+Srinivasauthor=P.+Jetterauthor=B.+J.+Ueberbacherauthor=M.+Werneburgauthor=K.+Zerbeauthor=J.+Steinmannauthor=B.+Van+der+Meijdenauthor=F.+Bernardiniauthor=A.+Ledererauthor=R.+L.+A.+Diasauthor=P.+E.+Missonauthor=H.+Henzeauthor=J.+Zumbrunnauthor=F.+O.+Gombertauthor=D.+Obrechtauthor=P.+Hunzikerauthor=S.+Schauerauthor=U.+Zieglerauthor=A.+K%C3%A4chauthor=L.+Eberlauthor=K.+Riedelauthor=S.+J.+DeMarcoauthor=J.+A.+Robinson&title=Peptidomimetic+Antibiotics+Target+Outer-Membrane+Biogenesis+in+Pseudomonas+Aeruginosa&doi=10.1126%2Fscience.1182749"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit139R"><div class="casContent"><span class="casTitleNuber">139</span><div class="casTitle"><span class="NLM_cas:atitle">Peptidomimetic Antibiotics Target Outer-Membrane Biogenesis in Pseudomonas aeruginosa</span></div><div class="casAuthors">Srinivas, Nityakalyani; Jetter, Peter; Ueberbacher, Bernhard J.; Werneburg, Martina; Zerbe, Katja; Steinmann, Jessica; Van der Meijden, Benjamin; Bernardini, Francesca; Lederer, Alexander; Dias, Ricardo L. A.; Misson, Pauline E.; Henze, Heiko; Zumbrunn, Juerg; Gombert, Frank O.; Obrecht, Daniel; Hunziker, Peter; Schauer, Stefan; Ziegler, Urs; Kaech, Andres; Eberl, Leo; Riedel, Kathrin; De Marco, Steven J.; Robinson, John A.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">327</span>
        (<span class="NLM_cas:issue">5968</span>),
    <span class="NLM_cas:pages">1010-1013</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Antibiotics with new mechanisms of action are urgently required to combat the growing health threat posed by resistant pathogenic microorganisms.  The authors synthesized a family of peptidomimetic antibiotics based on the antimicrobial peptide protegrin I.  Several rounds of optimization gave a lead compd. that was active in the nanomolar range against Gram-neg. Pseudomonas spp., but was largely inactive against other Gram-neg. and Gram-pos. bacteria.  Biochem. and genetic studies showed that the peptidomimetics had a non-membrane-lytic mechanism of action and identified a homolog of the β-barrel protein LptD (Imp/OstA), which functions in outer-membrane biogenesis, as a cellular target.  The peptidomimetic showed potent antimicrobial activity in a mouse septicemia infection model.  Drug-resistant strains of Pseudomonas are a serious health problem, so this family of antibiotics may have important therapeutic applications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpjVCgTY_lqGLVg90H21EOLACvtfcHk0ljDEjvJO6FvDA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXitVSjtbk%253D&md5=a59f74c897b26c6471c6d16910bcb018</span></div><a href="/servlet/linkout?suffix=cit139&amp;dbid=16384&amp;doi=10.1126%2Fscience.1182749&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1182749%26sid%3Dliteratum%253Aachs%26aulast%3DSrinivas%26aufirst%3DN.%26aulast%3DJetter%26aufirst%3DP.%26aulast%3DUeberbacher%26aufirst%3DB.%2BJ.%26aulast%3DWerneburg%26aufirst%3DM.%26aulast%3DZerbe%26aufirst%3DK.%26aulast%3DSteinmann%26aufirst%3DJ.%26aulast%3DVan%2Bder%2BMeijden%26aufirst%3DB.%26aulast%3DBernardini%26aufirst%3DF.%26aulast%3DLederer%26aufirst%3DA.%26aulast%3DDias%26aufirst%3DR.%2BL.%2BA.%26aulast%3DMisson%26aufirst%3DP.%2BE.%26aulast%3DHenze%26aufirst%3DH.%26aulast%3DZumbrunn%26aufirst%3DJ.%26aulast%3DGombert%26aufirst%3DF.%2BO.%26aulast%3DObrecht%26aufirst%3DD.%26aulast%3DHunziker%26aufirst%3DP.%26aulast%3DSchauer%26aufirst%3DS.%26aulast%3DZiegler%26aufirst%3DU.%26aulast%3DK%25C3%25A4ch%26aufirst%3DA.%26aulast%3DEberl%26aufirst%3DL.%26aulast%3DRiedel%26aufirst%3DK.%26aulast%3DDeMarco%26aufirst%3DS.%2BJ.%26aulast%3DRobinson%26aufirst%3DJ.%2BA.%26atitle%3DPeptidomimetic%2520Antibiotics%2520Target%2520Outer-Membrane%2520Biogenesis%2520in%2520Pseudomonas%2520Aeruginosa%26jtitle%3DScience%26date%3D2010%26volume%3D327%26spage%3D1010%26epage%3D1013%26doi%3D10.1126%2Fscience.1182749" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref140"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref140'); return false;" data-citation="" class="refNumLink">140</a></strong><div class="NLM_citation" id="cit140"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zerbe, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moehle, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robinson, J. A.</span></span> <span> </span><span class="NLM_article-title">Protein Epitope Mimetics: From New Antibiotics to Supramolecular Synthetic Vaccines</span>. <i>Acc. Chem. Res.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>50</i></span>,  <span class="NLM_fpage">1323</span>– <span class="NLM_lpage">1331</span>, <span class="refDoi"> DOI: 10.1021/acs.accounts.7b00129</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.accounts.7b00129" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2017&pages=1323-1331&author=K.+Zerbeauthor=K.+Moehleauthor=J.+A.+Robinson&title=Protein+Epitope+Mimetics%3A+From+New+Antibiotics+to+Supramolecular+Synthetic+Vaccines&doi=10.1021%2Facs.accounts.7b00129"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit140&amp;dbid=16384&amp;doi=10.1021%2Facs.accounts.7b00129&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.accounts.7b00129%26sid%3Dliteratum%253Aachs%26aulast%3DZerbe%26aufirst%3DK.%26aulast%3DMoehle%26aufirst%3DK.%26aulast%3DRobinson%26aufirst%3DJ.%2BA.%26atitle%3DProtein%2520Epitope%2520Mimetics%253A%2520From%2520New%2520Antibiotics%2520to%2520Supramolecular%2520Synthetic%2520Vaccines%26jtitle%3DAcc.%2520Chem.%2520Res.%26date%3D2017%26volume%3D50%26spage%3D1323%26epage%3D1331%26doi%3D10.1021%2Facs.accounts.7b00129" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref141"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref141'); return false;" data-citation="" class="refNumLink">141</a></strong><div class="NLM_citation" id="cit141"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Luther, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moehle, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chevalier, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dale, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Obrecht, D.</span></span> <span> </span><span class="NLM_article-title">Protein Epitope Mimetic Macrocycles as Biopharmaceuticals</span>. <i>Curr. Opin. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>38</i></span>,  <span class="NLM_fpage">45</span>– <span class="NLM_lpage">51</span>, <span class="refDoi"> DOI: 10.1016/j.cbpa.2017.02.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref141/cit141&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1016%2Fj.cbpa.2017.02.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref141/cit141&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=28319811" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref141/cit141&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC2sXksFSiu7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=2017&pages=45-51&author=A.+Lutherauthor=K.+Moehleauthor=E.+Chevalierauthor=G.+Daleauthor=D.+Obrecht&title=Protein+Epitope+Mimetic+Macrocycles+as+Biopharmaceuticals&doi=10.1016%2Fj.cbpa.2017.02.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit141R"><div class="casContent"><span class="casTitleNuber">141</span><div class="casTitle"><span class="NLM_cas:atitle">Protein epitope mimetic macrocycles as biopharmaceuticals</span></div><div class="casAuthors">Luther, Anatol; Moehle, Kerstin; Chevalier, Eric; Dale, Glenn; Obrecht, Daniel</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Chemical Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">45-51</span>CODEN:
                <span class="NLM_cas:coden">COCBF4</span>;
        ISSN:<span class="NLM_cas:issn">1367-5931</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Fully synthetic medium-sized macrocyclic peptides mimicking the key β-hairpin and α-helical protein epitopes relevant in many protein-protein interactions have emerged as a novel class of drugs with the potential to fill an important gap between small mols. and proteins.  Conformationally stabilized macrocyclic scaffolds represent ideal templates for medicinal chemists to incorporate bioactive peptide and protein pharmacophores in order to generate novel drugs to treat diseases with high unmet medical need.  This review describes recent approaches to design and generate large libraries of such macrocycles, for hit identification, and for their efficient optimization.  Finally, this review describes some of the most advanced protein epitope mimetic (PEM) macrocycles in clin. development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpKLA10A3orAbVg90H21EOLACvtfcHk0liofVHB3vYCkQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXksFSiu7Y%253D&md5=623f351d0e915d58f1ebd698da850a37</span></div><a href="/servlet/linkout?suffix=cit141&amp;dbid=16384&amp;doi=10.1016%2Fj.cbpa.2017.02.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cbpa.2017.02.004%26sid%3Dliteratum%253Aachs%26aulast%3DLuther%26aufirst%3DA.%26aulast%3DMoehle%26aufirst%3DK.%26aulast%3DChevalier%26aufirst%3DE.%26aulast%3DDale%26aufirst%3DG.%26aulast%3DObrecht%26aufirst%3DD.%26atitle%3DProtein%2520Epitope%2520Mimetic%2520Macrocycles%2520as%2520Biopharmaceuticals%26jtitle%3DCurr.%2520Opin.%2520Chem.%2520Biol.%26date%3D2017%26volume%3D38%26spage%3D45%26epage%3D51%26doi%3D10.1016%2Fj.cbpa.2017.02.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref142"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref142'); return false;" data-citation="" class="refNumLink">142</a></strong><div class="NLM_citation" id="cit142"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gavenonis, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheneman, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siegert, T. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eshelman, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kritzer, J. A.</span></span> <span> </span><span class="NLM_article-title">Comprehensive Analysis of Loops at Protein-Protein Interfaces for Macrocycle Design</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">716</span>– <span class="NLM_lpage">722</span>, <span class="refDoi"> DOI: 10.1038/nchembio.1580</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref142/cit142&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1038%2Fnchembio.1580" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref142/cit142&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=25038791" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref142/cit142&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtFygs7jE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2014&pages=716-722&author=J.+Gavenonisauthor=B.+A.+Shenemanauthor=T.+R.+Siegertauthor=M.+R.+Eshelmanauthor=J.+A.+Kritzer&title=Comprehensive+Analysis+of+Loops+at+Protein-Protein+Interfaces+for+Macrocycle+Design&doi=10.1038%2Fnchembio.1580"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit142R"><div class="casContent"><span class="casTitleNuber">142</span><div class="casTitle"><span class="NLM_cas:atitle">Comprehensive analysis of loops at protein-protein interfaces for macrocycle design</span></div><div class="casAuthors">Gavenonis, Jason; Sheneman, Bradley A.; Siegert, Timothy R.; Eshelman, Matthew R.; Kritzer, Joshua A.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">716-722</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Inhibiting protein-protein interactions (PPIs) with synthetic mols. remains a frontier of chem. biol.  Many PPIs have been successfully targeted by mimicking α-helixes at interfaces, but most PPIs are mediated by nonhelical, nonstrand peptide loops.  We sought to comprehensively identify and analyze these loop-mediated PPIs by writing and implementing LoopFinder, a customizable program that can identify loop-mediated PPIs within all of the protein-protein complexes in the Protein Data Bank.  Comprehensive anal. of the entire set of 25,005 interface loops revealed common structural motifs and unique features that distinguish loop-mediated PPIs from other PPIs.  'Hot loops', named in analogy to protein hot spots, were identified as loops with favorable properties for mimicry using synthetic mols.  The hot loops and their binding partners represent new and promising PPIs for the development of macrocycle and constrained peptide inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp96pqxP3_amLVg90H21EOLACvtfcHk0liofVHB3vYCkQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtFygs7jE&md5=bf07bd724f173ea74467ab7cd5aedb6a</span></div><a href="/servlet/linkout?suffix=cit142&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1580&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1580%26sid%3Dliteratum%253Aachs%26aulast%3DGavenonis%26aufirst%3DJ.%26aulast%3DSheneman%26aufirst%3DB.%2BA.%26aulast%3DSiegert%26aufirst%3DT.%2BR.%26aulast%3DEshelman%26aufirst%3DM.%2BR.%26aulast%3DKritzer%26aufirst%3DJ.%2BA.%26atitle%3DComprehensive%2520Analysis%2520of%2520Loops%2520at%2520Protein-Protein%2520Interfaces%2520for%2520Macrocycle%2520Design%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2014%26volume%3D10%26spage%3D716%26epage%3D722%26doi%3D10.1038%2Fnchembio.1580" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref143"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref143'); return false;" data-citation="" class="refNumLink">143</a></strong><div class="NLM_citation" id="cit143"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Siegert, T. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bird, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Makwana, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kritzer, J. A.</span></span> <span> </span><span class="NLM_article-title">Analysis of Loops That Mediate Protein–Protein Interactions and Translation into Submicromolar Inhibitors</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>138</i></span>,  <span class="NLM_fpage">12876</span>– <span class="NLM_lpage">12884</span>, <span class="refDoi"> DOI: 10.1021/jacs.6b05656</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.6b05656" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref143/cit143&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsVyqtL%252FK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=138&publication_year=2016&pages=12876-12884&author=T.+R.+Siegertauthor=M.+J.+Birdauthor=K.+M.+Makwanaauthor=J.+A.+Kritzer&title=Analysis+of+Loops+That+Mediate+Protein%E2%80%93Protein+Interactions+and+Translation+into+Submicromolar+Inhibitors&doi=10.1021%2Fjacs.6b05656"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit143R"><div class="casContent"><span class="casTitleNuber">143</span><div class="casTitle"><span class="NLM_cas:atitle">Analysis of Loops that Mediate Protein-Protein Interactions and Translation into Submicromolar Inhibitors</span></div><div class="casAuthors">Siegert, Timothy R.; Bird, Michael J.; Makwana, Kamlesh M.; Kritzer, Joshua A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">138</span>
        (<span class="NLM_cas:issue">39</span>),
    <span class="NLM_cas:pages">12876-12884</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Effective strategies for mimicking α-helix and β-strand epitopes have been developed, producing valuable inhibitors for some classes of protein-protein interactions (PPIs).  However, there are no general strategies for translating loop epitopes into useful PPI inhibitors.  In this work, we use the LoopFinder program to identify diverse sets of "hot loops," which are loop epitopes that mediate PPIs.  These include loops that are well-suited to mimicry with macrocyclic compds., and loops that are most similar to variable loops on antibodies and ankyrin repeat proteins.  We present data-driven criteria for scoring loop-mediated PPIs, uncovering a trove of potentially druggable interactions.  We also use unbiased clustering to identify common structures among the hot loops.  To translate these insights into real-world inhibitors, we describe a robust, diversity-oriented strategy for the rapid prodn. and evaluation of cyclized loops.  This method is applied to a computationally identified loop in the PPI between stonin2 and Eps15, producing sub-micromolar inhibitors.  The most potent inhibitor is well-structured in water and successfully mimics the native epitope.  Overall, these computational and exptl. strategies provide new opportunities to design inhibitors for an otherwise intractable set of PPIs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpWWUk81azbxLVg90H21EOLACvtfcHk0ljVx-dfcQPKww"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsVyqtL%252FK&md5=9a036f700e452a65257a0c0c8b52a937</span></div><a href="/servlet/linkout?suffix=cit143&amp;dbid=16384&amp;doi=10.1021%2Fjacs.6b05656&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.6b05656%26sid%3Dliteratum%253Aachs%26aulast%3DSiegert%26aufirst%3DT.%2BR.%26aulast%3DBird%26aufirst%3DM.%2BJ.%26aulast%3DMakwana%26aufirst%3DK.%2BM.%26aulast%3DKritzer%26aufirst%3DJ.%2BA.%26atitle%3DAnalysis%2520of%2520Loops%2520That%2520Mediate%2520Protein%25E2%2580%2593Protein%2520Interactions%2520and%2520Translation%2520into%2520Submicromolar%2520Inhibitors%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2016%26volume%3D138%26spage%3D12876%26epage%3D12884%26doi%3D10.1021%2Fjacs.6b05656" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref144"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref144'); return false;" data-citation="" class="refNumLink">144</a></strong><div class="NLM_citation" id="cit144"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rong, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luk, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qin, N.</span></span> <span> </span><span class="NLM_article-title">Targeting Hippo Pathway by Specific Interruption of YAP-TEAD Interaction Using Cyclic YAP-like Peptides</span>. <i>FASEB J.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">724</span>– <span class="NLM_lpage">732</span>, <span class="refDoi"> DOI: 10.1096/fj.14-262980</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref144/cit144&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1096%2Ffj.14-262980" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref144/cit144&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=25384421" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref144/cit144&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC2MXislGisbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2015&pages=724-732&author=Z.+Zhouauthor=T.+Huauthor=Z.+Xuauthor=Z.+Linauthor=Z.+Zhangauthor=T.+Fengauthor=L.+Zhuauthor=Y.+Rongauthor=H.+Shenauthor=J.+M.+Lukauthor=X.+Zhangauthor=N.+Qin&title=Targeting+Hippo+Pathway+by+Specific+Interruption+of+YAP-TEAD+Interaction+Using+Cyclic+YAP-like+Peptides&doi=10.1096%2Ffj.14-262980"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit144R"><div class="casContent"><span class="casTitleNuber">144</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting Hippo pathway by specific interruption of YAP-TEAD interaction using cyclic YAP-like peptides</span></div><div class="casAuthors">Zhou, Zheng; Hu, Taishan; Xu, Zhiheng; Lin, Zhaohu; Zhang, Zhisen; Feng, Teng; Zhu, Liangcheng; Rong, Yiping; Shen, Hong; Luk, John M.; Zhang, Xiongwen; Qin, Ning</div><div class="citationInfo"><span class="NLM_cas:title">FASEB Journal</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">724-732, 10.1096/fj.14-262980</span>CODEN:
                <span class="NLM_cas:coden">FAJOEC</span>;
        ISSN:<span class="NLM_cas:issn">0892-6638</span>.
    
            (<span class="NLM_cas:orgname">Federation of American Societies for Experimental Biology</span>)
        </div><div class="casAbstract">Hippo signaling pathway is emerging as a novel target for anticancer therapy because it plays key roles in organ size control and tumorigenesis.  As the downstream effectors, Yes-assocd. protein (YAP)-transcriptional enhancer activation domain family member (TEAD) assocn. is essential for YAP-driven oncogenic activity, while TEAD is largely dispensable for normal tissue growth.  We present the design of YAP-like peptides (17mer) to occupy the interface 3 on TEAD.  Introducing cysteines at YAP sites 87 and 96 can induce disulfide formation, as confirmed by crystallog.  The engineered peptide significantly improves the potency in disrupting YAP-TEAD interaction in vitro.  To confirm that blocking YAP-TEAD complex formation by directly targeting on TEAD is a valid approach, we report a significant redn. in tumor growth rate in a hepatocellular carcinoma xenograft model after introducing the dominant-neg. mutation (Y406H) of TEAD1 to abolish YAP-TEAD interaction.  Our results suggest that targeting TEAD is a promising strategy against YAP-induced oncogenesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqIJ7Idzph7ELVg90H21EOLACvtfcHk0ljVx-dfcQPKww"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXislGisbk%253D&md5=bd3ce9530d0b5587c0118bf32e975546</span></div><a href="/servlet/linkout?suffix=cit144&amp;dbid=16384&amp;doi=10.1096%2Ffj.14-262980&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1096%252Ffj.14-262980%26sid%3Dliteratum%253Aachs%26aulast%3DZhou%26aufirst%3DZ.%26aulast%3DHu%26aufirst%3DT.%26aulast%3DXu%26aufirst%3DZ.%26aulast%3DLin%26aufirst%3DZ.%26aulast%3DZhang%26aufirst%3DZ.%26aulast%3DFeng%26aufirst%3DT.%26aulast%3DZhu%26aufirst%3DL.%26aulast%3DRong%26aufirst%3DY.%26aulast%3DShen%26aufirst%3DH.%26aulast%3DLuk%26aufirst%3DJ.%2BM.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DQin%26aufirst%3DN.%26atitle%3DTargeting%2520Hippo%2520Pathway%2520by%2520Specific%2520Interruption%2520of%2520YAP-TEAD%2520Interaction%2520Using%2520Cyclic%2520YAP-like%2520Peptides%26jtitle%3DFASEB%2520J.%26date%3D2015%26volume%3D29%26spage%3D724%26epage%3D732%26doi%3D10.1096%2Ffj.14-262980" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref145"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref145'); return false;" data-citation="" class="refNumLink">145</a></strong><div class="NLM_citation" id="cit145"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, H. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, S. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mayweg, A. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qin, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rong, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fry, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, T.</span></span> <span> </span><span class="NLM_article-title">Structure-Based Design and Synthesis of Potent Cyclic Peptides Inhibiting the YAP–TEAD Protein–Protein Interaction</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">993</span>– <span class="NLM_lpage">998</span>, <span class="refDoi"> DOI: 10.1021/ml500160m</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ml500160m" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&pages=993-998&author=Z.+Zhangauthor=Z.+Linauthor=Z.+Zhouauthor=H.+C.+Shenauthor=S.+F.+Yanauthor=A.+V.+Maywegauthor=Z.+Xuauthor=N.+Qinauthor=J.+C.+Wongauthor=Z.+Zhangauthor=Y.+Rongauthor=D.+C.+Fryauthor=T.+Hu&title=Structure-Based+Design+and+Synthesis+of+Potent+Cyclic+Peptides+Inhibiting+the+YAP%E2%80%93TEAD+Protein%E2%80%93Protein+Interaction&doi=10.1021%2Fml500160m"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit145&amp;dbid=16384&amp;doi=10.1021%2Fml500160m&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fml500160m%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DZ.%26aulast%3DLin%26aufirst%3DZ.%26aulast%3DZhou%26aufirst%3DZ.%26aulast%3DShen%26aufirst%3DH.%2BC.%26aulast%3DYan%26aufirst%3DS.%2BF.%26aulast%3DMayweg%26aufirst%3DA.%2BV.%26aulast%3DXu%26aufirst%3DZ.%26aulast%3DQin%26aufirst%3DN.%26aulast%3DWong%26aufirst%3DJ.%2BC.%26aulast%3DZhang%26aufirst%3DZ.%26aulast%3DRong%26aufirst%3DY.%26aulast%3DFry%26aufirst%3DD.%2BC.%26aulast%3DHu%26aufirst%3DT.%26atitle%3DStructure-Based%2520Design%2520and%2520Synthesis%2520of%2520Potent%2520Cyclic%2520Peptides%2520Inhibiting%2520the%2520YAP%25E2%2580%2593TEAD%2520Protein%25E2%2580%2593Protein%2520Interaction%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2014%26volume%3D5%26spage%3D993%26epage%3D998%26doi%3D10.1021%2Fml500160m" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref146"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref146'); return false;" data-citation="" class="refNumLink">146</a></strong><div class="NLM_citation" id="cit146"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bhardwaj, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mulligan, V. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bahl, C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilmore, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harvey, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheneval, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buchko, G. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pulavarti, S. V. S. R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaas, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eletsky, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, P.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnsen, W. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greisen, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rocklin, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Linsky, T. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watkins, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rettie, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carter, L. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bonneau, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olson, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coutsias, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Correnti, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szyperski, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Craik, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baker, D.</span></span> <span> </span><span class="NLM_article-title">Accurate de Novo Design of Hyperstable Constrained Peptides</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>538</i></span>,  <span class="NLM_fpage">329</span>– <span class="NLM_lpage">335</span>, <span class="refDoi"> DOI: 10.1038/nature19791</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref146/cit146&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1038%2Fnature19791" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref146/cit146&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=27626386" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref146/cit146&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsFWiurnL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=538&publication_year=2016&pages=329-335&author=G.+Bhardwajauthor=V.+K.+Mulliganauthor=C.+D.+Bahlauthor=J.+M.+Gilmoreauthor=P.+J.+Harveyauthor=O.+Chenevalauthor=G.+W.+Buchkoauthor=S.+V.+S.+R.+K.+Pulavartiauthor=Q.+Kaasauthor=A.+Eletskyauthor=P.-S.+Huangauthor=W.+A.+Johnsenauthor=P.+J.+Greisenauthor=G.+J.+Rocklinauthor=Y.+Songauthor=T.+W.+Linskyauthor=A.+Watkinsauthor=S.+A.+Rettieauthor=X.+Xuauthor=L.+P.+Carterauthor=R.+Bonneauauthor=J.+M.+Olsonauthor=E.+Coutsiasauthor=C.+E.+Correntiauthor=T.+Szyperskiauthor=D.+J.+Craikauthor=D.+Baker&title=Accurate+de+Novo+Design+of+Hyperstable+Constrained+Peptides&doi=10.1038%2Fnature19791"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit146R"><div class="casContent"><span class="casTitleNuber">146</span><div class="casTitle"><span class="NLM_cas:atitle">Accurate de novo design of hyperstable constrained peptides</span></div><div class="casAuthors">Bhardwaj, Gaurav; Mulligan, Vikram Khipple; Bahl, Christopher D.; Gilmore, Jason M.; Harvey, Peta J.; Cheneval, Olivier; Buchko, Garry W.; Pulavarti, Surya V. S. R. K.; Kaas, Quentin; Eletsky, Alexander; Huang, Po-Ssu; Johnsen, William A.; Greisen, Per Jr; Rocklin, Gabriel J.; Song, Yifan; Linsky, Thomas W.; Watkins, Andrew; Rettie, Stephen A.; Xu, Xianzhong; Carter, Lauren P.; Bonneau, Richard; Olson, James M.; Coutsias, Evangelos; Correnti, Colin E.; Szyperski, Thomas; Craik, David J.; Baker, David</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">538</span>
        (<span class="NLM_cas:issue">7625</span>),
    <span class="NLM_cas:pages">329-335</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Naturally occurring, pharmacol. active peptides constrained with covalent crosslinks generally have shapes that have evolved to fit precisely into binding pockets on their targets.  Such peptides can have excellent pharmaceutical properties, combining the stability and tissue penetration of small-mol. drugs with the specificity of much larger protein therapeutics.  The ability to design constrained peptides with precisely specified tertiary structures would enable the design of shape-complementary inhibitors of arbitrary targets.  Here we describe the development of computational methods for accurate de novo design of conformationally restricted peptides, and the use of these methods to design 18-47 residue, disulfide-crosslinked peptides, a subset of which are heterochiral and/or N-C backbone-cyclized.  Both genetically encodable and non-canonical peptides are exceptionally stable to thermal and chem. denaturation, and 12 exptl. detd. X-ray and NMR structures are nearly identical to the computational design models.  The computational design methods and stable scaffolds presented here provide the basis for development of a new generation of peptide-based drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqHeMGaFeL_kbVg90H21EOLACvtfcHk0ljVx-dfcQPKww"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsFWiurnL&md5=2d2b551ad78a2d76c99eeaba0c763c8e</span></div><a href="/servlet/linkout?suffix=cit146&amp;dbid=16384&amp;doi=10.1038%2Fnature19791&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature19791%26sid%3Dliteratum%253Aachs%26aulast%3DBhardwaj%26aufirst%3DG.%26aulast%3DMulligan%26aufirst%3DV.%2BK.%26aulast%3DBahl%26aufirst%3DC.%2BD.%26aulast%3DGilmore%26aufirst%3DJ.%2BM.%26aulast%3DHarvey%26aufirst%3DP.%2BJ.%26aulast%3DCheneval%26aufirst%3DO.%26aulast%3DBuchko%26aufirst%3DG.%2BW.%26aulast%3DPulavarti%26aufirst%3DS.%2BV.%2BS.%2BR.%2BK.%26aulast%3DKaas%26aufirst%3DQ.%26aulast%3DEletsky%26aufirst%3DA.%26aulast%3DHuang%26aufirst%3DP.-S.%26aulast%3DJohnsen%26aufirst%3DW.%2BA.%26aulast%3DGreisen%26aufirst%3DP.%2BJ.%26aulast%3DRocklin%26aufirst%3DG.%2BJ.%26aulast%3DSong%26aufirst%3DY.%26aulast%3DLinsky%26aufirst%3DT.%2BW.%26aulast%3DWatkins%26aufirst%3DA.%26aulast%3DRettie%26aufirst%3DS.%2BA.%26aulast%3DXu%26aufirst%3DX.%26aulast%3DCarter%26aufirst%3DL.%2BP.%26aulast%3DBonneau%26aufirst%3DR.%26aulast%3DOlson%26aufirst%3DJ.%2BM.%26aulast%3DCoutsias%26aufirst%3DE.%26aulast%3DCorrenti%26aufirst%3DC.%2BE.%26aulast%3DSzyperski%26aufirst%3DT.%26aulast%3DCraik%26aufirst%3DD.%2BJ.%26aulast%3DBaker%26aufirst%3DD.%26atitle%3DAccurate%2520de%2520Novo%2520Design%2520of%2520Hyperstable%2520Constrained%2520Peptides%26jtitle%3DNature%26date%3D2016%26volume%3D538%26spage%3D329%26epage%3D335%26doi%3D10.1038%2Fnature19791" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref147"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref147'); return false;" data-citation="" class="refNumLink">147</a></strong><div class="NLM_citation" id="cit147"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Namoto, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sirockin, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sellner, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wiesmann, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Villard, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moreau, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valeur, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paulding, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schleeger, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schipp, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loup, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andrews, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Swale, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robinson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farady, C. J.</span></span> <span> </span><span class="NLM_article-title">Structure-Based Design and Synthesis of Macrocyclic Human Rhinovirus 3C Protease Inhibitors</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>28</i></span>,  <span class="NLM_fpage">906</span>– <span class="NLM_lpage">909</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2018.01.064</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref147/cit147&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1016%2Fj.bmcl.2018.01.064" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2018&pages=906-909&author=K.+Namotoauthor=F.+Sirockinauthor=H.+Sellnerauthor=C.+Wiesmannauthor=F.+Villardauthor=R.+J.+Moreauauthor=E.+Valeurauthor=S.+C.+Pauldingauthor=S.+Schleegerauthor=K.+Schippauthor=J.+Loupauthor=L.+Andrewsauthor=R.+Swaleauthor=M.+Robinsonauthor=C.+J.+Farady&title=Structure-Based+Design+and+Synthesis+of+Macrocyclic+Human+Rhinovirus+3C+Protease+Inhibitors&doi=10.1016%2Fj.bmcl.2018.01.064"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit147&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2018.01.064&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2018.01.064%26sid%3Dliteratum%253Aachs%26aulast%3DNamoto%26aufirst%3DK.%26aulast%3DSirockin%26aufirst%3DF.%26aulast%3DSellner%26aufirst%3DH.%26aulast%3DWiesmann%26aufirst%3DC.%26aulast%3DVillard%26aufirst%3DF.%26aulast%3DMoreau%26aufirst%3DR.%2BJ.%26aulast%3DValeur%26aufirst%3DE.%26aulast%3DPaulding%26aufirst%3DS.%2BC.%26aulast%3DSchleeger%26aufirst%3DS.%26aulast%3DSchipp%26aufirst%3DK.%26aulast%3DLoup%26aufirst%3DJ.%26aulast%3DAndrews%26aufirst%3DL.%26aulast%3DSwale%26aufirst%3DR.%26aulast%3DRobinson%26aufirst%3DM.%26aulast%3DFarady%26aufirst%3DC.%2BJ.%26atitle%3DStructure-Based%2520Design%2520and%2520Synthesis%2520of%2520Macrocyclic%2520Human%2520Rhinovirus%25203C%2520Protease%2520Inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2018%26volume%3D28%26spage%3D906%26epage%3D909%26doi%3D10.1016%2Fj.bmcl.2018.01.064" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref148"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref148'); return false;" data-citation="" class="refNumLink">148</a></strong><div class="NLM_citation" id="cit148"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bird, G. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mazzola, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Opoku-Nsiah, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lammert, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Godes, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neuberg, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walensky, L. D.</span></span> <span> </span><span class="NLM_article-title">Biophysical Determinants for Cellular Uptake of Hydrocarbon-Stapled Peptide Helices</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">845</span>– <span class="NLM_lpage">852</span>, <span class="refDoi"> DOI: 10.1038/nchembio.2153</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref148/cit148&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1038%2Fnchembio.2153" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref148/cit148&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=27547919" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref148/cit148&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtlyqsbbP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2016&pages=845-852&author=G.+H.+Birdauthor=E.+Mazzolaauthor=K.+Opoku-Nsiahauthor=M.+A.+Lammertauthor=M.+Godesauthor=D.+S.+Neubergauthor=L.+D.+Walensky&title=Biophysical+Determinants+for+Cellular+Uptake+of+Hydrocarbon-Stapled+Peptide+Helices&doi=10.1038%2Fnchembio.2153"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit148R"><div class="casContent"><span class="casTitleNuber">148</span><div class="casTitle"><span class="NLM_cas:atitle">Biophysical determinants for cellular uptake of hydrocarbon-stapled peptide helices</span></div><div class="casAuthors">Bird, Gregory H.; Mazzola, Emanuele; Opoku-Nsiah, Kwadwo; Lammert, Margaret A.; Godes, Marina; Neuberg, Donna S.; Walensky, Loren D.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">845-852</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Hydrocarbon-stapled peptides are a class of bioactive alpha-helical ligands developed to dissect and target protein interactions.  While there is consensus that stapled peptides can be effective chem. tools for investigating protein regulation, their broader utility for therapeutic modulation of intracellular interactions remains an active area of study.  In particular, the design principles for generating cell-permeable stapled peptides are empiric, yet consistent intracellular access is essential to in vivo application.  Here, we used an unbiased statistical approach to det. which biophys. parameters dictate the uptake of stapled-peptide libraries.  We found that staple placement at the amphipathic boundary combined with optimal hydrophobic and helical content are the key drivers of cellular uptake, whereas excess hydrophobicity and pos. charge at isolated amino acid positions can trigger membrane lysis at elevated peptide dosing.  Our results provide a design roadmap for maximizing the potential to generate cell-permeable stapled peptides with on-mechanism cellular activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGowcU10JcijIbVg90H21EOLACvtfcHk0lhFyJMvDvqsbw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtlyqsbbP&md5=f31d10106beebfe08f69ec9a1b215ec1</span></div><a href="/servlet/linkout?suffix=cit148&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.2153&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.2153%26sid%3Dliteratum%253Aachs%26aulast%3DBird%26aufirst%3DG.%2BH.%26aulast%3DMazzola%26aufirst%3DE.%26aulast%3DOpoku-Nsiah%26aufirst%3DK.%26aulast%3DLammert%26aufirst%3DM.%2BA.%26aulast%3DGodes%26aufirst%3DM.%26aulast%3DNeuberg%26aufirst%3DD.%2BS.%26aulast%3DWalensky%26aufirst%3DL.%2BD.%26atitle%3DBiophysical%2520Determinants%2520for%2520Cellular%2520Uptake%2520of%2520Hydrocarbon-Stapled%2520Peptide%2520Helices%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2016%26volume%3D12%26spage%3D845%26epage%3D852%26doi%3D10.1038%2Fnchembio.2153" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref149"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref149'); return false;" data-citation="" class="refNumLink">149</a></strong><div class="NLM_citation" id="cit149"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Craik, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daly, N. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bond, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waine, C.</span></span> <span> </span><span class="NLM_article-title">Plant Cyclotides: A Unique Family of Cyclic and Knotted Proteins That Defines the Cyclic Cystine Knot Structural motif1 1Edited by P. E. Wright</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>294</i></span>,  <span class="NLM_fpage">1327</span>– <span class="NLM_lpage">1336</span>, <span class="refDoi"> DOI: 10.1006/jmbi.1999.3383</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref149/cit149&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1006%2Fjmbi.1999.3383" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref149/cit149&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10600388" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref149/cit149&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADyaK1MXnvFaqsLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=294&publication_year=1999&pages=1327-1336&author=D.+J.+Craikauthor=N.+L.+Dalyauthor=T.+Bondauthor=C.+Waine&title=Plant+Cyclotides%3A+A+Unique+Family+of+Cyclic+and+Knotted+Proteins+That+Defines+the+Cyclic+Cystine+Knot+Structural+motif1+1Edited+by+P.+E.+Wright&doi=10.1006%2Fjmbi.1999.3383"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit149R"><div class="casContent"><span class="casTitleNuber">149</span><div class="casTitle"><span class="NLM_cas:atitle">Plant Cyclotides: A Unique Family of Cyclic and Knotted Proteins that Defines the Cyclic Cystine Knot Structural Motif</span></div><div class="casAuthors">Craik, David J.; Daly, Norelle L.; Bond, Trudy; Waine, Clement</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">294</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1327-1336</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Academic Press</span>)
        </div><div class="casAbstract">Several macrocyclic peptides (∼30 amino acids), with diverse biol. activities, have been isolated from the Rubiaceae and Violaceae plant families over recent years.  We have significantly expanded the range of known macrocyclic peptides with the discovery of 16 novel peptides from exts. of Viola hederaceae, Viola odorata and Oldenlandia affinis.  The Viola plants had not previously been examd. for these peptides and thus represent novel species in which these unusual macrocyclic peptides are produced.  Further, we have detd. the three-dimensional structure of one of these novel peptides, cycloviolacin O1, using 1H NMR spectroscopy.  The structure consists of a distorted triple-stranded β-sheet and a cystine-knot arrangement of the disulfide bonds.  This structure is similar to kalata B1 and circulin A, the only two macrocyclic peptides for which a structure was available, suggesting that despite the sequence variation throughout the peptides they form a family in which the overall fold is conserved.  We refer to these peptides as the cyclotide family and their embedded topol. as the cyclic cystine knot (CCK) motif.  The unique cyclic and knotted nature of these mols. makes them a fascinating example of topol. complex proteins.  Examn. of the sequences reveals they can be sepd. into two subfamilies, one of which tends to contain a larger no. of pos. charged residues and has a bracelet-like circularization of the backbone.  The second subfamily contains a backbone twist due to a cis-Pro peptide bond and may conceptually be regarded as a mol. Moebius strip.  Here we define the structural features of the two apparent subfamilies of the CCK peptides which may be significant for the likely defense related role of these peptides within plants.  (c) 1999 Academic Press.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrOe8K8-Up5fLVg90H21EOLACvtfcHk0lhFyJMvDvqsbw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXnvFaqsLs%253D&md5=5c27c83a6ed1896821d3c14a74370e75</span></div><a href="/servlet/linkout?suffix=cit149&amp;dbid=16384&amp;doi=10.1006%2Fjmbi.1999.3383&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1006%252Fjmbi.1999.3383%26sid%3Dliteratum%253Aachs%26aulast%3DCraik%26aufirst%3DD.%2BJ.%26aulast%3DDaly%26aufirst%3DN.%2BL.%26aulast%3DBond%26aufirst%3DT.%26aulast%3DWaine%26aufirst%3DC.%26atitle%3DPlant%2520Cyclotides%253A%2520A%2520Unique%2520Family%2520of%2520Cyclic%2520and%2520Knotted%2520Proteins%2520That%2520Defines%2520the%2520Cyclic%2520Cystine%2520Knot%2520Structural%2520motif1%25201Edited%2520by%2520P.%2520E.%2520Wright%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D1999%26volume%3D294%26spage%3D1327%26epage%3D1336%26doi%3D10.1006%2Fjmbi.1999.3383" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref150"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref150'); return false;" data-citation="" class="refNumLink">150</a></strong><div class="NLM_citation" id="cit150"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Colgrave, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Korsinczky, M. J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clark, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Foley, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Craik, D. J.</span></span> <span> </span><span class="NLM_article-title">Sunflower Trypsin Inhibitor-1, Proteolytic Studies on a Trypsin Inhibitor Peptide and Its Analogs</span>. <i>Biopolymers</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>94</i></span>,  <span class="NLM_fpage">665</span>– <span class="NLM_lpage">672</span>, <span class="refDoi"> DOI: 10.1002/bip.21415</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref150/cit150&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1002%2Fbip.21415" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref150/cit150&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=20564016" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref150/cit150&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC3cXht12qtr%252FE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=94&publication_year=2010&pages=665-672&author=M.+L.+Colgraveauthor=M.+J.+L.+Korsinczkyauthor=R.+J.+Clarkauthor=F.+Foleyauthor=D.+J.+Craik&title=Sunflower+Trypsin+Inhibitor-1%2C+Proteolytic+Studies+on+a+Trypsin+Inhibitor+Peptide+and+Its+Analogs&doi=10.1002%2Fbip.21415"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit150R"><div class="casContent"><span class="casTitleNuber">150</span><div class="casTitle"><span class="NLM_cas:atitle">Sunflower trypsin inhibitor-1, proteolytic studies on a trypsin inhibitor peptide and its analogs</span></div><div class="casAuthors">Colgrave, Michelle L.; Korsinczky, Michael J. L.; Clark, Richard J.; Foley, Fiona; Craik, David J.</div><div class="citationInfo"><span class="NLM_cas:title">Biopolymers</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">94</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">665-672</span>CODEN:
                <span class="NLM_cas:coden">BIPMAA</span>;
        ISSN:<span class="NLM_cas:issn">0006-3525</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">Sunflower trypsin inhibitor-1 (SFTI-1) is a 14 amino acid cyclic peptide from sunflower seeds, which possesses exceptionally potent trypsin-inhibitory activity, and has promise as a stable peptide-based drug template.  Within its compact structure, SFTI-1 combines a head-to-tail cyclized backbone and a disulfide bond.  In this study, we synthesized a range of acyclic and disulfide-deficient analogs of SFTI-1 to investigate enzyme-assisted cyclization of the peptide backbone and proteolytic degrdn. that occurs as a result of incubation with trypsin.  Electrospray and matrix-assisted laser desorption ionization mass spectrometry allowed the characterization of a range of novel degrdn. products and elucidation of the time-course for cyclization and/or proteolysis.  Trypsin displayed the ability to resynthesize the scissile bond(s) and hence cyclize two of the linear permutants, whereas irreversible degrdn. was obsd. for another two permutants.  An interesting ring contraction mediated by trypsin was obsd., supporting a role for protease catalyzed splicing as a way of increasing the combinatorial diversity of cyclic peptides in nature.  Disulfide-deficient mutants were degraded within minutes, emphasizing the crit. role of the cysteine bridge in maintaining proteolytic stability of SFTI-1.  Overall, the study provides addnl. support for the proposal that naturally occurring cyclic peptides like SFTI-1 are biosynthesized by proteolytic enzymes effectively catalyzing the reverse of their normal reaction to make, rather than break peptide bonds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqqlJnJbomu87Vg90H21EOLACvtfcHk0lhPADm-Qps2Pg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXht12qtr%252FE&md5=d709396ac3d3c319fbcc8eebf44cac85</span></div><a href="/servlet/linkout?suffix=cit150&amp;dbid=16384&amp;doi=10.1002%2Fbip.21415&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fbip.21415%26sid%3Dliteratum%253Aachs%26aulast%3DColgrave%26aufirst%3DM.%2BL.%26aulast%3DKorsinczky%26aufirst%3DM.%2BJ.%2BL.%26aulast%3DClark%26aufirst%3DR.%2BJ.%26aulast%3DFoley%26aufirst%3DF.%26aulast%3DCraik%26aufirst%3DD.%2BJ.%26atitle%3DSunflower%2520Trypsin%2520Inhibitor-1%252C%2520Proteolytic%2520Studies%2520on%2520a%2520Trypsin%2520Inhibitor%2520Peptide%2520and%2520Its%2520Analogs%26jtitle%3DBiopolymers%26date%3D2010%26volume%3D94%26spage%3D665%26epage%3D672%26doi%3D10.1002%2Fbip.21415" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref151"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref151'); return false;" data-citation="" class="refNumLink">151</a></strong><div class="NLM_citation" id="cit151"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Muttenthaler, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Akondi, K. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alewood, P. F.</span></span> <span> </span><span class="NLM_article-title">Structure-Activity Studies on Alpha-Conotoxins</span>. <i>Curr. Pharm. Des.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">4226</span>– <span class="NLM_lpage">4241</span>, <span class="refDoi"> DOI: 10.2174/138161211798999384</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref151/cit151&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.2174%2F138161211798999384" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref151/cit151&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=22204424" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref151/cit151&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC38XovFajsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2011&pages=4226-4241&author=M.+Muttenthalerauthor=K.+B.+Akondiauthor=P.+F.+Alewood&title=Structure-Activity+Studies+on+Alpha-Conotoxins&doi=10.2174%2F138161211798999384"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit151R"><div class="casContent"><span class="casTitleNuber">151</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-activity studies on alpha-conotoxins</span></div><div class="casAuthors">Muttenthaler, Markus; Akondi, Kalyana B.; Alewood, Paul F.</div><div class="citationInfo"><span class="NLM_cas:title">Current Pharmaceutical Design</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">38</span>),
    <span class="NLM_cas:pages">4226-4241</span>CODEN:
                <span class="NLM_cas:coden">CPDEFP</span>;
        ISSN:<span class="NLM_cas:issn">1381-6128</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Conotoxins are small bioactive highly structured peptides from the venom of marine cone snails (genus Conus).  Over the past 50 million years these molluscs have developed a complex venom cocktail for each species that is comprised of 100-2000 distinct cysteine-rich peptides for prey capture and defense.  This review focuses on an important and well-studied class of conotoxins, the α-conotoxins.  These α-conotoxins are potent and selective antagonists of various subtypes of the nicotinic acetylcholine receptors (nAChRs).  Key structure-activity relationship studies are presented to illustrate the common motifs, structural features and pharmacophores that define this interesting peptide class.  Addnl., their synthesis, chem. modifications, the development of more selective and stable analogs and their therapeutic potential are discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrLz6PhaCubtLVg90H21EOLACvtfcHk0lhPADm-Qps2Pg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XovFajsw%253D%253D&md5=34aa49c05aa9cc30a4df1545e9cc76fd</span></div><a href="/servlet/linkout?suffix=cit151&amp;dbid=16384&amp;doi=10.2174%2F138161211798999384&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F138161211798999384%26sid%3Dliteratum%253Aachs%26aulast%3DMuttenthaler%26aufirst%3DM.%26aulast%3DAkondi%26aufirst%3DK.%2BB.%26aulast%3DAlewood%26aufirst%3DP.%2BF.%26atitle%3DStructure-Activity%2520Studies%2520on%2520Alpha-Conotoxins%26jtitle%3DCurr.%2520Pharm.%2520Des.%26date%3D2011%26volume%3D17%26spage%3D4226%26epage%3D4241%26doi%3D10.2174%2F138161211798999384" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref152"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref152'); return false;" data-citation="" class="refNumLink">152</a></strong><div class="NLM_citation" id="cit152"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Craik, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Du, J.</span></span> <span> </span><span class="NLM_article-title">Cyclotides as Drug Design Scaffolds</span>. <i>Curr. Opin. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>38</i></span>,  <span class="NLM_fpage">8</span>– <span class="NLM_lpage">16</span>, <span class="refDoi"> DOI: 10.1016/j.cbpa.2017.01.018</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref152/cit152&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1016%2Fj.cbpa.2017.01.018" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref152/cit152&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=28249194" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref152/cit152&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC2sXjsFGksLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=2017&pages=8-16&author=D.+J.+Craikauthor=J.+Du&title=Cyclotides+as+Drug+Design+Scaffolds&doi=10.1016%2Fj.cbpa.2017.01.018"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit152R"><div class="casContent"><span class="casTitleNuber">152</span><div class="casTitle"><span class="NLM_cas:atitle">Cyclotides as drug design scaffolds</span></div><div class="casAuthors">Craik, David J.; Du, Junqiao</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Chemical Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">8-16</span>CODEN:
                <span class="NLM_cas:coden">COCBF4</span>;
        ISSN:<span class="NLM_cas:issn">1367-5931</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Cyclotides are ultra-stable peptides derived from plants.  They are around 30 amino acids in size and are characterized by their head-to-tail cyclic backbone and cystine knot.  Their exceptional stability and tolerance to sequence substitutions has led to their use as frameworks in drug design.  This article describes recent developments in this field, particularly developments over the last two years relating to the grafting of bioactive peptide sequences into the cyclic cystine knot framework of cyclotides to stabilize the sequences.  Grafted cyclotides have now been developed that interact with protein or enzyme targets, both extracellular and intracellular, as well as with cell surface receptors and membranes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpW8Rb0hr6lQbVg90H21EOLACvtfcHk0lhPADm-Qps2Pg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXjsFGksLk%253D&md5=e3c59d4eacaa4b08a43e33a9986ff2eb</span></div><a href="/servlet/linkout?suffix=cit152&amp;dbid=16384&amp;doi=10.1016%2Fj.cbpa.2017.01.018&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cbpa.2017.01.018%26sid%3Dliteratum%253Aachs%26aulast%3DCraik%26aufirst%3DD.%2BJ.%26aulast%3DDu%26aufirst%3DJ.%26atitle%3DCyclotides%2520as%2520Drug%2520Design%2520Scaffolds%26jtitle%3DCurr.%2520Opin.%2520Chem.%2520Biol.%26date%3D2017%26volume%3D38%26spage%3D8%26epage%3D16%26doi%3D10.1016%2Fj.cbpa.2017.01.018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref153"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref153'); return false;" data-citation="" class="refNumLink">153</a></strong><div class="NLM_citation" id="cit153"><span><span class="NLM_contrib-group"><span class="NLM_string-name">De Veer, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weidmann, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Craik, D. J.</span></span> <span> </span><span class="NLM_article-title">Cyclotides as Tools in Chemical Biology</span>. <i>Acc. Chem. Res.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>50</i></span>,  <span class="NLM_fpage">1557</span>– <span class="NLM_lpage">1565</span>, <span class="refDoi"> DOI: 10.1021/acs.accounts.7b00157</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.accounts.7b00157" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref153/cit153&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtVCgsL7N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2017&pages=1557-1565&author=S.+J.+De+Veerauthor=J.+Weidmannauthor=D.+J.+Craik&title=Cyclotides+as+Tools+in+Chemical+Biology&doi=10.1021%2Facs.accounts.7b00157"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit153R"><div class="casContent"><span class="casTitleNuber">153</span><div class="casTitle"><span class="NLM_cas:atitle">Cyclotides as Tools in Chemical Biology</span></div><div class="casAuthors">de Veer, Simon J.; Weidmann, Joachim; Craik, David J.</div><div class="citationInfo"><span class="NLM_cas:title">Accounts of Chemical Research</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1557-1565</span>CODEN:
                <span class="NLM_cas:coden">ACHRE4</span>;
        ISSN:<span class="NLM_cas:issn">0001-4842</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Among the various mols. that plants produce for defense against pests and pathogens, cyclotides stand out as exceptionally stable and structurally unique.  These ribosomally synthesized peptides are around 30 amino acids in size, and are stabilized by a head-to-tail cyclic peptide backbone and three disulfide bonds that form a cystine knot.  They occur in certain plants of the Rubiaceae, Violaceae, Cucurbitaceae, Fabaceae, and Solanaceae families, with an individual plant producing up to hundreds of different cyclotides.  Aside from being exploitable as crop protection agents based on their natural pesticidal activities, cyclotides are amenable to repurposing by chemists for use as drug leads or as tools in chem. biol.  Their macrocyclic peptide backbone and knotted arrangement of three disulfide bonds engenders cyclotides with resistance to proteolytic degrdn., high temps., and chem. chaotropes.  Furthermore, their small size makes them accessible to synthesis using solid-phase peptide chem. and so nonnatural cyclotides can be designed and synthesized for a variety of applications.  The focus here is on cyclotides as tools in chem. biol., and there are four main areas of application that have appeared in the literature so far: (i) cyclotides as probes of membrane binding; (ii) cyclotides as probes of biosynthetic pathways for peptide cyclization; (iii) cyclotides as probes of protease specificity and function; and (iv) cyclotides as probes of receptor binding and specificity, with the potential for them to be developed as drug leads.  The main methods used in these studies include solid-phase peptide chem. for synthesis and NMR spectroscopy for structural characterization, as well as a wide range of biochem. and biophys. techniques for probing intermol. interactions.  In addn., cyclotides have been examd. in diverse biol. assays, ranging from enzyme inhibition to cell penetration, intracellular targeting and cytotoxicity.  The main finding to have emerged from studies over the past decade is that cyclotides are exceptionally stable under a variety of conditions (in assay buffers, biol. fluids, membranes, and recombinant expression systems).  Furthermore, they are structurally very well-defined and amenable to sequence substitutions that can introduce new desired biol. activities, generally without compromising their exceptional stability.  Both features contribute to their use as peptide-based frameworks in drug design.  Finally, they occupy a size niche between traditional small-mol. drugs (<500 Da in mol. wt.) and protein-based biologics (>5000 Da) and thus can probe receptors, membranes, and protein-protein interactions in different ways to what is possible with either small mols. or biologics.  Overall, cyclotides are an exciting class of peptides that have great potential as ultrastable chem. biol. probes in a variety of applications.  They have the advantage of specificity (typical of proteins) combined with the synthetic accessibility of small mols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqkJnwpjFMOVrVg90H21EOLACvtfcHk0lhPADm-Qps2Pg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtVCgsL7N&md5=18517c515d7b5018fc7744f534d5a4e2</span></div><a href="/servlet/linkout?suffix=cit153&amp;dbid=16384&amp;doi=10.1021%2Facs.accounts.7b00157&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.accounts.7b00157%26sid%3Dliteratum%253Aachs%26aulast%3DDe%2BVeer%26aufirst%3DS.%2BJ.%26aulast%3DWeidmann%26aufirst%3DJ.%26aulast%3DCraik%26aufirst%3DD.%2BJ.%26atitle%3DCyclotides%2520as%2520Tools%2520in%2520Chemical%2520Biology%26jtitle%3DAcc.%2520Chem.%2520Res.%26date%3D2017%26volume%3D50%26spage%3D1557%26epage%3D1565%26doi%3D10.1021%2Facs.accounts.7b00157" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref154"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref154'); return false;" data-citation="" class="refNumLink">154</a></strong><div class="NLM_citation" id="cit154"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Camarero, J. A.</span></span> <span> </span><span class="NLM_article-title">Cyclotides, a Versatile Ultrastable Micro-Protein Scaffold for Biotechnological Applications</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">5089</span>– <span class="NLM_lpage">5099</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2017.10.051</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref154/cit154&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1016%2Fj.bmcl.2017.10.051" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref154/cit154&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=29110985" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref154/cit154&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhslemsLrO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2017&pages=5089-5099&author=J.+A.+Camarero&title=Cyclotides%2C+a+Versatile+Ultrastable+Micro-Protein+Scaffold+for+Biotechnological+Applications&doi=10.1016%2Fj.bmcl.2017.10.051"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit154R"><div class="casContent"><span class="casTitleNuber">154</span><div class="casTitle"><span class="NLM_cas:atitle">Cyclotides, a versatile ultrastable micro-protein scaffold for biotechnological applications</span></div><div class="casAuthors">Camarero, Julio A.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">5089-5099</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Cyclotides are fascinating microproteins (≈30-40 residues long) with a unique head-to-tail cyclized backbone, stabilized by three disulfide bonds forming a cystine knot.  This unique topol. makes them exceptionally stable to chem., thermal and biol. degrdn. compared to other peptides of similar size.  Cyclotides have been also found to be highly tolerant to sequence variability, aside from the conserved residues forming the cystine knot, able to cross cellular membranes and modulate intracellular protein-protein interactions both in vitro and in vivo.  These properties make them ideal scaffolds for many biotechnol. applications.  This article provides and overview of the properties of cyclotides and their applications as mol. imaging agents and peptide-based therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoBl9-w1yHZYbVg90H21EOLACvtfcHk0ljiLuiNiXzpKA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhslemsLrO&md5=3e5aa9896809895d04eb1cc95d949498</span></div><a href="/servlet/linkout?suffix=cit154&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2017.10.051&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2017.10.051%26sid%3Dliteratum%253Aachs%26aulast%3DCamarero%26aufirst%3DJ.%2BA.%26atitle%3DCyclotides%252C%2520a%2520Versatile%2520Ultrastable%2520Micro-Protein%2520Scaffold%2520for%2520Biotechnological%2520Applications%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2017%26volume%3D27%26spage%3D5089%26epage%3D5099%26doi%3D10.1016%2Fj.bmcl.2017.10.051" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref155"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref155'); return false;" data-citation="" class="refNumLink">155</a></strong><div class="NLM_citation" id="cit155"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Eliasen, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daly, N. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wulff, B. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andresen, T. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Conde-Frieboes, K. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Craik, D. J.</span></span> <span> </span><span class="NLM_article-title">Design, Synthesis, Structural and Functional Characterization of Novel Melanocortin Agonists Based on the Cyclotide Kalata B1</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>287</i></span>,  <span class="NLM_fpage">40493</span>– <span class="NLM_lpage">40501</span>, <span class="refDoi"> DOI: 10.1074/jbc.M112.395442</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref155/cit155&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1074%2Fjbc.M112.395442" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref155/cit155&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=23012369" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref155/cit155&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC38XhslansbbF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=287&publication_year=2012&pages=40493-40501&author=R.+Eliasenauthor=N.+L.+Dalyauthor=B.+S.+Wulffauthor=T.+L.+Andresenauthor=K.+W.+Conde-Frieboesauthor=D.+J.+Craik&title=Design%2C+Synthesis%2C+Structural+and+Functional+Characterization+of+Novel+Melanocortin+Agonists+Based+on+the+Cyclotide+Kalata+B1&doi=10.1074%2Fjbc.M112.395442"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit155R"><div class="casContent"><span class="casTitleNuber">155</span><div class="casTitle"><span class="NLM_cas:atitle">Design, Synthesis, Structural and Functional Characterization of Novel Melanocortin Agonists Based on the Cyclotide Kalata B1</span></div><div class="casAuthors">Eliasen, Rasmus; Daly, Norelle L.; Wulff, Birgitte S.; Andresen, Thomas L.; Conde-Frieboes, Kilian W.; Craik, David J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">287</span>
        (<span class="NLM_cas:issue">48</span>),
    <span class="NLM_cas:pages">40493-40501</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Obesity is an increasingly important global health problem that lacks current treatment options.  The melanocortin receptor 4 (MC4R) is a target for obesity therapies because its activation triggers appetite suppression and increases energy expenditure.  Cyclotides have been suggested as scaffolds for the insertion and stabilization of pharmaceutically active peptides.  In this study, we explored the development of appetite-reducing peptides by synthesizing MC4R agonists based on the insertion of the His-Phe-Arg-Trp sequence into the cyclotide kalata B1.  The ability of the analogs to fold similarly to kalata B1 but display MC4R activity were investigated.  Four peptides were synthesized using t-butoxycarbonyl peptide chem. with a C-terminal thioester to facilitate backbone cyclization.  The structures of the peptides were found to be similar to kalata B1, evaluated by Hα NMR chem. shifts.  KB1(GHFRWG;23-28) had a Ki of 29 nm at the MC4R and was 107 or 314 times more selective over this receptor than MC1R or MC5R, resp., and had no detectable binding to MC3R.  The peptide had higher affinity for the MC4R than the endogenous agonist, α-melanocyte stimulation hormone, but it was less potent at the MC4R, with an EC50 of 580 nm for activation of the MC4R.  In conclusion, we synthesized melanocortin analogs of kalata B1 that preserve the structural scaffold and display receptor binding and functional activity.  KB1(GHFRWG;23-28) is potent and selective for the MC4R.  This compd. validates the use of cyclotides as scaffolds and has the potential to be a new lead for the treatment of obesity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq_sNylzkkWFLVg90H21EOLACvtfcHk0ljiLuiNiXzpKA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhslansbbF&md5=49425ad83d5075c1e375ee1fec3849ec</span></div><a href="/servlet/linkout?suffix=cit155&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M112.395442&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M112.395442%26sid%3Dliteratum%253Aachs%26aulast%3DEliasen%26aufirst%3DR.%26aulast%3DDaly%26aufirst%3DN.%2BL.%26aulast%3DWulff%26aufirst%3DB.%2BS.%26aulast%3DAndresen%26aufirst%3DT.%2BL.%26aulast%3DConde-Frieboes%26aufirst%3DK.%2BW.%26aulast%3DCraik%26aufirst%3DD.%2BJ.%26atitle%3DDesign%252C%2520Synthesis%252C%2520Structural%2520and%2520Functional%2520Characterization%2520of%2520Novel%2520Melanocortin%2520Agonists%2520Based%2520on%2520the%2520Cyclotide%2520Kalata%2520B1%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2012%26volume%3D287%26spage%3D40493%26epage%3D40501%26doi%3D10.1074%2Fjbc.M112.395442" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref156"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref156'); return false;" data-citation="" class="refNumLink">156</a></strong><div class="NLM_citation" id="cit156"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ji, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Majumder, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Millard, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Borra, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bi, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elnagar, A. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neamati, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shekhtman, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Camarero, J. A.</span></span> <span> </span><span class="NLM_article-title">In Vivo Activation of the p53 Tumor Suppressor Pathway by an Engineered Cyclotide</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>135</i></span>,  <span class="NLM_fpage">11623</span>– <span class="NLM_lpage">11633</span>, <span class="refDoi"> DOI: 10.1021/ja405108p</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja405108p" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref156/cit156&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtFSis7nL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=135&publication_year=2013&pages=11623-11633&author=Y.+Jiauthor=S.+Majumderauthor=M.+Millardauthor=R.+Borraauthor=T.+Biauthor=A.+Y.+Elnagarauthor=N.+Neamatiauthor=A.+Shekhtmanauthor=J.+A.+Camarero&title=In+Vivo+Activation+of+the+p53+Tumor+Suppressor+Pathway+by+an+Engineered+Cyclotide&doi=10.1021%2Fja405108p"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit156R"><div class="casContent"><span class="casTitleNuber">156</span><div class="casTitle"><span class="NLM_cas:atitle">In Vivo Activation of the p53 Tumor Suppressor Pathway by an Engineered Cyclotide</span></div><div class="casAuthors">Ji, Yanbin; Majumder, Subhabrata; Millard, Melissa; Borra, Radhika; Bi, Tao; Elnagar, Ahmed Y.; Neamati, Nouri; Shekhtman, Alexander; Camarero, Julio A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">135</span>
        (<span class="NLM_cas:issue">31</span>),
    <span class="NLM_cas:pages">11623-11633</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The overexpression of Hdm2 and HdmX is a common mechanism used by many tumor cells to inactive the p53 tumor suppressor pathway promoting cell survival.  Targeting Hdm2 and HdmX has emerged as a validated therapeutic strategy for treating cancers with wild-type p53.  Small linear peptides mimicking the N-terminal fragment of p53 have been shown to be potent Hdm2/HdmX antagonists.  The potential therapeutic use of these peptides, however, is limited by their poor stability and bioavailability.  Here, the authors report the engineering of the cyclotide MCoTI-I to efficiently antagonize intracellular p53 degrdn.  The resulting cyclotide MCo-PMI was able to bind with low nanomolar affinity to both Hdm2 and HdmX, showed high stability in human serum, and was cytotoxic to wild-type p53 cancer cell lines by activating the p53 tumor suppressor pathway both in vitro and in vivo.  These features make the cyclotide MCoTI-I an optimal scaffold for targeting intracellular protein-protein interactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGra-dIq75V8XLVg90H21EOLACvtfcHk0ljiLuiNiXzpKA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtFSis7nL&md5=c2d41067e876cb97bcf3a5a21bebce17</span></div><a href="/servlet/linkout?suffix=cit156&amp;dbid=16384&amp;doi=10.1021%2Fja405108p&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja405108p%26sid%3Dliteratum%253Aachs%26aulast%3DJi%26aufirst%3DY.%26aulast%3DMajumder%26aufirst%3DS.%26aulast%3DMillard%26aufirst%3DM.%26aulast%3DBorra%26aufirst%3DR.%26aulast%3DBi%26aufirst%3DT.%26aulast%3DElnagar%26aufirst%3DA.%2BY.%26aulast%3DNeamati%26aufirst%3DN.%26aulast%3DShekhtman%26aufirst%3DA.%26aulast%3DCamarero%26aufirst%3DJ.%2BA.%26atitle%3DIn%2520Vivo%2520Activation%2520of%2520the%2520p53%2520Tumor%2520Suppressor%2520Pathway%2520by%2520an%2520Engineered%2520Cyclotide%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2013%26volume%3D135%26spage%3D11623%26epage%3D11633%26doi%3D10.1021%2Fja405108p" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref157"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref157'); return false;" data-citation="" class="refNumLink">157</a></strong><div class="NLM_citation" id="cit157"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mullard, A.</span></span> <span> </span><span class="NLM_article-title">The Drug-Maker’s Guide to the Galaxy</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>549</i></span>,  <span class="NLM_fpage">445</span>– <span class="NLM_lpage">447</span>, <span class="refDoi"> DOI: 10.1038/549445a</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref157/cit157&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1038%2F549445a" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref157/cit157&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=28959982" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref157/cit157&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A280%3ADC%252BC1M%252FjsFSmtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=549&publication_year=2017&pages=445-447&author=A.+Mullard&title=The+Drug-Maker%E2%80%99s+Guide+to+the+Galaxy&doi=10.1038%2F549445a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit157R"><div class="casContent"><span class="casTitleNuber">157</span><div class="casTitle"><span class="NLM_cas:atitle">The drug-maker's guide to the galaxy</span></div><div class="casAuthors">Mullard Asher</div><div class="citationInfo"><span class="NLM_cas:title">Nature</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">549</span>
        (<span class="NLM_cas:issue">7673</span>),
    <span class="NLM_cas:pages">445-447</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSItGWG-TCuJEg6wdEZmT2rfW6udTcc2ebPW8r_RA5DQrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1M%252FjsFSmtA%253D%253D&md5=4ff8347906c8f3b459d05a5fe7ad0b56</span></div><a href="/servlet/linkout?suffix=cit157&amp;dbid=16384&amp;doi=10.1038%2F549445a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F549445a%26sid%3Dliteratum%253Aachs%26aulast%3DMullard%26aufirst%3DA.%26atitle%3DThe%2520Drug-Maker%25E2%2580%2599s%2520Guide%2520to%2520the%2520Galaxy%26jtitle%3DNature%26date%3D2017%26volume%3D549%26spage%3D445%26epage%3D447%26doi%3D10.1038%2F549445a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref158"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref158'); return false;" data-citation="" class="refNumLink">158</a></strong><div class="NLM_citation" id="cit158"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pertusi, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Donnell, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Homsher, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Solly, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stahler, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riley, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Finley, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Finger, E. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adam, G. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bell</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zuck, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hudak, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weber, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nothstein, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Locco, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quinn, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amoss, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Squadroni, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hartnett, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heo, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">White, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">May, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boots, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cocchiarella, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wolicki, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kreamer, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kutchukian, P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wassermann, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uebele, V. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glick, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rusinko, A.,  III</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Culberson, J. C.</span></span> <span> </span><span class="NLM_article-title">Prospective Assessment of Virtual Screening Heuristics Derived Using a Novel Fusion Score</span>. <i>SLAS Discovery</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">995</span>– <span class="NLM_lpage">1006</span>, <span class="refDoi"> DOI: 10.1177/2472555217706058</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref158/cit158&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1177%2F2472555217706058" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2017&pages=995-1006&author=D.+A.+Pertusiauthor=G.+O%E2%80%99Donnellauthor=M.+F.+Homsherauthor=K.+Sollyauthor=A.+Patelauthor=S.+L.+Stahlerauthor=D.+Rileyauthor=M.+F.+Finleyauthor=E.+N.+Fingerauthor=G.+C.+Adamauthor=J.+Mengauthor=+Bellauthor=P.+D.+Zuckauthor=E.+M.+Hudakauthor=M.+J.+Weberauthor=J.+E.+Nothsteinauthor=L.+Loccoauthor=C.+Quinnauthor=A.+Amossauthor=B.+Squadroniauthor=M.+Hartnettauthor=M.+R.+Heoauthor=T.+Whiteauthor=S.+A.+Mayauthor=E.+Bootsauthor=K.+Robertsauthor=P.+Cocchiarellaauthor=A.+Wolickiauthor=A.+Kreamerauthor=P.+S.+Kutchukianauthor=A.+M.+Wassermannauthor=V.+N.+Uebeleauthor=M.+Glickauthor=A.+Rusinkoauthor=J.+C.+Culberson&title=Prospective+Assessment+of+Virtual+Screening+Heuristics+Derived+Using+a+Novel+Fusion+Score&doi=10.1177%2F2472555217706058"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit158&amp;dbid=16384&amp;doi=10.1177%2F2472555217706058&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F2472555217706058%26sid%3Dliteratum%253Aachs%26aulast%3DPertusi%26aufirst%3DD.%2BA.%26aulast%3DO%25E2%2580%2599Donnell%26aufirst%3DG.%26aulast%3DHomsher%26aufirst%3DM.%2BF.%26aulast%3DSolly%26aufirst%3DK.%26aulast%3DPatel%26aufirst%3DA.%26aulast%3DStahler%26aufirst%3DS.%2BL.%26aulast%3DRiley%26aufirst%3DD.%26aulast%3DFinley%26aufirst%3DM.%2BF.%26aulast%3DFinger%26aufirst%3DE.%2BN.%26aulast%3DAdam%26aufirst%3DG.%2BC.%26aulast%3DMeng%26aufirst%3DJ.%26aulast%3DBell%26aulast%3DZuck%26aufirst%3DP.%2BD.%26aulast%3DHudak%26aufirst%3DE.%2BM.%26aulast%3DWeber%26aufirst%3DM.%2BJ.%26aulast%3DNothstein%26aufirst%3DJ.%2BE.%26aulast%3DLocco%26aufirst%3DL.%26aulast%3DQuinn%26aufirst%3DC.%26aulast%3DAmoss%26aufirst%3DA.%26aulast%3DSquadroni%26aufirst%3DB.%26aulast%3DHartnett%26aufirst%3DM.%26aulast%3DHeo%26aufirst%3DM.%2BR.%26aulast%3DWhite%26aufirst%3DT.%26aulast%3DMay%26aufirst%3DS.%2BA.%26aulast%3DBoots%26aufirst%3DE.%26aulast%3DRoberts%26aufirst%3DK.%26aulast%3DCocchiarella%26aufirst%3DP.%26aulast%3DWolicki%26aufirst%3DA.%26aulast%3DKreamer%26aufirst%3DA.%26aulast%3DKutchukian%26aufirst%3DP.%2BS.%26aulast%3DWassermann%26aufirst%3DA.%2BM.%26aulast%3DUebele%26aufirst%3DV.%2BN.%26aulast%3DGlick%26aufirst%3DM.%26aulast%3DRusinko%26aufirst%3DA.%26aulast%3DCulberson%26aufirst%3DJ.%2BC.%26atitle%3DProspective%2520Assessment%2520of%2520Virtual%2520Screening%2520Heuristics%2520Derived%2520Using%2520a%2520Novel%2520Fusion%2520Score%26jtitle%3DSLAS%2520Discovery%26date%3D2017%26volume%3D22%26spage%3D995%26epage%3D1006%26doi%3D10.1177%2F2472555217706058" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref159"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref159'); return false;" data-citation="" class="refNumLink">159</a></strong><div class="NLM_citation" id="cit159"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Martin, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Polyakov, V. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tian, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perez, R. C.</span></span> <span> </span><span class="NLM_article-title">Profile-QSAR 2.0: Kinase Virtual Screening Accuracy Comparable to Four-Concentration IC50s for Realistically Novel Compounds</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">2077</span>– <span class="NLM_lpage">2088</span>, <span class="refDoi"> DOI: 10.1021/acs.jcim.7b00166</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.7b00166" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref159/cit159&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtVKgsLjF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2017&pages=2077-2088&author=E.+J.+Martinauthor=V.+R.+Polyakovauthor=L.+Tianauthor=R.+C.+Perez&title=Profile-QSAR+2.0%3A+Kinase+Virtual+Screening+Accuracy+Comparable+to+Four-Concentration+IC50s+for+Realistically+Novel+Compounds&doi=10.1021%2Facs.jcim.7b00166"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit159R"><div class="casContent"><span class="casTitleNuber">159</span><div class="casTitle"><span class="NLM_cas:atitle">Profile-QSAR 2.0: Kinase Virtual Screening Accuracy Comparable to Four-Concentration IC50s for Realistically Novel Compounds</span></div><div class="casAuthors">Martin, Eric J.; Polyakov, Valery R.; Tian, Li; Perez, Rolando C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2077-2088</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">While conventional random forest regression (RFR) virtual screening models appear to have excellent accuracy on random held-out test sets, they prove lacking in actual practice.  Anal. of 18 historical virtual screens showed that random test sets are far more similar to their training sets than are the compds. project teams actually order.  A new, cluster-based "realistic" training/test set split, which mirrors the chem. novelty of real-life virtual screens, recapitulates the poor predictive power of RFR models in real projects.  The original Profile-QSAR (pQSAR) method greatly broadened the domain of applicability over conventional models by using as independent variables a profile of activity predictions from all historical assays in a large protein family.  However, the accuracy still fell short of expt. on the realistic test sets.  The improved "pQSAR 2.0" method replaces probabilities of activity from naive Bayes categorical models at several thresholds with predicted IC50s from RFR models.  Unexpectedly, the high accuracy also requires removing the RFR model for the actual assay of interest from the independent variable profile.  With these improvements, pQSAR 2.0 activity predictions are now statistically comparable to medium-throughput 4-concn. IC50 measurements even on the realistic test set.  Beyond the yes/no activity predictions from a typical high-throughput screen (HTS) or conventional virtual screen, these semi-quant. IC50 predictions allow for predicted potency, ligand efficiency, lipophilic efficiency and selectivity against antitargets, greatly facilitating hit list triaging and enabling virtual screening panels such as toxicity panels and overall promiscuity predictions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoURHgPoAj7n7Vg90H21EOLACvtfcHk0lhXCe0qn5gZtw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtVKgsLjF&md5=3e215174b6d4b89f490f79e61c893ee1</span></div><a href="/servlet/linkout?suffix=cit159&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.7b00166&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.7b00166%26sid%3Dliteratum%253Aachs%26aulast%3DMartin%26aufirst%3DE.%2BJ.%26aulast%3DPolyakov%26aufirst%3DV.%2BR.%26aulast%3DTian%26aufirst%3DL.%26aulast%3DPerez%26aufirst%3DR.%2BC.%26atitle%3DProfile-QSAR%25202.0%253A%2520Kinase%2520Virtual%2520Screening%2520Accuracy%2520Comparable%2520to%2520Four-Concentration%2520IC50s%2520for%2520Realistically%2520Novel%2520Compounds%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2017%26volume%3D57%26spage%3D2077%26epage%3D2088%26doi%3D10.1021%2Facs.jcim.7b00166" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref160"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref160'); return false;" data-citation="" class="refNumLink">160</a></strong><div class="NLM_citation" id="cit160"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bohacek, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McMartin, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guida, W. C.</span></span> <span> </span><span class="NLM_article-title">The Art and Practice of Structure-Based Drug Design: A Molecular Modeling Perspective</span>. <i>Med. Res. Rev.</i> <span class="NLM_year" style="font-weight: bold;">1996</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">3</span>– <span class="NLM_lpage">50</span>, <span class="refDoi"> DOI: 10.1002/(SICI)1098-1128(199601)16:1<3::AID-MED1>3.0.CO;2-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref160/cit160&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1002%2F%28SICI%291098-1128%28199601%2916%3A1%3C3%3A%3AAID-MED1%3E3.0.CO%3B2-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref160/cit160&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=8788213" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref160/cit160&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADyaK28XhtFyls78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=1996&pages=3-50&author=R.+S.+Bohacekauthor=C.+McMartinauthor=W.+C.+Guida&title=The+Art+and+Practice+of+Structure-Based+Drug+Design%3A+A+Molecular+Modeling+Perspective&doi=10.1002%2F%28SICI%291098-1128%28199601%2916%3A1%3C3%3A%3AAID-MED1%3E3.0.CO%3B2-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit160R"><div class="casContent"><span class="casTitleNuber">160</span><div class="casTitle"><span class="NLM_cas:atitle">The art and practice of structure-based drug design: a molecular modeling perspective</span></div><div class="casAuthors">Bohacek, Regine S.; McMartin, Colin; Guida, Wayne C.</div><div class="citationInfo"><span class="NLM_cas:title">Medicinal Research Reviews</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">3-50</span>CODEN:
                <span class="NLM_cas:coden">MRREDD</span>;
        ISSN:<span class="NLM_cas:issn">0198-6325</span>.
    
            (<span class="NLM_cas:orgname">Wiley</span>)
        </div><div class="casAbstract">A review with 120 refs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrmfrWdtPosdrVg90H21EOLACvtfcHk0ljyv8JpJ1pT6g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28XhtFyls78%253D&md5=edfa592791c111479117feed600dead6</span></div><a href="/servlet/linkout?suffix=cit160&amp;dbid=16384&amp;doi=10.1002%2F%28SICI%291098-1128%28199601%2916%3A1%3C3%3A%3AAID-MED1%3E3.0.CO%3B2-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F%2528SICI%25291098-1128%2528199601%252916%253A1%253C3%253A%253AAID-MED1%253E3.0.CO%253B2-6%26sid%3Dliteratum%253Aachs%26aulast%3DBohacek%26aufirst%3DR.%2BS.%26aulast%3DMcMartin%26aufirst%3DC.%26aulast%3DGuida%26aufirst%3DW.%2BC.%26atitle%3DThe%2520Art%2520and%2520Practice%2520of%2520Structure-Based%2520Drug%2520Design%253A%2520A%2520Molecular%2520Modeling%2520Perspective%26jtitle%3DMed.%2520Res.%2520Rev.%26date%3D1996%26volume%3D16%26spage%3D3%26epage%3D50%26doi%3D10.1002%2F%28SICI%291098-1128%28199601%2916%3A1%3C3%3A%3AAID-MED1%3E3.0.CO%3B2-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref161"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref161'); return false;" data-citation="" class="refNumLink">161</a></strong><div class="NLM_citation" id="cit161"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Van Hoorn, W. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bell, A. S.</span></span> <span> </span><span class="NLM_article-title">Searching Chemical Space with the Bayesian Idea Generator</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>49</i></span>,  <span class="NLM_fpage">2211</span>– <span class="NLM_lpage">2220</span>, <span class="refDoi"> DOI: 10.1021/ci900072g</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci900072g" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref161/cit161&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtFyqsrfJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2009&pages=2211-2220&author=W.+P.+Van+Hoornauthor=A.+S.+Bell&title=Searching+Chemical+Space+with+the+Bayesian+Idea+Generator&doi=10.1021%2Fci900072g"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit161R"><div class="casContent"><span class="casTitleNuber">161</span><div class="casTitle"><span class="NLM_cas:atitle">Searching Chemical Space with the Bayesian Idea Generator</span></div><div class="casAuthors">van Hoorn, Willem P.; Bell, Andrew S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2211-2220</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The Pfizer Global Virtual Library (PGVL) is defined as a set compds. that could be synthesized using validated protocols and monomers.  However, it is too large (1012 compds.) to search by brute-force methods for close analogs of a given input structure.  In this paper the Bayesian Idea Generator is described which is based on a novel application of Bayesian statistics to narrow down the search space to a prioritized set of existing library arrays (the default is 16).  For each of these libraries the 6 closest neighbors are retrieved from the existing compd. file, resulting in a screenable hypothesis of 96 compds.  Using the Bayesian models for library space, the Pfizer file of singleton compds. has been mapped to library space and is optionally searched as well.  The method is >99% accurate in retrieving known library provenance from an independent test set.  The compds. retrieved strike a balance between similarity and diversity resulting in frequent scaffold hops.  Four examples of how the Bayesian Idea Generator has been successfully used in drug discovery are provided.  The methodol. of the Bayesian Idea Generator can be used for any collection of compds. contg. distinct clusters, and an example using compd. vendor catalogues has been included.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo8rVsn6tLJ5LVg90H21EOLACvtfcHk0ljyv8JpJ1pT6g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtFyqsrfJ&md5=dbd7bacc3ce21b2d0af8f5d8492dd990</span></div><a href="/servlet/linkout?suffix=cit161&amp;dbid=16384&amp;doi=10.1021%2Fci900072g&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci900072g%26sid%3Dliteratum%253Aachs%26aulast%3DVan%2BHoorn%26aufirst%3DW.%2BP.%26aulast%3DBell%26aufirst%3DA.%2BS.%26atitle%3DSearching%2520Chemical%2520Space%2520with%2520the%2520Bayesian%2520Idea%2520Generator%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2009%26volume%3D49%26spage%3D2211%26epage%3D2220%26doi%3D10.1021%2Fci900072g" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref162"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref162'); return false;" data-citation="" class="refNumLink">162</a></strong><div class="NLM_citation" id="cit162"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Allen, B. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mehta, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ember, S. W. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schonbrunn, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ayad, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schürer, S. C.</span></span> <span> </span><span class="NLM_article-title">Large-Scale Computational Screening Identifies First in Class Multitarget Inhibitor of EGFR Kinase and BRD4</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">16924</span>, <span class="refDoi"> DOI: 10.1038/srep16924</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref162/cit162&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1038%2Fsrep16924" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref162/cit162&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=26596901" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref162/cit162&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvFentrrI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2015&pages=16924&author=B.+K.+Allenauthor=S.+Mehtaauthor=S.+W.+J.+Emberauthor=E.+Schonbrunnauthor=N.+Ayadauthor=S.+C.+Sch%C3%BCrer&title=Large-Scale+Computational+Screening+Identifies+First+in+Class+Multitarget+Inhibitor+of+EGFR+Kinase+and+BRD4&doi=10.1038%2Fsrep16924"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit162R"><div class="casContent"><span class="casTitleNuber">162</span><div class="casTitle"><span class="NLM_cas:atitle">Large-Scale Computational Screening Identifies First in Class Multitarget Inhibitor of EGFR Kinase and BRD4</span></div><div class="casAuthors">Allen, Bryce K.; Mehta, Saurabh; Ember, Stewart W. J.; Schonbrunn, Ernst; Ayad, Nagi; Schurer, Stephan C.</div><div class="citationInfo"><span class="NLM_cas:title">Scientific Reports</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">16924</span>CODEN:
                <span class="NLM_cas:coden">SRCEC3</span>;
        ISSN:<span class="NLM_cas:issn">2045-2322</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Inhibition of cancer-promoting kinases is an established therapeutic strategy for the treatment of many cancers, although resistance to kinase inhibitors is common.  One way to overcome resistance is to target orthogonal cancer-promoting pathways.  Bromo and Extra-Terminal (BET) domain proteins, which belong to the family of epigenetic readers, have recently emerged as promising therapeutic targets in multiple cancers.  The development of multitarget drugs that inhibit kinase and BET proteins therefore may be a promising strategy to overcome tumor resistance and prolong therapeutic efficacy in the clinic.  We developed a general computational screening approach to identify novel dual kinase/bromodomain inhibitors from millions of com. available small mols.  Our method integrated machine learning using big datasets of kinase inhibitors and structure-based drug design.  Here we describe the computational methodol., including validation and characterization of our models and their application and integration into a scalable virtual screening pipeline.  We screened over 6 million com. available compds. and selected 24 for testing in BRD4 and EGFR biochem. assays.  We identified several novel BRD4 inhibitors, among them a first in class dual EGFR-BRD4 inhibitor.  Our studies suggest that this computational screening approach may be broadly applicable for identifying dual kinase/BET inhibitors with potential for treating various cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr2LBJWDL9se7Vg90H21EOLACvtfcHk0ljyv8JpJ1pT6g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvFentrrI&md5=341f9fedbdfd9e66829e5c43d602d510</span></div><a href="/servlet/linkout?suffix=cit162&amp;dbid=16384&amp;doi=10.1038%2Fsrep16924&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsrep16924%26sid%3Dliteratum%253Aachs%26aulast%3DAllen%26aufirst%3DB.%2BK.%26aulast%3DMehta%26aufirst%3DS.%26aulast%3DEmber%26aufirst%3DS.%2BW.%2BJ.%26aulast%3DSchonbrunn%26aufirst%3DE.%26aulast%3DAyad%26aufirst%3DN.%26aulast%3DSch%25C3%25BCrer%26aufirst%3DS.%2BC.%26atitle%3DLarge-Scale%2520Computational%2520Screening%2520Identifies%2520First%2520in%2520Class%2520Multitarget%2520Inhibitor%2520of%2520EGFR%2520Kinase%2520and%2520BRD4%26jtitle%3DSci.%2520Rep.%26date%3D2015%26volume%3D5%26spage%3D16924%26doi%3D10.1038%2Fsrep16924" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref163"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref163'); return false;" data-citation="" class="refNumLink">163</a></strong><div class="NLM_citation" id="cit163"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Boehm, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, T.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Claussen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lemmen, C.</span></span> <span> </span><span class="NLM_article-title">Similarity Searching and Scaffold Hopping in Synthetically Accessible Combinatorial Chemistry Spaces</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>51</i></span>,  <span class="NLM_fpage">2468</span>– <span class="NLM_lpage">2480</span>, <span class="refDoi"> DOI: 10.1021/jm0707727</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm0707727" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref163/cit163&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BD1cXktFShtL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2008&pages=2468-2480&author=M.+Boehmauthor=T.-Y.+Wuauthor=H.+Claussenauthor=C.+Lemmen&title=Similarity+Searching+and+Scaffold+Hopping+in+Synthetically+Accessible+Combinatorial+Chemistry+Spaces&doi=10.1021%2Fjm0707727"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit163R"><div class="casContent"><span class="casTitleNuber">163</span><div class="casTitle"><span class="NLM_cas:atitle">Similarity searching and scaffold hopping in synthetically accessible combinatorial chemistry spaces</span></div><div class="casAuthors">Boehm, Markus; Wu, Tong-Ying; Claussen, Holger; Lemmen, Christian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2468-2480</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Large collections of combinatorial libraries are an integral element in today's pharmaceutical industry.  It is of great interest to perform similarity searches against all virtual compds. that are synthetically accessible by any such library.  Here we describe the successful application of a new software tool CoLibri on 358 combinatorial libraries based on validated reaction protocols to create a single chem. space contg. over 1012 possible products.  Similarity searching with FTrees-FS allows the systematic exploration of this space without the need to enumerate all product structures.  The search result is a set of virtual hits which are synthetically accessible by one or more of the existing reaction protocols.  Grouping these virtual hits by their synthetic protocols allows the rapid design and synthesis of multiple follow-up libraries.  Such library ideas support hit-to-lead design efforts for tasks like follow-up from high-throughput screening hits or scaffold hopping from one hit to another attractive series.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrW_edQq9zwE7Vg90H21EOLACvtfcHk0lhIWSAWeJjLZQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXktFShtL0%253D&md5=c42c419eecf6d272820ed5237f8c1275</span></div><a href="/servlet/linkout?suffix=cit163&amp;dbid=16384&amp;doi=10.1021%2Fjm0707727&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0707727%26sid%3Dliteratum%253Aachs%26aulast%3DBoehm%26aufirst%3DM.%26aulast%3DWu%26aufirst%3DT.-Y.%26aulast%3DClaussen%26aufirst%3DH.%26aulast%3DLemmen%26aufirst%3DC.%26atitle%3DSimilarity%2520Searching%2520and%2520Scaffold%2520Hopping%2520in%2520Synthetically%2520Accessible%2520Combinatorial%2520Chemistry%2520Spaces%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2008%26volume%3D51%26spage%3D2468%26epage%3D2480%26doi%3D10.1021%2Fjm0707727" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref164"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref164'); return false;" data-citation="" class="refNumLink">164</a></strong><div class="NLM_citation" id="cit164"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lessel, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wellenzohn, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lilienthal, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Claussen, H.</span></span> <span> </span><span class="NLM_article-title">Searching Fragment Spaces with Feature Trees</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>49</i></span>,  <span class="NLM_fpage">270</span>– <span class="NLM_lpage">279</span>, <span class="refDoi"> DOI: 10.1021/ci800272a</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci800272a" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref164/cit164&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BD1MXpt1aluw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2009&pages=270-279&author=U.+Lesselauthor=B.+Wellenzohnauthor=M.+Lilienthalauthor=H.+Claussen&title=Searching+Fragment+Spaces+with+Feature+Trees&doi=10.1021%2Fci800272a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit164R"><div class="casContent"><span class="casTitleNuber">164</span><div class="casTitle"><span class="NLM_cas:atitle">Searching Fragment Spaces with Feature Trees</span></div><div class="casAuthors">Lessel, Uta; Wellenzohn, Bernd; Lilienthal, Markus; Claussen, Holger</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">270-279</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Virtual combinatorial chem. easily produces billions of compds., for which conventional virtual screening cannot be performed even with the fastest methods available.  An efficient soln. for such a scenario is the generation of Fragment Spaces, which encode huge nos. of virtual compds. by their fragments/reagents and rules of how to combine them.  Similarity-based searches can be performed in such spaces without ever fully enumerating all virtual products.  Here we describe the generation of a huge Fragment Space encoding about 5*1011 compds. based on established inhouse synthesis protocols for combinatorial libraries, i.e., we encode practically evaluated combinatorial chem. protocols in a machine readable form, rendering them accessible to in silico search methods.  We show how such searches in this Fragment Space can be integrated as a first step in an overall work-flow.  It reduces the extremely huge no. of virtual products by several orders of magnitude so that the resulting list of mols. becomes more manageable for further more elaborated and time-consuming anal. steps.  Results of a case study are presented and discussed, which lead to some general conclusions for an efficient expansion of the chem. space to be screened in pharmaceutical companies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq16bJX_82TS7Vg90H21EOLACvtfcHk0lhIWSAWeJjLZQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXpt1aluw%253D%253D&md5=323989844a58818ff57cf1d6bf38fcdf</span></div><a href="/servlet/linkout?suffix=cit164&amp;dbid=16384&amp;doi=10.1021%2Fci800272a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci800272a%26sid%3Dliteratum%253Aachs%26aulast%3DLessel%26aufirst%3DU.%26aulast%3DWellenzohn%26aufirst%3DB.%26aulast%3DLilienthal%26aufirst%3DM.%26aulast%3DClaussen%26aufirst%3DH.%26atitle%3DSearching%2520Fragment%2520Spaces%2520with%2520Feature%2520Trees%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2009%26volume%3D49%26spage%3D270%26epage%3D279%26doi%3D10.1021%2Fci800272a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref165"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref165'); return false;" data-citation="" class="refNumLink">165</a></strong><div class="NLM_citation" id="cit165"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Peng, Z.</span></span> <span> </span><span class="NLM_article-title">Very Large Virtual Compound Spaces: Construction, Storage and Utility in Drug Discovery</span>. <i>Drug Discovery Today: Technol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">e387</span>– <span class="NLM_lpage">e394</span>, <span class="refDoi"> DOI: 10.1016/j.ddtec.2013.01.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref165/cit165&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1016%2Fj.ddtec.2013.01.004" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2013&pages=e387-e394&author=Z.+Peng&title=Very+Large+Virtual+Compound+Spaces%3A+Construction%2C+Storage+and+Utility+in+Drug+Discovery&doi=10.1016%2Fj.ddtec.2013.01.004"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit165&amp;dbid=16384&amp;doi=10.1016%2Fj.ddtec.2013.01.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ddtec.2013.01.004%26sid%3Dliteratum%253Aachs%26aulast%3DPeng%26aufirst%3DZ.%26atitle%3DVery%2520Large%2520Virtual%2520Compound%2520Spaces%253A%2520Construction%252C%2520Storage%2520and%2520Utility%2520in%2520Drug%2520Discovery%26jtitle%3DDrug%2520Discovery%2520Today%253A%2520Technol.%26date%3D2013%26volume%3D10%26spage%3De387%26epage%3De394%26doi%3D10.1016%2Fj.ddtec.2013.01.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref166"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref166'); return false;" data-citation="" class="refNumLink">166</a></strong><div class="NLM_citation" id="cit166"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vainio, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kogej, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raubacher, F.</span></span> <span> </span><span class="NLM_article-title">Automated Recycling of Chemistry for Virtual Screening and Library Design</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>52</i></span>,  <span class="NLM_fpage">1777</span>– <span class="NLM_lpage">1786</span>, <span class="refDoi"> DOI: 10.1021/ci300157m</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci300157m" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref166/cit166&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC38XnvFehuro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2012&pages=1777-1786&author=M.+J.+Vainioauthor=T.+Kogejauthor=F.+Raubacher&title=Automated+Recycling+of+Chemistry+for+Virtual+Screening+and+Library+Design&doi=10.1021%2Fci300157m"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit166R"><div class="casContent"><span class="casTitleNuber">166</span><div class="casTitle"><span class="NLM_cas:atitle">Automated Recycling of Chemistry for Virtual Screening and Library Design</span></div><div class="casAuthors">Vainio, Mikko J.; Kogej, Thierry; Raubacher, Florian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1777-1786</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">An early stage drug discovery project needs to identify a no. of chem. diverse and attractive compds.  These hit compds. are typically found through high-throughput screening campaigns.  The diversity of the chem. libraries used in screening is therefore important.  In this study, we describe a virtual high-throughput screening system called Virtual Library.  The system automatically "recycles" validated synthetic protocols and available starting materials to generate a large no. of virtual compd. libraries, and allows for fast searches in the generated libraries using a 2D fingerprint based screening method.  Virtual Library links the returned virtual hit compds. back to exptl. protocols to quickly assess the synthetic accessibility of the hits.  The system can be used as an idea generator for library design to enrich the screening collection and to explore the structure-activity landscape around a specific active compd.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGovsyO-M3z0jbVg90H21EOLACvtfcHk0lhIWSAWeJjLZQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XnvFehuro%253D&md5=15fc68eff3e7716aae6a809a94bb1c1c</span></div><a href="/servlet/linkout?suffix=cit166&amp;dbid=16384&amp;doi=10.1021%2Fci300157m&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci300157m%26sid%3Dliteratum%253Aachs%26aulast%3DVainio%26aufirst%3DM.%2BJ.%26aulast%3DKogej%26aufirst%3DT.%26aulast%3DRaubacher%26aufirst%3DF.%26atitle%3DAutomated%2520Recycling%2520of%2520Chemistry%2520for%2520Virtual%2520Screening%2520and%2520Library%2520Design%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2012%26volume%3D52%26spage%3D1777%26epage%3D1786%26doi%3D10.1021%2Fci300157m" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref167"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref167'); return false;" data-citation="" class="refNumLink">167</a></strong><div class="NLM_citation" id="cit167"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nicolaou, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watson, I. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span> <span> </span><span class="NLM_article-title">The Proximal Lilly Collection: Mapping, Exploring and Exploiting Feasible Chemical Space</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">1253</span>– <span class="NLM_lpage">1266</span>, <span class="refDoi"> DOI: 10.1021/acs.jcim.6b00173</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.6b00173" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref167/cit167&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC28XpsFKjs7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2016&pages=1253-1266&author=C.+A.+Nicolaouauthor=I.+A.+Watsonauthor=H.+Huauthor=J.+Wang&title=The+Proximal+Lilly+Collection%3A+Mapping%2C+Exploring+and+Exploiting+Feasible+Chemical+Space&doi=10.1021%2Facs.jcim.6b00173"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit167R"><div class="casContent"><span class="casTitleNuber">167</span><div class="casTitle"><span class="NLM_cas:atitle">The Proximal Lilly Collection: Mapping, Exploring and Exploiting Feasible Chemical Space</span></div><div class="casAuthors">Nicolaou, Christos A.; Watson, Ian A.; Hu, Hong; Wang, Jibo</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1253-1266</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Venturing into the immensity of the small mol. universe to identify novel chem. structure is a much discussed objective of many methods proposed by the chemoinformatics community.  To this end, numerous approaches using techniques from the fields of computational de novo design, virtual screening and reaction informatics, among others, have been proposed.  Although in principle this objective is commendable, in practice there are several obstacles to useful exploitation of the chem. space.  Prime among them are the sheer no. of theor. feasible compds. and the practical concern regarding the synthesizability of the chem. structures conceived using in silico methods.  We present the Proximal Lilly Collection initiative implemented at Eli Lilly and Co. with the aims to (i) define the chem. space of small, drug-like compds. that could be synthesized using inhouse resources and (ii) facilitate access to compds. in this large space for the purposes of ongoing drug discovery efforts.  The implementation of PLC relies on coupling access to available synthetic knowledge and resources with chemo/reaction informatics techniques and tools developed for this purpose.  We describe in detail the computational framework supporting this initiative and elaborate on the characteristics of the PLC virtual collection of compds.  As an example of the opportunities provided to drug discovery researchers by easy access to a large, realistically feasible virtual collection such as the PLC, we describe a recent application of the technol. that led to the discovery of selective kinase inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrRp50JUV8h5rVg90H21EOLACvtfcHk0lhIWSAWeJjLZQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XpsFKjs7o%253D&md5=4689df1c7e0d2eddd93b14db4ac3e4b8</span></div><a href="/servlet/linkout?suffix=cit167&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.6b00173&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.6b00173%26sid%3Dliteratum%253Aachs%26aulast%3DNicolaou%26aufirst%3DC.%2BA.%26aulast%3DWatson%26aufirst%3DI.%2BA.%26aulast%3DHu%26aufirst%3DH.%26aulast%3DWang%26aufirst%3DJ.%26atitle%3DThe%2520Proximal%2520Lilly%2520Collection%253A%2520Mapping%252C%2520Exploring%2520and%2520Exploiting%2520Feasible%2520Chemical%2520Space%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2016%26volume%3D56%26spage%3D1253%26epage%3D1266%26doi%3D10.1021%2Facs.jcim.6b00173" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref168"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref168'); return false;" data-citation="" class="refNumLink">168</a></strong><div class="NLM_citation" id="cit168"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Baranczak, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tu, N. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marjanovic, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Searle, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vasudevan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Djuric, S. W.</span></span> <span> </span><span class="NLM_article-title">Integrated Platform for Expedited Synthesis–Purification–Testing of Small Molecule Libraries</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">461</span>– <span class="NLM_lpage">465</span>, <span class="refDoi"> DOI: 10.1021/acsmedchemlett.7b00054</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.7b00054" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref168/cit168&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC2sXltVSqsr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=461-465&author=A.+Baranczakauthor=N.+P.+Tuauthor=J.+Marjanovicauthor=P.+A.+Searleauthor=A.+Vasudevanauthor=S.+W.+Djuric&title=Integrated+Platform+for+Expedited+Synthesis%E2%80%93Purification%E2%80%93Testing+of+Small+Molecule+Libraries&doi=10.1021%2Facsmedchemlett.7b00054"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit168R"><div class="casContent"><span class="casTitleNuber">168</span><div class="casTitle"><span class="NLM_cas:atitle">Integrated Platform for Expedited Synthesis-Purification-Testing of Small Molecule Libraries</span></div><div class="casAuthors">Baranczak, Aleksandra; Tu, Noah P.; Marjanovic, Jasmina; Searle, Philip A.; Vasudevan, Anil; Djuric, Stevan W.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">461-465</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The productivity of medicinal chem. programs can be significantly increased through the introduction of automation, leading to shortened discovery cycle times.  Herein, the authors describe a platform that consolidates synthesis, purifn., quantitation, dissoln., and testing of small mol. libraries.  The system was validated through the synthesis and testing of two libraries of binders of polycomb protein EED, and excellent correlation of obtained data with results generated through conventional approaches was obsd.  The fully automated and integrated platform enables batch-supported compd. synthesis based on a broad array of chem. transformations with testing in a variety of biochem. assay formats.  A library turnaround time of between 24 and 36 h was achieved, and notably, each library synthesis produces sufficient amts. of compds. for further evaluation in secondary assays thereby contributing significantly to the shortening of medicinal chem. discovery cycles.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrKCbl3UVpOhLVg90H21EOLACvtfcHk0lg3Mk1-JQg3eg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXltVSqsr0%253D&md5=7c88f92c6d6e39d77571202db4b7e82b</span></div><a href="/servlet/linkout?suffix=cit168&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.7b00054&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.7b00054%26sid%3Dliteratum%253Aachs%26aulast%3DBaranczak%26aufirst%3DA.%26aulast%3DTu%26aufirst%3DN.%2BP.%26aulast%3DMarjanovic%26aufirst%3DJ.%26aulast%3DSearle%26aufirst%3DP.%2BA.%26aulast%3DVasudevan%26aufirst%3DA.%26aulast%3DDjuric%26aufirst%3DS.%2BW.%26atitle%3DIntegrated%2520Platform%2520for%2520Expedited%2520Synthesis%25E2%2580%2593Purification%25E2%2580%2593Testing%2520of%2520Small%2520Molecule%2520Libraries%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2017%26volume%3D8%26spage%3D461%26epage%3D465%26doi%3D10.1021%2Facsmedchemlett.7b00054" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref169"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref169'); return false;" data-citation="" class="refNumLink">169</a></strong><div class="NLM_citation" id="cit169"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Perera, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tucker, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brahmbhatt, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Helal, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chong, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farrell, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sach, N. W.</span></span> <span> </span><span class="NLM_article-title">A Platform for Automated Nanomole-Scale Reaction Screening and Micromole-Scale Synthesis in Flow</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>359</i></span>,  <span class="NLM_fpage">429</span>– <span class="NLM_lpage">434</span>, <span class="refDoi"> DOI: 10.1126/science.aap9112</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref169/cit169&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1126%2Fscience.aap9112" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref169/cit169&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=29371464" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref169/cit169&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhsVGhsrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=359&publication_year=2018&pages=429-434&author=D.+Pereraauthor=J.+W.+Tuckerauthor=S.+Brahmbhattauthor=C.+J.+Helalauthor=A.+Chongauthor=W.+Farrellauthor=P.+Richardsonauthor=N.+W.+Sach&title=A+Platform+for+Automated+Nanomole-Scale+Reaction+Screening+and+Micromole-Scale+Synthesis+in+Flow&doi=10.1126%2Fscience.aap9112"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit169R"><div class="casContent"><span class="casTitleNuber">169</span><div class="casTitle"><span class="NLM_cas:atitle">A platform for automated nanomole-scale reaction screening and micromole-scale synthesis in flow</span></div><div class="casAuthors">Perera, Damith; Tucker, Joseph W.; Brahmbhatt, Shalini; Helal, Christopher J.; Chong, Ashley; Farrell, William; Richardson, Paul; Sach, Neal W.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">359</span>
        (<span class="NLM_cas:issue">6374</span>),
    <span class="NLM_cas:pages">429-434</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">The scarcity of complex intermediates in pharmaceutical research motivates the pursuit of reaction optimization protocols on submilligram scales.  We report here the development of an automated flow-based synthesis platform, designed from com. available components, that integrates both rapid nanomole-scale reaction screening and micromole-scale synthesis into a single modular unit.  This system was validated by exploring a diverse range of reaction variables in a Suzuki-Miyaura coupling on nanomole scale at elevated temps., generating liq. chromatog.-mass spectrometry data points for 5760 reactions at a rate of >1500 reactions per 24 h.  Through multiple injections of the same segment, the system directly produced micromole quantities of desired material.  The optimal conditions were also replicated in traditional flow and batch mode at 50- to 200-mg scale to provide good to excellent yields.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpc_fiVqt95HLVg90H21EOLACvtfcHk0lg3Mk1-JQg3eg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhsVGhsrc%253D&md5=3c288152644b3574372bf5ef3e829a90</span></div><a href="/servlet/linkout?suffix=cit169&amp;dbid=16384&amp;doi=10.1126%2Fscience.aap9112&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.aap9112%26sid%3Dliteratum%253Aachs%26aulast%3DPerera%26aufirst%3DD.%26aulast%3DTucker%26aufirst%3DJ.%2BW.%26aulast%3DBrahmbhatt%26aufirst%3DS.%26aulast%3DHelal%26aufirst%3DC.%2BJ.%26aulast%3DChong%26aufirst%3DA.%26aulast%3DFarrell%26aufirst%3DW.%26aulast%3DRichardson%26aufirst%3DP.%26aulast%3DSach%26aufirst%3DN.%2BW.%26atitle%3DA%2520Platform%2520for%2520Automated%2520Nanomole-Scale%2520Reaction%2520Screening%2520and%2520Micromole-Scale%2520Synthesis%2520in%2520Flow%26jtitle%3DScience%26date%3D2018%26volume%3D359%26spage%3D429%26epage%3D434%26doi%3D10.1126%2Fscience.aap9112" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref170"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref170'); return false;" data-citation="" class="refNumLink">170</a></strong><div class="NLM_citation" id="cit170"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schneider, G.</span></span> <span> </span><span class="NLM_article-title">Automating Drug Discovery</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">97</span>– <span class="NLM_lpage">113</span>, <span class="refDoi"> DOI: 10.1038/nrd.2017.232</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref170/cit170&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1038%2Fnrd.2017.232" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref170/cit170&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=29242609" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref170/cit170&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvFOntbbK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2018&pages=97-113&author=G.+Schneider&title=Automating+Drug+Discovery&doi=10.1038%2Fnrd.2017.232"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit170R"><div class="casContent"><span class="casTitleNuber">170</span><div class="casTitle"><span class="NLM_cas:atitle">Automating drug discovery</span></div><div class="casAuthors">Schneider, Gisbert</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">97-113</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">A review.  Small mol. drug discovery can be viewed as a challenging multidimensional problem in which various characteristics of compds. including efficacy, pharmacokinetics and safety need to be optimized in parallel to provide drug candidates.  Recent advances in areas such as microfluidics-assisted chem. synthesis and biol. testing, as well as artificial intelligence systems that improve a design hypothesis through feedback anal., are now providing a basis for the introduction of greater automation into aspects of this process.  This could potentially accelerate time frames for compd. discovery and optimization and enable more effective searches of chem. space.  However, such approaches also raise considerable conceptual, tech. and organizational challenges, as well as scepticism about the current hype around them.  This article aims to identify the approaches and technologies that could be implemented robustly by medicinal chemists in the near future and to critically analyze the opportunities and challenges for their more widespread application.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpWyg9gWifCLrVg90H21EOLACvtfcHk0lg3Mk1-JQg3eg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvFOntbbK&md5=4fc1e441a0a31d7421642eb91fbac8c7</span></div><a href="/servlet/linkout?suffix=cit170&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2017.232&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2017.232%26sid%3Dliteratum%253Aachs%26aulast%3DSchneider%26aufirst%3DG.%26atitle%3DAutomating%2520Drug%2520Discovery%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2018%26volume%3D17%26spage%3D97%26epage%3D113%26doi%3D10.1038%2Fnrd.2017.232" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref171"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref171'); return false;" data-citation="" class="refNumLink">171</a></strong><div class="NLM_citation" id="cit171"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Magnani, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Serrano-Vega, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shibata, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abdul-Hussein, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lebon, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller-Gallacher, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singhal, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Strege, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tate, C. G.</span></span> <span> </span><span class="NLM_article-title">A Mutagenesis and Screening Strategy to Generate Optimally Thermostabilized Membrane Proteins for Structural Studies</span>. <i>Nat. Protoc.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">1554</span>– <span class="NLM_lpage">1571</span>, <span class="refDoi"> DOI: 10.1038/nprot.2016.088</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref171/cit171&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1038%2Fnprot.2016.088" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref171/cit171&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=27466713" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref171/cit171&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A280%3ADC%252BC2s3ls1Clug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2016&pages=1554-1571&author=F.+Magnaniauthor=M.+J.+Serrano-Vegaauthor=Y.+Shibataauthor=S.+Abdul-Husseinauthor=G.+Lebonauthor=J.+Miller-Gallacherauthor=A.+Singhalauthor=A.+Stregeauthor=J.+A.+Thomasauthor=C.+G.+Tate&title=A+Mutagenesis+and+Screening+Strategy+to+Generate+Optimally+Thermostabilized+Membrane+Proteins+for+Structural+Studies&doi=10.1038%2Fnprot.2016.088"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit171R"><div class="casContent"><span class="casTitleNuber">171</span><div class="casTitle"><span class="NLM_cas:atitle">A mutagenesis and screening strategy to generate optimally thermostabilized membrane proteins for structural studies</span></div><div class="casAuthors">Magnani Francesca; Serrano-Vega Maria J; Shibata Yoko; Abdul-Hussein Saba; Lebon Guillaume; Miller-Gallacher Jennifer; Singhal Ankita; Strege Annette; Thomas Jennifer A; Tate Christopher G</div><div class="citationInfo"><span class="NLM_cas:title">Nature protocols</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1554-71</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The thermostability of an integral membrane protein (MP) in detergent solution is a key parameter that dictates the likelihood of obtaining well-diffracting crystals that are suitable for structure determination.  However, many mammalian MPs are too unstable for crystallization.  We developed a thermostabilization strategy based on systematic mutagenesis coupled to a radioligand-binding thermostability assay that can be applied to receptors, ion channels and transporters.  It takes ∼6-12 months to thermostabilize a G-protein-coupled receptor (GPCR) containing 300 amino acid (aa) residues.  The resulting thermostabilized MPs are more easily crystallized and result in high-quality structures.  This methodology has facilitated structure-based drug design applied to GPCRs because it is possible to determine multiple structures of the thermostabilized receptors bound to low-affinity ligands.  Protocols and advice are given on how to develop thermostability assays for MPs and how to combine mutations to make an optimally stable mutant suitable for structural studies.  The steps in the procedure include the generation of ∼300 site-directed mutants by Ala/Leu scanning mutagenesis, the expression of each mutant in mammalian cells by transient transfection and the identification of thermostable mutants using a thermostability assay that is based on binding of an (125)I-labeled radioligand to the unpurified, detergent-solubilized MP.  Individual thermostabilizing point mutations are then combined to make an optimally stable MP that is suitable for structural biology and other biophysical studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRD72cWR2CMnxjv2lLSLOxFfW6udTcc2eZ82xpCEl72Orntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2s3ls1Clug%253D%253D&md5=23318e6bde65a88ce20cb3536ea5f97b</span></div><a href="/servlet/linkout?suffix=cit171&amp;dbid=16384&amp;doi=10.1038%2Fnprot.2016.088&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnprot.2016.088%26sid%3Dliteratum%253Aachs%26aulast%3DMagnani%26aufirst%3DF.%26aulast%3DSerrano-Vega%26aufirst%3DM.%2BJ.%26aulast%3DShibata%26aufirst%3DY.%26aulast%3DAbdul-Hussein%26aufirst%3DS.%26aulast%3DLebon%26aufirst%3DG.%26aulast%3DMiller-Gallacher%26aufirst%3DJ.%26aulast%3DSinghal%26aufirst%3DA.%26aulast%3DStrege%26aufirst%3DA.%26aulast%3DThomas%26aufirst%3DJ.%2BA.%26aulast%3DTate%26aufirst%3DC.%2BG.%26atitle%3DA%2520Mutagenesis%2520and%2520Screening%2520Strategy%2520to%2520Generate%2520Optimally%2520Thermostabilized%2520Membrane%2520Proteins%2520for%2520Structural%2520Studies%26jtitle%3DNat.%2520Protoc.%26date%3D2016%26volume%3D11%26spage%3D1554%26epage%3D1571%26doi%3D10.1038%2Fnprot.2016.088" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref172"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref172'); return false;" data-citation="" class="refNumLink">172</a></strong><div class="NLM_citation" id="cit172"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jazayeri, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rappas, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, A. J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kean, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Errey, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robertson, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fiez-Vandal, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andrews, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Congreve, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bortolato, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mason, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baig, A. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Teobald, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doré, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weir, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cooke, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marshall, F. H.</span></span> <span> </span><span class="NLM_article-title">Crystal Structure of the GLP-1 Receptor Bound to a Peptide Agonist</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>546</i></span>,  <span class="NLM_fpage">254</span>– <span class="NLM_lpage">258</span>, <span class="refDoi"> DOI: 10.1038/nature22800</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref172/cit172&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1038%2Fnature22800" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref172/cit172&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=28562585" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref172/cit172&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC2sXovFCrs70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=546&publication_year=2017&pages=254-258&author=A.+Jazayeriauthor=M.+Rappasauthor=A.+J.+H.+Brownauthor=J.+Keanauthor=J.+C.+Erreyauthor=N.+J.+Robertsonauthor=C.+Fiez-Vandalauthor=S.+P.+Andrewsauthor=M.+Congreveauthor=A.+Bortolatoauthor=J.+S.+Masonauthor=A.+H.+Baigauthor=I.+Teobaldauthor=A.+S.+Dor%C3%A9author=M.+Weirauthor=R.+M.+Cookeauthor=F.+H.+Marshall&title=Crystal+Structure+of+the+GLP-1+Receptor+Bound+to+a+Peptide+Agonist&doi=10.1038%2Fnature22800"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit172R"><div class="casContent"><span class="casTitleNuber">172</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal structure of the GLP-1 receptor bound to a peptide agonist</span></div><div class="casAuthors">Jazayeri, Ali; Rappas, Mathieu; Brown, Alastair J. H.; Kean, James; Errey, James C.; Robertson, Nathan J.; Fiez-Vandal, Cedric; Andrews, Stephen P.; Congreve, Miles; Bortolato, Andrea; Mason, Jonathan S.; Baig, Asma H.; Teobald, Iryna; Dore, Andrew S.; Weir, Malcolm; Cooke, Robert M.; Marshall, Fiona H.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">546</span>
        (<span class="NLM_cas:issue">7657</span>),
    <span class="NLM_cas:pages">254-258</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Glucagon-like peptide 1 (GLP-1) regulates glucose homeostasis through the control of insulin release from the pancreas.  GLP-1 peptide agonists are efficacious drugs for the treatment of diabetes.  To gain insight into the mol. mechanism of action of GLP-1 peptides, here we report the crystal structure of the full-length GLP-1 receptor bound to a truncated peptide agonist.  The peptide agonist retains an α-helical conformation as it sits deep within the receptor-binding pocket.  The arrangement of the transmembrane helixes reveals hallmarks of an active conformation similar to that obsd. in class A receptors.  Guided by this structural information, we design peptide agonists with potent in vivo activity in a mouse model of diabetes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoX6Ibc110QxrVg90H21EOLACvtfcHk0ljNZS7_NltyHg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXovFCrs70%253D&md5=0a5c99161af8bae506a184e0ab6306c1</span></div><a href="/servlet/linkout?suffix=cit172&amp;dbid=16384&amp;doi=10.1038%2Fnature22800&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature22800%26sid%3Dliteratum%253Aachs%26aulast%3DJazayeri%26aufirst%3DA.%26aulast%3DRappas%26aufirst%3DM.%26aulast%3DBrown%26aufirst%3DA.%2BJ.%2BH.%26aulast%3DKean%26aufirst%3DJ.%26aulast%3DErrey%26aufirst%3DJ.%2BC.%26aulast%3DRobertson%26aufirst%3DN.%2BJ.%26aulast%3DFiez-Vandal%26aufirst%3DC.%26aulast%3DAndrews%26aufirst%3DS.%2BP.%26aulast%3DCongreve%26aufirst%3DM.%26aulast%3DBortolato%26aufirst%3DA.%26aulast%3DMason%26aufirst%3DJ.%2BS.%26aulast%3DBaig%26aufirst%3DA.%2BH.%26aulast%3DTeobald%26aufirst%3DI.%26aulast%3DDor%25C3%25A9%26aufirst%3DA.%2BS.%26aulast%3DWeir%26aufirst%3DM.%26aulast%3DCooke%26aufirst%3DR.%2BM.%26aulast%3DMarshall%26aufirst%3DF.%2BH.%26atitle%3DCrystal%2520Structure%2520of%2520the%2520GLP-1%2520Receptor%2520Bound%2520to%2520a%2520Peptide%2520Agonist%26jtitle%3DNature%26date%3D2017%26volume%3D546%26spage%3D254%26epage%3D258%26doi%3D10.1038%2Fnature22800" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref173"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref173'); return false;" data-citation="" class="refNumLink">173</a></strong><div class="NLM_citation" id="cit173"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jazayeri, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doré, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lamb, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krishnamurthy, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Southall, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baig, A. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bortolato, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koglin, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robertson, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Errey, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andrews, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Teobald, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, A. J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cooke, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weir, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marshall, F. H.</span></span> <span> </span><span class="NLM_article-title">Extra-Helical Binding Site of a Glucagon Receptor Antagonist</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>533</i></span>,  <span class="NLM_fpage">274</span>– <span class="NLM_lpage">277</span>, <span class="refDoi"> DOI: 10.1038/nature17414</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref173/cit173&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1038%2Fnature17414" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref173/cit173&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=27111510" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref173/cit173&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC28XmvVGnt7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=533&publication_year=2016&pages=274-277&author=A.+Jazayeriauthor=A.+S.+Dor%C3%A9author=D.+Lambauthor=H.+Krishnamurthyauthor=S.+M.+Southallauthor=A.+H.+Baigauthor=A.+Bortolatoauthor=M.+Koglinauthor=N.+J.+Robertsonauthor=J.+C.+Erreyauthor=S.+P.+Andrewsauthor=I.+Teobaldauthor=A.+J.+H.+Brownauthor=R.+M.+Cookeauthor=M.+Weirauthor=F.+H.+Marshall&title=Extra-Helical+Binding+Site+of+a+Glucagon+Receptor+Antagonist&doi=10.1038%2Fnature17414"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit173R"><div class="casContent"><span class="casTitleNuber">173</span><div class="casTitle"><span class="NLM_cas:atitle">Extra-helical binding site of a glucagon receptor antagonist</span></div><div class="casAuthors">Jazayeri, Ali; Dore, Andrew S.; Lamb, Daniel; Krishnamurthy, Harini; Southall, Stacey M.; Baig, Asma H.; Bortolato, Andrea; Koglin, Markus; Robertson, Nathan J.; Errey, James C.; Andrews, Stephen P.; Teobald, Iryna; Brown, Alastair J. H.; Cooke, Robert M.; Weir, Malcolm; Marshall, Fiona H.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">533</span>
        (<span class="NLM_cas:issue">7602</span>),
    <span class="NLM_cas:pages">274-277</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Glucagon is a 29-amino-acid peptide released from the α-cells of the islet of Langerhans, which has a key role in glucose homeostasis.  Glucagon action is transduced by the class B G-protein-coupled glucagon receptor (GCGR), which is located on liver, kidney, intestinal smooth muscle, brain, adipose tissue, heart and pancreas cells, and this receptor has been considered an important drug target in the treatment of diabetes.  Administration of recently identified small-mol. GCGR antagonists in patients with type 2 diabetes results in a substantial redn. of fasting and postprandial glucose concns.  Although an x-ray structure of the transmembrane domain of the GCGR has previously been solved, the ligand (NNC0640) was not resolved.  Here the authors report the 2.5 Å structure of human GCGR in complex with the antagonist MK-0893 (ref. 4), which is found to bind to an allosteric site outside the seven transmembrane (7TM) helical bundle in a position between TM6 and TM7 extending into the lipid bilayer.  Mutagenesis of key residues identified in the x-ray structure confirms their role in the binding of MK-0893 to the receptor.  The unexpected position of the binding site for MK-0893, which is structurally similar to other GCGR antagonists, suggests that glucagon activation of the receptor is prevented by restriction of the outward helical movement of TM6 required for G-protein coupling.  Structural knowledge of class B receptors is limited, with only one other ligand-binding site defined-for the corticotropin-releasing hormone receptor 1 (CRF1R)-which was located deep within the 7TM bundle.  The authors describe a completely novel allosteric binding site for class B receptors, providing an opportunity for structure-based drug design for this receptor class and furthering the authors' understanding of the mechanisms of activation of these receptors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoj307lDqVDzrVg90H21EOLACvtfcHk0ljNZS7_NltyHg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XmvVGnt7w%253D&md5=5b45af53e0870f497967c113095da9b4</span></div><a href="/servlet/linkout?suffix=cit173&amp;dbid=16384&amp;doi=10.1038%2Fnature17414&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature17414%26sid%3Dliteratum%253Aachs%26aulast%3DJazayeri%26aufirst%3DA.%26aulast%3DDor%25C3%25A9%26aufirst%3DA.%2BS.%26aulast%3DLamb%26aufirst%3DD.%26aulast%3DKrishnamurthy%26aufirst%3DH.%26aulast%3DSouthall%26aufirst%3DS.%2BM.%26aulast%3DBaig%26aufirst%3DA.%2BH.%26aulast%3DBortolato%26aufirst%3DA.%26aulast%3DKoglin%26aufirst%3DM.%26aulast%3DRobertson%26aufirst%3DN.%2BJ.%26aulast%3DErrey%26aufirst%3DJ.%2BC.%26aulast%3DAndrews%26aufirst%3DS.%2BP.%26aulast%3DTeobald%26aufirst%3DI.%26aulast%3DBrown%26aufirst%3DA.%2BJ.%2BH.%26aulast%3DCooke%26aufirst%3DR.%2BM.%26aulast%3DWeir%26aufirst%3DM.%26aulast%3DMarshall%26aufirst%3DF.%2BH.%26atitle%3DExtra-Helical%2520Binding%2520Site%2520of%2520a%2520Glucagon%2520Receptor%2520Antagonist%26jtitle%3DNature%26date%3D2016%26volume%3D533%26spage%3D274%26epage%3D277%26doi%3D10.1038%2Fnature17414" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref174"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref174'); return false;" data-citation="" class="refNumLink">174</a></strong><div class="NLM_citation" id="cit174"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Oswald, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rappas, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kean, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doré, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Errey, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bennett, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deflorian, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Christopher, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jazayeri, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mason, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Congreve, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cooke, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marshall, F. H.</span></span> <span> </span><span class="NLM_article-title">Intracellular Allosteric Antagonism of the CCR9 Receptor</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>540</i></span>,  <span class="NLM_fpage">462</span>– <span class="NLM_lpage">465</span>, <span class="refDoi"> DOI: 10.1038/nature20606</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref174/cit174&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1038%2Fnature20606" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref174/cit174&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=27926729" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref174/cit174&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC28XitVWmurbM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=540&publication_year=2016&pages=462-465&author=C.+Oswaldauthor=M.+Rappasauthor=J.+Keanauthor=A.+S.+Dor%C3%A9author=J.+C.+Erreyauthor=K.+Bennettauthor=F.+Deflorianauthor=J.+A.+Christopherauthor=A.+Jazayeriauthor=J.+S.+Masonauthor=M.+Congreveauthor=R.+M.+Cookeauthor=F.+H.+Marshall&title=Intracellular+Allosteric+Antagonism+of+the+CCR9+Receptor&doi=10.1038%2Fnature20606"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit174R"><div class="casContent"><span class="casTitleNuber">174</span><div class="casTitle"><span class="NLM_cas:atitle">Intracellular allosteric antagonism of the CCR9 receptor</span></div><div class="casAuthors">Oswald, Christine; Rappas, Mathieu; Kean, James; Dore, Andrew S.; Errey, James C.; Bennett, Kirstie; Deflorian, Francesca; Christopher, John A.; Jazayeri, Ali; Mason, Jonathan S.; Congreve, Miles; Cooke, Robert M.; Marshall, Fiona H.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">540</span>
        (<span class="NLM_cas:issue">7633</span>),
    <span class="NLM_cas:pages">462-465</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Chemokines and their G-protein-coupled receptors play a diverse role in immune defense by controlling the migration, activation and survival of immune cells.  They are also involved in viral entry, tumor growth and metastasis and hence are important drug targets in a wide range of diseases.  Despite very significant efforts by the pharmaceutical industry to develop drugs, with over 50 small-mol. drugs directed at the family entering clin. development, only two compds. have reached the market: maraviroc (CCR5) for HIV infection and plerixafor (CXCR4) for stem-cell mobilization.  The high failure rate may in part be due to limited understanding of the mechanism of action of chemokine antagonists and an inability to optimize compds. in the absence of structural information.  CC chemokine receptor type 9 (CCR9) activation by CCL25 plays a key role in leukocyte recruitment to the gut and represents a therapeutic target in inflammatory bowel disease.  The selective CCR9 antagonist vercirnon progressed to phase 3 clin. trials in Crohn's disease but efficacy was limited, with the need for very high doses to block receptor activation.  Here we report the crystal structure of the CCR9 receptor in complex with vercirnon at 2.8 Å resoln.  Remarkably, vercirnon binds to the intracellular side of the receptor, exerting allosteric antagonism and preventing G-protein coupling.  This binding site explains the need for relatively lipophilic ligands and describes another example of an allosteric site on G-protein-coupled receptors that can be targeted for drug design, not only at CCR9, but potentially extending to other chemokine receptors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpHomIAvUxar7Vg90H21EOLACvtfcHk0ljPzkXV27pAmw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitVWmurbM&md5=253b630bd2b9b65fc192ff86de9df61a</span></div><a href="/servlet/linkout?suffix=cit174&amp;dbid=16384&amp;doi=10.1038%2Fnature20606&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature20606%26sid%3Dliteratum%253Aachs%26aulast%3DOswald%26aufirst%3DC.%26aulast%3DRappas%26aufirst%3DM.%26aulast%3DKean%26aufirst%3DJ.%26aulast%3DDor%25C3%25A9%26aufirst%3DA.%2BS.%26aulast%3DErrey%26aufirst%3DJ.%2BC.%26aulast%3DBennett%26aufirst%3DK.%26aulast%3DDeflorian%26aufirst%3DF.%26aulast%3DChristopher%26aufirst%3DJ.%2BA.%26aulast%3DJazayeri%26aufirst%3DA.%26aulast%3DMason%26aufirst%3DJ.%2BS.%26aulast%3DCongreve%26aufirst%3DM.%26aulast%3DCooke%26aufirst%3DR.%2BM.%26aulast%3DMarshall%26aufirst%3DF.%2BH.%26atitle%3DIntracellular%2520Allosteric%2520Antagonism%2520of%2520the%2520CCR9%2520Receptor%26jtitle%3DNature%26date%3D2016%26volume%3D540%26spage%3D462%26epage%3D465%26doi%3D10.1038%2Fnature20606" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref175"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref175'); return false;" data-citation="" class="refNumLink">175</a></strong><div class="NLM_citation" id="cit175"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Harpsøe, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Isberg, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tehan, B. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weiss, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arsova, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marshall, F. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bräuner-Osborne, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gloriam, D. E.</span></span> <span> </span><span class="NLM_article-title">Selective Negative Allosteric Modulation Of Metabotropic Glutamate Receptors – A Structural Perspective of Ligands and Mutants</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">13869</span>, <span class="refDoi"> DOI: 10.1038/srep13869</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref175/cit175&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1038%2Fsrep13869" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref175/cit175&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=26359761" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref175/cit175&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A280%3ADC%252BC283itVSqsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2015&pages=13869&author=K.+Harps%C3%B8eauthor=V.+Isbergauthor=B.+G.+Tehanauthor=D.+Weissauthor=A.+Arsovaauthor=F.+H.+Marshallauthor=H.+Br%C3%A4uner-Osborneauthor=D.+E.+Gloriam&title=Selective+Negative+Allosteric+Modulation+Of+Metabotropic+Glutamate+Receptors+%E2%80%93+A+Structural+Perspective+of+Ligands+and+Mutants&doi=10.1038%2Fsrep13869"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit175R"><div class="casContent"><span class="casTitleNuber">175</span><div class="casTitle"><span class="NLM_cas:atitle">Selective Negative Allosteric Modulation Of Metabotropic Glutamate Receptors - A Structural Perspective of Ligands and Mutants</span></div><div class="casAuthors">Harpsoe Kasper; Isberg Vignir; Arsova Angela; Brauner-Osborne Hans; Gloriam David E; Tehan Benjamin G; Weiss Dahlia; Marshall Fiona H</div><div class="citationInfo"><span class="NLM_cas:title">Scientific reports</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">13869</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The metabotropic glutamate receptors have a wide range of modulatory functions in the central nervous system.  They are among the most highly pursued drug targets, with relevance for several neurological diseases, and a number of allosteric modulators have entered clinical trials.  However, so far this has not led to a marketed drug, largely because of the difficulties in achieving subtype-selective compounds with desired properties.  Very recently the first crystal structures were published for the transmembrane domain of two metabotropic glutamate receptors in complex with negative allosteric modulators.  In this analysis, we make the first comprehensive structural comparison of all metabotropic glutamate receptors, placing selective negative allosteric modulators and critical mutants into the detailed context of the receptor binding sites.  A better understanding of how the different mGlu allosteric modulator binding modes relates to selective pharmacological actions will be very valuable for rational design of safer drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRY5UWfgHZTboYPsCkJIw00fW6udTcc2eZQIZX1mgKc3Lntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC283itVSqsg%253D%253D&md5=ddff5947451bbc826710b780f7e2df70</span></div><a href="/servlet/linkout?suffix=cit175&amp;dbid=16384&amp;doi=10.1038%2Fsrep13869&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsrep13869%26sid%3Dliteratum%253Aachs%26aulast%3DHarps%25C3%25B8e%26aufirst%3DK.%26aulast%3DIsberg%26aufirst%3DV.%26aulast%3DTehan%26aufirst%3DB.%2BG.%26aulast%3DWeiss%26aufirst%3DD.%26aulast%3DArsova%26aufirst%3DA.%26aulast%3DMarshall%26aufirst%3DF.%2BH.%26aulast%3DBr%25C3%25A4uner-Osborne%26aufirst%3DH.%26aulast%3DGloriam%26aufirst%3DD.%2BE.%26atitle%3DSelective%2520Negative%2520Allosteric%2520Modulation%2520Of%2520Metabotropic%2520Glutamate%2520Receptors%2520%25E2%2580%2593%2520A%2520Structural%2520Perspective%2520of%2520Ligands%2520and%2520Mutants%26jtitle%3DSci.%2520Rep.%26date%3D2015%26volume%3D5%26spage%3D13869%26doi%3D10.1038%2Fsrep13869" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref176"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref176'); return false;" data-citation="" class="refNumLink">176</a></strong><div class="NLM_citation" id="cit176"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kean, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bortolato, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hollenstein, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marshall, F. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jazayeri, A.</span></span> <span> </span><span class="NLM_article-title">Conformational Thermostabilisation of Corticotropin Releasing Factor Receptor 1</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">11954</span>, <span class="refDoi"> DOI: 10.1038/srep11954</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref176/cit176&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1038%2Fsrep11954" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref176/cit176&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=26159865" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref176/cit176&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A280%3ADC%252BC28%252FhvVGnuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2015&pages=11954&author=J.+Keanauthor=A.+Bortolatoauthor=K.+Hollensteinauthor=F.+H.+Marshallauthor=A.+Jazayeri&title=Conformational+Thermostabilisation+of+Corticotropin+Releasing+Factor+Receptor+1&doi=10.1038%2Fsrep11954"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit176R"><div class="casContent"><span class="casTitleNuber">176</span><div class="casTitle"><span class="NLM_cas:atitle">Conformational thermostabilisation of corticotropin releasing factor receptor 1</span></div><div class="casAuthors">Kean James; Bortolato Andrea; Hollenstein Kaspar; Marshall Fiona H; Jazayeri Ali</div><div class="citationInfo"><span class="NLM_cas:title">Scientific reports</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">11954</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Recent technical advances have greatly facilitated G-protein coupled receptors crystallography as evidenced by the number of successful x-ray structures that have been reported recently.  These technical advances include novel detergents, specialised crystallography techniques as well as protein engineering solutions such as fusions and conformational thermostabilisation.  Using conformational thermostabilisation, it is possible to generate variants of GPCRs that exhibit significantly increased stability in detergent micelles whilst preferentially occupying a single conformation.  In this paper we describe for the first time the application of this technique to a member of a class B GPCR, the corticotropin releasing factor receptor 1 (CRF1R).  Mutational screening in the presence of the inverse agonist, CP-376395, resulted in the identification of a construct with twelve point mutations that exhibited significantly increased thermal stability in a range of detergents.  We further describe the subsequent construct engineering steps that eventually yielded a crystallisation-ready construct which recently led to the solution of the first x-ray structure of a class B receptor.  Finally, we have used molecular dynamic simulation to provide structural insight into CRF1R instability as well as the stabilising effects of the mutants, which may be extended to other class B receptors considering the high degree of structural conservation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSjZbDL9WcLFO2qxNLjzXe8fW6udTcc2eZQIZX1mgKc3Lntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC28%252FhvVGnuw%253D%253D&md5=dd1bc3726119ed8d19b7ecfe4ee227b2</span></div><a href="/servlet/linkout?suffix=cit176&amp;dbid=16384&amp;doi=10.1038%2Fsrep11954&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsrep11954%26sid%3Dliteratum%253Aachs%26aulast%3DKean%26aufirst%3DJ.%26aulast%3DBortolato%26aufirst%3DA.%26aulast%3DHollenstein%26aufirst%3DK.%26aulast%3DMarshall%26aufirst%3DF.%2BH.%26aulast%3DJazayeri%26aufirst%3DA.%26atitle%3DConformational%2520Thermostabilisation%2520of%2520Corticotropin%2520Releasing%2520Factor%2520Receptor%25201%26jtitle%3DSci.%2520Rep.%26date%3D2015%26volume%3D5%26spage%3D11954%26doi%3D10.1038%2Fsrep11954" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref177"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref177'); return false;" data-citation="" class="refNumLink">177</a></strong><div class="NLM_citation" id="cit177"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, R. K. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fiez-Vandal, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schlenker, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edman, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aggeler, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, D. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cooke, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dumelin, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doré, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geschwindner, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grebner, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hermansson, N.-O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jazayeri, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johansson, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leong, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prihandoko, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rappas, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soutter, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Snijder, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sundström, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tehan, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thornton, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Troast, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wiggin, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhukov, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marshall, F. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dekker, N.</span></span> <span> </span><span class="NLM_article-title">Structural Insight into Allosteric Modulation of Protease-Activated Receptor 2</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>545</i></span>,  <span class="NLM_fpage">112</span>– <span class="NLM_lpage">115</span>, <span class="refDoi"> DOI: 10.1038/nature22309</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref177/cit177&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1038%2Fnature22309" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref177/cit177&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=28445455" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref177/cit177&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC2sXmslehsrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=545&publication_year=2017&pages=112-115&author=R.+K.+Y.+Chengauthor=C.+Fiez-Vandalauthor=O.+Schlenkerauthor=K.+Edmanauthor=B.+Aggelerauthor=D.+G.+Brownauthor=G.+A.+Brownauthor=R.+M.+Cookeauthor=C.+E.+Dumelinauthor=A.+S.+Dor%C3%A9author=S.+Geschwindnerauthor=C.+Grebnerauthor=N.-O.+Hermanssonauthor=A.+Jazayeriauthor=P.+Johanssonauthor=L.+Leongauthor=R.+Prihandokoauthor=M.+Rappasauthor=H.+Soutterauthor=A.+Snijderauthor=L.+Sundstr%C3%B6mauthor=B.+Tehanauthor=P.+Thorntonauthor=D.+Troastauthor=G.+Wigginauthor=A.+Zhukovauthor=F.+H.+Marshallauthor=N.+Dekker&title=Structural+Insight+into+Allosteric+Modulation+of+Protease-Activated+Receptor+2&doi=10.1038%2Fnature22309"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit177R"><div class="casContent"><span class="casTitleNuber">177</span><div class="casTitle"><span class="NLM_cas:atitle">Structural insight into allosteric modulation of protease-activated receptor 2</span></div><div class="casAuthors">Cheng, Robert K. Y.; Fiez-Vandal, Cedric; Schlenker, Oliver; Edman, Karl; Aggeler, Birte; Brown, Dean G.; Brown, Giles A.; Cooke, Robert M.; Dumelin, Christoph E.; Dore, Andrew S.; Geschwindner, Stefan; Grebner, Christoph; Hermansson, Nils-Olov; Jazayeri, Ali; Johansson, Patrik; Leong, Louis; Prihandoko, Rudi; Rappas, Mathieu; Soutter, Holly; Snijder, Arjan; Sundstrom, Linda; Tehan, Benjamin; Thornton, Peter; Troast, Dawn; Wiggin, Giselle; Zhukov, Andrei; Marshall, Fiona H.; Dekker, Niek</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">545</span>
        (<span class="NLM_cas:issue">7652</span>),
    <span class="NLM_cas:pages">112-115</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Protease-activated receptors (PARs) are a family of G-protein-coupled receptors (GPCRs) that are irreversibly activated by proteolytic cleavage of the N-terminus, which unmasks a tethered peptide ligand that binds and activates the transmembrane receptor domain, eliciting a cellular cascade in response to inflammatory signals and other stimuli.  PARs are implicated in a wide range of diseases, such as cancer and inflammation.  PARs have been the subject of major pharmaceutical research efforts, but the discovery of small-mol. antagonists that effectively bind them has proved challenging.  The only marketed drug targeting a PAR is vorapaxar, a selective antagonist of PAR1 used to prevent thrombosis.  The structure of PAR1 in complex with vorapaxar has been reported previously.  Despite sequence homol. across the PAR isoforms, discovery of PAR2 antagonists has been less successful, although GB88 has been described as a weak antagonist.  Here, we report crystal structures of PAR2 in complex with two distinct antagonists and a blocking antibody.  The antagonist AZ8838 binds in a fully occluded pocket near the extracellular surface.  Functional and binding studies reveal that AZ8838 exhibits slow binding kinetics, which is an attractive feature for a PAR2 antagonist competing against a tethered ligand.  Antagonist AZ3451 binds to a remote allosteric site outside the helical bundle.  We propose that antagonist binding prevents structural rearrangements required for receptor activation and signalling.  We also show that a blocking antibody antigen-binding fragment binds to the extracellular surface of PAR2, preventing access of the tethered ligand to the peptide-binding site.  These structures provide a basis for the development of selective PAR2 antagonists for a range of therapeutic uses.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoSz2vmZJ2f6LVg90H21EOLACvtfcHk0lhDQuhPxEpkmg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXmslehsrs%253D&md5=6a0737757c891336437d4419c3fd6c45</span></div><a href="/servlet/linkout?suffix=cit177&amp;dbid=16384&amp;doi=10.1038%2Fnature22309&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature22309%26sid%3Dliteratum%253Aachs%26aulast%3DCheng%26aufirst%3DR.%2BK.%2BY.%26aulast%3DFiez-Vandal%26aufirst%3DC.%26aulast%3DSchlenker%26aufirst%3DO.%26aulast%3DEdman%26aufirst%3DK.%26aulast%3DAggeler%26aufirst%3DB.%26aulast%3DBrown%26aufirst%3DD.%2BG.%26aulast%3DBrown%26aufirst%3DG.%2BA.%26aulast%3DCooke%26aufirst%3DR.%2BM.%26aulast%3DDumelin%26aufirst%3DC.%2BE.%26aulast%3DDor%25C3%25A9%26aufirst%3DA.%2BS.%26aulast%3DGeschwindner%26aufirst%3DS.%26aulast%3DGrebner%26aufirst%3DC.%26aulast%3DHermansson%26aufirst%3DN.-O.%26aulast%3DJazayeri%26aufirst%3DA.%26aulast%3DJohansson%26aufirst%3DP.%26aulast%3DLeong%26aufirst%3DL.%26aulast%3DPrihandoko%26aufirst%3DR.%26aulast%3DRappas%26aufirst%3DM.%26aulast%3DSoutter%26aufirst%3DH.%26aulast%3DSnijder%26aufirst%3DA.%26aulast%3DSundstr%25C3%25B6m%26aufirst%3DL.%26aulast%3DTehan%26aufirst%3DB.%26aulast%3DThornton%26aufirst%3DP.%26aulast%3DTroast%26aufirst%3DD.%26aulast%3DWiggin%26aufirst%3DG.%26aulast%3DZhukov%26aufirst%3DA.%26aulast%3DMarshall%26aufirst%3DF.%2BH.%26aulast%3DDekker%26aufirst%3DN.%26atitle%3DStructural%2520Insight%2520into%2520Allosteric%2520Modulation%2520of%2520Protease-Activated%2520Receptor%25202%26jtitle%3DNature%26date%3D2017%26volume%3D545%26spage%3D112%26epage%3D115%26doi%3D10.1038%2Fnature22309" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref178"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref178'); return false;" data-citation="" class="refNumLink">178</a></strong><div class="NLM_citation" id="cit178"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rasmussen, S. G. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choi, H.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fung, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pardon, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Casarosa, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chae, P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DeVree, B. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosenbaum, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thian, F. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kobilka, T. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schnapp, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Konetzki, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sunahara, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gellman, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pautsch, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steyaert, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weis, W. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kobilka, B. K.</span></span> <span> </span><span class="NLM_article-title">Structure of a Nanobody-Stabilized Active State of the β<sub>2</sub> Adrenoceptor</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>469</i></span>,  <span class="NLM_fpage">175</span>– <span class="NLM_lpage">180</span>, <span class="refDoi"> DOI: 10.1038/nature09648</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref178/cit178&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1038%2Fnature09648" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref178/cit178&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=21228869" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref178/cit178&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC3MXkvFartA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=469&publication_year=2011&pages=175-180&author=S.+G.+F.+Rasmussenauthor=H.-J.+Choiauthor=J.+J.+Fungauthor=E.+Pardonauthor=P.+Casarosaauthor=P.+S.+Chaeauthor=B.+T.+DeVreeauthor=D.+M.+Rosenbaumauthor=F.+S.+Thianauthor=T.+S.+Kobilkaauthor=A.+Schnappauthor=I.+Konetzkiauthor=R.+K.+Sunaharaauthor=S.+H.+Gellmanauthor=A.+Pautschauthor=J.+Steyaertauthor=W.+I.+Weisauthor=B.+K.+Kobilka&title=Structure+of+a+Nanobody-Stabilized+Active+State+of+the+%CE%B22+Adrenoceptor&doi=10.1038%2Fnature09648"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit178R"><div class="casContent"><span class="casTitleNuber">178</span><div class="casTitle"><span class="NLM_cas:atitle">Structure of a nanobody-stabilized active state of the β2 adrenoceptor</span></div><div class="casAuthors">Rasmussen, Soren G. F.; Choi, Hee-Jung; Fung, Juan Jose; Pardon, Els; Casarosa, Paola; Chae, Pil Seok; DeVree, Brian T.; Rosenbaum, Daniel M.; Thian, Foon Sun; Kobilka, Tong Sun; Schnapp, Andreas; Konetzki, Ingo; Sunahara, Roger K.; Gellman, Samuel H.; Pautsch, Alexander; Steyaert, Jan; Weis, William I.; Kobilka, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">469</span>
        (<span class="NLM_cas:issue">7329</span>),
    <span class="NLM_cas:pages">175-180</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">G protein coupled receptors (GPCRs) exhibit a spectrum of functional behaviors in response to natural and synthetic ligands.  Recent crystal structures provide insights into inactive states of several GPCRs.  Efforts to obtain an agonist-bound active-state GPCR structure have proven difficult due to the inherent instability of this state in the absence of a G protein.  We generated a camelid antibody fragment (nanobody) to the human β2 adrenergic receptor (β2AR) that exhibits G protein-like behavior, and obtained an agonist-bound, active-state crystal structure of the receptor-nanobody complex.  Comparison with the inactive β2AR structure reveals subtle changes in the binding pocket; however, these small changes are assocd. with an 11 Å outward movement of the cytoplasmic end of transmembrane segment 6, and rearrangements of transmembrane segments 5 and 7 that are remarkably similar to those obsd. in opsin, an active form of rhodopsin.  This structure provides insights into the process of agonist binding and activation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqITZxHz6Vi77Vg90H21EOLACvtfcHk0lir9GlzVENgbQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXkvFartA%253D%253D&md5=3a1d3bac6d92c9d54cf1ddd14d56ab8c</span></div><a href="/servlet/linkout?suffix=cit178&amp;dbid=16384&amp;doi=10.1038%2Fnature09648&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature09648%26sid%3Dliteratum%253Aachs%26aulast%3DRasmussen%26aufirst%3DS.%2BG.%2BF.%26aulast%3DChoi%26aufirst%3DH.-J.%26aulast%3DFung%26aufirst%3DJ.%2BJ.%26aulast%3DPardon%26aufirst%3DE.%26aulast%3DCasarosa%26aufirst%3DP.%26aulast%3DChae%26aufirst%3DP.%2BS.%26aulast%3DDeVree%26aufirst%3DB.%2BT.%26aulast%3DRosenbaum%26aufirst%3DD.%2BM.%26aulast%3DThian%26aufirst%3DF.%2BS.%26aulast%3DKobilka%26aufirst%3DT.%2BS.%26aulast%3DSchnapp%26aufirst%3DA.%26aulast%3DKonetzki%26aufirst%3DI.%26aulast%3DSunahara%26aufirst%3DR.%2BK.%26aulast%3DGellman%26aufirst%3DS.%2BH.%26aulast%3DPautsch%26aufirst%3DA.%26aulast%3DSteyaert%26aufirst%3DJ.%26aulast%3DWeis%26aufirst%3DW.%2BI.%26aulast%3DKobilka%26aufirst%3DB.%2BK.%26atitle%3DStructure%2520of%2520a%2520Nanobody-Stabilized%2520Active%2520State%2520of%2520the%2520%25CE%25B22%2520Adrenoceptor%26jtitle%3DNature%26date%3D2011%26volume%3D469%26spage%3D175%26epage%3D180%26doi%3D10.1038%2Fnature09648" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref179"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref179'); return false;" data-citation="" class="refNumLink">179</a></strong><div class="NLM_citation" id="cit179"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rasmussen, S. G. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DeVree, B. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zou, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kruse, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, K. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kobilka, T. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thian, F. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chae, P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pardon, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Calinski, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mathiesen, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shah, S. T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lyons, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caffrey, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gellman, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steyaert, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skiniotis, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weis, W. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sunahara, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kobilka, B. K.</span></span> <span> </span><span class="NLM_article-title">Crystal Structure of the β2 Adrenergic receptor–Gs Protein Complex</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>477</i></span>,  <span class="NLM_fpage">549</span>– <span class="NLM_lpage">555</span>, <span class="refDoi"> DOI: 10.1038/nature10361</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref179/cit179&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1038%2Fnature10361" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref179/cit179&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=21772288" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref179/cit179&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC3MXht1equrrL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=477&publication_year=2011&pages=549-555&author=S.+G.+F.+Rasmussenauthor=B.+T.+DeVreeauthor=Y.+Zouauthor=A.+C.+Kruseauthor=K.+Y.+Chungauthor=T.+S.+Kobilkaauthor=F.+S.+Thianauthor=P.+S.+Chaeauthor=E.+Pardonauthor=D.+Calinskiauthor=J.+M.+Mathiesenauthor=S.+T.+A.+Shahauthor=J.+A.+Lyonsauthor=M.+Caffreyauthor=S.+H.+Gellmanauthor=J.+Steyaertauthor=G.+Skiniotisauthor=W.+I.+Weisauthor=R.+K.+Sunaharaauthor=B.+K.+Kobilka&title=Crystal+Structure+of+the+%CE%B22+Adrenergic+receptor%E2%80%93Gs+Protein+Complex&doi=10.1038%2Fnature10361"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit179R"><div class="casContent"><span class="casTitleNuber">179</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal structure of the β2 adrenergic receptor-Gs protein complex</span></div><div class="casAuthors">Rasmussen, Soren G. F.; DeVree, Brian T.; Zou, Yao-Zhong; Kruse, Andrew C.; Chung, Ka-Young; Kobilka, Tong-Sun; Thian, Foon-Sun; Chae, Pil-Seok; Pardon, Els; Calinski, Diane; Mathiesen, Jesper M.; Shah, Syed T. A.; Lyons, Joseph A.; Caffrey, Martin; Gellman, Samuel H.; Steyaert, Jan; Skiniotis, Georgios; Weis, William I.; Sunahara, Roger K.; Kobilka, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">477</span>
        (<span class="NLM_cas:issue">7366</span>),
    <span class="NLM_cas:pages">549-555</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">G protein-coupled receptors (GPCRs) are responsible for the majority of cellular responses to hormones and neurotransmitters as well as the senses of sight, olfaction and taste.  The paradigm of GPCR signalling is the activation of a heterotrimeric GTP binding protein (G protein) by an agonist-occupied receptor.  The β2 adrenergic receptor (β2AR) activation of Gs, the stimulatory G protein for adenylyl cyclase, has long been a model system for GPCR signalling.  Here we present the crystal structure of the active state ternary complex composed of agonist-occupied monomeric β2AR and nucleotide-free Gs heterotrimer.  The principal interactions between the β2AR and Gs involve the amino- and carboxy-terminal α-helixes of Gs, with conformational changes propagating to the nucleotide-binding pocket.  The largest conformational changes in the β2AR include a 14 Å outward movement at the cytoplasmic end of transmembrane segment 6 (TM6) and an α-helical extension of the cytoplasmic end of TM5.  The most surprising observation is a major displacement of the α-helical domain of Gαs relative to the Ras-like GTPase domain.  This crystal structure represents the first high-resoln. view of transmembrane signalling by a GPCR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq-PdYRDvolqrVg90H21EOLACvtfcHk0lir9GlzVENgbQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXht1equrrL&md5=d22a43dd677ac255d138b1aedff357d3</span></div><a href="/servlet/linkout?suffix=cit179&amp;dbid=16384&amp;doi=10.1038%2Fnature10361&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature10361%26sid%3Dliteratum%253Aachs%26aulast%3DRasmussen%26aufirst%3DS.%2BG.%2BF.%26aulast%3DDeVree%26aufirst%3DB.%2BT.%26aulast%3DZou%26aufirst%3DY.%26aulast%3DKruse%26aufirst%3DA.%2BC.%26aulast%3DChung%26aufirst%3DK.%2BY.%26aulast%3DKobilka%26aufirst%3DT.%2BS.%26aulast%3DThian%26aufirst%3DF.%2BS.%26aulast%3DChae%26aufirst%3DP.%2BS.%26aulast%3DPardon%26aufirst%3DE.%26aulast%3DCalinski%26aufirst%3DD.%26aulast%3DMathiesen%26aufirst%3DJ.%2BM.%26aulast%3DShah%26aufirst%3DS.%2BT.%2BA.%26aulast%3DLyons%26aufirst%3DJ.%2BA.%26aulast%3DCaffrey%26aufirst%3DM.%26aulast%3DGellman%26aufirst%3DS.%2BH.%26aulast%3DSteyaert%26aufirst%3DJ.%26aulast%3DSkiniotis%26aufirst%3DG.%26aulast%3DWeis%26aufirst%3DW.%2BI.%26aulast%3DSunahara%26aufirst%3DR.%2BK.%26aulast%3DKobilka%26aufirst%3DB.%2BK.%26atitle%3DCrystal%2520Structure%2520of%2520the%2520%25CE%25B22%2520Adrenergic%2520receptor%25E2%2580%2593Gs%2520Protein%2520Complex%26jtitle%3DNature%26date%3D2011%26volume%3D477%26spage%3D549%26epage%3D555%26doi%3D10.1038%2Fnature10361" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref180"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref180'); return false;" data-citation="" class="refNumLink">180</a></strong><div class="NLM_citation" id="cit180"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sounier, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mas, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steyaert, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laeremans, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manglik, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kobilka, B. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Déméné, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Granier, S.</span></span> <span> </span><span class="NLM_article-title">Propagation of Conformational Changes during Μ-Opioid Receptor Activation</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>524</i></span>,  <span class="NLM_fpage">375</span>– <span class="NLM_lpage">378</span>, <span class="refDoi"> DOI: 10.1038/nature14680</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref180/cit180&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1038%2Fnature14680" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref180/cit180&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=26245377" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref180/cit180&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht12jtb%252FN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=524&publication_year=2015&pages=375-378&author=R.+Sounierauthor=C.+Masauthor=J.+Steyaertauthor=T.+Laeremansauthor=A.+Manglikauthor=W.+Huangauthor=B.+K.+Kobilkaauthor=H.+D%C3%A9m%C3%A9n%C3%A9author=S.+Granier&title=Propagation+of+Conformational+Changes+during+%CE%9C-Opioid+Receptor+Activation&doi=10.1038%2Fnature14680"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit180R"><div class="casContent"><span class="casTitleNuber">180</span><div class="casTitle"><span class="NLM_cas:atitle">Propagation of conformational changes during μ-opioid receptor activation</span></div><div class="casAuthors">Sounier, Remy; Mas, Camille; Steyaert, Jan; Laeremans, Toon; Manglik, Aashish; Huang, Weijiao; Kobilka, Brian K.; Demene, Helene; Granier, Sebastien</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">524</span>
        (<span class="NLM_cas:issue">7565</span>),
    <span class="NLM_cas:pages">375-378</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">μ-Opioid receptors (μORs) are G-protein-coupled receptors that are activated by a structurally diverse spectrum of natural and synthetic agonists including endogenous endorphin peptides, morphine and methadone.  The recent structures of the μOR in inactive and agonist-induced active states (W. Huang et al., 2015) provide snapshots of the receptor at the beginning and end of a signalling event, but little is known about the dynamic sequence of events that span these two states.  Here we use soln.-state NMR to examine the process of μOR activation using a purified receptor (mouse sequence) prepn. in an amphiphile membrane-like environment.  We obtain spectra of the μOR in the absence of ligand, and in the presence of the high-affinity agonist BU 72 alone, or with BU 72 and a G protein mimetic nanobody.  Our results show that conformational changes in transmembrane segments 5 and 6 (TM5 and TM6), which are required for the full engagement of a G protein, are almost completely dependent on the presence of both the agonist and the G protein mimetic nanobody, revealing a weak allosteric coupling between the agonist-binding pocket and the G-protein-coupling interface (TM5 and TM6), similar to that obsd. for the β2-adrenergic receptor.  Unexpectedly, in the presence of agonist alone, we find larger spectral changes involving intracellular loop 1 and helix 8 compared to changes in TM5 and TM6.  These results suggest that one or both of these domains may play a role in the initial interaction with the G protein, and that TM5 and TM6 are only engaged later in the process of complex formation.  The initial interactions between the G protein and intracellular loop 1 and/or helix 8 may be involved in G-protein coupling specificity, as has been suggested for other family A G-protein-coupled receptors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGonzj6Bmj4e5LVg90H21EOLACvtfcHk0liankgwmk4CnA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXht12jtb%252FN&md5=0dd0dedcce583a52b4adc3084726a862</span></div><a href="/servlet/linkout?suffix=cit180&amp;dbid=16384&amp;doi=10.1038%2Fnature14680&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature14680%26sid%3Dliteratum%253Aachs%26aulast%3DSounier%26aufirst%3DR.%26aulast%3DMas%26aufirst%3DC.%26aulast%3DSteyaert%26aufirst%3DJ.%26aulast%3DLaeremans%26aufirst%3DT.%26aulast%3DManglik%26aufirst%3DA.%26aulast%3DHuang%26aufirst%3DW.%26aulast%3DKobilka%26aufirst%3DB.%2BK.%26aulast%3DD%25C3%25A9m%25C3%25A9n%25C3%25A9%26aufirst%3DH.%26aulast%3DGranier%26aufirst%3DS.%26atitle%3DPropagation%2520of%2520Conformational%2520Changes%2520during%2520%25CE%259C-Opioid%2520Receptor%2520Activation%26jtitle%3DNature%26date%3D2015%26volume%3D524%26spage%3D375%26epage%3D378%26doi%3D10.1038%2Fnature14680" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref181"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref181'); return false;" data-citation="" class="refNumLink">181</a></strong><div class="NLM_citation" id="cit181"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pardon, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Betti, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laeremans, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chevillard, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guillemyn, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kolb, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ballet, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steyaert, J.</span></span> <span> </span><span class="NLM_article-title">Nanobody-Enabled Reverse Pharmacology on G-Protein-Coupled Receptors</span>. <i>Angew. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>130</i></span>,  <span class="NLM_fpage">5390</span>– <span class="NLM_lpage">5393</span>, <span class="refDoi"> DOI: 10.1002/ange.201712581</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref181/cit181&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1002%2Fange.201712581" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=130&publication_year=2018&pages=5390-5393&author=E.+Pardonauthor=C.+Bettiauthor=T.+Laeremansauthor=F.+Chevillardauthor=K.+Guillemynauthor=P.+Kolbauthor=S.+Balletauthor=J.+Steyaert&title=Nanobody-Enabled+Reverse+Pharmacology+on+G-Protein-Coupled+Receptors&doi=10.1002%2Fange.201712581"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit181&amp;dbid=16384&amp;doi=10.1002%2Fange.201712581&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fange.201712581%26sid%3Dliteratum%253Aachs%26aulast%3DPardon%26aufirst%3DE.%26aulast%3DBetti%26aufirst%3DC.%26aulast%3DLaeremans%26aufirst%3DT.%26aulast%3DChevillard%26aufirst%3DF.%26aulast%3DGuillemyn%26aufirst%3DK.%26aulast%3DKolb%26aufirst%3DP.%26aulast%3DBallet%26aufirst%3DS.%26aulast%3DSteyaert%26aufirst%3DJ.%26atitle%3DNanobody-Enabled%2520Reverse%2520Pharmacology%2520on%2520G-Protein-Coupled%2520Receptors%26jtitle%3DAngew.%2520Chem.%26date%3D2018%26volume%3D130%26spage%3D5390%26epage%3D5393%26doi%3D10.1002%2Fange.201712581" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref182"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref182'); return false;" data-citation="" class="refNumLink">182</a></strong><div class="NLM_citation" id="cit182"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Manglik, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aryal, D. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCorvy, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dengler, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Corder, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Levit, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kling, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernat, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hübner, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, X.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sassano, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giguère, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Löber, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duan, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scherrer, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kobilka, B. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gmeiner, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roth, B. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shoichet, B. K.</span></span> <span> </span><span class="NLM_article-title">Structure-Based Discovery of Opioid Analgesics with Reduced Side Effects</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>537</i></span>,  <span class="NLM_fpage">185</span>– <span class="NLM_lpage">190</span>, <span class="refDoi"> DOI: 10.1038/nature19112</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref182/cit182&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1038%2Fnature19112" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref182/cit182&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=27533032" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref182/cit182&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtlKmsbvI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=537&publication_year=2016&pages=185-190&author=A.+Manglikauthor=H.+Linauthor=D.+K.+Aryalauthor=J.+D.+McCorvyauthor=D.+Denglerauthor=G.+Corderauthor=A.+Levitauthor=R.+C.+Klingauthor=V.+Bernatauthor=H.+H%C3%BCbnerauthor=X.-P.+Huangauthor=M.+F.+Sassanoauthor=P.+M.+Gigu%C3%A8reauthor=S.+L%C3%B6berauthor=D.+Duanauthor=G.+Scherrerauthor=B.+K.+Kobilkaauthor=P.+Gmeinerauthor=B.+L.+Rothauthor=B.+K.+Shoichet&title=Structure-Based+Discovery+of+Opioid+Analgesics+with+Reduced+Side+Effects&doi=10.1038%2Fnature19112"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit182R"><div class="casContent"><span class="casTitleNuber">182</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-based discovery of opioid analgesics with reduced side effects</span></div><div class="casAuthors">Manglik, Aashish; Lin, Henry; Aryal, Dipendra K.; McCorvy, John D.; Dengler, Daniela; Corder, Gregory; Levit, Anat; Kling, Ralf C.; Bernat, Viachaslau; Hubner, Harald; Huang, Xi-Ping; Sassano, Maria F.; Giguere, Patrick M.; Lober, Stefan; Da Duan; Scherrer, Gregory; Kobilka, Brian K.; Gmeiner, Peter; Roth, Bryan L.; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">537</span>
        (<span class="NLM_cas:issue">7619</span>),
    <span class="NLM_cas:pages">185-190</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Morphine is an alkaloid from the opium poppy used to treat pain.  The potentially lethal side effects of morphine and related opioids-which include fatal respiratory depression-are thought to be mediated by μ-opioid-receptor (μOR) signalling through the β-arrestin pathway or by actions at other receptors.  Conversely, G-protein μOR signalling is thought to confer analgesia.  Here we computationally dock over 3 million mols. against the μOR structure and identify new scaffolds unrelated to known opioids.  Structure-based optimization yields PZM21-a potent Gi activator with exceptional selectivity for μOR and minimal β-arrestin-2 recruitment.  Unlike morphine, PZM21 is more efficacious for the affective component of analgesia vs. the reflexive component and is devoid of both respiratory depression and morphine-like reinforcing activity in mice at equi-analgesic doses.  PZM21 thus serves as both a probe to disentangle μOR signalling and a therapeutic lead that is devoid of many of the side effects of current opioids.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpiCff2w3U8iLVg90H21EOLACvtfcHk0liankgwmk4CnA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtlKmsbvI&md5=b8bb2d8575dadf4a6dc74d94c26d2c1a</span></div><a href="/servlet/linkout?suffix=cit182&amp;dbid=16384&amp;doi=10.1038%2Fnature19112&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature19112%26sid%3Dliteratum%253Aachs%26aulast%3DManglik%26aufirst%3DA.%26aulast%3DLin%26aufirst%3DH.%26aulast%3DAryal%26aufirst%3DD.%2BK.%26aulast%3DMcCorvy%26aufirst%3DJ.%2BD.%26aulast%3DDengler%26aufirst%3DD.%26aulast%3DCorder%26aufirst%3DG.%26aulast%3DLevit%26aufirst%3DA.%26aulast%3DKling%26aufirst%3DR.%2BC.%26aulast%3DBernat%26aufirst%3DV.%26aulast%3DH%25C3%25BCbner%26aufirst%3DH.%26aulast%3DHuang%26aufirst%3DX.-P.%26aulast%3DSassano%26aufirst%3DM.%2BF.%26aulast%3DGigu%25C3%25A8re%26aufirst%3DP.%2BM.%26aulast%3DL%25C3%25B6ber%26aufirst%3DS.%26aulast%3DDuan%26aufirst%3DD.%26aulast%3DScherrer%26aufirst%3DG.%26aulast%3DKobilka%26aufirst%3DB.%2BK.%26aulast%3DGmeiner%26aufirst%3DP.%26aulast%3DRoth%26aufirst%3DB.%2BL.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DStructure-Based%2520Discovery%2520of%2520Opioid%2520Analgesics%2520with%2520Reduced%2520Side%2520Effects%26jtitle%3DNature%26date%3D2016%26volume%3D537%26spage%3D185%26epage%3D190%26doi%3D10.1038%2Fnature19112" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref183"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref183'); return false;" data-citation="" class="refNumLink">183</a></strong><div class="NLM_citation" id="cit183"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Banerjee, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bartesaghi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Merk, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rao, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bulfer, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Green, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mroczkowski, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neitz, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wipf, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Falconieri, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deshaies, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Milne, J. L. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huryn, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arkin, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Subramaniam, S.</span></span> <span> </span><span class="NLM_article-title">2.3 Å Resolution Cryo-EM Structure of Human p97 and Mechanism of Allosteric Inhibition</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>351</i></span>,  <span class="NLM_fpage">871</span>– <span class="NLM_lpage">875</span>, <span class="refDoi"> DOI: 10.1126/science.aad7974</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref183/cit183&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1126%2Fscience.aad7974" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref183/cit183&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=26822609" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref183/cit183&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC28XisFOjsLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=351&publication_year=2016&pages=871-875&author=S.+Banerjeeauthor=A.+Bartesaghiauthor=A.+Merkauthor=P.+Raoauthor=S.+L.+Bulferauthor=Y.+Yanauthor=N.+Greenauthor=B.+Mroczkowskiauthor=R.+J.+Neitzauthor=P.+Wipfauthor=V.+Falconieriauthor=R.+J.+Deshaiesauthor=J.+L.+S.+Milneauthor=D.+Hurynauthor=M.+Arkinauthor=S.+Subramaniam&title=2.3+%C3%85+Resolution+Cryo-EM+Structure+of+Human+p97+and+Mechanism+of+Allosteric+Inhibition&doi=10.1126%2Fscience.aad7974"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit183R"><div class="casContent"><span class="casTitleNuber">183</span><div class="casTitle"><span class="NLM_cas:atitle">2.3 Å resolution cryo-EM structure of human p97 and mechanism of allosteric inhibition</span></div><div class="casAuthors">Banerjee, Soojay; Bartesaghi, Alberto; Merk, Alan; Rao, Prashant; Bulfer, Stacie L.; Yan, Yongzhao; Green, Neal; Mroczkowski, Barbara; Neitz, R. Jeffrey; Wipf, Peter; Falconieri, Veronica; Deshaies, Raymond J.; Milne, Jacqueline L. S.; Huryn, Donna; Arkin, Michelle; Subramaniam, Sriram</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">351</span>
        (<span class="NLM_cas:issue">6275</span>),
    <span class="NLM_cas:pages">871-875</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">P97 is a hexameric AAA+ ATPase that is an attractive target for cancer drug development.  We report cryo-electron microscopy (cryo-EM) structures for ADP-bound, full-length, hexameric wild-type p97 in the presence and absence of an allosteric inhibitor at resolns. of 2.3 and 2.4 angstroms, resp.  We also report cryo-EM structures (at resolns. of ∼3.3, 3.2, and 3.3 angstroms, resp.) for three distinct, coexisting functional states of p97 with occupancies of zero, one, or two mols. of adenosine 5'-O-(3-thiotriphosphate) (ATPγS) per protomer.  A large corkscrew-like change in mol. architecture, coupled with upward displacement of the N-terminal domain, is obsd. only when ATPγS is bound to both the D1 and D2 domains of the protomer.  These cryo-EM structures establish the sequence of nucleotide-driven structural changes in p97 at at. resoln.  They also enable elucidation of the binding mode of an allosteric small-mol. inhibitor to p97 and illustrate how inhibitor binding at the interface between the D1 and D2 domains prevents propagation of the conformational changes necessary for p97 function.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqhbtTnBPFaALVg90H21EOLACvtfcHk0liankgwmk4CnA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XisFOjsLY%253D&md5=14f41313af9dad93f934d07c8bdfc138</span></div><a href="/servlet/linkout?suffix=cit183&amp;dbid=16384&amp;doi=10.1126%2Fscience.aad7974&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.aad7974%26sid%3Dliteratum%253Aachs%26aulast%3DBanerjee%26aufirst%3DS.%26aulast%3DBartesaghi%26aufirst%3DA.%26aulast%3DMerk%26aufirst%3DA.%26aulast%3DRao%26aufirst%3DP.%26aulast%3DBulfer%26aufirst%3DS.%2BL.%26aulast%3DYan%26aufirst%3DY.%26aulast%3DGreen%26aufirst%3DN.%26aulast%3DMroczkowski%26aufirst%3DB.%26aulast%3DNeitz%26aufirst%3DR.%2BJ.%26aulast%3DWipf%26aufirst%3DP.%26aulast%3DFalconieri%26aufirst%3DV.%26aulast%3DDeshaies%26aufirst%3DR.%2BJ.%26aulast%3DMilne%26aufirst%3DJ.%2BL.%2BS.%26aulast%3DHuryn%26aufirst%3DD.%26aulast%3DArkin%26aufirst%3DM.%26aulast%3DSubramaniam%26aufirst%3DS.%26atitle%3D2.3%2520%25C3%2585%2520Resolution%2520Cryo-EM%2520Structure%2520of%2520Human%2520p97%2520and%2520Mechanism%2520of%2520Allosteric%2520Inhibition%26jtitle%3DScience%26date%3D2016%26volume%3D351%26spage%3D871%26epage%3D875%26doi%3D10.1126%2Fscience.aad7974" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref184"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref184'); return false;" data-citation="" class="refNumLink">184</a></strong><div class="NLM_citation" id="cit184"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Merk, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bartesaghi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Banerjee, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Falconieri, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rao, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, M. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pragani, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boxer, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Earl, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Milne, J. L. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Subramaniam, S.</span></span> <span> </span><span class="NLM_article-title">Breaking Cryo-EM Resolution Barriers to Facilitate Drug Discovery</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>165</i></span>,  <span class="NLM_fpage">1698</span>– <span class="NLM_lpage">1707</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2016.05.040</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref184/cit184&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1016%2Fj.cell.2016.05.040" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref184/cit184&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=27238019" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref184/cit184&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC28XptVSlsb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=165&publication_year=2016&pages=1698-1707&author=A.+Merkauthor=A.+Bartesaghiauthor=S.+Banerjeeauthor=V.+Falconieriauthor=P.+Raoauthor=M.+I.+Davisauthor=R.+Praganiauthor=M.+B.+Boxerauthor=L.+A.+Earlauthor=J.+L.+S.+Milneauthor=S.+Subramaniam&title=Breaking+Cryo-EM+Resolution+Barriers+to+Facilitate+Drug+Discovery&doi=10.1016%2Fj.cell.2016.05.040"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit184R"><div class="casContent"><span class="casTitleNuber">184</span><div class="casTitle"><span class="NLM_cas:atitle">Breaking Cryo-EM Resolution Barriers to Facilitate Drug Discovery</span></div><div class="casAuthors">Merk, Alan; Bartesaghi, Alberto; Banerjee, Soojay; Falconieri, Veronica; Rao, Prashant; Davis, Mindy I.; Pragani, Rajan; Boxer, Matthew B.; Earl, Lesley A.; Milne, Jacqueline L. S.; Subramaniam, Sriram</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">165</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1698-1707</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Recent advances in single-particle cryo-electron microscopy (cryo-EM) are enabling generation of numerous near-at. resoln. structures for well-ordered protein complexes with sizes ≥ ∼200 kDa.  Whether cryo-EM methods are equally useful for high-resoln. structural anal. of smaller, dynamic protein complexes such as those involved in cellular metab. remains an important question.  Here, we present 3.8 Å resoln. cryo-EM structures of the cancer target isocitrate dehydrogenase (93 kDa) and identify the nature of conformational changes induced by binding of the allosteric small-mol. inhibitor ML309.  We also report 2.8-Å- and 1.8-Å-resoln. structures of lactate dehydrogenase (145 kDa) and glutamate dehydrogenase (334 kDa), resp.  With these results, two perceived barriers in single-particle cryo-EM are overcome: (1) crossing 2 Å resoln. and (2) obtaining structures of proteins with sizes < 100 kDa, demonstrating that cryo-EM can be used to investigate a broad spectrum of drug-target interactions and dynamic conformational states.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrSVJ4hXEIN07Vg90H21EOLACvtfcHk0lid9cqRE4Sj2A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XptVSlsb8%253D&md5=8457195c1f1724244caa9f90e0ba4682</span></div><a href="/servlet/linkout?suffix=cit184&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2016.05.040&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2016.05.040%26sid%3Dliteratum%253Aachs%26aulast%3DMerk%26aufirst%3DA.%26aulast%3DBartesaghi%26aufirst%3DA.%26aulast%3DBanerjee%26aufirst%3DS.%26aulast%3DFalconieri%26aufirst%3DV.%26aulast%3DRao%26aufirst%3DP.%26aulast%3DDavis%26aufirst%3DM.%2BI.%26aulast%3DPragani%26aufirst%3DR.%26aulast%3DBoxer%26aufirst%3DM.%2BB.%26aulast%3DEarl%26aufirst%3DL.%2BA.%26aulast%3DMilne%26aufirst%3DJ.%2BL.%2BS.%26aulast%3DSubramaniam%26aufirst%3DS.%26atitle%3DBreaking%2520Cryo-EM%2520Resolution%2520Barriers%2520to%2520Facilitate%2520Drug%2520Discovery%26jtitle%3DCell%26date%3D2016%26volume%3D165%26spage%3D1698%26epage%3D1707%26doi%3D10.1016%2Fj.cell.2016.05.040" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref185"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref185'); return false;" data-citation="" class="refNumLink">185</a></strong><div class="NLM_citation" id="cit185"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bai, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scheres, S. H. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, Y.</span></span> <span> </span><span class="NLM_article-title">An Atomic Structure of Human Γ-Secretase</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>525</i></span>,  <span class="NLM_fpage">212</span>– <span class="NLM_lpage">217</span>, <span class="refDoi"> DOI: 10.1038/nature14892</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref185/cit185&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1038%2Fnature14892" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref185/cit185&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=26280335" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref185/cit185&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtlGgurfM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=525&publication_year=2015&pages=212-217&author=X.+Baiauthor=C.+Yanauthor=G.+Yangauthor=P.+Luauthor=D.+Maauthor=L.+Sunauthor=R.+Zhouauthor=S.+H.+W.+Scheresauthor=Y.+Shi&title=An+Atomic+Structure+of+Human+%CE%93-Secretase&doi=10.1038%2Fnature14892"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit185R"><div class="casContent"><span class="casTitleNuber">185</span><div class="casTitle"><span class="NLM_cas:atitle">An atomic structure of human γ-secretase</span></div><div class="casAuthors">Bai, Xiao-chen; Yan, Chuangye; Yang, Guanghui; Lu, Peilong; Ma, Dan; Sun, Linfeng; Zhou, Rui; Scheres, Sjors H. W.; Shi, Yigong</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">525</span>
        (<span class="NLM_cas:issue">7568</span>),
    <span class="NLM_cas:pages">212-217</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Dysfunction of the intramembrane protease γ-secretase is thought to cause Alzheimer's disease, with most mutations derived from Alzheimer's disease mapping to the catalytic subunit presenilin 1 (PS1).  Here we report an at. structure of human γ-secretase at 3.4 Å resoln., detd. by single-particle cryo-electron microscopy.  Mutations derived from Alzheimer's disease affect residues at two hotspots in PS1, each located at the center of a distinct four transmembrane segment (TM) bundle.  TM2 and, to a lesser extent, TM6 exhibit considerable flexibility, yielding a plastic active site and adaptable surrounding elements.  The active site of PS1 is accessible from the convex side of the TM horseshoe, suggesting considerable conformational changes in nicastrin extracellular domain after substrate recruitment.  Component protein APH-1 serves as a scaffold, anchoring the lone transmembrane helix from nicastrin and supporting the flexible conformation of PS1.  Ordered phospholipids stabilize the complex inside the membrane.  Our structure serves as a mol. basis for mechanistic understanding of γ-secretase function.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqGo99dE6LPbbVg90H21EOLACvtfcHk0lid9cqRE4Sj2A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtlGgurfM&md5=68dd390cae90f02f8c6a3d6d9c503326</span></div><a href="/servlet/linkout?suffix=cit185&amp;dbid=16384&amp;doi=10.1038%2Fnature14892&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature14892%26sid%3Dliteratum%253Aachs%26aulast%3DBai%26aufirst%3DX.%26aulast%3DYan%26aufirst%3DC.%26aulast%3DYang%26aufirst%3DG.%26aulast%3DLu%26aufirst%3DP.%26aulast%3DMa%26aufirst%3DD.%26aulast%3DSun%26aufirst%3DL.%26aulast%3DZhou%26aufirst%3DR.%26aulast%3DScheres%26aufirst%3DS.%2BH.%2BW.%26aulast%3DShi%26aufirst%3DY.%26atitle%3DAn%2520Atomic%2520Structure%2520of%2520Human%2520%25CE%2593-Secretase%26jtitle%3DNature%26date%3D2015%26volume%3D525%26spage%3D212%26epage%3D217%26doi%3D10.1038%2Fnature14892" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref186"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref186'); return false;" data-citation="" class="refNumLink">186</a></strong><div class="NLM_citation" id="cit186"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liao, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Julius, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, Y.</span></span> <span> </span><span class="NLM_article-title">Structure of the TRPV1 Ion Channel Determined by Electron Cryo-Microscopy</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>504</i></span>,  <span class="NLM_fpage">107</span>– <span class="NLM_lpage">112</span>, <span class="refDoi"> DOI: 10.1038/nature12822</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref186/cit186&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1038%2Fnature12822" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref186/cit186&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=24305160" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref186/cit186&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvVyis77N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=504&publication_year=2013&pages=107-112&author=M.+Liaoauthor=E.+Caoauthor=D.+Juliusauthor=Y.+Cheng&title=Structure+of+the+TRPV1+Ion+Channel+Determined+by+Electron+Cryo-Microscopy&doi=10.1038%2Fnature12822"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit186R"><div class="casContent"><span class="casTitleNuber">186</span><div class="casTitle"><span class="NLM_cas:atitle">Structure of the TRPV1 ion channel determined by electron cryo-microscopy</span></div><div class="casAuthors">Liao, Maofu; Cao, Erhu; Julius, David; Cheng, Yifan</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">504</span>
        (<span class="NLM_cas:issue">7478</span>),
    <span class="NLM_cas:pages">107-112</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Transient receptor potential (TRP) channels are sensors for a wide range of cellular and environmental signals, but elucidating how these channels respond to phys. and chem. stimuli has been hampered by a lack of detailed structural information.  Here, the authors exploited advances in electron cryo-microscopy to det. the structure of a mammalian TRP channel, TRPV1, at 3.4 Å resoln., breaking the side-chain resoln. barrier for membrane proteins without crystn.  Like voltage-gated channels, TRPV1 exhibited 4-fold symmetry around a central ion pathway formed by transmembrane segments 5-6 (S5-S6) and the intervening pore loop, which was flanked by S1-S4 voltage-sensor-like domains.  TRPV1 had a wide extracellular 'mouth' with a short selectivity filter.  The conserved 'TRP domain' interacted with the S4-S5 linker, consistent with its contribution to allosteric modulation.  Subunit organization was facilitated by interactions among cytoplasmic domains, including N-terminal ankyrin repeats.  These observations provide a structural blueprint for understanding unique aspects of TRP channel function.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqznsggkUfm_7Vg90H21EOLACvtfcHk0lid9cqRE4Sj2A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvVyis77N&md5=bfccb367080288f78abf04dbc114769e</span></div><a href="/servlet/linkout?suffix=cit186&amp;dbid=16384&amp;doi=10.1038%2Fnature12822&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature12822%26sid%3Dliteratum%253Aachs%26aulast%3DLiao%26aufirst%3DM.%26aulast%3DCao%26aufirst%3DE.%26aulast%3DJulius%26aufirst%3DD.%26aulast%3DCheng%26aufirst%3DY.%26atitle%3DStructure%2520of%2520the%2520TRPV1%2520Ion%2520Channel%2520Determined%2520by%2520Electron%2520Cryo-Microscopy%26jtitle%3DNature%26date%3D2013%26volume%3D504%26spage%3D107%26epage%3D112%26doi%3D10.1038%2Fnature12822" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref187"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref187'); return false;" data-citation="" class="refNumLink">187</a></strong><div class="NLM_citation" id="cit187"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span> <span> </span><span class="NLM_article-title">Atomic Structure of the Cystic Fibrosis Transmembrane Conductance Regulator</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>167</i></span>,  <span class="NLM_fpage">1586.e9</span>– <span class="NLM_lpage">1597.e9</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2016.11.014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref187/cit187&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1016%2Fj.cell.2016.11.014" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=167&publication_year=2016&pages=1586.e9-1597.e9&author=Z.+Zhangauthor=J.+Chen&title=Atomic+Structure+of+the+Cystic+Fibrosis+Transmembrane+Conductance+Regulator&doi=10.1016%2Fj.cell.2016.11.014"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit187&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2016.11.014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2016.11.014%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DZ.%26aulast%3DChen%26aufirst%3DJ.%26atitle%3DAtomic%2520Structure%2520of%2520the%2520Cystic%2520Fibrosis%2520Transmembrane%2520Conductance%2520Regulator%26jtitle%3DCell%26date%3D2016%26volume%3D167%26spage%3D1586.e9%26epage%3D1597.e9%26doi%3D10.1016%2Fj.cell.2016.11.014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref188"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref188'); return false;" data-citation="" class="refNumLink">188</a></strong><div class="NLM_citation" id="cit188"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Neutze, R.</span></span> <span> </span><span class="NLM_article-title">Opportunities and Challenges for Time-Resolved Studies of Protein Structural Dynamics at X-Ray Free-Electron Lasers</span>. <i>Philos. Trans. R. Soc., B</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>369</i></span>,  <span class="NLM_fpage">20130318</span>, <span class="refDoi"> DOI: 10.1098/rstb.2013.0318</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref188/cit188&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1098%2Frstb.2013.0318" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref188/cit188&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=24914150" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref188/cit188&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtlaitL3P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=369&publication_year=2014&pages=20130318&author=R.+Neutze&title=Opportunities+and+Challenges+for+Time-Resolved+Studies+of+Protein+Structural+Dynamics+at+X-Ray+Free-Electron+Lasers&doi=10.1098%2Frstb.2013.0318"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit188R"><div class="casContent"><span class="casTitleNuber">188</span><div class="casTitle"><span class="NLM_cas:atitle">Opportunities and challenges for time-resolved studies of protein structural dynamics at X-ray free-electron lasers</span></div><div class="casAuthors">Neutze, Richard</div><div class="citationInfo"><span class="NLM_cas:title">Philosophical Transactions of the Royal Society, B: Biological Sciences</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">369</span>
        (<span class="NLM_cas:issue">1647</span>),
    <span class="NLM_cas:pages">20130318/1-20130318/9</span>CODEN:
                <span class="NLM_cas:coden">PTRBAE</span>;
        ISSN:<span class="NLM_cas:issn">0962-8436</span>.
    
            (<span class="NLM_cas:orgname">Royal Society</span>)
        </div><div class="casAbstract">A review.  X-ray free-electron lasers (XFELs) are revolutionary X-ray sources.  Their time structure, providing X-ray pulses of a few tens of femtoseconds in duration; and their extreme peak brilliance, delivering approx. 1012 X-ray photons per pulse and facilitating sub-micrometre focusing, distinguish XFEL sources from synchrotron radiation.  In this opinion piece, I argue that these properties of XFEL radiation will facilitate new discoveries in life science.  I reason that time-resolved serial femtosecond crystallog. and time-resolved wide angle X-ray scattering are promising areas of scientific investigation that will be advanced by XFEL capabilities, allowing new scientific questions to be addressed that are not accessible using established methods at storage ring facilities.  These questions include visualizing ultrafast protein structural dynamics on the femtosecond to picosecond time-scale, as well as time-resolved diffraction studies of non-cyclic reactions.  I argue that these emerging opportunities will stimulate a renaissance of interest in time-resolved structural biochem.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqdL7GMhFWDu7Vg90H21EOLACvtfcHk0lidMPVXhuL4zA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtlaitL3P&md5=66c77f7e3ae2e7bd004946dcba2027dd</span></div><a href="/servlet/linkout?suffix=cit188&amp;dbid=16384&amp;doi=10.1098%2Frstb.2013.0318&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1098%252Frstb.2013.0318%26sid%3Dliteratum%253Aachs%26aulast%3DNeutze%26aufirst%3DR.%26atitle%3DOpportunities%2520and%2520Challenges%2520for%2520Time-Resolved%2520Studies%2520of%2520Protein%2520Structural%2520Dynamics%2520at%2520X-Ray%2520Free-Electron%2520Lasers%26jtitle%3DPhilos.%2520Trans.%2520R.%2520Soc.%252C%2520B%26date%3D2014%26volume%3D369%26spage%3D20130318%26doi%3D10.1098%2Frstb.2013.0318" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref189"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref189'); return false;" data-citation="" class="refNumLink">189</a></strong><div class="NLM_citation" id="cit189"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, G. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Batyuk, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ishchenko, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">White, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sadybekov, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zamlynny, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rudd, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hollenstein, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tolstikova, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">White, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hunter, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weierstall, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Babaoglu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moore, E. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Katz, R. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shipman, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garcia-Calvo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharma, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheth, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soisson, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stevens, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Katritch, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cherezov, V.</span></span> <span> </span><span class="NLM_article-title">Structural Basis for Selectivity and Diversity in Angiotensin II Receptors</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>544</i></span>,  <span class="NLM_fpage">327</span>– <span class="NLM_lpage">332</span>, <span class="refDoi"> DOI: 10.1038/nature22035</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref189/cit189&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1038%2Fnature22035" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref189/cit189&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=28379944" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref189/cit189&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC2sXlvVKhs7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=544&publication_year=2017&pages=327-332&author=H.+Zhangauthor=G.+W.+Hanauthor=A.+Batyukauthor=A.+Ishchenkoauthor=K.+L.+Whiteauthor=N.+Patelauthor=A.+Sadybekovauthor=B.+Zamlynnyauthor=M.+T.+Ruddauthor=K.+Hollensteinauthor=A.+Tolstikovaauthor=T.+A.+Whiteauthor=M.+S.+Hunterauthor=U.+Weierstallauthor=W.+Liuauthor=K.+Babaogluauthor=E.+L.+Mooreauthor=R.+D.+Katzauthor=J.+M.+Shipmanauthor=M.+Garcia-Calvoauthor=S.+Sharmaauthor=P.+Shethauthor=S.+M.+Soissonauthor=R.+C.+Stevensauthor=V.+Katritchauthor=V.+Cherezov&title=Structural+Basis+for+Selectivity+and+Diversity+in+Angiotensin+II+Receptors&doi=10.1038%2Fnature22035"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit189R"><div class="casContent"><span class="casTitleNuber">189</span><div class="casTitle"><span class="NLM_cas:atitle">Structural basis for selectivity and diversity in angiotensin II receptors</span></div><div class="casAuthors">Zhang, Haitao; Han, Gye Won; Batyuk, Alexander; Ishchenko, Andrii; White, Kate L.; Patel, Nilkanth; Sadybekov, Anastasiia; Zamlynny, Beata; Rudd, Michael T.; Hollenstein, Kaspar; Tolstikova, Alexandra; White, Thomas A.; Hunter, Mark S.; Weierstall, Uwe; Liu, Wei; Babaoglu, Kerim; Moore, Eric L.; Katz, Ryan D.; Shipman, Jennifer M.; Garcia-Calvo, Margarita; Sharma, Sujata; Sheth, Payal; Soisson, Stephen M.; Stevens, Raymond C.; Katritch, Vsevolod; Cherezov, Vadim</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">544</span>
        (<span class="NLM_cas:issue">7650</span>),
    <span class="NLM_cas:pages">327-332</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The angiotensin II receptors AT1R and AT2R serve as key components of the renin-angiotensin-aldosterone system.  AT1R has a central role in the regulation of blood pressure, but the function of AT2R is unclear and it has a variety of reported effects.  To identify the mechanisms that underlie the differences in function and ligand selectivity between these receptors, here the authors report crystal structures of human AT2R bound to an AT2R-selective ligand and to an AT1R/AT2R dual ligand, capturing the receptor in an active-like conformation.  Unexpectedly, helix VIII was found in a non-canonical position, stabilizing the active-like state, but at the same time preventing the recruitment of G proteins or β-arrestins, in agreement with the lack of signaling responses in std. cellular assays.  Structure-activity relationship, docking and mutagenesis studies revealed the crucial interactions for ligand binding and selectivity.  The authors' results thus provide insights into the structural basis of the distinct functions of the angiotensin receptors, and may guide the design of new selective ligands.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGonEzSnnYnMorVg90H21EOLACvtfcHk0lidMPVXhuL4zA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXlvVKhs7c%253D&md5=4473fb24c39c800b2126b1303c02afec</span></div><a href="/servlet/linkout?suffix=cit189&amp;dbid=16384&amp;doi=10.1038%2Fnature22035&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature22035%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DH.%26aulast%3DHan%26aufirst%3DG.%2BW.%26aulast%3DBatyuk%26aufirst%3DA.%26aulast%3DIshchenko%26aufirst%3DA.%26aulast%3DWhite%26aufirst%3DK.%2BL.%26aulast%3DPatel%26aufirst%3DN.%26aulast%3DSadybekov%26aufirst%3DA.%26aulast%3DZamlynny%26aufirst%3DB.%26aulast%3DRudd%26aufirst%3DM.%2BT.%26aulast%3DHollenstein%26aufirst%3DK.%26aulast%3DTolstikova%26aufirst%3DA.%26aulast%3DWhite%26aufirst%3DT.%2BA.%26aulast%3DHunter%26aufirst%3DM.%2BS.%26aulast%3DWeierstall%26aufirst%3DU.%26aulast%3DLiu%26aufirst%3DW.%26aulast%3DBabaoglu%26aufirst%3DK.%26aulast%3DMoore%26aufirst%3DE.%2BL.%26aulast%3DKatz%26aufirst%3DR.%2BD.%26aulast%3DShipman%26aufirst%3DJ.%2BM.%26aulast%3DGarcia-Calvo%26aufirst%3DM.%26aulast%3DSharma%26aufirst%3DS.%26aulast%3DSheth%26aufirst%3DP.%26aulast%3DSoisson%26aufirst%3DS.%2BM.%26aulast%3DStevens%26aufirst%3DR.%2BC.%26aulast%3DKatritch%26aufirst%3DV.%26aulast%3DCherezov%26aufirst%3DV.%26atitle%3DStructural%2520Basis%2520for%2520Selectivity%2520and%2520Diversity%2520in%2520Angiotensin%2520II%2520Receptors%26jtitle%3DNature%26date%3D2017%26volume%3D544%26spage%3D327%26epage%3D332%26doi%3D10.1038%2Fnature22035" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref190"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref190'); return false;" data-citation="" class="refNumLink">190</a></strong><div class="NLM_citation" id="cit190"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ginn, H. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Messerschmidt, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ji, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Axford, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gildea, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winter, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brewster, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hattne, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wagner, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grimes, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evans, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sauter, N. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sutton, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stuart, D. I.</span></span> <span> </span><span class="NLM_article-title">Structure of CPV17 Polyhedrin Determined by the Improved Analysis of Serial Femtosecond Crystallographic Data</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">6435</span>, <span class="refDoi"> DOI: 10.1038/ncomms7435</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref190/cit190&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1038%2Fncomms7435" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref190/cit190&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=25751308" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref190/cit190&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC2MXosFelt7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=6435&author=H.+M.+Ginnauthor=M.+Messerschmidtauthor=X.+Jiauthor=H.+Zhangauthor=D.+Axfordauthor=R.+J.+Gildeaauthor=G.+Winterauthor=A.+S.+Brewsterauthor=J.+Hattneauthor=A.+Wagnerauthor=J.+M.+Grimesauthor=G.+Evansauthor=N.+K.+Sauterauthor=G.+Suttonauthor=D.+I.+Stuart&title=Structure+of+CPV17+Polyhedrin+Determined+by+the+Improved+Analysis+of+Serial+Femtosecond+Crystallographic+Data&doi=10.1038%2Fncomms7435"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit190R"><div class="casContent"><span class="casTitleNuber">190</span><div class="casTitle"><span class="NLM_cas:atitle">Structure of CPV17 polyhedrin determined by the improved analysis of serial femtosecond crystallographic data</span></div><div class="casAuthors">Ginn, Helen M.; Messerschmidt, Marc; Ji, Xiaoyun; Zhang, Hanwen; Axford, Danny; Gildea, Richard J.; Winter, Graeme; Brewster, Aaron S.; Hattne, Johan; Wagner, Armin; Grimes, Jonathan M.; Evans, Gwyndaf; Sauter, Nicholas K.; Sutton, Geoff; Stuart, David I.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Communications</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">6435</span>CODEN:
                <span class="NLM_cas:coden">NCAOBW</span>;
        ISSN:<span class="NLM_cas:issn">2041-1723</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The X-ray free-electron laser (XFEL) allows the anal. of small weakly diffracting protein crystals, but has required very many crystals to obtain good data.  Here we use an XFEL to det. the room temp. at. structure for the smallest cytoplasmic polyhedrosis virus polyhedra yet characterized, which we failed to solve at a synchrotron.  These protein microcrystals, roughly a micron across, accrue within infected cells.  We use a new phys. model for XFEL diffraction, which better ests. the exptl. signal, delivering a high-resoln. XFEL structure (1.75 Å), using fewer crystals than previously required for this resoln.  The crystal lattice and protein core are conserved compared with a polyhedrin with less than 10% sequence identity.  We explain how the conserved biol. phenotype, the crystal lattice, is maintained in the face of extreme environmental challenge and massive evolutionary divergence.  Our improved methods should open up more challenging biol. samples to XFEL anal.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqk0LC5QFL7cLVg90H21EOLACvtfcHk0lidMPVXhuL4zA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXosFelt7o%253D&md5=de15ffb7fc082e8661e4355cc7397ae1</span></div><a href="/servlet/linkout?suffix=cit190&amp;dbid=16384&amp;doi=10.1038%2Fncomms7435&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms7435%26sid%3Dliteratum%253Aachs%26aulast%3DGinn%26aufirst%3DH.%2BM.%26aulast%3DMesserschmidt%26aufirst%3DM.%26aulast%3DJi%26aufirst%3DX.%26aulast%3DZhang%26aufirst%3DH.%26aulast%3DAxford%26aufirst%3DD.%26aulast%3DGildea%26aufirst%3DR.%2BJ.%26aulast%3DWinter%26aufirst%3DG.%26aulast%3DBrewster%26aufirst%3DA.%2BS.%26aulast%3DHattne%26aufirst%3DJ.%26aulast%3DWagner%26aufirst%3DA.%26aulast%3DGrimes%26aufirst%3DJ.%2BM.%26aulast%3DEvans%26aufirst%3DG.%26aulast%3DSauter%26aufirst%3DN.%2BK.%26aulast%3DSutton%26aufirst%3DG.%26aulast%3DStuart%26aufirst%3DD.%2BI.%26atitle%3DStructure%2520of%2520CPV17%2520Polyhedrin%2520Determined%2520by%2520the%2520Improved%2520Analysis%2520of%2520Serial%2520Femtosecond%2520Crystallographic%2520Data%26jtitle%3DNat.%2520Commun.%26date%3D2015%26volume%3D6%26spage%3D6435%26doi%3D10.1038%2Fncomms7435" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref191"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref191'); return false;" data-citation="" class="refNumLink">191</a></strong><div class="NLM_citation" id="cit191"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bublitz, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nass, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drachmann, N. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Markvardsen, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gutmann, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barends, T. R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mattle, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shoeman, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doak, R. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boutet, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Messerschmidt, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seibert, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williams, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Foucar, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reinhard, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sitsel, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gregersen, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clausen, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boesen, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gotfryd, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, K.-T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olesen, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Møller, J. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nissen, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schlichting, I.</span></span> <span> </span><span class="NLM_article-title">Structural Studies of P-Type ATPase–ligand Complexes Using an X-Ray Free-Electron Laser</span>. <i>IUCrJ</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>2</i></span>,  <span class="NLM_fpage">409</span>– <span class="NLM_lpage">420</span>, <span class="refDoi"> DOI: 10.1107/S2052252515008969</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref191/cit191&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1107%2FS2052252515008969" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2015&pages=409-420&author=M.+Bublitzauthor=K.+Nassauthor=N.+D.+Drachmannauthor=A.+J.+Markvardsenauthor=M.+J.+Gutmannauthor=T.+R.+M.+Barendsauthor=D.+Mattleauthor=R.+L.+Shoemanauthor=R.+B.+Doakauthor=S.+Boutetauthor=M.+Messerschmidtauthor=M.+M.+Seibertauthor=G.+J.+Williamsauthor=L.+Foucarauthor=L.+Reinhardauthor=O.+Sitselauthor=J.+L.+Gregersenauthor=J.+D.+Clausenauthor=T.+Boesenauthor=K.+Gotfrydauthor=K.-T.+Wangauthor=C.+Olesenauthor=J.+V.+M%C3%B8llerauthor=P.+Nissenauthor=I.+Schlichting&title=Structural+Studies+of+P-Type+ATPase%E2%80%93ligand+Complexes+Using+an+X-Ray+Free-Electron+Laser&doi=10.1107%2FS2052252515008969"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit191&amp;dbid=16384&amp;doi=10.1107%2FS2052252515008969&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS2052252515008969%26sid%3Dliteratum%253Aachs%26aulast%3DBublitz%26aufirst%3DM.%26aulast%3DNass%26aufirst%3DK.%26aulast%3DDrachmann%26aufirst%3DN.%2BD.%26aulast%3DMarkvardsen%26aufirst%3DA.%2BJ.%26aulast%3DGutmann%26aufirst%3DM.%2BJ.%26aulast%3DBarends%26aufirst%3DT.%2BR.%2BM.%26aulast%3DMattle%26aufirst%3DD.%26aulast%3DShoeman%26aufirst%3DR.%2BL.%26aulast%3DDoak%26aufirst%3DR.%2BB.%26aulast%3DBoutet%26aufirst%3DS.%26aulast%3DMesserschmidt%26aufirst%3DM.%26aulast%3DSeibert%26aufirst%3DM.%2BM.%26aulast%3DWilliams%26aufirst%3DG.%2BJ.%26aulast%3DFoucar%26aufirst%3DL.%26aulast%3DReinhard%26aufirst%3DL.%26aulast%3DSitsel%26aufirst%3DO.%26aulast%3DGregersen%26aufirst%3DJ.%2BL.%26aulast%3DClausen%26aufirst%3DJ.%2BD.%26aulast%3DBoesen%26aufirst%3DT.%26aulast%3DGotfryd%26aufirst%3DK.%26aulast%3DWang%26aufirst%3DK.-T.%26aulast%3DOlesen%26aufirst%3DC.%26aulast%3DM%25C3%25B8ller%26aufirst%3DJ.%2BV.%26aulast%3DNissen%26aufirst%3DP.%26aulast%3DSchlichting%26aufirst%3DI.%26atitle%3DStructural%2520Studies%2520of%2520P-Type%2520ATPase%25E2%2580%2593ligand%2520Complexes%2520Using%2520an%2520X-Ray%2520Free-Electron%2520Laser%26jtitle%3DIUCrJ%26date%3D2015%26volume%3D2%26spage%3D409%26epage%3D420%26doi%3D10.1107%2FS2052252515008969" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref192"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref192'); return false;" data-citation="" class="refNumLink">192</a></strong><div class="NLM_citation" id="cit192"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Thomaston, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woldeyes, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakane, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamashita, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanaka, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koiwai, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brewster, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barad, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lemmin, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uervirojnangkoorn, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arima, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kobayashi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Masuda, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suzuki, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sugahara, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sauter, N. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanaka, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nureki, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tono, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Joti, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nango, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iwata, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yumoto, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fraser, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DeGrado, W. F.</span></span> <span> </span><span class="NLM_article-title">XFEL Structures of the Influenza M2 Proton Channel: Room Temperature Water Networks and Insights into Proton Conduction</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>114</i></span>,  <span class="NLM_fpage">13357</span>– <span class="NLM_lpage">13362</span>, <span class="refDoi"> DOI: 10.1073/pnas.1705624114</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref192/cit192&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1073%2Fpnas.1705624114" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref192/cit192&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=28835537" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref192/cit192&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtlOhurbM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=114&publication_year=2017&pages=13357-13362&author=J.+L.+Thomastonauthor=R.+A.+Woldeyesauthor=T.+Nakaneauthor=A.+Yamashitaauthor=T.+Tanakaauthor=K.+Koiwaiauthor=A.+S.+Brewsterauthor=B.+A.+Baradauthor=Y.+Chenauthor=T.+Lemminauthor=M.+Uervirojnangkoornauthor=T.+Arimaauthor=J.+Kobayashiauthor=T.+Masudaauthor=M.+Suzukiauthor=M.+Sugaharaauthor=N.+K.+Sauterauthor=R.+Tanakaauthor=O.+Nurekiauthor=K.+Tonoauthor=Y.+Jotiauthor=E.+Nangoauthor=S.+Iwataauthor=F.+Yumotoauthor=J.+S.+Fraserauthor=W.+F.+DeGrado&title=XFEL+Structures+of+the+Influenza+M2+Proton+Channel%3A+Room+Temperature+Water+Networks+and+Insights+into+Proton+Conduction&doi=10.1073%2Fpnas.1705624114"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit192R"><div class="casContent"><span class="casTitleNuber">192</span><div class="casTitle"><span class="NLM_cas:atitle">XFEL structures of the influenza M2 proton channel: Room temperature water networks and insights into proton conduction</span></div><div class="casAuthors">Thomaston, Jessica L.; Woldeyes, Rahel A.; Nakane, Takanori; Yamashita, Ayumi; Tanaka, Tomoyuki; Koiwai, Kotaro; Brewster, Aaron S.; Barad, Benjamin A.; Chen, Yujie; Lemmin, Thomas; Uervirojnangkoorn, Monarin; Arima, Toshi; Kobayashi, Jun; Masuda, Tetsuya; Suzuki, Mamoru; Sugahara, Michihiro; Sauter, Nicholas K.; Tanaka, Rie; Nureki, Osamu; Tono, Kensuke; Joti, Yasumasa; Nango, Eriko; Iwata, So; Yumoto, Fumiaki; Fraser, James S.; DeGrado, William F.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">114</span>
        (<span class="NLM_cas:issue">51</span>),
    <span class="NLM_cas:pages">13357-13362</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The M2 proton channel of influenza A virus is a drug target that is essential for the reprodn. of the virus.  It is also a model system for the study of selective, unidirectional proton transport across a membrane.  Ordered water mols. arranged in "wires" inside the channel pore have been proposed to play a role in both the conduction of protons to the 4 gating His-37 residues and the stabilization of multiple pos. charges within the channel.  To visualize the solvent in the pore of the channel at room temp. while minimizing the effects of radiation damage, data were collected to a resoln. of 1.4 Å using an X-ray free-electron laser (XFEL) at 3 different pH conditions: pH 5.5, pH 6.5, and pH 8.0.  Data were collected on the Inwardopen state, which is an intermediate that accumulates at high protonation of the His-37 tetrad.  At pH 5.5, a continuous H-bonded network of water mols. spanned the vertical length of the channel, consistent with a Grotthuss mechanism model for proton transport to the His-37 tetrad.  This ordered solvent at pH 5.5 could act to stabilize the pos. charges that build up on the gating His-37 tetrad during the proton conduction cycle.  The no. of ordered pore waters decreased at pH 6.5 and 8.0, where the Inwardopen state was less stable.  These studies provide a graphical view of the response of water to a change in charge within a restricted channel environment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqgzczipMcns7Vg90H21EOLACvtfcHk0lhDKCjoDpBhAA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtlOhurbM&md5=d44a62000931d80660980d2613efa7e9</span></div><a href="/servlet/linkout?suffix=cit192&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1705624114&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1705624114%26sid%3Dliteratum%253Aachs%26aulast%3DThomaston%26aufirst%3DJ.%2BL.%26aulast%3DWoldeyes%26aufirst%3DR.%2BA.%26aulast%3DNakane%26aufirst%3DT.%26aulast%3DYamashita%26aufirst%3DA.%26aulast%3DTanaka%26aufirst%3DT.%26aulast%3DKoiwai%26aufirst%3DK.%26aulast%3DBrewster%26aufirst%3DA.%2BS.%26aulast%3DBarad%26aufirst%3DB.%2BA.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DLemmin%26aufirst%3DT.%26aulast%3DUervirojnangkoorn%26aufirst%3DM.%26aulast%3DArima%26aufirst%3DT.%26aulast%3DKobayashi%26aufirst%3DJ.%26aulast%3DMasuda%26aufirst%3DT.%26aulast%3DSuzuki%26aufirst%3DM.%26aulast%3DSugahara%26aufirst%3DM.%26aulast%3DSauter%26aufirst%3DN.%2BK.%26aulast%3DTanaka%26aufirst%3DR.%26aulast%3DNureki%26aufirst%3DO.%26aulast%3DTono%26aufirst%3DK.%26aulast%3DJoti%26aufirst%3DY.%26aulast%3DNango%26aufirst%3DE.%26aulast%3DIwata%26aufirst%3DS.%26aulast%3DYumoto%26aufirst%3DF.%26aulast%3DFraser%26aufirst%3DJ.%2BS.%26aulast%3DDeGrado%26aufirst%3DW.%2BF.%26atitle%3DXFEL%2520Structures%2520of%2520the%2520Influenza%2520M2%2520Proton%2520Channel%253A%2520Room%2520Temperature%2520Water%2520Networks%2520and%2520Insights%2520into%2520Proton%2520Conduction%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2017%26volume%3D114%26spage%3D13357%26epage%3D13362%26doi%3D10.1073%2Fpnas.1705624114" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref193"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref193'); return false;" data-citation="" class="refNumLink">193</a></strong><div class="NLM_citation" id="cit193"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Weierstall, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">James, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">White, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spence, J. C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doak, R. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nelson, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fromme, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fromme, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grotjohann, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kupitz, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zatsepin, N. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Basu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wacker, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, G. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Katritch, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boutet, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Messerschmidt, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williams, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koglin, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seibert, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klinker, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gati, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shoeman, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barty, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chapman, H. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirian, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beyerlein, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stevens, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shah, S. T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Howe, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caffrey, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cherezov, V.</span></span> <span> </span><span class="NLM_article-title">Lipidic Cubic Phase Injector Facilitates Membrane Protein Serial Femtosecond Crystallography</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">3309</span>, <span class="refDoi"> DOI: 10.1038/ncomms4309</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref193/cit193&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1038%2Fncomms4309" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref193/cit193&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=24525480" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref193/cit193&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A280%3ADC%252BC2cvltVeqsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&pages=3309&author=U.+Weierstallauthor=D.+Jamesauthor=C.+Wangauthor=T.+A.+Whiteauthor=D.+Wangauthor=W.+Liuauthor=J.+C.+H.+Spenceauthor=R.+B.+Doakauthor=G.+Nelsonauthor=P.+Frommeauthor=R.+Frommeauthor=I.+Grotjohannauthor=C.+Kupitzauthor=N.+A.+Zatsepinauthor=H.+Liuauthor=S.+Basuauthor=D.+Wackerauthor=G.+W.+Hanauthor=V.+Katritchauthor=S.+Boutetauthor=M.+Messerschmidtauthor=G.+J.+Williamsauthor=J.+E.+Koglinauthor=M.+M.+Seibertauthor=M.+Klinkerauthor=C.+Gatiauthor=R.+L.+Shoemanauthor=A.+Bartyauthor=H.+N.+Chapmanauthor=R.+A.+Kirianauthor=K.+R.+Beyerleinauthor=R.+C.+Stevensauthor=D.+Liauthor=S.+T.+A.+Shahauthor=N.+Howeauthor=M.+Caffreyauthor=V.+Cherezov&title=Lipidic+Cubic+Phase+Injector+Facilitates+Membrane+Protein+Serial+Femtosecond+Crystallography&doi=10.1038%2Fncomms4309"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit193R"><div class="casContent"><span class="casTitleNuber">193</span><div class="casTitle"><span class="NLM_cas:atitle">Lipidic cubic phase injector facilitates membrane protein serial femtosecond crystallography</span></div><div class="casAuthors">Weierstall Uwe; James Daniel; Wang Dingjie; Spence John C H; Bruce Doak R; Nelson Garrett; Zatsepin Nadia A; Liu Haiguang; Wang Chong; Liu Wei; Wacker Daniel; Han Gye Won; Katritch Vsevolod; Stevens Raymond C; Cherezov Vadim; White Thomas A; Gati Cornelius; Barty Anton; Beyerlein Kenneth R; Fromme Petra; Fromme Raimund; Grotjohann Ingo; Kupitz Christopher; Basu Shibom; Boutet Sebastien; Messerschmidt Marc; Williams Garth J; Koglin Jason E; Marvin Seibert M; Klinker Markus; Shoeman Robert L; Chapman Henry N; Kirian Richard A; Li Dianfan; Shah Syed T A; Howe Nicole; Caffrey Martin</div><div class="citationInfo"><span class="NLM_cas:title">Nature communications</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">3309</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Lipidic cubic phase (LCP) crystallization has proven successful for high-resolution structure determination of challenging membrane proteins.  Here we present a technique for extruding gel-like LCP with embedded membrane protein microcrystals, providing a continuously renewed source of material for serial femtosecond crystallography.  Data collected from sub-10-μm-sized crystals produced with less than 0.5 mg of purified protein yield structural insights regarding cyclopamine binding to the Smoothened receptor.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQDUg0uqQckxOWhZTslLrfffW6udTcc2eYL53PIwYaugrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2cvltVeqsA%253D%253D&md5=ce335fa0a5bfeb8990c836e7f425e2d2</span></div><a href="/servlet/linkout?suffix=cit193&amp;dbid=16384&amp;doi=10.1038%2Fncomms4309&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms4309%26sid%3Dliteratum%253Aachs%26aulast%3DWeierstall%26aufirst%3DU.%26aulast%3DJames%26aufirst%3DD.%26aulast%3DWang%26aufirst%3DC.%26aulast%3DWhite%26aufirst%3DT.%2BA.%26aulast%3DWang%26aufirst%3DD.%26aulast%3DLiu%26aufirst%3DW.%26aulast%3DSpence%26aufirst%3DJ.%2BC.%2BH.%26aulast%3DDoak%26aufirst%3DR.%2BB.%26aulast%3DNelson%26aufirst%3DG.%26aulast%3DFromme%26aufirst%3DP.%26aulast%3DFromme%26aufirst%3DR.%26aulast%3DGrotjohann%26aufirst%3DI.%26aulast%3DKupitz%26aufirst%3DC.%26aulast%3DZatsepin%26aufirst%3DN.%2BA.%26aulast%3DLiu%26aufirst%3DH.%26aulast%3DBasu%26aufirst%3DS.%26aulast%3DWacker%26aufirst%3DD.%26aulast%3DHan%26aufirst%3DG.%2BW.%26aulast%3DKatritch%26aufirst%3DV.%26aulast%3DBoutet%26aufirst%3DS.%26aulast%3DMesserschmidt%26aufirst%3DM.%26aulast%3DWilliams%26aufirst%3DG.%2BJ.%26aulast%3DKoglin%26aufirst%3DJ.%2BE.%26aulast%3DSeibert%26aufirst%3DM.%2BM.%26aulast%3DKlinker%26aufirst%3DM.%26aulast%3DGati%26aufirst%3DC.%26aulast%3DShoeman%26aufirst%3DR.%2BL.%26aulast%3DBarty%26aufirst%3DA.%26aulast%3DChapman%26aufirst%3DH.%2BN.%26aulast%3DKirian%26aufirst%3DR.%2BA.%26aulast%3DBeyerlein%26aufirst%3DK.%2BR.%26aulast%3DStevens%26aufirst%3DR.%2BC.%26aulast%3DLi%26aufirst%3DD.%26aulast%3DShah%26aufirst%3DS.%2BT.%2BA.%26aulast%3DHowe%26aufirst%3DN.%26aulast%3DCaffrey%26aufirst%3DM.%26aulast%3DCherezov%26aufirst%3DV.%26atitle%3DLipidic%2520Cubic%2520Phase%2520Injector%2520Facilitates%2520Membrane%2520Protein%2520Serial%2520Femtosecond%2520Crystallography%26jtitle%3DNat.%2520Commun.%26date%3D2014%26volume%3D5%26spage%3D3309%26doi%3D10.1038%2Fncomms4309" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref194"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref194'); return false;" data-citation="" class="refNumLink">194</a></strong><div class="NLM_citation" id="cit194"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wacker, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gati, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, G. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">James, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nelson, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weierstall, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Katritch, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barty, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zatsepin, N. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Messerschmidt, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boutet, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williams, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koglin, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seibert, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shah, S. T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Basu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fromme, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kupitz, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rendek, K. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grotjohann, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fromme, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirian, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beyerlein, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">White, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chapman, H. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caffrey, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spence, J. C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stevens, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cherezov, V.</span></span> <span> </span><span class="NLM_article-title">Serial Femtosecond Crystallography of G Protein–Coupled Receptors</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>342</i></span>,  <span class="NLM_fpage">1521</span>– <span class="NLM_lpage">1524</span>, <span class="refDoi"> DOI: 10.1126/science.1244142</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref194/cit194&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1126%2Fscience.1244142" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref194/cit194&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=24357322" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref194/cit194&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvFemu7zE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=342&publication_year=2013&pages=1521-1524&author=W.+Liuauthor=D.+Wackerauthor=C.+Gatiauthor=G.+W.+Hanauthor=D.+Jamesauthor=D.+Wangauthor=G.+Nelsonauthor=U.+Weierstallauthor=V.+Katritchauthor=A.+Bartyauthor=N.+A.+Zatsepinauthor=D.+Liauthor=M.+Messerschmidtauthor=S.+Boutetauthor=G.+J.+Williamsauthor=J.+E.+Koglinauthor=M.+M.+Seibertauthor=C.+Wangauthor=S.+T.+A.+Shahauthor=S.+Basuauthor=R.+Frommeauthor=C.+Kupitzauthor=K.+N.+Rendekauthor=I.+Grotjohannauthor=P.+Frommeauthor=R.+A.+Kirianauthor=K.+R.+Beyerleinauthor=T.+A.+Whiteauthor=H.+N.+Chapmanauthor=M.+Caffreyauthor=J.+C.+H.+Spenceauthor=R.+C.+Stevensauthor=V.+Cherezov&title=Serial+Femtosecond+Crystallography+of+G+Protein%E2%80%93Coupled+Receptors&doi=10.1126%2Fscience.1244142"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit194R"><div class="casContent"><span class="casTitleNuber">194</span><div class="casTitle"><span class="NLM_cas:atitle">Serial Femtosecond Crystallography of G Protein-Coupled Receptors</span></div><div class="casAuthors">Liu, Wei; Wacker, Daniel; Gati, Cornelius; Han, Gye Won; James, Daniel; Wang, Dingjie; Nelson, Garrett; Weierstall, Uwe; Katritch, Vsevolod; Barty, Anton; Zatsepin, Nadia A.; Li, Dianfan; Messerschmidt, Marc; Boutet, Sebastien; Williams, Garth J.; Koglin, Jason E.; Seibert, M. Marvin; Wang, Chong; Shah, Syed T. A.; Basu, Shibom; Fromme, Raimund; Kupitz, Christopher; Rendek, Kimberley N.; Grotjohann, Ingo; Fromme, Petra; Kirian, Richard A.; Beyerlein, Kenneth R.; White, Thomas A.; Chapman, Henry N.; Caffrey, Martin; Spence, John C. H.; Stevens, Raymond C.; Cherezov, Vadim</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">342</span>
        (<span class="NLM_cas:issue">6165</span>),
    <span class="NLM_cas:pages">1521-1524</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">X-ray crystallog. of G protein-coupled receptors and other membrane proteins is hampered by difficulties assocd. with growing sufficiently large crystals that withstand radiation damage and yield high-resoln. data at synchrotron sources.  The authors used an x-ray free-electron laser (XFEL) with individual 50-fs-duration x-ray pulses to minimize radiation damage and obtained a high-resoln. room-temp. structure of a human serotonin receptor using sub-10-μm microcrystals grown in a membrane mimetic matrix known as lipidic cubic phase.  Compared with the structure solved by using traditional microcrystallog. from cryo-cooled crystals of about two orders of magnitude larger vol., the room-temp. XFEL structure displays a distinct distribution of thermal motions and conformations of residues that likely more accurately represent the receptor structure and dynamics in a cellular environment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo5GiymqAvRprVg90H21EOLACvtfcHk0lhLYXpnrO7Kjw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvFemu7zE&md5=a632160c4265cddfebb28ef9b002b38a</span></div><a href="/servlet/linkout?suffix=cit194&amp;dbid=16384&amp;doi=10.1126%2Fscience.1244142&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1244142%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DW.%26aulast%3DWacker%26aufirst%3DD.%26aulast%3DGati%26aufirst%3DC.%26aulast%3DHan%26aufirst%3DG.%2BW.%26aulast%3DJames%26aufirst%3DD.%26aulast%3DWang%26aufirst%3DD.%26aulast%3DNelson%26aufirst%3DG.%26aulast%3DWeierstall%26aufirst%3DU.%26aulast%3DKatritch%26aufirst%3DV.%26aulast%3DBarty%26aufirst%3DA.%26aulast%3DZatsepin%26aufirst%3DN.%2BA.%26aulast%3DLi%26aufirst%3DD.%26aulast%3DMesserschmidt%26aufirst%3DM.%26aulast%3DBoutet%26aufirst%3DS.%26aulast%3DWilliams%26aufirst%3DG.%2BJ.%26aulast%3DKoglin%26aufirst%3DJ.%2BE.%26aulast%3DSeibert%26aufirst%3DM.%2BM.%26aulast%3DWang%26aufirst%3DC.%26aulast%3DShah%26aufirst%3DS.%2BT.%2BA.%26aulast%3DBasu%26aufirst%3DS.%26aulast%3DFromme%26aufirst%3DR.%26aulast%3DKupitz%26aufirst%3DC.%26aulast%3DRendek%26aufirst%3DK.%2BN.%26aulast%3DGrotjohann%26aufirst%3DI.%26aulast%3DFromme%26aufirst%3DP.%26aulast%3DKirian%26aufirst%3DR.%2BA.%26aulast%3DBeyerlein%26aufirst%3DK.%2BR.%26aulast%3DWhite%26aufirst%3DT.%2BA.%26aulast%3DChapman%26aufirst%3DH.%2BN.%26aulast%3DCaffrey%26aufirst%3DM.%26aulast%3DSpence%26aufirst%3DJ.%2BC.%2BH.%26aulast%3DStevens%26aufirst%3DR.%2BC.%26aulast%3DCherezov%26aufirst%3DV.%26atitle%3DSerial%2520Femtosecond%2520Crystallography%2520of%2520G%2520Protein%25E2%2580%2593Coupled%2520Receptors%26jtitle%3DScience%26date%3D2013%26volume%3D342%26spage%3D1521%26epage%3D1524%26doi%3D10.1126%2Fscience.1244142" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref195"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref195'); return false;" data-citation="" class="refNumLink">195</a></strong><div class="NLM_citation" id="cit195"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Charlesworth, C. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deshpande, P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dever, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dejene, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gomez-Ospina, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mantri, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pavel-Dinu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Camarena, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weinberg, K. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Porteus, M. H.</span></span> <span> </span><span class="NLM_article-title">Identification of Pre-Existing Adaptive Immunity to Cas9 Proteins in Humans</span>. <i>bioRxiv</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_fpage">243345</span>, <span class="refDoi"> DOI: 10.1101/243345</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref195/cit195&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1101%2F243345" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2018&pages=243345&author=C.+T.+Charlesworthauthor=P.+S.+Deshpandeauthor=D.+P.+Deverauthor=B.+Dejeneauthor=N.+Gomez-Ospinaauthor=S.+Mantriauthor=M.+Pavel-Dinuauthor=J.+Camarenaauthor=K.+I.+Weinbergauthor=M.+H.+Porteus&title=Identification+of+Pre-Existing+Adaptive+Immunity+to+Cas9+Proteins+in+Humans&doi=10.1101%2F243345"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit195&amp;dbid=16384&amp;doi=10.1101%2F243345&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252F243345%26sid%3Dliteratum%253Aachs%26aulast%3DCharlesworth%26aufirst%3DC.%2BT.%26aulast%3DDeshpande%26aufirst%3DP.%2BS.%26aulast%3DDever%26aufirst%3DD.%2BP.%26aulast%3DDejene%26aufirst%3DB.%26aulast%3DGomez-Ospina%26aufirst%3DN.%26aulast%3DMantri%26aufirst%3DS.%26aulast%3DPavel-Dinu%26aufirst%3DM.%26aulast%3DCamarena%26aufirst%3DJ.%26aulast%3DWeinberg%26aufirst%3DK.%2BI.%26aulast%3DPorteus%26aufirst%3DM.%2BH.%26atitle%3DIdentification%2520of%2520Pre-Existing%2520Adaptive%2520Immunity%2520to%2520Cas9%2520Proteins%2520in%2520Humans%26jtitle%3DbioRxiv%26date%3D2018%26spage%3D243345%26doi%3D10.1101%2F243345" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref196"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref196'); return false;" data-citation="" class="refNumLink">196</a></strong><div class="NLM_citation" id="cit196"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rizvi, N. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, G. F.</span></span> <span> </span><span class="NLM_article-title">RNA as a Small Molecule Druggable Target</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">5083</span>– <span class="NLM_lpage">5088</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2017.10.052</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref196/cit196&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1016%2Fj.bmcl.2017.10.052" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref196/cit196&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=29097169" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref196/cit196&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhslemtLfF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2017&pages=5083-5088&author=N.+F.+Rizviauthor=G.+F.+Smith&title=RNA+as+a+Small+Molecule+Druggable+Target&doi=10.1016%2Fj.bmcl.2017.10.052"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit196R"><div class="casContent"><span class="casTitleNuber">196</span><div class="casTitle"><span class="NLM_cas:atitle">RNA as a small molecule druggable target</span></div><div class="casAuthors">Rizvi, Noreen F.; Smith, Graham F.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">5083-5088</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Small mol. drugs have readily been developed against many proteins in the human proteome, but RNA has remained an elusive target for drug discovery.  Increasingly, the authors see that RNA, and to a lesser extent DNA elements, show a persistent tertiary structure responsible for many diverse and complex cellular functions.  In this digest, the authors have summarized recent advances in screening approaches for RNA targets and outlined the discovery of novel, drug-like small mols. against RNA targets from various classes and therapeutic areas.  The link of structure, function, and small-mol. Druggability validates now for the first time that RNA can be the targets of therapeutic agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpXQHvz_K9pGLVg90H21EOLACvtfcHk0lhLYXpnrO7Kjw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhslemtLfF&md5=ba213f6e8954c97239ccc23f7096c13e</span></div><a href="/servlet/linkout?suffix=cit196&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2017.10.052&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2017.10.052%26sid%3Dliteratum%253Aachs%26aulast%3DRizvi%26aufirst%3DN.%2BF.%26aulast%3DSmith%26aufirst%3DG.%2BF.%26atitle%3DRNA%2520as%2520a%2520Small%2520Molecule%2520Druggable%2520Target%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2017%26volume%3D27%26spage%3D5083%26epage%3D5088%26doi%3D10.1016%2Fj.bmcl.2017.10.052" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref197"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref197'); return false;" data-citation="" class="refNumLink">197</a></strong><div class="NLM_citation" id="cit197"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Collie, G. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parkinson, G. N.</span></span> <span> </span><span class="NLM_article-title">The Application of DNA and RNA G-Quadruplexes to Therapeutic Medicines</span>. <i>Chem. Soc. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>40</i></span>,  <span class="NLM_fpage">5867</span>– <span class="NLM_lpage">5892</span>, <span class="refDoi"> DOI: 10.1039/c1cs15067g</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref197/cit197&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1039%2Fc1cs15067g" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref197/cit197&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=21789296" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref197/cit197&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsVGrsb3L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2011&pages=5867-5892&author=G.+W.+Collieauthor=G.+N.+Parkinson&title=The+Application+of+DNA+and+RNA+G-Quadruplexes+to+Therapeutic+Medicines&doi=10.1039%2Fc1cs15067g"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit197R"><div class="casContent"><span class="casTitleNuber">197</span><div class="casTitle"><span class="NLM_cas:atitle">The application of DNA and RNA G-quadruplexes to therapeutic medicines</span></div><div class="casAuthors">Collie, Gavin W.; Parkinson, Gary N.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Society Reviews</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">5867-5892</span>CODEN:
                <span class="NLM_cas:coden">CSRVBR</span>;
        ISSN:<span class="NLM_cas:issn">0306-0012</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">A review.  The intriguing structural diversity in folded topologies available to guanine-rich nucleic acid repeat sequences have made 4-stranded G-quadruplex structures the focus of both basic and applied research, from cancer biol. and novel therapeutics through to nanoelectronics.  Distributed widely in the human genome as targets for regulating gene expression and chromosomal maintenance, they offer unique avenues for future cancer drug development.  In particular, the recent advances in chem. and structural biol. have enabled the construction of bespoke selective DNA based aptamers to be used as novel therapeutic agents and access to detailed structural models for structure based drug discovery.  In this crit. review, the authors will explore the important underlying characteristics of G-quadruplexes that make them functional, stable, and predictable nanoscaffolds.  The authors will review the current structural database of folding topologies, mol. interfaces and novel interaction surfaces, with a consideration to their future exploitation in drug discovery, mol. biol., supermol. assembly and aptamer design.  In recent years the no. of potential applications for G-quadruplex motifs has rapidly grown, so in this review the authors aim to explore the many future challenges and highlight where possible successes may lie.  The authors will highlight the similarities and differences between DNA and RNA folded G-quadruplexes in terms of stability, distribution, and exploitability as small mol. targets.  Finally, the authors will provide a detailed review of basic G-quadruplex geometry, exptl. tools used, and a crit. evaluation of the application of high-resoln. structural biol. and its ability to provide meaningful and valid models for future applications (255 refs.).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoYJF_jWcCZCLVg90H21EOLACvtfcHk0ljbkfpQFWLjtA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsVGrsb3L&md5=96e22ab0555ed0a3eac39ad091c93b2c</span></div><a href="/servlet/linkout?suffix=cit197&amp;dbid=16384&amp;doi=10.1039%2Fc1cs15067g&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc1cs15067g%26sid%3Dliteratum%253Aachs%26aulast%3DCollie%26aufirst%3DG.%2BW.%26aulast%3DParkinson%26aufirst%3DG.%2BN.%26atitle%3DThe%2520Application%2520of%2520DNA%2520and%2520RNA%2520G-Quadruplexes%2520to%2520Therapeutic%2520Medicines%26jtitle%3DChem.%2520Soc.%2520Rev.%26date%3D2011%26volume%3D40%26spage%3D5867%26epage%3D5892%26doi%3D10.1039%2Fc1cs15067g" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref198"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref198'); return false;" data-citation="" class="refNumLink">198</a></strong><div class="NLM_citation" id="cit198"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Weldon, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Behm-Ansmant, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hurley, L. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burley, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Branlant, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eperon, I. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dominguez, C.</span></span> <span> </span><span class="NLM_article-title">Identification of G-Quadruplexes in Long Functional RNAs Using 7-Deazaguanine RNA</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">18</span>– <span class="NLM_lpage">20</span>, <span class="refDoi"> DOI: 10.1038/nchembio.2228</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref198/cit198&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1038%2Fnchembio.2228" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref198/cit198&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=27820800" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref198/cit198&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvVSntLnN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2017&pages=18-20&author=C.+Weldonauthor=I.+Behm-Ansmantauthor=L.+H.+Hurleyauthor=G.+A.+Burleyauthor=C.+Branlantauthor=I.+C.+Eperonauthor=C.+Dominguez&title=Identification+of+G-Quadruplexes+in+Long+Functional+RNAs+Using+7-Deazaguanine+RNA&doi=10.1038%2Fnchembio.2228"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit198R"><div class="casContent"><span class="casTitleNuber">198</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of G-quadruplexes in long functional RNAs using 7-deazaguanine RNA</span></div><div class="casAuthors">Weldon, Carika; Behm-Ansmant, Isabelle; Hurley, Laurence H.; Burley, Glenn A.; Branlant, Christiane; Eperon, Ian C.; Dominguez, Cyril</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">18-20</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">RNA G-quadruplex (G4) structures are thought to affect biol. processes, including translation and pre-mRNA splicing, but it is not possible at present to demonstrate that they form naturally at specific sequences in long functional RNA mols.  We developed a new strategy, footprinting of long 7-deazaguanine-substituted RNAs (FOLDeR), that allows the formation of G4s to be confirmed in long RNAs and under functional conditions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoEnw-nGuf4ZLVg90H21EOLACvtfcHk0ljbkfpQFWLjtA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvVSntLnN&md5=b1f547bacfc316a5544d6901bab73533</span></div><a href="/servlet/linkout?suffix=cit198&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.2228&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.2228%26sid%3Dliteratum%253Aachs%26aulast%3DWeldon%26aufirst%3DC.%26aulast%3DBehm-Ansmant%26aufirst%3DI.%26aulast%3DHurley%26aufirst%3DL.%2BH.%26aulast%3DBurley%26aufirst%3DG.%2BA.%26aulast%3DBranlant%26aufirst%3DC.%26aulast%3DEperon%26aufirst%3DI.%2BC.%26aulast%3DDominguez%26aufirst%3DC.%26atitle%3DIdentification%2520of%2520G-Quadruplexes%2520in%2520Long%2520Functional%2520RNAs%2520Using%25207-Deazaguanine%2520RNA%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2017%26volume%3D13%26spage%3D18%26epage%3D20%26doi%3D10.1038%2Fnchembio.2228" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref199"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref199'); return false;" data-citation="" class="refNumLink">199</a></strong><div class="NLM_citation" id="cit199"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Palacino, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Swalley, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheung, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Hoosear, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shin, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chin, D. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keller, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beibel, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Renaud, N. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salcius, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hild, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Servais, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jain, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bullock, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McLellan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schuierer, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murphy, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blommers, M. J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blaustein, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berenshteyn, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lacoste, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roma, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michaud, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tseng, B. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Porter, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Myer, V. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tallarico, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamann, L. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Curtis, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fishman, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dietrich, W. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dales, N. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sivasankaran, R.</span></span> <span> </span><span class="NLM_article-title">SMN2 Splice Modulators Enhance U1–pre-mRNA Association and Rescue SMA Mice</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">511</span>– <span class="NLM_lpage">517</span>, <span class="refDoi"> DOI: 10.1038/nchembio.1837</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref199/cit199&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1038%2Fnchembio.1837" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref199/cit199&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=26030728" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref199/cit199&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtFeju7zN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2015&pages=511-517&author=J.+Palacinoauthor=S.+E.+Swalleyauthor=C.+Songauthor=A.+K.+Cheungauthor=L.+Shuauthor=X.+Zhangauthor=M.+Van+Hoosearauthor=Y.+Shinauthor=D.+N.+Chinauthor=C.+G.+Kellerauthor=M.+Beibelauthor=N.+A.+Renaudauthor=T.+M.+Smithauthor=M.+Salciusauthor=X.+Shiauthor=M.+Hildauthor=R.+Servaisauthor=M.+Jainauthor=L.+Dengauthor=C.+Bullockauthor=M.+McLellanauthor=S.+Schuiererauthor=L.+Murphyauthor=M.+J.+J.+Blommersauthor=C.+Blausteinauthor=F.+Berenshteynauthor=A.+Lacosteauthor=J.+R.+Thomasauthor=G.+Romaauthor=G.+A.+Michaudauthor=B.+S.+Tsengauthor=J.+A.+Porterauthor=V.+E.+Myerauthor=J.+A.+Tallaricoauthor=L.+G.+Hamannauthor=D.+Curtisauthor=M.+C.+Fishmanauthor=W.+F.+Dietrichauthor=N.+A.+Dalesauthor=R.+Sivasankaran&title=SMN2+Splice+Modulators+Enhance+U1%E2%80%93pre-mRNA+Association+and+Rescue+SMA+Mice&doi=10.1038%2Fnchembio.1837"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit199R"><div class="casContent"><span class="casTitleNuber">199</span><div class="casTitle"><span class="NLM_cas:atitle">SMN2 splice modulators enhance U1-pre-mRNA association and rescue SMA mice</span></div><div class="casAuthors">Palacino, James; Swalley, Susanne E.; Song, Cheng; Cheung, Atwood K.; Shu, Lei; Zhang, Xiaolu; Van Hoosear, Mailin; Shin, Youngah; Chin, Donovan N.; Keller, Caroline Gubser; Beibel, Martin; Renaud, Nicole A.; Smith, Thomas M.; Salcius, Michael; Shi, Xiaoying; Hild, Marc; Servais, Rebecca; Jain, Monish; Deng, Lin; Bullock, Caroline; McLellan, Michael; Schuierer, Sven; Murphy, Leo; Blommers, Marcel J. J.; Blaustein, Cecile; Berenshteyn, Frada; Lacoste, Arnaud; Thomas, Jason R.; Roma, Guglielmo; Michaud, Gregory A.; Tseng, Brian S.; Porter, Jeffery A.; Myer, Vic E.; Tallarico, John A.; Hamann, Lawrence G.; Curtis, Daniel; Fishman, Mark C.; Dietrich, William F.; Dales, Natalie A.; Sivasankaran, Rajeev</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">511-517</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Spinal muscular atrophy (SMA), which results from the loss of expression of the survival of motor neuron-1 (SMN1) gene, represents the most common genetic cause of pediatric mortality.  A duplicate copy (SMN2) is inefficiently spliced, producing a truncated and unstable protein.  We describe herein a potent, orally active, small-mol. enhancer of SMN2 splicing that elevates full-length SMN protein and extends survival in a severe SMA mouse model.  We demonstrate that the mol. mechanism of action is via stabilization of the transient double-strand RNA structure formed by the SMN2 pre-mRNA and U1 small nuclear ribonucleic protein (snRNP) complex.  The binding affinity of U1 snRNP to the 5' splice site is increased in a sequence-selective manner, discrete from constitutive recognition.  This new mechanism demonstrates the feasibility of small mol.-mediated, sequence-selective splice modulation and the potential for leveraging this strategy in other splicing diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqdMqLIC0lw17Vg90H21EOLACvtfcHk0ljbkfpQFWLjtA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtFeju7zN&md5=15e56c8b3f48feffa150e975a81633a0</span></div><a href="/servlet/linkout?suffix=cit199&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1837&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1837%26sid%3Dliteratum%253Aachs%26aulast%3DPalacino%26aufirst%3DJ.%26aulast%3DSwalley%26aufirst%3DS.%2BE.%26aulast%3DSong%26aufirst%3DC.%26aulast%3DCheung%26aufirst%3DA.%2BK.%26aulast%3DShu%26aufirst%3DL.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DVan%2BHoosear%26aufirst%3DM.%26aulast%3DShin%26aufirst%3DY.%26aulast%3DChin%26aufirst%3DD.%2BN.%26aulast%3DKeller%26aufirst%3DC.%2BG.%26aulast%3DBeibel%26aufirst%3DM.%26aulast%3DRenaud%26aufirst%3DN.%2BA.%26aulast%3DSmith%26aufirst%3DT.%2BM.%26aulast%3DSalcius%26aufirst%3DM.%26aulast%3DShi%26aufirst%3DX.%26aulast%3DHild%26aufirst%3DM.%26aulast%3DServais%26aufirst%3DR.%26aulast%3DJain%26aufirst%3DM.%26aulast%3DDeng%26aufirst%3DL.%26aulast%3DBullock%26aufirst%3DC.%26aulast%3DMcLellan%26aufirst%3DM.%26aulast%3DSchuierer%26aufirst%3DS.%26aulast%3DMurphy%26aufirst%3DL.%26aulast%3DBlommers%26aufirst%3DM.%2BJ.%2BJ.%26aulast%3DBlaustein%26aufirst%3DC.%26aulast%3DBerenshteyn%26aufirst%3DF.%26aulast%3DLacoste%26aufirst%3DA.%26aulast%3DThomas%26aufirst%3DJ.%2BR.%26aulast%3DRoma%26aufirst%3DG.%26aulast%3DMichaud%26aufirst%3DG.%2BA.%26aulast%3DTseng%26aufirst%3DB.%2BS.%26aulast%3DPorter%26aufirst%3DJ.%2BA.%26aulast%3DMyer%26aufirst%3DV.%2BE.%26aulast%3DTallarico%26aufirst%3DJ.%2BA.%26aulast%3DHamann%26aufirst%3DL.%2BG.%26aulast%3DCurtis%26aufirst%3DD.%26aulast%3DFishman%26aufirst%3DM.%2BC.%26aulast%3DDietrich%26aufirst%3DW.%2BF.%26aulast%3DDales%26aufirst%3DN.%2BA.%26aulast%3DSivasankaran%26aufirst%3DR.%26atitle%3DSMN2%2520Splice%2520Modulators%2520Enhance%2520U1%25E2%2580%2593pre-mRNA%2520Association%2520and%2520Rescue%2520SMA%2520Mice%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2015%26volume%3D11%26spage%3D511%26epage%3D517%26doi%3D10.1038%2Fnchembio.1837" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref200"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref200'); return false;" data-citation="" class="refNumLink">200</a></strong><div class="NLM_citation" id="cit200"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Naryshkin, N. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weetall, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dakka, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Narasimhan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ling, K. K. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karp, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woll, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gabbeta, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vazirani, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhattacharyya, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Furia, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Risher, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheedy, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kong, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turpoff, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, C.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moon, Y.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trifillis, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Welch, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Colacino, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Babiak, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Almstead, N. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peltz, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eng, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mull, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lynes, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rubin, L. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fontoura, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Santarelli, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haehnke, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCarthy, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmucki, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ebeling, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sivaramakrishnan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ko, C.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paushkin, S. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ratni, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerlach, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ghosh, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Metzger, F.</span></span> <span> </span><span class="NLM_article-title">SMN2 Splicing Modifiers Improve Motor Function and Longevity in Mice with Spinal Muscular Atrophy</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>345</i></span>,  <span class="NLM_fpage">688</span>– <span class="NLM_lpage">693</span>, <span class="refDoi"> DOI: 10.1126/science.1250127</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref200/cit200&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1126%2Fscience.1250127" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref200/cit200&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=25104390" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref200/cit200&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC2cXht12gtLfO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=345&publication_year=2014&pages=688-693&author=N.+A.+Naryshkinauthor=M.+Weetallauthor=A.+Dakkaauthor=J.+Narasimhanauthor=X.+Zhaoauthor=Z.+Fengauthor=K.+K.+Y.+Lingauthor=G.+M.+Karpauthor=H.+Qiauthor=M.+G.+Wollauthor=G.+Chenauthor=N.+Zhangauthor=V.+Gabbetaauthor=P.+Vaziraniauthor=A.+Bhattacharyyaauthor=B.+Furiaauthor=N.+Risherauthor=J.+Sheedyauthor=R.+Kongauthor=J.+Maauthor=A.+Turpoffauthor=C.-S.+Leeauthor=X.+Zhangauthor=Y.-C.+Moonauthor=P.+Trifillisauthor=E.+M.+Welchauthor=J.+M.+Colacinoauthor=J.+Babiakauthor=N.+G.+Almsteadauthor=S.+W.+Peltzauthor=L.+A.+Engauthor=K.+S.+Chenauthor=J.+L.+Mullauthor=M.+S.+Lynesauthor=L.+L.+Rubinauthor=P.+Fontouraauthor=L.+Santarelliauthor=D.+Haehnkeauthor=K.+D.+McCarthyauthor=R.+Schmuckiauthor=M.+Ebelingauthor=M.+Sivaramakrishnanauthor=C.-P.+Koauthor=S.+V.+Paushkinauthor=H.+Ratniauthor=I.+Gerlachauthor=A.+Ghoshauthor=F.+Metzger&title=SMN2+Splicing+Modifiers+Improve+Motor+Function+and+Longevity+in+Mice+with+Spinal+Muscular+Atrophy&doi=10.1126%2Fscience.1250127"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit200R"><div class="casContent"><span class="casTitleNuber">200</span><div class="casTitle"><span class="NLM_cas:atitle">SMN2 splicing modifiers improve motor function and longevity in mice with spinal muscular atrophy</span></div><div class="casAuthors">Naryshkin, Nikolai A.; Weetall, Marla; Dakka, Amal; Narasimhan, Jana; Zhao, Xin; Feng, Zhihua; Ling, Karen K. Y.; Karp, Gary M.; Qi, Hongyan; Woll, Matthew G.; Chen, Guangming; Zhang, Nanjing; Gabbeta, Vijayalakshmi; Vazirani, Priya; Bhattacharyya, Anuradha; Furia, Bansri; Risher, Nicole; Sheedy, Josephine; Kong, Ronald; Ma, Jiyuan; Turpoff, Anthony; Lee, Chang-Sun; Zhang, Xiaoyan; Moon, Young-Choon; Trifillis, Panayiota; Welch, Ellen M.; Colacino, Joseph M.; Babiak, John; Almstead, Neil G.; Peltz, Stuart W.; Eng, Loren A.; Chen, Karen S.; Mull, Jesse L.; Lynes, Maureen S.; Rubin, Lee L.; Fontoura, Paulo; Santarelli, Luca; Haehnke, Daniel; McCarthy, Kathleen D.; Schmucki, Roland; Ebeling, Martin; Sivaramakrishnan, Manaswini; Ko, Chien-Ping; Paushkin, Sergey V.; Ratni, Hasane; Gerlach, Irene; Ghosh, Anirvan; Metzger, Friedrich</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">345</span>
        (<span class="NLM_cas:issue">6197</span>),
    <span class="NLM_cas:pages">688-693</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Spinal muscular atrophy (SMA) is a genetic disease caused by mutation or deletion of the survival of motor neuron 1 (SMN1) gene.  A paralogous gene in humans, SMN2, produces low, insufficient levels of functional SMN protein due to alternative splicing that truncates the transcript.  The decreased levels of SMN protein lead to progressive neuromuscular degeneration and high rates of mortality.  Through chem. screening and optimization, we identified orally available small mols. that shift the balance of SMN2 splicing toward the prodn. of full-length SMN2 mRNA with high selectivity.  Administration of these compds. to Δ7 mice, a model of severe SMA, led to an increase in SMN protein levels, improvement of motor function, and protection of the neuromuscular circuit.  These compds. also extended the life span of the mice.  Selective SMN2 splicing modifiers may have therapeutic potential for patients with SMA.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoxZ8QSg5KSzLVg90H21EOLACvtfcHk0lh0Qlmec1t8Yw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXht12gtLfO&md5=f321c411bb384f95a9fe4d4debcbb195</span></div><a href="/servlet/linkout?suffix=cit200&amp;dbid=16384&amp;doi=10.1126%2Fscience.1250127&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1250127%26sid%3Dliteratum%253Aachs%26aulast%3DNaryshkin%26aufirst%3DN.%2BA.%26aulast%3DWeetall%26aufirst%3DM.%26aulast%3DDakka%26aufirst%3DA.%26aulast%3DNarasimhan%26aufirst%3DJ.%26aulast%3DZhao%26aufirst%3DX.%26aulast%3DFeng%26aufirst%3DZ.%26aulast%3DLing%26aufirst%3DK.%2BK.%2BY.%26aulast%3DKarp%26aufirst%3DG.%2BM.%26aulast%3DQi%26aufirst%3DH.%26aulast%3DWoll%26aufirst%3DM.%2BG.%26aulast%3DChen%26aufirst%3DG.%26aulast%3DZhang%26aufirst%3DN.%26aulast%3DGabbeta%26aufirst%3DV.%26aulast%3DVazirani%26aufirst%3DP.%26aulast%3DBhattacharyya%26aufirst%3DA.%26aulast%3DFuria%26aufirst%3DB.%26aulast%3DRisher%26aufirst%3DN.%26aulast%3DSheedy%26aufirst%3DJ.%26aulast%3DKong%26aufirst%3DR.%26aulast%3DMa%26aufirst%3DJ.%26aulast%3DTurpoff%26aufirst%3DA.%26aulast%3DLee%26aufirst%3DC.-S.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DMoon%26aufirst%3DY.-C.%26aulast%3DTrifillis%26aufirst%3DP.%26aulast%3DWelch%26aufirst%3DE.%2BM.%26aulast%3DColacino%26aufirst%3DJ.%2BM.%26aulast%3DBabiak%26aufirst%3DJ.%26aulast%3DAlmstead%26aufirst%3DN.%2BG.%26aulast%3DPeltz%26aufirst%3DS.%2BW.%26aulast%3DEng%26aufirst%3DL.%2BA.%26aulast%3DChen%26aufirst%3DK.%2BS.%26aulast%3DMull%26aufirst%3DJ.%2BL.%26aulast%3DLynes%26aufirst%3DM.%2BS.%26aulast%3DRubin%26aufirst%3DL.%2BL.%26aulast%3DFontoura%26aufirst%3DP.%26aulast%3DSantarelli%26aufirst%3DL.%26aulast%3DHaehnke%26aufirst%3DD.%26aulast%3DMcCarthy%26aufirst%3DK.%2BD.%26aulast%3DSchmucki%26aufirst%3DR.%26aulast%3DEbeling%26aufirst%3DM.%26aulast%3DSivaramakrishnan%26aufirst%3DM.%26aulast%3DKo%26aufirst%3DC.-P.%26aulast%3DPaushkin%26aufirst%3DS.%2BV.%26aulast%3DRatni%26aufirst%3DH.%26aulast%3DGerlach%26aufirst%3DI.%26aulast%3DGhosh%26aufirst%3DA.%26aulast%3DMetzger%26aufirst%3DF.%26atitle%3DSMN2%2520Splicing%2520Modifiers%2520Improve%2520Motor%2520Function%2520and%2520Longevity%2520in%2520Mice%2520with%2520Spinal%2520Muscular%2520Atrophy%26jtitle%3DScience%26date%3D2014%26volume%3D345%26spage%3D688%26epage%3D693%26doi%3D10.1126%2Fscience.1250127" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref201"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref201'); return false;" data-citation="" class="refNumLink">201</a></strong><div class="NLM_citation" id="cit201"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lintner, N. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McClure, K. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petersen, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Londregan, A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Piotrowski, D. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bolt, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loria, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maguire, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geoghegan, K. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rolph, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liras, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doudna, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dullea, R. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cate, J. H. D.</span></span> <span> </span><span class="NLM_article-title">Selective Stalling of Human Translation through Small-Molecule Engagement of the Ribosome Nascent Chain</span>. <i>PLoS Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">e2001882</span>, <span class="refDoi"> DOI: 10.1371/journal.pbio.2001882</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref201/cit201&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1371%2Fjournal.pbio.2001882" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref201/cit201&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=28323820" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref201/cit201&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC2sXht1WqurfE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2017&pages=e2001882&author=N.+G.+Lintnerauthor=K.+F.+McClureauthor=D.+Petersenauthor=A.+T.+Londreganauthor=D.+W.+Piotrowskiauthor=L.+Weiauthor=J.+Xiaoauthor=M.+Boltauthor=P.+M.+Loriaauthor=B.+Maguireauthor=K.+F.+Geogheganauthor=A.+Huangauthor=T.+Rolphauthor=S.+Lirasauthor=J.+A.+Doudnaauthor=R.+G.+Dulleaauthor=J.+H.+D.+Cate&title=Selective+Stalling+of+Human+Translation+through+Small-Molecule+Engagement+of+the+Ribosome+Nascent+Chain&doi=10.1371%2Fjournal.pbio.2001882"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit201R"><div class="casContent"><span class="casTitleNuber">201</span><div class="casTitle"><span class="NLM_cas:atitle">Selective stalling of human translation through small-molecule engagement of the ribosome nascent chain</span></div><div class="casAuthors">Lintner, Nathanael G.; McClure, Kim F.; Petersen, Donna; Londregan, Allyn T.; Piotrowski, David W.; Wei, Liuqing; Xiao, Jun; Bolt, Michael; Loria, Paula M.; Maguire, Bruce; Geoghegan, Kieran F.; Huang, Austin; Rolph, Tim; Liras, Spiros; Doudna, Jennifer A.; Dullea, Robert G.; Cate, Jamie H. D.</div><div class="citationInfo"><span class="NLM_cas:title">PLoS Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">e2001882/1-e2001882/36</span>CODEN:
                <span class="NLM_cas:coden">PBLIBG</span>;
        ISSN:<span class="NLM_cas:issn">1545-7885</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Proprotein convertase subtilisin/kexin type 9 (PCSK9) plays a key role in regulating the levels of plasma low-d. lipoprotein cholesterol (LDL-C).  Here, we demonstrate that the compd. PF-06446846 inhibits translation of PCSK9 by inducing the ribosome to stall around codon 34, mediated by the sequence of the nascent chain within the exit tunnel.  We further show that PF-06446846 reduces plasma PCSK9 and total cholesterol levels in rats following oral dosing.  Using ribosome profiling, we demonstrate that PF-06446846 is highly selective for the inhibition of PCSK9 translation.  The mechanism of action employed by PF-06446846 reveals a previously unexpected tunability of the human ribosome that allows small mols. to specifically block translation of individual transcripts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpqeQbFLtKlX7Vg90H21EOLACvtfcHk0lh0Qlmec1t8Yw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXht1WqurfE&md5=64f33ee234c2425258154ac8368d2518</span></div><a href="/servlet/linkout?suffix=cit201&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pbio.2001882&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pbio.2001882%26sid%3Dliteratum%253Aachs%26aulast%3DLintner%26aufirst%3DN.%2BG.%26aulast%3DMcClure%26aufirst%3DK.%2BF.%26aulast%3DPetersen%26aufirst%3DD.%26aulast%3DLondregan%26aufirst%3DA.%2BT.%26aulast%3DPiotrowski%26aufirst%3DD.%2BW.%26aulast%3DWei%26aufirst%3DL.%26aulast%3DXiao%26aufirst%3DJ.%26aulast%3DBolt%26aufirst%3DM.%26aulast%3DLoria%26aufirst%3DP.%2BM.%26aulast%3DMaguire%26aufirst%3DB.%26aulast%3DGeoghegan%26aufirst%3DK.%2BF.%26aulast%3DHuang%26aufirst%3DA.%26aulast%3DRolph%26aufirst%3DT.%26aulast%3DLiras%26aufirst%3DS.%26aulast%3DDoudna%26aufirst%3DJ.%2BA.%26aulast%3DDullea%26aufirst%3DR.%2BG.%26aulast%3DCate%26aufirst%3DJ.%2BH.%2BD.%26atitle%3DSelective%2520Stalling%2520of%2520Human%2520Translation%2520through%2520Small-Molecule%2520Engagement%2520of%2520the%2520Ribosome%2520Nascent%2520Chain%26jtitle%3DPLoS%2520Biol.%26date%3D2017%26volume%3D15%26spage%3De2001882%26doi%3D10.1371%2Fjournal.pbio.2001882" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref202"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref202'); return false;" data-citation="" class="refNumLink">202</a></strong><div class="NLM_citation" id="cit202"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Blakeley, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DePorter, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mohan, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burai, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tolbert, B. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McNaughton, B. R.</span></span> <span> </span><span class="NLM_article-title">Methods for Identifying and Characterizing Interactions Involving RNA</span>. <i>Tetrahedron</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>68</i></span>,  <span class="NLM_fpage">8837</span>– <span class="NLM_lpage">8855</span>, <span class="refDoi"> DOI: 10.1016/j.tet.2012.07.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref202/cit202&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1016%2Fj.tet.2012.07.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref202/cit202&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtVyjsLnP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=68&publication_year=2012&pages=8837-8855&author=B.+D.+Blakeleyauthor=S.+M.+DePorterauthor=U.+Mohanauthor=R.+Buraiauthor=B.+S.+Tolbertauthor=B.+R.+McNaughton&title=Methods+for+Identifying+and+Characterizing+Interactions+Involving+RNA&doi=10.1016%2Fj.tet.2012.07.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit202R"><div class="casContent"><span class="casTitleNuber">202</span><div class="casTitle"><span class="NLM_cas:atitle">Methods for identifying and characterizing interactions involving RNA</span></div><div class="casAuthors">Blakeley, Brett D.; DePorter, Sandra M.; Mohan, Utpal; Burai, Ritwik; Tolbert, Blanton S.; McNaughton, Brian R.</div><div class="citationInfo"><span class="NLM_cas:title">Tetrahedron</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">68</span>
        (<span class="NLM_cas:issue">43</span>),
    <span class="NLM_cas:pages">8837-8855</span>CODEN:
                <span class="NLM_cas:coden">TETRAB</span>;
        ISSN:<span class="NLM_cas:issn">0040-4020</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Topics discussed include fluorescence-based methods, microarrays, selection-based methods, and low-throughput methods for measuring ligand-RNA interactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpc9bjJ4rDA_LVg90H21EOLACvtfcHk0lgNjsRt9eBLlg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtVyjsLnP&md5=ecbfe5736239b30b9e615d3b51dc78ed</span></div><a href="/servlet/linkout?suffix=cit202&amp;dbid=16384&amp;doi=10.1016%2Fj.tet.2012.07.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tet.2012.07.001%26sid%3Dliteratum%253Aachs%26aulast%3DBlakeley%26aufirst%3DB.%2BD.%26aulast%3DDePorter%26aufirst%3DS.%2BM.%26aulast%3DMohan%26aufirst%3DU.%26aulast%3DBurai%26aufirst%3DR.%26aulast%3DTolbert%26aufirst%3DB.%2BS.%26aulast%3DMcNaughton%26aufirst%3DB.%2BR.%26atitle%3DMethods%2520for%2520Identifying%2520and%2520Characterizing%2520Interactions%2520Involving%2520RNA%26jtitle%3DTetrahedron%26date%3D2012%26volume%3D68%26spage%3D8837%26epage%3D8855%26doi%3D10.1016%2Fj.tet.2012.07.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref203"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref203'); return false;" data-citation="" class="refNumLink">203</a></strong><div class="NLM_citation" id="cit203"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Moon, M. H.</span>; <span class="NLM_string-name">Hilimire, T. A.</span>; <span class="NLM_string-name">Sanders, A. M.</span>; <span class="NLM_string-name">Schneekloth, J. S.</span></span> <span> </span><span class="NLM_article-title">Measuring RNA–Ligand Interactions with Microscale Thermophoresis</span>.  <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2018</span>, <span class="refDoi"> DOI: 10.1021/acs.biochem.7b01141</span> .</span><div class="citationLinks">[<a href="/doi/10.1021/acs.biochem.7b01141" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2018&author=M.+H.+Moon&author=T.+A.+Hilimire&author=A.+M.+Sanders&author=J.+S.+Schneekloth&title=Measuring+RNA%E2%80%93Ligand+Interactions+with+Microscale+Thermophoresis&doi=10.1021%2Facs.biochem.7b01141"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit203&amp;dbid=16384&amp;doi=10.1021%2Facs.biochem.7b01141&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.biochem.7b01141%26sid%3Dliteratum%253Aachs%26aulast%3DMoon%26aufirst%3DM.%2BH.%26atitle%3DMeasuring%2520RNA%25E2%2580%2593Ligand%2520Interactions%2520with%2520Microscale%2520Thermophoresis%26jtitle%3DBiochemistry%26date%3D2018%26doi%3D10.1021%2Facs.biochem.7b01141" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref204"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref204'); return false;" data-citation="" class="refNumLink">204</a></strong><div class="NLM_citation" id="cit204"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Disney, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winkelsas, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Velagapudi, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Southern, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fallahi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Childs-Disney, J. L.</span></span> <span> </span><span class="NLM_article-title">Inforna 2.0: A Platform for the Sequence-Based Design of Small Molecules Targeting Structured RNAs</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">1720</span>– <span class="NLM_lpage">1728</span>, <span class="refDoi"> DOI: 10.1021/acschembio.6b00001</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.6b00001" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref204/cit204&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC28Xmt1Cgsrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2016&pages=1720-1728&author=M.+D.+Disneyauthor=A.+M.+Winkelsasauthor=S.+P.+Velagapudiauthor=M.+Southernauthor=M.+Fallahiauthor=J.+L.+Childs-Disney&title=Inforna+2.0%3A+A+Platform+for+the+Sequence-Based+Design+of+Small+Molecules+Targeting+Structured+RNAs&doi=10.1021%2Facschembio.6b00001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit204R"><div class="casContent"><span class="casTitleNuber">204</span><div class="casTitle"><span class="NLM_cas:atitle">Inforna 2.0: A Platform for the Sequence-Based Design of Small Molecules Targeting Structured RNAs</span></div><div class="casAuthors">Disney, Matthew D.; Winkelsas, Audrey M.; Velagapudi, Sai Pradeep; Southern, Mark; Fallahi, Mohammad; Childs-Disney, Jessica L.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1720-1728</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The development of small mols. that target RNA is challenging yet, if successful, could advance the development of chem. probes to study RNA function or precision therapeutics to treat RNA-mediated disease.  Previously, we described Inforna, an approach that can mine motifs (secondary structures) within target RNAs, which is deduced from the RNA sequence, and compare them to a database of known RNA motif-small mol. binding partners.  Output generated by Inforna includes the motif found in both the database and the desired RNA target, lead small mols. for that target, and other related meta-data.  Lead small mols. can then be tested for binding and affecting cellular (dys)function.  Herein, we describe Inforna 2.0, which incorporates all known RNA motif-small mol. binding partners reported in the scientific literature, a chem. similarity searching feature, and an improved user interface and is freely available via an online web server.  By incorporation of interactions identified by other labs., the database has been doubled, contg. 1936 RNA motif-small mol. interactions, including 244 unique small mols. and 1331 motifs.  Interestingly, chemotype anal. of the compds. that bind RNA in the database reveals features in small mol. chemotypes that are privileged for binding.  Further, this updated database expanded the no. of cellular RNAs to which lead compds. can be identified.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoHmJfcxIk9yrVg90H21EOLACvtfcHk0lgNjsRt9eBLlg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xmt1Cgsrw%253D&md5=6086bc48066565e519f7ee9e9cf2542a</span></div><a href="/servlet/linkout?suffix=cit204&amp;dbid=16384&amp;doi=10.1021%2Facschembio.6b00001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.6b00001%26sid%3Dliteratum%253Aachs%26aulast%3DDisney%26aufirst%3DM.%2BD.%26aulast%3DWinkelsas%26aufirst%3DA.%2BM.%26aulast%3DVelagapudi%26aufirst%3DS.%2BP.%26aulast%3DSouthern%26aufirst%3DM.%26aulast%3DFallahi%26aufirst%3DM.%26aulast%3DChilds-Disney%26aufirst%3DJ.%2BL.%26atitle%3DInforna%25202.0%253A%2520A%2520Platform%2520for%2520the%2520Sequence-Based%2520Design%2520of%2520Small%2520Molecules%2520Targeting%2520Structured%2520RNAs%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2016%26volume%3D11%26spage%3D1720%26epage%3D1728%26doi%3D10.1021%2Facschembio.6b00001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref205"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref205'); return false;" data-citation="" class="refNumLink">205</a></strong><div class="NLM_citation" id="cit205"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rzuczek, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Colgan, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakai, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cameron, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Furling, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yasuda, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Disney, M. D.</span></span> <span> </span><span class="NLM_article-title">Precise Small-Molecule Recognition of a Toxic CUG RNA Repeat Expansion</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">188</span>– <span class="NLM_lpage">193</span>, <span class="refDoi"> DOI: 10.1038/nchembio.2251</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref205/cit205&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1038%2Fnchembio.2251" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref205/cit205&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=27941760" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref205/cit205&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC28XitVSrurbM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2017&pages=188-193&author=S.+G.+Rzuczekauthor=L.+A.+Colganauthor=Y.+Nakaiauthor=M.+D.+Cameronauthor=D.+Furlingauthor=R.+Yasudaauthor=M.+D.+Disney&title=Precise+Small-Molecule+Recognition+of+a+Toxic+CUG+RNA+Repeat+Expansion&doi=10.1038%2Fnchembio.2251"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit205R"><div class="casContent"><span class="casTitleNuber">205</span><div class="casTitle"><span class="NLM_cas:atitle">Precise small-molecule recognition of a toxic CUG RNA repeat expansion</span></div><div class="casAuthors">Rzuczek, Suzanne G.; Colgan, Lesley A.; Nakai, Yoshio; Cameron, Michael D.; Furling, Denis; Yasuda, Ryohei; Disney, Matthew D.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">188-193</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Excluding the ribosome and riboswitches, developing small mols. that selectively target RNA is a longstanding problem in chem. biol.  A typical cellular RNA is difficult to target because it has little tertiary, but abundant secondary structure.  We designed allele-selective compds. that target such an RNA, the toxic noncoding repeat expansion (r(CUG)exp) that causes myotonic dystrophy type 1 (DM1).  We developed several strategies to generate allele-selective small mols., including non-covalent binding, covalent binding, cleavage and on-site probe synthesis.  Covalent binding and cleavage enabled target profiling in cells derived from individuals with DM1, showing precise recognition of r(CUG)exp.  In the on-site probe synthesis approach, small mols. bound adjacent sites in r(CUG)exp and reacted to afford picomolar inhibitors via a proximity-based click reaction only in DM1-affected cells.  We expanded this approach to image r(CUG)exp in its natural context.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrC5kTu0Wkj9bVg90H21EOLACvtfcHk0lgNjsRt9eBLlg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitVSrurbM&md5=bb30d79fe6e8d513a4956d13ee20f69c</span></div><a href="/servlet/linkout?suffix=cit205&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.2251&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.2251%26sid%3Dliteratum%253Aachs%26aulast%3DRzuczek%26aufirst%3DS.%2BG.%26aulast%3DColgan%26aufirst%3DL.%2BA.%26aulast%3DNakai%26aufirst%3DY.%26aulast%3DCameron%26aufirst%3DM.%2BD.%26aulast%3DFurling%26aufirst%3DD.%26aulast%3DYasuda%26aufirst%3DR.%26aulast%3DDisney%26aufirst%3DM.%2BD.%26atitle%3DPrecise%2520Small-Molecule%2520Recognition%2520of%2520a%2520Toxic%2520CUG%2520RNA%2520Repeat%2520Expansion%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2017%26volume%3D13%26spage%3D188%26epage%3D193%26doi%3D10.1038%2Fnchembio.2251" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref206"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref206'); return false;" data-citation="" class="refNumLink">206</a></strong><div class="NLM_citation" id="cit206"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Velagapudi, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tran, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haniff, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakai, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fallahi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martinez, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Childs-Disney, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Disney, M. D.</span></span> <span> </span><span class="NLM_article-title">Defining RNA–Small Molecule Affinity Landscapes Enables Design of a Small Molecule Inhibitor of an Oncogenic Noncoding RNA</span>. <i>ACS Cent. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">205</span>– <span class="NLM_lpage">216</span>, <span class="refDoi"> DOI: 10.1021/acscentsci.7b00009</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acscentsci.7b00009" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref206/cit206&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC2sXjs1GqsLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2017&pages=205-216&author=S.+P.+Velagapudiauthor=Y.+Luoauthor=T.+Tranauthor=H.+S.+Haniffauthor=Y.+Nakaiauthor=M.+Fallahiauthor=G.+J.+Martinezauthor=J.+L.+Childs-Disneyauthor=M.+D.+Disney&title=Defining+RNA%E2%80%93Small+Molecule+Affinity+Landscapes+Enables+Design+of+a+Small+Molecule+Inhibitor+of+an+Oncogenic+Noncoding+RNA&doi=10.1021%2Facscentsci.7b00009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit206R"><div class="casContent"><span class="casTitleNuber">206</span><div class="casTitle"><span class="NLM_cas:atitle">Defining RNA-Small Molecule Affinity Landscapes Enables Design of a Small Molecule Inhibitor of an Oncogenic Noncoding RNA</span></div><div class="casAuthors">Velagapudi, Sai Pradeep; Luo, Yiling; Tran, Tuan; Haniff, Hafeez S.; Nakai, Yoshio; Fallahi, Mohammad; Martinez, Gustavo J.; Childs-Disney, Jessica L.; Disney, Matthew D.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Central Science</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">205-216</span>CODEN:
                <span class="NLM_cas:coden">ACSCII</span>;
        ISSN:<span class="NLM_cas:issn">2374-7951</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">RNA drug targets are pervasive in cells but methods to design small mols. that target them are sparse.  Herein, the authors report a general approach to score the affinity and selectivity of RNA motif-small mol. interactions identified via selection.  Named High Throughput Structure-Activity Relationships Through Sequencing (HiT-StARTS), HiT-StARTS is statistical in nature and compares input nucleic acid sequences to selected library members that bind a ligand via high throughput sequencing.  The approach allowed facile definition of the fitness landscape of hundreds of thousands of RNA motif-small mol. binding partners.  These results were mined against folded RNAs in the human transcriptome and identified an avid interaction between a small mol. and the Dicer nuclease-processing site in the oncogenic microRNA (miR)-18a hairpin precursor, which is a member of the miR-17-92 cluster.  Application of the small mol., Targapremir-18a, to prostate cancer cells inhibited prodn. of miR-18a from the cluster, derepressed serine/threonine protein kinase 4 protein (STK4), and triggered apoptosis.  Profiling the cellular targets of Targapremir-18a via Chem. Cross Linking and Isolation by Pull Down (Chem-CLIP), a covalent small mol.-RNA cellular profiling approach, and other studies showed specific binding of the compd. to the miR-18a precursor, revealing broadly applicable factors that govern small mol. drugging of noncoding RNAs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPmptGpjN_4rVg90H21EOLACvtfcHk0ljbzMvK-gCwKg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXjs1GqsLs%253D&md5=fefd2fe6e9d637c4aa04fd1e25fb1446</span></div><a href="/servlet/linkout?suffix=cit206&amp;dbid=16384&amp;doi=10.1021%2Facscentsci.7b00009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facscentsci.7b00009%26sid%3Dliteratum%253Aachs%26aulast%3DVelagapudi%26aufirst%3DS.%2BP.%26aulast%3DLuo%26aufirst%3DY.%26aulast%3DTran%26aufirst%3DT.%26aulast%3DHaniff%26aufirst%3DH.%2BS.%26aulast%3DNakai%26aufirst%3DY.%26aulast%3DFallahi%26aufirst%3DM.%26aulast%3DMartinez%26aufirst%3DG.%2BJ.%26aulast%3DChilds-Disney%26aufirst%3DJ.%2BL.%26aulast%3DDisney%26aufirst%3DM.%2BD.%26atitle%3DDefining%2520RNA%25E2%2580%2593Small%2520Molecule%2520Affinity%2520Landscapes%2520Enables%2520Design%2520of%2520a%2520Small%2520Molecule%2520Inhibitor%2520of%2520an%2520Oncogenic%2520Noncoding%2520RNA%26jtitle%3DACS%2520Cent.%2520Sci.%26date%3D2017%26volume%3D3%26spage%3D205%26epage%3D216%26doi%3D10.1021%2Facscentsci.7b00009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref207"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref207'); return false;" data-citation="" class="refNumLink">207</a></strong><div class="NLM_citation" id="cit207"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Costales, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haga, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Velagapudi, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Childs-Disney, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phinney, D. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Disney, M. D.</span></span> <span> </span><span class="NLM_article-title">Small Molecule Inhibition of microRNA-210 Reprograms an Oncogenic Hypoxic Circuit</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>139</i></span>,  <span class="NLM_fpage">3446</span>– <span class="NLM_lpage">3455</span>, <span class="refDoi"> DOI: 10.1021/jacs.6b11273</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.6b11273" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref207/cit207&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC2sXjtlCjt7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=139&publication_year=2017&pages=3446-3455&author=M.+G.+Costalesauthor=C.+L.+Hagaauthor=S.+P.+Velagapudiauthor=J.+L.+Childs-Disneyauthor=D.+G.+Phinneyauthor=M.+D.+Disney&title=Small+Molecule+Inhibition+of+microRNA-210+Reprograms+an+Oncogenic+Hypoxic+Circuit&doi=10.1021%2Fjacs.6b11273"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit207R"><div class="casContent"><span class="casTitleNuber">207</span><div class="casTitle"><span class="NLM_cas:atitle">Small Molecule Inhibition of microRNA-210 Reprograms an Oncogenic Hypoxic Circuit</span></div><div class="casAuthors">Costales, Matthew G.; Haga, Christopher L.; Velagapudi, Sai Pradeep; Childs-Disney, Jessica L.; Phinney, Donald G.; Disney, Matthew D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">139</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">3446-3455</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A hypoxic state is crit. to the metastatic and invasive characteristics of cancer.  Numerous pathways play crit. roles in cancer maintenance, many of which include noncoding RNAs such as microRNA (miR)-210 that regulates hypoxia inducible factors (HIFs).  Herein, we describe the identification of a small mol. named Targapremir-210 that binds to the Dicer site of the miR-210 hairpin precursor.  This interaction inhibits prodn. of the mature miRNA, derepresses glycerol-3-phosphate dehydrogenase 1-like enzyme (GPD1L), a hypoxia-assocd. protein neg. regulated by miR-210, decreases HIF-1α, and triggers apoptosis of triple neg. breast cancer cells only under hypoxic conditions.  Further, Targapremir-210 inhibits tumorigenesis in a mouse xenograft model of hypoxic triple neg. breast cancer.  Many factors govern mol. recognition of biol. targets by small mols.  For protein, chemoproteomics and activity-based protein profiling are invaluable tools to study small mol. target engagement and selectivity in cells.  Such approaches are lacking for RNA, leaving a void in the understanding of its druggability.  We applied Chem. Crosslinking and Isolation by Pull Down (Chem-CLIP) to study the cellular selectivity and the on- and off-targets of Targapremir-210.  Targapremir-210 selectively recognizes the miR-210 precursor and can differentially recognize RNAs in cells that have the same target motif but have different expression levels, revealing this important feature for selectively drugging RNAs for the first time.  These studies show that small mols. can be rapidly designed to selectively target RNAs and affect cellular responses to environmental conditions, resulting in favorable benefits against cancer.  Further, they help define rules for identifying druggable targets in the transcriptome.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpAY_YQQyzKvLVg90H21EOLACvtfcHk0ljbzMvK-gCwKg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXjtlCjt7w%253D&md5=c6725837e6f1713facb507b518998a8f</span></div><a href="/servlet/linkout?suffix=cit207&amp;dbid=16384&amp;doi=10.1021%2Fjacs.6b11273&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.6b11273%26sid%3Dliteratum%253Aachs%26aulast%3DCostales%26aufirst%3DM.%2BG.%26aulast%3DHaga%26aufirst%3DC.%2BL.%26aulast%3DVelagapudi%26aufirst%3DS.%2BP.%26aulast%3DChilds-Disney%26aufirst%3DJ.%2BL.%26aulast%3DPhinney%26aufirst%3DD.%2BG.%26aulast%3DDisney%26aufirst%3DM.%2BD.%26atitle%3DSmall%2520Molecule%2520Inhibition%2520of%2520microRNA-210%2520Reprograms%2520an%2520Oncogenic%2520Hypoxic%2520Circuit%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2017%26volume%3D139%26spage%3D3446%26epage%3D3455%26doi%3D10.1021%2Fjacs.6b11273" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref208"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref208'); return false;" data-citation="" class="refNumLink">208</a></strong><div class="NLM_citation" id="cit208"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Howe, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fischmann, T. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Balibar, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiao, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galgoci, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Malinverni, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mayhood, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Villafania, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nahvi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murgolo, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barbieri, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mann, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carr, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xia, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zuck, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riley, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Painter, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walker, S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sherborne, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Jesus, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pan, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plotkin, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rindgen, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cummings, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garlisi, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheth, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gill, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roemer, T.</span></span> <span> </span><span class="NLM_article-title">Selective Small-Molecule Inhibition of an RNA Structural Element</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>526</i></span>,  <span class="NLM_fpage">672</span>– <span class="NLM_lpage">677</span>, <span class="refDoi"> DOI: 10.1038/nature15542</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref208/cit208&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1038%2Fnature15542" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref208/cit208&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=26416753" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref208/cit208&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhslCnu7%252FL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=526&publication_year=2015&pages=672-677&author=J.+A.+Howeauthor=H.+Wangauthor=T.+O.+Fischmannauthor=C.+J.+Balibarauthor=L.+Xiaoauthor=A.+M.+Galgociauthor=J.+C.+Malinverniauthor=T.+Mayhoodauthor=A.+Villafaniaauthor=A.+Nahviauthor=N.+Murgoloauthor=C.+M.+Barbieriauthor=P.+A.+Mannauthor=D.+Carrauthor=E.+Xiaauthor=P.+Zuckauthor=D.+Rileyauthor=R.+E.+Painterauthor=S.+S.+Walkerauthor=B.+Sherborneauthor=R.+de+Jesusauthor=W.+Panauthor=M.+A.+Plotkinauthor=J.+Wuauthor=D.+Rindgenauthor=J.+Cummingsauthor=C.+G.+Garlisiauthor=R.+Zhangauthor=P.+R.+Shethauthor=C.+J.+Gillauthor=H.+Tangauthor=T.+Roemer&title=Selective+Small-Molecule+Inhibition+of+an+RNA+Structural+Element&doi=10.1038%2Fnature15542"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit208R"><div class="casContent"><span class="casTitleNuber">208</span><div class="casTitle"><span class="NLM_cas:atitle">Selective small-molecule inhibition of an RNA structural element</span></div><div class="casAuthors">Howe, John A.; Wang, Hao; Fischmann, Thierry O.; Balibar, Carl J.; Xiao, Li; Galgoci, Andrew M.; Malinverni, Juliana C.; Mayhood, Todd; Villafania, Artjohn; Nahvi, Ali; Murgolo, Nicholas; Barbieri, Christopher M.; Mann, Paul A.; Carr, Donna; Xia, Ellen; Zuck, Paul; Riley, Dan; Painter, Ronald E.; Walker, Scott S.; Sherborne, Brad; de Jesus, Reynalda; Pan, Weidong; Plotkin, Michael A.; Wu, Jin; Rindgen, Diane; Cummings, John; Garlisi, Charles G.; Zhang, Rumin; Sheth, Payal R.; Gill, Charles J.; Tang, Haifeng; Roemer, Terry</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">526</span>
        (<span class="NLM_cas:issue">7575</span>),
    <span class="NLM_cas:pages">672-677</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Riboswitches are non-coding RNA structures located in mRNAs that bind endogenous ligands, such as a specific metabolite or ion, to regulate gene expression.  As such, riboswitches serve as a novel, yet largely unexploited, class of emerging drug targets.  Demonstrating this potential, however, has proven difficult and is restricted to structurally similar antimetabolites and semi-synthetic analogs of their cognate ligand, thus greatly restricting the chem. space and selectivity sought for such inhibitors.  Here we report the discovery and characterization of ribocil, a highly selective chem. modulator of bacterial riboflavin riboswitches, which was identified in a phenotypic screen and acts as a structurally distinct synthetic mimic of the natural ligand, FMN, to repress riboswitch-mediated ribB gene expression and inhibit bacterial cell growth.  Our findings indicate that non-coding RNA structural elements may be more broadly targeted by synthetic small mols. than previously expected.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqWptaVLK3njLVg90H21EOLACvtfcHk0ljbzMvK-gCwKg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhslCnu7%252FL&md5=c0f8443348c10f69eb47f05a2ac5e893</span></div><a href="/servlet/linkout?suffix=cit208&amp;dbid=16384&amp;doi=10.1038%2Fnature15542&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature15542%26sid%3Dliteratum%253Aachs%26aulast%3DHowe%26aufirst%3DJ.%2BA.%26aulast%3DWang%26aufirst%3DH.%26aulast%3DFischmann%26aufirst%3DT.%2BO.%26aulast%3DBalibar%26aufirst%3DC.%2BJ.%26aulast%3DXiao%26aufirst%3DL.%26aulast%3DGalgoci%26aufirst%3DA.%2BM.%26aulast%3DMalinverni%26aufirst%3DJ.%2BC.%26aulast%3DMayhood%26aufirst%3DT.%26aulast%3DVillafania%26aufirst%3DA.%26aulast%3DNahvi%26aufirst%3DA.%26aulast%3DMurgolo%26aufirst%3DN.%26aulast%3DBarbieri%26aufirst%3DC.%2BM.%26aulast%3DMann%26aufirst%3DP.%2BA.%26aulast%3DCarr%26aufirst%3DD.%26aulast%3DXia%26aufirst%3DE.%26aulast%3DZuck%26aufirst%3DP.%26aulast%3DRiley%26aufirst%3DD.%26aulast%3DPainter%26aufirst%3DR.%2BE.%26aulast%3DWalker%26aufirst%3DS.%2BS.%26aulast%3DSherborne%26aufirst%3DB.%26aulast%3Dde%2BJesus%26aufirst%3DR.%26aulast%3DPan%26aufirst%3DW.%26aulast%3DPlotkin%26aufirst%3DM.%2BA.%26aulast%3DWu%26aufirst%3DJ.%26aulast%3DRindgen%26aufirst%3DD.%26aulast%3DCummings%26aufirst%3DJ.%26aulast%3DGarlisi%26aufirst%3DC.%2BG.%26aulast%3DZhang%26aufirst%3DR.%26aulast%3DSheth%26aufirst%3DP.%2BR.%26aulast%3DGill%26aufirst%3DC.%2BJ.%26aulast%3DTang%26aufirst%3DH.%26aulast%3DRoemer%26aufirst%3DT.%26atitle%3DSelective%2520Small-Molecule%2520Inhibition%2520of%2520an%2520RNA%2520Structural%2520Element%26jtitle%3DNature%26date%3D2015%26volume%3D526%26spage%3D672%26epage%3D677%26doi%3D10.1038%2Fnature15542" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref209"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref209'); return false;" data-citation="" class="refNumLink">209</a></strong><div class="NLM_citation" id="cit209"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Stelzer, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frank, A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kratz, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Swanson, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gonzalez-Hernandez, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andricioaei, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Markovitz, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Al-Hashimi, H. M.</span></span> <span> </span><span class="NLM_article-title">Discovery of Selective Bioactive Small Molecules by Targeting an RNA Dynamic Ensemble</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">553</span>– <span class="NLM_lpage">559</span>, <span class="refDoi"> DOI: 10.1038/nchembio.596</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref209/cit209&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1038%2Fnchembio.596" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref209/cit209&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=21706033" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref209/cit209&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC3MXotV2htbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2011&pages=553-559&author=A.+C.+Stelzerauthor=A.+T.+Frankauthor=J.+D.+Kratzauthor=M.+D.+Swansonauthor=M.+J.+Gonzalez-Hernandezauthor=J.+Leeauthor=I.+Andricioaeiauthor=D.+M.+Markovitzauthor=H.+M.+Al-Hashimi&title=Discovery+of+Selective+Bioactive+Small+Molecules+by+Targeting+an+RNA+Dynamic+Ensemble&doi=10.1038%2Fnchembio.596"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit209R"><div class="casContent"><span class="casTitleNuber">209</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of selective bioactive small molecules by targeting an RNA dynamic ensemble</span></div><div class="casAuthors">Stelzer, Andrew C.; Frank, Aaron T.; Kratz, Jeremy D.; Swanson, Michael D.; Gonzalez-Hernandez, Marta J.; Lee, Janghyun; Andricioaei, Ioan; Markovitz, David M.; Al-Hashimi, Hashim M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">553-559</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Current approaches used to identify protein-binding small mols. are not suited for identifying small mols. that can bind emerging RNA drug targets.  By docking small mols. onto an RNA dynamic ensemble constructed by combining NMR spectroscopy and computational mol. dynamics, the authors virtually screened small mols. that target the entire structure landscape of the transactivation response element (TAR) from HIV type 1 (HIV-1).  The authors quant. predict binding energies for small mols. that bind different RNA conformations and report the de novo discovery of six compds. that bind TAR with high affinity and inhibit its interaction with a Tat peptide in vitro (Ki values of 710 nM-169 μM).  One compd. binds HIV-1 TAR with marked selectivity and inhibits Tat-mediated activation of the HIV-1 long terminal repeat by 81% in T-cell lines and HIV replication in an HIV-1 indicator cell line (IC50 ∼23.1 μM).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoQ1jtoA-Z8RrVg90H21EOLACvtfcHk0lhv5-kG7o4IUw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXotV2htbw%253D&md5=cd04f57b1fececc2f4aa678fe3f1f3f2</span></div><a href="/servlet/linkout?suffix=cit209&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.596&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.596%26sid%3Dliteratum%253Aachs%26aulast%3DStelzer%26aufirst%3DA.%2BC.%26aulast%3DFrank%26aufirst%3DA.%2BT.%26aulast%3DKratz%26aufirst%3DJ.%2BD.%26aulast%3DSwanson%26aufirst%3DM.%2BD.%26aulast%3DGonzalez-Hernandez%26aufirst%3DM.%2BJ.%26aulast%3DLee%26aufirst%3DJ.%26aulast%3DAndricioaei%26aufirst%3DI.%26aulast%3DMarkovitz%26aufirst%3DD.%2BM.%26aulast%3DAl-Hashimi%26aufirst%3DH.%2BM.%26atitle%3DDiscovery%2520of%2520Selective%2520Bioactive%2520Small%2520Molecules%2520by%2520Targeting%2520an%2520RNA%2520Dynamic%2520Ensemble%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2011%26volume%3D7%26spage%3D553%26epage%3D559%26doi%3D10.1038%2Fnchembio.596" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref210"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref210'); return false;" data-citation="" class="refNumLink">210</a></strong><div class="NLM_citation" id="cit210"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rizvi, N. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Howe, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nahvi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klein, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fischmann, T. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, H.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walker, S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hruza, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richards, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chamberlin, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saradjian, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Butko, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mercado, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burchard, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Strickland, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dandliker, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, G. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nickbarg, E. B.</span></span> <span> </span><span class="NLM_article-title">Discovery of Selective RNA-Binding Small Molecules by Affinity-Selection Mass Spectrometry</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">820</span>– <span class="NLM_lpage">831</span>, <span class="refDoi"> DOI: 10.1021/acschembio.7b01013</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.7b01013" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref210/cit210&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC1cXit1yku7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2018&pages=820-831&author=N.+F.+Rizviauthor=J.+A.+Howeauthor=A.+Nahviauthor=D.+J.+Kleinauthor=T.+O.+Fischmannauthor=H.-Y.+Kimauthor=M.+A.+McCoyauthor=S.+S.+Walkerauthor=A.+Hruzaauthor=M.+P.+Richardsauthor=C.+Chamberlinauthor=P.+Saradjianauthor=M.+T.+Butkoauthor=G.+Mercadoauthor=J.+Burchardauthor=C.+Stricklandauthor=P.+J.+Dandlikerauthor=G.+F.+Smithauthor=E.+B.+Nickbarg&title=Discovery+of+Selective+RNA-Binding+Small+Molecules+by+Affinity-Selection+Mass+Spectrometry&doi=10.1021%2Facschembio.7b01013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit210R"><div class="casContent"><span class="casTitleNuber">210</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Selective RNA-Binding Small Molecules by Affinity-Selection Mass Spectrometry</span></div><div class="casAuthors">Rizvi, Noreen F.; Howe, John A.; Nahvi, Ali; Klein, Daniel J.; Fischmann, Thierry O.; Kim, Hai-Young; McCoy, Mark A.; Walker, Scott S.; Hruza, Alan; Richards, Matthew P.; Chamberlin, Chad; Saradjian, Peter; Butko, Margaret T.; Mercado, Gabriel; Burchard, Julja; Strickland, Corey; Dandliker, Peter J.; Smith, Graham F.; Nickbarg, Elliott B.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">820-831</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Recent advances in understanding the relevance of noncoding RNA (ncRNA) to disease have increased interest in drugging ncRNA with small mols.  The recent discovery of ribocil, a structurally distinct synthetic mimic of the natural ligand of the FMN riboswitch, revealed the potential chem. diversity of small mols. that target ncRNA.  Affinity-selection mass spectrometry (AS-MS) is theor. applicable to high-throughput screening (HTS) of small mols. binding to ncRNA.  Here, the authors report the first application of the Automated Ligand Detection System (ALIS), an indirect AS-MS technique, for the selective detection of small mol.-ncRNA interactions, high-throughput screening against large unbiased small-mol. libraries, and identification and characterization of novel compds. (structurally distinct from both FMN and ribocil) that target the FMN riboswitch.  Crystal structures reveal that different compds. induce various conformations of the FMN riboswitch, leading to different activity profiles.  The authors' findings validate the ALIS platform for HTS screening for RNA-binding small mols. and further demonstrate that ncRNA can be broadly targeted by chem. diverse yet selective small mols. as therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoW6w6M7wToLrVg90H21EOLACvtfcHk0lhv5-kG7o4IUw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXit1yku7o%253D&md5=b489e442832bcd12515e7c07ebc69432</span></div><a href="/servlet/linkout?suffix=cit210&amp;dbid=16384&amp;doi=10.1021%2Facschembio.7b01013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.7b01013%26sid%3Dliteratum%253Aachs%26aulast%3DRizvi%26aufirst%3DN.%2BF.%26aulast%3DHowe%26aufirst%3DJ.%2BA.%26aulast%3DNahvi%26aufirst%3DA.%26aulast%3DKlein%26aufirst%3DD.%2BJ.%26aulast%3DFischmann%26aufirst%3DT.%2BO.%26aulast%3DKim%26aufirst%3DH.-Y.%26aulast%3DMcCoy%26aufirst%3DM.%2BA.%26aulast%3DWalker%26aufirst%3DS.%2BS.%26aulast%3DHruza%26aufirst%3DA.%26aulast%3DRichards%26aufirst%3DM.%2BP.%26aulast%3DChamberlin%26aufirst%3DC.%26aulast%3DSaradjian%26aufirst%3DP.%26aulast%3DButko%26aufirst%3DM.%2BT.%26aulast%3DMercado%26aufirst%3DG.%26aulast%3DBurchard%26aufirst%3DJ.%26aulast%3DStrickland%26aufirst%3DC.%26aulast%3DDandliker%26aufirst%3DP.%2BJ.%26aulast%3DSmith%26aufirst%3DG.%2BF.%26aulast%3DNickbarg%26aufirst%3DE.%2BB.%26atitle%3DDiscovery%2520of%2520Selective%2520RNA-Binding%2520Small%2520Molecules%2520by%2520Affinity-Selection%2520Mass%2520Spectrometry%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2018%26volume%3D13%26spage%3D820%26epage%3D831%26doi%3D10.1021%2Facschembio.7b01013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref211"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref211'); return false;" data-citation="" class="refNumLink">211</a></strong><div class="NLM_citation" id="cit211"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lai, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Induced Protein Degradation: An Emerging Drug Discovery Paradigm</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">101</span>– <span class="NLM_lpage">114</span>, <span class="refDoi"> DOI: 10.1038/nrd.2016.211</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref211/cit211&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1038%2Fnrd.2016.211" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref211/cit211&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=27885283" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref211/cit211&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvFCit7rP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2017&pages=101-114&author=A.+C.+Laiauthor=C.+M.+Crews&title=Induced+Protein+Degradation%3A+An+Emerging+Drug+Discovery+Paradigm&doi=10.1038%2Fnrd.2016.211"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit211R"><div class="casContent"><span class="casTitleNuber">211</span><div class="casTitle"><span class="NLM_cas:atitle">Induced protein degradation: an emerging drug discovery paradigm</span></div><div class="casAuthors">Lai, Ashton C.; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">101-114</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Small-mol. drug discovery has traditionally focused on occupancy of a binding site that directly affects protein function, and this approach typically precludes targeting proteins that lack such amenable sites.  Furthermore, high systemic drug exposures may be needed to maintain sufficient target inhibition in vivo, increasing the risk of undesirable off-target effects.  Induced protein degrdn. is an alternative approach that is event-driven: upon drug binding, the target protein is tagged for elimination.  Emerging technologies based on proteolysis-targeting chimaeras (PROTACs) that exploit cellular quality control machinery to selectively degrade target proteins are attracting considerable attention in the pharmaceutical industry owing to the advantages they could offer over traditional small-mol. strategies.  These advantages include the potential to reduce systemic drug exposure, the ability to counteract increased target protein expression that often accompanies inhibition of protein function and the potential ability to target proteins that are not currently therapeutically tractable, such as transcription factors, scaffolding and regulatory proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoj-O7zXGOpXLVg90H21EOLACvtfcHk0lhv5-kG7o4IUw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvFCit7rP&md5=a01ff43e9b696fe4dc03f0815158a98f</span></div><a href="/servlet/linkout?suffix=cit211&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2016.211&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2016.211%26sid%3Dliteratum%253Aachs%26aulast%3DLai%26aufirst%3DA.%2BC.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DInduced%2520Protein%2520Degradation%253A%2520An%2520Emerging%2520Drug%2520Discovery%2520Paradigm%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2017%26volume%3D16%26spage%3D101%26epage%3D114%26doi%3D10.1038%2Fnrd.2016.211" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref212"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref212'); return false;" data-citation="" class="refNumLink">212</a></strong><div class="NLM_citation" id="cit212"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cromm, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Targeted Protein Degradation: From Chemical Biology to Drug Discovery</span>. <i>Cell Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">1181</span>– <span class="NLM_lpage">1190</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2017.05.024</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref212/cit212&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1016%2Fj.chembiol.2017.05.024" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref212/cit212&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=28648379" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref212/cit212&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtVOjtrrO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2017&pages=1181-1190&author=P.+M.+Crommauthor=C.+M.+Crews&title=Targeted+Protein+Degradation%3A+From+Chemical+Biology+to+Drug+Discovery&doi=10.1016%2Fj.chembiol.2017.05.024"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit212R"><div class="casContent"><span class="casTitleNuber">212</span><div class="casTitle"><span class="NLM_cas:atitle">Targeted Protein Degradation: from Chemical Biology to Drug Discovery</span></div><div class="casAuthors">Cromm, Philipp M.; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1181-1190</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  Traditional pharmaceutical drug discovery is almost exclusively focused on directly controlling protein activity to cure diseases.  Modulators of protein activity, esp. inhibitors, are developed and applied at high concn. to achieve maximal effects.  Thereby, reduced bioavailability and off-target effects can hamper compd. efficacy.  Nucleic acid-based strategies that control protein function by affecting expression have emerged as an alternative.  However, metabolic stability and broad bioavailability represent development hurdles that remain to be overcome for these approaches.  More recently, utilizing the cell's own protein destruction machinery for selective degrdn. of essential drivers of human disorders has opened up a new and exciting area of drug discovery.  Small-mol.-induced proteolysis of selected substrates offers the potential of reaching beyond the limitations of the current pharmaceutical paradigm to expand the druggable target space.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrosuhmnPE0RrVg90H21EOLACvtfcHk0ljSPnxTnwo5EQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtVOjtrrO&md5=3c8c6d9488396d5eb08ca51cee20bbf4</span></div><a href="/servlet/linkout?suffix=cit212&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2017.05.024&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2017.05.024%26sid%3Dliteratum%253Aachs%26aulast%3DCromm%26aufirst%3DP.%2BM.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DTargeted%2520Protein%2520Degradation%253A%2520From%2520Chemical%2520Biology%2520to%2520Drug%2520Discovery%26jtitle%3DCell%2520Chem.%2520Biol.%26date%3D2017%26volume%3D24%26spage%3D1181%26epage%3D1190%26doi%3D10.1016%2Fj.chembiol.2017.05.024" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref213"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref213'); return false;" data-citation="" class="refNumLink">213</a></strong><div class="NLM_citation" id="cit213"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lebraud, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heightman, T. D.</span></span> <span> </span><span class="NLM_article-title">Protein Degradation: A Validated Therapeutic Strategy with Exciting Prospects</span>. <i>Essays Biochem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">517</span>– <span class="NLM_lpage">527</span>, <span class="refDoi"> DOI: 10.1042/EBC20170030</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref213/cit213&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1042%2FEBC20170030" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref213/cit213&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=28970340" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref213/cit213&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A280%3ADC%252BC1M%252FksFCjsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2017&pages=517-527&author=H.+Lebraudauthor=T.+D.+Heightman&title=Protein+Degradation%3A+A+Validated+Therapeutic+Strategy+with+Exciting+Prospects&doi=10.1042%2FEBC20170030"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit213R"><div class="casContent"><span class="casTitleNuber">213</span><div class="casTitle"><span class="NLM_cas:atitle">Protein degradation: a validated therapeutic strategy with exciting prospects</span></div><div class="casAuthors">Lebraud Honorine; Heightman Tom D</div><div class="citationInfo"><span class="NLM_cas:title">Essays in biochemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">517-527</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">In a time of unprecedented challenges in developing potent, selective and well-tolerated protein inhibitors as therapeutics, drug hunters are increasingly seeking alternative modalities to modulate pharmacological targets.  Selective inhibitors are achievable for only a fraction of the proteome, and are not guaranteed to elicit the desired response in patients, especially when pursuing targets identified through genetic knockdown.  Targeted protein degradation holds the potential to expand the range of proteins that can be effectively modulated.  Drugs inducing protein degradation through misfolding or by modulating cereblon (CRBN) substrate recognition are already approved for treatment of cancer patients.  The last decade has seen the development of proteolysis targeting chimeras (PROTACs), small molecules that elicit proteasomal degradation by causing protein polyubiquitination.  These have been used to degrade a range of disease-relevant proteins in cells, and some show promising efficacy in preclinical animal models, although their clinical efficacy and tolerability is yet to be proven.  This review introduces current strategies for protein degradation with an emphasis on PROTACs and the role of click chemistry in PROTAC research through the formation of libraries of preclicked PROTACs or in-cell click-formed PROTACs (CLIPTACs).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSAv7HAvl2UWlQKX3ooHxWjfW6udTcc2eYcevcxJ7dR7bntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1M%252FksFCjsw%253D%253D&md5=00c39113eef9f1c1e6382fbea2d5f8d4</span></div><a href="/servlet/linkout?suffix=cit213&amp;dbid=16384&amp;doi=10.1042%2FEBC20170030&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252FEBC20170030%26sid%3Dliteratum%253Aachs%26aulast%3DLebraud%26aufirst%3DH.%26aulast%3DHeightman%26aufirst%3DT.%2BD.%26atitle%3DProtein%2520Degradation%253A%2520A%2520Validated%2520Therapeutic%2520Strategy%2520with%2520Exciting%2520Prospects%26jtitle%3DEssays%2520Biochem.%26date%3D2017%26volume%3D61%26spage%3D517%26epage%3D527%26doi%3D10.1042%2FEBC20170030" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref214"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref214'); return false;" data-citation="" class="refNumLink">214</a></strong><div class="NLM_citation" id="cit214"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Toure, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Small-Molecule PROTACS: New Approaches to Protein Degradation</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">1966</span>– <span class="NLM_lpage">1973</span>, <span class="refDoi"> DOI: 10.1002/anie.201507978</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref214/cit214&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1002%2Fanie.201507978" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref214/cit214&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC28XnsF2ltA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2016&pages=1966-1973&author=M.+Toureauthor=C.+M.+Crews&title=Small-Molecule+PROTACS%3A+New+Approaches+to+Protein+Degradation&doi=10.1002%2Fanie.201507978"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit214R"><div class="casContent"><span class="casTitleNuber">214</span><div class="casTitle"><span class="NLM_cas:atitle">Small-Molecule PROTACS: New Approaches to Protein Degradation</span></div><div class="casAuthors">Toure, Momar; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1966-1973</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A review.  The current inhibitor-based approach to therapeutics has inherent limitations owing to its occupancy-based model: there is a need to maintain high systemic exposure to ensure sufficient in vivo inhibition, high in vivo concns. bring potential for off-target side effects, and there is a need to bind to an active site, thus limiting the drug target space.  As an alternative, induced protein degrdn. lacks these limitations.  Based on an event-driven model, this approach offers a novel catalytic mechanism to irreversibly inhibit protein function by targeting protein destruction through recruitment to the cellular quality control machinery.  Prior protein degrading strategies have lacked therapeutic potential.  However, recent reports of small-mol.-based proteolysis-targeting chimeras (PROTACs) have demonstrated that this technol. can effectively decrease the cellular levels of several protein classes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrdaqlIjCSsgrVg90H21EOLACvtfcHk0ljSPnxTnwo5EQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XnsF2ltA%253D%253D&md5=47838cd15719757317cb8282c26cc59e</span></div><a href="/servlet/linkout?suffix=cit214&amp;dbid=16384&amp;doi=10.1002%2Fanie.201507978&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201507978%26sid%3Dliteratum%253Aachs%26aulast%3DToure%26aufirst%3DM.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DSmall-Molecule%2520PROTACS%253A%2520New%2520Approaches%2520to%2520Protein%2520Degradation%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2016%26volume%3D55%26spage%3D1966%26epage%3D1973%26doi%3D10.1002%2Fanie.201507978" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref215"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref215'); return false;" data-citation="" class="refNumLink">215</a></strong><div class="NLM_citation" id="cit215"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ottis, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Proteolysis-Targeting Chimeras: Induced Protein Degradation as a Therapeutic Strategy</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">892</span>– <span class="NLM_lpage">898</span>, <span class="refDoi"> DOI: 10.1021/acschembio.6b01068</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.6b01068" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref215/cit215&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC2sXjs12ns70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2017&pages=892-898&author=P.+Ottisauthor=C.+M.+Crews&title=Proteolysis-Targeting+Chimeras%3A+Induced+Protein+Degradation+as+a+Therapeutic+Strategy&doi=10.1021%2Facschembio.6b01068"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit215R"><div class="casContent"><span class="casTitleNuber">215</span><div class="casTitle"><span class="NLM_cas:atitle">Proteolysis-Targeting Chimeras: Induced Protein Degradation as a Therapeutic Strategy</span></div><div class="casAuthors">Ottis, Philipp; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">892-898</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Until recently, the only ways to reduce specific protein signaling were to either knock down the target by RNAi or to interfere with the signaling by inhibiting an enzyme or receptor within the signal transduction cascade.  Herein, we review an emerging class of small mol. pharmacol. agents, called PROTACs, that present a novel approach to specifically target proteins and their resp. signaling pathways.  These heterobifunctional mols. utilize endogenous cellular quality control machinery by recruiting it to target proteins in order to induce their degrdn.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGohB6xwkQMTprVg90H21EOLACvtfcHk0ljSPnxTnwo5EQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXjs12ns70%253D&md5=eada521c10faa1e456e0885fd390c554</span></div><a href="/servlet/linkout?suffix=cit215&amp;dbid=16384&amp;doi=10.1021%2Facschembio.6b01068&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.6b01068%26sid%3Dliteratum%253Aachs%26aulast%3DOttis%26aufirst%3DP.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DProteolysis-Targeting%2520Chimeras%253A%2520Induced%2520Protein%2520Degradation%2520as%2520a%2520Therapeutic%2520Strategy%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2017%26volume%3D12%26spage%3D892%26epage%3D898%26doi%3D10.1021%2Facschembio.6b01068" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref216"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref216'); return false;" data-citation="" class="refNumLink">216</a></strong><div class="NLM_citation" id="cit216"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Crew, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raina, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qian, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vigil, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Serebrenik, Y. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamman, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morgan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferraro, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neklesa, T. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winkler, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coleman, K. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Identification and Characterization of Von Hippel-Lindau-Recruiting Proteolysis Targeting Chimeras (PROTACs) of TANK-Binding Kinase 1</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">583</span>– <span class="NLM_lpage">598</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b00635</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b00635" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref216/cit216&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtFelu7rK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=583-598&author=A.+P.+Crewauthor=K.+Rainaauthor=H.+Dongauthor=Y.+Qianauthor=J.+Wangauthor=D.+Vigilauthor=Y.+V.+Serebrenikauthor=B.+D.+Hammanauthor=A.+Morganauthor=C.+Ferraroauthor=K.+Siuauthor=T.+K.+Neklesaauthor=J.+D.+Winklerauthor=K.+G.+Colemanauthor=C.+M.+Crews&title=Identification+and+Characterization+of+Von+Hippel-Lindau-Recruiting+Proteolysis+Targeting+Chimeras+%28PROTACs%29+of+TANK-Binding+Kinase+1&doi=10.1021%2Facs.jmedchem.7b00635"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit216R"><div class="casContent"><span class="casTitleNuber">216</span><div class="casTitle"><span class="NLM_cas:atitle">Identification and Characterization of Von Hippel-Lindau-Recruiting Proteolysis Targeting Chimeras (PROTACs) of TANK-Binding Kinase 1</span></div><div class="casAuthors">Crew, Andrew P.; Raina, Kanak; Dong, Hanqing; Qian, Yimin; Wang, Jing; Vigil, Dominico; Serebrenik, Yevgeniy V.; Hamman, Brian D.; Morgan, Alicia; Ferraro, Caterina; Siu, Kam; Neklesa, Taavi K.; Winkler, James D.; Coleman, Kevin G.; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">583-598</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Proteolysis Targeting Chimeras (PROTACs) are bifunctional mols. that recruit an E3 ligase to a target protein to facilitate ubiquitination and subsequent degrdn. of that protein.  While the field of targeted degraders is still relatively young, the potential for this modality to become a differentiated and therapeutic reality is strong, such that both academic and pharmaceutical institutions are now entering this interesting area of research.  In this paper, the authors describe a broadly applicable process for identifying degrader hits based around the serine/threonine kinase TANK-binding kinase 1 (TBK1), and have generalized the key structural elements assocd. with degrdn. activities.  Compd. I is a potent hit (TBK1 DC50 = 12 nM, Dmax = 96%) with excellent selectivity against a related kinase IKKε, which was further used as a chem. tool to assess TBK1 as a target in mutant K-Ras cancer cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoIJWCnFncQRLVg90H21EOLACvtfcHk0ljL3MPZuDSbAg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtFelu7rK&md5=41c3627e05ca31aee3ba85815685fcb2</span></div><a href="/servlet/linkout?suffix=cit216&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b00635&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b00635%26sid%3Dliteratum%253Aachs%26aulast%3DCrew%26aufirst%3DA.%2BP.%26aulast%3DRaina%26aufirst%3DK.%26aulast%3DDong%26aufirst%3DH.%26aulast%3DQian%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DVigil%26aufirst%3DD.%26aulast%3DSerebrenik%26aufirst%3DY.%2BV.%26aulast%3DHamman%26aufirst%3DB.%2BD.%26aulast%3DMorgan%26aufirst%3DA.%26aulast%3DFerraro%26aufirst%3DC.%26aulast%3DSiu%26aufirst%3DK.%26aulast%3DNeklesa%26aufirst%3DT.%2BK.%26aulast%3DWinkler%26aufirst%3DJ.%2BD.%26aulast%3DColeman%26aufirst%3DK.%2BG.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DIdentification%2520and%2520Characterization%2520of%2520Von%2520Hippel-Lindau-Recruiting%2520Proteolysis%2520Targeting%2520Chimeras%2520%2528PROTACs%2529%2520of%2520TANK-Binding%2520Kinase%25201%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D583%26epage%3D598%26doi%3D10.1021%2Facs.jmedchem.7b00635" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref217"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref217'); return false;" data-citation="" class="refNumLink">217</a></strong><div class="NLM_citation" id="cit217"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yin, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qin, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Z.</span></span> <span> </span><span class="NLM_article-title">Development of Stabilized Peptide-Based PROTACs against Estrogen Receptor Α</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">628</span>– <span class="NLM_lpage">635</span>, <span class="refDoi"> DOI: 10.1021/acschembio.7b00985</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.7b00985" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref217/cit217&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC2sXitVaksr3M" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2018&pages=628-635&author=Y.+Jiangauthor=Q.+Dengauthor=H.+Zhaoauthor=M.+Xieauthor=L.+Chenauthor=F.+Yinauthor=X.+Qinauthor=W.+Zhengauthor=Y.+Zhaoauthor=Z.+Li&title=Development+of+Stabilized+Peptide-Based+PROTACs+against+Estrogen+Receptor+%CE%91&doi=10.1021%2Facschembio.7b00985"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit217R"><div class="casContent"><span class="casTitleNuber">217</span><div class="casTitle"><span class="NLM_cas:atitle">Development of Stabilized Peptide-Based PROTACs against Estrogen Receptor α</span></div><div class="casAuthors">Jiang, Yanhong; Deng, Qiwen; Zhao, Hui; Xie, Mingsheng; Chen, Longjian; Yin, Feng; Qin, Xuan; Zheng, Weihao; Zhao, Yongjuan; Li, Zigang</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">628-635</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Peptide modulators targeting protein-protein interactions (PPIs) exhibit greater potential than small-mol. drugs in several important aspects including facile modification and relative large contact surface area.  Stabilized peptides constructed by variable chem. methods exhibit improved peptide stability and cell permeability compared to that of the linears.  Herein, we designed a stabilized peptide-based proteolysis-targeting chimera (PROTAC) targeting estrogen receptor α (ERα) by tethering an N-terminal aspartic acid cross-linked stabilized peptide ERα modulator (TD-PERM) with a pentapeptide that binds the Von Hippel-Lindau (VHL) E3 ubiquitin ligase complex.  The resulting heterobifunctional peptide (TD-PROTAC) selectively recruits ERα to the VHL E3 ligase complex, leading to the degrdn. of ERα in a proteasome-dependent manner.  Compared with the control peptides, TD-PROTAC shows significantly enhanced activities in reducing the transcription of the ERα-downstream genes and inhibiting the proliferation of ERα-pos. breast cancer cells.  In addn., in vivo expts. indicate that TD-PROTAC leads to tumor regression in the MCF-7 mouse xenograft model.  This work is a successful attempt to construct PROTACs based on cell-permeable stabilized peptides, which significantly broadens the chem. space of PROTACs and stabilized peptides.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGovSDWxTR3LbLVg90H21EOLACvtfcHk0ljL3MPZuDSbAg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXitVaksr3M&md5=abcd6f72223751db6d96f8027cbb8c23</span></div><a href="/servlet/linkout?suffix=cit217&amp;dbid=16384&amp;doi=10.1021%2Facschembio.7b00985&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.7b00985%26sid%3Dliteratum%253Aachs%26aulast%3DJiang%26aufirst%3DY.%26aulast%3DDeng%26aufirst%3DQ.%26aulast%3DZhao%26aufirst%3DH.%26aulast%3DXie%26aufirst%3DM.%26aulast%3DChen%26aufirst%3DL.%26aulast%3DYin%26aufirst%3DF.%26aulast%3DQin%26aufirst%3DX.%26aulast%3DZheng%26aufirst%3DW.%26aulast%3DZhao%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DZ.%26atitle%3DDevelopment%2520of%2520Stabilized%2520Peptide-Based%2520PROTACs%2520against%2520Estrogen%2520Receptor%2520%25CE%2591%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2018%26volume%3D13%26spage%3D628%26epage%3D635%26doi%3D10.1021%2Facschembio.7b00985" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref218"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref218'); return false;" data-citation="" class="refNumLink">218</a></strong><div class="NLM_citation" id="cit218"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiao, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ji, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tao, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">You, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, Z.</span></span> <span> </span><span class="NLM_article-title">Discovery of a Keap1-Dependent Peptide PROTAC to Knockdown Tau by Ubiquitination-Proteasome Degradation Pathway</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>146</i></span>,  <span class="NLM_fpage">251</span>– <span class="NLM_lpage">259</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2018.01.063</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref218/cit218&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1016%2Fj.ejmech.2018.01.063" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref218/cit218&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=29407955" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref218/cit218&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC1cXisFehsbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=146&publication_year=2018&pages=251-259&author=M.+Luauthor=T.+Liuauthor=Q.+Jiaoauthor=J.+Jiauthor=M.+Taoauthor=Y.+Liuauthor=Q.+Youauthor=Z.+Jiang&title=Discovery+of+a+Keap1-Dependent+Peptide+PROTAC+to+Knockdown+Tau+by+Ubiquitination-Proteasome+Degradation+Pathway&doi=10.1016%2Fj.ejmech.2018.01.063"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit218R"><div class="casContent"><span class="casTitleNuber">218</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a Keap1-dependent peptide PROTAC to knockdown Tau by ubiquitination-proteasome degradation pathway</span></div><div class="casAuthors">Lu, Mengchen; Liu, Tian; Jiao, Qiong; Ji, Jianai; Tao, Mengmin; Liu, Yijun; You, Qidong; Jiang, Zhengyu</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">146</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">251-259</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">Induced protein degrdn. by PROTACs has emerged as a promising strategy to target nonenzymic proteins inside the cell.  The aim of this study was to identify Keap1, a substrate adaptor protein for ubiquitin E3 ligase involved in oxidative stress regulation, as a novel candidate for PROTACs that can be applied in the degrdn. of the nonenzymic protein Tau.  A peptide PROTAC by recruiting Keap1-Cul3 ubiquitin E3 ligase was developed and applied in the degrdn. of intracellular Tau.  Peptide 1 showed strong in vitro binding with Keap1 and Tau.  With proper cell permeability, peptide 1 was found to colocalize with cellular Keap1 and resulted in the coimmunopptn. of Tau and Keap1.  The results of flow cytometry and western blotting assays showed that peptide 1 can downregulate the intracellular Tau level in both time- and concn.-dependent manner.  The application of Keap1 siRNA silencing and the proteasome inhibitor MG132 confirmed that peptide 1 could promote the Keap1-dependent poly-ubiquitination and proteasome-dependent degrdn. of Tau.  The results suggested that using PROTACs to recruit Keap1 to induce the degrdn. of Tau may show promising character in the treatment of neurodegenerative disease.  In addn., our research demonstrated that Keap1 should be a promising E3 ligase adaptor to be used in the design of novel PROTACs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp9KC7pnPqlL7Vg90H21EOLACvtfcHk0ljL3MPZuDSbAg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXisFehsbY%253D&md5=165e6f510e4e2024aca6c4135d2840b2</span></div><a href="/servlet/linkout?suffix=cit218&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2018.01.063&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2018.01.063%26sid%3Dliteratum%253Aachs%26aulast%3DLu%26aufirst%3DM.%26aulast%3DLiu%26aufirst%3DT.%26aulast%3DJiao%26aufirst%3DQ.%26aulast%3DJi%26aufirst%3DJ.%26aulast%3DTao%26aufirst%3DM.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DYou%26aufirst%3DQ.%26aulast%3DJiang%26aufirst%3DZ.%26atitle%3DDiscovery%2520of%2520a%2520Keap1-Dependent%2520Peptide%2520PROTAC%2520to%2520Knockdown%2520Tau%2520by%2520Ubiquitination-Proteasome%2520Degradation%2520Pathway%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2018%26volume%3D146%26spage%3D251%26epage%3D259%26doi%3D10.1016%2Fj.ejmech.2018.01.063" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref219"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref219'); return false;" data-citation="" class="refNumLink">219</a></strong><div class="NLM_citation" id="cit219"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Woelcke, J.</span></span> <span> </span><span class="NLM_article-title">Advancement of Lead Discovery in the Protease Platform</span>, <span class="NLM_year" style="font-weight: bold;">2013</span>, Drug Discovery Leaders Summit, Geneva, Switzerland.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2013&author=J.+Woelcke&title=Advancement+of+Lead+Discovery+in+the+Protease+Platform"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit219&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DWoelcke%26aufirst%3DJ.%26atitle%3DAdvancement%2520of%2520Lead%2520Discovery%2520in%2520the%2520Protease%2520Platform%26date%3D2013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref220"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref220'); return false;" data-citation="" class="refNumLink">220</a></strong><div class="NLM_citation" id="cit220"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Oueis, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nardone, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jaspars, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Westwood, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Naismith, J. H.</span></span> <span> </span><span class="NLM_article-title">Synthesis of Hybrid Cyclopeptides through Enzymatic Macrocyclization</span>. <i>ChemistryOpen</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">11</span>– <span class="NLM_lpage">14</span>, <span class="refDoi"> DOI: 10.1002/open.201600134</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref220/cit220&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1002%2Fopen.201600134" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref220/cit220&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=28168143" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref220/cit220&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC28XitVyjtLjM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2017&pages=11-14&author=E.+Oueisauthor=B.+Nardoneauthor=M.+Jasparsauthor=N.+J.+Westwoodauthor=J.+H.+Naismith&title=Synthesis+of+Hybrid+Cyclopeptides+through+Enzymatic+Macrocyclization&doi=10.1002%2Fopen.201600134"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit220R"><div class="casContent"><span class="casTitleNuber">220</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis of Hybrid Cyclopeptides through Enzymatic Macrocyclization</span></div><div class="casAuthors">Oueis, Emilia; Nardone, Brunello; Jaspars, Marcel; Westwood, Nicholas J.; Naismith, James H.</div><div class="citationInfo"><span class="NLM_cas:title">ChemistryOpen</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">11-14</span>CODEN:
                <span class="NLM_cas:coden">CHOPCK</span>;
        ISSN:<span class="NLM_cas:issn">2191-1363</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Natural products comprise a diverse array of mols., many of which are biol. active.  Most natural products are derived from combinations of polyketides, peptides, sugars, and fatty-acid building blocks.  Peptidic macrocycles have attracted attention as potential therapeutics possessing cell permeability, stability, and easy-to-control variability.  Here, we show that enzymes from the patellamide biosynthetic pathway can be harnessed to make macrocycles that are hybrids of amino acids and a variety of manmade chem. building blocks, including aryl rings, polyethers, and alkyl chains.  We have made macrocycles with only three amino acids, one of which can be converted to a thiazoline or a thiazole ring.  We report the synthesis of 18 peptide hybrid macrocycles, nine of which have been isolated and fully characterized.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpMkscQjq-H9rVg90H21EOLACvtfcHk0lg7trCumyh_Cg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitVyjtLjM&md5=180f31681c98e3739b8551392d0ddde9</span></div><a href="/servlet/linkout?suffix=cit220&amp;dbid=16384&amp;doi=10.1002%2Fopen.201600134&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fopen.201600134%26sid%3Dliteratum%253Aachs%26aulast%3DOueis%26aufirst%3DE.%26aulast%3DNardone%26aufirst%3DB.%26aulast%3DJaspars%26aufirst%3DM.%26aulast%3DWestwood%26aufirst%3DN.%2BJ.%26aulast%3DNaismith%26aufirst%3DJ.%2BH.%26atitle%3DSynthesis%2520of%2520Hybrid%2520Cyclopeptides%2520through%2520Enzymatic%2520Macrocyclization%26jtitle%3DChemistryOpen%26date%3D2017%26volume%3D6%26spage%3D11%26epage%3D14%26doi%3D10.1002%2Fopen.201600134" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref221"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref221'); return false;" data-citation="" class="refNumLink">221</a></strong><div class="NLM_citation" id="cit221"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Klein, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heal, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamilton, W. D. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boussemghoune, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tange, T. Ø.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Delegrange, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jaeschke, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hatsch, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heim, J.</span></span> <span> </span><span class="NLM_article-title">Yeast Synthetic Biology Platform Generates Novel Chemical Structures as Scaffolds for Drug Discovery</span>. <i>ACS Synth. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">314</span>– <span class="NLM_lpage">323</span>, <span class="refDoi"> DOI: 10.1021/sb400177x</span> </span><div class="citationLinks">[<a href="/doi/10.1021/sb400177x" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref221/cit221&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC2cXotVGlsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2014&pages=314-323&author=J.+Kleinauthor=J.+R.+Healauthor=W.+D.+O.+Hamiltonauthor=T.+Boussemghouneauthor=T.+%C3%98.+Tangeauthor=F.+Delegrangeauthor=G.+Jaeschkeauthor=A.+Hatschauthor=J.+Heim&title=Yeast+Synthetic+Biology+Platform+Generates+Novel+Chemical+Structures+as+Scaffolds+for+Drug+Discovery&doi=10.1021%2Fsb400177x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit221R"><div class="casContent"><span class="casTitleNuber">221</span><div class="casTitle"><span class="NLM_cas:atitle">Yeast Synthetic Biology Platform Generates Novel Chemical Structures as Scaffolds for Drug Discovery</span></div><div class="casAuthors">Klein, Jens; Heal, Jonathan R.; Hamilton, William D. O.; Boussemghoune, Thiamo; Tange, Thomas Oestergaard; Delegrange, Fanny; Jaeschke, Georg; Hatsch, Anaelle; Heim, Jutta</div><div class="citationInfo"><span class="NLM_cas:title">ACS Synthetic Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">314-323</span>CODEN:
                <span class="NLM_cas:coden">ASBCD6</span>;
        ISSN:<span class="NLM_cas:issn">2161-5063</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Synthetic biol. has been heralded as a new bioengineering platform for the prodn. of bulk and specialty chems., drugs, and fuels.  Here, the authors report for the first time a series of 74 novel compds. produced using a combinatorial genetics approach in baker's yeast.  Based on the concept of "coevolution" with target proteins in an intracellular primary survival assay, the identified, mostly scaffold-sized (200-350 MW) compds., which displayed excellent biol. activity, can be considered as prevalidated hits.  Of the mols. found, >75% have not been described previously; 20% of the compds. exhibit novel scaffolds.  Their structural and physicochem. properties comply with established rules of drug- and fragment-likeness and exhibit increased structural complexities compared to synthetically produced fragments.  In summary, the synthetic biol. approach described here represents a completely new, complementary strategy for hit and early lead identification that can be easily integrated into the existing drug discovery process.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrQ8fJGPPKWe7Vg90H21EOLACvtfcHk0lg7trCumyh_Cg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXotVGlsQ%253D%253D&md5=08b0ac399343c1230ca71808a53a6d30</span></div><a href="/servlet/linkout?suffix=cit221&amp;dbid=16384&amp;doi=10.1021%2Fsb400177x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fsb400177x%26sid%3Dliteratum%253Aachs%26aulast%3DKlein%26aufirst%3DJ.%26aulast%3DHeal%26aufirst%3DJ.%2BR.%26aulast%3DHamilton%26aufirst%3DW.%2BD.%2BO.%26aulast%3DBoussemghoune%26aufirst%3DT.%26aulast%3DTange%26aufirst%3DT.%2B%25C3%2598.%26aulast%3DDelegrange%26aufirst%3DF.%26aulast%3DJaeschke%26aufirst%3DG.%26aulast%3DHatsch%26aufirst%3DA.%26aulast%3DHeim%26aufirst%3DJ.%26atitle%3DYeast%2520Synthetic%2520Biology%2520Platform%2520Generates%2520Novel%2520Chemical%2520Structures%2520as%2520Scaffolds%2520for%2520Drug%2520Discovery%26jtitle%3DACS%2520Synth.%2520Biol.%26date%3D2014%26volume%3D3%26spage%3D314%26epage%3D323%26doi%3D10.1021%2Fsb400177x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref222"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref222'); return false;" data-citation="" class="refNumLink">222</a></strong><div class="NLM_citation" id="cit222"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lennard, K.
R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tavassoli, A.</span></span> <span> </span><span class="NLM_article-title">Peptides Come Round: Using SICLOPPS Libraries for Early Stage Drug Discovery</span>. <i>Chem. - Eur. J.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">10608</span>– <span class="NLM_lpage">10614</span>, <span class="refDoi"> DOI: 10.1002/chem.201403117</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref222/cit222&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1002%2Fchem.201403117" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref222/cit222&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=25043886" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref222/cit222&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtFCgtL%252FF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2014&pages=10608-10614&author=K.%0AR.+Lennardauthor=A.+Tavassoli&title=Peptides+Come+Round%3A+Using+SICLOPPS+Libraries+for+Early+Stage+Drug+Discovery&doi=10.1002%2Fchem.201403117"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit222R"><div class="casContent"><span class="casTitleNuber">222</span><div class="casTitle"><span class="NLM_cas:atitle">Peptides Come Round: Using SICLOPPS Libraries for Early Stage Drug Discovery</span></div><div class="casAuthors">Lennard, Katherine R.; Tavassoli, Ali</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry - A European Journal</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">34</span>),
    <span class="NLM_cas:pages">10608-10614</span>CODEN:
                <span class="NLM_cas:coden">CEUJED</span>;
        ISSN:<span class="NLM_cas:issn">0947-6539</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A review.  Cyclic peptides are an emerging class of mol. therapeutics that are increasingly viewed as ideal backbones for modulation of protein-protein interactions.  A split-intein based method, termed SICLOPPS, enables the rapid generation of genetically encoded cyclic peptide libraries of around a hundred million members.  Here we review recent approaches using SICLOPPS for the discovery of bioactive compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGre3O4emKUgCrVg90H21EOLACvtfcHk0lg7trCumyh_Cg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtFCgtL%252FF&md5=c9c4bfb37d3f3e64eadc300c33221469</span></div><a href="/servlet/linkout?suffix=cit222&amp;dbid=16384&amp;doi=10.1002%2Fchem.201403117&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fchem.201403117%26sid%3Dliteratum%253Aachs%26aulast%3DLennard%26aufirst%3DK.%2BR.%26aulast%3DTavassoli%26aufirst%3DA.%26atitle%3DPeptides%2520Come%2520Round%253A%2520Using%2520SICLOPPS%2520Libraries%2520for%2520Early%2520Stage%2520Drug%2520Discovery%26jtitle%3DChem.%2520-%2520Eur.%2520J.%26date%3D2014%26volume%3D20%26spage%3D10608%26epage%3D10614%26doi%3D10.1002%2Fchem.201403117" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref223"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref223'); return false;" data-citation="" class="refNumLink">223</a></strong><div class="NLM_citation" id="cit223"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Young, T. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, D. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ahmad, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Louis, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Benkovic, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schultz, P. G.</span></span> <span> </span><span class="NLM_article-title">Evolution of Cyclic Peptide Protease Inhibitors</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>108</i></span>,  <span class="NLM_fpage">11052</span>– <span class="NLM_lpage">11056</span>, <span class="refDoi"> DOI: 10.1073/pnas.1108045108</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref223/cit223&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1073%2Fpnas.1108045108" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref223/cit223&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=21690365" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref223/cit223&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC3MXptV2htL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=108&publication_year=2011&pages=11052-11056&author=T.+S.+Youngauthor=D.+D.+Youngauthor=I.+Ahmadauthor=J.+M.+Louisauthor=S.+J.+Benkovicauthor=P.+G.+Schultz&title=Evolution+of+Cyclic+Peptide+Protease+Inhibitors&doi=10.1073%2Fpnas.1108045108"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit223R"><div class="casContent"><span class="casTitleNuber">223</span><div class="casTitle"><span class="NLM_cas:atitle">Evolution of cyclic peptide protease inhibitors</span></div><div class="casAuthors">Young, Travis S.; Young, Douglas D.; Ahmad, Insha; Louis, John M.; Benkovic, Stephen J.; Schultz, Peter G.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">108</span>
        (<span class="NLM_cas:issue">27</span>),
    <span class="NLM_cas:pages">11052-11056, S11052/1-S11052/14</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">We report a bacterial system for the evolution of cyclic peptides that makes use of an expanded set of amino acid building blocks.  Orthogonal aminoacyl-tRNA synthetase/tRNACUA, together with a split intein system were used to biosynthesize a library of ribosomal peptides contg. amino acids with unique structures and reactivities.  This peptide library was subsequently used to evolve an inhibitor of HIV protease using a selection based on cellular viability.  Two of three cyclic peptides isolated after two rounds of selection contained the keto amino acid p-benzoylphenylalanine (pBzF).  The most potent peptide (G12: GIXVSL; X = pBzF) inhibited HIV protease through the formation of a covalent Schiff base adduct of the pBzF residue with the ε-amino group of Lys 14 on the protease.  This result suggests that an expanded genetic code can confer an evolutionary advantage in response to selective pressure.  Moreover, the combination of natural evolutionary processes with chem. biased building blocks provides another strategy for the generation of biol. active peptides using microbial systems.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGreESwOTK5p9rVg90H21EOLACvtfcHk0lhzx60HqBnr2g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXptV2htL0%253D&md5=b063a5c486cfa6a68801ccbb6a68237d</span></div><a href="/servlet/linkout?suffix=cit223&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1108045108&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1108045108%26sid%3Dliteratum%253Aachs%26aulast%3DYoung%26aufirst%3DT.%2BS.%26aulast%3DYoung%26aufirst%3DD.%2BD.%26aulast%3DAhmad%26aufirst%3DI.%26aulast%3DLouis%26aufirst%3DJ.%2BM.%26aulast%3DBenkovic%26aufirst%3DS.%2BJ.%26aulast%3DSchultz%26aufirst%3DP.%2BG.%26atitle%3DEvolution%2520of%2520Cyclic%2520Peptide%2520Protease%2520Inhibitors%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2011%26volume%3D108%26spage%3D11052%26epage%3D11056%26doi%3D10.1073%2Fpnas.1108045108" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref224"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref224'); return false;" data-citation="" class="refNumLink">224</a></strong><div class="NLM_citation" id="cit224"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Miranda, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nordgren, I. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Male, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lawrence, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoakwie, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cuda, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Court, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fox, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Townsend, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Packham, G. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eccles, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tavassoli, A.</span></span> <span> </span><span class="NLM_article-title">A Cyclic Peptide Inhibitor of HIF-1 Heterodimerization That Inhibits Hypoxia Signaling in Cancer Cells</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>135</i></span>,  <span class="NLM_fpage">10418</span>– <span class="NLM_lpage">10425</span>, <span class="refDoi"> DOI: 10.1021/ja402993u</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja402993u" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref224/cit224&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC3sXpvFaksro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=135&publication_year=2013&pages=10418-10425&author=E.+Mirandaauthor=I.+K.+Nordgrenauthor=A.+L.+Maleauthor=C.+E.+Lawrenceauthor=F.+Hoakwieauthor=F.+Cudaauthor=W.+Courtauthor=K.+R.+Foxauthor=P.+A.+Townsendauthor=G.+K.+Packhamauthor=S.+A.+Ecclesauthor=A.+Tavassoli&title=A+Cyclic+Peptide+Inhibitor+of+HIF-1+Heterodimerization+That+Inhibits+Hypoxia+Signaling+in+Cancer+Cells&doi=10.1021%2Fja402993u"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit224R"><div class="casContent"><span class="casTitleNuber">224</span><div class="casTitle"><span class="NLM_cas:atitle">A Cyclic Peptide Inhibitor of HIF-1 Heterodimerization That Inhibits Hypoxia Signaling in Cancer Cells</span></div><div class="casAuthors">Miranda, Elena; Nordgren, Ida K.; Male, Abigail L.; Lawrence, Charlotte E.; Hoakwie, Franciane; Cuda, Francesco; Court, William; Fox, Keith R.; Townsend, Paul A.; Packham, Graham K.; Eccles, Suzanne A.; Tavassoli, Ali</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">135</span>
        (<span class="NLM_cas:issue">28</span>),
    <span class="NLM_cas:pages">10418-10425</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Hypoxia inducible factor-1 (HIF-1) is a heterodimeric transcription factor that acts as the master regulator of cellular response to reduced oxygen levels, thus playing a key role in the adaptation, survival, and progression of tumors.  Here we report cyclo-CLLFVY, identified from a library of 3.2 million cyclic hexapeptides using a genetically encoded high-throughput screening platform, as an inhibitor of the HIF-1α/HIF-1β protein-protein interaction in vitro and in cells.  The identified compd. inhibits HIF-1 dimerization and transcription activity by binding to the PAS-B domain of HIF-1α, reducing HIF-1-mediated hypoxia response signaling in a variety of cell lines, without affecting the function of the closely related HIF-2 isoform.  The reported cyclic peptide demonstrates the utility of our high-throughput screening platform for the identification of protein-protein interaction inhibitors, and forms the starting point for the development of HIF-1 targeted cancer therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGodwKA_rNEr2LVg90H21EOLACvtfcHk0lhzx60HqBnr2g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXpvFaksro%253D&md5=3f44683ebfa19a9dbb6f505d50bb6032</span></div><a href="/servlet/linkout?suffix=cit224&amp;dbid=16384&amp;doi=10.1021%2Fja402993u&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja402993u%26sid%3Dliteratum%253Aachs%26aulast%3DMiranda%26aufirst%3DE.%26aulast%3DNordgren%26aufirst%3DI.%2BK.%26aulast%3DMale%26aufirst%3DA.%2BL.%26aulast%3DLawrence%26aufirst%3DC.%2BE.%26aulast%3DHoakwie%26aufirst%3DF.%26aulast%3DCuda%26aufirst%3DF.%26aulast%3DCourt%26aufirst%3DW.%26aulast%3DFox%26aufirst%3DK.%2BR.%26aulast%3DTownsend%26aufirst%3DP.%2BA.%26aulast%3DPackham%26aufirst%3DG.%2BK.%26aulast%3DEccles%26aufirst%3DS.%2BA.%26aulast%3DTavassoli%26aufirst%3DA.%26atitle%3DA%2520Cyclic%2520Peptide%2520Inhibitor%2520of%2520HIF-1%2520Heterodimerization%2520That%2520Inhibits%2520Hypoxia%2520Signaling%2520in%2520Cancer%2520Cells%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2013%26volume%3D135%26spage%3D10418%26epage%3D10425%26doi%3D10.1021%2Fja402993u" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref225"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref225'); return false;" data-citation="" class="refNumLink">225</a></strong><div class="NLM_citation" id="cit225"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Davis, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plowright, A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valeur, E.</span></span> <span> </span><span class="NLM_article-title">Directing Evolution: The next Revolution in Drug Discovery?</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">681</span>– <span class="NLM_lpage">698</span>, <span class="refDoi"> DOI: 10.1038/nrd.2017.146</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref225/cit225&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1038%2Fnrd.2017.146" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref225/cit225&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=28935911" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref225/cit225&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsFeqs7%252FP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2017&pages=681-698&author=A.+M.+Davisauthor=A.+T.+Plowrightauthor=E.+Valeur&title=Directing+Evolution%3A+The+next+Revolution+in+Drug+Discovery%3F&doi=10.1038%2Fnrd.2017.146"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit225R"><div class="casContent"><span class="casTitleNuber">225</span><div class="casTitle"><span class="NLM_cas:atitle">Directing evolution: the next revolution in drug discovery?</span></div><div class="casAuthors">Davis, Andrew M.; Plowright, Alleyn T.; Valeur, Eric</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">681-698</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">The strong biol. rationale to pursue challenging drug targets such as protein-protein interactions has stimulated the development of novel screening strategies, such as DNA-encoded libraries, to allow broader areas of chem. space to be searched.  There has also been renewed interest in screening natural products, which are the result of evolutionary selection for a function, such as interference with a key signalling pathway of a competing organism.  However, recent advances in several areas, such as understanding of the biosynthetic pathways for natural products, synthetic biol. and the development of biosensors to detect target mols., are now providing new opportunities to directly harness evolutionary pressure to identify and optimize compds. with desired bioactivities.  Here, we describe innovations in the key components of such strategies and highlight pioneering examples that indicate the potential of the directed-evolution concept.  We also discuss the scientific gaps and challenges that remain to be addressed to realize this potential more broadly in drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpy0QYUc4kt_bVg90H21EOLACvtfcHk0lhzx60HqBnr2g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsFeqs7%252FP&md5=3c9a88c827e47cb67751d9bc1607056b</span></div><a href="/servlet/linkout?suffix=cit225&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2017.146&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2017.146%26sid%3Dliteratum%253Aachs%26aulast%3DDavis%26aufirst%3DA.%2BM.%26aulast%3DPlowright%26aufirst%3DA.%2BT.%26aulast%3DValeur%26aufirst%3DE.%26atitle%3DDirecting%2520Evolution%253A%2520The%2520next%2520Revolution%2520in%2520Drug%2520Discovery%253F%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2017%26volume%3D16%26spage%3D681%26epage%3D698%26doi%3D10.1038%2Fnrd.2017.146" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref226"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref226'); return false;" data-citation="" class="refNumLink">226</a></strong><div class="NLM_citation" id="cit226"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Badran, A. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guzov, V. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huai, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kemp, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vishwanath, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kain, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nance, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evdokimov, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moshiri, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turner, K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Malvar, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, D. R.</span></span> <span> </span><span class="NLM_article-title">Continuous Evolution of <i>Bacillus Thuringiensis</i> Toxins Overcomes Insect Resistance</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>533</i></span>,  <span class="NLM_fpage">58</span>– <span class="NLM_lpage">63</span>, <span class="refDoi"> DOI: 10.1038/nature17938</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref226/cit226&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1038%2Fnature17938" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref226/cit226&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=27120167" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref226/cit226&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC28XmvVGnt7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=533&publication_year=2016&pages=58-63&author=A.+H.+Badranauthor=V.+M.+Guzovauthor=Q.+Huaiauthor=M.+M.+Kempauthor=P.+Vishwanathauthor=W.+Kainauthor=A.+M.+Nanceauthor=A.+Evdokimovauthor=F.+Moshiriauthor=K.+H.+Turnerauthor=P.+Wangauthor=T.+Malvarauthor=D.+R.+Liu&title=Continuous+Evolution+of+Bacillus+Thuringiensis+Toxins+Overcomes+Insect+Resistance&doi=10.1038%2Fnature17938"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit226R"><div class="casContent"><span class="casTitleNuber">226</span><div class="casTitle"><span class="NLM_cas:atitle">Continuous evolution of Bacillus thuringiensis toxins overcomes insect resistance</span></div><div class="casAuthors">Badran, Ahmed H.; Guzov, Victor M.; Huai, Qing; Kemp, Melissa M.; Vishwanath, Prashanth; Kain, Wendy; Nance, Autumn M.; Evdokimov, Artem; Moshiri, Farhad; Turner, Keith H.; Wang, Ping; Malvar, Thomas; Liu, David R.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">533</span>
        (<span class="NLM_cas:issue">7601</span>),
    <span class="NLM_cas:pages">58-63</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The Bacillus thuringiensis δ-endotoxins (Bt toxins) are widely used insecticidal proteins in engineered crops that provide agricultural, economic, and environmental benefits.  The development of insect resistance to Bt toxins endangers their long-term effectiveness.  Here we have developed a phage-assisted continuous evolution selection that rapidly evolves high-affinity protein-protein interactions, and applied this system to evolve variants of the Bt toxin Cry1Ac that bind a cadherin-like receptor from the insect pest Trichoplusia ni (TnCAD) that is not natively bound by wild-type Cry1Ac.  The resulting evolved Cry1Ac variants bind TnCAD with high affinity (dissocn. const. Kd = 11-41 nM), kill TnCAD-expressing insect cells that are not susceptible to wild-type Cry1Ac, and kill Cry1Ac-resistant T. ni insects up to 335-fold more potently than wild-type Cry1Ac.  Our findings establish that the evolution of Bt toxins with novel insect cell receptor affinity can overcome insect Bt toxin resistance and confer lethality approaching that of the wild-type Bt toxin against non-resistant insects.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo2tXjekTvoL7Vg90H21EOLACvtfcHk0lhli4tukQwMVg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XmvVGnt7Y%253D&md5=1f0de0f8d608073ccf34dd33a1d50282</span></div><a href="/servlet/linkout?suffix=cit226&amp;dbid=16384&amp;doi=10.1038%2Fnature17938&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature17938%26sid%3Dliteratum%253Aachs%26aulast%3DBadran%26aufirst%3DA.%2BH.%26aulast%3DGuzov%26aufirst%3DV.%2BM.%26aulast%3DHuai%26aufirst%3DQ.%26aulast%3DKemp%26aufirst%3DM.%2BM.%26aulast%3DVishwanath%26aufirst%3DP.%26aulast%3DKain%26aufirst%3DW.%26aulast%3DNance%26aufirst%3DA.%2BM.%26aulast%3DEvdokimov%26aufirst%3DA.%26aulast%3DMoshiri%26aufirst%3DF.%26aulast%3DTurner%26aufirst%3DK.%2BH.%26aulast%3DWang%26aufirst%3DP.%26aulast%3DMalvar%26aufirst%3DT.%26aulast%3DLiu%26aufirst%3DD.%2BR.%26atitle%3DContinuous%2520Evolution%2520of%2520Bacillus%2520Thuringiensis%2520Toxins%2520Overcomes%2520Insect%2520Resistance%26jtitle%3DNature%26date%3D2016%26volume%3D533%26spage%3D58%26epage%3D63%26doi%3D10.1038%2Fnature17938" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref227"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref227'); return false;" data-citation="" class="refNumLink">227</a></strong><div class="NLM_citation" id="cit227"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chi, C.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ahmed, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dereli-Korkut, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span> <span> </span><span class="NLM_article-title">Microfluidic Cell Chips for High-Throughput Drug Screening</span>. <i>Bioanalysis</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">921</span>– <span class="NLM_lpage">937</span>, <span class="refDoi"> DOI: 10.4155/bio-2016-0028</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref227/cit227&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.4155%2Fbio-2016-0028" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2016&pages=921-937&author=C.-W.+Chiauthor=A.+R.+Ahmedauthor=Z.+Dereli-Korkutauthor=S.+Wang&title=Microfluidic+Cell+Chips+for+High-Throughput+Drug+Screening&doi=10.4155%2Fbio-2016-0028"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit227&amp;dbid=16384&amp;doi=10.4155%2Fbio-2016-0028&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4155%252Fbio-2016-0028%26sid%3Dliteratum%253Aachs%26aulast%3DChi%26aufirst%3DC.-W.%26aulast%3DAhmed%26aufirst%3DA.%2BR.%26aulast%3DDereli-Korkut%26aufirst%3DZ.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DMicrofluidic%2520Cell%2520Chips%2520for%2520High-Throughput%2520Drug%2520Screening%26jtitle%3DBioanalysis%26date%3D2016%26volume%3D8%26spage%3D921%26epage%3D937%26doi%3D10.4155%2Fbio-2016-0028" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref228"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref228'); return false;" data-citation="" class="refNumLink">228</a></strong><div class="NLM_citation" id="cit228"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Klucznik, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mikulak-Klucznik, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCormack, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lima, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szymkuć, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhowmick, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Molga, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rickershauser, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gajewska, E. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toutchkine, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dittwald, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Startek, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirkovits, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roszak, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adamski, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sieredzińska, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mrksich, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trice, S. L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grzybowski, B. A.</span></span> <span> </span><span class="NLM_article-title">Efficient Syntheses of Diverse, Medicinally Relevant Targets Planned by Computer and Executed in the Laboratory</span>. <i>Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">522</span>– <span class="NLM_lpage">532</span>, <span class="refDoi"> DOI: 10.1016/j.chempr.2018.02.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref228/cit228&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1016%2Fj.chempr.2018.02.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref228/cit228&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC1cXktlyitb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2018&pages=522-532&author=T.+Klucznikauthor=B.+Mikulak-Klucznikauthor=M.+P.+McCormackauthor=H.+Limaauthor=S.+Szymku%C4%87author=M.+Bhowmickauthor=K.+Molgaauthor=Y.+Zhouauthor=L.+Rickershauserauthor=E.+P.+Gajewskaauthor=A.+Toutchkineauthor=P.+Dittwaldauthor=M.+P.+Startekauthor=G.+J.+Kirkovitsauthor=R.+Roszakauthor=A.+Adamskiauthor=B.+Sieredzi%C5%84skaauthor=M.+Mrksichauthor=S.+L.+J.+Triceauthor=B.+A.+Grzybowski&title=Efficient+Syntheses+of+Diverse%2C+Medicinally+Relevant+Targets+Planned+by+Computer+and+Executed+in+the+Laboratory&doi=10.1016%2Fj.chempr.2018.02.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit228R"><div class="casContent"><span class="casTitleNuber">228</span><div class="casTitle"><span class="NLM_cas:atitle">Efficient Syntheses of Diverse, Medicinally Relevant Targets Planned by Computer and Executed in the Laboratory</span></div><div class="casAuthors">Klucznik, Tomasz; Mikulak-Klucznik, Barbara; McCormack, Michael P.; Lima, Heather; Szymkuc, Sara; Bhowmick, Manishabrata; Molga, Karol; Zhou, Yubai; Rickershauser, Lindsey; Gajewska, Ewa P.; Toutchkine, Alexei; Dittwald, Piotr; Startek, Michal P.; Kirkovits, Gregory J.; Roszak, Rafal; Adamski, Ariel; Sieredzinska, Bianka; Mrksich, Milan; Trice, Sarah L. J.; Grzybowski, Bartosz A.</div><div class="citationInfo"><span class="NLM_cas:title">Chem</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">522-532</span>CODEN:
                <span class="NLM_cas:coden">CHEMVE</span>;
        ISSN:<span class="NLM_cas:issn">2451-9294</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The Chematica program was used to autonomously design synthetic pathways to eight structurally diverse targets, including seven com. valuable bioactive substances and one natural product.  All of these computer-planned routes were successfully executed in the lab. and offer significant yield improvements and cost savings over previous approaches, provide alternatives to patented routes, or produce targets that were not synthesized previously.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGotOimnTZh2zLVg90H21EOLACvtfcHk0lhli4tukQwMVg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXktlyitb8%253D&md5=2d5d22d2570ca3e146ba9c25149dfc9e</span></div><a href="/servlet/linkout?suffix=cit228&amp;dbid=16384&amp;doi=10.1016%2Fj.chempr.2018.02.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chempr.2018.02.002%26sid%3Dliteratum%253Aachs%26aulast%3DKlucznik%26aufirst%3DT.%26aulast%3DMikulak-Klucznik%26aufirst%3DB.%26aulast%3DMcCormack%26aufirst%3DM.%2BP.%26aulast%3DLima%26aufirst%3DH.%26aulast%3DSzymku%25C4%2587%26aufirst%3DS.%26aulast%3DBhowmick%26aufirst%3DM.%26aulast%3DMolga%26aufirst%3DK.%26aulast%3DZhou%26aufirst%3DY.%26aulast%3DRickershauser%26aufirst%3DL.%26aulast%3DGajewska%26aufirst%3DE.%2BP.%26aulast%3DToutchkine%26aufirst%3DA.%26aulast%3DDittwald%26aufirst%3DP.%26aulast%3DStartek%26aufirst%3DM.%2BP.%26aulast%3DKirkovits%26aufirst%3DG.%2BJ.%26aulast%3DRoszak%26aufirst%3DR.%26aulast%3DAdamski%26aufirst%3DA.%26aulast%3DSieredzi%25C5%2584ska%26aufirst%3DB.%26aulast%3DMrksich%26aufirst%3DM.%26aulast%3DTrice%26aufirst%3DS.%2BL.%2BJ.%26aulast%3DGrzybowski%26aufirst%3DB.%2BA.%26atitle%3DEfficient%2520Syntheses%2520of%2520Diverse%252C%2520Medicinally%2520Relevant%2520Targets%2520Planned%2520by%2520Computer%2520and%2520Executed%2520in%2520the%2520Laboratory%26jtitle%3DChem.%26date%3D2018%26volume%3D4%26spage%3D522%26epage%3D532%26doi%3D10.1016%2Fj.chempr.2018.02.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref229"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref229'); return false;" data-citation="" class="refNumLink">229</a></strong><div class="NLM_citation" id="cit229"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Segler, M. H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Preuss, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waller, M. P.</span></span> <span> </span><span class="NLM_article-title">Planning Chemical Syntheses with Deep Neural Networks and Symbolic AI.</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>555</i></span>,  <span class="NLM_fpage">604</span>– <span class="NLM_lpage">610</span>, <span class="refDoi"> DOI: 10.1038/nature25978</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref229/cit229&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1038%2Fnature25978" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=555&publication_year=2018&pages=604-610&author=M.+H.+S.+Seglerauthor=M.+Preussauthor=M.+P.+Waller&title=Planning+Chemical+Syntheses+with+Deep+Neural+Networks+and+Symbolic+AI.&doi=10.1038%2Fnature25978"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit229&amp;dbid=16384&amp;doi=10.1038%2Fnature25978&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature25978%26sid%3Dliteratum%253Aachs%26aulast%3DSegler%26aufirst%3DM.%2BH.%2BS.%26aulast%3DPreuss%26aufirst%3DM.%26aulast%3DWaller%26aufirst%3DM.%2BP.%26atitle%3DPlanning%2520Chemical%2520Syntheses%2520with%2520Deep%2520Neural%2520Networks%2520and%2520Symbolic%2520AI.%26jtitle%3DNature%26date%3D2018%26volume%3D555%26spage%3D604%26epage%3D610%26doi%3D10.1038%2Fnature25978" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref230"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref230'); return false;" data-citation="" class="refNumLink">230</a></strong><div class="NLM_citation" id="cit230"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fons, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Esquerré, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Versluys, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mambrini, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paillasse, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bell, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schreyer, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cox, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bickerton, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lisztwan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whittaker, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bono, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnstone, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hopkins, A.</span></span> <span> </span><span class="NLM_article-title">Abstract 3970: Targeting the Adenosine Immunosuppressive Pathway for Cancer Immunotherapy with Small Molecule Agents</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>77</i></span>,  <span class="NLM_fpage">3970</span>, <span class="refDoi"> DOI: 10.1158/1538-7445.AM2017-3970</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref230/cit230&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1158%2F1538-7445.AM2017-3970" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=77&publication_year=2017&pages=3970&author=P.+Fonsauthor=M.+Esquerr%C3%A9author=S.+Versluysauthor=G.+Mambriniauthor=M.+Paillasseauthor=A.+Bellauthor=A.+Schreyerauthor=R.+Coxauthor=R.+Bickertonauthor=J.+Lisztwanauthor=M.+Whittakerauthor=F.+Bonoauthor=C.+Johnstoneauthor=A.+Hopkins&title=Abstract+3970%3A+Targeting+the+Adenosine+Immunosuppressive+Pathway+for+Cancer+Immunotherapy+with+Small+Molecule+Agents&doi=10.1158%2F1538-7445.AM2017-3970"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit230&amp;dbid=16384&amp;doi=10.1158%2F1538-7445.AM2017-3970&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1538-7445.AM2017-3970%26sid%3Dliteratum%253Aachs%26aulast%3DFons%26aufirst%3DP.%26aulast%3DEsquerr%25C3%25A9%26aufirst%3DM.%26aulast%3DVersluys%26aufirst%3DS.%26aulast%3DMambrini%26aufirst%3DG.%26aulast%3DPaillasse%26aufirst%3DM.%26aulast%3DBell%26aufirst%3DA.%26aulast%3DSchreyer%26aufirst%3DA.%26aulast%3DCox%26aufirst%3DR.%26aulast%3DBickerton%26aufirst%3DR.%26aulast%3DLisztwan%26aufirst%3DJ.%26aulast%3DWhittaker%26aufirst%3DM.%26aulast%3DBono%26aufirst%3DF.%26aulast%3DJohnstone%26aufirst%3DC.%26aulast%3DHopkins%26aufirst%3DA.%26atitle%3DAbstract%25203970%253A%2520Targeting%2520the%2520Adenosine%2520Immunosuppressive%2520Pathway%2520for%2520Cancer%2520Immunotherapy%2520with%2520Small%2520Molecule%2520Agents%26jtitle%3DCancer%2520Res.%26date%3D2017%26volume%3D77%26spage%3D3970%26doi%3D10.1158%2F1538-7445.AM2017-3970" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref231"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref231'); return false;" data-citation="" class="refNumLink">231</a></strong><div class="NLM_citation" id="cit231"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ptacin, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fischer, E. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aerni, H. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caffaro, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">San Jose, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feldman, A. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turner, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Romesberg, F. E.</span></span> <span> </span><span class="NLM_article-title">A Semi-Synthetic Organism That Stores and Retrieves Increased Genetic Information</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>551</i></span>,  <span class="NLM_fpage">644</span>– <span class="NLM_lpage">647</span>, <span class="refDoi"> DOI: 10.1038/nature24659</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref231/cit231&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1038%2Fnature24659" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref231/cit231&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=29189780" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref231/cit231&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvFWhtL7L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=551&publication_year=2017&pages=644-647&author=Y.+Zhangauthor=J.+L.+Ptacinauthor=E.+C.+Fischerauthor=H.+R.+Aerniauthor=C.+E.+Caffaroauthor=K.+San+Joseauthor=A.+W.+Feldmanauthor=C.+R.+Turnerauthor=F.+E.+Romesberg&title=A+Semi-Synthetic+Organism+That+Stores+and+Retrieves+Increased+Genetic+Information&doi=10.1038%2Fnature24659"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit231R"><div class="casContent"><span class="casTitleNuber">231</span><div class="casTitle"><span class="NLM_cas:atitle">A semi-synthetic organism that stores and retrieves increased genetic information</span></div><div class="casAuthors">Zhang, Yorke; Ptacin, Jerod L.; Fischer, Emil C.; Aerni, Hans R.; Caffaro, Carolina E.; San Jose, Kristine; Feldman, Aaron W.; Turner, Court R.; Romesberg, Floyd E.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">551</span>
        (<span class="NLM_cas:issue">7682</span>),
    <span class="NLM_cas:pages">644-647</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Since at least the last common ancestor of all life on Earth, genetic information has been stored in a four-letter alphabet that is propagated and retrieved by the formation of two base pairs.  The central goal of synthetic biol. is to create new life forms and functions, and the most general route to this goal is the creation of semi-synthetic organisms whose DNA harbors two addnl. letters that form a third, unnatural base pair.  Previous efforts to generate such semi-synthetic organisms culminated in the creation of a strain of Escherichia coli that, by virtue of a nucleoside triphosphate transporter from Phaeodactylum tricornutum, imports the requisite unnatural triphosphates from its medium and then uses them to replicate a plasmid contg. the unnatural base pair dNaM-dTPT3.  Although the semi-synthetic organism stores increased information when compared to natural organisms, retrieval of the information requires in vivo transcription of the unnatural base pair into mRNA and tRNA, aminoacylation of the tRNA with a non-canonical amino acid, and efficient participation of the unnatural base pair in decoding at the ribosome.  Here we report the in vivo transcription of DNA contg. dNaM and dTPT3 into mRNAs with two different unnatural codons and tRNAs with cognate unnatural anticodons, and their efficient decoding at the ribosome to direct the site-specific incorporation of natural or non-canonical amino acids into superfolder green fluorescent protein.  The results demonstrate that interactions other than hydrogen bonding can contribute to every step of information storage and retrieval.  The resulting semi-synthetic organism both encodes and retrieves increased information and should serve as a platform for the creation of new life forms and functions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqjEtvNNzQsZrVg90H21EOLACvtfcHk0lhZvaaYT_3xWQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvFWhtL7L&md5=d04861904783b1a852d4089b28f09acf</span></div><a href="/servlet/linkout?suffix=cit231&amp;dbid=16384&amp;doi=10.1038%2Fnature24659&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature24659%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DPtacin%26aufirst%3DJ.%2BL.%26aulast%3DFischer%26aufirst%3DE.%2BC.%26aulast%3DAerni%26aufirst%3DH.%2BR.%26aulast%3DCaffaro%26aufirst%3DC.%2BE.%26aulast%3DSan%2BJose%26aufirst%3DK.%26aulast%3DFeldman%26aufirst%3DA.%2BW.%26aulast%3DTurner%26aufirst%3DC.%2BR.%26aulast%3DRomesberg%26aufirst%3DF.%2BE.%26atitle%3DA%2520Semi-Synthetic%2520Organism%2520That%2520Stores%2520and%2520Retrieves%2520Increased%2520Genetic%2520Information%26jtitle%3DNature%26date%3D2017%26volume%3D551%26spage%3D644%26epage%3D647%26doi%3D10.1038%2Fnature24659" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref232"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref232'); return false;" data-citation="" class="refNumLink">232</a></strong><div class="NLM_citation" id="cit232"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Valeur, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knerr, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ölwegård-Halvarsson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lemurell, M.</span></span> <span> </span><span class="NLM_article-title">Targeted Delivery for Regenerative Medicines: An Untapped Opportunity for Drug Conjugates</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">841</span>– <span class="NLM_lpage">847</span>, <span class="refDoi"> DOI: 10.1016/j.drudis.2016.12.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref232/cit232&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1016%2Fj.drudis.2016.12.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref232/cit232&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=27988360" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref232/cit232&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC28XitFSlsb3N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2017&pages=841-847&author=E.+Valeurauthor=L.+Knerrauthor=M.+%C3%96lweg%C3%A5rd-Halvarssonauthor=M.+Lemurell&title=Targeted+Delivery+for+Regenerative+Medicines%3A+An+Untapped+Opportunity+for+Drug+Conjugates&doi=10.1016%2Fj.drudis.2016.12.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit232R"><div class="casContent"><span class="casTitleNuber">232</span><div class="casTitle"><span class="NLM_cas:atitle">Targeted delivery for regenerative medicines: an untapped opportunity for drug conjugates</span></div><div class="casAuthors">Valeur, Eric; Knerr, Laurent; Oelwegaard-Halvarsson, Maria; Lemurell, Malin</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">841-847</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Regenerative approaches are promising avenues to effectively cure diseases rather than merely treating symptoms, but are assocd. with concerns around proliferation in other organs.  Given that targeted delivery holds the promise of delivering a drug precisely to its desired site of action, usually with the prospect of increasing the therapeutic index, it can be considered as an essential enabler of regenerative medicines.  Although significant progress has been made predominantly in oncol. for the delivery of cytotoxic drugs using antibody-drug conjugates (ADCs), the physiol. conditions and safety requirements for regenerative medicines are very different.  Drug conjugates need to be approached differently and, we herein suggest using a broader range of homing modalities and a specific framework to develop safe linkers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqADJijsrrGXLVg90H21EOLACvtfcHk0lhZvaaYT_3xWQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitFSlsb3N&md5=482bf655641afc22051dd4e222546e13</span></div><a href="/servlet/linkout?suffix=cit232&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2016.12.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2016.12.004%26sid%3Dliteratum%253Aachs%26aulast%3DValeur%26aufirst%3DE.%26aulast%3DKnerr%26aufirst%3DL.%26aulast%3D%25C3%2596lweg%25C3%25A5rd-Halvarsson%26aufirst%3DM.%26aulast%3DLemurell%26aufirst%3DM.%26atitle%3DTargeted%2520Delivery%2520for%2520Regenerative%2520Medicines%253A%2520An%2520Untapped%2520Opportunity%2520for%2520Drug%2520Conjugates%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2017%26volume%3D22%26spage%3D841%26epage%3D847%26doi%3D10.1016%2Fj.drudis.2016.12.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52'],'ref53':['cit53'],'ref54':['cit54'],'ref55':['cit55'],'ref56':['cit56'],'ref57':['cit57'],'ref58':['cit58'],'ref59':['cit59'],'ref60':['cit60'],'ref61':['cit61'],'ref62':['cit62'],'ref63':['cit63'],'ref64':['cit64'],'ref65':['cit65'],'ref66':['cit66'],'ref67':['cit67'],'ref68':['cit68'],'ref69':['cit69'],'ref70':['cit70'],'ref71':['cit71'],'ref72':['cit72'],'ref73':['cit73'],'ref74':['cit74'],'ref75':['cit75'],'ref76':['cit76'],'ref77':['cit77'],'ref78':['cit78'],'ref79':['cit79'],'ref80':['cit80'],'ref81':['cit81'],'ref82':['cit82'],'ref83':['cit83'],'ref84':['cit84'],'ref85':['cit85'],'ref86':['cit86'],'ref87':['cit87'],'ref88':['cit88'],'ref89':['cit89'],'ref90':['cit90'],'ref91':['cit91'],'ref92':['cit92'],'ref93':['cit93'],'ref94':['cit94'],'ref95':['cit95'],'ref96':['cit96'],'ref97':['cit97'],'ref98':['cit98'],'ref99':['cit99'],'ref100':['cit100'],'ref101':['cit101'],'ref102':['cit102'],'ref103':['cit103'],'ref104':['cit104'],'ref105':['cit105'],'ref106':['cit106'],'ref107':['cit107'],'ref108':['cit108'],'ref109':['cit109'],'ref110':['cit110'],'ref111':['cit111'],'ref112':['cit112'],'ref113':['cit113'],'ref114':['cit114'],'ref115':['cit115'],'ref116':['cit116'],'ref117':['cit117'],'ref118':['cit118'],'ref119':['cit119'],'ref120':['cit120'],'ref121':['cit121'],'ref122':['cit122'],'ref123':['cit123'],'ref124':['cit124'],'ref125':['cit125'],'ref126':['cit126'],'ref127':['cit127'],'ref128':['cit128'],'ref129':['cit129'],'ref130':['cit130'],'ref131':['cit131'],'ref132':['cit132'],'ref133':['cit133'],'ref134':['cit134'],'ref135':['cit135'],'ref136':['cit136'],'ref137':['cit137'],'ref138':['cit138'],'ref139':['cit139'],'ref140':['cit140'],'ref141':['cit141'],'ref142':['cit142'],'ref143':['cit143'],'ref144':['cit144'],'ref145':['cit145'],'ref146':['cit146'],'ref147':['cit147'],'ref148':['cit148'],'ref149':['cit149'],'ref150':['cit150'],'ref151':['cit151'],'ref152':['cit152'],'ref153':['cit153'],'ref154':['cit154'],'ref155':['cit155'],'ref156':['cit156'],'ref157':['cit157'],'ref158':['cit158'],'ref159':['cit159'],'ref160':['cit160'],'ref161':['cit161'],'ref162':['cit162'],'ref163':['cit163'],'ref164':['cit164'],'ref165':['cit165'],'ref166':['cit166'],'ref167':['cit167'],'ref168':['cit168'],'ref169':['cit169'],'ref170':['cit170'],'ref171':['cit171'],'ref172':['cit172'],'ref173':['cit173'],'ref174':['cit174'],'ref175':['cit175'],'ref176':['cit176'],'ref177':['cit177'],'ref178':['cit178'],'ref179':['cit179'],'ref180':['cit180'],'ref181':['cit181'],'ref182':['cit182'],'ref183':['cit183'],'ref184':['cit184'],'ref185':['cit185'],'ref186':['cit186'],'ref187':['cit187'],'ref188':['cit188'],'ref189':['cit189'],'ref190':['cit190'],'ref191':['cit191'],'ref192':['cit192'],'ref193':['cit193'],'ref194':['cit194'],'ref195':['cit195'],'ref196':['cit196'],'ref197':['cit197'],'ref198':['cit198'],'ref199':['cit199'],'ref200':['cit200'],'ref201':['cit201'],'ref202':['cit202'],'ref203':['cit203'],'ref204':['cit204'],'ref205':['cit205'],'ref206':['cit206'],'ref207':['cit207'],'ref208':['cit208'],'ref209':['cit209'],'ref210':['cit210'],'ref211':['cit211'],'ref212':['cit212'],'ref213':['cit213'],'ref214':['cit214'],'ref215':['cit215'],'ref216':['cit216'],'ref217':['cit217'],'ref218':['cit218'],'ref219':['cit219'],'ref220':['cit220'],'ref221':['cit221'],'ref222':['cit222'],'ref223':['cit223'],'ref224':['cit224'],'ref225':['cit225'],'ref226':['cit226'],'ref227':['cit227'],'ref228':['cit228'],'ref229':['cit229'],'ref230':['cit230'],'ref231':['cit231'],'ref232':['cit232']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 24 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Zhe Zhang, Christian Hug, Yuanqi Tao, Francis Bitsch, <span class="NLM_string-name hlFld-ContribAuthor">Yang Yang</span>. </span><span class="cited-content_cbyCitation_article-title">Solving Complex Biologics Truncation Problems by Top-Down Mass Spectrometry. </span><span class="cited-content_cbyCitation_journal-name">Journal of the American Society for Mass Spectrometry</span><span> <strong>2021,</strong> <em>32 </em>
                                    (8)
                                     , 1928-1935. <a href="https://doi.org/10.1021/jasms.0c00343" title="DOI URL">https://doi.org/10.1021/jasms.0c00343</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/jasms.0c00343&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fjasms.0c00343%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520the%2520American%2520Society%2520for%2520Mass%2520Spectrometry%26atitle%3DSolving%252BComplex%252BBiologics%252BTruncation%252BProblems%252Bby%252BTop-Down%252BMass%252BSpectrometry%26aulast%3DZhang%26aufirst%3DZhe%26date%3D2021%26date%3D2021%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D17092020%26date%3D23122020%26date%3D21122020%26date%3D04012021%26volume%3D32%26issue%3D8%26spage%3D1928%26epage%3D1935" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Sabrina E. Iskandar, Victoria A. Haberman, <span class="NLM_string-name hlFld-ContribAuthor">Albert A. Bowers</span>. </span><span class="cited-content_cbyCitation_article-title">Expanding the Chemical Diversity of Genetically Encoded Libraries. </span><span class="cited-content_cbyCitation_journal-name">ACS Combinatorial Science</span><span> <strong>2020,</strong> <em>22 </em>
                                    (12)
                                     , 712-733. <a href="https://doi.org/10.1021/acscombsci.0c00179" title="DOI URL">https://doi.org/10.1021/acscombsci.0c00179</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acscombsci.0c00179&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facscombsci.0c00179%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Combinatorial%2520Science%26atitle%3DExpanding%252Bthe%252BChemical%252BDiversity%252Bof%252BGenetically%252BEncoded%252BLibraries%26aulast%3DIskandar%26aufirst%3DSabrina%2BE.%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D08092020%26date%3D20102020%26date%3D09112020%26volume%3D22%26issue%3D12%26spage%3D712%26epage%3D733" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Shashank Kulkarni, Klaus Urbahns, <span class="NLM_string-name hlFld-ContribAuthor">Thomas Spangenberg</span>. </span><span class="cited-content_cbyCitation_article-title">Targeted Covalent Inhibitors for the Treatment of Malaria?. </span><span class="cited-content_cbyCitation_journal-name">ACS Infectious Diseases</span><span> <strong>2020,</strong> <em>6 </em>
                                    (11)
                                     , 2815-2817. <a href="https://doi.org/10.1021/acsinfecdis.0c00684" title="DOI URL">https://doi.org/10.1021/acsinfecdis.0c00684</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsinfecdis.0c00684&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsinfecdis.0c00684%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Infectious%2520Diseases%26atitle%3DTargeted%252BCovalent%252BInhibitors%252Bfor%252Bthe%252BTreatment%252Bof%252BMalaria%25253F%26aulast%3DKulkarni%26aufirst%3DShashank%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D29092020%26date%3D15102020%26volume%3D6%26issue%3D11%26spage%3D2815%26epage%3D2817" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Matthew G. Costales, Jessica L. Childs-Disney, Hafeez S. Haniff, <span class="NLM_string-name hlFld-ContribAuthor">Matthew D. Disney</span>. </span><span class="cited-content_cbyCitation_article-title">How We Think about Targeting RNA with Small Molecules. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (17)
                                     , 8880-8900. <a href="https://doi.org/10.1021/acs.jmedchem.9b01927" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b01927</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b01927&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.9b01927%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DHow%252BWe%252BThink%252Babout%252BTargeting%252BRNA%252Bwith%252BSmall%252BMolecules%26aulast%3DCostales%26aufirst%3DMatthew%2BG.%26date%3D2020%26date%3D2020%26date%3D2019%26date%3D21112019%26date%3D26032020%26volume%3D63%26issue%3D17%26spage%3D8880%26epage%3D8900" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Andrei Ursu, Jessica L. Childs-Disney, Alicia J. Angelbello, Matthew G. Costales, Samantha M. Meyer, <span class="NLM_string-name hlFld-ContribAuthor">Matthew D. Disney</span>. </span><span class="cited-content_cbyCitation_article-title">Gini Coefficients as a Single Value Metric to Define Chemical Probe Selectivity. </span><span class="cited-content_cbyCitation_journal-name">ACS Chemical Biology</span><span> <strong>2020,</strong> <em>15 </em>
                                    (8)
                                     , 2031-2040. <a href="https://doi.org/10.1021/acschembio.0c00486" title="DOI URL">https://doi.org/10.1021/acschembio.0c00486</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acschembio.0c00486&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facschembio.0c00486%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Chemical%2520Biology%26atitle%3DGini%252BCoefficients%252Bas%252Ba%252BSingle%252BValue%252BMetric%252Bto%252BDefine%252BChemical%252BProbe%252BSelectivity%26aulast%3DUrsu%26aufirst%3DAndrei%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D15062020%26date%3D22062020%26date%3D09072020%26date%3D22062020%26volume%3D15%26issue%3D8%26spage%3D2031%26epage%3D2040" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Julia Zinkus-Boltz, Craig DeValk, <span class="NLM_string-name hlFld-ContribAuthor">Bryan C. Dickinson</span>. </span><span class="cited-content_cbyCitation_article-title">A Phage-Assisted Continuous Selection Approach for Deep Mutational Scanning of Protein–Protein Interactions. </span><span class="cited-content_cbyCitation_journal-name">ACS Chemical Biology</span><span> <strong>2019,</strong> <em>14 </em>
                                    (12)
                                     , 2757-2767. <a href="https://doi.org/10.1021/acschembio.9b00669" title="DOI URL">https://doi.org/10.1021/acschembio.9b00669</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acschembio.9b00669&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facschembio.9b00669%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Chemical%2520Biology%26atitle%3DA%252BPhage-Assisted%252BContinuous%252BSelection%252BApproach%252Bfor%252BDeep%252BMutational%252BScanning%252Bof%252BProtein%2525E2%252580%252593Protein%252BInteractions%26aulast%3DZinkus-Boltz%26aufirst%3DJulia%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D18082019%26date%3D19112019%26date%3D06122019%26volume%3D14%26issue%3D12%26spage%3D2757%26epage%3D2767" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Quinn Coughlin, Allen T. Hopper, Maria-Jesus Blanco, Vijaya Tirunagaru, Albert J. Robichaud, <span class="NLM_string-name hlFld-ContribAuthor">Dario Doller</span>. </span><span class="cited-content_cbyCitation_article-title">Allosteric Modalities for Membrane-Bound Receptors: Insights from Drug Hunting for Brain Diseases. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2019,</strong> <em>62 </em>
                                    (13)
                                     , 5979-6002. <a href="https://doi.org/10.1021/acs.jmedchem.8b01651" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.8b01651</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.8b01651&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.8b01651%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DAllosteric%252BModalities%252Bfor%252BMembrane-Bound%252BReceptors%25253A%252BInsights%252Bfrom%252BDrug%252BHunting%252Bfor%252BBrain%252BDiseases%26aulast%3DCoughlin%26aufirst%3DQuinn%26date%3D2019%26date%3D2019%26date%3D2018%26date%3D23102018%26date%3D01032019%26date%3D05022019%26volume%3D62%26issue%3D13%26spage%3D5979%26epage%3D6002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Philipp
M. Cromm, Kusal T. G. Samarasinghe, John Hines, <span class="NLM_string-name hlFld-ContribAuthor">Craig M. Crews</span>. </span><span class="cited-content_cbyCitation_article-title">Addressing Kinase-Independent Functions of Fak via PROTAC-Mediated Degradation. </span><span class="cited-content_cbyCitation_journal-name">Journal of the American Chemical Society</span><span> <strong>2018,</strong> <em>140 </em>
                                    (49)
                                     , 17019-17026. <a href="https://doi.org/10.1021/jacs.8b08008" title="DOI URL">https://doi.org/10.1021/jacs.8b08008</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/jacs.8b08008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fjacs.8b08008%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520the%2520American%2520Chemical%2520Society%26atitle%3DAddressing%252BKinase-Independent%252BFunctions%252Bof%252BFak%252Bvia%252BPROTAC-Mediated%252BDegradation%26aulast%3DCromm%26aufirst%3DPhilipp%2BM.%26date%3D2018%26date%3D2018%26date%3D2018%26date%3D03082018%26date%3D28112018%26date%3D16112018%26volume%3D140%26issue%3D49%26spage%3D17019%26epage%3D17026" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Annely  Lorents</span>, <span class="hlFld-ContribAuthor ">Maria  Maloverjan</span>, <span class="hlFld-ContribAuthor ">Kärt  Padari</span>, <span class="hlFld-ContribAuthor ">Margus  Pooga</span>. </span><span class="cited-content_cbyCitation_article-title">Internalisation and Biological Activity of Nucleic Acids Delivering Cell-Penetrating Peptide Nanoparticles Is Controlled by the Biomolecular Corona. </span><span class="cited-content_cbyCitation_journal-name">Pharmaceuticals</span><span> <strong>2021,</strong> <em>14 </em>
                                    (7)
                                     , 667. <a href="https://doi.org/10.3390/ph14070667" title="DOI URL">https://doi.org/10.3390/ph14070667</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/ph14070667&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fph14070667%26sid%3Dliteratum%253Aachs%26jtitle%3DPharmaceuticals%26atitle%3DInternalisation%252Band%252BBiological%252BActivity%252Bof%252BNucleic%252BAcids%252BDelivering%252BCell-Penetrating%252BPeptide%252BNanoparticles%252BIs%252BControlled%252Bby%252Bthe%252BBiomolecular%252BCorona%26aulast%3DLorents%26aufirst%3DAnnely%26date%3D2021%26date%3D2021%26volume%3D14%26issue%3D7%26spage%3D667" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jing  Shi</span>, <span class="hlFld-ContribAuthor ">Xuesong  Chen</span>, <span class="hlFld-ContribAuthor ">Jianbo  Diao</span>, <span class="hlFld-ContribAuthor ">Liying  Jiang</span>, <span class="hlFld-ContribAuthor ">Lan  Li</span>, <span class="hlFld-ContribAuthor ">Stephen  Li</span>, <span class="hlFld-ContribAuthor ">Wenzhong  Liang</span>, <span class="hlFld-ContribAuthor ">Xiaoying  Jin</span>, <span class="hlFld-ContribAuthor ">Yonghui  Wang</span>, <span class="hlFld-ContribAuthor ">Colton  Wong</span>, <span class="hlFld-ContribAuthor ">Xiaolong Tom  Zhang</span>, <span class="hlFld-ContribAuthor ">Francis L.S.  Tse</span>. </span><span class="cited-content_cbyCitation_article-title">Bioanalysis in the Age of New Drug Modalities. </span><span class="cited-content_cbyCitation_journal-name">The AAPS Journal</span><span> <strong>2021,</strong> <em>23 </em>
                                    (3)
                                     <a href="https://doi.org/10.1208/s12248-021-00594-w" title="DOI URL">https://doi.org/10.1208/s12248-021-00594-w</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1208/s12248-021-00594-w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1208%2Fs12248-021-00594-w%26sid%3Dliteratum%253Aachs%26jtitle%3DThe%2520AAPS%2520Journal%26atitle%3DBioanalysis%252Bin%252Bthe%252BAge%252Bof%252BNew%252BDrug%252BModalities%26aulast%3DShi%26aufirst%3DJing%26date%3D2021%26date%3D2021%26volume%3D23%26issue%3D3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Scott D.  Edmondson</span>, <span class="hlFld-ContribAuthor ">Duane A.  Burnett</span>, <span class="hlFld-ContribAuthor ">Harry R.  Davis</span>. </span><span class="cited-content_cbyCitation_article-title">Atherosclerosis: Cholesterol Management. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2021,</strong>,, 1-36. <a href="https://doi.org/10.1002/0471266949.bmc185.pub2" title="DOI URL">https://doi.org/10.1002/0471266949.bmc185.pub2</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/0471266949.bmc185.pub2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2F0471266949.bmc185.pub2%26sid%3Dliteratum%253Aachs%26atitle%3DAtherosclerosis%25253A%252BCholesterol%252BManagement%26aulast%3DEdmondson%26aufirst%3DScott%2BD.%26date%3D2021%26date%3D2021%26spage%3D1%26epage%3D36%26pub%3DWiley%26atitle%3DBurger%252527s%252BMedicinal%252BChemistry%252Band%252BDrug%252BDiscovery%26aulast%3DAbraham%26aufirst%3DDonald%2BJ.%26date%3D2003%26date%3D2003%26volume%3D139" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">David  Janzén</span>, <span class="hlFld-ContribAuthor ">Peter  Gennemark</span>, <span class="hlFld-ContribAuthor ">Daniel  Hovdal</span>, <span class="hlFld-ContribAuthor ">Rasmus  Jansson-Löfmark</span>, <span class="hlFld-ContribAuthor ">Christine  Ahlström</span>. </span><span class="cited-content_cbyCitation_article-title">Dynamical Modeling of Different Drug Modalities in Drug Research. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2021,</strong>,, 232-242. <a href="https://doi.org/10.1016/B978-0-12-801238-3.11542-9" title="DOI URL">https://doi.org/10.1016/B978-0-12-801238-3.11542-9</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/B978-0-12-801238-3.11542-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2FB978-0-12-801238-3.11542-9%26sid%3Dliteratum%253Aachs%26atitle%3DDynamical%252BModeling%252Bof%252BDifferent%252BDrug%252BModalities%252Bin%252BDrug%252BResearch%26aulast%3DJanz%25C3%25A9n%26aufirst%3DDavid%26date%3D2021%26spage%3D232%26epage%3D242%26pub%3DElsevier%26atitle%3DSystems%252BMedicine%26date%3D2021%26volume%3D88" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Daniel  Conole</span>, <span class="hlFld-ContribAuthor ">James  H Hunter</span>, <span class="hlFld-ContribAuthor ">Michael  J Waring</span>. </span><span class="cited-content_cbyCitation_article-title">The maturation of DNA encoded libraries: opportunities for new users. </span><span class="cited-content_cbyCitation_journal-name">Future Medicinal Chemistry</span><span> <strong>2020,</strong> <em>39 </em><a href="https://doi.org/10.4155/fmc-2020-0285" title="DOI URL">https://doi.org/10.4155/fmc-2020-0285</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.4155/fmc-2020-0285&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.4155%2Ffmc-2020-0285%26sid%3Dliteratum%253Aachs%26jtitle%3DFuture%2520Medicinal%2520Chemistry%26atitle%3DThe%252Bmaturation%252Bof%252BDNA%252Bencoded%252Blibraries%25253A%252Bopportunities%252Bfor%252Bnew%252Busers%26aulast%3DConole%26aufirst%3DDaniel%26date%3D2020%26date%3D2020%26volume%3D39" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Martin  Braddock</span>. </span><span class="cited-content_cbyCitation_article-title">From Target Identification to Drug Development in Space: Using the Microgravity Assist. </span><span class="cited-content_cbyCitation_journal-name">Current Drug Discovery Technologies</span><span> <strong>2020,</strong> <em>17 </em>
                                    (1)
                                     , 45-56. <a href="https://doi.org/10.2174/1570163816666190112150014" title="DOI URL">https://doi.org/10.2174/1570163816666190112150014</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.2174/1570163816666190112150014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.2174%2F1570163816666190112150014%26sid%3Dliteratum%253Aachs%26jtitle%3DCurrent%2520Drug%2520Discovery%2520Technologies%26atitle%3DFrom%252BTarget%252BIdentification%252Bto%252BDrug%252BDevelopment%252Bin%252BSpace%25253A%252BUsing%252Bthe%252BMicrogravity%252BAssist%26aulast%3DBraddock%26aufirst%3DMartin%26date%3D2020%26volume%3D17%26issue%3D1%26spage%3D45%26epage%3D56" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jean‐Paul  Renaud</span>. </span><span class="cited-content_cbyCitation_article-title">The Evolving Role of Structural Biology in Drug Discovery. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2020,</strong>,, 1-22. <a href="https://doi.org/10.1002/9781118681121.ch1" title="DOI URL">https://doi.org/10.1002/9781118681121.ch1</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/9781118681121.ch1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2F9781118681121.ch1%26sid%3Dliteratum%253Aachs%26atitle%3DThe%252BEvolving%252BRole%252Bof%252BStructural%252BBiology%252Bin%252BDrug%252BDiscovery%26aulast%3DRenaud%26aufirst%3DJean%25E2%2580%2590Paul%26date%3D2020%26date%3D2020%26spage%3D1%26epage%3D22%26pub%3DWiley%26atitle%3DStructural%252BBiology%252Bin%252BDrug%252BDiscovery%26aulast%3DRenaud%26aufirst%3DJean%25E2%2580%2590Paul%26date%3D2020%26date%3D2020%26volume%3D10" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Céline  Martin</span>, <span class="hlFld-ContribAuthor ">Serena  De Piccoli</span>, <span class="hlFld-ContribAuthor ">Marc  Gaysinski</span>, <span class="hlFld-ContribAuthor ">Cécile  Becquart</span>, <span class="hlFld-ContribAuthor ">Stéphane  Azoulay</span>, <span class="hlFld-ContribAuthor ">Audrey  Di Giorgio</span>, <span class="hlFld-ContribAuthor ">Maria  Duca</span>. </span><span class="cited-content_cbyCitation_article-title">Unveiling RNA‐Binding Properties of Verapamil and Preparation of New Derivatives as Inhibitors of HIV‐1 Tat‐TAR Interaction. </span><span class="cited-content_cbyCitation_journal-name">ChemPlusChem</span><span> <strong>2020,</strong> <em>85 </em>
                                    (1)
                                     , 207-216. <a href="https://doi.org/10.1002/cplu.201900650" title="DOI URL">https://doi.org/10.1002/cplu.201900650</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/cplu.201900650&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fcplu.201900650%26sid%3Dliteratum%253Aachs%26jtitle%3DChemPlusChem%26atitle%3DUnveiling%252BRNA%2525E2%252580%252590Binding%252BProperties%252Bof%252BVerapamil%252Band%252BPreparation%252Bof%252BNew%252BDerivatives%252Bas%252BInhibitors%252Bof%252BHIV%2525E2%252580%2525901%252BTat%2525E2%252580%252590TAR%252BInteraction%26aulast%3DMartin%26aufirst%3DC%25C3%25A9line%26date%3D2020%26date%3D2020%26volume%3D85%26issue%3D1%26spage%3D207%26epage%3D216" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Phoebe  Ryder</span>, <span class="hlFld-ContribAuthor ">Martin  Braddock</span>. </span><span class="cited-content_cbyCitation_article-title">Harnessing the Space Environment for the Discovery and Development of New Medicines. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2020,</strong>,, 1-35. <a href="https://doi.org/10.1007/978-3-319-50909-9_32-1" title="DOI URL">https://doi.org/10.1007/978-3-319-50909-9_32-1</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/978-3-319-50909-9_32-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2F978-3-319-50909-9_32-1%26sid%3Dliteratum%253Aachs%26atitle%3DHarnessing%252Bthe%252BSpace%252BEnvironment%252Bfor%252Bthe%252BDiscovery%252Band%252BDevelopment%252Bof%252BNew%252BMedicines%26aulast%3DRyder%26aufirst%3DPhoebe%26date%3D2020%26date%3D2019%26spage%3D1%26epage%3D35%26pub%3DSpringer%2520International%2520Publishing%26atitle%3DHandbook%252Bof%252BSpace%252BPharmaceuticals%26aulast%3DPathak%26aufirst%3DYashwant%26date%3D2019%26volume%3D59" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Rick P. C.  Cousins</span>. </span><span class="cited-content_cbyCitation_article-title">Medicines discovery for auditory disorders: Challenges for industry. </span><span class="cited-content_cbyCitation_journal-name">The Journal of the Acoustical Society of America</span><span> <strong>2019,</strong> <em>146 </em>
                                    (5)
                                     , 3652-3667. <a href="https://doi.org/10.1121/1.5132706" title="DOI URL">https://doi.org/10.1121/1.5132706</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1121/1.5132706&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1121%2F1.5132706%26sid%3Dliteratum%253Aachs%26jtitle%3DThe%2520Journal%2520of%2520the%2520Acoustical%2520Society%2520of%2520America%26atitle%3DMedicines%252Bdiscovery%252Bfor%252Bauditory%252Bdisorders%25253A%252BChallenges%252Bfor%252Bindustry%26aulast%3DCousins%26aufirst%3DRick%2BP.%2BC.%26date%3D2019%26volume%3D146%26issue%3D5%26spage%3D3652%26epage%3D3667" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Liwen  Xia</span>, <span class="hlFld-ContribAuthor ">Wei  Liu</span>, <span class="hlFld-ContribAuthor ">Yinsen  Song</span>, <span class="hlFld-ContribAuthor ">Hailiang  Zhu</span>, <span class="hlFld-ContribAuthor ">Yongtao  Duan</span>. </span><span class="cited-content_cbyCitation_article-title">The Present and Future of Novel Protein Degradation Technology. </span><span class="cited-content_cbyCitation_journal-name">Current Topics in Medicinal Chemistry</span><span> <strong>2019,</strong> <em>19 </em>
                                    (20)
                                     , 1784-1788. <a href="https://doi.org/10.2174/1568026619666191011162955" title="DOI URL">https://doi.org/10.2174/1568026619666191011162955</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.2174/1568026619666191011162955&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.2174%2F1568026619666191011162955%26sid%3Dliteratum%253Aachs%26jtitle%3DCurrent%2520Topics%2520in%2520Medicinal%2520Chemistry%26atitle%3DThe%252BPresent%252Band%252BFuture%252Bof%252BNovel%252BProtein%252BDegradation%252BTechnology%26aulast%3DXia%26aufirst%3DLiwen%26date%3D2019%26volume%3D19%26issue%3D20%26spage%3D1784%26epage%3D1788" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Andrei  Ursu</span>, <span class="hlFld-ContribAuthor ">Simon  Vézina-Dawod</span>, <span class="hlFld-ContribAuthor ">Matthew D.  Disney</span>. </span><span class="cited-content_cbyCitation_article-title">Methods to identify and optimize small molecules interacting with RNA (SMIRNAs). </span><span class="cited-content_cbyCitation_journal-name">Drug Discovery Today</span><span> <strong>2019,</strong> <em>24 </em>
                                    (10)
                                     , 2002-2016. <a href="https://doi.org/10.1016/j.drudis.2019.06.019" title="DOI URL">https://doi.org/10.1016/j.drudis.2019.06.019</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.drudis.2019.06.019&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.drudis.2019.06.019%26sid%3Dliteratum%253Aachs%26jtitle%3DDrug%2520Discovery%2520Today%26atitle%3DMethods%252Bto%252Bidentify%252Band%252Boptimize%252Bsmall%252Bmolecules%252Binteracting%252Bwith%252BRNA%252B%252528SMIRNAs%252529%26aulast%3DUrsu%26aufirst%3DAndrei%26date%3D2019%26volume%3D24%26issue%3D10%26spage%3D2002%26epage%3D2016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Eric  Valeur</span>, <span class="hlFld-ContribAuthor ">Frank  Narjes</span>, <span class="hlFld-ContribAuthor ">Christian  Ottmann</span>, <span class="hlFld-ContribAuthor ">Alleyn T.  Plowright</span>. </span><span class="cited-content_cbyCitation_article-title">Emerging modes-of-action in drug discovery. </span><span class="cited-content_cbyCitation_journal-name">MedChemComm</span><span> <strong>2019,</strong> <em>10 </em>
                                    (9)
                                     , 1550-1568. <a href="https://doi.org/10.1039/C9MD00263D" title="DOI URL">https://doi.org/10.1039/C9MD00263D</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/C9MD00263D&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FC9MD00263D%26sid%3Dliteratum%253Aachs%26jtitle%3DMedChemComm%26atitle%3DEmerging%252Bmodes-of-action%252Bin%252Bdrug%252Bdiscovery%26aulast%3DValeur%26aufirst%3DEric%26date%3D2019%26date%3D2019%26volume%3D10%26issue%3D9%26spage%3D1550%26epage%3D1568" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Mariell  Pettersson</span>, <span class="hlFld-ContribAuthor ">Craig M.  Crews</span>. </span><span class="cited-content_cbyCitation_article-title">PROteolysis TArgeting Chimeras (PROTACs) — Past, present and future. </span><span class="cited-content_cbyCitation_journal-name">Drug Discovery Today: Technologies</span><span> <strong>2019,</strong> <em>31 </em>, 15-27. <a href="https://doi.org/10.1016/j.ddtec.2019.01.002" title="DOI URL">https://doi.org/10.1016/j.ddtec.2019.01.002</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ddtec.2019.01.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ddtec.2019.01.002%26sid%3Dliteratum%253Aachs%26jtitle%3DDrug%2520Discovery%2520Today%253A%2520Technologies%26atitle%3DPROteolysis%252BTArgeting%252BChimeras%252B%252528PROTACs%252529%252B%2525E2%252580%252594%252BPast%25252C%252Bpresent%252Band%252Bfuture%26aulast%3DPettersson%26aufirst%3DMariell%26date%3D2019%26volume%3D31%26spage%3D15%26epage%3D27" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Matthew  Williams</span>, <span class="hlFld-ContribAuthor ">Martin  Braddock</span>. </span><span class="cited-content_cbyCitation_article-title">AI Case Studies: Potential for Human Health, Space Exploration and Colonisation and a Proposed Superimposition of the Kubler-Ross Change Curve on the Hype Cycle. </span><span class="cited-content_cbyCitation_journal-name">Studia Humana</span><span> <strong>2019,</strong> <em>8 </em>
                                    (1)
                                     , 3-18. <a href="https://doi.org/10.2478/sh-2019-0001" title="DOI URL">https://doi.org/10.2478/sh-2019-0001</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.2478/sh-2019-0001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.2478%2Fsh-2019-0001%26sid%3Dliteratum%253Aachs%26jtitle%3DStudia%2520Humana%26atitle%3DAI%252BCase%252BStudies%25253A%252BPotential%252Bfor%252BHuman%252BHealth%25252C%252BSpace%252BExploration%252Band%252BColonisation%252Band%252Ba%252BProposed%252BSuperimposition%252Bof%252Bthe%252BKubler-Ross%252BChange%252BCurve%252Bon%252Bthe%252BHype%252BCycle%26aulast%3DWilliams%26aufirst%3DMatthew%26date%3D2019%26date%3D2019%26volume%3D8%26issue%3D1%26spage%3D3%26epage%3D18" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ülo  Langel</span>. </span><span class="cited-content_cbyCitation_article-title">Clinical Trials and Commercialization Using CPPs. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2019,</strong>,, 395-408. <a href="https://doi.org/10.1007/978-981-13-8747-0_11" title="DOI URL">https://doi.org/10.1007/978-981-13-8747-0_11</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/978-981-13-8747-0_11&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2F978-981-13-8747-0_11%26sid%3Dliteratum%253Aachs%26atitle%3DClinical%252BTrials%252Band%252BCommercialization%252BUsing%252BCPPs%26aulast%3DLangel%26aufirst%3D%25C3%259Clo%26date%3D2019%26date%3D2019%26spage%3D395%26epage%3D408%26pub%3DSpringer%2520Singapore%26atitle%3DCPP%25252C%252BCell-Penetrating%252BPeptides%26aulast%3DLangel%26aufirst%3D%25C3%259Clo%26date%3D2019%26volume%3D11" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b00378/20181019/images/medium/jm-2018-00378e_0022.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b00378/20181019/images/large/jm-2018-00378e_0022.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00378&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b00378/20181019/images/medium/jm-2018-00378e_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b00378/20181019/images/large/jm-2018-00378e_0001.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. MOAs affecting direct protein function and protein levels</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b00378/20181019/images/large/jm-2018-00378e_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00378&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b00378/20181019/images/medium/jm-2018-00378e_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b00378/20181019/images/large/jm-2018-00378e_0002.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. (a) modified phage display for the identification of bicyclic peptides. (b) Modified mRNA display for cyclic peptides containing unnatural amino acids.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b00378/20181019/images/large/jm-2018-00378e_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00378&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b00378/20181019/images/medium/jm-2018-00378e_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b00378/20181019/images/large/jm-2018-00378e_0003.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Examples of stabilized peptides. Color coding for amino acids: purple = <span class="smallcaps smallerCapital">d</span>-amino acid</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b00378/20181019/images/large/jm-2018-00378e_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00378&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b00378/20181019/images/medium/jm-2018-00378e_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b00378/20181019/images/large/jm-2018-00378e_0004.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Workflow for screening venoms.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b00378/20181019/images/large/jm-2018-00378e_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00378&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b00378/20181019/images/medium/jm-2018-00378e_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b00378/20181019/images/large/jm-2018-00378e_0005.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Principle of DELs.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b00378/20181019/images/large/jm-2018-00378e_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00378&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b00378/20181019/images/medium/jm-2018-00378e_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b00378/20181019/images/large/jm-2018-00378e_0006.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Examples of hits and probes identified by DEL.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b00378/20181019/images/large/jm-2018-00378e_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00378&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b00378/20181019/images/medium/jm-2018-00378e_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b00378/20181019/images/large/jm-2018-00378e_0007.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Examples of macrocycles in clinical phase.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b00378/20181019/images/large/jm-2018-00378e_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00378&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig8" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b00378/20181019/images/medium/jm-2018-00378e_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b00378/20181019/images/large/jm-2018-00378e_0008.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Overview of historical and new business models for expanding compound collections.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b00378/20181019/images/large/jm-2018-00378e_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00378&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig9" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b00378/20181019/images/medium/jm-2018-00378e_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b00378/20181019/images/large/jm-2018-00378e_0009.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. Protein structure mimetics strategies. (a) stabilization of α-helices, (b) stabilization of β-hairpins, (c) stabilization of loops.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b00378/20181019/images/large/jm-2018-00378e_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00378&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig10" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 10</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b00378/20181019/images/medium/jm-2018-00378e_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b00378/20181019/images/large/jm-2018-00378e_0010.jpeg" id="rightTab-gr10" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 10. Examples of protein-structure mimetics. Color coding for amino acids: light blue = β-amino acid, purple = <span class="smallcaps smallerCapital">d</span>-amino acid, and orange = unnatural amino acid used for ring closure metathesis. X indicates an amino acid whose identity has not been released.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b00378/20181019/images/large/jm-2018-00378e_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00378&amp;id=fig10"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig11" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Figure 11</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b00378/20181019/images/medium/jm-2018-00378e_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b00378/20181019/images/large/jm-2018-00378e_0011.jpeg" id="rightTab-gr11" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 11. Principle of scaffold grafting</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b00378/20181019/images/large/jm-2018-00378e_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00378&amp;id=fig11"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig12" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Figure 12</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b00378/20181019/images/medium/jm-2018-00378e_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b00378/20181019/images/large/jm-2018-00378e_0012.jpeg" id="rightTab-gr12" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 12. Example of cyclotide (Kalata B1) with a grafted MCR4 sequence.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b00378/20181019/images/large/jm-2018-00378e_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00378&amp;id=fig12"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig13" data-index="13" class="article__inlineFigure"><h2 class="fig-label">Figure 13</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b00378/20181019/images/medium/jm-2018-00378e_0013.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b00378/20181019/images/large/jm-2018-00378e_0013.jpeg" id="rightTab-gr13" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 13. MOAs of modalities regulating protein levels.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b00378/20181019/images/large/jm-2018-00378e_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00378&amp;id=fig13"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig14" data-index="14" class="article__inlineFigure"><h2 class="fig-label">Figure 14</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b00378/20181019/images/medium/jm-2018-00378e_0014.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b00378/20181019/images/large/jm-2018-00378e_0014.jpeg" id="rightTab-gr14" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 14. Examples of modified nucleotides. Modification of the carbohydrate: MOE = methoxyethyl; cEt = constrained ethyl; LNA = locked nucleic acid. The nucleobase is represented in green, and the replacement of the phosphate group by a thiophosphate is highlighted in light blue.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b00378/20181019/images/large/jm-2018-00378e_0014.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00378&amp;id=fig14"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig15" data-index="15" class="article__inlineFigure"><h2 class="fig-label">Figure 15</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b00378/20181019/images/medium/jm-2018-00378e_0015.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b00378/20181019/images/large/jm-2018-00378e_0015.jpeg" id="rightTab-gr15" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 15. Examples of small molecules binders to RNA.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b00378/20181019/images/large/jm-2018-00378e_0015.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00378&amp;id=fig15"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig16" data-index="16" class="article__inlineFigure"><h2 class="fig-label">Figure 16</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b00378/20181019/images/medium/jm-2018-00378e_0016.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b00378/20181019/images/large/jm-2018-00378e_0016.jpeg" id="rightTab-gr16" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 16. Principle of 2D combinatorial libraries and inforna.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b00378/20181019/images/large/jm-2018-00378e_0016.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00378&amp;id=fig16"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig17" data-index="17" class="article__inlineFigure"><h2 class="fig-label">Figure 17</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b00378/20181019/images/medium/jm-2018-00378e_0017.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b00378/20181019/images/large/jm-2018-00378e_0017.jpeg" id="rightTab-gr17" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 17. Principle of PROTAC.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b00378/20181019/images/large/jm-2018-00378e_0017.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00378&amp;id=fig17"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig18" data-index="18" class="article__inlineFigure"><h2 class="fig-label">Figure 18</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b00378/20181019/images/medium/jm-2018-00378e_0018.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b00378/20181019/images/large/jm-2018-00378e_0018.jpeg" id="rightTab-gr18" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 18. Examples of PROTACs. Color coding for amino acids: purple = <span class="smallcaps smallerCapital">d</span>-amino acid.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b00378/20181019/images/large/jm-2018-00378e_0018.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00378&amp;id=fig18"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig19" data-index="19" class="article__inlineFigure"><h2 class="fig-label">Figure 19</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b00378/20181019/images/medium/jm-2018-00378e_0019.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b00378/20181019/images/large/jm-2018-00378e_0019.jpeg" id="rightTab-gr19" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 19. Mode-of-action centric hit finding. DNA is transcribed into RNA, which is then translated into either an extracellular/membrane-bound protein or an intracellular protein. When the target modulation is agnostic and/or the target is unknown, phenotypic screening may be used directly as illustrated within the first level oval.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b00378/20181019/images/large/jm-2018-00378e_0019.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00378&amp;id=fig19"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig20" data-index="20" class="article__inlineFigure"><h2 class="fig-label">Figure 20</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b00378/20181019/images/medium/jm-2018-00378e_0020.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b00378/20181019/images/large/jm-2018-00378e_0020.jpeg" id="rightTab-gr20" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 20. Heat map for the strength and limitations of hit finding strategies and their associated modalities. *: Cell and tissue specific. P: Requires partnership with specialized company.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b00378/20181019/images/large/jm-2018-00378e_0020.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00378&amp;id=fig20"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig21" data-index="21" class="article__inlineFigure"><h2 class="fig-label">Figure 21</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b00378/20181019/images/medium/jm-2018-00378e_0021.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b00378/20181019/images/large/jm-2018-00378e_0021.jpeg" id="rightTab-gr21" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 21. Example of phased hit finding waves integrating multiple strategies and modalities.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2018/jmcmar.2018.61.issue-20/acs.jmedchem.8b00378/20181019/images/large/jm-2018-00378e_0021.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b00378&amp;id=fig21"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i35">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_46020" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_46020" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 232 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Verdine, G. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walensky, L. D.</span></span> <span> </span><span class="NLM_article-title">The Challenge of Drugging Undruggable Targets in Cancer: Lessons Learned from Targeting BCL-2 Family Members</span>. <i>Clin. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">7264</span>– <span class="NLM_lpage">7270</span>, <span class="refDoi"> DOI: 10.1158/1078-0432.CCR-07-2184</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1158%2F1078-0432.CCR-07-2184" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=18094406" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhsVCktr%252FK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2007&pages=7264-7270&author=G.+L.+Verdineauthor=L.+D.+Walensky&title=The+Challenge+of+Drugging+Undruggable+Targets+in+Cancer%3A+Lessons+Learned+from+Targeting+BCL-2+Family+Members&doi=10.1158%2F1078-0432.CCR-07-2184"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">The Challenge of Drugging Undruggable Targets in Cancer: Lessons Learned from Targeting BCL-2 Family Members</span></div><div class="casAuthors">Verdine, Gregory L.; Walensky, Loren D.</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">7264-7270</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">A review.  The genomic and proteomic revolutions have provided us with an ever-increasing no. of mechanistic insights into cancer pathogenesis.  Mutated genes and pathol. protein products have emerged as the basis for modern anticancer drug development.  With the increasing realization of the importance of disrupting oncogenic protein-protein interaction, new challenges have emerged for classical small mol. and protein-based drug modalities, i.e., the crit. need to target flat and extended protein surfaces.  Here, we highlight two distinct technologies that are being used to bridge the pharmacol. gap between small mols. and protein therapeutics.  With the BCL-2 family of survival proteins as their substrate for intracellular targeting, we conclude that peptide stapling and fragment-based drug discovery show promise to traverse the crit. surface features of proteins that drive human cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrgYQX3c_4_qLVg90H21EOLACvtfcHk0lhTf5fPUi55xA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhsVCktr%252FK&md5=1b7fcbe41063e04470f7c34b50410a42</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1158%2F1078-0432.CCR-07-2184&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1078-0432.CCR-07-2184%26sid%3Dliteratum%253Aachs%26aulast%3DVerdine%26aufirst%3DG.%2BL.%26aulast%3DWalensky%26aufirst%3DL.%2BD.%26atitle%3DThe%2520Challenge%2520of%2520Drugging%2520Undruggable%2520Targets%2520in%2520Cancer%253A%2520Lessons%2520Learned%2520from%2520Targeting%2520BCL-2%2520Family%2520Members%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D2007%26volume%3D13%26spage%3D7264%26epage%3D7270%26doi%3D10.1158%2F1078-0432.CCR-07-2184" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dugger, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Platt, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goldstein, D. B.</span></span> <span> </span><span class="NLM_article-title">Drug Development in the Era of Precision Medicine</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">183</span>– <span class="NLM_lpage">196</span>, <span class="refDoi"> DOI: 10.1038/nrd.2017.226</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1038%2Fnrd.2017.226" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=29217837" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvFCjur%252FM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2018&pages=183-196&author=S.+A.+Duggerauthor=A.+Plattauthor=D.+B.+Goldstein&title=Drug+Development+in+the+Era+of+Precision+Medicine&doi=10.1038%2Fnrd.2017.226"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Drug development in the era of precision medicine</span></div><div class="casAuthors">Dugger, Sarah A.; Platt, Adam; Goldstein, David B.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">183-196</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">A review.  For the past three decades, the use of genomics to inform drug discovery and development pipelines has generated both excitement and scepticism.  Although earlier efforts successfully identified some new drug targets, the overall clin. efficacy of developed drugs has remained unimpressive, owing in large part to the heterogeneous causes of disease.  Recent technol. and anal. advances in genomics, however, have now made it possible to rapidly identify and interpret the genetic variation underlying a single patient's disease, thereby providing a window into patient-specific mechanisms that cause or contribute to disease, which could ultimately enable the 'precise' targeting of these mechanisms.  Here, we first examine and highlight the successes and limitations of the earlier phases of genomics in drug discovery and development.  We then review the current major efforts in precision medicine and discuss the potential broader utility of mechanistically guided treatments going forward.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpshWBfUfoWf7Vg90H21EOLACvtfcHk0lhJnDFtWrj2Uw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvFCjur%252FM&md5=e06762106391d7646c686e6ef773398a</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2017.226&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2017.226%26sid%3Dliteratum%253Aachs%26aulast%3DDugger%26aufirst%3DS.%2BA.%26aulast%3DPlatt%26aufirst%3DA.%26aulast%3DGoldstein%26aufirst%3DD.%2BB.%26atitle%3DDrug%2520Development%2520in%2520the%2520Era%2520of%2520Precision%2520Medicine%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2018%26volume%3D17%26spage%3D183%26epage%3D196%26doi%3D10.1038%2Fnrd.2017.226" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hart, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tong, A. H. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Leeuwen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seetharaman, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aregger, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chandrashekhar, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hustedt, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seth, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Noonan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Habsid, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sizova, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nedyalkova, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Climie, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tworzyanski, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lawson, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sartori, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alibeh, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tieu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Masud, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mero, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weiss, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Usaj, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Billmann, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rahman, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Costanzo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Myers, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andrews, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boone, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Durocher, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moffat, J.</span></span> <span> </span><span class="NLM_article-title">Evaluation and Design of Genome-Wide CRISPR/SpCas9 Knockout Screens</span>. <i>G3: Genes, Genomes, Genet.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">2719</span>– <span class="NLM_lpage">2727</span>, <span class="refDoi"> DOI: 10.1534/g3.117.041277</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1534%2Fg3.117.041277" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2017&pages=2719-2727&author=T.+Hartauthor=A.+H.+Y.+Tongauthor=K.+Chanauthor=J.+Van+Leeuwenauthor=A.+Seetharamanauthor=M.+Areggerauthor=M.+Chandrashekharauthor=N.+Hustedtauthor=S.+Sethauthor=A.+Noonanauthor=A.+Habsidauthor=O.+Sizovaauthor=L.+Nedyalkovaauthor=R.+Climieauthor=L.+Tworzyanskiauthor=K.+Lawsonauthor=M.+A.+Sartoriauthor=S.+Alibehauthor=D.+Tieuauthor=S.+Masudauthor=P.+Meroauthor=A.+Weissauthor=K.+R.+Brownauthor=M.+Usajauthor=M.+Billmannauthor=M.+Rahmanauthor=M.+Costanzoauthor=C.+L.+Myersauthor=B.+J.+Andrewsauthor=C.+Booneauthor=D.+Durocherauthor=J.+Moffat&title=Evaluation+and+Design+of+Genome-Wide+CRISPR%2FSpCas9+Knockout+Screens&doi=10.1534%2Fg3.117.041277"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1534%2Fg3.117.041277&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1534%252Fg3.117.041277%26sid%3Dliteratum%253Aachs%26aulast%3DHart%26aufirst%3DT.%26aulast%3DTong%26aufirst%3DA.%2BH.%2BY.%26aulast%3DChan%26aufirst%3DK.%26aulast%3DVan%2BLeeuwen%26aufirst%3DJ.%26aulast%3DSeetharaman%26aufirst%3DA.%26aulast%3DAregger%26aufirst%3DM.%26aulast%3DChandrashekhar%26aufirst%3DM.%26aulast%3DHustedt%26aufirst%3DN.%26aulast%3DSeth%26aufirst%3DS.%26aulast%3DNoonan%26aufirst%3DA.%26aulast%3DHabsid%26aufirst%3DA.%26aulast%3DSizova%26aufirst%3DO.%26aulast%3DNedyalkova%26aufirst%3DL.%26aulast%3DClimie%26aufirst%3DR.%26aulast%3DTworzyanski%26aufirst%3DL.%26aulast%3DLawson%26aufirst%3DK.%26aulast%3DSartori%26aufirst%3DM.%2BA.%26aulast%3DAlibeh%26aufirst%3DS.%26aulast%3DTieu%26aufirst%3DD.%26aulast%3DMasud%26aufirst%3DS.%26aulast%3DMero%26aufirst%3DP.%26aulast%3DWeiss%26aufirst%3DA.%26aulast%3DBrown%26aufirst%3DK.%2BR.%26aulast%3DUsaj%26aufirst%3DM.%26aulast%3DBillmann%26aufirst%3DM.%26aulast%3DRahman%26aufirst%3DM.%26aulast%3DCostanzo%26aufirst%3DM.%26aulast%3DMyers%26aufirst%3DC.%2BL.%26aulast%3DAndrews%26aufirst%3DB.%2BJ.%26aulast%3DBoone%26aufirst%3DC.%26aulast%3DDurocher%26aufirst%3DD.%26aulast%3DMoffat%26aufirst%3DJ.%26atitle%3DEvaluation%2520and%2520Design%2520of%2520Genome-Wide%2520CRISPR%252FSpCas9%2520Knockout%2520Screens%26jtitle%3DG3%253A%2520Genes%252C%2520Genomes%252C%2520Genet.%26date%3D2017%26volume%3D7%26spage%3D2719%26epage%3D2727%26doi%3D10.1534%2Fg3.117.041277" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Disney, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Angelbello, A. J.</span></span> <span> </span><span class="NLM_article-title">Rational Design of Small Molecules Targeting Oncogenic Noncoding RNAs from Sequence</span>. <i>Acc. Chem. Res.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>49</i></span>,  <span class="NLM_fpage">2698</span>– <span class="NLM_lpage">2704</span>, <span class="refDoi"> DOI: 10.1021/acs.accounts.6b00326</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.accounts.6b00326" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvV2ht7nJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2016&pages=2698-2704&author=M.+D.+Disneyauthor=A.+J.+Angelbello&title=Rational+Design+of+Small+Molecules+Targeting+Oncogenic+Noncoding+RNAs+from+Sequence&doi=10.1021%2Facs.accounts.6b00326"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Rational Design of Small Molecules Targeting Oncogenic Noncoding RNAs from Sequence</span></div><div class="casAuthors">Disney, Matthew D.; Angelbello, Alicia J.</div><div class="citationInfo"><span class="NLM_cas:title">Accounts of Chemical Research</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2698-2704</span>CODEN:
                <span class="NLM_cas:coden">ACHRE4</span>;
        ISSN:<span class="NLM_cas:issn">0001-4842</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The discovery of RNA catalysis in the 1980s and the dissemination of the human genome sequence at the start of this century inspired investigations of the regulatory roles of noncoding RNAs in biol.  In fact, the Encyclopedia of DNA Elements (ENCODE) project has shown that only 1-2% of the human genome encodes protein, yet 75% is transcribed into RNA.  Functional studies both preceding and following the ENCODE project have shown that these noncoding RNAs have important roles that regulate gene expression, developmental timing, and other crit. functions.  RNA's diverse roles are often a consequence of the various folds that it adopts.  The single stranded nature of the biopolymer enables it to adopt intramol. folds with noncanonical pairings to lower its free energy (Figure ).  These folds can be scaffolds to bind proteins or to form frameworks to interact with other RNAs.  Not surprisingly, dysregulation of certain noncoding RNAs has been shown to be causative of disease.  Given this as a background, it is easy to see why it would be useful to develop methods to target RNA and manipulate its biol. in rational and predictable ways.  The antisense approach has afforded strategies to target RNAs via Watson-Crick based pairing and has typically focused on targeting partially unstructured regions of RNA.  Small mol. strategies to target RNA would be desirable, not only because compds. could be lead optimized via medicinal chem., but also because structured regions within an RNA of interest could be targeted in order to directly interfere with RNA folds that contribute to disease.  Addnl., small mols. have historically been the most successful drug candidates.  Until recently, the ability to design small mols. that target nonrRNAs has been elusive, creating the perception that they are "undruggable".  In this Account, approaches to demystify targeting RNA with small mols. are described.  Rather than bulk screening for compds. that bind to singular targets, which is the purview of the pharmaceutical industry and academic institutions with high throughput screening facilities, we focus on methods that allow for the rational design of small mols. toward biol. RNAs.  Enabling and foundational technologies that have been developed include two-dimensional combinatorial screening (2DCS), a library-vs.-library selection approach that allows the identification of the RNA motif binding preferences of small mols. from millions of combinations.  A landscape map of the 2DCS-defined and annotated RNA motif-small mol. interactions is then placed into Inforna, a computational tool that allows one to mine these interactions against an RNA of interest or an entire transcriptome.  Indeed, this approach has been enabled by tools to annotate RNA structure from sequence-an invaluable asset to the RNA community and this work-and has allowed for the rational identification of "druggable" RNAs in a target agnostic fashion.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpU-8j6q0xYYrVg90H21EOLACvtfcHk0lhJnDFtWrj2Uw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvV2ht7nJ&md5=14a75becddc19fd360e9b5dbd94eec48</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1021%2Facs.accounts.6b00326&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.accounts.6b00326%26sid%3Dliteratum%253Aachs%26aulast%3DDisney%26aufirst%3DM.%2BD.%26aulast%3DAngelbello%26aufirst%3DA.%2BJ.%26atitle%3DRational%2520Design%2520of%2520Small%2520Molecules%2520Targeting%2520Oncogenic%2520Noncoding%2520RNAs%2520from%2520Sequence%26jtitle%3DAcc.%2520Chem.%2520Res.%26date%3D2016%26volume%3D49%26spage%3D2698%26epage%3D2704%26doi%3D10.1021%2Facs.accounts.6b00326" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Valeur, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guéret, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adihou, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gopalakrishnan, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lemurell, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waldmann, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grossmann, T. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plowright, A. T.</span></span> <span> </span><span class="NLM_article-title">New Modalities for Challenging Targets in Drug Discovery</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">10294</span>– <span class="NLM_lpage">10323</span>, <span class="refDoi"> DOI: 10.1002/anie.201611914</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1002%2Fanie.201611914" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC2sXht1WlsLvP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2017&pages=10294-10323&author=E.+Valeurauthor=S.+M.+Gu%C3%A9retauthor=H.+Adihouauthor=R.+Gopalakrishnanauthor=M.+Lemurellauthor=H.+Waldmannauthor=T.+N.+Grossmannauthor=A.+T.+Plowright&title=New+Modalities+for+Challenging+Targets+in+Drug+Discovery&doi=10.1002%2Fanie.201611914"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">New Modalities for Challenging Targets in Drug Discovery</span></div><div class="casAuthors">Valeur, Eric; Gueret, Stephanie M.; Adihou, Helene; Gopalakrishnan, Ranganath; Lemurell, Malin; Waldmann, Herbert; Grossmann, Tom N.; Plowright, Alleyn T.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">35</span>),
    <span class="NLM_cas:pages">10294-10323</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A review.  The ever-increasing understanding of biol. systems is providing a range of exciting novel biol. targets, whose modulation may enable novel therapeutic options for many diseases.  These targets include protein-protein and protein-nucleic acid interactions, which are, however, often refractory to classical small-mol. approaches.  Other types of mols., or modalities, are therefore required to address these targets, which has led several academic research groups and pharmaceutical companies to increasingly use the concept of so-called "new modalities".  This review defines for the first time the scope of this term, which includes novel peptidic scaffolds, oligonucleotides, hybrids, mol. conjugates, as well as new uses of classical small mols.  The authors provide the most representative examples of these modalities to target large binding surface areas such as those found in protein-protein interactions and for biol. processes at the center of cell regulation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp3BpAx5oKYa7Vg90H21EOLACvtfcHk0lhJnDFtWrj2Uw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXht1WlsLvP&md5=e09ef5bb8dbbf09c05b1a2340aa431ec</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1002%2Fanie.201611914&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201611914%26sid%3Dliteratum%253Aachs%26aulast%3DValeur%26aufirst%3DE.%26aulast%3DGu%25C3%25A9ret%26aufirst%3DS.%2BM.%26aulast%3DAdihou%26aufirst%3DH.%26aulast%3DGopalakrishnan%26aufirst%3DR.%26aulast%3DLemurell%26aufirst%3DM.%26aulast%3DWaldmann%26aufirst%3DH.%26aulast%3DGrossmann%26aufirst%3DT.%2BN.%26aulast%3DPlowright%26aufirst%3DA.%2BT.%26atitle%3DNew%2520Modalities%2520for%2520Challenging%2520Targets%2520in%2520Drug%2520Discovery%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2017%26volume%3D56%26spage%3D10294%26epage%3D10323%26doi%3D10.1002%2Fanie.201611914" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bunnage, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilbert, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, L. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hett, E. C.</span></span> <span> </span><span class="NLM_article-title">Know Your Target, Know Your Molecule</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">368</span>– <span class="NLM_lpage">372</span>, <span class="refDoi"> DOI: 10.1038/nchembio.1813</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1038%2Fnchembio.1813" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=25978985" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtFemtrvK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2015&pages=368-372&author=M.+E.+Bunnageauthor=A.+M.+Gilbertauthor=L.+H.+Jonesauthor=E.+C.+Hett&title=Know+Your+Target%2C+Know+Your+Molecule&doi=10.1038%2Fnchembio.1813"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Know your target, know your molecule</span></div><div class="casAuthors">Bunnage, Mark E.; Gilbert, Adam M.; Jones, Lyn H.; Hett, Erik C.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">368-372</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review. The pharmaceutical industry continues to experience significant attrition of drug candidates during phase 2 proof-of-concept clin. studies.  We describe some questions about the characteristics of protein targets and small-mol. drugs that may be important to consider in drug-discovery projects and could improve prospects for future clin. success.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpRthCFSFvOXrVg90H21EOLACvtfcHk0liMcBJt0L6riw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtFemtrvK&md5=f9606135ccea6af432040d8afb630ede</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1813&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1813%26sid%3Dliteratum%253Aachs%26aulast%3DBunnage%26aufirst%3DM.%2BE.%26aulast%3DGilbert%26aufirst%3DA.%2BM.%26aulast%3DJones%26aufirst%3DL.%2BH.%26aulast%3DHett%26aufirst%3DE.%2BC.%26atitle%3DKnow%2520Your%2520Target%252C%2520Know%2520Your%2520Molecule%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2015%26volume%3D11%26spage%3D368%26epage%3D372%26doi%3D10.1038%2Fnchembio.1813" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fellmann, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gowen, B. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, P.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doudna, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Corn, J. E.</span></span> <span> </span><span class="NLM_article-title">Cornerstones of CRISPR–Cas in Drug Discovery and Therapy</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">89</span>– <span class="NLM_lpage">100</span>, <span class="refDoi"> DOI: 10.1038/nrd.2016.238</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1038%2Fnrd.2016.238" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2017&pages=89-100&author=C.+Fellmannauthor=B.+G.+Gowenauthor=P.-C.+Linauthor=J.+A.+Doudnaauthor=J.+E.+Corn&title=Cornerstones+of+CRISPR%E2%80%93Cas+in+Drug+Discovery+and+Therapy&doi=10.1038%2Fnrd.2016.238"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2016.238&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2016.238%26sid%3Dliteratum%253Aachs%26aulast%3DFellmann%26aufirst%3DC.%26aulast%3DGowen%26aufirst%3DB.%2BG.%26aulast%3DLin%26aufirst%3DP.-C.%26aulast%3DDoudna%26aufirst%3DJ.%2BA.%26aulast%3DCorn%26aufirst%3DJ.%2BE.%26atitle%3DCornerstones%2520of%2520CRISPR%25E2%2580%2593Cas%2520in%2520Drug%2520Discovery%2520and%2520Therapy%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2017%26volume%3D16%26spage%3D89%26epage%3D100%26doi%3D10.1038%2Fnrd.2016.238" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sander, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Joung, J. K.</span></span> <span> </span><span class="NLM_article-title">CRISPR-Cas Systems for Editing, Regulating and Targeting Genomes</span>. <i>Nat. Biotechnol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>32</i></span>,  <span class="NLM_fpage">347</span>– <span class="NLM_lpage">355</span>, <span class="refDoi"> DOI: 10.1038/nbt.2842</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1038%2Fnbt.2842" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=24584096" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC2cXjtlyrsLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2014&pages=347-355&author=J.+D.+Sanderauthor=J.+K.+Joung&title=CRISPR-Cas+Systems+for+Editing%2C+Regulating+and+Targeting+Genomes&doi=10.1038%2Fnbt.2842"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">CRISPR-Cas systems for editing, regulating and targeting genomes</span></div><div class="casAuthors">Sander, Jeffry D.; Joung, J. Keith</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">32</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">347-355</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Targeted genome editing using engineered nucleases has rapidly gone from being a niche technol. to a mainstream method used by many biol. researchers.  This widespread adoption has been largely fueled by the emergence of the clustered, regularly interspaced, short palindromic repeat (CRISPR) technol., an important new approach for generating RNA-guided nucleases, such as Cas9, with customizable specificities.  Genome editing mediated by these nucleases has been used to rapidly, easily and efficiently modify endogenous genes in a wide variety of biomedically important cell types and in organisms that have traditionally been challenging to manipulate genetically.  Furthermore, a modified version of the CRISPR-Cas9 system was developed to recruit heterologous domains that can regulate endogenous gene expression or label specific genomic loci in living cells.  Although the genome-wide specificities of CRISPR-Cas9 systems remain to be fully defined, the power of these systems to perform targeted, highly efficient alterations of genome sequence and gene expression will undoubtedly transform biol. research and spur the development of novel mol. therapeutics for human disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp4VCoNVtj1fLVg90H21EOLACvtfcHk0liMcBJt0L6riw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXjtlyrsLo%253D&md5=7be62d2d37b419010fb8bec273731c07</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1038%2Fnbt.2842&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt.2842%26sid%3Dliteratum%253Aachs%26aulast%3DSander%26aufirst%3DJ.%2BD.%26aulast%3DJoung%26aufirst%3DJ.%2BK.%26atitle%3DCRISPR-Cas%2520Systems%2520for%2520Editing%252C%2520Regulating%2520and%2520Targeting%2520Genomes%26jtitle%3DNat.%2520Biotechnol.%26date%3D2014%26volume%3D32%26spage%3D347%26epage%3D355%26doi%3D10.1038%2Fnbt.2842" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cox, D. B. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Platt, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, F.</span></span> <span> </span><span class="NLM_article-title">Therapeutic Genome Editing: Prospects and Challenges</span>. <i>Nat. Med.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">121</span>– <span class="NLM_lpage">131</span>, <span class="refDoi"> DOI: 10.1038/nm.3793</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1038%2Fnm.3793" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=25654603" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC2MXit1artbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2015&pages=121-131&author=D.+B.+T.+Coxauthor=R.+J.+Plattauthor=F.+Zhang&title=Therapeutic+Genome+Editing%3A+Prospects+and+Challenges&doi=10.1038%2Fnm.3793"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Therapeutic genome editing: prospects and challenges</span></div><div class="casAuthors">Cox, David Benjamin Turitz; Platt, Randall Jeffrey; Zhang, Feng</div><div class="citationInfo"><span class="NLM_cas:title">Nature Medicine (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">121-131</span>CODEN:
                <span class="NLM_cas:coden">NAMEFI</span>;
        ISSN:<span class="NLM_cas:issn">1078-8956</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Recent advances in the development of genome editing technologies based on programmable nucleases have substantially improved our ability to make precise changes in the genomes of eukaryotic cells.  Genome editing is already broadening our ability to elucidate the contribution of genetics to disease by facilitating the creation of more accurate cellular and animal models of pathol. processes.  A particularly tantalizing application of programmable nucleases is the potential to directly correct genetic mutations in affected tissues and cells to treat diseases that are refractory to traditional therapies.  Here we discuss current progress toward developing programmable nuclease-based therapies as well as future prospects and challenges.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp0EN1KkwY_dLVg90H21EOLACvtfcHk0liMcBJt0L6riw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXit1artbk%253D&md5=352473152693168e492dbdc6410cebb0</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1038%2Fnm.3793&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnm.3793%26sid%3Dliteratum%253Aachs%26aulast%3DCox%26aufirst%3DD.%2BB.%2BT.%26aulast%3DPlatt%26aufirst%3DR.%2BJ.%26aulast%3DZhang%26aufirst%3DF.%26atitle%3DTherapeutic%2520Genome%2520Editing%253A%2520Prospects%2520and%2520Challenges%26jtitle%3DNat.%2520Med.%26date%3D2015%26volume%3D21%26spage%3D121%26epage%3D131%26doi%3D10.1038%2Fnm.3793" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cox, D. B. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gootenberg, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abudayyeh, O. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Franklin, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kellner, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Joung, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, F.</span></span> <span> </span><span class="NLM_article-title">RNA Editing with CRISPR-Cas13</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>358</i></span>,  <span class="NLM_fpage">1019</span>, <span class="refDoi"> DOI: 10.1126/science.aaq0180</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1126%2Fscience.aaq0180" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=29070703" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvFahsb3J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=358&publication_year=2017&pages=1019&author=D.+B.+T.+Coxauthor=J.+S.+Gootenbergauthor=O.+O.+Abudayyehauthor=B.+Franklinauthor=M.+J.+Kellnerauthor=J.+Joungauthor=F.+Zhang&title=RNA+Editing+with+CRISPR-Cas13&doi=10.1126%2Fscience.aaq0180"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">RNA editing with CRISPR-Cas13</span></div><div class="casAuthors">Cox, David B. T.; Gootenberg, Jonathan S.; Abudayyeh, Omar O.; Franklin, Brian; Kellner, Max J.; Joung, Julia; Zhang, Feng</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">358</span>
        (<span class="NLM_cas:issue">6366</span>),
    <span class="NLM_cas:pages">1019-1027</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Nucleic acid editing holds promise for treating genetic disease, particularly at the RNA level, where disease-relevant sequences can be rescued to yield functional protein products.  Type VI CRISPR-Cas systems contain the programmable single-effector RNA-guided RNase Cas13.  We profiled type VI systems in order to engineer a Cas13 ortholog capable of robust knockdown and demonstrated RNA editing by using catalytically inactive Cas13 (dCas13) to direct adenosine-to-inosine deaminase activity by ADAR2 (adenosine deaminase acting on RNA type 2) to transcripts in mammalian cells.  This system, referred to as RNA editing for programmable A to I replacement (REPAIR), which has no strict sequence constraints, can be used to edit full-length transcripts contg. pathogenic mutations.  We further engineered this system to create a high-specificity variant and minimized the system to facilitate viral delivery.  REPAIR presents a promising RNA-editing platform with broad applicability for research, therapeutics, and biotechnol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrlQ-Cv1-eMdbVg90H21EOLACvtfcHk0ljDHpTPqPrIig"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvFahsb3J&md5=21ed8ebbd2cc334be0911129196e6b6e</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1126%2Fscience.aaq0180&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.aaq0180%26sid%3Dliteratum%253Aachs%26aulast%3DCox%26aufirst%3DD.%2BB.%2BT.%26aulast%3DGootenberg%26aufirst%3DJ.%2BS.%26aulast%3DAbudayyeh%26aufirst%3DO.%2BO.%26aulast%3DFranklin%26aufirst%3DB.%26aulast%3DKellner%26aufirst%3DM.%2BJ.%26aulast%3DJoung%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DF.%26atitle%3DRNA%2520Editing%2520with%2520CRISPR-Cas13%26jtitle%3DScience%26date%3D2017%26volume%3D358%26spage%3D1019%26doi%3D10.1126%2Fscience.aaq0180" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">An, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schneller, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frassetto, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, J.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Theisen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hong, S.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rajendran, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Levy, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Howell, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Besin, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Presnyak, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sabnis, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murphy-Benenato, K. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumarasinghe, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salerno, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mihai, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lukacs, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chandler, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guey, L. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Venditti, C. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martini, P. G. V.</span></span> <span> </span><span class="NLM_article-title">Systemic Messenger RNA Therapy as a Treatment for Methylmalonic Acidemia</span>. <i>Cell Rep.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">3548</span>– <span class="NLM_lpage">3558</span>, <span class="refDoi"> DOI: 10.1016/j.celrep.2017.11.081</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1016%2Fj.celrep.2017.11.081" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=29262333" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC2sXitVWlsrbP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2017&pages=3548-3558&author=D.+Anauthor=J.+L.+Schnellerauthor=A.+Frassettoauthor=S.+Liangauthor=X.+Zhuauthor=J.-S.+Parkauthor=M.+Theisenauthor=S.-J.+Hongauthor=J.+Zhouauthor=R.+Rajendranauthor=B.+Levyauthor=R.+Howellauthor=G.+Besinauthor=V.+Presnyakauthor=S.+Sabnisauthor=K.+E.+Murphy-Benenatoauthor=E.+S.+Kumarasingheauthor=T.+Salernoauthor=C.+Mihaiauthor=C.+M.+Lukacsauthor=R.+J.+Chandlerauthor=L.+T.+Gueyauthor=C.+P.+Vendittiauthor=P.+G.+V.+Martini&title=Systemic+Messenger+RNA+Therapy+as+a+Treatment+for+Methylmalonic+Acidemia&doi=10.1016%2Fj.celrep.2017.11.081"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Systemic Messenger RNA Therapy as a Treatment for Methylmalonic Acidemia</span></div><div class="casAuthors">An, Ding; Schneller, Jessica L.; Frassetto, Andrea; Liang, Shi; Zhu, Xuling; Park, Ji-Sun; Theisen, Matt; Hong, Sue-Jean; Zhou, Jenny; Rajendran, Raj; Levy, Becca; Howell, Rebecca; Besin, Gilles; Presnyak, Vladimir; Sabnis, Staci; Murphy-Benenato, Kerry E.; Kumarasinghe, E. Sathyajith; Salerno, Timothy; Mihai, Cosmin; Lukacs, Christine M.; Chandler, Randy J.; Guey, Lin T.; Venditti, Charles P.; Martini, Paolo G. V.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Reports</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">3548-3558</span>CODEN:
                <span class="NLM_cas:coden">CREED8</span>;
        ISSN:<span class="NLM_cas:issn">2211-1247</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Isolated methylmalonic acidemia/aciduria (MMA) is a devastating metabolic disorder with poor outcomes despite current medical treatments.  Like other mitochondrial enzymopathies, enzyme replacement therapy (ERT) is not available, and although promising, AAV gene therapy can be limited by pre-existing immunity and has been assocd. with genotoxicity in mice.  To develop a new class of therapy for MMA, we generated a pseudoU-modified codon-optimized mRNA encoding human methylmalonyl-CoA mutase (hMUT), the enzyme most frequently mutated in MMA, and encapsulated it into biodegradable lipid nanoparticles (LNPs).  I.v. (i.v.) administration of hMUT mRNA in two different mouse models of MMA resulted in a 75%-85% redn. in plasma methylmalonic acid and was assocd. with increased hMUT protein expression and activity in liver.  Repeat dosing of hMUT mRNA reduced circulating metabolites and dramatically improved survival and wt. gain.  Addnl., repeat i.v. dosing did not increase markers of liver toxicity or inflammation in heterozygote MMA mice.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqf0QTenUCp_rVg90H21EOLACvtfcHk0ljDHpTPqPrIig"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXitVWlsrbP&md5=1bd1d194653ce58f22f92e8aa77c7004</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1016%2Fj.celrep.2017.11.081&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.celrep.2017.11.081%26sid%3Dliteratum%253Aachs%26aulast%3DAn%26aufirst%3DD.%26aulast%3DSchneller%26aufirst%3DJ.%2BL.%26aulast%3DFrassetto%26aufirst%3DA.%26aulast%3DLiang%26aufirst%3DS.%26aulast%3DZhu%26aufirst%3DX.%26aulast%3DPark%26aufirst%3DJ.-S.%26aulast%3DTheisen%26aufirst%3DM.%26aulast%3DHong%26aufirst%3DS.-J.%26aulast%3DZhou%26aufirst%3DJ.%26aulast%3DRajendran%26aufirst%3DR.%26aulast%3DLevy%26aufirst%3DB.%26aulast%3DHowell%26aufirst%3DR.%26aulast%3DBesin%26aufirst%3DG.%26aulast%3DPresnyak%26aufirst%3DV.%26aulast%3DSabnis%26aufirst%3DS.%26aulast%3DMurphy-Benenato%26aufirst%3DK.%2BE.%26aulast%3DKumarasinghe%26aufirst%3DE.%2BS.%26aulast%3DSalerno%26aufirst%3DT.%26aulast%3DMihai%26aufirst%3DC.%26aulast%3DLukacs%26aufirst%3DC.%2BM.%26aulast%3DChandler%26aufirst%3DR.%2BJ.%26aulast%3DGuey%26aufirst%3DL.%2BT.%26aulast%3DVenditti%26aufirst%3DC.%2BP.%26aulast%3DMartini%26aufirst%3DP.%2BG.%2BV.%26atitle%3DSystemic%2520Messenger%2520RNA%2520Therapy%2520as%2520a%2520Treatment%2520for%2520Methylmalonic%2520Acidemia%26jtitle%3DCell%2520Rep.%26date%3D2017%26volume%3D21%26spage%3D3548%26epage%3D3558%26doi%3D10.1016%2Fj.celrep.2017.11.081" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zangi, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lui, K. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">von Gise, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ebina, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ptaszek, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Später, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tabebordbar, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gorbatov, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sena, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nahrendorf, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Briscoe, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wagers, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rossi, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pu, W. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chien, K. R.</span></span> <span> </span><span class="NLM_article-title">Modified mRNA Directs the Fate of Heart Progenitor Cells and Induces Vascular Regeneration after Myocardial Infarction</span>. <i>Nat. Biotechnol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>31</i></span>,  <span class="NLM_fpage">898</span>– <span class="NLM_lpage">907</span>, <span class="refDoi"> DOI: 10.1038/nbt.2682</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1038%2Fnbt.2682" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=24013197" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtl2ksbbF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2013&pages=898-907&author=L.+Zangiauthor=K.+O.+Luiauthor=A.+von+Giseauthor=Q.+Maauthor=W.+Ebinaauthor=L.+M.+Ptaszekauthor=D.+Sp%C3%A4terauthor=H.+Xuauthor=M.+Tabebordbarauthor=R.+Gorbatovauthor=B.+Senaauthor=M.+Nahrendorfauthor=D.+M.+Briscoeauthor=R.+A.+Liauthor=A.+J.+Wagersauthor=D.+J.+Rossiauthor=W.+T.+Puauthor=K.+R.+Chien&title=Modified+mRNA+Directs+the+Fate+of+Heart+Progenitor+Cells+and+Induces+Vascular+Regeneration+after+Myocardial+Infarction&doi=10.1038%2Fnbt.2682"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Modified mRNA directs the fate of heart progenitor cells and induces vascular regeneration after myocardial infarction</span></div><div class="casAuthors">Zangi, Lior; Lui, Kathy O.; von Gise, Alexander; Ma, Qing; Ebina, Wataru; Ptaszek, Leon M.; Spaeter, Daniela; Xu, Huansheng; Tabebordbar, Mohammadsharif; Gorbatov, Rostic; Sena, Brena; Nahrendorf, Matthias; Briscoe, David M.; Li, Ronald A.; Wagers, Amy J.; Rossi, Derrick J.; Pu, William T.; Chien, Kenneth R.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">898-907</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">In a cell-free approach to regenerative therapeutics, transient application of paracrine factors in vivo could be used to alter the behavior and fate of progenitor cells to achieve sustained clin. benefits.  Here we show that intramyocardial injection of synthetic modified RNA (modRNA) encoding human vascular endothelial growth factor-A (VEGF-A) results in the expansion and directed differentiation of endogenous heart progenitors in a mouse myocardial infarction model.  VEGF-A modRNA markedly improved heart function and enhanced long-term survival of recipients.  This improvement was in part due to mobilization of epicardial progenitor cells and redirection of their differentiation toward cardiovascular cell types.  Direct in vivo comparison with DNA vectors and temporal control with VEGF inhibitors revealed the greatly increased efficacy of pulse-like delivery of VEGF-A.  Our results suggest that modRNA is a versatile approach for expressing paracrine factors as cell fate switches to control progenitor cell fate and thereby enhance long-term organ repair.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrmPXkQ6XhLk7Vg90H21EOLACvtfcHk0ljDHpTPqPrIig"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtl2ksbbF&md5=911543ad4989377e233ec2f0dcbc900d</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1038%2Fnbt.2682&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt.2682%26sid%3Dliteratum%253Aachs%26aulast%3DZangi%26aufirst%3DL.%26aulast%3DLui%26aufirst%3DK.%2BO.%26aulast%3Dvon%2BGise%26aufirst%3DA.%26aulast%3DMa%26aufirst%3DQ.%26aulast%3DEbina%26aufirst%3DW.%26aulast%3DPtaszek%26aufirst%3DL.%2BM.%26aulast%3DSp%25C3%25A4ter%26aufirst%3DD.%26aulast%3DXu%26aufirst%3DH.%26aulast%3DTabebordbar%26aufirst%3DM.%26aulast%3DGorbatov%26aufirst%3DR.%26aulast%3DSena%26aufirst%3DB.%26aulast%3DNahrendorf%26aufirst%3DM.%26aulast%3DBriscoe%26aufirst%3DD.%2BM.%26aulast%3DLi%26aufirst%3DR.%2BA.%26aulast%3DWagers%26aufirst%3DA.%2BJ.%26aulast%3DRossi%26aufirst%3DD.%2BJ.%26aulast%3DPu%26aufirst%3DW.%2BT.%26aulast%3DChien%26aufirst%3DK.%2BR.%26atitle%3DModified%2520mRNA%2520Directs%2520the%2520Fate%2520of%2520Heart%2520Progenitor%2520Cells%2520and%2520Induces%2520Vascular%2520Regeneration%2520after%2520Myocardial%2520Infarction%26jtitle%3DNat.%2520Biotechnol.%26date%3D2013%26volume%3D31%26spage%3D898%26epage%3D907%26doi%3D10.1038%2Fnbt.2682" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Petersen, D. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hawkins, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruangsiriluk, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stevens, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maguire, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Connell, T. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rocke, B. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boehm, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruggeri, R. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rolph, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hepworth, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loria, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carpino, P. A.</span></span> <span> </span><span class="NLM_article-title">A Small-Molecule Anti-Secretagogue of PCSK9 Targets the 80S Ribosome to Inhibit PCSK9 Protein Translation</span>. <i>Cell Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">1362</span>– <span class="NLM_lpage">1371</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2016.08.016</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1016%2Fj.chembiol.2016.08.016" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=27746128" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC28Xhs1yqs7%252FJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2016&pages=1362-1371&author=D.+N.+Petersenauthor=J.+Hawkinsauthor=W.+Ruangsirilukauthor=K.+A.+Stevensauthor=B.+A.+Maguireauthor=T.+N.+O%E2%80%99Connellauthor=B.+N.+Rockeauthor=M.+Boehmauthor=R.+B.+Ruggeriauthor=T.+Rolphauthor=D.+Hepworthauthor=P.+M.+Loriaauthor=P.+A.+Carpino&title=A+Small-Molecule+Anti-Secretagogue+of+PCSK9+Targets+the+80S+Ribosome+to+Inhibit+PCSK9+Protein+Translation&doi=10.1016%2Fj.chembiol.2016.08.016"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">A Small-Molecule Anti-secretagogue of PCSK9 Targets the 80S Ribosome to Inhibit PCSK9 Protein Translation</span></div><div class="casAuthors">Petersen, Donna N.; Hawkins, Julie; Ruangsiriluk, Wanida; Stevens, Kimberly A.; Maguire, Bruce A.; O'Connell, Thomas N.; Rocke, Benjamin N.; Boehm, Markus; Ruggeri, Roger B.; Rolph, Tim; Hepworth, David; Loria, Paula M.; Carpino, Philip A.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1362-1371</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Proprotein convertase subtilisin/kexin type 9 (PCSK9) is a secreted protein that downregulates low-d. lipoprotein (LDL) receptor (LDL-R) levels on the surface of hepatocytes, resulting in decreased clearance of LDL-cholesterol (LDL-C).  Phenotypic screening of a small-mol. compd. collection was used to identify an inhibitor of PCSK9 secretion, (R)-N-(isoquinolin-1-yl)-3-(4-methoxyphenyl)-N-(piperidin-3-yl)propanamide (R-IMPP), which was shown to stimulate uptake of LDL-C in hepatoma cells by increasing LDL-R levels, without altering levels of secreted transferrin.  Systematic investigation of the mode of action revealed that R-IMPP did not decrease PCSK9 transcription or increase PCSK9 degrdn., but instead caused transcript-dependent inhibition of PCSK9 translation.  In support of this surprising mechanism of action, we found that R-IMPP was able to selectively bind to human, but not E. coli, ribosomes.  This study opens a new avenue for the development of drugs that modulate the activity of target proteins by mechanisms involving inhibition of eukaryotic translation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqbEnbBYD6NKbVg90H21EOLACvtfcHk0lj6PbDsn2B1Ww"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xhs1yqs7%252FJ&md5=c375439e6ed7c49daa4625baff8e4e48</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2016.08.016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2016.08.016%26sid%3Dliteratum%253Aachs%26aulast%3DPetersen%26aufirst%3DD.%2BN.%26aulast%3DHawkins%26aufirst%3DJ.%26aulast%3DRuangsiriluk%26aufirst%3DW.%26aulast%3DStevens%26aufirst%3DK.%2BA.%26aulast%3DMaguire%26aufirst%3DB.%2BA.%26aulast%3DO%25E2%2580%2599Connell%26aufirst%3DT.%2BN.%26aulast%3DRocke%26aufirst%3DB.%2BN.%26aulast%3DBoehm%26aufirst%3DM.%26aulast%3DRuggeri%26aufirst%3DR.%2BB.%26aulast%3DRolph%26aufirst%3DT.%26aulast%3DHepworth%26aufirst%3DD.%26aulast%3DLoria%26aufirst%3DP.%2BM.%26aulast%3DCarpino%26aufirst%3DP.%2BA.%26atitle%3DA%2520Small-Molecule%2520Anti-Secretagogue%2520of%2520PCSK9%2520Targets%2520the%252080S%2520Ribosome%2520to%2520Inhibit%2520PCSK9%2520Protein%2520Translation%26jtitle%3DCell%2520Chem.%2520Biol.%26date%3D2016%26volume%3D23%26spage%3D1362%26epage%3D1371%26doi%3D10.1016%2Fj.chembiol.2016.08.016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Churcher, I.</span></span> <span> </span><span class="NLM_article-title">Protac-Induced Protein Degradation in Drug Discovery: Breaking the Rules or Just Making New Ones?</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">444</span>– <span class="NLM_lpage">452</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b01272</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b01272" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvVWqtbvJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=444-452&author=I.+Churcher&title=Protac-Induced+Protein+Degradation+in+Drug+Discovery%3A+Breaking+the+Rules+or+Just+Making+New+Ones%3F&doi=10.1021%2Facs.jmedchem.7b01272"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Protac-Induced Protein Degradation in Drug Discovery: Breaking the Rules or Just Making New Ones?</span></div><div class="casAuthors">Churcher, Ian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">444-452</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Targeted protein degrdn., using bifunctional small mols. (Protacs) to remove specific proteins from within cells, has emerged as a novel drug discovery strategy with the potential to offer therapeutic interventions not achievable with existing approaches.  In this perspective, the brief history of the field is surveyed from a drug discovery perspective with a focus on the key advances in knowledge which have led to the definition and exemplification of protein degrdn. concepts, and their resulting applications to medicine discovery.  The approach has the potential to bring disruptive change to drug discovery - the many potential advantages and outstanding challenges which lie ahead of this technol. are discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr9n8I_0MhtkbVg90H21EOLACvtfcHk0lj6PbDsn2B1Ww"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvVWqtbvJ&md5=e60f1777de8bd860c5001137d8583f1f</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b01272&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b01272%26sid%3Dliteratum%253Aachs%26aulast%3DChurcher%26aufirst%3DI.%26atitle%3DProtac-Induced%2520Protein%2520Degradation%2520in%2520Drug%2520Discovery%253A%2520Breaking%2520the%2520Rules%2520or%2520Just%2520Making%2520New%2520Ones%253F%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D444%26epage%3D452%26doi%3D10.1021%2Facs.jmedchem.7b01272" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rossi, J.</span></span> <span> </span><span class="NLM_article-title">Aptamers as Targeted Therapeutics: Current Potential and Challenges</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">181</span>– <span class="NLM_lpage">202</span>, <span class="refDoi"> DOI: 10.1038/nrd.2016.199</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1038%2Fnrd.2016.199" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=27807347" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC28XhslOqtr3I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2017&pages=181-202&author=J.+Zhouauthor=J.+Rossi&title=Aptamers+as+Targeted+Therapeutics%3A+Current+Potential+and+Challenges&doi=10.1038%2Fnrd.2016.199"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Aptamers as targeted therapeutics: current potential and challenges</span></div><div class="casAuthors">Zhou, Jiehua; Rossi, John</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">181-202</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Nucleic acid aptamers, often termed 'chem. antibodies', are functionally comparable to traditional antibodies, but offer several advantages, including their relatively small phys. size, flexible structure, quick chem. prodn., versatile chem. modification, high stability and lack of immunogenicity.  In addn., many aptamers are internalized upon binding to cellular receptors, making them useful targeted delivery agents for small interfering RNAs (siRNAs), microRNAs and conventional drugs.  However, several crucial factors have delayed the clin. translation of therapeutic aptamers, such as their inherent physicochem. characteristics and lack of safety data.  This Review discusses these challenges, highlighting recent clin. developments and technol. advances that have revived the impetus for this promising class of therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr4KSEhC4dW0LVg90H21EOLACvtfcHk0lj6PbDsn2B1Ww"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhslOqtr3I&md5=3d79942589e20829b3e8de582a08d6e1</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2016.199&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2016.199%26sid%3Dliteratum%253Aachs%26aulast%3DZhou%26aufirst%3DJ.%26aulast%3DRossi%26aufirst%3DJ.%26atitle%3DAptamers%2520as%2520Targeted%2520Therapeutics%253A%2520Current%2520Potential%2520and%2520Challenges%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2017%26volume%3D16%26spage%3D181%26epage%3D202%26doi%3D10.1038%2Fnrd.2016.199" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Carter, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lazar, G. A.</span></span> <span> </span><span class="NLM_article-title">Next Generation Antibody Drugs: Pursuit of the “High-Hanging Fruit"</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">197</span>– <span class="NLM_lpage">223</span>, <span class="refDoi"> DOI: 10.1038/nrd.2017.227</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1038%2Fnrd.2017.227" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2018&pages=197-223&author=P.+J.+Carterauthor=G.+A.+Lazar&title=Next+Generation+Antibody+Drugs%3A+Pursuit+of+the+%E2%80%9CHigh-Hanging+Fruit%22&doi=10.1038%2Fnrd.2017.227"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2017.227&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2017.227%26sid%3Dliteratum%253Aachs%26aulast%3DCarter%26aufirst%3DP.%2BJ.%26aulast%3DLazar%26aufirst%3DG.%2BA.%26atitle%3DNext%2520Generation%2520Antibody%2520Drugs%253A%2520Pursuit%2520of%2520the%2520%25E2%2580%259CHigh-Hanging%2520Fruit%2522%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2018%26volume%3D17%26spage%3D197%26epage%3D223%26doi%3D10.1038%2Fnrd.2017.227" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Passioura, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suga, H.</span></span> <span> </span><span class="NLM_article-title">A RaPID Way to Discover Nonstandard Macrocyclic Peptide Modulators of Drug Targets</span>. <i>Chem. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">1931</span>– <span class="NLM_lpage">1940</span>, <span class="refDoi"> DOI: 10.1039/C6CC06951G</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1039%2FC6CC06951G" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=28091672" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC2sXnslCnsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2017&pages=1931-1940&author=T.+Passiouraauthor=H.+Suga&title=A+RaPID+Way+to+Discover+Nonstandard+Macrocyclic+Peptide+Modulators+of+Drug+Targets&doi=10.1039%2FC6CC06951G"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">A RaPID way to discover nonstandard macrocyclic peptide modulators of drug targets</span></div><div class="casAuthors">Passioura, Toby; Suga, Hiroaki</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Communications (Cambridge, United Kingdom)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1931-1940</span>CODEN:
                <span class="NLM_cas:coden">CHCOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-7345</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">A review.  Studies of the fundamental nature of RNA catalysis and the potential mechanism of a shift from the "RNA world" to proteinaceous life lead the authors to identify a set of ribozymes (flexizymes) capable of promiscuous tRNA acylation.  While theor. and mechanistically interesting in their own right, flexizymes have turned out to have immense practical value for the simple synthesis of tRNAs acylated with unusual amino acids, which in turn can be used for the ribosomal synthesis of peptides contg. non-canonical residues.  Using this technique, it is possible to synthesize peptides contg. a range of structural features (macrocyclic backbones, backbone N-methylation, D-stereochem., etc.) commonly obsd. in natural product secondary metabolites, a chem. class that has historically been a rich source of drug-like mols.  Moreover, when combined with biochem. display screening technologies, this synthetic approach can be used to generate (and screen for target affinity) extremely diverse (in excess of 1012 compd.) chem. libraries, making it an extraordinary tool for drug discovery.  The current review charts the history of flexizyme technol. and its use for non-canonical peptide synthesis and screening.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo9yxKNW7clerVg90H21EOLACvtfcHk0ljwAYlTgIWfdg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXnslCnsg%253D%253D&md5=987ef1038fb1ed65a88e07db3623a039</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1039%2FC6CC06951G&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC6CC06951G%26sid%3Dliteratum%253Aachs%26aulast%3DPassioura%26aufirst%3DT.%26aulast%3DSuga%26aufirst%3DH.%26atitle%3DA%2520RaPID%2520Way%2520to%2520Discover%2520Nonstandard%2520Macrocyclic%2520Peptide%2520Modulators%2520of%2520Drug%2520Targets%26jtitle%3DChem.%2520Commun.%26date%3D2017%26volume%3D53%26spage%3D1931%26epage%3D1940%26doi%3D10.1039%2FC6CC06951G" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, R. K. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abela, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hennig, M.</span></span> <span> </span><span class="NLM_article-title">X-Ray Free Electron Laser: Opportunities for Drug Discovery</span>. <i>Essays Biochem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">529</span>– <span class="NLM_lpage">542</span>, <span class="refDoi"> DOI: 10.1042/EBC20170031</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1042%2FEBC20170031" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=29118098" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A280%3ADC%252BC1M7pt1Glsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2017&pages=529-542&author=R.+K.+Y.+Chengauthor=R.+Abelaauthor=M.+Hennig&title=X-Ray+Free+Electron+Laser%3A+Opportunities+for+Drug+Discovery&doi=10.1042%2FEBC20170031"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">X-ray free electron laser: opportunities for drug discovery</span></div><div class="casAuthors">Cheng Robert K Y; Abela Rafael; Hennig Michael</div><div class="citationInfo"><span class="NLM_cas:title">Essays in biochemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">529-542</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Past decades have shown the impact of structural information derived from complexes of drug candidates with their protein targets to facilitate the discovery of safe and effective medicines.  Despite recent developments in single particle cryo-electron microscopy, X-ray crystallography has been the main method to derive structural information.  The unique properties of X-ray free electron laser (XFEL) with unmet peak brilliance and beam focus allow X-ray diffraction data recording and successful structure determination from smaller and weaker diffracting crystals shortening timelines in crystal optimization.  To further capitalize on the XFEL advantage, innovations in crystal sample delivery for the X-ray experiment, data collection and processing methods are required.  This development was a key contributor to serial crystallography allowing structure determination at room temperature yielding physiologically more relevant structures.  Adding the time resolution provided by the femtosecond X-ray pulse will enable monitoring and capturing of dynamic processes of ligand binding and associated conformational changes with great impact to the design of candidate drug compounds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTqZGQ3bfUY0bxanA7vODO6fW6udTcc2eY45TKsodCfELntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1M7pt1Glsg%253D%253D&md5=1b74caa7a7bbf89ca30736d76a02588e</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1042%2FEBC20170031&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252FEBC20170031%26sid%3Dliteratum%253Aachs%26aulast%3DCheng%26aufirst%3DR.%2BK.%2BY.%26aulast%3DAbela%26aufirst%3DR.%26aulast%3DHennig%26aufirst%3DM.%26atitle%3DX-Ray%2520Free%2520Electron%2520Laser%253A%2520Opportunities%2520for%2520Drug%2520Discovery%26jtitle%3DEssays%2520Biochem.%26date%3D2017%26volume%3D61%26spage%3D529%26epage%3D542%26doi%3D10.1042%2FEBC20170031" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Boland, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barford, D.</span></span> <span> </span><span class="NLM_article-title">The Potential of Cryo-Electron Microscopy for Structure-Based Drug Design</span>. <i>Essays Biochem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">543</span>– <span class="NLM_lpage">560</span>, <span class="refDoi"> DOI: 10.1042/EBC20170032</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1042%2FEBC20170032" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=29118099" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A280%3ADC%252BC1M7pt1Glsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2017&pages=543-560&author=A.+Bolandauthor=L.+Changauthor=D.+Barford&title=The+Potential+of+Cryo-Electron+Microscopy+for+Structure-Based+Drug+Design&doi=10.1042%2FEBC20170032"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">The potential of cryo-electron microscopy for structure-based drug design</span></div><div class="casAuthors">Boland Andreas; Chang Leifu; Barford David</div><div class="citationInfo"><span class="NLM_cas:title">Essays in biochemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">543-560</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Structure-based drug design plays a central role in therapeutic development.  Until recently, protein crystallography and NMR have dominated experimental approaches to obtain structural information of biological molecules.  However, in recent years rapid technical developments in single particle cryo-electron microscopy (cryo-EM) have enabled the determination to near-atomic resolution of macromolecules ranging from large multi-subunit molecular machines to proteins as small as 64 kDa.  These advances have revolutionized structural biology by hugely expanding both the range of macromolecules whose structures can be determined, and by providing a description of macromolecular dynamics.  Cryo-EM is now poised to similarly transform the discipline of structure-based drug discovery.  This article reviews the potential of cryo-EM for drug discovery with reference to protein ligand complex structures determined using this technique.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTqZGQ3bfUY0c-crCu0nI_sfW6udTcc2eY45TKsodCfELntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1M7pt1Glsw%253D%253D&md5=a5bdc609a7330f1b01dd7ef921c4afd4</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1042%2FEBC20170032&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252FEBC20170032%26sid%3Dliteratum%253Aachs%26aulast%3DBoland%26aufirst%3DA.%26aulast%3DChang%26aufirst%3DL.%26aulast%3DBarford%26aufirst%3DD.%26atitle%3DThe%2520Potential%2520of%2520Cryo-Electron%2520Microscopy%2520for%2520Structure-Based%2520Drug%2520Design%26jtitle%3DEssays%2520Biochem.%26date%3D2017%26volume%3D61%26spage%3D543%26epage%3D560%26doi%3D10.1042%2FEBC20170032" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cramer, R. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soltanshahi, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jilek, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campbell, B.</span></span> <span> </span><span class="NLM_article-title">AllChem: Generating and Searching 1020 Synthetically Accessible Structures</span>. <i>J. Comput.-Aided Mol. Des.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">341</span>– <span class="NLM_lpage">350</span>, <span class="refDoi"> DOI: 10.1007/s10822-006-9093-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1007%2Fs10822-006-9093-8" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2007&pages=341-350&author=R.+D.+Cramerauthor=F.+Soltanshahiauthor=R.+Jilekauthor=B.+Campbell&title=AllChem%3A+Generating+and+Searching+1020+Synthetically+Accessible+Structures&doi=10.1007%2Fs10822-006-9093-8"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1007%2Fs10822-006-9093-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-006-9093-8%26sid%3Dliteratum%253Aachs%26aulast%3DCramer%26aufirst%3DR.%2BD.%26aulast%3DSoltanshahi%26aufirst%3DF.%26aulast%3DJilek%26aufirst%3DR.%26aulast%3DCampbell%26aufirst%3DB.%26atitle%3DAllChem%253A%2520Generating%2520and%2520Searching%25201020%2520Synthetically%2520Accessible%2520Structures%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2007%26volume%3D21%26spage%3D341%26epage%3D350%26doi%3D10.1007%2Fs10822-006-9093-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Macarron, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Banks, M. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bojanic, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burns, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cirovic, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garyantes, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Green, D. V. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hertzberg, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Janzen, W. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paslay, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schopfer, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sittampalam, G. S.</span></span> <span> </span><span class="NLM_article-title">Impact of High-Throughput Screening in Biomedical Research</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">188</span>– <span class="NLM_lpage">195</span>, <span class="refDoi"> DOI: 10.1038/nrd3368</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1038%2Fnrd3368" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=21358738" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC3MXisFensbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2011&pages=188-195&author=R.+Macarronauthor=M.+N.+Banksauthor=D.+Bojanicauthor=D.+J.+Burnsauthor=D.+A.+Cirovicauthor=T.+Garyantesauthor=D.+V.+S.+Greenauthor=R.+P.+Hertzbergauthor=W.+P.+Janzenauthor=J.+W.+Paslayauthor=U.+Schopferauthor=G.+S.+Sittampalam&title=Impact+of+High-Throughput+Screening+in+Biomedical+Research&doi=10.1038%2Fnrd3368"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Impact of high-throughput screening in biomedical research</span></div><div class="casAuthors">Macarron, Ricardo; Banks, Martyn N.; Bojanic, Dejan; Burns, David J.; Cirovic, Dragan A.; Garyantes, Tina; Green, Darren V. S.; Hertzberg, Robert P.; Janzen, William P.; Paslay, Jeff W.; Schopfer, Ulrich; Sittampalam, G. Sitta</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">188-195</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  High-throughput screening (HTS) has been postulated in several quarters to be a contributory factor to the decline in productivity in the pharmaceutical industry.  Moreover, it has been blamed for stifling the creativity that drug discovery demands.  In this article, we aim to dispel these myths and present the case for the use of HTS as part of a proven scientific tool kit, the wider use of which is essential for the discovery of new chemotypes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpLnrSVehADbLVg90H21EOLACvtfcHk0ljxErf8r4wT7A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXisFensbY%253D&md5=cb9853227506f3a3d774791cff9fd79d</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1038%2Fnrd3368&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd3368%26sid%3Dliteratum%253Aachs%26aulast%3DMacarron%26aufirst%3DR.%26aulast%3DBanks%26aufirst%3DM.%2BN.%26aulast%3DBojanic%26aufirst%3DD.%26aulast%3DBurns%26aufirst%3DD.%2BJ.%26aulast%3DCirovic%26aufirst%3DD.%2BA.%26aulast%3DGaryantes%26aufirst%3DT.%26aulast%3DGreen%26aufirst%3DD.%2BV.%2BS.%26aulast%3DHertzberg%26aufirst%3DR.%2BP.%26aulast%3DJanzen%26aufirst%3DW.%2BP.%26aulast%3DPaslay%26aufirst%3DJ.%2BW.%26aulast%3DSchopfer%26aufirst%3DU.%26aulast%3DSittampalam%26aufirst%3DG.%2BS.%26atitle%3DImpact%2520of%2520High-Throughput%2520Screening%2520in%2520Biomedical%2520Research%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2011%26volume%3D10%26spage%3D188%26epage%3D195%26doi%3D10.1038%2Fnrd3368" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Besnard, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hopkins, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pannifer, A. D.</span></span> <span> </span><span class="NLM_article-title">The Joint European Compound Library: Boosting Precompetitive Research</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">181</span>– <span class="NLM_lpage">186</span>, <span class="refDoi"> DOI: 10.1016/j.drudis.2014.08.014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1016%2Fj.drudis.2014.08.014" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=25205347" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsFGgsLbE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2015&pages=181-186&author=J.+Besnardauthor=P.+S.+Jonesauthor=A.+L.+Hopkinsauthor=A.+D.+Pannifer&title=The+Joint+European+Compound+Library%3A+Boosting+Precompetitive+Research&doi=10.1016%2Fj.drudis.2014.08.014"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">The Joint European Compound Library: boosting precompetitive research</span></div><div class="casAuthors">Besnard, Jeremy; Jones, Philip S.; Hopkins, Andrew L.; Pannifer, Andrew D.</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">181-186</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">The Joint European Compd. Library (JECL) is a new high-throughput screening collection aimed at driving precompetitive drug discovery and target validation.  The JECL has been established with a core of over 321 000 compds. from the proprietary collections of seven pharmaceutical companies and will expand to around 500 000 compds.  Here, we analyze the physicochem. profile and chem. diversity of the core collection, showing that the collection is diverse and has a broad spectrum of predicted biol. activity.  We also describe a model for sharing compd. information from multiple proprietary collections, enabling diversity and quality anal. without disclosing structures.  The JECL is available for screening at no cost to European academic labs. and SMEs through the IMI European Lead Factory (http://www.europeanleadfactory.eu/).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrYUIRI3z5DAbVg90H21EOLACvtfcHk0ljxErf8r4wT7A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsFGgsLbE&md5=cbb46ec6091685a0f9908e5100c46c9c</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2014.08.014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2014.08.014%26sid%3Dliteratum%253Aachs%26aulast%3DBesnard%26aufirst%3DJ.%26aulast%3DJones%26aufirst%3DP.%2BS.%26aulast%3DHopkins%26aufirst%3DA.%2BL.%26aulast%3DPannifer%26aufirst%3DA.%2BD.%26atitle%3DThe%2520Joint%2520European%2520Compound%2520Library%253A%2520Boosting%2520Precompetitive%2520Research%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2015%26volume%3D20%26spage%3D181%26epage%3D186%26doi%3D10.1016%2Fj.drudis.2014.08.014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hamley, P.</span>; <span class="NLM_string-name">Jimonet, P.</span></span> <span> </span><span class="NLM_article-title">Advances in the Understanding of Drug Properties in Medicinal Chemistry</span>. In  <i>Small Molecule Medicinal Chemistry</i>; <span class="NLM_contrib-group"><span class="NLM_string-name">Czechtizky, W.</span>, <span class="NLM_string-name">Hamley, P.</span></span>, Eds.; <span class="NLM_publisher-name">John Wiley & Sons, Inc</span>: <span class="NLM_publisher-loc">Hoboken, NJ</span>, <span class="NLM_year">2015</span>; pp  <span class="NLM_fpage">305</span>– <span class="NLM_lpage">322</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1002%2F9781118771723.ch11" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2015&pages=305-322&author=P.+Hamley&author=P.+Jimonetauthor=W.+Czechtizky&author=P.+Hamley&title=Small+Molecule+Medicinal+Chemistry"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1002%2F9781118771723.ch11&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1002%252F9781118771723.ch11%26sid%3Dliteratum%253Aachs%26aulast%3DHamley%26aufirst%3DP.%26atitle%3DAdvances%2520in%2520the%2520Understanding%2520of%2520Drug%2520Properties%2520in%2520Medicinal%2520Chemistry%26btitle%3DSmall%2520Molecule%2520Medicinal%2520Chemistry%26aulast%3DCzechtizky%26aufirst%3DW.%26pub%3DJohn%2520Wiley%2520%2526%2520Sons%252C%2520Inc%26date%3D2015%26spage%3D305%26epage%3D322" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lerner, R. A.</span></span> <span> </span><span class="NLM_article-title">Combinatorial Antibody Libraries: New Advances, New Immunological Insights</span>. <i>Nat. Rev. Immunol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">498</span>– <span class="NLM_lpage">508</span>, <span class="refDoi"> DOI: 10.1038/nri.2016.67</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1038%2Fnri.2016.67" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=27374636" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtV2ntLvK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2016&pages=498-508&author=R.+A.+Lerner&title=Combinatorial+Antibody+Libraries%3A+New+Advances%2C+New+Immunological+Insights&doi=10.1038%2Fnri.2016.67"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Combinatorial antibody libraries: new advances, new immunological insights</span></div><div class="casAuthors">Lerner, Richard A.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Immunology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">498-508</span>CODEN:
                <span class="NLM_cas:coden">NRIABX</span>;
        ISSN:<span class="NLM_cas:issn">1474-1733</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Immunochemists have become quite proficient in engineering existing antibody mols. to control their pharmacol. properties.  However, in terms of generating new antibodies, the combinatorial antibody library has become a central feature of modern immunochem.  These libraries are essentially an immune system in a test tube and enable the selection of antibodies without the constraints of whole animal or cell-based systems.  This Review provides an overview of how antibody libraries are constructed and discusses what can be learnt from these synthetic systems.  In particular, the Review focuses on new biol. insights from antibody libraries - such as the concept of 'SOS antibodies' - and the growing use of intracellular antibodies to perturb cellular functions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoEEd11DiD6mLVg90H21EOLACvtfcHk0ljxErf8r4wT7A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtV2ntLvK&md5=5674760f51d0ded0d280b6f0758d1409</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1038%2Fnri.2016.67&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnri.2016.67%26sid%3Dliteratum%253Aachs%26aulast%3DLerner%26aufirst%3DR.%2BA.%26atitle%3DCombinatorial%2520Antibody%2520Libraries%253A%2520New%2520Advances%252C%2520New%2520Immunological%2520Insights%26jtitle%3DNat.%2520Rev.%2520Immunol.%26date%3D2016%26volume%3D16%26spage%3D498%26epage%3D508%26doi%3D10.1038%2Fnri.2016.67" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Darmostuk, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rimpelova, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gbelcova, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruml, T.</span></span> <span> </span><span class="NLM_article-title">Current Approaches in SELEX: An Update to Aptamer Selection Technology</span>. <i>Biotechnol. Adv.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>33</i></span>,  <span class="NLM_fpage">1141</span>– <span class="NLM_lpage">1161</span>, <span class="refDoi"> DOI: 10.1016/j.biotechadv.2015.02.008</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1016%2Fj.biotechadv.2015.02.008" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=25708387" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC2MXjslylsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2015&pages=1141-1161&author=M.+Darmostukauthor=S.+Rimpelovaauthor=H.+Gbelcovaauthor=T.+Ruml&title=Current+Approaches+in+SELEX%3A+An+Update+to+Aptamer+Selection+Technology&doi=10.1016%2Fj.biotechadv.2015.02.008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Current approaches in SELEX: An update to aptamer selection technology</span></div><div class="casAuthors">Darmostuk, Mariia; Rimpelova, Silvie; Gbelcova, Helena; Ruml, Tomas</div><div class="citationInfo"><span class="NLM_cas:title">Biotechnology Advances</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">6_Part_2</span>),
    <span class="NLM_cas:pages">1141-1161</span>CODEN:
                <span class="NLM_cas:coden">BIADDD</span>;
        ISSN:<span class="NLM_cas:issn">0734-9750</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">Systematic evolution of ligands by exponential enrichment (SELEX) is a well-established and efficient technol. for the generation of oligonucleotides with a high target affinity.  These SELEX-derived single stranded DNA and RNA mols., called aptamers, were selected against various targets, such as proteins, cells, microorganisms, chem. compds. etc.  They have a great potential in the use as novel antibodies, in cancer theragnostics and in biomedical research.  Vast interest in aptamers stimulated continuous development of SELEX, which underwent numerous modifications since its first application in 1990.  Novel modifications made the selection process more efficient, cost-effective and significantly less time-consuming.  This article brings a comprehensive and up-to-date review of recent advances in SELEX methods and pinpoints advantages, main obstacles and limitations.  The post-SELEX strategies and examples of application are also briefly outlined in this review.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpFs2pUhH9mnrVg90H21EOLACvtfcHk0lhnwxvigA9bmA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXjslylsb4%253D&md5=dc2fdb1a71c8f5eb7a77ce6d14bf4f28</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1016%2Fj.biotechadv.2015.02.008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.biotechadv.2015.02.008%26sid%3Dliteratum%253Aachs%26aulast%3DDarmostuk%26aufirst%3DM.%26aulast%3DRimpelova%26aufirst%3DS.%26aulast%3DGbelcova%26aufirst%3DH.%26aulast%3DRuml%26aufirst%3DT.%26atitle%3DCurrent%2520Approaches%2520in%2520SELEX%253A%2520An%2520Update%2520to%2520Aptamer%2520Selection%2520Technology%26jtitle%3DBiotechnol.%2520Adv.%26date%3D2015%26volume%3D33%26spage%3D1141%26epage%3D1161%26doi%3D10.1016%2Fj.biotechadv.2015.02.008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, R.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thapa, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Espiritu, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Menon, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bingham, J.-P.</span></span> <span> </span><span class="NLM_article-title">From Nature to Creation: Going around in Circles, the Art of Peptide Cyclization</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">1135</span>– <span class="NLM_lpage">1150</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2017.11.017</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1016%2Fj.bmc.2017.11.017" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=29295762" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitlWhtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2018&pages=1135-1150&author=R.-Y.+Zhangauthor=P.+Thapaauthor=M.+J.+Espirituauthor=V.+Menonauthor=J.-P.+Bingham&title=From+Nature+to+Creation%3A+Going+around+in+Circles%2C+the+Art+of+Peptide+Cyclization&doi=10.1016%2Fj.bmc.2017.11.017"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">From Nature to creation: Going around in circles, the art of peptide cyclization</span></div><div class="casAuthors">Zhang, Rui-Yang; Thapa, Parashar; Espiritu, Michael J.; Menon, Vinay; Bingham, Jon-Paul</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1135-1150</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Cyclic peptides and cyclotides are becoming common identities within present efforts seen in peptide engineering as we seek approaches to achieve potent biol. activity, pharmacol. selectivity, structurally stability, and oral bioavailability.  Yet, this unique family of peptides has faced uncommon hurdles in their discovery, synthesis, and bioengineering, retaining characteristics that truly deviate these from their linear counterparts.  Here, we take a board spectrum approach to introduce this novel family of biomols. and the troubles that they face in their sequence and disulfide connectivity assignment, together highlighting the present combined strategies involved in cyclic peptide/cyclotide synthesis and modification.  These efforts have circumvented otherwise impossible hurdles in their manipulation and prodn. that are only now advancing cyclic peptides/cyclotides as research probes and future pharmaceutical templates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqmjnZmnb594LVg90H21EOLACvtfcHk0lhnwxvigA9bmA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitlWhtw%253D%253D&md5=45e70431f57bc151655776f103acd39c</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2017.11.017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2017.11.017%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DR.-Y.%26aulast%3DThapa%26aufirst%3DP.%26aulast%3DEspiritu%26aufirst%3DM.%2BJ.%26aulast%3DMenon%26aufirst%3DV.%26aulast%3DBingham%26aufirst%3DJ.-P.%26atitle%3DFrom%2520Nature%2520to%2520Creation%253A%2520Going%2520around%2520in%2520Circles%252C%2520the%2520Art%2520of%2520Peptide%2520Cyclization%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2018%26volume%3D26%26spage%3D1135%26epage%3D1150%26doi%3D10.1016%2Fj.bmc.2017.11.017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Heinis, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rutherford, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Freund, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winter, G.</span></span> <span> </span><span class="NLM_article-title">Phage-Encoded Combinatorial Chemical Libraries Based on Bicyclic Peptides</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">502</span>– <span class="NLM_lpage">507</span>, <span class="refDoi"> DOI: 10.1038/nchembio.184</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1038%2Fnchembio.184" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=19483697" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BD1MXms1altbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2009&pages=502-507&author=C.+Heinisauthor=T.+Rutherfordauthor=S.+Freundauthor=G.+Winter&title=Phage-Encoded+Combinatorial+Chemical+Libraries+Based+on+Bicyclic+Peptides&doi=10.1038%2Fnchembio.184"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Phage-encoded combinatorial chemical libraries based on bicyclic peptides</span></div><div class="casAuthors">Heinis, Christian; Rutherford, Trevor; Freund, Stephan; Winter, Greg</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">502-507</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Here we describe a phage strategy for the selection of ligands based on bicyclic or linear peptides attached covalently to an org. core.  We designed peptide repertoires with three reactive cysteine residues, each spaced apart by several random amino acid residues, and we fused the repertoires to the phage gene-3-protein.  Conjugation with tris-(bromomethyl)benzene via the reactive cysteines generated repertoires of peptide conjugates with two peptide loops anchored to a mesitylene core.  Iterative affinity selections yielded several enzyme inhibitors; after further mutagenesis and selection, we were able to chem. synthesize a lead inhibitor (PK15; Ki = 1.5 nM) specific to human plasma kallikrein that efficiently interrupted the intrinsic coagulation pathway in human plasma tested ex vivo.  This approach offers a powerful means of generating and selecting bicyclic macrocycles (or if cleaved, linear derivs. thereof) as ligands poised at the interface of small-mol. drugs and biologics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrZhJRZyldisLVg90H21EOLACvtfcHk0lhnwxvigA9bmA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXms1altbs%253D&md5=0634460f77b185fbcf85309c6dbf93e3</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.184&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.184%26sid%3Dliteratum%253Aachs%26aulast%3DHeinis%26aufirst%3DC.%26aulast%3DRutherford%26aufirst%3DT.%26aulast%3DFreund%26aufirst%3DS.%26aulast%3DWinter%26aufirst%3DG.%26atitle%3DPhage-Encoded%2520Combinatorial%2520Chemical%2520Libraries%2520Based%2520on%2520Bicyclic%2520Peptides%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2009%26volume%3D5%26spage%3D502%26epage%3D507%26doi%3D10.1038%2Fnchembio.184" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morales-Sanfrutos, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Angelini, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cutting, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heinis, C.</span></span> <span> </span><span class="NLM_article-title">Structurally Diverse Cyclisation Linkers Impose Different Backbone Conformations in Bicyclic Peptides</span>. <i>ChemBioChem</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">1032</span>– <span class="NLM_lpage">1038</span>, <span class="refDoi"> DOI: 10.1002/cbic.201200049</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1002%2Fcbic.201200049" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=22492661" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC38Xlt1Kltb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2012&pages=1032-1038&author=S.+Chenauthor=J.+Morales-Sanfrutosauthor=A.+Angeliniauthor=B.+Cuttingauthor=C.+Heinis&title=Structurally+Diverse+Cyclisation+Linkers+Impose+Different+Backbone+Conformations+in+Bicyclic+Peptides&doi=10.1002%2Fcbic.201200049"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Structurally Diverse Cyclisation Linkers Impose Different Backbone Conformations in Bicyclic Peptides</span></div><div class="casAuthors">Chen, Shiyu; Morales-Sanfrutos, Julia; Angelini, Alessandro; Cutting, Brian; Heinis, Christian</div><div class="citationInfo"><span class="NLM_cas:title">ChemBioChem</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1032-1038</span>CODEN:
                <span class="NLM_cas:coden">CBCHFX</span>;
        ISSN:<span class="NLM_cas:issn">1439-4227</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Combinatorial libraries of structurally diverse peptide macrocycles offer a rich source for the development of high-affinity ligands to targets of interest.  In this work we have developed linkers for the generation of genetically encoded bicyclic peptides and tested whether the peptides cyclised by them have significant variations in their backbone conformations.  Two new cyclisation reagents, each contg. three thiol-reactive groups, efficiently and selectively cyclised linear peptides contg. three cysteine moieties.  When the mesitylene linker of the bicyclic peptide PK15, a potent inhibitor of plasma kallikrein (Ki=2 nM), was replaced by the new linkers, its inhibitory activity dropped by a factor of more than 1000, suggesting that the linkers impose different conformations on the peptide.  Indeed, structural anal. by soln.-state NMR revealed different NOE constraints in the three bicyclic peptides, indicating that these relatively small linkers at the centers of bicyclic peptide structures significantly influence the conformations of the peptides.  These results demonstrate the prominent structural role of linkers in peptide macrocycles and suggest that application of different cyclisation linkers in a combinatorial fashion could be an attractive means to generate topol. diverse macrocycle libraries.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGphdCp8KV5HeLVg90H21EOLACvtfcHk0lhnwxvigA9bmA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xlt1Kltb4%253D&md5=ddb3f7e338a22ea1a1e3ced71624781d</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1002%2Fcbic.201200049&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcbic.201200049%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DS.%26aulast%3DMorales-Sanfrutos%26aufirst%3DJ.%26aulast%3DAngelini%26aufirst%3DA.%26aulast%3DCutting%26aufirst%3DB.%26aulast%3DHeinis%26aufirst%3DC.%26atitle%3DStructurally%2520Diverse%2520Cyclisation%2520Linkers%2520Impose%2520Different%2520Backbone%2520Conformations%2520in%2520Bicyclic%2520Peptides%26jtitle%3DChemBioChem%26date%3D2012%26volume%3D13%26spage%3D1032%26epage%3D1038%26doi%3D10.1002%2Fcbic.201200049" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rebollo, I. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Angelini, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heinis, C.</span></span> <span> </span><span class="NLM_article-title">Phage Display Libraries of Differently Sized Bicyclic Peptides</span>. <i>MedChemComm</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">145</span>– <span class="NLM_lpage">150</span>, <span class="refDoi"> DOI: 10.1039/C2MD20171B</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1039%2FC2MD20171B" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvVymsr%252FO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2013&pages=145-150&author=I.+R.+Rebolloauthor=A.+Angeliniauthor=C.+Heinis&title=Phage+Display+Libraries+of+Differently+Sized+Bicyclic+Peptides&doi=10.1039%2FC2MD20171B"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Phage display libraries of differently sized bicyclic peptides</span></div><div class="casAuthors">Rebollo, Inmaculada Rentero; Angelini, Alessandro; Heinis, Christian</div><div class="citationInfo"><span class="NLM_cas:title">MedChemComm</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">145-150</span>CODEN:
                <span class="NLM_cas:coden">MCCEAY</span>;
        ISSN:<span class="NLM_cas:issn">2040-2503</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Phage selections with combinatorial libraries of uniformly sized bicyclic peptides have recently yielded potent and selective binders of several protein targets.  In this work we varied in a combinatorial fashion the ring sizes of bicyclic peptides in phage libraries, expecting that they would yield binders with higher affinities and/or more diverse binding motifs that could be affinity matured. 14 new phage peptide libraries of the format Cys-(Xaa)m-Cys-(Xaa)n-Cys (Xaa are random amino acids, m and n = 3, 4, 5 or 6) were generated and cyclized with tris-(bromomethyl)benzene.  Affinity selections against the tumor-assocd. serine protease urokinase-type plasminogen activator yielded bicyclic peptide inhibitors with a large variety of consensus sequences.  Several of the identified consensus sequences were exclusively found in bicyclic peptides having defined ring size combinations.  Some of these peptides may bind in orientations that allow affinity maturation of non-conserved regions, while others do not.  Having available multiple leads isolated from such bicyclic peptide libraries with variable ring sizes could therefore be a great asset for the generation of high affinity binders.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrYZtLlWd3zQbVg90H21EOLACvtfcHk0lh1XeQiT1k79g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvVymsr%252FO&md5=26f43d83d89ab90d12331b3614dd2dad</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1039%2FC2MD20171B&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC2MD20171B%26sid%3Dliteratum%253Aachs%26aulast%3DRebollo%26aufirst%3DI.%2BR.%26aulast%3DAngelini%26aufirst%3DA.%26aulast%3DHeinis%26aufirst%3DC.%26atitle%3DPhage%2520Display%2520Libraries%2520of%2520Differently%2520Sized%2520Bicyclic%2520Peptides%26jtitle%3DMedChemComm%26date%3D2013%26volume%3D4%26spage%3D145%26epage%3D150%26doi%3D10.1039%2FC2MD20171B" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rentero
Rebollo, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCallin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bertoldo, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Entenza, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moreillon, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heinis, C.</span></span> <span> </span><span class="NLM_article-title">Development of Potent and Selective S. Aureus Sortase A Inhibitors Based on Peptide Macrocycles</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">606</span>– <span class="NLM_lpage">611</span>, <span class="refDoi"> DOI: 10.1021/acsmedchemlett.6b00045</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.6b00045" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC28XmtFShu78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=606-611&author=I.+Rentero%0ARebolloauthor=S.+McCallinauthor=D.+Bertoldoauthor=J.+M.+Entenzaauthor=P.+Moreillonauthor=C.+Heinis&title=Development+of+Potent+and+Selective+S.+Aureus+Sortase+A+Inhibitors+Based+on+Peptide+Macrocycles&doi=10.1021%2Facsmedchemlett.6b00045"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Development of Potent and Selective S. aureus Sortase A Inhibitors Based on Peptide Macrocycles</span></div><div class="casAuthors">Rentero Rebollo, Inmaculada; McCallin, Shawna; Bertoldo, Davide; Entenza, Jose Manuel; Moreillon, Philippe; Heinis, Christian</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">606-611</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Sortases are transpeptidase enzymes that anchor surface proteins, including virulence factors, to the cell wall of Gram-pos. bacteria, and they are potential targets for the development of anti-infective agents.  While several large compd. libraries were searched by high-throughput screening, no high-affinity inhibitors of sortases could be developed to date.  Here, we applied phage display to screen billions of peptide macrocycles against sortase A (SrtA) of Staphylococcus aureus (S. aureus).  We were able to identify potent and selective inhibitors of SrtA that blocked SrtA-mediated anchoring of synthetic substrates to the surface of live S. aureus cells.  A region present in all inhibitory peptides (Leu-Pro-Pro) resembled the natural substrates of SrtA (Leu-Pro-Xaa-Thr-Gly), suggesting that the macrocycles bind to the enzyme's active site and that they form similar mol. contacts as natural substrates.  The evolved peptide macrocycles may be used as lead structures for the development of potent peptidomimetic SrtA inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqYzcM3aIF9WrVg90H21EOLACvtfcHk0lh1XeQiT1k79g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XmtFShu78%253D&md5=0768ae2e5722bac33b68e99272be06e4</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.6b00045&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.6b00045%26sid%3Dliteratum%253Aachs%26aulast%3DRentero%2BRebollo%26aufirst%3DI.%26aulast%3DMcCallin%26aufirst%3DS.%26aulast%3DBertoldo%26aufirst%3DD.%26aulast%3DEntenza%26aufirst%3DJ.%2BM.%26aulast%3DMoreillon%26aufirst%3DP.%26aulast%3DHeinis%26aufirst%3DC.%26atitle%3DDevelopment%2520of%2520Potent%2520and%2520Selective%2520S.%2520Aureus%2520Sortase%2520A%2520Inhibitors%2520Based%2520on%2520Peptide%2520Macrocycles%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2016%26volume%3D7%26spage%3D606%26epage%3D611%26doi%3D10.1021%2Facsmedchemlett.6b00045" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Diderich, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heinis, C.</span></span> <span> </span><span class="NLM_article-title">Phage Selection of Bicyclic Peptides Binding Her2</span>. <i>Tetrahedron</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>70</i></span>,  <span class="NLM_fpage">7733</span>– <span class="NLM_lpage">7739</span>, <span class="refDoi"> DOI: 10.1016/j.tet.2014.05.106</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1016%2Fj.tet.2014.05.106" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtVWgsr%252FE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=70&publication_year=2014&pages=7733-7739&author=P.+Diderichauthor=C.+Heinis&title=Phage+Selection+of+Bicyclic+Peptides+Binding+Her2&doi=10.1016%2Fj.tet.2014.05.106"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Phage selection of bicyclic peptides binding Her2</span></div><div class="casAuthors">Diderich, Philippe; Heinis, Christian</div><div class="citationInfo"><span class="NLM_cas:title">Tetrahedron</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">70</span>
        (<span class="NLM_cas:issue">42</span>),
    <span class="NLM_cas:pages">7733-7739</span>CODEN:
                <span class="NLM_cas:coden">TETRAB</span>;
        ISSN:<span class="NLM_cas:issn">0040-4020</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Aberrant expression of the epidermal growth factor receptor Her2 has been implicated in various malignancies including breast cancer.  Monoclonal antibodies and an antibody-drug conjugate targeting Her2 have found wide clin. application.  Herein, we aimed at developing Her2-specifc ligands based on peptides that have a 100-fold smaller mol. wt. than antibodies.  Such peptides could potentially offer advantages in the development of ligand-drug conjugates, such as ease of synthesis and conjugation, higher mol.-per-mass ratios, and better tumor penetration.  Panning of large bicyclic peptide phage display libraries against Her2 yielded a range of Her2-specific ligands having different formats and binding motifs.  Strong sequence similarities among several of the isolated peptides indicated that they interact with Her2 in a specific manner.  The best bicyclic peptide obtained after affinity maturation bound Her2 with a KD of 304 nM.  The diverse peptide ligands may offer valuable starting points for the development of high-affinity Her2 binders with potential application for tumor imaging and therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrBqi6pJaeyNLVg90H21EOLACvtfcHk0lh1XeQiT1k79g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtVWgsr%252FE&md5=33a503998bb2006f44f277b8d7e425ce</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1016%2Fj.tet.2014.05.106&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tet.2014.05.106%26sid%3Dliteratum%253Aachs%26aulast%3DDiderich%26aufirst%3DP.%26aulast%3DHeinis%26aufirst%3DC.%26atitle%3DPhage%2520Selection%2520of%2520Bicyclic%2520Peptides%2520Binding%2520Her2%26jtitle%3DTetrahedron%26date%3D2014%26volume%3D70%26spage%3D7733%26epage%3D7739%26doi%3D10.1016%2Fj.tet.2014.05.106" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Urech-Varenne, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Radtke, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heinis, C.</span></span> <span> </span><span class="NLM_article-title">Phage Selection of Bicyclic Peptide Ligands of the Notch1 Receptor</span>. <i>ChemMedChem</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">1754</span>– <span class="NLM_lpage">1761</span>, <span class="refDoi"> DOI: 10.1002/cmdc.201500261</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1002%2Fcmdc.201500261" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=26381087" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsFWrtLnO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&pages=1754-1761&author=C.+Urech-Varenneauthor=F.+Radtkeauthor=C.+Heinis&title=Phage+Selection+of+Bicyclic+Peptide+Ligands+of+the+Notch1+Receptor&doi=10.1002%2Fcmdc.201500261"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Phage Selection of Bicyclic Peptide Ligands of the Notch1 Receptor</span></div><div class="casAuthors">Urech-Varenne, Charlotte; Radtke, Freddy; Heinis, Christian</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1754-1761</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Precise regulation of the Notch signaling pathway is crucial, as increases or deficiencies in signaling are assocd. with diseases, including a wide range of cancers.  Recent studies have demonstrated that monoclonal antibodies that bind and stabilize the structure of the neg. regulatory region (NRR) in the extracellular domain of the Notch receptor can inhibit Notch signaling.  In this work, we posed the question whether bicyclic peptides, being around 100-fold smaller than antibodies, can also stabilize the NRR and inhibit Notch signaling.  Bicyclic peptides that bind the NRR of human Notch1 were isolated from combinatorial libraries by phage display.  Affinity maturation yielded ligands with dissocn. const. (Kd) values as low as 150 nΜ.  The bicyclic peptides increased the melting temp. of the NRR by up to 8 °C, thus substantially stabilizing the protein structure, but they did not inhibit Notch signaling in cellular assays.  Although Notch signaling could not be inhibited, this work demonstrates that phage-selected bicyclic peptides can stabilize proteins; this capacity of bicyclic peptides may be exploited for modulating the conformation of other disease targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqVL32Vw08kRLVg90H21EOLACvtfcHk0lh1XeQiT1k79g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsFWrtLnO&md5=13300060cc9bb51acd1ed15e4cceb56d</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.201500261&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.201500261%26sid%3Dliteratum%253Aachs%26aulast%3DUrech-Varenne%26aufirst%3DC.%26aulast%3DRadtke%26aufirst%3DF.%26aulast%3DHeinis%26aufirst%3DC.%26atitle%3DPhage%2520Selection%2520of%2520Bicyclic%2520Peptide%2520Ligands%2520of%2520the%2520Notch1%2520Receptor%26jtitle%3DChemMedChem%26date%3D2015%26volume%3D10%26spage%3D1754%26epage%3D1761%26doi%3D10.1002%2Fcmdc.201500261" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bertoldo, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khan, M. M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dessen, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Held, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huelsken, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heinis, C.</span></span> <span> </span><span class="NLM_article-title">Phage Selection of Peptide Macrocycles against Β-Catenin To Interfere with Wnt Signaling</span>. <i>ChemMedChem</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">834</span>– <span class="NLM_lpage">839</span>, <span class="refDoi"> DOI: 10.1002/cmdc.201500557</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1002%2Fcmdc.201500557" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2016&pages=834-839&author=D.+Bertoldoauthor=M.+M.+G.+Khanauthor=P.+Dessenauthor=W.+Heldauthor=J.+Huelskenauthor=C.+Heinis&title=Phage+Selection+of+Peptide+Macrocycles+against+%CE%92-Catenin+To+Interfere+with+Wnt+Signaling&doi=10.1002%2Fcmdc.201500557"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.201500557&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.201500557%26sid%3Dliteratum%253Aachs%26aulast%3DBertoldo%26aufirst%3DD.%26aulast%3DKhan%26aufirst%3DM.%2BM.%2BG.%26aulast%3DDessen%26aufirst%3DP.%26aulast%3DHeld%26aufirst%3DW.%26aulast%3DHuelsken%26aufirst%3DJ.%26aulast%3DHeinis%26aufirst%3DC.%26atitle%3DPhage%2520Selection%2520of%2520Peptide%2520Macrocycles%2520against%2520%25CE%2592-Catenin%2520To%2520Interfere%2520with%2520Wnt%2520Signaling%26jtitle%3DChemMedChem%26date%3D2016%26volume%3D11%26spage%3D834%26epage%3D839%26doi%3D10.1002%2Fcmdc.201500557" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Luther, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moehle, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chevalier, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dale, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Obrecht, D.</span></span> <span> </span><span class="NLM_article-title">Protein Epitope Mimetic Macrocycles as Biopharmaceuticals</span>. <i>Curr. Opin. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>38</i></span>,  <span class="NLM_fpage">45</span>– <span class="NLM_lpage">51</span>, <span class="refDoi"> DOI: 10.1016/j.cbpa.2017.02.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1016%2Fj.cbpa.2017.02.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=28319811" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC2sXksFSiu7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=2017&pages=45-51&author=A.+Lutherauthor=K.+Moehleauthor=E.+Chevalierauthor=G.+Daleauthor=D.+Obrecht&title=Protein+Epitope+Mimetic+Macrocycles+as+Biopharmaceuticals&doi=10.1016%2Fj.cbpa.2017.02.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Protein epitope mimetic macrocycles as biopharmaceuticals</span></div><div class="casAuthors">Luther, Anatol; Moehle, Kerstin; Chevalier, Eric; Dale, Glenn; Obrecht, Daniel</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Chemical Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">45-51</span>CODEN:
                <span class="NLM_cas:coden">COCBF4</span>;
        ISSN:<span class="NLM_cas:issn">1367-5931</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Fully synthetic medium-sized macrocyclic peptides mimicking the key β-hairpin and α-helical protein epitopes relevant in many protein-protein interactions have emerged as a novel class of drugs with the potential to fill an important gap between small mols. and proteins.  Conformationally stabilized macrocyclic scaffolds represent ideal templates for medicinal chemists to incorporate bioactive peptide and protein pharmacophores in order to generate novel drugs to treat diseases with high unmet medical need.  This review describes recent approaches to design and generate large libraries of such macrocycles, for hit identification, and for their efficient optimization.  Finally, this review describes some of the most advanced protein epitope mimetic (PEM) macrocycles in clin. development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpKLA10A3orAbVg90H21EOLACvtfcHk0li15YZ-bH8nrQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXksFSiu7Y%253D&md5=623f351d0e915d58f1ebd698da850a37</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1016%2Fj.cbpa.2017.02.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cbpa.2017.02.004%26sid%3Dliteratum%253Aachs%26aulast%3DLuther%26aufirst%3DA.%26aulast%3DMoehle%26aufirst%3DK.%26aulast%3DChevalier%26aufirst%3DE.%26aulast%3DDale%26aufirst%3DG.%26aulast%3DObrecht%26aufirst%3DD.%26atitle%3DProtein%2520Epitope%2520Mimetic%2520Macrocycles%2520as%2520Biopharmaceuticals%26jtitle%3DCurr.%2520Opin.%2520Chem.%2520Biol.%26date%3D2017%26volume%3D38%26spage%3D45%26epage%3D51%26doi%3D10.1016%2Fj.cbpa.2017.02.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Urban, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moosmeier, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aumüller, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thein, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bosma, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rink, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Groth, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zulley, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siegers, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tissot, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moll, G. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prassler, J.</span></span> <span> </span><span class="NLM_article-title">Phage Display and Selection of Lanthipeptides on the Carboxy-Terminus of the Gene-3 minor Coat Protein</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">1500</span>, <span class="refDoi"> DOI: 10.1038/s41467-017-01413-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1038%2Fs41467-017-01413-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=29138389" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A280%3ADC%252BC1M3ht1yisw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=1500&author=J.+H.+Urbanauthor=M.+A.+Moosmeierauthor=T.+Aum%C3%BCllerauthor=M.+Theinauthor=T.+Bosmaauthor=R.+Rinkauthor=K.+Grothauthor=M.+Zulleyauthor=K.+Siegersauthor=K.+Tissotauthor=G.+N.+Mollauthor=J.+Prassler&title=Phage+Display+and+Selection+of+Lanthipeptides+on+the+Carboxy-Terminus+of+the+Gene-3+minor+Coat+Protein&doi=10.1038%2Fs41467-017-01413-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Phage display and selection of lanthipeptides on the carboxy-terminus of the gene-3 minor coat protein</span></div><div class="casAuthors">Urban Johannes H; Moosmeier Markus A; Aumuller Tobias; Thein Marcus; Groth Katharina; Zulley Moritz; Siegers Katja; Tissot Kathrin; Prassler Josef; Bosma Tjibbe; Rink Rick; Moll Gert N</div><div class="citationInfo"><span class="NLM_cas:title">Nature communications</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1500</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Ribosomally synthesized and post-translationally modified peptides (RiPPs) are an emerging class of natural products with drug-like properties.  To fully exploit the potential of RiPPs as peptide drug candidates, tools for their systematic engineering are required.  Here we report the engineering of lanthipeptides, a subclass of RiPPs characterized by multiple thioether cycles that are enzymatically introduced in a regio- and stereospecific manner, by phage display.  This was achieved by heterologous co-expression of linear lanthipeptide precursors fused to the widely neglected C-terminus of the bacteriophage M13 minor coat protein pIII, rather than the conventionally used N-terminus, along with the modifying enzymes from distantly related bacteria.  We observe that C-terminal precursor peptide fusions to pIII are enzymatically modified in the cytoplasm of the producing cell and subsequently displayed as mature cyclic peptides on the phage surface.  Biopanning of large C-terminal display libraries readily identifies artificial lanthipeptide ligands specific to urokinase plasminogen activator (uPA) and streptavidin.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQsuQSi4xSarvh3YQQtAhNufW6udTcc2eZLzLiHxO-TzLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1M3ht1yisw%253D%253D&md5=8e6f2d606a659541adcd2a28aea4a953</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1038%2Fs41467-017-01413-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41467-017-01413-7%26sid%3Dliteratum%253Aachs%26aulast%3DUrban%26aufirst%3DJ.%2BH.%26aulast%3DMoosmeier%26aufirst%3DM.%2BA.%26aulast%3DAum%25C3%25BCller%26aufirst%3DT.%26aulast%3DThein%26aufirst%3DM.%26aulast%3DBosma%26aufirst%3DT.%26aulast%3DRink%26aufirst%3DR.%26aulast%3DGroth%26aufirst%3DK.%26aulast%3DZulley%26aufirst%3DM.%26aulast%3DSiegers%26aufirst%3DK.%26aulast%3DTissot%26aufirst%3DK.%26aulast%3DMoll%26aufirst%3DG.%2BN.%26aulast%3DPrassler%26aufirst%3DJ.%26atitle%3DPhage%2520Display%2520and%2520Selection%2520of%2520Lanthipeptides%2520on%2520the%2520Carboxy-Terminus%2520of%2520the%2520Gene-3%2520minor%2520Coat%2520Protein%26jtitle%3DNat.%2520Commun.%26date%3D2017%26volume%3D8%26spage%3D1500%26doi%3D10.1038%2Fs41467-017-01413-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ito, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Passioura, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suga, H.</span></span> <span> </span><span class="NLM_article-title">Technologies for the Synthesis of mRNA-Encoding Libraries and Discovery of Bioactive Natural Product-Inspired Non-Traditional Macrocyclic Peptides</span>. <i>Molecules</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">3502</span>– <span class="NLM_lpage">3528</span>, <span class="refDoi"> DOI: 10.3390/molecules18033502</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.3390%2Fmolecules18033502" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=23507778" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC3sXlt1GitLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2013&pages=3502-3528&author=K.+Itoauthor=T.+Passiouraauthor=H.+Suga&title=Technologies+for+the+Synthesis+of+mRNA-Encoding+Libraries+and+Discovery+of+Bioactive+Natural+Product-Inspired+Non-Traditional+Macrocyclic+Peptides&doi=10.3390%2Fmolecules18033502"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Technologies for the synthesis of mRNA-encoding libraries and discovery of bioactive natural product-inspired non-traditional macrocyclic peptides</span></div><div class="casAuthors">Ito, Kenichiro; Passioura, Toby; Suga, Hiroaki</div><div class="citationInfo"><span class="NLM_cas:title">Molecules</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">3502-3528</span>CODEN:
                <span class="NLM_cas:coden">MOLEFW</span>;
        ISSN:<span class="NLM_cas:issn">1420-3049</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">In this review, we discuss emerging technologies for drug discovery, which yields novel mol. scaffolds based on natural product-inspired non-traditional peptides expressed using the translation machinery.  Unlike natural products, these technologies allow for constructing mRNA-encoding libraries of macrocyclic peptides contg. non-canonical sidechains and N-methyl-modified backbones.  The complexity of sequence space in such libraries reaches as high as a trillion (>1012), affording initial hits of high affinity ligands against protein targets.  Although this article comprehensively covers several related technologies, we discuss in greater detail the tech. development and advantages of the Random non-std. Peptide Integration Discovery (RaPID) system, including the recent identification of inhibitors against various therapeutic targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq3aPtwP6VYNrVg90H21EOLACvtfcHk0lgOrRyS9EQ-GA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXlt1GitLw%253D&md5=71b8bd1771f6d01a7fce404ec6de8db5</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.3390%2Fmolecules18033502&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fmolecules18033502%26sid%3Dliteratum%253Aachs%26aulast%3DIto%26aufirst%3DK.%26aulast%3DPassioura%26aufirst%3DT.%26aulast%3DSuga%26aufirst%3DH.%26atitle%3DTechnologies%2520for%2520the%2520Synthesis%2520of%2520mRNA-Encoding%2520Libraries%2520and%2520Discovery%2520of%2520Bioactive%2520Natural%2520Product-Inspired%2520Non-Traditional%2520Macrocyclic%2520Peptides%26jtitle%3DMolecules%26date%3D2013%26volume%3D18%26spage%3D3502%26epage%3D3528%26doi%3D10.3390%2Fmolecules18033502" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jongkees, S. A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caner, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tysoe, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brayer, G. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Withers, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suga, H.</span></span> <span> </span><span class="NLM_article-title">Rapid Discovery of Potent and Selective Glycosidase-Inhibiting De Novo Peptides</span>. <i>Cell Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">381</span>– <span class="NLM_lpage">390</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2017.02.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1016%2Fj.chembiol.2017.02.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=28262556" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC2sXjvVOktr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2017&pages=381-390&author=S.+A.+K.+Jongkeesauthor=S.+Canerauthor=C.+Tysoeauthor=G.+D.+Brayerauthor=S.+G.+Withersauthor=H.+Suga&title=Rapid+Discovery+of+Potent+and+Selective+Glycosidase-Inhibiting+De+Novo+Peptides&doi=10.1016%2Fj.chembiol.2017.02.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Rapid Discovery of Potent and Selective Glycosidase-Inhibiting De Novo Peptides</span></div><div class="casAuthors">Jongkees, Seino A. K.; Caner, Sami; Tysoe, Christina; Brayer, Gary D.; Withers, Stephen G.; Suga, Hiroaki</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">381-390</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Human pancreatic α-amylase (HPA) is responsible for degrading starch to malto-oligosaccharides, thence to glucose, and is therefore an attractive therapeutic target for the treatment of diabetes and obesity.  Here we report the discovery of a unique lariat nonapeptide, by means of the RaPID (Random non-std. Peptides Integrated Discovery) system, composed of five amino acids in a head-to-side-chain thioether macrocycle and a further four amino acids in a 310 helical C terminus.  This is a potent inhibitor of HPA (Ki = 7 nM) yet exhibits selectivity for the target over other glycosidases tested.  Structural studies show that this nonapeptide forms a compact tertiary structure, and illustrate that a general inhibitory motif involving two phenolic groups is often accessed for tight binding of inhibitors to HPA.  Furthermore, the work reported here demonstrates the potential of this methodol. for the discovery of de novo peptide inhibitors against other glycosidases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqu0khtrOxH6bVg90H21EOLACvtfcHk0lgOrRyS9EQ-GA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXjvVOktr0%253D&md5=68dff6c2cd4c8fba0387a316802fc67b</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2017.02.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2017.02.001%26sid%3Dliteratum%253Aachs%26aulast%3DJongkees%26aufirst%3DS.%2BA.%2BK.%26aulast%3DCaner%26aufirst%3DS.%26aulast%3DTysoe%26aufirst%3DC.%26aulast%3DBrayer%26aufirst%3DG.%2BD.%26aulast%3DWithers%26aufirst%3DS.%2BG.%26aulast%3DSuga%26aufirst%3DH.%26atitle%3DRapid%2520Discovery%2520of%2520Potent%2520and%2520Selective%2520Glycosidase-Inhibiting%2520De%2520Novo%2520Peptides%26jtitle%3DCell%2520Chem.%2520Biol.%26date%3D2017%26volume%3D24%26spage%3D381%26epage%3D390%26doi%3D10.1016%2Fj.chembiol.2017.02.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yamagishi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shoji, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miyagawa, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kawakami, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Katoh, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goto, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suga, H.</span></span> <span> </span><span class="NLM_article-title">Natural Product-Like Macrocyclic N-Methyl-Peptide Inhibitors against a Ubiquitin Ligase Uncovered from a Ribosome-Expressed De Novo Library</span>. <i>Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">1562</span>– <span class="NLM_lpage">1570</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2011.09.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1016%2Fj.chembiol.2011.09.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=22195558" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhs12htrnI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2011&pages=1562-1570&author=Y.+Yamagishiauthor=I.+Shojiauthor=S.+Miyagawaauthor=T.+Kawakamiauthor=T.+Katohauthor=Y.+Gotoauthor=H.+Suga&title=Natural+Product-Like+Macrocyclic+N-Methyl-Peptide+Inhibitors+against+a+Ubiquitin+Ligase+Uncovered+from+a+Ribosome-Expressed+De+Novo+Library&doi=10.1016%2Fj.chembiol.2011.09.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Natural Product-Like Macrocyclic N-Methyl-Peptide Inhibitors against a Ubiquitin Ligase Uncovered from a Ribosome-Expressed De Novo Library</span></div><div class="casAuthors">Yamagishi, Yusuke; Shoji, Ikuo; Miyagawa, Shoji; Kawakami, Takashi; Katoh, Takayuki; Goto, Yuki; Suga, Hiroaki</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1562-1570</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Summary: Naturally occurring peptides often possess macrocyclic and N-methylated backbone.  These features grant them structural rigidity, high affinity to targets, proteolytic resistance, and occasionally membrane permeability.  Because such peptides are produced by either nonribosomal peptide synthetases or enzymic posttranslational modifications, it is yet a formidable challenge in degenerating sequence or length and prepg. libraries for screening bioactive mols.  Here, we report a new means of synthesizing a de novo library of "natural product-like" macrocyclic N-methyl-peptides using translation machinery under the reprogrammed genetic code, which is coupled with an in vitro display technique, referred to as RaPID (random nonstandard peptides integrated discovery) system.  This system allows for rapid selection of strong binders against an arbitrarily chosen therapeutic target.  Here, we have demonstrated the selection of anti-E6AP macrocyclic N-methyl-peptides, one of which strongly inhibits polyubiquitination of proteins such as p53.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq5j7qHsUITV7Vg90H21EOLACvtfcHk0lgOrRyS9EQ-GA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhs12htrnI&md5=14dd193429cd72bfb8dc87a9be5a7cf9</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2011.09.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2011.09.013%26sid%3Dliteratum%253Aachs%26aulast%3DYamagishi%26aufirst%3DY.%26aulast%3DShoji%26aufirst%3DI.%26aulast%3DMiyagawa%26aufirst%3DS.%26aulast%3DKawakami%26aufirst%3DT.%26aulast%3DKatoh%26aufirst%3DT.%26aulast%3DGoto%26aufirst%3DY.%26aulast%3DSuga%26aufirst%3DH.%26atitle%3DNatural%2520Product-Like%2520Macrocyclic%2520N-Methyl-Peptide%2520Inhibitors%2520against%2520a%2520Ubiquitin%2520Ligase%2520Uncovered%2520from%2520a%2520Ribosome-Expressed%2520De%2520Novo%2520Library%26jtitle%3DChem.%2520Biol.%26date%3D2011%26volume%3D18%26spage%3D1562%26epage%3D1570%26doi%3D10.1016%2Fj.chembiol.2011.09.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hayashi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morimoto, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suga, H.</span></span> <span> </span><span class="NLM_article-title">In Vitro Selection of Anti-Akt2 Thioether-Macrocyclic Peptides Leading to Isoform-Selective Inhibitors</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">607</span>– <span class="NLM_lpage">613</span>, <span class="refDoi"> DOI: 10.1021/cb200388k</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb200388k" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtFemtrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2012&pages=607-613&author=Y.+Hayashiauthor=J.+Morimotoauthor=H.+Suga&title=In+Vitro+Selection+of+Anti-Akt2+Thioether-Macrocyclic+Peptides+Leading+to+Isoform-Selective+Inhibitors&doi=10.1021%2Fcb200388k"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">In Vitro Selection of Anti-Akt2 Thioether-Macrocyclic Peptides Leading to Isoform-Selective Inhibitors</span></div><div class="casAuthors">Hayashi, Yuuki; Morimoto, Jumpei; Suga, Hiroaki</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">607-613</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The Akt kinase family, consisting of three isoforms in humans, is a well-validated class of drug target.  Through various screening campaigns in academics and pharmaceutical industries, several promising inhibitors have been developed to date.  However, due to the mechanistic and structural similarities of Akt kinases, it is yet a challenging task to discover selective inhibitors against a specific Akt isoform.  We here report Akt-selective and also Akt2 isoform-selective inhibitors based on a thioether-macrocyclic peptide scaffold.  Several anti-Akt2 peptides have been selected from a library by means of an in vitro display system, referred to as the RaPID (Random nonstandard Peptide Integrated Discovery) system.  Remarkably, the majority of these "binding-active" anti-Akt2 peptides turned out to be "inhibitory active", exhibiting IC50 values of approx. 100 nM.  Moreover, these peptides are not only selective to the Akt kinase family but also isoform-selective to Akt2.  Particularly, one referred to as Pakti-L1 is able to discriminate Akt2 250- and 40-fold over Akt1 and Akt3, resp.  This proof-of-concept case study suggests that the RaPID system has a tremendous potential for the discovery of unique inhibitors with high family- and isoform-selectivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrR40cWg05Nh7Vg90H21EOLACvtfcHk0lgOrRyS9EQ-GA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtFemtrs%253D&md5=2d4b48ab2653f7eace723a59f206af37</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1021%2Fcb200388k&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb200388k%26sid%3Dliteratum%253Aachs%26aulast%3DHayashi%26aufirst%3DY.%26aulast%3DMorimoto%26aufirst%3DJ.%26aulast%3DSuga%26aufirst%3DH.%26atitle%3DIn%2520Vitro%2520Selection%2520of%2520Anti-Akt2%2520Thioether-Macrocyclic%2520Peptides%2520Leading%2520to%2520Isoform-Selective%2520Inhibitors%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2012%26volume%3D7%26spage%3D607%26epage%3D613%26doi%3D10.1021%2Fcb200388k" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Song, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, L.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Passioura, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suga, H.</span></span> <span> </span><span class="NLM_article-title">Macrocyclic Peptide Inhibitors for the Protein-Protein Interaction of Zaire Ebola Virus Protein 24 and Karyopherin Alpha 5</span>. <i>Org. Biomol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">5155</span>– <span class="NLM_lpage">5160</span>, <span class="refDoi"> DOI: 10.1039/C7OB00012J</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1039%2FC7OB00012J" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2017&pages=5155-5160&author=X.+Songauthor=L.-Y.+Luauthor=T.+Passiouraauthor=H.+Suga&title=Macrocyclic+Peptide+Inhibitors+for+the+Protein-Protein+Interaction+of+Zaire+Ebola+Virus+Protein+24+and+Karyopherin+Alpha+5&doi=10.1039%2FC7OB00012J"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1039%2FC7OB00012J&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC7OB00012J%26sid%3Dliteratum%253Aachs%26aulast%3DSong%26aufirst%3DX.%26aulast%3DLu%26aufirst%3DL.-Y.%26aulast%3DPassioura%26aufirst%3DT.%26aulast%3DSuga%26aufirst%3DH.%26atitle%3DMacrocyclic%2520Peptide%2520Inhibitors%2520for%2520the%2520Protein-Protein%2520Interaction%2520of%2520Zaire%2520Ebola%2520Virus%2520Protein%252024%2520and%2520Karyopherin%2520Alpha%25205%26jtitle%3DOrg.%2520Biomol.%2520Chem.%26date%3D2017%26volume%3D15%26spage%3D5155%26epage%3D5160%26doi%3D10.1039%2FC7OB00012J" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Miller, M. M.</span>; <span class="NLM_string-name">Allen, M. P.</span>; <span class="NLM_string-name">Li, L.</span>; <span class="NLM_string-name">Bowsher, M. S.</span>; <span class="NLM_string-name">Gillis, E. P.</span>; <span class="NLM_string-name">Mull, E.</span>; <span class="NLM_string-name">Zhao, Q.</span>; <span class="NLM_string-name">Sun, L.-Q.</span>; <span class="NLM_string-name">Langley, D. R.</span>; <span class="NLM_string-name">Scola, P. M.</span></span> <span> </span><span class="NLM_article-title">Macrocyclic Inhibitors of the Pd-1/Pd-L1 and Cd80/Pd-L1 Protein/Protein Interactions</span>. <span class="NLM_patent">WO2017176608 (A1)</span>, <span class="NLM_year">2017</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2017&author=M.+M.+Miller&author=M.+P.+Allen&author=L.+Li&author=M.+S.+Bowsher&author=E.+P.+Gillis&author=E.+Mull&author=Q.+Zhao&author=L.-Q.+Sun&author=D.+R.+Langley&author=P.+M.+Scola&title=Macrocyclic+Inhibitors+of+the+Pd-1%2FPd-L1+and+Cd80%2FPd-L1+Protein%2FProtein+Interactions"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DMiller%26aufirst%3DM.%2BM.%26atitle%3DMacrocyclic%2520Inhibitors%2520of%2520the%2520Pd-1%252FPd-L1%2520and%2520Cd80%252FPd-L1%2520Protein%252FProtein%2520Interactions%26date%3D2017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kawamura, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Münzel, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kojima, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yapp, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhushan, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goto, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tumber, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Katoh, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">King, O. N. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Passioura, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walport, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hatch, S. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Madden, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brennan, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chowdhury, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hopkinson, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suga, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schofield, C. J.</span></span> <span> </span><span class="NLM_article-title">Highly Selective Inhibition of Histone Demethylases by de Novo Macrocyclic Peptides</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">14773</span>, <span class="refDoi"> DOI: 10.1038/ncomms14773</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1038%2Fncomms14773" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=28382930" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC2sXlvVersro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=14773&author=A.+Kawamuraauthor=M.+M%C3%BCnzelauthor=T.+Kojimaauthor=C.+Yappauthor=B.+Bhushanauthor=Y.+Gotoauthor=A.+Tumberauthor=T.+Katohauthor=O.+N.+F.+Kingauthor=T.+Passiouraauthor=L.+J.+Walportauthor=S.+B.+Hatchauthor=S.+Maddenauthor=S.+M%C3%BCllerauthor=P.+E.+Brennanauthor=R.+Chowdhuryauthor=R.+J.+Hopkinsonauthor=H.+Sugaauthor=C.+J.+Schofield&title=Highly+Selective+Inhibition+of+Histone+Demethylases+by+de+Novo+Macrocyclic+Peptides&doi=10.1038%2Fncomms14773"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Highly selective inhibition of histone demethylases by de novo macrocyclic peptides</span></div><div class="casAuthors">Kawamura, Akane; Munzel, Martin; Kojima, Tatsuya; Yapp, Clarence; Bhushan, Bhaskar; Goto, Yuki; Tumber, Anthony; Katoh, Takayuki; King, Oliver N. F.; Passioura, Toby; Walport, Louise J.; Hatch, Stephanie B.; Madden, Sarah; Muller, Susanne; Brennan, Paul E.; Chowdhury, Rasheduzzaman; Hopkinson, Richard J.; Suga, Hiroaki; Schofield, Christopher J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Communications</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">14773</span>CODEN:
                <span class="NLM_cas:coden">NCAOBW</span>;
        ISSN:<span class="NLM_cas:issn">2041-1723</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The JmjC histone demethylases (KDMs) are linked to tumor cell proliferation and are current cancer targets; however, very few highly selective inhibitors for these are available.  Here we report cyclic peptide inhibitors of the KDM4A-C with selectivity over other KDMs/2OG oxygenases, including closely related KDM4D/E isoforms.  Crystal structures and biochem. analyses of one of the inhibitors (CP2) with KDM4A reveals that CP2 binds differently to, but competes with, histone substrates in the active site.  Substitution of the active site binding arginine of CP2 to N-ε-trimethyl-lysine or methylated arginine results in cyclic peptide substrates, indicating that KDM4s may act on non-histone substrates.  Targeted modifications to CP2 based on crystallog. and mass spectrometry analyses results in variants with greater proteolytic robustness.  Peptide dosing in cells manifests KDM4A target stabilization.  Although further development is required to optimize cellular activity, the results reveal the feasibility of highly selective non-metal chelating, substrate-competitive inhibitors of the JmjC KDMs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrsOYIV1M7RVLVg90H21EOLACvtfcHk0ljErCJGce3WBQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXlvVersro%253D&md5=3e84efae6c9715ad9ecd33f42de87911</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1038%2Fncomms14773&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms14773%26sid%3Dliteratum%253Aachs%26aulast%3DKawamura%26aufirst%3DA.%26aulast%3DM%25C3%25BCnzel%26aufirst%3DM.%26aulast%3DKojima%26aufirst%3DT.%26aulast%3DYapp%26aufirst%3DC.%26aulast%3DBhushan%26aufirst%3DB.%26aulast%3DGoto%26aufirst%3DY.%26aulast%3DTumber%26aufirst%3DA.%26aulast%3DKatoh%26aufirst%3DT.%26aulast%3DKing%26aufirst%3DO.%2BN.%2BF.%26aulast%3DPassioura%26aufirst%3DT.%26aulast%3DWalport%26aufirst%3DL.%2BJ.%26aulast%3DHatch%26aufirst%3DS.%2BB.%26aulast%3DMadden%26aufirst%3DS.%26aulast%3DM%25C3%25BCller%26aufirst%3DS.%26aulast%3DBrennan%26aufirst%3DP.%2BE.%26aulast%3DChowdhury%26aufirst%3DR.%26aulast%3DHopkinson%26aufirst%3DR.%2BJ.%26aulast%3DSuga%26aufirst%3DH.%26aulast%3DSchofield%26aufirst%3DC.%2BJ.%26atitle%3DHighly%2520Selective%2520Inhibition%2520of%2520Histone%2520Demethylases%2520by%2520de%2520Novo%2520Macrocyclic%2520Peptides%26jtitle%3DNat.%2520Commun.%26date%3D2017%26volume%3D8%26spage%3D14773%26doi%3D10.1038%2Fncomms14773" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ito, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sakai, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suzuki, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ozawa, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hatta, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Natsume, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsumoto, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suga, H.</span></span> <span> </span><span class="NLM_article-title">Artificial Human Met Agonists Based on Macrocycle Scaffolds</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">6373</span>, <span class="refDoi"> DOI: 10.1038/ncomms7373</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1038%2Fncomms7373" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=25758345" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC2MXosFegu74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=6373&author=K.+Itoauthor=K.+Sakaiauthor=Y.+Suzukiauthor=N.+Ozawaauthor=T.+Hattaauthor=T.+Natsumeauthor=K.+Matsumotoauthor=H.+Suga&title=Artificial+Human+Met+Agonists+Based+on+Macrocycle+Scaffolds&doi=10.1038%2Fncomms7373"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Artificial human Met agonists based on macrocycle scaffolds</span></div><div class="casAuthors">Ito, Kenichiro; Sakai, Katsuya; Suzuki, Yoshinori; Ozawa, Naoya; Hatta, Tomohisa; Natsume, Tohru; Matsumoto, Kunio; Suga, Hiroaki</div><div class="citationInfo"><span class="NLM_cas:title">Nature Communications</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">6373</span>CODEN:
                <span class="NLM_cas:coden">NCAOBW</span>;
        ISSN:<span class="NLM_cas:issn">2041-1723</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Hepatocyte growth factor (HGF) receptor, also known as Met, is a member of the receptor tyrosine kinase family.  The Met-HGF interaction regulates various signalling pathways involving downstream kinases, such as Akt and Erk.  Met activation is implicated in wound healing of tissues via multiple biol. responses triggered by the above-mentioned signalling cascade.  Here we report the development of artificial Met-activating dimeric macrocycles.  We identify Met-binding monomeric macrocyclic peptides by means of the RaPID (random non-std. peptide integrated discovery) system, and dimerize the resp. monomers through rational design.  These dimeric macrocycles specifically and strongly activate Met signalling pathways through receptor dimerization and induce various HGF-like cellular responses, such as branching morphogenesis, in human cells.  This work suggests our approach for generating dimeric macrocycles as non-protein ligands for cell surface receptors can be useful for developing potential therapeutics with a broad range of potential applications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrAtMddCIHHmLVg90H21EOLACvtfcHk0ljErCJGce3WBQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXosFegu74%253D&md5=6b83f797358b402d7472f989b5913d0f</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1038%2Fncomms7373&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms7373%26sid%3Dliteratum%253Aachs%26aulast%3DIto%26aufirst%3DK.%26aulast%3DSakai%26aufirst%3DK.%26aulast%3DSuzuki%26aufirst%3DY.%26aulast%3DOzawa%26aufirst%3DN.%26aulast%3DHatta%26aufirst%3DT.%26aulast%3DNatsume%26aufirst%3DT.%26aulast%3DMatsumoto%26aufirst%3DK.%26aulast%3DSuga%26aufirst%3DH.%26atitle%3DArtificial%2520Human%2520Met%2520Agonists%2520Based%2520on%2520Macrocycle%2520Scaffolds%26jtitle%3DNat.%2520Commun.%26date%3D2015%26volume%3D6%26spage%3D6373%26doi%3D10.1038%2Fncomms7373" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lyamichev, V. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goodrich, L. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sullivan, E. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bannen, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Benz, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Albert, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, J. J.</span></span> <span> </span><span class="NLM_article-title">Stepwise Evolution Improves Identification of Diverse Peptides Binding to a Protein Target</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">12116</span>, <span class="refDoi"> DOI: 10.1038/s41598-017-12440-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1038%2Fs41598-017-12440-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=28935886" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A280%3ADC%252BC1M%252FhtFSguw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2017&pages=12116&author=V.+I.+Lyamichevauthor=L.+E.+Goodrichauthor=E.+H.+Sullivanauthor=R.+M.+Bannenauthor=J.+Benzauthor=T.+J.+Albertauthor=J.+J.+Patel&title=Stepwise+Evolution+Improves+Identification+of+Diverse+Peptides+Binding+to+a+Protein+Target&doi=10.1038%2Fs41598-017-12440-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Stepwise Evolution Improves Identification of Diverse Peptides Binding to a Protein Target</span></div><div class="casAuthors">Lyamichev Victor I; Goodrich Lauren E; Sullivan Eric H; Bannen Ryan M; Albert Thomas J; Patel Jigar J; Benz Joerg</div><div class="citationInfo"><span class="NLM_cas:title">Scientific reports</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">12116</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Considerable efforts have been made to develop technologies for selection of peptidic molecules that act as substrates or binders to a protein of interest.  Here we demonstrate the combination of rational peptide array library design, parallel screening and stepwise evolution, to discover novel peptide hotspots.  These hotspots can be systematically evolved to create high-affinity, high-specificity binding peptides to a protein target in a reproducible and digitally controlled process.  The method can be applied to synthesize both linear and cyclic peptides, as well as peptides composed of natural and non-natural amino acid analogs, thereby enabling screens in a much diverse chemical space.  We apply this method to stepwise evolve peptide binders to streptavidin, a protein studied for over two decades and report novel peptides that mimic key interactions of biotin to streptavidin.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSMFn84w0HfN3thwrmtH4OBfW6udTcc2eaKn73EPCjObbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1M%252FhtFSguw%253D%253D&md5=22bc53b44626b80450bf0dad2b2c1976</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1038%2Fs41598-017-12440-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41598-017-12440-1%26sid%3Dliteratum%253Aachs%26aulast%3DLyamichev%26aufirst%3DV.%2BI.%26aulast%3DGoodrich%26aufirst%3DL.%2BE.%26aulast%3DSullivan%26aufirst%3DE.%2BH.%26aulast%3DBannen%26aufirst%3DR.%2BM.%26aulast%3DBenz%26aufirst%3DJ.%26aulast%3DAlbert%26aufirst%3DT.%2BJ.%26aulast%3DPatel%26aufirst%3DJ.%2BJ.%26atitle%3DStepwise%2520Evolution%2520Improves%2520Identification%2520of%2520Diverse%2520Peptides%2520Binding%2520to%2520a%2520Protein%2520Target%26jtitle%3DSci.%2520Rep.%26date%3D2017%26volume%3D7%26spage%3D12116%26doi%3D10.1038%2Fs41598-017-12440-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hewitt, W. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leung, S. S. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pye, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ponkey, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bednarek, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacobson, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lokey, R. S.</span></span> <span> </span><span class="NLM_article-title">Cell-Permeable Cyclic Peptides from Synthetic Libraries Inspired by Natural Products</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>137</i></span>,  <span class="NLM_fpage">715</span>– <span class="NLM_lpage">721</span>, <span class="refDoi"> DOI: 10.1021/ja508766b</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja508766b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC2cXitFOhu7zF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=137&publication_year=2015&pages=715-721&author=W.+M.+Hewittauthor=S.+S.+F.+Leungauthor=C.+R.+Pyeauthor=A.+R.+Ponkeyauthor=M.+Bednarekauthor=M.+P.+Jacobsonauthor=R.+S.+Lokey&title=Cell-Permeable+Cyclic+Peptides+from+Synthetic+Libraries+Inspired+by+Natural+Products&doi=10.1021%2Fja508766b"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Cell-Permeable Cyclic Peptides from Synthetic Libraries Inspired by Natural Products</span></div><div class="casAuthors">Hewitt, William M.; Leung, Siegfried S. F.; Pye, Cameron R.; Ponkey, Alexandra R.; Bednarek, Maria; Jacobson, Matthew P.; Lokey, R. Scott</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">137</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">715-721</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Drug design efforts are turning to a new generation of therapeutic targets, such as protein-protein interactions (PPIs), that had previously been considered "undruggable" by typical small mols.  There is an emerging view that accessing these targets will require mols. that are larger and more complex than typical small mol. drugs.  Here, the authors present a methodol. for the discovery of geometrically diverse, membrane permeable cyclic peptide scaffolds based on the synthesis and permeability screening of a combinatorial library, followed by deconvolution of membrane-permeable scaffolds to identify cyclic peptides with good to excellent passive cell permeabilities.  The authors use a combination of exptl. and computational approaches to investigate structure-permeability relationships in one of these scaffolds, and uncover structural and conformational factors that govern passive membrane diffusion in a related set of cyclic peptide diastereomers.  Further, the authors investigate the dependency of permeability on side-chain identity of one of these scaffolds through single-point diversifications to show the adaptability of these scaffolds toward development of permeability-biased libraries suitable for bioactivity screens.  Overall, the results demonstrate that many novel, cell permeable scaffolds exist beyond those found in extant natural products, and that such scaffolds can be rapidly identified using a combination of synthesis and deconvolution which can, in principle, be applied to any type of macrocyclic template.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq5wbqbKaUG9rVg90H21EOLACvtfcHk0lhdfwIwTbGFzg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXitFOhu7zF&md5=0d112bf0da119b6a50b55997f64ac214</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1021%2Fja508766b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja508766b%26sid%3Dliteratum%253Aachs%26aulast%3DHewitt%26aufirst%3DW.%2BM.%26aulast%3DLeung%26aufirst%3DS.%2BS.%2BF.%26aulast%3DPye%26aufirst%3DC.%2BR.%26aulast%3DPonkey%26aufirst%3DA.%2BR.%26aulast%3DBednarek%26aufirst%3DM.%26aulast%3DJacobson%26aufirst%3DM.%2BP.%26aulast%3DLokey%26aufirst%3DR.%2BS.%26atitle%3DCell-Permeable%2520Cyclic%2520Peptides%2520from%2520Synthetic%2520Libraries%2520Inspired%2520by%2520Natural%2520Products%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2015%26volume%3D137%26spage%3D715%26epage%3D721%26doi%3D10.1021%2Fja508766b" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bockus, A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lexa, K. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pye, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kalgutkar, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gardner, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hund, K. C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hewitt, W. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwochert, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glassey, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Price, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mathiowetz, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liras, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacobson, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lokey, R. S.</span></span> <span> </span><span class="NLM_article-title">Probing the Physicochemical Boundaries of Cell Permeability and Oral Bioavailability in Lipophilic Macrocycles Inspired by Natural Products</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">4581</span>– <span class="NLM_lpage">4589</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b00128</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b00128" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC2MXnvV2ku7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=4581-4589&author=A.+T.+Bockusauthor=K.+W.+Lexaauthor=C.+R.+Pyeauthor=A.+S.+Kalgutkarauthor=J.+W.+Gardnerauthor=K.+C.+R.+Hundauthor=W.+M.+Hewittauthor=J.+A.+Schwochertauthor=E.+Glasseyauthor=D.+A.+Priceauthor=A.+M.+Mathiowetzauthor=S.+Lirasauthor=M.+P.+Jacobsonauthor=R.+S.+Lokey&title=Probing+the+Physicochemical+Boundaries+of+Cell+Permeability+and+Oral+Bioavailability+in+Lipophilic+Macrocycles+Inspired+by+Natural+Products&doi=10.1021%2Facs.jmedchem.5b00128"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Probing the Physicochemical Boundaries of Cell Permeability and Oral Bioavailability in Lipophilic Macrocycles Inspired by Natural Products</span></div><div class="casAuthors">Bockus, Andrew T.; Lexa, Katrina W.; Pye, Cameron R.; Kalgutkar, Amit S.; Gardner, Jarret W.; Hund, Kathryn C. R.; Hewitt, William M.; Schwochert, Joshua A.; Glassey, Emerson; Price, David A.; Mathiowetz, Alan M.; Liras, Spiros; Jacobson, Matthew P.; Lokey, R. Scott</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">4581-4589</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Cyclic peptide natural products contain a variety of conserved, nonproteinogenic structural elements such as D-amino acids and amide N-methylation.  In addn., many cyclic peptides incorporate γ-amino acids and other elements derived from polyketide synthases.  We hypothesized that the position and orientation of these extended backbone elements impact the ADME properties of these hybrid mols., esp. their ability to cross cell membranes and avoid metabolic degrdn.  Here we report the synthesis of cyclic hexapeptide diastereomers contg. γ-amino acids (e.g., statines) and systematically investigate their structure-permeability relationships.  These compds. were much more water-sol. and, in many cases, were both more membrane permeable and more stable to liver microsomes than a similar non-statine-contg. deriv.  Permeability correlated well with the extent of intramol. hydrogen bonding obsd. in the soln. structures detd. in the low-dielec. solvent CDCl3, and one compd. showed an oral bioavailability of 21% in rat.  Thus, the incorporation of γ-amino acids offers a route to increase backbone diversity and improve ADME properties in cyclic peptide scaffolds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoXlzvMDhSrerVg90H21EOLACvtfcHk0lhdfwIwTbGFzg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXnvV2ku7s%253D&md5=b50bfd139a4abcae5ede988a5b095ce1</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b00128&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b00128%26sid%3Dliteratum%253Aachs%26aulast%3DBockus%26aufirst%3DA.%2BT.%26aulast%3DLexa%26aufirst%3DK.%2BW.%26aulast%3DPye%26aufirst%3DC.%2BR.%26aulast%3DKalgutkar%26aufirst%3DA.%2BS.%26aulast%3DGardner%26aufirst%3DJ.%2BW.%26aulast%3DHund%26aufirst%3DK.%2BC.%2BR.%26aulast%3DHewitt%26aufirst%3DW.%2BM.%26aulast%3DSchwochert%26aufirst%3DJ.%2BA.%26aulast%3DGlassey%26aufirst%3DE.%26aulast%3DPrice%26aufirst%3DD.%2BA.%26aulast%3DMathiowetz%26aufirst%3DA.%2BM.%26aulast%3DLiras%26aufirst%3DS.%26aulast%3DJacobson%26aufirst%3DM.%2BP.%26aulast%3DLokey%26aufirst%3DR.%2BS.%26atitle%3DProbing%2520the%2520Physicochemical%2520Boundaries%2520of%2520Cell%2520Permeability%2520and%2520Oral%2520Bioavailability%2520in%2520Lipophilic%2520Macrocycles%2520Inspired%2520by%2520Natural%2520Products%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D4581%26epage%3D4589%26doi%3D10.1021%2Facs.jmedchem.5b00128" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schwochert, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turner, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berkeley, R. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ponkey, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rodriguez, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leung, S. S. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khunte, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goetz, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Limberakis, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kalgutkar, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eng, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shapiro, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mathiowetz, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Price, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liras, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacobson, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lokey, R. S.</span></span> <span> </span><span class="NLM_article-title">Peptide to Peptoid Substitutions Increase Cell Permeability in Cyclic Hexapeptides</span>. <i>Org. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">2928</span>– <span class="NLM_lpage">2931</span>, <span class="refDoi"> DOI: 10.1021/acs.orglett.5b01162</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.orglett.5b01162" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC2MXps1arsrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2015&pages=2928-2931&author=J.+Schwochertauthor=R.+Turnerauthor=M.+Thangauthor=R.+F.+Berkeleyauthor=A.+R.+Ponkeyauthor=K.+M.+Rodriguezauthor=S.+S.+F.+Leungauthor=B.+Khunteauthor=G.+Goetzauthor=C.+Limberakisauthor=A.+S.+Kalgutkarauthor=H.+Engauthor=M.+J.+Shapiroauthor=A.+M.+Mathiowetzauthor=D.+A.+Priceauthor=S.+Lirasauthor=M.+P.+Jacobsonauthor=R.+S.+Lokey&title=Peptide+to+Peptoid+Substitutions+Increase+Cell+Permeability+in+Cyclic+Hexapeptides&doi=10.1021%2Facs.orglett.5b01162"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Peptide to Peptoid Substitutions Increase Cell Permeability in Cyclic Hexapeptides</span></div><div class="casAuthors">Schwochert, Joshua; Turner, Rushia; Thang, Melissa; Berkeley, Ray F.; Ponkey, Alexandra R.; Rodriguez, Kelsie M.; Leung, Siegfried S. F.; Khunte, Bhagyashree; Goetz, Gilles; Limberakis, Chris; Kalgutkar, Amit S.; Eng, Heather; Shapiro, Michael J.; Mathiowetz, Alan M.; Price, David A.; Liras, Spiros; Jacobson, Matthew P.; Lokey, R. Scott</div><div class="citationInfo"><span class="NLM_cas:title">Organic Letters</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2928-2931</span>CODEN:
                <span class="NLM_cas:coden">ORLEF7</span>;
        ISSN:<span class="NLM_cas:issn">1523-7052</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The effect of peptide-to-peptoid substitutions on the passive membrane permeability of an N-methylated cyclic hexapeptide is examd.  In general, substitutions maintained permeability but increased conformational heterogeneity.  Diversification with nonproteinogenic side chains increased permeability up to 3-fold.  Addnl., the conformational impact of peptoid substitutions within a β-turn are explored.  Based on these results, the strategic incorporation of peptoid residues into cyclic peptides can maintain or improve cell permeability, while increasing access to diverse side-chain functionality.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGofUG1_hYdVn7Vg90H21EOLACvtfcHk0liFz0wLamnbuQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXps1arsrs%253D&md5=156985175ddcafd34a67e81713934040</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1021%2Facs.orglett.5b01162&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.orglett.5b01162%26sid%3Dliteratum%253Aachs%26aulast%3DSchwochert%26aufirst%3DJ.%26aulast%3DTurner%26aufirst%3DR.%26aulast%3DThang%26aufirst%3DM.%26aulast%3DBerkeley%26aufirst%3DR.%2BF.%26aulast%3DPonkey%26aufirst%3DA.%2BR.%26aulast%3DRodriguez%26aufirst%3DK.%2BM.%26aulast%3DLeung%26aufirst%3DS.%2BS.%2BF.%26aulast%3DKhunte%26aufirst%3DB.%26aulast%3DGoetz%26aufirst%3DG.%26aulast%3DLimberakis%26aufirst%3DC.%26aulast%3DKalgutkar%26aufirst%3DA.%2BS.%26aulast%3DEng%26aufirst%3DH.%26aulast%3DShapiro%26aufirst%3DM.%2BJ.%26aulast%3DMathiowetz%26aufirst%3DA.%2BM.%26aulast%3DPrice%26aufirst%3DD.%2BA.%26aulast%3DLiras%26aufirst%3DS.%26aulast%3DJacobson%26aufirst%3DM.%2BP.%26aulast%3DLokey%26aufirst%3DR.%2BS.%26atitle%3DPeptide%2520to%2520Peptoid%2520Substitutions%2520Increase%2520Cell%2520Permeability%2520in%2520Cyclic%2520Hexapeptides%26jtitle%3DOrg.%2520Lett.%26date%3D2015%26volume%3D17%26spage%3D2928%26epage%3D2931%26doi%3D10.1021%2Facs.orglett.5b01162" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Weinmüller, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rechenmacher, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kiran Marelli, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reichart, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kapp, T. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Räder, A. F. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Di Leva, F. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marinelli, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Novellino, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muñoz-Félix, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hodivala-Dilke, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schumacher, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fanous, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilon, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoffman, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kessler, H.</span></span> <span> </span><span class="NLM_article-title">Overcoming the Lack of Oral Availability of Cyclic Hexapeptides: Design of a Selective and Orally Available Ligand for the Integrin αvβ3</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">16405</span>– <span class="NLM_lpage">16409</span>, <span class="refDoi"> DOI: 10.1002/anie.201709709</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1002%2Fanie.201709709" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvVKqtLjF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2017&pages=16405-16409&author=M.+Weinm%C3%BCllerauthor=F.+Rechenmacherauthor=U.+Kiran+Marelliauthor=F.+Reichartauthor=T.+G.+Kappauthor=A.+F.+B.+R%C3%A4derauthor=F.+S.+Di+Levaauthor=L.+Marinelliauthor=E.+Novellinoauthor=J.+M.+Mu%C3%B1oz-F%C3%A9lixauthor=K.+Hodivala-Dilkeauthor=A.+Schumacherauthor=J.+Fanousauthor=C.+Gilonauthor=A.+Hoffmanauthor=H.+Kessler&title=Overcoming+the+Lack+of+Oral+Availability+of+Cyclic+Hexapeptides%3A+Design+of+a+Selective+and+Orally+Available+Ligand+for+the+Integrin+%CE%B1v%CE%B23&doi=10.1002%2Fanie.201709709"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Overcoming the Lack of Oral Availability of Cyclic Hexapeptides: Design of a Selective and Orally Available Ligand for the Integrin αvβ3</span></div><div class="casAuthors">Weinmueller, Michael; Rechenmacher, Florian; Kiran Marelli, Udaya; Reichart, Florian; Kapp, Tobias G.; Raeder, Andreas F. B.; Di Leva, Francesco Saverio; Marinelli, Luciana; Novellino, Ettore; Munoz-Felix, Jose M.; Hodivala-Dilke, Kairbaan; Schumacher, Adi; Fanous, Joseph; Gilon, Chaim; Hoffman, Amnon; Kessler, Horst</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">51</span>),
    <span class="NLM_cas:pages">16405-16409</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A highly systematic approach for the development of both orally bioavailable and bioactive cyclic N-methylated hexapeptides as high affinity ligands for the integrin αvβ3 is based on two concepts: (a) screening of systematically designed libraries with spatial diversity and (b) masking of the peptide charge with a lipophilic protecting group.  The key steps of the method are (1) initial design of a combinatorial library of N-methylated analogs of the stem peptide cyclo(D-Ala-Ala5); (2) selection of cyclic peptides with the highest intestinal permeability; (3) design of sublibraries with the bioactive RGD sequence in all possible positions; (4) selection of the best ligands for RGD-recognizing integrin subtypes; (5) fine-tuning of the affinity and selectivity by addnl. Ala to Xaa substitutions; (6) protection of the charged functional groups according to the prodrug concept to regain intestinal and oral permeability; (7) proof of biol. effects in mice after oral administration.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpUvvbSmX8C4rVg90H21EOLACvtfcHk0liFz0wLamnbuQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvVKqtLjF&md5=acd048e94a0525991c7019a2e334b60b</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1002%2Fanie.201709709&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201709709%26sid%3Dliteratum%253Aachs%26aulast%3DWeinm%25C3%25BCller%26aufirst%3DM.%26aulast%3DRechenmacher%26aufirst%3DF.%26aulast%3DKiran%2BMarelli%26aufirst%3DU.%26aulast%3DReichart%26aufirst%3DF.%26aulast%3DKapp%26aufirst%3DT.%2BG.%26aulast%3DR%25C3%25A4der%26aufirst%3DA.%2BF.%2BB.%26aulast%3DDi%2BLeva%26aufirst%3DF.%2BS.%26aulast%3DMarinelli%26aufirst%3DL.%26aulast%3DNovellino%26aufirst%3DE.%26aulast%3DMu%25C3%25B1oz-F%25C3%25A9lix%26aufirst%3DJ.%2BM.%26aulast%3DHodivala-Dilke%26aufirst%3DK.%26aulast%3DSchumacher%26aufirst%3DA.%26aulast%3DFanous%26aufirst%3DJ.%26aulast%3DGilon%26aufirst%3DC.%26aulast%3DHoffman%26aufirst%3DA.%26aulast%3DKessler%26aufirst%3DH.%26atitle%3DOvercoming%2520the%2520Lack%2520of%2520Oral%2520Availability%2520of%2520Cyclic%2520Hexapeptides%253A%2520Design%2520of%2520a%2520Selective%2520and%2520Orally%2520Available%2520Ligand%2520for%2520the%2520Integrin%2520%25CE%25B1v%25CE%25B23%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2017%26volume%3D56%26spage%3D16405%26epage%3D16409%26doi%3D10.1002%2Fanie.201709709" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Di, L.</span></span> <span> </span><span class="NLM_article-title">Strategic Approaches to Optimizing Peptide ADME Properties</span>. <i>AAPS J.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">134</span>– <span class="NLM_lpage">143</span>, <span class="refDoi"> DOI: 10.1208/s12248-014-9687-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1208%2Fs12248-014-9687-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=25366889" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvFSnurvJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2015&pages=134-143&author=L.+Di&title=Strategic+Approaches+to+Optimizing+Peptide+ADME+Properties&doi=10.1208%2Fs12248-014-9687-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Strategic Approaches to Optimizing Peptide ADME Properties</span></div><div class="casAuthors">Di, Li</div><div class="citationInfo"><span class="NLM_cas:title">AAPS Journal</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">134-143</span>CODEN:
                <span class="NLM_cas:coden">AJAOB6</span>;
        ISSN:<span class="NLM_cas:issn">1550-7416</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Development of peptide drugs is challenging but also quite rewarding.  Five blockbuster peptide drugs are currently on the market, and six new peptides received first marketing approval as new mol. entities in 2012.  Although peptides only represent 2% of the drug market, the market is growing twice as quickly and might soon occupy a larger niche.  Natural peptides typically have poor absorption, distribution, metab., and excretion (ADME) properties with rapid clearance, short half-life, low permeability, and sometimes low soly.  Strategies have been developed to improve peptide drugability through enhancing permeability, reducing proteolysis and renal clearance, and prolonging half-life.  In vivo, in vitro, and in silico tools are available to evaluate ADME properties of peptides, and structural modification strategies are in place to improve peptide developability.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpDLDsR-Zk0OLVg90H21EOLACvtfcHk0liFz0wLamnbuQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvFSnurvJ&md5=530597d3719d65509105096f433fee0a</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1208%2Fs12248-014-9687-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1208%252Fs12248-014-9687-3%26sid%3Dliteratum%253Aachs%26aulast%3DDi%26aufirst%3DL.%26atitle%3DStrategic%2520Approaches%2520to%2520Optimizing%2520Peptide%2520ADME%2520Properties%26jtitle%3DAAPS%2520J.%26date%3D2015%26volume%3D17%26spage%3D134%26epage%3D143%26doi%3D10.1208%2Fs12248-014-9687-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">van
Witteloostuijn, S. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pedersen, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jensen, K. J.</span></span> <span> </span><span class="NLM_article-title">Half-Life Extension of Biopharmaceuticals Using Chemical Methods: Alternatives to PEGylation</span>. <i>ChemMedChem</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">2474</span>– <span class="NLM_lpage">2495</span>, <span class="refDoi"> DOI: 10.1002/cmdc.201600374</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1002%2Fcmdc.201600374" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=27775236" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC28XhslSqtrbJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2016&pages=2474-2495&author=S.+B.+van%0AWitteloostuijnauthor=S.+L.+Pedersenauthor=K.+J.+Jensen&title=Half-Life+Extension+of+Biopharmaceuticals+Using+Chemical+Methods%3A+Alternatives+to+PEGylation&doi=10.1002%2Fcmdc.201600374"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Half-Life Extension of Biopharmaceuticals using Chemical Methods: Alternatives to PEGylation</span></div><div class="casAuthors">van Witteloostuijn, Soren B.; Pedersen, Soren L.; Jensen, Knud J.</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">2474-2495</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A review.  Peptides and proteins constitute a vast pool of excellent drug candidates.  Evolution has equipped these mols. with superior drug-like properties such as high specificity and potency.  However, native peptides and proteins suffer from an inadequate pharmacokinetic profile, and their outstanding pharmacol. potential can only be realized if this issue is addressed during drug development.  To overcome this challenge, a variety of half-life extension techniques relying on covalent chem. modification have been developed.  These methods include PEGylation, fusion to unstructured polypeptide-based PEG mimetics, conjugation of large polysaccharides, native-like glycosylation, lipidation, fusion to albumin or the Fc domain of IgG, and derivatization with bio-orthogonal moieties that direct self-assembly.  This review provides an overview of available conjugation chemistries, biophys. properties, and safety data assocd. with these concepts.  Moreover, the effects of these modifications on peptide and protein pharmacokinetics are demonstrated through key examples.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGodYDhxsEiYZrVg90H21EOLACvtfcHk0liFz0wLamnbuQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhslSqtrbJ&md5=b3811dad7e9a100fa235d8eebf7272c6</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.201600374&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.201600374%26sid%3Dliteratum%253Aachs%26aulast%3Dvan%2BWitteloostuijn%26aufirst%3DS.%2BB.%26aulast%3DPedersen%26aufirst%3DS.%2BL.%26aulast%3DJensen%26aufirst%3DK.%2BJ.%26atitle%3DHalf-Life%2520Extension%2520of%2520Biopharmaceuticals%2520Using%2520Chemical%2520Methods%253A%2520Alternatives%2520to%2520PEGylation%26jtitle%3DChemMedChem%26date%3D2016%26volume%3D11%26spage%3D2474%26epage%3D2495%26doi%3D10.1002%2Fcmdc.201600374" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">King, G. F.</span></span> <span> </span><span class="NLM_article-title">Venoms as a Platform for Human Drugs: Translating Toxins into Therapeutics</span>. <i>Expert Opin. Biol. Ther.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">1469</span>– <span class="NLM_lpage">1484</span>, <span class="refDoi"> DOI: 10.1517/14712598.2011.621940</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1517%2F14712598.2011.621940" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=21939428" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC3MXht1ylsrjF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2011&pages=1469-1484&author=G.+F.+King&title=Venoms+as+a+Platform+for+Human+Drugs%3A+Translating+Toxins+into+Therapeutics&doi=10.1517%2F14712598.2011.621940"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Venoms as a platform for human drugs: translating toxins into therapeutics</span></div><div class="casAuthors">King, Glenn F.</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Biological Therapy</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1469-1484</span>CODEN:
                <span class="NLM_cas:coden">EOBTA2</span>;
        ISSN:<span class="NLM_cas:issn">1471-2598</span>.
    
            (<span class="NLM_cas:orgname">Informa Healthcare</span>)
        </div><div class="casAbstract">A review.  Introduction:An extraordinarily diverse range of animals have evolved venoms for predation, defense, or competitor deterrence.  The major components of most venoms are peptides and proteins that are often protease-resistant due to their disulfide-rich architectures.  Some of these toxins have become valuable as pharmacol. tools and/or therapeutics due to their extremely high specificity and potency for particular mol. targets.  There are currently six FDA-approved drugs derived from venom peptides or proteins.  Areas covered:This article surveys the current pipeline of venom-derived therapeutics and critically examines the potential of peptide and protein drugs derived from venoms.  Emerging trends are identified, including an increasing industry focus on disulfide-rich venom peptides and the use of a broader array of mol. targets in order to develop venom-based therapeutics for treating a wider range of clin. conditions.  Expert opinion:Key tech. advances in combination with a renewed industry-wide focus on biologics have converged to provide a larger than ever pipeline of venom-derived therapeutics.  Disulfide-rich venom peptides obviate some of the traditional disadvantages of therapeutic peptides and some may be suitable for oral administration.  Moreover, some venom peptides can breach the blood brain barrier and translocate across cell membranes, which opens up the possibility of exploiting mol. targets not previously accessible to peptide drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpoYClUIGUrZbVg90H21EOLACvtfcHk0li0DqpW8xUerg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXht1ylsrjF&md5=4983d10289108fac346231191e6169f7</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1517%2F14712598.2011.621940&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1517%252F14712598.2011.621940%26sid%3Dliteratum%253Aachs%26aulast%3DKing%26aufirst%3DG.%2BF.%26atitle%3DVenoms%2520as%2520a%2520Platform%2520for%2520Human%2520Drugs%253A%2520Translating%2520Toxins%2520into%2520Therapeutics%26jtitle%3DExpert%2520Opin.%2520Biol.%2520Ther.%26date%3D2011%26volume%3D11%26spage%3D1469%26epage%3D1484%26doi%3D10.1517%2F14712598.2011.621940" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pennington, M. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Czerwinski, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Norton, R. S.</span></span> <span> </span><span class="NLM_article-title">Peptide Therapeutics from Venom: Current Status and Potential</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">2738</span>– <span class="NLM_lpage">2758</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2017.09.029</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1016%2Fj.bmc.2017.09.029" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=28988749" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhs1aitrzI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2018&pages=2738-2758&author=M.+W.+Penningtonauthor=A.+Czerwinskiauthor=R.+S.+Norton&title=Peptide+Therapeutics+from+Venom%3A+Current+Status+and+Potential&doi=10.1016%2Fj.bmc.2017.09.029"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Peptide therapeutics from venom: Current status and potential</span></div><div class="casAuthors">Pennington, Michael W.; Czerwinski, Andrzej; Norton, Raymond S.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2738-2758</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Peptides are recognized as being highly selective, potent and relatively safe as potential therapeutics.  Peptides isolated from the venom of different animals satisfy most of these criteria with the possible exception of safety, but when isolated as single compds. and used at appropriate concns., venom-derived peptides can become useful drugs.  Although the no. of venom-derived peptides that have successfully progressed to the clinic is currently limited, the prospects for venom-derived peptides look very optimistic.  As proteomic and transcriptomic approaches continue to identify new sequences, the potential of venom-derived peptides to find applications as therapeutics, cosmetics and insecticides grows accordingly.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp8haCt_E4Tu7Vg90H21EOLACvtfcHk0li0DqpW8xUerg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhs1aitrzI&md5=253b75540d1a2117338259c087b119f7</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2017.09.029&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2017.09.029%26sid%3Dliteratum%253Aachs%26aulast%3DPennington%26aufirst%3DM.%2BW.%26aulast%3DCzerwinski%26aufirst%3DA.%26aulast%3DNorton%26aufirst%3DR.%2BS.%26atitle%3DPeptide%2520Therapeutics%2520from%2520Venom%253A%2520Current%2520Status%2520and%2520Potential%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2018%26volume%3D26%26spage%3D2738%26epage%3D2758%26doi%3D10.1016%2Fj.bmc.2017.09.029" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Campolina
Rebello Horta, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chatzaki, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Almeida Rezende, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Freitas Magalhães, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guerra Duarte, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Figueiredo Felicori, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ribeiro Oliveira-Mendes, B. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oliveira
do Carmo, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chávez-Olórtegui, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kalapothakis, E.</span></span> <span> </span><span class="NLM_article-title">Cardiovascular-Active Venom Toxins: An Overview</span>. <i>Curr. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">603</span>– <span class="NLM_lpage">622</span>, <span class="refDoi"> DOI: 10.2174/0929867323666160126142837</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.2174%2F0929867323666160126142837" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=26812904" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2016&pages=603-622&author=C.+Campolina%0ARebello+Hortaauthor=M.+Chatzakiauthor=B.+Almeida+Rezendeauthor=B.+de+Freitas+Magalh%C3%A3esauthor=C.+Guerra+Duarteauthor=L.+Figueiredo+Felicoriauthor=B.+B.+Ribeiro+Oliveira-Mendesauthor=A.+Oliveira%0Ado+Carmoauthor=C.+Ch%C3%A1vez-Ol%C3%B3rteguiauthor=E.+Kalapothakis&title=Cardiovascular-Active+Venom+Toxins%3A+An+Overview&doi=10.2174%2F0929867323666160126142837"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.2174%2F0929867323666160126142837&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F0929867323666160126142837%26sid%3Dliteratum%253Aachs%26aulast%3DCampolina%2BRebello%2BHorta%26aufirst%3DC.%26aulast%3DChatzaki%26aufirst%3DM.%26aulast%3DAlmeida%2BRezende%26aufirst%3DB.%26aulast%3Dde%2BFreitas%2BMagalh%25C3%25A3es%26aufirst%3DB.%26aulast%3DGuerra%2BDuarte%26aufirst%3DC.%26aulast%3DFigueiredo%2BFelicori%26aufirst%3DL.%26aulast%3DRibeiro%2BOliveira-Mendes%26aufirst%3DB.%2BB.%26aulast%3DOliveira%2Bdo%2BCarmo%26aufirst%3DA.%26aulast%3DCh%25C3%25A1vez-Ol%25C3%25B3rtegui%26aufirst%3DC.%26aulast%3DKalapothakis%26aufirst%3DE.%26atitle%3DCardiovascular-Active%2520Venom%2520Toxins%253A%2520An%2520Overview%26jtitle%3DCurr.%2520Med.%2520Chem.%26date%3D2016%26volume%3D23%26spage%3D603%26epage%3D622%26doi%3D10.2174%2F0929867323666160126142837" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref54"><div class="reference"><strong class="refLabel"><a href="#ref54" class="rightTabRefNumLink">54</a></strong><div class="NLM_citation" id="rightTab-cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lewis, R. J.</span></span> <span> </span><span class="NLM_article-title">Case Study 1: Development of the Analgesic Drugs Prialt® and Xen2174 from Cone Snail Venoms</span>. In  <i>Venoms to Drugs</i>; <span class="NLM_contrib-group"><span class="NLM_string-name">King, G. F.</span></span>, Ed.; <span class="NLM_publisher-name">The Royal Society of Chemistry</span>: <span class="NLM_publisher-loc">Cambridge, U.K.</span>, <span class="NLM_year">2015</span>; pp  <span class="NLM_fpage">245</span>– <span class="NLM_lpage">254</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1039%2F9781849737876-00245" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2015&pages=245-254&author=R.+J.+Lewisauthor=G.+F.+King&title=Venoms+to+Drugs"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1039%2F9781849737876-00245&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1039%252F9781849737876-00245%26sid%3Dliteratum%253Aachs%26aulast%3DLewis%26aufirst%3DR.%2BJ.%26atitle%3DCase%2520Study%25201%253A%2520Development%2520of%2520the%2520Analgesic%2520Drugs%2520Prialt%25C2%25AE%2520and%2520Xen2174%2520from%2520Cone%2520Snail%2520Venoms%26btitle%3DVenoms%2520to%2520Drugs%26aulast%3DKing%26aufirst%3DG.%2BF.%26pub%3DThe%2520Royal%2520Society%2520of%2520Chemistry%26date%3D2015%26spage%3D245%26epage%3D254" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref55"><div class="reference"><strong class="refLabel"><a href="#ref55" class="rightTabRefNumLink">55</a></strong><div class="NLM_citation" id="rightTab-cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sanggaard, K. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dyrlund, T. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomsen, L. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nielsen, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brøndum, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thøgersen, I. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Enghild, J. J.</span></span> <span> </span><span class="NLM_article-title">Characterization of the Gila Monster (Heloderma Suspectum Suspectum) Venom Proteome</span>. <i>J. Proteomics</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>117</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">11</span>, <span class="refDoi"> DOI: 10.1016/j.jprot.2015.01.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1016%2Fj.jprot.2015.01.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=25603280" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtlWgu7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=117&publication_year=2015&pages=1-11&author=K.+W.+Sanggaardauthor=T.+F.+Dyrlundauthor=L.+R.+Thomsenauthor=T.+A.+Nielsenauthor=L.+Br%C3%B8ndumauthor=T.+Wangauthor=I.+B.+Th%C3%B8gersenauthor=J.+J.+Enghild&title=Characterization+of+the+Gila+Monster+%28Heloderma+Suspectum+Suspectum%29+Venom+Proteome&doi=10.1016%2Fj.jprot.2015.01.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Characterization of the gila monster (Heloderma suspectum suspectum) venom proteome</span></div><div class="casAuthors">Sanggaard, Kristian W.; Dyrlund, Thomas F.; Thomsen, Line R.; Nielsen, Tania A.; Broendum, Lars; Wang, Tobias; Thoegersen, Ida B.; Enghild, Jan J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Proteomics</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">117</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">1-11</span>CODEN:
                <span class="NLM_cas:coden">JPORFQ</span>;
        ISSN:<span class="NLM_cas:issn">1874-3919</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The archetypical venomous lizard species are the helodermatids, the gila monsters (Heloderma suspectum) and the beaded lizards (Heloderma horridum).  In the present study, the gila monster venom proteome was characterized using 2D-gel electrophoresis and tandem mass spectrometry-based de novo peptide sequencing followed by protein identification based on sequence homol.  A total of 39 different proteins were identified out of the 58 selected spots that represent the major constituents of venom.  Of these proteins, 19 have not previously been identified in helodermatid venom.  The data showed that helodermatid venom is complex and that this complexity is caused by genetic isoforms and post-translational modifications including proteolytic processing.  In addn., the venom proteome anal. revealed that the major constituents of the gila monster venom are kallikrein-like serine proteinases (EC 3.4.21) and phospholipase A2 (type III) enzymes (EC 3.1.1.4).  A neuroendocrine convertase 1 homolog that most likely converts the proforms of the previously identified bioactive exendins into the mature and active forms was identified suggesting that these peptide toxins are secreted as proforms that are activated by proteolytic cleavage following secretion as opposed to being activated intracellularly.  The presented global protein identification-anal. provides the first overview of the helodermatid venom compn.The helodermatid lizards are the classical venomous lizards, and the pharmacol. potential of the venom from these species has been known for years; best illustrated by the identification of exendin-4, which is now used in the treatment of type 2 diabetes.  Despite the potential, no global analyses of the protein components in the venom exist.  A hindrance is the lack of a genome sequence because it prevents protein identification using a conventional approach where MS data are searched against predicted protein sequences based on the genome sequence.  However, in the recent years the development of software tools for de novo sequencing and homol. searches have improved significantly facilitating the first global anal. of the major protein components of helodermatid venom presented in this study.  We have used a 2D-gel approach and detd. the protein components in the 58 major spots resulting in the identification of 39 unique proteins.  Of these, 19 have not previously been identified in helodermatid venom.  The anal. provides results with impact on our understanding of the function and evolution of venom proteins, and serves as a basis for further unraveling of the pharmaceutical potential of the venom components.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq0jT-qNadT-rVg90H21EOLACvtfcHk0liRipwTFJrVGg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtlWgu7g%253D&md5=7b20f196b8301fa4d769001e876ea332</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1016%2Fj.jprot.2015.01.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jprot.2015.01.004%26sid%3Dliteratum%253Aachs%26aulast%3DSanggaard%26aufirst%3DK.%2BW.%26aulast%3DDyrlund%26aufirst%3DT.%2BF.%26aulast%3DThomsen%26aufirst%3DL.%2BR.%26aulast%3DNielsen%26aufirst%3DT.%2BA.%26aulast%3DBr%25C3%25B8ndum%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DT.%26aulast%3DTh%25C3%25B8gersen%26aufirst%3DI.%2BB.%26aulast%3DEnghild%26aufirst%3DJ.%2BJ.%26atitle%3DCharacterization%2520of%2520the%2520Gila%2520Monster%2520%2528Heloderma%2520Suspectum%2520Suspectum%2529%2520Venom%2520Proteome%26jtitle%3DJ.%2520Proteomics%26date%3D2015%26volume%3D117%26spage%3D1%26epage%3D11%26doi%3D10.1016%2Fj.jprot.2015.01.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref56"><div class="reference"><strong class="refLabel"><a href="#ref56" class="rightTabRefNumLink">56</a></strong><div class="NLM_citation" id="rightTab-cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Akondi, K. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muttenthaler, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dutertre, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaas, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Craik, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lewis, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alewood, P. F.</span></span> <span> </span><span class="NLM_article-title">Discovery, Synthesis, and Structure-Activity Relationships of Conotoxins</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>114</i></span>,  <span class="NLM_fpage">5815</span>– <span class="NLM_lpage">5847</span>, <span class="refDoi"> DOI: 10.1021/cr400401e</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cr400401e" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC2cXmtVegt78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=114&publication_year=2014&pages=5815-5847&author=K.+B.+Akondiauthor=M.+Muttenthalerauthor=S.+Dutertreauthor=Q.+Kaasauthor=D.+J.+Craikauthor=R.+J.+Lewisauthor=P.+F.+Alewood&title=Discovery%2C+Synthesis%2C+and+Structure-Activity+Relationships+of+Conotoxins&doi=10.1021%2Fcr400401e"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery, Synthesis, and Structure-Activity Relationships of Conotoxins</span></div><div class="casAuthors">Akondi, Kalyana B.; Muttenthaler, Markus; Dutertre, Sebastien; Kaas, Quentin; Craik, David J.; Lewis, Richard J.; Alewood, Paul F.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">114</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">5815-5847</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Venom peptides, esp. conotoxins isolated from the venom of predatory marine snails, hold enormous promise for the development of peptide-based drugs.  Conotoxin classification, nomenclature, synthesis, structure activity relationships, and as candidates for drugs are reviewed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqBcE_sYX4b2LVg90H21EOLACvtfcHk0liRipwTFJrVGg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXmtVegt78%253D&md5=02f1253f5605757259436d2ecefd42a6</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1021%2Fcr400401e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcr400401e%26sid%3Dliteratum%253Aachs%26aulast%3DAkondi%26aufirst%3DK.%2BB.%26aulast%3DMuttenthaler%26aufirst%3DM.%26aulast%3DDutertre%26aufirst%3DS.%26aulast%3DKaas%26aufirst%3DQ.%26aulast%3DCraik%26aufirst%3DD.%2BJ.%26aulast%3DLewis%26aufirst%3DR.%2BJ.%26aulast%3DAlewood%26aufirst%3DP.%2BF.%26atitle%3DDiscovery%252C%2520Synthesis%252C%2520and%2520Structure-Activity%2520Relationships%2520of%2520Conotoxins%26jtitle%3DChem.%2520Rev.%26date%3D2014%26volume%3D114%26spage%3D5815%26epage%3D5847%26doi%3D10.1021%2Fcr400401e" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref57"><div class="reference"><strong class="refLabel"><a href="#ref57" class="rightTabRefNumLink">57</a></strong><div class="NLM_citation" id="rightTab-cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ciolek, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reinfrank, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quinton, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Viengchareun, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stura, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vera, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sigismeau, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mouillac, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Orcel, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peigneur, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tytgat, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Droctové, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beau, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nevoux, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lombès, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mourier, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Pauw, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Servent, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mendre, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Witzgall, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilles, N.</span></span> <span> </span><span class="NLM_article-title">Green Mamba Peptide Targets Type-2 Vasopressin Receptor against Polycystic Kidney Disease</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>114</i></span>,  <span class="NLM_fpage">7154</span>– <span class="NLM_lpage">7159</span>, <span class="refDoi"> DOI: 10.1073/pnas.1620454114</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1073%2Fpnas.1620454114" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=28630289" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtVWmtL3N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=114&publication_year=2017&pages=7154-7159&author=J.+Ciolekauthor=H.+Reinfrankauthor=L.+Quintonauthor=S.+Viengchareunauthor=E.+A.+Sturaauthor=L.+Veraauthor=S.+Sigismeauauthor=B.+Mouillacauthor=H.+Orcelauthor=S.+Peigneurauthor=J.+Tytgatauthor=L.+Droctov%C3%A9author=F.+Beauauthor=J.+Nevouxauthor=M.+Lomb%C3%A8sauthor=G.+Mourierauthor=E.+De+Pauwauthor=D.+Serventauthor=C.+Mendreauthor=R.+Witzgallauthor=N.+Gilles&title=Green+Mamba+Peptide+Targets+Type-2+Vasopressin+Receptor+against+Polycystic+Kidney+Disease&doi=10.1073%2Fpnas.1620454114"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">Green mamba peptide targets type-2 vasopressin receptor against polycystic kidney disease</span></div><div class="casAuthors">Ciolek, Justyna; Reinfrank, Helen; Quinton, Loic; Viengchareun, Say; Stura, Enrico A.; Vera, Laura; Sigismeau, Sabrina; Mouillac, Bernard; Orcel, Helene; Peigneur, Steve; Tytgat, Jan; Droctove, Laura; Beau, Fabrice; Nevoux, Jerome; Lombes, Marc; Mourier, Gilles; De Pauw, Edwin; Servent, Denis; Mendre, Christiane; Witzgall, Ralph; Gilles, Nicolas</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">114</span>
        (<span class="NLM_cas:issue">27</span>),
    <span class="NLM_cas:pages">7154-7159</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Polycystic kidney diseases (PKDs) are genetic disorders that can cause renal failure and death in children and adults.  Lowering cAMP in cystic tissues through the inhibition of the type-2 vasopressin receptor (V2R) constitutes a validated strategy to reduce disease progression.  The authors identified a peptide from green mamba venom that exhibits nanomolar affinity for the V2R without any activity on 155 other G-protein-coupled receptors or on 15 ionic channels.  Mambaquaretin-1 is a full antagonist of the V2R activation pathways studied: cAMP prodn., beta-arrestin interaction, and MAP kinase activity.  This peptide adopts the Kunitz fold known to mostly act on potassium channels and serine proteases.  Mambaquaretin-1 interacts selectively with the V2R through its first loop, in the same manner that aprotinin inhibits trypsin.  Injected in mice, mambaquaretin-1 increases in a dose-dependent manner urine outflow with concomitant redn. of urine osmolality, indicating a purely aquaretic effect assocd. with the in vivo blockade of V2R.  CD1-pcy/pcy mice, a juvenile model of PKD, daily treated with 13 μg of mambaquaretin-1 for 99 d, developed less abundant (by 33%) and smaller (by 47%) cysts than control mice.  Neither tachyphylaxis nor apparent toxicity has been noted.  Mambaquaretin-1 represents a promising therapeutic agent against PKDs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqI-J7A636tYrVg90H21EOLACvtfcHk0liRipwTFJrVGg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtVWmtL3N&md5=2cad6950c107df021baf5a66c6b086b0</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1620454114&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1620454114%26sid%3Dliteratum%253Aachs%26aulast%3DCiolek%26aufirst%3DJ.%26aulast%3DReinfrank%26aufirst%3DH.%26aulast%3DQuinton%26aufirst%3DL.%26aulast%3DViengchareun%26aufirst%3DS.%26aulast%3DStura%26aufirst%3DE.%2BA.%26aulast%3DVera%26aufirst%3DL.%26aulast%3DSigismeau%26aufirst%3DS.%26aulast%3DMouillac%26aufirst%3DB.%26aulast%3DOrcel%26aufirst%3DH.%26aulast%3DPeigneur%26aufirst%3DS.%26aulast%3DTytgat%26aufirst%3DJ.%26aulast%3DDroctov%25C3%25A9%26aufirst%3DL.%26aulast%3DBeau%26aufirst%3DF.%26aulast%3DNevoux%26aufirst%3DJ.%26aulast%3DLomb%25C3%25A8s%26aufirst%3DM.%26aulast%3DMourier%26aufirst%3DG.%26aulast%3DDe%2BPauw%26aufirst%3DE.%26aulast%3DServent%26aufirst%3DD.%26aulast%3DMendre%26aufirst%3DC.%26aulast%3DWitzgall%26aufirst%3DR.%26aulast%3DGilles%26aufirst%3DN.%26atitle%3DGreen%2520Mamba%2520Peptide%2520Targets%2520Type-2%2520Vasopressin%2520Receptor%2520against%2520Polycystic%2520Kidney%2520Disease%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2017%26volume%3D114%26spage%3D7154%26epage%3D7159%26doi%3D10.1073%2Fpnas.1620454114" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref58"><div class="reference"><strong class="refLabel"><a href="#ref58" class="rightTabRefNumLink">58</a></strong><div class="NLM_citation" id="rightTab-cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Diochot, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baron, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salinas, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Douguet, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scarzello, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dabert-Gay, A.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Debayle, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friend, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alloui, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lazdunski, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lingueglia, E.</span></span> <span> </span><span class="NLM_article-title">Black Mamba Venom Peptides Target Acid-Sensing Ion Channels to Abolish Pain</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>490</i></span>,  <span class="NLM_fpage">552</span>– <span class="NLM_lpage">555</span>, <span class="refDoi"> DOI: 10.1038/nature11494</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1038%2Fnature11494" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=23034652" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsVGjsrnL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=490&publication_year=2012&pages=552-555&author=S.+Diochotauthor=A.+Baronauthor=M.+Salinasauthor=D.+Douguetauthor=S.+Scarzelloauthor=A.-S.+Dabert-Gayauthor=D.+Debayleauthor=V.+Friendauthor=A.+Allouiauthor=M.+Lazdunskiauthor=E.+Lingueglia&title=Black+Mamba+Venom+Peptides+Target+Acid-Sensing+Ion+Channels+to+Abolish+Pain&doi=10.1038%2Fnature11494"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">Black mamba venom peptides target acid-sensing ion channels to abolish pain</span></div><div class="casAuthors">Diochot, Sylvie; Baron, Anne; Salinas, Miguel; Douguet, Dominique; Scarzello, Sabine; Dabert-Gay, Anne-Sophie; Debayle, Delphine; Friend, Valerie; Alloui, Abdelkrim; Lazdunski, Michel; Lingueglia, Eric</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">490</span>
        (<span class="NLM_cas:issue">7421</span>),
    <span class="NLM_cas:pages">552-555</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Polypeptide toxins have played a central part in understanding physiol. and physiopathol. functions of ion channels.  In the field of pain, they led to important advances in basic research and even to clin. applications.  Acid-sensing ion channels (ASICs) are generally considered principal players in the pain pathway, including in humans.  A snake toxin activating peripheral ASICs in nociceptive neurons has been recently shown to evoke pain.  Here we show that a new class of three-finger peptides from another snake, the black mamba, is able to abolish pain through inhibition of ASICs expressed either in central or peripheral neurons.  These peptides, which we call mambalgins, are not toxic in mice but show a potent analgesic effect upon central and peripheral injection that can be as strong as morphine.  This effect is, however, resistant to naloxone, and mambalgins cause much less tolerance than morphine and no respiratory distress.  Pharmacol. inhibition by mambalgins combined with the use of knockdown and knockout animals indicates that blockade of heteromeric channels made of ASIC1a and ASIC2a subunits in central neurons and of ASIC1b-contg. channels in nociceptors is involved in the analgesic effect of mambalgins.  These findings identify new potential therapeutic targets for pain and introduce natural peptides that block them to produce a potent analgesia.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqtE5BL138DF7Vg90H21EOLACvtfcHk0lj5TxLHpOhUfA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsVGjsrnL&md5=35cb1982de34af4d45935b3106f17787</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1038%2Fnature11494&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature11494%26sid%3Dliteratum%253Aachs%26aulast%3DDiochot%26aufirst%3DS.%26aulast%3DBaron%26aufirst%3DA.%26aulast%3DSalinas%26aufirst%3DM.%26aulast%3DDouguet%26aufirst%3DD.%26aulast%3DScarzello%26aufirst%3DS.%26aulast%3DDabert-Gay%26aufirst%3DA.-S.%26aulast%3DDebayle%26aufirst%3DD.%26aulast%3DFriend%26aufirst%3DV.%26aulast%3DAlloui%26aufirst%3DA.%26aulast%3DLazdunski%26aufirst%3DM.%26aulast%3DLingueglia%26aufirst%3DE.%26atitle%3DBlack%2520Mamba%2520Venom%2520Peptides%2520Target%2520Acid-Sensing%2520Ion%2520Channels%2520to%2520Abolish%2520Pain%26jtitle%3DNature%26date%3D2012%26volume%3D490%26spage%3D552%26epage%3D555%26doi%3D10.1038%2Fnature11494" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref59"><div class="reference"><strong class="refLabel"><a href="#ref59" class="rightTabRefNumLink">59</a></strong><div class="NLM_citation" id="rightTab-cit59"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Er, S. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cristofori-Armstrong, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Escoubas, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rash, L. D.</span></span> <span> </span><span class="NLM_article-title">Discovery and Molecular Interaction Studies of a Highly Stable, Tarantula Peptide Modulator of Acid-Sensing Ion Channel 1</span>. <i>Neuropharmacology</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>127</i></span>,  <span class="NLM_fpage">185</span>– <span class="NLM_lpage">195</span>, <span class="refDoi"> DOI: 10.1016/j.neuropharm.2017.03.020</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1016%2Fj.neuropharm.2017.03.020" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=127&publication_year=2017&pages=185-195&author=S.+Y.+Erauthor=B.+Cristofori-Armstrongauthor=P.+Escoubasauthor=L.+D.+Rash&title=Discovery+and+Molecular+Interaction+Studies+of+a+Highly+Stable%2C+Tarantula+Peptide+Modulator+of+Acid-Sensing+Ion+Channel+1&doi=10.1016%2Fj.neuropharm.2017.03.020"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1016%2Fj.neuropharm.2017.03.020&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.neuropharm.2017.03.020%26sid%3Dliteratum%253Aachs%26aulast%3DEr%26aufirst%3DS.%2BY.%26aulast%3DCristofori-Armstrong%26aufirst%3DB.%26aulast%3DEscoubas%26aufirst%3DP.%26aulast%3DRash%26aufirst%3DL.%2BD.%26atitle%3DDiscovery%2520and%2520Molecular%2520Interaction%2520Studies%2520of%2520a%2520Highly%2520Stable%252C%2520Tarantula%2520Peptide%2520Modulator%2520of%2520Acid-Sensing%2520Ion%2520Channel%25201%26jtitle%3DNeuropharmacology%26date%3D2017%26volume%3D127%26spage%3D185%26epage%3D195%26doi%3D10.1016%2Fj.neuropharm.2017.03.020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref60"><div class="reference"><strong class="refLabel"><a href="#ref60" class="rightTabRefNumLink">60</a></strong><div class="NLM_citation" id="rightTab-cit60"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chevalier, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Silva, D.-A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rocklin, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hicks, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vergara, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murapa, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernard, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lam, K.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bahl, C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miyashita, S.-I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goreshnik, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fuller, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koday, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jenkins, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Colvin, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carter, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bohn, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bryan, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fernández-Velasco, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stewart, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, I. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fuller, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baker, D.</span></span> <span> </span><span class="NLM_article-title">Massively Parallel <i>de Novo</i> Protein Design for Targeted Therapeutics</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>550</i></span>,  <span class="NLM_fpage">74</span>– <span class="NLM_lpage">79</span>, <span class="refDoi"> DOI: 10.1038/nature23912</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1038%2Fnature23912" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=28953867" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsFKmsL%252FL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=550&publication_year=2017&pages=74-79&author=A.+Chevalierauthor=D.-A.+Silvaauthor=G.+J.+Rocklinauthor=D.+R.+Hicksauthor=R.+Vergaraauthor=P.+Murapaauthor=S.+M.+Bernardauthor=L.+Zhangauthor=K.-H.+Lamauthor=G.+Yaoauthor=C.+D.+Bahlauthor=S.-I.+Miyashitaauthor=I.+Goreshnikauthor=J.+T.+Fullerauthor=M.+T.+Kodayauthor=C.+M.+Jenkinsauthor=T.+Colvinauthor=L.+Carterauthor=A.+Bohnauthor=C.+M.+Bryanauthor=D.+A.+Fern%C3%A1ndez-Velascoauthor=L.+Stewartauthor=M.+Dongauthor=X.+Huangauthor=R.+Jinauthor=I.+A.+Wilsonauthor=D.+H.+Fullerauthor=D.+Baker&title=Massively+Parallel+de+Novo+Protein+Design+for+Targeted+Therapeutics&doi=10.1038%2Fnature23912"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">Massively parallel de novo protein design for targeted therapeutics</span></div><div class="casAuthors">Chevalier, Aaron; Silva, Daniel-Adriano; Rocklin, Gabriel J.; Hicks, Derrick R.; Vergara, Renan; Murapa, Patience; Bernard, Steffen M.; Zhang, Lu; Lam, Kwok-Ho; Yao, Guorui; Bahl, Christopher D.; Miyashita, Shin-ichiro; Goreshnik, Inna; Fuller, James T.; Koday, Merika T.; Jenkins, Cody M.; Colvin, Tom; Carter, Lauren; Bohn, Alan; Bryan, Cassie M.; Fernandez-Velasco, D. Alejandro; Stewart, Lance; Dong, Min; Huang, Xuhui; Jin, Rongsheng; Wilson, Ian A.; Fuller, Deborah H.; Baker, David</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">550</span>
        (<span class="NLM_cas:issue">7674</span>),
    <span class="NLM_cas:pages">74-79</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">De novo protein design holds promise for creating small stable proteins with shapes customized to bind therapeutic targets.  We describe a massively parallel approach for designing, manufg. and screening mini-protein binders, integrating large-scale computational design, oligonucleotide synthesis, yeast display screening and next-generation sequencing.  We designed and tested 22,660 mini-proteins of 37-43 residues that target influenza hemagglutinin and botulinum neurotoxin B, along with 6286 control sequences to probe contributions to folding and binding, and identified 2618 high-affinity binders.  Comparison of the binding and non-binding design sets, which are two orders of magnitude larger than any previously investigated, enabled the evaluation and improvement of the computational model.  Biophys. characterization of a subset of the binder designs showed that they are extremely stable and, unlike antibodies, do not lose activity after exposure to high temps.  The designs elicit little or no immune response and provide potent prophylactic and therapeutic protection against influenza, even after extensive repeated dosing.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq_7ufRgBkWxLVg90H21EOLACvtfcHk0lj5TxLHpOhUfA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsFKmsL%252FL&md5=c9f52079bed31a8dd4b09f0c84df0b97</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1038%2Fnature23912&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature23912%26sid%3Dliteratum%253Aachs%26aulast%3DChevalier%26aufirst%3DA.%26aulast%3DSilva%26aufirst%3DD.-A.%26aulast%3DRocklin%26aufirst%3DG.%2BJ.%26aulast%3DHicks%26aufirst%3DD.%2BR.%26aulast%3DVergara%26aufirst%3DR.%26aulast%3DMurapa%26aufirst%3DP.%26aulast%3DBernard%26aufirst%3DS.%2BM.%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DLam%26aufirst%3DK.-H.%26aulast%3DYao%26aufirst%3DG.%26aulast%3DBahl%26aufirst%3DC.%2BD.%26aulast%3DMiyashita%26aufirst%3DS.-I.%26aulast%3DGoreshnik%26aufirst%3DI.%26aulast%3DFuller%26aufirst%3DJ.%2BT.%26aulast%3DKoday%26aufirst%3DM.%2BT.%26aulast%3DJenkins%26aufirst%3DC.%2BM.%26aulast%3DColvin%26aufirst%3DT.%26aulast%3DCarter%26aufirst%3DL.%26aulast%3DBohn%26aufirst%3DA.%26aulast%3DBryan%26aufirst%3DC.%2BM.%26aulast%3DFern%25C3%25A1ndez-Velasco%26aufirst%3DD.%2BA.%26aulast%3DStewart%26aufirst%3DL.%26aulast%3DDong%26aufirst%3DM.%26aulast%3DHuang%26aufirst%3DX.%26aulast%3DJin%26aufirst%3DR.%26aulast%3DWilson%26aufirst%3DI.%2BA.%26aulast%3DFuller%26aufirst%3DD.%2BH.%26aulast%3DBaker%26aufirst%3DD.%26atitle%3DMassively%2520Parallel%2520de%2520Novo%2520Protein%2520Design%2520for%2520Targeted%2520Therapeutics%26jtitle%3DNature%26date%3D2017%26volume%3D550%26spage%3D74%26epage%3D79%26doi%3D10.1038%2Fnature23912" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref61"><div class="reference"><strong class="refLabel"><a href="#ref61" class="rightTabRefNumLink">61</a></strong><div class="NLM_citation" id="rightTab-cit61"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Poillot, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bichraoui, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tisseyre, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bahemberae, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andreotti, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sabatier, J.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ronjat, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Waard, M.</span></span> <span> </span><span class="NLM_article-title">Small Efficient Cell-Penetrating Peptides Derived from Scorpion Toxin Maurocalcine</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>287</i></span>,  <span class="NLM_fpage">17331</span>– <span class="NLM_lpage">17342</span>, <span class="refDoi"> DOI: 10.1074/jbc.M112.360628</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1074%2Fjbc.M112.360628" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=287&publication_year=2012&pages=17331-17342&author=C.+Poillotauthor=H.+Bichraouiauthor=C.+Tisseyreauthor=E.+Bahemberaeauthor=N.+Andreottiauthor=J.-M.+Sabatierauthor=M.+Ronjatauthor=M.+De+Waard&title=Small+Efficient+Cell-Penetrating+Peptides+Derived+from+Scorpion+Toxin+Maurocalcine&doi=10.1074%2Fjbc.M112.360628"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M112.360628&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M112.360628%26sid%3Dliteratum%253Aachs%26aulast%3DPoillot%26aufirst%3DC.%26aulast%3DBichraoui%26aufirst%3DH.%26aulast%3DTisseyre%26aufirst%3DC.%26aulast%3DBahemberae%26aufirst%3DE.%26aulast%3DAndreotti%26aufirst%3DN.%26aulast%3DSabatier%26aufirst%3DJ.-M.%26aulast%3DRonjat%26aufirst%3DM.%26aulast%3DDe%2BWaard%26aufirst%3DM.%26atitle%3DSmall%2520Efficient%2520Cell-Penetrating%2520Peptides%2520Derived%2520from%2520Scorpion%2520Toxin%2520Maurocalcine%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2012%26volume%3D287%26spage%3D17331%26epage%3D17342%26doi%3D10.1074%2Fjbc.M112.360628" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref62"><div class="reference"><strong class="refLabel"><a href="#ref62" class="rightTabRefNumLink">62</a></strong><div class="NLM_citation" id="rightTab-cit62"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Goodnow, R. A.,  Jr.</span></span> <span> </span><span class="NLM_article-title">A Brief History of the Development of Combinatorial Chemistry and the Emerging Need for DNA-Encoded Chemistry</span>. In  <i>A Handbook for DNA-Encoded Chemistry</i>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goodnow, R. A.,  Jr.</span></span>, Ed.; <span class="NLM_publisher-name">John Wiley & Sons, Inc.</span>: <span class="NLM_publisher-loc">New York</span>, <span class="NLM_year">2014</span>; pp  <span class="NLM_fpage">19</span>– <span class="NLM_lpage">43</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1002%2F9781118832738.ch2" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2014&pages=19-43&author=R.+A.+Goodnowauthor=R.+A.+Goodnow&title=A+Handbook+for+DNA-Encoded+Chemistry"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1002%2F9781118832738.ch2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1002%252F9781118832738.ch2%26sid%3Dliteratum%253Aachs%26aulast%3DGoodnow%26aufirst%3DR.%2BA.%26atitle%3DA%2520Brief%2520History%2520of%2520the%2520Development%2520of%2520Combinatorial%2520Chemistry%2520and%2520the%2520Emerging%2520Need%2520for%2520DNA-Encoded%2520Chemistry%26btitle%3DA%2520Handbook%2520for%2520DNA-Encoded%2520Chemistry%26aulast%3DGoodnow%26aufirst%3DR.%2BA.%26pub%3DJohn%2520Wiley%2520%2526%2520Sons%252C%2520Inc%26date%3D2014%26spage%3D19%26epage%3D43" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref63"><div class="reference"><strong class="refLabel"><a href="#ref63" class="rightTabRefNumLink">63</a></strong><div class="NLM_citation" id="rightTab-cit63"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brenner, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lerner, R. A.</span></span> <span> </span><span class="NLM_article-title">Encoded Combinatorial Chemistry</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">1992</span>,  <span class="NLM_volume"><i>89</i></span>,  <span class="NLM_fpage">5381</span>– <span class="NLM_lpage">5383</span>, <span class="refDoi"> DOI: 10.1073/pnas.89.12.5381</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1073%2Fpnas.89.12.5381" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1608946" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADyaK38XksVeqsrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=89&publication_year=1992&pages=5381-5383&author=S.+Brennerauthor=R.+A.+Lerner&title=Encoded+Combinatorial+Chemistry&doi=10.1073%2Fpnas.89.12.5381"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit63R"><div class="casContent"><span class="casTitleNuber">63</span><div class="casTitle"><span class="NLM_cas:atitle">Encoded combinatorial chemistry</span></div><div class="casAuthors">Brenner, Sydney; Lerner, Richard A.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">1992</span>),
    <span class="NLM_cas:volume">89</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">5381-3</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    </div><div class="casAbstract">The diversity of chem. synthesis and the power of genetics are linked to provide a powerful, versatile method for drug screening.  A process of alternating parallel combinatorial synthesis is used to encode individual members of a large library of chems. with unique nucleotide sequences.  After the chem. entity is bound to a target, the genetic tag can be amplified by replication and utilized for enrichment of the bound mols. by serial hybridization to a subset of the library.  The library of the tagged chems. is termed an encoded combinatorial chem. library.  The nature of the chem. structure bound to the receptor is decoded by sequencing the nucleotide tag.  For example, with a library of peptides, each peptide was linked to a specifically designed oligodeoxynucleotide (genetic tag).  Each of these oligodeoxynucleotides had a signature (unique) sequence corresponding to the linked peptide and was flanked by sequences recognizable by designed DNA primers of the polymerase chain reaction (PCR).  After a desired member of this library of peptide-oligonucleotides was selected by screening (e.g., selected by binding to a target), this desired member was then identified by PCR amplification of the signature nucleotide sequence.  Detn. of the signature sequence from PCR amplified DNA gave the identity of the desired peptide.  Thus, this method can be used to identify a few members in a large chem. library.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoxFs-UqY_2JbVg90H21EOLACvtfcHk0li7Rg8op1kkow"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK38XksVeqsrs%253D&md5=3f03fc103c039ce7985cd7b25142c7ed</span></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1073%2Fpnas.89.12.5381&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.89.12.5381%26sid%3Dliteratum%253Aachs%26aulast%3DBrenner%26aufirst%3DS.%26aulast%3DLerner%26aufirst%3DR.%2BA.%26atitle%3DEncoded%2520Combinatorial%2520Chemistry%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D1992%26volume%3D89%26spage%3D5381%26epage%3D5383%26doi%3D10.1073%2Fpnas.89.12.5381" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref64"><div class="reference"><strong class="refLabel"><a href="#ref64" class="rightTabRefNumLink">64</a></strong><div class="NLM_citation" id="rightTab-cit64"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Neri, D.</span></span> <span> </span><span class="NLM_article-title">Twenty-Five Years of DNA-Encoded Chemical Libraries</span>. <i>ChemBioChem</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">827</span>– <span class="NLM_lpage">828</span>, <span class="refDoi"> DOI: 10.1002/cbic.201700130</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1002%2Fcbic.201700130" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=28318088" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC2sXksV2lur0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2017&pages=827-828&author=D.+Neri&title=Twenty-Five+Years+of+DNA-Encoded+Chemical+Libraries&doi=10.1002%2Fcbic.201700130"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit64R"><div class="casContent"><span class="casTitleNuber">64</span><div class="casTitle"><span class="NLM_cas:atitle">Twenty-five Years of DNA-Encoded Chemical Libraries</span></div><div class="casAuthors">Neri, Dario</div><div class="citationInfo"><span class="NLM_cas:title">ChemBioChem</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">827-828</span>CODEN:
                <span class="NLM_cas:coden">CBCHFX</span>;
        ISSN:<span class="NLM_cas:issn">1439-4227</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr_MwQL5lTsnbVg90H21EOLACvtfcHk0li7Rg8op1kkow"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXksV2lur0%253D&md5=02b1b7f3ce7e3f49d572c1166b7bc049</span></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=10.1002%2Fcbic.201700130&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcbic.201700130%26sid%3Dliteratum%253Aachs%26aulast%3DNeri%26aufirst%3DD.%26atitle%3DTwenty-Five%2520Years%2520of%2520DNA-Encoded%2520Chemical%2520Libraries%26jtitle%3DChemBioChem%26date%3D2017%26volume%3D18%26spage%3D827%26epage%3D828%26doi%3D10.1002%2Fcbic.201700130" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref65"><div class="reference"><strong class="refLabel"><a href="#ref65" class="rightTabRefNumLink">65</a></strong><div class="NLM_citation" id="rightTab-cit65"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Malone, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paegel, B. M.</span></span> <span> </span><span class="NLM_article-title">What Is a “DNA-Compatible” Reaction?</span>. <i>ACS Comb. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">182</span>– <span class="NLM_lpage">187</span>, <span class="refDoi"> DOI: 10.1021/acscombsci.5b00198</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acscombsci.5b00198" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC28XktFaqu7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2016&pages=182-187&author=M.+L.+Maloneauthor=B.+M.+Paegel&title=What+Is+a+%E2%80%9CDNA-Compatible%E2%80%9D+Reaction%3F&doi=10.1021%2Facscombsci.5b00198"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit65R"><div class="casContent"><span class="casTitleNuber">65</span><div class="casTitle"><span class="NLM_cas:atitle">What is a "DNA-Compatible" Reaction?</span></div><div class="casAuthors">Malone, Marie L.; Paegel, Brian M.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Combinatorial Science</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">182-187</span>CODEN:
                <span class="NLM_cas:coden">ACSCCC</span>;
        ISSN:<span class="NLM_cas:issn">2156-8944</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">DNA-encoded reaction rehearsal was developed as a method to assay the yields of a potential DNA-encoded reaction and the stability of the DNA strand in one pot under the same conditions.  Magnetic DNA-functionalized sensor beads quant. report the % DNA template mols. remaining viable for PCR amplification after exposure to test reaction conditions; resin-bound peptides with the appropriate functionality were used to det. the chem. yields of functionalized products.  Anal. of solid-phase bond forming (e.g., Suzuki-Miyaura cross-coupling, reductive amination) and deprotection reactions (e.g., allyl esters, silyl ethers) guided the definition and optimization of DNA-compatible reaction conditions (>90% yield, >30% viable DNA mols.), most notably in cases that involved known (H+, Pd) and more obscure (Δ, DMF) hazards to DNA integrity.  The data provide an empirical yet mechanistically consistent and predictive framework for designing successful DNA-encoded reaction sequences for combinatorial library synthesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGocOsuE1mW_x7Vg90H21EOLACvtfcHk0li7Rg8op1kkow"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XktFaqu7c%253D&md5=991cbb9fc51fe1de0160b79029fe325a</span></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=10.1021%2Facscombsci.5b00198&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facscombsci.5b00198%26sid%3Dliteratum%253Aachs%26aulast%3DMalone%26aufirst%3DM.%2BL.%26aulast%3DPaegel%26aufirst%3DB.%2BM.%26atitle%3DWhat%2520Is%2520a%2520%25E2%2580%259CDNA-Compatible%25E2%2580%259D%2520Reaction%253F%26jtitle%3DACS%2520Comb.%2520Sci.%26date%3D2016%26volume%3D18%26spage%3D182%26epage%3D187%26doi%3D10.1021%2Facscombsci.5b00198" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref66"><div class="reference"><strong class="refLabel"><a href="#ref66" class="rightTabRefNumLink">66</a></strong><div class="NLM_citation" id="rightTab-cit66"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Goodnow, R. A.,  Jr</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dumelin, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keefe, A. D.</span></span> <span> </span><span class="NLM_article-title">DNA-Encoded Chemistry: Enabling the Deeper Sampling of Chemical Space</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">131</span>– <span class="NLM_lpage">147</span>, <span class="refDoi"> DOI: 10.1038/nrd.2016.213</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1038%2Fnrd.2016.213" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=27932801" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC28XitVKkurjM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2017&pages=131-147&author=R.+A.+Goodnowauthor=C.+E.+Dumelinauthor=A.+D.+Keefe&title=DNA-Encoded+Chemistry%3A+Enabling+the+Deeper+Sampling+of+Chemical+Space&doi=10.1038%2Fnrd.2016.213"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit66R"><div class="casContent"><span class="casTitleNuber">66</span><div class="casTitle"><span class="NLM_cas:atitle">DNA-encoded chemistry: enabling the deeper sampling of chemical space</span></div><div class="casAuthors">Goodnow, Robert A. Jr; Dumelin, Christoph E.; Keefe, Anthony D.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">131-147</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  DNA-encoded chem. library technologies are increasingly being adopted in drug discovery for hit and lead generation.  DNA-encoded chem. enables the exploration of chem. spaces four to five orders of magnitude more deeply than is achievable by traditional high-throughput screening methods.  Operation of this technol. requires developing a range of capabilities including aq. synthetic chem., building block acquisition, oligonucleotide conjugation, large-scale mol. biol. transformations, selection methodologies, PCR, sequencing, sequence data anal. and the anal. of large chem. spaces.  This Review provides an overview of the development and applications of DNA-encoded chem., highlighting the challenges and future directions for the use of this technol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrdEd70MeexerVg90H21EOLACvtfcHk0li7Rg8op1kkow"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitVKkurjM&md5=7ca97b01d75e600378799baf77665c07</span></div><a href="/servlet/linkout?suffix=cit66&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2016.213&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2016.213%26sid%3Dliteratum%253Aachs%26aulast%3DGoodnow%26aufirst%3DR.%2BA.%26aulast%3DDumelin%26aufirst%3DC.%2BE.%26aulast%3DKeefe%26aufirst%3DA.%2BD.%26atitle%3DDNA-Encoded%2520Chemistry%253A%2520Enabling%2520the%2520Deeper%2520Sampling%2520of%2520Chemical%2520Space%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2017%26volume%3D16%26spage%3D131%26epage%3D147%26doi%3D10.1038%2Fnrd.2016.213" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref67"><div class="reference"><strong class="refLabel"><a href="#ref67" class="rightTabRefNumLink">67</a></strong><div class="NLM_citation" id="rightTab-cit67"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Harris, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berger, S. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeong, J. U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nagilla, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bandyopadhyay, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campobasso, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Capriotti, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cox, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dare, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eidam, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Finger, J. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoffman, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kasparcova, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">King, B. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lehr, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lan, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leister, L. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lich, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MacDonald, T. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, N. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ouellette, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pao, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rahman, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reilly, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rendina, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rivera, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schaeffer, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sehon, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singhaus, R. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, H. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Swift, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Totoritis, R. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vossenkämper, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ward, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wisnoski, D. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marquis, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gough, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bertin, J.</span></span> <span> </span><span class="NLM_article-title">Discovery of a First-in-Class Receptor Interacting Protein 1 (RIP1) Kinase Specific Clinical Candidate (GSK2982772) for the Treatment of Inflammatory Diseases</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">1247</span>– <span class="NLM_lpage">1261</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b01751</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01751" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC2sXitVeksLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=1247-1261&author=P.+A.+Harrisauthor=S.+B.+Bergerauthor=J.+U.+Jeongauthor=R.+Nagillaauthor=D.+Bandyopadhyayauthor=N.+Campobassoauthor=C.+A.+Capriottiauthor=J.+A.+Coxauthor=L.+Dareauthor=X.+Dongauthor=P.+M.+Eidamauthor=J.+N.+Fingerauthor=S.+J.+Hoffmanauthor=J.+Kangauthor=V.+Kasparcovaauthor=B.+W.+Kingauthor=R.+Lehrauthor=Y.+Lanauthor=L.+K.+Leisterauthor=J.+D.+Lichauthor=T.+T.+MacDonaldauthor=N.+A.+Millerauthor=M.+T.+Ouelletteauthor=C.+S.+Paoauthor=A.+Rahmanauthor=M.+A.+Reillyauthor=A.+R.+Rendinaauthor=E.+J.+Riveraauthor=M.+C.+Schaefferauthor=C.+A.+Sehonauthor=R.+R.+Singhausauthor=H.+H.+Sunauthor=B.+A.+Swiftauthor=R.+D.+Totoritisauthor=A.+Vossenk%C3%A4mperauthor=P.+Wardauthor=D.+D.+Wisnoskiauthor=D.+Zhangauthor=R.+W.+Marquisauthor=P.+J.+Goughauthor=J.+Bertin&title=Discovery+of+a+First-in-Class+Receptor+Interacting+Protein+1+%28RIP1%29+Kinase+Specific+Clinical+Candidate+%28GSK2982772%29+for+the+Treatment+of+Inflammatory+Diseases&doi=10.1021%2Facs.jmedchem.6b01751"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit67R"><div class="casContent"><span class="casTitleNuber">67</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a First-in-Class Receptor Interacting Protein 1 (RIP1) Kinase Specific Clinical Candidate (GSK2982772) for the Treatment of Inflammatory Diseases</span></div><div class="casAuthors">Harris, Philip A.; Berger, Scott B.; Jeong, Jae U.; Nagilla, Rakesh; Bandyopadhyay, Deepak; Campobasso, Nino; Capriotti, Carol A.; Cox, Julie A.; Dare, Lauren; Dong, Xiaoyang; Eidam, Patrick M.; Finger, Joshua N.; Hoffman, Sandra J.; Kang, James; Kasparcova, Viera; King, Bryan W.; Lehr, Ruth; Lan, Yunfeng; Leister, Lara K.; Lich, John D.; MacDonald, Thomas T.; Miller, Nathan A.; Ouellette, Michael T.; Pao, Christina S.; Rahman, Attiq; Reilly, Michael A.; Rendina, Alan R.; Rivera, Elizabeth J.; Schaeffer, Michelle C.; Sehon, Clark A.; Singhaus, Robert R.; Sun, Helen H.; Swift, Barbara A.; Totoritis, Rachel D.; Vossenkamper, Anna; Ward, Paris; Wisnoski, David D.; Zhang, Daohua; Marquis, Robert W.; Gough, Peter J.; Bertin, John</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1247-1261</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">RIP1 regulates necroptosis and inflammation and may play an important role in contributing to a variety of human pathologies, including immune-mediated inflammatory diseases.  Small-mol. inhibitors of RIP1 kinase that are suitable for advancement into the clinic have yet to be described.  Herein, the authors report the lead optimization of a benzoxazepinone hit from a DNA-encoded library and the discovery and profile of clin. candidate GSK2982772 (compd. I), currently in phase 2a clin. studies for psoriasis, rheumatoid arthritis, and ulcerative colitis.  Compd. I potently binds to RIP1 with exquisite kinase specificity and has excellent activity in blocking many TNF-dependent cellular responses.  Highlighting its potential as a novel anti-inflammatory agent, the inhibitor was also able to reduce spontaneous prodn. of cytokines from human ulcerative colitis explants.  The highly favorable physicochem. and ADMET properties of I, combined with high potency, led to a predicted low oral dose in humans.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqZGRDTQv-vs7Vg90H21EOLACvtfcHk0liY1DphFQDrCQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXitVeksLk%253D&md5=ee0c3c60bb1351dff696490ac9edb1cc</span></div><a href="/servlet/linkout?suffix=cit67&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01751&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01751%26sid%3Dliteratum%253Aachs%26aulast%3DHarris%26aufirst%3DP.%2BA.%26aulast%3DBerger%26aufirst%3DS.%2BB.%26aulast%3DJeong%26aufirst%3DJ.%2BU.%26aulast%3DNagilla%26aufirst%3DR.%26aulast%3DBandyopadhyay%26aufirst%3DD.%26aulast%3DCampobasso%26aufirst%3DN.%26aulast%3DCapriotti%26aufirst%3DC.%2BA.%26aulast%3DCox%26aufirst%3DJ.%2BA.%26aulast%3DDare%26aufirst%3DL.%26aulast%3DDong%26aufirst%3DX.%26aulast%3DEidam%26aufirst%3DP.%2BM.%26aulast%3DFinger%26aufirst%3DJ.%2BN.%26aulast%3DHoffman%26aufirst%3DS.%2BJ.%26aulast%3DKang%26aufirst%3DJ.%26aulast%3DKasparcova%26aufirst%3DV.%26aulast%3DKing%26aufirst%3DB.%2BW.%26aulast%3DLehr%26aufirst%3DR.%26aulast%3DLan%26aufirst%3DY.%26aulast%3DLeister%26aufirst%3DL.%2BK.%26aulast%3DLich%26aufirst%3DJ.%2BD.%26aulast%3DMacDonald%26aufirst%3DT.%2BT.%26aulast%3DMiller%26aufirst%3DN.%2BA.%26aulast%3DOuellette%26aufirst%3DM.%2BT.%26aulast%3DPao%26aufirst%3DC.%2BS.%26aulast%3DRahman%26aufirst%3DA.%26aulast%3DReilly%26aufirst%3DM.%2BA.%26aulast%3DRendina%26aufirst%3DA.%2BR.%26aulast%3DRivera%26aufirst%3DE.%2BJ.%26aulast%3DSchaeffer%26aufirst%3DM.%2BC.%26aulast%3DSehon%26aufirst%3DC.%2BA.%26aulast%3DSinghaus%26aufirst%3DR.%2BR.%26aulast%3DSun%26aufirst%3DH.%2BH.%26aulast%3DSwift%26aufirst%3DB.%2BA.%26aulast%3DTotoritis%26aufirst%3DR.%2BD.%26aulast%3DVossenk%25C3%25A4mper%26aufirst%3DA.%26aulast%3DWard%26aufirst%3DP.%26aulast%3DWisnoski%26aufirst%3DD.%2BD.%26aulast%3DZhang%26aufirst%3DD.%26aulast%3DMarquis%26aufirst%3DR.%2BW.%26aulast%3DGough%26aufirst%3DP.%2BJ.%26aulast%3DBertin%26aufirst%3DJ.%26atitle%3DDiscovery%2520of%2520a%2520First-in-Class%2520Receptor%2520Interacting%2520Protein%25201%2520%2528RIP1%2529%2520Kinase%2520Specific%2520Clinical%2520Candidate%2520%2528GSK2982772%2529%2520for%2520the%2520Treatment%2520of%2520Inflammatory%2520Diseases%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D1247%26epage%3D1261%26doi%3D10.1021%2Facs.jmedchem.6b01751" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref68"><div class="reference"><strong class="refLabel"><a href="#ref68" class="rightTabRefNumLink">68</a></strong><div class="NLM_citation" id="rightTab-cit68"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Škopić, M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salamon, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bugain, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jung, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gohla, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doetsch, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">dos Santos, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhat, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wagner, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brunschweiger, A.</span></span> <span> </span><span class="NLM_article-title">Acid- and Au(I)-Mediated Synthesis of Hexathymidine-DNA-Heterocycle Chimeras, an Efficient Entry to DNA-Encoded Libraries Inspired by Drug Structures</span>. <i>Chem. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">3356</span>– <span class="NLM_lpage">3361</span>, <span class="refDoi"> DOI: 10.1039/C7SC00455A</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1039%2FC7SC00455A" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=28507705" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC2sXjsVKmtrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=3356-3361&author=M.+K.+%C5%A0kopi%C4%87author=H.+Salamonauthor=O.+Bugainauthor=K.+Jungauthor=A.+Gohlaauthor=L.+J.+Doetschauthor=D.+dos+Santosauthor=A.+Bhatauthor=B.+Wagnerauthor=A.+Brunschweiger&title=Acid-+and+Au%28I%29-Mediated+Synthesis+of+Hexathymidine-DNA-Heterocycle+Chimeras%2C+an+Efficient+Entry+to+DNA-Encoded+Libraries+Inspired+by+Drug+Structures&doi=10.1039%2FC7SC00455A"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit68R"><div class="casContent"><span class="casTitleNuber">68</span><div class="casTitle"><span class="NLM_cas:atitle">Acid- and Au(I)-mediated synthesis of hexathymidine-DNA-heterocycle chimeras, an efficient entry to DNA-encoded libraries inspired by drug structures</span></div><div class="casAuthors">Skopic, Mateja Klika; Salamon, Hazem; Bugain, Olivia; Jung, Kathrin; Gohla, Anne; Doetsch, Lara J.; Santos, Denise dos; Bhat, Avinash; Wagner, Bernd; Brunschweiger, Andreas</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Science</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">3356-3361</span>CODEN:
                <span class="NLM_cas:coden">CSHCCN</span>;
        ISSN:<span class="NLM_cas:issn">2041-6520</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Libraries of DNA-tagged compds. are a validated screening technol. for drug discovery.  They are synthesized through combinatorial iterations of alternated coding and preparative synthesis steps.  Thus, large chem. space can be accessed for target-based screening.  However, the need to preserve the functionality of the DNA tag severely restricts the choice of chem. methods for library synthesis.  Acidic organocatalysts, transition metals, and oxidants furnish diverse drug-like structures from simple starting materials, but cause loss of genetic information by depurination.  A hexathymidine oligonucleotide, called "hexT" allows the chemist utilizing these classes of catalysts to access a potentially broad variety of structures in the initial step of library synthesis.  We exploited its catalyst tolerance to efficiently synthesize diverse substituted β-carbolines, pyrazolines, and pyrazoles from readily available starting materials as hexT conjugates by acid- and Au(I)-catalysis, resp.  The hexT conjugates were ligated to coding DNA sequences yielding encoded screening libraries inspired by drug structures.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGouSdRpFmyaN7Vg90H21EOLACvtfcHk0liY1DphFQDrCQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXjsVKmtrg%253D&md5=cc6a646c1e6ecf3cdfae7d36cca63440</span></div><a href="/servlet/linkout?suffix=cit68&amp;dbid=16384&amp;doi=10.1039%2FC7SC00455A&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC7SC00455A%26sid%3Dliteratum%253Aachs%26aulast%3D%25C5%25A0kopi%25C4%2587%26aufirst%3DM.%2BK.%26aulast%3DSalamon%26aufirst%3DH.%26aulast%3DBugain%26aufirst%3DO.%26aulast%3DJung%26aufirst%3DK.%26aulast%3DGohla%26aufirst%3DA.%26aulast%3DDoetsch%26aufirst%3DL.%2BJ.%26aulast%3Ddos%2BSantos%26aufirst%3DD.%26aulast%3DBhat%26aufirst%3DA.%26aulast%3DWagner%26aufirst%3DB.%26aulast%3DBrunschweiger%26aufirst%3DA.%26atitle%3DAcid-%2520and%2520Au%2528I%2529-Mediated%2520Synthesis%2520of%2520Hexathymidine-DNA-Heterocycle%2520Chimeras%252C%2520an%2520Efficient%2520Entry%2520to%2520DNA-Encoded%2520Libraries%2520Inspired%2520by%2520Drug%2520Structures%26jtitle%3DChem.%2520Sci.%26date%3D2017%26volume%3D8%26spage%3D3356%26epage%3D3361%26doi%3D10.1039%2FC7SC00455A" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref69"><div class="reference"><strong class="refLabel"><a href="#ref69" class="rightTabRefNumLink">69</a></strong><div class="NLM_citation" id="rightTab-cit69"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Graybill, T. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Platchek, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bodmer, V. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wisnoski, D. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coppo, F. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tamburino, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scavello, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Franklin, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mataruse, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bedard, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chai, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Summerfield, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Centrella, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Messer, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pope, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Israel, D. I.</span></span> <span> </span><span class="NLM_article-title">Cell-Based Selection Expands the Utility of DNA-Encoded Small-Molecule Library Technology to Cell Surface Drug Targets: Identification of Novel Antagonists of the NK3 Tachykinin Receptor</span>. <i>ACS Comb. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">722</span>– <span class="NLM_lpage">731</span>, <span class="refDoi"> DOI: 10.1021/acscombsci.5b00124</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acscombsci.5b00124" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvVaht7jM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2015&pages=722-731&author=Z.+Wuauthor=T.+L.+Graybillauthor=X.+Zengauthor=M.+Platchekauthor=J.+Zhangauthor=V.+Q.+Bodmerauthor=D.+D.+Wisnoskiauthor=J.+Dengauthor=F.+T.+Coppoauthor=G.+Yaoauthor=A.+Tamburinoauthor=G.+Scavelloauthor=G.+J.+Franklinauthor=S.+Mataruseauthor=K.+L.+Bedardauthor=Y.+Dingauthor=J.+Chaiauthor=J.+Summerfieldauthor=P.+A.+Centrellaauthor=J.+A.+Messerauthor=A.+J.+Popeauthor=D.+I.+Israel&title=Cell-Based+Selection+Expands+the+Utility+of+DNA-Encoded+Small-Molecule+Library+Technology+to+Cell+Surface+Drug+Targets%3A+Identification+of+Novel+Antagonists+of+the+NK3+Tachykinin+Receptor&doi=10.1021%2Facscombsci.5b00124"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit69R"><div class="casContent"><span class="casTitleNuber">69</span><div class="casTitle"><span class="NLM_cas:atitle">Cell-Based Selection Expands the Utility of DNA-Encoded Small-Molecule Library Technology to Cell Surface Drug Targets: Identification of Novel Antagonists of the NK3 Tachykinin Receptor</span></div><div class="casAuthors">Wu, Zining; Graybill, Todd L.; Zeng, Xin; Platchek, Michael; Zhang, Jean; Bodmer, Vera Q.; Wisnoski, David D.; Deng, Jianghe; Coppo, Frank T.; Yao, Gang; Tamburino, Alex; Scavello, Genaro; Franklin, G. Joseph; Mataruse, Sibongile; Bedard, Katie L.; Ding, Yun; Chai, Jing; Summerfield, Jennifer; Centrella, Paolo A.; Messer, Jeffrey A.; Pope, Andrew J.; Israel, David I.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Combinatorial Science</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">722-731</span>CODEN:
                <span class="NLM_cas:coden">ACSCCC</span>;
        ISSN:<span class="NLM_cas:issn">2156-8944</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">DNA-encoded small-mol. library technol. has recently emerged as a new paradigm for identifying ligands against drug targets.  To date, this technol. has been used with sol. protein targets that are produced and used in a purified state.  Here, we describe a cell-based method for identifying small-mol. ligands from DNA-encoded libraries against integral membrane protein targets.  We use this method to identify novel, potent, and specific inhibitors of NK3, a member of the tachykinin family of G-protein coupled receptors (GPCRs).  The method is simple and broadly applicable to other GPCRs and integral membrane proteins.  We have extended the application of DNA-encoded library technol. to membrane-assocd. targets and demonstrate the feasibility of selecting DNA-tagged, small-mol. ligands from complex combinatorial libraries against targets in a heterogeneous milieu, such as the surface of a cell.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpMG6EaclP47bVg90H21EOLACvtfcHk0lgdThH3WsfhQA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvVaht7jM&md5=eb7ebd0e31863dc8d756c14e9248791d</span></div><a href="/servlet/linkout?suffix=cit69&amp;dbid=16384&amp;doi=10.1021%2Facscombsci.5b00124&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facscombsci.5b00124%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DZ.%26aulast%3DGraybill%26aufirst%3DT.%2BL.%26aulast%3DZeng%26aufirst%3DX.%26aulast%3DPlatchek%26aufirst%3DM.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DBodmer%26aufirst%3DV.%2BQ.%26aulast%3DWisnoski%26aufirst%3DD.%2BD.%26aulast%3DDeng%26aufirst%3DJ.%26aulast%3DCoppo%26aufirst%3DF.%2BT.%26aulast%3DYao%26aufirst%3DG.%26aulast%3DTamburino%26aufirst%3DA.%26aulast%3DScavello%26aufirst%3DG.%26aulast%3DFranklin%26aufirst%3DG.%2BJ.%26aulast%3DMataruse%26aufirst%3DS.%26aulast%3DBedard%26aufirst%3DK.%2BL.%26aulast%3DDing%26aufirst%3DY.%26aulast%3DChai%26aufirst%3DJ.%26aulast%3DSummerfield%26aufirst%3DJ.%26aulast%3DCentrella%26aufirst%3DP.%2BA.%26aulast%3DMesser%26aufirst%3DJ.%2BA.%26aulast%3DPope%26aufirst%3DA.%2BJ.%26aulast%3DIsrael%26aufirst%3DD.%2BI.%26atitle%3DCell-Based%2520Selection%2520Expands%2520the%2520Utility%2520of%2520DNA-Encoded%2520Small-Molecule%2520Library%2520Technology%2520to%2520Cell%2520Surface%2520Drug%2520Targets%253A%2520Identification%2520of%2520Novel%2520Antagonists%2520of%2520the%2520NK3%2520Tachykinin%2520Receptor%26jtitle%3DACS%2520Comb.%2520Sci.%26date%3D2015%26volume%3D17%26spage%3D722%26epage%3D731%26doi%3D10.1021%2Facscombsci.5b00124" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref70"><div class="reference"><strong class="refLabel"><a href="#ref70" class="rightTabRefNumLink">70</a></strong><div class="NLM_citation" id="rightTab-cit70"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Henrich, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hein, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dötsch, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernhard, F.</span></span> <span> </span><span class="NLM_article-title">Membrane Protein Production in Escherichia Coli Cell-Free Lysates</span>. <i>FEBS Lett.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>589</i></span>,  <span class="NLM_fpage">1713</span>– <span class="NLM_lpage">1722</span>, <span class="refDoi"> DOI: 10.1016/j.febslet.2015.04.045</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1016%2Fj.febslet.2015.04.045" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=589&publication_year=2015&pages=1713-1722&author=E.+Henrichauthor=C.+Heinauthor=V.+D%C3%B6tschauthor=F.+Bernhard&title=Membrane+Protein+Production+in+Escherichia+Coli+Cell-Free+Lysates&doi=10.1016%2Fj.febslet.2015.04.045"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit70&amp;dbid=16384&amp;doi=10.1016%2Fj.febslet.2015.04.045&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.febslet.2015.04.045%26sid%3Dliteratum%253Aachs%26aulast%3DHenrich%26aufirst%3DE.%26aulast%3DHein%26aufirst%3DC.%26aulast%3DD%25C3%25B6tsch%26aufirst%3DV.%26aulast%3DBernhard%26aufirst%3DF.%26atitle%3DMembrane%2520Protein%2520Production%2520in%2520Escherichia%2520Coli%2520Cell-Free%2520Lysates%26jtitle%3DFEBS%2520Lett.%26date%3D2015%26volume%3D589%26spage%3D1713%26epage%3D1722%26doi%3D10.1016%2Fj.febslet.2015.04.045" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref71"><div class="reference"><strong class="refLabel"><a href="#ref71" class="rightTabRefNumLink">71</a></strong><div class="NLM_citation" id="rightTab-cit71"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shinoda, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shinya, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ito, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ishizuka-Katsura, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohsawa, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Terada, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hirata, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kawano, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamamoto, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomita, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ishibashi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hirabayashi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kimura-Someya, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shirouzu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yokoyama, S.</span></span> <span> </span><span class="NLM_article-title">Cell-Free Methods to Produce Structurally Intact Mammalian Membrane Proteins</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">30442</span>, <span class="refDoi"> DOI: 10.1038/srep30442</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1038%2Fsrep30442" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=27465719" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC2sXksFeltr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2016&pages=30442&author=T.+Shinodaauthor=N.+Shinyaauthor=K.+Itoauthor=Y.+Ishizuka-Katsuraauthor=N.+Ohsawaauthor=T.+Teradaauthor=K.+Hirataauthor=Y.+Kawanoauthor=M.+Yamamotoauthor=T.+Tomitaauthor=Y.+Ishibashiauthor=Y.+Hirabayashiauthor=T.+Kimura-Someyaauthor=M.+Shirouzuauthor=S.+Yokoyama&title=Cell-Free+Methods+to+Produce+Structurally+Intact+Mammalian+Membrane+Proteins&doi=10.1038%2Fsrep30442"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit71R"><div class="casContent"><span class="casTitleNuber">71</span><div class="casTitle"><span class="NLM_cas:atitle">Cell-free methods to produce structurally intact mammalian membrane proteins</span></div><div class="casAuthors">Shinoda, Takehiro; Shinya, Naoko; Ito, Kaori; Ishizuka-Katsura, Yoshiko; Ohsawa, Noboru; Terada, Takaho; Hirata, Kunio; Kawano, Yoshiaki; Yamamoto, Masaki; Tomita, Taisuke; Ishibashi, Yohei; Hirabayashi, Yoshio; Kimura-Someya, Tomomi; Shirouzu, Mikako; Yokoyama, Shigeyuki</div><div class="citationInfo"><span class="NLM_cas:title">Scientific Reports</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">30442</span>CODEN:
                <span class="NLM_cas:coden">SRCEC3</span>;
        ISSN:<span class="NLM_cas:issn">2045-2322</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The crystal structures of four membrane proteins, from bacteria or a unicellular alga, have been solved with samples produced by cell-free protein synthesis.  In this study, for mammalian membrane protein prodn., we established the pptg. and sol. membrane fragment methods: membrane proteins are synthesized with the Escherichia coli cell-free system in the presence of large and small membrane fragments, resp., and are simultaneously integrated into the lipid environments.  We applied the pptg. membrane fragment method to produce various mammalian membrane proteins, including human claudins, glucosylceramide synthase, and the γ-secretase subunits.  These proteins were produced at levels of about 0.1-1.0 mg per mL cell-free reaction under the initial conditions, and were obtained as ppts. by ultracentrifugation.  Larger amts. of membrane proteins were produced by the sol. membrane fragment method, collected in the ultracentrifugation supernatants, and purified directly by column chromatog.  For several proteins, the conditions of the membrane fragment methods were further optimized, such as by the addn. of specific lipids/detergents.  The functional and structural integrities of the purified proteins were confirmed by analyses of their ligand binding activities, size-exclusion chromatog. profiles, and/or thermal stabilities.  We successfully obtained high-quality crystals of the complex of human claudin-4 with an enterotoxin.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGptV1wwXNXR-LVg90H21EOLACvtfcHk0lgdThH3WsfhQA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXksFeltr8%253D&md5=4324f644d00ee011808931a4f410f6bf</span></div><a href="/servlet/linkout?suffix=cit71&amp;dbid=16384&amp;doi=10.1038%2Fsrep30442&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsrep30442%26sid%3Dliteratum%253Aachs%26aulast%3DShinoda%26aufirst%3DT.%26aulast%3DShinya%26aufirst%3DN.%26aulast%3DIto%26aufirst%3DK.%26aulast%3DIshizuka-Katsura%26aufirst%3DY.%26aulast%3DOhsawa%26aufirst%3DN.%26aulast%3DTerada%26aufirst%3DT.%26aulast%3DHirata%26aufirst%3DK.%26aulast%3DKawano%26aufirst%3DY.%26aulast%3DYamamoto%26aufirst%3DM.%26aulast%3DTomita%26aufirst%3DT.%26aulast%3DIshibashi%26aufirst%3DY.%26aulast%3DHirabayashi%26aufirst%3DY.%26aulast%3DKimura-Someya%26aufirst%3DT.%26aulast%3DShirouzu%26aufirst%3DM.%26aulast%3DYokoyama%26aufirst%3DS.%26atitle%3DCell-Free%2520Methods%2520to%2520Produce%2520Structurally%2520Intact%2520Mammalian%2520Membrane%2520Proteins%26jtitle%3DSci.%2520Rep.%26date%3D2016%26volume%3D6%26spage%3D30442%26doi%3D10.1038%2Fsrep30442" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref72"><div class="reference"><strong class="refLabel"><a href="#ref72" class="rightTabRefNumLink">72</a></strong><div class="NLM_citation" id="rightTab-cit72"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Desuzinges
Mandon, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Agez, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pellegrin, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Igonet, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jawhari, A.</span></span> <span> </span><span class="NLM_article-title">Novel Systematic Detergent Screening Method for Membrane Proteins Solubilization</span>. <i>Anal. Biochem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>517</i></span>,  <span class="NLM_fpage">40</span>– <span class="NLM_lpage">49</span>, <span class="refDoi"> DOI: 10.1016/j.ab.2016.11.008</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1016%2Fj.ab.2016.11.008" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=27847172" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvFKjs7vF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=517&publication_year=2017&pages=40-49&author=E.+Desuzinges%0AMandonauthor=M.+Agezauthor=R.+Pellegrinauthor=S.+Igonetauthor=A.+Jawhari&title=Novel+Systematic+Detergent+Screening+Method+for+Membrane+Proteins+Solubilization&doi=10.1016%2Fj.ab.2016.11.008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit72R"><div class="casContent"><span class="casTitleNuber">72</span><div class="casTitle"><span class="NLM_cas:atitle">Novel systematic detergent screening method for membrane proteins solubilization</span></div><div class="casAuthors">Desuzinges Mandon, Elodie; Agez, Morgane; Pellegrin, Rebecca; Igonet, Sebastien; Jawhari, Anass</div><div class="citationInfo"><span class="NLM_cas:title">Analytical Biochemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">517</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">40-49</span>CODEN:
                <span class="NLM_cas:coden">ANBCA2</span>;
        ISSN:<span class="NLM_cas:issn">0003-2697</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Membrane proteins play crucial role in many cellular processes including cell adhesion, cell-cell communication, signal transduction and transport.  To better understand the mol. basis of such central biol. machines and to specifically study their biol. and medical role, it is necessary to ext. them from their membrane environment.  To do so, it is challenging to find the best solubilization condition.  Here the authors describe, a systematic screening method called BMSS (Biotinylated Membranes Solubilization and Sepn.) that allow screening 96 conditions at once.  Streptavidin magnetic beads were used to sep. solubilized proteins from remaining biotinylated membranes after solubilization.  Relative quantification of dot blots help to select the best conditions to be confirmed by classical ultra-centrifugation and western blot.  Classical detergents with different phys.-chem. characteristics, novel calixarene based detergents and combination of both, were used for solubilization trials to obtain broad spectrum of conditions.  Here, the authors show the application of BMSS to discover solubilization conditions of a GPCR target (MP-A) and a transporter (MP-B).  The selected conditions allowed the solubilization and purifn. of non-aggregated and homogeneous native membrane proteins A and B.  Taken together, BMSS represent a rapid, reproducible and high throughput assessment of solubilization toward biochem./functional characterization, biophys. screening and structural studies of membrane proteins of high biol. and medical relevance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoaByHSw5OH8rVg90H21EOLACvtfcHk0lgdThH3WsfhQA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvFKjs7vF&md5=3e2118032081cc3e637147b009d2a7f6</span></div><a href="/servlet/linkout?suffix=cit72&amp;dbid=16384&amp;doi=10.1016%2Fj.ab.2016.11.008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ab.2016.11.008%26sid%3Dliteratum%253Aachs%26aulast%3DDesuzinges%2BMandon%26aufirst%3DE.%26aulast%3DAgez%26aufirst%3DM.%26aulast%3DPellegrin%26aufirst%3DR.%26aulast%3DIgonet%26aufirst%3DS.%26aulast%3DJawhari%26aufirst%3DA.%26atitle%3DNovel%2520Systematic%2520Detergent%2520Screening%2520Method%2520for%2520Membrane%2520Proteins%2520Solubilization%26jtitle%3DAnal.%2520Biochem.%26date%3D2017%26volume%3D517%26spage%3D40%26epage%3D49%26doi%3D10.1016%2Fj.ab.2016.11.008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref73"><div class="reference"><strong class="refLabel"><a href="#ref73" class="rightTabRefNumLink">73</a></strong><div class="NLM_citation" id="rightTab-cit73"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Arico-Muendel, C. C.</span></span> <span> </span><span class="NLM_article-title">From Haystack to Needle: Finding Value with DNA Encoded Library Technology at GSK</span>. <i>MedChemComm</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">1898</span>– <span class="NLM_lpage">1909</span>, <span class="refDoi"> DOI: 10.1039/C6MD00341A</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1039%2FC6MD00341A" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC28Xht1ehu7jL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=1898-1909&author=C.+C.+Arico-Muendel&title=From+Haystack+to+Needle%3A+Finding+Value+with+DNA+Encoded+Library+Technology+at+GSK&doi=10.1039%2FC6MD00341A"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit73R"><div class="casContent"><span class="casTitleNuber">73</span><div class="casTitle"><span class="NLM_cas:atitle">From haystack to needle: finding value with DNA encoded library technology at GSK</span></div><div class="casAuthors">Arico-Muendel, Christopher C.</div><div class="citationInfo"><span class="NLM_cas:title">MedChemComm</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1898-1909</span>CODEN:
                <span class="NLM_cas:coden">MCCEAY</span>;
        ISSN:<span class="NLM_cas:issn">2040-2503</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">DNA encoded library technol. (ELT) provides access to broad chem. diversity through affinity selection.  As an early adopter, GlaxoSmithKline enabled the development of ELT from proof of concept to full fledged contributor to the small mol. therapeutic pipeline.  Unique benefits of the GSK incubation include uptake of large nos. of building blocks and privileged scaffolds, guidance on library design, and opportunities to add value beyond lead discovery.  Sol. epoxide hydrolase and RIP1 kinase represent two case studies of ELT hits that have advanced into clin. studies.  ELT also shows promise as a strategy to forecast target tractability.  As an emerging technol., it benefits from incubation at the interface of industry and academia.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrGhx5inxOj_7Vg90H21EOLACvtfcHk0lj4auOC9u9jNg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xht1ehu7jL&md5=1862af051a956ff1893143e107cff4f4</span></div><a href="/servlet/linkout?suffix=cit73&amp;dbid=16384&amp;doi=10.1039%2FC6MD00341A&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC6MD00341A%26sid%3Dliteratum%253Aachs%26aulast%3DArico-Muendel%26aufirst%3DC.%2BC.%26atitle%3DFrom%2520Haystack%2520to%2520Needle%253A%2520Finding%2520Value%2520with%2520DNA%2520Encoded%2520Library%2520Technology%2520at%2520GSK%26jtitle%3DMedChemComm%26date%3D2016%26volume%3D7%26spage%3D1898%26epage%3D1909%26doi%3D10.1039%2FC6MD00341A" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref74"><div class="reference"><strong class="refLabel"><a href="#ref74" class="rightTabRefNumLink">74</a></strong><div class="NLM_citation" id="rightTab-cit74"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ahn, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kahsai, A. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pani, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Q.-T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wall, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Strachan, R. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Staus, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wingler, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, L. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sinnaeve, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cho, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, T. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hansen, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burnett, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lamerdin, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bassoni, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gavino, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Husemoen, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olsen, E. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Franch, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Costanzi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lefkowitz, R. J.</span></span> <span> </span><span class="NLM_article-title">Allosteric “beta-Blocker” Isolated from a DNA-Encoded Small Molecule Library</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>114</i></span>,  <span class="NLM_fpage">1708</span>– <span class="NLM_lpage">1713</span>, <span class="refDoi"> DOI: 10.1073/pnas.1620645114</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1073%2Fpnas.1620645114" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=28130548" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhs1Kjsbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=114&publication_year=2017&pages=1708-1713&author=S.+Ahnauthor=A.+W.+Kahsaiauthor=B.+Paniauthor=Q.-T.+Wangauthor=S.+Zhaoauthor=A.+L.+Wallauthor=R.+T.+Strachanauthor=D.+P.+Stausauthor=L.+M.+Winglerauthor=L.+D.+Sunauthor=J.+Sinnaeveauthor=M.+Choiauthor=T.+Choauthor=T.+T.+Xuauthor=G.+M.+Hansenauthor=M.+B.+Burnettauthor=J.+E.+Lamerdinauthor=D.+L.+Bassoniauthor=B.+J.+Gavinoauthor=G.+Husemoenauthor=E.+K.+Olsenauthor=T.+Franchauthor=S.+Costanziauthor=X.+Chenauthor=R.+J.+Lefkowitz&title=Allosteric+%E2%80%9Cbeta-Blocker%E2%80%9D+Isolated+from+a+DNA-Encoded+Small+Molecule+Library&doi=10.1073%2Fpnas.1620645114"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit74R"><div class="casContent"><span class="casTitleNuber">74</span><div class="casTitle"><span class="NLM_cas:atitle">Allosteric "beta-blocker" isolated from a DNA-encoded small molecule library</span></div><div class="casAuthors">Ahn, Seungkirl; Kahsai, Alem W.; Pani, Biswaranjan; Wang, Qin-Ting; Zhao, Shuai; Wall, Alissa L.; Strachan, Ryan T.; Staus, Dean P.; Wingler, Laura M.; Sun, Lillian D.; Sinnaeve, Justine; Choi, Minjung; Cho, Ted; Xu, Thomas T.; Hansen, Gwenn M.; Burnett, Michael B.; Lamerdin, Jane E.; Bassoni, Daniel L.; Gavino, Bryant J.; Husemoen, Gitte; Olsen, Eva K.; Franch, Thomas; Costanzi, Stefano; Chen, Xin; Lefkowitz, Robert J.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">114</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1708-1713</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The β2-adrenergic receptor (β2AR) has been a model system for understanding regulatory mechanisms of G-protein-coupled receptor (GPCR) actions and plays a significant role in cardiovascular and pulmonary diseases.  Because all known β-adrenergic receptor drugs target the orthosteric binding site of the receptor, we set out to isolate allosteric ligands for this receptor by panning DNA-encoded small-mol. libraries comprising 190 million distinct compds. against purified human β2AR.  Here, we report the discovery of a small-mol. neg. allosteric modulator (antagonist), compd. 15 [([4-((2S)-3-(((S)-3-(3-bromophenyl)-1-(methylamino)-1-oxopropan-2-yl)amino)-2-(2-cyclohexyl-2-phenylacetamido)-3-oxopropyl)benzamide], exhibiting a unique chemotype and low micromolar affinity for the β2AR.  Binding of 15 to the receptor cooperatively enhances orthosteric inverse agonist binding while neg. modulating binding of orthosteric agonists.  Studies with a specific antibody that binds to an intracellular region of the β2AR suggest that 15 binds in proximity to the G-protein binding site on the cytosolic surface of the β2AR.  In cell-signaling studies, 15 inhibits cAMP prodn. through the β2AR, but not that mediated by other Gs-coupled receptors.  Compd. 15 also similarly inhibits β-arrestin recruitment to the activated β2AR.  This study presents an allosteric small-mol. ligand for the β2AR and introduces a broadly applicable method for screening DNA-encoded small-mol. libraries against purified GPCR targets.  Importantly, such an approach could facilitate the discovery of GPCR drugs with tailored allosteric effects.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpE5o7Ib4LtSrVg90H21EOLACvtfcHk0lj4auOC9u9jNg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhs1Kjsbs%253D&md5=c42863673c8b91fcb4a2c42c2b253ebc</span></div><a href="/servlet/linkout?suffix=cit74&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1620645114&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1620645114%26sid%3Dliteratum%253Aachs%26aulast%3DAhn%26aufirst%3DS.%26aulast%3DKahsai%26aufirst%3DA.%2BW.%26aulast%3DPani%26aufirst%3DB.%26aulast%3DWang%26aufirst%3DQ.-T.%26aulast%3DZhao%26aufirst%3DS.%26aulast%3DWall%26aufirst%3DA.%2BL.%26aulast%3DStrachan%26aufirst%3DR.%2BT.%26aulast%3DStaus%26aufirst%3DD.%2BP.%26aulast%3DWingler%26aufirst%3DL.%2BM.%26aulast%3DSun%26aufirst%3DL.%2BD.%26aulast%3DSinnaeve%26aufirst%3DJ.%26aulast%3DChoi%26aufirst%3DM.%26aulast%3DCho%26aufirst%3DT.%26aulast%3DXu%26aufirst%3DT.%2BT.%26aulast%3DHansen%26aufirst%3DG.%2BM.%26aulast%3DBurnett%26aufirst%3DM.%2BB.%26aulast%3DLamerdin%26aufirst%3DJ.%2BE.%26aulast%3DBassoni%26aufirst%3DD.%2BL.%26aulast%3DGavino%26aufirst%3DB.%2BJ.%26aulast%3DHusemoen%26aufirst%3DG.%26aulast%3DOlsen%26aufirst%3DE.%2BK.%26aulast%3DFranch%26aufirst%3DT.%26aulast%3DCostanzi%26aufirst%3DS.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DLefkowitz%26aufirst%3DR.%2BJ.%26atitle%3DAllosteric%2520%25E2%2580%259Cbeta-Blocker%25E2%2580%259D%2520Isolated%2520from%2520a%2520DNA-Encoded%2520Small%2520Molecule%2520Library%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2017%26volume%3D114%26spage%3D1708%26epage%3D1713%26doi%3D10.1073%2Fpnas.1620645114" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref75"><div class="reference"><strong class="refLabel"><a href="#ref75" class="rightTabRefNumLink">75</a></strong><div class="NLM_citation" id="rightTab-cit75"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fernández-Montalván, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berger, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuropka, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koo, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Badock, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weiske, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Puetter, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holton, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stöckigt, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ter Laak, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Centrella, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clark, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dumelin, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sigel, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soutter, H. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Troast, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cuozzo, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keefe, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roche, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rodeschini, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chaikuad, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Díaz-Sáez, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bennett, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huber, K. V. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hübner, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weinmann, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hartung, I. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gorjánácz, M.</span></span> <span> </span><span class="NLM_article-title">Isoform-Selective ATAD2 Chemical Probe with Novel Chemical Structure and Unusual Mode of Action</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">2730</span>– <span class="NLM_lpage">2736</span>, <span class="refDoi"> DOI: 10.1021/acschembio.7b00708</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.7b00708" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhs1KqsrrL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2017&pages=2730-2736&author=A.+E.+Fern%C3%A1ndez-Montalv%C3%A1nauthor=M.+Bergerauthor=B.+Kuropkaauthor=S.+J.+Kooauthor=V.+Badockauthor=J.+Weiskeauthor=V.+Puetterauthor=S.+J.+Holtonauthor=D.+St%C3%B6ckigtauthor=A.+Ter+Laakauthor=P.+A.+Centrellaauthor=M.+A.+Clarkauthor=C.+E.+Dumelinauthor=E.+A.+Sigelauthor=H.+H.+Soutterauthor=D.+M.+Troastauthor=Y.+Zhangauthor=J.+W.+Cuozzoauthor=A.+D.+Keefeauthor=D.+Rocheauthor=V.+Rodeschiniauthor=A.+Chaikuadauthor=L.+D%C3%ADaz-S%C3%A1ezauthor=J.+M.+Bennettauthor=O.+Fedorovauthor=K.+V.+M.+Huberauthor=J.+H%C3%BCbnerauthor=H.+Weinmannauthor=I.+V.+Hartungauthor=M.+Gorj%C3%A1n%C3%A1cz&title=Isoform-Selective+ATAD2+Chemical+Probe+with+Novel+Chemical+Structure+and+Unusual+Mode+of+Action&doi=10.1021%2Facschembio.7b00708"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit75R"><div class="casContent"><span class="casTitleNuber">75</span><div class="casTitle"><span class="NLM_cas:atitle">Isoform-Selective ATAD2 Chemical Probe with Novel Chemical Structure and Unusual Mode of Action</span></div><div class="casAuthors">Fernandez-Montalvan, Amaury E.; Berger, Markus; Kuropka, Benno; Koo, Seong Joo; Badock, Volker; Weiske, Joerg; Puetter, Vera; Holton, Simon J.; Stoeckigt, Detlef; ter Laak, Antonius; Centrella, Paolo A.; Clark, Matthew A.; Dumelin, Christoph E.; Sigel, Eric A.; Soutter, Holly H.; Troast, Dawn M.; Zhang, Ying; Cuozzo, John W.; Keefe, Anthony D.; Roche, Didier; Rodeschini, Vincent; Chaikuad, Apirat; Diaz-Saez, Laura; Bennett, James M.; Fedorov, Oleg; Huber, Kilian V. M.; Huebner, Jan; Weinmann, Hilmar; Hartung, Ingo V.; Gorjanacz, Matyas</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2730-2736</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">ATAD2 (ANCCA) is an epigenetic regulator and transcriptional cofactor, whose overexpression has been linked to the progress of various cancer types.  Here, we report a DNA-encoded library screen leading to the discovery of BAY-850, a potent and isoform selective inhibitor that specifically induces ATAD2 bromodomain dimerization and prevents interactions with acetylated histones in vitro, as well as with chromatin in cells.  These features qualify BAY-850 as a chem. probe to explore ATAD2 biol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrCB3nyG9YvubVg90H21EOLACvtfcHk0lj4auOC9u9jNg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhs1KqsrrL&md5=1f09000a7d7e60e30540fbbae117acfe</span></div><a href="/servlet/linkout?suffix=cit75&amp;dbid=16384&amp;doi=10.1021%2Facschembio.7b00708&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.7b00708%26sid%3Dliteratum%253Aachs%26aulast%3DFern%25C3%25A1ndez-Montalv%25C3%25A1n%26aufirst%3DA.%2BE.%26aulast%3DBerger%26aufirst%3DM.%26aulast%3DKuropka%26aufirst%3DB.%26aulast%3DKoo%26aufirst%3DS.%2BJ.%26aulast%3DBadock%26aufirst%3DV.%26aulast%3DWeiske%26aufirst%3DJ.%26aulast%3DPuetter%26aufirst%3DV.%26aulast%3DHolton%26aufirst%3DS.%2BJ.%26aulast%3DSt%25C3%25B6ckigt%26aufirst%3DD.%26aulast%3DTer%2BLaak%26aufirst%3DA.%26aulast%3DCentrella%26aufirst%3DP.%2BA.%26aulast%3DClark%26aufirst%3DM.%2BA.%26aulast%3DDumelin%26aufirst%3DC.%2BE.%26aulast%3DSigel%26aufirst%3DE.%2BA.%26aulast%3DSoutter%26aufirst%3DH.%2BH.%26aulast%3DTroast%26aufirst%3DD.%2BM.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DCuozzo%26aufirst%3DJ.%2BW.%26aulast%3DKeefe%26aufirst%3DA.%2BD.%26aulast%3DRoche%26aufirst%3DD.%26aulast%3DRodeschini%26aufirst%3DV.%26aulast%3DChaikuad%26aufirst%3DA.%26aulast%3DD%25C3%25ADaz-S%25C3%25A1ez%26aufirst%3DL.%26aulast%3DBennett%26aufirst%3DJ.%2BM.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DHuber%26aufirst%3DK.%2BV.%2BM.%26aulast%3DH%25C3%25BCbner%26aufirst%3DJ.%26aulast%3DWeinmann%26aufirst%3DH.%26aulast%3DHartung%26aufirst%3DI.%2BV.%26aulast%3DGorj%25C3%25A1n%25C3%25A1cz%26aufirst%3DM.%26atitle%3DIsoform-Selective%2520ATAD2%2520Chemical%2520Probe%2520with%2520Novel%2520Chemical%2520Structure%2520and%2520Unusual%2520Mode%2520of%2520Action%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2017%26volume%3D12%26spage%3D2730%26epage%3D2736%26doi%3D10.1021%2Facschembio.7b00708" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref76"><div class="reference"><strong class="refLabel"><a href="#ref76" class="rightTabRefNumLink">76</a></strong><div class="NLM_citation" id="rightTab-cit76"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cuozzo, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Centrella, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gikunju, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Habeshian, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hupp, C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keefe, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sigel, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soutter, H. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomson, H. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clark, M. A.</span></span> <span> </span><span class="NLM_article-title">Discovery of a Potent BTK Inhibitor with a Novel Binding Mode by Using Parallel Selections with a DNA-Encoded Chemical Library</span>. <i>ChemBioChem</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">864</span>– <span class="NLM_lpage">871</span>, <span class="refDoi"> DOI: 10.1002/cbic.201600573</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1002%2Fcbic.201600573" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=28056160" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC2sXit1Chu7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2017&pages=864-871&author=J.+W.+Cuozzoauthor=P.+A.+Centrellaauthor=D.+Gikunjuauthor=S.+Habeshianauthor=C.+D.+Huppauthor=A.+D.+Keefeauthor=E.+A.+Sigelauthor=H.+H.+Soutterauthor=H.+A.+Thomsonauthor=Y.+Zhangauthor=M.+A.+Clark&title=Discovery+of+a+Potent+BTK+Inhibitor+with+a+Novel+Binding+Mode+by+Using+Parallel+Selections+with+a+DNA-Encoded+Chemical+Library&doi=10.1002%2Fcbic.201600573"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit76R"><div class="casContent"><span class="casTitleNuber">76</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a Potent BTK Inhibitor with a Novel Binding Mode by Using Parallel Selections with a DNA-Encoded Chemical Library</span></div><div class="casAuthors">Cuozzo, John W.; Centrella, Paolo A.; Gikunju, Diana; Habeshian, Sevan; Hupp, Christopher D.; Keefe, Anthony D.; Sigel, Eric A.; Soutter, Holly H.; Thomson, Heather A.; Zhang, Ying; Clark, Matthew A.</div><div class="citationInfo"><span class="NLM_cas:title">ChemBioChem</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">864-871</span>CODEN:
                <span class="NLM_cas:coden">CBCHFX</span>;
        ISSN:<span class="NLM_cas:issn">1439-4227</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">We have identified and characterized novel potent inhibitors of Bruton's tyrosine kinase (BTK) from a single DNA-encoded library of over 110 million compds. by using multiple parallel selection conditions, including variation in target concn. and addn. of known binders to provide competition information.  Distinct binding profiles were obsd. by comparing enrichments of library building block combinations under these conditions; one enriched only at high concns. of BTK and was competitive with ATP, and another enriched at both high and low concns. of BTK and was not competitive with ATP.  A compd. representing the latter profile showed low nanomolar potency in biochem. and cellular BTK assays.  Results from kinetic mechanism of action studies were consistent with the selection profiles.  Anal. of the co-crystal structure of the most potent compd. demonstrated a novel binding mode that revealed a new pocket in BTK.  Our results demonstrate that profile-based selection strategies using DNA-encoded libraries form the basis of a new methodol. to rapidly identify small mol. inhibitors with novel binding modes to clin. relevant targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqxaUUXy-jOi7Vg90H21EOLACvtfcHk0lih5siRBRXcdw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXit1Chu7Y%253D&md5=22423b0496eb68130236292433bc7251</span></div><a href="/servlet/linkout?suffix=cit76&amp;dbid=16384&amp;doi=10.1002%2Fcbic.201600573&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcbic.201600573%26sid%3Dliteratum%253Aachs%26aulast%3DCuozzo%26aufirst%3DJ.%2BW.%26aulast%3DCentrella%26aufirst%3DP.%2BA.%26aulast%3DGikunju%26aufirst%3DD.%26aulast%3DHabeshian%26aufirst%3DS.%26aulast%3DHupp%26aufirst%3DC.%2BD.%26aulast%3DKeefe%26aufirst%3DA.%2BD.%26aulast%3DSigel%26aufirst%3DE.%2BA.%26aulast%3DSoutter%26aufirst%3DH.%2BH.%26aulast%3DThomson%26aufirst%3DH.%2BA.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DClark%26aufirst%3DM.%2BA.%26atitle%3DDiscovery%2520of%2520a%2520Potent%2520BTK%2520Inhibitor%2520with%2520a%2520Novel%2520Binding%2520Mode%2520by%2520Using%2520Parallel%2520Selections%2520with%2520a%2520DNA-Encoded%2520Chemical%2520Library%26jtitle%3DChemBioChem%26date%3D2017%26volume%3D18%26spage%3D864%26epage%3D871%26doi%3D10.1002%2Fcbic.201600573" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref77"><div class="reference"><strong class="refLabel"><a href="#ref77" class="rightTabRefNumLink">77</a></strong><div class="NLM_citation" id="rightTab-cit77"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Petersen, L. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blakskjær, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chaikuad, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Christensen, A. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dietvorst, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holmkvist, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kořínek, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Larsen, L. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pedersen, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Röhm, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sløk, F. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hansen, N. J. V.</span></span> <span> </span><span class="NLM_article-title">Novel p38α MAP Kinase Inhibitors Identified from yoctoReactor DNA-Encoded Small Molecule Library</span>. <i>MedChemComm</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">1332</span>– <span class="NLM_lpage">1339</span>, <span class="refDoi"> DOI: 10.1039/C6MD00241B</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1039%2FC6MD00241B" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtVCis77J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=1332-1339&author=L.+K.+Petersenauthor=P.+Blakskj%C3%A6rauthor=A.+Chaikuadauthor=A.+B.+Christensenauthor=J.+Dietvorstauthor=J.+Holmkvistauthor=S.+Knappauthor=M.+Ko%C5%99%C3%ADnekauthor=L.+K.+Larsenauthor=A.+E.+Pedersenauthor=S.+R%C3%B6hmauthor=F.+A.+Sl%C3%B8kauthor=N.+J.+V.+Hansen&title=Novel+p38%CE%B1+MAP+Kinase+Inhibitors+Identified+from+yoctoReactor+DNA-Encoded+Small+Molecule+Library&doi=10.1039%2FC6MD00241B"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit77R"><div class="casContent"><span class="casTitleNuber">77</span><div class="casTitle"><span class="NLM_cas:atitle">Novel p38α MAP kinase inhibitors identified from yoctoReactor DNA-encoded small molecule library</span></div><div class="casAuthors">Petersen, L. K.; Blakskjaer, P.; Chaikuad, A.; Christensen, A. B.; Dietvorst, J.; Holmkvist, J.; Knapp, S.; Korinek, M.; Larsen, L. K.; Pedersen, A. E.; Rohm, S.; Sloek, F. A.; Hansen, N. J. V.</div><div class="citationInfo"><span class="NLM_cas:title">MedChemComm</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1332-1339</span>CODEN:
                <span class="NLM_cas:coden">MCCEAY</span>;
        ISSN:<span class="NLM_cas:issn">2040-2503</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">A highly specific and potent (7 nM cellular IC50) inhibitor of p38α kinase was identified directly from a 12.6 million membered DNA-encoded small mol. library.  This was achieved using the high fidelity yoctoReactor technol. (yR) for prepg. the DNA-encoded library, and a homogeneous screening technique - the binder trap enrichment technol. (BTE).  Although structurally atypical to other kinase blockers, this inhibitor was found by X-ray crystallog. to interact with the ATP binding site and provide strong distortion of the P-loop.  Remarkably, it assumed an alternative binding mode as it lacks key features of known kinase inhibitors such as typical hinge binding motifs.  Interestingly, the inhibitor bound assuming a canonical type-II ('DFG-out') binding mode by forming hinge hydrogen bonds with the backbone, showed excellent shape complementarity, and formed a no. of specific polar interactions.  Moreover, the crystal structure showed, that although buried in the p38α active site, the original DNA attachment point of the compd. was accessible through a channel created by the distorted P-loop conformation.  This study demonstrates the usability of DNA-encoded library technologies for identifying novel chem. matter with alternative binding modes to provide a good starting point for drug development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpgxosw66vOu7Vg90H21EOLACvtfcHk0lih5siRBRXcdw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtVCis77J&md5=8f4feb94165c51dcabc6110ff877dbe4</span></div><a href="/servlet/linkout?suffix=cit77&amp;dbid=16384&amp;doi=10.1039%2FC6MD00241B&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC6MD00241B%26sid%3Dliteratum%253Aachs%26aulast%3DPetersen%26aufirst%3DL.%2BK.%26aulast%3DBlakskj%25C3%25A6r%26aufirst%3DP.%26aulast%3DChaikuad%26aufirst%3DA.%26aulast%3DChristensen%26aufirst%3DA.%2BB.%26aulast%3DDietvorst%26aufirst%3DJ.%26aulast%3DHolmkvist%26aufirst%3DJ.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DKo%25C5%2599%25C3%25ADnek%26aufirst%3DM.%26aulast%3DLarsen%26aufirst%3DL.%2BK.%26aulast%3DPedersen%26aufirst%3DA.%2BE.%26aulast%3DR%25C3%25B6hm%26aufirst%3DS.%26aulast%3DSl%25C3%25B8k%26aufirst%3DF.%2BA.%26aulast%3DHansen%26aufirst%3DN.%2BJ.%2BV.%26atitle%3DNovel%2520p38%25CE%25B1%2520MAP%2520Kinase%2520Inhibitors%2520Identified%2520from%2520yoctoReactor%2520DNA-Encoded%2520Small%2520Molecule%2520Library%26jtitle%3DMedChemComm%26date%3D2016%26volume%3D7%26spage%3D1332%26epage%3D1339%26doi%3D10.1039%2FC6MD00241B" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref78"><div class="reference"><strong class="refLabel"><a href="#ref78" class="rightTabRefNumLink">78</a></strong><div class="NLM_citation" id="rightTab-cit78"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McMillan, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adler, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Auld, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baldwin, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blasko, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Browne, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chelsky, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davey, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dolle, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eagen, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Erickson, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feldman, R. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glaser, C. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mallari, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morrissey, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohlmeyer, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pan, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parkinson, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phillips, G. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Polokoff, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sigal, N. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vergona, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whitlow, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Devlin, J. J.</span></span> <span> </span><span class="NLM_article-title">Allosteric Inhibitors of Inducible Nitric Oxide Synthase Dimerization Discovered via Combinatorial Chemistry</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>97</i></span>,  <span class="NLM_fpage">1506</span>– <span class="NLM_lpage">1511</span>, <span class="refDoi"> DOI: 10.1073/pnas.97.4.1506</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1073%2Fpnas.97.4.1506" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=97&publication_year=2000&pages=1506-1511&author=K.+McMillanauthor=M.+Adlerauthor=D.+S.+Auldauthor=J.+J.+Baldwinauthor=E.+Blaskoauthor=L.+J.+Browneauthor=D.+Chelskyauthor=D.+Daveyauthor=R.+E.+Dolleauthor=K.+A.+Eagenauthor=S.+Ericksonauthor=R.+I.+Feldmanauthor=C.+B.+Glaserauthor=C.+Mallariauthor=M.+M.+Morrisseyauthor=M.+H.+Ohlmeyerauthor=G.+Panauthor=J.+F.+Parkinsonauthor=G.+B.+Phillipsauthor=M.+A.+Polokoffauthor=N.+H.+Sigalauthor=R.+Vergonaauthor=M.+Whitlowauthor=T.+A.+Youngauthor=J.+J.+Devlin&title=Allosteric+Inhibitors+of+Inducible+Nitric+Oxide+Synthase+Dimerization+Discovered+via+Combinatorial+Chemistry&doi=10.1073%2Fpnas.97.4.1506"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit78&amp;dbid=16384&amp;doi=10.1073%2Fpnas.97.4.1506&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.97.4.1506%26sid%3Dliteratum%253Aachs%26aulast%3DMcMillan%26aufirst%3DK.%26aulast%3DAdler%26aufirst%3DM.%26aulast%3DAuld%26aufirst%3DD.%2BS.%26aulast%3DBaldwin%26aufirst%3DJ.%2BJ.%26aulast%3DBlasko%26aufirst%3DE.%26aulast%3DBrowne%26aufirst%3DL.%2BJ.%26aulast%3DChelsky%26aufirst%3DD.%26aulast%3DDavey%26aufirst%3DD.%26aulast%3DDolle%26aufirst%3DR.%2BE.%26aulast%3DEagen%26aufirst%3DK.%2BA.%26aulast%3DErickson%26aufirst%3DS.%26aulast%3DFeldman%26aufirst%3DR.%2BI.%26aulast%3DGlaser%26aufirst%3DC.%2BB.%26aulast%3DMallari%26aufirst%3DC.%26aulast%3DMorrissey%26aufirst%3DM.%2BM.%26aulast%3DOhlmeyer%26aufirst%3DM.%2BH.%26aulast%3DPan%26aufirst%3DG.%26aulast%3DParkinson%26aufirst%3DJ.%2BF.%26aulast%3DPhillips%26aufirst%3DG.%2BB.%26aulast%3DPolokoff%26aufirst%3DM.%2BA.%26aulast%3DSigal%26aufirst%3DN.%2BH.%26aulast%3DVergona%26aufirst%3DR.%26aulast%3DWhitlow%26aufirst%3DM.%26aulast%3DYoung%26aufirst%3DT.%2BA.%26aulast%3DDevlin%26aufirst%3DJ.%2BJ.%26atitle%3DAllosteric%2520Inhibitors%2520of%2520Inducible%2520Nitric%2520Oxide%2520Synthase%2520Dimerization%2520Discovered%2520via%2520Combinatorial%2520Chemistry%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2000%26volume%3D97%26spage%3D1506%26epage%3D1511%26doi%3D10.1073%2Fpnas.97.4.1506" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref79"><div class="reference"><strong class="refLabel"><a href="#ref79" class="rightTabRefNumLink">79</a></strong><div class="NLM_citation" id="rightTab-cit79"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zambaldo, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daguer, J.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saarbach, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barluenga, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winssinger, N.</span></span> <span> </span><span class="NLM_article-title">Screening for Covalent Inhibitors Using DNA-Display of Small Molecule Libraries Functionalized with Cysteine Reactive Moieties</span>. <i>MedChemComm</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">1340</span>– <span class="NLM_lpage">1351</span>, <span class="refDoi"> DOI: 10.1039/C6MD00242K</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1039%2FC6MD00242K" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=1340-1351&author=C.+Zambaldoauthor=J.-P.+Daguerauthor=J.+Saarbachauthor=S.+Barluengaauthor=N.+Winssinger&title=Screening+for+Covalent+Inhibitors+Using+DNA-Display+of+Small+Molecule+Libraries+Functionalized+with+Cysteine+Reactive+Moieties&doi=10.1039%2FC6MD00242K"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit79&amp;dbid=16384&amp;doi=10.1039%2FC6MD00242K&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC6MD00242K%26sid%3Dliteratum%253Aachs%26aulast%3DZambaldo%26aufirst%3DC.%26aulast%3DDaguer%26aufirst%3DJ.-P.%26aulast%3DSaarbach%26aufirst%3DJ.%26aulast%3DBarluenga%26aufirst%3DS.%26aulast%3DWinssinger%26aufirst%3DN.%26atitle%3DScreening%2520for%2520Covalent%2520Inhibitors%2520Using%2520DNA-Display%2520of%2520Small%2520Molecule%2520Libraries%2520Functionalized%2520with%2520Cysteine%2520Reactive%2520Moieties%26jtitle%3DMedChemComm%26date%3D2016%26volume%3D7%26spage%3D1340%26epage%3D1351%26doi%3D10.1039%2FC6MD00242K" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref80"><div class="reference"><strong class="refLabel"><a href="#ref80" class="rightTabRefNumLink">80</a></strong><div class="NLM_citation" id="rightTab-cit80"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Salamon, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klika Škopić, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jung, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bugain, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brunschweiger, A.</span></span> <span> </span><span class="NLM_article-title">Chemical Biology Probes from Advanced DNA-Encoded Libraries</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">296</span>– <span class="NLM_lpage">307</span>, <span class="refDoi"> DOI: 10.1021/acschembio.5b00981</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.5b00981" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC28XitVyhtbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2016&pages=296-307&author=H.+Salamonauthor=M.+Klika+%C5%A0kopi%C4%87author=K.+Jungauthor=O.+Bugainauthor=A.+Brunschweiger&title=Chemical+Biology+Probes+from+Advanced+DNA-Encoded+Libraries&doi=10.1021%2Facschembio.5b00981"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit80R"><div class="casContent"><span class="casTitleNuber">80</span><div class="casTitle"><span class="NLM_cas:atitle">Chemical Biology Probes from Advanced DNA-encoded Libraries</span></div><div class="casAuthors">Salamon, Hazem; Klika Skopic, Mateja; Jung, Kathrin; Bugain, Olivia; Brunschweiger, Andreas</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">296-307</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The identification of bioactive compds. is a crucial step toward development of probes for chem. biol. studies.  Screening of DNA-encoded small mol. libraries (DELs) has emerged as a validated technol. to interrogate vast chem. space.  DELs consist of chimeric mols. composed of a low-mol. wt. compd. that is conjugated to a DNA identifier tag.  They are screened as pooled libraries using selection to identify "hits." Screening of DELs has identified numerous bioactive compds.  Some of these mols. were instrumental in gaining a deeper understanding of biol. systems.  One of the main challenges in the field is the development of synthesis methodol. for DELs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpoCCmTdlAe37Vg90H21EOLACvtfcHk0ljnZXO3Mqk7SA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitVyhtbk%253D&md5=a506c420e03fb229c928459b05c23406</span></div><a href="/servlet/linkout?suffix=cit80&amp;dbid=16384&amp;doi=10.1021%2Facschembio.5b00981&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.5b00981%26sid%3Dliteratum%253Aachs%26aulast%3DSalamon%26aufirst%3DH.%26aulast%3DKlika%2B%25C5%25A0kopi%25C4%2587%26aufirst%3DM.%26aulast%3DJung%26aufirst%3DK.%26aulast%3DBugain%26aufirst%3DO.%26aulast%3DBrunschweiger%26aufirst%3DA.%26atitle%3DChemical%2520Biology%2520Probes%2520from%2520Advanced%2520DNA-Encoded%2520Libraries%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2016%26volume%3D11%26spage%3D296%26epage%3D307%26doi%3D10.1021%2Facschembio.5b00981" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref81"><div class="reference"><strong class="refLabel"><a href="#ref81" class="rightTabRefNumLink">81</a></strong><div class="NLM_citation" id="rightTab-cit81"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Villar, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beglov, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chennamadhavuni, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Porco, J. A.,  Jr</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kozakov, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vajda, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whitty, A.</span></span> <span> </span><span class="NLM_article-title">How Proteins Bind Macrocycles</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">723</span>– <span class="NLM_lpage">731</span>, <span class="refDoi"> DOI: 10.1038/nchembio.1584</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1038%2Fnchembio.1584" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=25038790" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtFygs7jF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2014&pages=723-731&author=E.+A.+Villarauthor=D.+Beglovauthor=S.+Chennamadhavuniauthor=J.+A.+Porcoauthor=D.+Kozakovauthor=S.+Vajdaauthor=A.+Whitty&title=How+Proteins+Bind+Macrocycles&doi=10.1038%2Fnchembio.1584"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit81R"><div class="casContent"><span class="casTitleNuber">81</span><div class="casTitle"><span class="NLM_cas:atitle">How proteins bind macrocycles</span></div><div class="casAuthors">Villar, Elizabeth A.; Beglov, Dmitri; Chennamadhavuni, Spandan; Porco, John A. Jr; Kozakov, Dima; Vajda, Sandor; Whitty, Adrian</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">723-731</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The potential utility of synthetic macrocycles (MCs) as drugs, particularly against low-druggability targets such as protein-protein interactions, has been widely discussed.  There is little information, however, to guide the design of MCs for good target protein-binding activity or bioavailability.  To address this knowledge gap, we analyze the binding modes of a representative set of MC-protein complexes.  The results, combined with consideration of the physicochem. properties of approved macrocyclic drugs, allow us to propose specific guidelines for the design of synthetic MC libraries with structural and physicochem. features likely to favor strong binding to protein targets as well as good bioavailability.  We addnl. provide evidence that large, natural product-derived MCs can bind targets that are not druggable by conventional, drug-like compds., supporting the notion that natural product-inspired synthetic MCs can expand the no. of proteins that are druggable by synthetic small mols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqa3yuNCHU5d7Vg90H21EOLACvtfcHk0lhg7VKvQm5sZQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtFygs7jF&md5=0d41fba9267abb3a3d0065487937075a</span></div><a href="/servlet/linkout?suffix=cit81&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1584&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1584%26sid%3Dliteratum%253Aachs%26aulast%3DVillar%26aufirst%3DE.%2BA.%26aulast%3DBeglov%26aufirst%3DD.%26aulast%3DChennamadhavuni%26aufirst%3DS.%26aulast%3DPorco%26aufirst%3DJ.%2BA.%26aulast%3DKozakov%26aufirst%3DD.%26aulast%3DVajda%26aufirst%3DS.%26aulast%3DWhitty%26aufirst%3DA.%26atitle%3DHow%2520Proteins%2520Bind%2520Macrocycles%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2014%26volume%3D10%26spage%3D723%26epage%3D731%26doi%3D10.1038%2Fnchembio.1584" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref82"><div class="reference"><strong class="refLabel"><a href="#ref82" class="rightTabRefNumLink">82</a></strong><div class="NLM_citation" id="rightTab-cit82"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Giordanetto, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kihlberg, J.</span></span> <span> </span><span class="NLM_article-title">Macrocyclic Drugs and Clinical Candidates: What Can Medicinal Chemists Learn from Their Properties?</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">278</span>– <span class="NLM_lpage">295</span>, <span class="refDoi"> DOI: 10.1021/jm400887j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm400887j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtlGnurjI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=278-295&author=F.+Giordanettoauthor=J.+Kihlberg&title=Macrocyclic+Drugs+and+Clinical+Candidates%3A+What+Can+Medicinal+Chemists+Learn+from+Their+Properties%3F&doi=10.1021%2Fjm400887j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit82R"><div class="casContent"><span class="casTitleNuber">82</span><div class="casTitle"><span class="NLM_cas:atitle">Macrocyclic Drugs and Clinical Candidates: What Can Medicinal Chemists Learn from Their Properties?</span></div><div class="casAuthors">Giordanetto, Fabrizio; Kihlberg, Jan</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">278-295</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Macrocycles are ideal in efforts to tackle difficult targets, but our understanding of what makes them cell permeable and orally bioavailable is limited.  Anal. of approx. 100 macrocyclic drugs and clin. candidates revealed that macrocycles are predominantly used for infectious disease and in oncol. and that most belong to the macrolide or cyclic peptide class.  A significant no. (N = 34) of these macrocycles are administered orally, revealing that oral bioavailability can be obtained at mol. wts. up to and above 1 kDa and polar surface areas ranging toward 250 Å2.  Moreover, insight from a group of de novo designed oral macrocycles in clin. studies and understanding of how cyclosporin A and model cyclic hexapeptides cross cell membranes may unlock wider opportunities in drug discovery.  However, the no. of oral macrocycles is still low and it remains to be seen if they are outliers or if macrocycles will open up novel oral druggable space.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpaeM2DXwvmiLVg90H21EOLACvtfcHk0lhg7VKvQm5sZQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtlGnurjI&md5=a3e1743bd5ba34f73d8e78f8cb1bee5f</span></div><a href="/servlet/linkout?suffix=cit82&amp;dbid=16384&amp;doi=10.1021%2Fjm400887j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm400887j%26sid%3Dliteratum%253Aachs%26aulast%3DGiordanetto%26aufirst%3DF.%26aulast%3DKihlberg%26aufirst%3DJ.%26atitle%3DMacrocyclic%2520Drugs%2520and%2520Clinical%2520Candidates%253A%2520What%2520Can%2520Medicinal%2520Chemists%2520Learn%2520from%2520Their%2520Properties%253F%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D278%26epage%3D295%26doi%3D10.1021%2Fjm400887j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref83"><div class="reference"><strong class="refLabel"><a href="#ref83" class="rightTabRefNumLink">83</a></strong><div class="NLM_citation" id="rightTab-cit83"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Whitty, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Viarengo, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhong, M.</span></span> <span> </span><span class="NLM_article-title">Progress towards the Broad Use of Non-Peptide Synthetic Macrocycles in Drug Discovery</span>. <i>Org. Biomol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">7729</span>– <span class="NLM_lpage">7735</span>, <span class="refDoi"> DOI: 10.1039/C7OB00056A</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref83/cit83&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1039%2FC7OB00056A" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref83/cit83&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=28876025" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref83/cit83&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsVKgsLrO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2017&pages=7729-7735&author=A.+Whittyauthor=L.+A.+Viarengoauthor=M.+Zhong&title=Progress+towards+the+Broad+Use+of+Non-Peptide+Synthetic+Macrocycles+in+Drug+Discovery&doi=10.1039%2FC7OB00056A"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit83R"><div class="casContent"><span class="casTitleNuber">83</span><div class="casTitle"><span class="NLM_cas:atitle">Progress towards the broad use of non-peptide synthetic macrocycles in drug discovery</span></div><div class="casAuthors">Whitty, Adrian; Viarengo, Lauren A.; Zhong, Mengqi</div><div class="citationInfo"><span class="NLM_cas:title">Organic & Biomolecular Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">37</span>),
    <span class="NLM_cas:pages">7729-7735</span>CODEN:
                <span class="NLM_cas:coden">OBCRAK</span>;
        ISSN:<span class="NLM_cas:issn">1477-0520</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">A review.  The authors discuss progress towards addressing three key questions pertaining to the design of screening libraries of synthetic nonpeptidic macrocycles (MCs) for drug discovery: What structural and physicochem. properties of MCs maximize the likelihood of achieving strong and specific binding to protein targets.  What features render a protein target suitable for binding MCs, and can this information be used to identify suitable targets for inhibition by MCs.  What properties of synthetic MCs confer good pharmaceutical properties, and particularly good aq. soly. coupled with passive membrane permeability.  The authors addnl. discuss how the criteria that define a meaningful MC screening hit are linked to the size of the screening library and the synthetic methodol. employed in its prepn.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpeNdUQVdF6FrVg90H21EOLACvtfcHk0lhg7VKvQm5sZQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsVKgsLrO&md5=451ddddc4457d5a2ee84b384aa15508e</span></div><a href="/servlet/linkout?suffix=cit83&amp;dbid=16384&amp;doi=10.1039%2FC7OB00056A&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC7OB00056A%26sid%3Dliteratum%253Aachs%26aulast%3DWhitty%26aufirst%3DA.%26aulast%3DViarengo%26aufirst%3DL.%2BA.%26aulast%3DZhong%26aufirst%3DM.%26atitle%3DProgress%2520towards%2520the%2520Broad%2520Use%2520of%2520Non-Peptide%2520Synthetic%2520Macrocycles%2520in%2520Drug%2520Discovery%26jtitle%3DOrg.%2520Biomol.%2520Chem.%26date%3D2017%26volume%3D15%26spage%3D7729%26epage%3D7735%26doi%3D10.1039%2FC7OB00056A" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref84"><div class="reference"><strong class="refLabel"><a href="#ref84" class="rightTabRefNumLink">84</a></strong><div class="NLM_citation" id="rightTab-cit84"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Seigal, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Connors, W. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fraley, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Borzilleri, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carter, P. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Emanuel, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fargnoli, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lei, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Naglich, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pokross, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Posy, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Surti, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Talbott, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Terrett, N. K.</span></span> <span> </span><span class="NLM_article-title">The Discovery of Macrocyclic XIAP Antagonists from a DNA-Programmed Chemistry Library, and Their Optimization to Give Lead Compounds with in Vivo Antitumor Activity</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">2855</span>– <span class="NLM_lpage">2861</span>, <span class="refDoi"> DOI: 10.1021/jm501892g</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm501892g" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref84/cit84&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC2MXivF2qu78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=2855-2861&author=B.+A.+Seigalauthor=W.+H.+Connorsauthor=A.+Fraleyauthor=R.+M.+Borzilleriauthor=P.+H.+Carterauthor=S.+L.+Emanuelauthor=J.+Fargnoliauthor=K.+Kimauthor=M.+Leiauthor=J.+G.+Naglichauthor=M.+E.+Pokrossauthor=S.+L.+Posyauthor=H.+Shenauthor=N.+Surtiauthor=R.+Talbottauthor=Y.+Zhangauthor=N.+K.+Terrett&title=The+Discovery+of+Macrocyclic+XIAP+Antagonists+from+a+DNA-Programmed+Chemistry+Library%2C+and+Their+Optimization+to+Give+Lead+Compounds+with+in+Vivo+Antitumor+Activity&doi=10.1021%2Fjm501892g"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit84R"><div class="casContent"><span class="casTitleNuber">84</span><div class="casTitle"><span class="NLM_cas:atitle">The Discovery of Macrocyclic XIAP Antagonists from a DNA-Programmed Chemistry Library, and Their Optimization To Give Lead Compounds with in Vivo Antitumor Activity</span></div><div class="casAuthors">Seigal, Benjamin A.; Connors, William H.; Fraley, Andrew; Borzilleri, Robert M.; Carter, Percy H.; Emanuel, Stuart L.; Fargnoli, Joseph; Kim, Kyoung; Lei, Ming; Naglich, Joseph G.; Pokross, Matthew E.; Posy, Shana L.; Shen, Henry; Surti, Neha; Talbott, Randy; Zhang, Yong; Terrett, Nicholas K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2855-2861</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Affinity selection screening of macrocycle libraries derived from DNA-programmed chem. identified XIAP BIR2 and BIR3 domain inhibitors that displace bound pro-apoptotic caspases.  X-ray cocrystal structures of key compds. with XIAP BIR2 suggested potency-enhancing structural modifications.  Optimization of dimeric macrocycles with similar affinity for both domains were potent pro-apoptotic agents in cancer cell lines and efficacious in shrinking tumors in a mouse xenograft model.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrlB6i1KOFJjrVg90H21EOLACvtfcHk0lhg7VKvQm5sZQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXivF2qu78%253D&md5=9bd53cc3ed014796248e8048cecea0d1</span></div><a href="/servlet/linkout?suffix=cit84&amp;dbid=16384&amp;doi=10.1021%2Fjm501892g&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm501892g%26sid%3Dliteratum%253Aachs%26aulast%3DSeigal%26aufirst%3DB.%2BA.%26aulast%3DConnors%26aufirst%3DW.%2BH.%26aulast%3DFraley%26aufirst%3DA.%26aulast%3DBorzilleri%26aufirst%3DR.%2BM.%26aulast%3DCarter%26aufirst%3DP.%2BH.%26aulast%3DEmanuel%26aufirst%3DS.%2BL.%26aulast%3DFargnoli%26aufirst%3DJ.%26aulast%3DKim%26aufirst%3DK.%26aulast%3DLei%26aufirst%3DM.%26aulast%3DNaglich%26aufirst%3DJ.%2BG.%26aulast%3DPokross%26aufirst%3DM.%2BE.%26aulast%3DPosy%26aufirst%3DS.%2BL.%26aulast%3DShen%26aufirst%3DH.%26aulast%3DSurti%26aufirst%3DN.%26aulast%3DTalbott%26aufirst%3DR.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DTerrett%26aufirst%3DN.%2BK.%26atitle%3DThe%2520Discovery%2520of%2520Macrocyclic%2520XIAP%2520Antagonists%2520from%2520a%2520DNA-Programmed%2520Chemistry%2520Library%252C%2520and%2520Their%2520Optimization%2520to%2520Give%2520Lead%2520Compounds%2520with%2520in%2520Vivo%2520Antitumor%2520Activity%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D2855%26epage%3D2861%26doi%3D10.1021%2Fjm501892g" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref85"><div class="reference"><strong class="refLabel"><a href="#ref85" class="rightTabRefNumLink">85</a></strong><div class="NLM_citation" id="rightTab-cit85"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Groves, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dömling, A.</span></span> <span> </span><span class="NLM_article-title">Artificial Macrocycles as IL-17A/IL-17RA Antagonists</span>. <i>MedChemComm</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">22</span>– <span class="NLM_lpage">26</span>, <span class="refDoi"> DOI: 10.1039/C7MD00464H</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref85/cit85&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1039%2FC7MD00464H" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref85/cit85&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=29750108" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref85/cit85&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhslagtL3L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2018&pages=22-26&author=W.+Wangauthor=M.+R.+Grovesauthor=A.+D%C3%B6mling&title=Artificial+Macrocycles+as+IL-17A%2FIL-17RA+Antagonists&doi=10.1039%2FC7MD00464H"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit85R"><div class="casContent"><span class="casTitleNuber">85</span><div class="casTitle"><span class="NLM_cas:atitle">Artificial macrocycles as IL-17A/IL-17RA antagonists</span></div><div class="casAuthors">Wang, Wenjia; Groves, Matthew R.; Doemling, Alexander</div><div class="citationInfo"><span class="NLM_cas:title">MedChemComm</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">22-26</span>CODEN:
                <span class="NLM_cas:coden">MCCEAY</span>;
        ISSN:<span class="NLM_cas:issn">2040-2503</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">A review.  Interleukin 17(A) is a pro-inflammatory cytokine involved in several auto-immune and inflammatory diseases.  Current antagonists against IL-17(A) or its receptor (IL-17RA) that show efficacy in clin. trials are monoclonal antibodies (mAbs).  However, recently designed artificial macrocycles are potent IL-17A/IL-17RA antagonists.  Based on co-crystal structures, a better understanding of the biol. activity and SAR of the macrocycles has been gained, demonstrating that they can compete with mAbs for difficult targets such as PPIs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpMncYtypRDh7Vg90H21EOLACvtfcHk0lgl-NAxkQA94w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhslagtL3L&md5=62b3ef7b447d2367e337dad6e5a3a204</span></div><a href="/servlet/linkout?suffix=cit85&amp;dbid=16384&amp;doi=10.1039%2FC7MD00464H&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC7MD00464H%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DW.%26aulast%3DGroves%26aufirst%3DM.%2BR.%26aulast%3DD%25C3%25B6mling%26aufirst%3DA.%26atitle%3DArtificial%2520Macrocycles%2520as%2520IL-17A%252FIL-17RA%2520Antagonists%26jtitle%3DMedChemComm%26date%3D2018%26volume%3D9%26spage%3D22%26epage%3D26%26doi%3D10.1039%2FC7MD00464H" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref86"><div class="reference"><strong class="refLabel"><a href="#ref86" class="rightTabRefNumLink">86</a></strong><div class="NLM_citation" id="rightTab-cit86"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Maianti, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McFedries, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Foda, Z. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kleiner, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Du, X. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leissring, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, W.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Charron, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seeliger, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saghatelian, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, D. R.</span></span> <span> </span><span class="NLM_article-title">Anti-Diabetic Activity of Insulin-Degrading Enzyme Inhibitors Mediated by Multiple Hormones</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>511</i></span>,  <span class="NLM_fpage">94</span>– <span class="NLM_lpage">98</span>, <span class="refDoi"> DOI: 10.1038/nature13297</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref86/cit86&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1038%2Fnature13297" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref86/cit86&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=24847884" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref86/cit86&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC2cXpslGitbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=511&publication_year=2014&pages=94-98&author=J.+P.+Maiantiauthor=A.+McFedriesauthor=Z.+H.+Fodaauthor=R.+E.+Kleinerauthor=X.+Q.+Duauthor=M.+A.+Leissringauthor=W.-J.+Tangauthor=M.+J.+Charronauthor=M.+A.+Seeligerauthor=A.+Saghatelianauthor=D.+R.+Liu&title=Anti-Diabetic+Activity+of+Insulin-Degrading+Enzyme+Inhibitors+Mediated+by+Multiple+Hormones&doi=10.1038%2Fnature13297"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit86R"><div class="casContent"><span class="casTitleNuber">86</span><div class="casTitle"><span class="NLM_cas:atitle">Anti-diabetic activity of insulin-degrading enzyme inhibitors mediated by multiple hormones</span></div><div class="casAuthors">Maianti, Juan Pablo; McFedries, Amanda; Foda, Zachariah H.; Kleiner, Ralph E.; Du, Xiu Quan; Leissring, Malcolm A.; Tang, Wei-Jen; Charron, Maureen J.; Seeliger, Markus A.; Saghatelian, Alan; Liu, David R.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">511</span>
        (<span class="NLM_cas:issue">7507</span>),
    <span class="NLM_cas:pages">94-98</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Despite decades of speculation that inhibiting endogenous insulin degrdn. might treat type-2 diabetes, and the identification of IDE (insulin-degrading enzyme) as a diabetes susceptibility gene, the relationship between the activity of the zinc metalloprotein IDE and glucose homeostasis remains unclear.  Although Ide-/- mice have elevated insulin levels, they exhibit impaired, rather than improved, glucose tolerance that may arise from compensatory insulin signaling dysfunction.  IDE inhibitors that are active in vivo are therefore needed to elucidate IDE's physiol. roles and to det. its potential to serve as a target for the treatment of diabetes.  Here we report the discovery of a physiol. active IDE inhibitor identified from a DNA-templated macrocycle library.  An X-ray structure of the macrocycle bound to IDE reveals that it engages a binding pocket away from the catalytic site, which explains its remarkable selectivity.  Treatment of lean and obese mice with this inhibitor shows that IDE regulates the abundance and signaling of glucagon and amylin, in addn. to that of insulin.  Under physiol. conditions that augment insulin and amylin levels, such as oral glucose administration, acute IDE inhibition leads to substantially improved glucose tolerance and slower gastric emptying.  These findings demonstrate the feasibility of modulating IDE activity as a new therapeutic strategy to treat type-2 diabetes and expand our understanding of the roles of IDE in glucose and hormone regulation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqaRhLWpG6WfbVg90H21EOLACvtfcHk0lgl-NAxkQA94w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXpslGitbg%253D&md5=803357a4dc422275db54d15aa7518f89</span></div><a href="/servlet/linkout?suffix=cit86&amp;dbid=16384&amp;doi=10.1038%2Fnature13297&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature13297%26sid%3Dliteratum%253Aachs%26aulast%3DMaianti%26aufirst%3DJ.%2BP.%26aulast%3DMcFedries%26aufirst%3DA.%26aulast%3DFoda%26aufirst%3DZ.%2BH.%26aulast%3DKleiner%26aufirst%3DR.%2BE.%26aulast%3DDu%26aufirst%3DX.%2BQ.%26aulast%3DLeissring%26aufirst%3DM.%2BA.%26aulast%3DTang%26aufirst%3DW.-J.%26aulast%3DCharron%26aufirst%3DM.%2BJ.%26aulast%3DSeeliger%26aufirst%3DM.%2BA.%26aulast%3DSaghatelian%26aufirst%3DA.%26aulast%3DLiu%26aufirst%3DD.%2BR.%26atitle%3DAnti-Diabetic%2520Activity%2520of%2520Insulin-Degrading%2520Enzyme%2520Inhibitors%2520Mediated%2520by%2520Multiple%2520Hormones%26jtitle%3DNature%26date%3D2014%26volume%3D511%26spage%3D94%26epage%3D98%26doi%3D10.1038%2Fnature13297" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref87"><div class="reference"><strong class="refLabel"><a href="#ref87" class="rightTabRefNumLink">87</a></strong><div class="NLM_citation" id="rightTab-cit87"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Usanov, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, A. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maianti, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, D. R.</span></span> <span> </span><span class="NLM_article-title">Second-Generation DNA-Templated Macrocycle Libraries for the Discovery of Bioactive Small Molecules</span>. <i>Nat. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>, <span class="refDoi"> DOI: 10.1038/s41557-018-0033-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref87/cit87&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1038%2Fs41557-018-0033-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref87/cit87&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=29610462" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2018&author=D.+L.+Usanovauthor=A.+I.+Chanauthor=J.+P.+Maiantiauthor=D.+R.+Liu&title=Second-Generation+DNA-Templated+Macrocycle+Libraries+for+the+Discovery+of+Bioactive+Small+Molecules&doi=10.1038%2Fs41557-018-0033-8"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit87&amp;dbid=16384&amp;doi=10.1038%2Fs41557-018-0033-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41557-018-0033-8%26sid%3Dliteratum%253Aachs%26aulast%3DUsanov%26aufirst%3DD.%2BL.%26aulast%3DChan%26aufirst%3DA.%2BI.%26aulast%3DMaianti%26aufirst%3DJ.%2BP.%26aulast%3DLiu%26aufirst%3DD.%2BR.%26atitle%3DSecond-Generation%2520DNA-Templated%2520Macrocycle%2520Libraries%2520for%2520the%2520Discovery%2520of%2520Bioactive%2520Small%2520Molecules%26jtitle%3DNat.%2520Chem.%26date%3D2018%26doi%3D10.1038%2Fs41557-018-0033-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref88"><div class="reference"><strong class="refLabel"><a href="#ref88" class="rightTabRefNumLink">88</a></strong><div class="NLM_citation" id="rightTab-cit88"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Saupe, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kunz, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haustedt, L. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jakupovic, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mang, C.</span></span> <span> </span><span class="NLM_article-title">MacroEvoLution: A New Method for the Rapid Generation of Novel Scaffold-Diverse Macrocyclic Libraries</span>. <i>Chem. - Eur. J.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">11784</span>– <span class="NLM_lpage">11791</span>, <span class="refDoi"> DOI: 10.1002/chem.201703209</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref88/cit88&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1002%2Fchem.201703209" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref88/cit88&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=28715083" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref88/cit88&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC2sXht12hs7jO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2017&pages=11784-11791&author=J.+Saupeauthor=O.+Kunzauthor=L.+O.+Haustedtauthor=S.+Jakupovicauthor=C.+Mang&title=MacroEvoLution%3A+A+New+Method+for+the+Rapid+Generation+of+Novel+Scaffold-Diverse+Macrocyclic+Libraries&doi=10.1002%2Fchem.201703209"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit88R"><div class="casContent"><span class="casTitleNuber">88</span><div class="casTitle"><span class="NLM_cas:atitle">MacroEvoLution: A new method for the rapid generation of novel scaffold-diverse macrocyclic libraries</span></div><div class="casAuthors">Saupe, Joern; Kunz, Oliver; Haustedt, Lars Ole; Jakupovic, Sven; Mang, Christian</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry - A European Journal</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">49</span>),
    <span class="NLM_cas:pages">11784-11791</span>CODEN:
                <span class="NLM_cas:coden">CEUJED</span>;
        ISSN:<span class="NLM_cas:issn">0947-6539</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Macrocycles are a structural class bearing great promise for future challenges in medicinal chem.  Nevertheless, there are few flexible approaches for the rapid generation of structurally diverse macrocyclic compd. collections.  Here, an efficient method for the generation of novel macrocyclic peptide-based scaffolds is reported.  The process, named here as "MacroEvoLution", is based on a cyclization screening approach that gives reliable access to novel macrocyclic architectures.  Classification of building blocks into specific pools ensures that scaffolds with orthogonally addressable functionalities are generated, which can easily be used for the generation of structurally diverse compd. libraries.  The method grants rapid access to novel scaffolds with scalable synthesis (multi gram scale) and the introduction of further diversity at a late stage.  Despite being developed for peptidic systems, the approach can easily be extended for the synthesis of systems with a decreased peptidic character.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrAkGYsnFt95rVg90H21EOLACvtfcHk0lgl-NAxkQA94w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXht12hs7jO&md5=c266860b10199fde8140d50ec18ec9b6</span></div><a href="/servlet/linkout?suffix=cit88&amp;dbid=16384&amp;doi=10.1002%2Fchem.201703209&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fchem.201703209%26sid%3Dliteratum%253Aachs%26aulast%3DSaupe%26aufirst%3DJ.%26aulast%3DKunz%26aufirst%3DO.%26aulast%3DHaustedt%26aufirst%3DL.%2BO.%26aulast%3DJakupovic%26aufirst%3DS.%26aulast%3DMang%26aufirst%3DC.%26atitle%3DMacroEvoLution%253A%2520A%2520New%2520Method%2520for%2520the%2520Rapid%2520Generation%2520of%2520Novel%2520Scaffold-Diverse%2520Macrocyclic%2520Libraries%26jtitle%3DChem.%2520-%2520Eur.%2520J.%26date%3D2017%26volume%3D23%26spage%3D11784%26epage%3D11791%26doi%3D10.1002%2Fchem.201703209" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref89"><div class="reference"><strong class="refLabel"><a href="#ref89" class="rightTabRefNumLink">89</a></strong><div class="NLM_citation" id="rightTab-cit89"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ermert, P.</span></span> <span> </span><span class="NLM_article-title">Design, Properties and Recent Application of Macrocycles in Medicinal Chemistry</span>. <i>Chimia</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>71</i></span>,  <span class="NLM_fpage">678</span>– <span class="NLM_lpage">702</span>, <span class="refDoi"> DOI: 10.2533/chimia.2017.678</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref89/cit89&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.2533%2Fchimia.2017.678" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref89/cit89&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=29070413" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref89/cit89&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhsFeitbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=71&publication_year=2017&pages=678-702&author=P.+Ermert&title=Design%2C+Properties+and+Recent+Application+of+Macrocycles+in+Medicinal+Chemistry&doi=10.2533%2Fchimia.2017.678"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit89R"><div class="casContent"><span class="casTitleNuber">89</span><div class="casTitle"><span class="NLM_cas:atitle">Design, properties and recent application of macrocycles in medicinal chemistry</span></div><div class="casAuthors">Ermert, Philipp</div><div class="citationInfo"><span class="NLM_cas:title">Chimia</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">71</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">678-702</span>CODEN:
                <span class="NLM_cas:coden">CHIMAD</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    
            (<span class="NLM_cas:orgname">Swiss Chemical Society</span>)
        </div><div class="casAbstract">Macrocyclic compds. have recently received increasing attention in drug discovery as these compds. offer the potential to modulate difficult targets and to access novel chemotypes.  Approaches towards libraries of macrocyclic compds. based on modular org. synthesis and applications of these technol. platforms to find and improve biol. active compds. are introduced in this minireview.  Alternatively, lead compds. may be obtained by truncation and modification of macrocyclic natural products.  Selected medicinal chem. programs are discussed, illustrating a macrocyclization approach toward ligands with improved properties.  The design of such ligands is often informed by X-ray crystal structures of protein-ligand complexes.  Efforts to understand cellular permeability and oral bioavailability of cyclic peptides and non-peptidic macrocycles are summarized.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqo_L_ef2BYLrVg90H21EOLACvtfcHk0liN0cFYzRtmEA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhsFeitbg%253D&md5=aca0ab87a30863de362d9002c3d9f6cc</span></div><a href="/servlet/linkout?suffix=cit89&amp;dbid=16384&amp;doi=10.2533%2Fchimia.2017.678&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2533%252Fchimia.2017.678%26sid%3Dliteratum%253Aachs%26aulast%3DErmert%26aufirst%3DP.%26atitle%3DDesign%252C%2520Properties%2520and%2520Recent%2520Application%2520of%2520Macrocycles%2520in%2520Medicinal%2520Chemistry%26jtitle%3DChimia%26date%3D2017%26volume%3D71%26spage%3D678%26epage%3D702%26doi%3D10.2533%2Fchimia.2017.678" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref90"><div class="reference"><strong class="refLabel"><a href="#ref90" class="rightTabRefNumLink">90</a></strong><div class="NLM_citation" id="rightTab-cit90"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Boss, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hazemann, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kimmerlin, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">von Korff, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lüthi, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peter, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sander, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siegrist, R.</span></span> <span> </span><span class="NLM_article-title">The Screening Compound Collection: A Key Asset for Drug Discovery</span>. <i>Chimia</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>71</i></span>,  <span class="NLM_fpage">667</span>– <span class="NLM_lpage">677</span>, <span class="refDoi"> DOI: 10.2533/chimia.2017.667</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref90/cit90&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.2533%2Fchimia.2017.667" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=71&publication_year=2017&pages=667-677&author=C.+Bossauthor=J.+Hazemannauthor=T.+Kimmerlinauthor=M.+von+Korffauthor=U.+L%C3%BCthiauthor=O.+Peterauthor=T.+Sanderauthor=R.+Siegrist&title=The+Screening+Compound+Collection%3A+A+Key+Asset+for+Drug+Discovery&doi=10.2533%2Fchimia.2017.667"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit90&amp;dbid=16384&amp;doi=10.2533%2Fchimia.2017.667&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2533%252Fchimia.2017.667%26sid%3Dliteratum%253Aachs%26aulast%3DBoss%26aufirst%3DC.%26aulast%3DHazemann%26aufirst%3DJ.%26aulast%3DKimmerlin%26aufirst%3DT.%26aulast%3Dvon%2BKorff%26aufirst%3DM.%26aulast%3DL%25C3%25BCthi%26aufirst%3DU.%26aulast%3DPeter%26aufirst%3DO.%26aulast%3DSander%26aufirst%3DT.%26aulast%3DSiegrist%26aufirst%3DR.%26atitle%3DThe%2520Screening%2520Compound%2520Collection%253A%2520A%2520Key%2520Asset%2520for%2520Drug%2520Discovery%26jtitle%3DChimia%26date%3D2017%26volume%3D71%26spage%3D667%26epage%3D677%26doi%3D10.2533%2Fchimia.2017.667" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref91"><div class="reference"><strong class="refLabel"><a href="#ref91" class="rightTabRefNumLink">91</a></strong><div class="NLM_citation" id="rightTab-cit91"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Caenepeel, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Belmontes, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coxon, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moody, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hughes, P. E.</span></span> <span> </span><span class="NLM_article-title">Abstract 2027: Preclinical Evaluation of AMG 176, a Novel, Potent and Selective Mcl-1 Inhibitor with Robust Anti-Tumor Activity in Mcl-1 Dependent Cancer Models</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>77</i></span>,  <span class="NLM_fpage">2027</span>, <span class="refDoi"> DOI: 10.1158/1538-7445.AM2017-2027</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref91/cit91&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1158%2F1538-7445.AM2017-2027" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=77&publication_year=2017&pages=2027&author=S.+R.+Caenepeelauthor=B.+Belmontesauthor=J.+Sunauthor=A.+Coxonauthor=G.+Moodyauthor=P.+E.+Hughes&title=Abstract+2027%3A+Preclinical+Evaluation+of+AMG+176%2C+a+Novel%2C+Potent+and+Selective+Mcl-1+Inhibitor+with+Robust+Anti-Tumor+Activity+in+Mcl-1+Dependent+Cancer+Models&doi=10.1158%2F1538-7445.AM2017-2027"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit91&amp;dbid=16384&amp;doi=10.1158%2F1538-7445.AM2017-2027&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1538-7445.AM2017-2027%26sid%3Dliteratum%253Aachs%26aulast%3DCaenepeel%26aufirst%3DS.%2BR.%26aulast%3DBelmontes%26aufirst%3DB.%26aulast%3DSun%26aufirst%3DJ.%26aulast%3DCoxon%26aufirst%3DA.%26aulast%3DMoody%26aufirst%3DG.%26aulast%3DHughes%26aufirst%3DP.%2BE.%26atitle%3DAbstract%25202027%253A%2520Preclinical%2520Evaluation%2520of%2520AMG%2520176%252C%2520a%2520Novel%252C%2520Potent%2520and%2520Selective%2520Mcl-1%2520Inhibitor%2520with%2520Robust%2520Anti-Tumor%2520Activity%2520in%2520Mcl-1%2520Dependent%2520Cancer%2520Models%26jtitle%3DCancer%2520Res.%26date%3D2017%26volume%3D77%26spage%3D2027%26doi%3D10.1158%2F1538-7445.AM2017-2027" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref92"><div class="reference"><strong class="refLabel"><a href="#ref92" class="rightTabRefNumLink">92</a></strong><div class="NLM_citation" id="rightTab-cit92"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Facchinetti, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tiseo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Di Maio, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Graziano, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bria, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rossi, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Novello, S.</span></span> <span> </span><span class="NLM_article-title">Tackling ALK in Non-Small Cell Lung Cancer: The Role of Novel Inhibitors</span>. <i>Transl. Lung Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">301</span>– <span class="NLM_lpage">321</span>, <span class="refDoi"> DOI: 10.21037/tlcr.2016.06.10</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref92/cit92&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.21037%2Ftlcr.2016.06.10" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref92/cit92&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=27413712" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref92/cit92&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsFekt7jI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2016&pages=301-321&author=F.+Facchinettiauthor=M.+Tiseoauthor=M.+Di+Maioauthor=P.+Grazianoauthor=E.+Briaauthor=G.+Rossiauthor=S.+Novello&title=Tackling+ALK+in+Non-Small+Cell+Lung+Cancer%3A+The+Role+of+Novel+Inhibitors&doi=10.21037%2Ftlcr.2016.06.10"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit92R"><div class="casContent"><span class="casTitleNuber">92</span><div class="casTitle"><span class="NLM_cas:atitle">Tackling ALK in non-small cell lung cancer: the role of novel inhibitors</span></div><div class="casAuthors">Facchinetti, Francesco; Tiseo, Marcello; Di Maio, Massimo; Graziano, Paolo; Bria, Emilio; Rossi, Giulio; Novello, Silvia</div><div class="citationInfo"><span class="NLM_cas:title">Translational Lung Cancer Research</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">301-321</span>CODEN:
                <span class="NLM_cas:coden">TLCRA9</span>;
        ISSN:<span class="NLM_cas:issn">2226-4477</span>.
    
            (<span class="NLM_cas:orgname">Pioneer Bioscience Publishing Co.</span>)
        </div><div class="casAbstract">Crizotinib is an oral inhibitor of anaplastic lymphoma kinase (ALK) with remarkable clin. activity in patients suffering from ALK-rearranged non-small cell lung cancer (NSCLC), accounting to its superiority compared to chemotherapy.  Unfortunately, virtually all ALK-rearranged tumors acquire resistance to crizotinib, frequently within one year since the treatment initiation.  To date, therapeutic strategies to overcome crizotinib resistance have focused on the use of more potent and structurally different compds.  Second-generation ALK inhibitors such as ceritinib (LDK378), alectinib (CH5424802/ RO5424802) and brigatinib (AP26113) have shown relevant clin. activity, consequently fostering their rapid clin. development and their approval by health agencies.  The third-generation inhibitor lorlatinib (PF-06463922), selectively active against ALK and ROS1, harbors impressive biol. potency; its efficacy in reversing resistance to crizotinib and to other ALK inhibitors is being proven by early clin. trials.  The NTRK1-3 and ROS1 inhibitor entrectinib (RXDX-101) has been reported to act against NSCLC harboring ALK fusion proteins too.  Despite the quick development of these novel agents, several issues remain to be discussed in the treatment of patients suffering from ALK-rearranged NSCLC.  This position paper will discuss the development, the current evidence and approvals, as long as the future perspectives of new ALK inhibitors beyond crizotinib.  Clin. behaviors of ALK-rearranged NSCLC vary significantly among patients and differential mol. events responsible of crizotinib resistance account for the most important quote of this heterogeneity.  The precious availability of a wide range of active anti-ALK compds. should be approached in a crit. and careful perspective, in order to develop treatment strategies tailored on the disease evolution of every single patient.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGop_lMlX4zg0LVg90H21EOLACvtfcHk0liN0cFYzRtmEA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsFekt7jI&md5=4612d0c957b51f6407488ddb10ab41e0</span></div><a href="/servlet/linkout?suffix=cit92&amp;dbid=16384&amp;doi=10.21037%2Ftlcr.2016.06.10&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.21037%252Ftlcr.2016.06.10%26sid%3Dliteratum%253Aachs%26aulast%3DFacchinetti%26aufirst%3DF.%26aulast%3DTiseo%26aufirst%3DM.%26aulast%3DDi%2BMaio%26aufirst%3DM.%26aulast%3DGraziano%26aufirst%3DP.%26aulast%3DBria%26aufirst%3DE.%26aulast%3DRossi%26aufirst%3DG.%26aulast%3DNovello%26aufirst%3DS.%26atitle%3DTackling%2520ALK%2520in%2520Non-Small%2520Cell%2520Lung%2520Cancer%253A%2520The%2520Role%2520of%2520Novel%2520Inhibitors%26jtitle%3DTransl.%2520Lung%2520Cancer%2520Res.%26date%3D2016%26volume%3D5%26spage%3D301%26epage%3D321%26doi%3D10.21037%2Ftlcr.2016.06.10" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref93"><div class="reference"><strong class="refLabel"><a href="#ref93" class="rightTabRefNumLink">93</a></strong><div class="NLM_citation" id="rightTab-cit93"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Morgan, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, D. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lennard, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anderton, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barrett, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eriksson, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fidock, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamrén, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">March, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matcham, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mettetal, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nicholls, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Platz, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rees, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Snowden, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pangalos, M. N.</span></span> <span> </span><span class="NLM_article-title">Impact of a Five-Dimensional Framework on R&D Productivity at AstraZeneca</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">167</span>– <span class="NLM_lpage">181</span>, <span class="refDoi"> DOI: 10.1038/nrd.2017.244</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref93/cit93&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1038%2Fnrd.2017.244" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref93/cit93&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=29348681" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref93/cit93&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC1cXht1SitL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2018&pages=167-181&author=P.+Morganauthor=D.+G.+Brownauthor=S.+Lennardauthor=M.+J.+Andertonauthor=J.+C.+Barrettauthor=U.+Erikssonauthor=M.+Fidockauthor=B.+Hamr%C3%A9nauthor=A.+Johnsonauthor=R.+E.+Marchauthor=J.+Matchamauthor=J.+Mettetalauthor=D.+J.+Nichollsauthor=S.+Platzauthor=S.+Reesauthor=M.+A.+Snowdenauthor=M.+N.+Pangalos&title=Impact+of+a+Five-Dimensional+Framework+on+R%26D+Productivity+at+AstraZeneca&doi=10.1038%2Fnrd.2017.244"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit93R"><div class="casContent"><span class="casTitleNuber">93</span><div class="casTitle"><span class="NLM_cas:atitle">Impact of a five-dimensional framework on R&D productivity at AstraZeneca</span></div><div class="casAuthors">Morgan, Paul; Brown, Dean G.; Lennard, Simon; Anderton, Mark J.; Barrett, J. Carl; Eriksson, Ulf; Fidock, Mark; Hamren, Bengt; Johnson, Anthony; March, Ruth E.; Matcham, James; Mettetal, Jerome; Nicholls, David J.; Platz, Stefan; Rees, Steve; Snowden, Michael A.; Pangalos, Menelas N.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">167-181</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">In 2011, AstraZeneca embarked on a major revision of its research and development (R&D) strategy with the aim of improving R&D productivity, which was below industry avs. in 2005-2010.  A cornerstone of the revised strategy was to focus decision-making on five tech. determinants (the right target, right tissue, right safety, right patient and right com. potential).  In this article, we describe the progress made using this '5R framework' in the hope that our experience could be useful to other companies tackling R&D productivity issues.  We focus on the evolution of our approach to target validation, hit and lead optimization, pharmacokinetic/pharmacodynamic modeling and drug safety testing, which have helped improve the quality of candidate drug nomination, as well as the development of the right culture, where 'truth seeking' is encouraged by more rigorous and quant. decision-making.  We also discuss where the approach has failed and the lessons learned.  Overall, the continued evolution and application of the 5R framework are beginning to have an impact, with success rates from candidate drug nomination to phase III completion improving from 4% in 2005-2010 to 19% in 2012-2016.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpwdx6OlRZpzrVg90H21EOLACvtfcHk0lhAjo-CgaKpoA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXht1SitL4%253D&md5=06906e0bc59dc715941998660a514ffe</span></div><a href="/servlet/linkout?suffix=cit93&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2017.244&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2017.244%26sid%3Dliteratum%253Aachs%26aulast%3DMorgan%26aufirst%3DP.%26aulast%3DBrown%26aufirst%3DD.%2BG.%26aulast%3DLennard%26aufirst%3DS.%26aulast%3DAnderton%26aufirst%3DM.%2BJ.%26aulast%3DBarrett%26aufirst%3DJ.%2BC.%26aulast%3DEriksson%26aufirst%3DU.%26aulast%3DFidock%26aufirst%3DM.%26aulast%3DHamr%25C3%25A9n%26aufirst%3DB.%26aulast%3DJohnson%26aufirst%3DA.%26aulast%3DMarch%26aufirst%3DR.%2BE.%26aulast%3DMatcham%26aufirst%3DJ.%26aulast%3DMettetal%26aufirst%3DJ.%26aulast%3DNicholls%26aufirst%3DD.%2BJ.%26aulast%3DPlatz%26aufirst%3DS.%26aulast%3DRees%26aufirst%3DS.%26aulast%3DSnowden%26aufirst%3DM.%2BA.%26aulast%3DPangalos%26aufirst%3DM.%2BN.%26atitle%3DImpact%2520of%2520a%2520Five-Dimensional%2520Framework%2520on%2520R%2526D%2520Productivity%2520at%2520AstraZeneca%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2018%26volume%3D17%26spage%3D167%26epage%3D181%26doi%3D10.1038%2Fnrd.2017.244" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref94"><div class="reference"><strong class="refLabel"><a href="#ref94" class="rightTabRefNumLink">94</a></strong><div class="NLM_citation" id="rightTab-cit94"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Moffat, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vincent, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eder, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prunotto, M.</span></span> <span> </span><span class="NLM_article-title">Opportunities and Challenges in Phenotypic Drug Discovery: An Industry Perspective</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">531</span>– <span class="NLM_lpage">543</span>, <span class="refDoi"> DOI: 10.1038/nrd.2017.111</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref94/cit94&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1038%2Fnrd.2017.111" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref94/cit94&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=28685762" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref94/cit94&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtFelsLrJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2017&pages=531-543&author=J.+G.+Moffatauthor=F.+Vincentauthor=J.+A.+Leeauthor=J.+Ederauthor=M.+Prunotto&title=Opportunities+and+Challenges+in+Phenotypic+Drug+Discovery%3A+An+Industry+Perspective&doi=10.1038%2Fnrd.2017.111"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit94R"><div class="casContent"><span class="casTitleNuber">94</span><div class="casTitle"><span class="NLM_cas:atitle">Opportunities and challenges in phenotypic drug discovery: an industry perspective</span></div><div class="casAuthors">Moffat, John G.; Vincent, Fabien; Lee, Jonathan A.; Eder, Jorg; Prunotto, Marco</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">531-543</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Phenotypic drug discovery (PDD) approaches do not rely on knowledge of the identity of a specific drug target or a hypothesis about its role in disease, in contrast to the target-based strategies that have been widely used in the pharmaceutical industry in the past three decades.  However, in recent years, there has been a resurgence in interest in PDD approaches based on their potential to address the incompletely understood complexity of diseases and their promise of delivering first-in-class drugs, as well as major advances in the tools for cell-based phenotypic screening.  Nevertheless, PDD approaches also have considerable challenges, such as hit validation and target deconvolution.  This article focuses on the lessons learned by researchers engaged in PDD in the pharmaceutical industry and considers the impact of 'omics' knowledge in defining a cellular disease phenotype in the era of precision medicine, introducing the concept of a chain of translatability.  We particularly aim to identify features and areas in which PDD can best deliver value to drug discovery portfolios and can contribute to the identification and the development of novel medicines, and to illustrate the challenges and uncertainties that are assocd. with PDD in order to help set realistic expectations with regard to its benefits and costs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpGrD5TsU8_kLVg90H21EOLACvtfcHk0lhAjo-CgaKpoA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtFelsLrJ&md5=d83196715c879a81d9eabcf544e303db</span></div><a href="/servlet/linkout?suffix=cit94&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2017.111&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2017.111%26sid%3Dliteratum%253Aachs%26aulast%3DMoffat%26aufirst%3DJ.%2BG.%26aulast%3DVincent%26aufirst%3DF.%26aulast%3DLee%26aufirst%3DJ.%2BA.%26aulast%3DEder%26aufirst%3DJ.%26aulast%3DPrunotto%26aufirst%3DM.%26atitle%3DOpportunities%2520and%2520Challenges%2520in%2520Phenotypic%2520Drug%2520Discovery%253A%2520An%2520Industry%2520Perspective%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2017%26volume%3D16%26spage%3D531%26epage%3D543%26doi%3D10.1038%2Fnrd.2017.111" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref95"><div class="reference"><strong class="refLabel"><a href="#ref95" class="rightTabRefNumLink">95</a></strong><div class="NLM_citation" id="rightTab-cit95"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Plowright, A. T.</span>; <span class="NLM_string-name">Drowley, L.</span></span> <span> </span><span class="NLM_article-title">Phenotypic Screening</span>. In  <i>Platform Technologies in Drug Discovery and Validation</i>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goodnow, R. A.,  Jr.</span></span>, Ed.; <span class="NLM_publisher-name">Academic Press</span>: <span class="NLM_publisher-loc">Cambridge, MA</span>, <span class="NLM_year">2017</span>; pp  <span class="NLM_fpage">263</span>– <span class="NLM_lpage">299</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref95/cit95&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1016%2Fbs.armc.2017.07.001" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2017&pages=263-299&author=A.+T.+Plowright&author=L.+Drowleyauthor=R.+A.+Goodnow&title=Platform+Technologies+in+Drug+Discovery+and+Validation"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit95&amp;dbid=16384&amp;doi=10.1016%2Fbs.armc.2017.07.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1016%252Fbs.armc.2017.07.001%26sid%3Dliteratum%253Aachs%26aulast%3DPlowright%26aufirst%3DA.%2BT.%26atitle%3DPhenotypic%2520Screening%26btitle%3DPlatform%2520Technologies%2520in%2520Drug%2520Discovery%2520and%2520Validation%26aulast%3DGoodnow%26aufirst%3DR.%2BA.%26pub%3DAcademic%2520Press%26date%3D2017%26spage%3D263%26epage%3D299" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref96"><div class="reference"><strong class="refLabel"><a href="#ref96" class="rightTabRefNumLink">96</a></strong><div class="NLM_citation" id="rightTab-cit96"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lewis, K.</span></span> <span> </span><span class="NLM_article-title">Platforms for Antibiotic Discovery</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">371</span>– <span class="NLM_lpage">387</span>, <span class="refDoi"> DOI: 10.1038/nrd3975</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref96/cit96&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1038%2Fnrd3975" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref96/cit96&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=23629505" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref96/cit96&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC3sXms1Ojt70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2013&pages=371-387&author=K.+Lewis&title=Platforms+for+Antibiotic+Discovery&doi=10.1038%2Fnrd3975"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit96R"><div class="casContent"><span class="casTitleNuber">96</span><div class="casTitle"><span class="NLM_cas:atitle">Platforms for antibiotic discovery</span></div><div class="casAuthors">Lewis, Kim</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">371-387</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The spread of resistant bacteria, leading to untreatable infections, is a major public health threat but the pace of antibiotic discovery to combat these pathogens has slowed down.  Most antibiotics were originally isolated by screening soil-derived actinomycetes during the golden era of antibiotic discovery in the 1940s to 1960s.  However, diminishing returns from this discovery platform led to its collapse, and efforts to create a new platform based on target-focused screening of large libraries of synthetic compds. failed, in part owing to the lack of penetration of such compds. through the bacterial envelope.  This article considers strategies to re-establish viable platforms for antibiotic discovery.  These include investigating untapped natural product sources such as uncultured bacteria, establishing rules of compd. penetration to enable the development of synthetic antibiotics, developing species-specific antibiotics and identifying prodrugs that have the potential to eradicate dormant persisters, which are often responsible for hard-to-treat infections.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqOyTc7KWJVJrVg90H21EOLACvtfcHk0lhAjo-CgaKpoA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXms1Ojt70%253D&md5=4f69d8c4c1a25e952ae8ed7df2631115</span></div><a href="/servlet/linkout?suffix=cit96&amp;dbid=16384&amp;doi=10.1038%2Fnrd3975&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd3975%26sid%3Dliteratum%253Aachs%26aulast%3DLewis%26aufirst%3DK.%26atitle%3DPlatforms%2520for%2520Antibiotic%2520Discovery%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2013%26volume%3D12%26spage%3D371%26epage%3D387%26doi%3D10.1038%2Fnrd3975" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref97"><div class="reference"><strong class="refLabel"><a href="#ref97" class="rightTabRefNumLink">97</a></strong><div class="NLM_citation" id="rightTab-cit97"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ling, L. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schneider, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peoples, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spoering, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Engels, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Conlon, B. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mueller, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schäberle, T. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hughes, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Epstein, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lazarides, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steadman, V. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cohen, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Felix, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fetterman, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Millett, W. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nitti, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zullo, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lewis, K.</span></span> <span> </span><span class="NLM_article-title">A New Antibiotic Kills Pathogens without Detectable Resistance</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>517</i></span>,  <span class="NLM_fpage">455</span>– <span class="NLM_lpage">459</span>, <span class="refDoi"> DOI: 10.1038/nature14098</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref97/cit97&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1038%2Fnature14098" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref97/cit97&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=25561178" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref97/cit97&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtFOju7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=517&publication_year=2015&pages=455-459&author=L.+L.+Lingauthor=T.+Schneiderauthor=A.+J.+Peoplesauthor=A.+L.+Spoeringauthor=I.+Engelsauthor=B.+P.+Conlonauthor=A.+Muellerauthor=T.+F.+Sch%C3%A4berleauthor=D.+E.+Hughesauthor=S.+Epsteinauthor=M.+Jonesauthor=L.+Lazaridesauthor=V.+A.+Steadmanauthor=D.+R.+Cohenauthor=C.+R.+Felixauthor=K.+A.+Fettermanauthor=W.+P.+Millettauthor=A.+G.+Nittiauthor=A.+M.+Zulloauthor=C.+Chenauthor=K.+Lewis&title=A+New+Antibiotic+Kills+Pathogens+without+Detectable+Resistance&doi=10.1038%2Fnature14098"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit97R"><div class="casContent"><span class="casTitleNuber">97</span><div class="casTitle"><span class="NLM_cas:atitle">A new antibiotic kills pathogens without detectable resistance</span></div><div class="casAuthors">Ling, Losee L.; Schneider, Tanja; Peoples, Aaron J.; Spoering, Amy L.; Engels, Ina; Conlon, Brian P.; Mueller, Anna; Schaberle, Till F.; Hughes, Dallas E.; Epstein, Slava; Jones, Michael; Lazarides, Linos; Steadman, Victoria A.; Cohen, Douglas R.; Felix, Cintia R.; Fetterman, K. Ashley; Millett, William P.; Nitti, Anthony G.; Zullo, Ashley M.; Chen, Chao; Lewis, Kim</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">517</span>
        (<span class="NLM_cas:issue">7535</span>),
    <span class="NLM_cas:pages">455-459</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Antibiotic resistance is spreading faster than the introduction of new compds. into clin. practice, causing a public health crisis.  Most antibiotics were produced by screening soil microorganisms, but this limited resource of cultivable bacteria was overmined by the 1960s.  Synthetic approaches to produce antibiotics have been unable to replace this platform.  Uncultured bacteria make up approx. 99% of all species in external environments, and are an untapped source of new antibiotics.  We developed several methods to grow uncultured organisms by cultivation in situ or by using specific growth factors.  Here we report a new antibiotic that we term teixobactin, discovered in a screen of uncultured bacteria.  Teixobactin inhibits cell wall synthesis by binding to a highly conserved motif of lipid II (precursor of peptidoglycan) and lipid III (precursor of cell wall teichoic acid).  We did not obtain any mutants of Staphylococcus aureus or Mycobacterium tuberculosis resistant to teixobactin.  The properties of this compd. suggest a path towards developing antibiotics that are likely to avoid development of resistance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrV0hZrSZheoLVg90H21EOLACvtfcHk0liF0qL5JnsRVw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtFOju7w%253D&md5=27e302d7d44a549a91aa52113c3b4ad8</span></div><a href="/servlet/linkout?suffix=cit97&amp;dbid=16384&amp;doi=10.1038%2Fnature14098&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature14098%26sid%3Dliteratum%253Aachs%26aulast%3DLing%26aufirst%3DL.%2BL.%26aulast%3DSchneider%26aufirst%3DT.%26aulast%3DPeoples%26aufirst%3DA.%2BJ.%26aulast%3DSpoering%26aufirst%3DA.%2BL.%26aulast%3DEngels%26aufirst%3DI.%26aulast%3DConlon%26aufirst%3DB.%2BP.%26aulast%3DMueller%26aufirst%3DA.%26aulast%3DSch%25C3%25A4berle%26aufirst%3DT.%2BF.%26aulast%3DHughes%26aufirst%3DD.%2BE.%26aulast%3DEpstein%26aufirst%3DS.%26aulast%3DJones%26aufirst%3DM.%26aulast%3DLazarides%26aufirst%3DL.%26aulast%3DSteadman%26aufirst%3DV.%2BA.%26aulast%3DCohen%26aufirst%3DD.%2BR.%26aulast%3DFelix%26aufirst%3DC.%2BR.%26aulast%3DFetterman%26aufirst%3DK.%2BA.%26aulast%3DMillett%26aufirst%3DW.%2BP.%26aulast%3DNitti%26aufirst%3DA.%2BG.%26aulast%3DZullo%26aufirst%3DA.%2BM.%26aulast%3DChen%26aufirst%3DC.%26aulast%3DLewis%26aufirst%3DK.%26atitle%3DA%2520New%2520Antibiotic%2520Kills%2520Pathogens%2520without%2520Detectable%2520Resistance%26jtitle%3DNature%26date%3D2015%26volume%3D517%26spage%3D455%26epage%3D459%26doi%3D10.1038%2Fnature14098" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref98"><div class="reference"><strong class="refLabel"><a href="#ref98" class="rightTabRefNumLink">98</a></strong><div class="NLM_citation" id="rightTab-cit98"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Plowright, A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Engkvist, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gill, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knerr, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Q.-D.</span></span> <span> </span><span class="NLM_article-title">Heart Regeneration: Opportunities and Challenges for Drug Discovery with Novel Chemical and Therapeutic Methods or Agents</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">4056</span>– <span class="NLM_lpage">4075</span>, <span class="refDoi"> DOI: 10.1002/anie.201307034</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref98/cit98&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1002%2Fanie.201307034" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2014&pages=4056-4075&author=A.+T.+Plowrightauthor=O.+Engkvistauthor=A.+Gillauthor=L.+Knerrauthor=Q.-D.+Wang&title=Heart+Regeneration%3A+Opportunities+and+Challenges+for+Drug+Discovery+with+Novel+Chemical+and+Therapeutic+Methods+or+Agents&doi=10.1002%2Fanie.201307034"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit98&amp;dbid=16384&amp;doi=10.1002%2Fanie.201307034&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201307034%26sid%3Dliteratum%253Aachs%26aulast%3DPlowright%26aufirst%3DA.%2BT.%26aulast%3DEngkvist%26aufirst%3DO.%26aulast%3DGill%26aufirst%3DA.%26aulast%3DKnerr%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DQ.-D.%26atitle%3DHeart%2520Regeneration%253A%2520Opportunities%2520and%2520Challenges%2520for%2520Drug%2520Discovery%2520with%2520Novel%2520Chemical%2520and%2520Therapeutic%2520Methods%2520or%2520Agents%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2014%26volume%3D53%26spage%3D4056%26epage%3D4075%26doi%3D10.1002%2Fanie.201307034" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref99"><div class="reference"><strong class="refLabel"><a href="#ref99" class="rightTabRefNumLink">99</a></strong><div class="NLM_citation" id="rightTab-cit99"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Avior, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sagi, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Benvenisty, N.</span></span> <span> </span><span class="NLM_article-title">Pluripotent Stem Cells in Disease Modelling and Drug Discovery</span>. <i>Nat. Rev. Mol. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">170</span>– <span class="NLM_lpage">182</span>, <span class="refDoi"> DOI: 10.1038/nrm.2015.27</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref99/cit99&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1038%2Fnrm.2015.27" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref99/cit99&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=26818440" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref99/cit99&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsFygsro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2016&pages=170-182&author=Y.+Aviorauthor=I.+Sagiauthor=N.+Benvenisty&title=Pluripotent+Stem+Cells+in+Disease+Modelling+and+Drug+Discovery&doi=10.1038%2Fnrm.2015.27"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit99R"><div class="casContent"><span class="casTitleNuber">99</span><div class="casTitle"><span class="NLM_cas:atitle">Pluripotent stem cells in disease modelling and drug discovery</span></div><div class="casAuthors">Avior, Yishai; Sagi, Ido; Benvenisty, Nissim</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Molecular Cell Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">170-182</span>CODEN:
                <span class="NLM_cas:coden">NRMCBP</span>;
        ISSN:<span class="NLM_cas:issn">1471-0072</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Exptl. modeling of human disorders enables the definition of the cellular and mol. mechanisms underlying diseases and the development of therapies for treating them.  The availability of human pluripotent stem cells (PSCs), which are capable of self-renewal and have the potential to differentiate into virtually any cell type, can now help to overcome the limitations of animal models for certain disorders.  The ability to model human diseases using cultured PSCs has revolutionized the ways in which we study monogenic, complex and epigenetic disorders, as well as early- and late-onset diseases.  Several strategies are used to generate such disease models using either embryonic stem cells (ES cells) or patient-specific induced PSCs (iPSCs), creating new possibilities for the establishment of models and their use in drug screening.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpHtd9-bDU1BLVg90H21EOLACvtfcHk0liF0qL5JnsRVw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsFygsro%253D&md5=4072946787f6f29809eac02e1b2574e5</span></div><a href="/servlet/linkout?suffix=cit99&amp;dbid=16384&amp;doi=10.1038%2Fnrm.2015.27&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrm.2015.27%26sid%3Dliteratum%253Aachs%26aulast%3DAvior%26aufirst%3DY.%26aulast%3DSagi%26aufirst%3DI.%26aulast%3DBenvenisty%26aufirst%3DN.%26atitle%3DPluripotent%2520Stem%2520Cells%2520in%2520Disease%2520Modelling%2520and%2520Drug%2520Discovery%26jtitle%3DNat.%2520Rev.%2520Mol.%2520Cell%2520Biol.%26date%3D2016%26volume%3D17%26spage%3D170%26epage%3D182%26doi%3D10.1038%2Fnrm.2015.27" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref100"><div class="reference"><strong class="refLabel"><a href="#ref100" class="rightTabRefNumLink">100</a></strong><div class="NLM_citation" id="rightTab-cit100"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mills, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Titmarsh, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koenig, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parker, B. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ryall, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quaife-Ryan, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Voges, H. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hodson, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferguson, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drowley, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plowright, A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Needham, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Q.-D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gregorevic, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xin, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, W. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parton, R. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nielsen, L. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Launikonis, B. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">James, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elliott, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Porrello, E. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hudson, J. E.</span></span> <span> </span><span class="NLM_article-title">Functional Screening in Human Cardiac Organoids Reveals a Metabolic Mechanism for Cardiomyocyte Cell Cycle Arrest</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>114</i></span>,  <span class="NLM_fpage">E8372</span>– <span class="NLM_lpage">E8381</span>, <span class="refDoi"> DOI: 10.1073/pnas.1707316114</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref100/cit100&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1073%2Fpnas.1707316114" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=114&publication_year=2017&pages=E8372-E8381&author=R.+J.+Millsauthor=D.+M.+Titmarshauthor=X.+Koenigauthor=B.+L.+Parkerauthor=J.+G.+Ryallauthor=G.+A.+Quaife-Ryanauthor=H.+K.+Vogesauthor=M.+P.+Hodsonauthor=C.+Fergusonauthor=L.+Drowleyauthor=A.+T.+Plowrightauthor=E.+J.+Needhamauthor=Q.-D.+Wangauthor=P.+Gregorevicauthor=M.+Xinauthor=W.+G.+Thomasauthor=R.+G.+Partonauthor=L.+K.+Nielsenauthor=B.+S.+Launikonisauthor=D.+E.+Jamesauthor=D.+A.+Elliottauthor=E.+R.+Porrelloauthor=J.+E.+Hudson&title=Functional+Screening+in+Human+Cardiac+Organoids+Reveals+a+Metabolic+Mechanism+for+Cardiomyocyte+Cell+Cycle+Arrest&doi=10.1073%2Fpnas.1707316114"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit100&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1707316114&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1707316114%26sid%3Dliteratum%253Aachs%26aulast%3DMills%26aufirst%3DR.%2BJ.%26aulast%3DTitmarsh%26aufirst%3DD.%2BM.%26aulast%3DKoenig%26aufirst%3DX.%26aulast%3DParker%26aufirst%3DB.%2BL.%26aulast%3DRyall%26aufirst%3DJ.%2BG.%26aulast%3DQuaife-Ryan%26aufirst%3DG.%2BA.%26aulast%3DVoges%26aufirst%3DH.%2BK.%26aulast%3DHodson%26aufirst%3DM.%2BP.%26aulast%3DFerguson%26aufirst%3DC.%26aulast%3DDrowley%26aufirst%3DL.%26aulast%3DPlowright%26aufirst%3DA.%2BT.%26aulast%3DNeedham%26aufirst%3DE.%2BJ.%26aulast%3DWang%26aufirst%3DQ.-D.%26aulast%3DGregorevic%26aufirst%3DP.%26aulast%3DXin%26aufirst%3DM.%26aulast%3DThomas%26aufirst%3DW.%2BG.%26aulast%3DParton%26aufirst%3DR.%2BG.%26aulast%3DNielsen%26aufirst%3DL.%2BK.%26aulast%3DLaunikonis%26aufirst%3DB.%2BS.%26aulast%3DJames%26aufirst%3DD.%2BE.%26aulast%3DElliott%26aufirst%3DD.%2BA.%26aulast%3DPorrello%26aufirst%3DE.%2BR.%26aulast%3DHudson%26aufirst%3DJ.%2BE.%26atitle%3DFunctional%2520Screening%2520in%2520Human%2520Cardiac%2520Organoids%2520Reveals%2520a%2520Metabolic%2520Mechanism%2520for%2520Cardiomyocyte%2520Cell%2520Cycle%2520Arrest%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2017%26volume%3D114%26spage%3DE8372%26epage%3DE8381%26doi%3D10.1073%2Fpnas.1707316114" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref101"><div class="reference"><strong class="refLabel"><a href="#ref101" class="rightTabRefNumLink">101</a></strong><div class="NLM_citation" id="rightTab-cit101"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lee, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramirez, C. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeltner, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Radu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhinder, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, Y. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choi, I. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mukherjee-Clavin, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Djaballah, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Studer, L.</span></span> <span> </span><span class="NLM_article-title">Large-Scale Screening Using Familial Dysautonomia Induced Pluripotent Stem Cells Identifies Compounds That Rescue <i>IKBKAP</i> Expression</span>. <i>Nat. Biotechnol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">1244</span>– <span class="NLM_lpage">1248</span>, <span class="refDoi"> DOI: 10.1038/nbt.2435</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref101/cit101&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1038%2Fnbt.2435" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2012&pages=1244-1248&author=G.+Leeauthor=C.+N.+Ramirezauthor=H.+Kimauthor=N.+Zeltnerauthor=B.+Liuauthor=C.+Raduauthor=B.+Bhinderauthor=Y.+J.+Kimauthor=I.+Y.+Choiauthor=B.+Mukherjee-Clavinauthor=H.+Djaballahauthor=L.+Studer&title=Large-Scale+Screening+Using+Familial+Dysautonomia+Induced+Pluripotent+Stem+Cells+Identifies+Compounds+That+Rescue+IKBKAP+Expression&doi=10.1038%2Fnbt.2435"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit101&amp;dbid=16384&amp;doi=10.1038%2Fnbt.2435&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt.2435%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DG.%26aulast%3DRamirez%26aufirst%3DC.%2BN.%26aulast%3DKim%26aufirst%3DH.%26aulast%3DZeltner%26aufirst%3DN.%26aulast%3DLiu%26aufirst%3DB.%26aulast%3DRadu%26aufirst%3DC.%26aulast%3DBhinder%26aufirst%3DB.%26aulast%3DKim%26aufirst%3DY.%2BJ.%26aulast%3DChoi%26aufirst%3DI.%2BY.%26aulast%3DMukherjee-Clavin%26aufirst%3DB.%26aulast%3DDjaballah%26aufirst%3DH.%26aulast%3DStuder%26aufirst%3DL.%26atitle%3DLarge-Scale%2520Screening%2520Using%2520Familial%2520Dysautonomia%2520Induced%2520Pluripotent%2520Stem%2520Cells%2520Identifies%2520Compounds%2520That%2520Rescue%2520IKBKAP%2520Expression%26jtitle%3DNat.%2520Biotechnol.%26date%3D2012%26volume%3D30%26spage%3D1244%26epage%3D1248%26doi%3D10.1038%2Fnbt.2435" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref102"><div class="reference"><strong class="refLabel"><a href="#ref102" class="rightTabRefNumLink">102</a></strong><div class="NLM_citation" id="rightTab-cit102"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jones, L. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bunnage, M. E.</span></span> <span> </span><span class="NLM_article-title">Applications of Chemogenomic Library Screening in Drug Discovery</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">285</span>– <span class="NLM_lpage">296</span>, <span class="refDoi"> DOI: 10.1038/nrd.2016.244</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref102/cit102&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1038%2Fnrd.2016.244" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref102/cit102&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=28104905" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref102/cit102&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC2sXht1Ort7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2017&pages=285-296&author=L.+H.+Jonesauthor=M.+E.+Bunnage&title=Applications+of+Chemogenomic+Library+Screening+in+Drug+Discovery&doi=10.1038%2Fnrd.2016.244"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit102R"><div class="casContent"><span class="casTitleNuber">102</span><div class="casTitle"><span class="NLM_cas:atitle">Applications of chemogenomic library screening in drug discovery</span></div><div class="casAuthors">Jones, Lyn H.; Bunnage, Mark E.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">285-296</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The allure of phenotypic screening, combined with the industry preference for target-based approaches, has prompted the development of innovative chem. biol. technologies that facilitate the identification of new therapeutic targets for accelerated drug discovery.  A chemogenomic library is a collection of selective small-mol. pharmacol. agents, and a hit from such a set in a phenotypic screen suggests that the annotated target or targets of that pharmacol. agent may be involved in perturbing the observable phenotype.  In this Review, we describe opportunities for chemogenomic screening to considerably expedite the conversion of phenotypic screening projects into target-based drug discovery approaches.  Other applications are explored, including drug repositioning, predictive toxicol. and the discovery of novel pharmacol. modalities.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGruetMviqci8rVg90H21EOLACvtfcHk0lg4d_-tsbQPQA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXht1Ort7c%253D&md5=a51a882947528be18392170cfa48c554</span></div><a href="/servlet/linkout?suffix=cit102&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2016.244&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2016.244%26sid%3Dliteratum%253Aachs%26aulast%3DJones%26aufirst%3DL.%2BH.%26aulast%3DBunnage%26aufirst%3DM.%2BE.%26atitle%3DApplications%2520of%2520Chemogenomic%2520Library%2520Screening%2520in%2520Drug%2520Discovery%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2017%26volume%3D16%26spage%3D285%26epage%3D296%26doi%3D10.1038%2Fnrd.2016.244" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref103"><div class="reference"><strong class="refLabel"><a href="#ref103" class="rightTabRefNumLink">103</a></strong><div class="NLM_citation" id="rightTab-cit103"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cornett, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">King, F. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nigsch, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paris, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McAllister, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jenkins, J. L.</span></span> <span> </span><span class="NLM_article-title">Evidence-Based and Quantitative Prioritization of Tool Compounds in Phenotypic Drug Discovery</span>. <i>Cell Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">862</span>– <span class="NLM_lpage">874</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2016.05.016</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref103/cit103&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1016%2Fj.chembiol.2016.05.016" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref103/cit103&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=27427232" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref103/cit103&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtFyiur3E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2016&pages=862-874&author=Y.+Wangauthor=A.+Cornettauthor=F.+J.+Kingauthor=Y.+Maoauthor=F.+Nigschauthor=C.+G.+Parisauthor=G.+McAllisterauthor=J.+L.+Jenkins&title=Evidence-Based+and+Quantitative+Prioritization+of+Tool+Compounds+in+Phenotypic+Drug+Discovery&doi=10.1016%2Fj.chembiol.2016.05.016"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit103R"><div class="casContent"><span class="casTitleNuber">103</span><div class="casTitle"><span class="NLM_cas:atitle">Evidence-Based and Quantitative Prioritization of Tool Compounds in Phenotypic Drug Discovery</span></div><div class="casAuthors">Wang, Yuan; Cornett, Allen; King, Fred J.; Mao, Yi; Nigsch, Florian; Paris, C. Gregory; McAllister, Gregory; Jenkins, Jeremy L.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">862-874</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The use of potent and selective chem. tools with well-defined targets can help elucidate biol. processes driving phenotypes in phenotypic screens.  However, identification of selective compds. en masse to create targeted screening sets is non-trivial.  A systematic approach is needed to prioritize probes, which prevents the repeated use of published but unselective compds.  Here we performed a meta-anal. of integrated large-scale, heterogeneous bioactivity data to create an evidence-based, quant. metric to systematically rank tool compds. for targets.  Our tool score (TS) was then tested on hundreds of compds. by assessing their activity profiles in a panel of 41 cell-based pathway assays.  We demonstrate that high-TS tools show more reliably selective phenotypic profiles than lower-TS compds.  Addnl. we highlight frequently tested compds. that are non-selective tools and distinguish target family polypharmacol. from cross-family promiscuity.  TS can therefore be used to prioritize compds. from heterogeneous databases for phenotypic screening.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGquaPu2t_l4ZbVg90H21EOLACvtfcHk0lg4d_-tsbQPQA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtFyiur3E&md5=b990346556ac032b8cecbfc9455f8458</span></div><a href="/servlet/linkout?suffix=cit103&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2016.05.016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2016.05.016%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DY.%26aulast%3DCornett%26aufirst%3DA.%26aulast%3DKing%26aufirst%3DF.%2BJ.%26aulast%3DMao%26aufirst%3DY.%26aulast%3DNigsch%26aufirst%3DF.%26aulast%3DParis%26aufirst%3DC.%2BG.%26aulast%3DMcAllister%26aufirst%3DG.%26aulast%3DJenkins%26aufirst%3DJ.%2BL.%26atitle%3DEvidence-Based%2520and%2520Quantitative%2520Prioritization%2520of%2520Tool%2520Compounds%2520in%2520Phenotypic%2520Drug%2520Discovery%26jtitle%3DCell%2520Chem.%2520Biol.%26date%3D2016%26volume%3D23%26spage%3D862%26epage%3D874%26doi%3D10.1016%2Fj.chembiol.2016.05.016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref104"><div class="reference"><strong class="refLabel"><a href="#ref104" class="rightTabRefNumLink">104</a></strong><div class="NLM_citation" id="rightTab-cit104"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Santos, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ursu, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gaulton, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bento, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Donadi, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bologa, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karlsson, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Al-Lazikani, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hersey, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oprea, T. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Overington, J. P.</span></span> <span> </span><span class="NLM_article-title">A Comprehensive Map of Molecular Drug Targets</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">19</span>– <span class="NLM_lpage">34</span>, <span class="refDoi"> DOI: 10.1038/nrd.2016.230</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref104/cit104&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1038%2Fnrd.2016.230" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref104/cit104&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=27910877" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref104/cit104&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC28XitVSms7rK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2017&pages=19-34&author=R.+Santosauthor=O.+Ursuauthor=A.+Gaultonauthor=A.+P.+Bentoauthor=R.+S.+Donadiauthor=C.+G.+Bologaauthor=A.+Karlssonauthor=B.+Al-Lazikaniauthor=A.+Herseyauthor=T.+I.+Opreaauthor=J.+P.+Overington&title=A+Comprehensive+Map+of+Molecular+Drug+Targets&doi=10.1038%2Fnrd.2016.230"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit104R"><div class="casContent"><span class="casTitleNuber">104</span><div class="casTitle"><span class="NLM_cas:atitle">A comprehensive map of molecular drug targets</span></div><div class="casAuthors">Santos, Rita; Ursu, Oleg; Gaulton, Anna; Bento, A. Patricia; Donadi, Ramesh S.; Bologa, Cristian G.; Karlsson, Anneli; Al-Lazikani, Bissan; Hersey, Anne; Oprea, Tudor I.; Overington, John P.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">19-34</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The success of mechanism-based drug discovery depends on the definition of the drug target.  This definition becomes even more important as we try to link drug response to genetic variation, understand stratified clin. efficacy and safety, rationalize the differences between drugs in the same therapeutic class and predict drug utility in patient subgroups.  However, drug targets are often poorly defined in the literature, both for launched drugs and for potential therapeutic agents in discovery and development.  Here, we present an updated comprehensive map of mol. targets of approved drugs.  We curate a total of 893 human and pathogen-derived biomols. through which 1,578 US FDA-approved drugs act.  These biomols. include 667 human-genome-derived proteins targeted by drugs for human disease.  Anal. of these drug targets indicates the continued dominance of privileged target families across disease areas, but also the growth of novel first-in-class mechanisms, particularly in oncol.  We explore the relationships between bioactivity class and clin. success, as well as the presence of orthologues between human and animal models and between pathogen and human genomes.  Through the collaboration of three independent teams, we highlight some of the ongoing challenges in accurately defining the targets of mol. therapeutics and present conventions for deconvoluting the complexities of mol. pharmacol. and drug efficacy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpTfWwWkoZnbrVg90H21EOLACvtfcHk0lg4d_-tsbQPQA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitVSms7rK&md5=7437d4b678ac75dac0a5e3248df98761</span></div><a href="/servlet/linkout?suffix=cit104&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2016.230&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2016.230%26sid%3Dliteratum%253Aachs%26aulast%3DSantos%26aufirst%3DR.%26aulast%3DUrsu%26aufirst%3DO.%26aulast%3DGaulton%26aufirst%3DA.%26aulast%3DBento%26aufirst%3DA.%2BP.%26aulast%3DDonadi%26aufirst%3DR.%2BS.%26aulast%3DBologa%26aufirst%3DC.%2BG.%26aulast%3DKarlsson%26aufirst%3DA.%26aulast%3DAl-Lazikani%26aufirst%3DB.%26aulast%3DHersey%26aufirst%3DA.%26aulast%3DOprea%26aufirst%3DT.%2BI.%26aulast%3DOverington%26aufirst%3DJ.%2BP.%26atitle%3DA%2520Comprehensive%2520Map%2520of%2520Molecular%2520Drug%2520Targets%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2017%26volume%3D16%26spage%3D19%26epage%3D34%26doi%3D10.1038%2Fnrd.2016.230" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref105"><div class="reference"><strong class="refLabel"><a href="#ref105" class="rightTabRefNumLink">105</a></strong><div class="NLM_citation" id="rightTab-cit105"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kato, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Comer, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sakata-Kato, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharma, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharma, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maetani, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bastien, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brancucci, N. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bittker, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Corey, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clarke, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Derbyshire, E. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dornan, G. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duffy, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eckley, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Itoe, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koolen, K. M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lewis, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lui, P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lukens, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lund, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">March, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meibalan, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meier, B. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McPhail, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitasev, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moss, E. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sayes, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Gessel, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wawer, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yoshinaga, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeeman, A.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Avery, V. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhatia, S. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burke, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Catteruccia, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clardy, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clemons, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dechering, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duvall, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Foley, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gusovsky, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kocken, C. H. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marti, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morningstar, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Munoz, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neafsey, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharma, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winzeler, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wirth, D. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scherer, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schreiber, S. L.</span></span><span class="NLM_etal"></span> <span> </span><span class="NLM_article-title">Diversity-Oriented Synthesis Yields Novel Multistage Antimalarial Inhibitors</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>538</i></span>,  <span class="NLM_fpage">344</span>– <span class="NLM_lpage">349</span>, <span class="refDoi"> DOI: 10.1038/nature19804</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref105/cit105&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1038%2Fnature19804" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref105/cit105&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=27602946" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref105/cit105&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC28XhslWhsLrI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=538&publication_year=2016&pages=344-349&author=N.+Katoauthor=E.+Comerauthor=T.+Sakata-Katoauthor=A.+Sharmaauthor=M.+Sharmaauthor=M.+Maetaniauthor=J.+Bastienauthor=N.+M.+Brancucciauthor=J.+A.+Bittkerauthor=V.+Coreyauthor=D.+Clarkeauthor=E.+R.+Derbyshireauthor=G.+L.+Dornanauthor=S.+Duffyauthor=S.+Eckleyauthor=M.+A.+Itoeauthor=K.+M.+J.+Koolenauthor=T.+A.+Lewisauthor=P.+S.+Luiauthor=A.+K.+Lukensauthor=E.+Lundauthor=S.+Marchauthor=E.+Meibalanauthor=B.+C.+Meierauthor=J.+A.+McPhailauthor=B.+Mitasevauthor=E.+L.+Mossauthor=M.+Sayesauthor=Y.+Van+Gesselauthor=M.+J.+Wawerauthor=T.+Yoshinagaauthor=A.-M.+Zeemanauthor=V.+M.+Averyauthor=S.+N.+Bhatiaauthor=J.+E.+Burkeauthor=F.+Catterucciaauthor=J.+C.+Clardyauthor=P.+A.+Clemonsauthor=K.+J.+Decheringauthor=J.+R.+Duvallauthor=M.+A.+Foleyauthor=F.+Gusovskyauthor=C.+H.+M.+Kockenauthor=M.+Martiauthor=M.+L.+Morningstarauthor=B.+Munozauthor=D.+E.+Neafseyauthor=A.+Sharmaauthor=E.+A.+Winzelerauthor=D.+F.+Wirthauthor=C.+A.+Schererauthor=S.+L.+Schreiber&title=Diversity-Oriented+Synthesis+Yields+Novel+Multistage+Antimalarial+Inhibitors&doi=10.1038%2Fnature19804"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit105R"><div class="casContent"><span class="casTitleNuber">105</span><div class="casTitle"><span class="NLM_cas:atitle">Diversity-oriented synthesis yields novel multistage antimalarial inhibitors</span></div><div class="casAuthors">Kato, Nobutaka; Comer, Eamon; Sakata-Kato, Tomoyo; Sharma, Arvind; Sharma, Manmohan; Maetani, Micah; Bastien, Jessica; Brancucci, Nicolas M.; Bittker, Joshua A.; Corey, Victoria; Clarke, David; Derbyshire, Emily R.; Dornan, Gillian L.; Duffy, Sandra; Eckley, Sean; Itoe, Maurice A.; Koolen, Karin M. J.; Lewis, Timothy A.; Lui, Ping S.; Lukens, Amanda K.; Lund, Emily; March, Sandra; Meibalan, Elamaran; Meier, Bennett C.; McPhail, Jacob A.; Mitasev, Branko; Moss, Eli L.; Sayes, Morgane; Van Gessel, Yvonne; Wawer, Mathias J.; Yoshinaga, Takashi; Zeeman, Anne-Marie; Avery, Vicky M.; Bhatia, Sangeeta N.; Burke, John E.; Catteruccia, Flaminia; Clardy, Jon C.; Clemons, Paul A.; Dechering, Koen J.; Duvall, Jeremy R.; Foley, Michael A.; Gusovsky, Fabian; Kocken, Clemens H. M.; Marti, Matthias; Morningstar, Marshall L.; Munoz, Benito; Neafsey, Daniel E.; Sharma, Amit; Winzeler, Elizabeth A.; Wirth, Dyann F.; Scherer, Christina A.; Schreiber, Stuart L.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">538</span>
        (<span class="NLM_cas:issue">7625</span>),
    <span class="NLM_cas:pages">344-349</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Antimalarial drugs have thus far been chiefly derived from two sources-natural products and synthetic drug-like compds.  Here the authors investigate whether antimalarial agents with novel mechanisms of action could be discovered using a diverse collection of synthetic compds. that have three-dimensional features reminiscent of natural products and are underrepresented in typical screening collections.  The authors report the identification of such compds. with both previously reported and undescribed mechanisms of action, including a series of bicyclic azetidines that inhibit a new antimalarial target, phenylalanyl-tRNA synthetase.  These mols. are curative in mice at a single, low dose and show activity against all parasite life stages in multiple in vivo efficacy models.  The authors' findings identify bicyclic azetidines with the potential to both cure and prevent transmission of the disease as well as protect at-risk populations with a single oral dose, highlighting the strength of diversity-oriented synthesis in revealing promising therapeutic targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp9CdwKMqzmhLVg90H21EOLACvtfcHk0lg025mJoKkW2w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhslWhsLrI&md5=68e6e2018f2d49278d255ee02bfc8d16</span></div><a href="/servlet/linkout?suffix=cit105&amp;dbid=16384&amp;doi=10.1038%2Fnature19804&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature19804%26sid%3Dliteratum%253Aachs%26aulast%3DKato%26aufirst%3DN.%26aulast%3DComer%26aufirst%3DE.%26aulast%3DSakata-Kato%26aufirst%3DT.%26aulast%3DSharma%26aufirst%3DA.%26aulast%3DSharma%26aufirst%3DM.%26aulast%3DMaetani%26aufirst%3DM.%26aulast%3DBastien%26aufirst%3DJ.%26aulast%3DBrancucci%26aufirst%3DN.%2BM.%26aulast%3DBittker%26aufirst%3DJ.%2BA.%26aulast%3DCorey%26aufirst%3DV.%26aulast%3DClarke%26aufirst%3DD.%26aulast%3DDerbyshire%26aufirst%3DE.%2BR.%26aulast%3DDornan%26aufirst%3DG.%2BL.%26aulast%3DDuffy%26aufirst%3DS.%26aulast%3DEckley%26aufirst%3DS.%26aulast%3DItoe%26aufirst%3DM.%2BA.%26aulast%3DKoolen%26aufirst%3DK.%2BM.%2BJ.%26aulast%3DLewis%26aufirst%3DT.%2BA.%26aulast%3DLui%26aufirst%3DP.%2BS.%26aulast%3DLukens%26aufirst%3DA.%2BK.%26aulast%3DLund%26aufirst%3DE.%26aulast%3DMarch%26aufirst%3DS.%26aulast%3DMeibalan%26aufirst%3DE.%26aulast%3DMeier%26aufirst%3DB.%2BC.%26aulast%3DMcPhail%26aufirst%3DJ.%2BA.%26aulast%3DMitasev%26aufirst%3DB.%26aulast%3DMoss%26aufirst%3DE.%2BL.%26aulast%3DSayes%26aufirst%3DM.%26aulast%3DVan%2BGessel%26aufirst%3DY.%26aulast%3DWawer%26aufirst%3DM.%2BJ.%26aulast%3DYoshinaga%26aufirst%3DT.%26aulast%3DZeeman%26aufirst%3DA.-M.%26aulast%3DAvery%26aufirst%3DV.%2BM.%26aulast%3DBhatia%26aufirst%3DS.%2BN.%26aulast%3DBurke%26aufirst%3DJ.%2BE.%26aulast%3DCatteruccia%26aufirst%3DF.%26aulast%3DClardy%26aufirst%3DJ.%2BC.%26aulast%3DClemons%26aufirst%3DP.%2BA.%26aulast%3DDechering%26aufirst%3DK.%2BJ.%26aulast%3DDuvall%26aufirst%3DJ.%2BR.%26aulast%3DFoley%26aufirst%3DM.%2BA.%26aulast%3DGusovsky%26aufirst%3DF.%26aulast%3DKocken%26aufirst%3DC.%2BH.%2BM.%26aulast%3DMarti%26aufirst%3DM.%26aulast%3DMorningstar%26aufirst%3DM.%2BL.%26aulast%3DMunoz%26aufirst%3DB.%26aulast%3DNeafsey%26aufirst%3DD.%2BE.%26aulast%3DSharma%26aufirst%3DA.%26aulast%3DWinzeler%26aufirst%3DE.%2BA.%26aulast%3DWirth%26aufirst%3DD.%2BF.%26aulast%3DScherer%26aufirst%3DC.%2BA.%26aulast%3DSchreiber%26aufirst%3DS.%2BL.%26atitle%3DDiversity-Oriented%2520Synthesis%2520Yields%2520Novel%2520Multistage%2520Antimalarial%2520Inhibitors%26jtitle%3DNature%26date%3D2016%26volume%3D538%26spage%3D344%26epage%3D349%26doi%3D10.1038%2Fnature19804" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref106"><div class="reference"><strong class="refLabel"><a href="#ref106" class="rightTabRefNumLink">106</a></strong><div class="NLM_citation" id="rightTab-cit106"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Parker, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galmozzi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Correia, B. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sasaki, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Joslyn, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cavallaro, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lawrence, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Narvaiza, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saez, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cravatt, B. F.</span></span> <span> </span><span class="NLM_article-title">Ligand and Target Discovery by Fragment-Based Screening in Human Cells</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>168</i></span>,  <span class="NLM_fpage">527.e29</span>– <span class="NLM_lpage">541.e29</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2016.12.029</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref106/cit106&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1016%2Fj.cell.2016.12.029" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=168&publication_year=2017&pages=527.e29-541.e29&author=C.+G.+Parkerauthor=A.+Galmozziauthor=Y.+Wangauthor=B.+E.+Correiaauthor=K.+Sasakiauthor=C.+M.+Joslynauthor=A.+S.+Kimauthor=C.+L.+Cavallaroauthor=R.+M.+Lawrenceauthor=S.+R.+Johnsonauthor=I.+Narvaizaauthor=E.+Saezauthor=B.+F.+Cravatt&title=Ligand+and+Target+Discovery+by+Fragment-Based+Screening+in+Human+Cells&doi=10.1016%2Fj.cell.2016.12.029"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit106&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2016.12.029&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2016.12.029%26sid%3Dliteratum%253Aachs%26aulast%3DParker%26aufirst%3DC.%2BG.%26aulast%3DGalmozzi%26aufirst%3DA.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DCorreia%26aufirst%3DB.%2BE.%26aulast%3DSasaki%26aufirst%3DK.%26aulast%3DJoslyn%26aufirst%3DC.%2BM.%26aulast%3DKim%26aufirst%3DA.%2BS.%26aulast%3DCavallaro%26aufirst%3DC.%2BL.%26aulast%3DLawrence%26aufirst%3DR.%2BM.%26aulast%3DJohnson%26aufirst%3DS.%2BR.%26aulast%3DNarvaiza%26aufirst%3DI.%26aulast%3DSaez%26aufirst%3DE.%26aulast%3DCravatt%26aufirst%3DB.%2BF.%26atitle%3DLigand%2520and%2520Target%2520Discovery%2520by%2520Fragment-Based%2520Screening%2520in%2520Human%2520Cells%26jtitle%3DCell%26date%3D2017%26volume%3D168%26spage%3D527.e29%26epage%3D541.e29%26doi%3D10.1016%2Fj.cell.2016.12.029" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref107"><div class="reference"><strong class="refLabel"><a href="#ref107" class="rightTabRefNumLink">107</a></strong><div class="NLM_citation" id="rightTab-cit107"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wenlock, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Austin, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barton, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leeson, P. D.</span></span> <span> </span><span class="NLM_article-title">A Comparison of Physiochemical Property Profiles of Development and Marketed Oral Drugs</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>46</i></span>,  <span class="NLM_fpage">1250</span>– <span class="NLM_lpage">1256</span>, <span class="refDoi"> DOI: 10.1021/jm021053p</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm021053p" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref107/cit107&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BD3sXhslSlsbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2003&pages=1250-1256&author=M.+C.+Wenlockauthor=R.+P.+Austinauthor=P.+Bartonauthor=A.+M.+Davisauthor=P.+D.+Leeson&title=A+Comparison+of+Physiochemical+Property+Profiles+of+Development+and+Marketed+Oral+Drugs&doi=10.1021%2Fjm021053p"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit107R"><div class="casContent"><span class="casTitleNuber">107</span><div class="casTitle"><span class="NLM_cas:atitle">A Comparison of Physiochemical Property Profiles of Development and Marketed Oral Drugs</span></div><div class="casAuthors">Wenlock, Mark C.; Austin, Rupert P.; Barton, Patrick; Davis, Andrew M.; Leeson, Paul D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1250-1256</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The process of drug discovery applies rigorous selection pressures.  Marketed oral drugs will generally possess favorable physiochem. properties with respect to absorption, metab., distribution, and clearance.  This paper describes a study in which the distributions of physiochem. properties of oral drugs in different phases of clin. development are compared to those already marketed.  The aim is to identify the trends in physiochem. properties that favor a drug's successful passage through clin. development and on to the market.  Two libraries were created, one of current development oral drugs and one of marketed oral drugs.  Statistical anal. of the two showed that the mean mol. wt. of orally administered drugs in development decreases on passing through each of the different clin. phases and gradually converges toward the mean mol. wt. of marketed oral drugs.  It is also clear that the most lipophilic compds. are being discontinued from development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoF0PksUFJq0bVg90H21EOLACvtfcHk0lg025mJoKkW2w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXhslSlsbY%253D&md5=b9809c803a61ec99a1b4171e97952452</span></div><a href="/servlet/linkout?suffix=cit107&amp;dbid=16384&amp;doi=10.1021%2Fjm021053p&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm021053p%26sid%3Dliteratum%253Aachs%26aulast%3DWenlock%26aufirst%3DM.%2BC.%26aulast%3DAustin%26aufirst%3DR.%2BP.%26aulast%3DBarton%26aufirst%3DP.%26aulast%3DDavis%26aufirst%3DA.%2BM.%26aulast%3DLeeson%26aufirst%3DP.%2BD.%26atitle%3DA%2520Comparison%2520of%2520Physiochemical%2520Property%2520Profiles%2520of%2520Development%2520and%2520Marketed%2520Oral%2520Drugs%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2003%26volume%3D46%26spage%3D1250%26epage%3D1256%26doi%3D10.1021%2Fjm021053p" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref108"><div class="reference"><strong class="refLabel"><a href="#ref108" class="rightTabRefNumLink">108</a></strong><div class="NLM_citation" id="rightTab-cit108"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Leeson, P.
D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">St-Gallay, S. A.</span></span> <span> </span><span class="NLM_article-title">The Influence of the “Organizational Factor” on Compound Quality in Drug Discovery</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">749</span>– <span class="NLM_lpage">765</span>, <span class="refDoi"> DOI: 10.1038/nrd3552</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref108/cit108&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1038%2Fnrd3552" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2011&pages=749-765&author=P.%0AD.+Leesonauthor=S.+A.+St-Gallay&title=The+Influence+of+the+%E2%80%9COrganizational+Factor%E2%80%9D+on+Compound+Quality+in+Drug+Discovery&doi=10.1038%2Fnrd3552"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit108&amp;dbid=16384&amp;doi=10.1038%2Fnrd3552&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd3552%26sid%3Dliteratum%253Aachs%26aulast%3DLeeson%26aufirst%3DP.%2BD.%26aulast%3DSt-Gallay%26aufirst%3DS.%2BA.%26atitle%3DThe%2520Influence%2520of%2520the%2520%25E2%2580%259COrganizational%2520Factor%25E2%2580%259D%2520on%2520Compound%2520Quality%2520in%2520Drug%2520Discovery%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2011%26volume%3D10%26spage%3D749%26epage%3D765%26doi%3D10.1038%2Fnrd3552" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref109"><div class="reference"><strong class="refLabel"><a href="#ref109" class="rightTabRefNumLink">109</a></strong><div class="NLM_citation" id="rightTab-cit109"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cox, P. B.</span>; <span class="NLM_string-name">Vasudevan, A.</span></span> <span> </span><span class="NLM_article-title">Generating a High-Quality Compound Collection</span>. In  <i>Management of Chemical and Biological Samples for Screening Applications</i>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wigglesworth, M.</span>, <span class="NLM_string-name">Wood, T.</span></span>, Eds.; <span class="NLM_publisher-name">Wiley-VCH Verlag GmbH & Co. KGaA</span>: <span class="NLM_publisher-loc">Weinheim, Germany</span>, <span class="NLM_year">2012</span>; pp  <span class="NLM_fpage">9</span>– <span class="NLM_lpage">21</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref109/cit109&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1002%2F9783527645251.ch2" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2012&pages=9-21&author=P.+B.+Cox&author=A.+Vasudevanauthor=M.+Wigglesworth&author=T.+Wood&title=Management+of+Chemical+and+Biological+Samples+for+Screening+Applications"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit109&amp;dbid=16384&amp;doi=10.1002%2F9783527645251.ch2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1002%252F9783527645251.ch2%26sid%3Dliteratum%253Aachs%26aulast%3DCox%26aufirst%3DP.%2BB.%26atitle%3DGenerating%2520a%2520High-Quality%2520Compound%2520Collection%26btitle%3DManagement%2520of%2520Chemical%2520and%2520Biological%2520Samples%2520for%2520Screening%2520Applications%26aulast%3DWigglesworth%26aufirst%3DM.%26pub%3DWiley-VCH%2520Verlag%2520GmbH%2520%2526%2520Co.%2520KGaA%26date%3D2012%26spage%3D9%26epage%3D21" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref110"><div class="reference"><strong class="refLabel"><a href="#ref110" class="rightTabRefNumLink">110</a></strong><div class="NLM_citation" id="rightTab-cit110"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rankovic, Z.</span>; <span class="NLM_string-name">Jamieson, C.</span>; <span class="NLM_string-name">Morphy, R.</span></span> <span> </span><span class="NLM_article-title">High Throughput Screening Approach to Lead Discovery</span>. In  <i>Lead Generation Approaches in Drug Discovery</i>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rankovic, Z.</span>, <span class="NLM_string-name">Morphy, R.</span></span>, Eds.; <span class="NLM_publisher-name">John Wiley & Sons, Inc.</span>: <span class="NLM_publisher-loc">Hoboken, NJ</span>., <span class="NLM_year">2010</span>; pp  <span class="NLM_fpage">21</span>– <span class="NLM_lpage">71</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref110/cit110&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1002%2F9780470584170.ch2" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2010&pages=21-71&author=Z.+Rankovic&author=C.+Jamieson&author=R.+Morphyauthor=Z.+Rankovic&author=R.+Morphy&title=Lead+Generation+Approaches+in+Drug+Discovery"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit110&amp;dbid=16384&amp;doi=10.1002%2F9780470584170.ch2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1002%252F9780470584170.ch2%26sid%3Dliteratum%253Aachs%26aulast%3DRankovic%26aufirst%3DZ.%26atitle%3DHigh%2520Throughput%2520Screening%2520Approach%2520to%2520Lead%2520Discovery%26btitle%3DLead%2520Generation%2520Approaches%2520in%2520Drug%2520Discovery%26aulast%3DRankovic%26aufirst%3DZ.%26pub%3DJohn%2520Wiley%2520%2526%2520Sons%252C%2520Inc%26date%3D2010%26spage%3D21%26epage%3D71" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref111"><div class="reference"><strong class="refLabel"><a href="#ref111" class="rightTabRefNumLink">111</a></strong><div class="NLM_citation" id="rightTab-cit111"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kalliokoski, T.</span></span> <span> </span><span class="NLM_article-title">Price-Focused Analysis of Commercially Available Building Blocks for Combinatorial Library Synthesis</span>. <i>ACS Comb. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">600</span>– <span class="NLM_lpage">607</span>, <span class="refDoi"> DOI: 10.1021/acscombsci.5b00063</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acscombsci.5b00063" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref111/cit111&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsV2mtLrK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2015&pages=600-607&author=T.+Kalliokoski&title=Price-Focused+Analysis+of+Commercially+Available+Building+Blocks+for+Combinatorial+Library+Synthesis&doi=10.1021%2Facscombsci.5b00063"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit111R"><div class="casContent"><span class="casTitleNuber">111</span><div class="casTitle"><span class="NLM_cas:atitle">Price-Focused Analysis of Commercially Available Building Blocks for Combinatorial Library Synthesis</span></div><div class="casAuthors">Kalliokoski, Tuomo</div><div class="citationInfo"><span class="NLM_cas:title">ACS Combinatorial Science</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">600-607</span>CODEN:
                <span class="NLM_cas:coden">ACSCCC</span>;
        ISSN:<span class="NLM_cas:issn">2156-8944</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Combinatorial libraries are synthesized by combining smaller reagents (building blocks), the price of which is an important component of the total costs assocd. with the synthetic exercise.  A significant portion of com. available reagents are too expensive for large scale work.  In this study, 13 commonly used reagent classes in combinatorial library synthesis (primary and secondary amines, carboxylic acids, alcs., ketones, aldehydes, boronic acids, acyl halides, sulfonyl chlorides, isocyanates, isothiocyanates, azides and chloroformates) were analyzed with respect to the cost, physicochem. properties (mol. wt. and calcd. logP), chem. diversity, and 3D-likeness using a large data set.  The results define the chem. space accessible under a constraint of limited financial resources.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrlW1WcWLiiE7Vg90H21EOLACvtfcHk0lg7C6xIS0VBAQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsV2mtLrK&md5=a7f8c2a12a14ecb81a228c478c09ef06</span></div><a href="/servlet/linkout?suffix=cit111&amp;dbid=16384&amp;doi=10.1021%2Facscombsci.5b00063&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facscombsci.5b00063%26sid%3Dliteratum%253Aachs%26aulast%3DKalliokoski%26aufirst%3DT.%26atitle%3DPrice-Focused%2520Analysis%2520of%2520Commercially%2520Available%2520Building%2520Blocks%2520for%2520Combinatorial%2520Library%2520Synthesis%26jtitle%3DACS%2520Comb.%2520Sci.%26date%3D2015%26volume%3D17%26spage%3D600%26epage%3D607%26doi%3D10.1021%2Facscombsci.5b00063" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref112"><div class="reference"><strong class="refLabel"><a href="#ref112" class="rightTabRefNumLink">112</a></strong><div class="NLM_citation" id="rightTab-cit112"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Stocks, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilden, G. R. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pairaudeau, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perry, M. W. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steele, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stonehouse, J. P.</span></span> <span> </span><span class="NLM_article-title">A Practical Method for Targeted Library Design Balancing Lead-like Properties with Diversity</span>. <i>ChemMedChem</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">800</span>– <span class="NLM_lpage">808</span>, <span class="refDoi"> DOI: 10.1002/cmdc.200900050</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref112/cit112&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1002%2Fcmdc.200900050" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2009&pages=800-808&author=M.+J.+Stocksauthor=G.+R.+H.+Wildenauthor=G.+Pairaudeauauthor=M.+W.+D.+Perryauthor=J.+Steeleauthor=J.+P.+Stonehouse&title=A+Practical+Method+for+Targeted+Library+Design+Balancing+Lead-like+Properties+with+Diversity&doi=10.1002%2Fcmdc.200900050"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit112&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.200900050&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.200900050%26sid%3Dliteratum%253Aachs%26aulast%3DStocks%26aufirst%3DM.%2BJ.%26aulast%3DWilden%26aufirst%3DG.%2BR.%2BH.%26aulast%3DPairaudeau%26aufirst%3DG.%26aulast%3DPerry%26aufirst%3DM.%2BW.%2BD.%26aulast%3DSteele%26aufirst%3DJ.%26aulast%3DStonehouse%26aufirst%3DJ.%2BP.%26atitle%3DA%2520Practical%2520Method%2520for%2520Targeted%2520Library%2520Design%2520Balancing%2520Lead-like%2520Properties%2520with%2520Diversity%26jtitle%3DChemMedChem%26date%3D2009%26volume%3D4%26spage%3D800%26epage%3D808%26doi%3D10.1002%2Fcmdc.200900050" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref113"><div class="reference"><strong class="refLabel"><a href="#ref113" class="rightTabRefNumLink">113</a></strong><div class="NLM_citation" id="rightTab-cit113"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kallus, C.</span>; <span class="NLM_string-name">Hüser, J.</span>; <span class="NLM_string-name">Jones, P.
S.</span>; <span class="NLM_string-name">Nelson, A.</span></span> <span> </span><span class="NLM_article-title">The European Lead Factory</span>. In  <i>Small Molecule Medicinal Chemistry</i>; <span class="NLM_contrib-group"><span class="NLM_string-name">Czechtizky, W.</span>, <span class="NLM_string-name">Hamley, P.</span></span>, Eds.; <span class="NLM_publisher-name">John Wiley & Sons, Inc</span>: <span class="NLM_publisher-loc">Hoboken, NJ</span>., <span class="NLM_year">2015</span>; pp  <span class="NLM_fpage">37</span>– <span class="NLM_lpage">60</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref113/cit113&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1002%2F9781118771723.ch2" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2015&pages=37-60&author=C.+Kallus&author=J.+H%C3%BCser&author=P.%0AS.+Jones&author=A.+Nelsonauthor=W.+Czechtizky&author=P.+Hamley&title=Small+Molecule+Medicinal+Chemistry"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit113&amp;dbid=16384&amp;doi=10.1002%2F9781118771723.ch2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1002%252F9781118771723.ch2%26sid%3Dliteratum%253Aachs%26aulast%3DKallus%26aufirst%3DC.%26atitle%3DThe%2520European%2520Lead%2520Factory%26btitle%3DSmall%2520Molecule%2520Medicinal%2520Chemistry%26aulast%3DCzechtizky%26aufirst%3DW.%26pub%3DJohn%2520Wiley%2520%2526%2520Sons%252C%2520Inc%26date%3D2015%26spage%3D37%26epage%3D60" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref114"><div class="reference"><strong class="refLabel"><a href="#ref114" class="rightTabRefNumLink">114</a></strong><div class="NLM_citation" id="rightTab-cit114"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Karawajczyk, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giordanetto, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Benningshof, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamza, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kalliokoski, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pouwer, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morgentin, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nelson, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Piechot, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tzalis, D.</span></span> <span> </span><span class="NLM_article-title">Expansion of Chemical Space for Collaborative Lead Generation and Drug Discovery: The European Lead Factory Perspective. Drug Discov</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">1310</span>– <span class="NLM_lpage">1316</span>, <span class="refDoi"> DOI: 10.1016/j.drudis.2015.09.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref114/cit114&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1016%2Fj.drudis.2015.09.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref114/cit114&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=26429298" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref114/cit114&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhs1aktLnM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2015&pages=1310-1316&author=A.+Karawajczykauthor=F.+Giordanettoauthor=J.+Benningshofauthor=D.+Hamzaauthor=T.+Kalliokoskiauthor=K.+Pouwerauthor=R.+Morgentinauthor=A.+Nelsonauthor=G.+M%C3%BCllerauthor=A.+Piechotauthor=D.+Tzalis&title=Expansion+of+Chemical+Space+for+Collaborative+Lead+Generation+and+Drug+Discovery%3A+The+European+Lead+Factory+Perspective.+Drug+Discov&doi=10.1016%2Fj.drudis.2015.09.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit114R"><div class="casContent"><span class="casTitleNuber">114</span><div class="casTitle"><span class="NLM_cas:atitle">Expansion of chemical space for collaborative lead generation and drug discovery: the European Lead Factory Perspective</span></div><div class="casAuthors">Karawajczyk, Anna; Giordanetto, Fabrizio; Benningshof, Jorg; Hamza, Daniel; Kalliokoski, Tuomo; Pouwer, Kees; Morgentin, Remy; Nelson, Adam; Mueller, Gerhard; Piechot, Alexander; Tzalis, Dimitrios</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1310-1316</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">High-throughput screening (HTS) represents a major cornerstone of drug discovery.  The availability of an innovative, relevant and high-quality compd. collection to be screened often dictates the final fate of a drug discovery campaign.  Given that the chem. space to be sampled in research programs is practically infinite and sparsely populated, significant efforts and resources need to be invested in the generation and maintenance of a competitive compd. collection.  The European Lead Factory (ELF) project is addressing this challenge by leveraging the diverse experience and know-how of academic groups and small and medium enterprises (SMEs) engaged in synthetic and/or medicinal chem.  Here, we describe the novelty, diversity, structural complexity, physicochem. characteristics and overall attractiveness of this first batch of ELF compds. for HTS purposes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpkcY_bnknKv7Vg90H21EOLACvtfcHk0lg7C6xIS0VBAQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhs1aktLnM&md5=7a3bc7e49c3c9ff7d92c238295a62143</span></div><a href="/servlet/linkout?suffix=cit114&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2015.09.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2015.09.009%26sid%3Dliteratum%253Aachs%26aulast%3DKarawajczyk%26aufirst%3DA.%26aulast%3DGiordanetto%26aufirst%3DF.%26aulast%3DBenningshof%26aufirst%3DJ.%26aulast%3DHamza%26aufirst%3DD.%26aulast%3DKalliokoski%26aufirst%3DT.%26aulast%3DPouwer%26aufirst%3DK.%26aulast%3DMorgentin%26aufirst%3DR.%26aulast%3DNelson%26aufirst%3DA.%26aulast%3DM%25C3%25BCller%26aufirst%3DG.%26aulast%3DPiechot%26aufirst%3DA.%26aulast%3DTzalis%26aufirst%3DD.%26atitle%3DExpansion%2520of%2520Chemical%2520Space%2520for%2520Collaborative%2520Lead%2520Generation%2520and%2520Drug%2520Discovery%253A%2520The%2520European%2520Lead%2520Factory%2520Perspective.%2520Drug%2520Discov%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2015%26volume%3D20%26spage%3D1310%26epage%3D1316%26doi%3D10.1016%2Fj.drudis.2015.09.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref115"><div class="reference"><strong class="refLabel"><a href="#ref115" class="rightTabRefNumLink">115</a></strong><div class="NLM_citation" id="rightTab-cit115"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Medina-Franco, J. L.</span></span> <span> </span><span class="NLM_article-title">Chemoinformatic Characterization of the Chemical Space and Molecular Diversity of Compound Libraries</span>. In  <i>Diversity-Oriented Synthesis</i>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trabocchi, A.</span></span>, Ed.; <span class="NLM_publisher-name">John Wiley & Sons, Inc.</span>: <span class="NLM_publisher-loc">Hoboken, NJ</span>, <span class="NLM_year">2013</span>; pp  <span class="NLM_fpage">325</span>– <span class="NLM_lpage">352</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref115/cit115&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1002%2F9781118618110.ch10" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2013&pages=325-352&author=J.+L.+Medina-Francoauthor=A.+Trabocchi&title=Diversity-Oriented+Synthesis"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit115&amp;dbid=16384&amp;doi=10.1002%2F9781118618110.ch10&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1002%252F9781118618110.ch10%26sid%3Dliteratum%253Aachs%26aulast%3DMedina-Franco%26aufirst%3DJ.%2BL.%26atitle%3DChemoinformatic%2520Characterization%2520of%2520the%2520Chemical%2520Space%2520and%2520Molecular%2520Diversity%2520of%2520Compound%2520Libraries%26btitle%3DDiversity-Oriented%2520Synthesis%26aulast%3DTrabocchi%26aufirst%3DA.%26pub%3DJohn%2520Wiley%2520%2526%2520Sons%252C%2520Inc%26date%3D2013%26spage%3D325%26epage%3D352" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref116"><div class="reference"><strong class="refLabel"><a href="#ref116" class="rightTabRefNumLink">116</a></strong><div class="NLM_citation" id="rightTab-cit116"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mullin, R.</span></span> <span> </span><span class="NLM_article-title">AstraZeneca, Sanofi Exchange Compounds</span>. <i>Chem. Eng. News</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>93</i></span>,  <span class="NLM_fpage">4</span>, <span class="refDoi"> DOI: 10.1021/cen-09347-notw3</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cen-09347-notw3" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=93&publication_year=2015&pages=4&author=R.+Mullin&title=AstraZeneca%2C+Sanofi+Exchange+Compounds&doi=10.1021%2Fcen-09347-notw3"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit116&amp;dbid=16384&amp;doi=10.1021%2Fcen-09347-notw3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcen-09347-notw3%26sid%3Dliteratum%253Aachs%26aulast%3DMullin%26aufirst%3DR.%26atitle%3DAstraZeneca%252C%2520Sanofi%2520Exchange%2520Compounds%26jtitle%3DChem.%2520Eng.%2520News%26date%3D2015%26volume%3D93%26spage%3D4%26doi%3D10.1021%2Fcen-09347-notw3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref117"><div class="reference"><strong class="refLabel"><a href="#ref117" class="rightTabRefNumLink">117</a></strong><div class="NLM_citation" id="rightTab-cit117"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ten Holte, P.</span></span> <span> </span><span class="NLM_article-title">Access to Compound Collections</span>. In  <i>Small Molecule Medicinal Chemistry</i>; <span class="NLM_contrib-group"><span class="NLM_string-name">Czechtizky, W.</span>, <span class="NLM_string-name">Hamley, P.</span></span>, Eds.; <span class="NLM_publisher-name">John Wiley & Sons, Inc</span>: <span class="NLM_publisher-loc">Hoboken, NJ</span>, <span class="NLM_year">2015</span>; pp  <span class="NLM_fpage">61</span>– <span class="NLM_lpage">76</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref117/cit117&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1002%2F9781118771723.ch3" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2015&pages=61-76&author=P.+Ten+Holteauthor=W.+Czechtizky&author=P.+Hamley&title=Small+Molecule+Medicinal+Chemistry"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit117&amp;dbid=16384&amp;doi=10.1002%2F9781118771723.ch3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1002%252F9781118771723.ch3%26sid%3Dliteratum%253Aachs%26aulast%3DTen%2BHolte%26aufirst%3DP.%26atitle%3DAccess%2520to%2520Compound%2520Collections%26btitle%3DSmall%2520Molecule%2520Medicinal%2520Chemistry%26aulast%3DCzechtizky%26aufirst%3DW.%26pub%3DJohn%2520Wiley%2520%2526%2520Sons%252C%2520Inc%26date%3D2015%26spage%3D61%26epage%3D76" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref118"><div class="reference"><strong class="refLabel"><a href="#ref118" class="rightTabRefNumLink">118</a></strong><div class="NLM_citation" id="rightTab-cit118"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kogej, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blomberg, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greasley, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mundt, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vainio, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schamberger, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmidt, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hüser, J.</span></span> <span> </span><span class="NLM_article-title">Big Pharma Screening Collections: More of the Same or Unique Libraries? The AstraZeneca–Bayer Pharma AG Case. Drug Discov</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">1014</span>– <span class="NLM_lpage">1024</span>, <span class="refDoi"> DOI: 10.1016/j.drudis.2012.10.011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref118/cit118&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1016%2Fj.drudis.2012.10.011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref118/cit118&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=23127858" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref118/cit118&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A280%3ADC%252BC3s7htVeqtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2013&pages=1014-1024&author=T.+Kogejauthor=N.+Blombergauthor=P.+J.+Greasleyauthor=S.+Mundtauthor=M.+J.+Vainioauthor=J.+Schambergerauthor=G.+Schmidtauthor=J.+H%C3%BCser&title=Big+Pharma+Screening+Collections%3A+More+of+the+Same+or+Unique+Libraries%3F+The+AstraZeneca%E2%80%93Bayer+Pharma+AG+Case.+Drug+Discov&doi=10.1016%2Fj.drudis.2012.10.011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit118R"><div class="casContent"><span class="casTitleNuber">118</span><div class="casTitle"><span class="NLM_cas:atitle">Big pharma screening collections: more of the same or unique libraries? The AstraZeneca-Bayer Pharma AG case</span></div><div class="casAuthors">Kogej Thierry; Blomberg Niklas; Greasley Peter J; Mundt Stefan; Vainio Mikko J; Schamberger Jens; Schmidt Georg; Huser Jorg</div><div class="citationInfo"><span class="NLM_cas:title">Drug discovery today</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">19-20</span>),
    <span class="NLM_cas:pages">1014-24</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">In this study, the screening collections of two major pharmaceutical companies (AstraZeneca and Bayer Pharma AG) have been compared using a 2D molecular fingerprint by a nearest neighborhood approach.  Results revealed a low overlap between both collections in terms of compound identity and similarity.  This emphasizes the value of screening multiple compound collections to expand the chemical space that can be accessed by high-throughput screening (HTS).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQJl8Dg2rrY8XO4gDb7JggCfW6udTcc2eYvBxti7RTRFbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3s7htVeqtw%253D%253D&md5=86eb921c384773e2a9a6a2017cbef5a1</span></div><a href="/servlet/linkout?suffix=cit118&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2012.10.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2012.10.011%26sid%3Dliteratum%253Aachs%26aulast%3DKogej%26aufirst%3DT.%26aulast%3DBlomberg%26aufirst%3DN.%26aulast%3DGreasley%26aufirst%3DP.%2BJ.%26aulast%3DMundt%26aufirst%3DS.%26aulast%3DVainio%26aufirst%3DM.%2BJ.%26aulast%3DSchamberger%26aufirst%3DJ.%26aulast%3DSchmidt%26aufirst%3DG.%26aulast%3DH%25C3%25BCser%26aufirst%3DJ.%26atitle%3DBig%2520Pharma%2520Screening%2520Collections%253A%2520More%2520of%2520the%2520Same%2520or%2520Unique%2520Libraries%253F%2520The%2520AstraZeneca%25E2%2580%2593Bayer%2520Pharma%2520AG%2520Case.%2520Drug%2520Discov%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2013%26volume%3D18%26spage%3D1014%26epage%3D1024%26doi%3D10.1016%2Fj.drudis.2012.10.011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref119"><div class="reference"><strong class="refLabel"><a href="#ref119" class="rightTabRefNumLink">119</a></strong><div class="NLM_citation" id="rightTab-cit119"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ramsay, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kwasnoski, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caldwell, G. W.</span></span> <span> </span><span class="NLM_article-title">Small Molecule Compound Logistics Outsourcing–Going beyond the “Thought Experiment.</span>. <i>Curr. Top. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">1298</span>– <span class="NLM_lpage">1308</span>, <span class="refDoi"> DOI: 10.2174/156802612800672826</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref119/cit119&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.2174%2F156802612800672826" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref119/cit119&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=22571792" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref119/cit119&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC38XovFers74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2012&pages=1298-1308&author=D.+L.+Ramsayauthor=J.+D.+Kwasnoskiauthor=G.+W.+Caldwell&title=Small+Molecule+Compound+Logistics+Outsourcing%E2%80%93Going+beyond+the+%E2%80%9CThought+Experiment.&doi=10.2174%2F156802612800672826"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit119R"><div class="casContent"><span class="casTitleNuber">119</span><div class="casTitle"><span class="NLM_cas:atitle">Small molecule compound logistics outsourcing - going beyond the "thought experiment"</span></div><div class="casAuthors">Ramsay, Devon L.; Kwasnoski, Joseph D.; Caldwell, Gary W.</div><div class="citationInfo"><span class="NLM_cas:title">Current Topics in Medicinal Chemistry (Sharjah, United Arab Emirates)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1298-1308</span>CODEN:
                <span class="NLM_cas:coden">CTMCCL</span>;
        ISSN:<span class="NLM_cas:issn">1568-0266</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Increasing pressure on the pharmaceutical industry to reduce cost and focus internal resources on "high value" activities is driving a trend to outsource traditionally "inhouse" drug discovery activities.  Compd. collections are typically viewed as drug discovery's "crown jewels"; however, in late 2007, Johnson & Johnson Pharmaceutical Research & Development (J&J PRD) took a bold step to move their entire North American compd. inventory and processing capability to an external third party vendor.  The authors discuss the combination model implemented, that of local compd. logistics site support with an outsourced centralized processing center.  Some of the lessons learned over the past five years were predictable while others were unexpected.  The substantial cost savings, improved local service response and flexible platform to adjust to changing business needs resulted.  Continued sustainable success relies heavily upon maintaining internal headcount dedicated to vendor management, an open collaboration approach and a solid information technol. infrastructure with complete transparency and visibility.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpB941ZIlLXkbVg90H21EOLACvtfcHk0lgLE33frd3GYQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XovFers74%253D&md5=b69c0ba5d07fa83ad1ad1a897a1f13a2</span></div><a href="/servlet/linkout?suffix=cit119&amp;dbid=16384&amp;doi=10.2174%2F156802612800672826&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F156802612800672826%26sid%3Dliteratum%253Aachs%26aulast%3DRamsay%26aufirst%3DD.%2BL.%26aulast%3DKwasnoski%26aufirst%3DJ.%2BD.%26aulast%3DCaldwell%26aufirst%3DG.%2BW.%26atitle%3DSmall%2520Molecule%2520Compound%2520Logistics%2520Outsourcing%25E2%2580%2593Going%2520beyond%2520the%2520%25E2%2580%259CThought%2520Experiment.%26jtitle%3DCurr.%2520Top.%2520Med.%2520Chem.%26date%3D2012%26volume%3D12%26spage%3D1298%26epage%3D1308%26doi%3D10.2174%2F156802612800672826" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref120"><div class="reference"><strong class="refLabel"><a href="#ref120" class="rightTabRefNumLink">120</a></strong><div class="NLM_citation" id="rightTab-cit120"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mahuteau-Betzer, F.</span></span> <span> </span><span class="NLM_article-title">The French National Compound Library: advances and future prospects</span>. <i>Med. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>31</i></span>,  <span class="NLM_fpage">417</span>– <span class="NLM_lpage">422</span>, <span class="refDoi"> DOI: 10.1051/medsci/20153104016</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref120/cit120&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1051%2Fmedsci%2F20153104016" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref120/cit120&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A280%3ADC%252BC2Mfis1Cjuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2015&pages=417-422&author=F.+Mahuteau-Betzer&title=The+French+National+Compound+Library%3A+advances+and+future+prospects&doi=10.1051%2Fmedsci%2F20153104016"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit120R"><div class="casContent"><span class="casTitleNuber">120</span><div class="casTitle"><span class="NLM_cas:atitle">The French National Compound Library: advances and future prospects</span></div><div class="casAuthors">Mahuteau-Betzer Florence</div><div class="citationInfo"><span class="NLM_cas:title">Medecine sciences : M/S</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">417-22</span>
        ISSN:<span class="NLM_cas:issn">0767-0974</span>.
    </div><div class="casAbstract">The French National Compound Library (Chimiotheque Nationale) has been created in 2003 and is the federation of local collections.  It contains more than 56 000 small molecules and natural compounds synthesised or isolated in different laboratories over the past years.  This explains the diversity of the collection.  The strength of this initiative is the ability to connect chemists and biologists for the development of hits.  This development involves the synthesis of analogues or/and chemical tools to find new targets.  These collaborations lead to the identification of new chemical probes.  These probes able to modulate a biological function are essential to study biological pathways.  They can also be useful for therapeutic applications.  This article will describe the major achievements and perspectives of the French Chemical Library.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTjzt2Wqi1XWxgd2Wuil24tfW6udTcc2eYvBxti7RTRFbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2Mfis1Cjuw%253D%253D&md5=db4eaa72173176a4393c72ad8316365e</span></div><a href="/servlet/linkout?suffix=cit120&amp;dbid=16384&amp;doi=10.1051%2Fmedsci%2F20153104016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1051%252Fmedsci%252F20153104016%26sid%3Dliteratum%253Aachs%26aulast%3DMahuteau-Betzer%26aufirst%3DF.%26atitle%3DThe%2520French%2520National%2520Compound%2520Library%253A%2520advances%2520and%2520future%2520prospects%26jtitle%3DMed.%2520Sci.%26date%3D2015%26volume%3D31%26spage%3D417%26epage%3D422%26doi%3D10.1051%2Fmedsci%2F20153104016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref121"><div class="reference"><strong class="refLabel"><a href="#ref121" class="rightTabRefNumLink">121</a></strong><div class="NLM_citation" id="rightTab-cit121"><span> <i>Sanofi Signe un Partenariat avec le CNRS et Vingt-Six
Membres de
la Chimiothèque Nationale [Sanofi Signs a Partnerships with
CNRS and Twenty-Six Members of the French Nationale Compound Bank]</i>; <span class="NLM_publisher-name">Chimiotheque Nationale</span>, October
6, <span class="NLM_year">2015</span>; <a href="http://chimiotheque-nationale.cn.cnrs.fr" class="extLink">http://chimiotheque-nationale.cn.cnrs.fr</a> (accessed February 16, 2018].</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+Sanofi+Signe+un+Partenariat+avec+le+CNRS+et+Vingt-Six%0AMembres+de%0Ala+Chimioth%C3%A8que+Nationale+%5BSanofi+Signs+a+Partnerships+with%0ACNRS+and+Twenty-Six+Members+of+the+French+Nationale+Compound+Bank%5D%3B+Chimiotheque+Nationale%2C+October%0A6%2C+2015%3B+http%3A%2F%2Fchimiotheque-nationale.cn.cnrs.fr+%28accessed+February+16%2C+2018%5D."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit121&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26jtitle%3DSanofi%2520Signe%2520un%2520Partenariat%2520avec%2520le%2520CNRS%2520et%2520Vingt-Six%250AMembres%2520de%250Ala%2520Chimioth%25C3%25A8que%2520Nationale%2520%255BSanofi%2520Signs%2520a%2520Partnerships%2520with%250ACNRS%2520and%2520Twenty-Six%2520Members%2520of%2520the%2520French%2520Nationale%2520Compound%2520Bank%255D%26pub%3DChimiotheque%2520Nationale%26date%3D2015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref122"><div class="reference"><strong class="refLabel"><a href="#ref122" class="rightTabRefNumLink">122</a></strong><div class="NLM_citation" id="rightTab-cit122"><span> <i>Seeking Partners
for Macrocycle Drug Discovery (Challenge ID: 9933642)</i>; <span class="NLM_publisher-name">Innocentive</span>, February 5, <span class="NLM_year">2015</span>; <a href="https://www.innocentive.com/ar/challenge/9933642" class="extLink">https://www.innocentive.com/ar/challenge/9933642</a> (accessed February 16, 2018).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+Seeking+Partners%0Afor+Macrocycle+Drug+Discovery+%28Challenge+ID%3A+9933642%29%3B+Innocentive%2C+February+5%2C+2015%3B+https%3A%2F%2Fwww.innocentive.com%2Far%2Fchallenge%2F9933642+%28accessed+February+16%2C+2018%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit122&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26jtitle%3DSeeking%2520Partners%250Afor%2520Macrocycle%2520Drug%2520Discovery%2520%2528Challenge%2520ID%253A%25209933642%2529%26pub%3DInnocentive%26date%3D2015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref123"><div class="reference"><strong class="refLabel"><a href="#ref123" class="rightTabRefNumLink">123</a></strong><div class="NLM_citation" id="rightTab-cit123"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Price, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Howard, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cons, B. D.</span></span> <span> </span><span class="NLM_article-title">Fragment-Based Drug Discovery and Its Application to Challenging Drug Targets</span>. <i>Essays Biochem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">475</span>– <span class="NLM_lpage">484</span>, <span class="refDoi"> DOI: 10.1042/EBC20170029</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref123/cit123&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1042%2FEBC20170029" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref123/cit123&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=29118094" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref123/cit123&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A280%3ADC%252BC1M7pt1GmtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2017&pages=475-484&author=A.+J.+Priceauthor=S.+Howardauthor=B.+D.+Cons&title=Fragment-Based+Drug+Discovery+and+Its+Application+to+Challenging+Drug+Targets&doi=10.1042%2FEBC20170029"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit123R"><div class="casContent"><span class="casTitleNuber">123</span><div class="casTitle"><span class="NLM_cas:atitle">Fragment-based drug discovery and its application to challenging drug targets</span></div><div class="casAuthors">Price Amanda J; Howard Steven; Cons Benjamin D</div><div class="citationInfo"><span class="NLM_cas:title">Essays in biochemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">475-484</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Fragment-based drug discovery (FBDD) is a technique for identifying low molecular weight chemical starting points for drug discovery.  Since its inception 20 years ago, FBDD has grown in popularity to the point where it is now an established technique in industry and academia.  The approach involves the biophysical screening of proteins against collections of low molecular weight compounds (fragments).  Although fragments bind to proteins with relatively low affinity, they form efficient, high quality binding interactions with the protein architecture as they have to overcome a significant entropy barrier to bind.  Of the biophysical methods available for fragment screening, X-ray protein crystallography is one of the most sensitive and least prone to false positives.  It also provides detailed structural information of the protein-fragment complex at the atomic level.  Fragment-based screening using X-ray crystallography is therefore an efficient method for identifying binding hotspots on proteins, which can then be exploited by chemists and biologists for the discovery of new drugs.  The use of FBDD is illustrated here with a recently published case study of a drug discovery programme targeting the challenging protein-protein interaction Kelch-like ECH-associated protein 1:nuclear factor erythroid 2-related factor 2.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTqZGQ3bfUY0VHFYivNCdPRfW6udTcc2ebe7yeye9sTC7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1M7pt1GmtA%253D%253D&md5=f8572516bfb1ecc4a0ef57b0c5ba1bc0</span></div><a href="/servlet/linkout?suffix=cit123&amp;dbid=16384&amp;doi=10.1042%2FEBC20170029&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252FEBC20170029%26sid%3Dliteratum%253Aachs%26aulast%3DPrice%26aufirst%3DA.%2BJ.%26aulast%3DHoward%26aufirst%3DS.%26aulast%3DCons%26aufirst%3DB.%2BD.%26atitle%3DFragment-Based%2520Drug%2520Discovery%2520and%2520Its%2520Application%2520to%2520Challenging%2520Drug%2520Targets%26jtitle%3DEssays%2520Biochem.%26date%3D2017%26volume%3D61%26spage%3D475%26epage%3D484%26doi%3D10.1042%2FEBC20170029" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref124"><div class="reference"><strong class="refLabel"><a href="#ref124" class="rightTabRefNumLink">124</a></strong><div class="NLM_citation" id="rightTab-cit124"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lamoree, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hubbard, R. E.</span></span> <span> </span><span class="NLM_article-title">Current Perspectives in Fragment-Based Lead Discovery (FBLD)</span>. <i>Essays Biochem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">453</span>– <span class="NLM_lpage">464</span>, <span class="refDoi"> DOI: 10.1042/EBC20170028</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref124/cit124&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1042%2FEBC20170028" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref124/cit124&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=29118093" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref124/cit124&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A280%3ADC%252BC1M7pt1Gmtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2017&pages=453-464&author=B.+Lamoreeauthor=R.+E.+Hubbard&title=Current+Perspectives+in+Fragment-Based+Lead+Discovery+%28FBLD%29&doi=10.1042%2FEBC20170028"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit124R"><div class="casContent"><span class="casTitleNuber">124</span><div class="casTitle"><span class="NLM_cas:atitle">Current perspectives in fragment-based lead discovery (FBLD)</span></div><div class="casAuthors">Lamoree Bas; Hubbard Roderick E; Hubbard Roderick E</div><div class="citationInfo"><span class="NLM_cas:title">Essays in biochemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">453-464</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">It is over 20 years since the first fragment-based discovery projects were disclosed.  The methods are now mature for most 'conventional' targets in drug discovery such as enzymes (kinases and proteases) but there has also been growing success on more challenging targets, such as disruption of protein-protein interactions.  The main application is to identify tractable chemical startpoints that non-covalently modulate the activity of a biological molecule.  In this essay, we overview current practice in the methods and discuss how they have had an impact in lead discovery - generating a large number of fragment-derived compounds that are in clinical trials and two medicines treating patients.  In addition, we discuss some of the more recent applications of the methods in chemical biology - providing chemical tools to investigate biological molecules, mechanisms and systems.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTqZGQ3bfUY0f6BottYdIAWfW6udTcc2ebe7yeye9sTC7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1M7pt1Gmtw%253D%253D&md5=35bce3c5978329c6737ffd719135d363</span></div><a href="/servlet/linkout?suffix=cit124&amp;dbid=16384&amp;doi=10.1042%2FEBC20170028&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252FEBC20170028%26sid%3Dliteratum%253Aachs%26aulast%3DLamoree%26aufirst%3DB.%26aulast%3DHubbard%26aufirst%3DR.%2BE.%26atitle%3DCurrent%2520Perspectives%2520in%2520Fragment-Based%2520Lead%2520Discovery%2520%2528FBLD%2529%26jtitle%3DEssays%2520Biochem.%26date%3D2017%26volume%3D61%26spage%3D453%26epage%3D464%26doi%3D10.1042%2FEBC20170028" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref125"><div class="reference"><strong class="refLabel"><a href="#ref125" class="rightTabRefNumLink">125</a></strong><div class="NLM_citation" id="rightTab-cit125"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jochim, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arora, P. S.</span></span> <span> </span><span class="NLM_article-title">Assessment of Helical Interfaces in Protein–protein Interactions</span>. <i>Mol. BioSyst.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">924</span>– <span class="NLM_lpage">926</span>, <span class="refDoi"> DOI: 10.1039/b903202a</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref125/cit125&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1039%2Fb903202a" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref125/cit125&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=19668855" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref125/cit125&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BD1MXps1KksLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2009&pages=924-926&author=A.+L.+Jochimauthor=P.+S.+Arora&title=Assessment+of+Helical+Interfaces+in+Protein%E2%80%93protein+Interactions&doi=10.1039%2Fb903202a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit125R"><div class="casContent"><span class="casTitleNuber">125</span><div class="casTitle"><span class="NLM_cas:atitle">Assessment of helical interfaces in protein-protein interactions</span></div><div class="casAuthors">Jochim, Andrea L.; Arora, Paramjit S.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular BioSystems</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">924-926</span>CODEN:
                <span class="NLM_cas:coden">MBOIBW</span>;
        ISSN:<span class="NLM_cas:issn">1742-206X</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Herein we identify and analyze helical protein interfaces as potential targets for synthetic modulators of protein-protein interactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGptv4ZRquFYk7Vg90H21EOLACvtfcHk0liDVrB5L2y7tQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXps1KksLw%253D&md5=18d83b22339213b36dbe7df2d0d5cd91</span></div><a href="/servlet/linkout?suffix=cit125&amp;dbid=16384&amp;doi=10.1039%2Fb903202a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fb903202a%26sid%3Dliteratum%253Aachs%26aulast%3DJochim%26aufirst%3DA.%2BL.%26aulast%3DArora%26aufirst%3DP.%2BS.%26atitle%3DAssessment%2520of%2520Helical%2520Interfaces%2520in%2520Protein%25E2%2580%2593protein%2520Interactions%26jtitle%3DMol.%2520BioSyst.%26date%3D2009%26volume%3D5%26spage%3D924%26epage%3D926%26doi%3D10.1039%2Fb903202a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref126"><div class="reference"><strong class="refLabel"><a href="#ref126" class="rightTabRefNumLink">126</a></strong><div class="NLM_citation" id="rightTab-cit126"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Watkins, A.
M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wuo, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arora, P. S.</span></span> <span> </span><span class="NLM_article-title">Protein–Protein Interactions Mediated by Helical Tertiary Structure Motifs</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>137</i></span>,  <span class="NLM_fpage">11622</span>– <span class="NLM_lpage">11630</span>, <span class="refDoi"> DOI: 10.1021/jacs.5b05527</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.5b05527" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref126/cit126&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtlOqu7rK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=137&publication_year=2015&pages=11622-11630&author=A.%0AM.+Watkinsauthor=M.+G.+Wuoauthor=P.+S.+Arora&title=Protein%E2%80%93Protein+Interactions+Mediated+by+Helical+Tertiary+Structure+Motifs&doi=10.1021%2Fjacs.5b05527"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit126R"><div class="casContent"><span class="casTitleNuber">126</span><div class="casTitle"><span class="NLM_cas:atitle">Protein-protein interactions mediated by helical tertiary structure motifs</span></div><div class="casAuthors">Watkins, Andrew M.; Wuo, Michael G.; Arora, Paramjit S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">137</span>
        (<span class="NLM_cas:issue">36</span>),
    <span class="NLM_cas:pages">11622-11630</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The modulation of protein-protein interactions (PPIs) by means of creating or stabilizing secondary structure conformations is a rapidly growing area of research.  Recent success in the inhibition of difficult PPIs by secondary structure mimetics also points to potential limitations, because often, specific cases require tertiary structure mimetics.  To streamline protein structure-based inhibitor design, the authors have previously described the examn. of protein complexes in the Protein Data Bank where α-helixes or β-strands form crit. contacts.  Here, the authors examd. coiled-coils and helix bundles that mediate complex formation to create a platform for the discovery of potential tertiary structure mimetics.  Although there has been extensive anal. of coiled-coil motifs, the interactions between pre-formed coiled-coils and globular proteins have not been systematically analyzed.  Here, the authors identified crit. features of these helical interfaces with respect to coiled-coil and other helical PPIs.  The authors expect the anal. to prove useful for the rational design of modulators of this fundamental class of protein assemblies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrhNUkUL-2YLbVg90H21EOLACvtfcHk0lhQ27m-49QzCA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtlOqu7rK&md5=440d2e7bda4a4806e0d79721fe899304</span></div><a href="/servlet/linkout?suffix=cit126&amp;dbid=16384&amp;doi=10.1021%2Fjacs.5b05527&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.5b05527%26sid%3Dliteratum%253Aachs%26aulast%3DWatkins%26aufirst%3DA.%2BM.%26aulast%3DWuo%26aufirst%3DM.%2BG.%26aulast%3DArora%26aufirst%3DP.%2BS.%26atitle%3DProtein%25E2%2580%2593Protein%2520Interactions%2520Mediated%2520by%2520Helical%2520Tertiary%2520Structure%2520Motifs%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2015%26volume%3D137%26spage%3D11622%26epage%3D11630%26doi%3D10.1021%2Fjacs.5b05527" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref127"><div class="reference"><strong class="refLabel"><a href="#ref127" class="rightTabRefNumLink">127</a></strong><div class="NLM_citation" id="rightTab-cit127"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Checco, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gellman, S. H.</span></span> <span> </span><span class="NLM_article-title">Iterative Nonproteinogenic Residue Incorporation Yields Α/β-Peptides with a Helix–Loop–Helix Tertiary Structure and High Affinity for VEGF</span>. <i>ChemBioChem</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">291</span>– <span class="NLM_lpage">299</span>, <span class="refDoi"> DOI: 10.1002/cbic.201600545</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref127/cit127&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1002%2Fcbic.201600545" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref127/cit127&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=27897370" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref127/cit127&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC2sXltVanuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2017&pages=291-299&author=J.+W.+Checcoauthor=S.+H.+Gellman&title=Iterative+Nonproteinogenic+Residue+Incorporation+Yields+%CE%91%2F%CE%B2-Peptides+with+a+Helix%E2%80%93Loop%E2%80%93Helix+Tertiary+Structure+and+High+Affinity+for+VEGF&doi=10.1002%2Fcbic.201600545"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit127R"><div class="casContent"><span class="casTitleNuber">127</span><div class="casTitle"><span class="NLM_cas:atitle">Iterative Nonproteinogenic Residue Incorporation Yields α/β-Peptides with a Helix-Loop-Helix Tertiary Structure and High Affinity for VEGF</span></div><div class="casAuthors">Checco, James W.; Gellman, Samuel H.</div><div class="citationInfo"><span class="NLM_cas:title">ChemBioChem</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">291-299</span>CODEN:
                <span class="NLM_cas:coden">CBCHFX</span>;
        ISSN:<span class="NLM_cas:issn">1439-4227</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Inhibition of specific protein-protein interactions is attractive for a range of therapeutic applications, but the large and irregularly shaped contact surfaces involved in many such interactions make it challenging to design synthetic antagonists.  Here, we describe the development of backbone-modified peptides contg. both α- and β-amino acid residues (α/β-peptides) that target the receptor-binding surface of vascular endothelial growth factor (VEGF).  Our approach is based on the Z-domain, which adopts a three-helix bundle tertiary structure.  We show how a two-helix "mini-Z-domain" can be modified to contain β and other nonproteinogenic residues while retaining the target-binding epitope by using iterative unnatural residue incorporation.  The resulting α/β-peptides are less susceptible to proteolysis than is their parent α-peptide, and some of these α/β-peptides match the full-length Z-domain in terms of affinity for receptor-recognition surfaces on the VEGF homodimer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGozhiwqlXcFTrVg90H21EOLACvtfcHk0lhQ27m-49QzCA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXltVanuw%253D%253D&md5=6c653a8760340c0410d9f598be076a22</span></div><a href="/servlet/linkout?suffix=cit127&amp;dbid=16384&amp;doi=10.1002%2Fcbic.201600545&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcbic.201600545%26sid%3Dliteratum%253Aachs%26aulast%3DChecco%26aufirst%3DJ.%2BW.%26aulast%3DGellman%26aufirst%3DS.%2BH.%26atitle%3DIterative%2520Nonproteinogenic%2520Residue%2520Incorporation%2520Yields%2520%25CE%2591%252F%25CE%25B2-Peptides%2520with%2520a%2520Helix%25E2%2580%2593Loop%25E2%2580%2593Helix%2520Tertiary%2520Structure%2520and%2520High%2520Affinity%2520for%2520VEGF%26jtitle%3DChemBioChem%26date%3D2017%26volume%3D18%26spage%3D291%26epage%3D299%26doi%3D10.1002%2Fcbic.201600545" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref128"><div class="reference"><strong class="refLabel"><a href="#ref128" class="rightTabRefNumLink">128</a></strong><div class="NLM_citation" id="rightTab-cit128"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wuo, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mahon, A. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arora, P. S.</span></span> <span> </span><span class="NLM_article-title">An Effective Strategy for Stabilizing Minimal Coiled Coil Mimetics</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>137</i></span>,  <span class="NLM_fpage">11618</span>– <span class="NLM_lpage">11621</span>, <span class="refDoi"> DOI: 10.1021/jacs.5b05525</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.5b05525" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref128/cit128&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsVKit7fI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=137&publication_year=2015&pages=11618-11621&author=M.+G.+Wuoauthor=A.+B.+Mahonauthor=P.+S.+Arora&title=An+Effective+Strategy+for+Stabilizing+Minimal+Coiled+Coil+Mimetics&doi=10.1021%2Fjacs.5b05525"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit128R"><div class="casContent"><span class="casTitleNuber">128</span><div class="casTitle"><span class="NLM_cas:atitle">An Effective Strategy for Stabilizing Minimal Coiled Coil Mimetics</span></div><div class="casAuthors">Wuo, Michael G.; Mahon, Andrew B.; Arora, Paramjit S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">137</span>
        (<span class="NLM_cas:issue">36</span>),
    <span class="NLM_cas:pages">11618-11621</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Coiled coils are a major motif in proteins and orchestrate multimerization of various complexes important for biol. processes.  Inhibition of coiled coil-mediated interactions has significant biomedical potential.  However, general approaches that afford short peptides with defined coiled coil conformation remain elusive.  The authors evaluated several strategies to stabilize minimal helical bundles, with the dimer motif as the initial focus.  A stable dimeric scaffold was realized in a synthetic sequence by replacing an interhelical ionic bond with a covalent bond.  Application of this strategy to a more challenging native protein-protein interaction (PPI) suggested that an addnl. constraint, a disulfide bond at the internal a/d' position along with a linker at the e/e' position, is required for enhanced conformational stability.  The authors anticipate the coiled coil stabilization methodol. described herein to yield new classes of modulators for PPIs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqb0FEdmteRRLVg90H21EOLACvtfcHk0lhQ27m-49QzCA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsVKit7fI&md5=8efd6f5dd2576c989ed1542b650ec454</span></div><a href="/servlet/linkout?suffix=cit128&amp;dbid=16384&amp;doi=10.1021%2Fjacs.5b05525&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.5b05525%26sid%3Dliteratum%253Aachs%26aulast%3DWuo%26aufirst%3DM.%2BG.%26aulast%3DMahon%26aufirst%3DA.%2BB.%26aulast%3DArora%26aufirst%3DP.%2BS.%26atitle%3DAn%2520Effective%2520Strategy%2520for%2520Stabilizing%2520Minimal%2520Coiled%2520Coil%2520Mimetics%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2015%26volume%3D137%26spage%3D11618%26epage%3D11621%26doi%3D10.1021%2Fjacs.5b05525" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref129"><div class="reference"><strong class="refLabel"><a href="#ref129" class="rightTabRefNumLink">129</a></strong><div class="NLM_citation" id="rightTab-cit129"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Roschger, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neukirchen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elsässer, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schubert, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maeding, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verwanger, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krammer, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cabrele, C.</span></span> <span> </span><span class="NLM_article-title">Targeting of a Helix-Loop-Helix Transcriptional Regulator by a Short Helical Peptide</span>. <i>ChemMedChem</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">1497</span>– <span class="NLM_lpage">1503</span>, <span class="refDoi"> DOI: 10.1002/cmdc.201700305</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref129/cit129&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1002%2Fcmdc.201700305" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2017&pages=1497-1503&author=C.+Roschgerauthor=S.+Neukirchenauthor=B.+Els%C3%A4sserauthor=M.+Schubertauthor=N.+Maedingauthor=T.+Verwangerauthor=B.+Krammerauthor=C.+Cabrele&title=Targeting+of+a+Helix-Loop-Helix+Transcriptional+Regulator+by+a+Short+Helical+Peptide&doi=10.1002%2Fcmdc.201700305"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit129&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.201700305&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.201700305%26sid%3Dliteratum%253Aachs%26aulast%3DRoschger%26aufirst%3DC.%26aulast%3DNeukirchen%26aufirst%3DS.%26aulast%3DEls%25C3%25A4sser%26aufirst%3DB.%26aulast%3DSchubert%26aufirst%3DM.%26aulast%3DMaeding%26aufirst%3DN.%26aulast%3DVerwanger%26aufirst%3DT.%26aulast%3DKrammer%26aufirst%3DB.%26aulast%3DCabrele%26aufirst%3DC.%26atitle%3DTargeting%2520of%2520a%2520Helix-Loop-Helix%2520Transcriptional%2520Regulator%2520by%2520a%2520Short%2520Helical%2520Peptide%26jtitle%3DChemMedChem%26date%3D2017%26volume%3D12%26spage%3D1497%26epage%3D1503%26doi%3D10.1002%2Fcmdc.201700305" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref130"><div class="reference"><strong class="refLabel"><a href="#ref130" class="rightTabRefNumLink">130</a></strong><div class="NLM_citation" id="rightTab-cit130"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jochim, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arora, P. S.</span></span> <span> </span><span class="NLM_article-title">Systematic Analysis of Helical Protein Interfaces Reveals Targets for Synthetic Inhibitors</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">919</span>– <span class="NLM_lpage">923</span>, <span class="refDoi"> DOI: 10.1021/cb1001747</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb1001747" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref130/cit130&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtVKrsLnO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2010&pages=919-923&author=A.+L.+Jochimauthor=P.+S.+Arora&title=Systematic+Analysis+of+Helical+Protein+Interfaces+Reveals+Targets+for+Synthetic+Inhibitors&doi=10.1021%2Fcb1001747"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit130R"><div class="casContent"><span class="casTitleNuber">130</span><div class="casTitle"><span class="NLM_cas:atitle">Systematic Analysis of Helical Protein Interfaces Reveals Targets for Synthetic Inhibitors</span></div><div class="casAuthors">Jochim, Andrea L.; Arora, Paramjit S.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">919-923</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Synthetic inhibitors of protein-protein interactions are being discovered despite the inherent challenge in targeting large contact surfaces with small mols.  An anal. of available examples identifies common features of complexes that make them tractable for small mols.  We deduced that relative disposition and energetic contributions of "hot spot" residues provide a predictive scale for the potential of protein-protein interactions to be inhibited by small mols.  On the basis of this model, we analyzed the full set of helical protein interfaces in the Protein Data Bank to identify those that are potentially suitable candidates for synthetic ligands.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr3SMHxSvMd7LVg90H21EOLACvtfcHk0lh8RfhO93q3DQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtVKrsLnO&md5=ea2f564abefa82573fb2320fd70e9fe0</span></div><a href="/servlet/linkout?suffix=cit130&amp;dbid=16384&amp;doi=10.1021%2Fcb1001747&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb1001747%26sid%3Dliteratum%253Aachs%26aulast%3DJochim%26aufirst%3DA.%2BL.%26aulast%3DArora%26aufirst%3DP.%2BS.%26atitle%3DSystematic%2520Analysis%2520of%2520Helical%2520Protein%2520Interfaces%2520Reveals%2520Targets%2520for%2520Synthetic%2520Inhibitors%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2010%26volume%3D5%26spage%3D919%26epage%3D923%26doi%3D10.1021%2Fcb1001747" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref131"><div class="reference"><strong class="refLabel"><a href="#ref131" class="rightTabRefNumLink">131</a></strong><div class="NLM_citation" id="rightTab-cit131"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Moellering, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cornejo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, T. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bianco, C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aster, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blacklow, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kung, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilliland, D. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verdine, G. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradner, J. E.</span></span> <span> </span><span class="NLM_article-title">Direct Inhibition of the NOTCH Transcription Factor Complex</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>462</i></span>,  <span class="NLM_fpage">182</span>– <span class="NLM_lpage">188</span>, <span class="refDoi"> DOI: 10.1038/nature08543</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref131/cit131&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1038%2Fnature08543" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref131/cit131&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=19907488" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref131/cit131&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsVSjsr3F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=462&publication_year=2009&pages=182-188&author=R.+E.+Moelleringauthor=M.+Cornejoauthor=T.+N.+Davisauthor=C.+D.+Biancoauthor=J.+C.+Asterauthor=S.+C.+Blacklowauthor=A.+L.+Kungauthor=D.+G.+Gillilandauthor=G.+L.+Verdineauthor=J.+E.+Bradner&title=Direct+Inhibition+of+the+NOTCH+Transcription+Factor+Complex&doi=10.1038%2Fnature08543"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit131R"><div class="casContent"><span class="casTitleNuber">131</span><div class="casTitle"><span class="NLM_cas:atitle">Direct inhibition of the NOTCH transcription factor complex</span></div><div class="casAuthors">Moellering, Raymond E.; Cornejo, Melanie; Davis, Tina N.; Del Bianco, Cristina; Aster, Jon C.; Blacklow, Stephen C.; Kung, Andrew L.; Gilliland, D. Gary; Verdine, Gregory L.; Bradner, James E.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">462</span>
        (<span class="NLM_cas:issue">7270</span>),
    <span class="NLM_cas:pages">182-188</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Direct inhibition of transcription factor complexes remains a central challenge in the discipline of ligand discovery.  In general, these proteins lack surface involutions suitable for high-affinity binding by small mols.  Here the authors report the design of synthetic, cell-permeable, stabilized α-helical peptides that target a crit. protein-protein interface in the NOTCH transactivation complex.  The authors demonstrate that direct, high-affinity binding of the hydrocarbon-stapled peptide SAHM1 prevents assembly of the active transcriptional complex.  Inappropriate NOTCH activation is directly implicated in the pathogenesis of several disease states, including T-cell acute lymphoblastic leukemia (T-ALL).  The treatment of leukemic cells with SAHM1 results in genome-wide suppression of NOTCH-activated genes.  Direct antagonism of the NOTCH transcriptional program causes potent, NOTCH-specific anti-proliferative effects in cultured cells and in a mouse model of NOTCH1-driven T-ALL.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq_hyuFJHX0T7Vg90H21EOLACvtfcHk0lh8RfhO93q3DQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsVSjsr3F&md5=a12f19dd03303fd054e3052d7bbffd47</span></div><a href="/servlet/linkout?suffix=cit131&amp;dbid=16384&amp;doi=10.1038%2Fnature08543&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature08543%26sid%3Dliteratum%253Aachs%26aulast%3DMoellering%26aufirst%3DR.%2BE.%26aulast%3DCornejo%26aufirst%3DM.%26aulast%3DDavis%26aufirst%3DT.%2BN.%26aulast%3DBianco%26aufirst%3DC.%2BD.%26aulast%3DAster%26aufirst%3DJ.%2BC.%26aulast%3DBlacklow%26aufirst%3DS.%2BC.%26aulast%3DKung%26aufirst%3DA.%2BL.%26aulast%3DGilliland%26aufirst%3DD.%2BG.%26aulast%3DVerdine%26aufirst%3DG.%2BL.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26atitle%3DDirect%2520Inhibition%2520of%2520the%2520NOTCH%2520Transcription%2520Factor%2520Complex%26jtitle%3DNature%26date%3D2009%26volume%3D462%26spage%3D182%26epage%3D188%26doi%3D10.1038%2Fnature08543" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref132"><div class="reference"><strong class="refLabel"><a href="#ref132" class="rightTabRefNumLink">132</a></strong><div class="NLM_citation" id="rightTab-cit132"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Grossmann, T. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yeh, J. T.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bowman, B. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chu, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moellering, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verdine, G. L.</span></span> <span> </span><span class="NLM_article-title">Inhibition of Oncogenic Wnt Signaling through Direct Targeting of Β-Catenin</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>109</i></span>,  <span class="NLM_fpage">17942</span>– <span class="NLM_lpage">17947</span>, <span class="refDoi"> DOI: 10.1073/pnas.1208396109</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref132/cit132&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1073%2Fpnas.1208396109" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref132/cit132&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=23071338" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref132/cit132&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC38Xhsl2ktbvI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=109&publication_year=2012&pages=17942-17947&author=T.+N.+Grossmannauthor=J.+T.-H.+Yehauthor=B.+R.+Bowmanauthor=Q.+Chuauthor=R.+E.+Moelleringauthor=G.+L.+Verdine&title=Inhibition+of+Oncogenic+Wnt+Signaling+through+Direct+Targeting+of+%CE%92-Catenin&doi=10.1073%2Fpnas.1208396109"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit132R"><div class="casContent"><span class="casTitleNuber">132</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of oncogenic Wnt signaling through direct targeting of β-catenin</span></div><div class="casAuthors">Grossmann, Tom N.; Yeh, Johannes T.-H.; Bowman, Brian R.; Chu, Qian; Moellering, Raymond E.; Verdine, Gregory L.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">109</span>
        (<span class="NLM_cas:issue">44</span>),
    <span class="NLM_cas:pages">17942-17947, S17942/1-S17942/10</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Aberrant activation of signaling by the Wnt pathway is strongly implicated in the onset and progression of numerous types of cancer.  Owing to the persistent dependence of these tumors on Wnt signaling for growth and survival, inhibition of this pathway is considered an attractive mechanism-based therapeutic approach.  Oncogenic activation of Wnt signaling can ensue from a variety of distinct aberrations in the signaling pathway, but most share the common feature of causing increased cellular levels of β-catenin by interfering with its constitutive degrdn.  β-Catenin serves as a central hub in Wnt signaling by engaging in crucial protein-protein interactions with both neg. and pos. effectors of the pathway.  Direct interference with these protein-protein interactions is a biol. compelling approach toward suppression of β-catenin hyperactivity, but such interactions have proven intransigent with respect to small-mol. targeting.  Hence β-catenin remains an elusive target for translational cancer therapy.  Here the authors report the discovery of a hydrocarbon-stapled peptide that directly targets β-catenin and interferes with its ability to serve as a transcriptional coactivator for T-cell factor (TCF) proteins, the downstream transcriptional regulators of the Wnt pathway.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqD9Q6Y8-IAI7Vg90H21EOLACvtfcHk0lh8RfhO93q3DQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xhsl2ktbvI&md5=3f2fc0f4f616ef057317639fe7c465c6</span></div><a href="/servlet/linkout?suffix=cit132&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1208396109&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1208396109%26sid%3Dliteratum%253Aachs%26aulast%3DGrossmann%26aufirst%3DT.%2BN.%26aulast%3DYeh%26aufirst%3DJ.%2BT.-H.%26aulast%3DBowman%26aufirst%3DB.%2BR.%26aulast%3DChu%26aufirst%3DQ.%26aulast%3DMoellering%26aufirst%3DR.%2BE.%26aulast%3DVerdine%26aufirst%3DG.%2BL.%26atitle%3DInhibition%2520of%2520Oncogenic%2520Wnt%2520Signaling%2520through%2520Direct%2520Targeting%2520of%2520%25CE%2592-Catenin%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2012%26volume%3D109%26spage%3D17942%26epage%3D17947%26doi%3D10.1073%2Fpnas.1208396109" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref133"><div class="reference"><strong class="refLabel"><a href="#ref133" class="rightTabRefNumLink">133</a></strong><div class="NLM_citation" id="rightTab-cit133"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lau, Y. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rossmann, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, B. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Andrade, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, Y. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verma, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McKenzie, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Venkitaraman, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hyvönen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spring, D. R.</span></span> <span> </span><span class="NLM_article-title">Double Strain-Promoted Macrocyclization for the Rapid Selection of Cell-Active Stapled Peptides</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">15410</span>– <span class="NLM_lpage">15413</span>, <span class="refDoi"> DOI: 10.1002/anie.201508416</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref133/cit133&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1002%2Fanie.201508416" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref133/cit133&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhslClsrbE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2015&pages=15410-15413&author=Y.+H.+Lauauthor=Y.+Wuauthor=M.+Rossmannauthor=B.+X.+Tanauthor=P.+de+Andradeauthor=Y.+S.+Tanauthor=C.+Vermaauthor=G.+J.+McKenzieauthor=A.+R.+Venkitaramanauthor=M.+Hyv%C3%B6nenauthor=D.+R.+Spring&title=Double+Strain-Promoted+Macrocyclization+for+the+Rapid+Selection+of+Cell-Active+Stapled+Peptides&doi=10.1002%2Fanie.201508416"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit133R"><div class="casContent"><span class="casTitleNuber">133</span><div class="casTitle"><span class="NLM_cas:atitle">Double Strain-Promoted Macrocyclization for the Rapid Selection of Cell-Active Stapled Peptides</span></div><div class="casAuthors">Lau, Yu Heng; Wu, Yuteng; Rossmann, Maxim; Tan, Ban Xiong; de Andrade, Peterson; Tan, Yaw Sing; Verma, Chandra; McKenzie, Grahame J.; Venkitaraman, Ashok R.; Hyvoenen, Marko; Spring, David R.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">51</span>),
    <span class="NLM_cas:pages">15410-15413</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Peptide stapling is a method for designing macrocyclic alpha-helical inhibitors of protein-protein interactions.  However, obtaining a cell-active inhibitor can require significant optimization.  The authors report a novel stapling technique based on a double strain-promoted azide-alkyne reaction, and exploit its biocompatibility to accelerate the discovery of cell-active stapled peptides.  As a proof of concept, MDM2-binding peptides were stapled in parallel, directly in cell culture medium in 96-well plates, and simultaneously evaluated in a p53 reporter assay.  This in situ stapling/screening process gave an optimal candidate that showed improved proteolytic stability and nanomolar binding to MDM2 in subsequent biophys. assays.  α-Helicity was confirmed by a crystal structure of the MDM2-peptide complex.  This work introduces in situ stapling as a versatile biocompatible technique with many other potential high-throughput biol. applications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpbh5ftgHQc1LVg90H21EOLACvtfcHk0lhu93NJMYJ2RA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhslClsrbE&md5=7ce32efbb010623a6ee8247373d6701c</span></div><a href="/servlet/linkout?suffix=cit133&amp;dbid=16384&amp;doi=10.1002%2Fanie.201508416&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201508416%26sid%3Dliteratum%253Aachs%26aulast%3DLau%26aufirst%3DY.%2BH.%26aulast%3DWu%26aufirst%3DY.%26aulast%3DRossmann%26aufirst%3DM.%26aulast%3DTan%26aufirst%3DB.%2BX.%26aulast%3Dde%2BAndrade%26aufirst%3DP.%26aulast%3DTan%26aufirst%3DY.%2BS.%26aulast%3DVerma%26aufirst%3DC.%26aulast%3DMcKenzie%26aufirst%3DG.%2BJ.%26aulast%3DVenkitaraman%26aufirst%3DA.%2BR.%26aulast%3DHyv%25C3%25B6nen%26aufirst%3DM.%26aulast%3DSpring%26aufirst%3DD.%2BR.%26atitle%3DDouble%2520Strain-Promoted%2520Macrocyclization%2520for%2520the%2520Rapid%2520Selection%2520of%2520Cell-Active%2520Stapled%2520Peptides%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2015%26volume%3D54%26spage%3D15410%26epage%3D15413%26doi%3D10.1002%2Fanie.201508416" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref134"><div class="reference"><strong class="refLabel"><a href="#ref134" class="rightTabRefNumLink">134</a></strong><div class="NLM_citation" id="rightTab-cit134"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hilinski, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, Y.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hong, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kutchukian, P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crenshaw, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berkovitch, S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ham, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verdine, G. L.</span></span> <span> </span><span class="NLM_article-title">Stitched Α-Helical Peptides via Bis Ring-Closing Metathesis</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>136</i></span>,  <span class="NLM_fpage">12314</span>– <span class="NLM_lpage">12322</span>, <span class="refDoi"> DOI: 10.1021/ja505141j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja505141j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref134/cit134&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtlaisrfI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=136&publication_year=2014&pages=12314-12322&author=G.+J.+Hilinskiauthor=Y.-W.+Kimauthor=J.+Hongauthor=P.+S.+Kutchukianauthor=C.+M.+Crenshawauthor=S.+S.+Berkovitchauthor=A.+Changauthor=S.+Hamauthor=G.+L.+Verdine&title=Stitched+%CE%91-Helical+Peptides+via+Bis+Ring-Closing+Metathesis&doi=10.1021%2Fja505141j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit134R"><div class="casContent"><span class="casTitleNuber">134</span><div class="casTitle"><span class="NLM_cas:atitle">Stitched α-Helical Peptides via Bis Ring-Closing Metathesis</span></div><div class="casAuthors">Hilinski, Gerard J.; Kim, Young-Woo; Hong, Jooyeon; Kutchukian, Peter S.; Crenshaw, Charisse M.; Berkovitch, Shaunna S.; Chang, Andrew; Ham, Sihyun; Verdine, Gregory L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">136</span>
        (<span class="NLM_cas:issue">35</span>),
    <span class="NLM_cas:pages">12314-12322</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Conformationally stabilized α-helical peptides are capable of inhibiting disease-relevant intracellular or extracellular protein-protein interactions in vivo.  The authors have previously reported that the employment of ring-closing metathesis to introduce a single all-hydrocarbon staple along one face of an α-helical peptide greatly increases α-helical content, binding affinity to a target protein, cell penetration through active transport, and resistance to proteolytic degrdn.  In an effort to improve upon this technol. for stabilizing a peptide in a bioactive α-helical conformation, the authors report the discovery of an efficient and selective bis ring-closing metathesis reaction leading to peptides bearing multiple contiguous staples connected by a central spiro ring junction.  CD spectroscopy, NMR, and computational analyses have been used to study the conformation of these "stitched" peptides, which exhibit remarkable thermal stabilities.  Likewise, trypsin proteolysis assays confirm the achievement of a structural rigidity unmatched by peptides bearing a single staple.  Furthermore, fluorescence-activated cell sorting (FACS) and confocal microscopy assays demonstrate that stitched peptides display superior cell penetrating ability compared to their stapled counterparts, suggesting that this technol. may be useful not only in the context of enhancing the drug-like properties of α-helical peptides but also in producing potent agents for the intracellular delivery of proteins and oligonucleotides.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGob9a_dts0-bLVg90H21EOLACvtfcHk0lhu93NJMYJ2RA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtlaisrfI&md5=45e1542906e2062a3859ace862d0865d</span></div><a href="/servlet/linkout?suffix=cit134&amp;dbid=16384&amp;doi=10.1021%2Fja505141j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja505141j%26sid%3Dliteratum%253Aachs%26aulast%3DHilinski%26aufirst%3DG.%2BJ.%26aulast%3DKim%26aufirst%3DY.-W.%26aulast%3DHong%26aufirst%3DJ.%26aulast%3DKutchukian%26aufirst%3DP.%2BS.%26aulast%3DCrenshaw%26aufirst%3DC.%2BM.%26aulast%3DBerkovitch%26aufirst%3DS.%2BS.%26aulast%3DChang%26aufirst%3DA.%26aulast%3DHam%26aufirst%3DS.%26aulast%3DVerdine%26aufirst%3DG.%2BL.%26atitle%3DStitched%2520%25CE%2591-Helical%2520Peptides%2520via%2520Bis%2520Ring-Closing%2520Metathesis%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2014%26volume%3D136%26spage%3D12314%26epage%3D12322%26doi%3D10.1021%2Fja505141j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref135"><div class="reference"><strong class="refLabel"><a href="#ref135" class="rightTabRefNumLink">135</a></strong><div class="NLM_citation" id="rightTab-cit135"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gopalakrishnan, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frolov, A. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knerr, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drury, W. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valeur, E.</span></span> <span> </span><span class="NLM_article-title">Therapeutic Potential of Foldamers: From Chemical Biology Tools To Drug Candidates?</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">9599</span>– <span class="NLM_lpage">9621</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b00376</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b00376" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref135/cit135&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtV2gsbrE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=9599-9621&author=R.+Gopalakrishnanauthor=A.+I.+Frolovauthor=L.+Knerrauthor=W.+J.+Druryauthor=E.+Valeur&title=Therapeutic+Potential+of+Foldamers%3A+From+Chemical+Biology+Tools+To+Drug+Candidates%3F&doi=10.1021%2Facs.jmedchem.6b00376"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit135R"><div class="casContent"><span class="casTitleNuber">135</span><div class="casTitle"><span class="NLM_cas:atitle">Therapeutic Potential of Foldamers: From Chemical Biology Tools To Drug Candidates?</span></div><div class="casAuthors">Gopalakrishnan, Ranganath; Frolov, Andrey I.; Knerr, Laurent; Drury, William J.; Valeur, Eric</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">9599-9621</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Over the past decade, foldamers have progressively emerged as useful architectures to mimic secondary structures of proteins.  Peptidic foldamers, consisting of various amino acid based backbones, have been the most studied from a therapeutic perspective, while polyarom. foldamers have barely evolved from their nascency and remain perplexing for medicinal chemists due to their poor drug-like nature.  Despite these limitations, this compd. class may still offer opportunities to study challenging targets or provide chem. biol. tools.  The potential of foldamer drug candidates reaching the clinic is still a stretch.  Nevertheless, advances in the field have demonstrated their potential for the discovery of next generation therapeutics.  In this perspective, the current knowledge of foldamers is reviewed in a drug discovery context.  Recent advances in the early phases of drug discovery including hit finding, target validation, and optimization and mol. modeling are discussed.  In addn., challenges and focus areas are debated and gaps highlighted.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoI_-tbQEJhE7Vg90H21EOLACvtfcHk0lhu93NJMYJ2RA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtV2gsbrE&md5=7e6ff6d5e90ef9992e71c17ce30f565e</span></div><a href="/servlet/linkout?suffix=cit135&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b00376&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b00376%26sid%3Dliteratum%253Aachs%26aulast%3DGopalakrishnan%26aufirst%3DR.%26aulast%3DFrolov%26aufirst%3DA.%2BI.%26aulast%3DKnerr%26aufirst%3DL.%26aulast%3DDrury%26aufirst%3DW.%2BJ.%26aulast%3DValeur%26aufirst%3DE.%26atitle%3DTherapeutic%2520Potential%2520of%2520Foldamers%253A%2520From%2520Chemical%2520Biology%2520Tools%2520To%2520Drug%2520Candidates%253F%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D9599%26epage%3D9621%26doi%3D10.1021%2Facs.jmedchem.6b00376" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref136"><div class="reference"><strong class="refLabel"><a href="#ref136" class="rightTabRefNumLink">136</a></strong><div class="NLM_citation" id="rightTab-cit136"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cheloha, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maeda, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dean, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gardella, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gellman, S. H.</span></span> <span> </span><span class="NLM_article-title">Backbone Modification of a Polypeptide Drug Alters Duration of Action in Vivo</span>. <i>Nat. Biotechnol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>32</i></span>,  <span class="NLM_fpage">653</span>– <span class="NLM_lpage">655</span>, <span class="refDoi"> DOI: 10.1038/nbt.2920</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref136/cit136&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1038%2Fnbt.2920" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref136/cit136&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=24929976" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref136/cit136&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC2cXpslGis7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2014&pages=653-655&author=R.+W.+Chelohaauthor=A.+Maedaauthor=T.+Deanauthor=T.+J.+Gardellaauthor=S.+H.+Gellman&title=Backbone+Modification+of+a+Polypeptide+Drug+Alters+Duration+of+Action+in+Vivo&doi=10.1038%2Fnbt.2920"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit136R"><div class="casContent"><span class="casTitleNuber">136</span><div class="casTitle"><span class="NLM_cas:atitle">Backbone modification of a polypeptide drug alters duration of action in vivo</span></div><div class="casAuthors">Cheloha, Ross W.; Maeda, Akira; Dean, Thomas; Gardella, Thomas J.; Gellman, Samuel H.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">32</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">653-655</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Systematic modification of the backbone of bioactive polypeptides through β-amino acid residue incorporation could provide a strategy for generating mols. with improved drug properties, but such alterations can result in lower receptor affinity and potency.  Using an agonist of parathyroid hormone receptor-1 (PTHR1), a G protein-coupled receptor in the B-family, we present an approach for α→β residue replacement that enables both high activity and improved pharmacokinetic properties in vivo.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr8cWxIA-3shbVg90H21EOLACvtfcHk0lhu93NJMYJ2RA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXpslGis7k%253D&md5=100382f0d7e67fe9c11fd39886e7acac</span></div><a href="/servlet/linkout?suffix=cit136&amp;dbid=16384&amp;doi=10.1038%2Fnbt.2920&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt.2920%26sid%3Dliteratum%253Aachs%26aulast%3DCheloha%26aufirst%3DR.%2BW.%26aulast%3DMaeda%26aufirst%3DA.%26aulast%3DDean%26aufirst%3DT.%26aulast%3DGardella%26aufirst%3DT.%2BJ.%26aulast%3DGellman%26aufirst%3DS.%2BH.%26atitle%3DBackbone%2520Modification%2520of%2520a%2520Polypeptide%2520Drug%2520Alters%2520Duration%2520of%2520Action%2520in%2520Vivo%26jtitle%3DNat.%2520Biotechnol.%26date%3D2014%26volume%3D32%26spage%3D653%26epage%3D655%26doi%3D10.1038%2Fnbt.2920" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref137"><div class="reference"><strong class="refLabel"><a href="#ref137" class="rightTabRefNumLink">137</a></strong><div class="NLM_citation" id="rightTab-cit137"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hager, M. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wootten, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sexton, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gellman, S. H.</span></span> <span> </span><span class="NLM_article-title">Β-Arrestin-Biased Agonists of the GLP-1 Receptor from Β-Amino Acid Residue Incorporation into GLP-1 Analogues</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>138</i></span>,  <span class="NLM_fpage">14970</span>– <span class="NLM_lpage">14979</span>, <span class="refDoi"> DOI: 10.1021/jacs.6b08323</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.6b08323" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=138&publication_year=2016&pages=14970-14979&author=M.+V.+Hagerauthor=L.+M.+Johnsonauthor=D.+Woottenauthor=P.+M.+Sextonauthor=S.+H.+Gellman&title=%CE%92-Arrestin-Biased+Agonists+of+the+GLP-1+Receptor+from+%CE%92-Amino+Acid+Residue+Incorporation+into+GLP-1+Analogues&doi=10.1021%2Fjacs.6b08323"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit137&amp;dbid=16384&amp;doi=10.1021%2Fjacs.6b08323&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.6b08323%26sid%3Dliteratum%253Aachs%26aulast%3DHager%26aufirst%3DM.%2BV.%26aulast%3DJohnson%26aufirst%3DL.%2BM.%26aulast%3DWootten%26aufirst%3DD.%26aulast%3DSexton%26aufirst%3DP.%2BM.%26aulast%3DGellman%26aufirst%3DS.%2BH.%26atitle%3D%25CE%2592-Arrestin-Biased%2520Agonists%2520of%2520the%2520GLP-1%2520Receptor%2520from%2520%25CE%2592-Amino%2520Acid%2520Residue%2520Incorporation%2520into%2520GLP-1%2520Analogues%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2016%26volume%3D138%26spage%3D14970%26epage%3D14979%26doi%3D10.1021%2Fjacs.6b08323" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref138"><div class="reference"><strong class="refLabel"><a href="#ref138" class="rightTabRefNumLink">138</a></strong><div class="NLM_citation" id="rightTab-cit138"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Robinson, J. A.</span></span> <span> </span><span class="NLM_article-title">Β-Hairpin Peptidomimetics: Design, Structures and Biological Activities</span>. <i>Acc. Chem. Res.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>41</i></span>,  <span class="NLM_fpage">1278</span>– <span class="NLM_lpage">1288</span>, <span class="refDoi"> DOI: 10.1021/ar700259k</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ar700259k" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref138/cit138&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BD1cXkslChsbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2008&pages=1278-1288&author=J.+A.+Robinson&title=%CE%92-Hairpin+Peptidomimetics%3A+Design%2C+Structures+and+Biological+Activities&doi=10.1021%2Far700259k"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit138R"><div class="casContent"><span class="casTitleNuber">138</span><div class="casTitle"><span class="NLM_cas:atitle">β-Hairpin Peptidomimetics: Design, Structures and Biological Activities</span></div><div class="casAuthors">Robinson, John A.</div><div class="citationInfo"><span class="NLM_cas:title">Accounts of Chemical Research</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1278-1288</span>CODEN:
                <span class="NLM_cas:coden">ACHRE4</span>;
        ISSN:<span class="NLM_cas:issn">0001-4842</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The folded 3D structures of peptides and proteins provide excellent starting points for the design of synthetic mols. that mimic key epitopes (or surface patches) involved in protein-protein and protein-nucleic acid interactions.  Protein epitope mimetics (PEMs) may recapitulate not only the structural and conformational properties of the target epitope but also their biol. activities.  By transferring the epitope from a recombinant to a synthetic scaffold that can be produced by parallel combinatorial methods, it is possible to optimize properties through iterative cycles of library synthesis and screening, and even to evolve new biol. activities.  One very interesting scaffold is found in β-hairpin motifs, which are used by many proteins to mediate mol. recognition events.  This motif is readily amenable to PEM design, for example, by transplanting hairpin loop sequences from folded proteins onto hairpin-stabilizing templates, such as the dipeptide D-Pro-L-Pro.  In addn., β-hairpin peptidomimetics can also be exploited to mimic other types of epitopes, such as those based on α-helical secondary structures.  The size and shape of β-hairpin PEMs appear well suited for the design of inhibitors of both protein-protein and protein-nucleic acid interactions, endeavors that have so far proven difficult using small "drug-like" mols.  In recent work, it was shown that β-hairpin PEMs can be designed that mimic the canonical conformations of antibody hypervariable loops, suggesting that novel small-mol. antibody mimics may be feasible.  Using naturally occurring peptides as starting points, β-hairpin mimetics have been discovered that possess antimicrobial activity, while others are potent inhibitors of the chemokine receptor CXCR4. β-Hairpin PEMs have also been designed and optimized that mimic an α-helical epitope in p53 and so block its interaction with HDM2.  A crystal structure of one HDM2-mimetic complex revealed how the surface of the protein had adapted to the shape of the hairpin, thereby enhancing inhibitor affinity.  Small folded RNA motifs also make interesting targets for inhibitor design.  For example, β-hairpin mimetics have been designed and optimized that bind with high affinity and good selectivity to the TAR and RRE RNA motifs from HIV-1.  Soln. structures of the mimetics both free and bound to the RNA target provided some surprises, as well as an improved understanding of the mechanisms of binding.  These mimetics represent still a relatively new family of RNA-binding mols., but clearly one with potential for development into novel antiviral agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqJpSiBOoiIQLVg90H21EOLACvtfcHk0lgfBBCJCYcf7w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXkslChsbc%253D&md5=083014216e0fc0918cf096d5fcf446d4</span></div><a href="/servlet/linkout?suffix=cit138&amp;dbid=16384&amp;doi=10.1021%2Far700259k&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Far700259k%26sid%3Dliteratum%253Aachs%26aulast%3DRobinson%26aufirst%3DJ.%2BA.%26atitle%3D%25CE%2592-Hairpin%2520Peptidomimetics%253A%2520Design%252C%2520Structures%2520and%2520Biological%2520Activities%26jtitle%3DAcc.%2520Chem.%2520Res.%26date%3D2008%26volume%3D41%26spage%3D1278%26epage%3D1288%26doi%3D10.1021%2Far700259k" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref139"><div class="reference"><strong class="refLabel"><a href="#ref139" class="rightTabRefNumLink">139</a></strong><div class="NLM_citation" id="rightTab-cit139"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Srinivas, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jetter, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ueberbacher, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Werneburg, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zerbe, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steinmann, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van der Meijden, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernardini, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lederer, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dias, R. L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Misson, P. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Henze, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zumbrunn, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gombert, F. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Obrecht, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hunziker, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schauer, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ziegler, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Käch, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eberl, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riedel, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DeMarco, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robinson, J. A.</span></span> <span> </span><span class="NLM_article-title">Peptidomimetic Antibiotics Target Outer-Membrane Biogenesis in Pseudomonas Aeruginosa</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>327</i></span>,  <span class="NLM_fpage">1010</span>– <span class="NLM_lpage">1013</span>, <span class="refDoi"> DOI: 10.1126/science.1182749</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref139/cit139&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1126%2Fscience.1182749" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref139/cit139&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=20167788" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref139/cit139&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC3cXitVSjtbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=327&publication_year=2010&pages=1010-1013&author=N.+Srinivasauthor=P.+Jetterauthor=B.+J.+Ueberbacherauthor=M.+Werneburgauthor=K.+Zerbeauthor=J.+Steinmannauthor=B.+Van+der+Meijdenauthor=F.+Bernardiniauthor=A.+Ledererauthor=R.+L.+A.+Diasauthor=P.+E.+Missonauthor=H.+Henzeauthor=J.+Zumbrunnauthor=F.+O.+Gombertauthor=D.+Obrechtauthor=P.+Hunzikerauthor=S.+Schauerauthor=U.+Zieglerauthor=A.+K%C3%A4chauthor=L.+Eberlauthor=K.+Riedelauthor=S.+J.+DeMarcoauthor=J.+A.+Robinson&title=Peptidomimetic+Antibiotics+Target+Outer-Membrane+Biogenesis+in+Pseudomonas+Aeruginosa&doi=10.1126%2Fscience.1182749"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit139R"><div class="casContent"><span class="casTitleNuber">139</span><div class="casTitle"><span class="NLM_cas:atitle">Peptidomimetic Antibiotics Target Outer-Membrane Biogenesis in Pseudomonas aeruginosa</span></div><div class="casAuthors">Srinivas, Nityakalyani; Jetter, Peter; Ueberbacher, Bernhard J.; Werneburg, Martina; Zerbe, Katja; Steinmann, Jessica; Van der Meijden, Benjamin; Bernardini, Francesca; Lederer, Alexander; Dias, Ricardo L. A.; Misson, Pauline E.; Henze, Heiko; Zumbrunn, Juerg; Gombert, Frank O.; Obrecht, Daniel; Hunziker, Peter; Schauer, Stefan; Ziegler, Urs; Kaech, Andres; Eberl, Leo; Riedel, Kathrin; De Marco, Steven J.; Robinson, John A.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">327</span>
        (<span class="NLM_cas:issue">5968</span>),
    <span class="NLM_cas:pages">1010-1013</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Antibiotics with new mechanisms of action are urgently required to combat the growing health threat posed by resistant pathogenic microorganisms.  The authors synthesized a family of peptidomimetic antibiotics based on the antimicrobial peptide protegrin I.  Several rounds of optimization gave a lead compd. that was active in the nanomolar range against Gram-neg. Pseudomonas spp., but was largely inactive against other Gram-neg. and Gram-pos. bacteria.  Biochem. and genetic studies showed that the peptidomimetics had a non-membrane-lytic mechanism of action and identified a homolog of the β-barrel protein LptD (Imp/OstA), which functions in outer-membrane biogenesis, as a cellular target.  The peptidomimetic showed potent antimicrobial activity in a mouse septicemia infection model.  Drug-resistant strains of Pseudomonas are a serious health problem, so this family of antibiotics may have important therapeutic applications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpjVCgTY_lqGLVg90H21EOLACvtfcHk0lgfBBCJCYcf7w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXitVSjtbk%253D&md5=a59f74c897b26c6471c6d16910bcb018</span></div><a href="/servlet/linkout?suffix=cit139&amp;dbid=16384&amp;doi=10.1126%2Fscience.1182749&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1182749%26sid%3Dliteratum%253Aachs%26aulast%3DSrinivas%26aufirst%3DN.%26aulast%3DJetter%26aufirst%3DP.%26aulast%3DUeberbacher%26aufirst%3DB.%2BJ.%26aulast%3DWerneburg%26aufirst%3DM.%26aulast%3DZerbe%26aufirst%3DK.%26aulast%3DSteinmann%26aufirst%3DJ.%26aulast%3DVan%2Bder%2BMeijden%26aufirst%3DB.%26aulast%3DBernardini%26aufirst%3DF.%26aulast%3DLederer%26aufirst%3DA.%26aulast%3DDias%26aufirst%3DR.%2BL.%2BA.%26aulast%3DMisson%26aufirst%3DP.%2BE.%26aulast%3DHenze%26aufirst%3DH.%26aulast%3DZumbrunn%26aufirst%3DJ.%26aulast%3DGombert%26aufirst%3DF.%2BO.%26aulast%3DObrecht%26aufirst%3DD.%26aulast%3DHunziker%26aufirst%3DP.%26aulast%3DSchauer%26aufirst%3DS.%26aulast%3DZiegler%26aufirst%3DU.%26aulast%3DK%25C3%25A4ch%26aufirst%3DA.%26aulast%3DEberl%26aufirst%3DL.%26aulast%3DRiedel%26aufirst%3DK.%26aulast%3DDeMarco%26aufirst%3DS.%2BJ.%26aulast%3DRobinson%26aufirst%3DJ.%2BA.%26atitle%3DPeptidomimetic%2520Antibiotics%2520Target%2520Outer-Membrane%2520Biogenesis%2520in%2520Pseudomonas%2520Aeruginosa%26jtitle%3DScience%26date%3D2010%26volume%3D327%26spage%3D1010%26epage%3D1013%26doi%3D10.1126%2Fscience.1182749" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref140"><div class="reference"><strong class="refLabel"><a href="#ref140" class="rightTabRefNumLink">140</a></strong><div class="NLM_citation" id="rightTab-cit140"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zerbe, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moehle, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robinson, J. A.</span></span> <span> </span><span class="NLM_article-title">Protein Epitope Mimetics: From New Antibiotics to Supramolecular Synthetic Vaccines</span>. <i>Acc. Chem. Res.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>50</i></span>,  <span class="NLM_fpage">1323</span>– <span class="NLM_lpage">1331</span>, <span class="refDoi"> DOI: 10.1021/acs.accounts.7b00129</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.accounts.7b00129" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2017&pages=1323-1331&author=K.+Zerbeauthor=K.+Moehleauthor=J.+A.+Robinson&title=Protein+Epitope+Mimetics%3A+From+New+Antibiotics+to+Supramolecular+Synthetic+Vaccines&doi=10.1021%2Facs.accounts.7b00129"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit140&amp;dbid=16384&amp;doi=10.1021%2Facs.accounts.7b00129&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.accounts.7b00129%26sid%3Dliteratum%253Aachs%26aulast%3DZerbe%26aufirst%3DK.%26aulast%3DMoehle%26aufirst%3DK.%26aulast%3DRobinson%26aufirst%3DJ.%2BA.%26atitle%3DProtein%2520Epitope%2520Mimetics%253A%2520From%2520New%2520Antibiotics%2520to%2520Supramolecular%2520Synthetic%2520Vaccines%26jtitle%3DAcc.%2520Chem.%2520Res.%26date%3D2017%26volume%3D50%26spage%3D1323%26epage%3D1331%26doi%3D10.1021%2Facs.accounts.7b00129" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref141"><div class="reference"><strong class="refLabel"><a href="#ref141" class="rightTabRefNumLink">141</a></strong><div class="NLM_citation" id="rightTab-cit141"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Luther, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moehle, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chevalier, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dale, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Obrecht, D.</span></span> <span> </span><span class="NLM_article-title">Protein Epitope Mimetic Macrocycles as Biopharmaceuticals</span>. <i>Curr. Opin. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>38</i></span>,  <span class="NLM_fpage">45</span>– <span class="NLM_lpage">51</span>, <span class="refDoi"> DOI: 10.1016/j.cbpa.2017.02.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref141/cit141&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1016%2Fj.cbpa.2017.02.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref141/cit141&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=28319811" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref141/cit141&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC2sXksFSiu7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=2017&pages=45-51&author=A.+Lutherauthor=K.+Moehleauthor=E.+Chevalierauthor=G.+Daleauthor=D.+Obrecht&title=Protein+Epitope+Mimetic+Macrocycles+as+Biopharmaceuticals&doi=10.1016%2Fj.cbpa.2017.02.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit141R"><div class="casContent"><span class="casTitleNuber">141</span><div class="casTitle"><span class="NLM_cas:atitle">Protein epitope mimetic macrocycles as biopharmaceuticals</span></div><div class="casAuthors">Luther, Anatol; Moehle, Kerstin; Chevalier, Eric; Dale, Glenn; Obrecht, Daniel</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Chemical Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">45-51</span>CODEN:
                <span class="NLM_cas:coden">COCBF4</span>;
        ISSN:<span class="NLM_cas:issn">1367-5931</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Fully synthetic medium-sized macrocyclic peptides mimicking the key β-hairpin and α-helical protein epitopes relevant in many protein-protein interactions have emerged as a novel class of drugs with the potential to fill an important gap between small mols. and proteins.  Conformationally stabilized macrocyclic scaffolds represent ideal templates for medicinal chemists to incorporate bioactive peptide and protein pharmacophores in order to generate novel drugs to treat diseases with high unmet medical need.  This review describes recent approaches to design and generate large libraries of such macrocycles, for hit identification, and for their efficient optimization.  Finally, this review describes some of the most advanced protein epitope mimetic (PEM) macrocycles in clin. development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpKLA10A3orAbVg90H21EOLACvtfcHk0liG0-HQvPdKcg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXksFSiu7Y%253D&md5=623f351d0e915d58f1ebd698da850a37</span></div><a href="/servlet/linkout?suffix=cit141&amp;dbid=16384&amp;doi=10.1016%2Fj.cbpa.2017.02.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cbpa.2017.02.004%26sid%3Dliteratum%253Aachs%26aulast%3DLuther%26aufirst%3DA.%26aulast%3DMoehle%26aufirst%3DK.%26aulast%3DChevalier%26aufirst%3DE.%26aulast%3DDale%26aufirst%3DG.%26aulast%3DObrecht%26aufirst%3DD.%26atitle%3DProtein%2520Epitope%2520Mimetic%2520Macrocycles%2520as%2520Biopharmaceuticals%26jtitle%3DCurr.%2520Opin.%2520Chem.%2520Biol.%26date%3D2017%26volume%3D38%26spage%3D45%26epage%3D51%26doi%3D10.1016%2Fj.cbpa.2017.02.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref142"><div class="reference"><strong class="refLabel"><a href="#ref142" class="rightTabRefNumLink">142</a></strong><div class="NLM_citation" id="rightTab-cit142"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gavenonis, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheneman, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siegert, T. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eshelman, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kritzer, J. A.</span></span> <span> </span><span class="NLM_article-title">Comprehensive Analysis of Loops at Protein-Protein Interfaces for Macrocycle Design</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">716</span>– <span class="NLM_lpage">722</span>, <span class="refDoi"> DOI: 10.1038/nchembio.1580</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref142/cit142&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1038%2Fnchembio.1580" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref142/cit142&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=25038791" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref142/cit142&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtFygs7jE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2014&pages=716-722&author=J.+Gavenonisauthor=B.+A.+Shenemanauthor=T.+R.+Siegertauthor=M.+R.+Eshelmanauthor=J.+A.+Kritzer&title=Comprehensive+Analysis+of+Loops+at+Protein-Protein+Interfaces+for+Macrocycle+Design&doi=10.1038%2Fnchembio.1580"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit142R"><div class="casContent"><span class="casTitleNuber">142</span><div class="casTitle"><span class="NLM_cas:atitle">Comprehensive analysis of loops at protein-protein interfaces for macrocycle design</span></div><div class="casAuthors">Gavenonis, Jason; Sheneman, Bradley A.; Siegert, Timothy R.; Eshelman, Matthew R.; Kritzer, Joshua A.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">716-722</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Inhibiting protein-protein interactions (PPIs) with synthetic mols. remains a frontier of chem. biol.  Many PPIs have been successfully targeted by mimicking α-helixes at interfaces, but most PPIs are mediated by nonhelical, nonstrand peptide loops.  We sought to comprehensively identify and analyze these loop-mediated PPIs by writing and implementing LoopFinder, a customizable program that can identify loop-mediated PPIs within all of the protein-protein complexes in the Protein Data Bank.  Comprehensive anal. of the entire set of 25,005 interface loops revealed common structural motifs and unique features that distinguish loop-mediated PPIs from other PPIs.  'Hot loops', named in analogy to protein hot spots, were identified as loops with favorable properties for mimicry using synthetic mols.  The hot loops and their binding partners represent new and promising PPIs for the development of macrocycle and constrained peptide inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp96pqxP3_amLVg90H21EOLACvtfcHk0liG0-HQvPdKcg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtFygs7jE&md5=bf07bd724f173ea74467ab7cd5aedb6a</span></div><a href="/servlet/linkout?suffix=cit142&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1580&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1580%26sid%3Dliteratum%253Aachs%26aulast%3DGavenonis%26aufirst%3DJ.%26aulast%3DSheneman%26aufirst%3DB.%2BA.%26aulast%3DSiegert%26aufirst%3DT.%2BR.%26aulast%3DEshelman%26aufirst%3DM.%2BR.%26aulast%3DKritzer%26aufirst%3DJ.%2BA.%26atitle%3DComprehensive%2520Analysis%2520of%2520Loops%2520at%2520Protein-Protein%2520Interfaces%2520for%2520Macrocycle%2520Design%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2014%26volume%3D10%26spage%3D716%26epage%3D722%26doi%3D10.1038%2Fnchembio.1580" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref143"><div class="reference"><strong class="refLabel"><a href="#ref143" class="rightTabRefNumLink">143</a></strong><div class="NLM_citation" id="rightTab-cit143"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Siegert, T. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bird, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Makwana, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kritzer, J. A.</span></span> <span> </span><span class="NLM_article-title">Analysis of Loops That Mediate Protein–Protein Interactions and Translation into Submicromolar Inhibitors</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>138</i></span>,  <span class="NLM_fpage">12876</span>– <span class="NLM_lpage">12884</span>, <span class="refDoi"> DOI: 10.1021/jacs.6b05656</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.6b05656" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref143/cit143&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsVyqtL%252FK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=138&publication_year=2016&pages=12876-12884&author=T.+R.+Siegertauthor=M.+J.+Birdauthor=K.+M.+Makwanaauthor=J.+A.+Kritzer&title=Analysis+of+Loops+That+Mediate+Protein%E2%80%93Protein+Interactions+and+Translation+into+Submicromolar+Inhibitors&doi=10.1021%2Fjacs.6b05656"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit143R"><div class="casContent"><span class="casTitleNuber">143</span><div class="casTitle"><span class="NLM_cas:atitle">Analysis of Loops that Mediate Protein-Protein Interactions and Translation into Submicromolar Inhibitors</span></div><div class="casAuthors">Siegert, Timothy R.; Bird, Michael J.; Makwana, Kamlesh M.; Kritzer, Joshua A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">138</span>
        (<span class="NLM_cas:issue">39</span>),
    <span class="NLM_cas:pages">12876-12884</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Effective strategies for mimicking α-helix and β-strand epitopes have been developed, producing valuable inhibitors for some classes of protein-protein interactions (PPIs).  However, there are no general strategies for translating loop epitopes into useful PPI inhibitors.  In this work, we use the LoopFinder program to identify diverse sets of "hot loops," which are loop epitopes that mediate PPIs.  These include loops that are well-suited to mimicry with macrocyclic compds., and loops that are most similar to variable loops on antibodies and ankyrin repeat proteins.  We present data-driven criteria for scoring loop-mediated PPIs, uncovering a trove of potentially druggable interactions.  We also use unbiased clustering to identify common structures among the hot loops.  To translate these insights into real-world inhibitors, we describe a robust, diversity-oriented strategy for the rapid prodn. and evaluation of cyclized loops.  This method is applied to a computationally identified loop in the PPI between stonin2 and Eps15, producing sub-micromolar inhibitors.  The most potent inhibitor is well-structured in water and successfully mimics the native epitope.  Overall, these computational and exptl. strategies provide new opportunities to design inhibitors for an otherwise intractable set of PPIs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpWWUk81azbxLVg90H21EOLACvtfcHk0lgQ2KE9RfKZOw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsVyqtL%252FK&md5=9a036f700e452a65257a0c0c8b52a937</span></div><a href="/servlet/linkout?suffix=cit143&amp;dbid=16384&amp;doi=10.1021%2Fjacs.6b05656&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.6b05656%26sid%3Dliteratum%253Aachs%26aulast%3DSiegert%26aufirst%3DT.%2BR.%26aulast%3DBird%26aufirst%3DM.%2BJ.%26aulast%3DMakwana%26aufirst%3DK.%2BM.%26aulast%3DKritzer%26aufirst%3DJ.%2BA.%26atitle%3DAnalysis%2520of%2520Loops%2520That%2520Mediate%2520Protein%25E2%2580%2593Protein%2520Interactions%2520and%2520Translation%2520into%2520Submicromolar%2520Inhibitors%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2016%26volume%3D138%26spage%3D12876%26epage%3D12884%26doi%3D10.1021%2Fjacs.6b05656" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref144"><div class="reference"><strong class="refLabel"><a href="#ref144" class="rightTabRefNumLink">144</a></strong><div class="NLM_citation" id="rightTab-cit144"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rong, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luk, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qin, N.</span></span> <span> </span><span class="NLM_article-title">Targeting Hippo Pathway by Specific Interruption of YAP-TEAD Interaction Using Cyclic YAP-like Peptides</span>. <i>FASEB J.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">724</span>– <span class="NLM_lpage">732</span>, <span class="refDoi"> DOI: 10.1096/fj.14-262980</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref144/cit144&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1096%2Ffj.14-262980" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref144/cit144&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=25384421" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref144/cit144&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC2MXislGisbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2015&pages=724-732&author=Z.+Zhouauthor=T.+Huauthor=Z.+Xuauthor=Z.+Linauthor=Z.+Zhangauthor=T.+Fengauthor=L.+Zhuauthor=Y.+Rongauthor=H.+Shenauthor=J.+M.+Lukauthor=X.+Zhangauthor=N.+Qin&title=Targeting+Hippo+Pathway+by+Specific+Interruption+of+YAP-TEAD+Interaction+Using+Cyclic+YAP-like+Peptides&doi=10.1096%2Ffj.14-262980"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit144R"><div class="casContent"><span class="casTitleNuber">144</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting Hippo pathway by specific interruption of YAP-TEAD interaction using cyclic YAP-like peptides</span></div><div class="casAuthors">Zhou, Zheng; Hu, Taishan; Xu, Zhiheng; Lin, Zhaohu; Zhang, Zhisen; Feng, Teng; Zhu, Liangcheng; Rong, Yiping; Shen, Hong; Luk, John M.; Zhang, Xiongwen; Qin, Ning</div><div class="citationInfo"><span class="NLM_cas:title">FASEB Journal</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">724-732, 10.1096/fj.14-262980</span>CODEN:
                <span class="NLM_cas:coden">FAJOEC</span>;
        ISSN:<span class="NLM_cas:issn">0892-6638</span>.
    
            (<span class="NLM_cas:orgname">Federation of American Societies for Experimental Biology</span>)
        </div><div class="casAbstract">Hippo signaling pathway is emerging as a novel target for anticancer therapy because it plays key roles in organ size control and tumorigenesis.  As the downstream effectors, Yes-assocd. protein (YAP)-transcriptional enhancer activation domain family member (TEAD) assocn. is essential for YAP-driven oncogenic activity, while TEAD is largely dispensable for normal tissue growth.  We present the design of YAP-like peptides (17mer) to occupy the interface 3 on TEAD.  Introducing cysteines at YAP sites 87 and 96 can induce disulfide formation, as confirmed by crystallog.  The engineered peptide significantly improves the potency in disrupting YAP-TEAD interaction in vitro.  To confirm that blocking YAP-TEAD complex formation by directly targeting on TEAD is a valid approach, we report a significant redn. in tumor growth rate in a hepatocellular carcinoma xenograft model after introducing the dominant-neg. mutation (Y406H) of TEAD1 to abolish YAP-TEAD interaction.  Our results suggest that targeting TEAD is a promising strategy against YAP-induced oncogenesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqIJ7Idzph7ELVg90H21EOLACvtfcHk0lgQ2KE9RfKZOw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXislGisbk%253D&md5=bd3ce9530d0b5587c0118bf32e975546</span></div><a href="/servlet/linkout?suffix=cit144&amp;dbid=16384&amp;doi=10.1096%2Ffj.14-262980&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1096%252Ffj.14-262980%26sid%3Dliteratum%253Aachs%26aulast%3DZhou%26aufirst%3DZ.%26aulast%3DHu%26aufirst%3DT.%26aulast%3DXu%26aufirst%3DZ.%26aulast%3DLin%26aufirst%3DZ.%26aulast%3DZhang%26aufirst%3DZ.%26aulast%3DFeng%26aufirst%3DT.%26aulast%3DZhu%26aufirst%3DL.%26aulast%3DRong%26aufirst%3DY.%26aulast%3DShen%26aufirst%3DH.%26aulast%3DLuk%26aufirst%3DJ.%2BM.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DQin%26aufirst%3DN.%26atitle%3DTargeting%2520Hippo%2520Pathway%2520by%2520Specific%2520Interruption%2520of%2520YAP-TEAD%2520Interaction%2520Using%2520Cyclic%2520YAP-like%2520Peptides%26jtitle%3DFASEB%2520J.%26date%3D2015%26volume%3D29%26spage%3D724%26epage%3D732%26doi%3D10.1096%2Ffj.14-262980" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref145"><div class="reference"><strong class="refLabel"><a href="#ref145" class="rightTabRefNumLink">145</a></strong><div class="NLM_citation" id="rightTab-cit145"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, H. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, S. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mayweg, A. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qin, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rong, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fry, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, T.</span></span> <span> </span><span class="NLM_article-title">Structure-Based Design and Synthesis of Potent Cyclic Peptides Inhibiting the YAP–TEAD Protein–Protein Interaction</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">993</span>– <span class="NLM_lpage">998</span>, <span class="refDoi"> DOI: 10.1021/ml500160m</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ml500160m" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&pages=993-998&author=Z.+Zhangauthor=Z.+Linauthor=Z.+Zhouauthor=H.+C.+Shenauthor=S.+F.+Yanauthor=A.+V.+Maywegauthor=Z.+Xuauthor=N.+Qinauthor=J.+C.+Wongauthor=Z.+Zhangauthor=Y.+Rongauthor=D.+C.+Fryauthor=T.+Hu&title=Structure-Based+Design+and+Synthesis+of+Potent+Cyclic+Peptides+Inhibiting+the+YAP%E2%80%93TEAD+Protein%E2%80%93Protein+Interaction&doi=10.1021%2Fml500160m"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit145&amp;dbid=16384&amp;doi=10.1021%2Fml500160m&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fml500160m%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DZ.%26aulast%3DLin%26aufirst%3DZ.%26aulast%3DZhou%26aufirst%3DZ.%26aulast%3DShen%26aufirst%3DH.%2BC.%26aulast%3DYan%26aufirst%3DS.%2BF.%26aulast%3DMayweg%26aufirst%3DA.%2BV.%26aulast%3DXu%26aufirst%3DZ.%26aulast%3DQin%26aufirst%3DN.%26aulast%3DWong%26aufirst%3DJ.%2BC.%26aulast%3DZhang%26aufirst%3DZ.%26aulast%3DRong%26aufirst%3DY.%26aulast%3DFry%26aufirst%3DD.%2BC.%26aulast%3DHu%26aufirst%3DT.%26atitle%3DStructure-Based%2520Design%2520and%2520Synthesis%2520of%2520Potent%2520Cyclic%2520Peptides%2520Inhibiting%2520the%2520YAP%25E2%2580%2593TEAD%2520Protein%25E2%2580%2593Protein%2520Interaction%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2014%26volume%3D5%26spage%3D993%26epage%3D998%26doi%3D10.1021%2Fml500160m" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref146"><div class="reference"><strong class="refLabel"><a href="#ref146" class="rightTabRefNumLink">146</a></strong><div class="NLM_citation" id="rightTab-cit146"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bhardwaj, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mulligan, V. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bahl, C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilmore, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harvey, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheneval, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buchko, G. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pulavarti, S. V. S. R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaas, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eletsky, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, P.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnsen, W. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greisen, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rocklin, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Linsky, T. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watkins, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rettie, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carter, L. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bonneau, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olson, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coutsias, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Correnti, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szyperski, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Craik, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baker, D.</span></span> <span> </span><span class="NLM_article-title">Accurate de Novo Design of Hyperstable Constrained Peptides</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>538</i></span>,  <span class="NLM_fpage">329</span>– <span class="NLM_lpage">335</span>, <span class="refDoi"> DOI: 10.1038/nature19791</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref146/cit146&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1038%2Fnature19791" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref146/cit146&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=27626386" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref146/cit146&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsFWiurnL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=538&publication_year=2016&pages=329-335&author=G.+Bhardwajauthor=V.+K.+Mulliganauthor=C.+D.+Bahlauthor=J.+M.+Gilmoreauthor=P.+J.+Harveyauthor=O.+Chenevalauthor=G.+W.+Buchkoauthor=S.+V.+S.+R.+K.+Pulavartiauthor=Q.+Kaasauthor=A.+Eletskyauthor=P.-S.+Huangauthor=W.+A.+Johnsenauthor=P.+J.+Greisenauthor=G.+J.+Rocklinauthor=Y.+Songauthor=T.+W.+Linskyauthor=A.+Watkinsauthor=S.+A.+Rettieauthor=X.+Xuauthor=L.+P.+Carterauthor=R.+Bonneauauthor=J.+M.+Olsonauthor=E.+Coutsiasauthor=C.+E.+Correntiauthor=T.+Szyperskiauthor=D.+J.+Craikauthor=D.+Baker&title=Accurate+de+Novo+Design+of+Hyperstable+Constrained+Peptides&doi=10.1038%2Fnature19791"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit146R"><div class="casContent"><span class="casTitleNuber">146</span><div class="casTitle"><span class="NLM_cas:atitle">Accurate de novo design of hyperstable constrained peptides</span></div><div class="casAuthors">Bhardwaj, Gaurav; Mulligan, Vikram Khipple; Bahl, Christopher D.; Gilmore, Jason M.; Harvey, Peta J.; Cheneval, Olivier; Buchko, Garry W.; Pulavarti, Surya V. S. R. K.; Kaas, Quentin; Eletsky, Alexander; Huang, Po-Ssu; Johnsen, William A.; Greisen, Per Jr; Rocklin, Gabriel J.; Song, Yifan; Linsky, Thomas W.; Watkins, Andrew; Rettie, Stephen A.; Xu, Xianzhong; Carter, Lauren P.; Bonneau, Richard; Olson, James M.; Coutsias, Evangelos; Correnti, Colin E.; Szyperski, Thomas; Craik, David J.; Baker, David</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">538</span>
        (<span class="NLM_cas:issue">7625</span>),
    <span class="NLM_cas:pages">329-335</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Naturally occurring, pharmacol. active peptides constrained with covalent crosslinks generally have shapes that have evolved to fit precisely into binding pockets on their targets.  Such peptides can have excellent pharmaceutical properties, combining the stability and tissue penetration of small-mol. drugs with the specificity of much larger protein therapeutics.  The ability to design constrained peptides with precisely specified tertiary structures would enable the design of shape-complementary inhibitors of arbitrary targets.  Here we describe the development of computational methods for accurate de novo design of conformationally restricted peptides, and the use of these methods to design 18-47 residue, disulfide-crosslinked peptides, a subset of which are heterochiral and/or N-C backbone-cyclized.  Both genetically encodable and non-canonical peptides are exceptionally stable to thermal and chem. denaturation, and 12 exptl. detd. X-ray and NMR structures are nearly identical to the computational design models.  The computational design methods and stable scaffolds presented here provide the basis for development of a new generation of peptide-based drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqHeMGaFeL_kbVg90H21EOLACvtfcHk0liQw2oewmI_og"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsFWiurnL&md5=2d2b551ad78a2d76c99eeaba0c763c8e</span></div><a href="/servlet/linkout?suffix=cit146&amp;dbid=16384&amp;doi=10.1038%2Fnature19791&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature19791%26sid%3Dliteratum%253Aachs%26aulast%3DBhardwaj%26aufirst%3DG.%26aulast%3DMulligan%26aufirst%3DV.%2BK.%26aulast%3DBahl%26aufirst%3DC.%2BD.%26aulast%3DGilmore%26aufirst%3DJ.%2BM.%26aulast%3DHarvey%26aufirst%3DP.%2BJ.%26aulast%3DCheneval%26aufirst%3DO.%26aulast%3DBuchko%26aufirst%3DG.%2BW.%26aulast%3DPulavarti%26aufirst%3DS.%2BV.%2BS.%2BR.%2BK.%26aulast%3DKaas%26aufirst%3DQ.%26aulast%3DEletsky%26aufirst%3DA.%26aulast%3DHuang%26aufirst%3DP.-S.%26aulast%3DJohnsen%26aufirst%3DW.%2BA.%26aulast%3DGreisen%26aufirst%3DP.%2BJ.%26aulast%3DRocklin%26aufirst%3DG.%2BJ.%26aulast%3DSong%26aufirst%3DY.%26aulast%3DLinsky%26aufirst%3DT.%2BW.%26aulast%3DWatkins%26aufirst%3DA.%26aulast%3DRettie%26aufirst%3DS.%2BA.%26aulast%3DXu%26aufirst%3DX.%26aulast%3DCarter%26aufirst%3DL.%2BP.%26aulast%3DBonneau%26aufirst%3DR.%26aulast%3DOlson%26aufirst%3DJ.%2BM.%26aulast%3DCoutsias%26aufirst%3DE.%26aulast%3DCorrenti%26aufirst%3DC.%2BE.%26aulast%3DSzyperski%26aufirst%3DT.%26aulast%3DCraik%26aufirst%3DD.%2BJ.%26aulast%3DBaker%26aufirst%3DD.%26atitle%3DAccurate%2520de%2520Novo%2520Design%2520of%2520Hyperstable%2520Constrained%2520Peptides%26jtitle%3DNature%26date%3D2016%26volume%3D538%26spage%3D329%26epage%3D335%26doi%3D10.1038%2Fnature19791" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref147"><div class="reference"><strong class="refLabel"><a href="#ref147" class="rightTabRefNumLink">147</a></strong><div class="NLM_citation" id="rightTab-cit147"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Namoto, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sirockin, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sellner, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wiesmann, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Villard, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moreau, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valeur, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paulding, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schleeger, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schipp, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loup, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andrews, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Swale, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robinson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farady, C. J.</span></span> <span> </span><span class="NLM_article-title">Structure-Based Design and Synthesis of Macrocyclic Human Rhinovirus 3C Protease Inhibitors</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>28</i></span>,  <span class="NLM_fpage">906</span>– <span class="NLM_lpage">909</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2018.01.064</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref147/cit147&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1016%2Fj.bmcl.2018.01.064" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2018&pages=906-909&author=K.+Namotoauthor=F.+Sirockinauthor=H.+Sellnerauthor=C.+Wiesmannauthor=F.+Villardauthor=R.+J.+Moreauauthor=E.+Valeurauthor=S.+C.+Pauldingauthor=S.+Schleegerauthor=K.+Schippauthor=J.+Loupauthor=L.+Andrewsauthor=R.+Swaleauthor=M.+Robinsonauthor=C.+J.+Farady&title=Structure-Based+Design+and+Synthesis+of+Macrocyclic+Human+Rhinovirus+3C+Protease+Inhibitors&doi=10.1016%2Fj.bmcl.2018.01.064"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit147&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2018.01.064&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2018.01.064%26sid%3Dliteratum%253Aachs%26aulast%3DNamoto%26aufirst%3DK.%26aulast%3DSirockin%26aufirst%3DF.%26aulast%3DSellner%26aufirst%3DH.%26aulast%3DWiesmann%26aufirst%3DC.%26aulast%3DVillard%26aufirst%3DF.%26aulast%3DMoreau%26aufirst%3DR.%2BJ.%26aulast%3DValeur%26aufirst%3DE.%26aulast%3DPaulding%26aufirst%3DS.%2BC.%26aulast%3DSchleeger%26aufirst%3DS.%26aulast%3DSchipp%26aufirst%3DK.%26aulast%3DLoup%26aufirst%3DJ.%26aulast%3DAndrews%26aufirst%3DL.%26aulast%3DSwale%26aufirst%3DR.%26aulast%3DRobinson%26aufirst%3DM.%26aulast%3DFarady%26aufirst%3DC.%2BJ.%26atitle%3DStructure-Based%2520Design%2520and%2520Synthesis%2520of%2520Macrocyclic%2520Human%2520Rhinovirus%25203C%2520Protease%2520Inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2018%26volume%3D28%26spage%3D906%26epage%3D909%26doi%3D10.1016%2Fj.bmcl.2018.01.064" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref148"><div class="reference"><strong class="refLabel"><a href="#ref148" class="rightTabRefNumLink">148</a></strong><div class="NLM_citation" id="rightTab-cit148"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bird, G. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mazzola, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Opoku-Nsiah, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lammert, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Godes, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neuberg, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walensky, L. D.</span></span> <span> </span><span class="NLM_article-title">Biophysical Determinants for Cellular Uptake of Hydrocarbon-Stapled Peptide Helices</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">845</span>– <span class="NLM_lpage">852</span>, <span class="refDoi"> DOI: 10.1038/nchembio.2153</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref148/cit148&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1038%2Fnchembio.2153" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref148/cit148&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=27547919" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref148/cit148&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtlyqsbbP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2016&pages=845-852&author=G.+H.+Birdauthor=E.+Mazzolaauthor=K.+Opoku-Nsiahauthor=M.+A.+Lammertauthor=M.+Godesauthor=D.+S.+Neubergauthor=L.+D.+Walensky&title=Biophysical+Determinants+for+Cellular+Uptake+of+Hydrocarbon-Stapled+Peptide+Helices&doi=10.1038%2Fnchembio.2153"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit148R"><div class="casContent"><span class="casTitleNuber">148</span><div class="casTitle"><span class="NLM_cas:atitle">Biophysical determinants for cellular uptake of hydrocarbon-stapled peptide helices</span></div><div class="casAuthors">Bird, Gregory H.; Mazzola, Emanuele; Opoku-Nsiah, Kwadwo; Lammert, Margaret A.; Godes, Marina; Neuberg, Donna S.; Walensky, Loren D.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">845-852</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Hydrocarbon-stapled peptides are a class of bioactive alpha-helical ligands developed to dissect and target protein interactions.  While there is consensus that stapled peptides can be effective chem. tools for investigating protein regulation, their broader utility for therapeutic modulation of intracellular interactions remains an active area of study.  In particular, the design principles for generating cell-permeable stapled peptides are empiric, yet consistent intracellular access is essential to in vivo application.  Here, we used an unbiased statistical approach to det. which biophys. parameters dictate the uptake of stapled-peptide libraries.  We found that staple placement at the amphipathic boundary combined with optimal hydrophobic and helical content are the key drivers of cellular uptake, whereas excess hydrophobicity and pos. charge at isolated amino acid positions can trigger membrane lysis at elevated peptide dosing.  Our results provide a design roadmap for maximizing the potential to generate cell-permeable stapled peptides with on-mechanism cellular activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGowcU10JcijIbVg90H21EOLACvtfcHk0liQw2oewmI_og"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtlyqsbbP&md5=f31d10106beebfe08f69ec9a1b215ec1</span></div><a href="/servlet/linkout?suffix=cit148&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.2153&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.2153%26sid%3Dliteratum%253Aachs%26aulast%3DBird%26aufirst%3DG.%2BH.%26aulast%3DMazzola%26aufirst%3DE.%26aulast%3DOpoku-Nsiah%26aufirst%3DK.%26aulast%3DLammert%26aufirst%3DM.%2BA.%26aulast%3DGodes%26aufirst%3DM.%26aulast%3DNeuberg%26aufirst%3DD.%2BS.%26aulast%3DWalensky%26aufirst%3DL.%2BD.%26atitle%3DBiophysical%2520Determinants%2520for%2520Cellular%2520Uptake%2520of%2520Hydrocarbon-Stapled%2520Peptide%2520Helices%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2016%26volume%3D12%26spage%3D845%26epage%3D852%26doi%3D10.1038%2Fnchembio.2153" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref149"><div class="reference"><strong class="refLabel"><a href="#ref149" class="rightTabRefNumLink">149</a></strong><div class="NLM_citation" id="rightTab-cit149"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Craik, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daly, N. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bond, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waine, C.</span></span> <span> </span><span class="NLM_article-title">Plant Cyclotides: A Unique Family of Cyclic and Knotted Proteins That Defines the Cyclic Cystine Knot Structural motif1 1Edited by P. E. Wright</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>294</i></span>,  <span class="NLM_fpage">1327</span>– <span class="NLM_lpage">1336</span>, <span class="refDoi"> DOI: 10.1006/jmbi.1999.3383</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref149/cit149&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1006%2Fjmbi.1999.3383" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref149/cit149&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10600388" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref149/cit149&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADyaK1MXnvFaqsLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=294&publication_year=1999&pages=1327-1336&author=D.+J.+Craikauthor=N.+L.+Dalyauthor=T.+Bondauthor=C.+Waine&title=Plant+Cyclotides%3A+A+Unique+Family+of+Cyclic+and+Knotted+Proteins+That+Defines+the+Cyclic+Cystine+Knot+Structural+motif1+1Edited+by+P.+E.+Wright&doi=10.1006%2Fjmbi.1999.3383"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit149R"><div class="casContent"><span class="casTitleNuber">149</span><div class="casTitle"><span class="NLM_cas:atitle">Plant Cyclotides: A Unique Family of Cyclic and Knotted Proteins that Defines the Cyclic Cystine Knot Structural Motif</span></div><div class="casAuthors">Craik, David J.; Daly, Norelle L.; Bond, Trudy; Waine, Clement</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">294</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1327-1336</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Academic Press</span>)
        </div><div class="casAbstract">Several macrocyclic peptides (∼30 amino acids), with diverse biol. activities, have been isolated from the Rubiaceae and Violaceae plant families over recent years.  We have significantly expanded the range of known macrocyclic peptides with the discovery of 16 novel peptides from exts. of Viola hederaceae, Viola odorata and Oldenlandia affinis.  The Viola plants had not previously been examd. for these peptides and thus represent novel species in which these unusual macrocyclic peptides are produced.  Further, we have detd. the three-dimensional structure of one of these novel peptides, cycloviolacin O1, using 1H NMR spectroscopy.  The structure consists of a distorted triple-stranded β-sheet and a cystine-knot arrangement of the disulfide bonds.  This structure is similar to kalata B1 and circulin A, the only two macrocyclic peptides for which a structure was available, suggesting that despite the sequence variation throughout the peptides they form a family in which the overall fold is conserved.  We refer to these peptides as the cyclotide family and their embedded topol. as the cyclic cystine knot (CCK) motif.  The unique cyclic and knotted nature of these mols. makes them a fascinating example of topol. complex proteins.  Examn. of the sequences reveals they can be sepd. into two subfamilies, one of which tends to contain a larger no. of pos. charged residues and has a bracelet-like circularization of the backbone.  The second subfamily contains a backbone twist due to a cis-Pro peptide bond and may conceptually be regarded as a mol. Moebius strip.  Here we define the structural features of the two apparent subfamilies of the CCK peptides which may be significant for the likely defense related role of these peptides within plants.  (c) 1999 Academic Press.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrOe8K8-Up5fLVg90H21EOLACvtfcHk0lh7FDEvufwN1g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXnvFaqsLs%253D&md5=5c27c83a6ed1896821d3c14a74370e75</span></div><a href="/servlet/linkout?suffix=cit149&amp;dbid=16384&amp;doi=10.1006%2Fjmbi.1999.3383&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1006%252Fjmbi.1999.3383%26sid%3Dliteratum%253Aachs%26aulast%3DCraik%26aufirst%3DD.%2BJ.%26aulast%3DDaly%26aufirst%3DN.%2BL.%26aulast%3DBond%26aufirst%3DT.%26aulast%3DWaine%26aufirst%3DC.%26atitle%3DPlant%2520Cyclotides%253A%2520A%2520Unique%2520Family%2520of%2520Cyclic%2520and%2520Knotted%2520Proteins%2520That%2520Defines%2520the%2520Cyclic%2520Cystine%2520Knot%2520Structural%2520motif1%25201Edited%2520by%2520P.%2520E.%2520Wright%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D1999%26volume%3D294%26spage%3D1327%26epage%3D1336%26doi%3D10.1006%2Fjmbi.1999.3383" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref150"><div class="reference"><strong class="refLabel"><a href="#ref150" class="rightTabRefNumLink">150</a></strong><div class="NLM_citation" id="rightTab-cit150"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Colgrave, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Korsinczky, M. J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clark, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Foley, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Craik, D. J.</span></span> <span> </span><span class="NLM_article-title">Sunflower Trypsin Inhibitor-1, Proteolytic Studies on a Trypsin Inhibitor Peptide and Its Analogs</span>. <i>Biopolymers</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>94</i></span>,  <span class="NLM_fpage">665</span>– <span class="NLM_lpage">672</span>, <span class="refDoi"> DOI: 10.1002/bip.21415</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref150/cit150&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1002%2Fbip.21415" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref150/cit150&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=20564016" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref150/cit150&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC3cXht12qtr%252FE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=94&publication_year=2010&pages=665-672&author=M.+L.+Colgraveauthor=M.+J.+L.+Korsinczkyauthor=R.+J.+Clarkauthor=F.+Foleyauthor=D.+J.+Craik&title=Sunflower+Trypsin+Inhibitor-1%2C+Proteolytic+Studies+on+a+Trypsin+Inhibitor+Peptide+and+Its+Analogs&doi=10.1002%2Fbip.21415"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit150R"><div class="casContent"><span class="casTitleNuber">150</span><div class="casTitle"><span class="NLM_cas:atitle">Sunflower trypsin inhibitor-1, proteolytic studies on a trypsin inhibitor peptide and its analogs</span></div><div class="casAuthors">Colgrave, Michelle L.; Korsinczky, Michael J. L.; Clark, Richard J.; Foley, Fiona; Craik, David J.</div><div class="citationInfo"><span class="NLM_cas:title">Biopolymers</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">94</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">665-672</span>CODEN:
                <span class="NLM_cas:coden">BIPMAA</span>;
        ISSN:<span class="NLM_cas:issn">0006-3525</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">Sunflower trypsin inhibitor-1 (SFTI-1) is a 14 amino acid cyclic peptide from sunflower seeds, which possesses exceptionally potent trypsin-inhibitory activity, and has promise as a stable peptide-based drug template.  Within its compact structure, SFTI-1 combines a head-to-tail cyclized backbone and a disulfide bond.  In this study, we synthesized a range of acyclic and disulfide-deficient analogs of SFTI-1 to investigate enzyme-assisted cyclization of the peptide backbone and proteolytic degrdn. that occurs as a result of incubation with trypsin.  Electrospray and matrix-assisted laser desorption ionization mass spectrometry allowed the characterization of a range of novel degrdn. products and elucidation of the time-course for cyclization and/or proteolysis.  Trypsin displayed the ability to resynthesize the scissile bond(s) and hence cyclize two of the linear permutants, whereas irreversible degrdn. was obsd. for another two permutants.  An interesting ring contraction mediated by trypsin was obsd., supporting a role for protease catalyzed splicing as a way of increasing the combinatorial diversity of cyclic peptides in nature.  Disulfide-deficient mutants were degraded within minutes, emphasizing the crit. role of the cysteine bridge in maintaining proteolytic stability of SFTI-1.  Overall, the study provides addnl. support for the proposal that naturally occurring cyclic peptides like SFTI-1 are biosynthesized by proteolytic enzymes effectively catalyzing the reverse of their normal reaction to make, rather than break peptide bonds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqqlJnJbomu87Vg90H21EOLACvtfcHk0lh7FDEvufwN1g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXht12qtr%252FE&md5=d709396ac3d3c319fbcc8eebf44cac85</span></div><a href="/servlet/linkout?suffix=cit150&amp;dbid=16384&amp;doi=10.1002%2Fbip.21415&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fbip.21415%26sid%3Dliteratum%253Aachs%26aulast%3DColgrave%26aufirst%3DM.%2BL.%26aulast%3DKorsinczky%26aufirst%3DM.%2BJ.%2BL.%26aulast%3DClark%26aufirst%3DR.%2BJ.%26aulast%3DFoley%26aufirst%3DF.%26aulast%3DCraik%26aufirst%3DD.%2BJ.%26atitle%3DSunflower%2520Trypsin%2520Inhibitor-1%252C%2520Proteolytic%2520Studies%2520on%2520a%2520Trypsin%2520Inhibitor%2520Peptide%2520and%2520Its%2520Analogs%26jtitle%3DBiopolymers%26date%3D2010%26volume%3D94%26spage%3D665%26epage%3D672%26doi%3D10.1002%2Fbip.21415" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref151"><div class="reference"><strong class="refLabel"><a href="#ref151" class="rightTabRefNumLink">151</a></strong><div class="NLM_citation" id="rightTab-cit151"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Muttenthaler, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Akondi, K. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alewood, P. F.</span></span> <span> </span><span class="NLM_article-title">Structure-Activity Studies on Alpha-Conotoxins</span>. <i>Curr. Pharm. Des.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">4226</span>– <span class="NLM_lpage">4241</span>, <span class="refDoi"> DOI: 10.2174/138161211798999384</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref151/cit151&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.2174%2F138161211798999384" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref151/cit151&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=22204424" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref151/cit151&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC38XovFajsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2011&pages=4226-4241&author=M.+Muttenthalerauthor=K.+B.+Akondiauthor=P.+F.+Alewood&title=Structure-Activity+Studies+on+Alpha-Conotoxins&doi=10.2174%2F138161211798999384"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit151R"><div class="casContent"><span class="casTitleNuber">151</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-activity studies on alpha-conotoxins</span></div><div class="casAuthors">Muttenthaler, Markus; Akondi, Kalyana B.; Alewood, Paul F.</div><div class="citationInfo"><span class="NLM_cas:title">Current Pharmaceutical Design</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">38</span>),
    <span class="NLM_cas:pages">4226-4241</span>CODEN:
                <span class="NLM_cas:coden">CPDEFP</span>;
        ISSN:<span class="NLM_cas:issn">1381-6128</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Conotoxins are small bioactive highly structured peptides from the venom of marine cone snails (genus Conus).  Over the past 50 million years these molluscs have developed a complex venom cocktail for each species that is comprised of 100-2000 distinct cysteine-rich peptides for prey capture and defense.  This review focuses on an important and well-studied class of conotoxins, the α-conotoxins.  These α-conotoxins are potent and selective antagonists of various subtypes of the nicotinic acetylcholine receptors (nAChRs).  Key structure-activity relationship studies are presented to illustrate the common motifs, structural features and pharmacophores that define this interesting peptide class.  Addnl., their synthesis, chem. modifications, the development of more selective and stable analogs and their therapeutic potential are discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrLz6PhaCubtLVg90H21EOLACvtfcHk0lh7FDEvufwN1g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XovFajsw%253D%253D&md5=34aa49c05aa9cc30a4df1545e9cc76fd</span></div><a href="/servlet/linkout?suffix=cit151&amp;dbid=16384&amp;doi=10.2174%2F138161211798999384&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F138161211798999384%26sid%3Dliteratum%253Aachs%26aulast%3DMuttenthaler%26aufirst%3DM.%26aulast%3DAkondi%26aufirst%3DK.%2BB.%26aulast%3DAlewood%26aufirst%3DP.%2BF.%26atitle%3DStructure-Activity%2520Studies%2520on%2520Alpha-Conotoxins%26jtitle%3DCurr.%2520Pharm.%2520Des.%26date%3D2011%26volume%3D17%26spage%3D4226%26epage%3D4241%26doi%3D10.2174%2F138161211798999384" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref152"><div class="reference"><strong class="refLabel"><a href="#ref152" class="rightTabRefNumLink">152</a></strong><div class="NLM_citation" id="rightTab-cit152"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Craik, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Du, J.</span></span> <span> </span><span class="NLM_article-title">Cyclotides as Drug Design Scaffolds</span>. <i>Curr. Opin. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>38</i></span>,  <span class="NLM_fpage">8</span>– <span class="NLM_lpage">16</span>, <span class="refDoi"> DOI: 10.1016/j.cbpa.2017.01.018</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref152/cit152&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1016%2Fj.cbpa.2017.01.018" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref152/cit152&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=28249194" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref152/cit152&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC2sXjsFGksLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=2017&pages=8-16&author=D.+J.+Craikauthor=J.+Du&title=Cyclotides+as+Drug+Design+Scaffolds&doi=10.1016%2Fj.cbpa.2017.01.018"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit152R"><div class="casContent"><span class="casTitleNuber">152</span><div class="casTitle"><span class="NLM_cas:atitle">Cyclotides as drug design scaffolds</span></div><div class="casAuthors">Craik, David J.; Du, Junqiao</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Chemical Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">8-16</span>CODEN:
                <span class="NLM_cas:coden">COCBF4</span>;
        ISSN:<span class="NLM_cas:issn">1367-5931</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Cyclotides are ultra-stable peptides derived from plants.  They are around 30 amino acids in size and are characterized by their head-to-tail cyclic backbone and cystine knot.  Their exceptional stability and tolerance to sequence substitutions has led to their use as frameworks in drug design.  This article describes recent developments in this field, particularly developments over the last two years relating to the grafting of bioactive peptide sequences into the cyclic cystine knot framework of cyclotides to stabilize the sequences.  Grafted cyclotides have now been developed that interact with protein or enzyme targets, both extracellular and intracellular, as well as with cell surface receptors and membranes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpW8Rb0hr6lQbVg90H21EOLACvtfcHk0lh7FDEvufwN1g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXjsFGksLk%253D&md5=e3c59d4eacaa4b08a43e33a9986ff2eb</span></div><a href="/servlet/linkout?suffix=cit152&amp;dbid=16384&amp;doi=10.1016%2Fj.cbpa.2017.01.018&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cbpa.2017.01.018%26sid%3Dliteratum%253Aachs%26aulast%3DCraik%26aufirst%3DD.%2BJ.%26aulast%3DDu%26aufirst%3DJ.%26atitle%3DCyclotides%2520as%2520Drug%2520Design%2520Scaffolds%26jtitle%3DCurr.%2520Opin.%2520Chem.%2520Biol.%26date%3D2017%26volume%3D38%26spage%3D8%26epage%3D16%26doi%3D10.1016%2Fj.cbpa.2017.01.018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref153"><div class="reference"><strong class="refLabel"><a href="#ref153" class="rightTabRefNumLink">153</a></strong><div class="NLM_citation" id="rightTab-cit153"><span><span class="NLM_contrib-group"><span class="NLM_string-name">De Veer, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weidmann, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Craik, D. J.</span></span> <span> </span><span class="NLM_article-title">Cyclotides as Tools in Chemical Biology</span>. <i>Acc. Chem. Res.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>50</i></span>,  <span class="NLM_fpage">1557</span>– <span class="NLM_lpage">1565</span>, <span class="refDoi"> DOI: 10.1021/acs.accounts.7b00157</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.accounts.7b00157" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref153/cit153&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtVCgsL7N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2017&pages=1557-1565&author=S.+J.+De+Veerauthor=J.+Weidmannauthor=D.+J.+Craik&title=Cyclotides+as+Tools+in+Chemical+Biology&doi=10.1021%2Facs.accounts.7b00157"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit153R"><div class="casContent"><span class="casTitleNuber">153</span><div class="casTitle"><span class="NLM_cas:atitle">Cyclotides as Tools in Chemical Biology</span></div><div class="casAuthors">de Veer, Simon J.; Weidmann, Joachim; Craik, David J.</div><div class="citationInfo"><span class="NLM_cas:title">Accounts of Chemical Research</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1557-1565</span>CODEN:
                <span class="NLM_cas:coden">ACHRE4</span>;
        ISSN:<span class="NLM_cas:issn">0001-4842</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Among the various mols. that plants produce for defense against pests and pathogens, cyclotides stand out as exceptionally stable and structurally unique.  These ribosomally synthesized peptides are around 30 amino acids in size, and are stabilized by a head-to-tail cyclic peptide backbone and three disulfide bonds that form a cystine knot.  They occur in certain plants of the Rubiaceae, Violaceae, Cucurbitaceae, Fabaceae, and Solanaceae families, with an individual plant producing up to hundreds of different cyclotides.  Aside from being exploitable as crop protection agents based on their natural pesticidal activities, cyclotides are amenable to repurposing by chemists for use as drug leads or as tools in chem. biol.  Their macrocyclic peptide backbone and knotted arrangement of three disulfide bonds engenders cyclotides with resistance to proteolytic degrdn., high temps., and chem. chaotropes.  Furthermore, their small size makes them accessible to synthesis using solid-phase peptide chem. and so nonnatural cyclotides can be designed and synthesized for a variety of applications.  The focus here is on cyclotides as tools in chem. biol., and there are four main areas of application that have appeared in the literature so far: (i) cyclotides as probes of membrane binding; (ii) cyclotides as probes of biosynthetic pathways for peptide cyclization; (iii) cyclotides as probes of protease specificity and function; and (iv) cyclotides as probes of receptor binding and specificity, with the potential for them to be developed as drug leads.  The main methods used in these studies include solid-phase peptide chem. for synthesis and NMR spectroscopy for structural characterization, as well as a wide range of biochem. and biophys. techniques for probing intermol. interactions.  In addn., cyclotides have been examd. in diverse biol. assays, ranging from enzyme inhibition to cell penetration, intracellular targeting and cytotoxicity.  The main finding to have emerged from studies over the past decade is that cyclotides are exceptionally stable under a variety of conditions (in assay buffers, biol. fluids, membranes, and recombinant expression systems).  Furthermore, they are structurally very well-defined and amenable to sequence substitutions that can introduce new desired biol. activities, generally without compromising their exceptional stability.  Both features contribute to their use as peptide-based frameworks in drug design.  Finally, they occupy a size niche between traditional small-mol. drugs (<500 Da in mol. wt.) and protein-based biologics (>5000 Da) and thus can probe receptors, membranes, and protein-protein interactions in different ways to what is possible with either small mols. or biologics.  Overall, cyclotides are an exciting class of peptides that have great potential as ultrastable chem. biol. probes in a variety of applications.  They have the advantage of specificity (typical of proteins) combined with the synthetic accessibility of small mols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqkJnwpjFMOVrVg90H21EOLACvtfcHk0lhNx10e4J5z3A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtVCgsL7N&md5=18517c515d7b5018fc7744f534d5a4e2</span></div><a href="/servlet/linkout?suffix=cit153&amp;dbid=16384&amp;doi=10.1021%2Facs.accounts.7b00157&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.accounts.7b00157%26sid%3Dliteratum%253Aachs%26aulast%3DDe%2BVeer%26aufirst%3DS.%2BJ.%26aulast%3DWeidmann%26aufirst%3DJ.%26aulast%3DCraik%26aufirst%3DD.%2BJ.%26atitle%3DCyclotides%2520as%2520Tools%2520in%2520Chemical%2520Biology%26jtitle%3DAcc.%2520Chem.%2520Res.%26date%3D2017%26volume%3D50%26spage%3D1557%26epage%3D1565%26doi%3D10.1021%2Facs.accounts.7b00157" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref154"><div class="reference"><strong class="refLabel"><a href="#ref154" class="rightTabRefNumLink">154</a></strong><div class="NLM_citation" id="rightTab-cit154"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Camarero, J. A.</span></span> <span> </span><span class="NLM_article-title">Cyclotides, a Versatile Ultrastable Micro-Protein Scaffold for Biotechnological Applications</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">5089</span>– <span class="NLM_lpage">5099</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2017.10.051</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref154/cit154&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1016%2Fj.bmcl.2017.10.051" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref154/cit154&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=29110985" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref154/cit154&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhslemsLrO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2017&pages=5089-5099&author=J.+A.+Camarero&title=Cyclotides%2C+a+Versatile+Ultrastable+Micro-Protein+Scaffold+for+Biotechnological+Applications&doi=10.1016%2Fj.bmcl.2017.10.051"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit154R"><div class="casContent"><span class="casTitleNuber">154</span><div class="casTitle"><span class="NLM_cas:atitle">Cyclotides, a versatile ultrastable micro-protein scaffold for biotechnological applications</span></div><div class="casAuthors">Camarero, Julio A.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">5089-5099</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Cyclotides are fascinating microproteins (≈30-40 residues long) with a unique head-to-tail cyclized backbone, stabilized by three disulfide bonds forming a cystine knot.  This unique topol. makes them exceptionally stable to chem., thermal and biol. degrdn. compared to other peptides of similar size.  Cyclotides have been also found to be highly tolerant to sequence variability, aside from the conserved residues forming the cystine knot, able to cross cellular membranes and modulate intracellular protein-protein interactions both in vitro and in vivo.  These properties make them ideal scaffolds for many biotechnol. applications.  This article provides and overview of the properties of cyclotides and their applications as mol. imaging agents and peptide-based therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoBl9-w1yHZYbVg90H21EOLACvtfcHk0lhNx10e4J5z3A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhslemsLrO&md5=3e5aa9896809895d04eb1cc95d949498</span></div><a href="/servlet/linkout?suffix=cit154&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2017.10.051&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2017.10.051%26sid%3Dliteratum%253Aachs%26aulast%3DCamarero%26aufirst%3DJ.%2BA.%26atitle%3DCyclotides%252C%2520a%2520Versatile%2520Ultrastable%2520Micro-Protein%2520Scaffold%2520for%2520Biotechnological%2520Applications%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2017%26volume%3D27%26spage%3D5089%26epage%3D5099%26doi%3D10.1016%2Fj.bmcl.2017.10.051" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref155"><div class="reference"><strong class="refLabel"><a href="#ref155" class="rightTabRefNumLink">155</a></strong><div class="NLM_citation" id="rightTab-cit155"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Eliasen, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daly, N. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wulff, B. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andresen, T. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Conde-Frieboes, K. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Craik, D. J.</span></span> <span> </span><span class="NLM_article-title">Design, Synthesis, Structural and Functional Characterization of Novel Melanocortin Agonists Based on the Cyclotide Kalata B1</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>287</i></span>,  <span class="NLM_fpage">40493</span>– <span class="NLM_lpage">40501</span>, <span class="refDoi"> DOI: 10.1074/jbc.M112.395442</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref155/cit155&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1074%2Fjbc.M112.395442" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref155/cit155&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=23012369" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref155/cit155&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC38XhslansbbF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=287&publication_year=2012&pages=40493-40501&author=R.+Eliasenauthor=N.+L.+Dalyauthor=B.+S.+Wulffauthor=T.+L.+Andresenauthor=K.+W.+Conde-Frieboesauthor=D.+J.+Craik&title=Design%2C+Synthesis%2C+Structural+and+Functional+Characterization+of+Novel+Melanocortin+Agonists+Based+on+the+Cyclotide+Kalata+B1&doi=10.1074%2Fjbc.M112.395442"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit155R"><div class="casContent"><span class="casTitleNuber">155</span><div class="casTitle"><span class="NLM_cas:atitle">Design, Synthesis, Structural and Functional Characterization of Novel Melanocortin Agonists Based on the Cyclotide Kalata B1</span></div><div class="casAuthors">Eliasen, Rasmus; Daly, Norelle L.; Wulff, Birgitte S.; Andresen, Thomas L.; Conde-Frieboes, Kilian W.; Craik, David J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">287</span>
        (<span class="NLM_cas:issue">48</span>),
    <span class="NLM_cas:pages">40493-40501</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Obesity is an increasingly important global health problem that lacks current treatment options.  The melanocortin receptor 4 (MC4R) is a target for obesity therapies because its activation triggers appetite suppression and increases energy expenditure.  Cyclotides have been suggested as scaffolds for the insertion and stabilization of pharmaceutically active peptides.  In this study, we explored the development of appetite-reducing peptides by synthesizing MC4R agonists based on the insertion of the His-Phe-Arg-Trp sequence into the cyclotide kalata B1.  The ability of the analogs to fold similarly to kalata B1 but display MC4R activity were investigated.  Four peptides were synthesized using t-butoxycarbonyl peptide chem. with a C-terminal thioester to facilitate backbone cyclization.  The structures of the peptides were found to be similar to kalata B1, evaluated by Hα NMR chem. shifts.  KB1(GHFRWG;23-28) had a Ki of 29 nm at the MC4R and was 107 or 314 times more selective over this receptor than MC1R or MC5R, resp., and had no detectable binding to MC3R.  The peptide had higher affinity for the MC4R than the endogenous agonist, α-melanocyte stimulation hormone, but it was less potent at the MC4R, with an EC50 of 580 nm for activation of the MC4R.  In conclusion, we synthesized melanocortin analogs of kalata B1 that preserve the structural scaffold and display receptor binding and functional activity.  KB1(GHFRWG;23-28) is potent and selective for the MC4R.  This compd. validates the use of cyclotides as scaffolds and has the potential to be a new lead for the treatment of obesity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq_sNylzkkWFLVg90H21EOLACvtfcHk0lhNx10e4J5z3A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhslansbbF&md5=49425ad83d5075c1e375ee1fec3849ec</span></div><a href="/servlet/linkout?suffix=cit155&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M112.395442&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M112.395442%26sid%3Dliteratum%253Aachs%26aulast%3DEliasen%26aufirst%3DR.%26aulast%3DDaly%26aufirst%3DN.%2BL.%26aulast%3DWulff%26aufirst%3DB.%2BS.%26aulast%3DAndresen%26aufirst%3DT.%2BL.%26aulast%3DConde-Frieboes%26aufirst%3DK.%2BW.%26aulast%3DCraik%26aufirst%3DD.%2BJ.%26atitle%3DDesign%252C%2520Synthesis%252C%2520Structural%2520and%2520Functional%2520Characterization%2520of%2520Novel%2520Melanocortin%2520Agonists%2520Based%2520on%2520the%2520Cyclotide%2520Kalata%2520B1%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2012%26volume%3D287%26spage%3D40493%26epage%3D40501%26doi%3D10.1074%2Fjbc.M112.395442" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref156"><div class="reference"><strong class="refLabel"><a href="#ref156" class="rightTabRefNumLink">156</a></strong><div class="NLM_citation" id="rightTab-cit156"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ji, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Majumder, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Millard, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Borra, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bi, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elnagar, A. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neamati, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shekhtman, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Camarero, J. A.</span></span> <span> </span><span class="NLM_article-title">In Vivo Activation of the p53 Tumor Suppressor Pathway by an Engineered Cyclotide</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>135</i></span>,  <span class="NLM_fpage">11623</span>– <span class="NLM_lpage">11633</span>, <span class="refDoi"> DOI: 10.1021/ja405108p</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja405108p" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref156/cit156&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtFSis7nL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=135&publication_year=2013&pages=11623-11633&author=Y.+Jiauthor=S.+Majumderauthor=M.+Millardauthor=R.+Borraauthor=T.+Biauthor=A.+Y.+Elnagarauthor=N.+Neamatiauthor=A.+Shekhtmanauthor=J.+A.+Camarero&title=In+Vivo+Activation+of+the+p53+Tumor+Suppressor+Pathway+by+an+Engineered+Cyclotide&doi=10.1021%2Fja405108p"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit156R"><div class="casContent"><span class="casTitleNuber">156</span><div class="casTitle"><span class="NLM_cas:atitle">In Vivo Activation of the p53 Tumor Suppressor Pathway by an Engineered Cyclotide</span></div><div class="casAuthors">Ji, Yanbin; Majumder, Subhabrata; Millard, Melissa; Borra, Radhika; Bi, Tao; Elnagar, Ahmed Y.; Neamati, Nouri; Shekhtman, Alexander; Camarero, Julio A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">135</span>
        (<span class="NLM_cas:issue">31</span>),
    <span class="NLM_cas:pages">11623-11633</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The overexpression of Hdm2 and HdmX is a common mechanism used by many tumor cells to inactive the p53 tumor suppressor pathway promoting cell survival.  Targeting Hdm2 and HdmX has emerged as a validated therapeutic strategy for treating cancers with wild-type p53.  Small linear peptides mimicking the N-terminal fragment of p53 have been shown to be potent Hdm2/HdmX antagonists.  The potential therapeutic use of these peptides, however, is limited by their poor stability and bioavailability.  Here, the authors report the engineering of the cyclotide MCoTI-I to efficiently antagonize intracellular p53 degrdn.  The resulting cyclotide MCo-PMI was able to bind with low nanomolar affinity to both Hdm2 and HdmX, showed high stability in human serum, and was cytotoxic to wild-type p53 cancer cell lines by activating the p53 tumor suppressor pathway both in vitro and in vivo.  These features make the cyclotide MCoTI-I an optimal scaffold for targeting intracellular protein-protein interactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGra-dIq75V8XLVg90H21EOLACvtfcHk0liadiKWssFAEQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtFSis7nL&md5=c2d41067e876cb97bcf3a5a21bebce17</span></div><a href="/servlet/linkout?suffix=cit156&amp;dbid=16384&amp;doi=10.1021%2Fja405108p&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja405108p%26sid%3Dliteratum%253Aachs%26aulast%3DJi%26aufirst%3DY.%26aulast%3DMajumder%26aufirst%3DS.%26aulast%3DMillard%26aufirst%3DM.%26aulast%3DBorra%26aufirst%3DR.%26aulast%3DBi%26aufirst%3DT.%26aulast%3DElnagar%26aufirst%3DA.%2BY.%26aulast%3DNeamati%26aufirst%3DN.%26aulast%3DShekhtman%26aufirst%3DA.%26aulast%3DCamarero%26aufirst%3DJ.%2BA.%26atitle%3DIn%2520Vivo%2520Activation%2520of%2520the%2520p53%2520Tumor%2520Suppressor%2520Pathway%2520by%2520an%2520Engineered%2520Cyclotide%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2013%26volume%3D135%26spage%3D11623%26epage%3D11633%26doi%3D10.1021%2Fja405108p" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref157"><div class="reference"><strong class="refLabel"><a href="#ref157" class="rightTabRefNumLink">157</a></strong><div class="NLM_citation" id="rightTab-cit157"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mullard, A.</span></span> <span> </span><span class="NLM_article-title">The Drug-Maker’s Guide to the Galaxy</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>549</i></span>,  <span class="NLM_fpage">445</span>– <span class="NLM_lpage">447</span>, <span class="refDoi"> DOI: 10.1038/549445a</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref157/cit157&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1038%2F549445a" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref157/cit157&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=28959982" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref157/cit157&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A280%3ADC%252BC1M%252FjsFSmtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=549&publication_year=2017&pages=445-447&author=A.+Mullard&title=The+Drug-Maker%E2%80%99s+Guide+to+the+Galaxy&doi=10.1038%2F549445a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit157R"><div class="casContent"><span class="casTitleNuber">157</span><div class="casTitle"><span class="NLM_cas:atitle">The drug-maker's guide to the galaxy</span></div><div class="casAuthors">Mullard Asher</div><div class="citationInfo"><span class="NLM_cas:title">Nature</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">549</span>
        (<span class="NLM_cas:issue">7673</span>),
    <span class="NLM_cas:pages">445-447</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSItGWG-TCuJEg6wdEZmT2rfW6udTcc2ebSkYKSlzu3Mrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1M%252FjsFSmtA%253D%253D&md5=4ff8347906c8f3b459d05a5fe7ad0b56</span></div><a href="/servlet/linkout?suffix=cit157&amp;dbid=16384&amp;doi=10.1038%2F549445a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F549445a%26sid%3Dliteratum%253Aachs%26aulast%3DMullard%26aufirst%3DA.%26atitle%3DThe%2520Drug-Maker%25E2%2580%2599s%2520Guide%2520to%2520the%2520Galaxy%26jtitle%3DNature%26date%3D2017%26volume%3D549%26spage%3D445%26epage%3D447%26doi%3D10.1038%2F549445a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref158"><div class="reference"><strong class="refLabel"><a href="#ref158" class="rightTabRefNumLink">158</a></strong><div class="NLM_citation" id="rightTab-cit158"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pertusi, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Donnell, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Homsher, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Solly, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stahler, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riley, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Finley, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Finger, E. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adam, G. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bell</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zuck, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hudak, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weber, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nothstein, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Locco, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quinn, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amoss, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Squadroni, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hartnett, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heo, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">White, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">May, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boots, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roberts, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cocchiarella, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wolicki, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kreamer, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kutchukian, P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wassermann, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uebele, V. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glick, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rusinko, A.,  III</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Culberson, J. C.</span></span> <span> </span><span class="NLM_article-title">Prospective Assessment of Virtual Screening Heuristics Derived Using a Novel Fusion Score</span>. <i>SLAS Discovery</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">995</span>– <span class="NLM_lpage">1006</span>, <span class="refDoi"> DOI: 10.1177/2472555217706058</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref158/cit158&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1177%2F2472555217706058" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2017&pages=995-1006&author=D.+A.+Pertusiauthor=G.+O%E2%80%99Donnellauthor=M.+F.+Homsherauthor=K.+Sollyauthor=A.+Patelauthor=S.+L.+Stahlerauthor=D.+Rileyauthor=M.+F.+Finleyauthor=E.+N.+Fingerauthor=G.+C.+Adamauthor=J.+Mengauthor=+Bellauthor=P.+D.+Zuckauthor=E.+M.+Hudakauthor=M.+J.+Weberauthor=J.+E.+Nothsteinauthor=L.+Loccoauthor=C.+Quinnauthor=A.+Amossauthor=B.+Squadroniauthor=M.+Hartnettauthor=M.+R.+Heoauthor=T.+Whiteauthor=S.+A.+Mayauthor=E.+Bootsauthor=K.+Robertsauthor=P.+Cocchiarellaauthor=A.+Wolickiauthor=A.+Kreamerauthor=P.+S.+Kutchukianauthor=A.+M.+Wassermannauthor=V.+N.+Uebeleauthor=M.+Glickauthor=A.+Rusinkoauthor=J.+C.+Culberson&title=Prospective+Assessment+of+Virtual+Screening+Heuristics+Derived+Using+a+Novel+Fusion+Score&doi=10.1177%2F2472555217706058"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit158&amp;dbid=16384&amp;doi=10.1177%2F2472555217706058&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F2472555217706058%26sid%3Dliteratum%253Aachs%26aulast%3DPertusi%26aufirst%3DD.%2BA.%26aulast%3DO%25E2%2580%2599Donnell%26aufirst%3DG.%26aulast%3DHomsher%26aufirst%3DM.%2BF.%26aulast%3DSolly%26aufirst%3DK.%26aulast%3DPatel%26aufirst%3DA.%26aulast%3DStahler%26aufirst%3DS.%2BL.%26aulast%3DRiley%26aufirst%3DD.%26aulast%3DFinley%26aufirst%3DM.%2BF.%26aulast%3DFinger%26aufirst%3DE.%2BN.%26aulast%3DAdam%26aufirst%3DG.%2BC.%26aulast%3DMeng%26aufirst%3DJ.%26aulast%3DBell%26aulast%3DZuck%26aufirst%3DP.%2BD.%26aulast%3DHudak%26aufirst%3DE.%2BM.%26aulast%3DWeber%26aufirst%3DM.%2BJ.%26aulast%3DNothstein%26aufirst%3DJ.%2BE.%26aulast%3DLocco%26aufirst%3DL.%26aulast%3DQuinn%26aufirst%3DC.%26aulast%3DAmoss%26aufirst%3DA.%26aulast%3DSquadroni%26aufirst%3DB.%26aulast%3DHartnett%26aufirst%3DM.%26aulast%3DHeo%26aufirst%3DM.%2BR.%26aulast%3DWhite%26aufirst%3DT.%26aulast%3DMay%26aufirst%3DS.%2BA.%26aulast%3DBoots%26aufirst%3DE.%26aulast%3DRoberts%26aufirst%3DK.%26aulast%3DCocchiarella%26aufirst%3DP.%26aulast%3DWolicki%26aufirst%3DA.%26aulast%3DKreamer%26aufirst%3DA.%26aulast%3DKutchukian%26aufirst%3DP.%2BS.%26aulast%3DWassermann%26aufirst%3DA.%2BM.%26aulast%3DUebele%26aufirst%3DV.%2BN.%26aulast%3DGlick%26aufirst%3DM.%26aulast%3DRusinko%26aufirst%3DA.%26aulast%3DCulberson%26aufirst%3DJ.%2BC.%26atitle%3DProspective%2520Assessment%2520of%2520Virtual%2520Screening%2520Heuristics%2520Derived%2520Using%2520a%2520Novel%2520Fusion%2520Score%26jtitle%3DSLAS%2520Discovery%26date%3D2017%26volume%3D22%26spage%3D995%26epage%3D1006%26doi%3D10.1177%2F2472555217706058" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref159"><div class="reference"><strong class="refLabel"><a href="#ref159" class="rightTabRefNumLink">159</a></strong><div class="NLM_citation" id="rightTab-cit159"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Martin, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Polyakov, V. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tian, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perez, R. C.</span></span> <span> </span><span class="NLM_article-title">Profile-QSAR 2.0: Kinase Virtual Screening Accuracy Comparable to Four-Concentration IC50s for Realistically Novel Compounds</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">2077</span>– <span class="NLM_lpage">2088</span>, <span class="refDoi"> DOI: 10.1021/acs.jcim.7b00166</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.7b00166" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref159/cit159&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtVKgsLjF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2017&pages=2077-2088&author=E.+J.+Martinauthor=V.+R.+Polyakovauthor=L.+Tianauthor=R.+C.+Perez&title=Profile-QSAR+2.0%3A+Kinase+Virtual+Screening+Accuracy+Comparable+to+Four-Concentration+IC50s+for+Realistically+Novel+Compounds&doi=10.1021%2Facs.jcim.7b00166"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit159R"><div class="casContent"><span class="casTitleNuber">159</span><div class="casTitle"><span class="NLM_cas:atitle">Profile-QSAR 2.0: Kinase Virtual Screening Accuracy Comparable to Four-Concentration IC50s for Realistically Novel Compounds</span></div><div class="casAuthors">Martin, Eric J.; Polyakov, Valery R.; Tian, Li; Perez, Rolando C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2077-2088</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">While conventional random forest regression (RFR) virtual screening models appear to have excellent accuracy on random held-out test sets, they prove lacking in actual practice.  Anal. of 18 historical virtual screens showed that random test sets are far more similar to their training sets than are the compds. project teams actually order.  A new, cluster-based "realistic" training/test set split, which mirrors the chem. novelty of real-life virtual screens, recapitulates the poor predictive power of RFR models in real projects.  The original Profile-QSAR (pQSAR) method greatly broadened the domain of applicability over conventional models by using as independent variables a profile of activity predictions from all historical assays in a large protein family.  However, the accuracy still fell short of expt. on the realistic test sets.  The improved "pQSAR 2.0" method replaces probabilities of activity from naive Bayes categorical models at several thresholds with predicted IC50s from RFR models.  Unexpectedly, the high accuracy also requires removing the RFR model for the actual assay of interest from the independent variable profile.  With these improvements, pQSAR 2.0 activity predictions are now statistically comparable to medium-throughput 4-concn. IC50 measurements even on the realistic test set.  Beyond the yes/no activity predictions from a typical high-throughput screen (HTS) or conventional virtual screen, these semi-quant. IC50 predictions allow for predicted potency, ligand efficiency, lipophilic efficiency and selectivity against antitargets, greatly facilitating hit list triaging and enabling virtual screening panels such as toxicity panels and overall promiscuity predictions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoURHgPoAj7n7Vg90H21EOLACvtfcHk0liadiKWssFAEQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtVKgsLjF&md5=3e215174b6d4b89f490f79e61c893ee1</span></div><a href="/servlet/linkout?suffix=cit159&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.7b00166&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.7b00166%26sid%3Dliteratum%253Aachs%26aulast%3DMartin%26aufirst%3DE.%2BJ.%26aulast%3DPolyakov%26aufirst%3DV.%2BR.%26aulast%3DTian%26aufirst%3DL.%26aulast%3DPerez%26aufirst%3DR.%2BC.%26atitle%3DProfile-QSAR%25202.0%253A%2520Kinase%2520Virtual%2520Screening%2520Accuracy%2520Comparable%2520to%2520Four-Concentration%2520IC50s%2520for%2520Realistically%2520Novel%2520Compounds%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2017%26volume%3D57%26spage%3D2077%26epage%3D2088%26doi%3D10.1021%2Facs.jcim.7b00166" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref160"><div class="reference"><strong class="refLabel"><a href="#ref160" class="rightTabRefNumLink">160</a></strong><div class="NLM_citation" id="rightTab-cit160"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bohacek, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McMartin, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guida, W. C.</span></span> <span> </span><span class="NLM_article-title">The Art and Practice of Structure-Based Drug Design: A Molecular Modeling Perspective</span>. <i>Med. Res. Rev.</i> <span class="NLM_year" style="font-weight: bold;">1996</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">3</span>– <span class="NLM_lpage">50</span>, <span class="refDoi"> DOI: 10.1002/(SICI)1098-1128(199601)16:1<3::AID-MED1>3.0.CO;2-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref160/cit160&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1002%2F%28SICI%291098-1128%28199601%2916%3A1%3C3%3A%3AAID-MED1%3E3.0.CO%3B2-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref160/cit160&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=8788213" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref160/cit160&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADyaK28XhtFyls78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=1996&pages=3-50&author=R.+S.+Bohacekauthor=C.+McMartinauthor=W.+C.+Guida&title=The+Art+and+Practice+of+Structure-Based+Drug+Design%3A+A+Molecular+Modeling+Perspective&doi=10.1002%2F%28SICI%291098-1128%28199601%2916%3A1%3C3%3A%3AAID-MED1%3E3.0.CO%3B2-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit160R"><div class="casContent"><span class="casTitleNuber">160</span><div class="casTitle"><span class="NLM_cas:atitle">The art and practice of structure-based drug design: a molecular modeling perspective</span></div><div class="casAuthors">Bohacek, Regine S.; McMartin, Colin; Guida, Wayne C.</div><div class="citationInfo"><span class="NLM_cas:title">Medicinal Research Reviews</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">3-50</span>CODEN:
                <span class="NLM_cas:coden">MRREDD</span>;
        ISSN:<span class="NLM_cas:issn">0198-6325</span>.
    
            (<span class="NLM_cas:orgname">Wiley</span>)
        </div><div class="casAbstract">A review with 120 refs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrmfrWdtPosdrVg90H21EOLACvtfcHk0lgsz-lsty-iyg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28XhtFyls78%253D&md5=edfa592791c111479117feed600dead6</span></div><a href="/servlet/linkout?suffix=cit160&amp;dbid=16384&amp;doi=10.1002%2F%28SICI%291098-1128%28199601%2916%3A1%3C3%3A%3AAID-MED1%3E3.0.CO%3B2-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F%2528SICI%25291098-1128%2528199601%252916%253A1%253C3%253A%253AAID-MED1%253E3.0.CO%253B2-6%26sid%3Dliteratum%253Aachs%26aulast%3DBohacek%26aufirst%3DR.%2BS.%26aulast%3DMcMartin%26aufirst%3DC.%26aulast%3DGuida%26aufirst%3DW.%2BC.%26atitle%3DThe%2520Art%2520and%2520Practice%2520of%2520Structure-Based%2520Drug%2520Design%253A%2520A%2520Molecular%2520Modeling%2520Perspective%26jtitle%3DMed.%2520Res.%2520Rev.%26date%3D1996%26volume%3D16%26spage%3D3%26epage%3D50%26doi%3D10.1002%2F%28SICI%291098-1128%28199601%2916%3A1%3C3%3A%3AAID-MED1%3E3.0.CO%3B2-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref161"><div class="reference"><strong class="refLabel"><a href="#ref161" class="rightTabRefNumLink">161</a></strong><div class="NLM_citation" id="rightTab-cit161"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Van Hoorn, W. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bell, A. S.</span></span> <span> </span><span class="NLM_article-title">Searching Chemical Space with the Bayesian Idea Generator</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>49</i></span>,  <span class="NLM_fpage">2211</span>– <span class="NLM_lpage">2220</span>, <span class="refDoi"> DOI: 10.1021/ci900072g</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci900072g" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref161/cit161&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtFyqsrfJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2009&pages=2211-2220&author=W.+P.+Van+Hoornauthor=A.+S.+Bell&title=Searching+Chemical+Space+with+the+Bayesian+Idea+Generator&doi=10.1021%2Fci900072g"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit161R"><div class="casContent"><span class="casTitleNuber">161</span><div class="casTitle"><span class="NLM_cas:atitle">Searching Chemical Space with the Bayesian Idea Generator</span></div><div class="casAuthors">van Hoorn, Willem P.; Bell, Andrew S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2211-2220</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The Pfizer Global Virtual Library (PGVL) is defined as a set compds. that could be synthesized using validated protocols and monomers.  However, it is too large (1012 compds.) to search by brute-force methods for close analogs of a given input structure.  In this paper the Bayesian Idea Generator is described which is based on a novel application of Bayesian statistics to narrow down the search space to a prioritized set of existing library arrays (the default is 16).  For each of these libraries the 6 closest neighbors are retrieved from the existing compd. file, resulting in a screenable hypothesis of 96 compds.  Using the Bayesian models for library space, the Pfizer file of singleton compds. has been mapped to library space and is optionally searched as well.  The method is >99% accurate in retrieving known library provenance from an independent test set.  The compds. retrieved strike a balance between similarity and diversity resulting in frequent scaffold hops.  Four examples of how the Bayesian Idea Generator has been successfully used in drug discovery are provided.  The methodol. of the Bayesian Idea Generator can be used for any collection of compds. contg. distinct clusters, and an example using compd. vendor catalogues has been included.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo8rVsn6tLJ5LVg90H21EOLACvtfcHk0lgsz-lsty-iyg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtFyqsrfJ&md5=dbd7bacc3ce21b2d0af8f5d8492dd990</span></div><a href="/servlet/linkout?suffix=cit161&amp;dbid=16384&amp;doi=10.1021%2Fci900072g&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci900072g%26sid%3Dliteratum%253Aachs%26aulast%3DVan%2BHoorn%26aufirst%3DW.%2BP.%26aulast%3DBell%26aufirst%3DA.%2BS.%26atitle%3DSearching%2520Chemical%2520Space%2520with%2520the%2520Bayesian%2520Idea%2520Generator%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2009%26volume%3D49%26spage%3D2211%26epage%3D2220%26doi%3D10.1021%2Fci900072g" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref162"><div class="reference"><strong class="refLabel"><a href="#ref162" class="rightTabRefNumLink">162</a></strong><div class="NLM_citation" id="rightTab-cit162"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Allen, B. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mehta, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ember, S. W. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schonbrunn, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ayad, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schürer, S. C.</span></span> <span> </span><span class="NLM_article-title">Large-Scale Computational Screening Identifies First in Class Multitarget Inhibitor of EGFR Kinase and BRD4</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">16924</span>, <span class="refDoi"> DOI: 10.1038/srep16924</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref162/cit162&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1038%2Fsrep16924" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref162/cit162&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=26596901" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref162/cit162&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvFentrrI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2015&pages=16924&author=B.+K.+Allenauthor=S.+Mehtaauthor=S.+W.+J.+Emberauthor=E.+Schonbrunnauthor=N.+Ayadauthor=S.+C.+Sch%C3%BCrer&title=Large-Scale+Computational+Screening+Identifies+First+in+Class+Multitarget+Inhibitor+of+EGFR+Kinase+and+BRD4&doi=10.1038%2Fsrep16924"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit162R"><div class="casContent"><span class="casTitleNuber">162</span><div class="casTitle"><span class="NLM_cas:atitle">Large-Scale Computational Screening Identifies First in Class Multitarget Inhibitor of EGFR Kinase and BRD4</span></div><div class="casAuthors">Allen, Bryce K.; Mehta, Saurabh; Ember, Stewart W. J.; Schonbrunn, Ernst; Ayad, Nagi; Schurer, Stephan C.</div><div class="citationInfo"><span class="NLM_cas:title">Scientific Reports</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">16924</span>CODEN:
                <span class="NLM_cas:coden">SRCEC3</span>;
        ISSN:<span class="NLM_cas:issn">2045-2322</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Inhibition of cancer-promoting kinases is an established therapeutic strategy for the treatment of many cancers, although resistance to kinase inhibitors is common.  One way to overcome resistance is to target orthogonal cancer-promoting pathways.  Bromo and Extra-Terminal (BET) domain proteins, which belong to the family of epigenetic readers, have recently emerged as promising therapeutic targets in multiple cancers.  The development of multitarget drugs that inhibit kinase and BET proteins therefore may be a promising strategy to overcome tumor resistance and prolong therapeutic efficacy in the clinic.  We developed a general computational screening approach to identify novel dual kinase/bromodomain inhibitors from millions of com. available small mols.  Our method integrated machine learning using big datasets of kinase inhibitors and structure-based drug design.  Here we describe the computational methodol., including validation and characterization of our models and their application and integration into a scalable virtual screening pipeline.  We screened over 6 million com. available compds. and selected 24 for testing in BRD4 and EGFR biochem. assays.  We identified several novel BRD4 inhibitors, among them a first in class dual EGFR-BRD4 inhibitor.  Our studies suggest that this computational screening approach may be broadly applicable for identifying dual kinase/BET inhibitors with potential for treating various cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr2LBJWDL9se7Vg90H21EOLACvtfcHk0lgsz-lsty-iyg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvFentrrI&md5=341f9fedbdfd9e66829e5c43d602d510</span></div><a href="/servlet/linkout?suffix=cit162&amp;dbid=16384&amp;doi=10.1038%2Fsrep16924&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsrep16924%26sid%3Dliteratum%253Aachs%26aulast%3DAllen%26aufirst%3DB.%2BK.%26aulast%3DMehta%26aufirst%3DS.%26aulast%3DEmber%26aufirst%3DS.%2BW.%2BJ.%26aulast%3DSchonbrunn%26aufirst%3DE.%26aulast%3DAyad%26aufirst%3DN.%26aulast%3DSch%25C3%25BCrer%26aufirst%3DS.%2BC.%26atitle%3DLarge-Scale%2520Computational%2520Screening%2520Identifies%2520First%2520in%2520Class%2520Multitarget%2520Inhibitor%2520of%2520EGFR%2520Kinase%2520and%2520BRD4%26jtitle%3DSci.%2520Rep.%26date%3D2015%26volume%3D5%26spage%3D16924%26doi%3D10.1038%2Fsrep16924" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref163"><div class="reference"><strong class="refLabel"><a href="#ref163" class="rightTabRefNumLink">163</a></strong><div class="NLM_citation" id="rightTab-cit163"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Boehm, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, T.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Claussen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lemmen, C.</span></span> <span> </span><span class="NLM_article-title">Similarity Searching and Scaffold Hopping in Synthetically Accessible Combinatorial Chemistry Spaces</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>51</i></span>,  <span class="NLM_fpage">2468</span>– <span class="NLM_lpage">2480</span>, <span class="refDoi"> DOI: 10.1021/jm0707727</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm0707727" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref163/cit163&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BD1cXktFShtL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2008&pages=2468-2480&author=M.+Boehmauthor=T.-Y.+Wuauthor=H.+Claussenauthor=C.+Lemmen&title=Similarity+Searching+and+Scaffold+Hopping+in+Synthetically+Accessible+Combinatorial+Chemistry+Spaces&doi=10.1021%2Fjm0707727"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit163R"><div class="casContent"><span class="casTitleNuber">163</span><div class="casTitle"><span class="NLM_cas:atitle">Similarity searching and scaffold hopping in synthetically accessible combinatorial chemistry spaces</span></div><div class="casAuthors">Boehm, Markus; Wu, Tong-Ying; Claussen, Holger; Lemmen, Christian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2468-2480</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Large collections of combinatorial libraries are an integral element in today's pharmaceutical industry.  It is of great interest to perform similarity searches against all virtual compds. that are synthetically accessible by any such library.  Here we describe the successful application of a new software tool CoLibri on 358 combinatorial libraries based on validated reaction protocols to create a single chem. space contg. over 1012 possible products.  Similarity searching with FTrees-FS allows the systematic exploration of this space without the need to enumerate all product structures.  The search result is a set of virtual hits which are synthetically accessible by one or more of the existing reaction protocols.  Grouping these virtual hits by their synthetic protocols allows the rapid design and synthesis of multiple follow-up libraries.  Such library ideas support hit-to-lead design efforts for tasks like follow-up from high-throughput screening hits or scaffold hopping from one hit to another attractive series.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrW_edQq9zwE7Vg90H21EOLACvtfcHk0lidJjDa9aN7pw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXktFShtL0%253D&md5=c42c419eecf6d272820ed5237f8c1275</span></div><a href="/servlet/linkout?suffix=cit163&amp;dbid=16384&amp;doi=10.1021%2Fjm0707727&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0707727%26sid%3Dliteratum%253Aachs%26aulast%3DBoehm%26aufirst%3DM.%26aulast%3DWu%26aufirst%3DT.-Y.%26aulast%3DClaussen%26aufirst%3DH.%26aulast%3DLemmen%26aufirst%3DC.%26atitle%3DSimilarity%2520Searching%2520and%2520Scaffold%2520Hopping%2520in%2520Synthetically%2520Accessible%2520Combinatorial%2520Chemistry%2520Spaces%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2008%26volume%3D51%26spage%3D2468%26epage%3D2480%26doi%3D10.1021%2Fjm0707727" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref164"><div class="reference"><strong class="refLabel"><a href="#ref164" class="rightTabRefNumLink">164</a></strong><div class="NLM_citation" id="rightTab-cit164"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lessel, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wellenzohn, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lilienthal, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Claussen, H.</span></span> <span> </span><span class="NLM_article-title">Searching Fragment Spaces with Feature Trees</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>49</i></span>,  <span class="NLM_fpage">270</span>– <span class="NLM_lpage">279</span>, <span class="refDoi"> DOI: 10.1021/ci800272a</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci800272a" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref164/cit164&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BD1MXpt1aluw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2009&pages=270-279&author=U.+Lesselauthor=B.+Wellenzohnauthor=M.+Lilienthalauthor=H.+Claussen&title=Searching+Fragment+Spaces+with+Feature+Trees&doi=10.1021%2Fci800272a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit164R"><div class="casContent"><span class="casTitleNuber">164</span><div class="casTitle"><span class="NLM_cas:atitle">Searching Fragment Spaces with Feature Trees</span></div><div class="casAuthors">Lessel, Uta; Wellenzohn, Bernd; Lilienthal, Markus; Claussen, Holger</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">270-279</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Virtual combinatorial chem. easily produces billions of compds., for which conventional virtual screening cannot be performed even with the fastest methods available.  An efficient soln. for such a scenario is the generation of Fragment Spaces, which encode huge nos. of virtual compds. by their fragments/reagents and rules of how to combine them.  Similarity-based searches can be performed in such spaces without ever fully enumerating all virtual products.  Here we describe the generation of a huge Fragment Space encoding about 5*1011 compds. based on established inhouse synthesis protocols for combinatorial libraries, i.e., we encode practically evaluated combinatorial chem. protocols in a machine readable form, rendering them accessible to in silico search methods.  We show how such searches in this Fragment Space can be integrated as a first step in an overall work-flow.  It reduces the extremely huge no. of virtual products by several orders of magnitude so that the resulting list of mols. becomes more manageable for further more elaborated and time-consuming anal. steps.  Results of a case study are presented and discussed, which lead to some general conclusions for an efficient expansion of the chem. space to be screened in pharmaceutical companies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq16bJX_82TS7Vg90H21EOLACvtfcHk0lidJjDa9aN7pw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXpt1aluw%253D%253D&md5=323989844a58818ff57cf1d6bf38fcdf</span></div><a href="/servlet/linkout?suffix=cit164&amp;dbid=16384&amp;doi=10.1021%2Fci800272a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci800272a%26sid%3Dliteratum%253Aachs%26aulast%3DLessel%26aufirst%3DU.%26aulast%3DWellenzohn%26aufirst%3DB.%26aulast%3DLilienthal%26aufirst%3DM.%26aulast%3DClaussen%26aufirst%3DH.%26atitle%3DSearching%2520Fragment%2520Spaces%2520with%2520Feature%2520Trees%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2009%26volume%3D49%26spage%3D270%26epage%3D279%26doi%3D10.1021%2Fci800272a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref165"><div class="reference"><strong class="refLabel"><a href="#ref165" class="rightTabRefNumLink">165</a></strong><div class="NLM_citation" id="rightTab-cit165"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Peng, Z.</span></span> <span> </span><span class="NLM_article-title">Very Large Virtual Compound Spaces: Construction, Storage and Utility in Drug Discovery</span>. <i>Drug Discovery Today: Technol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">e387</span>– <span class="NLM_lpage">e394</span>, <span class="refDoi"> DOI: 10.1016/j.ddtec.2013.01.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref165/cit165&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1016%2Fj.ddtec.2013.01.004" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2013&pages=e387-e394&author=Z.+Peng&title=Very+Large+Virtual+Compound+Spaces%3A+Construction%2C+Storage+and+Utility+in+Drug+Discovery&doi=10.1016%2Fj.ddtec.2013.01.004"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit165&amp;dbid=16384&amp;doi=10.1016%2Fj.ddtec.2013.01.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ddtec.2013.01.004%26sid%3Dliteratum%253Aachs%26aulast%3DPeng%26aufirst%3DZ.%26atitle%3DVery%2520Large%2520Virtual%2520Compound%2520Spaces%253A%2520Construction%252C%2520Storage%2520and%2520Utility%2520in%2520Drug%2520Discovery%26jtitle%3DDrug%2520Discovery%2520Today%253A%2520Technol.%26date%3D2013%26volume%3D10%26spage%3De387%26epage%3De394%26doi%3D10.1016%2Fj.ddtec.2013.01.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref166"><div class="reference"><strong class="refLabel"><a href="#ref166" class="rightTabRefNumLink">166</a></strong><div class="NLM_citation" id="rightTab-cit166"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vainio, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kogej, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raubacher, F.</span></span> <span> </span><span class="NLM_article-title">Automated Recycling of Chemistry for Virtual Screening and Library Design</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>52</i></span>,  <span class="NLM_fpage">1777</span>– <span class="NLM_lpage">1786</span>, <span class="refDoi"> DOI: 10.1021/ci300157m</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci300157m" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref166/cit166&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC38XnvFehuro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2012&pages=1777-1786&author=M.+J.+Vainioauthor=T.+Kogejauthor=F.+Raubacher&title=Automated+Recycling+of+Chemistry+for+Virtual+Screening+and+Library+Design&doi=10.1021%2Fci300157m"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit166R"><div class="casContent"><span class="casTitleNuber">166</span><div class="casTitle"><span class="NLM_cas:atitle">Automated Recycling of Chemistry for Virtual Screening and Library Design</span></div><div class="casAuthors">Vainio, Mikko J.; Kogej, Thierry; Raubacher, Florian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1777-1786</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">An early stage drug discovery project needs to identify a no. of chem. diverse and attractive compds.  These hit compds. are typically found through high-throughput screening campaigns.  The diversity of the chem. libraries used in screening is therefore important.  In this study, we describe a virtual high-throughput screening system called Virtual Library.  The system automatically "recycles" validated synthetic protocols and available starting materials to generate a large no. of virtual compd. libraries, and allows for fast searches in the generated libraries using a 2D fingerprint based screening method.  Virtual Library links the returned virtual hit compds. back to exptl. protocols to quickly assess the synthetic accessibility of the hits.  The system can be used as an idea generator for library design to enrich the screening collection and to explore the structure-activity landscape around a specific active compd.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGovsyO-M3z0jbVg90H21EOLACvtfcHk0lidJjDa9aN7pw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XnvFehuro%253D&md5=15fc68eff3e7716aae6a809a94bb1c1c</span></div><a href="/servlet/linkout?suffix=cit166&amp;dbid=16384&amp;doi=10.1021%2Fci300157m&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci300157m%26sid%3Dliteratum%253Aachs%26aulast%3DVainio%26aufirst%3DM.%2BJ.%26aulast%3DKogej%26aufirst%3DT.%26aulast%3DRaubacher%26aufirst%3DF.%26atitle%3DAutomated%2520Recycling%2520of%2520Chemistry%2520for%2520Virtual%2520Screening%2520and%2520Library%2520Design%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2012%26volume%3D52%26spage%3D1777%26epage%3D1786%26doi%3D10.1021%2Fci300157m" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref167"><div class="reference"><strong class="refLabel"><a href="#ref167" class="rightTabRefNumLink">167</a></strong><div class="NLM_citation" id="rightTab-cit167"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nicolaou, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watson, I. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span> <span> </span><span class="NLM_article-title">The Proximal Lilly Collection: Mapping, Exploring and Exploiting Feasible Chemical Space</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">1253</span>– <span class="NLM_lpage">1266</span>, <span class="refDoi"> DOI: 10.1021/acs.jcim.6b00173</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.6b00173" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref167/cit167&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC28XpsFKjs7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2016&pages=1253-1266&author=C.+A.+Nicolaouauthor=I.+A.+Watsonauthor=H.+Huauthor=J.+Wang&title=The+Proximal+Lilly+Collection%3A+Mapping%2C+Exploring+and+Exploiting+Feasible+Chemical+Space&doi=10.1021%2Facs.jcim.6b00173"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit167R"><div class="casContent"><span class="casTitleNuber">167</span><div class="casTitle"><span class="NLM_cas:atitle">The Proximal Lilly Collection: Mapping, Exploring and Exploiting Feasible Chemical Space</span></div><div class="casAuthors">Nicolaou, Christos A.; Watson, Ian A.; Hu, Hong; Wang, Jibo</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1253-1266</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Venturing into the immensity of the small mol. universe to identify novel chem. structure is a much discussed objective of many methods proposed by the chemoinformatics community.  To this end, numerous approaches using techniques from the fields of computational de novo design, virtual screening and reaction informatics, among others, have been proposed.  Although in principle this objective is commendable, in practice there are several obstacles to useful exploitation of the chem. space.  Prime among them are the sheer no. of theor. feasible compds. and the practical concern regarding the synthesizability of the chem. structures conceived using in silico methods.  We present the Proximal Lilly Collection initiative implemented at Eli Lilly and Co. with the aims to (i) define the chem. space of small, drug-like compds. that could be synthesized using inhouse resources and (ii) facilitate access to compds. in this large space for the purposes of ongoing drug discovery efforts.  The implementation of PLC relies on coupling access to available synthetic knowledge and resources with chemo/reaction informatics techniques and tools developed for this purpose.  We describe in detail the computational framework supporting this initiative and elaborate on the characteristics of the PLC virtual collection of compds.  As an example of the opportunities provided to drug discovery researchers by easy access to a large, realistically feasible virtual collection such as the PLC, we describe a recent application of the technol. that led to the discovery of selective kinase inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrRp50JUV8h5rVg90H21EOLACvtfcHk0lidJjDa9aN7pw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XpsFKjs7o%253D&md5=4689df1c7e0d2eddd93b14db4ac3e4b8</span></div><a href="/servlet/linkout?suffix=cit167&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.6b00173&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.6b00173%26sid%3Dliteratum%253Aachs%26aulast%3DNicolaou%26aufirst%3DC.%2BA.%26aulast%3DWatson%26aufirst%3DI.%2BA.%26aulast%3DHu%26aufirst%3DH.%26aulast%3DWang%26aufirst%3DJ.%26atitle%3DThe%2520Proximal%2520Lilly%2520Collection%253A%2520Mapping%252C%2520Exploring%2520and%2520Exploiting%2520Feasible%2520Chemical%2520Space%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2016%26volume%3D56%26spage%3D1253%26epage%3D1266%26doi%3D10.1021%2Facs.jcim.6b00173" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref168"><div class="reference"><strong class="refLabel"><a href="#ref168" class="rightTabRefNumLink">168</a></strong><div class="NLM_citation" id="rightTab-cit168"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Baranczak, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tu, N. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marjanovic, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Searle, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vasudevan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Djuric, S. W.</span></span> <span> </span><span class="NLM_article-title">Integrated Platform for Expedited Synthesis–Purification–Testing of Small Molecule Libraries</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">461</span>– <span class="NLM_lpage">465</span>, <span class="refDoi"> DOI: 10.1021/acsmedchemlett.7b00054</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.7b00054" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref168/cit168&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC2sXltVSqsr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=461-465&author=A.+Baranczakauthor=N.+P.+Tuauthor=J.+Marjanovicauthor=P.+A.+Searleauthor=A.+Vasudevanauthor=S.+W.+Djuric&title=Integrated+Platform+for+Expedited+Synthesis%E2%80%93Purification%E2%80%93Testing+of+Small+Molecule+Libraries&doi=10.1021%2Facsmedchemlett.7b00054"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit168R"><div class="casContent"><span class="casTitleNuber">168</span><div class="casTitle"><span class="NLM_cas:atitle">Integrated Platform for Expedited Synthesis-Purification-Testing of Small Molecule Libraries</span></div><div class="casAuthors">Baranczak, Aleksandra; Tu, Noah P.; Marjanovic, Jasmina; Searle, Philip A.; Vasudevan, Anil; Djuric, Stevan W.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">461-465</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The productivity of medicinal chem. programs can be significantly increased through the introduction of automation, leading to shortened discovery cycle times.  Herein, the authors describe a platform that consolidates synthesis, purifn., quantitation, dissoln., and testing of small mol. libraries.  The system was validated through the synthesis and testing of two libraries of binders of polycomb protein EED, and excellent correlation of obtained data with results generated through conventional approaches was obsd.  The fully automated and integrated platform enables batch-supported compd. synthesis based on a broad array of chem. transformations with testing in a variety of biochem. assay formats.  A library turnaround time of between 24 and 36 h was achieved, and notably, each library synthesis produces sufficient amts. of compds. for further evaluation in secondary assays thereby contributing significantly to the shortening of medicinal chem. discovery cycles.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrKCbl3UVpOhLVg90H21EOLACvtfcHk0liydqMzdilL9A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXltVSqsr0%253D&md5=7c88f92c6d6e39d77571202db4b7e82b</span></div><a href="/servlet/linkout?suffix=cit168&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.7b00054&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.7b00054%26sid%3Dliteratum%253Aachs%26aulast%3DBaranczak%26aufirst%3DA.%26aulast%3DTu%26aufirst%3DN.%2BP.%26aulast%3DMarjanovic%26aufirst%3DJ.%26aulast%3DSearle%26aufirst%3DP.%2BA.%26aulast%3DVasudevan%26aufirst%3DA.%26aulast%3DDjuric%26aufirst%3DS.%2BW.%26atitle%3DIntegrated%2520Platform%2520for%2520Expedited%2520Synthesis%25E2%2580%2593Purification%25E2%2580%2593Testing%2520of%2520Small%2520Molecule%2520Libraries%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2017%26volume%3D8%26spage%3D461%26epage%3D465%26doi%3D10.1021%2Facsmedchemlett.7b00054" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref169"><div class="reference"><strong class="refLabel"><a href="#ref169" class="rightTabRefNumLink">169</a></strong><div class="NLM_citation" id="rightTab-cit169"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Perera, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tucker, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brahmbhatt, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Helal, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chong, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farrell, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sach, N. W.</span></span> <span> </span><span class="NLM_article-title">A Platform for Automated Nanomole-Scale Reaction Screening and Micromole-Scale Synthesis in Flow</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>359</i></span>,  <span class="NLM_fpage">429</span>– <span class="NLM_lpage">434</span>, <span class="refDoi"> DOI: 10.1126/science.aap9112</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref169/cit169&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1126%2Fscience.aap9112" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref169/cit169&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=29371464" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref169/cit169&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhsVGhsrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=359&publication_year=2018&pages=429-434&author=D.+Pereraauthor=J.+W.+Tuckerauthor=S.+Brahmbhattauthor=C.+J.+Helalauthor=A.+Chongauthor=W.+Farrellauthor=P.+Richardsonauthor=N.+W.+Sach&title=A+Platform+for+Automated+Nanomole-Scale+Reaction+Screening+and+Micromole-Scale+Synthesis+in+Flow&doi=10.1126%2Fscience.aap9112"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit169R"><div class="casContent"><span class="casTitleNuber">169</span><div class="casTitle"><span class="NLM_cas:atitle">A platform for automated nanomole-scale reaction screening and micromole-scale synthesis in flow</span></div><div class="casAuthors">Perera, Damith; Tucker, Joseph W.; Brahmbhatt, Shalini; Helal, Christopher J.; Chong, Ashley; Farrell, William; Richardson, Paul; Sach, Neal W.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">359</span>
        (<span class="NLM_cas:issue">6374</span>),
    <span class="NLM_cas:pages">429-434</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">The scarcity of complex intermediates in pharmaceutical research motivates the pursuit of reaction optimization protocols on submilligram scales.  We report here the development of an automated flow-based synthesis platform, designed from com. available components, that integrates both rapid nanomole-scale reaction screening and micromole-scale synthesis into a single modular unit.  This system was validated by exploring a diverse range of reaction variables in a Suzuki-Miyaura coupling on nanomole scale at elevated temps., generating liq. chromatog.-mass spectrometry data points for 5760 reactions at a rate of >1500 reactions per 24 h.  Through multiple injections of the same segment, the system directly produced micromole quantities of desired material.  The optimal conditions were also replicated in traditional flow and batch mode at 50- to 200-mg scale to provide good to excellent yields.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpc_fiVqt95HLVg90H21EOLACvtfcHk0liydqMzdilL9A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhsVGhsrc%253D&md5=3c288152644b3574372bf5ef3e829a90</span></div><a href="/servlet/linkout?suffix=cit169&amp;dbid=16384&amp;doi=10.1126%2Fscience.aap9112&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.aap9112%26sid%3Dliteratum%253Aachs%26aulast%3DPerera%26aufirst%3DD.%26aulast%3DTucker%26aufirst%3DJ.%2BW.%26aulast%3DBrahmbhatt%26aufirst%3DS.%26aulast%3DHelal%26aufirst%3DC.%2BJ.%26aulast%3DChong%26aufirst%3DA.%26aulast%3DFarrell%26aufirst%3DW.%26aulast%3DRichardson%26aufirst%3DP.%26aulast%3DSach%26aufirst%3DN.%2BW.%26atitle%3DA%2520Platform%2520for%2520Automated%2520Nanomole-Scale%2520Reaction%2520Screening%2520and%2520Micromole-Scale%2520Synthesis%2520in%2520Flow%26jtitle%3DScience%26date%3D2018%26volume%3D359%26spage%3D429%26epage%3D434%26doi%3D10.1126%2Fscience.aap9112" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref170"><div class="reference"><strong class="refLabel"><a href="#ref170" class="rightTabRefNumLink">170</a></strong><div class="NLM_citation" id="rightTab-cit170"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schneider, G.</span></span> <span> </span><span class="NLM_article-title">Automating Drug Discovery</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">97</span>– <span class="NLM_lpage">113</span>, <span class="refDoi"> DOI: 10.1038/nrd.2017.232</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref170/cit170&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1038%2Fnrd.2017.232" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref170/cit170&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=29242609" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref170/cit170&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvFOntbbK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2018&pages=97-113&author=G.+Schneider&title=Automating+Drug+Discovery&doi=10.1038%2Fnrd.2017.232"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit170R"><div class="casContent"><span class="casTitleNuber">170</span><div class="casTitle"><span class="NLM_cas:atitle">Automating drug discovery</span></div><div class="casAuthors">Schneider, Gisbert</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">97-113</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">A review.  Small mol. drug discovery can be viewed as a challenging multidimensional problem in which various characteristics of compds. including efficacy, pharmacokinetics and safety need to be optimized in parallel to provide drug candidates.  Recent advances in areas such as microfluidics-assisted chem. synthesis and biol. testing, as well as artificial intelligence systems that improve a design hypothesis through feedback anal., are now providing a basis for the introduction of greater automation into aspects of this process.  This could potentially accelerate time frames for compd. discovery and optimization and enable more effective searches of chem. space.  However, such approaches also raise considerable conceptual, tech. and organizational challenges, as well as scepticism about the current hype around them.  This article aims to identify the approaches and technologies that could be implemented robustly by medicinal chemists in the near future and to critically analyze the opportunities and challenges for their more widespread application.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpWyg9gWifCLrVg90H21EOLACvtfcHk0liydqMzdilL9A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvFOntbbK&md5=4fc1e441a0a31d7421642eb91fbac8c7</span></div><a href="/servlet/linkout?suffix=cit170&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2017.232&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2017.232%26sid%3Dliteratum%253Aachs%26aulast%3DSchneider%26aufirst%3DG.%26atitle%3DAutomating%2520Drug%2520Discovery%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2018%26volume%3D17%26spage%3D97%26epage%3D113%26doi%3D10.1038%2Fnrd.2017.232" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref171"><div class="reference"><strong class="refLabel"><a href="#ref171" class="rightTabRefNumLink">171</a></strong><div class="NLM_citation" id="rightTab-cit171"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Magnani, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Serrano-Vega, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shibata, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abdul-Hussein, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lebon, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller-Gallacher, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singhal, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Strege, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tate, C. G.</span></span> <span> </span><span class="NLM_article-title">A Mutagenesis and Screening Strategy to Generate Optimally Thermostabilized Membrane Proteins for Structural Studies</span>. <i>Nat. Protoc.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">1554</span>– <span class="NLM_lpage">1571</span>, <span class="refDoi"> DOI: 10.1038/nprot.2016.088</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref171/cit171&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1038%2Fnprot.2016.088" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref171/cit171&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=27466713" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref171/cit171&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A280%3ADC%252BC2s3ls1Clug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2016&pages=1554-1571&author=F.+Magnaniauthor=M.+J.+Serrano-Vegaauthor=Y.+Shibataauthor=S.+Abdul-Husseinauthor=G.+Lebonauthor=J.+Miller-Gallacherauthor=A.+Singhalauthor=A.+Stregeauthor=J.+A.+Thomasauthor=C.+G.+Tate&title=A+Mutagenesis+and+Screening+Strategy+to+Generate+Optimally+Thermostabilized+Membrane+Proteins+for+Structural+Studies&doi=10.1038%2Fnprot.2016.088"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit171R"><div class="casContent"><span class="casTitleNuber">171</span><div class="casTitle"><span class="NLM_cas:atitle">A mutagenesis and screening strategy to generate optimally thermostabilized membrane proteins for structural studies</span></div><div class="casAuthors">Magnani Francesca; Serrano-Vega Maria J; Shibata Yoko; Abdul-Hussein Saba; Lebon Guillaume; Miller-Gallacher Jennifer; Singhal Ankita; Strege Annette; Thomas Jennifer A; Tate Christopher G</div><div class="citationInfo"><span class="NLM_cas:title">Nature protocols</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1554-71</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The thermostability of an integral membrane protein (MP) in detergent solution is a key parameter that dictates the likelihood of obtaining well-diffracting crystals that are suitable for structure determination.  However, many mammalian MPs are too unstable for crystallization.  We developed a thermostabilization strategy based on systematic mutagenesis coupled to a radioligand-binding thermostability assay that can be applied to receptors, ion channels and transporters.  It takes ∼6-12 months to thermostabilize a G-protein-coupled receptor (GPCR) containing 300 amino acid (aa) residues.  The resulting thermostabilized MPs are more easily crystallized and result in high-quality structures.  This methodology has facilitated structure-based drug design applied to GPCRs because it is possible to determine multiple structures of the thermostabilized receptors bound to low-affinity ligands.  Protocols and advice are given on how to develop thermostability assays for MPs and how to combine mutations to make an optimally stable mutant suitable for structural studies.  The steps in the procedure include the generation of ∼300 site-directed mutants by Ala/Leu scanning mutagenesis, the expression of each mutant in mammalian cells by transient transfection and the identification of thermostable mutants using a thermostability assay that is based on binding of an (125)I-labeled radioligand to the unpurified, detergent-solubilized MP.  Individual thermostabilizing point mutations are then combined to make an optimally stable MP that is suitable for structural biology and other biophysical studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRD72cWR2CMnxjv2lLSLOxFfW6udTcc2eYVmhVYpXfxa7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2s3ls1Clug%253D%253D&md5=23318e6bde65a88ce20cb3536ea5f97b</span></div><a href="/servlet/linkout?suffix=cit171&amp;dbid=16384&amp;doi=10.1038%2Fnprot.2016.088&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnprot.2016.088%26sid%3Dliteratum%253Aachs%26aulast%3DMagnani%26aufirst%3DF.%26aulast%3DSerrano-Vega%26aufirst%3DM.%2BJ.%26aulast%3DShibata%26aufirst%3DY.%26aulast%3DAbdul-Hussein%26aufirst%3DS.%26aulast%3DLebon%26aufirst%3DG.%26aulast%3DMiller-Gallacher%26aufirst%3DJ.%26aulast%3DSinghal%26aufirst%3DA.%26aulast%3DStrege%26aufirst%3DA.%26aulast%3DThomas%26aufirst%3DJ.%2BA.%26aulast%3DTate%26aufirst%3DC.%2BG.%26atitle%3DA%2520Mutagenesis%2520and%2520Screening%2520Strategy%2520to%2520Generate%2520Optimally%2520Thermostabilized%2520Membrane%2520Proteins%2520for%2520Structural%2520Studies%26jtitle%3DNat.%2520Protoc.%26date%3D2016%26volume%3D11%26spage%3D1554%26epage%3D1571%26doi%3D10.1038%2Fnprot.2016.088" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref172"><div class="reference"><strong class="refLabel"><a href="#ref172" class="rightTabRefNumLink">172</a></strong><div class="NLM_citation" id="rightTab-cit172"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jazayeri, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rappas, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, A. J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kean, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Errey, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robertson, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fiez-Vandal, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andrews, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Congreve, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bortolato, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mason, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baig, A. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Teobald, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doré, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weir, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cooke, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marshall, F. H.</span></span> <span> </span><span class="NLM_article-title">Crystal Structure of the GLP-1 Receptor Bound to a Peptide Agonist</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>546</i></span>,  <span class="NLM_fpage">254</span>– <span class="NLM_lpage">258</span>, <span class="refDoi"> DOI: 10.1038/nature22800</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref172/cit172&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1038%2Fnature22800" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref172/cit172&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=28562585" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref172/cit172&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC2sXovFCrs70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=546&publication_year=2017&pages=254-258&author=A.+Jazayeriauthor=M.+Rappasauthor=A.+J.+H.+Brownauthor=J.+Keanauthor=J.+C.+Erreyauthor=N.+J.+Robertsonauthor=C.+Fiez-Vandalauthor=S.+P.+Andrewsauthor=M.+Congreveauthor=A.+Bortolatoauthor=J.+S.+Masonauthor=A.+H.+Baigauthor=I.+Teobaldauthor=A.+S.+Dor%C3%A9author=M.+Weirauthor=R.+M.+Cookeauthor=F.+H.+Marshall&title=Crystal+Structure+of+the+GLP-1+Receptor+Bound+to+a+Peptide+Agonist&doi=10.1038%2Fnature22800"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit172R"><div class="casContent"><span class="casTitleNuber">172</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal structure of the GLP-1 receptor bound to a peptide agonist</span></div><div class="casAuthors">Jazayeri, Ali; Rappas, Mathieu; Brown, Alastair J. H.; Kean, James; Errey, James C.; Robertson, Nathan J.; Fiez-Vandal, Cedric; Andrews, Stephen P.; Congreve, Miles; Bortolato, Andrea; Mason, Jonathan S.; Baig, Asma H.; Teobald, Iryna; Dore, Andrew S.; Weir, Malcolm; Cooke, Robert M.; Marshall, Fiona H.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">546</span>
        (<span class="NLM_cas:issue">7657</span>),
    <span class="NLM_cas:pages">254-258</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Glucagon-like peptide 1 (GLP-1) regulates glucose homeostasis through the control of insulin release from the pancreas.  GLP-1 peptide agonists are efficacious drugs for the treatment of diabetes.  To gain insight into the mol. mechanism of action of GLP-1 peptides, here we report the crystal structure of the full-length GLP-1 receptor bound to a truncated peptide agonist.  The peptide agonist retains an α-helical conformation as it sits deep within the receptor-binding pocket.  The arrangement of the transmembrane helixes reveals hallmarks of an active conformation similar to that obsd. in class A receptors.  Guided by this structural information, we design peptide agonists with potent in vivo activity in a mouse model of diabetes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoX6Ibc110QxrVg90H21EOLACvtfcHk0liZMbGRe_k02Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXovFCrs70%253D&md5=0a5c99161af8bae506a184e0ab6306c1</span></div><a href="/servlet/linkout?suffix=cit172&amp;dbid=16384&amp;doi=10.1038%2Fnature22800&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature22800%26sid%3Dliteratum%253Aachs%26aulast%3DJazayeri%26aufirst%3DA.%26aulast%3DRappas%26aufirst%3DM.%26aulast%3DBrown%26aufirst%3DA.%2BJ.%2BH.%26aulast%3DKean%26aufirst%3DJ.%26aulast%3DErrey%26aufirst%3DJ.%2BC.%26aulast%3DRobertson%26aufirst%3DN.%2BJ.%26aulast%3DFiez-Vandal%26aufirst%3DC.%26aulast%3DAndrews%26aufirst%3DS.%2BP.%26aulast%3DCongreve%26aufirst%3DM.%26aulast%3DBortolato%26aufirst%3DA.%26aulast%3DMason%26aufirst%3DJ.%2BS.%26aulast%3DBaig%26aufirst%3DA.%2BH.%26aulast%3DTeobald%26aufirst%3DI.%26aulast%3DDor%25C3%25A9%26aufirst%3DA.%2BS.%26aulast%3DWeir%26aufirst%3DM.%26aulast%3DCooke%26aufirst%3DR.%2BM.%26aulast%3DMarshall%26aufirst%3DF.%2BH.%26atitle%3DCrystal%2520Structure%2520of%2520the%2520GLP-1%2520Receptor%2520Bound%2520to%2520a%2520Peptide%2520Agonist%26jtitle%3DNature%26date%3D2017%26volume%3D546%26spage%3D254%26epage%3D258%26doi%3D10.1038%2Fnature22800" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref173"><div class="reference"><strong class="refLabel"><a href="#ref173" class="rightTabRefNumLink">173</a></strong><div class="NLM_citation" id="rightTab-cit173"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jazayeri, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doré, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lamb, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krishnamurthy, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Southall, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baig, A. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bortolato, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koglin, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robertson, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Errey, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andrews, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Teobald, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, A. J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cooke, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weir, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marshall, F. H.</span></span> <span> </span><span class="NLM_article-title">Extra-Helical Binding Site of a Glucagon Receptor Antagonist</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>533</i></span>,  <span class="NLM_fpage">274</span>– <span class="NLM_lpage">277</span>, <span class="refDoi"> DOI: 10.1038/nature17414</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref173/cit173&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1038%2Fnature17414" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref173/cit173&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=27111510" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref173/cit173&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC28XmvVGnt7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=533&publication_year=2016&pages=274-277&author=A.+Jazayeriauthor=A.+S.+Dor%C3%A9author=D.+Lambauthor=H.+Krishnamurthyauthor=S.+M.+Southallauthor=A.+H.+Baigauthor=A.+Bortolatoauthor=M.+Koglinauthor=N.+J.+Robertsonauthor=J.+C.+Erreyauthor=S.+P.+Andrewsauthor=I.+Teobaldauthor=A.+J.+H.+Brownauthor=R.+M.+Cookeauthor=M.+Weirauthor=F.+H.+Marshall&title=Extra-Helical+Binding+Site+of+a+Glucagon+Receptor+Antagonist&doi=10.1038%2Fnature17414"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit173R"><div class="casContent"><span class="casTitleNuber">173</span><div class="casTitle"><span class="NLM_cas:atitle">Extra-helical binding site of a glucagon receptor antagonist</span></div><div class="casAuthors">Jazayeri, Ali; Dore, Andrew S.; Lamb, Daniel; Krishnamurthy, Harini; Southall, Stacey M.; Baig, Asma H.; Bortolato, Andrea; Koglin, Markus; Robertson, Nathan J.; Errey, James C.; Andrews, Stephen P.; Teobald, Iryna; Brown, Alastair J. H.; Cooke, Robert M.; Weir, Malcolm; Marshall, Fiona H.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">533</span>
        (<span class="NLM_cas:issue">7602</span>),
    <span class="NLM_cas:pages">274-277</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Glucagon is a 29-amino-acid peptide released from the α-cells of the islet of Langerhans, which has a key role in glucose homeostasis.  Glucagon action is transduced by the class B G-protein-coupled glucagon receptor (GCGR), which is located on liver, kidney, intestinal smooth muscle, brain, adipose tissue, heart and pancreas cells, and this receptor has been considered an important drug target in the treatment of diabetes.  Administration of recently identified small-mol. GCGR antagonists in patients with type 2 diabetes results in a substantial redn. of fasting and postprandial glucose concns.  Although an x-ray structure of the transmembrane domain of the GCGR has previously been solved, the ligand (NNC0640) was not resolved.  Here the authors report the 2.5 Å structure of human GCGR in complex with the antagonist MK-0893 (ref. 4), which is found to bind to an allosteric site outside the seven transmembrane (7TM) helical bundle in a position between TM6 and TM7 extending into the lipid bilayer.  Mutagenesis of key residues identified in the x-ray structure confirms their role in the binding of MK-0893 to the receptor.  The unexpected position of the binding site for MK-0893, which is structurally similar to other GCGR antagonists, suggests that glucagon activation of the receptor is prevented by restriction of the outward helical movement of TM6 required for G-protein coupling.  Structural knowledge of class B receptors is limited, with only one other ligand-binding site defined-for the corticotropin-releasing hormone receptor 1 (CRF1R)-which was located deep within the 7TM bundle.  The authors describe a completely novel allosteric binding site for class B receptors, providing an opportunity for structure-based drug design for this receptor class and furthering the authors' understanding of the mechanisms of activation of these receptors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoj307lDqVDzrVg90H21EOLACvtfcHk0liZMbGRe_k02Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XmvVGnt7w%253D&md5=5b45af53e0870f497967c113095da9b4</span></div><a href="/servlet/linkout?suffix=cit173&amp;dbid=16384&amp;doi=10.1038%2Fnature17414&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature17414%26sid%3Dliteratum%253Aachs%26aulast%3DJazayeri%26aufirst%3DA.%26aulast%3DDor%25C3%25A9%26aufirst%3DA.%2BS.%26aulast%3DLamb%26aufirst%3DD.%26aulast%3DKrishnamurthy%26aufirst%3DH.%26aulast%3DSouthall%26aufirst%3DS.%2BM.%26aulast%3DBaig%26aufirst%3DA.%2BH.%26aulast%3DBortolato%26aufirst%3DA.%26aulast%3DKoglin%26aufirst%3DM.%26aulast%3DRobertson%26aufirst%3DN.%2BJ.%26aulast%3DErrey%26aufirst%3DJ.%2BC.%26aulast%3DAndrews%26aufirst%3DS.%2BP.%26aulast%3DTeobald%26aufirst%3DI.%26aulast%3DBrown%26aufirst%3DA.%2BJ.%2BH.%26aulast%3DCooke%26aufirst%3DR.%2BM.%26aulast%3DWeir%26aufirst%3DM.%26aulast%3DMarshall%26aufirst%3DF.%2BH.%26atitle%3DExtra-Helical%2520Binding%2520Site%2520of%2520a%2520Glucagon%2520Receptor%2520Antagonist%26jtitle%3DNature%26date%3D2016%26volume%3D533%26spage%3D274%26epage%3D277%26doi%3D10.1038%2Fnature17414" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref174"><div class="reference"><strong class="refLabel"><a href="#ref174" class="rightTabRefNumLink">174</a></strong><div class="NLM_citation" id="rightTab-cit174"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Oswald, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rappas, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kean, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doré, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Errey, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bennett, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deflorian, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Christopher, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jazayeri, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mason, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Congreve, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cooke, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marshall, F. H.</span></span> <span> </span><span class="NLM_article-title">Intracellular Allosteric Antagonism of the CCR9 Receptor</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>540</i></span>,  <span class="NLM_fpage">462</span>– <span class="NLM_lpage">465</span>, <span class="refDoi"> DOI: 10.1038/nature20606</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref174/cit174&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1038%2Fnature20606" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref174/cit174&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=27926729" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref174/cit174&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC28XitVWmurbM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=540&publication_year=2016&pages=462-465&author=C.+Oswaldauthor=M.+Rappasauthor=J.+Keanauthor=A.+S.+Dor%C3%A9author=J.+C.+Erreyauthor=K.+Bennettauthor=F.+Deflorianauthor=J.+A.+Christopherauthor=A.+Jazayeriauthor=J.+S.+Masonauthor=M.+Congreveauthor=R.+M.+Cookeauthor=F.+H.+Marshall&title=Intracellular+Allosteric+Antagonism+of+the+CCR9+Receptor&doi=10.1038%2Fnature20606"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit174R"><div class="casContent"><span class="casTitleNuber">174</span><div class="casTitle"><span class="NLM_cas:atitle">Intracellular allosteric antagonism of the CCR9 receptor</span></div><div class="casAuthors">Oswald, Christine; Rappas, Mathieu; Kean, James; Dore, Andrew S.; Errey, James C.; Bennett, Kirstie; Deflorian, Francesca; Christopher, John A.; Jazayeri, Ali; Mason, Jonathan S.; Congreve, Miles; Cooke, Robert M.; Marshall, Fiona H.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">540</span>
        (<span class="NLM_cas:issue">7633</span>),
    <span class="NLM_cas:pages">462-465</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Chemokines and their G-protein-coupled receptors play a diverse role in immune defense by controlling the migration, activation and survival of immune cells.  They are also involved in viral entry, tumor growth and metastasis and hence are important drug targets in a wide range of diseases.  Despite very significant efforts by the pharmaceutical industry to develop drugs, with over 50 small-mol. drugs directed at the family entering clin. development, only two compds. have reached the market: maraviroc (CCR5) for HIV infection and plerixafor (CXCR4) for stem-cell mobilization.  The high failure rate may in part be due to limited understanding of the mechanism of action of chemokine antagonists and an inability to optimize compds. in the absence of structural information.  CC chemokine receptor type 9 (CCR9) activation by CCL25 plays a key role in leukocyte recruitment to the gut and represents a therapeutic target in inflammatory bowel disease.  The selective CCR9 antagonist vercirnon progressed to phase 3 clin. trials in Crohn's disease but efficacy was limited, with the need for very high doses to block receptor activation.  Here we report the crystal structure of the CCR9 receptor in complex with vercirnon at 2.8 Å resoln.  Remarkably, vercirnon binds to the intracellular side of the receptor, exerting allosteric antagonism and preventing G-protein coupling.  This binding site explains the need for relatively lipophilic ligands and describes another example of an allosteric site on G-protein-coupled receptors that can be targeted for drug design, not only at CCR9, but potentially extending to other chemokine receptors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpHomIAvUxar7Vg90H21EOLACvtfcHk0liQO4MTAoagyg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitVWmurbM&md5=253b630bd2b9b65fc192ff86de9df61a</span></div><a href="/servlet/linkout?suffix=cit174&amp;dbid=16384&amp;doi=10.1038%2Fnature20606&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature20606%26sid%3Dliteratum%253Aachs%26aulast%3DOswald%26aufirst%3DC.%26aulast%3DRappas%26aufirst%3DM.%26aulast%3DKean%26aufirst%3DJ.%26aulast%3DDor%25C3%25A9%26aufirst%3DA.%2BS.%26aulast%3DErrey%26aufirst%3DJ.%2BC.%26aulast%3DBennett%26aufirst%3DK.%26aulast%3DDeflorian%26aufirst%3DF.%26aulast%3DChristopher%26aufirst%3DJ.%2BA.%26aulast%3DJazayeri%26aufirst%3DA.%26aulast%3DMason%26aufirst%3DJ.%2BS.%26aulast%3DCongreve%26aufirst%3DM.%26aulast%3DCooke%26aufirst%3DR.%2BM.%26aulast%3DMarshall%26aufirst%3DF.%2BH.%26atitle%3DIntracellular%2520Allosteric%2520Antagonism%2520of%2520the%2520CCR9%2520Receptor%26jtitle%3DNature%26date%3D2016%26volume%3D540%26spage%3D462%26epage%3D465%26doi%3D10.1038%2Fnature20606" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref175"><div class="reference"><strong class="refLabel"><a href="#ref175" class="rightTabRefNumLink">175</a></strong><div class="NLM_citation" id="rightTab-cit175"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Harpsøe, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Isberg, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tehan, B. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weiss, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arsova, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marshall, F. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bräuner-Osborne, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gloriam, D. E.</span></span> <span> </span><span class="NLM_article-title">Selective Negative Allosteric Modulation Of Metabotropic Glutamate Receptors – A Structural Perspective of Ligands and Mutants</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">13869</span>, <span class="refDoi"> DOI: 10.1038/srep13869</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref175/cit175&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1038%2Fsrep13869" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref175/cit175&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=26359761" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref175/cit175&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A280%3ADC%252BC283itVSqsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2015&pages=13869&author=K.+Harps%C3%B8eauthor=V.+Isbergauthor=B.+G.+Tehanauthor=D.+Weissauthor=A.+Arsovaauthor=F.+H.+Marshallauthor=H.+Br%C3%A4uner-Osborneauthor=D.+E.+Gloriam&title=Selective+Negative+Allosteric+Modulation+Of+Metabotropic+Glutamate+Receptors+%E2%80%93+A+Structural+Perspective+of+Ligands+and+Mutants&doi=10.1038%2Fsrep13869"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit175R"><div class="casContent"><span class="casTitleNuber">175</span><div class="casTitle"><span class="NLM_cas:atitle">Selective Negative Allosteric Modulation Of Metabotropic Glutamate Receptors - A Structural Perspective of Ligands and Mutants</span></div><div class="casAuthors">Harpsoe Kasper; Isberg Vignir; Arsova Angela; Brauner-Osborne Hans; Gloriam David E; Tehan Benjamin G; Weiss Dahlia; Marshall Fiona H</div><div class="citationInfo"><span class="NLM_cas:title">Scientific reports</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">13869</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The metabotropic glutamate receptors have a wide range of modulatory functions in the central nervous system.  They are among the most highly pursued drug targets, with relevance for several neurological diseases, and a number of allosteric modulators have entered clinical trials.  However, so far this has not led to a marketed drug, largely because of the difficulties in achieving subtype-selective compounds with desired properties.  Very recently the first crystal structures were published for the transmembrane domain of two metabotropic glutamate receptors in complex with negative allosteric modulators.  In this analysis, we make the first comprehensive structural comparison of all metabotropic glutamate receptors, placing selective negative allosteric modulators and critical mutants into the detailed context of the receptor binding sites.  A better understanding of how the different mGlu allosteric modulator binding modes relates to selective pharmacological actions will be very valuable for rational design of safer drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRY5UWfgHZTboYPsCkJIw00fW6udTcc2ebnEHMCy0fVBbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC283itVSqsg%253D%253D&md5=ddff5947451bbc826710b780f7e2df70</span></div><a href="/servlet/linkout?suffix=cit175&amp;dbid=16384&amp;doi=10.1038%2Fsrep13869&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsrep13869%26sid%3Dliteratum%253Aachs%26aulast%3DHarps%25C3%25B8e%26aufirst%3DK.%26aulast%3DIsberg%26aufirst%3DV.%26aulast%3DTehan%26aufirst%3DB.%2BG.%26aulast%3DWeiss%26aufirst%3DD.%26aulast%3DArsova%26aufirst%3DA.%26aulast%3DMarshall%26aufirst%3DF.%2BH.%26aulast%3DBr%25C3%25A4uner-Osborne%26aufirst%3DH.%26aulast%3DGloriam%26aufirst%3DD.%2BE.%26atitle%3DSelective%2520Negative%2520Allosteric%2520Modulation%2520Of%2520Metabotropic%2520Glutamate%2520Receptors%2520%25E2%2580%2593%2520A%2520Structural%2520Perspective%2520of%2520Ligands%2520and%2520Mutants%26jtitle%3DSci.%2520Rep.%26date%3D2015%26volume%3D5%26spage%3D13869%26doi%3D10.1038%2Fsrep13869" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref176"><div class="reference"><strong class="refLabel"><a href="#ref176" class="rightTabRefNumLink">176</a></strong><div class="NLM_citation" id="rightTab-cit176"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kean, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bortolato, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hollenstein, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marshall, F. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jazayeri, A.</span></span> <span> </span><span class="NLM_article-title">Conformational Thermostabilisation of Corticotropin Releasing Factor Receptor 1</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">11954</span>, <span class="refDoi"> DOI: 10.1038/srep11954</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref176/cit176&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1038%2Fsrep11954" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref176/cit176&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=26159865" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref176/cit176&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A280%3ADC%252BC28%252FhvVGnuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2015&pages=11954&author=J.+Keanauthor=A.+Bortolatoauthor=K.+Hollensteinauthor=F.+H.+Marshallauthor=A.+Jazayeri&title=Conformational+Thermostabilisation+of+Corticotropin+Releasing+Factor+Receptor+1&doi=10.1038%2Fsrep11954"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit176R"><div class="casContent"><span class="casTitleNuber">176</span><div class="casTitle"><span class="NLM_cas:atitle">Conformational thermostabilisation of corticotropin releasing factor receptor 1</span></div><div class="casAuthors">Kean James; Bortolato Andrea; Hollenstein Kaspar; Marshall Fiona H; Jazayeri Ali</div><div class="citationInfo"><span class="NLM_cas:title">Scientific reports</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">11954</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Recent technical advances have greatly facilitated G-protein coupled receptors crystallography as evidenced by the number of successful x-ray structures that have been reported recently.  These technical advances include novel detergents, specialised crystallography techniques as well as protein engineering solutions such as fusions and conformational thermostabilisation.  Using conformational thermostabilisation, it is possible to generate variants of GPCRs that exhibit significantly increased stability in detergent micelles whilst preferentially occupying a single conformation.  In this paper we describe for the first time the application of this technique to a member of a class B GPCR, the corticotropin releasing factor receptor 1 (CRF1R).  Mutational screening in the presence of the inverse agonist, CP-376395, resulted in the identification of a construct with twelve point mutations that exhibited significantly increased thermal stability in a range of detergents.  We further describe the subsequent construct engineering steps that eventually yielded a crystallisation-ready construct which recently led to the solution of the first x-ray structure of a class B receptor.  Finally, we have used molecular dynamic simulation to provide structural insight into CRF1R instability as well as the stabilising effects of the mutants, which may be extended to other class B receptors considering the high degree of structural conservation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSjZbDL9WcLFO2qxNLjzXe8fW6udTcc2ebnEHMCy0fVBbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC28%252FhvVGnuw%253D%253D&md5=dd1bc3726119ed8d19b7ecfe4ee227b2</span></div><a href="/servlet/linkout?suffix=cit176&amp;dbid=16384&amp;doi=10.1038%2Fsrep11954&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsrep11954%26sid%3Dliteratum%253Aachs%26aulast%3DKean%26aufirst%3DJ.%26aulast%3DBortolato%26aufirst%3DA.%26aulast%3DHollenstein%26aufirst%3DK.%26aulast%3DMarshall%26aufirst%3DF.%2BH.%26aulast%3DJazayeri%26aufirst%3DA.%26atitle%3DConformational%2520Thermostabilisation%2520of%2520Corticotropin%2520Releasing%2520Factor%2520Receptor%25201%26jtitle%3DSci.%2520Rep.%26date%3D2015%26volume%3D5%26spage%3D11954%26doi%3D10.1038%2Fsrep11954" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref177"><div class="reference"><strong class="refLabel"><a href="#ref177" class="rightTabRefNumLink">177</a></strong><div class="NLM_citation" id="rightTab-cit177"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, R. K. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fiez-Vandal, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schlenker, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edman, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aggeler, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, D. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cooke, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dumelin, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doré, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geschwindner, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grebner, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hermansson, N.-O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jazayeri, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johansson, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leong, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prihandoko, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rappas, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soutter, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Snijder, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sundström, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tehan, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thornton, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Troast, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wiggin, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhukov, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marshall, F. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dekker, N.</span></span> <span> </span><span class="NLM_article-title">Structural Insight into Allosteric Modulation of Protease-Activated Receptor 2</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>545</i></span>,  <span class="NLM_fpage">112</span>– <span class="NLM_lpage">115</span>, <span class="refDoi"> DOI: 10.1038/nature22309</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref177/cit177&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1038%2Fnature22309" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref177/cit177&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=28445455" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref177/cit177&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC2sXmslehsrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=545&publication_year=2017&pages=112-115&author=R.+K.+Y.+Chengauthor=C.+Fiez-Vandalauthor=O.+Schlenkerauthor=K.+Edmanauthor=B.+Aggelerauthor=D.+G.+Brownauthor=G.+A.+Brownauthor=R.+M.+Cookeauthor=C.+E.+Dumelinauthor=A.+S.+Dor%C3%A9author=S.+Geschwindnerauthor=C.+Grebnerauthor=N.-O.+Hermanssonauthor=A.+Jazayeriauthor=P.+Johanssonauthor=L.+Leongauthor=R.+Prihandokoauthor=M.+Rappasauthor=H.+Soutterauthor=A.+Snijderauthor=L.+Sundstr%C3%B6mauthor=B.+Tehanauthor=P.+Thorntonauthor=D.+Troastauthor=G.+Wigginauthor=A.+Zhukovauthor=F.+H.+Marshallauthor=N.+Dekker&title=Structural+Insight+into+Allosteric+Modulation+of+Protease-Activated+Receptor+2&doi=10.1038%2Fnature22309"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit177R"><div class="casContent"><span class="casTitleNuber">177</span><div class="casTitle"><span class="NLM_cas:atitle">Structural insight into allosteric modulation of protease-activated receptor 2</span></div><div class="casAuthors">Cheng, Robert K. Y.; Fiez-Vandal, Cedric; Schlenker, Oliver; Edman, Karl; Aggeler, Birte; Brown, Dean G.; Brown, Giles A.; Cooke, Robert M.; Dumelin, Christoph E.; Dore, Andrew S.; Geschwindner, Stefan; Grebner, Christoph; Hermansson, Nils-Olov; Jazayeri, Ali; Johansson, Patrik; Leong, Louis; Prihandoko, Rudi; Rappas, Mathieu; Soutter, Holly; Snijder, Arjan; Sundstrom, Linda; Tehan, Benjamin; Thornton, Peter; Troast, Dawn; Wiggin, Giselle; Zhukov, Andrei; Marshall, Fiona H.; Dekker, Niek</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">545</span>
        (<span class="NLM_cas:issue">7652</span>),
    <span class="NLM_cas:pages">112-115</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Protease-activated receptors (PARs) are a family of G-protein-coupled receptors (GPCRs) that are irreversibly activated by proteolytic cleavage of the N-terminus, which unmasks a tethered peptide ligand that binds and activates the transmembrane receptor domain, eliciting a cellular cascade in response to inflammatory signals and other stimuli.  PARs are implicated in a wide range of diseases, such as cancer and inflammation.  PARs have been the subject of major pharmaceutical research efforts, but the discovery of small-mol. antagonists that effectively bind them has proved challenging.  The only marketed drug targeting a PAR is vorapaxar, a selective antagonist of PAR1 used to prevent thrombosis.  The structure of PAR1 in complex with vorapaxar has been reported previously.  Despite sequence homol. across the PAR isoforms, discovery of PAR2 antagonists has been less successful, although GB88 has been described as a weak antagonist.  Here, we report crystal structures of PAR2 in complex with two distinct antagonists and a blocking antibody.  The antagonist AZ8838 binds in a fully occluded pocket near the extracellular surface.  Functional and binding studies reveal that AZ8838 exhibits slow binding kinetics, which is an attractive feature for a PAR2 antagonist competing against a tethered ligand.  Antagonist AZ3451 binds to a remote allosteric site outside the helical bundle.  We propose that antagonist binding prevents structural rearrangements required for receptor activation and signalling.  We also show that a blocking antibody antigen-binding fragment binds to the extracellular surface of PAR2, preventing access of the tethered ligand to the peptide-binding site.  These structures provide a basis for the development of selective PAR2 antagonists for a range of therapeutic uses.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoSz2vmZJ2f6LVg90H21EOLACvtfcHk0lhsEHQinKJMGQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXmslehsrs%253D&md5=6a0737757c891336437d4419c3fd6c45</span></div><a href="/servlet/linkout?suffix=cit177&amp;dbid=16384&amp;doi=10.1038%2Fnature22309&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature22309%26sid%3Dliteratum%253Aachs%26aulast%3DCheng%26aufirst%3DR.%2BK.%2BY.%26aulast%3DFiez-Vandal%26aufirst%3DC.%26aulast%3DSchlenker%26aufirst%3DO.%26aulast%3DEdman%26aufirst%3DK.%26aulast%3DAggeler%26aufirst%3DB.%26aulast%3DBrown%26aufirst%3DD.%2BG.%26aulast%3DBrown%26aufirst%3DG.%2BA.%26aulast%3DCooke%26aufirst%3DR.%2BM.%26aulast%3DDumelin%26aufirst%3DC.%2BE.%26aulast%3DDor%25C3%25A9%26aufirst%3DA.%2BS.%26aulast%3DGeschwindner%26aufirst%3DS.%26aulast%3DGrebner%26aufirst%3DC.%26aulast%3DHermansson%26aufirst%3DN.-O.%26aulast%3DJazayeri%26aufirst%3DA.%26aulast%3DJohansson%26aufirst%3DP.%26aulast%3DLeong%26aufirst%3DL.%26aulast%3DPrihandoko%26aufirst%3DR.%26aulast%3DRappas%26aufirst%3DM.%26aulast%3DSoutter%26aufirst%3DH.%26aulast%3DSnijder%26aufirst%3DA.%26aulast%3DSundstr%25C3%25B6m%26aufirst%3DL.%26aulast%3DTehan%26aufirst%3DB.%26aulast%3DThornton%26aufirst%3DP.%26aulast%3DTroast%26aufirst%3DD.%26aulast%3DWiggin%26aufirst%3DG.%26aulast%3DZhukov%26aufirst%3DA.%26aulast%3DMarshall%26aufirst%3DF.%2BH.%26aulast%3DDekker%26aufirst%3DN.%26atitle%3DStructural%2520Insight%2520into%2520Allosteric%2520Modulation%2520of%2520Protease-Activated%2520Receptor%25202%26jtitle%3DNature%26date%3D2017%26volume%3D545%26spage%3D112%26epage%3D115%26doi%3D10.1038%2Fnature22309" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref178"><div class="reference"><strong class="refLabel"><a href="#ref178" class="rightTabRefNumLink">178</a></strong><div class="NLM_citation" id="rightTab-cit178"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rasmussen, S. G. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choi, H.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fung, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pardon, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Casarosa, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chae, P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DeVree, B. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosenbaum, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thian, F. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kobilka, T. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schnapp, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Konetzki, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sunahara, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gellman, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pautsch, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steyaert, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weis, W. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kobilka, B. K.</span></span> <span> </span><span class="NLM_article-title">Structure of a Nanobody-Stabilized Active State of the β<sub>2</sub> Adrenoceptor</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>469</i></span>,  <span class="NLM_fpage">175</span>– <span class="NLM_lpage">180</span>, <span class="refDoi"> DOI: 10.1038/nature09648</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref178/cit178&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1038%2Fnature09648" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref178/cit178&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=21228869" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref178/cit178&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC3MXkvFartA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=469&publication_year=2011&pages=175-180&author=S.+G.+F.+Rasmussenauthor=H.-J.+Choiauthor=J.+J.+Fungauthor=E.+Pardonauthor=P.+Casarosaauthor=P.+S.+Chaeauthor=B.+T.+DeVreeauthor=D.+M.+Rosenbaumauthor=F.+S.+Thianauthor=T.+S.+Kobilkaauthor=A.+Schnappauthor=I.+Konetzkiauthor=R.+K.+Sunaharaauthor=S.+H.+Gellmanauthor=A.+Pautschauthor=J.+Steyaertauthor=W.+I.+Weisauthor=B.+K.+Kobilka&title=Structure+of+a+Nanobody-Stabilized+Active+State+of+the+%CE%B22+Adrenoceptor&doi=10.1038%2Fnature09648"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit178R"><div class="casContent"><span class="casTitleNuber">178</span><div class="casTitle"><span class="NLM_cas:atitle">Structure of a nanobody-stabilized active state of the β2 adrenoceptor</span></div><div class="casAuthors">Rasmussen, Soren G. F.; Choi, Hee-Jung; Fung, Juan Jose; Pardon, Els; Casarosa, Paola; Chae, Pil Seok; DeVree, Brian T.; Rosenbaum, Daniel M.; Thian, Foon Sun; Kobilka, Tong Sun; Schnapp, Andreas; Konetzki, Ingo; Sunahara, Roger K.; Gellman, Samuel H.; Pautsch, Alexander; Steyaert, Jan; Weis, William I.; Kobilka, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">469</span>
        (<span class="NLM_cas:issue">7329</span>),
    <span class="NLM_cas:pages">175-180</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">G protein coupled receptors (GPCRs) exhibit a spectrum of functional behaviors in response to natural and synthetic ligands.  Recent crystal structures provide insights into inactive states of several GPCRs.  Efforts to obtain an agonist-bound active-state GPCR structure have proven difficult due to the inherent instability of this state in the absence of a G protein.  We generated a camelid antibody fragment (nanobody) to the human β2 adrenergic receptor (β2AR) that exhibits G protein-like behavior, and obtained an agonist-bound, active-state crystal structure of the receptor-nanobody complex.  Comparison with the inactive β2AR structure reveals subtle changes in the binding pocket; however, these small changes are assocd. with an 11 Å outward movement of the cytoplasmic end of transmembrane segment 6, and rearrangements of transmembrane segments 5 and 7 that are remarkably similar to those obsd. in opsin, an active form of rhodopsin.  This structure provides insights into the process of agonist binding and activation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqITZxHz6Vi77Vg90H21EOLACvtfcHk0lhsEHQinKJMGQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXkvFartA%253D%253D&md5=3a1d3bac6d92c9d54cf1ddd14d56ab8c</span></div><a href="/servlet/linkout?suffix=cit178&amp;dbid=16384&amp;doi=10.1038%2Fnature09648&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature09648%26sid%3Dliteratum%253Aachs%26aulast%3DRasmussen%26aufirst%3DS.%2BG.%2BF.%26aulast%3DChoi%26aufirst%3DH.-J.%26aulast%3DFung%26aufirst%3DJ.%2BJ.%26aulast%3DPardon%26aufirst%3DE.%26aulast%3DCasarosa%26aufirst%3DP.%26aulast%3DChae%26aufirst%3DP.%2BS.%26aulast%3DDeVree%26aufirst%3DB.%2BT.%26aulast%3DRosenbaum%26aufirst%3DD.%2BM.%26aulast%3DThian%26aufirst%3DF.%2BS.%26aulast%3DKobilka%26aufirst%3DT.%2BS.%26aulast%3DSchnapp%26aufirst%3DA.%26aulast%3DKonetzki%26aufirst%3DI.%26aulast%3DSunahara%26aufirst%3DR.%2BK.%26aulast%3DGellman%26aufirst%3DS.%2BH.%26aulast%3DPautsch%26aufirst%3DA.%26aulast%3DSteyaert%26aufirst%3DJ.%26aulast%3DWeis%26aufirst%3DW.%2BI.%26aulast%3DKobilka%26aufirst%3DB.%2BK.%26atitle%3DStructure%2520of%2520a%2520Nanobody-Stabilized%2520Active%2520State%2520of%2520the%2520%25CE%25B22%2520Adrenoceptor%26jtitle%3DNature%26date%3D2011%26volume%3D469%26spage%3D175%26epage%3D180%26doi%3D10.1038%2Fnature09648" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref179"><div class="reference"><strong class="refLabel"><a href="#ref179" class="rightTabRefNumLink">179</a></strong><div class="NLM_citation" id="rightTab-cit179"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rasmussen, S. G. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DeVree, B. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zou, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kruse, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, K. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kobilka, T. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thian, F. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chae, P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pardon, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Calinski, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mathiesen, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shah, S. T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lyons, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caffrey, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gellman, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steyaert, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skiniotis, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weis, W. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sunahara, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kobilka, B. K.</span></span> <span> </span><span class="NLM_article-title">Crystal Structure of the β2 Adrenergic receptor–Gs Protein Complex</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>477</i></span>,  <span class="NLM_fpage">549</span>– <span class="NLM_lpage">555</span>, <span class="refDoi"> DOI: 10.1038/nature10361</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref179/cit179&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1038%2Fnature10361" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref179/cit179&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=21772288" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref179/cit179&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC3MXht1equrrL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=477&publication_year=2011&pages=549-555&author=S.+G.+F.+Rasmussenauthor=B.+T.+DeVreeauthor=Y.+Zouauthor=A.+C.+Kruseauthor=K.+Y.+Chungauthor=T.+S.+Kobilkaauthor=F.+S.+Thianauthor=P.+S.+Chaeauthor=E.+Pardonauthor=D.+Calinskiauthor=J.+M.+Mathiesenauthor=S.+T.+A.+Shahauthor=J.+A.+Lyonsauthor=M.+Caffreyauthor=S.+H.+Gellmanauthor=J.+Steyaertauthor=G.+Skiniotisauthor=W.+I.+Weisauthor=R.+K.+Sunaharaauthor=B.+K.+Kobilka&title=Crystal+Structure+of+the+%CE%B22+Adrenergic+receptor%E2%80%93Gs+Protein+Complex&doi=10.1038%2Fnature10361"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit179R"><div class="casContent"><span class="casTitleNuber">179</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal structure of the β2 adrenergic receptor-Gs protein complex</span></div><div class="casAuthors">Rasmussen, Soren G. F.; DeVree, Brian T.; Zou, Yao-Zhong; Kruse, Andrew C.; Chung, Ka-Young; Kobilka, Tong-Sun; Thian, Foon-Sun; Chae, Pil-Seok; Pardon, Els; Calinski, Diane; Mathiesen, Jesper M.; Shah, Syed T. A.; Lyons, Joseph A.; Caffrey, Martin; Gellman, Samuel H.; Steyaert, Jan; Skiniotis, Georgios; Weis, William I.; Sunahara, Roger K.; Kobilka, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">477</span>
        (<span class="NLM_cas:issue">7366</span>),
    <span class="NLM_cas:pages">549-555</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">G protein-coupled receptors (GPCRs) are responsible for the majority of cellular responses to hormones and neurotransmitters as well as the senses of sight, olfaction and taste.  The paradigm of GPCR signalling is the activation of a heterotrimeric GTP binding protein (G protein) by an agonist-occupied receptor.  The β2 adrenergic receptor (β2AR) activation of Gs, the stimulatory G protein for adenylyl cyclase, has long been a model system for GPCR signalling.  Here we present the crystal structure of the active state ternary complex composed of agonist-occupied monomeric β2AR and nucleotide-free Gs heterotrimer.  The principal interactions between the β2AR and Gs involve the amino- and carboxy-terminal α-helixes of Gs, with conformational changes propagating to the nucleotide-binding pocket.  The largest conformational changes in the β2AR include a 14 Å outward movement at the cytoplasmic end of transmembrane segment 6 (TM6) and an α-helical extension of the cytoplasmic end of TM5.  The most surprising observation is a major displacement of the α-helical domain of Gαs relative to the Ras-like GTPase domain.  This crystal structure represents the first high-resoln. view of transmembrane signalling by a GPCR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq-PdYRDvolqrVg90H21EOLACvtfcHk0lhsEHQinKJMGQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXht1equrrL&md5=d22a43dd677ac255d138b1aedff357d3</span></div><a href="/servlet/linkout?suffix=cit179&amp;dbid=16384&amp;doi=10.1038%2Fnature10361&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature10361%26sid%3Dliteratum%253Aachs%26aulast%3DRasmussen%26aufirst%3DS.%2BG.%2BF.%26aulast%3DDeVree%26aufirst%3DB.%2BT.%26aulast%3DZou%26aufirst%3DY.%26aulast%3DKruse%26aufirst%3DA.%2BC.%26aulast%3DChung%26aufirst%3DK.%2BY.%26aulast%3DKobilka%26aufirst%3DT.%2BS.%26aulast%3DThian%26aufirst%3DF.%2BS.%26aulast%3DChae%26aufirst%3DP.%2BS.%26aulast%3DPardon%26aufirst%3DE.%26aulast%3DCalinski%26aufirst%3DD.%26aulast%3DMathiesen%26aufirst%3DJ.%2BM.%26aulast%3DShah%26aufirst%3DS.%2BT.%2BA.%26aulast%3DLyons%26aufirst%3DJ.%2BA.%26aulast%3DCaffrey%26aufirst%3DM.%26aulast%3DGellman%26aufirst%3DS.%2BH.%26aulast%3DSteyaert%26aufirst%3DJ.%26aulast%3DSkiniotis%26aufirst%3DG.%26aulast%3DWeis%26aufirst%3DW.%2BI.%26aulast%3DSunahara%26aufirst%3DR.%2BK.%26aulast%3DKobilka%26aufirst%3DB.%2BK.%26atitle%3DCrystal%2520Structure%2520of%2520the%2520%25CE%25B22%2520Adrenergic%2520receptor%25E2%2580%2593Gs%2520Protein%2520Complex%26jtitle%3DNature%26date%3D2011%26volume%3D477%26spage%3D549%26epage%3D555%26doi%3D10.1038%2Fnature10361" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref180"><div class="reference"><strong class="refLabel"><a href="#ref180" class="rightTabRefNumLink">180</a></strong><div class="NLM_citation" id="rightTab-cit180"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sounier, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mas, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steyaert, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laeremans, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manglik, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kobilka, B. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Déméné, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Granier, S.</span></span> <span> </span><span class="NLM_article-title">Propagation of Conformational Changes during Μ-Opioid Receptor Activation</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>524</i></span>,  <span class="NLM_fpage">375</span>– <span class="NLM_lpage">378</span>, <span class="refDoi"> DOI: 10.1038/nature14680</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref180/cit180&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1038%2Fnature14680" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref180/cit180&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=26245377" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref180/cit180&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC2MXht12jtb%252FN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=524&publication_year=2015&pages=375-378&author=R.+Sounierauthor=C.+Masauthor=J.+Steyaertauthor=T.+Laeremansauthor=A.+Manglikauthor=W.+Huangauthor=B.+K.+Kobilkaauthor=H.+D%C3%A9m%C3%A9n%C3%A9author=S.+Granier&title=Propagation+of+Conformational+Changes+during+%CE%9C-Opioid+Receptor+Activation&doi=10.1038%2Fnature14680"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit180R"><div class="casContent"><span class="casTitleNuber">180</span><div class="casTitle"><span class="NLM_cas:atitle">Propagation of conformational changes during μ-opioid receptor activation</span></div><div class="casAuthors">Sounier, Remy; Mas, Camille; Steyaert, Jan; Laeremans, Toon; Manglik, Aashish; Huang, Weijiao; Kobilka, Brian K.; Demene, Helene; Granier, Sebastien</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">524</span>
        (<span class="NLM_cas:issue">7565</span>),
    <span class="NLM_cas:pages">375-378</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">μ-Opioid receptors (μORs) are G-protein-coupled receptors that are activated by a structurally diverse spectrum of natural and synthetic agonists including endogenous endorphin peptides, morphine and methadone.  The recent structures of the μOR in inactive and agonist-induced active states (W. Huang et al., 2015) provide snapshots of the receptor at the beginning and end of a signalling event, but little is known about the dynamic sequence of events that span these two states.  Here we use soln.-state NMR to examine the process of μOR activation using a purified receptor (mouse sequence) prepn. in an amphiphile membrane-like environment.  We obtain spectra of the μOR in the absence of ligand, and in the presence of the high-affinity agonist BU 72 alone, or with BU 72 and a G protein mimetic nanobody.  Our results show that conformational changes in transmembrane segments 5 and 6 (TM5 and TM6), which are required for the full engagement of a G protein, are almost completely dependent on the presence of both the agonist and the G protein mimetic nanobody, revealing a weak allosteric coupling between the agonist-binding pocket and the G-protein-coupling interface (TM5 and TM6), similar to that obsd. for the β2-adrenergic receptor.  Unexpectedly, in the presence of agonist alone, we find larger spectral changes involving intracellular loop 1 and helix 8 compared to changes in TM5 and TM6.  These results suggest that one or both of these domains may play a role in the initial interaction with the G protein, and that TM5 and TM6 are only engaged later in the process of complex formation.  The initial interactions between the G protein and intracellular loop 1 and/or helix 8 may be involved in G-protein coupling specificity, as has been suggested for other family A G-protein-coupled receptors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGonzj6Bmj4e5LVg90H21EOLACvtfcHk0lg_yMBs76WWMA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXht12jtb%252FN&md5=0dd0dedcce583a52b4adc3084726a862</span></div><a href="/servlet/linkout?suffix=cit180&amp;dbid=16384&amp;doi=10.1038%2Fnature14680&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature14680%26sid%3Dliteratum%253Aachs%26aulast%3DSounier%26aufirst%3DR.%26aulast%3DMas%26aufirst%3DC.%26aulast%3DSteyaert%26aufirst%3DJ.%26aulast%3DLaeremans%26aufirst%3DT.%26aulast%3DManglik%26aufirst%3DA.%26aulast%3DHuang%26aufirst%3DW.%26aulast%3DKobilka%26aufirst%3DB.%2BK.%26aulast%3DD%25C3%25A9m%25C3%25A9n%25C3%25A9%26aufirst%3DH.%26aulast%3DGranier%26aufirst%3DS.%26atitle%3DPropagation%2520of%2520Conformational%2520Changes%2520during%2520%25CE%259C-Opioid%2520Receptor%2520Activation%26jtitle%3DNature%26date%3D2015%26volume%3D524%26spage%3D375%26epage%3D378%26doi%3D10.1038%2Fnature14680" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref181"><div class="reference"><strong class="refLabel"><a href="#ref181" class="rightTabRefNumLink">181</a></strong><div class="NLM_citation" id="rightTab-cit181"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pardon, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Betti, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laeremans, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chevillard, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guillemyn, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kolb, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ballet, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steyaert, J.</span></span> <span> </span><span class="NLM_article-title">Nanobody-Enabled Reverse Pharmacology on G-Protein-Coupled Receptors</span>. <i>Angew. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>130</i></span>,  <span class="NLM_fpage">5390</span>– <span class="NLM_lpage">5393</span>, <span class="refDoi"> DOI: 10.1002/ange.201712581</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref181/cit181&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1002%2Fange.201712581" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=130&publication_year=2018&pages=5390-5393&author=E.+Pardonauthor=C.+Bettiauthor=T.+Laeremansauthor=F.+Chevillardauthor=K.+Guillemynauthor=P.+Kolbauthor=S.+Balletauthor=J.+Steyaert&title=Nanobody-Enabled+Reverse+Pharmacology+on+G-Protein-Coupled+Receptors&doi=10.1002%2Fange.201712581"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit181&amp;dbid=16384&amp;doi=10.1002%2Fange.201712581&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fange.201712581%26sid%3Dliteratum%253Aachs%26aulast%3DPardon%26aufirst%3DE.%26aulast%3DBetti%26aufirst%3DC.%26aulast%3DLaeremans%26aufirst%3DT.%26aulast%3DChevillard%26aufirst%3DF.%26aulast%3DGuillemyn%26aufirst%3DK.%26aulast%3DKolb%26aufirst%3DP.%26aulast%3DBallet%26aufirst%3DS.%26aulast%3DSteyaert%26aufirst%3DJ.%26atitle%3DNanobody-Enabled%2520Reverse%2520Pharmacology%2520on%2520G-Protein-Coupled%2520Receptors%26jtitle%3DAngew.%2520Chem.%26date%3D2018%26volume%3D130%26spage%3D5390%26epage%3D5393%26doi%3D10.1002%2Fange.201712581" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref182"><div class="reference"><strong class="refLabel"><a href="#ref182" class="rightTabRefNumLink">182</a></strong><div class="NLM_citation" id="rightTab-cit182"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Manglik, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aryal, D. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCorvy, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dengler, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Corder, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Levit, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kling, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernat, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hübner, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, X.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sassano, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giguère, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Löber, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duan, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scherrer, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kobilka, B. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gmeiner, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roth, B. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shoichet, B. K.</span></span> <span> </span><span class="NLM_article-title">Structure-Based Discovery of Opioid Analgesics with Reduced Side Effects</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>537</i></span>,  <span class="NLM_fpage">185</span>– <span class="NLM_lpage">190</span>, <span class="refDoi"> DOI: 10.1038/nature19112</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref182/cit182&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1038%2Fnature19112" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref182/cit182&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=27533032" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref182/cit182&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtlKmsbvI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=537&publication_year=2016&pages=185-190&author=A.+Manglikauthor=H.+Linauthor=D.+K.+Aryalauthor=J.+D.+McCorvyauthor=D.+Denglerauthor=G.+Corderauthor=A.+Levitauthor=R.+C.+Klingauthor=V.+Bernatauthor=H.+H%C3%BCbnerauthor=X.-P.+Huangauthor=M.+F.+Sassanoauthor=P.+M.+Gigu%C3%A8reauthor=S.+L%C3%B6berauthor=D.+Duanauthor=G.+Scherrerauthor=B.+K.+Kobilkaauthor=P.+Gmeinerauthor=B.+L.+Rothauthor=B.+K.+Shoichet&title=Structure-Based+Discovery+of+Opioid+Analgesics+with+Reduced+Side+Effects&doi=10.1038%2Fnature19112"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit182R"><div class="casContent"><span class="casTitleNuber">182</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-based discovery of opioid analgesics with reduced side effects</span></div><div class="casAuthors">Manglik, Aashish; Lin, Henry; Aryal, Dipendra K.; McCorvy, John D.; Dengler, Daniela; Corder, Gregory; Levit, Anat; Kling, Ralf C.; Bernat, Viachaslau; Hubner, Harald; Huang, Xi-Ping; Sassano, Maria F.; Giguere, Patrick M.; Lober, Stefan; Da Duan; Scherrer, Gregory; Kobilka, Brian K.; Gmeiner, Peter; Roth, Bryan L.; Shoichet, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">537</span>
        (<span class="NLM_cas:issue">7619</span>),
    <span class="NLM_cas:pages">185-190</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Morphine is an alkaloid from the opium poppy used to treat pain.  The potentially lethal side effects of morphine and related opioids-which include fatal respiratory depression-are thought to be mediated by μ-opioid-receptor (μOR) signalling through the β-arrestin pathway or by actions at other receptors.  Conversely, G-protein μOR signalling is thought to confer analgesia.  Here we computationally dock over 3 million mols. against the μOR structure and identify new scaffolds unrelated to known opioids.  Structure-based optimization yields PZM21-a potent Gi activator with exceptional selectivity for μOR and minimal β-arrestin-2 recruitment.  Unlike morphine, PZM21 is more efficacious for the affective component of analgesia vs. the reflexive component and is devoid of both respiratory depression and morphine-like reinforcing activity in mice at equi-analgesic doses.  PZM21 thus serves as both a probe to disentangle μOR signalling and a therapeutic lead that is devoid of many of the side effects of current opioids.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpiCff2w3U8iLVg90H21EOLACvtfcHk0lg_yMBs76WWMA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtlKmsbvI&md5=b8bb2d8575dadf4a6dc74d94c26d2c1a</span></div><a href="/servlet/linkout?suffix=cit182&amp;dbid=16384&amp;doi=10.1038%2Fnature19112&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature19112%26sid%3Dliteratum%253Aachs%26aulast%3DManglik%26aufirst%3DA.%26aulast%3DLin%26aufirst%3DH.%26aulast%3DAryal%26aufirst%3DD.%2BK.%26aulast%3DMcCorvy%26aufirst%3DJ.%2BD.%26aulast%3DDengler%26aufirst%3DD.%26aulast%3DCorder%26aufirst%3DG.%26aulast%3DLevit%26aufirst%3DA.%26aulast%3DKling%26aufirst%3DR.%2BC.%26aulast%3DBernat%26aufirst%3DV.%26aulast%3DH%25C3%25BCbner%26aufirst%3DH.%26aulast%3DHuang%26aufirst%3DX.-P.%26aulast%3DSassano%26aufirst%3DM.%2BF.%26aulast%3DGigu%25C3%25A8re%26aufirst%3DP.%2BM.%26aulast%3DL%25C3%25B6ber%26aufirst%3DS.%26aulast%3DDuan%26aufirst%3DD.%26aulast%3DScherrer%26aufirst%3DG.%26aulast%3DKobilka%26aufirst%3DB.%2BK.%26aulast%3DGmeiner%26aufirst%3DP.%26aulast%3DRoth%26aufirst%3DB.%2BL.%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DStructure-Based%2520Discovery%2520of%2520Opioid%2520Analgesics%2520with%2520Reduced%2520Side%2520Effects%26jtitle%3DNature%26date%3D2016%26volume%3D537%26spage%3D185%26epage%3D190%26doi%3D10.1038%2Fnature19112" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref183"><div class="reference"><strong class="refLabel"><a href="#ref183" class="rightTabRefNumLink">183</a></strong><div class="NLM_citation" id="rightTab-cit183"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Banerjee, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bartesaghi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Merk, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rao, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bulfer, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Green, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mroczkowski, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neitz, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wipf, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Falconieri, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deshaies, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Milne, J. L. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huryn, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arkin, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Subramaniam, S.</span></span> <span> </span><span class="NLM_article-title">2.3 Å Resolution Cryo-EM Structure of Human p97 and Mechanism of Allosteric Inhibition</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>351</i></span>,  <span class="NLM_fpage">871</span>– <span class="NLM_lpage">875</span>, <span class="refDoi"> DOI: 10.1126/science.aad7974</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref183/cit183&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1126%2Fscience.aad7974" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref183/cit183&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=26822609" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref183/cit183&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC28XisFOjsLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=351&publication_year=2016&pages=871-875&author=S.+Banerjeeauthor=A.+Bartesaghiauthor=A.+Merkauthor=P.+Raoauthor=S.+L.+Bulferauthor=Y.+Yanauthor=N.+Greenauthor=B.+Mroczkowskiauthor=R.+J.+Neitzauthor=P.+Wipfauthor=V.+Falconieriauthor=R.+J.+Deshaiesauthor=J.+L.+S.+Milneauthor=D.+Hurynauthor=M.+Arkinauthor=S.+Subramaniam&title=2.3+%C3%85+Resolution+Cryo-EM+Structure+of+Human+p97+and+Mechanism+of+Allosteric+Inhibition&doi=10.1126%2Fscience.aad7974"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit183R"><div class="casContent"><span class="casTitleNuber">183</span><div class="casTitle"><span class="NLM_cas:atitle">2.3 Å resolution cryo-EM structure of human p97 and mechanism of allosteric inhibition</span></div><div class="casAuthors">Banerjee, Soojay; Bartesaghi, Alberto; Merk, Alan; Rao, Prashant; Bulfer, Stacie L.; Yan, Yongzhao; Green, Neal; Mroczkowski, Barbara; Neitz, R. Jeffrey; Wipf, Peter; Falconieri, Veronica; Deshaies, Raymond J.; Milne, Jacqueline L. S.; Huryn, Donna; Arkin, Michelle; Subramaniam, Sriram</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">351</span>
        (<span class="NLM_cas:issue">6275</span>),
    <span class="NLM_cas:pages">871-875</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">P97 is a hexameric AAA+ ATPase that is an attractive target for cancer drug development.  We report cryo-electron microscopy (cryo-EM) structures for ADP-bound, full-length, hexameric wild-type p97 in the presence and absence of an allosteric inhibitor at resolns. of 2.3 and 2.4 angstroms, resp.  We also report cryo-EM structures (at resolns. of ∼3.3, 3.2, and 3.3 angstroms, resp.) for three distinct, coexisting functional states of p97 with occupancies of zero, one, or two mols. of adenosine 5'-O-(3-thiotriphosphate) (ATPγS) per protomer.  A large corkscrew-like change in mol. architecture, coupled with upward displacement of the N-terminal domain, is obsd. only when ATPγS is bound to both the D1 and D2 domains of the protomer.  These cryo-EM structures establish the sequence of nucleotide-driven structural changes in p97 at at. resoln.  They also enable elucidation of the binding mode of an allosteric small-mol. inhibitor to p97 and illustrate how inhibitor binding at the interface between the D1 and D2 domains prevents propagation of the conformational changes necessary for p97 function.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqhbtTnBPFaALVg90H21EOLACvtfcHk0lg7M8DIlBfmJg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XisFOjsLY%253D&md5=14f41313af9dad93f934d07c8bdfc138</span></div><a href="/servlet/linkout?suffix=cit183&amp;dbid=16384&amp;doi=10.1126%2Fscience.aad7974&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.aad7974%26sid%3Dliteratum%253Aachs%26aulast%3DBanerjee%26aufirst%3DS.%26aulast%3DBartesaghi%26aufirst%3DA.%26aulast%3DMerk%26aufirst%3DA.%26aulast%3DRao%26aufirst%3DP.%26aulast%3DBulfer%26aufirst%3DS.%2BL.%26aulast%3DYan%26aufirst%3DY.%26aulast%3DGreen%26aufirst%3DN.%26aulast%3DMroczkowski%26aufirst%3DB.%26aulast%3DNeitz%26aufirst%3DR.%2BJ.%26aulast%3DWipf%26aufirst%3DP.%26aulast%3DFalconieri%26aufirst%3DV.%26aulast%3DDeshaies%26aufirst%3DR.%2BJ.%26aulast%3DMilne%26aufirst%3DJ.%2BL.%2BS.%26aulast%3DHuryn%26aufirst%3DD.%26aulast%3DArkin%26aufirst%3DM.%26aulast%3DSubramaniam%26aufirst%3DS.%26atitle%3D2.3%2520%25C3%2585%2520Resolution%2520Cryo-EM%2520Structure%2520of%2520Human%2520p97%2520and%2520Mechanism%2520of%2520Allosteric%2520Inhibition%26jtitle%3DScience%26date%3D2016%26volume%3D351%26spage%3D871%26epage%3D875%26doi%3D10.1126%2Fscience.aad7974" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref184"><div class="reference"><strong class="refLabel"><a href="#ref184" class="rightTabRefNumLink">184</a></strong><div class="NLM_citation" id="rightTab-cit184"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Merk, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bartesaghi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Banerjee, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Falconieri, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rao, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, M. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pragani, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boxer, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Earl, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Milne, J. L. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Subramaniam, S.</span></span> <span> </span><span class="NLM_article-title">Breaking Cryo-EM Resolution Barriers to Facilitate Drug Discovery</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>165</i></span>,  <span class="NLM_fpage">1698</span>– <span class="NLM_lpage">1707</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2016.05.040</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref184/cit184&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1016%2Fj.cell.2016.05.040" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref184/cit184&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=27238019" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref184/cit184&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC28XptVSlsb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=165&publication_year=2016&pages=1698-1707&author=A.+Merkauthor=A.+Bartesaghiauthor=S.+Banerjeeauthor=V.+Falconieriauthor=P.+Raoauthor=M.+I.+Davisauthor=R.+Praganiauthor=M.+B.+Boxerauthor=L.+A.+Earlauthor=J.+L.+S.+Milneauthor=S.+Subramaniam&title=Breaking+Cryo-EM+Resolution+Barriers+to+Facilitate+Drug+Discovery&doi=10.1016%2Fj.cell.2016.05.040"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit184R"><div class="casContent"><span class="casTitleNuber">184</span><div class="casTitle"><span class="NLM_cas:atitle">Breaking Cryo-EM Resolution Barriers to Facilitate Drug Discovery</span></div><div class="casAuthors">Merk, Alan; Bartesaghi, Alberto; Banerjee, Soojay; Falconieri, Veronica; Rao, Prashant; Davis, Mindy I.; Pragani, Rajan; Boxer, Matthew B.; Earl, Lesley A.; Milne, Jacqueline L. S.; Subramaniam, Sriram</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">165</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1698-1707</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Recent advances in single-particle cryo-electron microscopy (cryo-EM) are enabling generation of numerous near-at. resoln. structures for well-ordered protein complexes with sizes ≥ ∼200 kDa.  Whether cryo-EM methods are equally useful for high-resoln. structural anal. of smaller, dynamic protein complexes such as those involved in cellular metab. remains an important question.  Here, we present 3.8 Å resoln. cryo-EM structures of the cancer target isocitrate dehydrogenase (93 kDa) and identify the nature of conformational changes induced by binding of the allosteric small-mol. inhibitor ML309.  We also report 2.8-Å- and 1.8-Å-resoln. structures of lactate dehydrogenase (145 kDa) and glutamate dehydrogenase (334 kDa), resp.  With these results, two perceived barriers in single-particle cryo-EM are overcome: (1) crossing 2 Å resoln. and (2) obtaining structures of proteins with sizes < 100 kDa, demonstrating that cryo-EM can be used to investigate a broad spectrum of drug-target interactions and dynamic conformational states.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrSVJ4hXEIN07Vg90H21EOLACvtfcHk0lg7M8DIlBfmJg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XptVSlsb8%253D&md5=8457195c1f1724244caa9f90e0ba4682</span></div><a href="/servlet/linkout?suffix=cit184&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2016.05.040&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2016.05.040%26sid%3Dliteratum%253Aachs%26aulast%3DMerk%26aufirst%3DA.%26aulast%3DBartesaghi%26aufirst%3DA.%26aulast%3DBanerjee%26aufirst%3DS.%26aulast%3DFalconieri%26aufirst%3DV.%26aulast%3DRao%26aufirst%3DP.%26aulast%3DDavis%26aufirst%3DM.%2BI.%26aulast%3DPragani%26aufirst%3DR.%26aulast%3DBoxer%26aufirst%3DM.%2BB.%26aulast%3DEarl%26aufirst%3DL.%2BA.%26aulast%3DMilne%26aufirst%3DJ.%2BL.%2BS.%26aulast%3DSubramaniam%26aufirst%3DS.%26atitle%3DBreaking%2520Cryo-EM%2520Resolution%2520Barriers%2520to%2520Facilitate%2520Drug%2520Discovery%26jtitle%3DCell%26date%3D2016%26volume%3D165%26spage%3D1698%26epage%3D1707%26doi%3D10.1016%2Fj.cell.2016.05.040" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref185"><div class="reference"><strong class="refLabel"><a href="#ref185" class="rightTabRefNumLink">185</a></strong><div class="NLM_citation" id="rightTab-cit185"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bai, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scheres, S. H. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, Y.</span></span> <span> </span><span class="NLM_article-title">An Atomic Structure of Human Γ-Secretase</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>525</i></span>,  <span class="NLM_fpage">212</span>– <span class="NLM_lpage">217</span>, <span class="refDoi"> DOI: 10.1038/nature14892</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref185/cit185&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1038%2Fnature14892" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref185/cit185&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=26280335" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref185/cit185&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtlGgurfM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=525&publication_year=2015&pages=212-217&author=X.+Baiauthor=C.+Yanauthor=G.+Yangauthor=P.+Luauthor=D.+Maauthor=L.+Sunauthor=R.+Zhouauthor=S.+H.+W.+Scheresauthor=Y.+Shi&title=An+Atomic+Structure+of+Human+%CE%93-Secretase&doi=10.1038%2Fnature14892"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit185R"><div class="casContent"><span class="casTitleNuber">185</span><div class="casTitle"><span class="NLM_cas:atitle">An atomic structure of human γ-secretase</span></div><div class="casAuthors">Bai, Xiao-chen; Yan, Chuangye; Yang, Guanghui; Lu, Peilong; Ma, Dan; Sun, Linfeng; Zhou, Rui; Scheres, Sjors H. W.; Shi, Yigong</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">525</span>
        (<span class="NLM_cas:issue">7568</span>),
    <span class="NLM_cas:pages">212-217</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Dysfunction of the intramembrane protease γ-secretase is thought to cause Alzheimer's disease, with most mutations derived from Alzheimer's disease mapping to the catalytic subunit presenilin 1 (PS1).  Here we report an at. structure of human γ-secretase at 3.4 Å resoln., detd. by single-particle cryo-electron microscopy.  Mutations derived from Alzheimer's disease affect residues at two hotspots in PS1, each located at the center of a distinct four transmembrane segment (TM) bundle.  TM2 and, to a lesser extent, TM6 exhibit considerable flexibility, yielding a plastic active site and adaptable surrounding elements.  The active site of PS1 is accessible from the convex side of the TM horseshoe, suggesting considerable conformational changes in nicastrin extracellular domain after substrate recruitment.  Component protein APH-1 serves as a scaffold, anchoring the lone transmembrane helix from nicastrin and supporting the flexible conformation of PS1.  Ordered phospholipids stabilize the complex inside the membrane.  Our structure serves as a mol. basis for mechanistic understanding of γ-secretase function.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqGo99dE6LPbbVg90H21EOLACvtfcHk0lg7M8DIlBfmJg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtlGgurfM&md5=68dd390cae90f02f8c6a3d6d9c503326</span></div><a href="/servlet/linkout?suffix=cit185&amp;dbid=16384&amp;doi=10.1038%2Fnature14892&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature14892%26sid%3Dliteratum%253Aachs%26aulast%3DBai%26aufirst%3DX.%26aulast%3DYan%26aufirst%3DC.%26aulast%3DYang%26aufirst%3DG.%26aulast%3DLu%26aufirst%3DP.%26aulast%3DMa%26aufirst%3DD.%26aulast%3DSun%26aufirst%3DL.%26aulast%3DZhou%26aufirst%3DR.%26aulast%3DScheres%26aufirst%3DS.%2BH.%2BW.%26aulast%3DShi%26aufirst%3DY.%26atitle%3DAn%2520Atomic%2520Structure%2520of%2520Human%2520%25CE%2593-Secretase%26jtitle%3DNature%26date%3D2015%26volume%3D525%26spage%3D212%26epage%3D217%26doi%3D10.1038%2Fnature14892" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref186"><div class="reference"><strong class="refLabel"><a href="#ref186" class="rightTabRefNumLink">186</a></strong><div class="NLM_citation" id="rightTab-cit186"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liao, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Julius, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, Y.</span></span> <span> </span><span class="NLM_article-title">Structure of the TRPV1 Ion Channel Determined by Electron Cryo-Microscopy</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>504</i></span>,  <span class="NLM_fpage">107</span>– <span class="NLM_lpage">112</span>, <span class="refDoi"> DOI: 10.1038/nature12822</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref186/cit186&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1038%2Fnature12822" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref186/cit186&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=24305160" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref186/cit186&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvVyis77N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=504&publication_year=2013&pages=107-112&author=M.+Liaoauthor=E.+Caoauthor=D.+Juliusauthor=Y.+Cheng&title=Structure+of+the+TRPV1+Ion+Channel+Determined+by+Electron+Cryo-Microscopy&doi=10.1038%2Fnature12822"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit186R"><div class="casContent"><span class="casTitleNuber">186</span><div class="casTitle"><span class="NLM_cas:atitle">Structure of the TRPV1 ion channel determined by electron cryo-microscopy</span></div><div class="casAuthors">Liao, Maofu; Cao, Erhu; Julius, David; Cheng, Yifan</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">504</span>
        (<span class="NLM_cas:issue">7478</span>),
    <span class="NLM_cas:pages">107-112</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Transient receptor potential (TRP) channels are sensors for a wide range of cellular and environmental signals, but elucidating how these channels respond to phys. and chem. stimuli has been hampered by a lack of detailed structural information.  Here, the authors exploited advances in electron cryo-microscopy to det. the structure of a mammalian TRP channel, TRPV1, at 3.4 Å resoln., breaking the side-chain resoln. barrier for membrane proteins without crystn.  Like voltage-gated channels, TRPV1 exhibited 4-fold symmetry around a central ion pathway formed by transmembrane segments 5-6 (S5-S6) and the intervening pore loop, which was flanked by S1-S4 voltage-sensor-like domains.  TRPV1 had a wide extracellular 'mouth' with a short selectivity filter.  The conserved 'TRP domain' interacted with the S4-S5 linker, consistent with its contribution to allosteric modulation.  Subunit organization was facilitated by interactions among cytoplasmic domains, including N-terminal ankyrin repeats.  These observations provide a structural blueprint for understanding unique aspects of TRP channel function.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqznsggkUfm_7Vg90H21EOLACvtfcHk0lg74ar8H-RZHA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvVyis77N&md5=bfccb367080288f78abf04dbc114769e</span></div><a href="/servlet/linkout?suffix=cit186&amp;dbid=16384&amp;doi=10.1038%2Fnature12822&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature12822%26sid%3Dliteratum%253Aachs%26aulast%3DLiao%26aufirst%3DM.%26aulast%3DCao%26aufirst%3DE.%26aulast%3DJulius%26aufirst%3DD.%26aulast%3DCheng%26aufirst%3DY.%26atitle%3DStructure%2520of%2520the%2520TRPV1%2520Ion%2520Channel%2520Determined%2520by%2520Electron%2520Cryo-Microscopy%26jtitle%3DNature%26date%3D2013%26volume%3D504%26spage%3D107%26epage%3D112%26doi%3D10.1038%2Fnature12822" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref187"><div class="reference"><strong class="refLabel"><a href="#ref187" class="rightTabRefNumLink">187</a></strong><div class="NLM_citation" id="rightTab-cit187"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span> <span> </span><span class="NLM_article-title">Atomic Structure of the Cystic Fibrosis Transmembrane Conductance Regulator</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>167</i></span>,  <span class="NLM_fpage">1586.e9</span>– <span class="NLM_lpage">1597.e9</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2016.11.014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref187/cit187&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1016%2Fj.cell.2016.11.014" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=167&publication_year=2016&pages=1586.e9-1597.e9&author=Z.+Zhangauthor=J.+Chen&title=Atomic+Structure+of+the+Cystic+Fibrosis+Transmembrane+Conductance+Regulator&doi=10.1016%2Fj.cell.2016.11.014"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit187&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2016.11.014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2016.11.014%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DZ.%26aulast%3DChen%26aufirst%3DJ.%26atitle%3DAtomic%2520Structure%2520of%2520the%2520Cystic%2520Fibrosis%2520Transmembrane%2520Conductance%2520Regulator%26jtitle%3DCell%26date%3D2016%26volume%3D167%26spage%3D1586.e9%26epage%3D1597.e9%26doi%3D10.1016%2Fj.cell.2016.11.014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref188"><div class="reference"><strong class="refLabel"><a href="#ref188" class="rightTabRefNumLink">188</a></strong><div class="NLM_citation" id="rightTab-cit188"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Neutze, R.</span></span> <span> </span><span class="NLM_article-title">Opportunities and Challenges for Time-Resolved Studies of Protein Structural Dynamics at X-Ray Free-Electron Lasers</span>. <i>Philos. Trans. R. Soc., B</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>369</i></span>,  <span class="NLM_fpage">20130318</span>, <span class="refDoi"> DOI: 10.1098/rstb.2013.0318</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref188/cit188&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1098%2Frstb.2013.0318" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref188/cit188&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=24914150" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref188/cit188&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtlaitL3P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=369&publication_year=2014&pages=20130318&author=R.+Neutze&title=Opportunities+and+Challenges+for+Time-Resolved+Studies+of+Protein+Structural+Dynamics+at+X-Ray+Free-Electron+Lasers&doi=10.1098%2Frstb.2013.0318"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit188R"><div class="casContent"><span class="casTitleNuber">188</span><div class="casTitle"><span class="NLM_cas:atitle">Opportunities and challenges for time-resolved studies of protein structural dynamics at X-ray free-electron lasers</span></div><div class="casAuthors">Neutze, Richard</div><div class="citationInfo"><span class="NLM_cas:title">Philosophical Transactions of the Royal Society, B: Biological Sciences</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">369</span>
        (<span class="NLM_cas:issue">1647</span>),
    <span class="NLM_cas:pages">20130318/1-20130318/9</span>CODEN:
                <span class="NLM_cas:coden">PTRBAE</span>;
        ISSN:<span class="NLM_cas:issn">0962-8436</span>.
    
            (<span class="NLM_cas:orgname">Royal Society</span>)
        </div><div class="casAbstract">A review.  X-ray free-electron lasers (XFELs) are revolutionary X-ray sources.  Their time structure, providing X-ray pulses of a few tens of femtoseconds in duration; and their extreme peak brilliance, delivering approx. 1012 X-ray photons per pulse and facilitating sub-micrometre focusing, distinguish XFEL sources from synchrotron radiation.  In this opinion piece, I argue that these properties of XFEL radiation will facilitate new discoveries in life science.  I reason that time-resolved serial femtosecond crystallog. and time-resolved wide angle X-ray scattering are promising areas of scientific investigation that will be advanced by XFEL capabilities, allowing new scientific questions to be addressed that are not accessible using established methods at storage ring facilities.  These questions include visualizing ultrafast protein structural dynamics on the femtosecond to picosecond time-scale, as well as time-resolved diffraction studies of non-cyclic reactions.  I argue that these emerging opportunities will stimulate a renaissance of interest in time-resolved structural biochem.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqdL7GMhFWDu7Vg90H21EOLACvtfcHk0lg74ar8H-RZHA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtlaitL3P&md5=66c77f7e3ae2e7bd004946dcba2027dd</span></div><a href="/servlet/linkout?suffix=cit188&amp;dbid=16384&amp;doi=10.1098%2Frstb.2013.0318&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1098%252Frstb.2013.0318%26sid%3Dliteratum%253Aachs%26aulast%3DNeutze%26aufirst%3DR.%26atitle%3DOpportunities%2520and%2520Challenges%2520for%2520Time-Resolved%2520Studies%2520of%2520Protein%2520Structural%2520Dynamics%2520at%2520X-Ray%2520Free-Electron%2520Lasers%26jtitle%3DPhilos.%2520Trans.%2520R.%2520Soc.%252C%2520B%26date%3D2014%26volume%3D369%26spage%3D20130318%26doi%3D10.1098%2Frstb.2013.0318" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref189"><div class="reference"><strong class="refLabel"><a href="#ref189" class="rightTabRefNumLink">189</a></strong><div class="NLM_citation" id="rightTab-cit189"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, G. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Batyuk, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ishchenko, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">White, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sadybekov, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zamlynny, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rudd, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hollenstein, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tolstikova, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">White, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hunter, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weierstall, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Babaoglu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moore, E. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Katz, R. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shipman, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garcia-Calvo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharma, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheth, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soisson, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stevens, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Katritch, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cherezov, V.</span></span> <span> </span><span class="NLM_article-title">Structural Basis for Selectivity and Diversity in Angiotensin II Receptors</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>544</i></span>,  <span class="NLM_fpage">327</span>– <span class="NLM_lpage">332</span>, <span class="refDoi"> DOI: 10.1038/nature22035</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref189/cit189&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1038%2Fnature22035" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref189/cit189&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=28379944" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref189/cit189&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC2sXlvVKhs7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=544&publication_year=2017&pages=327-332&author=H.+Zhangauthor=G.+W.+Hanauthor=A.+Batyukauthor=A.+Ishchenkoauthor=K.+L.+Whiteauthor=N.+Patelauthor=A.+Sadybekovauthor=B.+Zamlynnyauthor=M.+T.+Ruddauthor=K.+Hollensteinauthor=A.+Tolstikovaauthor=T.+A.+Whiteauthor=M.+S.+Hunterauthor=U.+Weierstallauthor=W.+Liuauthor=K.+Babaogluauthor=E.+L.+Mooreauthor=R.+D.+Katzauthor=J.+M.+Shipmanauthor=M.+Garcia-Calvoauthor=S.+Sharmaauthor=P.+Shethauthor=S.+M.+Soissonauthor=R.+C.+Stevensauthor=V.+Katritchauthor=V.+Cherezov&title=Structural+Basis+for+Selectivity+and+Diversity+in+Angiotensin+II+Receptors&doi=10.1038%2Fnature22035"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit189R"><div class="casContent"><span class="casTitleNuber">189</span><div class="casTitle"><span class="NLM_cas:atitle">Structural basis for selectivity and diversity in angiotensin II receptors</span></div><div class="casAuthors">Zhang, Haitao; Han, Gye Won; Batyuk, Alexander; Ishchenko, Andrii; White, Kate L.; Patel, Nilkanth; Sadybekov, Anastasiia; Zamlynny, Beata; Rudd, Michael T.; Hollenstein, Kaspar; Tolstikova, Alexandra; White, Thomas A.; Hunter, Mark S.; Weierstall, Uwe; Liu, Wei; Babaoglu, Kerim; Moore, Eric L.; Katz, Ryan D.; Shipman, Jennifer M.; Garcia-Calvo, Margarita; Sharma, Sujata; Sheth, Payal; Soisson, Stephen M.; Stevens, Raymond C.; Katritch, Vsevolod; Cherezov, Vadim</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">544</span>
        (<span class="NLM_cas:issue">7650</span>),
    <span class="NLM_cas:pages">327-332</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The angiotensin II receptors AT1R and AT2R serve as key components of the renin-angiotensin-aldosterone system.  AT1R has a central role in the regulation of blood pressure, but the function of AT2R is unclear and it has a variety of reported effects.  To identify the mechanisms that underlie the differences in function and ligand selectivity between these receptors, here the authors report crystal structures of human AT2R bound to an AT2R-selective ligand and to an AT1R/AT2R dual ligand, capturing the receptor in an active-like conformation.  Unexpectedly, helix VIII was found in a non-canonical position, stabilizing the active-like state, but at the same time preventing the recruitment of G proteins or β-arrestins, in agreement with the lack of signaling responses in std. cellular assays.  Structure-activity relationship, docking and mutagenesis studies revealed the crucial interactions for ligand binding and selectivity.  The authors' results thus provide insights into the structural basis of the distinct functions of the angiotensin receptors, and may guide the design of new selective ligands.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGonEzSnnYnMorVg90H21EOLACvtfcHk0lg74ar8H-RZHA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXlvVKhs7c%253D&md5=4473fb24c39c800b2126b1303c02afec</span></div><a href="/servlet/linkout?suffix=cit189&amp;dbid=16384&amp;doi=10.1038%2Fnature22035&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature22035%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DH.%26aulast%3DHan%26aufirst%3DG.%2BW.%26aulast%3DBatyuk%26aufirst%3DA.%26aulast%3DIshchenko%26aufirst%3DA.%26aulast%3DWhite%26aufirst%3DK.%2BL.%26aulast%3DPatel%26aufirst%3DN.%26aulast%3DSadybekov%26aufirst%3DA.%26aulast%3DZamlynny%26aufirst%3DB.%26aulast%3DRudd%26aufirst%3DM.%2BT.%26aulast%3DHollenstein%26aufirst%3DK.%26aulast%3DTolstikova%26aufirst%3DA.%26aulast%3DWhite%26aufirst%3DT.%2BA.%26aulast%3DHunter%26aufirst%3DM.%2BS.%26aulast%3DWeierstall%26aufirst%3DU.%26aulast%3DLiu%26aufirst%3DW.%26aulast%3DBabaoglu%26aufirst%3DK.%26aulast%3DMoore%26aufirst%3DE.%2BL.%26aulast%3DKatz%26aufirst%3DR.%2BD.%26aulast%3DShipman%26aufirst%3DJ.%2BM.%26aulast%3DGarcia-Calvo%26aufirst%3DM.%26aulast%3DSharma%26aufirst%3DS.%26aulast%3DSheth%26aufirst%3DP.%26aulast%3DSoisson%26aufirst%3DS.%2BM.%26aulast%3DStevens%26aufirst%3DR.%2BC.%26aulast%3DKatritch%26aufirst%3DV.%26aulast%3DCherezov%26aufirst%3DV.%26atitle%3DStructural%2520Basis%2520for%2520Selectivity%2520and%2520Diversity%2520in%2520Angiotensin%2520II%2520Receptors%26jtitle%3DNature%26date%3D2017%26volume%3D544%26spage%3D327%26epage%3D332%26doi%3D10.1038%2Fnature22035" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref190"><div class="reference"><strong class="refLabel"><a href="#ref190" class="rightTabRefNumLink">190</a></strong><div class="NLM_citation" id="rightTab-cit190"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ginn, H. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Messerschmidt, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ji, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Axford, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gildea, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winter, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brewster, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hattne, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wagner, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grimes, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evans, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sauter, N. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sutton, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stuart, D. I.</span></span> <span> </span><span class="NLM_article-title">Structure of CPV17 Polyhedrin Determined by the Improved Analysis of Serial Femtosecond Crystallographic Data</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">6435</span>, <span class="refDoi"> DOI: 10.1038/ncomms7435</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref190/cit190&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1038%2Fncomms7435" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref190/cit190&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=25751308" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref190/cit190&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC2MXosFelt7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=6435&author=H.+M.+Ginnauthor=M.+Messerschmidtauthor=X.+Jiauthor=H.+Zhangauthor=D.+Axfordauthor=R.+J.+Gildeaauthor=G.+Winterauthor=A.+S.+Brewsterauthor=J.+Hattneauthor=A.+Wagnerauthor=J.+M.+Grimesauthor=G.+Evansauthor=N.+K.+Sauterauthor=G.+Suttonauthor=D.+I.+Stuart&title=Structure+of+CPV17+Polyhedrin+Determined+by+the+Improved+Analysis+of+Serial+Femtosecond+Crystallographic+Data&doi=10.1038%2Fncomms7435"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit190R"><div class="casContent"><span class="casTitleNuber">190</span><div class="casTitle"><span class="NLM_cas:atitle">Structure of CPV17 polyhedrin determined by the improved analysis of serial femtosecond crystallographic data</span></div><div class="casAuthors">Ginn, Helen M.; Messerschmidt, Marc; Ji, Xiaoyun; Zhang, Hanwen; Axford, Danny; Gildea, Richard J.; Winter, Graeme; Brewster, Aaron S.; Hattne, Johan; Wagner, Armin; Grimes, Jonathan M.; Evans, Gwyndaf; Sauter, Nicholas K.; Sutton, Geoff; Stuart, David I.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Communications</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">6435</span>CODEN:
                <span class="NLM_cas:coden">NCAOBW</span>;
        ISSN:<span class="NLM_cas:issn">2041-1723</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The X-ray free-electron laser (XFEL) allows the anal. of small weakly diffracting protein crystals, but has required very many crystals to obtain good data.  Here we use an XFEL to det. the room temp. at. structure for the smallest cytoplasmic polyhedrosis virus polyhedra yet characterized, which we failed to solve at a synchrotron.  These protein microcrystals, roughly a micron across, accrue within infected cells.  We use a new phys. model for XFEL diffraction, which better ests. the exptl. signal, delivering a high-resoln. XFEL structure (1.75 Å), using fewer crystals than previously required for this resoln.  The crystal lattice and protein core are conserved compared with a polyhedrin with less than 10% sequence identity.  We explain how the conserved biol. phenotype, the crystal lattice, is maintained in the face of extreme environmental challenge and massive evolutionary divergence.  Our improved methods should open up more challenging biol. samples to XFEL anal.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqk0LC5QFL7cLVg90H21EOLACvtfcHk0liYR2ZTAhkcpg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXosFelt7o%253D&md5=de15ffb7fc082e8661e4355cc7397ae1</span></div><a href="/servlet/linkout?suffix=cit190&amp;dbid=16384&amp;doi=10.1038%2Fncomms7435&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms7435%26sid%3Dliteratum%253Aachs%26aulast%3DGinn%26aufirst%3DH.%2BM.%26aulast%3DMesserschmidt%26aufirst%3DM.%26aulast%3DJi%26aufirst%3DX.%26aulast%3DZhang%26aufirst%3DH.%26aulast%3DAxford%26aufirst%3DD.%26aulast%3DGildea%26aufirst%3DR.%2BJ.%26aulast%3DWinter%26aufirst%3DG.%26aulast%3DBrewster%26aufirst%3DA.%2BS.%26aulast%3DHattne%26aufirst%3DJ.%26aulast%3DWagner%26aufirst%3DA.%26aulast%3DGrimes%26aufirst%3DJ.%2BM.%26aulast%3DEvans%26aufirst%3DG.%26aulast%3DSauter%26aufirst%3DN.%2BK.%26aulast%3DSutton%26aufirst%3DG.%26aulast%3DStuart%26aufirst%3DD.%2BI.%26atitle%3DStructure%2520of%2520CPV17%2520Polyhedrin%2520Determined%2520by%2520the%2520Improved%2520Analysis%2520of%2520Serial%2520Femtosecond%2520Crystallographic%2520Data%26jtitle%3DNat.%2520Commun.%26date%3D2015%26volume%3D6%26spage%3D6435%26doi%3D10.1038%2Fncomms7435" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref191"><div class="reference"><strong class="refLabel"><a href="#ref191" class="rightTabRefNumLink">191</a></strong><div class="NLM_citation" id="rightTab-cit191"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bublitz, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nass, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drachmann, N. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Markvardsen, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gutmann, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barends, T. R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mattle, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shoeman, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doak, R. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boutet, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Messerschmidt, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seibert, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williams, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Foucar, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reinhard, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sitsel, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gregersen, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clausen, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boesen, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gotfryd, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, K.-T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olesen, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Møller, J. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nissen, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schlichting, I.</span></span> <span> </span><span class="NLM_article-title">Structural Studies of P-Type ATPase–ligand Complexes Using an X-Ray Free-Electron Laser</span>. <i>IUCrJ</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>2</i></span>,  <span class="NLM_fpage">409</span>– <span class="NLM_lpage">420</span>, <span class="refDoi"> DOI: 10.1107/S2052252515008969</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref191/cit191&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1107%2FS2052252515008969" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2015&pages=409-420&author=M.+Bublitzauthor=K.+Nassauthor=N.+D.+Drachmannauthor=A.+J.+Markvardsenauthor=M.+J.+Gutmannauthor=T.+R.+M.+Barendsauthor=D.+Mattleauthor=R.+L.+Shoemanauthor=R.+B.+Doakauthor=S.+Boutetauthor=M.+Messerschmidtauthor=M.+M.+Seibertauthor=G.+J.+Williamsauthor=L.+Foucarauthor=L.+Reinhardauthor=O.+Sitselauthor=J.+L.+Gregersenauthor=J.+D.+Clausenauthor=T.+Boesenauthor=K.+Gotfrydauthor=K.-T.+Wangauthor=C.+Olesenauthor=J.+V.+M%C3%B8llerauthor=P.+Nissenauthor=I.+Schlichting&title=Structural+Studies+of+P-Type+ATPase%E2%80%93ligand+Complexes+Using+an+X-Ray+Free-Electron+Laser&doi=10.1107%2FS2052252515008969"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit191&amp;dbid=16384&amp;doi=10.1107%2FS2052252515008969&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS2052252515008969%26sid%3Dliteratum%253Aachs%26aulast%3DBublitz%26aufirst%3DM.%26aulast%3DNass%26aufirst%3DK.%26aulast%3DDrachmann%26aufirst%3DN.%2BD.%26aulast%3DMarkvardsen%26aufirst%3DA.%2BJ.%26aulast%3DGutmann%26aufirst%3DM.%2BJ.%26aulast%3DBarends%26aufirst%3DT.%2BR.%2BM.%26aulast%3DMattle%26aufirst%3DD.%26aulast%3DShoeman%26aufirst%3DR.%2BL.%26aulast%3DDoak%26aufirst%3DR.%2BB.%26aulast%3DBoutet%26aufirst%3DS.%26aulast%3DMesserschmidt%26aufirst%3DM.%26aulast%3DSeibert%26aufirst%3DM.%2BM.%26aulast%3DWilliams%26aufirst%3DG.%2BJ.%26aulast%3DFoucar%26aufirst%3DL.%26aulast%3DReinhard%26aufirst%3DL.%26aulast%3DSitsel%26aufirst%3DO.%26aulast%3DGregersen%26aufirst%3DJ.%2BL.%26aulast%3DClausen%26aufirst%3DJ.%2BD.%26aulast%3DBoesen%26aufirst%3DT.%26aulast%3DGotfryd%26aufirst%3DK.%26aulast%3DWang%26aufirst%3DK.-T.%26aulast%3DOlesen%26aufirst%3DC.%26aulast%3DM%25C3%25B8ller%26aufirst%3DJ.%2BV.%26aulast%3DNissen%26aufirst%3DP.%26aulast%3DSchlichting%26aufirst%3DI.%26atitle%3DStructural%2520Studies%2520of%2520P-Type%2520ATPase%25E2%2580%2593ligand%2520Complexes%2520Using%2520an%2520X-Ray%2520Free-Electron%2520Laser%26jtitle%3DIUCrJ%26date%3D2015%26volume%3D2%26spage%3D409%26epage%3D420%26doi%3D10.1107%2FS2052252515008969" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref192"><div class="reference"><strong class="refLabel"><a href="#ref192" class="rightTabRefNumLink">192</a></strong><div class="NLM_citation" id="rightTab-cit192"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Thomaston, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woldeyes, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakane, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamashita, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanaka, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koiwai, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brewster, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barad, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lemmin, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uervirojnangkoorn, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arima, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kobayashi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Masuda, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suzuki, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sugahara, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sauter, N. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanaka, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nureki, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tono, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Joti, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nango, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iwata, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yumoto, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fraser, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DeGrado, W. F.</span></span> <span> </span><span class="NLM_article-title">XFEL Structures of the Influenza M2 Proton Channel: Room Temperature Water Networks and Insights into Proton Conduction</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>114</i></span>,  <span class="NLM_fpage">13357</span>– <span class="NLM_lpage">13362</span>, <span class="refDoi"> DOI: 10.1073/pnas.1705624114</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref192/cit192&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1073%2Fpnas.1705624114" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref192/cit192&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=28835537" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref192/cit192&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtlOhurbM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=114&publication_year=2017&pages=13357-13362&author=J.+L.+Thomastonauthor=R.+A.+Woldeyesauthor=T.+Nakaneauthor=A.+Yamashitaauthor=T.+Tanakaauthor=K.+Koiwaiauthor=A.+S.+Brewsterauthor=B.+A.+Baradauthor=Y.+Chenauthor=T.+Lemminauthor=M.+Uervirojnangkoornauthor=T.+Arimaauthor=J.+Kobayashiauthor=T.+Masudaauthor=M.+Suzukiauthor=M.+Sugaharaauthor=N.+K.+Sauterauthor=R.+Tanakaauthor=O.+Nurekiauthor=K.+Tonoauthor=Y.+Jotiauthor=E.+Nangoauthor=S.+Iwataauthor=F.+Yumotoauthor=J.+S.+Fraserauthor=W.+F.+DeGrado&title=XFEL+Structures+of+the+Influenza+M2+Proton+Channel%3A+Room+Temperature+Water+Networks+and+Insights+into+Proton+Conduction&doi=10.1073%2Fpnas.1705624114"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit192R"><div class="casContent"><span class="casTitleNuber">192</span><div class="casTitle"><span class="NLM_cas:atitle">XFEL structures of the influenza M2 proton channel: Room temperature water networks and insights into proton conduction</span></div><div class="casAuthors">Thomaston, Jessica L.; Woldeyes, Rahel A.; Nakane, Takanori; Yamashita, Ayumi; Tanaka, Tomoyuki; Koiwai, Kotaro; Brewster, Aaron S.; Barad, Benjamin A.; Chen, Yujie; Lemmin, Thomas; Uervirojnangkoorn, Monarin; Arima, Toshi; Kobayashi, Jun; Masuda, Tetsuya; Suzuki, Mamoru; Sugahara, Michihiro; Sauter, Nicholas K.; Tanaka, Rie; Nureki, Osamu; Tono, Kensuke; Joti, Yasumasa; Nango, Eriko; Iwata, So; Yumoto, Fumiaki; Fraser, James S.; DeGrado, William F.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">114</span>
        (<span class="NLM_cas:issue">51</span>),
    <span class="NLM_cas:pages">13357-13362</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The M2 proton channel of influenza A virus is a drug target that is essential for the reprodn. of the virus.  It is also a model system for the study of selective, unidirectional proton transport across a membrane.  Ordered water mols. arranged in "wires" inside the channel pore have been proposed to play a role in both the conduction of protons to the 4 gating His-37 residues and the stabilization of multiple pos. charges within the channel.  To visualize the solvent in the pore of the channel at room temp. while minimizing the effects of radiation damage, data were collected to a resoln. of 1.4 Å using an X-ray free-electron laser (XFEL) at 3 different pH conditions: pH 5.5, pH 6.5, and pH 8.0.  Data were collected on the Inwardopen state, which is an intermediate that accumulates at high protonation of the His-37 tetrad.  At pH 5.5, a continuous H-bonded network of water mols. spanned the vertical length of the channel, consistent with a Grotthuss mechanism model for proton transport to the His-37 tetrad.  This ordered solvent at pH 5.5 could act to stabilize the pos. charges that build up on the gating His-37 tetrad during the proton conduction cycle.  The no. of ordered pore waters decreased at pH 6.5 and 8.0, where the Inwardopen state was less stable.  These studies provide a graphical view of the response of water to a change in charge within a restricted channel environment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqgzczipMcns7Vg90H21EOLACvtfcHk0liYR2ZTAhkcpg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtlOhurbM&md5=d44a62000931d80660980d2613efa7e9</span></div><a href="/servlet/linkout?suffix=cit192&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1705624114&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1705624114%26sid%3Dliteratum%253Aachs%26aulast%3DThomaston%26aufirst%3DJ.%2BL.%26aulast%3DWoldeyes%26aufirst%3DR.%2BA.%26aulast%3DNakane%26aufirst%3DT.%26aulast%3DYamashita%26aufirst%3DA.%26aulast%3DTanaka%26aufirst%3DT.%26aulast%3DKoiwai%26aufirst%3DK.%26aulast%3DBrewster%26aufirst%3DA.%2BS.%26aulast%3DBarad%26aufirst%3DB.%2BA.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DLemmin%26aufirst%3DT.%26aulast%3DUervirojnangkoorn%26aufirst%3DM.%26aulast%3DArima%26aufirst%3DT.%26aulast%3DKobayashi%26aufirst%3DJ.%26aulast%3DMasuda%26aufirst%3DT.%26aulast%3DSuzuki%26aufirst%3DM.%26aulast%3DSugahara%26aufirst%3DM.%26aulast%3DSauter%26aufirst%3DN.%2BK.%26aulast%3DTanaka%26aufirst%3DR.%26aulast%3DNureki%26aufirst%3DO.%26aulast%3DTono%26aufirst%3DK.%26aulast%3DJoti%26aufirst%3DY.%26aulast%3DNango%26aufirst%3DE.%26aulast%3DIwata%26aufirst%3DS.%26aulast%3DYumoto%26aufirst%3DF.%26aulast%3DFraser%26aufirst%3DJ.%2BS.%26aulast%3DDeGrado%26aufirst%3DW.%2BF.%26atitle%3DXFEL%2520Structures%2520of%2520the%2520Influenza%2520M2%2520Proton%2520Channel%253A%2520Room%2520Temperature%2520Water%2520Networks%2520and%2520Insights%2520into%2520Proton%2520Conduction%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2017%26volume%3D114%26spage%3D13357%26epage%3D13362%26doi%3D10.1073%2Fpnas.1705624114" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref193"><div class="reference"><strong class="refLabel"><a href="#ref193" class="rightTabRefNumLink">193</a></strong><div class="NLM_citation" id="rightTab-cit193"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Weierstall, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">James, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">White, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spence, J. C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doak, R. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nelson, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fromme, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fromme, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grotjohann, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kupitz, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zatsepin, N. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Basu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wacker, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, G. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Katritch, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boutet, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Messerschmidt, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williams, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koglin, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seibert, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klinker, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gati, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shoeman, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barty, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chapman, H. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirian, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beyerlein, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stevens, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shah, S. T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Howe, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caffrey, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cherezov, V.</span></span> <span> </span><span class="NLM_article-title">Lipidic Cubic Phase Injector Facilitates Membrane Protein Serial Femtosecond Crystallography</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">3309</span>, <span class="refDoi"> DOI: 10.1038/ncomms4309</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref193/cit193&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1038%2Fncomms4309" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref193/cit193&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=24525480" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref193/cit193&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A280%3ADC%252BC2cvltVeqsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&pages=3309&author=U.+Weierstallauthor=D.+Jamesauthor=C.+Wangauthor=T.+A.+Whiteauthor=D.+Wangauthor=W.+Liuauthor=J.+C.+H.+Spenceauthor=R.+B.+Doakauthor=G.+Nelsonauthor=P.+Frommeauthor=R.+Frommeauthor=I.+Grotjohannauthor=C.+Kupitzauthor=N.+A.+Zatsepinauthor=H.+Liuauthor=S.+Basuauthor=D.+Wackerauthor=G.+W.+Hanauthor=V.+Katritchauthor=S.+Boutetauthor=M.+Messerschmidtauthor=G.+J.+Williamsauthor=J.+E.+Koglinauthor=M.+M.+Seibertauthor=M.+Klinkerauthor=C.+Gatiauthor=R.+L.+Shoemanauthor=A.+Bartyauthor=H.+N.+Chapmanauthor=R.+A.+Kirianauthor=K.+R.+Beyerleinauthor=R.+C.+Stevensauthor=D.+Liauthor=S.+T.+A.+Shahauthor=N.+Howeauthor=M.+Caffreyauthor=V.+Cherezov&title=Lipidic+Cubic+Phase+Injector+Facilitates+Membrane+Protein+Serial+Femtosecond+Crystallography&doi=10.1038%2Fncomms4309"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit193R"><div class="casContent"><span class="casTitleNuber">193</span><div class="casTitle"><span class="NLM_cas:atitle">Lipidic cubic phase injector facilitates membrane protein serial femtosecond crystallography</span></div><div class="casAuthors">Weierstall Uwe; James Daniel; Wang Dingjie; Spence John C H; Bruce Doak R; Nelson Garrett; Zatsepin Nadia A; Liu Haiguang; Wang Chong; Liu Wei; Wacker Daniel; Han Gye Won; Katritch Vsevolod; Stevens Raymond C; Cherezov Vadim; White Thomas A; Gati Cornelius; Barty Anton; Beyerlein Kenneth R; Fromme Petra; Fromme Raimund; Grotjohann Ingo; Kupitz Christopher; Basu Shibom; Boutet Sebastien; Messerschmidt Marc; Williams Garth J; Koglin Jason E; Marvin Seibert M; Klinker Markus; Shoeman Robert L; Chapman Henry N; Kirian Richard A; Li Dianfan; Shah Syed T A; Howe Nicole; Caffrey Martin</div><div class="citationInfo"><span class="NLM_cas:title">Nature communications</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">3309</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Lipidic cubic phase (LCP) crystallization has proven successful for high-resolution structure determination of challenging membrane proteins.  Here we present a technique for extruding gel-like LCP with embedded membrane protein microcrystals, providing a continuously renewed source of material for serial femtosecond crystallography.  Data collected from sub-10-μm-sized crystals produced with less than 0.5 mg of purified protein yield structural insights regarding cyclopamine binding to the Smoothened receptor.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQDUg0uqQckxOWhZTslLrfffW6udTcc2eZt9vKNly35Bbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2cvltVeqsA%253D%253D&md5=ce335fa0a5bfeb8990c836e7f425e2d2</span></div><a href="/servlet/linkout?suffix=cit193&amp;dbid=16384&amp;doi=10.1038%2Fncomms4309&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms4309%26sid%3Dliteratum%253Aachs%26aulast%3DWeierstall%26aufirst%3DU.%26aulast%3DJames%26aufirst%3DD.%26aulast%3DWang%26aufirst%3DC.%26aulast%3DWhite%26aufirst%3DT.%2BA.%26aulast%3DWang%26aufirst%3DD.%26aulast%3DLiu%26aufirst%3DW.%26aulast%3DSpence%26aufirst%3DJ.%2BC.%2BH.%26aulast%3DDoak%26aufirst%3DR.%2BB.%26aulast%3DNelson%26aufirst%3DG.%26aulast%3DFromme%26aufirst%3DP.%26aulast%3DFromme%26aufirst%3DR.%26aulast%3DGrotjohann%26aufirst%3DI.%26aulast%3DKupitz%26aufirst%3DC.%26aulast%3DZatsepin%26aufirst%3DN.%2BA.%26aulast%3DLiu%26aufirst%3DH.%26aulast%3DBasu%26aufirst%3DS.%26aulast%3DWacker%26aufirst%3DD.%26aulast%3DHan%26aufirst%3DG.%2BW.%26aulast%3DKatritch%26aufirst%3DV.%26aulast%3DBoutet%26aufirst%3DS.%26aulast%3DMesserschmidt%26aufirst%3DM.%26aulast%3DWilliams%26aufirst%3DG.%2BJ.%26aulast%3DKoglin%26aufirst%3DJ.%2BE.%26aulast%3DSeibert%26aufirst%3DM.%2BM.%26aulast%3DKlinker%26aufirst%3DM.%26aulast%3DGati%26aufirst%3DC.%26aulast%3DShoeman%26aufirst%3DR.%2BL.%26aulast%3DBarty%26aufirst%3DA.%26aulast%3DChapman%26aufirst%3DH.%2BN.%26aulast%3DKirian%26aufirst%3DR.%2BA.%26aulast%3DBeyerlein%26aufirst%3DK.%2BR.%26aulast%3DStevens%26aufirst%3DR.%2BC.%26aulast%3DLi%26aufirst%3DD.%26aulast%3DShah%26aufirst%3DS.%2BT.%2BA.%26aulast%3DHowe%26aufirst%3DN.%26aulast%3DCaffrey%26aufirst%3DM.%26aulast%3DCherezov%26aufirst%3DV.%26atitle%3DLipidic%2520Cubic%2520Phase%2520Injector%2520Facilitates%2520Membrane%2520Protein%2520Serial%2520Femtosecond%2520Crystallography%26jtitle%3DNat.%2520Commun.%26date%3D2014%26volume%3D5%26spage%3D3309%26doi%3D10.1038%2Fncomms4309" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref194"><div class="reference"><strong class="refLabel"><a href="#ref194" class="rightTabRefNumLink">194</a></strong><div class="NLM_citation" id="rightTab-cit194"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wacker, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gati, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, G. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">James, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nelson, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weierstall, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Katritch, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barty, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zatsepin, N. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Messerschmidt, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boutet, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williams, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koglin, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seibert, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shah, S. T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Basu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fromme, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kupitz, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rendek, K. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grotjohann, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fromme, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirian, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beyerlein, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">White, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chapman, H. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caffrey, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spence, J. C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stevens, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cherezov, V.</span></span> <span> </span><span class="NLM_article-title">Serial Femtosecond Crystallography of G Protein–Coupled Receptors</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>342</i></span>,  <span class="NLM_fpage">1521</span>– <span class="NLM_lpage">1524</span>, <span class="refDoi"> DOI: 10.1126/science.1244142</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref194/cit194&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1126%2Fscience.1244142" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref194/cit194&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=24357322" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref194/cit194&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvFemu7zE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=342&publication_year=2013&pages=1521-1524&author=W.+Liuauthor=D.+Wackerauthor=C.+Gatiauthor=G.+W.+Hanauthor=D.+Jamesauthor=D.+Wangauthor=G.+Nelsonauthor=U.+Weierstallauthor=V.+Katritchauthor=A.+Bartyauthor=N.+A.+Zatsepinauthor=D.+Liauthor=M.+Messerschmidtauthor=S.+Boutetauthor=G.+J.+Williamsauthor=J.+E.+Koglinauthor=M.+M.+Seibertauthor=C.+Wangauthor=S.+T.+A.+Shahauthor=S.+Basuauthor=R.+Frommeauthor=C.+Kupitzauthor=K.+N.+Rendekauthor=I.+Grotjohannauthor=P.+Frommeauthor=R.+A.+Kirianauthor=K.+R.+Beyerleinauthor=T.+A.+Whiteauthor=H.+N.+Chapmanauthor=M.+Caffreyauthor=J.+C.+H.+Spenceauthor=R.+C.+Stevensauthor=V.+Cherezov&title=Serial+Femtosecond+Crystallography+of+G+Protein%E2%80%93Coupled+Receptors&doi=10.1126%2Fscience.1244142"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit194R"><div class="casContent"><span class="casTitleNuber">194</span><div class="casTitle"><span class="NLM_cas:atitle">Serial Femtosecond Crystallography of G Protein-Coupled Receptors</span></div><div class="casAuthors">Liu, Wei; Wacker, Daniel; Gati, Cornelius; Han, Gye Won; James, Daniel; Wang, Dingjie; Nelson, Garrett; Weierstall, Uwe; Katritch, Vsevolod; Barty, Anton; Zatsepin, Nadia A.; Li, Dianfan; Messerschmidt, Marc; Boutet, Sebastien; Williams, Garth J.; Koglin, Jason E.; Seibert, M. Marvin; Wang, Chong; Shah, Syed T. A.; Basu, Shibom; Fromme, Raimund; Kupitz, Christopher; Rendek, Kimberley N.; Grotjohann, Ingo; Fromme, Petra; Kirian, Richard A.; Beyerlein, Kenneth R.; White, Thomas A.; Chapman, Henry N.; Caffrey, Martin; Spence, John C. H.; Stevens, Raymond C.; Cherezov, Vadim</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">342</span>
        (<span class="NLM_cas:issue">6165</span>),
    <span class="NLM_cas:pages">1521-1524</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">X-ray crystallog. of G protein-coupled receptors and other membrane proteins is hampered by difficulties assocd. with growing sufficiently large crystals that withstand radiation damage and yield high-resoln. data at synchrotron sources.  The authors used an x-ray free-electron laser (XFEL) with individual 50-fs-duration x-ray pulses to minimize radiation damage and obtained a high-resoln. room-temp. structure of a human serotonin receptor using sub-10-μm microcrystals grown in a membrane mimetic matrix known as lipidic cubic phase.  Compared with the structure solved by using traditional microcrystallog. from cryo-cooled crystals of about two orders of magnitude larger vol., the room-temp. XFEL structure displays a distinct distribution of thermal motions and conformations of residues that likely more accurately represent the receptor structure and dynamics in a cellular environment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo5GiymqAvRprVg90H21EOLACvtfcHk0ljPdg6oXySCIQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvFemu7zE&md5=a632160c4265cddfebb28ef9b002b38a</span></div><a href="/servlet/linkout?suffix=cit194&amp;dbid=16384&amp;doi=10.1126%2Fscience.1244142&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1244142%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DW.%26aulast%3DWacker%26aufirst%3DD.%26aulast%3DGati%26aufirst%3DC.%26aulast%3DHan%26aufirst%3DG.%2BW.%26aulast%3DJames%26aufirst%3DD.%26aulast%3DWang%26aufirst%3DD.%26aulast%3DNelson%26aufirst%3DG.%26aulast%3DWeierstall%26aufirst%3DU.%26aulast%3DKatritch%26aufirst%3DV.%26aulast%3DBarty%26aufirst%3DA.%26aulast%3DZatsepin%26aufirst%3DN.%2BA.%26aulast%3DLi%26aufirst%3DD.%26aulast%3DMesserschmidt%26aufirst%3DM.%26aulast%3DBoutet%26aufirst%3DS.%26aulast%3DWilliams%26aufirst%3DG.%2BJ.%26aulast%3DKoglin%26aufirst%3DJ.%2BE.%26aulast%3DSeibert%26aufirst%3DM.%2BM.%26aulast%3DWang%26aufirst%3DC.%26aulast%3DShah%26aufirst%3DS.%2BT.%2BA.%26aulast%3DBasu%26aufirst%3DS.%26aulast%3DFromme%26aufirst%3DR.%26aulast%3DKupitz%26aufirst%3DC.%26aulast%3DRendek%26aufirst%3DK.%2BN.%26aulast%3DGrotjohann%26aufirst%3DI.%26aulast%3DFromme%26aufirst%3DP.%26aulast%3DKirian%26aufirst%3DR.%2BA.%26aulast%3DBeyerlein%26aufirst%3DK.%2BR.%26aulast%3DWhite%26aufirst%3DT.%2BA.%26aulast%3DChapman%26aufirst%3DH.%2BN.%26aulast%3DCaffrey%26aufirst%3DM.%26aulast%3DSpence%26aufirst%3DJ.%2BC.%2BH.%26aulast%3DStevens%26aufirst%3DR.%2BC.%26aulast%3DCherezov%26aufirst%3DV.%26atitle%3DSerial%2520Femtosecond%2520Crystallography%2520of%2520G%2520Protein%25E2%2580%2593Coupled%2520Receptors%26jtitle%3DScience%26date%3D2013%26volume%3D342%26spage%3D1521%26epage%3D1524%26doi%3D10.1126%2Fscience.1244142" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref195"><div class="reference"><strong class="refLabel"><a href="#ref195" class="rightTabRefNumLink">195</a></strong><div class="NLM_citation" id="rightTab-cit195"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Charlesworth, C. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deshpande, P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dever, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dejene, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gomez-Ospina, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mantri, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pavel-Dinu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Camarena, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weinberg, K. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Porteus, M. H.</span></span> <span> </span><span class="NLM_article-title">Identification of Pre-Existing Adaptive Immunity to Cas9 Proteins in Humans</span>. <i>bioRxiv</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_fpage">243345</span>, <span class="refDoi"> DOI: 10.1101/243345</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref195/cit195&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1101%2F243345" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2018&pages=243345&author=C.+T.+Charlesworthauthor=P.+S.+Deshpandeauthor=D.+P.+Deverauthor=B.+Dejeneauthor=N.+Gomez-Ospinaauthor=S.+Mantriauthor=M.+Pavel-Dinuauthor=J.+Camarenaauthor=K.+I.+Weinbergauthor=M.+H.+Porteus&title=Identification+of+Pre-Existing+Adaptive+Immunity+to+Cas9+Proteins+in+Humans&doi=10.1101%2F243345"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit195&amp;dbid=16384&amp;doi=10.1101%2F243345&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252F243345%26sid%3Dliteratum%253Aachs%26aulast%3DCharlesworth%26aufirst%3DC.%2BT.%26aulast%3DDeshpande%26aufirst%3DP.%2BS.%26aulast%3DDever%26aufirst%3DD.%2BP.%26aulast%3DDejene%26aufirst%3DB.%26aulast%3DGomez-Ospina%26aufirst%3DN.%26aulast%3DMantri%26aufirst%3DS.%26aulast%3DPavel-Dinu%26aufirst%3DM.%26aulast%3DCamarena%26aufirst%3DJ.%26aulast%3DWeinberg%26aufirst%3DK.%2BI.%26aulast%3DPorteus%26aufirst%3DM.%2BH.%26atitle%3DIdentification%2520of%2520Pre-Existing%2520Adaptive%2520Immunity%2520to%2520Cas9%2520Proteins%2520in%2520Humans%26jtitle%3DbioRxiv%26date%3D2018%26spage%3D243345%26doi%3D10.1101%2F243345" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref196"><div class="reference"><strong class="refLabel"><a href="#ref196" class="rightTabRefNumLink">196</a></strong><div class="NLM_citation" id="rightTab-cit196"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rizvi, N. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, G. F.</span></span> <span> </span><span class="NLM_article-title">RNA as a Small Molecule Druggable Target</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">5083</span>– <span class="NLM_lpage">5088</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2017.10.052</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref196/cit196&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1016%2Fj.bmcl.2017.10.052" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref196/cit196&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=29097169" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref196/cit196&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhslemtLfF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2017&pages=5083-5088&author=N.+F.+Rizviauthor=G.+F.+Smith&title=RNA+as+a+Small+Molecule+Druggable+Target&doi=10.1016%2Fj.bmcl.2017.10.052"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit196R"><div class="casContent"><span class="casTitleNuber">196</span><div class="casTitle"><span class="NLM_cas:atitle">RNA as a small molecule druggable target</span></div><div class="casAuthors">Rizvi, Noreen F.; Smith, Graham F.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">5083-5088</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Small mol. drugs have readily been developed against many proteins in the human proteome, but RNA has remained an elusive target for drug discovery.  Increasingly, the authors see that RNA, and to a lesser extent DNA elements, show a persistent tertiary structure responsible for many diverse and complex cellular functions.  In this digest, the authors have summarized recent advances in screening approaches for RNA targets and outlined the discovery of novel, drug-like small mols. against RNA targets from various classes and therapeutic areas.  The link of structure, function, and small-mol. Druggability validates now for the first time that RNA can be the targets of therapeutic agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpXQHvz_K9pGLVg90H21EOLACvtfcHk0lg7oN3eZtks4A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhslemtLfF&md5=ba213f6e8954c97239ccc23f7096c13e</span></div><a href="/servlet/linkout?suffix=cit196&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2017.10.052&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2017.10.052%26sid%3Dliteratum%253Aachs%26aulast%3DRizvi%26aufirst%3DN.%2BF.%26aulast%3DSmith%26aufirst%3DG.%2BF.%26atitle%3DRNA%2520as%2520a%2520Small%2520Molecule%2520Druggable%2520Target%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2017%26volume%3D27%26spage%3D5083%26epage%3D5088%26doi%3D10.1016%2Fj.bmcl.2017.10.052" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref197"><div class="reference"><strong class="refLabel"><a href="#ref197" class="rightTabRefNumLink">197</a></strong><div class="NLM_citation" id="rightTab-cit197"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Collie, G. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parkinson, G. N.</span></span> <span> </span><span class="NLM_article-title">The Application of DNA and RNA G-Quadruplexes to Therapeutic Medicines</span>. <i>Chem. Soc. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>40</i></span>,  <span class="NLM_fpage">5867</span>– <span class="NLM_lpage">5892</span>, <span class="refDoi"> DOI: 10.1039/c1cs15067g</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref197/cit197&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1039%2Fc1cs15067g" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref197/cit197&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=21789296" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref197/cit197&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsVGrsb3L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2011&pages=5867-5892&author=G.+W.+Collieauthor=G.+N.+Parkinson&title=The+Application+of+DNA+and+RNA+G-Quadruplexes+to+Therapeutic+Medicines&doi=10.1039%2Fc1cs15067g"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit197R"><div class="casContent"><span class="casTitleNuber">197</span><div class="casTitle"><span class="NLM_cas:atitle">The application of DNA and RNA G-quadruplexes to therapeutic medicines</span></div><div class="casAuthors">Collie, Gavin W.; Parkinson, Gary N.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Society Reviews</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">5867-5892</span>CODEN:
                <span class="NLM_cas:coden">CSRVBR</span>;
        ISSN:<span class="NLM_cas:issn">0306-0012</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">A review.  The intriguing structural diversity in folded topologies available to guanine-rich nucleic acid repeat sequences have made 4-stranded G-quadruplex structures the focus of both basic and applied research, from cancer biol. and novel therapeutics through to nanoelectronics.  Distributed widely in the human genome as targets for regulating gene expression and chromosomal maintenance, they offer unique avenues for future cancer drug development.  In particular, the recent advances in chem. and structural biol. have enabled the construction of bespoke selective DNA based aptamers to be used as novel therapeutic agents and access to detailed structural models for structure based drug discovery.  In this crit. review, the authors will explore the important underlying characteristics of G-quadruplexes that make them functional, stable, and predictable nanoscaffolds.  The authors will review the current structural database of folding topologies, mol. interfaces and novel interaction surfaces, with a consideration to their future exploitation in drug discovery, mol. biol., supermol. assembly and aptamer design.  In recent years the no. of potential applications for G-quadruplex motifs has rapidly grown, so in this review the authors aim to explore the many future challenges and highlight where possible successes may lie.  The authors will highlight the similarities and differences between DNA and RNA folded G-quadruplexes in terms of stability, distribution, and exploitability as small mol. targets.  Finally, the authors will provide a detailed review of basic G-quadruplex geometry, exptl. tools used, and a crit. evaluation of the application of high-resoln. structural biol. and its ability to provide meaningful and valid models for future applications (255 refs.).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoYJF_jWcCZCLVg90H21EOLACvtfcHk0lg7oN3eZtks4A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsVGrsb3L&md5=96e22ab0555ed0a3eac39ad091c93b2c</span></div><a href="/servlet/linkout?suffix=cit197&amp;dbid=16384&amp;doi=10.1039%2Fc1cs15067g&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc1cs15067g%26sid%3Dliteratum%253Aachs%26aulast%3DCollie%26aufirst%3DG.%2BW.%26aulast%3DParkinson%26aufirst%3DG.%2BN.%26atitle%3DThe%2520Application%2520of%2520DNA%2520and%2520RNA%2520G-Quadruplexes%2520to%2520Therapeutic%2520Medicines%26jtitle%3DChem.%2520Soc.%2520Rev.%26date%3D2011%26volume%3D40%26spage%3D5867%26epage%3D5892%26doi%3D10.1039%2Fc1cs15067g" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref198"><div class="reference"><strong class="refLabel"><a href="#ref198" class="rightTabRefNumLink">198</a></strong><div class="NLM_citation" id="rightTab-cit198"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Weldon, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Behm-Ansmant, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hurley, L. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burley, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Branlant, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eperon, I. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dominguez, C.</span></span> <span> </span><span class="NLM_article-title">Identification of G-Quadruplexes in Long Functional RNAs Using 7-Deazaguanine RNA</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">18</span>– <span class="NLM_lpage">20</span>, <span class="refDoi"> DOI: 10.1038/nchembio.2228</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref198/cit198&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1038%2Fnchembio.2228" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref198/cit198&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=27820800" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref198/cit198&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvVSntLnN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2017&pages=18-20&author=C.+Weldonauthor=I.+Behm-Ansmantauthor=L.+H.+Hurleyauthor=G.+A.+Burleyauthor=C.+Branlantauthor=I.+C.+Eperonauthor=C.+Dominguez&title=Identification+of+G-Quadruplexes+in+Long+Functional+RNAs+Using+7-Deazaguanine+RNA&doi=10.1038%2Fnchembio.2228"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit198R"><div class="casContent"><span class="casTitleNuber">198</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of G-quadruplexes in long functional RNAs using 7-deazaguanine RNA</span></div><div class="casAuthors">Weldon, Carika; Behm-Ansmant, Isabelle; Hurley, Laurence H.; Burley, Glenn A.; Branlant, Christiane; Eperon, Ian C.; Dominguez, Cyril</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">18-20</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">RNA G-quadruplex (G4) structures are thought to affect biol. processes, including translation and pre-mRNA splicing, but it is not possible at present to demonstrate that they form naturally at specific sequences in long functional RNA mols.  We developed a new strategy, footprinting of long 7-deazaguanine-substituted RNAs (FOLDeR), that allows the formation of G4s to be confirmed in long RNAs and under functional conditions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoEnw-nGuf4ZLVg90H21EOLACvtfcHk0lg7oN3eZtks4A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvVSntLnN&md5=b1f547bacfc316a5544d6901bab73533</span></div><a href="/servlet/linkout?suffix=cit198&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.2228&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.2228%26sid%3Dliteratum%253Aachs%26aulast%3DWeldon%26aufirst%3DC.%26aulast%3DBehm-Ansmant%26aufirst%3DI.%26aulast%3DHurley%26aufirst%3DL.%2BH.%26aulast%3DBurley%26aufirst%3DG.%2BA.%26aulast%3DBranlant%26aufirst%3DC.%26aulast%3DEperon%26aufirst%3DI.%2BC.%26aulast%3DDominguez%26aufirst%3DC.%26atitle%3DIdentification%2520of%2520G-Quadruplexes%2520in%2520Long%2520Functional%2520RNAs%2520Using%25207-Deazaguanine%2520RNA%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2017%26volume%3D13%26spage%3D18%26epage%3D20%26doi%3D10.1038%2Fnchembio.2228" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref199"><div class="reference"><strong class="refLabel"><a href="#ref199" class="rightTabRefNumLink">199</a></strong><div class="NLM_citation" id="rightTab-cit199"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Palacino, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Swalley, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheung, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Hoosear, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shin, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chin, D. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keller, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beibel, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Renaud, N. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salcius, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hild, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Servais, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jain, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bullock, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McLellan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schuierer, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murphy, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blommers, M. J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blaustein, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berenshteyn, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lacoste, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roma, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michaud, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tseng, B. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Porter, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Myer, V. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tallarico, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamann, L. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Curtis, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fishman, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dietrich, W. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dales, N. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sivasankaran, R.</span></span> <span> </span><span class="NLM_article-title">SMN2 Splice Modulators Enhance U1–pre-mRNA Association and Rescue SMA Mice</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">511</span>– <span class="NLM_lpage">517</span>, <span class="refDoi"> DOI: 10.1038/nchembio.1837</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref199/cit199&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1038%2Fnchembio.1837" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref199/cit199&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=26030728" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref199/cit199&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtFeju7zN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2015&pages=511-517&author=J.+Palacinoauthor=S.+E.+Swalleyauthor=C.+Songauthor=A.+K.+Cheungauthor=L.+Shuauthor=X.+Zhangauthor=M.+Van+Hoosearauthor=Y.+Shinauthor=D.+N.+Chinauthor=C.+G.+Kellerauthor=M.+Beibelauthor=N.+A.+Renaudauthor=T.+M.+Smithauthor=M.+Salciusauthor=X.+Shiauthor=M.+Hildauthor=R.+Servaisauthor=M.+Jainauthor=L.+Dengauthor=C.+Bullockauthor=M.+McLellanauthor=S.+Schuiererauthor=L.+Murphyauthor=M.+J.+J.+Blommersauthor=C.+Blausteinauthor=F.+Berenshteynauthor=A.+Lacosteauthor=J.+R.+Thomasauthor=G.+Romaauthor=G.+A.+Michaudauthor=B.+S.+Tsengauthor=J.+A.+Porterauthor=V.+E.+Myerauthor=J.+A.+Tallaricoauthor=L.+G.+Hamannauthor=D.+Curtisauthor=M.+C.+Fishmanauthor=W.+F.+Dietrichauthor=N.+A.+Dalesauthor=R.+Sivasankaran&title=SMN2+Splice+Modulators+Enhance+U1%E2%80%93pre-mRNA+Association+and+Rescue+SMA+Mice&doi=10.1038%2Fnchembio.1837"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit199R"><div class="casContent"><span class="casTitleNuber">199</span><div class="casTitle"><span class="NLM_cas:atitle">SMN2 splice modulators enhance U1-pre-mRNA association and rescue SMA mice</span></div><div class="casAuthors">Palacino, James; Swalley, Susanne E.; Song, Cheng; Cheung, Atwood K.; Shu, Lei; Zhang, Xiaolu; Van Hoosear, Mailin; Shin, Youngah; Chin, Donovan N.; Keller, Caroline Gubser; Beibel, Martin; Renaud, Nicole A.; Smith, Thomas M.; Salcius, Michael; Shi, Xiaoying; Hild, Marc; Servais, Rebecca; Jain, Monish; Deng, Lin; Bullock, Caroline; McLellan, Michael; Schuierer, Sven; Murphy, Leo; Blommers, Marcel J. J.; Blaustein, Cecile; Berenshteyn, Frada; Lacoste, Arnaud; Thomas, Jason R.; Roma, Guglielmo; Michaud, Gregory A.; Tseng, Brian S.; Porter, Jeffery A.; Myer, Vic E.; Tallarico, John A.; Hamann, Lawrence G.; Curtis, Daniel; Fishman, Mark C.; Dietrich, William F.; Dales, Natalie A.; Sivasankaran, Rajeev</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">511-517</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Spinal muscular atrophy (SMA), which results from the loss of expression of the survival of motor neuron-1 (SMN1) gene, represents the most common genetic cause of pediatric mortality.  A duplicate copy (SMN2) is inefficiently spliced, producing a truncated and unstable protein.  We describe herein a potent, orally active, small-mol. enhancer of SMN2 splicing that elevates full-length SMN protein and extends survival in a severe SMA mouse model.  We demonstrate that the mol. mechanism of action is via stabilization of the transient double-strand RNA structure formed by the SMN2 pre-mRNA and U1 small nuclear ribonucleic protein (snRNP) complex.  The binding affinity of U1 snRNP to the 5' splice site is increased in a sequence-selective manner, discrete from constitutive recognition.  This new mechanism demonstrates the feasibility of small mol.-mediated, sequence-selective splice modulation and the potential for leveraging this strategy in other splicing diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqdMqLIC0lw17Vg90H21EOLACvtfcHk0lgGaYWxibxSYg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtFeju7zN&md5=15e56c8b3f48feffa150e975a81633a0</span></div><a href="/servlet/linkout?suffix=cit199&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1837&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1837%26sid%3Dliteratum%253Aachs%26aulast%3DPalacino%26aufirst%3DJ.%26aulast%3DSwalley%26aufirst%3DS.%2BE.%26aulast%3DSong%26aufirst%3DC.%26aulast%3DCheung%26aufirst%3DA.%2BK.%26aulast%3DShu%26aufirst%3DL.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DVan%2BHoosear%26aufirst%3DM.%26aulast%3DShin%26aufirst%3DY.%26aulast%3DChin%26aufirst%3DD.%2BN.%26aulast%3DKeller%26aufirst%3DC.%2BG.%26aulast%3DBeibel%26aufirst%3DM.%26aulast%3DRenaud%26aufirst%3DN.%2BA.%26aulast%3DSmith%26aufirst%3DT.%2BM.%26aulast%3DSalcius%26aufirst%3DM.%26aulast%3DShi%26aufirst%3DX.%26aulast%3DHild%26aufirst%3DM.%26aulast%3DServais%26aufirst%3DR.%26aulast%3DJain%26aufirst%3DM.%26aulast%3DDeng%26aufirst%3DL.%26aulast%3DBullock%26aufirst%3DC.%26aulast%3DMcLellan%26aufirst%3DM.%26aulast%3DSchuierer%26aufirst%3DS.%26aulast%3DMurphy%26aufirst%3DL.%26aulast%3DBlommers%26aufirst%3DM.%2BJ.%2BJ.%26aulast%3DBlaustein%26aufirst%3DC.%26aulast%3DBerenshteyn%26aufirst%3DF.%26aulast%3DLacoste%26aufirst%3DA.%26aulast%3DThomas%26aufirst%3DJ.%2BR.%26aulast%3DRoma%26aufirst%3DG.%26aulast%3DMichaud%26aufirst%3DG.%2BA.%26aulast%3DTseng%26aufirst%3DB.%2BS.%26aulast%3DPorter%26aufirst%3DJ.%2BA.%26aulast%3DMyer%26aufirst%3DV.%2BE.%26aulast%3DTallarico%26aufirst%3DJ.%2BA.%26aulast%3DHamann%26aufirst%3DL.%2BG.%26aulast%3DCurtis%26aufirst%3DD.%26aulast%3DFishman%26aufirst%3DM.%2BC.%26aulast%3DDietrich%26aufirst%3DW.%2BF.%26aulast%3DDales%26aufirst%3DN.%2BA.%26aulast%3DSivasankaran%26aufirst%3DR.%26atitle%3DSMN2%2520Splice%2520Modulators%2520Enhance%2520U1%25E2%2580%2593pre-mRNA%2520Association%2520and%2520Rescue%2520SMA%2520Mice%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2015%26volume%3D11%26spage%3D511%26epage%3D517%26doi%3D10.1038%2Fnchembio.1837" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref200"><div class="reference"><strong class="refLabel"><a href="#ref200" class="rightTabRefNumLink">200</a></strong><div class="NLM_citation" id="rightTab-cit200"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Naryshkin, N. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weetall, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dakka, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Narasimhan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ling, K. K. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karp, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woll, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gabbeta, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vazirani, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhattacharyya, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Furia, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Risher, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheedy, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kong, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turpoff, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, C.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moon, Y.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trifillis, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Welch, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Colacino, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Babiak, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Almstead, N. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peltz, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eng, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mull, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lynes, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rubin, L. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fontoura, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Santarelli, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haehnke, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCarthy, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmucki, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ebeling, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sivaramakrishnan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ko, C.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paushkin, S. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ratni, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerlach, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ghosh, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Metzger, F.</span></span> <span> </span><span class="NLM_article-title">SMN2 Splicing Modifiers Improve Motor Function and Longevity in Mice with Spinal Muscular Atrophy</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>345</i></span>,  <span class="NLM_fpage">688</span>– <span class="NLM_lpage">693</span>, <span class="refDoi"> DOI: 10.1126/science.1250127</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref200/cit200&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1126%2Fscience.1250127" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref200/cit200&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=25104390" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref200/cit200&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC2cXht12gtLfO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=345&publication_year=2014&pages=688-693&author=N.+A.+Naryshkinauthor=M.+Weetallauthor=A.+Dakkaauthor=J.+Narasimhanauthor=X.+Zhaoauthor=Z.+Fengauthor=K.+K.+Y.+Lingauthor=G.+M.+Karpauthor=H.+Qiauthor=M.+G.+Wollauthor=G.+Chenauthor=N.+Zhangauthor=V.+Gabbetaauthor=P.+Vaziraniauthor=A.+Bhattacharyyaauthor=B.+Furiaauthor=N.+Risherauthor=J.+Sheedyauthor=R.+Kongauthor=J.+Maauthor=A.+Turpoffauthor=C.-S.+Leeauthor=X.+Zhangauthor=Y.-C.+Moonauthor=P.+Trifillisauthor=E.+M.+Welchauthor=J.+M.+Colacinoauthor=J.+Babiakauthor=N.+G.+Almsteadauthor=S.+W.+Peltzauthor=L.+A.+Engauthor=K.+S.+Chenauthor=J.+L.+Mullauthor=M.+S.+Lynesauthor=L.+L.+Rubinauthor=P.+Fontouraauthor=L.+Santarelliauthor=D.+Haehnkeauthor=K.+D.+McCarthyauthor=R.+Schmuckiauthor=M.+Ebelingauthor=M.+Sivaramakrishnanauthor=C.-P.+Koauthor=S.+V.+Paushkinauthor=H.+Ratniauthor=I.+Gerlachauthor=A.+Ghoshauthor=F.+Metzger&title=SMN2+Splicing+Modifiers+Improve+Motor+Function+and+Longevity+in+Mice+with+Spinal+Muscular+Atrophy&doi=10.1126%2Fscience.1250127"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit200R"><div class="casContent"><span class="casTitleNuber">200</span><div class="casTitle"><span class="NLM_cas:atitle">SMN2 splicing modifiers improve motor function and longevity in mice with spinal muscular atrophy</span></div><div class="casAuthors">Naryshkin, Nikolai A.; Weetall, Marla; Dakka, Amal; Narasimhan, Jana; Zhao, Xin; Feng, Zhihua; Ling, Karen K. Y.; Karp, Gary M.; Qi, Hongyan; Woll, Matthew G.; Chen, Guangming; Zhang, Nanjing; Gabbeta, Vijayalakshmi; Vazirani, Priya; Bhattacharyya, Anuradha; Furia, Bansri; Risher, Nicole; Sheedy, Josephine; Kong, Ronald; Ma, Jiyuan; Turpoff, Anthony; Lee, Chang-Sun; Zhang, Xiaoyan; Moon, Young-Choon; Trifillis, Panayiota; Welch, Ellen M.; Colacino, Joseph M.; Babiak, John; Almstead, Neil G.; Peltz, Stuart W.; Eng, Loren A.; Chen, Karen S.; Mull, Jesse L.; Lynes, Maureen S.; Rubin, Lee L.; Fontoura, Paulo; Santarelli, Luca; Haehnke, Daniel; McCarthy, Kathleen D.; Schmucki, Roland; Ebeling, Martin; Sivaramakrishnan, Manaswini; Ko, Chien-Ping; Paushkin, Sergey V.; Ratni, Hasane; Gerlach, Irene; Ghosh, Anirvan; Metzger, Friedrich</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">345</span>
        (<span class="NLM_cas:issue">6197</span>),
    <span class="NLM_cas:pages">688-693</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Spinal muscular atrophy (SMA) is a genetic disease caused by mutation or deletion of the survival of motor neuron 1 (SMN1) gene.  A paralogous gene in humans, SMN2, produces low, insufficient levels of functional SMN protein due to alternative splicing that truncates the transcript.  The decreased levels of SMN protein lead to progressive neuromuscular degeneration and high rates of mortality.  Through chem. screening and optimization, we identified orally available small mols. that shift the balance of SMN2 splicing toward the prodn. of full-length SMN2 mRNA with high selectivity.  Administration of these compds. to Δ7 mice, a model of severe SMA, led to an increase in SMN protein levels, improvement of motor function, and protection of the neuromuscular circuit.  These compds. also extended the life span of the mice.  Selective SMN2 splicing modifiers may have therapeutic potential for patients with SMA.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoxZ8QSg5KSzLVg90H21EOLACvtfcHk0lgGaYWxibxSYg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXht12gtLfO&md5=f321c411bb384f95a9fe4d4debcbb195</span></div><a href="/servlet/linkout?suffix=cit200&amp;dbid=16384&amp;doi=10.1126%2Fscience.1250127&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1250127%26sid%3Dliteratum%253Aachs%26aulast%3DNaryshkin%26aufirst%3DN.%2BA.%26aulast%3DWeetall%26aufirst%3DM.%26aulast%3DDakka%26aufirst%3DA.%26aulast%3DNarasimhan%26aufirst%3DJ.%26aulast%3DZhao%26aufirst%3DX.%26aulast%3DFeng%26aufirst%3DZ.%26aulast%3DLing%26aufirst%3DK.%2BK.%2BY.%26aulast%3DKarp%26aufirst%3DG.%2BM.%26aulast%3DQi%26aufirst%3DH.%26aulast%3DWoll%26aufirst%3DM.%2BG.%26aulast%3DChen%26aufirst%3DG.%26aulast%3DZhang%26aufirst%3DN.%26aulast%3DGabbeta%26aufirst%3DV.%26aulast%3DVazirani%26aufirst%3DP.%26aulast%3DBhattacharyya%26aufirst%3DA.%26aulast%3DFuria%26aufirst%3DB.%26aulast%3DRisher%26aufirst%3DN.%26aulast%3DSheedy%26aufirst%3DJ.%26aulast%3DKong%26aufirst%3DR.%26aulast%3DMa%26aufirst%3DJ.%26aulast%3DTurpoff%26aufirst%3DA.%26aulast%3DLee%26aufirst%3DC.-S.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DMoon%26aufirst%3DY.-C.%26aulast%3DTrifillis%26aufirst%3DP.%26aulast%3DWelch%26aufirst%3DE.%2BM.%26aulast%3DColacino%26aufirst%3DJ.%2BM.%26aulast%3DBabiak%26aufirst%3DJ.%26aulast%3DAlmstead%26aufirst%3DN.%2BG.%26aulast%3DPeltz%26aufirst%3DS.%2BW.%26aulast%3DEng%26aufirst%3DL.%2BA.%26aulast%3DChen%26aufirst%3DK.%2BS.%26aulast%3DMull%26aufirst%3DJ.%2BL.%26aulast%3DLynes%26aufirst%3DM.%2BS.%26aulast%3DRubin%26aufirst%3DL.%2BL.%26aulast%3DFontoura%26aufirst%3DP.%26aulast%3DSantarelli%26aufirst%3DL.%26aulast%3DHaehnke%26aufirst%3DD.%26aulast%3DMcCarthy%26aufirst%3DK.%2BD.%26aulast%3DSchmucki%26aufirst%3DR.%26aulast%3DEbeling%26aufirst%3DM.%26aulast%3DSivaramakrishnan%26aufirst%3DM.%26aulast%3DKo%26aufirst%3DC.-P.%26aulast%3DPaushkin%26aufirst%3DS.%2BV.%26aulast%3DRatni%26aufirst%3DH.%26aulast%3DGerlach%26aufirst%3DI.%26aulast%3DGhosh%26aufirst%3DA.%26aulast%3DMetzger%26aufirst%3DF.%26atitle%3DSMN2%2520Splicing%2520Modifiers%2520Improve%2520Motor%2520Function%2520and%2520Longevity%2520in%2520Mice%2520with%2520Spinal%2520Muscular%2520Atrophy%26jtitle%3DScience%26date%3D2014%26volume%3D345%26spage%3D688%26epage%3D693%26doi%3D10.1126%2Fscience.1250127" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref201"><div class="reference"><strong class="refLabel"><a href="#ref201" class="rightTabRefNumLink">201</a></strong><div class="NLM_citation" id="rightTab-cit201"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lintner, N. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McClure, K. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petersen, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Londregan, A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Piotrowski, D. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bolt, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loria, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maguire, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geoghegan, K. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rolph, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liras, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doudna, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dullea, R. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cate, J. H. D.</span></span> <span> </span><span class="NLM_article-title">Selective Stalling of Human Translation through Small-Molecule Engagement of the Ribosome Nascent Chain</span>. <i>PLoS Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">e2001882</span>, <span class="refDoi"> DOI: 10.1371/journal.pbio.2001882</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref201/cit201&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1371%2Fjournal.pbio.2001882" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref201/cit201&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=28323820" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref201/cit201&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC2sXht1WqurfE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2017&pages=e2001882&author=N.+G.+Lintnerauthor=K.+F.+McClureauthor=D.+Petersenauthor=A.+T.+Londreganauthor=D.+W.+Piotrowskiauthor=L.+Weiauthor=J.+Xiaoauthor=M.+Boltauthor=P.+M.+Loriaauthor=B.+Maguireauthor=K.+F.+Geogheganauthor=A.+Huangauthor=T.+Rolphauthor=S.+Lirasauthor=J.+A.+Doudnaauthor=R.+G.+Dulleaauthor=J.+H.+D.+Cate&title=Selective+Stalling+of+Human+Translation+through+Small-Molecule+Engagement+of+the+Ribosome+Nascent+Chain&doi=10.1371%2Fjournal.pbio.2001882"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit201R"><div class="casContent"><span class="casTitleNuber">201</span><div class="casTitle"><span class="NLM_cas:atitle">Selective stalling of human translation through small-molecule engagement of the ribosome nascent chain</span></div><div class="casAuthors">Lintner, Nathanael G.; McClure, Kim F.; Petersen, Donna; Londregan, Allyn T.; Piotrowski, David W.; Wei, Liuqing; Xiao, Jun; Bolt, Michael; Loria, Paula M.; Maguire, Bruce; Geoghegan, Kieran F.; Huang, Austin; Rolph, Tim; Liras, Spiros; Doudna, Jennifer A.; Dullea, Robert G.; Cate, Jamie H. D.</div><div class="citationInfo"><span class="NLM_cas:title">PLoS Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">e2001882/1-e2001882/36</span>CODEN:
                <span class="NLM_cas:coden">PBLIBG</span>;
        ISSN:<span class="NLM_cas:issn">1545-7885</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Proprotein convertase subtilisin/kexin type 9 (PCSK9) plays a key role in regulating the levels of plasma low-d. lipoprotein cholesterol (LDL-C).  Here, we demonstrate that the compd. PF-06446846 inhibits translation of PCSK9 by inducing the ribosome to stall around codon 34, mediated by the sequence of the nascent chain within the exit tunnel.  We further show that PF-06446846 reduces plasma PCSK9 and total cholesterol levels in rats following oral dosing.  Using ribosome profiling, we demonstrate that PF-06446846 is highly selective for the inhibition of PCSK9 translation.  The mechanism of action employed by PF-06446846 reveals a previously unexpected tunability of the human ribosome that allows small mols. to specifically block translation of individual transcripts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpqeQbFLtKlX7Vg90H21EOLACvtfcHk0lgfqc4m9tvCTg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXht1WqurfE&md5=64f33ee234c2425258154ac8368d2518</span></div><a href="/servlet/linkout?suffix=cit201&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pbio.2001882&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pbio.2001882%26sid%3Dliteratum%253Aachs%26aulast%3DLintner%26aufirst%3DN.%2BG.%26aulast%3DMcClure%26aufirst%3DK.%2BF.%26aulast%3DPetersen%26aufirst%3DD.%26aulast%3DLondregan%26aufirst%3DA.%2BT.%26aulast%3DPiotrowski%26aufirst%3DD.%2BW.%26aulast%3DWei%26aufirst%3DL.%26aulast%3DXiao%26aufirst%3DJ.%26aulast%3DBolt%26aufirst%3DM.%26aulast%3DLoria%26aufirst%3DP.%2BM.%26aulast%3DMaguire%26aufirst%3DB.%26aulast%3DGeoghegan%26aufirst%3DK.%2BF.%26aulast%3DHuang%26aufirst%3DA.%26aulast%3DRolph%26aufirst%3DT.%26aulast%3DLiras%26aufirst%3DS.%26aulast%3DDoudna%26aufirst%3DJ.%2BA.%26aulast%3DDullea%26aufirst%3DR.%2BG.%26aulast%3DCate%26aufirst%3DJ.%2BH.%2BD.%26atitle%3DSelective%2520Stalling%2520of%2520Human%2520Translation%2520through%2520Small-Molecule%2520Engagement%2520of%2520the%2520Ribosome%2520Nascent%2520Chain%26jtitle%3DPLoS%2520Biol.%26date%3D2017%26volume%3D15%26spage%3De2001882%26doi%3D10.1371%2Fjournal.pbio.2001882" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref202"><div class="reference"><strong class="refLabel"><a href="#ref202" class="rightTabRefNumLink">202</a></strong><div class="NLM_citation" id="rightTab-cit202"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Blakeley, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DePorter, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mohan, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burai, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tolbert, B. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McNaughton, B. R.</span></span> <span> </span><span class="NLM_article-title">Methods for Identifying and Characterizing Interactions Involving RNA</span>. <i>Tetrahedron</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>68</i></span>,  <span class="NLM_fpage">8837</span>– <span class="NLM_lpage">8855</span>, <span class="refDoi"> DOI: 10.1016/j.tet.2012.07.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref202/cit202&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1016%2Fj.tet.2012.07.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref202/cit202&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtVyjsLnP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=68&publication_year=2012&pages=8837-8855&author=B.+D.+Blakeleyauthor=S.+M.+DePorterauthor=U.+Mohanauthor=R.+Buraiauthor=B.+S.+Tolbertauthor=B.+R.+McNaughton&title=Methods+for+Identifying+and+Characterizing+Interactions+Involving+RNA&doi=10.1016%2Fj.tet.2012.07.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit202R"><div class="casContent"><span class="casTitleNuber">202</span><div class="casTitle"><span class="NLM_cas:atitle">Methods for identifying and characterizing interactions involving RNA</span></div><div class="casAuthors">Blakeley, Brett D.; DePorter, Sandra M.; Mohan, Utpal; Burai, Ritwik; Tolbert, Blanton S.; McNaughton, Brian R.</div><div class="citationInfo"><span class="NLM_cas:title">Tetrahedron</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">68</span>
        (<span class="NLM_cas:issue">43</span>),
    <span class="NLM_cas:pages">8837-8855</span>CODEN:
                <span class="NLM_cas:coden">TETRAB</span>;
        ISSN:<span class="NLM_cas:issn">0040-4020</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Topics discussed include fluorescence-based methods, microarrays, selection-based methods, and low-throughput methods for measuring ligand-RNA interactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpc9bjJ4rDA_LVg90H21EOLACvtfcHk0lgfqc4m9tvCTg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtVyjsLnP&md5=ecbfe5736239b30b9e615d3b51dc78ed</span></div><a href="/servlet/linkout?suffix=cit202&amp;dbid=16384&amp;doi=10.1016%2Fj.tet.2012.07.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tet.2012.07.001%26sid%3Dliteratum%253Aachs%26aulast%3DBlakeley%26aufirst%3DB.%2BD.%26aulast%3DDePorter%26aufirst%3DS.%2BM.%26aulast%3DMohan%26aufirst%3DU.%26aulast%3DBurai%26aufirst%3DR.%26aulast%3DTolbert%26aufirst%3DB.%2BS.%26aulast%3DMcNaughton%26aufirst%3DB.%2BR.%26atitle%3DMethods%2520for%2520Identifying%2520and%2520Characterizing%2520Interactions%2520Involving%2520RNA%26jtitle%3DTetrahedron%26date%3D2012%26volume%3D68%26spage%3D8837%26epage%3D8855%26doi%3D10.1016%2Fj.tet.2012.07.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref203"><div class="reference"><strong class="refLabel"><a href="#ref203" class="rightTabRefNumLink">203</a></strong><div class="NLM_citation" id="rightTab-cit203"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Moon, M. H.</span>; <span class="NLM_string-name">Hilimire, T. A.</span>; <span class="NLM_string-name">Sanders, A. M.</span>; <span class="NLM_string-name">Schneekloth, J. S.</span></span> <span> </span><span class="NLM_article-title">Measuring RNA–Ligand Interactions with Microscale Thermophoresis</span>.  <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2018</span>, <span class="refDoi"> DOI: 10.1021/acs.biochem.7b01141</span> .</span><div class="citationLinks">[<a href="/doi/10.1021/acs.biochem.7b01141" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2018&author=M.+H.+Moon&author=T.+A.+Hilimire&author=A.+M.+Sanders&author=J.+S.+Schneekloth&title=Measuring+RNA%E2%80%93Ligand+Interactions+with+Microscale+Thermophoresis&doi=10.1021%2Facs.biochem.7b01141"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit203&amp;dbid=16384&amp;doi=10.1021%2Facs.biochem.7b01141&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.biochem.7b01141%26sid%3Dliteratum%253Aachs%26aulast%3DMoon%26aufirst%3DM.%2BH.%26atitle%3DMeasuring%2520RNA%25E2%2580%2593Ligand%2520Interactions%2520with%2520Microscale%2520Thermophoresis%26jtitle%3DBiochemistry%26date%3D2018%26doi%3D10.1021%2Facs.biochem.7b01141" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref204"><div class="reference"><strong class="refLabel"><a href="#ref204" class="rightTabRefNumLink">204</a></strong><div class="NLM_citation" id="rightTab-cit204"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Disney, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winkelsas, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Velagapudi, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Southern, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fallahi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Childs-Disney, J. L.</span></span> <span> </span><span class="NLM_article-title">Inforna 2.0: A Platform for the Sequence-Based Design of Small Molecules Targeting Structured RNAs</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">1720</span>– <span class="NLM_lpage">1728</span>, <span class="refDoi"> DOI: 10.1021/acschembio.6b00001</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.6b00001" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref204/cit204&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC28Xmt1Cgsrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2016&pages=1720-1728&author=M.+D.+Disneyauthor=A.+M.+Winkelsasauthor=S.+P.+Velagapudiauthor=M.+Southernauthor=M.+Fallahiauthor=J.+L.+Childs-Disney&title=Inforna+2.0%3A+A+Platform+for+the+Sequence-Based+Design+of+Small+Molecules+Targeting+Structured+RNAs&doi=10.1021%2Facschembio.6b00001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit204R"><div class="casContent"><span class="casTitleNuber">204</span><div class="casTitle"><span class="NLM_cas:atitle">Inforna 2.0: A Platform for the Sequence-Based Design of Small Molecules Targeting Structured RNAs</span></div><div class="casAuthors">Disney, Matthew D.; Winkelsas, Audrey M.; Velagapudi, Sai Pradeep; Southern, Mark; Fallahi, Mohammad; Childs-Disney, Jessica L.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1720-1728</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The development of small mols. that target RNA is challenging yet, if successful, could advance the development of chem. probes to study RNA function or precision therapeutics to treat RNA-mediated disease.  Previously, we described Inforna, an approach that can mine motifs (secondary structures) within target RNAs, which is deduced from the RNA sequence, and compare them to a database of known RNA motif-small mol. binding partners.  Output generated by Inforna includes the motif found in both the database and the desired RNA target, lead small mols. for that target, and other related meta-data.  Lead small mols. can then be tested for binding and affecting cellular (dys)function.  Herein, we describe Inforna 2.0, which incorporates all known RNA motif-small mol. binding partners reported in the scientific literature, a chem. similarity searching feature, and an improved user interface and is freely available via an online web server.  By incorporation of interactions identified by other labs., the database has been doubled, contg. 1936 RNA motif-small mol. interactions, including 244 unique small mols. and 1331 motifs.  Interestingly, chemotype anal. of the compds. that bind RNA in the database reveals features in small mol. chemotypes that are privileged for binding.  Further, this updated database expanded the no. of cellular RNAs to which lead compds. can be identified.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoHmJfcxIk9yrVg90H21EOLACvtfcHk0lgfqc4m9tvCTg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xmt1Cgsrw%253D&md5=6086bc48066565e519f7ee9e9cf2542a</span></div><a href="/servlet/linkout?suffix=cit204&amp;dbid=16384&amp;doi=10.1021%2Facschembio.6b00001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.6b00001%26sid%3Dliteratum%253Aachs%26aulast%3DDisney%26aufirst%3DM.%2BD.%26aulast%3DWinkelsas%26aufirst%3DA.%2BM.%26aulast%3DVelagapudi%26aufirst%3DS.%2BP.%26aulast%3DSouthern%26aufirst%3DM.%26aulast%3DFallahi%26aufirst%3DM.%26aulast%3DChilds-Disney%26aufirst%3DJ.%2BL.%26atitle%3DInforna%25202.0%253A%2520A%2520Platform%2520for%2520the%2520Sequence-Based%2520Design%2520of%2520Small%2520Molecules%2520Targeting%2520Structured%2520RNAs%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2016%26volume%3D11%26spage%3D1720%26epage%3D1728%26doi%3D10.1021%2Facschembio.6b00001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref205"><div class="reference"><strong class="refLabel"><a href="#ref205" class="rightTabRefNumLink">205</a></strong><div class="NLM_citation" id="rightTab-cit205"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rzuczek, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Colgan, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakai, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cameron, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Furling, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yasuda, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Disney, M. D.</span></span> <span> </span><span class="NLM_article-title">Precise Small-Molecule Recognition of a Toxic CUG RNA Repeat Expansion</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">188</span>– <span class="NLM_lpage">193</span>, <span class="refDoi"> DOI: 10.1038/nchembio.2251</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref205/cit205&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1038%2Fnchembio.2251" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref205/cit205&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=27941760" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref205/cit205&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC28XitVSrurbM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2017&pages=188-193&author=S.+G.+Rzuczekauthor=L.+A.+Colganauthor=Y.+Nakaiauthor=M.+D.+Cameronauthor=D.+Furlingauthor=R.+Yasudaauthor=M.+D.+Disney&title=Precise+Small-Molecule+Recognition+of+a+Toxic+CUG+RNA+Repeat+Expansion&doi=10.1038%2Fnchembio.2251"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit205R"><div class="casContent"><span class="casTitleNuber">205</span><div class="casTitle"><span class="NLM_cas:atitle">Precise small-molecule recognition of a toxic CUG RNA repeat expansion</span></div><div class="casAuthors">Rzuczek, Suzanne G.; Colgan, Lesley A.; Nakai, Yoshio; Cameron, Michael D.; Furling, Denis; Yasuda, Ryohei; Disney, Matthew D.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">188-193</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Excluding the ribosome and riboswitches, developing small mols. that selectively target RNA is a longstanding problem in chem. biol.  A typical cellular RNA is difficult to target because it has little tertiary, but abundant secondary structure.  We designed allele-selective compds. that target such an RNA, the toxic noncoding repeat expansion (r(CUG)exp) that causes myotonic dystrophy type 1 (DM1).  We developed several strategies to generate allele-selective small mols., including non-covalent binding, covalent binding, cleavage and on-site probe synthesis.  Covalent binding and cleavage enabled target profiling in cells derived from individuals with DM1, showing precise recognition of r(CUG)exp.  In the on-site probe synthesis approach, small mols. bound adjacent sites in r(CUG)exp and reacted to afford picomolar inhibitors via a proximity-based click reaction only in DM1-affected cells.  We expanded this approach to image r(CUG)exp in its natural context.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrC5kTu0Wkj9bVg90H21EOLACvtfcHk0lguGNdWEk3Z1A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitVSrurbM&md5=bb30d79fe6e8d513a4956d13ee20f69c</span></div><a href="/servlet/linkout?suffix=cit205&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.2251&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.2251%26sid%3Dliteratum%253Aachs%26aulast%3DRzuczek%26aufirst%3DS.%2BG.%26aulast%3DColgan%26aufirst%3DL.%2BA.%26aulast%3DNakai%26aufirst%3DY.%26aulast%3DCameron%26aufirst%3DM.%2BD.%26aulast%3DFurling%26aufirst%3DD.%26aulast%3DYasuda%26aufirst%3DR.%26aulast%3DDisney%26aufirst%3DM.%2BD.%26atitle%3DPrecise%2520Small-Molecule%2520Recognition%2520of%2520a%2520Toxic%2520CUG%2520RNA%2520Repeat%2520Expansion%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2017%26volume%3D13%26spage%3D188%26epage%3D193%26doi%3D10.1038%2Fnchembio.2251" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref206"><div class="reference"><strong class="refLabel"><a href="#ref206" class="rightTabRefNumLink">206</a></strong><div class="NLM_citation" id="rightTab-cit206"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Velagapudi, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tran, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haniff, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakai, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fallahi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martinez, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Childs-Disney, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Disney, M. D.</span></span> <span> </span><span class="NLM_article-title">Defining RNA–Small Molecule Affinity Landscapes Enables Design of a Small Molecule Inhibitor of an Oncogenic Noncoding RNA</span>. <i>ACS Cent. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">205</span>– <span class="NLM_lpage">216</span>, <span class="refDoi"> DOI: 10.1021/acscentsci.7b00009</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acscentsci.7b00009" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref206/cit206&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC2sXjs1GqsLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2017&pages=205-216&author=S.+P.+Velagapudiauthor=Y.+Luoauthor=T.+Tranauthor=H.+S.+Haniffauthor=Y.+Nakaiauthor=M.+Fallahiauthor=G.+J.+Martinezauthor=J.+L.+Childs-Disneyauthor=M.+D.+Disney&title=Defining+RNA%E2%80%93Small+Molecule+Affinity+Landscapes+Enables+Design+of+a+Small+Molecule+Inhibitor+of+an+Oncogenic+Noncoding+RNA&doi=10.1021%2Facscentsci.7b00009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit206R"><div class="casContent"><span class="casTitleNuber">206</span><div class="casTitle"><span class="NLM_cas:atitle">Defining RNA-Small Molecule Affinity Landscapes Enables Design of a Small Molecule Inhibitor of an Oncogenic Noncoding RNA</span></div><div class="casAuthors">Velagapudi, Sai Pradeep; Luo, Yiling; Tran, Tuan; Haniff, Hafeez S.; Nakai, Yoshio; Fallahi, Mohammad; Martinez, Gustavo J.; Childs-Disney, Jessica L.; Disney, Matthew D.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Central Science</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">205-216</span>CODEN:
                <span class="NLM_cas:coden">ACSCII</span>;
        ISSN:<span class="NLM_cas:issn">2374-7951</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">RNA drug targets are pervasive in cells but methods to design small mols. that target them are sparse.  Herein, the authors report a general approach to score the affinity and selectivity of RNA motif-small mol. interactions identified via selection.  Named High Throughput Structure-Activity Relationships Through Sequencing (HiT-StARTS), HiT-StARTS is statistical in nature and compares input nucleic acid sequences to selected library members that bind a ligand via high throughput sequencing.  The approach allowed facile definition of the fitness landscape of hundreds of thousands of RNA motif-small mol. binding partners.  These results were mined against folded RNAs in the human transcriptome and identified an avid interaction between a small mol. and the Dicer nuclease-processing site in the oncogenic microRNA (miR)-18a hairpin precursor, which is a member of the miR-17-92 cluster.  Application of the small mol., Targapremir-18a, to prostate cancer cells inhibited prodn. of miR-18a from the cluster, derepressed serine/threonine protein kinase 4 protein (STK4), and triggered apoptosis.  Profiling the cellular targets of Targapremir-18a via Chem. Cross Linking and Isolation by Pull Down (Chem-CLIP), a covalent small mol.-RNA cellular profiling approach, and other studies showed specific binding of the compd. to the miR-18a precursor, revealing broadly applicable factors that govern small mol. drugging of noncoding RNAs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPmptGpjN_4rVg90H21EOLACvtfcHk0lguGNdWEk3Z1A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXjs1GqsLs%253D&md5=fefd2fe6e9d637c4aa04fd1e25fb1446</span></div><a href="/servlet/linkout?suffix=cit206&amp;dbid=16384&amp;doi=10.1021%2Facscentsci.7b00009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facscentsci.7b00009%26sid%3Dliteratum%253Aachs%26aulast%3DVelagapudi%26aufirst%3DS.%2BP.%26aulast%3DLuo%26aufirst%3DY.%26aulast%3DTran%26aufirst%3DT.%26aulast%3DHaniff%26aufirst%3DH.%2BS.%26aulast%3DNakai%26aufirst%3DY.%26aulast%3DFallahi%26aufirst%3DM.%26aulast%3DMartinez%26aufirst%3DG.%2BJ.%26aulast%3DChilds-Disney%26aufirst%3DJ.%2BL.%26aulast%3DDisney%26aufirst%3DM.%2BD.%26atitle%3DDefining%2520RNA%25E2%2580%2593Small%2520Molecule%2520Affinity%2520Landscapes%2520Enables%2520Design%2520of%2520a%2520Small%2520Molecule%2520Inhibitor%2520of%2520an%2520Oncogenic%2520Noncoding%2520RNA%26jtitle%3DACS%2520Cent.%2520Sci.%26date%3D2017%26volume%3D3%26spage%3D205%26epage%3D216%26doi%3D10.1021%2Facscentsci.7b00009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref207"><div class="reference"><strong class="refLabel"><a href="#ref207" class="rightTabRefNumLink">207</a></strong><div class="NLM_citation" id="rightTab-cit207"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Costales, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haga, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Velagapudi, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Childs-Disney, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phinney, D. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Disney, M. D.</span></span> <span> </span><span class="NLM_article-title">Small Molecule Inhibition of microRNA-210 Reprograms an Oncogenic Hypoxic Circuit</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>139</i></span>,  <span class="NLM_fpage">3446</span>– <span class="NLM_lpage">3455</span>, <span class="refDoi"> DOI: 10.1021/jacs.6b11273</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.6b11273" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref207/cit207&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC2sXjtlCjt7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=139&publication_year=2017&pages=3446-3455&author=M.+G.+Costalesauthor=C.+L.+Hagaauthor=S.+P.+Velagapudiauthor=J.+L.+Childs-Disneyauthor=D.+G.+Phinneyauthor=M.+D.+Disney&title=Small+Molecule+Inhibition+of+microRNA-210+Reprograms+an+Oncogenic+Hypoxic+Circuit&doi=10.1021%2Fjacs.6b11273"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit207R"><div class="casContent"><span class="casTitleNuber">207</span><div class="casTitle"><span class="NLM_cas:atitle">Small Molecule Inhibition of microRNA-210 Reprograms an Oncogenic Hypoxic Circuit</span></div><div class="casAuthors">Costales, Matthew G.; Haga, Christopher L.; Velagapudi, Sai Pradeep; Childs-Disney, Jessica L.; Phinney, Donald G.; Disney, Matthew D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">139</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">3446-3455</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A hypoxic state is crit. to the metastatic and invasive characteristics of cancer.  Numerous pathways play crit. roles in cancer maintenance, many of which include noncoding RNAs such as microRNA (miR)-210 that regulates hypoxia inducible factors (HIFs).  Herein, we describe the identification of a small mol. named Targapremir-210 that binds to the Dicer site of the miR-210 hairpin precursor.  This interaction inhibits prodn. of the mature miRNA, derepresses glycerol-3-phosphate dehydrogenase 1-like enzyme (GPD1L), a hypoxia-assocd. protein neg. regulated by miR-210, decreases HIF-1α, and triggers apoptosis of triple neg. breast cancer cells only under hypoxic conditions.  Further, Targapremir-210 inhibits tumorigenesis in a mouse xenograft model of hypoxic triple neg. breast cancer.  Many factors govern mol. recognition of biol. targets by small mols.  For protein, chemoproteomics and activity-based protein profiling are invaluable tools to study small mol. target engagement and selectivity in cells.  Such approaches are lacking for RNA, leaving a void in the understanding of its druggability.  We applied Chem. Crosslinking and Isolation by Pull Down (Chem-CLIP) to study the cellular selectivity and the on- and off-targets of Targapremir-210.  Targapremir-210 selectively recognizes the miR-210 precursor and can differentially recognize RNAs in cells that have the same target motif but have different expression levels, revealing this important feature for selectively drugging RNAs for the first time.  These studies show that small mols. can be rapidly designed to selectively target RNAs and affect cellular responses to environmental conditions, resulting in favorable benefits against cancer.  Further, they help define rules for identifying druggable targets in the transcriptome.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpAY_YQQyzKvLVg90H21EOLACvtfcHk0lguGNdWEk3Z1A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXjtlCjt7w%253D&md5=c6725837e6f1713facb507b518998a8f</span></div><a href="/servlet/linkout?suffix=cit207&amp;dbid=16384&amp;doi=10.1021%2Fjacs.6b11273&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.6b11273%26sid%3Dliteratum%253Aachs%26aulast%3DCostales%26aufirst%3DM.%2BG.%26aulast%3DHaga%26aufirst%3DC.%2BL.%26aulast%3DVelagapudi%26aufirst%3DS.%2BP.%26aulast%3DChilds-Disney%26aufirst%3DJ.%2BL.%26aulast%3DPhinney%26aufirst%3DD.%2BG.%26aulast%3DDisney%26aufirst%3DM.%2BD.%26atitle%3DSmall%2520Molecule%2520Inhibition%2520of%2520microRNA-210%2520Reprograms%2520an%2520Oncogenic%2520Hypoxic%2520Circuit%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2017%26volume%3D139%26spage%3D3446%26epage%3D3455%26doi%3D10.1021%2Fjacs.6b11273" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref208"><div class="reference"><strong class="refLabel"><a href="#ref208" class="rightTabRefNumLink">208</a></strong><div class="NLM_citation" id="rightTab-cit208"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Howe, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fischmann, T. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Balibar, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiao, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galgoci, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Malinverni, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mayhood, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Villafania, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nahvi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murgolo, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barbieri, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mann, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carr, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xia, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zuck, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riley, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Painter, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walker, S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sherborne, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Jesus, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pan, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plotkin, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rindgen, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cummings, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garlisi, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheth, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gill, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roemer, T.</span></span> <span> </span><span class="NLM_article-title">Selective Small-Molecule Inhibition of an RNA Structural Element</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>526</i></span>,  <span class="NLM_fpage">672</span>– <span class="NLM_lpage">677</span>, <span class="refDoi"> DOI: 10.1038/nature15542</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref208/cit208&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1038%2Fnature15542" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref208/cit208&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=26416753" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref208/cit208&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhslCnu7%252FL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=526&publication_year=2015&pages=672-677&author=J.+A.+Howeauthor=H.+Wangauthor=T.+O.+Fischmannauthor=C.+J.+Balibarauthor=L.+Xiaoauthor=A.+M.+Galgociauthor=J.+C.+Malinverniauthor=T.+Mayhoodauthor=A.+Villafaniaauthor=A.+Nahviauthor=N.+Murgoloauthor=C.+M.+Barbieriauthor=P.+A.+Mannauthor=D.+Carrauthor=E.+Xiaauthor=P.+Zuckauthor=D.+Rileyauthor=R.+E.+Painterauthor=S.+S.+Walkerauthor=B.+Sherborneauthor=R.+de+Jesusauthor=W.+Panauthor=M.+A.+Plotkinauthor=J.+Wuauthor=D.+Rindgenauthor=J.+Cummingsauthor=C.+G.+Garlisiauthor=R.+Zhangauthor=P.+R.+Shethauthor=C.+J.+Gillauthor=H.+Tangauthor=T.+Roemer&title=Selective+Small-Molecule+Inhibition+of+an+RNA+Structural+Element&doi=10.1038%2Fnature15542"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit208R"><div class="casContent"><span class="casTitleNuber">208</span><div class="casTitle"><span class="NLM_cas:atitle">Selective small-molecule inhibition of an RNA structural element</span></div><div class="casAuthors">Howe, John A.; Wang, Hao; Fischmann, Thierry O.; Balibar, Carl J.; Xiao, Li; Galgoci, Andrew M.; Malinverni, Juliana C.; Mayhood, Todd; Villafania, Artjohn; Nahvi, Ali; Murgolo, Nicholas; Barbieri, Christopher M.; Mann, Paul A.; Carr, Donna; Xia, Ellen; Zuck, Paul; Riley, Dan; Painter, Ronald E.; Walker, Scott S.; Sherborne, Brad; de Jesus, Reynalda; Pan, Weidong; Plotkin, Michael A.; Wu, Jin; Rindgen, Diane; Cummings, John; Garlisi, Charles G.; Zhang, Rumin; Sheth, Payal R.; Gill, Charles J.; Tang, Haifeng; Roemer, Terry</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">526</span>
        (<span class="NLM_cas:issue">7575</span>),
    <span class="NLM_cas:pages">672-677</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Riboswitches are non-coding RNA structures located in mRNAs that bind endogenous ligands, such as a specific metabolite or ion, to regulate gene expression.  As such, riboswitches serve as a novel, yet largely unexploited, class of emerging drug targets.  Demonstrating this potential, however, has proven difficult and is restricted to structurally similar antimetabolites and semi-synthetic analogs of their cognate ligand, thus greatly restricting the chem. space and selectivity sought for such inhibitors.  Here we report the discovery and characterization of ribocil, a highly selective chem. modulator of bacterial riboflavin riboswitches, which was identified in a phenotypic screen and acts as a structurally distinct synthetic mimic of the natural ligand, FMN, to repress riboswitch-mediated ribB gene expression and inhibit bacterial cell growth.  Our findings indicate that non-coding RNA structural elements may be more broadly targeted by synthetic small mols. than previously expected.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqWptaVLK3njLVg90H21EOLACvtfcHk0lh4C4OeC_XeYg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhslCnu7%252FL&md5=c0f8443348c10f69eb47f05a2ac5e893</span></div><a href="/servlet/linkout?suffix=cit208&amp;dbid=16384&amp;doi=10.1038%2Fnature15542&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature15542%26sid%3Dliteratum%253Aachs%26aulast%3DHowe%26aufirst%3DJ.%2BA.%26aulast%3DWang%26aufirst%3DH.%26aulast%3DFischmann%26aufirst%3DT.%2BO.%26aulast%3DBalibar%26aufirst%3DC.%2BJ.%26aulast%3DXiao%26aufirst%3DL.%26aulast%3DGalgoci%26aufirst%3DA.%2BM.%26aulast%3DMalinverni%26aufirst%3DJ.%2BC.%26aulast%3DMayhood%26aufirst%3DT.%26aulast%3DVillafania%26aufirst%3DA.%26aulast%3DNahvi%26aufirst%3DA.%26aulast%3DMurgolo%26aufirst%3DN.%26aulast%3DBarbieri%26aufirst%3DC.%2BM.%26aulast%3DMann%26aufirst%3DP.%2BA.%26aulast%3DCarr%26aufirst%3DD.%26aulast%3DXia%26aufirst%3DE.%26aulast%3DZuck%26aufirst%3DP.%26aulast%3DRiley%26aufirst%3DD.%26aulast%3DPainter%26aufirst%3DR.%2BE.%26aulast%3DWalker%26aufirst%3DS.%2BS.%26aulast%3DSherborne%26aufirst%3DB.%26aulast%3Dde%2BJesus%26aufirst%3DR.%26aulast%3DPan%26aufirst%3DW.%26aulast%3DPlotkin%26aufirst%3DM.%2BA.%26aulast%3DWu%26aufirst%3DJ.%26aulast%3DRindgen%26aufirst%3DD.%26aulast%3DCummings%26aufirst%3DJ.%26aulast%3DGarlisi%26aufirst%3DC.%2BG.%26aulast%3DZhang%26aufirst%3DR.%26aulast%3DSheth%26aufirst%3DP.%2BR.%26aulast%3DGill%26aufirst%3DC.%2BJ.%26aulast%3DTang%26aufirst%3DH.%26aulast%3DRoemer%26aufirst%3DT.%26atitle%3DSelective%2520Small-Molecule%2520Inhibition%2520of%2520an%2520RNA%2520Structural%2520Element%26jtitle%3DNature%26date%3D2015%26volume%3D526%26spage%3D672%26epage%3D677%26doi%3D10.1038%2Fnature15542" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref209"><div class="reference"><strong class="refLabel"><a href="#ref209" class="rightTabRefNumLink">209</a></strong><div class="NLM_citation" id="rightTab-cit209"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Stelzer, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frank, A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kratz, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Swanson, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gonzalez-Hernandez, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andricioaei, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Markovitz, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Al-Hashimi, H. M.</span></span> <span> </span><span class="NLM_article-title">Discovery of Selective Bioactive Small Molecules by Targeting an RNA Dynamic Ensemble</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">553</span>– <span class="NLM_lpage">559</span>, <span class="refDoi"> DOI: 10.1038/nchembio.596</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref209/cit209&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1038%2Fnchembio.596" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref209/cit209&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=21706033" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref209/cit209&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC3MXotV2htbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2011&pages=553-559&author=A.+C.+Stelzerauthor=A.+T.+Frankauthor=J.+D.+Kratzauthor=M.+D.+Swansonauthor=M.+J.+Gonzalez-Hernandezauthor=J.+Leeauthor=I.+Andricioaeiauthor=D.+M.+Markovitzauthor=H.+M.+Al-Hashimi&title=Discovery+of+Selective+Bioactive+Small+Molecules+by+Targeting+an+RNA+Dynamic+Ensemble&doi=10.1038%2Fnchembio.596"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit209R"><div class="casContent"><span class="casTitleNuber">209</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of selective bioactive small molecules by targeting an RNA dynamic ensemble</span></div><div class="casAuthors">Stelzer, Andrew C.; Frank, Aaron T.; Kratz, Jeremy D.; Swanson, Michael D.; Gonzalez-Hernandez, Marta J.; Lee, Janghyun; Andricioaei, Ioan; Markovitz, David M.; Al-Hashimi, Hashim M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">553-559</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Current approaches used to identify protein-binding small mols. are not suited for identifying small mols. that can bind emerging RNA drug targets.  By docking small mols. onto an RNA dynamic ensemble constructed by combining NMR spectroscopy and computational mol. dynamics, the authors virtually screened small mols. that target the entire structure landscape of the transactivation response element (TAR) from HIV type 1 (HIV-1).  The authors quant. predict binding energies for small mols. that bind different RNA conformations and report the de novo discovery of six compds. that bind TAR with high affinity and inhibit its interaction with a Tat peptide in vitro (Ki values of 710 nM-169 μM).  One compd. binds HIV-1 TAR with marked selectivity and inhibits Tat-mediated activation of the HIV-1 long terminal repeat by 81% in T-cell lines and HIV replication in an HIV-1 indicator cell line (IC50 ∼23.1 μM).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoQ1jtoA-Z8RrVg90H21EOLACvtfcHk0lh4C4OeC_XeYg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXotV2htbw%253D&md5=cd04f57b1fececc2f4aa678fe3f1f3f2</span></div><a href="/servlet/linkout?suffix=cit209&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.596&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.596%26sid%3Dliteratum%253Aachs%26aulast%3DStelzer%26aufirst%3DA.%2BC.%26aulast%3DFrank%26aufirst%3DA.%2BT.%26aulast%3DKratz%26aufirst%3DJ.%2BD.%26aulast%3DSwanson%26aufirst%3DM.%2BD.%26aulast%3DGonzalez-Hernandez%26aufirst%3DM.%2BJ.%26aulast%3DLee%26aufirst%3DJ.%26aulast%3DAndricioaei%26aufirst%3DI.%26aulast%3DMarkovitz%26aufirst%3DD.%2BM.%26aulast%3DAl-Hashimi%26aufirst%3DH.%2BM.%26atitle%3DDiscovery%2520of%2520Selective%2520Bioactive%2520Small%2520Molecules%2520by%2520Targeting%2520an%2520RNA%2520Dynamic%2520Ensemble%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2011%26volume%3D7%26spage%3D553%26epage%3D559%26doi%3D10.1038%2Fnchembio.596" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref210"><div class="reference"><strong class="refLabel"><a href="#ref210" class="rightTabRefNumLink">210</a></strong><div class="NLM_citation" id="rightTab-cit210"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rizvi, N. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Howe, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nahvi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klein, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fischmann, T. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, H.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walker, S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hruza, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richards, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chamberlin, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saradjian, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Butko, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mercado, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burchard, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Strickland, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dandliker, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, G. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nickbarg, E. B.</span></span> <span> </span><span class="NLM_article-title">Discovery of Selective RNA-Binding Small Molecules by Affinity-Selection Mass Spectrometry</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">820</span>– <span class="NLM_lpage">831</span>, <span class="refDoi"> DOI: 10.1021/acschembio.7b01013</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.7b01013" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref210/cit210&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC1cXit1yku7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2018&pages=820-831&author=N.+F.+Rizviauthor=J.+A.+Howeauthor=A.+Nahviauthor=D.+J.+Kleinauthor=T.+O.+Fischmannauthor=H.-Y.+Kimauthor=M.+A.+McCoyauthor=S.+S.+Walkerauthor=A.+Hruzaauthor=M.+P.+Richardsauthor=C.+Chamberlinauthor=P.+Saradjianauthor=M.+T.+Butkoauthor=G.+Mercadoauthor=J.+Burchardauthor=C.+Stricklandauthor=P.+J.+Dandlikerauthor=G.+F.+Smithauthor=E.+B.+Nickbarg&title=Discovery+of+Selective+RNA-Binding+Small+Molecules+by+Affinity-Selection+Mass+Spectrometry&doi=10.1021%2Facschembio.7b01013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit210R"><div class="casContent"><span class="casTitleNuber">210</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Selective RNA-Binding Small Molecules by Affinity-Selection Mass Spectrometry</span></div><div class="casAuthors">Rizvi, Noreen F.; Howe, John A.; Nahvi, Ali; Klein, Daniel J.; Fischmann, Thierry O.; Kim, Hai-Young; McCoy, Mark A.; Walker, Scott S.; Hruza, Alan; Richards, Matthew P.; Chamberlin, Chad; Saradjian, Peter; Butko, Margaret T.; Mercado, Gabriel; Burchard, Julja; Strickland, Corey; Dandliker, Peter J.; Smith, Graham F.; Nickbarg, Elliott B.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">820-831</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Recent advances in understanding the relevance of noncoding RNA (ncRNA) to disease have increased interest in drugging ncRNA with small mols.  The recent discovery of ribocil, a structurally distinct synthetic mimic of the natural ligand of the FMN riboswitch, revealed the potential chem. diversity of small mols. that target ncRNA.  Affinity-selection mass spectrometry (AS-MS) is theor. applicable to high-throughput screening (HTS) of small mols. binding to ncRNA.  Here, the authors report the first application of the Automated Ligand Detection System (ALIS), an indirect AS-MS technique, for the selective detection of small mol.-ncRNA interactions, high-throughput screening against large unbiased small-mol. libraries, and identification and characterization of novel compds. (structurally distinct from both FMN and ribocil) that target the FMN riboswitch.  Crystal structures reveal that different compds. induce various conformations of the FMN riboswitch, leading to different activity profiles.  The authors' findings validate the ALIS platform for HTS screening for RNA-binding small mols. and further demonstrate that ncRNA can be broadly targeted by chem. diverse yet selective small mols. as therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoW6w6M7wToLrVg90H21EOLACvtfcHk0lh4C4OeC_XeYg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXit1yku7o%253D&md5=b489e442832bcd12515e7c07ebc69432</span></div><a href="/servlet/linkout?suffix=cit210&amp;dbid=16384&amp;doi=10.1021%2Facschembio.7b01013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.7b01013%26sid%3Dliteratum%253Aachs%26aulast%3DRizvi%26aufirst%3DN.%2BF.%26aulast%3DHowe%26aufirst%3DJ.%2BA.%26aulast%3DNahvi%26aufirst%3DA.%26aulast%3DKlein%26aufirst%3DD.%2BJ.%26aulast%3DFischmann%26aufirst%3DT.%2BO.%26aulast%3DKim%26aufirst%3DH.-Y.%26aulast%3DMcCoy%26aufirst%3DM.%2BA.%26aulast%3DWalker%26aufirst%3DS.%2BS.%26aulast%3DHruza%26aufirst%3DA.%26aulast%3DRichards%26aufirst%3DM.%2BP.%26aulast%3DChamberlin%26aufirst%3DC.%26aulast%3DSaradjian%26aufirst%3DP.%26aulast%3DButko%26aufirst%3DM.%2BT.%26aulast%3DMercado%26aufirst%3DG.%26aulast%3DBurchard%26aufirst%3DJ.%26aulast%3DStrickland%26aufirst%3DC.%26aulast%3DDandliker%26aufirst%3DP.%2BJ.%26aulast%3DSmith%26aufirst%3DG.%2BF.%26aulast%3DNickbarg%26aufirst%3DE.%2BB.%26atitle%3DDiscovery%2520of%2520Selective%2520RNA-Binding%2520Small%2520Molecules%2520by%2520Affinity-Selection%2520Mass%2520Spectrometry%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2018%26volume%3D13%26spage%3D820%26epage%3D831%26doi%3D10.1021%2Facschembio.7b01013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref211"><div class="reference"><strong class="refLabel"><a href="#ref211" class="rightTabRefNumLink">211</a></strong><div class="NLM_citation" id="rightTab-cit211"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lai, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Induced Protein Degradation: An Emerging Drug Discovery Paradigm</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">101</span>– <span class="NLM_lpage">114</span>, <span class="refDoi"> DOI: 10.1038/nrd.2016.211</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref211/cit211&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1038%2Fnrd.2016.211" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref211/cit211&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=27885283" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref211/cit211&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvFCit7rP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2017&pages=101-114&author=A.+C.+Laiauthor=C.+M.+Crews&title=Induced+Protein+Degradation%3A+An+Emerging+Drug+Discovery+Paradigm&doi=10.1038%2Fnrd.2016.211"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit211R"><div class="casContent"><span class="casTitleNuber">211</span><div class="casTitle"><span class="NLM_cas:atitle">Induced protein degradation: an emerging drug discovery paradigm</span></div><div class="casAuthors">Lai, Ashton C.; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">101-114</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Small-mol. drug discovery has traditionally focused on occupancy of a binding site that directly affects protein function, and this approach typically precludes targeting proteins that lack such amenable sites.  Furthermore, high systemic drug exposures may be needed to maintain sufficient target inhibition in vivo, increasing the risk of undesirable off-target effects.  Induced protein degrdn. is an alternative approach that is event-driven: upon drug binding, the target protein is tagged for elimination.  Emerging technologies based on proteolysis-targeting chimaeras (PROTACs) that exploit cellular quality control machinery to selectively degrade target proteins are attracting considerable attention in the pharmaceutical industry owing to the advantages they could offer over traditional small-mol. strategies.  These advantages include the potential to reduce systemic drug exposure, the ability to counteract increased target protein expression that often accompanies inhibition of protein function and the potential ability to target proteins that are not currently therapeutically tractable, such as transcription factors, scaffolding and regulatory proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoj-O7zXGOpXLVg90H21EOLACvtfcHk0lgaVAhpDNcs8g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvFCit7rP&md5=a01ff43e9b696fe4dc03f0815158a98f</span></div><a href="/servlet/linkout?suffix=cit211&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2016.211&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2016.211%26sid%3Dliteratum%253Aachs%26aulast%3DLai%26aufirst%3DA.%2BC.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DInduced%2520Protein%2520Degradation%253A%2520An%2520Emerging%2520Drug%2520Discovery%2520Paradigm%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2017%26volume%3D16%26spage%3D101%26epage%3D114%26doi%3D10.1038%2Fnrd.2016.211" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref212"><div class="reference"><strong class="refLabel"><a href="#ref212" class="rightTabRefNumLink">212</a></strong><div class="NLM_citation" id="rightTab-cit212"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cromm, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Targeted Protein Degradation: From Chemical Biology to Drug Discovery</span>. <i>Cell Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">1181</span>– <span class="NLM_lpage">1190</span>, <span class="refDoi"> DOI: 10.1016/j.chembiol.2017.05.024</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref212/cit212&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1016%2Fj.chembiol.2017.05.024" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref212/cit212&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=28648379" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref212/cit212&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtVOjtrrO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2017&pages=1181-1190&author=P.+M.+Crommauthor=C.+M.+Crews&title=Targeted+Protein+Degradation%3A+From+Chemical+Biology+to+Drug+Discovery&doi=10.1016%2Fj.chembiol.2017.05.024"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit212R"><div class="casContent"><span class="casTitleNuber">212</span><div class="casTitle"><span class="NLM_cas:atitle">Targeted Protein Degradation: from Chemical Biology to Drug Discovery</span></div><div class="casAuthors">Cromm, Philipp M.; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1181-1190</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  Traditional pharmaceutical drug discovery is almost exclusively focused on directly controlling protein activity to cure diseases.  Modulators of protein activity, esp. inhibitors, are developed and applied at high concn. to achieve maximal effects.  Thereby, reduced bioavailability and off-target effects can hamper compd. efficacy.  Nucleic acid-based strategies that control protein function by affecting expression have emerged as an alternative.  However, metabolic stability and broad bioavailability represent development hurdles that remain to be overcome for these approaches.  More recently, utilizing the cell's own protein destruction machinery for selective degrdn. of essential drivers of human disorders has opened up a new and exciting area of drug discovery.  Small-mol.-induced proteolysis of selected substrates offers the potential of reaching beyond the limitations of the current pharmaceutical paradigm to expand the druggable target space.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrosuhmnPE0RrVg90H21EOLACvtfcHk0lgaVAhpDNcs8g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtVOjtrrO&md5=3c8c6d9488396d5eb08ca51cee20bbf4</span></div><a href="/servlet/linkout?suffix=cit212&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2017.05.024&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2017.05.024%26sid%3Dliteratum%253Aachs%26aulast%3DCromm%26aufirst%3DP.%2BM.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DTargeted%2520Protein%2520Degradation%253A%2520From%2520Chemical%2520Biology%2520to%2520Drug%2520Discovery%26jtitle%3DCell%2520Chem.%2520Biol.%26date%3D2017%26volume%3D24%26spage%3D1181%26epage%3D1190%26doi%3D10.1016%2Fj.chembiol.2017.05.024" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref213"><div class="reference"><strong class="refLabel"><a href="#ref213" class="rightTabRefNumLink">213</a></strong><div class="NLM_citation" id="rightTab-cit213"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lebraud, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heightman, T. D.</span></span> <span> </span><span class="NLM_article-title">Protein Degradation: A Validated Therapeutic Strategy with Exciting Prospects</span>. <i>Essays Biochem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">517</span>– <span class="NLM_lpage">527</span>, <span class="refDoi"> DOI: 10.1042/EBC20170030</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref213/cit213&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1042%2FEBC20170030" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref213/cit213&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=28970340" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref213/cit213&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A280%3ADC%252BC1M%252FksFCjsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2017&pages=517-527&author=H.+Lebraudauthor=T.+D.+Heightman&title=Protein+Degradation%3A+A+Validated+Therapeutic+Strategy+with+Exciting+Prospects&doi=10.1042%2FEBC20170030"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit213R"><div class="casContent"><span class="casTitleNuber">213</span><div class="casTitle"><span class="NLM_cas:atitle">Protein degradation: a validated therapeutic strategy with exciting prospects</span></div><div class="casAuthors">Lebraud Honorine; Heightman Tom D</div><div class="citationInfo"><span class="NLM_cas:title">Essays in biochemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">517-527</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">In a time of unprecedented challenges in developing potent, selective and well-tolerated protein inhibitors as therapeutics, drug hunters are increasingly seeking alternative modalities to modulate pharmacological targets.  Selective inhibitors are achievable for only a fraction of the proteome, and are not guaranteed to elicit the desired response in patients, especially when pursuing targets identified through genetic knockdown.  Targeted protein degradation holds the potential to expand the range of proteins that can be effectively modulated.  Drugs inducing protein degradation through misfolding or by modulating cereblon (CRBN) substrate recognition are already approved for treatment of cancer patients.  The last decade has seen the development of proteolysis targeting chimeras (PROTACs), small molecules that elicit proteasomal degradation by causing protein polyubiquitination.  These have been used to degrade a range of disease-relevant proteins in cells, and some show promising efficacy in preclinical animal models, although their clinical efficacy and tolerability is yet to be proven.  This review introduces current strategies for protein degradation with an emphasis on PROTACs and the role of click chemistry in PROTAC research through the formation of libraries of preclicked PROTACs or in-cell click-formed PROTACs (CLIPTACs).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSAv7HAvl2UWlQKX3ooHxWjfW6udTcc2ebzQTYAl4v5Wrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1M%252FksFCjsw%253D%253D&md5=00c39113eef9f1c1e6382fbea2d5f8d4</span></div><a href="/servlet/linkout?suffix=cit213&amp;dbid=16384&amp;doi=10.1042%2FEBC20170030&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252FEBC20170030%26sid%3Dliteratum%253Aachs%26aulast%3DLebraud%26aufirst%3DH.%26aulast%3DHeightman%26aufirst%3DT.%2BD.%26atitle%3DProtein%2520Degradation%253A%2520A%2520Validated%2520Therapeutic%2520Strategy%2520with%2520Exciting%2520Prospects%26jtitle%3DEssays%2520Biochem.%26date%3D2017%26volume%3D61%26spage%3D517%26epage%3D527%26doi%3D10.1042%2FEBC20170030" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref214"><div class="reference"><strong class="refLabel"><a href="#ref214" class="rightTabRefNumLink">214</a></strong><div class="NLM_citation" id="rightTab-cit214"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Toure, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Small-Molecule PROTACS: New Approaches to Protein Degradation</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">1966</span>– <span class="NLM_lpage">1973</span>, <span class="refDoi"> DOI: 10.1002/anie.201507978</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref214/cit214&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1002%2Fanie.201507978" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref214/cit214&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC28XnsF2ltA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2016&pages=1966-1973&author=M.+Toureauthor=C.+M.+Crews&title=Small-Molecule+PROTACS%3A+New+Approaches+to+Protein+Degradation&doi=10.1002%2Fanie.201507978"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit214R"><div class="casContent"><span class="casTitleNuber">214</span><div class="casTitle"><span class="NLM_cas:atitle">Small-Molecule PROTACS: New Approaches to Protein Degradation</span></div><div class="casAuthors">Toure, Momar; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1966-1973</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A review.  The current inhibitor-based approach to therapeutics has inherent limitations owing to its occupancy-based model: there is a need to maintain high systemic exposure to ensure sufficient in vivo inhibition, high in vivo concns. bring potential for off-target side effects, and there is a need to bind to an active site, thus limiting the drug target space.  As an alternative, induced protein degrdn. lacks these limitations.  Based on an event-driven model, this approach offers a novel catalytic mechanism to irreversibly inhibit protein function by targeting protein destruction through recruitment to the cellular quality control machinery.  Prior protein degrading strategies have lacked therapeutic potential.  However, recent reports of small-mol.-based proteolysis-targeting chimeras (PROTACs) have demonstrated that this technol. can effectively decrease the cellular levels of several protein classes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrdaqlIjCSsgrVg90H21EOLACvtfcHk0lgaVAhpDNcs8g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XnsF2ltA%253D%253D&md5=47838cd15719757317cb8282c26cc59e</span></div><a href="/servlet/linkout?suffix=cit214&amp;dbid=16384&amp;doi=10.1002%2Fanie.201507978&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201507978%26sid%3Dliteratum%253Aachs%26aulast%3DToure%26aufirst%3DM.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DSmall-Molecule%2520PROTACS%253A%2520New%2520Approaches%2520to%2520Protein%2520Degradation%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2016%26volume%3D55%26spage%3D1966%26epage%3D1973%26doi%3D10.1002%2Fanie.201507978" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref215"><div class="reference"><strong class="refLabel"><a href="#ref215" class="rightTabRefNumLink">215</a></strong><div class="NLM_citation" id="rightTab-cit215"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ottis, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Proteolysis-Targeting Chimeras: Induced Protein Degradation as a Therapeutic Strategy</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">892</span>– <span class="NLM_lpage">898</span>, <span class="refDoi"> DOI: 10.1021/acschembio.6b01068</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.6b01068" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref215/cit215&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC2sXjs12ns70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2017&pages=892-898&author=P.+Ottisauthor=C.+M.+Crews&title=Proteolysis-Targeting+Chimeras%3A+Induced+Protein+Degradation+as+a+Therapeutic+Strategy&doi=10.1021%2Facschembio.6b01068"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit215R"><div class="casContent"><span class="casTitleNuber">215</span><div class="casTitle"><span class="NLM_cas:atitle">Proteolysis-Targeting Chimeras: Induced Protein Degradation as a Therapeutic Strategy</span></div><div class="casAuthors">Ottis, Philipp; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">892-898</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Until recently, the only ways to reduce specific protein signaling were to either knock down the target by RNAi or to interfere with the signaling by inhibiting an enzyme or receptor within the signal transduction cascade.  Herein, we review an emerging class of small mol. pharmacol. agents, called PROTACs, that present a novel approach to specifically target proteins and their resp. signaling pathways.  These heterobifunctional mols. utilize endogenous cellular quality control machinery by recruiting it to target proteins in order to induce their degrdn.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGohB6xwkQMTprVg90H21EOLACvtfcHk0lgZKiiRI4x5eA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXjs12ns70%253D&md5=eada521c10faa1e456e0885fd390c554</span></div><a href="/servlet/linkout?suffix=cit215&amp;dbid=16384&amp;doi=10.1021%2Facschembio.6b01068&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.6b01068%26sid%3Dliteratum%253Aachs%26aulast%3DOttis%26aufirst%3DP.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DProteolysis-Targeting%2520Chimeras%253A%2520Induced%2520Protein%2520Degradation%2520as%2520a%2520Therapeutic%2520Strategy%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2017%26volume%3D12%26spage%3D892%26epage%3D898%26doi%3D10.1021%2Facschembio.6b01068" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref216"><div class="reference"><strong class="refLabel"><a href="#ref216" class="rightTabRefNumLink">216</a></strong><div class="NLM_citation" id="rightTab-cit216"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Crew, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raina, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qian, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vigil, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Serebrenik, Y. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamman, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morgan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferraro, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neklesa, T. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winkler, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coleman, K. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crews, C. M.</span></span> <span> </span><span class="NLM_article-title">Identification and Characterization of Von Hippel-Lindau-Recruiting Proteolysis Targeting Chimeras (PROTACs) of TANK-Binding Kinase 1</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">583</span>– <span class="NLM_lpage">598</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b00635</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b00635" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref216/cit216&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtFelu7rK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=583-598&author=A.+P.+Crewauthor=K.+Rainaauthor=H.+Dongauthor=Y.+Qianauthor=J.+Wangauthor=D.+Vigilauthor=Y.+V.+Serebrenikauthor=B.+D.+Hammanauthor=A.+Morganauthor=C.+Ferraroauthor=K.+Siuauthor=T.+K.+Neklesaauthor=J.+D.+Winklerauthor=K.+G.+Colemanauthor=C.+M.+Crews&title=Identification+and+Characterization+of+Von+Hippel-Lindau-Recruiting+Proteolysis+Targeting+Chimeras+%28PROTACs%29+of+TANK-Binding+Kinase+1&doi=10.1021%2Facs.jmedchem.7b00635"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit216R"><div class="casContent"><span class="casTitleNuber">216</span><div class="casTitle"><span class="NLM_cas:atitle">Identification and Characterization of Von Hippel-Lindau-Recruiting Proteolysis Targeting Chimeras (PROTACs) of TANK-Binding Kinase 1</span></div><div class="casAuthors">Crew, Andrew P.; Raina, Kanak; Dong, Hanqing; Qian, Yimin; Wang, Jing; Vigil, Dominico; Serebrenik, Yevgeniy V.; Hamman, Brian D.; Morgan, Alicia; Ferraro, Caterina; Siu, Kam; Neklesa, Taavi K.; Winkler, James D.; Coleman, Kevin G.; Crews, Craig M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">583-598</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Proteolysis Targeting Chimeras (PROTACs) are bifunctional mols. that recruit an E3 ligase to a target protein to facilitate ubiquitination and subsequent degrdn. of that protein.  While the field of targeted degraders is still relatively young, the potential for this modality to become a differentiated and therapeutic reality is strong, such that both academic and pharmaceutical institutions are now entering this interesting area of research.  In this paper, the authors describe a broadly applicable process for identifying degrader hits based around the serine/threonine kinase TANK-binding kinase 1 (TBK1), and have generalized the key structural elements assocd. with degrdn. activities.  Compd. I is a potent hit (TBK1 DC50 = 12 nM, Dmax = 96%) with excellent selectivity against a related kinase IKKε, which was further used as a chem. tool to assess TBK1 as a target in mutant K-Ras cancer cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoIJWCnFncQRLVg90H21EOLACvtfcHk0lgZKiiRI4x5eA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtFelu7rK&md5=41c3627e05ca31aee3ba85815685fcb2</span></div><a href="/servlet/linkout?suffix=cit216&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b00635&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b00635%26sid%3Dliteratum%253Aachs%26aulast%3DCrew%26aufirst%3DA.%2BP.%26aulast%3DRaina%26aufirst%3DK.%26aulast%3DDong%26aufirst%3DH.%26aulast%3DQian%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DVigil%26aufirst%3DD.%26aulast%3DSerebrenik%26aufirst%3DY.%2BV.%26aulast%3DHamman%26aufirst%3DB.%2BD.%26aulast%3DMorgan%26aufirst%3DA.%26aulast%3DFerraro%26aufirst%3DC.%26aulast%3DSiu%26aufirst%3DK.%26aulast%3DNeklesa%26aufirst%3DT.%2BK.%26aulast%3DWinkler%26aufirst%3DJ.%2BD.%26aulast%3DColeman%26aufirst%3DK.%2BG.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26atitle%3DIdentification%2520and%2520Characterization%2520of%2520Von%2520Hippel-Lindau-Recruiting%2520Proteolysis%2520Targeting%2520Chimeras%2520%2528PROTACs%2529%2520of%2520TANK-Binding%2520Kinase%25201%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D583%26epage%3D598%26doi%3D10.1021%2Facs.jmedchem.7b00635" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref217"><div class="reference"><strong class="refLabel"><a href="#ref217" class="rightTabRefNumLink">217</a></strong><div class="NLM_citation" id="rightTab-cit217"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yin, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qin, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Z.</span></span> <span> </span><span class="NLM_article-title">Development of Stabilized Peptide-Based PROTACs against Estrogen Receptor Α</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">628</span>– <span class="NLM_lpage">635</span>, <span class="refDoi"> DOI: 10.1021/acschembio.7b00985</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.7b00985" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref217/cit217&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC2sXitVaksr3M" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2018&pages=628-635&author=Y.+Jiangauthor=Q.+Dengauthor=H.+Zhaoauthor=M.+Xieauthor=L.+Chenauthor=F.+Yinauthor=X.+Qinauthor=W.+Zhengauthor=Y.+Zhaoauthor=Z.+Li&title=Development+of+Stabilized+Peptide-Based+PROTACs+against+Estrogen+Receptor+%CE%91&doi=10.1021%2Facschembio.7b00985"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit217R"><div class="casContent"><span class="casTitleNuber">217</span><div class="casTitle"><span class="NLM_cas:atitle">Development of Stabilized Peptide-Based PROTACs against Estrogen Receptor α</span></div><div class="casAuthors">Jiang, Yanhong; Deng, Qiwen; Zhao, Hui; Xie, Mingsheng; Chen, Longjian; Yin, Feng; Qin, Xuan; Zheng, Weihao; Zhao, Yongjuan; Li, Zigang</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">628-635</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Peptide modulators targeting protein-protein interactions (PPIs) exhibit greater potential than small-mol. drugs in several important aspects including facile modification and relative large contact surface area.  Stabilized peptides constructed by variable chem. methods exhibit improved peptide stability and cell permeability compared to that of the linears.  Herein, we designed a stabilized peptide-based proteolysis-targeting chimera (PROTAC) targeting estrogen receptor α (ERα) by tethering an N-terminal aspartic acid cross-linked stabilized peptide ERα modulator (TD-PERM) with a pentapeptide that binds the Von Hippel-Lindau (VHL) E3 ubiquitin ligase complex.  The resulting heterobifunctional peptide (TD-PROTAC) selectively recruits ERα to the VHL E3 ligase complex, leading to the degrdn. of ERα in a proteasome-dependent manner.  Compared with the control peptides, TD-PROTAC shows significantly enhanced activities in reducing the transcription of the ERα-downstream genes and inhibiting the proliferation of ERα-pos. breast cancer cells.  In addn., in vivo expts. indicate that TD-PROTAC leads to tumor regression in the MCF-7 mouse xenograft model.  This work is a successful attempt to construct PROTACs based on cell-permeable stabilized peptides, which significantly broadens the chem. space of PROTACs and stabilized peptides.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGovSDWxTR3LbLVg90H21EOLACvtfcHk0lgZKiiRI4x5eA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXitVaksr3M&md5=abcd6f72223751db6d96f8027cbb8c23</span></div><a href="/servlet/linkout?suffix=cit217&amp;dbid=16384&amp;doi=10.1021%2Facschembio.7b00985&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.7b00985%26sid%3Dliteratum%253Aachs%26aulast%3DJiang%26aufirst%3DY.%26aulast%3DDeng%26aufirst%3DQ.%26aulast%3DZhao%26aufirst%3DH.%26aulast%3DXie%26aufirst%3DM.%26aulast%3DChen%26aufirst%3DL.%26aulast%3DYin%26aufirst%3DF.%26aulast%3DQin%26aufirst%3DX.%26aulast%3DZheng%26aufirst%3DW.%26aulast%3DZhao%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DZ.%26atitle%3DDevelopment%2520of%2520Stabilized%2520Peptide-Based%2520PROTACs%2520against%2520Estrogen%2520Receptor%2520%25CE%2591%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2018%26volume%3D13%26spage%3D628%26epage%3D635%26doi%3D10.1021%2Facschembio.7b00985" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref218"><div class="reference"><strong class="refLabel"><a href="#ref218" class="rightTabRefNumLink">218</a></strong><div class="NLM_citation" id="rightTab-cit218"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiao, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ji, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tao, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">You, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, Z.</span></span> <span> </span><span class="NLM_article-title">Discovery of a Keap1-Dependent Peptide PROTAC to Knockdown Tau by Ubiquitination-Proteasome Degradation Pathway</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>146</i></span>,  <span class="NLM_fpage">251</span>– <span class="NLM_lpage">259</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2018.01.063</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref218/cit218&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1016%2Fj.ejmech.2018.01.063" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref218/cit218&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=29407955" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref218/cit218&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC1cXisFehsbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=146&publication_year=2018&pages=251-259&author=M.+Luauthor=T.+Liuauthor=Q.+Jiaoauthor=J.+Jiauthor=M.+Taoauthor=Y.+Liuauthor=Q.+Youauthor=Z.+Jiang&title=Discovery+of+a+Keap1-Dependent+Peptide+PROTAC+to+Knockdown+Tau+by+Ubiquitination-Proteasome+Degradation+Pathway&doi=10.1016%2Fj.ejmech.2018.01.063"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit218R"><div class="casContent"><span class="casTitleNuber">218</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a Keap1-dependent peptide PROTAC to knockdown Tau by ubiquitination-proteasome degradation pathway</span></div><div class="casAuthors">Lu, Mengchen; Liu, Tian; Jiao, Qiong; Ji, Jianai; Tao, Mengmin; Liu, Yijun; You, Qidong; Jiang, Zhengyu</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">146</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">251-259</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">Induced protein degrdn. by PROTACs has emerged as a promising strategy to target nonenzymic proteins inside the cell.  The aim of this study was to identify Keap1, a substrate adaptor protein for ubiquitin E3 ligase involved in oxidative stress regulation, as a novel candidate for PROTACs that can be applied in the degrdn. of the nonenzymic protein Tau.  A peptide PROTAC by recruiting Keap1-Cul3 ubiquitin E3 ligase was developed and applied in the degrdn. of intracellular Tau.  Peptide 1 showed strong in vitro binding with Keap1 and Tau.  With proper cell permeability, peptide 1 was found to colocalize with cellular Keap1 and resulted in the coimmunopptn. of Tau and Keap1.  The results of flow cytometry and western blotting assays showed that peptide 1 can downregulate the intracellular Tau level in both time- and concn.-dependent manner.  The application of Keap1 siRNA silencing and the proteasome inhibitor MG132 confirmed that peptide 1 could promote the Keap1-dependent poly-ubiquitination and proteasome-dependent degrdn. of Tau.  The results suggested that using PROTACs to recruit Keap1 to induce the degrdn. of Tau may show promising character in the treatment of neurodegenerative disease.  In addn., our research demonstrated that Keap1 should be a promising E3 ligase adaptor to be used in the design of novel PROTACs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp9KC7pnPqlL7Vg90H21EOLACvtfcHk0lh0CZfqvr84rQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXisFehsbY%253D&md5=165e6f510e4e2024aca6c4135d2840b2</span></div><a href="/servlet/linkout?suffix=cit218&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2018.01.063&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2018.01.063%26sid%3Dliteratum%253Aachs%26aulast%3DLu%26aufirst%3DM.%26aulast%3DLiu%26aufirst%3DT.%26aulast%3DJiao%26aufirst%3DQ.%26aulast%3DJi%26aufirst%3DJ.%26aulast%3DTao%26aufirst%3DM.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DYou%26aufirst%3DQ.%26aulast%3DJiang%26aufirst%3DZ.%26atitle%3DDiscovery%2520of%2520a%2520Keap1-Dependent%2520Peptide%2520PROTAC%2520to%2520Knockdown%2520Tau%2520by%2520Ubiquitination-Proteasome%2520Degradation%2520Pathway%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2018%26volume%3D146%26spage%3D251%26epage%3D259%26doi%3D10.1016%2Fj.ejmech.2018.01.063" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref219"><div class="reference"><strong class="refLabel"><a href="#ref219" class="rightTabRefNumLink">219</a></strong><div class="NLM_citation" id="rightTab-cit219"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Woelcke, J.</span></span> <span> </span><span class="NLM_article-title">Advancement of Lead Discovery in the Protease Platform</span>, <span class="NLM_year" style="font-weight: bold;">2013</span>, Drug Discovery Leaders Summit, Geneva, Switzerland.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2013&author=J.+Woelcke&title=Advancement+of+Lead+Discovery+in+the+Protease+Platform"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit219&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DWoelcke%26aufirst%3DJ.%26atitle%3DAdvancement%2520of%2520Lead%2520Discovery%2520in%2520the%2520Protease%2520Platform%26date%3D2013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref220"><div class="reference"><strong class="refLabel"><a href="#ref220" class="rightTabRefNumLink">220</a></strong><div class="NLM_citation" id="rightTab-cit220"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Oueis, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nardone, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jaspars, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Westwood, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Naismith, J. H.</span></span> <span> </span><span class="NLM_article-title">Synthesis of Hybrid Cyclopeptides through Enzymatic Macrocyclization</span>. <i>ChemistryOpen</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">11</span>– <span class="NLM_lpage">14</span>, <span class="refDoi"> DOI: 10.1002/open.201600134</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref220/cit220&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1002%2Fopen.201600134" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref220/cit220&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=28168143" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref220/cit220&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC28XitVyjtLjM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2017&pages=11-14&author=E.+Oueisauthor=B.+Nardoneauthor=M.+Jasparsauthor=N.+J.+Westwoodauthor=J.+H.+Naismith&title=Synthesis+of+Hybrid+Cyclopeptides+through+Enzymatic+Macrocyclization&doi=10.1002%2Fopen.201600134"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit220R"><div class="casContent"><span class="casTitleNuber">220</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis of Hybrid Cyclopeptides through Enzymatic Macrocyclization</span></div><div class="casAuthors">Oueis, Emilia; Nardone, Brunello; Jaspars, Marcel; Westwood, Nicholas J.; Naismith, James H.</div><div class="citationInfo"><span class="NLM_cas:title">ChemistryOpen</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">11-14</span>CODEN:
                <span class="NLM_cas:coden">CHOPCK</span>;
        ISSN:<span class="NLM_cas:issn">2191-1363</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Natural products comprise a diverse array of mols., many of which are biol. active.  Most natural products are derived from combinations of polyketides, peptides, sugars, and fatty-acid building blocks.  Peptidic macrocycles have attracted attention as potential therapeutics possessing cell permeability, stability, and easy-to-control variability.  Here, we show that enzymes from the patellamide biosynthetic pathway can be harnessed to make macrocycles that are hybrids of amino acids and a variety of manmade chem. building blocks, including aryl rings, polyethers, and alkyl chains.  We have made macrocycles with only three amino acids, one of which can be converted to a thiazoline or a thiazole ring.  We report the synthesis of 18 peptide hybrid macrocycles, nine of which have been isolated and fully characterized.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpMkscQjq-H9rVg90H21EOLACvtfcHk0lh0CZfqvr84rQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitVyjtLjM&md5=180f31681c98e3739b8551392d0ddde9</span></div><a href="/servlet/linkout?suffix=cit220&amp;dbid=16384&amp;doi=10.1002%2Fopen.201600134&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fopen.201600134%26sid%3Dliteratum%253Aachs%26aulast%3DOueis%26aufirst%3DE.%26aulast%3DNardone%26aufirst%3DB.%26aulast%3DJaspars%26aufirst%3DM.%26aulast%3DWestwood%26aufirst%3DN.%2BJ.%26aulast%3DNaismith%26aufirst%3DJ.%2BH.%26atitle%3DSynthesis%2520of%2520Hybrid%2520Cyclopeptides%2520through%2520Enzymatic%2520Macrocyclization%26jtitle%3DChemistryOpen%26date%3D2017%26volume%3D6%26spage%3D11%26epage%3D14%26doi%3D10.1002%2Fopen.201600134" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref221"><div class="reference"><strong class="refLabel"><a href="#ref221" class="rightTabRefNumLink">221</a></strong><div class="NLM_citation" id="rightTab-cit221"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Klein, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heal, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamilton, W. D. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boussemghoune, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tange, T. Ø.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Delegrange, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jaeschke, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hatsch, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heim, J.</span></span> <span> </span><span class="NLM_article-title">Yeast Synthetic Biology Platform Generates Novel Chemical Structures as Scaffolds for Drug Discovery</span>. <i>ACS Synth. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">314</span>– <span class="NLM_lpage">323</span>, <span class="refDoi"> DOI: 10.1021/sb400177x</span> </span><div class="citationLinks">[<a href="/doi/10.1021/sb400177x" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref221/cit221&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC2cXotVGlsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2014&pages=314-323&author=J.+Kleinauthor=J.+R.+Healauthor=W.+D.+O.+Hamiltonauthor=T.+Boussemghouneauthor=T.+%C3%98.+Tangeauthor=F.+Delegrangeauthor=G.+Jaeschkeauthor=A.+Hatschauthor=J.+Heim&title=Yeast+Synthetic+Biology+Platform+Generates+Novel+Chemical+Structures+as+Scaffolds+for+Drug+Discovery&doi=10.1021%2Fsb400177x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit221R"><div class="casContent"><span class="casTitleNuber">221</span><div class="casTitle"><span class="NLM_cas:atitle">Yeast Synthetic Biology Platform Generates Novel Chemical Structures as Scaffolds for Drug Discovery</span></div><div class="casAuthors">Klein, Jens; Heal, Jonathan R.; Hamilton, William D. O.; Boussemghoune, Thiamo; Tange, Thomas Oestergaard; Delegrange, Fanny; Jaeschke, Georg; Hatsch, Anaelle; Heim, Jutta</div><div class="citationInfo"><span class="NLM_cas:title">ACS Synthetic Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">314-323</span>CODEN:
                <span class="NLM_cas:coden">ASBCD6</span>;
        ISSN:<span class="NLM_cas:issn">2161-5063</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Synthetic biol. has been heralded as a new bioengineering platform for the prodn. of bulk and specialty chems., drugs, and fuels.  Here, the authors report for the first time a series of 74 novel compds. produced using a combinatorial genetics approach in baker's yeast.  Based on the concept of "coevolution" with target proteins in an intracellular primary survival assay, the identified, mostly scaffold-sized (200-350 MW) compds., which displayed excellent biol. activity, can be considered as prevalidated hits.  Of the mols. found, >75% have not been described previously; 20% of the compds. exhibit novel scaffolds.  Their structural and physicochem. properties comply with established rules of drug- and fragment-likeness and exhibit increased structural complexities compared to synthetically produced fragments.  In summary, the synthetic biol. approach described here represents a completely new, complementary strategy for hit and early lead identification that can be easily integrated into the existing drug discovery process.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrQ8fJGPPKWe7Vg90H21EOLACvtfcHk0lh0CZfqvr84rQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXotVGlsQ%253D%253D&md5=08b0ac399343c1230ca71808a53a6d30</span></div><a href="/servlet/linkout?suffix=cit221&amp;dbid=16384&amp;doi=10.1021%2Fsb400177x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fsb400177x%26sid%3Dliteratum%253Aachs%26aulast%3DKlein%26aufirst%3DJ.%26aulast%3DHeal%26aufirst%3DJ.%2BR.%26aulast%3DHamilton%26aufirst%3DW.%2BD.%2BO.%26aulast%3DBoussemghoune%26aufirst%3DT.%26aulast%3DTange%26aufirst%3DT.%2B%25C3%2598.%26aulast%3DDelegrange%26aufirst%3DF.%26aulast%3DJaeschke%26aufirst%3DG.%26aulast%3DHatsch%26aufirst%3DA.%26aulast%3DHeim%26aufirst%3DJ.%26atitle%3DYeast%2520Synthetic%2520Biology%2520Platform%2520Generates%2520Novel%2520Chemical%2520Structures%2520as%2520Scaffolds%2520for%2520Drug%2520Discovery%26jtitle%3DACS%2520Synth.%2520Biol.%26date%3D2014%26volume%3D3%26spage%3D314%26epage%3D323%26doi%3D10.1021%2Fsb400177x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref222"><div class="reference"><strong class="refLabel"><a href="#ref222" class="rightTabRefNumLink">222</a></strong><div class="NLM_citation" id="rightTab-cit222"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lennard, K.
R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tavassoli, A.</span></span> <span> </span><span class="NLM_article-title">Peptides Come Round: Using SICLOPPS Libraries for Early Stage Drug Discovery</span>. <i>Chem. - Eur. J.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">10608</span>– <span class="NLM_lpage">10614</span>, <span class="refDoi"> DOI: 10.1002/chem.201403117</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref222/cit222&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1002%2Fchem.201403117" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref222/cit222&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=25043886" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref222/cit222&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtFCgtL%252FF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2014&pages=10608-10614&author=K.%0AR.+Lennardauthor=A.+Tavassoli&title=Peptides+Come+Round%3A+Using+SICLOPPS+Libraries+for+Early+Stage+Drug+Discovery&doi=10.1002%2Fchem.201403117"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit222R"><div class="casContent"><span class="casTitleNuber">222</span><div class="casTitle"><span class="NLM_cas:atitle">Peptides Come Round: Using SICLOPPS Libraries for Early Stage Drug Discovery</span></div><div class="casAuthors">Lennard, Katherine R.; Tavassoli, Ali</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry - A European Journal</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">34</span>),
    <span class="NLM_cas:pages">10608-10614</span>CODEN:
                <span class="NLM_cas:coden">CEUJED</span>;
        ISSN:<span class="NLM_cas:issn">0947-6539</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A review.  Cyclic peptides are an emerging class of mol. therapeutics that are increasingly viewed as ideal backbones for modulation of protein-protein interactions.  A split-intein based method, termed SICLOPPS, enables the rapid generation of genetically encoded cyclic peptide libraries of around a hundred million members.  Here we review recent approaches using SICLOPPS for the discovery of bioactive compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGre3O4emKUgCrVg90H21EOLACvtfcHk0ljyRkNklrEhTQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtFCgtL%252FF&md5=c9c4bfb37d3f3e64eadc300c33221469</span></div><a href="/servlet/linkout?suffix=cit222&amp;dbid=16384&amp;doi=10.1002%2Fchem.201403117&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fchem.201403117%26sid%3Dliteratum%253Aachs%26aulast%3DLennard%26aufirst%3DK.%2BR.%26aulast%3DTavassoli%26aufirst%3DA.%26atitle%3DPeptides%2520Come%2520Round%253A%2520Using%2520SICLOPPS%2520Libraries%2520for%2520Early%2520Stage%2520Drug%2520Discovery%26jtitle%3DChem.%2520-%2520Eur.%2520J.%26date%3D2014%26volume%3D20%26spage%3D10608%26epage%3D10614%26doi%3D10.1002%2Fchem.201403117" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref223"><div class="reference"><strong class="refLabel"><a href="#ref223" class="rightTabRefNumLink">223</a></strong><div class="NLM_citation" id="rightTab-cit223"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Young, T. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, D. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ahmad, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Louis, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Benkovic, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schultz, P. G.</span></span> <span> </span><span class="NLM_article-title">Evolution of Cyclic Peptide Protease Inhibitors</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>108</i></span>,  <span class="NLM_fpage">11052</span>– <span class="NLM_lpage">11056</span>, <span class="refDoi"> DOI: 10.1073/pnas.1108045108</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref223/cit223&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1073%2Fpnas.1108045108" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref223/cit223&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=21690365" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref223/cit223&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC3MXptV2htL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=108&publication_year=2011&pages=11052-11056&author=T.+S.+Youngauthor=D.+D.+Youngauthor=I.+Ahmadauthor=J.+M.+Louisauthor=S.+J.+Benkovicauthor=P.+G.+Schultz&title=Evolution+of+Cyclic+Peptide+Protease+Inhibitors&doi=10.1073%2Fpnas.1108045108"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit223R"><div class="casContent"><span class="casTitleNuber">223</span><div class="casTitle"><span class="NLM_cas:atitle">Evolution of cyclic peptide protease inhibitors</span></div><div class="casAuthors">Young, Travis S.; Young, Douglas D.; Ahmad, Insha; Louis, John M.; Benkovic, Stephen J.; Schultz, Peter G.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">108</span>
        (<span class="NLM_cas:issue">27</span>),
    <span class="NLM_cas:pages">11052-11056, S11052/1-S11052/14</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">We report a bacterial system for the evolution of cyclic peptides that makes use of an expanded set of amino acid building blocks.  Orthogonal aminoacyl-tRNA synthetase/tRNACUA, together with a split intein system were used to biosynthesize a library of ribosomal peptides contg. amino acids with unique structures and reactivities.  This peptide library was subsequently used to evolve an inhibitor of HIV protease using a selection based on cellular viability.  Two of three cyclic peptides isolated after two rounds of selection contained the keto amino acid p-benzoylphenylalanine (pBzF).  The most potent peptide (G12: GIXVSL; X = pBzF) inhibited HIV protease through the formation of a covalent Schiff base adduct of the pBzF residue with the ε-amino group of Lys 14 on the protease.  This result suggests that an expanded genetic code can confer an evolutionary advantage in response to selective pressure.  Moreover, the combination of natural evolutionary processes with chem. biased building blocks provides another strategy for the generation of biol. active peptides using microbial systems.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGreESwOTK5p9rVg90H21EOLACvtfcHk0ljyRkNklrEhTQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXptV2htL0%253D&md5=b063a5c486cfa6a68801ccbb6a68237d</span></div><a href="/servlet/linkout?suffix=cit223&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1108045108&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1108045108%26sid%3Dliteratum%253Aachs%26aulast%3DYoung%26aufirst%3DT.%2BS.%26aulast%3DYoung%26aufirst%3DD.%2BD.%26aulast%3DAhmad%26aufirst%3DI.%26aulast%3DLouis%26aufirst%3DJ.%2BM.%26aulast%3DBenkovic%26aufirst%3DS.%2BJ.%26aulast%3DSchultz%26aufirst%3DP.%2BG.%26atitle%3DEvolution%2520of%2520Cyclic%2520Peptide%2520Protease%2520Inhibitors%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2011%26volume%3D108%26spage%3D11052%26epage%3D11056%26doi%3D10.1073%2Fpnas.1108045108" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref224"><div class="reference"><strong class="refLabel"><a href="#ref224" class="rightTabRefNumLink">224</a></strong><div class="NLM_citation" id="rightTab-cit224"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Miranda, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nordgren, I. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Male, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lawrence, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoakwie, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cuda, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Court, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fox, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Townsend, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Packham, G. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eccles, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tavassoli, A.</span></span> <span> </span><span class="NLM_article-title">A Cyclic Peptide Inhibitor of HIF-1 Heterodimerization That Inhibits Hypoxia Signaling in Cancer Cells</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>135</i></span>,  <span class="NLM_fpage">10418</span>– <span class="NLM_lpage">10425</span>, <span class="refDoi"> DOI: 10.1021/ja402993u</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja402993u" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref224/cit224&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC3sXpvFaksro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=135&publication_year=2013&pages=10418-10425&author=E.+Mirandaauthor=I.+K.+Nordgrenauthor=A.+L.+Maleauthor=C.+E.+Lawrenceauthor=F.+Hoakwieauthor=F.+Cudaauthor=W.+Courtauthor=K.+R.+Foxauthor=P.+A.+Townsendauthor=G.+K.+Packhamauthor=S.+A.+Ecclesauthor=A.+Tavassoli&title=A+Cyclic+Peptide+Inhibitor+of+HIF-1+Heterodimerization+That+Inhibits+Hypoxia+Signaling+in+Cancer+Cells&doi=10.1021%2Fja402993u"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit224R"><div class="casContent"><span class="casTitleNuber">224</span><div class="casTitle"><span class="NLM_cas:atitle">A Cyclic Peptide Inhibitor of HIF-1 Heterodimerization That Inhibits Hypoxia Signaling in Cancer Cells</span></div><div class="casAuthors">Miranda, Elena; Nordgren, Ida K.; Male, Abigail L.; Lawrence, Charlotte E.; Hoakwie, Franciane; Cuda, Francesco; Court, William; Fox, Keith R.; Townsend, Paul A.; Packham, Graham K.; Eccles, Suzanne A.; Tavassoli, Ali</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">135</span>
        (<span class="NLM_cas:issue">28</span>),
    <span class="NLM_cas:pages">10418-10425</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Hypoxia inducible factor-1 (HIF-1) is a heterodimeric transcription factor that acts as the master regulator of cellular response to reduced oxygen levels, thus playing a key role in the adaptation, survival, and progression of tumors.  Here we report cyclo-CLLFVY, identified from a library of 3.2 million cyclic hexapeptides using a genetically encoded high-throughput screening platform, as an inhibitor of the HIF-1α/HIF-1β protein-protein interaction in vitro and in cells.  The identified compd. inhibits HIF-1 dimerization and transcription activity by binding to the PAS-B domain of HIF-1α, reducing HIF-1-mediated hypoxia response signaling in a variety of cell lines, without affecting the function of the closely related HIF-2 isoform.  The reported cyclic peptide demonstrates the utility of our high-throughput screening platform for the identification of protein-protein interaction inhibitors, and forms the starting point for the development of HIF-1 targeted cancer therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGodwKA_rNEr2LVg90H21EOLACvtfcHk0ljyRkNklrEhTQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXpvFaksro%253D&md5=3f44683ebfa19a9dbb6f505d50bb6032</span></div><a href="/servlet/linkout?suffix=cit224&amp;dbid=16384&amp;doi=10.1021%2Fja402993u&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja402993u%26sid%3Dliteratum%253Aachs%26aulast%3DMiranda%26aufirst%3DE.%26aulast%3DNordgren%26aufirst%3DI.%2BK.%26aulast%3DMale%26aufirst%3DA.%2BL.%26aulast%3DLawrence%26aufirst%3DC.%2BE.%26aulast%3DHoakwie%26aufirst%3DF.%26aulast%3DCuda%26aufirst%3DF.%26aulast%3DCourt%26aufirst%3DW.%26aulast%3DFox%26aufirst%3DK.%2BR.%26aulast%3DTownsend%26aufirst%3DP.%2BA.%26aulast%3DPackham%26aufirst%3DG.%2BK.%26aulast%3DEccles%26aufirst%3DS.%2BA.%26aulast%3DTavassoli%26aufirst%3DA.%26atitle%3DA%2520Cyclic%2520Peptide%2520Inhibitor%2520of%2520HIF-1%2520Heterodimerization%2520That%2520Inhibits%2520Hypoxia%2520Signaling%2520in%2520Cancer%2520Cells%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2013%26volume%3D135%26spage%3D10418%26epage%3D10425%26doi%3D10.1021%2Fja402993u" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref225"><div class="reference"><strong class="refLabel"><a href="#ref225" class="rightTabRefNumLink">225</a></strong><div class="NLM_citation" id="rightTab-cit225"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Davis, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plowright, A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valeur, E.</span></span> <span> </span><span class="NLM_article-title">Directing Evolution: The next Revolution in Drug Discovery?</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">681</span>– <span class="NLM_lpage">698</span>, <span class="refDoi"> DOI: 10.1038/nrd.2017.146</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref225/cit225&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1038%2Fnrd.2017.146" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref225/cit225&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=28935911" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref225/cit225&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsFeqs7%252FP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2017&pages=681-698&author=A.+M.+Davisauthor=A.+T.+Plowrightauthor=E.+Valeur&title=Directing+Evolution%3A+The+next+Revolution+in+Drug+Discovery%3F&doi=10.1038%2Fnrd.2017.146"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit225R"><div class="casContent"><span class="casTitleNuber">225</span><div class="casTitle"><span class="NLM_cas:atitle">Directing evolution: the next revolution in drug discovery?</span></div><div class="casAuthors">Davis, Andrew M.; Plowright, Alleyn T.; Valeur, Eric</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">681-698</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">The strong biol. rationale to pursue challenging drug targets such as protein-protein interactions has stimulated the development of novel screening strategies, such as DNA-encoded libraries, to allow broader areas of chem. space to be searched.  There has also been renewed interest in screening natural products, which are the result of evolutionary selection for a function, such as interference with a key signalling pathway of a competing organism.  However, recent advances in several areas, such as understanding of the biosynthetic pathways for natural products, synthetic biol. and the development of biosensors to detect target mols., are now providing new opportunities to directly harness evolutionary pressure to identify and optimize compds. with desired bioactivities.  Here, we describe innovations in the key components of such strategies and highlight pioneering examples that indicate the potential of the directed-evolution concept.  We also discuss the scientific gaps and challenges that remain to be addressed to realize this potential more broadly in drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpy0QYUc4kt_bVg90H21EOLACvtfcHk0liqj7eTCfKdRw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsFeqs7%252FP&md5=3c9a88c827e47cb67751d9bc1607056b</span></div><a href="/servlet/linkout?suffix=cit225&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2017.146&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2017.146%26sid%3Dliteratum%253Aachs%26aulast%3DDavis%26aufirst%3DA.%2BM.%26aulast%3DPlowright%26aufirst%3DA.%2BT.%26aulast%3DValeur%26aufirst%3DE.%26atitle%3DDirecting%2520Evolution%253A%2520The%2520next%2520Revolution%2520in%2520Drug%2520Discovery%253F%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2017%26volume%3D16%26spage%3D681%26epage%3D698%26doi%3D10.1038%2Fnrd.2017.146" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref226"><div class="reference"><strong class="refLabel"><a href="#ref226" class="rightTabRefNumLink">226</a></strong><div class="NLM_citation" id="rightTab-cit226"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Badran, A. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guzov, V. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huai, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kemp, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vishwanath, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kain, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nance, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evdokimov, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moshiri, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turner, K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Malvar, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, D. R.</span></span> <span> </span><span class="NLM_article-title">Continuous Evolution of <i>Bacillus Thuringiensis</i> Toxins Overcomes Insect Resistance</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>533</i></span>,  <span class="NLM_fpage">58</span>– <span class="NLM_lpage">63</span>, <span class="refDoi"> DOI: 10.1038/nature17938</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref226/cit226&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1038%2Fnature17938" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref226/cit226&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=27120167" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref226/cit226&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC28XmvVGnt7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=533&publication_year=2016&pages=58-63&author=A.+H.+Badranauthor=V.+M.+Guzovauthor=Q.+Huaiauthor=M.+M.+Kempauthor=P.+Vishwanathauthor=W.+Kainauthor=A.+M.+Nanceauthor=A.+Evdokimovauthor=F.+Moshiriauthor=K.+H.+Turnerauthor=P.+Wangauthor=T.+Malvarauthor=D.+R.+Liu&title=Continuous+Evolution+of+Bacillus+Thuringiensis+Toxins+Overcomes+Insect+Resistance&doi=10.1038%2Fnature17938"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit226R"><div class="casContent"><span class="casTitleNuber">226</span><div class="casTitle"><span class="NLM_cas:atitle">Continuous evolution of Bacillus thuringiensis toxins overcomes insect resistance</span></div><div class="casAuthors">Badran, Ahmed H.; Guzov, Victor M.; Huai, Qing; Kemp, Melissa M.; Vishwanath, Prashanth; Kain, Wendy; Nance, Autumn M.; Evdokimov, Artem; Moshiri, Farhad; Turner, Keith H.; Wang, Ping; Malvar, Thomas; Liu, David R.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">533</span>
        (<span class="NLM_cas:issue">7601</span>),
    <span class="NLM_cas:pages">58-63</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The Bacillus thuringiensis δ-endotoxins (Bt toxins) are widely used insecticidal proteins in engineered crops that provide agricultural, economic, and environmental benefits.  The development of insect resistance to Bt toxins endangers their long-term effectiveness.  Here we have developed a phage-assisted continuous evolution selection that rapidly evolves high-affinity protein-protein interactions, and applied this system to evolve variants of the Bt toxin Cry1Ac that bind a cadherin-like receptor from the insect pest Trichoplusia ni (TnCAD) that is not natively bound by wild-type Cry1Ac.  The resulting evolved Cry1Ac variants bind TnCAD with high affinity (dissocn. const. Kd = 11-41 nM), kill TnCAD-expressing insect cells that are not susceptible to wild-type Cry1Ac, and kill Cry1Ac-resistant T. ni insects up to 335-fold more potently than wild-type Cry1Ac.  Our findings establish that the evolution of Bt toxins with novel insect cell receptor affinity can overcome insect Bt toxin resistance and confer lethality approaching that of the wild-type Bt toxin against non-resistant insects.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo2tXjekTvoL7Vg90H21EOLACvtfcHk0liqj7eTCfKdRw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XmvVGnt7Y%253D&md5=1f0de0f8d608073ccf34dd33a1d50282</span></div><a href="/servlet/linkout?suffix=cit226&amp;dbid=16384&amp;doi=10.1038%2Fnature17938&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature17938%26sid%3Dliteratum%253Aachs%26aulast%3DBadran%26aufirst%3DA.%2BH.%26aulast%3DGuzov%26aufirst%3DV.%2BM.%26aulast%3DHuai%26aufirst%3DQ.%26aulast%3DKemp%26aufirst%3DM.%2BM.%26aulast%3DVishwanath%26aufirst%3DP.%26aulast%3DKain%26aufirst%3DW.%26aulast%3DNance%26aufirst%3DA.%2BM.%26aulast%3DEvdokimov%26aufirst%3DA.%26aulast%3DMoshiri%26aufirst%3DF.%26aulast%3DTurner%26aufirst%3DK.%2BH.%26aulast%3DWang%26aufirst%3DP.%26aulast%3DMalvar%26aufirst%3DT.%26aulast%3DLiu%26aufirst%3DD.%2BR.%26atitle%3DContinuous%2520Evolution%2520of%2520Bacillus%2520Thuringiensis%2520Toxins%2520Overcomes%2520Insect%2520Resistance%26jtitle%3DNature%26date%3D2016%26volume%3D533%26spage%3D58%26epage%3D63%26doi%3D10.1038%2Fnature17938" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref227"><div class="reference"><strong class="refLabel"><a href="#ref227" class="rightTabRefNumLink">227</a></strong><div class="NLM_citation" id="rightTab-cit227"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chi, C.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ahmed, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dereli-Korkut, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span> <span> </span><span class="NLM_article-title">Microfluidic Cell Chips for High-Throughput Drug Screening</span>. <i>Bioanalysis</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">921</span>– <span class="NLM_lpage">937</span>, <span class="refDoi"> DOI: 10.4155/bio-2016-0028</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref227/cit227&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.4155%2Fbio-2016-0028" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2016&pages=921-937&author=C.-W.+Chiauthor=A.+R.+Ahmedauthor=Z.+Dereli-Korkutauthor=S.+Wang&title=Microfluidic+Cell+Chips+for+High-Throughput+Drug+Screening&doi=10.4155%2Fbio-2016-0028"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit227&amp;dbid=16384&amp;doi=10.4155%2Fbio-2016-0028&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4155%252Fbio-2016-0028%26sid%3Dliteratum%253Aachs%26aulast%3DChi%26aufirst%3DC.-W.%26aulast%3DAhmed%26aufirst%3DA.%2BR.%26aulast%3DDereli-Korkut%26aufirst%3DZ.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DMicrofluidic%2520Cell%2520Chips%2520for%2520High-Throughput%2520Drug%2520Screening%26jtitle%3DBioanalysis%26date%3D2016%26volume%3D8%26spage%3D921%26epage%3D937%26doi%3D10.4155%2Fbio-2016-0028" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref228"><div class="reference"><strong class="refLabel"><a href="#ref228" class="rightTabRefNumLink">228</a></strong><div class="NLM_citation" id="rightTab-cit228"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Klucznik, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mikulak-Klucznik, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCormack, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lima, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szymkuć, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhowmick, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Molga, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rickershauser, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gajewska, E. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toutchkine, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dittwald, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Startek, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirkovits, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roszak, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adamski, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sieredzińska, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mrksich, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trice, S. L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grzybowski, B. A.</span></span> <span> </span><span class="NLM_article-title">Efficient Syntheses of Diverse, Medicinally Relevant Targets Planned by Computer and Executed in the Laboratory</span>. <i>Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">522</span>– <span class="NLM_lpage">532</span>, <span class="refDoi"> DOI: 10.1016/j.chempr.2018.02.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref228/cit228&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1016%2Fj.chempr.2018.02.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref228/cit228&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC1cXktlyitb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2018&pages=522-532&author=T.+Klucznikauthor=B.+Mikulak-Klucznikauthor=M.+P.+McCormackauthor=H.+Limaauthor=S.+Szymku%C4%87author=M.+Bhowmickauthor=K.+Molgaauthor=Y.+Zhouauthor=L.+Rickershauserauthor=E.+P.+Gajewskaauthor=A.+Toutchkineauthor=P.+Dittwaldauthor=M.+P.+Startekauthor=G.+J.+Kirkovitsauthor=R.+Roszakauthor=A.+Adamskiauthor=B.+Sieredzi%C5%84skaauthor=M.+Mrksichauthor=S.+L.+J.+Triceauthor=B.+A.+Grzybowski&title=Efficient+Syntheses+of+Diverse%2C+Medicinally+Relevant+Targets+Planned+by+Computer+and+Executed+in+the+Laboratory&doi=10.1016%2Fj.chempr.2018.02.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit228R"><div class="casContent"><span class="casTitleNuber">228</span><div class="casTitle"><span class="NLM_cas:atitle">Efficient Syntheses of Diverse, Medicinally Relevant Targets Planned by Computer and Executed in the Laboratory</span></div><div class="casAuthors">Klucznik, Tomasz; Mikulak-Klucznik, Barbara; McCormack, Michael P.; Lima, Heather; Szymkuc, Sara; Bhowmick, Manishabrata; Molga, Karol; Zhou, Yubai; Rickershauser, Lindsey; Gajewska, Ewa P.; Toutchkine, Alexei; Dittwald, Piotr; Startek, Michal P.; Kirkovits, Gregory J.; Roszak, Rafal; Adamski, Ariel; Sieredzinska, Bianka; Mrksich, Milan; Trice, Sarah L. J.; Grzybowski, Bartosz A.</div><div class="citationInfo"><span class="NLM_cas:title">Chem</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">522-532</span>CODEN:
                <span class="NLM_cas:coden">CHEMVE</span>;
        ISSN:<span class="NLM_cas:issn">2451-9294</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The Chematica program was used to autonomously design synthetic pathways to eight structurally diverse targets, including seven com. valuable bioactive substances and one natural product.  All of these computer-planned routes were successfully executed in the lab. and offer significant yield improvements and cost savings over previous approaches, provide alternatives to patented routes, or produce targets that were not synthesized previously.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGotOimnTZh2zLVg90H21EOLACvtfcHk0lj-pYNBSYhNUw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXktlyitb8%253D&md5=2d5d22d2570ca3e146ba9c25149dfc9e</span></div><a href="/servlet/linkout?suffix=cit228&amp;dbid=16384&amp;doi=10.1016%2Fj.chempr.2018.02.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chempr.2018.02.002%26sid%3Dliteratum%253Aachs%26aulast%3DKlucznik%26aufirst%3DT.%26aulast%3DMikulak-Klucznik%26aufirst%3DB.%26aulast%3DMcCormack%26aufirst%3DM.%2BP.%26aulast%3DLima%26aufirst%3DH.%26aulast%3DSzymku%25C4%2587%26aufirst%3DS.%26aulast%3DBhowmick%26aufirst%3DM.%26aulast%3DMolga%26aufirst%3DK.%26aulast%3DZhou%26aufirst%3DY.%26aulast%3DRickershauser%26aufirst%3DL.%26aulast%3DGajewska%26aufirst%3DE.%2BP.%26aulast%3DToutchkine%26aufirst%3DA.%26aulast%3DDittwald%26aufirst%3DP.%26aulast%3DStartek%26aufirst%3DM.%2BP.%26aulast%3DKirkovits%26aufirst%3DG.%2BJ.%26aulast%3DRoszak%26aufirst%3DR.%26aulast%3DAdamski%26aufirst%3DA.%26aulast%3DSieredzi%25C5%2584ska%26aufirst%3DB.%26aulast%3DMrksich%26aufirst%3DM.%26aulast%3DTrice%26aufirst%3DS.%2BL.%2BJ.%26aulast%3DGrzybowski%26aufirst%3DB.%2BA.%26atitle%3DEfficient%2520Syntheses%2520of%2520Diverse%252C%2520Medicinally%2520Relevant%2520Targets%2520Planned%2520by%2520Computer%2520and%2520Executed%2520in%2520the%2520Laboratory%26jtitle%3DChem.%26date%3D2018%26volume%3D4%26spage%3D522%26epage%3D532%26doi%3D10.1016%2Fj.chempr.2018.02.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref229"><div class="reference"><strong class="refLabel"><a href="#ref229" class="rightTabRefNumLink">229</a></strong><div class="NLM_citation" id="rightTab-cit229"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Segler, M. H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Preuss, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waller, M. P.</span></span> <span> </span><span class="NLM_article-title">Planning Chemical Syntheses with Deep Neural Networks and Symbolic AI.</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>555</i></span>,  <span class="NLM_fpage">604</span>– <span class="NLM_lpage">610</span>, <span class="refDoi"> DOI: 10.1038/nature25978</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref229/cit229&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1038%2Fnature25978" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=555&publication_year=2018&pages=604-610&author=M.+H.+S.+Seglerauthor=M.+Preussauthor=M.+P.+Waller&title=Planning+Chemical+Syntheses+with+Deep+Neural+Networks+and+Symbolic+AI.&doi=10.1038%2Fnature25978"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit229&amp;dbid=16384&amp;doi=10.1038%2Fnature25978&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature25978%26sid%3Dliteratum%253Aachs%26aulast%3DSegler%26aufirst%3DM.%2BH.%2BS.%26aulast%3DPreuss%26aufirst%3DM.%26aulast%3DWaller%26aufirst%3DM.%2BP.%26atitle%3DPlanning%2520Chemical%2520Syntheses%2520with%2520Deep%2520Neural%2520Networks%2520and%2520Symbolic%2520AI.%26jtitle%3DNature%26date%3D2018%26volume%3D555%26spage%3D604%26epage%3D610%26doi%3D10.1038%2Fnature25978" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref230"><div class="reference"><strong class="refLabel"><a href="#ref230" class="rightTabRefNumLink">230</a></strong><div class="NLM_citation" id="rightTab-cit230"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fons, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Esquerré, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Versluys, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mambrini, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paillasse, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bell, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schreyer, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cox, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bickerton, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lisztwan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whittaker, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bono, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnstone, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hopkins, A.</span></span> <span> </span><span class="NLM_article-title">Abstract 3970: Targeting the Adenosine Immunosuppressive Pathway for Cancer Immunotherapy with Small Molecule Agents</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>77</i></span>,  <span class="NLM_fpage">3970</span>, <span class="refDoi"> DOI: 10.1158/1538-7445.AM2017-3970</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref230/cit230&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1158%2F1538-7445.AM2017-3970" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=77&publication_year=2017&pages=3970&author=P.+Fonsauthor=M.+Esquerr%C3%A9author=S.+Versluysauthor=G.+Mambriniauthor=M.+Paillasseauthor=A.+Bellauthor=A.+Schreyerauthor=R.+Coxauthor=R.+Bickertonauthor=J.+Lisztwanauthor=M.+Whittakerauthor=F.+Bonoauthor=C.+Johnstoneauthor=A.+Hopkins&title=Abstract+3970%3A+Targeting+the+Adenosine+Immunosuppressive+Pathway+for+Cancer+Immunotherapy+with+Small+Molecule+Agents&doi=10.1158%2F1538-7445.AM2017-3970"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit230&amp;dbid=16384&amp;doi=10.1158%2F1538-7445.AM2017-3970&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1538-7445.AM2017-3970%26sid%3Dliteratum%253Aachs%26aulast%3DFons%26aufirst%3DP.%26aulast%3DEsquerr%25C3%25A9%26aufirst%3DM.%26aulast%3DVersluys%26aufirst%3DS.%26aulast%3DMambrini%26aufirst%3DG.%26aulast%3DPaillasse%26aufirst%3DM.%26aulast%3DBell%26aufirst%3DA.%26aulast%3DSchreyer%26aufirst%3DA.%26aulast%3DCox%26aufirst%3DR.%26aulast%3DBickerton%26aufirst%3DR.%26aulast%3DLisztwan%26aufirst%3DJ.%26aulast%3DWhittaker%26aufirst%3DM.%26aulast%3DBono%26aufirst%3DF.%26aulast%3DJohnstone%26aufirst%3DC.%26aulast%3DHopkins%26aufirst%3DA.%26atitle%3DAbstract%25203970%253A%2520Targeting%2520the%2520Adenosine%2520Immunosuppressive%2520Pathway%2520for%2520Cancer%2520Immunotherapy%2520with%2520Small%2520Molecule%2520Agents%26jtitle%3DCancer%2520Res.%26date%3D2017%26volume%3D77%26spage%3D3970%26doi%3D10.1158%2F1538-7445.AM2017-3970" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref231"><div class="reference"><strong class="refLabel"><a href="#ref231" class="rightTabRefNumLink">231</a></strong><div class="NLM_citation" id="rightTab-cit231"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ptacin, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fischer, E. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aerni, H. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caffaro, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">San Jose, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feldman, A. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turner, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Romesberg, F. E.</span></span> <span> </span><span class="NLM_article-title">A Semi-Synthetic Organism That Stores and Retrieves Increased Genetic Information</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>551</i></span>,  <span class="NLM_fpage">644</span>– <span class="NLM_lpage">647</span>, <span class="refDoi"> DOI: 10.1038/nature24659</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref231/cit231&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1038%2Fnature24659" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref231/cit231&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=29189780" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref231/cit231&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvFWhtL7L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=551&publication_year=2017&pages=644-647&author=Y.+Zhangauthor=J.+L.+Ptacinauthor=E.+C.+Fischerauthor=H.+R.+Aerniauthor=C.+E.+Caffaroauthor=K.+San+Joseauthor=A.+W.+Feldmanauthor=C.+R.+Turnerauthor=F.+E.+Romesberg&title=A+Semi-Synthetic+Organism+That+Stores+and+Retrieves+Increased+Genetic+Information&doi=10.1038%2Fnature24659"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit231R"><div class="casContent"><span class="casTitleNuber">231</span><div class="casTitle"><span class="NLM_cas:atitle">A semi-synthetic organism that stores and retrieves increased genetic information</span></div><div class="casAuthors">Zhang, Yorke; Ptacin, Jerod L.; Fischer, Emil C.; Aerni, Hans R.; Caffaro, Carolina E.; San Jose, Kristine; Feldman, Aaron W.; Turner, Court R.; Romesberg, Floyd E.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">551</span>
        (<span class="NLM_cas:issue">7682</span>),
    <span class="NLM_cas:pages">644-647</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Since at least the last common ancestor of all life on Earth, genetic information has been stored in a four-letter alphabet that is propagated and retrieved by the formation of two base pairs.  The central goal of synthetic biol. is to create new life forms and functions, and the most general route to this goal is the creation of semi-synthetic organisms whose DNA harbors two addnl. letters that form a third, unnatural base pair.  Previous efforts to generate such semi-synthetic organisms culminated in the creation of a strain of Escherichia coli that, by virtue of a nucleoside triphosphate transporter from Phaeodactylum tricornutum, imports the requisite unnatural triphosphates from its medium and then uses them to replicate a plasmid contg. the unnatural base pair dNaM-dTPT3.  Although the semi-synthetic organism stores increased information when compared to natural organisms, retrieval of the information requires in vivo transcription of the unnatural base pair into mRNA and tRNA, aminoacylation of the tRNA with a non-canonical amino acid, and efficient participation of the unnatural base pair in decoding at the ribosome.  Here we report the in vivo transcription of DNA contg. dNaM and dTPT3 into mRNAs with two different unnatural codons and tRNAs with cognate unnatural anticodons, and their efficient decoding at the ribosome to direct the site-specific incorporation of natural or non-canonical amino acids into superfolder green fluorescent protein.  The results demonstrate that interactions other than hydrogen bonding can contribute to every step of information storage and retrieval.  The resulting semi-synthetic organism both encodes and retrieves increased information and should serve as a platform for the creation of new life forms and functions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqjEtvNNzQsZrVg90H21EOLACvtfcHk0lj-pYNBSYhNUw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvFWhtL7L&md5=d04861904783b1a852d4089b28f09acf</span></div><a href="/servlet/linkout?suffix=cit231&amp;dbid=16384&amp;doi=10.1038%2Fnature24659&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature24659%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DPtacin%26aufirst%3DJ.%2BL.%26aulast%3DFischer%26aufirst%3DE.%2BC.%26aulast%3DAerni%26aufirst%3DH.%2BR.%26aulast%3DCaffaro%26aufirst%3DC.%2BE.%26aulast%3DSan%2BJose%26aufirst%3DK.%26aulast%3DFeldman%26aufirst%3DA.%2BW.%26aulast%3DTurner%26aufirst%3DC.%2BR.%26aulast%3DRomesberg%26aufirst%3DF.%2BE.%26atitle%3DA%2520Semi-Synthetic%2520Organism%2520That%2520Stores%2520and%2520Retrieves%2520Increased%2520Genetic%2520Information%26jtitle%3DNature%26date%3D2017%26volume%3D551%26spage%3D644%26epage%3D647%26doi%3D10.1038%2Fnature24659" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref232"><div class="reference"><strong class="refLabel"><a href="#ref232" class="rightTabRefNumLink">232</a></strong><div class="NLM_citation" id="rightTab-cit232"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Valeur, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knerr, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ölwegård-Halvarsson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lemurell, M.</span></span> <span> </span><span class="NLM_article-title">Targeted Delivery for Regenerative Medicines: An Untapped Opportunity for Drug Conjugates</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">841</span>– <span class="NLM_lpage">847</span>, <span class="refDoi"> DOI: 10.1016/j.drudis.2016.12.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref232/cit232&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=10.1016%2Fj.drudis.2016.12.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref232/cit232&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=27988360" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref232/cit232&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;key=1%3ACAS%3A528%3ADC%252BC28XitFSlsb3N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2017&pages=841-847&author=E.+Valeurauthor=L.+Knerrauthor=M.+%C3%96lweg%C3%A5rd-Halvarssonauthor=M.+Lemurell&title=Targeted+Delivery+for+Regenerative+Medicines%3A+An+Untapped+Opportunity+for+Drug+Conjugates&doi=10.1016%2Fj.drudis.2016.12.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit232R"><div class="casContent"><span class="casTitleNuber">232</span><div class="casTitle"><span class="NLM_cas:atitle">Targeted delivery for regenerative medicines: an untapped opportunity for drug conjugates</span></div><div class="casAuthors">Valeur, Eric; Knerr, Laurent; Oelwegaard-Halvarsson, Maria; Lemurell, Malin</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">841-847</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Regenerative approaches are promising avenues to effectively cure diseases rather than merely treating symptoms, but are assocd. with concerns around proliferation in other organs.  Given that targeted delivery holds the promise of delivering a drug precisely to its desired site of action, usually with the prospect of increasing the therapeutic index, it can be considered as an essential enabler of regenerative medicines.  Although significant progress has been made predominantly in oncol. for the delivery of cytotoxic drugs using antibody-drug conjugates (ADCs), the physiol. conditions and safety requirements for regenerative medicines are very different.  Drug conjugates need to be approached differently and, we herein suggest using a broader range of homing modalities and a specific framework to develop safe linkers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqADJijsrrGXLVg90H21EOLACvtfcHk0lj-pYNBSYhNUw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitFSlsb3N&md5=482bf655641afc22051dd4e222546e13</span></div><a href="/servlet/linkout?suffix=cit232&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2016.12.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2016.12.004%26sid%3Dliteratum%253Aachs%26aulast%3DValeur%26aufirst%3DE.%26aulast%3DKnerr%26aufirst%3DL.%26aulast%3D%25C3%2596lweg%25C3%25A5rd-Halvarsson%26aufirst%3DM.%26aulast%3DLemurell%26aufirst%3DM.%26atitle%3DTargeted%2520Delivery%2520for%2520Regenerative%2520Medicines%253A%2520An%2520Untapped%2520Opportunity%2520for%2520Drug%2520Conjugates%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2017%26volume%3D22%26spage%3D841%26epage%3D847%26doi%3D10.1016%2Fj.drudis.2016.12.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.8b00378&amp;pbContext=%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2018.61.issue-20%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.8b00378%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.8b00378" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                7MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"679932683ce63d66","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
