
<html lang="en"     class="pb-page"  data-request-id="47d1ddb7-cc3a-478a-9502-4268cae50b61"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";article:article:10.1021/acs.jmedchem.9b00855;requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;issue:issue:10.1021/jmcmar.2020.63.issue-16;ctype:string:Journal Content;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Deep Learning Enhancing Kinome-Wide Polypharmacology Profiling: Model Construction and Experiment Validation" /></meta><meta name="dc.Creator" content="Xutong  Li" /></meta><meta name="dc.Creator" content="Zhaojun  Li" /></meta><meta name="dc.Creator" content="Xiaolong  Wu" /></meta><meta name="dc.Creator" content="Zhaoping  Xiong" /></meta><meta name="dc.Creator" content="Tianbiao  Yang" /></meta><meta name="dc.Creator" content="Zunyun  Fu" /></meta><meta name="dc.Creator" content="Xiaohong  Liu" /></meta><meta name="dc.Creator" content="Xiaoqin  Tan" /></meta><meta name="dc.Creator" content="Feisheng  Zhong" /></meta><meta name="dc.Creator" content="Xiaozhe  Wan" /></meta><meta name="dc.Creator" content="Dingyan  Wang" /></meta><meta name="dc.Creator" content="Xiaoyu  Ding" /></meta><meta name="dc.Creator" content="Ruirui  Yang" /></meta><meta name="dc.Creator" content="Hui  Hou" /></meta><meta name="dc.Creator" content="Chunpu  Li" /></meta><meta name="dc.Creator" content="Hong  Liu" /></meta><meta name="dc.Creator" content="Kaixian  Chen" /></meta><meta name="dc.Creator" content="Hualiang  Jiang" /></meta><meta name="dc.Creator" content="Mingyue  Zheng" /></meta><meta name="dc.Description" content="The kinome-wide virtual profiling of small molecules with high-dimensional structure–activity data is a challenging task in drug discovery. Here, we present a virtual profiling model against a pane..." /></meta><meta name="Description" content="The kinome-wide virtual profiling of small molecules with high-dimensional structure–activity data is a challenging task in drug discovery. Here, we present a virtual profiling model against a pane..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="July 31, 2019" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.9b00855" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="Copyright © 2019 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.9b00855" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.9b00855" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.9b00855" /></link>
        
    
    

<title>Deep Learning Enhancing Kinome-Wide Polypharmacology Profiling: Model Construction and Experiment Validation | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.9b00855" /></meta><meta property="og:title" content="Deep Learning Enhancing Kinome-Wide Polypharmacology Profiling: Model Construction and Experiment Validation" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-16/acs.jmedchem.9b00855/20200820/images/large/jm9b00855_0010.jpeg" /></meta><meta property="og:description" content="The kinome-wide virtual profiling of small molecules with high-dimensional structure–activity data is a challenging task in drug discovery. Here, we present a virtual profiling model against a panel of 391 kinases based on large-scale bioactivity data and the multitask deep neural network algorithm. The obtained model yields excellent internal prediction capability with an auROC of 0.90 and consistently outperforms conventional single-task models on external tests, especially for kinases with insufficient activity data. Moreover, more rigorous experimental validations including 1410 kinase-compound pairs showed a high-quality average auROC of 0.75 and confirmed many novel predicted “off-target” activities. Given the verified generalizability, the model was further applied to various scenarios for depicting the kinome-wide selectivity and the association with certain diseases. Overall, the computational model enables us to create a comprehensive kinome interaction network for designing novel chemical modulators or drug repositioning and is of practical value for exploring previously less studied kinases." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.9b00855"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.9b00855">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.9b00855&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.9b00855&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.9b00855&amp;href=/doi/10.1021/acs.jmedchem.9b00855" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2020</span><span class="cit-fg-volume">, 63</span><span class="cit-fg-issue">, 16</span><span class="cit-fg-pageRange">, 8723-8737</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/63/16" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.0c00385" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.9b00867" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Deep Learning Enhancing Kinome-Wide Polypharmacology Profiling: Model Construction and Experiment Validation</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Xutong Li</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Xutong Li</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Drug Discovery and Design Center, State Key Laboratory of Drug Research, Shanghai Institute of Materia Medica, Chinese Academy of Sciences, 555 Zuchongzhi Road, Shanghai 201203, China</div><div class="loa-info-affiliations-info">University of Chinese Academy of Sciences, No. 19A Yuquan Road, Beijing 100049, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Xutong++Li">Xutong Li</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Zhaojun Li</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Zhaojun Li</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">School of Information Management, Dezhou University, 566 West University Road, Dezhou 253023, China</div><div class="loa-info-affiliations-info">Drug Discovery and Design Center, State Key Laboratory of Drug Research, Shanghai Institute of Materia Medica, Chinese Academy of Sciences, 555 Zuchongzhi Road, Shanghai 201203, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Zhaojun++Li">Zhaojun Li</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Xiaolong Wu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Xiaolong Wu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Drug Discovery and Design Center, State Key Laboratory of Drug Research, Shanghai Institute of Materia Medica, Chinese Academy of Sciences, 555 Zuchongzhi Road, Shanghai 201203, China</div><div class="loa-info-affiliations-info">School of Pharmacy, East China University of Science and Technology, 130 Meilong Road, Shanghai 200237, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Xiaolong++Wu">Xiaolong Wu</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Zhaoping Xiong</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Zhaoping Xiong</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Drug Discovery and Design Center, State Key Laboratory of Drug Research, Shanghai Institute of Materia Medica, Chinese Academy of Sciences, 555 Zuchongzhi Road, Shanghai 201203, China</div><div class="loa-info-affiliations-info">School of Life Science and Technology, ShanghaiTech University, 393 Huaxiazhong Road, Shanghai 200031, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Zhaoping++Xiong">Zhaoping Xiong</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Tianbiao Yang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Tianbiao Yang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Drug Discovery and Design Center, State Key Laboratory of Drug Research, Shanghai Institute of Materia Medica, Chinese Academy of Sciences, 555 Zuchongzhi Road, Shanghai 201203, China</div><div class="loa-info-affiliations-info">University of Chinese Academy of Sciences, No. 19A Yuquan Road, Beijing 100049, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Tianbiao++Yang">Tianbiao Yang</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Zunyun Fu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Zunyun Fu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Drug Discovery and Design Center, State Key Laboratory of Drug Research, Shanghai Institute of Materia Medica, Chinese Academy of Sciences, 555 Zuchongzhi Road, Shanghai 201203, China</div><div class="loa-info-affiliations-info">University of Chinese Academy of Sciences, No. 19A Yuquan Road, Beijing 100049, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Zunyun++Fu">Zunyun Fu</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Xiaohong Liu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Xiaohong Liu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Drug Discovery and Design Center, State Key Laboratory of Drug Research, Shanghai Institute of Materia Medica, Chinese Academy of Sciences, 555 Zuchongzhi Road, Shanghai 201203, China</div><div class="loa-info-affiliations-info">School of Life Science and Technology, ShanghaiTech University, 393 Huaxiazhong Road, Shanghai 200031, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Xiaohong++Liu">Xiaohong Liu</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Xiaoqin Tan</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Xiaoqin Tan</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Drug Discovery and Design Center, State Key Laboratory of Drug Research, Shanghai Institute of Materia Medica, Chinese Academy of Sciences, 555 Zuchongzhi Road, Shanghai 201203, China</div><div class="loa-info-affiliations-info">University of Chinese Academy of Sciences, No. 19A Yuquan Road, Beijing 100049, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Xiaoqin++Tan">Xiaoqin Tan</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Feisheng Zhong</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Feisheng Zhong</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Drug Discovery and Design Center, State Key Laboratory of Drug Research, Shanghai Institute of Materia Medica, Chinese Academy of Sciences, 555 Zuchongzhi Road, Shanghai 201203, China</div><div class="loa-info-affiliations-info">University of Chinese Academy of Sciences, No. 19A Yuquan Road, Beijing 100049, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Feisheng++Zhong">Feisheng Zhong</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Xiaozhe Wan</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Xiaozhe Wan</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Drug Discovery and Design Center, State Key Laboratory of Drug Research, Shanghai Institute of Materia Medica, Chinese Academy of Sciences, 555 Zuchongzhi Road, Shanghai 201203, China</div><div class="loa-info-affiliations-info">University of Chinese Academy of Sciences, No. 19A Yuquan Road, Beijing 100049, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Xiaozhe++Wan">Xiaozhe Wan</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Dingyan Wang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Dingyan Wang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Drug Discovery and Design Center, State Key Laboratory of Drug Research, Shanghai Institute of Materia Medica, Chinese Academy of Sciences, 555 Zuchongzhi Road, Shanghai 201203, China</div><div class="loa-info-affiliations-info">University of Chinese Academy of Sciences, No. 19A Yuquan Road, Beijing 100049, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Dingyan++Wang">Dingyan Wang</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Xiaoyu Ding</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Xiaoyu Ding</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Drug Discovery and Design Center, State Key Laboratory of Drug Research, Shanghai Institute of Materia Medica, Chinese Academy of Sciences, 555 Zuchongzhi Road, Shanghai 201203, China</div><div class="loa-info-affiliations-info">University of Chinese Academy of Sciences, No. 19A Yuquan Road, Beijing 100049, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Xiaoyu++Ding">Xiaoyu Ding</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Ruirui Yang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Ruirui Yang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Drug Discovery and Design Center, State Key Laboratory of Drug Research, Shanghai Institute of Materia Medica, Chinese Academy of Sciences, 555 Zuchongzhi Road, Shanghai 201203, China</div><div class="loa-info-affiliations-info">School of Life Science and Technology, ShanghaiTech University, 393 Huaxiazhong Road, Shanghai 200031, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Ruirui++Yang">Ruirui Yang</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Hui Hou</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Hui Hou</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Drug Discovery and Design Center, State Key Laboratory of Drug Research, Shanghai Institute of Materia Medica, Chinese Academy of Sciences, 555 Zuchongzhi Road, Shanghai 201203, China</div><div class="loa-info-affiliations-info">School of Pharmacy, Fudan University, 826 Zhangheng Road, Shanghai 201203, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Hui++Hou">Hui Hou</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Chunpu Li</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Chunpu Li</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Drug Discovery and Design Center, State Key Laboratory of Drug Research, Shanghai Institute of Materia Medica, Chinese Academy of Sciences, 555 Zuchongzhi Road, Shanghai 201203, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Chunpu++Li">Chunpu Li</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Hong Liu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Hong Liu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Drug Discovery and Design Center, State Key Laboratory of Drug Research, Shanghai Institute of Materia Medica, Chinese Academy of Sciences, 555 Zuchongzhi Road, Shanghai 201203, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Hong++Liu">Hong Liu</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-0863-7853" title="Orcid link">http://orcid.org/0000-0002-0863-7853</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Kaixian Chen</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Kaixian Chen</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Drug Discovery and Design Center, State Key Laboratory of Drug Research, Shanghai Institute of Materia Medica, Chinese Academy of Sciences, 555 Zuchongzhi Road, Shanghai 201203, China</div><div class="loa-info-affiliations-info">School of Life Science and Technology, ShanghaiTech University, 393 Huaxiazhong Road, Shanghai 200031, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Kaixian++Chen">Kaixian Chen</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Hualiang Jiang</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Hualiang Jiang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Drug Discovery and Design Center, State Key Laboratory of Drug Research, Shanghai Institute of Materia Medica, Chinese Academy of Sciences, 555 Zuchongzhi Road, Shanghai 201203, China</div><div class="loa-info-affiliations-info">School of Life Science and Technology, ShanghaiTech University, 393 Huaxiazhong Road, Shanghai 200031, China</div></div><span class="conrtib-corresp"><strong>*</strong>H.J.: phone, +86-21-50806600-1303; e-mail, <a href="/cdn-cgi/l/email-protection#264e4a4c4f47484166554f4b4b084745084548"><span class="__cf_email__" data-cfemail="eb838781828a858cab98828686c58a88c58885">[email protected]</span></a></span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Hualiang++Jiang">Hualiang Jiang</a></span></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Mingyue Zheng</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Mingyue Zheng</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Drug Discovery and Design Center, State Key Laboratory of Drug Research, Shanghai Institute of Materia Medica, Chinese Academy of Sciences, 555 Zuchongzhi Road, Shanghai 201203, China</div></div><span class="conrtib-corresp"><strong>*</strong>M.Z.: phone, +86-21-50806600-1308; e-mail, <a href="/cdn-cgi/l/email-protection#c3aebab9aba6ada483b0aaaeaeeda2a0eda0ad"><span class="__cf_email__" data-cfemail="4c2135362429222b0c3f252121622d2f622f22">[email protected]</span></a></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Mingyue++Zheng">Mingyue Zheng</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-3323-3092" title="Orcid link">http://orcid.org/0000-0002-3323-3092</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.9b00855&amp;href=/doi/10.1021%2Facs.jmedchem.9b00855" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2020</span></span><span class="cit-volume">, 63</span><span class="cit-issue">, 16</span><span class="cit-pageRange">, 8723–8737</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">July 31, 2019</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>26 May 2019</li><li><span class="item_label"><b>Published</b> online</span>31 July 2019</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 27 August 2020</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.9b00855" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b00855</a></div><div class="article_header-article-copyright"><strong>Copyright © 2019 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D8723%26pageCount%3D15%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DXutong%2BLi%252C%2BZhaojun%2BLi%252C%2BXiaolong%2BWu%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D63%26issueNum%3D16%26contentID%3Dacs.jmedchem.9b00855%26title%3DDeep%2BLearning%2BEnhancing%2BKinome-Wide%2BPolypharmacology%2BProfiling%253A%2BModel%2BConstruction%2Band%2BExperiment%2BValidation%26numPages%3D15%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D8737%26publicationDate%3DAugust%2B2020">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.9b00855"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">3986</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">12</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.9b00855" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Deep Learning Enhancing Kinome-Wide Polypharmacology Profiling: Model Construction and Experiment Validation&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Xutong&quot;,&quot;last_name&quot;:&quot;Li&quot;},{&quot;first_name&quot;:&quot;Zhaojun&quot;,&quot;last_name&quot;:&quot;Li&quot;},{&quot;first_name&quot;:&quot;Xiaolong&quot;,&quot;last_name&quot;:&quot;Wu&quot;},{&quot;first_name&quot;:&quot;Zhaoping&quot;,&quot;last_name&quot;:&quot;Xiong&quot;},{&quot;first_name&quot;:&quot;Tianbiao&quot;,&quot;last_name&quot;:&quot;Yang&quot;},{&quot;first_name&quot;:&quot;Zunyun&quot;,&quot;last_name&quot;:&quot;Fu&quot;},{&quot;first_name&quot;:&quot;Xiaohong&quot;,&quot;last_name&quot;:&quot;Liu&quot;},{&quot;first_name&quot;:&quot;Xiaoqin&quot;,&quot;last_name&quot;:&quot;Tan&quot;},{&quot;first_name&quot;:&quot;Feisheng&quot;,&quot;last_name&quot;:&quot;Zhong&quot;},{&quot;first_name&quot;:&quot;Xiaozhe&quot;,&quot;last_name&quot;:&quot;Wan&quot;},{&quot;first_name&quot;:&quot;Dingyan&quot;,&quot;last_name&quot;:&quot;Wang&quot;},{&quot;first_name&quot;:&quot;Xiaoyu&quot;,&quot;last_name&quot;:&quot;Ding&quot;},{&quot;first_name&quot;:&quot;Ruirui&quot;,&quot;last_name&quot;:&quot;Yang&quot;},{&quot;first_name&quot;:&quot;Hui&quot;,&quot;last_name&quot;:&quot;Hou&quot;},{&quot;first_name&quot;:&quot;Chunpu&quot;,&quot;last_name&quot;:&quot;Li&quot;},{&quot;first_name&quot;:&quot;Hong&quot;,&quot;last_name&quot;:&quot;Liu&quot;},{&quot;first_name&quot;:&quot;Kaixian&quot;,&quot;last_name&quot;:&quot;Chen&quot;},{&quot;first_name&quot;:&quot;Hualiang&quot;,&quot;last_name&quot;:&quot;Jiang&quot;},{&quot;first_name&quot;:&quot;Mingyue&quot;,&quot;last_name&quot;:&quot;Zheng&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2019&quot;,&quot;month&quot;:&quot;07&quot;,&quot;day&quot;:&quot;31&quot;,&quot;issue&quot;:&quot;16&quot;,&quot;volume&quot;:&quot;63&quot;,&quot;pages&quot;:&quot;8723-8737&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.9b00855&quot;},&quot;abstract&quot;:&quot;The kinome-wide virtual profiling of small molecules with high-dimensional structure–activity data is a challenging task in drug discovery. Here, we present a virtual profiling model against a panel of 391 kinases based on large-scale bioactivity data and the multitask deep neural network algorithm. The obtained model yields excellent internal prediction capability with an auROC of 0.90 and consistently outperforms conventional single-task models on external tests, especially for kinases with insufficient activity data. Moreover, more rigorous experimental validations including 1410 kinase-compound pairs showed a high-quality average auROC of 0.75 and confirmed many novel predicted “off-target” activities. Given the verified generalizability, the model was further applied to various scenarios for depicting the kinome-wide selectivity and the association with certain diseases. Overall, the computational model enables us to create a comprehensive kinome interaction network for designing novel chemical modulator&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b00855&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b00855" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b00855&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b00855" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b00855&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b00855" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.9b00855&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b00855&amp;href=/doi/10.1021/acs.jmedchem.9b00855" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.9b00855" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.9b00855" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (6 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b00855&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.9b00855%26sid%3Dliteratum%253Aachs%26pmid%3D31364850%26genre%3Darticle%26aulast%3DLi%26date%3D2020%26atitle%3DDeep%2BLearning%2BEnhancing%2BKinome-Wide%2BPolypharmacology%2BProfiling%253A%2BModel%2BConstruction%2Band%2BExperiment%2BValidation%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D63%26issue%3D16%26spage%3D8723%26epage%3D8737%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (5)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=291264" title="Kinase inhibitors">Kinase inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=291711" title="Selectivity">Selectivity</a>,</li><li><a href="/action/doSearch?ConceptID=291321" title="Bioactivity">Bioactivity</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/63/16" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-16/jmcmar.2020.63.issue-16/20200827/jmcmar.2020.63.issue-16.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-16/acs.jmedchem.9b00855/20200820/images/medium/jm9b00855_0010.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-16/acs.jmedchem.9b00855/20200820/images/large/jm9b00855_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00855&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">The kinome-wide virtual profiling of small molecules with high-dimensional structure–activity data is a challenging task in drug discovery. Here, we present a virtual profiling model against a panel of 391 kinases based on large-scale bioactivity data and the multitask deep neural network algorithm. The obtained model yields excellent internal prediction capability with an auROC of 0.90 and consistently outperforms conventional single-task models on external tests, especially for kinases with insufficient activity data. Moreover, more rigorous experimental validations including 1410 kinase-compound pairs showed a high-quality average auROC of 0.75 and confirmed many novel predicted “off-target” activities. Given the verified generalizability, the model was further applied to various scenarios for depicting the kinome-wide selectivity and the association with certain diseases. Overall, the computational model enables us to create a comprehensive kinome interaction network for designing novel chemical modulators or drug repositioning and is of practical value for exploring previously less studied kinases.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="specialIssueNotice" class="extra-info-sec articleNote"><h4>SPECIAL ISSUE</h4><p class="last">This article is part of the <a href="/toc/jmcmar/63/16"><issue-title>Artificial Intelligence in Drug Discovery</issue-title></a> special issue.</p></div><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_47567" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_47567" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">The protein kinase family is one of the largest enzyme families. The human kinome comprises more than 500 kinases, constituting approximately 1.7% of all human genes.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> Dysregulation of protein kinases plays causal roles in numerous human diseases, including cancers, inflammatory diseases, central nervous system disorders, cardiovascular diseases, and complications of diabetes.<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a> By December 2018, 48 small kinase inhibitors were approved by the U.S. FDA, approximately half of which were approved in the past 5 years. However, despite the success of the kinase inhibitor drugs, some deficiencies cannot be ignored. On the one hand, only a small number of human kinases (approximately 80) have been successfully targeted by these drugs, and many kinase inhibitor drugs are used against the same targets in oncology.<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> More than 100 kinases (∼25%) have completely unknown functions, and approximately 50% are largely uncharacterized, with little indication of how these targets influence the major signaling pathways.<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> Therefore, there are still many “untargeted” kinases and relevant diseases that require further investigation. On the other hand, many kinase inhibitor drugs have promiscuous profiles due to the conservation of the ATP binding site in the human kinome. As demonstrated by a recent study, even for kinase drugs in clinical use, the target space or bioactivities are still surprisingly poorly characterized.<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> Therefore, thorough characterization of the kinase spectrum of inhibitor drugs is important to explain undesired adverse effects and to enable drug repurposing, and the discovery of potent inhibitors with reasonable selectivity and polypharmacological profiles has become a very promising but challenging direction in novel drug development process.<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a></div><div class="NLM_p">A step toward accelerating the kinase drug discovery process is to quickly identify whether a compound interacts with a kinase. Recent advances in high-throughput screening technologies have enabled the bioactivity profiling of hundreds of compounds against a panel of protein kinases,<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> but the high cost and laborious process of chemical synthesis and biological characterization hinder its application in extensive chemical spaces. Many in silico modeling approaches have been developed to predict kinase inhibitory activity for large-scale compound libraries. Compared to traditional drug design methods (e.g., docking and standard QSAR), machine-learning-based models such as naive Bayesian (NB),<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> k-nearest neighbors (KNN),<a onclick="showRef(event, 'ref9 ref10'); return false;" href="javascript:void(0);" class="ref ref9 ref10">(9,10)</a> random forest (RF),<a onclick="showRef(event, 'ref11 ref12 ref13'); return false;" href="javascript:void(0);" class="ref ref11 ref12 ref13">(11−13)</a> support vector machine (SVM),<a onclick="showRef(event, 'ref8 ref9 ref10 ref14'); return false;" href="javascript:void(0);" class="ref ref8 ref9 ref10 ref14">(8−10,14)</a> and deep neural network (DNN)<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> have been established to predict a wider range of biological activities for a compound by employing high-dimensional data sets. Generally, these models have been separately trained with individual data sets relating to different tasks and often are more flexible and have high variance. For example, given a learning task with insufficient data, these models always show unsatisfactory predictive power and a tendency toward overfitting. In fact, standard machine learning algorithms are capable of learning meaningful chemical information from hundreds of compounds and become even more effective given additional data.<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> However, even a set of 100 compounds is often an unavailable resource for building a machine learning model for some less studied or uncharacterized kinases.</div><div class="NLM_p last">In this study, we established a multitask deep neural network (MTDNN) classification model to predict the bioactivities of small molecules at a kinome-wide level, including kinases with both large and small amounts of available data. Previous studies of MTDNN applied to drug discovery suggested that MTDNN can obtain significantly better predictive accuracies than single-task methods for problems with multiple related tasks<a onclick="showRef(event, 'ref17 ref18 ref19 ref20 ref21'); return false;" href="javascript:void(0);" class="ref ref17 ref18 ref19 ref20 ref21">(17−21)</a> due to the generalization ability and transfer learning effect of DNNs. These features make MTDNN an appropriate solution for bioactivity prediction against a spectrum of kinases with high homology to each other. Rodríguez-Perez et al. have showed that MTDNN resulted in a performance boost for other single-task and multitask machine learning models in the prediction of highly potent inhibitors of 103 human kinases.<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> These results are illuminating, but a wider kinase panel and more rigorous experimental validations are in high demand. Here, the MTDNN model has been trained with over 140 000 bioactivity data points for 391 kinases, where one task corresponds to bioactivity prediction against a specific type of kinase. Extensive computational (on four external data sets with significant sizes) and experimental validations (on 5 compounds against 282 kinases involving 1410 kinase–compound pairs) have been performed, and the MTDNN model consistently outperforms conventional single-task models in terms of auROC. In particular, it facilitates network information sharing across different kinases and compensates for the limited bioactivity data associated with any specific kinase, which enables reliable and comprehensive profiling of kinome-wide activity and reasonable estimations of overall and group-specific selectivity. We envisage that the MTDNN model could be used to quickly establish a comprehensive kinome interaction network for designing novel chemical modulators and for exploring previously less studied or untargeted kinases.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_65342" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_65342" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> Optimized MTDNN Model</h3><div class="NLM_p">The predictive performance on the testing data set of all the MTDNN models is summarized in <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>. The following points can be noted: (1) The influence of batch size in our search space is slight. (2) Models with very simple architecture show moderate performance, whereas those with complex structures are very sensitive to different learning rates. In general, a more complex architecture means a more heavily parametrized model with a higher capacity, and such a model operates in a space with a larger dimension and has a more complex error surface than a thinner one. Thus, in such a complex error regime, the convergence process may either easily deviate from meaningful locations with a large learning rate or become trapped in local minima with a smaller learning rate. (3) Models with less favorable learning rate values always yield weak predictive power. A large learning rate might cause drastic updates leading to divergent behaviors, while a model with a small learning rate might require too many updates to achieve convergence. Here, models with moderate depth and fine-tuned learning rates can show satisfactory performance on the testing data set, with auROCs higher than 0.89. With the layer size of [1024,391], the learning rate of 5 × 10<sup>–5</sup>, and the batch size of 128 on our internal test set, the model with the best performance has an auROC over 0.90, an F1-score of 0.74, and a BA (balanced accuracy) of 0.83. This model was chosen for further external evaluations.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-16/acs.jmedchem.9b00855/20200820/images/medium/jm9b00855_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-16/acs.jmedchem.9b00855/20200820/images/large/jm9b00855_0001.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Model performance results with different hyperparameters. The heat maps are colored according to the auROC of the testing data set, which is directly marked in each block. A darker color corresponds to better performance.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-16/acs.jmedchem.9b00855/20200820/images/large/jm9b00855_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00855&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5"> Evaluation on External Data Sets</h3><div class="NLM_p">Despite the satisfactory predictive power of the internal testing data set, it is valuable to investigate model performance on data sets that have different distributions from that of the modeled data set to evaluate the generalization capability of the neural network. Thus, we prepared four different external data sets with diverse experimental methods and value types to evaluate the model.</div><div class="NLM_p">The first data set was published by Davis et al.,<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> who tested 72 known inhibitors against a panel of 442 kinase assays, resulting in a total of 9424 <i>K</i><sub>d</sub> values for all potential protein–ligand pairs. The second data set was provided by Anastassiadis et al.<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> and reported the inhibitory percentage inhibition values of 178 compounds tested against 300 kinases at 0.5 μM. The other two data sets are PKIS1<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> and PKIS2,<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> also referred to as Published Kinase Inhibitor Set 1 and Published Kinase Inhibitor Set 2, respectively. PKIS1 is a collection of 367 kinase inhibitors representing 31 diverse chemotypes, and their inhibitory percentage values evaluated against 232 kinases at 1 μM were collected. PKIS2 is composed of 645 small-molecule inhibitors representing 86 diverse chemotypes, which were all profiled at a concentration of 1 μM against a broader panel of 392 kinases. For the data set of Anastassiadis et al., the activity threshold is set by converting the single point activity with the equation previously defined.<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> For data sets PKIS1 and PKIS2, an inhibition rate over 50% at 1 μM was defined as signifying activity.<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> Note that all compound–kinase pairs included in our training data set were removed from these external data sets before further evaluation.</div><div class="NLM_p">As summarized in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>, the classification performance results on these external data sets are lower than those from the testing data set. One of the potential reasons for this performance degradation could be the chemical distribution deviations. We first investigated the performance for the compounds dissimilar to the training samples by a 5-fold cluster cross-validation<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). Clearly, the predictive performance appreciably decreased. To provide an overview of the different distributions between the training data set and four external data sets, we used t-SNE<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> to visualize high-dimensional feature vectors by projecting each sample into three-dimensional space. As shown in <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>, the compounds in the training set were compared with the samples in each external data set. The Davis set is relatively small, and all of the samples are distributed within the chemical space of the training data set. For the Anastassiadis set and PKIS1, the chemical space of most of the samples overlaps with that of the training data set. In contrast, for PKIS2, a number of samples are scattered outside the training set space, suggesting a distinct data distribution. As a result, MTDNN achieves the worst performance on PKIS2 of which the chemical space shows apparent deviation from the training set.</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-16/acs.jmedchem.9b00855/20200820/images/medium/jm9b00855_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-16/acs.jmedchem.9b00855/20200820/images/large/jm9b00855_0002.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Visualizing data sets using t-SNE. The ECFP4 values of the compounds were used as input. The distribution of compounds from the training data set (gray scatters) is compared to that from (A) the Davis data set, (B) the Anastassiadis data set, (C) PKIS1, and (D) PKIS2 (blue scatters).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-16/acs.jmedchem.9b00855/20200820/images/large/jm9b00855_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00855&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Model Performance on Internal Testing, Cluster Cross-Validation, and External Validations</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="char" char="." /></col><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center" char=".">testing data set</th><th class="colsep0 rowsep0" align="center">cluster cross-validation</th><th class="colsep0 rowsep0" align="center" char=".">Davis</th><th class="colsep0 rowsep0" align="center" char=".">Anastassiadis</th><th class="colsep0 rowsep0" align="center" char=".">PKIS1</th><th class="colsep0 rowsep0" align="center" char=".">PKIS2</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">auROC</td><td class="colsep0 rowsep0" align="char" char=".">0.90</td><td class="colsep0 rowsep0" align="left">0.82 ± 0.02</td><td class="colsep0 rowsep0" align="char" char=".">0.78</td><td class="colsep0 rowsep0" align="char" char=".">0.72</td><td class="colsep0 rowsep0" align="char" char=".">0.79</td><td class="colsep0 rowsep0" align="char" char=".">0.69</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Recall</td><td class="colsep0 rowsep0" align="char" char=".">0.78</td><td class="colsep0 rowsep0" align="left">0.63 ± 0.05</td><td class="colsep0 rowsep0" align="char" char=".">0.58</td><td class="colsep0 rowsep0" align="char" char=".">0.44</td><td class="colsep0 rowsep0" align="char" char=".">0.62</td><td class="colsep0 rowsep0" align="char" char=".">0.43</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Precision</td><td class="colsep0 rowsep0" align="char" char=".">0.70</td><td class="colsep0 rowsep0" align="left">0.66 ± 0.09</td><td class="colsep0 rowsep0" align="char" char=".">0.39</td><td class="colsep0 rowsep0" align="char" char=".">0.38</td><td class="colsep0 rowsep0" align="char" char=".">0.15</td><td class="colsep0 rowsep0" align="char" char=".">0.18</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">F1-score</td><td class="colsep0 rowsep0" align="char" char=".">0.74</td><td class="colsep0 rowsep0" align="left">0.64 ± 0.07</td><td class="colsep0 rowsep0" align="char" char=".">0.47</td><td class="colsep0 rowsep0" align="char" char=".">0.41</td><td class="colsep0 rowsep0" align="char" char=".">0.24</td><td class="colsep0 rowsep0" align="char" char=".">0.26</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">BA</td><td class="colsep0 rowsep0" align="char" char=".">0.83</td><td class="colsep0 rowsep0" align="left">0.75 ± 0.02</td><td class="colsep0 rowsep0" align="char" char=".">0.71</td><td class="colsep0 rowsep0" align="char" char=".">0.66</td><td class="colsep0 rowsep0" align="char" char=".">0.72</td><td class="colsep0 rowsep0" align="char" char=".">0.64</td></tr></tbody></table></div></div><div class="NLM_p">One the other hand, the concordance of data from different sources could also limit the predictive performance. As highlighted by Sutherland et al., there is only modest concordance between assay panels from different sources, including the Davis set, the Anastassiadis set, and part of our training set, i.e., the Metz set.<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> The discrepancies between assays will certainly affect the predictive performance of the model on these external data sets, and it is therefore of interest to quantitatively analyze the data consistency between our training set and different external data sources. Toward this aim, the removed overlapping data between the training and external sources were used to perform diagnostic tests with auROCs, where a higher auROC value suggests a more consistent result between the different data sources. Specifically, given a molecule with multiple experimental data points relating to the same target, a TP (true positive) is counted when both values in the training set and external sources are “active”. The auROCs of the Davis set, the Anastassiadis set, PKIS1, and PKIS2 with respect to our training set are 0.91, 0.82, 0.84, and 0.86, respectively. These results suggest that despite the uncertainty in different sources and forms of activity measurements, significant consistency can be observed between our training and external data. Among them, the Davis set with the equilibrium dissociation constant data shares the highest concordance with our training data. The Anastassiadis set has the lowest concordance with the training set with an auROC of 0.82, and the MTDNN yielded a relatively low auROC of 0.72 even though it shows a similar distribution of chemical space to that of the training set.</div><div class="NLM_p">Overall, the MTDNN still shows impressive performance on these external data sets. We supposed that by expanding the scale and increasing the diversity of the chemotypes of the training data set, the generalization capability of the model could be further improved. To verify this point, with a series of increasing scales of modeling data, we built two series of models using random split and 5-fold cluster cross-validation, respectively. The size of the training data set was increased from 500 compounds to 25 000 compounds in increments of 2500 compounds. As shown in <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>, for the random split validation, there is a continuous uptrend of the auROC curve due to the rising probability of finding similar compounds in the training set. While for the cluster cross-validation, auROC curve stops to increase after modeling compounds reach 5500. Nonetheless, the performance of cluster cross-validation still demonstrated that the model is of decent predictive power for compounds with unseen structures. Moreover, we may find that the auROCs for external data sets of both series of models rise steadily with the increasing size of training data set, which supports the argument that the expansion of the chemical space of training set will improve model generalization capability.</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-16/acs.jmedchem.9b00855/20200820/images/medium/jm9b00855_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-16/acs.jmedchem.9b00855/20200820/images/large/jm9b00855_0003.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. AuROC on internal validation and external testing data sets of models with increasing numbers of compounds in the training data set: (A) random split validation; (B) 5-fold cluster cross-validation.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-16/acs.jmedchem.9b00855/20200820/images/large/jm9b00855_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00855&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i7" class="anchor-spacer"></div><h3 class="article-section__title" id="_i7"> Comparison with Previously Reported Models</h3><div class="NLM_p">In recent years, several models have been proposed for predicting the kinome-wide polypharmacological effect of small molecules.<a onclick="showRef(event, 'ref8 ref9 ref10 ref11 ref12 ref13 ref31 ref32'); return false;" href="javascript:void(0);" class="ref ref8 ref9 ref10 ref11 ref12 ref13 ref31 ref32">(8−13,31,32)</a> Niijima et al. developed a deconvolution approach to dissecting the Kinase SARfari database by which kinase–inhibitor pairs are represented by residues and fragments and built kinome-wide activity classification models with dual-component naive Bayes (DCNB) and dual-component SVMs (DCSVMs).<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> DCSVM achieved good performance on an internal validation set, showing an unbalanced accuracy over 0.85, but it did not achieve consistently good performance on external sets. Merget et al. reported ligand-based activity classification models for over 280 kinases by RF methods, based on an extensive data set combining both proprietary and open bioactivity data.<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> The models yielded good prediction results with an average auROC of 0.76, and high quality (auROC > 0.7) was achieved for over 200 kinases. Janssen et al. presented Drug Discovery Maps (DDM), a t-SNE-based model that can map chemical and target space and predict the activities of novel kinase inhibitors across the kinome.<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> Using DDM, they discovered new inhibitors for FMS-like tyrosine kinase 3 (FLT3).</div><div class="NLM_p">To verify the generalizability and transfer learning effect of our MTDNN model, a parallel comparison with previously reported models was carried out. Here, we chose the method of Merget et al. as a reference because it requires the same form of input and because its source code for deriving prediction models is available. On the basis of the same sets of training data used for MTDNN, we built a total of 391 single-task random forest models (RFs) with their provided code. First, we evaluated the RFs on our internal testing data set, among which 226 kinases had auROCs higher than 0.7 and 134 kinases had auROCs higher than 0.8. These results agree well with the high-quality performance (i.e., auROC > 0.7 was achieved for ∼200 kinases, of which 118 prediction models had auROC ≥ 0.8) reported by Merget et al., confirming that these RFs are correctly established. In comparison with these high-quality RFs, we may notice that MTDNN further improved the predictive capability. As shown in <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>A, the MTDNN showed consistently higher average auROC values than those of the RFs on all four external data sets and achieved more high-quality (auROC > 0.7) predictions than RFs.</div><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-16/acs.jmedchem.9b00855/20200820/images/medium/jm9b00855_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-16/acs.jmedchem.9b00855/20200820/images/large/jm9b00855_0004.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Prediction performance comparison between MTDNN and RF of Merget et al. (A) Violin plots of auROCs for kinases, grouped by underlying external data sets and colored by method (RF in blue and MTDNN in red). The white dots are the average auROCs. The upper and lower end points of the black segments are the first and the third quartile, respectively. The <i>p</i>-values of the <i>t</i> tests are shown above the violin. (B) Bar plots of model performance on tasks with different amounts of bioactivity data in external data sets. A bar indicates the average auROC of tasks with the number of bioactivity data points within the underlying range: (∗) 0.01 < <i>p</i> < 0.05; (∗∗) 0.001 < <i>p</i> < 0.01; (∗∗∗) <i>p</i> < 0.001.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-16/acs.jmedchem.9b00855/20200820/images/large/jm9b00855_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00855&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">For all individual tasks, we further investigated the relation between the size of the training data sets and their auROCs on the external data sets (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00855/suppl_file/jm9b00855_si_002.xlsx" class="ext-link">Table S2</a>). As shown in <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>B, MTDNN demonstrated decent performance even when the number of training bioactivity data points for the kinases was quite small. In contrast, for these small-data tasks, RF can give the results only barely better than random prediction (auROC of ∼0.5). Given a kinase with fewer than 100 training data points, the performance of MTDNN significantly exceeded that of RF. As the amount of training data increases, the gap between MTDNN and RF gradually narrows but remains. These results corroborate the transfer learning effect of MTDNN, which can take advantage of tasks with larger amounts of training data to improve the predictive performance of tasks with smaller training data. Obviously, the predictive capability for kinases without sufficient training data is an essential feature of our model, resulting in its practical value for developing a selective inhibitor for the previously less studied or untargeted kinases.</div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i8" class="anchor-spacer"></div><h3 class="article-section__title" id="_i8"> Experimental Validation</h3><div class="NLM_p">In addition to retrospective model validation with previously reported data, we also performed more rigorous prospective experimental validation to investigate the practical benefits of the model for drug discovery or repurposing applications. Five kinase inhibitors with unknown activity profiling data and diverse structures were used for this validation, including the clinically approved drugs BAY73-4506 and K-115 and the investigational inhibitors NVP-BHG712, E31, and DC381016 (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>). BAY73-4506, K-115, and NVP-BHG712 were purchased from Selleck, and E31 and DC381016 were provided by our co-workers.<a onclick="showRef(event, 'ref33 ref34'); return false;" href="javascript:void(0);" class="ref ref33 ref34">(33,34)</a> These five kinase inhibitors have different primary targets under investigation, while their complete kinase spectra have not been reported by the time we performed the experimental validation. All commercially available compounds were used without any further purification. Reversed-phase HPLC analyses were performed on an Agilent 1100 HPLC system with a DAD detector (area normalization). Purity of all compounds for biological evaluation was confirmed by HPLC to be >95%.</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Details for Experimental Compounds<a onclick="showRef(event, 'ref51'); return false;" href="javascript:void(0);" class="ref ref51">()</a><a onclick="showRef(event, 'ref52'); return false;" href="javascript:void(0);" class="ref ref52">()</a><a onclick="showRef(event, 'ref53'); return false;" href="javascript:void(0);" class="ref ref53">()</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-16/acs.jmedchem.9b00855/20200820/images/medium/jm9b00855_0009.gif" alt="" id="gr9" /></img><div></div></div><div class="NLM_p"><a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a> also summarizes the predicted kinase activity profile of the five molecules. To validate the prediction, the commercial KinaseProfiler Service (Eurofins Scientific, Inc.) was utilized to evaluate the experimental activity of each molecule against a panel of 405 kinases, among which 282 fall in our MTDNN capacity panel (391 kinases). Filter-binding radiometric kinase activity assays were performed at a concentration of 1 μM, with the active threshold defined as 50% (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00855/suppl_file/jm9b00855_si_003.xlsx" class="ext-link">Table S3</a>). It is gratifying to note that for the 1410 kinase-compound pairs in our MTDNN capacity panel, the model produced high-quality predictions that were generally in agreement with the experimental data with an average auROC of 0.75 (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>). Moreover, in addition to the activities reported previously, many novel “off-target” activities have been successfully predicted by the model and confirmed by subsequent experiments at lower compound concentrations.</div><figure id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-16/acs.jmedchem.9b00855/20200820/images/medium/jm9b00855_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-16/acs.jmedchem.9b00855/20200820/images/large/jm9b00855_0005.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Kinome maps of the predicted and experimental bioactivities. Kinases colored in red are experimental % activity of <50% at 1 μM or predicted active probability of >0.5 (left). Bar plots show predicted and experimental group selectivity. When a group has an odds ratio significantly greater than 1, the bar is colored red (right): (∗) 0.01 < <i>p</i> < 0.05; (∗∗) 0.001 < <i>p</i> < 0.01; (∗∗∗) <i>p</i> < 0.001.  Kinome maps were generated using the TREEspot software tool and reproduced with permission from KINOMEscan, a division of DiscoveRx Corporation, © DiscoveRx Corporation 2010.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-16/acs.jmedchem.9b00855/20200820/images/large/jm9b00855_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00855&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">BAY73-4506 (regorafenib) is an orally available antineoplastic agent launched for the treatment of colorectal cancer,<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> gastrointestinal stromal tumor (GIST),<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> and hepatocellular carcinoma.<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> On the one hand, many previously published inhibitory activities, e.g., its known potent inhibitory effects against VEGFR1, VEGFR2, VEGFR3, PDGFRβ, Kit, RET, and c-Raf,<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> have been accurately identified by our model. On the other hand, there are inhibitory activities for several other kinases that have not been reported (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>). Examples include the following: (1) Targeting p38α, ZAK, and TAO1, which are involved in p38 MAPK signal transduction pathways. As a member of the p38 MAPK family that plays a critical role in cancer cell biology, p38α is an investigational target for the treatment of several types of cancer, including colorectal cancer and cardiovascular disorders. ZAK and TAO1 have been shown to trigger p38 MAPK activation.<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> (2) Targeting TrkB and TrkC, which are important therapeutic targets for neuroblastoma, non-small-cell lung cancer (NSCLC) and colorectal cancer, for patients harboring alterations in TRK expression or activity.<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> (3) Targeting DDR1, which has recently been identified as a new therapeutic target for colorectal cancer<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a> and could also be a biomarker of epithelial ovarian cancer and a prognostic marker for NSCLC patients.<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a> (4) Targeting LOK, which is a previously untargeted kinase with little indication of its function.<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> Overall, the successful prediction of kinase “off-targets” of BAY73-4506 provides additional explanations for its anticancer mechanisms of action and informs the search for new clinical applications. Moreover, identifying BAY73-4506 as a potent inhibitor of LOK may help to accelerate the related functional study of this untargeted kinase.</div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. Predicted Active Probability and Experimental % Activity of BAY73-4506 against Kinases</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center" char="."> </th><th class="rowsep1 colsep0" colspan="3" align="center" char=".">% activity<a class="ref internalNav" href="#t3fn1" aria-label="a">a</a></th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">UniProt code</th><th class="colsep0 rowsep0" align="center">kinase<a class="ref internalNav" href="#t3fn2" aria-label="b">b</a></th><th class="colsep0 rowsep0" align="center">family</th><th class="colsep0 rowsep0" align="center" char=".">predicted probability</th><th class="colsep0 rowsep0" align="center" char=".">at 1 μM</th><th class="colsep0 rowsep0" align="center" char=".">at 0.1 μM</th><th class="colsep0 rowsep0" align="center" char=".">at 0.01 μM</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><a href="https://P10721" class="ext-link">P10721</a></td><td class="colsep0 rowsep0" align="left"><b>Kit</b></td><td class="colsep0 rowsep0" align="left">TK</td><td class="colsep0 rowsep0" align="char" char=".">0.7795</td><td class="colsep0 rowsep0" align="char" char=".">–1</td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="."> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><a href="https://P35916" class="ext-link">P35916</a></td><td class="colsep0 rowsep0" align="left"><b>VEGFR3</b></td><td class="colsep0 rowsep0" align="left">TK</td><td class="colsep0 rowsep0" align="char" char=".">0.9825</td><td class="colsep0 rowsep0" align="char" char=".">0</td><td class="colsep0 rowsep0" align="char" char=".">1</td><td class="colsep0 rowsep0" align="char" char=".">38</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><a href="https://P07949" class="ext-link">P07949</a></td><td class="colsep0 rowsep0" align="left"><b>Ret</b></td><td class="colsep0 rowsep0" align="left">TK</td><td class="colsep0 rowsep0" align="char" char=".">0.9928</td><td class="colsep0 rowsep0" align="char" char=".">0</td><td class="colsep0 rowsep0" align="char" char=".">6</td><td class="colsep0 rowsep0" align="char" char=".">50</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><a href="https://P17948" class="ext-link">P17948</a></td><td class="colsep0 rowsep0" align="left"><b>VEGFR1</b></td><td class="colsep0 rowsep0" align="left">TK</td><td class="colsep0 rowsep0" align="char" char=".">0.9885</td><td class="colsep0 rowsep0" align="char" char=".">3</td><td class="colsep0 rowsep0" align="char" char=".">3</td><td class="colsep0 rowsep0" align="char" char=".">43</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><a href="https://P04049" class="ext-link">P04049</a></td><td class="colsep0 rowsep0" align="left"><b>Raf1</b></td><td class="colsep0 rowsep0" align="left">TKL</td><td class="colsep0 rowsep0" align="char" char=".">0.8489</td><td class="colsep0 rowsep0" align="char" char=".">8</td><td class="colsep0 rowsep0" align="char" char=".">57</td><td class="colsep0 rowsep0" align="char" char=".">93</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><a href="https://P35968" class="ext-link">P35968</a></td><td class="colsep0 rowsep0" align="left"><b>VEGFR2</b></td><td class="colsep0 rowsep0" align="left">TK</td><td class="colsep0 rowsep0" align="char" char=".">0.9950</td><td class="colsep0 rowsep0" align="char" char=".">17</td><td class="colsep0 rowsep0" align="char" char=".">19</td><td class="colsep0 rowsep0" align="char" char=".">68</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><a href="https://P09619" class="ext-link">P09619</a></td><td class="colsep0 rowsep0" align="left"><b>PDGFRβ</b></td><td class="colsep0 rowsep0" align="left">TK</td><td class="colsep0 rowsep0" align="char" char=".">0.9713</td><td class="colsep0 rowsep0" align="char" char=".">55</td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="."> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><a href="https://Q16539" class="ext-link">Q16539</a></td><td class="colsep0 rowsep0" align="left">p38α</td><td class="colsep0 rowsep0" align="left">CMGC</td><td class="colsep0 rowsep0" align="char" char=".">0.5766</td><td class="colsep0 rowsep0" align="char" char=".">–6</td><td class="colsep0 rowsep0" align="char" char=".">52</td><td class="colsep0 rowsep0" align="char" char=".">100</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><a href="https://P00520" class="ext-link">P00520</a></td><td class="colsep0 rowsep0" align="left">Abl(m)</td><td class="colsep0 rowsep0" align="left">TK</td><td class="colsep0 rowsep0" align="char" char=".">0.9867</td><td class="colsep0 rowsep0" align="char" char=".">–1</td><td class="colsep0 rowsep0" align="char" char=".">–7</td><td class="colsep0 rowsep0" align="char" char=".">22</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><a href="https://Q9NYL2" class="ext-link">Q9NYL2</a></td><td class="colsep0 rowsep0" align="left">ZAK</td><td class="colsep0 rowsep0" align="left">TKL</td><td class="colsep0 rowsep0" align="char" char=".">0.9506</td><td class="colsep0 rowsep0" align="char" char=".">–1</td><td class="colsep0 rowsep0" align="char" char=".">2</td><td class="colsep0 rowsep0" align="char" char=".">27</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><a href="https://Q8NE63" class="ext-link">Q8NE63</a></td><td class="colsep0 rowsep0" align="left">HIPK4</td><td class="colsep0 rowsep0" align="left">CMGC</td><td class="colsep0 rowsep0" align="char" char=".">0.9961</td><td class="colsep0 rowsep0" align="char" char=".">–1</td><td class="colsep0 rowsep0" align="char" char=".">3</td><td class="colsep0 rowsep0" align="char" char=".">31</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><a href="https://Q9Y4K4" class="ext-link">Q9Y4K4</a></td><td class="colsep0 rowsep0" align="left">MAP4K5</td><td class="colsep0 rowsep0" align="left">STE</td><td class="colsep0 rowsep0" align="char" char=".">0.5848</td><td class="colsep0 rowsep0" align="char" char=".">0</td><td class="colsep0 rowsep0" align="char" char=".">37</td><td class="colsep0 rowsep0" align="char" char=".">104</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><a href="https://P16277" class="ext-link">P16277</a></td><td class="colsep0 rowsep0" align="left">Blk(m)</td><td class="colsep0 rowsep0" align="left">TK</td><td class="colsep0 rowsep0" align="char" char=".">0.5559</td><td class="colsep0 rowsep0" align="char" char=".">1</td><td class="colsep0 rowsep0" align="char" char=".">45</td><td class="colsep0 rowsep0" align="char" char=".">52</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><a href="https://Q16620" class="ext-link">Q16620</a></td><td class="colsep0 rowsep0" align="left">TrkB</td><td class="colsep0 rowsep0" align="left">TK</td><td class="colsep0 rowsep0" align="char" char=".">0.9862</td><td class="colsep0 rowsep0" align="char" char=".">1</td><td class="colsep0 rowsep0" align="char" char=".">53</td><td class="colsep0 rowsep0" align="char" char=".">100</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><a href="https://Q16288" class="ext-link">Q16288</a></td><td class="colsep0 rowsep0" align="left">TrkC</td><td class="colsep0 rowsep0" align="left">TK</td><td class="colsep0 rowsep0" align="char" char=".">0.9487</td><td class="colsep0 rowsep0" align="char" char=".">1</td><td class="colsep0 rowsep0" align="char" char=".">17</td><td class="colsep0 rowsep0" align="char" char=".">54</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><a href="https://P07948" class="ext-link">P07948</a></td><td class="colsep0 rowsep0" align="left">Lyn</td><td class="colsep0 rowsep0" align="left">TK</td><td class="colsep0 rowsep0" align="char" char=".">0.9373</td><td class="colsep0 rowsep0" align="char" char=".">1</td><td class="colsep0 rowsep0" align="char" char=".">21</td><td class="colsep0 rowsep0" align="char" char=".">32</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><a href="https://P42685" class="ext-link">P42685</a></td><td class="colsep0 rowsep0" align="left">PTK5</td><td class="colsep0 rowsep0" align="left">TK</td><td class="colsep0 rowsep0" align="char" char=".">0.9961</td><td class="colsep0 rowsep0" align="char" char=".">3</td><td class="colsep0 rowsep0" align="char" char=".">18</td><td class="colsep0 rowsep0" align="char" char=".">64</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><a href="https://P29322" class="ext-link">P29322</a></td><td class="colsep0 rowsep0" align="left">EphA8</td><td class="colsep0 rowsep0" align="left">TK</td><td class="colsep0 rowsep0" align="char" char=".">0.9835</td><td class="colsep0 rowsep0" align="char" char=".">5</td><td class="colsep0 rowsep0" align="char" char=".">44</td><td class="colsep0 rowsep0" align="char" char=".">85</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><a href="https://P29317" class="ext-link">P29317</a></td><td class="colsep0 rowsep0" align="left">EphA2</td><td class="colsep0 rowsep0" align="left">TK</td><td class="colsep0 rowsep0" align="char" char=".">0.9902</td><td class="colsep0 rowsep0" align="char" char=".">5</td><td class="colsep0 rowsep0" align="char" char=".">58</td><td class="colsep0 rowsep0" align="char" char=".">90</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><a href="https://P15056" class="ext-link">P15056</a></td><td class="colsep0 rowsep0" align="left">B-Raf</td><td class="colsep0 rowsep0" align="left">TKL</td><td class="colsep0 rowsep0" align="char" char=".">0.8673</td><td class="colsep0 rowsep0" align="char" char=".">8</td><td class="colsep0 rowsep0" align="char" char=".">66</td><td class="colsep0 rowsep0" align="char" char=".">101</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><a href="https://P00519" class="ext-link">P00519</a></td><td class="colsep0 rowsep0" align="left">Abl</td><td class="colsep0 rowsep0" align="left">TK</td><td class="colsep0 rowsep0" align="char" char=".">0.9352</td><td class="colsep0 rowsep0" align="char" char=".">8</td><td class="colsep0 rowsep0" align="char" char=".">49</td><td class="colsep0 rowsep0" align="char" char=".">79</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><a href="https://P04049" class="ext-link">P04049</a></td><td class="colsep0 rowsep0" align="left">c-RAF</td><td class="colsep0 rowsep0" align="left">TKL</td><td class="colsep0 rowsep0" align="char" char=".">0.8489</td><td class="colsep0 rowsep0" align="char" char=".">8</td><td class="colsep0 rowsep0" align="char" char=".">57</td><td class="colsep0 rowsep0" align="char" char=".">93</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><a href="https://Q4JIM5" class="ext-link">Q4JIM5</a></td><td class="colsep0 rowsep0" align="left">Arg(m)</td><td class="colsep0 rowsep0" align="left">TK</td><td class="colsep0 rowsep0" align="char" char=".">0.5631</td><td class="colsep0 rowsep0" align="char" char=".">9</td><td class="colsep0 rowsep0" align="char" char=".">29</td><td class="colsep0 rowsep0" align="char" char=".">97</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><a href="https://P07333" class="ext-link">P07333</a></td><td class="colsep0 rowsep0" align="left">Fms</td><td class="colsep0 rowsep0" align="left">TK</td><td class="colsep0 rowsep0" align="char" char=".">0.9437</td><td class="colsep0 rowsep0" align="char" char=".">10</td><td class="colsep0 rowsep0" align="char" char=".">36</td><td class="colsep0 rowsep0" align="char" char=".">73</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><a href="https://Q08345" class="ext-link">Q08345</a></td><td class="colsep0 rowsep0" align="left">DDR1</td><td class="colsep0 rowsep0" align="left">TK</td><td class="colsep0 rowsep0" align="char" char=".">0.9612</td><td class="colsep0 rowsep0" align="char" char=".">11</td><td class="colsep0 rowsep0" align="char" char=".">43</td><td class="colsep0 rowsep0" align="char" char=".">92</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><a href="https://Q9H422" class="ext-link">Q9H422</a></td><td class="colsep0 rowsep0" align="left">HIPK3</td><td class="colsep0 rowsep0" align="left">CMGC</td><td class="colsep0 rowsep0" align="char" char=".">0.9742</td><td class="colsep0 rowsep0" align="char" char=".">12</td><td class="colsep0 rowsep0" align="char" char=".">68</td><td class="colsep0 rowsep0" align="char" char=".">103</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><a href="https://Q9H2X6" class="ext-link">Q9H2X6</a></td><td class="colsep0 rowsep0" align="left">HIPK2</td><td class="colsep0 rowsep0" align="left">CMGC</td><td class="colsep0 rowsep0" align="char" char=".">0.7037</td><td class="colsep0 rowsep0" align="char" char=".">13</td><td class="colsep0 rowsep0" align="char" char=".">61</td><td class="colsep0 rowsep0" align="char" char=".">96</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><a href="https://O15146" class="ext-link">O15146</a></td><td class="colsep0 rowsep0" align="left">MuSK</td><td class="colsep0 rowsep0" align="left">TK</td><td class="colsep0 rowsep0" align="char" char=".">0.9955</td><td class="colsep0 rowsep0" align="char" char=".">21</td><td class="colsep0 rowsep0" align="char" char=".">92</td><td class="colsep0 rowsep0" align="char" char=".">89</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><a href="https://P36888" class="ext-link">P36888</a></td><td class="colsep0 rowsep0" align="left">Flt3</td><td class="colsep0 rowsep0" align="left">TK</td><td class="colsep0 rowsep0" align="char" char=".">0.9801</td><td class="colsep0 rowsep0" align="char" char=".">31</td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="."> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><a href="https://O94804" class="ext-link">O94804</a></td><td class="colsep0 rowsep0" align="left">LOK</td><td class="colsep0 rowsep0" align="left">STE</td><td class="colsep0 rowsep0" align="char" char=".">0.9604</td><td class="colsep0 rowsep0" align="char" char=".">34</td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="."> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><a href="https://Q9HBH9" class="ext-link">Q9HBH9</a></td><td class="colsep0 rowsep0" align="left">Mnk2</td><td class="colsep0 rowsep0" align="left">CAMK</td><td class="colsep0 rowsep0" align="char" char=".">0.8779</td><td class="colsep0 rowsep0" align="char" char=".">34</td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="."> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><a href="https://P16234" class="ext-link">P16234</a></td><td class="colsep0 rowsep0" align="left">PDGFRα</td><td class="colsep0 rowsep0" align="left">TK</td><td class="colsep0 rowsep0" align="char" char=".">0.9929</td><td class="colsep0 rowsep0" align="char" char=".">35</td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="."> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><a href="https://P23443" class="ext-link">P23443</a></td><td class="colsep0 rowsep0" align="left">p70S6K</td><td class="colsep0 rowsep0" align="left">AGC</td><td class="colsep0 rowsep0" align="char" char=".">0.6446</td><td class="colsep0 rowsep0" align="char" char=".">36</td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="."> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><a href="https://Q7L7X3" class="ext-link">Q7L7X3</a></td><td class="colsep0 rowsep0" align="left">TAO1</td><td class="colsep0 rowsep0" align="left">STE</td><td class="colsep0 rowsep0" align="char" char=".">0.9578</td><td class="colsep0 rowsep0" align="char" char=".">42</td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="."> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><a href="https://P53667" class="ext-link">P53667</a></td><td class="colsep0 rowsep0" align="left">LIMK1</td><td class="colsep0 rowsep0" align="left">TKL</td><td class="colsep0 rowsep0" align="char" char=".">0.8753</td><td class="colsep0 rowsep0" align="char" char=".">46</td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="."> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><a href="https://Q92772" class="ext-link">Q92772</a></td><td class="colsep0 rowsep0" align="left">CDKL2</td><td class="colsep0 rowsep0" align="left">CMGC</td><td class="colsep0 rowsep0" align="char" char=".">0.9881</td><td class="colsep0 rowsep0" align="char" char=".">47</td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="."> </td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t3fn1"><div class="footnote" id="t3fn1"><sup><sup>a</sup></sup><p class="last">The % activities of a kinase at BAY73-4506 concentrations of 1 μM, 0.1 μM, and 0.01 μM. A null value indicates that the activity was not tested.</p></div><div class="footnote" id="t3fn2"><sup><sup>b</sup></sup><p class="last">Kinases in bold type have been previously reported as targets of BAY73-4506.</p></div></div></div><div class="NLM_p last">K-115 is a Rho-kinase inhibitor approved and launched in Japan as an ophthalmic solution for the treatment of glaucoma and ocular hypertension in 2014.<a onclick="showRef(event, 'ref43'); return false;" href="javascript:void(0);" class="ref ref43">(43)</a> Several “off-target” inhibitory activities against AGC kinases, including PKG and PKC, that could contribute to clinical cancer therapy were revealed by the MTDNN model.<a onclick="showRef(event, 'ref44 ref45 ref46'); return false;" href="javascript:void(0);" class="ref ref44 ref45 ref46">(44−46)</a> NVP-BHG712 is a specific EphB4 receptor inhibitor that blocks vascular endothelial growth factor-mediated angiogenesis in vivo.<a onclick="showRef(event, 'ref47'); return false;" href="javascript:void(0);" class="ref ref47">(47)</a> In addition to c-Src, NVP-BHG712 was predicted to inhibit other Src family kinases (SFKs), including Lyn, Hck, Lck, and Yes. Thus, NVP-BHG712 could potentially inhibit the proliferation and survival of cancer cells, especially pancreatic cancer and non-small-cell lung cancer.<a onclick="showRef(event, 'ref48 ref49'); return false;" href="javascript:void(0);" class="ref ref48 ref49">(48,49)</a> E31<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> was predicted to inhibit additional SFKs (Src, Yes, Blk, Lyn, Fgr, Lck), but the subsequent experiment excluded the activity against Src. This false positive prediction may indicate molecular structural similarity with known Src inhibitors, which suggested that a relatively minor structural modification could improve the inhibitory activity of E31 against Src and thus increase its therapeutic potential. For DC381016, a compound designed and synthesized for c-Met inhibition,<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> the MTDNN model also revealed inhibitory effects against several important or novel therapeutic kinases, including Abl, Trk, DDR1, and ACK1.<a onclick="showRef(event, 'ref50'); return false;" href="javascript:void(0);" class="ref ref50">(50)</a> In summary, based on the impressive experimental verification results obtained here, in addition to its high throughput, we envisage that the MTDNN model can be used as a practical tool for large-scale target identification and drug redirection.</div></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i11" class="anchor-spacer"></div><h3 class="article-section__title" id="_i11"> Selectivity</h3><div class="NLM_p">Tracking the selectivity profile for the inhibitors is a particularly important step before large-scale biochemical assays for kinase drug discovery. It is of great interest to investigate whether the MTDNN model can provide reliable predictions of the selectivity. Here, we adopted a standard score<a onclick="showRef(event, 'ref54'); return false;" href="javascript:void(0);" class="ref ref54">(54)</a> to measure the model’s ability to rank compounds in terms of their overall selectivity, which is calculated as the number of kinase hits above or below a threshold value divided by the number of kinases tested. As an accepted quantitative measurement of selectivity, the standard score is simple but effective. The selectivity scores of the Davis data set were calculated based on reported and predicted inhibitory activities, respectively. We set <i>K</i><sub>d</sub> = 1 μM and predicted probability = 50% as the activity threshold for reported and predicted inhibitory activities, respectively. The results show that the predicted selectivity scores have a significant rank correlation with the reported experimental kinase selectivity, with a Spearman correlation of 0.747 (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>).</div><figure id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-16/acs.jmedchem.9b00855/20200820/images/medium/jm9b00855_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-16/acs.jmedchem.9b00855/20200820/images/large/jm9b00855_0006.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Scatter plot of correlation between the experimental and predicted overall selectivity. The <i>x</i>-axis and <i>y</i>-axis for a point are the standard selectivity scores calculated from predicted active probabilities (>50% as active) and reported <i>K</i><sub>d</sub> values (>1 μM as active), respectively.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-16/acs.jmedchem.9b00855/20200820/images/large/jm9b00855_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00855&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Moreover, we defined the odds ratio (OR)<a onclick="showRef(event, 'ref55'); return false;" href="javascript:void(0);" class="ref ref55">(55)</a> as a measure of group-specific selectivity, which reveals whether an inhibitor preferentially targets a specific kinase subfamily or a group of kinases of particular interest. For example, to calculate the strength of the association between an inhibitor for the group of TK, the OR statistic can be calculated as<span class="NLM_disp-formula" id="ueq1"><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-16/acs.jmedchem.9b00855/20200820/images/medium/jm9b00855_m001.gif" alt="" id="_i12" /></img></span>where <i>N</i><sub>TP</sub> refers to the number of positive interactions of the inhibitor within the TK family and <i>N</i><sub>OP</sub> refers to the number of its positive interactions with kinases other than TK. Similarly, <i>N</i><sub>TN</sub> and<i>N</i><sub>ON</sub> are defined as the number of negative interactions for TK and other kinases, respectively. If the OR is significantly greater than 1.0, the kinases inhibited by this inhibitor can be considered to be enriched in the TK group, indicating that the inhibitor is TK-selective. <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a> compares the ORs of the above-mentioned five inhibitors for each of the major groups described by the kinome phylogenetic tree based on the activity spectrum predicted by the MTDNN. Clearly, the predicted group-specific selectivity revealed a high correlation with the experimental results. Even for molecules with relatively low predictive precision results, e.g., K-115, the ORs still provide a pronounced effect separating the kinase subfamily most associated with their primary studied targets.</div><div class="NLM_p">In addition to subfamily selectivity, predicting disease-specific selectivity is a compelling application of the model in a clinical setting. Taking NSCLC as an example, which is known to be associated with a variety of oncogenes encoding protein kinases, we defined a group of NSCLC-related kinases, including EGFR, ALK, ROS1, B-raf, Ret, ErbB2, Met, FGFR1, IGF1R, and Src.<a onclick="showRef(event, 'ref49 ref56 ref57 ref58 ref59 ref60'); return false;" href="javascript:void(0);" class="ref ref49 ref56 ref57 ref58 ref59 ref60">(49,56−60)</a> A total of 48 FDA-approved small-molecule kinase inhibitors were collected for analysis of whether they can be used in NSCLC treatment or exhibit repositioning potential for such treatment based on the predicted group selectivity. As shown in <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>, some well-known NSCLC drugs, such as gefitinib, crizotinib, and erlotinib, are ranked high among all those drugs. Interestingly, the top ranked three drugs are not initially approved for NSCLC treatment, but all of them have been reported to exhibit monotherapy activities for NSCLC. Examples include the following: (1) Lapatinib is an ErbB2 and EGFR kinase inhibitor launched for the treatment of advanced or metastatic HER2 (ErbB2)-positive breast cancer.<a onclick="showRef(event, 'ref61'); return false;" href="javascript:void(0);" class="ref ref61">(61)</a> A randomized phase II study has shown that although lapatinib monotherapy does not induce a significant number of tumor regressions in NSCLC, it showed equivalent progression-free survival as first-line chemotherapy in the 1500 mg once daily group.<a onclick="showRef(event, 'ref62'); return false;" href="javascript:void(0);" class="ref ref62">(62)</a> In addition, another study has demonstrated that lapatinib single treatment may be an effective option for the therapy of KRAS-mutated NSCLC that is resistant to erlotinib and gefitinib. (2) Ponatinib is a potent, oral multitargeted kinase inhibitor of Abl, PDGFRα, VEGFR2, FGFR1, and Src and has been approved for the treatment of patients with resistant or intolerant chronic myeloid leukemia (CML).<a onclick="showRef(event, 'ref63'); return false;" href="javascript:void(0);" class="ref ref63">(63)</a> Recently, Ren et al. reported that ponatinib can be used as a treatment for established NSCLC cell lines with FGFR1 overexpression, resulting in marked cell growth inhibition.<a onclick="showRef(event, 'ref64'); return false;" href="javascript:void(0);" class="ref ref64">(64)</a> (3) Bosutinib is a dual Src/Abl inhibitor for the treatment of CML. A phase I study of bosutinib has shown good tolerance and efficiency in NSCLC patients.<a onclick="showRef(event, 'ref65'); return false;" href="javascript:void(0);" class="ref ref65">(65)</a> Also worth noting are vandetanib and ibrutinib, which have shown efficacy in patients with NSCLC harboring RET rearrangement<a onclick="showRef(event, 'ref66'); return false;" href="javascript:void(0);" class="ref ref66">(66)</a> and NSCLC cell lines carrying EGFR mutations,<a onclick="showRef(event, 'ref67'); return false;" href="javascript:void(0);" class="ref ref67">(67)</a> respectively. These results highlighted the practicality of our model in finding novel medical indications for pioneering drugs. With a clearer definition of therapeutic targets related to a certain disease, the capability of the MTDNN model to guide the development of drugs with precise multitargeting selectivity can be envisaged.</div><figure id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-16/acs.jmedchem.9b00855/20200820/images/medium/jm9b00855_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-16/acs.jmedchem.9b00855/20200820/images/large/jm9b00855_0007.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Odds ratio of 48 FDA-approved small-molecule kinase inhibitors for the group of NSCLC-related kinases. Red bars indicate the FDA-approved drugs for NSCLC. The bars with red stripes indicate that there have been reported evidence showing monotherapy efficacy against NSCLC for the drugs. Blue bars indicate that there have been reported evidence showing efficacy against NSCLC when combined with other drugs. The bars filled with blue stripes indicate drugs that, to the best of our knowledge, have not been reported for treating NSCLC. (∗) 0.01 < <i>p</i> < 0.05; (∗∗) 0.001 < <i>p</i> < 0.01; (∗∗∗) <i>p</i> < 0.001.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-16/acs.jmedchem.9b00855/20200820/images/large/jm9b00855_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00855&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">In summary, the above analyses illustrated how MTDNN can be used to better understand the intrafamily and interspecies selectivity of kinase inhibitors, which highlighted the significant value of the model in repositioning inhibitors for new kinase targets and exploring their unknown therapeutic potential. Moreover, MTDNN can also be used to prioritize compounds on the basis of their overall selectivity or selectivity toward certain kinase subfamilies or diseases.</div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i13">Conclusion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_87668" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_87668" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">In this study, a virtual kinase chemogenomics model was developed for predicting the interaction profiles of kinase inhibitors against a panel of 391 kinases based on large-scale bioactivity data and the MTDNN algorithm. As a result of the high relatedness among tasks (meaning widespread cross-reactivity of kinase inhibitors) and the transfer learning effect of MTDNN, the obtained model yields excellent prediction ability with an auROC of 0.90 on an internal testing data set. On external data sets, the MTDNN model also shows impressive high-quality prediction results, despite the apparent deviation of chemical diversity distribution and the uncertainty in different data sources. The analysis revealed that the prediction results of the model could be further improved by expanding the scale and increasing the diversity of chemotypes in the training data set. Compared with conventional single-task RF models, the model consistently shows higher auROCs on external data sets, especially for kinases with insufficient activity data. Moreover, rigorous experimental validations were performed using compounds with diverse structures and unknown kinase activity profiles. The predicted spectrum shows significant agreement with experimental data, with a high-quality average auROC of 0.75, and many novel and therapeutic “off-target” activities have been successfully predicted by the model and confirmed by subsequent experiments. On the basis of the predicted kinase profile, the MTDNN model can also be used to depict the overall selectivity, the selectivity toward a subfamily of kinases, and the strength of association with certain diseases such as NSCLC. Overall, MTDNN enables us to create a comprehensive kinome interaction network for designing novel chemical modulators or drug repositioning and is of practical value for exploring previously less studied kinases.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i14">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_61083" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_61083" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i15" class="anchor-spacer"></div><h3 class="article-section__title" id="_i15"> Data Sets</h3><div class="NLM_p last">The SARfari data set and Metz data set were merged into one set to generate the classification model. (1) The SARfari data set refers to Kinase SARfari database (accessed Nov 2017), an integrated chemogenomics workbench focused on kinases, which is composed of 54 189 compounds, 989 different kinase domains, and 532 155 data points in the form of IC<sub>50</sub>, <i>K</i><sub>i</sub>, <i>K</i><sub>d</sub>, and other values. (2) The second data set, the Metz data set,<a onclick="showRef(event, 'ref68'); return false;" href="javascript:void(0);" class="ref ref68">(68)</a> contains 1498 compounds with known structures, 173 human kinases, and 107 791 p<i>K</i><sub>i</sub> data points. The inhibition activity in the merged data set was converted to two classes: active (p<i>K</i><sub>i</sub>/p<i>K</i><sub>d</sub>/pIC<sub>50</sub> ≥ 6) and inactive (p<i>K</i><sub>i</sub>/p<i>K</i><sub>d</sub>/pIC<sub>50</sub> < 6). After the deletion of mutant kinases and kinases without both active and inactive data points, the final data set contains over 170 000 bioactivity data points composed of 391 kinases (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00855/suppl_file/jm9b00855_si_001.xlsx" class="ext-link">Table S1</a>) and ∼32 000 compounds. The data set is divided into 80% training and 20% testing data sets by random selection of compounds.</div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i16" class="anchor-spacer"></div><h3 class="article-section__title" id="_i16"> Cluster Cross-Validation</h3><div class="NLM_p">In contrast to conventional <i>k</i>-fold cross-validation, which distributes the compounds randomly cross the folds, cluster cross-validation<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> identifies clusters of compounds in the modeling data set and distributes them to folds to guarantee that compounds of the same cluster are present either only in the training or only in the test set.</div><div class="NLM_p last">We performed a 5-fold cluster cross-validation using the hierarchical clustering single linkage algorithm. Single linkage represents a way to measure the dissimilarity between groups of samples, in which the dissimilarity between groups G and H is the smallest dissimilarity between two points in opposite groups:<span class="NLM_disp-formula" id="ueq2"><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-16/acs.jmedchem.9b00855/20200820/images/medium/jm9b00855_m002.gif" alt="" id="_i17" /></img></span>With this property, single linkage cluster guarantees a minimum distance between compounds of all folds. The distances <i>d<sub>ij</sub></i> between compounds are measured by Jaccard distances (1 – Tanimoto similarity) on binarized ECFP4 compound representations. The minimum distance was set as 0.3. As a result, all compounds were clustered into 7861 clusters with 1640 compounds for the largest cluster, which were then merged to 5 different folds with about 6400 compounds each.</div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18"> Data Balancing Methods</h3><div class="NLM_p last">For many data sets, the numbers of data points that belong to different classes are significantly different. The direct development of machine learning models using imbalanced data sets will fail to properly represent the distributive characteristics of the data and thus provide unfavorable accuracies across the categories. Instead of creating balanced data distributions through different sampling strategies, a cost-sensitive method algorithm<a onclick="showRef(event, 'ref69'); return false;" href="javascript:void(0);" class="ref ref69">(69)</a> was applied in our model, in which the cross entropy is weighted based on the ratio of the numbers of different categories. To address the issue that there are far fewer active data points than inactive data points for many kinases, a penalty for misclassifying the active data points coefficient is introduced into the cost matrix. Accordingly, the cross entropy cost matrix of a data point for each kinase is converted to<span class="NLM_disp-formula" id="ueq3"><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-16/acs.jmedchem.9b00855/20200820/images/medium/jm9b00855_m003.gif" alt="" id="_i19" /></img></span>where <i>N</i><sub>i</sub> and <i>N</i><sub>a</sub> refer to the number of inactive data points and the number of active data points for a kinase, respectively, and <i>y<sub>m</sub></i>′ and <i>y<sub>m</sub></i> are the <i>m</i>th class of true labels and predicted outputs, respectively.</div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i20" class="anchor-spacer"></div><h3 class="article-section__title" id="_i20"> Multitask Deep Neural Networks (MTDNNs)</h3><div class="NLM_p">MTDNNs implement a straightforward solution to problems with multiple related labels. The shared hidden layers among all tasks can help the model learn a shared representation and thereby obtain relatively strong abstracting capabilities. Xu et al.<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> have demonstrated that an MTDNN can outperform single-task DNNs by utilizing the activities of molecules from the other tasks if the molecules from these tasks share similar structures and correlated activities (either positively or negatively correlated). Ma et al.<a onclick="showRef(event, 'ref70'); return false;" href="javascript:void(0);" class="ref ref70">(70)</a> also proposed that an MTDNN can take advantage of the tasks with a larger training data set to improve the predictive performance for the tasks with a smaller training set, and the regularization effect produced by various task can help the MTDNN avoid potential overfitting. For kinases, as most inhibitors interact with the hinge motif in the highly conserved catalytic domain, the structures and activities of inhibitors of different kinases are substantially correlated. Thus, MTDNN provides an appropriate solution to kinome spectrum prediction, and the cross-reaction of inhibitors can boost the predictive performance for related kinases, especially for kinases with insufficient interaction data. In this study, we constructed a multitask classification DNN architecture with shared hidden layers among all tasks (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>), where each task represents the activity to be predicted against a specific kinase. Extended connectivity fingerprints<a onclick="showRef(event, 'ref71'); return false;" href="javascript:void(0);" class="ref ref71">(71)</a> with a radius of 2 continuous bonds (ECPF4) and a length of 1024 bits were adopted to featurize each molecule and fed into the input layer, which means that only two-dimensional chemical structural information is needed for making predictions based on the resulting model. The input was fed into one or more (<i>L</i>) fully connected hidden layers, and the rectified linear unit (ReLU)<a onclick="showRef(event, 'ref72'); return false;" href="javascript:void(0);" class="ref ref72">(72)</a> was chosen to perform the nonlinear transformation as follows between any two adjacent layers:<span class="NLM_disp-formula" id="ueq4"><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-16/acs.jmedchem.9b00855/20200820/images/medium/jm9b00855_m004.gif" alt="" id="_i21" /></img></span>where <i>X</i><sub><i>l</i></sub>, <i>W<sub>l</sub></i>, and <i>b</i><sub><i>l</i></sub> represent the input, weight matrix, and bias for the <i>l</i>th layer, respectively, and σ refers to the ReLU activation function. After <i>L</i> transformations, the <i>L</i>th layer is fed into a task-specific output layer, which consists of <i>N</i> softmax classifiers corresponding to <i>N</i> tasks with <i>M</i> labels (<i>M</i> = 2 for binary classification) for predicting whether the input molecule is active or inactive toward a panel of <i>N</i> kinases. For the <i>n</i>th task, the probability that the input has a label of <i>m</i> is<span class="NLM_disp-formula" id="ueq5"><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-16/acs.jmedchem.9b00855/20200820/images/medium/jm9b00855_m005.gif" alt="" id="_i22" /></img></span>The cross entropy between true labels and predicted labels is calculated as the loss function, and thus, the weight matrix and bias vectors can be updated using the backpropagation algorithm:<span class="NLM_disp-formula" id="ueq6"><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-16/acs.jmedchem.9b00855/20200820/images/medium/jm9b00855_m006.gif" alt="" id="_i23" /></img></span>where <i>p</i> refers to <i>N</i><sub>i</sub>/<i>N</i><sub>a</sub> or 1 according to the class of true labels as mentioned in the balancing method. The process of network training will repeat multiple epochs until the loss or other evaluation metrics converge.</div><figure id="fig8" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-16/acs.jmedchem.9b00855/20200820/images/medium/jm9b00855_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-16/acs.jmedchem.9b00855/20200820/images/large/jm9b00855_0008.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Architecture of a multitask deep neural network.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-16/acs.jmedchem.9b00855/20200820/images/large/jm9b00855_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00855&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i24" class="anchor-spacer"></div><h3 class="article-section__title" id="_i24"> Optimization of Hyperparameters</h3><div class="NLM_p">In general, DNN is sensitive to the choice of hyper parameters, and so is MTDNN. To optimize the generalizability of the model, we explored a number of different hyperparameters through a grid search,<a onclick="showRef(event, 'ref73'); return false;" href="javascript:void(0);" class="ref ref73">(73)</a> including network architecture (layer size), learning rate, and batch size. An early stopping<a onclick="showRef(event, 'ref74'); return false;" href="javascript:void(0);" class="ref ref74">(74)</a> scheme was introduced during the process of training, where the auROC on the validation set was monitored every five epochs. The training is stopped once the auROC decreases four times continuously.</div><div class="NLM_p">The layer size was varied from shallow (one hidden layer with 1000 nodes, i.e., [1000]) to moderate (two hidden layers of 1500 and 1000 nodes, i.e., [1500,1000]) and deep ([2000,1000,500]). Additionally, as there are 1024 nodes in the input layer and 391 tasks in the output layer, we also tried another two-hidden-layer architecture [1024,391]. Since network architectures with different levels of complexity require different learning rates to achieve stable and improved performance, the Adaptive Moment Estimation (Adam) method<a onclick="showRef(event, 'ref75'); return false;" href="javascript:void(0);" class="ref ref75">(75)</a> was used for backpropagation with initial learning rates varying among 1 × 10<sup>–3</sup>, 1 × 10<sup>–4</sup>, 5 × 10<sup>–5</sup>, and 1 × 10<sup>–5</sup>. Two sizes of minibatch (64 and 128) were also tested. To regularize our network, we used a dropout of 0.5 as well as a moderate L2 weight decay of 0.002 for each hidden layer in every network. These two strategies have been proven to work in concert to avoid overfitting.<a onclick="showRef(event, 'ref76'); return false;" href="javascript:void(0);" class="ref ref76">(76)</a></div><div class="NLM_p last">MTDNN models are developed in Tensorflow (version 1.6.0) and DeepChem (version 2.1.0). All trainings are performed on standard NVIDIA GPUs. The code is developed in Python 3.6.</div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i25" class="anchor-spacer"></div><h3 class="article-section__title" id="_i25"> Evaluation Metrics</h3><div class="NLM_p">For model quality assessment, the auROC (area under the ROC curve), Recall, Precision, F1-score, and BA (balanced accuracy) were evaluated (<a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>).</div><div class="NLM_table-wrap" id="tbl4"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 4. Description of the Evaluation Metrics</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">evaluation metric</th><th class="colsep0 rowsep0" align="center">equation<a class="ref internalNav" href="#t4fn1" aria-label="a">a</a></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Recall</td><td class="colsep0 rowsep0" align="left">TP/(TP + FN)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Precision</td><td class="colsep0 rowsep0" align="left">TP/(TP + FP)</td></tr><tr valign="middle" class="colsep0"><td class="colsep0 rowsep0" align="left" valign="middle">F1-score</td><td class="colsep0 rowsep0" align="left" valign="middle"><span class="NLM_disp-formula" id="ueq7"><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-16/acs.jmedchem.9b00855/20200820/images/medium/jm9b00855_m007.gif" alt="" id="_i27" /></img></span></td></tr><tr valign="middle" class="colsep0"><td class="colsep0 rowsep0" align="left" valign="middle">BA</td><td class="colsep0 rowsep0" align="left" valign="middle"><span class="NLM_disp-formula" id="ueq8"><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-16/acs.jmedchem.9b00855/20200820/images/medium/jm9b00855_m008.gif" alt="" id="_i28" /></img></span></td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t4fn1"><div class="footnote" id="t4fn1"><sup><sup>a</sup></sup><p class="last">TP is the number of correctly predicted actives (true positives). TN is the number of correctly predicted inactives (true negatives). FP is the number of incorrectly identified actives (false positives), and FN is the number of incorrectly identified inactives (false negatives).</p></div></div></div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i29"><a href="/doi/suppl/10.1021/acs.jmedchem.9b00855" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_82124" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_82124" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a href="/" class="ext-link">ACS Publications website</a> at DOI: <a href="/doi/abs/10.1021/acs.jmedchem.9b00855" class="ext-link">10.1021/acs.jmedchem.9b00855</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Table S1 of kinase information (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00855/suppl_file/jm9b00855_si_001.xlsx" class="ext-link">XLSX</a>)</p></li><li><p class="inline">Table S2 of prediction performance comparisons between MTDNN and RF (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00855/suppl_file/jm9b00855_si_002.xlsx" class="ext-link">XLSX</a>)</p></li><li><p class="inline">Table S3 of experimental data (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00855/suppl_file/jm9b00855_si_003.xlsx" class="ext-link">XLSX</a>)</p></li><li><p class="inline">Molecular formula strings and the associated biochemical and biological data (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00855/suppl_file/jm9b00855_si_004.csv" class="ext-link">CSV</a>)</p></li><li><p class="inline">Molecular information (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00855/suppl_file/jm9b00855_si_005.pdf" class="ext-link">PDF</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00855/suppl_file/jm9b00855_si_001.xlsx">jm9b00855_si_001.xlsx (50.75 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00855/suppl_file/jm9b00855_si_002.xlsx">jm9b00855_si_002.xlsx (57.6 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00855/suppl_file/jm9b00855_si_003.xlsx">jm9b00855_si_003.xlsx (2.04 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00855/suppl_file/jm9b00855_si_004.csv">jm9b00855_si_004.csv (281.82 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00855/suppl_file/jm9b00855_si_005.pdf">jm9b00855_si_005.pdf (200.65 kb)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.9b00855" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_13738" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_13738" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Hualiang Jiang</span> - <span class="hlFld-Affiliation affiliation">Drug
Discovery and Design Center, State Key Laboratory of Drug Research, Shanghai Institute of Materia Medica, Chinese Academy
of Sciences, 555 Zuchongzhi Road, Shanghai 201203, China</span>; 
    <span class="hlFld-Affiliation affiliation">School
of Life Science and Technology, ShanghaiTech
University, 393 Huaxiazhong
Road, Shanghai 200031, China</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#2a424640434b444d6a59434747044b49044944"><span class="__cf_email__" data-cfemail="87efebedeee6e9e0c7f4eeeaeaa9e6e4a9e4e9">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Mingyue Zheng</span> - <span class="hlFld-Affiliation affiliation">Drug
Discovery and Design Center, State Key Laboratory of Drug Research, Shanghai Institute of Materia Medica, Chinese Academy
of Sciences, 555 Zuchongzhi Road, Shanghai 201203, China</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-3323-3092" title="Orcid link">http://orcid.org/0000-0002-3323-3092</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#ddb0a4a7b5b8b3ba9daeb4b0b0f3bcbef3beb3"><span class="__cf_email__" data-cfemail="afc2d6d5c7cac1c8efdcc6c2c281cecc81ccc1">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Xutong Li</span> - <span class="hlFld-Affiliation affiliation">Drug
Discovery and Design Center, State Key Laboratory of Drug Research, Shanghai Institute of Materia Medica, Chinese Academy
of Sciences, 555 Zuchongzhi Road, Shanghai 201203, China</span>; 
    <span class="hlFld-Affiliation affiliation">University
of Chinese Academy of Sciences, No. 19A Yuquan Road, Beijing 100049, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Zhaojun Li</span> - <span class="hlFld-Affiliation affiliation">School
of Information Management, Dezhou University, 566 West University Road, Dezhou 253023, China</span>; 
    <span class="hlFld-Affiliation affiliation">Drug
Discovery and Design Center, State Key Laboratory of Drug Research, Shanghai Institute of Materia Medica, Chinese Academy
of Sciences, 555 Zuchongzhi Road, Shanghai 201203, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Xiaolong Wu</span> - <span class="hlFld-Affiliation affiliation">Drug
Discovery and Design Center, State Key Laboratory of Drug Research, Shanghai Institute of Materia Medica, Chinese Academy
of Sciences, 555 Zuchongzhi Road, Shanghai 201203, China</span>; 
    <span class="hlFld-Affiliation affiliation">School
of Pharmacy, East China University of Science
and Technology, 130 Meilong Road, Shanghai 200237, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Zhaoping Xiong</span> - <span class="hlFld-Affiliation affiliation">Drug
Discovery and Design Center, State Key Laboratory of Drug Research, Shanghai Institute of Materia Medica, Chinese Academy
of Sciences, 555 Zuchongzhi Road, Shanghai 201203, China</span>; 
    <span class="hlFld-Affiliation affiliation">School
of Life Science and Technology, ShanghaiTech
University, 393 Huaxiazhong
Road, Shanghai 200031, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Tianbiao Yang</span> - <span class="hlFld-Affiliation affiliation">Drug
Discovery and Design Center, State Key Laboratory of Drug Research, Shanghai Institute of Materia Medica, Chinese Academy
of Sciences, 555 Zuchongzhi Road, Shanghai 201203, China</span>; 
    <span class="hlFld-Affiliation affiliation">University
of Chinese Academy of Sciences, No. 19A Yuquan Road, Beijing 100049, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Zunyun Fu</span> - <span class="hlFld-Affiliation affiliation">Drug
Discovery and Design Center, State Key Laboratory of Drug Research, Shanghai Institute of Materia Medica, Chinese Academy
of Sciences, 555 Zuchongzhi Road, Shanghai 201203, China</span>; 
    <span class="hlFld-Affiliation affiliation">University
of Chinese Academy of Sciences, No. 19A Yuquan Road, Beijing 100049, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Xiaohong Liu</span> - <span class="hlFld-Affiliation affiliation">Drug
Discovery and Design Center, State Key Laboratory of Drug Research, Shanghai Institute of Materia Medica, Chinese Academy
of Sciences, 555 Zuchongzhi Road, Shanghai 201203, China</span>; 
    <span class="hlFld-Affiliation affiliation">School
of Life Science and Technology, ShanghaiTech
University, 393 Huaxiazhong
Road, Shanghai 200031, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Xiaoqin Tan</span> - <span class="hlFld-Affiliation affiliation">Drug
Discovery and Design Center, State Key Laboratory of Drug Research, Shanghai Institute of Materia Medica, Chinese Academy
of Sciences, 555 Zuchongzhi Road, Shanghai 201203, China</span>; 
    <span class="hlFld-Affiliation affiliation">University
of Chinese Academy of Sciences, No. 19A Yuquan Road, Beijing 100049, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Feisheng Zhong</span> - <span class="hlFld-Affiliation affiliation">Drug
Discovery and Design Center, State Key Laboratory of Drug Research, Shanghai Institute of Materia Medica, Chinese Academy
of Sciences, 555 Zuchongzhi Road, Shanghai 201203, China</span>; 
    <span class="hlFld-Affiliation affiliation">University
of Chinese Academy of Sciences, No. 19A Yuquan Road, Beijing 100049, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Xiaozhe Wan</span> - <span class="hlFld-Affiliation affiliation">Drug
Discovery and Design Center, State Key Laboratory of Drug Research, Shanghai Institute of Materia Medica, Chinese Academy
of Sciences, 555 Zuchongzhi Road, Shanghai 201203, China</span>; 
    <span class="hlFld-Affiliation affiliation">University
of Chinese Academy of Sciences, No. 19A Yuquan Road, Beijing 100049, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Dingyan Wang</span> - <span class="hlFld-Affiliation affiliation">Drug
Discovery and Design Center, State Key Laboratory of Drug Research, Shanghai Institute of Materia Medica, Chinese Academy
of Sciences, 555 Zuchongzhi Road, Shanghai 201203, China</span>; 
    <span class="hlFld-Affiliation affiliation">University
of Chinese Academy of Sciences, No. 19A Yuquan Road, Beijing 100049, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Xiaoyu Ding</span> - <span class="hlFld-Affiliation affiliation">Drug
Discovery and Design Center, State Key Laboratory of Drug Research, Shanghai Institute of Materia Medica, Chinese Academy
of Sciences, 555 Zuchongzhi Road, Shanghai 201203, China</span>; 
    <span class="hlFld-Affiliation affiliation">University
of Chinese Academy of Sciences, No. 19A Yuquan Road, Beijing 100049, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ruirui Yang</span> - <span class="hlFld-Affiliation affiliation">Drug
Discovery and Design Center, State Key Laboratory of Drug Research, Shanghai Institute of Materia Medica, Chinese Academy
of Sciences, 555 Zuchongzhi Road, Shanghai 201203, China</span>; 
    <span class="hlFld-Affiliation affiliation">School
of Life Science and Technology, ShanghaiTech
University, 393 Huaxiazhong
Road, Shanghai 200031, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Hui Hou</span> - <span class="hlFld-Affiliation affiliation">Drug
Discovery and Design Center, State Key Laboratory of Drug Research, Shanghai Institute of Materia Medica, Chinese Academy
of Sciences, 555 Zuchongzhi Road, Shanghai 201203, China</span>; 
    <span class="hlFld-Affiliation affiliation">School of
Pharmacy, Fudan University, 826 Zhangheng Road, Shanghai 201203, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Chunpu Li</span> - <span class="hlFld-Affiliation affiliation">Drug
Discovery and Design Center, State Key Laboratory of Drug Research, Shanghai Institute of Materia Medica, Chinese Academy
of Sciences, 555 Zuchongzhi Road, Shanghai 201203, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Hong Liu</span> - <span class="hlFld-Affiliation affiliation">Drug
Discovery and Design Center, State Key Laboratory of Drug Research, Shanghai Institute of Materia Medica, Chinese Academy
of Sciences, 555 Zuchongzhi Road, Shanghai 201203, China</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-0863-7853" title="Orcid link">http://orcid.org/0000-0002-0863-7853</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Kaixian Chen</span> - <span class="hlFld-Affiliation affiliation">Drug
Discovery and Design Center, State Key Laboratory of Drug Research, Shanghai Institute of Materia Medica, Chinese Academy
of Sciences, 555 Zuchongzhi Road, Shanghai 201203, China</span>; 
    <span class="hlFld-Affiliation affiliation">School
of Life Science and Technology, ShanghaiTech
University, 393 Huaxiazhong
Road, Shanghai 200031, China</span></div></li></ul></li><li></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ack1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i30">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_36836" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_36836" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">We gratefully acknowledge financial support from the National Natural Science Foundation of China (Grant 81773634 to M.Z. and Grant 81430084 to K.C.), National Science & Technology Major Project “Key New Drug Creation and Manufacturing Program”, China (Grant 2018ZX09711002 to H.J.), and “Personalized Medicines—Molecular Signature-Based Drug Discovery and Development”, Strategic Priority Research Program of the Chinese Academy of Sciences (Grant XDA12050201 to M.Z.), and the Open Fund of State Key Laboratory of Pharmaceutical Biotechnology, Nanjing University, China (Grant KF-GN-201706 to H.J.).</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations Used</td></tr><tr><td class="NLM_term">NB</td><td class="NLM_def"><p class="first last">naive Bayesian</p></td></tr><tr><td class="NLM_term">KNN</td><td class="NLM_def"><p class="first last">k-nearest neighbors</p></td></tr><tr><td class="NLM_term">RF</td><td class="NLM_def"><p class="first last">random forest</p></td></tr><tr><td class="NLM_term">SVM</td><td class="NLM_def"><p class="first last">support vector machine</p></td></tr><tr><td class="NLM_term">DNN</td><td class="NLM_def"><p class="first last">deep neural network</p></td></tr><tr><td class="NLM_term">MTDNN</td><td class="NLM_def"><p class="first last">multitask deep neural network</p></td></tr><tr><td class="NLM_term">auROC</td><td class="NLM_def"><p class="first last">area under receiver operating characteristic</p></td></tr><tr><td class="NLM_term">BA</td><td class="NLM_def"><p class="first last">balanced accuracy</p></td></tr><tr><td class="NLM_term">PKIS1</td><td class="NLM_def"><p class="first last">published kinase inhibitor set 1</p></td></tr><tr><td class="NLM_term">PKIS2</td><td class="NLM_def"><p class="first last">published kinase inhibitor set 2</p></td></tr><tr><td class="NLM_term">t-SNE</td><td class="NLM_def"><p class="first last">t-distributed stochastic neighbor embedding</p></td></tr><tr><td class="NLM_term">ECFP4</td><td class="NLM_def"><p class="first last">extended connectivity fingerprint 4</p></td></tr><tr><td class="NLM_term">DCNB</td><td class="NLM_def"><p class="first last">dual-component naive Bayes</p></td></tr><tr><td class="NLM_term">DCSVM</td><td class="NLM_def"><p class="first last">dual-component SVM</p></td></tr><tr><td class="NLM_term">DDM</td><td class="NLM_def"><p class="first last">drug discovery map</p></td></tr><tr><td class="NLM_term">FLT3</td><td class="NLM_def"><p class="first last">FMS-like tyrosine kinase 3</p></td></tr><tr><td class="NLM_term">DAD</td><td class="NLM_def"><p class="first last">diode-array detector</p></td></tr><tr><td class="NLM_term">VEGFR</td><td class="NLM_def"><p class="first last">vascular endothelial growth factor receptor</p></td></tr><tr><td class="NLM_term">PDGFRβ</td><td class="NLM_def"><p class="first last">platelet-derived growth factor receptor β</p></td></tr><tr><td class="NLM_term">Kit</td><td class="NLM_def"><p class="first last">mast/stem cell growth factor receptor Kit</p></td></tr><tr><td class="NLM_term">RET</td><td class="NLM_def"><p class="first last">proto-oncogene tyrosine-protein kinase receptor Ret</p></td></tr><tr><td class="NLM_term">c-Raf</td><td class="NLM_def"><p class="first last">RAF proto-oncogene serine/threonine-protein kinase</p></td></tr><tr><td class="NLM_term">p38α</td><td class="NLM_def"><p class="first last">mitogen-activated protein kinase 14</p></td></tr><tr><td class="NLM_term">ZAK</td><td class="NLM_def"><p class="first last">sterile α motif and leucine zipper containing kinase AZK</p></td></tr><tr><td class="NLM_term">TAO1</td><td class="NLM_def"><p class="first last">serine/threonine-protein kinase</p></td></tr><tr><td class="NLM_term">MAPK</td><td class="NLM_def"><p class="first last">mitogen-activated protein kinase</p></td></tr><tr><td class="NLM_term">TrkB</td><td class="NLM_def"><p class="first last">BDNF/NT-3 growth factors receptor</p></td></tr><tr><td class="NLM_term">TrkC</td><td class="NLM_def"><p class="first last">NT-3 growth factor receptor</p></td></tr><tr><td class="NLM_term">NSCLC</td><td class="NLM_def"><p class="first last">non-small-cell lung cancer</p></td></tr><tr><td class="NLM_term">DDR1</td><td class="NLM_def"><p class="first last">epithelial discoidin domain-containing receptor 1</p></td></tr><tr><td class="NLM_term">LOK</td><td class="NLM_def"><p class="first last">serine/threonine-protein kinase 10</p></td></tr><tr><td class="NLM_term">PKG</td><td class="NLM_def"><p class="first last">protein kinase G</p></td></tr><tr><td class="NLM_term">PKC</td><td class="NLM_def"><p class="first last">protein kinase C</p></td></tr><tr><td class="NLM_term">Src</td><td class="NLM_def"><p class="first last">proto-oncogene tyrosine-protein kinase Src</p></td></tr><tr><td class="NLM_term">SFK</td><td class="NLM_def"><p class="first last">Src family kinase</p></td></tr><tr><td class="NLM_term">Lyn</td><td class="NLM_def"><p class="first last">tyrosine-protein kinase Lyn</p></td></tr><tr><td class="NLM_term">Hck</td><td class="NLM_def"><p class="first last">hematopoietic cell kinase</p></td></tr><tr><td class="NLM_term">Lck</td><td class="NLM_def"><p class="first last">leukocyte c-terminal Src kinase</p></td></tr><tr><td class="NLM_term">Yes</td><td class="NLM_def"><p class="first last">tyrosine-protein kinase Yes</p></td></tr><tr><td class="NLM_term">Blk</td><td class="NLM_def"><p class="first last">B lymphocyte kinase</p></td></tr><tr><td class="NLM_term">Fgr</td><td class="NLM_def"><p class="first last">tyrosine-protein kinase Fgr</p></td></tr><tr><td class="NLM_term">c-Met</td><td class="NLM_def"><p class="first last">hepatocyte growth factor receptor</p></td></tr><tr><td class="NLM_term">Abl</td><td class="NLM_def"><p class="first last">Abelson murine leukemia viral oncogene homologue 1</p></td></tr><tr><td class="NLM_term">ACK1</td><td class="NLM_def"><p class="first last">activated CDC42 kinase 1</p></td></tr><tr><td class="NLM_term">ROCK1</td><td class="NLM_def"><p class="first last">rho-associated protein kinase 1</p></td></tr><tr><td class="NLM_term">ROCK2</td><td class="NLM_def"><p class="first last">rho-associated protein kinase 2</p></td></tr><tr><td class="NLM_term">EGFR</td><td class="NLM_def"><p class="first last">epidermal growth factor receptor</p></td></tr><tr><td class="NLM_term">ERBB2</td><td class="NLM_def"><p class="first last">receptor tyrosine-protein kinase erbB-2</p></td></tr><tr><td class="NLM_term">ERBB4</td><td class="NLM_def"><p class="first last">receptor tyrosine-protein kinase erbB-4</p></td></tr><tr><td class="NLM_term">OR</td><td class="NLM_def"><p class="first last">odds ratio</p></td></tr><tr><td class="NLM_term">ALK</td><td class="NLM_def"><p class="first last">ALK tyrosine kinase receptor</p></td></tr><tr><td class="NLM_term">ROS1</td><td class="NLM_def"><p class="first last">proto-oncogene tyrosine-protein kinase ROS</p></td></tr><tr><td class="NLM_term">B-raf</td><td class="NLM_def"><p class="first last">serine/threonine-protein kinase B-raf</p></td></tr><tr><td class="NLM_term">FGFR1</td><td class="NLM_def"><p class="first last">fibroblast growth factor receptor 1</p></td></tr><tr><td class="NLM_term">IGF1R</td><td class="NLM_def"><p class="first last">insulin-like growth factor 1 receptor</p></td></tr><tr><td class="NLM_term">KRAS</td><td class="NLM_def"><p class="first last">GTPase KRas</p></td></tr><tr><td class="NLM_term">PDGFRα</td><td class="NLM_def"><p class="first last">platelet-derived growth factor receptor α</p></td></tr><tr><td class="NLM_term">VEGFR2</td><td class="NLM_def"><p class="first last">vascular endothelial growth factor receptor 2</p></td></tr><tr><td class="NLM_term">CML</td><td class="NLM_def"><p class="first last">chronic myeloid leukemia</p></td></tr><tr><td class="NLM_term">HER2</td><td class="NLM_def"><p class="first last">receptor tyrosine-protein kinase erbB-2</p></td></tr><tr><td class="NLM_term">ReLU</td><td class="NLM_def"><p class="first last">rectified linear unit</p></td></tr><tr><td class="NLM_term">GPU</td><td class="NLM_def"><p class="first last">graphics processing unit</p></td></tr><tr><td class="NLM_term">TP</td><td class="NLM_def"><p class="first last">true positive</p></td></tr><tr><td class="NLM_term">TN</td><td class="NLM_def"><p class="first last">true negative</p></td></tr><tr><td class="NLM_term">FP</td><td class="NLM_def"><p class="first last">false positive</p></td></tr><tr><td class="NLM_term">FN</td><td class="NLM_def"><p class="first last">false negative</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i32">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_37201" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_37201" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 76 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Manning, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whyte, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martinez, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hunter, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sudarsanam, S.</span></span> <span> </span><span class="NLM_article-title">The protein kinase complement of the human genome</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>298</i></span>,  <span class="NLM_fpage">1912</span>– <span class="NLM_lpage">1934</span>, <span class="refDoi"> DOI: 10.1126/science.1075762</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=10.1126%2Fscience.1075762" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=12471243" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=1%3ACAS%3A528%3ADC%252BD38Xpt1Wisb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=298&publication_year=2002&pages=1912-1934&author=G.+Manningauthor=D.+B.+Whyteauthor=R.+Martinezauthor=T.+Hunterauthor=S.+Sudarsanam&title=The+protein+kinase+complement+of+the+human+genome&doi=10.1126%2Fscience.1075762"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">The Protein Kinase Complement of the Human Genome</span></div><div class="casAuthors">Manning, G.; Whyte, D. B.; Martinez, R.; Hunter, T.; Sudarsanam, S.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">298</span>
        (<span class="NLM_cas:issue">5600</span>),
    <span class="NLM_cas:pages">1912-1916, 1933-1934</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">We have catalogued the protein kinase complement of the human genome (the "kinome") using public and proprietary genomic, complementary DNA, and expressed sequence tag (EST) sequences.  This provides a starting point for comprehensive anal. of protein phosphorylation in normal and disease states, as well as a detailed view of the current state of human genome anal. through a focus on one large gene family.  We identify 518 putative protein kinase genes, of which 71 have not previously been reported or described as kinases, and we extend or correct the protein sequences of 56 more kinases.  New genes include members of well-studied families as well as previously unidentified families, some of which are conserved in model organisms.  Classification and comparison with model organism kinomes identified orthologous groups and highlighted expansions specific to human and other lineages.  We also identified 106 protein kinase pseudogenes.  Chromosomal mapping revealed several small clusters of kinase genes and revealed that 244 kinases map to disease loci or cancer amplicons.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpTj1hf_0cQcrVg90H21EOLACvtfcHk0lj_hBjBgcxuTg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38Xpt1Wisb0%253D&md5=b3def83bace52257a3252a3347b8bb92</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1126%2Fscience.1075762&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1075762%26sid%3Dliteratum%253Aachs%26aulast%3DManning%26aufirst%3DG.%26aulast%3DWhyte%26aufirst%3DD.%2BB.%26aulast%3DMartinez%26aufirst%3DR.%26aulast%3DHunter%26aufirst%3DT.%26aulast%3DSudarsanam%26aufirst%3DS.%26atitle%3DThe%2520protein%2520kinase%2520complement%2520of%2520the%2520human%2520genome%26jtitle%3DScience%26date%3D2002%26volume%3D298%26spage%3D1912%26epage%3D1934%26doi%3D10.1126%2Fscience.1075762" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wu, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nielsen, T. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clausen, M. H.</span></span> <span> </span><span class="NLM_article-title">FDA-approved small-molecule kinase inhibitors</span>. <i>Trends Pharmacol. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>36</i></span>,  <span class="NLM_fpage">422</span>– <span class="NLM_lpage">439</span>, <span class="refDoi"> DOI: 10.1016/j.tips.2015.04.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=10.1016%2Fj.tips.2015.04.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=25975227" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=1%3ACAS%3A528%3ADC%252BC2MXns1KisLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2015&pages=422-439&author=P.+Wuauthor=T.+E.+Nielsenauthor=M.+H.+Clausen&title=FDA-approved+small-molecule+kinase+inhibitors&doi=10.1016%2Fj.tips.2015.04.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">FDA-approved small-molecule kinase inhibitors</span></div><div class="casAuthors">Wu, Peng; Nielsen, Thomas E.; Clausen, Mads H.</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Pharmacological Sciences</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">422-439</span>CODEN:
                <span class="NLM_cas:coden">TPHSDY</span>;
        ISSN:<span class="NLM_cas:issn">0165-6147</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Kinases have emerged as one of the most intensively pursued targets in current pharmacol. research, esp. for cancer, due to their crit. roles in cellular signaling.  To date, the US FDA has approved 28 small-mol. kinase inhibitors, half of which were approved in the past 3 years.  While the clin. data of these approved mols. are widely presented and structure-activity relationship (SAR) has been reported for individual mols., an updated review that analyzes all approved mols. and summarizes current achievements and trends in the field has yet to be found.  Here we present all approved small-mol. kinase inhibitors with an emphasis on binding mechanism and structural features, summarize current challenges, and discuss future directions in this field.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrudyvM3eO09bVg90H21EOLACvtfcHk0ljuEVnpPxZ10g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXns1KisLg%253D&md5=a6aa3dbb11e6fe3001fa5fa03a4c950a</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1016%2Fj.tips.2015.04.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tips.2015.04.005%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DP.%26aulast%3DNielsen%26aufirst%3DT.%2BE.%26aulast%3DClausen%26aufirst%3DM.%2BH.%26atitle%3DFDA-approved%2520small-molecule%2520kinase%2520inhibitors%26jtitle%3DTrends%2520Pharmacol.%2520Sci.%26date%3D2015%26volume%3D36%26spage%3D422%26epage%3D439%26doi%3D10.1016%2Fj.tips.2015.04.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fabbro, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowan-Jacob, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moebitz, H.</span></span> <span> </span><span class="NLM_article-title">Ten things you should know about protein kinases: IUPHAR Review 14</span>. <i>Br. J. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>172</i></span>,  <span class="NLM_fpage">2675</span>– <span class="NLM_lpage">2700</span>, <span class="refDoi"> DOI: 10.1111/bph.13096</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=10.1111%2Fbph.13096" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=25630872" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtVent7zP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=172&publication_year=2015&pages=2675-2700&author=D.+Fabbroauthor=S.+W.+Cowan-Jacobauthor=H.+Moebitz&title=Ten+things+you+should+know+about+protein+kinases%3A+IUPHAR+Review+14&doi=10.1111%2Fbph.13096"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Ten things you should know about protein kinases: IUPHAR Review 14</span></div><div class="casAuthors">Fabbro, Doriano; Cowan-Jacob, Sandra W.; Moebitz, Henrik</div><div class="citationInfo"><span class="NLM_cas:title">British Journal of Pharmacology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">172</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2675-2700</span>CODEN:
                <span class="NLM_cas:coden">BJPCBM</span>;
        ISSN:<span class="NLM_cas:issn">1476-5381</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">Many human malignancies are assocd. with aberrant regulation of protein or lipid kinases due to mutations, chromosomal rearrangements and/or gene amplification.  Protein and lipid kinases represent an important target class for treating human disorders.  This review focus on 'the 10 things you should know about protein kinases and their inhibitors', including a short introduction on the history of protein kinases and their inhibitors and ending with a perspective on kinase drug discovery.  Although the '10 things' have been, to a certain extent, chosen arbitrarily, they cover in a comprehensive way the past and present efforts in kinase drug discovery and summarize the status quo of the current kinase inhibitors as well as knowledge about kinase structure and binding modes.  Besides describing the potentials of protein kinase inhibitors as drugs, this review also focus on their limitations, particularly on how to circumvent emerging resistance against kinase inhibitors in oncol. indications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGryyzn-icN30rVg90H21EOLACvtfcHk0ljuEVnpPxZ10g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtVent7zP&md5=160b754efc7f29460712fa03bb2274b1</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1111%2Fbph.13096&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fbph.13096%26sid%3Dliteratum%253Aachs%26aulast%3DFabbro%26aufirst%3DD.%26aulast%3DCowan-Jacob%26aufirst%3DS.%2BW.%26aulast%3DMoebitz%26aufirst%3DH.%26atitle%3DTen%2520things%2520you%2520should%2520know%2520about%2520protein%2520kinases%253A%2520IUPHAR%2520Review%252014%26jtitle%3DBr.%2520J.%2520Pharmacol.%26date%3D2015%26volume%3D172%26spage%3D2675%26epage%3D2700%26doi%3D10.1111%2Fbph.13096" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span> <span> </span><span class="NLM_article-title">The (un)targeted cancer kinome</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">166</span>– <span class="NLM_lpage">169</span>, <span class="refDoi"> DOI: 10.1038/nchembio.297</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=10.1038%2Fnchembio.297" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=20154661" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhvFGntbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2010&pages=166-169&author=O.+Fedorovauthor=S.+M%C3%BCllerauthor=S.+Knapp&title=The+%28un%29targeted+cancer+kinome&doi=10.1038%2Fnchembio.297"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">The (un)targeted cancer kinome</span></div><div class="casAuthors">Fedorov, Oleg; Mueller, Susanne; Knapp, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">166-169</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review and discussion.  The complexity of cancer signaling and the resulting difficulties in target selection have strongly biased kinase drug discovery towards clin. validated targets.  Recently, novel kinase targets that are uncharacterized have emerged from genome sequencing and RNAi studies.  Chem. probes are urgently needed to functionally annotate these kinases and to stimulate new drug discovery efforts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpwcnq0-qjPirVg90H21EOLACvtfcHk0ljyoP1X8iBNpg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhvFGntbk%253D&md5=198ae8d39d40247e17aef5e9de0f2a99</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.297&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.297%26sid%3Dliteratum%253Aachs%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DM%25C3%25BCller%26aufirst%3DS.%26aulast%3DKnapp%26aufirst%3DS.%26atitle%3DThe%2520%2528un%2529targeted%2520cancer%2520kinome%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2010%26volume%3D6%26spage%3D166%26epage%3D169%26doi%3D10.1038%2Fnchembio.297" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Klaeger, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heinzlmeir, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilhelm, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Polzer, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vick, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koenig, P.-A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reinecke, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruprecht, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petzoldt, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meng, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zecha, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reiter, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qiao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Helm, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koch, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schoof, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Canevari, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Casale, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Depaolini, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feuchtinger, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmidt, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rueckert, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Becker, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huenges, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garz, A.-K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gohlke, B.-O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zolg, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kayser, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vooder, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Preissner, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hahne, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tonisson, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kramer, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gotze, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bassermann, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schlegl, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ehrlich, H.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aiche, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walch, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greif, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schneider, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Felder, E. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruland, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Medard, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeremias, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spiekermann, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuster, B.</span></span> <span> </span><span class="NLM_article-title">The target landscape of clinical kinase drugs</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>358</i></span>, <span class="NLM_elocation-id">eaan4368</span> <span class="refDoi"> DOI: 10.1126/science.aan4368</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=10.1126%2Fscience.aan4368" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=29191878" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=358&publication_year=2017&author=S.+Klaegerauthor=S.+Heinzlmeirauthor=M.+Wilhelmauthor=H.+Polzerauthor=B.+Vickauthor=P.-A.+Koenigauthor=M.+Reineckeauthor=B.+Ruprechtauthor=S.+Petzoldtauthor=C.+Mengauthor=J.+Zechaauthor=K.+Reiterauthor=H.+Qiaoauthor=D.+Helmauthor=H.+Kochauthor=M.+Schoofauthor=G.+Canevariauthor=E.+Casaleauthor=S.+R.+Depaoliniauthor=A.+Feuchtingerauthor=Z.+Wuauthor=T.+Schmidtauthor=L.+Rueckertauthor=W.+Beckerauthor=J.+Huengesauthor=A.-K.+Garzauthor=B.-O.+Gohlkeauthor=D.+P.+Zolgauthor=G.+Kayserauthor=T.+Vooderauthor=R.+Preissnerauthor=H.+Hahneauthor=N.+Tonissonauthor=K.+Kramerauthor=K.+Gotzeauthor=F.+Bassermannauthor=J.+Schleglauthor=H.-C.+Ehrlichauthor=S.+Aicheauthor=A.+Walchauthor=P.+A.+Greifauthor=S.+Schneiderauthor=E.+R.+Felderauthor=J.+Rulandauthor=G.+Medardauthor=I.+Jeremiasauthor=K.+Spiekermannauthor=B.+Kuster&title=The+target+landscape+of+clinical+kinase+drugs&doi=10.1126%2Fscience.aan4368"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1126%2Fscience.aan4368&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.aan4368%26sid%3Dliteratum%253Aachs%26aulast%3DKlaeger%26aufirst%3DS.%26aulast%3DHeinzlmeir%26aufirst%3DS.%26aulast%3DWilhelm%26aufirst%3DM.%26aulast%3DPolzer%26aufirst%3DH.%26aulast%3DVick%26aufirst%3DB.%26aulast%3DKoenig%26aufirst%3DP.-A.%26aulast%3DReinecke%26aufirst%3DM.%26aulast%3DRuprecht%26aufirst%3DB.%26aulast%3DPetzoldt%26aufirst%3DS.%26aulast%3DMeng%26aufirst%3DC.%26aulast%3DZecha%26aufirst%3DJ.%26aulast%3DReiter%26aufirst%3DK.%26aulast%3DQiao%26aufirst%3DH.%26aulast%3DHelm%26aufirst%3DD.%26aulast%3DKoch%26aufirst%3DH.%26aulast%3DSchoof%26aufirst%3DM.%26aulast%3DCanevari%26aufirst%3DG.%26aulast%3DCasale%26aufirst%3DE.%26aulast%3DDepaolini%26aufirst%3DS.%2BR.%26aulast%3DFeuchtinger%26aufirst%3DA.%26aulast%3DWu%26aufirst%3DZ.%26aulast%3DSchmidt%26aufirst%3DT.%26aulast%3DRueckert%26aufirst%3DL.%26aulast%3DBecker%26aufirst%3DW.%26aulast%3DHuenges%26aufirst%3DJ.%26aulast%3DGarz%26aufirst%3DA.-K.%26aulast%3DGohlke%26aufirst%3DB.-O.%26aulast%3DZolg%26aufirst%3DD.%2BP.%26aulast%3DKayser%26aufirst%3DG.%26aulast%3DVooder%26aufirst%3DT.%26aulast%3DPreissner%26aufirst%3DR.%26aulast%3DHahne%26aufirst%3DH.%26aulast%3DTonisson%26aufirst%3DN.%26aulast%3DKramer%26aufirst%3DK.%26aulast%3DGotze%26aufirst%3DK.%26aulast%3DBassermann%26aufirst%3DF.%26aulast%3DSchlegl%26aufirst%3DJ.%26aulast%3DEhrlich%26aufirst%3DH.-C.%26aulast%3DAiche%26aufirst%3DS.%26aulast%3DWalch%26aufirst%3DA.%26aulast%3DGreif%26aufirst%3DP.%2BA.%26aulast%3DSchneider%26aufirst%3DS.%26aulast%3DFelder%26aufirst%3DE.%2BR.%26aulast%3DRuland%26aufirst%3DJ.%26aulast%3DMedard%26aufirst%3DG.%26aulast%3DJeremias%26aufirst%3DI.%26aulast%3DSpiekermann%26aufirst%3DK.%26aulast%3DKuster%26aufirst%3DB.%26atitle%3DThe%2520target%2520landscape%2520of%2520clinical%2520kinase%2520drugs%26jtitle%3DScience%26date%3D2017%26volume%3D358%26doi%3D10.1126%2Fscience.aan4368" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Knight, Z. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shokat, K. M.</span></span> <span> </span><span class="NLM_article-title">Targeting the cancer kinome through polypharmacology</span>. <i>Nat. Rev. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">130</span>, <span class="refDoi"> DOI: 10.1038/nrc2787</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=10.1038%2Fnrc2787" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=20094047" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=1%3ACAS%3A528%3ADC%252BC3cXps1Clug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2010&pages=130&author=Z.+A.+Knightauthor=H.+Linauthor=K.+M.+Shokat&title=Targeting+the+cancer+kinome+through+polypharmacology&doi=10.1038%2Fnrc2787"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting the cancer kinome through polypharmacology</span></div><div class="casAuthors">Knight, Zachary A.; Lin, Henry; Shokat, Kevan M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">130-137</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Kinase inhibitors are the largest class of new cancer drugs.  However, it is already apparent that most tumors can escape from the inhibition of any single kinase.  If it is necessary to inhibit multiple kinases, how do we choose which ones.  In this Opinion article, we discuss some of the strategies that are currently being used to identify new therapeutic combinations of kinase targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpkDv2FeeH58rVg90H21EOLACvtfcHk0lgGutwOypXovA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXps1Clug%253D%253D&md5=9aaa314e54b43889e147a9f11ac77072</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1038%2Fnrc2787&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc2787%26sid%3Dliteratum%253Aachs%26aulast%3DKnight%26aufirst%3DZ.%2BA.%26aulast%3DLin%26aufirst%3DH.%26aulast%3DShokat%26aufirst%3DK.%2BM.%26atitle%3DTargeting%2520the%2520cancer%2520kinome%2520through%2520polypharmacology%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2010%26volume%3D10%26spage%3D130%26doi%3D10.1038%2Fnrc2787" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Goldstein, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zarrinkar, P. P.</span></span> <span> </span><span class="NLM_article-title">High-throughput kinase profiling as a platform for drug discovery</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">391</span>, <span class="refDoi"> DOI: 10.1038/nrd2541</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=10.1038%2Fnrd2541" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=18404149" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=1%3ACAS%3A528%3ADC%252BD1cXlt1agtbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2008&pages=391&author=D.+M.+Goldsteinauthor=N.+S.+Grayauthor=P.+P.+Zarrinkar&title=High-throughput+kinase+profiling+as+a+platform+for+drug+discovery&doi=10.1038%2Fnrd2541"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">High-throughput kinase profiling as a platform for drug discovery</span></div><div class="casAuthors">Goldstein, David M.; Gray, Nathanael S.; Zarrinkar, Patrick P.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">391-397</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  High-throughput profiling of compd. libraries against large panels of kinases is becoming tech. feasible.  In contrast to the traditional linear, target-centric approach to discovery, this approach may provide a choice of targets to pursue that is guided by the quality of lead compds. available, rather than by target biol. alone, and could thereby significantly improve the productivity of kinase inhibitor discovery.  To fully exploit the potential of kinases as drug targets, novel strategies for the efficient discovery of inhibitors are required.  In contrast to the traditional, linear process of inhibitor discovery, high-throughput kinase profiling enables a parallel approach by interrogating compds. against hundreds of targets in a single screen.  Compd. potency and selectivity are detd. simultaneously, providing a choice of targets to pursue that is guided by the quality of lead compds. available, rather than by target biol. alone.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpwnJ59cVxbJbVg90H21EOLACvtfcHk0lgGutwOypXovA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXlt1agtbs%253D&md5=eaa61fd2958ff04fecf86a993f91833d</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1038%2Fnrd2541&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd2541%26sid%3Dliteratum%253Aachs%26aulast%3DGoldstein%26aufirst%3DD.%2BM.%26aulast%3DGray%26aufirst%3DN.%2BS.%26aulast%3DZarrinkar%26aufirst%3DP.%2BP.%26atitle%3DHigh-throughput%2520kinase%2520profiling%2520as%2520a%2520platform%2520for%2520drug%2520discovery%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2008%26volume%3D7%26spage%3D391%26doi%3D10.1038%2Fnrd2541" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Niijima, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shiraishi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Okuno, Y.</span></span> <span> </span><span class="NLM_article-title">Dissecting kinase profiling data to predict activity and understand cross-reactivity of kinase inhibitors</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>52</i></span>,  <span class="NLM_fpage">901</span>– <span class="NLM_lpage">912</span>, <span class="refDoi"> DOI: 10.1021/ci200607f</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci200607f" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=1%3ACAS%3A528%3ADC%252BC38Xjs1Clsbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2012&pages=901-912&author=S.+Niijimaauthor=A.+Shiraishiauthor=Y.+Okuno&title=Dissecting+kinase+profiling+data+to+predict+activity+and+understand+cross-reactivity+of+kinase+inhibitors&doi=10.1021%2Fci200607f"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Dissecting Kinase Profiling Data to Predict Activity and Understand Cross-Reactivity of Kinase Inhibitors</span></div><div class="casAuthors">Niijima, Satoshi; Shiraishi, Akira; Okuno, Yasushi</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">901-912</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The development of selective and multitargeted kinase inhibitors has received much attention, because cross-reactivity with unintended targets may cause toxic side effects, while it can also give rise to efficacious multitargeted drugs.  Here we describe a deconvolution approach to dissecting kinase profiling data in order to gain knowledge about cross-reactivity of inhibitors from large-scale profiling data.  This approach not only enables activity predictions of given compds. on a kinome-wide scale, but also allows to ext. residue-fragment pairs that are assocd. with activity.  We demonstrate its effectiveness using a large-scale public chemogenomics data set and also apply our proposed model to a recently published bioactivity data set.  We further illustrate that the preference of given compds. for kinases of interest is better understood by residue-fragment pairs, which could provide both biol. and chem. insights into cross-reactivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpNgeRgAJen8LVg90H21EOLACvtfcHk0lgV02j8lgwWeA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xjs1Clsbo%253D&md5=5b3d8d0938f8109e21d17a3fb3c9da30</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1021%2Fci200607f&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci200607f%26sid%3Dliteratum%253Aachs%26aulast%3DNiijima%26aufirst%3DS.%26aulast%3DShiraishi%26aufirst%3DA.%26aulast%3DOkuno%26aufirst%3DY.%26atitle%3DDissecting%2520kinase%2520profiling%2520data%2520to%2520predict%2520activity%2520and%2520understand%2520cross-reactivity%2520of%2520kinase%2520inhibitors%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2012%26volume%3D52%26spage%3D901%26epage%3D912%26doi%3D10.1021%2Fci200607f" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lapins, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wikberg, J. E.</span></span> <span> </span><span class="NLM_article-title">Kinome-wide interaction modelling using alignment-based and alignment-independent approaches for kinase description and linear and non-linear data analysis techniques</span>. <i>BMC Bioinf.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">339</span>– <span class="NLM_lpage">339</span>, <span class="refDoi"> DOI: 10.1186/1471-2105-11-339</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=10.1186%2F1471-2105-11-339" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=20569422" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=1%3ACAS%3A280%3ADC%252BC3cjgtFekug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2010&pages=339-339&author=M.+Lapinsauthor=J.+E.+Wikberg&title=Kinome-wide+interaction+modelling+using+alignment-based+and+alignment-independent+approaches+for+kinase+description+and+linear+and+non-linear+data+analysis+techniques&doi=10.1186%2F1471-2105-11-339"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Kinome-wide interaction modelling using alignment-based and alignment-independent approaches for kinase description and linear and non-linear data analysis techniques</span></div><div class="casAuthors">Lapins Maris; Wikberg Jarl Es</div><div class="citationInfo"><span class="NLM_cas:title">BMC bioinformatics</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">339</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  Protein kinases play crucial roles in cell growth, differentiation, and apoptosis.  Abnormal function of protein kinases can lead to many serious diseases, such as cancer.  Kinase inhibitors have potential for treatment of these diseases.  However, current inhibitors interact with a broad variety of kinases and interfere with multiple vital cellular processes, which causes toxic effects.  Bioinformatics approaches that can predict inhibitor-kinase interactions from the chemical properties of the inhibitors and the kinase macromolecules might aid in design of more selective therapeutic agents, that show better efficacy and lower toxicity.  RESULTS:  We applied proteochemometric modelling to correlate the properties of 317 wild-type and mutated kinases and 38 inhibitors (12,046 inhibitor-kinase combinations) to the respective combination's interaction dissociation constant (Kd).  We compared six approaches for description of protein kinases and several linear and non-linear correlation methods.  The best performing models encoded kinase sequences with amino acid physico-chemical z-scale descriptors and used support vector machines or partial least- squares projections to latent structures for the correlations.  Modelling performance was estimated by double cross-validation.  The best models showed high predictive ability; the squared correlation coefficient for new kinase-inhibitor pairs ranging P2 = 0.67-0.73; for new kinases it ranged P2kin = 0.65-0.70.  Models could also separate interacting from non-interacting inhibitor-kinase pairs with high sensitivity and specificity; the areas under the ROC curves ranging AUC = 0.92-0.93.  We also investigated the relationship between the number of protein kinases in the dataset and the modelling results.  Using only 10% of all data still a valid model was obtained with P2 = 0.47, P2kin = 0.42 and AUC = 0.83.  CONCLUSIONS:  Our results strongly support the applicability of proteochemometrics for kinome-wide interaction modelling.  Proteochemometrics might be used to speed-up identification and optimization of protein kinase targeted and multi-targeted inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSQqQwOlHM3l0EksK8CslK8fW6udTcc2ebRySsAbiA6Yrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3cjgtFekug%253D%253D&md5=5bde7177dce4f3c5e0c26a89c7fc4d03</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1186%2F1471-2105-11-339&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1471-2105-11-339%26sid%3Dliteratum%253Aachs%26aulast%3DLapins%26aufirst%3DM.%26aulast%3DWikberg%26aufirst%3DJ.%2BE.%26atitle%3DKinome-wide%2520interaction%2520modelling%2520using%2520alignment-based%2520and%2520alignment-independent%2520approaches%2520for%2520kinase%2520description%2520and%2520linear%2520and%2520non-linear%2520data%2520analysis%2520techniques%26jtitle%3DBMC%2520Bioinf.%26date%3D2010%26volume%3D11%26spage%3D339%26epage%3D339%26doi%3D10.1186%2F1471-2105-11-339" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schurer, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muskal, S. M.</span></span> <span> </span><span class="NLM_article-title">Kinome-wide activity modeling from diverse public high-quality data sets</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">27</span>– <span class="NLM_lpage">38</span>, <span class="refDoi"> DOI: 10.1021/ci300403k</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci300403k" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvVygu77L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2013&pages=27-38&author=S.+C.+Schurerauthor=S.+M.+Muskal&title=Kinome-wide+activity+modeling+from+diverse+public+high-quality+data+sets&doi=10.1021%2Fci300403k"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Kinome-wide Activity Modeling from Diverse Public High-Quality Data Sets</span></div><div class="casAuthors">Schurer, Stephan C.; Muskal, Steven M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">27-38</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Large corpora of kinase small mol. inhibitor data are accessible to public sector research from thousands of journal article and patent publications.  These data have been generated employing a wide variety of assay methodologies and exptl. procedures by numerous labs.  Here we ask the question how applicable these heterogeneous data sets are to predict kinase activities and which characteristics of the data sets contribute to their utility.  We accessed almost 500 000 mols. from the Kinase Knowledge Base (KKB) and after rigorous aggregation and standardization generated over 180 distinct data sets covering all major groups of the human kinome.  To assess the value of the data sets, we generated hundreds of classification and regression models.  Their rigorous cross-validation and characterization demonstrated highly predictive classification and quant. models for the majority of kinase targets if a min. required no. of active compds. or structure-activity data points were available.  We then applied the best classifiers to compds. most recently profiled in the NIH Library of Integrated Network-based Cellular Signatures (LINCS) program and found good agreement of profiling results with predicted activities.  Our results indicate that, although heterogeneous in nature, the publically accessible data sets are exceedingly valuable and well suited to develop highly accurate predictors for practical Kinome-wide virtual screening applications and to complement exptl. kinase profiling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr1rhwNqRlmD7Vg90H21EOLACvtfcHk0lj6_rfMA31RQA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvVygu77L&md5=cea0a7b0c22b43e7660b430e8039f4e1</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1021%2Fci300403k&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci300403k%26sid%3Dliteratum%253Aachs%26aulast%3DSchurer%26aufirst%3DS.%2BC.%26aulast%3DMuskal%26aufirst%3DS.%2BM.%26atitle%3DKinome-wide%2520activity%2520modeling%2520from%2520diverse%2520public%2520high-quality%2520data%2520sets%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2013%26volume%3D53%26spage%3D27%26epage%3D38%26doi%3D10.1021%2Fci300403k" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cao, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, G. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Q. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, Y. Z.</span></span> <span> </span><span class="NLM_article-title">Large-scale prediction of human kinase–inhibitor interactions using protein sequences and molecular topological structures</span>. <i>Anal. Chim. Acta</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>792</i></span>,  <span class="NLM_fpage">10</span>– <span class="NLM_lpage">18</span>, <span class="refDoi"> DOI: 10.1016/j.aca.2013.07.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=10.1016%2Fj.aca.2013.07.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=23910962" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtFOhsrjF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=792&publication_year=2013&pages=10-18&author=D.+S.+Caoauthor=G.+H.+Zhouauthor=S.+Liuauthor=L.+X.+Zhangauthor=Q.+S.+Xuauthor=M.+Heauthor=Y.+Z.+Liang&title=Large-scale+prediction+of+human+kinase%E2%80%93inhibitor+interactions+using+protein+sequences+and+molecular+topological+structures&doi=10.1016%2Fj.aca.2013.07.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Large-scale prediction of human kinase-inhibitor interactions using protein sequences and molecular topological structures</span></div><div class="casAuthors">Cao, Dong-Sheng; Zhou, Guang-Hua; Liu, Shao; Zhang, Liu-Xia; Xu, Qing-Song; He, Min; Liang, Yi-Zeng</div><div class="citationInfo"><span class="NLM_cas:title">Analytica Chimica Acta</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">792</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">10-18</span>CODEN:
                <span class="NLM_cas:coden">ACACAM</span>;
        ISSN:<span class="NLM_cas:issn">0003-2670</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The kinase family is one of the largest target families in the human genome.  The family's key function in signal transduction for all organisms makes it a very attractive target class for the therapeutic interventions in many diseases states such as cancer, diabetes, inflammation and arthritis.  A first step toward accelerating kinase drug discovery process is to fast identify whether a chem. and a kinase interact or not.  Exptl., these interactions can be identified by in vitro binding assay - an expensive and laborious procedure that is not applicable on a large scale.  Therefore, there is an urgent need to develop statistically efficient approaches for identifying kinase-inhibitor interactions.  For the first time, the quant. binding affinities of kinase-inhibitor pairs are differentiated as a measurement to define if an inhibitor interacts with a kinase, and then a chemogenomics framework using an unbiased set of general integrated features (drug descriptors and protein descriptors) and random forest (RF) is employed to construct a predictive model which can accurately classify kinase-inhibitor pairs.  Our results show that RF with integrated features gave prediction accuracy of 93.76%, sensitivity of 92.26%, and specificity of 95.27%, resp.  The results are superior to those by only considering two sepd. spaces (chem. space and protein space), demonstrating that these integrated features contribute cooperatively.  Based on the constructed model, we provided a high confidence list of drug-target assocns. for subsequent exptl. investigation guidance at a low false discovery rate.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqSkjxlh2x_9bVg90H21EOLACvtfcHk0lj6_rfMA31RQA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtFOhsrjF&md5=d43c3262b9f166729bcfc418cc8bc3be</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1016%2Fj.aca.2013.07.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.aca.2013.07.003%26sid%3Dliteratum%253Aachs%26aulast%3DCao%26aufirst%3DD.%2BS.%26aulast%3DZhou%26aufirst%3DG.%2BH.%26aulast%3DLiu%26aufirst%3DS.%26aulast%3DZhang%26aufirst%3DL.%2BX.%26aulast%3DXu%26aufirst%3DQ.%2BS.%26aulast%3DHe%26aufirst%3DM.%26aulast%3DLiang%26aufirst%3DY.%2BZ.%26atitle%3DLarge-scale%2520prediction%2520of%2520human%2520kinase%25E2%2580%2593inhibitor%2520interactions%2520using%2520protein%2520sequences%2520and%2520molecular%2520topological%2520structures%26jtitle%3DAnal.%2520Chim.%2520Acta%26date%3D2013%26volume%3D792%26spage%3D10%26epage%3D18%26doi%3D10.1016%2Fj.aca.2013.07.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Merget, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turk, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eid, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rippmann, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fulle, S.</span></span> <span> </span><span class="NLM_article-title">Profiling prediction of kinase inhibitors: toward the virtual assay</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">474</span>– <span class="NLM_lpage">485</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b01611</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01611" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvFWgtLjM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=474-485&author=B.+Mergetauthor=S.+Turkauthor=S.+Eidauthor=F.+Rippmannauthor=S.+Fulle&title=Profiling+prediction+of+kinase+inhibitors%3A+toward+the+virtual+assay&doi=10.1021%2Facs.jmedchem.6b01611"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Profiling Prediction of Kinase Inhibitors: Toward the Virtual Assay</span></div><div class="casAuthors">Merget, Benjamin; Turk, Samo; Eid, Sameh; Rippmann, Friedrich; Fulle, Simone</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">474-485</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Kinome-wide screening would have the advantage of providing structure-activity relationships against hundreds of targets simultaneously.  Here, the authors report the generation of ligand-based activity prediction models for over 280 kinases by employing Machine Learning methods on an extensive data set of proprietary bioactivity data combined with open data.  High quality (AUC >0.7) was achieved for ∼200 kinases by combining open with proprietary data, choosing Random Forest over alternative tested Machine Learning methods, and balancing the training data sets.  Tests on left-out and external data indicate a high value for virtual screening projects.  Importantly, the derived models are evenly distributed across the kinome tree, allowing reliable profiling prediction for all kinase branches.  The prediction quality was further improved by employing exptl. bioactivity fingerprints of a small kinase subset.  Overall, the generated models can support various hit identification tasks, including virtual screening, compd. repurposing and the detection of potential off-targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoPeLeyo4D6IrVg90H21EOLACvtfcHk0licmnfxSN2aqw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvFWgtLjM&md5=2e0ad30702bcf11bed7cb3ec30c774c3</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01611%26sid%3Dliteratum%253Aachs%26aulast%3DMerget%26aufirst%3DB.%26aulast%3DTurk%26aufirst%3DS.%26aulast%3DEid%26aufirst%3DS.%26aulast%3DRippmann%26aufirst%3DF.%26aulast%3DFulle%26aufirst%3DS.%26atitle%3DProfiling%2520prediction%2520of%2520kinase%2520inhibitors%253A%2520toward%2520the%2520virtual%2520assay%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D474%26epage%3D485%26doi%3D10.1021%2Facs.jmedchem.6b01611" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bora, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Avram, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciucanu, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raica, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Avram, S.</span></span> <span> </span><span class="NLM_article-title">Predictive models for fast and effective profiling of kinase inhibitors</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">895</span>– <span class="NLM_lpage">905</span>, <span class="refDoi"> DOI: 10.1021/acs.jcim.5b00646</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.5b00646" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=1%3ACAS%3A528%3ADC%252BC28Xlslyqtr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2016&pages=895-905&author=A.+Boraauthor=S.+Avramauthor=I.+Ciucanuauthor=M.+Raicaauthor=S.+Avram&title=Predictive+models+for+fast+and+effective+profiling+of+kinase+inhibitors&doi=10.1021%2Facs.jcim.5b00646"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Predictive Models for Fast and Effective Profiling of Kinase Inhibitors</span></div><div class="casAuthors">Bora, Alina; Avram, Sorin; Ciucanu, Ionel; Raica, Marius; Avram, Stefana</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">895-905</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">In this study we developed two-dimensional pharmacophore-based random forest models for the effective profiling of kinase inhibitors.  One hundred seven prediction models were developed to address distinct kinases spanning over all kinase groups.  Rigorous external validation demonstrates excellent virtual screening and classification potential of the predictors and, more importantly, the capacity to prioritize novel chem. scaffolds in large chem. libraries.  The models built upon more diverse and more potent compds. tend to exert the highest predictive power.  The anal. of ColBioS-FlavRC (Collection of Bioselective Flavonoids and Related Compds.) highlighted several potentially promiscuous derivs. with undesirable selectivity against kinases.  The prediction models can be downloaded from www.chembioinf.ro.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqTZSs1GEkYHLVg90H21EOLACvtfcHk0licmnfxSN2aqw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xlslyqtr4%253D&md5=cd8d17ea9616165b115dd9d75352e0ac</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.5b00646&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.5b00646%26sid%3Dliteratum%253Aachs%26aulast%3DBora%26aufirst%3DA.%26aulast%3DAvram%26aufirst%3DS.%26aulast%3DCiucanu%26aufirst%3DI.%26aulast%3DRaica%26aufirst%3DM.%26aulast%3DAvram%26aufirst%3DS.%26atitle%3DPredictive%2520models%2520for%2520fast%2520and%2520effective%2520profiling%2520of%2520kinase%2520inhibitors%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2016%26volume%3D56%26spage%3D895%26epage%3D905%26doi%3D10.1021%2Facs.jcim.5b00646" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yabuuchi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Niijima, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takematsu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ida, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hirokawa, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hara, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ogawa, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minowa, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsujimoto, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Okuno, Y.</span></span> <span> </span><span class="NLM_article-title">Analysis of multiple compound–protein interactions reveals novel bioactive molecules</span>. <i>Mol. Syst. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">472</span>, <span class="refDoi"> DOI: 10.1038/msb.2011.5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=10.1038%2Fmsb.2011.5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=21364574" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=1%3ACAS%3A528%3ADC%252BC3MXltFShsbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2011&pages=472&author=H.+Yabuuchiauthor=S.+Niijimaauthor=H.+Takematsuauthor=T.+Idaauthor=T.+Hirokawaauthor=T.+Haraauthor=T.+Ogawaauthor=Y.+Minowaauthor=G.+Tsujimotoauthor=Y.+Okuno&title=Analysis+of+multiple+compound%E2%80%93protein+interactions+reveals+novel+bioactive+molecules&doi=10.1038%2Fmsb.2011.5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Analysis of multiple compound-protein interactions reveals novel bioactive molecules</span></div><div class="casAuthors">Yabuuchi, Hiroaki; Niijima, Satoshi; Takematsu, Hiromu; Ida, Tomomi; Hirokawa, Takatsugu; Hara, Takafumi; Ogawa, Teppei; Minowa, Yohsuke; Tsujimoto, Gozoh; Okuno, Yasushi</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Systems Biology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">472</span>CODEN:
                <span class="NLM_cas:coden">MSBOC3</span>;
        ISSN:<span class="NLM_cas:issn">1744-4292</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The discovery of novel bioactive mols. advances our systems-level understanding of biol. processes and is crucial for innovation in drug development.  For this purpose, the emerging field of chem. genomics is currently focused on accumulating large assay data sets describing compd.-protein interactions (CPIs).  Although new target proteins for known drugs have recently been identified through mining of CPI databases, using these resources to identify novel ligands remains unexplored.  Herein, we demonstrate that machine learning of multiple CPIs can not only assess drug polypharmacol. but can also efficiently identify novel bioactive scaffold-hopping compds.  Through a machine-learning technique that uses multiple CPIs, we have successfully identified novel lead compds. for two pharmaceutically important protein families, G-protein coupled receptors and protein kinases.  These novel compds. were not identified by existing computational ligand-screening methods in comparative studies.  The results of this study indicate that data derived from chem. genomics can be highly useful for exploring chem. space, and this systems biol. perspective could accelerate drug discovery processes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqh8pIgfxm7oLVg90H21EOLACvtfcHk0lggq0Z4N2NzFg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXltFShsbY%253D&md5=bc0caba6894d46de383042fa85accefa</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1038%2Fmsb.2011.5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fmsb.2011.5%26sid%3Dliteratum%253Aachs%26aulast%3DYabuuchi%26aufirst%3DH.%26aulast%3DNiijima%26aufirst%3DS.%26aulast%3DTakematsu%26aufirst%3DH.%26aulast%3DIda%26aufirst%3DT.%26aulast%3DHirokawa%26aufirst%3DT.%26aulast%3DHara%26aufirst%3DT.%26aulast%3DOgawa%26aufirst%3DT.%26aulast%3DMinowa%26aufirst%3DY.%26aulast%3DTsujimoto%26aufirst%3DG.%26aulast%3DOkuno%26aufirst%3DY.%26atitle%3DAnalysis%2520of%2520multiple%2520compound%25E2%2580%2593protein%2520interactions%2520reveals%2520novel%2520bioactive%2520molecules%26jtitle%3DMol.%2520Syst.%2520Biol.%26date%3D2011%26volume%3D7%26spage%3D472%26doi%3D10.1038%2Fmsb.2011.5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Manallack, D. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pitt, W. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gancia, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Montana, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Livingstone, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ford, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whitley, D. C.</span></span> <span> </span><span class="NLM_article-title">Selecting screening candidates for kinase and G protein-coupled receptor targets using neural networks</span>. <i>J. Chem. Inf. Comput. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>42</i></span>,  <span class="NLM_fpage">1256</span>– <span class="NLM_lpage">1262</span>, <span class="refDoi"> DOI: 10.1021/ci020267c</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci020267c" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=1%3ACAS%3A528%3ADC%252BD38XlvFSmtrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2002&pages=1256-1262&author=D.+T.+Manallackauthor=W.+R.+Pittauthor=E.+Ganciaauthor=J.+G.+Montanaauthor=D.+J.+Livingstoneauthor=M.+G.+Fordauthor=D.+C.+Whitley&title=Selecting+screening+candidates+for+kinase+and+G+protein-coupled+receptor+targets+using+neural+networks&doi=10.1021%2Fci020267c"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Selecting Screening Candidates for Kinase and G Protein-Coupled Receptor Targets Using Neural Networks</span></div><div class="casAuthors">Manallack, David T.; Pitt, Will R.; Gancia, Emanuela; Montana, John G.; Livingstone, David J.; Ford, Martyn G.; Whitley, David C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Computer Sciences</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1256-1262</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">0095-2338</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A series of neural networks has been trained, using consensus methods, to recognize compds. that act at biol. targets belonging to specific gene families.  The MDDR database was used to provide compds. targeted against gene families and sets of randomly selected mols.  BCUT parameters were employed as input descriptors that encode structural properties and information relevant to ligand-receptor interactions.  In each case, the networks identified over 80% of the compds. targeting a gene family.  The technique was applied to purchasing compds. from external suppliers, and results from screening against one gene family demonstrated impressive abilities to predict the activity of the majority of known hit compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp3B0ertK2pfbVg90H21EOLACvtfcHk0lggq0Z4N2NzFg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XlvFSmtrg%253D&md5=264cc5d5bc3657018ca0bd232954765c</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1021%2Fci020267c&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci020267c%26sid%3Dliteratum%253Aachs%26aulast%3DManallack%26aufirst%3DD.%2BT.%26aulast%3DPitt%26aufirst%3DW.%2BR.%26aulast%3DGancia%26aufirst%3DE.%26aulast%3DMontana%26aufirst%3DJ.%2BG.%26aulast%3DLivingstone%26aufirst%3DD.%2BJ.%26aulast%3DFord%26aufirst%3DM.%2BG.%26aulast%3DWhitley%26aufirst%3DD.%2BC.%26atitle%3DSelecting%2520screening%2520candidates%2520for%2520kinase%2520and%2520G%2520protein-coupled%2520receptor%2520targets%2520using%2520neural%2520networks%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Comput.%2520Sci.%26date%3D2002%26volume%3D42%26spage%3D1256%26epage%3D1262%26doi%3D10.1021%2Fci020267c" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Subramanian, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramsundar, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pande, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Denny, R. A.</span></span> <span> </span><span class="NLM_article-title">Computational modeling of β-secretase 1 (BACE-1) inhibitors using ligand based approaches</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">1936</span>– <span class="NLM_lpage">1949</span>, <span class="refDoi"> DOI: 10.1021/acs.jcim.6b00290</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.6b00290" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsF2nu73E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2016&pages=1936-1949&author=G.+Subramanianauthor=B.+Ramsundarauthor=V.+Pandeauthor=R.+A.+Denny&title=Computational+modeling+of+%CE%B2-secretase+1+%28BACE-1%29+inhibitors+using+ligand+based+approaches&doi=10.1021%2Facs.jcim.6b00290"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Computational Modeling of β-Secretase 1 (BACE-1) Inhibitors Using Ligand Based Approaches</span></div><div class="casAuthors">Subramanian, Govindan; Ramsundar, Bharath; Pande, Vijay; Denny, Rajiah Aldrin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1936-1949</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The binding affinities (IC50) reported for diverse structural classes of human β-secretase 1 (BACE-1) inhibitors in literature were modeled using multiple in silico ligand based modeling approaches and statistical techniques.  The descriptor space encompasses simple binary mol. fingerprint, 1- and 2-dimensional physicochem. and topol. descriptors, and sophisticated 3-dimensional mol. fields that require appropriate structural alignments of varied chem. scaffolds in one universal chem. space.  The affinities were modeled using qual. classification or quant. regression schemes involving linear, nonlinear, and deep neural network (DNN) machine-learning methods used in the scientific literature for quant. structure activity relationships (QSAR).  In a departure from tradition, ∼20% of the chem. diverse dataset (205 compds.) was used to train the model with the remaining ∼80% of the structural and chem. analogs used as part of an external validation (1273 compds.) and prospective test (69 compds.) sets resp. to ascertain the model performance.  The machine learning methods investigated herein performed well in both the qual. classification (∼70% accuracy) and quant. IC50 predictions (RMSE ∼ 1 log).  The success of the 2D descriptor based machine learning approach when compared against the 3D field based technique pursued here provides a strong impetus for systematically applying machine-learning methods during the lead identification and optimization efforts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGorevaLuMtBsbVg90H21EOLACvtfcHk0litX1BnDWAqjQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsF2nu73E&md5=5f916f908fb0b19f4f4ba86f568e22a0</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.6b00290&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.6b00290%26sid%3Dliteratum%253Aachs%26aulast%3DSubramanian%26aufirst%3DG.%26aulast%3DRamsundar%26aufirst%3DB.%26aulast%3DPande%26aufirst%3DV.%26aulast%3DDenny%26aufirst%3DR.%2BA.%26atitle%3DComputational%2520modeling%2520of%2520%25CE%25B2-secretase%25201%2520%2528BACE-1%2529%2520inhibitors%2520using%2520ligand%2520based%2520approaches%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2016%26volume%3D56%26spage%3D1936%26epage%3D1949%26doi%3D10.1021%2Facs.jcim.6b00290" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liaw, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheridan, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Svetnik, V.</span></span> <span> </span><span class="NLM_article-title">Demystifying multitask deep neural networks for quantitative structure–activity relationships</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">2490</span>– <span class="NLM_lpage">2504</span>, <span class="refDoi"> DOI: 10.1021/acs.jcim.7b00087</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.7b00087" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsVejs7jJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2017&pages=2490-2504&author=Y.+Xuauthor=J.+Maauthor=A.+Liawauthor=R.+P.+Sheridanauthor=V.+Svetnik&title=Demystifying+multitask+deep+neural+networks+for+quantitative+structure%E2%80%93activity+relationships&doi=10.1021%2Facs.jcim.7b00087"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Demystifying Multitask Deep Neural Networks for Quantitative Structure-Activity Relationships</span></div><div class="casAuthors">Xu, Yuting; Ma, Junshui; Liaw, Andy; Sheridan, Robert P.; Svetnik, Vladimir</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2490-2504</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Deep neural networks (DNNs) are complex computational models that have found great success in many artificial intelligence applications, such as computer vision and natural language processing.  In the past four years, DNNs also generated promising results for quant. structure-activity relationship (QSAR) tasks.  Previous work showed that DNNs can routinely make better predictions than traditional methods, such as random forests, on a diverse collection of QSAR datasets.  It was also found that multi-task DNN models - those trained on and predicting multiple QSAR properties simultaneously - outperform DNNs trained sep. on the individual datasets in many but not all tasks.  Up to now there is no satisfactory explanation as to why the QSAR of one task, embedded in a multi-task DNN, can borrow information from other unrelated QSAR tasks.  Thus using multi-task DNNs in a way that consistently provides a predictive advantage becomes a challenge.  The authors explore why multi-task DNNs make a difference in predictive performance.  The authors' results show that, during prediction, a multi-task DNN does borrow ''signal'' from mols. with similar structures in the training sets of the other tasks.  However, whether this borrowing leads to better or worse predictive performance depends on whether the activities are correlated.  Based on this realization, the authors develop a strategy to use multi-task DNNs that incorporate prior domain knowledge to select training sets with correlated activities, and the authors demonstrate its effectiveness on several examples.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGruAy4BN8PqXLVg90H21EOLACvtfcHk0litX1BnDWAqjQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsVejs7jJ&md5=efb621fcdcd07b5527666412375dd7db</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.7b00087&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.7b00087%26sid%3Dliteratum%253Aachs%26aulast%3DXu%26aufirst%3DY.%26aulast%3DMa%26aufirst%3DJ.%26aulast%3DLiaw%26aufirst%3DA.%26aulast%3DSheridan%26aufirst%3DR.%2BP.%26aulast%3DSvetnik%26aufirst%3DV.%26atitle%3DDemystifying%2520multitask%2520deep%2520neural%2520networks%2520for%2520quantitative%2520structure%25E2%2580%2593activity%2520relationships%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2017%26volume%3D57%26spage%3D2490%26epage%3D2504%26doi%3D10.1021%2Facs.jcim.7b00087" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ramsundar, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verras, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tudor, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheridan, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pande, V.</span></span> <span> </span><span class="NLM_article-title">Is multitask deep learning practical for pharma?</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">2068</span>– <span class="NLM_lpage">2076</span>, <span class="refDoi"> DOI: 10.1021/acs.jcim.7b00146</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.7b00146" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtFCisr7O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2017&pages=2068-2076&author=B.+Ramsundarauthor=B.+Liuauthor=Z.+Wuauthor=A.+Verrasauthor=M.+Tudorauthor=R.+P.+Sheridanauthor=V.+Pande&title=Is+multitask+deep+learning+practical+for+pharma%3F&doi=10.1021%2Facs.jcim.7b00146"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Is Multitask Deep Learning Practical for Pharma?</span></div><div class="casAuthors">Ramsundar, Bharath; Liu, Bowen; Wu, Zhenqin; Verras, Andreas; Tudor, Matthew; Sheridan, Robert P.; Pande, Vijay</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2068-2076</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Multitask deep learning has emerged as a powerful tool for computational drug discovery.  However, despite a no. of preliminary studies, multitask deep networks have yet to be widely deployed in the pharmaceutical and biotech industries.  This lack of acceptance stems from both software difficulties and lack of understanding of the robustness of multitask deep networks.  Our work aims to resolve both of these barriers to adoption.  We introduce a high-quality open-source implementation of multitask deep networks as part of the DeepChem open-source platform.  Our implementation enables simple python scripts to construct, fit, and evaluate sophisticated deep models.  We use our implementation to analyze the performance of multitask deep networks and related deep models on four collections of pharmaceutical data (three of which have not previously been analyzed in the literature).  We split these data sets into train/valid/test using time and neighbor splits to test multitask deep learning performance under challenging conditions.  Our results demonstrate that multitask deep networks are surprisingly robust and can offer strong improvement over random forests.  Our anal. and open-source implementation in DeepChem provide an argument that multitask deep networks are ready for widespread use in com. drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpo62Pjc9GpnbVg90H21EOLACvtfcHk0lj35CpRJ45VSQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtFCisr7O&md5=3d27a01e20455d4ee4c869b7cc91717d</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.7b00146&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.7b00146%26sid%3Dliteratum%253Aachs%26aulast%3DRamsundar%26aufirst%3DB.%26aulast%3DLiu%26aufirst%3DB.%26aulast%3DWu%26aufirst%3DZ.%26aulast%3DVerras%26aufirst%3DA.%26aulast%3DTudor%26aufirst%3DM.%26aulast%3DSheridan%26aufirst%3DR.%2BP.%26aulast%3DPande%26aufirst%3DV.%26atitle%3DIs%2520multitask%2520deep%2520learning%2520practical%2520for%2520pharma%253F%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2017%26volume%3D57%26spage%3D2068%26epage%3D2076%26doi%3D10.1021%2Facs.jcim.7b00146" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ramsundar, B.</span>; <span class="NLM_string-name">Kearnes, S.</span>; <span class="NLM_string-name">Riley, P.</span>; <span class="NLM_string-name">Webster, D.</span>; <span class="NLM_string-name">Konerding, D.</span>; <span class="NLM_string-name">Pande, V.</span></span> <span> </span><span class="NLM_article-title">Massively multitask networks for drug discovery</span>.  <i>arXiv:1502.02072</i>, <span class="NLM_year">2015</span>. arXiv.org e-Print archive.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2015&author=B.+Ramsundar&author=S.+Kearnes&author=P.+Riley&author=D.+Webster&author=D.+Konerding&author=V.+Pande&title=Massively+multitask+networks+for+drug+discovery"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dsimple%26sid%3Dliteratum%253Aachs%26aulast%3DRamsundar%26aufirst%3DB.%26atitle%3DMassively%2520multitask%2520networks%2520for%2520drug%2520discovery%26jtitle%3DarXiv%253A1502.02072%26date%3D2015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, G.-W.</span></span> <span> </span><span class="NLM_article-title">Quantitative toxicity prediction using topology based multitask deep neural networks</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">520</span>– <span class="NLM_lpage">531</span>, <span class="refDoi"> DOI: 10.1021/acs.jcim.7b00558</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.7b00558" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=1%3ACAS%3A528%3ADC%252BC1cXlt1altw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2018&pages=520-531&author=K.+Wuauthor=G.-W.+Wei&title=Quantitative+toxicity+prediction+using+topology+based+multitask+deep+neural+networks&doi=10.1021%2Facs.jcim.7b00558"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Quantitative Toxicity Prediction Using Topology Based Multitask Deep Neural Networks</span></div><div class="casAuthors">Wu, Kedi; Wei, Guo-Wei</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">520-531</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The understanding of toxicity is of paramount importance to human health and environmental protection.  Quant. toxicity anal. has become a new std. in the field.  This work introduces element specific persistent homol. (ESPH), an algebraic topol. approach, for quant. toxicity prediction.  ESPH retains crucial chem. information during the topol. abstraction of geometric complexity and provides a representation of small mols. that cannot be obtained by any other method.  To investigate the representability and predictive power of ESPH for small mols., ancillary descriptors have also been developed based on phys. models.  Topol. and phys. descriptors are paired with advanced machine learning algorithms, such as deep neural network (DNN), random forest (RF) and gradient boosting decision tree (GBDT), to facilitate their applications to quant. toxicity predictions.  A topol. based multi-task strategy is proposed to take the advantage of the availability of large data sets while dealing with small data sets.  Four benchmark toxicity data sets that involve quant. measurements are used to validate the proposed approaches.  Extensive numerical studies indicate that the proposed topol. are able to outperform the state-of-the-art methods in the literature for quant. toxicity anal.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo545VJUZNFCbVg90H21EOLACvtfcHk0lj35CpRJ45VSQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXlt1altw%253D%253D&md5=1c38990135876101457e2687e18319b0</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.7b00558&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.7b00558%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DK.%26aulast%3DWei%26aufirst%3DG.-W.%26atitle%3DQuantitative%2520toxicity%2520prediction%2520using%2520topology%2520based%2520multitask%2520deep%2520neural%2520networks%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2018%26volume%3D58%26spage%3D520%26epage%3D531%26doi%3D10.1021%2Facs.jcim.7b00558" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lenselink, E. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ten Dijke, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bongers, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Papadatos, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Vlijmen, H. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kowalczyk, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ijzerman, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Westen, G. J.</span></span> <span> </span><span class="NLM_article-title">Beyond the hype: deep neural networks outperform established methods using a ChEMBL bioactivity benchmark set</span>. <i>J. Cheminf.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">45</span>, <span class="refDoi"> DOI: 10.1186/s13321-017-0232-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=10.1186%2Fs13321-017-0232-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=1%3ACAS%3A528%3ADC%252BC1cXnsVygtLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2017&pages=45&author=E.+B.+Lenselinkauthor=N.+Ten+Dijkeauthor=B.+Bongersauthor=G.+Papadatosauthor=H.+W.+Van+Vlijmenauthor=W.+Kowalczykauthor=A.+P.+Ijzermanauthor=G.+J.+Van+Westen&title=Beyond+the+hype%3A+deep+neural+networks+outperform+established+methods+using+a+ChEMBL+bioactivity+benchmark+set&doi=10.1186%2Fs13321-017-0232-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Beyond the hype: deep neural networks outperform established methods using a ChEMBL bioactivity benchmark set</span></div><div class="casAuthors">Lenselink, Eelke B.; ten Dijke, Niels; Bongers, Brandon; Papadatos, George; van Vlijmen, Herman W. T.; Kowalczyk, Wojtek; Ijzerman, Adriaan P.; van Westen, Gerard J. P.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Cheminformatics</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">45/1-45/14</span>CODEN:
                <span class="NLM_cas:coden">JCOHB3</span>;
        ISSN:<span class="NLM_cas:issn">1758-2946</span>.
    
            (<span class="NLM_cas:orgname">Chemistry Central Ltd.</span>)
        </div><div class="casAbstract">The increase of publicly available bioactivity data in recent years has fueled and catalyzed research in chemogenomics, data mining, and modeling approaches.  As a direct result, over the past few years a multitude of different methods have been reported and evaluated, such as target fishing, nearest neighbor similarity-based methods, and Quant. Structure Activity Relationship (QSAR)-based protocols.  However, such studies are typically conducted on different datasets, using different validation strategies, and different metrics.  In this study, different methods were compared using one single standardized dataset obtained from ChEMBL, which is made available to the public, using standardized metrics (BEDROC and Matthews Correlation Coeff.).  Specifically, the performance of Naive Bayes, Random Forests, Support Vector Machines, Logistic Regression, and Deep Neural Networks was assessed using QSAR and proteochemometric (PCM) methods.  All methods were validated using both a random split validation and a temporal validation, with the latter being a more realistic benchmark of expected prospective execution.  Deep Neural Networks are the top performing classifiers, highlighting the added value of Deep Neural Networks over other more conventional methods.  Moreover, the best method ('DNN_PCM') performed significantly better at almost one std. deviation higher than the mean performance.  Furthermore, Multi-task and PCM implementations were shown to improve performance over single task Deep Neural Networks.  Conversely, target prediction performed almost two std. deviations under the mean performance.  Random Forests, Support Vector Machines, and Logistic Regression performed around mean performance.  Finally, using an ensemble of DNNs, alongside addnl. tuning, enhanced the relative performance by another 27% (compared with unoptimized 'DNN_PCM').  Here, a standardized set to test and evaluate different machine learning algorithms in the context of multi-task learning is offered by providing the data and the protocols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqc4r8snR0aaLVg90H21EOLACvtfcHk0ljk9x7XbW9AVA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXnsVygtLk%253D&md5=2afd1af603e16ed058b52a7359a650ed</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1186%2Fs13321-017-0232-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs13321-017-0232-0%26sid%3Dliteratum%253Aachs%26aulast%3DLenselink%26aufirst%3DE.%2BB.%26aulast%3DTen%2BDijke%26aufirst%3DN.%26aulast%3DBongers%26aufirst%3DB.%26aulast%3DPapadatos%26aufirst%3DG.%26aulast%3DVan%2BVlijmen%26aufirst%3DH.%2BW.%26aulast%3DKowalczyk%26aufirst%3DW.%26aulast%3DIjzerman%26aufirst%3DA.%2BP.%26aulast%3DVan%2BWesten%26aufirst%3DG.%2BJ.%26atitle%3DBeyond%2520the%2520hype%253A%2520deep%2520neural%2520networks%2520outperform%2520established%2520methods%2520using%2520a%2520ChEMBL%2520bioactivity%2520benchmark%2520set%26jtitle%3DJ.%2520Cheminf.%26date%3D2017%26volume%3D9%26spage%3D45%26doi%3D10.1186%2Fs13321-017-0232-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rodríguez-Pérez, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bajorath, J. r.</span></span> <span> </span><span class="NLM_article-title">Multitask Machine Learning for Classifying Highly and Weakly Potent Kinase Inhibitors</span>. <i>ACS Omega</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">4367</span>– <span class="NLM_lpage">4375</span>, <span class="refDoi"> DOI: 10.1021/acsomega.9b00298</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsomega.9b00298" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=1%3ACAS%3A528%3ADC%252BC1MXjs1Kmtbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2019&pages=4367-4375&author=R.+Rodr%C3%ADguez-P%C3%A9rezauthor=J.+r.+Bajorath&title=Multitask+Machine+Learning+for+Classifying+Highly+and+Weakly+Potent+Kinase+Inhibitors&doi=10.1021%2Facsomega.9b00298"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Multitask Machine Learning for Classifying Highly and Weakly Potent Kinase Inhibitors</span></div><div class="casAuthors">Rodriguez-Perez, Raquel; Bajorath, Juergen</div><div class="citationInfo"><span class="NLM_cas:title">ACS Omega</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">4367-4375</span>CODEN:
                <span class="NLM_cas:coden">ACSODF</span>;
        ISSN:<span class="NLM_cas:issn">2470-1343</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Compd. activity prediction is a major application of machine learning (ML) in pharmaceutical research.  Conventional single-task (ST) learning aims to predict active compds. for a given target.  In addn., multitask (MT) learning attempts to simultaneously predict active compds. for multiple targets.  The underlying rationale of MT learning is to guide and further improve modeling by exploring and exploiting related prediction tasks.  For MT learning, deep neural networks (DNNs) are often used, establishing a link between MT and deep learning.  In this work, ST and MT strategies for ML methods including DNN were compared in the systematic prediction of highly potent and weakly potent protein kinase inhibitors.  A total of 19 030 inhibitors with activity against 103 human kinases were used for modeling.  Given its compn., the data set provided many related prediction tasks.  DNN, support vector machine, and random forest ST and MT models were derived and compared.  Clear trends emerged.  Regardless of the method, MT learning consistently outperformed ST modeling.  Overall MT-DNNs achieved the highest prediction accuracy, but advantages over other MT-ML methods were only marginal.  Furthermore, feature wts. were extd. from models to evaluate correlation between different prediction tasks.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGryP6H8dy5cA7Vg90H21EOLACvtfcHk0ljk9x7XbW9AVA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXjs1Kmtbg%253D&md5=58ecd10e7f04a3da58a58d7b06a75fd7</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1021%2Facsomega.9b00298&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsomega.9b00298%26sid%3Dliteratum%253Aachs%26aulast%3DRodr%25C3%25ADguez-P%25C3%25A9rez%26aufirst%3DR.%26aulast%3DBajorath%26aufirst%3DJ.%2Br.%26atitle%3DMultitask%2520Machine%2520Learning%2520for%2520Classifying%2520Highly%2520and%2520Weakly%2520Potent%2520Kinase%2520Inhibitors%26jtitle%3DACS%2520Omega%26date%3D2019%26volume%3D4%26spage%3D4367%26epage%3D4375%26doi%3D10.1021%2Facsomega.9b00298" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Davis, M. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hunt, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herrgard, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciceri, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wodicka, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pallares, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hocker, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Treiber, D. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zarrinkar, P. P.</span></span> <span> </span><span class="NLM_article-title">Comprehensive analysis of kinase inhibitor selectivity</span>. <i>Nat. Biotechnol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">1046</span>– <span class="NLM_lpage">1051</span>, <span class="refDoi"> DOI: 10.1038/nbt.1990</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=10.1038%2Fnbt.1990" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=22037378" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtlyisrrK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2011&pages=1046-1051&author=M.+I.+Davisauthor=J.+P.+Huntauthor=S.+Herrgardauthor=P.+Ciceriauthor=L.+M.+Wodickaauthor=G.+Pallaresauthor=M.+Hockerauthor=D.+K.+Treiberauthor=P.+P.+Zarrinkar&title=Comprehensive+analysis+of+kinase+inhibitor+selectivity&doi=10.1038%2Fnbt.1990"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Comprehensive analysis of kinase inhibitor selectivity</span></div><div class="casAuthors">Davis, Mindy I.; Hunt, Jeremy P.; Herrgard, Sanna; Ciceri, Pietro; Wodicka, Lisa M.; Pallares, Gabriel; Hocker, Michael; Treiber, Daniel K.; Zarrinkar, Patrick P.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1046-1051</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">We tested the interaction of 72 kinase inhibitors with 442 kinases covering >80% of the human catalytic protein kinome.  Our data show that, as a class, type II inhibitors are more selective than type I inhibitors, but that there are important exceptions to this trend.  The data further illustrate that selective inhibitors have been developed against the majority of kinases targeted by the compds. tested.  Anal. of the interaction patterns reveals a class of 'group-selective' inhibitors broadly active against a single subfamily of kinases, but selective outside that subfamily.  The data set suggests compds. to use as tools to study kinases for which no dedicated inhibitors exist.  It also provides a foundation for further exploring kinase inhibitor biol. and toxicity, as well as for studying the structural basis of the obsd. interaction patterns.  Our findings will help to realize the direct enabling potential of genomics for drug development and basic research about cellular signaling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoEZnKVs6Xt5rVg90H21EOLACvtfcHk0ljgZPcBAnxhdw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtlyisrrK&md5=67639f702cb48ffb2c60e9a250f0df5e</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1038%2Fnbt.1990&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt.1990%26sid%3Dliteratum%253Aachs%26aulast%3DDavis%26aufirst%3DM.%2BI.%26aulast%3DHunt%26aufirst%3DJ.%2BP.%26aulast%3DHerrgard%26aufirst%3DS.%26aulast%3DCiceri%26aufirst%3DP.%26aulast%3DWodicka%26aufirst%3DL.%2BM.%26aulast%3DPallares%26aufirst%3DG.%26aulast%3DHocker%26aufirst%3DM.%26aulast%3DTreiber%26aufirst%3DD.%2BK.%26aulast%3DZarrinkar%26aufirst%3DP.%2BP.%26atitle%3DComprehensive%2520analysis%2520of%2520kinase%2520inhibitor%2520selectivity%26jtitle%3DNat.%2520Biotechnol.%26date%3D2011%26volume%3D29%26spage%3D1046%26epage%3D1051%26doi%3D10.1038%2Fnbt.1990" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Anastassiadis, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deacon, S. W</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Devarajan, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peterson, J. R</span></span> <span> </span><span class="NLM_article-title">Comprehensive assay of kinase catalytic activity reveals features of kinase inhibitor selectivity</span>. <i>Nat. Biotechnol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">1039</span>– <span class="NLM_lpage">1045</span>, <span class="refDoi"> DOI: 10.1038/nbt.2017</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=10.1038%2Fnbt.2017" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=22037377" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtlyisL7E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2011&pages=1039-1045&author=T.+Anastassiadisauthor=S.+W+Deaconauthor=K.+Devarajanauthor=H.+Maauthor=J.+R+Peterson&title=Comprehensive+assay+of+kinase+catalytic+activity+reveals+features+of+kinase+inhibitor+selectivity&doi=10.1038%2Fnbt.2017"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Comprehensive assay of kinase catalytic activity reveals features of kinase inhibitor selectivity</span></div><div class="casAuthors">Anastassiadis, Theonie; Deacon, Sean W.; Devarajan, Karthik; Ma, Haiching; Peterson, Jeffrey R.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1039-1045</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Small-mol. protein kinase inhibitors are widely used to elucidate cellular signaling pathways and are promising therapeutic agents.  Owing to evolutionary conservation of the ATP-binding site, most kinase inhibitors that target this site promiscuously inhibit multiple kinases.  Interpretation of expts. that use these compds. is confounded by a lack of data on the comprehensive kinase selectivity of most inhibitors.  Here we used functional assays to profile the activity of 178 com. available kinase inhibitors against a panel of 300 recombinant protein kinases.  Quant. anal. revealed complex and often unexpected interactions between protein kinases and kinase inhibitors, with a wide spectrum of promiscuity.  Many off-target interactions occur with seemingly unrelated kinases, revealing how large-scale profiling can identify multitargeted inhibitors of specific, diverse kinases.  The results have implications for drug development and provide a resource for selecting compds. to elucidate kinase function and for interpreting the results of expts. involving kinase inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpUljWIhZGPkrVg90H21EOLACvtfcHk0ljgZPcBAnxhdw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtlyisL7E&md5=b2f705509b0e09afa46b01d4669f0515</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1038%2Fnbt.2017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt.2017%26sid%3Dliteratum%253Aachs%26aulast%3DAnastassiadis%26aufirst%3DT.%26aulast%3DDeacon%26aufirst%3DS.%2BW%26aulast%3DDevarajan%26aufirst%3DK.%26aulast%3DMa%26aufirst%3DH.%26aulast%3DPeterson%26aufirst%3DJ.%2BR%26atitle%3DComprehensive%2520assay%2520of%2520kinase%2520catalytic%2520activity%2520reveals%2520features%2520of%2520kinase%2520inhibitor%2520selectivity%26jtitle%3DNat.%2520Biotechnol.%26date%3D2011%26volume%3D29%26spage%3D1039%26epage%3D1045%26doi%3D10.1038%2Fnbt.2017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Elkins, J. M</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedele, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szklarz, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abdul Azeez, K. R</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salah, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mikolajczyk, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Romanov, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sepetov, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, X.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roth, B. L</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Al Haj Zen, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fourches, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muratov, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tropsha, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morris, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Teicher, B. A</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kunkel, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Polley, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lackey, K. E</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Atkinson, F. L</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Overington, J. P</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bamborough, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Price, D. J</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Willson, T. M</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drewry, D. H</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zuercher, W. J</span></span> <span> </span><span class="NLM_article-title">Comprehensive characterization of the Published Kinase Inhibitor Set</span>. <i>Nat. Biotechnol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>34</i></span>,  <span class="NLM_fpage">95</span>– <span class="NLM_lpage">103</span>, <span class="refDoi"> DOI: 10.1038/nbt.3374</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=10.1038%2Fnbt.3374" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=26501955" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhslantL%252FM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=2016&pages=95-103&author=J.+M+Elkinsauthor=V.+Fedeleauthor=M.+Szklarzauthor=K.+R+Abdul+Azeezauthor=E.+Salahauthor=J.+Mikolajczykauthor=S.+Romanovauthor=N.+Sepetovauthor=X.-P.+Huangauthor=B.+L+Rothauthor=A.+Al+Haj+Zenauthor=D.+Fourchesauthor=E.+Muratovauthor=A.+Tropshaauthor=J.+Morrisauthor=B.+A+Teicherauthor=M.+Kunkelauthor=E.+Polleyauthor=K.+E+Lackeyauthor=F.+L+Atkinsonauthor=J.+P+Overingtonauthor=P.+Bamboroughauthor=S.+Mullerauthor=D.+J+Priceauthor=T.+M+Willsonauthor=D.+H+Drewryauthor=S.+Knappauthor=W.+J+Zuercher&title=Comprehensive+characterization+of+the+Published+Kinase+Inhibitor+Set&doi=10.1038%2Fnbt.3374"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Comprehensive characterization of the Published Kinase Inhibitor Set</span></div><div class="casAuthors">Elkins, Jonathan M.; Fedele, Vita; Szklarz, Marta; Abdul Azeez, Kamal R.; Salah, Eidarus; Mikolajczyk, Jowita; Romanov, Sergei; Sepetov, Nikolai; Huang, Xi-Ping; Roth, Bryan L.; Al Haj Zen, Ayman; Fourches, Denis; Muratov, Eugene; Tropsha, Alex; Morris, Joel; Teicher, Beverly A.; Kunkel, Mark; Polley, Eric; Lackey, Karen E.; Atkinson, Francis L.; Overington, John P.; Bamborough, Paul; Muller, Susanne; Price, Daniel J.; Willson, Timothy M.; Drewry, David H.; Knapp, Stefan; Zuercher, William J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">34</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">95-103</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Despite the success of protein kinase inhibitors as approved therapeutics, drug discovery has focused on a small subset of kinase targets.  Here we provide a thorough characterization of the Published Kinase Inhibitor Set (PKIS), a set of 367 small-mol. ATP-competitive kinase inhibitors that was recently made freely available with the aim of expanding research in this field and as an expt. in open-source target validation.  We screen the set in activity assays with 224 recombinant kinases and 24 G protein-coupled receptors and in cellular assays of cancer cell proliferation and angiogenesis.  We identify chem. starting points for designing new chem. probes of orphan kinases and illustrate the utility of these leads by developing a selective inhibitor for the previously untargeted kinases LOK and SLK.  Our cellular screens reveal compds. that modulate cancer cell growth and angiogenesis in vitro.  These reagents and assocd. data illustrate an efficient way forward to increasing understanding of the historically untargeted kinome.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo4P1WN3gdiXbVg90H21EOLACvtfcHk0lgiEpz5U22iTQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhslantL%252FM&md5=3f15096e79632dbaaadf4460a6848e60</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1038%2Fnbt.3374&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt.3374%26sid%3Dliteratum%253Aachs%26aulast%3DElkins%26aufirst%3DJ.%2BM%26aulast%3DFedele%26aufirst%3DV.%26aulast%3DSzklarz%26aufirst%3DM.%26aulast%3DAbdul%2BAzeez%26aufirst%3DK.%2BR%26aulast%3DSalah%26aufirst%3DE.%26aulast%3DMikolajczyk%26aufirst%3DJ.%26aulast%3DRomanov%26aufirst%3DS.%26aulast%3DSepetov%26aufirst%3DN.%26aulast%3DHuang%26aufirst%3DX.-P.%26aulast%3DRoth%26aufirst%3DB.%2BL%26aulast%3DAl%2BHaj%2BZen%26aufirst%3DA.%26aulast%3DFourches%26aufirst%3DD.%26aulast%3DMuratov%26aufirst%3DE.%26aulast%3DTropsha%26aufirst%3DA.%26aulast%3DMorris%26aufirst%3DJ.%26aulast%3DTeicher%26aufirst%3DB.%2BA%26aulast%3DKunkel%26aufirst%3DM.%26aulast%3DPolley%26aufirst%3DE.%26aulast%3DLackey%26aufirst%3DK.%2BE%26aulast%3DAtkinson%26aufirst%3DF.%2BL%26aulast%3DOverington%26aufirst%3DJ.%2BP%26aulast%3DBamborough%26aufirst%3DP.%26aulast%3DMuller%26aufirst%3DS.%26aulast%3DPrice%26aufirst%3DD.%2BJ%26aulast%3DWillson%26aufirst%3DT.%2BM%26aulast%3DDrewry%26aufirst%3DD.%2BH%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DZuercher%26aufirst%3DW.%2BJ%26atitle%3DComprehensive%2520characterization%2520of%2520the%2520Published%2520Kinase%2520Inhibitor%2520Set%26jtitle%3DNat.%2520Biotechnol.%26date%3D2016%26volume%3D34%26spage%3D95%26epage%3D103%26doi%3D10.1038%2Fnbt.3374" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Drewry, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wells, C. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andrews, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Angell, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Al-Ali, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Axtman, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Capuzzi, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elkins, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ettmayer, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frederiksen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gileadi, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hooper, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laufer, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luecking, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michaelides, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muratov, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Denny, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saikatendu, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Treiber, D. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zuercher, W. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Willson, T. M.</span></span> <span> </span><span class="NLM_article-title">Progress towards a public chemogenomic set for protein kinases and a call for contributions</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>12</i></span>, <span class="NLM_elocation-id">e0181585</span> <span class="refDoi"> DOI: 10.1371/journal.pone.0181585</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=10.1371%2Fjournal.pone.0181585" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=28767711" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhsFWktL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2017&author=D.+H.+Drewryauthor=C.+I.+Wellsauthor=D.+M.+Andrewsauthor=R.+Angellauthor=H.+Al-Aliauthor=A.+D.+Axtmanauthor=S.+J.+Capuzziauthor=J.+M.+Elkinsauthor=P.+Ettmayerauthor=M.+Frederiksenauthor=O.+Gileadiauthor=N.+Grayauthor=A.+Hooperauthor=S.+Knappauthor=S.+Lauferauthor=U.+Lueckingauthor=M.+Michaelidesauthor=S.+Mullerauthor=E.+Muratovauthor=R.+A.+Dennyauthor=K.+S.+Saikatenduauthor=D.+K.+Treiberauthor=W.+J.+Zuercherauthor=T.+M.+Willson&title=Progress+towards+a+public+chemogenomic+set+for+protein+kinases+and+a+call+for+contributions&doi=10.1371%2Fjournal.pone.0181585"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Progress towards a public chemogenomic set for protein kinases and a call for contributions</span></div><div class="casAuthors">Drewry, David H.; Wells, Carrow I.; Andrews, David M.; Angell, Richard; Al-Ali, Hassan; Axtman, Alison D.; Capuzzi, Stephen J.; Elkins, Jonathan M.; Ettmayer, Peter; Frederiksen, Mathias; Gileadi, Opher; Gray, Nathanael; Hooper, Alice; Knapp, Stefan; Laufer, Stefan; Luecking, Ulrich; Michaelides, Michael; Muller, Susanne; Muratov, Eugene; Aldrin, Denny R.; Saikatendu, Kumar S.; Treiber, Daniel K.; Zuercher, William J.; Willson, Timothy M.</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">e0181585/1-e0181585/20</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Protein kinases are highly tractable targets for drug discovery.  However, the biol. function and therapeutic potential of the majority of the 500+ human protein kinases remains unknown.  We have developed phys. and virtual collections of small mol. inhibitors, which we call chemogenomic sets, that are designed to inhibit the catalytic function of almost half the human protein kinases.  In this manuscript we share our progress towards generation of a comprehensive kinase chemogenomic set (KCGS), release kinome profiling data of a large inhibitor set (Published Kinase Inhibitor Set 2 (PKIS2)), and outline a process through which the community can openly collaborate to create a KCGS that probes the full complement of human protein kinases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpozOiNZl51XbVg90H21EOLACvtfcHk0lhawXL6WJ3PCw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhsFWktL0%253D&md5=9c27ba5cd21da43d30ce485ffa0e1675</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0181585&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0181585%26sid%3Dliteratum%253Aachs%26aulast%3DDrewry%26aufirst%3DD.%2BH.%26aulast%3DWells%26aufirst%3DC.%2BI.%26aulast%3DAndrews%26aufirst%3DD.%2BM.%26aulast%3DAngell%26aufirst%3DR.%26aulast%3DAl-Ali%26aufirst%3DH.%26aulast%3DAxtman%26aufirst%3DA.%2BD.%26aulast%3DCapuzzi%26aufirst%3DS.%2BJ.%26aulast%3DElkins%26aufirst%3DJ.%2BM.%26aulast%3DEttmayer%26aufirst%3DP.%26aulast%3DFrederiksen%26aufirst%3DM.%26aulast%3DGileadi%26aufirst%3DO.%26aulast%3DGray%26aufirst%3DN.%26aulast%3DHooper%26aufirst%3DA.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DLaufer%26aufirst%3DS.%26aulast%3DLuecking%26aufirst%3DU.%26aulast%3DMichaelides%26aufirst%3DM.%26aulast%3DMuller%26aufirst%3DS.%26aulast%3DMuratov%26aufirst%3DE.%26aulast%3DDenny%26aufirst%3DR.%2BA.%26aulast%3DSaikatendu%26aufirst%3DK.%2BS.%26aulast%3DTreiber%26aufirst%3DD.%2BK.%26aulast%3DZuercher%26aufirst%3DW.%2BJ.%26aulast%3DWillson%26aufirst%3DT.%2BM.%26atitle%3DProgress%2520towards%2520a%2520public%2520chemogenomic%2520set%2520for%2520protein%2520kinases%2520and%2520a%2520call%2520for%2520contributions%26jtitle%3DPLoS%2520One%26date%3D2017%26volume%3D12%26doi%3D10.1371%2Fjournal.pone.0181585" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sutherland, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cahya, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vieth, M.</span></span> <span> </span><span class="NLM_article-title">What general conclusions can we draw from kinase profiling data sets?</span>. <i>Biochim. Biophys. Acta, Proteins Proteomics</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>1834</i></span>,  <span class="NLM_fpage">1425</span>– <span class="NLM_lpage">1433</span>, <span class="refDoi"> DOI: 10.1016/j.bbapap.2012.12.023</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=10.1016%2Fj.bbapap.2012.12.023" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=23333421" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=1%3ACAS%3A528%3ADC%252BC3sXit1KksLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1834&publication_year=2013&pages=1425-1433&author=J.+J.+Sutherlandauthor=C.+Gaoauthor=S.+Cahyaauthor=M.+Vieth&title=What+general+conclusions+can+we+draw+from+kinase+profiling+data+sets%3F&doi=10.1016%2Fj.bbapap.2012.12.023"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">What general conclusions can we draw from kinase profiling data sets?</span></div><div class="casAuthors">Sutherland, Jeffrey J.; Gao, Cen; Cahya, Suntara; Vieth, Michal</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, Proteins and Proteomics</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">1834</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1425-1433</span>CODEN:
                <span class="NLM_cas:coden">BBAPBW</span>;
        ISSN:<span class="NLM_cas:issn">1570-9639</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B. V.</span>)
        </div><div class="casAbstract">A review.  Understanding general selectivity trends across the kinome has implications ranging from target selection, compd. prioritization, toxicity and patient tailoring.  Several recent publications have described the characterization of kinase inhibitors via large assay panels, offering a range of generalizations that influenced kinase inhibitor research trends.  Since a subset of profiled inhibitors overlap across reports, we evaluated the concordance of activity results for the same compd.-kinase pairs across four data sources generated from different kinase biochem. assay technologies.  Overall, 77% of all results are within 3 fold or qual. in agreement across sources.  However, the agreement for active compds. is only 37%, indicating that different profiling panels are in better agreement to det. a compd.'s lack of activity rather than degree of activity.  Low concordance is also found when comparing the promiscuity of kinase targets evaluated from different sources, and the pharmacol. similarity of kinases.  In contrast, the overall promiscuity of kinase inhibitors was consistent across sources.  We highlight the difficulty of drawing general conclusions from such data by showing that no significant selectivity difference distinguishes type I vs. type II inhibitors, and limited kinase space similarity that is consistent across different sources.  This article is part of a Special Issue entitled: Inhibitors of Protein Kinases (2012).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrWOFwnlgm0AbVg90H21EOLACvtfcHk0ljJEaRFbseqVw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXit1KksLk%253D&md5=59c6667035700272be4e4e0e890490ed</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1016%2Fj.bbapap.2012.12.023&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbapap.2012.12.023%26sid%3Dliteratum%253Aachs%26aulast%3DSutherland%26aufirst%3DJ.%2BJ.%26aulast%3DGao%26aufirst%3DC.%26aulast%3DCahya%26aufirst%3DS.%26aulast%3DVieth%26aufirst%3DM.%26atitle%3DWhat%2520general%2520conclusions%2520can%2520we%2520draw%2520from%2520kinase%2520profiling%2520data%2520sets%253F%26jtitle%3DBiochim.%2520Biophys.%2520Acta%252C%2520Proteins%2520Proteomics%26date%3D2013%26volume%3D1834%26spage%3D1425%26epage%3D1433%26doi%3D10.1016%2Fj.bbapap.2012.12.023" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Posy, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hermsmeier, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vaccaro, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ott, K.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Todderud, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lippy, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trainor, G. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loughney, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, S. R.</span></span> <span> </span><span class="NLM_article-title">Trends in kinase selectivity: insights for target class-focused library screening</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">54</span>– <span class="NLM_lpage">66</span>, <span class="refDoi"> DOI: 10.1021/jm101195a</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm101195a" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsFSms7jJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=54-66&author=S.+L.+Posyauthor=M.+A.+Hermsmeierauthor=W.+Vaccaroauthor=K.-H.+Ottauthor=G.+Todderudauthor=J.+S.+Lippyauthor=G.+L.+Trainorauthor=D.+A.+Loughneyauthor=S.+R.+Johnson&title=Trends+in+kinase+selectivity%3A+insights+for+target+class-focused+library+screening&doi=10.1021%2Fjm101195a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Trends in Kinase Selectivity: Insights for Target Class-Focused Library Screening</span></div><div class="casAuthors">Posy, Shana L.; Hermsmeier, Mark A.; Vaccaro, Wayne; Ott, Karl-Heinz; Todderud, Gordon; Lippy, Jonathan S.; Trainor, George L.; Loughney, Deborah A.; Johnson, Stephen R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">54-66</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A kinome-wide selectivity screen of >20000 compds. with a rich representation of many structural classes has been completed.  Anal. of the selectivity patterns for each class shows that a broad spectrum of structural scaffolds can achieve specificity for many kinase families.  Kinase selectivity and potency are inversely correlated, a trend that is also found in a large set of kinase functional data.  Although selective and nonselective compds. are mostly similar in their physicochem. characteristics, we identify specific features that are present more frequently in compds. that bind to many kinases.  Our results support a scaffold-oriented approach for building compd. collections to screen kinase targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpJlqg1ZszRmLVg90H21EOLACvtfcHk0ljJEaRFbseqVw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsFSms7jJ&md5=f824b454904fb99d1ddd1ac6396cb784</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1021%2Fjm101195a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm101195a%26sid%3Dliteratum%253Aachs%26aulast%3DPosy%26aufirst%3DS.%2BL.%26aulast%3DHermsmeier%26aufirst%3DM.%2BA.%26aulast%3DVaccaro%26aufirst%3DW.%26aulast%3DOtt%26aufirst%3DK.-H.%26aulast%3DTodderud%26aufirst%3DG.%26aulast%3DLippy%26aufirst%3DJ.%2BS.%26aulast%3DTrainor%26aufirst%3DG.%2BL.%26aulast%3DLoughney%26aufirst%3DD.%2BA.%26aulast%3DJohnson%26aufirst%3DS.%2BR.%26atitle%3DTrends%2520in%2520kinase%2520selectivity%253A%2520insights%2520for%2520target%2520class-focused%2520library%2520screening%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26spage%3D54%26epage%3D66%26doi%3D10.1021%2Fjm101195a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mayr, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klambauer, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Unterthiner, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steijaert, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wegner, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ceulemans, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clevert, D.-A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hochreiter, S.</span></span> <span> </span><span class="NLM_article-title">Large-scale comparison of machine learning methods for drug target prediction on ChEMBL</span>. <i>Chem. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">5441</span>– <span class="NLM_lpage">5451</span>, <span class="refDoi"> DOI: 10.1039/C8SC00148K</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=10.1039%2FC8SC00148K" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=30155234" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtVOlu7jE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2018&pages=5441-5451&author=A.+Mayrauthor=G.+Klambauerauthor=T.+Unterthinerauthor=M.+Steijaertauthor=J.+K.+Wegnerauthor=H.+Ceulemansauthor=D.-A.+Clevertauthor=S.+Hochreiter&title=Large-scale+comparison+of+machine+learning+methods+for+drug+target+prediction+on+ChEMBL&doi=10.1039%2FC8SC00148K"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Large-scale comparison of machine learning methods for drug target prediction on ChEMBL</span></div><div class="casAuthors">Mayr, Andreas; Klambauer, Guenter; Unterthiner, Thomas; Steijaert, Marvin; Wegner, Joerg K.; Ceulemans, Hugo; Clevert, Djork-Arne; Hochreiter, Sepp</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Science</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">5441-5451</span>CODEN:
                <span class="NLM_cas:coden">CSHCCN</span>;
        ISSN:<span class="NLM_cas:issn">2041-6520</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Deep learning is currently the most successful machine learning technique in a wide range of application areas and has recently been applied successfully in drug discovery research to predict potential drug targets and to screen for active mols.  However, due to (1) the lack of large-scale studies, (2) the compd. series bias that is characteristic of drug discovery datasets and (3) the hyperparameter selection bias that comes with the high no. of potential deep learning architectures, it remains unclear whether deep learning can indeed outperform existing computational methods in drug discovery tasks.  We therefore assessed the performance of several deep learning methods on a large-scale drug discovery dataset and compared the results with those of other machine learning and target prediction methods.  To avoid potential biases from hyperparameter selection or compd. series, we used a nested cluster-cross-validation strategy.  We found (1) that deep learning methods significantly outperform all competing methods and (2) that the predictive performance of deep learning is in many cases comparable to that of tests performed in wet labs (i.e., in vitro assays).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrGfZgi5YuZVrVg90H21EOLACvtfcHk0lhLveHMjc9Miw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtVOlu7jE&md5=e91e5905e357d0c49ce443d78125537f</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1039%2FC8SC00148K&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC8SC00148K%26sid%3Dliteratum%253Aachs%26aulast%3DMayr%26aufirst%3DA.%26aulast%3DKlambauer%26aufirst%3DG.%26aulast%3DUnterthiner%26aufirst%3DT.%26aulast%3DSteijaert%26aufirst%3DM.%26aulast%3DWegner%26aufirst%3DJ.%2BK.%26aulast%3DCeulemans%26aufirst%3DH.%26aulast%3DClevert%26aufirst%3DD.-A.%26aulast%3DHochreiter%26aufirst%3DS.%26atitle%3DLarge-scale%2520comparison%2520of%2520machine%2520learning%2520methods%2520for%2520drug%2520target%2520prediction%2520on%2520ChEMBL%26jtitle%3DChem.%2520Sci.%26date%3D2018%26volume%3D9%26spage%3D5441%26epage%3D5451%26doi%3D10.1039%2FC8SC00148K" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">vn
der Maaten, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hinton, G.</span></span> <span> </span><span class="NLM_article-title">Visualizing data using t-SNE</span>. <i>J. Mach. Learn. Res.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">2579</span>– <span class="NLM_lpage">2605</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2008&pages=2579-2605&author=L.+vn%0Ader+Maatenauthor=G.+Hinton&title=Visualizing+data+using+t-SNE"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3Dvn%2Bder%2BMaaten%26aufirst%3DL.%26aulast%3DHinton%26aufirst%3DG.%26atitle%3DVisualizing%2520data%2520using%2520t-SNE%26jtitle%3DJ.%2520Mach.%2520Learn.%2520Res.%26date%3D2008%26volume%3D9%26spage%3D2579%26epage%3D2605" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Avram, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bora, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Halip, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Curpǎn, R.</span></span> <span> </span><span class="NLM_article-title">Modeling kinase inhibition using highly confident data sets</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">957</span>– <span class="NLM_lpage">967</span>, <span class="refDoi"> DOI: 10.1021/acs.jcim.7b00729</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.7b00729" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=1%3ACAS%3A528%3ADC%252BC1cXosFensro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2018&pages=957-967&author=S.+Avramauthor=A.+Boraauthor=L.+Halipauthor=R.+Curp%C7%8En&title=Modeling+kinase+inhibition+using+highly+confident+data+sets&doi=10.1021%2Facs.jcim.7b00729"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Modeling Kinase Inhibition Using Highly Confident Data Sets</span></div><div class="casAuthors">Avram, Sorin; Bora, Alina; Halip, Liliana; Curpan, Ramona</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">957-967</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Protein kinases form a consistent class of promising drug targets, and several efforts have been made to predict the activities of small mols. against a representative part of the kinome.  This study continues our previous work aiming to build and measure the performance of ligand-based kinase inhibitor prediction models.  Here we analyzed kinase-inhibitor pairs with multiple activity points extd. from the ChEMBL database and identified the main sources of inconsistency.  Our results indicate that lower IC50 values are usually less affected by errors and reflect more accurately the structure-activity relationship of the mols. against the target, ideally for quant. structure-activity relationship studies.  Further, we modeled the activities of 104 kinases using unbiased target-specific activity points.  The performance of predictors built on extended connectivity fingerprints (ECFP4) and two-dimensional pharmacophore fingerprints (PFPs) are compared by means of tolerance intervals (TIs) (95%/95%) in virtual screening (VS) and classification tasks using external random (RandSets) and diversity-based (DivSets) test sets.  We found that the two encodings perform superior to each other on different kinases in VS and that PFP models perform consistently better in classifying actives (higher sensitivity).  Next, we combined the two encodings into a single one (PFPECFP) and demonstrated that esp. in VS (as indicated by the exponential receiver operating curve enrichment metric (eROCE)), for the vast majority of kinases the model performance increased compared with the individual fingerprint models.  These findings are highlighted in the more challenging DivSets compared with RandSets.  The current paper explores the boundaries of inhibitor predictors for individual kinases to enhance VS and ultimately aid the discovery of novel compds. with desirable polypharmacol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpvbuviAgeXNLVg90H21EOLACvtfcHk0ljgvxJya1flgQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXosFensro%253D&md5=31fe4e3cf0a44becc1a97b345e0975da</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.7b00729&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.7b00729%26sid%3Dliteratum%253Aachs%26aulast%3DAvram%26aufirst%3DS.%26aulast%3DBora%26aufirst%3DA.%26aulast%3DHalip%26aufirst%3DL.%26aulast%3DCurp%25C7%258En%26aufirst%3DR.%26atitle%3DModeling%2520kinase%2520inhibition%2520using%2520highly%2520confident%2520data%2520sets%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2018%26volume%3D58%26spage%3D957%26epage%3D967%26doi%3D10.1021%2Facs.jcim.7b00729" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Janssen, A. P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grimm, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wijdeven, R. H. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lenselink, E. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neefjes, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Boeckel, C. A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Westen, G. J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van der Stelt, M.</span></span> <span> </span><span class="NLM_article-title">Drug discovery maps, a machine learning model that visualizes and predicts kinome–inhibitor interaction landscapes</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">1221</span>– <span class="NLM_lpage">1229</span>, <span class="refDoi"> DOI: 10.1021/acs.jcim.8b00640</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.8b00640" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitVansb3K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2019&pages=1221-1229&author=A.+P.+A.+Janssenauthor=S.+H.+Grimmauthor=R.+H.+M.+Wijdevenauthor=E.+B.+Lenselinkauthor=J.+Neefjesauthor=C.+A.+A.+van+Boeckelauthor=G.+J.+P.+van+Westenauthor=M.+van+der+Stelt&title=Drug+discovery+maps%2C+a+machine+learning+model+that+visualizes+and+predicts+kinome%E2%80%93inhibitor+interaction+landscapes&doi=10.1021%2Facs.jcim.8b00640"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Drug Discovery Maps, a Machine Learning Model That Visualizes and Predicts Kinome-Inhibitor Interaction Landscapes</span></div><div class="casAuthors">Janssen, Antonius P. A.; Grimm, Sebastian H.; Wijdeven, Ruud H. M.; Lenselink, Eelke B.; Neefjes, Jacques; van Boeckel, Constant A. A.; van Westen, Gerard J. P.; van der Stelt, Mario</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1221-1229</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The interpretation of high-dimensional structure-activity data sets in drug discovery to predict ligand-protein interaction landscapes is a challenging task.  Here we present Drug Discovery Maps (DDM), a machine learning model that maps the activity profile of compds. across an entire protein family, as illustrated here for the kinase family.  DDM is based on the t-distributed stochastic neighbor embedding (t-SNE) algorithm to generate a visualization of mol. and biol. similarity.  DDM maps chem. and target space and predicts the activities of novel kinase inhibitors across the kinome.  The model was validated using independent data sets and in a prospective exptl. setting, where DDM predicted new inhibitors for FMS-like tyrosine kinase 3 (FLT3), a therapeutic target for the treatment of acute myeloid leukemia.  Compds. were resynthesized, yielding highly potent, cellularly active FLT3 inhibitors.  Biochem. assays confirmed most of the predicted off-targets.  DDM is further unique in that it is completely open-source and available as a ready-to-use executable to facilitate broad and easy adoption.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqJ32jhSPSIdLVg90H21EOLACvtfcHk0ljgvxJya1flgQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitVansb3K&md5=dc60f5872d49edde162e29327857e08f</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.8b00640&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.8b00640%26sid%3Dliteratum%253Aachs%26aulast%3DJanssen%26aufirst%3DA.%2BP.%2BA.%26aulast%3DGrimm%26aufirst%3DS.%2BH.%26aulast%3DWijdeven%26aufirst%3DR.%2BH.%2BM.%26aulast%3DLenselink%26aufirst%3DE.%2BB.%26aulast%3DNeefjes%26aufirst%3DJ.%26aulast%3Dvan%2BBoeckel%26aufirst%3DC.%2BA.%2BA.%26aulast%3Dvan%2BWesten%26aufirst%3DG.%2BJ.%2BP.%26aulast%3Dvan%2Bder%2BStelt%26aufirst%3DM.%26atitle%3DDrug%2520discovery%2520maps%252C%2520a%2520machine%2520learning%2520model%2520that%2520visualizes%2520and%2520predicts%2520kinome%25E2%2580%2593inhibitor%2520interaction%2520landscapes%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2019%26volume%3D59%26spage%3D1221%26epage%3D1229%26doi%3D10.1021%2Facs.jcim.8b00640" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ai, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ji, Y. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geng, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H.</span></span> <span> </span><span class="NLM_article-title">Novel 5-(benzyloxy)pyridin-2(1 H)-one derivatives as potent c-Met inhibitors</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">2408</span>– <span class="NLM_lpage">2413</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2013.02.037</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=10.1016%2Fj.bmcl.2013.02.037" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=23474386" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=1%3ACAS%3A528%3ADC%252BC3sXktFaisrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2013&pages=2408-2413&author=D.+Zhangauthor=J.+Aiauthor=Z.+Liangauthor=W.+Zhuauthor=X.+Pengauthor=X.+Chenauthor=Y.+C.+Jiauthor=H.+Jiangauthor=C.+Luoauthor=M.+Gengauthor=H.+Liu&title=Novel+5-%28benzyloxy%29pyridin-2%281+H%29-one+derivatives+as+potent+c-Met+inhibitors&doi=10.1016%2Fj.bmcl.2013.02.037"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Novel 5-(benzyloxy)pyridin-2(1H)-one derivatives as potent c-Met inhibitors</span></div><div class="casAuthors">Zhang, Dengyou; Ai, Jing; Liang, Zhongjie; Zhu, Wei; Peng, Xia; Chen, Xianjie; Ji, YinChun; Jiang, Hualiang; Luo, Cheng; Geng, Meiyu; Liu, Hong</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2408-2413</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A series of novel 5-(benzyloxy)pyridin-2(1H)-ones were designed, synthesized and biol. evaluated for c-Met inhibition.  Various amides and benzimidazoles at the C-3 position were investigated.  A potent compd. I with a c-Met IC50 of 12 nM was identified.  This compd. exhibited potent inhibition of EBC-1 cell assocd. with c-Met constitutive activation and showed high selectivity for c-Met than other tested 11 kinases.  The binding model I with c-Met was disclosed by docking anal.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqGU1nKpF0QGbVg90H21EOLACvtfcHk0lhsTeQBY7FCSA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXktFaisrc%253D&md5=1ae7dab74ba56e7cd1b5701959d474b7</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2013.02.037&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2013.02.037%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DD.%26aulast%3DAi%26aufirst%3DJ.%26aulast%3DLiang%26aufirst%3DZ.%26aulast%3DZhu%26aufirst%3DW.%26aulast%3DPeng%26aufirst%3DX.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DJi%26aufirst%3DY.%2BC.%26aulast%3DJiang%26aufirst%3DH.%26aulast%3DLuo%26aufirst%3DC.%26aulast%3DGeng%26aufirst%3DM.%26aulast%3DLiu%26aufirst%3DH.%26atitle%3DNovel%25205-%2528benzyloxy%2529pyridin-2%25281%2520H%2529-one%2520derivatives%2520as%2520potent%2520c-Met%2520inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2013%26volume%3D23%26spage%3D2408%26epage%3D2413%26doi%3D10.1016%2Fj.bmcl.2013.02.037" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peng, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ji, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tong, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H.</span></span> <span> </span><span class="NLM_article-title">Design, synthesis and biological evaluation of novel 4-anilinoquinazolines with C-6 urea-linked side chains as inhibitors of the epidermal growth factor receptor</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">7988</span>– <span class="NLM_lpage">7998</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2013.09.049</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=10.1016%2Fj.bmc.2013.09.049" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=24183742" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhslSlsLrI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2013&pages=7988-7998&author=X.+Zhangauthor=T.+Pengauthor=X.+Jiauthor=J.+Liauthor=L.+Tongauthor=Z.+Liauthor=W.+Yangauthor=Y.+Xuauthor=M.+Liauthor=J.+Dingauthor=H.+Jiangauthor=H.+Xieauthor=H.+Liu&title=Design%2C+synthesis+and+biological+evaluation+of+novel+4-anilinoquinazolines+with+C-6+urea-linked+side+chains+as+inhibitors+of+the+epidermal+growth+factor+receptor&doi=10.1016%2Fj.bmc.2013.09.049"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Design, synthesis and biological evaluation of novel 4-anilinoquinazolines with C-6 urea-linked side chains as inhibitors of the epidermal growth factor receptor</span></div><div class="casAuthors">Zhang, Xu; Peng, Ting; Ji, Xun; Li, Jian; Tong, Linjiang; Li, Zeng; Yang, Wei; Xu, Yungen; Li, Mengyuan; Ding, Jian; Jiang, Hualiang; Xie, Hua; Liu, Hong</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">7988-7998</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A novel series of anilinoquinazoline compds. with C-6 urea-linked side chains was designed and synthesized as reversible inhibitors of epidermal growth factor receptor (EGFR) based on the structure-activity relationships (SARs) of anilinoquinazoline inhibitors.  All compds. demonstrated good inhibition of EGFR wild type (EGFR wt) and inhibited proliferation of A431cell line.  Compds. (I) and (II) almost completely blocked the phosphorylation of EGFR in A431 cell line at 0.01 μM.  Interestingly, all of the compds. also demonstrated moderate inhibition of EGFR/T790M/L858R.  In addn., compds. (III) and (IV) blocked the autophosphorylation of EGFR in NCI-H1975 cells at high concn. (10 μM), and compd. 8f was confirmed to be an irreversible inhibitor through the diln. method.  Importantly, the compds. with C-6 urea-linked side chains which did not contain Michael acceptors demonstrated moderate to strong irreversible EGFR inhibition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqm9A3kxR7nTbVg90H21EOLACvtfcHk0lhsTeQBY7FCSA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhslSlsLrI&md5=30a7a86cd500a9073efe50cd1660bf4b</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2013.09.049&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2013.09.049%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DPeng%26aufirst%3DT.%26aulast%3DJi%26aufirst%3DX.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DTong%26aufirst%3DL.%26aulast%3DLi%26aufirst%3DZ.%26aulast%3DYang%26aufirst%3DW.%26aulast%3DXu%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DM.%26aulast%3DDing%26aufirst%3DJ.%26aulast%3DJiang%26aufirst%3DH.%26aulast%3DXie%26aufirst%3DH.%26aulast%3DLiu%26aufirst%3DH.%26atitle%3DDesign%252C%2520synthesis%2520and%2520biological%2520evaluation%2520of%2520novel%25204-anilinoquinazolines%2520with%2520C-6%2520urea-linked%2520side%2520chains%2520as%2520inhibitors%2520of%2520the%2520epidermal%2520growth%2520factor%2520receptor%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2013%26volume%3D21%26spage%3D7988%26epage%3D7998%26doi%3D10.1016%2Fj.bmc.2013.09.049" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Grothey, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cutsem, E. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sobrero, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siena, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Falcone, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ychou, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Humblet, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bouche, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mineur, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barone, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adenis, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tabernero, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yoshino, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lenz, H.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goldberg, R. M</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sargent, D. J</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cihon, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cupit, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wagner, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laurent, D.</span></span> <span> </span><span class="NLM_article-title">Regorafenib monotherapy for previously treated metastatic colorectal cancer (CORRECT): an international, multicentre, randomised, placebo-controlled, phase 3 trial</span>. <i>Lancet</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>381</i></span>,  <span class="NLM_fpage">303</span>– <span class="NLM_lpage">312</span>, <span class="refDoi"> DOI: 10.1016/S0140-6736(12)61900-X</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=10.1016%2FS0140-6736%2812%2961900-X" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=23177514" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=1%3ACAS%3A528%3ADC%252BC38Xhslarsr%252FE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=381&publication_year=2013&pages=303-312&author=A.+Grotheyauthor=E.+V.+Cutsemauthor=A.+Sobreroauthor=S.+Sienaauthor=A.+Falconeauthor=M.+Ychouauthor=Y.+Humbletauthor=O.+Boucheauthor=L.+Mineurauthor=C.+Baroneauthor=A.+Adenisauthor=J.+Taberneroauthor=T.+Yoshinoauthor=H.-J.+Lenzauthor=R.+M+Goldbergauthor=D.+J+Sargentauthor=F.+Cihonauthor=L.+Cupitauthor=A.+Wagnerauthor=D.+Laurent&title=Regorafenib+monotherapy+for+previously+treated+metastatic+colorectal+cancer+%28CORRECT%29%3A+an+international%2C+multicentre%2C+randomised%2C+placebo-controlled%2C+phase+3+trial&doi=10.1016%2FS0140-6736%2812%2961900-X"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Regorafenib monotherapy for previously treated metastatic colorectal cancer (CORRECT): an international, multicentre, randomised, placebo-controlled, phase 3 trial</span></div><div class="casAuthors">Grothey, Axel; Cutsem, Eric Van; Sobrero, Alberto; Siena, Salvatore; Falcone, Alfredo; Ychou, Marc; Humblet, Yves; Bouche, Olivier; Mineur, Laurent; Barone, Carlo; Adenis, Antoine; Tabernero, Josep; Yoshino, Takayuki; Lenz, Heinz-Josef; Goldberg, Richard M.; Sargent, Daniel J.; Cihon, Frank; Cupit, Lisa; Wagner, Andrea; Laurent, Dirk</div><div class="citationInfo"><span class="NLM_cas:title">Lancet</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">381</span>
        (<span class="NLM_cas:issue">9863</span>),
    <span class="NLM_cas:pages">303-312</span>CODEN:
                <span class="NLM_cas:coden">LANCAO</span>;
        ISSN:<span class="NLM_cas:issn">0140-6736</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">No treatment options are available for patients with metastatic colorectal cancer that progresses after all approved std. therapies, but many patients maintain a good performance status and could be candidates for further therapy.  An international phase 3 trial was done to assess the multikinase inhibitor regorafenib in these patients.  We did this trial at 114 centers in 16 countries.  Patients with documented metastatic colorectal cancer and progression during or within 3 mo after the last std. therapy were randomized (in a 2:1 ratio; by computer-generated randomization list and interactive voice response system; preallocated block design (block size six); stratified by previous treatment with VEGF-targeting drugs, time from diagnosis of metastatic disease, and geog. region) to receive best supportive care plus oral regorafenib 160 mg or placebo once daily, for the first 3 wk of each 4 wk cycle.  The primary endpoint was overall survival.  The study sponsor, participants, and investigators were masked to treatment assignment.  Efficacy analyses were by intention to treat.  This trial is registered at ClinicalTrials.gov, no. NCT01103323.Between Apr. 30, 2010, and March 22, 2011, 1052 patients were screened, 760 patients were randomized to receive regorafenib (n=505) or placebo (n=255), and 753 patients initiated treatment (regorafenib n=500; placebo n=253; population for safety analyses).  The primary endpoint of overall survival was met at a preplanned interim anal.; data cutoff was on July 21, 2011.  Median overall survival was 6·4 mo in the regorafenib group vs. 5·0 mo in the placebo group (hazard ratio 0·77; 95% CI 0·64-0·94; one-sided p=0·0052).  Treatment-related adverse events occurred in 465 (93%) patients assigned regorafenib and in 154 (61%) of those assigned placebo.  The most common adverse events of grade three or higher related to regorafenib were hand-foot skin reaction (83 patients, 17%), fatigue (48, 10%), diarrhea (36, 7%), hypertension (36, 7%), and rash or desquamation (29, 6%).Regorafenib is the first small-mol. multikinase inhibitor with survival benefits in metastatic colorectal cancer which has progressed after all std. therapies.  The present study provides evidence for a continuing role of targeted treatment after disease progression, with regorafenib offering a potential new line of therapy in this treatment-refractory population.Bayer HealthCare Pharmaceuticals.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGohw55Jw5C_4LVg90H21EOLACvtfcHk0lhuptuc-oykFg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xhslarsr%252FE&md5=ec2d5286add1c1837bfe7bdb50817b06</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1016%2FS0140-6736%2812%2961900-X&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0140-6736%252812%252961900-X%26sid%3Dliteratum%253Aachs%26aulast%3DGrothey%26aufirst%3DA.%26aulast%3DCutsem%26aufirst%3DE.%2BV.%26aulast%3DSobrero%26aufirst%3DA.%26aulast%3DSiena%26aufirst%3DS.%26aulast%3DFalcone%26aufirst%3DA.%26aulast%3DYchou%26aufirst%3DM.%26aulast%3DHumblet%26aufirst%3DY.%26aulast%3DBouche%26aufirst%3DO.%26aulast%3DMineur%26aufirst%3DL.%26aulast%3DBarone%26aufirst%3DC.%26aulast%3DAdenis%26aufirst%3DA.%26aulast%3DTabernero%26aufirst%3DJ.%26aulast%3DYoshino%26aufirst%3DT.%26aulast%3DLenz%26aufirst%3DH.-J.%26aulast%3DGoldberg%26aufirst%3DR.%2BM%26aulast%3DSargent%26aufirst%3DD.%2BJ%26aulast%3DCihon%26aufirst%3DF.%26aulast%3DCupit%26aufirst%3DL.%26aulast%3DWagner%26aufirst%3DA.%26aulast%3DLaurent%26aufirst%3DD.%26atitle%3DRegorafenib%2520monotherapy%2520for%2520previously%2520treated%2520metastatic%2520colorectal%2520cancer%2520%2528CORRECT%2529%253A%2520an%2520international%252C%2520multicentre%252C%2520randomised%252C%2520placebo-controlled%252C%2520phase%25203%2520trial%26jtitle%3DLancet%26date%3D2013%26volume%3D381%26spage%3D303%26epage%3D312%26doi%3D10.1016%2FS0140-6736%2812%2961900-X" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Demetri, G. D</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reichardt, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kang, Y.-K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blay, J.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rutkowski, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gelderblom, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hohenberger, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leahy, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">von Mehren, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Joensuu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Badalamenti, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blackstein, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Le Cesne, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schoffski, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maki, R. G</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bauer, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen, B. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nishida, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kappeler, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuss, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laurent, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Casali, P. G</span></span> <span> </span><span class="NLM_article-title">Efficacy and safety of regorafenib for advanced gastrointestinal stromal tumours after failure of imatinib and sunitinib (GRID): an international, multicentre, randomised, placebo-controlled, phase 3 trial</span>. <i>Lancet</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>381</i></span>,  <span class="NLM_fpage">295</span>– <span class="NLM_lpage">302</span>, <span class="refDoi"> DOI: 10.1016/S0140-6736(12)61857-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=10.1016%2FS0140-6736%2812%2961857-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=23177515" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=1%3ACAS%3A528%3ADC%252BC38XhslarsLvL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=381&publication_year=2013&pages=295-302&author=G.+D+Demetriauthor=P.+Reichardtauthor=Y.-K.+Kangauthor=J.-Y.+Blayauthor=P.+Rutkowskiauthor=H.+Gelderblomauthor=P.+Hohenbergerauthor=M.+Leahyauthor=M.+von+Mehrenauthor=H.+Joensuuauthor=G.+Badalamentiauthor=M.+Blacksteinauthor=A.+Le+Cesneauthor=P.+Schoffskiauthor=R.+G+Makiauthor=S.+Bauerauthor=B.+B.+Nguyenauthor=J.+Xuauthor=T.+Nishidaauthor=J.+Chungauthor=C.+Kappelerauthor=I.+Kussauthor=D.+Laurentauthor=P.+G+Casali&title=Efficacy+and+safety+of+regorafenib+for+advanced+gastrointestinal+stromal+tumours+after+failure+of+imatinib+and+sunitinib+%28GRID%29%3A+an+international%2C+multicentre%2C+randomised%2C+placebo-controlled%2C+phase+3+trial&doi=10.1016%2FS0140-6736%2812%2961857-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Efficacy and safety of regorafenib for advanced gastrointestinal stromal tumours after failure of imatinib and sunitinib (GRID): an international, multicentre, randomised, placebo-controlled, phase 3 trial</span></div><div class="casAuthors">Demetri, George D.; Reichardt, Peter; Kang, Yoon-Koo; Blay, Jean-Yves; Rutkowski, Piotr; Gelderblom, Hans; Hohenberger, Peter; Leahy, Michael; von Mehren, Margaret; Joensuu, Heikki; Badalamenti, Giuseppe; Blackstein, Martin; Le Cesne, Axel; Schoffski, Patrick; Maki, Robert G.; Bauer, Sebastian; Nguyen, Binh Bui; Xu, Jianming; Nishida, Toshirou; Chung, John; Kappeler, Christian; Kuss, Iris; Laurent, Dirk; Casali, Paolo G.</div><div class="citationInfo"><span class="NLM_cas:title">Lancet</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">381</span>
        (<span class="NLM_cas:issue">9863</span>),
    <span class="NLM_cas:pages">295-302</span>CODEN:
                <span class="NLM_cas:coden">LANCAO</span>;
        ISSN:<span class="NLM_cas:issn">0140-6736</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Until now, only imatinib and sunitinib have proven clin. benefit in patients with gastrointestinal stromal tumors (GIST), but almost all metastatic GIST eventually develop resistance to these agents, resulting in fatal disease progression.  We aimed to assess efficacy and safety of regorafenib in patients with metastatic or unresectable GIST progressing after failure of at least imatinib and sunitinib.We did this phase 3 trial at 57 hospitals in 17 countries.  Patients with histol. confirmed, metastatic or unresectable GIST, with failure of at least previous imatinib and sunitinib were randomised in a 2:1 ratio (by computer-generated randomisation list and interactive voice response system; preallocated block design (block size 12); stratified by treatment line and geog. region) to receive either oral regorafenib 160 mg daily or placebo, plus best supportive care in both groups, for the first 3 wk of each 4 wk cycle.  The study sponsor, participants, and investigators were masked to treatment assignment.  The primary endpoint was progression-free survival (PFS).  At disease progression, patients assigned placebo could crossover to open-label regorafenib.  Analyses were by intention to treat.  This trial is registered with ClinicalTrials.gov, no. NCT01271712.From Jan 4, to Aug 18, 2011, 240 patients were screened and 199 were randomised to receive regorafenib (n=133) or matching placebo (n=66).  Data cutoff was Jan 26, 2012.  Median PFS per independent blinded central review was 4·8 mo (IQR 1·4-9·2) for regorafenib and 0·9 mo (0·9-1·8) for placebo (hazard ratio [HR] 0·27, 95% CI 0·19-0·39; p<0·0001).  After progression, 56 patients (85%) assigned placebo crossed over to regorafenib.  Drug-related adverse events were reported in 130 (98%) patients assigned regorafenib and 45 (68%) patients assigned placebo.  The most common regorafenib-related adverse events of grade 3 or higher were hypertension (31 of 132, 23%), hand-foot skin reaction (26 of 132, 20%), and diarrhoea (seven of 132, 5%).The results of this study show that oral regorafenib can provide a significant improvement in progression-free survival compared with placebo in patients with metastatic GIST after progression on std. treatments.  As far as we are aware, this is the first clin. trial to show benefit from a kinase inhibitor in this highly refractory population of patients.Bayer HealthCare Pharmaceuticals.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoN2GIfYXBPyLVg90H21EOLACvtfcHk0ljtfI_ie1n1cA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhslarsLvL&md5=f8ee42424a9b3bd7558b1a2f1461af33</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1016%2FS0140-6736%2812%2961857-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0140-6736%252812%252961857-1%26sid%3Dliteratum%253Aachs%26aulast%3DDemetri%26aufirst%3DG.%2BD%26aulast%3DReichardt%26aufirst%3DP.%26aulast%3DKang%26aufirst%3DY.-K.%26aulast%3DBlay%26aufirst%3DJ.-Y.%26aulast%3DRutkowski%26aufirst%3DP.%26aulast%3DGelderblom%26aufirst%3DH.%26aulast%3DHohenberger%26aufirst%3DP.%26aulast%3DLeahy%26aufirst%3DM.%26aulast%3Dvon%2BMehren%26aufirst%3DM.%26aulast%3DJoensuu%26aufirst%3DH.%26aulast%3DBadalamenti%26aufirst%3DG.%26aulast%3DBlackstein%26aufirst%3DM.%26aulast%3DLe%2BCesne%26aufirst%3DA.%26aulast%3DSchoffski%26aufirst%3DP.%26aulast%3DMaki%26aufirst%3DR.%2BG%26aulast%3DBauer%26aufirst%3DS.%26aulast%3DNguyen%26aufirst%3DB.%2BB.%26aulast%3DXu%26aufirst%3DJ.%26aulast%3DNishida%26aufirst%3DT.%26aulast%3DChung%26aufirst%3DJ.%26aulast%3DKappeler%26aufirst%3DC.%26aulast%3DKuss%26aufirst%3DI.%26aulast%3DLaurent%26aufirst%3DD.%26aulast%3DCasali%26aufirst%3DP.%2BG%26atitle%3DEfficacy%2520and%2520safety%2520of%2520regorafenib%2520for%2520advanced%2520gastrointestinal%2520stromal%2520tumours%2520after%2520failure%2520of%2520imatinib%2520and%2520sunitinib%2520%2528GRID%2529%253A%2520an%2520international%252C%2520multicentre%252C%2520randomised%252C%2520placebo-controlled%252C%2520phase%25203%2520trial%26jtitle%3DLancet%26date%3D2013%26volume%3D381%26spage%3D295%26epage%3D302%26doi%3D10.1016%2FS0140-6736%2812%2961857-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bruix, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Merle, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Granito, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Y.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bodoky, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pracht, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yokosuka, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosmorduc, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Breder, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerolami, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Masi, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ross, P. J</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bronowicki, J.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ollivier-Hourmand, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kudo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, A.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Llovet, J. M</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Finn, R. S</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">LeBerre, M.-A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baumhauer, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meinhardt, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, G.</span></span> <span> </span><span class="NLM_article-title">Regorafenib for patients with hepatocellular carcinoma who progressed on sorafenib treatment (RESORCE): a randomised, double-blind, placebo-controlled, phase 3 trial</span>. <i>Lancet</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>389</i></span>,  <span class="NLM_fpage">56</span>– <span class="NLM_lpage">66</span>, <span class="refDoi"> DOI: 10.1016/S0140-6736(16)32453-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=10.1016%2FS0140-6736%2816%2932453-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=27932229" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=1%3ACAS%3A528%3ADC%252BC28XitVajsrrN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=389&publication_year=2017&pages=56-66&author=J.+Bruixauthor=S.+Qinauthor=P.+Merleauthor=A.+Granitoauthor=Y.-H.+Huangauthor=G.+Bodokyauthor=M.+Prachtauthor=O.+Yokosukaauthor=O.+Rosmorducauthor=V.+Brederauthor=R.+Gerolamiauthor=G.+Masiauthor=P.+J+Rossauthor=T.+Songauthor=J.-P.+Bronowickiauthor=I.+Ollivier-Hourmandauthor=M.+Kudoauthor=A.-L.+Chengauthor=J.+M+Llovetauthor=R.+S+Finnauthor=M.-A.+LeBerreauthor=A.+Baumhauerauthor=G.+Meinhardtauthor=G.+Han&title=Regorafenib+for+patients+with+hepatocellular+carcinoma+who+progressed+on+sorafenib+treatment+%28RESORCE%29%3A+a+randomised%2C+double-blind%2C+placebo-controlled%2C+phase+3+trial&doi=10.1016%2FS0140-6736%2816%2932453-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Regorafenib for patients with hepatocellular carcinoma who progressed on sorafenib treatment (RESORCE): a randomised, double-blind, placebo-controlled, phase 3 trial</span></div><div class="casAuthors">Bruix, Jordi; Qin, Shukui; Merle, Philippe; Granito, Alessandro; Huang, Yi-Hsiang; Bodoky, Gyorgy; Pracht, Marc; Yokosuka, Osamu; Rosmorduc, Olivier; Breder, Valeriy; Gerolami, Rene; Masi, Gianluca; Ross, Paul J.; Song, Tianqiang; Bronowicki, Jean-Pierre; Ollivier-Hourmand, Isabelle; Kudo, Masatoshi; Cheng, Ann-Lii; Llovet, Josep M.; Finn, Richard S.; LeBerre, Marie-Aude; Baumhauer, Annette; Meinhardt, Gerold; Han, Guohong</div><div class="citationInfo"><span class="NLM_cas:title">Lancet</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">389</span>
        (<span class="NLM_cas:issue">10064</span>),
    <span class="NLM_cas:pages">56-66</span>CODEN:
                <span class="NLM_cas:coden">LANCAO</span>;
        ISSN:<span class="NLM_cas:issn">0140-6736</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">There are no systemic treatments for patients with hepatocellular carcinoma (HCC) whose disease progresses during sorafenib treatment.  We aimed to assess the efficacy and safety of regorafenib in patients with HCC who have progressed during sorafenib treatment.  In this randomized, double-blind, parallel-group, phase 3 trial done at 152 sites in 21 countries, adults with HCC who tolerated sorafenib (≥400 mg/day for ≥20 of last 28 days of treatment), progressed on sorafenib, and had Child-Pugh A liver function were enrolled.  Participants were randomly assigned (2:1) by a computer-generated randomization list and interactive voice response system and stratified by geog. region, Eastern Cooperative Oncol. Group performance status, macrovascular invasion, extrahepatic disease, and α-fetoprotein level to best supportive care plus oral regorafenib 160 mg or placebo once daily during weeks 1-3 of each 4-wk cycle.  Investigators, patients, and the funder were masked to treatment assignment.  The primary endpoint was overall survival (defined as time from randomization to death due to any cause) and analyzed by intention to treat.  This trial is registered with ClinicalTrials.gov, no. NCT01774344.  Between May 14, 2013, and Dec 31, 2015, 843 patients were screened, of whom 573 were enrolled and randomized (379 to regorafenib and 194 to placebo; population for efficacy analyses), and 567 initiated treatment (374 received regorafenib and 193 received placebo; population for safety analyses).  Regorafenib improved overall survival with a hazard ratio of 0·63 (95% CI 0·50-0·79; one-sided p<0·0001); median survival was 10·6 mo (95% CI 9·1-12·1) for regorafenib vs. 7·8 mo (6·3-8·8) for placebo.  Adverse events were reported in all regorafenib recipients (374 [100%] of 374) and 179 (93%) of 193 placebo recipients.  The most common clin. relevant grade 3 or 4 treatment-emergent events were hypertension (57 patients [15%] in the regorafenib group vs nine patients [5%] in the placebo group), hand-foot skin reaction (47 patients [13%] vs one [1%]), fatigue (34 patients [9%] vs nine patients [5%]), and diarrhea (12 patients [3%] vs no patients).  Of the 88 deaths (grade 5 adverse events) reported during the study (50 patients [13%] assigned to regorafenib and 38 [20%] assigned to placebo), seven (2%) were considered by the investigator to be related to study drug in the regorafenib group and two (1%) in the placebo group, including two patients (1%) with hepatic failure in the placebo group.  Regorafenib is the only systemic treatment shown to provide survival benefit in HCC patients progressing on sorafenib treatment.  Future trials should explore combinations of regorafenib with other systemic agents and third-line treatments for patients who fail or who do not tolerate the sequence of sorafenib and regorafenib.Bayer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrY-TirMzLgvrVg90H21EOLACvtfcHk0lh5otK6KckIyg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitVajsrrN&md5=bdfcd7a16ed161777b9960bb52c0a719</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1016%2FS0140-6736%2816%2932453-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0140-6736%252816%252932453-9%26sid%3Dliteratum%253Aachs%26aulast%3DBruix%26aufirst%3DJ.%26aulast%3DQin%26aufirst%3DS.%26aulast%3DMerle%26aufirst%3DP.%26aulast%3DGranito%26aufirst%3DA.%26aulast%3DHuang%26aufirst%3DY.-H.%26aulast%3DBodoky%26aufirst%3DG.%26aulast%3DPracht%26aufirst%3DM.%26aulast%3DYokosuka%26aufirst%3DO.%26aulast%3DRosmorduc%26aufirst%3DO.%26aulast%3DBreder%26aufirst%3DV.%26aulast%3DGerolami%26aufirst%3DR.%26aulast%3DMasi%26aufirst%3DG.%26aulast%3DRoss%26aufirst%3DP.%2BJ%26aulast%3DSong%26aufirst%3DT.%26aulast%3DBronowicki%26aufirst%3DJ.-P.%26aulast%3DOllivier-Hourmand%26aufirst%3DI.%26aulast%3DKudo%26aufirst%3DM.%26aulast%3DCheng%26aufirst%3DA.-L.%26aulast%3DLlovet%26aufirst%3DJ.%2BM%26aulast%3DFinn%26aufirst%3DR.%2BS%26aulast%3DLeBerre%26aufirst%3DM.-A.%26aulast%3DBaumhauer%26aufirst%3DA.%26aulast%3DMeinhardt%26aufirst%3DG.%26aulast%3DHan%26aufirst%3DG.%26atitle%3DRegorafenib%2520for%2520patients%2520with%2520hepatocellular%2520carcinoma%2520who%2520progressed%2520on%2520sorafenib%2520treatment%2520%2528RESORCE%2529%253A%2520a%2520randomised%252C%2520double-blind%252C%2520placebo-controlled%252C%2520phase%25203%2520trial%26jtitle%3DLancet%26date%3D2017%26volume%3D389%26spage%3D56%26epage%3D66%26doi%3D10.1016%2FS0140-6736%2816%2932453-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wilhelm, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dumas, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adnane, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lynch, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carter, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schütz, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thierauch, K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zopf, D.</span></span> <span> </span><span class="NLM_article-title">Regorafenib (BAY 73-4506): a new oral multikinase inhibitor of angiogenic, stromal and oncogenic receptor tyrosine kinases with potent preclinical antitumor activity</span>. <i>Int. J. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>129</i></span>,  <span class="NLM_fpage">245</span>– <span class="NLM_lpage">255</span>, <span class="refDoi"> DOI: 10.1002/ijc.25864</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=10.1002%2Fijc.25864" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=21170960" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=1%3ACAS%3A528%3ADC%252BC3MXltFGlu7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=129&publication_year=2011&pages=245-255&author=S.+M.+Wilhelmauthor=J.+Dumasauthor=L.+Adnaneauthor=M.+Lynchauthor=C.+A.+Carterauthor=G.+Sch%C3%BCtzauthor=K.+H.+Thierauchauthor=D.+Zopf&title=Regorafenib+%28BAY+73-4506%29%3A+a+new+oral+multikinase+inhibitor+of+angiogenic%2C+stromal+and+oncogenic+receptor+tyrosine+kinases+with+potent+preclinical+antitumor+activity&doi=10.1002%2Fijc.25864"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Regorafenib (BAY 73-4506): A new oral multikinase inhibitor of angiogenic, stromal and oncogenic receptor tyrosine kinases with potent preclinical antitumor activity</span></div><div class="casAuthors">Wilhelm, Scott M.; Dumas, Jacques; Adnane, Lila; Lynch, Mark; Carter, Christopher A.; Schuetz, Gunnar; Thierauch, Karl-Heinz; Zopf, Dieter</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Cancer</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">129</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">245-255</span>CODEN:
                <span class="NLM_cas:coden">IJCNAW</span>;
        ISSN:<span class="NLM_cas:issn">0020-7136</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Liss, Inc.</span>)
        </div><div class="casAbstract">Angiogenesis, a crit. driver of tumor development, is controlled by interconnected signaling pathways.  Vascular endothelial growth factor receptor (VEGFR) 2 and tyrosine kinase with Ig and epidermal growth factor homol. domain 2 play crucial roles in the biol. of normal and tumor vasculature.  Regorafenib (BAY 73-4506), a novel oral multikinase inhibitor, potently inhibits these endothelial cell kinases in biochem. and cellular kinase phosphorylation assays.  Furthermore, regorafenib inhibits addnl. angiogenic kinases (VEGFR1/3, platelet-derived growth factor receptor-β and fibroblast growth factor receptor 1) and the mutant oncogenic kinases KIT, RET and B-RAF.  The antiangiogenic effect of regorafenib was demonstrated in vivo by dynamic contrast-enhanced magnetic resonance imaging.  Regorafenib administered once orally at 10 mg/kg significantly decreased the extravasation of Gadomer in the vasculature of rat GS9L glioblastoma tumor xenografts.  In a daily (qd)×4 dosing study, the pharmacodynamic effects persisted for 48 h after the last dosing and correlated with tumor growth inhibition (TGI).  A significant redn. in tumor microvessel area was obsd. in a human colorectal xenograft after qd×5 dosing at 10 and 30 mg/kg.  Regorafenib exhibited potent dose-dependent TGI in various preclin. human xenograft models in mice, with tumor shrinkages obsd. in breast MDA-MB-231 and renal 786-O carcinoma models.  Pharmacodynamic analyses of the breast model revealed strong redn. in staining of proliferation marker Ki-67 and phosphorylated extracellular regulated kinases 1/2.  These data demonstrate that regorafenib is a well-tolerated, orally active multikinase inhibitor with a distinct target profile that may have therapeutic benefit in human malignancies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrtQpQXfCq04rVg90H21EOLACvtfcHk0lh5otK6KckIyg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXltFGlu7c%253D&md5=13eee29ef1f1f03fd1ed311ea613f566</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1002%2Fijc.25864&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fijc.25864%26sid%3Dliteratum%253Aachs%26aulast%3DWilhelm%26aufirst%3DS.%2BM.%26aulast%3DDumas%26aufirst%3DJ.%26aulast%3DAdnane%26aufirst%3DL.%26aulast%3DLynch%26aufirst%3DM.%26aulast%3DCarter%26aufirst%3DC.%2BA.%26aulast%3DSch%25C3%25BCtz%26aufirst%3DG.%26aulast%3DThierauch%26aufirst%3DK.%2BH.%26aulast%3DZopf%26aufirst%3DD.%26atitle%3DRegorafenib%2520%2528BAY%252073-4506%2529%253A%2520a%2520new%2520oral%2520multikinase%2520inhibitor%2520of%2520angiogenic%252C%2520stromal%2520and%2520oncogenic%2520receptor%2520tyrosine%2520kinases%2520with%2520potent%2520preclinical%2520antitumor%2520activity%26jtitle%3DInt.%2520J.%2520Cancer%26date%3D2011%26volume%3D129%26spage%3D245%26epage%3D255%26doi%3D10.1002%2Fijc.25864" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Koul, H. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pal, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koul, S.</span></span> <span> </span><span class="NLM_article-title">Role of p38 MAP kinase signal transduction in solid tumors</span>. <i>Genes Cancer</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">342</span>– <span class="NLM_lpage">359</span>, <span class="refDoi"> DOI: 10.1177/1947601913507951</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=10.1177%2F1947601913507951" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=24349632" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsVGktbfP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2013&pages=342-359&author=H.+K.+Koulauthor=M.+Palauthor=S.+Koul&title=Role+of+p38+MAP+kinase+signal+transduction+in+solid+tumors&doi=10.1177%2F1947601913507951"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Role of p38 MAP kinase signal transduction in solid tumors</span></div><div class="casAuthors">Koul, Hari K.; Pal, Mintu; Koul, Sweaty</div><div class="citationInfo"><span class="NLM_cas:title">Genes & Cancer</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">9-10</span>),
    <span class="NLM_cas:pages">342-359, 18 pp.</span>CODEN:
                <span class="NLM_cas:coden">GCEAAY</span>;
        ISSN:<span class="NLM_cas:issn">1947-6019</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">A review.  Mitogen-activated protein kinases (MAPKs) mediate a wide variety of cellular behaviors in response to extracellular stimuli.  One of the main subgroups, the p38 MAP kinases, has been implicated in a wide range of complex biol. processes, such as cell proliferation, cell differentiation, cell death, cell migration, and invasion.  Dysregulation of p38 MAPK levels in patients are assocd. with advanced stages and short survival in cancer patients (e.g., prostate, breast, bladder, liver, and lung cancer).  P38 MAPK plays a dual role as a regulator of cell death, and it can either mediate cell survival or cell death depending not only on the type of stimulus but also in a cell type specific manner.  In addn. to modulating cell survival, an essential role of p38 MAPK in modulation of cell migration and invasion offers a distinct opportunity to target this pathway with respect to tumor metastasis.  The specific function of p38 MAPK appears to depend not only on the cell type but also on the stimuli and/or the isoform that is activated.  P38 MAPK signaling pathway is activated in response to diverse stimuli and mediates its function by components downstream of p38.  Extrapolation of the knowledge gained from lab. findings is essential to address the clin. significance of p38 MAPK signaling pathways.  The goal of this review is to provide an overview on recent progress made in defining the functions of p38 MAPK pathways with respect to solid tumor biol. and generate testable hypothesis with respect to the role of p38 MAPK as an attractive target for intervention of solid tumors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoB6A4yxL-gLLVg90H21EOLACvtfcHk0lhLZYtqx1CodQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsVGktbfP&md5=a7803ce5e47e58b277f614616e526209</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1177%2F1947601913507951&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1947601913507951%26sid%3Dliteratum%253Aachs%26aulast%3DKoul%26aufirst%3DH.%2BK.%26aulast%3DPal%26aufirst%3DM.%26aulast%3DKoul%26aufirst%3DS.%26atitle%3DRole%2520of%2520p38%2520MAP%2520kinase%2520signal%2520transduction%2520in%2520solid%2520tumors%26jtitle%3DGenes%2520Cancer%26date%3D2013%26volume%3D4%26spage%3D342%26epage%3D359%26doi%3D10.1177%2F1947601913507951" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lange, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lo, H.-W.</span></span> <span> </span><span class="NLM_article-title">Inhibiting TRK proteins in clinical cancer therapy</span>. <i>Cancers</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">105</span>, <span class="refDoi"> DOI: 10.3390/cancers10040105</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=10.3390%2Fcancers10040105" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhtV2gtL%252FI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2018&pages=105&author=A.+Langeauthor=H.-W.+Lo&title=Inhibiting+TRK+proteins+in+clinical+cancer+therapy&doi=10.3390%2Fcancers10040105"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibiting TRK proteins in clinical cancer therapy</span></div><div class="casAuthors">Lange, Allison M.; Lo, Hui-Wen</div><div class="citationInfo"><span class="NLM_cas:title">Cancers</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">105/1-105/15</span>CODEN:
                <span class="NLM_cas:coden">CANCCT</span>;
        ISSN:<span class="NLM_cas:issn">2072-6694</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">Gene rearrangements resulting in the aberrant activity of tyrosine kinases have been identified as drivers of oncogenesis in a variety of cancers.  The tropomyosin receptor kinase (TRK) family of tyrosine receptor kinases is emerging as an important target for cancer therapeutics.  The TRK family contains three members, TRKA, TRKB, and TRKC, and these proteins are encoded by the genes NTRK1, NTRK2, and NTRK3, resp.  To activate TRK receptors, neurotrophins bind to the extracellular region stimulating dimerization, phosphorylation, and activation of downstream signaling pathways.  Major known downstream pathways include RAS/MAPK/ERK, PLCγ, and PI3K/Akt.  While being rare in most cancers, TRK fusions with other proteins have been well-established as oncogenic events in specific malignancies, including glioblastoma, papillary thyroid carcinoma, and secretory breast carcinomas.  TRK protein amplification as well as alternative splicing events have also been described as contributors to cancer pathogenesis.  For patients harboring alterations in TRK expression or activity, TRK inhibition emerges as an important therapeutic target.  To date, multiple trials testing TRK-inhibiting compds. in various cancers are underway.  In this review, we will summarize the current therapeutic trials for neoplasms involving NTKR gene alterations, as well as the promises and setbacks that are assocd. with targeting gene fusions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqvepDLt-_JaLVg90H21EOLACvtfcHk0lhLZYtqx1CodQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhtV2gtL%252FI&md5=f421f06adf15f9357d88b80bf44479e8</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.3390%2Fcancers10040105&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fcancers10040105%26sid%3Dliteratum%253Aachs%26aulast%3DLange%26aufirst%3DA.%26aulast%3DLo%26aufirst%3DH.-W.%26atitle%3DInhibiting%2520TRK%2520proteins%2520in%2520clinical%2520cancer%2520therapy%26jtitle%3DCancers%26date%3D2018%26volume%3D10%26spage%3D105%26doi%3D10.3390%2Fcancers10040105" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sirvent, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lafitte, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roche, S.</span></span> <span> </span><span class="NLM_article-title">DDR1 inhibition as a new therapeutic strategy for colorectal cancer</span>. <i>Mol. Cell Oncol</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>5</i></span>, <span class="NLM_elocation-id">e1465882</span> <span class="refDoi"> DOI: 10.1080/23723556.2018.1465882</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=10.1080%2F23723556.2018.1465882" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=30250919" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtlyrtrbJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2018&author=A.+Sirventauthor=M.+Lafitteauthor=S.+Roche&title=DDR1+inhibition+as+a+new+therapeutic+strategy+for+colorectal+cancer&doi=10.1080%2F23723556.2018.1465882"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">DDR1 inhibition as a new therapeutic strategy for colorectal cancer</span></div><div class="casAuthors">Sirvent, Audrey; Lafitte, Marie; Roche, Serge</div><div class="citationInfo"><span class="NLM_cas:title">Molecular & Cellular Oncology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">e1465882/1-e1465882/3</span>CODEN:
                <span class="NLM_cas:coden">MCOOCO</span>;
        ISSN:<span class="NLM_cas:issn">2372-3556</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis, Inc.</span>)
        </div><div class="casAbstract">The clin. management of metastatic colorectal cancer (mCRC) is still a major challenge.  Recently, we discovered that nilotinib, an approved treatment for chronic myeloid leukemia, inhibits invasive and metastatic properties of CRC cells by targeting the kinase activity of receptor for collagens DDR1 (Discoidin Domain Receptor tyrosine kinase 1), suggesting that nilotinib could be an effective strategy to treat mCRC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpyexaLpIY9RLVg90H21EOLACvtfcHk0li9_gN9smGfow"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtlyrtrbJ&md5=794bf62b12a6327947f23c5a610e3bf9</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1080%2F23723556.2018.1465882&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F23723556.2018.1465882%26sid%3Dliteratum%253Aachs%26aulast%3DSirvent%26aufirst%3DA.%26aulast%3DLafitte%26aufirst%3DM.%26aulast%3DRoche%26aufirst%3DS.%26atitle%3DDDR1%2520inhibition%2520as%2520a%2520new%2520therapeutic%2520strategy%2520for%2520colorectal%2520cancer%26jtitle%3DMol.%2520Cell%2520Oncol%26date%3D2018%26volume%3D5%26doi%3D10.1080%2F23723556.2018.1465882" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ren, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, K.</span></span> <span> </span><span class="NLM_article-title">Small molecule discoidin domain receptor kinase inhibitors and potential medical applications</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">3287</span>– <span class="NLM_lpage">3301</span>, <span class="refDoi"> DOI: 10.1021/jm5012319</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm5012319" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=1%3ACAS%3A528%3ADC%252BC2MXns1ylsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=3287-3301&author=Y.+Liauthor=X.+Luauthor=X.+Renauthor=K.+Ding&title=Small+molecule+discoidin+domain+receptor+kinase+inhibitors+and+potential+medical+applications&doi=10.1021%2Fjm5012319"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Small Molecule Discoidin Domain Receptor Kinase Inhibitors and Potential Medical Applications</span></div><div class="casAuthors">Li, Yupeng; Lu, Xiaoyun; Ren, Xiaomei; Ding, Ke</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">3287-3301</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Discoidin domain receptors (DDRs) are members of the transmembrane receptor tyrosine kinase (RTK) superfamily which are distinguished from others by the presence of a discoidin motif in the extracellular domain and their utilization of collagens as internal ligands.  Two types of DDRs, DDR1 and DDR2, have been identified with distinct expression profiles and ligand specificities.  These DDRs play important roles in the regulation of fundamental cellular process, such as proliferation, survival, differentiation, adhesion, and matrix remodeling.  They have also been closely linked to a no. of human diseases, including various fibrotic disorders, atherosclerosis, and cancer.  As a consequence, DDRs have been considered as novel potential mol. targets for drug discovery and increasing efforts are being devoted to the identification of new small mol. inhibitors targeting the receptors.  In this review, we offer a contemporary overview on the discovery of DDRs inhibitors and their potential medical application for the treatment of cancer and inflammation related disorders.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoVUPSkCQYF-LVg90H21EOLACvtfcHk0li9_gN9smGfow"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXns1ylsQ%253D%253D&md5=71249675e4ed65d538956f8c9cb7d387</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1021%2Fjm5012319&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm5012319%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DY.%26aulast%3DLu%26aufirst%3DX.%26aulast%3DRen%26aufirst%3DX.%26aulast%3DDing%26aufirst%3DK.%26atitle%3DSmall%2520molecule%2520discoidin%2520domain%2520receptor%2520kinase%2520inhibitors%2520and%2520potential%2520medical%2520applications%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D3287%26epage%3D3301%26doi%3D10.1021%2Fjm5012319" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tanihara, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Inoue, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamamoto, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuwayama, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abe, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Araie, M.</span></span> <span> </span><span class="NLM_article-title">Phase 2 randomized clinical study of a Rho kinase inhibitor, K-115, in primary open-angle glaucoma and ocular hypertension</span>. <i>Am. J. Ophthalmol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>156</i></span>,  <span class="NLM_fpage">731</span>– <span class="NLM_lpage">736</span>, <span class="refDoi"> DOI: 10.1016/j.ajo.2013.05.016</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=10.1016%2Fj.ajo.2013.05.016" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=23831221" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtVKrur7K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=156&publication_year=2013&pages=731-736&author=H.+Taniharaauthor=T.+Inoueauthor=T.+Yamamotoauthor=Y.+Kuwayamaauthor=H.+Abeauthor=M.+Araie&title=Phase+2+randomized+clinical+study+of+a+Rho+kinase+inhibitor%2C+K-115%2C+in+primary+open-angle+glaucoma+and+ocular+hypertension&doi=10.1016%2Fj.ajo.2013.05.016"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Phase 2 Randomized Clinical Study of a Rho Kinase Inhibitor, K-115, in Primary Open-Angle Glaucoma and Ocular Hypertension</span></div><div class="casAuthors">Tanihara, Hidenobu; Inoue, Toshihiro; Yamamoto, Tetsuya; Kuwayama, Yasuaki; Abe, Haruki; Araie, Makoto</div><div class="citationInfo"><span class="NLM_cas:title">American Journal of Ophthalmology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">156</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">731-736.e2</span>CODEN:
                <span class="NLM_cas:coden">AJOPAA</span>;
        ISSN:<span class="NLM_cas:issn">0002-9394</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">To identify the optimal dose of a novel Rho kinase inhibitor, K-115, by assessing dose dependency of the intraocular pressure (IOP)-lowering effects and the safety in patients with primary open-angle glaucoma or ocular hypertension.  Multicenter, prospective, randomized, placebo-controlled, double-masked, parallel group comparison clin. study.  After appropriate washout periods, 210 patients with primary open-angle glaucoma or ocular hypertension were subdivided into 4 groups and were treated with K-115 in concns. of 0.1%, 0.2%, and 0.4% or placebo twice daily for 8 wk.  The dose response of IOP redn. and the incidence of adverse events by K-115 or placebo were investigated.The mean baseline IOP was between 23.0 and 23.4 mm Hg.  The mean IOP redns. of the last visit from baseline were -2.2 mm Hg, -3.4 mm Hg, -3.2 mm Hg, and -3.5 mm Hg, resp., in the placebo, 0.1%, 0.2%, and 0.4% groups at before instillation (9:00); -2.5 mm Hg, -3.7 mm Hg, -4.2 mm Hg, and -4.5 mm Hg at 2 h after instillation (11:00); and -1.9 mm Hg, -3.2 mm Hg, -2.7 mm Hg, and -3.1 mm Hg at 8 h after instillation (17:00).  The dose-dependent IOP-lowering effect of K-115 was statistically significant at all time points.  Also, conjunctival hyperemia was found in 7 (13.0%) of 54 patients for placebo, 23 (43.4%) of 53 patients for the 0.1% group, 31 (57.4%) of 54 patients for the 0.2% group, and 32 (65.3%) of 49 patients for the 0.4% group.  On the basis of this dose-response study, K-115 0.4% has been selected to be the optimal dose and has the potential to be a promising new agent for glaucoma to control 24-h IOP by twice-daily dosing.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrq9Y_00OjhnbVg90H21EOLACvtfcHk0lgQtmAA1FxCXA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtVKrur7K&md5=075dc5531be1a7bdf44d9e21448df783</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1016%2Fj.ajo.2013.05.016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ajo.2013.05.016%26sid%3Dliteratum%253Aachs%26aulast%3DTanihara%26aufirst%3DH.%26aulast%3DInoue%26aufirst%3DT.%26aulast%3DYamamoto%26aufirst%3DT.%26aulast%3DKuwayama%26aufirst%3DY.%26aulast%3DAbe%26aufirst%3DH.%26aulast%3DAraie%26aufirst%3DM.%26atitle%3DPhase%25202%2520randomized%2520clinical%2520study%2520of%2520a%2520Rho%2520kinase%2520inhibitor%252C%2520K-115%252C%2520in%2520primary%2520open-angle%2520glaucoma%2520and%2520ocular%2520hypertension%26jtitle%3DAm.%2520J.%2520Ophthalmol.%26date%3D2013%26volume%3D156%26spage%3D731%26epage%3D736%26doi%3D10.1016%2Fj.ajo.2013.05.016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Prêtre, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wicki, A.</span></span> <span> </span><span class="NLM_article-title">Inhibition of Akt and other AGC kinases: A target for clinical cancer therapy?</span>. <i>Semin. Cancer Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>48</i></span>,  <span class="NLM_fpage">70</span>– <span class="NLM_lpage">77</span>, <span class="refDoi"> DOI: 10.1016/j.semcancer.2017.04.011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=10.1016%2Fj.semcancer.2017.04.011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=28473255" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=1%3ACAS%3A528%3ADC%252BC2sXotlOrurc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2018&pages=70-77&author=V.+Pr%C3%AAtreauthor=A.+Wicki&title=Inhibition+of+Akt+and+other+AGC+kinases%3A+A+target+for+clinical+cancer+therapy%3F&doi=10.1016%2Fj.semcancer.2017.04.011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of Akt and other AGC kinases: A target for clinical cancer therapy?</span></div><div class="casAuthors">Pretre, Vincent; Wicki, Andreas</div><div class="citationInfo"><span class="NLM_cas:title">Seminars in Cancer Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">70-77</span>CODEN:
                <span class="NLM_cas:coden">SECBE7</span>;
        ISSN:<span class="NLM_cas:issn">1044-579X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  AGC kinases have been identified to contribute to cancer development and progression.  Currently, most AGC inhibitors in clin. development are Akt inhibitors such as MK-2206 or GDC-0068, which are known to promote cell growth arrest and to sensitize cancer cells to radiotherapy.  Response rates in clin. trials with single agent Akt inhibitors are typically low.  The obsd. adverse events are within the expected limits for compds. inhibiting the PI3K-mTOR axis.  Preclin. and early clin. data for combination therapies are accumulating.  Based on these data, several Akt inhibitors are about to enter phase 3 trials.  Besides drugs that target Akt, p70S6K inhibitors have entered clin. development.  Again, the response rates were rather low.  In addn., relevant toxicities were identified, including a risk for coagulopathies with these compds.  Multi-AGC kinase inhibitors are also in early clin. development but the data is not sufficient yet to draw conclusions regarding their efficacy and side-effect profile.  PKC inhibitors have been tested in the phase 3 setting but were found to lack efficacy.  More trials with isoform-specific PKC inhibitors are expected.  Taken together, therapies with AGC kinase inhibitors as single agents are unlikely to meet success.  However, combination therapies and a precise stratification of patients according to the activation of signaling axes may increase the probability to see relevant efficacy with these compds.  The emergence of onco-immunotherapies holds some new challenges for these agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoHcf3ttp3ohbVg90H21EOLACvtfcHk0lgQtmAA1FxCXA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXotlOrurc%253D&md5=9181c4968a872207d6b3b9bf3ba98881</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1016%2Fj.semcancer.2017.04.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.semcancer.2017.04.011%26sid%3Dliteratum%253Aachs%26aulast%3DPr%25C3%25AAtre%26aufirst%3DV.%26aulast%3DWicki%26aufirst%3DA.%26atitle%3DInhibition%2520of%2520Akt%2520and%2520other%2520AGC%2520kinases%253A%2520A%2520target%2520for%2520clinical%2520cancer%2520therapy%253F%26jtitle%3DSemin.%2520Cancer%2520Biol.%26date%3D2018%26volume%3D48%26spage%3D70%26epage%3D77%26doi%3D10.1016%2Fj.semcancer.2017.04.011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Browning, D. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kwon, I.-K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, R.</span></span> <span> </span><span class="NLM_article-title">cGMP-dependent protein kinases as potential targets for colon cancer prevention and treatment</span>. <i>Future Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>2</i></span>,  <span class="NLM_fpage">65</span>– <span class="NLM_lpage">80</span>, <span class="refDoi"> DOI: 10.4155/fmc.09.142</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=10.4155%2Ffmc.09.142" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=21426046" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=1%3ACAS%3A528%3ADC%252BC3cXosVSisw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2010&pages=65-80&author=D.+D.+Browningauthor=I.-K.+Kwonauthor=R.+Wang&title=cGMP-dependent+protein+kinases+as+potential+targets+for+colon+cancer+prevention+and+treatment&doi=10.4155%2Ffmc.09.142"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">cGMP-dependent protein kinases as potential targets for colon cancer prevention and treatment</span></div><div class="casAuthors">Browning, Darren D.; Kwon, In-Kiu; Wang, Rui</div><div class="citationInfo"><span class="NLM_cas:title">Future Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">65-80</span>CODEN:
                <span class="NLM_cas:coden">FMCUA7</span>;
        ISSN:<span class="NLM_cas:issn">1756-8919</span>.
    
            (<span class="NLM_cas:orgname">Future Science Ltd.</span>)
        </div><div class="casAbstract">A review.  In recent years, several antitumor signaling pathways mediated by the cGMP-dependent protein kinases have been identified in colon cancer cells.  This review aims to present the mounting evidence in favor of cGMP/protein kinase G (PKG) signaling as a therapeutic strategy in colon cancer.  The homeostatic and tumor suppressive effects of cGMP in the intestine are uncontested, but the signaling details are not understood.  PKG is the central cGMP effector, and can block proliferation and tumor angiogenesis by inhibiting β-catenin/TCF and SOX9 signaling.  Therapeutic activation of cGMP/PKG offers a promising avenue for the prevention and treatment of colon cancer, but addnl. preclin. studies are needed to fully understand the potential of this system.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq3mNwktPlmfbVg90H21EOLACvtfcHk0liRj-eNJtGVsA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXosVSisw%253D%253D&md5=987db32b246874d5ae418bf45efc3087</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.4155%2Ffmc.09.142&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4155%252Ffmc.09.142%26sid%3Dliteratum%253Aachs%26aulast%3DBrowning%26aufirst%3DD.%2BD.%26aulast%3DKwon%26aufirst%3DI.-K.%26aulast%3DWang%26aufirst%3DR.%26atitle%3DcGMP-dependent%2520protein%2520kinases%2520as%2520potential%2520targets%2520for%2520colon%2520cancer%2520prevention%2520and%2520treatment%26jtitle%3DFuture%2520Med.%2520Chem.%26date%3D2010%26volume%3D2%26spage%3D65%26epage%3D80%26doi%3D10.4155%2Ffmc.09.142" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">da
Rocha, A. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mans, D. R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Regner, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwartsmann, G.</span></span> <span> </span><span class="NLM_article-title">Targeting protein kinase C: new therapeutic opportunities against high-grade malignant gliomas?</span>. <i>Oncologist</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">17</span>– <span class="NLM_lpage">33</span>, <span class="refDoi"> DOI: 10.1634/theoncologist.7-1-17</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=10.1634%2Ftheoncologist.7-1-17" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=11854544" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=1%3ACAS%3A528%3ADC%252BD38XitVCht7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2002&pages=17-33&author=A.+B.+da%0ARochaauthor=D.+R.+A.+Mansauthor=A.+Regnerauthor=G.+Schwartsmann&title=Targeting+protein+kinase+C%3A+new+therapeutic+opportunities+against+high-grade+malignant+gliomas%3F&doi=10.1634%2Ftheoncologist.7-1-17"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting protein kinase C: New therapeutic opportunities against high-grade malignant gliomas?</span></div><div class="casAuthors">da Rocha, A. B.; Mans, D. R. A.; Regner, A.; Schwartsmann, G.</div><div class="citationInfo"><span class="NLM_cas:title">Oncologist</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">17-33</span>CODEN:
                <span class="NLM_cas:coden">OCOLF6</span>;
        ISSN:<span class="NLM_cas:issn">1083-7159</span>.
    
            (<span class="NLM_cas:orgname">AlphaMed Press</span>)
        </div><div class="casAbstract">A review.  A large body of evidence suggests that the abnormal phenotype of neoplastic astrocytes, including their excessive proliferation rate and high propensity to invade surrounding tissues, results from mutations in crit. genes involved in key cellular events.  These genetic alterations can affect cell-surface-assocd. receptors, elements of signaling pathways, or components of the cell cycle clock, conferring a gain or a loss of relevant metabolic functions of the cells.  The understanding of such phenomena may allow the development of more efficacious forms of cancer treatment.  Examples are therapies specifically directed against overexpressed epidermal growth factor receptor, hyperactive Ras, excessively stimulated Raf-1, overproduced ornithine decarboxylase, or aberrantly activated cyclindependent kinases.  The applicability of some of these approaches is now being assessed in patients suffering from primary malignant central nervous system tumors that are not amenable to current therapeutic modalities.  Another potentially useful therapeutic strategy against such tumors involves the inhibition of hyperactive or overexpressed protein kinase C (PKC).  This strategy is justified by the decrease in cell proliferation and invasion following inhibition of the activity of this enzyme obsd. in preclin. glioma models.  Thus, interference with PKC activity may represent a novel form of exptl. cancer treatment that may simultaneously restrain the hyperproliferative state and the invasive capacity of high-grade malignant gliomas without inducing the expected toxicity of classical cytotoxic agents.  Of note, the exptl. use of PKC-inhibiting agents in patients with refractory high-grade malignant gliomas has indeed led to some clin. responses.  The present paper reviews the current status of the biochem. and mol. biol. of PKC, as well as the possibilities for developing novel anti-PKC-based therapies for central nervous system malignancies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGresHYaWuTx5LVg90H21EOLACvtfcHk0liRj-eNJtGVsA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XitVCht7c%253D&md5=3b2b01910cb3bd93631dfa8b08e9d389</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1634%2Ftheoncologist.7-1-17&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1634%252Ftheoncologist.7-1-17%26sid%3Dliteratum%253Aachs%26aulast%3Dda%2BRocha%26aufirst%3DA.%2BB.%26aulast%3DMans%26aufirst%3DD.%2BR.%2BA.%26aulast%3DRegner%26aufirst%3DA.%26aulast%3DSchwartsmann%26aufirst%3DG.%26atitle%3DTargeting%2520protein%2520kinase%2520C%253A%2520new%2520therapeutic%2520opportunities%2520against%2520high-grade%2520malignant%2520gliomas%253F%26jtitle%3DOncologist%26date%3D2002%26volume%3D7%26spage%3D17%26epage%3D33%26doi%3D10.1634%2Ftheoncologist.7-1-17" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kathawala, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anreddy, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alqahtani, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.-K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sodani, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaddoumi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ashby, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z.-S.</span></span> <span> </span><span class="NLM_article-title">The small molecule tyrosine kinase inhibitor NVP-BHG712 antagonizes ABCC10-mediated paclitaxel resistance: a preclinical and pharmacokinetic study</span>. <i>Oncotarget</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">510</span>– <span class="NLM_lpage">521</span>, <span class="refDoi"> DOI: 10.18632/oncotarget.2638</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=10.18632%2Foncotarget.2638" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=25402202" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=1%3ACAS%3A280%3ADC%252BC2M3osFWjtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=510-521&author=R.+J.+Kathawalaauthor=L.+Weiauthor=N.+Anreddyauthor=K.+Chenauthor=A.+Patelauthor=S.+Alqahtaniauthor=Y.-K.+Zhangauthor=Y.-J.+Wangauthor=K.+Sodaniauthor=A.+Kaddoumiauthor=C.+R.+Ashbyauthor=Z.-S.+Chen&title=The+small+molecule+tyrosine+kinase+inhibitor+NVP-BHG712+antagonizes+ABCC10-mediated+paclitaxel+resistance%3A+a+preclinical+and+pharmacokinetic+study&doi=10.18632%2Foncotarget.2638"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">The small molecule tyrosine kinase inhibitor NVP-BHG712 antagonizes ABCC10-mediated paclitaxel resistance: a preclinical and pharmacokinetic study</span></div><div class="casAuthors">Kathawala Rishil J; Wei Liuya; Anreddy Nagaraju; Patel Atish; Zhang Yun-Kai; Wang Yi-Jun; Sodani Kamlesh; Ashby Charles R Jr; Chen Zhe-Sheng; Chen Kang; Alqahtani Saeed; Kaddoumi Amal</div><div class="citationInfo"><span class="NLM_cas:title">Oncotarget</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">510-21</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Paclitaxel exhibits clinical activity against a wide variety of solid tumors.  However, resistance to paclitaxel significantly attenuates the response to chemotherapy.  The ABC transporter subfamily C member 10 (ABCC10), also known as multi-drug resistance protein 7 (MRP7) efflux transporter, is a major mediator of paclitaxel resistance.  Here, we determine the effect of NVP-BHG712, a specific EphB4 receptor inhibitor, on 1) paclitaxel resistance in HEK293 cells transfected with ABCC10, 2) the growth of tumors in athymic nude mice that received NVP-BHG712 and paclitaxel systemically and 3) the pharmacokinetics of paclitaxel in presence or absence of NVP-BHG712.  NVP-BHG712 (0.5 μM), in HEK293/ABCC10 cells, significantly enhanced the intracellular accumulation of paclitaxel by inhibiting the efflux activity of ABCC10 without altering the expression level of the ABCC10 protein.  Furthermore, NVP-BHG712 (25 mg/kg, p.o., q3d x 6), in combination with paclitaxel (15 mg/kg, i.p., q3d x 6), significantly inhibited the growth of ABCC10-expressing tumors in athymic nude mice.  NVP-BHG712 administration significantly increased the levels of paclitaxel in the tumors but not in plasma compared to paclitaxel alone.  The combination of NVP-BHG712 and paclitaxel could serve as a novel and useful therapeutic strategy to attenuate paclitaxel resistance mediated by the expression of the ABCC10 transporter.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTxASDz7eI_LVZoq0fPSOTvfW6udTcc2ebQh9RH3-rFzrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2M3osFWjtQ%253D%253D&md5=75b7e4bb54af1483c51799cf1ab6642e</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.18632%2Foncotarget.2638&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncotarget.2638%26sid%3Dliteratum%253Aachs%26aulast%3DKathawala%26aufirst%3DR.%2BJ.%26aulast%3DWei%26aufirst%3DL.%26aulast%3DAnreddy%26aufirst%3DN.%26aulast%3DChen%26aufirst%3DK.%26aulast%3DPatel%26aufirst%3DA.%26aulast%3DAlqahtani%26aufirst%3DS.%26aulast%3DZhang%26aufirst%3DY.-K.%26aulast%3DWang%26aufirst%3DY.-J.%26aulast%3DSodani%26aufirst%3DK.%26aulast%3DKaddoumi%26aufirst%3DA.%26aulast%3DAshby%26aufirst%3DC.%2BR.%26aulast%3DChen%26aufirst%3DZ.-S.%26atitle%3DThe%2520small%2520molecule%2520tyrosine%2520kinase%2520inhibitor%2520NVP-BHG712%2520antagonizes%2520ABCC10-mediated%2520paclitaxel%2520resistance%253A%2520a%2520preclinical%2520and%2520pharmacokinetic%2520study%26jtitle%3DOncotarget%26date%3D2015%26volume%3D6%26spage%3D510%26epage%3D521%26doi%3D10.18632%2Foncotarget.2638" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Je, D. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O, Y. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ji, Y. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cho, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, D. H.</span></span> <span> </span><span class="NLM_article-title">The inhibition of SRC family kinase suppresses pancreatic cancer cell proliferation, migration, and invasion</span>. <i>Pancreas</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>43</i></span>,  <span class="NLM_fpage">768</span>– <span class="NLM_lpage">776</span>, <span class="refDoi"> DOI: 10.1097/MPA.0000000000000103</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=10.1097%2FMPA.0000000000000103" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=24763074" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=1%3ACAS%3A528%3ADC%252BC2cXps1Cjt74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2014&pages=768-776&author=D.+W.+Jeauthor=Y.+M.+Oauthor=Y.+G.+Jiauthor=Y.+Choauthor=D.+H.+Lee&title=The+inhibition+of+SRC+family+kinase+suppresses+pancreatic+cancer+cell+proliferation%2C+migration%2C+and+invasion&doi=10.1097%2FMPA.0000000000000103"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">The Inhibition of Src Family Kinase Suppresses Pancreatic Cancer Cell Proliferation, Migration, and Invasion</span></div><div class="casAuthors">Je, Dong Wook; O, Young Moon; Ji, Young Geon; Cho, Yunkyung; Lee, Dong Hyeon</div><div class="citationInfo"><span class="NLM_cas:title">Pancreas (Philadelphia, PA, United States)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">43</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">768-776</span>CODEN:
                <span class="NLM_cas:coden">PANCE4</span>;
        ISSN:<span class="NLM_cas:issn">0885-3177</span>.
    
            (<span class="NLM_cas:orgname">Lippincott Williams & Wilkins</span>)
        </div><div class="casAbstract">Objectives: Src is considered a rising therapeutic target for the treatment of solid tumors, and Src family kinases (SFKs) participate in cancer cell proliferation and survival.  The role of SFK suppression was investigated in the proliferation, migration, and invasion of pancreatic cancer cells.  Methods: Knockdown of the SFKs in pancreatic cancer cells was achieved by transfecting small interfering RNAs, and its effects were investigated using proliferation, wound, and invasion assays.  Results: The SFK inhibitors suppressed proliferation and induced cell cycle arrest in pancreatic cancer cells.  The SFK mRNA profiles showed that Yes1, Lyn, Fyn, Frk, Hck, and Src were expressed.  Specific small interfering RNA transfection suppressed the mRNA expressions of Yes1, Lyn, Fyn, Frk, and Src, and the knockdown suppressed cell proliferation by 16.7% to 47.3% in PANC-1 cells.  Knockdown of any of these 5 SFKs suppressed proliferation in other pancreatic cancer cell lines by 3.0% to 40.5%.  The knockdowns significantly reduced pancreatic cancer cell migration by 24.9% to 66.7% and completely inhibited invasion.  Conclusions: These results suggest that the knockdown of Yes1, Lyn, Fyn, Frk, or Src reduce human pancreatic cancer cell proliferation, migration, and invasion, and that SFKs should be viewed as crit. therapeutic targets of pancreatic cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWIIEBAqGDnrVg90H21EOLACvtfcHk0lg7knkI3Nxf_w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXps1Cjt74%253D&md5=27a2339f2ac9f4176cca71e77a315bbe</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1097%2FMPA.0000000000000103&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1097%252FMPA.0000000000000103%26sid%3Dliteratum%253Aachs%26aulast%3DJe%26aufirst%3DD.%2BW.%26aulast%3DO%26aufirst%3DY.%2BM.%26aulast%3DJi%26aufirst%3DY.%2BG.%26aulast%3DCho%26aufirst%3DY.%26aulast%3DLee%26aufirst%3DD.%2BH.%26atitle%3DThe%2520inhibition%2520of%2520SRC%2520family%2520kinase%2520suppresses%2520pancreatic%2520cancer%2520cell%2520proliferation%252C%2520migration%252C%2520and%2520invasion%26jtitle%3DPancreas%26date%3D2014%26volume%3D43%26spage%3D768%26epage%3D776%26doi%3D10.1097%2FMPA.0000000000000103" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kalyankrishna, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wislez, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thilaganathan, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saigal, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wistuba, I. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, F. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kurie, J. M.</span></span> <span> </span><span class="NLM_article-title">SRC-family kinases are activated in non-small cell lung cancer and promote the survival of epidermal growth factor receptor-dependent cell lines</span>. <i>Am. J. Pathol.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>170</i></span>,  <span class="NLM_fpage">366</span>– <span class="NLM_lpage">376</span>, <span class="refDoi"> DOI: 10.2353/ajpath.2007.060706</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=10.2353%2Fajpath.2007.060706" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=17200208" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=1%3ACAS%3A280%3ADC%252BD2s%252FgtlOnsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=170&publication_year=2007&pages=366-376&author=J.+Zhangauthor=S.+Kalyankrishnaauthor=M.+Wislezauthor=N.+Thilaganathanauthor=B.+Saigalauthor=W.+Weiauthor=L.+Maauthor=I.+I.+Wistubaauthor=F.+M.+Johnsonauthor=J.+M.+Kurie&title=SRC-family+kinases+are+activated+in+non-small+cell+lung+cancer+and+promote+the+survival+of+epidermal+growth+factor+receptor-dependent+cell+lines&doi=10.2353%2Fajpath.2007.060706"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">SRC-family kinases are activated in non-small cell lung cancer and promote the survival of epidermal growth factor receptor-dependent cell lines</span></div><div class="casAuthors">Zhang Jie; Kalyankrishna Shailaja; Wislez Marie; Thilaganathan Nishan; Saigal Babita; Wei Wei; Ma Long; Wistuba Ignacio I; Johnson Faye M; Kurie Jonathan M</div><div class="citationInfo"><span class="NLM_cas:title">The American journal of pathology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">170</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">366-76</span>
        ISSN:<span class="NLM_cas:issn">0002-9440</span>.
    </div><div class="casAbstract">The role of Src-family kinases (SFKs) in non-small cell lung cancer (NSCLC) has not been fully defined.  Here we addressed this question by examining SFK phosphorylation in NSCLC biopsy samples and using genetic and pharmacological approaches to inhibit SFK expression and activity in cultured NSCLC cells.  Immunohistochemical analysis of NSCLC biopsy samples using a Tyr416 phosphorylation-specific, pan-SFK antibody revealed staining in 123 (33%) of 370 tumors.  Because c-Src is known to be both an upstream activator and downstream mediator of epidermal growth factor receptor (EGFR), we next investigated SFK phosphorylation in a panel of NSCLC cell lines, including ones that depend on EGFR for survival.  The EGFR-dependent NSCLC cell lines HCC827 and H3255 had increased phosphorylation of SFKs, and treatment of these cells with an SFK inhibitor (PP1 or SKI-606) induced apoptosis.  PP1 decreased phosphorylation of EGFR, ErbB2, and ErbB3 and strikingly enhanced apoptosis by gefitinib, an EGFR inhibitor.  HCC827 cells transfected with c-Src short hairpin RNA exhibited diminished phosphorylation of EGFR and ErbB2 and decreased sensitivity to apoptosis by PP1 or gefitinib.  We conclude that SFKs are activated in NSCLC biopsy samples, promote the survival of EGFR-dependent NSCLC cells, and should be investigated as therapeutic targets in NSCLC patients.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQZ2f3qQMjhE0kUaC_kzhvyfW6udTcc2ea_z6YDHWaF-rntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2s%252FgtlOnsA%253D%253D&md5=391c770e7368377b4b9f5576a206d8d0</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.2353%2Fajpath.2007.060706&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2353%252Fajpath.2007.060706%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DKalyankrishna%26aufirst%3DS.%26aulast%3DWislez%26aufirst%3DM.%26aulast%3DThilaganathan%26aufirst%3DN.%26aulast%3DSaigal%26aufirst%3DB.%26aulast%3DWei%26aufirst%3DW.%26aulast%3DMa%26aufirst%3DL.%26aulast%3DWistuba%26aufirst%3DI.%2BI.%26aulast%3DJohnson%26aufirst%3DF.%2BM.%26aulast%3DKurie%26aufirst%3DJ.%2BM.%26atitle%3DSRC-family%2520kinases%2520are%2520activated%2520in%2520non-small%2520cell%2520lung%2520cancer%2520and%2520promote%2520the%2520survival%2520of%2520epidermal%2520growth%2520factor%2520receptor-dependent%2520cell%2520lines%26jtitle%3DAm.%2520J.%2520Pathol.%26date%3D2007%26volume%3D170%26spage%3D366%26epage%3D376%26doi%3D10.2353%2Fajpath.2007.060706" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zahari, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Renuse, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kelkar, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bharbuiya, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rojas, P. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stearns, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gabrielson, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Malla, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sukumar, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mahajan, N. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pandey, A.</span></span> <span> </span><span class="NLM_article-title">The non-receptor tyrosine kinase TNK2/ACK1 is a novel therapeutic target in triple negative breast cancer</span>. <i>Oncotarget</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">2971</span>– <span class="NLM_lpage">2983</span>, <span class="refDoi"> DOI: 10.18632/oncotarget.13579</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=10.18632%2Foncotarget.13579" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=27902967" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=1%3ACAS%3A280%3ADC%252BC2sjitVShsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=2971-2983&author=X.+Wuauthor=M.+S.+Zahariauthor=S.+Renuseauthor=D.+S.+Kelkarauthor=M.+A.+Bharbuiyaauthor=P.+L.+Rojasauthor=V.+Stearnsauthor=E.+Gabrielsonauthor=P.+Mallaauthor=S.+Sukumarauthor=N.+P.+Mahajanauthor=A.+Pandey&title=The+non-receptor+tyrosine+kinase+TNK2%2FACK1+is+a+novel+therapeutic+target+in+triple+negative+breast+cancer&doi=10.18632%2Foncotarget.13579"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">The non-receptor tyrosine kinase TNK2/ACK1 is a novel therapeutic target in triple negative breast cancer</span></div><div class="casAuthors">Wu Xinyan; Zahari Muhammad Saddiq; Renuse Santosh; Kelkar Dhanashree S; Rojas Pamela L; Pandey Akhilesh; Wu Xinyan; Zahari Muhammad Saddiq; Renuse Santosh; Kelkar Dhanashree S; Barbhuiya Mustafa A; Rojas Pamela L; Pandey Akhilesh; Renuse Santosh; Stearns Vered; Gabrielson Edward; Sukumar Saraswati; Pandey Akhilesh; Gabrielson Edward; Pandey Akhilesh; Malla Pavani; Mahajan Nupam P; Mahajan Nupam P</div><div class="citationInfo"><span class="NLM_cas:title">Oncotarget</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">2971-2983</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Breast cancer is the most prevalent cancer in women worldwide.  About 15-20% of all breast cancers do not express estrogen receptor, progesterone receptor or HER2 receptor and hence are collectively classified as triple negative breast cancer (TNBC).  These tumors are often relatively aggressive when compared to other types of breast cancer, and this issue is compounded by the lack of effective targeted therapy.  In our previous phosphoproteomic profiling effort, we identified the non-receptor tyrosine kinase TNK2 as activated in a majority of aggressive TNBC cell lines.  In the current study, we show that high expression of TNK2 in breast cancer cell lines correlates with high proliferation, invasion and colony forming ability.  We demonstrate that knockdown of TNK2 expression can substantially suppress the invasiveness and proliferation advantage of TNBC cells in vitro and tumor formation in xenograft mouse models.  Moreover, inhibition of TNK2 with small molecule inhibitor (R)-9bMS significantly compromised TNBC proliferation.Finally, we find that high levels of TNK2 expression in high-grade basal-like breast cancers correlates significantly with poorer patient outcome.  Taken together, our study suggests that TNK2 is a novel potential therapeutic target for the treatment of TNBC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQ8VkHWAOdcJDAyhZbj3lKdfW6udTcc2eYXjOgUWEGz4bntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2sjitVShsg%253D%253D&md5=71f888ec0dd35a99a9b5a779ab56f20a</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.18632%2Foncotarget.13579&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncotarget.13579%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DX.%26aulast%3DZahari%26aufirst%3DM.%2BS.%26aulast%3DRenuse%26aufirst%3DS.%26aulast%3DKelkar%26aufirst%3DD.%2BS.%26aulast%3DBharbuiya%26aufirst%3DM.%2BA.%26aulast%3DRojas%26aufirst%3DP.%2BL.%26aulast%3DStearns%26aufirst%3DV.%26aulast%3DGabrielson%26aufirst%3DE.%26aulast%3DMalla%26aufirst%3DP.%26aulast%3DSukumar%26aufirst%3DS.%26aulast%3DMahajan%26aufirst%3DN.%2BP.%26aulast%3DPandey%26aufirst%3DA.%26atitle%3DThe%2520non-receptor%2520tyrosine%2520kinase%2520TNK2%252FACK1%2520is%2520a%2520novel%2520therapeutic%2520target%2520in%2520triple%2520negative%2520breast%2520cancer%26jtitle%3DOncotarget%26date%3D2017%26volume%3D8%26spage%3D2971%26epage%3D2983%26doi%3D10.18632%2Foncotarget.13579" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Isobe, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mizuno, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaneko, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohta, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koide, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanabe, S.</span></span> <span> </span><span class="NLM_article-title">Effects of K-115, a Rho-kinase inhibitor, on aqueous humor dynamics in rabbits</span>. <i>Curr. Eye Res.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>39</i></span>,  <span class="NLM_fpage">813</span>– <span class="NLM_lpage">822</span>, <span class="refDoi"> DOI: 10.3109/02713683.2013.874444</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=10.3109%2F02713683.2013.874444" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=24502505" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtFylu7fP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=2014&pages=813-822&author=T.+Isobeauthor=K.+Mizunoauthor=Y.+Kanekoauthor=M.+Ohtaauthor=T.+Koideauthor=S.+Tanabe&title=Effects+of+K-115%2C+a+Rho-kinase+inhibitor%2C+on+aqueous+humor+dynamics+in+rabbits&doi=10.3109%2F02713683.2013.874444"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Effects of K-115, a Rho-Kinase Inhibitor, on Aqueous Humor Dynamics in Rabbits</span></div><div class="casAuthors">Isobe, Tomoyuki; Mizuno, Ken; Kaneko, Yoshio; Ohta, Masayuki; Koide, Takashi; Tanabe, Sohei</div><div class="citationInfo"><span class="NLM_cas:title">Current Eye Research</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">39</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">813-822</span>CODEN:
                <span class="NLM_cas:coden">CEYRDM</span>;
        ISSN:<span class="NLM_cas:issn">0271-3683</span>.
    
            (<span class="NLM_cas:orgname">Informa Healthcare</span>)
        </div><div class="casAbstract">Purpose: To evaluate the topical instillation of K-115, a selective Rho-assocd. coiled coil-contg. protein kinase (ROCK) inhibitor, on intraocular pressure (IOP), ocular distribution, and aq. humor dynamics in exptl. animals.  Methods: Kinase inhibition by K-115 was measured by biochem. assay.  IOP was monitored using a pneumatonometer in albino rabbits and monkeys after topical instillation of K-115.  The ocular distribution of [14C]K-115 was detd. by whole-head autoradiog.  The aq. flow rate was detd. by fluorophotometry.  The total outflow facility and uveoscleral outflow were measured by two-level const. pressure perfusion and perfusion technique using fluorescein isothiocyanate-dextran, resp.  Results: Biochem. assay showed that K-115 had selective and potent inhibitory effects on ROCKs.  In rabbits, topical instillation of K-115 significantly reduced IOP in a dose-dependent manner.  Maximum IOP redn. was obsd. 1 h after topical instillation, which was 8.55 ± 1.09 mmHg (mean ± SE) from the baseline IOP at 0.5%.  In monkeys, max. IOP redn. was obsd. 2 h after topical instillation, which was 4.36 ± 0.32 mmHg from the baseline IOP at 0.4%, and was significantly stronger than that of 0.005% latanoprost.  Whole-head autoradiog. showed that the radioactivity level was max. at 15 min after instillation of [14C]K-115 in the ipsilateral eye.  Single instillation of 0.4% K-115 showed no effect on aq. flow rate or uveoscleral outflow, but significantly increased conventional outflow facility by 2.2-fold compared to vehicle-treated eyes in rabbits.  Conclusions: These results indicated that K-115 ophthalmic soln., a selective and potent ROCK inhibitor, is a novel and potent antiglaucoma agent.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpC62-fukbe1rVg90H21EOLACvtfcHk0lhzcLfGvwt7gw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtFylu7fP&md5=1d0048c285f37c03f6f1c0f3bd26ec13</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.3109%2F02713683.2013.874444&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3109%252F02713683.2013.874444%26sid%3Dliteratum%253Aachs%26aulast%3DIsobe%26aufirst%3DT.%26aulast%3DMizuno%26aufirst%3DK.%26aulast%3DKaneko%26aufirst%3DY.%26aulast%3DOhta%26aufirst%3DM.%26aulast%3DKoide%26aufirst%3DT.%26aulast%3DTanabe%26aufirst%3DS.%26atitle%3DEffects%2520of%2520K-115%252C%2520a%2520Rho-kinase%2520inhibitor%252C%2520on%2520aqueous%2520humor%2520dynamics%2520in%2520rabbits%26jtitle%3DCurr.%2520Eye%2520Res.%26date%3D2014%26volume%3D39%26spage%3D813%26epage%3D822%26doi%3D10.3109%2F02713683.2013.874444" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yamamoto, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maruyama, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Himori, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Omodaka, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yokoyama, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shiga, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morin, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakazawa, T.</span></span> <span> </span><span class="NLM_article-title">The novel Rho kinase (ROCK) inhibitor K-115: a new candidate drug for neuroprotective treatment in glaucoma</span>. <i>Invest. Ophthalmol. Visual Sci.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">7126</span>– <span class="NLM_lpage">7136</span>, <span class="refDoi"> DOI: 10.1167/iovs.13-13842</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=10.1167%2Fiovs.13-13842" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=25277230" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=1%3ACAS%3A528%3ADC%252BC2MXitVOltbfN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2014&pages=7126-7136&author=K.+Yamamotoauthor=K.+Maruyamaauthor=N.+Himoriauthor=K.+Omodakaauthor=Y.+Yokoyamaauthor=Y.+Shigaauthor=R.+Morinauthor=T.+Nakazawa&title=The+novel+Rho+kinase+%28ROCK%29+inhibitor+K-115%3A+a+new+candidate+drug+for+neuroprotective+treatment+in+glaucoma&doi=10.1167%2Fiovs.13-13842"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">The novel Rho kinase (ROCK) inhibitor K-115: a new candidate drug for neuroprotective treatment in glaucoma</span></div><div class="casAuthors">Yamamoto, Kotaro; Maruyama, Kazuichi; Himori, Noriko; Omodaka, Kazuko; Yokoyama, Yu; Shiga, Yukihiro; Morin, Ryu; Nakazawa, Toru</div><div class="citationInfo"><span class="NLM_cas:title">Investigative Ophthalmology & Visual Science</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">7126-7136</span>CODEN:
                <span class="NLM_cas:coden">IOVSDA</span>;
        ISSN:<span class="NLM_cas:issn">1552-5783</span>.
    
            (<span class="NLM_cas:orgname">Association for Research in Vision and Ophthalmology</span>)
        </div><div class="casAbstract">To investigate the effect of K-115, a novel Rho kinase (ROCK) inhibitor, on retinal ganglion cell (RGC) survival in an optic nerve crush (NC) model.  Addnl., to det. the details of the mechanism of K-115's neuroprotective effect in vivo and in vitro.  ROCK inhibitors, including K-115 and fasudil (1 mg/kg/d), or vehicle were administered orally to C57BL/6 mice.  Retinal ganglion cell death was then induced with NC.  Retinal ganglion cell survival was evaluated by counting surviving retrogradely labeled cells and measuring RGC marker expression with quant. real-time polymerase chain reaction (qRT-PCR).  Total oxidized lipid levels were assessed with a thiobarbituric acid-reactive substances (TBARS) assay.  Reactive oxygen species (ROS) levels were assessed by co-labeling with CellROX and Fluorogold.  Expression of the NADPH oxidase (Nox) family of genes was evaluated with qRT-PCR.  The survival of RGCs after NC was increased 34 ± 3% with K-115, a significantly protective effect.  Moreover, a similar effect was revealed by the qRT-PCR anal. of Thy-1.2 and Brn3a, RGC markers.  Levels of oxidized lipids and ROS also increased with time after NC.  NC-induced oxidative stress, including oxidn. of lipids and prodn. of ROS, was significantly attenuated by K-115.  Furthermore, expression of the Nox gene family, esp. Nox1, which is involved in the NC-induced ROS prodn. pathway, was dramatically reduced by K-115.  The results indicated that oral K-115 administration delayed RGC death.  Although K-115 may be mediated through Nox1 downregulation, we found that it did not suppress ROS prodn. directly.  Our findings show that K-115 has a potential use in neuroprotective treatment for glaucoma and other neurodegenerative diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq7y2j7EK2KoLVg90H21EOLACvtfcHk0limEqOkUQo64A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXitVOltbfN&md5=f0db38d732b7244c73b3aeca80cf193a</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1167%2Fiovs.13-13842&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1167%252Fiovs.13-13842%26sid%3Dliteratum%253Aachs%26aulast%3DYamamoto%26aufirst%3DK.%26aulast%3DMaruyama%26aufirst%3DK.%26aulast%3DHimori%26aufirst%3DN.%26aulast%3DOmodaka%26aufirst%3DK.%26aulast%3DYokoyama%26aufirst%3DY.%26aulast%3DShiga%26aufirst%3DY.%26aulast%3DMorin%26aufirst%3DR.%26aulast%3DNakazawa%26aufirst%3DT.%26atitle%3DThe%2520novel%2520Rho%2520kinase%2520%2528ROCK%2529%2520inhibitor%2520K-115%253A%2520a%2520new%2520candidate%2520drug%2520for%2520neuroprotective%2520treatment%2520in%2520glaucoma%26jtitle%3DInvest.%2520Ophthalmol.%2520Visual%2520Sci.%26date%3D2014%26volume%3D55%26spage%3D7126%26epage%3D7136%26doi%3D10.1167%2Fiovs.13-13842" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Martiny-Baron, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holzer, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Billy, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schnell, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brueggen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferretti, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmiedeberg, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wood, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Furet, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Imbach, P.</span></span> <span> </span><span class="NLM_article-title">The small molecule specific EphB4 kinase inhibitor NVP-BHG712 inhibits VEGF driven angiogenesis</span>. <i>Angiogenesis</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">259</span>– <span class="NLM_lpage">267</span>, <span class="refDoi"> DOI: 10.1007/s10456-010-9183-z</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=10.1007%2Fs10456-010-9183-z" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=20803239" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtFyis7fN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2010&pages=259-267&author=G.+Martiny-Baronauthor=P.+Holzerauthor=E.+Billyauthor=C.+Schnellauthor=J.+Brueggenauthor=M.+Ferrettiauthor=N.+Schmiedebergauthor=J.+M.+Woodauthor=P.+Furetauthor=P.+Imbach&title=The+small+molecule+specific+EphB4+kinase+inhibitor+NVP-BHG712+inhibits+VEGF+driven+angiogenesis&doi=10.1007%2Fs10456-010-9183-z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">The small molecule specific EphB4 kinase inhibitor NVP-BHG712 inhibits VEGF driven angiogenesis</span></div><div class="casAuthors">Martiny-Baron, Georg; Holzer, Philipp; Billy, Eric; Schnell, Christian; Brueggen, Joseph; Ferretti, Mireille; Schmiedeberg, Niko; Wood, Jeanette M.; Furet, Pascal; Imbach, Patricia</div><div class="citationInfo"><span class="NLM_cas:title">Angiogenesis</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">259-267</span>CODEN:
                <span class="NLM_cas:coden">AGIOFT</span>;
        ISSN:<span class="NLM_cas:issn">0969-6970</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">EphB4 and its cognitive ligand ephrinB2 play an important role in embryonic vessel development and vascular remodeling.  In addn., several reports suggest that this receptor ligand pair is also involved in pathol. vessel formation in adults including tumor angiogenesis.  Eph/ephrin signaling is a complex phenomena characterized by receptor forward signaling through the tyrosine kinase of the receptor and ephrin reverse signaling through various protein-protein interaction domains and phosphorylation motifs of the ephrin ligands.  Therefore, interfering with EphR/ephrin signaling by the means of targeted gene ablation, sol. receptors, dominant neg. mutants or antisense mols. often does not allow to discriminate between inhibition of Eph/ephrin forward and reverse signaling.  We developed a specific small mol. wt. kinase inhibitor of the EphB4 kinase, NVP-BHG712, which inhibits EphB4 kinase activity in the low nanomolar range in cellular assays showed high selectivity for targeting the EphB4 kinase when profiled against other kinases in biochem. as well as in cell based assays.  Furthermore, NVP-BHG712 shows excellent pharmacokinetic properties and potently inhibits EphB4 autophosphorylation in tissues after oral administration.  In vivo, NVP-BHG712 inhibits VEGF driven vessel formation, while it has only little effects on VEGF receptor (VEGFR) activity in vitro or in cellular assays.  The data shown here suggest a close cross talk between the VEGFR and EphR signaling during vessel formation.  In addn. to its established function in vascular remodeling and endothelial arterio-venous differentiation, EphB4 forward signaling appears to be an important mediator of VEGF induced angiogenesis since inhibition of EphB4 forward signaling is sufficient to inhibit VEGF induced angiogenesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrWzqlHCnHYMLVg90H21EOLACvtfcHk0limEqOkUQo64A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtFyis7fN&md5=a7c8df9e330fc86001b36f3495ac6037</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1007%2Fs10456-010-9183-z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10456-010-9183-z%26sid%3Dliteratum%253Aachs%26aulast%3DMartiny-Baron%26aufirst%3DG.%26aulast%3DHolzer%26aufirst%3DP.%26aulast%3DBilly%26aufirst%3DE.%26aulast%3DSchnell%26aufirst%3DC.%26aulast%3DBrueggen%26aufirst%3DJ.%26aulast%3DFerretti%26aufirst%3DM.%26aulast%3DSchmiedeberg%26aufirst%3DN.%26aulast%3DWood%26aufirst%3DJ.%2BM.%26aulast%3DFuret%26aufirst%3DP.%26aulast%3DImbach%26aufirst%3DP.%26atitle%3DThe%2520small%2520molecule%2520specific%2520EphB4%2520kinase%2520inhibitor%2520NVP-BHG712%2520inhibits%2520VEGF%2520driven%2520angiogenesis%26jtitle%3DAngiogenesis%26date%3D2010%26volume%3D13%26spage%3D259%26epage%3D267%26doi%3D10.1007%2Fs10456-010-9183-z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref54"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref54'); return false;" data-citation="" class="refNumLink">54</a></strong><div class="NLM_citation" id="cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Karaman, M. W</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herrgard, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Treiber, D. K</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gallant, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Atteridge, C. E</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campbell, B. T</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, K. W</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciceri, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, M. I</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edeen, P. T</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Faraoni, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Floyd, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hunt, J. P</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lockhart, D. J</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Milanov, Z. V</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morrison, M. J</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pallares, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, H. K</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pritchard, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wodicka, L. M</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zarrinkar, P. P</span></span> <span> </span><span class="NLM_article-title">A quantitative analysis of kinase inhibitor selectivity</span>. <i>Nat. Biotechnol.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">127</span>– <span class="NLM_lpage">132</span>, <span class="refDoi"> DOI: 10.1038/nbt1358</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=10.1038%2Fnbt1358" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=18183025" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=1%3ACAS%3A528%3ADC%252BD1cXisFGlsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2008&pages=127-132&author=M.+W+Karamanauthor=S.+Herrgardauthor=D.+K+Treiberauthor=P.+Gallantauthor=C.+E+Atteridgeauthor=B.+T+Campbellauthor=K.+W+Chanauthor=P.+Ciceriauthor=M.+I+Davisauthor=P.+T+Edeenauthor=R.+Faraoniauthor=M.+Floydauthor=J.+P+Huntauthor=D.+J+Lockhartauthor=Z.+V+Milanovauthor=M.+J+Morrisonauthor=G.+Pallaresauthor=H.+K+Patelauthor=S.+Pritchardauthor=L.+M+Wodickaauthor=P.+P+Zarrinkar&title=A+quantitative+analysis+of+kinase+inhibitor+selectivity&doi=10.1038%2Fnbt1358"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">A quantitative analysis of kinase inhibitor selectivity</span></div><div class="casAuthors">Karaman, Mazen W.; Herrgard, Sanna; Treiber, Daniel K.; Gallant, Paul; Atteridge, Corey E.; Campbell, Brian T.; Chan, Katrina W.; Ciceri, Pietro; Davis, Mindy I.; Edeen, Philip T.; Faraoni, Raffaella; Floyd, Mark; Hunt, Jeremy P.; Lockhart, Daniel J.; Milanov, Zdravko V.; Morrison, Michael J.; Pallares, Gabriel; Patel, Hitesh K.; Pritchard, Stephanie; Wodicka, Lisa M.; Zarrinkar, Patrick P.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">127-132</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Kinase inhibitors are a new class of therapeutics with a propensity to inhibit multiple targets.  The biol. consequences of multikinase activity are poorly defined, and an important step toward understanding the relationship between selectivity, efficacy and safety is the exploration of how inhibitors interact with the human kinome.  The authors present interaction maps for 38 kinase inhibitors across a panel of 317 kinases representing >50% of the predicted human protein kinome.  The data constitute the most comprehensive study of kinase inhibitor selectivity to date and reveal a wide diversity of interaction patterns.  To enable a global anal. of the results, the authors introduce the concept of a selectivity score as a general tool to quantify and differentiate the obsd. interaction patterns.  The authors further investigate the impact of panel size and find that small assay panels do not provide a robust measure of selectivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpiArJSq2_lPrVg90H21EOLACvtfcHk0lgqEqc3Rjrm2w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXisFGlsQ%253D%253D&md5=346265d412853ced636ad4128ed8a76f</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1038%2Fnbt1358&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt1358%26sid%3Dliteratum%253Aachs%26aulast%3DKaraman%26aufirst%3DM.%2BW%26aulast%3DHerrgard%26aufirst%3DS.%26aulast%3DTreiber%26aufirst%3DD.%2BK%26aulast%3DGallant%26aufirst%3DP.%26aulast%3DAtteridge%26aufirst%3DC.%2BE%26aulast%3DCampbell%26aufirst%3DB.%2BT%26aulast%3DChan%26aufirst%3DK.%2BW%26aulast%3DCiceri%26aufirst%3DP.%26aulast%3DDavis%26aufirst%3DM.%2BI%26aulast%3DEdeen%26aufirst%3DP.%2BT%26aulast%3DFaraoni%26aufirst%3DR.%26aulast%3DFloyd%26aufirst%3DM.%26aulast%3DHunt%26aufirst%3DJ.%2BP%26aulast%3DLockhart%26aufirst%3DD.%2BJ%26aulast%3DMilanov%26aufirst%3DZ.%2BV%26aulast%3DMorrison%26aufirst%3DM.%2BJ%26aulast%3DPallares%26aufirst%3DG.%26aulast%3DPatel%26aufirst%3DH.%2BK%26aulast%3DPritchard%26aufirst%3DS.%26aulast%3DWodicka%26aufirst%3DL.%2BM%26aulast%3DZarrinkar%26aufirst%3DP.%2BP%26atitle%3DA%2520quantitative%2520analysis%2520of%2520kinase%2520inhibitor%2520selectivity%26jtitle%3DNat.%2520Biotechnol.%26date%3D2008%26volume%3D26%26spage%3D127%26epage%3D132%26doi%3D10.1038%2Fnbt1358" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref55"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref55'); return false;" data-citation="" class="refNumLink">55</a></strong><div class="NLM_citation" id="cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bland, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Altman, D. G.</span></span> <span> </span><span class="NLM_article-title">The odds ratio</span>. <i>BMJ.</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>320</i></span>,  <span class="NLM_fpage">1468</span>– <span class="NLM_lpage">1468</span>, <span class="refDoi"> DOI: 10.1136/bmj.320.7247.1468</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=10.1136%2Fbmj.320.7247.1468" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=10827061" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=1%3ACAS%3A280%3ADC%252BD3c3os1Cluw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=320&publication_year=2000&pages=1468-1468&author=J.+M.+Blandauthor=D.+G.+Altman&title=The+odds+ratio&doi=10.1136%2Fbmj.320.7247.1468"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Statistics notes. The odds ratio</span></div><div class="casAuthors">Bland J M; Altman D G</div><div class="citationInfo"><span class="NLM_cas:title">BMJ (Clinical research ed.)</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">320</span>
        (<span class="NLM_cas:issue">7247</span>),
    <span class="NLM_cas:pages">1468</span>
        ISSN:<span class="NLM_cas:issn">0959-8138</span>.
    </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSP_AzSFo_UuEGy1nwu7K2mfW6udTcc2eYpB3r3u3mnkbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD3c3os1Cluw%253D%253D&md5=48735af9458f863287a13b458bc84b79</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1136%2Fbmj.320.7247.1468&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1136%252Fbmj.320.7247.1468%26sid%3Dliteratum%253Aachs%26aulast%3DBland%26aufirst%3DJ.%2BM.%26aulast%3DAltman%26aufirst%3DD.%2BG.%26atitle%3DThe%2520odds%2520ratio%26jtitle%3DBMJ.%26date%3D2000%26volume%3D320%26spage%3D1468%26epage%3D1468%26doi%3D10.1136%2Fbmj.320.7247.1468" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref56"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref56'); return false;" data-citation="" class="refNumLink">56</a></strong><div class="NLM_citation" id="cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shtivelman, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hensing, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simon, G. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dennis, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Otterson, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bueno, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salgia, R.</span></span> <span> </span><span class="NLM_article-title">Molecular pathways and therapeutic targets in lung cancer</span>. <i>Oncotarget</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">1392</span>– <span class="NLM_lpage">1433</span>, <span class="refDoi"> DOI: 10.18632/oncotarget.1891</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=10.18632%2Foncotarget.1891" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=24722523" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=1%3ACAS%3A280%3ADC%252BC2cnisFyhtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&pages=1392-1433&author=E.+Shtivelmanauthor=T.+Hensingauthor=G.+R.+Simonauthor=P.+A.+Dennisauthor=G.+A.+Ottersonauthor=R.+Buenoauthor=R.+Salgia&title=Molecular+pathways+and+therapeutic+targets+in+lung+cancer&doi=10.18632%2Foncotarget.1891"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular pathways and therapeutic targets in lung cancer</span></div><div class="casAuthors">Shtivelman Emma; Hensing Thomas; Simon George R; Dennis Phillip A; Otterson Gregory A; Bueno Raphael; Salgia Ravi</div><div class="citationInfo"><span class="NLM_cas:title">Oncotarget</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1392-433</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Lung cancer is still the leading cause of cancer death worldwide.  Both histologically and molecularly lung cancer is heterogeneous.  This review summarizes the current knowledge of the pathways involved in the various types of lung cancer with an emphasis on the clinical implications of the increasing number of actionable molecular targets.  It describes the major pathways and molecular alterations implicated in the development and progression of non-small cell lung cancer (adenocarcinoma and squamous cancer), and of small cell carcinoma, emphasizing the molecular alterations comprising the specific blueprints in each group.  The approved and investigational targeted therapies as well as the immune therapies, and clinical trials exploring the variety of targeted approaches to treatment of lung cancer are the main focus of this review.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTTeNjg0C8I6m52AMpI4GK2fW6udTcc2eYpB3r3u3mnkbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2cnisFyhtQ%253D%253D&md5=9f0bff44699caa27f8c80f3c7d26d4c1</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.18632%2Foncotarget.1891&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncotarget.1891%26sid%3Dliteratum%253Aachs%26aulast%3DShtivelman%26aufirst%3DE.%26aulast%3DHensing%26aufirst%3DT.%26aulast%3DSimon%26aufirst%3DG.%2BR.%26aulast%3DDennis%26aufirst%3DP.%2BA.%26aulast%3DOtterson%26aufirst%3DG.%2BA.%26aulast%3DBueno%26aufirst%3DR.%26aulast%3DSalgia%26aufirst%3DR.%26atitle%3DMolecular%2520pathways%2520and%2520therapeutic%2520targets%2520in%2520lung%2520cancer%26jtitle%3DOncotarget%26date%3D2014%26volume%3D5%26spage%3D1392%26epage%3D1433%26doi%3D10.18632%2Foncotarget.1891" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref57"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref57'); return false;" data-citation="" class="refNumLink">57</a></strong><div class="NLM_citation" id="cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Janku, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garrido-Laguna, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petruzelka, L. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stewart, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kurzrock, R.</span></span> <span> </span><span class="NLM_article-title">Novel therapeutic targets in non-small cell lung cancer</span>. <i>J. Thorac. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">1601</span>– <span class="NLM_lpage">1612</span>, <span class="refDoi"> DOI: 10.1097/JTO.0b013e31822944b3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=10.1097%2FJTO.0b013e31822944b3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=21849857" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=1%3ACAS%3A280%3ADC%252BC3MjntFKmuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2011&pages=1601-1612&author=F.+Jankuauthor=I.+Garrido-Lagunaauthor=L.+B.+Petruzelkaauthor=D.+J.+Stewartauthor=R.+Kurzrock&title=Novel+therapeutic+targets+in+non-small+cell+lung+cancer&doi=10.1097%2FJTO.0b013e31822944b3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">Novel therapeutic targets in non-small cell lung cancer</span></div><div class="casAuthors">Janku Filip; Garrido-Laguna Ignacio; Petruzelka Lubos B; Stewart David J; Kurzrock Razelle</div><div class="citationInfo"><span class="NLM_cas:title">Journal of thoracic oncology : official publication of the International Association for the Study of Lung Cancer</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1601-12</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The development of personalized medicine with a focus on novel targeted therapies has supplanted the one-size-fits-all approach to the treatment of many cancers, including non-small cell lung cancer.  Targeted therapies, if given to a patient subpopulation enriched by the presence of relevant molecular targets, can often abrogate cell signaling that perpetuates cancer progression.  Critical targets activating procancer pathways include, but are not limited to, epidermal growth factor receptor (EGFR), hepatocyte growth factor receptor (MET), vascular endothelial growth factor (VEGF), VEGF receptor, GTPase KRAS (KRAS), receptor tyrosine protein kinase erbB-2 (HER2), echinoderm microtubule-associated protein-like 4-anaplastic lymphoma kinase (EML4-ALK), phosphatidylinositol-4,5-bisphosphate 3-kinase catalytic subunit alpha isoform (PIK3CA), serine/threonine-protein kinase B-raf (BRAF), and insulin-like growth factor 1 receptor (IGF-1R).  Some target-directed therapies, such as epidermal growth factor receptor tyrosine kinase inhibitors and anti-VEGF monoclonal antibody, have already been approved for clinical use.  Others, such as those targeted to MET, VEGFR, HER2, PIK3CA, and IGF-1R, are in clinical testing.  This review describes molecular targets in non-small cell lung cancer that are in development or being clinically applied and their implications for developing novel anticancer therapies for this previously refractory malignancy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcT-fAdn102Pmvh3YQQtAhNufW6udTcc2eZrvlMrmASBqLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3MjntFKmuw%253D%253D&md5=17357a9eb28282926945e3b7fc93b86f</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1097%2FJTO.0b013e31822944b3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1097%252FJTO.0b013e31822944b3%26sid%3Dliteratum%253Aachs%26aulast%3DJanku%26aufirst%3DF.%26aulast%3DGarrido-Laguna%26aufirst%3DI.%26aulast%3DPetruzelka%26aufirst%3DL.%2BB.%26aulast%3DStewart%26aufirst%3DD.%2BJ.%26aulast%3DKurzrock%26aufirst%3DR.%26atitle%3DNovel%2520therapeutic%2520targets%2520in%2520non-small%2520cell%2520lung%2520cancer%26jtitle%3DJ.%2520Thorac.%2520Oncol.%26date%3D2011%26volume%3D6%26spage%3D1601%26epage%3D1612%26doi%3D10.1097%2FJTO.0b013e31822944b3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref58"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref58'); return false;" data-citation="" class="refNumLink">58</a></strong><div class="NLM_citation" id="cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Reungwetwattana, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dy, G. K.</span></span> <span> </span><span class="NLM_article-title">Targeted therapies in development for non-small cell lung cancer</span>. <i>J. Carcinog.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">22</span>, <span class="refDoi"> DOI: 10.4103/1477-3163.123972</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=10.4103%2F1477-3163.123972" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=24574860" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=1%3ACAS%3A280%3ADC%252BC2cfls1Gmug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2013&pages=22&author=T.+Reungwetwattanaauthor=G.+K.+Dy&title=Targeted+therapies+in+development+for+non-small+cell+lung+cancer&doi=10.4103%2F1477-3163.123972"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">Targeted therapies in development for non-small cell lung cancer</span></div><div class="casAuthors">Reungwetwattana Thanyanan; Dy Grace Kho</div><div class="citationInfo"><span class="NLM_cas:title">Journal of carcinogenesis</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">22</span>
        ISSN:<span class="NLM_cas:issn">1477-3163</span>.
    </div><div class="casAbstract">The iterative discovery in various malignancies during the past decades that a number of aberrant tumorigenic processes and signal transduction pathways are mediated by "druggable" protein kinases has led to a revolutionary change in drug development.  In non-small cell lung cancer (NSCLC), the ErbB family of receptors (e.g., EGFR [epidermal growth factor receptor], HER2 [human epidermal growth factor receptor 2]), RAS (rat sarcoma gene), BRAF (v-raf murine sarcoma viral oncogene homolog B1), MAPK (mitogen-activated protein kinase) c-MET (c-mesenchymal-epithelial transition), FGFR (fibroblast growth factor receptor), DDR2 (discoidin domain receptor 2), PIK3CA (phosphatidylinositol-4,5-bisphosphate3-kinase, catalytic subunit alpha)), PTEN (phosphatase and tensin homolog), AKT (protein kinase B), ALK (anaplastic lym phoma kinase), RET (rearranged during transfection), ROS1 (reactive oxygen species 1) and EPH (erythropoietin-producing hepatoma) are key targets of various agents currently in clinical development.  These oncogenic targets exert their selective growth advantage through various intercommunicating pathways, such as through RAS/RAF/MEK, phosphoinositide 3-kinase/AKT/mammalian target of rapamycin and SRC-signal transduction and transcription signaling.  The recent clinical studies, EGFR tyrosine kinase inhibitors and crizotinib were considered as strongly effective targeted therapies in metastatic NSCLC.  Currently, five molecular targeted agents were approved for treatment of advanced NSCLC: Gefitinib, erlotinib and afatinib for positive EGFR mutation, crizotinib for positive echinoderm microtubule-associated protein-like 4 (EML4)-ALK translocation and bevacizumab.  Moreover, oncogenic mutant proteins are subject to regulation by protein trafficking pathways, specifically through the heat shock protein 90 system.  Drug combinations affecting various nodes in these signaling and intracellular processes are predicted and demonstrated to be synergistic and advantageous in overcoming treatment resistance compared with monotherapy approaches.  Understanding the role of the tumor microenvironment in the development and maintenance of the malignant phenotype provided additional therapeutic approaches as well.  More recently, improved knowledge on tumor immunology has set the stage for promising immunotherapies in NSCLC.  This review will focus on the rationale for the development of targeted therapies in NSCLC and the various strategies employed in preventing or overcoming the inevitable occurrence of treatment resistance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQlh8ppikYuFoqdXsLcGTEUfW6udTcc2eZrvlMrmASBqLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2cfls1Gmug%253D%253D&md5=7bd8f4f93f5c7a2b19523fde35af75b7</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.4103%2F1477-3163.123972&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4103%252F1477-3163.123972%26sid%3Dliteratum%253Aachs%26aulast%3DReungwetwattana%26aufirst%3DT.%26aulast%3DDy%26aufirst%3DG.%2BK.%26atitle%3DTargeted%2520therapies%2520in%2520development%2520for%2520non-small%2520cell%2520lung%2520cancer%26jtitle%3DJ.%2520Carcinog.%26date%3D2013%26volume%3D12%26spage%3D22%26doi%3D10.4103%2F1477-3163.123972" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref59"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref59'); return false;" data-citation="" class="refNumLink">59</a></strong><div class="NLM_citation" id="cit59"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chan, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hughes, B. G. M.</span></span> <span> </span><span class="NLM_article-title">Targeted therapy for non-small cell lung cancer: current standards and the promise of the future</span>. <i>Transl. Lung Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">36</span>– <span class="NLM_lpage">54</span>, <span class="refDoi"> DOI: 10.3978/j.issn.2218-6751.2014.05.01</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=10.3978%2Fj.issn.2218-6751.2014.05.01" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=25806345" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtFCms7vL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2015&pages=36-54&author=B.+A.+Chanauthor=B.+G.+M.+Hughes&title=Targeted+therapy+for+non-small+cell+lung+cancer%3A+current+standards+and+the+promise+of+the+future&doi=10.3978%2Fj.issn.2218-6751.2014.05.01"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">Targeted therapy for non-small cell lung cancer: current standards and the promise of the future</span></div><div class="casAuthors">Chan, Bryan A.; Hughes, Brett G. M.</div><div class="citationInfo"><span class="NLM_cas:title">Translational Lung Cancer Research</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">36-54</span>CODEN:
                <span class="NLM_cas:coden">TLCRA9</span>;
        ISSN:<span class="NLM_cas:issn">2226-4477</span>.
    
            (<span class="NLM_cas:orgname">Pioneer Bioscience Publishing Co.</span>)
        </div><div class="casAbstract">In recent years, there has been a major paradigm shift in the management of non-small cell lung cancer (NSCLC).  NSCLC should now be further sub-classified by histol. and driver mutation if one is known or present.  Translational research advances now allow such mutations to be inhibited by either receptor monoclonal antibodies (mAb) or small mol. tyrosine kinase inhibitors (TKI).  While empirical chemotherapy with a platinum-doublet remains the gold std. for advanced NSCLC without a known driver mutation, targeted therapy is pushing the boundary to significantly improve patient outcomes and quality of life.  In this review, we will examine the major subtypes of oncogenic drivers behind NSCLC as well as the development of targeted agents available to treat them both now and in the foreseeable future.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrpzORp4rfH87Vg90H21EOLACvtfcHk0lhXD7rDd_lbng"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtFCms7vL&md5=0e9254cd6c0bacd25d1776c295f45207</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.3978%2Fj.issn.2218-6751.2014.05.01&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3978%252Fj.issn.2218-6751.2014.05.01%26sid%3Dliteratum%253Aachs%26aulast%3DChan%26aufirst%3DB.%2BA.%26aulast%3DHughes%26aufirst%3DB.%2BG.%2BM.%26atitle%3DTargeted%2520therapy%2520for%2520non-small%2520cell%2520lung%2520cancer%253A%2520current%2520standards%2520and%2520the%2520promise%2520of%2520the%2520future%26jtitle%3DTransl.%2520Lung%2520Cancer%2520Res.%26date%3D2015%26volume%3D4%26spage%3D36%26epage%3D54%26doi%3D10.3978%2Fj.issn.2218-6751.2014.05.01" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref60"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref60'); return false;" data-citation="" class="refNumLink">60</a></strong><div class="NLM_citation" id="cit60"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Giaccone, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zucali, P. A.</span></span> <span> </span><span class="NLM_article-title">Src as a potential therapeutic target in non-small-cell lung cancer</span>. <i>Ann. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">1219</span>– <span class="NLM_lpage">1223</span>, <span class="refDoi"> DOI: 10.1093/annonc/mdn048</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=10.1093%2Fannonc%2Fmdn048" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=18388349" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=1%3ACAS%3A280%3ADC%252BD1cvgvVyrsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2008&pages=1219-1223&author=G.+Giacconeauthor=P.+A.+Zucali&title=Src+as+a+potential+therapeutic+target+in+non-small-cell+lung+cancer&doi=10.1093%2Fannonc%2Fmdn048"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">Src as a potential therapeutic target in non-small-cell lung cancer</span></div><div class="casAuthors">Giaccone G; Zucali P A</div><div class="citationInfo"><span class="NLM_cas:title">Annals of oncology : official journal of the European Society for Medical Oncology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1219-1223</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Lung cancer is the most common cause of cancer-related death, with non-small-cell lung cancer (NSCLC) accounting for 80%-85% of all cases.  Although survival rates are reasonably good for patients diagnosed with very early disease, the majority of patients present with advanced disease.  For these patients, palliation and improvements in quality of life are the primary goals of therapy.  Although chemotherapeutic agents remain the cornerstone of first-line therapy, these agents have limited use in patients who have relapsed and have metastatic disease.  Therefore, new strategies are required to improve survival and quality of life in this setting.  With the substantial advances in our understanding of tumour biology, it has been possible to identify signalling pathways involved in mediating tumour growth and progression.  These pathways offer targets for new biological agents such as small molecule inhibitors and monoclonal antibodies.  One such target is Src, a tyrosine kinase that is involved in multiple aspects of tumorigenesis including proliferation, migration and angiogenesis.  Increased levels of Src expression have been found in a range of cancers, especially breast, colorectal, prostate and lung.  Preliminary preclinical data and pharmacodynamic data suggest that Src inhibition is a viable therapeutic option in the treatment of advanced NSCLC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSDd4hYJocBQtj_Vg2vObihfW6udTcc2eYfrIDQfhIXwrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD1cvgvVyrsw%253D%253D&md5=9e95b141cb44a1999b0625e34f30861a</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1093%2Fannonc%2Fmdn048&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fannonc%252Fmdn048%26sid%3Dliteratum%253Aachs%26aulast%3DGiaccone%26aufirst%3DG.%26aulast%3DZucali%26aufirst%3DP.%2BA.%26atitle%3DSrc%2520as%2520a%2520potential%2520therapeutic%2520target%2520in%2520non-small-cell%2520lung%2520cancer%26jtitle%3DAnn.%2520Oncol.%26date%3D2008%26volume%3D19%26spage%3D1219%26epage%3D1223%26doi%3D10.1093%2Fannonc%2Fmdn048" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref61"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref61'); return false;" data-citation="" class="refNumLink">61</a></strong><div class="NLM_citation" id="cit61"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rusnak, D. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lackey, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Affleck, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wood, E. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alligood, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rhodes, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keith, B. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knight, W. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mullin, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilmer, T. M.</span></span> <span> </span><span class="NLM_article-title">The effects of the novel, reversible epidermal growth factor receptor/ErbB-2 tyrosine kinase inhibitor, GW2016, on the growth of human normal and tumor-derived cell lines in vitro and in vivo</span>. <i>Mol. Cancer Ther.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">85</span>– <span class="NLM_lpage">94</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=12467226" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=1%3ACAS%3A528%3ADC%252BD38XlsFSmtbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2001&pages=85-94&author=D.+W.+Rusnakauthor=K.+Lackeyauthor=K.+Affleckauthor=E.+R.+Woodauthor=K.+J.+Alligoodauthor=N.+Rhodesauthor=B.+R.+Keithauthor=D.+M.+Murrayauthor=W.+B.+Knightauthor=R.+J.+Mullinauthor=T.+M.+Gilmer&title=The+effects+of+the+novel%2C+reversible+epidermal+growth+factor+receptor%2FErbB-2+tyrosine+kinase+inhibitor%2C+GW2016%2C+on+the+growth+of+human+normal+and+tumor-derived+cell+lines+in+vitro+and+in+vivo"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">The effects of the novel, reversible epidermal growth factor receptor/ErbB-2 tyrosine kinase inhibitor, GW2016, on the growth of human normal and tumor-derived cell lines in vitro and in vivo</span></div><div class="casAuthors">Rusnak, David W.; Lackey, Karen; Affleck, Karen; Wood, Edgar R.; Alligood, Krystal J.; Rhodes, Nelson; Keith, Barry R.; Murray, Doris M.; Knight, W. Blaine; Mullin, Robert J.; Gilmer, Tona M.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cancer Therapeutics</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">85-94</span>CODEN:
                <span class="NLM_cas:coden">MCTOCF</span>;
        ISSN:<span class="NLM_cas:issn">1535-7163</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">The epidermal growth factor receptor (EGFR) and ErbB-2 transmembrane tyrosine kinases are currently being targeted by various mechanisms in the treatment of cancer.  GW2016 is a potent inhibitor of the ErbB-2 and EGFR tyrosine kinase domains with IC50 values against purified EGFR and ErbB-2 of 10.2 and 9.8 nM, resp.  This report describes the efficacy in cell growth assays of GW2016 on human tumor cell lines overexpressing either EGFR or ErbB-2: HN5 (head and neck), A-431 (vulva), BT474 (breast), CaLu-3 (lung), and N87 (gastric).  Normal human foreskin fibroblasts, nontumorigenic epithelial cells (HB4a), and nonoverexpressing tumor cells (MCF-7 and T47D) were tested as neg. controls.  After 3 days of compd. exposure, av. IC50 values for growth inhibition in the EGFR- and ErbB-2-overexpressing tumor cell lines were <0.16 μM.  The av. selectivity for the tumor cells vs. the human foreskin fibroblast cell line was 100-fold.  Inhibition of EGFR and ErbB-2 receptor autophosphorylation and phosphorylation of the downstream modulator, AKT, was verified by Western blot anal. in the BT474 and HN5 cell lines.  As a measure of cytotoxicity vs. growth arrest, the HN5 and BT474 cells were assessed in an outgrowth assay after a transient exposure to GW2016.  The cells were treated for 3 days in five concns. of GW2016, and cell growth was monitored for an addnl. 12 days after removal of the compd.  In each of these tumor cell lines, concns. of GW2016 were reached where outgrowth did not occur.  Furthermore, growth arrest and cell death were obsd. in parallel expts., as detd. by bromodeoxyuridine incorporation and propidium iodide staining.  GW2016 treatment inhibited tumor xenograft growth of the HN5 and BT474 cells in a dose-responsive manner at 30 and 100 mg/kg orally, twice daily, with complete inhibition of tumor growth at the higher dose.  Together, these results indicate that GW2016 achieves excellent potency on tumor cells with selectivity for tumor vs. normal cells and suggest that GW2016 has value as a therapy for patients with tumors overexpressing either EGFR or ErbB-2.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp5OJ41zsWu6rVg90H21EOLACvtfcHk0ljUQe_aQ3I4oA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XlsFSmtbs%253D&md5=64fd8cadce83eda3b9b82ab50f1e57b0</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DRusnak%26aufirst%3DD.%2BW.%26aulast%3DLackey%26aufirst%3DK.%26aulast%3DAffleck%26aufirst%3DK.%26aulast%3DWood%26aufirst%3DE.%2BR.%26aulast%3DAlligood%26aufirst%3DK.%2BJ.%26aulast%3DRhodes%26aufirst%3DN.%26aulast%3DKeith%26aufirst%3DB.%2BR.%26aulast%3DMurray%26aufirst%3DD.%2BM.%26aulast%3DKnight%26aufirst%3DW.%2BB.%26aulast%3DMullin%26aufirst%3DR.%2BJ.%26aulast%3DGilmer%26aufirst%3DT.%2BM.%26atitle%3DThe%2520effects%2520of%2520the%2520novel%252C%2520reversible%2520epidermal%2520growth%2520factor%2520receptor%252FErbB-2%2520tyrosine%2520kinase%2520inhibitor%252C%2520GW2016%252C%2520on%2520the%2520growth%2520of%2520human%2520normal%2520and%2520tumor-derived%2520cell%2520lines%2520in%2520vitro%2520and%2520in%2520vivo%26jtitle%3DMol.%2520Cancer%2520Ther.%26date%3D2001%26volume%3D1%26spage%3D85%26epage%3D94" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref62"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref62'); return false;" data-citation="" class="refNumLink">62</a></strong><div class="NLM_citation" id="cit62"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ross, H. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blumenschein, G. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aisner, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Damjanov, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dowlati, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garst, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rigas, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smylie, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hassani, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allen, K. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leopold, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zaks, T. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shepherd, F. A.</span></span> <span> </span><span class="NLM_article-title">Randomized phase II multicenter trial of two schedules of lapatinib as first-or second-line monotherapy in patients with advanced or metastatic non-small cell lung cancer</span>. <i>Clin. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">1938</span>– <span class="NLM_lpage">1949</span>, <span class="refDoi"> DOI: 10.1158/1078-0432.CCR-08-3328</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=10.1158%2F1078-0432.CCR-08-3328" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=20215545" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=1%3ACAS%3A528%3ADC%252BC3cXjtFyhurY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2010&pages=1938-1949&author=H.+J.+Rossauthor=G.+R.+Blumenscheinauthor=J.+Aisnerauthor=N.+Damjanovauthor=A.+Dowlatiauthor=J.+Garstauthor=J.+R.+Rigasauthor=M.+Smylieauthor=H.+Hassaniauthor=K.+E.+Allenauthor=L.+Leopoldauthor=T.+Z.+Zaksauthor=F.+A.+Shepherd&title=Randomized+phase+II+multicenter+trial+of+two+schedules+of+lapatinib+as+first-or+second-line+monotherapy+in+patients+with+advanced+or+metastatic+non-small+cell+lung+cancer&doi=10.1158%2F1078-0432.CCR-08-3328"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit62R"><div class="casContent"><span class="casTitleNuber">62</span><div class="casTitle"><span class="NLM_cas:atitle">Randomized Phase II Multicenter Trial of Two Schedules of Lapatinib as First- or Second-Line Monotherapy in Patients with Advanced or Metastatic Non-Small Cell Lung Cancer</span></div><div class="casAuthors">Ross, Helen J.; Blumenschein, George R., Jr.; Aisner, Joseph; Damjanov, Nevena; Dowlati, Afshin; Garst, Jennifer; Rigas, James R.; Smylie, Michael; Hassani, Habib; Allen, Kimberly E.; Leopold, Lance; Zaks, Tal Z.; Shepherd, Frances A.</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1938-1949</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">PURPOSE: This randomized phase II study was initially designed to test the activity of two dose schedules of lapatinib (GW572016H), an oral, reversible, dual tyrosine kinase inhibitor of epidermal growth factor receptor (EGFR) and human EGFR-2 (HER2/neu; HER2), in chemotherapy-naive patients with non-small cell lung cancer (NSCLC); it was later amended to target patients with bronchioloalveolar carcinoma or no smoking history.  Exptl. Design: Patients with good performance status and recurrent or metastatic NSCLC were randomized to lapatinib (orally, 1,500 mg once daily or 500 mg twice daily) until progression or intolerance.  Patients could have had a max. of one prior systemic therapy (chemotherapy or biol. therapy) for NSCLC.  Safety and activity were assessed every 4 and 8 wk, resp.  Tumors were analyzed for EGFR and HER2 mutations and/or amplifications.  RESULTS: Of 75 patients in the nontargeted population, 1 (1.3%) had partial response and 16 (21%) had stable disease of ≥24 wk.  No complete or partial responses were obsd. in 56 patients in the targeted population; 14 (25%) had stable disease of ≥24 wk.  No responses were seen in three patients with EGFR mutations and five with EGFR gene amplification.  No mutations in HER2 were found.  One of two patients with HER2 amplification had a 51% decrease in tumor size; however, this response was unconfirmed.  The most common adverse events were grade 1 or 2 diarrhea, rash, fatigue, nausea, and anorexia.  Adverse events were similar across dosing regimens.  CONCLUSIONS: Lapatinib was well tolerated, with no notable difference in toxicity between treatment groups.  Lapatinib monotherapy did not induce a significant no. of tumor regressions in NSCLC.  Further studies may be warranted to det. whether lapatinib is active in combination with other agents in the treatment of NSCLC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo-5ZsLpn50LrVg90H21EOLACvtfcHk0liGkJgIx53sgQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXjtFyhurY%253D&md5=0bfa99df35f30bb9e9ef0b2addb23e28</span></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1158%2F1078-0432.CCR-08-3328&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1078-0432.CCR-08-3328%26sid%3Dliteratum%253Aachs%26aulast%3DRoss%26aufirst%3DH.%2BJ.%26aulast%3DBlumenschein%26aufirst%3DG.%2BR.%26aulast%3DAisner%26aufirst%3DJ.%26aulast%3DDamjanov%26aufirst%3DN.%26aulast%3DDowlati%26aufirst%3DA.%26aulast%3DGarst%26aufirst%3DJ.%26aulast%3DRigas%26aufirst%3DJ.%2BR.%26aulast%3DSmylie%26aufirst%3DM.%26aulast%3DHassani%26aufirst%3DH.%26aulast%3DAllen%26aufirst%3DK.%2BE.%26aulast%3DLeopold%26aufirst%3DL.%26aulast%3DZaks%26aufirst%3DT.%2BZ.%26aulast%3DShepherd%26aufirst%3DF.%2BA.%26atitle%3DRandomized%2520phase%2520II%2520multicenter%2520trial%2520of%2520two%2520schedules%2520of%2520lapatinib%2520as%2520first-or%2520second-line%2520monotherapy%2520in%2520patients%2520with%2520advanced%2520or%2520metastatic%2520non-small%2520cell%2520lung%2520cancer%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D2010%26volume%3D16%26spage%3D1938%26epage%3D1949%26doi%3D10.1158%2F1078-0432.CCR-08-3328" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref63"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref63'); return false;" data-citation="" class="refNumLink">63</a></strong><div class="NLM_citation" id="cit63"><span><span class="NLM_contrib-group"><span class="NLM_string-name">O’Hare, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shakespeare, W. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eide, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rivera, V. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adrian, L. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, W.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Metcalf, C. A.,  III</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tyner, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loriaux, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Corbin, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wardwell, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ning, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keats, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sundaramoorthi, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Snodgrass, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Commodore, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sawyer, T. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dalgarno, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deininger, M. W. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Druker, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clackson, T.</span></span> <span> </span><span class="NLM_article-title">AP24534, a pan-BCR-ABL inhibitor for chronic myeloid leukemia, potently inhibits the T315I mutant and overcomes mutation-based resistance</span>. <i>Cancer Cell</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">401</span>– <span class="NLM_lpage">412</span>, <span class="refDoi"> DOI: 10.1016/j.ccr.2009.09.028</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=10.1016%2Fj.ccr.2009.09.028" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=19878872" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsFKltb3N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2009&pages=401-412&author=T.+O%E2%80%99Hareauthor=W.+C.+Shakespeareauthor=X.+Zhuauthor=C.+A.+Eideauthor=V.+M.+Riveraauthor=F.+Wangauthor=L.+T.+Adrianauthor=T.+Zhouauthor=W.-S.+Huangauthor=Q.+Xuauthor=C.+A.+Metcalfauthor=J.+W.+Tynerauthor=M.+M.+Loriauxauthor=A.+S.+Corbinauthor=S.+Wardwellauthor=Y.+Ningauthor=J.+A.+Keatsauthor=Y.+Wangauthor=R.+Sundaramoorthiauthor=M.+Thomasauthor=D.+Zhouauthor=J.+Snodgrassauthor=L.+Commodoreauthor=T.+K.+Sawyerauthor=D.+C.+Dalgarnoauthor=M.+W.+N.+Deiningerauthor=B.+J.+Drukerauthor=T.+Clackson&title=AP24534%2C+a+pan-BCR-ABL+inhibitor+for+chronic+myeloid+leukemia%2C+potently+inhibits+the+T315I+mutant+and+overcomes+mutation-based+resistance&doi=10.1016%2Fj.ccr.2009.09.028"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit63R"><div class="casContent"><span class="casTitleNuber">63</span><div class="casTitle"><span class="NLM_cas:atitle">AP24534, a pan-BCR-ABL inhibitor for chronic myeloid leukemia, potently inhibits the T315I mutant and overcomes mutation-based resistance</span></div><div class="casAuthors">O'Hare, Thomas; Shakespeare, William C.; Zhu, Xiaotian; Eide, Christopher A.; Rivera, Victor M.; Wang, Frank; Adrian, Lauren T.; Zhou, Tianjun; Huang, Wei-Sheng; Xu, Qihong; Metcalf, Chester A., III; Tyner, Jeffrey W.; Loriaux, Marc M.; Corbin, Amie S.; Wardwell, Scott; Ning, Yaoyu; Keats, Jeffrey A.; Wang, Yihan; Sundaramoorthi, Raji; Thomas, Mathew; Zhou, Dong; Snodgrass, Joseph; Commodore, Lois; Sawyer, Tomi K.; Dalgarno, David C.; Deininger, Michael W. N.; Druker, Brian J.; Clackson, Tim</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">401-412</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Inhibition of BCR-ABL by imatinib induces durable responses in many patients with chronic myeloid leukemia (CML), but resistance attributable to kinase domain mutations can lead to relapse and a switch to second-line therapy with nilotinib or dasatinib.  Despite three approved therapeutic options, the cross-resistant BCR-ABLT315I mutation and compd. mutants selected on sequential inhibitor therapy remain major clin. challenges.  We report design and preclin. evaluation of AP24534, a potent, orally available multitargeted kinase inhibitor active against T315I and other BCR-ABL mutants.  AP24534 inhibited all tested BCR-ABL mutants in cellular and biochem. assays, suppressed BCR-ABLT315I-driven tumor growth in mice, and completely abrogated resistance in cell-based mutagenesis screens.  Our work supports clin. evaluation of AP24534 as a pan-BCR-ABL inhibitor for treatment of CML.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGox3g7ZYNXwLbVg90H21EOLACvtfcHk0liILQ9dsmzxxQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsFKltb3N&md5=62a383734fc345b97647ba71aa7aee04</span></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1016%2Fj.ccr.2009.09.028&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccr.2009.09.028%26sid%3Dliteratum%253Aachs%26aulast%3DO%25E2%2580%2599Hare%26aufirst%3DT.%26aulast%3DShakespeare%26aufirst%3DW.%2BC.%26aulast%3DZhu%26aufirst%3DX.%26aulast%3DEide%26aufirst%3DC.%2BA.%26aulast%3DRivera%26aufirst%3DV.%2BM.%26aulast%3DWang%26aufirst%3DF.%26aulast%3DAdrian%26aufirst%3DL.%2BT.%26aulast%3DZhou%26aufirst%3DT.%26aulast%3DHuang%26aufirst%3DW.-S.%26aulast%3DXu%26aufirst%3DQ.%26aulast%3DMetcalf%26aufirst%3DC.%2BA.%26aulast%3DTyner%26aufirst%3DJ.%2BW.%26aulast%3DLoriaux%26aufirst%3DM.%2BM.%26aulast%3DCorbin%26aufirst%3DA.%2BS.%26aulast%3DWardwell%26aufirst%3DS.%26aulast%3DNing%26aufirst%3DY.%26aulast%3DKeats%26aufirst%3DJ.%2BA.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DSundaramoorthi%26aufirst%3DR.%26aulast%3DThomas%26aufirst%3DM.%26aulast%3DZhou%26aufirst%3DD.%26aulast%3DSnodgrass%26aufirst%3DJ.%26aulast%3DCommodore%26aufirst%3DL.%26aulast%3DSawyer%26aufirst%3DT.%2BK.%26aulast%3DDalgarno%26aufirst%3DD.%2BC.%26aulast%3DDeininger%26aufirst%3DM.%2BW.%2BN.%26aulast%3DDruker%26aufirst%3DB.%2BJ.%26aulast%3DClackson%26aufirst%3DT.%26atitle%3DAP24534%252C%2520a%2520pan-BCR-ABL%2520inhibitor%2520for%2520chronic%2520myeloid%2520leukemia%252C%2520potently%2520inhibits%2520the%2520T315I%2520mutant%2520and%2520overcomes%2520mutation-based%2520resistance%26jtitle%3DCancer%2520Cell%26date%3D2009%26volume%3D16%26spage%3D401%26epage%3D412%26doi%3D10.1016%2Fj.ccr.2009.09.028" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref64"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref64'); return false;" data-citation="" class="refNumLink">64</a></strong><div class="NLM_citation" id="cit64"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ren, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hong, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Biddinger, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Silva, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowell, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hao, Z.</span></span> <span> </span><span class="NLM_article-title">Novel FGFR inhibitor ponatinib suppresses the growth of non-small cell lung cancer cells overexpressing FGFR1</span>. <i>Oncol. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">2181</span>– <span class="NLM_lpage">2190</span>, <span class="refDoi"> DOI: 10.3892/or.2013.2386</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=10.3892%2For.2013.2386" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=23563700" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=1%3ACAS%3A528%3ADC%252BC3sXpsFCksbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2013&pages=2181-2190&author=M.+Renauthor=M.+Hongauthor=G.+Liuauthor=H.+Wangauthor=V.+Patelauthor=P.+Biddingerauthor=J.+Silvaauthor=J.+Cowellauthor=Z.+Hao&title=Novel+FGFR+inhibitor+ponatinib+suppresses+the+growth+of+non-small+cell+lung+cancer+cells+overexpressing+FGFR1&doi=10.3892%2For.2013.2386"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit64R"><div class="casContent"><span class="casTitleNuber">64</span><div class="casTitle"><span class="NLM_cas:atitle">Novel FGFR inhibitor ponatinib suppresses the growth of non-small cell lung cancer cells overexpressing FGFR1</span></div><div class="casAuthors">Ren, Mingqiang; Hong, Mei; Liu, Gentao; Wang, Hongjin; Patel, Vijay; Biddinger, Paul; Silva, Jeane; Cowell, John; Hao, Zhonglin</div><div class="citationInfo"><span class="NLM_cas:title">Oncology Reports</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2181-2190</span>CODEN:
                <span class="NLM_cas:coden">OCRPEW</span>;
        ISSN:<span class="NLM_cas:issn">1021-335X</span>.
    
            (<span class="NLM_cas:orgname">Spandidos Publications Ltd.</span>)
        </div><div class="casAbstract">Lung cancer is still the leading cause of cancer-related deaths worldwide.  Identifying new oncogenic drivers and developing efficient inhibitors through mol. targeting approaches are crucial for improving therapies.  The aim of this study was to investigate whether targeting fibroblast growth factor receptor 1 (FGFR1) with ponatinib inhibits the cell growth in both established and primary lung cancer cells over-expressing FGFR1.  Eighty-eight non-small cell lung cancer (NSCLC) and paired normal tissue specimens were analyzed by real-time RT-PCR for FGFR1 gene expression.  We identified four cell lines and two newly established primary lung cancer cultures that showed high FGFR1 expression levels, and evaluated the effect of the novel FGFR1 inhibitor ponatinib on cell growth.  Approx. 50% (30 out of 59) NSCLC specimens expressed FGFR1 >2-fold compared with their adjacent normal counterparts using quant. RT-PCR.  Ponatinib treatment of established NSCLC cell lines expressing higher levels of FGFR1 resulted in marked cell growth inhibition and suppression of clonogenicity.  This growth inhibition was assocd. with inactivation of FGFR1 and its downstream targets.  FGFR1 knockdown by shRNA achieved similar results when compared to treatment with ponatinib.  Furthermore, ponatinib was able to significantly inhibit the growth of primary lung cancer cultures in vitro.  Our data indicate that pharmacol. inhibition of FGFR1 kinase activity with ponatinib may be effective for the treatment of lung cancer patients whose tumors overexpress FGFR1.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpai6Rg1CUEubVg90H21EOLACvtfcHk0liILQ9dsmzxxQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXpsFCksbc%253D&md5=52bd8443e6319dd96736d4b2628c24d9</span></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=10.3892%2For.2013.2386&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3892%252For.2013.2386%26sid%3Dliteratum%253Aachs%26aulast%3DRen%26aufirst%3DM.%26aulast%3DHong%26aufirst%3DM.%26aulast%3DLiu%26aufirst%3DG.%26aulast%3DWang%26aufirst%3DH.%26aulast%3DPatel%26aufirst%3DV.%26aulast%3DBiddinger%26aufirst%3DP.%26aulast%3DSilva%26aufirst%3DJ.%26aulast%3DCowell%26aufirst%3DJ.%26aulast%3DHao%26aufirst%3DZ.%26atitle%3DNovel%2520FGFR%2520inhibitor%2520ponatinib%2520suppresses%2520the%2520growth%2520of%2520non-small%2520cell%2520lung%2520cancer%2520cells%2520overexpressing%2520FGFR1%26jtitle%3DOncol.%2520Rep.%26date%3D2013%26volume%3D29%26spage%3D2181%26epage%3D2190%26doi%3D10.3892%2For.2013.2386" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref65"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref65'); return false;" data-citation="" class="refNumLink">65</a></strong><div class="NLM_citation" id="cit65"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Daud, A. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krishnamurthi, S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saleh, M. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gitlitz, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Borad, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gold, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiorean, E. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Springett, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abbas, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Agarwal, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bardy-Bouxin, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hsyu, P.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leip, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turnbull, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zacharchuk, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Messersmith, W. A.</span></span> <span> </span><span class="NLM_article-title">Phase I study of bosutinib, a src/abl tyrosine kinase inhibitor, administered to patients with advanced solid tumors</span>. <i>Clin. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">1092</span>– <span class="NLM_lpage">1100</span>, <span class="refDoi"> DOI: 10.1158/1078-0432.CCR-11-2378</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=10.1158%2F1078-0432.CCR-11-2378" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=22179664" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=1%3ACAS%3A528%3ADC%252BC38XisVGhu7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2012&pages=1092-1100&author=A.+I.+Daudauthor=S.+S.+Krishnamurthiauthor=M.+N.+Salehauthor=B.+J.+Gitlitzauthor=M.+J.+Boradauthor=P.+J.+Goldauthor=E.+G.+Chioreanauthor=G.+M.+Springettauthor=R.+Abbasauthor=S.+Agarwalauthor=N.+Bardy-Bouxinauthor=P.-H.+Hsyuauthor=E.+Leipauthor=K.+Turnbullauthor=C.+Zacharchukauthor=W.+A.+Messersmith&title=Phase+I+study+of+bosutinib%2C+a+src%2Fabl+tyrosine+kinase+inhibitor%2C+administered+to+patients+with+advanced+solid+tumors&doi=10.1158%2F1078-0432.CCR-11-2378"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit65R"><div class="casContent"><span class="casTitleNuber">65</span><div class="casTitle"><span class="NLM_cas:atitle">Phase I Study of Bosutinib, a Src/Abl Tyrosine Kinase Inhibitor, Administered to Patients with Advanced Solid Tumors</span></div><div class="casAuthors">Daud, Adil I.; Krishnamurthi, Smitha S.; Saleh, Mansoor N.; Gitlitz, Barbara J.; Borad, Mitesh J.; Gold, Philip J.; Chiorean, Elena G.; Springett, Gregory M.; Abbas, Richat; Agarwal, Shefali; Bardy-Bouxin, Nathalie; Hsyu, Poe-Hirr; Leip, Eric; Turnbull, Kathleen; Zacharchuk, Charles; Messersmith, Wells A.</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1092-1100</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">PURPOSE: Bosutinib, a potent ATP-competitive, quinolinecarbonitrile Src/Abl kinase inhibitor, was tested in this first-in-human phase I trial in patients with advanced solid tumor malignancies.  PATIENTS AND METHODS: This trial was conducted in 2 parts.  In part 1 (dose escalation), increasing oral bosutinib doses were administered using a 3 + 3 design.  In part 2 (dose expansion), approx. 30 patients each with refractory colorectal, pancreas, or non-small cell lung cancer were treated at the recommended phase II dose (RP2D).  Primary efficacy endpoints for part 2 were median progression-free survival (colorectal and non-small cell lung) and median overall survival (pancreas).  RESULTS: In part 1, dose-limiting toxicities of grade 3 diarrhea (two patients) and grade 3 rash occurred with bosutinib 600 mg/day and the max. tolerated dose identified was 500 mg/day.  However, the majority of patients treated with 500 mg/day had grade 2 or greater gastrointestinal toxicity, and 400 mg/day was identified as the RP2D.  The most common bosutinib-related adverse events were nausea (60% patients), diarrhea (47%), vomiting (40%), fatigue (38%), and anorexia (36%).  Bosutinib had a mean half-life of 19 to 20 h at the RP2D.  A partial response (breast) and unconfirmed complete response (pancreas) were obsd.; 8 of 112 evaluable patients had stable disease for 22 to 101 wk.  However, the primary efficacy endpoints for part 2 were not met.  CONCLUSIONS: Bosutinib was generally well tolerated in patients with solid tumors, with the main toxicity being gastrointestinal.  The RP2D was 400 mg/day orally.  Further study of bosutinib is planned in combination regimens.  Clin Cancer Res; 18(4); 1092-100.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrYIJzJMnax9LVg90H21EOLACvtfcHk0liSZYHzRXrejQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XisVGhu7Y%253D&md5=6ed35e2929a66d5f7f8507cfdc085ba2</span></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=10.1158%2F1078-0432.CCR-11-2378&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1078-0432.CCR-11-2378%26sid%3Dliteratum%253Aachs%26aulast%3DDaud%26aufirst%3DA.%2BI.%26aulast%3DKrishnamurthi%26aufirst%3DS.%2BS.%26aulast%3DSaleh%26aufirst%3DM.%2BN.%26aulast%3DGitlitz%26aufirst%3DB.%2BJ.%26aulast%3DBorad%26aufirst%3DM.%2BJ.%26aulast%3DGold%26aufirst%3DP.%2BJ.%26aulast%3DChiorean%26aufirst%3DE.%2BG.%26aulast%3DSpringett%26aufirst%3DG.%2BM.%26aulast%3DAbbas%26aufirst%3DR.%26aulast%3DAgarwal%26aufirst%3DS.%26aulast%3DBardy-Bouxin%26aufirst%3DN.%26aulast%3DHsyu%26aufirst%3DP.-H.%26aulast%3DLeip%26aufirst%3DE.%26aulast%3DTurnbull%26aufirst%3DK.%26aulast%3DZacharchuk%26aufirst%3DC.%26aulast%3DMessersmith%26aufirst%3DW.%2BA.%26atitle%3DPhase%2520I%2520study%2520of%2520bosutinib%252C%2520a%2520src%252Fabl%2520tyrosine%2520kinase%2520inhibitor%252C%2520administered%2520to%2520patients%2520with%2520advanced%2520solid%2520tumors%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D2012%26volume%3D18%26spage%3D1092%26epage%3D1100%26doi%3D10.1158%2F1078-0432.CCR-11-2378" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref66"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref66'); return false;" data-citation="" class="refNumLink">66</a></strong><div class="NLM_citation" id="cit66"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lee, S.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J.-K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ahn, M.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, D.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, J.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keam, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heo, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ahn, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choi, Y.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Min, H.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeon, Y.-K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, K.</span></span> <span> </span><span class="NLM_article-title">A phase II study of vandetanib in patients with non-small cell lung cancer harboring RET rearrangement</span>. <i>J. Clin. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>34</i></span>,  <span class="NLM_fpage">9013</span>– <span class="NLM_lpage">9013</span>, <span class="refDoi"> DOI: 10.1200/JCO.2016.34.15_suppl.9013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=10.1200%2FJCO.2016.34.15_suppl.9013" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=2016&pages=9013-9013&author=S.-H.+Leeauthor=J.-K.+Leeauthor=M.-J.+Ahnauthor=D.-W.+Kimauthor=J.-M.+Sunauthor=B.+Keamauthor=T.+M.+Kimauthor=D.+S.+Heoauthor=J.+S.+Ahnauthor=Y.-L.+Choiauthor=H.-S.+Minauthor=Y.-K.+Jeonauthor=K.+Park&title=A+phase+II+study+of+vandetanib+in+patients+with+non-small+cell+lung+cancer+harboring+RET+rearrangement&doi=10.1200%2FJCO.2016.34.15_suppl.9013"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit66&amp;dbid=16384&amp;doi=10.1200%2FJCO.2016.34.15_suppl.9013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1200%252FJCO.2016.34.15_suppl.9013%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DS.-H.%26aulast%3DLee%26aufirst%3DJ.-K.%26aulast%3DAhn%26aufirst%3DM.-J.%26aulast%3DKim%26aufirst%3DD.-W.%26aulast%3DSun%26aufirst%3DJ.-M.%26aulast%3DKeam%26aufirst%3DB.%26aulast%3DKim%26aufirst%3DT.%2BM.%26aulast%3DHeo%26aufirst%3DD.%2BS.%26aulast%3DAhn%26aufirst%3DJ.%2BS.%26aulast%3DChoi%26aufirst%3DY.-L.%26aulast%3DMin%26aufirst%3DH.-S.%26aulast%3DJeon%26aufirst%3DY.-K.%26aulast%3DPark%26aufirst%3DK.%26atitle%3DA%2520phase%2520II%2520study%2520of%2520vandetanib%2520in%2520patients%2520with%2520non-small%2520cell%2520lung%2520cancer%2520harboring%2520RET%2520rearrangement%26jtitle%3DJ.%2520Clin.%2520Oncol.%26date%3D2016%26volume%3D34%26spage%3D9013%26epage%3D9013%26doi%3D10.1200%2FJCO.2016.34.15_suppl.9013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref67"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref67'); return false;" data-citation="" class="refNumLink">67</a></strong><div class="NLM_citation" id="cit67"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gao, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dai, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ou, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heymach, J. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gold, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minna, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roth, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hofstetter, W. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Swisher, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fang, B.</span></span> <span> </span><span class="NLM_article-title">Selective antitumor activity of ibrutinib in EGFR-mutant non-small cell lung cancer cells</span>. <i>J. Natl. Cancer Inst.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>106</i></span>, <span class="NLM_elocation-id">dju204</span> <span class="refDoi"> DOI: 10.1093/jnci/dju204</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=10.1093%2Fjnci%2Fdju204" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=25214559" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=106&publication_year=2014&author=W.+Gaoauthor=M.+Wangauthor=L.+Wangauthor=H.+Luauthor=S.+Wuauthor=B.+Daiauthor=Z.+Ouauthor=L.+Zhangauthor=J.+V.+Heymachauthor=K.+A.+Goldauthor=J.+Minnaauthor=J.+A.+Rothauthor=W.+L.+Hofstetterauthor=S.+G.+Swisherauthor=B.+Fang&title=Selective+antitumor+activity+of+ibrutinib+in+EGFR-mutant+non-small+cell+lung+cancer+cells&doi=10.1093%2Fjnci%2Fdju204"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit67&amp;dbid=16384&amp;doi=10.1093%2Fjnci%2Fdju204&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fjnci%252Fdju204%26sid%3Dliteratum%253Aachs%26aulast%3DGao%26aufirst%3DW.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DLu%26aufirst%3DH.%26aulast%3DWu%26aufirst%3DS.%26aulast%3DDai%26aufirst%3DB.%26aulast%3DOu%26aufirst%3DZ.%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DHeymach%26aufirst%3DJ.%2BV.%26aulast%3DGold%26aufirst%3DK.%2BA.%26aulast%3DMinna%26aufirst%3DJ.%26aulast%3DRoth%26aufirst%3DJ.%2BA.%26aulast%3DHofstetter%26aufirst%3DW.%2BL.%26aulast%3DSwisher%26aufirst%3DS.%2BG.%26aulast%3DFang%26aufirst%3DB.%26atitle%3DSelective%2520antitumor%2520activity%2520of%2520ibrutinib%2520in%2520EGFR-mutant%2520non-small%2520cell%2520lung%2520cancer%2520cells%26jtitle%3DJ.%2520Natl.%2520Cancer%2520Inst.%26date%3D2014%26volume%3D106%26doi%3D10.1093%2Fjnci%2Fdju204" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref68"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref68'); return false;" data-citation="" class="refNumLink">68</a></strong><div class="NLM_citation" id="cit68"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Metz, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, E. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soni, N. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Merta, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kifle, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hajduk, P. J.</span></span> <span> </span><span class="NLM_article-title">Navigating the kinome</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">200</span>– <span class="NLM_lpage">202</span>, <span class="refDoi"> DOI: 10.1038/nchembio.530</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=10.1038%2Fnchembio.530" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=21336281" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=1%3ACAS%3A528%3ADC%252BC3MXit1aitrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2011&pages=200-202&author=J.+T.+Metzauthor=E.+F.+Johnsonauthor=N.+B.+Soniauthor=P.+J.+Mertaauthor=L.+Kifleauthor=P.+J.+Hajduk&title=Navigating+the+kinome&doi=10.1038%2Fnchembio.530"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit68R"><div class="casContent"><span class="casTitleNuber">68</span><div class="casTitle"><span class="NLM_cas:atitle">Navigating the kinome</span></div><div class="casAuthors">Metz, James T.; Johnson, Eric F.; Soni, Niru B.; Merta, Philip J.; Kifle, Lemma; Hajduk, Philip J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">200-202</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Although it is increasingly being recognized that drug-target interaction networks can be powerful tools for the interrogation of systems biol. and the rational design of multitargeted drugs, there is no generalized, statistically validated approach to harmonizing sequence-dependent and pharmacol.-dependent networks.  Here we demonstrate the creation of a comprehensive kinome interaction network based not only on sequence comparisons but also on multiple pharmacol. parameters derived from activity profiling data.  The framework described for statistical interpretation of these network connections also enables rigorous investigation of chemotype-specific interaction networks, which is crit. for multitargeted drug design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoOgzF9Q0eH87Vg90H21EOLACvtfcHk0lh_g3cbk9yewQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXit1aitrc%253D&md5=09234ec451a4918e5aca6d65e0b65aee</span></div><a href="/servlet/linkout?suffix=cit68&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.530&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.530%26sid%3Dliteratum%253Aachs%26aulast%3DMetz%26aufirst%3DJ.%2BT.%26aulast%3DJohnson%26aufirst%3DE.%2BF.%26aulast%3DSoni%26aufirst%3DN.%2BB.%26aulast%3DMerta%26aufirst%3DP.%2BJ.%26aulast%3DKifle%26aufirst%3DL.%26aulast%3DHajduk%26aufirst%3DP.%2BJ.%26atitle%3DNavigating%2520the%2520kinome%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2011%26volume%3D7%26spage%3D200%26epage%3D202%26doi%3D10.1038%2Fnchembio.530" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref69"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref69'); return false;" data-citation="" class="refNumLink">69</a></strong><div class="NLM_citation" id="cit69"><span><span class="NLM_contrib-group"><span class="NLM_string-name">He, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garcia, E. A.</span></span> <span> </span><span class="NLM_article-title">Learning from imbalanced data</span>. <i>IEEE Trans. Knowl. Data. Eng.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">1263</span>– <span class="NLM_lpage">1284</span>, <span class="refDoi"> DOI: 10.1109/TKDE.2008.239</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=10.1109%2FTKDE.2008.239" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2009&pages=1263-1284&author=H.+Heauthor=E.+A.+Garcia&title=Learning+from+imbalanced+data&doi=10.1109%2FTKDE.2008.239"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit69&amp;dbid=16384&amp;doi=10.1109%2FTKDE.2008.239&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1109%252FTKDE.2008.239%26sid%3Dliteratum%253Aachs%26aulast%3DHe%26aufirst%3DH.%26aulast%3DGarcia%26aufirst%3DE.%2BA.%26atitle%3DLearning%2520from%2520imbalanced%2520data%26jtitle%3DIEEE%2520Trans.%2520Knowl.%2520Data.%2520Eng.%26date%3D2009%26volume%3D21%26spage%3D1263%26epage%3D1284%26doi%3D10.1109%2FTKDE.2008.239" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref70"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref70'); return false;" data-citation="" class="refNumLink">70</a></strong><div class="NLM_citation" id="cit70"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ma, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheridan, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liaw, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dahl, G. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Svetnik, V.</span></span> <span> </span><span class="NLM_article-title">Deep neural nets as a method for quantitative structure-activity relationships</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">263</span>– <span class="NLM_lpage">274</span>, <span class="refDoi"> DOI: 10.1021/ci500747n</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci500747n" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvFGns70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2015&pages=263-274&author=J.+Maauthor=R.+P.+Sheridanauthor=A.+Liawauthor=G.+E.+Dahlauthor=V.+Svetnik&title=Deep+neural+nets+as+a+method+for+quantitative+structure-activity+relationships&doi=10.1021%2Fci500747n"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit70R"><div class="casContent"><span class="casTitleNuber">70</span><div class="casTitle"><span class="NLM_cas:atitle">Deep Neural Nets as a Method for Quantitative Structure-Activity Relationships</span></div><div class="casAuthors">Ma, Junshui; Sheridan, Robert P.; Liaw, Andy; Dahl, George E.; Svetnik, Vladimir</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">263-274</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Neural networks were widely used for quant. structure-activity relationships (QSAR) in the 1990s.  Because of various practical issues (e.g., slow on large problems, difficult to train, prone to overfitting, etc.), they were superseded by more robust methods like support vector machine (SVM) and random forest (RF), which arose in the early 2000s.  The last 10 years has witnessed a revival of neural networks in the machine learning community thanks to new methods for preventing overfitting, more efficient training algorithms, and advancements in computer hardware.  In particular, deep neural nets (DNNs), i.e. neural nets with more than one hidden layer, have found great successes in many applications, such as computer vision and natural language processing.  Here we show that DNNs can routinely make better prospective predictions than RF on a set of large diverse QSAR data sets that are taken from Merck's drug discovery effort.  The no. of adjustable parameters needed for DNNs is fairly large, but our results show that it is not necessary to optimize them for individual data sets, and a single set of recommended parameters can achieve better performance than RF for most of the data sets we studied.  The usefulness of the parameters is demonstrated on addnl. data sets not used in the calibration.  Although training DNNs is still computationally intensive, using graphical processing units (GPUs) can make this issue manageable.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrwVMQ-gQwZd7Vg90H21EOLACvtfcHk0liGLLBcGTZJ5Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvFGns70%253D&md5=2c6a3f33f85a4abc32ebc33108426667</span></div><a href="/servlet/linkout?suffix=cit70&amp;dbid=16384&amp;doi=10.1021%2Fci500747n&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci500747n%26sid%3Dliteratum%253Aachs%26aulast%3DMa%26aufirst%3DJ.%26aulast%3DSheridan%26aufirst%3DR.%2BP.%26aulast%3DLiaw%26aufirst%3DA.%26aulast%3DDahl%26aufirst%3DG.%2BE.%26aulast%3DSvetnik%26aufirst%3DV.%26atitle%3DDeep%2520neural%2520nets%2520as%2520a%2520method%2520for%2520quantitative%2520structure-activity%2520relationships%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2015%26volume%3D55%26spage%3D263%26epage%3D274%26doi%3D10.1021%2Fci500747n" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref71"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref71'); return false;" data-citation="" class="refNumLink">71</a></strong><div class="NLM_citation" id="cit71"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rogers, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hahn, M.</span></span> <span> </span><span class="NLM_article-title">Extended-connectivity fingerprints</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>50</i></span>,  <span class="NLM_fpage">742</span>– <span class="NLM_lpage">754</span>, <span class="refDoi"> DOI: 10.1021/ci100050t</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci100050t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=1%3ACAS%3A528%3ADC%252BC3cXlt1Onsbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2010&pages=742-754&author=D.+Rogersauthor=M.+Hahn&title=Extended-connectivity+fingerprints&doi=10.1021%2Fci100050t"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit71R"><div class="casContent"><span class="casTitleNuber">71</span><div class="casTitle"><span class="NLM_cas:atitle">Extended-Connectivity Fingerprints</span></div><div class="casAuthors">Rogers, David; Hahn, Mathew</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">742-754</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Extended-connectivity fingerprints (ECFPs) are a novel class of topol. fingerprints for mol. characterization.  Historically, topol. fingerprints were developed for substructure and similarity searching.  ECFPs were developed specifically for structure-activity modeling.  ECFPs are circular fingerprints with a no. of useful qualities: they can be very rapidly calcd.; they are not predefined and can represent an essentially infinite no. of different mol. features (including stereochem. information); their features represent the presence of particular substructures, allowing easier interpretation of anal. results; and the ECFP algorithm can be tailored to generate different types of circular fingerprints, optimized for different uses.  While the use of ECFPs has been widely adopted and validated, a description of their implementation has not previously been presented in the literature.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGra43orI-Y7lbVg90H21EOLACvtfcHk0liGLLBcGTZJ5Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXlt1Onsbg%253D&md5=cd6c736cd7a3d280b67f5316acce8006</span></div><a href="/servlet/linkout?suffix=cit71&amp;dbid=16384&amp;doi=10.1021%2Fci100050t&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci100050t%26sid%3Dliteratum%253Aachs%26aulast%3DRogers%26aufirst%3DD.%26aulast%3DHahn%26aufirst%3DM.%26atitle%3DExtended-connectivity%2520fingerprints%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2010%26volume%3D50%26spage%3D742%26epage%3D754%26doi%3D10.1021%2Fci100050t" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref72"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref72'); return false;" data-citation="" class="refNumLink">72</a></strong><div class="NLM_citation" id="cit72"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nair, V.</span>; <span class="NLM_string-name">Hinton, G. E.</span></span><span> </span><span class="NLM_article-title">Rectified linear units improve restricted Boltzmann machines</span> <i>Proceedings of the 27th International Conference on Machine Learning (ICML-10)</i>; <span class="NLM_publisher-name">International Machine Learning Society</span>, <span class="NLM_year">2010</span>; pp  <span class="NLM_fpage">807</span>– <span class="NLM_lpage">814</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2010&pages=807-814&author=V.+Nair&author=G.+E.+Hinton&title=Proceedings+of+the+27th+International+Conference+on+Machine+Learning+%28ICML-10%29"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit72&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DNair%26aufirst%3DV.%26atitle%3DRectified%2520linear%2520units%2520improve%2520restricted%2520Boltzmann%2520machines%26btitle%3DProceedings%2520of%2520the%252027th%2520International%2520Conference%2520on%2520Machine%2520Learning%2520%2528ICML-10%2529%26pub%3DInternational%2520Machine%2520Learning%2520Society%26date%3D2010%26spage%3D807%26epage%3D814" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref73"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref73'); return false;" data-citation="" class="refNumLink">73</a></strong><div class="NLM_citation" id="cit73"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hinton, G. E.</span></span> <span> </span><span class="NLM_article-title">A Practical Guide to Training Restricted Boltzmann Machines</span>. In  <i>Neural Networks: Tricks of the Trade</i>, <span class="NLM_edition">2</span>nd ed.; <span class="NLM_contrib-group"><span class="NLM_string-name">Montavon, G.</span>, <span class="NLM_string-name">Orr, G. B.</span>, <span class="NLM_string-name">Müller, K.-R.</span></span>, Eds.; <span class="NLM_publisher-name">Springer</span>: <span class="NLM_publisher-loc">Berlin</span>, <span class="NLM_year">2012</span>; pp  <span class="NLM_fpage">599</span>– <span class="NLM_lpage">619</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=10.1007%2F978-3-642-35289-8_32" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2012&pages=599-619&author=G.+E.+Hintonauthor=G.+Montavon&author=G.+B.+Orr&author=K.-R.+M%C3%BCller&title=Neural+Networks%3A+Tricks+of+the+Trade"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit73&amp;dbid=16384&amp;doi=10.1007%2F978-3-642-35289-8_32&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1007%252F978-3-642-35289-8_32%26sid%3Dliteratum%253Aachs%26aulast%3DHinton%26aufirst%3DG.%2BE.%26atitle%3DA%2520Practical%2520Guide%2520to%2520Training%2520Restricted%2520Boltzmann%2520Machines%26btitle%3DNeural%2520Networks%253A%2520Tricks%2520of%2520the%2520Trade%26aulast%3DMontavon%26aufirst%3DG.%26pub%3DSpringer%26date%3D2012%26spage%3D599%26epage%3D619" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref74"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref74'); return false;" data-citation="" class="refNumLink">74</a></strong><div class="NLM_citation" id="cit74"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Prechelt, L.</span></span> <span> </span><span class="NLM_article-title">Early Stopping—But When?</span> In  <i>Neural Networks: Tricks of the Trade</i>, <span class="NLM_edition">2</span>nd ed.; <span class="NLM_contrib-group"><span class="NLM_string-name">Montavon, G.</span>, <span class="NLM_string-name">Orr, G. B.</span>, <span class="NLM_string-name">Müller, K.-R.</span></span>, Eds.; <span class="NLM_publisher-name">Springer</span>: <span class="NLM_publisher-loc">Berlin</span>, <span class="NLM_year">2012</span>; pp  <span class="NLM_fpage">53</span>– <span class="NLM_lpage">67</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=10.1007%2F978-3-642-35289-8_5" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2012&pages=53-67&author=L.+Precheltauthor=G.+Montavon&author=G.+B.+Orr&author=K.-R.+M%C3%BCller&title=Neural+Networks%3A+Tricks+of+the+Trade"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit74&amp;dbid=16384&amp;doi=10.1007%2F978-3-642-35289-8_5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1007%252F978-3-642-35289-8_5%26sid%3Dliteratum%253Aachs%26aulast%3DPrechelt%26aufirst%3DL.%26atitle%3DEarly%2520Stopping%25E2%2580%2594But%2520When%253F%26btitle%3DNeural%2520Networks%253A%2520Tricks%2520of%2520the%2520Trade%26aulast%3DMontavon%26aufirst%3DG.%26pub%3DSpringer%26date%3D2012%26spage%3D53%26epage%3D67" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref75"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref75'); return false;" data-citation="" class="refNumLink">75</a></strong><div class="NLM_citation" id="cit75"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kingma, D. P.</span>; <span class="NLM_string-name">Ba, J.</span></span> <span> </span><span class="NLM_article-title">Adam: A method for stochastic optimization</span>.  <i>arXiv:1412.6980</i>, <span class="NLM_year">2014</span>. arXiv.org e-Print archive.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2014&author=D.+P.+Kingma&author=J.+Ba&title=Adam%3A+A+method+for+stochastic+optimization"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit75&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dsimple%26sid%3Dliteratum%253Aachs%26aulast%3DKingma%26aufirst%3DD.%2BP.%26atitle%3DAdam%253A%2520A%2520method%2520for%2520stochastic%2520optimization%26jtitle%3DarXiv%253A1412.6980%26date%3D2014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref76"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref76'); return false;" data-citation="" class="refNumLink">76</a></strong><div class="NLM_citation" id="cit76"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Srivastava, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hinton, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krizhevsky, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sutskever, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salakhutdinov, R.</span></span> <span> </span><span class="NLM_article-title">Dropout: A simple way to prevent neural networks from overfitting</span>. <i>J. Mach. Learn. Res.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">1929</span>– <span class="NLM_lpage">1958</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2014&pages=1929-1958&author=N.+Srivastavaauthor=G.+Hintonauthor=A.+Krizhevskyauthor=I.+Sutskeverauthor=R.+Salakhutdinov&title=Dropout%3A+A+simple+way+to+prevent+neural+networks+from+overfitting"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit76&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DSrivastava%26aufirst%3DN.%26aulast%3DHinton%26aufirst%3DG.%26aulast%3DKrizhevsky%26aufirst%3DA.%26aulast%3DSutskever%26aufirst%3DI.%26aulast%3DSalakhutdinov%26aufirst%3DR.%26atitle%3DDropout%253A%2520A%2520simple%2520way%2520to%2520prevent%2520neural%2520networks%2520from%2520overfitting%26jtitle%3DJ.%2520Mach.%2520Learn.%2520Res.%26date%3D2014%26volume%3D15%26spage%3D1929%26epage%3D1958" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52'],'ref53':['cit53'],'ref54':['cit54'],'ref55':['cit55'],'ref56':['cit56'],'ref57':['cit57'],'ref58':['cit58'],'ref59':['cit59'],'ref60':['cit60'],'ref61':['cit61'],'ref62':['cit62'],'ref63':['cit63'],'ref64':['cit64'],'ref65':['cit65'],'ref66':['cit66'],'ref67':['cit67'],'ref68':['cit68'],'ref69':['cit69'],'ref70':['cit70'],'ref71':['cit71'],'ref72':['cit72'],'ref73':['cit73'],'ref74':['cit74'],'ref75':['cit75'],'ref76':['cit76']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 12 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Chenjing Cai, Shiwei Wang, Youjun Xu, Weilin Zhang, Ke Tang, Qi Ouyang, Luhua Lai, <span class="NLM_string-name hlFld-ContribAuthor">Jianfeng Pei</span>. </span><span class="cited-content_cbyCitation_article-title">Transfer Learning for Drug Discovery. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (16)
                                     , 8683-8694. <a href="https://doi.org/10.1021/acs.jmedchem.9b02147" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b02147</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b02147&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.9b02147%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DTransfer%252BLearning%252Bfor%252BDrug%252BDiscovery%26aulast%3DCai%26aufirst%3DChenjing%26date%3D2020%26date%3D2020%26date%3D2019%26date%3D23122019%26date%3D24072020%26date%3D16072020%26volume%3D63%26issue%3D16%26spage%3D8683%26epage%3D8694" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Yanxing Wang, Jianxing Hu, Junyong Lai, Yibo Li, Hongwei Jin, Lihe Zhang, Liang-Ren Zhang, <span class="NLM_string-name hlFld-ContribAuthor">Zhen-ming Liu</span>. </span><span class="cited-content_cbyCitation_article-title">TF3P: Three-Dimensional Force Fields Fingerprint Learned by Deep Capsular Network. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2020,</strong> <em>60 </em>
                                    (6)
                                     , 2754-2765. <a href="https://doi.org/10.1021/acs.jcim.0c00005" title="DOI URL">https://doi.org/10.1021/acs.jcim.0c00005</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.0c00005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.0c00005%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DTF3P%25253A%252BThree-Dimensional%252BForce%252BFields%252BFingerprint%252BLearned%252Bby%252BDeep%252BCapsular%252BNetwork%26aulast%3DWang%26aufirst%3DYanxing%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D03012020%26date%3D28052020%26date%3D11052020%26volume%3D60%26issue%3D6%26spage%3D2754%26epage%3D2765" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Fabio  Urbina</span>, <span class="hlFld-ContribAuthor ">Ana C.  Puhl</span>, <span class="hlFld-ContribAuthor ">Sean  Ekins</span>. </span><span class="cited-content_cbyCitation_article-title">Recent advances in drug repurposing using machine learning. </span><span class="cited-content_cbyCitation_journal-name">Current Opinion in Chemical Biology</span><span> <strong>2021,</strong> <em>65 </em>, 74-84. <a href="https://doi.org/10.1016/j.cbpa.2021.06.001" title="DOI URL">https://doi.org/10.1016/j.cbpa.2021.06.001</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.cbpa.2021.06.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.cbpa.2021.06.001%26sid%3Dliteratum%253Aachs%26jtitle%3DCurrent%2520Opinion%2520in%2520Chemical%2520Biology%26atitle%3DRecent%252Badvances%252Bin%252Bdrug%252Brepurposing%252Busing%252Bmachine%252Blearning%26aulast%3DUrbina%26aufirst%3DFabio%26date%3D2021%26volume%3D65%26spage%3D74%26epage%3D84" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Mateus Sá Magalhães  Serafim</span>, <span class="hlFld-ContribAuthor ">Valtair Severino  dos Santos Júnior</span>, <span class="hlFld-ContribAuthor ">Jadson Castro  Gertrudes</span>, <span class="hlFld-ContribAuthor ">Vinícius Gonçalves  Maltarollo</span>, <span class="hlFld-ContribAuthor ">Kathia Maria  Honorio</span>. </span><span class="cited-content_cbyCitation_article-title">Machine learning techniques applied to the drug design and discovery of new antivirals: a brief look over the past decade. </span><span class="cited-content_cbyCitation_journal-name">Expert Opinion on Drug Discovery</span><span> <strong>2021,</strong> <em>11 </em>, 1-15. <a href="https://doi.org/10.1080/17460441.2021.1918098" title="DOI URL">https://doi.org/10.1080/17460441.2021.1918098</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/17460441.2021.1918098&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F17460441.2021.1918098%26sid%3Dliteratum%253Aachs%26jtitle%3DExpert%2520Opinion%2520on%2520Drug%2520Discovery%26atitle%3DMachine%252Blearning%252Btechniques%252Bapplied%252Bto%252Bthe%252Bdrug%252Bdesign%252Band%252Bdiscovery%252Bof%252Bnew%252Bantivirals%25253A%252Ba%252Bbrief%252Blook%252Bover%252Bthe%252Bpast%252Bdecade%26aulast%3DSerafim%26aufirst%3DMateus%2BS%25C3%25A1%2BMagalh%25C3%25A3es%26date%3D2021%26date%3D2021%26volume%3D11%26spage%3D1%26epage%3D15" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Bo  Qiang</span>, <span class="hlFld-ContribAuthor ">Junyong  Lai</span>, <span class="hlFld-ContribAuthor ">Hongwei  Jin</span>, <span class="hlFld-ContribAuthor ">Liangren  Zhang</span>, <span class="hlFld-ContribAuthor ">Zhenming  Liu</span>. </span><span class="cited-content_cbyCitation_article-title">Target Prediction Model for Natural Products Using Transfer Learning. </span><span class="cited-content_cbyCitation_journal-name">International Journal of Molecular Sciences</span><span> <strong>2021,</strong> <em>22 </em>
                                    (9)
                                     , 4632. <a href="https://doi.org/10.3390/ijms22094632" title="DOI URL">https://doi.org/10.3390/ijms22094632</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/ijms22094632&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fijms22094632%26sid%3Dliteratum%253Aachs%26jtitle%3DInternational%2520Journal%2520of%2520Molecular%2520Sciences%26atitle%3DTarget%252BPrediction%252BModel%252Bfor%252BNatural%252BProducts%252BUsing%252BTransfer%252BLearning%26aulast%3DQiang%26aufirst%3DBo%26date%3D2021%26date%3D2021%26volume%3D22%26issue%3D9%26spage%3D4632" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Cong  Shen</span>, <span class="hlFld-ContribAuthor ">Jiawei  Luo</span>, <span class="hlFld-ContribAuthor ">Wenjue  Ouyang</span>, <span class="hlFld-ContribAuthor ">Pingjian  Ding</span>, <span class="hlFld-ContribAuthor ">Xiangtao  Chen</span>, . </span><span class="cited-content_cbyCitation_article-title">IDDkin: network-based influence deep diffusion model for enhancing prediction of kinase inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Bioinformatics</span><span> <strong>2021,</strong> <em>36 </em>
                                    (22-23)
                                     , 5481-5491. <a href="https://doi.org/10.1093/bioinformatics/btaa1058" title="DOI URL">https://doi.org/10.1093/bioinformatics/btaa1058</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1093/bioinformatics/btaa1058&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1093%2Fbioinformatics%2Fbtaa1058%26sid%3Dliteratum%253Aachs%26jtitle%3DBioinformatics%26atitle%3DIDDkin%25253A%252Bnetwork-based%252Binfluence%252Bdeep%252Bdiffusion%252Bmodel%252Bfor%252Benhancing%252Bprediction%252Bof%252Bkinase%252Binhibitors%26aulast%3DShen%26aufirst%3DCong%26date%3D2021%26date%3D2020%26volume%3D36%26issue%3D22-23%26spage%3D5481%26epage%3D5491" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Qianqian  Deng</span>, <span class="hlFld-ContribAuthor ">Jing  Hu</span>, <span class="hlFld-ContribAuthor ">Liying  Wang</span>, <span class="hlFld-ContribAuthor ">Yijing  Liu</span>, <span class="hlFld-ContribAuthor ">Yanzhi  Guo</span>, <span class="hlFld-ContribAuthor ">Tao  Xu</span>, <span class="hlFld-ContribAuthor ">Xuemei  Pu</span>. </span><span class="cited-content_cbyCitation_article-title">Probing impact of molecular structure on mechanical property and sensitivity of energetic materials by machine learning methods. </span><span class="cited-content_cbyCitation_journal-name">Chemometrics and Intelligent Laboratory Systems</span><span> <strong>2021,</strong> <em>151 </em>, 104331. <a href="https://doi.org/10.1016/j.chemolab.2021.104331" title="DOI URL">https://doi.org/10.1016/j.chemolab.2021.104331</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.chemolab.2021.104331&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.chemolab.2021.104331%26sid%3Dliteratum%253Aachs%26jtitle%3DChemometrics%2520and%2520Intelligent%2520Laboratory%2520Systems%26atitle%3DProbing%252Bimpact%252Bof%252Bmolecular%252Bstructure%252Bon%252Bmechanical%252Bproperty%252Band%252Bsensitivity%252Bof%252Benergetic%252Bmaterials%252Bby%252Bmachine%252Blearning%252Bmethods%26aulast%3DDeng%26aufirst%3DQianqian%26date%3D2021%26volume%3D151%26spage%3D104331" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Hannes  Whittingham</span>, <span class="hlFld-ContribAuthor ">Stephanie Kay  Ashenden</span>. </span><span class="cited-content_cbyCitation_article-title">Hit discovery. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2021,</strong>,, 81-102. <a href="https://doi.org/10.1016/B978-0-12-820045-2.00006-4" title="DOI URL">https://doi.org/10.1016/B978-0-12-820045-2.00006-4</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/B978-0-12-820045-2.00006-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2FB978-0-12-820045-2.00006-4%26sid%3Dliteratum%253Aachs%26atitle%3DHit%252Bdiscovery%26aulast%3DWhittingham%26aufirst%3DHannes%26date%3D2021%26spage%3D81%26epage%3D102%26pub%3DElsevier%26atitle%3DThe%252BEra%252Bof%252BArtificial%252BIntelligence%25252C%252BMachine%252BLearning%25252C%252Band%252BData%252BScience%252Bin%252Bthe%252BPharmaceutical%252BIndustry%26date%3D2021%26volume%3D162" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Christin  Neuber</span>, <span class="hlFld-ContribAuthor ">Alix  Tröster</span>, <span class="hlFld-ContribAuthor ">Reik  Löser</span>, <span class="hlFld-ContribAuthor ">Birgit  Belter</span>, <span class="hlFld-ContribAuthor ">Harald  Schwalbe</span>, <span class="hlFld-ContribAuthor ">Jens  Pietzsch</span>. </span><span class="cited-content_cbyCitation_article-title">The Pyrazolo[3,4-d]pyrimidine-Based Kinase Inhibitor NVP-BHG712: Effects of Regioisomers on Tumor Growth, Perfusion, and Hypoxia in EphB4-Positive A375 Melanoma Xenografts. </span><span class="cited-content_cbyCitation_journal-name">Molecules</span><span> <strong>2020,</strong> <em>25 </em>
                                    (21)
                                     , 5115. <a href="https://doi.org/10.3390/molecules25215115" title="DOI URL">https://doi.org/10.3390/molecules25215115</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/molecules25215115&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fmolecules25215115%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecules%26atitle%3DThe%252BPyrazolo%25255B3%25252C4-d%25255Dpyrimidine-Based%252BKinase%252BInhibitor%252BNVP-BHG712%25253A%252BEffects%252Bof%252BRegioisomers%252Bon%252BTumor%252BGrowth%25252C%252BPerfusion%25252C%252Band%252BHypoxia%252Bin%252BEphB4-Positive%252BA375%252BMelanoma%252BXenografts%26aulast%3DNeuber%26aufirst%3DChristin%26date%3D2020%26date%3D2020%26volume%3D25%26issue%3D21%26spage%3D5115" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Rajan  Chaudhari</span>, <span class="hlFld-ContribAuthor ">Long Wolf  Fong</span>, <span class="hlFld-ContribAuthor ">Zhi  Tan</span>, <span class="hlFld-ContribAuthor ">Beibei  Huang</span>, <span class="hlFld-ContribAuthor ">Shuxing  Zhang</span>. </span><span class="cited-content_cbyCitation_article-title">An up-to-date overview of computational polypharmacology in modern drug discovery. </span><span class="cited-content_cbyCitation_journal-name">Expert Opinion on Drug Discovery</span><span> <strong>2020,</strong> <em>15 </em>
                                    (9)
                                     , 1025-1044. <a href="https://doi.org/10.1080/17460441.2020.1767063" title="DOI URL">https://doi.org/10.1080/17460441.2020.1767063</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/17460441.2020.1767063&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F17460441.2020.1767063%26sid%3Dliteratum%253Aachs%26jtitle%3DExpert%2520Opinion%2520on%2520Drug%2520Discovery%26atitle%3DAn%252Bup-to-date%252Boverview%252Bof%252Bcomputational%252Bpolypharmacology%252Bin%252Bmodern%252Bdrug%252Bdiscovery%26aulast%3DChaudhari%26aufirst%3DRajan%26date%3D2020%26date%3D2020%26volume%3D15%26issue%3D9%26spage%3D1025%26epage%3D1044" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Filip  Miljković</span>, <span class="hlFld-ContribAuthor ">Jürgen  Bajorath</span>. </span><span class="cited-content_cbyCitation_article-title">Data structures for computational compound promiscuity analysis and exemplary applications to inhibitors of the human kinome. </span><span class="cited-content_cbyCitation_journal-name">Journal of Computer-Aided Molecular Design</span><span> <strong>2020,</strong> <em>34 </em>
                                    (1)
                                     , 1-10. <a href="https://doi.org/10.1007/s10822-019-00266-0" title="DOI URL">https://doi.org/10.1007/s10822-019-00266-0</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s10822-019-00266-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs10822-019-00266-0%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Computer-Aided%2520Molecular%2520Design%26atitle%3DData%252Bstructures%252Bfor%252Bcomputational%252Bcompound%252Bpromiscuity%252Banalysis%252Band%252Bexemplary%252Bapplications%252Bto%252Binhibitors%252Bof%252Bthe%252Bhuman%252Bkinome%26aulast%3DMiljkovi%25C4%2587%26aufirst%3DFilip%26date%3D2020%26date%3D2019%26volume%3D34%26issue%3D1%26spage%3D1%26epage%3D10" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Patroklos  Samaras</span>, <span class="hlFld-ContribAuthor ">Tobias  Schmidt</span>, <span class="hlFld-ContribAuthor ">Martin  Frejno</span>, <span class="hlFld-ContribAuthor ">Siegfried  Gessulat</span>, <span class="hlFld-ContribAuthor ">Maria  Reinecke</span>, <span class="hlFld-ContribAuthor ">Anna  Jarzab</span>, <span class="hlFld-ContribAuthor ">Jana  Zecha</span>, <span class="hlFld-ContribAuthor ">Julia  Mergner</span>, <span class="hlFld-ContribAuthor ">Piero  Giansanti</span>, <span class="hlFld-ContribAuthor ">Hans-Christian  Ehrlich</span>, <span class="hlFld-ContribAuthor ">Stephan  Aiche</span>, <span class="hlFld-ContribAuthor ">Johannes  Rank</span>, <span class="hlFld-ContribAuthor ">Harald  Kienegger</span>, <span class="hlFld-ContribAuthor ">Helmut  Krcmar</span>, <span class="hlFld-ContribAuthor ">Bernhard  Kuster</span>, <span class="hlFld-ContribAuthor ">Mathias  Wilhelm</span>. </span><span class="cited-content_cbyCitation_article-title">ProteomicsDB: a multi-omics and multi-organism resource for life science research. </span><span class="cited-content_cbyCitation_journal-name">Nucleic Acids Research</span><span> <strong>2019,</strong> <em>509 </em><a href="https://doi.org/10.1093/nar/gkz974" title="DOI URL">https://doi.org/10.1093/nar/gkz974</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1093/nar/gkz974&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1093%2Fnar%2Fgkz974%26sid%3Dliteratum%253Aachs%26jtitle%3DNucleic%2520Acids%2520Research%26atitle%3DProteomicsDB%25253A%252Ba%252Bmulti-omics%252Band%252Bmulti-organism%252Bresource%252Bfor%252Blife%252Bscience%252Bresearch%26aulast%3DSamaras%26aufirst%3DPatroklos%26date%3D2019%26date%3D2019%26volume%3D509" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-16/acs.jmedchem.9b00855/20200820/images/medium/jm9b00855_0010.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-16/acs.jmedchem.9b00855/20200820/images/large/jm9b00855_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00855&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-16/acs.jmedchem.9b00855/20200820/images/medium/jm9b00855_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-16/acs.jmedchem.9b00855/20200820/images/large/jm9b00855_0001.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Model performance results with different hyperparameters. The heat maps are colored according to the auROC of the testing data set, which is directly marked in each block. A darker color corresponds to better performance.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-16/acs.jmedchem.9b00855/20200820/images/large/jm9b00855_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00855&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-16/acs.jmedchem.9b00855/20200820/images/medium/jm9b00855_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-16/acs.jmedchem.9b00855/20200820/images/large/jm9b00855_0002.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Visualizing data sets using t-SNE. The ECFP4 values of the compounds were used as input. The distribution of compounds from the training data set (gray scatters) is compared to that from (A) the Davis data set, (B) the Anastassiadis data set, (C) PKIS1, and (D) PKIS2 (blue scatters).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-16/acs.jmedchem.9b00855/20200820/images/large/jm9b00855_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00855&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-16/acs.jmedchem.9b00855/20200820/images/medium/jm9b00855_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-16/acs.jmedchem.9b00855/20200820/images/large/jm9b00855_0003.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. AuROC on internal validation and external testing data sets of models with increasing numbers of compounds in the training data set: (A) random split validation; (B) 5-fold cluster cross-validation.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-16/acs.jmedchem.9b00855/20200820/images/large/jm9b00855_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00855&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-16/acs.jmedchem.9b00855/20200820/images/medium/jm9b00855_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-16/acs.jmedchem.9b00855/20200820/images/large/jm9b00855_0004.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Prediction performance comparison between MTDNN and RF of Merget et al. (A) Violin plots of auROCs for kinases, grouped by underlying external data sets and colored by method (RF in blue and MTDNN in red). The white dots are the average auROCs. The upper and lower end points of the black segments are the first and the third quartile, respectively. The <i>p</i>-values of the <i>t</i> tests are shown above the violin. (B) Bar plots of model performance on tasks with different amounts of bioactivity data in external data sets. A bar indicates the average auROC of tasks with the number of bioactivity data points within the underlying range: (∗) 0.01 < <i>p</i> < 0.05; (∗∗) 0.001 < <i>p</i> < 0.01; (∗∗∗) <i>p</i> < 0.001.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-16/acs.jmedchem.9b00855/20200820/images/large/jm9b00855_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00855&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-16/acs.jmedchem.9b00855/20200820/images/medium/jm9b00855_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-16/acs.jmedchem.9b00855/20200820/images/large/jm9b00855_0005.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Kinome maps of the predicted and experimental bioactivities. Kinases colored in red are experimental % activity of <50% at 1 μM or predicted active probability of >0.5 (left). Bar plots show predicted and experimental group selectivity. When a group has an odds ratio significantly greater than 1, the bar is colored red (right): (∗) 0.01 < <i>p</i> < 0.05; (∗∗) 0.001 < <i>p</i> < 0.01; (∗∗∗) <i>p</i> < 0.001.  Kinome maps were generated using the TREEspot software tool and reproduced with permission from KINOMEscan, a division of DiscoveRx Corporation, © DiscoveRx Corporation 2010.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-16/acs.jmedchem.9b00855/20200820/images/large/jm9b00855_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00855&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-16/acs.jmedchem.9b00855/20200820/images/medium/jm9b00855_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-16/acs.jmedchem.9b00855/20200820/images/large/jm9b00855_0006.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Scatter plot of correlation between the experimental and predicted overall selectivity. The <i>x</i>-axis and <i>y</i>-axis for a point are the standard selectivity scores calculated from predicted active probabilities (>50% as active) and reported <i>K</i><sub>d</sub> values (>1 μM as active), respectively.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-16/acs.jmedchem.9b00855/20200820/images/large/jm9b00855_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00855&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-16/acs.jmedchem.9b00855/20200820/images/medium/jm9b00855_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-16/acs.jmedchem.9b00855/20200820/images/large/jm9b00855_0007.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Odds ratio of 48 FDA-approved small-molecule kinase inhibitors for the group of NSCLC-related kinases. Red bars indicate the FDA-approved drugs for NSCLC. The bars with red stripes indicate that there have been reported evidence showing monotherapy efficacy against NSCLC for the drugs. Blue bars indicate that there have been reported evidence showing efficacy against NSCLC when combined with other drugs. The bars filled with blue stripes indicate drugs that, to the best of our knowledge, have not been reported for treating NSCLC. (∗) 0.01 < <i>p</i> < 0.05; (∗∗) 0.001 < <i>p</i> < 0.01; (∗∗∗) <i>p</i> < 0.001.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-16/acs.jmedchem.9b00855/20200820/images/large/jm9b00855_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00855&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig8" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-16/acs.jmedchem.9b00855/20200820/images/medium/jm9b00855_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-16/acs.jmedchem.9b00855/20200820/images/large/jm9b00855_0008.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Architecture of a multitask deep neural network.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-16/acs.jmedchem.9b00855/20200820/images/large/jm9b00855_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00855&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i32">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_57885" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_57885" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 76 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Manning, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whyte, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martinez, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hunter, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sudarsanam, S.</span></span> <span> </span><span class="NLM_article-title">The protein kinase complement of the human genome</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>298</i></span>,  <span class="NLM_fpage">1912</span>– <span class="NLM_lpage">1934</span>, <span class="refDoi"> DOI: 10.1126/science.1075762</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=10.1126%2Fscience.1075762" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=12471243" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=1%3ACAS%3A528%3ADC%252BD38Xpt1Wisb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=298&publication_year=2002&pages=1912-1934&author=G.+Manningauthor=D.+B.+Whyteauthor=R.+Martinezauthor=T.+Hunterauthor=S.+Sudarsanam&title=The+protein+kinase+complement+of+the+human+genome&doi=10.1126%2Fscience.1075762"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">The Protein Kinase Complement of the Human Genome</span></div><div class="casAuthors">Manning, G.; Whyte, D. B.; Martinez, R.; Hunter, T.; Sudarsanam, S.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">298</span>
        (<span class="NLM_cas:issue">5600</span>),
    <span class="NLM_cas:pages">1912-1916, 1933-1934</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">We have catalogued the protein kinase complement of the human genome (the "kinome") using public and proprietary genomic, complementary DNA, and expressed sequence tag (EST) sequences.  This provides a starting point for comprehensive anal. of protein phosphorylation in normal and disease states, as well as a detailed view of the current state of human genome anal. through a focus on one large gene family.  We identify 518 putative protein kinase genes, of which 71 have not previously been reported or described as kinases, and we extend or correct the protein sequences of 56 more kinases.  New genes include members of well-studied families as well as previously unidentified families, some of which are conserved in model organisms.  Classification and comparison with model organism kinomes identified orthologous groups and highlighted expansions specific to human and other lineages.  We also identified 106 protein kinase pseudogenes.  Chromosomal mapping revealed several small clusters of kinase genes and revealed that 244 kinases map to disease loci or cancer amplicons.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpTj1hf_0cQcrVg90H21EOLACvtfcHk0lh1obDhuRDV-w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38Xpt1Wisb0%253D&md5=b3def83bace52257a3252a3347b8bb92</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1126%2Fscience.1075762&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1075762%26sid%3Dliteratum%253Aachs%26aulast%3DManning%26aufirst%3DG.%26aulast%3DWhyte%26aufirst%3DD.%2BB.%26aulast%3DMartinez%26aufirst%3DR.%26aulast%3DHunter%26aufirst%3DT.%26aulast%3DSudarsanam%26aufirst%3DS.%26atitle%3DThe%2520protein%2520kinase%2520complement%2520of%2520the%2520human%2520genome%26jtitle%3DScience%26date%3D2002%26volume%3D298%26spage%3D1912%26epage%3D1934%26doi%3D10.1126%2Fscience.1075762" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wu, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nielsen, T. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clausen, M. H.</span></span> <span> </span><span class="NLM_article-title">FDA-approved small-molecule kinase inhibitors</span>. <i>Trends Pharmacol. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>36</i></span>,  <span class="NLM_fpage">422</span>– <span class="NLM_lpage">439</span>, <span class="refDoi"> DOI: 10.1016/j.tips.2015.04.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=10.1016%2Fj.tips.2015.04.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=25975227" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=1%3ACAS%3A528%3ADC%252BC2MXns1KisLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2015&pages=422-439&author=P.+Wuauthor=T.+E.+Nielsenauthor=M.+H.+Clausen&title=FDA-approved+small-molecule+kinase+inhibitors&doi=10.1016%2Fj.tips.2015.04.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">FDA-approved small-molecule kinase inhibitors</span></div><div class="casAuthors">Wu, Peng; Nielsen, Thomas E.; Clausen, Mads H.</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Pharmacological Sciences</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">422-439</span>CODEN:
                <span class="NLM_cas:coden">TPHSDY</span>;
        ISSN:<span class="NLM_cas:issn">0165-6147</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Kinases have emerged as one of the most intensively pursued targets in current pharmacol. research, esp. for cancer, due to their crit. roles in cellular signaling.  To date, the US FDA has approved 28 small-mol. kinase inhibitors, half of which were approved in the past 3 years.  While the clin. data of these approved mols. are widely presented and structure-activity relationship (SAR) has been reported for individual mols., an updated review that analyzes all approved mols. and summarizes current achievements and trends in the field has yet to be found.  Here we present all approved small-mol. kinase inhibitors with an emphasis on binding mechanism and structural features, summarize current challenges, and discuss future directions in this field.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrudyvM3eO09bVg90H21EOLACvtfcHk0lh1obDhuRDV-w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXns1KisLg%253D&md5=a6aa3dbb11e6fe3001fa5fa03a4c950a</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1016%2Fj.tips.2015.04.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tips.2015.04.005%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DP.%26aulast%3DNielsen%26aufirst%3DT.%2BE.%26aulast%3DClausen%26aufirst%3DM.%2BH.%26atitle%3DFDA-approved%2520small-molecule%2520kinase%2520inhibitors%26jtitle%3DTrends%2520Pharmacol.%2520Sci.%26date%3D2015%26volume%3D36%26spage%3D422%26epage%3D439%26doi%3D10.1016%2Fj.tips.2015.04.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fabbro, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowan-Jacob, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moebitz, H.</span></span> <span> </span><span class="NLM_article-title">Ten things you should know about protein kinases: IUPHAR Review 14</span>. <i>Br. J. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>172</i></span>,  <span class="NLM_fpage">2675</span>– <span class="NLM_lpage">2700</span>, <span class="refDoi"> DOI: 10.1111/bph.13096</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=10.1111%2Fbph.13096" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=25630872" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtVent7zP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=172&publication_year=2015&pages=2675-2700&author=D.+Fabbroauthor=S.+W.+Cowan-Jacobauthor=H.+Moebitz&title=Ten+things+you+should+know+about+protein+kinases%3A+IUPHAR+Review+14&doi=10.1111%2Fbph.13096"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Ten things you should know about protein kinases: IUPHAR Review 14</span></div><div class="casAuthors">Fabbro, Doriano; Cowan-Jacob, Sandra W.; Moebitz, Henrik</div><div class="citationInfo"><span class="NLM_cas:title">British Journal of Pharmacology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">172</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2675-2700</span>CODEN:
                <span class="NLM_cas:coden">BJPCBM</span>;
        ISSN:<span class="NLM_cas:issn">1476-5381</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">Many human malignancies are assocd. with aberrant regulation of protein or lipid kinases due to mutations, chromosomal rearrangements and/or gene amplification.  Protein and lipid kinases represent an important target class for treating human disorders.  This review focus on 'the 10 things you should know about protein kinases and their inhibitors', including a short introduction on the history of protein kinases and their inhibitors and ending with a perspective on kinase drug discovery.  Although the '10 things' have been, to a certain extent, chosen arbitrarily, they cover in a comprehensive way the past and present efforts in kinase drug discovery and summarize the status quo of the current kinase inhibitors as well as knowledge about kinase structure and binding modes.  Besides describing the potentials of protein kinase inhibitors as drugs, this review also focus on their limitations, particularly on how to circumvent emerging resistance against kinase inhibitors in oncol. indications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGryyzn-icN30rVg90H21EOLACvtfcHk0li03sz0mJJhcA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtVent7zP&md5=160b754efc7f29460712fa03bb2274b1</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1111%2Fbph.13096&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fbph.13096%26sid%3Dliteratum%253Aachs%26aulast%3DFabbro%26aufirst%3DD.%26aulast%3DCowan-Jacob%26aufirst%3DS.%2BW.%26aulast%3DMoebitz%26aufirst%3DH.%26atitle%3DTen%2520things%2520you%2520should%2520know%2520about%2520protein%2520kinases%253A%2520IUPHAR%2520Review%252014%26jtitle%3DBr.%2520J.%2520Pharmacol.%26date%3D2015%26volume%3D172%26spage%3D2675%26epage%3D2700%26doi%3D10.1111%2Fbph.13096" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Müller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span> <span> </span><span class="NLM_article-title">The (un)targeted cancer kinome</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">166</span>– <span class="NLM_lpage">169</span>, <span class="refDoi"> DOI: 10.1038/nchembio.297</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=10.1038%2Fnchembio.297" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=20154661" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhvFGntbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2010&pages=166-169&author=O.+Fedorovauthor=S.+M%C3%BCllerauthor=S.+Knapp&title=The+%28un%29targeted+cancer+kinome&doi=10.1038%2Fnchembio.297"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">The (un)targeted cancer kinome</span></div><div class="casAuthors">Fedorov, Oleg; Mueller, Susanne; Knapp, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">166-169</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review and discussion.  The complexity of cancer signaling and the resulting difficulties in target selection have strongly biased kinase drug discovery towards clin. validated targets.  Recently, novel kinase targets that are uncharacterized have emerged from genome sequencing and RNAi studies.  Chem. probes are urgently needed to functionally annotate these kinases and to stimulate new drug discovery efforts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpwcnq0-qjPirVg90H21EOLACvtfcHk0li03sz0mJJhcA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhvFGntbk%253D&md5=198ae8d39d40247e17aef5e9de0f2a99</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.297&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.297%26sid%3Dliteratum%253Aachs%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DM%25C3%25BCller%26aufirst%3DS.%26aulast%3DKnapp%26aufirst%3DS.%26atitle%3DThe%2520%2528un%2529targeted%2520cancer%2520kinome%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2010%26volume%3D6%26spage%3D166%26epage%3D169%26doi%3D10.1038%2Fnchembio.297" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Klaeger, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heinzlmeir, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilhelm, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Polzer, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vick, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koenig, P.-A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reinecke, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruprecht, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petzoldt, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meng, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zecha, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reiter, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qiao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Helm, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koch, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schoof, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Canevari, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Casale, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Depaolini, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feuchtinger, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmidt, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rueckert, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Becker, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huenges, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garz, A.-K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gohlke, B.-O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zolg, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kayser, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vooder, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Preissner, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hahne, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tonisson, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kramer, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gotze, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bassermann, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schlegl, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ehrlich, H.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aiche, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walch, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greif, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schneider, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Felder, E. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ruland, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Medard, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeremias, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spiekermann, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuster, B.</span></span> <span> </span><span class="NLM_article-title">The target landscape of clinical kinase drugs</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>358</i></span>, <span class="NLM_elocation-id">eaan4368</span> <span class="refDoi"> DOI: 10.1126/science.aan4368</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=10.1126%2Fscience.aan4368" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=29191878" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=358&publication_year=2017&author=S.+Klaegerauthor=S.+Heinzlmeirauthor=M.+Wilhelmauthor=H.+Polzerauthor=B.+Vickauthor=P.-A.+Koenigauthor=M.+Reineckeauthor=B.+Ruprechtauthor=S.+Petzoldtauthor=C.+Mengauthor=J.+Zechaauthor=K.+Reiterauthor=H.+Qiaoauthor=D.+Helmauthor=H.+Kochauthor=M.+Schoofauthor=G.+Canevariauthor=E.+Casaleauthor=S.+R.+Depaoliniauthor=A.+Feuchtingerauthor=Z.+Wuauthor=T.+Schmidtauthor=L.+Rueckertauthor=W.+Beckerauthor=J.+Huengesauthor=A.-K.+Garzauthor=B.-O.+Gohlkeauthor=D.+P.+Zolgauthor=G.+Kayserauthor=T.+Vooderauthor=R.+Preissnerauthor=H.+Hahneauthor=N.+Tonissonauthor=K.+Kramerauthor=K.+Gotzeauthor=F.+Bassermannauthor=J.+Schleglauthor=H.-C.+Ehrlichauthor=S.+Aicheauthor=A.+Walchauthor=P.+A.+Greifauthor=S.+Schneiderauthor=E.+R.+Felderauthor=J.+Rulandauthor=G.+Medardauthor=I.+Jeremiasauthor=K.+Spiekermannauthor=B.+Kuster&title=The+target+landscape+of+clinical+kinase+drugs&doi=10.1126%2Fscience.aan4368"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1126%2Fscience.aan4368&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.aan4368%26sid%3Dliteratum%253Aachs%26aulast%3DKlaeger%26aufirst%3DS.%26aulast%3DHeinzlmeir%26aufirst%3DS.%26aulast%3DWilhelm%26aufirst%3DM.%26aulast%3DPolzer%26aufirst%3DH.%26aulast%3DVick%26aufirst%3DB.%26aulast%3DKoenig%26aufirst%3DP.-A.%26aulast%3DReinecke%26aufirst%3DM.%26aulast%3DRuprecht%26aufirst%3DB.%26aulast%3DPetzoldt%26aufirst%3DS.%26aulast%3DMeng%26aufirst%3DC.%26aulast%3DZecha%26aufirst%3DJ.%26aulast%3DReiter%26aufirst%3DK.%26aulast%3DQiao%26aufirst%3DH.%26aulast%3DHelm%26aufirst%3DD.%26aulast%3DKoch%26aufirst%3DH.%26aulast%3DSchoof%26aufirst%3DM.%26aulast%3DCanevari%26aufirst%3DG.%26aulast%3DCasale%26aufirst%3DE.%26aulast%3DDepaolini%26aufirst%3DS.%2BR.%26aulast%3DFeuchtinger%26aufirst%3DA.%26aulast%3DWu%26aufirst%3DZ.%26aulast%3DSchmidt%26aufirst%3DT.%26aulast%3DRueckert%26aufirst%3DL.%26aulast%3DBecker%26aufirst%3DW.%26aulast%3DHuenges%26aufirst%3DJ.%26aulast%3DGarz%26aufirst%3DA.-K.%26aulast%3DGohlke%26aufirst%3DB.-O.%26aulast%3DZolg%26aufirst%3DD.%2BP.%26aulast%3DKayser%26aufirst%3DG.%26aulast%3DVooder%26aufirst%3DT.%26aulast%3DPreissner%26aufirst%3DR.%26aulast%3DHahne%26aufirst%3DH.%26aulast%3DTonisson%26aufirst%3DN.%26aulast%3DKramer%26aufirst%3DK.%26aulast%3DGotze%26aufirst%3DK.%26aulast%3DBassermann%26aufirst%3DF.%26aulast%3DSchlegl%26aufirst%3DJ.%26aulast%3DEhrlich%26aufirst%3DH.-C.%26aulast%3DAiche%26aufirst%3DS.%26aulast%3DWalch%26aufirst%3DA.%26aulast%3DGreif%26aufirst%3DP.%2BA.%26aulast%3DSchneider%26aufirst%3DS.%26aulast%3DFelder%26aufirst%3DE.%2BR.%26aulast%3DRuland%26aufirst%3DJ.%26aulast%3DMedard%26aufirst%3DG.%26aulast%3DJeremias%26aufirst%3DI.%26aulast%3DSpiekermann%26aufirst%3DK.%26aulast%3DKuster%26aufirst%3DB.%26atitle%3DThe%2520target%2520landscape%2520of%2520clinical%2520kinase%2520drugs%26jtitle%3DScience%26date%3D2017%26volume%3D358%26doi%3D10.1126%2Fscience.aan4368" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Knight, Z. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shokat, K. M.</span></span> <span> </span><span class="NLM_article-title">Targeting the cancer kinome through polypharmacology</span>. <i>Nat. Rev. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">130</span>, <span class="refDoi"> DOI: 10.1038/nrc2787</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=10.1038%2Fnrc2787" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=20094047" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=1%3ACAS%3A528%3ADC%252BC3cXps1Clug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2010&pages=130&author=Z.+A.+Knightauthor=H.+Linauthor=K.+M.+Shokat&title=Targeting+the+cancer+kinome+through+polypharmacology&doi=10.1038%2Fnrc2787"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting the cancer kinome through polypharmacology</span></div><div class="casAuthors">Knight, Zachary A.; Lin, Henry; Shokat, Kevan M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">130-137</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Kinase inhibitors are the largest class of new cancer drugs.  However, it is already apparent that most tumors can escape from the inhibition of any single kinase.  If it is necessary to inhibit multiple kinases, how do we choose which ones.  In this Opinion article, we discuss some of the strategies that are currently being used to identify new therapeutic combinations of kinase targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpkDv2FeeH58rVg90H21EOLACvtfcHk0lh-KGCPuBjDig"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXps1Clug%253D%253D&md5=9aaa314e54b43889e147a9f11ac77072</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1038%2Fnrc2787&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc2787%26sid%3Dliteratum%253Aachs%26aulast%3DKnight%26aufirst%3DZ.%2BA.%26aulast%3DLin%26aufirst%3DH.%26aulast%3DShokat%26aufirst%3DK.%2BM.%26atitle%3DTargeting%2520the%2520cancer%2520kinome%2520through%2520polypharmacology%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2010%26volume%3D10%26spage%3D130%26doi%3D10.1038%2Fnrc2787" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Goldstein, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zarrinkar, P. P.</span></span> <span> </span><span class="NLM_article-title">High-throughput kinase profiling as a platform for drug discovery</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">391</span>, <span class="refDoi"> DOI: 10.1038/nrd2541</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=10.1038%2Fnrd2541" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=18404149" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=1%3ACAS%3A528%3ADC%252BD1cXlt1agtbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2008&pages=391&author=D.+M.+Goldsteinauthor=N.+S.+Grayauthor=P.+P.+Zarrinkar&title=High-throughput+kinase+profiling+as+a+platform+for+drug+discovery&doi=10.1038%2Fnrd2541"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">High-throughput kinase profiling as a platform for drug discovery</span></div><div class="casAuthors">Goldstein, David M.; Gray, Nathanael S.; Zarrinkar, Patrick P.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">391-397</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  High-throughput profiling of compd. libraries against large panels of kinases is becoming tech. feasible.  In contrast to the traditional linear, target-centric approach to discovery, this approach may provide a choice of targets to pursue that is guided by the quality of lead compds. available, rather than by target biol. alone, and could thereby significantly improve the productivity of kinase inhibitor discovery.  To fully exploit the potential of kinases as drug targets, novel strategies for the efficient discovery of inhibitors are required.  In contrast to the traditional, linear process of inhibitor discovery, high-throughput kinase profiling enables a parallel approach by interrogating compds. against hundreds of targets in a single screen.  Compd. potency and selectivity are detd. simultaneously, providing a choice of targets to pursue that is guided by the quality of lead compds. available, rather than by target biol. alone.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpwnJ59cVxbJbVg90H21EOLACvtfcHk0limzAImiiLoLA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXlt1agtbs%253D&md5=eaa61fd2958ff04fecf86a993f91833d</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1038%2Fnrd2541&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd2541%26sid%3Dliteratum%253Aachs%26aulast%3DGoldstein%26aufirst%3DD.%2BM.%26aulast%3DGray%26aufirst%3DN.%2BS.%26aulast%3DZarrinkar%26aufirst%3DP.%2BP.%26atitle%3DHigh-throughput%2520kinase%2520profiling%2520as%2520a%2520platform%2520for%2520drug%2520discovery%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2008%26volume%3D7%26spage%3D391%26doi%3D10.1038%2Fnrd2541" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Niijima, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shiraishi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Okuno, Y.</span></span> <span> </span><span class="NLM_article-title">Dissecting kinase profiling data to predict activity and understand cross-reactivity of kinase inhibitors</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>52</i></span>,  <span class="NLM_fpage">901</span>– <span class="NLM_lpage">912</span>, <span class="refDoi"> DOI: 10.1021/ci200607f</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci200607f" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=1%3ACAS%3A528%3ADC%252BC38Xjs1Clsbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2012&pages=901-912&author=S.+Niijimaauthor=A.+Shiraishiauthor=Y.+Okuno&title=Dissecting+kinase+profiling+data+to+predict+activity+and+understand+cross-reactivity+of+kinase+inhibitors&doi=10.1021%2Fci200607f"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Dissecting Kinase Profiling Data to Predict Activity and Understand Cross-Reactivity of Kinase Inhibitors</span></div><div class="casAuthors">Niijima, Satoshi; Shiraishi, Akira; Okuno, Yasushi</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">901-912</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The development of selective and multitargeted kinase inhibitors has received much attention, because cross-reactivity with unintended targets may cause toxic side effects, while it can also give rise to efficacious multitargeted drugs.  Here we describe a deconvolution approach to dissecting kinase profiling data in order to gain knowledge about cross-reactivity of inhibitors from large-scale profiling data.  This approach not only enables activity predictions of given compds. on a kinome-wide scale, but also allows to ext. residue-fragment pairs that are assocd. with activity.  We demonstrate its effectiveness using a large-scale public chemogenomics data set and also apply our proposed model to a recently published bioactivity data set.  We further illustrate that the preference of given compds. for kinases of interest is better understood by residue-fragment pairs, which could provide both biol. and chem. insights into cross-reactivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpNgeRgAJen8LVg90H21EOLACvtfcHk0limzAImiiLoLA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xjs1Clsbo%253D&md5=5b3d8d0938f8109e21d17a3fb3c9da30</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1021%2Fci200607f&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci200607f%26sid%3Dliteratum%253Aachs%26aulast%3DNiijima%26aufirst%3DS.%26aulast%3DShiraishi%26aufirst%3DA.%26aulast%3DOkuno%26aufirst%3DY.%26atitle%3DDissecting%2520kinase%2520profiling%2520data%2520to%2520predict%2520activity%2520and%2520understand%2520cross-reactivity%2520of%2520kinase%2520inhibitors%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2012%26volume%3D52%26spage%3D901%26epage%3D912%26doi%3D10.1021%2Fci200607f" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lapins, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wikberg, J. E.</span></span> <span> </span><span class="NLM_article-title">Kinome-wide interaction modelling using alignment-based and alignment-independent approaches for kinase description and linear and non-linear data analysis techniques</span>. <i>BMC Bioinf.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">339</span>– <span class="NLM_lpage">339</span>, <span class="refDoi"> DOI: 10.1186/1471-2105-11-339</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=10.1186%2F1471-2105-11-339" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=20569422" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=1%3ACAS%3A280%3ADC%252BC3cjgtFekug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2010&pages=339-339&author=M.+Lapinsauthor=J.+E.+Wikberg&title=Kinome-wide+interaction+modelling+using+alignment-based+and+alignment-independent+approaches+for+kinase+description+and+linear+and+non-linear+data+analysis+techniques&doi=10.1186%2F1471-2105-11-339"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Kinome-wide interaction modelling using alignment-based and alignment-independent approaches for kinase description and linear and non-linear data analysis techniques</span></div><div class="casAuthors">Lapins Maris; Wikberg Jarl Es</div><div class="citationInfo"><span class="NLM_cas:title">BMC bioinformatics</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">339</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  Protein kinases play crucial roles in cell growth, differentiation, and apoptosis.  Abnormal function of protein kinases can lead to many serious diseases, such as cancer.  Kinase inhibitors have potential for treatment of these diseases.  However, current inhibitors interact with a broad variety of kinases and interfere with multiple vital cellular processes, which causes toxic effects.  Bioinformatics approaches that can predict inhibitor-kinase interactions from the chemical properties of the inhibitors and the kinase macromolecules might aid in design of more selective therapeutic agents, that show better efficacy and lower toxicity.  RESULTS:  We applied proteochemometric modelling to correlate the properties of 317 wild-type and mutated kinases and 38 inhibitors (12,046 inhibitor-kinase combinations) to the respective combination's interaction dissociation constant (Kd).  We compared six approaches for description of protein kinases and several linear and non-linear correlation methods.  The best performing models encoded kinase sequences with amino acid physico-chemical z-scale descriptors and used support vector machines or partial least- squares projections to latent structures for the correlations.  Modelling performance was estimated by double cross-validation.  The best models showed high predictive ability; the squared correlation coefficient for new kinase-inhibitor pairs ranging P2 = 0.67-0.73; for new kinases it ranged P2kin = 0.65-0.70.  Models could also separate interacting from non-interacting inhibitor-kinase pairs with high sensitivity and specificity; the areas under the ROC curves ranging AUC = 0.92-0.93.  We also investigated the relationship between the number of protein kinases in the dataset and the modelling results.  Using only 10% of all data still a valid model was obtained with P2 = 0.47, P2kin = 0.42 and AUC = 0.83.  CONCLUSIONS:  Our results strongly support the applicability of proteochemometrics for kinome-wide interaction modelling.  Proteochemometrics might be used to speed-up identification and optimization of protein kinase targeted and multi-targeted inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSQqQwOlHM3l0EksK8CslK8fW6udTcc2ebA4aE6XtuHgbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3cjgtFekug%253D%253D&md5=5bde7177dce4f3c5e0c26a89c7fc4d03</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1186%2F1471-2105-11-339&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1471-2105-11-339%26sid%3Dliteratum%253Aachs%26aulast%3DLapins%26aufirst%3DM.%26aulast%3DWikberg%26aufirst%3DJ.%2BE.%26atitle%3DKinome-wide%2520interaction%2520modelling%2520using%2520alignment-based%2520and%2520alignment-independent%2520approaches%2520for%2520kinase%2520description%2520and%2520linear%2520and%2520non-linear%2520data%2520analysis%2520techniques%26jtitle%3DBMC%2520Bioinf.%26date%3D2010%26volume%3D11%26spage%3D339%26epage%3D339%26doi%3D10.1186%2F1471-2105-11-339" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schurer, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muskal, S. M.</span></span> <span> </span><span class="NLM_article-title">Kinome-wide activity modeling from diverse public high-quality data sets</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">27</span>– <span class="NLM_lpage">38</span>, <span class="refDoi"> DOI: 10.1021/ci300403k</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci300403k" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvVygu77L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2013&pages=27-38&author=S.+C.+Schurerauthor=S.+M.+Muskal&title=Kinome-wide+activity+modeling+from+diverse+public+high-quality+data+sets&doi=10.1021%2Fci300403k"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Kinome-wide Activity Modeling from Diverse Public High-Quality Data Sets</span></div><div class="casAuthors">Schurer, Stephan C.; Muskal, Steven M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">27-38</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Large corpora of kinase small mol. inhibitor data are accessible to public sector research from thousands of journal article and patent publications.  These data have been generated employing a wide variety of assay methodologies and exptl. procedures by numerous labs.  Here we ask the question how applicable these heterogeneous data sets are to predict kinase activities and which characteristics of the data sets contribute to their utility.  We accessed almost 500 000 mols. from the Kinase Knowledge Base (KKB) and after rigorous aggregation and standardization generated over 180 distinct data sets covering all major groups of the human kinome.  To assess the value of the data sets, we generated hundreds of classification and regression models.  Their rigorous cross-validation and characterization demonstrated highly predictive classification and quant. models for the majority of kinase targets if a min. required no. of active compds. or structure-activity data points were available.  We then applied the best classifiers to compds. most recently profiled in the NIH Library of Integrated Network-based Cellular Signatures (LINCS) program and found good agreement of profiling results with predicted activities.  Our results indicate that, although heterogeneous in nature, the publically accessible data sets are exceedingly valuable and well suited to develop highly accurate predictors for practical Kinome-wide virtual screening applications and to complement exptl. kinase profiling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr1rhwNqRlmD7Vg90H21EOLACvtfcHk0lgZqBL27zvbHA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvVygu77L&md5=cea0a7b0c22b43e7660b430e8039f4e1</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1021%2Fci300403k&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci300403k%26sid%3Dliteratum%253Aachs%26aulast%3DSchurer%26aufirst%3DS.%2BC.%26aulast%3DMuskal%26aufirst%3DS.%2BM.%26atitle%3DKinome-wide%2520activity%2520modeling%2520from%2520diverse%2520public%2520high-quality%2520data%2520sets%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2013%26volume%3D53%26spage%3D27%26epage%3D38%26doi%3D10.1021%2Fci300403k" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cao, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, G. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Q. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, Y. Z.</span></span> <span> </span><span class="NLM_article-title">Large-scale prediction of human kinase–inhibitor interactions using protein sequences and molecular topological structures</span>. <i>Anal. Chim. Acta</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>792</i></span>,  <span class="NLM_fpage">10</span>– <span class="NLM_lpage">18</span>, <span class="refDoi"> DOI: 10.1016/j.aca.2013.07.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=10.1016%2Fj.aca.2013.07.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=23910962" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtFOhsrjF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=792&publication_year=2013&pages=10-18&author=D.+S.+Caoauthor=G.+H.+Zhouauthor=S.+Liuauthor=L.+X.+Zhangauthor=Q.+S.+Xuauthor=M.+Heauthor=Y.+Z.+Liang&title=Large-scale+prediction+of+human+kinase%E2%80%93inhibitor+interactions+using+protein+sequences+and+molecular+topological+structures&doi=10.1016%2Fj.aca.2013.07.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Large-scale prediction of human kinase-inhibitor interactions using protein sequences and molecular topological structures</span></div><div class="casAuthors">Cao, Dong-Sheng; Zhou, Guang-Hua; Liu, Shao; Zhang, Liu-Xia; Xu, Qing-Song; He, Min; Liang, Yi-Zeng</div><div class="citationInfo"><span class="NLM_cas:title">Analytica Chimica Acta</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">792</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">10-18</span>CODEN:
                <span class="NLM_cas:coden">ACACAM</span>;
        ISSN:<span class="NLM_cas:issn">0003-2670</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The kinase family is one of the largest target families in the human genome.  The family's key function in signal transduction for all organisms makes it a very attractive target class for the therapeutic interventions in many diseases states such as cancer, diabetes, inflammation and arthritis.  A first step toward accelerating kinase drug discovery process is to fast identify whether a chem. and a kinase interact or not.  Exptl., these interactions can be identified by in vitro binding assay - an expensive and laborious procedure that is not applicable on a large scale.  Therefore, there is an urgent need to develop statistically efficient approaches for identifying kinase-inhibitor interactions.  For the first time, the quant. binding affinities of kinase-inhibitor pairs are differentiated as a measurement to define if an inhibitor interacts with a kinase, and then a chemogenomics framework using an unbiased set of general integrated features (drug descriptors and protein descriptors) and random forest (RF) is employed to construct a predictive model which can accurately classify kinase-inhibitor pairs.  Our results show that RF with integrated features gave prediction accuracy of 93.76%, sensitivity of 92.26%, and specificity of 95.27%, resp.  The results are superior to those by only considering two sepd. spaces (chem. space and protein space), demonstrating that these integrated features contribute cooperatively.  Based on the constructed model, we provided a high confidence list of drug-target assocns. for subsequent exptl. investigation guidance at a low false discovery rate.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqSkjxlh2x_9bVg90H21EOLACvtfcHk0lgZqBL27zvbHA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtFOhsrjF&md5=d43c3262b9f166729bcfc418cc8bc3be</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1016%2Fj.aca.2013.07.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.aca.2013.07.003%26sid%3Dliteratum%253Aachs%26aulast%3DCao%26aufirst%3DD.%2BS.%26aulast%3DZhou%26aufirst%3DG.%2BH.%26aulast%3DLiu%26aufirst%3DS.%26aulast%3DZhang%26aufirst%3DL.%2BX.%26aulast%3DXu%26aufirst%3DQ.%2BS.%26aulast%3DHe%26aufirst%3DM.%26aulast%3DLiang%26aufirst%3DY.%2BZ.%26atitle%3DLarge-scale%2520prediction%2520of%2520human%2520kinase%25E2%2580%2593inhibitor%2520interactions%2520using%2520protein%2520sequences%2520and%2520molecular%2520topological%2520structures%26jtitle%3DAnal.%2520Chim.%2520Acta%26date%3D2013%26volume%3D792%26spage%3D10%26epage%3D18%26doi%3D10.1016%2Fj.aca.2013.07.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Merget, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turk, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eid, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rippmann, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fulle, S.</span></span> <span> </span><span class="NLM_article-title">Profiling prediction of kinase inhibitors: toward the virtual assay</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">474</span>– <span class="NLM_lpage">485</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b01611</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01611" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvFWgtLjM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=474-485&author=B.+Mergetauthor=S.+Turkauthor=S.+Eidauthor=F.+Rippmannauthor=S.+Fulle&title=Profiling+prediction+of+kinase+inhibitors%3A+toward+the+virtual+assay&doi=10.1021%2Facs.jmedchem.6b01611"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Profiling Prediction of Kinase Inhibitors: Toward the Virtual Assay</span></div><div class="casAuthors">Merget, Benjamin; Turk, Samo; Eid, Sameh; Rippmann, Friedrich; Fulle, Simone</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">474-485</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Kinome-wide screening would have the advantage of providing structure-activity relationships against hundreds of targets simultaneously.  Here, the authors report the generation of ligand-based activity prediction models for over 280 kinases by employing Machine Learning methods on an extensive data set of proprietary bioactivity data combined with open data.  High quality (AUC >0.7) was achieved for ∼200 kinases by combining open with proprietary data, choosing Random Forest over alternative tested Machine Learning methods, and balancing the training data sets.  Tests on left-out and external data indicate a high value for virtual screening projects.  Importantly, the derived models are evenly distributed across the kinome tree, allowing reliable profiling prediction for all kinase branches.  The prediction quality was further improved by employing exptl. bioactivity fingerprints of a small kinase subset.  Overall, the generated models can support various hit identification tasks, including virtual screening, compd. repurposing and the detection of potential off-targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoPeLeyo4D6IrVg90H21EOLACvtfcHk0lgWzbw4_77USQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvFWgtLjM&md5=2e0ad30702bcf11bed7cb3ec30c774c3</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01611&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01611%26sid%3Dliteratum%253Aachs%26aulast%3DMerget%26aufirst%3DB.%26aulast%3DTurk%26aufirst%3DS.%26aulast%3DEid%26aufirst%3DS.%26aulast%3DRippmann%26aufirst%3DF.%26aulast%3DFulle%26aufirst%3DS.%26atitle%3DProfiling%2520prediction%2520of%2520kinase%2520inhibitors%253A%2520toward%2520the%2520virtual%2520assay%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D474%26epage%3D485%26doi%3D10.1021%2Facs.jmedchem.6b01611" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bora, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Avram, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciucanu, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raica, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Avram, S.</span></span> <span> </span><span class="NLM_article-title">Predictive models for fast and effective profiling of kinase inhibitors</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">895</span>– <span class="NLM_lpage">905</span>, <span class="refDoi"> DOI: 10.1021/acs.jcim.5b00646</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.5b00646" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=1%3ACAS%3A528%3ADC%252BC28Xlslyqtr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2016&pages=895-905&author=A.+Boraauthor=S.+Avramauthor=I.+Ciucanuauthor=M.+Raicaauthor=S.+Avram&title=Predictive+models+for+fast+and+effective+profiling+of+kinase+inhibitors&doi=10.1021%2Facs.jcim.5b00646"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Predictive Models for Fast and Effective Profiling of Kinase Inhibitors</span></div><div class="casAuthors">Bora, Alina; Avram, Sorin; Ciucanu, Ionel; Raica, Marius; Avram, Stefana</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">895-905</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">In this study we developed two-dimensional pharmacophore-based random forest models for the effective profiling of kinase inhibitors.  One hundred seven prediction models were developed to address distinct kinases spanning over all kinase groups.  Rigorous external validation demonstrates excellent virtual screening and classification potential of the predictors and, more importantly, the capacity to prioritize novel chem. scaffolds in large chem. libraries.  The models built upon more diverse and more potent compds. tend to exert the highest predictive power.  The anal. of ColBioS-FlavRC (Collection of Bioselective Flavonoids and Related Compds.) highlighted several potentially promiscuous derivs. with undesirable selectivity against kinases.  The prediction models can be downloaded from www.chembioinf.ro.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqTZSs1GEkYHLVg90H21EOLACvtfcHk0lgWzbw4_77USQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xlslyqtr4%253D&md5=cd8d17ea9616165b115dd9d75352e0ac</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.5b00646&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.5b00646%26sid%3Dliteratum%253Aachs%26aulast%3DBora%26aufirst%3DA.%26aulast%3DAvram%26aufirst%3DS.%26aulast%3DCiucanu%26aufirst%3DI.%26aulast%3DRaica%26aufirst%3DM.%26aulast%3DAvram%26aufirst%3DS.%26atitle%3DPredictive%2520models%2520for%2520fast%2520and%2520effective%2520profiling%2520of%2520kinase%2520inhibitors%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2016%26volume%3D56%26spage%3D895%26epage%3D905%26doi%3D10.1021%2Facs.jcim.5b00646" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yabuuchi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Niijima, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takematsu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ida, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hirokawa, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hara, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ogawa, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minowa, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsujimoto, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Okuno, Y.</span></span> <span> </span><span class="NLM_article-title">Analysis of multiple compound–protein interactions reveals novel bioactive molecules</span>. <i>Mol. Syst. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">472</span>, <span class="refDoi"> DOI: 10.1038/msb.2011.5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=10.1038%2Fmsb.2011.5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=21364574" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=1%3ACAS%3A528%3ADC%252BC3MXltFShsbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2011&pages=472&author=H.+Yabuuchiauthor=S.+Niijimaauthor=H.+Takematsuauthor=T.+Idaauthor=T.+Hirokawaauthor=T.+Haraauthor=T.+Ogawaauthor=Y.+Minowaauthor=G.+Tsujimotoauthor=Y.+Okuno&title=Analysis+of+multiple+compound%E2%80%93protein+interactions+reveals+novel+bioactive+molecules&doi=10.1038%2Fmsb.2011.5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Analysis of multiple compound-protein interactions reveals novel bioactive molecules</span></div><div class="casAuthors">Yabuuchi, Hiroaki; Niijima, Satoshi; Takematsu, Hiromu; Ida, Tomomi; Hirokawa, Takatsugu; Hara, Takafumi; Ogawa, Teppei; Minowa, Yohsuke; Tsujimoto, Gozoh; Okuno, Yasushi</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Systems Biology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">472</span>CODEN:
                <span class="NLM_cas:coden">MSBOC3</span>;
        ISSN:<span class="NLM_cas:issn">1744-4292</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The discovery of novel bioactive mols. advances our systems-level understanding of biol. processes and is crucial for innovation in drug development.  For this purpose, the emerging field of chem. genomics is currently focused on accumulating large assay data sets describing compd.-protein interactions (CPIs).  Although new target proteins for known drugs have recently been identified through mining of CPI databases, using these resources to identify novel ligands remains unexplored.  Herein, we demonstrate that machine learning of multiple CPIs can not only assess drug polypharmacol. but can also efficiently identify novel bioactive scaffold-hopping compds.  Through a machine-learning technique that uses multiple CPIs, we have successfully identified novel lead compds. for two pharmaceutically important protein families, G-protein coupled receptors and protein kinases.  These novel compds. were not identified by existing computational ligand-screening methods in comparative studies.  The results of this study indicate that data derived from chem. genomics can be highly useful for exploring chem. space, and this systems biol. perspective could accelerate drug discovery processes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqh8pIgfxm7oLVg90H21EOLACvtfcHk0liDzfMoR4U6Sg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXltFShsbY%253D&md5=bc0caba6894d46de383042fa85accefa</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1038%2Fmsb.2011.5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fmsb.2011.5%26sid%3Dliteratum%253Aachs%26aulast%3DYabuuchi%26aufirst%3DH.%26aulast%3DNiijima%26aufirst%3DS.%26aulast%3DTakematsu%26aufirst%3DH.%26aulast%3DIda%26aufirst%3DT.%26aulast%3DHirokawa%26aufirst%3DT.%26aulast%3DHara%26aufirst%3DT.%26aulast%3DOgawa%26aufirst%3DT.%26aulast%3DMinowa%26aufirst%3DY.%26aulast%3DTsujimoto%26aufirst%3DG.%26aulast%3DOkuno%26aufirst%3DY.%26atitle%3DAnalysis%2520of%2520multiple%2520compound%25E2%2580%2593protein%2520interactions%2520reveals%2520novel%2520bioactive%2520molecules%26jtitle%3DMol.%2520Syst.%2520Biol.%26date%3D2011%26volume%3D7%26spage%3D472%26doi%3D10.1038%2Fmsb.2011.5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Manallack, D. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pitt, W. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gancia, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Montana, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Livingstone, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ford, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whitley, D. C.</span></span> <span> </span><span class="NLM_article-title">Selecting screening candidates for kinase and G protein-coupled receptor targets using neural networks</span>. <i>J. Chem. Inf. Comput. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>42</i></span>,  <span class="NLM_fpage">1256</span>– <span class="NLM_lpage">1262</span>, <span class="refDoi"> DOI: 10.1021/ci020267c</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci020267c" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=1%3ACAS%3A528%3ADC%252BD38XlvFSmtrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2002&pages=1256-1262&author=D.+T.+Manallackauthor=W.+R.+Pittauthor=E.+Ganciaauthor=J.+G.+Montanaauthor=D.+J.+Livingstoneauthor=M.+G.+Fordauthor=D.+C.+Whitley&title=Selecting+screening+candidates+for+kinase+and+G+protein-coupled+receptor+targets+using+neural+networks&doi=10.1021%2Fci020267c"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Selecting Screening Candidates for Kinase and G Protein-Coupled Receptor Targets Using Neural Networks</span></div><div class="casAuthors">Manallack, David T.; Pitt, Will R.; Gancia, Emanuela; Montana, John G.; Livingstone, David J.; Ford, Martyn G.; Whitley, David C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Computer Sciences</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1256-1262</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">0095-2338</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A series of neural networks has been trained, using consensus methods, to recognize compds. that act at biol. targets belonging to specific gene families.  The MDDR database was used to provide compds. targeted against gene families and sets of randomly selected mols.  BCUT parameters were employed as input descriptors that encode structural properties and information relevant to ligand-receptor interactions.  In each case, the networks identified over 80% of the compds. targeting a gene family.  The technique was applied to purchasing compds. from external suppliers, and results from screening against one gene family demonstrated impressive abilities to predict the activity of the majority of known hit compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp3B0ertK2pfbVg90H21EOLACvtfcHk0liDzfMoR4U6Sg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XlvFSmtrg%253D&md5=264cc5d5bc3657018ca0bd232954765c</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1021%2Fci020267c&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci020267c%26sid%3Dliteratum%253Aachs%26aulast%3DManallack%26aufirst%3DD.%2BT.%26aulast%3DPitt%26aufirst%3DW.%2BR.%26aulast%3DGancia%26aufirst%3DE.%26aulast%3DMontana%26aufirst%3DJ.%2BG.%26aulast%3DLivingstone%26aufirst%3DD.%2BJ.%26aulast%3DFord%26aufirst%3DM.%2BG.%26aulast%3DWhitley%26aufirst%3DD.%2BC.%26atitle%3DSelecting%2520screening%2520candidates%2520for%2520kinase%2520and%2520G%2520protein-coupled%2520receptor%2520targets%2520using%2520neural%2520networks%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Comput.%2520Sci.%26date%3D2002%26volume%3D42%26spage%3D1256%26epage%3D1262%26doi%3D10.1021%2Fci020267c" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Subramanian, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramsundar, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pande, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Denny, R. A.</span></span> <span> </span><span class="NLM_article-title">Computational modeling of β-secretase 1 (BACE-1) inhibitors using ligand based approaches</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">1936</span>– <span class="NLM_lpage">1949</span>, <span class="refDoi"> DOI: 10.1021/acs.jcim.6b00290</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.6b00290" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsF2nu73E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2016&pages=1936-1949&author=G.+Subramanianauthor=B.+Ramsundarauthor=V.+Pandeauthor=R.+A.+Denny&title=Computational+modeling+of+%CE%B2-secretase+1+%28BACE-1%29+inhibitors+using+ligand+based+approaches&doi=10.1021%2Facs.jcim.6b00290"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Computational Modeling of β-Secretase 1 (BACE-1) Inhibitors Using Ligand Based Approaches</span></div><div class="casAuthors">Subramanian, Govindan; Ramsundar, Bharath; Pande, Vijay; Denny, Rajiah Aldrin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1936-1949</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The binding affinities (IC50) reported for diverse structural classes of human β-secretase 1 (BACE-1) inhibitors in literature were modeled using multiple in silico ligand based modeling approaches and statistical techniques.  The descriptor space encompasses simple binary mol. fingerprint, 1- and 2-dimensional physicochem. and topol. descriptors, and sophisticated 3-dimensional mol. fields that require appropriate structural alignments of varied chem. scaffolds in one universal chem. space.  The affinities were modeled using qual. classification or quant. regression schemes involving linear, nonlinear, and deep neural network (DNN) machine-learning methods used in the scientific literature for quant. structure activity relationships (QSAR).  In a departure from tradition, ∼20% of the chem. diverse dataset (205 compds.) was used to train the model with the remaining ∼80% of the structural and chem. analogs used as part of an external validation (1273 compds.) and prospective test (69 compds.) sets resp. to ascertain the model performance.  The machine learning methods investigated herein performed well in both the qual. classification (∼70% accuracy) and quant. IC50 predictions (RMSE ∼ 1 log).  The success of the 2D descriptor based machine learning approach when compared against the 3D field based technique pursued here provides a strong impetus for systematically applying machine-learning methods during the lead identification and optimization efforts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGorevaLuMtBsbVg90H21EOLACvtfcHk0liDzfMoR4U6Sg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsF2nu73E&md5=5f916f908fb0b19f4f4ba86f568e22a0</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.6b00290&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.6b00290%26sid%3Dliteratum%253Aachs%26aulast%3DSubramanian%26aufirst%3DG.%26aulast%3DRamsundar%26aufirst%3DB.%26aulast%3DPande%26aufirst%3DV.%26aulast%3DDenny%26aufirst%3DR.%2BA.%26atitle%3DComputational%2520modeling%2520of%2520%25CE%25B2-secretase%25201%2520%2528BACE-1%2529%2520inhibitors%2520using%2520ligand%2520based%2520approaches%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2016%26volume%3D56%26spage%3D1936%26epage%3D1949%26doi%3D10.1021%2Facs.jcim.6b00290" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liaw, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheridan, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Svetnik, V.</span></span> <span> </span><span class="NLM_article-title">Demystifying multitask deep neural networks for quantitative structure–activity relationships</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">2490</span>– <span class="NLM_lpage">2504</span>, <span class="refDoi"> DOI: 10.1021/acs.jcim.7b00087</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.7b00087" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsVejs7jJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2017&pages=2490-2504&author=Y.+Xuauthor=J.+Maauthor=A.+Liawauthor=R.+P.+Sheridanauthor=V.+Svetnik&title=Demystifying+multitask+deep+neural+networks+for+quantitative+structure%E2%80%93activity+relationships&doi=10.1021%2Facs.jcim.7b00087"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Demystifying Multitask Deep Neural Networks for Quantitative Structure-Activity Relationships</span></div><div class="casAuthors">Xu, Yuting; Ma, Junshui; Liaw, Andy; Sheridan, Robert P.; Svetnik, Vladimir</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2490-2504</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Deep neural networks (DNNs) are complex computational models that have found great success in many artificial intelligence applications, such as computer vision and natural language processing.  In the past four years, DNNs also generated promising results for quant. structure-activity relationship (QSAR) tasks.  Previous work showed that DNNs can routinely make better predictions than traditional methods, such as random forests, on a diverse collection of QSAR datasets.  It was also found that multi-task DNN models - those trained on and predicting multiple QSAR properties simultaneously - outperform DNNs trained sep. on the individual datasets in many but not all tasks.  Up to now there is no satisfactory explanation as to why the QSAR of one task, embedded in a multi-task DNN, can borrow information from other unrelated QSAR tasks.  Thus using multi-task DNNs in a way that consistently provides a predictive advantage becomes a challenge.  The authors explore why multi-task DNNs make a difference in predictive performance.  The authors' results show that, during prediction, a multi-task DNN does borrow ''signal'' from mols. with similar structures in the training sets of the other tasks.  However, whether this borrowing leads to better or worse predictive performance depends on whether the activities are correlated.  Based on this realization, the authors develop a strategy to use multi-task DNNs that incorporate prior domain knowledge to select training sets with correlated activities, and the authors demonstrate its effectiveness on several examples.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGruAy4BN8PqXLVg90H21EOLACvtfcHk0ljsqge2CFU8tA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsVejs7jJ&md5=efb621fcdcd07b5527666412375dd7db</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.7b00087&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.7b00087%26sid%3Dliteratum%253Aachs%26aulast%3DXu%26aufirst%3DY.%26aulast%3DMa%26aufirst%3DJ.%26aulast%3DLiaw%26aufirst%3DA.%26aulast%3DSheridan%26aufirst%3DR.%2BP.%26aulast%3DSvetnik%26aufirst%3DV.%26atitle%3DDemystifying%2520multitask%2520deep%2520neural%2520networks%2520for%2520quantitative%2520structure%25E2%2580%2593activity%2520relationships%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2017%26volume%3D57%26spage%3D2490%26epage%3D2504%26doi%3D10.1021%2Facs.jcim.7b00087" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ramsundar, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verras, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tudor, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheridan, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pande, V.</span></span> <span> </span><span class="NLM_article-title">Is multitask deep learning practical for pharma?</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">2068</span>– <span class="NLM_lpage">2076</span>, <span class="refDoi"> DOI: 10.1021/acs.jcim.7b00146</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.7b00146" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtFCisr7O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2017&pages=2068-2076&author=B.+Ramsundarauthor=B.+Liuauthor=Z.+Wuauthor=A.+Verrasauthor=M.+Tudorauthor=R.+P.+Sheridanauthor=V.+Pande&title=Is+multitask+deep+learning+practical+for+pharma%3F&doi=10.1021%2Facs.jcim.7b00146"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Is Multitask Deep Learning Practical for Pharma?</span></div><div class="casAuthors">Ramsundar, Bharath; Liu, Bowen; Wu, Zhenqin; Verras, Andreas; Tudor, Matthew; Sheridan, Robert P.; Pande, Vijay</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2068-2076</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Multitask deep learning has emerged as a powerful tool for computational drug discovery.  However, despite a no. of preliminary studies, multitask deep networks have yet to be widely deployed in the pharmaceutical and biotech industries.  This lack of acceptance stems from both software difficulties and lack of understanding of the robustness of multitask deep networks.  Our work aims to resolve both of these barriers to adoption.  We introduce a high-quality open-source implementation of multitask deep networks as part of the DeepChem open-source platform.  Our implementation enables simple python scripts to construct, fit, and evaluate sophisticated deep models.  We use our implementation to analyze the performance of multitask deep networks and related deep models on four collections of pharmaceutical data (three of which have not previously been analyzed in the literature).  We split these data sets into train/valid/test using time and neighbor splits to test multitask deep learning performance under challenging conditions.  Our results demonstrate that multitask deep networks are surprisingly robust and can offer strong improvement over random forests.  Our anal. and open-source implementation in DeepChem provide an argument that multitask deep networks are ready for widespread use in com. drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpo62Pjc9GpnbVg90H21EOLACvtfcHk0ljsqge2CFU8tA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtFCisr7O&md5=3d27a01e20455d4ee4c869b7cc91717d</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.7b00146&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.7b00146%26sid%3Dliteratum%253Aachs%26aulast%3DRamsundar%26aufirst%3DB.%26aulast%3DLiu%26aufirst%3DB.%26aulast%3DWu%26aufirst%3DZ.%26aulast%3DVerras%26aufirst%3DA.%26aulast%3DTudor%26aufirst%3DM.%26aulast%3DSheridan%26aufirst%3DR.%2BP.%26aulast%3DPande%26aufirst%3DV.%26atitle%3DIs%2520multitask%2520deep%2520learning%2520practical%2520for%2520pharma%253F%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2017%26volume%3D57%26spage%3D2068%26epage%3D2076%26doi%3D10.1021%2Facs.jcim.7b00146" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ramsundar, B.</span>; <span class="NLM_string-name">Kearnes, S.</span>; <span class="NLM_string-name">Riley, P.</span>; <span class="NLM_string-name">Webster, D.</span>; <span class="NLM_string-name">Konerding, D.</span>; <span class="NLM_string-name">Pande, V.</span></span> <span> </span><span class="NLM_article-title">Massively multitask networks for drug discovery</span>.  <i>arXiv:1502.02072</i>, <span class="NLM_year">2015</span>. arXiv.org e-Print archive.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2015&author=B.+Ramsundar&author=S.+Kearnes&author=P.+Riley&author=D.+Webster&author=D.+Konerding&author=V.+Pande&title=Massively+multitask+networks+for+drug+discovery"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dsimple%26sid%3Dliteratum%253Aachs%26aulast%3DRamsundar%26aufirst%3DB.%26atitle%3DMassively%2520multitask%2520networks%2520for%2520drug%2520discovery%26jtitle%3DarXiv%253A1502.02072%26date%3D2015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, G.-W.</span></span> <span> </span><span class="NLM_article-title">Quantitative toxicity prediction using topology based multitask deep neural networks</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">520</span>– <span class="NLM_lpage">531</span>, <span class="refDoi"> DOI: 10.1021/acs.jcim.7b00558</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.7b00558" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=1%3ACAS%3A528%3ADC%252BC1cXlt1altw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2018&pages=520-531&author=K.+Wuauthor=G.-W.+Wei&title=Quantitative+toxicity+prediction+using+topology+based+multitask+deep+neural+networks&doi=10.1021%2Facs.jcim.7b00558"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Quantitative Toxicity Prediction Using Topology Based Multitask Deep Neural Networks</span></div><div class="casAuthors">Wu, Kedi; Wei, Guo-Wei</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">520-531</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The understanding of toxicity is of paramount importance to human health and environmental protection.  Quant. toxicity anal. has become a new std. in the field.  This work introduces element specific persistent homol. (ESPH), an algebraic topol. approach, for quant. toxicity prediction.  ESPH retains crucial chem. information during the topol. abstraction of geometric complexity and provides a representation of small mols. that cannot be obtained by any other method.  To investigate the representability and predictive power of ESPH for small mols., ancillary descriptors have also been developed based on phys. models.  Topol. and phys. descriptors are paired with advanced machine learning algorithms, such as deep neural network (DNN), random forest (RF) and gradient boosting decision tree (GBDT), to facilitate their applications to quant. toxicity predictions.  A topol. based multi-task strategy is proposed to take the advantage of the availability of large data sets while dealing with small data sets.  Four benchmark toxicity data sets that involve quant. measurements are used to validate the proposed approaches.  Extensive numerical studies indicate that the proposed topol. are able to outperform the state-of-the-art methods in the literature for quant. toxicity anal.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo545VJUZNFCbVg90H21EOLACvtfcHk0lhlQ-Uuqvc1Zw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXlt1altw%253D%253D&md5=1c38990135876101457e2687e18319b0</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.7b00558&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.7b00558%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DK.%26aulast%3DWei%26aufirst%3DG.-W.%26atitle%3DQuantitative%2520toxicity%2520prediction%2520using%2520topology%2520based%2520multitask%2520deep%2520neural%2520networks%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2018%26volume%3D58%26spage%3D520%26epage%3D531%26doi%3D10.1021%2Facs.jcim.7b00558" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lenselink, E. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ten Dijke, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bongers, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Papadatos, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Vlijmen, H. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kowalczyk, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ijzerman, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Westen, G. J.</span></span> <span> </span><span class="NLM_article-title">Beyond the hype: deep neural networks outperform established methods using a ChEMBL bioactivity benchmark set</span>. <i>J. Cheminf.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">45</span>, <span class="refDoi"> DOI: 10.1186/s13321-017-0232-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=10.1186%2Fs13321-017-0232-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=1%3ACAS%3A528%3ADC%252BC1cXnsVygtLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2017&pages=45&author=E.+B.+Lenselinkauthor=N.+Ten+Dijkeauthor=B.+Bongersauthor=G.+Papadatosauthor=H.+W.+Van+Vlijmenauthor=W.+Kowalczykauthor=A.+P.+Ijzermanauthor=G.+J.+Van+Westen&title=Beyond+the+hype%3A+deep+neural+networks+outperform+established+methods+using+a+ChEMBL+bioactivity+benchmark+set&doi=10.1186%2Fs13321-017-0232-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Beyond the hype: deep neural networks outperform established methods using a ChEMBL bioactivity benchmark set</span></div><div class="casAuthors">Lenselink, Eelke B.; ten Dijke, Niels; Bongers, Brandon; Papadatos, George; van Vlijmen, Herman W. T.; Kowalczyk, Wojtek; Ijzerman, Adriaan P.; van Westen, Gerard J. P.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Cheminformatics</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">45/1-45/14</span>CODEN:
                <span class="NLM_cas:coden">JCOHB3</span>;
        ISSN:<span class="NLM_cas:issn">1758-2946</span>.
    
            (<span class="NLM_cas:orgname">Chemistry Central Ltd.</span>)
        </div><div class="casAbstract">The increase of publicly available bioactivity data in recent years has fueled and catalyzed research in chemogenomics, data mining, and modeling approaches.  As a direct result, over the past few years a multitude of different methods have been reported and evaluated, such as target fishing, nearest neighbor similarity-based methods, and Quant. Structure Activity Relationship (QSAR)-based protocols.  However, such studies are typically conducted on different datasets, using different validation strategies, and different metrics.  In this study, different methods were compared using one single standardized dataset obtained from ChEMBL, which is made available to the public, using standardized metrics (BEDROC and Matthews Correlation Coeff.).  Specifically, the performance of Naive Bayes, Random Forests, Support Vector Machines, Logistic Regression, and Deep Neural Networks was assessed using QSAR and proteochemometric (PCM) methods.  All methods were validated using both a random split validation and a temporal validation, with the latter being a more realistic benchmark of expected prospective execution.  Deep Neural Networks are the top performing classifiers, highlighting the added value of Deep Neural Networks over other more conventional methods.  Moreover, the best method ('DNN_PCM') performed significantly better at almost one std. deviation higher than the mean performance.  Furthermore, Multi-task and PCM implementations were shown to improve performance over single task Deep Neural Networks.  Conversely, target prediction performed almost two std. deviations under the mean performance.  Random Forests, Support Vector Machines, and Logistic Regression performed around mean performance.  Finally, using an ensemble of DNNs, alongside addnl. tuning, enhanced the relative performance by another 27% (compared with unoptimized 'DNN_PCM').  Here, a standardized set to test and evaluate different machine learning algorithms in the context of multi-task learning is offered by providing the data and the protocols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqc4r8snR0aaLVg90H21EOLACvtfcHk0lhlQ-Uuqvc1Zw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXnsVygtLk%253D&md5=2afd1af603e16ed058b52a7359a650ed</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1186%2Fs13321-017-0232-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs13321-017-0232-0%26sid%3Dliteratum%253Aachs%26aulast%3DLenselink%26aufirst%3DE.%2BB.%26aulast%3DTen%2BDijke%26aufirst%3DN.%26aulast%3DBongers%26aufirst%3DB.%26aulast%3DPapadatos%26aufirst%3DG.%26aulast%3DVan%2BVlijmen%26aufirst%3DH.%2BW.%26aulast%3DKowalczyk%26aufirst%3DW.%26aulast%3DIjzerman%26aufirst%3DA.%2BP.%26aulast%3DVan%2BWesten%26aufirst%3DG.%2BJ.%26atitle%3DBeyond%2520the%2520hype%253A%2520deep%2520neural%2520networks%2520outperform%2520established%2520methods%2520using%2520a%2520ChEMBL%2520bioactivity%2520benchmark%2520set%26jtitle%3DJ.%2520Cheminf.%26date%3D2017%26volume%3D9%26spage%3D45%26doi%3D10.1186%2Fs13321-017-0232-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rodríguez-Pérez, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bajorath, J. r.</span></span> <span> </span><span class="NLM_article-title">Multitask Machine Learning for Classifying Highly and Weakly Potent Kinase Inhibitors</span>. <i>ACS Omega</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">4367</span>– <span class="NLM_lpage">4375</span>, <span class="refDoi"> DOI: 10.1021/acsomega.9b00298</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsomega.9b00298" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=1%3ACAS%3A528%3ADC%252BC1MXjs1Kmtbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2019&pages=4367-4375&author=R.+Rodr%C3%ADguez-P%C3%A9rezauthor=J.+r.+Bajorath&title=Multitask+Machine+Learning+for+Classifying+Highly+and+Weakly+Potent+Kinase+Inhibitors&doi=10.1021%2Facsomega.9b00298"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Multitask Machine Learning for Classifying Highly and Weakly Potent Kinase Inhibitors</span></div><div class="casAuthors">Rodriguez-Perez, Raquel; Bajorath, Juergen</div><div class="citationInfo"><span class="NLM_cas:title">ACS Omega</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">4367-4375</span>CODEN:
                <span class="NLM_cas:coden">ACSODF</span>;
        ISSN:<span class="NLM_cas:issn">2470-1343</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Compd. activity prediction is a major application of machine learning (ML) in pharmaceutical research.  Conventional single-task (ST) learning aims to predict active compds. for a given target.  In addn., multitask (MT) learning attempts to simultaneously predict active compds. for multiple targets.  The underlying rationale of MT learning is to guide and further improve modeling by exploring and exploiting related prediction tasks.  For MT learning, deep neural networks (DNNs) are often used, establishing a link between MT and deep learning.  In this work, ST and MT strategies for ML methods including DNN were compared in the systematic prediction of highly potent and weakly potent protein kinase inhibitors.  A total of 19 030 inhibitors with activity against 103 human kinases were used for modeling.  Given its compn., the data set provided many related prediction tasks.  DNN, support vector machine, and random forest ST and MT models were derived and compared.  Clear trends emerged.  Regardless of the method, MT learning consistently outperformed ST modeling.  Overall MT-DNNs achieved the highest prediction accuracy, but advantages over other MT-ML methods were only marginal.  Furthermore, feature wts. were extd. from models to evaluate correlation between different prediction tasks.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGryP6H8dy5cA7Vg90H21EOLACvtfcHk0lhlQ-Uuqvc1Zw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXjs1Kmtbg%253D&md5=58ecd10e7f04a3da58a58d7b06a75fd7</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1021%2Facsomega.9b00298&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsomega.9b00298%26sid%3Dliteratum%253Aachs%26aulast%3DRodr%25C3%25ADguez-P%25C3%25A9rez%26aufirst%3DR.%26aulast%3DBajorath%26aufirst%3DJ.%2Br.%26atitle%3DMultitask%2520Machine%2520Learning%2520for%2520Classifying%2520Highly%2520and%2520Weakly%2520Potent%2520Kinase%2520Inhibitors%26jtitle%3DACS%2520Omega%26date%3D2019%26volume%3D4%26spage%3D4367%26epage%3D4375%26doi%3D10.1021%2Facsomega.9b00298" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Davis, M. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hunt, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herrgard, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciceri, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wodicka, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pallares, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hocker, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Treiber, D. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zarrinkar, P. P.</span></span> <span> </span><span class="NLM_article-title">Comprehensive analysis of kinase inhibitor selectivity</span>. <i>Nat. Biotechnol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">1046</span>– <span class="NLM_lpage">1051</span>, <span class="refDoi"> DOI: 10.1038/nbt.1990</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=10.1038%2Fnbt.1990" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=22037378" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtlyisrrK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2011&pages=1046-1051&author=M.+I.+Davisauthor=J.+P.+Huntauthor=S.+Herrgardauthor=P.+Ciceriauthor=L.+M.+Wodickaauthor=G.+Pallaresauthor=M.+Hockerauthor=D.+K.+Treiberauthor=P.+P.+Zarrinkar&title=Comprehensive+analysis+of+kinase+inhibitor+selectivity&doi=10.1038%2Fnbt.1990"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Comprehensive analysis of kinase inhibitor selectivity</span></div><div class="casAuthors">Davis, Mindy I.; Hunt, Jeremy P.; Herrgard, Sanna; Ciceri, Pietro; Wodicka, Lisa M.; Pallares, Gabriel; Hocker, Michael; Treiber, Daniel K.; Zarrinkar, Patrick P.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1046-1051</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">We tested the interaction of 72 kinase inhibitors with 442 kinases covering >80% of the human catalytic protein kinome.  Our data show that, as a class, type II inhibitors are more selective than type I inhibitors, but that there are important exceptions to this trend.  The data further illustrate that selective inhibitors have been developed against the majority of kinases targeted by the compds. tested.  Anal. of the interaction patterns reveals a class of 'group-selective' inhibitors broadly active against a single subfamily of kinases, but selective outside that subfamily.  The data set suggests compds. to use as tools to study kinases for which no dedicated inhibitors exist.  It also provides a foundation for further exploring kinase inhibitor biol. and toxicity, as well as for studying the structural basis of the obsd. interaction patterns.  Our findings will help to realize the direct enabling potential of genomics for drug development and basic research about cellular signaling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoEZnKVs6Xt5rVg90H21EOLACvtfcHk0ljNmvt6YeIFxQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtlyisrrK&md5=67639f702cb48ffb2c60e9a250f0df5e</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1038%2Fnbt.1990&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt.1990%26sid%3Dliteratum%253Aachs%26aulast%3DDavis%26aufirst%3DM.%2BI.%26aulast%3DHunt%26aufirst%3DJ.%2BP.%26aulast%3DHerrgard%26aufirst%3DS.%26aulast%3DCiceri%26aufirst%3DP.%26aulast%3DWodicka%26aufirst%3DL.%2BM.%26aulast%3DPallares%26aufirst%3DG.%26aulast%3DHocker%26aufirst%3DM.%26aulast%3DTreiber%26aufirst%3DD.%2BK.%26aulast%3DZarrinkar%26aufirst%3DP.%2BP.%26atitle%3DComprehensive%2520analysis%2520of%2520kinase%2520inhibitor%2520selectivity%26jtitle%3DNat.%2520Biotechnol.%26date%3D2011%26volume%3D29%26spage%3D1046%26epage%3D1051%26doi%3D10.1038%2Fnbt.1990" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Anastassiadis, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deacon, S. W</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Devarajan, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peterson, J. R</span></span> <span> </span><span class="NLM_article-title">Comprehensive assay of kinase catalytic activity reveals features of kinase inhibitor selectivity</span>. <i>Nat. Biotechnol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">1039</span>– <span class="NLM_lpage">1045</span>, <span class="refDoi"> DOI: 10.1038/nbt.2017</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=10.1038%2Fnbt.2017" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=22037377" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtlyisL7E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2011&pages=1039-1045&author=T.+Anastassiadisauthor=S.+W+Deaconauthor=K.+Devarajanauthor=H.+Maauthor=J.+R+Peterson&title=Comprehensive+assay+of+kinase+catalytic+activity+reveals+features+of+kinase+inhibitor+selectivity&doi=10.1038%2Fnbt.2017"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Comprehensive assay of kinase catalytic activity reveals features of kinase inhibitor selectivity</span></div><div class="casAuthors">Anastassiadis, Theonie; Deacon, Sean W.; Devarajan, Karthik; Ma, Haiching; Peterson, Jeffrey R.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1039-1045</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Small-mol. protein kinase inhibitors are widely used to elucidate cellular signaling pathways and are promising therapeutic agents.  Owing to evolutionary conservation of the ATP-binding site, most kinase inhibitors that target this site promiscuously inhibit multiple kinases.  Interpretation of expts. that use these compds. is confounded by a lack of data on the comprehensive kinase selectivity of most inhibitors.  Here we used functional assays to profile the activity of 178 com. available kinase inhibitors against a panel of 300 recombinant protein kinases.  Quant. anal. revealed complex and often unexpected interactions between protein kinases and kinase inhibitors, with a wide spectrum of promiscuity.  Many off-target interactions occur with seemingly unrelated kinases, revealing how large-scale profiling can identify multitargeted inhibitors of specific, diverse kinases.  The results have implications for drug development and provide a resource for selecting compds. to elucidate kinase function and for interpreting the results of expts. involving kinase inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpUljWIhZGPkrVg90H21EOLACvtfcHk0ljNmvt6YeIFxQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtlyisL7E&md5=b2f705509b0e09afa46b01d4669f0515</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1038%2Fnbt.2017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt.2017%26sid%3Dliteratum%253Aachs%26aulast%3DAnastassiadis%26aufirst%3DT.%26aulast%3DDeacon%26aufirst%3DS.%2BW%26aulast%3DDevarajan%26aufirst%3DK.%26aulast%3DMa%26aufirst%3DH.%26aulast%3DPeterson%26aufirst%3DJ.%2BR%26atitle%3DComprehensive%2520assay%2520of%2520kinase%2520catalytic%2520activity%2520reveals%2520features%2520of%2520kinase%2520inhibitor%2520selectivity%26jtitle%3DNat.%2520Biotechnol.%26date%3D2011%26volume%3D29%26spage%3D1039%26epage%3D1045%26doi%3D10.1038%2Fnbt.2017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Elkins, J. M</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedele, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szklarz, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abdul Azeez, K. R</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salah, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mikolajczyk, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Romanov, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sepetov, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, X.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roth, B. L</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Al Haj Zen, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fourches, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muratov, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tropsha, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morris, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Teicher, B. A</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kunkel, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Polley, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lackey, K. E</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Atkinson, F. L</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Overington, J. P</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bamborough, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Price, D. J</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Willson, T. M</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drewry, D. H</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zuercher, W. J</span></span> <span> </span><span class="NLM_article-title">Comprehensive characterization of the Published Kinase Inhibitor Set</span>. <i>Nat. Biotechnol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>34</i></span>,  <span class="NLM_fpage">95</span>– <span class="NLM_lpage">103</span>, <span class="refDoi"> DOI: 10.1038/nbt.3374</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=10.1038%2Fnbt.3374" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=26501955" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhslantL%252FM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=2016&pages=95-103&author=J.+M+Elkinsauthor=V.+Fedeleauthor=M.+Szklarzauthor=K.+R+Abdul+Azeezauthor=E.+Salahauthor=J.+Mikolajczykauthor=S.+Romanovauthor=N.+Sepetovauthor=X.-P.+Huangauthor=B.+L+Rothauthor=A.+Al+Haj+Zenauthor=D.+Fourchesauthor=E.+Muratovauthor=A.+Tropshaauthor=J.+Morrisauthor=B.+A+Teicherauthor=M.+Kunkelauthor=E.+Polleyauthor=K.+E+Lackeyauthor=F.+L+Atkinsonauthor=J.+P+Overingtonauthor=P.+Bamboroughauthor=S.+Mullerauthor=D.+J+Priceauthor=T.+M+Willsonauthor=D.+H+Drewryauthor=S.+Knappauthor=W.+J+Zuercher&title=Comprehensive+characterization+of+the+Published+Kinase+Inhibitor+Set&doi=10.1038%2Fnbt.3374"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Comprehensive characterization of the Published Kinase Inhibitor Set</span></div><div class="casAuthors">Elkins, Jonathan M.; Fedele, Vita; Szklarz, Marta; Abdul Azeez, Kamal R.; Salah, Eidarus; Mikolajczyk, Jowita; Romanov, Sergei; Sepetov, Nikolai; Huang, Xi-Ping; Roth, Bryan L.; Al Haj Zen, Ayman; Fourches, Denis; Muratov, Eugene; Tropsha, Alex; Morris, Joel; Teicher, Beverly A.; Kunkel, Mark; Polley, Eric; Lackey, Karen E.; Atkinson, Francis L.; Overington, John P.; Bamborough, Paul; Muller, Susanne; Price, Daniel J.; Willson, Timothy M.; Drewry, David H.; Knapp, Stefan; Zuercher, William J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">34</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">95-103</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Despite the success of protein kinase inhibitors as approved therapeutics, drug discovery has focused on a small subset of kinase targets.  Here we provide a thorough characterization of the Published Kinase Inhibitor Set (PKIS), a set of 367 small-mol. ATP-competitive kinase inhibitors that was recently made freely available with the aim of expanding research in this field and as an expt. in open-source target validation.  We screen the set in activity assays with 224 recombinant kinases and 24 G protein-coupled receptors and in cellular assays of cancer cell proliferation and angiogenesis.  We identify chem. starting points for designing new chem. probes of orphan kinases and illustrate the utility of these leads by developing a selective inhibitor for the previously untargeted kinases LOK and SLK.  Our cellular screens reveal compds. that modulate cancer cell growth and angiogenesis in vitro.  These reagents and assocd. data illustrate an efficient way forward to increasing understanding of the historically untargeted kinome.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo4P1WN3gdiXbVg90H21EOLACvtfcHk0ljNmvt6YeIFxQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhslantL%252FM&md5=3f15096e79632dbaaadf4460a6848e60</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1038%2Fnbt.3374&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt.3374%26sid%3Dliteratum%253Aachs%26aulast%3DElkins%26aufirst%3DJ.%2BM%26aulast%3DFedele%26aufirst%3DV.%26aulast%3DSzklarz%26aufirst%3DM.%26aulast%3DAbdul%2BAzeez%26aufirst%3DK.%2BR%26aulast%3DSalah%26aufirst%3DE.%26aulast%3DMikolajczyk%26aufirst%3DJ.%26aulast%3DRomanov%26aufirst%3DS.%26aulast%3DSepetov%26aufirst%3DN.%26aulast%3DHuang%26aufirst%3DX.-P.%26aulast%3DRoth%26aufirst%3DB.%2BL%26aulast%3DAl%2BHaj%2BZen%26aufirst%3DA.%26aulast%3DFourches%26aufirst%3DD.%26aulast%3DMuratov%26aufirst%3DE.%26aulast%3DTropsha%26aufirst%3DA.%26aulast%3DMorris%26aufirst%3DJ.%26aulast%3DTeicher%26aufirst%3DB.%2BA%26aulast%3DKunkel%26aufirst%3DM.%26aulast%3DPolley%26aufirst%3DE.%26aulast%3DLackey%26aufirst%3DK.%2BE%26aulast%3DAtkinson%26aufirst%3DF.%2BL%26aulast%3DOverington%26aufirst%3DJ.%2BP%26aulast%3DBamborough%26aufirst%3DP.%26aulast%3DMuller%26aufirst%3DS.%26aulast%3DPrice%26aufirst%3DD.%2BJ%26aulast%3DWillson%26aufirst%3DT.%2BM%26aulast%3DDrewry%26aufirst%3DD.%2BH%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DZuercher%26aufirst%3DW.%2BJ%26atitle%3DComprehensive%2520characterization%2520of%2520the%2520Published%2520Kinase%2520Inhibitor%2520Set%26jtitle%3DNat.%2520Biotechnol.%26date%3D2016%26volume%3D34%26spage%3D95%26epage%3D103%26doi%3D10.1038%2Fnbt.3374" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Drewry, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wells, C. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andrews, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Angell, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Al-Ali, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Axtman, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Capuzzi, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elkins, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ettmayer, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frederiksen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gileadi, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hooper, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laufer, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luecking, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michaelides, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muratov, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Denny, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saikatendu, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Treiber, D. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zuercher, W. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Willson, T. M.</span></span> <span> </span><span class="NLM_article-title">Progress towards a public chemogenomic set for protein kinases and a call for contributions</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>12</i></span>, <span class="NLM_elocation-id">e0181585</span> <span class="refDoi"> DOI: 10.1371/journal.pone.0181585</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=10.1371%2Fjournal.pone.0181585" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=28767711" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhsFWktL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2017&author=D.+H.+Drewryauthor=C.+I.+Wellsauthor=D.+M.+Andrewsauthor=R.+Angellauthor=H.+Al-Aliauthor=A.+D.+Axtmanauthor=S.+J.+Capuzziauthor=J.+M.+Elkinsauthor=P.+Ettmayerauthor=M.+Frederiksenauthor=O.+Gileadiauthor=N.+Grayauthor=A.+Hooperauthor=S.+Knappauthor=S.+Lauferauthor=U.+Lueckingauthor=M.+Michaelidesauthor=S.+Mullerauthor=E.+Muratovauthor=R.+A.+Dennyauthor=K.+S.+Saikatenduauthor=D.+K.+Treiberauthor=W.+J.+Zuercherauthor=T.+M.+Willson&title=Progress+towards+a+public+chemogenomic+set+for+protein+kinases+and+a+call+for+contributions&doi=10.1371%2Fjournal.pone.0181585"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Progress towards a public chemogenomic set for protein kinases and a call for contributions</span></div><div class="casAuthors">Drewry, David H.; Wells, Carrow I.; Andrews, David M.; Angell, Richard; Al-Ali, Hassan; Axtman, Alison D.; Capuzzi, Stephen J.; Elkins, Jonathan M.; Ettmayer, Peter; Frederiksen, Mathias; Gileadi, Opher; Gray, Nathanael; Hooper, Alice; Knapp, Stefan; Laufer, Stefan; Luecking, Ulrich; Michaelides, Michael; Muller, Susanne; Muratov, Eugene; Aldrin, Denny R.; Saikatendu, Kumar S.; Treiber, Daniel K.; Zuercher, William J.; Willson, Timothy M.</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">e0181585/1-e0181585/20</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Protein kinases are highly tractable targets for drug discovery.  However, the biol. function and therapeutic potential of the majority of the 500+ human protein kinases remains unknown.  We have developed phys. and virtual collections of small mol. inhibitors, which we call chemogenomic sets, that are designed to inhibit the catalytic function of almost half the human protein kinases.  In this manuscript we share our progress towards generation of a comprehensive kinase chemogenomic set (KCGS), release kinome profiling data of a large inhibitor set (Published Kinase Inhibitor Set 2 (PKIS2)), and outline a process through which the community can openly collaborate to create a KCGS that probes the full complement of human protein kinases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpozOiNZl51XbVg90H21EOLACvtfcHk0liWIopYfgY8YQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhsFWktL0%253D&md5=9c27ba5cd21da43d30ce485ffa0e1675</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0181585&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0181585%26sid%3Dliteratum%253Aachs%26aulast%3DDrewry%26aufirst%3DD.%2BH.%26aulast%3DWells%26aufirst%3DC.%2BI.%26aulast%3DAndrews%26aufirst%3DD.%2BM.%26aulast%3DAngell%26aufirst%3DR.%26aulast%3DAl-Ali%26aufirst%3DH.%26aulast%3DAxtman%26aufirst%3DA.%2BD.%26aulast%3DCapuzzi%26aufirst%3DS.%2BJ.%26aulast%3DElkins%26aufirst%3DJ.%2BM.%26aulast%3DEttmayer%26aufirst%3DP.%26aulast%3DFrederiksen%26aufirst%3DM.%26aulast%3DGileadi%26aufirst%3DO.%26aulast%3DGray%26aufirst%3DN.%26aulast%3DHooper%26aufirst%3DA.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DLaufer%26aufirst%3DS.%26aulast%3DLuecking%26aufirst%3DU.%26aulast%3DMichaelides%26aufirst%3DM.%26aulast%3DMuller%26aufirst%3DS.%26aulast%3DMuratov%26aufirst%3DE.%26aulast%3DDenny%26aufirst%3DR.%2BA.%26aulast%3DSaikatendu%26aufirst%3DK.%2BS.%26aulast%3DTreiber%26aufirst%3DD.%2BK.%26aulast%3DZuercher%26aufirst%3DW.%2BJ.%26aulast%3DWillson%26aufirst%3DT.%2BM.%26atitle%3DProgress%2520towards%2520a%2520public%2520chemogenomic%2520set%2520for%2520protein%2520kinases%2520and%2520a%2520call%2520for%2520contributions%26jtitle%3DPLoS%2520One%26date%3D2017%26volume%3D12%26doi%3D10.1371%2Fjournal.pone.0181585" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sutherland, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cahya, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vieth, M.</span></span> <span> </span><span class="NLM_article-title">What general conclusions can we draw from kinase profiling data sets?</span>. <i>Biochim. Biophys. Acta, Proteins Proteomics</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>1834</i></span>,  <span class="NLM_fpage">1425</span>– <span class="NLM_lpage">1433</span>, <span class="refDoi"> DOI: 10.1016/j.bbapap.2012.12.023</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=10.1016%2Fj.bbapap.2012.12.023" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=23333421" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=1%3ACAS%3A528%3ADC%252BC3sXit1KksLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1834&publication_year=2013&pages=1425-1433&author=J.+J.+Sutherlandauthor=C.+Gaoauthor=S.+Cahyaauthor=M.+Vieth&title=What+general+conclusions+can+we+draw+from+kinase+profiling+data+sets%3F&doi=10.1016%2Fj.bbapap.2012.12.023"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">What general conclusions can we draw from kinase profiling data sets?</span></div><div class="casAuthors">Sutherland, Jeffrey J.; Gao, Cen; Cahya, Suntara; Vieth, Michal</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, Proteins and Proteomics</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">1834</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1425-1433</span>CODEN:
                <span class="NLM_cas:coden">BBAPBW</span>;
        ISSN:<span class="NLM_cas:issn">1570-9639</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B. V.</span>)
        </div><div class="casAbstract">A review.  Understanding general selectivity trends across the kinome has implications ranging from target selection, compd. prioritization, toxicity and patient tailoring.  Several recent publications have described the characterization of kinase inhibitors via large assay panels, offering a range of generalizations that influenced kinase inhibitor research trends.  Since a subset of profiled inhibitors overlap across reports, we evaluated the concordance of activity results for the same compd.-kinase pairs across four data sources generated from different kinase biochem. assay technologies.  Overall, 77% of all results are within 3 fold or qual. in agreement across sources.  However, the agreement for active compds. is only 37%, indicating that different profiling panels are in better agreement to det. a compd.'s lack of activity rather than degree of activity.  Low concordance is also found when comparing the promiscuity of kinase targets evaluated from different sources, and the pharmacol. similarity of kinases.  In contrast, the overall promiscuity of kinase inhibitors was consistent across sources.  We highlight the difficulty of drawing general conclusions from such data by showing that no significant selectivity difference distinguishes type I vs. type II inhibitors, and limited kinase space similarity that is consistent across different sources.  This article is part of a Special Issue entitled: Inhibitors of Protein Kinases (2012).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrWOFwnlgm0AbVg90H21EOLACvtfcHk0liWIopYfgY8YQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXit1KksLk%253D&md5=59c6667035700272be4e4e0e890490ed</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1016%2Fj.bbapap.2012.12.023&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbapap.2012.12.023%26sid%3Dliteratum%253Aachs%26aulast%3DSutherland%26aufirst%3DJ.%2BJ.%26aulast%3DGao%26aufirst%3DC.%26aulast%3DCahya%26aufirst%3DS.%26aulast%3DVieth%26aufirst%3DM.%26atitle%3DWhat%2520general%2520conclusions%2520can%2520we%2520draw%2520from%2520kinase%2520profiling%2520data%2520sets%253F%26jtitle%3DBiochim.%2520Biophys.%2520Acta%252C%2520Proteins%2520Proteomics%26date%3D2013%26volume%3D1834%26spage%3D1425%26epage%3D1433%26doi%3D10.1016%2Fj.bbapap.2012.12.023" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Posy, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hermsmeier, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vaccaro, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ott, K.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Todderud, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lippy, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trainor, G. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loughney, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, S. R.</span></span> <span> </span><span class="NLM_article-title">Trends in kinase selectivity: insights for target class-focused library screening</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">54</span>– <span class="NLM_lpage">66</span>, <span class="refDoi"> DOI: 10.1021/jm101195a</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm101195a" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsFSms7jJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=54-66&author=S.+L.+Posyauthor=M.+A.+Hermsmeierauthor=W.+Vaccaroauthor=K.-H.+Ottauthor=G.+Todderudauthor=J.+S.+Lippyauthor=G.+L.+Trainorauthor=D.+A.+Loughneyauthor=S.+R.+Johnson&title=Trends+in+kinase+selectivity%3A+insights+for+target+class-focused+library+screening&doi=10.1021%2Fjm101195a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Trends in Kinase Selectivity: Insights for Target Class-Focused Library Screening</span></div><div class="casAuthors">Posy, Shana L.; Hermsmeier, Mark A.; Vaccaro, Wayne; Ott, Karl-Heinz; Todderud, Gordon; Lippy, Jonathan S.; Trainor, George L.; Loughney, Deborah A.; Johnson, Stephen R.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">54-66</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A kinome-wide selectivity screen of >20000 compds. with a rich representation of many structural classes has been completed.  Anal. of the selectivity patterns for each class shows that a broad spectrum of structural scaffolds can achieve specificity for many kinase families.  Kinase selectivity and potency are inversely correlated, a trend that is also found in a large set of kinase functional data.  Although selective and nonselective compds. are mostly similar in their physicochem. characteristics, we identify specific features that are present more frequently in compds. that bind to many kinases.  Our results support a scaffold-oriented approach for building compd. collections to screen kinase targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpJlqg1ZszRmLVg90H21EOLACvtfcHk0ljlC8zI1naZLQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsFSms7jJ&md5=f824b454904fb99d1ddd1ac6396cb784</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1021%2Fjm101195a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm101195a%26sid%3Dliteratum%253Aachs%26aulast%3DPosy%26aufirst%3DS.%2BL.%26aulast%3DHermsmeier%26aufirst%3DM.%2BA.%26aulast%3DVaccaro%26aufirst%3DW.%26aulast%3DOtt%26aufirst%3DK.-H.%26aulast%3DTodderud%26aufirst%3DG.%26aulast%3DLippy%26aufirst%3DJ.%2BS.%26aulast%3DTrainor%26aufirst%3DG.%2BL.%26aulast%3DLoughney%26aufirst%3DD.%2BA.%26aulast%3DJohnson%26aufirst%3DS.%2BR.%26atitle%3DTrends%2520in%2520kinase%2520selectivity%253A%2520insights%2520for%2520target%2520class-focused%2520library%2520screening%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26spage%3D54%26epage%3D66%26doi%3D10.1021%2Fjm101195a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mayr, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klambauer, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Unterthiner, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steijaert, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wegner, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ceulemans, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clevert, D.-A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hochreiter, S.</span></span> <span> </span><span class="NLM_article-title">Large-scale comparison of machine learning methods for drug target prediction on ChEMBL</span>. <i>Chem. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">5441</span>– <span class="NLM_lpage">5451</span>, <span class="refDoi"> DOI: 10.1039/C8SC00148K</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=10.1039%2FC8SC00148K" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=30155234" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtVOlu7jE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2018&pages=5441-5451&author=A.+Mayrauthor=G.+Klambauerauthor=T.+Unterthinerauthor=M.+Steijaertauthor=J.+K.+Wegnerauthor=H.+Ceulemansauthor=D.-A.+Clevertauthor=S.+Hochreiter&title=Large-scale+comparison+of+machine+learning+methods+for+drug+target+prediction+on+ChEMBL&doi=10.1039%2FC8SC00148K"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Large-scale comparison of machine learning methods for drug target prediction on ChEMBL</span></div><div class="casAuthors">Mayr, Andreas; Klambauer, Guenter; Unterthiner, Thomas; Steijaert, Marvin; Wegner, Joerg K.; Ceulemans, Hugo; Clevert, Djork-Arne; Hochreiter, Sepp</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Science</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">5441-5451</span>CODEN:
                <span class="NLM_cas:coden">CSHCCN</span>;
        ISSN:<span class="NLM_cas:issn">2041-6520</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Deep learning is currently the most successful machine learning technique in a wide range of application areas and has recently been applied successfully in drug discovery research to predict potential drug targets and to screen for active mols.  However, due to (1) the lack of large-scale studies, (2) the compd. series bias that is characteristic of drug discovery datasets and (3) the hyperparameter selection bias that comes with the high no. of potential deep learning architectures, it remains unclear whether deep learning can indeed outperform existing computational methods in drug discovery tasks.  We therefore assessed the performance of several deep learning methods on a large-scale drug discovery dataset and compared the results with those of other machine learning and target prediction methods.  To avoid potential biases from hyperparameter selection or compd. series, we used a nested cluster-cross-validation strategy.  We found (1) that deep learning methods significantly outperform all competing methods and (2) that the predictive performance of deep learning is in many cases comparable to that of tests performed in wet labs (i.e., in vitro assays).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrGfZgi5YuZVrVg90H21EOLACvtfcHk0ljlC8zI1naZLQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtVOlu7jE&md5=e91e5905e357d0c49ce443d78125537f</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1039%2FC8SC00148K&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC8SC00148K%26sid%3Dliteratum%253Aachs%26aulast%3DMayr%26aufirst%3DA.%26aulast%3DKlambauer%26aufirst%3DG.%26aulast%3DUnterthiner%26aufirst%3DT.%26aulast%3DSteijaert%26aufirst%3DM.%26aulast%3DWegner%26aufirst%3DJ.%2BK.%26aulast%3DCeulemans%26aufirst%3DH.%26aulast%3DClevert%26aufirst%3DD.-A.%26aulast%3DHochreiter%26aufirst%3DS.%26atitle%3DLarge-scale%2520comparison%2520of%2520machine%2520learning%2520methods%2520for%2520drug%2520target%2520prediction%2520on%2520ChEMBL%26jtitle%3DChem.%2520Sci.%26date%3D2018%26volume%3D9%26spage%3D5441%26epage%3D5451%26doi%3D10.1039%2FC8SC00148K" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">vn
der Maaten, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hinton, G.</span></span> <span> </span><span class="NLM_article-title">Visualizing data using t-SNE</span>. <i>J. Mach. Learn. Res.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">2579</span>– <span class="NLM_lpage">2605</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2008&pages=2579-2605&author=L.+vn%0Ader+Maatenauthor=G.+Hinton&title=Visualizing+data+using+t-SNE"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3Dvn%2Bder%2BMaaten%26aufirst%3DL.%26aulast%3DHinton%26aufirst%3DG.%26atitle%3DVisualizing%2520data%2520using%2520t-SNE%26jtitle%3DJ.%2520Mach.%2520Learn.%2520Res.%26date%3D2008%26volume%3D9%26spage%3D2579%26epage%3D2605" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Avram, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bora, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Halip, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Curpǎn, R.</span></span> <span> </span><span class="NLM_article-title">Modeling kinase inhibition using highly confident data sets</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">957</span>– <span class="NLM_lpage">967</span>, <span class="refDoi"> DOI: 10.1021/acs.jcim.7b00729</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.7b00729" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=1%3ACAS%3A528%3ADC%252BC1cXosFensro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2018&pages=957-967&author=S.+Avramauthor=A.+Boraauthor=L.+Halipauthor=R.+Curp%C7%8En&title=Modeling+kinase+inhibition+using+highly+confident+data+sets&doi=10.1021%2Facs.jcim.7b00729"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Modeling Kinase Inhibition Using Highly Confident Data Sets</span></div><div class="casAuthors">Avram, Sorin; Bora, Alina; Halip, Liliana; Curpan, Ramona</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">957-967</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Protein kinases form a consistent class of promising drug targets, and several efforts have been made to predict the activities of small mols. against a representative part of the kinome.  This study continues our previous work aiming to build and measure the performance of ligand-based kinase inhibitor prediction models.  Here we analyzed kinase-inhibitor pairs with multiple activity points extd. from the ChEMBL database and identified the main sources of inconsistency.  Our results indicate that lower IC50 values are usually less affected by errors and reflect more accurately the structure-activity relationship of the mols. against the target, ideally for quant. structure-activity relationship studies.  Further, we modeled the activities of 104 kinases using unbiased target-specific activity points.  The performance of predictors built on extended connectivity fingerprints (ECFP4) and two-dimensional pharmacophore fingerprints (PFPs) are compared by means of tolerance intervals (TIs) (95%/95%) in virtual screening (VS) and classification tasks using external random (RandSets) and diversity-based (DivSets) test sets.  We found that the two encodings perform superior to each other on different kinases in VS and that PFP models perform consistently better in classifying actives (higher sensitivity).  Next, we combined the two encodings into a single one (PFPECFP) and demonstrated that esp. in VS (as indicated by the exponential receiver operating curve enrichment metric (eROCE)), for the vast majority of kinases the model performance increased compared with the individual fingerprint models.  These findings are highlighted in the more challenging DivSets compared with RandSets.  The current paper explores the boundaries of inhibitor predictors for individual kinases to enhance VS and ultimately aid the discovery of novel compds. with desirable polypharmacol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpvbuviAgeXNLVg90H21EOLACvtfcHk0lj8j97wtPca7w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXosFensro%253D&md5=31fe4e3cf0a44becc1a97b345e0975da</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.7b00729&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.7b00729%26sid%3Dliteratum%253Aachs%26aulast%3DAvram%26aufirst%3DS.%26aulast%3DBora%26aufirst%3DA.%26aulast%3DHalip%26aufirst%3DL.%26aulast%3DCurp%25C7%258En%26aufirst%3DR.%26atitle%3DModeling%2520kinase%2520inhibition%2520using%2520highly%2520confident%2520data%2520sets%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2018%26volume%3D58%26spage%3D957%26epage%3D967%26doi%3D10.1021%2Facs.jcim.7b00729" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Janssen, A. P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grimm, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wijdeven, R. H. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lenselink, E. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neefjes, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Boeckel, C. A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Westen, G. J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van der Stelt, M.</span></span> <span> </span><span class="NLM_article-title">Drug discovery maps, a machine learning model that visualizes and predicts kinome–inhibitor interaction landscapes</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">1221</span>– <span class="NLM_lpage">1229</span>, <span class="refDoi"> DOI: 10.1021/acs.jcim.8b00640</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.8b00640" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitVansb3K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2019&pages=1221-1229&author=A.+P.+A.+Janssenauthor=S.+H.+Grimmauthor=R.+H.+M.+Wijdevenauthor=E.+B.+Lenselinkauthor=J.+Neefjesauthor=C.+A.+A.+van+Boeckelauthor=G.+J.+P.+van+Westenauthor=M.+van+der+Stelt&title=Drug+discovery+maps%2C+a+machine+learning+model+that+visualizes+and+predicts+kinome%E2%80%93inhibitor+interaction+landscapes&doi=10.1021%2Facs.jcim.8b00640"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Drug Discovery Maps, a Machine Learning Model That Visualizes and Predicts Kinome-Inhibitor Interaction Landscapes</span></div><div class="casAuthors">Janssen, Antonius P. A.; Grimm, Sebastian H.; Wijdeven, Ruud H. M.; Lenselink, Eelke B.; Neefjes, Jacques; van Boeckel, Constant A. A.; van Westen, Gerard J. P.; van der Stelt, Mario</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1221-1229</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The interpretation of high-dimensional structure-activity data sets in drug discovery to predict ligand-protein interaction landscapes is a challenging task.  Here we present Drug Discovery Maps (DDM), a machine learning model that maps the activity profile of compds. across an entire protein family, as illustrated here for the kinase family.  DDM is based on the t-distributed stochastic neighbor embedding (t-SNE) algorithm to generate a visualization of mol. and biol. similarity.  DDM maps chem. and target space and predicts the activities of novel kinase inhibitors across the kinome.  The model was validated using independent data sets and in a prospective exptl. setting, where DDM predicted new inhibitors for FMS-like tyrosine kinase 3 (FLT3), a therapeutic target for the treatment of acute myeloid leukemia.  Compds. were resynthesized, yielding highly potent, cellularly active FLT3 inhibitors.  Biochem. assays confirmed most of the predicted off-targets.  DDM is further unique in that it is completely open-source and available as a ready-to-use executable to facilitate broad and easy adoption.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqJ32jhSPSIdLVg90H21EOLACvtfcHk0lj8j97wtPca7w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitVansb3K&md5=dc60f5872d49edde162e29327857e08f</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.8b00640&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.8b00640%26sid%3Dliteratum%253Aachs%26aulast%3DJanssen%26aufirst%3DA.%2BP.%2BA.%26aulast%3DGrimm%26aufirst%3DS.%2BH.%26aulast%3DWijdeven%26aufirst%3DR.%2BH.%2BM.%26aulast%3DLenselink%26aufirst%3DE.%2BB.%26aulast%3DNeefjes%26aufirst%3DJ.%26aulast%3Dvan%2BBoeckel%26aufirst%3DC.%2BA.%2BA.%26aulast%3Dvan%2BWesten%26aufirst%3DG.%2BJ.%2BP.%26aulast%3Dvan%2Bder%2BStelt%26aufirst%3DM.%26atitle%3DDrug%2520discovery%2520maps%252C%2520a%2520machine%2520learning%2520model%2520that%2520visualizes%2520and%2520predicts%2520kinome%25E2%2580%2593inhibitor%2520interaction%2520landscapes%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2019%26volume%3D59%26spage%3D1221%26epage%3D1229%26doi%3D10.1021%2Facs.jcim.8b00640" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ai, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ji, Y. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geng, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H.</span></span> <span> </span><span class="NLM_article-title">Novel 5-(benzyloxy)pyridin-2(1 H)-one derivatives as potent c-Met inhibitors</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">2408</span>– <span class="NLM_lpage">2413</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2013.02.037</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=10.1016%2Fj.bmcl.2013.02.037" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=23474386" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=1%3ACAS%3A528%3ADC%252BC3sXktFaisrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2013&pages=2408-2413&author=D.+Zhangauthor=J.+Aiauthor=Z.+Liangauthor=W.+Zhuauthor=X.+Pengauthor=X.+Chenauthor=Y.+C.+Jiauthor=H.+Jiangauthor=C.+Luoauthor=M.+Gengauthor=H.+Liu&title=Novel+5-%28benzyloxy%29pyridin-2%281+H%29-one+derivatives+as+potent+c-Met+inhibitors&doi=10.1016%2Fj.bmcl.2013.02.037"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Novel 5-(benzyloxy)pyridin-2(1H)-one derivatives as potent c-Met inhibitors</span></div><div class="casAuthors">Zhang, Dengyou; Ai, Jing; Liang, Zhongjie; Zhu, Wei; Peng, Xia; Chen, Xianjie; Ji, YinChun; Jiang, Hualiang; Luo, Cheng; Geng, Meiyu; Liu, Hong</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2408-2413</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A series of novel 5-(benzyloxy)pyridin-2(1H)-ones were designed, synthesized and biol. evaluated for c-Met inhibition.  Various amides and benzimidazoles at the C-3 position were investigated.  A potent compd. I with a c-Met IC50 of 12 nM was identified.  This compd. exhibited potent inhibition of EBC-1 cell assocd. with c-Met constitutive activation and showed high selectivity for c-Met than other tested 11 kinases.  The binding model I with c-Met was disclosed by docking anal.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqGU1nKpF0QGbVg90H21EOLACvtfcHk0lj8j97wtPca7w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXktFaisrc%253D&md5=1ae7dab74ba56e7cd1b5701959d474b7</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2013.02.037&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2013.02.037%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DD.%26aulast%3DAi%26aufirst%3DJ.%26aulast%3DLiang%26aufirst%3DZ.%26aulast%3DZhu%26aufirst%3DW.%26aulast%3DPeng%26aufirst%3DX.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DJi%26aufirst%3DY.%2BC.%26aulast%3DJiang%26aufirst%3DH.%26aulast%3DLuo%26aufirst%3DC.%26aulast%3DGeng%26aufirst%3DM.%26aulast%3DLiu%26aufirst%3DH.%26atitle%3DNovel%25205-%2528benzyloxy%2529pyridin-2%25281%2520H%2529-one%2520derivatives%2520as%2520potent%2520c-Met%2520inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2013%26volume%3D23%26spage%3D2408%26epage%3D2413%26doi%3D10.1016%2Fj.bmcl.2013.02.037" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peng, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ji, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tong, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H.</span></span> <span> </span><span class="NLM_article-title">Design, synthesis and biological evaluation of novel 4-anilinoquinazolines with C-6 urea-linked side chains as inhibitors of the epidermal growth factor receptor</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">7988</span>– <span class="NLM_lpage">7998</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2013.09.049</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=10.1016%2Fj.bmc.2013.09.049" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=24183742" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhslSlsLrI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2013&pages=7988-7998&author=X.+Zhangauthor=T.+Pengauthor=X.+Jiauthor=J.+Liauthor=L.+Tongauthor=Z.+Liauthor=W.+Yangauthor=Y.+Xuauthor=M.+Liauthor=J.+Dingauthor=H.+Jiangauthor=H.+Xieauthor=H.+Liu&title=Design%2C+synthesis+and+biological+evaluation+of+novel+4-anilinoquinazolines+with+C-6+urea-linked+side+chains+as+inhibitors+of+the+epidermal+growth+factor+receptor&doi=10.1016%2Fj.bmc.2013.09.049"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Design, synthesis and biological evaluation of novel 4-anilinoquinazolines with C-6 urea-linked side chains as inhibitors of the epidermal growth factor receptor</span></div><div class="casAuthors">Zhang, Xu; Peng, Ting; Ji, Xun; Li, Jian; Tong, Linjiang; Li, Zeng; Yang, Wei; Xu, Yungen; Li, Mengyuan; Ding, Jian; Jiang, Hualiang; Xie, Hua; Liu, Hong</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">7988-7998</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A novel series of anilinoquinazoline compds. with C-6 urea-linked side chains was designed and synthesized as reversible inhibitors of epidermal growth factor receptor (EGFR) based on the structure-activity relationships (SARs) of anilinoquinazoline inhibitors.  All compds. demonstrated good inhibition of EGFR wild type (EGFR wt) and inhibited proliferation of A431cell line.  Compds. (I) and (II) almost completely blocked the phosphorylation of EGFR in A431 cell line at 0.01 μM.  Interestingly, all of the compds. also demonstrated moderate inhibition of EGFR/T790M/L858R.  In addn., compds. (III) and (IV) blocked the autophosphorylation of EGFR in NCI-H1975 cells at high concn. (10 μM), and compd. 8f was confirmed to be an irreversible inhibitor through the diln. method.  Importantly, the compds. with C-6 urea-linked side chains which did not contain Michael acceptors demonstrated moderate to strong irreversible EGFR inhibition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqm9A3kxR7nTbVg90H21EOLACvtfcHk0lipsSt0xF_KWw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhslSlsLrI&md5=30a7a86cd500a9073efe50cd1660bf4b</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2013.09.049&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2013.09.049%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DPeng%26aufirst%3DT.%26aulast%3DJi%26aufirst%3DX.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DTong%26aufirst%3DL.%26aulast%3DLi%26aufirst%3DZ.%26aulast%3DYang%26aufirst%3DW.%26aulast%3DXu%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DM.%26aulast%3DDing%26aufirst%3DJ.%26aulast%3DJiang%26aufirst%3DH.%26aulast%3DXie%26aufirst%3DH.%26aulast%3DLiu%26aufirst%3DH.%26atitle%3DDesign%252C%2520synthesis%2520and%2520biological%2520evaluation%2520of%2520novel%25204-anilinoquinazolines%2520with%2520C-6%2520urea-linked%2520side%2520chains%2520as%2520inhibitors%2520of%2520the%2520epidermal%2520growth%2520factor%2520receptor%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2013%26volume%3D21%26spage%3D7988%26epage%3D7998%26doi%3D10.1016%2Fj.bmc.2013.09.049" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Grothey, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cutsem, E. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sobrero, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siena, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Falcone, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ychou, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Humblet, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bouche, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mineur, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barone, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adenis, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tabernero, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yoshino, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lenz, H.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goldberg, R. M</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sargent, D. J</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cihon, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cupit, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wagner, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laurent, D.</span></span> <span> </span><span class="NLM_article-title">Regorafenib monotherapy for previously treated metastatic colorectal cancer (CORRECT): an international, multicentre, randomised, placebo-controlled, phase 3 trial</span>. <i>Lancet</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>381</i></span>,  <span class="NLM_fpage">303</span>– <span class="NLM_lpage">312</span>, <span class="refDoi"> DOI: 10.1016/S0140-6736(12)61900-X</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=10.1016%2FS0140-6736%2812%2961900-X" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=23177514" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=1%3ACAS%3A528%3ADC%252BC38Xhslarsr%252FE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=381&publication_year=2013&pages=303-312&author=A.+Grotheyauthor=E.+V.+Cutsemauthor=A.+Sobreroauthor=S.+Sienaauthor=A.+Falconeauthor=M.+Ychouauthor=Y.+Humbletauthor=O.+Boucheauthor=L.+Mineurauthor=C.+Baroneauthor=A.+Adenisauthor=J.+Taberneroauthor=T.+Yoshinoauthor=H.-J.+Lenzauthor=R.+M+Goldbergauthor=D.+J+Sargentauthor=F.+Cihonauthor=L.+Cupitauthor=A.+Wagnerauthor=D.+Laurent&title=Regorafenib+monotherapy+for+previously+treated+metastatic+colorectal+cancer+%28CORRECT%29%3A+an+international%2C+multicentre%2C+randomised%2C+placebo-controlled%2C+phase+3+trial&doi=10.1016%2FS0140-6736%2812%2961900-X"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Regorafenib monotherapy for previously treated metastatic colorectal cancer (CORRECT): an international, multicentre, randomised, placebo-controlled, phase 3 trial</span></div><div class="casAuthors">Grothey, Axel; Cutsem, Eric Van; Sobrero, Alberto; Siena, Salvatore; Falcone, Alfredo; Ychou, Marc; Humblet, Yves; Bouche, Olivier; Mineur, Laurent; Barone, Carlo; Adenis, Antoine; Tabernero, Josep; Yoshino, Takayuki; Lenz, Heinz-Josef; Goldberg, Richard M.; Sargent, Daniel J.; Cihon, Frank; Cupit, Lisa; Wagner, Andrea; Laurent, Dirk</div><div class="citationInfo"><span class="NLM_cas:title">Lancet</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">381</span>
        (<span class="NLM_cas:issue">9863</span>),
    <span class="NLM_cas:pages">303-312</span>CODEN:
                <span class="NLM_cas:coden">LANCAO</span>;
        ISSN:<span class="NLM_cas:issn">0140-6736</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">No treatment options are available for patients with metastatic colorectal cancer that progresses after all approved std. therapies, but many patients maintain a good performance status and could be candidates for further therapy.  An international phase 3 trial was done to assess the multikinase inhibitor regorafenib in these patients.  We did this trial at 114 centers in 16 countries.  Patients with documented metastatic colorectal cancer and progression during or within 3 mo after the last std. therapy were randomized (in a 2:1 ratio; by computer-generated randomization list and interactive voice response system; preallocated block design (block size six); stratified by previous treatment with VEGF-targeting drugs, time from diagnosis of metastatic disease, and geog. region) to receive best supportive care plus oral regorafenib 160 mg or placebo once daily, for the first 3 wk of each 4 wk cycle.  The primary endpoint was overall survival.  The study sponsor, participants, and investigators were masked to treatment assignment.  Efficacy analyses were by intention to treat.  This trial is registered at ClinicalTrials.gov, no. NCT01103323.Between Apr. 30, 2010, and March 22, 2011, 1052 patients were screened, 760 patients were randomized to receive regorafenib (n=505) or placebo (n=255), and 753 patients initiated treatment (regorafenib n=500; placebo n=253; population for safety analyses).  The primary endpoint of overall survival was met at a preplanned interim anal.; data cutoff was on July 21, 2011.  Median overall survival was 6·4 mo in the regorafenib group vs. 5·0 mo in the placebo group (hazard ratio 0·77; 95% CI 0·64-0·94; one-sided p=0·0052).  Treatment-related adverse events occurred in 465 (93%) patients assigned regorafenib and in 154 (61%) of those assigned placebo.  The most common adverse events of grade three or higher related to regorafenib were hand-foot skin reaction (83 patients, 17%), fatigue (48, 10%), diarrhea (36, 7%), hypertension (36, 7%), and rash or desquamation (29, 6%).Regorafenib is the first small-mol. multikinase inhibitor with survival benefits in metastatic colorectal cancer which has progressed after all std. therapies.  The present study provides evidence for a continuing role of targeted treatment after disease progression, with regorafenib offering a potential new line of therapy in this treatment-refractory population.Bayer HealthCare Pharmaceuticals.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGohw55Jw5C_4LVg90H21EOLACvtfcHk0lipsSt0xF_KWw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xhslarsr%252FE&md5=ec2d5286add1c1837bfe7bdb50817b06</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1016%2FS0140-6736%2812%2961900-X&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0140-6736%252812%252961900-X%26sid%3Dliteratum%253Aachs%26aulast%3DGrothey%26aufirst%3DA.%26aulast%3DCutsem%26aufirst%3DE.%2BV.%26aulast%3DSobrero%26aufirst%3DA.%26aulast%3DSiena%26aufirst%3DS.%26aulast%3DFalcone%26aufirst%3DA.%26aulast%3DYchou%26aufirst%3DM.%26aulast%3DHumblet%26aufirst%3DY.%26aulast%3DBouche%26aufirst%3DO.%26aulast%3DMineur%26aufirst%3DL.%26aulast%3DBarone%26aufirst%3DC.%26aulast%3DAdenis%26aufirst%3DA.%26aulast%3DTabernero%26aufirst%3DJ.%26aulast%3DYoshino%26aufirst%3DT.%26aulast%3DLenz%26aufirst%3DH.-J.%26aulast%3DGoldberg%26aufirst%3DR.%2BM%26aulast%3DSargent%26aufirst%3DD.%2BJ%26aulast%3DCihon%26aufirst%3DF.%26aulast%3DCupit%26aufirst%3DL.%26aulast%3DWagner%26aufirst%3DA.%26aulast%3DLaurent%26aufirst%3DD.%26atitle%3DRegorafenib%2520monotherapy%2520for%2520previously%2520treated%2520metastatic%2520colorectal%2520cancer%2520%2528CORRECT%2529%253A%2520an%2520international%252C%2520multicentre%252C%2520randomised%252C%2520placebo-controlled%252C%2520phase%25203%2520trial%26jtitle%3DLancet%26date%3D2013%26volume%3D381%26spage%3D303%26epage%3D312%26doi%3D10.1016%2FS0140-6736%2812%2961900-X" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Demetri, G. D</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reichardt, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kang, Y.-K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blay, J.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rutkowski, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gelderblom, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hohenberger, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leahy, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">von Mehren, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Joensuu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Badalamenti, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blackstein, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Le Cesne, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schoffski, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maki, R. G</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bauer, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen, B. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nishida, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chung, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kappeler, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuss, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laurent, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Casali, P. G</span></span> <span> </span><span class="NLM_article-title">Efficacy and safety of regorafenib for advanced gastrointestinal stromal tumours after failure of imatinib and sunitinib (GRID): an international, multicentre, randomised, placebo-controlled, phase 3 trial</span>. <i>Lancet</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>381</i></span>,  <span class="NLM_fpage">295</span>– <span class="NLM_lpage">302</span>, <span class="refDoi"> DOI: 10.1016/S0140-6736(12)61857-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=10.1016%2FS0140-6736%2812%2961857-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=23177515" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=1%3ACAS%3A528%3ADC%252BC38XhslarsLvL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=381&publication_year=2013&pages=295-302&author=G.+D+Demetriauthor=P.+Reichardtauthor=Y.-K.+Kangauthor=J.-Y.+Blayauthor=P.+Rutkowskiauthor=H.+Gelderblomauthor=P.+Hohenbergerauthor=M.+Leahyauthor=M.+von+Mehrenauthor=H.+Joensuuauthor=G.+Badalamentiauthor=M.+Blacksteinauthor=A.+Le+Cesneauthor=P.+Schoffskiauthor=R.+G+Makiauthor=S.+Bauerauthor=B.+B.+Nguyenauthor=J.+Xuauthor=T.+Nishidaauthor=J.+Chungauthor=C.+Kappelerauthor=I.+Kussauthor=D.+Laurentauthor=P.+G+Casali&title=Efficacy+and+safety+of+regorafenib+for+advanced+gastrointestinal+stromal+tumours+after+failure+of+imatinib+and+sunitinib+%28GRID%29%3A+an+international%2C+multicentre%2C+randomised%2C+placebo-controlled%2C+phase+3+trial&doi=10.1016%2FS0140-6736%2812%2961857-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Efficacy and safety of regorafenib for advanced gastrointestinal stromal tumours after failure of imatinib and sunitinib (GRID): an international, multicentre, randomised, placebo-controlled, phase 3 trial</span></div><div class="casAuthors">Demetri, George D.; Reichardt, Peter; Kang, Yoon-Koo; Blay, Jean-Yves; Rutkowski, Piotr; Gelderblom, Hans; Hohenberger, Peter; Leahy, Michael; von Mehren, Margaret; Joensuu, Heikki; Badalamenti, Giuseppe; Blackstein, Martin; Le Cesne, Axel; Schoffski, Patrick; Maki, Robert G.; Bauer, Sebastian; Nguyen, Binh Bui; Xu, Jianming; Nishida, Toshirou; Chung, John; Kappeler, Christian; Kuss, Iris; Laurent, Dirk; Casali, Paolo G.</div><div class="citationInfo"><span class="NLM_cas:title">Lancet</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">381</span>
        (<span class="NLM_cas:issue">9863</span>),
    <span class="NLM_cas:pages">295-302</span>CODEN:
                <span class="NLM_cas:coden">LANCAO</span>;
        ISSN:<span class="NLM_cas:issn">0140-6736</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Until now, only imatinib and sunitinib have proven clin. benefit in patients with gastrointestinal stromal tumors (GIST), but almost all metastatic GIST eventually develop resistance to these agents, resulting in fatal disease progression.  We aimed to assess efficacy and safety of regorafenib in patients with metastatic or unresectable GIST progressing after failure of at least imatinib and sunitinib.We did this phase 3 trial at 57 hospitals in 17 countries.  Patients with histol. confirmed, metastatic or unresectable GIST, with failure of at least previous imatinib and sunitinib were randomised in a 2:1 ratio (by computer-generated randomisation list and interactive voice response system; preallocated block design (block size 12); stratified by treatment line and geog. region) to receive either oral regorafenib 160 mg daily or placebo, plus best supportive care in both groups, for the first 3 wk of each 4 wk cycle.  The study sponsor, participants, and investigators were masked to treatment assignment.  The primary endpoint was progression-free survival (PFS).  At disease progression, patients assigned placebo could crossover to open-label regorafenib.  Analyses were by intention to treat.  This trial is registered with ClinicalTrials.gov, no. NCT01271712.From Jan 4, to Aug 18, 2011, 240 patients were screened and 199 were randomised to receive regorafenib (n=133) or matching placebo (n=66).  Data cutoff was Jan 26, 2012.  Median PFS per independent blinded central review was 4·8 mo (IQR 1·4-9·2) for regorafenib and 0·9 mo (0·9-1·8) for placebo (hazard ratio [HR] 0·27, 95% CI 0·19-0·39; p<0·0001).  After progression, 56 patients (85%) assigned placebo crossed over to regorafenib.  Drug-related adverse events were reported in 130 (98%) patients assigned regorafenib and 45 (68%) patients assigned placebo.  The most common regorafenib-related adverse events of grade 3 or higher were hypertension (31 of 132, 23%), hand-foot skin reaction (26 of 132, 20%), and diarrhoea (seven of 132, 5%).The results of this study show that oral regorafenib can provide a significant improvement in progression-free survival compared with placebo in patients with metastatic GIST after progression on std. treatments.  As far as we are aware, this is the first clin. trial to show benefit from a kinase inhibitor in this highly refractory population of patients.Bayer HealthCare Pharmaceuticals.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoN2GIfYXBPyLVg90H21EOLACvtfcHk0ljy1b4zDr2yyQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhslarsLvL&md5=f8ee42424a9b3bd7558b1a2f1461af33</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1016%2FS0140-6736%2812%2961857-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0140-6736%252812%252961857-1%26sid%3Dliteratum%253Aachs%26aulast%3DDemetri%26aufirst%3DG.%2BD%26aulast%3DReichardt%26aufirst%3DP.%26aulast%3DKang%26aufirst%3DY.-K.%26aulast%3DBlay%26aufirst%3DJ.-Y.%26aulast%3DRutkowski%26aufirst%3DP.%26aulast%3DGelderblom%26aufirst%3DH.%26aulast%3DHohenberger%26aufirst%3DP.%26aulast%3DLeahy%26aufirst%3DM.%26aulast%3Dvon%2BMehren%26aufirst%3DM.%26aulast%3DJoensuu%26aufirst%3DH.%26aulast%3DBadalamenti%26aufirst%3DG.%26aulast%3DBlackstein%26aufirst%3DM.%26aulast%3DLe%2BCesne%26aufirst%3DA.%26aulast%3DSchoffski%26aufirst%3DP.%26aulast%3DMaki%26aufirst%3DR.%2BG%26aulast%3DBauer%26aufirst%3DS.%26aulast%3DNguyen%26aufirst%3DB.%2BB.%26aulast%3DXu%26aufirst%3DJ.%26aulast%3DNishida%26aufirst%3DT.%26aulast%3DChung%26aufirst%3DJ.%26aulast%3DKappeler%26aufirst%3DC.%26aulast%3DKuss%26aufirst%3DI.%26aulast%3DLaurent%26aufirst%3DD.%26aulast%3DCasali%26aufirst%3DP.%2BG%26atitle%3DEfficacy%2520and%2520safety%2520of%2520regorafenib%2520for%2520advanced%2520gastrointestinal%2520stromal%2520tumours%2520after%2520failure%2520of%2520imatinib%2520and%2520sunitinib%2520%2528GRID%2529%253A%2520an%2520international%252C%2520multicentre%252C%2520randomised%252C%2520placebo-controlled%252C%2520phase%25203%2520trial%26jtitle%3DLancet%26date%3D2013%26volume%3D381%26spage%3D295%26epage%3D302%26doi%3D10.1016%2FS0140-6736%2812%2961857-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bruix, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Merle, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Granito, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Y.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bodoky, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pracht, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yokosuka, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosmorduc, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Breder, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerolami, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Masi, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ross, P. J</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bronowicki, J.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ollivier-Hourmand, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kudo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, A.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Llovet, J. M</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Finn, R. S</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">LeBerre, M.-A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baumhauer, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meinhardt, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, G.</span></span> <span> </span><span class="NLM_article-title">Regorafenib for patients with hepatocellular carcinoma who progressed on sorafenib treatment (RESORCE): a randomised, double-blind, placebo-controlled, phase 3 trial</span>. <i>Lancet</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>389</i></span>,  <span class="NLM_fpage">56</span>– <span class="NLM_lpage">66</span>, <span class="refDoi"> DOI: 10.1016/S0140-6736(16)32453-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=10.1016%2FS0140-6736%2816%2932453-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=27932229" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=1%3ACAS%3A528%3ADC%252BC28XitVajsrrN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=389&publication_year=2017&pages=56-66&author=J.+Bruixauthor=S.+Qinauthor=P.+Merleauthor=A.+Granitoauthor=Y.-H.+Huangauthor=G.+Bodokyauthor=M.+Prachtauthor=O.+Yokosukaauthor=O.+Rosmorducauthor=V.+Brederauthor=R.+Gerolamiauthor=G.+Masiauthor=P.+J+Rossauthor=T.+Songauthor=J.-P.+Bronowickiauthor=I.+Ollivier-Hourmandauthor=M.+Kudoauthor=A.-L.+Chengauthor=J.+M+Llovetauthor=R.+S+Finnauthor=M.-A.+LeBerreauthor=A.+Baumhauerauthor=G.+Meinhardtauthor=G.+Han&title=Regorafenib+for+patients+with+hepatocellular+carcinoma+who+progressed+on+sorafenib+treatment+%28RESORCE%29%3A+a+randomised%2C+double-blind%2C+placebo-controlled%2C+phase+3+trial&doi=10.1016%2FS0140-6736%2816%2932453-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Regorafenib for patients with hepatocellular carcinoma who progressed on sorafenib treatment (RESORCE): a randomised, double-blind, placebo-controlled, phase 3 trial</span></div><div class="casAuthors">Bruix, Jordi; Qin, Shukui; Merle, Philippe; Granito, Alessandro; Huang, Yi-Hsiang; Bodoky, Gyorgy; Pracht, Marc; Yokosuka, Osamu; Rosmorduc, Olivier; Breder, Valeriy; Gerolami, Rene; Masi, Gianluca; Ross, Paul J.; Song, Tianqiang; Bronowicki, Jean-Pierre; Ollivier-Hourmand, Isabelle; Kudo, Masatoshi; Cheng, Ann-Lii; Llovet, Josep M.; Finn, Richard S.; LeBerre, Marie-Aude; Baumhauer, Annette; Meinhardt, Gerold; Han, Guohong</div><div class="citationInfo"><span class="NLM_cas:title">Lancet</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">389</span>
        (<span class="NLM_cas:issue">10064</span>),
    <span class="NLM_cas:pages">56-66</span>CODEN:
                <span class="NLM_cas:coden">LANCAO</span>;
        ISSN:<span class="NLM_cas:issn">0140-6736</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">There are no systemic treatments for patients with hepatocellular carcinoma (HCC) whose disease progresses during sorafenib treatment.  We aimed to assess the efficacy and safety of regorafenib in patients with HCC who have progressed during sorafenib treatment.  In this randomized, double-blind, parallel-group, phase 3 trial done at 152 sites in 21 countries, adults with HCC who tolerated sorafenib (≥400 mg/day for ≥20 of last 28 days of treatment), progressed on sorafenib, and had Child-Pugh A liver function were enrolled.  Participants were randomly assigned (2:1) by a computer-generated randomization list and interactive voice response system and stratified by geog. region, Eastern Cooperative Oncol. Group performance status, macrovascular invasion, extrahepatic disease, and α-fetoprotein level to best supportive care plus oral regorafenib 160 mg or placebo once daily during weeks 1-3 of each 4-wk cycle.  Investigators, patients, and the funder were masked to treatment assignment.  The primary endpoint was overall survival (defined as time from randomization to death due to any cause) and analyzed by intention to treat.  This trial is registered with ClinicalTrials.gov, no. NCT01774344.  Between May 14, 2013, and Dec 31, 2015, 843 patients were screened, of whom 573 were enrolled and randomized (379 to regorafenib and 194 to placebo; population for efficacy analyses), and 567 initiated treatment (374 received regorafenib and 193 received placebo; population for safety analyses).  Regorafenib improved overall survival with a hazard ratio of 0·63 (95% CI 0·50-0·79; one-sided p<0·0001); median survival was 10·6 mo (95% CI 9·1-12·1) for regorafenib vs. 7·8 mo (6·3-8·8) for placebo.  Adverse events were reported in all regorafenib recipients (374 [100%] of 374) and 179 (93%) of 193 placebo recipients.  The most common clin. relevant grade 3 or 4 treatment-emergent events were hypertension (57 patients [15%] in the regorafenib group vs nine patients [5%] in the placebo group), hand-foot skin reaction (47 patients [13%] vs one [1%]), fatigue (34 patients [9%] vs nine patients [5%]), and diarrhea (12 patients [3%] vs no patients).  Of the 88 deaths (grade 5 adverse events) reported during the study (50 patients [13%] assigned to regorafenib and 38 [20%] assigned to placebo), seven (2%) were considered by the investigator to be related to study drug in the regorafenib group and two (1%) in the placebo group, including two patients (1%) with hepatic failure in the placebo group.  Regorafenib is the only systemic treatment shown to provide survival benefit in HCC patients progressing on sorafenib treatment.  Future trials should explore combinations of regorafenib with other systemic agents and third-line treatments for patients who fail or who do not tolerate the sequence of sorafenib and regorafenib.Bayer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrY-TirMzLgvrVg90H21EOLACvtfcHk0ljy1b4zDr2yyQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitVajsrrN&md5=bdfcd7a16ed161777b9960bb52c0a719</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1016%2FS0140-6736%2816%2932453-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0140-6736%252816%252932453-9%26sid%3Dliteratum%253Aachs%26aulast%3DBruix%26aufirst%3DJ.%26aulast%3DQin%26aufirst%3DS.%26aulast%3DMerle%26aufirst%3DP.%26aulast%3DGranito%26aufirst%3DA.%26aulast%3DHuang%26aufirst%3DY.-H.%26aulast%3DBodoky%26aufirst%3DG.%26aulast%3DPracht%26aufirst%3DM.%26aulast%3DYokosuka%26aufirst%3DO.%26aulast%3DRosmorduc%26aufirst%3DO.%26aulast%3DBreder%26aufirst%3DV.%26aulast%3DGerolami%26aufirst%3DR.%26aulast%3DMasi%26aufirst%3DG.%26aulast%3DRoss%26aufirst%3DP.%2BJ%26aulast%3DSong%26aufirst%3DT.%26aulast%3DBronowicki%26aufirst%3DJ.-P.%26aulast%3DOllivier-Hourmand%26aufirst%3DI.%26aulast%3DKudo%26aufirst%3DM.%26aulast%3DCheng%26aufirst%3DA.-L.%26aulast%3DLlovet%26aufirst%3DJ.%2BM%26aulast%3DFinn%26aufirst%3DR.%2BS%26aulast%3DLeBerre%26aufirst%3DM.-A.%26aulast%3DBaumhauer%26aufirst%3DA.%26aulast%3DMeinhardt%26aufirst%3DG.%26aulast%3DHan%26aufirst%3DG.%26atitle%3DRegorafenib%2520for%2520patients%2520with%2520hepatocellular%2520carcinoma%2520who%2520progressed%2520on%2520sorafenib%2520treatment%2520%2528RESORCE%2529%253A%2520a%2520randomised%252C%2520double-blind%252C%2520placebo-controlled%252C%2520phase%25203%2520trial%26jtitle%3DLancet%26date%3D2017%26volume%3D389%26spage%3D56%26epage%3D66%26doi%3D10.1016%2FS0140-6736%2816%2932453-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wilhelm, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dumas, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adnane, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lynch, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carter, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schütz, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thierauch, K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zopf, D.</span></span> <span> </span><span class="NLM_article-title">Regorafenib (BAY 73-4506): a new oral multikinase inhibitor of angiogenic, stromal and oncogenic receptor tyrosine kinases with potent preclinical antitumor activity</span>. <i>Int. J. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>129</i></span>,  <span class="NLM_fpage">245</span>– <span class="NLM_lpage">255</span>, <span class="refDoi"> DOI: 10.1002/ijc.25864</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=10.1002%2Fijc.25864" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=21170960" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=1%3ACAS%3A528%3ADC%252BC3MXltFGlu7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=129&publication_year=2011&pages=245-255&author=S.+M.+Wilhelmauthor=J.+Dumasauthor=L.+Adnaneauthor=M.+Lynchauthor=C.+A.+Carterauthor=G.+Sch%C3%BCtzauthor=K.+H.+Thierauchauthor=D.+Zopf&title=Regorafenib+%28BAY+73-4506%29%3A+a+new+oral+multikinase+inhibitor+of+angiogenic%2C+stromal+and+oncogenic+receptor+tyrosine+kinases+with+potent+preclinical+antitumor+activity&doi=10.1002%2Fijc.25864"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Regorafenib (BAY 73-4506): A new oral multikinase inhibitor of angiogenic, stromal and oncogenic receptor tyrosine kinases with potent preclinical antitumor activity</span></div><div class="casAuthors">Wilhelm, Scott M.; Dumas, Jacques; Adnane, Lila; Lynch, Mark; Carter, Christopher A.; Schuetz, Gunnar; Thierauch, Karl-Heinz; Zopf, Dieter</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Cancer</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">129</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">245-255</span>CODEN:
                <span class="NLM_cas:coden">IJCNAW</span>;
        ISSN:<span class="NLM_cas:issn">0020-7136</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Liss, Inc.</span>)
        </div><div class="casAbstract">Angiogenesis, a crit. driver of tumor development, is controlled by interconnected signaling pathways.  Vascular endothelial growth factor receptor (VEGFR) 2 and tyrosine kinase with Ig and epidermal growth factor homol. domain 2 play crucial roles in the biol. of normal and tumor vasculature.  Regorafenib (BAY 73-4506), a novel oral multikinase inhibitor, potently inhibits these endothelial cell kinases in biochem. and cellular kinase phosphorylation assays.  Furthermore, regorafenib inhibits addnl. angiogenic kinases (VEGFR1/3, platelet-derived growth factor receptor-β and fibroblast growth factor receptor 1) and the mutant oncogenic kinases KIT, RET and B-RAF.  The antiangiogenic effect of regorafenib was demonstrated in vivo by dynamic contrast-enhanced magnetic resonance imaging.  Regorafenib administered once orally at 10 mg/kg significantly decreased the extravasation of Gadomer in the vasculature of rat GS9L glioblastoma tumor xenografts.  In a daily (qd)×4 dosing study, the pharmacodynamic effects persisted for 48 h after the last dosing and correlated with tumor growth inhibition (TGI).  A significant redn. in tumor microvessel area was obsd. in a human colorectal xenograft after qd×5 dosing at 10 and 30 mg/kg.  Regorafenib exhibited potent dose-dependent TGI in various preclin. human xenograft models in mice, with tumor shrinkages obsd. in breast MDA-MB-231 and renal 786-O carcinoma models.  Pharmacodynamic analyses of the breast model revealed strong redn. in staining of proliferation marker Ki-67 and phosphorylated extracellular regulated kinases 1/2.  These data demonstrate that regorafenib is a well-tolerated, orally active multikinase inhibitor with a distinct target profile that may have therapeutic benefit in human malignancies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrtQpQXfCq04rVg90H21EOLACvtfcHk0lgoVAmRWH9XSw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXltFGlu7c%253D&md5=13eee29ef1f1f03fd1ed311ea613f566</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1002%2Fijc.25864&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fijc.25864%26sid%3Dliteratum%253Aachs%26aulast%3DWilhelm%26aufirst%3DS.%2BM.%26aulast%3DDumas%26aufirst%3DJ.%26aulast%3DAdnane%26aufirst%3DL.%26aulast%3DLynch%26aufirst%3DM.%26aulast%3DCarter%26aufirst%3DC.%2BA.%26aulast%3DSch%25C3%25BCtz%26aufirst%3DG.%26aulast%3DThierauch%26aufirst%3DK.%2BH.%26aulast%3DZopf%26aufirst%3DD.%26atitle%3DRegorafenib%2520%2528BAY%252073-4506%2529%253A%2520a%2520new%2520oral%2520multikinase%2520inhibitor%2520of%2520angiogenic%252C%2520stromal%2520and%2520oncogenic%2520receptor%2520tyrosine%2520kinases%2520with%2520potent%2520preclinical%2520antitumor%2520activity%26jtitle%3DInt.%2520J.%2520Cancer%26date%3D2011%26volume%3D129%26spage%3D245%26epage%3D255%26doi%3D10.1002%2Fijc.25864" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Koul, H. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pal, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koul, S.</span></span> <span> </span><span class="NLM_article-title">Role of p38 MAP kinase signal transduction in solid tumors</span>. <i>Genes Cancer</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">342</span>– <span class="NLM_lpage">359</span>, <span class="refDoi"> DOI: 10.1177/1947601913507951</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=10.1177%2F1947601913507951" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=24349632" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsVGktbfP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2013&pages=342-359&author=H.+K.+Koulauthor=M.+Palauthor=S.+Koul&title=Role+of+p38+MAP+kinase+signal+transduction+in+solid+tumors&doi=10.1177%2F1947601913507951"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Role of p38 MAP kinase signal transduction in solid tumors</span></div><div class="casAuthors">Koul, Hari K.; Pal, Mintu; Koul, Sweaty</div><div class="citationInfo"><span class="NLM_cas:title">Genes & Cancer</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">9-10</span>),
    <span class="NLM_cas:pages">342-359, 18 pp.</span>CODEN:
                <span class="NLM_cas:coden">GCEAAY</span>;
        ISSN:<span class="NLM_cas:issn">1947-6019</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">A review.  Mitogen-activated protein kinases (MAPKs) mediate a wide variety of cellular behaviors in response to extracellular stimuli.  One of the main subgroups, the p38 MAP kinases, has been implicated in a wide range of complex biol. processes, such as cell proliferation, cell differentiation, cell death, cell migration, and invasion.  Dysregulation of p38 MAPK levels in patients are assocd. with advanced stages and short survival in cancer patients (e.g., prostate, breast, bladder, liver, and lung cancer).  P38 MAPK plays a dual role as a regulator of cell death, and it can either mediate cell survival or cell death depending not only on the type of stimulus but also in a cell type specific manner.  In addn. to modulating cell survival, an essential role of p38 MAPK in modulation of cell migration and invasion offers a distinct opportunity to target this pathway with respect to tumor metastasis.  The specific function of p38 MAPK appears to depend not only on the cell type but also on the stimuli and/or the isoform that is activated.  P38 MAPK signaling pathway is activated in response to diverse stimuli and mediates its function by components downstream of p38.  Extrapolation of the knowledge gained from lab. findings is essential to address the clin. significance of p38 MAPK signaling pathways.  The goal of this review is to provide an overview on recent progress made in defining the functions of p38 MAPK pathways with respect to solid tumor biol. and generate testable hypothesis with respect to the role of p38 MAPK as an attractive target for intervention of solid tumors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoB6A4yxL-gLLVg90H21EOLACvtfcHk0lgoVAmRWH9XSw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsVGktbfP&md5=a7803ce5e47e58b277f614616e526209</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1177%2F1947601913507951&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1947601913507951%26sid%3Dliteratum%253Aachs%26aulast%3DKoul%26aufirst%3DH.%2BK.%26aulast%3DPal%26aufirst%3DM.%26aulast%3DKoul%26aufirst%3DS.%26atitle%3DRole%2520of%2520p38%2520MAP%2520kinase%2520signal%2520transduction%2520in%2520solid%2520tumors%26jtitle%3DGenes%2520Cancer%26date%3D2013%26volume%3D4%26spage%3D342%26epage%3D359%26doi%3D10.1177%2F1947601913507951" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lange, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lo, H.-W.</span></span> <span> </span><span class="NLM_article-title">Inhibiting TRK proteins in clinical cancer therapy</span>. <i>Cancers</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">105</span>, <span class="refDoi"> DOI: 10.3390/cancers10040105</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=10.3390%2Fcancers10040105" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhtV2gtL%252FI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2018&pages=105&author=A.+Langeauthor=H.-W.+Lo&title=Inhibiting+TRK+proteins+in+clinical+cancer+therapy&doi=10.3390%2Fcancers10040105"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibiting TRK proteins in clinical cancer therapy</span></div><div class="casAuthors">Lange, Allison M.; Lo, Hui-Wen</div><div class="citationInfo"><span class="NLM_cas:title">Cancers</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">105/1-105/15</span>CODEN:
                <span class="NLM_cas:coden">CANCCT</span>;
        ISSN:<span class="NLM_cas:issn">2072-6694</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">Gene rearrangements resulting in the aberrant activity of tyrosine kinases have been identified as drivers of oncogenesis in a variety of cancers.  The tropomyosin receptor kinase (TRK) family of tyrosine receptor kinases is emerging as an important target for cancer therapeutics.  The TRK family contains three members, TRKA, TRKB, and TRKC, and these proteins are encoded by the genes NTRK1, NTRK2, and NTRK3, resp.  To activate TRK receptors, neurotrophins bind to the extracellular region stimulating dimerization, phosphorylation, and activation of downstream signaling pathways.  Major known downstream pathways include RAS/MAPK/ERK, PLCγ, and PI3K/Akt.  While being rare in most cancers, TRK fusions with other proteins have been well-established as oncogenic events in specific malignancies, including glioblastoma, papillary thyroid carcinoma, and secretory breast carcinomas.  TRK protein amplification as well as alternative splicing events have also been described as contributors to cancer pathogenesis.  For patients harboring alterations in TRK expression or activity, TRK inhibition emerges as an important therapeutic target.  To date, multiple trials testing TRK-inhibiting compds. in various cancers are underway.  In this review, we will summarize the current therapeutic trials for neoplasms involving NTKR gene alterations, as well as the promises and setbacks that are assocd. with targeting gene fusions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqvepDLt-_JaLVg90H21EOLACvtfcHk0liex3LjB4BUXw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhtV2gtL%252FI&md5=f421f06adf15f9357d88b80bf44479e8</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.3390%2Fcancers10040105&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fcancers10040105%26sid%3Dliteratum%253Aachs%26aulast%3DLange%26aufirst%3DA.%26aulast%3DLo%26aufirst%3DH.-W.%26atitle%3DInhibiting%2520TRK%2520proteins%2520in%2520clinical%2520cancer%2520therapy%26jtitle%3DCancers%26date%3D2018%26volume%3D10%26spage%3D105%26doi%3D10.3390%2Fcancers10040105" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sirvent, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lafitte, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roche, S.</span></span> <span> </span><span class="NLM_article-title">DDR1 inhibition as a new therapeutic strategy for colorectal cancer</span>. <i>Mol. Cell Oncol</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>5</i></span>, <span class="NLM_elocation-id">e1465882</span> <span class="refDoi"> DOI: 10.1080/23723556.2018.1465882</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=10.1080%2F23723556.2018.1465882" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=30250919" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtlyrtrbJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2018&author=A.+Sirventauthor=M.+Lafitteauthor=S.+Roche&title=DDR1+inhibition+as+a+new+therapeutic+strategy+for+colorectal+cancer&doi=10.1080%2F23723556.2018.1465882"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">DDR1 inhibition as a new therapeutic strategy for colorectal cancer</span></div><div class="casAuthors">Sirvent, Audrey; Lafitte, Marie; Roche, Serge</div><div class="citationInfo"><span class="NLM_cas:title">Molecular & Cellular Oncology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">e1465882/1-e1465882/3</span>CODEN:
                <span class="NLM_cas:coden">MCOOCO</span>;
        ISSN:<span class="NLM_cas:issn">2372-3556</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis, Inc.</span>)
        </div><div class="casAbstract">The clin. management of metastatic colorectal cancer (mCRC) is still a major challenge.  Recently, we discovered that nilotinib, an approved treatment for chronic myeloid leukemia, inhibits invasive and metastatic properties of CRC cells by targeting the kinase activity of receptor for collagens DDR1 (Discoidin Domain Receptor tyrosine kinase 1), suggesting that nilotinib could be an effective strategy to treat mCRC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpyexaLpIY9RLVg90H21EOLACvtfcHk0liex3LjB4BUXw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtlyrtrbJ&md5=794bf62b12a6327947f23c5a610e3bf9</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1080%2F23723556.2018.1465882&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F23723556.2018.1465882%26sid%3Dliteratum%253Aachs%26aulast%3DSirvent%26aufirst%3DA.%26aulast%3DLafitte%26aufirst%3DM.%26aulast%3DRoche%26aufirst%3DS.%26atitle%3DDDR1%2520inhibition%2520as%2520a%2520new%2520therapeutic%2520strategy%2520for%2520colorectal%2520cancer%26jtitle%3DMol.%2520Cell%2520Oncol%26date%3D2018%26volume%3D5%26doi%3D10.1080%2F23723556.2018.1465882" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ren, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, K.</span></span> <span> </span><span class="NLM_article-title">Small molecule discoidin domain receptor kinase inhibitors and potential medical applications</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">3287</span>– <span class="NLM_lpage">3301</span>, <span class="refDoi"> DOI: 10.1021/jm5012319</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm5012319" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=1%3ACAS%3A528%3ADC%252BC2MXns1ylsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=3287-3301&author=Y.+Liauthor=X.+Luauthor=X.+Renauthor=K.+Ding&title=Small+molecule+discoidin+domain+receptor+kinase+inhibitors+and+potential+medical+applications&doi=10.1021%2Fjm5012319"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Small Molecule Discoidin Domain Receptor Kinase Inhibitors and Potential Medical Applications</span></div><div class="casAuthors">Li, Yupeng; Lu, Xiaoyun; Ren, Xiaomei; Ding, Ke</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">3287-3301</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Discoidin domain receptors (DDRs) are members of the transmembrane receptor tyrosine kinase (RTK) superfamily which are distinguished from others by the presence of a discoidin motif in the extracellular domain and their utilization of collagens as internal ligands.  Two types of DDRs, DDR1 and DDR2, have been identified with distinct expression profiles and ligand specificities.  These DDRs play important roles in the regulation of fundamental cellular process, such as proliferation, survival, differentiation, adhesion, and matrix remodeling.  They have also been closely linked to a no. of human diseases, including various fibrotic disorders, atherosclerosis, and cancer.  As a consequence, DDRs have been considered as novel potential mol. targets for drug discovery and increasing efforts are being devoted to the identification of new small mol. inhibitors targeting the receptors.  In this review, we offer a contemporary overview on the discovery of DDRs inhibitors and their potential medical application for the treatment of cancer and inflammation related disorders.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoVUPSkCQYF-LVg90H21EOLACvtfcHk0liex3LjB4BUXw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXns1ylsQ%253D%253D&md5=71249675e4ed65d538956f8c9cb7d387</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1021%2Fjm5012319&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm5012319%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DY.%26aulast%3DLu%26aufirst%3DX.%26aulast%3DRen%26aufirst%3DX.%26aulast%3DDing%26aufirst%3DK.%26atitle%3DSmall%2520molecule%2520discoidin%2520domain%2520receptor%2520kinase%2520inhibitors%2520and%2520potential%2520medical%2520applications%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D3287%26epage%3D3301%26doi%3D10.1021%2Fjm5012319" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tanihara, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Inoue, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamamoto, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuwayama, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abe, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Araie, M.</span></span> <span> </span><span class="NLM_article-title">Phase 2 randomized clinical study of a Rho kinase inhibitor, K-115, in primary open-angle glaucoma and ocular hypertension</span>. <i>Am. J. Ophthalmol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>156</i></span>,  <span class="NLM_fpage">731</span>– <span class="NLM_lpage">736</span>, <span class="refDoi"> DOI: 10.1016/j.ajo.2013.05.016</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=10.1016%2Fj.ajo.2013.05.016" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=23831221" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtVKrur7K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=156&publication_year=2013&pages=731-736&author=H.+Taniharaauthor=T.+Inoueauthor=T.+Yamamotoauthor=Y.+Kuwayamaauthor=H.+Abeauthor=M.+Araie&title=Phase+2+randomized+clinical+study+of+a+Rho+kinase+inhibitor%2C+K-115%2C+in+primary+open-angle+glaucoma+and+ocular+hypertension&doi=10.1016%2Fj.ajo.2013.05.016"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Phase 2 Randomized Clinical Study of a Rho Kinase Inhibitor, K-115, in Primary Open-Angle Glaucoma and Ocular Hypertension</span></div><div class="casAuthors">Tanihara, Hidenobu; Inoue, Toshihiro; Yamamoto, Tetsuya; Kuwayama, Yasuaki; Abe, Haruki; Araie, Makoto</div><div class="citationInfo"><span class="NLM_cas:title">American Journal of Ophthalmology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">156</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">731-736.e2</span>CODEN:
                <span class="NLM_cas:coden">AJOPAA</span>;
        ISSN:<span class="NLM_cas:issn">0002-9394</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">To identify the optimal dose of a novel Rho kinase inhibitor, K-115, by assessing dose dependency of the intraocular pressure (IOP)-lowering effects and the safety in patients with primary open-angle glaucoma or ocular hypertension.  Multicenter, prospective, randomized, placebo-controlled, double-masked, parallel group comparison clin. study.  After appropriate washout periods, 210 patients with primary open-angle glaucoma or ocular hypertension were subdivided into 4 groups and were treated with K-115 in concns. of 0.1%, 0.2%, and 0.4% or placebo twice daily for 8 wk.  The dose response of IOP redn. and the incidence of adverse events by K-115 or placebo were investigated.The mean baseline IOP was between 23.0 and 23.4 mm Hg.  The mean IOP redns. of the last visit from baseline were -2.2 mm Hg, -3.4 mm Hg, -3.2 mm Hg, and -3.5 mm Hg, resp., in the placebo, 0.1%, 0.2%, and 0.4% groups at before instillation (9:00); -2.5 mm Hg, -3.7 mm Hg, -4.2 mm Hg, and -4.5 mm Hg at 2 h after instillation (11:00); and -1.9 mm Hg, -3.2 mm Hg, -2.7 mm Hg, and -3.1 mm Hg at 8 h after instillation (17:00).  The dose-dependent IOP-lowering effect of K-115 was statistically significant at all time points.  Also, conjunctival hyperemia was found in 7 (13.0%) of 54 patients for placebo, 23 (43.4%) of 53 patients for the 0.1% group, 31 (57.4%) of 54 patients for the 0.2% group, and 32 (65.3%) of 49 patients for the 0.4% group.  On the basis of this dose-response study, K-115 0.4% has been selected to be the optimal dose and has the potential to be a promising new agent for glaucoma to control 24-h IOP by twice-daily dosing.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrq9Y_00OjhnbVg90H21EOLACvtfcHk0lgw4zrEZFpdQw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtVKrur7K&md5=075dc5531be1a7bdf44d9e21448df783</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1016%2Fj.ajo.2013.05.016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ajo.2013.05.016%26sid%3Dliteratum%253Aachs%26aulast%3DTanihara%26aufirst%3DH.%26aulast%3DInoue%26aufirst%3DT.%26aulast%3DYamamoto%26aufirst%3DT.%26aulast%3DKuwayama%26aufirst%3DY.%26aulast%3DAbe%26aufirst%3DH.%26aulast%3DAraie%26aufirst%3DM.%26atitle%3DPhase%25202%2520randomized%2520clinical%2520study%2520of%2520a%2520Rho%2520kinase%2520inhibitor%252C%2520K-115%252C%2520in%2520primary%2520open-angle%2520glaucoma%2520and%2520ocular%2520hypertension%26jtitle%3DAm.%2520J.%2520Ophthalmol.%26date%3D2013%26volume%3D156%26spage%3D731%26epage%3D736%26doi%3D10.1016%2Fj.ajo.2013.05.016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Prêtre, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wicki, A.</span></span> <span> </span><span class="NLM_article-title">Inhibition of Akt and other AGC kinases: A target for clinical cancer therapy?</span>. <i>Semin. Cancer Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>48</i></span>,  <span class="NLM_fpage">70</span>– <span class="NLM_lpage">77</span>, <span class="refDoi"> DOI: 10.1016/j.semcancer.2017.04.011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=10.1016%2Fj.semcancer.2017.04.011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=28473255" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=1%3ACAS%3A528%3ADC%252BC2sXotlOrurc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2018&pages=70-77&author=V.+Pr%C3%AAtreauthor=A.+Wicki&title=Inhibition+of+Akt+and+other+AGC+kinases%3A+A+target+for+clinical+cancer+therapy%3F&doi=10.1016%2Fj.semcancer.2017.04.011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of Akt and other AGC kinases: A target for clinical cancer therapy?</span></div><div class="casAuthors">Pretre, Vincent; Wicki, Andreas</div><div class="citationInfo"><span class="NLM_cas:title">Seminars in Cancer Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">70-77</span>CODEN:
                <span class="NLM_cas:coden">SECBE7</span>;
        ISSN:<span class="NLM_cas:issn">1044-579X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  AGC kinases have been identified to contribute to cancer development and progression.  Currently, most AGC inhibitors in clin. development are Akt inhibitors such as MK-2206 or GDC-0068, which are known to promote cell growth arrest and to sensitize cancer cells to radiotherapy.  Response rates in clin. trials with single agent Akt inhibitors are typically low.  The obsd. adverse events are within the expected limits for compds. inhibiting the PI3K-mTOR axis.  Preclin. and early clin. data for combination therapies are accumulating.  Based on these data, several Akt inhibitors are about to enter phase 3 trials.  Besides drugs that target Akt, p70S6K inhibitors have entered clin. development.  Again, the response rates were rather low.  In addn., relevant toxicities were identified, including a risk for coagulopathies with these compds.  Multi-AGC kinase inhibitors are also in early clin. development but the data is not sufficient yet to draw conclusions regarding their efficacy and side-effect profile.  PKC inhibitors have been tested in the phase 3 setting but were found to lack efficacy.  More trials with isoform-specific PKC inhibitors are expected.  Taken together, therapies with AGC kinase inhibitors as single agents are unlikely to meet success.  However, combination therapies and a precise stratification of patients according to the activation of signaling axes may increase the probability to see relevant efficacy with these compds.  The emergence of onco-immunotherapies holds some new challenges for these agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoHcf3ttp3ohbVg90H21EOLACvtfcHk0lgw4zrEZFpdQw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXotlOrurc%253D&md5=9181c4968a872207d6b3b9bf3ba98881</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1016%2Fj.semcancer.2017.04.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.semcancer.2017.04.011%26sid%3Dliteratum%253Aachs%26aulast%3DPr%25C3%25AAtre%26aufirst%3DV.%26aulast%3DWicki%26aufirst%3DA.%26atitle%3DInhibition%2520of%2520Akt%2520and%2520other%2520AGC%2520kinases%253A%2520A%2520target%2520for%2520clinical%2520cancer%2520therapy%253F%26jtitle%3DSemin.%2520Cancer%2520Biol.%26date%3D2018%26volume%3D48%26spage%3D70%26epage%3D77%26doi%3D10.1016%2Fj.semcancer.2017.04.011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Browning, D. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kwon, I.-K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, R.</span></span> <span> </span><span class="NLM_article-title">cGMP-dependent protein kinases as potential targets for colon cancer prevention and treatment</span>. <i>Future Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>2</i></span>,  <span class="NLM_fpage">65</span>– <span class="NLM_lpage">80</span>, <span class="refDoi"> DOI: 10.4155/fmc.09.142</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=10.4155%2Ffmc.09.142" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=21426046" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=1%3ACAS%3A528%3ADC%252BC3cXosVSisw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2010&pages=65-80&author=D.+D.+Browningauthor=I.-K.+Kwonauthor=R.+Wang&title=cGMP-dependent+protein+kinases+as+potential+targets+for+colon+cancer+prevention+and+treatment&doi=10.4155%2Ffmc.09.142"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">cGMP-dependent protein kinases as potential targets for colon cancer prevention and treatment</span></div><div class="casAuthors">Browning, Darren D.; Kwon, In-Kiu; Wang, Rui</div><div class="citationInfo"><span class="NLM_cas:title">Future Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">65-80</span>CODEN:
                <span class="NLM_cas:coden">FMCUA7</span>;
        ISSN:<span class="NLM_cas:issn">1756-8919</span>.
    
            (<span class="NLM_cas:orgname">Future Science Ltd.</span>)
        </div><div class="casAbstract">A review.  In recent years, several antitumor signaling pathways mediated by the cGMP-dependent protein kinases have been identified in colon cancer cells.  This review aims to present the mounting evidence in favor of cGMP/protein kinase G (PKG) signaling as a therapeutic strategy in colon cancer.  The homeostatic and tumor suppressive effects of cGMP in the intestine are uncontested, but the signaling details are not understood.  PKG is the central cGMP effector, and can block proliferation and tumor angiogenesis by inhibiting β-catenin/TCF and SOX9 signaling.  Therapeutic activation of cGMP/PKG offers a promising avenue for the prevention and treatment of colon cancer, but addnl. preclin. studies are needed to fully understand the potential of this system.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq3mNwktPlmfbVg90H21EOLACvtfcHk0lgw4zrEZFpdQw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXosVSisw%253D%253D&md5=987db32b246874d5ae418bf45efc3087</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.4155%2Ffmc.09.142&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4155%252Ffmc.09.142%26sid%3Dliteratum%253Aachs%26aulast%3DBrowning%26aufirst%3DD.%2BD.%26aulast%3DKwon%26aufirst%3DI.-K.%26aulast%3DWang%26aufirst%3DR.%26atitle%3DcGMP-dependent%2520protein%2520kinases%2520as%2520potential%2520targets%2520for%2520colon%2520cancer%2520prevention%2520and%2520treatment%26jtitle%3DFuture%2520Med.%2520Chem.%26date%3D2010%26volume%3D2%26spage%3D65%26epage%3D80%26doi%3D10.4155%2Ffmc.09.142" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">da
Rocha, A. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mans, D. R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Regner, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwartsmann, G.</span></span> <span> </span><span class="NLM_article-title">Targeting protein kinase C: new therapeutic opportunities against high-grade malignant gliomas?</span>. <i>Oncologist</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">17</span>– <span class="NLM_lpage">33</span>, <span class="refDoi"> DOI: 10.1634/theoncologist.7-1-17</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=10.1634%2Ftheoncologist.7-1-17" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=11854544" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=1%3ACAS%3A528%3ADC%252BD38XitVCht7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2002&pages=17-33&author=A.+B.+da%0ARochaauthor=D.+R.+A.+Mansauthor=A.+Regnerauthor=G.+Schwartsmann&title=Targeting+protein+kinase+C%3A+new+therapeutic+opportunities+against+high-grade+malignant+gliomas%3F&doi=10.1634%2Ftheoncologist.7-1-17"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting protein kinase C: New therapeutic opportunities against high-grade malignant gliomas?</span></div><div class="casAuthors">da Rocha, A. B.; Mans, D. R. A.; Regner, A.; Schwartsmann, G.</div><div class="citationInfo"><span class="NLM_cas:title">Oncologist</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">17-33</span>CODEN:
                <span class="NLM_cas:coden">OCOLF6</span>;
        ISSN:<span class="NLM_cas:issn">1083-7159</span>.
    
            (<span class="NLM_cas:orgname">AlphaMed Press</span>)
        </div><div class="casAbstract">A review.  A large body of evidence suggests that the abnormal phenotype of neoplastic astrocytes, including their excessive proliferation rate and high propensity to invade surrounding tissues, results from mutations in crit. genes involved in key cellular events.  These genetic alterations can affect cell-surface-assocd. receptors, elements of signaling pathways, or components of the cell cycle clock, conferring a gain or a loss of relevant metabolic functions of the cells.  The understanding of such phenomena may allow the development of more efficacious forms of cancer treatment.  Examples are therapies specifically directed against overexpressed epidermal growth factor receptor, hyperactive Ras, excessively stimulated Raf-1, overproduced ornithine decarboxylase, or aberrantly activated cyclindependent kinases.  The applicability of some of these approaches is now being assessed in patients suffering from primary malignant central nervous system tumors that are not amenable to current therapeutic modalities.  Another potentially useful therapeutic strategy against such tumors involves the inhibition of hyperactive or overexpressed protein kinase C (PKC).  This strategy is justified by the decrease in cell proliferation and invasion following inhibition of the activity of this enzyme obsd. in preclin. glioma models.  Thus, interference with PKC activity may represent a novel form of exptl. cancer treatment that may simultaneously restrain the hyperproliferative state and the invasive capacity of high-grade malignant gliomas without inducing the expected toxicity of classical cytotoxic agents.  Of note, the exptl. use of PKC-inhibiting agents in patients with refractory high-grade malignant gliomas has indeed led to some clin. responses.  The present paper reviews the current status of the biochem. and mol. biol. of PKC, as well as the possibilities for developing novel anti-PKC-based therapies for central nervous system malignancies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGresHYaWuTx5LVg90H21EOLACvtfcHk0ljSPIu7C_0wVw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XitVCht7c%253D&md5=3b2b01910cb3bd93631dfa8b08e9d389</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1634%2Ftheoncologist.7-1-17&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1634%252Ftheoncologist.7-1-17%26sid%3Dliteratum%253Aachs%26aulast%3Dda%2BRocha%26aufirst%3DA.%2BB.%26aulast%3DMans%26aufirst%3DD.%2BR.%2BA.%26aulast%3DRegner%26aufirst%3DA.%26aulast%3DSchwartsmann%26aufirst%3DG.%26atitle%3DTargeting%2520protein%2520kinase%2520C%253A%2520new%2520therapeutic%2520opportunities%2520against%2520high-grade%2520malignant%2520gliomas%253F%26jtitle%3DOncologist%26date%3D2002%26volume%3D7%26spage%3D17%26epage%3D33%26doi%3D10.1634%2Ftheoncologist.7-1-17" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kathawala, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anreddy, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alqahtani, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.-K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sodani, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaddoumi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ashby, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z.-S.</span></span> <span> </span><span class="NLM_article-title">The small molecule tyrosine kinase inhibitor NVP-BHG712 antagonizes ABCC10-mediated paclitaxel resistance: a preclinical and pharmacokinetic study</span>. <i>Oncotarget</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">510</span>– <span class="NLM_lpage">521</span>, <span class="refDoi"> DOI: 10.18632/oncotarget.2638</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=10.18632%2Foncotarget.2638" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=25402202" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=1%3ACAS%3A280%3ADC%252BC2M3osFWjtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=510-521&author=R.+J.+Kathawalaauthor=L.+Weiauthor=N.+Anreddyauthor=K.+Chenauthor=A.+Patelauthor=S.+Alqahtaniauthor=Y.-K.+Zhangauthor=Y.-J.+Wangauthor=K.+Sodaniauthor=A.+Kaddoumiauthor=C.+R.+Ashbyauthor=Z.-S.+Chen&title=The+small+molecule+tyrosine+kinase+inhibitor+NVP-BHG712+antagonizes+ABCC10-mediated+paclitaxel+resistance%3A+a+preclinical+and+pharmacokinetic+study&doi=10.18632%2Foncotarget.2638"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">The small molecule tyrosine kinase inhibitor NVP-BHG712 antagonizes ABCC10-mediated paclitaxel resistance: a preclinical and pharmacokinetic study</span></div><div class="casAuthors">Kathawala Rishil J; Wei Liuya; Anreddy Nagaraju; Patel Atish; Zhang Yun-Kai; Wang Yi-Jun; Sodani Kamlesh; Ashby Charles R Jr; Chen Zhe-Sheng; Chen Kang; Alqahtani Saeed; Kaddoumi Amal</div><div class="citationInfo"><span class="NLM_cas:title">Oncotarget</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">510-21</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Paclitaxel exhibits clinical activity against a wide variety of solid tumors.  However, resistance to paclitaxel significantly attenuates the response to chemotherapy.  The ABC transporter subfamily C member 10 (ABCC10), also known as multi-drug resistance protein 7 (MRP7) efflux transporter, is a major mediator of paclitaxel resistance.  Here, we determine the effect of NVP-BHG712, a specific EphB4 receptor inhibitor, on 1) paclitaxel resistance in HEK293 cells transfected with ABCC10, 2) the growth of tumors in athymic nude mice that received NVP-BHG712 and paclitaxel systemically and 3) the pharmacokinetics of paclitaxel in presence or absence of NVP-BHG712.  NVP-BHG712 (0.5 μM), in HEK293/ABCC10 cells, significantly enhanced the intracellular accumulation of paclitaxel by inhibiting the efflux activity of ABCC10 without altering the expression level of the ABCC10 protein.  Furthermore, NVP-BHG712 (25 mg/kg, p.o., q3d x 6), in combination with paclitaxel (15 mg/kg, i.p., q3d x 6), significantly inhibited the growth of ABCC10-expressing tumors in athymic nude mice.  NVP-BHG712 administration significantly increased the levels of paclitaxel in the tumors but not in plasma compared to paclitaxel alone.  The combination of NVP-BHG712 and paclitaxel could serve as a novel and useful therapeutic strategy to attenuate paclitaxel resistance mediated by the expression of the ABCC10 transporter.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTxASDz7eI_LVZoq0fPSOTvfW6udTcc2ebTlWOHwH084bntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2M3osFWjtQ%253D%253D&md5=75b7e4bb54af1483c51799cf1ab6642e</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.18632%2Foncotarget.2638&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncotarget.2638%26sid%3Dliteratum%253Aachs%26aulast%3DKathawala%26aufirst%3DR.%2BJ.%26aulast%3DWei%26aufirst%3DL.%26aulast%3DAnreddy%26aufirst%3DN.%26aulast%3DChen%26aufirst%3DK.%26aulast%3DPatel%26aufirst%3DA.%26aulast%3DAlqahtani%26aufirst%3DS.%26aulast%3DZhang%26aufirst%3DY.-K.%26aulast%3DWang%26aufirst%3DY.-J.%26aulast%3DSodani%26aufirst%3DK.%26aulast%3DKaddoumi%26aufirst%3DA.%26aulast%3DAshby%26aufirst%3DC.%2BR.%26aulast%3DChen%26aufirst%3DZ.-S.%26atitle%3DThe%2520small%2520molecule%2520tyrosine%2520kinase%2520inhibitor%2520NVP-BHG712%2520antagonizes%2520ABCC10-mediated%2520paclitaxel%2520resistance%253A%2520a%2520preclinical%2520and%2520pharmacokinetic%2520study%26jtitle%3DOncotarget%26date%3D2015%26volume%3D6%26spage%3D510%26epage%3D521%26doi%3D10.18632%2Foncotarget.2638" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Je, D. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O, Y. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ji, Y. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cho, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, D. H.</span></span> <span> </span><span class="NLM_article-title">The inhibition of SRC family kinase suppresses pancreatic cancer cell proliferation, migration, and invasion</span>. <i>Pancreas</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>43</i></span>,  <span class="NLM_fpage">768</span>– <span class="NLM_lpage">776</span>, <span class="refDoi"> DOI: 10.1097/MPA.0000000000000103</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=10.1097%2FMPA.0000000000000103" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=24763074" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=1%3ACAS%3A528%3ADC%252BC2cXps1Cjt74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2014&pages=768-776&author=D.+W.+Jeauthor=Y.+M.+Oauthor=Y.+G.+Jiauthor=Y.+Choauthor=D.+H.+Lee&title=The+inhibition+of+SRC+family+kinase+suppresses+pancreatic+cancer+cell+proliferation%2C+migration%2C+and+invasion&doi=10.1097%2FMPA.0000000000000103"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">The Inhibition of Src Family Kinase Suppresses Pancreatic Cancer Cell Proliferation, Migration, and Invasion</span></div><div class="casAuthors">Je, Dong Wook; O, Young Moon; Ji, Young Geon; Cho, Yunkyung; Lee, Dong Hyeon</div><div class="citationInfo"><span class="NLM_cas:title">Pancreas (Philadelphia, PA, United States)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">43</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">768-776</span>CODEN:
                <span class="NLM_cas:coden">PANCE4</span>;
        ISSN:<span class="NLM_cas:issn">0885-3177</span>.
    
            (<span class="NLM_cas:orgname">Lippincott Williams & Wilkins</span>)
        </div><div class="casAbstract">Objectives: Src is considered a rising therapeutic target for the treatment of solid tumors, and Src family kinases (SFKs) participate in cancer cell proliferation and survival.  The role of SFK suppression was investigated in the proliferation, migration, and invasion of pancreatic cancer cells.  Methods: Knockdown of the SFKs in pancreatic cancer cells was achieved by transfecting small interfering RNAs, and its effects were investigated using proliferation, wound, and invasion assays.  Results: The SFK inhibitors suppressed proliferation and induced cell cycle arrest in pancreatic cancer cells.  The SFK mRNA profiles showed that Yes1, Lyn, Fyn, Frk, Hck, and Src were expressed.  Specific small interfering RNA transfection suppressed the mRNA expressions of Yes1, Lyn, Fyn, Frk, and Src, and the knockdown suppressed cell proliferation by 16.7% to 47.3% in PANC-1 cells.  Knockdown of any of these 5 SFKs suppressed proliferation in other pancreatic cancer cell lines by 3.0% to 40.5%.  The knockdowns significantly reduced pancreatic cancer cell migration by 24.9% to 66.7% and completely inhibited invasion.  Conclusions: These results suggest that the knockdown of Yes1, Lyn, Fyn, Frk, or Src reduce human pancreatic cancer cell proliferation, migration, and invasion, and that SFKs should be viewed as crit. therapeutic targets of pancreatic cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWIIEBAqGDnrVg90H21EOLACvtfcHk0lisGDpLqB7BVA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXps1Cjt74%253D&md5=27a2339f2ac9f4176cca71e77a315bbe</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1097%2FMPA.0000000000000103&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1097%252FMPA.0000000000000103%26sid%3Dliteratum%253Aachs%26aulast%3DJe%26aufirst%3DD.%2BW.%26aulast%3DO%26aufirst%3DY.%2BM.%26aulast%3DJi%26aufirst%3DY.%2BG.%26aulast%3DCho%26aufirst%3DY.%26aulast%3DLee%26aufirst%3DD.%2BH.%26atitle%3DThe%2520inhibition%2520of%2520SRC%2520family%2520kinase%2520suppresses%2520pancreatic%2520cancer%2520cell%2520proliferation%252C%2520migration%252C%2520and%2520invasion%26jtitle%3DPancreas%26date%3D2014%26volume%3D43%26spage%3D768%26epage%3D776%26doi%3D10.1097%2FMPA.0000000000000103" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kalyankrishna, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wislez, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thilaganathan, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saigal, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wistuba, I. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, F. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kurie, J. M.</span></span> <span> </span><span class="NLM_article-title">SRC-family kinases are activated in non-small cell lung cancer and promote the survival of epidermal growth factor receptor-dependent cell lines</span>. <i>Am. J. Pathol.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>170</i></span>,  <span class="NLM_fpage">366</span>– <span class="NLM_lpage">376</span>, <span class="refDoi"> DOI: 10.2353/ajpath.2007.060706</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=10.2353%2Fajpath.2007.060706" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=17200208" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=1%3ACAS%3A280%3ADC%252BD2s%252FgtlOnsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=170&publication_year=2007&pages=366-376&author=J.+Zhangauthor=S.+Kalyankrishnaauthor=M.+Wislezauthor=N.+Thilaganathanauthor=B.+Saigalauthor=W.+Weiauthor=L.+Maauthor=I.+I.+Wistubaauthor=F.+M.+Johnsonauthor=J.+M.+Kurie&title=SRC-family+kinases+are+activated+in+non-small+cell+lung+cancer+and+promote+the+survival+of+epidermal+growth+factor+receptor-dependent+cell+lines&doi=10.2353%2Fajpath.2007.060706"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">SRC-family kinases are activated in non-small cell lung cancer and promote the survival of epidermal growth factor receptor-dependent cell lines</span></div><div class="casAuthors">Zhang Jie; Kalyankrishna Shailaja; Wislez Marie; Thilaganathan Nishan; Saigal Babita; Wei Wei; Ma Long; Wistuba Ignacio I; Johnson Faye M; Kurie Jonathan M</div><div class="citationInfo"><span class="NLM_cas:title">The American journal of pathology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">170</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">366-76</span>
        ISSN:<span class="NLM_cas:issn">0002-9440</span>.
    </div><div class="casAbstract">The role of Src-family kinases (SFKs) in non-small cell lung cancer (NSCLC) has not been fully defined.  Here we addressed this question by examining SFK phosphorylation in NSCLC biopsy samples and using genetic and pharmacological approaches to inhibit SFK expression and activity in cultured NSCLC cells.  Immunohistochemical analysis of NSCLC biopsy samples using a Tyr416 phosphorylation-specific, pan-SFK antibody revealed staining in 123 (33%) of 370 tumors.  Because c-Src is known to be both an upstream activator and downstream mediator of epidermal growth factor receptor (EGFR), we next investigated SFK phosphorylation in a panel of NSCLC cell lines, including ones that depend on EGFR for survival.  The EGFR-dependent NSCLC cell lines HCC827 and H3255 had increased phosphorylation of SFKs, and treatment of these cells with an SFK inhibitor (PP1 or SKI-606) induced apoptosis.  PP1 decreased phosphorylation of EGFR, ErbB2, and ErbB3 and strikingly enhanced apoptosis by gefitinib, an EGFR inhibitor.  HCC827 cells transfected with c-Src short hairpin RNA exhibited diminished phosphorylation of EGFR and ErbB2 and decreased sensitivity to apoptosis by PP1 or gefitinib.  We conclude that SFKs are activated in NSCLC biopsy samples, promote the survival of EGFR-dependent NSCLC cells, and should be investigated as therapeutic targets in NSCLC patients.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQZ2f3qQMjhE0kUaC_kzhvyfW6udTcc2eZLthpEABcOerntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2s%252FgtlOnsA%253D%253D&md5=391c770e7368377b4b9f5576a206d8d0</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.2353%2Fajpath.2007.060706&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2353%252Fajpath.2007.060706%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DKalyankrishna%26aufirst%3DS.%26aulast%3DWislez%26aufirst%3DM.%26aulast%3DThilaganathan%26aufirst%3DN.%26aulast%3DSaigal%26aufirst%3DB.%26aulast%3DWei%26aufirst%3DW.%26aulast%3DMa%26aufirst%3DL.%26aulast%3DWistuba%26aufirst%3DI.%2BI.%26aulast%3DJohnson%26aufirst%3DF.%2BM.%26aulast%3DKurie%26aufirst%3DJ.%2BM.%26atitle%3DSRC-family%2520kinases%2520are%2520activated%2520in%2520non-small%2520cell%2520lung%2520cancer%2520and%2520promote%2520the%2520survival%2520of%2520epidermal%2520growth%2520factor%2520receptor-dependent%2520cell%2520lines%26jtitle%3DAm.%2520J.%2520Pathol.%26date%3D2007%26volume%3D170%26spage%3D366%26epage%3D376%26doi%3D10.2353%2Fajpath.2007.060706" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zahari, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Renuse, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kelkar, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bharbuiya, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rojas, P. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stearns, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gabrielson, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Malla, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sukumar, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mahajan, N. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pandey, A.</span></span> <span> </span><span class="NLM_article-title">The non-receptor tyrosine kinase TNK2/ACK1 is a novel therapeutic target in triple negative breast cancer</span>. <i>Oncotarget</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">2971</span>– <span class="NLM_lpage">2983</span>, <span class="refDoi"> DOI: 10.18632/oncotarget.13579</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=10.18632%2Foncotarget.13579" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=27902967" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=1%3ACAS%3A280%3ADC%252BC2sjitVShsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=2971-2983&author=X.+Wuauthor=M.+S.+Zahariauthor=S.+Renuseauthor=D.+S.+Kelkarauthor=M.+A.+Bharbuiyaauthor=P.+L.+Rojasauthor=V.+Stearnsauthor=E.+Gabrielsonauthor=P.+Mallaauthor=S.+Sukumarauthor=N.+P.+Mahajanauthor=A.+Pandey&title=The+non-receptor+tyrosine+kinase+TNK2%2FACK1+is+a+novel+therapeutic+target+in+triple+negative+breast+cancer&doi=10.18632%2Foncotarget.13579"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">The non-receptor tyrosine kinase TNK2/ACK1 is a novel therapeutic target in triple negative breast cancer</span></div><div class="casAuthors">Wu Xinyan; Zahari Muhammad Saddiq; Renuse Santosh; Kelkar Dhanashree S; Rojas Pamela L; Pandey Akhilesh; Wu Xinyan; Zahari Muhammad Saddiq; Renuse Santosh; Kelkar Dhanashree S; Barbhuiya Mustafa A; Rojas Pamela L; Pandey Akhilesh; Renuse Santosh; Stearns Vered; Gabrielson Edward; Sukumar Saraswati; Pandey Akhilesh; Gabrielson Edward; Pandey Akhilesh; Malla Pavani; Mahajan Nupam P; Mahajan Nupam P</div><div class="citationInfo"><span class="NLM_cas:title">Oncotarget</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">2971-2983</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Breast cancer is the most prevalent cancer in women worldwide.  About 15-20% of all breast cancers do not express estrogen receptor, progesterone receptor or HER2 receptor and hence are collectively classified as triple negative breast cancer (TNBC).  These tumors are often relatively aggressive when compared to other types of breast cancer, and this issue is compounded by the lack of effective targeted therapy.  In our previous phosphoproteomic profiling effort, we identified the non-receptor tyrosine kinase TNK2 as activated in a majority of aggressive TNBC cell lines.  In the current study, we show that high expression of TNK2 in breast cancer cell lines correlates with high proliferation, invasion and colony forming ability.  We demonstrate that knockdown of TNK2 expression can substantially suppress the invasiveness and proliferation advantage of TNBC cells in vitro and tumor formation in xenograft mouse models.  Moreover, inhibition of TNK2 with small molecule inhibitor (R)-9bMS significantly compromised TNBC proliferation.Finally, we find that high levels of TNK2 expression in high-grade basal-like breast cancers correlates significantly with poorer patient outcome.  Taken together, our study suggests that TNK2 is a novel potential therapeutic target for the treatment of TNBC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQ8VkHWAOdcJDAyhZbj3lKdfW6udTcc2eZLthpEABcOerntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2sjitVShsg%253D%253D&md5=71f888ec0dd35a99a9b5a779ab56f20a</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.18632%2Foncotarget.13579&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncotarget.13579%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DX.%26aulast%3DZahari%26aufirst%3DM.%2BS.%26aulast%3DRenuse%26aufirst%3DS.%26aulast%3DKelkar%26aufirst%3DD.%2BS.%26aulast%3DBharbuiya%26aufirst%3DM.%2BA.%26aulast%3DRojas%26aufirst%3DP.%2BL.%26aulast%3DStearns%26aufirst%3DV.%26aulast%3DGabrielson%26aufirst%3DE.%26aulast%3DMalla%26aufirst%3DP.%26aulast%3DSukumar%26aufirst%3DS.%26aulast%3DMahajan%26aufirst%3DN.%2BP.%26aulast%3DPandey%26aufirst%3DA.%26atitle%3DThe%2520non-receptor%2520tyrosine%2520kinase%2520TNK2%252FACK1%2520is%2520a%2520novel%2520therapeutic%2520target%2520in%2520triple%2520negative%2520breast%2520cancer%26jtitle%3DOncotarget%26date%3D2017%26volume%3D8%26spage%3D2971%26epage%3D2983%26doi%3D10.18632%2Foncotarget.13579" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Isobe, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mizuno, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaneko, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohta, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koide, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanabe, S.</span></span> <span> </span><span class="NLM_article-title">Effects of K-115, a Rho-kinase inhibitor, on aqueous humor dynamics in rabbits</span>. <i>Curr. Eye Res.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>39</i></span>,  <span class="NLM_fpage">813</span>– <span class="NLM_lpage">822</span>, <span class="refDoi"> DOI: 10.3109/02713683.2013.874444</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=10.3109%2F02713683.2013.874444" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=24502505" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtFylu7fP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=2014&pages=813-822&author=T.+Isobeauthor=K.+Mizunoauthor=Y.+Kanekoauthor=M.+Ohtaauthor=T.+Koideauthor=S.+Tanabe&title=Effects+of+K-115%2C+a+Rho-kinase+inhibitor%2C+on+aqueous+humor+dynamics+in+rabbits&doi=10.3109%2F02713683.2013.874444"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Effects of K-115, a Rho-Kinase Inhibitor, on Aqueous Humor Dynamics in Rabbits</span></div><div class="casAuthors">Isobe, Tomoyuki; Mizuno, Ken; Kaneko, Yoshio; Ohta, Masayuki; Koide, Takashi; Tanabe, Sohei</div><div class="citationInfo"><span class="NLM_cas:title">Current Eye Research</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">39</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">813-822</span>CODEN:
                <span class="NLM_cas:coden">CEYRDM</span>;
        ISSN:<span class="NLM_cas:issn">0271-3683</span>.
    
            (<span class="NLM_cas:orgname">Informa Healthcare</span>)
        </div><div class="casAbstract">Purpose: To evaluate the topical instillation of K-115, a selective Rho-assocd. coiled coil-contg. protein kinase (ROCK) inhibitor, on intraocular pressure (IOP), ocular distribution, and aq. humor dynamics in exptl. animals.  Methods: Kinase inhibition by K-115 was measured by biochem. assay.  IOP was monitored using a pneumatonometer in albino rabbits and monkeys after topical instillation of K-115.  The ocular distribution of [14C]K-115 was detd. by whole-head autoradiog.  The aq. flow rate was detd. by fluorophotometry.  The total outflow facility and uveoscleral outflow were measured by two-level const. pressure perfusion and perfusion technique using fluorescein isothiocyanate-dextran, resp.  Results: Biochem. assay showed that K-115 had selective and potent inhibitory effects on ROCKs.  In rabbits, topical instillation of K-115 significantly reduced IOP in a dose-dependent manner.  Maximum IOP redn. was obsd. 1 h after topical instillation, which was 8.55 ± 1.09 mmHg (mean ± SE) from the baseline IOP at 0.5%.  In monkeys, max. IOP redn. was obsd. 2 h after topical instillation, which was 4.36 ± 0.32 mmHg from the baseline IOP at 0.4%, and was significantly stronger than that of 0.005% latanoprost.  Whole-head autoradiog. showed that the radioactivity level was max. at 15 min after instillation of [14C]K-115 in the ipsilateral eye.  Single instillation of 0.4% K-115 showed no effect on aq. flow rate or uveoscleral outflow, but significantly increased conventional outflow facility by 2.2-fold compared to vehicle-treated eyes in rabbits.  Conclusions: These results indicated that K-115 ophthalmic soln., a selective and potent ROCK inhibitor, is a novel and potent antiglaucoma agent.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpC62-fukbe1rVg90H21EOLACvtfcHk0lgzYlM5b74byg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtFylu7fP&md5=1d0048c285f37c03f6f1c0f3bd26ec13</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.3109%2F02713683.2013.874444&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3109%252F02713683.2013.874444%26sid%3Dliteratum%253Aachs%26aulast%3DIsobe%26aufirst%3DT.%26aulast%3DMizuno%26aufirst%3DK.%26aulast%3DKaneko%26aufirst%3DY.%26aulast%3DOhta%26aufirst%3DM.%26aulast%3DKoide%26aufirst%3DT.%26aulast%3DTanabe%26aufirst%3DS.%26atitle%3DEffects%2520of%2520K-115%252C%2520a%2520Rho-kinase%2520inhibitor%252C%2520on%2520aqueous%2520humor%2520dynamics%2520in%2520rabbits%26jtitle%3DCurr.%2520Eye%2520Res.%26date%3D2014%26volume%3D39%26spage%3D813%26epage%3D822%26doi%3D10.3109%2F02713683.2013.874444" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yamamoto, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maruyama, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Himori, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Omodaka, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yokoyama, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shiga, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morin, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakazawa, T.</span></span> <span> </span><span class="NLM_article-title">The novel Rho kinase (ROCK) inhibitor K-115: a new candidate drug for neuroprotective treatment in glaucoma</span>. <i>Invest. Ophthalmol. Visual Sci.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">7126</span>– <span class="NLM_lpage">7136</span>, <span class="refDoi"> DOI: 10.1167/iovs.13-13842</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=10.1167%2Fiovs.13-13842" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=25277230" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=1%3ACAS%3A528%3ADC%252BC2MXitVOltbfN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2014&pages=7126-7136&author=K.+Yamamotoauthor=K.+Maruyamaauthor=N.+Himoriauthor=K.+Omodakaauthor=Y.+Yokoyamaauthor=Y.+Shigaauthor=R.+Morinauthor=T.+Nakazawa&title=The+novel+Rho+kinase+%28ROCK%29+inhibitor+K-115%3A+a+new+candidate+drug+for+neuroprotective+treatment+in+glaucoma&doi=10.1167%2Fiovs.13-13842"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">The novel Rho kinase (ROCK) inhibitor K-115: a new candidate drug for neuroprotective treatment in glaucoma</span></div><div class="casAuthors">Yamamoto, Kotaro; Maruyama, Kazuichi; Himori, Noriko; Omodaka, Kazuko; Yokoyama, Yu; Shiga, Yukihiro; Morin, Ryu; Nakazawa, Toru</div><div class="citationInfo"><span class="NLM_cas:title">Investigative Ophthalmology & Visual Science</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">7126-7136</span>CODEN:
                <span class="NLM_cas:coden">IOVSDA</span>;
        ISSN:<span class="NLM_cas:issn">1552-5783</span>.
    
            (<span class="NLM_cas:orgname">Association for Research in Vision and Ophthalmology</span>)
        </div><div class="casAbstract">To investigate the effect of K-115, a novel Rho kinase (ROCK) inhibitor, on retinal ganglion cell (RGC) survival in an optic nerve crush (NC) model.  Addnl., to det. the details of the mechanism of K-115's neuroprotective effect in vivo and in vitro.  ROCK inhibitors, including K-115 and fasudil (1 mg/kg/d), or vehicle were administered orally to C57BL/6 mice.  Retinal ganglion cell death was then induced with NC.  Retinal ganglion cell survival was evaluated by counting surviving retrogradely labeled cells and measuring RGC marker expression with quant. real-time polymerase chain reaction (qRT-PCR).  Total oxidized lipid levels were assessed with a thiobarbituric acid-reactive substances (TBARS) assay.  Reactive oxygen species (ROS) levels were assessed by co-labeling with CellROX and Fluorogold.  Expression of the NADPH oxidase (Nox) family of genes was evaluated with qRT-PCR.  The survival of RGCs after NC was increased 34 ± 3% with K-115, a significantly protective effect.  Moreover, a similar effect was revealed by the qRT-PCR anal. of Thy-1.2 and Brn3a, RGC markers.  Levels of oxidized lipids and ROS also increased with time after NC.  NC-induced oxidative stress, including oxidn. of lipids and prodn. of ROS, was significantly attenuated by K-115.  Furthermore, expression of the Nox gene family, esp. Nox1, which is involved in the NC-induced ROS prodn. pathway, was dramatically reduced by K-115.  The results indicated that oral K-115 administration delayed RGC death.  Although K-115 may be mediated through Nox1 downregulation, we found that it did not suppress ROS prodn. directly.  Our findings show that K-115 has a potential use in neuroprotective treatment for glaucoma and other neurodegenerative diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq7y2j7EK2KoLVg90H21EOLACvtfcHk0lgzYlM5b74byg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXitVOltbfN&md5=f0db38d732b7244c73b3aeca80cf193a</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1167%2Fiovs.13-13842&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1167%252Fiovs.13-13842%26sid%3Dliteratum%253Aachs%26aulast%3DYamamoto%26aufirst%3DK.%26aulast%3DMaruyama%26aufirst%3DK.%26aulast%3DHimori%26aufirst%3DN.%26aulast%3DOmodaka%26aufirst%3DK.%26aulast%3DYokoyama%26aufirst%3DY.%26aulast%3DShiga%26aufirst%3DY.%26aulast%3DMorin%26aufirst%3DR.%26aulast%3DNakazawa%26aufirst%3DT.%26atitle%3DThe%2520novel%2520Rho%2520kinase%2520%2528ROCK%2529%2520inhibitor%2520K-115%253A%2520a%2520new%2520candidate%2520drug%2520for%2520neuroprotective%2520treatment%2520in%2520glaucoma%26jtitle%3DInvest.%2520Ophthalmol.%2520Visual%2520Sci.%26date%3D2014%26volume%3D55%26spage%3D7126%26epage%3D7136%26doi%3D10.1167%2Fiovs.13-13842" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Martiny-Baron, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holzer, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Billy, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schnell, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brueggen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferretti, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmiedeberg, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wood, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Furet, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Imbach, P.</span></span> <span> </span><span class="NLM_article-title">The small molecule specific EphB4 kinase inhibitor NVP-BHG712 inhibits VEGF driven angiogenesis</span>. <i>Angiogenesis</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">259</span>– <span class="NLM_lpage">267</span>, <span class="refDoi"> DOI: 10.1007/s10456-010-9183-z</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=10.1007%2Fs10456-010-9183-z" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=20803239" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtFyis7fN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2010&pages=259-267&author=G.+Martiny-Baronauthor=P.+Holzerauthor=E.+Billyauthor=C.+Schnellauthor=J.+Brueggenauthor=M.+Ferrettiauthor=N.+Schmiedebergauthor=J.+M.+Woodauthor=P.+Furetauthor=P.+Imbach&title=The+small+molecule+specific+EphB4+kinase+inhibitor+NVP-BHG712+inhibits+VEGF+driven+angiogenesis&doi=10.1007%2Fs10456-010-9183-z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">The small molecule specific EphB4 kinase inhibitor NVP-BHG712 inhibits VEGF driven angiogenesis</span></div><div class="casAuthors">Martiny-Baron, Georg; Holzer, Philipp; Billy, Eric; Schnell, Christian; Brueggen, Joseph; Ferretti, Mireille; Schmiedeberg, Niko; Wood, Jeanette M.; Furet, Pascal; Imbach, Patricia</div><div class="citationInfo"><span class="NLM_cas:title">Angiogenesis</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">259-267</span>CODEN:
                <span class="NLM_cas:coden">AGIOFT</span>;
        ISSN:<span class="NLM_cas:issn">0969-6970</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">EphB4 and its cognitive ligand ephrinB2 play an important role in embryonic vessel development and vascular remodeling.  In addn., several reports suggest that this receptor ligand pair is also involved in pathol. vessel formation in adults including tumor angiogenesis.  Eph/ephrin signaling is a complex phenomena characterized by receptor forward signaling through the tyrosine kinase of the receptor and ephrin reverse signaling through various protein-protein interaction domains and phosphorylation motifs of the ephrin ligands.  Therefore, interfering with EphR/ephrin signaling by the means of targeted gene ablation, sol. receptors, dominant neg. mutants or antisense mols. often does not allow to discriminate between inhibition of Eph/ephrin forward and reverse signaling.  We developed a specific small mol. wt. kinase inhibitor of the EphB4 kinase, NVP-BHG712, which inhibits EphB4 kinase activity in the low nanomolar range in cellular assays showed high selectivity for targeting the EphB4 kinase when profiled against other kinases in biochem. as well as in cell based assays.  Furthermore, NVP-BHG712 shows excellent pharmacokinetic properties and potently inhibits EphB4 autophosphorylation in tissues after oral administration.  In vivo, NVP-BHG712 inhibits VEGF driven vessel formation, while it has only little effects on VEGF receptor (VEGFR) activity in vitro or in cellular assays.  The data shown here suggest a close cross talk between the VEGFR and EphR signaling during vessel formation.  In addn. to its established function in vascular remodeling and endothelial arterio-venous differentiation, EphB4 forward signaling appears to be an important mediator of VEGF induced angiogenesis since inhibition of EphB4 forward signaling is sufficient to inhibit VEGF induced angiogenesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrWzqlHCnHYMLVg90H21EOLACvtfcHk0ljGv3fGZa4Lig"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtFyis7fN&md5=a7c8df9e330fc86001b36f3495ac6037</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1007%2Fs10456-010-9183-z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10456-010-9183-z%26sid%3Dliteratum%253Aachs%26aulast%3DMartiny-Baron%26aufirst%3DG.%26aulast%3DHolzer%26aufirst%3DP.%26aulast%3DBilly%26aufirst%3DE.%26aulast%3DSchnell%26aufirst%3DC.%26aulast%3DBrueggen%26aufirst%3DJ.%26aulast%3DFerretti%26aufirst%3DM.%26aulast%3DSchmiedeberg%26aufirst%3DN.%26aulast%3DWood%26aufirst%3DJ.%2BM.%26aulast%3DFuret%26aufirst%3DP.%26aulast%3DImbach%26aufirst%3DP.%26atitle%3DThe%2520small%2520molecule%2520specific%2520EphB4%2520kinase%2520inhibitor%2520NVP-BHG712%2520inhibits%2520VEGF%2520driven%2520angiogenesis%26jtitle%3DAngiogenesis%26date%3D2010%26volume%3D13%26spage%3D259%26epage%3D267%26doi%3D10.1007%2Fs10456-010-9183-z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref54"><div class="reference"><strong class="refLabel"><a href="#ref54" class="rightTabRefNumLink">54</a></strong><div class="NLM_citation" id="rightTab-cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Karaman, M. W</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herrgard, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Treiber, D. K</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gallant, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Atteridge, C. E</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campbell, B. T</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, K. W</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciceri, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, M. I</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edeen, P. T</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Faraoni, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Floyd, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hunt, J. P</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lockhart, D. J</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Milanov, Z. V</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morrison, M. J</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pallares, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, H. K</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pritchard, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wodicka, L. M</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zarrinkar, P. P</span></span> <span> </span><span class="NLM_article-title">A quantitative analysis of kinase inhibitor selectivity</span>. <i>Nat. Biotechnol.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">127</span>– <span class="NLM_lpage">132</span>, <span class="refDoi"> DOI: 10.1038/nbt1358</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=10.1038%2Fnbt1358" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=18183025" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=1%3ACAS%3A528%3ADC%252BD1cXisFGlsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2008&pages=127-132&author=M.+W+Karamanauthor=S.+Herrgardauthor=D.+K+Treiberauthor=P.+Gallantauthor=C.+E+Atteridgeauthor=B.+T+Campbellauthor=K.+W+Chanauthor=P.+Ciceriauthor=M.+I+Davisauthor=P.+T+Edeenauthor=R.+Faraoniauthor=M.+Floydauthor=J.+P+Huntauthor=D.+J+Lockhartauthor=Z.+V+Milanovauthor=M.+J+Morrisonauthor=G.+Pallaresauthor=H.+K+Patelauthor=S.+Pritchardauthor=L.+M+Wodickaauthor=P.+P+Zarrinkar&title=A+quantitative+analysis+of+kinase+inhibitor+selectivity&doi=10.1038%2Fnbt1358"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">A quantitative analysis of kinase inhibitor selectivity</span></div><div class="casAuthors">Karaman, Mazen W.; Herrgard, Sanna; Treiber, Daniel K.; Gallant, Paul; Atteridge, Corey E.; Campbell, Brian T.; Chan, Katrina W.; Ciceri, Pietro; Davis, Mindy I.; Edeen, Philip T.; Faraoni, Raffaella; Floyd, Mark; Hunt, Jeremy P.; Lockhart, Daniel J.; Milanov, Zdravko V.; Morrison, Michael J.; Pallares, Gabriel; Patel, Hitesh K.; Pritchard, Stephanie; Wodicka, Lisa M.; Zarrinkar, Patrick P.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">127-132</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Kinase inhibitors are a new class of therapeutics with a propensity to inhibit multiple targets.  The biol. consequences of multikinase activity are poorly defined, and an important step toward understanding the relationship between selectivity, efficacy and safety is the exploration of how inhibitors interact with the human kinome.  The authors present interaction maps for 38 kinase inhibitors across a panel of 317 kinases representing >50% of the predicted human protein kinome.  The data constitute the most comprehensive study of kinase inhibitor selectivity to date and reveal a wide diversity of interaction patterns.  To enable a global anal. of the results, the authors introduce the concept of a selectivity score as a general tool to quantify and differentiate the obsd. interaction patterns.  The authors further investigate the impact of panel size and find that small assay panels do not provide a robust measure of selectivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpiArJSq2_lPrVg90H21EOLACvtfcHk0ljGv3fGZa4Lig"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXisFGlsQ%253D%253D&md5=346265d412853ced636ad4128ed8a76f</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1038%2Fnbt1358&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt1358%26sid%3Dliteratum%253Aachs%26aulast%3DKaraman%26aufirst%3DM.%2BW%26aulast%3DHerrgard%26aufirst%3DS.%26aulast%3DTreiber%26aufirst%3DD.%2BK%26aulast%3DGallant%26aufirst%3DP.%26aulast%3DAtteridge%26aufirst%3DC.%2BE%26aulast%3DCampbell%26aufirst%3DB.%2BT%26aulast%3DChan%26aufirst%3DK.%2BW%26aulast%3DCiceri%26aufirst%3DP.%26aulast%3DDavis%26aufirst%3DM.%2BI%26aulast%3DEdeen%26aufirst%3DP.%2BT%26aulast%3DFaraoni%26aufirst%3DR.%26aulast%3DFloyd%26aufirst%3DM.%26aulast%3DHunt%26aufirst%3DJ.%2BP%26aulast%3DLockhart%26aufirst%3DD.%2BJ%26aulast%3DMilanov%26aufirst%3DZ.%2BV%26aulast%3DMorrison%26aufirst%3DM.%2BJ%26aulast%3DPallares%26aufirst%3DG.%26aulast%3DPatel%26aufirst%3DH.%2BK%26aulast%3DPritchard%26aufirst%3DS.%26aulast%3DWodicka%26aufirst%3DL.%2BM%26aulast%3DZarrinkar%26aufirst%3DP.%2BP%26atitle%3DA%2520quantitative%2520analysis%2520of%2520kinase%2520inhibitor%2520selectivity%26jtitle%3DNat.%2520Biotechnol.%26date%3D2008%26volume%3D26%26spage%3D127%26epage%3D132%26doi%3D10.1038%2Fnbt1358" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref55"><div class="reference"><strong class="refLabel"><a href="#ref55" class="rightTabRefNumLink">55</a></strong><div class="NLM_citation" id="rightTab-cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bland, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Altman, D. G.</span></span> <span> </span><span class="NLM_article-title">The odds ratio</span>. <i>BMJ.</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>320</i></span>,  <span class="NLM_fpage">1468</span>– <span class="NLM_lpage">1468</span>, <span class="refDoi"> DOI: 10.1136/bmj.320.7247.1468</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=10.1136%2Fbmj.320.7247.1468" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=10827061" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=1%3ACAS%3A280%3ADC%252BD3c3os1Cluw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=320&publication_year=2000&pages=1468-1468&author=J.+M.+Blandauthor=D.+G.+Altman&title=The+odds+ratio&doi=10.1136%2Fbmj.320.7247.1468"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Statistics notes. The odds ratio</span></div><div class="casAuthors">Bland J M; Altman D G</div><div class="citationInfo"><span class="NLM_cas:title">BMJ (Clinical research ed.)</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">320</span>
        (<span class="NLM_cas:issue">7247</span>),
    <span class="NLM_cas:pages">1468</span>
        ISSN:<span class="NLM_cas:issn">0959-8138</span>.
    </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSP_AzSFo_UuEGy1nwu7K2mfW6udTcc2ebaRxkj1vqyN7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD3c3os1Cluw%253D%253D&md5=48735af9458f863287a13b458bc84b79</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1136%2Fbmj.320.7247.1468&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1136%252Fbmj.320.7247.1468%26sid%3Dliteratum%253Aachs%26aulast%3DBland%26aufirst%3DJ.%2BM.%26aulast%3DAltman%26aufirst%3DD.%2BG.%26atitle%3DThe%2520odds%2520ratio%26jtitle%3DBMJ.%26date%3D2000%26volume%3D320%26spage%3D1468%26epage%3D1468%26doi%3D10.1136%2Fbmj.320.7247.1468" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref56"><div class="reference"><strong class="refLabel"><a href="#ref56" class="rightTabRefNumLink">56</a></strong><div class="NLM_citation" id="rightTab-cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shtivelman, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hensing, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simon, G. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dennis, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Otterson, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bueno, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salgia, R.</span></span> <span> </span><span class="NLM_article-title">Molecular pathways and therapeutic targets in lung cancer</span>. <i>Oncotarget</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">1392</span>– <span class="NLM_lpage">1433</span>, <span class="refDoi"> DOI: 10.18632/oncotarget.1891</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=10.18632%2Foncotarget.1891" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=24722523" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=1%3ACAS%3A280%3ADC%252BC2cnisFyhtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&pages=1392-1433&author=E.+Shtivelmanauthor=T.+Hensingauthor=G.+R.+Simonauthor=P.+A.+Dennisauthor=G.+A.+Ottersonauthor=R.+Buenoauthor=R.+Salgia&title=Molecular+pathways+and+therapeutic+targets+in+lung+cancer&doi=10.18632%2Foncotarget.1891"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular pathways and therapeutic targets in lung cancer</span></div><div class="casAuthors">Shtivelman Emma; Hensing Thomas; Simon George R; Dennis Phillip A; Otterson Gregory A; Bueno Raphael; Salgia Ravi</div><div class="citationInfo"><span class="NLM_cas:title">Oncotarget</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1392-433</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Lung cancer is still the leading cause of cancer death worldwide.  Both histologically and molecularly lung cancer is heterogeneous.  This review summarizes the current knowledge of the pathways involved in the various types of lung cancer with an emphasis on the clinical implications of the increasing number of actionable molecular targets.  It describes the major pathways and molecular alterations implicated in the development and progression of non-small cell lung cancer (adenocarcinoma and squamous cancer), and of small cell carcinoma, emphasizing the molecular alterations comprising the specific blueprints in each group.  The approved and investigational targeted therapies as well as the immune therapies, and clinical trials exploring the variety of targeted approaches to treatment of lung cancer are the main focus of this review.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTTeNjg0C8I6m52AMpI4GK2fW6udTcc2eYj_L-rIxCFeLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2cnisFyhtQ%253D%253D&md5=9f0bff44699caa27f8c80f3c7d26d4c1</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.18632%2Foncotarget.1891&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncotarget.1891%26sid%3Dliteratum%253Aachs%26aulast%3DShtivelman%26aufirst%3DE.%26aulast%3DHensing%26aufirst%3DT.%26aulast%3DSimon%26aufirst%3DG.%2BR.%26aulast%3DDennis%26aufirst%3DP.%2BA.%26aulast%3DOtterson%26aufirst%3DG.%2BA.%26aulast%3DBueno%26aufirst%3DR.%26aulast%3DSalgia%26aufirst%3DR.%26atitle%3DMolecular%2520pathways%2520and%2520therapeutic%2520targets%2520in%2520lung%2520cancer%26jtitle%3DOncotarget%26date%3D2014%26volume%3D5%26spage%3D1392%26epage%3D1433%26doi%3D10.18632%2Foncotarget.1891" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref57"><div class="reference"><strong class="refLabel"><a href="#ref57" class="rightTabRefNumLink">57</a></strong><div class="NLM_citation" id="rightTab-cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Janku, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garrido-Laguna, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petruzelka, L. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stewart, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kurzrock, R.</span></span> <span> </span><span class="NLM_article-title">Novel therapeutic targets in non-small cell lung cancer</span>. <i>J. Thorac. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">1601</span>– <span class="NLM_lpage">1612</span>, <span class="refDoi"> DOI: 10.1097/JTO.0b013e31822944b3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=10.1097%2FJTO.0b013e31822944b3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=21849857" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=1%3ACAS%3A280%3ADC%252BC3MjntFKmuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2011&pages=1601-1612&author=F.+Jankuauthor=I.+Garrido-Lagunaauthor=L.+B.+Petruzelkaauthor=D.+J.+Stewartauthor=R.+Kurzrock&title=Novel+therapeutic+targets+in+non-small+cell+lung+cancer&doi=10.1097%2FJTO.0b013e31822944b3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">Novel therapeutic targets in non-small cell lung cancer</span></div><div class="casAuthors">Janku Filip; Garrido-Laguna Ignacio; Petruzelka Lubos B; Stewart David J; Kurzrock Razelle</div><div class="citationInfo"><span class="NLM_cas:title">Journal of thoracic oncology : official publication of the International Association for the Study of Lung Cancer</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1601-12</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The development of personalized medicine with a focus on novel targeted therapies has supplanted the one-size-fits-all approach to the treatment of many cancers, including non-small cell lung cancer.  Targeted therapies, if given to a patient subpopulation enriched by the presence of relevant molecular targets, can often abrogate cell signaling that perpetuates cancer progression.  Critical targets activating procancer pathways include, but are not limited to, epidermal growth factor receptor (EGFR), hepatocyte growth factor receptor (MET), vascular endothelial growth factor (VEGF), VEGF receptor, GTPase KRAS (KRAS), receptor tyrosine protein kinase erbB-2 (HER2), echinoderm microtubule-associated protein-like 4-anaplastic lymphoma kinase (EML4-ALK), phosphatidylinositol-4,5-bisphosphate 3-kinase catalytic subunit alpha isoform (PIK3CA), serine/threonine-protein kinase B-raf (BRAF), and insulin-like growth factor 1 receptor (IGF-1R).  Some target-directed therapies, such as epidermal growth factor receptor tyrosine kinase inhibitors and anti-VEGF monoclonal antibody, have already been approved for clinical use.  Others, such as those targeted to MET, VEGFR, HER2, PIK3CA, and IGF-1R, are in clinical testing.  This review describes molecular targets in non-small cell lung cancer that are in development or being clinically applied and their implications for developing novel anticancer therapies for this previously refractory malignancy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcT-fAdn102Pmvh3YQQtAhNufW6udTcc2eYj_L-rIxCFeLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3MjntFKmuw%253D%253D&md5=17357a9eb28282926945e3b7fc93b86f</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1097%2FJTO.0b013e31822944b3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1097%252FJTO.0b013e31822944b3%26sid%3Dliteratum%253Aachs%26aulast%3DJanku%26aufirst%3DF.%26aulast%3DGarrido-Laguna%26aufirst%3DI.%26aulast%3DPetruzelka%26aufirst%3DL.%2BB.%26aulast%3DStewart%26aufirst%3DD.%2BJ.%26aulast%3DKurzrock%26aufirst%3DR.%26atitle%3DNovel%2520therapeutic%2520targets%2520in%2520non-small%2520cell%2520lung%2520cancer%26jtitle%3DJ.%2520Thorac.%2520Oncol.%26date%3D2011%26volume%3D6%26spage%3D1601%26epage%3D1612%26doi%3D10.1097%2FJTO.0b013e31822944b3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref58"><div class="reference"><strong class="refLabel"><a href="#ref58" class="rightTabRefNumLink">58</a></strong><div class="NLM_citation" id="rightTab-cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Reungwetwattana, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dy, G. K.</span></span> <span> </span><span class="NLM_article-title">Targeted therapies in development for non-small cell lung cancer</span>. <i>J. Carcinog.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">22</span>, <span class="refDoi"> DOI: 10.4103/1477-3163.123972</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=10.4103%2F1477-3163.123972" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=24574860" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=1%3ACAS%3A280%3ADC%252BC2cfls1Gmug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2013&pages=22&author=T.+Reungwetwattanaauthor=G.+K.+Dy&title=Targeted+therapies+in+development+for+non-small+cell+lung+cancer&doi=10.4103%2F1477-3163.123972"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">Targeted therapies in development for non-small cell lung cancer</span></div><div class="casAuthors">Reungwetwattana Thanyanan; Dy Grace Kho</div><div class="citationInfo"><span class="NLM_cas:title">Journal of carcinogenesis</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">22</span>
        ISSN:<span class="NLM_cas:issn">1477-3163</span>.
    </div><div class="casAbstract">The iterative discovery in various malignancies during the past decades that a number of aberrant tumorigenic processes and signal transduction pathways are mediated by "druggable" protein kinases has led to a revolutionary change in drug development.  In non-small cell lung cancer (NSCLC), the ErbB family of receptors (e.g., EGFR [epidermal growth factor receptor], HER2 [human epidermal growth factor receptor 2]), RAS (rat sarcoma gene), BRAF (v-raf murine sarcoma viral oncogene homolog B1), MAPK (mitogen-activated protein kinase) c-MET (c-mesenchymal-epithelial transition), FGFR (fibroblast growth factor receptor), DDR2 (discoidin domain receptor 2), PIK3CA (phosphatidylinositol-4,5-bisphosphate3-kinase, catalytic subunit alpha)), PTEN (phosphatase and tensin homolog), AKT (protein kinase B), ALK (anaplastic lym phoma kinase), RET (rearranged during transfection), ROS1 (reactive oxygen species 1) and EPH (erythropoietin-producing hepatoma) are key targets of various agents currently in clinical development.  These oncogenic targets exert their selective growth advantage through various intercommunicating pathways, such as through RAS/RAF/MEK, phosphoinositide 3-kinase/AKT/mammalian target of rapamycin and SRC-signal transduction and transcription signaling.  The recent clinical studies, EGFR tyrosine kinase inhibitors and crizotinib were considered as strongly effective targeted therapies in metastatic NSCLC.  Currently, five molecular targeted agents were approved for treatment of advanced NSCLC: Gefitinib, erlotinib and afatinib for positive EGFR mutation, crizotinib for positive echinoderm microtubule-associated protein-like 4 (EML4)-ALK translocation and bevacizumab.  Moreover, oncogenic mutant proteins are subject to regulation by protein trafficking pathways, specifically through the heat shock protein 90 system.  Drug combinations affecting various nodes in these signaling and intracellular processes are predicted and demonstrated to be synergistic and advantageous in overcoming treatment resistance compared with monotherapy approaches.  Understanding the role of the tumor microenvironment in the development and maintenance of the malignant phenotype provided additional therapeutic approaches as well.  More recently, improved knowledge on tumor immunology has set the stage for promising immunotherapies in NSCLC.  This review will focus on the rationale for the development of targeted therapies in NSCLC and the various strategies employed in preventing or overcoming the inevitable occurrence of treatment resistance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQlh8ppikYuFoqdXsLcGTEUfW6udTcc2eYj_L-rIxCFeLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2cfls1Gmug%253D%253D&md5=7bd8f4f93f5c7a2b19523fde35af75b7</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.4103%2F1477-3163.123972&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4103%252F1477-3163.123972%26sid%3Dliteratum%253Aachs%26aulast%3DReungwetwattana%26aufirst%3DT.%26aulast%3DDy%26aufirst%3DG.%2BK.%26atitle%3DTargeted%2520therapies%2520in%2520development%2520for%2520non-small%2520cell%2520lung%2520cancer%26jtitle%3DJ.%2520Carcinog.%26date%3D2013%26volume%3D12%26spage%3D22%26doi%3D10.4103%2F1477-3163.123972" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref59"><div class="reference"><strong class="refLabel"><a href="#ref59" class="rightTabRefNumLink">59</a></strong><div class="NLM_citation" id="rightTab-cit59"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chan, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hughes, B. G. M.</span></span> <span> </span><span class="NLM_article-title">Targeted therapy for non-small cell lung cancer: current standards and the promise of the future</span>. <i>Transl. Lung Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">36</span>– <span class="NLM_lpage">54</span>, <span class="refDoi"> DOI: 10.3978/j.issn.2218-6751.2014.05.01</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=10.3978%2Fj.issn.2218-6751.2014.05.01" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=25806345" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtFCms7vL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2015&pages=36-54&author=B.+A.+Chanauthor=B.+G.+M.+Hughes&title=Targeted+therapy+for+non-small+cell+lung+cancer%3A+current+standards+and+the+promise+of+the+future&doi=10.3978%2Fj.issn.2218-6751.2014.05.01"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">Targeted therapy for non-small cell lung cancer: current standards and the promise of the future</span></div><div class="casAuthors">Chan, Bryan A.; Hughes, Brett G. M.</div><div class="citationInfo"><span class="NLM_cas:title">Translational Lung Cancer Research</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">36-54</span>CODEN:
                <span class="NLM_cas:coden">TLCRA9</span>;
        ISSN:<span class="NLM_cas:issn">2226-4477</span>.
    
            (<span class="NLM_cas:orgname">Pioneer Bioscience Publishing Co.</span>)
        </div><div class="casAbstract">In recent years, there has been a major paradigm shift in the management of non-small cell lung cancer (NSCLC).  NSCLC should now be further sub-classified by histol. and driver mutation if one is known or present.  Translational research advances now allow such mutations to be inhibited by either receptor monoclonal antibodies (mAb) or small mol. tyrosine kinase inhibitors (TKI).  While empirical chemotherapy with a platinum-doublet remains the gold std. for advanced NSCLC without a known driver mutation, targeted therapy is pushing the boundary to significantly improve patient outcomes and quality of life.  In this review, we will examine the major subtypes of oncogenic drivers behind NSCLC as well as the development of targeted agents available to treat them both now and in the foreseeable future.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrpzORp4rfH87Vg90H21EOLACvtfcHk0liTYRdNj4xr2Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtFCms7vL&md5=0e9254cd6c0bacd25d1776c295f45207</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.3978%2Fj.issn.2218-6751.2014.05.01&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3978%252Fj.issn.2218-6751.2014.05.01%26sid%3Dliteratum%253Aachs%26aulast%3DChan%26aufirst%3DB.%2BA.%26aulast%3DHughes%26aufirst%3DB.%2BG.%2BM.%26atitle%3DTargeted%2520therapy%2520for%2520non-small%2520cell%2520lung%2520cancer%253A%2520current%2520standards%2520and%2520the%2520promise%2520of%2520the%2520future%26jtitle%3DTransl.%2520Lung%2520Cancer%2520Res.%26date%3D2015%26volume%3D4%26spage%3D36%26epage%3D54%26doi%3D10.3978%2Fj.issn.2218-6751.2014.05.01" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref60"><div class="reference"><strong class="refLabel"><a href="#ref60" class="rightTabRefNumLink">60</a></strong><div class="NLM_citation" id="rightTab-cit60"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Giaccone, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zucali, P. A.</span></span> <span> </span><span class="NLM_article-title">Src as a potential therapeutic target in non-small-cell lung cancer</span>. <i>Ann. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">1219</span>– <span class="NLM_lpage">1223</span>, <span class="refDoi"> DOI: 10.1093/annonc/mdn048</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=10.1093%2Fannonc%2Fmdn048" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=18388349" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=1%3ACAS%3A280%3ADC%252BD1cvgvVyrsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2008&pages=1219-1223&author=G.+Giacconeauthor=P.+A.+Zucali&title=Src+as+a+potential+therapeutic+target+in+non-small-cell+lung+cancer&doi=10.1093%2Fannonc%2Fmdn048"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">Src as a potential therapeutic target in non-small-cell lung cancer</span></div><div class="casAuthors">Giaccone G; Zucali P A</div><div class="citationInfo"><span class="NLM_cas:title">Annals of oncology : official journal of the European Society for Medical Oncology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1219-1223</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Lung cancer is the most common cause of cancer-related death, with non-small-cell lung cancer (NSCLC) accounting for 80%-85% of all cases.  Although survival rates are reasonably good for patients diagnosed with very early disease, the majority of patients present with advanced disease.  For these patients, palliation and improvements in quality of life are the primary goals of therapy.  Although chemotherapeutic agents remain the cornerstone of first-line therapy, these agents have limited use in patients who have relapsed and have metastatic disease.  Therefore, new strategies are required to improve survival and quality of life in this setting.  With the substantial advances in our understanding of tumour biology, it has been possible to identify signalling pathways involved in mediating tumour growth and progression.  These pathways offer targets for new biological agents such as small molecule inhibitors and monoclonal antibodies.  One such target is Src, a tyrosine kinase that is involved in multiple aspects of tumorigenesis including proliferation, migration and angiogenesis.  Increased levels of Src expression have been found in a range of cancers, especially breast, colorectal, prostate and lung.  Preliminary preclinical data and pharmacodynamic data suggest that Src inhibition is a viable therapeutic option in the treatment of advanced NSCLC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSDd4hYJocBQtj_Vg2vObihfW6udTcc2eYQ9egKNey8Irntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD1cvgvVyrsw%253D%253D&md5=9e95b141cb44a1999b0625e34f30861a</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1093%2Fannonc%2Fmdn048&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fannonc%252Fmdn048%26sid%3Dliteratum%253Aachs%26aulast%3DGiaccone%26aufirst%3DG.%26aulast%3DZucali%26aufirst%3DP.%2BA.%26atitle%3DSrc%2520as%2520a%2520potential%2520therapeutic%2520target%2520in%2520non-small-cell%2520lung%2520cancer%26jtitle%3DAnn.%2520Oncol.%26date%3D2008%26volume%3D19%26spage%3D1219%26epage%3D1223%26doi%3D10.1093%2Fannonc%2Fmdn048" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref61"><div class="reference"><strong class="refLabel"><a href="#ref61" class="rightTabRefNumLink">61</a></strong><div class="NLM_citation" id="rightTab-cit61"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rusnak, D. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lackey, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Affleck, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wood, E. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alligood, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rhodes, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keith, B. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knight, W. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mullin, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilmer, T. M.</span></span> <span> </span><span class="NLM_article-title">The effects of the novel, reversible epidermal growth factor receptor/ErbB-2 tyrosine kinase inhibitor, GW2016, on the growth of human normal and tumor-derived cell lines in vitro and in vivo</span>. <i>Mol. Cancer Ther.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">85</span>– <span class="NLM_lpage">94</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=12467226" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=1%3ACAS%3A528%3ADC%252BD38XlsFSmtbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2001&pages=85-94&author=D.+W.+Rusnakauthor=K.+Lackeyauthor=K.+Affleckauthor=E.+R.+Woodauthor=K.+J.+Alligoodauthor=N.+Rhodesauthor=B.+R.+Keithauthor=D.+M.+Murrayauthor=W.+B.+Knightauthor=R.+J.+Mullinauthor=T.+M.+Gilmer&title=The+effects+of+the+novel%2C+reversible+epidermal+growth+factor+receptor%2FErbB-2+tyrosine+kinase+inhibitor%2C+GW2016%2C+on+the+growth+of+human+normal+and+tumor-derived+cell+lines+in+vitro+and+in+vivo"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">The effects of the novel, reversible epidermal growth factor receptor/ErbB-2 tyrosine kinase inhibitor, GW2016, on the growth of human normal and tumor-derived cell lines in vitro and in vivo</span></div><div class="casAuthors">Rusnak, David W.; Lackey, Karen; Affleck, Karen; Wood, Edgar R.; Alligood, Krystal J.; Rhodes, Nelson; Keith, Barry R.; Murray, Doris M.; Knight, W. Blaine; Mullin, Robert J.; Gilmer, Tona M.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cancer Therapeutics</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">85-94</span>CODEN:
                <span class="NLM_cas:coden">MCTOCF</span>;
        ISSN:<span class="NLM_cas:issn">1535-7163</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">The epidermal growth factor receptor (EGFR) and ErbB-2 transmembrane tyrosine kinases are currently being targeted by various mechanisms in the treatment of cancer.  GW2016 is a potent inhibitor of the ErbB-2 and EGFR tyrosine kinase domains with IC50 values against purified EGFR and ErbB-2 of 10.2 and 9.8 nM, resp.  This report describes the efficacy in cell growth assays of GW2016 on human tumor cell lines overexpressing either EGFR or ErbB-2: HN5 (head and neck), A-431 (vulva), BT474 (breast), CaLu-3 (lung), and N87 (gastric).  Normal human foreskin fibroblasts, nontumorigenic epithelial cells (HB4a), and nonoverexpressing tumor cells (MCF-7 and T47D) were tested as neg. controls.  After 3 days of compd. exposure, av. IC50 values for growth inhibition in the EGFR- and ErbB-2-overexpressing tumor cell lines were <0.16 μM.  The av. selectivity for the tumor cells vs. the human foreskin fibroblast cell line was 100-fold.  Inhibition of EGFR and ErbB-2 receptor autophosphorylation and phosphorylation of the downstream modulator, AKT, was verified by Western blot anal. in the BT474 and HN5 cell lines.  As a measure of cytotoxicity vs. growth arrest, the HN5 and BT474 cells were assessed in an outgrowth assay after a transient exposure to GW2016.  The cells were treated for 3 days in five concns. of GW2016, and cell growth was monitored for an addnl. 12 days after removal of the compd.  In each of these tumor cell lines, concns. of GW2016 were reached where outgrowth did not occur.  Furthermore, growth arrest and cell death were obsd. in parallel expts., as detd. by bromodeoxyuridine incorporation and propidium iodide staining.  GW2016 treatment inhibited tumor xenograft growth of the HN5 and BT474 cells in a dose-responsive manner at 30 and 100 mg/kg orally, twice daily, with complete inhibition of tumor growth at the higher dose.  Together, these results indicate that GW2016 achieves excellent potency on tumor cells with selectivity for tumor vs. normal cells and suggest that GW2016 has value as a therapy for patients with tumors overexpressing either EGFR or ErbB-2.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp5OJ41zsWu6rVg90H21EOLACvtfcHk0ljdEkLBwaAyng"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XlsFSmtbs%253D&md5=64fd8cadce83eda3b9b82ab50f1e57b0</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DRusnak%26aufirst%3DD.%2BW.%26aulast%3DLackey%26aufirst%3DK.%26aulast%3DAffleck%26aufirst%3DK.%26aulast%3DWood%26aufirst%3DE.%2BR.%26aulast%3DAlligood%26aufirst%3DK.%2BJ.%26aulast%3DRhodes%26aufirst%3DN.%26aulast%3DKeith%26aufirst%3DB.%2BR.%26aulast%3DMurray%26aufirst%3DD.%2BM.%26aulast%3DKnight%26aufirst%3DW.%2BB.%26aulast%3DMullin%26aufirst%3DR.%2BJ.%26aulast%3DGilmer%26aufirst%3DT.%2BM.%26atitle%3DThe%2520effects%2520of%2520the%2520novel%252C%2520reversible%2520epidermal%2520growth%2520factor%2520receptor%252FErbB-2%2520tyrosine%2520kinase%2520inhibitor%252C%2520GW2016%252C%2520on%2520the%2520growth%2520of%2520human%2520normal%2520and%2520tumor-derived%2520cell%2520lines%2520in%2520vitro%2520and%2520in%2520vivo%26jtitle%3DMol.%2520Cancer%2520Ther.%26date%3D2001%26volume%3D1%26spage%3D85%26epage%3D94" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref62"><div class="reference"><strong class="refLabel"><a href="#ref62" class="rightTabRefNumLink">62</a></strong><div class="NLM_citation" id="rightTab-cit62"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ross, H. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blumenschein, G. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aisner, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Damjanov, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dowlati, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garst, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rigas, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smylie, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hassani, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allen, K. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leopold, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zaks, T. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shepherd, F. A.</span></span> <span> </span><span class="NLM_article-title">Randomized phase II multicenter trial of two schedules of lapatinib as first-or second-line monotherapy in patients with advanced or metastatic non-small cell lung cancer</span>. <i>Clin. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">1938</span>– <span class="NLM_lpage">1949</span>, <span class="refDoi"> DOI: 10.1158/1078-0432.CCR-08-3328</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=10.1158%2F1078-0432.CCR-08-3328" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=20215545" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=1%3ACAS%3A528%3ADC%252BC3cXjtFyhurY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2010&pages=1938-1949&author=H.+J.+Rossauthor=G.+R.+Blumenscheinauthor=J.+Aisnerauthor=N.+Damjanovauthor=A.+Dowlatiauthor=J.+Garstauthor=J.+R.+Rigasauthor=M.+Smylieauthor=H.+Hassaniauthor=K.+E.+Allenauthor=L.+Leopoldauthor=T.+Z.+Zaksauthor=F.+A.+Shepherd&title=Randomized+phase+II+multicenter+trial+of+two+schedules+of+lapatinib+as+first-or+second-line+monotherapy+in+patients+with+advanced+or+metastatic+non-small+cell+lung+cancer&doi=10.1158%2F1078-0432.CCR-08-3328"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit62R"><div class="casContent"><span class="casTitleNuber">62</span><div class="casTitle"><span class="NLM_cas:atitle">Randomized Phase II Multicenter Trial of Two Schedules of Lapatinib as First- or Second-Line Monotherapy in Patients with Advanced or Metastatic Non-Small Cell Lung Cancer</span></div><div class="casAuthors">Ross, Helen J.; Blumenschein, George R., Jr.; Aisner, Joseph; Damjanov, Nevena; Dowlati, Afshin; Garst, Jennifer; Rigas, James R.; Smylie, Michael; Hassani, Habib; Allen, Kimberly E.; Leopold, Lance; Zaks, Tal Z.; Shepherd, Frances A.</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1938-1949</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">PURPOSE: This randomized phase II study was initially designed to test the activity of two dose schedules of lapatinib (GW572016H), an oral, reversible, dual tyrosine kinase inhibitor of epidermal growth factor receptor (EGFR) and human EGFR-2 (HER2/neu; HER2), in chemotherapy-naive patients with non-small cell lung cancer (NSCLC); it was later amended to target patients with bronchioloalveolar carcinoma or no smoking history.  Exptl. Design: Patients with good performance status and recurrent or metastatic NSCLC were randomized to lapatinib (orally, 1,500 mg once daily or 500 mg twice daily) until progression or intolerance.  Patients could have had a max. of one prior systemic therapy (chemotherapy or biol. therapy) for NSCLC.  Safety and activity were assessed every 4 and 8 wk, resp.  Tumors were analyzed for EGFR and HER2 mutations and/or amplifications.  RESULTS: Of 75 patients in the nontargeted population, 1 (1.3%) had partial response and 16 (21%) had stable disease of ≥24 wk.  No complete or partial responses were obsd. in 56 patients in the targeted population; 14 (25%) had stable disease of ≥24 wk.  No responses were seen in three patients with EGFR mutations and five with EGFR gene amplification.  No mutations in HER2 were found.  One of two patients with HER2 amplification had a 51% decrease in tumor size; however, this response was unconfirmed.  The most common adverse events were grade 1 or 2 diarrhea, rash, fatigue, nausea, and anorexia.  Adverse events were similar across dosing regimens.  CONCLUSIONS: Lapatinib was well tolerated, with no notable difference in toxicity between treatment groups.  Lapatinib monotherapy did not induce a significant no. of tumor regressions in NSCLC.  Further studies may be warranted to det. whether lapatinib is active in combination with other agents in the treatment of NSCLC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo-5ZsLpn50LrVg90H21EOLACvtfcHk0lj7oKCd8m5maw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXjtFyhurY%253D&md5=0bfa99df35f30bb9e9ef0b2addb23e28</span></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1158%2F1078-0432.CCR-08-3328&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1078-0432.CCR-08-3328%26sid%3Dliteratum%253Aachs%26aulast%3DRoss%26aufirst%3DH.%2BJ.%26aulast%3DBlumenschein%26aufirst%3DG.%2BR.%26aulast%3DAisner%26aufirst%3DJ.%26aulast%3DDamjanov%26aufirst%3DN.%26aulast%3DDowlati%26aufirst%3DA.%26aulast%3DGarst%26aufirst%3DJ.%26aulast%3DRigas%26aufirst%3DJ.%2BR.%26aulast%3DSmylie%26aufirst%3DM.%26aulast%3DHassani%26aufirst%3DH.%26aulast%3DAllen%26aufirst%3DK.%2BE.%26aulast%3DLeopold%26aufirst%3DL.%26aulast%3DZaks%26aufirst%3DT.%2BZ.%26aulast%3DShepherd%26aufirst%3DF.%2BA.%26atitle%3DRandomized%2520phase%2520II%2520multicenter%2520trial%2520of%2520two%2520schedules%2520of%2520lapatinib%2520as%2520first-or%2520second-line%2520monotherapy%2520in%2520patients%2520with%2520advanced%2520or%2520metastatic%2520non-small%2520cell%2520lung%2520cancer%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D2010%26volume%3D16%26spage%3D1938%26epage%3D1949%26doi%3D10.1158%2F1078-0432.CCR-08-3328" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref63"><div class="reference"><strong class="refLabel"><a href="#ref63" class="rightTabRefNumLink">63</a></strong><div class="NLM_citation" id="rightTab-cit63"><span><span class="NLM_contrib-group"><span class="NLM_string-name">O’Hare, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shakespeare, W. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eide, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rivera, V. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adrian, L. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, W.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Metcalf, C. A.,  III</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tyner, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loriaux, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Corbin, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wardwell, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ning, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keats, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sundaramoorthi, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Snodgrass, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Commodore, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sawyer, T. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dalgarno, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deininger, M. W. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Druker, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clackson, T.</span></span> <span> </span><span class="NLM_article-title">AP24534, a pan-BCR-ABL inhibitor for chronic myeloid leukemia, potently inhibits the T315I mutant and overcomes mutation-based resistance</span>. <i>Cancer Cell</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">401</span>– <span class="NLM_lpage">412</span>, <span class="refDoi"> DOI: 10.1016/j.ccr.2009.09.028</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=10.1016%2Fj.ccr.2009.09.028" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=19878872" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsFKltb3N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2009&pages=401-412&author=T.+O%E2%80%99Hareauthor=W.+C.+Shakespeareauthor=X.+Zhuauthor=C.+A.+Eideauthor=V.+M.+Riveraauthor=F.+Wangauthor=L.+T.+Adrianauthor=T.+Zhouauthor=W.-S.+Huangauthor=Q.+Xuauthor=C.+A.+Metcalfauthor=J.+W.+Tynerauthor=M.+M.+Loriauxauthor=A.+S.+Corbinauthor=S.+Wardwellauthor=Y.+Ningauthor=J.+A.+Keatsauthor=Y.+Wangauthor=R.+Sundaramoorthiauthor=M.+Thomasauthor=D.+Zhouauthor=J.+Snodgrassauthor=L.+Commodoreauthor=T.+K.+Sawyerauthor=D.+C.+Dalgarnoauthor=M.+W.+N.+Deiningerauthor=B.+J.+Drukerauthor=T.+Clackson&title=AP24534%2C+a+pan-BCR-ABL+inhibitor+for+chronic+myeloid+leukemia%2C+potently+inhibits+the+T315I+mutant+and+overcomes+mutation-based+resistance&doi=10.1016%2Fj.ccr.2009.09.028"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit63R"><div class="casContent"><span class="casTitleNuber">63</span><div class="casTitle"><span class="NLM_cas:atitle">AP24534, a pan-BCR-ABL inhibitor for chronic myeloid leukemia, potently inhibits the T315I mutant and overcomes mutation-based resistance</span></div><div class="casAuthors">O'Hare, Thomas; Shakespeare, William C.; Zhu, Xiaotian; Eide, Christopher A.; Rivera, Victor M.; Wang, Frank; Adrian, Lauren T.; Zhou, Tianjun; Huang, Wei-Sheng; Xu, Qihong; Metcalf, Chester A., III; Tyner, Jeffrey W.; Loriaux, Marc M.; Corbin, Amie S.; Wardwell, Scott; Ning, Yaoyu; Keats, Jeffrey A.; Wang, Yihan; Sundaramoorthi, Raji; Thomas, Mathew; Zhou, Dong; Snodgrass, Joseph; Commodore, Lois; Sawyer, Tomi K.; Dalgarno, David C.; Deininger, Michael W. N.; Druker, Brian J.; Clackson, Tim</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">401-412</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Inhibition of BCR-ABL by imatinib induces durable responses in many patients with chronic myeloid leukemia (CML), but resistance attributable to kinase domain mutations can lead to relapse and a switch to second-line therapy with nilotinib or dasatinib.  Despite three approved therapeutic options, the cross-resistant BCR-ABLT315I mutation and compd. mutants selected on sequential inhibitor therapy remain major clin. challenges.  We report design and preclin. evaluation of AP24534, a potent, orally available multitargeted kinase inhibitor active against T315I and other BCR-ABL mutants.  AP24534 inhibited all tested BCR-ABL mutants in cellular and biochem. assays, suppressed BCR-ABLT315I-driven tumor growth in mice, and completely abrogated resistance in cell-based mutagenesis screens.  Our work supports clin. evaluation of AP24534 as a pan-BCR-ABL inhibitor for treatment of CML.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGox3g7ZYNXwLbVg90H21EOLACvtfcHk0lj7oKCd8m5maw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsFKltb3N&md5=62a383734fc345b97647ba71aa7aee04</span></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1016%2Fj.ccr.2009.09.028&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccr.2009.09.028%26sid%3Dliteratum%253Aachs%26aulast%3DO%25E2%2580%2599Hare%26aufirst%3DT.%26aulast%3DShakespeare%26aufirst%3DW.%2BC.%26aulast%3DZhu%26aufirst%3DX.%26aulast%3DEide%26aufirst%3DC.%2BA.%26aulast%3DRivera%26aufirst%3DV.%2BM.%26aulast%3DWang%26aufirst%3DF.%26aulast%3DAdrian%26aufirst%3DL.%2BT.%26aulast%3DZhou%26aufirst%3DT.%26aulast%3DHuang%26aufirst%3DW.-S.%26aulast%3DXu%26aufirst%3DQ.%26aulast%3DMetcalf%26aufirst%3DC.%2BA.%26aulast%3DTyner%26aufirst%3DJ.%2BW.%26aulast%3DLoriaux%26aufirst%3DM.%2BM.%26aulast%3DCorbin%26aufirst%3DA.%2BS.%26aulast%3DWardwell%26aufirst%3DS.%26aulast%3DNing%26aufirst%3DY.%26aulast%3DKeats%26aufirst%3DJ.%2BA.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DSundaramoorthi%26aufirst%3DR.%26aulast%3DThomas%26aufirst%3DM.%26aulast%3DZhou%26aufirst%3DD.%26aulast%3DSnodgrass%26aufirst%3DJ.%26aulast%3DCommodore%26aufirst%3DL.%26aulast%3DSawyer%26aufirst%3DT.%2BK.%26aulast%3DDalgarno%26aufirst%3DD.%2BC.%26aulast%3DDeininger%26aufirst%3DM.%2BW.%2BN.%26aulast%3DDruker%26aufirst%3DB.%2BJ.%26aulast%3DClackson%26aufirst%3DT.%26atitle%3DAP24534%252C%2520a%2520pan-BCR-ABL%2520inhibitor%2520for%2520chronic%2520myeloid%2520leukemia%252C%2520potently%2520inhibits%2520the%2520T315I%2520mutant%2520and%2520overcomes%2520mutation-based%2520resistance%26jtitle%3DCancer%2520Cell%26date%3D2009%26volume%3D16%26spage%3D401%26epage%3D412%26doi%3D10.1016%2Fj.ccr.2009.09.028" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref64"><div class="reference"><strong class="refLabel"><a href="#ref64" class="rightTabRefNumLink">64</a></strong><div class="NLM_citation" id="rightTab-cit64"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ren, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hong, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Biddinger, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Silva, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowell, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hao, Z.</span></span> <span> </span><span class="NLM_article-title">Novel FGFR inhibitor ponatinib suppresses the growth of non-small cell lung cancer cells overexpressing FGFR1</span>. <i>Oncol. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">2181</span>– <span class="NLM_lpage">2190</span>, <span class="refDoi"> DOI: 10.3892/or.2013.2386</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=10.3892%2For.2013.2386" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=23563700" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=1%3ACAS%3A528%3ADC%252BC3sXpsFCksbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2013&pages=2181-2190&author=M.+Renauthor=M.+Hongauthor=G.+Liuauthor=H.+Wangauthor=V.+Patelauthor=P.+Biddingerauthor=J.+Silvaauthor=J.+Cowellauthor=Z.+Hao&title=Novel+FGFR+inhibitor+ponatinib+suppresses+the+growth+of+non-small+cell+lung+cancer+cells+overexpressing+FGFR1&doi=10.3892%2For.2013.2386"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit64R"><div class="casContent"><span class="casTitleNuber">64</span><div class="casTitle"><span class="NLM_cas:atitle">Novel FGFR inhibitor ponatinib suppresses the growth of non-small cell lung cancer cells overexpressing FGFR1</span></div><div class="casAuthors">Ren, Mingqiang; Hong, Mei; Liu, Gentao; Wang, Hongjin; Patel, Vijay; Biddinger, Paul; Silva, Jeane; Cowell, John; Hao, Zhonglin</div><div class="citationInfo"><span class="NLM_cas:title">Oncology Reports</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2181-2190</span>CODEN:
                <span class="NLM_cas:coden">OCRPEW</span>;
        ISSN:<span class="NLM_cas:issn">1021-335X</span>.
    
            (<span class="NLM_cas:orgname">Spandidos Publications Ltd.</span>)
        </div><div class="casAbstract">Lung cancer is still the leading cause of cancer-related deaths worldwide.  Identifying new oncogenic drivers and developing efficient inhibitors through mol. targeting approaches are crucial for improving therapies.  The aim of this study was to investigate whether targeting fibroblast growth factor receptor 1 (FGFR1) with ponatinib inhibits the cell growth in both established and primary lung cancer cells over-expressing FGFR1.  Eighty-eight non-small cell lung cancer (NSCLC) and paired normal tissue specimens were analyzed by real-time RT-PCR for FGFR1 gene expression.  We identified four cell lines and two newly established primary lung cancer cultures that showed high FGFR1 expression levels, and evaluated the effect of the novel FGFR1 inhibitor ponatinib on cell growth.  Approx. 50% (30 out of 59) NSCLC specimens expressed FGFR1 >2-fold compared with their adjacent normal counterparts using quant. RT-PCR.  Ponatinib treatment of established NSCLC cell lines expressing higher levels of FGFR1 resulted in marked cell growth inhibition and suppression of clonogenicity.  This growth inhibition was assocd. with inactivation of FGFR1 and its downstream targets.  FGFR1 knockdown by shRNA achieved similar results when compared to treatment with ponatinib.  Furthermore, ponatinib was able to significantly inhibit the growth of primary lung cancer cultures in vitro.  Our data indicate that pharmacol. inhibition of FGFR1 kinase activity with ponatinib may be effective for the treatment of lung cancer patients whose tumors overexpress FGFR1.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpai6Rg1CUEubVg90H21EOLACvtfcHk0liQbLgccRfEeg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXpsFCksbc%253D&md5=52bd8443e6319dd96736d4b2628c24d9</span></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=10.3892%2For.2013.2386&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3892%252For.2013.2386%26sid%3Dliteratum%253Aachs%26aulast%3DRen%26aufirst%3DM.%26aulast%3DHong%26aufirst%3DM.%26aulast%3DLiu%26aufirst%3DG.%26aulast%3DWang%26aufirst%3DH.%26aulast%3DPatel%26aufirst%3DV.%26aulast%3DBiddinger%26aufirst%3DP.%26aulast%3DSilva%26aufirst%3DJ.%26aulast%3DCowell%26aufirst%3DJ.%26aulast%3DHao%26aufirst%3DZ.%26atitle%3DNovel%2520FGFR%2520inhibitor%2520ponatinib%2520suppresses%2520the%2520growth%2520of%2520non-small%2520cell%2520lung%2520cancer%2520cells%2520overexpressing%2520FGFR1%26jtitle%3DOncol.%2520Rep.%26date%3D2013%26volume%3D29%26spage%3D2181%26epage%3D2190%26doi%3D10.3892%2For.2013.2386" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref65"><div class="reference"><strong class="refLabel"><a href="#ref65" class="rightTabRefNumLink">65</a></strong><div class="NLM_citation" id="rightTab-cit65"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Daud, A. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krishnamurthi, S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saleh, M. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gitlitz, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Borad, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gold, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chiorean, E. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Springett, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abbas, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Agarwal, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bardy-Bouxin, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hsyu, P.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leip, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turnbull, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zacharchuk, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Messersmith, W. A.</span></span> <span> </span><span class="NLM_article-title">Phase I study of bosutinib, a src/abl tyrosine kinase inhibitor, administered to patients with advanced solid tumors</span>. <i>Clin. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">1092</span>– <span class="NLM_lpage">1100</span>, <span class="refDoi"> DOI: 10.1158/1078-0432.CCR-11-2378</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=10.1158%2F1078-0432.CCR-11-2378" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=22179664" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=1%3ACAS%3A528%3ADC%252BC38XisVGhu7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2012&pages=1092-1100&author=A.+I.+Daudauthor=S.+S.+Krishnamurthiauthor=M.+N.+Salehauthor=B.+J.+Gitlitzauthor=M.+J.+Boradauthor=P.+J.+Goldauthor=E.+G.+Chioreanauthor=G.+M.+Springettauthor=R.+Abbasauthor=S.+Agarwalauthor=N.+Bardy-Bouxinauthor=P.-H.+Hsyuauthor=E.+Leipauthor=K.+Turnbullauthor=C.+Zacharchukauthor=W.+A.+Messersmith&title=Phase+I+study+of+bosutinib%2C+a+src%2Fabl+tyrosine+kinase+inhibitor%2C+administered+to+patients+with+advanced+solid+tumors&doi=10.1158%2F1078-0432.CCR-11-2378"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit65R"><div class="casContent"><span class="casTitleNuber">65</span><div class="casTitle"><span class="NLM_cas:atitle">Phase I Study of Bosutinib, a Src/Abl Tyrosine Kinase Inhibitor, Administered to Patients with Advanced Solid Tumors</span></div><div class="casAuthors">Daud, Adil I.; Krishnamurthi, Smitha S.; Saleh, Mansoor N.; Gitlitz, Barbara J.; Borad, Mitesh J.; Gold, Philip J.; Chiorean, Elena G.; Springett, Gregory M.; Abbas, Richat; Agarwal, Shefali; Bardy-Bouxin, Nathalie; Hsyu, Poe-Hirr; Leip, Eric; Turnbull, Kathleen; Zacharchuk, Charles; Messersmith, Wells A.</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1092-1100</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">PURPOSE: Bosutinib, a potent ATP-competitive, quinolinecarbonitrile Src/Abl kinase inhibitor, was tested in this first-in-human phase I trial in patients with advanced solid tumor malignancies.  PATIENTS AND METHODS: This trial was conducted in 2 parts.  In part 1 (dose escalation), increasing oral bosutinib doses were administered using a 3 + 3 design.  In part 2 (dose expansion), approx. 30 patients each with refractory colorectal, pancreas, or non-small cell lung cancer were treated at the recommended phase II dose (RP2D).  Primary efficacy endpoints for part 2 were median progression-free survival (colorectal and non-small cell lung) and median overall survival (pancreas).  RESULTS: In part 1, dose-limiting toxicities of grade 3 diarrhea (two patients) and grade 3 rash occurred with bosutinib 600 mg/day and the max. tolerated dose identified was 500 mg/day.  However, the majority of patients treated with 500 mg/day had grade 2 or greater gastrointestinal toxicity, and 400 mg/day was identified as the RP2D.  The most common bosutinib-related adverse events were nausea (60% patients), diarrhea (47%), vomiting (40%), fatigue (38%), and anorexia (36%).  Bosutinib had a mean half-life of 19 to 20 h at the RP2D.  A partial response (breast) and unconfirmed complete response (pancreas) were obsd.; 8 of 112 evaluable patients had stable disease for 22 to 101 wk.  However, the primary efficacy endpoints for part 2 were not met.  CONCLUSIONS: Bosutinib was generally well tolerated in patients with solid tumors, with the main toxicity being gastrointestinal.  The RP2D was 400 mg/day orally.  Further study of bosutinib is planned in combination regimens.  Clin Cancer Res; 18(4); 1092-100.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrYIJzJMnax9LVg90H21EOLACvtfcHk0liQbLgccRfEeg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XisVGhu7Y%253D&md5=6ed35e2929a66d5f7f8507cfdc085ba2</span></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=10.1158%2F1078-0432.CCR-11-2378&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1078-0432.CCR-11-2378%26sid%3Dliteratum%253Aachs%26aulast%3DDaud%26aufirst%3DA.%2BI.%26aulast%3DKrishnamurthi%26aufirst%3DS.%2BS.%26aulast%3DSaleh%26aufirst%3DM.%2BN.%26aulast%3DGitlitz%26aufirst%3DB.%2BJ.%26aulast%3DBorad%26aufirst%3DM.%2BJ.%26aulast%3DGold%26aufirst%3DP.%2BJ.%26aulast%3DChiorean%26aufirst%3DE.%2BG.%26aulast%3DSpringett%26aufirst%3DG.%2BM.%26aulast%3DAbbas%26aufirst%3DR.%26aulast%3DAgarwal%26aufirst%3DS.%26aulast%3DBardy-Bouxin%26aufirst%3DN.%26aulast%3DHsyu%26aufirst%3DP.-H.%26aulast%3DLeip%26aufirst%3DE.%26aulast%3DTurnbull%26aufirst%3DK.%26aulast%3DZacharchuk%26aufirst%3DC.%26aulast%3DMessersmith%26aufirst%3DW.%2BA.%26atitle%3DPhase%2520I%2520study%2520of%2520bosutinib%252C%2520a%2520src%252Fabl%2520tyrosine%2520kinase%2520inhibitor%252C%2520administered%2520to%2520patients%2520with%2520advanced%2520solid%2520tumors%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D2012%26volume%3D18%26spage%3D1092%26epage%3D1100%26doi%3D10.1158%2F1078-0432.CCR-11-2378" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref66"><div class="reference"><strong class="refLabel"><a href="#ref66" class="rightTabRefNumLink">66</a></strong><div class="NLM_citation" id="rightTab-cit66"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lee, S.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J.-K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ahn, M.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, D.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, J.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keam, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heo, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ahn, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choi, Y.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Min, H.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jeon, Y.-K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, K.</span></span> <span> </span><span class="NLM_article-title">A phase II study of vandetanib in patients with non-small cell lung cancer harboring RET rearrangement</span>. <i>J. Clin. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>34</i></span>,  <span class="NLM_fpage">9013</span>– <span class="NLM_lpage">9013</span>, <span class="refDoi"> DOI: 10.1200/JCO.2016.34.15_suppl.9013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=10.1200%2FJCO.2016.34.15_suppl.9013" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=2016&pages=9013-9013&author=S.-H.+Leeauthor=J.-K.+Leeauthor=M.-J.+Ahnauthor=D.-W.+Kimauthor=J.-M.+Sunauthor=B.+Keamauthor=T.+M.+Kimauthor=D.+S.+Heoauthor=J.+S.+Ahnauthor=Y.-L.+Choiauthor=H.-S.+Minauthor=Y.-K.+Jeonauthor=K.+Park&title=A+phase+II+study+of+vandetanib+in+patients+with+non-small+cell+lung+cancer+harboring+RET+rearrangement&doi=10.1200%2FJCO.2016.34.15_suppl.9013"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit66&amp;dbid=16384&amp;doi=10.1200%2FJCO.2016.34.15_suppl.9013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1200%252FJCO.2016.34.15_suppl.9013%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DS.-H.%26aulast%3DLee%26aufirst%3DJ.-K.%26aulast%3DAhn%26aufirst%3DM.-J.%26aulast%3DKim%26aufirst%3DD.-W.%26aulast%3DSun%26aufirst%3DJ.-M.%26aulast%3DKeam%26aufirst%3DB.%26aulast%3DKim%26aufirst%3DT.%2BM.%26aulast%3DHeo%26aufirst%3DD.%2BS.%26aulast%3DAhn%26aufirst%3DJ.%2BS.%26aulast%3DChoi%26aufirst%3DY.-L.%26aulast%3DMin%26aufirst%3DH.-S.%26aulast%3DJeon%26aufirst%3DY.-K.%26aulast%3DPark%26aufirst%3DK.%26atitle%3DA%2520phase%2520II%2520study%2520of%2520vandetanib%2520in%2520patients%2520with%2520non-small%2520cell%2520lung%2520cancer%2520harboring%2520RET%2520rearrangement%26jtitle%3DJ.%2520Clin.%2520Oncol.%26date%3D2016%26volume%3D34%26spage%3D9013%26epage%3D9013%26doi%3D10.1200%2FJCO.2016.34.15_suppl.9013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref67"><div class="reference"><strong class="refLabel"><a href="#ref67" class="rightTabRefNumLink">67</a></strong><div class="NLM_citation" id="rightTab-cit67"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gao, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dai, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ou, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heymach, J. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gold, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minna, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roth, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hofstetter, W. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Swisher, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fang, B.</span></span> <span> </span><span class="NLM_article-title">Selective antitumor activity of ibrutinib in EGFR-mutant non-small cell lung cancer cells</span>. <i>J. Natl. Cancer Inst.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>106</i></span>, <span class="NLM_elocation-id">dju204</span> <span class="refDoi"> DOI: 10.1093/jnci/dju204</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=10.1093%2Fjnci%2Fdju204" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=25214559" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=106&publication_year=2014&author=W.+Gaoauthor=M.+Wangauthor=L.+Wangauthor=H.+Luauthor=S.+Wuauthor=B.+Daiauthor=Z.+Ouauthor=L.+Zhangauthor=J.+V.+Heymachauthor=K.+A.+Goldauthor=J.+Minnaauthor=J.+A.+Rothauthor=W.+L.+Hofstetterauthor=S.+G.+Swisherauthor=B.+Fang&title=Selective+antitumor+activity+of+ibrutinib+in+EGFR-mutant+non-small+cell+lung+cancer+cells&doi=10.1093%2Fjnci%2Fdju204"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit67&amp;dbid=16384&amp;doi=10.1093%2Fjnci%2Fdju204&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fjnci%252Fdju204%26sid%3Dliteratum%253Aachs%26aulast%3DGao%26aufirst%3DW.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DLu%26aufirst%3DH.%26aulast%3DWu%26aufirst%3DS.%26aulast%3DDai%26aufirst%3DB.%26aulast%3DOu%26aufirst%3DZ.%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DHeymach%26aufirst%3DJ.%2BV.%26aulast%3DGold%26aufirst%3DK.%2BA.%26aulast%3DMinna%26aufirst%3DJ.%26aulast%3DRoth%26aufirst%3DJ.%2BA.%26aulast%3DHofstetter%26aufirst%3DW.%2BL.%26aulast%3DSwisher%26aufirst%3DS.%2BG.%26aulast%3DFang%26aufirst%3DB.%26atitle%3DSelective%2520antitumor%2520activity%2520of%2520ibrutinib%2520in%2520EGFR-mutant%2520non-small%2520cell%2520lung%2520cancer%2520cells%26jtitle%3DJ.%2520Natl.%2520Cancer%2520Inst.%26date%3D2014%26volume%3D106%26doi%3D10.1093%2Fjnci%2Fdju204" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref68"><div class="reference"><strong class="refLabel"><a href="#ref68" class="rightTabRefNumLink">68</a></strong><div class="NLM_citation" id="rightTab-cit68"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Metz, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, E. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soni, N. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Merta, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kifle, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hajduk, P. J.</span></span> <span> </span><span class="NLM_article-title">Navigating the kinome</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">200</span>– <span class="NLM_lpage">202</span>, <span class="refDoi"> DOI: 10.1038/nchembio.530</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=10.1038%2Fnchembio.530" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=21336281" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=1%3ACAS%3A528%3ADC%252BC3MXit1aitrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2011&pages=200-202&author=J.+T.+Metzauthor=E.+F.+Johnsonauthor=N.+B.+Soniauthor=P.+J.+Mertaauthor=L.+Kifleauthor=P.+J.+Hajduk&title=Navigating+the+kinome&doi=10.1038%2Fnchembio.530"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit68R"><div class="casContent"><span class="casTitleNuber">68</span><div class="casTitle"><span class="NLM_cas:atitle">Navigating the kinome</span></div><div class="casAuthors">Metz, James T.; Johnson, Eric F.; Soni, Niru B.; Merta, Philip J.; Kifle, Lemma; Hajduk, Philip J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">200-202</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Although it is increasingly being recognized that drug-target interaction networks can be powerful tools for the interrogation of systems biol. and the rational design of multitargeted drugs, there is no generalized, statistically validated approach to harmonizing sequence-dependent and pharmacol.-dependent networks.  Here we demonstrate the creation of a comprehensive kinome interaction network based not only on sequence comparisons but also on multiple pharmacol. parameters derived from activity profiling data.  The framework described for statistical interpretation of these network connections also enables rigorous investigation of chemotype-specific interaction networks, which is crit. for multitargeted drug design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoOgzF9Q0eH87Vg90H21EOLACvtfcHk0ljDs6Q7fmuUhA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXit1aitrc%253D&md5=09234ec451a4918e5aca6d65e0b65aee</span></div><a href="/servlet/linkout?suffix=cit68&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.530&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.530%26sid%3Dliteratum%253Aachs%26aulast%3DMetz%26aufirst%3DJ.%2BT.%26aulast%3DJohnson%26aufirst%3DE.%2BF.%26aulast%3DSoni%26aufirst%3DN.%2BB.%26aulast%3DMerta%26aufirst%3DP.%2BJ.%26aulast%3DKifle%26aufirst%3DL.%26aulast%3DHajduk%26aufirst%3DP.%2BJ.%26atitle%3DNavigating%2520the%2520kinome%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2011%26volume%3D7%26spage%3D200%26epage%3D202%26doi%3D10.1038%2Fnchembio.530" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref69"><div class="reference"><strong class="refLabel"><a href="#ref69" class="rightTabRefNumLink">69</a></strong><div class="NLM_citation" id="rightTab-cit69"><span><span class="NLM_contrib-group"><span class="NLM_string-name">He, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garcia, E. A.</span></span> <span> </span><span class="NLM_article-title">Learning from imbalanced data</span>. <i>IEEE Trans. Knowl. Data. Eng.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">1263</span>– <span class="NLM_lpage">1284</span>, <span class="refDoi"> DOI: 10.1109/TKDE.2008.239</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=10.1109%2FTKDE.2008.239" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2009&pages=1263-1284&author=H.+Heauthor=E.+A.+Garcia&title=Learning+from+imbalanced+data&doi=10.1109%2FTKDE.2008.239"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit69&amp;dbid=16384&amp;doi=10.1109%2FTKDE.2008.239&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1109%252FTKDE.2008.239%26sid%3Dliteratum%253Aachs%26aulast%3DHe%26aufirst%3DH.%26aulast%3DGarcia%26aufirst%3DE.%2BA.%26atitle%3DLearning%2520from%2520imbalanced%2520data%26jtitle%3DIEEE%2520Trans.%2520Knowl.%2520Data.%2520Eng.%26date%3D2009%26volume%3D21%26spage%3D1263%26epage%3D1284%26doi%3D10.1109%2FTKDE.2008.239" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref70"><div class="reference"><strong class="refLabel"><a href="#ref70" class="rightTabRefNumLink">70</a></strong><div class="NLM_citation" id="rightTab-cit70"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ma, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheridan, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liaw, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dahl, G. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Svetnik, V.</span></span> <span> </span><span class="NLM_article-title">Deep neural nets as a method for quantitative structure-activity relationships</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">263</span>– <span class="NLM_lpage">274</span>, <span class="refDoi"> DOI: 10.1021/ci500747n</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci500747n" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvFGns70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2015&pages=263-274&author=J.+Maauthor=R.+P.+Sheridanauthor=A.+Liawauthor=G.+E.+Dahlauthor=V.+Svetnik&title=Deep+neural+nets+as+a+method+for+quantitative+structure-activity+relationships&doi=10.1021%2Fci500747n"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit70R"><div class="casContent"><span class="casTitleNuber">70</span><div class="casTitle"><span class="NLM_cas:atitle">Deep Neural Nets as a Method for Quantitative Structure-Activity Relationships</span></div><div class="casAuthors">Ma, Junshui; Sheridan, Robert P.; Liaw, Andy; Dahl, George E.; Svetnik, Vladimir</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">263-274</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Neural networks were widely used for quant. structure-activity relationships (QSAR) in the 1990s.  Because of various practical issues (e.g., slow on large problems, difficult to train, prone to overfitting, etc.), they were superseded by more robust methods like support vector machine (SVM) and random forest (RF), which arose in the early 2000s.  The last 10 years has witnessed a revival of neural networks in the machine learning community thanks to new methods for preventing overfitting, more efficient training algorithms, and advancements in computer hardware.  In particular, deep neural nets (DNNs), i.e. neural nets with more than one hidden layer, have found great successes in many applications, such as computer vision and natural language processing.  Here we show that DNNs can routinely make better prospective predictions than RF on a set of large diverse QSAR data sets that are taken from Merck's drug discovery effort.  The no. of adjustable parameters needed for DNNs is fairly large, but our results show that it is not necessary to optimize them for individual data sets, and a single set of recommended parameters can achieve better performance than RF for most of the data sets we studied.  The usefulness of the parameters is demonstrated on addnl. data sets not used in the calibration.  Although training DNNs is still computationally intensive, using graphical processing units (GPUs) can make this issue manageable.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrwVMQ-gQwZd7Vg90H21EOLACvtfcHk0ljtM4-1ABGAJw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvFGns70%253D&md5=2c6a3f33f85a4abc32ebc33108426667</span></div><a href="/servlet/linkout?suffix=cit70&amp;dbid=16384&amp;doi=10.1021%2Fci500747n&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci500747n%26sid%3Dliteratum%253Aachs%26aulast%3DMa%26aufirst%3DJ.%26aulast%3DSheridan%26aufirst%3DR.%2BP.%26aulast%3DLiaw%26aufirst%3DA.%26aulast%3DDahl%26aufirst%3DG.%2BE.%26aulast%3DSvetnik%26aufirst%3DV.%26atitle%3DDeep%2520neural%2520nets%2520as%2520a%2520method%2520for%2520quantitative%2520structure-activity%2520relationships%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2015%26volume%3D55%26spage%3D263%26epage%3D274%26doi%3D10.1021%2Fci500747n" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref71"><div class="reference"><strong class="refLabel"><a href="#ref71" class="rightTabRefNumLink">71</a></strong><div class="NLM_citation" id="rightTab-cit71"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rogers, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hahn, M.</span></span> <span> </span><span class="NLM_article-title">Extended-connectivity fingerprints</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>50</i></span>,  <span class="NLM_fpage">742</span>– <span class="NLM_lpage">754</span>, <span class="refDoi"> DOI: 10.1021/ci100050t</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci100050t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=1%3ACAS%3A528%3ADC%252BC3cXlt1Onsbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2010&pages=742-754&author=D.+Rogersauthor=M.+Hahn&title=Extended-connectivity+fingerprints&doi=10.1021%2Fci100050t"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit71R"><div class="casContent"><span class="casTitleNuber">71</span><div class="casTitle"><span class="NLM_cas:atitle">Extended-Connectivity Fingerprints</span></div><div class="casAuthors">Rogers, David; Hahn, Mathew</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">742-754</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Extended-connectivity fingerprints (ECFPs) are a novel class of topol. fingerprints for mol. characterization.  Historically, topol. fingerprints were developed for substructure and similarity searching.  ECFPs were developed specifically for structure-activity modeling.  ECFPs are circular fingerprints with a no. of useful qualities: they can be very rapidly calcd.; they are not predefined and can represent an essentially infinite no. of different mol. features (including stereochem. information); their features represent the presence of particular substructures, allowing easier interpretation of anal. results; and the ECFP algorithm can be tailored to generate different types of circular fingerprints, optimized for different uses.  While the use of ECFPs has been widely adopted and validated, a description of their implementation has not previously been presented in the literature.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGra43orI-Y7lbVg90H21EOLACvtfcHk0ljtM4-1ABGAJw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXlt1Onsbg%253D&md5=cd6c736cd7a3d280b67f5316acce8006</span></div><a href="/servlet/linkout?suffix=cit71&amp;dbid=16384&amp;doi=10.1021%2Fci100050t&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci100050t%26sid%3Dliteratum%253Aachs%26aulast%3DRogers%26aufirst%3DD.%26aulast%3DHahn%26aufirst%3DM.%26atitle%3DExtended-connectivity%2520fingerprints%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2010%26volume%3D50%26spage%3D742%26epage%3D754%26doi%3D10.1021%2Fci100050t" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref72"><div class="reference"><strong class="refLabel"><a href="#ref72" class="rightTabRefNumLink">72</a></strong><div class="NLM_citation" id="rightTab-cit72"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nair, V.</span>; <span class="NLM_string-name">Hinton, G. E.</span></span><span> </span><span class="NLM_article-title">Rectified linear units improve restricted Boltzmann machines</span> <i>Proceedings of the 27th International Conference on Machine Learning (ICML-10)</i>; <span class="NLM_publisher-name">International Machine Learning Society</span>, <span class="NLM_year">2010</span>; pp  <span class="NLM_fpage">807</span>– <span class="NLM_lpage">814</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2010&pages=807-814&author=V.+Nair&author=G.+E.+Hinton&title=Proceedings+of+the+27th+International+Conference+on+Machine+Learning+%28ICML-10%29"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit72&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DNair%26aufirst%3DV.%26atitle%3DRectified%2520linear%2520units%2520improve%2520restricted%2520Boltzmann%2520machines%26btitle%3DProceedings%2520of%2520the%252027th%2520International%2520Conference%2520on%2520Machine%2520Learning%2520%2528ICML-10%2529%26pub%3DInternational%2520Machine%2520Learning%2520Society%26date%3D2010%26spage%3D807%26epage%3D814" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref73"><div class="reference"><strong class="refLabel"><a href="#ref73" class="rightTabRefNumLink">73</a></strong><div class="NLM_citation" id="rightTab-cit73"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hinton, G. E.</span></span> <span> </span><span class="NLM_article-title">A Practical Guide to Training Restricted Boltzmann Machines</span>. In  <i>Neural Networks: Tricks of the Trade</i>, <span class="NLM_edition">2</span>nd ed.; <span class="NLM_contrib-group"><span class="NLM_string-name">Montavon, G.</span>, <span class="NLM_string-name">Orr, G. B.</span>, <span class="NLM_string-name">Müller, K.-R.</span></span>, Eds.; <span class="NLM_publisher-name">Springer</span>: <span class="NLM_publisher-loc">Berlin</span>, <span class="NLM_year">2012</span>; pp  <span class="NLM_fpage">599</span>– <span class="NLM_lpage">619</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=10.1007%2F978-3-642-35289-8_32" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2012&pages=599-619&author=G.+E.+Hintonauthor=G.+Montavon&author=G.+B.+Orr&author=K.-R.+M%C3%BCller&title=Neural+Networks%3A+Tricks+of+the+Trade"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit73&amp;dbid=16384&amp;doi=10.1007%2F978-3-642-35289-8_32&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1007%252F978-3-642-35289-8_32%26sid%3Dliteratum%253Aachs%26aulast%3DHinton%26aufirst%3DG.%2BE.%26atitle%3DA%2520Practical%2520Guide%2520to%2520Training%2520Restricted%2520Boltzmann%2520Machines%26btitle%3DNeural%2520Networks%253A%2520Tricks%2520of%2520the%2520Trade%26aulast%3DMontavon%26aufirst%3DG.%26pub%3DSpringer%26date%3D2012%26spage%3D599%26epage%3D619" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref74"><div class="reference"><strong class="refLabel"><a href="#ref74" class="rightTabRefNumLink">74</a></strong><div class="NLM_citation" id="rightTab-cit74"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Prechelt, L.</span></span> <span> </span><span class="NLM_article-title">Early Stopping—But When?</span> In  <i>Neural Networks: Tricks of the Trade</i>, <span class="NLM_edition">2</span>nd ed.; <span class="NLM_contrib-group"><span class="NLM_string-name">Montavon, G.</span>, <span class="NLM_string-name">Orr, G. B.</span>, <span class="NLM_string-name">Müller, K.-R.</span></span>, Eds.; <span class="NLM_publisher-name">Springer</span>: <span class="NLM_publisher-loc">Berlin</span>, <span class="NLM_year">2012</span>; pp  <span class="NLM_fpage">53</span>– <span class="NLM_lpage">67</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;key=10.1007%2F978-3-642-35289-8_5" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2012&pages=53-67&author=L.+Precheltauthor=G.+Montavon&author=G.+B.+Orr&author=K.-R.+M%C3%BCller&title=Neural+Networks%3A+Tricks+of+the+Trade"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit74&amp;dbid=16384&amp;doi=10.1007%2F978-3-642-35289-8_5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1007%252F978-3-642-35289-8_5%26sid%3Dliteratum%253Aachs%26aulast%3DPrechelt%26aufirst%3DL.%26atitle%3DEarly%2520Stopping%25E2%2580%2594But%2520When%253F%26btitle%3DNeural%2520Networks%253A%2520Tricks%2520of%2520the%2520Trade%26aulast%3DMontavon%26aufirst%3DG.%26pub%3DSpringer%26date%3D2012%26spage%3D53%26epage%3D67" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref75"><div class="reference"><strong class="refLabel"><a href="#ref75" class="rightTabRefNumLink">75</a></strong><div class="NLM_citation" id="rightTab-cit75"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kingma, D. P.</span>; <span class="NLM_string-name">Ba, J.</span></span> <span> </span><span class="NLM_article-title">Adam: A method for stochastic optimization</span>.  <i>arXiv:1412.6980</i>, <span class="NLM_year">2014</span>. arXiv.org e-Print archive.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2014&author=D.+P.+Kingma&author=J.+Ba&title=Adam%3A+A+method+for+stochastic+optimization"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit75&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dsimple%26sid%3Dliteratum%253Aachs%26aulast%3DKingma%26aufirst%3DD.%2BP.%26atitle%3DAdam%253A%2520A%2520method%2520for%2520stochastic%2520optimization%26jtitle%3DarXiv%253A1412.6980%26date%3D2014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref76"><div class="reference"><strong class="refLabel"><a href="#ref76" class="rightTabRefNumLink">76</a></strong><div class="NLM_citation" id="rightTab-cit76"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Srivastava, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hinton, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krizhevsky, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sutskever, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salakhutdinov, R.</span></span> <span> </span><span class="NLM_article-title">Dropout: A simple way to prevent neural networks from overfitting</span>. <i>J. Mach. Learn. Res.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">1929</span>– <span class="NLM_lpage">1958</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2014&pages=1929-1958&author=N.+Srivastavaauthor=G.+Hintonauthor=A.+Krizhevskyauthor=I.+Sutskeverauthor=R.+Salakhutdinov&title=Dropout%3A+A+simple+way+to+prevent+neural+networks+from+overfitting"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit76&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DSrivastava%26aufirst%3DN.%26aulast%3DHinton%26aufirst%3DG.%26aulast%3DKrizhevsky%26aufirst%3DA.%26aulast%3DSutskever%26aufirst%3DI.%26aulast%3DSalakhutdinov%26aufirst%3DR.%26atitle%3DDropout%253A%2520A%2520simple%2520way%2520to%2520prevent%2520neural%2520networks%2520from%2520overfitting%26jtitle%3DJ.%2520Mach.%2520Learn.%2520Res.%26date%3D2014%26volume%3D15%26spage%3D1929%26epage%3D1958" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i29"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b00855">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_32235"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a class="ext-link" href="/">ACS Publications website</a> at DOI: <a class="ext-link" href="/doi/abs/10.1021/acs.jmedchem.9b00855">10.1021/acs.jmedchem.9b00855</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Table S1 of kinase information (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b00855/suppl_file/jm9b00855_si_001.xlsx">XLSX</a>)</p></li><li><p class="inline">Table S2 of prediction performance comparisons between MTDNN and RF (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b00855/suppl_file/jm9b00855_si_002.xlsx">XLSX</a>)</p></li><li><p class="inline">Table S3 of experimental data (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b00855/suppl_file/jm9b00855_si_003.xlsx">XLSX</a>)</p></li><li><p class="inline">Molecular formula strings and the associated biochemical and biological data (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b00855/suppl_file/jm9b00855_si_004.csv">CSV</a>)</p></li><li><p class="inline">Molecular information (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b00855/suppl_file/jm9b00855_si_005.pdf">PDF</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00855/suppl_file/jm9b00855_si_001.xlsx">jm9b00855_si_001.xlsx (50.75 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00855/suppl_file/jm9b00855_si_002.xlsx">jm9b00855_si_002.xlsx (57.6 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00855/suppl_file/jm9b00855_si_003.xlsx">jm9b00855_si_003.xlsx (2.04 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00855/suppl_file/jm9b00855_si_004.csv">jm9b00855_si_004.csv (281.82 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00855/suppl_file/jm9b00855_si_005.pdf">jm9b00855_si_005.pdf (200.65 kb)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.9b00855&amp;pbContext=%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.9b00855%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2020.63.issue-16%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.9b00855" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                6MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"679906ebbaad3d5a","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
